<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2018/02/08/261628.atom" />
<meta name="HW.pisa" content="biorxiv;261628v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Generating genomic platforms to study Candida albicans pathogenesis" />
<meta name="DC.Identifier" content="10.1101/261628" />
<meta name="DC.Date" content="2018-02-08" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2018, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="The advent of the genomic era has made elucidating gene function at large scale a pressing challenge. ORFeome collections, whereby almost all ORFs of a given species are cloned and can be subsequently leveraged in multiple functional genomic approaches, represent valuable resources towards this endeavor. Here we provide novel, genome-scale tools for the study of Candida albicans , a commensal yeast that is also responsible for frequent superficial and disseminated infections in humans. We have generated an ORFeome collection composed of 5,102 ORFs cloned in a Gateway™ donor vector, representing 83% of the currently annotated coding sequences of C. albicans . Sequencing data of the cloned ORFs are available in the CandidaOrfDB database at &lt;http://candidaorfeome.eu&gt;. We also engineered 49 expression vectors with a choice of promoters, tags, and selection markers and demonstrated their applicability to the study of target ORFs transferred from the C. albicans ORFeome. In addition, the use of the ORFeome in the detection of protein-protein interaction was demonstrated. Mating-compatible strains as well as Gateway™-compatible two-hybrid vectors were engineered, validated and used in a proof of concept experiment. These unique and valuable resources should greatly facilitate future functional studies in C. albicans and the elucidation of mechanisms that underlie its pathogenicity." />
<meta name="DC.Contributor" content="Mélanie Legrand" />
<meta name="DC.Contributor" content="Sophie Bachellier-Bassi" />
<meta name="DC.Contributor" content="Keunsook K. Lee" />
<meta name="DC.Contributor" content="Yogesh Chaudhari" />
<meta name="DC.Contributor" content="Hélène Tournu" />
<meta name="DC.Contributor" content="Laurence Arbogast" />
<meta name="DC.Contributor" content="Hélène Boyer" />
<meta name="DC.Contributor" content="Murielle Chauvel" />
<meta name="DC.Contributor" content="Vitor Cabral" />
<meta name="DC.Contributor" content="Corinne Maufrais" />
<meta name="DC.Contributor" content="Audrey Nesseir" />
<meta name="DC.Contributor" content="Irena Maslanka" />
<meta name="DC.Contributor" content="Emmanuelle Permal" />
<meta name="DC.Contributor" content="Tristan Rossignol" />
<meta name="DC.Contributor" content="Louise A. Walker" />
<meta name="DC.Contributor" content="Ute Zeidler" />
<meta name="DC.Contributor" content="Sadri Znaidi" />
<meta name="DC.Contributor" content="Floris Schoeters" />
<meta name="DC.Contributor" content="Charlotte Majgier" />
<meta name="DC.Contributor" content="Renaud A. Julien" />
<meta name="DC.Contributor" content="Laurence Ma" />
<meta name="DC.Contributor" content="Magali Tichit" />
<meta name="DC.Contributor" content="Christiane Bouchier" />
<meta name="DC.Contributor" content="Patrick Van Dijck" />
<meta name="DC.Contributor" content="Carol A. Munro" />
<meta name="DC.Contributor" content="Christophe d’Enfert" />
<meta name="article:published_time" content="2018-02-08" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Generating genomic platforms to study Candida albicans pathogenesis" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;ABSTRACT&lt;/h3&gt;
&lt;p&gt;The advent of the genomic era has made elucidating gene function at large scale a pressing challenge. ORFeome collections, whereby almost all ORFs of a given species are cloned and can be subsequently leveraged in multiple functional genomic approaches, represent valuable resources towards this endeavor. Here we provide novel, genome-scale tools for the study of &lt;i&gt;Candida albicans&lt;/i&gt;, a commensal yeast that is also responsible for frequent superficial and disseminated infections in humans. We have generated an ORFeome collection composed of 5,102 ORFs cloned in a Gateway&lt;sup&gt;™&lt;/sup&gt; donor vector, representing 83% of the currently annotated coding sequences of &lt;i&gt;C. albicans&lt;/i&gt;. Sequencing data of the cloned ORFs are available in the CandidaOrfDB database at http://candidaorfeome.eu. We also engineered 49 expression vectors with a choice of promoters, tags, and selection markers and demonstrated their applicability to the study of target ORFs transferred from the &lt;i&gt;C. albicans&lt;/i&gt; ORFeome. In addition, the use of the ORFeome in the detection of protein-protein interaction was demonstrated. Mating-compatible strains as well as Gateway&lt;sup&gt;™&lt;/sup&gt;-compatible two-hybrid vectors were engineered, validated and used in a proof of concept experiment. These unique and valuable resources should greatly facilitate future functional studies in &lt;i&gt;C. albicans&lt;/i&gt; and the elucidation of mechanisms that underlie its pathogenicity.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2018/01/01" />
<meta name="citation_mjid" content="biorxiv;261628v1" />
<meta name="citation_id" content="261628v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/261628v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/261628v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/261628v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261628.full.pdf" />
<meta name="citation_doi" content="10.1101/261628" />
<meta name="citation_num_pages" content="34" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="261628" />
<meta name="citation_author" content="Mélanie Legrand" />
<meta name="citation_author_institution" content="Fungal Biology and Pathogenicity Unit, Department of Mycology, Institut Pasteur" />
<meta name="citation_author" content="Sophie Bachellier-Bassi" />
<meta name="citation_author_institution" content="Fungal Biology and Pathogenicity Unit, Department of Mycology, Institut Pasteur" />
<meta name="citation_author" content="Keunsook K. Lee" />
<meta name="citation_author_institution" content="MRC Centre for Medical Mycology, Institute of Medical Sciences, University of Aberdeen" />
<meta name="citation_author" content="Yogesh Chaudhari" />
<meta name="citation_author_institution" content="MRC Centre for Medical Mycology, Institute of Medical Sciences, University of Aberdeen" />
<meta name="citation_author" content="Hélène Tournu" />
<meta name="citation_author_institution" content="VIB-KU Leuven Center for Microbiology" />
<meta name="citation_author_institution" content="Laboratory of Molecular Cell Biology" />
<meta name="citation_author" content="Laurence Arbogast" />
<meta name="citation_author_institution" content="Fungal Biology and Pathogenicity Unit, Department of Mycology, Institut Pasteur" />
<meta name="citation_author" content="Hélène Boyer" />
<meta name="citation_author_institution" content="Fungal Biology and Pathogenicity Unit, Department of Mycology, Institut Pasteur" />
<meta name="citation_author" content="Murielle Chauvel" />
<meta name="citation_author_institution" content="Fungal Biology and Pathogenicity Unit, Department of Mycology, Institut Pasteur" />
<meta name="citation_author" content="Vitor Cabral" />
<meta name="citation_author_institution" content="Fungal Biology and Pathogenicity Unit, Department of Mycology, Institut Pasteur" />
<meta name="citation_author_institution" content="Univ. Paris Diderot, Sorbonne Paris Cité, Cellule Pasteur" />
<meta name="citation_author" content="Corinne Maufrais" />
<meta name="citation_author_institution" content="Fungal Biology and Pathogenicity Unit, Department of Mycology, Institut Pasteur" />
<meta name="citation_author_institution" content="Institut Pasteur - Bioinformatics and Biostatistics Hub - C3BI" />
<meta name="citation_author" content="Audrey Nesseir" />
<meta name="citation_author_institution" content="Fungal Biology and Pathogenicity Unit, Department of Mycology, Institut Pasteur" />
<meta name="citation_author_institution" content="Univ. Paris Diderot, Sorbonne Paris Cité, Cellule Pasteur" />
<meta name="citation_author" content="Irena Maslanka" />
<meta name="citation_author_institution" content="MRC Centre for Medical Mycology, Institute of Medical Sciences, University of Aberdeen" />
<meta name="citation_author" content="Emmanuelle Permal" />
<meta name="citation_author_institution" content="Fungal Biology and Pathogenicity Unit, Department of Mycology, Institut Pasteur" />
<meta name="citation_author" content="Tristan Rossignol" />
<meta name="citation_author_institution" content="Fungal Biology and Pathogenicity Unit, Department of Mycology, Institut Pasteur" />
<meta name="citation_author" content="Louise A. Walker" />
<meta name="citation_author_institution" content="MRC Centre for Medical Mycology, Institute of Medical Sciences, University of Aberdeen" />
<meta name="citation_author" content="Ute Zeidler" />
<meta name="citation_author_institution" content="Fungal Biology and Pathogenicity Unit, Department of Mycology, Institut Pasteur" />
<meta name="citation_author" content="Sadri Znaidi" />
<meta name="citation_author_institution" content="Fungal Biology and Pathogenicity Unit, Department of Mycology, Institut Pasteur" />
<meta name="citation_author" content="Floris Schoeters" />
<meta name="citation_author_institution" content="VIB-KU Leuven Center for Microbiology" />
<meta name="citation_author_institution" content="Laboratory of Molecular Cell Biology" />
<meta name="citation_author" content="Charlotte Majgier" />
<meta name="citation_author_institution" content="Modul-Bio; Parc Scientifique Luminy Biotech II" />
<meta name="citation_author" content="Renaud A. Julien" />
<meta name="citation_author_institution" content="Modul-Bio; Parc Scientifique Luminy Biotech II" />
<meta name="citation_author" content="Laurence Ma" />
<meta name="citation_author_institution" content="Institut Pasteur - Biomics Pole - CITECH - Paris" />
<meta name="citation_author" content="Magali Tichit" />
<meta name="citation_author_institution" content="Institut Pasteur - Biomics Pole - CITECH - Paris" />
<meta name="citation_author" content="Christiane Bouchier" />
<meta name="citation_author_institution" content="Institut Pasteur - Biomics Pole - CITECH - Paris" />
<meta name="citation_author" content="Patrick Van Dijck" />
<meta name="citation_author_institution" content="VIB-KU Leuven Center for Microbiology" />
<meta name="citation_author_institution" content="Laboratory of Molecular Cell Biology" />
<meta name="citation_author" content="Carol A. Munro" />
<meta name="citation_author_institution" content="MRC Centre for Medical Mycology, Institute of Medical Sciences, University of Aberdeen" />
<meta name="citation_author_email" content="christophe.denfert@pasteur.fr" />
<meta name="citation_author_email" content="c.a.munro@abdn.ac.uk" />
<meta name="citation_author" content="Christophe d’Enfert" />
<meta name="citation_author_institution" content="Fungal Biology and Pathogenicity Unit, Department of Mycology, Institut Pasteur" />
<meta name="citation_author_email" content="christophe.denfert@pasteur.fr" />
<meta name="citation_author_email" content="c.a.munro@abdn.ac.uk" />
<meta name="citation_reference" content="Jones, T., Federspiel, N.A., Chibana, H., Dungan, J., Kalman, S., Magee, B.B., Newport, G., Thorstenson, Y.R., Agabian, N., Magee, P.T. et al. (2004) The diploid genome sequence of Candida albicans. Proc. Natl. Acad. Sci. U. S. A., 101, 7329–7334." />
<meta name="citation_reference" content="Muzzey, D., Schwartz, K., Weissman, J.S. and Sherlock, G. (2013) Assembly of a phased diploid Candida albicans pathogenomics albicans genome facilitates allele-specific measurements and provides a simple model for repeat and indel structure. Genome biology, 14, R97." />
<meta name="citation_reference" content="d’;Enfert, C., Goyard, S., Rodriguez-Arnaveilhe, S., Frangeul, L., Jones, L., Tekaia, F., Bader, O., Albrecht, A., Castillo, L., Dominguez, A. et al. (2005) CandidaDB: a genome database for Candida albicans pathogenomics. Nucl. Acids Res., 33 Database Issue, D353–357." />
<meta name="citation_reference" content="Braun, B.R., van Het Hoog, M., d’;Enfert, C., Martchenko, M., Dungan, J., Kuo, A., Inglis, D.O., Uhl, M.A., Hogues, H., Berriman, M. et al. (2005) A Human-Curated Annotation of the Candida albicans Genome. PLoS Genet., 1, 36–57." />
<meta name="citation_reference" content="Skrzypek, M.S., Binkley, J., Binkley, G., Miyasato, S.R., Simison, M. and Sherlock, G. (2017) The Candida Genome Database (CGD): incorporation of Assembly 22, systematic identifiers and visualization of high throughput sequencing data. Nucleic Acids Res, 45, D592–D596." />
<meta name="citation_reference" content="Roemer, T., Jiang, B., Davison, J., Ketela, T., Veillette, K., Breton, A., Tandia, F., Linteau, A., Sillaots, S., Marta, C. et al. (2003) Large-scale essential gene identification in Candida albicans and applications to antifungal drug discovery. Mol. Microbiol., 50, 167–181." />
<meta name="citation_reference" content="Xu, D., Jiang, B., Ketela, T., Lemieux, S., Veillette, K., Martel, N., Davison, J., Sillaots, S., Trosok, S., Bachewich, C. et al. (2007) Genome-wide fitness test and mechanism-of-action studies of inhibitory compounds in Candida albicans. PLoS pathogens, 3, e92." />
<meta name="citation_reference" content="Homann, O.R., Dea, J., Noble, S.M. and Johnson, A.D. (2009) A phenotypic profile of the Candida albicans regulatory network. PLoS genetics, 5, e1000783." />
<meta name="citation_reference" content="Noble, S.M., French, S., Kohn, L.A., Chen, V. and Johnson, A.D. (2010) Systematic screens of a Candida albicans homozygous deletion library decouple morphogenetic switching and pathogenicity. Nat Genet, 42, 590–598." />
<meta name="citation_reference" content="Sahni, N., Yi, S., Daniels, K.J., Huang, G., Srikantha, T. and Soll, D.R. (2010) Tec1 mediates the pheromone response of the white phenotype of Candida albicans: insights into the evolution of new signal transduction pathways. PLoS Biol, 8, e1000363." />
<meta name="citation_reference" content="Chauvel, M., Nesseir, A., Cabral, V., Znaidi, S., Goyard, S., Bachellier-Bassi, S., Firon, A., Legrand, M., Diogo, D., Naulleau, C. et al. (2012) A versatile overexpression strategy in the pathogenic yeast Candida albicans: identification of regulators of morphogenesis and fitness. PloS one, 7, e45912." />
<meta name="citation_reference" content="Schillig, R. and Morschhauser, J. (2013) Analysis of a fungus-specific transcription factor family, the Candida albicans zinc cluster proteins, by artificial activation. Mol. Microbiol., 89, 1003–1017." />
<meta name="citation_reference" content="Rual, J.F., Hill, D.E. and Vidal, M. (2004) ORFeomeprojects: gateway between genomics and omics. Curr. Opin. Chem. Biol., 8, 20–25." />
<meta name="citation_reference" content="Ghaemmaghami, S., Huh, W.K., Bower, K., Howson, R.W., Belle, A., Dephoure, N., O’;Shea, E.K. and Weissman, J.S. (2003) Global analysis of protein expression in yeast. Nature, 425, 737–741." />
<meta name="citation_reference" content="Reboul, J., Vaglio, P., Rual, J.F., Lamesch, P., Martinez, M., Armstrong, C.M., Li, S., Jacotot, L., Bertin, N., Janky, R. et al. (2003) C. elegans ORFeome version 1.1: experimental verification of the genome annotation and resource for proteome-scale protein expression. Nat. Genet., 34, 35–41." />
<meta name="citation_reference" content="Gelperin, D.M., White, M.A., Wilkinson, M.L., Kon, Y., Kung, L.A., Wise, K.J., Lopez-Hoyo, N., Jiang, L., Piccirillo, S., Yu, H. et al.(2005) Biochemical and genetic analysis of the yeast proteome with a movable ORF collection. Genes Dev., 19, 2816–2826." />
<meta name="citation_reference" content="Matsuyama, A. and Yoshida, M. (2009) Systematic cloning of an ORFeome using the Gateway system. Methods Mol. Biol., 577, 11–24." />
<meta name="citation_reference" content="Rajagopala, S.V., Yamamoto, N., Zweifel, A.E., Nakamichi, T., Huang, H.K., Mendez-Rios,J.D., Franca-Koh, J., Boorgula, M.P., Fujita, K., Suzuki, K. et al.(2010) The Escherichia coliK-12 ORFeome: a resource for comparative molecular microbiology. BMC genomics, 11, 470." />
<meta name="citation_reference" content="Gong, W., Shen, Y.P., Ma, L.G., Pan, Y., Du, Y.L., Wang, D.H., Yang, J.Y., Hu, L.D., Liu, X.F., Dong, C.X. et al.(2004) Genome-wide ORFeome cloning and analysis of Arabidopsis transcription factor genes. Plant Physiol., 135, 773–782." />
<meta name="citation_reference" content="Underwood, B.A., Vanderhaeghen, R., Whitford, R., Town, C.D. and Hilson, P. (2006) Simultaneous high-throughput recombinational cloning of open reading frames in closed and open configurations. Plant Biotechnol J, 4, 317–324." />
<meta name="citation_reference" content="Grant, I.M., Balcha, D., Hao, T., Shen, Y., Trivedi, P., Patrushev, I., Fortriede, J.D., Karpinka, J.B., Liu, L., Zorn, A.M. et al.(2015) The Xenopus ORFeome: A resource that enables functional genomics. Dev. Biol., 408, 345–357." />
<meta name="citation_reference" content="Bischof, J., Bjorklund, M., Furger, E., Schertel, C., Taipale, J. and Basler, K. (2013) A versatile platform for creating a comprehensive UAS-ORFeome library in Drosophila. Development, 140, 2434–2442." />
<meta name="citation_reference" content="Dricot, A., Rual, J.F., Lamesch, P., Bertin, N., Dupuy, D., Hao, T., Lambert, C., Hallez, R., Delroisse, J.M., Vandenhaute, J. et al.(2004) Generation of the Brucella melitensisORFeome version 1.1. Genome Res., 14, 2201–2206." />
<meta name="citation_reference" content="Aguiar, J.C., LaBaer, J., Blair, P.L., Shamailova, V.Y., Koundinya, M., Russell, J.A., Huang, F., Mar, W., Anthony, R.M., Witney, A. et al.(2004) High-throughput generation of P. falciparum functional molecules by recombinational cloning. Genome Res., 14, 2076–2082." />
<meta name="citation_reference" content="Hauser, R., Ceol, A., Rajagopala, S.V., Mosca, R., Siszler, G., Wermke, N., Sikorski, P., Schwarz, F., Schick, M., Wuchty, S. et al.(2014) A second-generation protein-protein interaction network of Helicobacter pylori. Molecular &amp; cellular proteomics: MCP, 13, 1318–1329." />
<meta name="citation_reference" content="Maier, C.J., Maier, R.H., Virok, D.P., Maass, M., Hintner, H., Bauer, J.W. and Onder, K. (2012) Construction of a highly flexible and comprehensive gene collection representing the ORFeome of the human pathogen Chlamydia pneumoniae. BMC genomics, 13, 632." />
<meta name="citation_reference" content="Brandner, C.J., Maier, R.H., Henderson, D.S., Hintner, H., Bauer, J.W. and Onder, K. (2008) The ORFeome of Staphylococcus aureusv 1.1. BMC genomics, 9, 321." />
<meta name="citation_reference" content="von Brunn, A., Teepe, C., Simpson, J.C., Pepperkok, R., Friedel, C.C., Zimmer, R., Roberts, R., Baric, R. and Haas, J. (2007) Analysis of intraviral protein-protein interactions of the SARS coronavirus ORFeome. PloS one, 2, e459." />
<meta name="citation_reference" content="Lamesch, P., Li, N., Milstein, S., Fan, C., Hao, T., Szabo, G., Hu, Z., Venkatesan, K., Bethel, G., Martin, P. et al.(2007) hORFeomev3.1: a resource of human open reading frames representing over 10,000 human genes. Genomics, 89, 307–315." />
<meta name="citation_reference" content="Walhout, A.J., Temple, G.F., Brasch, M.A., Hartley, J.L., Lorson, M.A., van den Heuvel, S. and Vidal, M. (2000) GATEWAY recombinational cloning: application to the cloning of large numbers of open reading frames or ORFeomes. Methods Enzymol., 328, 575–592." />
<meta name="citation_reference" content="Cabral, V., Znaidi, S., Walker, L.A., Martin-Yken, H., Dague, E., Legrand, M., Lee, K., Chauvel, M., Firon, A., Rossignol, T. et al.(2014) Targeted changes of the cell wall proteome influence Candida albicans ability to form single- and multi-strain biofilms. PLoS pathogens, 10, e1004542." />
<meta name="citation_reference" content="Stynen, B., Van Dijck, P. and Tournu, H. (2010) A CUG codon adapted two-hybrid system for the pathogenic fungus Candida albicans. Nucleic Acids Res, 38, e184." />
<meta name="citation_reference" content="Hanahan, D. (1983) Studies on transformation of Escherichia coli with plasmids. J. Mol. Biol., 166, 557–580." />
<meta name="citation_reference" content="Cabral, V., Chauvel, M., Firon, A., Legrand, M., Nesseir, A., Bachellier-Bassi, S., Chaudhari, Y., Munro, C.A. and d’;Enfert, C. (2012) Modular Gene Over-expression Strategies for Candida albicans. Methods Mol Biol, 845, 227–244." />
<meta name="citation_reference" content="Gola, S., Martin, R., Walther, A., Dunkler, A. and Wendland, J. (2003) New modules for PCR-based gene targeting in Candida albicans: rapid and efficient gene targeting using 100 bp of flanking homology region. Yeast, 20, 1339–1347." />
<meta name="citation_reference" content="Liu, T.T., Znaidi, S., Barker, K.S., Xu, L., Homayouni, R., Saidane, S., Morschhauser, J., Nantel, A., Raymond, M. and Rogers, P.D. (2007) Genome-wide expression and location analyses of the Candida albicans Tac1p regulon. Eukaryotic cell, 6, 2122–2138." />
<meta name="citation_reference" content="Enjalbert, B., Rachini, A., Vediyappan, G., Pietrella, D., Spaccapelo, R., Vecchiarelli, A., Brown, A.J. and d’;Enfert, C. (2009) A multifunctional, synthetic Gaussia princeps luciferase reporter for live imaging of Candida albicans infections. Infect Immun, 77, 4847–4858." />
<meta name="citation_reference" content="Reuss, O., Vik, A., Kolter, R. and Morschhauser, J. (2004) The SAT1flipper, an optimized tool for gene disruption in Candida albicans. Gene, 341, 119–127." />
<meta name="citation_reference" content="Walther, A. and Wendland, J. (2008) PCR-based gene targeting in Candida albicans. Nat Protoc, 3, 1414–1421." />
<meta name="citation_reference" content="Park, Y.N. and Morschhauser, J. (2005) Tetracycline-inducible gene expression and gene deletion in Candida albicans. Eukaryotic cell, 4, 1328–1342." />
<meta name="citation_reference" content="Huang, G., Wang, H., Chou, S., Nie, X., Chen, J. and Liu, H. (2006) Bistable expression of WOR1, a master regulator of white-opaque switching in Candida albicans. Proc. Natl. Acad. Sci. U. S. A., 103, 12813–12818." />
<meta name="citation_reference" content="Liu, H., Kohler, J. and Fink, G.R. (1994) Suppression of hyphal formation in Candida albicansby mutation of a STE12 homolog. Science, 266, 1723–1726." />
<meta name="citation_reference" content="Bennett, R.J. and Johnson, A.D. (2003) Completion of a parasexual cycle in Candida albicans by induced chromosome loss in tetraploid strains. EMBO J., 22, 2505–2515." />
<meta name="citation_reference" content="van het Hoog, M., Rast, T.J., Martchenko, M., Grindle, S., Dignard, D., Hogues, H., Cuomo, C., Berriman, M., Scherer, S., Magee, B.B. et al.(2007) Assembly of the Candida albicans genome into sixteen supercontigs aligned on the eight chromosomes. Genome biology, 8, R52." />
<meta name="citation_reference" content="Fonzi, W.A. and Irwin, M.Y. (1993) Isogenic strain construction and gene mapping in Candida albicans. Genetics, 134, 717–728." />
<meta name="citation_reference" content="Skrzypek, M.S., Binkley, J., Binkley, G., Miyasato, S.R., Simison, M. and Sherlock, G. “Candida Genome Database” http://www.candidagenome.org/ (2015)" />
<meta name="citation_reference" content="Alberti, S., Gitler, A.D. and Lindquist, S. (2007) A suite of Gateway cloning vectors for high-throughput genetic analysis in Saccharomyces cerevisiae. Yeast, 24, 913–919." />
<meta name="citation_reference" content="Hallez, R., Letesson, J.J., Vandenhaute, J. and De Bolle, X. (2007) Gateway-based destination vectors for functional analyses of bacterial ORFeomes: application to the Min system in Brucella abortus. Appl. Environ. Microbiol., 73, 1375–1379." />
<meta name="citation_reference" content="Carlisle, P.L., Banerjee, M., Lazzell, A., Monteagudo, C., Lopez-Ribot, J.L. and Kadosh, D. (2009) Expression levels of a filament-specific transcriptional regulator are sufficient to determine Candida albicans morphology and virulence. Proc. Natl. Acad. Sci. U. S. A., 106, 599–604." />
<meta name="citation_reference" content="Zeidler, U., Lettner, T., Lassnig, C., Muller, M., Lajko, R., Hintner, H., Breitenbach, M. and Bito, A. (2009) UME6 is a crucial downstream target of other transcriptional regulators of true hyphal development in Candida albicans. FEMS yeast research, 9, 126–142." />
<meta name="citation_reference" content="Bennett, R.J. (2015) The parasexual lifestyle of Candida albicans. Curr. Opin. Microbiol., 28, 10–17." />
<meta name="citation_reference" content="Miller, M.G. and Johnson, A.D. (2002) White-opaque switching in Candida albicans is controlled by mating-type locus homeodomain proteins and allows efficient mating. Cell, 110, 293–302." />
<meta name="citation_reference" content="Zordan, R.E., Galgoczy, D.J. and Johnson, A.D. (2006) Epigenetic properties of white-opaque switching in Candida albicans are based on a self-sustaining transcriptional feedback loop. Proc. Natl. Acad. Sci. U. S. A., 103, 12807–12812." />
<meta name="citation_reference" content="Loll-Krippleber, R., Feri, A., Nguyen, M., Maufrais, C., Yansouni, J., d’;Enfert, C. and Legrand, M. (2015) A FACS-optimized screen identifies regulators of genome stability in Candida albicans. Eukaryotic cell, 14, 311–322." />
<meta name="citation_reference" content="Mitrovich, Q.M., Tuch, B.B., Guthrie, C. and Johnson, A.D. (2007) Computational and experimental approaches double the number of known introns in the pathogenic yeast Candida albicans. Genome Res., 17, 492–502." />
<meta name="citation_reference" content="Santos, M.A. and Tuite, M.F. (1995) The CUG codon is decoded in vivo as serine and not leucine in Candida albicans. Nucleic Acids Res, 23, 1481–1486." />
<meta name="citation_reference" content="Coste, A., Turner, V., Ischer, F., Morschhauser, J., Forche, A., Selmecki, A., Berman, J., Bille, J. and Sanglard, D. (2006) A mutation in Tac1p, a transcription factor regulating CDR1 and CDR2, is coupled with loss of heterozygosity at chromosome 5 to mediate antifungal resistance in Candida albicans. Genetics, 172, 2139–2156." />
<meta name="citation_reference" content="Holmes, A.R., Tsao, S., Ong, S.W., Lamping, E., Niimi, K., Monk, B.C., Niimi, M., Kaneko, A., Holland, B.R., Schmid, J. et al.(2006) Heterozygosity and functional allelic variation in the Candida albicans efflux pump genes CDR1 and CDR2. Mol. Microbiol., 62, 170–186." />
<meta name="citation_reference" content="Noble, S.M. and Johnson, A.D. (2005) Strains and strategies for large-scale gene deletion studies of the diploid human fungal pathogen Candida albicans. Eukaryotic cell, 4, 298–309." />
<meta name="twitter:title" content="Generating genomic platforms to study Candida albicans pathogenesis" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="The advent of the genomic era has made elucidating gene function at large scale a pressing challenge. ORFeome collections, whereby almost all ORFs of a given species are cloned and can be subsequently leveraged in multiple functional genomic approaches, represent valuable resources towards this endeavor. Here we provide novel, genome-scale tools for the study of Candida albicans , a commensal yeast that is also responsible for frequent superficial and disseminated infections in humans. We have generated an ORFeome collection composed of 5,102 ORFs cloned in a Gateway™ donor vector, representing 83% of the currently annotated coding sequences of C. albicans . Sequencing data of the cloned ORFs are available in the CandidaOrfDB database at &lt;http://candidaorfeome.eu&gt;. We also engineered 49 expression vectors with a choice of promoters, tags, and selection markers and demonstrated their applicability to the study of target ORFs transferred from the C. albicans ORFeome. In addition, the use of the ORFeome in the detection of protein-protein interaction was demonstrated. Mating-compatible strains as well as Gateway™-compatible two-hybrid vectors were engineered, validated and used in a proof of concept experiment. These unique and valuable resources should greatly facilitate future functional studies in C. albicans and the elucidation of mechanisms that underlie its pathogenicity." />
<meta name="og-title" property="og:title" content="Generating genomic platforms to study Candida albicans pathogenesis" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/261628v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="The advent of the genomic era has made elucidating gene function at large scale a pressing challenge. ORFeome collections, whereby almost all ORFs of a given species are cloned and can be subsequently leveraged in multiple functional genomic approaches, represent valuable resources towards this endeavor. Here we provide novel, genome-scale tools for the study of Candida albicans , a commensal yeast that is also responsible for frequent superficial and disseminated infections in humans. We have generated an ORFeome collection composed of 5,102 ORFs cloned in a Gateway™ donor vector, representing 83% of the currently annotated coding sequences of C. albicans . Sequencing data of the cloned ORFs are available in the CandidaOrfDB database at &lt;http://candidaorfeome.eu&gt;. We also engineered 49 expression vectors with a choice of promoters, tags, and selection markers and demonstrated their applicability to the study of target ORFs transferred from the C. albicans ORFeome. In addition, the use of the ORFeome in the detection of protein-protein interaction was demonstrated. Mating-compatible strains as well as Gateway™-compatible two-hybrid vectors were engineered, validated and used in a proof of concept experiment. These unique and valuable resources should greatly facilitate future functional studies in C. albicans and the elucidation of mechanisms that underlie its pathogenicity." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2018-02-08" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/261628v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/261628v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/261628v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/261628v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/80349" />
    <title>Generating genomic platforms to study Candida albicans pathogenesis | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__vup9YXfxWOBOGbthJCUtxdPS6xRY2OQ_vXqbojfxPAk__ziC21uj-cnpEf3LG0kjRMEQHSEtjX5ljVuosklz2DZ4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"vfGDvqepysWVQvV-kNRwztCgISsSokT2Mn5SY1YHjs4","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"80349","apath":"\/biorxiv\/early\/2018\/02\/08\/261628.atom","pisa":"biorxiv;261628v1","markup":[{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;261628\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;261628\/DC1"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;261628v1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;261628\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;261628\/DC1"}],"processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/261628v1","title":"Generating genomic platforms to study Candida albicans pathogenesis","identifier":"node\/80349"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00225+ErL82J30POX0dC2CRL6NjJ3pdmaMp0L7Bfvl1YoI5y8yL3eZBu5Tk1ngQYvQjG+OxvA0fh1kxSX+hXEM0Evm00h3hiSuPIjwPn86q\\\\\\\/uD2sq3LsUJPmj9PZIZOf5RRoKGcPfVHKp87sCxBBa4E7aBPjOioyhIn7NYBJz42stXTYiD\\\\\\\/FlqjvKL2W1HrYvXm0yWyYCypuT9\\\\\\\/rsLwJLW69TJdgAMlpFMslOnbjWVXm72RzA\\\\\\\/sMLMlOEeTSBTVupbQgoo\\\\\\\/fzR+t7hKkGvmaUntiBVe3VAhrnmJ7Y6yAfYh5hzFPvX2sykGf45isIgiXDgZP9dvcWHWykSlFqQjbBVh\\\\\\\/f9KFTnYPudRWqq+ogk6JdqXyiBc1wsowgWAeG0KsOenazmeXb7tmlmqNoFY0wWflNTnFLKRvc0T4UbNPXXFge3Za4X\\\\\\\/i4ClZwvRZSy5\\\\\\\/tnznkCESvPkHh3qmMn+ZaUQS46dV4hKlCf1D1HEJJSnQmGOeG4612S6jTY81mADn3s0al4ijESoAtmF4MGodI9xPZkzG6wnm5NyDgj17PX7UZZ3U6z99xtxwoFQ3fSv8XQcGseVKycT3DROEON3Fc9bU2uiQYHCT1MbfsjXBT7s3M33YeVJuvW2XezGXUg24ntPYuAG10ufUyI4YW9JnWm57JhL1pPwIcxAVraNxElrGaZfbgFvhkVTL1KuM1JkbgmPWAyhtumUrrW9dyFDWTejLrSxqylGCYLRRL0QKbhESpEDquReFMBeNvilIccSGAyRgMr8Z\\\\\\\/CZ\\\\\\\/4P00iKcgCuXeuK5sXq5RDniMYNaQfO5CP7y+16Q\\\\\\\/1gs3ozEbNRM3TWonbR93k+1FzmgVrmgzfklKSPNMDAG3bd5sX8TAKQtEFZVz\\\\\\\/L0CwiDzZm4Wt1vEWRNgdItypQ3lIDNiZ0ZIH6THI\\\\\\\/xMvHcaAEEBgrXjNXLMDuZ4uJ977phF3Cq0JWMlr+6NbeOtcHPzN6gQdxvNtAE1PecvCLzUnDwVu\\\\\\\/EzA41Lgw4703kpxZwOHBiXviZ3tCcFEKYsncNeev5It6IfJKOI319deCq9AIRZtRe7HURSPxRc2U27esl2d4BHabx7aoOvAdiTnu0hSCP0JVXFhLzep7VolW2nRYTCvVJsaLMI1GumW7onkXEMsE1xgy\\\\\\\/lftiP3gjjlLj8Aa\\\\\\\/D8ujaTnuccgs2rt4IVi6pIY+JujwsuRCduhsCvpNfzyx\\\\\\\/E\\\\\\\/xNg+CN\\\\\\\/GTk6s1iIKN3erudVuYvfRRC9qo7y\\\\\\\/ssUHEDDgXxDJLFcVA9K8jW8JD2mcKa2FWOiOnQEFFhR1aC0n9eDgE7+M0HQuZB9551PuJ2F9jyle8=\\\u0022,\\\u0022iv\\\u0022:\\\u0022VdzajoMpmyw1EeRFmzkpoQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022480cb87fefb18a54afc53c52c9cd34ceea3e1bbde297414294cf1b627e0eada7\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/261628v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-80349 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/261628v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-2jCWh0gTO4TkvSBuXeghe9YUCN3eFaRiTGJD-egBb-0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="80349" id="node80349" data-pisa="biorxiv;261628v1" data-pisa-master="biorxiv;261628" data-apath="/biorxiv/early/2018/02/08/261628.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Generating genomic platforms to study <em>Candida albicans</em> pathogenesis</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Mélanie</span> <span class="nlm-surname">Legrand</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Sophie</span> <span class="nlm-surname">Bachellier-Bassi</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Keunsook K.</span> <span class="nlm-surname">Lee</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Yogesh</span> <span class="nlm-surname">Chaudhari</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Hélène</span> <span class="nlm-surname">Tournu</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Laurence</span> <span class="nlm-surname">Arbogast</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Hélène</span> <span class="nlm-surname">Boyer</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Murielle</span> <span class="nlm-surname">Chauvel</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Vitor</span> <span class="nlm-surname">Cabral</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Corinne</span> <span class="nlm-surname">Maufrais</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Audrey</span> <span class="nlm-surname">Nesseir</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Irena</span> <span class="nlm-surname">Maslanka</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Emmanuelle</span> <span class="nlm-surname">Permal</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Tristan</span> <span class="nlm-surname">Rossignol</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Louise A.</span> <span class="nlm-surname">Walker</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Ute</span> <span class="nlm-surname">Zeidler</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">Sadri</span> <span class="nlm-surname">Znaidi</span></span>, <span class="highwire-citation-author" data-delta="17"><span class="nlm-given-names">Floris</span> <span class="nlm-surname">Schoeters</span></span>, <span class="highwire-citation-author" data-delta="18"><span class="nlm-given-names">Charlotte</span> <span class="nlm-surname">Majgier</span></span>, <span class="highwire-citation-author" data-delta="19"><span class="nlm-given-names">Renaud A.</span> <span class="nlm-surname">Julien</span></span>, <span class="highwire-citation-author" data-delta="20"><span class="nlm-given-names">Laurence</span> <span class="nlm-surname">Ma</span></span>, <span class="highwire-citation-author" data-delta="21"><span class="nlm-given-names">Magali</span> <span class="nlm-surname">Tichit</span></span>, <span class="highwire-citation-author" data-delta="22"><span class="nlm-given-names">Christiane</span> <span class="nlm-surname">Bouchier</span></span>, <span class="highwire-citation-author" data-delta="23"><span class="nlm-given-names">Patrick Van</span> <span class="nlm-surname">Dijck</span></span>, <span class="highwire-citation-author" data-delta="24"><span class="nlm-given-names">Carol A.</span> <span class="nlm-surname">Munro</span></span>, <span class="highwire-citation-author" data-delta="25"><span class="nlm-given-names">Christophe</span> <span class="nlm-surname">d’Enfert</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/261628 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node80349" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mélanie Legrand </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Fungal Biology and Pathogenicity Unit, Department of Mycology, Institut Pasteur</span>, INRA, Paris, 75015, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M%C3%A9lanie%2BLegrand%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Legrand%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM%25C3%25A9lanie%2BLegrand%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sophie Bachellier-Bassi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Fungal Biology and Pathogenicity Unit, Department of Mycology, Institut Pasteur</span>, INRA, Paris, 75015, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sophie%2BBachellier-Bassi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bachellier-Bassi%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASophie%2BBachellier-Bassi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Keunsook K. Lee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>MRC Centre for Medical Mycology, Institute of Medical Sciences, University of Aberdeen</span>, Foresterhill, Aberdeen, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Keunsook%2BK.%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lee%20KK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKeunsook%2BK.%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Yogesh Chaudhari </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>MRC Centre for Medical Mycology, Institute of Medical Sciences, University of Aberdeen</span>, Foresterhill, Aberdeen, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yogesh%2BChaudhari%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chaudhari%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYogesh%2BChaudhari%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Hélène Tournu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>VIB-KU Leuven Center for Microbiology</span>, Leuven, <span class='nlm-country'>Belgium</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Laboratory of Molecular Cell Biology</span>, KU Leuven, Leuven, <span class='nlm-country'>Belgium</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=H%C3%A9l%C3%A8ne%2BTournu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tournu%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AH%25C3%25A9l%25C3%25A8ne%2BTournu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Laurence Arbogast </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Fungal Biology and Pathogenicity Unit, Department of Mycology, Institut Pasteur</span>, INRA, Paris, 75015, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laurence%2BArbogast%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Arbogast%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALaurence%2BArbogast%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Hélène Boyer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Fungal Biology and Pathogenicity Unit, Department of Mycology, Institut Pasteur</span>, INRA, Paris, 75015, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=H%C3%A9l%C3%A8ne%2BBoyer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boyer%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AH%25C3%25A9l%25C3%25A8ne%2BBoyer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Murielle Chauvel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Fungal Biology and Pathogenicity Unit, Department of Mycology, Institut Pasteur</span>, INRA, Paris, 75015, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Murielle%2BChauvel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chauvel%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMurielle%2BChauvel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Vitor Cabral </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Fungal Biology and Pathogenicity Unit, Department of Mycology, Institut Pasteur</span>, INRA, Paris, 75015, <span class='nlm-country'>France</span></div><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Univ. Paris Diderot, Sorbonne Paris Cité, Cellule Pasteur</span>, Paris, 75015, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vitor%2BCabral%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cabral%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVitor%2BCabral%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Corinne Maufrais </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Fungal Biology and Pathogenicity Unit, Department of Mycology, Institut Pasteur</span>, INRA, Paris, 75015, <span class='nlm-country'>France</span></div><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Institut Pasteur - Bioinformatics and Biostatistics Hub - C3BI</span>, USR 3756 IP CNRS - Paris, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Corinne%2BMaufrais%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maufrais%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACorinne%2BMaufrais%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Audrey Nesseir </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Fungal Biology and Pathogenicity Unit, Department of Mycology, Institut Pasteur</span>, INRA, Paris, 75015, <span class='nlm-country'>France</span></div><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Univ. Paris Diderot, Sorbonne Paris Cité, Cellule Pasteur</span>, Paris, 75015, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Audrey%2BNesseir%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nesseir%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAudrey%2BNesseir%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Irena Maslanka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>MRC Centre for Medical Mycology, Institute of Medical Sciences, University of Aberdeen</span>, Foresterhill, Aberdeen, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Irena%2BMaslanka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maslanka%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIrena%2BMaslanka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Emmanuelle Permal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Fungal Biology and Pathogenicity Unit, Department of Mycology, Institut Pasteur</span>, INRA, Paris, 75015, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emmanuelle%2BPermal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Permal%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEmmanuelle%2BPermal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Tristan Rossignol </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Fungal Biology and Pathogenicity Unit, Department of Mycology, Institut Pasteur</span>, INRA, Paris, 75015, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tristan%2BRossignol%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rossignol%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATristan%2BRossignol%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Louise A. Walker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>MRC Centre for Medical Mycology, Institute of Medical Sciences, University of Aberdeen</span>, Foresterhill, Aberdeen, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Louise%2BA.%2BWalker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Walker%20LA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALouise%2BA.%2BWalker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Ute Zeidler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Fungal Biology and Pathogenicity Unit, Department of Mycology, Institut Pasteur</span>, INRA, Paris, 75015, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ute%2BZeidler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zeidler%20U&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AUte%2BZeidler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-16"><div class="author-tooltip-name">Sadri Znaidi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Fungal Biology and Pathogenicity Unit, Department of Mycology, Institut Pasteur</span>, INRA, Paris, 75015, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sadri%2BZnaidi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Znaidi%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASadri%2BZnaidi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-17"><div class="author-tooltip-name">Floris Schoeters </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>VIB-KU Leuven Center for Microbiology</span>, Leuven, <span class='nlm-country'>Belgium</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Laboratory of Molecular Cell Biology</span>, KU Leuven, Leuven, <span class='nlm-country'>Belgium</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Floris%2BSchoeters%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schoeters%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFloris%2BSchoeters%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-18"><div class="author-tooltip-name">Charlotte Majgier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>7</span><span class='nlm-institution'>Modul-Bio; Parc Scientifique Luminy Biotech II</span>; Marseille, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charlotte%2BMajgier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Majgier%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharlotte%2BMajgier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-19"><div class="author-tooltip-name">Renaud A. Julien </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>7</span><span class='nlm-institution'>Modul-Bio; Parc Scientifique Luminy Biotech II</span>; Marseille, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Renaud%2BA.%2BJulien%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Julien%20RA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARenaud%2BA.%2BJulien%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-20"><div class="author-tooltip-name">Laurence Ma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>8</span><span class='nlm-institution'>Institut Pasteur - Biomics Pole - CITECH - Paris</span>, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laurence%2BMa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ma%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALaurence%2BMa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-21"><div class="author-tooltip-name">Magali Tichit </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>8</span><span class='nlm-institution'>Institut Pasteur - Biomics Pole - CITECH - Paris</span>, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Magali%2BTichit%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tichit%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMagali%2BTichit%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-22"><div class="author-tooltip-name">Christiane Bouchier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>8</span><span class='nlm-institution'>Institut Pasteur - Biomics Pole - CITECH - Paris</span>, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christiane%2BBouchier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bouchier%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristiane%2BBouchier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-23"><div class="author-tooltip-name">Patrick Van Dijck </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>VIB-KU Leuven Center for Microbiology</span>, Leuven, <span class='nlm-country'>Belgium</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Laboratory of Molecular Cell Biology</span>, KU Leuven, Leuven, <span class='nlm-country'>Belgium</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BVan%2BDijck%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dijck%20PV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BVan%2BDijck%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-24"><div class="author-tooltip-name">Carol A. Munro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>MRC Centre for Medical Mycology, Institute of Medical Sciences, University of Aberdeen</span>, Foresterhill, Aberdeen, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carol%2BA.%2BMunro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Munro%20CA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACarol%2BA.%2BMunro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:christophe.denfert@pasteur.fr" class="" data-icon-position="" data-hide-link-title="0">christophe.denfert@pasteur.fr</a>
<a href="mailto:c.a.munro@abdn.ac.uk" class="" data-icon-position="" data-hide-link-title="0">c.a.munro@abdn.ac.uk</a></span></li></ul></div><div class="author-tooltip-25"><div class="author-tooltip-name">Christophe d’Enfert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Fungal Biology and Pathogenicity Unit, Department of Mycology, Institut Pasteur</span>, INRA, Paris, 75015, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christophe%2Bd%E2%80%99Enfert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=d%E2%80%99Enfert%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristophe%2Bd%25E2%2580%2599Enfert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:christophe.denfert@pasteur.fr" class="" data-icon-position="" data-hide-link-title="0">christophe.denfert@pasteur.fr</a>
<a href="mailto:c.a.munro@abdn.ac.uk" class="" data-icon-position="" data-hide-link-title="0">c.a.munro@abdn.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/261628v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:80349" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:80349/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/261628v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:80349" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:80349/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/261628v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:80349" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:80349/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/261628v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:80349" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:80349/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/261628v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:80349" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:80349/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/261628v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:80349" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:80349/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">ABSTRACT</h2><p id="p-4">The advent of the genomic era has made elucidating gene function at large scale a pressing challenge. ORFeome collections, whereby almost all ORFs of a given species are cloned and can be subsequently leveraged in multiple functional genomic approaches, represent valuable resources towards this endeavor. Here we provide novel, genome-scale tools for the study of <em>Candida albicans</em>, a commensal yeast that is also responsible for frequent superficial and disseminated infections in humans. We have generated an ORFeome collection composed of 5,102 ORFs cloned in a Gateway<sup>™</sup> donor vector, representing 83% of the currently annotated coding sequences of <em>C. albicans</em>. Sequencing data of the cloned ORFs are available in the CandidaOrfDB database at <a href="http://candidaorfeome.eu">http://candidaorfeome.eu</a>. We also engineered 49 expression vectors with a choice of promoters, tags, and selection markers and demonstrated their applicability to the study of target ORFs transferred from the <em>C. albicans</em> ORFeome. In addition, the use of the ORFeome in the detection of protein-protein interaction was demonstrated. Mating-compatible strains as well as Gateway<sup>™</sup>-compatible two-hybrid vectors were engineered, validated and used in a proof of concept experiment. These unique and valuable resources should greatly facilitate future functional studies in <em>C. albicans</em> and the elucidation of mechanisms that underlie its pathogenicity.</p></div><div class="section" id="sec-1"><h2 class="">INTRODUCTION</h2><p id="p-5">Over the last decade, there has been an exponential growth in the quantity of available genome sequence data due to the very rapid progress in sequencing technology. In 2004, the genome sequence of the human fungal pathogen <em>Candida albicans</em> was released as Assembly 19 (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>). With the challenge of working with a heterozygous diploid organism, new computational methods had to be developed and resulted in the release in 2013 of Assembly 22, an assembly of a completely phased diploid genome sequence for the standard <em>C. albicans</em> reference strain SC5314 (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>). This opened new perspectives to understand the genetic basis and functional mechanisms that underlie pathogenesis and evolution in this organism. Although <em>C. albicans</em> gene sequences have been available to the community for more than a decade (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>,<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>), only 1,670 out of the 6,198 predicted protein-coding genes have been characterized as of January 31st, 2018 according to the Candida Genome Database (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>). Nowadays, the growing availability of whole-genome datasets has encouraged a shift towards the development of functional genomics and systems biology, enabling analysis of high-throughput whole-genome assays to better understand biological networks. In this context, major efforts have been made to generate large-scale deletion (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>-<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>) or overexpression (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>-<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>) mutant collections. Genome-wide ORF libraries or ORFeomes represent useful resources for the implementation of approaches used to elucidate gene function (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>). Besides the development of collections of overexpression mutants (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>), ORFeomes facilitate approaches aimed at evaluating protein subcellular localization and identifying protein-protein interactions (yeast two-hybrid) both at steady state and in response to environmental stimuli (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>).</p><p id="p-6">Large-scale cloning projects, with the goal of cloning all predicted ORFs into flexible recombinational vectors, have been described for several model organisms, including <em>Caenorhabditis elegans, Saccharomyces cerevisiae, Schizosaccharomyces pombe</em>, <em>Escherichia coli</em> K-12, <em>Arabidopsis thaliana</em>, <em>Xenopus laevis</em> and <em>Drosophila melanogaster</em>, as well as infectious microorganisms such as <em>Brucella melitensis, Plasmodium falciparum</em>, <em>Helicobacter pylori</em>, <em>Chlamydia pneumonia</em>, <em>Staphylococcus aureus</em> and viruses (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>-<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>). Human ORFeomes have also been generated and made publicly available, with the latest version containing sequence-confirmed ORFs for more than 11,000 human genes (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>). Most ORFeome libraries are created using the highly versatile Gateway<sup>™</sup> cloning approach (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>). The Gateway<sup>™</sup> technology employs the recombination system of bacteriophage lambda with two sets of reactions: the BP reaction, catalysed by Gateway<sup>™</sup> BP Clonase<sup>™</sup>, facilitates recombination between <em>attB</em> and <em>attP</em> sequences, and the LR reaction, catalysed by Gateway<sup>™</sup> LR Clonase<sup>™</sup>, promotes recombination between <em>attL</em> and <em>attR</em> sequences (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a>). Previously, we have reported the development of a first generation <em>C. albicans</em> ORFeome, in which 644 full-length ORFs, encoding signaling proteins (transcription factors and kinases), as well as cell wall proteins and proteins involved in DNA processes (repair, replication and recombination), were cloned in the Gateway<sup>™</sup> vector pDONR207 (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>). Here we describe the generation and validation of the latest <em>Candida albicans</em> ORFeome collection, which consists of 5,102 ORFs in pDONR207, allowing the transfer of the cloned genes into a variety of <em>C. albicans</em> Gateway<sup>™</sup>-compatible expression vectors. To this end, we also provide a panel of 15 expression vectors that differ in the combination of promoter/selection marker they are carrying, and give different options regarding the tagging of the cloned gene product. These 15 expression vectors have been validated using the filament-specific transcription factor <em>UME6</em>. In addition, we also provide a secondary collection of 34 expression vectors with additional options regarding the choice of the promoter and the tagging of the cloned gene product. Finally, among the numerous applications of ORFeome collections, we present a powerful association of the <em>C. albicans</em> ORFeome and a <em>C. albicans</em>-adapted two-hybrid system (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>) that will allow systematic protein-protein interaction screening in <em>C. albicans</em>.</p></div><div class="section" id="sec-2"><h2 class="">MATERIAL AND METHODS</h2><div id="sec-3" class="subsection"><h3>Strains and growth conditions</h3><p id="p-7"><em>C. albicans</em> strains used in this study are listed in <strong>Table 1</strong>. <em>C. albicans</em> strains were routinely cultured at 30°C in YPD medium (1% yeast extract, 2% peptone, 2% dextrose), synthetic dextrose (SD) medium (0.67% yeast nitrogen base, 2% dextrose) or nourseothricin-containing YPD medium (YPD + 200 μg/ml Nourseothricin). Solid media were obtained by adding 2% agar.</p><p id="p-8"><em>E. coli</em> strains were routinely cultured at 30°C or 37°C in LB or 2YT supplemented with 10 μg/ml gentamicin, 50 μg/ml kanamycin or 50 μg/ml ticarcillin. Solid media were obtained by adding 2% agar.</p><p id="p-9">DH5α <em>E. coli</em> cells competent for transformation with DNA were prepared according to Hanahan <em>et al</em>. (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>).</p><p id="p-10">The ORFeome collection is being distributed by the Centre International de Ressources Microbiennes (CIRM). The 49 expression vectors are available from Addgene, while two-hybrid strains and vectors are now accessible from the Belgian Co-ordinated Collections of Micro-organisms (BCCM).</p></div><div id="sec-4" class="subsection"><h3>Generation of an entry clone collection</h3><div id="sec-5" class="subsection"><h4>Primer design</h4><p id="p-11">The DNA sequence of <em>C. albicans</em> was obtained from the Candida Genome Database Release 21 (<a href="http://www.candidagenome.org">www.candidagenome.org</a>), before Assembly 22 was released. Gene-specific forward primers were designed by adding the sequence 5’-GGGGACAAGTTTGTACAAAAAAGCAGGCTTG-3’ to the 5’ end of the first 30 nucleotides of each ORF. Gene-specific reverse primers were designed by adding the sequence 5’-GGGGACCACTTTGTACAAGAAAGCTGGGTC-3’ to the 5’ end of the last 30 nucleotides of each ORF excluding the Stop codon. PCR fragments amplified from these primers are compatible for C-terminal tagging with destination vectors containing the Gateway<sup>™</sup> cassette A. A total of 6,205 primer pairs were obtained from Invitrogen in a 96-well format and are listed in <strong>Supplemental Table S1</strong>.</p></div><div id="sec-6" class="subsection"><h4>Gateway<sup>™</sup> cloning of the C. albicans ORFeome</h4><p id="p-12">The detailed method for the cloning of <em>C. albicans</em> ORFs in the pDONR207 vector has been described (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>). Briefly, ORFs, ranging from 90 bp to 5,295 bp, were amplified from genomic DNA of <em>C. albicans</em> strain SC5314 in 96-well plates using the Thermo Scientific Phusion High-Fidelity DNA Polymerase and 30 cycles of amplification, with elongation time varying from 1 to 3 min according to the ORF size. After ethanol precipitation, Gateway<sup>™</sup>-compatible amplified ORFs were recombined into pDONR207 (Invitrogen) using the Gateway<sup>™</sup> BP Clonase<sup>™</sup> II Enzyme Mix (Invitrogen). Reaction mixes containing pDONR207, the PCR products and the BP Clonase<sup>™</sup> were incubated overnight in 96-well plates, at room temperature. After adding proteinase K (Invitrogen) and incubating 10 min at 37°C, the BP reactions were directly used for bacterial transformation. 45-50 μl of chemically competent <em>E. coli</em> DH5α were added to the BP reactions and incubated for 30 min at 4°C. After heat-shock at 42°C for 35 s, 150 μl of SOC medium (20 ml YPD + 2 ml LB + 1 ml Hepes) was added to the transformation reactions and the samples were covered with breathing films and incubated for 1.5 h at 37°C with shaking. Then 100 μl of each sample were plated onto LB agar containing 10 μg/ml Gentamicin and incubated overnight at 37°C. The remainder of the transformation reactions were stored at −80°C in 30% glycerol. A single colony from each transformation reaction was inoculated in 400 μl 2YT + 10 μg/ml Gentamicin in 96 deep-well microplates. After 36h-growth at 37°C, the cultures were used for plasmid extraction and for −80°C storage.</p></div><div id="sec-7" class="subsection"><h4>Validation of entry clones by DNA sequencing and bioinformatics analysis</h4><p id="p-13">Sanger sequencing of the 5’-end of each BP clone was performed with the 207VER-F oligonucleotide (see <strong>Supplemental Table 2</strong> for oligonucleotide sequences except those used to amplify ORFs) to confirm that the expected ORF had been cloned. The 3’-ends were also sequenced with the 207VER-R oligonucleotide to ensure that the oligonucleotides did not carry deletions. All inserts validated by Sanger sequencing were systematically subjected to full-length sequence analysis using Illumina technology as described in Chauvel <em>et al</em>. (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>). If nonsense or frameshift mutations were observed in a cloned ORF, another colony was checked, or the ORF reamplified and cloned again. If these attempts were unsuccessful, cloning of the ORF was abandoned. Clones containing contiguous deletions or insertions of multiples of 3 bp were accepted. Missense mutations and those located within introns were accepted. It should be noted that further analysis of each mutation-containing ORF is required to determine whether or not mutations affect the function of the ORF.</p></div></div><div id="sec-8" class="subsection"><h3>CandidaOrfDB database</h3><p id="p-14">CandidaOrfDB was created to integrate the data of the <em>C. albicans</em> ORFeome project and is available at <a href="http://candidaorfeome.eu">http://candidaorfeome.eu</a>. The database is a relational database using the SQL database server Oracle with a web interface developed using J2EE. The alignment algorithm uses <em>biojava3-alignment</em> Java library, implemented with a NeedlemanWunsch aligner, configured with a gap open penalty of 5 and a gap extension penalty of 2. The substitution matrix used is named “nuc-4_4” and was created by Todd Lowe (<a href="https://github.com/sbliven/biojava/blob/master/biojava3-alignment/src/main/resources/nuc-4_4.txt">https://github.com/sbliven/biojava/blob/master/biojava3-alignment/src/main/resources/nuc-4_4.txt</a>). The alignment algorithm was specifically customized for CandidaOrfDB to work on segments of 500 codons, which provides an optimized alignment performance for <em>C. albicans</em> average sequence length.</p></div><div id="sec-9" class="subsection"><h3>Clone access</h3><p id="p-15">The <em>C. albicans</em> ORFeome clones are available at the Centre International de Ressources Microbiennes (CIRM - <a href="https://www6.inra.fr/cirm_eng/">https://www6.inra.fr/cirm_eng/)</a>.</p></div><div id="sec-10" class="subsection"><h3>Destination plasmids</h3><p id="p-16">All destination plasmids constructed in this study were derived from CIp-P<em><sub>TET</sub></em>-GTW (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">11</a>) and were confirmed by Sanger sequencing. Plasmid sequences have been submitted to GenBank. Accession numbers are given in <strong>Table 2</strong>. All oligonucleotides used for PCR and/or sequencing are listed in <strong>Supplemental Table 2</strong>.</p><div id="sec-11" class="subsection"><h4>SP cloning</h4><p id="p-17">First, a spacer sequence (SP) was amplified from the <em>E. coli kanR</em> gene borne on pCR-topo-blunt plasmid (Invitrogen) with primers SP3 and SP4. The PCR product was cloned in the SacII site of CIp-P<em><sub>TET</sub></em>-GTW, yielding CIp-P<em><sub>TET</sub></em>-GTW-SP, also referred to as pCA-Dest1100. This 390 bp-long sequence is flanked by the SP1 and SP2 Illumina paired-end sequencing primers 1 and 2 and can be used to insert molecular barcodes if expression plasmids are to be used in signature-tagged mutagenesis approaches.</p></div><div id="sec-12" class="subsection"><h4>Epitope tags</h4><p id="p-18">We then inserted in frame tags either upstream of attR1 or downstream of attR2 to allow N-terminal or C-terminal protein tagging, respectively. For the cloning of N-terminal tags, we used pUC-attR1-CmR, containing a PciI-BglII fragment from CIp-GTW cloned into the PciI and BamHI sites of pUC18. CIp-GTW was obtained after deleting P<em><sub>TET</sub></em> from CIp-P<em><sub>TET</sub></em>-GTW with Acc65I and HpaI. For cloning of the 3xHA coding sequence, two oligonucleotides (oligo1_PciHABsrG and oligo2_PciHABsrG) containing three HA epitopes were hybridized, gel purified, and cloned into pUC-attR1-CmR cut with PciI and BsrGI. The resulting plasmid was then used as a PCR template with primers GTW02 and GTW03. The resulting PCR product was cut with BspEI and HpaI, and cloned into the same sites of pCA-Dest1100 to yield pCA-Dest1110. For cloning of the GFP tag, the GFP gene was amplified from pFA-GFP-URA3 (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>) with primers GTW04 and GTW05. The PCR product was cut with PciI and EcoRV and ligated into the same sites of pUC-attR1-CmR. The resulting plasmid was then used as a PCR template with primers GTW07 and GTW03. The resulting PCR product was cut with ScaI and BspEI and ligated into pCA-Dest1100 cut with HpaI and BspEI, yielding pCA-Dest1120. For cloning of the TAP-tag, the TAP-tag coding region was PCR amplified from pFA-TAP-URA3 (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">11</a>) with primers GTW13 and GTW14. The resulting PCR product was then cut with PciI and BsrGI, and ligated into the same sites of pUC-attR1-CmR. The resulting plasmid was then digested with EcoRV and BspEI and the smallest restriction fragment was ligated into pCA-Dest1100 cut with HpaI and BspEI to yield pCA-Dest1130. For the cloning of C-terminal tags, we generated pUC-attR2 by inserting a SalI-HindIII fragment from pCA-Dest1100 into the same sites of pUC18. For cloning of the 3xHA epitope, pCaMPY-3xHA (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>) was used as a PCR template with primers GTW20 and GTW21, and the resulting PCR product cut with BsrGI and NsiI was ligated in the same sites of pUC-attR2. The resulting plasmid was cut with SalI and NsiI and the fragment cloned in the same sites of pCA-Dest1100, yielding pCA-Dest1101. For cloning the GFP tag, the GFP gene was amplified from pFA-GFP-URA3 (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">35</a>) with primers GTW11 and GTW12. The resulting PCR product was cut with NsiI and EcoRV and ligated into the same sites of pUC-attR2. The resulting plasmid was then cut with SalI and NsiI and the GFP-bearing fragment ligated into the same sites of pCA-Dest1100, yielding pCA-Dest1102. For cloning of the TAP-tag, the TAP-tag coding region was amplified from CIp10-P<em><sub>PCK1</sub></em>-GTW-TAPtag (<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">11</a>) with primers GTW15 and GTW16. The PCR product was cut with BsrGI and NsiI and cloned in the same sites of pUC-attR2. The resulting plasmid was digested with SalI and NsiI, and the TAP-containing fragment ligated in the same sites of pCA-Dest1100, yielding pCA-Dest1103.</p></div><div id="sec-13" class="subsection"><h4>Promoters</h4><p id="p-19">We replaced P<em><sub>TET</sub></em> with either P<em><sub>PCK1</sub></em> (obtained from CIp10-P<em><sub>PCK1</sub></em>-GTW-TAPtag, (<a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">11</a>)), yielding the pCA-Dest12xx series; P<em><sub>TDH3</sub></em> (cut from pKS-P<em><sub>TDH3</sub></em>; P<em><sub>TDH3</sub></em> was PCR amplified from SC5314 genomic DNA with oligos SZ11 and SZ12, and the fragment inserted into pBluescript-KS (+) cut with XhoI and EcoRV) yielding the pCA-Dest13xx series; or P<em><sub>ACT1</sub></em>, cut from CIp10::P<em><sub>ACT1</sub></em>-gLUC59 (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>), yielding the pCA-Dest14xx series.</p></div><div id="sec-14" class="subsection"><h4>Selection markers</h4><p id="p-20">In all plasmids except the P<sub><em>TET</em></sub> series, the auxotrophic marker <em>URA3</em> was replaced with the <em>NAT1</em> marker conferring nourseothricin resistance. Briefly, a XbaI-DraIII fragment or SpeI-NaeI fragment was cut from pUC-NAT1 and inserted into the similarly cut vectors. pUC-NAT1 was built by amplifying <em>NAT1</em> under the control of P<em><sub>TEF1</sub></em> promoter with oligonucleotides UZ50 and UZ51, and using pFA6-SAT1 (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>) as a template, and cloning the AatII-cut fragment into the same site of pUC18.</p><p id="p-21">The 49 expression vectors are available from Addgene <a href="(https://www.addgene.org)">(https://www.addgene.org)</a>.</p></div></div><div id="sec-15" class="subsection"><h3>Validation of 15 expression plasmids</h3><div id="sec-16" class="subsection"><h4>Construction of C. albicans UME6-overexpression strains</h4><p id="p-22">Detailed methods for the transfer of <em>C. albicans</em> ORFs from pDONR207 into the expression plasmids as well as the integration of the resulting expression plasmids at the <em>RPS1</em> locus have been described (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a>). Briefly, the <em>UME6</em> ORF was transferred from the entry clone into each one of 15 expression vectors by using the Gateway<sup>™</sup> LR Clonase<sup>™</sup> II Enzyme Mix (Invitrogen). After <em>E. coli</em> transformation, the plasmids were verified by EcoRV digest. The <em>URA3-</em> and <em>NAT1</em>-bearing expression plasmids were digested by StuI and transformed into <em>C. albicans</em> strain CEC161 or CEC369, respectively, according to Walther and Wendland (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>). Transformants were selected for prototrophy or nourseothricin resistance, respectively, and verified by PCR using primer CIpUL with (i) primer CIpUR for the <em>URA3</em>-bearing plasmids and (ii) primer CgSAT1-rev for the <em>SAT1</em>-bearing plasmids, that yield 1 kb and 1.6 kb products, respectively, if integration of the OE plasmid has occurred at the <em>RPS1</em> locus.</p></div><div id="sec-17" class="subsection"><h4>Induction of the Tet-On system</h4><p id="p-23">Overexpression from P<em><sub>TET</sub></em> was achieved by the addition of anhydrotetracycline (ATc, 3 μg/ml; Fisher Bioblock Scientific) in YPD at 30°C (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>). Overexpression experiments were carried out in the dark, as ATc is light sensitive.</p></div><div id="sec-18" class="subsection"><h4>Microscope analysis for filamentation</h4><p id="p-24">Cells were observed with a Leica DM RXA microscope (Leica Microsystems) with an x40 oil-immersion objective.</p></div></div><div id="sec-19" class="subsection"><h3>Analysis of TAP- and 3HA-tagged proteins by Western blots</h3><p id="p-25">For the P<em><sub>TET</sub></em>, P<em><sub>TDH3</sub></em> and P<em><sub>ACT1</sub></em> strains, a 30 ml culture in YPD or YPD+ATc3 was inoculated at OD600 = 0.2 with a freshly grown colony and incubated at 30°C with shaking until OD600 reaches ∼1. For the P<em><sub>PCK1</sub></em> strain, cells were scraped off the agar of Petri dish cultures and resuspended in 1 ml dH<sub>2</sub>O after centrifugation. The cell suspension was used to inoculate a 40 ml SD culture at OD600 = 0.2.</p><p id="p-26">20 ODs of exponentially growing cells were collected by centrifugation and resuspended in lysis buffer (9M Urea, 1.5% w/v dithiothreitol, 2-4% w/v CHAPS, and 1.5 M Tris pH9.5). Homogenization of the cells was achieved using a FastPrep bead beater. After homogenization, the lysed cells were centrifuged at 13,000 rpm for 10 min at room temperature, and the supernatant containing the solubilized proteins was used directly or stored at −80 °C.</p><p id="p-27">Proteins were separated on an Invitrogen 8% NuPage gel, transferred onto nitrocellulose. TAP- and 3HA-tagged proteins were detected using peroxidase-coupled anti-peroxidase and anti-HA-peroxydase antibodies (Sigma and Roche, respectively) and an ECL kit (GE Healthcare).</p></div><div id="sec-20" class="subsection"><h3>Construction of opaque mating-compatible two-hybrid strains and Gateway<sup>™</sup>-compatible two-hybrid vectors</h3><p id="p-28">Mating-compatible strains were generated by deletion of MTL<strong>a</strong> or MTLα locus using the <em>SAT1</em> flipper cassette (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">38</a>) in the two-hybrid strain background SC2H3 (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">32</a>). To delete the MTL<strong>a</strong> locus, flanking regions were amplified with primers MTLa-5’F and MTLa-5’R, and with MTLa-3’F and MTLa-3’R. To delete the MTLα locus, homologous regions were amplified with primers MTLα-5’F and MTLα-5’R, and with MTLα-3’F and MTLα-3’R. Both fragments for each locus were cloned into pSAT1 (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">38</a>), at SacI/NotI and XhoI/KpnI sites respectively. SacI/KpnI deletion constructs were transformed in SC2H3 (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">32</a>) and transformants were selected on YPD medium containing 200 μg/ml of nourseothricin. Positive SC2H3a and SC2H3α strains were subsequently grown in maltose medium for induction of the recombinase and loss of the <em>SAT1</em> flipper cassette.</p><p id="p-29">For induction of opaque switching, a <em>WOR1</em> fragment was amplified with primers WOR1F and WOR1R and cloned into pNIM1 vector (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">40</a>) at SalI and BglII sites. SC2H3a and SC2H3α strains were transformed with pNIM1-WOR1 with selection on nourseothricin-containing medium. The resulting strains, SC2H3a-pWOR1 and SC2H3α−pWOR1, were grown in presence of doxycycline (50 μg/ml) to induce the opaque state (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>). Opaque mating-compatible strains were finally transformed with prey or bait plasmid and selected on SC-ARG or SC-LEU respectively.</p><p id="p-30">Bait and prey two-hybrid vectors, pC2HB and pC2HP respectively (<a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">32</a>) were converted into Gateway<sup>™</sup> destination vectors using the Gateway<sup>™</sup> Vector Conversion System to generate pC2HB-GC and pC2HP-GC respectively. The genes encoding the bait protein Hst7 (Orf19.469) and the prey Cek1 (Orf19.2886) were transferred by LR reactions from the donor collection of vectors into pC2HB-GC and pC2HP-GC destination vectors respectively.</p></div><div id="sec-21" class="subsection"><h3>Mating approach and protein-protein interaction detection</h3><p id="p-31">Opaque cells of SC2H3a and SC2H3α, expressing VP16-Cek1 (Arg+) and lexA-Hst7 (Leu+) respectively, were crossed as follows. Opaque cells of both types were mixed at 1 × 10<sup>6</sup>cells each in Spider medium (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>) and incubated at 23°C for 24h. Resulting tetraploid cells were selected on SC-LEU-ARG medium for 48h incubation at 30°C, and transferred to SC-HIS-MET medium for protein-protein interaction detection. Chromosome loss was induced on pre-sporulation medium at 37°C for 10 days as described previously (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>).</p></div></div><div class="section" id="sec-22"><h2 class="">RESULTS</h2><div id="sec-23" class="subsection"><h3>Establishment of a sequence-validated Candida albicans ORFeome</h3><p id="p-32">Our objective was to develop a <em>C. albicans</em> ORFeome encompassing as many of the 6,205 ORFs predicted in Assembly 21 of the <em>C. albicans</em> genome as possible (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>). To this aim, forward and reverse oligonucleotides with, respectively, <em>attB</em> and <em>attP</em> sequences at their 5’ ends were synthesized for each of the 6,205 ORFs (<strong>Supplemental Table S1</strong>), used in independent PCR reactions with genomic DNA of <em>C. albicans</em> strain SC5314 (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>) and the resulting PCR products were cloned in pDONR207 using Gateway<sup>™</sup> recombinational cloning (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">30</a>). The resulting plasmids were then subjected to Sanger sequencing at the 5’ and 3’ ends of the cloned ORFs and to Illumina sequencing throughout the cloned ORFs.</p><p id="p-33">Among the 6,205 ORF sequences (from Assembly 21) used for oligonucleotide design, 6 are no longer present in Assembly 22 while 20 are annotated as mitochondrial genes in Assembly 22. Among the 6,179 remaining ORFs, 5,102 (82.6%) were successfully cloned and full-length sequence-validated. Sequences were validated by comparing the entire sequence of each cloned ORF against the reference sequences for haplotype A and B of the <em>C. albicans</em> SC5314 genome available at the Candida Genome Database (CGD; version_A22-s07-m01-r18; (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>)). We defined 4 groups: (i) 4,209 sequences that are 100% identical with a reference sequence (<strong>No SNP, no DIP</strong>; where SNP stands for Single Nucleotide Polymorphism and DIP stands for Deletion/Insertion Polymorphism); (ii) 116 sequences that do not carry any SNP but carry DIPs (<strong>No SNP, DIPs</strong>); (iii) 682 sequences that carry SNPs but no DIP (<strong>SNPs, no DIP</strong>); and (iv) 95 sequences that carry both SNPs and DIPs (<strong>SNPs and DIPs</strong>) (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1A</a>; <strong>Supplemental table S1)</strong>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261628/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-217047756" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261628/F1.medium.gif" width="440" height="328"/><noscript><img class="highwire-fragment fragment-image" alt="Figure1" src="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261628/F1.medium.gif" width="440" height="328"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261628/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261628/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div><p id="p-34">Overall, 4,209/6,179 variation-free <em>C. albicans</em> ORFs (68.1%) are now available to the community. For 35.9% of the validated ORFs, there was no difference between the two alleles in the reference sequences (<strong>HapA/B;</strong> <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1B</a>). When nucleotide differences were present in the two alleles of the reference ORF sequence, we did not notice any bias in the haplotype that was cloned: haplotype A ORFs were cloned in 33% of cases (<strong>HapA;</strong> <a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1B</a>) and haplotype B ORFs were cloned in 31% of cases (<strong>HapB;</strong> <a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1B</a>). On chromosomes R, 1, 2, 3, 4, 5 and 7, ORF cloning was uniformly successful, with a cloning success rate ranging from ∼82% to ∼86%. Chromosome 6 ORFs were slightly underrepresented (76.9%) (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1C</a>). Although ORFs up to 1.5kb could be cloned with a success rate around 87% and ORFs between 1.5 and 4kb could be cloned with a success rate around 77%, we observed a decreased success rate for the longer ORFs (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig. 1D</a>).</p><p id="p-35">The analysis of the 6,880,620 nucleotides of the 5,102 sequence-validated ORFs revealed 2077 SNPs. These nucleotide substitutions could be real SNPs between our strain and the reference strain, or due to poor quality of some reference sequences, or mutations in the primers, or misincorporation by the DNA polymerase. Overall, the resulting maximum error rate amounts to 3 × 10<sup>−4</sup>or 1 SNP every 3,312 cloned nucleotides. Notably, the <em>C. albicans</em> SC5314 genome sequence available at CGD (version_A22-s07-m01-r18) contained 272 ORFs with sequence ambiguities for at least one of the alleles. These ambiguities included stretches of ‘Ns’ or IUPAC code nucleotides (Y, R, S, M, K, W). The ORFeome-generated sequencing data resolved sequence ambiguities for 110 of these ORFs (<strong>Supplemental Tables 3 and 4</strong>). In this study, we also detected recombinant haplotypes for 116 of the cloned ORFs (<strong>Supplemental Table 5</strong>) that display characteristics of both reference haplotypes.</p><p id="p-36">Taken together, our study provides an almost comprehensive, extremely high quality <em>C. albicans</em> ORFeome.</p></div><div id="sec-24" class="subsection"><h3>CandidaOrfDB: a database for the C. albicans ORFeome</h3><p id="p-37">CandidaOrfDB was created to integrate the data of the <em>C. albicans</em> ORFeome project and is available at <a href="http://candidaorfeome.eu">http://candidaorfeome.eu</a>. CandidaOrfDB enables the scientific community to search for availability and quality of the clones. All data pertinent to the cloning process are stored in the database. This information includes the name and sequence of the primers used to amplify the ORFs from <em>C. albicans</em> SC5314 genomic DNA, the coordinates of the primers in their 96-well storage plates, and any sequencing information available on the clones. SNPs and DIPs, their location in the ORF and their influence on the corresponding amino acid sequences are shown (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>). Nucleotide sequence of the cloned ORF and the corresponding amino acid sequence are displayed with highlights of SNPs or DIPs relative to the closest haplotype sequence (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2</a>). The reference sequence data have been extracted from CGD (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">46</a>). The database can be queried through the ORF number (orf19.xxxx or 19.xxxx; (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>)), the Assembly 22 name (Ci_XXXXXW or Ci_XXXXXC; (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">5</a>)) or the gene name.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261628/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-217047756" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261628/F2.medium.gif" width="440" height="362"/><noscript><img class="highwire-fragment fragment-image" alt="Figure2" src="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261628/F2.medium.gif" width="440" height="362"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261628/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261628/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div></div><div id="sec-25" class="subsection"><h3>A collection of destination vectors for exploiting the C. albicans ORFeome</h3><p id="p-38">The <em>C. albicans</em> ORFeome described above was developed in pDONR207 as this allows subsequent transfer of the cloned ORFs in Gateway<sup>™</sup>-adapted destination vectors. While destination vectors for ORF expression in hosts such as <em>E. coli</em> or <em>S. cerevisiae</em> are available (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>,<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>), only a few destination vectors for ORF expression in <em>C. albicans</em> have been reported (<a id="xref-ref-11-7" class="xref-bibr" href="#ref-11">11</a>). Therefore, we set out to establish a collection of Gateway<sup>™</sup>-adapted destination vectors for constitutive or conditional expression of untagged or tagged ORFs in <em>C. albicans</em>. Our primary collection of 15 <em>C. albicans</em> destination vectors derived from CIp10S (<a id="xref-ref-11-8" class="xref-bibr" href="#ref-11">11</a>) provides a choice of two promoters, either the constitutive promoter P<em><sub>TDH3</sub></em> or the inducible promoter P<em><sub>TET</sub></em>, and the option for N- or C-terminal fusion to various epitope tags (3xHA or TAP) (<strong>Table 2</strong> and <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3</a>). Integration of the destination vectors and their derivatives at the <em>RPS1</em> locus in the <em>C. albicans</em> genome is promoted by StuI or I-SceI linearization. These CIp10-derived vectors carry either the auxotrophic marker <em>URA3</em> or the <em>NAT1</em> marker that confers resistance to nourseothricin and can be used with clinical isolates. However, P<em><sub>TET</sub></em>-bearing plasmids cannot carry the <em>NAT1</em> marker since the transactivator needed for tetracycline-mediated induction of P<em><sub>TET</sub></em> is borne on the <em>NAT1</em>-carrying pNIMX plasmid (<a id="xref-ref-11-9" class="xref-bibr" href="#ref-11">11</a>). All vectors harbor a so-called spacer sequence (SP) originating from the <em>E. coli</em> kanamycin resistance gene, for barcode insertion and subsequent Illumina-based barcode sequencing. A second set of 34 plasmids was also generated with the constitutive promoters P<em><sub>ACT1</sub></em> or the inducible promoter P<em><sub>PCK1</sub></em>, and the option for N- or C-terminal fusion to GFP (<strong>Table 2</strong>). All 49 destination vectors were designated with a standardized nomenclature, pCA-DESTijkl, whereby i stands for the transformation marker (1, <em>URA3</em>; 2, <em>NAT1</em>); j stands for the promoter (1, P<em><sub>TET</sub></em>; 2, P<em><sub>PCK1</sub></em>; 3, P<em><sub>TDH3</sub></em>; 4, P<em><sub>ACT1</sub></em>); k stands for N-terminal tagging (0, no tag; 1, 3xHA; 2, GFP; 3, TAP); and l stands for C-terminal tagging (0, no tag; 1, 3xHA; 2, GFP; 3, TAP) (<strong>Table 2</strong> and <a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3</a>). All destination vectors are available from Addgene <a href="(https://www.addgene.org)">(https://www.addgene.org)</a>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261628/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-217047756" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261628/F3.medium.gif" width="440" height="363"/><noscript><img class="highwire-fragment fragment-image" alt="Figure3" src="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261628/F3.medium.gif" width="440" height="363"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261628/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261628/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div><p id="p-39">In order to validate that the primary set of 15 destination vectors could drive constitutive or conditional expression of untagged or tagged ORFs, the <em>UME6</em> ORF was transferred into each of them using LR Clonase<sup>™</sup>. <em>UME6</em> encodes a transcription factor whose overexpression has been shown to force filamentation in <em>C. albicans</em> (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>,<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>). The validation criteria for the destination vectors were (i) success of the LR reaction, (ii) integration in the <em>C. albicans</em> genome at the <em>RPS1</em> locus, (iii) filamentation phenotype and/or (iv) detection of the 3xHA- or TAP-tagged Ume6 protein by western blot. The LR reaction was successfully used to transfer <em>UME6</em> from the pDONR207:: <em>UME6</em> plasmid to all 15 destination vectors. The resulting expression plasmids were successfully integrated at the <em>C. albicans RPS1</em> locus. The 5 strains containing P<em><sub>TET</sub></em> were induced overnight at 30°C by adding 3 μg/ml anhydrotetracycline (ATc3) to the culture medium, while the 10 strains containing P<em><sub>TDH3</sub></em> were grown overnight at 30°C in YPD. As described in Chauvel <em>et al</em>. (<a id="xref-ref-11-10" class="xref-bibr" href="#ref-11">11</a>), overexpression of Ume6 led to hyperfilamentation. This phenotype was observed for all strains, regardless the tags (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4A</a> and <a id="xref-fig-4-2" class="xref-fig" href="#F4">4B</a>). 3xHA- or TAP-tagged Ume6 was detected in the relevant strains and under the appropriate growth conditions (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5A</a> and <a id="xref-fig-5-2" class="xref-fig" href="#F5">5B</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261628/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-217047756" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261628/F4.medium.gif" width="386" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure4" src="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261628/F4.medium.gif" width="386" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261628/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261628/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261628/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-217047756" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261628/F5.medium.gif" width="440" height="427"/><noscript><img class="highwire-fragment fragment-image" alt="Figure5" src="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261628/F5.medium.gif" width="440" height="427"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261628/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261628/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div><p id="p-40">Taken together, these data present a collection of 49 available destination vectors and validate 15 of those for application with the <em>C. albicans</em> ORFeome.</p></div><div id="sec-26" class="subsection"><h3>Proof of concept of a two-hybrid matrix approach of protein-protein interaction detection via mating in C. albicans</h3><p id="p-41">The availability of the ORFeome collection is a prerequisite to large, systematic two-hybrid (2H) screenings in <em>C. albicans</em>. The Candida 2H (C2H) system was developed for targeted one to one protein-protein interaction detection (<a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">32</a>). Hence, there was a need to engineer the system for high throughput screening based on rapid and convenient recombination cloning systems for the generation of prey/bait arrays of proteins, and to use a mating approach to circumvent the low efficiency of transformation of this fungus. The former was obtained by adapting the prey and bait recipient vectors for Gateway<sup>™</sup>-mediated transfer of the ORFeome library, yielding pC2HB-GC and pC2HP-GC, respectively. The latter involved the combined processes of white-opaque switching and the preparation of mating-compatible 2H strains. The pleiomorphic fungus <em>C. albicans</em> is characterized by a parasexual cycle, with the possible mating of diploids, and generation of tetraploids, the absence of meiosis and the reversion to the diploid state by chromosome loss (reviewed in (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>)). An essential pre-requisite to mating in <em>C. albicans</em> is the phenotypic switch from white to opaque, a stable but reversible switch regulated by environmental stimuli such as low temperatures, carbon dioxide and nutrients. The epigenetic switch is tightly linked to mating since the a1/α2 complex, encoded at the mating type like (<em>MTL</em>) loci, acts as a repressor of opaque switching (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>). In that context, the two-hybrid strain SC2H3 originating from the diploid a/α SN152 strain was deleted for the <em>MTL</em>a or <em>MTLα</em> locus to construct SC2H3α and SC2H3a strains. The efficient switch to opaque in the hemizygote strains was achieved by the doxycycline-controlled expression of <em>WOR1</em>, a major regulator of the white to opaque switch (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">41</a>,<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>).</p><p id="p-42">A proof of principle of the whole procedure for protein-protein detection is shown in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>. The MAP kinase kinase Hst7 was used as bait and linked to the DNA binding domain lexA, as part of the pC2HB-GC vector. Similarly, the Cek1 kinase was expressed as a prey protein, fused to the activation domain VP16, component of the pC2HP-GC vector. pC2HB-Hst7 and pC2HP-Cek1 were transformed into opaque SC2H3α and SC2H3a strains, respectively. Mating of the resulting transformants was achieved through selection for leucine and arginine prototrophy. Upon interaction of the two proteins, the reconstituted transcriptional module promoted the expression of the <em>HIS4</em> marker, allowing only the mating-derived strains to grow on a histidine-free medium, whereas the original diploid strains could not (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig. 6B</a>). Through a mechanism of chromosome loss, the crossed tetraploid strains return back to a diploid state. We showed that the plasmids were stably expressed in the diploid offspring strains as these strains conserved their ability to grow in absence of arginine and leucine, indicating the presence of the prey and bait vectors respectively. In addition, all these strains were able to grow in absence of histidine, indicative of the targeted protein-protein interaction.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261628/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-217047756" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261628/F6.medium.gif" width="440" height="296"/><noscript><img class="highwire-fragment fragment-image" alt="Figure6" src="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261628/F6.medium.gif" width="440" height="296"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261628/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261628/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div></div></div><div class="section" id="sec-27"><h2 class="">DISCUSSION</h2><p id="p-43">A partial <em>C. albicans</em> Gateway<sup>™</sup>-adapted ORFeome (ORFeomeV1; 644 ORFs) and the corresponding <em>C. albicans</em> overexpression strains have already been successfully used to investigate morphogenesis, biofilm formation and genome dynamics in <em>C. albicans</em> (<a id="xref-ref-11-11" class="xref-bibr" href="#ref-11">11</a>,<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a>,<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>). In this study, we have now generated the <em>C. albicans</em> ORFeomeV2 within the Gateway<sup>™</sup> recombination cloning system, as well as a collection of destination vectors suitable for expression in <em>C. albicans</em>.</p><p id="p-44">This new <em>C. albicans</em> ORFeome resource encompasses 5,102 ORFs (83% of the annotated ORFs) and will be made available to the community upon request. The resource is supported by the CandidaOrfDB database, providing information on the individual plasmids and their sequences. The <em>C. albicans</em> ORFeomeV2 is of unprecedented quality, as all ORFs in the collection are fully sequenced, which remains an exception in large-scale ORFeome projects where ORFs are rarely sequenced in their entirety and only the 5’- and 3’-ends of the cloned ORFs are sequenced to confirm identity and the absence of frameshift mutations in the primers (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>,<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>,<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>,<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>,<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>,<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a>). Given these high standards, the success rate of 83% that we achieved with the <em>C. albicans</em> ORFeomeV2 is quite remarkable.</p><p id="p-45">In eukaryotes, ORFeomes are usually cloned from full-length cDNA libraries to account for the presence of introns and therefore splicing variants. However, only 6% of <em>C. albicans</em> genes have introns (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>), rendering the generation of the <em>C. albicans</em> ORFeome more straightforward than for pluricellular eukaryotes. Indeed, ORFs were simply amplified from the genomic DNA of the reference strain SC5314 and therefore, 360 ORFs in the <em>C. albicans</em> ORFeomeV2 harbor introns, a matter that should be taken into consideration when expressing ORFs in a prokaryotic host and, possibly, other eukaryotic hosts than <em>C. albicans</em>. A second aspect to be taken into consideration when using the ORFeome in hosts other than <em>C. albicans</em> is the unusual codon usage in this species (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>). Indeed, in <em>C. albicans</em>, CUG is decoded as a serine instead of a leucine the majority of the time. Hence, improper translation in hosts with a standard genetic code may distort protein structure and function.</p><p id="p-46">Sanger sequencing of both 5’ and 3’ ends was performed to confirm ORF identity and exclude clones containing primer or recombination errors. Previous studies have reported mutation rates in primers of 3-10% (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>,<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">29</a>). Similar rates were observed in this work. Unlike other Gateway<sup>™</sup> recombinational cloning projects (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">15</a>,<a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">29</a>), we did not see major differences in cloning efficiency for ORFs up to 4 kb. A decrease in success rate was observed only for ORFs &gt;4 kb. This size bias could be attributed to an increased difficulty to amplify the ORF by PCR, a reduced efficiency of the Gateway<sup>™</sup> BP Clonase<sup>™</sup> reactions with long PCR products, and the error rate of the PCR polymerase, which increases with longer products. In addition, we also corrected the sequences of 110 ORFs with sequence ambiguities (N-tracts and IUPAC code nucleotides) according to the Candida Genome Database and identified 116 ORFs that display a recombinant haplotype. Recombinant haplotypes could be explained by template switching during PCR cycles or by incorrectly phased SNPs in reference sequences. This could be addressed by performing long-read sequencing of the <em>C. albicans</em> SC5314 genome.</p><p id="p-47">However, many challenges remain to be addressed. For heterozygous genes, only one allele is present in the ORFeome. Because functional differences have been reported between the 2 alleles of a heterozygous gene (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>,<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>), it would be relevant to also clone and validate the second allele in these instances. In addition, oligonucleotides have been designed based on annotation of the haploid set of Assembly 19 of the <em>C. albicans</em> genome. As a consequence, genes of the <em>MTLα</em> locus have not been included in the ORFeome. Despite our efforts, the collection is still missing 1,081 clones. The next version of the <em>C. albicans</em> ORFeome could extend gene coverage by adding the missing ORFs, allelic variants or strain-specific variations.</p><p id="p-48">ORFeomes are essential to bridge the gap between genome annotation and systems biology and allow large-scale gene and protein characterization. The development of a collection of <em>C. albicans</em> constitutive or conditional overexpression strains is one of the applications of the <em>C. albicans</em> ORFeome that we have successfully explored (<a id="xref-ref-11-12" class="xref-bibr" href="#ref-11">11</a>,<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">31</a>,<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">54</a>). In this respect, the <em>C. albicans</em> ORFeome project is currently completing its second and third phases, which consist of transferring the 5,102 cloned ORFs into barcoded destination vectors and generating a collection of <em>C. albicans</em> barcoded overexpression mutants, each mutant carrying one of the 5,102 cloned ORFs under the control of the inducible P<em><sub>TET</sub></em> promoter. CandidaOrfDB already provides information on the available overexpression plasmids and strains. In this study, we have paved the way for a second application of the <em>C. albicans</em> ORFeome through the development of tools for a 2H matrix approach of protein-protein interaction detection via mating in <em>C. albicans</em>. Our data show that mating of diploid <em>C. albicans</em> expressing a bait fused to the LexA DNA binding domain and a prey fused to the VP16 activation domain, respectively, allows protein-protein interactions to be tested. In <em>S. cerevisiae</em>, large-scale 2H screens can be performed in a matrix design whereby haploid strains expressing baits and preys are mated and protein-protein interactions are scored in the resulting diploids. Our toolkit now enables the implementation of such a matrix design for large-scale 2H screens in <em>C. albicans</em>. To this aim, a collection of 1500 prey clones has already been generated. As mentioned above, <em>C. albicans</em> codon usage is unusual, which limits the development of applications of the <em>C. albicans</em> ORFeome in species that use standard decoding such as <em>S. cerevisiae</em>. Our development of tools for large-scale 2H screens in <em>C. albicans</em> circumvents this limitation, opening the path for the characterization of the <em>C. albicans</em> interactome. Defining the <em>C. albicans</em> interactome will undoubtedly impact our understanding of <em>C. albicans</em> pathogenesis in humans.</p><p id="p-49">In summary, high-level gene coverage coupled with the versatility of the Gateway<sup>™</sup> recombinational cloning, <em>C. albicans-</em> adapted functional genomics tools and full access to clones, make the <em>C. albicans</em> ORFeomeV2 a unique and valuable resource for the scientific community that should greatly facilitate future functional studies in <em>C. albicans</em>.</p></div><div class="section" id="sec-28"><h2 class="">FUNDING</h2><p id="p-51">This work has been supported by a Biomedical Resources grant from the Wellcome Trust (The <em>Candida albicans</em> ORFeome project, 088858/Z/09/Z to CAM and CD), the European Commission (FINSysB, PITN-GA-2008-214004 to CD), the Agence Nationale de la Recherche (KANJI, ANR-08-MIE-033-01 to CD; CANDICOL, ANR-10-01 to CD) and the Interuniversity Attraction Poles Programme initiated by the Belgian Science Policy Office for funding (IAP P7/28 to PVD). CAM would like to acknowledge support from a Medical Research Council New Investigator Award (G0400284) and the MRC Centre for Medical Mycology and the University of Aberdeen (MR/M026663/1). VC was the recipient of a PhD fellowship in the framework of the FINSysB consortium (PITN-GA-2008-214004). AN was the recipient of a PhD fellowship from the DIM-MalInf Région Ile-de-France. EP was the recipient of a post-doctoral fellowship in the framework of the <em>C. albicans</em> ORFeome project (WT088858MA). TR was the recipient of a postdoctoral fellowship in the framework of the NPARI consortium (LSHE-CT-2006-037692). UZ was the recipient of a postdoctoral fellowship in the framework of Programme Fungi from Institut Carnot-Pasteur Maladies Infectieuses. SZ was the recipient from post-doctoral fellowships in the framework of the FINSysB (PITN-GA-2008-214004) and KANJI (ANR-08-MIE-033-01) consortia. HT was the recipient of a KU Leuven CREA grant (2011-2013). High throughput sequencing has been performed on the Genomics Platform, member of France Génomique consortium (ANR10-INBS-09-08). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript. We acknowledge support from the French Government’s Investissement d’Avenir program (Laboratoire d’Excellence Integrative Biology of Emerging Infectious Diseases, ANR-10-LABX-62-IBEID).</p></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGEMENT</h2><p id="p-50">We are grateful to other members of our groups for their support throughout the development of the <em>C. albicans</em> ORFeome.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label">§</span> The authors wish it to be known that, in their opinion, the first two authors should be regarded as joint First Authors</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">REFERENCES</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.1" data-doi="10.1073/pnas.0401648101"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Federspiel</span>,  <span class="cit-name-given-names">N.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chibana</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dungan</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kalman</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Magee</span>,  <span class="cit-name-given-names">B.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Newport</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thorstenson</span>,  <span class="cit-name-given-names">Y.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Agabian</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Magee</span>,  <span class="cit-name-given-names">P.T.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">(2004)</span> <span class="cit-article-title">The diploid genome sequence of <em>Candida albicans</em></span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>., <span class="cit-vol">101</span>, <span class="cit-fpage">7329</span>–<span class="cit-lpage">7334</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0401648101%26rft_id%253Dinfo%253Apmid%252F15123810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzE5LzczMjkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wMi8wOC8yNjE2MjguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.2" data-doi="10.1186/gb-2013-14-9-r97"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Muzzey</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schwartz</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weissman</span>,  <span class="cit-name-given-names">J.S.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Sherlock</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-pub-date">(2013)</span> <span class="cit-article-title">Assembly of a phased diploid <em>Candida albicans</em> pathogenomics albicans genome facilitates allele-specific measurements and provides a simple model for repeat and indel structure</span>. <abbr class="cit-jnl-abbrev">Genome biology</abbr>, <span class="cit-vol">14</span>, <span class="cit-fpage">R97</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.volume%253D14%26rft.spage%253DR97%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-9-r97%26rft_id%253Dinfo%253Apmid%252F24025428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-9-r97&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24025428&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.3" data-doi="10.1093/nar/gki124"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">d’;Enfert</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goyard</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rodriguez-Arnaveilhe</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frangeul</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tekaia</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bader</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Albrecht</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Castillo</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dominguez</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">(2005)</span> <span class="cit-article-title">CandidaDB: a genome database for <em>Candida albicans</em> pathogenomics</span>. <abbr class="cit-jnl-abbrev">Nucl. Acids Res</abbr>., <span class="cit-vol">33 Database Issue</span>, <span class="cit-fpage">D353</span>–<span class="cit-lpage">357</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253Dd%2527Enfert%26rft.auinit1%253DC.%26rft.volume%253D33%26rft.issue%253Dsuppl_1%26rft.spage%253DD353%26rft.epage%253DD357%26rft.atitle%253DCandidaDB%253A%2Ba%2Bgenome%2Bdatabase%2Bfor%2BCandida%2Balbicans%2Bpathogenomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki124%26rft_id%253Dinfo%253Apmid%252F15608215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gki124&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15608215&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226524300073&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.4" data-doi="10.1371/journal.pgen.0010036"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Braun</span>,  <span class="cit-name-given-names">B.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Het Hoog</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">d’;Enfert</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martchenko</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dungan</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuo</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Inglis</span>,  <span class="cit-name-given-names">D.O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Uhl</span>,  <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hogues</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berriman</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">(2005)</span> <span class="cit-article-title">A Human-Curated Annotation of the <em>Candida albicans</em> Genome</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr>., <span class="cit-vol">1</span>, <span class="cit-fpage">36</span>–<span class="cit-lpage">57</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBraun%26rft.auinit1%253DB.%2BR.%26rft.volume%253D1%26rft.issue%253D1%26rft.spage%253D36%26rft.epage%253D57%26rft.atitle%253DA%2Bhuman-curated%2Bannotation%2Bof%2Bthe%2BCandida%2Balbicans%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0010036%26rft_id%253Dinfo%253Apmid%252F16103911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0010036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16103911&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234295900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.5" data-doi="10.1093/nar/gkw924"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Skrzypek</span>,  <span class="cit-name-given-names">M.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Binkley</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Binkley</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miyasato</span>,  <span class="cit-name-given-names">S.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simison</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Sherlock</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-pub-date">(2017)</span> <span class="cit-article-title">The Candida Genome Database (CGD): incorporation of Assembly 22, systematic identifiers and visualization of high throughput sequencing data</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">45</span>, <span class="cit-fpage">D592</span>–<span class="cit-lpage">D596</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw924%26rft_id%253Dinfo%253Apmid%252F27738138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw924&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27738138&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.6" data-doi="10.1046/j.1365-2958.2003.03697.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Roemer</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davison</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ketela</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Veillette</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Breton</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tandia</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Linteau</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sillaots</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marta</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">(2003)</span> <span class="cit-article-title">Large-scale essential gene identification in <em>Candida albicans</em> and applications to antifungal drug discovery</span>. <abbr class="cit-jnl-abbrev">Mol. Microbiol</abbr>., <span class="cit-vol">50</span>, <span class="cit-fpage">167</span>–<span class="cit-lpage">181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DRoemer%26rft.auinit1%253DT.%26rft.volume%253D50%26rft.issue%253D1%26rft.spage%253D167%26rft.epage%253D181%26rft.atitle%253DLarge-scale%2Bessential%2Bgene%2Bidentification%2Bin%2BCandida%2Balbicans%2Band%2Bapplications%2Bto%2Bantifungal%2Bdrug%2Bdiscovery.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.2003.03697.x%26rft_id%253Dinfo%253Apmid%252F14507372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.2003.03697.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14507372&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185446300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">7.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-261628v1.7" data-doi="10.1371/journal.ppat.0030092"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ketela</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lemieux</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Veillette</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martel</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davison</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sillaots</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Trosok</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bachewich</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">(2007)</span> <span class="cit-article-title">Genome-wide fitness test and mechanism-of-action studies of inhibitory compounds in <em>Candida albicans</em></span>. <abbr class="cit-jnl-abbrev">PLoS pathogens</abbr>, <span class="cit-vol">3</span>, <span class="cit-fpage">e92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DXu%26rft.auinit1%253DD.%26rft.volume%253D3%26rft.issue%253D6%26rft.spage%253De92%26rft.epage%253De92%26rft.atitle%253DGenome-wide%2Bfitness%2Btest%2Band%2Bmechanism-of-action%2Bstudies%2Bof%2Binhibitory%2Bcompounds%2Bin%2BCandida%2Balbicans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.0030092%26rft_id%253Dinfo%253Apmid%252F17604452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.0030092&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17604452&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">8.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-261628v1.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Homann</span>,  <span class="cit-name-given-names">O.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dea</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Noble</span>,  <span class="cit-name-given-names">S.M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Johnson</span>,  <span class="cit-name-given-names">A.D.</span></span> <span class="cit-pub-date">(2009)</span> <span class="cit-article-title">A phenotypic profile of the <em>Candida albicans</em> regulatory network</span>. <abbr class="cit-jnl-abbrev">PLoS genetics</abbr>, <span class="cit-vol">5</span>, <span class="cit-fpage">e1000783</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bgenetics%26rft.volume%253D5%26rft.spage%253D1000783e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.9" data-doi="10.1038/ng.605"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Noble</span>,  <span class="cit-name-given-names">S.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">French</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kohn</span>,  <span class="cit-name-given-names">L.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">V.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Johnson</span>,  <span class="cit-name-given-names">A.D.</span></span> <span class="cit-pub-date">(2010)</span> <span class="cit-article-title">Systematic screens of a <em>Candida albicans</em> homozygous deletion library decouple morphogenetic switching and pathogenicity</span>. <abbr class="cit-jnl-abbrev">Nat Genet</abbr>, <span class="cit-vol">42</span>, <span class="cit-fpage">590</span>–<span class="cit-lpage">598</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DNoble%26rft.auinit1%253DS.%2BM.%26rft.volume%253D42%26rft.issue%253D7%26rft.spage%253D590%26rft.epage%253D598%26rft.atitle%253DSystematic%2Bscreens%2Bof%2Ba%2BCandida%2Balbicans%2Bhomozygous%2Bdeletion%2Blibrary%2Bdecouple%2Bmorphogenetic%2Bswitching%2Band%2Bpathogenicity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.605%26rft_id%253Dinfo%253Apmid%252F20543849%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.605&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20543849&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.10" data-doi="10.1371/journal.pbio.1000363"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sahni</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yi</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Daniels</span>,  <span class="cit-name-given-names">K.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Srikantha</span>,  <span class="cit-name-given-names">T.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Soll</span>,  <span class="cit-name-given-names">D.R.</span></span> <span class="cit-pub-date">(2010)</span> <span class="cit-article-title">Tec1 mediates the pheromone response of the white phenotype of <em>Candida albicans</em>: insights into the evolution of new signal transduction pathways</span>. <abbr class="cit-jnl-abbrev">PLoS Biol</abbr>, <span class="cit-vol">8</span>, <span class="cit-fpage">e1000363</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DSahni%26rft.auinit1%253DN.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253De1000363%26rft.epage%253De1000363%26rft.atitle%253DTec1%2Bmediates%2Bthe%2Bpheromone%2Bresponse%2Bof%2Bthe%2Bwhite%2Bphenotype%2Bof%2BCandida%2Balbicans%253A%2Binsights%2Binto%2Bthe%2Bevolution%2Bof%2Bnew%2Bsignal%2Btransduction%2Bpathways.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000363%26rft_id%253Dinfo%253Apmid%252F20454615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20454615&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.11" data-doi="10.1371/journal.pone.0045912"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chauvel</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nesseir</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cabral</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Znaidi</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goyard</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bachellier-Bassi</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Firon</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Legrand</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Diogo</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Naulleau</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">(2012)</span> <span class="cit-article-title">A versatile overexpression strategy in the pathogenic yeast <em>Candida albicans</em>: identification of regulators of morphogenesis and fitness</span>. <abbr class="cit-jnl-abbrev">PloS one</abbr>, <span class="cit-vol">7</span>, <span class="cit-fpage">e45912</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DChauvel%26rft.auinit1%253DM.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253De45912%26rft.epage%253De45912%26rft.atitle%253DA%2BVersatile%2BOverexpression%2BStrategy%2Bin%2Bthe%2BPathogenic%2BYeast%2BCandida%2Balbicans%253A%2BIdentification%2Bof%2BRegulators%2Bof%2BMorphogenesis%2Band%2BFitness.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0045912%26rft_id%253Dinfo%253Apmid%252F23049891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0045912&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23049891&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.12" data-doi="10.1111/mmi.12327"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schillig</span>,  <span class="cit-name-given-names">R.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Morschhauser</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">(2013)</span> <span class="cit-article-title">Analysis of a fungus-specific transcription factor family, the <em>Candida albicans</em> zinc cluster proteins, by artificial activation</span>. <abbr class="cit-jnl-abbrev">Mol. Microbiol</abbr>., <span class="cit-vol">89</span>, <span class="cit-fpage">1003</span>–<span class="cit-lpage">1017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BMicrobiol%26rft.volume%253D89%26rft.spage%253D1003%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmmi.12327%26rft_id%253Dinfo%253Apmid%252F23844834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mmi.12327&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23844834&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.13" data-doi="10.1016/j.cbpa.2003.12.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rual</span>,  <span class="cit-name-given-names">J.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hill</span>,  <span class="cit-name-given-names">D.E.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Vidal</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">(2004)</span> <span class="cit-article-title">ORFeomeprojects: gateway between genomics and omics</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Chem. Biol</abbr>., <span class="cit-vol">8</span>, <span class="cit-fpage">20</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bchemical%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BChem%2BBiol%26rft.aulast%253DRual%26rft.auinit1%253DJ.%2BF.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253D20%26rft.epage%253D25%26rft.atitle%253DORFeome%2Bprojects%253A%2Bgateway%2Bbetween%2Bgenomics%2Band%2Bomics.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cbpa.2003.12.002%26rft_id%253Dinfo%253Apmid%252F15036152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cbpa.2003.12.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15036152&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000189359000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.14" data-doi="10.1038/nature02046"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ghaemmaghami</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huh</span>,  <span class="cit-name-given-names">W.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bower</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Howson</span>,  <span class="cit-name-given-names">R.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Belle</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dephoure</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’;Shea</span>,  <span class="cit-name-given-names">E.K.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Weissman</span>,  <span class="cit-name-given-names">J.S.</span></span> <span class="cit-pub-date">(2003)</span> <span class="cit-article-title">Global analysis of protein expression in yeast</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">425</span>, <span class="cit-fpage">737</span>–<span class="cit-lpage">741</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGhaemmaghami%26rft.auinit1%253DS.%26rft.volume%253D425%26rft.issue%253D6959%26rft.spage%253D737%26rft.epage%253D741%26rft.atitle%253DGlobal%2Banalysis%2Bof%2Bprotein%2Bexpression%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02046%26rft_id%253Dinfo%253Apmid%252F14562106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14562106&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185924500046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.15" data-doi="10.1038/ng1140"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reboul</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vaglio</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rual</span>,  <span class="cit-name-given-names">J.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lamesch</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martinez</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Armstrong</span>,  <span class="cit-name-given-names">C.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jacotot</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bertin</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Janky</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">(2003)</span> <span class="cit-article-title">C. elegans ORFeome version 1.1: experimental verification of the genome annotation and resource for proteome-scale protein expression</span>. <abbr class="cit-jnl-abbrev">Nat. Genet</abbr>., <span class="cit-vol">34</span>, <span class="cit-fpage">35</span>–<span class="cit-lpage">41</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DReboul%26rft.auinit1%253DJ.%26rft.volume%253D34%26rft.issue%253D1%26rft.spage%253D35%26rft.epage%253D41%26rft.atitle%253DC.%2Belegans%2BORFeome%2Bversion%2B1.1%253A%2Bexperimental%2Bverification%2Bof%2Bthe%2Bgenome%2Bannotation%2Band%2Bresource%2Bfor%2Bproteome-scale%2Bprotein%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1140%26rft_id%253Dinfo%253Apmid%252F12679813%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1140&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12679813&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182667900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.16" data-doi="10.1101/gad.1362105"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gelperin</span>,  <span class="cit-name-given-names">D.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">White</span>,  <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilkinson</span>,  <span class="cit-name-given-names">M.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kon</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kung</span>,  <span class="cit-name-given-names">L.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wise</span>,  <span class="cit-name-given-names">K.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lopez-Hoyo</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Piccirillo</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-etal">et al.</span><span class="cit-pub-date">(2005)</span> <span class="cit-article-title">Biochemical and genetic analysis of the yeast proteome with a movable ORF collection</span>. <abbr class="cit-jnl-abbrev">Genes Dev</abbr>., <span class="cit-vol">19</span>, <span class="cit-fpage">2816</span>–<span class="cit-lpage">2826</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DGelperin%26rft.auinit1%253DD.%2BM.%26rft.volume%253D19%26rft.issue%253D23%26rft.spage%253D2816%26rft.epage%253D2826%26rft.atitle%253DBiochemical%2Band%2Bgenetic%2Banalysis%2Bof%2Bthe%2Byeast%2Bproteome%2Bwith%2Ba%2Bmovable%2BORF%2Bcollection%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1362105%26rft_id%253Dinfo%253Apmid%252F16322557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE5LzIzLzI4MTYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wMi8wOC8yNjE2MjguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">17.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-261628v1.17" data-doi="10.1007/978-1-60761-232-2_2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Matsuyama</span>,  <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Yoshida</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">(2009)</span> <span class="cit-article-title">Systematic cloning of an ORFeome using the Gateway system</span>. <abbr class="cit-jnl-abbrev">Methods Mol. Biol</abbr>., <span class="cit-vol">577</span>, <span class="cit-fpage">11</span>–<span class="cit-lpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DMatsuyama%26rft.auinit1%253DA.%26rft.volume%253D577%26rft.spage%253D11%26rft.epage%253D24%26rft.atitle%253DSystematic%2Bcloning%2Bof%2Ban%2BORFeome%2Busing%2Bthe%2BGateway%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-60761-232-2_2%26rft_id%253Dinfo%253Apmid%252F19718505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-60761-232-2_2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19718505&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.18" data-doi="10.1186/1471-2164-11-470"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rajagopala</span>,  <span class="cit-name-given-names">S.V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yamamoto</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zweifel</span>,  <span class="cit-name-given-names">A.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nakamichi</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">H.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mendez-Rios</span>, <span class="cit-name-given-names">J.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Franca-Koh</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boorgula</span>,  <span class="cit-name-given-names">M.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fujita</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suzuki</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-etal">et al.</span><span class="cit-pub-date">(2010)</span> <span class="cit-article-title">The Escherichia coliK-12 ORFeome: a resource for comparative molecular microbiology</span>. <abbr class="cit-jnl-abbrev">BMC genomics</abbr>, <span class="cit-vol">11</span>, <span class="cit-fpage">470</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DRajagopala%26rft.auinit1%253DS.%2BV.%26rft.volume%253D11%26rft.spage%253D470%26rft.epage%253D470%26rft.atitle%253DThe%2BEscherichia%2Bcoli%2BK-12%2BORFeome%253A%2Ba%2Bresource%2Bfor%2Bcomparative%2Bmolecular%2Bmicrobiology.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-470%26rft_id%253Dinfo%253Apmid%252F20701780%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-470&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20701780&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">19.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-261628v1.19" data-doi="10.1104/pp.104.042176"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gong</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shen</span>,  <span class="cit-name-given-names">Y.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ma</span>,  <span class="cit-name-given-names">L.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pan</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Du</span>,  <span class="cit-name-given-names">Y.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">D.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">J.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">L.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">X.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dong</span>,  <span class="cit-name-given-names">C.X.</span></span> <span class="cit-etal">et al.</span><span class="cit-pub-date">(2004)</span> <span class="cit-article-title">Genome-wide ORFeome cloning and analysis of Arabidopsis transcription factor genes</span>. <abbr class="cit-jnl-abbrev">Plant Physiol</abbr>., <span class="cit-vol">135</span>, <span class="cit-fpage">773</span>–<span class="cit-lpage">782</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DGong%26rft.auinit1%253DW.%26rft.volume%253D135%26rft.issue%253D2%26rft.spage%253D773%26rft.epage%253D782%26rft.atitle%253DGenome-Wide%2BORFeome%2BCloning%2Band%2BAnalysis%2Bof%2BArabidopsis%2BTranscription%2BFactor%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.104.042176%26rft_id%253Dinfo%253Apmid%252F15208423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxMzUvMi83NzMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wMi8wOC8yNjE2MjguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">20.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-261628v1.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Underwood</span>,  <span class="cit-name-given-names">B.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vanderhaeghen</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whitford</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Town</span>,  <span class="cit-name-given-names">C.D.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Hilson</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-pub-date">(2006)</span> <span class="cit-article-title">Simultaneous high-throughput recombinational cloning of open reading frames in closed and open configurations</span>. <abbr class="cit-jnl-abbrev">Plant Biotechnol J</abbr>, <span class="cit-vol">4</span>, <span class="cit-fpage">317</span>–<span class="cit-lpage">324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BBiotechnol%2BJ%26rft.aulast%253DUnderwood%26rft.auinit1%253DB.%2BA.%26rft.volume%253D4%26rft.issue%253D3%26rft.spage%253D317%26rft.epage%253D324%26rft.atitle%253DSimultaneous%2Bhigh-throughput%2Brecombinational%2Bcloning%2Bof%2Bopen%2Breading%2Bframes%2Bin%2Bclosed%2Band%2Bopen%2Bconfigurations.%26rft_id%253Dinfo%253Apmid%252F17147637%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17147637&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.21" data-doi="10.1016/j.ydbio.2015.09.004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grant</span>,  <span class="cit-name-given-names">I.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Balcha</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hao</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shen</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Trivedi</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Patrushev</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fortriede</span>,  <span class="cit-name-given-names">J.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Karpinka</span>,  <span class="cit-name-given-names">J.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zorn</span>,  <span class="cit-name-given-names">A.M.</span></span> <span class="cit-etal">et al.</span><span class="cit-pub-date">(2015)</span> <span class="cit-article-title">The Xenopus ORFeome: A resource that enables functional genomics</span>. <abbr class="cit-jnl-abbrev">Dev. Biol</abbr>., <span class="cit-vol">408</span>, <span class="cit-fpage">345</span>–<span class="cit-lpage">357</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDev.%2BBiol%26rft.volume%253D408%26rft.spage%253D345%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2015.09.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2015.09.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">22.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-261628v1.22" data-doi="10.1242/dev.088757"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bischof</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bjorklund</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Furger</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schertel</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taipale</span>,  <span class="cit-name-given-names">J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Basler</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-pub-date">(2013)</span> <span class="cit-article-title">A versatile platform for creating a comprehensive UAS-ORFeome library in Drosophila</span>. <abbr class="cit-jnl-abbrev">Development</abbr>, <span class="cit-vol">140</span>, <span class="cit-fpage">2434</span>–<span class="cit-lpage">2442</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.088757%26rft_id%253Dinfo%253Apmid%252F23637332%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTQwLzExLzI0MzQiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wMi8wOC8yNjE2MjguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.23" data-doi="10.1101/gr.2456204"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dricot</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rual</span>,  <span class="cit-name-given-names">J.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lamesch</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bertin</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dupuy</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hao</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lambert</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hallez</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Delroisse</span>,  <span class="cit-name-given-names">J.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vandenhaute</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span><span class="cit-pub-date">(2004)</span> <span class="cit-article-title">Generation of the Brucella melitensisORFeome version 1.1</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr>., <span class="cit-vol">14</span>, <span class="cit-fpage">2201</span>–<span class="cit-lpage">2206</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DDricot%26rft.auinit1%253DA.%26rft.volume%253D14%26rft.issue%253D10b%26rft.spage%253D2201%26rft.epage%253D2206%26rft.atitle%253DGeneration%2Bof%2Bthe%2BBrucella%2Bmelitensis%2BORFeome%2BVersion%2B1.1%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.2456204%26rft_id%253Dinfo%253Apmid%252F15489343%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjExOiIxNC8xMGIvMjIwMSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzAyLzA4LzI2MTYyOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">24.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-261628v1.24" data-doi="10.1101/gr.2416604"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aguiar</span>,  <span class="cit-name-given-names">J.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">LaBaer</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blair</span>,  <span class="cit-name-given-names">P.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shamailova</span>,  <span class="cit-name-given-names">V.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koundinya</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Russell</span>,  <span class="cit-name-given-names">J.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mar</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anthony</span>,  <span class="cit-name-given-names">R.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Witney</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span><span class="cit-pub-date">(2004)</span> <span class="cit-article-title">High-throughput generation of P. falciparum functional molecules by recombinational cloning</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr>., <span class="cit-vol">14</span>, <span class="cit-fpage">2076</span>–<span class="cit-lpage">2082</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DAguiar%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D14%26rft.issue%253D10b%26rft.spage%253D2076%26rft.epage%253D2082%26rft.atitle%253DHigh-Throughput%2BGeneration%2Bof%2BP.%2Bfalciparum%2BFunctional%2BMolecules%2Bby%2BRecombinational%2BCloning%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.2416604%26rft_id%253Dinfo%253Apmid%252F15489329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjExOiIxNC8xMGIvMjA3NiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzAyLzA4LzI2MTYyOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">25.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-261628v1.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hauser</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ceol</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rajagopala</span>,  <span class="cit-name-given-names">S.V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mosca</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Siszler</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wermke</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sikorski</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schwarz</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schick</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wuchty</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-etal">et al.</span><span class="cit-pub-date">(2014)</span> <span class="cit-article-title">A second-generation protein-protein interaction network of Helicobacter pylori</span>. <abbr class="cit-jnl-abbrev">Molecular &amp; cellular proteomics: MCP</abbr>, <span class="cit-vol">13</span>, <span class="cit-fpage">1318</span>–<span class="cit-lpage">1329</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2B%2526%2Bcellular%2Bproteomics%253A%2BMCP%26rft.volume%253D13%26rft.spage%253D1318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.26" data-doi="10.1186/1471-2164-13-632"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maier</span>,  <span class="cit-name-given-names">C.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maier</span>,  <span class="cit-name-given-names">R.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Virok</span>,  <span class="cit-name-given-names">D.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maass</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hintner</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bauer</span>,  <span class="cit-name-given-names">J.W.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Onder</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-pub-date">(2012)</span> <span class="cit-article-title">Construction of a highly flexible and comprehensive gene collection representing the ORFeome of the human pathogen Chlamydia pneumoniae</span>. <abbr class="cit-jnl-abbrev">BMC genomics</abbr>, <span class="cit-vol">13</span>, <span class="cit-fpage">632</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMaier%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D632%26rft.epage%253D632%26rft.atitle%253DConstruction%2Bof%2Ba%2Bhighly%2Bflexible%2Band%2Bcomprehensive%2Bgene%2Bcollection%2Brepresenting%2Bthe%2BORFeome%2Bof%2Bthe%2Bhuman%2Bpathogen%2BChlamydia%2Bpneumoniae.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-632%26rft_id%253Dinfo%253Apmid%252F23157390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-632&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23157390&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">27.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-261628v1.27" data-doi="10.1186/1471-2164-9-321"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brandner</span>,  <span class="cit-name-given-names">C.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maier</span>,  <span class="cit-name-given-names">R.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Henderson</span>,  <span class="cit-name-given-names">D.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hintner</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bauer</span>,  <span class="cit-name-given-names">J.W.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Onder</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-pub-date">(2008)</span> <span class="cit-article-title">The ORFeome of Staphylococcus aureusv 1.1</span>. <abbr class="cit-jnl-abbrev">BMC genomics</abbr>, <span class="cit-vol">9</span>, <span class="cit-fpage">321</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DBrandner%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D9%26rft.spage%253D321%26rft.epage%253D321%26rft.atitle%253DThe%2BORFeome%2Bof%2BStaphylococcus%2Baureus%2Bv%2B1.1.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-321%26rft_id%253Dinfo%253Apmid%252F18605992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-321&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18605992&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.28" data-doi="10.1371/journal.pone.0000459"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">von Brunn</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Teepe</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simpson</span>,  <span class="cit-name-given-names">J.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pepperkok</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Friedel</span>,  <span class="cit-name-given-names">C.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zimmer</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roberts</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>,  <span class="cit-name-given-names">R.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Haas</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">(2007)</span> <span class="cit-article-title">Analysis of intraviral protein-protein interactions of the SARS coronavirus ORFeome</span>. <abbr class="cit-jnl-abbrev">PloS one</abbr>, <span class="cit-vol">2</span>, <span class="cit-fpage">e459</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253Dvon%2BBrunn%26rft.auinit1%253DA.%26rft.volume%253D2%26rft.issue%253D5%26rft.spage%253De459%26rft.epage%253De459%26rft.atitle%253DAnalysis%2Bof%2Bintraviral%2Bprotein-protein%2Binteractions%2Bof%2Bthe%2BSARS%2Bcoronavirus%2BORFeome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000459%26rft_id%253Dinfo%253Apmid%252F17520018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0000459&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17520018&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.29" data-doi="10.1016/j.ygeno.2006.11.012"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lamesch</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Milstein</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fan</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hao</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Szabo</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Venkatesan</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bethel</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martin</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-etal">et al.</span><span class="cit-pub-date">(2007)</span> <span class="cit-article-title">hORFeomev3.1: a resource of human open reading frames representing over 10,000 human genes</span>. <abbr class="cit-jnl-abbrev">Genomics</abbr>, <span class="cit-vol">89</span>, <span class="cit-fpage">307</span>–<span class="cit-lpage">315</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DLamesch%26rft.auinit1%253DP.%26rft.volume%253D89%26rft.issue%253D3%26rft.spage%253D307%26rft.epage%253D315%26rft.atitle%253DhORFeome%2Bv3.1%253A%2Ba%2Bresource%2Bof%2Bhuman%2Bopen%2Breading%2Bframes%2Brepresenting%2Bover%2B10%252C000%2Bhuman%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygeno.2006.11.012%26rft_id%253Dinfo%253Apmid%252F17207965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ygeno.2006.11.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17207965&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244572900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.30" data-doi="10.1016/S0076-6879(00)28419-X"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Walhout</span>,  <span class="cit-name-given-names">A.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Temple</span>,  <span class="cit-name-given-names">G.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brasch</span>,  <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hartley</span>,  <span class="cit-name-given-names">J.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lorson</span>,  <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van den Heuvel</span>,  <span class="cit-name-given-names">S.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Vidal</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">(2000)</span> <span class="cit-article-title">GATEWAY recombinational cloning: application to the cloning of large numbers of open reading frames or ORFeomes</span>. <abbr class="cit-jnl-abbrev">Methods Enzymol</abbr>., <span class="cit-vol">328</span>, <span class="cit-fpage">575</span>–<span class="cit-lpage">592</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Benzymology%26rft.stitle%253DMethods%2BEnzymol%26rft.aulast%253DWalhout%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D328%26rft.spage%253D575%26rft.epage%253D592%26rft.atitle%253DGATEWAY%2Brecombinational%2Bcloning%253A%2Bapplication%2Bto%2Bthe%2Bcloning%2Bof%2Blarge%2Bnumbers%2Bof%2Bopen%2Breading%2Bframes%2Bor%2BORFeomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0076-6879%252800%252928419-X%26rft_id%253Dinfo%253Apmid%252F11075367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0076-6879(00)28419-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11075367&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166565300034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.31" data-doi="10.1371/journal.ppat.1004542"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cabral</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Znaidi</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Walker</span>,  <span class="cit-name-given-names">L.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martin-Yken</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dague</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Legrand</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chauvel</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Firon</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rossignol</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-etal">et al.</span><span class="cit-pub-date">(2014)</span> <span class="cit-article-title">Targeted changes of the cell wall proteome influence <em>Candida albicans</em> ability to form single- and multi-strain biofilms</span>. <abbr class="cit-jnl-abbrev">PLoS pathogens</abbr>, <span class="cit-vol">10</span>, <span class="cit-fpage">e1004542</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bpathogens%26rft.volume%253D10%26rft.spage%253De1004542%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1004542%26rft_id%253Dinfo%253Apmid%252F25502890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1004542&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25502890&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.32" data-doi="10.1093/nar/gkq725"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stynen</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van Dijck</span>,  <span class="cit-name-given-names">P.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Tournu</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-pub-date">(2010)</span> <span class="cit-article-title">A CUG codon adapted two-hybrid system for the pathogenic fungus <em>Candida albicans</em></span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">38</span>, <span class="cit-fpage">e184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq725%26rft_id%253Dinfo%253Apmid%252F20719741%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq725&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20719741&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.33" data-doi="10.1016/S0022-2836(83)80284-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hanahan</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-pub-date">(1983)</span> <span class="cit-article-title">Studies on transformation of Escherichia coli with plasmids</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol</abbr>., <span class="cit-vol">166</span>, <span class="cit-fpage">557</span>–<span class="cit-lpage">580</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DHanahan%26rft.auinit1%253DD.%26rft.volume%253D166%26rft.issue%253D4%26rft.spage%253D557%26rft.epage%253D580%26rft.atitle%253DStudies%2Bon%2Btransformation%2Bof%2BEscherichia%2Bcoli%2Bwith%2Bplasmids.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0022-2836%252883%252980284-8%26rft_id%253Dinfo%253Apmid%252F6345791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0022-2836(83)80284-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6345791&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1983QW37900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.34" data-doi="10.1007/978-1-61779-539-8_15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cabral</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chauvel</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Firon</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Legrand</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nesseir</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bachellier-Bassi</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chaudhari</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Munro</span>,  <span class="cit-name-given-names">C.A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">d’;Enfert</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-pub-date">(2012)</span> <span class="cit-article-title">Modular Gene Over-expression Strategies for <em>Candida albicans</em></span>. <abbr class="cit-jnl-abbrev">Methods Mol Biol</abbr>, <span class="cit-vol">845</span>, <span class="cit-fpage">227</span>–<span class="cit-lpage">244</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DCabral%26rft.auinit1%253DV.%26rft.volume%253D845%26rft.spage%253D227%26rft.epage%253D244%26rft.atitle%253DModular%2Bgene%2Bover-expression%2Bstrategies%2Bfor%2BCandida%2Balbicans.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-61779-539-8_15%26rft_id%253Dinfo%253Apmid%252F22328378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-61779-539-8_15&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22328378&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.35" data-doi="10.1002/yea.1044"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gola</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martin</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Walther</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dunkler</span>,  <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Wendland</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">(2003)</span> <span class="cit-article-title">New modules for PCR-based gene targeting in <em>Candida albicans</em>: rapid and efficient gene targeting using 100 bp of flanking homology region</span>. <abbr class="cit-jnl-abbrev">Yeast</abbr>, <span class="cit-vol">20</span>, <span class="cit-fpage">1339</span>–<span class="cit-lpage">1347</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DGola%26rft.auinit1%253DS.%26rft.volume%253D20%26rft.issue%253D16%26rft.spage%253D1339%26rft.epage%253D1347%26rft.atitle%253DNew%2Bmodules%2Bfor%2BPCR-based%2Bgene%2Btargeting%2Bin%2BCandida%2Balbicans%253A%2Brapid%2Band%2Befficient%2Bgene%2Btargeting%2Busing%2B100%2Bbp%2Bof%2Bflanking%2Bhomology%2Bregion.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.1044%26rft_id%253Dinfo%253Apmid%252F14663826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.1044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14663826&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187610000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.36" data-doi="10.1128/EC.00327-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">T.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Znaidi</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barker</span>,  <span class="cit-name-given-names">K.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Homayouni</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saidane</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morschhauser</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nantel</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Raymond</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Rogers</span>,  <span class="cit-name-given-names">P.D.</span></span> <span class="cit-pub-date">(2007)</span> <span class="cit-article-title">Genome-wide expression and location analyses of the <em>Candida albicans</em> Tac1p regulon</span>. <abbr class="cit-jnl-abbrev">Eukaryotic cell</abbr>, <span class="cit-vol">6</span>, <span class="cit-fpage">2122</span>–<span class="cit-lpage">2138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2Bcell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00327-07%26rft_id%253Dinfo%253Apmid%252F17905926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo5OiI2LzExLzIxMjIiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wMi8wOC8yNjE2MjguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.37" data-doi="10.1128/IAI.00223-09"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Enjalbert</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rachini</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vediyappan</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pietrella</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spaccapelo</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vecchiarelli</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brown</span>,  <span class="cit-name-given-names">A.J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">d’;Enfert</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-pub-date">(2009)</span> <span class="cit-article-title">A multifunctional, synthetic Gaussia princeps luciferase reporter for live imaging of <em>Candida albicans</em> infections</span>. <abbr class="cit-jnl-abbrev">Infect Immun</abbr>, <span class="cit-vol">77</span>, <span class="cit-fpage">4847</span>–<span class="cit-lpage">4858</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfect%2BImmun%26rft_id%253Dinfo%253Adoi%252F10.1128%252FIAI.00223-09%26rft_id%253Dinfo%253Apmid%252F19687206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaWFpIjtzOjU6InJlc2lkIjtzOjEwOiI3Ny8xMS80ODQ3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDIvMDgvMjYxNjI4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.38" data-doi="10.1016/j.gene.2004.06.021"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reuss</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vik</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kolter</span>,  <span class="cit-name-given-names">R.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Morschhauser</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">(2004)</span> <span class="cit-article-title">The SAT1flipper, an optimized tool for gene disruption in <em>Candida albicans</em></span>. <abbr class="cit-jnl-abbrev">Gene</abbr>, <span class="cit-vol">341</span>, <span class="cit-fpage">119</span>–<span class="cit-lpage">127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DReuss%26rft.auinit1%253DO.%26rft.volume%253D341%26rft.spage%253D119%26rft.epage%253D127%26rft.atitle%253DThe%2BSAT1%2Bflipper%252C%2Ban%2Boptimized%2Btool%2Bfor%2Bgene%2Bdisruption%2Bin%2BCandida%2Balbicans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2004.06.021%26rft_id%253Dinfo%253Apmid%252F15474295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2004.06.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15474295&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224765100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.39" data-doi="10.1038/nprot.2008.137"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Walther</span>,  <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Wendland</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">(2008)</span> <span class="cit-article-title">PCR-based gene targeting in <em>Candida albicans</em></span>. <abbr class="cit-jnl-abbrev">Nat Protoc</abbr>, <span class="cit-vol">3</span>, <span class="cit-fpage">1414</span>–<span class="cit-lpage">1421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DWalther%26rft.auinit1%253DA.%26rft.volume%253D3%26rft.issue%253D9%26rft.spage%253D1414%26rft.epage%253D1421%26rft.atitle%253DPCR-based%2Bgene%2Btargeting%2Bin%2BCandida%2Balbicans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2008.137%26rft_id%253Dinfo%253Apmid%252F18772868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2008.137&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18772868&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.40" data-doi="10.1128/EC.4.8.1328-1342.2005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Park</span>,  <span class="cit-name-given-names">Y.N.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Morschhauser</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">(2005)</span> <span class="cit-article-title">Tetracycline-inducible gene expression and gene deletion in <em>Candida albicans</em></span>. <abbr class="cit-jnl-abbrev">Eukaryotic cell</abbr>, <span class="cit-vol">4</span>, <span class="cit-fpage">1328</span>–<span class="cit-lpage">1342</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DPark%26rft.auinit1%253DY.-N.%26rft.volume%253D4%26rft.issue%253D8%26rft.spage%253D1328%26rft.epage%253D1342%26rft.atitle%253DTetracycline-Inducible%2BGene%2BExpression%2Band%2BGene%2BDeletion%2Bin%2BCandida%2Balbicans%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.4.8.1328-1342.2005%26rft_id%253Dinfo%253Apmid%252F16087738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo4OiI0LzgvMTMyOCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzAyLzA4LzI2MTYyOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.41" data-doi="10.1073/pnas.0605270103"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chou</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nie</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-pub-date">(2006)</span> <span class="cit-article-title">Bistable expression of WOR1, a master regulator of white-opaque switching in <em>Candida albicans</em></span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>., <span class="cit-vol">103</span>, <span class="cit-fpage">12813</span>–<span class="cit-lpage">12818</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0605270103%26rft_id%253Dinfo%253Apmid%252F16905649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzM0LzEyODEzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDIvMDgvMjYxNjI4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.42" data-doi="10.1126/science.7992058"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kohler</span>,  <span class="cit-name-given-names">J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Fink</span>,  <span class="cit-name-given-names">G.R.</span></span> <span class="cit-pub-date">(1994)</span> <span class="cit-article-title">Suppression of hyphal formation in <em>Candida albicans</em>by mutation of a STE12 homolog</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-vol">266</span>, <span class="cit-fpage">1723</span>–<span class="cit-lpage">1726</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLiu%26rft.auinit1%253DH%26rft.volume%253D266%26rft.issue%253D5191%26rft.spage%253D1723%26rft.epage%253D1726%26rft.atitle%253DSuppression%2Bof%2Bhyphal%2Bformation%2Bin%2BCandida%2Balbicans%2Bby%2Bmutation%2Bof%2Ba%2BSTE12%2Bhomolog%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.7992058%26rft_id%253Dinfo%253Apmid%252F7992058%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNjYvNTE5MS8xNzIzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDIvMDgvMjYxNjI4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.43" data-doi="10.1093/emboj/cdg235"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bennett</span>,  <span class="cit-name-given-names">R.J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Johnson</span>,  <span class="cit-name-given-names">A.D.</span></span> <span class="cit-pub-date">(2003)</span> <span class="cit-article-title">Completion of a parasexual cycle in <em>Candida albicans</em> by induced chromosome loss in tetraploid strains</span>. <abbr class="cit-jnl-abbrev">EMBO J</abbr>., <span class="cit-vol">22</span>, <span class="cit-fpage">2505</span>–<span class="cit-lpage">2515</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DBennett%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D22%26rft.issue%253D10%26rft.spage%253D2505%26rft.epage%253D2515%26rft.atitle%253DCompletion%2Bof%2Ba%2Bparasexual%2Bcycle%2Bin%2BCandida%2Balbicans%2Bby%2Binduced%2Bchromosome%2Bloss%2Bin%2Btetraploid%2Bstrains%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252Fcdg235%26rft_id%253Dinfo%253Apmid%252F12743044%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjIvMTAvMjUwNSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzAyLzA4LzI2MTYyOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.44" data-doi="10.1186/gb-2007-8-4-r52"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van het Hoog</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rast</span>,  <span class="cit-name-given-names">T.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martchenko</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grindle</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dignard</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hogues</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cuomo</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berriman</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scherer</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Magee</span>,  <span class="cit-name-given-names">B.B.</span></span> <span class="cit-etal">et al.</span><span class="cit-pub-date">(2007)</span> <span class="cit-article-title">Assembly of the <em>Candida albicans</em> genome into sixteen supercontigs aligned on the eight chromosomes</span>. <abbr class="cit-jnl-abbrev">Genome biology</abbr>, <span class="cit-vol">8</span>, <span class="cit-fpage">R52</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253Dvan%2BHet%2BHoog%26rft.auinit1%253DM.%26rft.volume%253D8%26rft.issue%253D4%26rft.spage%253DR52%26rft.epage%253DR52%26rft.atitle%253DAssembly%2Bof%2Bthe%2BCandida%2Balbicans%2Bgenome%2Binto%2Bsixteen%2Bsupercontigs%2Baligned%2Bon%2Bthe%2Beight%2Bchromosomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2007-8-4-r52%26rft_id%253Dinfo%253Apmid%252F17419877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2007-8-4-r52&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17419877&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fonzi</span>,  <span class="cit-name-given-names">W.A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Irwin</span>,  <span class="cit-name-given-names">M.Y.</span></span> <span class="cit-pub-date">(1993)</span> <span class="cit-article-title">Isogenic strain construction and gene mapping in <em>Candida albicans</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">134</span>, <span class="cit-fpage">717</span>–<span class="cit-lpage">728</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFonzi%26rft.auinit1%253DW.%2BA.%26rft.volume%253D134%26rft.issue%253D3%26rft.spage%253D717%26rft.epage%253D728%26rft.atitle%253DIsogenic%2Bstrain%2Bconstruction%2Band%2Bgene%2Bmapping%2Bin%2BCandida%2Balbicans.%26rft_id%253Dinfo%253Apmid%252F8349105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM0LzMvNzE3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDIvMDgvMjYxNjI4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-web" id="cit-261628v1.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Skrzypek</span>,  <span class="cit-name-given-names">M.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Binkley</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Binkley</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miyasato</span>,  <span class="cit-name-given-names">S.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simison</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Sherlock</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">“Candida Genome Database”</span> <a href="http://www.candidagenome.org/">http://www.candidagenome.org/</a> <span class="cit-pub-date">(2015)</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.47" data-doi="10.1002/yea.1502"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Alberti</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gitler</span>,  <span class="cit-name-given-names">A.D.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Lindquist</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-pub-date">(2007)</span> <span class="cit-article-title">A suite of Gateway cloning vectors for high-throughput genetic analysis in Saccharomyces cerevisiae</span>. <abbr class="cit-jnl-abbrev">Yeast</abbr>, <span class="cit-vol">24</span>, <span class="cit-fpage">913</span>–<span class="cit-lpage">919</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DAlberti%26rft.auinit1%253DS.%26rft.volume%253D24%26rft.issue%253D10%26rft.spage%253D913%26rft.epage%253D919%26rft.atitle%253DA%2Bsuite%2Bof%2BGateway%2Bcloning%2Bvectors%2Bfor%2Bhigh-throughput%2Bgenetic%2Banalysis%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.1502%26rft_id%253Dinfo%253Apmid%252F17583893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.1502&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17583893&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250352000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.48" data-doi="10.1128/AEM.01873-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hallez</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Letesson</span>,  <span class="cit-name-given-names">J.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vandenhaute</span>,  <span class="cit-name-given-names">J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">De Bolle</span>,  <span class="cit-name-given-names">X.</span></span> <span class="cit-pub-date">(2007)</span> <span class="cit-article-title">Gateway-based destination vectors for functional analyses of bacterial ORFeomes: application to the Min system in Brucella abortus</span>. <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol</abbr>., <span class="cit-vol">73</span>, <span class="cit-fpage">1375</span>–<span class="cit-lpage">1379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.01873-06%26rft_id%253Dinfo%253Apmid%252F17172460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjk6IjczLzQvMTM3NSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzAyLzA4LzI2MTYyOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.49" data-doi="10.1073/pnas.0804061106"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carlisle</span>,  <span class="cit-name-given-names">P.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Banerjee</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lazzell</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Monteagudo</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lopez-Ribot</span>,  <span class="cit-name-given-names">J.L.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Kadosh</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-pub-date">(2009)</span> <span class="cit-article-title">Expression levels of a filament-specific transcriptional regulator are sufficient to determine <em>Candida albicans</em> morphology and virulence</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>., <span class="cit-vol">106</span>, <span class="cit-fpage">599</span>–<span class="cit-lpage">604</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0804061106%26rft_id%253Dinfo%253Apmid%252F19116272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMDYvMi81OTkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wMi8wOC8yNjE2MjguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.50" data-doi="10.1111/j.1567-1364.2008.00459.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zeidler</span>,  <span class="cit-name-given-names">U.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lettner</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lassnig</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muller</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lajko</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hintner</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Breitenbach</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Bito</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">(2009)</span> <span class="cit-article-title">UME6 is a crucial downstream target of other transcriptional regulators of true hyphal development in <em>Candida albicans</em></span>. <abbr class="cit-jnl-abbrev">FEMS yeast research</abbr>, <span class="cit-vol">9</span>, <span class="cit-fpage">126</span>–<span class="cit-lpage">142</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2Byeast%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2008.00459.x%26rft_id%253Dinfo%253Apmid%252F19054126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1567-1364.2008.00459.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19054126&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262314000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.51" data-doi="10.1016/j.mib.2015.06.017"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bennett</span>,  <span class="cit-name-given-names">R.J.</span></span> <span class="cit-pub-date">(2015)</span> <span class="cit-article-title">The parasexual lifestyle of <em>Candida albicans</em></span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Microbiol</abbr>., <span class="cit-vol">28</span>, <span class="cit-fpage">10</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BMicrobiol%26rft.volume%253D28%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mib.2015.06.017%26rft_id%253Dinfo%253Apmid%252F26210747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mib.2015.06.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26210747&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.52" data-doi="10.1016/S0092-8674(02)00837-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Miller</span>,  <span class="cit-name-given-names">M.G.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Johnson</span>,  <span class="cit-name-given-names">A.D.</span></span> <span class="cit-pub-date">(2002)</span> <span class="cit-article-title">White-opaque switching in <em>Candida albicans</em> is controlled by mating-type locus homeodomain proteins and allows efficient mating</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>, <span class="cit-vol">110</span>, <span class="cit-fpage">293</span>–<span class="cit-lpage">302</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMiller%26rft.auinit1%253DM.%2BG.%26rft.volume%253D110%26rft.issue%253D3%26rft.spage%253D293%26rft.epage%253D302%26rft.atitle%253DWhite-opaque%2Bswitching%2Bin%2BCandida%2Balbicans%2Bis%2Bcontrolled%2Bby%2Bmating-type%2Blocus%2Bhomeodomain%2Bproteins%2Band%2Ballows%2Befficient%2Bmating.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252802%252900837-1%26rft_id%253Dinfo%253Apmid%252F12176317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(02)00837-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12176317&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177411800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.53" data-doi="10.1073/pnas.0605138103"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zordan</span>,  <span class="cit-name-given-names">R.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Galgoczy</span>,  <span class="cit-name-given-names">D.J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Johnson</span>,  <span class="cit-name-given-names">A.D.</span></span> <span class="cit-pub-date">(2006)</span> <span class="cit-article-title">Epigenetic properties of white-opaque switching in <em>Candida albicans</em> are based on a self-sustaining transcriptional feedback loop</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>., <span class="cit-vol">103</span>, <span class="cit-fpage">12807</span>–<span class="cit-lpage">12812</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0605138103%26rft_id%253Dinfo%253Apmid%252F16899543%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzM0LzEyODA3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDIvMDgvMjYxNjI4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.54" data-doi="10.1128/EC.00286-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Loll-Krippleber</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feri</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nguyen</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maufrais</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yansouni</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">d’;Enfert</span>,  <span class="cit-name-given-names">C.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Legrand</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">(2015)</span> <span class="cit-article-title">A FACS-optimized screen identifies regulators of genome stability in <em>Candida albicans</em></span>. <abbr class="cit-jnl-abbrev">Eukaryotic cell</abbr>, <span class="cit-vol">14</span>, <span class="cit-fpage">311</span>–<span class="cit-lpage">322</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2Bcell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00286-14%26rft_id%253Dinfo%253Apmid%252F25595446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo4OiIxNC8zLzMxMSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzAyLzA4LzI2MTYyOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.55" data-doi="10.1101/gr.6111907"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mitrovich</span>,  <span class="cit-name-given-names">Q.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tuch</span>,  <span class="cit-name-given-names">B.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guthrie</span>,  <span class="cit-name-given-names">C.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Johnson</span>,  <span class="cit-name-given-names">A.D.</span></span> <span class="cit-pub-date">(2007)</span> <span class="cit-article-title">Computational and experimental approaches double the number of known introns in the pathogenic yeast <em>Candida albicans</em></span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr>., <span class="cit-vol">17</span>, <span class="cit-fpage">492</span>–<span class="cit-lpage">502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.6111907%26rft_id%253Dinfo%253Apmid%252F17351132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE3LzQvNDkyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDIvMDgvMjYxNjI4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.56" data-doi="10.1093/nar/23.9.1481"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Santos</span>,  <span class="cit-name-given-names">M.A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Tuite</span>,  <span class="cit-name-given-names">M.F.</span></span> <span class="cit-pub-date">(1995)</span> <span class="cit-article-title">The CUG codon is decoded in vivo as serine and not leucine in <em>Candida albicans</em></span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">23</span>, <span class="cit-fpage">1481</span>–<span class="cit-lpage">1486</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSantos%26rft.auinit1%253DM.%2BA.S.%26rft.volume%253D23%26rft.issue%253D9%26rft.spage%253D1481%26rft.epage%253D1486%26rft.atitle%253DThe%2BCUG%2Bcodon%2Bis%2Bdecoded%2Bin%2Bvivo%2Bas%2Bserine%2Band%2Bnot%2Bleucine%2Bin%2BCandida%2Balbicans%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F23.9.1481%26rft_id%253Dinfo%253Apmid%252F7784200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/23.9.1481&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7784200&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QZ47700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.57" data-doi="10.1534/genetics.105.054767"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Coste</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Turner</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ischer</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morschhauser</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Forche</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Selmecki</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berman</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bille</span>,  <span class="cit-name-given-names">J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Sanglard</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-pub-date">(2006)</span> <span class="cit-article-title">A mutation in Tac1p, a transcription factor regulating CDR1 and CDR2, is coupled with loss of heterozygosity at chromosome 5 to mediate antifungal resistance in <em>Candida albicans</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">172</span>, <span class="cit-fpage">2139</span>–<span class="cit-lpage">2156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.054767%26rft_id%253Dinfo%253Apmid%252F16452151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi80LzIxMzkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wMi8wOC8yNjE2MjguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-261628v1.58" data-doi="10.1111/j.1365-2958.2006.05357.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">A.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tsao</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ong</span>,  <span class="cit-name-given-names">S.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lamping</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Niimi</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Monk</span>,  <span class="cit-name-given-names">B.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Niimi</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kaneko</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holland</span>,  <span class="cit-name-given-names">B.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmid</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span><span class="cit-pub-date">(2006)</span> <span class="cit-article-title">Heterozygosity and functional allelic variation in the <em>Candida albicans</em> efflux pump genes CDR1 and CDR2</span>. <abbr class="cit-jnl-abbrev">Mol. Microbiol</abbr>., <span class="cit-vol">62</span>, <span class="cit-fpage">170</span>–<span class="cit-lpage">186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DHolmes%26rft.auinit1%253DA.%2BR.%26rft.volume%253D62%26rft.issue%253D1%26rft.spage%253D170%26rft.epage%253D186%26rft.atitle%253DHeterozygosity%2Band%2Bfunctional%2Ballelic%2Bvariation%2Bin%2Bthe%2BCandida%2Balbicans%2Befflux%2Bpump%2Bgenes%2BCDR1%2Band%2BCDR2.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2006.05357.x%26rft_id%253Dinfo%253Apmid%252F16942600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2006.05357.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16942600&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261628.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">59.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-261628v1.59" data-doi="10.1128/EC.4.2.298-309.2005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Noble</span>,  <span class="cit-name-given-names">S.M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Johnson</span>,  <span class="cit-name-given-names">A.D.</span></span> <span class="cit-pub-date">(2005)</span> <span class="cit-article-title">Strains and strategies for large-scale gene deletion studies of the diploid human fungal pathogen <em>Candida albicans</em></span>. <abbr class="cit-jnl-abbrev">Eukaryotic cell</abbr>, <span class="cit-vol">4</span>, <span class="cit-fpage">298</span>–<span class="cit-lpage">309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DNoble%26rft.auinit1%253DS.%2BM.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253D298%26rft.epage%253D309%26rft.atitle%253DStrains%2Band%2BStrategies%2Bfor%2BLarge-Scale%2BGene%2BDeletion%2BStudies%2Bof%2Bthe%2BDiploid%2BHuman%2BFungal%2BPathogen%2BCandida%2Balbicans%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.4.2.298-309.2005%26rft_id%253Dinfo%253Apmid%252F15701792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiI0LzIvMjk4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDIvMDgvMjYxNjI4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/261628v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F261628v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/262022v1" title="Descriptors of Pain Quality Associate with Pulpal and Periapical Diagnostic Tests in Patients Experiencing Acute Tooth Pain" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/262246v1" title="Neuroligins mediate presynaptic maturation through brain-derived neurotrophic factor signaling" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;February 08, 2018.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/261628v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/261628v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/261628v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/261628v1" class="active" data-icon-position="" data-hide-link-title="0">Generating genomic platforms to study Candida albicans pathogenesis</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/80349" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-eN2cH2Nkv15Uf5q6P2zwjkGBFgn3ax7sjFv-Pkrx2n8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="80349" id="node80349--2" data-pisa="biorxiv;261628v1" data-pisa-master="biorxiv;261628" data-seqnum="80349" data-apath="/biorxiv/early/2018/02/08/261628.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Generating genomic platforms to study <em>Candida albicans</em> pathogenesis</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Mélanie</span> <span class="nlm-surname">Legrand</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Sophie</span> <span class="nlm-surname">Bachellier-Bassi</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Keunsook K.</span> <span class="nlm-surname">Lee</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Yogesh</span> <span class="nlm-surname">Chaudhari</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Hélène</span> <span class="nlm-surname">Tournu</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Laurence</span> <span class="nlm-surname">Arbogast</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Hélène</span> <span class="nlm-surname">Boyer</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Murielle</span> <span class="nlm-surname">Chauvel</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Vitor</span> <span class="nlm-surname">Cabral</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Corinne</span> <span class="nlm-surname">Maufrais</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Audrey</span> <span class="nlm-surname">Nesseir</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Irena</span> <span class="nlm-surname">Maslanka</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Emmanuelle</span> <span class="nlm-surname">Permal</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Tristan</span> <span class="nlm-surname">Rossignol</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Louise A.</span> <span class="nlm-surname">Walker</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Ute</span> <span class="nlm-surname">Zeidler</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">Sadri</span> <span class="nlm-surname">Znaidi</span></span>, <span class="highwire-citation-author" data-delta="17"><span class="nlm-given-names">Floris</span> <span class="nlm-surname">Schoeters</span></span>, <span class="highwire-citation-author" data-delta="18"><span class="nlm-given-names">Charlotte</span> <span class="nlm-surname">Majgier</span></span>, <span class="highwire-citation-author" data-delta="19"><span class="nlm-given-names">Renaud A.</span> <span class="nlm-surname">Julien</span></span>, <span class="highwire-citation-author" data-delta="20"><span class="nlm-given-names">Laurence</span> <span class="nlm-surname">Ma</span></span>, <span class="highwire-citation-author" data-delta="21"><span class="nlm-given-names">Magali</span> <span class="nlm-surname">Tichit</span></span>, <span class="highwire-citation-author" data-delta="22"><span class="nlm-given-names">Christiane</span> <span class="nlm-surname">Bouchier</span></span>, <span class="highwire-citation-author" data-delta="23"><span class="nlm-given-names">Patrick Van</span> <span class="nlm-surname">Dijck</span></span>, <span class="highwire-citation-author" data-delta="24"><span class="nlm-given-names">Carol A.</span> <span class="nlm-surname">Munro</span></span>, <span class="highwire-citation-author" data-delta="25"><span class="nlm-given-names">Christophe</span> <span class="nlm-surname">d’Enfert</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">261628; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/261628 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/261628v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/261628v1&amp;title=Generating%20genomic%20platforms%20to%20study%20Candida%20albicans%20pathogenesis" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/261628v1&amp;title=Generating%20genomic%20platforms%20to%20study%20Candida%20albicans%20pathogenesis" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/261628v1&amp;text=Generating%20genomic%20platforms%20to%20study%20Candida%20albicans%20pathogenesis" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/261628v1&amp;title=Generating%20genomic%20platforms%20to%20study%20Candida%20albicans%20pathogenesis" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/261628v1&amp;t=Generating%20genomic%20platforms%20to%20study%20Candida%20albicans%20pathogenesis" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/261628v1&amp;title=Generating%20genomic%20platforms%20to%20study%20Candida%20albicans%20pathogenesis" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/261628v1&amp;title=Generating%20genomic%20platforms%20to%20study%20Candida%20albicans%20pathogenesis" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="80349" id="node80349--3" data-pisa="biorxiv;261628v1" data-pisa-master="biorxiv;261628" data-seqnum="80349" data-apath="/biorxiv/early/2018/02/08/261628.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Generating genomic platforms to study <em>Candida albicans</em> pathogenesis</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Mélanie</span> <span class="nlm-surname">Legrand</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Sophie</span> <span class="nlm-surname">Bachellier-Bassi</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Keunsook K.</span> <span class="nlm-surname">Lee</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Yogesh</span> <span class="nlm-surname">Chaudhari</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Hélène</span> <span class="nlm-surname">Tournu</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Laurence</span> <span class="nlm-surname">Arbogast</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Hélène</span> <span class="nlm-surname">Boyer</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Murielle</span> <span class="nlm-surname">Chauvel</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Vitor</span> <span class="nlm-surname">Cabral</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Corinne</span> <span class="nlm-surname">Maufrais</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Audrey</span> <span class="nlm-surname">Nesseir</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Irena</span> <span class="nlm-surname">Maslanka</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Emmanuelle</span> <span class="nlm-surname">Permal</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Tristan</span> <span class="nlm-surname">Rossignol</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Louise A.</span> <span class="nlm-surname">Walker</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Ute</span> <span class="nlm-surname">Zeidler</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">Sadri</span> <span class="nlm-surname">Znaidi</span></span>, <span class="highwire-citation-author" data-delta="17"><span class="nlm-given-names">Floris</span> <span class="nlm-surname">Schoeters</span></span>, <span class="highwire-citation-author" data-delta="18"><span class="nlm-given-names">Charlotte</span> <span class="nlm-surname">Majgier</span></span>, <span class="highwire-citation-author" data-delta="19"><span class="nlm-given-names">Renaud A.</span> <span class="nlm-surname">Julien</span></span>, <span class="highwire-citation-author" data-delta="20"><span class="nlm-given-names">Laurence</span> <span class="nlm-surname">Ma</span></span>, <span class="highwire-citation-author" data-delta="21"><span class="nlm-given-names">Magali</span> <span class="nlm-surname">Tichit</span></span>, <span class="highwire-citation-author" data-delta="22"><span class="nlm-given-names">Christiane</span> <span class="nlm-surname">Bouchier</span></span>, <span class="highwire-citation-author" data-delta="23"><span class="nlm-given-names">Patrick Van</span> <span class="nlm-surname">Dijck</span></span>, <span class="highwire-citation-author" data-delta="24"><span class="nlm-given-names">Carol A.</span> <span class="nlm-surname">Munro</span></span>, <span class="highwire-citation-author" data-delta="25"><span class="nlm-given-names">Christophe</span> <span class="nlm-surname">d’Enfert</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">261628; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/261628 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Generating%20genomic%20platforms%20to%20study%20Candida%20albicans%20pathogenesis&amp;rft.volume&amp;rft.issue&amp;rft.date=2018-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Legrand%2C+M%C3%A9lanie&amp;rft.au=Bachellier-Bassi%2C+Sophie&amp;rft.au=Lee%2C+Keunsook+K.&amp;rft.au=Chaudhari%2C+Yogesh&amp;rft.au=Tournu%2C+H%C3%A9l%C3%A8ne&amp;rft.au=Arbogast%2C+Laurence&amp;rft.au=Boyer%2C+H%C3%A9l%C3%A8ne&amp;rft.au=Chauvel%2C+Murielle&amp;rft.au=Cabral%2C+Vitor&amp;rft.au=Maufrais%2C+Corinne&amp;rft.au=Nesseir%2C+Audrey&amp;rft.au=Maslanka%2C+Irena&amp;rft.au=Permal%2C+Emmanuelle&amp;rft.au=Rossignol%2C+Tristan&amp;rft.au=Walker%2C+Louise+A.&amp;rft.au=Zeidler%2C+Ute&amp;rft.au=Znaidi%2C+Sadri&amp;rft.au=Schoeters%2C+Floris&amp;rft.au=Majgier%2C+Charlotte&amp;rft.au=Julien%2C+Renaud+A.&amp;rft.au=Ma%2C+Laurence&amp;rft.au=Tichit%2C+Magali&amp;rft.au=Bouchier%2C+Christiane&amp;rft.au=Dijck%2C+Patrick+Van&amp;rft.au=Munro%2C+Carol+A.&amp;rft.au=d%E2%80%99Enfert%2C+Christophe"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/80349/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/80349/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/80349/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/80349/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/80349/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/80349/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/80349/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/80349/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/80349/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/80349/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/80349/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/80349/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/261628v1&amp;count=horizontal&amp;via=&amp;text=Generating%20genomic%20platforms%20to%20study%20Candida%20albicans%20pathogenesis&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/261628v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/261628v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/261628v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/genomics" class="highlight" data-icon-position="" data-hide-link-title="0">Genomics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20277)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
