<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Nat Microbiol</span><span class="iso-abbrev" title="iso-abbrev">Nat Microbiol</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Nature Microbiology</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2058-5276</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Nature Publishing Group UK</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7097208">7097208</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/31969714">31969714</a></span><span class="publisher-id" title="publisher-id">670</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1038/s41564-020-0670-8">10.1038/s41564-020-0670-8</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Editorial</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Rapid outbreak response requires trust</div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-1-1</span></span><span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2020</span></span><span class="volume" tagx="volume" title="volume">5</span><span class="issue" tagx="issue" title="issue">2</span><span class="fpage" tagx="fpage" title="fpage">227</span><span class="lpage" tagx="lpage" title="lpage">228</span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p">This article is made available via the PMC Open Access Subset for unrestricted research re-use and secondary analysis in any form or by any means with acknowledgement of the original source. These permissions are granted for the duration of the World Health Organization (WHO) declaration of COVID-19 as a global pandemic.</span></span></div> <div class="abstract" title="abstract"> <p id="Par1">As the international community responds to an outbreak of coronavirus-induced pneumonia in Wuhan, China, early and open data sharing â€" which are vital for its control â€" depend on the trust that the data will not be used without proper attribution to those who generated it.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Subject terms</div> <div class="kwd" title="kwd">
SARS virus</div> <div class="kwd" title="kwd">
Metagenomics</div> <div class="kwd" title="kwd">
Viral epidemiology</div> <div class="kwd" title="kwd">
Viral transmission</div> <div class="kwd" title="kwd">
Epidemiology</div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="Â© The Author(s), under exclusive licence to Springer Nature Limited 2020" /> </div> </div> </div> <div class="body" title="body"> <p id="Par2">As 2019 drew to a close, news emerged of an outbreak of pneumonia of unknown aetiology, with cases clustered around the Huanan Seafood Wholesale Market in Wuhan, China. The cases were detected from 8 December and the cluster was initially reported on 31 December, when the WHOâ€™s Chinese office was informed. The market was closed on 1 January 2020 and the Chinese authorities reported having isolated a new type of coronavirus on 7 January. All suspected cases identified through active case finding and retrospective review were tested. Close to 300 cases were <a href="https://www.scmp.com/news/china/society/article/3046908/new-china-virus-likely-human-transmission-stage-infections">confirmed</a>, at the time this Editorial went to press, to have been infected by this new virus in Wuhan and four have died. Although initial reports were of very limited human to human spread, the recent <a href="https://www.scmp.com/news/china/society/article/3046908/new-china-virus-likely-human-transmission-stage-infections">increase in cases</a>, including 15 health care workers, reports of super-spreaders and the spread of the virus to other Chinese cities and various Asian countries have evidenced that there is indeed human-to-human transmission. Health authorities in the region are on alert given that millions of people are expected to travel in the region during the upcoming Lunar New Year holiday.</p> <p id="Par3">Wuhan, a strong virology hub in China, was well-equipped to detect and deal with the outbreak. However, it has put Chinaâ€™s epidemic preparedness to the test in a part of the world that still actively remembers the severe acute respiratory syndrome (SARS) coronavirus outbreak in 2003. Then, the <a href="https://www.who.int/ith/diseases/sars/en/">virus spread </a> from China to 25 other countries, infecting more than 8,000 people and killing almost 800 before it was contained. In the present case, the speed with which the Chinese authorities reported the outbreak to the international community was commendable and suggests that lessons have been learned from past outbreaks. Within ten days of having communicated the initial cluster of cases, the first genome sequence of the so-called 2019-nCoV was reported in the open access repository <a href="http://virological.org/t/initial-genome-release-of-novel-coronoavirus/319">Virological.org</a> on behalf of a consortium led by Professor Yong-Zhen Zhang of Fudan University, Shanghai, with clinicians and scientists from the Shanghai Public Health Clinical Center and School of Public Health, the Central Hospital of Wuhan, Huazhong University of Science and Technology, the Wuhan Center for Disease Control and Prevention, the National Institute for Communicable Disease Control and Prevention, the Chinese Center for Disease Control and the University of Sydney, Australia. Five additional virus sequences related to the outbreak were deposited two days later in <a href="https://www.gisaid.org/">GISAID</a>, and the Wuhan Municipal Health Commission is posting <a href="http://wjw.wuhan.gov.cn/front/web/list2nd/no/710">regular updates</a> with epidemiological information. The sequence information has identified 2019-nCoV as a betacoronavirus closely related to SARS and other coronaviruses that originate from bats, although its source remains unclear as various animals are sold at the market where it emerged.</p> <p id="Par4">Such a quick response is possible when surveillance works and there is on-site capacity to isolate and identify new viruses. An essential element of this response is prompt, transparent and thorough outbreak reporting and public sharing of the generated data. The sequence information has enabled the design of primers to develop <a href="https://www.who.int/docs/default-source/coronaviruse/wuhan-virus-assay-v1991527e5122341d99287a1b17c111902.pdf?sfvrsn=d381fc88_2">surveillance tests</a> that have enabled subsequent diagnoses and some coronavirus laboratories are already trying to rescue the virus, which will allow the development of antibody-based tests and, possibly, experimentation in animal models if they are susceptible. Although of vital importance and something that has been called for on several occasions, rapid outbreak reporting and open access posting of outbreak data before publication has not always been forthcoming and is thus something to support and celebrate. But it requires trust that the community will not use the data without proper attribution. Indeed, the consortium that published the Wuhan coronavirus sequences encourages the sharing, use and analysis of the data, and only asks that researchers communicate with them if they wish to publish results that use these data in a journal. The trust evidenced by this open sharing of information should not be broken, as the response to future outbreaks could be at stake.</p> <p id="Par5">Another characteristic of this response is that the community has made use of social media for communication and discussion in real time, enabling any interested member of the public to follow data analysis and commentary from experts in outbreak control, genomic epidemiology, virus evolution and coronavirus pathogenesis. The conversations on Twitter initially strongly called for the release of viral sequences to the international community, rapidly spread the word when they are made available and subsequently flagged instances where researchers considered the data generators had not been given proper attribution by the people analysing it. This self-regulation is important in maintaining the necessary trust that will allow the local communities, wherever outbreaks take place, to be open with the rest of the world. Beyond social media, we should think of ways of ensuring openness and proper attribution, lack of which is not only unethical, but can be considered research misconduct. In studies reporting on data derived from outbreaks and/or epidemics, journals could, for example, require that data be available before peer review instead of upon publication (something we already do) and specifically ask the referees whether the data is being properly credited.</p> <p id="Par6">As highlighted by the Executive Director of the WHO Health Emergencies Programme, we are entering a new phase of high-impact epidemics where <a href="https://www.bbc.co.uk/news/health-48547983">constant outbreaks</a> are the new normal, and one should not expect their frequency to decrease. This increase â€" which is linked to rapid urbanization, population growth, global travel, deforestation, climate change and political instability â€" emphasizes the need to improve outbreak responses, and open sharing of information as early as possible is a vital part of this effort. Once the outbreak is contained, the sharing of information should extend to the approaches used and know-how gained in the response to empower other local communities, often in areas where public health and research infrastructure is lacking, to better prepare for future outbreaks.</p> <p id="Par7">The epidemiological characteristics of the pathogen will, of course, influence how quickly an outbreak can be contained. However, a response based on equipping local communities with surveillance and control capacity at the source, rapid and open communication of epidemiological and genetic information, and international community support, will increase our chances of controlling outbreaks earlier and thus potentially save lives.</p> </div>  </body></html>