<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/04/10/2020.04.06.028902.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.04.06.028902v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Optimization of SARS-CoV-2 detection by RT-QPCR without RNA extraction"/>
  <meta name="DC.Identifier" content="10.1101/2020.04.06.028902"/>
  <meta name="DC.Date" content="2020-04-10"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Rapid and reliable screening of SARS-CoV-2 is fundamental to assess viral spread and limit the pandemic we are facing. In this study we evaluated the reliability and the efficiency of a direct RT-QPCR method (without RNA extraction) using SeeGene Allplex™ 2019-nCoV RT-QPCR and the influence of swab storage media composition on further viral detection. We show that SeeGene’s assay provides similar efficiency as the RealStar® SARS-CoV-2 RT-PCR kit (Altona Diagnostics), and that RNA extraction is not necessary nor advantageous if samples are stored in UTM or molecular water but is recommended if samples are stored in saline solution and in Hanks medium. ### Competing Interest Statement"/>
  <meta name="DC.Contributor" content="Natacha Merindol"/>
  <meta name="DC.Contributor" content="Geneviève Pépin"/>
  <meta name="DC.Contributor" content="Caroline Marchand"/>
  <meta name="DC.Contributor" content="Marylène Rheault"/>
  <meta name="DC.Contributor" content="Christine Peterson"/>
  <meta name="DC.Contributor" content="André Poirier"/>
  <meta name="DC.Contributor" content="Hugo Germain"/>
  <meta name="DC.Contributor" content="Alexis Danylo"/>
  <meta name="article:published_time" content="2020-04-10"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Optimization of SARS-CoV-2 detection by RT-QPCR without RNA extraction"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;Rapid and reliable screening of SARS-CoV-2 is fundamental to assess viral spread and limit the pandemic we are facing. In this study we evaluated the reliability and the efficiency of a direct RT-QPCR method (without RNA extraction) using SeeGene Allplex&lt;;sup&gt;;TM&lt;;/sup&gt;; 2019-nCoV RT-QPCR and the influence of swab storage media composition on further viral detection. We show that SeeGene9s assay provides similar efficiency as the RealStar® SARS-CoV-2 RT-PCR kit (Altona Diagnostics), and that RNA extraction is not necessary nor advantageous if samples are stored in UTM or molecular water but is recommended if samples are stored in saline solution and in Hanks medium.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.04.06.028902v1"/>
  <meta name="citation_id" content="2020.04.06.028902v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.04.06.028902v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.06.028902v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.06.028902v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.06.028902.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.04.06.028902"/>
  <meta name="citation_num_pages" content="17"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.04.06.028902"/>
  <meta name="citation_author" content="Natacha Merindol"/>
  <meta name="citation_author_institution" content="Département de chimie, biochimie et physique, Université du Québec à Trois-Rivières"/>
  <meta name="citation_author_institution" content="Centre Intégré Universitaire de santé et services sociaux de la Mauricie et Centre du Québec"/>
  <meta name="citation_author" content="Geneviève Pépin"/>
  <meta name="citation_author_institution" content="Département de biologie médicale, Université du Québec à Trois-Rivières"/>
  <meta name="citation_author" content="Caroline Marchand"/>
  <meta name="citation_author_institution" content="Centre Intégré Universitaire de santé et services sociaux de la Mauricie et Centre du Québec"/>
  <meta name="citation_author" content="Marylène Rheault"/>
  <meta name="citation_author_institution" content="Centre Intégré Universitaire de santé et services sociaux de la Mauricie et Centre du Québec"/>
  <meta name="citation_author" content="Christine Peterson"/>
  <meta name="citation_author_institution" content="Centre Intégré Universitaire de santé et services sociaux de la Mauricie et Centre du Québec"/>
  <meta name="citation_author" content="André Poirier"/>
  <meta name="citation_author_institution" content="Centre Intégré Universitaire de santé et services sociaux de la Mauricie et Centre du Québec"/>
  <meta name="citation_author_institution" content="Département de microbiologie infectiologie et immunologie, faculté de médecine, Université de Montréal"/>
  <meta name="citation_author" content="Hugo Germain"/>
  <meta name="citation_author_institution" content="Département de chimie, biochimie et physique, Université du Québec à Trois-Rivières"/>
  <meta name="citation_author_institution" content="Centre Intégré Universitaire de santé et services sociaux de la Mauricie et Centre du Québec"/>
  <meta name="citation_author" content="Alexis Danylo"/>
  <meta name="citation_author_institution" content="Centre Intégré Universitaire de santé et services sociaux de la Mauricie et Centre du Québec"/>
  <meta name="citation_author_institution" content="Département de microbiologie infectiologie et immunologie, faculté de médecine, Université de Montréal"/>
  <meta name="citation_author_email" content="Alexis_Danylo@ssss.gouv.qc.ca"/>
  <meta name="citation_reference" content="Coronaviridae Study Group of the International Committee on Taxonomy of V. The species Severe acute respiratory syndrome-related coronavirus: classifying 2019-nCoV and naming it SARS-CoV-2. Nat Microbiol. 2020;5(4):536–44."/>
  <meta name="citation_reference" content="Corman VM, Landt O, Kaiser M, Molenkamp R, Meijer A, Chu DKW, et al. Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR. Euro Surveill. 2020;25(3)."/>
  <meta name="citation_reference" content="Jin YH, Cai L, Cheng ZS, Cheng H, Deng T, Fan YP, et al. A rapid advice guideline for the diagnosis and treatment of 2019 novel coronavirus (2019-nCoV) infected pneumonia (standard version). Mil Med Res. 2020;7(1):4."/>
  <meta name="citation_reference" content="Chan JF, Yip CC, To KK, Tang TH, Wong SC, Leung KH, et al. Improved molecular diagnosis of COVID-19 by the novel, highly sensitive and specific COVID-19-RdRp/Hel real-time reverse transcription-polymerase chain reaction assay validated in vitro and with clinical specimens. J Clin Microbiol. 2020."/>
  <meta name="citation_reference" content="Tang YW, Schmitz JE, Persing DH, Stratton CW. The Laboratory Diagnosis of COVID-19 Infection: Current Issues and Challenges. J Clin Microbiol. 2020."/>
  <meta name="citation_reference" content="Lippi G, Simundic AM, Plebani M. Potential preanalytical and analytical vulnerabilities in the laboratory diagnosis of coronavirus disease 2019 (COVID-19). Clin Chem Lab Med. 2020."/>
  <meta name="twitter:title" content="Optimization of SARS-CoV-2 detection by RT-QPCR without RNA extraction"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Rapid and reliable screening of SARS-CoV-2 is fundamental to assess viral spread and limit the pandemic we are facing. In this study we evaluated the reliability and the efficiency of a direct RT-QPCR method (without RNA extraction) using SeeGene Allplex™ 2019-nCoV RT-QPCR and the influence of swab storage media composition on further viral detection. We show that SeeGene’s assay provides similar efficiency as the RealStar® SARS-CoV-2 RT-PCR kit (Altona Diagnostics), and that RNA extraction is not necessary nor advantageous if samples are stored in UTM or molecular water but is recommended if samples are stored in saline solution and in Hanks medium. ### Competing Interest Statement"/>
  <meta name="og-title" property="og:title" content="Optimization of SARS-CoV-2 detection by RT-QPCR without RNA extraction"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.04.06.028902v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Rapid and reliable screening of SARS-CoV-2 is fundamental to assess viral spread and limit the pandemic we are facing. In this study we evaluated the reliability and the efficiency of a direct RT-QPCR method (without RNA extraction) using SeeGene Allplex™ 2019-nCoV RT-QPCR and the influence of swab storage media composition on further viral detection. We show that SeeGene’s assay provides similar efficiency as the RealStar® SARS-CoV-2 RT-PCR kit (Altona Diagnostics), and that RNA extraction is not necessary nor advantageous if samples are stored in UTM or molecular water but is recommended if samples are stored in saline solution and in Hanks medium. ### Competing Interest Statement"/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-04-10"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Optimization of SARS-CoV-2 detection by RT-QPCR without RNA extraction | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1238533 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-4">Rapid and reliable screening of SARS-CoV-2 is fundamental to assess viral spread and limit the pandemic we are facing. In this study we evaluated the reliability and the efficiency of a direct RT-QPCR method (without RNA extraction) using SeeGene Allplex™ 2019-nCoV RT-QPCR and the influence of swab storage media composition on further viral detection.</p>
    <p id="p-5">We show that SeeGene’s assay provides similar efficiency as the RealStar
     <sup>®</sup> SARS-CoV-2 RT-PCR kit (Altona Diagnostics), and that RNA extraction is not necessary nor advantageous if samples are stored in UTM or molecular water but is recommended if samples are stored in saline solution and in Hanks medium.
    </p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Introduction</h2>
    <p id="p-7">The SARS-CoV-2 pandemic that the world is fighting since January 2020 has already caused devastating mortality (
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>). As we are facing our biggest public health threat in the modern history, there is a huge number of samples testing to be performed. Most of the current methods described so far are based on viral genes detection by RT-QPCR from patients fluids, including from the respiratory tract (
     <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>–
     <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>).
    </p>
    <p id="p-8">To slow the spread of the pandemic, screening is the key (
     <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>). This allows to isolate positive patients and stop the transmission. Many medical laboratories now have automated protocol to test the samples. These protocols are based on a Real-time PCR amplification on extracted RNA from fluids or swabs stored in media. However, in many laboratories worldwide, fear of shortage in the chain of supplies is growing. Shortage of RNA extraction kit, PCR detection kits and even swabs storage media will likely arise in many testing laboratories.
    </p>
    <p id="p-9">This outstanding situation called for alternatives protocols to ensure the continuity of rapid testing in medical laboratories. In addition, the development of new methods could also be used in case of an increase in demand for testing. In this paper, we have compared the concordance rate of positive viral genes detection of extracted RNA from the Abott M2000sp automated plate preparation system using Altona RealStar
     <sup>®</sup> SARS-CoV-2 RT-PCR Kit RUO with the SeeGene Allplex™ 2019-nCoV RT-QPCR Assay running on CFX96 BioRad. In addition, we demonstrate that direct testing of viral genes by skipping the RNA extraction step on swabs storage media could be achieved when the samples originate from standard UTM media and from molecular water.
    </p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Methods</h2>
    <p id="p-10">Specimens were collected with nasopharyngeal and oropharyngeal swabs, and were stored in different mediums. Mediums used were UTM
     <sup>®</sup> (Remel RE12569), homemade Hanks medium, Saline and molecular grade H2O.
    </p>
    <p id="p-11">Oro-nasopharyngeal swabs were inactivated at 95°C for 5 min and stored at 4°C or −80°C.RNA was extracted from 700 μl of 2 ml of patient’s swab medium using Abbott 
     <em>m</em>Sample Preparation Systems DNA kit on m2000sp instrument; the eluate was 90 μl. RT-QPCR plates were automated: prepared on the Abbott m2000sp and routinely detected on m2000rt using the Altona RealStar
     <sup>®</sup> SARS-CoV-2 RT-PCR Kit RUO according to the manufacturer’s instruction. 10μl of RNA was used in 20 μl.
    </p>
    <p id="p-12">SeeGene Allplex™ 2019-nCoV RT-QPCR Assay was manually performed on a CFX96 BioRad machine according to the manufacturer’s instruction. 7 μl of RNA or patient’ swab medium was used in a total of 25 μl and 45 cycles of amplification were performed unless specified otherwise. A patient was considered positive when a positive signal (gene E or S in Altona, genes E, N or RdRP in SeeGene) was detected at any Cts. A patient was considered negative if the internal control was amplified but not the viral genes (N, E, S and RdRP).</p>
   </div>
   <div class="section" id="sec-3">
    <h2 class="">Results and discussion</h2>
    <div id="sec-4" class="subsection">
     <h3>Following RNA extraction and RT-QPCR SeeGene vs Altona</h3>
     <p id="p-13">To determine the concordance between the standard Altona method and the SeeGene protocol, RNA extracted from 65 SARS-CoV-2-positive and 23 -negative samples were tested using SeeGene kit in three consecutive assays. 23 negative samples were confirmed negative, 64 out of 65 positive samples were confirmed positive (96,92 %). One sample, detected at Ct= 35.6 (gene S only) with Altona’s kit was missed using SeeGene’s kit (
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, 
      <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Tables 1</a> and 
      <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">2</a>). Ct means between the two methods cannot be directly compared as SeeGene uses less RNA, which can have important and this assay was performed following an additional cycle of freeze/thaw. Moreover, both manufacturers target different genes and likely do not use the same primers for the E gene in the PCR which can also results in Ct variation. Although both manufacturers neither use the same amount of RNA in the RT-QPCR reaction nor detect the same viral genes, all compounding factors which could lead to important differences in Ct between the two methods, we still observed a strong correlation between the Ct values obtained by both methods (r=0.9746; 
      <em>p</em>&lt;;0.0001).
     </p>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.06.028902/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Ranges of Cts detected from RNA extracted from patients Swabs. Each dot represents 1 of 65 reaction, each corresponding to a patient. The grey dot represents the only sample that was missed using SeeGene, detected at Ct= 35.6 with Altona." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383703716" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Ranges of Cts detected from RNA extracted from patients Swabs. Each dot represents 1 of 65 reaction, each corresponding to a patient. The grey dot represents the only sample that was missed using SeeGene, detected at Ct= 35.6 with Altona.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.06.028902/F1.medium.gif" width="273" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.06.028902/F1.medium.gif" width="273" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.06.028902/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.06.028902/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 1.</span>
       <p id="p-14" class="first-child">Ranges of Cts detected from RNA extracted from patients Swabs. Each dot represents 1 of 65 reaction, each corresponding to a patient. The grey dot represents the only sample that was missed using SeeGene, detected at Ct= 35.6 with Altona.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1242334/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1242334/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1242334" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">TABLE 1-</span>
       <p id="p-15" class="first-child">Number of positive samples detected using different methods</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="T2" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1242330/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1242330/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1242330" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">TABLE 2-</span>
       <p id="p-16" class="first-child">Mean Ct values (targets with best concordance)</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-5" class="subsection">
     <h3>RT-QPCR directly from swabs: without RNA extraction</h3>
     <p id="p-17">The swab storage media for respiratory virus detection can have important impact on the RT-PCR efficiency, when this media is used directly for PCR reaction, without RNA extraction. To assess the feasibility of direct RT-QPCR using different media we performed the following experiments.</p>
     <div id="sec-6" class="subsection">
      <h4>Samples stored in UTM</h4>
      <p id="p-18">17 samples stored in UTM media, previously assessed as positive by the Altona method with RNA extraction, were used for direct RT-QPCR in two independent experiments. 100% were detected using SeeGene (gene N) and Ct means were equivalent whether or not RNA was extracted before the RT-QPCR (
       <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>; 
       <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> and 
       <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">2</a>). This experiment was also repeated in 12.5 μl of final volume and similar results were obtained.
      </p>
      <div id="F2" class="fig pos-float type-figure odd">
       <div class="highwire-figure">
        <div class="fig-inline-img-wrapper">
         <div class="fig-inline-img">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.06.028902/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Detection of SARS-CoV-2 by SeeGene RT-QPCR from swabs stored in UTM medium, in the absence of RNA extraction. Each dot represents one reaction corresponding to one patient. Results are plotted as bar graphs with standard deviation. [[#x2018]]n[[#x2019]] stands for the number of positive RT-QPCR detection." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383703716" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Detection of SARS-CoV-2 by SeeGene RT-QPCR from swabs stored in UTM medium, in the absence of RNA extraction. Each dot represents one reaction corresponding to one patient. Results are plotted as bar graphs with standard deviation. [[#x2018]]n[[#x2019]] stands for the number of positive RT-QPCR detection.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
           <span class="hw-responsive-img">
            <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.06.028902/F2.medium.gif" width="440" height="437"/>
            <noscript>
             <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.06.028902/F2.medium.gif" width="440" height="437"/>
            </noscript>
           </span>
          </a>
         </div>
        </div>
        <ul class="highwire-figure-links inline">
         <li class="download-fig first">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.06.028902/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
         </li>
         <li class="new-tab last">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.06.028902/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
         </li>
        </ul>
       </div>
       <div class="fig-caption">
        <span class="fig-label">Figure 2.</span>
        <p id="p-19" class="first-child">Detection of SARS-CoV-2 by SeeGene RT-QPCR from swabs stored in UTM medium, in the absence of RNA extraction. Each dot represents one reaction corresponding to one patient. Results are plotted as bar graphs with standard deviation. ‘n’ stands for the number of positive RT-QPCR detection.</p>
        <div class="sb-div caption-clear"/>
       </div>
      </div>
     </div>
     <div id="sec-7" class="subsection">
      <h4>Samples stored in Hanks</h4>
      <p id="p-20">Due to a shortage of UTM media, swabs were stored in Hanks medium and analysed for direct RT-QPCR. 94 samples previously assessed as positive by the Altona method (with RNA extraction) stored in Hanks medium were used for direct RT-QPCR in two independent experiments. 21 (2,3%) reactions did not show amplification neither of viral genes nor of the internal control suggesting RT-QPCR inhibition. 75,53% (n=71) were positive following analysis (
       <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Tables 1</a> and 
       <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">2</a>, 
       <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). The addition of 5 cycles (from 45 cycles to 50 cycles) for 64 of the samples helped with the detection of 5 samples that showed delayed Cts.
      </p>
      <div id="F3" class="fig pos-float type-figure odd">
       <div class="highwire-figure">
        <div class="fig-inline-img-wrapper">
         <div class="fig-inline-img">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.06.028902/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Detection of SARS-CoV-2 from swabs stored in Hanks medium. Results are plotted as bar graphs with standard deviation. Ct values from amplification of extracted RNA from swabs stored in Hanks medium using Altona[[#x2019]]s kit (Altona, S gene) and Seegen[[#x2019]]s kit (RNA_N; N gene, following an additional freeze/thaw cycle), or directly from Swabs in Hanks medium (No_Extr_N; N gene) using SeeGene[[#x2019]]s kit." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383703716" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Detection of SARS-CoV-2 from swabs stored in Hanks medium. Results are plotted as bar graphs with standard deviation. Ct values from amplification of extracted RNA from swabs stored in Hanks medium using Altona[[#x2019]]s kit (Altona, S gene) and Seegen[[#x2019]]s kit (RNA_N; N gene, following an additional freeze/thaw cycle), or directly from Swabs in Hanks medium (No_Extr_N; N gene) using SeeGene[[#x2019]]s kit.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
           <span class="hw-responsive-img">
            <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.06.028902/F3.medium.gif" width="313" height="440"/>
            <noscript>
             <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.06.028902/F3.medium.gif" width="313" height="440"/>
            </noscript>
           </span>
          </a>
         </div>
        </div>
        <ul class="highwire-figure-links inline">
         <li class="download-fig first">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.06.028902/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
         </li>
         <li class="new-tab last">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.06.028902/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
         </li>
        </ul>
       </div>
       <div class="fig-caption">
        <span class="fig-label">Figure 3.</span>
        <p id="p-21" class="first-child">Detection of SARS-CoV-2 from swabs stored in Hanks medium. Results are plotted as bar graphs with standard deviation. Ct values from amplification of extracted RNA from swabs stored in Hanks medium using Altona’s kit (Altona, S gene) and Seegen’s kit (RNA_N; N gene, following an additional freeze/thaw cycle), or directly from Swabs in Hanks medium (No_Extr_N; N gene) using SeeGene’s kit.</p>
        <div class="sb-div caption-clear"/>
       </div>
      </div>
      <p id="p-22">Ct values cannot be directly compared to Altona’s, as less volume of sample was added in the mastermix, the assays were performed following an additional cycle of freeze/thaw, and the target genes and primers used by both manufacturers are not the same.</p>
     </div>
     <div id="sec-8" class="subsection">
      <h4>Samples stored in water</h4>
      <p id="p-23">To determine if molecular grade water could be used as swabs storage media without impacting direct RT-QPCR, three positive swabs were stored in water at 4°C. Eight serial dilutions (1:10) were performed on two of these samples. From the first patient, the undiluted sample and 4 of 7 dilutions showed amplification of SARS-CoV-2 N gene and S gene using RNA with both Altona and SeeGene, respectively. In this same sample comparable amounts of viral N gene was detected without RNA extraction using SeeGene (
       <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Cts can be compared between the two methods in this case as there was no additional Freeze/thaw cycle for this experiment. This experiment was confirmed using 12 μl as sample volume in the mastermix. An addition of 5 other patients were efficiently amplified (
       <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>).
      </p>
      <div id="F4" class="fig pos-float type-figure odd">
       <div class="highwire-figure">
        <div class="fig-inline-img-wrapper">
         <div class="fig-inline-img">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.06.028902/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Detection of SARS-CoV-2 by SeeGene RT-QPCR from swabs stored in molecular water. Results are plotted as bar graphs with standard deviation. Serial dilutions were performed in water from a swab stored in water. Ct values from amplification of extracted RNA from swabs stored in water medium using Altona[[#x2019]]s kit (Altona_RNA, S and E genes) and Seegen[[#x2019]]s kit (SeeGene_RNA; N and E genes), or directly from Swabs in water medium (H20; N and E gene) using SeeGene[[#x2019]]s kit. [[#x2018]]na[[#x2019]] means [[#x2018]]not available[[#x2019]]." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383703716" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Detection of SARS-CoV-2 by SeeGene RT-QPCR from swabs stored in molecular water. Results are plotted as bar graphs with standard deviation. Serial dilutions were performed in water from a swab stored in water. Ct values from amplification of extracted RNA from swabs stored in water medium using Altona[[#x2019]]s kit (Altona_RNA, S and E genes) and Seegen[[#x2019]]s kit (SeeGene_RNA; N and E genes), or directly from Swabs in water medium (H20; N and E gene) using SeeGene[[#x2019]]s kit. [[#x2018]]na[[#x2019]] means [[#x2018]]not available[[#x2019]].&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
           <span class="hw-responsive-img">
            <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.06.028902/F4.medium.gif" width="375" height="440"/>
            <noscript>
             <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.06.028902/F4.medium.gif" width="375" height="440"/>
            </noscript>
           </span>
          </a>
         </div>
        </div>
        <ul class="highwire-figure-links inline">
         <li class="download-fig first">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.06.028902/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
         </li>
         <li class="new-tab last">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.06.028902/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
         </li>
        </ul>
       </div>
       <div class="fig-caption">
        <span class="fig-label">Figure 4.</span>
        <p id="p-24" class="first-child">Detection of SARS-CoV-2 by SeeGene RT-QPCR from swabs stored in molecular water. Results are plotted as bar graphs with standard deviation. Serial dilutions were performed in water from a swab stored in water. Ct values from amplification of extracted RNA from swabs stored in water medium using Altona’s kit (Altona_RNA, S and E genes) and Seegen’s kit (SeeGene_RNA; N and E genes), or directly from Swabs in water medium (H20; N and E gene) using SeeGene’s kit. ‘na’ means ‘not available’.</p>
        <div class="sb-div caption-clear"/>
       </div>
      </div>
      <div id="F5" class="fig pos-float type-figure odd">
       <div class="highwire-figure">
        <div class="fig-inline-img-wrapper">
         <div class="fig-inline-img">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.06.028902/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Detection of SARS-CoV-2 by SeeGene RT-QPCR from RNA from swabs diluted in saline water. Results are plotted as bar graphs with standard deviation. One hundredth dilution of 5 samples with high Ct value were performed in molecular water and in saline water. Ct values obtained from RNA eluates diluted in water medium (H2O) or in saline water (Saline) using Seegen[[#x2019]]s kit (SeeGene_RNA; RDRP, N and E genes)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383703716" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Detection of SARS-CoV-2 by SeeGene RT-QPCR from RNA from swabs diluted in saline water. Results are plotted as bar graphs with standard deviation. One hundredth dilution of 5 samples with high Ct value were performed in molecular water and in saline water. Ct values obtained from RNA eluates diluted in water medium (H2O) or in saline water (Saline) using Seegen[[#x2019]]s kit (SeeGene_RNA; RDRP, N and E genes).&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
           <span class="hw-responsive-img">
            <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.06.028902/F5.medium.gif" width="440" height="436"/>
            <noscript>
             <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.06.028902/F5.medium.gif" width="440" height="436"/>
            </noscript>
           </span>
          </a>
         </div>
        </div>
        <ul class="highwire-figure-links inline">
         <li class="download-fig first">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.06.028902/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a>
         </li>
         <li class="new-tab last">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.06.028902/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
         </li>
        </ul>
       </div>
       <div class="fig-caption">
        <span class="fig-label">Figure 5.</span>
        <p id="p-25" class="first-child">Detection of SARS-CoV-2 by SeeGene RT-QPCR from RNA from swabs diluted in saline water. Results are plotted as bar graphs with standard deviation. One hundredth dilution of 5 samples with high Ct value were performed in molecular water and in saline water. Ct values obtained from RNA eluates diluted in water medium (H2O) or in saline water (Saline) using Seegen’s kit (SeeGene_RNA; RDRP, N and E genes).</p>
        <div class="sb-div caption-clear"/>
       </div>
      </div>
      <div id="F6" class="fig pos-float type-figure odd">
       <div class="highwire-figure">
        <div class="fig-inline-img-wrapper">
         <div class="fig-inline-img">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.06.028902/F6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Stability of swabs (not extracted) stored in H2O at 4[[#xB0]]C. Serial dilutions were performed in molecular grade water from a swab stored in RNAse inhibitor containing water. Ct values from direct amplification from swab and dilutions was measured using Seegen[[#x2019]]s kit (SeeGene_RNA; RdRP, N and E genes) following storage for 0, 1, 3 and 5 days." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1383703716" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Stability of swabs (not extracted) stored in H2O at 4[[#xB0]]C. Serial dilutions were performed in molecular grade water from a swab stored in RNAse inhibitor containing water. Ct values from direct amplification from swab and dilutions was measured using Seegen[[#x2019]]s kit (SeeGene_RNA; RdRP, N and E genes) following storage for 0, 1, 3 and 5 days.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
           <span class="hw-responsive-img">
            <img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.06.028902/F6.medium.gif" width="440" height="117"/>
            <noscript>
             <img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.06.028902/F6.medium.gif" width="440" height="117"/>
            </noscript>
           </span>
          </a>
         </div>
        </div>
        <ul class="highwire-figure-links inline">
         <li class="download-fig first">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.06.028902/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a>
         </li>
         <li class="new-tab last">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/10/2020.04.06.028902/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
         </li>
        </ul>
       </div>
       <div class="fig-caption">
        <span class="fig-label">Figure 6.</span>
        <p id="p-26" class="first-child">Stability of swabs (not extracted) stored in H2O at 4°C. Serial dilutions were performed in molecular grade water from a swab stored in RNAse inhibitor containing water. Ct values from direct amplification from swab and dilutions was measured using Seegen’s kit (SeeGene_RNA; RdRP, N and E genes) following storage for 0, 1, 3 and 5 days.</p>
        <div class="sb-div caption-clear"/>
       </div>
      </div>
      <p id="p-27">In the second sample, SARS-CoV-2 could be detected only from the RNA extracted in the undiluted specimen using Altona’s kit at Ct=35,6. The experiment was repeated using SeeGene and SARS-CoV-2 N gene was detected from H2O (stored at 4°C for 24 hours) at Ct=37.1 but not from extracted RNA (
       <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a> and 
       <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">2</a>).
      </p>
     </div>
     <div id="sec-9" class="subsection">
      <h4>Samples stored in saline solution</h4>
      <p id="p-28">To determine if saline solution could be used as swabs storage media without impacting direct RT-PCR, five RNA eluate from samples with high Cts (between 12,61 and 14,22) following Altona’s amplification were diluted in water (1:100) and in saline solution (1:100). Dilutions were directly used for RT-QPCR. As previously observed, amplification could be reliably detected in samples stored in water. However, when samples were stored in saline water, a loss of 10 Cts for the N gene was measured following SeeGene RT-QPCR and the E gene could not be amplified despite 50 cycles.</p>
     </div>
     <div id="sec-10" class="subsection">
      <h4>Stability of long-term storage of swabs in water at 4°C</h4>
      <p id="p-29">Centralization and increased number of screenings may lead to swabs storage at 4°C for several days. To assess viral RNA stability at 4°C, we quantified levels of N, E and RDRP genes directly from a swab following four 1:10 serial dilutions stored for 0, 24, 72 hours and for 5 days at 4°C. RNase inhibitor was added to the patient swab and serial dilutions in molecular grade water were performed (hence the RNAse inhibitor was also diluted). Viral RNA was detected at 0, 24 and 72 hours at all dilutions but 1:10 000. Ct values were highly constant at all timepoints for the three viral genes for undiluted, 1:10 and 1:100 diluted swabs (E gene, data not shown). The addition of RNase inhibitor probably increased RNA stability of the undiluted swab. Nevertheless, RNA remained highly stable even when diluted hundred times. Samples diluted 1000 times were less accurately amplified: RDRP was detected up to 24h.</p>
     </div>
    </div>
   </div>
   <div class="section" id="sec-11">
    <h2 class="">Conclusion</h2>
    <p id="p-30">Our results suggest that: i) the SeeGene Allplex™ 2019-nCoV RT-QPCR Assay provides similar efficiency as the Altona RealStar
     <sup>®</sup> SARS-CoV-2 RT-PCR Kit RUO, ii) RNA extraction is not necessary if samples are stored in UTM or molecular grade water; iii) RNA extraction is recommended if samples are stored in saline water and in Hanks medium. The number of cycles can be increased to 50 to detect low-producers.
    </p>
   </div>
   <div class="section fn-group" id="fn-group-1">
    <h2>Footnotes</h2>
    <ul>
     <li class="fn-other" id="fn-2">
      <p id="p-2">NM: 
       <span class="em-link">
        <span class="em-addr">Natacha.Merindol{at}uqtr.ca</span>
       </span>, HG: 
       <span class="em-link">
        <span class="em-addr">Hugo.Germain{at}uqtr.ca</span>
       </span>, GP: 
       <span class="em-link">
        <span class="em-addr">Genevieve.Pepin3{at}uqtr.ca</span>
       </span>, CM: 
       <span class="em-link">
        <span class="em-addr">caroline_marchand_labo{at}ssss.gouv.qc.ca</span>
       </span>, MR: 
       <span class="em-link">
        <span class="em-addr">marylene_rheault{at}ssss.gouv.qc.ca</span>
       </span>, CP: 
       <span class="em-link">
        <span class="em-addr">christine_peterson{at}ssss.gouv.qc.ca</span>
       </span>, AP: 
       <span class="em-link">
        <span class="em-addr">andre_poirier_chrtr{at}ssss.gouv.qc.ca</span>
       </span>
      </p>
     </li>
    </ul>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.06.028902v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">Coronaviridae Study Group of the International Committee on Taxonomy of V</span>. 
         <span class="cit-article-title">The species Severe acute respiratory syndrome-related coronavirus: classifying 2019-nCoV and naming it SARS-CoV-2</span>. 
         <abbr class="cit-jnl-abbrev">Nat Microbiol</abbr>. 
         <span class="cit-pub-date">2020</span>;
         <span class="cit-vol">5</span>(
         <span class="cit-issue">4</span>):
         <span class="cit-fpage">536</span>–
         <span class="cit-lpage">44</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BMicrobiol%26rft.volume%253D5%26rft.spage%253D536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.06.028902v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Corman</span>
          <span class="cit-name-given-names">VM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Landt</span>
          <span class="cit-name-given-names">O</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kaiser</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Molenkamp</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Meijer</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chu</span>
          <span class="cit-name-given-names">DKW</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR</span>. 
         <abbr class="cit-jnl-abbrev">Euro Surveill</abbr>. 
         <span class="cit-pub-date">2020</span>;
         <span class="cit-vol">25</span>(
         <span class="cit-issue">3</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.06.028902v1.3" data-doi="10.1186/s40779-020-0233-6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jin</span>
          <span class="cit-name-given-names">YH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cai</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cheng</span>
          <span class="cit-name-given-names">ZS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cheng</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Deng</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fan</span>
          <span class="cit-name-given-names">YP</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A rapid advice guideline for the diagnosis and treatment of 2019 novel coronavirus (2019-nCoV) infected pneumonia (standard version)</span>. 
         <abbr class="cit-jnl-abbrev">Mil Med Res</abbr>. 
         <span class="cit-pub-date">2020</span>;
         <span class="cit-vol">7</span>(
         <span class="cit-issue">1</span>):
         <span class="cit-fpage">4</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMil%2BMed%2BRes%26rft.volume%253D7%26rft.spage%253D4%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs40779-020-0233-6%26rft_id%253Dinfo%253Apmid%252F32029004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/s40779-020-0233-6&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=32029004&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F10%2F2020.04.06.028902.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.06.028902v1.4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chan</span>
          <span class="cit-name-given-names">JF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yip</span>
          <span class="cit-name-given-names">CC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">To</span>
          <span class="cit-name-given-names">KK</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tang</span>
          <span class="cit-name-given-names">TH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wong</span>
          <span class="cit-name-given-names">SC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Leung</span>
          <span class="cit-name-given-names">KH</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Improved molecular diagnosis of COVID-19 by the novel, highly sensitive and specific COVID-19-RdRp/Hel real-time reverse transcription-polymerase chain reaction assay validated in vitro and with clinical specimens</span>. 
         <abbr class="cit-jnl-abbrev">J Clin Microbiol</abbr>. 
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.06.028902v1.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Tang</span>
          <span class="cit-name-given-names">YW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schmitz</span>
          <span class="cit-name-given-names">JE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Persing</span>
          <span class="cit-name-given-names">DH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Stratton</span>
          <span class="cit-name-given-names">CW.</span>
         </span>
         <span class="cit-article-title">The Laboratory Diagnosis of COVID-19 Infection: Current Issues and Challenges</span>. 
         <abbr class="cit-jnl-abbrev">J Clin Microbiol</abbr>. 
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.06.028902v1.6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lippi</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Simundic</span>
          <span class="cit-name-given-names">AM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Plebani</span>
          <span class="cit-name-given-names">M.</span>
         </span>
         <span class="cit-article-title">Potential preanalytical and analytical vulnerabilities in the laboratory diagnosis of coronavirus disease 2019 (COVID-19)</span>. 
         <abbr class="cit-jnl-abbrev">Clin Chem Lab Med</abbr>. 
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
