<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.13.990267/embed/graphic-6.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/03/15/2020.03.13.990267.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.13.990267v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Differential Antibody Recognition by Novel SARS-CoV-2 and SARS-CoV Spike Protein Receptor Binding Domains: Mechanistic Insights and Implications for the Design of Diagnostics and Therapeutics" />
<meta name="DC.Identifier" content="10.1101/2020.03.13.990267" />
<meta name="DC.Date" content="2020-03-15" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="The appearance of the novel betacoronavirus SARS-CoV-2 represents a major threat to human health, and its diffusion around the world is causing dramatic consequences. The knowledge of the 3D structures of SARS-CoV-2 proteins can facilitate the development of therapeutic and diagnostic molecules. Specifically, comparative analyses of the structures of SARS-CoV-2 proteins and homologous proteins from previously characterized viruses, such as SARS-CoV, can reveal the common and/or distinctive traits that underlie the mechanisms of recognition of cell receptors and of molecules of the immune system.

Herein, we apply our recently developed energy-based methods for the prediction of antibody-binding epitopes and protein-protein interaction regions to the Receptor Binding Domain (RBD) of the Spike proteins from SARS-CoV-2 and SARS-CoV. Our analysis focusses only on the study of the structure of RBDs in isolation, without making use of any previous knowledge of binding properties. Importantly, our results highlight structural and sequence differences among the regions that are predicted to be immunoreactive and bind/elicit antibodies. These results provide a rational basis to the observation that several SARS-CoV RDB-specific monoclonal antibodies fail to appreciably bind the SARS-CoV-2 counterpart. Furthermore, we correctly identify the region of SARS-CoV-2 RBD that is engaged by the cell receptor ACE2 during viral entry into host cells.

The data, sequences and structures we present here can be useful for the development of novel therapeutic and diagnostic interventions." />
<meta name="DC.Contributor" content="Ilda D’Annessa" />
<meta name="DC.Contributor" content="Filippo Marchetti" />
<meta name="DC.Contributor" content="Giorgio Colombo" />
<meta name="article:published_time" content="2020-03-15" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Differential Antibody Recognition by Novel SARS-CoV-2 and SARS-CoV Spike Protein Receptor Binding Domains: Mechanistic Insights and Implications for the Design of Diagnostics and Therapeutics" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The appearance of the novel betacoronavirus SARS-CoV-2 represents a major threat to human health, and its diffusion around the world is causing dramatic consequences. The knowledge of the 3D structures of SARS-CoV-2 proteins can facilitate the development of therapeutic and diagnostic molecules. Specifically, comparative analyses of the structures of SARS-CoV-2 proteins and homologous proteins from previously characterized viruses, such as SARS-CoV, can reveal the common and/or distinctive traits that underlie the mechanisms of recognition of cell receptors and of molecules of the immune system. Herein, we apply our recently developed energy-based methods for the prediction of antibody-binding epitopes and protein-protein interaction regions to the Receptor Binding Domain (RBD) of the Spike proteins from SARS-CoV-2 and SARS-CoV. Our analysis focusses only on the study of the structure of RBDs in isolation, without making use of any previous knowledge of binding properties. Importantly, our results highlight structural and sequence differences among the regions that are predicted to be immunoreactive and bind/elicit antibodies. These results provide a rational basis to the observation that several SARS-CoV RDB-specific monoclonal antibodies fail to appreciably bind the SARS-CoV-2 counterpart. Furthermore, we correctly identify the region of SARS-CoV-2 RBD that is engaged by the cell receptor ACE2 during viral entry into host cells. The data, sequences and structures we present here can be useful for the development of novel therapeutic and diagnostic interventions.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.13.990267v2" />
<meta name="citation_id" content="2020.03.13.990267v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.13.990267v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.13.990267v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.13.990267v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.13.990267.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.13.990267" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.13.990267" />
<meta name="citation_author" content="Ilda D’Annessa" />
<meta name="citation_author_institution" content="SCITEC-CNR" />
<meta name="citation_author" content="Filippo Marchetti" />
<meta name="citation_author_institution" content="University of Pavia, Department of Chemistry" />
<meta name="citation_author" content="Giorgio Colombo" />
<meta name="citation_author_institution" content="University of Pavia, Department of Chemistry" />
<meta name="citation_author_email" content="g.colombo@unipv.it" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1318-668X" />
<meta name="citation_reference" content="Chan, J. F.-W.; Yuan, S.; Kok, K.-H.; To, K. K.-W.; Chu, H.; Yang, J.; Xing, F.; Liu, J.; Yip, C. C.-Y.; Poon, R. W.-S.; Tsoi, H.-W.; Lo, S. K.-F.; Chan, K.-H.; Poon, V. K.-M.; Chan, W.-M.; Ip, J. D.; Cai, J.-P.; Cheng, V. C.-C.; Chen, H.; Hui, C. K.-M.; Yuen, K.-Y., A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster. The Lancet 2020, 395 (10223), 514–523." />
<meta name="citation_reference" content="Huang, C.; Wang, Y.; Li, X.; Ren, L.; Zhao, J.; Hu, Y.; Zhang, L.; Fan, G.; Xu, J.; Gu, X.; Cheng, Z.; Yu, T.; Xia, J.; Wei, Y.; Wu, W.; Xie, X.; Yin, W.; Li, H.; Liu, M.; Xiao, Y.; Gao, H.; Guo, L.; Xie, J.; Wang, G.; Jiang, R.; Gao, Z.; Jin, Q.; Wang, J.; Cao, B., Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China. The Lancet 2020, 395 (10223), 497–506." />
<meta name="citation_reference" content="Scarabelli, G.; Morra, G.; Colombo, G., Predicting interaction sited from the energetics of isolated proteins: a new approach to epitope mapping. Biophys. J. 2010, 98 (9), 1966–1975." />
<meta name="citation_reference" content="Marchetti, F.; Capelli, R.; Rizzato, F.; Laio, A.; Colombo, G., The Subtle Trade-Off between Evolutionary and Energetic Constraints in Protein-Protein Interactions. J. Phys. Chem. Lett. 2019, 10 (7), 1489–1497." />
<meta name="citation_reference" content="Soriani, M.; Petit, P.; Grifantini, R.; Petracca, R.; Gancitano, G.; Frigimelica, E.; Nardelli, F.; Garcia, C.; Spinelli, S.; Scarabelli, G.; Fiorucci, S.; Affentranger, R.; Ferrer-Navarro, M.; Zacharias, M.; Colombo, G.; Vuillard, L.; Daura, X.; Grandi, G., Exploiting antigenic diversity for vaccine design: the chlamydia ArtJ paradigm. J. Biol. Chem. 2010, 285 (39), 30126–30138." />
<meta name="citation_reference" content="Gourlay, L. J.; Peri, C.; Ferrer-Navarro, M.; Conchillo-Solé, O.; Gori, A.; Rinchai, D.; Thomas, R. J.; Champion, O. L.; Michell, S. L.; Kewcharoenwong, C.; Nithichanon, A.; Lassaux, P.; Perletti, L.; Longhi, R.; Lertmemongkolchai, G.; Titball, R. W.; Daura, X.; Colombo, G.; Bolognesi, M., Exploiting the Burkholderia pseudomallei Acute Phase Antigen BPSL2765 for Structure-Based Epitope Discovery/Design in Structural Vaccinology. Chem. Biol. 2013, 20, 1147–1156." />
<meta name="citation_reference" content="Lassaux, P.; Peri, C.; Ferrer-Navarro, M.; Gourlay, L.; Gori, A.; Conchillo-Solé, O.; Rinchai, D.; Lertmemongkolchai, G.; Longhi, R.; Daura, X.; Colombo, G.; Bolognesi, M., A structure-based strategy for epitope discovery in Burkholderia pseudomallei OppA antigen. Structure 2013, 21, 1–9." />
<meta name="citation_reference" content="Gourlay, L. J.; Lassaux, P.; Thomas, R. J.; Peri, C.; Conchillo-Sole, O.; Nithichanon, A.; Ferrer-Navarro, M.; Vila, J.; Daura, X.; Lertmemongkolchai, G.; Titball, R.; Colombo, G.; Bolognesi, M., Flagellar subunits as targets for structure-based epitope discovery approaches and melioidosis vaccine development. Febs Journal 2015, 282, 338–338." />
<meta name="citation_reference" content="Gourlay, L. J.; Thomas, R. J.; Peri, C.; Conchillo-Sole, O.; Ferrer-Navarro, M.; Nithichanon, A.; Vila, J.; Daura, X.; Lertmemongkolchai, G.; Titball, R.; Colombo, G.; Bolognesi, M., From crystal structure to in silico epitope discovery in the Burkholderia pseudomallei flagellar hook-associated protein FlgK. Febs Journal 2015, 282 (7), 1319–1333." />
<meta name="citation_reference" content="Nithichanon, A.; Rinchai, D.; Gori, A.; Lassaux, P.; Peri, C.; Conchillio-Sole, O.; Ferrer-Navarro, M.; Gourlay, L. J.; Nardini, M.; Vila, J.; Daura, X.; Colombo, G.; Bolognesi, M.; Lertmemonkolchai, G., Sequence- and Structure-Based Immunoreactive Epitope Discovery for Burkholderia pseudomallei Flagellin. Plos Neglected Tropical Diseases 2015, 9 (7)." />
<meta name="citation_reference" content="Gori, A.; Peri, C.; Quilici, G.; Nithichanon, A.; Gaudesi, D.; Longhi, R.; Gourlay, L.; Bolognesi, M.; Lertmemongkolchai, G.; Musco, G.; Colombo, G., Flexible vs Rigid Epitope Conformations for Diagnostic- and Vaccine-Oriented Applications: Novel Insights from the Burkholderia pseudomallei BPSL2765 Pa13 Epitope. Acs Infectious Diseases 2016, 2 (3), 221–230." />
<meta name="citation_reference" content="Gori, A.; Sola, L.; Gagni, P.; Bruni, G.; Liprino, M.; Peri, C.; Colombo, G.; Cretich, M.; Chiari, M., Screening Complex Biological Samples with Peptide Microarrays: The Favorable Impact of Probe Orientation via Chemoselective Immobilization Strategies on Clickable Polymeric Coatings. Bioconjugate Chemistry 2016, 27 (11), 2669–2677." />
<meta name="citation_reference" content="Sola, L.; Gagni, P.; D’Annessa, I.; Capelli, R.; Bertino, C.; Romanato, A.; Damin, F.; Bergamaschi, G.; Marchisio, E.; Cuzzocrea, A.; Bombaci, M.; Grifantini, R.; Chiari, M.; Colombo, G.; Gori, A.; Cretich, M., Enhancing Antibody Serodiagnosis Using a Controlled Peptide Coimmobilization Strategy. ACS Infectious Diseases 2018, 4 (6), 998–1006." />
<meta name="citation_reference" content="Bergamaschi, G.; Fassi, E. M. A.; Romanato, A.; D’Annessa, I.; Odinolfi, M. T.; Brambilla, D.; Damin, F.; Chiari, M.; Gori, A.; Colombo, G.; Cretich, M., Computational Analysis of Dengue Virus Envelope Protein (E) Reveals an Epitope with Flavivirus Immunodiagnostic Potential in Peptide Microarrays. International journal of molecular sciences 2019, 20 (8), 1921." />
<meta name="citation_reference" content="Wrapp, D.; Wang, N.; Corbett, K. S.; Goldsmith, J. A.; Hsieh, C.-L.; Abiona, O.; Graham, B. S.; McLellan, J. S., Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation. Science 2020, eabb2507." />
<meta name="citation_reference" content="Hwang, W. C.; Lin, Y.; Santelli, E.; Sui, J.; Jaroszewski, L.; Stec, B.; Farzan, M.; Marasco, W. A.; Liddington, R. C., Structural Basis of Neutralization by a Human Anti-severe Acute Respiratory Syndrome Spike Protein Antibody, 80R. Journal of Biological Chemistry 2006, 281 (45), 34610–34616." />
<meta name="citation_reference" content="Rockx, B.; Corti, D.; Donaldson, E.; Sheahan, T.; Stadler, K.; Lanzavecchia, A.; Baric, R., Structural Basis for Potent Cross-Neutralizing Human Monoclonal Antibody Protection Against Lethal Human and Zoonotic SARS-CoV Challenge. Journal of Virology 2008." />
<meta name="citation_reference" content="Zhu, Z.; Chakraborti, S.; He, Y.; Roberts, A.; Sheahan, T.; Xiao, X.; Hensley, L. E.; Prabakaran, P.; Rockx, B.; Sidorov, I. A.; Corti, D.; Vogel, L.; Feng, Y.; Kim, J.-O.; Wang, L.-F.; Baric, R.; Lanzavecchia, A.; Curtis, K. M.; Nabel, G. J.; Subbarao, K.; Jiang, S.; Dimitrov, D. S., Potent crossreactive neutralization of SARS coronavirus isolates by human monoclonal antibodies. Proceedings of the National Academy of Sciences 2007, 104 (29), 12123." />
<meta name="citation_reference" content="Dormitzer, P. R.; Grandi, G.; Rappuoli, R., Structural vaccinology starts to deliver. Nat Rev Microbiol 2012, 10 (12), 807–13." />
<meta name="citation_reference" content="De Gregorio, E.; Rappuoli, R., From empiricism to rational design: a personal perspective of the evolution of vaccine development. Nat. Rev. Immunol. 2014, 14 (7), 505–514." />
<meta name="citation_reference" content="Thomas, S.; Dilbarova, R.; Rappuoli, R., Future Challenges for Vaccinologists. Methods Mol Biol 2016, 1403, 41–55." />
<meta name="citation_reference" content="Peri, C.; Gagni, P.; Combi, F.; Gori, A.; Chiari, M.; Longhi, R.; Cretich, M.; Colombo, G., Rational epitope design for protein targeting. . ACS Chemical Biology 2013, 8, 397–404" />
<meta name="citation_reference" content="Cretich, M.; Gori, A.; D’Annessa, I.; Chiari, M.; Colombo, G., Peptides for Infectious Diseases: From Probe Design to Diagnostic Microarrays. Antibodies (Basel) 2019, 8 (1), 23." />
<meta name="citation_reference" content="Lan, J.; Ge, J.; Yu, J.; Shan, S.; Zhou, H.; Fan, S.; Zhang, Q.; Shi, X.; Wang, Q.; Zhang, L.; Wang, X., Crystal structure of the 2019-nCoV spike receptor-binding domain bound with the ACE2 receptor. bioRxiv 2020, 2020.02.19.956235." />
<meta name="citation_reference" content="Yan, R.; Zhang, Y.; Li, Y.; Xia, L.; Guo, Y.; Zhou, Q., Structural basis for the recognition of the SARS-CoV-2 by full-length human ACE2. Science 2020, eabb2762." />
<meta name="citation_reference" content="Li, F.; Li, W.; Farzan, M.; Harrison, S. C., Structure of SARS Coronavirus Spike ReceptorBinding Domain Complexed with Receptor. Science 2005, 309 (5742), 1864." />
<meta name="citation_reference" content="Tian, X.; Li, C.; Huang, A.; Xia, S.; Lu, S.; Shi, Z.; Lu, L.; Jiang, S.; Yang, Z.; Wu, Y.; Ying, T., Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody. Emerging Microbes &amp; Infections 2020, 9 (1), 382–385." />
<meta name="citation_reference" content="Morra, G.; Colombo, G., Relationship between energy distribution and fold stability: Insights from molecular dynamics simulations of native and mutant proteins. Proteins: Struct. Funct. and Bioinf. 2008, 72 (2), 660–672." />
<meta name="citation_reference" content="Genoni, A.; Morra, G.; Colombo, G., Identification of Domains in Protein Structures from the Analysis of Intramolecular Interactions. J. Phys. Chem. B. 2012, 116 (10), 3331–3343." />
<meta name="citation_reference" content="Morra, G.; Genoni, A.; Colombo, G., Mechanisms of Differential Allosteric Modulation in Homologous Proteins: Insights from the Analysis of Internal Dynamics and Energetics of PDZ Domains. J Chem Theory Comput. 2014, 10 (12), 5677–5689" />
<meta name="citation_reference" content="Paladino, A.; Morra, G.; Colombo, G., Structural Stability and Flexibility Direct the Selection of Activating Mutations in Epidermal Growth Factor Receptor Kinase. J Chem Inf Model 2015, 55 (7), 1377–87." />
<meta name="twitter:title" content="Differential Antibody Recognition by Novel SARS-CoV-2 and SARS-CoV Spike Protein Receptor Binding Domains: Mechanistic Insights and Implications for the Design of Diagnostics and Therapeutics" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="The appearance of the novel betacoronavirus SARS-CoV-2 represents a major threat to human health, and its diffusion around the world is causing dramatic consequences. The knowledge of the 3D structures of SARS-CoV-2 proteins can facilitate the development of therapeutic and diagnostic molecules. Specifically, comparative analyses of the structures of SARS-CoV-2 proteins and homologous proteins from previously characterized viruses, such as SARS-CoV, can reveal the common and/or distinctive traits that underlie the mechanisms of recognition of cell receptors and of molecules of the immune system.

Herein, we apply our recently developed energy-based methods for the prediction of antibody-binding epitopes and protein-protein interaction regions to the Receptor Binding Domain (RBD) of the Spike proteins from SARS-CoV-2 and SARS-CoV. Our analysis focusses only on the study of the structure of RBDs in isolation, without making use of any previous knowledge of binding properties. Importantly, our results highlight structural and sequence differences among the regions that are predicted to be immunoreactive and bind/elicit antibodies. These results provide a rational basis to the observation that several SARS-CoV RDB-specific monoclonal antibodies fail to appreciably bind the SARS-CoV-2 counterpart. Furthermore, we correctly identify the region of SARS-CoV-2 RBD that is engaged by the cell receptor ACE2 during viral entry into host cells.

The data, sequences and structures we present here can be useful for the development of novel therapeutic and diagnostic interventions." />
<meta name="og-title" property="og:title" content="Differential Antibody Recognition by Novel SARS-CoV-2 and SARS-CoV Spike Protein Receptor Binding Domains: Mechanistic Insights and Implications for the Design of Diagnostics and Therapeutics" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.13.990267v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="The appearance of the novel betacoronavirus SARS-CoV-2 represents a major threat to human health, and its diffusion around the world is causing dramatic consequences. The knowledge of the 3D structures of SARS-CoV-2 proteins can facilitate the development of therapeutic and diagnostic molecules. Specifically, comparative analyses of the structures of SARS-CoV-2 proteins and homologous proteins from previously characterized viruses, such as SARS-CoV, can reveal the common and/or distinctive traits that underlie the mechanisms of recognition of cell receptors and of molecules of the immune system.

Herein, we apply our recently developed energy-based methods for the prediction of antibody-binding epitopes and protein-protein interaction regions to the Receptor Binding Domain (RBD) of the Spike proteins from SARS-CoV-2 and SARS-CoV. Our analysis focusses only on the study of the structure of RBDs in isolation, without making use of any previous knowledge of binding properties. Importantly, our results highlight structural and sequence differences among the regions that are predicted to be immunoreactive and bind/elicit antibodies. These results provide a rational basis to the observation that several SARS-CoV RDB-specific monoclonal antibodies fail to appreciably bind the SARS-CoV-2 counterpart. Furthermore, we correctly identify the region of SARS-CoV-2 RBD that is engaged by the cell receptor ACE2 during viral entry into host cells.

The data, sequences and structures we present here can be useful for the development of novel therapeutic and diagnostic interventions." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-03-15" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.13.990267v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.13.990267v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.13.990267v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.13.990267v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1196794" />
    <title>Differential Antibody Recognition by Novel SARS-CoV-2 and SARS-CoV Spike Protein Receptor Binding Domains: Mechanistic Insights and Implications for the Design of Diagnostics and Therapeutics | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"pm1XX-xbJswdPME4mS1JistyAnfNDYgZ8i9Z7WR9nbo","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1196794","apath":"\/biorxiv\/early\/2020\/03\/15\/2020.03.13.990267.atom","pisa":"biorxiv;2020.03.13.990267v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.13.990267v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.13.990267v2","title":"Differential Antibody Recognition by Novel SARS-CoV-2 and SARS-CoV Spike Protein Receptor Binding Domains: Mechanistic Insights and Implications for the Design of Diagnostics and Therapeutics","identifier":"node\/1196794"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222RG\\\\\\\/ssReIiJRl3z8fnRkrK3Ood7wXY1s39rMYX9I4SMrCiclIbPi0p6cTKlP82NBL5BpRE9LjY5\\\\\\\/2\\\\\\\/+ojbrw3ytOE69q3YyUvMZpJvTjlc1lJSeuk3bUx9Ss9Np3g9LYV6o4h\\\\\\\/bmeMejLAhAE0Q6cKC1S20CvINA1UY2\\\\\\\/SWaJ+fLZBHM3TRp66iBvkG1TZy2NR+QMBPb9S0TSsS8Erl+Bxc9Zl0fSzDL5WR8bL8VSFEf8F6F\\\\\\\/knDFp7tl2qWXnJhhZzhAjtIaK0Xt437LV060nF5EkCDOuFVnzQUmcK+c+hCnnQxLR2UhIauOteyFAhlIlv+am5jwqv40t+yvl7bmfXrBeDc9p\\\\\\\/xNTqtCX5iqkCM37zLBV3SClPnDPQZJvXHZkRQR8Ranq2mKmxipjiDZ7LezKHGoAn2hFFDhOc7DR+vJGpiALOroHkhu1b9fXdPNpznBcimLgQG1pNU\\\\\\\/YbfKecyDX\\\\\\\/QSlC9PDgIlmOJpSyrogdq94iwj3a2w2l1Rb\\\\\\\/JuqFNJwTtiGIWvY4+HJV6f2mBANZroHwCOm22IWVCBJKyuvQ6W4Bui7CBfPbibMsPtbgL97nJ2Kik\\\\\\\/OLjtjecVLq2YZrca3SqPs3wuVb+F6xArIsCHEEy5Gk\\\\\\\/Syf1UbSIlj2i07akZB\\\\\\\/YtlnpWGHuXWJZrS0rQd4RrLceWfi6HV8PiKdJhl5UT\\\\\\\/lXLdOOy9hekeQ2TanYBpSCr31x+IOfTNX6kxJ7R5eZ7yWfxYJMeOuFTEQSnKki1VPE8NN9256jyTD7SUw3wLXZ1oTXIK\\\\\\\/1tNonXj3rCoYCWbnSJJDhNFYoFmMKBSlgFPdi8yrirHkA+Cynpakbew2wGGiYhk9OZXOswHhLhuz6Utf88dHsdfIlVKQDMbbUQJKyfnAmzTgJY9gdt\\\\\\\/+fb1ExkIL9BqNB2HTTGY8haai9DcGvb9TLpe+insSOBwmZ63xABFYcHs5oXfnniyBCg32ynXey8Pwk+d9DJ8MQap85hbiakiqLprLlCfWJga2PCX1f1KatfAISDXpi7cmFqeiJQw6lpxZp75C8qdu2UWI6\\\\\\\/DzKQi4D9Zn7XKrFLr7cPZBase9CPA2K1P2ueeaS8Zh+3\\\\\\\/1tw4aOpKitVLPDad8fmEOlAK+Hxp9kwOETBRBVcLwq3p7Q8btb09SWPiJlw4bC9nqCOYCfnd2XuMvRIq59DubHyCU3fGt2KSiPi7o+M\\\\\\\/Xjq\\\\\\\/VhMsm\\\\\\\/dG2Zm3T2rwBsLC4tku54pmuZtfwOueAnnAVhITQnrynHjM\\\\\\\/JxZIrsV7hVoFv5U7xyitIiq1YRwa8bJOiWb\\\\\\\/1ysiSNV7OdexKLtGX6G9CrJiQGe\\\\\\\/mDMlPYUGz9wMgiTJX\\\u0022,\\\u0022iv\\\u0022:\\\u0022oHNCfhACoS8IGX1bVOKSuA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022f3511642990c908c87db5759ea737cbbaefe5d750d2841e35c1c54f95dd37a56\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.13.990267v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1196794 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.13.990267v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vMUWZTNaHrZ-m7CMPbJALfYs_Qr2XS6Vtm9TaP4JpgU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1196794" id="node1196794" data-pisa="biorxiv;2020.03.13.990267v2" data-pisa-master="biorxiv;2020.03.13.990267" data-apath="/biorxiv/early/2020/03/15/2020.03.13.990267.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Differential Antibody Recognition by Novel SARS-CoV-2 and SARS-CoV Spike Protein Receptor Binding Domains: Mechanistic Insights and Implications for the Design of Diagnostics and Therapeutics</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Ilda</span> <span class="nlm-surname">D’Annessa</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Filippo</span> <span class="nlm-surname">Marchetti</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-1318-668X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Giorgio</span> <span class="nlm-surname">Colombo</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.13.990267 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1196794" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ilda D’Annessa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>SCITEC-CNR</span>, via Mario Bianco 9, 20131 Milano (<span class='nlm-country'>Italy</span>)</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ilda%2BD%E2%80%99Annessa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=D%E2%80%99Annessa%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIlda%2BD%25E2%2580%2599Annessa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Filippo Marchetti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>University of Pavia, Department of Chemistry</span>, viale Taramelli 12, 27100 Pavia (<span class='nlm-country'>Italy</span>)</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Filippo%2BMarchetti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marchetti%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFilippo%2BMarchetti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Giorgio Colombo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>University of Pavia, Department of Chemistry</span>, viale Taramelli 12, 27100 Pavia (<span class='nlm-country'>Italy</span>)</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Giorgio%2BColombo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Colombo%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGiorgio%2BColombo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1318-668X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Giorgio Colombo</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:g.colombo@unipv.it" class="" data-icon-position="" data-hide-link-title="0">g.colombo@unipv.it</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.13.990267v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1196794" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1196794/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.13.990267v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1196794" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1196794/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.13.990267v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1196794" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1196794/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.13.990267v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1196794" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1196794/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.13.990267v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1196794" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1196794/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">The appearance of the novel betacoronavirus SARS-CoV-2 represents a major threat to human health, and its diffusion around the world is causing dramatic consequences. The knowledge of the 3D structures of SARS-CoV-2 proteins can facilitate the development of therapeutic and diagnostic molecules. Specifically, comparative analyses of the structures of SARS-CoV-2 proteins and homologous proteins from previously characterized viruses, such as SARS-CoV, can reveal the common and/or distinctive traits that underlie the mechanisms of recognition of cell receptors and of molecules of the immune system.</p><p id="p-3">Herein, we apply our recently developed energy-based methods for the prediction of antibody-binding epitopes and protein-protein interaction regions to the Receptor Binding Domain (RBD) of the Spike proteins from SARS-CoV-2 and SARS-CoV. Our analysis focusses only on the study of the structure of RBDs in isolation, without making use of any previous knowledge of binding properties. Importantly, our results highlight structural and sequence differences among the regions that are predicted to be immunoreactive and bind/elicit antibodies. These results provide a rational basis to the observation that several SARS-CoV RDB-specific monoclonal antibodies fail to appreciably bind the SARS-CoV-2 counterpart. Furthermore, we correctly identify the region of SARS-CoV-2 RBD that is engaged by the cell receptor ACE2 during viral entry into host cells.</p><p id="p-4">The data, sequences and structures we present here can be useful for the development of novel therapeutic and diagnostic interventions.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-6">The outbreak of the new betacoronavirus 2019-nCoV, formally named SARS-CoV-2, represents a world-wide epidemic threat that has been causing major disruptions in societies and economies<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>-<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a></sup>. For these reasons the new virus has been internationally declared a public health emergency.</p><p id="p-7">In this context, structural knowledge of the proteins of the virus can provide important information to develop strategies against the infection, ranging from diagnosis to therapy. In general, coronaviruses use this protein localized on the envelope to bind their cell receptors. After binding, a series of molecular events lead to the fusion of the viral membrane with that of the host cell, eventually leading to virus entry and invasion.</p><p id="p-8">One of the necessary steps in this mechanism is the binding of the spike protein Receptor Binding Domain (RBD) to the cell receptor Angiotensin-Converting Enzyme 2 (ACE2). Indeed, knock out of ACE2 in HeLa cells was shown to prevent infection.</p><p id="p-9">For these reasons the development of antibodies that target the spike RBD of coronaviruses and block their interaction with ACE2, disrupting a fundamental infection mechanism, represent attractive therapeutic solutions. Monoclonal antibodies with high affinities had been previously developed against the spike RBD of SARS-CoV. However, despite the correlations between the two viruses, no RBD-targeted SARS-CoV monoclonal antibodies have shown the ability to bind and neutralize 2019-nCoV.</p><p id="p-10">The recent publication of the Cryo-EM structure of the 2019-nCoV spike trimer at 3.5 Å resolution can provide a useful basis to link these observations to molecular causes, guide the design of SARS-CoV-2 specific antibodies, and possibly suggest the basis for molecules able to block cell viral entry.</p><p id="p-11">Here, we carry out a comparative analysis of the spike protein RBDs from SARS-CoV-2 and SARS-CoV to reveal the key properties of their surfaces that can be reconnected to observed differences in antibody (Ab) and receptor binding. Starting from the atomic resolution information available from the solved structures of the two proteins, the MLCE (Matrix of Low Coupling Energies) method is applied to identify the subsets of surface residues that can be defined as "interacting". MLCE analysis of protein energetics accounts for the interactions that each residue establishes with all other residues of the protein it belongs to. The correlation of this parameter with the structural properties of the protein thus highlights which substructures are (pre)organized to interact with a potential partner (a cell receptor or an antibody).</p><p id="p-12">MLCE analyzes the interaction energies of all of the amino acids in a protein. In particular, it computes the nonbonded part of the potential (van der Waals, electrostatic interactions, solvent effects) via an MM/GBSA calculation, obtaining, for a protein composed by <em>N</em> residues, an <em>N</em> × <em>N</em> symmetric interaction matrix <em>M<sub>ij</sub></em>. Eigenvalue decomposition of the matrix highlights the regions of strongest and weakest couplings: the fragments that are on the surface, contiguous in space and weakly coupled to the protein core, define the potential interaction regions. In other words, the putative interacting patches are assumed to be characterized by frustrated intramolecular interactions. The actual interaction with a partner will actually occur if favorable interactions determine a lower free energy for the bound than the unbound state<sup><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>-<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a></sup>. Indeed, Minimal energetic coupling with the rest of the protein allows these substructures to undergo conformational changes, to be recognized by a binding partner (Antibodies, Receptors). All these properties are hallmarks of Antibody binding epitopes and PPI substructures.</p><p id="p-13">The MLCE method for the prediction of Protein-Protein Interaction regions and Antibody binding epitopes has been validated in several previous studies<sup><a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>-<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></sup>.</p></div><div class="section" id="sec-2"><h2 class="">Results and Discussion</h2><p id="p-14">Here, we took advantage of the recent publication of the structures of SARS-CoV-2 spike protein in its pre-fusion state (doi.org/10.1126/science.abb2507) and of known structure of the SARS-CoV spike protein to comparatively apply our interaction epitope prediction method MLCE.</p><p id="p-15">The application of MLCE to the Receptor Binding Domain (RBD, aa 319-591) of the SARS-CoV-2 spike protein (pdb code <span class="underline">6vsb</span>; doi: <span class="underline">10.2210/pdb6vsb/pdb</span>)<sup><a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a></sup> and of the SARS-CoV spike protein (pdb code <span class="underline">5×58</span>; doi: 10.2210/<span class="underline">pdb5X58</span>/pdb) <strong>(<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>)</strong> shows that the two protein regions that contain potential interaction regions/Ab-binding epitopes share the same topological localization at the two opposite ends of the 3D structures <strong>(<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>)</strong>. However, this trait appears to be the only common one. The linear sequences that correspond to the predicted interacting epitope regions are reported in <strong><a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.</strong></p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.13.990267/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of the Receptor Binding Domains of the SARS-CoV-2 and SARS-CoV spike proteins. The predicted epitopes, whose sequences are reported in Tables 1 and 2 are highlighted with colors." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1799408330" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of the Receptor Binding Domains of the SARS-CoV-2 and SARS-CoV spike proteins. The predicted epitopes, whose sequences are reported in Tables 1 and 2 are highlighted with colors.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.13.990267/F1.medium.gif" width="440" height="241"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.13.990267/F1.medium.gif" width="440" height="241"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.13.990267/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.13.990267/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-16" class="first-child">Comparison of the Receptor Binding Domains of the SARS-CoV-2 and SARS-CoV spike proteins. The predicted epitopes, whose sequences are reported in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Tables 1</a> and <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">2</a> are highlighted with colors.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1200247/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1200247/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1200247" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-17" class="first-child">Epitope sequences for the Receptor Binding Domain (RBD, aa 319-591) of the SARS-CoV-2 spike protein (pdb code <span class="underline">6vsb</span>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">Importantly, the predicted epitopes for SARS-CoV spike RBD show a good degree of overlap with the epitopes mapped for neutralizing Ab 80R<sup><a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a></sup>, S230<sup><a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></sup> and m396<sup><a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a></sup>, providing a relevant validation of our approach. However, in spite of the high degree of structural similarity between the two spike proteins, these mAbs failed to bind SARS-CoV-2 RBD, probably due to the low degree of sequence conservation in the regions contacted by these antibodies. The fact that most of the SARS-CoV-directed mAbs are not reactive against the novel betacoronavirus once more highlights the need to discover specific epitopes in SARS-CoV-2.</p><p id="p-19">Actually, except for the common topological localization, the predicted Ab-binding epitopes differ between the two proteins, both in their sequence and 3D structural organization <strong>(<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>, <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">2</a> and <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>)</strong>. In some cases, linear sequences spanning distant segments of the primary structure come together in the three-dimensional structure to form extended substructures potentially (pre)organized for binding. This is particularly true for the conformational epitope at the N- and C-terminal region of SARS-CoV-2 RBD: the interacting region spans a large part of a well-defined betasheet subdomain <strong>(<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>, <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">2</a> and <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>)</strong>. It is tempting to suggest that the expression of such subdomain in isolation could give rise to a highly immunoreactive molecule<sup><a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>-<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a></sup>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.13.990267/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The space filling regions indicate possible conformational epitopes for SARS-CoV-2 formed by the juxtapositions of the sequences reports in Table 1. The region binding to ACE2 and correctly predicted is also highlighted." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1799408330" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The space filling regions indicate possible conformational epitopes for SARS-CoV-2 formed by the juxtapositions of the sequences reports in Table 1. The region binding to ACE2 and correctly predicted is also highlighted.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.13.990267/F2.medium.gif" width="440" height="368"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.13.990267/F2.medium.gif" width="440" height="368"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.13.990267/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.13.990267/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-20" class="first-child">The space filling regions indicate possible conformational epitopes for SARS-CoV-2 formed by the juxtapositions of the sequences reports in <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>. The region binding to ACE2 and correctly predicted is also highlighted.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1200246/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1200246/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1200246" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2.</span> <p id="p-21" class="first-child">Epitope sequences for the Receptor Binding Domain (RBD, aa 319-591) of the SARS-CoV spike protein (pdb code <span class="underline">5×58</span>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">In SARS-CoV-2 RBD, the knowledge of substructures representing potential conformational epitopes may be helpful in guiding the development of peptide-based immunogens able to elicit antibodies that specifically target the cognate protein to which the predicted epitopes belong (<strong><a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a></strong>). Simply speaking, the termini of the linear segments making up the conformational epitopes could be aptly bridged with a number of Gly residues sufficient to approximate the distance in the experimental structure (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Antibodies raised against these molecules, which recapitulate the main determinants of immunoreactivity of the full protein, could have useful applications in a therapeutic setting<sup><a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a>-<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a></sup>. Furthermore, information on potentially specific SARS-CoV-2 spike RBD-specific epitopes can guide the synthesis of mimics of the epitope sequences in the form of peptides for use in diagnostic settings. Serological approaches aimed at revealing antibodies elicited in the host in response to the infection or/and at using specifically selected antibodies to detect relevant 2019-nCoV antigens could indeed be used for the screening of people who may have come into contact with the virus as well as for epidemiological surveillance, a relevant issue considering the number of asymptomatic patients which caused a fast and widespread diffusion of SARS-CoV-2. In general, serological approaches of this kind can be simple and quick to perform and could aptly represent a valid complement to swabs followed by molecular nucleic acid-based methods, currently used as the front-line response to detect the virus in the window period in which host B-cell responses have not yet been elicited.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1200249/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1200249/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1200249" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3.</span> <p id="p-23" class="first-child">Proposed sequences for the mimics of the conformational epitopes predicted for the Receptor Binding Domain (RBD, aa 319-591) of the SARS-CoV-2 spike protein (pdb code 6vsb). Boldface G’s indicate glycines inserted to bridge two parts of a conformational epitope; Bold-face, underlined S’s or T’s indicate serines inserted to substitute for original cysteines or methionines, respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">The specific SARS-CoV-2 RBD epitope sequences and structures identified here can thus be used to design new molecular probes (immunoprobes) with facile synthetic accessibility and the potential to function both as antigen-mimicking baits to target virus-specific antibodies in patients’ fluids, and as immunogens to generate new Abs able to detect relevant antigens<sup><a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>-<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></sup>.</p><p id="p-25">Importantly, after we ran our predictions on the SARS-CoV-2 RBD, crystal structures of this domain in complex with the cellular receptor protein ACE2 was reported by Lan et al. (aa 480-488)<sup><a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a></sup> and by Yan et al.<sup><a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a></sup>. Of note, one predicted interaction region almost completely overlaps with the substructure of spike protein RDB engaged in binding with ACE2 (478 TPCNGVEGF 486). Synthetic mimicry of this sequence could aptly generate high priority candidates aimed at disrupting PPIs between SARS-CoV-2 RBD and its cell receptor ACE2. Such molecules could be used to block viral entry into cells.</p><p id="p-26">Similarly, our approach also predicts a patch on SARS-CoV spike RBD, which is partly involved in binding ACE2 (see<sup><a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a></sup>).</p><p id="p-27">Overall, while the two RBD proteins likely evolved from a common ancestor and share a common fold, they are characterized by different sequences. The physico-chemical properties determined by such sequence differences, efficaciously unveiled by MLCE, reverberate in the modulation of the networks of intraprotein interactions that underlie the (pre)organization of regions for potential recognition of binding partners (cell receptors or antibodies).</p><p id="p-28">These results represent, to the best of our knowledge, one of the first molecular physical-chemistry based rationalization of the reasons why some of the tested SARS-CoV RBD monoclonal Abs do not have appreciable cross-reactivity to SARS-CoV-2 RBD<sup><a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a></sup>, despite the fact that the two domains were recently reported to display the same affinity for the ACE2, with K<sub>D</sub> values of 15.2 nM and 15 nM for SARS-CoV-2 and SARS-CoV, respectively<sup><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a></sup>.</p><p id="p-29">In this complex scenario, our predictions can help disentangle some of the issues related to betacoronavirues infectivity and immunological response.</p></div><div class="section" id="sec-3"><h2 class="">Methods</h2><p id="p-31">The coordinates of the proteins analyzed through the text were downloaded from the pdb with the following codes:</p><p id="p-32">2019-nCoV spike glycoprotein Receptor Binding Domain: PDB ID <span class="underline">6vsb</span></p><p id="p-33">SARS-nCoV spike glycoprotein Receptor Binding Domain: PDB ID <span class="underline">5×58</span></p><p id="p-34">Missing residues belonging to highly flexible loops not solved in the structure of the Spike protein were modeled by homology using the SwissModel web server (<a href="https://swissmodel.expasy.org/">https://swissmodel.expasy.org/</a>), that retrieved the structure of the SARS-CoV <span class="underline">5×58</span> as the best template.</p><p id="p-35">The structures were used as input for epitope prediction using the Matrix of Local Coupling Energies method (MLCE), which combines the analysis of structural determinants of a given protein with its energetic properties<sup><a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">3</a>-<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a></sup>. This approach allows to identify nonoptimized, low-intensity energetic interaction-networks, corresponding to those substructures that can be more prone to establish interactions with Antibodies, and be suitably recognized by binding partners. Briefly, the contiguous regions on the protein surface that are deemed to have minimal coupling energies with the rest of the structure are selected on the basis of the eigenvalue decomposition of the matrix reporting the non-bonded interaction of all residue-pairs. The eigenvector associated to the most negative eigenvalue permits to reconstruct a simplified matrix which reports the maximal and minimal stabilizing residue-pairs in the protein structure. Filtering of the simplified matrix with the contact matrix allows to identify contiguous residue-pairs characterized by their essential degree of coupling to the rest of the protein. Selection of proximal pairs showing minimal coupling with the rest of the protein defines putative epitopes. Selection is carried out on the basis of a threshold value (called softness), which defines the percentage of the set of putative interaction sites by including increasing residue-residue coupling values until the number of couplings that correspond to the lowest contact-filtered pairs under the threshold was reached.</p></div><div class="section" id="sec-4"><h2 class="">Detailed Prediction of protein interaction surfaces: MLCE</h2><p id="p-36">MLCE is a technique based on the analysis of the interaction energies of all the amino acids in a protein<sup><a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">3</a>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>-<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a></sup>. In particular, it computes the non-bonded part of the potential (van der Waals, electrostatic interactions, solvent effects) via a MM/GBSA calculation, obtaining, for a protein composed by <em>N</em> residues, a <em>N × N</em> symmetric interaction matrix <em>M<sub>ij</sub></em>. This matrix can be expressed in terms of its eigenvalues and eigenvectors as
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/15/2020.03.13.990267/embed/graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/15/2020.03.13.990267/embed/graphic-6.gif"/></noscript></span>

</span>
where <em>λ<sub>a</sub></em> is the <em>α</em>-th eigenvalue and <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/15/2020.03.13.990267/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/15/2020.03.13.990267/embed/inline-graphic-1.gif"/></noscript></span></span> is the i-th component of the corresponding eigenvector. The eigenvector with the most negative correspondent eigenvalue contains most of the interaction information for the stabilizing interaction of the system. An approximated interaction matrix <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/15/2020.03.13.990267/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/15/2020.03.13.990267/embed/inline-graphic-2.gif"/></noscript></span></span> is thus given by
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/15/2020.03.13.990267/embed/graphic-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/15/2020.03.13.990267/embed/graphic-7.gif"/></noscript></span>

</span></p><p id="p-37">If the structure of the protein is known, one can estimate a contact matrix <em>C<sub>ij</sub></em> by assuming two amino acids in contact if the distance between two of their heavy atoms is smaller than a threshold. The Hadamard product of the two matrices gives us the matrix of the local coupling energies
<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/15/2020.03.13.990267/embed/graphic-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/15/2020.03.13.990267/embed/graphic-8.gif"/></noscript></span>

</span></p><p id="p-38">We will select as possible interacting zones sets of close by residues that show weak or frustrated interactions.</p><p id="p-39">The analysis of the energetic properties of the surface residues is based on the MLCE method. Basically, we perform a MM/GBSA analysis of the structure in a force field, obtaining a symmetric per-residue interaction matrix <em>M<sub>ij</sub></em> keeping only non-bonded interaction (i.e. electrostatic, van der Waals and solvation contributions). We diagonalize the matrix, obtaining a set of eigenvectors <em>x</em><sup>(<em>i</em>)</sup> sorted following the increasing value of their eigenvectors <em>λ<sub>i</sub></em> where <em>N</em> is the number of amino acids in the sequence. We thus can write the original matrix <em>M<sub>ij</sub></em> as
<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/15/2020.03.13.990267/embed/graphic-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/15/2020.03.13.990267/embed/graphic-9.gif"/></noscript></span>

</span></p><p id="p-40">It has been shown that the first eigenvector alone can be used to build an approximate interaction matrix <em>M</em>: <em>M</em><sub>ij</sub>=<em>λ</em><sub>1</sub><em>x</em><sup>(1)</sup>(<em>i</em>)<em>x</em><sup>(1)</sup>(<em>j</em>), which recapitulates the interactions most relevant for the stabilization of a certain conformation of a defined protein or protein substructure.</p><p id="p-41">The MM/GBSA is made with Amber 14 software with the ff14SB forcefield.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgement</h2><p id="p-30">We acknowledge funding by Regione Lombardia, project READY (Regional Network for developing diagnostic methods in rapid response to emerging epidemics and bio-emergencies)</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-update fn-group-summary-of-updates" id="fn-1"><p id="p-5">The title was changed (upon suggestion by a reader) to better highlight the applicative implications of our mechanistic studies.
We also added a table of designs to better explain the gly-bridging concept we suggested for conformational epitopes.</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990267v2.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chan</span>,  <span class="cit-name-given-names">J. F.-W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yuan</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kok</span>,  <span class="cit-name-given-names">K.-H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">To</span>,  <span class="cit-name-given-names">K. K.-W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chu</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Xing</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yip</span>,  <span class="cit-name-given-names">C. C.-Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Poon</span>,  <span class="cit-name-given-names">R. W.-S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Tsoi</span>,  <span class="cit-name-given-names">H.-W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lo</span>,  <span class="cit-name-given-names">S. K.-F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chan</span>,  <span class="cit-name-given-names">K.-H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Poon</span>,  <span class="cit-name-given-names">V. K.-M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chan</span>,  <span class="cit-name-given-names">W.-M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ip</span>,  <span class="cit-name-given-names">J. D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cai</span>,  <span class="cit-name-given-names">J.-P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cheng</span>,  <span class="cit-name-given-names">V. C.-C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hui</span>,  <span class="cit-name-given-names">C. K.-M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yuen</span>,  <span class="cit-name-given-names">K.-Y.</span></span>, <span class="cit-article-title">A familial cluster of pneumonia associated with the 2019 novel coronavirus indicating person-to-person transmission: a study of a family cluster</span>. <abbr class="cit-jnl-abbrev">The Lancet</abbr> <span class="cit-pub-date">2020</span>, <span class="cit-vol">395</span> (<span class="cit-issue">10223</span>), <span class="cit-fpage">514</span>–<span class="cit-lpage">523</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BLancet%26rft.volume%253D395%26rft.spage%253D514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990267v2.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ren</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Fan</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gu</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cheng</span>,  <span class="cit-name-given-names">Z.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Xia</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wei</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Xie</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yin</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Xiao</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gao</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Xie</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gao</span>,  <span class="cit-name-given-names">Z.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Jin</span>,  <span class="cit-name-given-names">Q.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cao</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-article-title">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</span>. <abbr class="cit-jnl-abbrev">The Lancet</abbr> <span class="cit-pub-date">2020</span>, <span class="cit-vol">395</span> (<span class="cit-issue">10223</span>), <span class="cit-fpage">497</span>–<span class="cit-lpage">506</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BLancet%26rft.volume%253D395%26rft.spage%253D497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990267v2.3" data-doi="10.1016/j.bpj.2010.01.014"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Scarabelli</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Morra</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Colombo</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-article-title">Predicting interaction sited from the energetics of isolated proteins: a new approach to epitope mapping</span>. <abbr class="cit-jnl-abbrev">Biophys. J</abbr>. <span class="cit-pub-date">2010</span>, <span class="cit-vol">98</span> (<span class="cit-issue">9</span>), <span class="cit-fpage">1966</span>–<span class="cit-lpage">1975</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiophysical%2BJournal%26rft.stitle%253DBiophys.%2BJ%26rft.aulast%253DScarabelli%26rft.auinit1%253DG.%26rft.volume%253D98%26rft.issue%253D9%26rft.spage%253D1966%26rft.epage%253D1975%26rft.atitle%253DPredicting%2Binteraction%2Bsites%2Bfrom%2Bthe%2Benergetics%2Bof%2Bisolated%2Bproteins%253A%2Ba%2Bnew%2Bapproach%2Bto%2Bepitope%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bpj.2010.01.014%26rft_id%253Dinfo%253Apmid%252F20441761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bpj.2010.01.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20441761&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F15%2F2020.03.13.990267.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990267v2.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Marchetti</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Capelli</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Rizzato</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Laio</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Colombo</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-article-title">The Subtle Trade-Off between Evolutionary and Energetic Constraints in Protein-Protein Interactions</span>. <abbr class="cit-jnl-abbrev">J. Phys. Chem. Lett</abbr>. <span class="cit-pub-date">2019</span>, <span class="cit-vol">10</span> (<span class="cit-issue">7</span>), <span class="cit-fpage">1489</span>–<span class="cit-lpage">1497</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BPhys.%2BChem.%2BLett%26rft.volume%253D10%26rft.spage%253D1489%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">5.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.13.990267v2.5" data-doi="10.1074/jbc.M110.118513"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Soriani</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Petit</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Grifantini</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Petracca</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gancitano</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Frigimelica</span>,  <span class="cit-name-given-names">E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Nardelli</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Garcia</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Spinelli</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Scarabelli</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Fiorucci</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Affentranger</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ferrer-Navarro</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zacharias</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Colombo</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Vuillard</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Daura</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Grandi</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-article-title">Exploiting antigenic diversity for vaccine design: the chlamydia ArtJ paradigm</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem</abbr>. <span class="cit-pub-date">2010</span>, <span class="cit-vol">285</span> (<span class="cit-issue">39</span>), <span class="cit-fpage">30126</span>–<span class="cit-lpage">30138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M110.118513%26rft_id%253Dinfo%253Apmid%252F20592031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODUvMzkvMzAxMjYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8xNS8yMDIwLjAzLjEzLjk5MDI2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">6.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.13.990267v2.6" data-doi="10.1016/j.chembiol.2013.07.010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gourlay</span>,  <span class="cit-name-given-names">L. J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Peri</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ferrer-Navarro</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Conchillo-Solé</span>,  <span class="cit-name-given-names">O.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gori</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Rinchai</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Thomas</span>,  <span class="cit-name-given-names">R. J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Champion</span>,  <span class="cit-name-given-names">O. L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Michell</span>,  <span class="cit-name-given-names">S. L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kewcharoenwong</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Nithichanon</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lassaux</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Perletti</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Longhi</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lertmemongkolchai</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Titball</span>,  <span class="cit-name-given-names">R. W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Daura</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Colombo</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bolognesi</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-article-title">Exploiting the Burkholderia pseudomallei Acute Phase Antigen BPSL2765 for Structure-Based Epitope Discovery/Design in Structural Vaccinology</span>. <abbr class="cit-jnl-abbrev">Chem. Biol</abbr>. <span class="cit-pub-date">2013</span>, <span class="cit-vol">20</span>, <span class="cit-fpage">1147</span>–<span class="cit-lpage">1156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChem.%2BBiol%26rft.volume%253D20%26rft.spage%253D1147%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chembiol.2013.07.010%26rft_id%253Dinfo%253Apmid%252F23993463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chembiol.2013.07.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23993463&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F15%2F2020.03.13.990267.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">7.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.13.990267v2.7" data-doi="10.1016/j.str.2012.12.004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lassaux</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Peri</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ferrer-Navarro</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gourlay</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gori</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Conchillo-Solé</span>,  <span class="cit-name-given-names">O.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Rinchai</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lertmemongkolchai</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Longhi</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Daura</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Colombo</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bolognesi</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-article-title">A structure-based strategy for epitope discovery in Burkholderia pseudomallei OppA antigen</span>. <abbr class="cit-jnl-abbrev">Structure</abbr> <span class="cit-pub-date">2013</span>, <span class="cit-vol">21</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStructure%26rft.volume%253D21%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.str.2012.12.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.str.2012.12.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">8.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.13.990267v2.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gourlay</span>,  <span class="cit-name-given-names">L. J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lassaux</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Thomas</span>,  <span class="cit-name-given-names">R. J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Peri</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Conchillo-Sole</span>,  <span class="cit-name-given-names">O.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Nithichanon</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ferrer-Navarro</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Vila</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Daura</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lertmemongkolchai</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Titball</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Colombo</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bolognesi</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-article-title">Flagellar subunits as targets for structure-based epitope discovery approaches and melioidosis vaccine development</span>. <abbr class="cit-jnl-abbrev">Febs Journal</abbr> <span class="cit-pub-date">2015</span>, <span class="cit-vol">282</span>, <span class="cit-fpage">338</span>–<span class="cit-lpage">338</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFebs%2BJournal%26rft.volume%253D282%26rft.spage%253D338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">9.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.13.990267v2.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gourlay</span>,  <span class="cit-name-given-names">L. J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Thomas</span>,  <span class="cit-name-given-names">R. J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Peri</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Conchillo-Sole</span>,  <span class="cit-name-given-names">O.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ferrer-Navarro</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Nithichanon</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Vila</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Daura</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lertmemongkolchai</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Titball</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Colombo</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bolognesi</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-article-title">From crystal structure to in silico epitope discovery in the Burkholderia pseudomallei flagellar hook-associated protein FlgK</span>. <abbr class="cit-jnl-abbrev">Febs Journal</abbr> <span class="cit-pub-date">2015</span>, <span class="cit-vol">282</span> (<span class="cit-issue">7</span>), <span class="cit-fpage">1319</span>–<span class="cit-lpage">1333</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFebs%2BJournal%26rft.volume%253D282%26rft.spage%253D1319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">10.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.13.990267v2.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nithichanon</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Rinchai</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gori</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lassaux</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Peri</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Conchillio-Sole</span>,  <span class="cit-name-given-names">O.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ferrer-Navarro</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gourlay</span>,  <span class="cit-name-given-names">L. J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Nardini</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Vila</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Daura</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Colombo</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bolognesi</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lertmemonkolchai</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-article-title">Sequence- and Structure-Based Immunoreactive Epitope Discovery for Burkholderia pseudomallei Flagellin</span>. <abbr class="cit-jnl-abbrev">Plos Neglected Tropical Diseases</abbr> <span class="cit-pub-date">2015</span>, <span class="cit-vol">9</span> (<span class="cit-issue">7</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">11.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.13.990267v2.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gori</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Peri</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Quilici</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Nithichanon</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gaudesi</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Longhi</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gourlay</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bolognesi</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lertmemongkolchai</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Musco</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Colombo</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-article-title">Flexible vs Rigid Epitope Conformations for Diagnostic- and Vaccine-Oriented Applications: Novel Insights from the Burkholderia pseudomallei BPSL2765 Pa13 Epitope</span>. <abbr class="cit-jnl-abbrev">Acs Infectious Diseases</abbr> <span class="cit-pub-date">2016</span>, <span class="cit-vol">2</span> (<span class="cit-issue">3</span>), <span class="cit-fpage">221</span>–<span class="cit-lpage">230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAcs%2BInfectious%2BDiseases%26rft.volume%253D2%26rft.spage%253D221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">12.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.13.990267v2.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gori</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sola</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gagni</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bruni</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Liprino</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Peri</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Colombo</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cretich</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chiari</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-article-title">Screening Complex Biological Samples with Peptide Microarrays: The Favorable Impact of Probe Orientation via Chemoselective Immobilization Strategies on Clickable Polymeric Coatings</span>. <abbr class="cit-jnl-abbrev">Bioconjugate Chemistry</abbr> <span class="cit-pub-date">2016</span>, <span class="cit-vol">27</span> (<span class="cit-issue">11</span>), <span class="cit-fpage">2669</span>–<span class="cit-lpage">2677</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioconjugate%2BChemistry%26rft.volume%253D27%26rft.spage%253D2669%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">13.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.13.990267v2.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sola</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gagni</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">D’Annessa</span>,  <span class="cit-name-given-names">I.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Capelli</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bertino</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Romanato</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Damin</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bergamaschi</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Marchisio</span>,  <span class="cit-name-given-names">E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cuzzocrea</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Bombaci</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Grifantini</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chiari</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Colombo</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gori</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cretich</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-article-title">Enhancing Antibody Serodiagnosis Using a Controlled Peptide Coimmobilization Strategy</span>. <abbr class="cit-jnl-abbrev">ACS Infectious Diseases</abbr> <span class="cit-pub-date">2018</span>, <span class="cit-vol">4</span> (<span class="cit-issue">6</span>), <span class="cit-fpage">998</span>–<span class="cit-lpage">1006</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DACS%2BInfectious%2BDiseases%26rft.volume%253D4%26rft.spage%253D998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990267v2.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bergamaschi</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Fassi</span>,  <span class="cit-name-given-names">E. M. A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Romanato</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">D’Annessa</span>,  <span class="cit-name-given-names">I.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Odinolfi</span>,  <span class="cit-name-given-names">M. T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Brambilla</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Damin</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chiari</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gori</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Colombo</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cretich</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-article-title">Computational Analysis of Dengue Virus Envelope Protein (E) Reveals an Epitope with Flavivirus Immunodiagnostic Potential in Peptide Microarrays</span>. <abbr class="cit-jnl-abbrev">International journal of molecular sciences</abbr> <span class="cit-pub-date">2019</span>, <span class="cit-vol">20</span> (<span class="cit-issue">8</span>), <span class="cit-fpage">1921</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2Bjournal%2Bof%2Bmolecular%2Bsciences%26rft.volume%253D20%26rft.spage%253D1921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990267v2.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wrapp</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">N.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Corbett</span>,  <span class="cit-name-given-names">K. S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Goldsmith</span>,  <span class="cit-name-given-names">J. A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hsieh</span>,  <span class="cit-name-given-names">C.-L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Abiona</span>,  <span class="cit-name-given-names">O.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Graham</span>,  <span class="cit-name-given-names">B. S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">McLellan</span>,  <span class="cit-name-given-names">J. S.</span></span>, <span class="cit-article-title">Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-pub-date">2020</span>, <span class="cit-fpage">eabb2507</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990267v2.16" data-doi="10.1074/jbc.M603275200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hwang</span>,  <span class="cit-name-given-names">W. C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Santelli</span>,  <span class="cit-name-given-names">E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sui</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Jaroszewski</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Stec</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Farzan</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Marasco</span>,  <span class="cit-name-given-names">W. A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Liddington</span>,  <span class="cit-name-given-names">R. C.</span></span>, <span class="cit-article-title">Structural Basis of Neutralization by a Human Anti-severe Acute Respiratory Syndrome Spike Protein Antibody, 80R</span>. <abbr class="cit-jnl-abbrev">Journal of Biological Chemistry</abbr> <span class="cit-pub-date">2006</span>, <span class="cit-vol">281</span> (<span class="cit-issue">45</span>), <span class="cit-fpage">34610</span>–<span class="cit-lpage">34616</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M603275200%26rft_id%253Dinfo%253Apmid%252F16954221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODEvNDUvMzQ2MTAiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8xNS8yMDIwLjAzLjEzLjk5MDI2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990267v2.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rockx</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Corti</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Donaldson</span>,  <span class="cit-name-given-names">E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sheahan</span>,  <span class="cit-name-given-names">T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Stadler</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lanzavecchia</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Baric</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-article-title">Structural Basis for Potent Cross-Neutralizing Human Monoclonal Antibody Protection Against Lethal Human and Zoonotic SARS-CoV Challenge</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr> <span class="cit-pub-date">2008</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990267v2.18" data-doi="10.1073/pnas.0701000104"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">Z.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chakraborti</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">He</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Roberts</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sheahan</span>,  <span class="cit-name-given-names">T.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Xiao</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Hensley</span>,  <span class="cit-name-given-names">L. E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Prabakaran</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Rockx</span>,  <span class="cit-name-given-names">B.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Sidorov</span>,  <span class="cit-name-given-names">I. A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Corti</span>,  <span class="cit-name-given-names">D.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Vogel</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Feng</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">J.-O.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">L.-F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Baric</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lanzavecchia</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Curtis</span>,  <span class="cit-name-given-names">K. M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Nabel</span>,  <span class="cit-name-given-names">G. J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Subbarao</span>,  <span class="cit-name-given-names">K.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Dimitrov</span>,  <span class="cit-name-given-names">D. S.</span></span>, <span class="cit-article-title">Potent crossreactive neutralization of SARS coronavirus isolates by human monoclonal antibodies</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr> <span class="cit-pub-date">2007</span>, <span class="cit-vol">104</span> (<span class="cit-issue">29</span>), <span class="cit-fpage">12123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0701000104%26rft_id%253Dinfo%253Apmid%252F17620608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzI5LzEyMTIzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTUvMjAyMC4wMy4xMy45OTAyNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990267v2.19" data-doi="10.1038/nrmicro2893"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dormitzer</span>,  <span class="cit-name-given-names">P. R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Grandi</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Rappuoli</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-article-title">Structural vaccinology starts to deliver</span>. <abbr class="cit-jnl-abbrev">Nat Rev Microbiol</abbr> <span class="cit-pub-date">2012</span>, <span class="cit-vol">10</span> (<span class="cit-issue">12</span>), <span class="cit-fpage">807</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DDormitzer%26rft.auinit1%253DP.%2BR.%26rft.volume%253D10%26rft.issue%253D12%26rft.spage%253D807%26rft.epage%253D813%26rft.atitle%253DStructural%2Bvaccinology%2Bstarts%2Bto%2Bdeliver.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro2893%26rft_id%253Dinfo%253Apmid%252F23154260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro2893&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23154260&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F15%2F2020.03.13.990267.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">20.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.13.990267v2.20" data-doi="10.1038/nri3694"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">De Gregorio</span>,  <span class="cit-name-given-names">E.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Rappuoli</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-article-title">From empiricism to rational design: a personal perspective of the evolution of vaccine development</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Immunol</abbr>. <span class="cit-pub-date">2014</span>, <span class="cit-vol">14</span> (<span class="cit-issue">7</span>), <span class="cit-fpage">505</span>–<span class="cit-lpage">514</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BImmunol%26rft.volume%253D14%26rft.spage%253D505%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnri3694%26rft_id%253Dinfo%253Apmid%252F24925139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nri3694&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24925139&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F15%2F2020.03.13.990267.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990267v2.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thomas</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Dilbarova</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Rappuoli</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-article-title">Future Challenges for Vaccinologists</span>. <abbr class="cit-jnl-abbrev">Methods Mol Biol</abbr> <span class="cit-pub-date">2016</span>, <span class="cit-vol">1403</span>, <span class="cit-fpage">41</span>–<span class="cit-lpage">55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol%2BBiol%26rft.volume%253D1403%26rft.spage%253D41%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990267v2.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peri</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gagni</span>,  <span class="cit-name-given-names">P.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Combi</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gori</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chiari</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Longhi</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Cretich</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Colombo</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-article-title">Rational epitope design for protein targeting</span>. . <abbr class="cit-jnl-abbrev">ACS Chemical Biology</abbr> <span class="cit-pub-date">2013</span>, <span class="cit-vol">8</span>, <span class="cit-fpage">397</span>–<span class="cit-lpage">404</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DACS%2BChemical%2BBiology%26rft.volume%253D8%26rft.spage%253D397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990267v2.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cretich</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Gori</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">D’Annessa</span>,  <span class="cit-name-given-names">I.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Chiari</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Colombo</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-article-title">Peptides for Infectious Diseases: From Probe Design to Diagnostic Microarrays</span>. <abbr class="cit-jnl-abbrev">Antibodies (Basel)</abbr> <span class="cit-pub-date">2019</span>, <span class="cit-vol">8</span> (<span class="cit-issue">1</span>), <span class="cit-fpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntibodies%2B%2528Basel%2529%26rft.volume%253D8%26rft.spage%253D23%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990267v2.24" data-doi="2020.02.19.956235"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lan</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ge</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">J.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Shan</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">H.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Fan</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Q.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Q.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-article-title">Crystal structure of the 2019-nCoV spike receptor-binding domain bound with the ACE2 receptor</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-pub-date">2020</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>2020.02.19.956235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F2020.02.19.956235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=2020.02.19.956235&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990267v2.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yan</span>,  <span class="cit-name-given-names">R.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Xia</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-article-title">Structural basis for the recognition of the SARS-CoV-2 by full-length human ACE2</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-pub-date">2020</span>, <span class="cit-fpage">eabb2762</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990267v2.26" data-doi="10.1126/science.1116480"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">F.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Farzan</span>,  <span class="cit-name-given-names">M.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Harrison</span>,  <span class="cit-name-given-names">S. C.</span></span>, <span class="cit-article-title">Structure of SARS Coronavirus Spike ReceptorBinding Domain Complexed with Receptor</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-pub-date">2005</span>, <span class="cit-vol">309</span> (<span class="cit-issue">5742</span>), <span class="cit-fpage">1864</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLi%26rft.auinit1%253DF.%26rft.volume%253D309%26rft.issue%253D5742%26rft.spage%253D1864%26rft.epage%253D1868%26rft.atitle%253DStructure%2Bof%2BSARS%2BCoronavirus%2BSpike%2BReceptor-Binding%2BDomain%2BComplexed%2Bwith%2BReceptor%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1116480%26rft_id%253Dinfo%253Apmid%252F16166518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDkvNTc0Mi8xODY0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTUvMjAyMC4wMy4xMy45OTAyNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990267v2.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tian</span>,  <span class="cit-name-given-names">X.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">C.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Xia</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">Z.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">L.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">S.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Z.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">Y.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Ying</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-article-title">Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody</span>. <abbr class="cit-jnl-abbrev">Emerging Microbes &amp; Infections</abbr> <span class="cit-pub-date">2020</span>, <span class="cit-vol">9</span> (<span class="cit-issue">1</span>), <span class="cit-fpage">382</span>–<span class="cit-lpage">385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2BMicrobes%2B%2526%2BInfections%26rft.volume%253D9%26rft.spage%253D382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990267v2.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Morra</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Colombo</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-article-title">Relationship between energy distribution and fold stability: Insights from molecular dynamics simulations of native and mutant proteins</span>. <abbr class="cit-jnl-abbrev">Proteins: Struct. Funct. and Bioinf</abbr>. <span class="cit-pub-date">2008</span>, <span class="cit-vol">72</span> (<span class="cit-issue">2</span>), <span class="cit-fpage">660</span>–<span class="cit-lpage">672</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteins%253A%2BStruct.%2BFunct.%2Band%2BBioinf%26rft.volume%253D72%26rft.spage%253D660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">29.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.13.990267v2.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Genoni</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Morra</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Colombo</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-article-title">Identification of Domains in Protein Structures from the Analysis of Intramolecular Interactions</span>. <abbr class="cit-jnl-abbrev">J. Phys. Chem. B</abbr>. <span class="cit-pub-date">2012</span>, <span class="cit-vol">116</span> (<span class="cit-issue">10</span>), <span class="cit-fpage">3331</span>–<span class="cit-lpage">3343</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BPhysical%2BChemistry.%2BB%252C%2BCondensed%2BMatter%252C%2BMaterials%252C%2BSurfaces%252C%2BInterfaces%2Band%2BBiophysical%26rft.stitle%253DJournal%2Bof%2BPhysical%2BChemistry.%2BB%252C%2BCondensed%2BMatter%252C%2BMaterials%252C%2BSurfaces%252C%2BInterfaces%2Band%2BBiophysical%26rft.aulast%253DGenoni%26rft.auinit1%253DA.%26rft.volume%253D116%26rft.issue%253D10%26rft.spage%253D3331%26rft.epage%253D3343%26rft.atitle%253DIdentification%2Bof%2Bdomains%2Bin%2Bprotein%2Bstructures%2Bfrom%2Bthe%2Banalysis%2Bof%2Bintramolecular%2Binteractions.%26rft_id%253Dinfo%253Apmid%252F22384792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22384792&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F15%2F2020.03.13.990267.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">30.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.13.990267v2.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Morra</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Genoni</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Colombo</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-article-title">Mechanisms of Differential Allosteric Modulation in Homologous Proteins: Insights from the Analysis of Internal Dynamics and Energetics of PDZ Domains</span>. <abbr class="cit-jnl-abbrev">J Chem Theory Comput</abbr>. <span class="cit-pub-date">2014</span>, <span class="cit-vol">10</span> (<span class="cit-issue">12</span>), <span class="cit-fpage">5677</span>–<span class="cit-lpage">5689</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BChem%2BTheory%2BComput%26rft.volume%253D10%26rft.spage%253D5677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.13.990267v2.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Paladino</span>,  <span class="cit-name-given-names">A.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Morra</span>,  <span class="cit-name-given-names">G.</span></span>; <span class="cit-auth"><span class="cit-name-surname">Colombo</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-article-title">Structural Stability and Flexibility Direct the Selection of Activating Mutations in Epidermal Growth Factor Receptor Kinase</span>. <abbr class="cit-jnl-abbrev">J Chem Inf Model</abbr> <span class="cit-pub-date">2015</span>, <span class="cit-vol">55</span> (<span class="cit-issue">7</span>), <span class="cit-fpage">1377</span>–<span class="cit-lpage">87</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BChem%2BInf%2BModel%26rft.volume%253D55%26rft.spage%253D1377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.13.990267v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.13.990267v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.02.24.961565v2" title="Beyond locomotion: in the mouse the mapping between sensations and behaviours unfolds in a higher dimensional space" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.02.974246v2" title="Multi-dimensional predictions of psychotic symptoms via machine learning" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 15, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.13.990267v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.13.990267v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.13.990267v2" class="active" data-icon-position="" data-hide-link-title="0">Differential Antibody Recognition by Novel SARS-CoV-2 and SARS-CoV Spike Protein Receptor Binding Domains: Mechanistic Insights and Implications for the Design of Diagnostics and Therapeutics</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1196794" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Ps8fZay2nw43W7f84AFgZIz3kpH87qB7rWxtVv3DVlg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1196794" id="node1196794" data-pisa="biorxiv;2020.03.13.990267v2" data-pisa-master="biorxiv;2020.03.13.990267" data-seqnum="1196794" data-apath="/biorxiv/early/2020/03/15/2020.03.13.990267.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Differential Antibody Recognition by Novel SARS-CoV-2 and SARS-CoV Spike Protein Receptor Binding Domains: Mechanistic Insights and Implications for the Design of Diagnostics and Therapeutics</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Ilda</span> <span class="nlm-surname">D’Annessa</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Filippo</span> <span class="nlm-surname">Marchetti</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Giorgio</span> <span class="nlm-surname">Colombo</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.13.990267; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.13.990267 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.13.990267v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.990267v2&amp;title=Differential%20Antibody%20Recognition%20by%20Novel%20SARS-CoV-2%20and%20SARS-CoV%20Spike%20Protein%20Receptor%20Binding%20Domains%3A%20Mechanistic%20Insights%20and%20Implications%20for%20the%20Design%20of%20Diagnostics%20and%20Therapeutics" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.990267v2&amp;title=Differential%20Antibody%20Recognition%20by%20Novel%20SARS-CoV-2%20and%20SARS-CoV%20Spike%20Protein%20Receptor%20Binding%20Domains%3A%20Mechanistic%20Insights%20and%20Implications%20for%20the%20Design%20of%20Diagnostics%20and%20Therapeutics" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.990267v2&amp;text=Differential%20Antibody%20Recognition%20by%20Novel%20SARS-CoV-2%20and%20SARS-CoV%20Spike%20Protein%20Receptor%20Binding%20Domains%3A%20Mechanistic%20Insights%20and%20Implications%20for%20the%20Design%20of%20Diagnostics%20and%20Therapeutics" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.990267v2&amp;title=Differential%20Antibody%20Recognition%20by%20Novel%20SARS-CoV-2%20and%20SARS-CoV%20Spike%20Protein%20Receptor%20Binding%20Domains%3A%20Mechanistic%20Insights%20and%20Implications%20for%20the%20Design%20of%20Diagnostics%20and%20Therapeutics" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.990267v2&amp;t=Differential%20Antibody%20Recognition%20by%20Novel%20SARS-CoV-2%20and%20SARS-CoV%20Spike%20Protein%20Receptor%20Binding%20Domains%3A%20Mechanistic%20Insights%20and%20Implications%20for%20the%20Design%20of%20Diagnostics%20and%20Therapeutics" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.990267v2&amp;title=Differential%20Antibody%20Recognition%20by%20Novel%20SARS-CoV-2%20and%20SARS-CoV%20Spike%20Protein%20Receptor%20Binding%20Domains%3A%20Mechanistic%20Insights%20and%20Implications%20for%20the%20Design%20of%20Diagnostics%20and%20Therapeutics" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.990267v2&amp;title=Differential%20Antibody%20Recognition%20by%20Novel%20SARS-CoV-2%20and%20SARS-CoV%20Spike%20Protein%20Receptor%20Binding%20Domains%3A%20Mechanistic%20Insights%20and%20Implications%20for%20the%20Design%20of%20Diagnostics%20and%20Therapeutics" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1196794" id="node1196794--2" data-pisa="biorxiv;2020.03.13.990267v2" data-pisa-master="biorxiv;2020.03.13.990267" data-seqnum="1196794" data-apath="/biorxiv/early/2020/03/15/2020.03.13.990267.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Differential Antibody Recognition by Novel SARS-CoV-2 and SARS-CoV Spike Protein Receptor Binding Domains: Mechanistic Insights and Implications for the Design of Diagnostics and Therapeutics</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Ilda</span> <span class="nlm-surname">D’Annessa</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Filippo</span> <span class="nlm-surname">Marchetti</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Giorgio</span> <span class="nlm-surname">Colombo</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.13.990267; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.13.990267 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Differential%20Antibody%20Recognition%20by%20Novel%20SARS-CoV-2%20and%20SARS-CoV%20Spike%20Protein%20Receptor%20Binding%20Domains%3A%20Mechanistic%20Insights%20and%20Implications%20for%20the%20Design%20of%20Diagnostics%20and%20Therapeutics&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=D%E2%80%99Annessa%2C+Ilda&amp;rft.au=Marchetti%2C+Filippo&amp;rft.au=Colombo%2C+Giorgio"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1196794/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1196794/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1196794/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1196794/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1196794/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1196794/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1196794/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1196794/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1196794/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1196794/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1196794/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1196794/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.990267v2&amp;count=horizontal&amp;via=&amp;text=Differential%20Antibody%20Recognition%20by%20Novel%20SARS-CoV-2%20and%20SARS-CoV%20Spike%20Protein%20Receptor%20Binding%20Domains%3A%20Mechanistic%20Insights%20and%20Implications%20for%20the%20Design%20of%20Diagnostics%20and%20Therapeutics&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.990267v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.13.990267v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.13.990267v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/biochemistry" class="highlight" data-icon-position="" data-hide-link-title="0">Biochemistry<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
