<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557/embed/graphic-5.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/04/07/2020.04.06.027557.atom" />
<meta name="HW.pisa" content="biorxiv;2020.04.06.027557v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Translational adaptation of human viruses to the tissues they infect" />
<meta name="DC.Identifier" content="10.1101/2020.04.06.027557" />
<meta name="DC.Date" content="2020-04-07" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Viruses need to hijack the translational machinery of the host cell for a productive infection to happen. However, given the dynamic landscape of tRNA pools among tissues, it is unclear whether different viruses infecting different tissues have adapted their codon usage toward their tropism. Here, we collect the coding sequences of over 500 human-infecting viruses and determine that tropism explains changes in codon usage. Using an in silico model of translational efficiency, we validate the correspondence of the viral codon usage with the translational machinery of their tropism. In particular, we propose that the improved translational adaptation to the upper respiratory airways of the pandemic agent SARS-CoV-2 coronavirus could enhance its transmissibility. Furthermore, this correspondence is specifically defined in early viral proteins, as upon infection cells undergo reprogramming of tRNA pools that favors the translation of late counterparts." />
<meta name="DC.Contributor" content="Xavier Hernandez-Alias" />
<meta name="DC.Contributor" content="Martin H. Schaefer" />
<meta name="DC.Contributor" content="Luis Serrano" />
<meta name="article:published_time" content="2020-04-07" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Translational adaptation of human viruses to the tissues they infect" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Viruses need to hijack the translational machinery of the host cell for a productive infection to happen. However, given the dynamic landscape of tRNA pools among tissues, it is unclear whether different viruses infecting different tissues have adapted their codon usage toward their tropism. Here, we collect the coding sequences of over 500 human-infecting viruses and determine that tropism explains changes in codon usage. Using an in silico model of translational efficiency, we validate the correspondence of the viral codon usage with the translational machinery of their tropism. In particular, we propose that the improved translational adaptation to the upper respiratory airways of the pandemic agent SARS-CoV-2 coronavirus could enhance its transmissibility. Furthermore, this correspondence is specifically defined in early viral proteins, as upon infection cells undergo reprogramming of tRNA pools that favors the translation of late counterparts.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.04.06.027557v1" />
<meta name="citation_id" content="2020.04.06.027557v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.04.06.027557v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.06.027557v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.06.027557v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.04.06.027557" />
<meta name="citation_num_pages" content="23" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.04.06.027557" />
<meta name="citation_author" content="Xavier Hernandez-Alias" />
<meta name="citation_author_institution" content="Centre for Genomic Regulation (CRG), The Barcelona Institute of Science and Technology" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8633-499X" />
<meta name="citation_author" content="Martin H. Schaefer" />
<meta name="citation_author_institution" content="IEO European Institute of Oncology IRCCS, Department of Experimental Oncology" />
<meta name="citation_author_email" content="martin.schaefer@ieo.it" />
<meta name="citation_author_email" content="luis.serrano@crg.eu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7503-6364" />
<meta name="citation_author" content="Luis Serrano" />
<meta name="citation_author_institution" content="Centre for Genomic Regulation (CRG), The Barcelona Institute of Science and Technology" />
<meta name="citation_author_institution" content="Universitat Pompeu Fabra (UPF)" />
<meta name="citation_author_institution" content="ICREA" />
<meta name="citation_author_email" content="martin.schaefer@ieo.it" />
<meta name="citation_author_email" content="luis.serrano@crg.eu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5276-1392" />
<meta name="citation_reference" content="Grantham, R., Gautier, C., Gouy, M., Jacobzone, M. &amp; Mercier, R. Codon catalog usage is a genome strategy modulated for gene expressivity. Nucleic Acids Res. 9, r43–74 (1981)." />
<meta name="citation_reference" content="Hernandez-Alias, X., Benisty, H., Schaefer, M. H. &amp; Serrano, L. Translational efficiency across healthy and tumor tissues is proliferation-related. Mol. Syst. Biol. 16, e9275 (2020)." />
<meta name="citation_reference" content="Plotkin, J. B. &amp; Kudla, G. Synonymous but not the same: the causes and consequences of codon bias. Nat. Rev. Genet. 12, 32–42 (2011)." />
<meta name="citation_reference" content="Sharp, P. M., Stenico, M., Peden, J. F. &amp; Lloyd, A. T. Codon usage: mutational bias, translational selection, or both? Biochem. Soc. Trans. 21, 835–841 (1993)." />
<meta name="citation_reference" content="Morgado, S. &amp; Vicente, A. C. Global In-Silico Scenario of tRNA Genes and Their Organization in Virus Genomes. Viruses-Basel 11, 180 (2019)." />
<meta name="citation_reference" content="Lucks, J. B., Nelson, D. R., Kudla, G. R. &amp; Plotkin, J. B. Genome Landscapes and Bacteriophage Codon Usage. Plos Comput. Biol. 4, e1000001 (2008)." />
<meta name="citation_reference" content="Carbone, A. Codon Bias is a Major Factor Explaining Phage Evolution in Translationally Biased Hosts. J. Mol. Evol. 66, 210–223 (2008)." />
<meta name="citation_reference" content="Ahlgren, N. A., Ren, J., Lu, Y. Y., Fuhrman, J. A. &amp; Sun, F. Alignment-free d2* oligonucleotide frequency dissimilarity measure improves prediction of hosts from metagenomically-derived viral sequences. Nucleic Acids Res. 45, 39–53 (2017)." />
<meta name="citation_reference" content="Ren, J., Ahlgren, N. A., Lu, Y. Y., Fuhrman, J. A. &amp; Sun, F. VirFinder: a novel k-mer based tool for identifying viral sequences from assembled metagenomic data. Microbiome 5, 69 (2017)." />
<meta name="citation_reference" content="Bahir, I., Fromer, M., Prat, Y. &amp; Linial, M. Viral adaptation to host: a proteome-based analysis of codon usage and amino acid preferences. Mol. Syst. Biol. 5, (2009)." />
<meta name="citation_reference" content="Jenkins, G. M. &amp; Holmes, E. C. The extent of codon usage bias in human RNA viruses and its evolutionary origin. Virus Res. 92, 1–7 (2003)." />
<meta name="citation_reference" content="Lauring, A. S., Jones, J. O. &amp; Andino, R. Rationalizing the development of live attenuated virus vaccines. Nat. Biotechnol. 28, 573–579 (2010)." />
<meta name="citation_reference" content="Pouyet, F., Mouchiroud, D., Duret, L. &amp; Sémon, M. Recombination, meiotic expression and human codon usage. eLife 6, e27344 (2017)." />
<meta name="citation_reference" content="Gingold, H. et al. A Dual Program for Translation Regulation in Cellular Proliferation and Differentiation. Cell 158, 1281–1292 (2014)." />
<meta name="citation_reference" content="Zhao, K.-N., Gu, W., Fang, N. X., Saunders, N. A. &amp; Frazer, I. H. Gene codon composition determines differentiation-dependent expression of a viral capsid gene in keratinocytes in vitro and in vivo. Mol. Cell. Biol. 25, 8643–8655 (2005)." />
<meta name="citation_reference" content="Zhou, J., Liu, W. J., Peng, S. W., Sun, X. Y. &amp; Frazer, I. Papillomavirus capsid protein expression level depends on the match between codon usage and tRNA availability. J. Virol. 73, 4972–4982 (1999)." />
<meta name="citation_reference" content="van Weringh, A. et al. HIV-1 modulates the tRNA pool to improve translation efficiency. Mol. Biol. Evol. 28, 1827–1834 (2011)." />
<meta name="citation_reference" content="Li, M. et al. Codon-usage-based inhibition of HIV protein synthesis by human schlafen 11. Nature 491, 125–128 (2012)." />
<meta name="citation_reference" content="B. Miller, J., Hippen, A. A., M. Wright, S., Morris, C. &amp; G. Ridge, P. Human viruses have codon usage biases that match highly expressed proteins in the tissues they infect. Biomed. Genet. Genomics 2, (2017)." />
<meta name="citation_reference" content="World Health Organization. Novel Coronavirus (2019-nCoV) situation reports. https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports." />
<meta name="citation_reference" content="Sungnak, W., Huang, N., Bécavin, C., Berg, M. &amp; Network, H. L. B. SARS-CoV-2 Entry Genes Are Most Highly Expressed in Nasal Goblet and Ciliated Cells within Human Airways. ArXiv200306122 Q-Bio (2020)." />
<meta name="citation_reference" content="Zhao, Y. et al. Single-cell RNA expression profiling of ACE2, the putative receptor of Wuhan 2019-nCov. bioRxiv 2020.01.26.919985 (2020) doi:10.1101/2020.01.26.919985." />
<meta name="citation_reference" content="Baig, A. M., Khaleeq, A., Ali, U. &amp; Syeda, H. Evidence of the COVID-19 Virus Targeting the CNS: Tissue Distribution, Host–Virus Interaction, and Proposed Neurotropic Mechanisms. ACS Chem. Neurosci. (2020) doi:10.1021/acschemneuro.0c00122." />
<meta name="citation_reference" content="Zhang, H. et al. The digestive system is a potential route of 2019-nCov infection: a bioinformatics analysis based on single-cell transcriptomes. bioRxiv 2020.01.30.927806 (2020) doi:10.1101/2020.01.30.927806." />
<meta name="citation_reference" content="Hulo, C. et al. ViralZone: a knowledge resource to understand virus diversity. Nucleic Acids Res. 39, D576–582 (2011)." />
<meta name="citation_reference" content="Aiewsakun, P. &amp; Simmonds, P. The genomic underpinnings of eukaryotic virus taxonomy: creating a sequence-based framework for family-level virus classification. Microbiome 6, 38 (2018)." />
<meta name="citation_reference" content="Gogakos, T. et al. Characterizing Expression and Processing of Precursor and Mature Human tRNAs by Hydro-tRNAseq and PAR-CLIP. Cell Rep. 20, 1463–1475 (2017)." />
<meta name="citation_reference" content="Zhang, Z. et al. Global analysis of tRNA and translation factor expression reveals a dynamic landscape of translational regulation in human cancers. Commun. Biol. 1, 234 (2018)." />
<meta name="citation_reference" content="Belalov, I. S. &amp; Lukashev, A. N. Causes and Implications of Codon Usage Bias in RNA Viruses. PLoS ONE 8, e56642 (2013)." />
<meta name="citation_reference" content="Shackelton, L. A., Parrish, C. R. &amp; Holmes, E. C. Evolutionary basis of codon usage and nucleotide composition bias in vertebrate DNA viruses. J. Mol. Evol. 62, 551–563 (2006)." />
<meta name="citation_reference" content="Drosten, C. et al. Identification of a novel coronavirus in patients with severe acute respiratory syndrome. N. Engl. J. Med. 348, 1967–1976 (2003)." />
<meta name="citation_reference" content="Zaki, A. M., van Boheemen, S., Bestebroer, T. M., Osterhaus, A. D. M. E. &amp; Fouchier, R. A. M. Isolation of a novel coronavirus from a man with pneumonia in Saudi Arabia. N. Engl. J. Med. 367, 1814–1820 (2012)." />
<meta name="citation_reference" content="Zhou, P. et al. A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature 579, 270–273 (2020)." />
<meta name="citation_reference" content="Broszeit, F. et al. N-Glycolylneuraminic Acid as a Receptor for Influenza A Viruses. Cell Rep. 27, 3284–3294.e6 (2019)." />
<meta name="citation_reference" content="Raj, V. S. et al. Dipeptidyl peptidase 4 is a functional receptor for the emerging human coronavirus-EMC. Nature 495, 251–254 (2013)." />
<meta name="citation_reference" content="Hoffmann, M. et al. SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor. Cell 0, (2020)." />
<meta name="citation_reference" content="Wang, K. et al. SARS-CoV-2 invades host cells via a novel route: CD147-spike protein. bioRxiv 2020.03.14.988345 (2020) doi:10.1101/2020.03.14.988345." />
<meta name="citation_reference" content="Lv, L., Li, G., Chen, J., Liang, X. &amp; Li, Y. Comparative genomic analysis revealed specific mutation pattern between human coronavirus SARS-CoV-2 and Bat-SARSr-CoV RaTG13. bioRxiv 2020.02.27.969006 (2020) doi:10.1101/2020.02.27.969006." />
<meta name="citation_reference" content="Knipe, D. M. &amp; Howley, P. M. Fields Virology. (Lippincott Williams&amp;Wilki, 2013)." />
<meta name="citation_reference" content="Chang, S. T. et al. Next-generation sequencing of small RNAs from HIV-infected cells identifies phased microrna expression patterns and candidate novel microRNAs differentially expressed upon infection. mBio 4, e00549–00512 (2013)." />
<meta name="citation_reference" content="Shi, J. et al. Deep RNA Sequencing Reveals a Repertoire of Human Fibroblast Circular RNAs Associated with Cellular Responses to Herpes Simplex Virus 1 Infection. Cell. Physiol. Biochem. Int. J. Exp. Cell. Physiol. Biochem. Pharmacol. 47, 2031–2045 (2018)." />
<meta name="citation_reference" content="Stark, T. J., Arnold, J. D., Spector, D. H. &amp; Yeo, G. W. High-resolution profiling and analysis of viral and host small RNAs during human cytomegalovirus infection. J. Virol. 86, 226–235 (2012)." />
<meta name="citation_reference" content="Franzo, G., Tucciarone, C. M., Cecchinato, M. &amp; Drigo, M. Canine parvovirus type 2 (CPV-2) and Feline panleukopenia virus (FPV) codon bias analysis reveals a progressive adaptation to the new niche after the host jump. Mol. Phylogenet. Evol. 114, 82–92 (2017)." />
<meta name="citation_reference" content="Luo, W. et al. The fit of codon usage of human-isolated avian influenza A viruses to human. Infect. Genet. Evol. 104181 (2020) doi:10.1016/j.meegid.2020.104181." />
<meta name="citation_reference" content="Wong, E. H. M., Smith, D. K., Rabadan, R., Peiris, M. &amp; Poon, L. L. M. Codon usage bias and the evolution of influenza A viruses. Codon Usage Biases of Influenza Virus. BMC Evol. Biol. 10, 253 (2010)." />
<meta name="citation_reference" content="Pan, L. et al. Clinical characteristics of COVID-19 patients with digestive symptoms in Hubei, China: a descriptive, cross-sectional, multicenter study. 25." />
<meta name="citation_reference" content="Zhou, Z. et al. Effect of gastrointestinal symptoms on patients infected with COVID-19. Gastroenterology 0, (2020)." />
<meta name="citation_reference" content="Li, Y.-C., Bai, W.-Z. &amp; Hashikawa, T. The neuroinvasive potential of SARS-CoV2 may play a role in the respiratory failure of COVID-19 patients. J. Med. Virol. n/a,." />
<meta name="citation_reference" content="Mao, L. et al. Neurological Manifestations of Hospitalized Patients with COVID-19 in Wuhan, China: a retrospective case series study. medRxiv 2020.02.22.20026500 (2020) doi:10.1101/2020.02.22.20026500." />
<meta name="citation_reference" content="Andersen, K. G., Rambaut, A., Lipkin, W. I., Holmes, E. C. &amp; Garry, R. F. The proximal origin of SARS-CoV-2. Nat. Med. 1–3 (2020) doi:10.1038/s41591-020-0820-9." />
<meta name="citation_reference" content="Chen, F. et al. Dissimilation of synonymous codon usage bias in virus-host coevolution due to translational selection. Nat. Ecol. Evol. (2020) doi:10.1038/s41559-020-1124-7." />
<meta name="citation_reference" content="Pavon-Eternod, M. et al. Vaccinia and influenza A viruses select rather than adjust tRNAs to optimize translation. Nucleic Acids Res. 41, 1914–1921 (2013)." />
<meta name="citation_reference" content="Mioduser, O., Goz, E. &amp; Tuller, T. Significant differences in terms of codon usage bias between bacteriophage early and late genes: a comparative genomics analysis. Bmc Genomics 18, 866 (2017)." />
<meta name="citation_reference" content="Walker, P. J. et al. Changes to virus taxonomy and the International Code of Virus Classification and Nomenclature ratified by the International Committee on Taxonomy of Viruses (2019). Arch. Virol. 164, 2417–2429 (2019)." />
<meta name="citation_reference" content="Alexaki, A. et al. Codon and Codon-Pair Usage Tables (CoCoPUTs): Facilitating Genetic Variation Analyses and Recombinant Gene Design. J. Mol. Biol. 431, 2434–2441 (2019)." />
<meta name="citation_reference" content="Athey, J. et al. A new and updated resource for codon usage tables. BMC Bioinformatics 18, 391 (2017)." />
<meta name="citation_reference" content="Chan, P. P. &amp; Lowe, T. M. tRNAscan-SE: Searching for tRNA Genes in Genomic Sequences. Methods Mol. Biol. Clifton NJ 1962, 1–14 (2019)." />
<meta name="citation_reference" content="Hoffmann, A. et al. Accurate mapping of tRNA reads. Bioinformatics 34, 1116–1124 (2018)." />
<meta name="citation_reference" content="Hoffmann, S. et al. Fast mapping of short sequences with mismatches, insertions and deletions using index structures. PLoS Comput. Biol. 5, e1000502 (2009)." />
<meta name="citation_reference" content="McKenna, A. et al. The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. Genome Res. 20, 1297–1303 (2010)." />
<meta name="citation_reference" content="dos Reis, M., Wernisch, L. &amp; Savva, R. Unexpected correlations between gene expression and codon usage bias from microarray data for the whole Escherichia coli K-12 genome. Nucleic Acids Res. 31, 6976–6985 (2003)." />
<meta name="citation_reference" content="dos Reis, M., Savva, R. &amp; Wernisch, L. Solving the riddle of codon usage preferences: a test for translational selection. Nucleic Acids Res. 32, 5036–5044 (2004)." />
<meta name="citation_reference" content="Frumkin, I. et al. Codon usage of highly expressed genes affects proteome-wide translation efficiency. Proc. Natl. Acad. Sci. U. S. A. 115, E4940–E4949 (2018)." />
<meta name="citation_reference" content="Gingold, H., Dahan, O. &amp; Pilpel, Y. Dynamic changes in translational efficiency are deduced from codon usage of the transcriptome. Nucleic Acids Res. 40, 10053–10063 (2012)." />
<meta name="citation_reference" content="Pechmann, S. &amp; Frydman, J. Evolutionary conservation of codon optimality reveals hidden signatures of cotranslational folding. Nat. Struct. Mol. Biol. 20, 237–243 (2013)." />
<meta name="citation_reference" content="Zhao, Q. &amp; Fränti, P. WB-index: A sum-of-squares based index for cluster validity. Data Knowl. Eng. 92, 77–89 (2014)." />
<meta name="citation_reference" content="Dunn, J. C. Well-Separated Clusters and Optimal Fuzzy Partitions. J. Cybern. 4, 95–104 (1974)." />
<meta name="citation_reference" content="Al-Zoubi, M. B. &amp; Raw, M. An Efficient Approach for Computing Silhouette Coefficients. J. Comput. Sci. 4, (2008)." />
<meta name="citation_reference" content="Pedregosa, F. et al. Scikit-learn: Machine Learning in Python. J. Mach. Learn. Res. 12, 2825–2830 (2011)." />
<meta name="citation_reference" content="Subramanian, A. et al. Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles. Proc. Natl. Acad. Sci. U. S. A. 102, 15545–15550 (2005)." />
<meta name="twitter:title" content="Translational adaptation of human viruses to the tissues they infect" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Viruses need to hijack the translational machinery of the host cell for a productive infection to happen. However, given the dynamic landscape of tRNA pools among tissues, it is unclear whether different viruses infecting different tissues have adapted their codon usage toward their tropism. Here, we collect the coding sequences of over 500 human-infecting viruses and determine that tropism explains changes in codon usage. Using an in silico model of translational efficiency, we validate the correspondence of the viral codon usage with the translational machinery of their tropism. In particular, we propose that the improved translational adaptation to the upper respiratory airways of the pandemic agent SARS-CoV-2 coronavirus could enhance its transmissibility. Furthermore, this correspondence is specifically defined in early viral proteins, as upon infection cells undergo reprogramming of tRNA pools that favors the translation of late counterparts." />
<meta name="og-title" property="og:title" content="Translational adaptation of human viruses to the tissues they infect" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.04.06.027557v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Viruses need to hijack the translational machinery of the host cell for a productive infection to happen. However, given the dynamic landscape of tRNA pools among tissues, it is unclear whether different viruses infecting different tissues have adapted their codon usage toward their tropism. Here, we collect the coding sequences of over 500 human-infecting viruses and determine that tropism explains changes in codon usage. Using an in silico model of translational efficiency, we validate the correspondence of the viral codon usage with the translational machinery of their tropism. In particular, we propose that the improved translational adaptation to the upper respiratory airways of the pandemic agent SARS-CoV-2 coronavirus could enhance its transmissibility. Furthermore, this correspondence is specifically defined in early viral proteins, as upon infection cells undergo reprogramming of tRNA pools that favors the translation of late counterparts." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-04-07" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.04.06.027557v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.04.06.027557v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.04.06.027557v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.04.06.027557v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1232152" />
    <title>Translational adaptation of human viruses to the tissues they infect | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Lw1gyD48-NSe1DWC7bgHrA7RK0jg1sZ4xRAfUHk2nnw","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1232152","apath":"\/biorxiv\/early\/2020\/04\/07\/2020.04.06.027557.atom","pisa":"biorxiv;2020.04.06.027557v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.04.06.027557v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.04.06.027557v1","title":"Translational adaptation of human viruses to the tissues they infect","identifier":"node\/1232152"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00223uWuktnFqWuHy6Joag33J9lMfqywfOWETkjadLDBkzwHBtWjBYDyx6rz1L5AJb8uVZZhpNOZlo+7IzNAspA0I0uXLoTn0K3AIPOQq5uaY4Ph4FBzhpfRemIbC4U+vLtUies\\\\\\\/tqj\\\\\\\/LAjZpzEtVnmfvF1gXja+d8aGIuZm2TRZp+cpelqavAiX72Ke4bghSjv7jeIDQSaktGV9AlZaZcBai2Rzjq+hBOY5+b9qtERNaEWIzATZU+fLWg40VTmnp862mJaev7Rn8oC5ELdwnwdDxkKx1jk89MBGtxC+jo6Q9p1ZMmjBDEvMa3b6MWsu+fFtscnCD1SLTegrjxesHVbWOhlu0+dcpGvy4ZAhFROtJXUUKj45SxbcwaR0i9IuePnK66dQWlVhW10TK8pRC029tyGQVXV+YCRH9\\\\\\\/kOqibaXQQIXmJXt1Tcp3PWrRDT4wKakXyB+1OyXKBbsScuucAgTLoEsatVdgxvPZSzlOZPuQXcXGj5Nd14awBkQlmsdTwqbj9cPYt53QsfhPhG+vlRCvDNg\\\\\\\/3oNf7+LeY9DqccWO1psvi+JtqBDu65P4PO9F5XR2mn7qy6S7AdmmqfLqc\\\\\\\/WUf3ESeEJDLhBg6Bqza\\\\\\\/PDbsES5FNOvu8Z1+eDGOY885E1UQTe8uba5Ba7psuInUUHerv3W46+kIYvmBNpJtEM7qR1f3KHH6L8oNjLXH6YugQRCD6OscA7BGmDS1sjBvFn61mOX8vhpmLY1AdEsmu9swjZIBSQK1vCkTzUSZRDd\\\\\\\/QwCutrvaUHljKMSQL0qmlx3Ib5dDK5ig+VpdgoSoYClmylvobubSG51K+EAr4jGOucJpC82qN+u+BT6Mk31nS3nC\\\\\\\/7Vde5k1C1Bo07qDcpZq+NCoWZzyFGP\\\\\\\/GSu0uetXO7K6f\\\\\\\/I\\\\\\\/K79DxlMBeB2QwwHpZNKdOfwUrKgF1kkcceCKUK6vFbnRdTGHMNxDfL8HdVY9jw04uXDzA33hhbaCmq4exBu5dEAnLZDegHhE7ynyRe0+PIGaQAsy5qINY1qBCm2vqPhx3WFtvUSy\\\\\\\/I+vIDDVO9XoEH6baALyBzuSaqgpXzyZf1nDx0PuwkQBWLLq7kBwnuvfGKufvKBtJxXaIbysbN7U+CHeYNiJVWdHWlcK7U\\\\\\\/5ugG6QKitKCn3X6x3kcptaB+6\\\\\\\/jxIRAm8yqK4LjqnZ\\\\\\\/5LPMGoXLeN7aVp7Qq0wBuw4p20c3v66etgkO01W3P\\\\\\\/CWE1ZLZHgHyNu6\\\\\\\/fQLG5p32fXyfrvwmEkP6Z1mtdxuBhzBTpP+0uM5lspiCxZJ9foOOE6VtmhFFxhkbstXyD+6Is4UrFGW714JLj3tUbwBPzOUl8mcllhUm8e6fD\\\u0022,\\\u0022iv\\\u0022:\\\u00221yRSE7Rmf9ga2GZf6hZe+Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00222f3a07503d1d00fe5f863b1d724ae9c0ee7ba84025bf6bb0e1179f3e3e5ebdc2\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.04.06.027557v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1232152 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.04.06.027557v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-QdqwaColmaC50iLP0Yg2157iqNNj1kteLBlIMjdUA34" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1232152" id="node1232152" data-pisa="biorxiv;2020.04.06.027557v1" data-pisa-master="biorxiv;2020.04.06.027557" data-apath="/biorxiv/early/2020/04/07/2020.04.06.027557.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Translational adaptation of human viruses to the tissues they infect</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-8633-499X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Xavier</span> <span class="nlm-surname">Hernandez-Alias</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-7503-6364" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Martin H.</span> <span class="nlm-surname">Schaefer</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-5276-1392" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Luis</span> <span class="nlm-surname">Serrano</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.04.06.027557 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1232152" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Xavier Hernandez-Alias </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Centre for Genomic Regulation (CRG), The Barcelona Institute of Science and Technology</span>, Dr. Aiguader 88, Barcelona, 08003, <span class='nlm-country'>Spain</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xavier%2BHernandez-Alias%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hernandez-Alias%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXavier%2BHernandez-Alias%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8633-499X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xavier Hernandez-Alias</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Martin H. Schaefer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>IEO European Institute of Oncology IRCCS, Department of Experimental Oncology</span>, Via Adamello 16, Milan, 20139, <span class='nlm-country'>Italy</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BH.%2BSchaefer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schaefer%20MH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMartin%2BH.%2BSchaefer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7503-6364" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Martin H. Schaefer</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:martin.schaefer@ieo.it" class="" data-icon-position="" data-hide-link-title="0">martin.schaefer@ieo.it</a>
<a href="mailto:luis.serrano@crg.eu" class="" data-icon-position="" data-hide-link-title="0">luis.serrano@crg.eu</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Luis Serrano </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Centre for Genomic Regulation (CRG), The Barcelona Institute of Science and Technology</span>, Dr. Aiguader 88, Barcelona, 08003, <span class='nlm-country'>Spain</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Universitat Pompeu Fabra (UPF)</span>, Barcelona, 08002, <span class='nlm-country'>Spain</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>ICREA</span>, Pg. Lluís Companys 23, Barcelona, 08010, <span class='nlm-country'>Spain</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luis%2BSerrano%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Serrano%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALuis%2BSerrano%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5276-1392" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Luis Serrano</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:martin.schaefer@ieo.it" class="" data-icon-position="" data-hide-link-title="0">martin.schaefer@ieo.it</a>
<a href="mailto:luis.serrano@crg.eu" class="" data-icon-position="" data-hide-link-title="0">luis.serrano@crg.eu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.04.06.027557v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1232152" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1232152/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.06.027557v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1232152" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1232152/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.06.027557v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1232152" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1232152/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.06.027557v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1232152" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1232152/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.06.027557v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1232152" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1232152/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.04.06.027557v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1232152" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1232152/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">ABSTRACT</h2><p id="p-2">Viruses need to hijack the translational machinery of the host cell for a productive infection to happen. However, given the dynamic landscape of tRNA pools among tissues, it is unclear whether different viruses infecting different tissues have adapted their codon usage toward their tropism. Here, we collect the coding sequences of over 500 human-infecting viruses and determine that tropism explains changes in codon usage. Using an <em>in silico</em> model of translational efficiency, we validate the correspondence of the viral codon usage with the translational machinery of their tropism. In particular, we propose that the improved translational adaptation to the upper respiratory airways of the pandemic agent SARS-CoV-2 coronavirus could enhance its transmissibility. Furthermore, this correspondence is specifically defined in early viral proteins, as upon infection cells undergo reprogramming of tRNA pools that favors the translation of late counterparts.</p></div><div class="section" id="sec-1"><h2 class="">INTRODUCTION</h2><p id="p-3">Given the degeneracy of the genetic code, multiple 3-letter combinations of nucleotides can code for the same amino acid. Such synonymous codons are nevertheless not uniformly distributed along the genomes and can significantly deviate between organisms<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a></sup>. Evolutionary forces that explain the existence of the so-called codon bias include (1) a mutation pressure for a certain GC base composition depending on the species and chromosomal location, and (2) the translational selection for codons corresponding to highly expressed tRNA isoacceptors<sup><a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>–<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a></sup>.</p><p id="p-4">Viruses strongly depend on the translational machinery of the host for the expression of their own proteins and, ultimately, their replication. For instance, given the small size of most viral genomes, no or very few tRNA genes are generally autonomously encoded<sup><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a></sup>. In terms of codon usage, it has indeed been shown that bacteriophages are specifically adapted to their microbial hosts<sup><a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>,<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a></sup>. This information has been applied in the prediction of viral hosts from metagenomics data<sup><a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>,<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a></sup>. The codon usage of human-infecting viruses is similarly adapted to the host<sup><a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>,<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a></sup>, and actually the concept of codon deoptimization has been applied in the design of attenuated vaccines<sup><a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a></sup>.</p><p id="p-5">Although translational selection has long been under debate in human<sup><a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></sup>, recent studies indicate that different tissues and conditions showcase distinct tRNA expression profiles, leading to changes in their respective translational efficiency<sup><a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>,<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></sup>. In agreement with this observation, the codon usage of papillomavirus capsid proteins is adapted to the tRNAs of differentiated keratinocytes, where their translation becomes specifically efficient<sup><a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>,<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a></sup>. In addition, upon HIV-1 infection, the host tRNA pool is reprogrammed to favor translation of late viral genes<sup><a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></sup>, a phenomenon that is indeed exploited by host antiviral mechanisms<sup><a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a></sup>. Furthermore, some viruses with a specific tissue tropism resemble the codon bias of highly expressed proteins of their respective infecting tissues<sup><a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a></sup>. Nevertheless, despite the few aforementioned studies, a high-throughput analysis of the translational selection of viral genomes to their tissue tropism has been heretofore hindered by the absence of tissue-wide tRNA expression data.</p><p id="p-6">Here, we systematically analyze the relative codon usage landscape of over 500 humaninfecting viruses together with the recently published tRNA expression profiles of human tissues<sup><a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">2</a></sup>. Among other viral annotated features, including phylogeny and Baltimore classification, their tissue tropism explains more variance in codon usage than the other tested features. In consequence, tropism corresponds with codon optimization patterns that can be associated with tissue-specific profiles of tRNA-based translation efficiencies.</p><p id="p-7">The SARS-CoV-2 coronavirus constitutes the etiologic agent of the biggest pandemic of the 21<sup>st</sup> century, causing the COVID-19 pneumonia-like disease. Originated in Wuhan (China), the virus has already caused 40,598 deaths as of 1<sup>st</sup> April 2020<sup><a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a></sup>. In this context, recent scientific efforts suggest that the novel coronavirus strain has evolved to preferentially infect nasal goblet and ciliated cells as well as the type-2 alveolar epithelial cells<sup><a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>,<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a></sup>. However, some non-respiratory-related symptoms such as loss of smell and taste hint as well to other tropism potential outside the respiratory tract<sup><a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>,<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a></sup>. In terms of translational efficiency, we here discover that the coronavirus proteome is especially adapted not only to upper respiratory airways and alveoli, but also to the gastrointestinal tract and brain.</p><p id="p-8">Further, by studying the tissue-adaptation among the viral proteome, we also determine that early replication-related proteins are more translationally-adapted than the late structural counterparts. Overall, we observe a tropism-specific adaptation of the viral proteome to the tRNA profiles of their target tissues, which opens the door to the development of tissue-specific codon-deoptimized vaccines and targeted antiviral therapies.</p></div><div class="section" id="sec-2"><h2 class="">RESULTS</h2><div id="sec-3" class="subsection"><h3>Tropism corresponds with differences in Relative Codon Usage of human-infecting viruses</h3><p id="p-9">Publicly available genomic data comprised a total of 564 human-infecting viruses, distributed among 33 families and covering all seven Baltimore categories (Sup. Table 1). Across this diversity, six main viral tropisms were defined for 182 viruses based on the ViralZone curated database<sup><a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a></sup>: neurons, immune cells, respiratory tract, hepatocytes, intestine, and epithelial cells (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1A</a>). Their corresponding coding sequences constituted a total of 4935 viral proteins (Sup. Table 1), for which we determined the Relative Codon Usage (RCU, i.e. the contribution of each synonymous codon to the amino acid it encodes, see Methods).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Tropism corresponds with differences in relative codon usage of humaninfecting viruses. (A) A total of 564 viruses was distributed among 33 families and covered all seven Baltimore groups. From there, 182 viruses were classified in six general tropisms based on ViralZone annotations25. (B) Three internal clustering indexes were computed to assess the validity of each viral classification in terms of their RCU. Good cluster performances lead to low WB indexes, but to high Silhouette and Dunn values (as shown in the color code). (C) Linear Discriminant Analysis of the RCU of the 182 tropism-defined viruses. In brackets, the percentage of variance explained by each of the components." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1251490112" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Tropism corresponds with differences in relative codon usage of humaninfecting viruses.&lt;/span&gt; (A) A total of 564 viruses was distributed among 33 families and covered all seven Baltimore groups. From there, 182 viruses were classified in six general tropisms based on ViralZone annotations&lt;sup&gt;25&lt;/sup&gt;. (B) Three internal clustering indexes were computed to assess the validity of each viral classification in terms of their RCU. Good cluster performances lead to low WB indexes, but to high Silhouette and Dunn values (as shown in the color code). (C) Linear Discriminant Analysis of the RCU of the 182 tropism-defined viruses. In brackets, the percentage of variance explained by each of the components.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557/F1.medium.gif" width="440" height="306"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557/F1.medium.gif" width="440" height="306"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">Tropism corresponds with differences in relative codon usage of humaninfecting viruses.</span><p id="p-10" class="first-child">(A) A total of 564 viruses was distributed among 33 families and covered all seven Baltimore groups. From there, 182 viruses were classified in six general tropisms based on ViralZone annotations<sup><a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a></sup>. (B) Three internal clustering indexes were computed to assess the validity of each viral classification in terms of their RCU. Good cluster performances lead to low WB indexes, but to high Silhouette and Dunn values (as shown in the color code). (C) Linear Discriminant Analysis of the RCU of the 182 tropism-defined viruses. In brackets, the percentage of variance explained by each of the components.</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">In order to understand the main factors driving differences between viral RCU, we used three internal clustering indexes that assess how similar each virus is to a certain group compared to other groups. Taking the average RCU over each of the 564 viral proteomes, we applied this framework to assess the grouping performance of six different viral features: tropism, type of genetic material, family, genus, Baltimore category, and a sequence-based classification by <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Aiewsakun and Simmonds (2018)</a>. In such analysis, the tropism leads the best classification of viral RCUs, followed by the viral genetic type (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1B</a>). On the other hand, classical and sequence-based phylogenetic classifications show poor clustering performances.</p><p id="p-12">Given the impact of viral tropism on the RCU, we sought to determine the main codon differences between the six defined target tissues. By using a linear discriminant analysis (see Methods), the 182 tropism-defined viruses were classified in six clear clusters, regardless of other factors such as the phylogenetic lineage (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1C</a>). In consequence, we observe that specific codon usage profiles are associated with the tissue tropism of human-infecting viruses.</p></div><div id="sec-4" class="subsection"><h3>Viruses are adapted to the tRNA-based translational efficiencies of their target tissues</h3><p id="p-13">Based on the RCU differences between viruses with distinct tropism, we hypothesize that distinct tissues impose selection towards a certain set of translationally-efficient codons. However, a validation for this hypothesis requires the accurate quantification of tissue-specific tRNA profiles, which has been hitherto missing. With the advent of such high-throughput expression data<sup><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>,<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a></sup>, here we retrieved the previously-published Supply-to-Demand Adaptation (SDA) estimate for translational efficiency<sup><a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">2</a></sup>, which computes the balance between the supply (i.e., the anticodon tRNA abundances) and demand (i.e., the codons expressed in mRNAs) of each codon (see Methods).</p><p id="p-14">Using a total of 620 healthy samples from The Cancer Genome Atlas (TCGA) dataset<sup><a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">2</a></sup>, we first computed the SDA of all viral-protein-coding sequences based on the SDA weights of their constituent codons. Therefore, taking the average of all healthy samples across each of the 23 TCGA cancer types (Sup. Table 2), we determined the estimated translational efficiencies of viral proteins in different human tissues (Sup. Table 3).</p><p id="p-15">Next, from the perspective of the translational selection hypothesis, we would expect that viral proteins are translationally adapted to their target tissues. In consequence, we tried to test our hypothesis using a completely blind and unbiased random forest classifier, which applies machine learning in order to predict the tropism of each viral protein based on the SDA to different tissues (see Methods). The resulting performance of the models, based on the Area Under the Curve (AUC) of their Receiver Operating Characteristic (ROC) curves, ranges between 0.74-0.91 (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2A</a>), clearly higher than the no-skill model of 0.5. Similar results are also obtained from complementary prediction performance metrics such as Precision-Recall curves (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2A</a>). These results indicate that our machine learning model is able to predict the tropism of a viral protein based on its SDA to tissues with high accuracy. In concordance, a linear discriminant analysis of the average SDA of each virus across tissues can similarly separate different clusters of viral tropism based on their translational efficiencies (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Extended Data Fig. 1</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Viruses are adapted to the tRNA-based translational efficiencies of their target tissues. (A) Receiver Operating Characteristic (ROC) and Precision-Recall (PR) curves of a Random Forest Classifier, in which the average Supply-to-Demand Adaptation of viral proteins to each of the 23 TCGA tissues is used to predict their corresponding viral tropism (see Methods). The area under the curves (AUC) &#xB1; SD summarizes the performance of the model. (B) Relative feature weights of each of the 23 TCGA tissues for each of the six tropisms. The dendrograms show a hierarchical clustering among tissues (left) and among tropisms (top). Refer to Sup. Table 2 for full TCGA cancer type names. See also Extended Data Fig. 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1251490112" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Viruses are adapted to the tRNA-based translational efficiencies of their target tissues.&lt;/span&gt; (A) Receiver Operating Characteristic (ROC) and Precision-Recall (PR) curves of a Random Forest Classifier, in which the average Supply-to-Demand Adaptation of viral proteins to each of the 23 TCGA tissues is used to predict their corresponding viral tropism (see Methods). The area under the curves (AUC) &#xB1; SD summarizes the performance of the model. (B) Relative feature weights of each of the 23 TCGA tissues for each of the six tropisms. The dendrograms show a hierarchical clustering among tissues (left) and among tropisms (top). Refer to Sup. Table 2 for full TCGA cancer type names. See also Extended Data Fig. 1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557/F2.medium.gif" width="440" height="348"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557/F2.medium.gif" width="440" height="348"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Viruses are adapted to the tRNA-based translational efficiencies of their target tissues.</span><p id="p-16" class="first-child">(A) Receiver Operating Characteristic (ROC) and Precision-Recall (PR) curves of a Random Forest Classifier, in which the average Supply-to-Demand Adaptation of viral proteins to each of the 23 TCGA tissues is used to predict their corresponding viral tropism (see Methods). The area under the curves (AUC) ± SD summarizes the performance of the model. (B) Relative feature weights of each of the 23 TCGA tissues for each of the six tropisms. The dendrograms show a hierarchical clustering among tissues (left) and among tropisms (top). Refer to Sup. Table 2 for full TCGA cancer type names. See also <a id="xref-fig-5-2" class="xref-fig" href="#F5">Extended Data Fig. 1</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">In an attempt to understand which tissues are the most predictive in identifying the viral tropism of proteins, we analyzed the relative feature importance within each random forest classifier, which measures the contribution of each tissue SDA in the decision trees (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2B</a>). The main observation is that no single tissue alone is able to discriminate against the specific tropism, since all feature importances lie below 0.10. However, it is also clear that translational adaptation to bile duct (CHOL, for healthy samples of cholangiocarcinoma) is a recurrent discriminant feature, while other tissues are specifically important for just one or few tropisms, such as rectum (READ, for healthy samples of rectum adenocarcinoma) in predicting intestinal viruses. In any case, the directionality of these features cannot be established.</p><p id="p-18">Overall, as tropism of proteins can be predicted from their translational adaptation to tissues, these results indicate that viral proteomes are specifically adapted to certain tRNA-based translational efficiencies. In consequence, and complementary to the observations of mutational pressure driving viral codon bias<sup><a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>,<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>,<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a></sup>, we describe the basis for a potential tissuespecific translational selection of the viral codon usage.</p></div><div id="sec-5" class="subsection"><h3>SARS-CoV-2 is translationally adapted to upper respiratory airways and alveoli</h3><p id="p-19">As our systematic analysis suggests that the codon usage of viruses tend to be adapted to the tissue they infect, we selected the novel coronavirus SARS-CoV-2 and other related respiratory viruses to further explore their translational adaptation profile over tissues.</p><p id="p-20">We initially reconstructed tRNA expression profiles along the respiratory tract making use of the spatial information associated with healthy TCGA samples from head and neck squamous cell carcinoma (HNSC), lung squamous cell carcinoma (LUSC) and lung adenocarcinoma (LUAD) (Sup. Table 4). We then computed the SDA of viral proteins from the three pandemic coronaviruses of the last two decades SARS-CoV<sup><a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a></sup>, MERS-CoV<sup><a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></sup>, and SARS-CoV-2<sup><a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a></sup>, as well as the common flu-causing influenza A virus (H1N1) along the respiratory tract. We find that the new coronavirus strain SARS-CoV-2 is the most translationally adapted virus across all tissues, especially in comparison with MERS-CoV and influenza A virus (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3A</a>). Further, the novel coronavirus is highly adapted throughout the whole respiratory tract, and specifically the upper respiratory airways and the alveolar region in the lung periphery are the most efficient tissues.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SARS-CoV-2 is translationally adapted to upper respiratory airways and alveoli. (A) Using the codon adaptation weights of all TCGA samples along the respiratory tract (HNSC, LUAD, LUSC), we compute the SDA of proteins from coronaviruses (and influenza A virus. Center values within the violin plot represent the median. Using a two-tailed Wilcoxon rank-sum test, only significant differences between viruses are shown: * (p " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1251490112" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;SARS-CoV-2 is translationally adapted to upper respiratory airways and alveoli.&lt;/span&gt; (A) Using the codon adaptation weights of all TCGA samples along the respiratory tract (HNSC, LUAD, LUSC), we compute the SDA of proteins from coronaviruses (and influenza A virus. Center values within the violin plot represent the median. Using a two-tailed Wilcoxon rank-sum test, only significant differences between viruses are shown: * (p &lt;= 0.05), ** (p &lt;= 0.01), *** (p &lt;= 0.001), **** (p &lt;= 0.0001). (B) Average RSEM expression of the corresponding entry receptors of the viruses: MERS-CoV binds to DPP4&lt;sup&gt;35&lt;/sup&gt;, SARS-CoV and SARS-CoV-2 bind to ACE2 or BSG with priming by the protease TMPRSS2&lt;sup&gt;36,37&lt;/sup&gt;, influenza A virus binds to sialic acids, which are synthesised by the enzymes ST3GAL4 and ST6GAL1&lt;sup&gt;34&lt;/sup&gt;. RSEM receptor expression is re-scaled between 0 and 1 among all tissues, being 1 the tissue with highest expression. Refer to Sup. Table 3 for full TCGA cancer type names. (C) Schematic representation of the potential of infection of each virus based on the coincidence of both translational adaptation and entry receptor expression. See also Extended Data Fig. 2.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557/F3.medium.gif" width="440" height="393"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557/F3.medium.gif" width="440" height="393"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">SARS-CoV-2 is translationally adapted to upper respiratory airways and alveoli.</span><p id="p-21" class="first-child">(A) Using the codon adaptation weights of all TCGA samples along the respiratory tract (HNSC, LUAD, LUSC), we compute the SDA of proteins from coronaviruses (and influenza A virus. Center values within the violin plot represent the median. Using a two-tailed Wilcoxon rank-sum test, only significant differences between viruses are shown: * (p &lt;= 0.05), ** (p &lt;= 0.01), *** (p &lt;= 0.001), **** (p &lt;= 0.0001). (B) Average RSEM expression of the corresponding entry receptors of the viruses: MERS-CoV binds to DPP4<sup><a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup>, SARS-CoV and SARS-CoV-2 bind to ACE2 or BSG with priming by the protease TMPRSS2<sup><a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>,<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a></sup>, influenza A virus binds to sialic acids, which are synthesised by the enzymes ST3GAL4 and ST6GAL1<sup><a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a></sup>. RSEM receptor expression is re-scaled between 0 and 1 among all tissues, being 1 the tissue with highest expression. Refer to Sup. Table 3 for full TCGA cancer type names. (C) Schematic representation of the potential of infection of each virus based on the coincidence of both translational adaptation and entry receptor expression. See also <a id="xref-fig-6-1" class="xref-fig" href="#F6">Extended Data Fig. 2</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">Moreover, given that the tropisms not only depend on the translational adaptation to the host, but also on the expression of the required entry receptors, we measure the respective receptors of each virus. Influenza A viruses bind to α(2,6)-linked and α(2,3)-linked sialic acids, which are synthesized by the enzymes ST6GAL1 and ST3GAL4, respectively<sup><a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a></sup>. While their expression is relatively uniform along the airways, the MERS-CoV uses the parenchymaspecific receptor DPP4 (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3B</a>)<sup><a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">35</a></sup>. In both cases, the expression of their entry receptor coincides with an optimal translational adaptation in the lower respiratory tract. On the other hand, both the SARS-CoV and SARS-CoV-2 strains bind to the ACE2 protein and require the proteolytic priming of the viral spike protein by TMPRSS2<sup><a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">36</a></sup>, although the receptor BSG/CD147 has also been proposed<sup><a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">37</a></sup>. In all cases, their expression throughout the airways (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3B</a>) alongside their efficient translational adaptation defines the upper respiratory tract and the alveoli as their particular optimal tropism (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3C</a>). This is in agreement with recent single-cell transcriptomic studies reporting the expression of ACE2 in the nasal goblet and ciliated cells as well as the type-2 alveolar epithelial cells<sup><a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>,<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a></sup>.</p><p id="p-23">Apart from the clear viral tropism of SARS-CoV-2 to the respiratory tract, recent studies propose that their tropism can expand to other tissues such as the digestive system or the brain<sup><a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>,<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a></sup>. For this reason, we also extended our translational analysis to all the 23 tissues of the TCGA dataset (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Extended Data Fig. 2A</a>), together with the expression of the corresponding receptors (Sup. Table 4). In agreement with the clinical findings<sup><a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">23</a>,<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">24</a></sup>, the gastrointestinal tract emerges as the most translationally adapted tissue, followed by the other epithelial-like tissues and the brain. Therefore, in terms of translational efficiency, the novel SARS-CoV-2 is widely adapted across tissues.</p><p id="p-24">In an attempt to elucidate the translational selection that could have benefitted the evolution of the new coronavirus, we also compared the SDA adaptation of SARS-CoV-2 to those of close phylogenetic strains: the human-infecting SARS-CoV and the bat coronavirus RatG13, with 79.6% and 96.2% of sequence identity, respectively<sup><a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">33</a></sup>. We observe that all four main proteins of the new strain have optimized their codon usage with regard to the previous 2003-pandemic SARS-CoV (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Extended Data Fig. 2B</a>). On the other hand, even though the ratio of synonymous versus nonsynonymous mutation rate of the bat coronavirus compared to SARS-CoV-2 is exceptionally high<sup><a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a></sup>, their proteins are very similarly adapted to most human tissues.</p><p id="p-25">In short, we report that the new pandemic SARS-CoV-2 coronavirus has evolved to significantly increase its translational adaptation to the human tissues. While it might not be sufficient on its own, the coincidence of both translational efficiency and entry receptor expression supports an optimal tropism of the alveoli but also the upper respiratory tract (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig. 3C</a>), which has recently been related to the higher transmissibility of the novel strain<sup><a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">21</a></sup>.</p></div><div id="sec-6" class="subsection"><h3>Early viral proteins are better adapted and translationally deoptimized throughout the process of infection</h3><p id="p-26">Given the tropism-specific adaptation of viral RCU towards the translational machinery of tissues, we wondered whether certain genomic subsets were specifically adapted to the tissue of infection. In particular, we speculated that early replication-related proteins would further benefit from such adaptation than late structural proteins, since once the virus takes control of the cell it could change its tRNA expression program<sup><a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a></sup>.</p><p id="p-27">To estimate the adaptation of each protein to the tRNA-based codon efficiencies of each tissue, we computed their SDA<sup><a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">2</a></sup> (Sup. Table 5). For that purpose, we matched each virus to the tRNAs of their tissues of infection (Sup. Table 5). In concordance with our hypothesis, based on current viral annotations (VOGdb, <a href="http://vogdb.org">vogdb.org</a>), we observed a small but highly significant shift in SDA between structural and replication proteins in all but hepatocyteinfecting viruses (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4A</a>). However, the proteomes of this latter group are poorly annotated at the viral orthology database VOGdb, and separation between classes is often hindered by the polyprotein structure of their genomes<sup><a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a></sup>. Similarly, we performed a Gene Set Enrichment Analysis to identify which Virus Orthologous Groups (VOGs) were enriched in high-SDA or low-SDA proteins (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4B</a>). As determined by current annotations<sup><a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">39</a></sup>, top-VOGs mostly contained replication-related early proteins, whereas bottom-VOGs had structural late functions.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Early viral proteins are better adapted and translationally deoptimized throughout the process of infection. (A) Average Supply-to-Demand Adaptation of replication (Xr) and structural (Xr) proteins of a total of 104 annotated tropism-specific viruses, respectively matched to 461 samples of their tissues of infection (Sup. Table 5). Boxes expand from the first to the third quartile, with the center values indicating the median. The whiskers define a confidence interval of median &#xB1; 1.58*IQR/sqrt(n). Statistical significance is determined by paired (structural against replication proteins of each virus) and two-tailed Wilcoxon rank-sum test. (B) Top positive and negative Virus Orthologous Groups upon Gene Set Enrichment Analysis of the SDA of all proteins of tropism-specific viruses (Sup. Table 5). Based on their annotations, proteins groups are colored based on their early/replication or late/structural function39. (C) Differences of the relative tRNA Adaptation Index (see Methods) between mock and effective viral infections in different cell lines. Multiple time points are specified when available. Proteins are allocated to different time expression classes based on current viral knowledge39 (Sup. Table 6). Center values within the violin plot represent the median. Using a two-tailed Wilcoxon rank-sum test, only significant differences are shown: * (p " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1251490112" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Early viral proteins are better adapted and translationally deoptimized throughout the process of infection.&lt;/span&gt; (A) Average Supply-to-Demand Adaptation of replication (Xr) and structural (Xr) proteins of a total of 104 annotated tropism-specific viruses, respectively matched to 461 samples of their tissues of infection (Sup. Table 5). Boxes expand from the first to the third quartile, with the center values indicating the median. The whiskers define a confidence interval of median &#xB1; 1.58*IQR/sqrt(n). Statistical significance is determined by paired (structural against replication proteins of each virus) and two-tailed Wilcoxon rank-sum test. (B) Top positive and negative Virus Orthologous Groups upon Gene Set Enrichment Analysis of the SDA of all proteins of tropism-specific viruses (Sup. Table 5). Based on their annotations, proteins groups are colored based on their early/replication or late/structural function&lt;sup&gt;39&lt;/sup&gt;. (C) Differences of the relative tRNA Adaptation Index (see Methods) between mock and effective viral infections in different cell lines. Multiple time points are specified when available. Proteins are allocated to different time expression classes based on current viral knowledge&lt;sup&gt;39&lt;/sup&gt; (Sup. Table 6). Center values within the violin plot represent the median. Using a two-tailed Wilcoxon rank-sum test, only significant differences are shown: * (p &lt;= 0.05), ** (p &lt;= 0.01), *** (p &lt;= 0.001), **** (p &lt;= 0.0001).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557/F4.medium.gif" width="433" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557/F4.medium.gif" width="433" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">Early viral proteins are better adapted and translationally deoptimized throughout the process of infection.</span><p id="p-28" class="first-child">(A) Average Supply-to-Demand Adaptation of replication (Xr) and structural (Xr) proteins of a total of 104 annotated tropism-specific viruses, respectively matched to 461 samples of their tissues of infection (Sup. Table 5). Boxes expand from the first to the third quartile, with the center values indicating the median. The whiskers define a confidence interval of median ± 1.58*IQR/sqrt(n). Statistical significance is determined by paired (structural against replication proteins of each virus) and two-tailed Wilcoxon rank-sum test. (B) Top positive and negative Virus Orthologous Groups upon Gene Set Enrichment Analysis of the SDA of all proteins of tropism-specific viruses (Sup. Table 5). Based on their annotations, proteins groups are colored based on their early/replication or late/structural function<sup><a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">39</a></sup>. (C) Differences of the relative tRNA Adaptation Index (see Methods) between mock and effective viral infections in different cell lines. Multiple time points are specified when available. Proteins are allocated to different time expression classes based on current viral knowledge<sup><a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">39</a></sup> (Sup. Table 6). Center values within the violin plot represent the median. Using a two-tailed Wilcoxon rank-sum test, only significant differences are shown: * (p &lt;= 0.05), ** (p &lt;= 0.01), *** (p &lt;= 0.001), **** (p &lt;= 0.0001).</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">Previous studies on the translational adaptation of the human immunodeficiency virus 1 suggested that the host tRNA pool is reprogramed upon viral infection in order to favor the expression of late genes<sup><a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">17</a></sup>. In this direction, we wanted to test whether this tRNA reprogramming is a general adaptive mechanism among viral species. Using three previously published small RNA-sequencing datasets of human cell lines upon viral infection<sup><a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>–<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a></sup>, we quantified the tRNA of mock and productive infections at different time points (Sup. Table 6). Therefore, for all datasets, we detected a general decrease in translational efficiency upon viral infection, with late viral proteins being more favored than early ones specifically at longer infection times (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4C</a>).</p><p id="p-30">Overall, we determine that the tropism-specific adaptation of viruses is specifically pronounced among early proteins. However, later in infection, the tRNA pool of the host cells is reprogrammed to favor the expression of late viral genes, reproducing previous findings of HIV1 infection and extending the mechanism to dsDNA-based viruses such as human cytomegaloviruses and human simplex viruses.</p></div></div><div class="section" id="sec-7"><h2 class="">DISCUSSION</h2><p id="p-31">Tropism is determined by an ensemble of different factors, including the mechanism of viral entry to the host, the immune responses to the infection, or the viral hijacking of the cellular machinery in the interest of replication and propagation. In this article, we study the latter by focusing on the translational adaptation of viral genomes to the host.</p><p id="p-32">While previous studies on the base composition and codon usage of both DNA and RNA viruses<sup><a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">11</a>,<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a></sup> have attributed most of the codon usage variability to the mutational pressure of viral genomes, our analysis proposes tropism as an important driving force. By systematically interrogating all human-infecting viruses, we uncover that tissue tropism explains changes in their codon usage more than other viral properties such as type or family. Therefore, as mutational pressure would act more similarly within phylogenetically closer species, such tropism-related differences in codon usage suggests that tissue-specific tRNA expression could be driving a translational selection on viral genomes.</p><p id="p-33">Although high-throughput sequencing of tRNAs has been only recently developed, cases of natural selection of codon usage towards the host have been previously proposed. For instance, codon usage of <em>Parvovirus</em> has been progressively adapted from dogs to cats after the host jump<sup><a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a></sup>. <em>Influenzaviruses</em> show a similar adaptation over time of viral isolation, deviating from the codon usage of avian hosts<sup><a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>,<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a></sup>. However, whether these progressive changes in codon usage over time are directly driven by translational selection has remained elusive. With the advent of tissue-wide datasets of tRNAs and their translational efficiencies<sup><a id="xref-ref-2-7" class="xref-bibr" href="#ref-2">2</a></sup>, we can now compute the Supply-to-Demand Adaptation (SDA) of all viral proteomes in different tissues. From there, we then created a random forest model that predicts with high accuracy the viral tropism of proteins based on their profile of adaptation to human tissues. In consequence, the tRNA-based adaptation profile of a protein is descriptive of their viral tropism, indicating that translational selection could indeed drive tropism differences of codon usage. It is important to remark that viruses could still have a good SDA to non-target tissues with similar tRNA expression patterns that are not infected because they are not exposed to the virus.</p><p id="p-34">Specifically, we therefore focus our analysis on the new SARS-CoV-2 coronavirus, causative of the current COVID-19 pandemic. As has been recently suggested based on the expression of their entry receptor<sup><a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">21</a>,<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">22</a></sup>, SARS-CoV-2 is also translationally adapted to the upper respiratory airways, as well as the lung parenchyma. This upper tract tropism has indeed been linked to the higher transmissibility of the strain<sup><a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">21</a></sup>. The novel coronavirus showcases the highest SDA among all studied coronaviruses, including lung but also other tissues such as the digestive system and brain. Interestingly, recent reports have shown that COVID-19 patients very frequently have problems in the digestive tract<sup><a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">24</a>,<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>,<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a></sup> as well as in neural tissue<sup><a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">23</a>,<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>,<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a></sup>.</p><p id="p-35">One major open question in the field still persists: how much natural selection before or after zoonosis shapes the current coronavirus<sup><a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a></sup>. On the one hand, viruses tend to gradually change their codon usage after a zoonotic jump<sup><a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">43</a>,<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">45</a></sup>. On the other hand, given the similarity of SARS-CoV-2 SDA with the phylogenetically closest bat coronavirus, it seems that a translational selection to increase SDA would have acted before the putative zoonosis from bats or other intermediate hosts. Furthermore, in agreement with the highest translational potential of SARS-CoV-2 in their target tissues, a recent model of viral tropism suggested that a tradeoff exists between the efficiency of viral translation and the translational load on the host, indicating that an improved codon usage can make the difference between symptomatic and natural hosts<sup><a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a></sup>.</p><p id="p-36">On the other hand, it is known that host tRNA pools undergo reprogramming upon viral infection of HIV-1, vaccinia virus, and influenza A virus<sup><a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">17</a>,<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a></sup>. In this context, differences in codon usage between early and late viral genes have been previously reported, but the directionality of such changes remained unclear<sup><a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>,<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a></sup>. Based on our concept of tissue-specific adaptation, we therefore validate that early replication-related proteins are better adapted to the tissue of infection. Upon infection, we then unveil that changes in tRNA abundance switch the adaptation towards the expression of late structural proteins, confirming a general trend that had previously only been observed in HIV-1<sup><a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">17</a></sup>.</p><p id="p-37">Overall, this is the first systematic analysis establishing a link between the codon usage of human viruses and the translational efficiency of their tissue of infection. This correspondence is particularly observed in early viral proteins, while late counterparts benefit from the tRNA reprogramming that underlies the process of infection. We therefore envision the development of <em>ad hoc</em> gene therapies specifically targeting the tissue of interest.</p></div><div class="section" id="sec-8"><h2 class="">METHODS</h2><div id="sec-9" class="subsection"><h3>DATA SOURCES</h3><div id="sec-10" class="subsection"><h4>Viruses and annotations</h4><p id="p-38">We included in the analysis all human-infecting viruses from the NCBI Viral Genome Browser, downloaded as of August 29, 2019. Additionally, for its interest, we added <em>a posteriori</em> the new SARS-CoV-2 virus. Viral metadata including family, genus, genetic material type and Baltimore category were retrieved either from the ICTV 2018b Master Species List<sup><a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a></sup> or the ICTV Virus Metadata Resource (talk.ictvonline.org/taxonomy/vmr/). The sequence-based phylogenetic information was obtained from <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Aiewsakun and Simmonds (2018)</a>. Tissue and cell type tropism was determined based on the curated database ViralZone<sup><a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">25</a></sup>, and allocated to each of the six main classes based on the main annotation. Sup. Table 1 contains all human-infecting viruses and their associated metadata.</p></div><div id="sec-11" class="subsection"><h4>Coding sequences</h4><p id="p-39">The coding sequences of human-infecting viruses from RefSeq were downloaded from the Codon/Codon Pair Usage Tables (CoCoPUTs) project release as of August 29, 2019<sup><a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>,<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a></sup> (Sup. Table 1). The SARS-CoV-2 and RaTG13 sequences were directly downloaded from GenBank (Sup. Table 4).</p></div><div id="sec-12" class="subsection"><h4>Virus Orthologous Groups</h4><p id="p-40">Virus Orthologous Groups and their functional annotations (virus structure and replication) were downloaded from VOGdb (<a href="http://vogdb.org">vogdb.org</a>, release number vog94). The protein sets of each VOG were formatted to a Gene Matrix Transposed (GMT) file for custom GSEA analyses.</p></div><div id="sec-13" class="subsection"><h4>TCGA translational efficiency</h4><p id="p-41">The Supply-to-Demand Adaptation (SDA) is the balance between the supply (i.e. the anticodon tRNA abundances) and demand (i.e. the weighted codon usage based on the mRNA levels) for each of the 60 codons (excluding methionine and Stop codons)<sup><a id="xref-ref-2-8" class="xref-bibr" href="#ref-2">2</a></sup>. The SDA weights of all TCGA samples were downloaded from Synapse (<a href="http://www.synapse.org/tRNA_TCGA">www.synapse.org/tRNA_TCGA</a>, syn20640275).</p></div><div id="sec-14" class="subsection"><h4>Small RNA-sequencing datasets upon viral infection</h4><p id="p-42">Three small RNA-sequencing datasets were downloaded to analyze the tRNA content of human cell lines upon viral infection. The non-productive mock infections of each dataset were also analyzed for normalization. In <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Stark et al. (2012)</a>, samples of Human Foreskin Fibroblasts (HFF) infected by human cytomegalovirus (HCMV) strain Towne at a multiplicity of infection (MOI) of 3, analyzed at 24 and 72 hours post-infection (GSE33584). In Shi et al. (2018), samples of cellosaurus KMB-17 infected by Human Simplex Virus type 1 (HSV1) strain 17 at a MOI of 1, analyzed at 48 hpi (GSE102470). In Chang et al. (2013), samples of lymphoblastoid SUP-T1 cells infected by Human immunodeficiency virus 1 (HIV1) strain LAI at a MOI of 5, at 5, 12 and 24 hpi (GSE57763).</p></div></div><div id="sec-15" class="subsection"><h3>COMPUTATIONAL ANALYSIS</h3><div id="sec-16" class="subsection"><h4>tRNA quantification</h4><p id="p-43">In small RNA-seq FASTQ files, sequencing adapters were trimmed using BBDuk from the BBMap toolkit [v38.22] (<a href="https://sourceforge.net/projects/bbmap">https://sourceforge.net/projects/bbmap</a>): k-mer=10 (allowing 8 at the end of the read), Hamming distance=1, length=10-75bp, Phred&gt;25. Using the human reference genome GRCh38 (Genome Reference Consortium Human Reference 38, GCA_000001405.15), a total of 856 nuclear tRNAs and 21 mitochondrial tRNAs were annotated with tRNAscan-SE [v2.0]<sup><a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a></sup>.</p><p id="p-44">Trimmed FASTQ files were then mapped using a specific pipeline for tRNAs<sup><a id="xref-ref-2-9" class="xref-bibr" href="#ref-2">2</a>,<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a></sup>. Summarizing, an artificial genome is first generated by masking all annotated tRNA genes and adding pre-tRNAs (i.e. tRNA genes with 3’ and 5’ genomic flanking regions) as extra chromosomes. Upon mapping to this artificial genome with Segemehl [v0.3.1]<sup><a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a></sup>, reads that map to the tRNA-masked chromosomes or to the tRNA flanking regions are filtered out in order to remove non-tRNA reads and unmature-tRNA reads respectively.</p><p id="p-45">After this first mapping step, a second library is generated by adding 3’ CCA tails and removing introns from tRNA genes. All 100% identical sequences of this so-called <em>mature</em> tRNAs are clustered to avoid redundancy. Next, the subset of filtered reads from the first mapping is aligned against the clustered mature tRNAs using Segemehl [v0.3.1]<sup><a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">59</a></sup>. Mapped reads are then realigned with GATK IndelRealigner [v3.8]<sup><a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a></sup> to reduce the number of mismatching bases across all reads.</p><p id="p-46">For quantification, isoacceptors were quantified as reads per million (RPM). In order to increase the coverage for anticodon-level quantification, we consider all reads that map unambiguously to a certain isoacceptor, even though they ambiguously map to different isodecoders (i.e. tRNA genes that differ in their sequence but share the same anticodon). Ambiguous reads mapping to genes of different isoacceptors were discarded.</p></div><div id="sec-17" class="subsection"><h4>Relative Codon Usage (RCU)</h4><p id="p-47">The RCU is defined as the contribution of a certain codon to the amino acid it belongs to. The RCU of all synonymous codons therefore sum up to 1.
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/07/2020.04.06.027557/embed/graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/07/2020.04.06.027557/embed/graphic-5.gif"/></noscript></span></span>
where <em>x<sub>C</sub></em> refers to the abundance of the codon <em>C</em>, and <em>C<sub>aa</sub></em> is the set of all synonymous codons.</p></div><div id="sec-18" class="subsection"><h4>Relative tRNA Adaptation Index (RtAI)</h4><p id="p-48">As described by <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">dos Reis et al. (2003</a>, <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">2004</a>), the tAI weights every codon based on the wobble-base codon-anticodon interaction rules. Let <em>c</em> be a codon, then the decoding weight is a weighted sum of the square-root-normalized tRNA abundances <em>tRNA<sub>cj</sub></em> for all tRNA isoacceptors <em>j</em> that bind with affinity (1 - <em>s<sub>cj</sub></em>) given the wobble-base pairing rules <em>n<sub>c</sub></em>. However, while <a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">dos Reis et al., (2004)</a> assumes that highly expressed genes are codon-optimized, here we use the non-optimized s-values to avoid a circularity in our reasoning:
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/07/2020.04.06.027557/embed/graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/07/2020.04.06.027557/embed/graphic-6.gif"/></noscript></span></span>
</p><p id="p-49">For better comparison with the SDA, an amino-acid-normalized tAI measure is defined by dividing each tAI weight by the maximum weight among all codons within each amino acid family.
<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/07/2020.04.06.027557/embed/graphic-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/07/2020.04.06.027557/embed/graphic-7.gif"/></noscript></span></span>
</p><p id="p-50">And therefore the RtAI of a certain protein is the product of weights <em>Rw</em> of each codon <em>i<sub>k</sub></em> at the triplet position <em>k</em> throughout the full gene length <em>l<sub>g</sub></em>, and normalized by the length.
<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/07/2020.04.06.027557/embed/graphic-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/07/2020.04.06.027557/embed/graphic-8.gif"/></noscript></span></span>
</p></div><div id="sec-19" class="subsection"><h4>Supply-to-Demand Adaptation (SDA)</h4><p id="p-51">The SDA aims to consider not only tRNA abundances, but also the codon usage demand. In doing so, it constitutes a global measure of translation control, since the efficiency of a certain codon depends both on its complementary anticodon abundance as well as the demand for such anticodon by other transcripts. This global control has been indeed established to play an important role in defining optimal translation programs<sup><a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a></sup>.</p><p id="p-52">The definition of the SDA is based on similar previously published metrics<sup><a id="xref-ref-2-10" class="xref-bibr" href="#ref-2">2</a>,<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a>,<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a></sup>, which consists of a ratio between the anticodon supply and demand. On the one hand, the anticodon supply is defined as the relative tAI weights <em>Rw</em> (see previous section). On the other, the anticodon demand is estimated from the codon usage at the transcriptome level. It is computed as the frequency of each codon in a transcript weighted by the corresponding transcript expression, and finally summing up over all transcripts. Let <em>c</em> be a codon, then the codon usage is a weighted sum of the counts of codon <em>c<sub>i</sub></em> in gene <em>j</em> weighted by the mRNA-seq abundance <em>mRNA<sub>j</sub></em> for all genes in the genome <em>g:</em>
<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/07/2020.04.06.027557/embed/graphic-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/07/2020.04.06.027557/embed/graphic-9.gif"/></noscript></span></span>
</p><p id="p-53">Similarly to the supply, the anticodon demand is then normalized within each amino acid family:
<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/07/2020.04.06.027557/embed/graphic-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/07/2020.04.06.027557/embed/graphic-10.gif"/></noscript></span></span>
</p><p id="p-54">Finally, the SDA weights (SDAw) are defined as the ratio between the codon supply <em>S<sub>c</sub></em> and demand <em>D<sub>c</sub></em>:
<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/07/2020.04.06.027557/embed/graphic-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/07/2020.04.06.027557/embed/graphic-11.gif"/></noscript></span></span>
</p><p id="p-55">And therefore the SDA of a certain protein is the product of weights <em>SDAw</em> of each codon <em>i<sub>k</sub></em> at the triplet position <em>k</em> throughout the full gene length <em>l<sub>g</sub></em>, and normalized by the length.
<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/07/2020.04.06.027557/embed/graphic-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/07/2020.04.06.027557/embed/graphic-12.gif"/></noscript></span></span>
</p></div><div id="sec-20" class="subsection"><h4>Internal clustering validity</h4><p>Three indexes were used to determine the clustering performance of the RCUs based on different viral features. These are “internal” metrics, since they evaluate the quality of a certain grouping using measures of the dataset itself (homogeneity of clusters, distances within and between clusters, etc.).
</p><ul class="list-unord " id="list-1"><li id="list-item-1"><p id="p-57">WB index is a ratio of the sum-of-squares (SS) within clusters and the SS between clusters, normalized by the number of clusters<sup><a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">66</a></sup>. Therefore, low values of the WB index are indicative of good clustering.</p></li><li id="list-item-2"><p id="p-58">Dunn index considers the inter-cluster distance and diameter of the cluster hypersphere<sup><a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">67</a></sup>. A higher Dunn index indicates better clustering.</p></li><li id="list-item-3"><p id="p-59">Silhouette Coefficient ranges from −1 to +1, and measures how similar an object is to its own cluster (intra-cluster distance) compared to other clusters (nearest-cluster distance)<sup><a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">68</a></sup>. A high value indicates a correct clustering.</p></li></ul></div><div id="sec-21" class="subsection"><h4>Linear Discriminant Analysis of viral RCU</h4><p id="p-60">We applied a Linear Discriminant Analysis (LDA) to the viral RCUs, taking for each virus the average RCU of its proteins. We assigned each virus to its corresponding tropism (Sup. Table 1) in order to find the linear combination of codon features that maximized differences between viral target tissues. Given the collinear nature of RCUs by definition, the estimated coefficients are impossible to interpret, although it does not hamper the classification performance.</p></div><div id="sec-22" class="subsection"><h4>Random Forest Classifier</h4><p id="p-61">To evaluate the adaptation of the viral proteins to the SDAw of human tissues, we computed their average SDA to each of the 23 TCGA tissues (Sup. Table 3). Using the set of 182 tropism-defined viruses, we had a total of 2891 viral proteins. Taking the 23 tissue-specific SDAs as features, we applied a Random Forest (RF) Classifier, populated with 100 decision trees, using the <em>scikit-learn</em> package<sup><a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">69</a></sup>. Therefore, for each of the six viral tropisms, we developed a model for predicting the tropism-positive versus tropism-negative proteins based on the translational adaptation across tissues. Given that the size of the tropism-positive and tropism-negative groups were often unbalanced, we iteratively sampled equal-sized groups, for n=100 iterations. Furthermore, we validated the results with a stratified 5-fold crossvalidation.</p><p id="p-62">In order to evaluate the performance of the RF models, we computed the Area Under the Curve (AUC) of Receiver Operating Characteristic (ROC) and Precision-Recall (PR) plots (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2A</a>). We took the average and standard deviation across all iterations. Similarly, we computed the relative feature weights corresponding to each of the 23 TCGA tissues (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig. 2B</a>).</p></div><div id="sec-23" class="subsection"><h4>Linear Discriminant Analysis of tissue-specific SDAs</h4><p id="p-63">Similar to the RF classifier, we also computed the average proteome SDA per virus in each of the 23 tissues. We then applied a Linear Discriminant Analysis (LDA) to these averaged SDAs. We assigned each virus to its corresponding tropism (Sup. Table 1) in order to find the linear combination of tissue adaptation features that maximized differences between viral target tissues (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Extended Data Fig. 1</a>, Sup. Table 3).</p></div><div id="sec-24" class="subsection"><h4>Gene Set Enrichment Analysis (GSEA)</h4><p id="p-64">We analyzed the enrichment of gene sets of the Virus Orthologous Groups using the GSEA algorithm<sup><a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">70</a></sup>. The score used to generate the ranked list input is specified in the text.</p></div></div><div id="sec-25" class="subsection"><h3>STATISTICAL ANALYSIS</h3><p id="p-65">All details of the statistical analyses can be found in the figure legends. For hypothesis testing, a Wilcoxon rank-sum test was performed. We used a significance value of 0.05.</p></div><div id="sec-26" class="subsection"><h3>DATA AND CODE AVAILABILITY</h3><p id="p-66">The code used in this study is available at GitHub (<a href="https://github.com/hexavier/tRNA_viruses">https://github.com/hexavier/tRNA_viruses</a>, <a href="https://github.com/hexavier/tRNA_mapping">https://github.com/hexavier/tRNA_mapping</a>). The published article includes all datasets generated or analyzed during this study.</p></div></div><div class="section" id="sec-28"><h2 class="">AUTHOR CONTRIBUTIONS</h2><p id="p-68">Conceptualization, X.H., M.H.S. and L.S.; Methodology, X.H., M.H.S., L.S.; Software, X.H.; Investigation, X.H.; Validation, X.H., M.H.S.; Formal analysis, X.H.; Writing-Original Draft, X.H.; Writing-Review &amp; Editing, X.H., M.H.S., L.S.; Visualisation: X.H., M.H.S., L.S.; Funding Acquisition, L.S.; Supervision, M.H.S. and L.S.</p></div><div class="section" id="sec-29"><h2 class="">COMPETING INTEREST DECLARATION</h2><p id="p-69">The authors declare no competing interests.</p></div><div class="section" id="sec-30"><h2 class="">ADDITIONAL INFORMATION</h2><p id="p-70">Further information and requests for resources and reagents should be directed to and will be fulfilled by the corresponding authors L.S. (<span class="em-link"><span class="em-addr">luis.serrano{at}crg.eu</span></span>) and M.H.S. (<span class="em-link"><span class="em-addr">martin.schaefer{at}ieo.it</span></span>).</p></div><div class="section" id="sec-31"><h2 class="">EXTENDED DATA FIGURE LEGENDS</h2><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Viruses are adapted to the tRNA-based translational efficiencies of their target tissues, related to Fig. 2. Linear Discriminant Analysis of the Supply-to-Demand Adaptation of the 182 tropism-defined viruses to each of the 23 TCGA tissues, averaged among all the viral proteins and TCGA samples. The features describing the LDA components are available in Sup. Table 3. In brackets, the percentage of variance explained by each of the components." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1251490112" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Viruses are adapted to the tRNA-based translational efficiencies of their target tissues, related to Fig. 2.&lt;/span&gt; Linear Discriminant Analysis of the Supply-to-Demand Adaptation of the 182 tropism-defined viruses to each of the 23 TCGA tissues, averaged among all the viral proteins and TCGA samples. The features describing the LDA components are available in Sup. Table 3. In brackets, the percentage of variance explained by each of the components.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Extended Data Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557/F5.medium.gif" width="440" height="296"/><noscript><img class="highwire-fragment fragment-image" alt="Extended Data Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557/F5.medium.gif" width="440" height="296"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Extended Data Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Extended Data Figure 1.</span> <span class="caption-title">Viruses are adapted to the tRNA-based translational efficiencies of their target tissues, related to <a id="xref-fig-2-6" class="xref-fig" href="#F2">Fig. 2</a>.</span><p id="p-71" class="first-child">Linear Discriminant Analysis of the Supply-to-Demand Adaptation of the 182 tropism-defined viruses to each of the 23 TCGA tissues, averaged among all the viral proteins and TCGA samples. The features describing the LDA components are available in Sup. Table 3. In brackets, the percentage of variance explained by each of the components.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Translational adaptation of SARS-CoV-2 and related viruses among human tissues, related to Fig. 3. (A) Using the codon adaptation weights of all TCGA samples, we compute the average SDA of proteins from coronaviruses and influenza A virus. (B) Difference between the SDA of SARS-CoV-2 and the closest phylogenetically related RaTG13 coronavirus (from bat) or the closest human coronavirus (SARS-CoV). For each strain, the SDA of four of the proteins are shown. Refer to Sup. Table 2 for full TCGA cancer type names." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1251490112" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Translational adaptation of SARS-CoV-2 and related viruses among human tissues, related to Fig. 3.&lt;/span&gt; (A) Using the codon adaptation weights of all TCGA samples, we compute the average SDA of proteins from coronaviruses and influenza A virus. (B) Difference between the SDA of SARS-CoV-2 and the closest phylogenetically related RaTG13 coronavirus (from bat) or the closest human coronavirus (SARS-CoV). For each strain, the SDA of four of the proteins are shown. Refer to Sup. Table 2 for full TCGA cancer type names.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Extended Data Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557/F6.medium.gif" width="440" height="388"/><noscript><img class="highwire-fragment fragment-image" alt="Extended Data Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557/F6.medium.gif" width="440" height="388"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Extended Data Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/07/2020.04.06.027557/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Extended Data Figure 2.</span> <span class="caption-title">Translational adaptation of SARS-CoV-2 and related viruses among human tissues, related to <a id="xref-fig-3-6" class="xref-fig" href="#F3">Fig. 3</a>.</span><p id="p-72" class="first-child">(A) Using the codon adaptation weights of all TCGA samples, we compute the average SDA of proteins from coronaviruses and influenza A virus. (B) Difference between the SDA of SARS-CoV-2 and the closest phylogenetically related RaTG13 coronavirus (from bat) or the closest human coronavirus (SARS-CoV). For each strain, the SDA of four of the proteins are shown. Refer to Sup. Table 2 for full TCGA cancer type names.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-32"><h2 class="">SUPPLEMENTARY TABLES</h2><p id="p-73"><strong>Sup. Table 1. Human-infecting viruses, related to <a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1</a></strong>. List of all viruses included in this study, together with their related information (genetic material, Baltimore classification, genus, family, tropism). Codon usage and annotation of all proteins of human-infecting viruses.</p><p id="p-74"><strong>Sup. Table 2. TCGA samples, related to <a id="xref-fig-2-7" class="xref-fig" href="#F2">Fig. 2</a>–<a id="xref-fig-3-7" class="xref-fig" href="#F3">3</a></strong>. Number and abbreviations of TCGA samples covering 23 cancer types. NT and TP correspond to “Normal Tissue” and “Tumor Primary” respectively.</p><p id="p-75"><strong>Sup. Table 3. Random Forest Dataset, related to <a id="xref-fig-2-8" class="xref-fig" href="#F2">Fig. 2</a></strong>. SDA of all viral proteins across the healthy samples of all TCGA tissues. Feature weights describing the components of the Linear Discriminant Analysis (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Extended Data Fig. 1</a>).</p><p id="p-76"><strong>Sup. Table 4. Translational adaptation of SARS-CoV-2, related to <a id="xref-fig-3-8" class="xref-fig" href="#F3">Fig. 3</a></strong>. SDA of viral proteins across the healthy samples of all TCGA tissues, including MERS-CoV, SARS-CoV, SARS-CoV-2 and Influenza A virus. RSEM expression of receptors and related proteins for the entry of MERS-CoV, SARS-CoV, SARS-CoV-2 and Influenza A virus.</p><p id="p-77"><strong>Sup. Table 5. Gene Set Enrichment Analysis of tropism SDA, related to <a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig. 4</a></strong>. Defined correspondences between matching tissues to viral tropisms. Supply-to-Demand Adaptation (SDA) of all tropism-defined viral proteins, with regard to their respective tissue of infection. Among the SDA of all proteins, the Gene Set Enrichment Analysis shows the Virus Orthologous Groups that are better or worse adapted.</p><p id="p-78"><strong>Sup. Table 6. Translational reprogramming upon infection, related to <a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig. 4</a></strong>. tRNA isoacceptor quantification in Reads Per Million (RPM) of all small RNA-seq samples from cell lines upon viral infection. Using their tRNA quantification, the translational efficiency of viral proteins in each sample is estimated using the relative tAI (RtAI).</p></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGEMENTS</h2><p id="p-67">We thank Hannah Benisty and Samuel Miravet-Verde for stimulating and critical discussions. The results published here are in part based on data generated by the TCGA Research Network: <a href="https://www.cancer.gov/tcga">https://www.cancer.gov/tcga</a>. We acknowledge the support of the Spanish Ministry of Science and Innovation (MICINN) (PGC2018-101271-B-I00 Plan Estatal), ‘Centro de Excelencia Severo Ochoa’, the CERCA Programme/Generalitat de Catalunya, and the Spanish Ministry of Science and Innovation (MICINN) to the EMBL partnership. The work of X.H. has been supported by a PhD fellowship from the Fundación Ramón Areces.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">REFERENCES</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.1" data-doi="10.1093/nar/9.1.213-b"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grantham</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gautier</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gouy</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jacobzone</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Mercier</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Codon catalog usage is a genome strategy modulated for gene expressivity</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">9</span>, <span class="cit-fpage">r43</span>–<span class="cit-lpage">74</span> (<span class="cit-pub-date">1981</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D9%26rft.issue%253D1%26rft.spage%253Dr43%26rft.epage%253D74%26rft.atitle%253DCodon%2Bcatalog%2Busage%2Bis%2Ba%2Bgenome%2Bstrategy%2Bmodulated%2Bfor%2Bgene%2Bexpressivity%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F9.1.213-b%26rft_id%253Dinfo%253Apmid%252F7208352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/9.1.213-b&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7208352&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.06.027557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1981KZ75700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hernandez-Alias</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Benisty</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schaefer</span>,  <span class="cit-name-given-names">M. H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Serrano</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">Translational efficiency across healthy and tumor tissues is proliferation-related</span>. <abbr class="cit-jnl-abbrev">Mol. Syst. Biol</abbr>. <span class="cit-vol">16</span>, <span class="cit-fpage">e9275</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BSyst.%2BBiol%26rft.volume%253D16%26rft.spage%253De9275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">3.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.06.027557v1.3" data-doi="10.1038/nrg2899"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Plotkin</span>,  <span class="cit-name-given-names">J. B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Kudla</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">Synonymous but not the same: the causes and consequences of codon bias</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet</abbr>. <span class="cit-vol">12</span>, <span class="cit-fpage">32</span>–<span class="cit-lpage">42</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPlotkin%26rft.auinit1%253DJ.%2BB.%26rft.volume%253D12%26rft.issue%253D1%26rft.spage%253D32%26rft.epage%253D42%26rft.atitle%253DSynonymous%2Bbut%2Bnot%2Bthe%2Bsame%253A%2Bthe%2Bcauses%2Band%2Bconsequences%2Bof%2Bcodon%2Bbias.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2899%26rft_id%253Dinfo%253Apmid%252F21102527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2899&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21102527&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.06.027557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285410500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.4" data-doi="10.1042/bst0210835"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sharp</span>,  <span class="cit-name-given-names">P. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stenico</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peden</span>,  <span class="cit-name-given-names">J. F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Lloyd</span>,  <span class="cit-name-given-names">A. T.</span></span> <span class="cit-article-title">Codon usage: mutational bias, translational selection, or both?</span> <abbr class="cit-jnl-abbrev">Biochem. Soc. Trans</abbr>. <span class="cit-vol">21</span>, <span class="cit-fpage">835</span>–<span class="cit-lpage">841</span> (<span class="cit-pub-date">1993</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochem.%2BSoc.%2BTrans%26rft_id%253Dinfo%253Adoi%252F10.1042%252Fbst0210835%26rft_id%253Dinfo%253Apmid%252F8132077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo3OiJwcGJpb3N0IjtzOjU6InJlc2lkIjtzOjg6IjIxLzQvODM1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDcvMjAyMC4wNC4wNi4wMjc1NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Morgado</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Vicente</span>,  <span class="cit-name-given-names">A. C.</span></span> <span class="cit-article-title">Global In-Silico Scenario of tRNA Genes and Their Organization in Virus Genomes</span>. <abbr class="cit-jnl-abbrev">Viruses-Basel</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">180</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses-Basel%26rft.volume%253D11%26rft.spage%253D180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.6" data-doi="10.1371/journal.pcbi.1000001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lucks</span>,  <span class="cit-name-given-names">J. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nelson</span>,  <span class="cit-name-given-names">D. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kudla</span>,  <span class="cit-name-given-names">G. R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Plotkin</span>,  <span class="cit-name-given-names">J. B.</span></span> <span class="cit-article-title">Genome Landscapes and Bacteriophage Codon Usage</span>. <abbr class="cit-jnl-abbrev">Plos Comput. Biol</abbr>. <span class="cit-vol">4</span>, <span class="cit-fpage">e1000001</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DLucks%26rft.auinit1%253DJ.%2BB.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253De1000001%26rft.epage%253De1000001%26rft.atitle%253DGenome%2Blandscapes%2Band%2Bbacteriophage%2Bcodon%2Busage.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1000001%26rft_id%253Dinfo%253Apmid%252F18463708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1000001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18463708&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.06.027557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.7" data-doi="10.1007/s00239-008-9068-6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carbone</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Codon Bias is a Major Factor Explaining Phage Evolution in Translationally Biased Hosts</span>. <abbr class="cit-jnl-abbrev">J. Mol. Evol</abbr>. <span class="cit-vol">66</span>, <span class="cit-fpage">210</span>–<span class="cit-lpage">223</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DCarbone%26rft.auinit1%253DA.%26rft.volume%253D66%26rft.issue%253D3%26rft.spage%253D210%26rft.epage%253D223%26rft.atitle%253DCodon%2Bbias%2Bis%2Ba%2Bmajor%2Bfactor%2Bexplaining%2Bphage%2Bevolution%2Bin%2Btranslationally%2Bbiased%2Bhosts.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-008-9068-6%26rft_id%253Dinfo%253Apmid%252F18286220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-008-9068-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18286220&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.06.027557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255629600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.8" data-doi="10.1093/nar/gkw1002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ahlgren</span>,  <span class="cit-name-given-names">N. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ren</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">Y. Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fuhrman</span>,  <span class="cit-name-given-names">J. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">Alignment-free d2* oligonucleotide frequency dissimilarity measure improves prediction of hosts from metagenomically-derived viral sequences</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">45</span>, <span class="cit-fpage">39</span>–<span class="cit-lpage">53</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw1002%26rft_id%253Dinfo%253Apmid%252F27899557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw1002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27899557&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.06.027557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.9" data-doi="10.1186/s40168-017-0283-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ren</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ahlgren</span>,  <span class="cit-name-given-names">N. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">Y. Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fuhrman</span>,  <span class="cit-name-given-names">J. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">VirFinder: a novel k-mer based tool for identifying viral sequences from assembled metagenomic data</span>. <abbr class="cit-jnl-abbrev">Microbiome</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">69</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiome%26rft.volume%253D5%26rft.spage%253D69%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs40168-017-0283-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s40168-017-0283-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bahir</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fromer</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Prat</span>,  <span class="cit-name-given-names">Y.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Linial</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Viral adaptation to host: a proteome-based analysis of codon usage and amino acid preferences</span>. <abbr class="cit-jnl-abbrev">Mol. Syst. Biol</abbr>. <span class="cit-vol">5</span>, (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.11" data-doi="10.1016/S0168-1702(02)00309-X"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>,  <span class="cit-name-given-names">G. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E. C.</span></span> <span class="cit-article-title">The extent of codon usage bias in human RNA viruses and its evolutionary origin</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr>. <span class="cit-vol">92</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">7</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DJenkins%26rft.auinit1%253DG.%2BM.%26rft.volume%253D92%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D7%26rft.atitle%253DThe%2Bextent%2Bof%2Bcodon%2Busage%2Bbias%2Bin%2Bhuman%2BRNA%2Bviruses%2Band%2Bits%2Bevolutionary%2Borigin.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-1702%252802%252900309-X%26rft_id%253Dinfo%253Apmid%252F12606071%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-1702(02)00309-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12606071&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.06.027557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181372400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.12" data-doi="10.1038/nbt.1635"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lauring</span>,  <span class="cit-name-given-names">A. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">J. O.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Andino</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Rationalizing the development of live attenuated virus vaccines</span>. <abbr class="cit-jnl-abbrev">Nat. Biotechnol</abbr>. <span class="cit-vol">28</span>, <span class="cit-fpage">573</span>–<span class="cit-lpage">579</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DLauring%26rft.auinit1%253DA.%2BS.%26rft.volume%253D28%26rft.issue%253D6%26rft.spage%253D573%26rft.epage%253D579%26rft.atitle%253DRationalizing%2Bthe%2Bdevelopment%2Bof%2Blive%2Battenuated%2Bvirus%2Bvaccines.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1635%26rft_id%253Dinfo%253Apmid%252F20531338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1635&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20531338&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.06.027557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pouyet</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mouchiroud</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duret</span>,  <span class="cit-name-given-names">L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Sémon</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Recombination, meiotic expression and human codon usage</span>. <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">e27344</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D6%26rft.spage%253De27344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.14" data-doi="10.1016/j.cell.2014.08.011"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gingold</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A Dual Program for Translation Regulation in Cellular Proliferation and Differentiation</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">158</span>, <span class="cit-fpage">1281</span>–<span class="cit-lpage">1292</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D158%26rft.spage%253D1281%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2014.08.011%26rft_id%253Dinfo%253Apmid%252F25215487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2014.08.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25215487&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.06.027557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.15" data-doi="10.1128/MCB.25.19.8643-8655.2005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">K.-N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gu</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fang</span>,  <span class="cit-name-given-names">N. X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saunders</span>,  <span class="cit-name-given-names">N. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Frazer</span>,  <span class="cit-name-given-names">I. H.</span></span> <span class="cit-article-title">Gene codon composition determines differentiation-dependent expression of a viral capsid gene in keratinocytes in vitro and in vivo</span>. <abbr class="cit-jnl-abbrev">Mol. Cell. Biol</abbr>. <span class="cit-vol">25</span>, <span class="cit-fpage">8643</span>–<span class="cit-lpage">8655</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DZhao%26rft.auinit1%253DK.-N.%26rft.volume%253D25%26rft.issue%253D19%26rft.spage%253D8643%26rft.epage%253D8655%26rft.atitle%253DGene%2BCodon%2BComposition%2BDetermines%2BDifferentiation-Dependent%2BExpression%2Bof%2Ba%2BViral%2BCapsid%2BGene%2Bin%2BKeratinocytes%2BIn%2BVitro%2Band%2BIn%2BVivo%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.25.19.8643-8655.2005%26rft_id%253Dinfo%253Apmid%252F16166644%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xOS84NjQzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDcvMjAyMC4wNC4wNi4wMjc1NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">W. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peng</span>,  <span class="cit-name-given-names">S. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">X. Y.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Frazer</span>,  <span class="cit-name-given-names">I.</span></span> <span class="cit-article-title">Papillomavirus capsid protein expression level depends on the match between codon usage and tRNA availability</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">73</span>, <span class="cit-fpage">4972</span>–<span class="cit-lpage">4982</span> (<span class="cit-pub-date">1999</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DZhou%26rft.auinit1%253DJ.%26rft.volume%253D73%26rft.issue%253D6%26rft.spage%253D4972%26rft.epage%253D4982%26rft.atitle%253DPapillomavirus%2BCapsid%2BProtein%2BExpression%2BLevel%2BDepends%2Bon%2Bthe%2BMatch%2Bbetween%2BCodon%2BUsage%2Band%2BtRNA%2BAvailability%26rft_id%253Dinfo%253Apmid%252F10233959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjczLzYvNDk3MiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA3LzIwMjAuMDQuMDYuMDI3NTU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.17" data-doi="10.1093/molbev/msr005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van Weringh</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">HIV-1 modulates the tRNA pool to improve translation efficiency</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol</abbr>. <span class="cit-vol">28</span>, <span class="cit-fpage">1827</span>–<span class="cit-lpage">1834</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr005%26rft_id%253Dinfo%253Apmid%252F21216840%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msr005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21216840&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.06.027557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000290814600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.18" data-doi="10.1038/nature11433"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Codon-usage-based inhibition of HIV protein synthesis by human schlafen 11</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">491</span>, <span class="cit-fpage">125</span>–<span class="cit-lpage">128</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLi%26rft.auinit1%253DM.%26rft.volume%253D491%26rft.issue%253D7422%26rft.spage%253D125%26rft.epage%253D128%26rft.atitle%253DCodon-usage-based%2Binhibition%2Bof%2BHIV%2Bprotein%2Bsynthesis%2Bby%2Bhuman%2Bschlafen%2B11.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11433%26rft_id%253Dinfo%253Apmid%252F23000900%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11433&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23000900&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.06.027557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310434500043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">B. Miller</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hippen</span>,  <span class="cit-name-given-names">A. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">M. Wright</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morris</span>,  <span class="cit-name-given-names">C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">G. Ridge</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">Human viruses have codon usage biases that match highly expressed proteins in the tissues they infect</span>. <abbr class="cit-jnl-abbrev">Biomed. Genet. Genomics</abbr> <span class="cit-vol">2</span>, (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.06.027557v1.20"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">World Health Organization</span>. <span class="cit-source">Novel Coronavirus (2019-nCoV) situation reports</span>. <a href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports">https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sungnak</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bécavin</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berg</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Network</span>,  <span class="cit-name-given-names">H. L. B.</span></span> <span class="cit-article-title">SARS-CoV-2 Entry Genes Are Most Highly Expressed in Nasal Goblet and Ciliated Cells within Human Airways</span>. <abbr class="cit-jnl-abbrev">ArXiv200306122 Q-Bio</abbr> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.22" data-doi="10.1101/2020.01.26.919985"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Single-cell RNA expression profiling of ACE2, the putative receptor of Wuhan 2019-nCov</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-fpage">2020.01.26.919985</span> (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/2020.01.26.919985</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.01.26.919985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wMS4yNi45MTk5ODV2MiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA3LzIwMjAuMDQuMDYuMDI3NTU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.23" data-doi="10.1021/acschemneuro.0c00122"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Baig</span>,  <span class="cit-name-given-names">A. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Khaleeq</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ali</span>,  <span class="cit-name-given-names">U.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Syeda</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-article-title">Evidence of the COVID-19 Virus Targeting the CNS: Tissue Distribution, Host–Virus Interaction, and Proposed Neurotropic Mechanisms</span>. <abbr class="cit-jnl-abbrev">ACS Chem. Neurosci</abbr>. (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1021/acschemneuro.0c00122</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DACS%2BChem.%2BNeurosci%26rft_id%253Dinfo%253Adoi%252F10.1021%252Facschemneuro.0c00122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/acschemneuro.0c00122&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.24" data-doi="10.1101/2020.01.30.927806"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The digestive system is a potential route of 2019-nCov infection: a bioinformatics analysis based on single-cell transcriptomes</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-fpage">2020.01.30.927806</span> (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/2020.01.30.927806</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.01.30.927806%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wMS4zMC45Mjc4MDZ2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA3LzIwMjAuMDQuMDYuMDI3NTU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.25" data-doi="10.1093/nar/gkq901"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hulo</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">ViralZone: a knowledge resource to understand virus diversity</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">39</span>, <span class="cit-fpage">D576</span>–<span class="cit-lpage">582</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq901%26rft_id%253Dinfo%253Apmid%252F20947564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq901&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20947564&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.06.027557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285831700091&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.26" data-doi="10.1186/s40168-018-0422-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aiewsakun</span>,  <span class="cit-name-given-names">P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Simmonds</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">The genomic underpinnings of eukaryotic virus taxonomy: creating a sequence-based framework for family-level virus classification</span>. <abbr class="cit-jnl-abbrev">Microbiome</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">38</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiome%26rft.volume%253D6%26rft.spage%253D38%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs40168-018-0422-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s40168-018-0422-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.27" data-doi="10.1016/j.celrep.2017.07.029"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gogakos</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Characterizing Expression and Processing of Precursor and Mature Human tRNAs by Hydro-tRNAseq and PAR-CLIP</span>. <abbr class="cit-jnl-abbrev">Cell Rep</abbr>. <span class="cit-vol">20</span>, <span class="cit-fpage">1463</span>–<span class="cit-lpage">1475</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRep%26rft.volume%253D20%26rft.spage%253D1463%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2017.07.029%26rft_id%253Dinfo%253Apmid%252F28793268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2017.07.029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28793268&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.06.027557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Z.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Global analysis of tRNA and translation factor expression reveals a dynamic landscape of translational regulation in human cancers</span>. <abbr class="cit-jnl-abbrev">Commun. Biol</abbr>. <span class="cit-vol">1</span>, <span class="cit-fpage">234</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCommun.%2BBiol%26rft.volume%253D1%26rft.spage%253D234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.29" data-doi="10.1371/journal.pone.0056642"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Belalov</span>,  <span class="cit-name-given-names">I. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Lukashev</span>,  <span class="cit-name-given-names">A. N.</span></span> <span class="cit-article-title">Causes and Implications of Codon Usage Bias in RNA Viruses</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">e56642</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D8%26rft.spage%253De56642%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0056642%26rft_id%253Dinfo%253Apmid%252F23451064%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0056642&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23451064&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.06.027557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.30" data-doi="10.1007/s00239-005-0221-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shackelton</span>,  <span class="cit-name-given-names">L. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Parrish</span>,  <span class="cit-name-given-names">C. R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E. C.</span></span> <span class="cit-article-title">Evolutionary basis of codon usage and nucleotide composition bias in vertebrate DNA viruses</span>. <abbr class="cit-jnl-abbrev">J. Mol. Evol</abbr>. <span class="cit-vol">62</span>, <span class="cit-fpage">551</span>–<span class="cit-lpage">563</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DShackelton%26rft.auinit1%253DL.%2BA.%26rft.volume%253D62%26rft.issue%253D5%26rft.spage%253D551%26rft.epage%253D563%26rft.atitle%253DEvolutionary%2Bbasis%2Bof%2Bcodon%2Busage%2Band%2Bnucleotide%2Bcomposition%2Bbias%2Bin%2Bvertebrate%2BDNA%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-005-0221-1%26rft_id%253Dinfo%253Apmid%252F16557338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-005-0221-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16557338&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.06.027557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237487400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.31" data-doi="10.1056/NEJMoa030747"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drosten</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Identification of a novel coronavirus in patients with severe acute respiratory syndrome</span>. <abbr class="cit-jnl-abbrev">N. Engl. J. Med</abbr>. <span class="cit-vol">348</span>, <span class="cit-fpage">1967</span>–<span class="cit-lpage">1976</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BEngl.%2BJ.%2BMed%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa030747%26rft_id%253Dinfo%253Apmid%252F12690091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa030747&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12690091&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.06.027557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182823400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.32" data-doi="10.1056/NEJMoa1211721"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zaki</span>,  <span class="cit-name-given-names">A. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Boheemen</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bestebroer</span>,  <span class="cit-name-given-names">T. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Osterhaus</span>,  <span class="cit-name-given-names">A. D. M. E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Fouchier</span>,  <span class="cit-name-given-names">R. A. M.</span></span> <span class="cit-article-title">Isolation of a novel coronavirus from a man with pneumonia in Saudi Arabia</span>. <abbr class="cit-jnl-abbrev">N. Engl. J. Med</abbr>. <span class="cit-vol">367</span>, <span class="cit-fpage">1814</span>–<span class="cit-lpage">1820</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.stitle%253DNEJM%26rft.aulast%253DZaki%26rft.auinit1%253DA.%2BM.%26rft.volume%253D367%26rft.issue%253D19%26rft.spage%253D1814%26rft.epage%253D1820%26rft.atitle%253DIsolation%2Bof%2Ba%2Bnovel%2Bcoronavirus%2Bfrom%2Ba%2Bman%2Bwith%2Bpneumonia%2Bin%2BSaudi%2BArabia.%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1211721%26rft_id%253Dinfo%253Apmid%252F23075143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa1211721&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23075143&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.06.027557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310773200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.33" data-doi="10.1038/s41586-020-2012-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">579</span>, <span class="cit-fpage">270</span>–<span class="cit-lpage">273</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D579%26rft.spage%253D270%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2012-7%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-020-2012-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=http://www.n&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.06.027557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.34" data-doi="10.1016/j.celrep.2019.05.048"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Broszeit</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">N-Glycolylneuraminic Acid as a Receptor for Influenza A Viruses</span>. <abbr class="cit-jnl-abbrev">Cell Rep</abbr>. <span class="cit-vol">27</span>, <span class="cit-fpage">3284</span>–<span class="cit-lpage">3294</span>.<span class="cit-page-range">e6</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRep%26rft.volume%253D27%26rft.spage%253D3284%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2019.05.048%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2019.05.048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.35" data-doi="10.1038/nature12005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Raj</span>,  <span class="cit-name-given-names">V. S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Dipeptidyl peptidase 4 is a functional receptor for the emerging human coronavirus-EMC</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">495</span>, <span class="cit-fpage">251</span>–<span class="cit-lpage">254</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D495%26rft.spage%253D251%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12005%26rft_id%253Dinfo%253Apmid%252F23486063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23486063&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.06.027557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316652300054&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">0</span>, (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.37" data-doi="10.1101/2020.03.14.988345"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">SARS-CoV-2 invades host cells via a novel route: CD147-spike protein</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-fpage">2020.03.14.988345</span> (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/2020.03.14.988345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.03.14.988345%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wMy4xNC45ODgzNDV2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA3LzIwMjAuMDQuMDYuMDI3NTU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.38" data-doi="10.1101/2020.02.27.969006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lv</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liang</span>,  <span class="cit-name-given-names">X.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-article-title">Comparative genomic analysis revealed specific mutation pattern between human coronavirus SARS-CoV-2 and Bat-SARSr-CoV RaTG13</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-fpage">2020.02.27.969006</span> (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/2020.02.27.969006</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.02.27.969006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wMi4yNy45NjkwMDZ2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA3LzIwMjAuMDQuMDYuMDI3NTU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-book" id="cit-2020.04.06.027557v1.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Knipe</span>,  <span class="cit-name-given-names">D. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Howley</span>,  <span class="cit-name-given-names">P. M.</span></span> <span class="cit-source">Fields Virology</span>. (<span class="cit-publ-name">Lippincott Williams&amp;Wilki</span>, <span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.40" data-doi="10.1128/mBio.00549-12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">S. T.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Next-generation sequencing of small RNAs from HIV-infected cells identifies phased microrna expression patterns and candidate novel microRNAs differentially expressed upon infection</span>. <abbr class="cit-jnl-abbrev">mBio</abbr> <span class="cit-vol">4</span>, <span class="cit-fpage">e00549</span>–<span class="cit-lpage">00512</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DmBio%26rft.volume%253D4%26rft.spage%253De00549%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.00549-12%26rft_id%253Dinfo%253Apmid%252F23386435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/mBio.00549-12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23386435&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.06.027557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">41.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.06.027557v1.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Deep RNA Sequencing Reveals a Repertoire of Human Fibroblast Circular RNAs Associated with Cellular Responses to Herpes Simplex Virus 1 Infection</span>. <abbr class="cit-jnl-abbrev">Cell. Physiol. Biochem. Int. J. Exp. Cell. Physiol. Biochem. Pharmacol</abbr>. <span class="cit-vol">47</span>, <span class="cit-fpage">2031</span>–<span class="cit-lpage">2045</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell.%2BPhysiol.%2BBiochem.%2BInt.%2BJ.%2BExp.%2BCell.%2BPhysiol.%2BBiochem.%2BPharmacol%26rft.volume%253D47%26rft.spage%253D2031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.42" data-doi="10.1128/JVI.05903-11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stark</span>,  <span class="cit-name-given-names">T. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arnold</span>,  <span class="cit-name-given-names">J. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spector</span>,  <span class="cit-name-given-names">D. H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Yeo</span>,  <span class="cit-name-given-names">G. W.</span></span> <span class="cit-article-title">High-resolution profiling and analysis of viral and host small RNAs during human cytomegalovirus infection</span>. <abbr class="cit-jnl-abbrev">J. Virol</abbr>. <span class="cit-vol">86</span>, <span class="cit-fpage">226</span>–<span class="cit-lpage">235</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.05903-11%26rft_id%253Dinfo%253Apmid%252F22013051%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6Ijg2LzEvMjI2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDcvMjAyMC4wNC4wNi4wMjc1NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Franzo</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tucciarone</span>,  <span class="cit-name-given-names">C. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cecchinato</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Drigo</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Canine parvovirus type 2 (CPV-2) and Feline panleukopenia virus (FPV) codon bias analysis reveals a progressive adaptation to the new niche after the host jump</span>. <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol</abbr>. <span class="cit-vol">114</span>, <span class="cit-fpage">82</span>–<span class="cit-lpage">92</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPhylogenet.%2BEvol%26rft.volume%253D114%26rft.spage%253D82%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.44" data-doi="10.1016/j.meegid.2020.104181"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Luo</span>,  <span class="cit-name-given-names">W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The fit of codon usage of human-isolated avian influenza A viruses to human</span>. <abbr class="cit-jnl-abbrev">Infect. Genet. Evol</abbr>. <span class="cit-fpage">104181</span> (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.meegid.2020.104181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfect.%2BGenet.%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.meegid.2020.104181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.meegid.2020.104181&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.45" data-doi="10.1186/1471-2148-10-253"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wong</span>,  <span class="cit-name-given-names">E. H. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">D. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rabadan</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peiris</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Poon</span>,  <span class="cit-name-given-names">L. L. M.</span></span> <span class="cit-article-title">Codon usage bias and the evolution of influenza A viruses. Codon Usage Biases of Influenza Virus</span>. <abbr class="cit-jnl-abbrev">BMC Evol. Biol</abbr>. <span class="cit-vol">10</span>, <span class="cit-fpage">253</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DWong%26rft.auinit1%253DE.%2BH.%26rft.volume%253D10%26rft.spage%253D253%26rft.epage%253D253%26rft.atitle%253DCodon%2Busage%2Bbias%2Band%2Bthe%2Bevolution%2Bof%2Binfluenza%2BA%2Bviruses.%2BCodon%2BUsage%2BBiases%2Bof%2BInfluenza%2BVirus.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-10-253%26rft_id%253Dinfo%253Apmid%252F20723216%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-10-253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20723216&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.06.027557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pan</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-etal">et al.</span> <abbr class="cit-jnl-abbrev">Clinical characteristics of COVID-19 patients with digestive symptoms in Hubei, China: a descriptive, cross-sectional, multicenter study</abbr>. <span class="cit-fpage">25</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">Z.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Effect of gastrointestinal symptoms on patients infected with COVID-19</span>. <abbr class="cit-jnl-abbrev">Gastroenterology</abbr> <span class="cit-vol">0</span>, (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.48"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Y.-C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bai</span>,  <span class="cit-name-given-names">W.-Z.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Hashikawa</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-article-title">The neuroinvasive potential of SARS-CoV2 may play a role in the respiratory failure of COVID-19 patients</span>. <abbr class="cit-jnl-abbrev">J. Med. Virol</abbr>. n/a,.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.49" data-doi="10.1101/2020.02.22.20026500"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mao</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Neurological Manifestations of Hospitalized Patients with COVID-19 in Wuhan, China: a retrospective case series study</span>. <abbr class="cit-jnl-abbrev">medRxiv</abbr> <span class="cit-fpage">2020.02.22.20026500</span> (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/2020.02.22.20026500</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DmedRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.02.22.20026500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoibWVkcnhpdiI7czo1OiJyZXNpZCI7czoyMToiMjAyMC4wMi4yMi4yMDAyNjUwMHYxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDcvMjAyMC4wNC4wNi4wMjc1NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.50" data-doi="10.1038/s41591-020-0820-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Andersen</span>,  <span class="cit-name-given-names">K. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lipkin</span>,  <span class="cit-name-given-names">W. I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E. C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Garry</span>,  <span class="cit-name-given-names">R. F.</span></span> <span class="cit-article-title">The proximal origin of SARS-CoV-2</span>. <abbr class="cit-jnl-abbrev">Nat. Med</abbr>. <span class="cit-fpage">1</span>–<span class="cit-lpage">3</span> (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41591-020-0820-9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMed%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41591-020-0820-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41591-020-0820-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.51" data-doi="10.1038/s41559-020-1124-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Dissimilation of synonymous codon usage bias in virus-host coevolution due to translational selection</span>. <abbr class="cit-jnl-abbrev">Nat. Ecol. Evol</abbr>. (<span class="cit-pub-date">2020</span>)<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41559-020-1124-7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BEcol.%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41559-020-1124-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41559-020-1124-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.52" data-doi="10.1093/nar/gks986"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pavon-Eternod</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Vaccinia and influenza A viruses select rather than adjust tRNAs to optimize translation</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">41</span>, <span class="cit-fpage">1914</span>–<span class="cit-lpage">1921</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks986%26rft_id%253Dinfo%253Apmid%252F23254333%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks986&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23254333&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.06.027557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316351800051&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.53"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mioduser</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goz</span>,  <span class="cit-name-given-names">E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Tuller</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-article-title">Significant differences in terms of codon usage bias between bacteriophage early and late genes: a comparative genomics analysis</span>. <abbr class="cit-jnl-abbrev">Bmc Genomics</abbr> <span class="cit-vol">18</span>, <span class="cit-fpage">866</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBmc%2BGenomics%26rft.volume%253D18%26rft.spage%253D866%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.54"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Walker</span>,  <span class="cit-name-given-names">P. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Changes to virus taxonomy and the International Code of Virus Classification and Nomenclature ratified by the International Committee on Taxonomy of Viruses (2019)</span>. <abbr class="cit-jnl-abbrev">Arch. Virol</abbr>. <span class="cit-vol">164</span>, <span class="cit-fpage">2417</span>–<span class="cit-lpage">2429</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArch.%2BVirol%26rft.volume%253D164%26rft.spage%253D2417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.55" data-doi="10.1016/j.jmb.2019.04.021"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Alexaki</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Codon and Codon-Pair Usage Tables (CoCoPUTs): Facilitating Genetic Variation Analyses and Recombinant Gene Design</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol</abbr>. <span class="cit-vol">431</span>, <span class="cit-fpage">2434</span>–<span class="cit-lpage">2441</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMol.%2BBiol%26rft.volume%253D431%26rft.spage%253D2434%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2019.04.021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2019.04.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.56" data-doi="10.1186/s12859-017-1793-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Athey</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A new and updated resource for codon usage tables</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">18</span>, <span class="cit-fpage">391</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D18%26rft.spage%253D391%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-017-1793-7%26rft_id%253Dinfo%253Apmid%252F28865429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-017-1793-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28865429&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.06.027557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.57"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chan</span>,  <span class="cit-name-given-names">P. P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Lowe</span>,  <span class="cit-name-given-names">T. M.</span></span> <span class="cit-article-title">tRNAscan-SE: Searching for tRNA Genes in Genomic Sequences</span>. <abbr class="cit-jnl-abbrev">Methods Mol. Biol. Clifton NJ</abbr> <span class="cit-vol">1962</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol.%2BBiol.%2BClifton%2BNJ%26rft.volume%253D1962%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.58"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Accurate mapping of tRNA reads</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">34</span>, <span class="cit-fpage">1116</span>–<span class="cit-lpage">1124</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D1116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.59" data-doi="10.1371/journal.pcbi.1000502"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Fast mapping of short sequences with mismatches, insertions and deletions using index structures</span>. <abbr class="cit-jnl-abbrev">PLoS Comput. Biol</abbr>. <span class="cit-vol">5</span>, <span class="cit-fpage">e1000502</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DHoffmann%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.issue%253D9%26rft.spage%253De1000502%26rft.epage%253De1000502%26rft.atitle%253DFast%2Bmapping%2Bof%2Bshort%2Bsequences%2Bwith%2Bmismatches%252C%2Binsertions%2Band%2Bdeletions%2Busing%2Bindex%2Bstructures.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1000502%26rft_id%253Dinfo%253Apmid%252F19750212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1000502&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19750212&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.06.027557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.60" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McKenna</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr>. <span class="cit-vol">20</span>, <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA3LzIwMjAuMDQuMDYuMDI3NTU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.61" data-doi="10.1093/nar/gkg897"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">dos Reis</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wernisch</span>,  <span class="cit-name-given-names">L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Savva</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Unexpected correlations between gene expression and codon usage bias from microarray data for the whole Escherichia coli K-12 genome</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">31</span>, <span class="cit-fpage">6976</span>–<span class="cit-lpage">6985</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253Ddos%2BReis%26rft.auinit1%253DM.%26rft.volume%253D31%26rft.issue%253D23%26rft.spage%253D6976%26rft.epage%253D6985%26rft.atitle%253DUnexpected%2Bcorrelations%2Bbetween%2Bgene%2Bexpression%2Band%2Bcodon%2Busage%2Bbias%2Bfrom%2Bmicroarray%2Bdata%2Bfor%2Bthe%2Bwhole%2BEscherichia%2Bcoli%2BK-12%2Bgenome%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg897%26rft_id%253Dinfo%253Apmid%252F14627830%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkg897&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14627830&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.06.027557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186802500035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">62.</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.62" data-doi="10.1093/nar/gkh834"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">dos Reis</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Savva</span>,  <span class="cit-name-given-names">R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Wernisch</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">Solving the riddle of codon usage preferences: a test for translational selection</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">32</span>, <span class="cit-fpage">5036</span>–<span class="cit-lpage">5044</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DReis%26rft.auinit1%253DM.%2Bd.%26rft.volume%253D32%26rft.issue%253D17%26rft.spage%253D5036%26rft.epage%253D5044%26rft.atitle%253DSolving%2Bthe%2Briddle%2Bof%2Bcodon%2Busage%2Bpreferences%253A%2Ba%2Btest%2Bfor%2Btranslational%2Bselection%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh834%26rft_id%253Dinfo%253Apmid%252F15448185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkh834&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15448185&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.06.027557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224355400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">63.</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.63" data-doi="10.1073/pnas.1719375115"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Frumkin</span>,  <span class="cit-name-given-names">I.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Codon usage of highly expressed genes affects proteome-wide translation efficiency</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>. <span class="cit-vol">115</span>, <span class="cit-fpage">E4940</span>–<span class="cit-lpage">E4949</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1719375115%26rft_id%253Dinfo%253Apmid%252F29735666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE1LzIxL0U0OTQwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDcvMjAyMC4wNC4wNi4wMjc1NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">64.</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.64" data-doi="10.1093/nar/gks772"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gingold</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dahan</span>,  <span class="cit-name-given-names">O.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Pilpel</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-article-title">Dynamic changes in translational efficiency are deduced from codon usage of the transcriptome</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">40</span>, <span class="cit-fpage">10053</span>–<span class="cit-lpage">10063</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks772%26rft_id%253Dinfo%253Apmid%252F22941644%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks772&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22941644&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.06.027557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310970700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">65.</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.65" data-doi="10.1038/nsmb.2466"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pechmann</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Frydman</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Evolutionary conservation of codon optimality reveals hidden signatures of cotranslational folding</span>. <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol</abbr>. <span class="cit-vol">20</span>, <span class="cit-fpage">237</span>–<span class="cit-lpage">243</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BStruct.%2BMol.%2BBiol%26rft.volume%253D20%26rft.spage%253D237%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.2466%26rft_id%253Dinfo%253Apmid%252F23262490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.2466&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23262490&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.06.027557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">66.</span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 66. in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.66"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">Q.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Fränti</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">WB-index: A sum-of-squares based index for cluster validity</span>. <abbr class="cit-jnl-abbrev">Data Knowl. Eng</abbr>. <span class="cit-vol">92</span>, <span class="cit-fpage">77</span>–<span class="cit-lpage">89</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DData%2BKnowl.%2BEng%26rft.volume%253D92%26rft.spage%253D77%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">67.</span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference 67. in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.67" data-doi="10.1080/01969727408546059"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dunn</span>,  <span class="cit-name-given-names">J. C.</span></span> <span class="cit-article-title">Well-Separated Clusters and Optimal Fuzzy Partitions</span>. <abbr class="cit-jnl-abbrev">J. Cybern</abbr>. <span class="cit-vol">4</span>, <span class="cit-fpage">95</span>–<span class="cit-lpage">104</span> (<span class="cit-pub-date">1974</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCybern%26rft.volume%253D4%26rft.spage%253D95%26rft_id%253Dinfo%253Adoi%252F10.1080%252F01969727408546059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/01969727408546059&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">68.</span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference 68. in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.68"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Al-Zoubi</span>,  <span class="cit-name-given-names">M. B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Raw</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">An Efficient Approach for Computing Silhouette Coefficients</span>. <abbr class="cit-jnl-abbrev">J. Comput. Sci</abbr>. <span class="cit-vol">4</span>, (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">69.</span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference 69. in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.69" data-doi="10.1016/j.cpc.2010.04.018"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pedregosa</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Scikit-learn: Machine Learning in Python</span>. <abbr class="cit-jnl-abbrev">J. Mach. Learn. Res</abbr>. <span class="cit-vol">12</span>, <span class="cit-fpage">2825</span>–<span class="cit-lpage">2830</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMach.%2BLearn.%2BRes%26rft.volume%253D12%26rft.spage%253D2825%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cpc.2010.04.018%26rft_id%253Dinfo%253Apmid%252F23755062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cpc.2010.04.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23755062&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F07%2F2020.04.06.027557.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">70.</span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference 70. in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.06.027557v1.70" data-doi="10.1073/pnas.0506580102"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Subramanian</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>. <span class="cit-vol">102</span>, <span class="cit-fpage">15545</span>–<span class="cit-lpage">15550</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0506580102%26rft_id%253Dinfo%253Apmid%252F16199517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAyLzQzLzE1NTQ1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDcvMjAyMC4wNC4wNi4wMjc1NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.04.06.027557v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.04.06.027557v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.04.06.027797v1" title="Larger active site in an ancestral hydroxynitrile lyase increases catalytically promiscuous esterase activity" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.04.06.027862v1" title="Concerted action of kinesin-1 KIF5B and kinesin-3 KIF13B promotes efficient transport of exocytotic vesicles to microtubule plus ends" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;April 07, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.04.06.027557v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.04.06.027557v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.04.06.027557v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.04.06.027557v1" class="active" data-icon-position="" data-hide-link-title="0">Translational adaptation of human viruses to the tissues they infect</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1232152" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-aTTVNQS8A-ym_2qCPimaBmAdrBhb1I-Gs149_1ZaHnE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1232152" id="node1232152--2" data-pisa="biorxiv;2020.04.06.027557v1" data-pisa-master="biorxiv;2020.04.06.027557" data-seqnum="1232152" data-apath="/biorxiv/early/2020/04/07/2020.04.06.027557.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Translational adaptation of human viruses to the tissues they infect</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Xavier</span> <span class="nlm-surname">Hernandez-Alias</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Martin H.</span> <span class="nlm-surname">Schaefer</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Luis</span> <span class="nlm-surname">Serrano</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.04.06.027557; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.04.06.027557 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.04.06.027557v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.04.06.027557v1&amp;title=Translational%20adaptation%20of%20human%20viruses%20to%20the%20tissues%20they%20infect" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.06.027557v1&amp;title=Translational%20adaptation%20of%20human%20viruses%20to%20the%20tissues%20they%20infect" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.06.027557v1&amp;text=Translational%20adaptation%20of%20human%20viruses%20to%20the%20tissues%20they%20infect" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.06.027557v1&amp;title=Translational%20adaptation%20of%20human%20viruses%20to%20the%20tissues%20they%20infect" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.04.06.027557v1&amp;t=Translational%20adaptation%20of%20human%20viruses%20to%20the%20tissues%20they%20infect" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.04.06.027557v1&amp;title=Translational%20adaptation%20of%20human%20viruses%20to%20the%20tissues%20they%20infect" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.06.027557v1&amp;title=Translational%20adaptation%20of%20human%20viruses%20to%20the%20tissues%20they%20infect" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1232152" id="node1232152--3" data-pisa="biorxiv;2020.04.06.027557v1" data-pisa-master="biorxiv;2020.04.06.027557" data-seqnum="1232152" data-apath="/biorxiv/early/2020/04/07/2020.04.06.027557.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Translational adaptation of human viruses to the tissues they infect</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Xavier</span> <span class="nlm-surname">Hernandez-Alias</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Martin H.</span> <span class="nlm-surname">Schaefer</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Luis</span> <span class="nlm-surname">Serrano</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.04.06.027557; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.04.06.027557 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Translational%20adaptation%20of%20human%20viruses%20to%20the%20tissues%20they%20infect&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Hernandez-Alias%2C+Xavier&amp;rft.au=Schaefer%2C+Martin+H.&amp;rft.au=Serrano%2C+Luis"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1232152/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1232152/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1232152/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1232152/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1232152/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1232152/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1232152/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1232152/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1232152/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1232152/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1232152/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1232152/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.06.027557v1&amp;count=horizontal&amp;via=&amp;text=Translational%20adaptation%20of%20human%20viruses%20to%20the%20tissues%20they%20infect&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.04.06.027557v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.04.06.027557v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.04.06.027557v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/systems-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Systems Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
