<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2018/02/08/261776.atom" />
<meta name="HW.pisa" content="biorxiv;261776v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="IFIT3 and IFIT2/3 promote IFIT1-mediated translation inhibition by enhancing binding to non-self RNA" />
<meta name="DC.Identifier" content="10.1101/261776" />
<meta name="DC.Date" content="2018-02-08" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2018, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Interferon-induced proteins with tetratricopeptide repeats (IFITs) are highly expressed during the cell-intrinsic immune response to viral infection. IFIT1 inhibits translation by binding directly to the 5′ end of foreign RNAs, particularly those with non-self cap structures, precluding the recruitment of the cap-binding eukaryotic translation initiation factor 4F and subsequent 40S recruitment. Interaction of different IFIT family members is well described, but little is known of the molecular basis of IFIT association or its impact on function. Here, we reconstituted different complexes of IFIT1, IFIT2 and IFIT3 in vitro , which enabled us to reveal critical aspects of IFIT complex assembly. IFIT1 interacts rapidly and strongly with IFIT3 forming a stable heterotetramer. IFIT2 and IFIT3 homodimers dissociate to form a more stable heterodimer that associates with IFIT1, forming an IFIT1:IFIT2:IFIT3 trimer. Site-directed mutagenesis revealed a C-terminal ‘YxxxL’ motif in IFIT1 that mediates its association with IFIT3. Using various reporter mRNAs, we demonstrate for the first time that IFIT3 stabilises IFIT1 binding to cap0-mRNA and enhances its translation inhibition activity. Disrupting the binding interface between IFIT1 and IFIT3 abrogated this enhancement. This work reveals molecular aspects of IFIT assembly and provides an important ‘missing link’ between IFIT interaction and function." />
<meta name="DC.Contributor" content="Renata C Fleith" />
<meta name="DC.Contributor" content="Harriet V Mears" />
<meta name="DC.Contributor" content="Edward Emmott" />
<meta name="DC.Contributor" content="Stephen C Graham" />
<meta name="DC.Contributor" content="Daniel S Mansur" />
<meta name="DC.Contributor" content="Trevor R Sweeney" />
<meta name="article:published_time" content="2018-02-08" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="IFIT3 and IFIT2/3 promote IFIT1-mediated translation inhibition by enhancing binding to non-self RNA" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Interferon-induced proteins with tetratricopeptide repeats (IFITs) are highly expressed during the cell-intrinsic immune response to viral infection. IFIT1 inhibits translation by binding directly to the 5′ end of foreign RNAs, particularly those with non-self cap structures, precluding the recruitment of the cap-binding eukaryotic translation initiation factor 4F and subsequent 40S recruitment. Interaction of different IFIT family members is well described, but little is known of the molecular basis of IFIT association or its impact on function. Here, we reconstituted different complexes of IFIT1, IFIT2 and IFIT3 &lt;i&gt;in vitro&lt;/i&gt;, which enabled us to reveal critical aspects of IFIT complex assembly. IFIT1 interacts rapidly and strongly with IFIT3 forming a stable heterotetramer. IFIT2 and IFIT3 homodimers dissociate to form a more stable heterodimer that associates with IFIT1, forming an IFIT1:IFIT2:IFIT3 trimer. Site-directed mutagenesis revealed a C-terminal ‘YxxxL’ motif in IFIT1 that mediates its association with IFIT3. Using various reporter mRNAs, we demonstrate for the first time that IFIT3 stabilises IFIT1 binding to cap0-mRNA and enhances its translation inhibition activity. Disrupting the binding interface between IFIT1 and IFIT3 abrogated this enhancement. This work reveals molecular aspects of IFIT assembly and provides an important ‘missing link’ between IFIT interaction and function.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2018/01/01" />
<meta name="citation_mjid" content="biorxiv;261776v1" />
<meta name="citation_id" content="261776v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/261776v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/261776v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/261776v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261776.full.pdf" />
<meta name="citation_doi" content="10.1101/261776" />
<meta name="citation_num_pages" content="25" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="261776" />
<meta name="citation_author" content="Renata C Fleith" />
<meta name="citation_author_institution" content="Division of Virology, Department of Pathology, University of Cambridge" />
<meta name="citation_author_institution" content="Laboratory of Immunobiology, Department of Microbiology, Immunology and Parasitology, Universidade Federal de Santa Catarina" />
<meta name="citation_author_institution" content="Universidade Federal de Minas Gerais" />
<meta name="citation_author" content="Harriet V Mears" />
<meta name="citation_author_institution" content="Division of Virology, Department of Pathology, University of Cambridge" />
<meta name="citation_author" content="Edward Emmott" />
<meta name="citation_author_institution" content="Division of Virology, Department of Pathology, University of Cambridge" />
<meta name="citation_author" content="Stephen C Graham" />
<meta name="citation_author_institution" content="Division of Virology, Department of Pathology, University of Cambridge" />
<meta name="citation_author" content="Daniel S Mansur" />
<meta name="citation_author_institution" content="Laboratory of Immunobiology, Department of Microbiology, Immunology and Parasitology, Universidade Federal de Santa Catarina" />
<meta name="citation_author" content="Trevor R Sweeney" />
<meta name="citation_author_institution" content="Division of Virology, Department of Pathology, University of Cambridge" />
<meta name="citation_author_email" content="ts629@cam.ac.uk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4016-7326" />
<meta name="citation_reference" content="Randall,R.E. and Goodbourn,S. (2008) Interferons and viruses: An interplay between induction, signalling, antiviral responses and virus countermeasures. J. Gen. Virol.,89, 1–47." />
<meta name="citation_reference" content="Schoggins,J.W., Wilson,S.J., Panis,M., Murphy,M.Y., Jones,C.T., Bieniasz,P. and Rice,C.M. (2011) A diverse range of gene products are effectors of the type i interferon antiviral response. Nature, 472, 481–485." />
<meta name="citation_reference" content="Zhou,X., Michal,J.J., Zhang,L., Ding,B., Lunney,J.K., Liu,B. and Jiang,Z. (2013) Interferon induced IFIT family genes in host antiviral defense. Int. J. Biol. Sci., 9, 200–208." />
<meta name="citation_reference" content="Grandvaux,N., Servant,M.J., TenOever,B., Sen,G.C., Balachandran,S., Barber,G.N., Lin,R. and Hiscott,J. (2002) Transcriptional profiling of interferon regulatory factor 3 target genes: direct involvement in the regulation of interferon-stimulated genes. J. Virol., 76, 5532–9." />
<meta name="citation_reference" content="Liu,Y., Zhang,Y.B., Liu,T.K. and Gui,J.F. (2013) Lineage-Specific Expansion of IFIT Gene Family: An Insight into Coevolution with IFN Gene Family. PLoS One, 8." />
<meta name="citation_reference" content="Daugherty,M.D., Schaller,A.M., Geballe,A.P. and Malik,H.S. (2016) Evolution-guided functional analyses reveal diverse antiviral specificities encoded by ifit1 genes in mammals. Elife,5." />
<meta name="citation_reference" content="Yang,Z., Liang,H., Zhou,Q., Li,Y., Chen,H., Ye,W., Chen,D., Fleming,J., Shu,H. and Liu,Y. (2012) Crystal structure of ISG54 reveals a novel RNA binding structure and potential functional mechanisms. Cell Res.,22, 1328–1338." />
<meta name="citation_reference" content="Abbas,Y.M., Pichlmair,A., Górna,M.W., Superti-Furga,G. and Nagar,B. (2013) Structural basis for viral 5’-PPP-RNA recognition by human IFIT proteins. Nature,494, 60–64." />
<meta name="citation_reference" content="Abbas,Y.M., Laudenbach,B.T., Martínez-Montero,S., Cencic,R., Habjan,M., Pichlmair,A., Damha,M.J., Pelletier,J. and Nagar,B. (2017) Structure of human IFIT1 with capped RNA reveals adaptable mRNA binding and mechanisms for sensing N1 and N2 ribose 2’-O methylations. Proc. Natl. Acad. Sci., 114, E2106–E2115." />
<meta name="citation_reference" content="Katibah,G.E., Lee,H.J., Huizar,J.P., Vogan,J.M., Alber,T. and Collins,K. (2013) TRNA Binding, Structure, and Localization of the Human Interferon-Induced Protein IFIT5. Mol. Cell,49, 743–750." />
<meta name="citation_reference" content="Kenneth Allan,R. and Ratajczak,T. (2011) Versatile TPR domains accommodate different modes of target protein recognition and function. Cell Stress Chaperones,16, 353–367." />
<meta name="citation_reference" content="Pichlmair,A., Lassnig,C., Eberle,C.-A., Górna,M.W., Baumann,C.L., Burkard,T.R., Bürckstümmer,T., Stefanovic,A., Krieger,S., Bennett,K.L., et al.(2011) IFIT1 is an antiviral protein that recognizes 5[prime]-triphosphate RNA. Nat. Immunol.,12, 624–630 LA–en." />
<meta name="citation_reference" content="Kumar,P., Sweeney,T.R., Skabkin,M.A., Skabkina,O. V., Hellen,C.U.T. and Pestova,T. V. (2014) Inhibition of translation by IFIT family members is determined by their ability to interact selectively with the 5’-terminal regions of cap0-, cap1- and 5’ppp-mRNAs. Nucleic Acids Res., 42, 3228–3245." />
<meta name="citation_reference" content="Habjan,M., Hubel,P., Lacerda,L., Benda,C., Holze,C., Eberl,C.H., Mann,A., Kindler,E., Gil-Cruz,C., Ziebuhr,J., et al.(2013) Sequestration by IFIT1 Impairs Translation of 2’O-unmethylated Capped RNA. PLoS Pathog.,9." />
<meta name="citation_reference" content="Kimura,T., Katoh,H., Kayama,H., Saiga,H., Okuyama,M., Okamoto,T., Umemoto,E., Matsuura,Y., Yamamoto,M. and Takeda,K. (2013) Ifit1 Inhibits Japanese Encephalitis Virus Replication through Binding to 5’ Capped 2’-O Unmethylated RNA. J. Virol.,87, 9997–10003." />
<meta name="citation_reference" content="Li,S.-H., Dong,H., Li,X.-F., Xie,X., Zhao,H., Deng,Y.-Q., Wang,X.-Y., Ye,Q., Zhu,S.-Y., Wang,H.-J., et al.(2013) Rational Design of a Flavivirus Vaccine by Abolishing Viral RNA 2’-O Methylation. J. Virol.,87, 5812–5819." />
<meta name="citation_reference" content="Daffis,S., Szretter,K.J., Schriewer,J., Li,J., Youn,S., Errett,J., Lin,T.Y., Schneller,S., Zust,R., Dong,H., et al. (2010) 2’-O methylation of the viral mRNA cap evades host restriction by IFIT family members. Nature,468, 452–456." />
<meta name="citation_reference" content="Szretter,K.J., Daniels,B.P., Cho,H., Gainey,M.D., Yokoyama,W.M., Gale,M., Virgin,H.W., Klein,R.S., Sen,G.C. and Diamond,M.S. (2012) 2’-O methylation of the viral mRNA cap by West Nile virus evades Ifit1-dependent and -independent mechanisms of host restriction in vivo. PLoS Pathog., 8." />
<meta name="citation_reference" content="Züst,R., Dong,H., Li,X.F., Chang,D.C., Zhang,B., Balakrishnan,T., Toh,Y.X., Jiang,T., Li,S.H., Deng,Y.Q., et al.(2013) Rational Design of a Live Attenuated Dengue Vaccine: 2’-O-Methyltransferase Mutants Are Highly Attenuated and Immunogenic in Mice and Macaques. PLoS Pathog.,9." />
<meta name="citation_reference" content="Züst,R., Cervantes-Barragan,L., Habjan,M., Maier,R., Neuman,B.W., Ziebuhr,J., Szretter,K.J., Baker,S.C., Barchet,W., Diamond,M.S., et al.(2011) Ribose 2’-O-methylation provides a molecular signature for the distinction of self and non-self mRNA dependent on the RNA sensor Mda5. Nat. Immunol.,12, 137–143." />
<meta name="citation_reference" content="Menachery,V.D., Yount,B.L., Josset,L., Gralinski,L.E., Scobey,T., Agnihothram,S., Katze,M.G. and Baric,R.S. (2014) Attenuation and Restoration of SARS-CoV Mutant lacking 2’ O Methyltransferase Activity. J. Virol.,88, 4251–64." />
<meta name="citation_reference" content="Young,D.F., Andrejeva,J., Li,X., Inesta-Vaquera,F., Dong,C., Cowling,V.H., Goodbourn,S. and Randall,R.E. (2016) Human IFIT1 Inhibits mRNA Translation of Rubulaviruses but Not Other Members of the Paramyxoviridae Family. J. Virol.,90, 9446–9456." />
<meta name="citation_reference" content="Menachery,V.D., Gralinski,L.E., Mitchell,H.D, Dinnon,K.H., Leist,S.R, Yount,B.L., Graham,R.L., McAnarney,E.T., Stratton,K.G., Cockrell,A.S., Debbink,K, Sims,A.C., Waters,K.M., Baric,R.S. (2017) Middle East Respiratory Syndrome Coronavirus Nonstructural Protein 16 Is Necessary for Interferon Resistance and Viral Pathogenesis. mSphere, 2, e00346–17." />
<meta name="citation_reference" content="Dubin,D.T., Stollar,V., Hsuchen,C.C., Timko,K. and Guild,G.M. (1977) Sindbis virus messenger RNA: the 5’-termini and methylated residues of 26 and 42 S RNA. Virology,77, 457–470." />
<meta name="citation_reference" content="Hyde,J.L., Gardner,C.L., Kimura,T., White,J.P., Liu,G., Trobaugh,D.W., Huang,C., Tonelli,M., Paessler,S., Takeda,K., et al.(2014) A Viral RNA Structural Element Alters Host Recognition of Nonself RNA. Science (80-. ).,343, 783–787." />
<meta name="citation_reference" content="Reynaud,J.M., Kim,D.Y., Atasheva,S., Rasalouskaya,A., White,J.P., Diamond,M.S., Weaver,S.C., Frolova,E.I. and Frolov,I. (2015) IFIT1 Differentially Interferes with Translation and Replication of Alphavirus Genomes and Promotes Induction of Type I Interferon. PLoS Pathog., 11." />
<meta name="citation_reference" content="Guo,J., Hui,D.J., Merrick,W.C. and Sen,G.C. (2000) A new pathway of translational regulation mediated by eukaryotic initiation factor 3. EMBO J.,19, 6891–9." />
<meta name="citation_reference" content="Saikia,P., Fensterl,V. and Sen,G.C. (2010) The Inhibitory Action of P56 on Select Functions of E1 Mediates Interferon’s Effect on Human Papillomavirus DNA Replication. J. Virol.,84, 13036–13039." />
<meta name="citation_reference" content="Stawowczyk,M., Van Scoy,S., Kumar,K.P. and Reich,N.C. (2011) The interferon stimulated gene 54 promotes apoptosis. J. Biol. Chem.,286, 7257–7266." />
<meta name="citation_reference" content="Chavali,P.L., Stojic,L., Meredith,L.W., Joseph,N., Nahorski,M.S., Sanford,T.J., Sweeney,T.R., Krishna,B.A., Hosmillo,M., Firth,A.E., et al.(2017) Neurodevelopmental protein Musashi-1 interacts with the Zika genome and promotes viral replication. Science (80-. ).,357, 83–88." />
<meta name="citation_reference" content="Niesen,F.H., Berglund,H. and Vedadi,M. (2007) The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability. Nat. Protoc.,2, 2212–2221." />
<meta name="citation_reference" content="Emmott,E. and Goodfellow,I. (2014) Identification of Protein Interaction Partners in Mammalian Cells Using SILAC-immunoprecipitation Quantitative Proteomics. J. Vis. Exp., 10.3791/51656." />
<meta name="citation_reference" content="Tyanova,S., Temu,T., Carlson,A., Sinitcyn,P., Mann,M. and Cox,J. (2015) Visualization of LC-MS/MS proteomics data in MaxQuant. Proteomics,15, 1453–1456." />
<meta name="citation_reference" content="Tyanova,S., Temu,T., Sinitcyn,P., Carlson,A., Hein,M.Y., Geiger,T., Mann,M. and Cox,J. (2016) The Perseus computational platform for comprehensive analysis of (prote)omics data. Nat. Methods,13, 731–740." />
<meta name="citation_reference" content="Dolinsky,T.J., Czodrowski,P., Li,H., Nielsen,J.E., Jensen,J.H., Klebe,G. and Baker,N.A. (2007) PDB2PQR: Expanding and upgrading automated preparation of biomolecular structures for molecular simulations. Nucleic Acids Res., 35." />
<meta name="citation_reference" content="Dolinsky,T.J., Nielsen,J.E., McCammon,J.A. and Baker,N.A. (2004) PDB2PQR: An automated pipeline for the setup of Poisson-Boltzmann electrostatics calculations. Nucleic Acids Res., 32." />
<meta name="citation_reference" content="Baker,N.A., Sept,D., Joseph,S., Holst,M.J. and McCammon,J.A. (2001) Electrostatics of nanosystems: Application to microtubules and the ribosome. Proc. Natl. Acad. Sci.,98, 10037–10041." />
<meta name="citation_reference" content="Rabbani,M.A.G., Ribaudo,M., Guo,J.-T. and Barik,S. (2016) Identification of IFN-stimulated gene (ISG) proteins that inhibit human parainfluenza virus type 3. J. Virol., 10.1128/JVI.01551-16." />
<meta name="citation_reference" content="Reich,N.C. (2013) A death-promoting role for ISG54/IFIT2. J. Interferon Cytokine Res.,33, 199–205." />
<meta name="citation_reference" content="Hsu,Y.L., Shi,S.F., Wu,W.L., Ho,L.J. and Lai,J.H. (2013) Protective roles of interferon-induced protein with tetratricopeptide repeats 3 (IFIT3) in dengue virus infection of human lung epithelial cells. PLoS One, 8." />
<meta name="citation_reference" content="Göertz,G.P., Abbo,S.R., Fros,J.J. and Pijlman,G.P. (2017) Functional RNA during Zika virus infection. Virus Res., 10.1016/j.virusres.2017.08.015." />
<meta name="twitter:title" content="IFIT3 and IFIT2/3 promote IFIT1-mediated translation inhibition by enhancing binding to non-self RNA" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Interferon-induced proteins with tetratricopeptide repeats (IFITs) are highly expressed during the cell-intrinsic immune response to viral infection. IFIT1 inhibits translation by binding directly to the 5′ end of foreign RNAs, particularly those with non-self cap structures, precluding the recruitment of the cap-binding eukaryotic translation initiation factor 4F and subsequent 40S recruitment. Interaction of different IFIT family members is well described, but little is known of the molecular basis of IFIT association or its impact on function. Here, we reconstituted different complexes of IFIT1, IFIT2 and IFIT3 in vitro , which enabled us to reveal critical aspects of IFIT complex assembly. IFIT1 interacts rapidly and strongly with IFIT3 forming a stable heterotetramer. IFIT2 and IFIT3 homodimers dissociate to form a more stable heterodimer that associates with IFIT1, forming an IFIT1:IFIT2:IFIT3 trimer. Site-directed mutagenesis revealed a C-terminal ‘YxxxL’ motif in IFIT1 that mediates its association with IFIT3. Using various reporter mRNAs, we demonstrate for the first time that IFIT3 stabilises IFIT1 binding to cap0-mRNA and enhances its translation inhibition activity. Disrupting the binding interface between IFIT1 and IFIT3 abrogated this enhancement. This work reveals molecular aspects of IFIT assembly and provides an important ‘missing link’ between IFIT interaction and function." />
<meta name="og-title" property="og:title" content="IFIT3 and IFIT2/3 promote IFIT1-mediated translation inhibition by enhancing binding to non-self RNA" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/261776v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Interferon-induced proteins with tetratricopeptide repeats (IFITs) are highly expressed during the cell-intrinsic immune response to viral infection. IFIT1 inhibits translation by binding directly to the 5′ end of foreign RNAs, particularly those with non-self cap structures, precluding the recruitment of the cap-binding eukaryotic translation initiation factor 4F and subsequent 40S recruitment. Interaction of different IFIT family members is well described, but little is known of the molecular basis of IFIT association or its impact on function. Here, we reconstituted different complexes of IFIT1, IFIT2 and IFIT3 in vitro , which enabled us to reveal critical aspects of IFIT complex assembly. IFIT1 interacts rapidly and strongly with IFIT3 forming a stable heterotetramer. IFIT2 and IFIT3 homodimers dissociate to form a more stable heterodimer that associates with IFIT1, forming an IFIT1:IFIT2:IFIT3 trimer. Site-directed mutagenesis revealed a C-terminal ‘YxxxL’ motif in IFIT1 that mediates its association with IFIT3. Using various reporter mRNAs, we demonstrate for the first time that IFIT3 stabilises IFIT1 binding to cap0-mRNA and enhances its translation inhibition activity. Disrupting the binding interface between IFIT1 and IFIT3 abrogated this enhancement. This work reveals molecular aspects of IFIT assembly and provides an important ‘missing link’ between IFIT interaction and function." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2018-02-08" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/261776v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/261776v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/261776v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/261776v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/80278" />
    <title>IFIT3 and IFIT2/3 promote IFIT1-mediated translation inhibition by enhancing binding to non-self RNA | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__vup9YXfxWOBOGbthJCUtxdPS6xRY2OQ_vXqbojfxPAk__ziC21uj-cnpEf3LG0kjRMEQHSEtjX5ljVuosklz2DZ4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"RNe060pKx8lQcbIeWIqi4DwGerodiM6qDoFRFegpo-M","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"80278","apath":"\/biorxiv\/early\/2018\/02\/08\/261776.atom","pisa":"biorxiv;261776v1","markup":[{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;261776\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;261776\/DC1"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;261776v1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;261776\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;261776\/DC1"}],"processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/261776v1","title":"IFIT3 and IFIT2\/3 promote IFIT1-mediated translation inhibition by enhancing binding to non-self RNA","identifier":"node\/80278"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022olnR65HW+4ylH+zpUYsI20Lpt8MDWVRrPOOIPJ+sPSHG\\\\\\\/4kIIoDyULTwU0V7GKOFZz1Gd7rkvhGmZfGLfkFwPBNfh7LRWH6eDxULrSsjJ8xxt6iinx0lH\\\\\\\/wokpKMcc40hKSGVPw5dUYGrAcRsOir0ctQeYpekZsrFr51BEQeJCS2L952ngtkzPqthc0Pj5X0UpVkPVTRjcO9PCzRkt20TyVbUbQtxMhbLPvkw4wUokaK\\\\\\\/6vot4BlfZGgxKpn0M20WMdsYAwPS2RbXou9qYGpO\\\\\\\/N+m09miJ5f7jBuI1hWn26I6vJDIcm0Y6j1zgxUx8ITZt\\\\\\\/EA8fH19FTM4RcOPlu+8QkB1XBkrQo7QUAZjuRrtWOpi5lObTnv\\\\\\\/\\\\\\\/01tcgkWkz1ZGEgsPxH05atjRWRa1dcoCZK3Ns+zn7HC3TIo4VQsy2dQQy8BwcMJFYCVGdkvI7JypTup2IBXv8FF7KoqgAIvN8TQW7PeZ7SttkRD4XEA4qH67DgDltG96XTS0wNWece6uizkY6FMUJbEHJs3pF8zC7R0vJ72Qwh\\\\\\\/QRBzxIoTlvVN64TrVwqzrWtoHgu5BiIODM+PUQzhNWhcxU4R5y1e6ZACz2IU\\\\\\\/E\\\\\\\/pAxuV8BGsBEbsr4UEGGu57Q8Ee6iKn6zgA53n74oZJdVdsh0+H\\\\\\\/RMHVnT0Ewt9ohlzm8RAp\\\\\\\/yO\\\\\\\/HZBBJuBWHbYP7iLgM0oFWnBXzH9ZU16WRNMHtFCH1jCtOn9S8DFrpLQgQ5li\\\\\\\/OK\\\\\\\/PFoePZRDSc9\\\\\\\/mPtgBPI06WVQQ\\\\\\\/Ojun\\\\\\\/a9ss9a+7LWHZIw2bawAAsJbI4JXvZ9E\\\\\\\/I2Vj0haFMNE04Nao3NkbOmShOShhKe0o27YIuTPZ24AJ+kqq4oUdrtJ74XEtPwDPta4RVbcZr\\\\\\\/7s1pvfaA\\\\\\\/XniN+rbWc8ryMznlZI7fBVqb29HAq8mmV2R8dnU+w5UcN1S1g1wtrEaAeP4ATunVYRAmKMpcm6Rv4PAm26tkcpZqulUDoO2CeGh6nTY3WhT7lWmePiF\\\\\\\/XC7ZUpJ3dmY8PXNisjPvX+6Z30glltXlbTpMLquJDivtxJvc2maxYUvFd8zoFX2mc39P2YNU7wdmrgGWrc77enGWT7U982INwcN\\\\\\\/64bCExr1WWU6Os\\\\\\\/XFbOudjlBpAD4sLtdWgD6GKIQPIn6AJmcnOeZ4RB0M\\\\\\\/DgJ+fMHJlFC0gRJzMd8H4G6KWdl0SF61LsuFZyRNgHRk8q7t1VaLBy+crlXN9fW\\\\\\\/TWMRr5eMuRE6\\\\\\\/G3bE6bSiX9+1ChIoMBTVBsID4edSxbQfczW1PAaLZ9yHrAYVt1aiPbJZN7\\\\\\\/\\\\\\\/BB3yrOdkXWjsqvOL9M=\\\u0022,\\\u0022iv\\\u0022:\\\u0022wtSAK9aJpgLGMbQ\\\\\\\/EEkIQA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022f4b0464a476f03c59bb9fe84109d21406873d5779a5455eeab580dd748b253dc\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/261776v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-80278 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/261776v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-RiB2EgiZvm0ihx0Ypr-CbGS5Jpw4oESgSEnA4UO1lA0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="80278" id="node80278" data-pisa="biorxiv;261776v1" data-pisa-master="biorxiv;261776" data-apath="/biorxiv/early/2018/02/08/261776.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">IFIT3 and IFIT2/3 promote IFIT1-mediated translation inhibition by enhancing binding to non-self RNA</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Renata C</span> <span class="nlm-surname">Fleith</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Harriet V</span> <span class="nlm-surname">Mears</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Edward</span> <span class="nlm-surname">Emmott</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Stephen C</span> <span class="nlm-surname">Graham</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Daniel S</span> <span class="nlm-surname">Mansur</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-4016-7326" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Trevor R</span> <span class="nlm-surname">Sweeney</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/261776 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node80278" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Renata C Fleith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Division of Virology, Department of Pathology, University of Cambridge</span>, Addenbrooke’s Hospital, Hills Road, Cambridge, <span class='nlm-country'>UK</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Laboratory of Immunobiology, Department of Microbiology, Immunology and Parasitology, Universidade Federal de Santa Catarina</span>, Florianópolis, <span class='nlm-country'>Brazil</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Universidade Federal de Minas Gerais</span>, Belo Horizonte, <span class='nlm-country'>Brazil</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Renata%2BC%2BFleith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fleith%20RC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARenata%2BC%2BFleith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Harriet V Mears </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Division of Virology, Department of Pathology, University of Cambridge</span>, Addenbrooke’s Hospital, Hills Road, Cambridge, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Harriet%2BV%2BMears%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mears%20HV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHarriet%2BV%2BMears%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Edward Emmott </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Division of Virology, Department of Pathology, University of Cambridge</span>, Addenbrooke’s Hospital, Hills Road, Cambridge, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edward%2BEmmott%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Emmott%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEdward%2BEmmott%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Stephen C Graham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Division of Virology, Department of Pathology, University of Cambridge</span>, Addenbrooke’s Hospital, Hills Road, Cambridge, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BC%2BGraham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Graham%20SC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephen%2BC%2BGraham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Daniel S Mansur </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Laboratory of Immunobiology, Department of Microbiology, Immunology and Parasitology, Universidade Federal de Santa Catarina</span>, Florianópolis, <span class='nlm-country'>Brazil</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BS%2BMansur%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mansur%20DS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BS%2BMansur%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Trevor R Sweeney </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Division of Virology, Department of Pathology, University of Cambridge</span>, Addenbrooke’s Hospital, Hills Road, Cambridge, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Trevor%2BR%2BSweeney%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sweeney%20TR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATrevor%2BR%2BSweeney%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4016-7326" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Trevor R Sweeney</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ts629@cam.ac.uk" class="" data-icon-position="" data-hide-link-title="0">ts629@cam.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/261776v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:80278" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:80278/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/261776v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:80278" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:80278/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/261776v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:80278" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:80278/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/261776v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:80278" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:80278/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/261776v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:80278" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:80278/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/261776v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:80278" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:80278/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-4">Interferon-induced proteins with tetratricopeptide repeats (IFITs) are highly expressed during the cell-intrinsic immune response to viral infection. IFIT1 inhibits translation by binding directly to the 5′ end of foreign RNAs, particularly those with non-self cap structures, precluding the recruitment of the cap-binding eukaryotic translation initiation factor 4F and subsequent 40S recruitment. Interaction of different IFIT family members is well described, but little is known of the molecular basis of IFIT association or its impact on function. Here, we reconstituted different complexes of IFIT1, IFIT2 and IFIT3 <em>in vitro</em>, which enabled us to reveal critical aspects of IFIT complex assembly. IFIT1 interacts rapidly and strongly with IFIT3 forming a stable heterotetramer. IFIT2 and IFIT3 homodimers dissociate to form a more stable heterodimer that associates with IFIT1, forming an IFIT1:IFIT2:IFIT3 trimer. Site-directed mutagenesis revealed a C-terminal ‘YxxxL’ motif in IFIT1 that mediates its association with IFIT3. Using various reporter mRNAs, we demonstrate for the first time that IFIT3 stabilises IFIT1 binding to cap0-mRNA and enhances its translation inhibition activity. Disrupting the binding interface between IFIT1 and IFIT3 abrogated this enhancement. This work reveals molecular aspects of IFIT assembly and provides an important ‘missing link’ between IFIT interaction and function.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-5">The host innate immune response provides a first line defence against invading pathogens. Following infection, pathogen recognition receptors (PRRs) sense non-self, pathogen-associated molecular patterns (PAMP<sub>s</sub>) triggering signalling pathways that activate an immune response (reviewed in 1). Detection of viral signatures by PRRs, such as RIG I-like receptor sensing of double stranded RNA, induces production of Type I and Type III interferon (IFN). Through binding of cell surface IFN receptors and subsequent activation of the JAK-STAT pathway, IFN activates the transcription of hundreds of IFN-stimulated genes (ISGs) (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>), many with known antiviral properties, priming neighbouring cells to restrict viral spread.</p><p id="p-6">The <strong><span class="underline"><em>i</em></span></strong>nter<strong><span class="underline"><em>f</em></span></strong>eron <strong><span class="underline"><em>i</em></span></strong>nduced protein with <strong><span class="underline"><em>t</em></span></strong>etratricopeptide repeats (IFIT) protein family, present in all vertebrates, include some of the most highly expressed ISGs. IFIT1, IFIT2, IFIT3 and IFIT5 contain multiple IFN-stimulated response elements (ISREs) in their promoters (reviewed in 3) and are also induced directly by the IFN-regulatory factor 3 (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>), downstream of initial PRR activation. In contrast, the promoter of the poorly characterised human IFIT1B lacks ISREs. Different species have varying complements of IFITs, but most mammals possess IFIT1, IFIT1B, IFIT2, IFIT3 and IFIT5 (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>). Phylogenetic analysis revealed that rodents, including mice, have lost IFIT1 and instead Ifit1b1 is IFN responsive (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>). In mice Ifit1b1 has undergone duplication twice (Ifit1b2 and Ifit1b3) and Ifit3 once (Ifit3b) (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>). IFITs are composed of sequential tetratricopeptide repeat (TPR) motifs that form globular N- and C-terminal domains joined by a linker of variable flexibility (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>–<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>).</p><p id="p-7">TPR motifs are frequently involved in protein-protein interactions and are commonly found in scaffolding proteins (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>). The crystal structures of IFIT1 and IFIT5 revealed a positively charged pocket formed in the groove between the N and C domains that interacts with single-stranded RNA (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>–<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>). IFIT1 RNA binding activity was first reported by Pichlmair <em>et al.</em> (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>) who identified proteins in lysates from IFN stimulated cells that interacted with 5’-ppp RNAs. Subsequently, we and others demonstrated that IFIT1 tightly binds capped RNAs lacking methylation on the first cap-proximal nucleotide (cap0) with low nanomolar affinity (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>–<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>). While similar positively charged tunnels in IFIT1 and IFIT5 interact with the phosphate backbone of bound RNAs (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>, <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>), only IFIT1 possesses a large hydrophobic cavity at the rear of the tunnel that can accommodate the cap structure (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>). These findings support a model whereby IFIT1 out-competes eukaryotic initiation factor (eIF) 4E/4F for binding to cap0-mRNAs, thereby inhibiting their translation. As host mRNAs are generally methylated on the first or first and second bases (cap1 and cap2, respectively), this selectivity offers a mechanism of recognising and blocking translation of non-self RNAs.</p><p id="p-8">Viruses have evolved numerous mechanisms to circumvent detection and inhibition by IFIT1. For example, members of the <em>Flaviviridae</em> and <em>Coronaviridae</em> families that replicate in the cytoplasm and rely on cap-dependent translation of their single-stranded positive-sense RNA genomes, encode their own 2’-<em>O</em>-methyltransferases. Disruption of methyltransferase activity increases susceptibility of the flaviviruses West Nile virus, japanese encephalitis virus and dengue virus (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>–<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>) and the coronaviruses murine hepatitis virus and severe acute respiratory syndrome virus (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>) to IFIT1 restriction. Interestingly, enzymatically 2’-<em>O</em>-methylated, capped mRNAs from parainfluenza virus 5 display differential translational sensitivity to IFIT1 <em>in vitro</em>, while wild type middle east respiratory syndrome coronavirus (MERS) replication was enhanced upon IFIT1 depletion suggesting that factors other than 5’ end methylation can influence IFIT1 recognition (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>). By contrast, alphaviruses, such as the emerging human pathogen chikungunya virus, also rely on cap-dependent translation but lack a virally encoded 2’-<em>O</em>-methyltranferase, thus possessing viral mRNAs with a cap0 structure at the 5’ end (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>). Recent evidence suggests that stable secondary structure at the 5’ end of alphaviral genomes protects the viral RNAs from IFIT1 restriction (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>). IFIT1 may also affect translation through interaction with eIF3 (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>), while its direct binding to the viral E1 protein restricts human papilloma virus replication (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>).</p><p id="p-9">An intriguing feature of IFITs is their propensity to homo and heterooligomerise. Using pull-down experiments of differentially tagged IFITs, Stawowczyk <em>et al.</em> (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>) demonstrated that IFIT2 could interact both with itself and with IFIT1 and IFIT3 in HeLa cells. In the same study, an IFIT1, IFIT2 and IFIT3 containing complex in HeLa cytoplasmic lysates was also reported to migrate between 150-200 kDa when analysed by glycerol gradient sedimentation. Deletion analysis identified the first four TPRs of IFIT2 as being important for interaction with IFIT3 while the TPR(s) of IFIT2 that promote interaction with IFIT1 could not be elucidated (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a>). IFIT1 was later reported to interact with IFIT2 or IFIT3 by size exclusion chromatography (SEC) (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>). The crystal structure of IFIT2 revealed it forms a stable, domain-swapped dimer, with TPRs of the N-terminal domain exchanged between each monomer (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>). Using native gel electrophoresis, we previously demonstrated IFIT1 and IFIT3 could homooligomerise (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">13</a>), while a recently-reported crystal structure of IFIT1 identified a motif in the C terminus responsible for IFIT1 dimerisation (BioRxiv: <a href="https://doi.org/10.1101/152850)">https://doi.org/10.1101/152850</a>).</p><p id="p-10">Despite considerable evidence for IFIT oligomerisation, little is known about how different IFITs interact and what impact this interaction has on function. To address this, here we have reconstituted different IFIT complexes from individually purified proteins, determined the oligomeric state of each and reveal for the first time that interaction with IFIT3 or a heterocomplex of IFIT2 and IFIT3 enhances the cap0 RNA binding and translation inhibition activity of IFIT1. We also demonstrate <em>in vitro</em> that although IFIT2 and IFIT3 individually form stable homodimers, the IFIT2:IFIT3 dimer complex is the energetically more stable species. Our results provide a critical missing link between IFIT oligomerisation and function, and present a mechanistic framework for understanding the role of IFITs in the host immune response.</p><div id="sec-2" class="subsection"><h3>Material and Methods</h3><div id="sec-3" class="subsection"><h4>Plasmids</h4><p id="p-11">For mammalian cell expression, sequences for human IFIT1 (BC007091.1) and IFIT5 (BC025786.1) were PCR amplified to include a 5′ Kozak sequence, 3′ FLAG tag and 5′ BamHI and 3′ XhoI sites, to facilitate cloning into pCDNA3.1. The plasmid for expression of IFIT1 in <em>Escherichia coli</em> was previously described (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">10</a>) and was used as a template for site directed mutagenesis to generate the mutant IFIT1 expression vectors. Sequences for IFIT2 (NM_001547.4) and IFIT3 (NM_001549.5) were PCR amplified to contain 5′ NdeI and 3′ XhoI restriction sites for cloning into pET28b (Novagen) producing a full-length protein with thrombin cleavable N-terminal 6-His tag. For reporter RNA transcription, the firefly luciferase reporter gene (Fluc) was PCR amplified using primers containing the 5′ UTR and 3′ UTR sequences of human β-globin (NM_000518.4), including a 5′ T7 promotor and EcoRI and PstI sites to facilitate cloning into pUC57. pUC57-ZIKV-Fluc was previously described (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>).</p></div><div id="sec-4" class="subsection"><h4>Protein expression and purification</h4><p id="p-12">Recombinant IFITs were expressed in Rosetta 2 (DE3) pLysS (Novagen). Cells were grown to an OD<sub>600</sub> of approximately 1 in 2x TY media at 37 °C. Expression was induced by adding 1 mM isopropyl β-D-1-thiogalactopyranoside. The induced culture was incubated at 22 °C for 16 hours. Cells were harvested and lysed in a buffer containing 20 mM Tris-Cl pH 7.5, 400 mM KCl, 5 % glycerol, 1 mM DTT and 0.5 mM phenylmethylsulfonyl fluoride and 0.5 mg/ml lysozyme (from hen egg). IFITs were isolated by affinity chromatography on Ni-NTA Agarose beads (Qiagen). IFIT1 and IFIT1 mutants were additionally purified by FPLC on MonoQ (Q buffer: 20 mM Tris-Cl pH 7.5, 5 % glycerol, 1 mM DTT and 100-500 mM KCl), followed by MonoS 5/50 GL (S buffer: 30 mM HEPES pH 7.5, 5 % glycerol, 1 mM DTT and 100-500 mM KCl). IFIT2 and IFIT3 were treated with thrombin (from bovine plasma).</p><p id="p-13">IFIT2 and IFIT3 were further purified on MonoQ 5/50 (Q buffer), followed by size exclusion chromatography (SEC) on Superdex 200 increase 10/300 GL or HiLoad 16/600 Superdex 200 pg columns (SEC buffer: 20 mM Tris-Cl pH 7.5, 150 mM KCl and 1 mM DTT). All FPLC columns are from GE Healthcare and all buffer reagents and enzymes were from Sigma.</p></div><div id="sec-5" class="subsection"><h4>Western blotting</h4><p id="p-14">Proteins were resolved by 12.5 % SDS-PAGE and transferred to 0.45 μm nitrocellulose membrane. For immunoprecipitations, membranes were probed with anti-FLAG M2-Peroxidase (A8592, Sigma), detected by chemiluminescence using Westar Supernova substrate (Cyanagen) and visualised on Super RX-N film (Fugifilm). As a loading control, membranes were probed with anti-GAPDH (AM4300, ThermoFisher), detected using IRDye 800CW Goat anti-Mouse and visualised on an Odyssey CLx Imaging System (Li-Cor). To normalise recombinant IFIT proteins and complexes, membranes were probed with anti-penta-His (34660, Qiagen) and quantified using ImageJ.</p></div><div id="sec-6" class="subsection"><h4>IFIT complex assembly</h4><p id="p-15">All complexes were assembled in SEC buffer.</p><p id="p-16"><strong>IFIT1:IFIT2</strong> and <strong>IFIT1:IFIT3-</strong> 1 mg/mL of each protein was mixed and incubated for one hour at 4 or 30 °C. <strong>IFIT2:IFIT3-</strong> 0.18 mg/mL of each protein was mixed and incubated for one hour at 37 °C and concentrated to 2 mg/mL. <strong>IFIT1:IFIT2:IFIT3 trimer-</strong> 0.4 mg/mL of purified IFIT2:IFIT3 complex and 0.2 mg/mL of IFIT1 were incubated for 1 hour at 30 °C. The complex was concentrated to 3 mg/mL. <strong>IFIT1:IFIT2:IFIT3 tetramer-</strong> 0.4 mg/mL of purified IFIT2:IFIT3 complex and 0.4 mg/mL of IFIT1 were incubated for 1 hour at 30 C. The complex was concentrated to 4 mg/mL. Complexes were concentrated using Amicon Ultra 0.5 mL 10 kDa molecular weight cut off filters (Millipore).</p></div><div id="sec-7" class="subsection"><h4>SEC analysis of mutant IFIT1 and IFIT3 complexes</h4><p id="p-17">Wild type or mutant IFIT1 was combined with IFIT3 as described in <em>IFIT complex assembly,</em> above. 150 µL of each assembly reaction was injected on to a Superdex 200 Increase 10/300 GL column, at 0.3 mL/min flow rate and UV280 readings were monitored. Peak fractions were analysed by SDS-PAGE and Coomassie staining.</p></div><div id="sec-8" class="subsection"><h4>SEC - Multi-Angle Light Scatter (SEC-MALS)</h4><p id="p-18">Proteins/complexes were injected (100 µL at the concentration described in <em>IFIT complex assembly, above</em>) onto an analytical Superdex 200 Increase 10/300 gel filtration column. MALS analysis was performed at room temperature, by inline measurement of static light scattering (DAWN 8+, Wyatt Technology), differential refractive index (Optilab T-rEX, Wyatt Technology), and 280 nm absorbance (Agilent 1260 UV, Agilent Technologies) following SEC at a flow rate of 0.4 mL/min. Molecular masses were calculated using the ASTRA6 software package (Wyatt Technology).</p></div><div id="sec-9" class="subsection"><h4>Differential scanning fluorimetry</h4><p id="p-19">Differential scanning fluorimetry experiments to determine the melting temperature (T<sub>M</sub>) of IFIT2, IFIT3 and IFIT2:IFIT3 were performed using a Viia7 Real-Time PCR system (Applied Biosystems). In an optical 96-Well Reaction Plate (Applied Biosystems 4366932), 1:500 Protein Thermal Shift dye (Life Technologies, 4461146) was mixed with 0.1 mg/mL protein in a final buffer composition of 20 mM HEPES pH 7.5, 150 mM KOAc, 2.5 mM MgOAc, 5% glycerol, and 1 mM DTT in a final volume of 20 μL. Emission from quadruplicate samples was measured at 623 nm while ramping from 25 – 95 °C stepwise at a rate of 1 °C per 20 seconds. To determine T<sub>m</sub>, data were analysed by non-linear regression using the Boltzmann equation y = LL + (UL – LL)/(1 + exp(T<sub>m</sub> – x)/a) where LL and UL are the minimum and maximum fluorescence intensities, respectively (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>).</p></div><div id="sec-10" class="subsection"><h4>In vitro transcription</h4><p id="p-20">pUC57-globin-Fluc was linearised with FspI and pUC57-ZIKV-Fluc was linearised with HindIII. RNA was transcribed using recombinant T7 polymerase at a final concentration of 50 ng/μL in transcription buffer (40 mM HEPES pH 7.5, 32 mM MgOAc, 40 mM DTT, 2 mM Spermidine, 10 mM NTPs, 0.2 U/μL RNaseOUT (Invitrogen)) for 2-4 hours at 37 °C. RNA was purified by DNaseI treatment, acidic phenol extraction and ethanol precipitation. Residual nucleotides were removed using Illustra MicroSpin G-50 columns (GE Healthcare). RNA was capped using ScriptCap system and ScriptCap 2′-<em>O</em>-methyltransferase (CellScript).</p></div><div id="sec-11" class="subsection"><h4>In vitro translation</h4><p id="p-21">IFIT proteins or complexes were diluted in bovine serum albumin (BSA) diluent buffer (0.5 mg/mL BSA, 20 mM Tris pH 7.5, 160 mM KCl, 5 % glycerol, 2 mM DTT, 1U/μL RNaseOUT), and incubated with 125 ng reporter RNA for 15 minutes at 37 °C to allow RNA binding. <em>In vitro</em> translation was performed using the Flexi Rabbit Reticulocyte Lysate (RRL) System (Promega) for 90 minutes at 30 °C. Reactions were terminated by incubation on ice, followed by addition of 50 volumes of passive lysis buffer (Promega). After addition of an equal volume of firefly luciferase substrate (20 mM Tricine, 2.67 mM MgSO<sub>4</sub>, 0.1 mM EDTA, 33.3 mM DTT, 530 μM ATP, 270 μM Acetyl-CoA, 30 mg/mL luciferin, 250 μM magnesium carbonate hydroxide), luciferase signal was measured by GloMax (Promega). Luciferase values were normalised to the diluent buffer-only control for each experiment.</p></div><div id="sec-12" class="subsection"><h4>Analysis of the IFIT1-mRNA interaction by inhibition of primer extension (toeprinting)</h4><p id="p-22">IFIT1/mRNA interaction was performed essentially as described previously (<a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">13</a>) with minor modifications. Cap0-ZV or cap0-β-globin reporter mRNAs (1 nM) were incubated with IFIT1 or IFIT1 containing complexes (at concentrations indicated in Figures) for 10 min at 37 °C in 20 μL reactions containing 20 mM Tris pH 7.5, 100 mM KCl, 2.5 mM MgCl<sub>2</sub>, 1 mM ATP, 0.2 mM GTP, 1 mM DTT, 0.25 mM spermidine and 0.5 mg/mL BSA. IFIT1/mRNA interaction was monitored by inhibition of primer extension using avian myeloblastosis virus reverse transcriptase (2.5 U) (Promega) and a <sup>32</sup>P-labelled primer in the presence of 4 mM MgCl<sub>2</sub> and 0.5 mM dNTPs. Full length and truncated cDNA products were separated in a denaturing 6 % acrylamide gel and detected by autoradiography using an FLA7000 Typhoon Scanner (GE). Analysis was performed using Image-Quant TL.</p></div><div id="sec-13" class="subsection"><h4>Stable isotope labelling with amino acids in cell culture and immunoprecipitation</h4><p id="p-23">HEK293T cells were cultured in Arg/Lys-free DMEM, supplemented with light (R0K0), medium (R6K4) or heavy (R10K8) amino acids, as described (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>). 1x10<sup>7</sup> cells were transfected with 10 μg plasmid DNA using Lipofectamine 2000 (ThermoFisher). After 24 hours, media was replaced to contain 1000 U/mL human interferon α-2a (Roferon-A, Roche) for a further 16 hours. Cells were harvested in lysis buffer (50 mM Tris pH 7.5, 150 mM NaCl, 1 mM EDTA, 1 % Triton X100) containing 1:200 Protease Inhibitor Cocktail Set III (Merck) and 1:200 Benzonase nuclease (Sigma-Aldrich). Lysates were normalised to 3 mg/mL of protein before incubation with anti-FLAG-M2 affinity gel (Sigma) at 4 C for 18 hours. Beads were washed 3 times in Tris-buffered saline, then resuspended in 2X SDS-sample buffer (Invitrogen) and boiled for 5 minutes to elute bound proteins.</p></div><div id="sec-14" class="subsection"><h4>LC-MS/MS and sample preparation</h4><p id="p-24">Following immunoprecipitation, the combined samples were subjected to SDS-PAGE electrophoresis on a precast gel, and extracted as a single band for in-gel trypsinisation. The resulting peptides were fractionated using an Ultimate 3000 nanoHPLC system in line with an Orbitrap Fusion Tribrid mass spectrometer (Thermo Scientific). In brief, peptides in 1% (vol/vol) formic acid were injected onto an Acclaim PepMap C18 nano-trap column (Thermo Scientific). After washing with 0.5 % (vol/vol) acetonitrile 0.1 % (vol/vol) formic acid peptides were resolved on a 250 mm × 75 μm Acclaim PepMap C18 reverse phase analytical column (Thermo Scientific) over a 150 min organic gradient, using 7 gradient segments (1-6 % solvent B over 1 min., 6-15 % B over 58 min., 15-32 %B over 58 min., 32-40 %B over 5 min., 40-90 %B over 1 min., held at 90 %B for 6 min and then reduced to 1 %B over 1 min.) with a flow rate of 300 nL min<sup>−1</sup>. Solvent A was 0.1 % formic acid and Solvent B was aqueous 80 % acetonitrile in 0.1 % formic acid. Peptides were ionized by nano-electrospray ionization at 2.0 kV using a stainless steel emitter with an internal diameter of 30 μm (Thermo Scientific) and a capillary temperature of 275 °C.</p><p id="p-25">All spectra were acquired using an Orbitrap Fusion Tribrid mass spectrometer controlled by Xcalibur 2.1 software (Thermo Scientific) and operated in data-dependent acquisition mode. FTMS1 spectra were collected at a resolution of 120 000 over a scan range (m/z) of 350-1550, with an automatic gain control (AGC) target of 300 000 and a max injection time of 100 ms. Precursors were filtered using an Intensity Range of 1E4 to 1E20 and according to charge state (to include charge states 2-6) and with monoisotopic precursor selection. Previously interrogated precursors were excluded using a dynamic window (40 s +/-10 ppm). The MS2 precursors were isolated with a quadrupole mass filter set to a width of 1.4 m/z. ITMS2 spectra were collected with an AGC target of 20 000, max injection time of 40 ms and CID collision energy of 35 %.</p></div><div id="sec-15" class="subsection"><h4>Mass spectrometry data analysis</h4><p id="p-26">The raw data files were processed and quantified using MaxQuant v1.5.7.4 (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>) and searched against the Uniprot Human database (70,550 entries, dated 19th September 2016) using the built-in Andromeda search engine. Peptide precursor mass tolerance was set a 4.5 ppm, and MS/MS tolerance was set at 0.5 Da. Search criteria included carbaminomethylation of Cys as a fixed modification. Oxidation of Met and N-terminal acetylation were selected as variable modifications. Quantification was based on Light (Arg 0, Lys 0), Medium (Arg 6, Lys 4), and Heavy (Arg 10, Lys 8) SILAC labels. Searches were performed with tryptic digestion, a minimum peptide length of seven amino acids, and a maximum of two missed cleavages were allowed. The reverse database search option was enabled and the maximum false discovery rate for both peptide and protein identifications was set to 0.01. Quantitation was performed using a mass precision of 2 ppm. The full MaxQuant output is provided as part of PRIDE submission PXD007584 permitting viewing of annotated spectra in MaxQuant v1.5.7.4. Downstream analysis was accomplished in the Perseus software (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>). Contaminants and reverse database hits were removed, and protein ratios were log2-transformed. Proteins were considered to represent putative interaction partners if they showed a significant (t-test, p&lt;0.05) increase in their abundance compared with the control pulldown and had to have been identified in at least two of the three replicates.</p></div><div id="sec-16" class="subsection"><h4>Protein structure modelling and analysis</h4><p id="p-27">Protein structure analysis and generation of protein structure images was performed using PyMOL (The PyMOL Molecular Graphics System, Version 2.0 Schrödinger, LLC). The IFIT3 model was generated by submitting the IFIT3 amino acid sequence to the Swiss Model server. The crystal structure of the IFIT2 domain-swapped homodimer (PDB:4G1T) was used as a template for model generation. The electrostatic surface potential of IFIT2 and the IFIT3 model were analysed using PDB2PQR and APBS software (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>–<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>).</p></div></div></div><div class="section" id="sec-17"><h2 class="">Results</h2><div id="sec-18" class="subsection"><h3>IFIT1 co-precipitates IFIT2 and IFIT3 independently of RNA association</h3><p id="p-28">IFIT1 was originally identified as an RNA binding protein after being precipitated by 5’-ppp RNA from lysates of IFN stimulated HEK293 cells. IFIT2 and IFIT3, as well as several well-characterised RNA binding proteins, co-precipitated with IFIT1 (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">12</a>). We used SILAC proteomics to examine if IFIT1 could interact directly with IFIT2 and IFIT3 in nuclease treated lysates from IFN-stimulated cells. HEK293 cells were passaged in differentially isotopically labelled media and transfected with either a plasmid expressing FLAG-tagged IFIT1, FLAG-tagged IFIT5 (previously reported not to interact with other IFITs (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">29</a>)), or an empty vector control. 24 hours after transfection the cells were treated with IFN-αand incubated for a further 16 hours. Preparation of nuclease treated cell lysates and pull-down experiments are described in Materials and Methods. Consistent with previous reports (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>), IFIT1 was poorly overexpressed while IFIT5 was strongly expressed (Figure S1). However, as shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>, IFIT2 and IFIT3 co-precipitated with FLAG-tagged IFIT1, while FLAG-tagged IFIT5 did not precipitate other IFIT family members (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). The full SILAC data set is available on the PRIDE server. These results independently confirm the interaction of IFIT1, 2 and 3 in IFN-stimulated cell lysates and further demonstrate that this interaction is maintained after nuclease treatment. Both IFIT2 and IFIT3 were enriched to a similar extent in the IFIT1 pull downs (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261776/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Interaction between IFIT1, IFIT2 and IFIT3 occurs independently of bound RNA. Volcano plots of SILAC proteomics data showing statistical analysis of proteins that immuno-precipitated with (A) FLAG-tagged IFIT1 or (B) FLAG-tagged IFIT5 following nuclease treatment of IFN-stimulated cell lysates. Data points corresponding to IFIT family members are coloured and labelled." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-815547595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Interaction between IFIT1, IFIT2 and IFIT3 occurs independently of bound RNA. Volcano plots of SILAC proteomics data showing statistical analysis of proteins that immuno-precipitated with (A) FLAG-tagged IFIT1 or (B) FLAG-tagged IFIT5 following nuclease treatment of IFN-stimulated cell lysates. Data points corresponding to IFIT family members are coloured and labelled.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261776/F1.medium.gif" width="440" height="267"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261776/F1.medium.gif" width="440" height="267"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261776/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261776/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-29" class="first-child">Interaction between IFIT1, IFIT2 and IFIT3 occurs independently of bound RNA. Volcano plots of SILAC proteomics data showing statistical analysis of proteins that immuno-precipitated with (A) FLAG-tagged IFIT1 or (B) FLAG-tagged IFIT5 following nuclease treatment of IFN-stimulated cell lysates. Data points corresponding to IFIT family members are coloured and labelled.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h3>IFIT1:IFIT2:IFIT3 hetero-complexes can be reconstituted from bacterially expressed proteins</h3><p id="p-30">To investigate human IFIT family oligomerisation and examine the influence of interaction with IFIT2 and IFIT3 on IFIT1 mRNA cap0 binding activity we reconstituted the IFIT1:IFIT2:IFIT3 hetero-complex <em>in vitro</em>. To this end, His-tagged IFIT1, IFIT2 and IFIT3 were individually expressed in bacteria and purified as described in Materials and Methods. The His-tag was removed from IFIT2 and IFIT3 but retained on IFIT1 for later detection. Each protein was subjected to size exclusion chromatography with multi angle light scattering (SEC-MALS) to analyse IFIT assembly. SEC-MALS reveals the molecular mass of species eluting at different volumes from a size exclusion column providing information about the oligomeric state of these species. Varying concentrations of IFIT were analysed by SEC-MALS and, consistent with a recent report (BioRxiv: <a href="https://doi.org/10.1101/152850">https://doi.org/10.1101/152850</a>), IFIT1 oligomerised in a concentration dependent manner (Figure S2A). In contrast, IFIT2 eluted as two species corresponding to a stable dimer or tetramer (Figure S2B). Surprisingly, the IFIT2 dimer had a similar elution volume to the lowest concentration of IFIT examined. This demonstrates the importance of using the SEC-MALS technique which directly determines the mass of particles in solution from their Rayleigh scattering, instead of relying on the elution volumes of molecular weight standards during SEC to infer oligomeric state. When analysed alone, IFIT3 also eluted as a mostly dimeric species, with a smaller peak corresponding to a monomer (Figure S2C).</p><p id="p-31">We next examined the oligomeric status of complexes containing mixtures of the individually purified IFIT proteins. When incubated at 4 °C for 1 hour IFIT1 and IFIT3 formed a stable complex that eluted with a molecular mass of 221 kDa (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>, peak a). Analysis of the protein in peak a by SDS-PAGE (gel inset) shows that IFIT1 and IFIT3 are equimolar, indicating that this complex represents stable IFIT1:IFIT3 tetramers. A later eluting species likely corresponding to IFIT1:IFIT3 dimers (123 kDa) is also evident (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>, peak b). IFIT1 and IFIT2 were incubated at 30 °C for 1 hour and analysed by SEC-MALS. The IFIT1:IFIT2 complex was less defined and eluted as multiple species ranging from 148 to &gt;234 kDa as measured by SEC-MALS (Figure S2D). While the most abundant IFIT1:IFIT2 complex form is likely tetrameric, the presence of several overlapping peaks precludes reliable determination of specific molecular mass and hence oligomeric state.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261776/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="In vitro reconstitution of IFIT oligomeric assemblies. (A,B) Indicated IFIT complexes were assembled and analysed by SEC-MALS. (C) SEC-MALS analysis of the purified IFIT2:IFIT3 dimer alone or after incubation with an equimolar amount (E) or a 2-fold excess (F) of IFIT1. (D) Differential scanning fluorimetry analysis of IFIT2, IFIT3 and IFIT2:IFIT3 dimers. Gel inset shows SDS-PAGE analysis of proteins used for analysis. For A-C, D and F, complexes were formed at the indicated temperatures. Normalised differential refractive index (nRI) is shown as dotted or broken lines on the left y-axis. Calculated molecular masses (kDa) of eluting species are shown as solid, red lines on the right y-axis. Gel insets below each trace show SDS-PAGE analysis of each run. Protein gel lanes and corresponding peaks are indicated by lower case letters. The position of IFIT1, IFIT2 and IFIT3 on the protein gels is indicated. The calculated molecular masses of individual IFITs are: IFIT1(with 6His tag)-58.6 kDa, IFIT2-54.9 kD and IFIT3-56.2 kDa." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-815547595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;In vitro&lt;/em&gt; reconstitution of IFIT oligomeric assemblies. (&lt;strong&gt;A,B&lt;/strong&gt;) Indicated IFIT complexes were assembled and analysed by SEC-MALS. (&lt;strong&gt;C&lt;/strong&gt;) SEC-MALS analysis of the purified IFIT2:IFIT3 dimer alone or after incubation with an equimolar amount (&lt;strong&gt;E&lt;/strong&gt;) or a 2-fold excess (&lt;strong&gt;F&lt;/strong&gt;) of IFIT1. (&lt;strong&gt;D&lt;/strong&gt;) Differential scanning fluorimetry analysis of IFIT2, IFIT3 and IFIT2:IFIT3 dimers. Gel inset shows SDS-PAGE analysis of proteins used for analysis. For A-C, D and F, complexes were formed at the indicated temperatures. Normalised differential refractive index (nRI) is shown as dotted or broken lines on the left &lt;em&gt;y-axis&lt;/em&gt;. Calculated molecular masses (kDa) of eluting species are shown as solid, red lines on the right &lt;em&gt;y-axis&lt;/em&gt;. Gel insets below each trace show SDS-PAGE analysis of each run. Protein gel lanes and corresponding peaks are indicated by lower case letters. The position of IFIT1, IFIT2 and IFIT3 on the protein gels is indicated. The calculated molecular masses of individual IFITs are: IFIT1(with 6His tag)-58.6 kDa, IFIT2-54.9 kD and IFIT3-56.2 kDa.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261776/F2.medium.gif" width="350" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261776/F2.medium.gif" width="350" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261776/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261776/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-32" class="first-child"><em>In vitro</em> reconstitution of IFIT oligomeric assemblies. (<strong>A,B</strong>) Indicated IFIT complexes were assembled and analysed by SEC-MALS. (<strong>C</strong>) SEC-MALS analysis of the purified IFIT2:IFIT3 dimer alone or after incubation with an equimolar amount (<strong>E</strong>) or a 2-fold excess (<strong>F</strong>) of IFIT1. (<strong>D</strong>) Differential scanning fluorimetry analysis of IFIT2, IFIT3 and IFIT2:IFIT3 dimers. Gel inset shows SDS-PAGE analysis of proteins used for analysis. For A-C, D and F, complexes were formed at the indicated temperatures. Normalised differential refractive index (nRI) is shown as dotted or broken lines on the left <em>y-axis</em>. Calculated molecular masses (kDa) of eluting species are shown as solid, red lines on the right <em>y-axis</em>. Gel insets below each trace show SDS-PAGE analysis of each run. Protein gel lanes and corresponding peaks are indicated by lower case letters. The position of IFIT1, IFIT2 and IFIT3 on the protein gels is indicated. The calculated molecular masses of individual IFITs are: IFIT1(with 6His tag)-58.6 kDa, IFIT2-54.9 kD and IFIT3-56.2 kDa.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">Because IFIT2 and IFIT3 were enriched to a similar degree in our IFIT1 pull down SILAC experiments we hypothesised that IFIT1 interacts with a heterodimer of IFIT2 and IFIT3. We therefore next examined if IFIT2 and IFIT3 could interact. Equal amounts of IFIT2 and IFIT3 were mixed and incubated at 4 °C for 1 hour. When analysed by SEC-MALS two peaks corresponding to IFIT2 and IFIT3 homodimers were clearly separated (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>, grey dotted line). In contrast, when IFIT2 and IFIT3 were instead incubated at 37 °C for 1 hour an IFIT2:IFIT3 heterodimer was formed (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>, black dashed line). The molecular weight of the eluting species (110 kDa) and analysis by SDS-PAGE (gel inset) are consistent with this complex representing an IFIT2:IFIT3 heterodimer. To test the stability of the IFIT2:IFIT3 heterodimer, corresponding peak fractions were concentrated and reanalysed by SEC-MALS. As evident in <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2C</a>, there was no observed dissociation of the complex into constituent components, confirming that the IFIT2:IFIT3 complex remains stable.</p><p id="p-34">To determine the relative stabilities of the different dimeric complexes, we examined IFIT2, IFIT3 and the IFIT2:IFIT3 heterodimer using differential scanning fluorimetry. Using this approach, protein unfolding is monitored by measuring the signal from a dye that fluoresces in hydrophobic environments, at increasing temperature. More stable proteins unfold at higher temperatures than less stable proteins. Unfolding exposes hydrophobic regions of the protein making them more accessible to the dye. <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2D</a> shows the thermal melt curves for IFIT2 and IFIT3 homodimers and the IFIT2:IFIT3 heterodimer. The protein used in this assay is shown in the gel inset below the graph. IFIT2 and IFIT3 exhibit a monophasic melt curve from with T<sub>m</sub>’s of 37.3 °C and 34.1 °C, respectively, were calculated as described in the Material and Methods. This indicates that the IFIT2 is more thermodynamically stable than IFIT3. In contrast, the IFIT2:IFIT3 heterodimer exhibits a biphasic melt curve from which a reliable T<sub>m</sub> could not be calculated. However, as is evident from comparison with the IFIT2 and IFIT3 melt curves that the melting temperature of the IFIT2:IFIT3 heterodimer, and thus the stability, is greater than that of either the homodimers alone.</p><p id="p-35">Having successfully formed a stable IFIT2:IFIT3 complex, we attempted to reconstitute the full IFIT1:IFIT2:IFIT3 heterotrimeric complex. An equimolar ratio of IFIT1 was incubated with the purified (preformed) IFIT2:IFIT3 heterodimeric complex. Most of the IFIT1 was incorporated into a larger complex that eluted with a molecular mass of 165 kDa corresponding to a trimer (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2E</a>). Analysis of this complex by SDS-PAGE (gel inset) reveals that it contains equimolar amounts of IFIT1, IFIT2 and IFIT3. The remainder of the IFIT1 complexed with free IFIT3, a minor contaminant of the purified IFIT2:IFIT3 heterodimer, and eluted slightly earlier as a hetero-tetrameric complex with a molecular mass of 218 kDa. Finally, we incubated the IFIT2:IFIT3 heterodimer with a two-fold molar excess of IFIT1 at 30 °C for 1 hour. Analysis by SEC-MALS reveals a clear shift of the heterotrimeric complex to a heterotetrameric complex with a molecular mass of 236 kDa (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2F</a>). SDS-PAGE analysis of this complex (gel inset) reveals a molar excess of IFIT1 over IFIT2 and IFIT3 indicating that the heterotetrameric complex consists of two IFIT1 molecules to one molecule each of IFIT2 and 3 (compare gel insets in Figures <a id="xref-fig-2-9" class="xref-fig" href="#F2">2 E</a> and <a id="xref-fig-2-10" class="xref-fig" href="#F2">F</a>). The IFIT1:IFIT2:IFIT3 heterotetramer was less stable than the trimer, precipitating when incubated at 4 °C. Subsequently, only the IFIT1:IFIT2:IFIT3 heterotrimer was used for further experiments.</p></div><div id="sec-20" class="subsection"><h3>IFIT3 and IFIT2:IFIT3 stimulate the cap0-RNA translation inhibition activity of IFIT1</h3><p id="p-36">To investigate the impact of hetero-oligomerisation on the ability of IFIT1 to inhibit cap0 dependent translation we used a similar approach to that previously described by Young <em>et al.</em> (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a>). <em>In vitro</em> transcribed mRNAs comprising a firefly luciferase (Fluc) reporter flanked by either the human β- globin or zika virus (ZV) 5' and 3' untranslated regions (UTRs) were post-transcriptionally capped as described in the Materials and Methods. The β-globin and ZV constructs represent mRNAs with weak or strong secondary structures in the 5' UTR, respectively. Schematics of the two constructs are shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>. These mRNAs were incubated with different IFIT heterocomplexes before addition of rabbit reticulocyte lysate (RRL). Translation was quantified by measuring the luminescence from the Fluc reporter. The amount of IFIT1 added was equalised by western blotting against the His-tag on the purified IFIT1 in each complex. The upper panel of <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a> shows an SDS-PAGE analysis of the protein complexes used in these experiments while the lower panel shows an anti-His-tag western blot of the same complexes. The linearity of the western blot signal to protein concentration was confirmed as shown in Figure S3.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261776/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="IFIT3 and IFIT2:IFIT3 enhance IFIT1 cap0 translation inhibition in vitro. (A)Schematic representation of the cap0-mRNAs used in in vitro translation assays in RRL. (B) IFIT1 containing complexes included in in vitro translation assaysUpper panel, Coomassie stained SDS-PAGE gel analysis of individually purified and complexed IFITs. Lower panel, analysis of the same samples as in the upper panel by western blotting against the His-tag. Note, the His-tag was removed from IFIT2 and IFIT3 but not IFIT1 during purification. (C-F) Luciferase activity from RRL incubated with cap0- or cap1--globin Fluc RNA (C, E) or cap0- or cap1-ZV Fluc RNA (D, F) in the presence of increasing concentrations of IFIT1 or IFIT1 containing complexes as indicated. Data are normalised to the luciferase activity in the absence of IFITs and shown as the mean the standard error of three separate experiments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-815547595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;IFIT3 and IFIT2:IFIT3 enhance IFIT1 cap0 translation inhibition &lt;em&gt;in vitro&lt;/em&gt;. (&lt;strong&gt;A&lt;/strong&gt;)Schematic representation of the cap0-mRNAs used in &lt;em&gt;in vitro&lt;/em&gt; translation assays in RRL. (&lt;strong&gt;B&lt;/strong&gt;) IFIT1 containing complexes included in &lt;em&gt;in vitro&lt;/em&gt; translation assays&lt;em&gt;Upper panel&lt;/em&gt;, Coomassie stained SDS-PAGE gel analysis of individually purified and complexed IFITs. &lt;em&gt;Lower panel&lt;/em&gt;, analysis of the same samples as in the upper panel by western blotting against the His-tag. Note, the His-tag was removed from IFIT2 and IFIT3 but not IFIT1 during purification. (&lt;strong&gt;C-F&lt;/strong&gt;) Luciferase activity from RRL incubated with cap0- or cap1--globin Fluc RNA (C, E) or cap0- or cap1-ZV Fluc RNA (D, F) in the presence of increasing concentrations of IFIT1 or IFIT1 containing complexes as indicated. Data are normalised to the luciferase activity in the absence of IFITs and shown as the mean the standard error of three separate experiments.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261776/F3.medium.gif" width="407" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261776/F3.medium.gif" width="407" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261776/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261776/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <p id="p-37" class="first-child">IFIT3 and IFIT2:IFIT3 enhance IFIT1 cap0 translation inhibition <em>in vitro</em>. (<strong>A</strong>)Schematic representation of the cap0-mRNAs used in <em>in vitro</em> translation assays in RRL. (<strong>B</strong>) IFIT1 containing complexes included in <em>in vitro</em> translation assays<em>Upper panel</em>, Coomassie stained SDS-PAGE gel analysis of individually purified and complexed IFITs. <em>Lower panel</em>, analysis of the same samples as in the upper panel by western blotting against the His-tag. Note, the His-tag was removed from IFIT2 and IFIT3 but not IFIT1 during purification. (<strong>C-F</strong>) Luciferase activity from RRL incubated with cap0- or cap1--globin Fluc RNA (C, E) or cap0- or cap1-ZV Fluc RNA (D, F) in the presence of increasing concentrations of IFIT1 or IFIT1 containing complexes as indicated. Data are normalised to the luciferase activity in the absence of IFITs and shown as the mean the standard error of three separate experiments.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">IFIT1 inhibited translation of the cap0-β-globin Fluc reporter in a concentration-dependent manner (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>). Pre-incubation of IFIT1 and IFIT2 had no detectable impact on the inhibition of translation of the cap0-β-globin Fluc reporter and so was not examined further. In contrast, complexing with IFIT3 reproducibly decreased the concentration of IFIT1 required to cause 50 % inhibition of the same reporter (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a>). IFIT1 inhibited translation of the cap0-ZV reporter less than the β-globin reporter overall, but complexing with IFIT3 again enhanced IFIT1’s translation inhibitory effect (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3D</a>). Because the heterotetrameric IFIT1:IFIT2:IFIT3 complex was unstable we focused on the heterotrimeric complex for further analysis. As can be seen in <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3E</a> and <a id="xref-fig-3-7" class="xref-fig" href="#F3">3F</a>, stimulation of IFIT1 translation inhibition was reproducibly observed in the context of the IFIT1:IFIT2:IFIT3 complex. Reporter mRNAs bearing cap1 structures were also examined (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3C-F</a> and Figure S4). Inhibition of cap1 mRNAs was only observed at very high concentrations of IFIT1.</p></div><div id="sec-21" class="subsection"><h3>IFIT3 and IFIT2:IFIT3 enhance cap0-RNA binding by IFIT1</h3><p id="p-39">We and others have previously demonstrated that IFIT1 binds preferentially to mRNA with a cap0 structure at the 5 end (<a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">13</a>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>). Using our purified complexes, we next examined if the interaction of IFIT1 with cap0 mRNA was altered when part of a larger IFIT1:IFIT3 or IFIT1:IFIT2:IFIT3 complex. We used a primer extension inhibition assay to monitor the IFIT1/cap0 mRNA interaction as previously described (<a id="xref-ref-13-6" class="xref-bibr" href="#ref-13">13</a>). An advantage of this technique over other methods to analyse protein-RNA interactions, such as electrophoretic mobility shift assays, is that the primer extension reaction is performed in equilibrium binding conditions. IFIT1 alone or as part of an IFIT heterocomplex was incubated with the <em>in vitro</em> transcribed and capped model β-globin and ZV mRNAs prior to the addition of a radiolabelled primer that binds within the Fluc mRNA sequence. A reverse transcription reaction is performed in which a full-length cDNA is produced in the absence of IFIT1, whereas a 7 nucleotide truncated cDNA, corresponding to the length of the IFIT1 RNA-binding surface, is produced in the presence of IFIT1 (<a id="xref-ref-13-7" class="xref-bibr" href="#ref-13">13</a>). The cDNA products are subsequently separated by denaturing PAGE and detected by autoradiography. IFIT protein complexes shown in <a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3B</a> were used in the binding reactions. Representative autoradiographs are shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>. Quantification of the cDNA products was performed as described in the Materials and Methods and the binding curves shown in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B and C</a> show the fraction of RNA bound at varying IFIT concentrations.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261776/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="IFIT3 containing complexes stabilise IFIT1 binding to cap0 RNA. (A)Toeprinting analysis of the interaction of IFIT1 and IFIT1 containing complexes with cap0 RNA. The full-length and 7 nucleotide (nt) truncated cDNA product produced by IFIT1 binding are indicated. Protein complexes and RNAs are the same as those used in Figure 3. (B, C) Graphs represent fraction of RNA bound by IFIT1 and IFIT1 containing complexes at varying IFIT1 concentrations. Curves representative of three separate experiments were fitted using the nonlinear Hill equation, Fraction[bound]=[IFIT1]h &#x2022; Fraction[bound]max/([IFIT1]h+Kh1/2,app) from data where [IFIT1] was &#x2265; 10 &#x2022; [mRNA]. K1/2, appand Hill coefficients (h) are listed in the text." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-815547595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;IFIT3 containing complexes stabilise IFIT1 binding to cap0 RNA. (&lt;strong&gt;A&lt;/strong&gt;)Toeprinting analysis of the interaction of IFIT1 and IFIT1 containing complexes with cap0 RNA. The full-length and 7 nucleotide (nt) truncated cDNA product produced by IFIT1 binding are indicated. Protein complexes and RNAs are the same as those used in Figure 3. (B, C) Graphs represent fraction of RNA bound by IFIT1 and IFIT1 containing complexes at varying IFIT1 concentrations. Curves representative of three separate experiments were fitted using the nonlinear Hill equation, Fraction&lt;sup&gt;[bound]&lt;/sup&gt;=[IFIT1]&lt;sup&gt;h&lt;/sup&gt; &#x2022; Fraction&lt;sup&gt;[bound]&lt;/sup&gt;&lt;sub&gt;max&lt;/sub&gt;/([IFIT1]&lt;sup&gt;h&lt;/sup&gt;+K&lt;sup&gt;h&lt;/sup&gt;&lt;sub&gt;1/2,app&lt;/sub&gt;) from data where [IFIT1] was &#x2265; 10 &#x2022; [mRNA]. K&lt;sub&gt;1/2, &lt;em&gt;&lt;sub&gt;app&lt;/sub&gt;&lt;/em&gt;&lt;/sub&gt;and Hill coefficients (&lt;em&gt;h&lt;/em&gt;) are listed in the text.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261776/F4.medium.gif" width="440" height="381"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261776/F4.medium.gif" width="440" height="381"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261776/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261776/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <p id="p-40" class="first-child">IFIT3 containing complexes stabilise IFIT1 binding to cap0 RNA. (<strong>A</strong>)Toeprinting analysis of the interaction of IFIT1 and IFIT1 containing complexes with cap0 RNA. The full-length and 7 nucleotide (nt) truncated cDNA product produced by IFIT1 binding are indicated. Protein complexes and RNAs are the same as those used in <a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3</a>. (B, C) Graphs represent fraction of RNA bound by IFIT1 and IFIT1 containing complexes at varying IFIT1 concentrations. Curves representative of three separate experiments were fitted using the nonlinear Hill equation, Fraction<sup>[bound]</sup>=[IFIT1]<sup>h</sup> • Fraction<sup>[bound]</sup><sub>max</sub>/([IFIT1]<sup>h</sup>+K<sup>h</sup><sub>1/2,app</sub>) from data where [IFIT1] was ≥ 10 • [mRNA]. K<sub>1/2, <em><sub>app</sub></em></sub>and Hill coefficients (<em>h</em>) are listed in the text.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">As previously reported (<a id="xref-ref-13-8" class="xref-bibr" href="#ref-13">13</a>), IFIT1 binds cap0-β-globin mRNA with very high affinity (K<sub>1/2,app</sub>= 40 ± 1.2 nM, <em>h</em> = 2.8 ± 0.2). Complexing with IFIT3 or the IFIT2:IFIT3 heterodimer reduces the K<sub>1/2,app</sub> of IFIT1 on this mRNA to 19 ± 0.8 nM ( <em>h</em> = 2.5 ± 0.2) and 19 ± 0.7 nM ( <em>h</em> = 1.98 ± 0.1), respectively. IFIT1 heterocomplexes also has the effect of saturating the binding on the mRNA as evident in the autoradiograms. This effect was even more pronounced when the cap0-ZV reporter was analysed. On this more structured RNA the K<sub>1/2,app</sub> of IFIT1 binding was 69 ± 1.8 nM ( <em>h</em> = 3.5 ± 0.3) and only reached 60% saturation. In contrast, when complexed with IFIT3 or the IFIT2:IFIT3 heterodimer IFIT1 bound the ZV reporter with a K<sub>1/2,app</sub> of 29 ± 1.1 nM ( <em>h</em> = 4.1 ± 0.6) and 58 ± 2.3 nM ( <em>h</em> = 3.1 ± 0.32), respectively. Again, as is clear from the autoradiograms, addition of IFIT3 or the IFIT2:IFIT3 heterodimer led to saturation of RNA binding. In all cases, and similar to our previous findings (<a id="xref-ref-13-9" class="xref-bibr" href="#ref-13">13</a>), the Hill coefficient was greater than 1, indicating a degree of cooperativity in IFIT1 cap0-mRNA binding.</p></div><div id="sec-22" class="subsection"><h3>IFIT1 and IFIT3 interact through a C-terminal motif</h3><p id="p-42">Our data reveal that IFIT3 or the IFIT2:IFIT3 heterodimer can stimulate cap0 mRNA binding, and thus translation inhibition, by IFIT1. We next sought to identify how IFIT1 and IFIT3 interact. Murine Ifit3, which does not precipitate with murine Ifit1b1 (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>), has a large deletion at the C terminus (Figure S5) when compared to human IFIT3. As a result of this deletion, mouse Ifit3 lacks a ‘YxxxL’ structural motif present in both human IFIT3 and IFIT1 (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A and B</a>) recently reported to promote IFIT1 concentration dependent dimerisation (BioRxiv: <a href="https://doi.org/10.1101/152850">https://doi.org/10.1101/152850</a>). Our SEC-MALS analysis demonstrates that the IFIT1:IFIT3 interaction is more stable than the IFIT1:IFIT1 interaction (compare <a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2A</a> and Figure S2A). We therefore hypothesised that the proposed IFIT1 dimerisation motif is the site of interaction between IFIT1 and IFIT3.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261776/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mutational analysis reveals a key IFIT1:IFIT3 interaction motif. (A)Crystal structure of wild type IFIT1 (PDB: 5W5H). The two chains in the asymmetric unit are coloured yellow and green with m7GpppAAAA bound in the cap-binding pocket highlighted in orange. The interface region between the two IFIT1 molecules with Y460 and L464 side chains is enlarged (only labelled for the chain coloured green). (B) Sequence alignment of the C-terminal regions of IFIT1 and IFIT3 generated by Clustal Omega. The &#x2018;YxxxL&#x2019; motif is boxed. (C, D, E and F) UV280 absorbance traces of SEC analysis (SuperdexS200 Increase 10/300 column) of wild type (WT) and mutant IFIT1 alone or incubated with IFIT3. Gel insets below each trace show SDS-PAGE analysis of each run. ( D, E and F) Protein gel lanes and corresponding peaks are indicated by lower case letters. The position of IFIT1 and IFIT3 on the protein gels is indicated. The Y460E/L464E+IFIT3 trace IFIT1+IFIT3 is shown in grey shadow for reference. (G) Luciferase activity from RRL incubated with cap0--globin Fluc RNA and WT or mutant IFIT1 with or without IFIT3 as indicated. Data are normalised to the luciferase activity in the absence of IFIT1 and shown as the mean the standard error of three separate experiments. Statistical analysis was performed comparing conditions joined by dotted lines using an unpaired, two-tailed Students T-test. P values are indicated, and * denotes statistical significance." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-815547595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Mutational analysis reveals a key IFIT1:IFIT3 interaction motif. (&lt;strong&gt;A&lt;/strong&gt;)Crystal structure of wild type IFIT1 (PDB: 5W5H). The two chains in the asymmetric unit are coloured yellow and green with m7GpppAAAA bound in the cap-binding pocket highlighted in orange. The interface region between the two IFIT1 molecules with Y460 and L464 side chains is enlarged (only labelled for the chain coloured green). (&lt;strong&gt;B&lt;/strong&gt;) Sequence alignment of the C-terminal regions of IFIT1 and IFIT3 generated by Clustal Omega. The &#x2018;YxxxL&#x2019; motif is boxed. (&lt;strong&gt;C&lt;/strong&gt;, &lt;strong&gt;D, E&lt;/strong&gt; and &lt;strong&gt;F&lt;/strong&gt;) UV&lt;sub&gt;280&lt;/sub&gt; absorbance traces of SEC analysis (SuperdexS200 Increase 10/300 column) of wild type (WT) and mutant IFIT1 alone or incubated with IFIT3. Gel insets below each trace show SDS-PAGE analysis of each run. ( &lt;strong&gt;D, E&lt;/strong&gt; and &lt;strong&gt;F&lt;/strong&gt;) Protein gel lanes and corresponding peaks are indicated by lower case letters. The position of IFIT1 and IFIT3 on the protein gels is indicated. The Y460E/L464E+IFIT3 trace IFIT1+IFIT3 is shown in grey shadow for reference. (&lt;strong&gt;G&lt;/strong&gt;) Luciferase activity from RRL incubated with cap0--globin Fluc RNA and WT or mutant IFIT1 with or without IFIT3 as indicated. Data are normalised to the luciferase activity in the absence of IFIT1 and shown as the mean the standard error of three separate experiments. Statistical analysis was performed comparing conditions joined by dotted lines using an unpaired, two-tailed Students T-test. P values are indicated, and * denotes statistical significance.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261776/F5.medium.gif" width="355" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261776/F5.medium.gif" width="355" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261776/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261776/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <p id="p-43" class="first-child">Mutational analysis reveals a key IFIT1:IFIT3 interaction motif. (<strong>A</strong>)Crystal structure of wild type IFIT1 (PDB: 5W5H). The two chains in the asymmetric unit are coloured yellow and green with m7GpppAAAA bound in the cap-binding pocket highlighted in orange. The interface region between the two IFIT1 molecules with Y460 and L464 side chains is enlarged (only labelled for the chain coloured green). (<strong>B</strong>) Sequence alignment of the C-terminal regions of IFIT1 and IFIT3 generated by Clustal Omega. The ‘YxxxL’ motif is boxed. (<strong>C</strong>, <strong>D, E</strong> and <strong>F</strong>) UV<sub>280</sub> absorbance traces of SEC analysis (SuperdexS200 Increase 10/300 column) of wild type (WT) and mutant IFIT1 alone or incubated with IFIT3. Gel insets below each trace show SDS-PAGE analysis of each run. ( <strong>D, E</strong> and <strong>F</strong>) Protein gel lanes and corresponding peaks are indicated by lower case letters. The position of IFIT1 and IFIT3 on the protein gels is indicated. The Y460E/L464E+IFIT3 trace IFIT1+IFIT3 is shown in grey shadow for reference. (<strong>G</strong>) Luciferase activity from RRL incubated with cap0--globin Fluc RNA and WT or mutant IFIT1 with or without IFIT3 as indicated. Data are normalised to the luciferase activity in the absence of IFIT1 and shown as the mean the standard error of three separate experiments. Statistical analysis was performed comparing conditions joined by dotted lines using an unpaired, two-tailed Students T-test. P values are indicated, and * denotes statistical significance.</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">We generated a panel of IFIT1 mutants based on the IFIT1 dimer crystal structure (PDB: 5W5H). The point mutants Y460E and L464E expressed and purified similarly to the wild type protein. Both mutants eluted as monomeric species during SEC (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5C</a>) consistent with disruption of IFIT1 homodimerisation as previously reported (BioRxiv: <a href="https://doi.org/10.1101/152850)">https://doi.org/10.1101/152850)</a>. However, the L464E substitution only had a modest effect on the interaction of IFIT1 with IFIT3 (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5D</a>). While a small portion of IFIT1-L464E eluted as a monomer, a stable tetramer with IFIT3 was the predominant species, like the wild type IFIT1 shown in <a id="xref-fig-2-12" class="xref-fig" href="#F2">Figure 2A</a>. The Y460E substitution destabilises IFIT1:IFIT3 oligomerisation to a greater extent than the L464E substitution, with less IFIT1-Y460E sequestered to the IFIT1:IFIT3 complex (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5E</a>) but, as is clear in the SDS-PAGE analysis of this experiment (gel inset), IFIT1-Y460E can still associate with IFIT3. We therefore generated a Y460E/L464E double mutant of IFIT1 (IFIT1-YL). IFIT1-YL eluted as a monomeric species on SEC, similarly to the Y460E and L464E single mutants (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5C</a>). However, in contrast to these two single point mutants, IFIT1-YL did not interact with IFIT3 during SEC (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5F</a>).</p><p id="p-45">Disruption of dimerisation was previously reported not to affect the translation inhibition activity of IFIT1 (BioRxiv: <a href="https://doi.org/10.1101/152850">https://doi.org/10.1101/152850</a> and <a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5G</a>, white bars). Therefore, we examined what impact mutations in the ‘YxxxL’ motif had on the ability of IFIT3 to stimulate IFIT1 translation inhibition activity. IFIT1 mutants were combined with IFIT3, reporter mRNA and RRL as described in Materials and Methods and Fluc dependent luminescence was measured (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5G</a>). At 40 nM IFIT1, ‘YxxxL’ motif mutants displayed similar translation inhibition. IFIT3 significantly enhanced translation inhibition of both wild type IFIT1 and the IFIT1-L464E mutant. In contrast, IFIT3 did not stimulate translation inhibition of either the IFIT1-Y460E mutant or the L464E/Y460E double mutant.</p></div></div><div class="section" id="sec-23"><h2 class="">Discussion</h2><p id="p-46">The IFIT family of ISGs are among the most highly upregulated proteins during cellular response to viral infection and, while a role for IFITs in regulating translation has long been postulated, the mechanisms by which these proteins function are only recently being revealed. Here, we have presented, to our knowledge, the first <em>in vitro</em> reconstitution of the IFIT1:IFIT2:IFIT3 complex. This has enabled us to examine the impact of hetero oligomerisation on RNA recognition by IFIT1 and to begin to understand how this complex assembles.</p><div id="sec-24" class="subsection"><h3>IFIT oligomerisation</h3><p id="p-47">We initially confirmed that IFIT1, IFIT2 and IFIT3 interact in lysates from HEK293T cells using proteomics. In contrast to previous studies (<a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">14</a>) we used nuclease treated samples confirming that this interaction occurs in cells in an RNA-independent manner. Moreover, we confirmed that IFIT1 could interact directly with IFIT2 or IFIT3 by SEC. We then employed SEC-MALS to examine IFIT heterocomplex assembly for the first time. Unlike standard SEC, when coupled with protein gel analysis SEC-MALS enables reliable identification of oligomeric status. <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> shows schematic representations of each of the IFIT complexes reconstituted <em>in vitro</em> in this study. Consistent with previous reports ((<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>) and BioRxiv: <a href="https://doi.org/10.1101/152850)">https://doi.org/10.1101/152850)</a>, IFIT1 was found to reversibly dimerise in a concentration dependent manner (Figure S2A). While both IFIT2 and IFIT3 individually form stable homodimers as previously reported (<a id="xref-ref-13-10" class="xref-bibr" href="#ref-13">13</a>, <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">7</a>), an unexpected tetrameric form of IFIT2 was also detected (Figure S2B and C). Surprisingly, we found that the IFIT1:IFIT3 interaction is more stable than the IFIT1:IFIT1 complex (compare <a id="xref-fig-2-13" class="xref-fig" href="#F2">Figures 2A</a> and S2A). When analysed by SEC-MALS the predominant IFIT1:IFIT3 oligomeric state was a stable tetramer containing equimolar amounts of both proteins. The IFIT2 dimer eluted from the SEC at a volume expected for a monomeric species (Figure S2B) and as such had a similar elution volume as IFIT1. Very weak interaction of IFIT1 and IFIT2 was detected by SEC when these proteins were incubated at 4° C (<em>RF and TS unpublished data</em>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261776/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic representation of the IFIT complexes analysed in this study. Cartoons depicting IFIT1 (yellow), IFIT2 (green) and IFIT3 (blue) complexes reconstituted in vitro from individually purified proteins. Weak interactions are indicated by reversible arrows. IFIT1 and IFIT2 structures and dimerisation interactions have been characterised by X-ray crystallography (BioRxiv: https://doi.org/10.1101/152850 and (30)). No experimentally derived structure for IFIT3 homo- or heterodimers are currently available. Therefore, IFIT3 and IFIT2:IFIT3 dimerisation, and IFIT1 heterooligomerisation are modelled on the IFIT1 dimer (PDB: 5W5H) and IFIT2 dimer (PDB: 4G1T) crystal structures and supported by experimental evidence as discussed in the text. The association of IFIT1 and IFIT2 is less well defined as indicated by the question marks." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-815547595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Schematic representation of the IFIT complexes analysed in this study. Cartoons depicting IFIT1 (yellow), IFIT2 (green) and IFIT3 (blue) complexes reconstituted &lt;em&gt;in vitro&lt;/em&gt; from individually purified proteins. Weak interactions are indicated by reversible arrows. IFIT1 and IFIT2 structures and dimerisation interactions have been characterised by X-ray crystallography (BioRxiv: https://doi.org/10.1101/152850 and (30)). No experimentally derived structure for IFIT3 homo- or heterodimers are currently available. Therefore, IFIT3 and IFIT2:IFIT3 dimerisation, and IFIT1 heterooligomerisation are modelled on the IFIT1 dimer (PDB: 5W5H) and IFIT2 dimer (PDB: 4G1T) crystal structures and supported by experimental evidence as discussed in the text. The association of IFIT1 and IFIT2 is less well defined as indicated by the question marks.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261776/F6.medium.gif" width="282" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261776/F6.medium.gif" width="282" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261776/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/02/08/261776/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6.</span> <p id="p-48" class="first-child">Schematic representation of the IFIT complexes analysed in this study. Cartoons depicting IFIT1 (yellow), IFIT2 (green) and IFIT3 (blue) complexes reconstituted <em>in vitro</em> from individually purified proteins. Weak interactions are indicated by reversible arrows. IFIT1 and IFIT2 structures and dimerisation interactions have been characterised by X-ray crystallography (BioRxiv: <a href="https://doi.org/10.1101/152850">https://doi.org/10.1101/152850</a> and (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a>)). No experimentally derived structure for IFIT3 homo- or heterodimers are currently available. Therefore, IFIT3 and IFIT2:IFIT3 dimerisation, and IFIT1 heterooligomerisation are modelled on the IFIT1 dimer (PDB: 5W5H) and IFIT2 dimer (PDB: 4G1T) crystal structures and supported by experimental evidence as discussed in the text. The association of IFIT1 and IFIT2 is less well defined as indicated by the question marks.</p><div class="sb-div caption-clear"></div></div></div><p id="p-49">However, when the incubation temperature was increased to 30 °C multiple oligomeric states of the IFIT1:IFIT2 complex were present (Figure S2D). These data suggest that IFIT1 interacts with IFIT2 in a different manner to IFIT3. Results of previous attempts to narrow down the IFIT1:IFIT2 binding interface using co-precipitation from lysates overexpressing IFIT2 truncation mutants were ambiguous and suggested a lack of specificity in the interaction (<a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">29</a>).</p><p id="p-50">Our SILAC data indicated that IFIT2 and IFIT3 were enriched to a similar degree in IFIT1-bound samples (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>), suggesting that a heterocomplex of IFIT2:IFIT3 was the more likely form to interact with IFIT1 in the cell. This prompted us to investigate the IFIT2:IFIT3 interaction in more detail. The delayed elution of the dimeric IFIT2 from the size exclusion column fortuitously enabled us to analyse the IFIT2:IFIT3 interaction by SEC-MALS. Although no interaction between these two proteins was detected when incubated at 4 °C, we observed the appearance of a heterodimeric complex when these proteins were incubated at 37 °C (<a id="xref-fig-2-14" class="xref-fig" href="#F2">Figure 2B</a>) indicating that addition of energy was required for this heterocomplex to form. The crystal structure of IFIT2 revealed that dimerisation of the protein was due to a domain swap of three α-helices that constitute one and a half TPRs in the N-terminal domain (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">7</a>) (Figure S6A and shown schematically in <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). Nucleic acid binding activity of IFIT2 was localised to the dimer interface surfaces of the C-terminal domain that form a large positively charged pocket (Figure S6B). The electrostatic surface potential of an IFIT3 molecular model, based on the IFIT2 crystal structure, is shown in Figure S6C. The IFIT3 model lacks a positively-charged nucleic acid binding surface like that of IFIT2.</p><p id="p-51">The ability of IFIT3 to sequester IFIT2 into a heterodimeric form demonstrates that IFIT3 can disrupt the domain swapped architecture of IFIT2. It is therefore possible that IFIT3 could interact with IFIT2 in a similar domain swapped manner (modelled schematically in <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>, <em>top right</em>), consistent with TPRs 1-4 in the N terminus of IFIT2 being sufficient for the interaction as reported previously (<a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">29</a>). Results of SEC analysis and differential scanning fluorimetry experiments indicate that the IFIT2:IFIT3 complex is more stable than separate IFIT2 and IFT3 homodimers (<a id="xref-fig-2-15" class="xref-fig" href="#F2">Figure 2C and D</a>). The reason for this is currently unclear but comparison of the thermal melt curves of the three complexes shows that unlike the IFIT2 and IFIT3 homodimers, unfolding of the IFIT2:IFIT3 complex is biphasic (<a id="xref-fig-2-16" class="xref-fig" href="#F2">Figure 2D</a>). A possible explanation for this is that the IFIT2 and IFIT3 proteins unfold uniformly, whereas the interacting regions of the IFIT2:IFIT3 heterocomplex are stabilised and unfold at different rates to the rest of the protein domains in the complex. The IFIT2:IFIT3 complex does not display unfolding kinetics averaged between those observed for IFIT2 and IFIT3 suggesting that the heterocomplex is indeed more stable than each of its constituent parts alone. High-resolution structures of the IFIT2:IFIT3 complex are required to confirm how these proteins interact and are a focus of our ongoing investigation.</p><p id="p-52">Our IFIT2:IFIT3 interaction studies have important implications for our understanding of IFIT complex assembly and biology. Overexpression of IFIT2 was previously reported to induce apoptosis (<a id="xref-ref-29-6" class="xref-bibr" href="#ref-29">29</a>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>), while co-expression with IFIT3 but not IFIT1 blocked this effect (<a id="xref-ref-29-7" class="xref-bibr" href="#ref-29">29</a>). Moreover, depletion of IFIT3 induced cell death in the U549 human carcinoma cell line, an effect potentiated by co-infection with Dengue virus (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>). Our data would suggest that the homodimeric form of IFIT2 may be responsible for this phenotype, and that co-expression of IFIT3 may mitigate this effect by disrupting IFIT2 dimerisation. It is currently not clear why the cell would evolve such a mechanism for inducing programmed cell death. One potential hypothesis is that dysregulation of ISG induction, perhaps due to infection, could perturb the balance of IFIT2 and IFIT3, promoting cell death to restrict pathogen spread.</p><p id="p-53">Combining IFIT1 in equimolar amounts or in two-fold excess with the purified IFIT2:IFIT3 heterodimer resulted in the formation of heterotrimeric or heterotetrameric IFIT complexes, respectively (<a id="xref-fig-2-17" class="xref-fig" href="#F2">Figure 2E and F</a> and shown schematically in <a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6</a>, <em>bottom row</em>). Importantly, this complex closely resembles the IFIT1:IFIT2:IFIT3-containing 150-200 kDa complex previously identified in HeLa cell lysates (<a id="xref-ref-29-8" class="xref-bibr" href="#ref-29">29</a>), supporting its biological relevance. Unlike the IFIT1:IFIT3 tetramer, the IFIT1:IFIT2:IFIT3 tetramer was unstable and precipitated during incubation at 4 °C. In contrast, the heterotrimer was stable and enabled us to investigate the impact of IFIT2:IFIT3 complexing on the translation inhibition activity of IFIT1 (discussed below). A recent report described the crystal structure of wild type IFIT1 and revealed a dimerisation interface between C-terminal TPRs (BioRxiv: <a href="https://doi.org/10.1101/152850)">https://doi.org/10.1101/152850)</a>, responsible for the concentration dependent homo dimerisation of the protein, confirmed here. Importantly, mutation of this interface had no impact on the translation inhibition activity of IFIT1 (BioRxiv: <a href="https://doi.org/10.1101/152850">https://doi.org/10.1101/152850</a> and <a id="xref-fig-5-9" class="xref-fig" href="#F5">Figure 5E</a>) and <a id="xref-fig-5-10" class="xref-fig" href="#F5">Figure 5E</a>). We observed that IFIT3 shares multiple key residues involved in the IFIT1 ‘YxxxL’ dimerisation motif, also in a C-terminal TPR. We therefore hypothesized that this motif is the site of interaction between IFIT1 and IFIT3. Mutational analysis of IFIT1 confirmed that this motif is indeed required for interaction with IFIT3 (<a id="xref-fig-5-11" class="xref-fig" href="#F5">Figure 5</a>). The double-mutant IFIT1-Y460E/L464E did not interact with IFIT3 when analysed by SEC. Since the wild type IFIT1 homodimer is less stable than the IFIT1:IFIT3 complex, as shown by our SEC-MALS analysis, we propose that the functionally-important role of the ‘YxxxL’ motif in humans is mediating interaction between IFIT1 and IFIT3 (shown schematically in <a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6</a>). Importantly, the crystal structure of IFIT5 reveals that, although critical interacting residues may be conserved, they are buried in an interface with a terminal helix not present in IFIT1 (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">8</a>–<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">10</a>), explaining why IFIT5 does not interact with IFIT3 ((<a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">12</a>) and <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>). While the ‘YxxxL’ motif is conserved in mouse Ifit1b1, Ifit3 is truncated such that the ‘YxxxL’ motif is absent. In mice and other rodents, the Ifit1b1 gene has been duplicated (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">6</a>) and it is possible that these extra IFITs could compensate for the disrupted murine Ifit1b1-Ifit3 interaction. This suggests species differences in the role of this motif, and in IFIT oligomerisation in general, that must be considered when examining phenotypes of small animal models used to examine the impact of IFIT depletion.</p></div><div id="sec-25" class="subsection"><h3>Impact of oligomerisation on IFIT1 cap0 binding and translation inhibition</h3><p id="p-54">Our <em>in vitro</em> reconstitution of the human IFIT heterocomplex enabled us to examine the impact of oligomerisation on the cap0 mRNA binding and translation inhibition activity of IFIT1. We opted to use an RRL based translation inhibition assay system as this was previously used to demonstrate differential inhibitory effects of IFIT1 on mRNAs from parainfluenza virus 5 (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">22</a>). We used two reporters, one with a well characterised (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>), highly structured 5' UTR (ZV, <em>dG</em>=-33.4 kcal/mol) and one with very little structure (β-globin, <em>dG</em>=-10.5 kcal/mol), upstream of a Fluc reporter and with the corresponding 3' UTRs. IFIT1 strongly inhibited translation of both reporters (<a id="xref-fig-3-11" class="xref-fig" href="#F3">Figure 3</a>), but had a stronger effect on the less-structured β-globin reporter, consistent with an emerging consensus that both the methylation state and RNA structure at the 5' end of an mRNA can influence its susceptibility to IFIT1 inhibition (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">9</a>, <a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>). IFIT2 had no detectable impact on the ability of IFIT1 to inhibit translation in the RRL system (<a id="xref-fig-3-12" class="xref-fig" href="#F3">Figure 3C</a>). In contrast, IFIT3 and to a lesser extent the IFIT2:IFIT3 heterodimer enhanced the translation inhibition effect of IFIT1 on both reporters (<a id="xref-fig-3-13" class="xref-fig" href="#F3">Figure 3C-F</a>).</p><p id="p-55">It is possible that the RNA binding surface of IFIT1 may be extended by interaction with its binding partners. This has the potential to expand the repertoire of RNAs IFIT1 can interact with by providing additional stabilising interactions. Moreover, it is unclear why inclusion of IFIT3 alone has a stronger effect on translation inhibition than inclusion of the IFIT2:IFIT3 heterodimer. Since inclusion of IFIT2 alone had no impact on the translation inhibition activity of IFIT1 it is possible that the stimulatory effect is a result of the IFIT3 in the IFIT2:IFIT3 complex. The physiological role for this stimulation of binding is not immediately clear as we have previously demonstrated that IFIT1 binding to cap0-mRNA is sufficient to prevent association of the cap-binding translation initiation factor, eIF4F, <em>in vitro</em> (<a id="xref-ref-13-11" class="xref-bibr" href="#ref-13">13</a>). However, a requirement for co-expression of other IFITs for the full anti-viral activity of IFIT1 has been reported (<a id="xref-ref-12-7" class="xref-bibr" href="#ref-12">12</a>). During infection the enhanced binding of IFIT1, as a result of interaction with IFIT3, may promote IFIT1-dependent restriction in regions where access to cellular proteins is limited, such as in viral replication complexes.</p><p id="p-56">While multiple cellular factors such as competing cap-binding proteins complicate analysis in cells, our <em>in vitro</em> mRNA binding assays reveal a clear cap0-mRNA binding advantage conferred by IFIT1 heterooligomerisation. When examined in our reverse transcriptase inhibition assays apparent IFIT1 binding was much less efficient for the structured cap0-ZV reporter than for the less structured - globin mRNA, and in fact on the ZV construct failed to reach saturation (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). This is not due to incomplete capping of the mRNA as saturation is possible when IFIT3 is present. Instead, it is more likely that the reverse transcriptase may remove a proportion of the bound IFIT1 as it proceeds in a 5 to 3 direction resulting in the production of a full-length signal even if IFIT1 was initially bound. We conclude therefore that IFIT3, and the IFIT2:IFIT3 heterodimer, stabilise the interaction of IFIT1 with the cap0 mRNA in such a way that it is no longer removed by the reverse transcriptase. Interestingly, there was no evidence of a shift in the IFIT dependent toeprint to indicate that the other IFITs in the complex were interacting with the mRNA downstream of the IFIT1 cap0 binding cleft. IFIT5 changes conformation when transitioning between the apo- and RNA-bound state (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">8</a>), positioning key residues for optimal RNA binding. The crystal structures of IFIT1 bound to different short RNAs show the protein is in a similar closed conformation as the RNA-bound structure of IFIT5 (<a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">9</a>). It is therefore likely that IFIT1 cycles through a similar open/closed conformation to interact with RNA. Binding of IFIT3 may promote the closed conformation of IFIT1, stabilising its interaction with target RNAs. Future efforts will focus on obtaining high-resolution structures of IFIT1:IFIT3 and IFIT1:IFIT2:IFIT3 complexes with and without RNA to address these questions.</p><p id="p-57">Together, our results demonstrate the first <em>in vitro</em> reconstitution of the human IFIT2:IFIT3 and IFIT1:IFIT2:IFIT3 heterocomplexes, provide novel details about IFIT complex assembly and reveal for the first time that IFIT3 enhances IFIT1 cap0-mRNA binding and translation inhibition. Our reconstituted complexes provide a solid foundation for future molecular analysis of IFIT assembly and function.</p></div></div><div class="section" id="sec-26"><h2 class="">Funding</h2><p id="p-58">This work was supported by a joint Royal Society/Wellcome Trust Sir Henry Dale Fellowship (202471/Z/16/Z) and a Royal Society Research Grant (RG140708) to TRS. SCG is a Sir Henry Dale Fellow (098406/Z/12/Z) co-funded by the Wellcome Trust and Royal Society. HVM is supported by a University of Cambridge, Department of Pathology PhD studentship. RCF and DSM are supported by CAPES Computational Biology (23038.010048/2013-27). DSM is also supported by the Academy of Medical Sciences/UK (NAF004/1005).</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-59">We are grateful to Janet Deane for providing access to equipment for SEC-MALS experiments and to George Katibah and Kathleen Collins for the gift of the IFIT1 expression plasmid.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-2">The authors wish it to be known that, in their opinion, the first two authors should be regarded as Joint First Authors</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-261776v1.1" data-doi="10.1099/vir.0.83391-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Randall</span>, <span class="cit-name-given-names">R.E.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Goodbourn</span>, <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">Interferons and viruses: An interplay between induction, signalling, antiviral responses and virus countermeasures</span>. <abbr class="cit-jnl-abbrev">J. Gen. Virol.</abbr>,<span class="cit-vol">89</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DRandall%26rft.auinit1%253DR.%2BE.%26rft.volume%253D89%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D47%26rft.atitle%253DInterferons%2Band%2Bviruses%253A%2Ban%2Binterplay%2Bbetween%2Binduction%252C%2Bsignalling%252C%2Bantiviral%2Bresponses%2Band%2Bvirus%2Bcountermeasures%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.83391-0%26rft_id%253Dinfo%253Apmid%252F18089727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.83391-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18089727&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252700900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.2" data-doi="10.1038/nature09907"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schoggins</span>, <span class="cit-name-given-names">J.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilson</span>, <span class="cit-name-given-names">S.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Panis</span>, <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murphy</span>, <span class="cit-name-given-names">M.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>, <span class="cit-name-given-names">C.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bieniasz</span>, <span class="cit-name-given-names">P.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Rice</span>, <span class="cit-name-given-names">C.M.</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">A diverse range of gene products are effectors of the type i interferon antiviral response</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">472</span>, <span class="cit-fpage">481</span>–<span class="cit-lpage">485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSchoggins%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D472%26rft.issue%253D7344%26rft.spage%253D481%26rft.epage%253D485%26rft.atitle%253DA%2Bdiverse%2Brange%2Bof%2Bgene%2Bproducts%2Bare%2Beffectors%2Bof%2Bthe%2Btype%2BI%2Binterferon%2Bantiviral%2Bresponse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09907%26rft_id%253Dinfo%253Apmid%252F21478870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09907&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478870&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289949600036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">3.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-261776v1.3" data-doi="10.7150/ijbs.5613"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>, <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Michal</span>, <span class="cit-name-given-names">J.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>, <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ding</span>, <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lunney</span>, <span class="cit-name-given-names">J.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>, <span class="cit-name-given-names">B.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Jiang</span>, <span class="cit-name-given-names">Z.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Interferon induced IFIT family genes in host antiviral defense</span>. <abbr class="cit-jnl-abbrev">Int. J. Biol. Sci.</abbr>, <span class="cit-vol">9</span>, <span class="cit-fpage">200</span>–<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BBiol.%2BSci.%26rft.volume%253D9%26rft.spage%253D200%26rft_id%253Dinfo%253Adoi%252F10.7150%252Fijbs.5613%26rft_id%253Dinfo%253Apmid%252F23459883%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7150/ijbs.5613&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23459883&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.4" data-doi="10.1128/JVI.76.11.5532-5539.2002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grandvaux</span>, <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Servant</span>, <span class="cit-name-given-names">M.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">TenOever</span>, <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sen</span>, <span class="cit-name-given-names">G.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Balachandran</span>, <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barber</span>, <span class="cit-name-given-names">G.N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>, <span class="cit-name-given-names">R.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Hiscott</span>, <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2002</span>) <span class="cit-article-title">Transcriptional profiling of interferon regulatory factor 3 target genes: direct involvement in the regulation of interferon-stimulated genes</span>. <abbr class="cit-jnl-abbrev">J. Virol.</abbr>, <span class="cit-vol">76</span>, <span class="cit-fpage">5532</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DGrandvaux%26rft.auinit1%253DN.%26rft.volume%253D76%26rft.issue%253D11%26rft.spage%253D5532%26rft.epage%253D5539%26rft.atitle%253DTranscriptional%2BProfiling%2Bof%2BInterferon%2BRegulatory%2BFactor%2B3%2BTarget%2BGenes%253A%2BDirect%2BInvolvement%2Bin%2Bthe%2BRegulation%2Bof%2BInterferon-Stimulated%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.76.11.5532-5539.2002%26rft_id%253Dinfo%253Apmid%252F11991981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI3Ni8xMS81NTMyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDIvMDgvMjYxNzc2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>, <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>, <span class="cit-name-given-names">Y.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>, <span class="cit-name-given-names">T.K.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Gui</span>, <span class="cit-name-given-names">J.F.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Lineage-Specific Expansion of IFIT Gene Family: An Insight into Coevolution with IFN Gene Family</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>, <span class="cit-vol">8</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Daugherty</span>, <span class="cit-name-given-names">M.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schaller</span>, <span class="cit-name-given-names">A.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Geballe</span>, <span class="cit-name-given-names">A.P.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Malik</span>, <span class="cit-name-given-names">H.S.</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Evolution-guided functional analyses reveal diverse antiviral specificities encoded by ifit1 genes in mammals</span>. <abbr class="cit-jnl-abbrev">Elife</abbr>,<span class="cit-vol">5</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.7" data-doi="10.1038/cr.2012.111"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>, <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liang</span>, <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>, <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>, <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>, <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ye</span>, <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>, <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fleming</span>, <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shu</span>, <span class="cit-name-given-names">H.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Liu</span>, <span class="cit-name-given-names">Y.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Crystal structure of ISG54 reveals a novel RNA binding structure and potential functional mechanisms</span>. <abbr class="cit-jnl-abbrev">Cell Res.</abbr>,<span class="cit-vol">22</span>, <span class="cit-fpage">1328</span>–<span class="cit-lpage">1338</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Bresearch%26rft.stitle%253DCell%2BRes%26rft.aulast%253DYang%26rft.auinit1%253DZ.%26rft.volume%253D22%26rft.issue%253D9%26rft.spage%253D1328%26rft.epage%253D1338%26rft.atitle%253DCrystal%2Bstructure%2Bof%2BISG54%2Breveals%2Ba%2Bnovel%2BRNA%2Bbinding%2Bstructure%2Band%2Bpotential%2Bfunctional%2Bmechanisms.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2012.111%26rft_id%253Dinfo%253Apmid%252F22825553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/cr.2012.111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22825553&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308611900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.8" data-doi="10.1038/nature11783"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Abbas</span>, <span class="cit-name-given-names">Y.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pichlmair</span>, <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Górna</span>, <span class="cit-name-given-names">M.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Superti-Furga</span>, <span class="cit-name-given-names">G.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Nagar</span>, <span class="cit-name-given-names">B.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Structural basis for viral 5’-PPP-RNA recognition by human IFIT proteins</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>,<span class="cit-vol">494</span>, <span class="cit-fpage">60</span>–<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D494%26rft.spage%253D60%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11783%26rft_id%253Dinfo%253Apmid%252F23334420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11783&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23334420&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000314741200033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.9" data-doi="10.1073/pnas.1612444114"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Abbas</span>, <span class="cit-name-given-names">Y.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Laudenbach</span>, <span class="cit-name-given-names">B.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martínez-Montero</span>, <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cencic</span>, <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Habjan</span>, <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pichlmair</span>, <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Damha</span>, <span class="cit-name-given-names">M.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pelletier</span>, <span class="cit-name-given-names">J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Nagar</span>, <span class="cit-name-given-names">B.</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Structure of human IFIT1 with capped RNA reveals adaptable mRNA binding and mechanisms for sensing N1 and N2 ribose 2’-O methylations</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci.</abbr>, <span class="cit-vol">114</span>, <span class="cit-fpage">E2106</span>–<span class="cit-lpage">E2115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1612444114%26rft_id%253Dinfo%253Apmid%252F28251928%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE0LzExL0UyMTA2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDIvMDgvMjYxNzc2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.10" data-doi="10.1016/j.molcel.2012.12.015"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Katibah</span>, <span class="cit-name-given-names">G.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>, <span class="cit-name-given-names">H.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huizar</span>, <span class="cit-name-given-names">J.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vogan</span>, <span class="cit-name-given-names">J.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alber</span>, <span class="cit-name-given-names">T.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Collins</span>, <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">TRNA Binding, Structure, and Localization of the Human Interferon-Induced Protein IFIT5</span>. <abbr class="cit-jnl-abbrev">Mol. Cell</abbr>,<span class="cit-vol">49</span>, <span class="cit-fpage">743</span>–<span class="cit-lpage">750</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D49%26rft.spage%253D743%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2012.12.015%26rft_id%253Dinfo%253Apmid%252F23317505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2012.12.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23317505&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000315555200017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.11" data-doi="10.1007/s12192-010-0248-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kenneth Allan</span>, <span class="cit-name-given-names">R.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Ratajczak</span>, <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Versatile TPR domains accommodate different modes of target protein recognition and function</span>. <abbr class="cit-jnl-abbrev">Cell Stress Chaperones</abbr>,<span class="cit-vol">16</span>, <span class="cit-fpage">353</span>–<span class="cit-lpage">367</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Bstress%2B%2526%2Bchaperones%26rft.stitle%253DCell%2BStress%2BChaperones%26rft.aulast%253DAllan%26rft.auinit1%253DR.%2BK.%26rft.volume%253D16%26rft.issue%253D4%26rft.spage%253D353%26rft.epage%253D367%26rft.atitle%253DVersatile%2BTPR%2Bdomains%2Baccommodate%2Bdifferent%2Bmodes%2Bof%2Btarget%2Bprotein%2Brecognition%2Band%2Bfunction.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12192-010-0248-0%26rft_id%253Dinfo%253Apmid%252F21153002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12192-010-0248-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21153002&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291883900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.12" data-doi="10.1038/ni.2048"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pichlmair</span>, <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lassnig</span>, <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eberle</span>, <span class="cit-name-given-names">C.-A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Górna</span>, <span class="cit-name-given-names">M.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baumann</span>, <span class="cit-name-given-names">C.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burkard</span>, <span class="cit-name-given-names">T.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bürckstümmer</span>, <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stefanovic</span>, <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krieger</span>, <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bennett</span>, <span class="cit-name-given-names">K.L.</span></span>, <span class="cit-etal">et al.</span>(<span class="cit-pub-date">2011</span>) <span class="cit-article-title">IFIT1 is an antiviral protein that recognizes 5[prime]-triphosphate RNA</span>. <abbr class="cit-jnl-abbrev">Nat. Immunol.</abbr>,<span class="cit-vol">12</span>, <span class="cit-fpage">624</span>–<span class="cit-lpage">630</span> LA–en.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bimmunology%26rft.stitle%253DNat%2BImmunol%26rft.aulast%253DPichlmair%26rft.auinit1%253DA.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D624%26rft.epage%253D630%26rft.atitle%253DIFIT1%2Bis%2Ban%2Bantiviral%2Bprotein%2Bthat%2Brecognizes%2B5%2527-triphosphate%2BRNA.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fni.2048%26rft_id%253Dinfo%253Apmid%252F21642987%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ni.2048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21642987&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.13" data-doi="10.1093/nar/gkt1321"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kumar</span>, <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sweeney</span>, <span class="cit-name-given-names">T.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Skabkin</span>, <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Skabkina</span>, <span class="cit-name-given-names">O. V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hellen</span>, <span class="cit-name-given-names">C.U.T.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Pestova</span>, <span class="cit-name-given-names">T. V.</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Inhibition of translation by IFIT family members is determined by their ability to interact selectively with the 5’-terminal regions of cap0-, cap1- and 5’ppp-mRNAs</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>, <span class="cit-vol">42</span>, <span class="cit-fpage">3228</span>–<span class="cit-lpage">3245</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1321%26rft_id%253Dinfo%253Apmid%252F24371270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt1321&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24371270&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000333093600042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Habjan</span>, <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hubel</span>, <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lacerda</span>, <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Benda</span>, <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holze</span>, <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eberl</span>, <span class="cit-name-given-names">C.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mann</span>, <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kindler</span>, <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gil-Cruz</span>, <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>, <span class="cit-name-given-names">J.</span></span>, <span class="cit-etal">et al.</span>(<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Sequestration by IFIT1 Impairs Translation of 2’O-unmethylated Capped RNA</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr>,<span class="cit-vol">9</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.15" data-doi="10.1128/JVI.00883-13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kimura</span>, <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Katoh</span>, <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kayama</span>, <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saiga</span>, <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Okuyama</span>, <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Okamoto</span>, <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Umemoto</span>, <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matsuura</span>, <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yamamoto</span>, <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Takeda</span>, <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Ifit1 Inhibits Japanese Encephalitis Virus Replication through Binding to 5’ Capped 2’-O Unmethylated RNA</span>. <abbr class="cit-jnl-abbrev">J. Virol.</abbr>,<span class="cit-vol">87</span>, <span class="cit-fpage">9997</span>–<span class="cit-lpage">10003</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00883-13%26rft_id%253Dinfo%253Apmid%252F23824812%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Ny8xOC85OTk3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDIvMDgvMjYxNzc2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">16.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-261776v1.16" data-doi="10.1128/JVI.02806-12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>, <span class="cit-name-given-names">S.-H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dong</span>, <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>, <span class="cit-name-given-names">X.-F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>, <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>, <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Deng</span>, <span class="cit-name-given-names">Y.-Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>, <span class="cit-name-given-names">X.-Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ye</span>, <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>, <span class="cit-name-given-names">S.-Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>, <span class="cit-name-given-names">H.-J.</span></span>, <span class="cit-etal">et al.</span>(<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Rational Design of a Flavivirus Vaccine by Abolishing Viral RNA 2’-O Methylation</span>. <abbr class="cit-jnl-abbrev">J. Virol.</abbr>,<span class="cit-vol">87</span>, <span class="cit-fpage">5812</span>–<span class="cit-lpage">5819</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02806-12%26rft_id%253Dinfo%253Apmid%252F23487465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Ny8xMC81ODEyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDIvMDgvMjYxNzc2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">17.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-261776v1.17" data-doi="10.1038/nature09489"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Daffis</span>, <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Szretter</span>, <span class="cit-name-given-names">K.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schriewer</span>, <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>, <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Youn</span>, <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Errett</span>, <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>, <span class="cit-name-given-names">T.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schneller</span>, <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zust</span>, <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dong</span>, <span class="cit-name-given-names">H.</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">(2010) 2’-O</span> <span class="cit-article-title">methylation of the viral mRNA cap evades host restriction by IFIT family members</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>,<span class="cit-vol">468</span>, <span class="cit-fpage">452</span>–<span class="cit-lpage">456</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDaffis%26rft.auinit1%253DS.%26rft.volume%253D468%26rft.issue%253D7322%26rft.spage%253D452%26rft.epage%253D456%26rft.atitle%253D2%2527-O%2Bmethylation%2Bof%2Bthe%2Bviral%2BmRNA%2Bcap%2Bevades%2Bhost%2Brestriction%2Bby%2BIFIT%2Bfamily%2Bmembers.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09489%26rft_id%253Dinfo%253Apmid%252F21085181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09489&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21085181&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284313100046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">18.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-261776v1.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Szretter</span>, <span class="cit-name-given-names">K.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Daniels</span>, <span class="cit-name-given-names">B.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cho</span>, <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gainey</span>, <span class="cit-name-given-names">M.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yokoyama</span>, <span class="cit-name-given-names">W.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gale</span>, <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Virgin</span>, <span class="cit-name-given-names">H.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klein</span>, <span class="cit-name-given-names">R.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sen</span>, <span class="cit-name-given-names">G.C.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Diamond</span>, <span class="cit-name-given-names">M.S.</span></span> <span class="cit-pub-date">(2012) 2’-O</span> <span class="cit-article-title">methylation of the viral mRNA cap by West Nile virus evades Ifit1-dependent and -independent mechanisms of host restriction in vivo</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr>, <span class="cit-vol">8</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Züst</span>, <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dong</span>, <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>, <span class="cit-name-given-names">X.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chang</span>, <span class="cit-name-given-names">D.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>, <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Balakrishnan</span>, <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Toh</span>, <span class="cit-name-given-names">Y.X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>, <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>, <span class="cit-name-given-names">S.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Deng</span>, <span class="cit-name-given-names">Y.Q.</span></span>, <span class="cit-etal">et al.</span>(<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Rational Design of a Live Attenuated Dengue Vaccine: 2’-O-Methyltransferase Mutants Are Highly Attenuated and Immunogenic in Mice and Macaques</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr>,<span class="cit-vol">9</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.20" data-doi="10.1038/ni.1979"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Züst</span>, <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cervantes-Barragan</span>, <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Habjan</span>, <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maier</span>, <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neuman</span>, <span class="cit-name-given-names">B.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>, <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Szretter</span>, <span class="cit-name-given-names">K.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baker</span>, <span class="cit-name-given-names">S.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barchet</span>, <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Diamond</span>, <span class="cit-name-given-names">M.S.</span></span>, <span class="cit-etal">et al.</span>(<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Ribose 2’-O-methylation provides a molecular signature for the distinction of self and non-self mRNA dependent on the RNA sensor Mda5</span>. <abbr class="cit-jnl-abbrev">Nat. Immunol.</abbr>,<span class="cit-vol">12</span>, <span class="cit-fpage">137</span>–<span class="cit-lpage">143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bimmunology%26rft.stitle%253DNat%2BImmunol%26rft.aulast%253DZust%26rft.auinit1%253DR.%26rft.volume%253D12%26rft.issue%253D2%26rft.spage%253D137%26rft.epage%253D143%26rft.atitle%253DRibose%2B2%2527-O-methylation%2Bprovides%2Ba%2Bmolecular%2Bsignature%2Bfor%2Bthe%2Bdistinction%2Bof%2Bself%2Band%2Bnon-self%2BmRNA%2Bdependent%2Bon%2Bthe%2BRNA%2Bsensor%2BMda5.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fni.1979%26rft_id%253Dinfo%253Apmid%252F21217758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ni.1979&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21217758&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286378400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.21" data-doi="10.1128/JVI.03571-13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Menachery</span>, <span class="cit-name-given-names">V.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yount</span>, <span class="cit-name-given-names">B.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Josset</span>, <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gralinski</span>, <span class="cit-name-given-names">L.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scobey</span>, <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Agnihothram</span>, <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Katze</span>, <span class="cit-name-given-names">M.G.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Baric</span>, <span class="cit-name-given-names">R.S.</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Attenuation and Restoration of SARS-CoV Mutant lacking 2’ O Methyltransferase Activity</span>. <abbr class="cit-jnl-abbrev">J. Virol.</abbr>,<span class="cit-vol">88</span>, <span class="cit-fpage">4251</span>–<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.03571-13%26rft_id%253Dinfo%253Apmid%252F24478444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg4LzgvNDI1MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzAyLzA4LzI2MTc3Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.22" data-doi="10.1128/JVI.01056-16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Young</span>, <span class="cit-name-given-names">D.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andrejeva</span>, <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>, <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Inesta-Vaquera</span>, <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dong</span>, <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cowling</span>, <span class="cit-name-given-names">V.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goodbourn</span>, <span class="cit-name-given-names">S.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Randall</span>, <span class="cit-name-given-names">R.E.</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Human IFIT1 Inhibits mRNA Translation of Rubulaviruses but Not Other Members of the Paramyxoviridae Family</span>. <abbr class="cit-jnl-abbrev">J. Virol.</abbr>,<span class="cit-vol">90</span>, <span class="cit-fpage">9446</span>–<span class="cit-lpage">9456</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01056-16%26rft_id%253Dinfo%253Apmid%252F27512068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI5MC8yMC85NDQ2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDIvMDgvMjYxNzc2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Menachery</span>, <span class="cit-name-given-names">V.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gralinski</span>, <span class="cit-name-given-names">L.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mitchell</span>, <span class="cit-name-given-names">H.D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dinnon</span>, <span class="cit-name-given-names">K.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leist</span>, <span class="cit-name-given-names">S.R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yount</span>, <span class="cit-name-given-names">B.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>, <span class="cit-name-given-names">R.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McAnarney</span>, <span class="cit-name-given-names">E.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stratton</span>, <span class="cit-name-given-names">K.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cockrell</span>, <span class="cit-name-given-names">A.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Debbink</span>, <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sims</span>, <span class="cit-name-given-names">A.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Waters</span>, <span class="cit-name-given-names">K.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>, <span class="cit-name-given-names">R.S.</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Middle East Respiratory Syndrome Coronavirus Nonstructural Protein 16 Is Necessary for Interferon Resistance and Viral Pathogenesis</span>. <abbr class="cit-jnl-abbrev">mSphere</abbr>, <span class="cit-vol">2</span>, <span class="cit-fpage">e00346</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DmSphere%26rft.volume%253D2%26rft.spage%253De00346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.24" data-doi="10.1016/0042-6822(77)90471-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dubin</span>, <span class="cit-name-given-names">D.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stollar</span>, <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hsuchen</span>, <span class="cit-name-given-names">C.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Timko</span>, <span class="cit-name-given-names">K.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Guild</span>, <span class="cit-name-given-names">G.M.</span></span> (<span class="cit-pub-date">1977</span>) <span class="cit-article-title">Sindbis virus messenger RNA: the 5’-termini and methylated residues of 26 and 42 S RNA</span>. <abbr class="cit-jnl-abbrev">Virology</abbr>,<span class="cit-vol">77</span>, <span class="cit-fpage">457</span>–<span class="cit-lpage">470</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DDubin%26rft.auinit1%253DD.%2BT.%26rft.volume%253D77%26rft.issue%253D2%26rft.spage%253D457%26rft.epage%253D470%26rft.atitle%253DSindbis%2Bvirus%2Bmessenger%2BRNA%253A%2Bthe%2B5%2527-termini%2Band%2Bmethylated%2Bresidues%2Bof%2B26%2Band%2B42%2BS%2BRNA.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0042-6822%252877%252990471-8%26rft_id%253Dinfo%253Apmid%252F193248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0042-6822(77)90471-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=193248&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.25" data-doi="10.1126/science.1248465"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hyde</span>, <span class="cit-name-given-names">J.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gardner</span>, <span class="cit-name-given-names">C.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kimura</span>, <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">White</span>, <span class="cit-name-given-names">J.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>, <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Trobaugh</span>, <span class="cit-name-given-names">D.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>, <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tonelli</span>, <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Paessler</span>, <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Takeda</span>, <span class="cit-name-given-names">K.</span></span>, <span class="cit-etal">et al.</span>(<span class="cit-pub-date">2014</span>) <span class="cit-article-title">A Viral RNA Structural Element Alters Host Recognition of Nonself RNA</span>. <abbr class="cit-jnl-abbrev">Science (80-. ).</abbr>,<span class="cit-vol">343</span>, <span class="cit-fpage">783</span>–<span class="cit-lpage">787</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2B%252880-.%2B%2529.%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1248465%26rft_id%253Dinfo%253Apmid%252F24482115%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDMvNjE3Mi83ODMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wMi8wOC8yNjE3NzYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reynaud</span>, <span class="cit-name-given-names">J.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kim</span>, <span class="cit-name-given-names">D.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Atasheva</span>, <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rasalouskaya</span>, <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">White</span>, <span class="cit-name-given-names">J.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Diamond</span>, <span class="cit-name-given-names">M.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weaver</span>, <span class="cit-name-given-names">S.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frolova</span>, <span class="cit-name-given-names">E.I.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Frolov</span>, <span class="cit-name-given-names">I.</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">IFIT1 Differentially Interferes with Translation and Replication of Alphavirus Genomes and Promotes Induction of Type I Interferon</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr>, <span class="cit-vol">11</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.27" data-doi="10.1093/emboj/19.24.6891"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guo</span>, <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hui</span>, <span class="cit-name-given-names">D.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Merrick</span>, <span class="cit-name-given-names">W.C.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Sen</span>, <span class="cit-name-given-names">G.C.</span></span> (<span class="cit-pub-date">2000</span>) <span class="cit-article-title">A new pathway of translational regulation mediated by eukaryotic initiation factor 3</span>. <abbr class="cit-jnl-abbrev">EMBO J.</abbr>,<span class="cit-vol">19</span>, <span class="cit-fpage">6891</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DGuo%26rft.auinit1%253DJ.%26rft.volume%253D19%26rft.issue%253D24%26rft.spage%253D6891%26rft.epage%253D6899%26rft.atitle%253DA%2Bnew%2Bpathway%2Bof%2Btranslational%2Bregulation%2Bmediated%2Bby%2Beukaryotic%2Binitiation%2Bfactor%2B3%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F19.24.6891%26rft_id%253Dinfo%253Apmid%252F11118224%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMTkvMjQvNjg5MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzAyLzA4LzI2MTc3Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.28" data-doi="10.1128/JVI.01194-10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Saikia</span>, <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fensterl</span>, <span class="cit-name-given-names">V.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Sen</span>, <span class="cit-name-given-names">G.C.</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">The Inhibitory Action of P56 on Select Functions of E1 Mediates Interferon’s Effect on Human Papillomavirus DNA Replication</span>. <abbr class="cit-jnl-abbrev">J. Virol.</abbr>,<span class="cit-vol">84</span>, <span class="cit-fpage">13036</span>–<span class="cit-lpage">13039</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01194-10%26rft_id%253Dinfo%253Apmid%252F20926571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4NC8yNC8xMzAzNiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzAyLzA4LzI2MTc3Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.29" data-doi="10.1074/jbc.M110.207068"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stawowczyk</span>, <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van Scoy</span>, <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kumar</span>, <span class="cit-name-given-names">K.P.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Reich</span>, <span class="cit-name-given-names">N.C.</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">The interferon stimulated gene 54 promotes apoptosis</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr>,<span class="cit-vol">286</span>, <span class="cit-fpage">7257</span>–<span class="cit-lpage">7266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M110.207068%26rft_id%253Dinfo%253Apmid%252F21190939%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyODYvOS83MjU3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDIvMDgvMjYxNzc2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.30" data-doi="10.1126/science.aam9243"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chavali</span>, <span class="cit-name-given-names">P.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stojic</span>, <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meredith</span>, <span class="cit-name-given-names">L.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joseph</span>, <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nahorski</span>, <span class="cit-name-given-names">M.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sanford</span>, <span class="cit-name-given-names">T.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sweeney</span>, <span class="cit-name-given-names">T.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krishna</span>, <span class="cit-name-given-names">B.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hosmillo</span>, <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Firth</span>, <span class="cit-name-given-names">A.E.</span></span>, <span class="cit-etal">et al.</span>(<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Neurodevelopmental protein Musashi-1 interacts with the Zika genome and promotes viral replication</span>. <abbr class="cit-jnl-abbrev">Science (80-. ).</abbr>,<span class="cit-vol">357</span>, <span class="cit-fpage">83</span>–<span class="cit-lpage">88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2B%252880-.%2B%2529.%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aam9243%26rft_id%253Dinfo%253Apmid%252F28572454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzNTcvNjM0Ni84MyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzAyLzA4LzI2MTc3Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.31" data-doi="10.1038/nprot.2007.321"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Niesen</span>, <span class="cit-name-given-names">F.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berglund</span>, <span class="cit-name-given-names">H.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Vedadi</span>, <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span>. <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr>,<span class="cit-vol">2</span>, <span class="cit-fpage">2212</span>–<span class="cit-lpage">2221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DNiesen%26rft.auinit1%253DF.%2BH.%26rft.volume%253D2%26rft.issue%253D9%26rft.spage%253D2212%26rft.epage%253D2221%26rft.atitle%253DThe%2Buse%2Bof%2Bdifferential%2Bscanning%2Bfluorimetry%2Bto%2Bdetect%2Bligand%2Binteractions%2Bthat%2Bpromote%2Bprotein%2Bstability.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2007.321%26rft_id%253Dinfo%253Apmid%252F17853878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2007.321&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17853878&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253139600019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Emmott</span>, <span class="cit-name-given-names">E.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Goodfellow</span>, <span class="cit-name-given-names">I.</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Identification of Protein Interaction Partners in Mammalian Cells Using SILAC-immunoprecipitation Quantitative Proteomics</span>. <abbr class="cit-jnl-abbrev">J. Vis. Exp.</abbr>, 10.3791/51656.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.33" data-doi="10.1002/pmic.201400449"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tyanova</span>, <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Temu</span>, <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carlson</span>, <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sinitcyn</span>, <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mann</span>, <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Cox</span>, <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Visualization of LC-MS/MS proteomics data in MaxQuant</span>. <abbr class="cit-jnl-abbrev">Proteomics</abbr>,<span class="cit-vol">15</span>, <span class="cit-fpage">1453</span>–<span class="cit-lpage">1456</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteomics%26rft.volume%253D15%26rft.spage%253D1453%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpmic.201400449%26rft_id%253Dinfo%253Apmid%252F25644178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/pmic.201400449&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25644178&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.34" data-doi="10.1038/nmeth.3901"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tyanova</span>, <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Temu</span>, <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sinitcyn</span>, <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carlson</span>, <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hein</span>, <span class="cit-name-given-names">M.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Geiger</span>, <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mann</span>, <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Cox</span>, <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">The Perseus computational platform for comprehensive analysis of (prote)omics data</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr>,<span class="cit-vol">13</span>, <span class="cit-fpage">731</span>–<span class="cit-lpage">740</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D13%26rft.spage%253D731%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3901%26rft_id%253Dinfo%253Apmid%252F27348712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3901&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27348712&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dolinsky</span>, <span class="cit-name-given-names">T.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Czodrowski</span>, <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>, <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nielsen</span>, <span class="cit-name-given-names">J.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jensen</span>, <span class="cit-name-given-names">J.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klebe</span>, <span class="cit-name-given-names">G.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Baker</span>, <span class="cit-name-given-names">N.A.</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">PDB2PQR: Expanding and upgrading automated preparation of biomolecular structures for molecular simulations</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>, <span class="cit-vol">35</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">36.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-261776v1.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dolinsky</span>, <span class="cit-name-given-names">T.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nielsen</span>, <span class="cit-name-given-names">J.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McCammon</span>, <span class="cit-name-given-names">J.A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Baker</span>, <span class="cit-name-given-names">N.A.</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">PDB2PQR: An automated pipeline for the setup of Poisson-Boltzmann electrostatics calculations</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>, <span class="cit-vol">32</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.37" data-doi="10.1073/pnas.181342398"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Baker</span>, <span class="cit-name-given-names">N.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sept</span>, <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joseph</span>, <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holst</span>, <span class="cit-name-given-names">M.J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">McCammon</span>, <span class="cit-name-given-names">J.A.</span></span> (<span class="cit-pub-date">2001</span>) <span class="cit-article-title">Electrostatics of nanosystems: Application to microtubules and the ribosome</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci.</abbr>,<span class="cit-vol">98</span>, <span class="cit-fpage">10037</span>–<span class="cit-lpage">10041</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.181342398%26rft_id%253Dinfo%253Apmid%252F11517324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTgvMTgvMTAwMzciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wMi8wOC8yNjE3NzYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rabbani</span>, <span class="cit-name-given-names">M.A.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ribaudo</span>, <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>, <span class="cit-name-given-names">J.-T.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Barik</span>, <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Identification of IFN-stimulated gene (ISG) proteins that inhibit human parainfluenza virus type 3</span>. <abbr class="cit-jnl-abbrev">J. Virol.</abbr>, 10.1128/JVI.01551-16.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.39" data-doi="10.1089/jir.2012.0159"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reich</span>, <span class="cit-name-given-names">N.C.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">A death-promoting role for ISG54/IFIT2</span>. <abbr class="cit-jnl-abbrev">J. Interferon Cytokine Res.</abbr>,<span class="cit-vol">33</span>, <span class="cit-fpage">199</span>–<span class="cit-lpage">205</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BInterferon%2BCytokine%2BRes.%26rft.volume%253D33%26rft.spage%253D199%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fjir.2012.0159%26rft_id%253Dinfo%253Apmid%252F23570386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/jir.2012.0159&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23570386&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F02%2F08%2F261776.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000317463400008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hsu</span>, <span class="cit-name-given-names">Y.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>, <span class="cit-name-given-names">S.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>, <span class="cit-name-given-names">W.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ho</span>, <span class="cit-name-given-names">L.J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Lai</span>, <span class="cit-name-given-names">J.H.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Protective roles of interferon-induced protein with tetratricopeptide repeats 3 (IFIT3) in dengue virus infection of human lung epithelial cells</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>, <span class="cit-vol">8</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-261776v1.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Göertz</span>, <span class="cit-name-given-names">G.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Abbo</span>, <span class="cit-name-given-names">S.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fros</span>, <span class="cit-name-given-names">J.J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Pijlman</span>, <span class="cit-name-given-names">G.P.</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Functional RNA during Zika virus infection</span>. <abbr class="cit-jnl-abbrev">Virus Res.</abbr>, 10.1016/j.virusres.2017.08.015.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/261776v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F261776v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/163691v3" title="Phylodynamic assessment of intervention strategies for the West African Ebola virus outbreak" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/261610v1" title="Regulation of ATM and ATR by Smarcal1 and BRG1" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;February 08, 2018.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/261776v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/261776v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/261776v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/261776v1" class="active" data-icon-position="" data-hide-link-title="0">IFIT3 and IFIT2/3 promote IFIT1-mediated translation inhibition by enhancing binding to non-self RNA</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/80278" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-W5Jib4P8FeVMpX-XPplbzSXhyAiP1jl7zwTy6SQODXM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="80278" id="node80278--2" data-pisa="biorxiv;261776v1" data-pisa-master="biorxiv;261776" data-seqnum="80278" data-apath="/biorxiv/early/2018/02/08/261776.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">IFIT3 and IFIT2/3 promote IFIT1-mediated translation inhibition by enhancing binding to non-self RNA</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Renata C</span> <span class="nlm-surname">Fleith</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Harriet V</span> <span class="nlm-surname">Mears</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Edward</span> <span class="nlm-surname">Emmott</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Stephen C</span> <span class="nlm-surname">Graham</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Daniel S</span> <span class="nlm-surname">Mansur</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Trevor R</span> <span class="nlm-surname">Sweeney</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">261776; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/261776 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/261776v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/261776v1&amp;title=IFIT3%20and%20IFIT2/3%20promote%20IFIT1-mediated%20translation%20inhibition%20by%20enhancing%20binding%20to%20non-self%20RNA" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/261776v1&amp;title=IFIT3%20and%20IFIT2/3%20promote%20IFIT1-mediated%20translation%20inhibition%20by%20enhancing%20binding%20to%20non-self%20RNA" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/261776v1&amp;text=IFIT3%20and%20IFIT2/3%20promote%20IFIT1-mediated%20translation%20inhibition%20by%20enhancing%20binding%20to%20non-self%20RNA" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/261776v1&amp;title=IFIT3%20and%20IFIT2/3%20promote%20IFIT1-mediated%20translation%20inhibition%20by%20enhancing%20binding%20to%20non-self%20RNA" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/261776v1&amp;t=IFIT3%20and%20IFIT2/3%20promote%20IFIT1-mediated%20translation%20inhibition%20by%20enhancing%20binding%20to%20non-self%20RNA" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/261776v1&amp;title=IFIT3%20and%20IFIT2/3%20promote%20IFIT1-mediated%20translation%20inhibition%20by%20enhancing%20binding%20to%20non-self%20RNA" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/261776v1&amp;title=IFIT3%20and%20IFIT2/3%20promote%20IFIT1-mediated%20translation%20inhibition%20by%20enhancing%20binding%20to%20non-self%20RNA" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="80278" id="node80278--3" data-pisa="biorxiv;261776v1" data-pisa-master="biorxiv;261776" data-seqnum="80278" data-apath="/biorxiv/early/2018/02/08/261776.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">IFIT3 and IFIT2/3 promote IFIT1-mediated translation inhibition by enhancing binding to non-self RNA</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Renata C</span> <span class="nlm-surname">Fleith</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Harriet V</span> <span class="nlm-surname">Mears</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Edward</span> <span class="nlm-surname">Emmott</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Stephen C</span> <span class="nlm-surname">Graham</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Daniel S</span> <span class="nlm-surname">Mansur</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Trevor R</span> <span class="nlm-surname">Sweeney</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">261776; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/261776 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=IFIT3%20and%20IFIT2/3%20promote%20IFIT1-mediated%20translation%20inhibition%20by%20enhancing%20binding%20to%20non-self%20RNA&amp;rft.volume&amp;rft.issue&amp;rft.date=2018-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Fleith%2C+Renata+C&amp;rft.au=Mears%2C+Harriet+V&amp;rft.au=Emmott%2C+Edward&amp;rft.au=Graham%2C+Stephen+C&amp;rft.au=Mansur%2C+Daniel+S&amp;rft.au=Sweeney%2C+Trevor+R"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/80278/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/80278/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/80278/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/80278/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/80278/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/80278/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/80278/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/80278/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/80278/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/80278/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/80278/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/80278/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/261776v1&amp;count=horizontal&amp;via=&amp;text=IFIT3%20and%20IFIT2/3%20promote%20IFIT1-mediated%20translation%20inhibition%20by%20enhancing%20binding%20to%20non-self%20RNA&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/261776v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/261776v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/261776v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/immunology" class="highlight" data-icon-position="" data-hide-link-title="0">Immunology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
