<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/03/15/2020.03.12.988865.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.12.988865v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The architecture of SARS-CoV-2 transcriptome" />
<meta name="DC.Identifier" content="10.1101/2020.03.12.988865" />
<meta name="DC.Date" content="2020-03-15" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="SARS-CoV-2 is a betacoronavirus that is responsible for the COVID-19 pandemic. The genome of SARS-CoV-2 was reported recently, but its transcriptomic architecture is unknown. Utilizing two complementary sequencing techniques, we here present a high-resolution map of the SARS-CoV-2 transcriptome and epitranscriptome. DNA nanoball sequencing shows that the transcriptome is highly complex owing to numerous recombination events, both canonical and noncanonical. In addition to the genomic RNA and subgenomic RNAs common in all coronaviruses, SARS-CoV-2 produces a large number of transcripts encoding unknown ORFs with fusion, deletion, and/or frameshift. Using nanopore direct RNA sequencing, we further find at least 41 RNA modification sites on viral transcripts, with the most frequent motif being AAGAA. Modified RNAs have shorter poly(A) tails than unmodified RNAs, suggesting a link between the internal modification and the 3′ tail. Functional investigation of the unknown ORFs and RNA modifications discovered in this study will open new directions to our understanding of the life cycle and pathogenicity of SARS-CoV-2.

Highlights" />
<meta name="DC.Contributor" content="Dongwan Kim" />
<meta name="DC.Contributor" content="Joo-Yeon Lee" />
<meta name="DC.Contributor" content="Jeong-Sun Yang" />
<meta name="DC.Contributor" content="Jun Won Kim" />
<meta name="DC.Contributor" content="V. Narry Kim" />
<meta name="DC.Contributor" content="Hyeshik Chang" />
<meta name="article:published_time" content="2020-03-15" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="The architecture of SARS-CoV-2 transcriptome" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;SARS-CoV-2 is a betacoronavirus that is responsible for the COVID-19 pandemic. The genome of SARS-CoV-2 was reported recently, but its transcriptomic architecture is unknown. Utilizing two complementary sequencing techniques, we here present a high-resolution map of the SARS-CoV-2 transcriptome and epitranscriptome. DNA nanoball sequencing shows that the transcriptome is highly complex owing to numerous recombination events, both canonical and noncanonical. In addition to the genomic RNA and subgenomic RNAs common in all coronaviruses, SARS-CoV-2 produces a large number of transcripts encoding unknown ORFs with fusion, deletion, and/or frameshift. Using nanopore direct RNA sequencing, we further find at least 41 RNA modification sites on viral transcripts, with the most frequent motif being AAGAA. Modified RNAs have shorter poly(A) tails than unmodified RNAs, suggesting a link between the internal modification and the 3′ tail. Functional investigation of the unknown ORFs and RNA modifications discovered in this study will open new directions to our understanding of the life cycle and pathogenicity of SARS-CoV-2.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.12.988865v2" />
<meta name="citation_id" content="2020.03.12.988865v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.12.988865v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.12.988865v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.12.988865v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.12.988865" />
<meta name="citation_num_pages" content="29" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.12.988865" />
<meta name="citation_author" content="Dongwan Kim" />
<meta name="citation_author_institution" content="Center for RNA Research, Institute for Basic Science (IBS)" />
<meta name="citation_author_institution" content="School of Biological Sciences, Seoul National University" />
<meta name="citation_author" content="Joo-Yeon Lee" />
<meta name="citation_author_institution" content="Korea National Institute of Health, Korea Centers for Disease Control &amp; Prevention" />
<meta name="citation_author" content="Jeong-Sun Yang" />
<meta name="citation_author_institution" content="Korea National Institute of Health, Korea Centers for Disease Control &amp; Prevention" />
<meta name="citation_author" content="Jun Won Kim" />
<meta name="citation_author_institution" content="Korea National Institute of Health, Korea Centers for Disease Control &amp; Prevention" />
<meta name="citation_author" content="V. Narry Kim" />
<meta name="citation_author_institution" content="Center for RNA Research, Institute for Basic Science (IBS)" />
<meta name="citation_author_institution" content="School of Biological Sciences, Seoul National University" />
<meta name="citation_author_email" content="narrykim@snu.ac.kr" />
<meta name="citation_author_email" content="hyeshik@snu.ac.kr" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3213-4965" />
<meta name="citation_author" content="Hyeshik Chang" />
<meta name="citation_author_institution" content="Center for RNA Research, Institute for Basic Science (IBS)" />
<meta name="citation_author_institution" content="School of Biological Sciences, Seoul National University" />
<meta name="citation_author_email" content="narrykim@snu.ac.kr" />
<meta name="citation_author_email" content="hyeshik@snu.ac.kr" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9812-8015" />
<meta name="citation_reference" content="Courtney, D.G., Kennedy, E.M., Dumm, R.E., Bogerd, H.P., Tsai, K., Heaton, N.S., and Cullen, B.R. (2017). Epitranscriptomic Enhancement of Influenza A Virus Gene Expression and Replication. Cell Host Microbe 22, 377–386 e375." />
<meta name="citation_reference" content="Dobin, A., Davis, C.A., Schlesinger, F., Drenkow, J., Zaleski, C., Jha, S., Batut, P., Chaisson, M., and Gingeras, T.R. (2013). STAR: ultrafast universal RNA-seq aligner. Bioinformatics 29, 15–21." />
<meta name="citation_reference" content="Furuya, T., and Lai, M.M. (1993). Three different cellular proteins bind to complementary sites on the 5′-end-positive and 3′-end-negative strands of mouse hepatitis virus RNA. J Virol 67, 7215–7222." />
<meta name="citation_reference" content="Gokhale, N.S., and Horner, S.M. (2017). RNA modifications go viral. PLoS Pathog 13, e1006188." />
<meta name="citation_reference" content="Gokhale, N.S., McIntyre, A.B.R., McFadden, M.J., Roder, A.E., Kennedy, E.M., Gandara, J.A., Hopcraft, S.E., Quicke, K.M., Vazquez, C., Willer, J., et al. (2016). N6-Methyladenosine in Flaviviridae Viral RNA Genomes Regulates Infection. Cell Host Microbe 20, 654–665." />
<meta name="citation_reference" content="Kariko, K., Buckstein, M., Ni, H., and Weissman, D. (2005). Suppression of RNA recognition by Toll-like receptors: the impact of nucleoside modification and the evolutionary origin of RNA. Immunity 23, 165–175." />
<meta name="citation_reference" content="Kim, J.M., Chung, Y.S., Jo, H.J., Lee, N.J., Kim, M.S., Woo, S.H., Park, S., Kim, J.W., Kim, H.M., and Han, M.G. (2020). Identification of Coronavirus Isolated from a Patient in Korea with COVID-19. Osong Public Health Res Perspect 11, 3–7." />
<meta name="citation_reference" content="Krug, R.M., Morgan, M.A., and Shatkin, A.J. (1976). Influenza viral mRNA contains internal N6-methyladenosine and 5′-terminal 7-methylguanosine in cap structures. J Virol 20, 45–53." />
<meta name="citation_reference" content="Lai, M.M., and Stohlman, S.A. (1981). Comparative analysis of RNA genomes of mouse hepatitis viruses. J Virol 38, 661–670." />
<meta name="citation_reference" content="Li, H. (2018). Minimap2: pairwise alignment for nucleotide sequences. Bioinformatics 34, 3094–3100." />
<meta name="citation_reference" content="Liao, C.L., and Lai, M.M. (1992). RNA recombination in a coronavirus: recombination between viral genomic RNA and transfected RNA fragments. J Virol 66, 6117–6124." />
<meta name="citation_reference" content="Lichinchi, G., Zhao, B.S., Wu, Y., Lu, Z., Qin, Y., He, C., and Rana, T.M. (2016). Dynamics of Human and Viral RNA Methylation during Zika Virus Infection. Cell Host Microbe 20, 666–673." />
<meta name="citation_reference" content="Luytjes, W., Gerritsma, H., and Spaan, W.J. (1996). Replication of synthetic defective interfering RNAs derived from coronavirus mouse hepatitis virus-A59. Virology 216, 174–183." />
<meta name="citation_reference" content="Menachery, V.D., Graham, R.L., and Baric, R.S. (2017). Jumping species-a mechanism for coronavirus persistence and survival. Curr Opin Virol 23, 1–7." />
<meta name="citation_reference" content="Narayan, P., Ayers, D.F., Rottman, F.M., Maroney, P.A., and Nilsen, T.W. (1987). Unequal distribution of N6-methyladenosine in influenza virus mRNAs. Mol Cell Biol 7, 1572–1575." />
<meta name="citation_reference" content="Pathak, K.B., and Nagy, P.D. (2009). Defective Interfering RNAs: Foes of Viruses and Friends of Virologists. Viruses 1, 895–919." />
<meta name="citation_reference" content="Snijder, E.J., Decroly, E., and Ziebuhr, J. (2016). The Nonstructural Proteins Directing Coronavirus RNA Synthesis and Processing. Adv Virus Res 96, 59–126." />
<meta name="citation_reference" content="Sola, I., Almazan, F., Zuniga, S., and Enjuanes, L. (2015). Continuous and Discontinuous RNA Synthesis in Coronaviruses. Annu Rev Virol 2, 265–288." />
<meta name="citation_reference" content="Stoiber, M., Quick, J., Egan, R., Eun Lee, J., Celniker, S., Neely, R.K., Loman, N., Pennacchio, L.A., and Brown, J. (2017). De novo Identification of DNA Modifications Enabled by Genome-Guided Nanopore Signal Processing. bioRxiv, 094672." />
<meta name="citation_reference" content="Tomecki, R., Dmochowska, A., Gewartowski, K., Dziembowski, A., and Stepien, P.P. (2004). Identification of a novel human nuclear-encoded mitochondrial poly(A) polymerase. Nucleic Acids Res 32, 6001–6014." />
<meta name="citation_reference" content="Tvarogova, J., Madhugiri, R., Bylapudi, G., Ferguson, L.J., Karl, N., and Ziebuhr, J. (2019). Identification and Characterization of a Human Coronavirus 229E Nonstructural Protein 8-Associated RNA 3′-Terminal Adenylyltransferase Activity. J Virol 93." />
<meta name="citation_reference" content="Viehweger, A., Krautwurst, S., Lamkiewicz, K., Madhugiri, R., Ziebuhr, J., Holzer, M., and Marz, M. (2019). Direct RNA nanopore sequencing of full-length coronavirus genomes provides novel insights into structural variants and enables modification analysis. Genome Res 29, 1545–1554." />
<meta name="citation_reference" content="Virtanen, P., Gommers, R., Oliphant, T.E., Haberland, M., Reddy, T., Cournapeau, D., Burovski, E., Peterson, P., Weckesser, W., Bright, J., et al. (2020). SciPy 1.0: fundamental algorithms for scientific computing in Python. Nat Methods 17, 261–272." />
<meta name="citation_reference" content="Warkocki, Z., Liudkovska, V., Gewartowska, O., Mroczek, S., and Dziembowski, A. (2018). Terminal nucleotidyl transferases (TENTs) in mammalian RNA metabolism. Philos Trans R Soc Lond B Biol Sci 373." />
<meta name="citation_reference" content="Williams, G.D., Gokhale, N.S., and Horner, S.M. (2019). Regulation of Viral Infection by the RNA Modification N6-Methyladenosine. Annu Rev Virol 6, 235–253." />
<meta name="citation_reference" content="Wu, H.Y., Ke, T.Y., Liao, W.Y., and Chang, N.Y. (2013). Regulation of coronaviral poly(A) tail length during infection. PLoS One 8, e70548." />
<meta name="citation_reference" content="Yogo, Y., Hirano, N., Hino, S., Shibuta, H., and Matumoto, M. (1977). Polyadenylate in the virion RNA of mouse hepatitis virus. J Biochem 82, 1103–1108." />
<meta name="citation_reference" content="Zhou, P., Yang, X.L., Wang, X.G., Hu, B., Zhang, L., Zhang, W., Si, H.R., Zhu, Y., Li, B., Huang, C.L., et al. (2020). A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature." />
<meta name="citation_reference" content="Zhu, N., Zhang, D., Wang, W., Li, X., Yang, B., Song, J., Zhao, X., Huang, B., Shi, W., Lu, R., et al.(2020). A Novel Coronavirus from Patients with Pneumonia in China, 2019. N Engl J Med 382, 727–733." />
<meta name="twitter:title" content="The architecture of SARS-CoV-2 transcriptome" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="SARS-CoV-2 is a betacoronavirus that is responsible for the COVID-19 pandemic. The genome of SARS-CoV-2 was reported recently, but its transcriptomic architecture is unknown. Utilizing two complementary sequencing techniques, we here present a high-resolution map of the SARS-CoV-2 transcriptome and epitranscriptome. DNA nanoball sequencing shows that the transcriptome is highly complex owing to numerous recombination events, both canonical and noncanonical. In addition to the genomic RNA and subgenomic RNAs common in all coronaviruses, SARS-CoV-2 produces a large number of transcripts encoding unknown ORFs with fusion, deletion, and/or frameshift. Using nanopore direct RNA sequencing, we further find at least 41 RNA modification sites on viral transcripts, with the most frequent motif being AAGAA. Modified RNAs have shorter poly(A) tails than unmodified RNAs, suggesting a link between the internal modification and the 3′ tail. Functional investigation of the unknown ORFs and RNA modifications discovered in this study will open new directions to our understanding of the life cycle and pathogenicity of SARS-CoV-2.

Highlights" />
<meta name="og-title" property="og:title" content="The architecture of SARS-CoV-2 transcriptome" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.12.988865v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="SARS-CoV-2 is a betacoronavirus that is responsible for the COVID-19 pandemic. The genome of SARS-CoV-2 was reported recently, but its transcriptomic architecture is unknown. Utilizing two complementary sequencing techniques, we here present a high-resolution map of the SARS-CoV-2 transcriptome and epitranscriptome. DNA nanoball sequencing shows that the transcriptome is highly complex owing to numerous recombination events, both canonical and noncanonical. In addition to the genomic RNA and subgenomic RNAs common in all coronaviruses, SARS-CoV-2 produces a large number of transcripts encoding unknown ORFs with fusion, deletion, and/or frameshift. Using nanopore direct RNA sequencing, we further find at least 41 RNA modification sites on viral transcripts, with the most frequent motif being AAGAA. Modified RNAs have shorter poly(A) tails than unmodified RNAs, suggesting a link between the internal modification and the 3′ tail. Functional investigation of the unknown ORFs and RNA modifications discovered in this study will open new directions to our understanding of the life cycle and pathogenicity of SARS-CoV-2.

Highlights" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-03-15" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.12.988865v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.12.988865v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.12.988865v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.12.988865v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1196690" />
    <title>The architecture of SARS-CoV-2 transcriptome | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"OGUmb0M-LAoYV6JIgc4lmGzf4OxKzEjtdlEHbk1B-6E","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1196690","apath":"\/biorxiv\/early\/2020\/03\/15\/2020.03.12.988865.atom","pisa":"biorxiv;2020.03.12.988865v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.12.988865v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.12.988865v2","title":"The architecture of SARS-CoV-2 transcriptome","identifier":"node\/1196690"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022cY2Vtoh+OrGFY5BTZMlPcM\\\\\\\/gsRPSipXBdR\\\\\\\/g\\\\\\\/AOTWqZooRN\\\\\\\/UgFvQhNJfbFsKEfkgclRWJ2E7UBpvRM697+bD2PLBrAGo\\\\\\\/E2xOqIzJcbGwEC38P+CKYfDyFxYMG\\\\\\\/8afapMC\\\\\\\/ZOhZMzgBmcPk2Oes\\\\\\\/GR5jS5WCYywGy5wIf62lr1ByxvPgtfkKVnZDXcz+w1bJbh\\\\\\\/NAfd\\\\\\\/wN9KzC6XUHEP6xdmeHiMVwjMOUrIHwofA7ApPb26e+R+VI+VgdAmh6zhRAR6TkDnrMqzzByFvKS72OfTJlI8mQ6tK0bmBtPKs1aD5dKLvELm3h4iTaZYbeDOXOdsjZ1CkWgLF\\\\\\\/EvEPEyiUpMOs0GsVFYzlRwo5VWS6\\\\\\\/fwHKcqgSDKWMgPRd\\\\\\\/sGsh7p7rfEiPcKCx4FCQe87qCFbzpWb4T7\\\\\\\/4oOq4iweFXc7I2TyTzv+Re0G4IdYv1fju4vT0Aqzs5vG2AUqng1E16f0Abosaqp53JEZWlFbXUdO5so\\\\\\\/HtKfCZwsI4R9lL6XXTY8\\\\\\\/3RXW+jScnqCUh2jJ6PPD+MEbgoWq3kmM45KXzQqeNXDPP8YdR9wtVv8HkIvolSRm\\\\\\\/yNGtwnZC4qWtMneZb13LPtE6bLQ8rzbVM6gaN+Jer3pG4b+yjiE5\\\\\\\/sW2du7VWTBItKqf9Vb4DlZnTaiub7HbqatNcy3RpWciJhpukE7B29Ra+9RWGQa+9DKCqpj0e64X0E43DDhtRQrgeCxv3NUnKE9K7EsXAem91Yw8MX6a88O4KMllB8sQ0VeprsgV5VshNUVXgolAK9+pII+urWBxMdsUxvuFRdWTUHLT0yG5NH5xG46PqKfRz68Kd5qnd4uk7s6+03Q9faMGCtl2ZiC2cl1PufZXsMy\\\\\\\/882ZuTArwjNArMIAAykoGwa5WHtAo0U8Y9uX5HZfquV5xNV4RSE6LTOy4WFrnKt4UsYDnSlDSUYOJvl5JmtXk9pCsd1tfF1ZboFIDfijEyG6ago0mE+UiriXWGfd42d1MFPlCPTaECtsmug4LYpl2cFB+RqVZ\\\\\\\/Z72A61ShQ0ZQJ32rlkImPhq5Bm+\\\\\\\/jhYCofyBL4sSVnZfAgiA228yw7pYi+8QZZ+oXmpnVCtmYNu3g9SgNEoc4t5NF+55VvE0LHUz8PNfgyzb7rWNN3mNJH4upIZ5tf7BvcaYS2scqJqo31ZzKt7lnvIoX31HDZZ\\\\\\\/bKtHQTlZc8\\\\\\\/NZfipf55rNRehoOyS0uMJKC5KbJHHicd9i25UnLmi\\\\\\\/ghFpx\\\\\\\/q8ivbSJOVab1t18Dd35X8DYw5XQFl64Zah+7TuqYrqF7KTafThY99rRITo2ghr7ypFfcdM6cpYc\\\\\\\/tNi2Dlf96wjRb0HJgjt9i\\\u0022,\\\u0022iv\\\u0022:\\\u0022zM73DlHgGw6vAju+kTtDgw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002236bc234cdfb3f4884de19e58a1435ad433414ee2ad46c019573924341fb9505d\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.12.988865v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1196690 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.12.988865v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-HodL3qSlNq1pjU3aTajhRR8OwoU91qU9vtmODFBxwIM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1196690" id="node1196690" data-pisa="biorxiv;2020.03.12.988865v2" data-pisa-master="biorxiv;2020.03.12.988865" data-apath="/biorxiv/early/2020/03/15/2020.03.12.988865.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">The architecture of SARS-CoV-2 transcriptome</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Dongwan</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Joo-Yeon</span> <span class="nlm-surname">Lee</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jeong-Sun</span> <span class="nlm-surname">Yang</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Jun Won</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-3213-4965" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">V. Narry</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-9812-8015" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Hyeshik</span> <span class="nlm-surname">Chang</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.12.988865 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1196690" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Dongwan Kim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for RNA Research, Institute for Basic Science (IBS)</span>, Seoul 08826, <span class='nlm-country'>Republic of Korea</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>School of Biological Sciences, Seoul National University</span>, Seoul 08826, <span class='nlm-country'>Republic of Korea</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dongwan%2BKim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kim%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADongwan%2BKim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Joo-Yeon Lee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Korea National Institute of Health, Korea Centers for Disease Control &#38; Prevention</span>, Osong 28159, <span class='nlm-country'>Republic of Korea</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joo-Yeon%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lee%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoo-Yeon%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jeong-Sun Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Korea National Institute of Health, Korea Centers for Disease Control &#38; Prevention</span>, Osong 28159, <span class='nlm-country'>Republic of Korea</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeong-Sun%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeong-Sun%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jun Won Kim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Korea National Institute of Health, Korea Centers for Disease Control &#38; Prevention</span>, Osong 28159, <span class='nlm-country'>Republic of Korea</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jun%2BWon%2BKim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kim%20JW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJun%2BWon%2BKim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">V. Narry Kim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for RNA Research, Institute for Basic Science (IBS)</span>, Seoul 08826, <span class='nlm-country'>Republic of Korea</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>School of Biological Sciences, Seoul National University</span>, Seoul 08826, <span class='nlm-country'>Republic of Korea</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=V.%2BNarry%2BKim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kim%20VN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AV.%2BNarry%2BKim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3213-4965" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for V. Narry Kim</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:narrykim@snu.ac.kr" class="" data-icon-position="" data-hide-link-title="0">narrykim@snu.ac.kr</a>
<a href="mailto:hyeshik@snu.ac.kr" class="" data-icon-position="" data-hide-link-title="0">hyeshik@snu.ac.kr</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Hyeshik Chang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for RNA Research, Institute for Basic Science (IBS)</span>, Seoul 08826, <span class='nlm-country'>Republic of Korea</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>School of Biological Sciences, Seoul National University</span>, Seoul 08826, <span class='nlm-country'>Republic of Korea</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hyeshik%2BChang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHyeshik%2BChang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9812-8015" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Hyeshik Chang</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:narrykim@snu.ac.kr" class="" data-icon-position="" data-hide-link-title="0">narrykim@snu.ac.kr</a>
<a href="mailto:hyeshik@snu.ac.kr" class="" data-icon-position="" data-hide-link-title="0">hyeshik@snu.ac.kr</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.12.988865v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1196690" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1196690/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.12.988865v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1196690" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1196690/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.12.988865v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1196690" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1196690/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.12.988865v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1196690" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1196690/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.12.988865v2.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:1196690" data-trigger="external-links" data-url-enabled="1">Data/Code</a><a href="/panels_ajax_tab/article_tab_data_code/node:1196690/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.12.988865v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1196690" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1196690/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Summary</h2><p id="p-3">SARS-CoV-2 is a betacoronavirus that is responsible for the COVID-19 pandemic. The genome of SARS-CoV-2 was reported recently, but its transcriptomic architecture is unknown. Utilizing two complementary sequencing techniques, we here present a high-resolution map of the SARS-CoV-2 transcriptome and epitranscriptome. DNA nanoball sequencing shows that the transcriptome is highly complex owing to numerous recombination events, both canonical and noncanonical. In addition to the genomic RNA and subgenomic RNAs common in all coronaviruses, SARS-CoV-2 produces a large number of transcripts encoding unknown ORFs with fusion, deletion, and/or frameshift. Using nanopore direct RNA sequencing, we further find at least 41 RNA modification sites on viral transcripts, with the most frequent motif being AAGAA. Modified RNAs have shorter poly(A) tails than unmodified RNAs, suggesting a link between the internal modification and the 3′ tail. Functional investigation of the unknown ORFs and RNA modifications discovered in this study will open new directions to our understanding of the life cycle and pathogenicity of SARS-CoV-2.</p><div id="sec-1" class="subsection"><p><strong>Highlights</strong></p><ul class="list-unord " id="list-1"><li id="list-item-1"><p id="p-4">We provide a high-resolution map of SARS-CoV-2 transcriptome and epitranscriptome using nanopore direct RNA sequencing and DNA nanoball sequencing.</p></li><li id="list-item-2"><p id="p-5">The transcriptome is highly complex owing to numerous recombination events, both canonical and noncanonical.</p></li><li id="list-item-3"><p id="p-6">In addition to the genomic and subgenomic RNAs common in all coronaviruses, SARS-CoV-2 produces transcripts encoding unknown ORFs.</p></li><li id="list-item-4"><p id="p-7">We discover at least 41 potential RNA modification sites with an AAGAA motif.</p></li></ul></div></div><div class="section" id="sec-2"><h2 class="">Introduction</h2><p id="p-9">Coronavirus disease 19 (COVID-19) is caused by a novel coronavirus designated as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Zhou et al., 2020</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Zhu et al., 2020</a>). Like other coronaviruses (order <em>Nidovirales</em>, family <em>Coronaviridae</em>, subfamily <em>Coronavirinae</em>), SARS-CoV-2 is an enveloped virus with a positive-sense, single-stranded RNA genome of ~30 kb. SARS-CoV-2 belongs to the genus <em>betacoronavirus</em>, together with SARS-CoV and Middle East respiratory syndrome coronavirus (MERS-CoV) (with 80% and 50% homology, respectively) (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Kim et al., 2020</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Zhou et al., 2020</a>). Coronaviruses (CoVs) were thought to primarily cause enzootic infections in birds and mammals. But, the recurring outbreaks of SARS, MERS, and now COVID-19 have clearly demonstrated the remarkable ability of CoVs to cross species barriers and transmit between humans (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Menachery et al., 2017</a>).</p><p id="p-10">CoVs carry the largest genomes (26-32 kb) among all RNA virus families (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Each viral transcript has a 5′-cap structure and a 3′ poly(A) tail (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Lai and Stohlman, 1981</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Yogo et al., 1977</a>). Upon cell entry, the genomic RNA is translated to produce nonstructural proteins (nsps) from two open reading frames (ORFs), ORF1a, and ORF1b. The ORF1a produces polypeptide 1a (pp1a, 440-500 kDa) that is cleaved into 11 nsps. The –1 ribosome frameshift occurs immediately upstream of the ORF1a stop codon, which allows continued translation of ORF1b, yielding a large polypeptide (pp1ab, 740-810 kDa) which is cleaved into 16 nsps. The proteolytic cleavage is mediated by viral proteases nsp3 and nsp5 that harbor a papain-like protease domain and a 3C-like protease domain, respectively.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic presentation of the SARS-CoV-2 genome organization, the canonical subgenomic mRNAs, and the virion structure. From the full-length genomic RNA (29,903 nt) which also serves as an mRNA, ORF1a and ORF1b are translated. In addition to the genomic RNA, nine major subgenomic RNAs are produced. The sizes of the boxes representing small accessory proteins are bigger than the actual size of the ORF for better visualization. The black box indicates leader sequence. Note that ORF10 is not included here because our data show no evidence for ORF10 expression." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1618217866" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Schematic presentation of the SARS-CoV-2 genome organization, the canonical subgenomic mRNAs, and the virion structure.&lt;/span&gt; From the full-length genomic RNA (29,903 nt) which also serves as an mRNA, ORF1a and ORF1b are translated. In addition to the genomic RNA, nine major subgenomic RNAs are produced. The sizes of the boxes representing small accessory proteins are bigger than the actual size of the ORF for better visualization. The black box indicates leader sequence. Note that ORF10 is not included here because our data show no evidence for ORF10 expression.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F1.medium.gif" width="440" height="203"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F1.medium.gif" width="440" height="203"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <span class="caption-title">Schematic presentation of the SARS-CoV-2 genome organization, the canonical subgenomic mRNAs, and the virion structure.</span><p id="p-11" class="first-child">From the full-length genomic RNA (29,903 nt) which also serves as an mRNA, ORF1a and ORF1b are translated. In addition to the genomic RNA, nine major subgenomic RNAs are produced. The sizes of the boxes representing small accessory proteins are bigger than the actual size of the ORF for better visualization. The black box indicates leader sequence. Note that ORF10 is not included here because our data show no evidence for ORF10 expression.</p><div class="sb-div caption-clear"></div></div></div><p id="p-12">The viral genome is also used as the template for replication and transcription, which is mediated by nsp12 harboring RNA-dependent RNA polymerase (RdRP) activity (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Snijder et al., 2016</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Sola et al., 2015</a>). Negative-strand RNA intermediates are generated to serve as the templates for the synthesis of positive-sense genomic RNA (gRNA) and subgenomic RNAs (sgRNAs). The gRNA is packaged by the structural proteins to assemble progeny virions. Shorter sgRNAs encode conserved structural proteins (spike protein (S), envelope protein (E), membrane protein (M), and nucleocapsid protein (N)), and several accessory proteins. SARS-CoV-2 is known to have six accessory proteins (3a, 6, 7a, 7b, 8, and 10) according to the current annotation (NCBI Reference Sequence: NC_045512.2). But the ORFs have not yet been experimentally verified for expression. Therefore, it is currently unclear which accessory genes are actually expressed from this compact genome.</p><p id="p-13">Each coronaviral RNA contains the common 5′ “leader” sequence of 70-100 nt fused to the “body” sequence from the 3′ end of the genome (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Lai and Stohlman, 1981</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Sola et al., 2015</a>) (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). According to the prevailing model, leader-to-body fusion occurs during negative-strand synthesis at short motifs called transcription-regulating sequences (TRSs) that are located immediately adjacent to ORFs (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). TRSs contain a conserved 6-7 nt core sequence (CS) surrounded by variable sequences. During negative-strand synthesis, RdRP pauses when it crosses a TRS in the body (TRS-B), and switches the template to the TRS in the leader (TRS-L), which results in the leader-body fusion. From the fused negative-strand intermediates, positive-strand mRNAs are transcribed. The replication and transcription mechanism has been studied in other coronaviruses. However, it is unclear whether the general mechanism also applies to SARS-CoV-2 and if there are any unknown components in the SARS-CoV-2 transcriptome. For the development of diagnostic and therapeutic tools and the understanding of this new virus, it is critical to define the organization of the SARS-CoV-2 genome.</p><p id="p-14">Deep sequencing technologies offer a powerful means to investigate viral transcriptome. The “sequencing-by-synthesis (SBS)” methods such as the Illumina and MGI platforms confer high accuracy and coverage. But they are limited by short read length (200-400 nt), so the fragmented sequences should be re-assembled computationally, during which the haplotype information is lost. More recently introduced is the nanopore-based direct RNA sequencing (DRS) approach. While nanopore DRS is limited in sequencing accuracy, it enables long-read sequencing, which would be particularly useful for the analysis of long nested CoV transcripts. Moreover, because DRS does not require reverse transcription to generate cDNA, the RNA modification information can be detected directly during sequencing. Numerous RNA modifications have been found to control eukaryotic RNAs and viral RNAs (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Williams et al., 2019</a>). Terminal RNA modifications such as RNA tailing also plays a critical role in cellular and viral RNA regulation (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Warkocki et al., 2018</a>).</p><p id="p-15">In this study, we combined two complementary sequencing approaches, DRS and SBS. We unambiguously mapped the sgRNAs, ORFs, and TRSs of SARS-CoV-2. Additionally, we found numerous unconventional recombination events that are distinct from canonical TRS-mediated joining. We further discovered RNA modification sites and measured the poly(A) tail length of gRNAs and sgRNAs.</p></div><div class="section" id="sec-3"><h2 class="">Results and Discussion</h2><p id="p-16">To delineate the SARS-CoV-2 transcriptome, we first performed DRS runs on a MinION nanopore sequencer using total RNA extracted from Vero cells infected with SARS-CoV-2 (BetaCoV/Korea/KCDC03/2020). The virus was isolated from a patient who was diagnosed with COVID-19 on January 26, 2020, after traveling from Wuhan, China (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Kim et al., 2020</a>). We obtained 879,679 reads from infected cells (corresponding to a throughput of 1.9 Gb) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). The majority (65.4%) of the reads mapped to SARS-CoV-2, indicating that viral transcripts dominate the transcriptome while the host gene expression is strongly suppressed. Although nanopore DRS has the 3′ bias due to directional sequencing from the 3′-ends of RNAs, approximately half of the viral reads still contained the 5′ leader.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Statistics of sequencing data. A, Read counts from nanopore direct RNA sequencing of total RNA from Vero cells infected with SARS-CoV-2. &#x201C;Leader+&#x201D; indicates the viral reads that contain the 5&#x2032; end leader sequence. &#x201C;No leader&#x201D; denotes the viral reads lacking the leader sequence. &#x201C;Nuclear&#x201D; reads match to mRNAs from the nuclear chromosome while &#x201C;mitochondrial&#x201D; reads are derived from the mitochondrial genome. &#x201C;Control&#x201D; indicates quality control RNA for nanopore sequencing. B, Genome coverage of the nanopore direct RNA sequencing data shown in panel A. The stepwise reduction in coverage corresponds to the borders expected for the canonical sgRNAs. The smaller inner plot magnifies the 5&#x2032; part of the genome. C, Read counts from DNA nanoball sequencing using MGISEQ. Total RNA from Vero cells infected with SARS-CoV-2 was used for sequencing. D, Genome coverage of the DNA nanoball sequencing (DNBseq) data shown in panel C." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1618217866" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Statistics of sequencing data.&lt;/span&gt; &lt;strong&gt;A,&lt;/strong&gt; Read counts from nanopore direct RNA sequencing of total RNA from Vero cells infected with SARS-CoV-2. &#x201C;Leader+&#x201D; indicates the viral reads that contain the &lt;em&gt;5&#x2032;&lt;/em&gt; end leader sequence. &#x201C;No leader&#x201D; denotes the viral reads lacking the leader sequence. &#x201C;Nuclear&#x201D; reads match to mRNAs from the nuclear chromosome while &#x201C;mitochondrial&#x201D; reads are derived from the mitochondrial genome. &#x201C;Control&#x201D; indicates quality control RNA for nanopore sequencing. &lt;strong&gt;B,&lt;/strong&gt; Genome coverage of the nanopore direct RNA sequencing data shown in panel A. The stepwise reduction in coverage corresponds to the borders expected for the canonical sgRNAs. The smaller inner plot magnifies the 5&#x2032; part of the genome. &lt;strong&gt;C,&lt;/strong&gt; Read counts from DNA nanoball sequencing using MGISEQ. Total RNA from Vero cells infected with SARS-CoV-2 was used for sequencing. &lt;strong&gt;D,&lt;/strong&gt; Genome coverage of the DNA nanoball sequencing (DNBseq) data shown in panel C.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F2.medium.gif" width="440" height="143"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F2.medium.gif" width="440" height="143"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <span class="caption-title">Statistics of sequencing data.</span><p id="p-17" class="first-child"><strong>A,</strong> Read counts from nanopore direct RNA sequencing of total RNA from Vero cells infected with SARS-CoV-2. “Leader+” indicates the viral reads that contain the <em>5′</em> end leader sequence. “No leader” denotes the viral reads lacking the leader sequence. “Nuclear” reads match to mRNAs from the nuclear chromosome while “mitochondrial” reads are derived from the mitochondrial genome. “Control” indicates quality control RNA for nanopore sequencing. <strong>B,</strong> Genome coverage of the nanopore direct RNA sequencing data shown in panel A. The stepwise reduction in coverage corresponds to the borders expected for the canonical sgRNAs. The smaller inner plot magnifies the 5′ part of the genome. <strong>C,</strong> Read counts from DNA nanoball sequencing using MGISEQ. Total RNA from Vero cells infected with SARS-CoV-2 was used for sequencing. <strong>D,</strong> Genome coverage of the DNA nanoball sequencing (DNBseq) data shown in panel C.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">The SARS-CoV-2 genome was fully covered, missing only 12 nt from the 5′ end (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). The longest tags (111 reads) correspond to the full-length gRNA (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). The coverage of the 3′ side of the viral genome is substantially higher than that of the 5′ side, which reflects the nested sgRNAs. This is also partly due to the 3′ bias of the directional DRS technique. The presence of the leader sequence (72 nt) in viral RNAs results in a prominent coverage peak at the 5′ end, as expected. We could also clearly detect vertical drops in the coverage, whose positions correspond to the leader-body junction in sgRNAs. All known sgRNAs are supported by DRS reads, with the exception of ORF10 (see below).</p><p id="p-19">In addition, we observed unexpected reads reflecting noncanonical recombination events. Such fusion transcripts resulted in the increased coverage towards the 5′ end (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>, inner box). Early studies on coronavirus mouse hepatitis virus reported that recombination frequently occurs (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Furuya and Lai, 1993</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Liao and Lai, 1992</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Luytjes et al., 1996</a>). Some viral RNAs contain the 5′ and 3<sup>’</sup> proximal sequences resulting from “illegitimate” recombination events.</p><p id="p-20">To further validate sgRNAs and their junction sites, we performed DNA nanoball sequencing based on the sequencing-by-synthesis principle (DNBseq) and obtained 305,065,029 reads (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2C</a>). The results are overall consistent with the DRS data. The leader-body junctions are frequently sequenced, giving rise to a sharp peak at the 5′ end in the coverage plot (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2D</a>). The 3′ end exhibits a high coverage as expected for the nested transcripts.</p><p id="p-21">The depth of DNB sequencing allowed us to confirm and examine the junctions on an unprecedented scale for a CoV genome. We mapped the 5′ and 3′ sites at the fused junctions and estimated the recombination frequency by counting the reads spanning the junctions (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). The leader represents the most prominent 5′ site, as expected (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>, red asterisk on the x-axis). The known TRSs are detected as the top 3′ sites (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>, red dots on the y-axis). These results confirm that SARS-CoV-2 uses the canonical TRS-mediated mechanism for discontinuous transcription to produce major sgRNAs (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>). Quantitative comparison of the junction-spanning reads shows that the N RNA is the most abundantly expressed transcript, followed by S, 7a, 3a, 8, M, E, 6, and 7b (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3C</a>). It is important to note that ORF10 is represented by only one read in DNB data (0.000009 % of viral junction-spanning reads) and that it was not supported at all by DRS data. ORF10 does not show significant homology to known proteins. Thus, ORF10 is unlikely to be expressed, and the annotation of ORF10 should be reconsidered. Taken together, SARS-CoV-2 expresses nine canonical sgRNAs (S, 3a, E, M, 6, 7a, 7b, 8, and N) together with the gRNA (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figures 1</a> and <a id="xref-fig-3-6" class="xref-fig" href="#F3">3C</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Viral subgenomic RNAs and their recombination sites. A, Positions of recombination sites determined by using junction-spanning reads from DNBseq data. The x-axis and y-axis show the positions of the 5&#x2032; sites and their joined 3&#x2032; sites, respectively. The frequency of the recombination was estimated by the read counts of the junction-spanning reads. The red asterisk on the x-axis is to indicate that the leader sequence. Please note that the left-most bins containing the leader TRS was expanded horizontally on this heatmap to improve visualization. The red dots on the sub-plot alongside the y-axis denote local peaks which coincide with the 5&#x2032; end of the body of each sgRNA. B, Transcript abundance was estimated by counting the DNBseq reads that span the junction of the corresponding RNA. C, Top 50 sgRNAs. The asterisk indicates an ORF beginning at 27,825 which may encode the 7b protein with an N-terminal truncation of 23 amino acid. The grey bars denote minor transcripts that encode proteins with an N-terminal truncation compared with the corresponding overlapping transcript. The black bars indicate minor transcripts that encode proteins in a different reading frame from the overlapping major mRNA. D, Canonical recombination. E, TRS-L-dependent noncanonical recombination between the leader TRS and a noncanonical 3&#x2032; site in the body. F, TRS-L-independent long-distance (&gt;5,000 nt) recombination. G, TRS-L-independent local recombination yielding a deletion between proximal sites (20-5,000 nt distance)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1618217866" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Viral subgenomic RNAs and their recombination sites.&lt;/span&gt; &lt;strong&gt;A,&lt;/strong&gt; Positions of recombination sites determined by using junction-spanning reads from DNBseq data. The x-axis and y-axis show the positions of the 5&#x2032; sites and their joined 3&#x2032; sites, respectively. The frequency of the recombination was estimated by the read counts of the junction-spanning reads. The red asterisk on the x-axis is to indicate that the leader sequence. Please note that the left-most bins containing the leader TRS was expanded horizontally on this heatmap to improve visualization. The red dots on the sub-plot alongside the y-axis denote local peaks which coincide with the 5&#x2032; end of the body of each sgRNA. &lt;strong&gt;B,&lt;/strong&gt; Transcript abundance was estimated by counting the DNBseq reads that span the junction of the corresponding RNA. &lt;strong&gt;C,&lt;/strong&gt; Top 50 sgRNAs. The asterisk indicates an ORF beginning at 27,825 which may encode the 7b protein with an N-terminal truncation of 23 amino acid. The grey bars denote minor transcripts that encode proteins with an N-terminal truncation compared with the corresponding overlapping transcript. The black bars indicate minor transcripts that encode proteins in a different reading frame from the overlapping major mRNA. &lt;strong&gt;D,&lt;/strong&gt; Canonical recombination. &lt;strong&gt;E,&lt;/strong&gt; TRS-L-dependent noncanonical recombination between the leader TRS and a noncanonical 3&#x2032; site in the body. &lt;strong&gt;F,&lt;/strong&gt; TRS-L-independent long-distance (&gt;5,000 nt) recombination. &lt;strong&gt;G,&lt;/strong&gt; TRS-L-independent local recombination yielding a deletion between proximal sites (20-5,000 nt distance).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F3.medium.gif" width="440" height="351"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F3.medium.gif" width="440" height="351"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <span class="caption-title">Viral subgenomic RNAs and their recombination sites.</span><p id="p-22" class="first-child"><strong>A,</strong> Positions of recombination sites determined by using junction-spanning reads from DNBseq data. The x-axis and y-axis show the positions of the 5′ sites and their joined 3′ sites, respectively. The frequency of the recombination was estimated by the read counts of the junction-spanning reads. The red asterisk on the x-axis is to indicate that the leader sequence. Please note that the left-most bins containing the leader TRS was expanded horizontally on this heatmap to improve visualization. The red dots on the sub-plot alongside the y-axis denote local peaks which coincide with the 5′ end of the body of each sgRNA. <strong>B,</strong> Transcript abundance was estimated by counting the DNBseq reads that span the junction of the corresponding RNA. <strong>C,</strong> Top 50 sgRNAs. The asterisk indicates an ORF beginning at 27,825 which may encode the 7b protein with an N-terminal truncation of 23 amino acid. The grey bars denote minor transcripts that encode proteins with an N-terminal truncation compared with the corresponding overlapping transcript. The black bars indicate minor transcripts that encode proteins in a different reading frame from the overlapping major mRNA. <strong>D,</strong> Canonical recombination. <strong>E,</strong> TRS-L-dependent noncanonical recombination between the leader TRS and a noncanonical 3′ site in the body. <strong>F,</strong> TRS-L-independent long-distance (&gt;5,000 nt) recombination. <strong>G,</strong> TRS-L-independent local recombination yielding a deletion between proximal sites (20-5,000 nt distance).</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">In addition to the canonical sgRNAs with expected structure and length (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3D</a>), our results show many minor recombination sites (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figures 3E</a>-<a id="xref-fig-3-9" class="xref-fig" href="#F3">3G</a>, Table S2). There are three main types of such recombinant events. The RNAs in the first cluster have the leader combined with the body at unexpected 3′ sites in the middle of ORFs or UTRs (<a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3E</a>, TRS-L-dependent noncanonical recombination; Table S3). The second cluster shows a long-distance fusion between sequences that do not have similarity to the leader (<a id="xref-fig-3-11" class="xref-fig" href="#F3">Figure 3F</a>, TRS-L-independent distant recombination). The last group undergoes local recombination, which leads to smaller deletions, mainly in structural and accessory genes, including S (<a id="xref-fig-3-12" class="xref-fig" href="#F3">Figure 3G</a>, TRS-L-independent local recombination).</p><p id="p-24">Of note, the junctions in these noncanonical transcripts do not contain a known TRS-B, indicating that at least some of these transcripts may be generated through a different mechanism(s). It was previously shown in other coronaviruses that transcripts with partial sequences are produced (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Furuya and Lai, 1993</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Liao and Lai, 1992</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Luytjes et al., 1996</a>). These RNAs are considered as parasites that compete for viral proteins, hence referred to as “defective interfering RNAs” (DI-RNAs) (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Pathak and Nagy, 2009</a>). Similar sgRNAs have also been described in a recent sequencing analysis on alphacoronavirus HCoV-229E (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Viehweger et al., 2019</a>), suggesting this mechanism may be conserved among coronaviruses. While this may be due to erroneous replicase activity, it remains an open question if the noncanonical recombinations have an active role in the viral life cycle and evolution. Although individual RNA species are not abundant, the combined read numbers are often comparable to the levels of accessory transcripts. Most of the RNAs have coding potential to yield proteins. Many transcripts (that belong to the “TRS-L-independent distant” group) encode the upstream part of ORF1a, including nsp1, nsp2, and truncated nsp3, which may change the stoichiometry between nsps (<a id="xref-fig-3-13" class="xref-fig" href="#F3">Figure 3F</a>, Table S4). Another notable example is the 7b protein with an N-terminal truncation that may be produced at a level similar to the annotated full-length 7b (<a id="xref-fig-3-14" class="xref-fig" href="#F3">Figure 3C</a>, asterisk). Frame-shifted ORFs may also generate short peptides that are different from known viral proteins (<a id="xref-fig-3-15" class="xref-fig" href="#F3">Figure 3B</a>). It will be interesting in the future to examine if these unknown ORFs are actually translated and yield functional products.</p><p id="p-25">As nanopore DRS is based on single-molecule detection of RNA, it offers a unique opportunity to examine multiple epitranscriptomic features of individual RNA molecules. We recently developed software to measure the length of poly(A) tail from DRS data (Choi and Chang, unpublished). Using this software, we confirm that, like other CoVs, SARS-CoV-2 RNAs carry poly(A) tails (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figures 4A-B</a>). The tail is likely to be critical for both translation and replication. The tail of viral RNAs is 47 nt in median length. The full-length viral RNA has a relatively longer tail than sgRNAs. Notably, sgRNAs have two tail populations: a minor peak at ~30 nt and a major peak at ~45 nt (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>, arrowheads). Wu and colleagues previously observed that the poly(A) tail length of bovine CoV mRNAs change during infection: from ~45 nt immediately after virus entry to ~65 nt at 6-9 h.p.i. and ~30 nt at 120-144 h.p.i. (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Wu et al., 2013</a>). Thus, the short tails of ~30 nt observed in this study may represent aged RNAs that are prone to decay. Viral RNAs exhibit a homogenous length distribution, unlike host nuclear genome-encoded mRNAs (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>). The distribution is similar to that of mitochondrial chromosome-encoded RNAs whose tail is generated by MTPAP (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Tomecki et al., 2004</a>). It was recently shown that HCoV-229E nsp8 has an adenylyltransferase activity, which may extend poly(A) tail of viral RNA (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Tvarogova et al., 2019</a>). Because poly(A) tail should be constantly attacked by host deadenylases, the regulation of viral RNA tailing is likely to be important for viral end replication.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Length of poly(A) tail. A-B, Poly(A) tail length distribution of viral transcripts. Arrowheads indicate peaks at ~30 and ~45 nt. C, Poly(A) tail length distribution of quality control RNA which has a 30-nt poly(A) tail, host mRNAs from the nuclear chromosome, or host RNAs from the mitochondrial chromosome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1618217866" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Length of poly(A) tail.&lt;/span&gt; &lt;strong&gt;A-B,&lt;/strong&gt; Poly(A) tail length distribution of viral transcripts. Arrowheads indicate peaks at ~30 and ~45 nt. &lt;strong&gt;C,&lt;/strong&gt; Poly(A) tail length distribution of quality control RNA which has a 30-nt poly(A) tail, host mRNAs from the nuclear chromosome, or host RNAs from the mitochondrial chromosome.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F4.medium.gif" width="440" height="227"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F4.medium.gif" width="440" height="227"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <span class="caption-title">Length of poly(A) tail.</span><p id="p-26" class="first-child"><strong>A-B,</strong> Poly(A) tail length distribution of viral transcripts. Arrowheads indicate peaks at ~30 and ~45 nt. <strong>C,</strong> Poly(A) tail length distribution of quality control RNA which has a 30-nt poly(A) tail, host mRNAs from the nuclear chromosome, or host RNAs from the mitochondrial chromosome.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">Next, we examined the epitranscriptomic landscape of SARS-CoV-2 by using the DRS data. Viral RNA modification was first described more than 40 years ago (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Gokhale and Horner, 2017</a>). N6-methyladenosine (m6A) is the most widely used modification (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Courtney et al., 2017</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Gokhale et al., 2016</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Krug et al., 1976</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Lichinchi et al., 2016</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Narayan et al., 1987</a>), but other modifications have also been reported on viral RNAs, including 5-methylcytosine methylation (5mC), 2′-O-methylation (Nm), deamination, and terminal uridylation. In a recent analysis of HCoV-229E using DRS, modification calling suggested frequent 5mC signal across viral RNAs (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Viehweger et al., 2019</a>). But since no direct control group was included in the analysis, the proposed modification needed validation. To unambiguously investigate the modifications, we generated negative control RNAs by in vitro transcription of the viral sequences and performed a DRS run on these unmodified controls (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure S1A</a>). The partially overlapping control RNAs are ~2.3 kb or ~4.4 kb each and cover the entire length of the genome (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure S1B</a>). Detection using pre-trained models reported numerous signal level changes corresponding to 5mC modification, even with the unmodified controls (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure S1C</a>). We obtained highly comparable results from the viral RNAs from infected cells (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure S1D</a>), demonstrating that the 5mC sites detected without a control may be false positives.</p><p id="p-28">We, however, noticed intriguing differences in the ionic current (called “squiggles”) between negative control and viral transcripts (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). At least 41 sites displayed substantial differences (over 20% frequency), indicating potential RNA modifications (Table S5). Notably, some of the sites showed different frequencies depending on the sgRNA species. <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figures 5A</a>-<a id="xref-fig-5-3" class="xref-fig" href="#F5">5C</a> show an example that is modified more heavily on the S RNA than the N RNA, while <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figures S2A</a>-<a id="xref-fig-7-2" class="xref-fig" href="#F7">S2C</a> present a site that is modified more frequently on the ORF8 RNA compared with the S RNA. Moreover, the dwell time of the modified base is longer than that of the unmodified base (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5D</a>, right), suggesting that the modification interferes with the passing of RNA molecules through the pore.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Frequent RNA modification sites. A, Distinct ionic current signals (&#x201C;squiggles&#x201D;) from viral S transcript (green lines) and in vitro transcribed control (IVT, black lines) indicate RNA modification at the genomic position 29,016. B, The ionic current signals from viral N transcript at the genomic position 29,016 (yellow lines) are similar to those from IVT control (black lines), indicating that modification is rare on the N sgRNA. C, Ionic current distribution at A29016. Blue line shows the signal distribution in the standard model of tombo 1.5. D, Dwell time difference supports RNA modification. The dwell time of the region 29,015-29,017 of the S RNA (right) is significantly longer than those of IVT control and N RNAs. On the contrary, the neighboring region 28,995-28,997 of IVT, N, and S RNA is indistinguishable (left). E, Position-specific base frequency of a motif enriched in the frequently modified sites. F, Genomic location of modification sites with the AAGAA-like motif (top row) and the others grouped by the detected nucleotide base. G, Location and modification levels in different RNA species. H, Poly(A) length distribution of gRNA (left) and S RNA (right). Modified viral RNAs carry shorter poly(A) tails." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1618217866" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Frequent RNA modification sites.&lt;/span&gt; &lt;strong&gt;A,&lt;/strong&gt; Distinct ionic current signals (&#x201C;squiggles&#x201D;) from viral S transcript (green lines) and in vitro transcribed control (IVT, black lines) indicate RNA modification at the genomic position 29,016. &lt;strong&gt;B,&lt;/strong&gt; The ionic current signals from viral N transcript at the genomic position 29,016 (yellow lines) are similar to those from IVT control (black lines), indicating that modification is rare on the N sgRNA. &lt;strong&gt;C,&lt;/strong&gt; Ionic current distribution at A29016. Blue line shows the signal distribution in the standard model of tombo 1.5. &lt;strong&gt;D,&lt;/strong&gt; Dwell time difference supports RNA modification. The dwell time of the region 29,015-29,017 of the S RNA (right) is significantly longer than those of IVT control and N RNAs. On the contrary, the neighboring region 28,995-28,997 of IVT, N, and S RNA is indistinguishable (left). &lt;strong&gt;E,&lt;/strong&gt; Position-specific base frequency of a motif enriched in the frequently modified sites. &lt;strong&gt;F,&lt;/strong&gt; Genomic location of modification sites with the AAGAA-like motif (top row) and the others grouped by the detected nucleotide base. &lt;strong&gt;G,&lt;/strong&gt; Location and modification levels in different RNA species. &lt;strong&gt;H,&lt;/strong&gt; Poly(A) length distribution of gRNA (left) and S RNA (right). Modified viral RNAs carry shorter poly(A) tails.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F5.medium.gif" width="440" height="311"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F5.medium.gif" width="440" height="311"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <span class="caption-title">Frequent RNA modification sites.</span><p id="p-29" class="first-child"><strong>A,</strong> Distinct ionic current signals (“squiggles”) from viral S transcript (green lines) and in vitro transcribed control (IVT, black lines) indicate RNA modification at the genomic position 29,016. <strong>B,</strong> The ionic current signals from viral N transcript at the genomic position 29,016 (yellow lines) are similar to those from IVT control (black lines), indicating that modification is rare on the N sgRNA. <strong>C,</strong> Ionic current distribution at A29016. Blue line shows the signal distribution in the standard model of tombo 1.5. <strong>D,</strong> Dwell time difference supports RNA modification. The dwell time of the region 29,015-29,017 of the S RNA (right) is significantly longer than those of IVT control and N RNAs. On the contrary, the neighboring region 28,995-28,997 of IVT, N, and S RNA is indistinguishable (left). <strong>E,</strong> Position-specific base frequency of a motif enriched in the frequently modified sites. <strong>F,</strong> Genomic location of modification sites with the AAGAA-like motif (top row) and the others grouped by the detected nucleotide base. <strong>G,</strong> Location and modification levels in different RNA species. <strong>H,</strong> Poly(A) length distribution of gRNA (left) and S RNA (right). Modified viral RNAs carry shorter poly(A) tails.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">Among the 41 potential modification sites, the most frequently observed motif is AAGAA (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figures 5E</a> and <a id="xref-fig-7-3" class="xref-fig" href="#F7">S2D</a>). The modification sites on the ‘AAGAA-like’ motif (including AAGAA and other A/G-rich sequences) are found throughout the viral genome but particularly enriched in genomic positions 28,500-29,500 (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5F</a>). Long viral transcripts (gRNA, S, 3a, E, and M) are more frequently modified than shorter RNAs (6, 7a, 7b, 8, and N) (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5G</a>), suggesting a modification mechanism that is specific for certain RNA species.</p><p id="p-31">Since DRS allows simultaneous detection of multiple features on individual molecules, we cross-examined the poly(A) tail length and internal modification sites. Interestingly, modified RNA molecules have shorter poly(A) tails than unmodified ones (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Figures 5H</a> and <a id="xref-fig-8-1" class="xref-fig" href="#F8">S3</a>). These results suggest a link between the internal modification and 3′ end tail. Since poly(A) tail plays an important role in RNA turnover, it is tempting to speculate that the observed internal modification is involved in viral RNA stability control. It is also plausible that RNA modification is a mechanism to evade the host immune response. The type of modification(s) is yet to be identified, although we can exclude METTL3-mediated m6A (for lack of consensus motif RRACH), ADAR-mediated deamination (for lack of A-to-G sequence change in the DNBseq data), and m1A (for lack of the evidence for RT stop). Our finding implicates a hidden layer of CoV regulation. It will be interesting in the future to identify the chemical nature, enzymology, and biological functions of the modification(s).</p><p id="p-32">In this study, we delineate the transcriptomic and epitranscriptomic architecture of SARS-CoV-2. Unambiguous mapping of the expressed sgRNAs and ORFs is a prerequisite for the functional investigation of viral proteins, replication mechanism, and host-viral interactions involved in pathogenicity. An in-depth analysis of the joint reads revealed a highly complex landscape of viral RNA synthesis. Like other RNA viruses, CoVs undergo frequent recombination, which may allow rapid evolution to change their host/tissue specificity and drug sensitivity. The ORFs found in this study may serve as accessory proteins that modulate viral replication and host immune response. The RNA modifications may also contribute to viral survival and immune evasion in infected tissues as the innate immune system is known to be less sensitive to RNAs with nucleoside modification (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Kariko et al., 2005</a>). It is yet to be examined if the ORFs and RNA modifications are unique to SARS-CoV-2 or conserved in other coronaviruses. Comparative studies on their distribution and functional significance will help us to gain a deeper understanding of SARS-CoV-2 and coronaviruses in general. Our data provide a rich resource and open new directions to investigate the mechanisms underlying the pathogenicity of SARS-CoV-2.</p></div><div class="section" id="sec-4"><h2 class="">Ethics Statement</h2><p id="p-33">This study was carried out in accordance with the biosafety guideline by the KCDC. The Institutional Biosafety Committee of Seoul National University approved the protocol used in these studies (SNUIBC-200219-10).</p></div><div class="section" id="sec-5"><h2 class="">Author Contributions</h2><p id="p-36">H.C, J.Y.L, and V.N.K. designed the study. D.K., J.S.Y., and J.W.K. performed molecular and cell biological experiments. H.C. carried out computational analyses. H.C., J.Y.L, and V.N.K. wrote the manuscript.</p></div><div class="section" id="sec-6"><h2 class="">Declaration of Interests</h2><p id="p-37">The authors declare no competing interests.</p></div><div class="section" id="sec-7"><h2 class="">Accession Numbers</h2><p id="p-38">The sequencing data were deposited into the Open Science Framework (OSF) with an accession number doi:10.17605/OSF.IO/8F6N9.</p></div><div class="section" id="sec-8"><h2 class="">Supplementary Information</h2><div class="section" id="sec-9"><h3>Supplemental Table Legends</h3><p id="p-39"><strong>Supplementary Table 1 | Oligonucleotide sequences used in preparation of IVT control RNAs.</strong></p><p id="p-40">Refer to Methods for the experimental procedure.</p><p id="p-41"><strong>Supplementary Table 2 | Recombination sites detected from junction-spanning reads.</strong></p><p id="p-42">The genomic coordinates in the “5′ site” and “3′ site” point to the 3′-most and the 5′-most nucleotides that survive the recombination event, respectively. “Count” is the number of the junction-spanning reads that supports the recombination event. “Skip” is the size of the deletion. “5p_before” is the 3′-most sequence from the 5′-side of the recombination event. “5p_after” is the bases immediately following the “5′ site.” The sequence is the first nucleotides of the region that is excluded by the recombination event. “3p_before” is the last nucleotide sequence of the region that is excluded by the recombination. “3p_after” is the 5′-most nucleotides of the 3′-side of the recombination product.</p><p id="p-43"><strong>Supplementary Table 3 | TRS-L-dependent recombination events resulting in noncanonical leader-body junction (related to <a id="xref-fig-3-16" class="xref-fig" href="#F3">Fig. 3E</a>).</strong></p><p id="p-44">This table extends the information for the recombination events using the canonical leader TRS joined to noncanonical 3′ sites. “Startpos” is the genomic coordinate where the first AUG appears in the resulting recombination product. “Resultingsequence” shows the sequence near the recombination site with the nucleotides from the 5′-side in lowercase, and those from the 3′-side in uppercase. “Annot_cds” is the name of an ORF that shares the start codon position with the recombination product by the “first AUG” rule. “Annot_cds_start” and “annot_cds_end” indicate the interval of the annotated canonical ORF that matches to the start codon in the recombination product. “Startmatch_name” and “startmatch_offset” show the canonical ORF that overlaps with the position of the first appearing AUG in the recombination product, and the distance between the AUG and the start codon position of the ORF. “Stopmatch_name” and “stopmatch_offset” show the canonical ORF that overlaps with the expected stop codon position in case the translation starts at the first appearing AUG, and the relative coordinate of the “expected” stop codon position from the stop codon of the canonical ORF. “Translation” shows the peptide sequence that is expected to be produced by the first ORF in the recombination product.</p><p id="p-45"><strong>Supplementary Table 4 | TRS-L-independent distant recombination events from the 5′ sites located within the “hot spots” (related to <a id="xref-fig-3-17" class="xref-fig" href="#F3">Fig. 3F</a>).</strong></p><p id="p-46">Many distant recombination events do not involve the leader sequence. Instead, the 5′ sites are often found in “hot spots” near ORFs or nsp2 and nsp3. “Count” indicates the number of reads that share the same 5′ and 3′ sites. “Skip” is the size of the deletion in the given combination of 5′ and 3′ sites. “Truncated_prod” is the name of a mature peptide that is truncated by recombination. “Lost_aa” is the number of amino acids lost from the mature peptide by recombination. “Added_aa” is the number of amino acids introduced to the potentially truncated product by recombination. By the recombination event, the sequence indicated in “ctx_orig” turns into the sequence in “ctx_new”.</p><p id="p-47"><strong>Supplementary Table 5 | Detected modification sites in sgRNAs.</strong></p><p id="p-48">The “pos” column shows all genomic positions reported to be modified at least 20% in any species of sgRNA. “Location” indicates the name of ORF or mature peptide that the position codes. The percentage numbers below the names of sgRNAs are the estimated fraction of modified reads at the position. “Context” shows the sequence context around the modified bases.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="False positive calling of 5mC modification demonstrated by using unmodified negative control RNAs. A, Read counts from nanopore direct RNA sequencing of in vitro transcribed (IVT) RNAs that have viral sequences. &#x201C;Control&#x201D; indicates quality control RNA for nanopore sequencing. B, The 15 partially overlapping patches cover the entire genome (blue rectangles at the bottom). Each RNA is ~2.3 kb in length. One fragment marked with a green rectangle is longer than others (~4.4 kb) to circumvent difficulties in the PCR amplification. The sequenced reads were downsampled so that every region is equally covered. The resulting balanced coverage is shown in the chart at the top. C, Detected 5mC modification from in vitro transcribed unmodified RNAs (IVT product) by the &#x201C;alternative base detection&#x201D; mode in Tombo. Black dots indicate the sites that satisfy the estimated false discovery rate cut-off calculated using unmodified yeast ENO2 mRNA (Viehweger et al., 2019). D, Comparison between the sites called from unmodified IVT products and those from viral RNAs expressed in Vero cells. Related to Figure 5" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1618217866" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;False positive calling of 5mC modification demonstrated by using unmodified negative control RNAs.&lt;/span&gt; &lt;strong&gt;A,&lt;/strong&gt; Read counts from nanopore direct RNA sequencing of in vitro transcribed (IVT) RNAs that have viral sequences. &#x201C;Control&#x201D; indicates quality control RNA for nanopore sequencing. &lt;strong&gt;B,&lt;/strong&gt; The 15 partially overlapping patches cover the entire genome (blue rectangles at the bottom). Each RNA is ~2.3 kb in length. One fragment marked with a green rectangle is longer than others (~4.4 kb) to circumvent difficulties in the PCR amplification. The sequenced reads were downsampled so that every region is equally covered. The resulting balanced coverage is shown in the chart at the top. &lt;strong&gt;C,&lt;/strong&gt; Detected 5mC modification from in vitro transcribed unmodified RNAs (IVT product) by the &#x201C;alternative base detection&#x201D; mode in Tombo. Black dots indicate the sites that satisfy the estimated false discovery rate cut-off calculated using unmodified yeast &lt;em&gt;ENO2&lt;/em&gt; mRNA (Viehweger et al., 2019). &lt;strong&gt;D,&lt;/strong&gt; Comparison between the sites called from unmodified IVT products and those from viral RNAs expressed in Vero cells. &lt;strong&gt;Related to Figure 5&lt;/strong&gt;&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F6.medium.gif" width="440" height="435"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 1" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F6.medium.gif" width="440" height="435"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 1</span> <span class="caption-title">False positive calling of 5mC modification demonstrated by using unmodified negative control RNAs.</span><p id="p-49" class="first-child"><strong>A,</strong> Read counts from nanopore direct RNA sequencing of in vitro transcribed (IVT) RNAs that have viral sequences. “Control” indicates quality control RNA for nanopore sequencing. <strong>B,</strong> The 15 partially overlapping patches cover the entire genome (blue rectangles at the bottom). Each RNA is ~2.3 kb in length. One fragment marked with a green rectangle is longer than others (~4.4 kb) to circumvent difficulties in the PCR amplification. The sequenced reads were downsampled so that every region is equally covered. The resulting balanced coverage is shown in the chart at the top. <strong>C,</strong> Detected 5mC modification from in vitro transcribed unmodified RNAs (IVT product) by the “alternative base detection” mode in Tombo. Black dots indicate the sites that satisfy the estimated false discovery rate cut-off calculated using unmodified yeast <em>ENO2</em> mRNA (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Viehweger et al., 2019</a>). <strong>D,</strong> Comparison between the sites called from unmodified IVT products and those from viral RNAs expressed in Vero cells. <strong>Related to <a id="xref-fig-5-9" class="xref-fig" href="#F5">Figure 5</a></strong></p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Detected modified sites in viral RNAs. A, Ionic current levels near the genomic position 27,947 in viral S RNA (green lines) and IVT control RNA (black lines). B, Ionic current levels for the identical region in the viral ORF8 RNA (orange lines) and IVT control RNA (black lines). C, Signal distributions at the position 27,947 in the different RNAs. The blue line shows the standard model used for modification detections without controls (&#x201C;alternative base detection&#x201D; and &#x201C;de novo&#x201D; modes) in Tombo. D, Sequence alignment of the detected modification sites with &#x201C;AAGAA&#x201D;-type motif. Base positions on the left hand side correspond to the genomic coordinates denoted with red arrowhead. Related to Figure 5" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1618217866" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Detected modified sites in viral RNAs.&lt;/span&gt; &lt;strong&gt;A,&lt;/strong&gt; Ionic current levels near the genomic position 27,947 in viral S RNA (green lines) and IVT control RNA (black lines). &lt;strong&gt;B,&lt;/strong&gt; Ionic current levels for the identical region in the viral ORF8 RNA (orange lines) and IVT control RNA (black lines). &lt;strong&gt;C,&lt;/strong&gt; Signal distributions at the position 27,947 in the different RNAs. The blue line shows the standard model used for modification detections without controls (&#x201C;alternative base detection&#x201D; and &#x201C;de novo&#x201D; modes) in Tombo. &lt;strong&gt;D,&lt;/strong&gt; Sequence alignment of the detected modification sites with &#x201C;AAGAA&#x201D;-type motif. Base positions on the left hand side correspond to the genomic coordinates denoted with red arrowhead. &lt;strong&gt;Related to Figure 5&lt;/strong&gt;&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F7.medium.gif" width="358" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 2" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F7.medium.gif" width="358" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 2</span> <span class="caption-title">Detected modified sites in viral RNAs.</span><p id="p-50" class="first-child"><strong>A,</strong> Ionic current levels near the genomic position 27,947 in viral S RNA (green lines) and IVT control RNA (black lines). <strong>B,</strong> Ionic current levels for the identical region in the viral ORF8 RNA (orange lines) and IVT control RNA (black lines). <strong>C,</strong> Signal distributions at the position 27,947 in the different RNAs. The blue line shows the standard model used for modification detections without controls (“alternative base detection” and “de novo” modes) in Tombo. <strong>D,</strong> Sequence alignment of the detected modification sites with “AAGAA”-type motif. Base positions on the left hand side correspond to the genomic coordinates denoted with red arrowhead. <strong>Related to <a id="xref-fig-5-10" class="xref-fig" href="#F5">Figure 5</a></strong></p><div class="sb-div caption-clear"></div></div></div><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Highly modified viral RNAs carry shorter poly(A) tails. Poly(A) tail length distribution of each viral transcript other than shown in Figure 5. Related to Figure 5" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1618217866" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Highly modified viral RNAs carry shorter poly(A) tails.&lt;/span&gt; Poly(A) tail length distribution of each viral transcript other than shown in Figure 5. &lt;strong&gt;Related to Figure 5&lt;/strong&gt;&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F8.medium.gif" width="440" height="321"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 3" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F8.medium.gif" width="440" height="321"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/15/2020.03.12.988865/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 3</span> <span class="caption-title">Highly modified viral RNAs carry shorter poly(A) tails.</span><p id="p-51" class="first-child">Poly(A) tail length distribution of each viral transcript other than shown in <a id="xref-fig-5-11" class="xref-fig" href="#F5">Figure 5</a>. <strong>Related to <a id="xref-fig-5-12" class="xref-fig" href="#F5">Figure 5</a></strong></p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-10"><h2 class="">STAR Methods</h2><div class="section" id="sec-11"><h3>SARS-CoV-2 sample preparation</h3><p id="p-52">SARS-CoV-2 viral RNA was prepared by extracting total RNA from Vero cells infected with BetaCoV/Korea/KCDC03/2020, at a multiplicity of infection (MOI) of 0.05, and cultured in DMEM supplemented with 2% fetal bovine serum and penicillin-strepamycin. The virus is the fourth passage and not plaque-isolated. Cells were harvested at 24 hours post-infection and washed once with PBS before adding TRIzol. Viral culture was conducted in a biosafety level-3 facility.</p></div><div class="section" id="sec-12"><h3>In vitro transcription</h3><p id="p-53">Total RNA from SARS-CoV-2-infected Vero cell was extracted by using TRIzol (Invitrogen) followed by DNaseI (Takara) treatment. Reverse transcription (SuperScript IV Reverse Transcriptase [Invitrogen]) was done with virus-specific RT primers. Templates for in vitro transcription were prepared by PCR (Q5® High-Fidelity DNA Polymerase [NEB]) with virus-specific PCR primers followed by in vitro transcription (MEGAscript™ T7 Transcription Kit [Invitrogen]). The oligonucleotides used in this study were listed in Table S1.</p></div><div class="section" id="sec-13"><h3>Nanopore direct RNA sequencing</h3><p id="p-54">For nanopore sequencing on non-infected and SARS-CoV-2-infected Vero cells, each 4 <em>μ</em>g of DNase I (Takara)-treated total RNA in 8 <em>μ</em>l was used for library preparation following the manufacturer’s instruction (the Oxford Nanopore DRS protocol, SQK-RNA002) with minor adaptations. 20 U of SUPERase-In RNase inhibitor (Ambion, 20 U/<em>μ</em>l) was added to both adapter ligation steps. SuperScript IV Reverse Transcriptase (Invitrogen) was adopted instead of SuperScript III, and the reaction time of reverse transcription was lengthened by 2 hours. The library was loaded on FLO-MIN106D flow cell followed by 42 hours sequencing run on MinION device (Oxford Nanopore Technologies). For nanopore sequencing on SARS-CoV-2 RNA fragments by in vitro transcription, the same method was applied except for a total 2 <em>μ</em>g of fragment RNAs and 30 minutes reaction time of reverse transcription. The nanopore direct sequencing data were basecalled using guppy 3.4.5 (Oxford Nanopore Technologies) using the high-accuracy model. The sequence reads were aligned to the reference sequence database composed of the <em>C. sabaeus</em> genome (ENSEMBL release 99), a SARS-CoV-2 genome, yeast <em>ENO2</em> cDNA (YHR174W), and human ribosomal DNA complete repeat unit (GenBank <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=U13369.1&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F15%2F2020.03.12.988865.atom">U13369.1</a>) using minimap2 2.17 (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Li, 2018</a>) with options “-k 13 -x splice -N 32 -un”. We used the sequence of the Wuhan-Hu-1 strain (GenBank <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=NC_045512.2&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F15%2F2020.03.12.988865.atom">NC_045512.2</a>) as a backbone for the viral reference genome, then corrected the four single nucleotide variants found in BetaCoV/Korea/KCDC03/2020; T4402C, G5062T, C8782T, and T28143C (GISAID). The sequence alignments were further improved by re-mapping the identified viral reads to the viral genome using minimap2 options “-k 8 -w 1 --splice -g 30000 -G 30000 -A1 -B2 -O2,24 -E1,0 -C0 -z 400,200 --no-end-flt --junc-bonus=100 -F 40000 -N 32 --splice-flank=no --max-chain-skip=40 -un --junc-bed=FILE -p 0.7”. Chimeric reads were filtered out according to the flag from minimap2. The mapped reads from canonical sgRNAs were identified using the start and end positions of large deletions ≥ 10000 nt. For a valid assignment to a species of sgRNA, we required that the start position is between 55 and 85 in the genomic coordinate. The first AUG in the downstream of the end position of a large deletion was used for identification of the “spliced” product. The dwell time of poly(A) tails were measured using poreplex 0.5.0 (<a href="https://github.com/hyeshik/poreplex">https://github.com/hyeshik/poreplex</a>). For the conversion from a dwell time to a nucleotide length, we divided a poly(A) dwell measurement by 1/30 of the mode of the poly(A) dwell time of the ONT sequencing calibration control which has a 30 nt-long poly(A) tail.</p></div><div class="section" id="sec-14"><h3>DNBseq RNA sequencing</h3><p id="p-55">Total RNA from SARS-CoV-2-infected Vero cell was extracted by using TRIzol (Invitrogen) followed by DNaseI (Takara) treatment. Dynabeads<sup>®</sup> mRNA Purification Kit (Invitrogen) was applied to 1 <em>μ</em>g of total RNA for rRNA depletion. RNA-seq library for 250 bp insert size was constructed following the manufacturer’s instruction (MGIEasy RNA Directional Library Prep Set). The library was loaded on MGISEQ-200RS Sequencing flow cell with MGISEQ-200RS High-throughput Sequencing Kit (PE 100), and the library was run on DNBSEQ-G50RS (paired-end run, 100 × 100 cycles). The sequences from DNBseq were aligned to the reference sequences used in nanopore DRS. We used STAR (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Dobin et al., 2013</a>) with many switches to completely turn off the penalties of non-canonical eukaryotic splicing: “--outFilterType BySJout -- outFilterMultimapNmax 20 --alignSJoverhangMin 8 --outSJfilterOverhangMin 12 12 12 12 --outSJfilterCountUniqueMin 1 1 1 1 --outSJfilterCountTotalMin 1 1 1 1 --outSJfilterDistToOtherSJmin 0 0 0 0 --outFilterMismatchNmax 999 --outFilterMismatchNoverReadLmax 0.04 --scoreGapNoncan -4 --scoreGapATAC -4 --chimOutType WithinBAM HardClip --chimScoreJunctionNonGTAG 0 --alignSJstitchMismatchNmax -1 -1 -1 -1 --alignIntronMin 20 --alignIntronMax 1000000 --alignMatesGapMax 1000000”.</p></div><div class="section" id="sec-15"><h3>Analysis of junction-spanning reads</h3><p id="p-56">The junction-spanning reads (JSR) were prepared by extracting all large deletion (≥ 20 nt) events from the sequence alignments. <strong>[<a id="xref-fig-3-18" class="xref-fig" href="#F3">Figure 3A</a>]</strong> We counted the number of 5′ and 3′ sites in 100nt-wide bins. The red dots that indicate the frequently used 3′ sites were identified using the SciPy’s scipy.signals.find_peaks(prominence=2) (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Virtanen et al., 2020</a>). <strong>[<a id="xref-fig-3-19" class="xref-fig" href="#F3">Figures 3B</a>-<a id="xref-fig-3-20" class="xref-fig" href="#F3">3G</a>]</strong> The JSRs were categorized by the position of 5′ and 3′ site positions. A JSR was marked as a leader-to-body junction when the 5′ site of the deletion is between 55 and 85. The sgRNA identity and the frame matching were determined by the first AUG position in the downstream sequence of the 3′ site for the JSRs that the 5′ site is in a region that is annotated as a UTR. For the cases that the 5′ site is in a known ORF or an AUG is introduced by recombination, we checked if the concatenated sequence generates a protein product with the same reading frame as a canonical ORF after the 3′ site.</p></div><div class="section" id="sec-16"><h3>Modified base detection and the downstream analyses</h3><p id="p-57">The DRS reads of the IVT RNAs were downsampled to balance the coverage between different fragments that were split into equal-sized patches. Sampling frequency of a fragment was controlled by the read counts within a 100-nt bin with the lowest coverage in each fragment. We sampled the reads so that the result contain roughly 10,000 reads from every IVT fragment. The viral RNA reads and the downsampled IVT reads were processed for squiggle analyses by ONT tombo 1.5 (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Stoiber et al., 2017</a>) with a minor tune to improve the sensitivity of sequence alignments (-k8 -w1). The modified base detection was done by using the “model_sample_compare” mode with an option “--sample-only-estimates” unless otherwise specified. <strong>[<a id="xref-fig-5-13" class="xref-fig" href="#F5">Figure 5E</a>-<a id="xref-fig-5-14" class="xref-fig" href="#F5">5F</a>]</strong> The classification of sequence context near the modified sites was first done by the existence of four consecutive purine bases within 5-nt from the position with the highest modification fraction reported by tombo. Then, the rest were further divided into four groups according to the nucleotide base with the highest modification fraction. <strong>[<a id="xref-fig-5-15" class="xref-fig" href="#F5">Figures 5H</a> and <a id="xref-fig-8-2" class="xref-fig" href="#F8">S3</a>]</strong> “Highly modified” sgRNA reads were detected by referring to eight modification sites which were at least 40% modified in any species of sgRNAs: 28591, 28612, 28653, 28860, 28958, 29016, 29088, 29127. We used the reads that were reported as modified at three or more sites with a statistic &lt; 0.01 as “highly modified” reads. “Not modified” reads were reported with the statistic ≥ 0.01 in all eight sites.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-34">We thank members of our laboratories for discussion and help, particularly Eun-jin Chang and Young-suk Lee. We are grateful to Dr. Jung-Hye Roe and Inhye Park for their advice. We thank Kyung-Chang Kim and Sung Soon Kim at Korea National Institute of Health for their support. The pathogen resource (NCCP43326) for this study was provided by the National Culture Collection for Pathogens, Korea National Institute of Health.</p><p id="p-35">Funding: This work was supported by IBS-R008-D1 of Institute for Basic Science from the Ministry of Science, ICT and Future Planning of Korea (D.K., H.C. and V.N.K.), BK21 Research Fellowship from the Ministry of Education of Korea (D.K.), the New Faculty Startup Fund from Seoul National University (H.C.), and 4845-300(2019-NG044) of Korea Centers for Disease Control and Prevention (J.Y.L., J.S.Y., and J.W.K.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-other" id="fn-1"><p id="p-1"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label">4</span> Lead Contact</p></li><li class="fn-dataset fn-group-external-links" id="fn-2"><p id="p-8">
<a href="https://osf.io/8f6n9/">https://osf.io/8f6n9/</a>
</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.988865v2.1" data-doi="10.1016/j.chom.2017.08.004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Courtney</span>,  <span class="cit-name-given-names">D.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kennedy</span>,  <span class="cit-name-given-names">E.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dumm</span>,  <span class="cit-name-given-names">R.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bogerd</span>,  <span class="cit-name-given-names">H.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tsai</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heaton</span>,  <span class="cit-name-given-names">N.S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Cullen</span>,  <span class="cit-name-given-names">B.R.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Epitranscriptomic Enhancement of Influenza A Virus Gene Expression and Replication</span>. <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr> <span class="cit-vol">22</span>, <span class="cit-fpage">377</span>–<span class="cit-lpage">386</span> <span class="cit-page-range">e375</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2BMicrobe%26rft.volume%253D22%26rft.spage%253D377%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2017.08.004%26rft_id%253Dinfo%253Apmid%252F28910636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2017.08.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28910636&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F15%2F2020.03.12.988865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.988865v2.2" data-doi="10.1093/bioinformatics/bts635"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dobin</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davis</span>,  <span class="cit-name-given-names">C.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schlesinger</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drenkow</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zaleski</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jha</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Batut</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chaisson</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Gingeras</span>,  <span class="cit-name-given-names">T.R.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">STAR: ultrafast universal RNA-seq aligner</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>, <span class="cit-fpage">15</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts635%26rft_id%253Dinfo%253Apmid%252F23104886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts635&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23104886&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F15%2F2020.03.12.988865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312654600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.988865v2.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Furuya</span>,  <span class="cit-name-given-names">T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Lai</span>,  <span class="cit-name-given-names">M.M.</span></span> (<span class="cit-pub-date">1993</span>). <span class="cit-article-title">Three different cellular proteins bind to complementary sites on the 5′-end-positive and 3′-end-negative strands of mouse hepatitis virus RNA</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">67</span>, <span class="cit-fpage">7215</span>–<span class="cit-lpage">7222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DFuruya%26rft.auinit1%253DT%26rft.volume%253D67%26rft.issue%253D12%26rft.spage%253D7215%26rft.epage%253D7222%26rft.atitle%253DThree%2Bdifferent%2Bcellular%2Bproteins%2Bbind%2Bto%2Bcomplementary%2Bsites%2Bon%2Bthe%2B5%2527-end-positive%2Band%2B3%2527-end-negative%2Bstrands%2Bof%2Bmouse%2Bhepatitis%2Bvirus%2BRNA.%26rft_id%253Dinfo%253Apmid%252F8230443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI2Ny8xMi83MjE1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTUvMjAyMC4wMy4xMi45ODg4NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.988865v2.4" data-doi="10.1371/journal.ppat.1006188"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gokhale</span>,  <span class="cit-name-given-names">N.S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Horner</span>,  <span class="cit-name-given-names">S.M.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">RNA modifications go viral</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-vol">13</span>, <span class="cit-fpage">e1006188</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D13%26rft.spage%253De1006188%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1006188%26rft_id%253Dinfo%253Apmid%252F28278189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1006188&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28278189&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F15%2F2020.03.12.988865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.988865v2.5" data-doi="10.1016/j.chom.2016.09.015"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gokhale</span>,  <span class="cit-name-given-names">N.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McIntyre</span>,  <span class="cit-name-given-names">A.B.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McFadden</span>,  <span class="cit-name-given-names">M.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roder</span>,  <span class="cit-name-given-names">A.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kennedy</span>,  <span class="cit-name-given-names">E.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gandara</span>,  <span class="cit-name-given-names">J.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hopcraft</span>,  <span class="cit-name-given-names">S.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Quicke</span>,  <span class="cit-name-given-names">K.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vazquez</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Willer</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">N6-Methyladenosine in Flaviviridae Viral RNA Genomes Regulates Infection</span>. <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr> <span class="cit-vol">20</span>, <span class="cit-fpage">654</span>–<span class="cit-lpage">665</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2BMicrobe%26rft.volume%253D20%26rft.spage%253D654%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2016.09.015%26rft_id%253Dinfo%253Apmid%252F27773535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2016.09.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27773535&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F15%2F2020.03.12.988865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.988865v2.6" data-doi="10.1016/j.immuni.2005.06.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kariko</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buckstein</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ni</span>,  <span class="cit-name-given-names">H.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Weissman</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">Suppression of RNA recognition by Toll-like receptors: the impact of nucleoside modification and the evolutionary origin of RNA</span>. <abbr class="cit-jnl-abbrev">Immunity</abbr> <span class="cit-vol">23</span>, <span class="cit-fpage">165</span>–<span class="cit-lpage">175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunity%26rft.stitle%253DImmunity%26rft.aulast%253DKarik%25C3%25B3%26rft.auinit1%253DK.%26rft.volume%253D23%26rft.issue%253D2%26rft.spage%253D165%26rft.epage%253D175%26rft.atitle%253DSuppression%2Bof%2BRNA%2Brecognition%2Bby%2BToll-like%2Breceptors%253A%2Bthe%2Bimpact%2Bof%2Bnucleoside%2Bmodification%2Band%2Bthe%2Bevolutionary%2Borigin%2Bof%2BRNA.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.immuni.2005.06.008%26rft_id%253Dinfo%253Apmid%252F16111635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.immuni.2005.06.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16111635&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F15%2F2020.03.12.988865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231721700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.988865v2.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">J.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chung</span>,  <span class="cit-name-given-names">Y.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jo</span>,  <span class="cit-name-given-names">H.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">N.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">M.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Woo</span>,  <span class="cit-name-given-names">S.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Park</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">J.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">H.M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">M.G.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Identification of Coronavirus Isolated from a Patient in Korea with COVID-19</span>. <abbr class="cit-jnl-abbrev">Osong Public Health Res Perspect</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">3</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOsong%2BPublic%2BHealth%2BRes%2BPerspect%26rft.volume%253D11%26rft.spage%253D3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.988865v2.8" data-doi="10.1177/1091581816683642"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Krug</span>,  <span class="cit-name-given-names">R.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morgan</span>,  <span class="cit-name-given-names">M.A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Shatkin</span>,  <span class="cit-name-given-names">A.J.</span></span> (<span class="cit-pub-date">1976</span>). <span class="cit-article-title">Influenza viral mRNA contains internal N6-methyladenosine and 5′-terminal 7-methylguanosine in cap structures</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">20</span>, <span class="cit-fpage">45</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DKrug%26rft.auinit1%253DR%2BM%26rft.volume%253D20%26rft.issue%253D1%26rft.spage%253D45%26rft.epage%253D53%26rft.atitle%253DInfluenza%2Bviral%2BmRNA%2Bcontains%2Binternal%2BN6-methyladenosine%2Band%2B5%2527-terminal%2B7-methylguanosine%2Bin%2Bcap%2Bstructures.%26rft_id%253Dinfo%253Adoi%252F10.1177%252F1091581816683642%26rft_id%253Dinfo%253Apmid%252F1086370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjc6IjIwLzEvNDUiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8xNS8yMDIwLjAzLjEyLjk4ODg2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.988865v2.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lai</span>,  <span class="cit-name-given-names">M.M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Stohlman</span>,  <span class="cit-name-given-names">S.A.</span></span> (<span class="cit-pub-date">1981</span>). <span class="cit-article-title">Comparative analysis of RNA genomes of mouse hepatitis viruses</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">38</span>, <span class="cit-fpage">661</span>–<span class="cit-lpage">670</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DLai%26rft.auinit1%253DM%2BM%26rft.volume%253D38%26rft.issue%253D2%26rft.spage%253D661%26rft.epage%253D670%26rft.atitle%253DComparative%2Banalysis%2Bof%2BRNA%2Bgenomes%2Bof%2Bmouse%2Bhepatitis%2Bviruses.%26rft_id%253Dinfo%253Apmid%252F6165837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjM4LzIvNjYxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTUvMjAyMC4wMy4xMi45ODg4NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.988865v2.10" data-doi="10.1093/bioinformatics/bty191"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Minimap2: pairwise alignment for nucleotide sequences</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">34</span>, <span class="cit-fpage">3094</span>–<span class="cit-lpage">3100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D3094%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty191%26rft_id%253Dinfo%253Apmid%252F29750242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty191&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29750242&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F15%2F2020.03.12.988865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.988865v2.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liao</span>,  <span class="cit-name-given-names">C.L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Lai</span>,  <span class="cit-name-given-names">M.M.</span></span> (<span class="cit-pub-date">1992</span>). <span class="cit-article-title">RNA recombination in a coronavirus: recombination between viral genomic RNA and transfected RNA fragments</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">66</span>, <span class="cit-fpage">6117</span>–<span class="cit-lpage">6124</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DLiao%26rft.auinit1%253DC%2BL%26rft.volume%253D66%26rft.issue%253D10%26rft.spage%253D6117%26rft.epage%253D6124%26rft.atitle%253DRNA%2Brecombination%2Bin%2Ba%2Bcoronavirus%253A%2Brecombination%2Bbetween%2Bviral%2Bgenomic%2BRNA%2Band%2Btransfected%2BRNA%2Bfragments.%26rft_id%253Dinfo%253Apmid%252F1326662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI2Ni8xMC82MTE3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTUvMjAyMC4wMy4xMi45ODg4NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.988865v2.12" data-doi="10.1016/j.chom.2016.10.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lichinchi</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">B.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qin</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">He</span>,  <span class="cit-name-given-names">C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Rana</span>,  <span class="cit-name-given-names">T.M.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Dynamics of Human and Viral RNA Methylation during Zika Virus Infection</span>. <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr> <span class="cit-vol">20</span>, <span class="cit-fpage">666</span>–<span class="cit-lpage">673</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2BMicrobe%26rft.volume%253D20%26rft.spage%253D666%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2016.10.002%26rft_id%253Dinfo%253Apmid%252F27773536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2016.10.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27773536&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F15%2F2020.03.12.988865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.988865v2.13" data-doi="10.1006/viro.1996.0044"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Luytjes</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gerritsma</span>,  <span class="cit-name-given-names">H.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Spaan</span>,  <span class="cit-name-given-names">W.J.</span></span> (<span class="cit-pub-date">1996</span>). <span class="cit-article-title">Replication of synthetic defective interfering RNAs derived from coronavirus mouse hepatitis virus-A59</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">216</span>, <span class="cit-fpage">174</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DLuytjes%26rft.auinit1%253DW.%26rft.volume%253D216%26rft.issue%253D1%26rft.spage%253D174%26rft.epage%253D183%26rft.atitle%253DReplication%2Bof%2Bsynthetic%2Bdefective%2Binterfering%2BRNAs%2Bderived%2Bfrom%2Bcoronavirus%2Bmouse%2Bhepatitis%2Bvirus-A59.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fviro.1996.0044%26rft_id%253Dinfo%253Apmid%252F8614984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/viro.1996.0044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8614984&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F15%2F2020.03.12.988865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.988865v2.14" data-doi="10.1016/j.coviro.2017.01.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Menachery</span>,  <span class="cit-name-given-names">V.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>,  <span class="cit-name-given-names">R.L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Baric</span>,  <span class="cit-name-given-names">R.S.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Jumping species-a mechanism for coronavirus persistence and survival</span>. <abbr class="cit-jnl-abbrev">Curr Opin Virol</abbr> <span class="cit-vol">23</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr%2BOpin%2BVirol%26rft.volume%253D23%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.coviro.2017.01.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.coviro.2017.01.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.988865v2.15" data-doi="10.1128/MCB.7.4.1572"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Narayan</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ayers</span>,  <span class="cit-name-given-names">D.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rottman</span>,  <span class="cit-name-given-names">F.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maroney</span>,  <span class="cit-name-given-names">P.A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Nilsen</span>,  <span class="cit-name-given-names">T.W.</span></span> (<span class="cit-pub-date">1987</span>). <span class="cit-article-title">Unequal distribution of N6-methyladenosine in influenza virus mRNAs</span>. <abbr class="cit-jnl-abbrev">Mol Cell Biol</abbr> <span class="cit-vol">7</span>, <span class="cit-fpage">1572</span>–<span class="cit-lpage">1575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DNarayan%26rft.auinit1%253DP%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253D1572%26rft.epage%253D1575%26rft.atitle%253DUnequal%2Bdistribution%2Bof%2BN6-methyladenosine%2Bin%2Binfluenza%2Bvirus%2BmRNAs.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.7.4.1572%26rft_id%253Dinfo%253Apmid%252F3600638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjcvNC8xNTcyIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTUvMjAyMC4wMy4xMi45ODg4NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.988865v2.16" data-doi="10.3390/v1030895"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pathak</span>,  <span class="cit-name-given-names">K.B.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Nagy</span>,  <span class="cit-name-given-names">P.D.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Defective Interfering RNAs: Foes of Viruses and Friends of Virologists</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">1</span>, <span class="cit-fpage">895</span>–<span class="cit-lpage">919</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DViruses%26rft.aulast%253DPathak%26rft.auinit1%253DK.%2BB.%26rft.volume%253D1%26rft.issue%253D3%26rft.spage%253D895%26rft.epage%253D919%26rft.atitle%253DDefective%2BInterfering%2BRNAs%253A%2BFoes%2Bof%2BViruses%2Band%2BFriends%2Bof%2BVirologists.%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv1030895%26rft_id%253Dinfo%253Apmid%252F21994575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v1030895&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21994575&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F15%2F2020.03.12.988865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280413200027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.988865v2.17" data-doi="10.1016/bs.aivir.2016.08.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Snijder</span>,  <span class="cit-name-given-names">E.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Decroly</span>,  <span class="cit-name-given-names">E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">The Nonstructural Proteins Directing Coronavirus RNA Synthesis and Processing</span>. <abbr class="cit-jnl-abbrev">Adv Virus Res</abbr> <span class="cit-vol">96</span>, <span class="cit-fpage">59</span>–<span class="cit-lpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv%2BVirus%2BRes%26rft.volume%253D96%26rft.spage%253D59%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fbs.aivir.2016.08.008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/bs.aivir.2016.08.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.988865v2.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sola</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Almazan</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zuniga</span>,  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Enjuanes</span>,  <span class="cit-name-given-names">L.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Continuous and Discontinuous RNA Synthesis in Coronaviruses</span>. <abbr class="cit-jnl-abbrev">Annu Rev Virol</abbr> <span class="cit-vol">2</span>, <span class="cit-fpage">265</span>–<span class="cit-lpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu%2BRev%2BVirol%26rft.volume%253D2%26rft.spage%253D265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.988865v2.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stoiber</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Quick</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Egan</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eun Lee</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Celniker</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neely</span>,  <span class="cit-name-given-names">R.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Loman</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pennacchio</span>,  <span class="cit-name-given-names">L.A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Brown</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title"><em>De novo</em> Identification of DNA Modifications Enabled by Genome-Guided Nanopore Signal Processing</span>. bioRxiv, 094672.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.988865v2.20" data-doi="10.1093/nar/gkh923"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tomecki</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dmochowska</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gewartowski</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dziembowski</span>,  <span class="cit-name-given-names">A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Stepien</span>,  <span class="cit-name-given-names">P.P.</span></span> (<span class="cit-pub-date">2004</span>). <span class="cit-article-title">Identification of a novel human nuclear-encoded mitochondrial poly(A) polymerase</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">32</span>, <span class="cit-fpage">6001</span>–<span class="cit-lpage">6014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DTomecki%26rft.auinit1%253DR.%26rft.volume%253D32%26rft.issue%253D20%26rft.spage%253D6001%26rft.epage%253D6014%26rft.atitle%253DIdentification%2Bof%2Ba%2Bnovel%2Bhuman%2Bnuclear-encoded%2Bmitochondrial%2Bpoly%2528A%2529%2Bpolymerase%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh923%26rft_id%253Dinfo%253Apmid%252F15547249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkh923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15547249&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F15%2F2020.03.12.988865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225362100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.988865v2.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tvarogova</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Madhugiri</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bylapudi</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ferguson</span>,  <span class="cit-name-given-names">L.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Karl</span>,  <span class="cit-name-given-names">N.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Identification and Characterization of a Human Coronavirus 229E Nonstructural Protein 8-Associated RNA 3′-Terminal Adenylyltransferase Activity</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">93</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.988865v2.22" data-doi="10.1101/gr.247064.118"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Viehweger</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krautwurst</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lamkiewicz</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Madhugiri</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holzer</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Marz</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Direct RNA nanopore sequencing of full-length coronavirus genomes provides novel insights into structural variants and enables modification analysis</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr> <span class="cit-vol">29</span>, <span class="cit-fpage">1545</span>–<span class="cit-lpage">1554</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.247064.118%26rft_id%253Dinfo%253Apmid%252F31439691%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI5LzkvMTU0NSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzE1LzIwMjAuMDMuMTIuOTg4ODY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.988865v2.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Virtanen</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gommers</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oliphant</span>,  <span class="cit-name-given-names">T.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haberland</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reddy</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cournapeau</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burovski</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peterson</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weckesser</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bright</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">SciPy 1.0: fundamental algorithms for scientific computing in Python</span>. <abbr class="cit-jnl-abbrev">Nat Methods</abbr> <span class="cit-vol">17</span>, <span class="cit-fpage">261</span>–<span class="cit-lpage">272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D17%26rft.spage%253D261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.988865v2.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Warkocki</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liudkovska</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gewartowska</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mroczek</span>,  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Dziembowski</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Terminal nucleotidyl transferases (TENTs) in mammalian RNA metabolism</span>. <abbr class="cit-jnl-abbrev">Philos Trans R Soc Lond B Biol Sci</abbr> <span class="cit-vol">373</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.988865v2.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Williams</span>,  <span class="cit-name-given-names">G.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gokhale</span>,  <span class="cit-name-given-names">N.S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Horner</span>,  <span class="cit-name-given-names">S.M.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Regulation of Viral Infection by the RNA Modification N6-Methyladenosine</span>. <abbr class="cit-jnl-abbrev">Annu Rev Virol</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">235</span>–<span class="cit-lpage">253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu%2BRev%2BVirol%26rft.volume%253D6%26rft.spage%253D235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.988865v2.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">H.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ke</span>,  <span class="cit-name-given-names">T.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liao</span>,  <span class="cit-name-given-names">W.Y.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">N.Y.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Regulation of coronaviral poly(A) tail length during infection</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">e70548</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De70548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.988865v2.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yogo</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hirano</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hino</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shibuta</span>,  <span class="cit-name-given-names">H.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Matumoto</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">1977</span>). <span class="cit-article-title">Polyadenylate in the virion RNA of mouse hepatitis virus</span>. <abbr class="cit-jnl-abbrev">J Biochem</abbr> <span class="cit-vol">82</span>, <span class="cit-fpage">1103</span>–<span class="cit-lpage">1108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiochemistry%26rft.stitle%253DJ%2BBiochem%26rft.aulast%253DYOGO%26rft.auinit1%253DY.%26rft.volume%253D82%26rft.issue%253D4%26rft.spage%253D1103%26rft.epage%253D1108%26rft.atitle%253DPolyadenylate%2Bin%2Bthe%2BVirion%2BRNA%2Bof%2BMouse%2BHepatitis%2BVirus%26rft_id%253Dinfo%253Apmid%252F200604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=200604&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F15%2F2020.03.12.988865.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.988865v2.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">X.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">X.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Si</span>,  <span class="cit-name-given-names">H.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">C.L.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.12.988865v2.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-etal">et al.</span>(<span class="cit-pub-date">2020</span>). <span class="cit-article-title">A Novel Coronavirus from Patients with Pneumonia in China, 2019</span>. <abbr class="cit-jnl-abbrev">N Engl J Med</abbr> <span class="cit-vol">382</span>, <span class="cit-fpage">727</span>–<span class="cit-lpage">733</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft.volume%253D382%26rft.spage%253D727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.12.988865v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.12.988865v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.03.06.979625v2" title="Dataset Augmentation Allows Deep Learning-Based Virtual Screening To Better Generalize To Unseen Target Classes, And Highlight Important Binding Interactions" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.02.974246v2" title="Multi-dimensional predictions of psychotic symptoms via machine learning" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 15, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.12.988865v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/early/2020/03/15/2020.03.12.988865.external-links" target="_self" class="link-icon"><i class="icon-file"></i> <span class="title">Data/Code</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.12.988865v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.12.988865v2" class="active" data-icon-position="" data-hide-link-title="0">The architecture of SARS-CoV-2 transcriptome</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1196690" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-tf3oKcMAEqcxzaTK68fawNQrSwyr-xwZS7wW7aixucc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1196690" id="node1196690" data-pisa="biorxiv;2020.03.12.988865v2" data-pisa-master="biorxiv;2020.03.12.988865" data-seqnum="1196690" data-apath="/biorxiv/early/2020/03/15/2020.03.12.988865.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">The architecture of SARS-CoV-2 transcriptome</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Dongwan</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Joo-Yeon</span> <span class="nlm-surname">Lee</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jeong-Sun</span> <span class="nlm-surname">Yang</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Jun Won</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">V. Narry</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Hyeshik</span> <span class="nlm-surname">Chang</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.12.988865; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.12.988865 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.12.988865v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.12.988865v2&amp;title=The%20architecture%20of%20SARS-CoV-2%20transcriptome" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.12.988865v2&amp;title=The%20architecture%20of%20SARS-CoV-2%20transcriptome" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.12.988865v2&amp;text=The%20architecture%20of%20SARS-CoV-2%20transcriptome" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.12.988865v2&amp;title=The%20architecture%20of%20SARS-CoV-2%20transcriptome" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.12.988865v2&amp;t=The%20architecture%20of%20SARS-CoV-2%20transcriptome" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.12.988865v2&amp;title=The%20architecture%20of%20SARS-CoV-2%20transcriptome" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.12.988865v2&amp;title=The%20architecture%20of%20SARS-CoV-2%20transcriptome" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1196690" id="node1196690" data-pisa="biorxiv;2020.03.12.988865v2" data-pisa-master="biorxiv;2020.03.12.988865" data-seqnum="1196690" data-apath="/biorxiv/early/2020/03/15/2020.03.12.988865.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">The architecture of SARS-CoV-2 transcriptome</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Dongwan</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Joo-Yeon</span> <span class="nlm-surname">Lee</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jeong-Sun</span> <span class="nlm-surname">Yang</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Jun Won</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">V. Narry</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Hyeshik</span> <span class="nlm-surname">Chang</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.12.988865; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.12.988865 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=The%20architecture%20of%20SARS-CoV-2%20transcriptome&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Kim%2C+Dongwan&amp;rft.au=Lee%2C+Joo-Yeon&amp;rft.au=Yang%2C+Jeong-Sun&amp;rft.au=Kim%2C+Jun+Won&amp;rft.au=Kim%2C+V.+Narry&amp;rft.au=Chang%2C+Hyeshik"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1196690/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1196690/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1196690/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1196690/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1196690/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1196690/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1196690/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1196690/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1196690/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1196690/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1196690/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1196690/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.12.988865v2&amp;count=horizontal&amp;via=&amp;text=The%20architecture%20of%20SARS-CoV-2%20transcriptome&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.12.988865v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.12.988865v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.12.988865v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/genomics" class="highlight" data-icon-position="" data-hide-link-title="0">Genomics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
