<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">PLoS Comput Biol</span><span class="iso-abbrev" title="iso-abbrev">PLoS Comput. Biol</span><span class="publisher-id" title="publisher-id">plos</span><span class="pmc" title="pmc">ploscomp</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">PLoS Computational Biology</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1553-734X</span><span class="issn-epub" tagx="issn" title="issn-epub">1553-7358</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, USA</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/2740835">2740835</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/19779555">19779555</a></span><span class="publisher-id" title="publisher-id">09-PLCB-RA-0208R2</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pcbi.1000520">10.1371/journal.pcbi.1000520</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span>: <span class="subject" title="subject">Computational Biology/Comparative Sequence Analysis</span><span class="subject" title="subject">Evolutionary Biology/Bioinformatics</span><span class="subject" title="subject">Virology/Emerging Viral Diseases</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Bayesian Phylogeography Finds Its Roots</div> <div class="alt-title" title="alt-title">
Bayesian Phylogeography Finds Its Roots</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemey</span><span class="given-names" tagx="given-names" title="given-names">Philippe</span></span><a href="#aff1"><sup>1</sup></a><a href="#cor1"><sup>*</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">Andrew</span></span><a href="#aff2"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">Alexei J.</span></span><a href="#aff3"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">Marc A.</span></span><a href="#aff4"><sup>4</sup></a><a href="#aff5"><sup>5</sup></a></span></div><span class="citation_author_institution" id="aff1">[1], <span class="addr-line" title="addr-line">Department of Microbiology and Immunology, Katholieke Universiteit Leuven, Leuven, Belgium</span></span><span class="citation_author_institution" id="aff2">[2], <span class="addr-line" title="addr-line">Institute of Evolutionary Biology, University of Edinburgh, Edinburgh, United Kingdom</span></span><span class="citation_author_institution" id="aff3">[3], <span class="addr-line" title="addr-line">Department of Computer Science, University of Auckland, Auckland, New Zealand</span></span><span class="citation_author_institution" id="aff4">[4], <span class="addr-line" title="addr-line">Departments of Biomathematics and Human Genetics, David Geffen School of Medicine, University of California, Los Angeles, California, United States of America</span></span><span class="citation_author_institution" id="aff5">[5], <span class="addr-line" title="addr-line">Department of Biostatistics, School of Public Health, University of California, Los Angeles, California, United States of America</span></span><div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fraser</span><span class="given-names" tagx="given-names" title="given-names">Christophe</span></span><span class="role" tagx="role" title="role">Editor</span><a href="#edit1" /></span></div><span class="citation_author_institution" id="edit1">[], </span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">philippe.lemey@uz.kuleuven.be</span></div> <div class="fn-type-con" title="con"> <p>Analyzed the data: PL. Wrote the paper: PL MAS. Designed the study: PL MAS. Provided programming assistance: AR AJD. Conceived the original idea and developed the software implementation: MAS.</p> </div> </div><span class="pub-date-collection" title="pub-date-collection">collection: <span>2009-9</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2009-9-9</span></span><span class="volume" tagx="volume" title="volume">5</span><span class="issue" tagx="issue" title="issue">9</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1000520</span><span class="history" title="history"><span class="received" title="received">received: 2009-2-27</span><span class="accepted" title="accepted">accepted: 2009-8-25</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2009</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are properly credited.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>As a key factor in endemic and epidemic dynamics, the geographical distribution of viruses has been frequently interpreted in the light of their genetic histories. Unfortunately, inference of historical dispersal or migration patterns of viruses has mainly been restricted to model-free heuristic approaches that provide little insight into the temporal setting of the spatial dynamics. The introduction of probabilistic models of evolution, however, offers unique opportunities to engage in this statistical endeavor. Here we introduce a Bayesian framework for inference, visualization and hypothesis testing of phylogeographic history. By implementing character mapping in a Bayesian software that samples time-scaled phylogenies, we enable the reconstruction of timed viral dispersal patterns while accommodating phylogenetic uncertainty. Standard Markov model inference is extended with a stochastic search variable selection procedure that identifies the parsimonious descriptions of the diffusion process. In addition, we propose priors that can incorporate geographical sampling distributions or characterize alternative hypotheses about the spatial dynamics. To visualize the spatial and temporal information, we summarize inferences using virtual globe software. We describe how Bayesian phylogeography compares with previous parsimony analysis in the investigation of the influenza A H5N1 origin and H5N1 epidemiological linkage among sampling localities. Analysis of rabies in West African dog populations reveals how virus diffusion may enable endemic maintenance through continuous epidemic cycles. From these analyses, we conclude that our phylogeographic framework will make an important asset in molecular epidemiology that can be easily generalized to infer biogeogeography from genetic data for many organisms.</p> </div> <div class="abstract" title="abstract"> <div class="title" tagx="title" title="title">
Author Summary</div> <p>Spreading in time and space, rapidly evolving viruses can accumulate a considerable amount of genetic variation. As a consequence, viral genomes become valuable resources to reconstruct the spatial and temporal processes that are shaping epidemic or endemic dynamics. In molecular epidemiology, spatial inference is often limited to the interpretation of evolutionary histories with respect to the sampling locations of the pathogens. To test hypotheses about the spatial diffusion patterns of viruses, analytical techniques are required that enable us to reconstruct how viruses migrated in the past. Here, we develop a model to infer diffusion processes among discrete locations in timed evolutionary histories in a statistically efficient fashion. Applications to Avian Influenza A H5N1 and Rabies virus in Central and West African dogs demonstrate several advantages of simultaneously inferring spatial and temporal processes from gene sequences.</p> </div> <div class="counts" title="counts"><span class="page-count" title="page-count">page-count: </span></div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Phylogenetic inference from molecular sequences is becoming an increasingly popular tool to trace the patterns of pathogen dispersal. The time-scale of epidemic spread usually provides ample time for rapidly evolving viruses to accumulate informative mutations in their genomes <a href="#pcbi.1000520-Holmes1">[1]</a>. As a consequence, spatial diffusionâ€"among other processesâ€"can leave a measurable footprint in sampled gene sequences from these viruses <a href="#pcbi.1000520-Holmes1">[1]</a>. Reconstructing both the evolutionary history and spatial process from these sequences provides fundamental understanding of the evolutionary dynamics underlying epidemics, e.g. <a href="#pcbi.1000520-Rambaut1">[2]</a>,<a href="#pcbi.1000520-Russell1">[3]</a>. It is also hoped that these insights can be translated to effective intervention and prevention strategies <a href="#pcbi.1000520-Wallace1">[4]</a> and elucidating the key factors in viral transmission and gene flow over larger distances is central in formulating such strategies, e.g. <a href="#pcbi.1000520-Olsen1">[5]</a>.</p> <p>Phylogeographic analyses are a common approach in molecular ecology, connecting historical processes in evolution with spatial distributions that traditionally scale over millions of years <a href="#pcbi.1000520-Knowles1">[6]</a>. Many popular phylogeographic approaches <a href="#pcbi.1000520-Slatkin1">[7]</a>,<a href="#pcbi.1000520-Templeton1">[8]</a> can be remiss in ignoring the interaction between evolutionary processes and spatial-temporal domains. One first reconstructs a phylogeny omitting spatial information and then conditions the phylogeographic inferences on this reconstruction <a href="#pcbi.1000520-Holmes1">[1]</a>,<a href="#pcbi.1000520-Lanciotti1">[9]</a>, exploiting non-parametric tests to evaluate the significance of this conditional structure, e.g. <a href="#pcbi.1000520-Slatkin1">[7]</a>,<a href="#pcbi.1000520-Parker1">[10]</a>,<a href="#pcbi.1000520-Zarate1">[11]</a>. To draw conclusions about the epidemic origin or epidemiological linkage between locations, however, we require a reconstruction of the dispersal patterns and process throughout the evolutionary history. Considering locations as discrete states, this boils down to the well-known problem of ancestral state inference <a href="#pcbi.1000520-Slatkin1">[7]</a>. Parsimony is a popular heuristic approach to map characters onto a single phylogenetic tree <a href="#pcbi.1000520-DSwofford1">[12]</a>. Unfortunately, parsimony reconstructions ignore important sources of model uncertainty, including both uncertainty in the dispersal process as well as in the unknown phylogeny <a href="#pcbi.1000520-Ronquist1">[13]</a>. In addition, minimizing the number of state exchanges over a phylogeny is misleading when rates of evolution are rapid and when the state exchange probabilities are unequal <a href="#pcbi.1000520-Cunningham1">[14]</a>.</p> <p>Probabilistic methods draw on an explicit model of state evolution, permitting the ability to glimpse the complete state history over the entire phylogeny and conveniently draw statistical inferences <a href="#pcbi.1000520-Felsenstein1">[15]</a>â€"<a href="#pcbi.1000520-Minin1">[17]</a>. These analyses typically employ continuous-time Markov chain models for discrete state evolution analogous to common nucleotide, codon or amino acid substitution models <a href="#pcbi.1000520-Li1">[18]</a>. In contrast to parsimony, maximum likelihood-based reconstructions incorporate branch length differences in calculating the conditional probability of each ancestral state given the observed states at the phylogeny tips <a href="#pcbi.1000520-Cunningham1">[14]</a>. Bayesian reconstruction methods enable further generalization of this conditional probability analysis by removing the necessity to fix the Markov model parameters to obtain ancestral states and the necessity to specify a fixed tree topology with known branch lengths. Bayesian inference integrates conclusions over all possible parameter values but to achieve this, however, requires prior probability distributions for all aspects of the model.</p> <p>While probabilistic methods have been previously presented in a bio- or phylogeographic context, in particular Bayesian methods that integrate over phylogenetic uncertainty and Markov model parameter uncertainty <a href="#pcbi.1000520-Sanmartin1">[19]</a>, viral phylogeography studies have rarely made use of these developments. This may be a consequence of low awareness of existing software implementations for arbitrary continuous-time Markov chain models <a href="#pcbi.1000520-Pagel2">[20]</a>,<a href="#pcbi.1000520-Pond1">[21]</a> or a lack of appreciation for the uncertainty intrinsic in these reconstructions and the ease with which one can formally access epidemiological linkage through probabilistic approaches. A recent phylogeographic study of influenza A H5N1 introduces a heuristic non-parametric test to evaluate whether parsimony-inferred migration events between two particular locations occur at significantly high frequency <a href="#pcbi.1000520-Wallace2">[22]</a>. Null distributions for these frequencies arise from randomizing tip localities after false discovery rate correction to control for simultaneous testing issues. Although this procedure addresses concerns about statistical inference on sparse frequency matrices, the multiple comparison correction still results in a conservative estimate of significant migration events. Fully probabilistic approaches may further ease statistical inference, yet similar tests remain lacking for likelihood-based phylogeographic models.</p> <p>Advances in evolutionary inference methodology have frequently demonstrated how novel approaches can be appended to a sequence of analyses, in many cases starting from alignment to parameter estimation conditional on tree reconstructions. For example, demographic inference has involved genealogy reconstruction, estimating a time scale for the evolutionary history, and coalescent theory to quantify the demographic impact on this tree shape <a href="#pcbi.1000520-Pybus1">[23]</a>. It is well acknowledged that such sequential procedures ignore important sources of uncertainty because they generally purge error associated with each intermediate estimate. With the advent of novel computational techniques like Markov chain Monte Carlo (MCMC) sampling, it has become feasible to integrate many of the models involved and simultaneously estimate parameters of interest. Demographic inference is a well-known example of genealogy-based population genetics that benefited from these advances <a href="#pcbi.1000520-Drummond1">[24]</a>,<a href="#pcbi.1000520-Drummond2">[25]</a>. Bayesian MCMC methods also enable ancestral state reconstruction while simultaneously accounting for both phylogenetic and mapping uncertainty. Although this adds much needed credibility to ancestral reconstruction <a href="#pcbi.1000520-Ronquist1">[13]</a>, phylogeographic analysis would benefit even more from fully integrating spatial, temporal and demographic inference.</p> <p>Here, we implement ancestral reconstruction of discrete states in a Bayesian statistical framework for evolutionary hypothesis testing that is geared towards rooted, time-measured phylogenies. This allows character mapping in natural time scales, calibrated under a strict or relaxed molecular clock, in combination with several models of population size change. We use this full probabilistic approach to study viral phylogeography and extend the Bayesian implementation to a mixture model in which exchange rates in the Markov model are allowed to be zero with some probability. This Bayesian stochastic search variable selection (BSSVS) enables us to construct a Bayes factor test that identifies the most parsimonious description of the phylogeographic diffusion process. We also demonstrate how the geographical distribution of the sampling locations can be incorporated as prior specifications. Through feature-rich visual summaries of the space-time process, we demonstrate how this approach can offer insights into the spatial epidemic history of Avian influenza A-H5N1 and rabies viruses in Africa.</p> <p>The highly pathogenic avian influenza A-H5N1 viruses have been present for over a decade in Southern China and spread in multiple waves to different types of poultry in countries across Asia, Africa and Europe <a href="#pcbi.1000520-Wang1">[26]</a>. As a result, highly pathogenic A-H5N1 is now a panzootic disease and represents a continuous threat for human spill-over. Strong surveillance has been in place since these viruses caused extensive outbreaks, but the source and early dissemination pathways have remained uncertain. Because parsimony analysis has attempted to shed light on the latter <a href="#pcbi.1000520-Wallace2">[22]</a>, A-H5N1 provides an ideal example for comparison with Bayesian phylogeographic inference. Rabies is endemic in Asia and Africa, where the primary reservoir and vector for rabies virus (RABV) is the domestic dog. Phylogenetic analysis has revealed several genotypes of lyssaviruses (family <i>Rhabdoviridae</i>); genotype 1 has been found responsible for classical rabies, a fatal disease in terrestrial mammals throughout the world <a href="#pcbi.1000520-Bourhy1">[27]</a>,<a href="#pcbi.1000520-Bourhy2">[28]</a>. Here, we explore the phylogeographic history of RABV in domestic dogs in West Central Africa, using recently obtained sequence data, and evaluate the role of viral dispersal in maintaining RABV epidemic cycles.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p>We examine the evolution and spatial dispersion of two viral pathogens, Avian influenza A-H5N1 and rabies, to demonstrate the strengths and limitations of our discretized stochastic model for phylogeography.</p> <div class="avianinfluenzaa-h5n1" title="sec"> <div class="title" tagx="title" title="title">
Avian influenza A-H5N1</div> <p>To reconstruct the spatial dispersion patterns of Avian influenza A-H5N1, we analyze the hemagglutinin (HA) and neuraminidase (NA) gene datasets previously compiled by <a href="#pcbi.1000520-Wallace2">[22]</a>. Both datasets contain whole gene sequences from 192 A-H5N1 strains sampled from 20 localities across Eurasia. <a href="#pcbi.1000520-Wallace2">[22]</a> explore these genes individually, as well as concatenated together, through a strictly parsimony-based ancestral reconstruction method. Our Bayesian approach builds upon stochastic models and naturally affords quantification of uncertainty in both the ancestral state reconstructions and the underlying phylogeographic process. Further, as we are able to infer plausible root positions unlike the original analysis, we are not required to include outgroup sequences. To model sequence evolution, we employ the <a href="#pcbi.1000520-Hasegawa1">[29]</a> (HKY85) CTMC model of nucleotide substitution; we include discrete gamma-distributed rate variation <a href="#pcbi.1000520-Yang1">[30]</a> and assume an unknown, constant population-size coalescent process prior over the unknown phylogeny <a href="#pcbi.1000520-Kingman1">[31]</a>. Exploratory analyses using the less restrictive Bayesian skyline plot model indicate that the demographic prior has little influence on the phylogeographic inference (data not shown).</p> <p><a href="#pcbi-1000520-g001">Figure 1</a> summarizes the Bayesian maximum clade credibility (MCC) trees for the A-H5N1 HA and NA segments. An MCC tree is a point-estimate characterizing the posterior distribution of trees and represents the tree topology yielding the highest product of individual clade probabilities in their posterior sample <a href="#pcbi.1000520-Rambaut1">[2]</a>; branch lengths in these MCC trees are posterior median estimates.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pcbi.1000520.g001"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <div class="title" tagx="title" title="title">
Maximum clade credibility (MCC) phylogenies for hemagglutinin (HA) and neuraminidase (NA) genes of Avian influenza A-H5N1.</div> <p>We color branches according to the most probable location state of their descendent nodes. We use the same color coding as <a href="#pcbi.1000520-Wallace2">[22]</a>. To the upper left of both phylogenies are their root location state posterior probability distributions. A white arrow indicates the A/goose/Guangdong/1/96 sequence; a filled white circle identifies the most recent common ancestor of the Gs/GD lineage named after this strain.</p>   </div> <p>We further annotated the tree nodes with their most probable (modal) location states via color labelings. Although the nucleotide substitution rates are very similar across genes (HA: posterior mean <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, 95% Bayesian credible interval <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>; NA: <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span><span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>), the root lies considerably deeper in the NA tree resulting in a time scale that spans about twice the time of the HA tree. In combination with other topological differences between the trees, this difference strongly suggests past reassortment events between both segments, with the progenitor virus of the basal Hong Kong clade and a chicken strain from Hebei having acquired an NA segment from different lineages. Such events are not surprising given frequent reports of A-H5N1 reassortment in China, e.g. <a href="#pcbi.1000520-Wang1">[26]</a>, and the particular reassortment event for the basal Hong Kong clade has very recently been confirmed <a href="#pcbi.1000520-Vijaykrishna1">[32]</a>.</p> <p>Despite different time scales for HA and NA, most probable location states agree on Guangdong as the predominant location of these sequences throughout the majority of their evolutionary history. As an indication of the A-H5N1 epidemic origin, we consider the inferred location at the root of the trees (<a href="#pcbi-1000520-g001">Figure 1</a>). In the HA tree, Guangdong and Hong Kong share a vast majority of the posterior mass, neighboring locations in which surveillance efforts report early Avian influenza cases <a href="#pcbi.1000520-Xu1">[33]</a>,<a href="#pcbi.1000520-Subbarao1">[34]</a>. In the NA tree, although Hong Kong and Guangdong still obtain marginally higher support than other locations, all posterior root state probabilities are much closer to their prior probability. The substantially deeper NA root explains this difference as the depth greatly increases uncertainty on the root state. <a href="#pcbi-1000520-t001">Table 1</a> quantifies differences in ancestral state reconstruction uncertainty between the HA and NA trees using the Kullback-Leibler (KL) divergence measure (see <a href="#s4">Methods</a>). The NA tree results in considerably lower KL divergence than the HA tree, signifying a much smaller deviation of the posterior distribution of the root location from the prior. However, lack of phylogeographic structure in the data does not contribute to this difference because the NA trees return a lower association index (AI). This measure of spatial admixture is based on a sum across all nodes in the tree of the complement of the frequency of the most abundant location among all descendent taxa weighted by the depth of the node in the tree <a href="#pcbi.1000520-Wang2">[35]</a>, and thus bears some relationship with an entropy value for descendent taxa locations. The AI rescales this sum by its expectation for randomized location assignments and results in low values for relatively strong phylogeny-locality correlation whereas AI values close to one reflect complete spatial admixture. If the basal Hong Kong clade and a chicken strain from Hebei have indeed acquired a different NA through reassortment, the root state might be difficult to interpret for NA and is not necessarily the same as that for HA. Therefore, we also list uncertainty measures for the marginal posterior distribution of the most recent common ancestor (MRCA) of the Gs/GD lineage, named after the A/goose/Guangdong/1/96 strain very close to this node (indicated in <a href="#pcbi-1000520-g001">Figure 1</a>). KL divergence is again lower for this node in the NA phylogeny, but the difference is not as pronounced as for the root node.</p> <div class="table-wrap_UNKNOWN" id="pcbi-1000520-t001" position="float"><a href="https://dx.doi.org/10.1371/journal.pcbi.1000520.t001">10.1371/journal.pcbi.1000520.t001</a><span class="label" tagx="label" title="label">Table 1</span> <div class="title" tagx="title" title="title">
Mapping uncertainty and model exploration for Avian influenza A-H5N1 hemagglutinin (HA) and neuroaminidase (NA) genes.</div>  <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi-1000520-t001-1" /> <table frame="hsides" rules="groups"> <colgroup span="1"> <col align="left" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup>  <thead> <tr> <td align="left" rowspan="1" colspan="1">Data</td> <td align="left" rowspan="1" colspan="1">Model</td> <td colspan="2" align="left" rowspan="1">Kullback-Leibler</td> <td align="left" rowspan="1" colspan="1">Association index</td> </tr> <tr> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">root</td> <td align="left" rowspan="1" colspan="1">GsGD</td> <td align="left" rowspan="1" colspan="1" /> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">HA</td> <td align="left" rowspan="1" colspan="1">C</td> <td align="left" rowspan="1" colspan="1">1.4464</td> <td align="left" rowspan="1" colspan="1">2.1999</td> <td align="left" rowspan="1" colspan="1">0.21 (0.17â€"0.25)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">NA</td> <td align="left" rowspan="1" colspan="1">C</td> <td align="left" rowspan="1" colspan="1">0.0184</td> <td align="left" rowspan="1" colspan="1">1.6679</td> <td align="left" rowspan="1" colspan="1">0.14 (0.09â€"0.18)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">HA</td> <td align="left" rowspan="1" colspan="1">C, BSSVS</td> <td align="left" rowspan="1" colspan="1">1.7895</td> <td align="left" rowspan="1" colspan="1">1.4383</td> <td align="left" rowspan="1" colspan="1">0.24 (0.19â€"0.29)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">NA</td> <td align="left" rowspan="1" colspan="1">C, BSSVS</td> <td align="left" rowspan="1" colspan="1">0.5660</td> <td align="left" rowspan="1" colspan="1">1.1185</td> <td align="left" rowspan="1" colspan="1">0.20 (0.14â€"0.26)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">HA</td> <td align="left" rowspan="1" colspan="1">D, BSSVS</td> <td align="left" rowspan="1" colspan="1">1.7861</td> <td align="left" rowspan="1" colspan="1">1.4059</td> <td align="left" rowspan="1" colspan="1">0.25 (0.20â€"0.30)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">NA</td> <td align="left" rowspan="1" colspan="1">D, BSSVS</td> <td align="left" rowspan="1" colspan="1">0.5811</td> <td align="left" rowspan="1" colspan="1">1.1889</td> <td align="left" rowspan="1" colspan="1">0.23 (0.17â€"0.29)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Shared</td> <td align="left" rowspan="1" colspan="1">C</td> <td align="left" rowspan="1" colspan="1">HA: 1.4704</td> <td align="left" rowspan="1" colspan="1">HA: 2.2303</td> <td align="left" rowspan="1" colspan="1">HA: 0.21 (0.17â€"0.25)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">NA: 0.0321</td> <td align="left" rowspan="1" colspan="1">NA: 1.7281</td> <td align="left" rowspan="1" colspan="1">NA: 0.15 (0.10â€"0.19)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Shared</td> <td align="left" rowspan="1" colspan="1">C, BSSVS</td> <td align="left" rowspan="1" colspan="1">HA: 1.8965</td> <td align="left" rowspan="1" colspan="1">HA: 1.5844</td> <td align="left" rowspan="1" colspan="1">HA: 0.25 (0.21â€"0.30)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">NA: 0.7813</td> <td align="left" rowspan="1" colspan="1">NA: 1.2511</td> <td align="left" rowspan="1" colspan="1">NA: 0.22 (0.16â€"0.28)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Shared</td> <td align="left" rowspan="1" colspan="1">DI, BSSVS</td> <td align="left" rowspan="1" colspan="1">HA: 1.8038</td> <td align="left" rowspan="1" colspan="1">HA: 1.6086</td> <td align="left" rowspan="1" colspan="1">HA: 0.26 (0.21â€"0.31)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">NA: 0.7748</td> <td align="left" rowspan="1" colspan="1">NA: 1.3195</td> <td align="left" rowspan="1" colspan="1">NA: 0.23 (0.17â€"0.29)</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">HA (fixed)</td> <td align="left" rowspan="1" colspan="1">C</td> <td align="left" rowspan="1" colspan="1">1.5030</td> <td align="left" rowspan="1" colspan="1">2.5626</td> <td align="left" rowspan="1" colspan="1">0.18</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">HA (fixed)</td> <td align="left" rowspan="1" colspan="1">C, BSSVS</td> <td align="left" rowspan="1" colspan="1">1.7578</td> <td align="left" rowspan="1" colspan="1">1.7026</td> <td align="left" rowspan="1" colspan="1">0.18</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">HA (fixed)</td> <td align="left" rowspan="1" colspan="1">DI, BSSVS</td> <td align="left" rowspan="1" colspan="1">1.7235</td> <td align="left" rowspan="1" colspan="1">1.7364</td> <td align="left" rowspan="1" colspan="1">0.18</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>We report the Kullback-Leibler divergence between the posterior and prior location distributions of the root and the GsGD most recent common ancestor (MRCA), as well as a phylogeographic association index. We analyze genes independently, assuming equal phylogeographic models (Shared) and by fixing the HA phylogeny through phylogeographic models with prior rates proportional to a constant (C) or distance-informed (DI) and using Bayesian stochastic search variable selection (BSSVS).</p> </div> </div> </div> <p><a href="#pcbi-1000520-t001">Table 1</a> also explores the effects of distance-informed priors and BSSVS on location reconstruction. In general, the distance-informed priors furnish little advantage while inferring the root locations for both the HA and NA trees. If anything, KL divergences are slightly smaller for models involving distance-informed priors than those with flat priors. For these data, this finding is unsurprising as physical distances can be poor proxies for inverse-diffusion rates when dispersal results from a heterogenous mix of migratory birds, transport of poultry and poultry products, and trade of wild birds <a href="#pcbi.1000520-Kilpatrick1">[36]</a>. Finally, we also investigated the uncertainty that is accommodated by averaging over plausible trees by analyzing the HA data using a fixed tree topology and branch lengths (<a href="#pcbi-1000520-t001">Table 1</a>). The state reconstructions for the Gs/GD node in the fixed tree topology appear to ignore some uncertainty in comparison to integrating trees, which is not that evident for the root node. Although state reconstruction uncertainty is expected to be correlated among nodes, we also compared the KL divergence summed over all internal nodes, indicating much higher KL divergences using a fixed tree topology, e.g. for HA, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>: 292 vs 523 for integrating trees and a fixed tree respectively.</p> <p>Under BSSVS, we assume a truncated Poisson prior that assigns 50% prior probability on the minimal rate configuration, comprising 19 non-zero rates connecting the 20 locations. This model strongly favors reduced parameterizations. A sensitivity analysis with respect to larger Poisson prior means reinforces that the data prefer a minimal number of rates, as increasing the mean leads to lower overall marginal likelihoods (<a href="#pcbi-1000520-t002">Table 2</a>). BSSVS has a strong impact on root location reconstruction (<a href="#pcbi-1000520-g002">Figure 2</a>). Many localities that are weakly supported as the root location without BSSVS obtain negligible posterior probability under BSSVS. Consequentially, BSSVS leads to larger KL divergences for both the HA and NA root nodes (<a href="#pcbi-1000520-t001">Table 1</a>), suggesting that these reduced models more efficiently exploit the information content of the data. Interestingly, the posterior support for Guangxi increases under BSSVS at the expense of Guangdong in the HA phylogeny (<a href="#pcbi-1000520-g002">Figure 2</a>). This may be an artifact of the reversible CTMC assumption we enforce. Specifically, at the tips of the phylogeny, several pathways of migration into Guangxi are highly likely. Assuming reversibility dictates that migration out of this location occurs as well; placing these emigrations deeper in the phylogeny is most consistent with the data. Because many locations already receive very low posterior probabilities at the GsGD node, the increase in posterior probability for a few locations now seems to outweigh the marginal reductions in posterior probabilities for most other locations and results in lower KL divergences at this node.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pcbi.1000520.g002"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <div class="title" tagx="title" title="title">
Posterior location probabilities at two different nodes, the root and GsGD MRCA, for different Bayesian phylogeographic analyses of Avian influenza A-H5N1 HA and NA.</div> <p>The posterior probabilities are shown for different expectations, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, for the gamma priors on the rates; either <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, where <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> is an arbitrary constant, or <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, where <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> is the distance between location <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. A truncated Poisson (TP) prior with <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> was used in the Bayesian stochastic search variable selection (BSSVS) procedure.</p>   </div> <div class="table-wrap_UNKNOWN" id="pcbi-1000520-t002" position="float"><a href="https://dx.doi.org/10.1371/journal.pcbi.1000520.t002">10.1371/journal.pcbi.1000520.t002</a><span class="label" tagx="label" title="label">Table 2</span> <div class="title" tagx="title" title="title">
Sensitivity analysis to the expected number of migration rates for A-H5N1.</div>  <div class="alternatives" title="alternatives"> <div class="graphic" id="pcbi-1000520-t002-2" /> <table frame="hsides" rules="groups"> <colgroup span="1"> <col align="left" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup>  <thead> <tr> <td align="left" rowspan="1" colspan="1">Prior Mean</td> <td align="left" rowspan="1" colspan="1">ML (stdev)</td> <td align="left" rowspan="1" colspan="1">Posterior median (BCIs)</td> <td align="left" rowspan="1" colspan="1">KL divergence</td> </tr> <tr> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">root</td> <td align="left" rowspan="1" colspan="1">Gs/GD</td> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">log(2)</td> <td align="left" rowspan="1" colspan="1">âˆ’11339.343 (0.856)</td> <td align="left" rowspan="1" colspan="1">21 (19â€"22)</td> <td align="left" rowspan="1" colspan="1">1.7895</td> <td align="left" rowspan="1" colspan="1">1.4383</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">1</td> <td align="left" rowspan="1" colspan="1">âˆ’11339.670 (0.636)</td> <td align="left" rowspan="1" colspan="1">21 (19â€"23)</td> <td align="left" rowspan="1" colspan="1">1.7991</td> <td align="left" rowspan="1" colspan="1">1.4540</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">5</td> <td align="left" rowspan="1" colspan="1">âˆ’11341.197 (0.955)</td> <td align="left" rowspan="1" colspan="1">25 (22â€"29)</td> <td align="left" rowspan="1" colspan="1">1.7804</td> <td align="left" rowspan="1" colspan="1">1.4533</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">10</td> <td align="left" rowspan="1" colspan="1">âˆ’11342.463 (0.883)</td> <td align="left" rowspan="1" colspan="1">29 (24â€"34)</td> <td align="left" rowspan="1" colspan="1">1.7940</td> <td align="left" rowspan="1" colspan="1">1.7940</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">20</td> <td align="left" rowspan="1" colspan="1">âˆ’11343.429 (0.957)</td> <td align="left" rowspan="1" colspan="1">36 (29â€"43)</td> <td align="left" rowspan="1" colspan="1">1.7691</td> <td align="left" rowspan="1" colspan="1">1.5691</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>We report estimates of <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> marginal likelihoods (ML) with Monte Carlo error standard deviations, posterior medians and 95% Bayesian credible intervals (BCIs) of the inferred number of rates and KL divergences across a range of prior expectations.</p> </div> </div> </div> <p>By specifying a prior on the number of non-zero rates, we are able to construct Bayes factor (BF) tests for significance of individual rates (<a href="#pcbi-1000520-g003">Figure 3</a>). To visualize the epidemiological linkage that this test establishes, we employ Google Earth to display all rates with a non-zero expectancy that results in a BF larger than three. The majority of well-supported rates (16 out of 25 for both genes) are concordant between HA and NA. Some variation in support for different migration pathways between HA and NA was also noted in the original parsimony analysis <a href="#pcbi.1000520-Wallace2">[22]</a>. Importantly, Guangdong presents as an end-point in three well-supported epidemiological links in HA as well in NA. For HA, four migration links previously identified using the parsimony sFDR test (Guangdong to Fujian, Bangkok to Vietnam, Uthai Thani to Phitsanulok, and Qinghai to Novosibirsk) are also present in our well-supported symmetric rates. We can, however, not confirm epidemiological linkage directly between Guangdong and Indonesia. Despite having more supported rates by this Bayes factor test as compared to the parsimony sFDR test, it remains difficult to univocally identify the pathways seeding remote localities as Japan and Indonesia, and to connect the eastern diffusion network with the Chinese/Russian inlands. Distance-informed priors do not have strong influence on the Bayes factor test for significant rates.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pcbi.1000520.g003"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <div class="title" tagx="title" title="title">
Bayes factor (BF) test for significant non-zero rates in Avian influenza A-H5N1.</div> <p>Only rates supported by a BF greater than 3 are indicated. The color and thickness of the line represent the relative strength by which the rates are supported; thin white lines and thick red lines suggest relatively weak and strong support respectively. The maps are based on satellite pictures made available in Google Earth (<a href="http://earth.google.com">http://earth.google.com</a>).</p>   </div> <p>The presence of reassortment amongst the gene segments obfuscates phylogenetic inference for concatenated HA/NA sequence data. In this respect, it is interesting to note that previous parsimony reconstructions on a phylogeny for the concatenated HA and NA segments result in fewer significant diffusion rates compared to the separate analyses; <a href="#pcbi.1000520-Wallace2">[22]</a> found 2 for the concatenated alignment vs. 5 and 10 for HA and NA separately. The Bayesian framework enables a flexible combination of the data without having to specify a single phylogeny for both segments. To this end, we share the instantaneous rate matrix <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> between both segment phylogenies and sample all parameters in a single MCMC analysis. Without BSSVS, sharing the rate matrix results in slightly higher KL divergences for both the root node and the Gs/GD node in the HA and NA phylogenies (<a href="#pcbi-1000520-t001">Table 1</a>). <a href="#pcbi-1000520-g004">Figure 4</a> illustrates the well-supported rates based on the Bayes factor test of the shared rate matrix with a distance-informed prior. The shared data bring to light two possible pathways seeding the remote localities of Japan and Indonesia; these pathways suggest Guangxi and Hunan as possible source for Indonesia, and Hunan and Hebei as possible source for Japan.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pcbi.1000520.g004"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <div class="title" tagx="title" title="title">
Bayes factor test for significant non-zero rates for the combined HA and NA analysis.</div> <p>Only rates supported by a BF greater than 3 are indicated. The color and thickness of the line represent the relative strength by which the rates are supported; thin white lines and thick red lines suggest relatively weak and strong support respectively. The maps are based on satellite pictures made available in Google Earth (<a href="http://earth.google.com">http://earth.google.com</a>).</p>   </div> <p>A major advantage of the current phylogeography implementation is the ability to infer the migration process in natural time scales. The panels in <a href="#pcbi-1000520-g005">Figure 5</a> summarize the temporal dynamics of A-H5N1 spatial diffusion inferred using the shared rate matrix (KML files, <a href="#pcbi.1000520.s001">Dataset S1</a> and <a href="#pcbi.1000520.s002">S2</a> for HA and NA respectively, which enable visualizing the spread over time in Google Earth are available as supporting information). The lines connecting different locations represent branches in the MCC tree on which state exchanges occur and circle areas reflect the number of branches maintaining a particular state at that time point. By May 1997, Avian influenza lineages have accumulated in Guangdong, where the virus was originally isolated from a farmed goose <a href="#pcbi.1000520-Xu1">[33]</a>, and to a large extent in Hong Kong (both circles overlap in the figure), where 18 cases of human infection occurred in 1997. Although significant poultry culling efforts have been made in Hong Kong, the virus continues circulating in Southern China. By May 2001, the virus appears to have spread to Guangxi, Fujian, Shanghai and Hebei in the north of China. The diffusion process intensifies by May 2003; the virus reaches more remote locations like Japan, Vietnam and Indonesia. This is known as â€˜wave 1â€™ in Southeast Asia resulting in severe A-H5N1 outbreaks in 2003. Finally, A-H5N1 virus also spreads to the west in a second major transmission wave. Since this occurs after a major outbreak in migratory waterfowl at Qinghai Lake in Northern China, migratory birds could play a prominent role in this dissemination pathway <a href="#pcbi.1000520-Chen1">[37]</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pcbi.1000520.g005"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <div class="title" tagx="title" title="title">
Temporal dynamics of spatial Avian influenza A-H5N1 diffusion.</div> <p>We provide snapshots of the dispersal pattern for May 1997, 2001, 2003 and 2005. Lines between locations represent branches in the MCC tree along which the relevant location transition occurs. Location circle diameters are proportional to square root of the number of MCC branches maintaining a particular location state at each time-point. The white-green and yellow-magenta color gradients inform the relative age of the transitions for HA and NA respectively (older-recent). The maps are based on satellite pictures made available in Google Earth (<a href="http://earth.google.com">http://earth.google.com</a>).</p>   </div> </div> <div class="rabiesindogsinwestandcentralafrica" title="sec"> <div class="title" tagx="title" title="title">
Rabies in dogs in West and Central Africa</div> <p>We investigate the â€œAfrica 2â€� lineage of rabies transmitted by African dogs. This lineage forms one of the most divergent African rabies virus clades <a href="#pcbi.1000520-Bourhy2">[28]</a>,<a href="#pcbi.1000520-Davis1">[38]</a>. The data set we analyze here comprises 101 complete nucleoprotein (N) gene sequences sampled across 12 African countries including Chad, Niger, Cameroon, the Central African Republic, Benin, Sierra Leone, Mali, Mauritania, Guinea, Ivory Coast and Burkina Faso <a href="#pcbi.1000520-Talbi1">[39]</a>.</p> <p><a href="#pcbi-1000520-g006">Figure 6A</a> illustrates the location-annotated MCC phylogeny and demographic history for the African dog rabies lineages. We make this initial inference without either BSSVS or a distance-informed prior. To allow for variation in the underlying coalescent process giving rise to the phylogeny, we assume a piece-wise constant multiple change-point model on the effective population-size with 20 coalecent-interval groups <a href="#pcbi.1000520-Drummond2">[25]</a>. As generally observed for rabies viruses <a href="#pcbi.1000520-Bourhy2">[28]</a>, there exists strong signal of phylogenetic clustering according to sampling location. This observation is also reflected in a low AI (0.087 [0.043â€"0.132]). In contrast to the influenza phylogenies, however, there is no single location for which sampled sequences are phylogenetically dispersed throughout the whole phylogeny. Together with the relative deep time scale of the phylogeny and the absence of sequences sampled closer to the root, this hampers precise inference of the location state at the root. The root state posterior probabilities for all locations range between 0.059 and 0.125, with Chad and Guinea receiving the highest probability (0.115 and 0.124 respectively). These two locations are geographically distant from each other, but they both host viruses from the most basal lineage in the phylogeny (<a href="#pcbi-1000520-g006">Figure 6A</a>). Root inference is somewhat different using BSSVS and a distance-informed prior on the rates (<a href="#pcbi-1000520-g006">Figure 6B</a>). In this case, a more central location, Niger, obtains the highest posterior probability (0.144) but the KL divergence for the root state reconstruction is only marginally greater than zero (0.0645). The exploitation of BSSVS contributes to this effect; as for Avian influenza A-H5N1, distance-informed priors, alone, on the rates have little impact (data not shown).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pcbi.1000520.g006"><span class="label" tagx="label" title="label">Figure 6</span></a></div>   <div class="title" tagx="title" title="title">
Inferred phylogeny, demographic history and root location for Africa rabies virus.</div> <p>(A) MCC phylogeny with branches colored according to the most probable posterior location of their child nodes; superimposed under the phylogeny lies the inferred demographic history. (B) Root location posterior probabilities are shown for the standard discrete model (opaque) and for the BSSVS extension with, in addition, distance-informed priors on the infinitesimal migration rates (transparent). The distance-informed priors in the latter had little impact on the results (data not shown). Both the height and width of the cylinders are proportional to root location posterior probability; the same colors as the tree branches in (A) are used. The maps are based on satellite pictures made available in Google Earth (<a href="http://earth.google.com">http://earth.google.com</a>.).</p>   </div> <p>Although geographic origins remain elusive, we are able to identify locations that are epidemiologically linked using the BF test under BSSVS (<a href="#pcbi-1000520-g007">Figure 7</a>). Panel A in the figure highlights the rates yielding a BF<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>3. The resulting migration graph is markedly parsimonious with a distinctive East-West axis running along the Southern border of the Saharan desert. To glean how this graph reflects the migratory process acting along the rabies phylogeny, panel B projects each of the branches of the MCC phylogeny onto the geographic map. In this projection, we translate each branch into a geographic link that connects the branch's most probable starting and ending location states. The height of a link represents the relative length of the time elapsed on the link's corresponding branch, while the color gradient reflects the relative age of the migration. Many recent (magenta) migration events that occur in a relatively short time contribute to the well-supported East-West axis.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pcbi.1000520.g007"><span class="label" tagx="label" title="label">Figure 7</span></a></div>   <div class="title" tagx="title" title="title">
Inferred migration graph for African rabies virus and its reflection of the events reconstructed from the MCC tree.</div> <p>(A) Significantly non-zero migration rate using a Bayes factor test. Line thicknesses and the white-red color gradient relate to relative posterior migration rate expectations. (B) Projection of reconstructed migration events. Link heights indicate the relative durations of the branches upon which the inferred migration occurs, while the yellow-magenta color gradient informs the relative age of the transition (older-recent). The maps are based on satellite pictures made available in Google Earth (<a href="http://earth.google.com">http://earth.google.com</a>).</p>   </div> <p>Although the best supported rates mainly form an East-West axis, many transitions along this axis occur in the last three decades; this suggests that the axis is not representative of a relatively slow unidirectional migration wave. <a href="#pcbi-1000520-g008">Figure 8</a> reports the migration pathways over the last thirty years. These migration events accumulating over time, contingent on the estimated time of the branches on which they occur, demonstrate that RABV diffusion in West Africa is characterized by different simultaneous migration events in various directions rather than a unidirectional pattern, and that most of these migrations are short-range, occurring between neighboring countries.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pcbi.1000520.g008"><span class="label" tagx="label" title="label">Figure 8</span></a></div>   <div class="title" tagx="title" title="title">
West African dog rabies virus migration over the last three decades.</div> <p>The different panels represent temporal projections of reconstructed migration events every 15 years: A) 1977, B) 1992 and C) 2007. In these projections, each MCC branch is again translated into a geographic link that connects the branch's most probable starting and ending location states. The panels only show migration events or partial migration events that have occurred up to a particular date, assuming that the virus migrates at a constant rate over the inferred time span of the branch. The maps are based on satellite pictures made available in Google Earth (<a href="http://earth.google.com">http://earth.google.com</a>).</p>   </div> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>The Bayesian phylogeographic inference framework we present here incorporates the spatial and temporal dynamics of gene flow. In this study, we focus on pathogen diffusion because viral sequence sampling on a time-scale commensurate with the rate of substitution permits the inference of spatial patterns in real-time units. In addition, elucidating the phylodynamics of viral epidemics has important implications for public health management. We selected the Avian influenza A-H5N1 example to allow a convenient comparison of Bayesian ancestral state inference with the previous parsimony analysis; on the other hand, statistical analysis of the rabies migration in Africa up to this point has been largely unexplored. Both zoonoses represent a clear threat to human health. The frequent transmission of A-H5N1 from poultry or wild birds to humans suggest that the virus could emerge as or contribute genetically to the next human flu pandemic. Although the lack of a human-to-human transmission mechanism means that rabies will not emerge as a purely human disease, rabies infection causes a fatal neurological disease and at least 55,000 people die from this disease every year, mainly in the developing world <a href="#pcbi.1000520-Knobel1">[40]</a>.</p> <p>A Bayesian statistical approach presents many advantages over parsimony inference of ancestral states. First, MCMC offers a computational technique to integrate over an unknown phylogeny and unknown migration process as the former is not directly observable in nature and the latter is poorly understood. Accommodating this lack of knowledge protects against potentially severe bias, but can reduce the power to make inferences; phylogeographic analyses are no exception to this. One can regard this uncertainty itself as a â€˜mixed blessingâ€™ because whilst it can hamper drawing definitive conclusions <a href="#pcbi.1000520-Ronquist1">[13]</a>, it protect us from making overstated conclusions. For example, parsimony analysis of the influenza data establishes an epidemiological link between Guangdong and Indonesia <a href="#pcbi.1000520-Wallace2">[22]</a>. Bayesian inference does not confirm this conclusion and phylogenetic analysis of more recently obtained sequence data now identifies the progenitors of Indonesian strains in the Chinese province of Hunan <a href="#pcbi.1000520-Wang1">[26]</a>, a site which our shared analysis also identified as a possible source. Further, unlike parsimony, likelihood-based probabilistic methods consider branch lengths in ancestral reconstructions. The impact of the tree depth on root state reconstructions for the A-H5N1 genome segments clearly illustrates the importance of branch lengths. Moreover, probabilistic methods allow for estimating the relative posterior probability of each location state at any position along the phylogeny; this ability is indispensable in a hypothesis testing framework. As introduced by <a href="#pcbi.1000520-Sanmartin1">[19]</a> in a different setting, we also demonstrate how phylogeographic parameters can be estimated from different genomic segments without assuming the same evolutionary history. H5N1 reassortment, however, will have not have fully unlinked HA and NA evolutionary histories and partially shared ancestry may lead to overstated credibility in some aspects of the phylogeographic inference.</p> <p>Bayesian inference also proffers particular benefits within the class of likelihood-based methods, for example, by allowing for straightforward approaches to control model complexity. BSSVS naturally provides a BF test to identify significant non-zero migration rates. Further prior specification easily incorporates geographical detail of the sequence data. Although distance-informed priors appear to have little impact on the phylogeographic analyses presented here, both BSSVS and informed priors furnish new opportunities for hypothesis testing when comparing competing prior scenarios of the diffusion process. Examples include â€œgravity modelsâ€� <a href="#pcbi.1000520-Viboud1">[41]</a> in which infinitesimal rates become functions of the host population-sizes at the end-point locations and <i>a priori</i> structurally-fixed graphs <a href="#pcbi.1000520-Sanmartin1">[19]</a>. Finally, it has been recognized that an MCMC-based Bayesian framework is well-suited to bring together information of different kinds <a href="#pcbi.1000520-Drummond3">[42]</a>. The BEAST software, which has a strong focus on calibrated phylogenies and genealogies, elegantly illustrates this by offering a large number of complementary evolutionary models including substitution models, demographic and relaxed clock models that can be combined into a full probabilistic model <a href="#pcbi.1000520-Drummond4">[43]</a>. By adding spatial reconstruction to this arsenal of evolutionary models, the full probabilistic inference now brings us much closer to biogeographical history reconstruction from genetic data.</p> <p>Our primary motivation for exploiting BSSVS to select among all possible migration graphs is to elucidate the limited number of epidemiological links that appropriately explain the viral diffusion process. This parsimonious set both informs major modes of migration and reduces the high statistical variance that burdens estimation of all pairwise transition rates. Following this argument, less uncertainty on node state reconstructions would be expected when focusing on a parsimonious parameterization of the instantaneous rate matrix. The A-H5N1 analysis indeed indicates lower uncertainty of root state reconstructions. However, for some other internal nodes, we note the opposite behavior. We attribute this to the reversibility assumption in the rate matrix. Selection of reversible rates by BSSVS imposes more balanced transitions in the phylogeny among locations that could have unidirectional links in reality. Therefore, work is in progress to develop non-reversible models that may better fit a spatially expanding epidemic like A-H5N1 or recurring epidemic influenza emergence through source-sink dynamics <a href="#pcbi.1000520-Rambaut1">[2]</a>. Considerable technical hurdles remain to incorporate BSSVS procedures under such models. Because BSSVS places non-negligible probability on structural zeros in the rate matrix, we can not guarantee that all resulting rate matrices are diagonalizable, challenging stable computation. Bearing in mind the reversibility assumption, we pass no judgement on the origin of the A-H5N1 epidemic based on the frequency by which a location is present in well-supported rates, as was previously done in the parsimony analysis <a href="#pcbi.1000520-Wallace2">[22]</a>. Instead, we focus on node location state reconstructions throughout the phylogeny and their posterior probabilities. <a href="#pcbi-1000520-g001">Figure 1</a> suggests that, although Hong Kong and Guangdong both receive posterior support as root location states, the dominant location throughout the phylogeny and hence the more likely hub of diffusion is Guangdong. An inherent assumption of the discrete model of location change is that ancestral viruses necessarily reside at only the sampled locations of the extant viruses. In this respect, it is important to realize that the CTMC process should describe the underlying spatial dynamics more accurately as the sampling density increases. E.g., for A-H5N1 <a href="#pcbi.1000520-Wang1">[26]</a>, provide more recently obtained sequence data across a larger set of geographic locations; the data could inform further pathways seeding remote localities that remain elusive in our present analysis. In addition to tackling the reversibility assumption, it may also prove necessary to relax constant diffusion rates through time to realistically model phylogeographic processes in many situations. Covarion-like models <a href="#pcbi.1000520-Fitch1">[44]</a> and allowing different diffusion matrices across different time intervals in the phylogeny may help achieve these aims.</p> <p>Our rabies phylogeographic analysis confirms a longstanding presence of this viral lineage in West Africa <a href="#pcbi.1000520-Bourhy2">[28]</a>. The virus appears to have a constant population size for about 150 years during which, extrapolating from the more recent spatial dynamics, diffusion occurs continuously with no particular directionality (<a href="#pcbi-1000520-g006">Figure 6A</a>). These continuous dynamics explain why we can not achieve precise root state location inference based only on samples from the last 20 years. In the light of the constant population dynamics, however, the location of the MRCA may be epidemiologically irrelevant as the location probably does not necessarily represent the ultimate source of the rabies endemic. We note that our analysis does not include all currently available strains originating from Chad, which may add to the weak East-to-West dispersal signal revealed by a recent parsimony analysis <a href="#pcbi.1000520-Talbi1">[39]</a>. Our analysis confirms the model proposed for dog RABV in general; that is, of a series of spatially distinct clusters that experience relatively little contact among them <a href="#pcbi.1000520-Bourhy2">[28]</a>,<a href="#pcbi.1000520-Talbi1">[39]</a>. By providing a time scale for the seeding of these spatial clusters, we again demonstrate a clear advantage of the Bayesian inference over parsimony analysis. The ability to draw migrations over time also promotes a more precise dissection of local and temporal RABV movement on smaller geographical scales. After migration, the virus appears to establish local populations maintaining the viral lineage for at least a limited amount of time. These dynamics are reminiscent of a metapopulation model with continuous turnover of locally established viral populations. A long-standing rabies presence in West Africa is not surprising; already recognized since the late 60s, the territory plays a major role in the rabies-canid ecological balance <a href="#pcbi.1000520-Chalmers1">[45]</a>. It remains a remarkable feat that an acute and mainly fatal disease achieves prolonged endemicity. Because disease-induced mortality can rapidly deplete the number of susceptibles in a population, one expects epidemic cycles with oscillatory dynamics to occur. Rabies cycles and traveling waves have been well documented in wildlife across Europe and North America, e.g. <a href="#pcbi.1000520-Anderson1">[46]</a>, and more recently <a href="#pcbi.1000520-Hampson1">[47]</a>, demonstrate such cycles in African dogs. Because their periodicity is notably shorter than expected from epidemiological models, the authors argue that intervention responses also impact the epidemic cycles <a href="#pcbi.1000520-Hampson1">[47]</a>. Importantly, there is also a remarkable phase synchrony in rabies outbreaks across southern and eastern Africa, most pronounced for distances up to 1,000 km <a href="#pcbi.1000520-Hampson1">[47]</a>. For oscillating systems in particular, it is well known that dispersal can generate population synchrony <a href="#pcbi.1000520-Bjornstad1">[48]</a>. Because previous studies illustrate that even limited amounts of relatively local dispersal can generate synchrony in cyclical dynamics over large spatial scales <a href="#pcbi.1000520-Bjornstad1">[48]</a>, and that the resulting synchrony tends to decline as distance increases and varies through time <a href="#pcbi.1000520-Ranta1">[49]</a>, <a href="#pcbi.1000520-Hampson1">[47]</a> argue that dispersal could enforce synchrony in dog rabies epidemics across different countries. Our analysis clearly reveals rabies dispersal as a continuous dynamic process that could indeed be essential in maintaining epidemic cycles. As argued by <a href="#pcbi.1000520-Talbi1">[39]</a>, however, the rate of dispersal is probably not sufficiently high to explain the short epidemic cycles as suggested by <a href="#pcbi.1000520-Hampson1">[47]</a>. Nevertheless, we underscore that sustained and coordinated responses across political boundaries are necessary to control domestic dog rabies in Africa.</p> <p>Many questions in evolutionary biology require a biogeographical perspective on the population under investigation. We hope to have demonstrated that Bayesian phylogeographic framework can contribute significantly to evolutionary hypothesis testing, and, although we have focused on viral phylodynamics, this approach is generally applicable in molecular evolution. Employing geographically-informed priors delivers a first step in incorporating GIS information. Future developments like irreversible CTMC processes may offer even more biological realism.</p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <p>For many spatial scales and problems, geography can naturally be partitioned into a finite number of discrete sites <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> for <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. Examples of these situations include individual cities, islands or countries. Starting from the observed data, at the tips of the phylogeny <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> we record discretized locations <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, where <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> pin-points the sampling site of taxon <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. Unobserved in the spatial process are the locations of the most common recent ancestor <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> drawing from root distribution <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, the times at which the descendent taxa move and amongst which discrete sites these moves occur, a process which ultimately gives rise to <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. Conditioning on <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and the unobserved locations realized at each internal node <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span><a href="#pcbi.1000520-Ronquist1">[13]</a>,<a href="#pcbi.1000520-Pagel1">[16]</a>, suggest modeling the instantaneous locations <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> for taxa along each branch in <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> as independent continuous-time Markov chains (CTMCs). CTMCs are the same processes one commonly exploits to model sequence character evolution <a href="#pcbi.1000520-Felsenstein1">[15]</a>,<a href="#pcbi.1000520-Jukes1">[50]</a>. Although many readers are familiar with CTMCs, we here highlight several chain properties to which we turn later when discussing CTMC modeling limitations. CTMCs are the simplest stochastic processes that emit discrete outcomes as a continuous function of time. The processes are memoryless, in that the probability of transitioning to a new location only depends on the current location and not the past history. A <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> infinitesimal rate matrix <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> completely characterizes the CTMC process. Rate matrix <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> contains non-negative off-diagonal entries and all rows sum to 0, yielding a stochastic matrix upon exponentiation. Solving the Chapman-Kolmogorov equation that specifies the behavior of the chain yields the finite-time transition probabilities <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. In matrix form, </p><div class="disp-formula" title="disp-formula"> <div class="graphic" mimetype="image" position="float" /><span class="label" tagx="label" title="label">(1)</span></div>Determining the finite-time transition probabilities involves matrix exponentiation, generally accomplished through an eigen-decomposition of <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. Here, we restrict ourselves to infinitesimal rate matrices that yield only real eigen-values and eigen-vectors. Any matrix similar to a symmetric matrix ensures a real eigen-decomposition; consequentially, we formulate <div class="disp-formula" title="disp-formula"> <div class="graphic" mimetype="image" position="float" /><span class="label" tagx="label" title="label">(2)</span></div>where <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> is an overall rate scalar, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> is a <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> symmetric matrix and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. Infinitesimal rate matrices of this form generate reversible Markov chains, such that <div class="disp-formula" title="disp-formula"> <div class="graphic" mimetype="image" position="float" /><span class="label" tagx="label" title="label">(3)</span></div>placing many restrictions on the underlying geographic process. We discuss these limitations and modeling extensions that allow for irreversible chains in the <a href="#s3">Discussion</a>. In its most general time-reversibile (GTR) form, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> contains <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> free parameters, with <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> donating <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> together with <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>'s <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> off-diagonal entries. Following standard practice, we normalize entries in <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> such that <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> measures the expected (with respect to <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>) number of transitions per unit time <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>.<p /> <p>One illuminating perspective from which to view a CTMC is that of a random walk on a graph <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. From this perspective, the possible realizations of the chain <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> correspond to the vertex set of <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. Between the vertices lie edges that record the infinitesimal transition rates. For example, between <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> sits <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. As a continuous-time random walk, a particle, starting at vertex <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> at time <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, first waits an Exponential amount of time with rate <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and then randomly decides to which neighboring vertix <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> to move with probability <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. Now on <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, the process repeats. Neighboring vertices are those for which a single edge connects them. For character evolution, â€œcompleteâ€� graphs find almost exclusive use, such that edges exist between all pairs of vertices. At a minimum, however, the graph must remain â€œconnectedâ€�, such that it remains possible to walk between any two vertices on <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>.</p> <div class="" title="sec"> <div class="title" tagx="title" title="title" /> <div class="bayesianstochasticsearchvariableselection" title="sec"> <div class="title" tagx="title" title="title">
Bayesian stochastic search variable selection</div> <p>When GTR models find use modeling nucleotide substitution, most of the <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> possible transitions have non-neglible probability of occurring and are observed over the evolutionary history. Such is unlikely to be the case for geographical locations; given that there may be many sites and each taxon only has one location (the equivalent of just one single alignment site), we expect most transitions to rarely, if ever, occur. Consequentially, we suspect <i>a priori</i> that many infinitesimal rates are zero. From a statistical perspective, so many degrees of freedom fit to the limited data lead to extremely high variance estimates. These poor estimates arise not only for <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, but, more critically, for inference of the unobserved ancestral locations and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. We circumvent this sparse data problem by invoking BSSVS to select a parsimonious parameterization of <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. BSSVS enables us to simultaneously determine which infinitesimal rates are zero depending on the evidence in the data and efficiently infer the ancestral locations. As a beneficial by-product of BSSVS, directly quantifying the evidence about which rates are non-zero furnishes both the most likely migration patterns and the ability to test between competing migratory hypotheses.</p> <p>BSSVS is traditionally applied to model selection problems in a linear regression framework, in which statisticians start with a large number of potential predictors <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and ask which among these associate linearly with an <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>-dimensional outcome variable <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. For example, the full model becomes <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, where <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> is a <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>-dimensional vector of regression coefficients and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> is an <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>-dimensional vector of normally distributed errors with mean <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. When <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> for <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> differs significantly from 0, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> helps predict <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, otherwise <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> contributes little additional information and warrants removal from the model via forcing <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. Given potentially high correlation between the predictors, deterministic model search strategies tend not to find the optimal set of predictors unless one explores all possible subsets. This exploration is generally computationally impractical as there exist <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> such subsets and completely fails for <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>.</p> <p>Recent work in BSSVS <a href="#pcbi.1000520-Kuo1">[51]</a>,<a href="#pcbi.1000520-Chipman1">[52]</a> efficiently performs the exploration in two steps. In the first step, the approach augments the model state-space with a set of <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> binary indicator variables <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and imposes a prior <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> on the regression coefficients that has expectation <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and variance proportional to a <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> diagonal matrix with its entries equal to <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. If <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, then the prior variance on <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> shrinks to 0 and enforces <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> in the posterior. In the second step, MCMC explores the joint space of <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> simultaneously.</p> <p>To map BSSVS into the phylogeography setting, we consider selection among the <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> random graphs in which each of the <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> edges either exists or does not exist in <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. Let <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> be the binary indicator that an edge exists connecting <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. An equivalent parameterization specifies that <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> when <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> otherwise. So, rate matrix <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> plays an analogous role to the regression coefficients in BSSVS. An important difference is that <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> while <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, mandating alternative prior formulations.</p> </div> <div class="priorspecification" title="sec"> <div class="title" tagx="title" title="title">
Prior specification</div> <p>To specify a prior distribution over <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, we assume that each indicator acts <i>a priori</i> as an independent Bernoulli random variable (RV) with small success probability <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. The sum of independent Bernoulli RVs yields a Binomial distribution over their sum <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. In the limit that <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, this prior conveniently collapses to </p><div class="disp-formula" title="disp-formula"> <div class="graphic" mimetype="image" position="float" /><span class="label" tagx="label" title="label">(4)</span></div>where <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> is the prior expected number of edges in graph <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>.<p /> <p>We entertain two prior choices for <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. Diagonal vector <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> is the stationary distribution for the CTMC when all edges are included in the graph <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. For this complete graph, as the length of the random walk <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, one expects that <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> with probability <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. One natural choice says that there exists no spatial preference over the long-run and fixes <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> for all <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. However, sites may expound spatial preference over the long-run; for example, such preference can relate to known or unknown quantities such as population-size or geographic size of the site. In these situations, we estimate <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> simultaneously with the rest of the model by imposing the flat prior <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. Also non-informatively for small values, we take <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>.</p> <p>To complete the CTMC specification, we assume that all unnormalized rates in <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> are <i>a priori</i> independent and Gamma-distributed with prior expectation <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and variance <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, following in the vein of Bayesian SSVS. However, little previous work on prior formulation helps inform our choices of <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. This represents a critically important area of research. A common, yet arbitrary choice in the Bayesian phylogenetic literature assumes that rates draw from Exponential distributions, forcing <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. We follow this practice in light of there being no obvious way to elicit information on the variance of these rates. Finally, we explore two choices for setting the means. The first assumes no preference over rates, setting all <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, where <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> is an arbitrary constant; as, after normalization, only ratios of infinitesimal rates participate in the data likelihood, the actual value of <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> has no influence on the likelihood. The second is informed by the geographical distance between sites.</p> </div> <div class="distanceinformedprior" title="sec"> <div class="title" tagx="title" title="title">
Distance informed prior</div> <p>Considerable additional information exists about the sites <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and remains unused. Most notably, the geographic distances <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> between (the centroids) of sites is readily measurable. <i>A priori</i> we may believe that more distantly separated sites have the smallest infinitesimal migration rates, yielding </p><div class="disp-formula" title="disp-formula"> <div class="graphic" mimetype="image" position="float" /><span class="label" tagx="label" title="label">(5)</span></div>Other information is also surely helpful and application-specific. One example involving human hosts quantifies the availability of motorized transportation, such as air flights, between sites. We explore the utility BSSVS and distance-informed priors in our phylogeographic models.<p /> </div> <div class="bayesfactortestofsignificantdiffusionrates" title="sec"> <div class="title" tagx="title" title="title">
Bayes factor test of significant diffusion rates</div> <p>The Bayes factor (BF) for a particular rate <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> contributing to the migration graph is the posterior odds that rate <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> is non-zero divided by the equivalent prior odds, </p><div class="disp-formula" title="disp-formula"> <div class="graphic" mimetype="image" position="float" /><span class="label" tagx="label" title="label">(6)</span></div>where <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> is the posterior probability that rate <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> is non-zero, in this case the posterior expectation of indicator <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. Since we employ a truncated Poisson prior with mean <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, that assigns 50% prior probability on the minimal rate configuration (<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>), the prior probability <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> reduces to <div class="disp-formula" title="disp-formula"> <div class="graphic" mimetype="image" position="float" /><span class="label" tagx="label" title="label">(7)</span></div>We consider rates yielding a BF<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>3 as well supported diffusion rates constituting the migration graph.<p /> </div> </div> <div class="sequencedata,jointmodelingandposteriorsampling" title="sec"> <div class="title" tagx="title" title="title">
Sequence data, joint modeling and posterior sampling</div> <p>A strength of the Bayesian approach we exploit in this paper is the ability to integrate together into a joint model of spatial locations <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and aligned molecular sequence data <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> collected from the <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> taxa. The joint model affords a natural way to incorporate uncertainty about the unobserved phylogeny <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and the character substitution process giving rise to <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. We take a standard statistical phylogenetic approach and assume that a separate CTMC characterized by <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> generates <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. While we discuss specific choices about this process in the <a href="#s2">Results</a> sections, we do assume that the sequence and location CTMCs are independent given <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, enabling us to write the joint model posterior distribution as </p><div class="disp-formula" title="disp-formula"> <div class="graphic" mimetype="image" position="float" /><span class="label" tagx="label" title="label">(8)</span></div>Likelihoods <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> follow directly from Felsenstein's pruning algorithm <a href="#pcbi.1000520-Felsenstein1">[15]</a>, efficiently integrating over all possible locations and sequences at the root and internal nodes in <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>.<p /> <p>We approximate the joint posterior (8) and its marginalizations using MCMC implemented in the software package BEAST <a href="#pcbi.1000520-Drummond4">[43]</a>. We employ standard transition kernels over the parameter spaces of <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. To sample realizations of <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, we consider random-walk operators on the continuous portions and a specialized â€œbit-flipâ€� operator on the Bernoulli rate indicators <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. <a href="#pcbi.1000520-Drummond5">[53]</a> discuss this transition kernel further. Finally, in many situations, inference on the posterior distribution of the root and internal node states is of paramount interest. We implement a pre-order, tree-traversal algorithm in BEAST that allows researchers to generate realizations of the root and internal node states following <a href="#pcbi.1000520-Pagel2">[20]</a> and produce posterior summaries. Importantly, this procedure is not limited to phylogeographic models, making general ancestral state reconstruction available in BEAST for the first time.</p> <div class="summarizingposteriorlocationuncertainty" title="sec"> <div class="title" tagx="title" title="title">
Summarizing posterior location uncertainty</div> <p>An important statistical question asks to what extent the data inform our inference when fitting different phylogeographic models. A model of low statistical power makes poor use of the information in the data, while a successful model exploits this information to generate posterior distributions that are maximally different from prior beliefs. One primary outcome of a Bayesian phylogeographic study is the marginal posterior distribution of the root location <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. We calculate the Kullback-Leibler (KL) divergence <a href="#pcbi.1000520-Kullback1">[54]</a> from the root location prior <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> to summarize this information gain, </p><div class="disp-formula" title="disp-formula"> <div class="graphic" mimetype="image" position="float" /><span class="label" tagx="label" title="label">(9)</span></div>where <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. When the posterior and prior distributions are equal, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. In the examples in this paper, we fix <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> achieves its maximum <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> when the posterior places all estimable mass on a single location. From this perspective, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> plays the role of a measure of dispersion <a href="#pcbi.1000520-Gilulua1">[55]</a> or uncertainty. As a simple numerical summary, we also use <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> to explore the utility of BSSVS and distance-informed priors on drawing inference from phylogeographic models. Larger values signify that the model extracts more information from the data. To calculate KL divergence, we employ a uniform discrete distribution as reference distribution.<p /> </div> <div class="associationindex" title="sec"> <div class="title" tagx="title" title="title">
Association index</div> <p>Following existing phylogeographic approaches, we finally score the degree of spatial admixture using a modified association index (AI) <a href="#pcbi.1000520-Wang2">[35]</a>. For a given phylogeny <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and tip locations <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, we obtain the association value <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> by summing over each internal node <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, </p><div class="disp-formula" title="disp-formula"> <div class="graphic" mimetype="image" position="float" /><span class="label" tagx="label" title="label">(10)</span></div>where <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> counts the number of sampled locations descendent to <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> is the frequency of the highest frequency location amongst these descendents. Similar to <a href="#pcbi.1000520-Parker1">[10]</a>, we report the posterior distributions <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and the AI compares these distributions to those obtained by random permutation of the tip locations <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. Deviation from these permuted distributions reflected in low AI values suggests phylogeographic structure whereas AI values close to 1 suggest spatial admixture.<p /> </div> <div class="visualizingphylogeographicdiffusion" title="sec"> <div class="title" tagx="title" title="title">
Visualizing phylogeographic diffusion</div> <p>To summarize the posterior distribution of ancestral location states, we annotate nodes in the MCC tree with the modal location state for each node using TreeAnnotator, and visualize this tree using FigTree (available at <a href="http://tree.bio.ed.ac.uk/software">http://tree.bio.ed.ac.uk/software</a>). To provide a spatial projection, we convert the tree into a keyhole markup language (KML) file suitable for viewing with Google Earth (<a href="http://earth.google.com">http://earth.google.com</a>). We introduce the temporal information on the marked-up tree using the <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> KML-function to animate viral dispersal over the time. Example KML files for the Avian Influenza A HA and NA genes are included as supplementary files and software to convert annotated trees to KML is available from the authors on request.</p> </div> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting Information</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Dataset S1</span> <p>KML file for H5N1 diffusion over time as inferred from HA</p> <p>(2.13 MB XML)</p>  <div class="media" title="media"><a href="pcbi.1000520.s001.xml">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Dataset S2</span> <p>Supplementary information: KML file for H5N1 diffusion over time as inferred from NA</p> <p>(2.10 MB XML)</p>  <div class="media" title="media"><a href="pcbi.1000520.s002.xml">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We thank the Isaac Newton Institute for Mathematical Sciences, Cambridge, UK, for hosting us during its Phylogenetics Programme from which this research grew. We thank HervÃ© Bourhy and Shiraz Talbi for providing the rabies data and commenting on the manuscript.</p> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-COI-statement" title="COI-statement"> <p>The authors have declared that no competing interests exist.</p> </div> <div class="fn-type-financial-disclosure" title="financial-disclosure"> <p>PL was supported by a postdoctoral fellowship from the Fund for Scientific Research (FWO) Flanders and FWO grant G.0513.06. AR is supported by The Royal Society. MAS is supported by NIH R01 GM086887 and the Marsden Fund (with AJD). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="pcbi.1000520-Holmes1" /><span class="label" tagx="label" title="label">1</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">The phylogeography of human viruses.</span><span class="source" tagx="source" title="source">Molecular Ecology</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">745</span><span class="lpage" tagx="lpage" title="lpage">756</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15012753">15012753</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Rambaut1" /><span class="label" tagx="label" title="label">2</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nelson</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Viboud</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taubennerger</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">The genomic and epidemiological dynamics of human influenza A virus.</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">453</span><span class="fpage" tagx="fpage" title="fpage">615</span><span class="lpage" tagx="lpage" title="lpage">619</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18418375">18418375</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Russell1" /><span class="label" tagx="label" title="label">3</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russell</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">TC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barr</span><span class="given-names" tagx="given-names" title="given-names">IG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cox</span><span class="given-names" tagx="given-names" title="given-names">NJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garten</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">The global circulation of seasonal influenza A (H3N2) viruses.</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">320</span><span class="fpage" tagx="fpage" title="fpage">340</span><span class="lpage" tagx="lpage" title="lpage">346</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18420927">18420927</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Wallace1" /><span class="label" tagx="label" title="label">4</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wallace</span><span class="given-names" tagx="given-names" title="given-names">RG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fitch</span><span class="given-names" tagx="given-names" title="given-names">WM</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Influenza A H5N1 immigration is filtered out at some international borders.</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">e1697</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18301773">18301773</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Olsen1" /><span class="label" tagx="label" title="label">5</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olsen</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">HL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheung</span><span class="given-names" tagx="given-names" title="given-names">TYY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">AFY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fisk</span><span class="given-names" tagx="given-names" title="given-names">TL</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">Transmission of the severe acute respiratory syndrome on aircraft.</span><span class="source" tagx="source" title="source">N Engl J Med</span><span class="volume" tagx="volume" title="volume">349</span><span class="fpage" tagx="fpage" title="fpage">2416</span><span class="lpage" tagx="lpage" title="lpage">2422</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14681507">14681507</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Knowles1" /><span class="label" tagx="label" title="label">6</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knowles</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maddison</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">Statistical phylogeography.</span><span class="source" tagx="source" title="source">Molecular Ecology</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">2623</span><span class="lpage" tagx="lpage" title="lpage">2635</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12453245">12453245</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Slatkin1" /><span class="label" tagx="label" title="label">7</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Slatkin</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maddison</span><span class="given-names" tagx="given-names" title="given-names">WP</span></span></span><span class="year" tagx="year" title="year">1989</span><span class="mixed-article-title" title="mixed-article-title">A cladistic measure of gene flow inferred from the phylogenies of alleles.</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">123</span><span class="fpage" tagx="fpage" title="fpage">603</span><span class="lpage" tagx="lpage" title="lpage">613</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2599370">2599370</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Templeton1" /><span class="label" tagx="label" title="label">8</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Templeton</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Statistical phylogeography: methods for evaluating and minimizing inference errors.</span><span class="source" tagx="source" title="source">Molecular Ecology</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">789</span><span class="lpage" tagx="lpage" title="lpage">809</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15012756">15012756</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Lanciotti1" /><span class="label" tagx="label" title="label">9</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lanciotti</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gubler</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trent</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">1997</span><span class="mixed-article-title" title="mixed-article-title">Molecular evolution and phylogeny of dengue-4 viruses.</span><span class="source" tagx="source" title="source">Journal of General Virology</span><span class="volume" tagx="volume" title="volume">78</span><span class="fpage" tagx="fpage" title="fpage">2279</span><span class="lpage" tagx="lpage" title="lpage">2286</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9292015">9292015</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Parker1" /><span class="label" tagx="label" title="label">10</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parker</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">OG</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Correlating viral phenotypes with phylogeny: accounting for phylogenetic uncertainty.</span><span class="source" tagx="source" title="source">Infect Genet Evol</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">239</span><span class="lpage" tagx="lpage" title="lpage">246</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17921073">17921073</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Zarate1" /><span class="label" tagx="label" title="label">11</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zarate</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pond</span><span class="given-names" tagx="given-names" title="given-names">SLK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shapshak</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frost</span><span class="given-names" tagx="given-names" title="given-names">SDW</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Comparative study of methods for detecting sequence compartmentalization in human immunodeficiency virus type 1.</span><span class="source" tagx="source" title="source">J Virol</span><span class="volume" tagx="volume" title="volume">81</span><span class="fpage" tagx="fpage" title="fpage">6643</span><span class="lpage" tagx="lpage" title="lpage">6651</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17428864">17428864</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-DSwofford1" /><span class="label" tagx="label" title="label">12</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">D Swofford</span><span class="given-names" tagx="given-names" title="given-names">WM</span></span></span><span class="year" tagx="year" title="year">1992</span><span class="fpage" tagx="fpage" title="fpage">186</span><span class="lpage" tagx="lpage" title="lpage">223</span><span class="comment">Systematics, Historical Ecology, and North American Freshwater Fishes, Stanford University Press, chapter Parsimony, character-state reconstructions and evolutionary inferences</span></span></li> <li tag="ref"><a name="pcbi.1000520-Ronquist1" /><span class="label" tagx="label" title="label">13</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ronquist</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Bayesian inference of character evolution.</span><span class="source" tagx="source" title="source">Trends in Ecology and Evolution</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">475</span><span class="lpage" tagx="lpage" title="lpage">481</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16701310">16701310</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Cunningham1" /><span class="label" tagx="label" title="label">14</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cunningham</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Omland</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oakley</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="year" tagx="year" title="year">1998</span><span class="mixed-article-title" title="mixed-article-title">Reconstructing ancestral character states: a critical reappraisal.</span><span class="source" tagx="source" title="source">Trends in Ecology &amp;amp; Evolution</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">361</span><span class="lpage" tagx="lpage" title="lpage">366</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21238344">21238344</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Felsenstein1" /><span class="label" tagx="label" title="label">15</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Felsenstein</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">1981</span><span class="mixed-article-title" title="mixed-article-title">Evolutionary trees from DNA sequences: a maximum likelihood approach.</span><span class="source" tagx="source" title="source">Journal of Molecular Evolution</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">368</span><span class="lpage" tagx="lpage" title="lpage">376</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7288891">7288891</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Pagel1" /><span class="label" tagx="label" title="label">16</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pagel</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">1999</span><span class="mixed-article-title" title="mixed-article-title">The maximum likelihood approach to reconstructing ancestral character states of discrete characters on phylogenies.</span><span class="source" tagx="source" title="source">Systematic Biology</span><span class="volume" tagx="volume" title="volume">48</span><span class="fpage" tagx="fpage" title="fpage">612</span><span class="lpage" tagx="lpage" title="lpage">622</span></span></li> <li tag="ref"><a name="pcbi.1000520-Minin1" /><span class="label" tagx="label" title="label">17</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Minin</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Fast, accurate and simulation-free stochastic mapping.</span><span class="source" tagx="source" title="source">Philosophical Transactions of the Royal Society B: Biological Sciences</span><span class="volume" tagx="volume" title="volume">363</span><span class="fpage" tagx="fpage" title="fpage">2985</span><span class="lpage" tagx="lpage" title="lpage">2995</span></span></li> <li tag="ref"><a name="pcbi.1000520-Li1" /><span class="label" tagx="label" title="label">18</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">LiÃ²</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goldman</span><span class="given-names" tagx="given-names" title="given-names">N</span></span></span><span class="year" tagx="year" title="year">1998</span><span class="mixed-article-title" title="mixed-article-title">Models of molecular evolution and phylogeny.</span><span class="source" tagx="source" title="source">Genome Research</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">1233</span><span class="lpage" tagx="lpage" title="lpage">1244</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9872979">9872979</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Sanmartin1" /><span class="label" tagx="label" title="label">19</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanmartin</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van der Mark</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ronquist</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Inferring disperal: a Bayesian approach to phylogeny-based island biogeography, with special reference to the Canary Islands.</span><span class="source" tagx="source" title="source">Journal of Biogeography</span><span class="volume" tagx="volume" title="volume">35</span><span class="fpage" tagx="fpage" title="fpage">428</span><span class="lpage" tagx="lpage" title="lpage">449</span></span></li> <li tag="ref"><a name="pcbi.1000520-Pagel2" /><span class="label" tagx="label" title="label">20</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pagel</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meade</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barker</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Bayesian estimation of ancestral character states on phylogenies.</span><span class="source" tagx="source" title="source">Systematic Biology</span><span class="volume" tagx="volume" title="volume">53</span><span class="fpage" tagx="fpage" title="fpage">673</span><span class="lpage" tagx="lpage" title="lpage">684</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15545248">15545248</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Pond1" /><span class="label" tagx="label" title="label">21</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pond</span><span class="given-names" tagx="given-names" title="given-names">SLK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frost</span><span class="given-names" tagx="given-names" title="given-names">SDW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muse</span><span class="given-names" tagx="given-names" title="given-names">SV</span></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">Hyphy: hypothesis testing using phylogenies.</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">21</span><span class="fpage" tagx="fpage" title="fpage">676</span><span class="lpage" tagx="lpage" title="lpage">679</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15509596">15509596</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Wallace2" /><span class="label" tagx="label" title="label">22</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wallace</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">HoDac</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lathrop</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fitch</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">A statistical phylogeography of influenza A H5N1.</span><span class="source" tagx="source" title="source">Proceedings of the National Academy of Sciences, USA</span><span class="volume" tagx="volume" title="volume">104</span><span class="fpage" tagx="fpage" title="fpage">4473</span><span class="lpage" tagx="lpage" title="lpage">4478</span></span></li> <li tag="ref"><a name="pcbi.1000520-Pybus1" /><span class="label" tagx="label" title="label">23</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harvey</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">2000</span><span class="mixed-article-title" title="mixed-article-title">An integrated framework for the inference of viral population history from reconstructed genealogies.</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">155</span><span class="fpage" tagx="fpage" title="fpage">1429</span><span class="lpage" tagx="lpage" title="lpage">1437</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10880500">10880500</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Drummond1" /><span class="label" tagx="label" title="label">24</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Forsberg</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rodrigo</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">Measurably evolving populations.</span><span class="source" tagx="source" title="source">Trends in Ecology &amp;amp; Evolution</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">481</span><span class="lpage" tagx="lpage" title="lpage">488</span></span></li> <li tag="ref"><a name="pcbi.1000520-Drummond2" /><span class="label" tagx="label" title="label">25</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shapiro</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">O</span></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">Bayesian coalescent inference of past population dynamics from molecular sequences.</span><span class="source" tagx="source" title="source">Molecular Biology and Evolution</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">1185</span><span class="lpage" tagx="lpage" title="lpage">1192</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15703244">15703244</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Wang1" /><span class="label" tagx="label" title="label">26</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vijaykrishna</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duan</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bahl</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">JX</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Identification of the progenitors of Indonesian and Vietnamese avian influenza A (H5N1) viruses from southern China.</span><span class="source" tagx="source" title="source">J Virol</span><span class="volume" tagx="volume" title="volume">82</span><span class="fpage" tagx="fpage" title="fpage">3405</span><span class="lpage" tagx="lpage" title="lpage">3414</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18216109">18216109</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Bourhy1" /><span class="label" tagx="label" title="label">27</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bourhy</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kissi</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tordo</span><span class="given-names" tagx="given-names" title="given-names">N</span></span></span><span class="year" tagx="year" title="year">1993</span><span class="mixed-article-title" title="mixed-article-title">Molecular diversity of the lyssavirus genus.</span><span class="source" tagx="source" title="source">Virology</span><span class="volume" tagx="volume" title="volume">194</span><span class="fpage" tagx="fpage" title="fpage">70</span><span class="lpage" tagx="lpage" title="lpage">81</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8386891">8386891</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Bourhy2" /><span class="label" tagx="label" title="label">28</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bourhy</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reynes</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dunham</span><span class="given-names" tagx="given-names" title="given-names">EJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dacheux</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larrous</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">The origin and phylogeography of dog rabies virus.</span><span class="source" tagx="source" title="source">J Gen Virol</span><span class="volume" tagx="volume" title="volume">89</span><span class="fpage" tagx="fpage" title="fpage">2673</span><span class="lpage" tagx="lpage" title="lpage">2681</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18931062">18931062</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Hasegawa1" /><span class="label" tagx="label" title="label">29</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hasegawa</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kishino</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yano</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="year" tagx="year" title="year">1985</span><span class="mixed-article-title" title="mixed-article-title">Dating the human-ape splitting by a molecular clock of mitochondrial DNA.</span><span class="source" tagx="source" title="source">Journal of Molecular Evolution</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">160</span><span class="lpage" tagx="lpage" title="lpage">174</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3934395">3934395</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Yang1" /><span class="label" tagx="label" title="label">30</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span></span><span class="year" tagx="year" title="year">1995</span><span class="mixed-article-title" title="mixed-article-title">A space-time process model for the evolution of DNA sequences.</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">139</span><span class="fpage" tagx="fpage" title="fpage">993</span><span class="lpage" tagx="lpage" title="lpage">1005</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7713447">7713447</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Kingman1" /><span class="label" tagx="label" title="label">31</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kingman</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">2000</span><span class="mixed-article-title" title="mixed-article-title">Origins of the coalescent: 1974â€"1982.</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">156</span><span class="fpage" tagx="fpage" title="fpage">1461</span><span class="lpage" tagx="lpage" title="lpage">1463</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11102348">11102348</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Vijaykrishna1" /><span class="label" tagx="label" title="label">32</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vijaykrishna</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bahl</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riley</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duan</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">JX</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Evolutionary dynamics and emergence of panzootic H5N1 influenza viruses.</span><span class="source" tagx="source" title="source">PLoS Pathog</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">e1000161</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18818732">18818732</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Xu1" /><span class="label" tagx="label" title="label">33</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cox</span><span class="given-names" tagx="given-names" title="given-names">NJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="year" tagx="year" title="year">1999</span><span class="mixed-article-title" title="mixed-article-title">Genetic characterization of the pathogenic influenza A/Goose/Guangdong/1/96 (H5N1) virus: similarity of its hemagglutinin gene to those of H5N1 viruses from the 1997 outbreaks in Hong Kong.</span><span class="source" tagx="source" title="source">Virology</span><span class="volume" tagx="volume" title="volume">261</span><span class="fpage" tagx="fpage" title="fpage">15</span><span class="lpage" tagx="lpage" title="lpage">19</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10484749">10484749</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Subbarao1" /><span class="label" tagx="label" title="label">34</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Subbarao</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klimov</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katz</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Regnery</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lim</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">1998</span><span class="mixed-article-title" title="mixed-article-title">Characterization of an avian influenza A (H5N1) virus isolated from a child with a fatal respiratory illness.</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">279</span><span class="fpage" tagx="fpage" title="fpage">393</span><span class="lpage" tagx="lpage" title="lpage">396</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9430591">9430591</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Wang2" /><span class="label" tagx="label" title="label">35</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">TH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Donaldson</span><span class="given-names" tagx="given-names" title="given-names">YK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brettle</span><span class="given-names" tagx="given-names" title="given-names">RP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bell</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmonds</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">Identification of shared populations of human immunodeficiency virus type 1 infecting microglia and tissue macrophages outside the central nervous system.</span><span class="source" tagx="source" title="source">J Virol</span><span class="volume" tagx="volume" title="volume">75</span><span class="fpage" tagx="fpage" title="fpage">11686</span><span class="lpage" tagx="lpage" title="lpage">11699</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11689650">11689650</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Kilpatrick1" /><span class="label" tagx="label" title="label">36</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kilpatrick</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chmura</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibbons</span><span class="given-names" tagx="given-names" title="given-names">DW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fleischer</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marra</span><span class="given-names" tagx="given-names" title="given-names">PP</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Predicting the global spread of H5N1 avian influenza.</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span><span class="volume" tagx="volume" title="volume">103</span><span class="fpage" tagx="fpage" title="fpage">19368</span><span class="lpage" tagx="lpage" title="lpage">19373</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17158217">17158217</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Chen1" /><span class="label" tagx="label" title="label">37</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shinya</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Properties and dissemination of H5N1 viruses isolated during an influenza outbreak in migratory waterfowl in western China.</span><span class="source" tagx="source" title="source">J Virol</span><span class="volume" tagx="volume" title="volume">80</span><span class="fpage" tagx="fpage" title="fpage">5976</span><span class="lpage" tagx="lpage" title="lpage">5983</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16731936">16731936</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Davis1" /><span class="label" tagx="label" title="label">38</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davis</span><span class="given-names" tagx="given-names" title="given-names">PL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bourhy</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">The evolutionary dynamics of canid and mongoose rabies virus in Southern Africa.</span><span class="source" tagx="source" title="source">Arch Virol</span><span class="volume" tagx="volume" title="volume">152</span><span class="fpage" tagx="fpage" title="fpage">1251</span><span class="lpage" tagx="lpage" title="lpage">1258</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17401615">17401615</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Talbi1" /><span class="label" tagx="label" title="label">39</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Talbi</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Benedictis</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Faye</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">NakounÃ©</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Evolutionary history and dynamics of dog rabies virus in western and Central Africa.</span><span class="source" tagx="source" title="source">J Gen Virol</span><span class="volume" tagx="volume" title="volume">90</span><span class="fpage" tagx="fpage" title="fpage">783</span><span class="lpage" tagx="lpage" title="lpage">91</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19264663">19264663</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Knobel1" /><span class="label" tagx="label" title="label">40</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knobel</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cleaveland</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coleman</span><span class="given-names" tagx="given-names" title="given-names">PG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fevre</span><span class="given-names" tagx="given-names" title="given-names">EM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meltzer</span><span class="given-names" tagx="given-names" title="given-names">MI</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">Re-evaluating the burden of rabies in Africa and Asia.</span><span class="source" tagx="source" title="source">Bull World Health Organ</span><span class="volume" tagx="volume" title="volume">83</span><span class="fpage" tagx="fpage" title="fpage">360</span><span class="lpage" tagx="lpage" title="lpage">368</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15976877">15976877</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Viboud1" /><span class="label" tagx="label" title="label">41</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Viboud</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bjornstad</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simonsen</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Synchrony, waves and spatial hierarchies in the spread of influenza.</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">312</span><span class="fpage" tagx="fpage" title="fpage">447</span><span class="lpage" tagx="lpage" title="lpage">451</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16574822">16574822</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Drummond3" /><span class="label" tagx="label" title="label">42</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nicholls</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rodrigo</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Solomon</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="mixed-article-title" title="mixed-article-title">Estimating mutation parameters, population history and genealogy simultaneously from temporally spaced sequence data.</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">161</span><span class="fpage" tagx="fpage" title="fpage">1307</span><span class="lpage" tagx="lpage" title="lpage">1320</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12136032">12136032</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Drummond4" /><span class="label" tagx="label" title="label">43</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Beast: Bayesian evolutionary analysis by sampling trees.</span><span class="source" tagx="source" title="source">BMC Evol Biol</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">214</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17996036">17996036</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Fitch1" /><span class="label" tagx="label" title="label">44</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fitch</span><span class="given-names" tagx="given-names" title="given-names">WM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Markowitz</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="year" tagx="year" title="year">1970</span><span class="mixed-article-title" title="mixed-article-title">An improved method for determining codon variability in a gene and its application to the rate of fixation of mutations in evolution.</span><span class="source" tagx="source" title="source">Biochem Genet</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">579</span><span class="lpage" tagx="lpage" title="lpage">593</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5489762">5489762</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Chalmers1" /><span class="label" tagx="label" title="label">45</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chalmers</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scott</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="year" tagx="year" title="year">1969</span><span class="mixed-article-title" title="mixed-article-title">Ecology of rabies.</span><span class="source" tagx="source" title="source">Tropical Animal Health and Production</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">33</span><span class="lpage" tagx="lpage" title="lpage">55</span></span></li> <li tag="ref"><a name="pcbi.1000520-Anderson1" /><span class="label" tagx="label" title="label">46</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anderson</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jackson</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">May</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">1981</span><span class="mixed-article-title" title="mixed-article-title">Population dynamics of fox rabies in Europe.</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">289</span><span class="fpage" tagx="fpage" title="fpage">765</span><span class="lpage" tagx="lpage" title="lpage">771</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7464941">7464941</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Hampson1" /><span class="label" tagx="label" title="label">47</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hampson</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dushoff</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bingham</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bruckner</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ali</span><span class="given-names" tagx="given-names" title="given-names">YH</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Synchronous cycles of domestic dog rabies in sub-Saharan Africa and the impact of control efforts.</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span><span class="volume" tagx="volume" title="volume">104</span><span class="fpage" tagx="fpage" title="fpage">7717</span><span class="lpage" tagx="lpage" title="lpage">7722</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17452645">17452645</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Bjornstad1" /><span class="label" tagx="label" title="label">48</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bjornstad</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ims</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lambin</span><span class="given-names" tagx="given-names" title="given-names">X</span></span></span><span class="year" tagx="year" title="year">1999</span><span class="mixed-article-title" title="mixed-article-title">Spatial population dynamics: analyzing patterns and processes of population synchrony.</span><span class="source" tagx="source" title="source">Trends Ecol Evol</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">427</span><span class="lpage" tagx="lpage" title="lpage">432</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10511718">10511718</a></span></span></li> <li tag="ref"><a name="pcbi.1000520-Ranta1" /><span class="label" tagx="label" title="label">49</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ranta</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaitala</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lundberg</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">1998</span><span class="mixed-article-title" title="mixed-article-title">Population variability in space and time: the dynamics of synchronous population fluctuations.</span><span class="source" tagx="source" title="source">Oikos</span><span class="volume" tagx="volume" title="volume">83</span><span class="fpage" tagx="fpage" title="fpage">376</span><span class="lpage" tagx="lpage" title="lpage">382</span></span></li> <li tag="ref"><a name="pcbi.1000520-Jukes1" /><span class="label" tagx="label" title="label">50</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jukes</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cantor</span><span class="given-names" tagx="given-names" title="given-names">C</span></span></span><span class="year" tagx="year" title="year">1969</span><span class="mixed-article-title" title="mixed-article-title">Evolution of protein molecules.</span><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Munro</span><span class="given-names" tagx="given-names" title="given-names">H</span></span></span><span class="source" tagx="source" title="source">Mammaliam Protein Metabolism</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Academic Press</span><span class="fpage" tagx="fpage" title="fpage">21</span><span class="lpage" tagx="lpage" title="lpage">132</span></span></li> <li tag="ref"><a name="pcbi.1000520-Kuo1" /><span class="label" tagx="label" title="label">51</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuo</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mallick</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="year" tagx="year" title="year">1998</span><span class="mixed-article-title" title="mixed-article-title">Variable selection for regression models.</span><span class="source" tagx="source" title="source">Sankhya B</span><span class="volume" tagx="volume" title="volume">60</span><span class="fpage" tagx="fpage" title="fpage">65</span><span class="lpage" tagx="lpage" title="lpage">81</span></span></li> <li tag="ref"><a name="pcbi.1000520-Chipman1" /><span class="label" tagx="label" title="label">52</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chipman</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">George</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCulloch</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">2001</span><span class="mixed-article-title" title="mixed-article-title">The practical implementation of Bayesian model selection.</span><span class="source" tagx="source" title="source">IMS Lecture Notes â€" Monograph Series</span><span class="volume" tagx="volume" title="volume">38</span><span class="fpage" tagx="fpage" title="fpage">67</span><span class="lpage" tagx="lpage" title="lpage">134</span></span></li> <li tag="ref"><a name="pcbi.1000520-Drummond5" /><span class="label" tagx="label" title="label">53</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">In submission</span><span class="mixed-article-title" title="mixed-article-title">Bayesian random local clocks, or one rate to rule them all.</span><span class="source" tagx="source" title="source">Systematic Biology?</span></span></li> <li tag="ref"><a name="pcbi.1000520-Kullback1" /><span class="label" tagx="label" title="label">54</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kullback</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leibler</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">1951</span><span class="mixed-article-title" title="mixed-article-title">On information and sufficiency.</span><span class="source" tagx="source" title="source">Annals of Mathematical Statistics</span><span class="volume" tagx="volume" title="volume">22</span><span class="fpage" tagx="fpage" title="fpage">79</span><span class="lpage" tagx="lpage" title="lpage">86</span></span></li> <li tag="ref"><a name="pcbi.1000520-Gilulua1" /><span class="label" tagx="label" title="label">55</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gilulua</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haberman</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">1995</span><span class="mixed-article-title" title="mixed-article-title">Dispersion of categorical variables and penalty functions: derivation, estimation and comparability.</span><span class="source" tagx="source" title="source">Journal of the American Statistical Association</span><span class="volume" tagx="volume" title="volume">90</span><span class="fpage" tagx="fpage" title="fpage">290</span><span class="lpage" tagx="lpage" title="lpage">295</span></span></li> </ul> </div> </div>  </body></html>