<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Curr Opin Virol</span><span class="iso-abbrev" title="iso-abbrev">Curr Opin Virol</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Current Opinion in Virology</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1879-6257</span><span class="issn-epub" tagx="issn" title="issn-epub">1879-6265</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Elsevier B.V.</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7102768">7102768</a></span><span class="publisher-id" title="publisher-id">S1879-6257(18)30108-1</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1016/j.coviro.2018.12.006">10.1016/j.coviro.2018.12.006</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Parallel molecular evolution and adaptation in viruses</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gutierrez</span><span class="given-names" tagx="given-names" title="given-names">Bernardo</span></span><span class="email" tagx="email" title="email">bernardo.gutierrez@zoo.ox.ac.uk</span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Escalera-Zamudio</span><span class="given-names" tagx="given-names" title="given-names">Marina</span></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">Oliver G</span></span><span class="email" tagx="email" title="email">oliver.pybus@zoo.ox.ac.uk</span></span><span class="citation_author_institution" id="aff0005">[], </span></div><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2019-1-1</span></span><span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2019-2</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-1-1</span></span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">90</span><span class="lpage" tagx="lpage" title="lpage">96</span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Elsevier B.V.</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2018</span></span><span class="license" title="license"><span class="license-p" title="license-p">Since January 2020 Elsevier has created a COVID-19 resource centre with free information in English and Mandarin on the novel coronavirus COVID-19. The COVID-19 resource centre is hosted on Elsevier Connect, the company's public news and information website. Elsevier hereby grants permission to make all its COVID-19-related research that is available on the COVID-19 resource centre - including this research content - immediately available in PubMed Central and other publicly funded repositories, such as the WHO COVID database with rights for unrestricted research re-use and analyses in any form or by any means with acknowledgement of the original source. These permissions are granted for free by Elsevier for as long as the COVID-19 resource centre remains active.</span></span></div> <div class="abstract" title="abstract"> <div class="title" tagx="title" title="title">
Graphical abstract</div> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/" /></div>  </div> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Parallel molecular evolution is the independent evolution of the same genotype or phenotype from distinct ancestors. The simple genomes and rapid evolution of many viruses mean they are useful model systems for studying parallel evolution by natural selection. Parallel adaptation occurs in the context of several viral behaviours, including cross-species transmission, drug resistance, and host immune escape, and its existence suggests that at least some aspects of virus evolution and emergence are repeatable and predictable. We introduce examples of virus parallel evolution and summarise key concepts. We outline the difficulties in detecting parallel adaptation using virus genomes, with a particular focus on phylogenetic and structural approaches, and we discuss future approaches that may improve our understanding of the phenomenon.</p> </div> </div> </div> <div class="body" title="body"> <p id="par0140"> </p><div class="boxed-text" tagx="boxed-text" title="boxed-text"> <p id="par0005"><b>Current Opinion in Virology</b> 2019, <b>34</b>:90–96</p> <p id="par0010">This review comes from a themed issue on <b><b>Emerging viruses: interspecies transmission</b></b></p> <p id="par0015">Edited by <b>Adolfo García-Sastre</b> and <b>Juergen A Richt</b></p> <p id="par0020">For a complete overview see the <a href="http://www.sciencedirect.com/science/journal/18796257/34"><u>Issue</u></a> and the <a href="https://doi.org/10.1016/j.coviro.2019.02.003"><u>Editorial</u></a></p> <p id="par0025">Available online 28th January 2019</p> <p id="par0030"><a href="https://doi.org/10.1016/j.coviro.2018.12.006"><b>https://doi.org/10.1016/j.coviro.2018.12.006</b></a></p> <p id="par0035">1879-6257/© 2018 Elsevier B.V. All rights reserved.</p> </div> <p /> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="par0040">The processes that drive the cross-species transmission and emergence of viruses in natural systems are diverse and involve a range of ecological, evolutionary, and genetic factors. Amongst the evolutionary mechanisms, molecular adaptation by natural selection is a key phenomenon that entails the generation and spread of beneficial mutations that increase virus fitness in a specific environment. If rapidly evolving virus populations repeatedly experience comparable environmental changes and similar selective pressures, then viruses may exhibit what is known as parallel evolution.</p> <p id="par0045">The concept of parallel evolution is applied to organisms across the tree of life and can refer to all manners of phenotypes and traits at different levels of biological organisation [<a href="#bib0005">1</a>]. Parallel evolution manifests itself in a variety of ways, from the repeated fixation of single point mutations or larger genetic changes (e.g. indels or genome rearrangements) [<a href="#bib0010">2</a>], to the evolution of structural, functional or behavioural phenotypes. The terms ‘parallel’ and ‘convergent’ evolution are sometimes given distinct definitions: convergent evolution refers to the independent evolution of similar traits from different ancestral starting points [<a href="#bib0015">3</a>], whereas parallel evolution describes the independent emergence of similar traits from the same state [<a href="#bib0020">4</a>]. Here, we use ‘parallel’ to refer to both situations. Further, we use ‘adaptation’ to refer to evolutionary change through positive natural selection, and ‘evolution’ to encompass change via any process (e.g. by random genetic drift).</p> <p id="par0050">Parallel evolution in viruses may arise from adaptation to new host species [<a href="#bib0025">5</a>] or different host demographic structures [<a href="#bib0030">6</a>], evasion of host immune responses [<a href="#bib0035">7</a>,<a href="#bib0040">8</a>] or to circumvent anti-viral drugs [<a href="#bib0045">9</a>]. These situations involve changes to the virus’ environment that generate strong selective pressures, favouring the recurrent evolution of certain genotypes. Here, we summarise the theoretical concepts behind parallel molecular evolution in viruses, and explore some of the methods that can be used to identify such phenomena. We highlight the importance of phylogenetic methods to detect recurring evolutionary patterns generated by natural selection.</p> </div> <div class="parallelmolecularevolutioninviruses" title="sec"> <div class="title" tagx="title" title="title">
Parallel molecular evolution in viruses</div> <p id="par0055">Viruses are useful model systems for studying parallel evolution and adaptation. Many viruses, particularly RNA viruses, can adapt rapidly due to a combination of high mutation rates, large population sizes, short generation times, and large mutational selection coefficients [<a href="#bib0050">10<sup>••</sup></a>]. Further, the small genomes of RNA viruses (and some DNA viruses) may limit the range of genetic solutions available to viruses as they respond to environmental change. In contrast, the larger genomes of DNA viruses, such as Myxoma [<a href="#bib0055">11</a>], may offer more potential genetic routes to the same phenotype.</p> <p id="par0060">Perhaps the best described example of virus parallel evolution is the development of anti-viral drug resistance by the human immunodeficiency virus (HIV). HIV exhibits all the features of rapid adaptation listed above [<a href="#bib0060">12<sup>•</sup></a>]; consequently, many HIV drug resistance mutations have been identified, which independently and repeatedly arise during chronic infection in different HIV patients [<a href="#bib0060">12<sup>•</sup></a>,<a href="#bib0065">13</a>,<a href="#bib0070">14</a>]. Interestingly, even within a genome as constrained as HIV’s, different mutations can confer resistance to specific drugs [<a href="#bib0075">15</a>]. HIV also displays rapid and repeatable evolution to host immune responses, as demonstrated by the parallel evolution in different patients of escape mutations to HLA-restricted T-cell responses [<a href="#bib0035">7</a>].</p> <p id="par0065">Viruses that register frequent cross-species transmission events might also exhibit parallel evolution as a result of adaptation to new host environments [<a href="#bib0080">16</a>]. Barriers to virus replication in a new host may include (i) lack suitable receptors for virus cell entry, (ii) innate and adaptive immune responses, (iii) reduced replication efficacy in specific cell types, and (iv) mechanisms of virion release from infected cells [<a href="#bib0085">17<sup>••</sup></a>]. For example, HIV-1 groups M, N and O are derived from separate cross-species transmissions to humans of simian immunodeficiency viruses (SIVs) in chimpanzees and gorillas. Following spillover, these groups independently acquired a Met → Arg amino acid change in the retroviral Gag protein [<a href="#bib0090">18</a>]. Under experimental conditions, this mutation increases SIV replication in human lymphoid tissue [<a href="#bib0095">19</a>]. Rabies virus (RABV) offers another example: the RABV lineage that infects dogs has changed host species on multiple occasions, causing outbreaks in other <i>Carnivora</i> species. Amongst these events, parallel amino acid changes were observed in two separate RABV zoonoses from dogs to ferret-badgers (a Leu → Ser mutation in the RABV nucleoprotein and a Lys → Arg change in the polymerase) [<a href="#bib0025">5</a>].</p> <p id="par0070">Parallel evolution and adaptation may also target larger genome regions, and sites involved in post-translational modification. For example, the hemagglutinin of H7 subtype avian influenza virus (AIV) can acquire a N-glycosylation site when the virus jumps from wild to domestic birds [<a href="#bib0100">20</a>], and highly pathogenic strains of AIV have emerged independently many times, through the evolution of different polybasic cleavage sites in hemagglutinin [<a href="#bib0105">21</a>]. For HIV-1, it is now understood that, after zoonosis, groups M and O independently evolved the ability to antagonise human tetherin (an anti-viral restriction factor), but did so using entirely different retroviral accessory genes [<a href="#bib0110">22</a>,<a href="#bib0115">23</a>].</p> </div> <div class="parallelmolecularevolution:selectionandadaptation" title="sec"> <div class="title" tagx="title" title="title">
Parallel molecular evolution: selection and adaptation</div> <p id="par0075">Parallel molecular evolution has been described for many viruses both under laboratory conditions and in natural systems. <a href="#tbl0005">Table 1</a> provides an illustrative selection of RNA virus examples. Whilst theoretical studies have focused on the ability of natural selection to increase the probability of parallel evolution [<a href="#bib0120">24</a>], experimental studies have indicated the potential impact of other factors, including epistatic interactions amongst sites and heterogeneity of mutation rates across the genome [<a href="#bib0125">25<sup>••</sup></a>,<a href="#bib0130">26</a>]. <div class="table-wrap_UNKNOWN" position="float" id="tbl0005"><span class="label" tagx="label" title="label">Table 1</span> </div></p><p>Selected examples of virus parallel evolution</p> [alt-text] Table 1 <table frame="hsides" rules="groups"> <thead> <tr> <th align="left">Virus</th> <th align="left">Gene</th> <th align="left">Mutation</th> <th align="center">Details</th> <th align="left">Reference</th> </tr> </thead> <tbody> <tr> <td colspan="5" align="center"><i>Laboratory conditions</i></td> </tr> <tr> <td align="left">Vesicular stomatitis virus (VSV)</td> <td align="left">Various</td> <td align="left">Various</td> <td align="left">Independent populations sequentially passaged showed 19 loci that evolved shared alleles.</td> <td align="left">[<a href="#bib0320">64</a>]</td> </tr> <tr> <td align="left">Middle East respiratory syndrome coronavirus (MERS-CoV)</td> <td align="left">S</td> <td align="left">S465F</td> <td align="left">Replicated sequential passaging in DDP4-expressing cells resulted in the mutation Ser → Phe in the spike protein</td> <td align="left">[<a href="#bib0325">65</a>]</td> </tr> <tr> <td colspan="5" align="left">  </td> </tr> <tr> <td colspan="5" align="center"><i>Natural systems</i></td> </tr> <tr> <td align="left">Human immunodeficiency virus (HIV)</td> <td align="left">Gag</td> <td align="left">M30R, Y30R</td> <td align="left">A change from Met or Lys → Arg observed in HIV groups M, N and O.</td> <td align="left">[<a href="#bib0090">18</a>]</td> </tr> <tr> <td align="left" valign="middle" rowspan="2">Rabies virus (RABV)</td> <td align="left">N</td> <td align="left">L374S</td> <td align="left" valign="middle" rowspan="2">Substitutions in the nucleoprotein and polymerase observed in two independent zoonoses.</td> <td align="left" valign="middle" rowspan="2">[<a href="#bib0025">5</a>]</td> </tr> <tr> <td align="left">L</td> <td align="left">K200R</td> </tr> <tr> <td align="left" valign="middle" rowspan="2">Influenza A virus (IAV)</td> <td align="left">HA</td> <td align="left">G228S</td> <td align="left">A Gly → Ser change observed in different HX subtypes that circulate in humans, compared to avian and equine sequences.</td> <td align="left">[<a href="#bib0330">66</a>]</td> </tr> <tr> <td align="left">PB2</td> <td align="left">E627K</td> <td align="left">A Glu → Lys change at position 627 of PB2 increases virulence in mammalian hosts, in both H5N1 and H3N2 subtypes.</td> <td align="left">[<a href="#bib0335">67</a>]</td> </tr> <tr> <td align="left">West Nile Virus (WNV)</td> <td align="left">NS3</td> <td align="left">T249P</td> <td align="left">A single change (Thr→Pro) in the viral helicase increased virulence for American crows, and was observed in different bird outbreaks in Egypt, Romania, Russia, Israel and the United States.</td> <td align="left">[<a href="#bib0340">68</a>]</td> </tr> <tr> <td align="left" valign="middle" rowspan="3">Circulating vaccine-derived polioviruses (cVDPV)</td> <td align="center">VP1</td> <td align="left">F280Y</td> <td align="left" valign="middle" rowspan="3">Multiple parallel changes observed in vaccine derived polioviruses in 29 different epidemics in relation to a known, ancestral oral polio vaccine (OPV) strain sequence.</td> <td align="left" valign="middle" rowspan="3">[<a href="#bib0345">69</a>]<a href="#tblfn0005">a</a></td> </tr> <tr> <td align="center">VP2</td> <td align="left">F141Y</td> </tr> <tr> <td align="center">VP3</td> <td align="left">T80K</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">a</span><p id="npar0005">An additional non-coding substitution was also reported by the authors and is not included in this table.</p> </div> </div>  <p /> <p id="par0080">It is commonly hypothesised that natural selection is the key process involved in parallel molecular evolution. Negative selection eliminates many mutations from viral populations because strong functional constraints render most genetic changes disadvantageous. In contrast, positive selection favours the accumulation of mutations that are beneficial to virus replication and transmission. Determining whether a given site or genomic region has evolved under natural selection can be achieved by comparing the relative rates of synonymous (<i>d<sub>S</sub></i>) and non-synonymous (<i>d<sub>N</sub></i>) changes, quantified as a <i>d<sub>N</sub></i>/<i>d<sub>S</sub></i> ratio [<a href="#bib0135">27</a>]. Because positive selection can act on specific sites at particular points in time, it is important to explore variation in <i>d<sub>N</sub></i>/<i>d<sub>S</sub></i> amongst sites and amongst virus lineages. Several methods are available to undertake such calculations [<a href="#bib0140">28</a>, <a href="#bib0145">29</a>, <a href="#bib0150">30</a>]. If <i>d<sub>N</sub></i>/<i>d<sub>S</sub></i> &amp;gt; 1 for a specific codon, then multiple amino acid changes have likely occurred at that codon, and the codon is a candidate for the detection of parallel evolution (see below). However, <i>d<sub>N</sub></i>/<i>d<sub>S</sub></i> methods have limitations and may fail to detect instances of sites under selection [<a href="#bib0145">29</a>]. In particular, some studies report that <i>d<sub>N</sub></i>/<i>d<sub>S</sub></i> tests have failed to identify sites that appear to have undergone parallel adaptation [<a href="#bib0025">5</a>,<a href="#bib0155">31</a>].</p> </div> <div class="detectingparallelmolecularevolution" title="sec"> <div class="title" tagx="title" title="title">
Detecting parallel molecular evolution</div> <p id="par0085">The robust identification of parallel evolution and adaptation can be broken down into four steps: (<i>i</i>) detection of parallel changes (usually amino acid changes), (<i>ii</i>) association of those changes with a phenotype or environmental change of interest, (<i>iii</i>) characterisation of the evolutionary forces behind the repeated occurrence of such changes, and (<i>iv</i>) evaluation of the functional effects of mutations in the context of host–pathogen biology. A powerful method for detecting repeated, independent evolution is comparative phylogenetics, which infers the evolutionary history of a given trait (including reconstruction of ancestral states) using parsimony or other phylogenetic models [<a href="#bib0160">32</a>]. This approach is typically robust, so long as recombination or horizontal gene transfer (which can mimic parallel evolution) is absent.</p> <p id="par0090">Steps (<i>ii</i>) and (<i>iii</i>) are less straightforward. It is tempting to detect associations between a mutation and a phenotype by calculating the mutation frequencies in viruses that do or do not exhibit the relevant phenotype. This generates a contingency table of observed counts (<a href="#fig0005">Figure 1</a> ) that is evaluated using a Chi-squared or Fisher’s exact test. However, it has long been known in the evolutionary literature that this approach can lead to false positive conclusions, because the counts are not independent observations, but instead are correlated due to shared ancestry [<a href="#bib0165">33</a>,<a href="#bib0170">34</a>]. The problem can be resolved by considering the phylogenetic history of the trait in question (<a href="#fig0005">Figure 1</a>). Specifically, recurrent acquisition of a mutation in the context of independent changes in phenotype (<a href="#fig0005">Figure 1</a>b) indicates that the mutation is evolutionarily linked to the phenotype. However, if mutation and phenotypic change occur only once in the phylogeny (<a href="#fig0005">Figure 1</a>a) then their association could be due to chance. Bhattacharya <i>et al.</i> [<a href="#bib0035">7</a>] referred to the latter situation as the ‘founder effect’ and used a phylogenetically corrected statistical test to demonstrate that previously reported HLA-associated escape mutations in HIV [<a href="#bib0175">35</a>] were not statistically robust. Another phenomenon that may cause mutations to appear artefactually linked to a specific phenotype or environment is genetic hitchhiking, whereby an observed mutation does not affect the associated phenotype, but is genetically linked to a mutation that does. Further, the phylogenetic association between a mutation and a phenotype may be weak if the phenotype is polygenic in nature, or depends on an interaction between the viral genotype and the environment. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <p>Different evolutionary scenarios underlying the same apparent association between virus mutation and phenotype. <b>(a)</b> An apparently perfect association between the presence/absence of a mutation (blue/red dots) and a specific phenotype/environment (shading) can arise from a single evolutionary change (red cross). This association is typically represented and tested using a contingency table (bottom). However, each virus sequence state is <i>not</i> an independent observation due to this evolutionary history. <b>(b)</b> The same apparent association can arise through multiple independent mutations (red crosses) linked to change in phenotype/environment. This latter scenario is less likely to occur by chance and provides evidence for parallel adaptation. <b>(c)</b> Experimental evolution studies are often designed to observe changes (red crosses) in variants from a known ancestral strain (blue dot, centre) that is introduced to different environments (shading).</p>   </div> <p /> <p id="par0095">It is useful to place virus experimental evolution studies in the same phylogenetic context (<a href="#fig0005">Figure 1</a>c). Experimental evolution was, for example, used to identify adaptive mutations that increase viral fitness in vesicular stomatitis virus [<a href="#bib0130">26</a>]. Such studies often initiate independent virus populations from the same source inoculum. The replicate populations are passaged in different environments and later scanned for shared evolutionary changes. This experimental design is, thus, equivalent to a star-shaped phylogeny with a known ancestor (<a href="#fig0005">Figure 1</a>c). Since the replicate populations are not correlated by shared ancestry, differences amongst them are independent and can be tested using standard statistics.</p> </div> <div class="parallelevolutionbeyondthesequencelevel" title="sec"> <div class="title" tagx="title" title="title">
Parallel evolution beyond the sequence level</div> <p id="par0100">As explained above, phylogenetic analyses of viral sequences may be able to detect parallel evolution, but are insufficient to establish whether a given trait has arisen repeatedly through adaptation. Experimental evidence, from molecular biology or animal challenge studies, is ultimately required to determine the functional consequences of parallel mutations. However, it may be possible to further refine candidate sites for experimental confirmation by undertaking <i>in silico</i> analyses that combine structural and evolutionary information. This is especially true if virus proteins evolve the same structural change (e.g. a change in charge) via a variety of mutational paths. Mapping of virus mutations onto resolved protein structures can provide information about the functional effects of genetic changes, including thermodynamic stability, or interactions with other proteins or biologically important molecules [<a href="#bib0180">36<sup>•</sup></a>]. For example, a recent study of human influenza B virus genomes observed parallel amino acid changes occurring three times in the HA gene of the Yamagata lineage [<a href="#bib0190">38</a>]. Computational structural analyses showed that these were located in a major antigenic epitope [<a href="#bib0185">37</a>]. These approaches include structural alignment methods [<a href="#bib0190">38</a>] that can compare divergent virus proteins that exhibit little homology at the sequence level, but which share homology at the structural level. Other combined phylogenetic approaches can estimate the effect of specific mutations on protein stability, as demonstrated in a functional analysis of influenza A virus hemagglutinin gene variation [<a href="#bib0195">39</a>]. We envisage that structural analyses could form part of a phylogenetically informed workflow for investigating the significance of parallel evolutionary changes (<a href="#fig0010">Figure 2</a> ), and for selecting appropriate candidates for experimental validation. The results obtained would also provide insights into the evolvability, mutability and robustness of virus proteins [<a href="#bib0200">40</a>]. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <p>A framework for evaluating parallel evolution in virus sequences. <b>(a)</b> An alignment is estimated for a set of viral sequences. Polymorphic sites associated with a given phenotype or environment are identified. <b>(b)</b> A phylogeny is estimated, and the lineages in which the phenotype of interest is observed are identified (orange). <b>(c)</b> Ancestral states (here, S or V) are reconstructed for each of the polymorphic sites identified in step A. <b>(d)</b> Branches upon which a genetic change (red cross) is associated with the trait of interest are tested for positive selection. <b>(e)</b> Sites that exhibit evidence of molecular adaptation are mapped into a structure of the protein in question to determine possible functional effects.</p>   </div> <p /> <p id="par0105">Other functional constraints, beyond those imposed by protein structure, could also play a role in parallel molecular evolution during emergence or zoonosis. Non-coding viral genetic elements, such as untranslated regions (UTRs) [<a href="#bib0205">41</a>], internal ribosome entry sites (IRES) [<a href="#bib0210">42</a>,<a href="#bib0215">43</a>], or noncoding RNAs [<a href="#bib0220">44</a>,<a href="#bib0225">45</a>], can be functionally important for virus replication and infection. Flaviviruses provide many examples of the importance of RNA structures for viral replication [<a href="#bib0230">46</a>] and host adaptation [<a href="#bib0205">41</a>], some of which are proposed to have arisen through parallel evolution [<a href="#bib0235">47</a>]; different functional roles have been inferred for RNA structures in other viruses [<a href="#bib0240">48</a>,<a href="#bib0245">49</a>]. Evaluating the effects of natural selection on RNA genome regions encoding for secondary structures represents an additional challenge because (i) our understanding of how such structures evolve is poor, and (ii) it is difficult to define neutrally evolving sites in such regions [<a href="#bib0250">50</a>], which limits the use of <i>d<sub>N</sub></i>/<i>d<sub>S</sub></i> estimation methods, and our limited knowledge of the degree to which non-coding elements are structurally constrained. Various approaches have been developed to test for selection on non-coding regions [<a href="#bib0255">51</a>] and to predict the secondary structure of RNA molecules [<a href="#bib0260">52</a>,<a href="#bib0265">53</a>], but to date their use has been limited in evolutionary analyses of viruses. Further development of these methods could improve our understanding of the role that RNA structures play in virus evolution [<a href="#bib0270">54</a>].</p> </div> <div class="conclusionsandfutureperspectives" title="sec"> <div class="title" tagx="title" title="title">
Conclusions and future perspectives</div> <p id="par0110">Detecting parallel evolution in viruses, and inferring the mechanisms that underlie such changes, can benefit both basic and applied problems. If, in a rapidly evolving virus population, parallel mutations repeatedly arise in the context of the same change in phenotype or environment (i.e. <a href="#fig0005">Figure 1</a>b), then at least some aspects of virus evolution are predictable. Such repeating patterns, at the sequence or structural level, may help to forecast virus emergence and cross-species transmission events in the future. Further, placing these changes within a structural context could be used to triage newly discovered viruses, enabling surveillance and research to focus on those with the greatest predicted propensity for cross-species transmission or emergence. We note, however, that instances of parallel evolution identified by computational means must be interpreted with caution; confirmation of the fitness costs of evolutionary change ultimately depends on experimental validation, as shown for some recent outbreak scenarios [<a href="#bib0275">55</a>,<a href="#bib0280">56</a>]. A number of experimental techniques are now available to link virus genotypes to phenotypes, including pseudotyping [<a href="#bib0285">57<sup>•</sup></a>], minigenome systems [<a href="#bib0290">58</a>], viral reverse genetics [<a href="#bib0295">59</a>], and site-directed mutagenesis [<a href="#bib0300">60</a>]. These and related approaches are vital to determining the functional role of viral mutations and their effects on virus and host fitness.</p> <p id="par0115">Current challenges for inferring parallel molecular evolution are twofold. First, to better understand and predict parallel adaptation, and apply it to the specific scenario of virus emergence, we need a model that describes how the phenomenon depends on factors such as mutation rates, population sizes, selection coefficients and genome evolvability. Second, improvements in statistical methods and software are needed to make it easier to test parallel evolution hypotheses in the correct way. Advances have been made in testing specific phenomena associated with parallel evolution, such as coevolution [<a href="#bib0305">61</a>] and episodic selection [<a href="#bib0140">28</a>,<a href="#bib0150">30</a>]; however, an integrated inference framework that maximises the accuracy of site identification, and is robust to confounding processes such as recombination and genetic hitchhiking is still required. New high throughput experimental approaches, such as deep mutational scanning, have been applied to virus pathogens and will provide more comprehensive information on the mutability of virus proteins [<a href="#bib0310">62<sup>•</sup></a>,<a href="#bib0315">63</a>]. Results from similar empirical approaches, combined with the rapid growth of genome sequences for viruses from multiple hosts and ecosystems, will provide a broader evidence base for virus parallel evolution during virus emergence.</p> </div> <div class="referencesandrecommendedreading" title="sec"> <div class="title" tagx="title" title="title">
References and recommended reading</div> <p id="par0120">Papers of particular interest, published within the period of review, have been highlighted as: </p><ul> <li> <p id="par0125">• of special interest</p> </li> <li> <p id="par0130">•• of outstanding interest</p> </li> </ul> <p /> </div> </div> <div class="back" title="back"> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="bib0005" /><span class="label" tagx="label" title="label">1</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stern</span><span class="given-names" tagx="given-names" title="given-names">D.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The genetic causes of convergent evolution</span><span class="source" tagx="source" title="source">Nat Rev Genet</span><span class="volume" tagx="volume" title="volume">14</span><span class="year" tagx="year" title="year">2013</span><span class="fpage" tagx="fpage" title="fpage">751</span><span class="lpage" tagx="lpage" title="lpage">764</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24105273">24105273</a></span></span></li> <li tag="ref"><a name="bib0010" /><span class="label" tagx="label" title="label">2</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wood</span><span class="given-names" tagx="given-names" title="given-names">T.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burke</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rieseberg</span><span class="given-names" tagx="given-names" title="given-names">L.H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Parallel genotypic adaptation: when evolution repeats itself</span><span class="source" tagx="source" title="source">Genetica</span><span class="volume" tagx="volume" title="volume">123</span><span class="year" tagx="year" title="year">2005</span><span class="fpage" tagx="fpage" title="fpage">157</span><span class="lpage" tagx="lpage" title="lpage">170</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15881688">15881688</a></span></span></li> <li tag="ref"><a name="bib0015" /><span class="label" tagx="label" title="label">3</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doolittle</span><span class="given-names" tagx="given-names" title="given-names">R.F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Convergent evolution: the need to be explicit</span><span class="source" tagx="source" title="source">Trends Biochem Sci</span><span class="volume" tagx="volume" title="volume">19</span><span class="year" tagx="year" title="year">1994</span><span class="fpage" tagx="fpage" title="fpage">15</span><span class="lpage" tagx="lpage" title="lpage">18</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8140615">8140615</a></span></span></li> <li tag="ref"><a name="bib0020" /><span class="label" tagx="label" title="label">4</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zakon</span><span class="given-names" tagx="given-names" title="given-names">H.H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Convergent evolution on the molecular level</span><span class="source" tagx="source" title="source">Brain Behav Evol</span><span class="volume" tagx="volume" title="volume">59</span><span class="year" tagx="year" title="year">2002</span><span class="fpage" tagx="fpage" title="fpage">250</span><span class="lpage" tagx="lpage" title="lpage">261</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12207082">12207082</a></span></span></li> <li tag="ref"><a name="bib0025" /><span class="label" tagx="label" title="label">5</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Troupin</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Large-scale phylogenomic analysis reveals the complex evolutionary history of rabies virus in multiple carnivore hosts</span><span class="source" tagx="source" title="source">PLoS Pathog</span><span class="volume" tagx="volume" title="volume">12</span><span class="year" tagx="year" title="year">2016</span>e1006041</span></li> <li tag="ref"><a name="bib0030" /><span class="label" tagx="label" title="label">6</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kerr</span><span class="given-names" tagx="given-names" title="given-names">P.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Myxomatosis in Australia and Europe: a model for emerging infectious diseases</span><span class="source" tagx="source" title="source">Antivir Res</span><span class="volume" tagx="volume" title="volume">93</span><span class="year" tagx="year" title="year">2012</span><span class="fpage" tagx="fpage" title="fpage">387</span><span class="lpage" tagx="lpage" title="lpage">415</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22333483">22333483</a></span></span></li> <li tag="ref"><a name="bib0035" /><span class="label" tagx="label" title="label">7</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhattacharya</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Founder effects in the assessment of HIV polymorphisms and HLA allele associations</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">315</span><span class="year" tagx="year" title="year">2007</span><span class="fpage" tagx="fpage" title="fpage">1583</span><span class="lpage" tagx="lpage" title="lpage">1586</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17363674">17363674</a></span></span></li> <li tag="ref"><a name="bib0040" /><span class="label" tagx="label" title="label">8</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leslie</span><span class="given-names" tagx="given-names" title="given-names">A.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">HIV evolution: CTL escape mutation and reversion after transmission</span><span class="source" tagx="source" title="source">Nat Med</span><span class="volume" tagx="volume" title="volume">10</span><span class="year" tagx="year" title="year">2004</span><span class="fpage" tagx="fpage" title="fpage">282</span><span class="lpage" tagx="lpage" title="lpage">289</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14770175">14770175</a></span></span></li> <li tag="ref"><a name="bib0045" /><span class="label" tagx="label" title="label">9</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clutter</span><span class="given-names" tagx="given-names" title="given-names">D.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">HIV-1 drug resistance and resistance testing</span><span class="source" tagx="source" title="source">Infect Genet Evol</span><span class="volume" tagx="volume" title="volume">46</span><span class="year" tagx="year" title="year">2016</span><span class="fpage" tagx="fpage" title="fpage">292</span><span class="lpage" tagx="lpage" title="lpage">307</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27587334">27587334</a></span></span></li> <li tag="ref"><a name="bib0050" /><span class="label" tagx="label" title="label">10••</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woolhouse</span><span class="given-names" tagx="given-names" title="given-names">M.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Assessing the epidemic potential of RNA and DNA viruses</span><span class="source" tagx="source" title="source">Emerg Infect Dis</span><span class="volume" tagx="volume" title="volume">22</span><span class="year" tagx="year" title="year">2016</span><span class="fpage" tagx="fpage" title="fpage">2037</span><span class="lpage" tagx="lpage" title="lpage">2044</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27869592">27869592</a></span></span><span class="note" tagx="note" title="note"> <p>This work provides an overview of DNA and RNA viruses with emergence potential, paired with simulated data to assess factors that determine the ability to establish outbreaks.</p></span></li> <li tag="ref"><a name="bib0055" /><span class="label" tagx="label" title="label">11</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kerr</span><span class="given-names" tagx="given-names" title="given-names">P.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Evolutionary history and attenuation of myxoma virus on two continents</span><span class="source" tagx="source" title="source">PLoS Pathog</span><span class="volume" tagx="volume" title="volume">8</span><span class="year" tagx="year" title="year">2012</span>e1002950</span></li> <li tag="ref"><a name="bib0060" /><span class="label" tagx="label" title="label">12•</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wensing</span><span class="given-names" tagx="given-names" title="given-names">A.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">2017 update of the drug resistance mutations in HIV-1</span><span class="source" tagx="source" title="source">Top Antivir Med</span><span class="volume" tagx="volume" title="volume">24</span><span class="year" tagx="year" title="year">2017</span><span class="fpage" tagx="fpage" title="fpage">132</span><span class="lpage" tagx="lpage" title="lpage">133</span></span><span class="note" tagx="note" title="note"> <p>Summary of currently reported mutations associated with drug resistance in HIV-1, which includes many cases of parallel molecular evolution.</p></span></li> <li tag="ref"><a name="bib0065" /><span class="label" tagx="label" title="label">13</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jabara</span><span class="given-names" tagx="given-names" title="given-names">C.B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Accurate sampling and deep sequencing of the HIV-1 protease gene using a Primer ID</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span><span class="volume" tagx="volume" title="volume">108</span><span class="year" tagx="year" title="year">2011</span><span class="fpage" tagx="fpage" title="fpage">20166</span><span class="lpage" tagx="lpage" title="lpage">20171</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22135472">22135472</a></span></span></li> <li tag="ref"><a name="bib0070" /><span class="label" tagx="label" title="label">14</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palmer</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Multiple, linked human immunodeficiency virus type 1 drug resistance mutations in treatment-experienced patients are missed by standard genotype analysis</span><span class="source" tagx="source" title="source">J Clin Microbiol</span><span class="volume" tagx="volume" title="volume">43</span><span class="year" tagx="year" title="year">2005</span><span class="fpage" tagx="fpage" title="fpage">406</span><span class="lpage" tagx="lpage" title="lpage">413</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15635002">15635002</a></span></span></li> <li tag="ref"><a name="bib0075" /><span class="label" tagx="label" title="label">15</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iyidogan</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anderson</span><span class="given-names" tagx="given-names" title="given-names">K.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Current perspectives on HIV-1 antiretroviral drug resistance</span><span class="source" tagx="source" title="source">Viruses</span><span class="volume" tagx="volume" title="volume">6</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">4095</span><span class="lpage" tagx="lpage" title="lpage">4139</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25341668">25341668</a></span></span></li> <li tag="ref"><a name="bib0080" /><span class="label" tagx="label" title="label">16</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Longdon</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The evolution and genetics of virus host shifts</span><span class="source" tagx="source" title="source">PLoS Pathog</span><span class="volume" tagx="volume" title="volume">10</span><span class="year" tagx="year" title="year">2014</span>e1004395</span></li> <li tag="ref"><a name="bib0085" /><span class="label" tagx="label" title="label">17••</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Plowright</span><span class="given-names" tagx="given-names" title="given-names">R.K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Pathways to zoonotic spillover</span><span class="source" tagx="source" title="source">Nat Rev Microbiol</span><span class="volume" tagx="volume" title="volume">15</span><span class="year" tagx="year" title="year">2017</span><span class="fpage" tagx="fpage" title="fpage">502</span><span class="lpage" tagx="lpage" title="lpage">510</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28555073">28555073</a></span></span><span class="note" tagx="note" title="note"> <p>This work explores the scenarios and factors that can prompt spillovers from animals to humans and the barriers that a pathogens sorts during zoonotic events.</p></span></li> <li tag="ref"><a name="bib0090" /><span class="label" tagx="label" title="label">18</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wain</span><span class="given-names" tagx="given-names" title="given-names">L.V.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Adaptation of HIV-1 to its human host</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="volume" tagx="volume" title="volume">24</span><span class="year" tagx="year" title="year">2007</span><span class="fpage" tagx="fpage" title="fpage">1853</span><span class="lpage" tagx="lpage" title="lpage">1860</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17545188">17545188</a></span></span></li> <li tag="ref"><a name="bib0095" /><span class="label" tagx="label" title="label">19</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bibollet-Ruche</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Efficient SIVcpz replication in human lymphoid tissue requires viral matrix protein adaptation</span><span class="source" tagx="source" title="source">J Clin Invest</span><span class="volume" tagx="volume" title="volume">122</span><span class="year" tagx="year" title="year">2012</span><span class="fpage" tagx="fpage" title="fpage">1644</span><span class="lpage" tagx="lpage" title="lpage">1652</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22505456">22505456</a></span></span></li> <li tag="ref"><a name="bib0100" /><span class="label" tagx="label" title="label">20</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lebarbenchon</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stallknecht</span><span class="given-names" tagx="given-names" title="given-names">D.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Host shifts and molecular evolution of H7 avian influenza virus hemagglutinin</span><span class="source" tagx="source" title="source">Virol J</span><span class="volume" tagx="volume" title="volume">8</span><span class="year" tagx="year" title="year">2011</span><span class="fpage" tagx="fpage" title="fpage">328</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21711553">21711553</a></span></span></li> <li tag="ref"><a name="bib0105" /><span class="label" tagx="label" title="label">21</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hulse</span><span class="given-names" tagx="given-names" title="given-names">D.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular determinants within the surface proteins involved in the pathogenicity of H5N1 influenza viruses in chickens</span><span class="source" tagx="source" title="source">J Virol</span><span class="volume" tagx="volume" title="volume">78</span><span class="year" tagx="year" title="year">2004</span><span class="fpage" tagx="fpage" title="fpage">9954</span><span class="lpage" tagx="lpage" title="lpage">9964</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15331729">15331729</a></span></span></li> <li tag="ref"><a name="bib0110" /><span class="label" tagx="label" title="label">22</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sauter</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Tetherin-driven adaptation of Vpu and Nef function and the evolution of pandemic and nonpandemic HIV-1 strains</span><span class="source" tagx="source" title="source">Cell Host Microbe</span><span class="volume" tagx="volume" title="volume">6</span><span class="year" tagx="year" title="year">2009</span><span class="fpage" tagx="fpage" title="fpage">409</span><span class="lpage" tagx="lpage" title="lpage">421</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19917496">19917496</a></span></span></li> <li tag="ref"><a name="bib0115" /><span class="label" tagx="label" title="label">23</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kluge</span><span class="given-names" tagx="given-names" title="given-names">S.F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Nef proteins of epidemic HIV-1 group O strains antagonize human tetherin</span><span class="source" tagx="source" title="source">Cell Host Microbe</span><span class="volume" tagx="volume" title="volume">16</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">639</span><span class="lpage" tagx="lpage" title="lpage">650</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25525794">25525794</a></span></span></li> <li tag="ref"><a name="bib0120" /><span class="label" tagx="label" title="label">24</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Orr</span><span class="given-names" tagx="given-names" title="given-names">H.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The probability of parallel evolution</span><span class="source" tagx="source" title="source">Evolution</span><span class="volume" tagx="volume" title="volume">59</span><span class="year" tagx="year" title="year">2005</span><span class="fpage" tagx="fpage" title="fpage">216</span><span class="lpage" tagx="lpage" title="lpage">220</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15792240">15792240</a></span></span></li> <li tag="ref"><a name="bib0125" /><span class="label" tagx="label" title="label">25••</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bailey</span><span class="given-names" tagx="given-names" title="given-names">S.F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">What drives parallel evolution?: How population size and mutational variation contribute to repeated evolution</span><span class="source" tagx="source" title="source">Bioessays</span><span class="volume" tagx="volume" title="volume">39</span><span class="year" tagx="year" title="year">2017</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">9</span></span><span class="note" tagx="note" title="note"> <p>Metanalysis of experimental evolution datasets to analyse the effect of population sizes on the probability of occurrence of parallel molecular evolution.</p></span></li> <li tag="ref"><a name="bib0130" /><span class="label" tagx="label" title="label">26</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cuevas</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elena</span><span class="given-names" tagx="given-names" title="given-names">S.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moya</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular basis of adaptive convergence in experimental populations of RNA viruses</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">162</span><span class="year" tagx="year" title="year">2002</span><span class="fpage" tagx="fpage" title="fpage">533</span><span class="lpage" tagx="lpage" title="lpage">542</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12399369">12399369</a></span></span></li> <li tag="ref"><a name="bib0135" /><span class="label" tagx="label" title="label">27</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salemi</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vandamme</span><span class="given-names" tagx="given-names" title="given-names">A.-M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemey</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="edition" tagx="edition" title="edition">edn 2</span><span class="series" tagx="series" title="series">The Phylogenetic Handbook: A Practical Approach to Phylogenetic Analysis and Hypothesis Testing</span><span class="volume" tagx="volume" title="volume">vol. xxvi</span><span class="year" tagx="year" title="year">2009</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Cambridge University Press</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Cambridge, UK; New York</span><span class="comment">723 p</span></span></li> <li tag="ref"><a name="bib0140" /><span class="label" tagx="label" title="label">28</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">dos Reis</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Statistical properties of the branch-site test of positive selection</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="volume" tagx="volume" title="volume">28</span><span class="year" tagx="year" title="year">2011</span><span class="fpage" tagx="fpage" title="fpage">1217</span><span class="lpage" tagx="lpage" title="lpage">1228</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21087944">21087944</a></span></span></li> <li tag="ref"><a name="bib0145" /><span class="label" tagx="label" title="label">29</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anisimova</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bielawski</span><span class="given-names" tagx="given-names" title="given-names">J.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Accuracy and power of bayes prediction of amino acid sites under positive selection</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="volume" tagx="volume" title="volume">19</span><span class="year" tagx="year" title="year">2002</span><span class="fpage" tagx="fpage" title="fpage">950</span><span class="lpage" tagx="lpage" title="lpage">958</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12032251">12032251</a></span></span></li> <li tag="ref"><a name="bib0150" /><span class="label" tagx="label" title="label">30</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murrell</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Detecting individual sites subject to episodic diversifying selection</span><span class="source" tagx="source" title="source">PLoS Genet</span><span class="volume" tagx="volume" title="volume">8</span><span class="year" tagx="year" title="year">2012</span>e1002764</span></li> <li tag="ref"><a name="bib0155" /><span class="label" tagx="label" title="label">31</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crandall</span><span class="given-names" tagx="given-names" title="given-names">K.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Parallel evolution of drug resistance in HIV: failure of nonsynonymous/synonymous substitution rate ratio to detect selection</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="volume" tagx="volume" title="volume">16</span><span class="year" tagx="year" title="year">1999</span><span class="fpage" tagx="fpage" title="fpage">372</span><span class="lpage" tagx="lpage" title="lpage">382</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10331263">10331263</a></span></span></li> <li tag="ref"><a name="bib0160" /><span class="label" tagx="label" title="label">32</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Avise</span><span class="given-names" tagx="given-names" title="given-names">J.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nicholson</span><span class="given-names" tagx="given-names" title="given-names">T.H.</span></span></span><span class="series" tagx="series" title="series">Evolutionary Pathways in Nature: A Phylogenetic Approach</span><span class="volume" tagx="volume" title="volume">vol. x</span><span class="year" tagx="year" title="year">2006</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Cambridge University Press</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Cambridge</span><span class="comment">286 p</span></span></li> <li tag="ref"><a name="bib0165" /><span class="label" tagx="label" title="label">33</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harvey</span><span class="given-names" tagx="given-names" title="given-names">P.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pagel</span><span class="given-names" tagx="given-names" title="given-names">M.D.</span></span></span><span class="series" tagx="series" title="series">The Comparative Method in Evolutionary Biology. Oxford Series in Ecology and Evolution</span><span class="volume" tagx="volume" title="volume">vol. viii</span><span class="year" tagx="year" title="year">1991</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford University Press</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Oxford; New York</span><span class="comment">239 p</span></span></li> <li tag="ref"><a name="bib0170" /><span class="label" tagx="label" title="label">34</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pagel</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Detecting correlated evolution on phylogenies: a general method for the comparative analysis of discrete characters</span><span class="source" tagx="source" title="source">Proc R Soc Lond B</span><span class="volume" tagx="volume" title="volume">255</span><span class="year" tagx="year" title="year">1994</span><span class="fpage" tagx="fpage" title="fpage">37</span><span class="lpage" tagx="lpage" title="lpage">45</span></span></li> <li tag="ref"><a name="bib0175" /><span class="label" tagx="label" title="label">35</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moore</span><span class="given-names" tagx="given-names" title="given-names">C.B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Evidence of HIV-1 adaptation to HLA-restricted immune responses at a population level</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">296</span><span class="year" tagx="year" title="year">2002</span><span class="fpage" tagx="fpage" title="fpage">1439</span><span class="lpage" tagx="lpage" title="lpage">1443</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12029127">12029127</a></span></span></li> <li tag="ref"><a name="bib0180" /><span class="label" tagx="label" title="label">36•</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glusman</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Mapping genetic variations to three-dimensional protein structures to enhance variant interpretation: a proposed framework</span><span class="source" tagx="source" title="source">Genome Med</span><span class="volume" tagx="volume" title="volume">9</span><span class="year" tagx="year" title="year">2017</span><span class="fpage" tagx="fpage" title="fpage">113</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29254494">29254494</a></span></span><span class="note" tagx="note" title="note"> <p>Proposed integrated framework for mapping the effects of structural variants of proteins in the context of the human genome (part of the GVto3D Workshop).</p></span></li> <li tag="ref"><a name="bib0185" /><span class="label" tagx="label" title="label">37</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Langat</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genome-wide evolutionary dynamics of influenza B viruses on a global scale</span><span class="source" tagx="source" title="source">PLoS Pathog</span><span class="volume" tagx="volume" title="volume">13</span><span class="year" tagx="year" title="year">2017</span>e1006749</span></li> <li tag="ref"><a name="bib0190" /><span class="label" tagx="label" title="label">38</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Algorithms, applications, and challenges of protein structure alignment</span><span class="source" tagx="source" title="source">Adv Protein Chem Struct Biol</span><span class="volume" tagx="volume" title="volume">94</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">121</span><span class="lpage" tagx="lpage" title="lpage">175</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24629187">24629187</a></span></span></li> <li tag="ref"><a name="bib0195" /><span class="label" tagx="label" title="label">39</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bloom</span><span class="given-names" tagx="given-names" title="given-names">J.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glassman</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Inferring stabilizing mutations from protein phylogenies: application to influenza hemagglutinin</span><span class="source" tagx="source" title="source">PLoS Comput Biol</span><span class="volume" tagx="volume" title="volume">5</span><span class="year" tagx="year" title="year">2009</span>e1000349</span></li> <li tag="ref"><a name="bib0200" /><span class="label" tagx="label" title="label">40</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tokuriki</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tawfik</span><span class="given-names" tagx="given-names" title="given-names">D.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Stability effects of mutations and protein evolvability</span><span class="source" tagx="source" title="source">Curr Opin Struct Biol</span><span class="volume" tagx="volume" title="volume">19</span><span class="year" tagx="year" title="year">2009</span><span class="fpage" tagx="fpage" title="fpage">596</span><span class="lpage" tagx="lpage" title="lpage">604</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19765975">19765975</a></span></span></li> <li tag="ref"><a name="bib0205" /><span class="label" tagx="label" title="label">41</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Villordo</span><span class="given-names" tagx="given-names" title="given-names">S.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">RNA structure duplications and Flavivirus host adaptation</span><span class="source" tagx="source" title="source">Trends Microbiol</span><span class="volume" tagx="volume" title="volume">24</span><span class="year" tagx="year" title="year">2016</span><span class="fpage" tagx="fpage" title="fpage">270</span><span class="lpage" tagx="lpage" title="lpage">283</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26850219">26850219</a></span></span></li> <li tag="ref"><a name="bib0210" /><span class="label" tagx="label" title="label">42</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kieft</span><span class="given-names" tagx="given-names" title="given-names">J.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Viral IRES RNA structures and ribosome interactions</span><span class="source" tagx="source" title="source">Trends Biochem Sci</span><span class="volume" tagx="volume" title="volume">33</span><span class="year" tagx="year" title="year">2008</span><span class="fpage" tagx="fpage" title="fpage">274</span><span class="lpage" tagx="lpage" title="lpage">283</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18468443">18468443</a></span></span></li> <li tag="ref"><a name="bib0215" /><span class="label" tagx="label" title="label">43</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pfingsten</span><span class="given-names" tagx="given-names" title="given-names">J.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Costantino</span><span class="given-names" tagx="given-names" title="given-names">D.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kieft</span><span class="given-names" tagx="given-names" title="given-names">J.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structural basis for ribosome recruitment and manipulation by a viral IRES RNA</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">314</span><span class="year" tagx="year" title="year">2006</span><span class="fpage" tagx="fpage" title="fpage">1450</span><span class="lpage" tagx="lpage" title="lpage">1454</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17124290">17124290</a></span></span></li> <li tag="ref"><a name="bib0220" /><span class="label" tagx="label" title="label">44</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Akiyama</span><span class="given-names" tagx="given-names" title="given-names">B.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Zika virus produces noncoding RNAs using a multi-pseudoknot structure that confounds a cellular exonuclease</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">354</span><span class="year" tagx="year" title="year">2016</span><span class="fpage" tagx="fpage" title="fpage">1148</span><span class="lpage" tagx="lpage" title="lpage">1152</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27934765">27934765</a></span></span></li> <li tag="ref"><a name="bib0225" /><span class="label" tagx="label" title="label">45</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tycowski</span><span class="given-names" tagx="given-names" title="given-names">K.T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Viral noncoding RNAs: more surprises</span><span class="source" tagx="source" title="source">Genes Dev</span><span class="volume" tagx="volume" title="volume">29</span><span class="year" tagx="year" title="year">2015</span><span class="fpage" tagx="fpage" title="fpage">567</span><span class="lpage" tagx="lpage" title="lpage">584</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25792595">25792595</a></span></span></li> <li tag="ref"><a name="bib0230" /><span class="label" tagx="label" title="label">46</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alvarez</span><span class="given-names" tagx="given-names" title="given-names">D.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Role of RNA structures present at the 3′UTR of dengue virus on translation, RNA synthesis, and viral replication</span><span class="source" tagx="source" title="source">Virology</span><span class="volume" tagx="volume" title="volume">339</span><span class="year" tagx="year" title="year">2005</span><span class="fpage" tagx="fpage" title="fpage">200</span><span class="lpage" tagx="lpage" title="lpage">212</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16002117">16002117</a></span></span></li> <li tag="ref"><a name="bib0235" /><span class="label" tagx="label" title="label">47</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gritsun</span><span class="given-names" tagx="given-names" title="given-names">D.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular archaeology of Flaviviridae untranslated regions: duplicated RNA structures in the replication enhancer of flaviviruses and pestiviruses emerged via convergent evolution</span><span class="source" tagx="source" title="source">PLoS One</span><span class="volume" tagx="volume" title="volume">9</span><span class="year" tagx="year" title="year">2014</span>e92056</span></li> <li tag="ref"><a name="bib0240" /><span class="label" tagx="label" title="label">48</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gultyaev</span><span class="given-names" tagx="given-names" title="given-names">A.P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">RNA structural constraints in the evolution of the influenza A virus genome NP segment</span><span class="source" tagx="source" title="source">RNA Biol</span><span class="volume" tagx="volume" title="volume">11</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">942</span><span class="lpage" tagx="lpage" title="lpage">952</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25180940">25180940</a></span></span></li> <li tag="ref"><a name="bib0245" /><span class="label" tagx="label" title="label">49</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bailey</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Functional analysis of RNA structures present at the 3′ extremity of the murine norovirus genome: the variable polypyrimidine tract plays a role in viral virulence</span><span class="source" tagx="source" title="source">J Virol</span><span class="volume" tagx="volume" title="volume">84</span><span class="year" tagx="year" title="year">2010</span><span class="fpage" tagx="fpage" title="fpage">2859</span><span class="lpage" tagx="lpage" title="lpage">2870</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20053745">20053745</a></span></span></li> <li tag="ref"><a name="bib0250" /><span class="label" tagx="label" title="label">50</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">W.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nielsen</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Detecting selection in noncoding regions of nucleotide sequences</span><span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">167</span><span class="year" tagx="year" title="year">2004</span><span class="fpage" tagx="fpage" title="fpage">949</span><span class="lpage" tagx="lpage" title="lpage">958</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15238543">15238543</a></span></span></li> <li tag="ref"><a name="bib0255" /><span class="label" tagx="label" title="label">51</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhen</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andolfatto</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Methods to detect selection on noncoding DNA</span><span class="source" tagx="source" title="source">Methods Mol Biol</span><span class="volume" tagx="volume" title="volume">856</span><span class="year" tagx="year" title="year">2012</span><span class="fpage" tagx="fpage" title="fpage">141</span><span class="lpage" tagx="lpage" title="lpage">159</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22399458">22399458</a></span></span></li> <li tag="ref"><a name="bib0260" /><span class="label" tagx="label" title="label">52</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bernhart</span><span class="given-names" tagx="given-names" title="given-names">S.H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">RNAalifold: improved consensus structure prediction for RNA alignments</span><span class="source" tagx="source" title="source">BMC Bioinform</span><span class="volume" tagx="volume" title="volume">9</span><span class="year" tagx="year" title="year">2008</span><span class="fpage" tagx="fpage" title="fpage">474</span></span></li> <li tag="ref"><a name="bib0265" /><span class="label" tagx="label" title="label">53</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knudsen</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hein</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Pfold: RNA secondary structure prediction using stochastic context-free grammars</span><span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="volume" tagx="volume" title="volume">31</span><span class="year" tagx="year" title="year">2003</span><span class="fpage" tagx="fpage" title="fpage">3423</span><span class="lpage" tagx="lpage" title="lpage">3428</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12824339">12824339</a></span></span></li> <li tag="ref"><a name="bib0270" /><span class="label" tagx="label" title="label">54</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmonds</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">D.B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structural constraints on RNA virus evolution</span><span class="source" tagx="source" title="source">J Virol</span><span class="volume" tagx="volume" title="volume">73</span><span class="year" tagx="year" title="year">1999</span><span class="fpage" tagx="fpage" title="fpage">5787</span><span class="lpage" tagx="lpage" title="lpage">5794</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10364330">10364330</a></span></span></li> <li tag="ref"><a name="bib0275" /><span class="label" tagx="label" title="label">55</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diehl</span><span class="given-names" tagx="given-names" title="given-names">W.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Ebola virus glycoprotein with increased infectivity dominated the 2013-2016 epidemic</span><span class="source" tagx="source" title="source">Cell</span><span class="volume" tagx="volume" title="volume">167</span><span class="year" tagx="year" title="year">2016</span><span class="fpage" tagx="fpage" title="fpage">1088</span><span class="lpage" tagx="lpage" title="lpage">1098</span><span class="comment">e6</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27814506">27814506</a></span></span></li> <li tag="ref"><a name="bib0280" /><span class="label" tagx="label" title="label">56</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Urbanowicz</span><span class="given-names" tagx="given-names" title="given-names">R.A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Human adaptation of Ebola virus during the West African outbreak</span><span class="source" tagx="source" title="source">Cell</span><span class="volume" tagx="volume" title="volume">167</span><span class="year" tagx="year" title="year">2016</span><span class="fpage" tagx="fpage" title="fpage">1079</span><span class="lpage" tagx="lpage" title="lpage">1087</span><span class="comment">e5</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27814505">27814505</a></span></span></li> <li tag="ref"><a name="bib0285" /><span class="label" tagx="label" title="label">57•</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steffen</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmons</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Pseudotyping viral vectors with emerging virus envelope proteins</span><span class="source" tagx="source" title="source">Curr Gene Ther</span><span class="volume" tagx="volume" title="volume">16</span><span class="year" tagx="year" title="year">2016</span><span class="fpage" tagx="fpage" title="fpage">47</span><span class="lpage" tagx="lpage" title="lpage">55</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26785737">26785737</a></span></span><span class="note" tagx="note" title="note"> <p>Review of the use of pseudotyping methods to investigate emerging viruses such as SARS-CoV, EBOV, CHIKV, MERS-CoV, amongst others.</p></span></li> <li tag="ref"><a name="bib0290" /><span class="label" tagx="label" title="label">58</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoenen</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Minigenomes, transcription and replication competent virus-like particles and beyond: reverse genetics systems for filoviruses and other negative stranded hemorrhagic fever viruses</span><span class="source" tagx="source" title="source">Antivir Res</span><span class="volume" tagx="volume" title="volume">91</span><span class="year" tagx="year" title="year">2011</span><span class="fpage" tagx="fpage" title="fpage">195</span><span class="lpage" tagx="lpage" title="lpage">208</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21699921">21699921</a></span></span></li> <li tag="ref"><a name="bib0295" /><span class="label" tagx="label" title="label">59</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stobart</span><span class="given-names" tagx="given-names" title="given-names">C.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moore</span><span class="given-names" tagx="given-names" title="given-names">M.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">RNA virus reverse genetics and vaccine design</span><span class="source" tagx="source" title="source">Viruses</span><span class="volume" tagx="volume" title="volume">6</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">2531</span><span class="lpage" tagx="lpage" title="lpage">2550</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24967693">24967693</a></span></span></li> <li tag="ref"><a name="bib0300" /><span class="label" tagx="label" title="label">60</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carter</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Site-directed mutagenesis</span><span class="source" tagx="source" title="source">Biochem J</span><span class="volume" tagx="volume" title="volume">237</span><span class="year" tagx="year" title="year">1986</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">7</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3541892">3541892</a></span></span></li> <li tag="ref"><a name="bib0305" /><span class="label" tagx="label" title="label">61</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dutheil</span><span class="given-names" tagx="given-names" title="given-names">J.Y.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Detecting coevolving positions in a molecule: why and how to account for phylogeny</span><span class="source" tagx="source" title="source">Brief Bioinform</span><span class="volume" tagx="volume" title="volume">13</span><span class="year" tagx="year" title="year">2012</span><span class="fpage" tagx="fpage" title="fpage">228</span><span class="lpage" tagx="lpage" title="lpage">243</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21949241">21949241</a></span></span></li> <li tag="ref"><a name="bib0310" /><span class="label" tagx="label" title="label">62•</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doud</span><span class="given-names" tagx="given-names" title="given-names">M.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bloom</span><span class="given-names" tagx="given-names" title="given-names">J.D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Accurate measurement of the effects of all amino-acid mutations on influenza hemagglutinin</span><span class="source" tagx="source" title="source">Viruses</span><span class="volume" tagx="volume" title="volume">8</span><span class="year" tagx="year" title="year">2016</span></span><span class="note" tagx="note" title="note"> <p>Example of the use of deep mutational scanning to determine the mutability of the hemagglutinin in Influenza A viruses, a powerful tool for predicting sites that could be subject to parallel evolution.</p></span></li> <li tag="ref"><a name="bib0315" /><span class="label" tagx="label" title="label">63</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haddox</span><span class="given-names" tagx="given-names" title="given-names">H.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dingens</span><span class="given-names" tagx="given-names" title="given-names">A.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bloom</span><span class="given-names" tagx="given-names" title="given-names">J.D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Experimental estimation of the effects of all amino-acid mutations to HIV’s envelope protein on viral replication in cell culture</span><span class="source" tagx="source" title="source">PLoS Pathog</span><span class="volume" tagx="volume" title="volume">12</span><span class="year" tagx="year" title="year">2016</span>e1006114</span></li> <li tag="ref"><a name="bib0320" /><span class="label" tagx="label" title="label">64</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Remold</span><span class="given-names" tagx="given-names" title="given-names">S.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Turner</span><span class="given-names" tagx="given-names" title="given-names">P.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Evolutionary genomics of host adaptation in vesicular stomatitis virus</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="volume" tagx="volume" title="volume">25</span><span class="year" tagx="year" title="year">2008</span><span class="fpage" tagx="fpage" title="fpage">1138</span><span class="lpage" tagx="lpage" title="lpage">1147</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18353798">18353798</a></span></span></li> <li tag="ref"><a name="bib0325" /><span class="label" tagx="label" title="label">65</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Letko</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Adaptive evolution of MERS-CoV to species variation in DPP4</span><span class="source" tagx="source" title="source">Cell Rep</span><span class="volume" tagx="volume" title="volume">24</span><span class="year" tagx="year" title="year">2018</span><span class="fpage" tagx="fpage" title="fpage">1730</span><span class="lpage" tagx="lpage" title="lpage">1737</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30110630">30110630</a></span></span></li> <li tag="ref"><a name="bib0330" /><span class="label" tagx="label" title="label">66</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Connor</span><span class="given-names" tagx="given-names" title="given-names">R.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Receptor specificity in human, avian, and equine H2 and H3 influenza virus isolates</span><span class="source" tagx="source" title="source">Virology</span><span class="volume" tagx="volume" title="volume">205</span><span class="year" tagx="year" title="year">1994</span><span class="fpage" tagx="fpage" title="fpage">17</span><span class="lpage" tagx="lpage" title="lpage">23</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7975212">7975212</a></span></span></li> <li tag="ref"><a name="bib0335" /><span class="label" tagx="label" title="label">67</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steel</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Transmission of influenza virus in a mammalian host is increased by PB2 amino acids 627K or 627E/701N</span><span class="source" tagx="source" title="source">PLoS Pathog</span><span class="volume" tagx="volume" title="volume">5</span><span class="year" tagx="year" title="year">2009</span>e1000252</span></li> <li tag="ref"><a name="bib0340" /><span class="label" tagx="label" title="label">68</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brault</span><span class="given-names" tagx="given-names" title="given-names">A.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">C.Y.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Langevin</span><span class="given-names" tagx="given-names" title="given-names">S.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kinney</span><span class="given-names" tagx="given-names" title="given-names">R.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bowen</span><span class="given-names" tagx="given-names" title="given-names">R.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramey</span><span class="given-names" tagx="given-names" title="given-names">W.N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Panella</span><span class="given-names" tagx="given-names" title="given-names">N.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">E.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Powers</span><span class="given-names" tagx="given-names" title="given-names">A.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">B.R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A single positively selected West Nile viral mutation confers increased virogenesis in American crows</span><span class="source" tagx="source" title="source">Nat Genet</span><span class="volume" tagx="volume" title="volume">39</span><span class="year" tagx="year" title="year">2007</span><span class="fpage" tagx="fpage" title="fpage">1162</span><span class="lpage" tagx="lpage" title="lpage">1166</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17694056">17694056</a></span></span></li> <li tag="ref"><a name="bib0345" /><span class="label" tagx="label" title="label">69</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stern</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yeh</span><span class="given-names" tagx="given-names" title="given-names">M.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zinger</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wright</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ling</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nielsen</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Macadam</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andino</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The evolutionary pathway to virulence of an RNA virus</span><span class="source" tagx="source" title="source">Cell</span><span class="volume" tagx="volume" title="volume">169</span><span class="year" tagx="year" title="year">2017</span><span class="fpage" tagx="fpage" title="fpage">35</span><span class="lpage" tagx="lpage" title="lpage">46</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28340348">28340348</a></span></span></li> </ul> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>We thank Michael Golden for discussion and feedback. B.G. is supported by the </p><div class="funding-source" title="funding-source">
2017 Universities of Academic Excellence Scholarship Program of the Secretariat for Higher Education, Science, Technology and Innovation of the Republic of Ecuador</div> (ARSEQ-BEC-003163-2017) and by <div class="funding-source" title="funding-source">
Universidad San Francisco de Quito</div>. M.E.Z. is supported by an <div class="funding-source" title="funding-source">
EMBO Long Term Fellowship</div> (ALTF376-2017). O.G.P. is supported by the <div class="funding-source" title="funding-source">
European Union Seventh Framework Programme</div> (FP7/2007–2013)/ <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">10.13039/501100000781</span><span class="institution" title="institution">European Research Council</span></div> </div> (614725-PATHPHYLODYN).<p /> </div> </div>  </body></html>