<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/05/03/627158.atom" />
<meta name="HW.pisa" content="biorxiv;627158v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Extensive genetic diversity of bat-borne polyomaviruses reveals inter-family host-switching events" />
<meta name="DC.Identifier" content="10.1101/627158" />
<meta name="DC.Date" content="2019-05-03" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="Â© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Polyomaviruses (PyVs) are small, double-stranded DNA tumor viruses carried by diverse vertebrates. PyVs have previously been considered highly host restricted in mammalian hosts, with host-switching events thought rare or nonexistent. Prior investigations have revealed short-range host-switching events of PyVs in two different African bat species within the horseshoe bat genus Rhinolophus . Herein, we have conducted a systematic investigation of PyVs in 1,083 archived bat samples collected from five provinces across China, and identified 192 PyVs from 186 bats from 15 host species within 6 families (Rhinolophidae, Vespertilionidae, Hipposideridae, Emballonuridae, Miniopteridae and Pteropodidae) representing 28 newly-described PyVs, indicative of extensive genetic diversity of bat PyVs. Surprisingly, two PyVs were identified in multiple bat species from different families, and another PyV clustered phylogenetically with PyVs carried by bats from a different host family, indicative of three inter-family PyV host-switching events. The time to most recent common ancestor (tMRCA) of the three events was estimated at 0.02-11.6 million years ago (MYA), which is inconsistent with the estimated tMRCA of their respective bat hosts (36.3-66.7 MYA), and is most parsimoniously explained by host-switching events. PyVs identified from geographically separated Chinese horseshoe bat species in the present study showed close genetic identities, and clustered with each other and with PyVs from African horseshoe bats, allowing assessment of the effects of positive selection in VP1 within the horseshoe bat family Rhinolophidae. Correlation analysis indicated that co-evolution with their hosts contributed much more to evolutionary divergence of PyV than geographic distance. In conclusion, our findings provide the first evidence of inter-family host-switching events of PyV in mammals and challenge the prevailing evolutionary paradigm for strict host restriction of mammalian PyVs.

Author summary Since the discovery of murine polyomavirus in the 1950s, polyomaviruses (PyVs) have been considered both genetically stable and highly host-restricted in their mammalian hosts. In this study, we have identified multiple cases of host-switching events of PyVs by large scale surveillance in diverse bat species collected in China. These host-switching events occurred between bat families living in the same colony, indicating that a large population with frequent contacts between different bat species may represent an ecological niche facilitating PyV host-switching. The cases studied involved members of bats from several families, including horseshoe bats, which were previously found to harbor a number of highly virulent viruses to both humans and domestic animals. Our findings have provided evidence that even highly host-specific DNA viruses can transmit between bats of different species and indicate an increased propensity for spillover events involving horseshoe bats. We propose an evolutionary scheme for bat-borne PyVs in which intra-host divergence and host-switching has generated the diverse PyVs in present day bats. This scheme provides a useful model to study the evolution of PyVs in other hosts and, potentially, the modeling of bat zoonoses and the transmission of other DNA viruses in other mammals, including humans." />
<meta name="DC.Contributor" content="Zhizhou Tan" />
<meta name="DC.Contributor" content="Gabriel Gonzalez" />
<meta name="DC.Contributor" content="Jinliang Sheng" />
<meta name="DC.Contributor" content="Jianmin Wu" />
<meta name="DC.Contributor" content="Fuqiang Zhang" />
<meta name="DC.Contributor" content="Lin Xu" />
<meta name="DC.Contributor" content="Peisheng Zhang" />
<meta name="DC.Contributor" content="Aiwei Zhu" />
<meta name="DC.Contributor" content="Yonggang Qu" />
<meta name="DC.Contributor" content="Changchun Tu" />
<meta name="DC.Contributor" content="Michael J. Carr" />
<meta name="DC.Contributor" content="Biao He" />
<meta name="article:published_time" content="2019-05-03" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Extensive genetic diversity of bat-borne polyomaviruses reveals inter-family host-switching events" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Polyomaviruses (PyVs) are small, double-stranded DNA tumor viruses carried by diverse vertebrates. PyVs have previously been considered highly host restricted in mammalian hosts, with host-switching events thought rare or nonexistent. Prior investigations have revealed short-range host-switching events of PyVs in two different African bat species within the horseshoe bat genus &lt;i&gt;Rhinolophus&lt;/i&gt;. Herein, we have conducted a systematic investigation of PyVs in 1,083 archived bat samples collected from five provinces across China, and identified 192 PyVs from 186 bats from 15 host species within 6 families (Rhinolophidae, Vespertilionidae, Hipposideridae, Emballonuridae, Miniopteridae and Pteropodidae) representing 28 newly-described PyVs, indicative of extensive genetic diversity of bat PyVs. Surprisingly, two PyVs were identified in multiple bat species from different families, and another PyV clustered phylogenetically with PyVs carried by bats from a different host family, indicative of three inter-family PyV host-switching events. The time to most recent common ancestor (tMRCA) of the three events was estimated at 0.02-11.6 million years ago (MYA), which is inconsistent with the estimated tMRCA of their respective bat hosts (36.3-66.7 MYA), and is most parsimoniously explained by host-switching events. PyVs identified from geographically separated Chinese horseshoe bat species in the present study showed close genetic identities, and clustered with each other and with PyVs from African horseshoe bats, allowing assessment of the effects of positive selection in VP1 within the horseshoe bat family Rhinolophidae. Correlation analysis indicated that co-evolution with their hosts contributed much more to evolutionary divergence of PyV than geographic distance. In conclusion, our findings provide the first evidence of inter-family host-switching events of PyV in mammals and challenge the prevailing evolutionary paradigm for strict host restriction of mammalian PyVs.&lt;/p&gt;&lt;h3&gt;Author summary&lt;/h3&gt;
&lt;p&gt;Since the discovery of murine polyomavirus in the 1950s, polyomaviruses (PyVs) have been considered both genetically stable and highly host-restricted in their mammalian hosts. In this study, we have identified multiple cases of host-switching events of PyVs by large scale surveillance in diverse bat species collected in China. These host-switching events occurred between bat families living in the same colony, indicating that a large population with frequent contacts between different bat species may represent an ecological niche facilitating PyV host-switching. The cases studied involved members of bats from several families, including horseshoe bats, which were previously found to harbor a number of highly virulent viruses to both humans and domestic animals. Our findings have provided evidence that even highly host-specific DNA viruses can transmit between bats of different species and indicate an increased propensity for spillover events involving horseshoe bats. We propose an evolutionary scheme for bat-borne PyVs in which intra-host divergence and host-switching has generated the diverse PyVs in present day bats. This scheme provides a useful model to study the evolution of PyVs in other hosts and, potentially, the modeling of bat zoonoses and the transmission of other DNA viruses in other mammals, including humans.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;627158v1" />
<meta name="citation_id" content="627158v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/627158v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/627158v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/627158v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/05/03/627158.full.pdf" />
<meta name="citation_doi" content="10.1101/627158" />
<meta name="citation_num_pages" content="61" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="627158" />
<meta name="citation_author" content="Zhizhou Tan" />
<meta name="citation_author_institution" content="Key Laboratory of Jilin Province for Zoonosis Prevention and Control, Institute of Military Veterinary Medicine, Academy of Military Medical Sciences, Changchun" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2844-9673" />
<meta name="citation_author" content="Gabriel Gonzalez" />
<meta name="citation_author_institution" content="National Advanced Computing Collaboratory, National Center for High Technology" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2180-2120" />
<meta name="citation_author" content="Jinliang Sheng" />
<meta name="citation_author_institution" content="College of Animal Science and Technology, Shihezi University, Shihezi, Xinjiang Uyghur Autonomous Region" />
<meta name="citation_author" content="Jianmin Wu" />
<meta name="citation_author_institution" content="Guangxi Key Laboratory of Veterinary Biotechnology, Guangxi Veterinary Research Institute, Nanning" />
<meta name="citation_author" content="Fuqiang Zhang" />
<meta name="citation_author_institution" content="Center for Disease Control and Prevention of Southern Theater Command" />
<meta name="citation_author" content="Lin Xu" />
<meta name="citation_author_institution" content="Key Laboratory of Jilin Province for Zoonosis Prevention and Control, Institute of Military Veterinary Medicine, Academy of Military Medical Sciences, Changchun" />
<meta name="citation_author" content="Peisheng Zhang" />
<meta name="citation_author_institution" content="Key Laboratory of Jilin Province for Zoonosis Prevention and Control, Institute of Military Veterinary Medicine, Academy of Military Medical Sciences, Changchun" />
<meta name="citation_author_institution" content="College of Animal Science and Technology, Shihezi University, Shihezi, Xinjiang Uyghur Autonomous Region" />
<meta name="citation_author" content="Aiwei Zhu" />
<meta name="citation_author_institution" content="Key Laboratory of Jilin Province for Zoonosis Prevention and Control, Institute of Military Veterinary Medicine, Academy of Military Medical Sciences, Changchun" />
<meta name="citation_author" content="Yonggang Qu" />
<meta name="citation_author_institution" content="College of Animal Science and Technology, Shihezi University, Shihezi, Xinjiang Uyghur Autonomous Region" />
<meta name="citation_author" content="Changchun Tu" />
<meta name="citation_author_institution" content="Key Laboratory of Jilin Province for Zoonosis Prevention and Control, Institute of Military Veterinary Medicine, Academy of Military Medical Sciences, Changchun" />
<meta name="citation_author_institution" content="Jiangsu Co-innovation Center for Prevention and Control of Important Animal Infectious Diseases and Zoonosis, Yangzhou" />
<meta name="citation_author_email" content="heb-001001@163.com" />
<meta name="citation_author_email" content="michael.carr@ucd.ie" />
<meta name="citation_author_email" content="changchun_tu@hotmail.com" />
<meta name="citation_author" content="Michael J. Carr" />
<meta name="citation_author_institution" content="National Virus Reference Laboratory, School of Medicine, University College Dublin" />
<meta name="citation_author_email" content="heb-001001@163.com" />
<meta name="citation_author_email" content="michael.carr@ucd.ie" />
<meta name="citation_author_email" content="changchun_tu@hotmail.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0011-6209" />
<meta name="citation_author" content="Biao He" />
<meta name="citation_author_institution" content="Key Laboratory of Jilin Province for Zoonosis Prevention and Control, Institute of Military Veterinary Medicine, Academy of Military Medical Sciences, Changchun" />
<meta name="citation_author_institution" content="Jiangsu Co-innovation Center for Prevention and Control of Important Animal Infectious Diseases and Zoonosis, Yangzhou" />
<meta name="citation_author_email" content="heb-001001@163.com" />
<meta name="citation_author_email" content="michael.carr@ucd.ie" />
<meta name="citation_author_email" content="changchun_tu@hotmail.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7587-152X" />
<meta name="citation_reference" content="Wilson DE, Reeder DM. Mammal species of the world: a taxonomic and geographic reference, 3rd ed, Order Chiroptera. Baltimore, MD: Johns Hopkins University Press; 2005." />
<meta name="citation_reference" content="Streicker DG, Turmelle AS, Vonhof MJ, Kuzmin IV, McCracken GF, Rupprecht CE. Host phylogeny constrains cross-species emergence and establishment of rabies virus in bats. Science. 2010;329(5992):676. DOI:10.1126/science.1188836." />
<meta name="citation_reference" content="Towner JS, Pourrut X, AlbariÃ±o CG, Nkogue CN, Bird BH, Grard G, et al. Marburg virus infection detected in a common African bat. PLoS One. 2007;2(8):e764. DOI:10.1371/journal.pone.0000764." />
<meta name="citation_reference" content="Leroy EM, Kumulungui B, Pourrut X, Rouquet P, Hassanin A, Yaba P, et al. Fruit bats as reservoirs of Ebola virus. Nature. 2005;438(7068):575. DOI:10.1038/438575a." />
<meta name="citation_reference" content="Chua KB, Lek Koh C, Hooi PS, Wee KF, Khong JH, Chua BH, et al. Isolation of Nipah virus from Malaysian Island flying-foxes. Microb Infect. 2002;4(2):145â51. DOI:10.1016/S1286-4579(01)01522-2." />
<meta name="citation_reference" content="Li W, Shi Z, Yu M, Ren W, Smith C, Epstein JH, et al. Bats are natural reservoirs of SARS-like coronaviruses. Science. 2005;310(5748):676â9. DOI:10.1126/science.1118391." />
<meta name="citation_reference" content="Lau SK, Woo PC, Li KS, Huang Y, Tsoi H-W, Wong BH, et al. Severe acute respiratory syndrome coronavirus-like virus in Chinese horseshoe bats. Proc Natl Acad Sci U S A. 2005;102(39):14040â5. DOI:10.1073/pnas.0506735102." />
<meta name="citation_reference" content="He B, Zhang Y, Xu L, Yang W, Yang F, Feng Y, et al. Identification of diverse alphacoronaviruses and genomic characterization of a novel severe acute respiratory syndrome-like coronavirus from bats in China. J Virol. 2014;88(12):7070â82. DOI:10.1128/JVI.00631-14." />
<meta name="citation_reference" content="Zhou P, Fan H, Lan T, Yang X-L, Shi W-F, Zhang W, et al. Fatal swine acute diarrhoea syndrome caused by an HKU2-related coronavirus of bat origin. Nature. 2018;556:255â8. DOI:10.1038/s41586-018-0010-9." />
<meta name="citation_reference" content="Wolfe ND, Dunavan CP, Diamond J. Origins of major human infectious diseases. Nature. 2007;447(7142):279â83. DOI:10.1038/nature05775." />
<meta name="citation_reference" content="DeCaprio JA, Imperiale MJ, Major EO. Polyomaviridae, Fields virology. 6th ed. Philadelphia: WoltersKluwer; 2013." />
<meta name="citation_reference" content="Moens U, Krumbholz A, Ehlers B, Zell R, Johne R, Calvignac-Spencer S, et al. Biology, evolution, and medical importance of polyomaviruses: An update. Infect Genet Evol. 2017;54:18â38. DOI:10.1016/j.meegid.2017.06.011." />
<meta name="citation_reference" content="Feng H, Shuda M, Chang Y, Moore PS. Clonal integration of a polyomavirus in human Merkel cell carcinoma. Science. 2008;319(5866):1096. DOI:10.1126/science.1152586." />
<meta name="citation_reference" content="Dalianis T, Hirsch HH. Human polyomaviruses in disease and cancer. Virology. 2013;437(2):63â72. DOI:10.1016/j.virol.2012.12.015." />
<meta name="citation_reference" content="Dela Cruz FN, Giannitti F, Li L, Woods LW, Del Valle L, Delwart E, et al. Novel polyomavirus associated with brain tumors in free-ranging raccoons, western United States. Emerg Infect Dis. 2013;19(1):77â84. DOI:10.3201/eid1901.121078." />
<meta name="citation_reference" content="Moens U, Calvignac-Spencer S, Lauber C, Ramqvist T, Feltkamp MCW, Daugherty MD, et al. ICTV virus taxonomy profile: Polyomaviridae. J Gen Virol. 2017;98(6):1159â60. DOI:10.1099/jgv.0.000839." />
<meta name="citation_reference" content="Buck CB, Van Doorslaer K, Peretti A, Geoghegan EM, Tisza MJ, An P, et al. The ancient evolutionary history of polyomaviruses. PLoS Pathog. 2016;12(4):e1005574. DOI:10.1371/journal.ppat.1005574." />
<meta name="citation_reference" content="Perez-Losada M, Christensen RG, McClellan DA, Adams BJ, Viscidi RP, Demma JC, et al. Comparing phylogenetic codivergence between polyomaviruses and their hosts. J Virol. 2006;80(12):5663â9. DOI:10.1128/jvi.00056-06." />
<meta name="citation_reference" content="Misra V, Dumonceaux T, Dubois J, Willis C, Nadin-Davis S, Severini A, et al. Detection of polyoma and corona viruses in bats of Canada. J Gen Virol. 2009;90(8):2015â22. DOI:10.1099/vir.0.010694-0." />
<meta name="citation_reference" content="Fagrouch Z, Sarwari R, Lavergne A, Delaval M, De Thoisy B, Lacoste V, et al. Novel polyomaviruses in South American bats and their relationship to other members of the family Polyomaviridae. J Gen Virol. 2012;93(12):2652â7. DOI:10.1099/vir.0.044149-0." />
<meta name="citation_reference" content="Tao Y, Shi M, Conrardy C, Kuzmin IV, Recuenco S, Agwanda B, et al. Discovery of diverse polyomaviruses in bats and the evolutionary history of the Polyomaviridae. J Gen Virol. 2013;94(4):738â48. DOI:10.1099/vir.0.047928-0." />
<meta name="citation_reference" content="Lima FES, Cibulski SP, Witt AA, Franco AC, Roehe PM. Genomic characterization of two novel polyomaviruses in Brazilian insectivorous bats. Arch Virol. 2015;160(7):1831â6. DOI:10.1007/s00705-015-2447-6." />
<meta name="citation_reference" content="Kobayashi S, Sasaki M, Nakao R, Setiyono A, Handharyani E, Orba Y, et al. Detection of novel polyomaviruses in fruit bats in Indonesia. Arch Virol. 2015;160(4):1075â82. DOI:10.1007/s00705-015-2349-7." />
<meta name="citation_reference" content="Wang J, Moore NE, Murray ZL, McInnes K, White DJ, Tompkins DM, et al. Discovery of novel virus sequences in an isolated and threatened bat species, the New Zealand lesser short-tailed bat (Mystacina tuberculata). J Gen Virol. 2015;96(8):2442â52. DOI:10.1099/vir.0.000158." />
<meta name="citation_reference" content="Carr M, Gonzalez G, Sasaki M, Ito K, Ishii A, Hangâombe BM, et al. Discovery of African bat polyomaviruses and infrequent recombination in the large T antigen in the Polyomaviridae. J Gen Virol. 2017;98(4):726â38. DOI:10.1099/jgv.0.000737." />
<meta name="citation_reference" content="Carr M, Gonzalez G, Sasaki M, Dool SE, Ito K, Ishii A, et al. Identification of the same polyomavirus species in different African horseshoe bat species is indicative of short-range host-switching events. J Gen Virol. 2017;98(11):2771â85. DOI:10.1099/jgv.0.000935." />
<meta name="citation_reference" content="Johne R, Enderlein D, Nieper H, MÃ¼ller H. Novel polyomavirus detected in the feces of a chimpanzee by nested broad-spectrum PCR. J Virol. 2005;79(6):3883â7. DOI:10.1128/JVI.79.6.3883-3887.2005." />
<meta name="citation_reference" content="Wu Z, Yang L, Ren X, He G, Zhang J, Yang J, et al. Deciphering the bat virome catalog to better understand tha ecological diversity of bat viruses and the bat origin of emerging infectious diseases. The ISME Journal. 2015;2015:1â12. DOI:10.1038/ismej.2015.138." />
<meta name="citation_reference" content="Cantalupo PG, Buck CB, Pipas JM. Complete genome sequence of a polyomavirus recovered from a pomona leaf-nosed bat (Hipposideros pomona) metagenome data set. Genome Announc. 2017;5(3):e01053â16. DOI:10.1128/genomeA.01053-16." />
<meta name="citation_reference" content="Mishra N, Fagbo SF, Alagaili AN, Nitido A, Williams SH, Ng J, et al. A viral metagenomic survey identifies known and novel mammalian viruses in bats from Saudi Arabia. PLoS One. 2019;14(4):e0214227. DOI:10.1371/journal.pone.0214227." />
<meta name="citation_reference" content="Hsieh H-M, Chiang H-L, Tsai L-C, Lai S-Y, Huang N-E, Linacre A, et al. Cytochrome bgene for species identification of the conservation animals. Forensic Sci Int. 2001;122(1):7â18." />
<meta name="citation_reference" content="He B, Huang X, Zhang F, Tan W, Matthijnssens J, Qin S, et al. Group A rotaviruses in Chinese bats: Genetic composition, serology, and evidence for bat-to-human transmission and reassortment. J Virol. 2017;91(12). DOI:10.1128/jvi.02493-16." />
<meta name="citation_reference" content="Xu L, Wu J, Li Q, Wei Y, Tan Z, Cai J, et al. Seroprevalence, cross antigenicity and circulation sphere of bat-borne hantaviruses revealed by serological and antigenic analyses. PLoS Pathog. 2019;15(1):e1007545. DOI:10.1371/journal.ppat.1007545." />
<meta name="citation_reference" content="He B, Li Z, Yang F, Zheng J, Feng Y, Guo H, et al. Virome profiling of bats from Myanmar by metagenomic analysis of tissue samples reveals more novel mammalian viruses. PLoS One. 2013;8(4):e61950. DOI:10.1371/journal.pone.0061950." />
<meta name="citation_reference" content="Ehlers B, Richter D, Matuschka FR, Ulrich RG. Genome sequences of a rat polyomavirus related to murine polyomavirus, Rattus norvegicus polyomavirus 1. Genome Announc. 2015;3(5). DOI:10.1128/genomeA.00997-15." />
<meta name="citation_reference" content="Calvignac-Spencer S, Feltkamp MC, Daugherty MD, Moens U, Ramqvist T, Johne R, et al. A taxonomy update for the family Polyomaviridae. Arch Virol. 2016;161(6):1739â50." />
<meta name="citation_reference" content="Sun K, Feng J, Zhang Z, Xu L, Liu Y. Cryptic diversity in Chinese rhinolophids and hipposiderids (Chiroptera: Rhinolophidae and Hipposideridae). Mammalia. 2009;73(2):135â41. DOI:10.1515/MAMM.2009.022." />
<meta name="citation_reference" content="Tan Z, Yu H, Xu L, Zhao Z, Zhang P, Qu Y, et al. Virome profiling of rodents in Xinjiang Uygur Autonomous Region, China: Isolation and characterization of a new strain of Wenzhou virus. Virology. 2019;529:122â34. DOI:10.1016/j.virol.2019.01.010." />
<meta name="citation_reference" content="Dool SE, Puechmaille SJ, Foley NM, Allegrini B, Bastian A, Mutumi GL, et al. Nuclear introns outperform mitochondrial DNA in inter-specific phylogenetic reconstruction: lessons from horseshoe bats (Rhinolophidae: Chiroptera). Mol Phylogenet Evol. 2016;97:196â212. DOI:10.1016/j.ympev.2016.01.003." />
<meta name="citation_reference" content="Agnarsson I, Zambrana-Torrelio CM, Flores-Saldana NP, May-Collado LJ. A time-calibrated species-level phylogeny of bats (Chiroptera, Mammalia). PLoS currents. 2011;3:Rrn1212. DOI:10.1371/currents.RRN1212." />
<meta name="citation_reference" content="Meredith RW, Janecka JE, Gatesy J, Ryder OA, Fisher CA, Teeling EC, et al. Impacts of the Cretaceous Terrestrial Revolution and KPg extinction on mammal diversification. Science. 2011;334(6055):521. DOI:10.1126/science.1211028." />
<meta name="citation_reference" content="Stehle T, Yan Y, Benjamin TL, Harrison SC. Structure of murine polyomavirus complexed with an oligosaccharide receptor fragment. Nature. 1994;369(6476):160â3. DOI:10.1038/369160a0." />
<meta name="citation_reference" content="Neu U, Woellner K, Gauglitz G, Stehle T. Structural basis of GM1 ganglioside recognition by simian virus 40. Proc Natl Acad Sci U S A. 2008;105(13):5219â24. DOI:10.1073/pnas.0710301105." />
<meta name="citation_reference" content="Neu U, Maginnis MS, Palma AS, Stroh LJ, Nelson CD, Feizi T, et al. Structure-function analysis of the human JC polyomavirus establishes the LSTc pentasaccharide as a functional receptor motif. Cell Host Microbe. 2010;8(4):309â19. DOI:10.1016/j.chom.2010.09.004." />
<meta name="citation_reference" content="Neu U, Hengel H, Blaum BS, Schowalter RM, Macejak D, Gilbert M, et al. Structures of Merkel cell polyomavirus VP1 complexes define a sialic acid binding site required for infection. PLoS Pathog. 2012;8(7):e1002738. DOI:10.1371/journal.ppat.1002738." />
<meta name="citation_reference" content="Hurdiss DL, Frank M, Snowden JS, Macdonald A, Ranson NA. The structure of an infectious human polyomavirus and its interactions with cellular receptors. Structure. 2018;26(6):839-47.e3. DOI:10.1016/j.str.2018.03.019." />
<meta name="citation_reference" content="Warden CD, Lacey SF. Updated phylogenetic analysis of polyomavirus-host co-evolution. J Bioinfo Res. 2012;1:46â9." />
<meta name="citation_reference" content="Krumbholz A, Bininda-Emonds OR, Wutzler P, Zell R. Evolution of four BK virus subtypes. Infect Genet Evol. 2008;8(5):632â43. DOI:10.1016/j.meegid.2008.05.006." />
<meta name="citation_reference" content="Johne R, Muller H. Polyomaviruses of birds: etiologic agents of inflammatory diseases in a tumor virus family. J Virol. 2007;81(21):11554â9. DOI:10.1128/jvi.01178-07." />
<meta name="citation_reference" content="Nie FY, Lin XD, Hao ZY, Chen XN, Wang ZX, Wang MR, et al. Extensive diversity and evolution of hepadnaviruses in bats in China. Virology. 2018;514:88â97. DOI:10.1016/j.virol.2017.11.005." />
<meta name="citation_reference" content="Eckerle I, Ehlen L, Kallies R, Wollny R, Corman VM, Cottontail VM, et al. Bat airway epithelial cells: a novel tool for the study of zoonotic viruses. PLoS One. 2014;9(1):e84679. DOI:10.1371/journal.pone.0084679." />
<meta name="citation_reference" content="Banerjee A, Rapin N, Miller M, Griebel P, Zhou Y, Munster V, et al. Generation and characterization of Eptesicus fuscus (Big brown bat) kidney cell lines immortalized using the Myotis polyomavirus large T-antigen. J Virol Methods. 2016;237:166â73. DOI:10.1016/j.jviromet.2016.09.008." />
<meta name="citation_reference" content="GarcÃ­a-PÃ©rez R, IbÃ¡Ã±ez C, GodÃ­nez JM, ArÃ©chiga N, Garin I, PÃ©rez-SuÃ¡rez G, et al. Novel papillomaviruses in free-ranging Iberian bats: no virusâhost co-evolution, no strict host specificity, and hints for recombination. Genome Biol Evol. 2014;6(1):94â104. DOI:10.1093/gbe/evt211" />
<meta name="citation_reference" content="Corman VM, Ithete NL, Richards LR, Schoeman MC, Preiser W, Drosten C, et al. Rooting the phylogenetic tree of middle East respiratory syndrome coronavirus by characterization of a conspecific virus from an African bat. J Virol. 2014;88(19):11297â303. DOI:10.1128/jvi.01498-14." />
<meta name="citation_reference" content="Peretti A, FitzGerald PC, Bliskovsky V, Buck CB, Pastrana DV. Hamburger polyomaviruses. J Gen Virol. 2015;96(Pt 4):833â9. DOI:10.1099/vir.0.000033." />
<meta name="citation_reference" content="Cook L. Polyomaviruses, Diagnostic microbiology of the immunocompromised host. 2 ed. Washington, DC: ASM Press; 2016." />
<meta name="citation_reference" content="Beckervordersandforth J, Pujari S, Rennspiess D, Speel EJ, Winnepenninckx V, Diaz C, et al. Frequent detection of human polyomavirus 6 in keratoacanthomas. Diagn Pathol. 2016;11(1):58. DOI:10.1186/s13000-016-0509-z." />
<meta name="citation_reference" content="Hausen Hz. Red meat consumption and cancer: reasons to suspect involvement of bovine infectious factors in colorectal cancer. Int J Cancer. 2012;130(11):2475â83. DOI:10.1002/ijc.27413." />
<meta name="citation_reference" content="Sauerbrei A, Wutzler P. Testing thermal resistance of viruses. Arch Virol. 2009;154(1):115â9. DOI:10.1007/s00705-008-0264-x." />
<meta name="citation_reference" content="Anthony SJ, Islam A, Johnson C, Navarrete-Macias I, Liang E, Jain K, et al. Non-random patterns in viral diversity. Nat Commun. 2015;6:8147. DOI:10.1038/ncomms9147." />
<meta name="citation_reference" content="Campanero-Rhodes MA, Smith A, Chai W, Sonnino S, Mauri L, Childs RA, et al. N-glycolyl GM1 ganglioside as a receptor for simian virus 40. J Virol. 2007;81(23):12846â58. DOI:10.1128/jvi.01311-07." />
<meta name="citation_reference" content="OâHara SD, Stehle T, Garcea R. Glycan receptors of the Polyomaviridae: Structure, function, and pathogenesis. Curr Opin Virol. 2014;7:73â8. DOI:10.1016/j.coviro.2014.05.004." />
<meta name="citation_reference" content="Jin L, Gibson PE, Knowles WA, Clewley JP. BK virus antigenic variants: sequence analysis within the capsid VP1 epitope. J Med Virol. 1993;39(1):50â6. DOI:10.1002/jmv.1890390110" />
<meta name="citation_reference" content="Erickson KD, Garcea RL, Tsai B. Ganglioside GT1b is a putative host cell receptor for the Merkel cell polyomavirus. J Virol. 2009;83(19):10275â9. DOI:10.1128/jvi.00949-09." />
<meta name="citation_reference" content="Murata H, Teferedegne B, Sheng L, Lewis AM, Peden K. Identification of a neutralization epitope in the VP1 capsid protein of SV40. Virology. 2008;381(1):116â22. DOI:10.1016/j.virol.2008.07.032." />
<meta name="citation_reference" content="Fleury MJ, Nicol JT, Samimi M, Arnold F, Cazal R, Ballaire R, et al. Identification of the neutralizing epitopes of Merkel cell polyomavirus major capsid protein within the BC and EF surface loops. PLoS One. 2015;10(3):e0121751. DOI:10.1371/journal.pone.0121751." />
<meta name="citation_reference" content="StrÃ¶h LJ, Neu U, Blaum BS, Buch MHC, Garcea RL, Stehle T. Structure analysis of the major capsid proteins of human polyomaviruses 6 and 7 reveals an obstructed sialic acid binding site. J Virol. 2014;88(18):10831. DOI:10.1128/JVI.01084-14." />
<meta name="citation_reference" content="Teeling EC, Vernes SC, Davalos LM, Ray DA, Gilbert MTP, Myers E. Bat biology, genomes, and the Bat1K project: To generate chromosome-level genomes for all Living bat species. Annu Rev Anim Biosci. 2018;6:23â46. DOI:10.1146/annurev-animal-022516-022811." />
<meta name="citation_reference" content="Orba Y, Kobayashi S, Nakamura I, Ishii A, Hang, Apos, et al. Detection and characterization of a novel polyomavirus in wild rodents. J Gen Virol. 2011;92(4):789â95. DOI:DOI:10.1099/vir.0.027854-0." />
<meta name="citation_reference" content="Muhire BM, Varsani A, Martin DP. SDT: A virus classification tool based on pairwise sequence alignment and identity calculation. PLoS One. 2014;9(9):e108277. DOI:10.1371/journal.pone.0108277." />
<meta name="citation_reference" content="Katoh K, Standley DM. MAFFT multiple sequence alignment software version 7: Improvements in performance and usability. Mol Biol Evol. 2013;30(4):772â80. DOI:10.1093/molbev/mst010." />
<meta name="citation_reference" content="Ronquist F, Teslenko M, van der Mark P, Ayres DL, Darling A, HÃ¶hna S, et al. MrBayes 3.2: Efficient Bayesian phylogenetic inference and model choice across a large model space. Syst Biol. 2012;61(3):539â42. DOI:10.1093/sysbio/sys029." />
<meta name="citation_reference" content="Carr M, Kawaguchi A, Sasaki M, Gonzalez G, Ito K, Thomas Y, et al. Isolation of a simian immunodeficiency virus from a malbrouck (Chlorocebus cynosuros). Arch Virol. 2017;162(2):543-8. Epub 2016/11/03. DOI:10.1007/s00705-016-3129-8." />
<meta name="citation_reference" content="Kumar S, Stecher G, Tamura K. MEGA7: molecular evolutionary genetics analysis version 7.0 for bigger datasets. Mol Biol Evol. 2016;33(7):1870â4. DOI:10.1093/molbev/msw054." />
<meta name="citation_reference" content="Drummond AJ, Suchard MA, Xie D, Rambaut A. Bayesian phylogenetics with BEAUti and the BEAST 1.7. Mol Biol Evol. 2012;29(8):1969â73. DOI:10.1093/molbev/mss075." />
<meta name="citation_reference" content="Perelman P, Johnson WE, Roos C, SeuÃ¡nez HN, Horvath JE, Moreira MAM, et al. A molecular phylogeny of living primates. PLoS Genet. 2011;7(3):e1001342. DOI:10.1371/journal.pgen.1001342." />
<meta name="citation_reference" content="Conow C, Fielder D, Ovadia Y, Libeskind-Hadas R. Jane: a new tool for the cophylogeny reconstruction problem. Algorithms Mol Biol. 2010;5(1):16. DOI:10.1186/1748-7188-5-16." />
<meta name="citation_reference" content="Waterhouse A, Bertoni M, Bienert S, Studer G, Tauriello G, Gumienny R, et al. SWISS-MODEL: homology modelling of protein structures and complexes. Nucleic Acids Res. 2018;46(W1):W296-W303. DOI:10.1093/nar/gky427." />
<meta name="twitter:title" content="Extensive genetic diversity of bat-borne polyomaviruses reveals inter-family host-switching events" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Polyomaviruses (PyVs) are small, double-stranded DNA tumor viruses carried by diverse vertebrates. PyVs have previously been considered highly host restricted in mammalian hosts, with host-switching events thought rare or nonexistent. Prior investigations have revealed short-range host-switching events of PyVs in two different African bat species within the horseshoe bat genus Rhinolophus . Herein, we have conducted a systematic investigation of PyVs in 1,083 archived bat samples collected from five provinces across China, and identified 192 PyVs from 186 bats from 15 host species within 6 families (Rhinolophidae, Vespertilionidae, Hipposideridae, Emballonuridae, Miniopteridae and Pteropodidae) representing 28 newly-described PyVs, indicative of extensive genetic diversity of bat PyVs. Surprisingly, two PyVs were identified in multiple bat species from different families, and another PyV clustered phylogenetically with PyVs carried by bats from a different host family, indicative of three inter-family PyV host-switching events. The time to most recent common ancestor (tMRCA) of the three events was estimated at 0.02-11.6 million years ago (MYA), which is inconsistent with the estimated tMRCA of their respective bat hosts (36.3-66.7 MYA), and is most parsimoniously explained by host-switching events. PyVs identified from geographically separated Chinese horseshoe bat species in the present study showed close genetic identities, and clustered with each other and with PyVs from African horseshoe bats, allowing assessment of the effects of positive selection in VP1 within the horseshoe bat family Rhinolophidae. Correlation analysis indicated that co-evolution with their hosts contributed much more to evolutionary divergence of PyV than geographic distance. In conclusion, our findings provide the first evidence of inter-family host-switching events of PyV in mammals and challenge the prevailing evolutionary paradigm for strict host restriction of mammalian PyVs.

Author summary Since the discovery of murine polyomavirus in the 1950s, polyomaviruses (PyVs) have been considered both genetically stable and highly host-restricted in their mammalian hosts. In this study, we have identified multiple cases of host-switching events of PyVs by large scale surveillance in diverse bat species collected in China. These host-switching events occurred between bat families living in the same colony, indicating that a large population with frequent contacts between different bat species may represent an ecological niche facilitating PyV host-switching. The cases studied involved members of bats from several families, including horseshoe bats, which were previously found to harbor a number of highly virulent viruses to both humans and domestic animals. Our findings have provided evidence that even highly host-specific DNA viruses can transmit between bats of different species and indicate an increased propensity for spillover events involving horseshoe bats. We propose an evolutionary scheme for bat-borne PyVs in which intra-host divergence and host-switching has generated the diverse PyVs in present day bats. This scheme provides a useful model to study the evolution of PyVs in other hosts and, potentially, the modeling of bat zoonoses and the transmission of other DNA viruses in other mammals, including humans." />
<meta name="og-title" property="og:title" content="Extensive genetic diversity of bat-borne polyomaviruses reveals inter-family host-switching events" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/627158v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Polyomaviruses (PyVs) are small, double-stranded DNA tumor viruses carried by diverse vertebrates. PyVs have previously been considered highly host restricted in mammalian hosts, with host-switching events thought rare or nonexistent. Prior investigations have revealed short-range host-switching events of PyVs in two different African bat species within the horseshoe bat genus Rhinolophus . Herein, we have conducted a systematic investigation of PyVs in 1,083 archived bat samples collected from five provinces across China, and identified 192 PyVs from 186 bats from 15 host species within 6 families (Rhinolophidae, Vespertilionidae, Hipposideridae, Emballonuridae, Miniopteridae and Pteropodidae) representing 28 newly-described PyVs, indicative of extensive genetic diversity of bat PyVs. Surprisingly, two PyVs were identified in multiple bat species from different families, and another PyV clustered phylogenetically with PyVs carried by bats from a different host family, indicative of three inter-family PyV host-switching events. The time to most recent common ancestor (tMRCA) of the three events was estimated at 0.02-11.6 million years ago (MYA), which is inconsistent with the estimated tMRCA of their respective bat hosts (36.3-66.7 MYA), and is most parsimoniously explained by host-switching events. PyVs identified from geographically separated Chinese horseshoe bat species in the present study showed close genetic identities, and clustered with each other and with PyVs from African horseshoe bats, allowing assessment of the effects of positive selection in VP1 within the horseshoe bat family Rhinolophidae. Correlation analysis indicated that co-evolution with their hosts contributed much more to evolutionary divergence of PyV than geographic distance. In conclusion, our findings provide the first evidence of inter-family host-switching events of PyV in mammals and challenge the prevailing evolutionary paradigm for strict host restriction of mammalian PyVs.

Author summary Since the discovery of murine polyomavirus in the 1950s, polyomaviruses (PyVs) have been considered both genetically stable and highly host-restricted in their mammalian hosts. In this study, we have identified multiple cases of host-switching events of PyVs by large scale surveillance in diverse bat species collected in China. These host-switching events occurred between bat families living in the same colony, indicating that a large population with frequent contacts between different bat species may represent an ecological niche facilitating PyV host-switching. The cases studied involved members of bats from several families, including horseshoe bats, which were previously found to harbor a number of highly virulent viruses to both humans and domestic animals. Our findings have provided evidence that even highly host-specific DNA viruses can transmit between bats of different species and indicate an increased propensity for spillover events involving horseshoe bats. We propose an evolutionary scheme for bat-borne PyVs in which intra-host divergence and host-switching has generated the diverse PyVs in present day bats. This scheme provides a useful model to study the evolution of PyVs in other hosts and, potentially, the modeling of bat zoonoses and the transmission of other DNA viruses in other mammals, including humans." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-05-03" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/627158v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/627158v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/627158v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/627158v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/682555" />
    <title>Extensive genetic diversity of bat-borne polyomaviruses reveals inter-family host-switching events | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__uGxiwDeWBOA2NAqiRKvJS7CKqv2_wE38cKwCd6tvrVo__XFIGL9IJsJXY4c62YMBljQoIaCQJsgoTa5cp7itS8y4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"F5ybwjnwyfLhv6dwvlN5e6NOVsdLTN0KYXYIB_4HuV4","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"682555","apath":"\/biorxiv\/early\/2019\/05\/03\/627158.atom","pisa":"biorxiv;627158v1","ac":{"\/biorxiv\/early\/2019\/05\/03\/627158.atom":{"access":{"full":true},"pisa_id":"","apath":"\/biorxiv\/early\/2019\/05\/03\/627158.atom","jcode":"biorxiv"}},"processed":["highwire_math"],"markup":[{"requested":"abstract","variant":"abstract","view":"abstract","pisa":"biorxiv;627158v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/627158v1","title":"Extensive genetic diversity of bat-borne polyomaviruses reveals inter-family host-switching events","identifier":"node\/682555"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022NyHFqDjtt3XpmlbIgjxbMVERoZou8ExzHfeUNIast52uVNBAFsIb0XsXo5rBdDjNyWYF0eZDobP91l5h3iI5ezpxyU9s2woq4IoC6YA+5opidIWMElj\\\\\\\/YrudI2tnTZgakKUAT+Mzbla19cz4kDZ2VBV0eprr7i8AYtg\\\\\\\/AIpoS8VNvbuQCs8p\\\\\\\/p4czfCSSbbTBEau3b8l5ufCnRGZpqllmshJxFJrkIQeRhOBOScDSjtq21deNn8ryHc8S4fdvRG57fCduKNhXfJvMVMFGqBpUGo9NPWholUu459bA0i0NEyAzRcOxo+Phf7QamOm0+VL\\\\\\\/N17NaH94OL9KI\\\\\\\/xgsPv2w9dZ3b6AkDqi0q4HGQ0ZMDfuFBj8N63gnDR0Ne4Zk\\\\\\\/vdjC3P6GPtNdxI99IRn6lXa4DQXtZfRBL215W8hEVaYsIRguOVt\\\\\\\/nxcusf2TWp3CnSOHXggIKg0tJnUYhkrCprw9fDtw8TTU12FVz6\\\\\\\/IUL+I8Lr2OFKA7YpAEuB2jPnI3BvGl8ez5e0mrFxW4hvm4ZHDp7IrxYy61ex1WdcJ7ypAoFIy\\\\\\\/wrs5RbKIWXHYCbpM5sHhOX8SwU4nHzL6a7v4tr4c8LAnyyunhdfbwwRotNMtvpu6gvOOYiWMU8YmtYjQ3tAb5xgs7lWz14s6uUo\\\\\\\/6NxRYZK4Sow5+NBl1bViiVvLWNGn98qCxh\\\\\\\/umBFMUyBhJSn0IYuuR3XZImhzow2wxayzxP6XrdfjQlHW7RcRJMh\\\\\\\/8cxEk6X1o2Zx3JU1aYvoQg6iHkGKglzBIE1KeZcu+5p1HlpzBNOhiDpq3bW5\\\\\\\/TrUrmVbvRyvADiOSlsmOqaq0OpCcmxZ7bZJSk\\\\\\\/5ObnlOk38+kOl0xX51NSbYGogbm0j4Wy31SF9DVchUsL7oseqUqziwLhA0MXf7CJWuIxdOwD2Yg8MznGpWnWlAzv5UQkIBJziZ9sMdCv+N3cxhEiigFCK8GCSz5hxMC0YwsenrzQThUbIYxU0KXLW2Fc0aZndMMDdS6z8h1\\\\\\\/65WD2k0ixc0WL6lTVi7M93hb28GknB4qULopm83961rwlNVKUF4pI7NkYbORQC00eZHo7gkXR61ZL3wDIqHmPHmSVDEhjZICfoFk37TcJL0y51h0Dwj\\\\\\\/JV1JJpJdzmykzcHGTBhIdhe\\\\\\\/7MO8hXm79XDd8QIkTGSIJbZG0ksW6p0sVy8hV9WNrFR5+G\\\\\\\/b82UbjfJTsybE3EziIlPbCT9pGdrLRAoV0Bqpz9u+Rlz9J6gdrBBbLuwtuP\\\\\\\/BE1KFt75zxUAFrVam2\\\u0022,\\\u0022iv\\\u0022:\\\u0022WpLgzc4ukR1yT3qJv\\\\\\\/kBnA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002219d89b2b8176ea152b69664c04698db3230d2878179b416182086b61149a7dc3\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/627158v1":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-682555 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/627158v1" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-hzgQ7lnoBKFRNJ__IhK8X0sdeDO0D4htjhT3ltV2tsQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="682555" id="node682555" data-pisa="biorxiv;627158v1" data-pisa-master="biorxiv;627158" data-apath="/biorxiv/early/2019/05/03/627158.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Extensive genetic diversity of bat-borne polyomaviruses reveals inter-family host-switching events</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-2844-9673" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Zhizhou</span> <span class="nlm-surname">Tan</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2180-2120" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Gabriel</span> <span class="nlm-surname">Gonzalez</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jinliang</span> <span class="nlm-surname">Sheng</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Jianmin</span> <span class="nlm-surname">Wu</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Fuqiang</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Lin</span> <span class="nlm-surname">Xu</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Peisheng</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Aiwei</span> <span class="nlm-surname">Zhu</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Yonggang</span> <span class="nlm-surname">Qu</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Changchun</span> <span class="nlm-surname">Tu</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0003-0011-6209" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Michael J.</span> <span class="nlm-surname">Carr</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="11"><a href="http://orcid.org/0000-0001-7587-152X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Biao</span> <span class="nlm-surname">He</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/627158 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node682555" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhizhou Tan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Key Laboratory of Jilin Province for Zoonosis Prevention and Control, Institute of Military Veterinary Medicine, Academy of Military Medical Sciences, Changchun</span>, Jilin Province, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhizhou%2BTan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tan%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZhizhou%2BTan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2844-9673" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Zhizhou Tan</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gabriel Gonzalez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>National Advanced Computing Collaboratory, National Center for High Technology</span>, San Jose, Costa Rica</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gabriel%2BGonzalez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gonzalez%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGabriel%2BGonzalez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2180-2120" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gabriel Gonzalez</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jinliang Sheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>College of Animal Science and Technology, Shihezi University, Shihezi, Xinjiang Uyghur Autonomous Region</span>, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jinliang%2BSheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sheng%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJinliang%2BSheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jianmin Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Guangxi Key Laboratory of Veterinary Biotechnology, Guangxi Veterinary Research Institute, Nanning</span>, Guangxi Province, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jianmin%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJianmin%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Fuqiang Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Center for Disease Control and Prevention of Southern Theater Command</span>, Kunming, Yunnan Province, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fuqiang%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFuqiang%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Lin Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Key Laboratory of Jilin Province for Zoonosis Prevention and Control, Institute of Military Veterinary Medicine, Academy of Military Medical Sciences, Changchun</span>, Jilin Province, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lin%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALin%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Peisheng Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Key Laboratory of Jilin Province for Zoonosis Prevention and Control, Institute of Military Veterinary Medicine, Academy of Military Medical Sciences, Changchun</span>, Jilin Province, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>College of Animal Science and Technology, Shihezi University, Shihezi, Xinjiang Uyghur Autonomous Region</span>, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peisheng%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeisheng%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Aiwei Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Key Laboratory of Jilin Province for Zoonosis Prevention and Control, Institute of Military Veterinary Medicine, Academy of Military Medical Sciences, Changchun</span>, Jilin Province, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aiwei%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAiwei%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Yonggang Qu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>College of Animal Science and Technology, Shihezi University, Shihezi, Xinjiang Uyghur Autonomous Region</span>, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yonggang%2BQu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Qu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYonggang%2BQu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Changchun Tu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Key Laboratory of Jilin Province for Zoonosis Prevention and Control, Institute of Military Veterinary Medicine, Academy of Military Medical Sciences, Changchun</span>, Jilin Province, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Jiangsu Co-innovation Center for Prevention and Control of Important Animal Infectious Diseases and Zoonosis, Yangzhou</span>, Jiangsu Province, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Changchun%2BTu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tu%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChangchun%2BTu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:heb-001001@163.com" class="" data-icon-position="" data-hide-link-title="0">heb-001001@163.com</a>
<a href="mailto:michael.carr@ucd.ie" class="" data-icon-position="" data-hide-link-title="0">michael.carr@ucd.ie</a>
<a href="mailto:changchun_tu@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">changchun_tu@hotmail.com</a></span></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Michael J. Carr </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>7</span><span class='nlm-institution'>National Virus Reference Laboratory, School of Medicine, University College Dublin</span>, Dublin 4, <span class='nlm-country'>Ireland</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BJ.%2BCarr%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carr%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BJ.%2BCarr%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0011-6209" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael J. Carr</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:heb-001001@163.com" class="" data-icon-position="" data-hide-link-title="0">heb-001001@163.com</a>
<a href="mailto:michael.carr@ucd.ie" class="" data-icon-position="" data-hide-link-title="0">michael.carr@ucd.ie</a>
<a href="mailto:changchun_tu@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">changchun_tu@hotmail.com</a></span></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Biao He </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Key Laboratory of Jilin Province for Zoonosis Prevention and Control, Institute of Military Veterinary Medicine, Academy of Military Medical Sciences, Changchun</span>, Jilin Province, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Jiangsu Co-innovation Center for Prevention and Control of Important Animal Infectious Diseases and Zoonosis, Yangzhou</span>, Jiangsu Province, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Biao%2BHe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=He%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABiao%2BHe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7587-152X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Biao He</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:heb-001001@163.com" class="" data-icon-position="" data-hide-link-title="0">heb-001001@163.com</a>
<a href="mailto:michael.carr@ucd.ie" class="" data-icon-position="" data-hide-link-title="0">michael.carr@ucd.ie</a>
<a href="mailto:changchun_tu@hotmail.com" class="" data-icon-position="" data-hide-link-title="0">changchun_tu@hotmail.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/627158v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:682555" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:682555/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/627158v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:682555" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:682555/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/627158v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:682555" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:682555/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/627158v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:682555" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:682555/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/627158v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:682555" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:682555/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="biorxiv_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-biorxiv_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article abstract-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">Polyomaviruses (PyVs) are small, double-stranded DNA tumor viruses carried by diverse vertebrates. PyVs have previously been considered highly host restricted in mammalian hosts, with host-switching events thought rare or nonexistent. Prior investigations have revealed short-range host-switching events of PyVs in two different African bat species within the horseshoe bat genus <em>Rhinolophus</em>. Herein, we have conducted a systematic investigation of PyVs in 1,083 archived bat samples collected from five provinces across China, and identified 192 PyVs from 186 bats from 15 host species within 6 families (Rhinolophidae, Vespertilionidae, Hipposideridae, Emballonuridae, Miniopteridae and Pteropodidae) representing 28 newly-described PyVs, indicative of extensive genetic diversity of bat PyVs. Surprisingly, two PyVs were identified in multiple bat species from different families, and another PyV clustered phylogenetically with PyVs carried by bats from a different host family, indicative of three inter-family PyV host-switching events. The time to most recent common ancestor (tMRCA) of the three events was estimated at 0.02-11.6 million years ago (MYA), which is inconsistent with the estimated tMRCA of their respective bat hosts (36.3-66.7 MYA), and is most parsimoniously explained by host-switching events. PyVs identified from geographically separated Chinese horseshoe bat species in the present study showed close genetic identities, and clustered with each other and with PyVs from African horseshoe bats, allowing assessment of the effects of positive selection in VP1 within the horseshoe bat family Rhinolophidae. Correlation analysis indicated that co-evolution with their hosts contributed much more to evolutionary divergence of PyV than geographic distance. In conclusion, our findings provide the first evidence of inter-family host-switching events of PyV in mammals and challenge the prevailing evolutionary paradigm for strict host restriction of mammalian PyVs.</p><div id="sec-1" class="subsection"><p id="p-4"><strong>Author summary</strong> Since the discovery of murine polyomavirus in the 1950s, polyomaviruses (PyVs) have been considered both genetically stable and highly host-restricted in their mammalian hosts. In this study, we have identified multiple cases of host-switching events of PyVs by large scale surveillance in diverse bat species collected in China. These host-switching events occurred between bat families living in the same colony, indicating that a large population with frequent contacts between different bat species may represent an ecological niche facilitating PyV host-switching. The cases studied involved members of bats from several families, including horseshoe bats, which were previously found to harbor a number of highly virulent viruses to both humans and domestic animals. Our findings have provided evidence that even highly host-specific DNA viruses can transmit between bats of different species and indicate an increased propensity for spillover events involving horseshoe bats. We propose an evolutionary scheme for bat-borne PyVs in which intra-host divergence and host-switching has generated the diverse PyVs in present day bats. This scheme provides a useful model to study the evolution of PyVs in other hosts and, potentially, the modeling of bat zoonoses and the transmission of other DNA viruses in other mammals, including humans.</p></div></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-copyright" >
  
      
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-copyright field-type-text field-label-inline clearfix"><div class="field-label">Copyright&nbsp;</div><div class="field-items"><div class="field-item even">The copyright holder for this preprint is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.<span class="license-type"> It is made available under a <a href="http://creativecommons.org/licenses/by/4.0/" class="" data-icon-position="" data-hide-link-title="0">CC-BY 4.0 International license</a>.</span></div></div></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F627158v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/627133v1" title="Surprise response as a probe for compressed memory states" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;May 03, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/627158v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/627158v1" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/627158v1" class="active" data-icon-position="" data-hide-link-title="0">Extensive genetic diversity of bat-borne polyomaviruses reveals inter-family host-switching events</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/682555" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-8ibr0qpk1P97DZ58nJDlMr9TAOkPCpugAfrPWOBCduA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="682555" id="node682555--2" data-pisa="biorxiv;627158v1" data-pisa-master="biorxiv;627158" data-seqnum="682555" data-apath="/biorxiv/early/2019/05/03/627158.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Extensive genetic diversity of bat-borne polyomaviruses reveals inter-family host-switching events</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Zhizhou</span> <span class="nlm-surname">Tan</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Gabriel</span> <span class="nlm-surname">Gonzalez</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jinliang</span> <span class="nlm-surname">Sheng</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Jianmin</span> <span class="nlm-surname">Wu</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Fuqiang</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Lin</span> <span class="nlm-surname">Xu</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Peisheng</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Aiwei</span> <span class="nlm-surname">Zhu</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Yonggang</span> <span class="nlm-surname">Qu</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Changchun</span> <span class="nlm-surname">Tu</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Michael J.</span> <span class="nlm-surname">Carr</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Biao</span> <span class="nlm-surname">He</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">627158; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/627158 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/627158v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/627158v1&amp;title=Extensive%20genetic%20diversity%20of%20bat-borne%20polyomaviruses%20reveals%20inter-family%20host-switching%20events" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/627158v1&amp;title=Extensive%20genetic%20diversity%20of%20bat-borne%20polyomaviruses%20reveals%20inter-family%20host-switching%20events" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/627158v1&amp;text=Extensive%20genetic%20diversity%20of%20bat-borne%20polyomaviruses%20reveals%20inter-family%20host-switching%20events" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/627158v1&amp;title=Extensive%20genetic%20diversity%20of%20bat-borne%20polyomaviruses%20reveals%20inter-family%20host-switching%20events" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/627158v1&amp;t=Extensive%20genetic%20diversity%20of%20bat-borne%20polyomaviruses%20reveals%20inter-family%20host-switching%20events" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/627158v1&amp;title=Extensive%20genetic%20diversity%20of%20bat-borne%20polyomaviruses%20reveals%20inter-family%20host-switching%20events" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/627158v1&amp;title=Extensive%20genetic%20diversity%20of%20bat-borne%20polyomaviruses%20reveals%20inter-family%20host-switching%20events" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="682555" id="node682555--3" data-pisa="biorxiv;627158v1" data-pisa-master="biorxiv;627158" data-seqnum="682555" data-apath="/biorxiv/early/2019/05/03/627158.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Extensive genetic diversity of bat-borne polyomaviruses reveals inter-family host-switching events</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Zhizhou</span> <span class="nlm-surname">Tan</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Gabriel</span> <span class="nlm-surname">Gonzalez</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jinliang</span> <span class="nlm-surname">Sheng</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Jianmin</span> <span class="nlm-surname">Wu</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Fuqiang</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Lin</span> <span class="nlm-surname">Xu</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Peisheng</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Aiwei</span> <span class="nlm-surname">Zhu</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Yonggang</span> <span class="nlm-surname">Qu</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Changchun</span> <span class="nlm-surname">Tu</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Michael J.</span> <span class="nlm-surname">Carr</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Biao</span> <span class="nlm-surname">He</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">627158; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/627158 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Extensive%20genetic%20diversity%20of%20bat-borne%20polyomaviruses%20reveals%20inter-family%20host-switching%20events&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Tan%2C+Zhizhou&amp;rft.au=Gonzalez%2C+Gabriel&amp;rft.au=Sheng%2C+Jinliang&amp;rft.au=Wu%2C+Jianmin&amp;rft.au=Zhang%2C+Fuqiang&amp;rft.au=Xu%2C+Lin&amp;rft.au=Zhang%2C+Peisheng&amp;rft.au=Zhu%2C+Aiwei&amp;rft.au=Qu%2C+Yonggang&amp;rft.au=Tu%2C+Changchun&amp;rft.au=Carr%2C+Michael+J.&amp;rft.au=He%2C+Biao"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/682555/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/682555/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/682555/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/682555/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/682555/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/682555/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/682555/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/682555/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/682555/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/682555/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/682555/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/682555/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/627158v1&amp;count=horizontal&amp;via=&amp;text=Extensive%20genetic%20diversity%20of%20bat-borne%20polyomaviruses%20reveals%20inter-family%20host-switching%20events&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/627158v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/627158v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/627158v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">Microbiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__Vh5BcRYb6VtLoN3uam6O4DIKltYUVMjVDWtakoysPq0__zA_9xHyAk-lRGLS85oOPfK7yEWnMzbYSmnTOXQEsvJw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
