<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/03/30/2020.03.28.013920.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.03.28.013920v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Crystal structure of the SARS-CoV-2 non-structural protein 9, Nsp9"/>
  <meta name="DC.Identifier" content="10.1101/2020.03.28.013920"/>
  <meta name="DC.Date" content="2020-03-30"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NoDerivs 4.0 International), CC BY-ND 4.0, as described at http://creativecommons.org/licenses/by-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Many of the proteins produced by SARS-CoV-2 have related counterparts across the Severe Acute Respiratory Syndrome (SARS-CoV) family. One such protein is non-structural protein 9 (Nsp9), which is thought to mediate both viral replication and virulence. Current understanding suggests that Nsp9 is involved in viral genomic RNA reproduction. Nsp9 is thought to bind RNA via a fold that is unique to this class of betacoronoaviruses although the molecular basis for this remains ill-defined. We sought to better characterise the SARS-CoV-2 Nsp9 protein and subsequently solved its X-ray crystal structure, in an apo-form and, unexpectedly, in a peptide-bound form with a sequence originating from a rhinoviral 3C protease sequence (LEVL). The structure of the SARS-CoV-2 Nsp9 revealed the high level of structural conservation within the Nsp9 family. The exogenous peptide binding site is close to the dimer interface and impacted on the relative juxtaposition of the monomers within the homodimer. Together we have established a protocol for the production of SARS-CoV-2 Nsp9, determined its structure and identified a peptide-binding site that may warrant further study from the perspective of understanding Nsp9 function."/>
  <meta name="DC.Contributor" content="D. R. Littler"/>
  <meta name="DC.Contributor" content="B. S. Gully"/>
  <meta name="DC.Contributor" content="R. N. Colson"/>
  <meta name="DC.Contributor" content="J Rossjohn"/>
  <meta name="article:published_time" content="2020-03-30"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Crystal structure of the SARS-CoV-2 non-structural protein 9, Nsp9"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;Many of the proteins produced by SARS-CoV-2 have related counterparts across the Severe Acute Respiratory Syndrome (SARS-CoV) family. One such protein is non-structural protein 9 (Nsp9), which is thought to mediate both viral replication and virulence. Current understanding suggests that Nsp9 is involved in viral genomic RNA reproduction. Nsp9 is thought to bind RNA via a fold that is unique to this class of betacoronoaviruses although the molecular basis for this remains ill-defined. We sought to better characterise the SARS-CoV-2 Nsp9 protein and subsequently solved its X-ray crystal structure, in an apo-form and, unexpectedly, in a peptide-bound form with a sequence originating from a rhinoviral 3C protease sequence (LEVL). The structure of the SARS-CoV-2 Nsp9 revealed the high level of structural conservation within the Nsp9 family. The exogenous peptide binding site is close to the dimer interface and impacted on the relative juxtaposition of the monomers within the homodimer. Together we have established a protocol for the production of SARS-CoV-2 Nsp9, determined its structure and identified a peptide-binding site that may warrant further study from the perspective of understanding Nsp9 function.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.03.28.013920v1"/>
  <meta name="citation_id" content="2020.03.28.013920v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.28.013920v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.28.013920v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.28.013920v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.28.013920.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.03.28.013920"/>
  <meta name="citation_num_pages" content="6"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.03.28.013920"/>
  <meta name="citation_author" content="D. R. Littler"/>
  <meta name="citation_author_institution" content="Infection and Immunity Program, Department of Biochemistry and Molecular Biology, Biomedicine Discovery Institute, Monash University"/>
  <meta name="citation_author_institution" content="Australian Research Council Centre of Excellence for Advanced Molecular Imaging, Monash University"/>
  <meta name="citation_author_email" content="dene.littler@monash.edu"/>
  <meta name="citation_author_email" content="Jamie.rossjohn@monash.edu"/>
  <meta name="citation_author" content="B. S. Gully"/>
  <meta name="citation_author_institution" content="Infection and Immunity Program, Department of Biochemistry and Molecular Biology, Biomedicine Discovery Institute, Monash University"/>
  <meta name="citation_author_institution" content="Australian Research Council Centre of Excellence for Advanced Molecular Imaging, Monash University"/>
  <meta name="citation_author" content="R. N. Colson"/>
  <meta name="citation_author_institution" content="Infection and Immunity Program, Department of Biochemistry and Molecular Biology, Biomedicine Discovery Institute, Monash University"/>
  <meta name="citation_author" content="J Rossjohn"/>
  <meta name="citation_author_institution" content="Infection and Immunity Program, Department of Biochemistry and Molecular Biology, Biomedicine Discovery Institute, Monash University"/>
  <meta name="citation_author_institution" content="Australian Research Council Centre of Excellence for Advanced Molecular Imaging, Monash University"/>
  <meta name="citation_author_institution" content="Institute of Infection and Immunity, Cardiff University School of Medicine"/>
  <meta name="citation_author_email" content="dene.littler@monash.edu"/>
  <meta name="citation_author_email" content="Jamie.rossjohn@monash.edu"/>
  <meta name="citation_reference" content="Aragao, D., Aishima, J., Cherukuvada, H., Clarken, R., Clift, M., Cowieson, N.P., Ericsson, D.J., Gee, C.L., Macedo, S., Mudie, N., et al. (2018). MX2: a high-flux undulator microfocus beamline serving both the chemical and macromolecular crystallography communities at the Australian Synchrotron. J Synchrotron Radiat 25, 885–891."/>
  <meta name="citation_reference" content="Casanal, A., Lohkamp, B., and Emsley, P. (2019). Current developments in Coot for macromolecular model building of Electron Cryo-microscopy and Crystallographic Data. Protein Sci."/>
  <meta name="citation_reference" content="Aragao, D., Aishima, J., Cherukuvada, H., Clarken, R., Clift, M., Cowieson, N.P., Ericsson, D.J., Gee, C.L., Macedo, S., Mudie, N., et al. (2018). MX2: a high-flux undulator microfocus beamline serving both the chemical and macromolecular crystallography communities at the Australian Synchrotron. J Synchrotron Radiat 25, 885–891."/>
  <meta name="citation_reference" content="Casanal, A., Lohkamp, B., and Emsley, P. (2019). Current developments in Coot for macromolecular model building of Electron Cryo-microscopy and Crystallographic Data. Protein Sci."/>
  <meta name="citation_reference" content="Chen, J., Xu, X., Tao, H., Li, Y., Nan, H., Wang, Y., Tian, M., and Chen, H. (2017). Structural Analysis of Porcine Reproductive and Respiratory Syndrome Virus Non-structural Protein 7alpha (NSP7alpha) and Identification of Its Interaction with NSP9. Front Microbiol 8, 853."/>
  <meta name="citation_reference" content="Egloff, M.P., Ferron, F., Campanacci, V., Longhi, S., Rancurel, C., Dutartre, H., Snijder, E.J., Gorbalenya, A.E., Cambillau, C., and Canard, B. (2004). The severe acute respiratory syndrome-coronavirus replicative protein nsp9 is a single-stranded RNA-binding subunit unique in the RNA virus world. Proc Natl Acad Sci U S A 101, 3792–3796."/>
  <meta name="citation_reference" content="Frieman, M., Yount, B., Agnihothram, S., Page, C., Donaldson, E., Roberts, A., Vogel, L., Woodruff, B., Scorpio, D., Subbarao, K., et al. (2012). Molecular determinants of severe acute respiratory syndrome coronavirus pathogenesis and virulence in young and aged mouse models of human disease. J Virol 86, 884–897."/>
  <meta name="citation_reference" content="Goh, K.J., Choong, M.C., Cheong, E.H., Kalimuddin, S., Duu Wen, S., Phua, G.C., Chan, K.S., and Haja Mohideen, S. (2020). Rapid Progression to Acute Respiratory Distress Syndrome: Review of Current Understanding of Critical Illness from COVID-19 Infection. Ann Acad Med Singapore 49, 1–9."/>
  <meta name="citation_reference" content="Hu, T., Chen, C., Li, H., Dou, Y., Zhou, M., Lu, D., Zong, Q., Li, Y., Yang, C., Zhong, Z., et al. (2017). Structural basis for dimerization and RNA binding of avian infectious bronchitis virus nsp9. Protein Sci 26, 1037–1048."/>
  <meta name="citation_reference" content="Huang, C., Wang, Y., Li, X., Ren, L., Zhao, J., Hu, Y., Zhang, L., Fan, G., Xu, J., Gu, X., et al. (2020). Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China. Lancet 395, 497–506."/>
  <meta name="citation_reference" content="Kabsch, W. (2010). Xds. Acta Crystallogr D Biol Crystallogr 66, 125–132."/>
  <meta name="citation_reference" content="Liebschner, D., Afonine, P.V., Baker, M.L., Bunkoczi, G., Chen, V.B., Croll, T.I., Hintze, B., Hung, L.W., Jain, S., McCoy, A.J., et al. (2019). Macromolecular structure determination using X-rays, neutrons and electrons: recent developments in Phenix. Acta Crystallogr D Struct Biol 75, 861–877."/>
  <meta name="citation_reference" content="Zhenming, J., Xiaoyu, D., Yechun, X., Yongqiang, D., Meiqin, L., Yao, Z., Bing, Z., Xiaofeng, L., Leike, Z., Chao, P., Yinkai, D., Jing, Y., Lin, W., Kailin, Y., Fengjiang, L., Rendi, J., Xinglou, Y., Tian, Y., Xiaoce, L., Xiuna, Y., Fang, B., Hong, L., Xiang, L., Luke, W. G., Wenqing, X., Gengfu, X., Chengfeng, Q., Zhengli, S., Hualiang, J., Zihe, R., &amp;; Haitao, Y. (2020) Structure of Mpro 1 from COVID-19 virus and discovery of its inhibitors bioRxiv doi: https://doi.org/10.1101/2020.02.26.964882"/>
  <meta name="citation_reference" content="Miknis, Z.J., Donaldson, E.F., Umland, T.C., Rimmer, R.A., Baric, R.S., and Schultz, L.W. (2009). Severe acute respiratory syndrome coronavirus nsp9 dimerization is essential for efficient viral growth. J Virol 83, 3007–3018."/>
  <meta name="citation_reference" content="Ponnusamy, R., Moll, R., Weimar, T., Mesters, J.R., and Hilgenfeld, R. (2008). Variable oligomerization modes in coronavirus non-structural protein 9. J Mol Biol 383, 1081–1096."/>
  <meta name="citation_reference" content="Rothan, H.A., and Byrareddy, S.N. (2020). The epidemiology and pathogenesis of coronavirus disease (COVID-19) outbreak. J Autoimmun, 102433."/>
  <meta name="citation_reference" content="Sutton, G., Fry, E., Carter, L., Sainsbury, S., Walter, T., Nettleship, J., Berrow, N., Owens, R., Gilbert, R., Davidson, A., et al. (2004). The nsp9 replicase protein of SARS-coronavirus, structure and functional insights. Structure 12, 341–353."/>
  <meta name="citation_reference" content="Wrapp, D., Wang, N., Corbett, K.S., Goldsmith, J.A., Hsieh, C.L., Abiona, O., Graham, B.S., and McLellan, J.S. (2020). Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation. Science 367, 1260–1263."/>
  <meta name="citation_reference" content="Yan, G.,Liming, Y., Yucen, H., Fengjiang, L., Yao, Z., Lin, C., Tao, W., Qianqian, S., Zhenhua, M., Lianqi, Z., Ji, G., Litao, Z., Ying, Z., Haofeng, W., Yan, Z., Chen, Z., Tianyu, H., Tian, H., Bing, Z., Xiuna, Y., Jun, L., Haitao, Y., Zhijie, L., Wenqing, X., Luke,W. G.,Quan Wang, Zhiyong, L., Zihe, R. (2020) Structure of RNA-dependent RNA polymerase from 2019-nCoV, a major antiviral drug target, bioRxiv doi: https://doi.org/10.1101/2020.03.16.993386"/>
  <meta name="citation_reference" content="Yang, H., Xie, W., Xue, X., Yang, K., Ma, J., Liang, W., Zhao, Q., Zhou, Z., Pei, D., Ziebuhr, J., et al. (2005). Design of wide-spectrum inhibitors targeting coronavirus main proteases. PLoS Biol 3, e324."/>
  <meta name="citation_reference" content="Zeng, Z., Deng, F., Shi, K., Ye, G., Wang, G., Fang, L., Xiao, S., Fu, Z., and Peng, G. (2018). Dimerization of Coronavirus nsp9 with Diverse Modes Enhances Its Nucleic Acid Binding Affinity. J Virol 92."/>
  <meta name="citation_reference" content="Zhu, L., George, S., Schmidt, M.F., Al-Gharabli, S.I., Rademann, J., and Hilgenfeld, R. (2011). Peptide aldehyde inhibitors challenge the substrate specificity of the SARS-coronavirus main protease. Antiviral Res 92, 204–212."/>
  <meta name="citation_reference" content="Zhu, N., Zhang, D., Wang, W., Li, X., Yang, B., Song, J., Zhao, X., Huang, B., Shi, W., Lu, R., et al. (2020). A Novel Coronavirus from Patients with Pneumonia in China, 2019. N Engl J Med 382, 727–733."/>
  <meta name="twitter:title" content="Crystal structure of the SARS-CoV-2 non-structural protein 9, Nsp9"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Many of the proteins produced by SARS-CoV-2 have related counterparts across the Severe Acute Respiratory Syndrome (SARS-CoV) family. One such protein is non-structural protein 9 (Nsp9), which is thought to mediate both viral replication and virulence. Current understanding suggests that Nsp9 is involved in viral genomic RNA reproduction. Nsp9 is thought to bind RNA via a fold that is unique to this class of betacoronoaviruses although the molecular basis for this remains ill-defined. We sought to better characterise the SARS-CoV-2 Nsp9 protein and subsequently solved its X-ray crystal structure, in an apo-form and, unexpectedly, in a peptide-bound form with a sequence originating from a rhinoviral 3C protease sequence (LEVL). The structure of the SARS-CoV-2 Nsp9 revealed the high level of structural conservation within the Nsp9 family. The exogenous peptide binding site is close to the dimer interface and impacted on the relative juxtaposition of the monomers within the homodimer. Together we have established a protocol for the production of SARS-CoV-2 Nsp9, determined its structure and identified a peptide-binding site that may warrant further study from the perspective of understanding Nsp9 function."/>
  <meta name="og-title" property="og:title" content="Crystal structure of the SARS-CoV-2 non-structural protein 9, Nsp9"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.28.013920v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Many of the proteins produced by SARS-CoV-2 have related counterparts across the Severe Acute Respiratory Syndrome (SARS-CoV) family. One such protein is non-structural protein 9 (Nsp9), which is thought to mediate both viral replication and virulence. Current understanding suggests that Nsp9 is involved in viral genomic RNA reproduction. Nsp9 is thought to bind RNA via a fold that is unique to this class of betacoronoaviruses although the molecular basis for this remains ill-defined. We sought to better characterise the SARS-CoV-2 Nsp9 protein and subsequently solved its X-ray crystal structure, in an apo-form and, unexpectedly, in a peptide-bound form with a sequence originating from a rhinoviral 3C protease sequence (LEVL). The structure of the SARS-CoV-2 Nsp9 revealed the high level of structural conservation within the Nsp9 family. The exogenous peptide binding site is close to the dimer interface and impacted on the relative juxtaposition of the monomers within the homodimer. Together we have established a protocol for the production of SARS-CoV-2 Nsp9, determined its structure and identified a peptide-binding site that may warrant further study from the perspective of understanding Nsp9 function."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-03-30"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Crystal structure of the SARS-CoV-2 non-structural protein 9, Nsp9 | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1218105 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-2">Many of the proteins produced by SARS-CoV-2 have related counterparts across the Severe Acute Respiratory Syndrome (SARS-CoV) family. One such protein is non-structural protein 9 (Nsp9), which is thought to mediate both viral replication and virulence. Current understanding suggests that Nsp9 is involved in viral genomic RNA reproduction. Nsp9 is thought to bind RNA via a fold that is unique to this class of betacoronoaviruses although the molecular basis for this remains ill-defined. We sought to better characterise the SARS-CoV-2 Nsp9 protein and subsequently solved its X-ray crystal structure, in an apo-form and, unexpectedly, in a peptide-bound form with a sequence originating from a rhinoviral 3C protease sequence (LEVL). The structure of the SARS-CoV-2 Nsp9 revealed the high level of structural conservation within the Nsp9 family. The exogenous peptide binding site is close to the dimer interface and impacted on the relative juxtaposition of the monomers within the homodimer. Together we have established a protocol for the production of SARS-CoV-2 Nsp9, determined its structure and identified a peptide-binding site that may warrant further study from the perspective of understanding Nsp9 function.</p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Introduction</h2>
    <p id="p-3">Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is comprised of a large single stranded positive polarity RNA genome that acts as messenger RNA after entering the host. Post infection, the ssRNA encodes two open reading frames, produced through host ribosomal frameshifting that transcribes two polyprotein products. The polyprotein products are subsequently cleaved into 27 viral proteins by internally encoded proteases. Further processing of the polyprotein releases an RNA-polymerase along with several non-structural proteins that facilitate RNA synthesis and may play a role in the enveloping process but are not included in the viral coat.</p>
    <p id="p-4">Treatment of infections caused by betacoronoaviruses have focused on three therapeutic strategies; vaccination with the spike glycoprotein of the SARS-Cov-2 envelope (
     <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Wrapp 
      <em>et al.</em>, 2020
     </a>), and small-molecule targeting of conserved viral enzymes (
     <em>e.g.</em> the Mpro protease (
     <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Zhenming 
      <em>et al.</em> 2020
     </a>) (
     <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Yang 
      <em>et al.</em>, 2005
     </a>) and the RNA-polymerase (
     <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Yan 
      <em>et al.</em> 2020
     </a>). Nevertheless, some of the betacoronaviral non-structural proteins appear important for viral replication within SARS-CoV and influence pathogenesis (
     <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Frieman 
      <em>et al.</em>, 2012
     </a>). Despite their close homology between viruses, such non-structural proteins remain of interest as they may have conserved roles within the viral lifecycle of SARS-CoV-2.
    </p>
    <p id="p-5">During infection of human cells, SARS-CoV Non-structural protein 9 (Nsp9
     <sub>SARS</sub>) was found to be important for replication (
     <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Frieman 
      <em>et al.</em>, 2012
     </a>). Homologs of the Nsp9 protein have been identified in numerous coronaviruses including SARS-Cov-2 (Nsp9
     <sub>COV19</sub>), human coronavirus 229E (Nsp9
     <sub>HcoV</sub>), avian infectious bronchitis virus (Nsp9
     <sub>IBV</sub>), porcine epidemic diarrhea virus (Nsp9
     <sub>PEDV</sub>) and porcine delta virus (Nsp9
     <sub>PDCoV</sub>). Nsp9
     <sub>SARS</sub> has been shown to have modest affinity for long oligonucleotides with binding thought to be dependent on oligomerisation state (
     <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Egloff 
      <em>et al.</em>, 2004
     </a>) (
     <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Sutton 
      <em>et al.</em>, 2004
     </a>). Nsp9
     <sub>SARS</sub> dimerises in solution via a conserved α-helical ‘GxxxG’ motif. Disruption of key residues within this motif reduces both RNA-binding (
     <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Sutton 
      <em>et al.</em>, 2004
     </a>) and SARS-CoV viral proliferation (
     <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Frieman 
      <em>et al.</em>, 2012
     </a>). The mechanism of RNA binding within the Nsp9 protein family is not understood as these proteins have an unusual structural fold not previously seen in RNA binding proteins (
     <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Egloff 
      <em>et al.</em>, 2004
     </a>) (
     <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Sutton 
      <em>et al.</em>, 2004
     </a>). The fold’s Greek-key motif exhibits topological similarities with OB-fold proteins and small protein B but such vestiges have proven insufficient to provide clear insight into Nsp9 function (
     <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Egloff 
      <em>et al.</em>, 2004
     </a>). As a consequence of the weak affinity of Nsp9
     <sub>SARS</sub> for long oligonucleotide stretches it was suggested that the natural RNA substrate may instead be conserved features at the 3’ end of the viral-genome (the stem-loop II RNA-motif) (
     <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Ponnusamy 
      <em>et al.</em>, 2008
     </a>). Furthermore, potential direct interactions with the co-factors of the RNA polymerase have been reported (
     <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chen 
      <em>et al.</em>, 2017
     </a>). However, it remains to be determined how the oligonucleotide-binding activity of Nsp9 proteins promote viral replication during infection.
    </p>
    <p id="p-6">The sequence of Nsp9 homologues are conserved amongst betacoronoaviruses, yet there remains the potential for functional differences in different viruses. Nsp9
     <sub>COV19</sub> exhibits 97% sequence identity with Nsp9
     <sub>SARS</sub> but only 44% sequence identity with Nsp9
     <sub>HCoV.</sub> The structure of the HCoV-229E Nsp9 protein suggested a potential oligomeric switch induced upon the formation of an intersubunit disulfide bond. Here, disulfide bond formation shifts the relative orientation of the Nsp9 monomers, which was suggested to promote higher-order oligomerisation (
     <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Ponnusamy 
      <em>et al.</em>, 2008
     </a>). The resultant rod-like higher order Nsp9
     <sub>HCoV</sub> assemblies had increased affinity for the RNA oligonucleotides. Cysteine mutants of Nsp9
     <sub>HCoV</sub> that are unable to produce the disulfide displayed reduced RNA-binding affinity (
     <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Ponnusamy 
      <em>et al.</em>, 2008
     </a>). The observation of a redox-induced structural switch of Nsp9
     <sub>HCoV</sub> led to the hypothesis that Nsp9
     <sub>HCoV</sub> may have a functional role in sensing the redox status of the host cell (
     <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Ponnusamy 
      <em>et al.</em>, 2008
     </a>). While the “redox-switch” cysteine responsible for oligomer formation in Nsp9
     <sub>HCoV</sub> is conserved amongst different viral Nsp9 homologues the higher-order oligomers were not observed for Nsp9
     <sub>SARS</sub> (
     <a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">Ponnusamy 
      <em>et al.</em>, 2008
     </a>). Because of these potential differences between Nsp9 proteins we sought to further characterise the nature of Nsp9
     <sub>COV19</sub>.
    </p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Results</h2>
    <div id="sec-3" class="subsection">
     <h3>Expression and purification of the SARS-CoV-2 Nsp9 protein</h3>
     <p id="p-7">The Nsp9 protein from SARS-CoV-2 (Nsp9
      <sub>COV19</sub>) was cloned and recombinantly expressed in 
      <em>E. coli</em>. The expression construct included an N-terminal Hexa-His tag attached via a rhinoviral 3C-protease site. Following Ni-affinity chromatography Nsp9
      <sub>COV19</sub> was further purified via size-exclusion chromatography to yield &gt;; 95% pure and homogeneous protein. Nsp9
      <sub>COV19</sub> eluted from gel filtration columns with the apparent molecular weight of a dimer suggesting that, as with other Nsp9 proteins, Nsp9
      <sub>COV19</sub> is an obligate homodimer. The N-terminal tag was removed prior to any biochemical experiments via overnight digestion with precision protease, as reported for Nsp9
      <sub>SARS</sub> (
      <a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Sutton 
       <em>et al.</em>, 2004
      </a>).
     </p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>Nucleotide binding of Nsp9COV19 protein</h3>
     <p id="p-8">The affinity of viral Nsp9 homologues for oligonucleotides has a range of binding affinities reported, some of which are dependent on oligomerisation state and nucleotide length, ranging from 20-400 μM (
      <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Zeng 
       <em>et al.</em>, 2018
      </a>). We therefore sought to assess the potential for Nsp9
      <sub>COV19</sub> to bind to fluorescently labelled oligonucleotides using fluorescence anisotropy. Preliminary experiments were performed under conditions similar to those previously identified for Nsp9
      <sub>SARS</sub> (
      <a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">Sutton 
       <em>et al.</em>, 2004
      </a>). Oligonucleotide affinity was very limited under our assay conditions (
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a>). Indeed, protein concentrations up to 200 μM of Nsp9
      <sub>COV19</sub> did not result in saturated binding and thus indicated an incredibly low affinity 
      <em>K</em>
      <sub>D</sub>, or no affinity for these oligonucleotides at all under these assay conditions.
     </p>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.28.013920/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Nsp9COV19 nucleotide-binding assay. Fluorescence polarization anisotropy assays were used to examine the possibility that Nsp9COV19 could bind to labelled 17-mer and 10-mer single stranded oligonucleotides. The plot shows corrected anisotropy for each Nsp9COV19 protein concentration, error bars represent the SD from the mean of triplicate measurements after 60 minutes incubation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-975893719" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Nsp9&lt;;sub&gt;;COV19&lt;;/sub&gt;; nucleotide-binding assay.&lt;;/span&gt;; Fluorescence polarization anisotropy assays were used to examine the possibility that Nsp9&lt;;sub&gt;;COV19&lt;;/sub&gt;; could bind to labelled 17-mer and 10-mer single stranded oligonucleotides. The plot shows corrected anisotropy for each Nsp9&lt;;sub&gt;;COV19&lt;;/sub&gt;; protein concentration, error bars represent the SD from the mean of triplicate measurements after 60 minutes incubation.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.28.013920/F1.medium.gif" width="440" height="234"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. 1" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.28.013920/F1.medium.gif" width="440" height="234"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.28.013920/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.28.013920/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. 1</span>
       <span class="caption-title">Nsp9
        <sub>COV19</sub> nucleotide-binding assay.
       </span>
       <p id="p-9" class="first-child">Fluorescence polarization anisotropy assays were used to examine the possibility that Nsp9
        <sub>COV19</sub> could bind to labelled 17-mer and 10-mer single stranded oligonucleotides. The plot shows corrected anisotropy for each Nsp9
        <sub>COV19</sub> protein concentration, error bars represent the SD from the mean of triplicate measurements after 60 minutes incubation.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Crystal structure of apo-Nsp9
      <sub>COV19</sub>
     </h3>
     <p id="p-10">We next determined the structure of apo-Nsp9
      <sub>COV19</sub> (
      <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The apo-Nsp9
      <sub>COV19</sub> structure aligned closely to that of Nsp9
      <sub>SARS</sub> (R.M.S.D of 0.57Å over 113 C
      <sub>α</sub>, 
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2A-C</a>) (
      <a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Egloff 
       <em>et al.</em>, 2004
      </a>). Like other Nsp9 homologues it exhibits an unusual fold that is yet to be observed outside of coronaviruses (
      <a id="xref-ref-17-6" class="xref-bibr" href="#ref-17">Sutton 
       <em>et al.</em>, 2004
      </a>). The core of the fold is a small 6-stranded enclosed β-barrel, from which a series of extended loops project outward (
      <a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig 2A</a>). The elongated loops link the individual β-strands of the barrel, along with a projecting N-terminal β-strand and C-terminal α1-helix; the latter two elements make up the main components of the dimer interface (
      <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3A</a>). Two loops project from the open-face of the barrel: the β2-3- and β3-4-loops are both positively charged, glycine rich, and are proposed to be involved in RNA-binding. The only protrusion on the enclosed barrel-side is the β6-7-loop; the C-terminal half of the β7-strand is an integral part of the fold’s barrel-core but its other half extended outward to pair with the external β6-strand and create a twisted β-hairpin, cupping the α1-helix and interacting with subsequent C-terminal residues.
     </p>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1223822/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1223822/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1223822" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 1</span>
       <p id="p-11" class="first-child">Data collection and refinement statistics</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.28.013920/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="apo-Nsp9COV19 is structurally similar to Nsp9SARS. Cartoon representation of the monomeric units of a) apo-Nsp9COV19 b) apo-Nsp9SARS (Sutton et al., 2004) and c) a backbone alignment of the two structures. The COV19 structures are colored with [[#x3B2]]-strands in marine and the [[#x3B1]]-helix in wheat; the SARS structures are in teal and orange respectively. d) the bound peptide is highlighted in red." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-975893719" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;apo-Nsp9&lt;;sub&gt;;COV19&lt;;/sub&gt;; is structurally similar to Nsp9&lt;;sub&gt;;SARS&lt;;/sub&gt;;.&lt;;/span&gt;; Cartoon representation of the monomeric units of &lt;;strong&gt;;a)&lt;;/strong&gt;; apo-Nsp9&lt;;sub&gt;;COV19&lt;;/sub&gt;; &lt;;strong&gt;;b)&lt;;/strong&gt;; apo-Nsp9&lt;;sub&gt;;SARS&lt;;/sub&gt;; (Sutton &lt;;em&gt;;et al.&lt;;/em&gt;;, 2004) and &lt;;strong&gt;;c)&lt;;/strong&gt;; a backbone alignment of the two structures. The COV19 structures are colored with [[#x3B2]]-strands in &lt;;em&gt;;marine&lt;;/em&gt;; and the [[#x3B1]]-helix in &lt;;em&gt;;wheat;&lt;;/em&gt;; the SARS structures are in &lt;;em&gt;;teal&lt;;/em&gt;; and &lt;;em&gt;;orange&lt;;/em&gt;; respectively. &lt;;strong&gt;;d)&lt;;/strong&gt;; the bound peptide is highlighted in &lt;;em&gt;;red&lt;;/em&gt;;.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.28.013920/F2.medium.gif" width="401" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. 2" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.28.013920/F2.medium.gif" width="401" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.28.013920/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.28.013920/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. 2</span>
       <span class="caption-title">apo-Nsp9
        <sub>COV19</sub> is structurally similar to Nsp9
        <sub>SARS</sub>.
       </span>
       <p id="p-12" class="first-child">Cartoon representation of the monomeric units of 
        <strong>a)</strong> apo-Nsp9
        <sub>COV19</sub>
        <strong>b)</strong> apo-Nsp9
        <sub>SARS</sub> (
        <a id="xref-ref-17-7" class="xref-bibr" href="#ref-17">Sutton 
         <em>et al.</em>, 2004
        </a>) and 
        <strong>c)</strong> a backbone alignment of the two structures. The COV19 structures are colored with β-strands in 
        <em>marine</em> and the α-helix in 
        <em>wheat;</em> the SARS structures are in 
        <em>teal</em> and 
        <em>orange</em> respectively. 
        <strong>d)</strong> the bound peptide is highlighted in 
        <em>red</em>.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.28.013920/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Peptide binding in Nsp9COV19 alters the dimer interface. Top-down views of the dimer interface highlighting the interaction helices for a) unbound Nsp9COV19 in which the surface of the hydrophobic interface cavity is displayed labelled b) an equivalent representation of peptide-occupied 3c-Nsp9COV19 dimer. c) Stick representation of the GxxxG protein-protein interaction helices at the dimer interface for apo-Nsp9COV19. d) C[[#x3B1]] backbone overlay of the Nsp9COV19 interface in the apo and peptide-occupied states. The GxxxG motif residues are colored light purple." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-975893719" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Peptide binding in Nsp9&lt;;sub&gt;;COV19&lt;;/sub&gt;; alters the dimer interface.&lt;;/span&gt;; Top-down views of the dimer interface highlighting the interaction helices for &lt;;strong&gt;;a)&lt;;/strong&gt;; unbound Nsp9&lt;;sub&gt;;COV19&lt;;/sub&gt;; in which the surface of the hydrophobic interface cavity is displayed labelled &lt;;strong&gt;;b)&lt;;/strong&gt;; an equivalent representation of peptide-occupied 3c-Nsp9&lt;;sub&gt;;COV19&lt;;/sub&gt;; dimer. &lt;;strong&gt;;c)&lt;;/strong&gt;; Stick representation of the GxxxG protein-protein interaction helices at the dimer interface for apo-Nsp9&lt;;sub&gt;;COV19&lt;;/sub&gt;;. &lt;;strong&gt;;d)&lt;;/strong&gt;; C[[#x3B1]] backbone overlay of the Nsp9&lt;;sub&gt;;COV19&lt;;/sub&gt;; interface in the apo and peptide-occupied states. The GxxxG motif residues are colored &lt;;em&gt;;light purple&lt;;/em&gt;;.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.28.013920/F3.medium.gif" width="440" height="347"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. 3" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.28.013920/F3.medium.gif" width="440" height="347"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.28.013920/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.28.013920/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. 3</span>
       <span class="caption-title">Peptide binding in Nsp9
        <sub>COV19</sub> alters the dimer interface.
       </span>
       <p id="p-13" class="first-child">Top-down views of the dimer interface highlighting the interaction helices for 
        <strong>a)</strong> unbound Nsp9
        <sub>COV19</sub> in which the surface of the hydrophobic interface cavity is displayed labelled 
        <strong>b)</strong> an equivalent representation of peptide-occupied 3c-Nsp9
        <sub>COV19</sub> dimer. 
        <strong>c)</strong> Stick representation of the GxxxG protein-protein interaction helices at the dimer interface for apo-Nsp9
        <sub>COV19</sub>. 
        <strong>d)</strong> Cα backbone overlay of the Nsp9
        <sub>COV19</sub> interface in the apo and peptide-occupied states. The GxxxG motif residues are colored 
        <em>light purple</em>.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-14">The arrangement of monomers within Nsp9-dimers is well-conserved in different viruses and is maintained within Nsp9
      <sub>COV19</sub> (R.M.S.D of 0.66Å over 226C
      <sub>α</sub> compared to the dimeric unit of Nsp9
      <sub>SARS</sub>). The main component of the intersubunit interaction is the self-association of the conserved GxxxG protein-protein binding motif (
      <a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3C</a>) that allowed backbone van der Waals interactions between interfacing copies of the C-terminal α1-helix (
      <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Hu 
       <em>et al.</em> 2017
      </a>). Here Gly-100 of the respective parallel α1-helices, formed complementary backbone van der Waals interactions. These interactions were replicated after a full helical turn by Gly-104 of the respective chains, thereby forming the molecular basis of the Nsp9
      <sub>COVID19</sub> dimer interface (
      <a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3C</a>). The 2-fold axis that created the dimer ran at a ~15 ° angle through the GxxxG motif allowing the 14-residue helix to cross its counterpart (
      <a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4A</a>), the N-terminal turns of the helix were relatively isolated, only making contacts with counterpart protomer residues. In contrast, the C-terminal portions were encircled by hydrophobic residues, albeit at a distance that created funnel-like hydrophobic cavities either side of the interfacing helices (
      <a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3A</a>). Strands β1, β6 and the protein’s C-terminus served to provide a ring of residues that encircled the paired helices. The first 10 residues of Nsp9
      <sub>COV19</sub> exchanged across the dimer-interface to form a strand-like extension of β1 that ran alongside β6 from the other protomer (
      <a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig. 3A</a>). The interaction these strands made did not appear optimal, indeed the remaining four C-terminal residues projected sideways across the dimer interface, inserting between the two strands while contributing a hydrophobic backing to the main helix.
     </p>
     <div id="F4" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.28.013920/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Movements within the GxxxG motif. A side-view of the N- and C-terminal structural elements at the dimer interface are shown for the a) apo-and b) peptide-occupied forms. c) Overlay of the Nsp9COV19 dimer in the apo and peptide-bound forms indicating respective shifts in subunit orientation. The center-of-mass of the nonaligned subunit is depicted with a light-pink and dark-pink point respectively. d) Unbiased omit map contoured at 3.2[[#x3C3]] near the hydrophobic cavities into which the exogenous bound peptide was refined." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-975893719" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Movements within the GxxxG motif.&lt;;/span&gt;; A side-view of the N- and C-terminal structural elements at the dimer interface are shown for the &lt;;strong&gt;;a)&lt;;/strong&gt;; apo-and &lt;;strong&gt;;b)&lt;;/strong&gt;; peptide-occupied forms. &lt;;strong&gt;;c)&lt;;/strong&gt;; Overlay of the Nsp9&lt;;sub&gt;;COV19&lt;;/sub&gt;; dimer in the apo and peptide-bound forms indicating respective shifts in subunit orientation. The center-of-mass of the nonaligned subunit is depicted with a &lt;;em&gt;;light-pink&lt;;/em&gt;; and &lt;;em&gt;;dark-pink&lt;;/em&gt;; point respectively. &lt;;strong&gt;;d)&lt;;/strong&gt;; Unbiased omit map contoured at 3.2[[#x3C3]] near the hydrophobic cavities into which the exogenous bound peptide was refined.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.28.013920/F4.medium.gif" width="340" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. 4" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.28.013920/F4.medium.gif" width="340" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.28.013920/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 4" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.28.013920/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. 4</span>
       <span class="caption-title">Movements within the GxxxG motif.</span>
       <p id="p-15" class="first-child">A side-view of the N- and C-terminal structural elements at the dimer interface are shown for the 
        <strong>a)</strong> apo-and 
        <strong>b)</strong> peptide-occupied forms. 
        <strong>c)</strong> Overlay of the Nsp9
        <sub>COV19</sub> dimer in the apo and peptide-bound forms indicating respective shifts in subunit orientation. The center-of-mass of the nonaligned subunit is depicted with a 
        <em>light-pink</em> and 
        <em>dark-pink</em> point respectively. 
        <strong>d)</strong> Unbiased omit map contoured at 3.2σ near the hydrophobic cavities into which the exogenous bound peptide was refined.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-6" class="subsection">
     <h3>Extraneous peptides occupy the hydrophobic cavities of Nsp9</h3>
     <p id="p-16">In a separate crystallisation experiment we determined the structure of Nsp9
      <sub>COV19</sub> that included the N-terminal tag together with a rhinoviral 3C protease sequence (termed 3C-Nsp9
      <sub>COV19</sub>). The 3C-Nsp9
      <sub>COV19</sub> crystal form diffracted to 2.05 Å resolution in space group P4
      <sub>3</sub>22, and had 1 molecule within the asymmetric unit, with the dimer being created across the crystallographic 2-fold axis.
     </p>
     <p id="p-17">Unexpectedly the high-resolution structure of 3C-Nsp9
      <sub>COV19</sub> diverged from that of the apo-Nsp9
      <sub>COV19</sub> (R.M.S.D 0.86 Å for the monomer and 2.23 Å when superimposing a dimer). The 3C sequence folded-around either side of the paired intersubunit helices to fill two funnel-like hydrophobic cavities (
      <a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2D</a>, 
      <a id="xref-fig-3-6" class="xref-fig" href="#F3">3B</a>, 
      <a id="xref-fig-4-2" class="xref-fig" href="#F4">4C, D</a>). Namely, 3C residues LEVL, inserted into the opposing cavities either side of the dimer interface and ran parallel to the paired GxxxG motif. Moreover, the 3C sequence formed additional β-sheet interactions with the N-terminus of the protein from the other protomer (
      <a id="xref-fig-3-7" class="xref-fig" href="#F3">Fig. 3B</a>). To accommodate the 3C residues the N-terminal strand residues moved outward by ~1.6 Å (residues 6-10). This movement allowed the N-terminus to increase the number of β-sheet interactions it formed with β6’. The β-barrel core of the fold remained unchanged but the increase in interactions between β1 and β6’ served to exclude the C-terminus, prompting residues 106-111 to condense into a bent extension of the α-helix (
      <a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4A, B</a>). The subtle structural changes near the interacting GxxxG motifs (
      <a id="xref-fig-3-8" class="xref-fig" href="#F3">Fig. 3D</a>) are amplified at the periphery of the dimer resulting in ~6 Å shift in the β-barrel core (
      <a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig. 4C</a>)
     </p>
    </div>
    <div id="sec-7" class="subsection">
     <h3>Conserved cavity residues accommodate a peptide backbone</h3>
     <p id="p-18">When comparing apo-Nsp9
      <sub>COV19</sub> with 3C-Nsp9
      <sub>COV19</sub> the point where the N-terminal interface strand diverges is near Leu-9 (
      <a id="xref-fig-3-9" class="xref-fig" href="#F3">Fig. 3D</a>). Within the apo form it makes van der Waals interactions with the sidechains of Met-101, Asn-33 and Ser-105; this latter serine is important as it immediately follows the conserved protein-binding motif (
      <sup>100</sup>
      <strong>G</strong> MVL
      <strong>G</strong> S
      <sup>105</sup>), while also specifically interacting with Gly104’ from the opposing protomer. Within the 3C-Nsp9
      <sub>COV19</sub> structure the extraneous LEVL residues insert at this point (
      <a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5B</a>), the hydrophobic side chains clasp either side of Ser-105 and allowed its hydroxyl group to form backbone hydrogen bonds to the glutamate within the extraneous sequence (
      <a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5B</a>). Meanwhile the C-terminal Leucine from the extraneous residues inserted behind the α-helix of the other protomer (
      <a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig. 5B</a>). Cumulatively these changes allow for a ~5° rotation of the protomer subunits about the 2-fold axis compared to apo-Nsp9
      <sub>COV19</sub> (
      <a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig. 4C</a>).
     </p>
     <div id="F5" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.28.013920/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Sequence conservation within Nsp9 homologues. Sequence alignment for viral Nsp9 proteins encoded by SARS-CoV-2, SARS-CoV, Human coronavirus 229E and Porcine epidemic diarrhea virus. The extent of secondary structural elements observed in the 3C-Nsp9COV19 structure are shown and labelled above. The GxxxG motif residues are highlighted with purple and those making up the extraneous peptide binding site in pink. b) Cartoon and stick representation of the peptide-binding site observed in 3C-Nsp9COV19. The sidechains of the extraneous 3C residues on one side of the paired helical-interface are displayed with carbon atoms colored pink. Nearby residues that make up the binding site are displayed and labelled and listed in the accompanying contacts table." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-975893719" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Sequence conservation within Nsp9 homologues.&lt;;/span&gt;; Sequence alignment for viral Nsp9 proteins encoded by SARS-CoV-2, SARS-CoV, Human coronavirus 229E and Porcine epidemic diarrhea virus. The extent of secondary structural elements observed in the 3C-Nsp9&lt;;sub&gt;;COV19&lt;;/sub&gt;; structure are shown and labelled above. The GxxxG motif residues are highlighted with &lt;;em&gt;;purple&lt;;/em&gt;; and those making up the extraneous peptide binding site in &lt;;em&gt;;pink&lt;;/em&gt;;. &lt;;strong&gt;;b)&lt;;/strong&gt;; Cartoon and stick representation of the peptide-binding site observed in 3C-Nsp9&lt;;sub&gt;;COV19&lt;;/sub&gt;;. The sidechains of the extraneous 3C residues on one side of the paired helical-interface are displayed with carbon atoms colored &lt;;em&gt;;pink&lt;;/em&gt;;. Nearby residues that make up the binding site are displayed and labelled and listed in the accompanying contacts table.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.28.013920/F5.medium.gif" width="419" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. 5" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.28.013920/F5.medium.gif" width="419" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.28.013920/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 5" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.28.013920/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. 5</span>
       <span class="caption-title">Sequence conservation within Nsp9 homologues.</span>
       <p id="p-19" class="first-child">Sequence alignment for viral Nsp9 proteins encoded by SARS-CoV-2, SARS-CoV, Human coronavirus 229E and Porcine epidemic diarrhea virus. The extent of secondary structural elements observed in the 3C-Nsp9
        <sub>COV19</sub> structure are shown and labelled above. The GxxxG motif residues are highlighted with 
        <em>purple</em> and those making up the extraneous peptide binding site in 
        <em>pink</em>. 
        <strong>b)</strong> Cartoon and stick representation of the peptide-binding site observed in 3C-Nsp9
        <sub>COV19</sub>. The sidechains of the extraneous 3C residues on one side of the paired helical-interface are displayed with carbon atoms colored 
        <em>pink</em>. Nearby residues that make up the binding site are displayed and labelled and listed in the accompanying contacts table.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-20">Most residues involved in protein-binding within the hydrophobic cavity and the structural changes needed to accommodate them appear broadly conserved amongst other Nsp9 viral homologues (
      <em>red</em> highlights in 
      <a id="xref-fig-5-4" class="xref-fig" href="#F5">Fig. 5A</a>). The main exception to this is Ser-105, which is a Tyrosine in the distantly related Nsp9
      <sub>HCoV</sub> and Nsp9
      <sub>PEDV</sub> proteins (
      <a id="xref-ref-15-6" class="xref-bibr" href="#ref-15">Ponnusamy et al., 2008</a>), (
      <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Zeng 
       <em>et al.</em>, 2018
      </a>). However, the N-terminal interface β-strand in these homologues is known to be involved in interface re-organisation of the subunits (
      <a id="xref-ref-15-7" class="xref-bibr" href="#ref-15">Ponnusamy et al., 2008</a>) and thus denotes other structural differences at this site.
     </p>
    </div>
   </div>
   <div class="section" id="sec-8">
    <h2 class="">Discussion</h2>
    <p id="p-21">Here we describe the structure of the recombinantly expressed Nsp9
     <sub>COV19</sub> as part of a global effort to characterise the virus causing a current global pandemic. Nsp9 is important for virulence in SARS-CoV (
     <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Miknis et al., 2009</a>). It remains to be understood whether Nsp9
     <sub>COV19</sub> plays a similar role in SARS-CoV-2, however the 97% sequence identity suggests a high degree of functional conservation. The CoV Nsp9 proteins are seemingly obligate dimers comprising a unique fold that associates via an unusual α-helical GxxxG interaction motif. The integrity of this motif is considered important for viral replication (
     <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Miknis et al., 2009</a>), leading to a proposal that disruption of the unusual dimer interface impacts on RNA binding and function (
     <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Hu et al., 2017</a>). Mutation of the same interaction motif in the porcine delta coronavirus Nsp9
     <sub>PDCoV</sub> also disrupted nucleotide binding capacity (
     <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Zeng 
      <em>et al.</em>, 2018
     </a>).
    </p>
    <p id="p-22">We describe the ability to produce homogenous Nsp9
     <sub>COV19</sub> which purifies as an obligate dimer, consistent with other Nsp9 proteins. Our preliminary nucleotide binding assays brought into question the RNA binding capacity of Nsp9
     <sub>COV19</sub>. The structure of the Nsp9
     <sub>COV19</sub> showed conservation of the unique Nsp9 fold when compared with homologues from SARS (
     <a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">Egloff 
      <em>et al.</em>, 2004
     </a>) (
     <a id="xref-ref-17-8" class="xref-bibr" href="#ref-17">Sutton 
      <em>et al.</em>, 2004
     </a>). Indeed, the topological fold was conserved as was the Nsp9 specific α-helical GxxxG dimerisation interface. This α-helical interface is encircled by hydrophobic residues but the interface includes considerable cavities as observed previously (
     <a id="xref-ref-6-6" class="xref-bibr" href="#ref-6">Egloff 
      <em>et al.</em>, 2004
     </a>). We made a serendipitous discovery in our 3C-Nsp9
     <sub>COV19</sub> structure, whereby the hydrophobic cavity captured the 3C cleavage sequence LEVL. The extraneous residues were tightly bound on all sides within the site situating themselves proximal to the conserved GxxxG motif. Coordination of the 3C sequence induced changes within interfacing residues, serving to both restructure key structural elements and cause a modest shift in subunit orientation.
    </p>
    <p id="p-23">At this stage it is unclear whether the bound residues within our structure have any bearing on the physiological function of Nsp9
     <sub>COV19</sub>. In the first instance this would seem unlikely, however our sequence is that of a rhinoviral 3C protease site and the SARS-CoV main protease cleaves consensus sequences following an LQ sequence (
     <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Zhu 
      <em>et al.</em>, 2011
     </a>). The bound 3C-LE residues have hallmarks of the LQ motif and are proximal to the highly conserved GxxxG motif. There are no obvious structural features to preclude, or select for, a Glu to Gln substitution within our bound sequence. Notably the M
     <sup>pro</sup> cleavage sequence occurs at multiple points throughout the CoV genome as the majority of viral proteins are released by its activity, thus it remains possible that Nsp9 may associate with unprocessed viral polyproteins retaining them near the viral RNA. Within our structure Met-101 provides contacts with the bound valine sidechain but the presence of Asn-33 nearby may also accommodate residues such as lysine at this position. Peptide-binding assays will need to be developed to rigorously assess such possibilities.
    </p>
    <p id="p-24">In summary we have established a protocol for the production and purification of SARS-CoV-2 Nsp9 protein. We determined the structure of the Nsp9
     <sub>COV19</sub> and described the conservation of the unique fold and dimerisation interface identified previously for members of this protein family. We also determined structure of Nsp9
     <sub>COV19</sub> in complex with a 3C sequence, although the significance of this is yet to be established. The structures we describe here could potentially be utilised in drug screening and targeting experiments to disrupt a dimer interface known to be important for coronavirus replication.
    </p>
   </div>
   <div class="section" id="sec-9">
    <h2 class="">Materials and Methods</h2>
    <div id="sec-10" class="subsection">
     <h3>Protein production, crystallisation, structure determination, and refinement</h3>
     <p id="p-25">Synthetic cDNA for the Nsp9
      <sub>COV19</sub> protein was cloned into pET28-LIC expression vector bearing an N-terminal His-tag with a Rhinovirus 3C protease cleavage site (MAHHHHHHSAA
      <em>
       <strong>
        <em>LEVL</em>
       </strong>
      </em>FQGPG). The plasmid was transformed into 
      <em>E. coli</em> BL21(DE3) cells which were grown in Luria Broth at 37°C until reaching an Absorbance at 600nm of ~1.0 before being induced with 0.5mM Isopropyl β-d-1-thiogalactopyranoside for 4 hours. Cells were harvested in 20mM HEPES pH 7.0, 150mM NaCl, 20mM Imidazole, 2mM MgCl
      <sub>2</sub> and 0.5mM TCEP and frozen until required. Lysis was achieved by sonicating the cells in the presence of 1mg of Lysozyme and 1mg of DNAase on ice. The lysate was then cleared by centrifugation at 10,000xg for 20 minutes and loaded onto a nickel affinity column. Bound protein was washed extensively with 20 column volumes of 20mM HEPES pH 7.0, 150mM NaCl, 0.5mM TCEP before being eluted in the same buffer with the addition of 400mM Imidazole. For His-tag removal samples were incubated with precision 3c protease overnight at 4°C. All samples were subjected to gel filtration (S75 16/60; GE Healthcare) in 20mM HEPES pH 7.0, 150mM NaCl before being concentrated to 50mg/mL for crystallization trials.
     </p>
     <p id="p-26">Nsp9
      <sub>COV19</sub> crystallized in 2.0-2.2M NH
      <sub>4</sub>SO
      <sub>4</sub> and 0.1M Phosphate-citrate buffer pH 4.0. Crystals of the His-tag samples grew with rectangular morphology in space group P4
      <sub>1</sub>22, however if the His-tag was removed the crystals grew in space group P6
      <sub>1</sub>22 form with hexagonal morphology. All diffraction data were collected at the Australian synchrotron’s MX2 beamline at the Australian synchrotron (
      <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aragao 
       <em>et al.</em>, 2018
      </a>) (see 
      <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> for details). Data were integrated in XDS (
      <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Kabsch, 2010</a>), processed with SCALA, phases were obtained through molecular replacement using PDB 1QZ8 (
      <a id="xref-ref-6-7" class="xref-bibr" href="#ref-6">Egloff 
       <em>et al.</em>, 2004
      </a>) as a search model. Subsequent rounds of manual building and refinement were performed in Coot (
      <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Casanal 
       <em>et al.</em>, 2019
      </a>) and PHENIX (
      <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Liebschner 
       <em>et al.</em>, 2019
      </a>).
     </p>
    </div>
    <div id="sec-11" class="subsection">
     <h3>Nucleotide binding assay</h3>
     <p id="p-27">To examine the RNA-binding affinity, an 18-point serial dilution (212-0 μM) of Nsp9
      <sub>COV19</sub> was incubated with 1 nM 5’-Fluorescein labelled 17mer poly-U single-stranded RNA (Dharmacon GE, USA) or 10mer PolyT single-stranded DNA (IDT, USA) in assay buffer (20mM HEPES pH 7.0, 150mM NaCl, 2mM MgCl
      <sub>2</sub>) at room temperature.
     </p>
     <p id="p-28">The assay was performed in 96-well non-binding black plates (Greiner Bio-One), with fluorescence anisotropy measured in triplicate using the PHERAstar FS (BMG) with FP 488-520-520 nm filters. The data was corrected using the anisotropy of RNA sample alone, then fitted by a one-site binding model using the Equation, A = (Amax [L])/(KD+[L]), where A is the corrected fluorescence anisotropy; Amax is maximum binding fluorescence anisotropy signal, [L] is the Nsp9
      <sub>COV19</sub> concentration, and 
      <em>K</em>
      <sub>
       <em>D</em>
      </sub> is the dissociation equilibrium constant. Amax and 
      <em>K</em>
      <sub>
       <em>D</em>
      </sub> were used as fitting parameters and nonlinear regression was performed using GraphPad Prism. Measurements were taken after 60 minutes incubation between protein and RNA.
     </p>
    </div>
   </div>
   <div class="section" id="sec-12">
    <h2 class="">Supplementary note</h2>
    <p id="p-29">During manuscript preparation, the Center for Structural Genomics of Infectious Diseases (CSGID) deposited the structure of Nsp9
     <sub>COV19</sub> (6W4B) to a resolution of 2.95Å
    </p>
   </div>
   <div class="section" id="sec-13">
    <h2 class="">End Matter</h2>
    <div class="section" id="sec-14">
     <h3>Author Contributions and Notes</h3>
     <p id="p-30">D.R.L, B.S.G, and J.R. designed the project and wrote the manuscript. D.R.L. cloned, purified and crystallised Nsp9
      <sub>Cov19</sub> and refined the structures. R.C.N. performed the RNA binding assays. Diffraction data and models have been deposited at the protein databank (
      <a href="http://www.rcsb.org">www.rcsb.org</a>) with accession codes 6W9Q and 6WC1.
     </p>
    </div>
    <div id="sec-15" class="subsection COI-statement">
     <p id="p-31">The authors declare no conflict of interest.</p>
     <p id="p-32">This article contains supporting information online.</p>
    </div>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgments</h2>
    <p id="p-33">Funding for the work originated from the Australian Research Council Centre of Excellence for Advanced Molecular Imaging. This research was undertaken in part using the MX2 beamline at the Australian Synchrotron, part of ANSTO, and made use of the Australian Cancer Research Foundation (ACRF) detector. We thank A. Riboldi-Tunnicliffe for assistance with data collection and J. Whisstock and G. Watson for advice on the manuscript.</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.28.013920v1.1" data-doi="10.1107/S1600577518003120">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Aragao</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Aishima</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cherukuvada</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Clarken</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Clift</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cowieson</span>, 
          <span class="cit-name-given-names">N.P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ericsson</span>, 
          <span class="cit-name-given-names">D.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gee</span>, 
          <span class="cit-name-given-names">C.L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Macedo</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mudie</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2018</span>). 
         <span class="cit-article-title">MX2: a high-flux undulator microfocus beamline serving both the chemical and macromolecular crystallography communities at the Australian Synchrotron</span>. 
         <abbr class="cit-jnl-abbrev">J Synchrotron Radiat</abbr>
         <span class="cit-vol">25</span>, 
         <span class="cit-fpage">885</span>–
         <span class="cit-lpage">891</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BSynchrotron%2BRadiat%26rft.volume%253D25%26rft.spage%253D885%26rft_id%253Dinfo%253Adoi%252F10.1107%252FS1600577518003120%26rft_id%253Dinfo%253Apmid%252F29714201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1107/S1600577518003120&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=29714201&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.28.013920.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.28.013920v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Casanal</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lohkamp</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Emsley</span>, 
          <span class="cit-name-given-names">P.</span>
         </span> (
         <span class="cit-pub-date">2019</span>). 
         <span class="cit-article-title">Current developments in Coot for macromolecular model building of Electron Cryo-microscopy and Crystallographic Data</span>. 
         <abbr class="cit-jnl-abbrev">Protein Sci.</abbr>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.28.013920v1.3" data-doi="10.1107/S1600577518003120">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Aragao</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Aishima</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cherukuvada</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Clarken</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Clift</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cowieson</span>, 
          <span class="cit-name-given-names">N.P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ericsson</span>, 
          <span class="cit-name-given-names">D.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gee</span>, 
          <span class="cit-name-given-names">C.L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Macedo</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mudie</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2018</span>). 
         <span class="cit-article-title">MX2: a high-flux undulator microfocus beamline serving both the chemical and macromolecular crystallography communities at the Australian Synchrotron</span>. 
         <abbr class="cit-jnl-abbrev">J Synchrotron Radiat</abbr>
         <span class="cit-vol">25</span>, 
         <span class="cit-fpage">885</span>–
         <span class="cit-lpage">891</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BSynchrotron%2BRadiat%26rft.volume%253D25%26rft.spage%253D885%26rft_id%253Dinfo%253Adoi%252F10.1107%252FS1600577518003120%26rft_id%253Dinfo%253Apmid%252F29714201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1107/S1600577518003120&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=29714201&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.28.013920.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.28.013920v1.4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Casanal</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lohkamp</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Emsley</span>, 
          <span class="cit-name-given-names">P.</span>
         </span> (
         <span class="cit-pub-date">2019</span>). 
         <span class="cit-article-title">Current developments in Coot for macromolecular model building of Electron Cryo-microscopy and Crystallographic Data</span>. 
         <abbr class="cit-jnl-abbrev">Protein Sci.</abbr>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.28.013920v1.5" data-doi="10.3389/fmicb.2017.00853">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tao</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nan</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tian</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">H.</span>
         </span> (
         <span class="cit-pub-date">2017</span>). 
         <span class="cit-article-title">Structural Analysis of Porcine Reproductive and Respiratory Syndrome Virus Non-structural Protein 7alpha (NSP7alpha) and Identification of Its Interaction with NSP9</span>. 
         <abbr class="cit-jnl-abbrev">Front Microbiol</abbr>
         <span class="cit-vol">8</span>, 
         <span class="cit-fpage">853</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DFront%2BMicrobiol%26rft.volume%253D8%26rft.spage%253D853%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffmicb.2017.00853%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.3389/fmicb.2017.00853&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.28.013920v1.6" data-doi="10.1073/pnas.0307877101">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Egloff</span>, 
          <span class="cit-name-given-names">M.P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ferron</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Campanacci</span>, 
          <span class="cit-name-given-names">V.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Longhi</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rancurel</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dutartre</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Snijder</span>, 
          <span class="cit-name-given-names">E.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gorbalenya</span>, 
          <span class="cit-name-given-names">A.E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cambillau</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Canard</span>, 
          <span class="cit-name-given-names">B.</span>
         </span> (
         <span class="cit-pub-date">2004</span>). 
         <span class="cit-article-title">The severe acute respiratory syndrome-coronavirus replicative protein nsp9 is a single-stranded RNA-binding subunit unique in the RNA virus world</span>. 
         <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>
         <span class="cit-vol">101</span>, 
         <span class="cit-fpage">3792</span>–
         <span class="cit-lpage">3796</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0307877101%26rft_id%253Dinfo%253Apmid%252F15007178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzExLzM3OTIiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8zMC8yMDIwLjAzLjI4LjAxMzkyMC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.28.013920v1.7" data-doi="10.1128/JVI.05957-11">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Frieman</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yount</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Agnihothram</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Page</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Donaldson</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Roberts</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vogel</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Woodruff</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Scorpio</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Subbarao</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2012</span>). 
         <span class="cit-article-title">Molecular determinants of severe acute respiratory syndrome coronavirus pathogenesis and virulence in young and aged mouse models of human disease</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>
         <span class="cit-vol">86</span>, 
         <span class="cit-fpage">884</span>–
         <span class="cit-lpage">897</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.05957-11%26rft_id%253Dinfo%253Apmid%252F22072787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6Ijg2LzIvODg0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMzAvMjAyMC4wMy4yOC4wMTM5MjAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.28.013920v1.8">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Goh</span>, 
          <span class="cit-name-given-names">K.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Choong</span>, 
          <span class="cit-name-given-names">M.C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cheong</span>, 
          <span class="cit-name-given-names">E.H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kalimuddin</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Duu Wen</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Phua</span>, 
          <span class="cit-name-given-names">G.C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chan</span>, 
          <span class="cit-name-given-names">K.S.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Haja Mohideen</span>, 
          <span class="cit-name-given-names">S.</span>
         </span> (
         <span class="cit-pub-date">2020</span>). 
         <span class="cit-article-title">Rapid Progression to Acute Respiratory Distress Syndrome: Review of Current Understanding of Critical Illness from COVID-19 Infection</span>. 
         <abbr class="cit-jnl-abbrev">Ann Acad Med Singapore</abbr>
         <span class="cit-vol">49</span>, 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">9</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAnn%2BAcad%2BMed%2BSingapore%26rft.volume%253D49%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.28.013920v1.9" data-doi="10.1002/pro.3150">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dou</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lu</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zong</span>, 
          <span class="cit-name-given-names">Q.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhong</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2017</span>). 
         <span class="cit-article-title">Structural basis for dimerization and RNA binding of avian infectious bronchitis virus nsp9</span>. 
         <abbr class="cit-jnl-abbrev">Protein Sci</abbr>
         <span class="cit-vol">26</span>, 
         <span class="cit-fpage">1037</span>–
         <span class="cit-lpage">1048</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProtein%2BSci%26rft.volume%253D26%26rft.spage%253D1037%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpro.3150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1002/pro.3150&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.28.013920v1.10" data-doi="10.1016/S0140-6736(20)30183-5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ren</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fan</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gu</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2020</span>). 
         <span class="cit-article-title">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</span>. 
         <abbr class="cit-jnl-abbrev">Lancet</abbr>
         <span class="cit-vol">395</span>, 
         <span class="cit-fpage">497</span>–
         <span class="cit-lpage">506</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D395%26rft.spage%253D497%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252820%252930183-5%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0140-6736(20)30183-5&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=http://www.n&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.28.013920.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.28.013920v1.11" data-doi="10.1107/S0907444909047337">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kabsch</span>, 
          <span class="cit-name-given-names">W.</span>
         </span> (
         <span class="cit-pub-date">2010</span>). 
         <span class="cit-article-title">Xds</span>. 
         <abbr class="cit-jnl-abbrev">Acta Crystallogr D Biol Crystallogr</abbr>
         <span class="cit-vol">66</span>, 
         <span class="cit-fpage">125</span>–
         <span class="cit-lpage">132</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DActa%2BCrystallographica%252C%2BSection%2BD%253A%2BBiological%2BCrystallography%26rft.stitle%253DActa%2BCrystallographica%252C%2BSection%2BD%253A%2BBiological%2BCrystallography%26rft.aulast%253DKabsch%26rft.auinit1%253DW.%26rft.volume%253D66%26rft.issue%253DPt%2B2%26rft.spage%253D125%26rft.epage%253D132%26rft.atitle%253DXDS.%26rft_id%253Dinfo%253Adoi%252F10.1107%252FS0907444909047337%26rft_id%253Dinfo%253Apmid%252F20124692%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1107/S0907444909047337&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20124692&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.28.013920.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000273820800003&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.28.013920v1.12" data-doi="10.1107/S2059798319011471">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Liebschner</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Afonine</span>, 
          <span class="cit-name-given-names">P.V.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Baker</span>, 
          <span class="cit-name-given-names">M.L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bunkoczi</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">V.B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Croll</span>, 
          <span class="cit-name-given-names">T.I.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hintze</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hung</span>, 
          <span class="cit-name-given-names">L.W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jain</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">McCoy</span>, 
          <span class="cit-name-given-names">A.J.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2019</span>). 
         <span class="cit-article-title">Macromolecular structure determination using X-rays, neutrons and electrons: recent developments in Phenix</span>. 
         <abbr class="cit-jnl-abbrev">Acta Crystallogr D Struct Biol</abbr>
         <span class="cit-vol">75</span>, 
         <span class="cit-fpage">861</span>–
         <span class="cit-lpage">877</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DActa%2BCrystallogr%2BD%2BStruct%2BBiol%26rft.volume%253D75%26rft.spage%253D861%26rft_id%253Dinfo%253Adoi%252F10.1107%252FS2059798319011471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1107/S2059798319011471&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.28.013920v1.13">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhenming</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xiaoyu</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yechun</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yongqiang</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Meiqin</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yao</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bing</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xiaofeng</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Leike</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chao</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yinkai</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jing</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lin</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kailin</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fengjiang</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rendi</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xinglou</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tian</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xiaoce</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xiuna</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fang</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hong</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xiang</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Luke</span>, 
          <span class="cit-name-given-names">W. G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wenqing</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gengfu</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chengfeng</span>, 
          <span class="cit-name-given-names">Q.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhengli</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hualiang</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zihe</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Haitao</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span> (
         <span class="cit-pub-date">2020</span>) 
         <span class="cit-article-title">Structure of Mpro 1 from COVID-19 virus and discovery of its inhibitors</span>
         <abbr class="cit-jnl-abbrev">bioRxiv</abbr> doi: 
         <a href="https://doi.org/10.1101/2020.02.26.964882">https://doi.org/10.1101/2020.02.26.964882</a>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.28.013920v1.14" data-doi="10.1128/JVI.01505-08">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Miknis</span>, 
          <span class="cit-name-given-names">Z.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Donaldson</span>, 
          <span class="cit-name-given-names">E.F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Umland</span>, 
          <span class="cit-name-given-names">T.C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rimmer</span>, 
          <span class="cit-name-given-names">R.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Baric</span>, 
          <span class="cit-name-given-names">R.S.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Schultz</span>, 
          <span class="cit-name-given-names">L.W.</span>
         </span> (
         <span class="cit-pub-date">2009</span>). 
         <span class="cit-article-title">Severe acute respiratory syndrome coronavirus nsp9 dimerization is essential for efficient viral growth</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>
         <span class="cit-vol">83</span>, 
         <span class="cit-fpage">3007</span>–
         <span class="cit-lpage">3018</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01505-08%26rft_id%253Dinfo%253Apmid%252F19153232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgzLzcvMzAwNyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzMwLzIwMjAuMDMuMjguMDEzOTIwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.28.013920v1.15" data-doi="10.1016/j.jmb.2008.07.071">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ponnusamy</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Moll</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Weimar</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mesters</span>, 
          <span class="cit-name-given-names">J.R.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Hilgenfeld</span>, 
          <span class="cit-name-given-names">R.</span>
         </span> (
         <span class="cit-pub-date">2008</span>). 
         <span class="cit-article-title">Variable oligomerization modes in coronavirus non-structural protein 9</span>. 
         <abbr class="cit-jnl-abbrev">J Mol Biol</abbr>
         <span class="cit-vol">383</span>, 
         <span class="cit-fpage">1081</span>–
         <span class="cit-lpage">1096</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DPonnusamy%26rft.auinit1%253DR.%26rft.volume%253D383%26rft.issue%253D5%26rft.spage%253D1081%26rft.epage%253D1096%26rft.atitle%253DVariable%2Boligomerization%2Bmodes%2Bin%2Bcoronavirus%2Bnon-structural%2Bprotein%2B9.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2008.07.071%26rft_id%253Dinfo%253Apmid%252F18694760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.jmb.2008.07.071&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18694760&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.28.013920.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.28.013920v1.16">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rothan</span>, 
          <span class="cit-name-given-names">H.A.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Byrareddy</span>, 
          <span class="cit-name-given-names">S.N.</span>
         </span> (
         <span class="cit-pub-date">2020</span>). 
         <span class="cit-article-title">The epidemiology and pathogenesis of coronavirus disease (COVID-19) outbreak</span>. 
         <abbr class="cit-jnl-abbrev">J Autoimmun</abbr>, 
         <span class="cit-fpage">102433</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.28.013920v1.17" data-doi="10.1016/j.str.2004.01.016">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Sutton</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fry</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Carter</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sainsbury</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Walter</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nettleship</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Berrow</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Owens</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gilbert</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Davidson</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2004</span>). 
         <span class="cit-article-title">The nsp9 replicase protein of SARS-coronavirus, structure and functional insights</span>. 
         <abbr class="cit-jnl-abbrev">Structure</abbr>
         <span class="cit-vol">12</span>, 
         <span class="cit-fpage">341</span>–
         <span class="cit-lpage">353</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DStructure%2B%2528London%2529%26rft.stitle%253DStructure%2B%2528London%2529%26rft.aulast%253DSutton%26rft.auinit1%253DG.%26rft.volume%253D12%26rft.issue%253D2%26rft.spage%253D341%26rft.epage%253D353%26rft.atitle%253DThe%2Bnsp9%2Breplicase%2Bprotein%2Bof%2BSARS-coronavirus%252C%2Bstructure%2Band%2Bfunctional%2Binsights.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.str.2004.01.016%26rft_id%253Dinfo%253Apmid%252F14962394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.str.2004.01.016&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=14962394&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.28.013920.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.28.013920v1.18" data-doi="10.1126/science.abb2507">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wrapp</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Corbett</span>, 
          <span class="cit-name-given-names">K.S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Goldsmith</span>, 
          <span class="cit-name-given-names">J.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hsieh</span>, 
          <span class="cit-name-given-names">C.L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Abiona</span>, 
          <span class="cit-name-given-names">O.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Graham</span>, 
          <span class="cit-name-given-names">B.S.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">McLellan</span>, 
          <span class="cit-name-given-names">J.S.</span>
         </span> (
         <span class="cit-pub-date">2020</span>). 
         <span class="cit-article-title">Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>
         <span class="cit-vol">367</span>, 
         <span class="cit-fpage">1260</span>–
         <span class="cit-lpage">1263</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.abb2507%26rft_id%253Dinfo%253Apmid%252F32075877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNjcvNjQ4My8xMjYwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMzAvMjAyMC4wMy4yOC4wMTM5MjAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.28.013920v1.19">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yan</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>,
         <span class="cit-auth">
          <span class="cit-name-surname">Liming</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yucen</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fengjiang</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yao</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lin</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tao</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Qianqian</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhenhua</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lianqi</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ji</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Litao</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ying</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Haofeng</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yan</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tianyu</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tian</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bing</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xiuna</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jun</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Haitao</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhijie</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wenqing</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Luke</span>, 
          <span class="cit-name-given-names">W. G.</span>
         </span>,
         <span class="cit-auth">
          <span class="cit-name-surname">Quan</span>
          <span class="cit-name-given-names">Wang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhiyong</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zihe</span>, 
          <span class="cit-name-given-names">R.</span>
         </span> (
         <span class="cit-pub-date">2020</span>) 
         <span class="cit-article-title">Structure of RNA-dependent RNA polymerase from 2019-nCoV, a major antiviral drug target</span>, 
         <abbr class="cit-jnl-abbrev">bioRxiv</abbr> doi: 
         <a href="https://doi.org/10.1101/2020.03.16.993386">https://doi.org/10.1101/2020.03.16.993386</a>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.28.013920v1.20" data-doi="10.1371/journal.pbio.0030324">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xie</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xue</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ma</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Liang</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>, 
          <span class="cit-name-given-names">Q.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pei</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ziebuhr</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2005</span>). 
         <span class="cit-article-title">Design of wide-spectrum inhibitors targeting coronavirus main proteases</span>. 
         <abbr class="cit-jnl-abbrev">PLoS Biol</abbr>
         <span class="cit-vol">3</span>, 
         <span class="cit-fpage">e324</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DYang%26rft.auinit1%253DH.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De324%26rft.epage%253De324%26rft.atitle%253DDesign%2Bof%2Bwide-spectrum%2Binhibitors%2Btargeting%2Bcoronavirus%2Bmain%2Bproteases.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0030324%26rft_id%253Dinfo%253Apmid%252F16128623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0030324&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16128623&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.28.013920.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.28.013920v1.21">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zeng</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Deng</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shi</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ye</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fang</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xiao</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fu</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Peng</span>, 
          <span class="cit-name-given-names">G.</span>
         </span> (
         <span class="cit-pub-date">2018</span>). 
         <span class="cit-article-title">Dimerization of Coronavirus nsp9 with Diverse Modes Enhances Its Nucleic Acid Binding Affinity</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>
         <span class="cit-vol">92</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.28.013920v1.22" data-doi="10.1016/j.antiviral.2011.08.001">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">George</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schmidt</span>, 
          <span class="cit-name-given-names">M.F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Al-Gharabli</span>, 
          <span class="cit-name-given-names">S.I.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rademann</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Hilgenfeld</span>, 
          <span class="cit-name-given-names">R.</span>
         </span> (
         <span class="cit-pub-date">2011</span>). 
         <span class="cit-article-title">Peptide aldehyde inhibitors challenge the substrate specificity of the SARS-coronavirus main protease</span>. 
         <abbr class="cit-jnl-abbrev">Antiviral Res</abbr>
         <span class="cit-vol">92</span>, 
         <span class="cit-fpage">204</span>–
         <span class="cit-lpage">212</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAntiviral%2Bresearch%26rft.stitle%253DAntiviral%2BRes%26rft.aulast%253DZhu%26rft.auinit1%253DL.%26rft.volume%253D92%26rft.issue%253D2%26rft.spage%253D204%26rft.epage%253D212%26rft.atitle%253DPeptide%2Baldehyde%2Binhibitors%2Bchallenge%2Bthe%2Bsubstrate%2Bspecificity%2Bof%2Bthe%2BSARS-coronavirus%2Bmain%2Bprotease.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.antiviral.2011.08.001%26rft_id%253Dinfo%253Apmid%252F21854807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.antiviral.2011.08.001&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21854807&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.28.013920.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000297427200008&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.28.013920v1.23" data-doi="10.1056/NEJMoa2001017">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Song</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shi</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lu</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2020</span>). 
         <span class="cit-article-title">A Novel Coronavirus from Patients with Pneumonia in China, 2019</span>. 
         <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>
         <span class="cit-vol">382</span>, 
         <span class="cit-fpage">727</span>–
         <span class="cit-lpage">733</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft.volume%253D382%26rft.spage%253D727%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa2001017%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1056/NEJMoa2001017&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=http://www.n&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.28.013920.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
