<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2016/07/01/061788.atom" />
<meta name="HW.pisa" content="biorxiv;061788v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Dysregulation of Long Non-coding RNA (lncRNA) Genes and Predicted lncRNA-Protein Interactions during Zika Virus Infection" />
<meta name="DC.Identifier" content="10.1101/061788" />
<meta name="DC.Date" content="2016-07-01" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2016, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Zika Virus (ZIKV) is a causative agent for poor pregnancy outcome and fetal developmental abnormalities, including microcephaly and eye defects. As a result, ZIKV is now a confirmed teratogen. Understanding host-pathogen interactions, specifically cellular perturbations caused by ZIKV, can provide novel therapeutic targets. In order to complete viral replication, viral pathogens control the host cellular machineries and regulate various factors, including long noncoding RNA (lncRNA) genes, at transcriptional levels. The role of lncRNA genes in the pathogenesis of ZIKV-mediated microcephaly and eye defects is currently unknown. To gain additional insights, we focused on profiling the differentially expressed lncRNA genes during ZIKV infection in mammalian cells. For this study, we employed a contemporary clinical Zika viral isolate, PRVABC59, of Asian genotype. We utilized an unbiased RNA sequencing approach to profile the lncRNA transcriptome in ZIKV infected Vero cells. We identified a total of 121 lncRNA genes that are differentially regulated at 48 hours post-infection. The majority of these genes are independently validated by reverse-transcription qPCR. A notable observation was that the lncRNAs, MALAT1 (Metastasis Associated Lung Adenocarcinoma Transcript 1) and NEAT1 (Nuclear Paraspeckle Assembly Transcript 1), are down-regulated upon Zika viral infection. MALAT1 and NEAT1 are known as nuclear localized RNAs that regulate gene expression and cell proliferation. Protein-lncRNA interaction maps revealed that MALAT1 and NEAT1 share common interacting partners and form a larger network comprising of 71 cellular factors. ZIKV-mediated dysregulation of these two regulatory lncRNAs can alter the expression of respective target genes and associated biological functions, an important one being cell division. In conclusion, this investigation is the first to provide insight into the biological connection of lncRNAs and ZIKV which can be further explored for developing antiviral therapy and understanding fetal developmental processes." />
<meta name="DC.Contributor" content="Arunachalam Ramaiah" />
<meta name="DC.Contributor" content="Deisy Contreras" />
<meta name="DC.Contributor" content="Vineela Gangalapudi" />
<meta name="DC.Contributor" content="Masumi Sameer Padhye" />
<meta name="DC.Contributor" content="Jie Tang" />
<meta name="DC.Contributor" content="Vaithilingaraja Arumugaswami" />
<meta name="article:published_time" content="2016-07-01" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Dysregulation of Long Non-coding RNA (lncRNA) Genes and Predicted lncRNA-Protein Interactions during Zika Virus Infection" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;ABSTRACT&lt;/h3&gt;
&lt;p&gt;Zika Virus (ZIKV) is a causative agent for poor pregnancy outcome and fetal developmental abnormalities, including microcephaly and eye defects. As a result, ZIKV is now a confirmed teratogen. Understanding host-pathogen interactions, specifically cellular perturbations caused by ZIKV, can provide novel therapeutic targets. In order to complete viral replication, viral pathogens control the host cellular machineries and regulate various factors, including long noncoding RNA (lncRNA) genes, at transcriptional levels. The role of lncRNA genes in the pathogenesis of ZIKV-mediated microcephaly and eye defects is currently unknown. To gain additional insights, we focused on profiling the differentially expressed lncRNA genes during ZIKV infection in mammalian cells. For this study, we employed a contemporary clinical Zika viral isolate, PRVABC59, of Asian genotype. We utilized an unbiased RNA sequencing approach to profile the lncRNA transcriptome in ZIKV infected Vero cells. We identified a total of 121 lncRNA genes that are differentially regulated at 48 hours post-infection. The majority of these genes are independently validated by reverse-transcription qPCR. A notable observation was that the lncRNAs, MALAT1 (Metastasis Associated Lung Adenocarcinoma Transcript 1) and NEAT1 (Nuclear Paraspeckle Assembly Transcript 1), are down-regulated upon Zika viral infection. MALAT1 and NEAT1 are known as nuclear localized RNAs that regulate gene expression and cell proliferation. Protein-lncRNA interaction maps revealed that MALAT1 and NEAT1 share common interacting partners and form a larger network comprising of 71 cellular factors. ZIKV-mediated dysregulation of these two regulatory lncRNAs can alter the expression of respective target genes and associated biological functions, an important one being cell division. In conclusion, this investigation is the first to provide insight into the biological connection of lncRNAs and ZIKV which can be further explored for developing antiviral therapy and understanding fetal developmental processes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2016/01/01" />
<meta name="citation_mjid" content="biorxiv;061788v1" />
<meta name="citation_id" content="061788v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/061788v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/061788v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/061788v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788.full.pdf" />
<meta name="citation_doi" content="10.1101/061788" />
<meta name="citation_num_pages" content="24" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="061788" />
<meta name="citation_author" content="Arunachalam Ramaiah" />
<meta name="citation_author_institution" content="Centre for Infectious Disease Research, Indian Institute of Science" />
<meta name="citation_author" content="Deisy Contreras" />
<meta name="citation_author_institution" content="Board of Governors Regenerative Medicine Institute, Cedars-Sinai Medical Center" />
<meta name="citation_author" content="Vineela Gangalapudi" />
<meta name="citation_author_institution" content="Cedars-Sinai Genomics Core, Medical Genetics Institute, Cedars-Sinai Medical Center Los Angeles" />
<meta name="citation_author" content="Masumi Sameer Padhye" />
<meta name="citation_author_institution" content="Board of Governors Regenerative Medicine Institute, Cedars-Sinai Medical Center" />
<meta name="citation_author" content="Jie Tang" />
<meta name="citation_author_institution" content="Cedars-Sinai Genomics Core, Medical Genetics Institute, Cedars-Sinai Medical Center Los Angeles" />
<meta name="citation_author" content="Vaithilingaraja Arumugaswami" />
<meta name="citation_author_institution" content="Board of Governors Regenerative Medicine Institute, Cedars-Sinai Medical Center" />
<meta name="citation_author_institution" content="Department of Surgery, Cedars-Sinai Medical Center" />
<meta name="citation_author_institution" content="Department of Surgery, University of California at Los Angeles" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6872-5118" />
<meta name="citation_reference" content="Brasil, P., et al., Zika Virus Infection in Pregnant Women in Rio de Janeiro – Preliminary Report. N Engl J Med, 2016." />
<meta name="citation_reference" content="Schuler-Faccini, L., et al., Possible Association Between Zika Virus Infection and Microcephaly – Brazil, 2015. MMWR Morb Mortal Wkly Rep, 2016. 65(3): p. 59–62." />
<meta name="citation_reference" content="Mlakar, J., et al., Zika Virus Associated with Microcephaly. N Engl J Med, 2016." />
<meta name="citation_reference" content="Dick, G.W., Zika virus. II. Pathogenicity and physical properties. Trans R Soc Trop Med Hyg, 1952. 46(5): p. 521–34." />
<meta name="citation_reference" content="Dick, G.W., S.F. Kitchen, and A.J. Haddow, Zika virus. I. Isolations and serological specificity. Trans R Soc Trop Med Hyg, 1952. 46(5): p. 509–20." />
<meta name="citation_reference" content="Hamel, R., et al., Biology of Zika Virus Infection in Human Skin Cells. J Virol, 2015. 89(17): p. 8880–96." />
<meta name="citation_reference" content="Lanciotti, R.S., et al., Genetic and serologic properties of Zika virus associated with an epidemic, Yap State, Micronesia, 2007. Emerg Infect Dis, 2008. 14(8): p. 1232–9." />
<meta name="citation_reference" content="Djebali, S., et al., Landscape of transcription in human cells. Nature, 2012. 489(7414): p. 101–8." />
<meta name="citation_reference" content="Huarte, M., et al., A large intergenic noncoding RNA induced by p53 mediates global gene repression in the p53 response. Cell, 2010. 142(3): p. 409–19." />
<meta name="citation_reference" content="Kung, J.T., D. Colognori, and J.T. Lee, Long noncoding RNAs: past, present, and future. Genetics, 2013. 193(3): p. 651–69." />
<meta name="citation_reference" content="Guttman, M., et al., Ab initio reconstruction of cell type-specific transcriptomes in mouse reveals the conserved multi-exonic structure of lincRNAs. Nat Biotechnol, 2010. 28(5): p. 503–10." />
<meta name="citation_reference" content="Cabili, M.N., et al., Integrative annotation of human large intergenic noncoding RNAs reveals global properties and specific subclasses. Genes Dev, 2011. 25(18): p. 1915–27." />
<meta name="citation_reference" content="Consortium, E.P., et al., Identification and analysis of functional elements in 1% of the human genome by the ENCODE pilot project. Nature, 2007. 447(7146): p. 799–816." />
<meta name="citation_reference" content="Dinger, M.E., et al., Long noncoding RNAs in mouse embryonic stem cell pluripotency and differentiation. Genome Res, 2008. 18(9): p. 1433–45." />
<meta name="citation_reference" content="Guttman, M., et al., Chromatin signature reveals over a thousand highly conserved large non-coding RNAs in mammals. Nature, 2009. 458(7235): p. 223–7." />
<meta name="citation_reference" content="Loewer, S., et al., Large intergenic non-coding RNA-RoR modulates reprogramming of human induced pluripotent stem cells. Nat Genet, 2010. 42(12): p. 1113–7." />
<meta name="citation_reference" content="Hung, T., et al., Extensive and coordinated transcription of noncoding RNAs within cell-cycle promoters. Nat Genet, 2011. 43(7): p. 621–9." />
<meta name="citation_reference" content="Rinn, J.L. and H.Y. Chang, Genome regulation by long noncoding RNAs. Annu Rev Biochem, 2012. 81: p. 145–66." />
<meta name="citation_reference" content="Bawa, P., et al., Integrative Analysis of Normal Long Intergenic Non-Coding RNAs in Prostate Cancer. PloS One, 2015. 10(5): p. e0122143." />
<meta name="citation_reference" content="Lanciotti RS, L.A., Holodniy M, Saavedra S, del Carmen Castillo Signor L, Phylogeny of Zika virus in Western Hemisphere, 2015 [letter]. 2016." />
<meta name="citation_reference" content="Contreras, D. and V. Arumugaswami, Zika Virus Infectious Cell Culture System and the In Vitro Prophylactic Effect of Interferons. Journal of Visualized Experiments, 2016(e54767)." />
<meta name="citation_reference" content="Langmead, B., et al., Ultrafast and memory-efficient alignment of short DNA sequences to the human genome. Genome Biol, 2009. 10(3): p. R25." />
<meta name="citation_reference" content="Love, M.I., W. Huber, and S. Anders, Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol, 2014. 15(12): p. 550." />
<meta name="citation_reference" content="Benjamini, Y. and Y. Hochberg, Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing. Journal of the Royal Statistical Society. Series B (Methodological), 1995. 57(1): p. 289–300." />
<meta name="citation_reference" content="Szklarczyk, D., et al., STRING v10: protein-protein interaction networks, integrated over the tree of life. Nucleic Acids Res, 2015. 43(Database issue): p. D447–52." />
<meta name="citation_reference" content="Rhodes, D.R., et al., Probabilistic model of the human protein-protein interaction network. Nat Biotechnol, 2005. 23(8): p. 951–9." />
<meta name="citation_reference" content="Mittal, N., et al., Dissecting the expression dynamics of RNA-binding proteins in posttranscriptional regulatory networks. Proc Natl Acad Sci U S A, 2009. 106(48): p. 20300–5." />
<meta name="citation_reference" content="Kishore, S., S. Luber, and M. Zavolan, Deciphering the role of RNA-binding proteins in the post-transcriptional control of gene expression. Brief Funct Genomics, 2010. 9(5-6): p. 391–404." />
<meta name="citation_reference" content="Hafner, M., et al., Transcriptome-wide identification of RNA-binding protein and microRNA target sites by PAR-CLIP. Cell, 2010. 141(1): p. 129–41." />
<meta name="citation_reference" content="Wang, P., et al., Long noncoding RNA NEAT1 promotes laryngeal squamous cell cancer through regulating miR-107/CDK6 pathway. J Exp Clin Cancer Res, 2016. 35: p. 22." />
<meta name="citation_reference" content="Tripathi, V., et al., Long noncoding RNA MALAT1 controls cell cycle progression by regulating the expression of oncogenic transcription factor B-MYB. PLoS Genet, 2013. 9(3): p. e1003368." />
<meta name="citation_reference" content="Amaral, P.P., et al., lncRNAdb: a reference database for long noncoding RNAs. Nucleic Acids Res, 2011. 39(Database issue): p. D146–51." />
<meta name="citation_reference" content="Bernard, D., et al., A long nuclear-retained non-coding RNA regulates synaptogenesis by modulating gene expression. EMBO J, 2010. 29(18): p. 3082–93." />
<meta name="citation_reference" content="Hutchinson, J.N., et al., A screen for nuclear transcripts identifies two linked noncoding RNAs associated with SC35 splicing domains. BMC Genomics, 2007. 8: p. 39." />
<meta name="citation_reference" content="Hirose, T., et al., NEAT1 long noncoding RNA regulates transcription via protein sequestration within subnuclear bodies. Mol Biol Cell, 2014. 25(1): p. 169–83." />
<meta name="citation_reference" content="Naganuma, T. and T. Hirose, Paraspeckle formation during the biogenesis of long noncoding RNAs. RNA Biol, 2013. 10(3): p. 456–61." />
<meta name="citation_reference" content="Zeng, C., et al., Inhibition of long non-coding RNA NEAT1 impairs myeloid differentiation in acute promyelocytic leukemia cells. BMC Cancer, 2014. 14: p. 693." />
<meta name="citation_reference" content="Guo, S., et al., Clinical implication of long non-coding RNA NEAT1 expression in hepatocellular carcinoma patients. Int J Clin Exp Pathol, 2015. 8(5): p. 5395–402." />
<meta name="citation_reference" content="Gibb, E.A., et al., Human cancer long non-coding RNA transcriptomes. PloS One, 2011. 6(10): p. e25915." />
<meta name="citation_reference" content="Standaert, L., et al., The long noncoding RNA Neat1 is required for mammary gland development and lactation. RNA, 2014. 20(12): p. 1844–9." />
<meta name="citation_reference" content="Tang, H., et al., Zika Virus Infects Human Cortical Neural Progenitors and Attenuates Their Growth. Cell Stem Cell, 2016." />
<meta name="citation_reference" content="Lees, J.G., et al., Systematic computational prediction of protein interaction networks. Phys Biol, 2011. 8(3): p. 035008." />
<meta name="citation_reference" content="Wang, T.Y., et al., A predicted protein-protein interaction network of the filamentous fungus Neurospora crassa. Mol Biosyst, 2011. 7(7): p. 2278–85." />
<meta name="citation_reference" content="Baroni, T.E., et al., Advances in RIP-chip analysis: RNA-binding protein immunoprecipitation-microarray profiling. Methods Mol Biol, 2008. 419: p. 93–108." />
<meta name="citation_reference" content="Barkan, A., Genome-wide analysis of RNA-protein interactions in plants. Methods Mol Biol, 2009. 553: p. 13–37." />
<meta name="citation_reference" content="Kim, M.Y., J. Hur, and S. Jeong, Emerging roles of RNA and RNA-binding protein network in cancer cells. BMB Rep, 2009. 42(3): p. 125–30." />
<meta name="citation_reference" content="Hogan, D.J., et al., Diverse RNA-binding proteins interact with functionally related sets of RNAs, suggesting an extensive regulatory system. PLoS Biol, 2008. 6(10): p. e255." />
<meta name="citation_reference" content="Licatalosi, D.D. and R.B. Darnell, RNA processing and its regulation: global insights into biological networks. Nat Rev Genet, 2010. 11(1): p. 75–87." />
<meta name="citation_reference" content="Sola, I., et al., RNA-RNA and RNA-protein interactions in coronavirus replication and transcription. RNA Biol, 2011. 8(2): p. 237–48." />
<meta name="citation_reference" content="Li, Z. and P.D. Nagy, Diverse roles of host RNA binding proteins in RNA virus replication. RNA Biol, 2011. 8(2): p. 305–15." />
<meta name="twitter:title" content="Dysregulation of Long Non-coding RNA (lncRNA) Genes and Predicted lncRNA-Protein Interactions during Zika Virus Infection" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Zika Virus (ZIKV) is a causative agent for poor pregnancy outcome and fetal developmental abnormalities, including microcephaly and eye defects. As a result, ZIKV is now a confirmed teratogen. Understanding host-pathogen interactions, specifically cellular perturbations caused by ZIKV, can provide novel therapeutic targets. In order to complete viral replication, viral pathogens control the host cellular machineries and regulate various factors, including long noncoding RNA (lncRNA) genes, at transcriptional levels. The role of lncRNA genes in the pathogenesis of ZIKV-mediated microcephaly and eye defects is currently unknown. To gain additional insights, we focused on profiling the differentially expressed lncRNA genes during ZIKV infection in mammalian cells. For this study, we employed a contemporary clinical Zika viral isolate, PRVABC59, of Asian genotype. We utilized an unbiased RNA sequencing approach to profile the lncRNA transcriptome in ZIKV infected Vero cells. We identified a total of 121 lncRNA genes that are differentially regulated at 48 hours post-infection. The majority of these genes are independently validated by reverse-transcription qPCR. A notable observation was that the lncRNAs, MALAT1 (Metastasis Associated Lung Adenocarcinoma Transcript 1) and NEAT1 (Nuclear Paraspeckle Assembly Transcript 1), are down-regulated upon Zika viral infection. MALAT1 and NEAT1 are known as nuclear localized RNAs that regulate gene expression and cell proliferation. Protein-lncRNA interaction maps revealed that MALAT1 and NEAT1 share common interacting partners and form a larger network comprising of 71 cellular factors. ZIKV-mediated dysregulation of these two regulatory lncRNAs can alter the expression of respective target genes and associated biological functions, an important one being cell division. In conclusion, this investigation is the first to provide insight into the biological connection of lncRNAs and ZIKV which can be further explored for developing antiviral therapy and understanding fetal developmental processes." />
<meta name="og-title" property="og:title" content="Dysregulation of Long Non-coding RNA (lncRNA) Genes and Predicted lncRNA-Protein Interactions during Zika Virus Infection" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/061788v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Zika Virus (ZIKV) is a causative agent for poor pregnancy outcome and fetal developmental abnormalities, including microcephaly and eye defects. As a result, ZIKV is now a confirmed teratogen. Understanding host-pathogen interactions, specifically cellular perturbations caused by ZIKV, can provide novel therapeutic targets. In order to complete viral replication, viral pathogens control the host cellular machineries and regulate various factors, including long noncoding RNA (lncRNA) genes, at transcriptional levels. The role of lncRNA genes in the pathogenesis of ZIKV-mediated microcephaly and eye defects is currently unknown. To gain additional insights, we focused on profiling the differentially expressed lncRNA genes during ZIKV infection in mammalian cells. For this study, we employed a contemporary clinical Zika viral isolate, PRVABC59, of Asian genotype. We utilized an unbiased RNA sequencing approach to profile the lncRNA transcriptome in ZIKV infected Vero cells. We identified a total of 121 lncRNA genes that are differentially regulated at 48 hours post-infection. The majority of these genes are independently validated by reverse-transcription qPCR. A notable observation was that the lncRNAs, MALAT1 (Metastasis Associated Lung Adenocarcinoma Transcript 1) and NEAT1 (Nuclear Paraspeckle Assembly Transcript 1), are down-regulated upon Zika viral infection. MALAT1 and NEAT1 are known as nuclear localized RNAs that regulate gene expression and cell proliferation. Protein-lncRNA interaction maps revealed that MALAT1 and NEAT1 share common interacting partners and form a larger network comprising of 71 cellular factors. ZIKV-mediated dysregulation of these two regulatory lncRNAs can alter the expression of respective target genes and associated biological functions, an important one being cell division. In conclusion, this investigation is the first to provide insight into the biological connection of lncRNAs and ZIKV which can be further explored for developing antiviral therapy and understanding fetal developmental processes." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2016-07-01" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/061788v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/061788v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/061788v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/061788v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/16978" />
    <title>Dysregulation of Long Non-coding RNA (lncRNA) Genes and Predicted lncRNA-Protein Interactions during Zika Virus Infection | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__vup9YXfxWOBOGbthJCUtxdPS6xRY2OQ_vXqbojfxPAk__ziC21uj-cnpEf3LG0kjRMEQHSEtjX5ljVuosklz2DZ4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Yd5cArTrHVOLZ4Xft-LNdZ_nKxvg9pd6sb1M8XIKRIo","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"16978","apath":"\/biorxiv\/early\/2016\/07\/01\/061788.atom","pisa":"biorxiv;061788v1","markup":[{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;061788\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;061788\/DC1"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;061788v1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;061788\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;061788\/DC1"}],"processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/061788v1","title":"Dysregulation of Long Non-coding RNA (lncRNA) Genes and Predicted lncRNA-Protein Interactions during Zika Virus Infection","identifier":"node\/16978"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022dCqxP0aZpy6QN21Diq4PLX5irfnV4web41dd8UZaEZgf6x1fzP2rFMXuuvzqilArMGB8hAurzbKDap6Yvdc+vcMWJoe4bw5doUF1X5dQKQ4kfw2qc7KGODkJ69AIsL\\\\\\\/4h84L4nNyypswzdZG9mXFNVi2bOobO6OZECsTZCZ\\\\\\\/jA0Yh5s8vn4DZ5Dl7GMMRnITdUqakIXHKVyDNAj83SApEkukiWCblcaW+U6f26wSIGxLjsQb7sU+mcTC5lE7IAnu5C2N9XLz9bEvr0Hhs8jFPps2EP72\\\\\\\/2LPqztoYOa\\\\\\\/MhOudtYCpXcifvFL5SCezqF8N4iTH4llqjGIW8GAbR8H9t9OY0zy\\\\\\\/+LfGquxYVOvn1K0x1Vw78yfzozLVIjQs19QV4GwxGwnwQ7aBCa4G\\\\\\\/0pwtZl07VBur8dUy1z\\\\\\\/BUGmOa4tAQEp5S5cYm2j9KWLajpPImwevBSLRSzz1BJoMPqxCYhAZeO2EFZ3cmw4zmDK\\\\\\\/FCk4eZvVbXd4\\\\\\\/3ZjTqPUErVojXun7tChXgB2E4r85yoVIHSAQJ6nGXBNMlUKC166YO9mgcaGyQxbfhtDelEh8jxamUBU6TC7+2MRhdTzyeitKmstk5OB4ri1PlnigYLd3m5ZxoQ8GJhJUK9lYbbn9LqsUu\\\\\\\/gK3nF6dTCuN3avLHYAN766bfw2hESMrRvyDXx1gZbEF8MLqylqUbSZyMerkerO54rnkojGrzJV8vrQM6vUO8fEFBDcTgs+QmO5Br7E4j0DBn405dnKmG8BrsyUddglu2FrkmSJe1idWV6puQLlLWT8eh\\\\\\\/pgNkEuG8kzK49egQG0675E1oMZTMVsIY+EHQ7204Jl4ZwmsaSgju2z1jD3gGUznbR\\\\\\\/hJDv8EQ1Id7KpCwm8YXwkcbM1V4IioMMjWJ15ZXd4sObWw+jpeViTA0Pdl+cKaESDvXaKXAGzC3C1A\\\\\\\/BjK1kPsVst7vRcagu\\\\\\\/0waTfI+rK4bQ9wU7PImaqOjzPKUh70H6UNkn2qOn+0Mw8pEtzuG9ZEGz82cbcAersvH3WVT3yyqEA7SUoi5a0mlpfoZF+tv3DlTi4Lz8KNkJY5DqFGippm+RAgXoIgldhu2NXspXKpSsZv2Ly37AXjXYVxF3UJfgGp4Zr255U+wd+H8+Uuyxn53\\\\\\\/\\\\\\\/aIjHHC22FLixuq8yB6FWWswOXW45hY1zyqPzJYBM2yuSHYEumQelJOfrJQ0Ie6Ba52YHBLUGyKImn20t\\\\\\\/s8mizPNBeJSv00BVzz7rTbDsCw21Ln0fRfvsqYAc7z4L\\\\\\\/7bSjjnLpTxFLR002KAsS\\\\\\\/g9Z4l292eJJLvUsJmgmV+lzRThzv5c=\\\u0022,\\\u0022iv\\\u0022:\\\u0022Lo+JIyxkK7WFVOFC\\\\\\\/GGzbA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002249bc34dd81cabf57e7696ab6f3c7205a7c8c1146e41d36ae6f84b4c8a49daecb\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/061788v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-16978 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/061788v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-e3XgT5kjFnusqe927K8hcXKgJkpqWYaS-NyQbCpBjfU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="16978" id="node16978" data-pisa="biorxiv;061788v1" data-pisa-master="biorxiv;061788" data-apath="/biorxiv/early/2016/07/01/061788.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Dysregulation of Long Non-coding RNA (lncRNA) Genes and Predicted lncRNA-Protein Interactions during Zika Virus Infection</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Arunachalam</span> <span class="nlm-surname">Ramaiah</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Deisy</span> <span class="nlm-surname">Contreras</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Vineela</span> <span class="nlm-surname">Gangalapudi</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Masumi</span> <span class="nlm-surname">Sameer Padhye</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Jie</span> <span class="nlm-surname">Tang</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-6872-5118" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Vaithilingaraja</span> <span class="nlm-surname">Arumugaswami</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/061788 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node16978" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Arunachalam Ramaiah </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>a</span><span class='nlm-institution'>Centre for Infectious Disease Research, Indian Institute of Science</span>, Bangalore, Karnataka 560012, <span class='nlm-country'>India</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Arunachalam%2BRamaiah%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ramaiah%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AArunachalam%2BRamaiah%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Deisy Contreras </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>b</span><span class='nlm-institution'>Board of Governors Regenerative Medicine Institute, Cedars-Sinai Medical Center</span>, Los Angeles, CA 90048, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Deisy%2BContreras%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Contreras%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADeisy%2BContreras%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Vineela Gangalapudi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>c</span><span class='nlm-institution'>Cedars-Sinai Genomics Core, Medical Genetics Institute, Cedars-Sinai Medical Center Los Angeles</span>, CA 90048, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vineela%2BGangalapudi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gangalapudi%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVineela%2BGangalapudi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Masumi Sameer Padhye </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>b</span><span class='nlm-institution'>Board of Governors Regenerative Medicine Institute, Cedars-Sinai Medical Center</span>, Los Angeles, CA 90048, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Masumi%2BSameer%2BPadhye%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sameer%20Padhye%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMasumi%2BSameer%2BPadhye%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jie Tang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>c</span><span class='nlm-institution'>Cedars-Sinai Genomics Core, Medical Genetics Institute, Cedars-Sinai Medical Center Los Angeles</span>, CA 90048, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jie%2BTang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJie%2BTang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Vaithilingaraja Arumugaswami </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>b</span><span class='nlm-institution'>Board of Governors Regenerative Medicine Institute, Cedars-Sinai Medical Center</span>, Los Angeles, CA 90048, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>d</span><span class='nlm-institution'>Department of Surgery, Cedars-Sinai Medical Center</span>, Los Angeles, CA 90048, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>e</span><span class='nlm-institution'>Department of Surgery, University of California at Los Angeles</span>, Los Angeles CA 90095, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vaithilingaraja%2BArumugaswami%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Arumugaswami%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVaithilingaraja%2BArumugaswami%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6872-5118" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vaithilingaraja Arumugaswami</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/061788v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:16978" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:16978/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/061788v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:16978" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:16978/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/061788v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:16978" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:16978/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/061788v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:16978" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:16978/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/061788v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:16978" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:16978/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/061788v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:16978" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:16978/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">ABSTRACT</h2><p id="p-2">Zika Virus (ZIKV) is a causative agent for poor pregnancy outcome and fetal developmental abnormalities, including microcephaly and eye defects. As a result, ZIKV is now a confirmed teratogen. Understanding host-pathogen interactions, specifically cellular perturbations caused by ZIKV, can provide novel therapeutic targets. In order to complete viral replication, viral pathogens control the host cellular machineries and regulate various factors, including long noncoding RNA (lncRNA) genes, at transcriptional levels. The role of lncRNA genes in the pathogenesis of ZIKV-mediated microcephaly and eye defects is currently unknown. To gain additional insights, we focused on profiling the differentially expressed lncRNA genes during ZIKV infection in mammalian cells. For this study, we employed a contemporary clinical Zika viral isolate, PRVABC59, of Asian genotype. We utilized an unbiased RNA sequencing approach to profile the lncRNA transcriptome in ZIKV infected Vero cells. We identified a total of 121 lncRNA genes that are differentially regulated at 48 hours post-infection. The majority of these genes are independently validated by reverse-transcription qPCR. A notable observation was that the lncRNAs, MALAT1 (Metastasis Associated Lung Adenocarcinoma Transcript 1) and NEAT1 (Nuclear Paraspeckle Assembly Transcript 1), are down-regulated upon Zika viral infection. MALAT1 and NEAT1 are known as nuclear localized RNAs that regulate gene expression and cell proliferation. Protein-lncRNA interaction maps revealed that MALAT1 and NEAT1 share common interacting partners and form a larger network comprising of 71 cellular factors. ZIKV-mediated dysregulation of these two regulatory lncRNAs can alter the expression of respective target genes and associated biological functions, an important one being cell division. In conclusion, this investigation is the first to provide insight into the biological connection of lncRNAs and ZIKV which can be further explored for developing antiviral therapy and understanding fetal developmental processes.</p></div><div class="section" id="sec-1"><h2 class="">INTRODUCTION</h2><p id="p-3">The human mosquito-borne pathogen, Zika Virus (ZIKV), is strongly associated with fetal developmental abnormalities, such as microcephaly and eye defects, as well as poor pregnancy outcomes [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>–<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>]. The teratogenic ZIKV belongs to <em>Flaviviridae</em> family and the viral genome comprises of a positive-sense, single-stranded RNA molecule that encodes a large polyprotein. This polyprotein is proteolytically cleaved into 11 functional peptides of viral structural and non-structural components. ZIKV was first isolated from a Rhesus monkey in 1947 [<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>]. The recent adaptation of ZIKV to a human host resulted in several epidemics across the globe [<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>]. The virus is transmitted by mosquitos and through sexual contact. The major concern is the increasing geographical distribution of mosquito vectors <em>Aedes aegypti</em>, and <em>Aedes albopictus</em>, which is responsible for ZIKV transmission in humans. Understanding the interaction of ZIKV with its human host is crucial for developing effective vaccines and therapeutic measures.</p><p id="p-4">The main focus of this study is to discern the effect of Zika viral infection on the expression of long non-coding RNA (lncRNA) genes. The presence of lncRNAs has been frequently determined with high confidence from transcriptomes and microarray technologies [<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>]. The Encyclopedia of DNA Elements (ENCODE) Project Consortium (GENCODE release 24) estimates that the human genome incorporates around 62% of 25,823 ncRNA genes as lncRNA genes that encode 28,031 well-defined lncRNA transcripts (<span class="underline"><a href="http://www.gencodegenes.org">http://www.gencodegenes.org</a></span>). The minority of the non-coding transcripts are small, whereas, the majority of them exceed 200 nucleotides in length, and they are consequently cataloged as lncRNAs [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>]. While this practical definition is fairly arbitrary, the size cutoff clearly differentiates lncRNAs from small regulatory RNAs, such as microRNAs (miRNAs), endogenous small interfering RNAs (endo-siRNAs) that participate in RNA interference (RNAi), and Piwi-associated RNAs (piRNAs). It is noted that the classical ncRNAs are ribosomal RNAs (rRNAs), ribozymes, transfer RNAs (tRNAs), small nuclear RNAs (snRNAs), small nucleolar RNAs (snoRNAs), and telomere-associated RNAs (TERC, TERRA) [<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>]. The majority of the lncRNA transcripts are expressed from intergenic regions and, to a lesser extent, in the protein-coding gene regions as sense or antisense transcripts.</p><p id="p-5">Previous studies have provided an unbiased finding of non-coding transcripts across many cell types and tissues [<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>] and made the conclusion that a vast majority of the genome was transcribed [<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>]. Interestingly, lncRNAs are playing a critical role in the developmental processes and etiology of a wide range of diseases. The expression patterns of many lncRNAs are associated with numerous key cellular processes, such as embryonic stem cell pluripotency, immune response, regulation of the cell cycle, and diseases such as cancer [<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>–<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>]. However, the function of the majority of lncRNAs remains unknown [<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>]. Even though lncRNAs exert a spectrum of regulatory mechanisms across different cellular pathways, a broad theme is emerging in which lncRNAs are driving the assembly of RNA-protein complexes that can facilitate the regulation of gene expression. Knowing that there is an interaction between lncRNAs and protein coding genes in both normal and disease states is essential to uncover the key functional roles of lncRNAs in cellular and developmental processes [<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>]. Thus, we carried out transcriptome analysis to determine expression patterns of lncRNAs in the ZIKV infected Vero cells that serve as a mammalian cell model. This study provides new insight into the interaction between ZIKV and its host.</p></div><div class="section" id="sec-2"><h2 class="">MATERIALS AND METHODS</h2><div id="sec-3" class="subsection"><h3>Cells</h3><p id="p-6">The Vero cell line (ATCC, USA) (kidney cells from an African green monkey) was cultured using complete Dulbecco’s modified Eagle’s medium (DMEM) (Fisher Scientific, USA). DMEM was supplemented with 10% fetal bovine serum (FBS), penicillin (100 units/ml), streptomycin (100 mg/ml), 10 mM Hepes, and 2 mM L-glutamine (Fisher Scientific, USA). The cells were cultured at 37°C with 5% CO<sub>2</sub> conditions and passaged every second or third day for experimental assays or when the cells reached over 80% confluency.</p></div><div id="sec-4" class="subsection"><h3>Zika virus</h3><p id="p-7">A ZIKV clinical isolate, PRVABC59 (GenBank accession number <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KU501215&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom">KU501215</a>), belonging to an Asian genotype was used in this study [<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>]. Early passage virus was obtained from the Centers for Disease Control and Prevention (CDC). Viral stocks were generated in Vero cells. Viral titer was estimated by a plaque assay, as reported previously [<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>].</p></div><div id="sec-5" class="subsection"><h3>Zika viral infection</h3><p id="p-8">Vero cells were seeded in a 12-well plate at a density of 1 × 10<sup>5</sup> cells per well. The following day, Zika viral inoculum was prepared in serum-free DMEM media at a multiplicity of infection (MOI) of 1 and was added to each well in triplicate. To allow virus attachment and entry, the plates were incubated in 37°C with 5% CO<sub>2</sub> for 4-6 hrs. After the designated incubation period, the inoculum was replaced with serum supplemented media. At 12, 24 and 48 hrs postinoculation, cells were harvested for RNA isolation. Uninfected control (mock) cells were included for each time point for comparison and were routinely treated with such the media without any viral inoculum.</p></div><div id="sec-6" class="subsection"><h3>RNA sequencing sample preparation</h3><p id="p-9">Total RNA was isolated from the cells using an RNeasy Mini Kit (QIAGEN, USA). RNase-Free DNase treatment was performed on the column to remove residual DNA. Fourteen RNA samples [duplicate mock and infected samples for 12 and 24 hr time points (total 8 samples) and triplicate samples for 48 hr time point (6 samples)] were assessed for concentration and quality using the Thermo Scientific (Waltham, MA) NanoDrop 8000 Spectrophotometer. RNA sequencing libraries were constructed using Illumina TruSeq RNA Sample Preparation Kit v2 (Illumina, San Diego, CA) per manufacturer’s instructions. Briefly, total RNA with RNA integrity (RIN) scores of 9 or better using Agilent Bioanalyzer RNA 6000 Nano kit were used for RNA library preparation. The poly (A)+ RNA was then purified from one microgram of total RNA from each sample using oligo-dT attached magnetic beads with two rounds of purification. Subsequently, the poly (A)+ RNA was fragmented and primed for cDNA synthesis according to manufacturer’s recommendations. RNA adapters and barcodes were ligated to cDNA to allow for clonal amplification and multiplexing. Sequencing was done on an Illumina NextSeq 500 using 75bp single-end sequencing kit to yield an average read depth of 28 million reads per sample with a minimum number of reads for a sample of at least 16 million reads. The sequencing data were deposited to Gene Expression Omnibus (GEO) database with the accession number GSE83900.</p></div><div id="sec-7" class="subsection"><h3>Data Analysis of RNA sequencing</h3><p id="p-10">Raw reads obtained from RNA-seq were aligned to the human genome using STAR (version 2.5) [<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>] with a custom human GRCh38 transcriptome reference downloaded from <a href="http://www.gencodegenes.org">http://www.gencodegenes.org</a>, which contains all protein-coding and long non-coding RNA genes based on human GENCODE version 23 annotation. A less stringent set of mapping parameters (allowing 5 mismatches per 75 bp reads) was used to align primate Vero cells to human reference.</p><p id="p-11">Expression counts for each gene in all of the samples were normalized by a modified trimmed mean of the M-values normalization method. The unsupervised principal component analysis (PCA) was performed with DESeq2 Bioconductor package version 1.10.1 in R version 3.2.2 [<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>]. Each gene was fitted into a negative binomial generalized linear model. The Wald test was applied to assess the differential expressions between two sample groups by DESeq2. The Benjamini and Hochberg procedure [<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>] was applied to adjust for multiple hypothesis testing and differential expressed gene candidates were selected with a false discovery rate of less than 0.10. For visualization of coordinated gene expression in the samples, a hierarchical clustering with the Pearson correlation distance matrix was performed with differential expressed gene candidates using the Bioconductor gplots package (version 2.14.2) in R. Significantly expressed genes were assessed for pathway enrichment using DAVID release 6.7 (<a href="https://david.ncifcrf.gov/">https://david.ncifcrf.gov/</a>) and Ingenuity Pathway Analysis. (<a href="http://www.ingenuity.com/products/ipa">http://www.ingenuity.com/products/ipa</a>) (QIAGEN, Redwood City). The significantly enriched canonical pathways were defined as having a q-value of &lt;.01.</p></div><div id="sec-8" class="subsection"><h3>Reverse transcription-quantitative PCR (RT-qPCR) Analysis</h3><p id="p-12">Total RNA was extracted from uninfected and ZIKV-infected cells using an RNeasy Mini Kit (QIAGEN). cDNA was synthesized using 3μg of total RNA and SuperScript III Reverse Transcriptase kit (Fisher Scientific, USA) and random hexamer primers as recommended by the manufacturers. Gene expression was quantified using Platinum SYBR Green qPCR SuperMix-UDG with ROX Kit (Life Technologies) by QuantStudio<sup>TM</sup> 12K Flex Real-Time PCR System (Life Technologies). The reaction for each gene expression was completed in duplicate and housekeeping human gene, Glyceraldehyde 3-phosphate dehydrogenase (GAPDH), was used as an internal reference control. The list of qPCR primers and their sequences are given in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Supplementary Table 1</a>. The following conditions were used for real-time amplification of the cDNA: 50°C for 2 min; 95°C for 2 min followed by 40 cycles of 95°C for 15 sec and 60°C for 1 min. The mRNA expression of each target gene was calculated based on 2<sup>−ΔCt</sup> values and fold-change was measured to the endogenous control, GAPDH.</p></div><div id="sec-9" class="subsection"><h3>Immunofluorescence Assay</h3><p id="p-13">Both the mock and infected cells were fixed with methanol for 30 mins in −20°C. Following three PBS washes, the cells were blocked with a blocking buffer (10% fetal bovine serum, 3% BSA, 0. 1% Triton-x 100 in PBS). Subsequently, the cells were incubated with mouse monoclonal antibody for Flavivirus group antigen [D1-4G2-4-15 (4G2)] (Absolute Antibody Ltd.) at a 1:200 dilution overnight at 4°C. The secondary antibody, mouse anti-goat polyclonal antibody-FITC (Santa Cruz Biotechnology) (Life Technologies, USA) was added at a 1:1000 dilution and was incubated for 1 hr at room temperature. Between antibody changes, the cells were washed five times with PBS. The nuclei were stained with a DAPI dye (Life Technologies, USA).</p></div><div id="sec-10" class="subsection"><h3>Non-coding RNA and protein interaction networks</h3><p id="p-14">To identify the high confident lncRNA-protein interaction network for NEAT1 and MALAT1, we used RNA-protein Association and Interaction Network’s (RAIN) web interface (<a href="http://rth.dk/resources/rain/">http://rth.dk/resources/rain/</a>). Subsequently, the combined NEAT1 and MALAT1 interaction analyses were carried out in the Search Tool for the Retrieval of Interacting Genes/Proteins (STRING) v10.0 database [<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>]. To avoid false positive results, a high confidence interaction score of 0.7 was used as a cut-off to obtain a maximum of 20 interactions under <em>evidence mode</em> for each of the two lncRNAs. The combined interacting network of these two lncRNAs with other human proteins was also predicted. The comprehensive interaction network was predicted using the 11 identified differentially co-expressed genes in normal and ZIKV infected conditions. Typically, the interaction network view summarizes the predicted associations for a specific group of proteins. The network nodes are proteins whereas the edges represent the predicted functional associations. The <em>evidence mode</em> based edges are drawn in with up to 7 differently colored lines representing the existence of the seven types of evidences. The evidence falls under 3 broad categories, namely known interactions, predicted interactions and other interactions, which was used in predicting the associations. The known interactions include the evidence that was experimentally determined (pink) or the evidence that was curated from database (sky blue). The predicted interactions depict the evidence of gene neighborhood (green), gene fusion (red) or gene co-occurrence (blue). The other interactions show evidence of co-expression (black), text-mining (light green) and protein homology (purple). Larger nodes reflect the availability of structural information associated with the factors. The Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway analysis was done based on factors in the interaction networks.</p></div><div id="sec-11" class="subsection"><h3>Statistical analysis</h3><p id="p-15">Error bars in the graph reflect the standard deviation. P-values were determined by the twotailed Student’s t-test and significance was reported if the p value is &lt;0.05.</p></div></div><div class="section" id="sec-12"><h2 class="">RESULTS</h2><div id="sec-13" class="subsection"><h3>Zika virus infectious cell culture system</h3><p id="p-16">We utilized a Vero cell-based <em>in vitro</em> infectious culture system to investigate the ZIKV effect on the expression of lncRNAs. Vero cells supported robust lytic infection by the Asian ZIKV genotype (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1A</a>). The viral-mediated cytopathic effects (CPEs), consisting of cell rounding and lysis, were observed at 48 hpi. The Zika viral genome replication was quantified by RT-qPCR, as described in the Methods section. At 48 hpi, several log increases in the ZIKV genome replication was observed (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1B</a>). The virus infection in the Vero cells was also confirmed by detecting viral envelope proteins in the cytoplasm by Immunofluorescence Assay (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1C</a>). These virological assays confirm the <em>in vitro</em> establishment of Zika virus infection, as well as a visible impact on cell health. Subsequently, we investigated the ZIKV-specific alterations in cellular gene expression profiles.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Vero cell-based ZIKV infectious culture system (A) Bright field image presents ZIKV infected cells at 48 hours post-infection (hpi). Uninfected control (mock) cells are shown as well. (B) Graph depicts Zika viral genome replication at 48 hpi, and fold-change was calculated to that of mock control cells. (C) Immunofluorescence analysis shows ZIKV infected cells are positive for viral envelope protein." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-796616661" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Vero cell-based ZIKV infectious culture system&lt;/span&gt; (A) Bright field image presents ZIKV infected cells at 48 hours post-infection (hpi). Uninfected control (mock) cells are shown as well. (B) Graph depicts Zika viral genome replication at 48 hpi, and fold-change was calculated to that of mock control cells. (C) Immunofluorescence analysis shows ZIKV infected cells are positive for viral envelope protein.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F1.medium.gif" width="440" height="345"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F1.medium.gif" width="440" height="345"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">Vero cell-based ZIKV infectious culture system</span><p id="p-17" class="first-child">(A) Bright field image presents ZIKV infected cells at 48 hours post-infection (hpi). Uninfected control (mock) cells are shown as well. (B) Graph depicts Zika viral genome replication at 48 hpi, and fold-change was calculated to that of mock control cells. (C) Immunofluorescence analysis shows ZIKV infected cells are positive for viral envelope protein.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Transcriptome analysis of Zika viral infected cells</h3><p id="p-18">The transcriptome of all 14 samples from the Vero cells both with and without Zika viral infection were characterized at three different time points (12, 24, and 48 hpi) by RNA-Seq. After mapping with human transcriptome, 65-69% of overall mapping was observed per sample, &gt;80% of the reads were successfully mapped to the exonic region in each sample and &gt;80% of the aligned reads were uniquely mapped, which confirms the high quality of our samples and sequencing. We focused on identifying lncRNA mRNA that are differentially expressed between uninfected and ZIKV infected cells. The changes in lncRNA expression were observed only at the later time point of 48 hpi. For the 48 hpi time point, a total of 121 lncRNA genes were differentially expressed with an adjusted p value of &lt;0.05 and 87 of these genes had an adjusted p value of &lt;0.01 (Supplementary Table 2). A heat map of these differentially expressed lncRNAs is given in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Global transcriptional analysis of ZIKV infected cells Heat map depicts the differentially expressed lncRNA genes at 48 hour post-infection. (Red: up-regulated genes; Green: down-regulated genes). The hierarchical clustering was done using 121 DE lncRNAs for infected and control samples." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-796616661" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Global transcriptional analysis of ZIKV infected cells&lt;/span&gt; Heat map depicts the differentially expressed lncRNA genes at 48 hour post-infection. (Red: up-regulated genes; Green: down-regulated genes). The hierarchical clustering was done using 121 DE lncRNAs for infected and control samples.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F2.medium.gif" width="262" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F2.medium.gif" width="262" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Global transcriptional analysis of ZIKV infected cells</span><p id="p-19" class="first-child">Heat map depicts the differentially expressed lncRNA genes at 48 hour post-infection. (Red: up-regulated genes; Green: down-regulated genes). The hierarchical clustering was done using 121 DE lncRNAs for infected and control samples.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Validation of lncRNAs differentially expressed during Zika virus infection</h3><p id="p-20">For validation, ZIKV infection experiment was repeated in Vero cells. Total RNA isolated at 48 hpi was subjected to RT-qPCR analysis to verify the lncRNAs. We confirmed the following up-regulated genes: LINC00184, LINC00672, RP11-213H15.1, EGOT, RP11-671J11.7, MIR222HG, KB-1732A1.1, C11orf97 and CITF22-1A6.3 (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3</a>). Several of the down-regulated genes were also verified. Notably, MALAT1 (Metastasis Associated Lung Adenocarcinoma Transcript 1) and NEAT1 (Nuclear Paraspeckle Assembly Transcript 1) lncRNA levels were highly reduced in Zika virus infected cells (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3</a>). HUCL (Hepatocellular Carcinoma Up-Regulated Long Non-Coding RNA) lncRNA was included as a control, which did not show changes during ZIKV infection. Since most of the lncRNAs are functionally uncharacterized, we focused on the previously known NEAT1 and MALAT1 for downstream studies.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Validation of differentially expressed lncRNAs Graph shows the expression levels of selected lncRNAs in ZIKV infected cells. Gene expression was quantified by RT-qPCR. Housekeeping gene GAPDH was used to normalize mRNA expression between ZIKV infected and mock control cells. For each gene, fold-change was calculated to that of uninfected cell control (mock) and presented as a bar graph." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-796616661" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Validation of differentially expressed lncRNAs&lt;/span&gt; Graph shows the expression levels of selected lncRNAs in ZIKV infected cells. Gene expression was quantified by RT-qPCR. Housekeeping gene GAPDH was used to normalize mRNA expression between ZIKV infected and mock control cells. For each gene, fold-change was calculated to that of uninfected cell control (mock) and presented as a bar graph.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F3.medium.gif" width="440" height="177"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F3.medium.gif" width="440" height="177"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">Validation of differentially expressed lncRNAs</span><p id="p-21" class="first-child">Graph shows the expression levels of selected lncRNAs in ZIKV infected cells. Gene expression was quantified by RT-qPCR. Housekeeping gene GAPDH was used to normalize mRNA expression between ZIKV infected and mock control cells. For each gene, fold-change was calculated to that of uninfected cell control (mock) and presented as a bar graph.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Functional interactions of differentially expressed lncRNAs and proteins in ZIKV infected cells</h3><p id="p-22">Identifying an interaction network of lncRNAs and protein coding genes, which are differentially regulated in ZIKV infected cells relative to the uninfected control, is essential to uncover the key biological functions of lncRNAs in cell injury processes. Hence, we focused on identifying the factors that can interact with lncRNAs NEAT1 and MALAT1 using RAIN and STRING. Initial seed interaction networks for NEAT1 and MALAT1 predicted a maximum of 20 interacting factors (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Supplementary Figures 1</a> and 2). There were 20 and 19 high confidence nodes that involved in 31 and 32 interactions for NEAT1 and MALAT1 networks, respectively. Almost, all the interactions in these two independent networks were supported by text-mining. Afterwards, the factors interacted with each of the two lncRNAs and were combined to find the common interacting factors (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). We identified two lncRNAs namely MIAT and TUG1 as common interacting factors while no common proteins were identified. A total of 35 factors were identified to have 75 interaction networks. Subsequently, we analyzed the expression levels of these 35 factors in ZIKV infected cells. This approach yielded a total of 11 factors with differential expression, which included 6 proteins (ABCB1, HCN2, KIAA1147, PRRG1, PSPC1, SFPQ) and 5 lncRNAs (LINC00657, MALAT1, NEAT1, SNHG11, GAS5) (Supplementary Table 3). Based on the evidence provided by the interaction networks, 3 (SFPQ, NONO, PSPC1) of 35 factors are known to be involved in common cellular and biological processes. Based on this study, 9 of 11 differentially expressed genes identified to constitute novel RNA-Protein interaction (RPI) networks (ABCB1, GAS5, HCN2, KIAA1147, LINC00657, MALAT1, NEAT 1, PRRG1, and SNHG11).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="LncRNAs NEAT1 and MALAT1 interaction networks (A) The network represents lncRNAs NEAT1 and MALAT1 interactions with 17 lncRNAs and 18 proteins, in which 5 lncRNAs and 6 proteins were co-regulated in ZIKV infected cells (underlined genes). There are 35 high confidence nodes involved in 75 interactions (PPI enrichment p-value of 0.0 and clustering coefficient of 0.856). (B) Heat map of differentially expressed genes in ZIKV infected cells, which are present in the interaction network. (Red: up-regulated genes; Green: down-regulated genes)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-796616661" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;LncRNAs NEAT1 and MALAT1 interaction networks&lt;/span&gt; (A) The network represents lncRNAs NEAT1 and MALAT1 interactions with 17 lncRNAs and 18 proteins, in which 5 lncRNAs and 6 proteins were co-regulated in ZIKV infected cells (underlined genes). There are 35 high confidence nodes involved in 75 interactions (PPI enrichment p-value of 0.0 and clustering coefficient of 0.856). (B) Heat map of differentially expressed genes in ZIKV infected cells, which are present in the interaction network. (Red: up-regulated genes; Green: down-regulated genes).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F4.medium.gif" width="440" height="236"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F4.medium.gif" width="440" height="236"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">LncRNAs NEAT1 and MALAT1 interaction networks</span><p id="p-23" class="first-child">(A) The network represents lncRNAs NEAT1 and MALAT1 interactions with 17 lncRNAs and 18 proteins, in which 5 lncRNAs and 6 proteins were co-regulated in ZIKV infected cells (underlined genes). There are 35 high confidence nodes involved in 75 interactions (PPI enrichment p-value of 0.0 and clustering coefficient of 0.856). (B) Heat map of differentially expressed genes in ZIKV infected cells, which are present in the interaction network. (Red: up-regulated genes; Green: down-regulated genes).</p><div class="sb-div caption-clear"></div></div></div><p id="p-24"><span class="underline"><em>Comprehensive IncRNA-Protein interaction network</em></span>. Proteins collectively contribute to a shared and specific function. For instance, if a single edge appears with different lines/colors between two nodes, this clearly indicates that the interaction between two given genes/nodes is supported by more than one verification [<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>]. It is ambiguous if current co-regulated gene data can be correlated to the predicted human lncRNA-protein networks even though it is widely known that interacting factors are frequently co-expressed [<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>]. With the evidence from the interacting network, we identified known co-expressing genes that are differentially regulated during ZIKV infection from our genome-wide gene expression data sets. We extended our analysis using the 11 interacting factors defined in our previous analysis for further comprehensive interacting network analysis in order to identify more high confidence coexpressing interacting factors and their possible functional associations. Bioinformatics analysis of the interacting data uncovered a network of 289 interactions between 71 factors, including 11 seeds with high statistical significance (PPI enrichment <em>p</em>-value 0; clustering co-efficient 0.699). Among the 71 interacting factors in the network, 51 factors (46 protein-coding, 2 ncRNAs and 3 lncRNAs) were differentially expressed in our study (Supplementary Table 3). As evidenced by the interaction network, 28 of 71 factors were known to be co-expressed in many cellular and biological processes. Notably, 29 of the 51 differentially expressed factors identified during ZIKV infection are novel co-expressing factors (AVL9, CYP3A5, DENND6A, ETS1, FBXO15, FOXO3, GAS5, HAX1, HCN1, HCN2, HIF1A, HSF1, HSPB1, KIAA1147, LANCL2, LINC00657, MALAT1, MAPK8, NEAT1, PIM1, PRRG1, PTBP1, RNF43, SNHG11, SP1, TAF12, TAF7, TP53, UGCG) (Supplementary Table 3). It is possible that 5 lncRNAs having interactions with 24 proteins may form post-transcriptional regulatory networks [<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>–<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>], and control fundamental cellular and developmental processes. We also observed that apart from the text-mining, the majority of the lncRNA-protein interactions were supported by experimental evidence.</p><p id="p-25"><span class="underline"><em>Pathway analysis of interacting factors in the network</em></span>. Subsequently, we performed the KEGG functional enrichment analysis for all 71 interacting factors to identify their functional associations. We noted that the factors interacting in the network were involved in 33 functional KEGG pathways, in which 79% (N=26) of the pathways consist of less than six proteins (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5</a>; Supplementary Table 4). Interestingly, several of the enriched pathways are associated with viral infections. Furthermore, in the context of ZIKV and microcephaly, factors involved in the Neurotrophin signaling pathway (FOXO3, TP53, ABL1, JUN, MAPK8) and MAPK signaling pathway (HSPB1, TP53, FOS, JUN, MAPK8) were identified. As a result of this study, we generated a comprehensive lncRNA-protein interactions map based on the differentially expressed genes during ZIKV infection.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A comprehensive RNA-protein interaction network for 11 factors co-regulated in the differentially expressed gene data set. The network represents 11 factors (5 lncRNAs and 6 proteins), and additional 40 co-regulated proteins identified in our differentially expressed gene data sets. A total of 71 high confidence nodes are involved in 289 interactions, which was supported by the PPI enrichment p-value of 0.0 and clustering coefficient of 0.699." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-796616661" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;A comprehensive RNA-protein interaction network for 11 factors co-regulated in the differentially expressed gene data set.&lt;/span&gt; The network represents 11 factors (5 lncRNAs and 6 proteins), and additional 40 co-regulated proteins identified in our differentially expressed gene data sets. A total of 71 high confidence nodes are involved in 289 interactions, which was supported by the PPI enrichment &lt;em&gt;p&lt;/em&gt;-value of 0.0 and clustering coefficient of 0.699.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F5.medium.gif" width="440" height="287"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F5.medium.gif" width="440" height="287"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <span class="caption-title">A comprehensive RNA-protein interaction network for 11 factors co-regulated in the differentially expressed gene data set.</span><p id="p-26" class="first-child">The network represents 11 factors (5 lncRNAs and 6 proteins), and additional 40 co-regulated proteins identified in our differentially expressed gene data sets. A total of 71 high confidence nodes are involved in 289 interactions, which was supported by the PPI enrichment <em>p</em>-value of 0.0 and clustering coefficient of 0.699.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">MALAT1 and NEAT1 have been shown to be involved in cell proliferation and cell cycle control [<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>]. Down-regulation of MALAT1 and NEAT1 by Zika virus can inhibit cell cycle progression. We found that the cell cycle pathways and various cell cycle genes including CDK1, Cyclin B, E2F, and p21 were deregulated in ZIKV infected cells (Supplementary <a id="xref-fig-8-1" class="xref-fig" href="#F8">Fig. 3</a>). Our data suggests that the lncRNAs MALAT1 and NEAT1 can play important roles in the pathogenesis of Zika virus-mediated developmental diseases.</p></div></div><div class="section" id="sec-17"><h2 class="">DISCUSSION</h2><p id="p-28">Here, we have provided the first comprehensive analysis of the effect of ZIKV infection on cellular lncRNA gene expression. We utilized an unbiased RNA sequencing approach to profile differentially regulated genes during ZIKV infection of mammalian cells. We identified a total of 121 lncRNA genes that were differentially regulated during infection. Though over 25,000 lncRNAs are catalogued in the human genome, the functions of only a few hundred of them are known [<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>]. We identified that MALAT1 and NEAT1, two lncRNAs with known functions, were down-regulated upon ZIKV infection.</p><p id="p-29">MALAT1 is localized at nuclear speckles. These sub-nuclear structures are involved in RNA polymerase II transcription, splicing of pre-mRNA and export of processed mRNA ([<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>]). NEAT1 is present in the nuclear paraspeckle structures [<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>, <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>]. Both of these lncRNAs have been shown to be upregulated in cancer cells and to be involved in cell proliferation [<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a>, <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">33</a>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>–<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>]. Depletion of these lncRNAs can cause loss of cell proliferation and cell cycle arrest. ZIKV has been shown to cause cell cycle inhibition and neuronal cell death [<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>]. These observations allow us to formulate a hypothesis that ZIKV may regulate MALAT1 and NEAT1 for cell cycle control towards completing virus replication. This investigation was carried out in a ZIKV infected model cell line. Profiling the deregulated lncRNAs during ZIKV infection of human neuro-and retinal progenitor cells (the cell types frequently affected by ZIKV <em>in utero</em>) can yield a tissue type-specific lncRNA expression profile and functional relevance.</p><p id="p-30">The majority of important molecular functions in cells are governed by interactions among proteins and other biomolecules [<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>, <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>]. The RNA-protein interactions, which are known to facilitate post-transcriptional regulatory networks, are beginning to be understood [<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a>–<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a>]. Through detailed bioinformatics analysis of our differentially expressed gene database, we have identified 11 proteins/lncRNAs that form the core factors in the RPI network involving MALAT1 and NEAT1. Further comprehensive analysis revealed the formation of a larger interaction network comprised of 71 factors. This includes 51 DE genes from ZIKV infected cells. This analysis provides a comprehensive view on the complexity of lncRNA and ZIKV interaction. Although recent studies have found evidence of many RNA binding proteins and RPIs in cells [<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>–<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>], huge numbers of lncRNAs and proteins that participate in RPIs need to be characterized. Apart from regulating gene expression at the post-transcriptional level, RPIs can control many fundamental biological processes including, but not limited to, DNA replication, transcription, pathogen resistance and viral replication [<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>–<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>]. As ZIKV causes developmental abnormalities, a further detailed investigation on the functions of lncRNAs in fetal development and Zika viral disease pathogenesis could yield additional clues to prevent the teratogenic effect.</p></div><div class="section supplemental-materials" id="sec-18"><h2 class="">Supplementary Figures and Tables</h2><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="LncRNA NEAT1 interaction network Lncrna NEAT1 interacts with 10 lncRNAs and 10 proteins, in which 2 lncRNAs and 4 proteins, were co-regulated in our data sets. A total of 20 high confidence nodes are involved in 31 interactions. This was supported by the PPI enrichment p-value of 0.007 and clustering coefficient of 0.895." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-796616661" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;LncRNA NEAT1 interaction network&lt;/span&gt; Lncrna NEAT1 interacts with 10 lncRNAs and 10 proteins, in which 2 lncRNAs and 4 proteins, were co-regulated in our data sets. A total of 20 high confidence nodes are involved in 31 interactions. This was supported by the PPI enrichment &lt;em&gt;p&lt;/em&gt;-value of 0.007 and clustering coefficient of 0.895.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F6.medium.gif" width="440" height="394"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F6.medium.gif" width="440" height="394"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 1.</span> <span class="caption-title">LncRNA NEAT1 interaction network</span><p id="p-32" class="first-child">Lncrna NEAT1 interacts with 10 lncRNAs and 10 proteins, in which 2 lncRNAs and 4 proteins, were co-regulated in our data sets. A total of 20 high confidence nodes are involved in 31 interactions. This was supported by the PPI enrichment <em>p</em>-value of 0.007 and clustering coefficient of 0.895.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="LncRNA MALAT1 interaction network This network represents lncRNA MALAT1 interaction with biomolecules, including 11 lncRNAs and 8 proteins, in which 5 and 2 lncRNAs and proteins, respectively were co-regulated in our differentially expressed gene data sets. A total of 19 high confidence nodes are involved in 32 interactions, which was supported by the PPI enrichment p-value of 0.002 and clustering coefficient of 0.866." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-796616661" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;LncRNA MALAT1 interaction network&lt;/span&gt; This network represents lncRNA MALAT1 interaction with biomolecules, including 11 lncRNAs and 8 proteins, in which 5 and 2 lncRNAs and proteins, respectively were co-regulated in our differentially expressed gene data sets. A total of 19 high confidence nodes are involved in 32 interactions, which was supported by the PPI enrichment &lt;em&gt;p&lt;/em&gt;-value of 0.002 and clustering coefficient of 0.866.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F7.medium.gif" width="440" height="415"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F7.medium.gif" width="440" height="415"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 2.</span> <span class="caption-title">LncRNA MALAT1 interaction network</span><p id="p-33" class="first-child">This network represents lncRNA MALAT1 interaction with biomolecules, including 11 lncRNAs and 8 proteins, in which 5 and 2 lncRNAs and proteins, respectively were co-regulated in our differentially expressed gene data sets. A total of 19 high confidence nodes are involved in 32 interactions, which was supported by the PPI enrichment <em>p</em>-value of 0.002 and clustering coefficient of 0.866.</p><div class="sb-div caption-clear"></div></div></div><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Differentially expressed (DE) genes involved in cell cycle during ZIKV infection at 48 hpi. (A) Heat map of DE genes is shown. (B) Pathway analysis of genes involved in cell cycle regulation. The positive cell cycle regulators, CDK1 and Cyclin B, are down-regulated in ZIKV infected cells." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-796616661" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Differentially expressed (DE) genes involved in cell cycle during ZIKV infection at 48 hpi. (A) Heat map of DE genes is shown. (B) Pathway analysis of genes involved in cell cycle regulation. The positive cell cycle regulators, CDK1 and Cyclin B, are down-regulated in ZIKV infected cells.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F8.medium.gif" width="440" height="346"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F8.medium.gif" width="440" height="346"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/01/061788/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 3.</span> <p id="p-34" class="first-child">Differentially expressed (DE) genes involved in cell cycle during ZIKV infection at 48 hpi. (A) Heat map of DE genes is shown. (B) Pathway analysis of genes involved in cell cycle regulation. The positive cell cycle regulators, CDK1 and Cyclin B, are down-regulated in ZIKV infected cells.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/221012/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/221012/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/221012" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Supplementary Table 1.</span> <p id="p-35" class="first-child">List of primers used in this study.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36"><strong>Supplementary Table 2</strong>. List of differentially expressed lncRNA genes in Zika virus infected cells at 48 hpi.</p><p id="p-37"><strong>Supplementary Table 3. Co-expressing biomolecules from the comprehensive interaction networks that were identified in our data sets</strong>. <strong>(A)</strong> List of co-expressing differentially expressed 46 protein coding genes and 5 lncRNAs identified in our data sets. <strong>(B)</strong> List of 20 protein-coding genes that were not present in our data sets.</p><p id="p-38"><strong>Supplementary Table 4. The KEGG pathway enrichment analysis for 71 factors</strong>. The 71 factors include 5 lncRNAs and 66 proteins from the comprehensive interaction networks (shown in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>), found to be involved in 33 functional pathways.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-31">We are thankful to Dr. Aaron Brault and Dr. Brandy Russell of the Centers for Disease Control and Prevention (CDC), USA for sharing Zika viral strain PRVABC59. We also thank Nikhil Chakravarty of Cedars-Sinai Medical Center for editing this manuscript. This work was supported by Cedars-Sinai Medical Center’s Institutional Research Award to V.A.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">REFERENCES</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brasil</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Zika Virus Infection in Pregnant Women in Rio de Janeiro – Preliminary Report</span>. <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>, <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">2.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-061788v1.2" data-doi="10.15585/mmwr.mm6503e2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schuler-Faccini</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Possible Association Between Zika Virus Infection and Microcephaly – Brazil, 2015</span>. <abbr class="cit-jnl-abbrev">MMWR Morb Mortal Wkly Rep</abbr>, <span class="cit-pub-date">2016</span>. <span class="cit-vol">65</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">59</span>–<span class="cit-lpage">62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMMWR%2BMorb%2BMortal%2BWkly%2BRep%26rft.volume%253D65%26rft.spage%253D59%26rft.atitle%253DPOSSIBLE%2BASSOCIATION%2BBETWEEN%2BZIKA%2BVIRUS%2BINFECTION%2BAND%2BMICROCEPHALY%2B%2B%2BBRAZIL%252C%2B2015.%26rft_id%253Dinfo%253Adoi%252F10.15585%252Fmmwr.mm6503e2%26rft_id%253Dinfo%253Apmid%252F26820244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.15585/mmwr.mm6503e2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26820244&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mlakar</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Zika Virus Associated with Microcephaly</span>. <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>, <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.4" data-doi="10.1016/0035-9203(52)90043-6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dick</span>,  <span class="cit-name-given-names">G.W.</span></span>, <span class="cit-article-title">Zika virus. II. Pathogenicity and physical properties</span>. <abbr class="cit-jnl-abbrev">Trans R Soc Trop Med Hyg</abbr>, <span class="cit-pub-date">1952</span>. <span class="cit-vol">46</span>(<span class="cit-issue">5</span>): p. <span class="cit-fpage">521</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrans%2BR%2BSoc%2BTrop%2BMed%2BHyg%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0035-9203%252852%252990043-6%26rft_id%253Dinfo%253Apmid%252F12995441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0035-9203(52)90043-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12995441&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.5" data-doi="10.1016/0035-9203(52)90042-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dick</span>,  <span class="cit-name-given-names">G.W.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.F.</span> <span class="cit-name-surname">Kitchen</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A.J.</span> <span class="cit-name-surname">Haddow</span></span>, <span class="cit-article-title">Zika virus. I. Isolations and serological specificity</span>. <abbr class="cit-jnl-abbrev">Trans R Soc Trop Med Hyg</abbr>, <span class="cit-pub-date">1952</span>. <span class="cit-vol">46</span>(<span class="cit-issue">5</span>): p. <span class="cit-fpage">509</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrans%2BR%2BSoc%2BTrop%2BMed%2BHyg%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0035-9203%252852%252990042-4%26rft_id%253Dinfo%253Apmid%252F12995440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0035-9203(52)90042-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12995440&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.6" data-doi="10.1128/JVI.00354-15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hamel</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Biology of Zika Virus Infection in Human Skin Cells</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>, <span class="cit-pub-date">2015</span>. <span class="cit-vol">89</span>(<span class="cit-issue">17</span>): p. <span class="cit-fpage">8880</span>–<span class="cit-lpage">96</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00354-15%26rft_id%253Dinfo%253Apmid%252F26085147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4OS8xNy84ODgwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTYvMDcvMDEvMDYxNzg4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.7" data-doi="10.3201/eid1408.080287"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lanciotti</span>,  <span class="cit-name-given-names">R.S.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Genetic and serologic properties of Zika virus associated with an epidemic, Yap State, Micronesia, 2007</span>. <abbr class="cit-jnl-abbrev">Emerg Infect Dis</abbr>, <span class="cit-pub-date">2008</span>. <span class="cit-vol">14</span>(<span class="cit-issue">8</span>): p. <span class="cit-fpage">1232</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2Binfectious%2Bdiseases%26rft.stitle%253DEmerg%2BInfect%2BDis%26rft.aulast%253DLanciotti%26rft.auinit1%253DR.%2BS.%26rft.volume%253D14%26rft.issue%253D8%26rft.spage%253D1232%26rft.epage%253D1239%26rft.atitle%253DGenetic%2Band%2Bserologic%2Bproperties%2Bof%2BZika%2Bvirus%2Bassociated%2Bwith%2Ban%2Bepidemic%252C%2BYap%2BState%252C%2BMicronesia%252C%2B2007.%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid1408.080287%26rft_id%253Dinfo%253Apmid%252F18680646%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid1408.080287&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18680646&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258216900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.8" data-doi="10.1038/nature11233"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Djebali</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Landscape of transcription in human cells</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-pub-date">2012</span>. <span class="cit-vol">489</span>(<span class="cit-issue">7414</span>): p. <span class="cit-fpage">101</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDjebali%26rft.auinit1%253DS.%26rft.volume%253D489%26rft.issue%253D7414%26rft.spage%253D101%26rft.epage%253D108%26rft.atitle%253DLandscape%2Bof%2Btranscription%2Bin%2Bhuman%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11233%26rft_id%253Dinfo%253Apmid%252F22955620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11233&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22955620&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308347000043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.9" data-doi="10.1016/j.cell.2010.06.040"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huarte</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">A large intergenic noncoding RNA induced by p53 mediates global gene repression in the p53 response</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>, <span class="cit-pub-date">2010</span>. <span class="cit-vol">142</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">409</span>–<span class="cit-lpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHuarte%26rft.auinit1%253DM.%26rft.volume%253D142%26rft.issue%253D3%26rft.spage%253D409%26rft.epage%253D419%26rft.atitle%253DA%2Blarge%2Bintergenic%2Bnoncoding%2BRNA%2Binduced%2Bby%2Bp53%2Bmediates%2Bglobal%2Bgene%2Brepression%2Bin%2Bthe%2Bp53%2Bresponse.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2010.06.040%26rft_id%253Dinfo%253Apmid%252F20673990%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2010.06.040&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20673990&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280609100018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.10" data-doi="10.1534/genetics.112.146704"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kung</span>,  <span class="cit-name-given-names">J.T.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Colognori</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.T.</span> <span class="cit-name-surname">Lee</span></span>, <span class="cit-article-title">Long noncoding RNAs: past, present, and future</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-pub-date">2013</span>. <span class="cit-vol">193</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">651</span>–<span class="cit-lpage">69</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKung%26rft.auinit1%253DJ.%2BT.%2BY.%26rft.volume%253D193%26rft.issue%253D3%26rft.spage%253D651%26rft.epage%253D669%26rft.atitle%253DLong%2BNoncoding%2BRNAs%253A%2BPast%252C%2BPresent%252C%2Band%2BFuture%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.146704%26rft_id%253Dinfo%253Apmid%252F23463798%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzMvNjUxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTYvMDcvMDEvMDYxNzg4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.11" data-doi="10.1038/nbt.1633"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guttman</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Ab initio reconstruction of cell type-specific transcriptomes in mouse reveals the conserved multi-exonic structure of lincRNAs</span>. <abbr class="cit-jnl-abbrev">Nat Biotechnol</abbr>, <span class="cit-pub-date">2010</span>. <span class="cit-vol">28</span>(<span class="cit-issue">5</span>): p. <span class="cit-fpage">503</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DGuttman%26rft.auinit1%253DM.%26rft.volume%253D28%26rft.issue%253D5%26rft.spage%253D503%26rft.epage%253D510%26rft.atitle%253DAb%2Binitio%2Breconstruction%2Bof%2Bcell%2Btype-specific%2Btranscriptomes%2Bin%2Bmouse%2Breveals%2Bthe%2Bconserved%2Bmulti-exonic%2Bstructure%2Bof%2BlincRNAs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1633%26rft_id%253Dinfo%253Apmid%252F20436462%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1633&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20436462&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277452700031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.12" data-doi="10.1101/gad.17446611"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cabili</span>,  <span class="cit-name-given-names">M.N.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Integrative annotation of human large intergenic noncoding RNAs reveals global properties and specific subclasses</span>. <abbr class="cit-jnl-abbrev">Genes Dev</abbr>, <span class="cit-pub-date">2011</span>. <span class="cit-vol">25</span>(<span class="cit-issue">18</span>): p. <span class="cit-fpage">1915</span>–<span class="cit-lpage">27</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.17446611%26rft_id%253Dinfo%253Apmid%252F21890647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjI1LzE4LzE5MTUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNi8wNy8wMS8wNjE3ODguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.13" data-doi="10.1038/nature05874"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Consortium</span>,  <span class="cit-name-given-names">E.P.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Identification and analysis of functional elements in 1% of the human genome by the ENCODE pilot project</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-pub-date">2007</span>. <span class="cit-vol">447</span>(<span class="cit-issue">7146</span>): p. <span class="cit-fpage">799</span>–<span class="cit-lpage">816</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D447%26rft.issue%253D7146%26rft.spage%253D799%26rft.epage%253D816%26rft.atitle%253DIdentification%2Band%2Banalysis%2Bof%2Bfunctional%2Belements%2Bin%2B1%2525%2Bof%2Bthe%2Bhuman%2Bgenome%2Bby%2Bthe%2BENCODE%2Bpilot%2Bproject.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05874%26rft_id%253Dinfo%253Apmid%252F17571346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05874&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17571346&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247207500034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.14" data-doi="10.1101/gr.078378.108"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dinger</span>,  <span class="cit-name-given-names">M.E.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Long noncoding RNAs in mouse embryonic stem cell pluripotency and differentiation</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr>, <span class="cit-pub-date">2008</span>. <span class="cit-vol">18</span>(<span class="cit-issue">9</span>): p. <span class="cit-fpage">1433</span>–<span class="cit-lpage">45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.078378.108%26rft_id%253Dinfo%253Apmid%252F18562676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE4LzkvMTQzMyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE2LzA3LzAxLzA2MTc4OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">15.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-061788v1.15" data-doi="10.1038/nature07672"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guttman</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Chromatin signature reveals over a thousand highly conserved large non-coding RNAs in mammals</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-pub-date">2009</span>. <span class="cit-vol">458</span>(<span class="cit-issue">7235</span>): p. <span class="cit-fpage">223</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGuttman%26rft.auinit1%253DM.%26rft.volume%253D458%26rft.issue%253D7235%26rft.spage%253D223%26rft.epage%253D227%26rft.atitle%253DChromatin%2Bsignature%2Breveals%2Bover%2Ba%2Bthousand%2Bhighly%2Bconserved%2Blarge%2Bnon-coding%2BRNAs%2Bin%2Bmammals.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07672%26rft_id%253Dinfo%253Apmid%252F19182780%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07672&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19182780&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264059700048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">16.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-061788v1.16" data-doi="10.1038/ng.710"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Loewer</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Large intergenic non-coding RNA-RoR modulates reprogramming of human induced pluripotent stem cells</span>. <abbr class="cit-jnl-abbrev">Nat Genet</abbr>, <span class="cit-pub-date">2010</span>. <span class="cit-vol">42</span>(<span class="cit-issue">12</span>): p. <span class="cit-fpage">1113</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DLoewer%26rft.auinit1%253DS.%26rft.volume%253D42%26rft.issue%253D12%26rft.spage%253D1113%26rft.epage%253D1117%26rft.atitle%253DLarge%2Bintergenic%2Bnon-coding%2BRNA-RoR%2Bmodulates%2Breprogramming%2Bof%2Bhuman%2Binduced%2Bpluripotent%2Bstem%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.710%26rft_id%253Dinfo%253Apmid%252F21057500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.710&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21057500&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284578800015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">17.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-061788v1.17" data-doi="10.1038/ng.848"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hung</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Extensive and coordinated transcription of noncoding RNAs within cell-cycle promoters</span>. <abbr class="cit-jnl-abbrev">Nat Genet</abbr>, <span class="cit-pub-date">2011</span>. <span class="cit-vol">43</span>(<span class="cit-issue">7</span>): p. <span class="cit-fpage">621</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHung%26rft.auinit1%253DT.%26rft.volume%253D43%26rft.issue%253D7%26rft.spage%253D621%26rft.epage%253D629%26rft.atitle%253DExtensive%2Band%2Bcoordinated%2Btranscription%2Bof%2Bnoncoding%2BRNAs%2Bwithin%2Bcell-cycle%2Bpromoters.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.848%26rft_id%253Dinfo%253Apmid%252F21642992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.848&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21642992&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.18" data-doi="10.1146/annurev-biochem-051410-092902"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rinn</span>,  <span class="cit-name-given-names">J.L.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">H.Y.</span> <span class="cit-name-surname">Chang</span></span>, <span class="cit-article-title">Genome regulation by long noncoding RNAs</span>. <abbr class="cit-jnl-abbrev">Annu Rev Biochem</abbr>, <span class="cit-pub-date">2012</span>. <span class="cit-vol">81</span>: p. <span class="cit-fpage">145</span>–<span class="cit-lpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.aulast%253DRinn%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D81%26rft.spage%253D145%26rft.epage%253D166%26rft.atitle%253DGenome%2Bregulation%2Bby%2Blong%2Bnoncoding%2BRNAs.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-biochem-051410-092902%26rft_id%253Dinfo%253Apmid%252F22663078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-biochem-051410-092902&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22663078&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305765500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bawa</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Integrative Analysis of Normal Long Intergenic Non-Coding RNAs in Prostate Cancer</span>. <abbr class="cit-jnl-abbrev">PloS One</abbr>, <span class="cit-pub-date">2015</span>. <span class="cit-vol">10</span>(<span class="cit-issue">5</span>): p. <span class="cit-fpage">e0122143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPloS%2BOne%26rft.volume%253D10%26rft.spage%253De0122143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lanciotti RS</span>,  <span class="cit-name-given-names">L.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holodniy</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saavedra</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">del Carmen Castillo Signor</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-article-title">Phylogeny of Zika virus in Western Hemisphere</span>, <abbr class="cit-jnl-abbrev">2015 [letter]</abbr>. <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Contreras</span>,  <span class="cit-name-given-names">D.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Arumugaswami</span></span>, <span class="cit-article-title">Zika Virus Infectious Cell Culture System and the In Vitro Prophylactic Effect of Interferons</span>. <abbr class="cit-jnl-abbrev">Journal of Visualized Experiments</abbr>, <span class="cit-pub-date">2016</span>(<span class="cit-fpage">e54767</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.22" data-doi="10.1186/gb-2009-10-3-r25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Langmead</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</span>. <abbr class="cit-jnl-abbrev">Genome Biol</abbr>, <span class="cit-pub-date">2009</span>. <span class="cit-vol">10</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">R25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253DR25%26rft.epage%253DR25%26rft.atitle%253DUltrafast%2Band%2Bmemory-efficient%2Balignment%2Bof%2Bshort%2BDNA%2Bsequences%2Bto%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-3-r25%26rft_id%253Dinfo%253Apmid%252F19261174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-3-r25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19261174&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.23" data-doi="10.1186/s13059-014-0550-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Love</span>,  <span class="cit-name-given-names">M.I.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Huber</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Anders</span></span>, <span class="cit-article-title">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</span>. <abbr class="cit-jnl-abbrev">Genome Biol</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">15</span>(<span class="cit-issue">12</span>): p. <span class="cit-fpage">550</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D15%26rft.spage%253D550%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0550-8%26rft_id%253Dinfo%253Apmid%252F25516281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0550-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25516281&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.24" data-doi="10.2307/2346101"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>,  <span class="cit-name-given-names">Y.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Hochberg</span></span>, <span class="cit-article-title">Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing</span>. <abbr class="cit-jnl-abbrev">Journal of the Royal Statistical Society. Series B (Methodological)</abbr>, <span class="cit-pub-date">1995</span>. <span class="cit-vol">57</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">289</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BRoyal%2BStatistical%2BSociety.%2BSeries%2BB%2B%2528Methodological%2529%26rft.volume%253D57%26rft.spage%253D289%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2346101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2346101&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995QE45300017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.25" data-doi="10.1093/nar/gku1003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Szklarczyk</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">STRING v10: protein-protein interaction networks, integrated over the tree of life</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-pub-date">2015</span>. <span class="cit-vol">43</span>(<span class="cit-issue">Database issue</span>): p. <span class="cit-fpage">D447</span>–<span class="cit-lpage">52</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1003%26rft_id%253Dinfo%253Apmid%252F25352553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku1003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25352553&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.26" data-doi="10.1038/nbt1103"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rhodes</span>,  <span class="cit-name-given-names">D.R.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Probabilistic model of the human protein-protein interaction network</span>. <abbr class="cit-jnl-abbrev">Nat Biotechnol</abbr>, <span class="cit-pub-date">2005</span>. <span class="cit-vol">23</span>(<span class="cit-issue">8</span>): p. <span class="cit-fpage">951</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DRhodes%26rft.auinit1%253DD.%2BR.%26rft.volume%253D23%26rft.issue%253D8%26rft.spage%253D951%26rft.epage%253D959%26rft.atitle%253DProbabilistic%2Bmodel%2Bof%2Bthe%2Bhuman%2Bprotein-protein%2Binteraction%2Bnetwork.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt1103%26rft_id%253Dinfo%253Apmid%252F16082366%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt1103&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16082366&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231019900027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-confproc" id="cit-061788v1.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mittal</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Dissecting the expression dynamics of RNA-binding proteins in posttranscriptional regulatory networks</span>. <span class="cit-conf-name">Proc Natl Acad Sci</span> <span class="cit-conf-loc">U S A</span>, <span class="cit-conf-date">2009</span>. <span class="cit-vol">106</span>(<span class="cit-issue">48</span>): p. <span class="cit-fpage">20300</span>–<span class="cit-lpage">5</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">28.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-061788v1.28" data-doi="10.1093/bfgp/elq028"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kishore</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Luber</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Zavolan</span></span>, <span class="cit-article-title">Deciphering the role of RNA-binding proteins in the post-transcriptional control of gene expression</span>. <abbr class="cit-jnl-abbrev">Brief Funct Genomics</abbr>, <span class="cit-pub-date">2010</span>. <span class="cit-vol">9</span>(<span class="cit-issue">5-6</span>): p. <span class="cit-fpage">391</span>–<span class="cit-lpage">404</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief%2BFunct%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbfgp%252Felq028%26rft_id%253Dinfo%253Apmid%252F21127008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bfgp/elq028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21127008&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286675100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.29" data-doi="10.1016/j.cell.2010.03.009"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hafner</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Transcriptome-wide identification of RNA-binding protein and microRNA target sites by PAR-CLIP</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>, <span class="cit-pub-date">2010</span>. <span class="cit-vol">141</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">129</span>–<span class="cit-lpage">41</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHafner%26rft.auinit1%253DM.%26rft.volume%253D141%26rft.issue%253D1%26rft.spage%253D129%26rft.epage%253D141%26rft.atitle%253DTranscriptome-wide%2Bidentification%2Bof%2BRNA-binding%2Bprotein%2Band%2BmicroRNA%2Btarget%2Bsites%2Bby%2BPAR-CLIP.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2010.03.009%26rft_id%253Dinfo%253Apmid%252F20371350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2010.03.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20371350&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276211100018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.30" data-doi="10.1186/s13046-016-0297-z"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Long noncoding RNA NEAT1 promotes laryngeal squamous cell cancer through regulating miR-107/CDK6 pathway</span>. <abbr class="cit-jnl-abbrev">J Exp Clin Cancer Res</abbr>, <span class="cit-pub-date">2016</span>. <span class="cit-vol">35</span>: p. <span class="cit-fpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BExp%2BClin%2BCancer%2BRes%26rft.volume%253D35%26rft.spage%253D22%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13046-016-0297-z%26rft_id%253Dinfo%253Apmid%252F26822763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13046-016-0297-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26822763&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.31" data-doi="10.1371/journal.pgen.1003368"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tripathi</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Long noncoding RNA MALAT1 controls cell cycle progression by regulating the expression of oncogenic transcription factor B-MYB</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr>, <span class="cit-pub-date">2013</span>. <span class="cit-vol">9</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">e1003368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet%26rft.volume%253D9%26rft.spage%253De1003368%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003368%26rft_id%253Dinfo%253Apmid%252F23555285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003368&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23555285&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.32" data-doi="10.1093/nar/gkq1138"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Amaral</span>,  <span class="cit-name-given-names">P.P.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">lncRNAdb: a reference database for long noncoding RNAs</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-pub-date">2011</span>. <span class="cit-vol">39</span>(<span class="cit-issue">Database issue</span>): p. <span class="cit-fpage">D146</span>–<span class="cit-lpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq1138%26rft_id%253Dinfo%253Apmid%252F21112873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq1138&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21112873&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285831700026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.33" data-doi="10.1038/emboj.2010.199"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bernard</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">A long nuclear-retained non-coding RNA regulates synaptogenesis by modulating gene expression</span>. <abbr class="cit-jnl-abbrev">EMBO J</abbr>, <span class="cit-pub-date">2010</span>. <span class="cit-vol">29</span>(<span class="cit-issue">18</span>): p. <span class="cit-fpage">3082</span>–<span class="cit-lpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DBernard%26rft.auinit1%253DD.%26rft.volume%253D29%26rft.issue%253D18%26rft.spage%253D3082%26rft.epage%253D3093%26rft.atitle%253DA%2Blong%2Bnuclear-retained%2Bnon-coding%2BRNA%2Bregulates%2Bsynaptogenesis%2Bby%2Bmodulating%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Femboj.2010.199%26rft_id%253Dinfo%253Apmid%252F20729808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/emboj.2010.199&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20729808&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282991900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.34" data-doi="10.1186/1471-2164-8-39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hutchinson</span>,  <span class="cit-name-given-names">J.N.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">A screen for nuclear transcripts identifies two linked noncoding RNAs associated with SC35 splicing domains</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr>, <span class="cit-pub-date">2007</span>. <span class="cit-vol">8</span>: p. <span class="cit-fpage">39</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DHutchinson%26rft.auinit1%253DJ.%2BN.%26rft.volume%253D8%26rft.spage%253D39%26rft.epage%253D39%26rft.atitle%253DA%2Bscreen%2Bfor%2Bnuclear%2Btranscripts%2Bidentifies%2Btwo%2Blinked%2Bnoncoding%2BRNAs%2Bassociated%2Bwith%2BSC35%2Bsplicing%2Bdomains.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-8-39%26rft_id%253Dinfo%253Apmid%252F17270048%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-8-39&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17270048&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.35" data-doi="10.1091/mbc.E13-09-0558"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hirose</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">NEAT1 long noncoding RNA regulates transcription via protein sequestration within subnuclear bodies</span>. <abbr class="cit-jnl-abbrev">Mol Biol Cell</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">25</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">169</span>–<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E13-09-0558%26rft_id%253Dinfo%253Apmid%252F24173718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjI1LzEvMTY5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTYvMDcvMDEvMDYxNzg4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.36" data-doi="10.4161/rna.23547"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Naganuma</span>,  <span class="cit-name-given-names">T.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Hirose</span></span>, <span class="cit-article-title">Paraspeckle formation during the biogenesis of long noncoding RNAs</span>. <abbr class="cit-jnl-abbrev">RNA Biol</abbr>, <span class="cit-pub-date">2013</span>. <span class="cit-vol">10</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">456</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%2BBiol%26rft.volume%253D10%26rft.spage%253D456%26rft_id%253Dinfo%253Adoi%252F10.4161%252Frna.23547%26rft_id%253Dinfo%253Apmid%252F23324609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/rna.23547&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23324609&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316784400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zeng</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Inhibition of long non-coding RNA NEAT1 impairs myeloid differentiation in acute promyelocytic leukemia cells</span>. <abbr class="cit-jnl-abbrev">BMC Cancer</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">14</span>: p. <span class="cit-fpage">693</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BCancer%26rft.volume%253D14%26rft.spage%253D693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">38.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-061788v1.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Clinical implication of long non-coding RNA NEAT1 expression in hepatocellular carcinoma patients</span>. <abbr class="cit-jnl-abbrev">Int J Clin Exp Pathol</abbr>, <span class="cit-pub-date">2015</span>. <span class="cit-vol">8</span>(<span class="cit-issue">5</span>): p. <span class="cit-fpage">5395</span>–<span class="cit-lpage">402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt%2BJ%2BClin%2BExp%2BPathol%26rft.volume%253D8%26rft.spage%253D5395%26rft_id%253Dinfo%253Apmid%252F26191242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=26191242&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">39.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-061788v1.39" data-doi="10.1371/journal.pone.0025915"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gibb</span>,  <span class="cit-name-given-names">E.A.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Human cancer long non-coding RNA transcriptomes</span>. <abbr class="cit-jnl-abbrev">PloS One</abbr>, <span class="cit-pub-date">2011</span>. <span class="cit-vol">6</span>(<span class="cit-issue">10</span>): p. <span class="cit-fpage">e25915</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DGibb%26rft.auinit1%253DE.%2BA.%26rft.volume%253D6%26rft.issue%253D10%26rft.spage%253De25915%26rft.epage%253De25915%26rft.atitle%253DHuman%2Bcancer%2Blong%2Bnon-coding%2BRNA%2Btranscriptomes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0025915%26rft_id%253Dinfo%253Apmid%252F21991387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0025915&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21991387&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.40" data-doi="10.1261/rna.047332.114"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Standaert</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">The long noncoding RNA Neat1 is required for mammary gland development and lactation</span>. <abbr class="cit-jnl-abbrev">RNA</abbr>, <span class="cit-pub-date">2014</span>. <span class="cit-vol">20</span>(<span class="cit-issue">12</span>): p. <span class="cit-fpage">1844</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.047332.114%26rft_id%253Dinfo%253Apmid%252F25316907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjEwOiIyMC8xMi8xODQ0IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTYvMDcvMDEvMDYxNzg4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tang</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Zika Virus Infects Human Cortical Neural Progenitors and Attenuates Their Growth</span>. <abbr class="cit-jnl-abbrev">Cell Stem Cell</abbr>, <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.42" data-doi="10.1088/1478-3975/8/3/035008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lees</span>,  <span class="cit-name-given-names">J.G.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Systematic computational prediction of protein interaction networks</span>. <abbr class="cit-jnl-abbrev">Phys Biol</abbr>, <span class="cit-pub-date">2011</span>. <span class="cit-vol">8</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">035008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPhys%2BBiol%26rft.aulast%253DLees%26rft.auinit1%253DJ.%2BG.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D035008%26rft.epage%253D035008%26rft.atitle%253DSystematic%2Bcomputational%2Bprediction%2Bof%2Bprotein%2Binteraction%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1088%252F1478-3975%252F8%252F3%252F035008%26rft_id%253Dinfo%253Apmid%252F21572181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1088/1478-3975/8/3/035008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21572181&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">T.Y.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">A predicted protein-protein interaction network of the filamentous fungus Neurospora crassa</span>. <abbr class="cit-jnl-abbrev">Mol Biosyst</abbr>, <span class="cit-pub-date">2011</span>. <span class="cit-vol">7</span>(<span class="cit-issue">7</span>): p. <span class="cit-fpage">2278</span>–<span class="cit-lpage">85</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BBiosyst%26rft.aulast%253DWang%26rft.auinit1%253DT.%2BY.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253D2278%26rft.epage%253D2285%26rft.atitle%253DA%2Bpredicted%2Bprotein-protein%2Binteraction%2Bnetwork%2Bof%2Bthe%2Bfilamentous%2Bfungus%2BNeurospora%2Bcrassa.%26rft_id%253Dinfo%253Apmid%252F21584303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21584303&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.44" data-doi="10.1007/978-1-59745-033-1_6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Baroni</span>,  <span class="cit-name-given-names">T.E.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Advances in RIP-chip analysis: RNA-binding protein immunoprecipitation-microarray profiling</span>. <abbr class="cit-jnl-abbrev">Methods Mol Biol</abbr>, <span class="cit-pub-date">2008</span>. <span class="cit-vol">419</span>: p. <span class="cit-fpage">93</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DBaroni%26rft.auinit1%253DT.%2BE.%26rft.volume%253D419%26rft.spage%253D93%26rft.epage%253D108%26rft.atitle%253DAdvances%2Bin%2BRIP-chip%2Banalysis%2B%253A%2BRNA-binding%2Bprotein%2Bimmunoprecipitation-microarray%2Bprofiling.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-59745-033-1_6%26rft_id%253Dinfo%253Apmid%252F18369977%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-59745-033-1_6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18369977&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">45.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-061788v1.45" data-doi="10.1007/978-1-60327-563-7_2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Barkan</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-article-title">Genome-wide analysis of RNA-protein interactions in plants</span>. <abbr class="cit-jnl-abbrev">Methods Mol Biol</abbr>, <span class="cit-pub-date">2009</span>. <span class="cit-vol">553</span>: p. <span class="cit-fpage">13</span>–<span class="cit-lpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DBarkan%26rft.auinit1%253DA.%26rft.volume%253D553%26rft.spage%253D13%26rft.epage%253D37%26rft.atitle%253DGenome-wide%2Banalysis%2Bof%2BRNA-protein%2Binteractions%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-60327-563-7_2%26rft_id%253Dinfo%253Apmid%252F19588099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-60327-563-7_2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19588099&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271177400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.46" data-doi="10.5483/BMBRep.2009.42.3.125"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">M.Y.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hur</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Jeong</span></span>, <span class="cit-article-title">Emerging roles of RNA and RNA-binding protein network in cancer cells</span>. <abbr class="cit-jnl-abbrev">BMB Rep</abbr>, <span class="cit-pub-date">2009</span>. <span class="cit-vol">42</span>(<span class="cit-issue">3</span>): p. <span class="cit-fpage">125</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMB%2BRep%26rft.aulast%253DKim%26rft.auinit1%253DM.%2BY.%26rft.volume%253D42%26rft.issue%253D3%26rft.spage%253D125%26rft.epage%253D130%26rft.atitle%253DEmerging%2Broles%2Bof%2BRNA%2Band%2BRNA-binding%2Bprotein%2Bnetwork%2Bin%2Bcancer%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.5483%252FBMBRep.2009.42.3.125%26rft_id%253Dinfo%253Apmid%252F19335997%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5483/BMBRep.2009.42.3.125&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19335997&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264839200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.47" data-doi="10.1371/journal.pbio.0060255"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hogan</span>,  <span class="cit-name-given-names">D.J.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Diverse RNA-binding proteins interact with functionally related sets of RNAs, suggesting an extensive regulatory system</span>. <abbr class="cit-jnl-abbrev">PLoS Biol</abbr>, <span class="cit-pub-date">2008</span>. <span class="cit-vol">6</span>(<span class="cit-issue">10</span>): p. <span class="cit-fpage">e255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DHogan%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D6%26rft.issue%253D10%26rft.spage%253De255%26rft.epage%253De255%26rft.atitle%253DDiverse%2BRNA-binding%2Bproteins%2Binteract%2Bwith%2Bfunctionally%2Brelated%2Bsets%2Bof%2BRNAs%252C%2Bsuggesting%2Ban%2Bextensive%2Bregulatory%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0060255%26rft_id%253Dinfo%253Apmid%252F18959479%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0060255&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18959479&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">48.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-061788v1.48" data-doi="10.1038/nrg2673"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Licatalosi</span>,  <span class="cit-name-given-names">D.D.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">R.B.</span> <span class="cit-name-surname">Darnell</span></span>, <span class="cit-article-title">RNA processing and its regulation: global insights into biological networks</span>. <abbr class="cit-jnl-abbrev">Nat Rev Genet</abbr>, <span class="cit-pub-date">2010</span>. <span class="cit-vol">11</span>(<span class="cit-issue">1</span>): p. <span class="cit-fpage">75</span>–<span class="cit-lpage">87</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DLicatalosi%26rft.auinit1%253DD.%2BD.%26rft.volume%253D11%26rft.issue%253D1%26rft.spage%253D75%26rft.epage%253D87%26rft.atitle%253DRNA%2Bprocessing%2Band%2Bits%2Bregulation%253A%2Bglobal%2Binsights%2Binto%2Bbiological%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2673%26rft_id%253Dinfo%253Apmid%252F20019688%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2673&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20019688&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272867700014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">49.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-061788v1.49" data-doi="10.4161/rna.8.2.14991"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sola</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">RNA-RNA and RNA-protein interactions in coronavirus replication and transcription</span>. <abbr class="cit-jnl-abbrev">RNA Biol</abbr>, <span class="cit-pub-date">2011</span>. <span class="cit-vol">8</span>(<span class="cit-issue">2</span>): p. <span class="cit-fpage">237</span>–<span class="cit-lpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DRNA%2BBiol%26rft.aulast%253DSola%26rft.auinit1%253DI.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253D237%26rft.epage%253D248%26rft.atitle%253DRNA-RNA%2Band%2BRNA-protein%2Binteractions%2Bin%2Bcoronavirus%2Breplication%2Band%2Btranscription.%26rft_id%253Dinfo%253Adoi%252F10.4161%252Frna.8.2.14991%26rft_id%253Dinfo%253Apmid%252F21378501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/rna.8.2.14991&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21378501&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289934300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-061788v1.50" data-doi="10.4161/rna.8.2.15391"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Z.</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">P.D.</span> <span class="cit-name-surname">Nagy</span></span>, <span class="cit-article-title">Diverse roles of host RNA binding proteins in RNA virus replication</span>. <abbr class="cit-jnl-abbrev">RNA Biol</abbr>, <span class="cit-pub-date">2011</span>. <span class="cit-vol">8</span>(<span class="cit-issue">2</span>): p. <span class="cit-fpage">305</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DRNA%2BBiol%26rft.aulast%253DLi%26rft.auinit1%253DZ.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253D305%26rft.epage%253D315%26rft.atitle%253DDiverse%2Broles%2Bof%2Bhost%2BRNA%2Bbinding%2Bproteins%2Bin%2BRNA%2Bvirus%2Breplication.%26rft_id%253Dinfo%253Adoi%252F10.4161%252Frna.8.2.15391%26rft_id%253Dinfo%253Apmid%252F21505273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/rna.8.2.15391&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21505273&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F01%2F061788.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289934300017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/061788v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F061788v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/061770v1" title="Forward genetic screen of human transposase genomic rearrangements" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/036897v2" title="A high-quality reference panel reveals the complexity and distribution of structural genome changes in a human population" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;July 01, 2016.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/061788v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/061788v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/061788v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/061788v1" class="active" data-icon-position="" data-hide-link-title="0">Dysregulation of Long Non-coding RNA (lncRNA) Genes and Predicted lncRNA-Protein Interactions during Zika Virus Infection</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/16978" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-aonEL3WvNFO_PwqBCx8_GTa8928ENeKL8jb-6Yo5a6I" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="16978" id="node16978--2" data-pisa="biorxiv;061788v1" data-pisa-master="biorxiv;061788" data-seqnum="16978" data-apath="/biorxiv/early/2016/07/01/061788.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Dysregulation of Long Non-coding RNA (lncRNA) Genes and Predicted lncRNA-Protein Interactions during Zika Virus Infection</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Arunachalam</span> <span class="nlm-surname">Ramaiah</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Deisy</span> <span class="nlm-surname">Contreras</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Vineela</span> <span class="nlm-surname">Gangalapudi</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Masumi</span> <span class="nlm-surname">Sameer Padhye</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Jie</span> <span class="nlm-surname">Tang</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Vaithilingaraja</span> <span class="nlm-surname">Arumugaswami</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">061788; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/061788 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/061788v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/061788v1&amp;title=Dysregulation%20of%20Long%20Non-coding%20RNA%20%28lncRNA%29%20Genes%20and%20Predicted%20lncRNA-Protein%20Interactions%20during%20Zika%20Virus%20Infection" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/061788v1&amp;title=Dysregulation%20of%20Long%20Non-coding%20RNA%20%28lncRNA%29%20Genes%20and%20Predicted%20lncRNA-Protein%20Interactions%20during%20Zika%20Virus%20Infection" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/061788v1&amp;text=Dysregulation%20of%20Long%20Non-coding%20RNA%20%28lncRNA%29%20Genes%20and%20Predicted%20lncRNA-Protein%20Interactions%20during%20Zika%20Virus%20Infection" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/061788v1&amp;title=Dysregulation%20of%20Long%20Non-coding%20RNA%20%28lncRNA%29%20Genes%20and%20Predicted%20lncRNA-Protein%20Interactions%20during%20Zika%20Virus%20Infection" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/061788v1&amp;t=Dysregulation%20of%20Long%20Non-coding%20RNA%20%28lncRNA%29%20Genes%20and%20Predicted%20lncRNA-Protein%20Interactions%20during%20Zika%20Virus%20Infection" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/061788v1&amp;title=Dysregulation%20of%20Long%20Non-coding%20RNA%20%28lncRNA%29%20Genes%20and%20Predicted%20lncRNA-Protein%20Interactions%20during%20Zika%20Virus%20Infection" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/061788v1&amp;title=Dysregulation%20of%20Long%20Non-coding%20RNA%20%28lncRNA%29%20Genes%20and%20Predicted%20lncRNA-Protein%20Interactions%20during%20Zika%20Virus%20Infection" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="16978" id="node16978--3" data-pisa="biorxiv;061788v1" data-pisa-master="biorxiv;061788" data-seqnum="16978" data-apath="/biorxiv/early/2016/07/01/061788.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Dysregulation of Long Non-coding RNA (lncRNA) Genes and Predicted lncRNA-Protein Interactions during Zika Virus Infection</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Arunachalam</span> <span class="nlm-surname">Ramaiah</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Deisy</span> <span class="nlm-surname">Contreras</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Vineela</span> <span class="nlm-surname">Gangalapudi</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Masumi</span> <span class="nlm-surname">Sameer Padhye</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Jie</span> <span class="nlm-surname">Tang</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Vaithilingaraja</span> <span class="nlm-surname">Arumugaswami</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">061788; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/061788 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Dysregulation%20of%20Long%20Non-coding%20RNA%20%28lncRNA%29%20Genes%20and%20Predicted%20lncRNA-Protein%20Interactions%20during%20Zika%20Virus%20Infection&amp;rft.volume&amp;rft.issue&amp;rft.date=2016-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Ramaiah%2C+Arunachalam&amp;rft.au=Contreras%2C+Deisy&amp;rft.au=Gangalapudi%2C+Vineela&amp;rft.au=Sameer+Padhye%2C+Masumi&amp;rft.au=Tang%2C+Jie&amp;rft.au=Arumugaswami%2C+Vaithilingaraja"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/16978/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/16978/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/16978/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/16978/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/16978/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/16978/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/16978/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/16978/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/16978/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/16978/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/16978/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/16978/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/061788v1&amp;count=horizontal&amp;via=&amp;text=Dysregulation%20of%20Long%20Non-coding%20RNA%20%28lncRNA%29%20Genes%20and%20Predicted%20lncRNA-Protein%20Interactions%20during%20Zika%20Virus%20Infection&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/061788v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/061788v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/061788v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">Microbiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20277)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
