<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Virus Res</span><span class="iso-abbrev" title="iso-abbrev">Virus Res</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Virus Research</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">0168-1702</span><span class="issn-epub" tagx="issn" title="issn-epub">1872-7492</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Elsevier B.V.</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7229464">7229464</a></span><span class="publisher-id" title="publisher-id">S0168-1702(19)30677-X</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1016/j.virusres.2020.198024">10.1016/j.virusres.2020.198024</a></span><span class="publisher-id" title="publisher-id">198024</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Swine enteric alphacoronavirus (swine acute diarrhea syndrome coronavirus): An update three years after its discovery</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Yong-Le</span></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">Jia-Qi</span></span></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Yao-Wei</span></span><span class="email" tagx="email" title="email">yhuang@zju.edu.cn</span><a href="#cor0005">*</a></span><span class="citation_author_institution" id="aff0005">[], </span></div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">â�Ž</span>Corresponding author. <span class="email" tagx="email" title="email">yhuang@zju.edu.cn</span></div> </div><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2020-5-5</span></span><span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2020-8</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-5-5</span></span><span class="volume" tagx="volume" title="volume">285</span><span class="fpage" tagx="fpage" title="fpage">198024</span><span class="lpage" tagx="lpage" title="lpage">198024</span><span class="history" title="history"><span class="received" title="received">received: 2020-3-31</span><span class="rev-recd" title="rev-recd">rev-recd: 2020-5-12</span><span class="accepted" title="accepted">accepted: 2020-5-13</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Elsevier B.V.</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">Since January 2020 Elsevier has created a COVID-19 resource centre with free information in English and Mandarin on the novel coronavirus COVID-19. The COVID-19 resource centre is hosted on Elsevier Connect, the company's public news and information website. Elsevier hereby grants permission to make all its COVID-19-related research that is available on the COVID-19 resource centre - including this research content - immediately available in PubMed Central and other publicly funded repositories, such as the WHO COVID database with rights for unrestricted research re-use and analyses in any form or by any means with acknowledgement of the original source. These permissions are granted for free by Elsevier for as long as the COVID-19 resource centre remains active.</span></span></div> <div class="abstract" title="abstract"> <div class="title" tagx="title" title="title">
Highlights</div> <p> </p><ul> <li><span class="label" tagx="label" title="label">â€¢</span><p id="par0005">A summary of the research progress in SeACoV (SADS-CoV) from 2017 to 2020.</p> </li> <li><span class="label" tagx="label" title="label">â€¢</span><p id="par0010">Bat-derived SeACoV was most recently recognized prior to SARS-CoV-2 associated with COVID-19.</p> </li> <li><span class="label" tagx="label" title="label">â€¢</span><p id="par0015">Focusing on the etiology, epidemiology, evolutionary perspective, potential for interspecies transmission, pathogenesis and diagnosis.</p> </li> </ul> <p /> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Discovered in 2017, swine enteric alphacoronavirus (SeACoV), also known as swine acute diarrhea syndrome coronavirus (SADS-CoV) or porcine enteric alphacoronavirus (PEAV), is the fifth porcine CoV identified in diarrheal piglets. The presumed name â€œSADS-CoVâ€� may not be appropriate since current studies have not provided strong evidence for high pathogenicity of the virus. SeACoV was the most recently recognized CoV of potential bat origin prior to the novel human <i>severe acute respiratory syndrome CoV 2</i> (SARS-CoV-2), associated with the pandemic CoV disease 2019 (COVID-19). Although SeACoV is recognized as a regional epizootic virus currently, it possesses the most extensive cell species tropism <i>in vitro</i> among known CoVs. This review summarizes the emergence of SeACoV and updates the research progress made from 2017 to early 2020, mainly focusing on the etiology, epidemiology, evolutionary perspective, potential for interspecies transmission, pathogenesis and diagnosis.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
Swine enteric alphacoronavirus (SeACoV)</div> <div class="kwd" title="kwd">
Swine acute diarrhea syndrome coronavirus (SADS-CoV)</div> <div class="kwd" title="kwd">
Etiology</div> <div class="kwd" title="kwd">
Pathogenicity</div> <div class="kwd" title="kwd">
Transmission</div> <div class="kwd" title="kwd">
Diagnosis</div> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"><span class="label" tagx="label" title="label">1</span><div class="title" tagx="title" title="title">
Introduction</div> <p id="par0020">Coronaviruses (CoVs; order <i>Nidovirales</i>, family <i>Coronaviridae</i>, subfamily <i>Orthocoronavirinae</i>) are single-stranded, positive-sense, enveloped RNA viruses that cause subclinical, mild or lethal respiratory and gastrointestinal diseases in humans and animals (<a href="#bib0010">de Groot et al., 2011</a>). To date, there have been five different species of CoV identified which naturally infect pigs (<a href="#bib0140">Wang et al., 2019</a>). These include three alphacoronaviruses: transmissible gastroenteritis virus (TGEV) together with its variant porcine respiratory virus (PRCV) (<a href="#bib0110">Pensaert et al., 1986</a>), porcine epidemic diarrhea virus (PEDV) (<a href="#bib0115">Pensaert and de Bouck, 1978</a>), and swine enteric alphacoronavirus (SeACoV) (<a href="#bib0105">Pan et al., 2017</a>); one betacoronavirus: porcine hemagglutinating encephalomyelitis virus (PHEV) (<a href="#bib0025">Greig et al., 1962</a>); and porcine deltacoronavirus (PDCoV) (<a href="#bib0155">Woo et al., 2012</a>). TGEV, PEDV, PDCoV and SeACoV cause swine enteric coronavirus disease (SECD), characterized by acute diarrhea in neonatal piglets leading to moderate or high mortality and significant economic losses for the pork industry, in Asia since 2010 (<a href="#bib0100">Pan et al., 2012</a>) and North America since 2013 (<a href="#bib0035">Huang et al., 2013</a>; <a href="#bib0140">Wang et al., 2019</a>).</p> <p id="par0025">SeACoV was discovered in Guangdong province, China in 2017. Initially, outbreaks of severe diarrhea of suckling piglets occurred in four swine herds in a mountainous area of northern Guangdong (<a href="#fig0005">Fig. 1</a> ). Our laboratory was the first group to isolate and characterize this new virus in Vero cells, and determined its genome to be closely related to the species <i>Rhinolophus bat coronavirus HKU2</i> in the subgenus <i>Rhinacovirus</i> (<a href="#bib0105">Pan et al., 2017</a>). We also demonstrated that SeACoV was able to infect pigs orally (<a href="#bib0105">Pan et al., 2017</a>). SeACoV is known by other names, such as porcine enteric alphacoronavirus (PEAV) (<a href="#bib0020">Gong et al., 2017</a>), or swine acute diarrhea syndrome (SADS)-CoV (<a href="#bib0200">Zhou et al., 2018b</a>). The latter nomenclature is attractive, likely due to name similarity to SARS-CoV (<i>severe acute respiratory syndrome CoV</i> of humans) which may make it easier to gain media attention. However, since the actual pathogenicity of this new virus is still controversial based on current evidence (see main text below), the presumed name â€œSADSâ€� (also easily confused with â€œSARSâ€�) may not be appropriate in the future in our opinion. Therefore, SeACoV is the name we will use to refer to this new virus in this review. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <p><b>Map showing geographic location of farms with SeACoV outbreaks.</b> SeACoV was first isolated from clinically sick animals in commercial pig herds of Guangdong province of China in 2017. There were reports of reemergence in Fujian in 2018 and again in Guangdong in 2019. The right panel showed the outbreak of newborn-piglet diarrhea first occurred in four commercial pig farms (marked by yellow stars) in northern Guangdong in 2017.</p>   </div> <p /> <p id="par0030">Very little is known about the molecular biology of SeACoV. This article will comprehensively review the current knowledge of SeACoV with regards to its origin, etiology, epidemiology, evolutionary perspective, potential for interspecies transmission, pathogenesis and diagnosis during the period from 2017 to early 2020.</p> </div> <div class="etiology" title="sec"><span class="label" tagx="label" title="label">2</span><div class="title" tagx="title" title="title">
Etiology</div> <div class="virionandgenomestructure" title="sec"><span class="label" tagx="label" title="label">2.1</span><div class="title" tagx="title" title="title">
Virion and genome structure</div> <p id="par0035">Like the other CoVs, morphological observations by electron microscopy have revealed that SeACoV has the crown-like characteristics typical of the spike (S) protein distribution on the surface of the viral envelope (<a href="#bib0105">Pan et al., 2017</a>; <a href="#bib0165">Yang et al., 2019a</a>). SeACoV particles are round with a diameter of 100â€"120 nm (<a href="#fig0010">Fig. 2</a> A), and are covered with dimly visible trimers of S protein (<a href="#fig0010">Fig. 2</a>A and B). Bat enteric alphacoronavirus HKU2 strains were first reported from Guangdong province and Hong Kong in 2004 and 2006 during the search for the source of SARS-CoV (<a href="#bib0050">Lau et al., 2007</a>). The prototype SeACoV (CH/GD-01/2017 strain) shares 94.9 % nucleotide (nt) sequence identity with the four bat HKU2 strains (<a href="#bib0105">Pan et al., 2017</a>). The SeACoV genome is 27,155 nt long with a 5Ê¹ cap and a 3Ê¹ polyadenylated tail, and contains a 5Ê¹ untranslated region (UTR) followed by nine open reading frames (ORFs) and a 3Ê¹ UTR (<a href="#fig0010">Fig. 2</a>C). RNA synthesis in SeACoV is carried out by a replicase-transcriptase composed of 16 nonstructural proteins (Nsp1-16) encoded by ORF1a and ORF1b, which are located in the 5Ê¹ two-thirds of the genome (<a href="#bib0105">Pan et al., 2017</a>; <a href="#bib0200">Zhou et al., 2018b</a>). The 3Ê¹ third harbors ORFs that encode four structural proteins [S; envelope (E); membrane (M); nucleocapsid (N)], an accessory ORF3 between S and E, and two overlapping ORFs (NS7a and NS7b) following the N gene (<a href="#bib0165">Yang et al., 2019a</a>; <a href="#bib0200">Zhou et al., 2018b</a>) (<a href="#fig0010">Fig. 2</a>B and C). These structural and accessory genes are expressed from six subgenomic mRNAs including a bicistronic mRNA containing the accessory NS7a and NS7b genes (<a href="#fig0010">Fig. 2</a>D), which have been experimentally confirmed in SeACoV-infected cells (<a href="#bib0165">Yang et al., 2019a</a>). The leader-body junction sequences of these subgenomic mRNAs are identical to the leader core sequence AACTAAA (<a href="#bib0165">Yang et al., 2019a</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <p><b>Schematic representations of SeACoV genome organization and virion structure.</b> (A) Electron micrograph of a purified SeACoV virus particle, clearly showing the typical viral surface projections of S protein. The scale bar represents 100 nm. (B) Schematic diagram of SeACoV virion structure with color-coded protein components. (C) The structure of SeACoV genomic RNA (27,155 bp) is shown at the top with 5Ê¹ and 3Ê¹ UTRs. ORF1a and ORF1b are co-translationally or post-translationally processed to encode a replicase-transcriptase complex composed of 16 nonstructural proteins (Nsp1-16). Four structural proteins [spike (S); envelope (E); membrane (M); and nucleocapsid (N)] are encoded along with three accessory proteins (ORF3, NS7a and NS7b). (D) Genomic and subgenomic mRNAs containing the leader-body junction sites (LS) are shown with colors corresponding to the genome structure.</p>   </div> <p /> </div> <div class="viralproteins" title="sec"><span class="label" tagx="label" title="label">2.2</span><div class="title" tagx="title" title="title">
Viral proteins</div> <p id="par0040">Among the four viral structural proteins, the S glycoprotein of SeACoV is the major protein involved in the viral entry <i>via</i> cellular receptor binding, and it induces host immune responses through the induction of neutralizing antibodies. The SeACoV S protein shares 86.4 % amino acid (aa) identity with the bat HKU2/GD430 strain, supporting a common ancestry (<a href="#bib0105">Pan et al., 2017</a>). Interestingly, SeACoV and HKU2 possess unique S genes that are closely related to the betacoronaviruses (<a href="#bib0050">Lau et al., 2007</a>; <a href="#bib0105">Pan et al., 2017</a>). Most recently, Yu et al reported cryo-EM structures for HKU2 and SeACoV S protein trimers, which are available in the open access preprint repository (<a href="#bib0175">Yu et al., 2020</a>). The trimeric structures of HKU2 and SeACoV S proteins (resolutions of 2.38 Ã… and 2.83 Ã…, respectively) are very similar, with differences mainly found in the N- and C-terminal domains of the S1 subunit, which is responsible for cell attachment and receptor binding. Both the HKU2 and SeACoV S protein carboxy-terminal domain (CTD) structures have a single-layer core consisting of a twisted, five-stranded antiparallel Î² sheet similar to the betacoronaviruses (<a href="#bib0175">Yu et al., 2020</a>). This suggests recombination between alpha- and betacoronavirus S glycoproteins, and reflects the evolutionary origin of the HKU2/SeACoV S protein at a structural level. Although the S2 subunit (which mediates membrane fusion) is also conserved, the conformation of the region after the fusion peptide of the HKU2-CoV and SeACoV S2 subunits are different from other CoVs. These results provide structural evidence of the close evolutionary relationship between HKU2-related CoV and betacoronavirus S proteins, and provide new insights into CoV evolution and cross-species transmission (<a href="#bib0175">Yu et al., 2020</a>).</p> <p id="par0045">M and E proteins are major components of the viral envelope and are required for the viral assembly process (<a href="#bib0015">de Haan et al., 2000</a>). Apart from that, the N protein is the most highly conserved structural protein among CoVs, and plays an important role in the packaging of virus particles <i>via</i> binding to the viral genomic RNA (<a href="#bib0085">Nelson et al., 2000</a>). Specific polyclonal antibodies (pAbs) are usually generated based on viral structural proteins, and immunofluorescence assay using pAbs against N, M and S1 proteins have been used to detect SeACoV infection in cultured cells. Anti-N and anti-M pAbs have also been used for specific detection of SeACoV infection with expected molecular size (42- and 25-KDa, respectively) by western blot (<a href="#bib0165">Yang et al., 2019a</a>) or immunohistochemistry (<a href="#bib0170">Yang et al., 2019b</a>; <a href="#bib0200">Zhou et al., 2018b</a>). In addition, rabbit antiserum against SeACoV Nsp3 acidic domain has been validated for use in the time course analysis of viral protein expression using confocal images, which detected the Nsp3 expression as early as 4 h post-infection in SeACoV-infected cells (<a href="#bib0165">Yang et al., 2019a</a>). Accessory proteins play an important role in the immune modulation and viral pathogenesis of CoVs (<a href="#bib0075">Liu et al., 2014</a>). Three accessory proteins (ORF3, NS7a and NS7b) have been identified within the SeACoV genome based on analysis of leader-body junctions in specific subgenomic mRNAs, though their functions remain to be confirmed experimentally (<a href="#bib0165">Yang et al., 2019a</a>). Similar to the other CoVs like PEDV and TGEV (<a href="#bib0040">Ji et al., 2018</a>; <a href="#bib0095">Ortego et al., 2003</a>), ORF3 is dispensable for SeACoV infection <i>in vitro</i> since replacement with green fluorescent protein (GFP) in this position does not affect recovery of the recombinant virus (unpublished data).</p> </div> <div class="cellandspeciestropism" title="sec"><span class="label" tagx="label" title="label">2.3</span><div class="title" tagx="title" title="title">
Cell and species tropism</div> <p id="par0050">SeACoV was first isolated from the feces of piglets with severe watery diarrhea and high rates of mortality (<a href="#bib0105">Pan et al., 2017</a>). Among the small intestine, the jejunum is the major target of SeACoV infection <i>in vivo</i>. Vero cells are routinely used for the isolation of novel CoVs; they were used to isolate SeACoV initially (<a href="#bib0105">Pan et al., 2017</a>). Virus-positive samples were inoculated and blind-passaged on Vero cell lines plus exogenous trypsin, with cytopathic effect characterized by syncytia formation (<a href="#bib0105">Pan et al., 2017</a>). A single-cycle of replication by SeACoV in Vero cells takes approximately 4â€"6 h (<a href="#bib0165">Yang et al., 2019a</a>).</p> <p id="par0055">Furthermore, cell lines originating from various tissues of humans and several animal species including bats, mice, rats, gerbils, hamsters, pigs, chickens and nonhuman primates, were tested for viral susceptibility and species tropism. Twenty-one of the 24 tested cell lines showed significant susceptibility to SeACoV infection, defined by efficient viral antigen expression and increased viral RNA titers (<a href="#bib0170">Yang et al., 2019b</a>). Notably, SeACoV infects both primary and passaged cell lines of rodent origin, suggesting that rodents may be susceptible to SeACoV infection. In most cases, exogenous trypsin enhanced the propagation of SeACoV in susceptible cell lines. HeLa cells showed the highest susceptibility based on viral load detected in culture supernatants by quantitative RT-PCR (<a href="#bib0170">Yang et al., 2019b</a>). This study revealed a remarkably broad spectrum of SeACoV tropism <i>in vitro</i>, and suggested exogenous trypsin treatment and endogenous protease on the cell membrane may activate the fusion of SeACoV S protein for efficient cell entry and release (<a href="#bib0170">Yang et al., 2019b</a>).</p> <p id="par0060">CoVs are known to utilize different primary cellular receptors to gain entry into host cells, including angiotensin converting enzyme 2 (ACE2) for SARS-CoV and SARS-CoV-2, dipeptidyl peptidase 4 (DPP4) for Middle East respiratory syndrome CoV (MERS-CoV), aminopeptidase N (APN) for TGEV and PDCoV and mouse carcinoembryonic antigen-related cell adhesion molecule 1a (mCEACAM1a) for mouse hepatitis virus (MHV) (<a href="#bib0060">Li et al., 2003</a>; <a href="#bib0120">Raj et al., 2013</a>; <a href="#bib0130">Wang et al., 2018a</a>; <a href="#bib0150">Williams et al., 1991</a>; <a href="#bib0205">Zhou et al., 2020</a>). However, it has been demonstrated that SeACoV does not utilize any of the known CoV protein receptors for cellular entry (<a href="#bib0170">Yang et al., 2019b</a>; <a href="#bib0200">Zhou et al., 2018b</a>). Though the receptor for SeACoV has not yet been identified, the results have revealed the most extensive cell tropism <i>in vitro</i> among known CoVs, implying that the functional receptor(s) for SeACoV is likely to be a very common molecule (<a href="#bib0170">Yang et al., 2019b</a>). Receptor discovery is a critical direction for future research.</p> </div> <div class="reversegenetics" title="sec"><span class="label" tagx="label" title="label">2.4</span><div class="title" tagx="title" title="title">
Reverse genetics</div> <p id="par0065">Recovery of infectious virus from plasmids or RNA transcripts encoding cloned cDNAs corresponding to the viral genome is one of the most important experimental platforms for CoV research (<a href="#bib0005">Almazan et al., 2014</a>). Recently, our lab developed a DNA-launched reverse genetics system for SeACoV (the infectious clone â€œpSEAâ€�) (<a href="#bib0165">Yang et al., 2019a</a>). The full-length consensus cDNA sequence of SeACoV (27,155 nt) was assembled and inserted into a modified bacterial artificial chromosome (BAC) backbone plasmid that contains a cytomegalovirus promoter, a hepatitis delta virus ribozyme sequence and a bovine growth hormone polyadenylation sequence and terminator. In addition, two silent mutations (A24222T and G24223C) in ORF3 were introduced in pSEA as a genetic marker to differentiate the parental virus. The recombinant SeACoV with the genetic marker can be recovered by direct transfection of cells with pSEA and a helper plasmid expressing the N gene, with similar growth kinetics to the parental virus (<a href="#bib0165">Yang et al., 2019a</a>). This plasmid-based system was applied to determine whether a SeACoV non-susceptible cell line, MDCK (canine kidney cells), could confer SeACoV replication competency by transfection of pSEA (<a href="#bib0170">Yang et al., 2019b</a>). By introducing desired mutations into viral structural and nonstructural genes, the SeACoV reverse genetics system will facilitate functional analysis of replication and packaging signals, and can be exploited to engineer recombinant SeACoV for design of attenuated vaccines. Most recently, a reporter virus expressing GFP by replacement of ORF3 was generated to track SeACoV in cell culture based on this system in our lab (unpublished data), which will also allow for high-throughout screening of antivirals in the future.</p> </div> </div> <div class="molecularepidemiology" title="sec"><span class="label" tagx="label" title="label">3</span><div class="title" tagx="title" title="title">
Molecular epidemiology</div> <div class="theemergenceofseacovinguangdong,2017" title="sec"><span class="label" tagx="label" title="label">3.1</span><div class="title" tagx="title" title="title">
The emergence of SeACoV in Guangdong, 2017</div> <p id="par0070">Beginning from February 2017, a remarkable outbreak of diarrhea in newborn piglets occurred at four pig farms located in Guangdong province of southern China (<a href="#fig0005">Fig. 1</a>). The clinical signs of affected pigs were similar to other swine enteric CoVs, characterized by acute vomiting and watery diarrhea. The original diarrheal samples analyzed by RT-PCR were negative for PEDV, TGEV, PDCoV and PHEV, and the pigs had been vaccinated against PEDV. Subsequently a bat-HKU2 origin SeACoV was detected by a pan-CoV RT-PCR assay, and was responsible for a large-scale outbreak that killed 24,693 piglets in commercial farms (<a href="#bib0020">Gong et al., 2017</a>; <a href="#bib0105">Pan et al., 2017</a>; <a href="#bib0200">Zhou et al., 2018b</a>). The four affected farms were distributed around the foot of a mountain (<a href="#fig0005">Fig. 1</a>), and there were reports of bat populations nearby.</p> <p id="par0075">During early outbreaks, the SeACoV genome sequences identified from three independent groups were 99.8â€"100 % identical to each other (<a href="#fig0015">Fig. 3</a> ). These SeACoV strains from the same source include â€œPEAVâ€� GDS04 and GDS04-P12 (GenBank accession nos. MF167434 and MH697599) (<a href="#bib0020">Gong et al., 2017</a>; <a href="#bib0160">Xu et al., 2019</a>), â€œSeACoVâ€� CH/GD-01/2017 (MF370205) (<a href="#bib0105">Pan et al., 2017</a>), and â€œSADS-CoVâ€� CN/GDWT/2017 (MG557844), Farm-A to Farm-D (MF094681-MF094684) (<a href="#bib0200">Zhou et al., 2018b</a>). Except for an extra 7 nt (ACATGGG) at the 5â€²-terminus of the genome, no insertions/deletions were found among the five SADS-CoV strains in comparison with SeACoV and its cell adapted strain P10 (MK977618) which is 27,155 nt in genomic length (<a href="#fig0010">Fig. 2</a>C). Since SeACoV-p10 could be rescued from the infectious cDNA clone pSEA (<a href="#bib0165">Yang et al., 2019a</a>), we suggest that MK977618 may represent a reference strain for sequence comparison. MK977618 is 6 nt longer than the prototype bat CoV HKU2/GD430 strain (27,149 nt), including a 3-nt (TTG) insertion at nt 4,555-4,557 in the nsp3 region, a 6-nt (GGCCTC) insertion at nt 20,508-20,513 in the S gene, and a 3-nt (GTT) deletion at nt 24,780- 24,781 in the M gene. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3</span></a></div>   <p><b>Phylogenetic analysis of the species <i>Rhinolophus bat coronavirus HKU2</i> and other representative coronaviruses based on nucleotide sequences of the complete genomes.</b> The tree was constructed by the neighbor-joining method using the MEGA X 10.1 software. Bootstrap values are indicated for each node from 1000 resamplings. The names of the viruses and strains as well as their GenBank accession numbers are depicted. The black solid circle indicates the reference sequence SeACoV p10 corresponding to the infectious clone capable of recovering the virus.</p>   </div> <p /> <p id="par0080">Subsequently, a retrospective investigation of SeACoV at 45 pig farms from Guangdong province based on 236 diarrhea samples showed that SeACoV had emerged in Guangdong at least as early as August 2016 (<a href="#bib0195">Zhou et al., 2018a</a>). This study also determined two additional genomic sequences CN/GDGL/2017 and CN/GDLS/2017 (MG605090 and MG605091) that shared nearly 100 % nt identity with the sequence CN/GDWT/2017 (MG557844) based on our analysis (<a href="#fig0015">Fig. 3</a>).</p> <p id="par0085">Since 2010, PEDV variant strains associated with large-scale outbreaks of diarrhea in neonatal piglets have emerged in Mainland China (<a href="#bib0035">Huang et al., 2013</a>; <a href="#bib0100">Pan et al., 2012</a>). The successive discovery of PDCoV in 2015 and SeACoV in 2017 has taught us that the cause of piglet diarrhea in China may not be as simple as PEDV or TGEV. At present, there remains a lack of nationwide epidemiological investigation of newly emerging porcine enteric CoVs beyond PEDV and TGEV, and thus comprehensive, systematic research is urgently needed.</p> </div> <div class="currentsituation" title="sec"><span class="label" tagx="label" title="label">3.2</span><div class="title" tagx="title" title="title">
Current situation</div> <p id="par0090">From May 2017 to January 2019, there were no new SeACoV cases arising in pig herds in Guangdong beyond the first outbreaks. In 2018, a new SeACoV strain (CH/FJWT/2018) was identified in pig stool and small intestine samples from seven pig farms in Fujian outside the Guangdong province (<a href="#fig0005">Fig. 1</a>) (<a href="#bib0055">Li et al., 2018a</a>). The complete genome (GenBank accession no. MH615810) was 27,168 nt in length, and there were a total of 13 nt insertions compared with the reference strain SeACoV-p10 (99.5 % nt identity) (<a href="#fig0015">Fig. 3</a>), including an extra seven nt (ACATGGG) at the 5â€²-end, a 3-nt (GGC) insertion at nt 22,473-22,474 in the S gene, and a 3-nt (GTA) insertion at nt 24,780- 24,781 in the M gene.</p> <p id="par0095">SeACoV reemerged in pig herds in Guangdong starting in February 2019 (<a href="#bib0190">Zhou et al., 2019a</a>), with an outbreak of diarrhea that caused the death of about 2000 pigs at another pig farm near the original farms with the first outbreaks. This strain (CN/GDLX/2019, GenBank accession no. MK651076) shared approximately 100 % nt identity with the â€œFarm-Dâ€� strain, and had an S gene that shares high nt identity (99.2â€"99.9 %) with previously reported SeACoV strains from Guangdong in 2017, but lower identity (97.5 %) with CH/FJWT/2018 from Fujian in 2018 (<a href="#fig0015">Fig. 3</a>).</p> <p id="par0100">SeACoV has not been reported in other provinces in China or in any other regions in the world. All the identified sequences in pigs to date belong to a sublineage â€œSeACoV/PEAV/SADS-CoVâ€� which shares &amp;gt;99.5 % nt identities, and is closely related to the bat â€œHKU2-like CoVâ€� sublineage, together forming the species <i>Rhinolophus bat coronavirus HKU2</i> in the genus <i>Alphacoronavirus</i> (<a href="#fig0015">Fig. 3</a>). Interestingly, some globally distributed rodent CoVs are phylogenetically related to this species (<a href="#fig0015">Fig. 3</a>) (<a href="#bib0125">Tsoleridis et al., 2019</a>). Taken together, it is not clear whether the sporadic emergences of SeACoV in Guangdong and Fujian from 2016 to 2019 resulted from multiple instances of bat HKU2-like CoVs spillover. Thus, it is necessary to track the evolution of the species <i>Rhinolophus bat coronavirus HKU2</i> in order to better understand its epidemiology and prevent further outbreaks and spread in China. Further discussion is in Section <a href="#sec0055">5</a>: Transmission.</p> </div> </div> <div class="pathogenesis" title="sec"><span class="label" tagx="label" title="label">4</span><div class="title" tagx="title" title="title">
Pathogenesis</div> <p id="par0105">The molecular mechanism of how SeACoV antagonizes the host innate immune response has been preliminarily studied. SeACoV interfered with the activity of IPS-1 and RIG-I to inhibit the phosphorylation and nuclear translocation of IRF3 to block IFN-Î² production (<a href="#bib0210">Zhou et al., 2019b</a>). In addition, SeACoV can induce apoptosis <i>via</i> caspase-dependent FasL and mitochondria-mediated apoptotic pathways (<a href="#bib0185">Zhang et al., 2020</a>). These studies provide insights into the comprehensive understanding of the interactions between SeACoV and susceptible hosts. A mouse infection model has been established for SeACoV with active viral replication in splenic dendritic cells, which suggests that SeACoV may be susceptible to rodents (<a href="#bib0170">Yang et al., 2019b</a>). In this study, mice exhibited only subclinical infection, and seem to be more susceptible by the intraperitoneal route, with modest replication in intestinal tissues and titers declining around 3 days post-infection (<a href="#bib0170">Yang et al., 2019b</a>).</p> <p id="par0110">The pathogenicity of SeACoV in pigs remains controversial. Initially, it was believed that SeACoV causes enteric disease with a mortality rate over 35 % in piglets less than 10 days old (<a href="#bib0200">Zhou et al., 2018b</a>). Its clinical symptoms were similar to PEDV and TGEV, including severe diarrhea, vomiting, and dehydration (<a href="#bib0105">Pan et al., 2017</a>; <a href="#bib0200">Zhou et al., 2018b</a>). Although the viruses isolated from different groups were able to infect piglets by the oral route, they each caused mild or severe diarrhea symptom (<a href="#bib0105">Pan et al., 2017</a>; <a href="#bib0160">Xu et al., 2019</a>; <a href="#bib0200">Zhou et al., 2018b</a>). In a challenge experiment on 4-day-old piglets using a cell culture strain GDS04-P12, clinical signs developed gradually and the virus was shed in the feces throughout the experimental period (<a href="#bib0160">Xu et al., 2019</a>). All piglets (n = 10) challenged with the virus died from 5 to 12 days post-infection (dpi). Examination of the affected pigs revealed that the small intestine had accumulated a large amount of yellow liquid in the colon and the cecum, the intestine was inflated, and the intestinal wall was thin and transparent. SeACoV antigen was detected in the cytoplasm of the villous enterocytes of the challenged piglets by immunohistochemical analysis. Histopathology revealed that the intestinal villi of the duodenum, jejunum, and other places had varying degrees of atrophy, and the intestinal epithelium also showed necrosis (<a href="#bib0160">Xu et al., 2019</a>). These results indicated that GDS04-P12 is highly pathogenic to the newborn piglets. A pathogenesis study using the cell culture strain CN/GDWT/2017 from the group that designated the virus â€œSADS-CoVâ€� also showed high mortality (3/6) in the inoculated 3-day-old piglets (<a href="#bib0200">Zhou et al., 2018b</a>). However, immunohistochemical analysis displayed only a few jejunal epithelial cells that were stained with viral antigen in challenged piglets, which may not be in line with the 50 % mortality (<a href="#bib0200">Zhou et al., 2018b</a>). We have conducted a couple of experimental infection of neonatal piglets with a moderate or high dose of purified SeACoV (isolate CH/GD-01/2017), and they resulted only in mild-moderate diarrheal signs or subclinical infection compared to the first outbreaks in pig herds [(<a href="#bib0105">Pan et al., 2017</a>) and unpublished data], implying a lower pathogenicity of SeACoV.</p> <p id="par0115">The discrepancies in the results from different challenged studies are very confusing since these SeACoV strains were obtained from the same source in the first outbreaks. One explanation may be attributed to the sequence differences in the SeACoV cell adapted strains used for challenge. By comparative analysis, we found that the sequences of CH/GD-01/2017 and CN/GDWT/2017 are almost identical at the genome level (99.9 %), with no insertions and deletions, whereas GDS04-P12 shares 99.7 % or 99.8 % nt identity with CH/GD-01/2017 or CN/GDWT/2017, repectively. GDS04-P12 has 14 nt in gaps, including a 1-nt deletion (A) at nt 179, a 10-nt insertion (GACTAGAGCC) at nt 12,483-12,494, a 1-nt insertion (A) at nt 27,058-27,059, a 1-nt insertion (T) at nt 27,067-27,068, and a 1-nt insertion (A) in nt 27,074-27,075. The other factors, such as the age of the piglets (all were 3- or 4-day-old), the source of the piglets (conventional status), and the geographical environment where these experiments were performed (all were done in Guangdong), may not contribute to the observed differences.</p> <p id="par0120">Therefore, future studies based on our recently developed SeACoV reverse genetics system are warranted to investigate this important question. We hope to reconstruct the wild-type strain from the original sample by using an infectious clone, and conduct a comparative challenge experiment to further clarify the true pathogenicity of SeACoV.</p> </div> <div class="transmission" title="sec"><span class="label" tagx="label" title="label">5</span><div class="title" tagx="title" title="title">
Transmission</div> <p id="par0125">Like other porcine enteric viruses, SeACoV is transmitted through the fecal-oral route. This has been demonstrated by challenge experiments in pigs using the cell-cultured SeACoV strains (<a href="#bib0105">Pan et al., 2017</a>; <a href="#bib0160">Xu et al., 2019</a>; <a href="#bib0200">Zhou et al., 2018b</a>). Diarrhea or vomitus from infected piglets is likely the major transmission route of SeACoV. Other swine CoVs may be able to utilize different routes of infection. For example, PEDV invasion through the nasal cavity (respiratory tract) can also cause intestinal mucosal disease (<a href="#bib0065">Li et al., 2018b</a>), and PRCV is a pathogenic respiratory mutant of TGEV (<a href="#bib0145">Wesley and Lager, 2003</a>). Whether aerosolized SeACoV is also infectious should be investigated in the future.</p> <p id="par0130">Considering that the zoonotic SARS-CoV and MERS-CoV originated from bats and jumped between intermediate hosts (civets and camels, respectively), SeACoV may pose a similar risk to human health through transmission from pigs or other potential intermediate hosts (<a href="#bib0170">Yang et al., 2019b</a>). The zoonotic potential of SeACoV was assessed <i>in vitro</i>, showing that SeACoV was able to grow efficiently in nine human and monkey cell lines. As a conserved receptor across different animal species for SeACoV cell entry has been proposed, SeACoV may pose a similar risk to human health through interspecies transmission from pigs in the future (<a href="#fig0020">Fig. 4</a> ). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 4</span></a></div>   <p><b>Potential routes of interspecies transmission between HKU2-origin SeACoV and different hosts.</b> The ability of SeACoV to infect various rodent cell lines and replicate in a murine infection model suggests rodents as a potentially susceptible host. The solid arrow indicates the known SeACoV transmission event from bat to pig that occurred in 2017. The dashed arrows with question marks indicate hypothetical or potential transmissions from bat to rodent, pig to rodent or <i>vice versa</i>, and pig to human. See Section <a href="#sec0055">5</a> for details of the proposed model.</p>   </div> <p /> <p id="par0135">The direct progenitor of SeACoV is not the prototype HKU2-CoV identified in <i>Rhinolophus affinis</i> in 2004 and 2006, due to a low sequence identity (86 %) of the S gene (<a href="#bib0105">Pan et al., 2017</a>; <a href="#bib0200">Zhou et al., 2018b</a>). Rather, a couple of HKU2-like CoV sequences, designated SADSr-CoV (<a href="#fig0015">Fig. 3</a>), were determined in several species of horseshoe bats such as <i>Rhinolophus affinis</i>, <i>Rhinolophus sinicus</i>, and <i>Rhinolophus rex</i> from seven different locations in Guangdong province between 2013 and 2016 (<a href="#bib0200">Zhou et al., 2018b</a>). The S proteins of two SADSr-CoVs identified from <i>Rhinolophus affinis</i> were more closely related to that of SeACoV (âˆ¼98 % aa sequence identity), indicating that horseshoe bats are reservoirs of HKU2-like CoVs. The phylogenetic, haplotype network and recombination analyses suggested that SeACoV had probably been introduced from these HKU2-like CoVs independently, and that some strains might have undergone further intragenotypic recombination (<a href="#bib0200">Zhou et al., 2018b</a>). However, it is still not clear how the virus was transmitted from bats to pigs. In fact, it is hardly possible that pigs in captivity could contact flying bats directly in the four pig farms with disease outbreaks.</p> <p id="par0140">A possible scenario for such transmission may include bats infected with HKU2-like CoVs preying on insects near pig facilities, dropping contaminated feces that were later introduced into the pens somehow, by pig feed or some kind of animal (<a href="#fig0020">Fig. 4</a>). According to our onsite observation in 2017, rodents were frequently visible in these pig farms. Notably, bat HKU2-like CoVs are clustered with rat CoVs in the genus <i>Alphacoronavirus</i> (<a href="#fig0015">Fig. 3</a>). As we found that SeACoV infects different cell lines originating from rodents, and mice may be susceptible to SeACoV experimental infection (<a href="#bib0170">Yang et al., 2019b</a>), we hypothesize that in such field conditions, rodents (especially wild rats) in the farms may eat pig feed contaminated by bat feces, becoming carriers of SeACoV (<a href="#fig0020">Fig. 4</a>). Alternatively, if pigs became infected and shed SeACoV-positive feces, the virus could begin circulating in pig facilities. Contamination of pig feed, pig feces and water supplies by rodents could accumulate and develop into outbreaks of diarrhea in neonatal piglets (<a href="#fig0020">Fig. 4</a>). Future studies on identifying HKU2-like CoV positive samples in rodents near pig farms are warranted to test this hypothesis.</p> </div> <div class="diagnosticmethods" title="sec"><span class="label" tagx="label" title="label">6</span><div class="title" tagx="title" title="title">
Diagnostic methods</div> <p id="par0145">With increasing throughput and decreasing costs, next-generation sequencing (NGS) technology is increasingly being used for diagnosis of pathogens in clinical samples. In fact, except for initial isolation (<a href="#bib0105">Pan et al., 2017</a>), complete genome sequences of SeACoV strains have been determined by assembling and mapping sequencing reads with NGS data initially by two independent groups (<a href="#bib0020">Gong et al., 2017</a>; <a href="#bib0200">Zhou et al., 2018b</a>). Current SeACoV diagnostic assays can be divided into two categories: molecular and serological methods. Molecular methods detect viral nucleic acid, while serological assays detect the antibodies induced in host immune responses to infection. Since the initial outbreak of SeACoV in 2017, multiple detection methods have been established to monitor the epidemic. A summary of current SeACoV diagnostic assays and their properties are presented in <a href="#tbl0005">Table 1</a>, <a href="#tbl0010">Table 2</a> . <div class="table-wrap_UNKNOWN" position="float" id="tbl0005"><span class="label" tagx="label" title="label">Table 1</span> </div></p><p>Summary of virological methods for SeACoV RNA detection.</p>  <div tag="alt-text">
Table 1</div> <table frame="hsides" rules="groups"> <thead> <tr> <th align="left">Method</th> <th align="left">Target Gene</th> <th align="left">Primer or Probe</th> <th align="left">Sequences (5'â€"3')</th> <th align="left">Limit of Detection</th> <th align="left">Principle</th> <th align="left">Reference</th> </tr> </thead> <tbody> <tr> <td align="left" valign="middle" rowspan="4">RT-PCR assay</td> <td align="left" valign="middle" rowspan="2">N gene</td> <td align="left">SEAF</td> <td align="left">ATGGATAAACCTGAATGGAAGCG</td> <td align="left" valign="middle" rowspan="2">Not defined</td> <td align="left" valign="middle" rowspan="2">Identified SeACoV from fecal samples.</td> <td align="left" valign="middle" rowspan="2"><a href="#bib0105">Pan et al. (2017)</a></td> </tr> <tr> <td align="left">SEAR</td> <td align="left">CACCATCTCAACCTCTTCCTCAG</td> </tr> <tr> <td align="left" valign="middle" rowspan="2">M gene</td> <td align="left">Forward primer</td> <td align="left">GGTCCCTGTGACCGAAGTTTTAG</td> <td align="left" valign="middle" rowspan="2">Not defined</td> <td align="left" valign="middle" rowspan="2">Retrospective detection of SeACoV in Guangdong Province.</td> <td align="left" valign="middle" rowspan="2"><a href="#bib0195">Zhou et al. (2018a)</a></td> </tr> <tr> <td align="left">Reverse primer</td> <td align="left">GCGTTCTGCGATAAAGCTTAAAACTATTA</td> </tr> <tr> <td align="left" valign="middle" rowspan="6">LAMP-RT-PCR assay</td> <td align="left" valign="middle" rowspan="6">N gene</td> <td align="left">Outer primers: SADS-F3</td> <td align="left">CAGCCTTCTAACTGGCACTT</td> <td align="left" valign="middle" rowspan="6">10 RNA copies/Î¼L</td> <td align="left" valign="middle" rowspan="6">A simple and rapid detected method for the result can be directly observed.</td> <td align="left" valign="middle" rowspan="6"><a href="#bib0135">Wang et al. (2018b)</a></td> </tr> <tr> <td align="left">Outer primers: SADS-B3</td> <td align="left">ACAGTCAGGTCTGGTGGTAA</td> </tr> <tr> <td align="left">Inner primers: SADS -FIP</td> <td align="left">CGTCAACAGCGACCCAATGCATCCTCACGCAGATGCTCC</td> </tr> <tr> <td align="left">Inner primers: SADS -BIP</td> <td align="left">AACTAGCCCCACAGGTCTTGGTAACCCAAACTGAGGTGTAGC</td> </tr> <tr> <td align="left">Loop primers: SADS -LB</td> <td align="left">TCGCAATCGTAACAAAGAACCT</td> </tr> <tr> <td align="left">Loop primers: SADS -LF</td> <td align="left">CACCCTGAATCCGTTTCCTG</td> </tr> <tr> <td align="left" valign="middle" rowspan="9">TaqMan-based real-time RT-PCR assay</td> <td align="left" valign="middle" rowspan="3">N gene</td> <td align="left">qSADS-N-F</td> <td align="left">CTGACTGTTGTTGAGGTTAC</td> <td align="left" valign="middle" rowspan="3">30 DNA copies/Î¼L</td> <td align="left" valign="middle" rowspan="3">More sensitive than the conventional PCR.</td> <td align="left" valign="middle" rowspan="3"><a href="#bib0195">Zhou et al. (2018a)</a></td> </tr> <tr> <td align="left">qSADS-N-R</td> <td align="left">TCTGCCAAAGCTTGTTTAAC</td> </tr> <tr> <td align="left">Probe</td> <td align="left">FAM-TCACAGTCTCGTTCTCGCAATCA-TAMRA</td> </tr> <tr> <td align="left" valign="middle" rowspan="3">N gene</td> <td align="left">Forward primer</td> <td align="left">CTAAAACTAGCCCCACAGGTC</td> <td align="left" valign="middle" rowspan="3">100 RNA copies/Î¼L</td> <td align="left" valign="middle" rowspan="3">To detect viral RNA in feces and various tissues of the mouse model experiment.</td> <td align="left" valign="middle" rowspan="3"><a href="#bib0170">Yang et al. (2019b)</a></td> </tr> <tr> <td align="left">Reverse primer</td> <td align="left">TGATTGCGAGAACGAGACTG</td> </tr> <tr> <td align="left">Probe</td> <td align="left">FAM-GAAACCCAAACTGAGGTGTAGCAGG-TAMRA</td> </tr> <tr> <td align="left" valign="middle" rowspan="3">N gene</td> <td align="left">Forward primer</td> <td align="left">GCACTTTTATTACCTTGGTA</td> <td align="left" valign="middle" rowspan="3">Not defined</td> <td align="left" valign="middle" rowspan="3">To detect viral RNA in feces and various tissues of the piglets challenge experiment.</td> <td align="left" valign="middle" rowspan="3"><a href="#bib0160">Xu et al. (2019)</a></td> </tr> <tr> <td align="left">Reverse primer</td> <td align="left">GTAGCAGGTTCTTTGTTAC</td> </tr> <tr> <td align="left">Probe</td> <td align="left">FAM-TCCTCACGCAGATGCTCCTT-TAMRA</td> </tr> <tr> <td align="left" valign="middle" rowspan="3">TaqMan-based Multiplex real-time RT-PCR assay</td> <td align="left" valign="middle" rowspan="3">N gene</td> <td align="left">Forward primer</td> <td align="left">TCTCGGCTTACTCTAAACCC</td> <td align="left" valign="middle" rowspan="3">110 RNA copies/Î¼L</td> <td align="left" valign="middle" rowspan="3">A multiplex real-time RT-qPCR assay for detection of swine enteric coronaviruses.</td> <td align="left" valign="middle" rowspan="3"><a href="#bib0030">Huang et al. (2019)</a></td> </tr> <tr> <td align="left">Reverse primer</td> <td align="left">CATCCACCATCTCAACCTC</td> </tr> <tr> <td align="left">Probe</td> <td align="left">TexasRed-AAGACCTAAATGCTGATGCCCCA-BHQ2</td> </tr> <tr> <td align="left" valign="middle" rowspan="10">SYBR green-based real-time RT-PCR assay</td> <td align="left" valign="middle" rowspan="2">N gene</td> <td align="left">SADS-N-F</td> <td align="left">CTAAAACTAGCCCCACAGGTC</td> <td align="left" valign="middle" rowspan="2">Not defined</td> <td align="left" valign="middle" rowspan="2">To detect SeACoV in acutely sick piglets and sows with diarrhea.</td> <td align="left" valign="middle" rowspan="2"><a href="#bib0200">Zhou et al. (2018b)</a></td> </tr> <tr> <td align="left">SADS-N-R</td> <td align="left">TGATTGCGAGAACGAGACTG</td> </tr> <tr> <td align="left" valign="middle" rowspan="2">RdRp</td> <td align="left">SADS-RdRp-F</td> <td align="left">GTTGATTGTAAGGCTTGGCG</td> <td align="left" valign="middle" rowspan="2">Not defined</td> <td align="left" valign="middle" rowspan="2">To detect bat anal swabs samples and determined SADSr-CoV.</td> <td align="left" valign="middle" rowspan="2"><a href="#bib0200">Zhou et al. (2018b)</a></td> </tr> <tr> <td align="left">SADS-RdRp-R</td> <td align="left">AACCACACTTCCACTCAGC</td> </tr> <tr> <td align="left" valign="middle" rowspan="2">M gene</td> <td align="left">Forward primer</td> <td align="left">ATGTGGCTCCTATGGCCCTT</td> <td align="left" valign="middle" rowspan="2">10 RNA copies/Î¼L</td> <td align="left" valign="middle" rowspan="2">More sensitive than probe-based RT-qPCR.</td> <td align="left" valign="middle" rowspan="2"><a href="#bib0080">Ma et al. (2018)</a></td> </tr> <tr> <td align="left">Reverse primer</td> <td align="left">ACGTTCGCGTTCTGCGATAA</td> </tr> <tr> <td align="left" valign="middle" rowspan="2">RdRp</td> <td align="left">qSADS-CoV-ORF1 F</td> <td align="left">AGTGAAAGACCACAGCAAACAG</td> <td align="left" valign="middle" rowspan="2">Not defined</td> <td align="left" valign="middle" rowspan="4">To detect multiple-shRNA-mediated inhibition of viral RNA replication.</td> <td align="left" valign="middle" rowspan="4"><a href="#bib0070">Li et al. (2019)</a></td> </tr> <tr> <td align="left">qSADS-CoV-ORF1 R</td> <td align="left">TAACATACGCCCAGCAACATAG</td> </tr> <tr> <td align="left" valign="middle" rowspan="2">S gene</td> <td align="left">qSADS-CoV-S F</td> <td align="left">TTGTGAGAGTGATGAATTGGGT</td> <td align="left" valign="middle" rowspan="2">Not defined</td> </tr> <tr> <td align="left">qSADS-CoV-S R</td> <td align="left">GATTTTCTGGTTTGTAAAGGTT</td> </tr> </tbody> </table>  <div class="table-wrap_UNKNOWN" position="float" id="tbl0010"><span class="label" tagx="label" title="label">Table 2</span> <p>Summary of serological methods for SeACoV antibody detection.</p>  <div tag="alt-text">
Table 2</div> <table frame="hsides" rules="groups"> <thead> <tr> <th align="left">Method</th> <th align="left">Target protein</th> <th align="left">Principle</th> <th align="left">Reference</th> </tr> </thead> <tbody> <tr> <td align="left">IFA</td> <td align="left">Live virus</td> <td align="left">To examine SeACoV antibody from pig sera in SeACoV-infected cells</td> <td align="left"><a href="#bib0105">Pan et al. (2017)</a></td> </tr> <tr> <td align="left" valign="middle" rowspan="3">ELISA</td> <td align="left">S1 domain</td> <td align="left">Antibody assay based on the S1 domain of the S protein using a luciferase immunoprecipitation system</td> <td align="left"><a href="#bib0200">Zhou et al. (2018b)</a></td> </tr> <tr> <td align="left">Virus particles</td> <td align="left">Purified virus particles as antigen to detect SeACoV antibodies in the serum from the infected mice</td> <td align="left"><a href="#bib0170">Yang et al. (2019b)</a></td> </tr> <tr> <td align="left">Not defined</td> <td align="left">Serological investigation of recent SADS cases implied that SADS-CoV, rather than PEDV, may directly contribute to the diarrhea</td> <td align="left"><a href="#bib0190">Zhou et al. (2019a)</a></td> </tr> </tbody> </table> </div> <p /> <div class="pcr-basedassays" title="sec"><span class="label" tagx="label" title="label">6.1</span><div class="title" tagx="title" title="title">
PCR-based assays</div> <p id="par0150">PCR is one of the most commonly used rapid detection methods in clinical diagnosis. In order to identify the etiological agent associated with outbreaks of diarrhea in newborn piglets, a pair of conserved primers was used to confirm a 251bp fragment in ORF1b by conventional RT-PCR of a region 100 % identical to the corresponding part of four bat HKU2-CoV strains (<a href="#bib0105">Pan et al., 2017</a>). After the complete genome of SeACoV was determined, multiple rapid PCR detection methods were subsequently developed (<a href="#tbl0005">Table 1</a>). Reverse transcription loop-mediated isothermal amplification (RT-LAMP) and real-time RT-PCR assays were established for the detection of SeACoV (<a href="#bib0135">Wang et al., 2018b</a>; <a href="#bib0180">Zhang et al., 2019</a>; <a href="#bib0195">Zhou et al., 2018a</a>). Real-time RT-PCR is more powerful than RT-LAMP, with greater sensitivity and reproducibility for better quantification of pathogen. SYBR green-based and TaqMan-based methods have been developed for several SeACoV isolates (<a href="#bib0080">Ma et al., 2018</a>). Since the S gene is more prone to mutation and recombination, PCR-based methods mainly target the relatively conserved N and M gene. In addition there is a multiplex real-time RT-qPCR assay for simultaneous detection of swine enteric PEDV, PDCoV (both targeting the M gene), TGEV and SeACoV (both targeting the N gene) (<a href="#bib0030">Huang et al., 2019</a>).</p> </div> <div class="serologicalassays" title="sec"><span class="label" tagx="label" title="label">6.2</span><div class="title" tagx="title" title="title">
Serological assays</div> <p id="par0155">The most commonly used serological assays include the indirect fluorescent assay (IFA) and enzyme linked immunosorbent assay (ELISA). The IFA detection method is mainly used in laboratory research on the mechanism of viral replication and infection. SeACoV is antigenically distinct from PEDV, TGEV and PDCoV, and antibodies against the N protein do not cross-react (<a href="#bib0105">Pan et al., 2017</a>). In addition, our preliminary result showed that virus-specific sera from SeACoV, PEDV, TGEV and PDCoV-infected pigs do not cross-react in IFA tests (unpublished data). Since SeACoV was first isolated in pigs vaccinated with PEDV vaccines (<a href="#bib0105">Pan et al., 2017</a>; <a href="#bib0200">Zhou et al., 2018b</a>), this evidence suggests that the existing porcine CoV vaccines may not be effective against SeACoV infection.</p> <p id="par0160">While ELISA can be used to monitor changes of antibody levels in field herds under clinical situations, it can also be used retrospectively to detect viruses and IgA/IgG levels in serum to provide guidance for vaccine use. The ELISA detection methods for SeACoV are only available for use in the laboratory, for the epidemiological investigation of the initial virus outbreak and reemergence (<a href="#tbl0010">Table 2</a>). The ELISA uses a luciferase immunoprecipitation system based on the S1 protein, and has been used to investigate possible zoonotic transmission (<a href="#bib0200">Zhou et al., 2018b</a>). In addition, there is a virion-based ELISA detection method designed for sample detection in mouse model experiments, which can also be used for pig serum samples (<a href="#bib0170">Yang et al., 2019b</a>).</p> </div> </div> <div class="conclusion" title="sec"><span class="label" tagx="label" title="label">7</span><div class="title" tagx="title" title="title">
Conclusion</div> <p id="par0165">As early as 2004, the HKU2-related <i>Rhinolophus</i> bat CoV was identified (<a href="#bib0050">Lau et al., 2007</a>), and the first outbreak of acute diarrhea caused by SeACoV in pigs appeared in 2017 (<a href="#bib0105">Pan et al., 2017</a>). Although the virus reemerged in Fujian in 2018 and Guangdong in 2019, there have not been large-scale losses reported, which is positive news for its prevention and control. However, it is strange that SeACoV has not resulted in a large-scale epidemic in the pig industry in Guangdong in recent years, similar to the way SARS-CoV emerged in 2003 and then basically disappeared (<a href="#bib0045">Ksiazek et al., 2003</a>) until the emergence of SARS-CoV-2 at the end of 2019 (<a href="#bib0205">Zhou et al., 2020</a>). This outbreak of the novel SARS-CoV-2 and current pandemic also requires strict control to ensure the health of people around the world. Since all CoVs have the capability for recombination, no CoV epidemic can be taken lightly, even those in pigs (<a href="#bib0090">Opriessnig and Huang, 2020</a>). To prevent future outbreaks, unified detection methods are needed to establish better surveillance of SeACoV, including virological and serological methods.</p> <p id="par0170">More research is needed to understand the infectious properties of SeACoV and HKU2-related CoVs, especially given their broad ability to infect cell lines from various animal origins (<a href="#bib0170">Yang et al., 2019b</a>). The presumed name â€œSADS-CoVâ€� may not be appropriate since the available pathogenicity studies have not provided strong evidence of high virulence. The reverse genetic system for SeACoV has been established successfully in our lab (<a href="#bib0165">Yang et al., 2019a</a>), which will facilitate future studies exploring the molecular mechanisms of SeACoV replication and pathogenicity. With the structural analysis of SeACoV and HKU2 S glycoproteins, it is also important to identify the primary receptor and study the mechanism of receptor binding. It is clear now more than ever that CoV-induced diseases not only pose a huge threat to the pig industry but also to human health. If HKU2-related CoVs emerge again, causing unknown damage to the pig industry, such basic research on SeACoV will be necessary to prevent and control future epidemics.</p> </div> </div> <div class="back" title="back"> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="bib0005" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Almazan</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sola</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zuniga</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marquez-Jurado</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Morales</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Becares</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Enjuanes</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Coronavirus reverse genetic systems: infectious clones and replicons</span><span class="source" tagx="source" title="source">Virus Res.</span><span class="volume" tagx="volume" title="volume">189</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">262</span><span class="lpage" tagx="lpage" title="lpage">270</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24930446">24930446</a></span></span></li> <li tag="ref"><a name="bib0010" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Groot</span><span class="given-names" tagx="given-names" title="given-names">R.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baker</span><span class="given-names" tagx="given-names" title="given-names">S.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baric</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Enjuanes</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gorbalenya</span><span class="given-names" tagx="given-names" title="given-names">A.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">K.V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perlman</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poon</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rottier</span><span class="given-names" tagx="given-names" title="given-names">P.J.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Talbot</span><span class="given-names" tagx="given-names" title="given-names">P.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woo</span><span class="given-names" tagx="given-names" title="given-names">P.C.Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ziebuhr</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Coronaviridae</span><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">King</span><span class="given-names" tagx="given-names" title="given-names">A.M.Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adams</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carstens</span><span class="given-names" tagx="given-names" title="given-names">E.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lefkowitz</span><span class="given-names" tagx="given-names" title="given-names">E.J.</span></span></span><span class="source" tagx="source" title="source">Virus Taxonomy: Ninth Report of the International Committee on Taxonomy of Viruses</span><span class="year" tagx="year" title="year">2011</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Elsevier Academic Press</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span><span class="fpage" tagx="fpage" title="fpage">806</span><span class="lpage" tagx="lpage" title="lpage">828</span></span></li> <li tag="ref"><a name="bib0015" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Haan</span><span class="given-names" tagx="given-names" title="given-names">C.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vennema</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rottier</span><span class="given-names" tagx="given-names" title="given-names">P.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Assembly of the coronavirus envelope: homotypic interactions between the M proteins</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="volume" tagx="volume" title="volume">74</span><span class="issue" tagx="issue" title="issue">11</span><span class="year" tagx="year" title="year">2000</span><span class="fpage" tagx="fpage" title="fpage">4967</span><span class="lpage" tagx="lpage" title="lpage">4978</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10799570">10799570</a></span></span></li> <li tag="ref"><a name="bib0020" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gong</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xue</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wen</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A new Bat-HKU2-like coronavirus in swine, China, 2017</span><span class="source" tagx="source" title="source">Emerg. Infect. Dis.</span><span class="volume" tagx="volume" title="volume">23</span><span class="issue" tagx="issue" title="issue">9</span><span class="year" tagx="year" title="year">2017</span><span class="fpage" tagx="fpage" title="fpage">1607</span><span class="lpage" tagx="lpage" title="lpage">1609</span></span></li> <li tag="ref"><a name="bib0025" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greig</span><span class="given-names" tagx="given-names" title="given-names">A.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mitchell</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Corner</span><span class="given-names" tagx="given-names" title="given-names">A.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bannister</span><span class="given-names" tagx="given-names" title="given-names">G.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meads</span><span class="given-names" tagx="given-names" title="given-names">E.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Julian</span><span class="given-names" tagx="given-names" title="given-names">R.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A hemagglutinating virus producing encephalomyelitis in baby pigs</span><span class="source" tagx="source" title="source">Can. J. Comp. Med. Vet. Sci.</span><span class="volume" tagx="volume" title="volume">26</span><span class="issue" tagx="issue" title="issue">3</span><span class="year" tagx="year" title="year">1962</span><span class="fpage" tagx="fpage" title="fpage">49</span><span class="lpage" tagx="lpage" title="lpage">56</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17649356">17649356</a></span></span></li> <li tag="ref"><a name="bib0030" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yao</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A TaqMan-probe-based multiplex real-time RT-qPCR for simultaneous detection of porcine enteric coronaviruses</span><span class="source" tagx="source" title="source">Appl. Microbiol. Biotechnol.</span><span class="volume" tagx="volume" title="volume">103</span><span class="issue" tagx="issue" title="issue">12</span><span class="year" tagx="year" title="year">2019</span><span class="fpage" tagx="fpage" title="fpage">4943</span><span class="lpage" tagx="lpage" title="lpage">4952</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31025076">31025076</a></span></span></li> <li tag="ref"><a name="bib0035" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Y.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dickerman</span><span class="given-names" tagx="given-names" title="given-names">A.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pineyro</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kiehne</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Opriessnig</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meng</span><span class="given-names" tagx="given-names" title="given-names">X.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Origin, evolution, and genotyping of emergent porcine epidemic diarrhea virus strains in the United States</span><span class="source" tagx="source" title="source">MBio</span><span class="volume" tagx="volume" title="volume">4</span><span class="issue" tagx="issue" title="issue">5</span><span class="year" tagx="year" title="year">2013</span><span class="fpage" tagx="fpage" title="fpage">e00737</span><span class="lpage" tagx="lpage" title="lpage">00713</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24129257">24129257</a></span></span></li> <li tag="ref"><a name="bib0040" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ji</span><span class="given-names" tagx="given-names" title="given-names">C.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Y.W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Aminopeptidase-N-independent entry of porcine epidemic diarrhea virus into Vero or porcine small intestine epithelial cells</span><span class="source" tagx="source" title="source">Virology</span><span class="volume" tagx="volume" title="volume">517</span><span class="year" tagx="year" title="year">2018</span><span class="fpage" tagx="fpage" title="fpage">16</span><span class="lpage" tagx="lpage" title="lpage">23</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29502803">29502803</a></span></span></li> <li tag="ref"><a name="bib0045" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ksiazek</span><span class="given-names" tagx="given-names" title="given-names">T.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Erdman</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goldsmith</span><span class="given-names" tagx="given-names" title="given-names">C.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zaki</span><span class="given-names" tagx="given-names" title="given-names">S.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peret</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Emery</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tong</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Urbani</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Comer</span><span class="given-names" tagx="given-names" title="given-names">J.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lim</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rollin</span><span class="given-names" tagx="given-names" title="given-names">P.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dowell</span><span class="given-names" tagx="given-names" title="given-names">S.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ling</span><span class="given-names" tagx="given-names" title="given-names">A.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Humphrey</span><span class="given-names" tagx="given-names" title="given-names">C.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shieh</span><span class="given-names" tagx="given-names" title="given-names">W.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guarner</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paddock</span><span class="given-names" tagx="given-names" title="given-names">C.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rota</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fields</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DeRisi</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">J.Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cox</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hughes</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">LeDuc</span><span class="given-names" tagx="given-names" title="given-names">J.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bellini</span><span class="given-names" tagx="given-names" title="given-names">W.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anderson</span><span class="given-names" tagx="given-names" title="given-names">L.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Group</span><span class="given-names" tagx="given-names" title="given-names">S.W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A novel coronavirus associated with severe acute respiratory syndrome</span><span class="source" tagx="source" title="source">N. Engl. J. Med.</span><span class="volume" tagx="volume" title="volume">348</span><span class="issue" tagx="issue" title="issue">20</span><span class="year" tagx="year" title="year">2003</span><span class="fpage" tagx="fpage" title="fpage">1953</span><span class="lpage" tagx="lpage" title="lpage">1966</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12690092">12690092</a></span></span></li> <li tag="ref"><a name="bib0050" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lau</span><span class="given-names" tagx="given-names" title="given-names">S.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woo</span><span class="given-names" tagx="given-names" title="given-names">P.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">K.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lam</span><span class="given-names" tagx="given-names" title="given-names">C.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">K.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">B.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuen</span><span class="given-names" tagx="given-names" title="given-names">K.Y.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Complete genome sequence of bat coronavirus HKU2 from Chinese horseshoe bats revealed a much smaller spike gene with a different evolutionary lineage from the rest of the genome</span><span class="source" tagx="source" title="source">Virology</span><span class="volume" tagx="volume" title="volume">367</span><span class="issue" tagx="issue" title="issue">2</span><span class="year" tagx="year" title="year">2007</span><span class="fpage" tagx="fpage" title="fpage">428</span><span class="lpage" tagx="lpage" title="lpage">439</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17617433">17617433</a></span></span></li> <li tag="ref"><a name="bib0055" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bi</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gong</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ye</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Complete genome sequence of a novel swine acute diarrhea syndrome coronavirus, CH/FJWT/2018, isolated in Fujian, China, in 2018</span><span class="source" tagx="source" title="source">Microb. Resour. Announc.</span><span class="volume" tagx="volume" title="volume">7</span><span class="issue" tagx="issue" title="issue">22</span><span class="year" tagx="year" title="year">2018</span><span class="comment">pii: e01259-01218</span></span></li> <li tag="ref"><a name="bib0060" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moore</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vasilieva</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sui</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wong</span><span class="given-names" tagx="given-names" title="given-names">S.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berne</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Somasundaran</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sullivan</span><span class="given-names" tagx="given-names" title="given-names">J.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luzuriaga</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greenough</span><span class="given-names" tagx="given-names" title="given-names">T.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Choe</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Farzan</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">426</span><span class="issue" tagx="issue" title="issue">6965</span><span class="year" tagx="year" title="year">2003</span><span class="fpage" tagx="fpage" title="fpage">450</span><span class="lpage" tagx="lpage" title="lpage">454</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14647384">14647384</a></span></span></li> <li tag="ref"><a name="bib0065" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span></span><span class="mixed-article-title" title="mixed-article-title">An alternative pathway of enteric PEDV dissemination from nasal cavity to intestinal mucosa in swine</span><span class="source" tagx="source" title="source">Nat. Commun.</span><span class="volume" tagx="volume" title="volume">9</span><span class="issue" tagx="issue" title="issue">1</span><span class="year" tagx="year" title="year">2018</span><span class="fpage" tagx="fpage" title="fpage">3811</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30232333">30232333</a></span></span></li> <li tag="ref"><a name="bib0070" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bi</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lei</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gong</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ye</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Significant inhibition of re-emerged and emerging swine enteric coronavirus in vitro using the multiple shRNA expression vector</span><span class="source" tagx="source" title="source">Antiviral Res.</span><span class="volume" tagx="volume" title="volume">166</span><span class="year" tagx="year" title="year">2019</span><span class="fpage" tagx="fpage" title="fpage">11</span><span class="lpage" tagx="lpage" title="lpage">18</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30905822">30905822</a></span></span></li> <li tag="ref"><a name="bib0075" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">D.X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fung</span><span class="given-names" tagx="given-names" title="given-names">T.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chong</span><span class="given-names" tagx="given-names" title="given-names">K.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shukla</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hilgenfeld</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Accessory proteins of SARS-CoV and other coronaviruses</span><span class="source" tagx="source" title="source">Antiviral Res.</span><span class="volume" tagx="volume" title="volume">109</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">97</span><span class="lpage" tagx="lpage" title="lpage">109</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24995382">24995382</a></span></span></li> <li tag="ref"><a name="bib0080" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zeng</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cong</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Development of a SYBR green-based real-time RT-PCR assay for rapid detection of the emerging swine acute diarrhea syndrome coronavirus</span><span class="source" tagx="source" title="source">J. Virol. Methods</span><span class="volume" tagx="volume" title="volume">265</span><span class="year" tagx="year" title="year">2018</span><span class="fpage" tagx="fpage" title="fpage">66</span><span class="lpage" tagx="lpage" title="lpage">70</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30593837">30593837</a></span></span></li> <li tag="ref"><a name="bib0085" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nelson</span><span class="given-names" tagx="given-names" title="given-names">G.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stohlman</span><span class="given-names" tagx="given-names" title="given-names">S.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tahara</span><span class="given-names" tagx="given-names" title="given-names">S.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">High affinity interaction between nucleocapsid protein and leader/intergenic sequence of mouse hepatitis virus RNA</span><span class="source" tagx="source" title="source">J. Gen. Virol.</span><span class="volume" tagx="volume" title="volume">81</span><span class="issue" tagx="issue" title="issue">Pt. 1</span><span class="year" tagx="year" title="year">2000</span><span class="fpage" tagx="fpage" title="fpage">181</span><span class="lpage" tagx="lpage" title="lpage">188</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10640556">10640556</a></span></span></li> <li tag="ref"><a name="bib0090" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Opriessnig</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Y.W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Coronavirus disease 2019 (COVID-19) outbreak: could pigs be vectors for human infections?</span><span class="source" tagx="source" title="source">Xenotransplantation</span><span class="volume" tagx="volume" title="volume">27</span><span class="issue" tagx="issue" title="issue">2</span><span class="year" tagx="year" title="year">2020</span><span class="fpage" tagx="fpage" title="fpage">e12591</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32222047">32222047</a></span></span></li> <li tag="ref"><a name="bib0095" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ortego</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sola</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Almazan</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ceriani</span><span class="given-names" tagx="given-names" title="given-names">J.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riquelme</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balasch</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Plana</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Enjuanes</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Transmissible gastroenteritis coronavirus gene 7 is not essential but influences in vivo virus replication and virulence</span><span class="source" tagx="source" title="source">Virology</span><span class="volume" tagx="volume" title="volume">308</span><span class="issue" tagx="issue" title="issue">1</span><span class="year" tagx="year" title="year">2003</span><span class="fpage" tagx="fpage" title="fpage">13</span><span class="lpage" tagx="lpage" title="lpage">22</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12706086">12706086</a></span></span></li> <li tag="ref"><a name="bib0100" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pan</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tian</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bi</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Isolation and characterization of a variant porcine epidemic diarrhea virus in China</span><span class="source" tagx="source" title="source">Virol. J.</span><span class="volume" tagx="volume" title="volume">9</span><span class="year" tagx="year" title="year">2012</span><span class="fpage" tagx="fpage" title="fpage">195</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22967434">22967434</a></span></span></li> <li tag="ref"><a name="bib0105" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pan</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tian</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qin</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Y.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Y.W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Discovery of a novel swine enteric alphacoronavirus (SeACoV) in southern China</span><span class="source" tagx="source" title="source">Vet. Microbiol.</span><span class="volume" tagx="volume" title="volume">211</span><span class="year" tagx="year" title="year">2017</span><span class="fpage" tagx="fpage" title="fpage">15</span><span class="lpage" tagx="lpage" title="lpage">21</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29102111">29102111</a></span></span></li> <li tag="ref"><a name="bib0110" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pensaert</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Callebaut</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vergote</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Isolation of a porcine respiratory, non-enteric coronavirus related to transmissible gastroenteritis</span><span class="source" tagx="source" title="source">Vet. Q.</span><span class="volume" tagx="volume" title="volume">8</span><span class="issue" tagx="issue" title="issue">3</span><span class="year" tagx="year" title="year">1986</span><span class="fpage" tagx="fpage" title="fpage">257</span><span class="lpage" tagx="lpage" title="lpage">261</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3018993">3018993</a></span></span></li> <li tag="ref"><a name="bib0115" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pensaert</span><span class="given-names" tagx="given-names" title="given-names">M.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Bouck</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A new coronavirus-like particle associated with diarrhea in swine</span><span class="source" tagx="source" title="source">Arch. Virol.</span><span class="volume" tagx="volume" title="volume">58</span><span class="issue" tagx="issue" title="issue">3</span><span class="year" tagx="year" title="year">1978</span><span class="fpage" tagx="fpage" title="fpage">243</span><span class="lpage" tagx="lpage" title="lpage">247</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/83132">83132</a></span></span></li> <li tag="ref"><a name="bib0120" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raj</span><span class="given-names" tagx="given-names" title="given-names">V.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mou</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smits</span><span class="given-names" tagx="given-names" title="given-names">S.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dekkers</span><span class="given-names" tagx="given-names" title="given-names">D.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muller</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dijkman</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muth</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Demmers</span><span class="given-names" tagx="given-names" title="given-names">J.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zaki</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fouchier</span><span class="given-names" tagx="given-names" title="given-names">R.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thiel</span><span class="given-names" tagx="given-names" title="given-names">V.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drosten</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rottier</span><span class="given-names" tagx="given-names" title="given-names">P.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Osterhaus</span><span class="given-names" tagx="given-names" title="given-names">A.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bosch</span><span class="given-names" tagx="given-names" title="given-names">B.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haagmans</span><span class="given-names" tagx="given-names" title="given-names">B.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Dipeptidyl peptidase 4 is a functional receptor for the emerging human coronavirus-EMC</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">495</span><span class="issue" tagx="issue" title="issue">7440</span><span class="year" tagx="year" title="year">2013</span><span class="fpage" tagx="fpage" title="fpage">251</span><span class="lpage" tagx="lpage" title="lpage">254</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23486063">23486063</a></span></span></li> <li tag="ref"><a name="bib0125" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsoleridis</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chappell</span><span class="given-names" tagx="given-names" title="given-names">J.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Onianwa</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marston</span><span class="given-names" tagx="given-names" title="given-names">D.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fooks</span><span class="given-names" tagx="given-names" title="given-names">A.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Monchatre-Leroy</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Umhang</span><span class="given-names" tagx="given-names" title="given-names">G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muller</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drexler</span><span class="given-names" tagx="given-names" title="given-names">J.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drosten</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tarlinton</span><span class="given-names" tagx="given-names" title="given-names">R.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McClure</span><span class="given-names" tagx="given-names" title="given-names">C.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">E.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ball</span><span class="given-names" tagx="given-names" title="given-names">J.K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Shared common ancestry of rodent alphacoronaviruses sampled globally</span><span class="source" tagx="source" title="source">Viruses</span><span class="volume" tagx="volume" title="volume">11</span><span class="issue" tagx="issue" title="issue">2</span><span class="year" tagx="year" title="year">2019</span></span></li> <li tag="ref"><a name="bib0130" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ji</span><span class="given-names" tagx="given-names" title="given-names">C.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Y.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">Q.Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">L.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lei</span><span class="given-names" tagx="given-names" title="given-names">X.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">W.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qin</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Y.W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Porcine deltacoronavirus engages the transmissible gastroenteritis virus functional receptor porcine aminopeptidase N for infectious cellular entry</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="volume" tagx="volume" title="volume">92</span><span class="issue" tagx="issue" title="issue">12</span><span class="year" tagx="year" title="year">2018</span><span class="fpage" tagx="fpage" title="fpage">e00318</span><span class="lpage" tagx="lpage" title="lpage">318</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29618640">29618640</a></span></span></li> <li tag="ref"><a name="bib0135" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cong</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zeng</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lian</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Development of a real time reverse transcription loop-mediated isothermal amplification method (RT-LAMP) for detection of a novel swine acute diarrhea syndrome coronavirus (SADS-CoV)</span><span class="source" tagx="source" title="source">J. Virol. Methods</span><span class="volume" tagx="volume" title="volume">260</span><span class="year" tagx="year" title="year">2018</span><span class="fpage" tagx="fpage" title="fpage">45</span><span class="lpage" tagx="lpage" title="lpage">48</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29936095">29936095</a></span></span></li> <li tag="ref"><a name="bib0140" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vlasova</span><span class="given-names" tagx="given-names" title="given-names">A.N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kenney</span><span class="given-names" tagx="given-names" title="given-names">S.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saif</span><span class="given-names" tagx="given-names" title="given-names">L.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Emerging and re-emerging coronaviruses in pigs</span><span class="source" tagx="source" title="source">Curr. Opin. Virol.</span><span class="volume" tagx="volume" title="volume">34</span><span class="year" tagx="year" title="year">2019</span><span class="fpage" tagx="fpage" title="fpage">39</span><span class="lpage" tagx="lpage" title="lpage">49</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30654269">30654269</a></span></span></li> <li tag="ref"><a name="bib0145" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wesley</span><span class="given-names" tagx="given-names" title="given-names">R.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lager</span><span class="given-names" tagx="given-names" title="given-names">K.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Increased litter survival rates, reduced clinical illness and better lactogenic immunity against TGEV in gilts that were primed as neonates with porcine respiratory coronavirus (PRCV)</span><span class="source" tagx="source" title="source">Vet. Microbiol.</span><span class="volume" tagx="volume" title="volume">95</span><span class="issue" tagx="issue" title="issue">3</span><span class="year" tagx="year" title="year">2003</span><span class="fpage" tagx="fpage" title="fpage">175</span><span class="lpage" tagx="lpage" title="lpage">186</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12935745">12935745</a></span></span></li> <li tag="ref"><a name="bib0150" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Williams</span><span class="given-names" tagx="given-names" title="given-names">R.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">G.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">K.V.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Receptor for mouse hepatitis virus is a member of the carcinoembryonic antigen family of glycoproteins</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U. S. A.</span><span class="volume" tagx="volume" title="volume">88</span><span class="issue" tagx="issue" title="issue">13</span><span class="year" tagx="year" title="year">1991</span><span class="fpage" tagx="fpage" title="fpage">5533</span><span class="lpage" tagx="lpage" title="lpage">5536</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1648219">1648219</a></span></span></li> <li tag="ref"><a name="bib0155" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woo</span><span class="given-names" tagx="given-names" title="given-names">P.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lau</span><span class="given-names" tagx="given-names" title="given-names">S.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lam</span><span class="given-names" tagx="given-names" title="given-names">C.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lau</span><span class="given-names" tagx="given-names" title="given-names">C.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsang</span><span class="given-names" tagx="given-names" title="given-names">A.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lau</span><span class="given-names" tagx="given-names" title="given-names">J.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bai</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Teng</span><span class="given-names" tagx="given-names" title="given-names">J.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsang</span><span class="given-names" tagx="given-names" title="given-names">C.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">B.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chan</span><span class="given-names" tagx="given-names" title="given-names">K.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuen</span><span class="given-names" tagx="given-names" title="given-names">K.Y.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Discovery of seven novel Mammalian and avian coronaviruses in the genus deltacoronavirus supports bat coronaviruses as the gene source of alphacoronavirus and betacoronavirus and avian coronaviruses as the gene source of gammacoronavirus and deltacoronavirus</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="volume" tagx="volume" title="volume">86</span><span class="issue" tagx="issue" title="issue">7</span><span class="year" tagx="year" title="year">2012</span><span class="fpage" tagx="fpage" title="fpage">3995</span><span class="lpage" tagx="lpage" title="lpage">4008</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22278237">22278237</a></span></span></li> <li tag="ref"><a name="bib0160" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gong</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qin</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Du</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xue</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Isolation and characterization of A Highly Pathogenic Strain of Porcine enteric alphacoronavirus Causing Watery Diarrhea and High Mortality in Newborn Piglets</span><span class="source" tagx="source" title="source">Transbound. Emerg. Dis.</span><span class="volume" tagx="volume" title="volume">66</span><span class="issue" tagx="issue" title="issue">1</span><span class="year" tagx="year" title="year">2019</span><span class="fpage" tagx="fpage" title="fpage">119</span><span class="lpage" tagx="lpage" title="lpage">130</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30103259">30103259</a></span></span></li> <li tag="ref"><a name="bib0165" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Y.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">Q.Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">S.Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mazing</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">G.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peng</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qin</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Y.W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Characterization of a novel bat-HKU2-like swine enteric alphacoronavirus (SeACoV) infection in cultured cells and development of a SeACoV infectious clone</span><span class="source" tagx="source" title="source">Virology</span><span class="volume" tagx="volume" title="volume">536</span><span class="year" tagx="year" title="year">2019</span><span class="fpage" tagx="fpage" title="fpage">110</span><span class="lpage" tagx="lpage" title="lpage">118</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31419711">31419711</a></span></span></li> <li tag="ref"><a name="bib0170" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">Y.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qin</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">G.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peng</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">S.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Y.W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Broad cross-species infection of cultured cells by bat HKU2-related swine acute diarrhea syndrome coronavirus and identification of its replication in murine dendritic cells in vivo highlight its potential for diverse interspecies transmission</span><span class="source" tagx="source" title="source">J. Virol.</span><span class="volume" tagx="volume" title="volume">93</span><span class="issue" tagx="issue" title="issue">24</span><span class="year" tagx="year" title="year">2019</span><span class="fpage" tagx="fpage" title="fpage">e01448</span><span class="lpage" tagx="lpage" title="lpage">1419</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31554686">31554686</a></span></span></li> <li tag="ref"><a name="bib0175" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qiao</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cryo-EM structures of HKU2 and SADS-CoV spike glycoproteins and insights into coronavirus evolution</span><span class="source" tagx="source" title="source">bioRxiv</span><span class="year" tagx="year" title="year">2020</span><span class="comment">2020.2002.2023.961912</span></span></li> <li tag="ref"><a name="bib0180" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yuan</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ye</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ding</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Prevalence and phylogenetic analysis of porcine diarrhea associated viruses in southern China from 2012 to 2018</span><span class="source" tagx="source" title="source">BMC Vet. Res.</span><span class="volume" tagx="volume" title="volume">15</span><span class="issue" tagx="issue" title="issue">1</span><span class="year" tagx="year" title="year">2019</span><span class="fpage" tagx="fpage" title="fpage">470</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31881873">31881873</a></span></span></li> <li tag="ref"><a name="bib0185" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ji</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jing</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Swine acute diarrhea syndrome coronavirus-induced apoptosis is caspase- and cyclophilin D- dependent</span><span class="source" tagx="source" title="source">Emerg. Microbes Infect.</span><span class="volume" tagx="volume" title="volume">9</span><span class="issue" tagx="issue" title="issue">1</span><span class="year" tagx="year" title="year">2020</span><span class="fpage" tagx="fpage" title="fpage">439</span><span class="lpage" tagx="lpage" title="lpage">456</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32090691">32090691</a></span></span></li> <li tag="ref"><a name="bib0190" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Q.N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">J.N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">G.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">Z.X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">R.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lan</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">J.Y.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The re-emerging of SADS-CoV infection in pig herds in Southern China</span><span class="source" tagx="source" title="source">Transbound. Emerg. Dis.</span><span class="volume" tagx="volume" title="volume">66</span><span class="issue" tagx="issue" title="issue">5</span><span class="year" tagx="year" title="year">2019</span><span class="fpage" tagx="fpage" title="fpage">2180</span><span class="lpage" tagx="lpage" title="lpage">2183</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31207129">31207129</a></span></span></li> <li tag="ref"><a name="bib0195" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lan</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">R.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">J.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">Z.X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">Q.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">X.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">J.Y.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Retrospective detection and phylogenetic analysis of swine acute diarrhea syndrome coronavirus in pigs in southern China</span><span class="source" tagx="source" title="source">Transbound. Emerg. Dis.</span><span class="volume" tagx="volume" title="volume">66</span><span class="issue" tagx="issue" title="issue">2</span><span class="year" tagx="year" title="year">2018</span><span class="fpage" tagx="fpage" title="fpage">687</span><span class="lpage" tagx="lpage" title="lpage">695</span></span></li> <li tag="ref"><a name="bib0200" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fan</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lan</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">X.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">W.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Y.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">Q.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mani</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">X.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pei</span><span class="given-names" tagx="given-names" title="given-names">G.Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">An</span><span class="given-names" tagx="given-names" title="given-names">X.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">J.W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mai</span><span class="given-names" tagx="given-names" title="given-names">K.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">Z.X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anderson</span><span class="given-names" tagx="given-names" title="given-names">D.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L.B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">S.Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mi</span><span class="given-names" tagx="given-names" title="given-names">Z.Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">T.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cong</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">P.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">X.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">X.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y.Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xing</span><span class="given-names" tagx="given-names" title="given-names">S.Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Y.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daszak</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">L.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">Z.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tong</span><span class="given-names" tagx="given-names" title="given-names">Y.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">J.Y.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Fatal swine acute diarrhoea syndrome caused by an HKU2-related coronavirus of bat origin</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">556</span><span class="issue" tagx="issue" title="issue">7700</span><span class="year" tagx="year" title="year">2018</span><span class="fpage" tagx="fpage" title="fpage">255</span><span class="lpage" tagx="lpage" title="lpage">258</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29618817">29618817</a></span></span></li> <li tag="ref"><a name="bib0205" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">X.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Si</span><span class="given-names" tagx="given-names" title="given-names">H.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">C.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">H.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jiang</span><span class="given-names" tagx="given-names" title="given-names">R.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">M.Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">X.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zheng</span><span class="given-names" tagx="given-names" title="given-names">X.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Q.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deng</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">L.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yan</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhan</span><span class="given-names" tagx="given-names" title="given-names">F.X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y.Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiao</span><span class="given-names" tagx="given-names" title="given-names">G.F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shi</span><span class="given-names" tagx="given-names" title="given-names">Z.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">579</span><span class="issue" tagx="issue" title="issue">7798</span><span class="year" tagx="year" title="year">2020</span><span class="fpage" tagx="fpage" title="fpage">270</span><span class="lpage" tagx="lpage" title="lpage">273</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32015507">32015507</a></span></span></li> <li tag="ref"><a name="bib0210" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">Z.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yan</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">X.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lan</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Swine acute diarrhea syndrome coronavirus (SADS-CoV) antagonizes interferon-beta production via blocking IPS-1 and RIG-I</span><span class="source" tagx="source" title="source">Virus Res.</span><span class="volume" tagx="volume" title="volume">278</span><span class="year" tagx="year" title="year">2019</span>197843</span></li> </ul> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>This study was supported by the </p><div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">10.13039/501100001809</span><span class="institution" title="institution">National Natural Science Foundation of China</span></div> </div> (31872488 and 32041003), <div class="funding-source" title="funding-source">
Zhejiang University special scientific research fund for COVID-19 prevention and control</div> (2020XGZX084), and the <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">10.13039/501100012226</span><span class="institution" title="institution">Fundamental Research Funds for the Central Universities of China</span></div> </div> (2019FZA6014).<p /> </div> </div>  </body></html>