<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/embed/graphic-17.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2018/12/04/484675.atom" />
<meta name="HW.pisa" content="biorxiv;484675v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Concerted 2-5A-Mediated mRNA Decay and Transcription Reprogram Protein Synthesis in dsRNA Response" />
<meta name="DC.Identifier" content="10.1101/484675" />
<meta name="DC.Date" content="2018-12-04" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2018, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="RNA degradation by RNase L during 2-5A-mediated decay (2-5AMD) is a conserved mammalian stress response to viral and endogenous double-stranded RNA (dsRNA). 2-5AMD onsets rapidly and facilitates a switch of protein synthesis from homeostasis to production of interferons (IFNs). To understand the mechanism of this protein synthesis reprogramming, we examined 2-5AMD in human cells. 2-5AMD triggers polysome collapse characteristic of a translation initiation defect, but translation initiation complexes and ribosomes purified from the translation-arrested cells remain functional. Using spike-in RNA-seq we found that basal messenger RNAs (mRNAs) rapidly decay, while mRNAs encoding IFNs and IFN-stimulated genes evade 2-5AMD and accumulate. The IFN evasion results from the combined effect of better mRNA stability and positive feedback amplification in the IFN response. Therefore, 2-5AMD and transcription act in concert to revamp the cellular mRNA composition. The resulting preferential accumulation of innate immune mRNAs establishes “prioritized” synthesis of defense proteins." />
<meta name="DC.Contributor" content="Sneha Rath" />
<meta name="DC.Contributor" content="Eliza Prangley" />
<meta name="DC.Contributor" content="Jesse Donovan" />
<meta name="DC.Contributor" content="Kaitlin Demarest" />
<meta name="DC.Contributor" content="Ned S. Wingreen" />
<meta name="DC.Contributor" content="Yigal Meir" />
<meta name="DC.Contributor" content="Alexei Korennykh" />
<meta name="article:published_time" content="2018-12-04" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Concerted 2-5A-Mediated mRNA Decay and Transcription Reprogram Protein Synthesis in dsRNA Response" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;RNA degradation by RNase L during 2-5A-mediated decay (2-5AMD) is a conserved mammalian stress response to viral and endogenous double-stranded RNA (dsRNA). 2-5AMD onsets rapidly and facilitates a switch of protein synthesis from homeostasis to production of interferons (IFNs). To understand the mechanism of this protein synthesis reprogramming, we examined 2-5AMD in human cells. 2-5AMD triggers polysome collapse characteristic of a translation initiation defect, but translation initiation complexes and ribosomes purified from the translation-arrested cells remain functional. Using spike-in RNA-seq we found that basal messenger RNAs (mRNAs) rapidly decay, while mRNAs encoding IFNs and IFN-stimulated genes evade 2-5AMD and accumulate. The IFN evasion results from the combined effect of better mRNA stability and positive feedback amplification in the IFN response. Therefore, 2-5AMD and transcription act in concert to revamp the cellular mRNA composition. The resulting preferential accumulation of innate immune mRNAs establishes “prioritized” synthesis of defense proteins.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2018/01/01" />
<meta name="citation_mjid" content="biorxiv;484675v1" />
<meta name="citation_id" content="484675v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/484675v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/484675v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/484675v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675.full.pdf" />
<meta name="citation_doi" content="10.1101/484675" />
<meta name="citation_num_pages" content="60" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="484675" />
<meta name="citation_author" content="Sneha Rath" />
<meta name="citation_author_institution" content="Department of Molecular Biology" />
<meta name="citation_author" content="Eliza Prangley" />
<meta name="citation_author_institution" content="Department of Molecular Biology" />
<meta name="citation_author" content="Jesse Donovan" />
<meta name="citation_author_institution" content="Department of Molecular Biology" />
<meta name="citation_author" content="Kaitlin Demarest" />
<meta name="citation_author_institution" content="Department of Molecular Biology" />
<meta name="citation_author" content="Ned S. Wingreen" />
<meta name="citation_author_institution" content="Department of Molecular Biology" />
<meta name="citation_author" content="Yigal Meir" />
<meta name="citation_author_institution" content="Department of Physics, Ben Gurion University" />
<meta name="citation_author" content="Alexei Korennykh" />
<meta name="citation_author_institution" content="Department of Molecular Biology" />
<meta name="citation_author_email" content="akorenny@princeton.edu" />
<meta name="citation_reference" content="Al-Ahmadi, W., Al-Haj, L., Al-Mohanna, F. A., Silverman, R. H., and Khabar, K. S. (2009). RNase L downmodulation of the RNA-binding protein, HuR, and cellular growth. Oncogene 28, 1782–1791." />
<meta name="citation_reference" content="Alisha Chitrakar, Sneha Rath, Jesse Donovan, Kaitlin Demarest, Yize Li, Raghavendra Rao Sridhar, Susan R. Weiss, Sergei V. Kotenko, Ned S. Wingreen, and Korennykh, A. (2018). Realtime 2-5A kinetics suggests interferons β and lambda evade global arrest of translation by RNase L. BioRxiv https://doiorg/101101/476341." />
<meta name="citation_reference" content="Banerjee, S., Li, G., Li, Y., Gaughan, C., Baskar, D., Parker, Y., Lindner, D. J., Weiss, S. R., and Silverman, R. H. (2015). RNase L is a negative regulator of cell migration. Oncotarget 6, 44360–44372." />
<meta name="citation_reference" content="Chakrabarti, A., Jha, B. K., and Silverman, R. H. (2011). New insights into the role of RNase L in innate immunity. J Interferon Cytokine Res 31, 49–57." />
<meta name="citation_reference" content="Chiappinelli, K. B., Strissel, P. L., Desrichard, A., Li, H., Henke, C., Akman, B., Hein, A., Rote, N. S., Cope, L. M., Snyder, A., et al. (2015). Inhibiting DNA Methylation Causes an Interferon Response in Cancer via dsRNA Including Endogenous Retroviruses. Cell 162, 974–986." />
<meta name="citation_reference" content="Civril, F., Deimling, T., de Oliveira Mann, C. C., Ablasser, A., Moldt, M., Witte, G., Hornung, V., and Hopfner, K. P. (2013). Structural mechanism of cytosolic DNA sensing by cGAS. Nature 498, 332–337." />
<meta name="citation_reference" content="Clemens, M. J., and Williams, B. R. (1978). Inhibition of cell-free protein synthesis by pppA2′p5′A2′p5′A: a novel oligonucleotide synthesized by interferon-treated L cell extracts. Cell 13, 565–572." />
<meta name="citation_reference" content="Cooper, D. A., Banerjee, S., Chakrabarti, A., Garcia-Sastre, A., Hesselberth, J. R., Silverman, R. H., and Barton, D. J. (2014a). Ribonuclease L Targets Distinct Sites in Influenza A Virus RNAs. J Virol." />
<meta name="citation_reference" content="Cooper, D. A., Jha, B. K., Silverman, R. H., Hesselberth, J. R., and Barton, D. J. (2014b). Ribonuclease L and metal-ion-independent endoribonuclease cleavage sites in host and viral RNAs. Nucleic Acids Res 42, 5202–5216." />
<meta name="citation_reference" content="Donovan, J., Dufner, M., and Korennykh, A. (2013). Structural basis for cytosolic double-stranded RNA surveillance by human oligoadenylate synthetase 1. Proc Natl Acad Sci U S A 110, 1652–1657." />
<meta name="citation_reference" content="Donovan, J., Rath, S., Kolet-Mandrikov, D., and Korennykh, A. (2017). Rapid RNase L-driven arrest of protein synthesis in the dsRNA response without degradation of translation machinery. RNA 23, 16601671." />
<meta name="citation_reference" content="Donovan, J., Whitney, G., Rath, S., and Korennykh, A. (2015). Structural mechanism of sensing long dsRNA via a noncatalytic domain in human oligoadenylate synthetase 3. Proc Natl Acad Sci U S A 112, 3949–3954." />
<meta name="citation_reference" content="Drappier, M., Jha, B. K., Stone, S., Elliott, R., Zhang, R., Vertommen, D., Weiss, S. R., Silverman, R. H., and Michiels, T. (2018). A novel mechanism of RNase L inhibition: Theiler’s virus L* protein prevents 2-5A from binding to RNase L. PLoS Pathog 14, e1006989." />
<meta name="citation_reference" content="Feldman, M. E., Apsel, B., Uotila, A., Loewith, R., Knight, Z. A., Ruggero, D., and Shokat, K. M. (2009). Active-site inhibitors of mTOR target rapamycin-resistant outputs of mTORC1 and mTORC2. PLoS Biol 7, e38." />
<meta name="citation_reference" content="Floyd-Smith, G., Slattery, E., and Lengyel, P. (1981). Interferon action: RNA cleavage pattern of a (2′-5′) oligoadenylate-dependent endonuclease. Science 212, 1030–1032." />
<meta name="citation_reference" content="Gandin, V., Sikstrom, K., Alain, T., Morita, M., McLaughlan, S., Larsson, O., and Topisirovic, I. (2014). Polysome fractionation and analysis of mammalian translatomes on a genomewide scale. J Vis Exp." />
<meta name="citation_reference" content="Gluck, A., Endo, Y., and Wool, I. G. (1994). The ribosomal RNA identity elements for ricin and for alpha-sarcin: mutations in the putative CG pair that closes a GAGA tetraloop. Nucleic Acids Res 22, 321–324." />
<meta name="citation_reference" content="Gusho, E., Zhang, R., Jha, B. K., Thornbrough, J. M., Dong, B., Gaughan, C., Elliott, R., Weiss, S. R., and Silverman, R. H. (2014). Murine AKAP7 has a 2′,5′-phosphodiesterase domain that can complement an inactive murine coronavirus ns2 gene. MBio 5, e01312–01314." />
<meta name="citation_reference" content="Han, J. Q., Wroblewski, G., Xu, Z., Silverman, R. H., and Barton, D. J. (2004). Sensitivity of hepatitis C virus RNA to the antiviral enzyme ribonuclease L is determined by a subset of efficient cleavage sites. J Interferon Cytokine Res 24, 664–676." />
<meta name="citation_reference" content="Han, Y., Donovan, J., Rath, S., Whitney, G., Chitrakar, A., and Korennykh, A. (2014). Structure of human RNase L reveals the basis for regulated RNA decay in the IFN response. Science 343, 1244–1248." />
<meta name="citation_reference" content="Han, Y., Whitney, G., Donovan, J., and Korennykh, A. (2012). Innate Immune Messenger 2-5A Tethers Human RNase L into Active High-Order Complexes. Cell Rep 2, 902–913." />
<meta name="citation_reference" content="Hovanessian, A. G., Brown, R. E., and Kerr, I. M. (1977). Synthesis of low molecular weight inhibitor of protein synthesis with enzyme from interferon-treated cells. Nature 268, 537–540." />
<meta name="citation_reference" content="Hsieh, A. C., Liu, Y., Edlind, M. P., Ingolia, N. T., Janes, M. R., Sher, A., Shi, E. Y., Stumpf, C. R., Christensen, C., Bonham, M. J., et al. (2012). The translational landscape of mTOR signalling steers cancer initiation and metastasis. Nature 485, 55–61." />
<meta name="citation_reference" content="Iordanov, M. S., Paranjape, J. M., Zhou, A., Wong, J., Williams, B. R., Meurs, E. F., Silverman, R. H., and Magun, B. E. (2000). Activation of p38 mitogen-activated protein kinase and c-Jun NH(2)-terminal kinase by double-stranded RNA and encephalomyocarditis virus: involvement of RNase L, protein kinase R, and alternative pathways. Mol Cell Biol 20, 617–627." />
<meta name="citation_reference" content="Iwasaki, S., Floor, S. N., and Ingolia, N. T. (2016). Rocaglates convert DEAD-box protein eIF4A into a sequence-selective translational repressor. Nature 534, 558–561." />
<meta name="citation_reference" content="Kawai, T., Takahashi, K., Sato, S., Coban, C., Kumar, H., Kato, H., Ishii, K. J., Takeuchi, O., and Akira, S. (2005). I PS-1, an adaptor triggering RIG-I-and Mda5-mediated type I interferon induction. Nat Immunol 6, 981–988." />
<meta name="citation_reference" content="Khatter, H., Myasnikov, A. G., Natchiar, S. K., and Klaholz, B. P. (2015). Structure of the human 80S ribosome. Nature 520, 640–645." />
<meta name="citation_reference" content="Korennykh, A. V., Egea, P. F., Korostelev, A. A., Finer-Moore, J., Zhang, C., Shokat, K. M., Stroud, R. M., and Walter, P. (2009). The unfolded protein response signals through high-order assembly of Ire1. Nature 457, 687–693." />
<meta name="citation_reference" content="Korennykh, A. V., Piccirilli, J. A., and Correll, C. C. (2006). The electrostatic character of the ribosomal surface enables extraordinarily rapid target location by ribotoxins. Nat Struct Mol Biol 13, 436–443." />
<meta name="citation_reference" content="Le Roy, F., Silhol, M., Salehzada, T., and Bisbal, C. (2007). Regulation of mitochondrial mRNA stability by RNase L is translation-dependent and controls IFNalpha-induced apoptosis. Cell Death Differ 14, 14061413." />
<meta name="citation_reference" content="Lee, K. P., Dey, M., Neculai, D., Cao, C., Dever, T. E., and Sicheri, F. (2008). Structure of the dual enzyme Ire1 reveals the basis for catalysis and regulation in nonconventional RNA splicing. Cell 132, 89-100." />
<meta name="citation_reference" content="Leonova, K. I., Brodsky, L., Lipchick, B., Pal, M., Novototskaya, L., Chenchik, A. A., Sen, G. C., Komarova, E. A., and Gudkov, A. V. (2013). p53 cooperates with DNA methylation and a suicidal interferon response to maintain epigenetic silencing of repeats and noncoding RNAs. Proc Natl Acad Sci U S A 110, E89–98." />
<meta name="citation_reference" content="Li, Y., Banerjee, S., Goldstein, S. A., Dong, B., Gaughan, C., Rath, S., Donovan, J., Korennykh, A., Silverman, R. H., and Weiss, S. R. (2017). Ribonuclease L mediates the cell-lethal phenotype of doublestranded RNA editing enzyme ADAR1 deficiency in a human cell line. Elife 6." />
<meta name="citation_reference" content="Li, Y., Banerjee, S., Wang, Y., Goldstein, S. A., Dong, B., Gaughan, C., Silverman, R. H., and Weiss, S. R. (2016). Activation of RNase L is dependent on OAS3 expression during infection with diverse human viruses. Proc Natl Acad Sci U S A 113, 2241–2246." />
<meta name="citation_reference" content="Liu, L., Botos, I., Wang, Y., Leonard, J. N., Shiloach, J., Segal, D. M., and Davies, D. R. (2008). Structural basis of toll-like receptor 3 signaling with double-stranded RNA. Science 320, 379–381." />
<meta name="citation_reference" content="Malathi, K., Dong, B., Gale, M., Jr., and Silverman, R. H. (2007). Small self-RNA generated by RNase L amplifies antiviral innate immunity. Nature 448, 816–819." />
<meta name="citation_reference" content="Marques-Ramos, A., Candeias, M. M., Menezes, J., Lacerda, R., Willcocks, M., Teixeira, A., Locker, N., and Romao, L. (2017). Cap-independent translation ensures mTOR expression and function upon protein synthesis inhibition. RNA 23, 1712–1728." />
<meta name="citation_reference" content="Michalska, A., Blaszczyk, K., Wesoly, J., and Bluyssen, H. A. R. (2018). A Positive Feedback Amplifier Circuit That Regulates Interferon (IFN)-Stimulated Gene Expression and Controls Type I and Type II IFN Responses. Front Immunol 9, 1135." />
<meta name="citation_reference" content="Nilsen, T. W., and Baglioni, C. (1979). Mechanism for discrimination between viral and host mRNA in interferon-treated cells. Proc Natl Acad Sci U S A 76, 2600–2604." />
<meta name="citation_reference" content="Nogimori, T., Nishiura, K., Kawashima, S., Nagai, T., Oishi, Y., Hosoda, N., Imataka, H., Kitamura, Y., Kitade, Y., and Hoshino, S. I. (2018). Dom34 mediates targeting of exogenous RNA in the antiviral OAS/RNase L pathway. Nucleic Acids Res." />
<meta name="citation_reference" content="Parsyan, A., Shahbazian, D., Martineau, Y., Petroulakis, E., Alain, T., Larsson, O., Mathonnet, G., Tettweiler, G., Hellen, C. U., Pestova, T. V., et al. (2009). The helicase protein DHX29 promotes translation initiation, cell proliferation, and tumorigenesis. Proc Natl Acad Sci U S A 106, 22217–22222." />
<meta name="citation_reference" content="Rath, S., Donovan, J., Whitney, G., Chitrakar, A., Wang, W., and Korennykh, A. (2015). Human RNase L tunes gene expression by selectively destabilizing the microRNA-regulated transcriptome. Proc Natl Acad Sci U S A 112, 15916–15921." />
<meta name="citation_reference" content="Taniuchi, S., Miyake, M., Tsugawa, K., Oyadomari, M., and Oyadomari, S. (2016). Integrated stress response of vertebrates is regulated by four eIF2alpha kinases. Sci Rep 6, 32886." />
<meta name="citation_reference" content="van den Elzen, A. M., Schuller, A., Green, R., and Seraphin, B. (2014). Dom34-Hbs1 mediated dissociation of inactive 80S ribosomes promotes restart of translation after stress. EMBO J 33, 265–276." />
<meta name="citation_reference" content="Zhou, A., Nie, H., and Silverman, R. H. (2000). Analysis and origins of the human and mouse RNase L genes: mediators of interferon action. Mamm Genome 11, 989–992." />
<meta name="citation_reference" content="Zhou, A., Paranjape, J., Brown, T. L., Nie, H., Naik, S., Dong, B., Chang, A., Trapp, B., Fairchild, R., Colmenares, C., and Silverman, R. H. (1997). Interferon action and apoptosis are defective in mice devoid of 2′,5′-oligoadenylate-dependent RNase L. Embo J 16, 6355–6363." />
<meta name="citation_reference" content="Zoncu, R., Efeyan, A., and Sabatini, D. M. (2011). mTOR: from growth signal integration to cancer, diabetes and ageing. Nat Rev Mol Cell Biol 12, 21–35." />
<meta name="citation_reference" content="Anders, S., Pyl, P. T., and Huber, W. (2015). HTSeq - a Python framework to work with high-throughput sequencing data. Bioinformatics 31, 166–169." />
<meta name="citation_reference" content="Bern, M., Kil, Y. J., and Becker, C. (2012). Byonic: advanced peptide and protein identification software. Curr Protoc Bioinformatics Chapter 13, Unit13 20." />
<meta name="citation_reference" content="Donovan, J., Rath, S., Kolet-Mandrikov, D., and Korennykh, A. (2017). Rapid RNase L-driven arrest of protein synthesis in the dsRNA response without degradation of translation machinery. RNA 23, 16601671." />
<meta name="citation_reference" content="Dorfer, V., Pichler, P., Stranzl, T., Stadlmann, J., Taus, T., Winkler, S., and Mechtler, K. (2014). MS Amanda, a universal identification algorithm optimized for high accuracy tandem mass spectra. J Proteome Res 13, 3679–3684." />
<meta name="citation_reference" content="Eng, J. K., McCormack, A. L., and Yates, J. R. (1994). An approach to correlate tandem mass spectral data of peptides with amino acid sequences in a protein database. J Am Soc Mass Spectrom 5, 976–989." />
<meta name="citation_reference" content="Gupta, N., DeMong, L. W., Banda, S., and Copeland, P. R. (2013). Reconstitution of selenocysteine incorporation reveals intrinsic regulation by SECIS elements. J Mol Biol 425, 2415–2422." />
<meta name="citation_reference" content="Kim, D., Pertea, G., Trapnell, C., Pimentel, H., Kelley, R., and Salzberg, S. L. (2013). TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions. Genome Biol 14, R36." />
<meta name="citation_reference" content="Lorsch, J. R., and Herschlag, D. (1999). Kinetic dissection of fundamental processes of eukaryotic translation initiation in vitro. EMBO J 18, 6705–6717." />
<meta name="citation_reference" content="Nesvizhskii, A. I., Keller, A., Kolker, E., and Aebersold, R. (2003). A statistical model for identifying proteins by tandem mass spectrometry. Anal Chem 75, 4646–4658." />
<meta name="citation_reference" content="Rath, S., Donovan, J., Whitney, G., Chitrakar, A., Wang, W., and Korennykh, A. (2015). Human RNase L tunes gene expression by selectively destabilizing the microRNA-regulated transcriptome. Proc Natl Acad Sci U S A 112, 15916–15921." />
<meta name="citation_reference" content="Robinson, J. T., Thorvaldsdottir, H., Winckler, W., Guttman, M., Lander, E. S., Getz, G., and Mesirov, J. P. (2011). Integrative genomics viewer. Nat Biotechnol 29, 24–26." />
<meta name="twitter:title" content="Concerted 2-5A-Mediated mRNA Decay and Transcription Reprogram Protein Synthesis in dsRNA Response" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="RNA degradation by RNase L during 2-5A-mediated decay (2-5AMD) is a conserved mammalian stress response to viral and endogenous double-stranded RNA (dsRNA). 2-5AMD onsets rapidly and facilitates a switch of protein synthesis from homeostasis to production of interferons (IFNs). To understand the mechanism of this protein synthesis reprogramming, we examined 2-5AMD in human cells. 2-5AMD triggers polysome collapse characteristic of a translation initiation defect, but translation initiation complexes and ribosomes purified from the translation-arrested cells remain functional. Using spike-in RNA-seq we found that basal messenger RNAs (mRNAs) rapidly decay, while mRNAs encoding IFNs and IFN-stimulated genes evade 2-5AMD and accumulate. The IFN evasion results from the combined effect of better mRNA stability and positive feedback amplification in the IFN response. Therefore, 2-5AMD and transcription act in concert to revamp the cellular mRNA composition. The resulting preferential accumulation of innate immune mRNAs establishes “prioritized” synthesis of defense proteins." />
<meta name="og-title" property="og:title" content="Concerted 2-5A-Mediated mRNA Decay and Transcription Reprogram Protein Synthesis in dsRNA Response" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/484675v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="RNA degradation by RNase L during 2-5A-mediated decay (2-5AMD) is a conserved mammalian stress response to viral and endogenous double-stranded RNA (dsRNA). 2-5AMD onsets rapidly and facilitates a switch of protein synthesis from homeostasis to production of interferons (IFNs). To understand the mechanism of this protein synthesis reprogramming, we examined 2-5AMD in human cells. 2-5AMD triggers polysome collapse characteristic of a translation initiation defect, but translation initiation complexes and ribosomes purified from the translation-arrested cells remain functional. Using spike-in RNA-seq we found that basal messenger RNAs (mRNAs) rapidly decay, while mRNAs encoding IFNs and IFN-stimulated genes evade 2-5AMD and accumulate. The IFN evasion results from the combined effect of better mRNA stability and positive feedback amplification in the IFN response. Therefore, 2-5AMD and transcription act in concert to revamp the cellular mRNA composition. The resulting preferential accumulation of innate immune mRNAs establishes “prioritized” synthesis of defense proteins." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2018-12-04" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/484675v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/484675v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/484675v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/484675v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/145109" />
    <title>Concerted 2-5A-Mediated mRNA Decay and Transcription Reprogram Protein Synthesis in dsRNA Response | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"bC5r7jbZ6HizJZrBX6TJombBjWMfZa-iAnKcQmjRbsc","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"145109","apath":"\/biorxiv\/early\/2018\/12\/04\/484675.atom","pisa":"biorxiv;484675v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;484675v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/484675v1","title":"Concerted 2-5A-Mediated mRNA Decay and Transcription Reprogram Protein Synthesis in dsRNA Response","identifier":"node\/145109"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Trr1\\\\\\\/rd5irT56PAmOfENAZEjGbRRbt31P9e0TqkyQTCeYPmTkWlwAHmj5u+YowGy9YaRMO\\\\\\\/zi5OzQ6brrAcl5wXl9yLzQJVV3kWnNbpNsEgdlvNjs7I2LmVOUyjAx3Ie0JRFeQ7BBIPbIsCmiS94xmMqOPKy+r8PTK1YwlZx4XY4TCQddhscYFWsGp6ubmyAvxZNfQr\\\\\\\/wTGARG8bUcBPWjhWcAsUn+CpOTe1\\\\\\\/IIWyN0uOBnTMFIQVa5F+\\\\\\\/4j69suCW3HpBmnjD6lED6tiXm2JIK+3GaCO5SCSvyudoN+rRCuWRjtMqFTlXuCFXzdM015kkWDYZeN33aj9+s2RE7qJlt+RROYZF1u7h95y9v0B4yRIB50ozw6UVkCftf8AjJAylrEVH2qWjekWxYUirSDlrNBqM9lUN7RdQfc6r9LLmuOJZ3\\\\\\\/M1ubFzNDy2qBIt\\\\\\\/trzndpFwWJHyYKSPidGXkC0gY3vIYHxtok5qk1gBkdCmFF5AS4+hssdw1pQMgVFG3hJPao0cEJOpu9jrfbTBIs2R\\\\\\\/YsOybhaHdDDGivMCkdzzcHN\\\\\\\/IVUqdNlwQsVLnOhxcc3JGjIGxY2TVE4UkDeuwXHtCocjjbr\\\\\\\/XVfBgMo96kJfYM4ijkpjbjnU93dhg7NtEKF8pAVl85y+dPagBkibFARRZiQTe5KVtNzaDr\\\\\\\/OrR7n8+dzjy7KxVcfS\\\\\\\/i8wBfe45f8xFdLOesFeINhgspbfmSUwpYhHPbGFiNV856+zjtNczh3FZ6oeYqbN0BnqqJ76PSRTPjUGXAeI1iJ5IQjcxU0I\\\\\\\/LbgbSGMGdfVWR8DzU+VRJvYf7aU4h1JQLmxwL22CyoBFWT2Mxv2GDN2TGWCM0K\\\\\\\/oOyqadWFGBBZ6I9jjAms1KtzMXbYqk5aj08zpXRU1ZMeuNwCVNXXpHqZtusOYVZyEjPYi\\\\\\\/zPrBIhI2qyhIOXa5cZw+FRrZhT8Z16X0X6lmwwhq9ugQKjmCZ0v0pkJQKDO9C7z6FT4cic3NzFmaTH05Rsx3dgDyPauWhsbi+XJpDpeRgEptKuMlSY9OwmBM71RPRs7bO4bD1JHOqvvAIwA8ch96IL9ZYEuA77WV0qjPjFDnEBTiAiQYTzGWS1S3wk8eKkhLwcJb4Fuy6uWztE91DHjoeCaB1SmetH774seXVvLTHRLVyKfqPtY+LOchVRpdeIirqOVlUgbF4xkRQkDDDy5ukWdMF+Jum1YCUn+Ft2Ol4Yu17N2sq1py3lbyf3aT8\\\\\\\/NPTJoDDVvnoH+Lqkf2QGK6XSjPiInuxS3Bru9JYFZmVsHpuQzIKhCTYHOqelRRLOHwigAxmAHZ0Nn8=\\\u0022,\\\u0022iv\\\u0022:\\\u0022RNDZCFS7Ezo7xoVwthBn9g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022427a5410d88fd3c81ec774518cd4a627383edf5417e1366aa0aa4ea87306ebd7\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/484675v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-145109 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/484675v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4gLzaIo7f1YhS1q0ielvfqnRZjlqobjM14QDIUBEbZ0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="145109" id="node145109" data-pisa="biorxiv;484675v1" data-pisa-master="biorxiv;484675" data-apath="/biorxiv/early/2018/12/04/484675.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Concerted 2-5A-Mediated mRNA Decay and Transcription Reprogram Protein Synthesis in dsRNA Response</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Sneha</span> <span class="nlm-surname">Rath</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Eliza</span> <span class="nlm-surname">Prangley</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jesse</span> <span class="nlm-surname">Donovan</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Kaitlin</span> <span class="nlm-surname">Demarest</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Ned S.</span> <span class="nlm-surname">Wingreen</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Yigal</span> <span class="nlm-surname">Meir</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Alexei</span> <span class="nlm-surname">Korennykh</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/484675 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node145109" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sneha Rath </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Molecular Biology</span>, Princeton, NJ 08544, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sneha%2BRath%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rath%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASneha%2BRath%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Eliza Prangley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Molecular Biology</span>, Princeton, NJ 08544, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eliza%2BPrangley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Prangley%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEliza%2BPrangley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jesse Donovan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Molecular Biology</span>, Princeton, NJ 08544, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jesse%2BDonovan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Donovan%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJesse%2BDonovan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kaitlin Demarest </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Molecular Biology</span>, Princeton, NJ 08544, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kaitlin%2BDemarest%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Demarest%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaitlin%2BDemarest%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ned S. Wingreen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Molecular Biology</span>, Princeton, NJ 08544, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ned%2BS.%2BWingreen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wingreen%20NS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANed%2BS.%2BWingreen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Yigal Meir </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Physics, Ben Gurion University</span>, Beer-Sheva 84105, <span class='nlm-country'>Israel</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yigal%2BMeir%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Meir%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYigal%2BMeir%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Alexei Korennykh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Molecular Biology</span>, Princeton, NJ 08544, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexei%2BKorennykh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Korennykh%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlexei%2BKorennykh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:akorenny@princeton.edu" class="" data-icon-position="" data-hide-link-title="0">akorenny@princeton.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/484675v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:145109" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:145109/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/484675v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:145109" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:145109/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/484675v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:145109" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:145109/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/484675v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:145109" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:145109/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/484675v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:145109" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:145109/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">RNA degradation by RNase L during 2-5A-mediated decay (2-5AMD) is a conserved mammalian stress response to viral and endogenous double-stranded RNA (dsRNA). 2-5AMD onsets rapidly and facilitates a switch of protein synthesis from homeostasis to production of interferons (IFNs). To understand the mechanism of this protein synthesis reprogramming, we examined 2-5AMD in human cells. 2-5AMD triggers polysome collapse characteristic of a translation initiation defect, but translation initiation complexes and ribosomes purified from the translation-arrested cells remain functional. Using spike-in RNA-seq we found that basal messenger RNAs (mRNAs) rapidly decay, while mRNAs encoding IFNs and IFN-stimulated genes evade 2-5AMD and accumulate. The IFN evasion results from the combined effect of better mRNA stability and positive feedback amplification in the IFN response. Therefore, 2-5AMD and transcription act in concert to revamp the cellular mRNA composition. The resulting preferential accumulation of innate immune mRNAs establishes “prioritized” synthesis of defense proteins.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-4">The innate immune system is activated without the delay needed for antibody production and provides an immediate defense barrier against infections and out-of-control host cells posing danger. In higher vertebrates, the innate immune system relies on interferon (IFN) signaling coupled with a vertebrae-specific pathway of regulated RNA degradation, 2-5A-mediated decay (2-5AMD) (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chakrabarti et al., 2011</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cooper et al., 2014b</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Donovan et al., 2017</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Rath et al., 2015</a>). 2-5AMD is activated in the presence of double-stranded RNA (dsRNA), a pathogen-associated and damage-associated molecular pattern that signals the presence of viruses (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Li et al., 2016</a>) and pathologic derepression of endogenous repeat elements of the host (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chiappinelli et al., 2015</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Leonova et al., 2013</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Li et al., 2017</a>).</p><p id="p-5">2-5AMD requires the coordinated activity of 2-5A oligonucleotide synthetases (OASs) and the pseudokinase-endoribonuclease, RNase L. The action of the OASs closely parallels that action of a structurally similar dsDNA sensor cGAS. cGAS synthesizes a second messenger cGAMP (cyclic-G<sub>2′,5′</sub>A<sub>3′,5′</sub>p) to activate the IFN response via the cGAMP receptor, STING (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Civril et al., 2013</a>). The OASs use the same mechanism of regulation, but function as sensors of dsRNA (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Civril et al., 2013</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Donovan et al., 2013</a>). The OASs synthesize the second messenger 2-5A (5′-ppp-A<sub>2′p5′</sub>A(<sub>2′p5′</sub>A)<sub>n≥0</sub>) to activate RNA decay by the 2-5A receptor, RNase L (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Chakrabarti et al., 2011</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Donovan et al., 2013</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Donovan et al., 2015</a>).</p><p id="p-6">RNase L is a mammalian pseudokinase-endoribonuclease uniquely related to the kinase-RNase Ire1 in the unfolded protein response (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Korennykh et al., 2009</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lee et al., 2008</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Zhou et al., 2000</a>). 2-5A binding to the ankyrin-repeat sensor domain of RNase L facilitates its dimerization and high-order oligomerization, which provides the switch for activation of RNA cleavage by 2-5AMD (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Han et al., 2014</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Han et al., 2012</a>). Upon activation, RNase L cleaves single-stranded RNA molecules at UN^N sites (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Floyd-Smith et al., 1981</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Han et al., 2014</a>). The prevalence of this short motif results in 2-5AMD sensitivity of many tRNAs, rRNAs, mRNAs, Y-RNAs and vault RNAs (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Cooper et al., 2014b</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Donovan et al., 2017</a>). During homeostasis, 2-5AMD regulates adhesion and migration activity of mammalian cells (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Banerjee et al., 2015</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Rath et al., 2015</a>). Upon acute activation by dsRNA, 2-5AMD arrests global translation by an immediate and poorly understood mechanism (Alisha <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Chitrakar et al., 2018</a>; <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Donovan et al., 2017</a>). In mammalian cells, dsRNA inhibits translation not only via 2-5AMD, but also via phosphorylation of the translation initiation factor eIF2α by the serine-threonine kinase protein kinase R (PKR) (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1A</a>). 2-5AMD and the PKR pathway are separated in time and in A549 human lung epithelial cells, 2-5AMD is the only mechanism responsible for the rapid translational inhibition (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Donovan et al., 2017</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The effect of 2-5AMD and RNase L activation on polysomes in A549 human cells. (A) Schematic overview of the dsRNA sensing pathways and the location of 2-5AMD. (B) Polysome sedimentation profiles in WT and RNase L-/- cells during poly I:C treatment. (C) Translation activity in WT and RNase L-/- cells after poly I:C transfection measured using 35S metabolic labeling. Total proteins stained with coomassie show lane loading. Independent measurements using an orthogonal readout are shown in related Fig. S1. (D) Quantification of new protein synthesis, normalized to loading control. Error bars are S.E. from three biological replicates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1172769221" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;The effect of 2-5AMD and RNase L activation on polysomes in A549 human cells.&lt;/span&gt; (A) Schematic overview of the dsRNA sensing pathways and the location of 2-5AMD. (B) Polysome sedimentation profiles in WT and RNase L&lt;sup&gt;-/-&lt;/sup&gt; cells during poly I:C treatment. (C) Translation activity in WT and RNase L&lt;sup&gt;-/-&lt;/sup&gt; cells after poly I:C transfection measured using &lt;sup&gt;35&lt;/sup&gt;S metabolic labeling. Total proteins stained with coomassie show lane loading. Independent measurements using an orthogonal readout are shown in related Fig. S1. (D) Quantification of new protein synthesis, normalized to loading control. Error bars are S.E. from three biological replicates.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F1.medium.gif" width="440" height="279"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F1.medium.gif" width="440" height="279"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">The effect of 2-5AMD and RNase L activation on polysomes in A549 human cells.</span><p id="p-7" class="first-child">(A) Schematic overview of the dsRNA sensing pathways and the location of 2-5AMD. (B) Polysome sedimentation profiles in WT and RNase L<sup>-/-</sup> cells during poly I:C treatment. (C) Translation activity in WT and RNase L<sup>-/-</sup> cells after poly I:C transfection measured using <sup>35</sup>S metabolic labeling. Total proteins stained with coomassie show lane loading. Independent measurements using an orthogonal readout are shown in related <a id="xref-fig-8-1" class="xref-fig" href="#F8">Fig. S1</a>. (D) Quantification of new protein synthesis, normalized to loading control. Error bars are S.E. from three biological replicates.</p><div class="sb-div caption-clear"></div></div></div><p id="p-8">Initial studies of 2-5AMD in rabbit reticulocyte lysate (RRL) reported polysome disaggregation and degradation of mRNAs (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Clemens and Williams, 1978</a>). RRL does not have transcription, which limits biologic interpretations of these results: in the cell-free system, the loss of mRNAs and polysomes due to a non-specific RNase is inevitable. In live cells, 2-5AMD has sufficient potency to cause degradation of 18S and 28S rRNAs. The characteristic rRNA cleavage pattern provides a reliable readout of RNase L activation (<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Donovan et al., 2017</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Malathi et al., 2007</a>). Furthermore, 2-5AMD causes degradation of tRNA-His and tRNA-Pro, as well as multiple mRNAs in live cells (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Al-Ahmadi et al., 2009</a>; <a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">Donovan et al., 2017</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Le Roy et al., 2007</a>; <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Rath et al., 2015</a>). Due to the complexity of the RNA degradation program, the arrest of global translation by 2-5AMD could not be linked to any specific RNA (Alisha <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Chitrakar et al., 2018</a>; <a id="xref-ref-11-7" class="xref-bibr" href="#ref-11">Donovan et al., 2017</a>).</p><p id="p-9">Shortly after activation of 2-5AMD, dsRNA triggers the transcriptional IFN response, leading to upregulation of innate immune mRNAs (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1A</a>) (Alisha <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Chitrakar et al., 2018</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kawai et al., 2005</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Liu et al., 2008</a>). While global translation remains silenced by actively ongoing 2-5AMD, the mRNAs encoding the interferons IFN-β (type I) and IFN-λ (type III) bypass the global inhibition and are actively translated (Alisha <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Chitrakar et al., 2018</a>). Here we employ cell biology, proteomics, transcriptomics and modeling approaches to determine the translation reprogramming mechanism of 2-5AMD and to explain how innate immune mRNAs can evade the translational arrest.</p></div><div class="section" id="sec-2"><h2 class="">Results</h2><div id="sec-3" class="subsection"><h3>2-5AMD Inhibits Translation Initiation without Disrupting Cap-binding Complex and 40S Subunit Loading</h3><p id="p-10">To begin deciphering the mechanism of protein synthesis regulation 2-5AMD, we examined whether it affects polysomes in WT and RNase L<sup>-/-</sup> cells by sucrose gradient sedimentation. In the presence of immunogenic dsRNA (poly-I:C), the 80S monosome peak became dominant within 30 minutes, whereas polysomes progressively disassembled (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1B</a>). In RNase L<sup>-/-</sup> cells that experience weak translation inhibition by PKR, the polysome profiles did not change until five hours, and the 80S peak never dominated. The changes in the polysome profiles were accompanied by a global, RNase L-dependent arrest of translation (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1C-D</a>; <a id="xref-fig-8-2" class="xref-fig" href="#F8">Fig. S1A-B</a>). The loss of polysomes and accumulation of the 80S monosomes during 2-5AMD is a signature of inhibited initiation of capped mRNAs. A similar polysome change takes place upon deletion of the RNA helicase DHX29 involved in recognition of the 5′-untranslated region (5′-UTR) (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Parsyan et al., 2009</a>) or inhibition of the mammalian target of rapamycin (mTOR) that maintains translation initiation active (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gandin et al., 2014</a>). The 80S species that form upon DHX29 and mTOR defects are non-translating, as are the 80S species formed during 2-5AMD (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1C</a>; <a id="xref-fig-8-3" class="xref-fig" href="#F8">Fig. S1A</a>). Non-translating 80S species devoid of mRNA form readily in A549 cells following translation release with puromycin (<a id="xref-fig-8-4" class="xref-fig" href="#F8">Fig. S1C</a>). Puromycin-induced and 2-5AMD-induced 80S monosomes are stable only at ∼100 mM KCl, but dissociate at 500 mM KCl (<a id="xref-fig-8-5" class="xref-fig" href="#F8">Fig. S1C-D</a>), as expected for vacant ribosomes (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">van den Elzen et al., 2014</a>).</p><p id="p-11">Inhibition of mTOR kinase is a common strategy to arrest bulk translation during stress responses (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hsieh et al., 2012</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Zoncu et al., 2011</a>), suggesting that inhibition of translation initiation by 2-5AMD could depend on mTOR. To test this link, we assessed mTOR activity by measuring phosphorylation of the translation initiation factor 4E-BP1 whose phosphorylation by mTOR is required for translation initiation (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Feldman et al., 2009</a>). Activation of 2-5AMD did not affect 4E-BP1 phosphorylation, whereas a control treatment with the small molecule mTOR inhibitor INK128 (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Feldman et al., 2009</a>) worked (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2B</a>). Considering that 2-5A and INK128 inhibited bulk translation comparably but with different effects on 4E-BP1 phosphorylation (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2A-B</a>), our data suggest that 2-5AMD inhibits translation initiation independently from mTOR.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Translation initiation status during 2-5AMD. (A) Activity of mTOR kinase monitored by western blot for phosphorylated 4E-BP1. Matching translational activity was measured by 35S metabolic labeling. The small molecule mTOR inhibitor INK128 is used as a control. (B) Quantification of the gels in (A). (C) Western blot analysis of core components of the cap-binding eIF4F complex: eIF4E, eIF4A and eIF4G. Data from control (Flag) and eIF4E immunoprecipitation (IP) experiments are shown with and without 2-5A treatment. (D) BioAnalyzer NanoChip profiling of rRNA in the samples in (C). (E) Mass spectrometry analysis of proteins that co-immunoprecipitate with the cap-binding translation initiation factor eIF4E (Table S1)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1172769221" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Translation initiation status during 2-5AMD.&lt;/span&gt; (A) Activity of mTOR kinase monitored by western blot for phosphorylated 4E-BP1. Matching translational activity was measured by &lt;sup&gt;35&lt;/sup&gt;S metabolic labeling. The small molecule mTOR inhibitor INK128 is used as a control. (B) Quantification of the gels in (A). (C) Western blot analysis of core components of the cap-binding eIF4F complex: eIF4E, eIF4A and eIF4G. Data from control (Flag) and eIF4E immunoprecipitation (IP) experiments are shown with and without 2-5A treatment. (D) BioAnalyzer NanoChip profiling of rRNA in the samples in (C). (E) Mass spectrometry analysis of proteins that co-immunoprecipitate with the cap-binding translation initiation factor eIF4E (Table S1).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F2.medium.gif" width="440" height="319"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F2.medium.gif" width="440" height="319"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Translation initiation status during 2-5AMD.</span><p id="p-12" class="first-child">(A) Activity of mTOR kinase monitored by western blot for phosphorylated 4E-BP1. Matching translational activity was measured by <sup>35</sup>S metabolic labeling. The small molecule mTOR inhibitor INK128 is used as a control. (B) Quantification of the gels in (A). (C) Western blot analysis of core components of the cap-binding eIF4F complex: eIF4E, eIF4A and eIF4G. Data from control (Flag) and eIF4E immunoprecipitation (IP) experiments are shown with and without 2-5A treatment. (D) BioAnalyzer NanoChip profiling of rRNA in the samples in (C). (E) Mass spectrometry analysis of proteins that co-immunoprecipitate with the cap-binding translation initiation factor eIF4E (Table S1).</p><div class="sb-div caption-clear"></div></div></div><p id="p-13">To test whether 2-5AMD disrupts assembly of cap binding initiation complexes, we pulled down the cap-binding initiation factor eIF4E and examined its association with the key partner factors eIF4A and eIF4G that together form the eIF4F complex. This tripartite complex was readily identified using the pulldown and remained unchanged by 2-5AMD (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2C</a>). Total RNA profiling by NanoChip revealed that eIF4E additionally pulled down the 40S ribosomal subunit both in naïve cells and in cells with activated 2-5AMD, suggesting normal loading of the small subunit. As expected, 18S and 28S rRNAs were degraded during 2-5AMD and exhibited the characteristic pattern of RNase L activity (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2D</a>). The 18S rRNA from the 40S subunit pulled down with eIF4E following 2-5AMD was cleaved as in the input rRNA. Thus, binding of the core components of the translation initiation complex is not disrupted during 2-5AMD.</p><p id="p-14">To examine the initiation complex more completely, we performed mass spectrometry (MS) of proteins that co-purify with eIF4E (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Fig. S2</a>). This analysis detected ribosomal proteins from the small subunit, translation initiation factors and poly-A binding proteins, but not proteins from the large ribosomal subunit, as expected due to polysome run off. The identified components did not change in response to 2-5A (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig. 2E</a>; Table S1), indicating 2-5AMD does not block the cap-binding complex and cap-mediated loading of the small ribosomal subunit.</p></div><div id="sec-4" class="subsection"><h3>2-5AMD Degrades 18S and 28S rRNAs but Leaves Ribosomes Functional</h3><p id="p-15">2-5AMD does not interfere with the initiation step. To further define the mechanism, we analyzed the hallmark substrate of RNase L, the ribosome, and examined whether 2-5AMD inhibits global translation by directly affecting the ribosomal translation activity. 18S and 28S rRNAs are both cleaved by RNase L (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Fig. 2D</a>) (<a id="xref-ref-11-8" class="xref-bibr" href="#ref-11">Donovan et al., 2017</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Malathi et al., 2007</a>). Although a high-resolution structure of the human 80S ribosome has become available (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Khatter et al., 2015</a>), mechanistic understanding of how rRNA cleavage by 2-5AMD could affect the ribosome is critically limited by the absence of reliable mapping of the cleavage sites. Based on primer extension analysis it has been proposed that RNase L cleaves nucleotides 4032, and to a smaller extent 4031, in 28S rRNA (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Iordanov et al., 2000</a>). In contrast, subsequent RNA-seq analyses of cleaved rRNAs captured using <em>Arabidopsis</em> tRNA ligase did not detect cleavage at either of the 28S rRNA sites, but observed predominantly RNase L-independent 18S/28S rRNA background cleavage events (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Cooper et al., 2014b</a>).</p><p id="p-16">To <em>de novo</em> identify the cleavage sites in human rRNA, we adapted RtcB RNA-seq suitable for single-nucleotide resolution mapping (<a id="xref-ref-11-9" class="xref-bibr" href="#ref-11">Donovan et al., 2017</a>). RtcB RNA-seq analysis of rRNA from cells with activated 2-5AMD (<a id="xref-fig-10-1" class="xref-fig" href="#F10">Fig. S3A</a>) revealed cleavage sites with a UN^N consensus (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3A</a>) that distinguishes the sequence-specific activity of RNase L (Table S2) (<a id="xref-ref-11-10" class="xref-bibr" href="#ref-11">Donovan et al., 2017</a>; <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Han et al., 2014</a>). We analyzed the data to identify predominant sites that simultaneously exhibit induction and high read count in the 2-5AMD sample, as shown in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>. A single high-scoring site 771 was detected in 18S rRNA and two dominant sites, 1056 and 4032, were detected in 28S rRNA. Our analysis detected both 28S rRNA sites 4031 and 4032 identified previously by primer extension assay (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Iordanov et al., 2000</a>), providing important validation for the RtcB RNA-seq approach. The dominant sites 771 (18S) and 1056 (28S) were detected for the first time.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of ribosomes during 2-5AMD. (A) RNase L cleavage positions in 18S and 28S rRNA found by RtcB RNA-seq. The Y-axis provides a unified metric of read abundance and cleavage induction strength. (B) Mapping the RNase L sites onto 3D structures of 18S and 28S rRNAs. Structures are colored by cleavage strength as defined in (A). (D) Status of rRNA and translation activity of purified ribosomes obtained from rabbit reticulocyte lysate (RRL) or A549 cells. A titration with purified ribosomes was done in the presence of 50 ng capped luciferase mRNA. New translation was measured by luminescence. (D) Comparison of translation loss and rRNA cleavage over the duration of dsRNA response. Fraction of intact rRNA observed by NanoChip was quantified in GelQuant.NET. New translation relative to the untreated condition was measured by 35S metabolic labeling and ribopuromycilation (Fig. 1C; S1A). Kinetic parameters of the time profiles are shown below the graph." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1172769221" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of ribosomes during 2-5AMD.&lt;/span&gt; (A) RNase L cleavage positions in 18S and 28S rRNA found by RtcB RNA-seq. The Y-axis provides a unified metric of read abundance and cleavage induction strength. (B) Mapping the RNase L sites onto 3D structures of 18S and 28S rRNAs. Structures are colored by cleavage strength as defined in (A). (D) Status of rRNA and translation activity of purified ribosomes obtained from rabbit reticulocyte lysate (RRL) or A549 cells. A titration with purified ribosomes was done in the presence of 50 ng capped luciferase mRNA. New translation was measured by luminescence. (D) Comparison of translation loss and rRNA cleavage over the duration of dsRNA response. Fraction of intact rRNA observed by NanoChip was quantified in GelQuant.NET. New translation relative to the untreated condition was measured by &lt;sup&gt;35&lt;/sup&gt;S metabolic labeling and ribopuromycilation (Fig. 1C; S1A). Kinetic parameters of the time profiles are shown below the graph.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F3.medium.gif" width="317" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F3.medium.gif" width="317" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">Analysis of ribosomes during 2-5AMD.</span><p id="p-17" class="first-child">(A) RNase L cleavage positions in 18S and 28S rRNA found by RtcB RNA-seq. The Y-axis provides a unified metric of read abundance and cleavage induction strength. (B) Mapping the RNase L sites onto 3D structures of 18S and 28S rRNAs. Structures are colored by cleavage strength as defined in (A). (D) Status of rRNA and translation activity of purified ribosomes obtained from rabbit reticulocyte lysate (RRL) or A549 cells. A titration with purified ribosomes was done in the presence of 50 ng capped luciferase mRNA. New translation was measured by luminescence. (D) Comparison of translation loss and rRNA cleavage over the duration of dsRNA response. Fraction of intact rRNA observed by NanoChip was quantified in GelQuant.NET. New translation relative to the untreated condition was measured by <sup>35</sup>S metabolic labeling and ribopuromycilation (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig. 1C</a>; <a id="xref-fig-8-6" class="xref-fig" href="#F8">S1A</a>). Kinetic parameters of the time profiles are shown below the graph.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">Mapping of the identified sites onto the 3D structure of human rRNA shows that except for the nucleotides 4031/4032 in the L1 stalk, 2-5AMD targets surface loops away from vital parts of the ribosome. The location of sensitive sites at distant ribosomal positions suggests that 2-5AMD is not optimized for targeting a defined ribosomal position, as observed with <em>bona fide</em> ribosome-inactivating nucleases such as <em>α</em>-sarcin (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gluck et al., 1994</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Korennykh et al., 2006</a>). The RNase L cleavage sites appear opportunistic rather than intended for ribosomal inhibition.</p><p id="p-19">To test this prediction, we directly assessed the translation activity of the cleaved ribosomes in rabbit reticulocyte lysate (RRL). We depleted this lysate of ribosomes by centrifugation (<a id="xref-fig-10-2" class="xref-fig" href="#F10">Fig. S3B</a>) and re-supplied intact ribosomes purified from RRL, A549 cells, or cleaved ribosomes from dsRNA-transfected human cells (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3C</a>). Using capped luciferase mRNA translation as the readout, we assessed the activity of each ribosome type. In ribosome-depleted RRL, luciferase translation was absent, suggesting that we obtained a suitable assay. Addition of either rabbit ribosomes from nuclease-treated RRL, intact human ribosomes, or human ribosomes with rRNA degraded by 2-5AMD, readily supported translation. We observed the same specific activity for intact and cleaved ribosomes, which we reproduced over a range of ribosomal concentrations to exclude saturation effects (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3C</a>). Therefore, 2-5AMD does not functionally damage human ribosomes even after a nearly complete degradation of full-length rRNA (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig. 3C</a>, last lane). In agreement with this observation, the single-exponential decay kinetics for rRNA lags behind the kinetics of translational shutdown (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Fig. 3D</a>). Our data and the previously reported disconnect between rRNA cleavage and translation (<a id="xref-ref-11-11" class="xref-bibr" href="#ref-11">Donovan et al., 2017</a>) together indicate that the loss of global translation during 2-5AMD involves a process physically distinct from rRNA degradation.</p></div><div id="sec-5" class="subsection"><h3>Spike-in poly-A<sup>+</sup> RNA-seq Reveals Global Decay of Messenger RNAs during 2-5AMD in Live Cells</h3><p id="p-20">In the presence of normal cap-dependent initiation and functional ribosomes, the loss of cell-wide protein synthesis (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Fig. 1C</a>, <a id="xref-fig-8-7" class="xref-fig" href="#F8">Fig. S1A-B</a>) may arise from cleavage of a non-ribosomal RNA essential for global translation. Cleavage of a tRNA would satisfy this criterion, and tRNA cleavage has been previously demonstrated (<a id="xref-ref-11-12" class="xref-bibr" href="#ref-11">Donovan et al., 2017</a>). However, even the most sensitive tRNAs were intact at the time of translational inhibition, suggesting that the only RNA substrates that could account for the translational inhibition are mRNAs. RNase L has been shown to cleave exogenous mRNAs, viral mRNAs and select host mRNAs with a preference for longer RNAs with many AU-rich elements due to the specificity of RNase L for UN^N sites (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Al-Ahmadi et al., 2009</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Le Roy et al., 2007</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Nogimori et al., 2018</a>; <a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Rath et al., 2015</a>). To produce the uniform loss of global translation by mRNA decay, 2-5AMD must act similarly on all housekeeping mRNAs. Indeed, there are no protein bands or protein groups that stand out during the time-dependent progressive loss of translation (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Fig. 1C</a>, <a id="xref-fig-8-8" class="xref-fig" href="#F8">S1A-B</a>).</p><p id="p-21">To test whether a uniform mRNA decay is taking place, we used qPCR that we designed to detect full-length mRNAs (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4A</a>). Using this assay we observed a rapid decay of a several abundant basal mRNAs (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4B</a>). As expected, the decay was absent in RNase L<sup>-/-</sup> cells (<a id="xref-fig-11-1" class="xref-fig" href="#F11">Fig. S4A</a>). All tested mRNAs were cleaved rapidly. The decay traces for all mRNAs leveled before 100% cleavage, suggesting that cells have 2-5AMD-sensitive and 2-5AMD-resistant mRNA pools. The size of the resistant fraction was higher for PRKDC and SON mRNAs (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4B</a>, dotted lines). By quantifying log-linear regions of the data (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig. 4B</a>, inset graphs), we determined first-order decay kinetics for each mRNA (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig. 4C</a>). The housekeeping mRNAs decay considerably faster than rRNA, and on the same time scale as the translational arrest (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Fig. 3D</a> vs <a id="xref-fig-4-6" class="xref-fig" href="#F4">4C</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of decay kinetics for select basal mRNAs during 2-5AMD. (A) Design of qPCR for detection of full-length mRNAs. (B) Decay of select highly expressed housekeeping mRNAs upon poly I:C treatment of A549 cells measured by qPCR. The decay of SON and PKRDC shows distinctly biphasic character indicating the presence of non-cleaved mRNA fraction (2-5AMD-resistant pool) in the cells. The inset graphs show log-linear parts of the decay profiles used to measure the first-order decay kinetic parameters. Dotted lines mark approximate levels of 2-5AMD-resistant mRNA fractions. (C) Decay half-life (T1/2) obtained for each mRNA from the qPCR in (B). The last line shows T1/2 for 5,000 mRNAs calculated based on RNA-seq (Fig S4B). (*) Excluding GAPDH due to its stability." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1172769221" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Analysis of decay kinetics for select basal mRNAs during 2-5AMD.&lt;/span&gt; (A) Design of qPCR for detection of full-length mRNAs. (B) Decay of select highly expressed housekeeping mRNAs upon poly I:C treatment of A549 cells measured by qPCR. The decay of SON and PKRDC shows distinctly biphasic character indicating the presence of non-cleaved mRNA fraction (2-5AMD-resistant pool) in the cells. The inset graphs show log-linear parts of the decay profiles used to measure the first-order decay kinetic parameters. Dotted lines mark approximate levels of 2-5AMD-resistant mRNA fractions. (C) Decay half-life (T&lt;sub&gt;1/2&lt;/sub&gt;) obtained for each mRNA from the qPCR in (B). The last line shows T&lt;sub&gt;1/2&lt;/sub&gt; for 5,000 mRNAs calculated based on RNA-seq (Fig S4B). (*) Excluding GAPDH due to its stability.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F4.medium.gif" width="331" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F4.medium.gif" width="331" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">Analysis of decay kinetics for select basal mRNAs during 2-5AMD.</span><p id="p-22" class="first-child">(A) Design of qPCR for detection of full-length mRNAs. (B) Decay of select highly expressed housekeeping mRNAs upon poly I:C treatment of A549 cells measured by qPCR. The decay of SON and PKRDC shows distinctly biphasic character indicating the presence of non-cleaved mRNA fraction (2-5AMD-resistant pool) in the cells. The inset graphs show log-linear parts of the decay profiles used to measure the first-order decay kinetic parameters. Dotted lines mark approximate levels of 2-5AMD-resistant mRNA fractions. (C) Decay half-life (T<sub>1/2</sub>) obtained for each mRNA from the qPCR in (B). The last line shows T<sub>1/2</sub> for 5,000 mRNAs calculated based on RNA-seq (<a id="xref-fig-11-2" class="xref-fig" href="#F11">Fig S4B</a>). (*) Excluding GAPDH due to its stability.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">Using poly-A<sup>+</sup> RNA-seq we extended our analysis to the transcriptome. Widely used RNA-seq normalization and differential expression analysis techniques presume that levels of most mRNAs remain unchanged. This assumption would be violated if 2-5AMD inhibited translation by global mRNA decay. To correctly quantify mRNA levels during the course of decay, we supplemented our RNA samples with internal standards (<em>Drosophila Melanogaster</em> RNA spike-ins). 2-5AMD profiling revealed a time-dependent loss of almost all cellular mRNAs (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5A</a>; Tables S3). The RNA-seq data agreed well with our qPCR analysis (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5B</a>). Reads for all decaying mRNAs disappeared across the entire transcript length, suggesting that once RNase L endonucleolytically cleaves a transcript, the resulting mRNA fragments are rapidly cleared. The decay kinetics determined from the top 5000 most abundant transcripts matches the time scale of translational inhibition (<a id="xref-fig-11-3" class="xref-fig" href="#F11">Fig. S4B</a>; 4C). Together, our qPCR and RNA-seq data link translation arrest by 2-5AMD to mRNA decay, which can explain the accumulation of empty 80S monosomes in cells.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="RNA-seq profiling of mRNA decay by RNase L in live cells and in cytosolic extracts. (A) Time-dependent decay of the most abundant 5,000 mRNAs measured by spike-in RNA-seq. To obtain mRNAs levels, total reads for each sample were normalized using D. melanogaster RNA spike in as an internal standard. Transcripts are ordered from the highest to the lowest expression level in the untreated sample. (B) RNA-seq profiles for select individual mRNAs from (A). (C) Cleavage profiles and kinetic parameters for the mRNAs in (B) obtained in cell-free experiments (Rath et al., 2015)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1172769221" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;RNA-seq profiling of mRNA decay by RNase L in live cells and in cytosolic extracts.&lt;/span&gt; (A) Time-dependent decay of the most abundant 5,000 mRNAs measured by spike-in RNA-seq. To obtain mRNAs levels, total reads for each sample were normalized using &lt;em&gt;D. melanogaster&lt;/em&gt; RNA spike in as an internal standard. Transcripts are ordered from the highest to the lowest expression level in the untreated sample. (B) RNA-seq profiles for select individual mRNAs from (A). (C) Cleavage profiles and kinetic parameters for the mRNAs in (B) obtained in cell-free experiments (Rath et al., 2015).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F5.medium.gif" width="288" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F5.medium.gif" width="288" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <span class="caption-title">RNA-seq profiling of mRNA decay by RNase L in live cells and in cytosolic extracts.</span><p id="p-24" class="first-child">(A) Time-dependent decay of the most abundant 5,000 mRNAs measured by spike-in RNA-seq. To obtain mRNAs levels, total reads for each sample were normalized using <em>D. melanogaster</em> RNA spike in as an internal standard. Transcripts are ordered from the highest to the lowest expression level in the untreated sample. (B) RNA-seq profiles for select individual mRNAs from (A). (C) Cleavage profiles and kinetic parameters for the mRNAs in (B) obtained in cell-free experiments (<a id="xref-ref-42-5" class="xref-bibr" href="#ref-42">Rath et al., 2015</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">Of note, evaluation of RNase L activity in cytosolic cell extracts showed that mRNA decay depends on both, mRNA length and AU content (<a id="xref-ref-42-6" class="xref-bibr" href="#ref-42">Rath et al., 2015</a>) (<a id="xref-fig-12-1" class="xref-fig" href="#F12">Fig. S5</a>). In S10 cytosolic extracts treated with 2-5A, at the time point when ∼60% ACTB mRNA still remains, only 0.1% of FAT1 mRNA and 0.2% of PRKDC mRNAs survive (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig. 5C</a>). The high sensitivity of FAT1 and PRKDC transcripts in the S10 extract is in line with their lower GC content and greater length leading to more net UN^N sites per mRNA. In contrast to these findings, 2-5AMD in live cells shows no dependence on GC content and leads to decay of PRKDC, FAT1, ACTB and most other mRNAs with comparable, within several-fold, kinetics (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Fig. 5A-B</a>; Discussion). Therefore, cellular mRNA decay agrees with the fast timing of translational inhibition and the loss of global protein synthesis.</p></div><div id="sec-6" class="subsection"><h3>Decay and Synthesis Kinetics Protect IFN mRNAs from 2-5AMD</h3><p id="p-26">Spike-in RNA-seq indicates that the 2-5AMD-sensitive RNAs that decay (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig. 6A</a>, 88-89% of poly-A<sup>+</sup> RNA) account for more than 99.7% of protein synthesis (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Fig. 1C</a>, <a id="xref-fig-8-9" class="xref-fig" href="#F8">S1A</a>). These data suggest that RNase L eliminates the most actively translating mRNAs. As much as 11-12% of the poly-A<sup>+</sup> RNA is resistant (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig. 6A</a>), indicating that resistant mRNAs must be shielded from RNase L, perhaps by being in the nucleus, in phase-separated complexes or in stress granules. The pool of RNase L-resistant poly-A<sup>+</sup> transcripts is enriched with non-coding RNAs (ncRNAs; <a id="xref-fig-6-3" class="xref-fig" href="#F6">Fig. 6A</a>), suggesting that translation could render mRNAs more sensitive to RNase L compared to ncRNAs. The potential dependence on translation appears to be in line with the recently proposed model for translation-dependent RNase L cleavage based on analysis of decay of exogenous RNA (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Nogimori et al., 2018</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Poly-A+ transcriptome composition and dynamics during 2-5AMD. (A) Time evolution of the poly-A+ transcriptome obtained from spike-in RNA-seq. Induced transcripts (red) were upregulated by &#x2265; 5-fold at four hours of poly I:C treatment. (B) Illustration of read increase for a gene that is induced. (C) Levels of select basal and defense mRNAs measured by spike-in RNAs-seq. (D) Induction of innate immune mRNAs in response to poly I:C with and without 2-5AMD. (E) Experimental outline for measurement of innate immune mRNA decay. Innate immune mRNAs are allowed to accumulate followed by transcriptional inhibition with actinomycin D and decay profiling. (F) Kinetics parameters of defense mRNAs measured as in Fig. 4B. P-value compares T1/2 for defense vs basal mRNAs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1172769221" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Poly-A&lt;sup&gt;+&lt;/sup&gt; transcriptome composition and dynamics during 2-5AMD.&lt;/span&gt; (A) Time evolution of the poly-A+ transcriptome obtained from spike-in RNA-seq. Induced transcripts (red) were upregulated by &#x2265; 5-fold at four hours of poly I:C treatment. (B) Illustration of read increase for a gene that is induced. (C) Levels of select basal and defense mRNAs measured by spike-in RNAs-seq. (D) Induction of innate immune mRNAs in response to poly I:C with and without 2-5AMD. (E) Experimental outline for measurement of innate immune mRNA decay. Innate immune mRNAs are allowed to accumulate followed by transcriptional inhibition with actinomycin D and decay profiling. (F) Kinetics parameters of defense mRNAs measured as in Fig. 4B. P-value compares T&lt;sub&gt;1/2&lt;/sub&gt; for defense vs basal mRNAs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F6.medium.gif" width="344" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F6.medium.gif" width="344" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6.</span> <span class="caption-title">Poly-A<sup>+</sup> transcriptome composition and dynamics during 2-5AMD.</span><p id="p-27" class="first-child">(A) Time evolution of the poly-A+ transcriptome obtained from spike-in RNA-seq. Induced transcripts (red) were upregulated by ≥ 5-fold at four hours of poly I:C treatment. (B) Illustration of read increase for a gene that is induced. (C) Levels of select basal and defense mRNAs measured by spike-in RNAs-seq. (D) Induction of innate immune mRNAs in response to poly I:C with and without 2-5AMD. (E) Experimental outline for measurement of innate immune mRNA decay. Innate immune mRNAs are allowed to accumulate followed by transcriptional inhibition with actinomycin D and decay profiling. (F) Kinetics parameters of defense mRNAs measured as in <a id="xref-fig-4-7" class="xref-fig" href="#F4">Fig. 4B</a>. P-value compares T<sub>1/2</sub> for defense vs basal mRNAs.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">As basal mRNAs decay, defense mRNAs encoding IFNs and ISGs are upregulated (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Fig. 6A</a>). The mRNAs are initially absent, but by four hours IFN/ISG mRNAs account for 25% of the poly-A<sup>+</sup> pool. In log-linear coordinates, the loss of basal mRNAs and the increase in innate immune transcripts obeys a linear law (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Fig. 6C</a>, <a id="xref-fig-14-1" class="xref-fig" href="#F14">S7A</a>). The induction of defense transcripts bypasses RNase L (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Fig. 6D</a>; <a id="xref-fig-13-1" class="xref-fig" href="#F13">S6A</a>; <a id="xref-fig-14-2" class="xref-fig" href="#F14">S7A</a>), as previously reported (Alisha <a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Chitrakar et al., 2018</a>). To test whether defense mRNAs could survive in the presence of RNase L, we measured their decay kinetics during 2-5AMD in the presence of actinomycin D treatment, which was used to stop new transcription (<a id="xref-fig-6-7" class="xref-fig" href="#F6">Fig. 6E</a>). RNase L-dependent decay of the innate immune mRNAs was readily detected (<a id="xref-fig-6-8" class="xref-fig" href="#F6">Fig. 6F</a>; <a id="xref-fig-13-2" class="xref-fig" href="#F13">S6B</a>), indicating that defense mRNAs can be cleaved by RNase L. However, the decay half-lives (T<sub>1/2</sub>) for defense mRNAs were ∼2-3-fold longer compared to those of basal mRNAs (<a id="xref-fig-6-9" class="xref-fig" href="#F6">Fig. 6F</a>).</p><p id="p-29">The log-linear decay of basal mRNAs indicates that 2-5AMD degrades these transcripts with single-exponential kinetics. In contrast, the log-linear increase of IFN/ISG mRNAs does not match single-exponential accumulation or steady influx laws, and indicates induction with a positive feedback (<a id="xref-fig-14-3" class="xref-fig" href="#F14">Fig. S7B</a>; <a id="xref-fig-14-4" class="xref-fig" href="#F14">S7A-D</a>; Methods). In accord with our data, the presence of positive feedback has been previously described for IFN signaling (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Michalska et al., 2018</a>). It is important to note that positive feedback in the IFN response involves not only RNase L-sensitive mediators (IFN/ISG mRNAs), but also RNase L-resistant activators (accumulation of IFN proteins and phosphorylation of the transcription factor STAT) (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Fig. 7A</a>). We developed a mathematical description of this positive feedback model to examine whether it predicts survival of defense mRNAs in the presence of RNase L. To define experimental parameters, we used the key observations that (1) basal mRNAs are downregulated by ∼100-fold after 4 hours, (2) 2-5AMD has a small effect on the dynamics of defense mRNAs, and (3) measured half-lives of defense mRNAs are longer than half-lives of basal mRNAs (<a id="xref-fig-14-5" class="xref-fig" href="#F14">Fig. S7A</a>; <a id="xref-fig-6-10" class="xref-fig" href="#F6">Fig. 6F</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Proposed model for coordination between transcription and decay in 2-5AMD. (A) A kinetic scheme for 2-5AMD and the IFN response with positive feedback. Parameters I (IFN mRNAs), A (2-5AMD-resistant stable activator within positive feedback), &#x3B1; and &#x3B2; are described in &#x201C;Decay and transcriptional dynamics analysis section&#x201D; (Methods). (B) Modeled dynamics of basal and defense mRNAs when all transcripts have the same sensitivity to 2-5AMD. Defense mRNAs are induced with positive feedback. (C) Model in (B) modified to include a stable activator of positive feedback. (D) Model in (B) modified to include both, a stable activator and stability of defense mRNAs. The modeling analysis in panels A-C is described in Methods. (D) The proposed model for mRNA decay and IFN transcriptional response leading to RNase L-mediated reorganization global translation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1172769221" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Proposed model for coordination between transcription and decay in 2-5AMD.&lt;/span&gt; (A) A kinetic scheme for 2-5AMD and the IFN response with positive feedback. Parameters I (IFN mRNAs), A (2-5AMD-resistant stable activator within positive feedback), &#x3B1; and &#x3B2; are described in &#x201C;Decay and transcriptional dynamics analysis section&#x201D; (Methods). (B) Modeled dynamics of basal and defense mRNAs when all transcripts have the same sensitivity to 2-5AMD. Defense mRNAs are induced with positive feedback. (C) Model in (B) modified to include a stable activator of positive feedback. (D) Model in (B) modified to include both, a stable activator and stability of defense mRNAs. The modeling analysis in panels A-C is described in Methods. (D) The proposed model for mRNA decay and IFN transcriptional response leading to RNase L-mediated reorganization global translation.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F7.medium.gif" width="326" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7." src="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F7.medium.gif" width="326" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/12/04/484675/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7.</span> <span class="caption-title">Proposed model for coordination between transcription and decay in 2-5AMD.</span><p id="p-30" class="first-child">(A) A kinetic scheme for 2-5AMD and the IFN response with positive feedback. Parameters I (IFN mRNAs), A (2-5AMD-resistant stable activator within positive feedback), α and β are described in “Decay and transcriptional dynamics analysis section” (Methods). (B) Modeled dynamics of basal and defense mRNAs when all transcripts have the same sensitivity to 2-5AMD. Defense mRNAs are induced with positive feedback. (C) Model in (B) modified to include a stable activator of positive feedback. (D) Model in (B) modified to include both, a stable activator and stability of defense mRNAs. The modeling analysis in panels A-C is described in Methods. (D) The proposed model for mRNA decay and IFN transcriptional response leading to RNase L-mediated reorganization global translation.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">A simple model with positive feedback and similar 2-5AMD sensitivity of all mRNAs predicts equal attenuation by RNase L of basal and transcriptionally induced mRNAs (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Fig. 7B</a>). Therefore, positive feedback <em>per se</em> is not protecting defense mRNAs from 2-5AMD. In contrast, protection is observed once a stable activator is included in the positive feedback loop (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Fig. 7C</a>). The protection arises from the inherent resistance of proteins (IFNs, phosphorylated STAT) to RNase L. IFN proteins steadily accumulate and promote increasing transcriptional activity even in the presence of 2-5AMD. The evasion from 2-5AMD is further enhanced once 2.6-fold greater mRNA stability is incorporated in the model (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Fig. 7D</a>). Our modeling results show that longer half-lives of defense mRNAs, as well as, independently, a positive feedback loop built around a stable activator can result in evasion of defense mRNAs from global decay.</p></div></div><div class="section" id="sec-7"><h2 class="">Discussion</h2><p id="p-32">Our work suggests that in mammalian cells decay of abundant mRNAs can be efficiently coupled with a kinetically matched transcriptional response to steer translation toward stress proteins. Since the discovery of translational inhibition by 2-5AMD in 1977 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hovanessian et al., 1977</a>), various hypotheses have been proposed to explain how this mechanism contributes to the IFN response and how mammals may use 2-5AMD to cope with stress conditions associated with dsRNA. The hypothesized functions ranged from antiviral defense due to viral RNA decay (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cooper et al., 2014a</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Han et al., 2004</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Nilsen and Baglioni, 1979</a>) to IFN amplification by cleaved self-RNAs (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Malathi et al., 2007</a>) or destruction of host RNAs as a mechanism to kill infected cells during the late pro-apoptotic stages of dsRNA sensing (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Chakrabarti et al., 2011</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Zhou et al., 1997</a>). The observations that 2-5AMD can begin early and precede the IFN response, and that IFNs bypass 2-5AMD (Alisha <a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">Chitrakar et al., 2018</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Donovan et al., 2017</a>) revealed unanticipated translational reprogramming in this pathway, leading to the present investigation. Our model, based on analysis of 2-5AMD in human A549 cells, is summarized in <a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7F</a>. We show that the loss of protein synthesis can be explained by depletion of thousands of host mRNAs. The same mechanistic conclusion is made in a manuscript submitted back to back with our work (Burke et al., and Roy Parker [ref will be added]; See not added on submission). This group too attributed the action of RNase L to global loss of mRNAs. We show that approximately 90% of the total mRNA pool (by mass) is sensitive to 2-5AMD and accounts for nearly the entire translational activity detected by both metabolic and ribopuromycilation assays. The remaining 11-12% of the poly-A<sup>+</sup> transcriptome is neither accessible to RNase L nor supporting translation, suggesting that RNase L may have a mechanism to preferentially target translationally-active endogenous mRNAs.</p><p id="p-33">RNase L cleaves most basal mRNAs with similar rate constants and does not exhibit a preference for longer and AU-rich mRNAs. The uniform mRNA decay is a central feature of 2-5AMD, which is responsible for arrest of all housekeeping proteins rather than just proteins encoded by long and AU-rich mRNAs. We propose that this uniformity indicates that in live cells mRNAs are cleaved in Briggs-Haldane kinetic regime (Methods). Under Briggs-Haldane conditions, RNase L will cleave mRNAs independently of the binding (K<sub>m</sub>) and catalytic constants (k<sub>cat</sub>) of individual mRNAs, thereby acquiring a mechanism for uniform decay of the transcriptome. A notable feature of Briggs-Haldane regime is that once RNase L encounters an mRNA, it makes a cut before dissociating, i.e. every RNase L-mRNA binding event is productive. In cell extracts, RNase L is sensitive to length and AU content in mRNAs, indicating cleavage under Michaelis-Menten regime. Although it remains to be explained precisely how live cells achieve Briggs-Haldane regime, our observations could be explained by ribosome-assisted RNase L access to mRNAs. This model agrees with the recently proposed mechanism for Dom34 rescue of RNase L-ribosome complex, which postulates a translation-dependent mRNA cleavage mechanism (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Nogimori et al., 2018</a>). If ribosome-RNase L recognition (rather than mRNA-RNase L recognition) determines kinetics mRNA decay by RNase L, 2-5AMD would target all actively translating mRNAs. Further, RNase L could dwell on translating ribosomes, which would ensure efficient cleavage of mRNAs and achieve Briggs-Haldane conditions.</p><p id="p-34">The decay rate constants are similar, within several-fold, for basal mRNAs and for mRNAs encoding IFNs (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Fig. 4C</a>; 6F). We show that once this several-fold stability advantage is coupled with the positive feedback of the IFN response, defense mRNAs become desensitized to RNase L. A minimal model with experimentally determined kinetic parameters can account for decay of basal mRNAs and explain how IFNs and ISGs can accumulate to nearly the same levels in WT and RNase L<sup>-/-</sup> cells (<a id="xref-fig-7-6" class="xref-fig" href="#F7">Fig. 7A-B</a>). The experimental observation that 2-5AMD can clear the cytoplasm from unneeded host mRNAs without compromising the innate immune system suggest that, by analogy, 2-5AMD could also eliminate viral mRNAs while allowing the IFN response to develop. The antiviral activity of RNase L poses a serious obstacle for some viruses, which evolved dedicated viral proteins to disarm 2-5AMD usually by degrading 2-5A or binding to RNase L (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Drappier et al., 2018</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gusho et al., 2014</a>). Our data raise the possibility that some viruses may alternatively rely on replication kinetics as a strategy of escape from the innate immune action of 2-5AMD, without using inhibitory viral proteins. Understanding whether viruses take advantage of such a kinetic mechanism will be important for understanding and improvement of antiviral treatments.</p><p id="p-35">Reprogramming protein synthesis is a central strategy by which mammalian cells achieve energy conservation and adapt to stressful environments. To our knowledge, the mechanism of prioritizing stress protein translation by 2-5AMD is different from mechanisms of previously described mammalian pathways. Whereas mammalian protein synthesis is usually regulated by interference with translation initiation factors (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Iwasaki et al., 2016</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Marques-Ramos et al., 2017</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Taniuchi et al., 2016</a>), 2-5AMD acts directly on messenger RNAs. If global mRNA decay is matched with a complementary transcriptional response, the transcriptional induction can outpace the decay. In the described example, activation of this mechanism orchestrates cell commitment to the IFN response.</p><div id="sec-8" class="subsection"><h3>Database entries</h3><p id="p-36">RNA-seq data have been submitted to GEO database under accession ID GSE123034.</p></div></div><div class="section" id="sec-9"><h2 class="">Funding</h2><p id="p-38">This study was funded by NIH grant 1R01GM110161-01 (to A.K.), Sidney Kimmel Foundation grant AWD1004002 (to A.K.), Burroughs Wellcome Foundation Grant 1013579 (to A.K.), The Vallee Foundation (A.K.); by NIH grants 5T32GM007388 and F99 CA212468-01 (to S.R.), and NSF PHY-1607612 grant to N.S.W.</p></div><div class="section COI-statement" id="sec-10"><h2 class="">Competing interests</h2><p id="p-39">The authors declare no competing interests.</p></div><div class="section" id="sec-11"><h2 class="">Author contributions</h2><p id="p-40">S.R. and E.P. conducted the core experimental work. J.D. and K.D. conducted select experiments. N.S.W. and Y.M. conducted theoretical analysis of decay and transcription kinetics. S.R., E.P. and A.K. wrote the manuscript. A.K. supervised the work.</p></div><div class="section" id="sec-12"><h2 class="">A note added on submission</h2><p id="p-41">Our work was submitted to BioRxiv back to back with a related and independent study by Burke et al., and R. Parker [ref will be added here]. Our work employs a considerably different set of experimental approaches, but is synergistic with the co-submitted paper. Our manuscripts converge on a similar model of mRNA regulation by 2-5AMD.</p></div><div class="section" id="sec-14"><h2 class="">Methods</h2><div class="section" id="sec-15"><h3>Cell culture</h3><p id="p-49">Human cells were grown in RPMI with 10% FBS. For poly I:C transfections, 1 µg/mL poly I:C was transfected using Lipofectamine 2000 Reagent (Thermo Fisher Scientific) for the indicated durations. For experiments aimed at measuring mRNA decay rates, RNA polymerase II transcription was blocked by adding 1 µg/ml actinomycin D directly to the cell culture medium for the indicated durations. To measure decay rates of poly I:C-induced transcripts, WT and RNase L<sup>-/-</sup> cells were treated with 1 µg/ml poly I:C for 4 hours, followed by actinomycin D treatment.</p></div><div class="section" id="sec-16"><h3>Nascent translation analysis by <sup>35</sup>S and ribopuromycilation</h3><p id="p-50">To conduct <sup>35</sup>S metabolic labeling, cells were incubated in methionine-free RPMI (Gibco) containing 11 µCi EasyTag EXRESS35S Protein Labeling mix (Perkin Elmer) for 15 minutes at 37 °C. Cells were directly lysed in NuPage LDS sample buffer. Lysates were boiled at 95 °C for 10 minutes, then separated on 10% BisTris PAGE gels (Invitrogen). Gels were stained with coomassie for total protein visualization, then analyzed by phosphorimaging (Typhoon FLA 7000, GE). For ribopuromycilation assay, cells were treated with 10 µg/ml puromycin (Invitrogen) in culture medium for 5 minutes at 37 °C. Cells were lysed and separated by PAGE as above. For western blotting proteins were transferred to PVDF membranes (Life Technologies) and stained with Ponceau Red to visualize total proteins. The membrane was washed and blocked in 5% non-fat dry milk in TBST. Membranes were probed with 1:1000 dilution of mouse anti-puromycin antibody (EMD Millipore), followed by 1:10,000 dilution of horseradish peroxidase-conjugated anti-mouse secondary antibody (Jackson ImmunoResearch).</p></div><div class="section" id="sec-17"><h3>RtcB RNA-seq</h3><p id="p-51">RtcB RNA-seq was conducted as described previously (<a id="xref-ref-11-13" class="xref-bibr" href="#ref-11">Donovan et al., 2017</a>), but without short RNAs purification step. Briefly, 1 µg RNeasy-purified RNA was ligated to 10 μM adaptor (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, oligo 1). Ligation reactions were performed using 10 μM RtcB, 20 mM HEPES pH 7.5, 110 mM NaCl, 2 mM MnCl<sub>2</sub>, 100 μM GTP, 40U RiboLock RNase inhibitor (Fermentas), 4 mM DTT, and 0.05% Triton X-100 for 1 hour at 37 °C. Reactions were quenched using 3 mM EDTA. Owing to its short length, free adapter was removed by purifying the ligated RNA with the RNeasy kit. On-column DNase treatment was omitted so that ligated adapter remained intact. Oligonucleotides were reverse-transcribed using MultiScribe reverse transcriptase (RT) and 2 pmol of RT primer complimentary to the ligation adaptor (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, oligo 2). RNA, RT primer, and dNTPs were incubated for 3 min at 65°C and snap-cooled on ice. MgCl<sub>2</sub> (3 mM f/c) was added to ensure efficient Mg<sup>2+</sup>-dependent RT reaction.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/685474/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/685474/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/685474" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <span class="caption-title">Oligonucleotides used for RtcB RNA-seq library preparation</span><div class="sb-div caption-clear"></div></div></div><p id="p-52">A 2x mastermix containing RT buffer, RT (Applied Biosystems), and 40U Ribolock was added to snap-cooled samples to a final volume of 20 μL. Reactions were incubated at 25°C for 10 min, 37°C for 1.5 h, and 95°C for 5 min. cDNA reaction was brought up to 200 μl with water and extracted using 1 volume of 25:24:1 Phenol:Chloroform:Isoamyl Alcohol saturated with 10 mM TRIS (pH 8.0) and 1 mM EDTA (Sigma Aldrich). The aqueous phase was precipitated using 20 μg glycogen as a carrier, 2/3 5M ammonium acetate (vol/vol) and 3 volumes of 100% ethanol. Mixture was incubated at –80 °C for 30 minutes, followed by a 17,000 x g spin at 4 °C, for 30 minutes. Pellets of cDNA were washed with 75% ethanol (vol/vol) and resuspended in 25 μl DI water. Reactions contained 30% of the cDNA from the previous step, 1 μM adaptor (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>, oligo 3), 1 U/μl CircLigase (Epicentre), and buffer contents as per manufacturers guidelines. CircLigase reactions were incubated for 1 h at 65°C and quenched by adding EDTA to a final concentration of 8 mM. 1/3 of the quenched CircLigase reaction was PCR amplified using Phusion DNA polymerase (NEB) and primers 4-5 in <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>. Libraries were analyzed by a BioAnalyzer high sensitivity DNA 1000 chip (Agilent). Equimolar amounts were pooled and gel purified from native page. RtcB RNA-seq was performed on an Illumina HiSeq 2500 and processed as described previously (<a id="xref-ref-11-14" class="xref-bibr" href="#ref-11">Donovan et al., 2017</a>).</p></div><div class="section" id="sec-18"><h3>Poly-A<sup>+</sup> RNA-seq</h3><p id="p-53">RNA purified by RNeasy kit (1 μg) was used for poly-A<sup>+</sup> enrichment with oligo-dT beads. Pulldown was followed by standard fragmentation, adapter ligation and PCR amplification for sequencing on the Illumina HiSeq 2000 platform. Sequencing reads were mapped to the human genome hg19 using TopHat 2 (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Kim et al., 2013</a>), set to map stranded reads with default parameters. Reads mapping to exons of each gene were counted using HTseq-count in union mode (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Anders et al., 2015</a>). RNA-seq data were visualized using the Integrative Genomics Viewer (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Robinson et al., 2011</a>).</p></div><div class="section" id="sec-19"><h3>Polysome sedimentation analysis</h3><p id="p-54">Cells in 10 cm dishes were flash frozen in liquid nitrogen, scraped in cold PBS with 100 µg/ml CHX and pelleted at 500 x g for 5 minutes at 4°C. The cell pellet was lysed in 5 mM HEPES, 1.5 mM KCl, 2.5 mM MgCl<sub>2</sub>, 100 µg/ml CHX, 1x Protease inhibitor cocktail, 100 U/mL RNase inhibitor (NEB), 0.5 % Triton X-100, and 0.5 % Na-Deoxycholate. The lysate was vortexed, rotated end-over-end for 7 minutes at 4°C and centrifuged at 10,000 x g for 10 minutes at 4 °C. Clarified lysate was layered over a 12 ml 10-50 % sucrose gradient made by GradientMaster (BioComp). The 10 % and 50 % sucrose solutions were made with 20 mM HEPES, 100 mM KCl, 5 mM MgCl<sub>2</sub>, 100 μg/mL CHX, 1x Protease inhibitor cocktail, and 100 U/ml RNase inhibitor. In experiments designed to distinguish mRNA bound 80S vs empty 80S complexes, both lysis and sucrose gradient buffers were adjusted to a final concentration of either 100 mM or 500 mM KCl. To create empty 80S as a control, WT cells were pre-treated with 50 μg/mL puromycin for 20 min and the sucrose gradient buffer also contained 50 μg/ml puromycin in place of CHX. The lysate was spun through the gradient in an SW41Ti rotor in an Optima XE-100 Ultracentrifuge (Beckman Coulter) at 200,000 x g for two hours at 4 °C. BioComp Gradient Fractionator was used to fractionate the gradients and the 254 nm absorbance was read by an EM-1 ultraviolet monitor (BioRad).</p></div><div class="section" id="sec-20"><h3>qPCR</h3><p id="p-55">RNA was purified using the RNeasy kit (Qiagen). RNA quality was assessed using BioAnalyzer NanoChip (Agilent) and extent of rRNA cleavage was quantified using GelQuant.NET (Biochem Lab Solutions, <a href="http://biochemlabsolutions.com/">http://biochemlabsolutions.com/</a>). Within each experiment, equal amounts (ng) of RNA were converted to cDNA using oligo-dT<sub>18</sub> as primer and the High Capacity Reverse Transcriptase kit (Applied Biosystems). qPCR was performed using Power SYBR Green PCR Master Mix (Life Technologies).</p></div><div class="section" id="sec-21"><h3>eIF4E immunoprecipitation</h3><p id="p-56">Magnetic protein A beads were incubated with 2 µg anti-FLAG (Sigma) or anti-eIF2α (Santa Cruz) antibodies in IP buffer (10 mM HEPES (pH 7.5), 150 mM NaCl, 0.1% Nonidet P-40 (NP-40), 1x complete protease inhibitor cocktail, RNase inhibitor) at 4°C for two hours. Excess unbound antibody was removed by washing the beads twice in IP buffer for 5 minutes. Cells transfected with or without 2-5A for three hours were lysed in 10 mM HEPES (pH 7.5), 10 mM NaCl, 2 mM EDTA, 0.5% Triton X-100, 1x complete protease inhibitor cocktail and RNase inhibitor for 7 minutes while rotating, at 4 °C. Lysates were clarified at by spinning at 10,000 x g for 10 minutes and incubated with antibody-bound beads for two hours at 4 °C. After two hours, beads were subject to 5 x 2 min washes with IP buffer. RNA and protein from inputs, supernatants (unbound) and IPs were analyzed using qPCR, western blot and mass spectrometry, respectively.</p></div><div class="section" id="sec-22"><h3>Mass spectrometry</h3><p id="p-57">Gel bands were digested using 1.5 µg Trypsin (Promega). Samples were dried completely in a SpeedVac and resuspended with 21 µl of 0.1% formic acid (pH 3). Next, 5 µl was injected per run using an Easy-nLC 1200 UPLC system. Samples were loaded directly onto a 45 cm long 75 µm inner diameter nano capillary column packed with 1.9 µm C18-AQ (Dr. Maisch, Germany) mated to metal emitter in-line with an Orbitrap Fusion Lumos (Thermo Scientific, USA). The mass spectrometer was operated in data dependent mode with the 120,000 resolution MS1 scan (AGC 4e5, Max IT 50ms, 400-1500 m/z) in the Orbitrap followed by up to 20 MS/MS scans with CID fragmentation in the ion trap. Dynamic exclusion list was invoked to exclude previously sequenced peptides for 60s if sequenced within the last 30s and maximum cycle time of 3s was used. Peptides were isolated for fragmentation using the quadrupole (1.6 Da) window. Ion-trap was operated in Rapid mode with AGC 2e3, maximum IT of 300 ms and minimum of 5000 ions.</p><p id="p-58">Raw files were searched using Byonic (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Bern et al., 2012</a>), MS-Amanda (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Dorfer et al., 2014</a>) and Sequest HT algorithms (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Eng et al., 1994</a>) within the Proteome Discoverer 2.2 suite (Thermo Scientific, USA). 10 ppm MS1 and 0.4 Da MS2 mass tolerances were specified. Carbamidomethylation of cysteine was used as fixed modification, oxidation of methionine, acetylation of protein N-termini, conversion of glutamine to pyro-glutamate and deamidation of asparagine were specified as dynamic modifications. Trypsin digestion with maximum of two missed cleavages were allowed. Files searched against the Uniprot <em>Homo Sapiens</em> database downloaded on February 23, 2017 and supplemented with common contaminants. Scaffold (version 4.8.4, Proteome Software Inc., Portland, OR) was used to validate MS/MS based peptide and protein identifications. Peptide identifications were accepted if they could be established at greater than 90.0% probability by the Scaffold Local FDR algorithm. Protein identifications were accepted if they could be established at greater than 99% probability and contained at least 2 identified peptides. Protein probabilities were assigned by the Protein Prophet algorithm (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Nesvizhskii et al., 2003</a>). Proteins that contained similar peptides and could not be differentiated based on MS/MS analysis alone were grouped to satisfy the principles of parsimony.</p></div><div class="section" id="sec-23"><h3><em>In vitro</em> transcription</h3><p id="p-59">An internal ribosome entry site (IRES)-containing dual luciferase plasmid was a gift from Dr. Paul Copeland (Rutgers University). Monocistronic firefly luciferase was obtained by PCR amplification of the coding region from the dual luciferase construct and cloning into BamHI/NotI digested pcDNA3.1. Plasmids were linearized with AgeI (firefly luciferase) or BamHI (dual luciferase) and purified by phenol extraction and ethanol precipitation. Capped mRNAs were transcribed using reagents from the MEGA ShortScript Kit, except for nucleoside triphosphates, and 12 mM anti-reverse cap analog (NEB). NTPs were added using a custom 10X mixture containing 75 mM each of ATP, UTP, and CTP, and 15 mM GTP. Transcription was carried out for two hours at 37 °C followed by addition of Turbo DNase I and incubation for 20 minutes at 37 °C. Messenger RNAs were phenol extracted and purified on P30 micro spin columns (Bio-Rad).</p></div><div class="section" id="sec-24"><h3>Ribosome-depleted Rabbit Reticulocyte Lysate (RRL)</h3><p id="p-60">Ribosome-free RRL was prepared essentially as described (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Gupta et al., 2013</a>). Briefly, nuclease treated RRL (Promega) was centrifuged 2 x 1 hour at 300,000 x g, 4 °C with care to not disturb the ribosome pellet when removing the supernatant. The pellet from the first centrifugation was saved for purifying RRL ribosomes from the salt-wash step.</p></div><div class="section" id="sec-25"><h3>Ribosome purification</h3><p id="p-61">Frozen A549 cell pellets (∼200 µL) were resuspended in 500 µL of 20 mM HEPES-KOH (pH 7.5), 100 mM KCl, 5 mM MgCl<sub>2</sub>, 4 mM DTT, 0.2% NP-40, 1x phosphatase inhibitors 2/3 (Sigma), 2x complete protease inhibitor (Roche), and 0.4 U/ml RNase inhibitor (NEB), as described previously (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Lorsch and Herschlag, 1999</a>). Resuspended cells were rotated for 15 minutes at 4°C, followed by centrifugation for 20 minutes at 16,000 x g, 4°C. Obtained supernatants were centrifuged for 30 min at 21,000 x g, 4°C. The resulting supernatants were centrifuged for 80 minutes at 300,000 x g and 4 °C to yield a crude ribosomal pellet. Pellets were washed with 5 mM HEPES-KOH (pH 7.5), 50 mM KCl, 1.5 mM MgCl<sub>2</sub>, 4 mM DTT and resuspended in 100 µL of the fresh wash buffer. KCl was adjusted to 0.5 M and ribosomes were incubated for additional 30 minutes on ice, followed by centrifugation for 5 minutes at 10,000 x g and at 4 °C to remove debris. Salt-washed ribosomes (130 µl) were layered onto a 100 µl 0.5 M sucrose cushion containing 20 mM HEPES-KOH (pH 7.5), 100 mM KCl, 2 mM MgCl<sub>2</sub>, and 4 mM DTT. Tubes were centrifuged for 90 minutes at 300,000 x g, at 4°C and the obtained pellets were rinsed with 3 x 50 µL of the ribosome storage buffer (20 mM HEPES KOH pH 7.5, 50 mM KCl, 2 mM MgCl<sub>2</sub>, 4 mM DTT, 10% glycerol) and then dissolved. Debris was removed by centrifugation as above. A small quantity of purified ribosomes (1 μl) was processed with Trizol for rRNA extraction. Remaining ribosomes were quantified, aliquoted, and flash-frozen in LN2. Ribosomes were quantified using 5×10<sup>7</sup> M<sup>−1</sup>cm<sup>−1</sup> as the molar extinction coefficient.</p></div><div class="section" id="sec-26"><h3>Cell-free translation analysis</h3><p id="p-62">Cell-free translation experiments were conducted using nuclease-treated rabbit reticulocyte lysate (Promega). Reactions were 12.5 µl and contained 8 µl RRL (ribosome-depleted or not, as indicated), 0.25 µl 40 U/ml RNase inhibitor, 0.25 µl 1 mM amino acids, 50 ng capped firefly luciferase mRNA or 300 ng dual luciferase mRNA (3 µl combined volume of mRNA and H<sub>2</sub>O), and 1 µl ribosome storage buffer or ribosomes to achieve the indicated final ribosome concentrations. For firefly luciferase mRNA, reactions were incubated for 30 minutes (firefly luciferase mRNA) at 30 °C and then quenched by adding 50 µl 20 mM HEPES pH 7.5, 100 mM NaCl and 1 mM MgCl<sub>2</sub>. The terminated reactions were transferred to a 96-well plate and supplemented with 10 µl of 6X luciferin mix (20 mM HEPES pH 7.5, 100 mM NaCl, 36 mM MgCl<sub>2</sub>, 2.4 mM D-luciferin, 18 mM ATP). Luminescence was measured for 10 seconds using a Tristar2 Multi-Mode Plate Reader (Berthold Technologies).</p></div><div class="section" id="sec-27"><h3>Decay and transcriptional dynamics analysis</h3><p id="p-63">The main observation is that while basal mRNAs decay, the rate of increase of IFNs is only slightly reduced in the presence of RNase L (<a id="xref-fig-14-6" class="xref-fig" href="#F14">Fig. S7A</a>). The simplest model to describe the dynamics of IFN mRNAs is that, in the absence of RNase L, they increase exponentially due to direct positive feedback according to the rate equation:
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/12/04/484675/embed/graphic-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/12/04/484675/embed/graphic-17.gif"/></noscript></span>

</span>
with the solution:
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/12/04/484675/embed/graphic-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/12/04/484675/embed/graphic-18.gif"/></noscript></span>

</span>
</p><p id="p-64">With RNase L, IFN mRNA loss due to decay can be accounted for by adding a decay rate constant <em>β</em><sub>1</sub>:
<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/12/04/484675/embed/graphic-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/12/04/484675/embed/graphic-19.gif"/></noscript></span>

</span>
</p><p id="p-65">With solution:
<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/12/04/484675/embed/graphic-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/12/04/484675/embed/graphic-20.gif"/></noscript></span>

</span>
</p><p id="p-66">However, the observed decrease of the rate of accumulation of IFN mRNAs due to the presence of RNase L is much smaller than this model would predict (<a id="xref-fig-14-7" class="xref-fig" href="#F14">Fig. S7A</a>).</p><p id="p-67">In order to explain the observation that the rate of increase of IFNs is only slightly reduced in the presence of RNase L, we generalize the above model by assuming that the IFN positive feedback loop is mediated by a stable activator (e.g. the IFN protein and phosphorylation of the transcription factor STAT) which do not get cleaved by RNase L. They provide a gradually accumulating activator, increasing stimulating IFN mRNA transcription with time. Denoting by <em>I</em>(<em>t</em>) and <em>A</em>(<em>t</em>) the concentrations of IFN mRNAs and the IFN proteins, respectively, the rate equations that describe the system become:
<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/12/04/484675/embed/graphic-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/12/04/484675/embed/graphic-21.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/12/04/484675/embed/graphic-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/12/04/484675/embed/graphic-22.gif"/></noscript></span>

</span>
</p><p id="p-68">RNase L does not degrade <em>A</em>, therefore we set <em>β</em><sub>2</sub> = 0. The solution of these equations is given by <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/12/04/484675/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/12/04/484675/embed/inline-graphic-1.gif"/></noscript></span></span> where λ is given by
<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/12/04/484675/embed/graphic-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/12/04/484675/embed/graphic-23.gif"/></noscript></span>

</span>
where we only consider the relevant increasing solution and define <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/12/04/484675/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/12/04/484675/embed/inline-graphic-2.gif"/></noscript></span></span>. In the absence of decay of the IFN mRNAs (<em>β</em><sub>1</sub> = 0), one obtains λ = <em>α</em>, i.e. IFN induction depends only on IFN mRNA induction and activator induction but not on RNase L. If IFN mRNA decay is present and <em>β</em><sub>1</sub> &lt;&lt; <em>α</em> (which is the case, see <a id="xref-fig-14-8" class="xref-fig" href="#F14">Fig. S7A</a>), then λ ≈ <em>α</em> – <em>β</em><sub>1</sub>/2. Under these conditions, the effect of RNase L on IFN accumulation will be a contribution to decay at just 1/2 the potency of the bare rate of RNase L-mediated decay of IFN mRNA. For decay of basal mRNA, <em>β</em><sub>basal</sub> = 0.007 (in natural logarithm scale and with units ∼1/time). The innate immune mRNA decay ∼2.6-fold slower on average, i.e. <em>β</em><sub>1</sub> ∼ 0.003. The effect of a stable activator will attenuate this value 2-fold to give λ ≈ <em>α</em> – 0.0015. Considering that decay-free IFN accumulation occurs with α ∼ 0.013 (<a id="xref-fig-14-9" class="xref-fig" href="#F14">Fig. S7A</a>), subtraction of 0.0015 will have a negligible effect, which explains why RNase L does not strongly inhibit IFN production. A graphical representation of these results is provided in <a id="xref-fig-7-7" class="xref-fig" href="#F7">Figures 7B</a>-D.</p><p id="p-69">For more general parameter values, note that if we define <em>γ</em> = <em>β</em><sub>1</sub> / <em>α</em>, we can write <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/12/04/484675/embed/inline-graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/12/04/484675/embed/inline-graphic-3.gif"/></noscript></span></span>, which is always positive. Based on this relationship, even if the decay exponent <em>β</em><sub>1</sub> is very large (i.e. 2-5AMD is very strong), as long as a stable activator is present in the positive feedback loop there will be exponential growth of IFNs due to the activator gradually accumulating and leading to faster IFN mRNA synthesis.</p><p id="p-70">Lastly, we note that if the activator does decay (<em>β</em><sub>2</sub> ≠ 0), the growth exponent is given by the expression
<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/12/04/484675/embed/graphic-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/12/04/484675/embed/graphic-24.gif"/></noscript></span>

</span>
</p><p id="p-71">In particular, when <em>β</em><sub>1</sub> = <em>β</em><sub>2</sub> (IFN mRNA and the activator both decay at the same rate), λ = <em>α</em> - <em>β</em><sub>1</sub>, and the kinetics then reproduces the case of IFN mRNA decay at full potency (E4).</p></div><div class="section" id="sec-28"><h3>Application of ribosome-equivalent messenger RNA length (REML) for calculation of mRNA decay in cell-free systems</h3><p id="p-72">Using the RNA-seq data (GEO ID GSE75530) we determined that fraction of intact mRNA left in a cell-free system in the course of 2-5AMD can be calculated from mRNA length L, GC content, and 28S rRNA cleavage observed by NanoChip as follows:
<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/12/04/484675/embed/graphic-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/12/04/484675/embed/graphic-25.gif"/></noscript></span>

</span>
</p><p id="p-73">In the case of ACTB (mRNA length L = 1808 nt and GC content of 55.2%), REML = 350 nt (<a id="xref-fig-12-2" class="xref-fig" href="#F12">Fig. S5</a>). Using the expression for f(mRNA<sup>left</sup>), it is determined that under 2-5AMD conditions that degrade 10% of 28S rRNA, 58% of ATCB mRNA will be remaining: (0.9)<sup>(1808/350)</sup> = 0.58. Under conditions of 50% 28S rRNA cleavage, (0.5)<sup>(1808/350)</sup> = 0.028 (2.8%) of ACTB mRNA will remain. The stability of mRNAs will vary with mRNA length and GC composition. For a transcript with 40% GC (REML = 200 nt) and length 10,000 bases, under conditions of 10% of 28S rRNA degradation only (0.9)<sup>(10000/200)</sup> = 0.005 (0.5%) of the uncleaved mRNA will be remaining. Therefore under conditions of 10% 28S rRNA cleavage, the latter mRNA will appear to be ∼ 100-fold more sensitive to RNase L than mRNA of ACTB.</p></div><div class="section" id="sec-29"><h3>Error analysis</h3><p id="p-74">For decay rates, data points from three biological replicates were plotted together. Statistical significance (P) was from Welch two-tailed unpaired t-test (James McCaffrey implementation, Microsoft, <a href="https://msdn.microsoft.com/en-us/magazine/mt620016.aspx">https://msdn.microsoft.com/en-us/magazine/mt620016.aspx</a>). *P≤0.05, **P≤0.01, ***P≤0.001, *P≤0.0001, NS: non-significant. Error bars represent S.E.</p></div><div class="section" id="sec-30"><h3>Briggs-Haldane kinetics application to 2-5AMD</h3><p id="p-75">Michael-Menten kinetics requires rapid enzyme-substrate binding equilibrium E+S. In contrast, in Briggs-Haldane regime the enzyme-substrate binding equilibrium is not achieved because product formation from the ES complex is faster than ES complex dissociation to give free E+S. Under Michaelis-Menten conditions, enzyme preference for two different substrates (specificity) is defined as the ratio: S1/S2, whether S1 is k<sub>cat</sub><sup>1</sup>/K<sub>m</sub><sup>1</sup> for substrate 1 and S2 is k<sub>cat</sub><sup>2</sup>/K<sub>m</sub><sup>2</sup> for substrate 2. Catalytic activity (k<sub>cat</sub>) and binding (K<sub>m</sub>) both determine the relative reaction rates of the two substrates. In Briggs-Haldane regime, the rate constant for the product formation from ES (k<sub>cat</sub>) is much larger than the rate constant for the ES complex dissociation (k<sub>off</sub>), such that k<sub>cat</sub>/K<sub>m</sub> = k<sub>cat</sub>/(k<sub>off</sub>+k<sub>cat</sub>/k<sub>on</sub>) ∼ k<sub>on</sub>. The specificity ratio S1/S2 is simplified to k<sub>on</sub><sup>1</sup>/k<sub>on</sub><sup>2</sup>. S1/S2 no longer depends on the binding affinity (K<sub>m</sub>) or the catalytic activity (k<sub>cat</sub>) and depends only on the association rate constant (k<sub>on</sub>). For similar substrates such as different mRNAs k<sub>on</sub> will depend primarily on the hydrodynamic radius and should be similar for most mRNAs. If association is mediated by the ribosome, k<sub>on</sub> for different mRNAs could be determined by ribosome-mRNA or ribosome-RNase L association.</p></div></div><div class="section display-objects" id="sec-31"><h2 class="">Tables</h2><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/685056/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/685056/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/685056" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2.</span> <span class="caption-title">qPCR Primers used in this work</span><div class="sb-div caption-clear"></div></div></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-37">The authors thank Prof. Kevan Shokat for the gift of INK128 and helpful discussions, Dr. Paul Copeland (Rutgers University) for the gift of internal ribosome entry site (IRES)-containing dual luciferase plasmid, staff at the Proteomics and Mass Spectrometry Core facility, and staff at the Genomics Core Facility at Princeton University. We are grateful to Prof. Zemer Gitai and to Dr. Sophia Li for help with polysome analysis, to Prof. Bonnie Bassler, Prof. Martin Jonikas, and Prof. Sabine Petry for sharing instrumental resources and we thank members of the Korennykh lab for great help throughout the project.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.1" data-doi="10.1038/onc.2009.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Al-Ahmadi</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Al-Haj</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Al-Mohanna</span>,  <span class="cit-name-given-names">F. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Silverman</span>,  <span class="cit-name-given-names">R. H.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Khabar</span>,  <span class="cit-name-given-names">K. S.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">RNase L downmodulation of the RNA-binding protein, HuR, and cellular growth</span>. <abbr class="cit-jnl-abbrev">Oncogene</abbr> <span class="cit-vol">28</span>, <span class="cit-fpage">1782</span>–<span class="cit-lpage">1791</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOncogene%26rft.stitle%253DOncogene%26rft.aulast%253DAl-Ahmadi%26rft.auinit1%253DW.%26rft.volume%253D28%26rft.issue%253D15%26rft.spage%253D1782%26rft.epage%253D1791%26rft.atitle%253DRNase%2BL%2Bdownmodulation%2Bof%2Bthe%2BRNA-binding%2Bprotein%252C%2BHuR%252C%2Band%2Bcellular%2Bgrowth.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fonc.2009.16%26rft_id%253Dinfo%253Apmid%252F19252527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/onc.2009.16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19252527&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265640400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.2"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Alisha</span> <span class="cit-name-surname">Chitrakar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Sneha</span> <span class="cit-name-surname">Rath</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Jesse</span> <span class="cit-name-surname">Donovan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Kaitlin</span> <span class="cit-name-surname">Demarest</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Yize</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Raghavendra Rao</span> <span class="cit-name-surname">Sridhar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Susan R.</span> <span class="cit-name-surname">Weiss</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Sergei V.</span> <span class="cit-name-surname">Kotenko</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Ned S.</span> <span class="cit-name-surname">Wingreen</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Korennykh</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Realtime 2-5A kinetics suggests interferons β and lambda evade global arrest of translation by RNase L</span>. <abbr class="cit-jnl-abbrev">BioRxiv</abbr> <a href="https://doiorg/101101/476341">https://doiorg/101101/476341</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Banerjee</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gaughan</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baskar</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Parker</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lindner</span>,  <span class="cit-name-given-names">D. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weiss</span>,  <span class="cit-name-given-names">S. R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Silverman</span>,  <span class="cit-name-given-names">R. H.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">RNase L is a negative regulator of cell migration</span>. <abbr class="cit-jnl-abbrev">Oncotarget</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">44360</span>–<span class="cit-lpage">44372</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOncotarget%26rft.volume%253D6%26rft.spage%253D44360%26rft_id%253Dinfo%253Apmid%252F26517238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=26517238&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.4" data-doi="10.1089/jir.2010.0120"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chakrabarti</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jha</span>,  <span class="cit-name-given-names">B. K.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Silverman</span>,  <span class="cit-name-given-names">R. H.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">New insights into the role of RNase L in innate immunity</span>. <abbr class="cit-jnl-abbrev">J Interferon Cytokine Res</abbr> <span class="cit-vol">31</span>, <span class="cit-fpage">49</span>–<span class="cit-lpage">57</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Binterferon%2B%2526%2Bcytokine%2Bresearch%2B%253A%2B%2Bthe%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BSociety%2Bfor%2BInterferon%2Band%2BCytokine%2BResearch%26rft.stitle%253DJ%2BInterferon%2BCytokine%2BRes%26rft.aulast%253DChakrabarti%26rft.auinit1%253DA.%26rft.volume%253D31%26rft.issue%253D1%26rft.spage%253D49%26rft.epage%253D57%26rft.atitle%253DNew%2Binsights%2Binto%2Bthe%2Brole%2Bof%2BRNase%2BL%2Bin%2Binnate%2Bimmunity.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fjir.2010.0120%26rft_id%253Dinfo%253Apmid%252F21190483%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/jir.2010.0120&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21190483&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286380000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.5" data-doi="10.1016/j.cell.2015.07.011"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chiappinelli</span>,  <span class="cit-name-given-names">K. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Strissel</span>,  <span class="cit-name-given-names">P. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Desrichard</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Henke</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Akman</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hein</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rote</span>,  <span class="cit-name-given-names">N. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cope</span>,  <span class="cit-name-given-names">L. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snyder</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">et</span>  <span class="cit-name-given-names">al.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Inhibiting DNA Methylation Causes an Interferon Response in Cancer via dsRNA Including Endogenous Retroviruses</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">162</span>, <span class="cit-fpage">974</span>–<span class="cit-lpage">986</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D162%26rft.spage%253D974%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2015.07.011%26rft_id%253Dinfo%253Apmid%252F26317466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2015.07.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26317466&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.6" data-doi="10.1038/nature12305"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Civril</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Deimling</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Oliveira Mann</span>,  <span class="cit-name-given-names">C. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ablasser</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moldt</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Witte</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hornung</span>,  <span class="cit-name-given-names">V.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Hopfner</span>,  <span class="cit-name-given-names">K. P.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Structural mechanism of cytosolic DNA sensing by cGAS</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">498</span>, <span class="cit-fpage">332</span>–<span class="cit-lpage">337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D498%26rft.spage%253D332%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12305%26rft_id%253Dinfo%253Apmid%252F23722159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12305&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23722159&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000320592900033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.7" data-doi="10.1016/0092-8674(78)90329-X"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Clemens</span>,  <span class="cit-name-given-names">M. J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Williams</span>,  <span class="cit-name-given-names">B. R.</span></span> (<span class="cit-pub-date">1978</span>). <span class="cit-article-title">Inhibition of cell-free protein synthesis by pppA2′p5′A2′p5′A: a novel oligonucleotide synthesized by interferon-treated L cell extracts</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">13</span>, <span class="cit-fpage">565</span>–<span class="cit-lpage">572</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DClemens%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D13%26rft.issue%253D3%26rft.spage%253D565%26rft.epage%253D572%26rft.atitle%253DInhibition%2Bof%2Bcell-free%2Bprotein%2Bsynthesis%2Bby%2BpppA2%2527p5%2527A2%2527p5%2527A%253A%2Ba%2Bnovel%2Boligonucleotide%2Bsynthesized%2Bby%2Binterferon-treated%2BL%2Bcell%2Bextracts.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252878%252990329-X%26rft_id%253Dinfo%253Apmid%252F657268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(78)90329-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=657268&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1978ES36300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cooper</span>,  <span class="cit-name-given-names">D. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Banerjee</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chakrabarti</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garcia-Sastre</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hesselberth</span>,  <span class="cit-name-given-names">J. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Silverman</span>,  <span class="cit-name-given-names">R. H.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Barton</span>,  <span class="cit-name-given-names">D. J.</span></span> (<span class="cit-pub-date">2014a</span>). <span class="cit-article-title">Ribonuclease L Targets Distinct Sites in Influenza A Virus RNAs</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.9" data-doi="10.1093/nar/gku118"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cooper</span>,  <span class="cit-name-given-names">D. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jha</span>,  <span class="cit-name-given-names">B. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Silverman</span>,  <span class="cit-name-given-names">R. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hesselberth</span>,  <span class="cit-name-given-names">J. R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Barton</span>,  <span class="cit-name-given-names">D. J.</span></span> (<span class="cit-pub-date">2014b</span>). <span class="cit-article-title">Ribonuclease L and metal-ion-independent endoribonuclease cleavage sites in host and viral RNAs</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">42</span>, <span class="cit-fpage">5202</span>–<span class="cit-lpage">5216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku118%26rft_id%253Dinfo%253Apmid%252F24500209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku118&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24500209&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.10" data-doi="10.1073/pnas.1218528110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Donovan</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dufner</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Korennykh</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Structural basis for cytosolic double-stranded RNA surveillance by human oligoadenylate synthetase 1</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">110</span>, <span class="cit-fpage">1652</span>–<span class="cit-lpage">1657</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1218528110%26rft_id%253Dinfo%253Apmid%252F23319625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTEwLzUvMTY1MiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzEyLzA0LzQ4NDY3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Donovan</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rath</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kolet-Mandrikov</span>,  <span class="cit-name-given-names">D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Korennykh</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Rapid RNase L-driven arrest of protein synthesis in the dsRNA response without degradation of translation machinery</span>. <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">23</span>, <span class="cit-fpage">16601671</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft.volume%253D23%26rft.spage%253D16601671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.12" data-doi="10.1073/pnas.1419409112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Donovan</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whitney</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rath</span>,  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Korennykh</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Structural mechanism of sensing long dsRNA via a noncatalytic domain in human oligoadenylate synthetase 3</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">112</span>, <span class="cit-fpage">3949</span>–<span class="cit-lpage">3954</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1419409112%26rft_id%253Dinfo%253Apmid%252F25775560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzEzLzM5NDkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8xMi8wNC80ODQ2NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drappier</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jha</span>,  <span class="cit-name-given-names">B. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stone</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elliott</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vertommen</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weiss</span>,  <span class="cit-name-given-names">S. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Silverman</span>,  <span class="cit-name-given-names">R. H.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Michiels</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">A novel mechanism of RNase L inhibition: Theiler’s virus L* protein prevents 2-5A from binding to RNase L</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">e1006989</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D14%26rft.spage%253De1006989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.14" data-doi="10.1371/journal.pbio.1000038"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Feldman</span>,  <span class="cit-name-given-names">M. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Apsel</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Uotila</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Loewith</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knight</span>,  <span class="cit-name-given-names">Z. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ruggero</span>,  <span class="cit-name-given-names">D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Shokat</span>,  <span class="cit-name-given-names">K. M.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Active-site inhibitors of mTOR target rapamycin-resistant outputs of mTORC1 and mTORC2</span>. <abbr class="cit-jnl-abbrev">PLoS Biol</abbr> <span class="cit-vol">7</span>, <span class="cit-fpage">e38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DFeldman%26rft.auinit1%253DM.%2BE.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De38%26rft.epage%253De38%26rft.atitle%253DActive-site%2Binhibitors%2Bof%2BmTOR%2Btarget%2Brapamycin-resistant%2Boutputs%2Bof%2BmTORC1%2Band%2BmTORC2.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000038%26rft_id%253Dinfo%253Apmid%252F19209957%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19209957&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.15" data-doi="10.1126/science.6165080"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Floyd-Smith</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Slattery</span>,  <span class="cit-name-given-names">E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Lengyel</span>,  <span class="cit-name-given-names">P.</span></span> (<span class="cit-pub-date">1981</span>). <span class="cit-article-title">Interferon action: RNA cleavage pattern of a (2′-5′) oligoadenylate-dependent endonuclease</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">212</span>, <span class="cit-fpage">1030</span>–<span class="cit-lpage">1032</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DFloyd-Smith%26rft.auinit1%253DG%26rft.volume%253D212%26rft.issue%253D4498%26rft.spage%253D1030%26rft.epage%253D1032%26rft.atitle%253DInterferon%2Baction%253A%2BRNA%2Bcleavage%2Bpattern%2Bof%2Ba%2B%25282%2527-5%2527%2529oligoadenylate--dependent%2Bendonuclease%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.6165080%26rft_id%253Dinfo%253Apmid%252F6165080%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyMTIvNDQ5OC8xMDMwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMTIvMDQvNDg0Njc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gandin</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sikstrom</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alain</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morita</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McLaughlan</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Larsson</span>,  <span class="cit-name-given-names">O.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Topisirovic</span>,  <span class="cit-name-given-names">I.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Polysome fractionation and analysis of mammalian translatomes on a genomewide scale</span>. <abbr class="cit-jnl-abbrev">J Vis Exp</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.17" data-doi="10.1093/nar/22.3.321"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gluck</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Endo</span>,  <span class="cit-name-given-names">Y.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Wool</span>,  <span class="cit-name-given-names">I. G.</span></span> (<span class="cit-pub-date">1994</span>). <span class="cit-article-title">The ribosomal RNA identity elements for ricin and for alpha-sarcin: mutations in the putative CG pair that closes a GAGA tetraloop</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">22</span>, <span class="cit-fpage">321</span>–<span class="cit-lpage">324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DGluck%26rft.auinit1%253DA.%26rft.volume%253D22%26rft.issue%253D3%26rft.spage%253D321%26rft.epage%253D324%26rft.atitle%253DThe%2Bribosomal%2BRNA%2Bidentity%2Belements%2Bfor%2Bricin%2Band%2Bfor%2B%257Balpha%257D-sarcin%253A%2Bmutations%2Bin%2Bthe%2Bputative%2BCG%2Bpair%2Bthat%2Bcloses%2Ba%2BGAGA%2Btetraloop%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F22.3.321%26rft_id%253Dinfo%253Apmid%252F8127668%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/22.3.321&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8127668&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994MX90800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gusho</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jha</span>,  <span class="cit-name-given-names">B. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thornbrough</span>,  <span class="cit-name-given-names">J. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dong</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gaughan</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elliott</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weiss</span>,  <span class="cit-name-given-names">S. R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Silverman</span>,  <span class="cit-name-given-names">R. H.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Murine AKAP7 has a 2′,5′-phosphodiesterase domain that can complement an inactive murine coronavirus ns2 gene</span>. <abbr class="cit-jnl-abbrev">MBio</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">e01312</span>–<span class="cit-lpage">01314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMBio%26rft.volume%253D5%26rft.spage%253De01312%26rft_id%253Dinfo%253Apmid%252F24987090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24987090&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.19" data-doi="10.1089/jir.2004.24.664"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">J. Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wroblewski</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Silverman</span>,  <span class="cit-name-given-names">R. H.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Barton</span>,  <span class="cit-name-given-names">D. J.</span></span> (<span class="cit-pub-date">2004</span>). <span class="cit-article-title">Sensitivity of hepatitis C virus RNA to the antiviral enzyme ribonuclease L is determined by a subset of efficient cleavage sites</span>. <abbr class="cit-jnl-abbrev">J Interferon Cytokine Res</abbr> <span class="cit-vol">24</span>, <span class="cit-fpage">664</span>–<span class="cit-lpage">676</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Binterferon%2B%2526%2Bcytokine%2Bresearch%2B%253A%2B%2Bthe%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BSociety%2Bfor%2BInterferon%2Band%2BCytokine%2BResearch%26rft.stitle%253DJ%2BInterferon%2BCytokine%2BRes%26rft.aulast%253DHan%26rft.auinit1%253DJ.%2BQ.%26rft.volume%253D24%26rft.issue%253D11%26rft.spage%253D664%26rft.epage%253D676%26rft.atitle%253DSensitivity%2Bof%2Bhepatitis%2BC%2Bvirus%2BRNA%2Bto%2Bthe%2Bantiviral%2Benzyme%2Bribonuclease%2BL%2Bis%2Bdetermined%2Bby%2Ba%2Bsubset%2Bof%2Befficient%2Bcleavage%2Bsites.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fjir.2004.24.664%26rft_id%253Dinfo%253Apmid%252F15684820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/jir.2004.24.664&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15684820&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225268900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.20" data-doi="10.1126/science.1249845"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Donovan</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rath</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whitney</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chitrakar</span>,  <span class="cit-name-given-names">A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Korennykh</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Structure of human RNase L reveals the basis for regulated RNA decay in the IFN response</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">343</span>, <span class="cit-fpage">1244</span>–<span class="cit-lpage">1248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1249845%26rft_id%253Dinfo%253Apmid%252F24578532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDMvNjE3Ni8xMjQ0IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMTIvMDQvNDg0Njc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.21" data-doi="10.1016/j.celrep.2012.09.004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whitney</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Donovan</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Korennykh</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Innate Immune Messenger 2-5A Tethers Human RNase L into Active High-Order Complexes</span>. <abbr class="cit-jnl-abbrev">Cell Rep</abbr> <span class="cit-vol">2</span>, <span class="cit-fpage">902</span>–<span class="cit-lpage">913</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCell%2BRep%26rft.aulast%253DHan%26rft.auinit1%253DY.%26rft.volume%253D2%26rft.issue%253D4%26rft.spage%253D902%26rft.epage%253D913%26rft.atitle%253DInnate%2BImmune%2BMessenger%2B2-5A%2BTethers%2BHuman%2BRNase%2BL%2Binto%2BActive%2BHigh-Order%2BComplexes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2012.09.004%26rft_id%253Dinfo%253Apmid%252F23084743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2012.09.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23084743&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000314455600021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.22" data-doi="10.1038/268537a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hovanessian</span>,  <span class="cit-name-given-names">A. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brown</span>,  <span class="cit-name-given-names">R. E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kerr</span>,  <span class="cit-name-given-names">I. M.</span></span> (<span class="cit-pub-date">1977</span>). <span class="cit-article-title">Synthesis of low molecular weight inhibitor of protein synthesis with enzyme from interferon-treated cells</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">268</span>, <span class="cit-fpage">537</span>–<span class="cit-lpage">540</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHovanessian%26rft.auinit1%253DA.%2BG.%26rft.volume%253D268%26rft.issue%253D5620%26rft.spage%253D537%26rft.epage%253D540%26rft.atitle%253DSynthesis%2Bof%2Blow%2Bmolecular%2Bweight%2Binhibitor%2Bof%2Bprotein%2Bsynthesis%2Bwith%2Benzyme%2Bfrom%2Binterferon-treated%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F268537a0%26rft_id%253Dinfo%253Apmid%252F560630%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/268537a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=560630&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.23" data-doi="10.1038/nature10912"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hsieh</span>,  <span class="cit-name-given-names">A. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Edlind</span>,  <span class="cit-name-given-names">M. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ingolia</span>,  <span class="cit-name-given-names">N. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Janes</span>,  <span class="cit-name-given-names">M. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sher</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">E. Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stumpf</span>,  <span class="cit-name-given-names">C. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Christensen</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bonham</span>,  <span class="cit-name-given-names">M. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">et</span>  <span class="cit-name-given-names">al.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">The translational landscape of mTOR signalling steers cancer initiation and metastasis</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">485</span>, <span class="cit-fpage">55</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHsieh%26rft.auinit1%253DA.%2BC.%26rft.volume%253D485%26rft.issue%253D7396%26rft.spage%253D55%26rft.epage%253D61%26rft.atitle%253DThe%2Btranslational%2Blandscape%2Bof%2BmTOR%2Bsignalling%2Bsteers%2Bcancer%2Binitiation%2Band%2Bmetastasis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10912%26rft_id%253Dinfo%253Apmid%252F22367541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10912&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22367541&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303451900033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.24" data-doi="10.1128/MCB.20.2.617-627.2000"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Iordanov</span>,  <span class="cit-name-given-names">M. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Paranjape</span>,  <span class="cit-name-given-names">J. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wong</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Williams</span>,  <span class="cit-name-given-names">B. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meurs</span>,  <span class="cit-name-given-names">E. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Silverman</span>,  <span class="cit-name-given-names">R. H.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Magun</span>,  <span class="cit-name-given-names">B. E.</span></span> (<span class="cit-pub-date">2000</span>). <span class="cit-article-title">Activation of p38 mitogen-activated protein kinase and c-Jun NH(2)-terminal kinase by double-stranded RNA and encephalomyocarditis virus: involvement of RNase L, protein kinase R, and alternative pathways</span>. <abbr class="cit-jnl-abbrev">Mol Cell Biol</abbr> <span class="cit-vol">20</span>, <span class="cit-fpage">617</span>–<span class="cit-lpage">627</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DIordanov%26rft.auinit1%253DM.%2BS.%26rft.volume%253D20%26rft.issue%253D2%26rft.spage%253D617%26rft.epage%253D627%26rft.atitle%253DActivation%2Bof%2Bp38%2BMitogen-Activated%2BProtein%2BKinase%2Band%2Bc-Jun%2BNH2-Terminal%2BKinase%2Bby%2BDouble-Stranded%2BRNA%2Band%2BEncephalomyocarditis%2BVirus%253A%2BInvolvement%2Bof%2BRNase%2BL%252C%2BProtein%2BKinase%2BR%252C%2Band%2BAlternative%2BPathways%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.20.2.617-627.2000%26rft_id%253Dinfo%253Apmid%252F10611240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjIwLzIvNjE3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMTIvMDQvNDg0Njc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.25" data-doi="10.1038/nature17978"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Iwasaki</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Floor</span>,  <span class="cit-name-given-names">S. N.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Ingolia</span>,  <span class="cit-name-given-names">N. T.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Rocaglates convert DEAD-box protein eIF4A into a sequence-selective translational repressor</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">534</span>, <span class="cit-fpage">558</span>–<span class="cit-lpage">561</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D534%26rft.spage%253D558%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature17978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature17978&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.26" data-doi="10.1038/ni1243"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kawai</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Takahashi</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sato</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coban</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kumar</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kato</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ishii</span>,  <span class="cit-name-given-names">K. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Takeuchi</span>,  <span class="cit-name-given-names">O.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Akira</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">I PS-1, an adaptor triggering RIG-I-and Mda5-mediated type I interferon induction</span>. <abbr class="cit-jnl-abbrev">Nat Immunol</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">981</span>–<span class="cit-lpage">988</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bimmunology%26rft.stitle%253DNat%2BImmunol%26rft.aulast%253DKawai%26rft.auinit1%253DT.%26rft.volume%253D6%26rft.issue%253D10%26rft.spage%253D981%26rft.epage%253D988%26rft.atitle%253DIPS-1%252C%2Ban%2Badaptor%2Btriggering%2BRIG-I-%2Band%2BMda5-mediated%2Btype%2BI%2Binterferon%2Binduction.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fni1243%26rft_id%253Dinfo%253Apmid%252F16127453%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ni1243&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16127453&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232027200016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.27" data-doi="10.1038/nature14427"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Khatter</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Myasnikov</span>,  <span class="cit-name-given-names">A. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Natchiar</span>,  <span class="cit-name-given-names">S. K.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Klaholz</span>,  <span class="cit-name-given-names">B. P.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Structure of the human 80S ribosome</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">520</span>, <span class="cit-fpage">640</span>–<span class="cit-lpage">645</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D520%26rft.spage%253D640%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14427%26rft_id%253Dinfo%253Apmid%252F25901680%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14427&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25901680&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.28" data-doi="10.1038/nature07661"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Korennykh</span>,  <span class="cit-name-given-names">A. V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Egea</span>,  <span class="cit-name-given-names">P. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Korostelev</span>,  <span class="cit-name-given-names">A. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Finer-Moore</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shokat</span>,  <span class="cit-name-given-names">K. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stroud</span>,  <span class="cit-name-given-names">R. M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Walter</span>,  <span class="cit-name-given-names">P.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">The unfolded protein response signals through high-order assembly of Ire1</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">457</span>, <span class="cit-fpage">687</span>–<span class="cit-lpage">693</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKorennykh%26rft.auinit1%253DA.%2BV.%26rft.volume%253D457%26rft.issue%253D7230%26rft.spage%253D687%26rft.epage%253D693%26rft.atitle%253DThe%2Bunfolded%2Bprotein%2Bresponse%2Bsignals%2Bthrough%2Bhigh-order%2Bassembly%2Bof%2BIre1.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07661%26rft_id%253Dinfo%253Apmid%252F19079236%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07661&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19079236&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263064700033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.29" data-doi="10.1038/nsmb1082"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Korennykh</span>,  <span class="cit-name-given-names">A. V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Piccirilli</span>,  <span class="cit-name-given-names">J. A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Correll</span>,  <span class="cit-name-given-names">C. C.</span></span> (<span class="cit-pub-date">2006</span>). <span class="cit-article-title">The electrostatic character of the ribosomal surface enables extraordinarily rapid target location by ribotoxins</span>. <abbr class="cit-jnl-abbrev">Nat Struct Mol Biol</abbr> <span class="cit-vol">13</span>, <span class="cit-fpage">436</span>–<span class="cit-lpage">443</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DKorennykh%26rft.auinit1%253DA.%2BV.%26rft.volume%253D13%26rft.issue%253D5%26rft.spage%253D436%26rft.epage%253D443%26rft.atitle%253DThe%2Belectrostatic%2Bcharacter%2Bof%2Bthe%2Bribosomal%2Bsurface%2Benables%2Bextraordinarily%2Brapid%2Btarget%2Blocation%2Bby%2Bribotoxins.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb1082%26rft_id%253Dinfo%253Apmid%252F16604082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb1082&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16604082&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237359700018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Le Roy</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Silhol</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Salehzada</span>,  <span class="cit-name-given-names">T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Bisbal</span>,  <span class="cit-name-given-names">C.</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">Regulation of mitochondrial mRNA stability by RNase L is translation-dependent and controls IFNalpha-induced apoptosis</span>. <abbr class="cit-jnl-abbrev">Cell Death Differ</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">14061413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BDeath%2BDiffer%26rft.volume%253D14%26rft.spage%253D14061413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.31" data-doi="10.1016/j.cell.2007.10.057"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">K. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dey</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neculai</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cao</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dever</span>,  <span class="cit-name-given-names">T. E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Sicheri</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">Structure of the dual enzyme Ire1 reveals the basis for catalysis and regulation in nonconventional RNA splicing</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">132</span>, <span class="cit-fpage">89</span>-<span class="cit-lpage">100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLee%26rft.auinit1%253DK.%2BP.%26rft.volume%253D132%26rft.issue%253D1%26rft.spage%253D89%26rft.epage%253D100%26rft.atitle%253DStructure%2Bof%2Bthe%2Bdual%2Benzyme%2BIre1%2Breveals%2Bthe%2Bbasis%2Bfor%2Bcatalysis%2Band%2Bregulation%2Bin%2Bnonconventional%2BRNA%2Bsplicing.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2007.10.057%26rft_id%253Dinfo%253Apmid%252F18191223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2007.10.057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18191223&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253427600018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.32" data-doi="10.1073/pnas.1216922110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Leonova</span>,  <span class="cit-name-given-names">K. I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brodsky</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lipchick</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pal</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Novototskaya</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chenchik</span>,  <span class="cit-name-given-names">A. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sen</span>,  <span class="cit-name-given-names">G. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Komarova</span>,  <span class="cit-name-given-names">E. A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Gudkov</span>,  <span class="cit-name-given-names">A. V.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">p53 cooperates with DNA methylation and a suicidal interferon response to maintain epigenetic silencing of repeats and noncoding RNAs</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">110</span>, <span class="cit-fpage">E89</span>–<span class="cit-lpage">98</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1216922110%26rft_id%253Dinfo%253Apmid%252F23236145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMTAvMS9FODkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8xMi8wNC80ODQ2NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Banerjee</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goldstein</span>,  <span class="cit-name-given-names">S. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dong</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gaughan</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rath</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Donovan</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Korennykh</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Silverman</span>,  <span class="cit-name-given-names">R. H.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Weiss</span>,  <span class="cit-name-given-names">S. R.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Ribonuclease L mediates the cell-lethal phenotype of doublestranded RNA editing enzyme ADAR1 deficiency in a human cell line</span>. <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-vol">6</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.34" data-doi="10.1073/pnas.1519657113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Banerjee</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goldstein</span>,  <span class="cit-name-given-names">S. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dong</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gaughan</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Silverman</span>,  <span class="cit-name-given-names">R. H.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Weiss</span>,  <span class="cit-name-given-names">S. R.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Activation of RNase L is dependent on OAS3 expression during infection with diverse human viruses</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">113</span>, <span class="cit-fpage">2241</span>–<span class="cit-lpage">2246</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1519657113%26rft_id%253Dinfo%253Apmid%252F26858407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTEzLzgvMjI0MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzEyLzA0LzQ4NDY3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.35" data-doi="10.1126/science.1155406"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Botos</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leonard</span>,  <span class="cit-name-given-names">J. N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shiloach</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Segal</span>,  <span class="cit-name-given-names">D. M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Davies</span>,  <span class="cit-name-given-names">D. R.</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">Structural basis of toll-like receptor 3 signaling with double-stranded RNA</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">320</span>, <span class="cit-fpage">379</span>–<span class="cit-lpage">381</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLiu%26rft.auinit1%253DL.%26rft.volume%253D320%26rft.issue%253D5874%26rft.spage%253D379%26rft.epage%253D381%26rft.atitle%253DStructural%2BBasis%2Bof%2BToll-Like%2BReceptor%2B3%2BSignaling%2Bwith%2BDouble-Stranded%2BRNA%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1155406%26rft_id%253Dinfo%253Apmid%252F18420935%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjAvNTg3NC8zNzkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8xMi8wNC80ODQ2NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.36" data-doi="10.1038/nature06042"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Malathi</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dong</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gale</span>,  <span class="cit-name-given-names">M.</span>,   <span class="cit-name-suffix">Jr.</span>.</span>, and <span class="cit-auth"><span class="cit-name-surname">Silverman</span>,  <span class="cit-name-given-names">R. H.</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">Small self-RNA generated by RNase L amplifies antiviral innate immunity</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">448</span>, <span class="cit-fpage">816</span>–<span class="cit-lpage">819</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMalathi%26rft.auinit1%253DK.%26rft.volume%253D448%26rft.issue%253D7155%26rft.spage%253D816%26rft.epage%253D819%26rft.atitle%253DSmall%2Bself-RNA%2Bgenerated%2Bby%2BRNase%2BL%2Bamplifies%2Bantiviral%2Binnate%2Bimmunity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06042%26rft_id%253Dinfo%253Apmid%252F17653195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17653195&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248754200042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.37" data-doi="10.1261/rna.063040.117"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Marques-Ramos</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Candeias</span>,  <span class="cit-name-given-names">M. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Menezes</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lacerda</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Willcocks</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Teixeira</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Locker</span>,  <span class="cit-name-given-names">N.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Romao</span>,  <span class="cit-name-given-names">L.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Cap-independent translation ensures mTOR expression and function upon protein synthesis inhibition</span>. <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">23</span>, <span class="cit-fpage">1712</span>–<span class="cit-lpage">1728</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.063040.117%26rft_id%253Dinfo%253Apmid%252F28821580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjEwOiIyMy8xMS8xNzEyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMTIvMDQvNDg0Njc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Michalska</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blaszczyk</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wesoly</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Bluyssen</span>,  <span class="cit-name-given-names">H. A. R.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">A Positive Feedback Amplifier Circuit That Regulates Interferon (IFN)-Stimulated Gene Expression and Controls Type I and Type II IFN Responses</span>. <abbr class="cit-jnl-abbrev">Front Immunol</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">1135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront%2BImmunol%26rft.volume%253D9%26rft.spage%253D1135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.39" data-doi="10.1073/pnas.76.6.2600"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nilsen</span>,  <span class="cit-name-given-names">T. W.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Baglioni</span>,  <span class="cit-name-given-names">C.</span></span> (<span class="cit-pub-date">1979</span>). <span class="cit-article-title">Mechanism for discrimination between viral and host mRNA in interferon-treated cells</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">76</span>, <span class="cit-fpage">2600</span>–<span class="cit-lpage">2604</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DNilsen%26rft.auinit1%253DT.%2BW.%26rft.volume%253D76%26rft.issue%253D6%26rft.spage%253D2600%26rft.epage%253D2604%26rft.atitle%253DMechanism%2Bfor%2Bdiscrimination%2Bbetween%2Bviral%2Band%2Bhost%2BmRNA%2Bin%2Binterferon-treated%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.76.6.2600%26rft_id%253Dinfo%253Apmid%252F223150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI3Ni82LzI2MDAiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8xMi8wNC80ODQ2NzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nogimori</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nishiura</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kawashima</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nagai</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oishi</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hosoda</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Imataka</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kitamura</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kitade</span>,  <span class="cit-name-given-names">Y.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Hoshino</span>,  <span class="cit-name-given-names">S. I.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Dom34 mediates targeting of exogenous RNA in the antiviral OAS/RNase L pathway</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.41" data-doi="10.1073/pnas.0909773106"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Parsyan</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shahbazian</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martineau</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Petroulakis</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alain</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Larsson</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mathonnet</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tettweiler</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hellen</span>,  <span class="cit-name-given-names">C. U.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pestova</span>,  <span class="cit-name-given-names">T. V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">et</span>  <span class="cit-name-given-names">al.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">The helicase protein DHX29 promotes translation initiation, cell proliferation, and tumorigenesis</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">106</span>, <span class="cit-fpage">22217</span>–<span class="cit-lpage">22222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0909773106%26rft_id%253Dinfo%253Apmid%252F20018725%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA2LzUyLzIyMjE3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMTIvMDQvNDg0Njc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.42" data-doi="10.1073/pnas.1513034112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rath</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Donovan</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whitney</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chitrakar</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">W.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Korennykh</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Human RNase L tunes gene expression by selectively destabilizing the microRNA-regulated transcriptome</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">112</span>, <span class="cit-fpage">15916</span>–<span class="cit-lpage">15921</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1513034112%26rft_id%253Dinfo%253Apmid%252F26668391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEyLzUyLzE1OTE2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMTIvMDQvNDg0Njc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.43" data-doi="10.1038/srep32886"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Taniuchi</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miyake</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tsugawa</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oyadomari</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Oyadomari</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Integrated stress response of vertebrates is regulated by four eIF2alpha kinases</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">32886</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D6%26rft.spage%253D32886%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep32886%26rft_id%253Dinfo%253Apmid%252F27633668%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep32886&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27633668&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.44" data-doi="10.1002/embj.201386123"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van den Elzen</span>,  <span class="cit-name-given-names">A. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schuller</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Green</span>,  <span class="cit-name-given-names">R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Seraphin</span>,  <span class="cit-name-given-names">B.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Dom34-Hbs1 mediated dissociation of inactive 80S ribosomes promotes restart of translation after stress</span>. <abbr class="cit-jnl-abbrev">EMBO J</abbr> <span class="cit-vol">33</span>, <span class="cit-fpage">265</span>–<span class="cit-lpage">276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ%26rft.volume%253D33%26rft.spage%253D265%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fembj.201386123%26rft_id%253Dinfo%253Apmid%252F24424461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/embj.201386123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24424461&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.45" data-doi="10.1007/s003350010194"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nie</span>,  <span class="cit-name-given-names">H.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Silverman</span>,  <span class="cit-name-given-names">R. H.</span></span> (<span class="cit-pub-date">2000</span>). <span class="cit-article-title">Analysis and origins of the human and mouse RNase L genes: mediators of interferon action</span>. <abbr class="cit-jnl-abbrev">Mamm Genome</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">989</span>–<span class="cit-lpage">992</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DZhou%26rft.auinit1%253DA.%26rft.volume%253D11%26rft.issue%253D11%26rft.spage%253D989%26rft.epage%253D992%26rft.atitle%253DAnalysis%2Band%2Borigins%2Bof%2Bthe%2Bhuman%2Band%2Bmouse%2BRNase%2BL%2Bgenes%253A%2Bmediators%2Bof%2Binterferon%2Baction.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs003350010194%26rft_id%253Dinfo%253Apmid%252F11063255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s003350010194&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11063255&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.46" data-doi="10.1093/emboj/16.21.6355"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Paranjape</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brown</span>,  <span class="cit-name-given-names">T. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nie</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Naik</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dong</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Trapp</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fairchild</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Colmenares</span>,  <span class="cit-name-given-names">C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Silverman</span>,  <span class="cit-name-given-names">R. H.</span></span> (<span class="cit-pub-date">1997</span>). <span class="cit-article-title">Interferon action and apoptosis are defective in mice devoid of 2′,5′-oligoadenylate-dependent RNase L</span>. <abbr class="cit-jnl-abbrev">Embo J</abbr> <span class="cit-vol">16</span>, <span class="cit-fpage">6355</span>–<span class="cit-lpage">6363</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DZhou%26rft.auinit1%253DA.%26rft.volume%253D16%26rft.issue%253D21%26rft.spage%253D6355%26rft.epage%253D6363%26rft.atitle%253DInterferon%2Baction%2Band%2Bapoptosis%2Bare%2Bdefective%2Bin%2Bmice%2Bdevoid%2Bof%2B2%2527%252C5%2527-oligoadenylate-dependent%2BRNase%2BL.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F16.21.6355%26rft_id%253Dinfo%253Apmid%252F9351818%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMTYvMjEvNjM1NSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzEyLzA0LzQ4NDY3NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.47" data-doi="10.1038/nrm3025"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zoncu</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Efeyan</span>,  <span class="cit-name-given-names">A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Sabatini</span>,  <span class="cit-name-given-names">D. M.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">mTOR: from growth signal integration to cancer, diabetes and ageing</span>. <abbr class="cit-jnl-abbrev">Nat Rev Mol Cell Biol</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">21</span>–<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DZoncu%26rft.auinit1%253DR.%26rft.volume%253D12%26rft.issue%253D1%26rft.spage%253D21%26rft.epage%253D35%26rft.atitle%253DmTOR%253A%2Bfrom%2Bgrowth%2Bsignal%2Bintegration%2Bto%2Bcancer%252C%2Bdiabetes%2Band%2Bageing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm3025%26rft_id%253Dinfo%253Apmid%252F21157483%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm3025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21157483&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285544000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><div class="section ref-list" id="ref-list-2"><h2 class="">REFERENCES</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.48" data-doi="10.1093/bioinformatics/btu638"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anders</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pyl</span>,  <span class="cit-name-given-names">P. T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Huber</span>,  <span class="cit-name-given-names">W.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">HTSeq - a Python framework to work with high-throughput sequencing data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>, <span class="cit-fpage">166</span>–<span class="cit-lpage">169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu638%26rft_id%253Dinfo%253Apmid%252F25260700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu638&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25260700&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000347832300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bern</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kil</span>,  <span class="cit-name-given-names">Y. J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Becker</span>,  <span class="cit-name-given-names">C.</span></span> (<span class="cit-pub-date">2012</span>). <abbr class="cit-jnl-abbrev">Byonic: advanced peptide and protein identification software. Curr Protoc Bioinformatics Chapter 13, Unit13 20</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Donovan</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rath</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kolet-Mandrikov</span>,  <span class="cit-name-given-names">D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Korennykh</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Rapid RNase L-driven arrest of protein synthesis in the dsRNA response without degradation of translation machinery</span>. <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">23</span>, <span class="cit-fpage">16601671</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft.volume%253D23%26rft.spage%253D16601671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.51" data-doi="10.1021/pr500202e"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dorfer</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pichler</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stranzl</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stadlmann</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taus</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Winkler</span>,  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Mechtler</span>,  <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">MS Amanda, a universal identification algorithm optimized for high accuracy tandem mass spectra</span>. <abbr class="cit-jnl-abbrev">J Proteome Res</abbr> <span class="cit-vol">13</span>, <span class="cit-fpage">3679</span>–<span class="cit-lpage">3684</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BProteome%2BRes%26rft.volume%253D13%26rft.spage%253D3679%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fpr500202e%26rft_id%253Dinfo%253Apmid%252F24909410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/pr500202e&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24909410&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.52" data-doi="10.1016/1044-0305(94)80016-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eng</span>,  <span class="cit-name-given-names">J. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McCormack</span>,  <span class="cit-name-given-names">A. L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Yates</span>,  <span class="cit-name-given-names">J. R.</span></span> (<span class="cit-pub-date">1994</span>). <span class="cit-article-title">An approach to correlate tandem mass spectral data of peptides with amino acid sequences in a protein database</span>. <abbr class="cit-jnl-abbrev">J Am Soc Mass Spectrom</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">976</span>–<span class="cit-lpage">989</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BAm%2BSoc%2BMass%2BSpectrom%26rft.volume%253D5%26rft.spage%253D976%26rft_id%253Dinfo%253Adoi%252F10.1016%252F1044-0305%252894%252980016-2%26rft_id%253Dinfo%253Apmid%252F24226387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/1044-0305(94)80016-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24226387&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PP71300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.53" data-doi="10.1016/j.jmb.2013.04.016"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gupta</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">DeMong</span>,  <span class="cit-name-given-names">L. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Banda</span>,  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Copeland</span>,  <span class="cit-name-given-names">P. R.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Reconstitution of selenocysteine incorporation reveals intrinsic regulation by SECIS elements</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr> <span class="cit-vol">425</span>, <span class="cit-fpage">2415</span>–<span class="cit-lpage">2422</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BMol%2BBiol%26rft.volume%253D425%26rft.spage%253D2415%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2013.04.016%26rft_id%253Dinfo%253Apmid%252F23624110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2013.04.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23624110&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.54" data-doi="10.1186/gb-2013-14-4-r36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pertea</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Trapnell</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pimentel</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kelley</span>,  <span class="cit-name-given-names">R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Salzberg</span>,  <span class="cit-name-given-names">S. L.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions</span>. <abbr class="cit-jnl-abbrev">Genome Biol</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">R36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D14%26rft.spage%253DR36%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-4-r36%26rft_id%253Dinfo%253Apmid%252F23618408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-4-r36&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23618408&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.55" data-doi="10.1093/emboj/18.23.6705"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lorsch</span>,  <span class="cit-name-given-names">J. R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Herschlag</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">1999</span>). <span class="cit-article-title">Kinetic dissection of fundamental processes of eukaryotic translation initiation in vitro</span>. <abbr class="cit-jnl-abbrev">EMBO J</abbr> <span class="cit-vol">18</span>, <span class="cit-fpage">6705</span>–<span class="cit-lpage">6717</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DLorsch%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D18%26rft.issue%253D23%26rft.spage%253D6705%26rft.epage%253D6717%26rft.atitle%253DKinetic%2Bdissection%2Bof%2Bfundamental%2Bprocesses%2Bof%2Beukaryotic%2Btranslation%2Binitiation%2Bin%2Bvitro%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F18.23.6705%26rft_id%253Dinfo%253Apmid%252F10581244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/emboj/18.23.6705&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10581244&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084177700012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.56" data-doi="10.1021/ac0341261"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nesvizhskii</span>,  <span class="cit-name-given-names">A. I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keller</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kolker</span>,  <span class="cit-name-given-names">E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Aebersold</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2003</span>). <span class="cit-article-title">A statistical model for identifying proteins by tandem mass spectrometry</span>. <abbr class="cit-jnl-abbrev">Anal Chem</abbr> <span class="cit-vol">75</span>, <span class="cit-fpage">4646</span>–<span class="cit-lpage">4658</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnalytical%2BChemistry%2B%2528Washington%252C%2BDC%2529%26rft.stitle%253DAnalytical%2BChemistry%2B%2528Washington%252C%2BDC%2529%26rft.aulast%253DNesvizhskii%26rft.auinit1%253DA.%2BI.%26rft.volume%253D75%26rft.issue%253D17%26rft.spage%253D4646%26rft.epage%253D4658%26rft.atitle%253DA%2Bstatistical%2Bmodel%2Bfor%2Bidentifying%2Bproteins%2Bby%2Btandem%2Bmass%2Bspectrometry.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fac0341261%26rft_id%253Dinfo%253Apmid%252F14632076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ac0341261&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14632076&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.57" data-doi="10.1073/pnas.1513034112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rath</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Donovan</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whitney</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chitrakar</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">W.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Korennykh</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Human RNase L tunes gene expression by selectively destabilizing the microRNA-regulated transcriptome</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">112</span>, <span class="cit-fpage">15916</span>–<span class="cit-lpage">15921</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1513034112%26rft_id%253Dinfo%253Apmid%252F26668391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEyLzUyLzE1OTE2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMTIvMDQvNDg0Njc1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-484675v1.58" data-doi="10.1038/nbt.1754"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Robinson</span>,  <span class="cit-name-given-names">J. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thorvaldsdottir</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Winckler</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guttman</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lander</span>,  <span class="cit-name-given-names">E. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Getz</span>,  <span class="cit-name-given-names">G.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Mesirov</span>,  <span class="cit-name-given-names">J. P.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Integrative genomics viewer</span>. <abbr class="cit-jnl-abbrev">Nat Biotechnol</abbr> <span class="cit-vol">29</span>, <span class="cit-fpage">24</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DRobinson%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D24%26rft.epage%253D26%26rft.atitle%253DIntegrative%2Bgenomics%2Bviewer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1754%26rft_id%253Dinfo%253Apmid%252F21221095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1754&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21221095&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F12%2F04%2F484675.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286048900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/484675v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F484675v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/487330v1" title="Super-resolution Molecular Map of Basal Foot Reveals Novel Cilium in Airway Multiciliated Cells" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;December 04, 2018.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/484675v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/484675v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/484675v1" class="active" data-icon-position="" data-hide-link-title="0">Concerted 2-5A-Mediated mRNA Decay and Transcription Reprogram Protein Synthesis in dsRNA Response</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/145109" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-CdLwwacF9ZbQ1WYbfsh3OTuApOAw1j9ykiHjLJbAVi4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="145109" id="node145109--2" data-pisa="biorxiv;484675v1" data-pisa-master="biorxiv;484675" data-seqnum="145109" data-apath="/biorxiv/early/2018/12/04/484675.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Concerted 2-5A-Mediated mRNA Decay and Transcription Reprogram Protein Synthesis in dsRNA Response</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Sneha</span> <span class="nlm-surname">Rath</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Eliza</span> <span class="nlm-surname">Prangley</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jesse</span> <span class="nlm-surname">Donovan</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Kaitlin</span> <span class="nlm-surname">Demarest</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Ned S.</span> <span class="nlm-surname">Wingreen</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Yigal</span> <span class="nlm-surname">Meir</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Alexei</span> <span class="nlm-surname">Korennykh</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">484675; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/484675 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/484675v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/484675v1&amp;title=Concerted%202-5A-Mediated%20mRNA%20Decay%20and%20Transcription%20Reprogram%20Protein%20Synthesis%20in%20dsRNA%20Response" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/484675v1&amp;title=Concerted%202-5A-Mediated%20mRNA%20Decay%20and%20Transcription%20Reprogram%20Protein%20Synthesis%20in%20dsRNA%20Response" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/484675v1&amp;text=Concerted%202-5A-Mediated%20mRNA%20Decay%20and%20Transcription%20Reprogram%20Protein%20Synthesis%20in%20dsRNA%20Response" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/484675v1&amp;title=Concerted%202-5A-Mediated%20mRNA%20Decay%20and%20Transcription%20Reprogram%20Protein%20Synthesis%20in%20dsRNA%20Response" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/484675v1&amp;t=Concerted%202-5A-Mediated%20mRNA%20Decay%20and%20Transcription%20Reprogram%20Protein%20Synthesis%20in%20dsRNA%20Response" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/484675v1&amp;title=Concerted%202-5A-Mediated%20mRNA%20Decay%20and%20Transcription%20Reprogram%20Protein%20Synthesis%20in%20dsRNA%20Response" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/484675v1&amp;title=Concerted%202-5A-Mediated%20mRNA%20Decay%20and%20Transcription%20Reprogram%20Protein%20Synthesis%20in%20dsRNA%20Response" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="145109" id="node145109--3" data-pisa="biorxiv;484675v1" data-pisa-master="biorxiv;484675" data-seqnum="145109" data-apath="/biorxiv/early/2018/12/04/484675.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Concerted 2-5A-Mediated mRNA Decay and Transcription Reprogram Protein Synthesis in dsRNA Response</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Sneha</span> <span class="nlm-surname">Rath</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Eliza</span> <span class="nlm-surname">Prangley</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jesse</span> <span class="nlm-surname">Donovan</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Kaitlin</span> <span class="nlm-surname">Demarest</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Ned S.</span> <span class="nlm-surname">Wingreen</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Yigal</span> <span class="nlm-surname">Meir</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Alexei</span> <span class="nlm-surname">Korennykh</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">484675; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/484675 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Concerted%202-5A-Mediated%20mRNA%20Decay%20and%20Transcription%20Reprogram%20Protein%20Synthesis%20in%20dsRNA%20Response&amp;rft.volume&amp;rft.issue&amp;rft.date=2018-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Rath%2C+Sneha&amp;rft.au=Prangley%2C+Eliza&amp;rft.au=Donovan%2C+Jesse&amp;rft.au=Demarest%2C+Kaitlin&amp;rft.au=Wingreen%2C+Ned+S.&amp;rft.au=Meir%2C+Yigal&amp;rft.au=Korennykh%2C+Alexei"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/145109/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/145109/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/145109/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/145109/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/145109/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/145109/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/145109/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/145109/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/145109/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/145109/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/145109/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/145109/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/484675v1&amp;count=horizontal&amp;via=&amp;text=Concerted%202-5A-Mediated%20mRNA%20Decay%20and%20Transcription%20Reprogram%20Protein%20Synthesis%20in%20dsRNA%20Response&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/484675v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/484675v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/484675v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/cell-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Cell Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20277)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
