<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/03/12/2020.03.11.987958.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.03.11.987958v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="A human monoclonal antibody blocking SARS-CoV-2 infection"/>
  <meta name="DC.Identifier" content="10.1101/2020.03.11.987958"/>
  <meta name="DC.Date" content="2020-03-12"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="The emergence of the novel human coronavirus SARS-CoV-2 in Wuhan, China has caused a worldwide epidemic of respiratory disease (COVID-19). Vaccines and targeted therapeutics for treatment of this disease are currently lacking. Here we report a human monoclonal antibody that neutralizes SARS-CoV-2 (and SARS-CoV). This cross-neutralizing antibody targets a communal epitope on these viruses and offers potential for prevention and treatment of COVID-19."/>
  <meta name="DC.Contributor" content="Chunyan Wang"/>
  <meta name="DC.Contributor" content="Wentao Li"/>
  <meta name="DC.Contributor" content="Dubravka Drabek"/>
  <meta name="DC.Contributor" content="Nisreen M.A. Okba"/>
  <meta name="DC.Contributor" content="Rien van Haperen"/>
  <meta name="DC.Contributor" content="Albert D.M.E. Osterhaus"/>
  <meta name="DC.Contributor" content="Frank J.M. van Kuppeveld"/>
  <meta name="DC.Contributor" content="Bart L. Haagmans"/>
  <meta name="DC.Contributor" content="Frank Grosveld"/>
  <meta name="DC.Contributor" content="Berend-Jan Bosch"/>
  <meta name="article:published_time" content="2020-03-12"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="A human monoclonal antibody blocking SARS-CoV-2 infection"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;The emergence of the novel human coronavirus SARS-CoV-2 in Wuhan, China has caused a worldwide epidemic of respiratory disease (COVID-19). Vaccines and targeted therapeutics for treatment of this disease are currently lacking. Here we report a human monoclonal antibody that neutralizes SARS-CoV-2 (and SARS-CoV). This cross-neutralizing antibody targets a communal epitope on these viruses and offers potential for prevention and treatment of COVID-19.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.03.11.987958v1"/>
  <meta name="citation_id" content="2020.03.11.987958v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.11.987958v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.11.987958v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.11.987958v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.03.11.987958"/>
  <meta name="citation_num_pages" content="24"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.03.11.987958"/>
  <meta name="citation_author" content="Chunyan Wang"/>
  <meta name="citation_author_institution" content="Virology Division, Department of Biomolecular Health Sciences, Faculty of Veterinary Medicine, Utrecht University"/>
  <meta name="citation_author" content="Wentao Li"/>
  <meta name="citation_author_institution" content="Virology Division, Department of Biomolecular Health Sciences, Faculty of Veterinary Medicine, Utrecht University"/>
  <meta name="citation_author" content="Dubravka Drabek"/>
  <meta name="citation_author_institution" content="Department of Cell Biology, Erasmus MC"/>
  <meta name="citation_author_institution" content="Harbour Antibodies B.V."/>
  <meta name="citation_author" content="Nisreen M.A. Okba"/>
  <meta name="citation_author_institution" content="Department of Molecular and Cell Biology, Department of Viroscience, Erasmus Medical Center"/>
  <meta name="citation_author" content="Rien van Haperen"/>
  <meta name="citation_author_institution" content="Department of Cell Biology, Erasmus MC"/>
  <meta name="citation_author_institution" content="Harbour Antibodies B.V."/>
  <meta name="citation_author" content="Albert D.M.E. Osterhaus"/>
  <meta name="citation_author_institution" content="University of Veterinary Medicine"/>
  <meta name="citation_author" content="Frank J.M. van Kuppeveld"/>
  <meta name="citation_author_institution" content="Virology Division, Department of Biomolecular Health Sciences, Faculty of Veterinary Medicine, Utrecht University"/>
  <meta name="citation_author" content="Bart L. Haagmans"/>
  <meta name="citation_author_institution" content="Department of Molecular and Cell Biology, Department of Viroscience, Erasmus Medical Center"/>
  <meta name="citation_author" content="Frank Grosveld"/>
  <meta name="citation_author_institution" content="Department of Cell Biology, Erasmus MC"/>
  <meta name="citation_author_institution" content="Harbour Antibodies B.V."/>
  <meta name="citation_author" content="Berend-Jan Bosch"/>
  <meta name="citation_author_institution" content="Virology Division, Department of Biomolecular Health Sciences, Faculty of Veterinary Medicine, Utrecht University"/>
  <meta name="citation_author_email" content="b.j.bosch@uu.nl"/>
  <meta name="citation_reference" content="Zhou, P. et al. A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature, 1–4 (2020)."/>
  <meta name="citation_reference" content="World Health Organization. https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200310-sitrep-50-covid-19.pdf?sfvrsn=55e904fb_2."/>
  <meta name="citation_reference" content="Coronaviridae Study Group of the International Committee on Taxonomy of Viruses. The species Severe acute respiratory syndrome-related coronavirus: classifying 2019-nCoV and naming it SARS-CoV-2. Nat. Microbiol. (2020)."/>
  <meta name="citation_reference" content="Prabakaran, P. et al. Potent human monoclonal antibodies against SARS CoV, Nipah and Hendra viruses. Expert opinion on biological therapy 9, 355–368 (2009)."/>
  <meta name="citation_reference" content="Saphire, E. O., Schendel, S. L., Gunn, B. M., Milligan, J. C. &amp;; Alter, G. Antibody-mediated protection against Ebola virus. Nat. Immunol. 19, 1169–1178 (2018)."/>
  <meta name="citation_reference" content="Reguera, J. et al. Structural bases of coronavirus attachment to host aminopeptidase N and its inhibition by neutralizing antibodies. PLoS Pathog. 8, e1002859 (2012)."/>
  <meta name="citation_reference" content="Yu, X. et al. Structural basis for the neutralization of MERS-CoV by a human monoclonal antibody MERS-27. Sci. Rep. 5, 13133 (2015)."/>
  <meta name="citation_reference" content="Prabakaran, P. et al. Structure of severe acute respiratory syndrome coronavirus receptor-binding domain complexed with neutralizing antibody. J. Biol. Chem. 281, 15829–15836 (2006)."/>
  <meta name="citation_reference" content="Hwang, W. C. et al. Structural basis of neutralization by a human anti-severe acute respiratory syndrome spike protein antibody, 80R. J. Biol. Chem. 281, 34610–34616 (2006)."/>
  <meta name="citation_reference" content="Rockx, B. et al. Structural basis for potent cross-neutralizing human monoclonal antibody protection against lethal human and zoonotic severe acute respiratory syndrome coronavirus challenge. J. Virol. 82, 3220–3235 (2008)."/>
  <meta name="citation_reference" content="Widjaja, I. et al. Towards a solution to MERS: protective human monoclonal antibodies targeting different domains and functions of the MERS-coronavirus spike glycoprotein. Emerg. Microbes Infect. 8, 516–530 (2019)."/>
  <meta name="citation_reference" content="Wrapp, D. et al. Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation. Science (2020)."/>
  <meta name="citation_reference" content="Walls, A. C. et al. Structure, Function, and Antigenicity of the SARS-CoV-2 Spike Glycoprotein. Cell (2020)."/>
  <meta name="citation_reference" content="Li, W. et al. Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus. Nature 426, 450–454 (2003)."/>
  <meta name="citation_reference" content="Walls, A. C. et al. Unexpected receptor functional mimicry elucidates activation of coronavirus fusion. Cell 176, 1026–1039. e15 (2019)."/>
  <meta name="citation_reference" content="Tian, X. et al. Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody. Emerging Microbes &amp;; Infections 9, 382–385 (2020)."/>
  <meta name="citation_reference" content="Menachery, V. D. et al. A SARS-like cluster of circulating bat coronaviruses shows potential for human emergence. Nat. Med. 21, 1508 (2015)."/>
  <meta name="citation_reference" content="Li, F., Li, W., Farzan, M. &amp;; Harrison, S. C. Structure of SARS coronavirus spike receptor-binding domain complexed with receptor. Science 309, 1864–1868 (2005)."/>
  <meta name="citation_reference" content="Raj, V. S. et al. Dipeptidyl peptidase 4 is a functional receptor for the emerging human coronavirus-EMC. Nature 495, 251–254 (2013)."/>
  <meta name="citation_reference" content="Okba, N. M. A. et al. Sensitive and Specific Detection of Low-Level Antibody Responses in Mild Middle East Respiratory Syndrome Coronavirus Infections. Emerg. Infect. Dis. 25, 1868–1877 (2019)."/>
  <meta name="citation_reference" content="Widjaja, I. et al. Towards a solution to MERS: protective human monoclonal antibodies targeting different domains and functions of the MERS-coronavirus spike glycoprotein. Emerg. Microbes Infect. 8, 516–530 (2019)."/>
  <meta name="citation_reference" content="Widjaja, I. et al. Towards a solution to MERS: protective human monoclonal antibodies targeting different domains and functions of the MERS-coronavirus spike glycoprotein. Emerg. Microbes Infect. 8, 516–530 (2019)."/>
  <meta name="citation_reference" content="Widjaja, I. et al. Towards a solution to MERS: protective human monoclonal antibodies targeting different domains and functions of the MERS-coronavirus spike glycoprotein. Emerg. Microbes Infect. 8, 516–530 (2019)."/>
  <meta name="citation_reference" content="Li, F., Li, W., Farzan, M. &amp;; Harrison, S.C. Structure of SARS coronavirus spike receptor-binding domain complexed with receptor. Science 309, 1864–1868 (2005)."/>
  <meta name="twitter:title" content="A human monoclonal antibody blocking SARS-CoV-2 infection"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="The emergence of the novel human coronavirus SARS-CoV-2 in Wuhan, China has caused a worldwide epidemic of respiratory disease (COVID-19). Vaccines and targeted therapeutics for treatment of this disease are currently lacking. Here we report a human monoclonal antibody that neutralizes SARS-CoV-2 (and SARS-CoV). This cross-neutralizing antibody targets a communal epitope on these viruses and offers potential for prevention and treatment of COVID-19."/>
  <meta name="og-title" property="og:title" content="A human monoclonal antibody blocking SARS-CoV-2 infection"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.11.987958v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="The emergence of the novel human coronavirus SARS-CoV-2 in Wuhan, China has caused a worldwide epidemic of respiratory disease (COVID-19). Vaccines and targeted therapeutics for treatment of this disease are currently lacking. Here we report a human monoclonal antibody that neutralizes SARS-CoV-2 (and SARS-CoV). This cross-neutralizing antibody targets a communal epitope on these viruses and offers potential for prevention and treatment of COVID-19."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-03-12"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>A human monoclonal antibody blocking SARS-CoV-2 infection | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1192808 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-4">The emergence of the novel human coronavirus SARS-CoV-2 in Wuhan, China has caused a worldwide epidemic of respiratory disease (COVID-19). Vaccines and targeted therapeutics for treatment of this disease are currently lacking. Here we report a human monoclonal antibody that neutralizes SARS-CoV-2 (and SARS-CoV). This cross-neutralizing antibody targets a communal epitope on these viruses and offers potential for prevention and treatment of COVID-19.</p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Main text</h2>
    <p id="p-5">The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the etiological agent of the coronavirus induced disease 19 (COVID-19) that emerged in China late 2019 and causing a worldwide epidemic
     <sup>
      <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>
     </sup>. As of March 10
     <sup>th</sup> 2020, over 115,113 cases have been reported in 109 countries, of which 4,063 (3.5%) succumbed to the infection
     <sup>
      <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>
     </sup>. SARS-CoV-2 belongs to the 
     <em>Sarbecovirus</em> subgenus (genus 
     <em>Betacoronavirus</em>, family 
     <em>Coronaviridae</em>)
     <sup>
      <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>
     </sup> together with SARS-CoV that emerged in 2002 causing approximately 8000 infections with a lethality of 10%. Both viruses crossed species barriers from an animal reservoir and can cause a life-threatening respiratory illness in humans. Presently no approved targeted therapeutics are available for COVID-19. Monoclonal antibodies targeting vulnerable sites on viral surface proteins are increasingly recognised as a promising class of drugs against infectious diseases and have shown therapeutic efficacy for a number of viruses
     <sup>
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>, 
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>
     </sup>.
    </p>
    <p id="p-6">Coronavirus neutralizing antibodies primarily target the trimeric spike (S) glycoproteins on the viral surface that mediate entry into host cells. The S protein has two functional subunits that mediate cell attachment (the S1 subunit, existing of four core domains S1
     <sub>A</sub> through S1
     <sub>D</sub>) and fusion of the viral and cellular membrane (the S2 subunit). Potent neutralizing antibodies often target the receptor interaction site in S1, disabling receptor interactions
     <sup>
      <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>–
      <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>
     </sup>. The spike proteins of SARS-CoV-2 (SARS2-S; 1,273 residues, strain Wuhan-Hu-1) and SARS-CoV (SARS-S, 1,255 residues, strain Urbani) are 77.5% identical by primary amino acid sequence, are structurally very similar
     <sup>
      <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>, 
      <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>
     </sup> and commonly bind the human angiotensin coverting enzyme 2 (ACE2) protein as a host receptor
     <sup>
      <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>, 
      <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>
     </sup> through their S1
     <sub>B</sub> domain. Receptor interaction is known to trigger irreversible conformational changes in coronavirus spike proteins enabling membrane fusion
     <sup>
      <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>
     </sup>.
    </p>
    <p id="p-7">In order to identify SARS-CoV-2 neutralizing antibodies, ELISA-(cross)reactivity was assessed of antibody-containing supernatants of a collection of 51 SARS-S hybridoma’s derived from immunized transgenic H2L2 mice that encode chimeric immunoglobulins with human variable heavy and light chains and constant regions of rat origin (
     <a id="xref-fig-3-1" class="xref-fig" href="#F3">Suppl.Fig.1</a>). Four of 51 SARS-S hybridoma supernatants displayed ELISA-cross-reactivity with the SARS2-S1 subunit (S residues 1-681; 
     <a id="xref-fig-3-2" class="xref-fig" href="#F3">Suppl.Fig.1</a>), of which one (47D11) exhibited cross-neutralizing activity of SARS-S and SARS2-S pseudotyped VSV infection. The chimeric 47D11 H2L2 antibody was reformatted and recombinantly expressed as a fully human IgG1 isotype antibody for further characterization.
    </p>
    <div id="F3" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="ELISA cross-reactivity of antibody-containing supernatants of SARS-S H2L2 hybridomas towards SARS2-S1. SARS-S targeting hybridomas developed by conventional hybridoma technology from immunized H2L2 transgenic mice (Harbour Biomed), as described before1. These mice - carrying encoding the heavy and light chain human immunoglobulin repertoire - were sequentially immunized with 2-week intervals with trimeric spike protein ectodomains (Secto) of three human coronaviruses from the betacoronavirus genus in the following order: 1. HCoV-OC43-Secto, 2. SARS-CoV-Secto, 3. MERS-CoV-Secto, 4. HCoV-OC43-Secto, 5. SARS-CoV-Secto, 6. MERS-CoV-Secto. Four days after the last immunization, splenocytes and lymph node lymphocytes were harvested and hybridomas were generated. Antibodies in the cell supernatants were tested for ELISA-reactivity against SARS-Secto, SARS-S1, SARS-S1A and SARS2-S1. Of the 51 hybridoma supernatants that reacted with SARS-Secto only, 23 reacted with SARS-S1A, 22 with SARS-S1 but not SARS-S1A, 6 with SARS-Secto but not SARS-S1. Four of the 51 SARS-Secto hybridoma supernatants reacted with SARS2-S1 (see column on the right). The table displays ELISA-signal intensities (OD450nm values) of hybridoma supernatants for the different antigens." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1705609420" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;ELISA cross-reactivity of antibody-containing supernatants of SARS-S H2L2 hybridomas towards SARS2-S1.&lt;;/span&gt;; SARS-S targeting hybridomas developed by conventional hybridoma technology from immunized H2L2 transgenic mice (Harbour Biomed), as described before&lt;;sup&gt;;1&lt;;/sup&gt;;. These mice - carrying encoding the heavy and light chain human immunoglobulin repertoire - were sequentially immunized with 2-week intervals with trimeric spike protein ectodomains (S&lt;;sub&gt;;ecto&lt;;/sub&gt;;) of three human coronaviruses from the betacoronavirus genus in the following order: 1. HCoV-OC43-S&lt;;sub&gt;;ecto&lt;;/sub&gt;;, 2. SARS-CoV-S&lt;;sub&gt;;ecto&lt;;/sub&gt;;, 3. MERS-CoV-S&lt;;sub&gt;;ecto&lt;;/sub&gt;;, 4. HCoV-OC43-S&lt;;sub&gt;;ecto&lt;;/sub&gt;;, 5. SARS-CoV-S&lt;;sub&gt;;ecto&lt;;/sub&gt;;, 6. MERS-CoV-S&lt;;sub&gt;;ecto&lt;;/sub&gt;;. Four days after the last immunization, splenocytes and lymph node lymphocytes were harvested and hybridomas were generated. Antibodies in the cell supernatants were tested for ELISA-reactivity against SARS-S&lt;;sub&gt;;ecto&lt;;/sub&gt;;, SARS-S1, SARS-S1&lt;;sub&gt;;A&lt;;/sub&gt;; and SARS2-S1. Of the 51 hybridoma supernatants that reacted with SARS-S&lt;;sub&gt;;ecto&lt;;/sub&gt;; only, 23 reacted with SARS-S1&lt;;sub&gt;;A&lt;;/sub&gt;;, 22 with SARS-S1 but not SARS-S1&lt;;sub&gt;;A&lt;;/sub&gt;;, 6 with SARS-S&lt;;sub&gt;;ecto&lt;;/sub&gt;; but not SARS-S1. Four of the 51 SARS-S&lt;;sub&gt;;ecto&lt;;/sub&gt;; hybridoma supernatants reacted with SARS2-S1 (see column on the right). The table displays ELISA-signal intensities (OD&lt;;sub&gt;;450nm&lt;;/sub&gt;; values) of hybridoma supernatants for the different antigens.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Suppl. Fig.1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F3.medium.gif" width="438" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Suppl. Fig.1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F3.medium.gif" width="438" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Suppl. Fig.1." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Suppl. Fig.1.</span>
      <span class="caption-title">ELISA cross-reactivity of antibody-containing supernatants of SARS-S H2L2 hybridomas towards SARS2-S1.</span>
      <p id="p-23" class="first-child">SARS-S targeting hybridomas developed by conventional hybridoma technology from immunized H2L2 transgenic mice (Harbour Biomed), as described before
       <sup>
        <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">1</a>
       </sup>. These mice - carrying encoding the heavy and light chain human immunoglobulin repertoire - were sequentially immunized with 2-week intervals with trimeric spike protein ectodomains (S
       <sub>ecto</sub>) of three human coronaviruses from the betacoronavirus genus in the following order: 1. HCoV-OC43-S
       <sub>ecto</sub>, 2. SARS-CoV-S
       <sub>ecto</sub>, 3. MERS-CoV-S
       <sub>ecto</sub>, 4. HCoV-OC43-S
       <sub>ecto</sub>, 5. SARS-CoV-S
       <sub>ecto</sub>, 6. MERS-CoV-S
       <sub>ecto</sub>. Four days after the last immunization, splenocytes and lymph node lymphocytes were harvested and hybridomas were generated. Antibodies in the cell supernatants were tested for ELISA-reactivity against SARS-S
       <sub>ecto</sub>, SARS-S1, SARS-S1
       <sub>A</sub> and SARS2-S1. Of the 51 hybridoma supernatants that reacted with SARS-S
       <sub>ecto</sub> only, 23 reacted with SARS-S1
       <sub>A</sub>, 22 with SARS-S1 but not SARS-S1
       <sub>A</sub>, 6 with SARS-S
       <sub>ecto</sub> but not SARS-S1. Four of the 51 SARS-S
       <sub>ecto</sub> hybridoma supernatants reacted with SARS2-S1 (see column on the right). The table displays ELISA-signal intensities (OD
       <sub>450nm</sub> values) of hybridoma supernatants for the different antigens.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-8">The human 47D11 antibody binds to cells expressing the full-length spike proteins of SARS-CoV and SARS-CoV-2 (
     <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig.1a</a>). The 47D11 antibody was found to potently inhibit infection of VeroE6 cells with SARS-S and SARS2-S pseudotyped VSV with IC
     <sub>50</sub> values of 0.06 and 0.08 μg/ml (
     <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig.1b</a>), respectively. Authentic infection of VeroE6 cells with SARS-CoV and SARS-CoV-2 was neutralized with IC
     <sub>50</sub> values of 0.19 and 0.57 μg/ml (
     <a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig.1c</a>). Using ELISA 47D11 was shown to target the S1
     <sub>B</sub> receptor binding domain (RBD) of SARS-S and SARS2-S. 47D11 bound the S1
     <sub>B</sub> of both viruses with similar affinities as shown by the ELISA-based half maximal effective concentration (EC
     <sub>50</sub>) values (0.02 and 0.03 μg/ml, respectively; 
     <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig.2a</a>). ELISA-based binding affinity of 47D11 for the spike ectodomain (S
     <sub>ecto</sub>) of SARS-CoV was higher relative to that of SARS-CoV-2 (EC
     <sub>50</sub> values: 0.018 and 0.15 μg/ml, respectively), despite equimolar antigen coating (
     <a id="xref-fig-4-1" class="xref-fig" href="#F4">Suppl.Fig.2</a>). Congruent with the ELISA-reactivities, measurement of binding kinetics of 47D11 by biolayer interferometry showed that 47D11 binds SARS-S
     <sub>ecto</sub> with higher affinity (equilibrium dissociation constant [
     <em>K
      <sub>D</sub>
     </em>]: 0.745 nM) relative to SARS2-S
     <sub>ecto</sub> (
     <em>K
      <sub>D</sub>
     </em> 10.8 nM) whereas affinity for SARS-S1
     <sub>B</sub> and SARS2-S1
     <sub>B</sub> was in a similar range (16.1 and 9.6 nM, respectively, 
     <a id="xref-fig-5-1" class="xref-fig" href="#F5">Suppl.Fig.3</a>). This difference may originate from differences in epitope accessibility in SARS-S versus SARS2-S, as domain B can adopt a closed and open conformation in the prefusion spike homotrimer
     <sup>
      <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>, 
      <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>
     </sup>. Remarkably, binding of 47D11 to SARS-S1
     <sub>B</sub> and SARS2-S1
     <sub>B</sub> did not compete with S1
     <sub>B</sub> binding to the ACE2 receptor expressed at the cell surface as shown by flow cytometry (
     <a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig.2b</a>; 
     <a id="xref-fig-6-1" class="xref-fig" href="#F6">Suppl.Fig.4</a>) nor with S
     <sub>ecto</sub> and S1
     <sub>B</sub> binding to soluble ACE2 in solid-phase based assay (
     <a id="xref-fig-7-1" class="xref-fig" href="#F7">Suppl.Fig.5</a>), whereas two SARS-S1 specific antibodies 35F4 and 43C6 that neutralize SARS-S (but not SARS2-S) pseudotyped VSV infection (
     <a id="xref-fig-8-1" class="xref-fig" href="#F8">Suppl.Fig.6</a>) do block binding of SARS-S
     <sub>ecto</sub> and SARS-S1
     <sub>B</sub> to ACE2. Using a trypsin-triggered cell-cell fusion assay, 47D11 was shown to impair SARS-S and SARS2-S mediated syncytia formation (
     <a id="xref-fig-9-1" class="xref-fig" href="#F9">Suppl.Fig.7</a>). Our data show that 47D11 neutralizes SARS-CoV and SARS-CoV-2 through a yet unknown mechanism that is different from receptor binding interference. Alternative mechanisms of coronavirus neutralization by RBD-targeting antibodies have been reported including spike inactivation through antibody-induced destabilization of its prefusion structure
     <sup>
      <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>
     </sup>, which may also apply for 47D11.
    </p>
    <div id="F4" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="ELISA binding curve of the anti-StrepMAb (IBA) antibody to Strep-tagged spike antigens to corroborate equimolar ELISA plate coating of SARS-Secto / SARS2-Secto (upper panel), SARS-S1B / SARS2-S1B (middle panel) and SARS-S1A / SARS2-S1A (lowerpanel) antigens used in Fig.2a." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1705609420" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;ELISA binding curve of the anti-StrepMAb (IBA) antibody to Strep-tagged spike antigens to corroborate equimolar ELISA plate coating of SARS-S&lt;;sub&gt;;ecto&lt;;/sub&gt;; / SARS2-S&lt;;sub&gt;;ecto&lt;;/sub&gt;; (upper panel), SARS-S1&lt;;sub&gt;;B&lt;;/sub&gt;; / SARS2-S1&lt;;sub&gt;;B&lt;;/sub&gt;; (middle panel) and SARS-S1&lt;;sub&gt;;A&lt;;/sub&gt;; / SARS2-S1&lt;;sub&gt;;A&lt;;/sub&gt;; (lowerpanel) antigens used in Fig.2a.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Suppl. Fig.2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F4.medium.gif" width="206" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Suppl. Fig.2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F4.medium.gif" width="206" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Suppl. Fig.2." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Suppl. Fig.2.</span>
      <p id="p-24" class="first-child">ELISA binding curve of the anti-StrepMAb (IBA) antibody to Strep-tagged spike antigens to corroborate equimolar ELISA plate coating of SARS-S
       <sub>ecto</sub> / SARS2-S
       <sub>ecto</sub> (upper panel), SARS-S1
       <sub>B</sub> / SARS2-S1
       <sub>B</sub> (middle panel) and SARS-S1
       <sub>A</sub> / SARS2-S1
       <sub>A</sub> (lowerpanel) antigens used in 
       <a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig.2a</a>.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F5" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Binding kinetics of 47D11 to the S ectodomain and S1B of SARS-CoV and SARS-CoV-2. Binding kinetics of 47D11 to immobilized recombinant SARS-Secto, SARS2-Secto, SARS-S1B and SARS2-S1B was measured using biolayer interferometry at 25[[#xB0]]C, as described previously21. Kinetic binding assay was performed by loading 47D11 mAb at optimal concentration (42 nM) on anti-human Fc biosensor for 10 mins. Antigen association step was performed by incubating the sensor with a range of concentrations of the recombinant spike ectodomain (1600-800-400-200-100-50-25 nM) for 10 min, followed by a dissociation step in PBS for 60 min. The kinetics constants were calculated using 1:1 Langmuir binding model on Fortebio Data Analysis 7.0 software." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1705609420" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Binding kinetics of 47D11 to the S ectodomain and S1&lt;;sub&gt;;B&lt;;/sub&gt;; of SARS-CoV and SARS-CoV-2.&lt;;/span&gt;; Binding kinetics of 47D11 to immobilized recombinant SARS-S&lt;;sub&gt;;ecto&lt;;/sub&gt;;, SARS2-S&lt;;sub&gt;;ecto&lt;;/sub&gt;;, SARS-S1&lt;;sub&gt;;B&lt;;/sub&gt;; and SARS2-S1&lt;;sub&gt;;B&lt;;/sub&gt;; was measured using biolayer interferometry at 25[[#xB0]]C, as described previously&lt;;sup&gt;;21&lt;;/sup&gt;;. Kinetic binding assay was performed by loading 47D11 mAb at optimal concentration (42 nM) on anti-human Fc biosensor for 10 mins. Antigen association step was performed by incubating the sensor with a range of concentrations of the recombinant spike ectodomain (1600-800-400-200-100-50-25 nM) for 10 min, followed by a dissociation step in PBS for 60 min. The kinetics constants were calculated using 1:1 Langmuir binding model on Fortebio Data Analysis 7.0 software.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Suppl. Fig.3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F5.medium.gif" width="416" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Suppl. Fig.3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F5.medium.gif" width="416" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Suppl. Fig.3." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Suppl. Fig.3.</span>
      <span class="caption-title">Binding kinetics of 47D11 to the S ectodomain and S1
       <sub>B</sub> of SARS-CoV and SARS-CoV-2.
      </span>
      <p id="p-25" class="first-child">Binding kinetics of 47D11 to immobilized recombinant SARS-S
       <sub>ecto</sub>, SARS2-S
       <sub>ecto</sub>, SARS-S1
       <sub>B</sub> and SARS2-S1
       <sub>B</sub> was measured using biolayer interferometry at 25°C, as described previously
       <sup>21</sup>. Kinetic binding assay was performed by loading 47D11 mAb at optimal concentration (42 nM) on anti-human Fc biosensor for 10 mins. Antigen association step was performed by incubating the sensor with a range of concentrations of the recombinant spike ectodomain (1600-800-400-200-100-50-25 nM) for 10 min, followed by a dissociation step in PBS for 60 min. The kinetics constants were calculated using 1:1 Langmuir binding model on Fortebio Data Analysis 7.0 software.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F6" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F6/graphic-6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="47D11 does not prevent binding of SARS-S1B and SARS2-S1B to ACE2-expressing cells. Human HEK-293T cells expressing human ACE2-GFP proteins (see Methods) were detached and fixed with 2% PFA, incubated with a fixed amount of human Fc-tagged S1B domain of SARS-S or SARS2-S that was preincubated for 1h with mAb (mAbs 47D11, 35F4, 43C6, 7.7G6, in H2L2 format) at the indicated mAb:S1B molar ratios, and analysed by flow cytometry using a Alexa Flour 594-conjugated secondary antibody targeting the human Fc tag. Cells are analysed for GFP expression (x-axis, GFP signal) and antibody binding (y-axis, Alexa 594 signal). Percentages of cells that scored negative, single positive, or double positive are shown in each quadrant. Binding controls include PBS-treated cells (mock), treatment of cells with SARS-S1B and SARS2-S1B in the absence of antibody, and cells treated with antibodies only. The experiment was performed twice, data from a representative experiment are shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1705609420" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;47D11 does not prevent binding of SARS-S1&lt;;sub&gt;;B&lt;;/sub&gt;; and SARS2-S1&lt;;sub&gt;;B&lt;;/sub&gt;; to ACE2-expressing cells.&lt;;/span&gt;; Human HEK-293T cells expressing human ACE2-GFP proteins (see Methods) were detached and fixed with 2% PFA, incubated with a fixed amount of human Fc-tagged S1&lt;;sub&gt;;B&lt;;/sub&gt;; domain of SARS-S or SARS2-S that was preincubated for 1h with mAb (mAbs 47D11, 35F4, 43C6, 7.7G6, in H2L2 format) at the indicated mAb:S1&lt;;sub&gt;;B&lt;;/sub&gt;; molar ratios, and analysed by flow cytometry using a Alexa Flour 594-conjugated secondary antibody targeting the human Fc tag. Cells are analysed for GFP expression (x-axis, GFP signal) and antibody binding (y-axis, Alexa 594 signal). Percentages of cells that scored negative, single positive, or double positive are shown in each quadrant. Binding controls include PBS-treated cells (mock), treatment of cells with SARS-S1&lt;;sub&gt;;B&lt;;/sub&gt;; and SARS2-S1&lt;;sub&gt;;B&lt;;/sub&gt;; in the absence of antibody, and cells treated with antibodies only. The experiment was performed twice, data from a representative experiment are shown.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F6/graphic-6.medium.gif" width="325" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F6/graphic-6.medium.gif" width="325" height="440"/>
          </noscript>
         </span>
        </a>
        <ul class="highwire-figure-links inline">
         <li class="download-fig first">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F6/graphic-6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a>
         </li>
         <li class="new-tab last">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F6/graphic-6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
         </li>
        </ul>
       </div>
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F6/graphic-7.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="47D11 does not prevent binding of SARS-S1B and SARS2-S1B to ACE2-expressing cells. Human HEK-293T cells expressing human ACE2-GFP proteins (see Methods) were detached and fixed with 2% PFA, incubated with a fixed amount of human Fc-tagged S1B domain of SARS-S or SARS2-S that was preincubated for 1h with mAb (mAbs 47D11, 35F4, 43C6, 7.7G6, in H2L2 format) at the indicated mAb:S1B molar ratios, and analysed by flow cytometry using a Alexa Flour 594-conjugated secondary antibody targeting the human Fc tag. Cells are analysed for GFP expression (x-axis, GFP signal) and antibody binding (y-axis, Alexa 594 signal). Percentages of cells that scored negative, single positive, or double positive are shown in each quadrant. Binding controls include PBS-treated cells (mock), treatment of cells with SARS-S1B and SARS2-S1B in the absence of antibody, and cells treated with antibodies only. The experiment was performed twice, data from a representative experiment are shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1705609420" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;47D11 does not prevent binding of SARS-S1&lt;;sub&gt;;B&lt;;/sub&gt;; and SARS2-S1&lt;;sub&gt;;B&lt;;/sub&gt;; to ACE2-expressing cells.&lt;;/span&gt;; Human HEK-293T cells expressing human ACE2-GFP proteins (see Methods) were detached and fixed with 2% PFA, incubated with a fixed amount of human Fc-tagged S1&lt;;sub&gt;;B&lt;;/sub&gt;; domain of SARS-S or SARS2-S that was preincubated for 1h with mAb (mAbs 47D11, 35F4, 43C6, 7.7G6, in H2L2 format) at the indicated mAb:S1&lt;;sub&gt;;B&lt;;/sub&gt;; molar ratios, and analysed by flow cytometry using a Alexa Flour 594-conjugated secondary antibody targeting the human Fc tag. Cells are analysed for GFP expression (x-axis, GFP signal) and antibody binding (y-axis, Alexa 594 signal). Percentages of cells that scored negative, single positive, or double positive are shown in each quadrant. Binding controls include PBS-treated cells (mock), treatment of cells with SARS-S1&lt;;sub&gt;;B&lt;;/sub&gt;; and SARS2-S1&lt;;sub&gt;;B&lt;;/sub&gt;; in the absence of antibody, and cells treated with antibodies only. The experiment was performed twice, data from a representative experiment are shown.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F6/graphic-7.medium.gif" width="440" height="353"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F6/graphic-7.medium.gif" width="440" height="353"/>
          </noscript>
         </span>
        </a>
        <ul class="highwire-figure-links inline">
         <li class="download-fig first">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F6/graphic-7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a>
         </li>
         <li class="new-tab last">
          <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F6/graphic-7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
         </li>
        </ul>
       </div>
      </div>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Suppl. Fig.4.</span>
      <span class="caption-title">47D11 does not prevent binding of SARS-S1
       <sub>B</sub> and SARS2-S1
       <sub>B</sub> to ACE2-expressing cells.
      </span>
      <p id="p-26" class="first-child">Human HEK-293T cells expressing human ACE2-GFP proteins (see Methods) were detached and fixed with 2% PFA, incubated with a fixed amount of human Fc-tagged S1
       <sub>B</sub> domain of SARS-S or SARS2-S that was preincubated for 1h with mAb (mAbs 47D11, 35F4, 43C6, 7.7G6, in H2L2 format) at the indicated mAb:S1
       <sub>B</sub> molar ratios, and analysed by flow cytometry using a Alexa Flour 594-conjugated secondary antibody targeting the human Fc tag. Cells are analysed for GFP expression (x-axis, GFP signal) and antibody binding (y-axis, Alexa 594 signal). Percentages of cells that scored negative, single positive, or double positive are shown in each quadrant. Binding controls include PBS-treated cells (mock), treatment of cells with SARS-S1
       <sub>B</sub> and SARS2-S1
       <sub>B</sub> in the absence of antibody, and cells treated with antibodies only. The experiment was performed twice, data from a representative experiment are shown.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F7" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F7.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="ELISA-based receptor binding inhibition assay. The ELISA-based receptor binding inhibition assay was performed as described previously with some adaptions1. Recombinant soluble human ACE2 was coated on NUNC Maxisorp plates (Thermo Scientific) at 4[[#xB0]]C overnight. Plates were washed three times with PBS containing 0.05% Tween-20 and blocked with 3% BSA in PBS containing 0.1% Tween-20 at room temperature for 2 hours. Recombinant Secto and S1B of SARS-S or SARS2-S (300 ng) and serially diluted mAbs (mAbs 47D11, 35F4, 43C6, 7.7G6, in H2L2 format) were mixed for 1 hour at RT, added to the plate for 1 t room temperature, after which the plates were washed three times. Binding to ACE2 was detected using HRP-conjugated StrepMAb (IBA) that recognizes the C-terminal Streptag on the Secto and S1B proteins." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1705609420" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;ELISA-based receptor binding inhibition assay.&lt;;/span&gt;; The ELISA-based receptor binding inhibition assay was performed as described previously with some adaptions&lt;;sup&gt;;1&lt;;/sup&gt;;. Recombinant soluble human ACE2 was coated on NUNC Maxisorp plates (Thermo Scientific) at 4[[#xB0]]C overnight. Plates were washed three times with PBS containing 0.05% Tween-20 and blocked with 3% BSA in PBS containing 0.1% Tween-20 at room temperature for 2 hours. Recombinant S&lt;;sub&gt;;ecto&lt;;/sub&gt;; and S1&lt;;sub&gt;;B&lt;;/sub&gt;; of SARS-S or SARS2-S (300 ng) and serially diluted mAbs (mAbs 47D11, 35F4, 43C6, 7.7G6, in H2L2 format) were mixed for 1 hour at RT, added to the plate for 1 t room temperature, after which the plates were washed three times. Binding to ACE2 was detected using HRP-conjugated StrepMAb (IBA) that recognizes the C-terminal Streptag on the S&lt;;sub&gt;;ecto&lt;;/sub&gt;; and S1&lt;;sub&gt;;B&lt;;/sub&gt;; proteins.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Suppl. Fig.5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F7.medium.gif" width="411" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Suppl. Fig.5." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F7.medium.gif" width="411" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Suppl. Fig.5." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Suppl. Fig.5.</span>
      <span class="caption-title">ELISA-based receptor binding inhibition assay.</span>
      <p id="p-27" class="first-child">The ELISA-based receptor binding inhibition assay was performed as described previously with some adaptions
       <sup>
        <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">1</a>
       </sup>. Recombinant soluble human ACE2 was coated on NUNC Maxisorp plates (Thermo Scientific) at 4°C overnight. Plates were washed three times with PBS containing 0.05% Tween-20 and blocked with 3% BSA in PBS containing 0.1% Tween-20 at room temperature for 2 hours. Recombinant S
       <sub>ecto</sub> and S1
       <sub>B</sub> of SARS-S or SARS2-S (300 ng) and serially diluted mAbs (mAbs 47D11, 35F4, 43C6, 7.7G6, in H2L2 format) were mixed for 1 hour at RT, added to the plate for 1 t room temperature, after which the plates were washed three times. Binding to ACE2 was detected using HRP-conjugated StrepMAb (IBA) that recognizes the C-terminal Streptag on the S
       <sub>ecto</sub> and S1
       <sub>B</sub> proteins.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F8" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F8.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="H2L2 monoclonal antibodies 35F4 and 43C6 neutralize SARS-CoV but not SARS-CoV-2. Antibody-mediated neutralization of infection of VSV particles pseudotyped with spike proteins of SARS-CoV (upper panel) and SARS-CoV-2 (lower panel) by the 35F4 and 43C6 H2L2 antibodies targeting SARS-S1 but not SARS2-S1 (see Suppl.Fig.1). An irrelevant antibody was taken along as a human IgG1 isotype control. Means [[#xB1]] SD of triplicates are shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1705609420" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;H2L2 monoclonal antibodies 35F4 and 43C6 neutralize SARS-CoV but not SARS-CoV-2. Antibody-mediated neutralization of infection of VSV particles pseudotyped with spike proteins of SARS-CoV (upper panel) and SARS-CoV-2 (lower panel) by the 35F4 and 43C6 H2L2 antibodies targeting SARS-S1 but not SARS2-S1 (see Suppl.Fig.1). An irrelevant antibody was taken along as a human IgG1 isotype control. Means [[#xB1]] SD of triplicates are shown.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Suppl. Fig.6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F8.medium.gif" width="300" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Suppl. Fig.6." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F8.medium.gif" width="300" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Suppl. Fig.6." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Suppl. Fig.6.</span>
      <p id="p-28" class="first-child">H2L2 monoclonal antibodies 35F4 and 43C6 neutralize SARS-CoV but not SARS-CoV-2. Antibody-mediated neutralization of infection of VSV particles pseudotyped with spike proteins of SARS-CoV (upper panel) and SARS-CoV-2 (lower panel) by the 35F4 and 43C6 H2L2 antibodies targeting SARS-S1 but not SARS2-S1 (see 
       <a id="xref-fig-3-3" class="xref-fig" href="#F3">Suppl.Fig.1</a>). An irrelevant antibody was taken along as a human IgG1 isotype control. Means ± SD of triplicates are shown.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F9" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F9.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Cell-cell fusion inhibition assay. The cell-cell-fusion inbition assay was performed as described previously with some adaptations1. VeroE6 cells were seeded with density of 105 cells per ml. After reaching 70~80% confluency, cells were transfected with plasmids encoding full length SARS-S, SARS2-S and MERS-S [[#x2013]] C-terminally fused to GFP - using Lipofectamine 2000 (Invitrogen). The furin recognition site in the SARS2-S was mutated (R682RAR to A682AAR) to inhibit cleavage of the protein by endogenous furin and allow trypsin-induced syncytia formation. Two days post transfection, cells were pretreated DMEM only or DMEM with 20 [[#x3BC]]g/ml mAbs for 1 h and subsequently treated with DMEM with 15 [[#x3BC]]g/ml trypsin (to activate the spike fusion function) in the absence or presence of 20 [[#x3BC]]g/ml mAbs (47D11 crossreactive SARS-S and SARS2-S, 35F4 reactive to SARS-S, 7.7G6 reactive to MERS-S). After incubation at 37[[#xB0]]C for 2 hrs, the cells were fixed with 2% PFA in PBS for 20 min at room temperature and stained for nuclei with 4,6-diamidino-2-phenylindole (DAPI). Cells expressing the S-GFP proteins were detected by fluorescence microscopy and S-mediated cell-cell fusion was observed by the formation of fluorescent) multi-nucleated syncytia. The fluorescence images were recorded using a Leica SpeII confocal microscope. The experiment was performed twice, data from a representative experiment are shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1705609420" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Cell-cell fusion inhibition assay.&lt;;/span&gt;; The cell-cell-fusion inbition assay was performed as described previously with some adaptations&lt;;sup&gt;;1&lt;;/sup&gt;;. VeroE6 cells were seeded with density of 10&lt;;sup&gt;;5&lt;;/sup&gt;; cells per ml. After reaching 70~80% confluency, cells were transfected with plasmids encoding full length SARS-S, SARS2-S and MERS-S [[#x2013]] C-terminally fused to GFP - using Lipofectamine 2000 (Invitrogen). The furin recognition site in the SARS2-S was mutated (R&lt;;sup&gt;;682&lt;;/sup&gt;;RAR to A&lt;;sup&gt;;682&lt;;/sup&gt;;AAR) to inhibit cleavage of the protein by endogenous furin and allow trypsin-induced syncytia formation. Two days post transfection, cells were pretreated DMEM only or DMEM with 20 [[#x3BC]]g/ml mAbs for 1 h and subsequently treated with DMEM with 15 [[#x3BC]]g/ml trypsin (to activate the spike fusion function) in the absence or presence of 20 [[#x3BC]]g/ml mAbs (47D11 crossreactive SARS-S and SARS2-S, 35F4 reactive to SARS-S, 7.7G6 reactive to MERS-S). After incubation at 37[[#xB0]]C for 2 hrs, the cells were fixed with 2% PFA in PBS for 20 min at room temperature and stained for nuclei with 4,6-diamidino-2-phenylindole (DAPI). Cells expressing the S-GFP proteins were detected by fluorescence microscopy and S-mediated cell-cell fusion was observed by the formation of fluorescent) multi-nucleated syncytia. The fluorescence images were recorded using a Leica SpeII confocal microscope. The experiment was performed twice, data from a representative experiment are shown.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Suppl. Fig.7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F9.medium.gif" width="392" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Suppl. Fig.7." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F9.medium.gif" width="392" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Suppl. Fig.7." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Suppl. Fig.7.</span>
      <span class="caption-title">Cell-cell fusion inhibition assay.</span>
      <p id="p-29" class="first-child">The cell-cell-fusion inbition assay was performed as described previously with some adaptations
       <sup>
        <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">1</a>
       </sup>. VeroE6 cells were seeded with density of 10
       <sup>5</sup> cells per ml. After reaching 70~80% confluency, cells were transfected with plasmids encoding full length SARS-S, SARS2-S and MERS-S – C-terminally fused to GFP - using Lipofectamine 2000 (Invitrogen). The furin recognition site in the SARS2-S was mutated (R
       <sup>682</sup>RAR to A
       <sup>682</sup>AAR) to inhibit cleavage of the protein by endogenous furin and allow trypsin-induced syncytia formation. Two days post transfection, cells were pretreated DMEM only or DMEM with 20 μg/ml mAbs for 1 h and subsequently treated with DMEM with 15 μg/ml trypsin (to activate the spike fusion function) in the absence or presence of 20 μg/ml mAbs (47D11 crossreactive SARS-S and SARS2-S, 35F4 reactive to SARS-S, 7.7G6 reactive to MERS-S). After incubation at 37°C for 2 hrs, the cells were fixed with 2% PFA in PBS for 20 min at room temperature and stained for nuclei with 4,6-diamidino-2-phenylindole (DAPI). Cells expressing the S-GFP proteins were detected by fluorescence microscopy and S-mediated cell-cell fusion was observed by the formation of fluorescent) multi-nucleated syncytia. The fluorescence images were recorded using a Leica SpeII confocal microscope. The experiment was performed twice, data from a representative experiment are shown.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F1" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="47D11 neutralizes SARS-CoV and SARS-CoV-2. a) Binding of 47D11 to HEK-293T cells expressing GFP-tagged spike proteins of SARS-CoV and SARS-CoV-2 detected by immunofluorescence assay. The human mAb 7.7G6 targeting the MERS-CoV S1B spike domain was taken along as a negative control, cell nuclei in the overlay images are visualized with DAPI. b) Antibody-mediated neutralization of infection of luciferase-encoding VSV particles pseudotyped with spike proteins of SARS-CoV and SARS-CoV-2. Pseudotyped VSV particles pre-incubated with antibodies at indicated concentrations (see methods) were used to infect VeroE6 cells and luciferase activities in cell lysates were determined at 24 h post transduction to calculate infection (%) relative to non-antibody-treated controls. The average [[#xB1]] SD from at least two independent experiments performed is shown. Iso-CTRL: irrelevant isotype monoclonal antibody. c) Antibody-mediated neutralization of SARS-CoV and SARS-CoV-2 infection on VeroE6 cells. The experiment was performed with triplicate samples, the average [[#xB1]] SD is shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1705609420" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;47D11 neutralizes SARS-CoV and SARS-CoV-2.&lt;;/span&gt;; a) Binding of 47D11 to HEK-293T cells expressing GFP-tagged spike proteins of SARS-CoV and SARS-CoV-2 detected by immunofluorescence assay. The human mAb 7.7G6 targeting the MERS-CoV S1&lt;;sub&gt;;B&lt;;/sub&gt;; spike domain was taken along as a negative control, cell nuclei in the overlay images are visualized with DAPI. b) Antibody-mediated neutralization of infection of luciferase-encoding VSV particles pseudotyped with spike proteins of SARS-CoV and SARS-CoV-2. Pseudotyped VSV particles pre-incubated with antibodies at indicated concentrations (see methods) were used to infect VeroE6 cells and luciferase activities in cell lysates were determined at 24 h post transduction to calculate infection (%) relative to non-antibody-treated controls. The average [[#xB1]] SD from at least two independent experiments performed is shown. Iso-CTRL: irrelevant isotype monoclonal antibody. c) Antibody-mediated neutralization of SARS-CoV and SARS-CoV-2 infection on VeroE6 cells. The experiment was performed with triplicate samples, the average [[#xB1]] SD is shown.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig.1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F1.medium.gif" width="270" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig.1" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F1.medium.gif" width="270" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig.1" data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig.1</span>
      <span class="caption-title">47D11 neutralizes SARS-CoV and SARS-CoV-2.</span>
      <p id="p-9" class="first-child">a) Binding of 47D11 to HEK-293T cells expressing GFP-tagged spike proteins of SARS-CoV and SARS-CoV-2 detected by immunofluorescence assay. The human mAb 7.7G6 targeting the MERS-CoV S1
       <sub>B</sub> spike domain was taken along as a negative control, cell nuclei in the overlay images are visualized with DAPI. b) Antibody-mediated neutralization of infection of luciferase-encoding VSV particles pseudotyped with spike proteins of SARS-CoV and SARS-CoV-2. Pseudotyped VSV particles pre-incubated with antibodies at indicated concentrations (see methods) were used to infect VeroE6 cells and luciferase activities in cell lysates were determined at 24 h post transduction to calculate infection (%) relative to non-antibody-treated controls. The average ± SD from at least two independent experiments performed is shown. Iso-CTRL: irrelevant isotype monoclonal antibody. c) Antibody-mediated neutralization of SARS-CoV and SARS-CoV-2 infection on VeroE6 cells. The experiment was performed with triplicate samples, the average ± SD is shown.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F2" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="The neutralizing 47D11 monoclonal antibody binds the receptor binding domain of SARS-CoV and SARS-CoV-2 spike proteins without eliminating S1B/ACE2 receptor interaction. a) ELISA binding curves of 47D11 to Secto (upper panel) or S1A and S1B (RBD) (lower panel) of SARS-S and SARS2-S coated at equimolar concentrations. The average [[#xB1]] SD from at least two independent experiments performed is shown. b) Interference of antibodies with binding of the S-S1B of SARS-CoV and SARS-CoV-2 to cell surface ACE2-GFP analysed by flow cytometry. Prior to cell binding, S1B was mixed with mAb (mAbs 47D11, 35F4, 43C6, 7.7G6, in H2L2 format) with indicated specificity in a mAb:S1B molar ratio of 8:1 (see Suppl.Fig.4 for an extensive analysis using different mAb:S1B molar ratio[[#x2019]]s). Cells are analysed for (ACE2-)GFP expression (x-axis) and S1B binding (y-axis). Percentages of cells that scored negative, single positive, or double positive are shown in each quadrant. c) Divergence in surface residues in S1B of SARS-CoV and SARS-CoV-2. Upper panel: Structure of the SARS-CoV spike protein S1B RBD in complex with human ACE2 receptor (PDB: 2AJF)18. ACE2 (wheat color) is visualized in ribbon presentation. The S1B core domain (blue) and subdomain (orange) are displayed in surface presentation using PyMOL, and are visualized with the same colors in the linear diagram of the spike protein above, with positions of the S1 and S2 subunits, the S ectodomain (Secto), the S1 domains S1A-D and the transmembrane domain (TM) indicated. Lower panel: Similar as panel above with surface residues on S1B of SARS-CoV that are at variance with SARS-CoV-2 colorored in white." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1705609420" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;The neutralizing 47D11 monoclonal antibody binds the receptor binding domain of SARS-CoV and SARS-CoV-2 spike proteins without eliminating S1&lt;;sub&gt;;B&lt;;/sub&gt;;/ACE2 receptor interaction.&lt;;/span&gt;; a) ELISA binding curves of 47D11 to S&lt;;sub&gt;;ecto&lt;;/sub&gt;; (upper panel) or S1&lt;;sub&gt;;A&lt;;/sub&gt;; and S1&lt;;sub&gt;;B&lt;;/sub&gt;; (RBD) (lower panel) of SARS-S and SARS2-S coated at equimolar concentrations. The average [[#xB1]] SD from at least two independent experiments performed is shown. b) Interference of antibodies with binding of the S-S1&lt;;sub&gt;;B&lt;;/sub&gt;; of SARS-CoV and SARS-CoV-2 to cell surface ACE2-GFP analysed by flow cytometry. Prior to cell binding, S1&lt;;sub&gt;;B&lt;;/sub&gt;; was mixed with mAb (mAbs 47D11, 35F4, 43C6, 7.7G6, in H2L2 format) with indicated specificity in a mAb:S1&lt;;sub&gt;;B&lt;;/sub&gt;; molar ratio of 8:1 (see Suppl.Fig.4 for an extensive analysis using different mAb:S1&lt;;sub&gt;;B&lt;;/sub&gt;; molar ratio[[#x2019]]s). Cells are analysed for (ACE2-)GFP expression (x-axis) and S1&lt;;sub&gt;;B&lt;;/sub&gt;; binding (y-axis). Percentages of cells that scored negative, single positive, or double positive are shown in each quadrant. c) Divergence in surface residues in S1&lt;;sub&gt;;B&lt;;/sub&gt;; of SARS-CoV and SARS-CoV-2. Upper panel: Structure of the SARS-CoV spike protein S1&lt;;sub&gt;;B&lt;;/sub&gt;; RBD in complex with human ACE2 receptor (PDB: 2AJF)&lt;;sup&gt;;18&lt;;/sup&gt;;. ACE2 (wheat color) is visualized in ribbon presentation. The S1&lt;;sub&gt;;B&lt;;/sub&gt;; core domain (blue) and subdomain (orange) are displayed in surface presentation using PyMOL, and are visualized with the same colors in the linear diagram of the spike protein above, with positions of the S1 and S2 subunits, the S ectodomain (S&lt;;sub&gt;;ecto&lt;;/sub&gt;;), the S1 domains S1&lt;;sub&gt;;A-D&lt;;/sub&gt;; and the transmembrane domain (TM) indicated. Lower panel: Similar as panel above with surface residues on S1&lt;;sub&gt;;B&lt;;/sub&gt;; of SARS-CoV that are at variance with SARS-CoV-2 colorored in white.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Fig.2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F2.medium.gif" width="415" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Fig.2" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F2.medium.gif" width="415" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig.2" data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Fig.2</span>
      <span class="caption-title">The neutralizing 47D11 monoclonal antibody binds the receptor binding domain of SARS-CoV and SARS-CoV-2 spike proteins without eliminating S1
       <sub>B</sub>/ACE2 receptor interaction.
      </span>
      <p id="p-10" class="first-child">a) ELISA binding curves of 47D11 to S
       <sub>ecto</sub> (upper panel) or S1
       <sub>A</sub> and S1
       <sub>B</sub> (RBD) (lower panel) of SARS-S and SARS2-S coated at equimolar concentrations. The average ± SD from at least two independent experiments performed is shown. b) Interference of antibodies with binding of the S-S1
       <sub>B</sub> of SARS-CoV and SARS-CoV-2 to cell surface ACE2-GFP analysed by flow cytometry. Prior to cell binding, S1
       <sub>B</sub> was mixed with mAb (mAbs 47D11, 35F4, 43C6, 7.7G6, in H2L2 format) with indicated specificity in a mAb:S1
       <sub>B</sub> molar ratio of 8:1 (see 
       <a id="xref-fig-6-2" class="xref-fig" href="#F6">Suppl.Fig.4</a> for an extensive analysis using different mAb:S1
       <sub>B</sub> molar ratio’s). Cells are analysed for (ACE2-)GFP expression (x-axis) and S1
       <sub>B</sub> binding (y-axis). Percentages of cells that scored negative, single positive, or double positive are shown in each quadrant. c) Divergence in surface residues in S1
       <sub>B</sub> of SARS-CoV and SARS-CoV-2. Upper panel: Structure of the SARS-CoV spike protein S1
       <sub>B</sub> RBD in complex with human ACE2 receptor (PDB: 2AJF)
       <sup>
        <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>
       </sup>. ACE2 (wheat color) is visualized in ribbon presentation. The S1
       <sub>B</sub> core domain (blue) and subdomain (orange) are displayed in surface presentation using PyMOL, and are visualized with the same colors in the linear diagram of the spike protein above, with positions of the S1 and S2 subunits, the S ectodomain (S
       <sub>ecto</sub>), the S1 domains S1
       <sub>A-D</sub> and the transmembrane domain (TM) indicated. Lower panel: Similar as panel above with surface residues on S1
       <sub>B</sub> of SARS-CoV that are at variance with SARS-CoV-2 colorored in white.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-11">The SARS2-S1
     <sub>B</sub> receptor binding domain (residues 338-506) consists of a core domain and a receptor binding subdomain (residues 438-498) looping out from the antiparallel betasheet core domain structure that directly engages the receptor. Compared to the S1
     <sub>B</sub> core domain, the protein sequence identity of the S1
     <sub>B</sub> receptor interacting subdomain of SARS-S and SARS2-S is substantially lower (46.7% versus 86.3%; 
     <a id="xref-fig-10-1" class="xref-fig" href="#F10">Suppl.Fig.8</a> and 
     <a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig.2c</a>). Potent neutralizing antibodies often target this receptor binding subdomain. However, due to common variations in this subdomain, these antibodies are often virus-specific and bind and neutralize related viruses poorly
     <sup>
      <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>, 
      <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>
     </sup>. The cross-reactive nature of 47D11 indicates that the antibody is more likely to target the conserved core structure of the S1
     <sub>B</sub> receptor binding domain. S1
     <sub>B</sub> binding by 47D11 further away from the receptor binding interface explains its inability to compromise spike-receptor interaction.
    </p>
    <div id="F10" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F10.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Protein sequence alignment of the S1B receptor binding in (RBD) of the SARS-CoV and SARS-CoV-2 spike proteins by ClustalW. Numbering denotes the residue position in the full-length spike protein of SARS-CoV (Genbank: AAP13441.1) and SARS-CoV-2 (Genbank: QHD43416.1). Asterisks (*) indicated fully conserved residues, the colon symbol (:) indicates conservation between groups of very similar properties, and the period symbol (.) indicates conservation between groups of weakly similar properties. Sequences corresponding to the S1B receptor binding core domain and the receptor binding subdomain are colored in blue and orange, respectively. The fourteen residues that are involved in binding of SARS-CoV S1B human ACE2 are highlighted in grey1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1705609420" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Protein sequence alignment of the S1&lt;;sub&gt;;B&lt;;/sub&gt;; receptor binding in (RBD) of the SARS-CoV and SARS-CoV-2 spike proteins by ClustalW.&lt;;/span&gt;; Numbering denotes the residue position in the full-length spike protein of SARS-CoV (Genbank: AAP13441.1) and SARS-CoV-2 (Genbank: QHD43416.1). Asterisks (*) indicated fully conserved residues, the colon symbol (:) indicates conservation between groups of very similar properties, and the period symbol (.) indicates conservation between groups of weakly similar properties. Sequences corresponding to the S1&lt;;sub&gt;;B&lt;;/sub&gt;; receptor binding core domain and the receptor binding subdomain are colored in blue and orange, respectively. The fourteen residues that are involved in binding of SARS-CoV S1&lt;;sub&gt;;B&lt;;/sub&gt;; human ACE2 are highlighted in grey&lt;;sup&gt;;1&lt;;/sup&gt;;.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Suppl. Fig.8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F10.medium.gif" width="440" height="195"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Suppl. Fig.8." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F10.medium.gif" width="440" height="195"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Suppl. Fig.8." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.11.987958/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Suppl. Fig.8.</span>
      <span class="caption-title">Protein sequence alignment of the S1
       <sub>B</sub> receptor binding in (RBD) of the SARS-CoV and SARS-CoV-2 spike proteins by ClustalW.
      </span>
      <p id="p-30" class="first-child">Numbering denotes the residue position in the full-length spike protein of SARS-CoV (Genbank: AAP13441.1) and SARS-CoV-2 (Genbank: QHD43416.1). Asterisks (*) indicated fully conserved residues, the colon symbol (:) indicates conservation between groups of very similar properties, and the period symbol (.) indicates conservation between groups of weakly similar properties. Sequences corresponding to the S1
       <sub>B</sub> receptor binding core domain and the receptor binding subdomain are colored in blue and orange, respectively. The fourteen residues that are involved in binding of SARS-CoV S1
       <sub>B</sub> human ACE2 are highlighted in grey
       <sup>
        <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">1</a>
       </sup>.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-12">In conclusion, this is the first report on a (human) monoclonal antibody that neutralizes SARS-CoV-2. 47D11 binds a conserved epitope on the spike receptor binding domain explaining its ability to cross-neutralize SARS-CoV and SARS-CoV-2, using a mechanism that is independent of receptor binding inhibition. This antibody will be useful for development of antigen detection tests and serological assays targeting SARS-CoV-2. Neutralizing antibodies can alter the course of infection in the infected host supporting virus clearance or protect an uninfected host that is exposed to the virus
     <sup>
      <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>
     </sup>. Hence, this antibody offers the potential to prevent and/or treat COVID-19, and possibly also other future emerging diseases in humans caused by viruses from the 
     <em>Sarbecovirus</em> subgenus.
    </p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Author Contributions</h2>
    <p id="p-14">B.J.B. designed and coordinated the study. C.W., W.L., N.M.A.O., R.v.H. and D.D conducted the experiments. D.D., B.L.H. and B.J.B. supervised part of the experiments. All authors contributed to the interpretations and conclusions presented. B.J.B. wrote the manuscript, B.L.H., F.J.M.K., A.D.M.E.O. and F.G. participated in editing the manuscript.</p>
   </div>
   <div class="section" id="sec-3">
    <h2 class="">Materials and Methods</h2>
    <div class="section" id="sec-4">
     <h3>Expression and purification of coronavirus spike proteins</h3>
     <p id="p-15">Coronavirus spike ectodomains (S
      <sub>ecto</sub>) of SARS-CoV (residues 1–1,182; strain CUHK-W1; GenBank: AAP13567.1) and SARS-CoV-2 (residues 1–1,213; strain Wuhan-Hu-1; GenBank: QHD43416.1) were expressed transiently in HEK-293T cells with a C-terminal trimerization motif and Strep-tag using the pCAGGS expression plasmid. Similarly, pCAGGS expression vectors encoding S1 or its subdomains of SARS-CoV (S1, residues 1-676; S1
      <sub>A</sub>, residues 1-302; S1
      <sub>B</sub>, residues, 325-533), and SARS-CoV-2 (S1, residues 1-682; S1
      <sub>A</sub>, residues 1-294; S1
      <sub>B</sub>, residues 329-538) C-terminally tagged with Fc domain of human or mouse IgG or Strep-tag were generated as described before
      <sup>
       <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>
      </sup>. Recombinant proteins were expressed transiently in HEK-293T cells and affinity purified from the culture supernatant by protein-A sepharose beads (GE Healthcare) or streptactin beads (IBA) purification. Purity and integrity of all purified recombinant proteins was checked by coomassie stained SDS-PAGE.
     </p>
    </div>
    <div class="section" id="sec-5">
     <h3>Generation of H2L2 mAbs</h3>
     <p id="p-16">H2L2 mice were sequentially immunized in two weeks intervals with purified S
      <sub>ecto</sub> of different CoVs in the following order: HCoV-OC43, SARS-CoV, MERS-CoV, HCoV-OC43, SARS-CoV and MERS-CoV. Antigens were injected at 20-25 μg/mouse using Stimune Adjuvant (Prionics) freshly prepared according to the manufacturer instruction for first injection, while boosting was done using Ribi (Sigma) adjuvant. Injections were done subcutaneously into the left and right groin each (50 μl) and 100 μl intraperitoneally. Four days after the last injection, spleen and lymph nodes are harvested, and hybridomas made by standard method using SP 2/0 myeloma cell line (ATCC#CRL-1581) as a fusion partner. Hybridomas were screened in antigen-specific ELISA and those selected for further development, subcloned and produced on a small scale (100 ml of medium). For this purpose, hybridomas are cultured in serum- and protein-free medium for hybridoma culturing (PFHM-II (1X), Gibco) with addition of non-essential amino acids 100X NEAA, Biowhittaker Lonza, Cat BE13-114E). H2L2 antibodies were purified from hybridoma culture supernatants using Protein-A affinity chromatography. Purified antibodies were stored at 4°C until use.
     </p>
    </div>
    <div class="section" id="sec-6">
     <h3>Production of human monoclonal antibody 47D11</h3>
     <p id="p-17">For recombinant human mAb production, the cDNA’s encoding the 47D11 H2L2 mAb variable regions of the heavy and light chains were cloned into expression plasmids containing the human IgG1 heavy chain and Ig kappa light chain constant regions, respectively (InvivoGen). Both plasmids contain the interleukin-2 signal sequence to enable efficient secretion of recombinant antibodies. Recombinant human 47D11 mAb and previously described Isotype-control (anti-Streptag mAb) or 7.7G6 mAb were produced in HEK-293T cells following transfection with pairs of the IgG1 heavy and light chain expression plasmids according to protocols from InvivoGen. Human antibodies were purified from cell culture supernatants using Protein-A affinity chromatography. Purified antibodies were stored at 4°C until use.</p>
    </div>
    <div class="section" id="sec-7">
     <h3>Immunofluorescence microscopy</h3>
     <p id="p-18">Antibody binding to cell surface spike proteins of SARS-CoV, SARS-CoV-2 and MERS-CoV was measured by immunofluoresence microscopy. HEK-293T cells seeded on glass slides were transfected with plasmids encoding SARS-S, SARS2-S or MERS-S - C-terminally fused to the green fluorescence protein (GFP) - using Lipofectamine 2000 (Invitrogen). Two days post transfection, cells were fixed by incubation with 2% paraformaldehyde in PBS for 20 min at room temperature and stained for nuclei with 4,6-diamidino-2-phenylindole (DAPI). Cells were subsequently incubated with mAbs at a concentration of 10 μg/ml for 1 h at room temperature, followed by incubation with Alexa Fluor 594 conjugated goat anti-human IgG antibodies (Invitrogen, Thermo Fisher Scientific) for 45 min at room temperature. The fluorescence images were recorded using a Leica SpeII confocal microscope.</p>
    </div>
    <div class="section" id="sec-8">
     <h3>Flow cytometry-based receptor binding inhibition assay</h3>
     <p id="p-19">Antibody interference of S1
      <sub>B</sub> binding to human ACE2 receptor on the cell surface was measured by flow cytometry. HEK-293T cells were seeded at a density of 2.5×10
      <sup>5</sup> cells per ml in a T75 flask. After reaching 70~80% confluency, cells were transfected with an expression plasmid encoding human ACE2 - C-terminally fused to the GFP - using Lipofectamine 2000 (Invitrogen). Two days post transfection, cells were dissociated by cell dissociation solution (Sigma-aldrich, Merck KGaA; cat. no. C5914). 2.5 μg/ml of human Fc tagged SARS-S1
      <sub>B</sub> and SARS2-S1
      <sub>B</sub> was preincubated with mAb at the indicated mAb:S1
      <sub>B</sub> molar ratios for 1 hour on ice and subjected to flow cytometry. Single cell suspensions in FACS buffer were centrifuged at 400×g for 10 min. Cells were subsequently incubated with S1
      <sub>B</sub> and mAb mixture for 1 h on ice, followed by incubation with Alexa Fluor 594 conjugated goat anti-human IgG antibodies (Invitrogen, Thermo Fisher Scientific) for 45 min at room temperature. Cells were subjected to flow cytometric analysis with a CytoFLEX Flow Cytometer (Beckman Coulter). The results were analysed by FlowJo (version 10).
     </p>
    </div>
    <div class="section" id="sec-9">
     <h3>Pseudotyped virus neutralization assay</h3>
     <p id="p-20">Production of VSV pseudotyped with SARS-S and SARS2-S was performed as described previously with some adaptations
      <sup>
       <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>
      </sup>. Briefly, HEK-293T cells were transfected with pCAGGS expression vectors encoding SARS-S or SARS2-S carrying a 28- or 18-a.a. cytoplasmic tail truncation, respectively. One day post transfection, cells were infected with the VSV-G pseudotyped VSVΔG bearing the firefly (
      <em>Photinus pyralis</em>) luciferase reporter gene. Twenty-four hours later, supernatants containing SARS-S/SARS2-S pseudotyped VSV particles were harvested and titrated on African green monkey kidney VeroE6 cells. In the virus neutralization assay, mAbs were serially diluted at two times the desired final concentration in DMEM supplemented with 1% fetal calf serum (Bodinco), 100 U/ml Penicillin and 100 μg/ml Streptomycin. Diluted mAbs were incubated with an equal volume of pseudotyped VSV particles for 1 hour at room temperature, inoculated on confluent VeroE6 monolayers in 96-well plated, and further incubated at 37°C for 24 hours. Luciferase activity was measured on a Berthold Centro LB 960 plate luminometer using D-luciferin as a substrate (Promega). The percentage of infectivity was calculated as ratio of luciferase readout in the presence of mAbs normalized to luciferase readout in the absence of mAb. The half maximal inhibitory concentrations (IC
      <sub>50</sub>) were determined using 4-parameter logistic regression (GraphPad Prism version 8).
     </p>
    </div>
    <div class="section" id="sec-10">
     <h3>Virus neutralization assay</h3>
     <p id="p-21">Neutralization of authentic SARS-CoV and SARS-CoV-2 was performed using a plaque reduction neutralization test (PRNT) as described earlier, with some modifications
      <sup>
       <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>
      </sup>. In brief, mAbs were two-fold serially diluted and mixed with SARS-CoV or SARS-CoV-2 for 1 hour. The mixture was then added to VeroE6 cells and incubated for 1 hr, after which the cells were washed and further incubated in medium for 8 hrs. The cells were then fixed and stained using a rabbit anti-SARS-CoV serum (Sino Biological) and a secondary peroxidase-labelled goat anti-rabbit IgG (Dako). The signal was developed using a precipitate forming TMB substrate (True Blue, KPL) and the number of infected cells per well were counted using the ImmunoSpot® Image analyzer (CTL Europe GmbH). The half maximal inhibitory concentrations (IC
      <sub>50</sub>) were determined using 4-parameter logistic regression (GraphPad Prism version 8).
     </p>
    </div>
    <div class="section" id="sec-11">
     <h3>ELISA analysis of antibody binding to CoV spike antigens</h3>
     <p id="p-22">NUNC Maxisorp plates (Thermo Scientific) were coated with equimolar antigen amounts at 4°C overnight. Plates were washed three times with Phosphate Saline Buffer (PBS) containing 0.05% Tween-20 and blocked with 3% Bovine Serum Albumin (BSA) in PBS containing 0.1% Tween-20 at room temperature for 2 hours. Four-folds serial dilutions of mAbs starting at 10 μg/ml (diluted in blocking buffer) were added and plates were incubated for 1 hour at room temperature. Plates were washed three times and incubated with HRP-conjugated goat anti-human secondary antibody (ITK Southern Biotech) diluted 1:2000 in blocking buffer for 1 hour at room temperature. An HRP-conjugated anti-StrepMAb (IBA, Cat.no: 2-1509-001) antibody was used to corroborate equimolar coating of the Strep-tagged spike antigens. HRP activity was measured at 450 nanometer using tetramethylbenzidine substrate (BioFX) and an ELISA plate reader (EL-808, Biotek). Half-maximum effective concentration (EC
      <sub>50</sub>) binding values were calculated by non-linear regression analysis on the binding curves using GraphPad Prism (version 8).
     </p>
    </div>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgements</h2>
    <p id="p-13">We thank dr. Yoshiharu Matsuura (Osaka University, Japan) for the provision of the luciferase-encoding VSV-G pseudotyped VSVΔG-luc virus, and Yongle Yang, Michael van der Reijden and Rick Janssens for technical support. We thank Christian Drosten (Charité Universitätsmedizin Berlin, Germany) for provision of the SARS-CoV-2 virus. This study was done within the framework of National Centre for One Health (NCOH) and the Innovative Medicines Initiative (IMI) Zoonotic Anticipation and Preparedness Initiative [ZAPI project; grant agreement no. 115760]. The mice used in this study were provided by Harbour Antibodies BV, a daughter company of Harbour Biomed (
     <a href="http://www.harbourbiomed.com">http://www.harbourbiomed.com</a>). C. Wang was supported by a grant from the Chinese Scholarship Council (file number CSC201708620178).
    </p>
   </div>
   <div class="section fn-group" id="fn-group-1">
    <h2>Footnotes</h2>
    <ul>
     <li class="fn-study-group-members" id="fn-1">
      <p id="p-1">
       <a class="rev-xref" href="#xref-fn-1-1">↵</a>
       <span class="fn-label">**</span> These authors are joint first authors.
      </p>
     </li>
     <li class="fn-study-group-members" id="fn-2">
      <p id="p-2">
       <a class="rev-xref" href="#xref-fn-2-1">↵</a>
       <span class="fn-label">$</span> These authors are joint senior authors.
      </p>
     </li>
    </ul>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987958v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>, 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">4</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.03.11.987958v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">World Health Organization</span>. 
         <a href="https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200310-sitrep-50-covid-19.pdf?sfvrsn=55e904fb_2">https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200310-sitrep-50-covid-19.pdf?sfvrsn=55e904fb_2</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987958v1.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">Coronaviridae Study Group of the International Committee on Taxonomy of Viruses</span>. 
         <span class="cit-article-title">The species Severe acute respiratory syndrome-related coronavirus: classifying 2019-nCoV and naming it SARS-CoV-2</span>. 
         <abbr class="cit-jnl-abbrev">Nat. Microbiol.</abbr> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987958v1.4" data-doi="10.1517/14712590902763755">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Prabakaran</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Potent human monoclonal antibodies against SARS CoV, Nipah and Hendra viruses</span>. 
         <abbr class="cit-jnl-abbrev">Expert opinion on biological therapy</abbr>
         <span class="cit-vol">9</span>, 
         <span class="cit-fpage">355</span>–
         <span class="cit-lpage">368</span> (
         <span class="cit-pub-date">2009</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DExpert%2Bopinion%2Bon%2Bbiological%2Btherapy%26rft.stitle%253DExpert%2BOpin%2BBiol%2BTher%26rft.aulast%253DPrabakaran%26rft.auinit1%253DP.%26rft.volume%253D9%26rft.issue%253D3%26rft.spage%253D355%26rft.epage%253D368%26rft.atitle%253DPotent%2Bhuman%2Bmonoclonal%2Bantibodies%2Bagainst%2BSARS%2BCoV%252C%2BNipah%2Band%2BHendra%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1517%252F14712590902763755%26rft_id%253Dinfo%253Apmid%252F19216624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1517/14712590902763755&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19216624&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.11.987958.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987958v1.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Saphire</span>, 
          <span class="cit-name-given-names">E. O.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schendel</span>, 
          <span class="cit-name-given-names">S. L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gunn</span>, 
          <span class="cit-name-given-names">B. M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Milligan</span>, 
          <span class="cit-name-given-names">J. C.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Alter</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>
         <span class="cit-article-title">Antibody-mediated protection against Ebola virus</span>. 
         <abbr class="cit-jnl-abbrev">Nat. Immunol.</abbr>
         <span class="cit-vol">19</span>, 
         <span class="cit-fpage">1169</span>–
         <span class="cit-lpage">1178</span> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat.%2BImmunol.%26rft.volume%253D19%26rft.spage%253D1169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987958v1.6" data-doi="10.1371/journal.ppat.1002859">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Reguera</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Structural bases of coronavirus attachment to host aminopeptidase N and its inhibition by neutralizing antibodies</span>. 
         <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr>
         <span class="cit-vol">8</span>, 
         <span class="cit-fpage">e1002859</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DReguera%26rft.auinit1%253DJ.%26rft.volume%253D8%26rft.issue%253D8%26rft.spage%253De1002859%26rft.epage%253De1002859%26rft.atitle%253DStructural%2Bbases%2Bof%2Bcoronavirus%2Battachment%2Bto%2Bhost%2Baminopeptidase%2BN%2Band%2Bits%2Binhibition%2Bby%2Bneutralizing%2Bantibodies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002859%26rft_id%253Dinfo%253Apmid%252F22876187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002859&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22876187&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.11.987958.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.11.987958v1.7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yu</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Structural basis for the neutralization of MERS-CoV by a human monoclonal antibody MERS-27</span>. 
         <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr>
         <span class="cit-vol">5</span>, 
         <span class="cit-fpage">13133</span> (
         <span class="cit-pub-date">2015</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D5%26rft.spage%253D13133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.11.987958v1.8" data-doi="10.1074/jbc.M600697200">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Prabakaran</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Structure of severe acute respiratory syndrome coronavirus receptor-binding domain complexed with neutralizing antibody</span>. 
         <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr>
         <span class="cit-vol">281</span>, 
         <span class="cit-fpage">15829</span>–
         <span class="cit-lpage">15836</span> (
         <span class="cit-pub-date">2006</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M600697200%26rft_id%253Dinfo%253Apmid%252F16597622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODEvMjMvMTU4MjkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8xMi8yMDIwLjAzLjExLjk4Nzk1OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.11.987958v1.9" data-doi="10.1074/jbc.M603275200">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hwang</span>, 
          <span class="cit-name-given-names">W. C.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Structural basis of neutralization by a human anti-severe acute respiratory syndrome spike protein antibody, 80R</span>. 
         <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr>
         <span class="cit-vol">281</span>, 
         <span class="cit-fpage">34610</span>–
         <span class="cit-lpage">34616</span> (
         <span class="cit-pub-date">2006</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M603275200%26rft_id%253Dinfo%253Apmid%252F16954221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODEvNDUvMzQ2MTAiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8xMi8yMDIwLjAzLjExLjk4Nzk1OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.11.987958v1.10" data-doi="10.1128/JVI.02377-07">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rockx</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Structural basis for potent cross-neutralizing human monoclonal antibody protection against lethal human and zoonotic severe acute respiratory syndrome coronavirus challenge</span>. 
         <abbr class="cit-jnl-abbrev">J. Virol.</abbr>
         <span class="cit-vol">82</span>, 
         <span class="cit-fpage">3220</span>–
         <span class="cit-lpage">3235</span> (
         <span class="cit-pub-date">2008</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02377-07%26rft_id%253Dinfo%253Apmid%252F18199635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgyLzcvMzIyMCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzEyLzIwMjAuMDMuMTEuOTg3OTU4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987958v1.11" data-doi="10.1080/22221751.2019.1597644">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Widjaja</span>, 
          <span class="cit-name-given-names">I.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Towards a solution to MERS: protective human monoclonal antibodies targeting different domains and functions of the MERS-coronavirus spike glycoprotein</span>. 
         <abbr class="cit-jnl-abbrev">Emerg. Microbes Infect.</abbr>
         <span class="cit-vol">8</span>, 
         <span class="cit-fpage">516</span>–
         <span class="cit-lpage">530</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEmerg.%2BMicrobes%2BInfect.%26rft.volume%253D8%26rft.spage%253D516%26rft_id%253Dinfo%253Adoi%252F10.1080%252F22221751.2019.1597644%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1080/22221751.2019.1597644&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987958v1.12">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wrapp</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987958v1.13">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Walls</span>, 
          <span class="cit-name-given-names">A. C.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Structure, Function, and Antigenicity of the SARS-CoV-2 Spike Glycoprotein</span>. 
         <abbr class="cit-jnl-abbrev">Cell</abbr> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987958v1.14" data-doi="10.1038/nature02145">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">426</span>, 
         <span class="cit-fpage">450</span>–
         <span class="cit-lpage">454</span> (
         <span class="cit-pub-date">2003</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLi%26rft.auinit1%253DW.%26rft.volume%253D426%26rft.issue%253D6965%26rft.spage%253D450%26rft.epage%253D454%26rft.atitle%253DAngiotensin-converting%2Benzyme%2B2%2Bis%2Ba%2Bfunctional%2Breceptor%2Bfor%2Bthe%2BSARS%2Bcoronavirus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02145%26rft_id%253Dinfo%253Apmid%252F14647384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature02145&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=14647384&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.11.987958.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987958v1.15" data-doi="10.1016/j.cell.2018.12.028">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Walls</span>, 
          <span class="cit-name-given-names">A. C.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Unexpected receptor functional mimicry elucidates activation of coronavirus fusion</span>. 
         <abbr class="cit-jnl-abbrev">Cell</abbr>
         <span class="cit-vol">176</span>, 
         <span class="cit-fpage">1026</span>–
         <span class="cit-lpage">1039. e15</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D176%26rft.spage%253D1026%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2018.12.028%26rft_id%253Dinfo%253Apmid%252F30712865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.cell.2018.12.028&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=30712865&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.11.987958.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987958v1.16">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Tian</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody</span>. 
         <abbr class="cit-jnl-abbrev">Emerging Microbes &amp;; Infections</abbr>
         <span class="cit-vol">9</span>, 
         <span class="cit-fpage">382</span>–
         <span class="cit-lpage">385</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEmerging%2BMicrobes%2B%2526%2BInfections%26rft.volume%253D9%26rft.spage%253D382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">17.</span>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987958v1.17" data-doi="10.1038/nm.3985">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Menachery</span>, 
          <span class="cit-name-given-names">V. D.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A SARS-like cluster of circulating bat coronaviruses shows potential for human emergence</span>. 
         <abbr class="cit-jnl-abbrev">Nat. Med.</abbr>
         <span class="cit-vol">21</span>, 
         <span class="cit-fpage">1508</span> (
         <span class="cit-pub-date">2015</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat.%2BMed.%26rft.volume%253D21%26rft.spage%253D1508%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm.3985%26rft_id%253Dinfo%253Apmid%252F26552008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nm.3985&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26552008&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.11.987958.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">18.</span>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987958v1.18" data-doi="10.1126/science.1116480">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Farzan</span>, 
          <span class="cit-name-given-names">M.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Harrison</span>, 
          <span class="cit-name-given-names">S. C.</span>
         </span>
         <span class="cit-article-title">Structure of SARS coronavirus spike receptor-binding domain complexed with receptor</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>
         <span class="cit-vol">309</span>, 
         <span class="cit-fpage">1864</span>–
         <span class="cit-lpage">1868</span> (
         <span class="cit-pub-date">2005</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLi%26rft.auinit1%253DF.%26rft.volume%253D309%26rft.issue%253D5742%26rft.spage%253D1864%26rft.epage%253D1868%26rft.atitle%253DStructure%2Bof%2BSARS%2BCoronavirus%2BSpike%2BReceptor-Binding%2BDomain%2BComplexed%2Bwith%2BReceptor%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1116480%26rft_id%253Dinfo%253Apmid%252F16166518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDkvNTc0Mi8xODY0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTIvMjAyMC4wMy4xMS45ODc5NTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">19.</span>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987958v1.19" data-doi="10.1038/nature12005">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Raj</span>, 
          <span class="cit-name-given-names">V. S.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Dipeptidyl peptidase 4 is a functional receptor for the emerging human coronavirus-EMC</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">495</span>, 
         <span class="cit-fpage">251</span>–
         <span class="cit-lpage">254</span> (
         <span class="cit-pub-date">2013</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D495%26rft.spage%253D251%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12005%26rft_id%253Dinfo%253Apmid%252F23486063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature12005&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23486063&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.11.987958.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000316652300054&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">20.</span>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987958v1.20">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Okba</span>, 
          <span class="cit-name-given-names">N. M. A.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Sensitive and Specific Detection of Low-Level Antibody Responses in Mild Middle East Respiratory Syndrome Coronavirus Infections</span>. 
         <abbr class="cit-jnl-abbrev">Emerg. Infect. Dis.</abbr>
         <span class="cit-vol">25</span>, 
         <span class="cit-fpage">1868</span>–
         <span class="cit-lpage">1877</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEmerg.%2BInfect.%2BDis.%26rft.volume%253D25%26rft.spage%253D1868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <div class="section ref-list" id="ref-list-2">
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1).</span>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 1) in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987958v1.21" data-doi="10.1080/22221751.2019.1597644">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Widjaja</span>, 
          <span class="cit-name-given-names">I.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Towards a solution to MERS: protective human monoclonal antibodies targeting different domains and functions of the MERS-coronavirus spike glycoprotein</span>. 
         <abbr class="cit-jnl-abbrev">Emerg. Microbes Infect.</abbr>
         <span class="cit-vol">8</span>, 
         <span class="cit-fpage">516</span>–
         <span class="cit-lpage">530</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEmerg.%2BMicrobes%2BInfect.%26rft.volume%253D8%26rft.spage%253D516%26rft_id%253Dinfo%253Adoi%252F10.1080%252F22221751.2019.1597644%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1080/22221751.2019.1597644&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">1).</span>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 1) in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987958v1.22" data-doi="10.1080/22221751.2019.1597644">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Widjaja</span>, 
          <span class="cit-name-given-names">I.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Towards a solution to MERS: protective human monoclonal antibodies targeting different domains and functions of the MERS-coronavirus spike glycoprotein</span>. 
         <abbr class="cit-jnl-abbrev">Emerg. Microbes Infect.</abbr>
         <span class="cit-vol">8</span>, 
         <span class="cit-fpage">516</span>–
         <span class="cit-lpage">530</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEmerg.%2BMicrobes%2BInfect.%26rft.volume%253D8%26rft.spage%253D516%26rft_id%253Dinfo%253Adoi%252F10.1080%252F22221751.2019.1597644%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1080/22221751.2019.1597644&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">1).</span>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 1) in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987958v1.23" data-doi="10.1080/22221751.2019.1597644">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Widjaja</span>, 
          <span class="cit-name-given-names">I.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Towards a solution to MERS: protective human monoclonal antibodies targeting different domains and functions of the MERS-coronavirus spike glycoprotein</span>. 
         <abbr class="cit-jnl-abbrev">Emerg. Microbes Infect.</abbr>
         <span class="cit-vol">8</span>, 
         <span class="cit-fpage">516</span>–
         <span class="cit-lpage">530</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEmerg.%2BMicrobes%2BInfect.%26rft.volume%253D8%26rft.spage%253D516%26rft_id%253Dinfo%253Adoi%252F10.1080%252F22221751.2019.1597644%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1080/22221751.2019.1597644&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">1).</span>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 1) in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.11.987958v1.24" data-doi="10.1126/science.1116480">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Farzan</span>, 
          <span class="cit-name-given-names">M.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Harrison</span>, 
          <span class="cit-name-given-names">S.C.</span>
         </span>
         <span class="cit-article-title">Structure of SARS coronavirus spike receptor-binding domain complexed with receptor</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>
         <span class="cit-vol">309</span>, 
         <span class="cit-fpage">1864</span>–
         <span class="cit-lpage">1868</span> (
         <span class="cit-pub-date">2005</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLi%26rft.auinit1%253DF.%26rft.volume%253D309%26rft.issue%253D5742%26rft.spage%253D1864%26rft.epage%253D1868%26rft.atitle%253DStructure%2Bof%2BSARS%2BCoronavirus%2BSpike%2BReceptor-Binding%2BDomain%2BComplexed%2Bwith%2BReceptor%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1116480%26rft_id%253Dinfo%253Apmid%252F16166518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDkvNTc0Mi8xODY0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMTIvMjAyMC4wMy4xMS45ODc5NTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
