<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2019/01/08/513333.atom"/>
  <meta name="HW.pisa" content="biorxiv;513333v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Contesting the evidence for −1 frameshifting in immune-functioning C-C chemokine receptor 5 (CCR5) – the HIV-1 co-receptor"/>
  <meta name="DC.Identifier" content="10.1101/513333"/>
  <meta name="DC.Date" content="2019-01-08"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="During the decoding of a subset of mRNAs a proportion of ribosomes productively shift to the −1 reading frame at a specific, slippage-prone site[1][1]. While the great majority of occurrences of the “Programmed −1 Ribosomal Frameshifting” (−1 PRF) involve viruses[2][2], other mobile"/>
  <meta name="DC.Contributor" content="Yousuf A. Khan"/>
  <meta name="DC.Contributor" content="Gary Loughran"/>
  <meta name="DC.Contributor" content="John F. Atkins"/>
  <meta name="article:published_time" content="2019-01-08"/>
  <meta name="article:section" content="Contradictory Results"/>
  <meta name="citation_title" content="Contesting the evidence for −1 frameshifting in immune-functioning C-C chemokine receptor 5 (CCR5) – the HIV-1 co-receptor"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2019/01/01"/>
  <meta name="citation_mjid" content="biorxiv;513333v1"/>
  <meta name="citation_id" content="513333v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/513333v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/513333v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/513333v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/01/08/513333.full.pdf"/>
  <meta name="citation_doi" content="10.1101/513333"/>
  <meta name="citation_num_pages" content="23"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="Contradictory Results"/>
  <meta name="citation_firstpage" content="513333"/>
  <meta name="citation_author" content="Yousuf A. Khan"/>
  <meta name="citation_author_institution" content="Division of Virology, Department of Pathology, University of Cambridge"/>
  <meta name="citation_author" content="Gary Loughran"/>
  <meta name="citation_author_institution" content="School of Biochemistry and Cell Biology, University College Cork"/>
  <meta name="citation_author" content="John F. Atkins"/>
  <meta name="citation_author_institution" content="School of Biochemistry and Cell Biology, University College Cork"/>
  <meta name="citation_reference" content="Atkins JF, Loughran G, Bhatt PR, Firth AE, Baranov P V. Ribosomal frameshifting and transcriptional slippage: From genetic steganography and cryptography to adventitious use. Nucleic Acids Res. 2016. doi:10.1093/nar/gkw530"/>
  <meta name="citation_reference" content="Jacks T, Varmus HE. Expression of the Rous sarcoma virus pol gene by ribosomal frameshifting. Science (80-). 1985. doi:10.1126/science.2416054"/>
  <meta name="citation_reference" content="Belew AT, Meskauskas A, Musalgaonkar S, et al. Ribosomal frameshifting in the CCR5 mRNA is regulated by miRNAs and the NMD pathway. Nature. 2014. doi:10.1038/nature13429"/>
  <meta name="citation_reference" content="Advani VM, Dinman JD. Reprogramming the genetic code: The emerging role of ribosomal frameshifting in regulating cellular gene expression. BioEssays. 2016. doi:10.1002/bies.201500131"/>
  <meta name="citation_reference" content="Belew AT, Dinman JD. Cell cycle control (and more) by programmed −1 ribosomal frameshifting: Implications for disease and therapeutics. Cell Cycle. 2015. doi:10.4161/15384101.2014.989123"/>
  <meta name="citation_reference" content="Brierley I, Jenner AJ, Inglis SC. Mutational analysis of the “slippery-sequence” component of a coronavirus ribosomal frameshifting signal. J Mol Biol. 1992. doi:10.1016/0022-2836(92)90901-U"/>
  <meta name="citation_reference" content="Loughran G, Howard MT, Firth AE, Atkins JF. Avoidance of reporter assay distortions from fused dual reporters. RNA. 2017. doi:10.1261/rna.061051.117"/>
  <meta name="citation_reference" content="Grentzmann G, Ingram JA, Kelly PJ, Gesteland RF, Atkins JF. A dual-luciferase reporter system for studying recoding signals. RNA. 1998."/>
  <meta name="citation_reference" content="Biswas P, Jiang X, Pacchia AL, Dougherty JP, Peltz SW. The Human Immunodeficiency Virus Type 1 Ribosomal Frameshifting Site Is an Invariant Sequence Determinant and an Important Target for Antiviral Therapy. J Virol. 2004. doi:10.1128/JVI.78.4.2082-2087.2004"/>
  <meta name="twitter:title" content="Contesting the evidence for −1 frameshifting in immune-functioning C-C chemokine receptor 5 (CCR5) – the HIV-1 co-receptor"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="During the decoding of a subset of mRNAs a proportion of ribosomes productively shift to the −1 reading frame at a specific, slippage-prone site[1][1]. While the great majority of occurrences of the “Programmed −1 Ribosomal Frameshifting” (−1 PRF) involve viruses[2][2], other mobile"/>
  <meta name="og-title" property="og:title" content="Contesting the evidence for −1 frameshifting in immune-functioning C-C chemokine receptor 5 (CCR5) – the HIV-1 co-receptor"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/513333v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="During the decoding of a subset of mRNAs a proportion of ribosomes productively shift to the −1 reading frame at a specific, slippage-prone site[1][1]. While the great majority of occurrences of the “Programmed −1 Ribosomal Frameshifting” (−1 PRF) involve viruses[2][2], other mobile "/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2019-01-08"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Contesting the evidence for −1 frameshifting in immune-functioning C-C chemokine receptor 5 (CCR5) – the HIV-1 co-receptor | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-151688 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-contradictory-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <p id="p-2">During the decoding of a subset of mRNAs a proportion of ribosomes productively shift to the −1 reading frame at a specific, slippage-prone site
    <sup>
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>
    </sup>. While the great majority of occurrences of the “Programmed −1 Ribosomal Frameshifting” (−1 PRF) involve viruses
    <sup>
     <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>
    </sup>, other mobile elements or retroelements
    <sup>
     <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>
    </sup>, a dramatic instance of utilized human cellular −1 frameshifting in a non-retroelement derived mRNA has been reported. The mRNA for which −1 frameshifting was claimed due to a stimulatory pseudoknot is that which encodes immune functioning C-C chemokine receptor 5 (CCR5), the HIV-1 co-receptor
    <sup>
     <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>
    </sup>. The publication ascribed the importance of the reporter 
    <em>CCR5</em> mRNA frameshifting directing translating ribosomes to a premature termination codon leading to mRNA destabilization via the nonsense-mediated decay pathway (NMD), rather than creating a C-terminally unique peptide. The same lab estimates that up to 10% of eukaryotic genes may be regulated by this frameshift-NMD mechanism
    <sup>
     <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>
    </sup> and point to the potential for novel therapeutic strategies aimed at treating disease by −1 PRF
    <sup>
     <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>
    </sup>.
   </p>
   <p id="p-3">Efficient −1 frameshifting requires a slippery site together with an appropriately positioned stimulatory RNA structure. The reported 
    <em>CCR5</em> slip site (U_UUA_AAA where underscores set of the reading frame) has previously been shown to facilitate high levels of −1 frameshifting in vitro when positioned 5’ of a known stimulatory pseudoknot
    <sup>
     <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>
    </sup>. However, frameshifting efficiencies in vivo could be substantially different and further, at least as of 2016, there were no known virus frameshift sites with the reported 
    <em>CCR5</em> shift site
    <sup>
     <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">1</a>
    </sup>.
   </p>
   <p id="p-4">We challenge the reported results by showing that frameshifting at the relevant 
    <em>CCR5</em> sequence cassette does not occur above background levels and that the claimed pertinent sequence is not as distinctively conserved as previously presented. Lastly, we demonstrate that previously published retroelement derived human frameshift signals do indeed harbor frameshift activity.
   </p>
   <p id="p-5">Using BLAST, we first identified orthologues of human 
    <em>CCR5</em> in the NCBI RefSeq database. Sequences were aligned and, on examining the frameshift region, we observed substitutions in the slippery site sequence which would likely decrease or eliminate frameshifting in related organisms. Indeed, none of the nucleotides in the slippery site are conserved across vertebrates (
    <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The predicted frameshift site in 
    <em>CCR5</em> was previously reported as most highly conserved in primates, except for 
    <em>Lemur catta</em>
    <sup>
     <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>
    </sup>. Primate 
    <em>CCR5</em> CDS sequences were aligned and nucleotide conservation was measured at single nucleotide resolution (
    <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>) and smoothed in a 10 (
    <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>) nucleotide sliding window. This analysis showed that the entire CDS, not just the predicted frameshift site, is highly conserved among primates. Thus, the observation that the frameshift site is mostly conserved in primates does not indicate that the (putative) ability to frameshift is subject to purifying selection.
   </p>
   <div id="F1" class="fig pos-float type-figure odd">
    <div class="highwire-figure">
     <div class="fig-inline-img-wrapper">
      <div class="fig-inline-img">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/08/513333/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Alignment of putative frameshift site: Sequence alignment of the CCR5 CDS regions from BLAST hits to human CCR5, trimmed to only include the frameshift region. The red box indicates the putative slippery site." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-795348926" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Alignment of putative frameshift site:&lt;;/span&gt;; Sequence alignment of the &lt;;em&gt;;CCR5&lt;;/em&gt;; CDS regions from BLAST hits to human &lt;;em&gt;;CCR5&lt;;/em&gt;;, trimmed to only include the frameshift region. The red box indicates the putative slippery site.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
        <span class="hw-responsive-img">
         <img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/01/08/513333/F1.medium.gif" width="310" height="440"/>
         <noscript>
          <img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2019/01/08/513333/F1.medium.gif" width="310" height="440"/>
         </noscript>
        </span>
       </a>
      </div>
     </div>
     <ul class="highwire-figure-links inline">
      <li class="download-fig first">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/08/513333/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a>
      </li>
      <li class="new-tab last">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/08/513333/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
      </li>
     </ul>
    </div>
    <div class="fig-caption">
     <span class="fig-label">Figure 1:</span>
     <span class="caption-title">Alignment of putative frameshift site:</span>
     <p id="p-6" class="first-child">Sequence alignment of the 
      <em>CCR5</em> CDS regions from BLAST hits to human 
      <em>CCR5</em>, trimmed to only include the frameshift region. The red box indicates the putative slippery site.
     </p>
     <div class="sb-div caption-clear"/>
    </div>
   </div>
   <div id="F2" class="fig pos-float type-figure odd">
    <div class="highwire-figure">
     <div class="fig-inline-img-wrapper">
      <div class="fig-inline-img">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/08/513333/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Sliding window conservation analysis of the CCR5 CDS: a. Single nucleotide resolution in the CCR5 CDS for primates only, excluding Lemur catta. b. 10-nucleotide sliding window of conservation in the CCR5 CDS for primates only, excluding Lemur catta. The grey highlight represents the CCR5 predicted frameshift region." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-795348926" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Sliding window conservation analysis of the &lt;;em&gt;;CCR5&lt;;/em&gt;; CDS:&lt;;/span&gt;; &lt;;strong&gt;;a.&lt;;/strong&gt;; Single nucleotide resolution in the &lt;;em&gt;;CCR5&lt;;/em&gt;; CDS for primates only, excluding &lt;;em&gt;;Lemur catta&lt;;/em&gt;;. &lt;;strong&gt;;b.&lt;;/strong&gt;; 10-nucleotide sliding window of conservation in the &lt;;em&gt;;CCR5&lt;;/em&gt;; CDS for primates only, excluding &lt;;em&gt;;Lemur catta&lt;;/em&gt;;. The grey highlight represents the &lt;;em&gt;;CCR5&lt;;/em&gt;; predicted frameshift region.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
        <span class="hw-responsive-img">
         <img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/01/08/513333/F2.medium.gif" width="440" height="269"/>
         <noscript>
          <img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2019/01/08/513333/F2.medium.gif" width="440" height="269"/>
         </noscript>
        </span>
       </a>
      </div>
     </div>
     <ul class="highwire-figure-links inline">
      <li class="download-fig first">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/08/513333/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a>
      </li>
      <li class="new-tab last">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/08/513333/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
      </li>
     </ul>
    </div>
    <div class="fig-caption">
     <span class="fig-label">Figure 2:</span>
     <span class="caption-title">Sliding window conservation analysis of the 
      <em>CCR5</em> CDS:
     </span>
     <p id="p-7" class="first-child">
      <strong>a.</strong> Single nucleotide resolution in the 
      <em>CCR5</em> CDS for primates only, excluding 
      <em>Lemur catta</em>. 
      <strong>b.</strong> 10-nucleotide sliding window of conservation in the 
      <em>CCR5</em> CDS for primates only, excluding 
      <em>Lemur catta</em>. The grey highlight represents the 
      <em>CCR5</em> predicted frameshift region.
     </p>
     <div class="sb-div caption-clear"/>
    </div>
   </div>
   <p id="p-8">Dual-luciferase vectors in which the test cassette is sandwiched between and fused to both Firefly and Renilla luciferase encoding sequences, have been widely used for assaying translational recoding events (
    <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). The frameshifting efficiency is calculated by dividing Firefly luciferase activity by Renilla luciferase activity. This ratio is then normalized to an in-frame control vector in which Renilla and Firefly luciferases are both situated in the zero-frame to account for differences in activity between the two enzymes (a ratio of a ratio) (
    <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>).
   </p>
   <div id="F3" class="fig pos-float type-figure odd">
    <div class="highwire-figure">
     <div class="fig-inline-img-wrapper">
      <div class="fig-inline-img">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/08/513333/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Schematic of the dual luciferase vector: a. Graphical representation of experimental and in-frame control for dual luciferase assays. b. Mathematical formula for calculating rates of ribosomal frameshifting. c. Graphical representation of negative controls." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-795348926" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Schematic of the dual luciferase vector:&lt;;/span&gt;; &lt;;strong&gt;;a.&lt;;/strong&gt;; Graphical representation of experimental and in-frame control for dual luciferase assays. &lt;;strong&gt;;b.&lt;;/strong&gt;; Mathematical formula for calculating rates of ribosomal frameshifting. &lt;;strong&gt;;c.&lt;;/strong&gt;; Graphical representation of negative controls.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
        <span class="hw-responsive-img">
         <img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/01/08/513333/F3.medium.gif" width="242" height="440"/>
         <noscript>
          <img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2019/01/08/513333/F3.medium.gif" width="242" height="440"/>
         </noscript>
        </span>
       </a>
      </div>
     </div>
     <ul class="highwire-figure-links inline">
      <li class="download-fig first">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/08/513333/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a>
      </li>
      <li class="new-tab last">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/08/513333/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
      </li>
     </ul>
    </div>
    <div class="fig-caption">
     <span class="fig-label">Figure 3:</span>
     <span class="caption-title">Schematic of the dual luciferase vector:</span>
     <p id="p-9" class="first-child">
      <strong>a.</strong> Graphical representation of experimental and in-frame control for dual luciferase assays. 
      <strong>b.</strong> Mathematical formula for calculating rates of ribosomal frameshifting. 
      <strong>c.</strong> Graphical representation of negative controls.
     </p>
     <div class="sb-div caption-clear"/>
    </div>
   </div>
   <p id="p-10">In nearly all cases the fused test encoded polypeptide has only a small or no effect on the upstream encoded luciferase activity. However, in one case luciferase activity was anomalously low (unpublished) and we developed a variant vector system (pSGDluc
    <sup>
     <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>
    </sup>) containing a StopGo sequence directly 3’ of the first and immediately 5’ of the second luciferase encoding sequence. Subsequently we made another variant by removing a potential splice site, pSGDlucV3.0 (deposited with Addgene). Thus in earlier constructs, where the insert-encoded peptides were appended to the C-terminus of Renilla luciferase, the Renilla luciferase activity could drop – particularly for the short zero-frame encoded peptide, causing an artificial inflation of translation recoding rates. This problem has been overcome by inserting flanking StopGo sequences that result in co-translational excision of the insert-encoded peptides, thus releasing independent Renilla and Firefly proteins that are identical in sequence for both the test and in-frame control constructs. The publication on 
    <em>CCR5</em> frameshifting utilized the very first iteration of a dual luciferase vector that we developed in 1998
    <sup>
     <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>
    </sup>.
   </p>
   <p id="p-11">We recloned the same 
    <em>CCR5</em> frameshift sequence tested in the original article into pSGDlucV3.0. For 
    <em>CCR5</em> we used negative controls comparable to those in the original article, which include a slip site mutant (UUUAAAA → GCGCGCG for 
    <strong>Y.A.K.</strong>, UUUAAAA → AUUCAAA for 
    <strong>G.L.</strong>), a 5`-termination control (also known as a premature termination codon control or PTC control) and a 3`-termination control (also known as an out-of-frame stop codon control or OOF control). Lastly, we included measurements from the empty vector alone (
    <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>). All four of these controls are meant to establish a baseline for background levels of luminescence
   </p>
   <p id="p-12">The dual luciferase assays were carried out in the same cell line used in the original article, HeLa. Renilla luciferase, the reference luciferase, is stable for all the constructs (
    <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). Similarly Firefly luciferase activities for the in-frame controls of 
    <em>CCR5 and</em> a HIV-1 control are consistent as well, indicating that when forced to express Firefly luciferase, both constructs produce proteins with similar activity. The Firefly luciferase activity for the WT HIV-1 frameshift construct is &gt;;40 fold higher than that of WT 
    <em>CCR5</em> or any of its negative controls. Furthermore, the Firefly luciferase activity for 
    <em>CCR5</em> and all its negative controls are all comparable to the activity of the empty vector control, which is the baseline for background levels of Firefly luciferase activity (
    <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). After normalizing the luciferase activities for the test vectors to their respective in-frame control vectors, we find that HIV-1 shows frameshift rates that are comparable to previously reported findings in HeLa cells
    <sup>
     <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>
    </sup>. However, 
    <em>CCR5</em> WT frameshifting is ~0.15%, indistinguishable from its negative controls, indicating that 
    <em>CCR5</em> does not frameshift above background at our limit of sensitivity and is certainly far less than the previously reported level of 9-11%
    <sup>
     <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">3</a>
    </sup>. In fact, 
    <em>CCR5</em> SSM demonstrates slightly higher rates of FS than the WT, indicating that translational recoding values this low are indicative of background (
    <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>). These experiments were repeated in HEK293T cells to control for cell-line specific results and we found that the results followed identical trends to those obtained in HeLa cells (
    <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4D-F</a>).
   </p>
   <div id="F4" class="fig pos-float type-figure odd">
    <div class="highwire-figure">
     <div class="fig-inline-img-wrapper">
      <div class="fig-inline-img">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/08/513333/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="The CCR5 sequence does not promote efficient frameshifting above its negative controls: a. Absolute Renilla luciferase values for all experiments performed in HeLa cells. b. Absolute Firefly luciferase values for all experiments performed in HeLa cells. c. Frameshifting efficiency for all experiments performed in HeLa cells, normalized to either the HIV-1 or CCR5 in-frame control vector as appropriate. d. Absolute Renilla luciferase values for all experiments performed in HEK293T cells. e. Absolute Firefly luciferase values for all experiments performed in HEK293T cells. f. Frameshifting efficiency for all experiments performed in HEK293T cells, normalized to either the HIV-1 or CCR5 in-frame control vector as appropriate. Every data point represents an independently transfected well of cells and error bars represent standard deviations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-795348926" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;The &lt;;em&gt;;CCR5&lt;;/em&gt;; sequence does not promote efficient frameshifting above its negative controls:&lt;;/span&gt;; &lt;;strong&gt;;a.&lt;;/strong&gt;; Absolute Renilla luciferase values for all experiments performed in HeLa cells. &lt;;strong&gt;;b.&lt;;/strong&gt;; Absolute Firefly luciferase values for all experiments performed in HeLa cells. &lt;;strong&gt;;c.&lt;;/strong&gt;; Frameshifting efficiency for all experiments performed in HeLa cells, normalized to either the HIV-1 or &lt;;em&gt;;CCR5&lt;;/em&gt;; in-frame control vector as appropriate. &lt;;strong&gt;;d.&lt;;/strong&gt;; Absolute Renilla luciferase values for all experiments performed in HEK293T cells. &lt;;strong&gt;;e.&lt;;/strong&gt;; Absolute Firefly luciferase values for all experiments performed in HEK293T cells. &lt;;strong&gt;;f.&lt;;/strong&gt;; Frameshifting efficiency for all experiments performed in HEK293T cells, normalized to either the HIV-1 or &lt;;em&gt;;CCR5&lt;;/em&gt;; in-frame control vector as appropriate. Every data point represents an independently transfected well of cells and error bars represent standard deviations.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
        <span class="hw-responsive-img">
         <img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/01/08/513333/F4.medium.gif" width="313" height="440"/>
         <noscript>
          <img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2019/01/08/513333/F4.medium.gif" width="313" height="440"/>
         </noscript>
        </span>
       </a>
      </div>
     </div>
     <ul class="highwire-figure-links inline">
      <li class="download-fig first">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/08/513333/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">Download figure</a>
      </li>
      <li class="new-tab last">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/08/513333/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
      </li>
     </ul>
    </div>
    <div class="fig-caption">
     <span class="fig-label">Figure 4:</span>
     <span class="caption-title">The 
      <em>CCR5</em> sequence does not promote efficient frameshifting above its negative controls:
     </span>
     <p id="p-13" class="first-child">
      <strong>a.</strong> Absolute Renilla luciferase values for all experiments performed in HeLa cells. 
      <strong>b.</strong> Absolute Firefly luciferase values for all experiments performed in HeLa cells. 
      <strong>c.</strong> Frameshifting efficiency for all experiments performed in HeLa cells, normalized to either the HIV-1 or 
      <em>CCR5</em> in-frame control vector as appropriate. 
      <strong>d.</strong> Absolute Renilla luciferase values for all experiments performed in HEK293T cells. 
      <strong>e.</strong> Absolute Firefly luciferase values for all experiments performed in HEK293T cells. 
      <strong>f.</strong> Frameshifting efficiency for all experiments performed in HEK293T cells, normalized to either the HIV-1 or 
      <em>CCR5</em> in-frame control vector as appropriate. Every data point represents an independently transfected well of cells and error bars represent standard deviations.
     </p>
     <div class="sb-div caption-clear"/>
    </div>
   </div>
   <p id="p-14">To ensure that these results were repeatable, the cloning and assaying of 
    <em>CCR5</em> frameshifting was completed independently by a different laboratory (
    <strong>G.L., J.F.A.</strong>), which used modestly different cloning and assaying strategies. When utilizing both HeLa and HEK293T cells, we found that 
    <em>CCR5</em> frameshifted at an insignificant rate as well (
    <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A-F</a>).
   </p>
   <div id="F5" class="fig pos-float type-figure odd">
    <div class="highwire-figure">
     <div class="fig-inline-img-wrapper">
      <div class="fig-inline-img">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/08/513333/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Independent confirmation that the CCR5 sequence does not promote frameshifting above its negative controls. a. Absolute Renilla luciferase values for all experiments performed in HeLa cells. b. Absolute Firefly luciferase values for all experiments performed in HeLa cells. c. Frameshifting efficiency for all experiments performed in HeLa cells, normalized to the CCR5 in-frame control vector. d. Absolute Renilla luciferase values for all experiments performed in HEK293T cells. e. Absolute Firefly luciferase values for all experiments performed in HEK293T cells. f. Frameshifting efficiency for all experiments performed in HEK293T cells, normalized to the CCR5 in-frame control vector. Every data point represents an independently transfected well of cells and error bars represent standard deviations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-795348926" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Independent confirmation that the &lt;;em&gt;;CCR5&lt;;/em&gt;; sequence does not promote frameshifting above its negative controls.&lt;;/span&gt;; &lt;;strong&gt;;a.&lt;;/strong&gt;; Absolute Renilla luciferase values for all experiments performed in HeLa cells. &lt;;strong&gt;;b.&lt;;/strong&gt;; Absolute Firefly luciferase values for all experiments performed in HeLa cells. &lt;;strong&gt;;c.&lt;;/strong&gt;; Frameshifting efficiency for all experiments performed in HeLa cells, normalized to the &lt;;em&gt;;CCR5&lt;;/em&gt;; in-frame control vector. &lt;;strong&gt;;d.&lt;;/strong&gt;; Absolute Renilla luciferase values for all experiments performed in HEK293T cells. &lt;;strong&gt;;e.&lt;;/strong&gt;; Absolute Firefly luciferase values for all experiments performed in HEK293T cells. &lt;;strong&gt;;f.&lt;;/strong&gt;; Frameshifting efficiency for all experiments performed in HEK293T cells, normalized to the &lt;;em&gt;;CCR5&lt;;/em&gt;; in-frame control vector. Every data point represents an independently transfected well of cells and error bars represent standard deviations.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
        <span class="hw-responsive-img">
         <img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/01/08/513333/F5.medium.gif" width="327" height="440"/>
         <noscript>
          <img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2019/01/08/513333/F5.medium.gif" width="327" height="440"/>
         </noscript>
        </span>
       </a>
      </div>
     </div>
     <ul class="highwire-figure-links inline">
      <li class="download-fig first">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/08/513333/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">Download figure</a>
      </li>
      <li class="new-tab last">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/08/513333/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
      </li>
     </ul>
    </div>
    <div class="fig-caption">
     <span class="fig-label">Figure 5:</span>
     <span class="caption-title">Independent confirmation that the 
      <em>CCR5</em> sequence does not promote frameshifting above its negative controls.
     </span>
     <p id="p-15" class="first-child">
      <strong>a.</strong> Absolute Renilla luciferase values for all experiments performed in HeLa cells. 
      <strong>b.</strong> Absolute Firefly luciferase values for all experiments performed in HeLa cells. 
      <strong>c.</strong> Frameshifting efficiency for all experiments performed in HeLa cells, normalized to the 
      <em>CCR5</em> in-frame control vector. 
      <strong>d.</strong> Absolute Renilla luciferase values for all experiments performed in HEK293T cells. 
      <strong>e.</strong> Absolute Firefly luciferase values for all experiments performed in HEK293T cells. 
      <strong>f.</strong> Frameshifting efficiency for all experiments performed in HEK293T cells, normalized to the 
      <em>CCR5</em> in-frame control vector. Every data point represents an independently transfected well of cells and error bars represent standard deviations.
     </p>
     <div class="sb-div caption-clear"/>
    </div>
   </div>
   <p id="p-16">Retrotransposons and retroviruses share a common origin. Ribosomal frameshifting has been well documented in viruses and thus it stands to reason that retrotransposons and/or derivatives of retrotransposons in the human genome could frameshift. Three derivatives of retrotransposons in the human genome, 
    <em>PNMA3 (Ma3), PNMA5 (Ma5)</em>, and 
    <em>PEG10</em>, have been previously demonstrated to frameshift in-vivo and in-vitro
    <sup>
     <a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">1</a>
    </sup>. All three encode zero-frame proteins with homology to the viral protein 
    <em>gag</em> and their trans-frame protein have similarity to 
    <em>gag-pol</em>. All three share a slippery site common with some viruses in the 
    <em>Retroviridae</em> family, GGGAAAC. These sequences were assayed in both HeLa and HEK293T cells (
    <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). Although the rates of −1 PRF were slightly different from previously reported, all three signals stimulated frameshifting far above background. These results further confirm the fidelity of the pSGDlucV3.0 vector for assaying −1 PRF, already seen for other translational recoding events
    <sup>
     <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>
    </sup>.
   </p>
   <div id="F6" class="fig pos-float type-figure odd">
    <div class="highwire-figure">
     <div class="fig-inline-img-wrapper">
      <div class="fig-inline-img">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/08/513333/F6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Previously validated retrotransposon-derived frameshift signals in human still demonstrate efficient frameshifting. a. Absolute Renilla luciferase values for all experiments performed in HeLa cells. b. Absolute Firefly luciferase values for all experiments performed in HeLa cells. c. Frameshifting efficiency for all experiments performed in HeLa cells, normalized to the respective in-frame controls Ma3, Ma5, or PEG10, as appropriate. d. Absolute Renilla luciferase values for all experiments performed in HEK293T cells. e. Absolute Firefly luciferase values for all experiments performed in HEK293T cells. f. Frameshifting efficiency for all experiments performed in HEK293T cells, normalized to the respective in-frame controls Ma3, Ma5, or PEG10, as appropriate." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-795348926" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Previously validated retrotransposon-derived frameshift signals in human still demonstrate efficient frameshifting.&lt;;/span&gt;; &lt;;strong&gt;;a.&lt;;/strong&gt;; Absolute Renilla luciferase values for all experiments performed in HeLa cells. &lt;;strong&gt;;b.&lt;;/strong&gt;; Absolute Firefly luciferase values for all experiments performed in HeLa cells. &lt;;strong&gt;;c.&lt;;/strong&gt;; Frameshifting efficiency for all experiments performed in HeLa cells, normalized to the respective in-frame controls &lt;;em&gt;;Ma3, Ma5&lt;;/em&gt;;, or &lt;;em&gt;;PEG10&lt;;/em&gt;;, as appropriate. &lt;;strong&gt;;d.&lt;;/strong&gt;; Absolute Renilla luciferase values for all experiments performed in HEK293T cells. &lt;;strong&gt;;e.&lt;;/strong&gt;; Absolute Firefly luciferase values for all experiments performed in HEK293T cells. &lt;;strong&gt;;f.&lt;;/strong&gt;; Frameshifting efficiency for all experiments performed in HEK293T cells, normalized to the respective in-frame controls &lt;;em&gt;;Ma3, Ma5&lt;;/em&gt;;, or &lt;;em&gt;;PEG10&lt;;/em&gt;;, as appropriate.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
        <span class="hw-responsive-img">
         <img class="highwire-fragment fragment-image lazyload" alt="Figure 6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/01/08/513333/F6.medium.gif" width="323" height="440"/>
         <noscript>
          <img class="highwire-fragment fragment-image" alt="Figure 6:" src="https://www.biorxiv.org/content/biorxiv/early/2019/01/08/513333/F6.medium.gif" width="323" height="440"/>
         </noscript>
        </span>
       </a>
      </div>
     </div>
     <ul class="highwire-figure-links inline">
      <li class="download-fig first">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/08/513333/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6:" data-icon-position="" data-hide-link-title="0">Download figure</a>
      </li>
      <li class="new-tab last">
       <a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/08/513333/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
      </li>
     </ul>
    </div>
    <div class="fig-caption">
     <span class="fig-label">Figure 6:</span>
     <span class="caption-title">Previously validated retrotransposon-derived frameshift signals in human still demonstrate efficient frameshifting.</span>
     <p id="p-17" class="first-child">
      <strong>a.</strong> Absolute Renilla luciferase values for all experiments performed in HeLa cells. 
      <strong>b.</strong> Absolute Firefly luciferase values for all experiments performed in HeLa cells. 
      <strong>c.</strong> Frameshifting efficiency for all experiments performed in HeLa cells, normalized to the respective in-frame controls 
      <em>Ma3, Ma5</em>, or 
      <em>PEG10</em>, as appropriate. 
      <strong>d.</strong> Absolute Renilla luciferase values for all experiments performed in HEK293T cells. 
      <strong>e.</strong> Absolute Firefly luciferase values for all experiments performed in HEK293T cells. 
      <strong>f.</strong> Frameshifting efficiency for all experiments performed in HEK293T cells, normalized to the respective in-frame controls 
      <em>Ma3, Ma5</em>, or 
      <em>PEG10</em>, as appropriate.
     </p>
     <div class="sb-div caption-clear"/>
    </div>
   </div>
   <p id="p-18">In conclusion, using an improved reporter assay system, we find no evidence of −1 frameshifting above background levels on the CCR5 mRNA. Additionally, a lack of statistically significant purifying selection acting on the slip site (a hallmark for functionally relevant frameshift signals) supports the argument that 
    <em>CCR5</em> does not contain a biologically important frameshift signal. There are several possible explanations for the discrepancy between the reported 
    <em>CCR5</em> results and the results presented here but the most obvious one is the standard decoding of a significant percentage of mRNAs generated by cryptic splicing.
   </p>
   <div class="section" id="sec-1">
    <h2 class="">Methods</h2>
    <div id="sec-2" class="subsection">
     <h3>Alignment of 
      <em>CCR5</em> slippery sites
     </h3>
     <p id="p-19">The NCBI mRNA and RefSeq BLAST databases were searched with blastn using the human CCR5 mRNA (accession 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=NM_000579.3&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F08%2F513333.atom">NM_000579.3</a>) as query on December 4th, 2018 using default parameters, with the number of hits expanded from 100 to 500. The best sequence per organism, as determined by ‘Max Score’, was taken up until 70% query cover. These sequences were aligned with MUSCLE. This alignment was trimmed to only include the putative frameshift region of CCR5 that was generated by code written in Python 3. The alignment image was generated using the box-shade server (
      <a href="https://embnet.vital-it.ch/software/BOX_form.html">https://embnet.vital-it.ch/software/BOX_form.html</a>).
     </p>
    </div>
    <div id="sec-3" class="subsection">
     <h3>
      <em>CCR5</em> Primate CDS analysis
     </h3>
     <p id="p-20">Of the sequences obtained in the ortholog search, only those belonging to the taxonomic descriptor of ‘primate’ (taxid 9443) were used. 
      <em>Lemur catta</em> sequences were ignored, since the original manuscript states that the 
      <em>CCR5</em> frameshift signal is not conserved in 
      <em>Lemur catta</em>. All CDS sequences were aligned with MUSCLE and then either at single nucleotide resolution or a smoothed 10 nucleotide window plotted in Python 3.
     </p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>Vector Construction</h3>
     <p id="p-21">pSGDlucV3.0 was generated by introducing silent mutations into the Renilla coding sequence to disrupt two potential donor splice sites (TGGgtaagt).</p>
     <p id="p-22">For 
      <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figures 4</a> and 
      <a id="xref-fig-6-2" class="xref-fig" href="#F6">6</a> (
      <strong>Y.A.K.</strong>) the vector was digested with 
      <em>Psp</em>XI and 
      <em>Bgl</em>-II. Overlapping oligonucleotides for insert sequences were ordered from Sigma Aldrich (
      <strong>Supplementary Table 1</strong>), resuspended at 100 μM, diluted to 10 μM, and then mixed together at a 1:100 dilution. This 1:100 dilution of the forward and reverse oligonucleotide was heated to 95°C. 2 μL of this mix was ligated into 40 ng of linearized backbone vector using NEB T4 ligase.
     </p>
     <p id="p-23">For 
      <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a> (
      <strong>G.L.</strong>) The vector was digested with 
      <em>Psp</em>XI and 
      <em>Bgl</em>-II. Using forward and reverse PCR primers (
      <strong>Supplementary Table 1</strong>), the 
      <em>CCR5</em> WT, SSM and IFC were all amplified from genomic DNA with enzyme overhangs. These PCR products were subsequently digested and ligated into pSGDlucV3.0.
     </p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Transfections and Dual Luciferase Assays</h3>
     <p id="p-24">For 
      <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figures 4</a> and 
      <a id="xref-fig-6-3" class="xref-fig" href="#F6">6</a> (
      <strong>Y.A.K.</strong>) HeLa and HEK293T cells were reverse transfected with 100 ng of plasmid using 0.2 μL Lipofectamine 2000 in a 96-well format. Transfection mix was added to 30,000 cells and incubated for 1-2 minutes at room temperature, before adding Fetal Calf Serum. The next day, media was removed from cells and 100 μL of fresh 1X passive lysis buffer (Promega) was added to each well. Samples were frozen and thawed before luciferase activity was measured by adding 20 μL of lysate and 20 μL of each reagent, as per the Promega protocol.
     </p>
     <p id="p-25">For 
      <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a> (
      <strong>G.L.</strong>) HeLa and HEK293T cells were reverse transfected with 25 ng of plasmid using 0.2 μL Lipofectamine 2000 in a half-area 96-well format. Transfection mix was added to 40,000 cells. The next day, transfected cells were lysed in 12.6 μL of 1X passive lysis buffer (Promega) and luciferase activity was measured by injection of 20 μL of each reagent.
     </p>
     <p id="p-26">Background level readings (measured from water transfected cells) were subtracted from all Firefly and Renilla luciferase readings. Frameshift efficiencies were calculated by dividing Firefly luciferase values by Renilla luciferase values and then dividing the relative ratios by the average F-luc/R-luc ratio of the in-frame control.</p>
    </div>
   </div>
   <div class="section con" id="sec-6">
    <h2 class="">Author Contributions</h2>
    <p id="p-27">Y.A.K. conceived and coordinated the project. G.L. established the pSGDlucV3.0 vector used in this study. Y.A.K. performed the experiments and analyzed the data shown in 
     <a id="xref-fig-4-7" class="xref-fig" href="#F4">Figures 4</a> and 
     <a id="xref-fig-6-4" class="xref-fig" href="#F6">6</a> whereas G.L. independently conducted experiments and analyzed the data shown in 
     <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>. Y.A.K. performed the comparative phylogenetic analyses shown in 
     <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figures 1</a> and 
     <a id="xref-fig-2-3" class="xref-fig" href="#F2">2</a>. Y.A.K. wrote the manuscript and created the figures. All authors edited and approved the manuscript.
    </p>
   </div>
   <div class="section COI-statement" id="sec-7">
    <h2 class="">Competing Interests Statement</h2>
    <p id="p-28">
     <span class="named-content COI-statement" id="named-content-1">All authors declare no competing interests.</span>
    </p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgements</h2>
    <p id="p-29">This work was supported by grants from Science Foundation Ireland (12/IP/1492 and 13/1A/1853) to J.F.A. Y.A.K. was supported by an award from the Winston Churchill Foundation of the United States of America.</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-513333v1.1" data-doi="10.1093/nar/gkw530">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Atkins</span>
          <span class="cit-name-given-names">JF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Loughran</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bhatt</span>
          <span class="cit-name-given-names">PR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Firth</span>
          <span class="cit-name-given-names">AE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Baranov</span>
          <span class="cit-name-given-names">P V</span>
         </span>. 
         <span class="cit-article-title">Ribosomal frameshifting and transcriptional slippage: From genetic steganography and cryptography to adventitious use</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>
         <span class="cit-pub-date">2016</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gkw530</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw530%26rft_id%253Dinfo%253Apmid%252F27436286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkw530&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=27436286&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F08%2F513333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-513333v1.2" data-doi="10.1126/science.2416054">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jacks</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Varmus</span>
          <span class="cit-name-given-names">HE</span>
         </span>. 
         <span class="cit-article-title">Expression of the Rous sarcoma virus pol gene by ribosomal frameshifting</span>. 
         <abbr class="cit-jnl-abbrev">Science (80-)</abbr>. 
         <span class="cit-pub-date">1985</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1126/science.2416054</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DJacks%26rft.auinit1%253DT%26rft.volume%253D230%26rft.issue%253D4731%26rft.spage%253D1237%26rft.epage%253D1242%26rft.atitle%253DExpression%2Bof%2Bthe%2BRous%2Bsarcoma%2Bvirus%2Bpol%2Bgene%2Bby%2Bribosomal%2Bframeshifting%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.2416054%26rft_id%253Dinfo%253Apmid%252F2416054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyMzAvNDczMS8xMjM3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDEvMDgvNTEzMzMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-513333v1.3" data-doi="10.1038/nature13429">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Belew</span>
          <span class="cit-name-given-names">AT</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Meskauskas</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Musalgaonkar</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Ribosomal frameshifting in the CCR5 mRNA is regulated by miRNAs and the NMD pathway</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>. 
         <span class="cit-pub-date">2014</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/nature13429</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D512%26rft.spage%253D265%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13429%26rft_id%253Dinfo%253Apmid%252F25043019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature13429&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25043019&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F08%2F513333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000340508200025&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-513333v1.4" data-doi="10.1002/bies.201500131">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Advani</span>
          <span class="cit-name-given-names">VM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dinman</span>
          <span class="cit-name-given-names">JD</span>
         </span>. 
         <span class="cit-article-title">Reprogramming the genetic code: The emerging role of ribosomal frameshifting in regulating cellular gene expression</span>. 
         <abbr class="cit-jnl-abbrev">BioEssays</abbr>. 
         <span class="cit-pub-date">2016</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1002/bies.201500131</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioEssays%26rft.volume%253D38%26rft.spage%253D21%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.201500131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1002/bies.201500131&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-513333v1.5" data-doi="10.4161/15384101.2014.989123">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Belew</span>
          <span class="cit-name-given-names">AT</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dinman</span>
          <span class="cit-name-given-names">JD</span>
         </span>. 
         <span class="cit-article-title">Cell cycle control (and more) by programmed −1 ribosomal frameshifting: Implications for disease and therapeutics</span>. 
         <abbr class="cit-jnl-abbrev">Cell Cycle</abbr>. 
         <span class="cit-pub-date">2015</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.4161/15384101.2014.989123</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%2BCycle%26rft.volume%253D14%26rft.spage%253D172%26rft_id%253Dinfo%253Adoi%252F10.4161%252F15384101.2014.989123%26rft_id%253Dinfo%253Apmid%252F25584829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.4161/15384101.2014.989123&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25584829&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F08%2F513333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-513333v1.6" data-doi="10.1016/0022-2836(92)90901-U">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Brierley</span>
          <span class="cit-name-given-names">I</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jenner</span>
          <span class="cit-name-given-names">AJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Inglis</span>
          <span class="cit-name-given-names">SC</span>
         </span>. 
         <span class="cit-article-title">Mutational analysis of the “slippery-sequence” component of a coronavirus ribosomal frameshifting signal</span>. 
         <abbr class="cit-jnl-abbrev">J Mol Biol.</abbr>
         <span class="cit-pub-date">1992</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1016/0022-2836(92)90901-U</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DBrierley%26rft.auinit1%253DI.%26rft.volume%253D227%26rft.issue%253D2%26rft.spage%253D463%26rft.epage%253D479%26rft.atitle%253DMutational%2Banalysis%2Bof%2Bthe%2B%2526quot%253Bslippery-sequence%2526quot%253B%2Bcomponent%2Bof%2Ba%2Bcoronavirus%2Bribosomal%2Bframeshifting%2Bsignal.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252892%252990901-U%26rft_id%253Dinfo%253Apmid%252F1404364%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/0022-2836(92)90901-U&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=1404364&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F08%2F513333.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1992JQ37300012&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-513333v1.7" data-doi="10.1261/rna.061051.117">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Loughran</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Howard</span>
          <span class="cit-name-given-names">MT</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Firth</span>
          <span class="cit-name-given-names">AE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Atkins</span>
          <span class="cit-name-given-names">JF</span>
         </span>. 
         <span class="cit-article-title">Avoidance of reporter assay distortions from fused dual reporters</span>. 
         <abbr class="cit-jnl-abbrev">RNA</abbr>. 
         <span class="cit-pub-date">2017</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1261/rna.061051.117</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.061051.117%26rft_id%253Dinfo%253Apmid%252F28442579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjk6IjIzLzgvMTI4NSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAxLzA4LzUxMzMzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-513333v1.8">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Grentzmann</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ingram</span>
          <span class="cit-name-given-names">JA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kelly</span>
          <span class="cit-name-given-names">PJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gesteland</span>
          <span class="cit-name-given-names">RF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Atkins</span>
          <span class="cit-name-given-names">JF</span>
         </span>. 
         <span class="cit-article-title">A dual-luciferase reporter system for studying recoding signals</span>. 
         <abbr class="cit-jnl-abbrev">RNA</abbr>. 
         <span class="cit-pub-date">1998</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-513333v1.9" data-doi="10.1128/JVI.78.4.2082-2087.2004">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Biswas</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jiang</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pacchia</span>
          <span class="cit-name-given-names">AL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dougherty</span>
          <span class="cit-name-given-names">JP</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Peltz</span>
          <span class="cit-name-given-names">SW</span>
         </span>. 
         <span class="cit-article-title">The Human Immunodeficiency Virus Type 1 Ribosomal Frameshifting Site Is an Invariant Sequence Determinant and an Important Target for Antiviral Therapy</span>. 
         <abbr class="cit-jnl-abbrev">J Virol.</abbr>
         <span class="cit-pub-date">2004</span>.
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.78.4.2082-2087.2004</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DBiswas%26rft.auinit1%253DP.%26rft.volume%253D78%26rft.issue%253D4%26rft.spage%253D2082%26rft.epage%253D2087%26rft.atitle%253DThe%2BHuman%2BImmunodeficiency%2BVirus%2BType%2B1%2BRibosomal%2BFrameshifting%2BSite%2BIs%2Ban%2BInvariant%2BSequence%2BDeterminant%2Band%2Ban%2BImportant%2BTarget%2Bfor%2BAntiviral%2BTherapy%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.78.4.2082-2087.2004%26rft_id%253Dinfo%253Apmid%252F14747573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijc4LzQvMjA4MiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAxLzA4LzUxMzMzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
