<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/02/13/2020.02.07.939207.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.02.07.939207v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Evidence of recombination in coronaviruses implicating pangolin origins of nCoV-2019"/>
  <meta name="DC.Identifier" content="10.1101/2020.02.07.939207"/>
  <meta name="DC.Date" content="2020-02-13"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="A novel coronavirus (nCoV-2019) was the cause of an outbreak of respiratory illness detected in Wuhan, Hubei Province, China in December of 2019. Genomic analyses of nCoV-2019 determined a 96% resemblance with a coronavirus isolated from a bat in 2013 (RaTG13); however, the receptor binding motif (RBM) of these two genomes share low sequence similarity. This divergence suggests a possible alternative source for the RBM coding sequence in nCoV-2019. We identified high sequence similarity in the RBM between nCoV-2019 and a coronavirus genome reconstructed from a viral metagenomic dataset from pangolins possibly indicating a more complex origin for nCoV-2019."/>
  <meta name="DC.Contributor" content="Matthew C. Wong"/>
  <meta name="DC.Contributor" content="Sara J. Javornik Cregeen"/>
  <meta name="DC.Contributor" content="Nadim J. Ajami"/>
  <meta name="DC.Contributor" content="Joseph F. Petrosino"/>
  <meta name="article:published_time" content="2020-02-13"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Evidence of recombination in coronaviruses implicating pangolin origins of nCoV-2019"/>
  <meta name="citation_abstract" lang="en" content="&lt;;h3&gt;;SUMMARY&lt;;/h3&gt;; &lt;;p&gt;;A novel coronavirus (nCoV-2019) was the cause of an outbreak of respiratory illness detected in Wuhan, Hubei Province, China in December of 2019. Genomic analyses of nCoV-2019 determined a 96% resemblance with a coronavirus isolated from a bat in 2013 (RaTG13); however, the receptor binding motif (RBM) of these two genomes share low sequence similarity. This divergence suggests a possible alternative source for the RBM coding sequence in nCoV-2019. We identified high sequence similarity in the RBM between nCoV-2019 and a coronavirus genome reconstructed from a viral metagenomic dataset from pangolins possibly indicating a more complex origin for nCoV-2019.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.02.07.939207v1"/>
  <meta name="citation_id" content="2020.02.07.939207v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.07.939207v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.07.939207v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.07.939207v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.07.939207.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.02.07.939207"/>
  <meta name="citation_num_pages" content="9"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.02.07.939207"/>
  <meta name="citation_author" content="Matthew C. Wong"/>
  <meta name="citation_author_institution" content="Alkek Center for Metagenomics and Microbiome Research, Department of Molecular Virology and Microbiology, Baylor College of Medicine"/>
  <meta name="citation_author" content="Sara J. Javornik Cregeen"/>
  <meta name="citation_author_institution" content="Alkek Center for Metagenomics and Microbiome Research, Department of Molecular Virology and Microbiology, Baylor College of Medicine"/>
  <meta name="citation_author" content="Nadim J. Ajami"/>
  <meta name="citation_author_institution" content="Program for Innovative Microbiome and Translational Research, Department of Genomic Medicine, The University of Texas MD Anderson Cancer Center"/>
  <meta name="citation_author" content="Joseph F. Petrosino"/>
  <meta name="citation_author_institution" content="Alkek Center for Metagenomics and Microbiome Research, Department of Molecular Virology and Microbiology, Baylor College of Medicine"/>
  <meta name="citation_author_email" content="jpetrosi@bcm.edu"/>
  <meta name="citation_reference" content="Wu F, Zhao S, Yu B, Chen YM, Wang W, Song ZG, Hu Y et al. A new coronavirus associated with human respiratory disease in China. Nature 2020;453"/>
  <meta name="citation_reference" content="Zhou P, Yang XL, Wang XG, Hu B, Zhang L, Zhang W et al. A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature 2020;1221"/>
  <meta name="citation_reference" content="Liu P, Chen W, Chen JP. Viral Metagenomics Revealed Sendai Virus and Coronavirus Infection of Malayan Pangolins (Manis javanica). Viruses 2019; 11(11): 979"/>
  <meta name="citation_reference" content="Li, F. Structure, Function, and Evolution of Coronavirus Spike Proteins. Annual Review of Virology 2016;3(1): 237–261"/>
  <meta name="citation_reference" content="Ajami NJ, Wong MC, Ross MC, Lloyd RE, Petrosino JF. Maximal viral information recovery from sequence data using VirMAP. Nature Communications 2018; 9: 3205"/>
  <meta name="citation_reference" content="Hu D, Zhu C, Ai L, He T, Wang Y, Ye F, et al. Genomic characterization and infectivity of a novel SARS-like coronavirus in Chinese bats. Emerging microbes and infections. 2018; 7(1): 1–10"/>
  <meta name="citation_reference" content="Bushnell B. BBMap Short Read Aligner. Berkeley, California: University of California; 2016; sourceforge.net/projects/bbmap/"/>
  <meta name="citation_reference" content="Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ. Basic local alignment search tool. Journal of Molecular Biology 1990; 215:403–410."/>
  <meta name="citation_reference" content="Camacho C, Coulouris G, Avagyan V, Ma N, Papadopoulos J, Bealer K, Madden TL. BLAST+: architecture and applications. BMC Bioinformatics 2008; 10:421."/>
  <meta name="citation_reference" content="Martin DP, Murrell B, Golden M, Khoosal A, Muhire B. RDP4: Detection and analysis of recombination patterns in virus genomes. Virus Evolution 2015; 1–5"/>
  <meta name="citation_reference" content="R Core Team (2019). R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. URL https://www.R-project.org/"/>
  <meta name="citation_reference" content="H. Wickham. ggplot2: Elegant Graphics for Data Analysis. Springer-Verlag New York, 2016."/>
  <meta name="citation_reference" content="Katoh K, Standley DM. MAFFT Multiple Sequence Alignment Software Version 7: Improvements in Performance and Usability. Molecular Biology and Evolution 2013; 30(4): 772–780."/>
  <meta name="citation_reference" content="Kumar S, Stecher G, Li M, Knyaz C, Tamura K. MEGA X: Molecular Evolutionary Genetics Analysis across computing platforms. Molecular Biology and Evolution 2018; 35:1547–1549."/>
  <meta name="citation_reference" content="Lau SK, Feng Y, Chen H, et al. Severe Acute Respiratory Syndrome (SARS) Coronavirus ORF8 Protein Is Acquired from SARS-Related Coronavirus from Greater Horseshoe Bats through Recombination. Journal of Virology 2015;89(20):10532–10547."/>
  <meta name="citation_reference" content="Zhang Z, Shen L, Gu X. Evolutionary Dynamics of MERS-CoV: Potential Recombination, Positive Selection and Transmission. Scientific Reports 2016; 6:25049."/>
  <meta name="twitter:title" content="Evidence of recombination in coronaviruses implicating pangolin origins of nCoV-2019"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="A novel coronavirus (nCoV-2019) was the cause of an outbreak of respiratory illness detected in Wuhan, Hubei Province, China in December of 2019. Genomic analyses of nCoV-2019 determined a 96% resemblance with a coronavirus isolated from a bat in 2013 (RaTG13); however, the receptor binding motif (RBM) of these two genomes share low sequence similarity. This divergence suggests a possible alternative source for the RBM coding sequence in nCoV-2019. We identified high sequence similarity in the RBM between nCoV-2019 and a coronavirus genome reconstructed from a viral metagenomic dataset from pangolins possibly indicating a more complex origin for nCoV-2019."/>
  <meta name="og-title" property="og:title" content="Evidence of recombination in coronaviruses implicating pangolin origins of nCoV-2019"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.07.939207v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="A novel coronavirus (nCoV-2019) was the cause of an outbreak of respiratory illness detected in Wuhan, Hubei Province, China in December of 2019. Genomic analyses of nCoV-2019 determined a 96% resemblance with a coronavirus isolated from a bat in 2013 (RaTG13); however, the receptor binding motif (RBM) of these two genomes share low sequence similarity. This divergence suggests a possible alternative source for the RBM coding sequence in nCoV-2019. We identified high sequence similarity in the RBM between nCoV-2019 and a coronavirus genome reconstructed from a viral metagenomic dataset from pangolins possibly indicating a more complex origin for nCoV-2019."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-02-13"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Evidence of recombination in coronaviruses implicating pangolin origins of nCoV-2019 | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1148609 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">SUMMARY</h2>
    <p id="p-3">A novel coronavirus (nCoV-2019) was the cause of an outbreak of respiratory illness detected in Wuhan, Hubei Province, China in December of 2019. Genomic analyses of nCoV-2019 determined a 96% resemblance with a coronavirus isolated from a bat in 2013 (RaTG13); however, the receptor binding motif (RBM) of these two genomes share low sequence similarity. This divergence suggests a possible alternative source for the RBM coding sequence in nCoV-2019. We identified high sequence similarity in the RBM between nCoV-2019 and a coronavirus genome reconstructed from a viral metagenomic dataset from pangolins possibly indicating a more complex origin for nCoV-2019.</p>
   </div>
   <div id="sec-1" class="subsection">
    <p id="p-4">An outbreak of respiratory illness caused by a novel coronavirus (nCoV-2019)
     <sup>
      <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>
     </sup>, first identified in Wuhan China, has resulted in over twenty thousand confirmed cases worldwide (WHO Situation Report, February 5 2020). Data available to date has led to the hypothesis that the outbreak strain originated from bats. The nCoV-2019 has been reported to share 96% sequence identity to the RaTG13 genome, a coronavirus isolated from an intermediate horseshoe bat (
     <em>Rhinolophus affinis</em>)
     <sup>
      <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>
     </sup>. RaTG13 is the most closely related coronavirus genome to nCoV-2019, however other viral strains may have contributed to its genomic content. To explore this hypothesis, we examined several viral metagenomic datasets for potential coronaviruses that could harbor RBMs closer to nCoV-2019. We report evidence of a coronavirus recovered from a Malayan pangolin (
     <em>Manis javanica</em>) viral metagenomic dataset
     <sup>
      <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>
     </sup> that shares a higher sequence identity at a crucial segment of the genome involved in host infection (
     <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). This result indicates a potential recombination event and a more complex origin for the nCoV-2019 strain.
    </p>
    <div id="F1" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.07.939207/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Nucleotide and amino acid identity across three coronavirus genomes relative to the outbreak strain (nCoV-2019). Panel A: Distances of selected coronavirus genomes relative to nCov-2019 across the entire genome length. The gaps in the Pangolin-CoV are a result of the incompleteness of the genome used for comparison. Grey box indicates S1 and S2 domains of the spike protein. Panel B: Close up of the S1 domain with surrounding regions. Grey boxes highlight the S1-NTD subdomain and the S1-CTD or RBD subdomain. In the RBD subdomain the homology of the Pangolin-CoV genome exceeds that of RaTG13-CoV as compared to the nCoV-2019 strain. Panel C: Multiple alignment of the receptor biding motif at the amino acid level. Boxes highlight the key amino acid residues involved in ACE2 receptor binding. The nCoV-2019 and Pangolin-CoV strains are identical at all five amino acid residues, whereas RaTG13-CoV strain differs at four out of five compared to nCoV-2019. Labels: nCoV-2019 [[#x2013]] outbreak strain; Pangolin-CoV [[#x2013]] pangolin coronavirus identified in this study; RaTG13-CoV [[#x2013]] R. affinis coronavirus; Bat-CoV [[#x2013]] bat coronavirus (MG772933.1)" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2136161409" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Nucleotide and amino acid identity across three coronavirus genomes relative to the outbreak strain (nCoV-2019). Panel A: Distances of selected coronavirus genomes relative to nCov-2019 across the entire genome length. The gaps in the Pangolin-CoV are a result of the incompleteness of the genome used for comparison. Grey box indicates S1 and S2 domains of the spike protein. Panel B: Close up of the S1 domain with surrounding regions. Grey boxes highlight the S1-NTD subdomain and the S1-CTD or RBD subdomain. In the RBD subdomain the homology of the Pangolin-CoV genome exceeds that of RaTG13-CoV as compared to the nCoV-2019 strain. Panel C: Multiple alignment of the receptor biding motif at the amino acid level. Boxes highlight the key amino acid residues involved in ACE2 receptor binding. The nCoV-2019 and Pangolin-CoV strains are identical at all five amino acid residues, whereas RaTG13-CoV strain differs at four out of five compared to nCoV-2019. Labels: nCoV-2019 [[#x2013]] outbreak strain; Pangolin-CoV [[#x2013]] pangolin coronavirus identified in this study; RaTG13-CoV [[#x2013]] &lt;;em&gt;;R. affinis&lt;;/em&gt;; coronavirus; Bat-CoV [[#x2013]] bat coronavirus (MG772933.1)&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.07.939207/F1.medium.gif" width="440" height="240"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.07.939207/F1.medium.gif" width="440" height="240"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.07.939207/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.07.939207/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 1:</span>
      <p id="p-5" class="first-child">Nucleotide and amino acid identity across three coronavirus genomes relative to the outbreak strain (nCoV-2019). Panel A: Distances of selected coronavirus genomes relative to nCov-2019 across the entire genome length. The gaps in the Pangolin-CoV are a result of the incompleteness of the genome used for comparison. Grey box indicates S1 and S2 domains of the spike protein. Panel B: Close up of the S1 domain with surrounding regions. Grey boxes highlight the S1-NTD subdomain and the S1-CTD or RBD subdomain. In the RBD subdomain the homology of the Pangolin-CoV genome exceeds that of RaTG13-CoV as compared to the nCoV-2019 strain. Panel C: Multiple alignment of the receptor biding motif at the amino acid level. Boxes highlight the key amino acid residues involved in ACE2 receptor binding. The nCoV-2019 and Pangolin-CoV strains are identical at all five amino acid residues, whereas RaTG13-CoV strain differs at four out of five compared to nCoV-2019. Labels: nCoV-2019 – outbreak strain; Pangolin-CoV – pangolin coronavirus identified in this study; RaTG13-CoV – 
       <em>R. affinis</em> coronavirus; Bat-CoV – bat coronavirus (MG772933.1)
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-6">The coronavirus spike protein is a trimeric protein that mediates virus entry into the host cell. It is composed of two domains: S1 and S2. The S1 domain contains two subdomains: S1-NTD and S1-CTD, also known as the receptor binding domain (RBD). The S2 domain encodes the stalk of the spike protein and is highly conserved across the SARS-like coronaviruses
     <sup>
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>
     </sup>. In order to infect a host cell, the RBD must first bind to a surface protein on the host cell. In the case of SARS-CoV and nCoV-2019, this is the angiotensin converting enzyme 2 (ACE2) receptor, expressed in human airway epithelia as well as lung parenchyma among other tissues. Following RBD binding, the S1/S2 junction must be cleaved by a surface protease
     <sup>
      <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>
     </sup>. Zhou et al. (2020)
     <sup>
      <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>
     </sup> have reported noticeable decreased amino acid homology between the RBD of the spike protein in RaTG13 and nCoV-2019. A further analysis of the RBD in RaTG13 showed the majority of the divergence was restricted to amino acid residues 435 to 510 (75% nucleotide identity and 78% amino acid identity). This region corresponds to the receptor binding motif (RBM), which is responsible for host specificity. Such divergence at the RBM could indicate the possibility of an alternate source for the RBM coding sequence in nCoV-2019. To explore this, we examined several viral metagenomic datasets for potential coronaviruses that could harbor RBMs closer to nCoV-2019. Using VirMAP
     <sup>
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>
     </sup>, we aimed to reconstruct coronavirus genomes to compare recovered RBM sequences to that of nCoV-2019.
    </p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Methods</h2>
    <div id="sec-3" class="subsection">
     <h3>Data used in the study</h3>
     <p id="p-7">Viral metagenomic datasets of hosts that could potentially harbor coronaviruses were downloaded from the NCBI BioProject database. These included PRJNA573298
      <sup>
       <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>
      </sup> (pangolin), PRJNA597258 (fruit bats) and PRJNA379515 (bats). The coronavirus genome (recovered from bat, GenBank accession number 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=MG772933&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F13%2F2020.02.07.939207.atom">MG772933</a>.1
      <sup>
       <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>
      </sup>) reported to be the most homologous to the nCoV-2019 outbreak strain prior to the publication the RaTG13 (GenBank accession number 
      <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;;access_num=MN996532&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F13%2F2020.02.07.939207.atom">MN996532</a>) was also downloaded.
     </p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>Analysis</h3>
     <p id="p-8">Sequences from each project were trimmed (minimum quality score 20 and minimum length 75bp) and filtered for Illumina adapter and PhiX presence using bbduk.sh v38.71
      <sup>
       <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>
      </sup>. Prior to analysis all sequences were filtered against a the relevant host reference genome (pangolin ManJav1.0, PRJNA256023 or bat mRhiFer1_v1.p, PRJNA489106) using bbmap.sh v38.71(k=15 and minid=0.9)
      <sup>
       <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>
      </sup>. After trimming and filtering, sequence reads were processed using VirMAP
      <sup>
       <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>
      </sup> to obtain and identify coronavirus genomes present in the viral metagenomic dataset. Two coronavirus genomes sharing high homology to nCoV-2019 were recovered from the pangolin dataset in two samples (SRR10168377 and SRR10168378). The two genomes were merged using the easymerge.pl subcommand from VirMAP to create the final pangolin-associated coronavirus (Pangolin-CoV) genome.
     </p>
     <p id="p-9">All further analysis was done using four coronaviruses genome sequences: Pangolin-CoV, RaTG13, Bat-CoV and nCoV-2019. They were pairwise aligned on NCBI using the blastn and blastx algorithms
      <sup>
       <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>,
       <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>
      </sup> and sequence homology plots were calculated using RDP4 beta 99
      <sup>
       <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>
      </sup>. Distances of the selected coronavirus genomes relative to nCov-2019 were calculated using a 300bp window sliding every 10bp across the entire genome length. Sequence homology plots were visualized using R version 3.6.2
      <sup>
       <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>
      </sup> and the ggplot2 visualization package
      <sup>
       <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>
      </sup>. The RBM segments as well as the Pro-Arg-Arg-Ala (PRRA) amino acid insertion region were excised from the full genome and pairwise aligned on NCBI using blastn and blastx. Multiple alignments of the both these regions were done using MAFFT v7.313
      <sup>
       <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>
      </sup> and visualized with MEGAX v10.1.7
      <sup>
       <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>
      </sup>.
     </p>
    </div>
   </div>
   <div class="section" id="sec-5">
    <h2 class="">Results</h2>
    <p id="p-10">Of the three viral metagenomic datasets inspected, only the pangolin dataset contained coronavirus genomes sharing significant homology with the outbreak strain. Two samples in this dataset yielded two partial coronavirus genomes that overlapped &gt;;8.4kb at &gt;;99% nucleotide identity with each other. After merging, the final Pangolin-CoV draft genome was 90.5% similar to nCoV-2019 (median nucleotide identity). 
     <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> shows the sequence homology of Pangolin-CoV, RaTG13-CoV and Bat-CoV relative to nCoV-2019 (
     <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a> shows the homology across the entire genome length). The multiple alignments across the RBM segments revealed an 89% nucleotide (
     <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>) and 98% amino acid identity (
     <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1C</a>) of the Pangolin-CoV compared to nCoV-2019. The sequence homology plots showed that the Pangolin-CoV was more similar to nCoV-2019 at the RBD region than the RaTG13-CoV and Bat-CoV (
     <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1B</a>). The multiple alignment comparing the PRRA insertion region at the S1/S2 junction shows the insertion present in nCoV-2019 and missing in the other three CoV genomes (
     <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).
    </p>
    <div id="F2" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.07.939207/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Multiple alignment at the amino acid level of PRRA insertion region and flanking areas across three coronavirus genomes relative to the outbreak strain (nCoV-2019). Box show the PRRA insertion present in nCoV-2019 and missing in the other three CoV genomes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2136161409" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Multiple alignment at the amino acid level of PRRA insertion region and flanking areas across three coronavirus genomes relative to the outbreak strain (nCoV-2019). Box show the PRRA insertion present in nCoV-2019 and missing in the other three CoV genomes.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.07.939207/F2.medium.gif" width="440" height="55"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.07.939207/F2.medium.gif" width="440" height="55"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.07.939207/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/13/2020.02.07.939207/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 2:</span>
      <p id="p-11" class="first-child">Multiple alignment at the amino acid level of PRRA insertion region and flanking areas across three coronavirus genomes relative to the outbreak strain (nCoV-2019). Box show the PRRA insertion present in nCoV-2019 and missing in the other three CoV genomes.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
   </div>
   <div class="section" id="sec-6">
    <h2 class="">Discussion</h2>
    <p id="p-12">Recent coronavirus zoonotic transmissions have involved non-bat intermediates. Indeed, both SARS
     <sup>
      <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>
     </sup> and MERS
     <sup>
      <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>
     </sup> are believed to have arisen from coronaviruses recombining in civet cats and camels, respectively. While early indications for nCoV-2019 showed that the virus may have jumped directly from bats to humans
     <sup>
      <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">2</a>
     </sup>, here we present evidence that nCoV-2019 likely arose from yet another recombined coronavirus in an intermediate host. A dataset describing the viral diversity in Malayan pangolins
     <sup>
      <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">3</a>
     </sup> yielded a coronavirus genome that shares 89% nucleotide (
     <a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1A</a> and 
     <a id="xref-fig-1-8" class="xref-fig" href="#F1">1B</a>) and 98% amino acid (
     <a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1C</a>) identity across the same RBM segment with nCoV-2019. Additionally, the RaTG13 genome only shares one out of five key amino acids involved in receptor binding motif
     <sup>
      <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">2</a>
     </sup>, whereas the pangolin coronavirus shares all five key residues (
     <a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1C</a>). It therefore appears that the RBM in nCoV-2019 was introduced during a recombination event between the strain of coronavirus found in the pangolin and RaTG13. Given the heavy selective pressure under which the RBM is constrained, it is unlikely that RaTG13 acquired the mutations found in the pangolin coronavirus genome through random chance. We posit that the homology at the RBM between nCoV-2019 and a pangolin coronavirus may have arisen from a recombination event based on sequence analysis presented herein.
    </p>
    <p id="p-13">Amino acid alignment between pangolin coronavirus and nCoV-2019 at the S1-NTD subdomain results in low sequence similarity. However, increased similarity is observed from the RBD through the S2 domain (
     <a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1A-B</a>). This suggests a possible recombination event between a bat coronavirus and the pangolin coronavirus at the S1-NTD and RBD junction. While there is high conservation at the amino-acid level between the pangolin coronavirus and nCoV-2019, this is not reflected in the nucleotide identity, suggesting that the proposed recombination event occurred in the more distant past, allowing subsequent time for genetic drift. One such mutation could be the insertion of the amino acid residues PRRA near the S1/S2 junction which induces a furin cleavage motif. The secondary requirement for entry into the host cell is proteolytic cleavage of the S1/S2 domains in order to mediate membrane fusion post ACE2 binding 
     <sup>
      <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">4</a>
     </sup>, and introduction of the furin cleavage motif may be another factor in nCoV-2019 being able to replicate in humans (
     <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>).
    </p>
    <p id="p-14">The main limitations of this analysis are the lack of additional viral datasets arising from pangolins or the availability of coronavirus genomes isolated from pangolins that could be used for multiple genome alignments with nCoV-2019. In addition, the current analysis only focuses on the spike protein of nCoV-2019 without exploring additional recombination events that may lead to changes in host tropism. Nonetheless, the identification of a possible recombination event in the host-receptor binding protein of nCoV-2019, suggesting an intermediate host, supports a call for continuous surveillance of species known to be infected by this group of viruses, including pangolins, as well as all small mammals sold in such wet markets as the Huanan Seafood Wholesale Market in Wuhan to better understand and characterize zoonotic transmission events.</p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgements</h2>
    <p id="p-15">Research reported in this publication was supported by the National Institute Of Allergy And Infectious Diseases of the National Institutes of Health under Award Number U19AI144297. The authors thank the Alkek foundation for ongoing support of the Baylor College of Medicine Alkek Center for Metagenomics and Microbiome Research. We thank P. Hotez, R. Atmar, and G. Metcalf for critical review of the manuscript.</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.939207v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yu</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">YM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Song</span>
          <span class="cit-name-given-names">ZG</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>
          <span class="cit-name-given-names">Y</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A new coronavirus associated with human respiratory disease in China</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-pub-date">2020</span>;
         <span class="cit-vol">453</span>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.939207v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">XL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">XG</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">W</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-pub-date">2020</span>;
         <span class="cit-vol">1221</span>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.939207v1.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">JP</span>
         </span>. 
         <span class="cit-article-title">Viral Metagenomics Revealed Sendai Virus and Coronavirus Infection of Malayan Pangolins (
          <em>Manis javanica</em>)
         </span>. 
         <abbr class="cit-jnl-abbrev">Viruses</abbr>
         <span class="cit-pub-date">2019</span>; 
         <span class="cit-vol">11</span>(
         <span class="cit-issue">11</span>): 
         <span class="cit-fpage">979</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D11%26rft.spage%253D979%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.939207v1.4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>
         <span class="cit-article-title">Structure, Function, and Evolution of Coronavirus Spike Proteins</span>. 
         <abbr class="cit-jnl-abbrev">Annual Review of Virology</abbr>
         <span class="cit-pub-date">2016</span>;
         <span class="cit-vol">3</span>(
         <span class="cit-issue">1</span>): 
         <span class="cit-fpage">237</span>–
         <span class="cit-lpage">261</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BVirology%26rft.volume%253D3%26rft.spage%253D237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.939207v1.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ajami</span>
          <span class="cit-name-given-names">NJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wong</span>
          <span class="cit-name-given-names">MC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ross</span>
          <span class="cit-name-given-names">MC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lloyd</span>
          <span class="cit-name-given-names">RE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Petrosino</span>
          <span class="cit-name-given-names">JF</span>
         </span>. 
         <span class="cit-article-title">Maximal viral information recovery from sequence data using VirMAP</span>. 
         <abbr class="cit-jnl-abbrev">Nature Communications</abbr>
         <span class="cit-pub-date">2018</span>; 
         <span class="cit-vol">9</span>: 
         <span class="cit-fpage">3205</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D9%26rft.spage%253D3205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.939207v1.6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ai</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">He</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ye</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Genomic characterization and infectivity of a novel SARS-like coronavirus in Chinese bats</span>. 
         <abbr class="cit-jnl-abbrev">Emerging microbes and infections</abbr>. 
         <span class="cit-pub-date">2018</span>; 
         <span class="cit-vol">7</span>(
         <span class="cit-issue">1</span>): 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">10</span>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEmerging%2Bmicrobes%2Band%2Binfections%26rft.volume%253D7%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-book" id="cit-2020.02.07.939207v1.7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bushnell</span>
          <span class="cit-name-given-names">B.</span>
         </span>
         <span class="cit-source">BBMap Short Read Aligner</span>. 
         <span class="cit-publ-loc">Berkeley, California</span>: 
         <span class="cit-publ-name">University of California</span>; 
         <span class="cit-pub-date">2016</span>; sourceforge.net/projects/bbmap/
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.939207v1.8" data-doi="10.1006/jmbi.1990.9999">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Altschul</span>
          <span class="cit-name-given-names">SF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gish</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Miller</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Myers</span>
          <span class="cit-name-given-names">EW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lipman</span>
          <span class="cit-name-given-names">DJ</span>
         </span>. 
         <span class="cit-article-title">Basic local alignment search tool</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Molecular Biology</abbr>
         <span class="cit-pub-date">1990</span>; 
         <span class="cit-vol">215</span>:
         <span class="cit-fpage">403</span>–
         <span class="cit-lpage">410</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=2231712&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F13%2F2020.02.07.939207.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1990ED16700008&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.939207v1.9" data-doi="10.1186/1471-2105-10-421">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Camacho</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Coulouris</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Avagyan</span>
          <span class="cit-name-given-names">V</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ma</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Papadopoulos</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bealer</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Madden</span>
          <span class="cit-name-given-names">TL</span>
         </span>. 
         <span class="cit-article-title">BLAST+: architecture and applications</span>. 
         <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr>
         <span class="cit-pub-date">2008</span>; 
         <span class="cit-vol">10</span>:
         <span class="cit-fpage">421</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D10%26rft.spage%253D421%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-421&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.939207v1.10">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Martin</span>
          <span class="cit-name-given-names">DP</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Murrell</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Golden</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Khoosal</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Muhire</span>
          <span class="cit-name-given-names">B.</span>
         </span>
         <span class="cit-article-title">RDP4: Detection and analysis of recombination patterns in virus genomes</span>. 
         <abbr class="cit-jnl-abbrev">Virus Evolution</abbr>
         <span class="cit-pub-date">2015</span>; 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">5</span>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.02.07.939207v1.11">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">R Core Team</span> (
         <span class="cit-pub-date">2019</span>). 
         <span class="cit-source">R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria</span>. URL 
         <a href="https://www.R-project.org/">https://www.R-project.org/</a>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-book" id="cit-2020.02.07.939207v1.12">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-given-names">H.</span>
          <span class="cit-name-surname">Wickham</span>
         </span>. 
         <span class="cit-source">ggplot2: Elegant Graphics for Data Analysis</span>. 
         <span class="cit-publ-loc">Springer-Verlag New York</span>, 
         <span class="cit-pub-date">2016</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.939207v1.13" data-doi="10.1093/molbev/mst010">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Katoh</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Standley</span>
          <span class="cit-name-given-names">DM</span>
         </span>. 
         <span class="cit-article-title">MAFFT Multiple Sequence Alignment Software Version 7: Improvements in Performance and Usability</span>. 
         <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr>
         <span class="cit-pub-date">2013</span>; 
         <span class="cit-vol">30</span>(
         <span class="cit-issue">4</span>): 
         <span class="cit-fpage">772</span>–
         <span class="cit-lpage">780</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst010%26rft_id%253Dinfo%253Apmid%252F23329690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/molbev/mst010&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23329690&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F13%2F2020.02.07.939207.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000317002300004&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.939207v1.14" data-doi="10.1093/molbev/msy096">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kumar</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Stecher</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Knyaz</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tamura</span>
          <span class="cit-name-given-names">K.</span>
         </span>
         <span class="cit-article-title">MEGA X: Molecular Evolutionary Genetics Analysis across computing platforms</span>. 
         <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr>
         <span class="cit-pub-date">2018</span>; 
         <span class="cit-vol">35</span>:
         <span class="cit-fpage">1547</span>–
         <span class="cit-lpage">1549</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.volume%253D35%26rft.spage%253D1547%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsy096%26rft_id%253Dinfo%253Apmid%252F29722887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/molbev/msy096&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=29722887&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F13%2F2020.02.07.939207.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.939207v1.15" data-doi="10.1128/JVI.01048-15">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lau</span>
          <span class="cit-name-given-names">SK</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Feng</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Severe Acute Respiratory Syndrome (SARS) Coronavirus ORF8 Protein Is Acquired from SARS-Related Coronavirus from Greater Horseshoe Bats through Recombination</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>
         <span class="cit-pub-date">2015</span>;
         <span class="cit-vol">89</span>(
         <span class="cit-issue">20</span>):
         <span class="cit-fpage">10532</span>–
         <span class="cit-lpage">10547</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01048-15%26rft_id%253Dinfo%253Apmid%252F26269185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4OS8yMC8xMDUzMiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzEzLzIwMjAuMDIuMDcuOTM5MjA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.939207v1.16">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shen</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gu</span>
          <span class="cit-name-given-names">X.</span>
         </span>
         <span class="cit-article-title">Evolutionary Dynamics of MERS-CoV: Potential Recombination, Positive Selection and Transmission</span>. 
         <abbr class="cit-jnl-abbrev">Scientific Reports</abbr>
         <span class="cit-pub-date">2016</span>; 
         <span class="cit-vol">6</span>:
         <span class="cit-fpage">25049</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScientific%2BReports%26rft.volume%253D6%26rft.spage%253D25049%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
