<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">PLoS Pathog</span><span class="iso-abbrev" title="iso-abbrev">PLoS Pathog</span><span class="publisher-id" title="publisher-id">plos</span><span class="pmc" title="pmc">plospath</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">PLoS Pathogens</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1553-7366</span><span class="issn-epub" tagx="issn" title="issn-epub">1553-7374</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, CA USA</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6422253">6422253</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.ppat.1007601">10.1371/journal.ppat.1007601</a></span><span class="publisher-id" title="publisher-id">PPATHOGENS-D-15-02950</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Genetics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Gene expression</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Gene regulation</span>: <span class="subject" title="subject">Small interfering RNAs</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biochemistry</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Nucleic acids</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">RNA</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Non-coding RNA</span>: <span class="subject" title="subject">Small interfering RNAs</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">RNA viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Orthomyxoviruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Influenza viruses</span>: <span class="subject" title="subject">Influenza A virus</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medical microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Orthomyxoviruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Influenza viruses</span>: <span class="subject" title="subject">Influenza A virus</span></div> </div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and health sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathology and laboratory medicine</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Orthomyxoviruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Influenza viruses</span>: <span class="subject" title="subject">Influenza A virus</span></div> </div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Orthomyxoviruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Influenza viruses</span>: <span class="subject" title="subject">Influenza A virus</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Bioengineering</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biotechnology</span>: <span class="subject" title="subject">Small Molecules</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Engineering and Technology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Bioengineering</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biotechnology</span>: <span class="subject" title="subject">Small Molecules</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Physical Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Chemistry</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Chemical Compounds</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organic Compounds</span>: <span class="subject" title="subject">Small Molecules</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Physical Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Chemistry</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organic Chemistry</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organic Compounds</span>: <span class="subject" title="subject">Small Molecules</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Virology</span>: <span class="subject" title="subject">Viral Replication</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Control</span>: <span class="subject" title="subject">Antimicrobial Resistance</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pharmacology</span>: <span class="subject" title="subject">Antimicrobial Resistance</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pharmacology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Drugs</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Antimicrobials</span>: <span class="subject" title="subject">Antivirals</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Control</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Antimicrobials</span>: <span class="subject" title="subject">Antivirals</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Virology</span>: <span class="subject" title="subject">Antivirals</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computer and Information Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Network Analysis</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Signaling Networks</span>: <span class="subject" title="subject">Wireless Sensor Networks</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">RNA viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Rhabdoviruses</span>: <span class="subject" title="subject">Vesicular Stomatitis Virus</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medical Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span>: <span class="subject" title="subject">Vesicular Stomatitis Virus</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathology and Laboratory Medicine</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span>: <span class="subject" title="subject">Vesicular Stomatitis Virus</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral Pathogens</span>: <span class="subject" title="subject">Vesicular Stomatitis Virus</span></div> </div> </div> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
RNAi-based small molecule repositioning reveals clinically approved urea-based kinase inhibitors as broadly active antivirals</div> <div class="alt-title" title="alt-title">
UBKI are broadly active antivirals</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lesch</span><span class="given-names" tagx="given-names" title="given-names">Markus</span></span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Validation</span><span class="role" tagx="role" title="role">Visualization</span><span class="role" tagx="role" title="role">Writing – original draft</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff001"><sup>1</sup></a><a href="#aff002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luckner</span><span class="given-names" tagx="given-names" title="given-names">Madlen</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#aff003"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyer</span><span class="given-names" tagx="given-names" title="given-names">Michael</span></span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weege</span><span class="given-names" tagx="given-names" title="given-names">Friderike</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#aff001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gravenstein</span><span class="given-names" tagx="given-names" title="given-names">Isabella</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#aff002"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raftery</span><span class="given-names" tagx="given-names" title="given-names">Martin</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Resources</span><a href="#aff004"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sieben</span><span class="given-names" tagx="given-names" title="given-names">Christian</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Resources</span><a href="#aff003"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin-Sancho</span><span class="given-names" tagx="given-names" title="given-names">Laura</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#aff001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Imai-Matsushima</span><span class="given-names" tagx="given-names" title="given-names">Aki</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><a href="#aff001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Welke</span><span class="given-names" tagx="given-names" title="given-names">Robert-William</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Resources</span><a href="#aff003"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frise</span><span class="given-names" tagx="given-names" title="given-names">Rebecca</span></span><span class="role" tagx="role" title="role">Investigation</span><a href="#aff005"><sup>5</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barclay</span><span class="given-names" tagx="given-names" title="given-names">Wendy</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Resources</span><a href="#aff005"><sup>5</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schönrich</span><span class="given-names" tagx="given-names" title="given-names">Günther</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Resources</span><a href="#aff004"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herrmann</span><span class="given-names" tagx="given-names" title="given-names">Andreas</span></span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Resources</span><a href="#aff003"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyer</span><span class="given-names" tagx="given-names" title="given-names">Thomas F.</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Funding acquisition</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Project administration</span><span class="role" tagx="role" title="role">Resources</span><span class="role" tagx="role" title="role">Supervision</span><span class="role" tagx="role" title="role">Writing – original draft</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff001"><sup>1</sup></a><a href="#aff002"><sup>2</sup></a><a href="#cor001">*</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-2187-2057</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karlas</span><span class="given-names" tagx="given-names" title="given-names">Alexander</span></span><span class="role" tagx="role" title="role">Conceptualization</span><span class="role" tagx="role" title="role">Data curation</span><span class="role" tagx="role" title="role">Formal analysis</span><span class="role" tagx="role" title="role">Investigation</span><span class="role" tagx="role" title="role">Methodology</span><span class="role" tagx="role" title="role">Project administration</span><span class="role" tagx="role" title="role">Supervision</span><span class="role" tagx="role" title="role">Validation</span><span class="role" tagx="role" title="role">Visualization</span><span class="role" tagx="role" title="role">Writing – original draft</span><span class="role" tagx="role" title="role">Writing – review &amp;amp; editing</span><a href="#aff001"><sup>1</sup></a><a href="#aff002"><sup>2</sup></a><a href="#currentaff001"><sup>¤</sup></a><a href="#cor001">*</a></span></div><span class="citation_author_institution" id="aff001">[1], <span class="addr-line" title="addr-line">Department of Molecular Biology, Max Planck Institute for Infection Biology, Berlin, Germany</span></span><span class="citation_author_institution" id="aff002">[2], <span class="addr-line" title="addr-line">Steinbeis Innovation Center for Systems Biomedicine, Falkensee, Germany</span></span><span class="citation_author_institution" id="aff003">[3], <span class="addr-line" title="addr-line">Group of Molecular Biophysics, Department of Biology, Humboldt-Universität zu Berlin, Berlin, Germany</span></span><span class="citation_author_institution" id="aff004">[4], <span class="addr-line" title="addr-line">Institute of Virology, Charité University Medicine, Berlin, Germany</span></span><span class="citation_author_institution" id="aff005">[5], <span class="addr-line" title="addr-line">Section of Virology, Department of Medicine, Imperial College London, St Mary's Campus, London, United Kingdom</span></span><div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fernandez-Sesma</span><span class="given-names" tagx="given-names" title="given-names">Ana</span></span><span class="role" tagx="role" title="role">Editor</span><a href="#edit1" /></span></div><span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">Icahn School of Medicine at Mount Sinai, UNITED STATES</span></span><div class="author-notes" title="author-notes"> <div class="fn-type-COI-statement" title="COI-statement"> <p>The authors have declared that no competing interests exist.</p> </div> <div class="fn-type-current-aff" title="current-aff"><span class="label" tagx="label" title="label">¤</span><p>Current address: Viral Vaccines Development, ProBioGen AG, Berlin, Germany</p> </div> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">meyer@mpiib-berlin.mpg.de</span> (TFM); <span class="email" tagx="email" title="email">karlas@mpiib-berlin.mpg.de</span> (AK)</div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-3-3</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2019-3</span></span><span class="volume" tagx="volume" title="volume">15</span><span class="issue" tagx="issue" title="issue">3</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1007601</span><span class="history" title="history"><span class="received" title="received">received: 2015-12-21</span><span class="accepted" title="accepted">accepted: 2019-1-29</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Lesch et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2019</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</span></span></div><a class="self-uri" href="ppat.1007601.pdf" title="self-uri">ppat.1007601.pdf</a><div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Influenza viruses (IVs) tend to rapidly develop resistance to virus-directed vaccines and common antivirals targeting pathogen determinants, but novel host-directed approaches might preclude resistance development. To identify the most promising cellular targets for a host-directed approach against influenza, we performed a comparative small interfering RNA (siRNA) loss-of-function screen of IV replication in A549 cells. Analysis of four different IV strains including a highly pathogenic avian H5N1 strain, an influenza B virus (IBV) and two human influenza A viruses (IAVs) revealed 133 genes required by all four IV strains. According to gene enrichment analyses, these strain-independent host genes were particularly enriched for nucleocytoplasmic trafficking. In addition, 360 strain-specific genes were identified with distinct patterns of usage for IAVs versus IBV and human versus avian IVs. The strain-independent host genes served to define 43 experimental and otherwise clinically approved drugs, targeting reportedly fourteen of the encoded host factors. Amongst the approved drugs, the urea-based kinase inhibitors (UBKIs) regorafenib and sorafenib exhibited a superior therapeutic window of high IV antiviral activity and low cytotoxicity. Both UBKIs appeared to block a cell signaling pathway involved in IV replication after internalization, yet prior to vRNP uncoating. Interestingly, both compounds were active also against unrelated viruses including cowpox virus (CPXV), hantavirus (HTV), herpes simplex virus 1 (HSV1) and vesicular stomatitis virus (VSV) and showed antiviral efficacy in human primary respiratory cells. An <i>in vitro</i> resistance development analysis for regorafenib failed to detect IV resistance development against this drug. Taken together, the otherwise clinically approved UBKIs regorafenib and sorafenib possess high and broad-spectrum antiviral activity along with substantial robustness against resistance development and thus constitute attractive host-directed drug candidates against a range of viral infections including influenza.</p> </div> <div class="abstract" title="abstract"> <div class="title" tagx="title" title="title">
Author summary</div> <p>Conventional medications against influenza infections, including vaccination and antiviral drug therapy, are targeted against viral determinants–an approach collectively referred to as pathogen-directed. However, influenza viruses mutate fast and quickly develop resistance to these pathogen-directed treatments. An alternative, yet not well established, is to block host cellular molecules required by the virus to successfully multiply. Such a host-directed approach is anticipated to be more robust against the development of drug resistance. This notion is founded on the different modes of action of the two principal approaches: Virus-directed therapeutics target the virus itself. Thus, just a single mutation could abrogate sensitivity to a virus-directed therapeutic. In contrast, it is unlikely that viruses can easily circumvent a pharmacological blockage of a cellular factor by means of just a few mutations. Instead, the virus needs to either exploit an immediate parallel cellular pathway or adjust its replication cycle to a different cellular factor–the latter being a process likely to require multiple mutations, if possible at all. To identify the most promising targets for a host-directed therapy, we performed a small interfering RNA (siRNA) screen with four different influenza virus strains using a lung epithelial cell line. Subsequently, we tested a series of drugs, specific for the products of the genes that are required for replication of all four influenza virus strains tested. Regorafenib and sorafenib, two chemically related urea-based kinase inhibitors already clinically approved for cancer treatment, turned out to be effective inhibitors of all influenza viruses and displayed low cytotoxicity. These drugs blocked viral replication at an early stage of the life cycle not only in cell lines but also in human primary respiratory cells. Moreover, these drugs exhibited high efficacy even against unrelated viruses. In addition, no development of resistance was observed against regorafenib, which was used in an <i>in vitro</i> assay representatively of urea-based kinase inhibitors. Our results suggest that regorafenib and sorafenib are promising drug candidates for a host-directed therapy of influenza and other viral infections.</p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">BMBF</span></div><span class="award-id" tagx="award-id" title="award-id">FK0316180C</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meyer</span><span class="given-names" tagx="given-names" title="given-names">Thomas F.</span></span></span></div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100001659</span><span class="institution" title="institution">Deutsche Forschungsgemeinschaft</span></div> </div><span class="award-id" tagx="award-id" title="award-id">SFB740HE3763/15</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herrmann</span><span class="given-names" tagx="given-names" title="given-names">Andreas</span></span></span></div> <div class="funding-statement" title="funding-statement">
This work was supported by the German Ministry of Education and Research through the eBio project ViroSign (FK0316180C) to TFM and AH, and by the German Research Foundation (SFB740 HE3763/15) to AH. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="Data Availability" value="All relevant data are within the paper and its Supporting Information files." /> </div> </div> <div class="title" tagx="title" title="title">
Data Availability</div> <p>All relevant data are within the paper and its Supporting Information files.</p> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
Introduction</div> <p>IVs are the cause of seasonal epidemics with up to 500,000 fatalities per year [<a href="#ppat.1007601.ref001">1</a>] as well as occasional pandemics with increased infection and mortality rates [<a href="#ppat.1007601.ref002">2</a>]. Epidemics are usually caused by human IAV strains of the subtypes H1N1 and H3N2, as well as IBVs. The source for influenza pandemics has in the past often been provided by avian IAVs that managed to adapt to humans as a host. Current preventive and therapeutic measures target viral proteins and include vaccination as well as direct-acting antivirals (DAAs). Unfortunately, a single mutation of the targeted site can lower the efficacy of a virus-directed drug by two to three orders of magnitude. Consequently, isolates resistant against all approved DAAs have been identified [<a href="#ppat.1007601.ref003">3</a>–<a href="#ppat.1007601.ref005">5</a>]. Thus, there is an urgent need for new anti-IV drugs which are less prone to the development of resistance.</p> <p>IAVs and IBVs encode only 10 to 16 proteins and rely heavily on various host cellular functions to multiply. As cellular proteins are integral parts of the viral life cycle, they also represent medical intervention points. The CCR5-inhibitor maraviroc, which is used in the therapy of human immunodeficiency virus infection, provides the proof-of-concept for host-directed therapy of an infectious disease [<a href="#ppat.1007601.ref006">6</a>]. The expected advantage of targeting cellular factors over viral factors is that resistance is less likely to develop: even if the virus mutates, the cellular protein will remain blocked. The only way for the virus to develop drug resistance is to utilize alternative cellular pathways. Such alternatives exist for certain pathways, highlighted by the ability of IVs to adapt to different types of sialic acids as receptors [<a href="#ppat.1007601.ref005">5</a>] or different importins for nuclear import of viral ribonucleoprotein particles (vRNPs) [<a href="#ppat.1007601.ref007">7</a>, <a href="#ppat.1007601.ref008">8</a>]. However, there might also be cellular bottlenecks to which development of resistance is impeded. One example for a drug targeting such a bottleneck is the microRNA-122 inhibitor miravirsen, which underwent clinical trials for treatment of hepatitis C virus infection [<a href="#ppat.1007601.ref009">9</a>], and for which a very high barrier against development of resistance was observed [<a href="#ppat.1007601.ref010">10</a>]. To systematically identify the human genes exploited by IVs, several loss-of-function screens have been conducted by independent laboratories including our own [<a href="#ppat.1007601.ref011">11</a>–<a href="#ppat.1007601.ref013">13</a>]. In these studies, more than 1,000 genes have been proposed to be growth relevant. However, mainly IAVs of subtype H1N1, such as A/WSN/1933(H1N1) (WSN) and A/Puerto Rico/8/1934(H1N1), have been used in the published screens. Thus, it is unclear to which extent the published genes are relevant for other IV types, such as H3N2 IAV, avian-adapted IAVs, or IBVs, and which genes are best suited for a pan-specific host-directed therapy of influenza.</p> <p>Here, we performed a large-scale, comparative small interfering RNA (siRNA) screen with a human H1N1 IAV (WSN), a human H3N2 IAV (A/Panama/2007/1999(H3N2), PAN), an avian H5N1 IAV (A/Vietnam/1203/2004(H5N1), VN), and an IBV (B/Thuringia/02/2006xB/Vienna/33/2006, THW) to identify suitable targets for a host-directed influenza therapy.</p> <p>Next, we screened drugs targeting genes required for replication of all four viruses for antiviral activity. Costs for development of a new drug are tremendously high. A more cost-efficient way to generate drugs for a new indication is to repurpose existing drugs approved for other indications. Successful examples of repurposed drugs include bupropion, chlorpromazine, and celecoxib [<a href="#ppat.1007601.ref014">14</a>]. We identified the UBKIs regorafenib [<a href="#ppat.1007601.ref015">15</a>] and sorafenib [<a href="#ppat.1007601.ref016">16</a>]–approved for specific cancer indications–as highly promising candidates for a host-directed therapy of influenza.</p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
Results</div> <div class="comparativesirnascreenidentifiesivstrain-independenthostfactors" title="sec"> <div class="title" tagx="title" title="title">
Comparative siRNA screen identifies IV strain-independent host factors</div> <p>The siRNA screen was set up as shown in <a href="#ppat.1007601.g001">Fig 1A</a>. Based on previous loss-of-function studies with IVs, we designed an siRNA library targeting 128 genes identified to be growth relevant in at least two out of a panel of six published screens [<a href="#ppat.1007601.ref012">12</a>], the 461 genes identified in the primary screen of our own group [<a href="#ppat.1007601.ref011">11</a>], and 717 genes associated with the human kinome, since kinase genes were the most overrepresented gene category in several loss-of-function-screens [<a href="#ppat.1007601.ref017">17</a>]. In total, this library targeted 1,208 genes with 3,482 siRNAs (Supplementary <a href="#ppat.1007601.s013">S1 Table</a>). We first screened the library for immunogenicity and cytotoxicity to exclude siRNAs which induce an interferon (IFN) response and cause cell death, respectively. We identified 678 cytotoxic siRNAs, but no immunogenic siRNAs. In total, we excluded 211 genes from further analyses due to the cytotoxicity of the corresponding siRNAs, leaving 997 genes. Subsequently, the library was screened for production of infectious virus progeny. First, the raw data of the virus replication screens were normalized to a non-targeting siRNA and an siRNA targeting a proviral gene (IAV: viral NP gene, IBV: cellular gene XPO1). Then, the normalized data for all siRNAs targeting a given gene were summarized by the collective mean and the collective strictly standardized mean difference (cSSMD), as described previously [<a href="#ppat.1007601.ref018">18</a>]. Employing collective mean &amp;gt; 0.5 and cSSMD ≥ 1 as cut-off criteria, we identified between 258 (WSN) and 468 (VN) proviral genes for each virus (<a href="#ppat.1007601.g001">Fig 1B</a>). A set of 133 genes was identified as proviral in all four virus replication screens (strain-independent genes) (<a href="#ppat.1007601.g001">Fig 1C</a>). Based on the average effect of the siRNAs tested, knockdown of thirteen genes impaired viral replication by an order of magnitude or more (<a href="#ppat.1007601.g001">Fig 1D</a>). We analyzed these strain-independent genes for enrichment of gene groups using the Database for Annotation, Visualization and Integrated Discovery (DAVID) [<a href="#ppat.1007601.ref019">19</a>] (<a href="#ppat.1007601.g001">Fig 1E</a>) and/or Ingenuity Pathway Analysis (IPA, Ingenuity H Systems; <a href="http://www.ingenuity.com/">http://www.ingenuity.com</a>) (<a href="#ppat.1007601.g001">Fig 1F</a>). We combined the results depicted in <a href="#ppat.1007601.g001">Fig 1D–1F</a> and visualized them as interaction map (<a href="#ppat.1007601.s001">S1 Fig</a>). Taking these results together, the nucleocytoplasmic transport appears to be the cellular process which is most sensitive to perturbation with respect to replication across all IV strains tested. Consequently, nucleocytoplasmic trafficking likely represents an attractive target for the development of host-directed therapeutics. Further highly sensitive processes include cell cycle regulation, RNA splicing, translation, and tRNA charging. Interestingly, the latter has not been identified as enriched in previous studies. The full results of the screens on siRNA and gene level, as well as the gene group enrichment analyses, are shown in Supplementary <a href="#ppat.1007601.s014">S2</a>–<a href="#ppat.1007601.s017">S5</a> Tables. Of note, we did not identify any factor which was antiviral for all four strains, probably because our gene set was selected to identify proviral genes.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1007601.g001"><span class="label" tagx="label" title="label">Fig 1</span></a></div>   <div class="title" tagx="title" title="title">
Result of the siRNA screen.</div> <p>(A) Setup of the siRNA screening campaign. (B) Classification of the 997 genes investigated with the four IV strains. (C) Venn diagram of genes required for replication of the individual viruses. (D) Virus titers relative to negative control upon knockdown of genes most strictly required for IV replication. Data represent the mean of all siRNAs targeting the specified gene. Data are from the siRNA screen. Positive control: IAV segment 5 (WSN, PAN, VN), XPO1 (THW). Negative control: Non-targeting scrambled sequence siRNA. (E) The genes required for replication of all four IV strains were subjected to gene group enrichment analysis using DAVID functional annotation clustering. Data represent the geometric means of the clusters’ p�?values for the top 10 ranking clusters. (F) The same data as in (E) were analyzed using IPA for enrichment of canonical pathways. Data represent the pathways’ p�?values for the top 10 ranking pathways.</p>   </div> </div> <div class="comparativesirnascreenidentifiesstrain-specifichost-factorsandcellularprocesses" title="sec"> <div class="title" tagx="title" title="title">
Comparative siRNA screen identifies strain-specific host-factors and cellular processes</div> <p>Although the primary aim of this study was to identify genes universally required by all prototypes of IVs, we also searched for entities differentially required between the four viral strains. We therefore analyzed the data sets with a hierarchical statistical model based on linear mixed effects. Subsequently, we filtered the results and obtained 360 strain-specific genes. By clustering these genes we found four clusters (<a href="#ppat.1007601.s002">S2A Fig</a>). Genes in cluster 1 were especially relevant for the avian IAV VN but much less for the human H1N1 IAV WSN (<a href="#ppat.1007601.g002">Fig 2A</a>). The genes in cluster 2 were especially relevant for the IBV strain THW and more or less dispensable for WSN (<a href="#ppat.1007601.g002">Fig 2B</a>). Genes in cluster 3 were especially relevant for VN and the human H3N2 IAV PAN but much less for THW (<a href="#ppat.1007601.g002">Fig 2C</a>). Genes in cluster 4 were most essential to replication of THW but much less for VN (<a href="#ppat.1007601.g002">Fig 2D</a>). Twenty-one genes could not be assigned to any cluster (<a href="#ppat.1007601.s002">S2B Fig</a>). Taking the results from the cluster analysis of the strain-specific genes together, most of the differences were identified between the IAVs and the IBV, or the human IVs and the avian IV.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1007601.g002"><span class="label" tagx="label" title="label">Fig 2</span></a></div>   <div class="title" tagx="title" title="title">
Several genes are strain-specifically required.</div> <p>Strain-specific genes were identified by mixed effects analysis and clustered using the CLICK algorithm [<a href="#ppat.1007601.ref079">79</a>]. Data represent the mean of the normalized viral load for the siRNAs targeting the individual genes. Data analyzed are from the screen outlined in <a href="#ppat.1007601.g001">Fig 1A</a>.</p>   </div> <p>We extended our search from differentially required genes to differentially required pathways, cellular compartments, and molecular functions. For this, we first queried the Gene Ontology database [<a href="#ppat.1007601.ref020">20</a>], the Kyoto Encyclopedia of Genes and Genomes database [<a href="#ppat.1007601.ref021">21</a>], and the Reactome database [<a href="#ppat.1007601.ref022">22</a>] for terms represented by ≥ 2 tested genes and again identified terms with strain-specific effects by a mixed effects model. Overall, 3,235 terms were represented by our siRNA library and 884 of these displayed significantly different effects on the four viral strains. Two processes for which the avian IV strain VN was less sensitive than the three IAVs were nuclear factor kappa-light-chain-enhancer of activated B cells (NF-кB) activation (<a href="#ppat.1007601.s003">S3A Fig</a> and <a href="#ppat.1007601.s018">S6 Table</a>) and regulation of RNA splicing (<a href="#ppat.1007601.s003">S3B Fig</a> and <a href="#ppat.1007601.s019">S7 Table</a>). Another cellular function with strain-dependent relevance is the cellular RNA polymerase activity, for which interference affected the IBV strain the most and the avian IV the least (<a href="#ppat.1007601.s003">S3C Fig</a> and <a href="#ppat.1007601.s020">S8 Table</a>). In summary, we often observed strain-specific host cell genes or processes to differ between IAVs and the IBV, or between human IVs and the avian IV. The full list of differentially required genes and gene ontologies and pathways can be found in <a href="#ppat.1007601.s021">S9</a> and <a href="#ppat.1007601.s022">S10</a> Tables.</p> </div> <div class="smallmoleculescreenidentifiestheurea-basedkinaseinhibitorsregorafenibandsorafenibasdrugssuitableforrepurposingforinfluenzatreatment" title="sec"> <div class="title" tagx="title" title="title">
Small molecule screen identifies the urea-based kinase inhibitors regorafenib and sorafenib as drugs suitable for repurposing for influenza treatment</div> <p>To identify drugs suitable for repositioning towards a host-directed anti-influenza therapy, we screened the 133 strain-independent pro-IV genes for approved drugs or drugs in clinical trials, using the Ingenuity database, identifying 41 drugs targeting 14 genes (<a href="#ppat.1007601.t001">Table 1</a>). We screened 39 of these drugs and four related compounds for antiviral activity and cytotoxicity in A549 cells. Seventeen drugs inhibited multiple round replication of WSN with an IC<sub>50</sub> &amp;lt; 10 μM (Supplementary <a href="#ppat.1007601.s023">S11 Table</a>). For fourteen of these highly active compounds, the selectivity index (SI) was &amp;gt; 10, which we considered a useful therapeutic window. Three of these drugs targeted the gene FLT4, which was also among the top three hits in the siRNA screen (<a href="#ppat.1007601.g001">Fig 1D</a>). Therefore, we decided to focus on two of the FLT4 inhibitors—regorafenib and sorafenib—in more detail. We determined the SIs of regorafenib and sorafenib to be 16 and 82, respectively (<a href="#ppat.1007601.g003">Fig 3A and 3B</a>), when adding the agents 2 h prior to infection. After quantifying the dose dependency of the antiviral effect of regorafenib and sorafenib, we investigated the time dependency of both compounds, employing the highest non-cytotoxic dose (3 μM each). While the virus titer in the supernatant of DMSO-treated cells increased up to 8 x 10<sup>6</sup> PFU/ml until 48 h post infection (p.i.), we could not detect virus in the supernatant of regorafenib- or sorafenib-treated cells (<a href="#ppat.1007601.g003">Fig 3C</a>). Subsequently, we compared preventive (2 h prior to infection) and therapeutic (4 h post infection) application of both drugs (<a href="#ppat.1007601.g003">Fig 3D</a>). Using the preventive approach, no virus could be detected with either of the two compounds. When applying the drugs therapeutically, a more than 10<sup>9</sup>-fold reduction of the viral load was observed with sorafenib, and no virus could be detected in the supernatant of regorafenib-treated cells.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1007601.g003"><span class="label" tagx="label" title="label">Fig 3</span></a></div>   <div class="title" tagx="title" title="title">
Several clinically approved and experimental drugs block multiple round and primary infection of IV.</div> <p>(A &amp;amp; B) For determination of virus replication, A549 cells were pre-treated with small molecules at different concentrations for 2 h. Cells were infected with IV strain WSN and cultivated for 36 h in presence of small molecules. Finally, virus titers in supernatants were determined and the infection rate was selected as read-out. IC<sub>50</sub>: half maximal inhibitory concentration of drug in virus replication assay. For determination of cell viability, A549 cells were cultivated for 36 h in presence of small molecules at different concentrations prior to conduction of WST-1 assay. Data represent signal in WST-1 assay relative to the vehicle control expressed as mean ± SEM of n = 3 technical replicates. CC<sub>50</sub>: half maximal inhibitory concentration in WST-1 assay. (A) Regorafenib. (B) Sorafenib. (C) A549 cells were pre-treated with small molecules at non-toxic concentrations (regorafenib &amp;amp; sorafenib, 3 μM; zanamivir, 1 μM) for 2 h. Cells were infected and cultivated for 48 h in presence of small molecules. Virus load was assessed in 12-hour intervals by plaque assay. Data represent mean ± SD of technical replicates. l.o.d.: limit of detection. Log-transformed virus titer for zanamivir treatment was compared to control using multiple linear regression. The titer for DMSO exceeds the one for zanamivir treatment significantly by a factor of 10<sup>4</sup> (p &amp;lt; 0.001; 95%CI: 10<sup>2</sup> to 10<sup>6</sup>). (D) A549 cells were treated with small molecules (same doses as in C) either starting 2 h prior to infection (preventive, black bars) or 4 h post-infection (therapeutic, grey bars) with WSN. Infected cells were cultivated for 36 h in presence of small molecules and virus load was assessed by plaque assay. Data represent mean ± SD of technical replicates. Log-transformed virus titer data were analysed with a linear model. The titer for preventive treatment exceeds the titer for therapeutic treatment (p = 0.041). It was lower for zanamivir treatment compared to DMSO (p ≤ 0.001). For preventive treatment with sorafenib and for both treatments with regorafenib the titer was below the limit of detection (l.o.d). Please note that zanamivir was applied at suboptimal concentration in (C) and (D).</p>   </div> <div class="table-wrap_UNKNOWN" id="ppat.1007601.t001" orientation="portrait" position="float"><a href="https://dx.doi.org/10.1371/journal.ppat.1007601.t001">10.1371/journal.ppat.1007601.t001</a><span class="label" tagx="label" title="label">Table 1</span> <div class="title" tagx="title" title="title">
Druggable genes required for IV replication.</div>  <div class="alternatives" title="alternatives"> <div class="graphic_UNKNOWN" id="ppat.1007601.t001g" /> <table frame="hsides" rules="groups"> <div class="colgroup_UNKNOWN" span="1"> <div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" align="left" valign="middle" span="1" /></div>  <thead> <tr> <th align="justify" rowspan="1" colspan="1">GeneID</th> <th align="justify" rowspan="1" colspan="1">GeneSymbol</th> <th align="justify" rowspan="1" colspan="1">Virus titer upon knockdown<a href="#t001fn001"><sup>§</sup></a> (% of neg ctl)</th> <th align="justify" rowspan="1" colspan="1">Inhibitory drugs</th> </tr> </thead> <tbody> <tr> <td align="justify" rowspan="1" colspan="1">2324</td> <td align="justify" rowspan="1" colspan="1">FLT4</td> <td align="justify" rowspan="1" colspan="1">2</td> <td align="justify" rowspan="1" colspan="1"><a href="https://reports.ingenuity.com/rs/nodeview/nodeview.jsp?nodeid=ING:1izby">CEP 7055</a>, <a href="https://reports.ingenuity.com/rs/nodeview/nodeview.jsp?nodeid=ING:3tyvj">motesanib</a>, <a href="https://reports.ingenuity.com/rs/nodeview/nodeview.jsp?nodeid=ING:4b6p8">telatinib</a>, <a href="https://reports.ingenuity.com/rs/nodeview/nodeview.jsp?nodeid=ING:1md4g">tivozanib</a>, <a href="https://reports.ingenuity.com/rs/nodeview/nodeview.jsp?nodeid=ING:55lx6">cabozantinib</a><a href="#t001fn002"><sup>†</sup></a>, <a href="https://reports.ingenuity.com/rs/nodeview/nodeview.jsp?nodeid=ING:1cqbp">axitinib</a><a href="#t001fn002"><sup>†</sup></a>, <a href="https://reports.ingenuity.com/rs/nodeview/nodeview.jsp?nodeid=ING:n2a">sorafenib</a><a href="#t001fn002"><sup>†</sup></a>, <a href="https://reports.ingenuity.com/rs/nodeview/nodeview.jsp?nodeid=ING:72eay">regorafenib</a><a href="#t001fn002"><sup>†</sup></a>, <a href="https://reports.ingenuity.com/rs/nodeview/nodeview.jsp?nodeid=ING:1agfe">sunitinib</a><a href="#t001fn002"><sup>†</sup></a>, <a href="https://reports.ingenuity.com/rs/nodeview/nodeview.jsp?nodeid=ING:1bpdy">pazopanib</a><a href="#t001fn002"><sup>†</sup></a><a href="#t001fn003"><sup><i>a</i></sup></a>, <a href="https://reports.ingenuity.com/rs/nodeview/nodeview.jsp?nodeid=ING:ouj">vandetanib</a><a href="#t001fn002"><sup>†</sup></a></td> </tr> <tr> <td align="justify" rowspan="1" colspan="1">10188</td> <td align="justify" rowspan="1" colspan="1">TNK2</td> <td align="justify" rowspan="1" colspan="1">12</td> <td align="justify" rowspan="1" colspan="1">vemurafenib<a href="#t001fn002"><sup>†</sup></a><a href="#t001fn003"><sup><i>a</i></sup></a></td> </tr> <tr> <td align="justify" rowspan="1" colspan="1">983</td> <td align="justify" rowspan="1" colspan="1">CDK1</td> <td align="justify" rowspan="1" colspan="1">15</td> <td align="justify" rowspan="1" colspan="1">alvocidib<a href="#t001fn004"><sup>b</sup></a></td> </tr> <tr> <td align="justify" rowspan="1" colspan="1">2475</td> <td align="justify" rowspan="1" colspan="1">MTOR</td> <td align="justify" rowspan="1" colspan="1">22</td> <td align="justify" rowspan="1" colspan="1"><a href="https://reports.ingenuity.com/rs/nodeview/nodeview.jsp?nodeid=ING:4ujsv">dactolisib</a><a href="#t001fn003"><sup><i>a</i></sup></a>, <a href="https://reports.ingenuity.com/rs/nodeview/nodeview.jsp?nodeid=ING:3vfyl">OSI-027</a>, <a href="https://reports.ingenuity.com/rs/nodeview/nodeview.jsp?nodeid=ING:1cq7q">ridaforolimus</a>, <a href="https://reports.ingenuity.com/rs/nodeview/nodeview.jsp?nodeid=ING:mwi">tacrolimus</a><sup>†</sup>, <a href="https://reports.ingenuity.com/rs/nodeview/nodeview.jsp?nodeid=ING:o40">everolimus</a><a href="#t001fn002"><sup>†</sup></a>, <a href="https://reports.ingenuity.com/rs/nodeview/nodeview.jsp?nodeid=ING:mci">sirolimus</a><a href="#t001fn002"><sup>†</sup></a>, <a href="https://reports.ingenuity.com/rs/nodeview/nodeview.jsp?nodeid=ING:muj">temsirolimus</a><a href="#t001fn002"><sup>†</sup></a></td> </tr> <tr> <td align="justify" rowspan="1" colspan="1">1017</td> <td align="justify" rowspan="1" colspan="1">CDK2</td> <td align="justify" rowspan="1" colspan="1">26</td> <td align="justify" rowspan="1" colspan="1">BMS-387032, alvocidib<a href="#t001fn004"><sup>b</sup></a></td> </tr> <tr> <td align="justify" rowspan="1" colspan="1">5293</td> <td align="justify" rowspan="1" colspan="1">PIK3CD</td> <td align="justify" rowspan="1" colspan="1">27</td> <td align="justify" rowspan="1" colspan="1">SF 1126, PX-866, dactolisib<a href="#t001fn003"><sup><i>a</i></sup></a>, pictrelisib, buparlisib, XL147, idelalisib</td> </tr> <tr> <td align="justify" rowspan="1" colspan="1">2268</td> <td align="justify" rowspan="1" colspan="1">FGR</td> <td align="justify" rowspan="1" colspan="1">27</td> <td align="justify" rowspan="1" colspan="1">vemurafenib<a href="#t001fn002"><sup>†</sup></a><a href="#t001fn003"><sup><i>a</i></sup></a></td> </tr> <tr> <td align="justify" rowspan="1" colspan="1">6300</td> <td align="justify" rowspan="1" colspan="1">MAPK12</td> <td align="justify" rowspan="1" colspan="1">30</td> <td align="justify" rowspan="1" colspan="1">talmapimod</td> </tr> <tr> <td align="justify" rowspan="1" colspan="1">64805</td> <td align="justify" rowspan="1" colspan="1">P2RY12</td> <td align="justify" rowspan="1" colspan="1">31</td> <td align="justify" rowspan="1" colspan="1"><a href="https://reports.ingenuity.com/rs/nodeview/nodeview.jsp?nodeid=ING:1amh9">ticagrelor</a><a href="#t001fn002"><sup>†</sup></a>, <a href="https://reports.ingenuity.com/rs/nodeview/nodeview.jsp?nodeid=ING:1amdt">prasugrel</a><a href="#t001fn002"><sup>†</sup></a>, <a href="https://reports.ingenuity.com/rs/nodeview/nodeview.jsp?nodeid=ING:o55">clopidogrel</a><a href="#t001fn002"><sup>†</sup></a>, <a href="https://reports.ingenuity.com/rs/nodeview/nodeview.jsp?nodeid=ING:ngr">ticlopidine</a><a href="#t001fn002"><sup>†</sup></a></td> </tr> <tr> <td align="justify" rowspan="1" colspan="1">1586</td> <td align="justify" rowspan="1" colspan="1">CYP17A1</td> <td align="justify" rowspan="1" colspan="1">33</td> <td align="justify" rowspan="1" colspan="1">abiraterone, ketoconazole<a href="#t001fn002"><sup>†</sup></a>, abiraterone acetate<a href="#t001fn002"><sup>†</sup></a></td> </tr> <tr> <td align="justify" rowspan="1" colspan="1">3716</td> <td align="justify" rowspan="1" colspan="1">JAK1</td> <td align="justify" rowspan="1" colspan="1">34</td> <td align="justify" rowspan="1" colspan="1">tofacitinib<a href="#t001fn002"><sup>†</sup></a>, ruxolitinib<a href="#t001fn002"><sup>†</sup></a></td> </tr> <tr> <td align="justify" rowspan="1" colspan="1">3932</td> <td align="justify" rowspan="1" colspan="1">LCK</td> <td align="justify" rowspan="1" colspan="1">35</td> <td align="justify" rowspan="1" colspan="1">dasatinib<a href="#t001fn002"><sup>†</sup></a>, pazopanib<a href="#t001fn002"><sup>†</sup></a><a href="#t001fn003"><sup><i>a</i></sup></a></td> </tr> <tr> <td align="justify" rowspan="1" colspan="1">1018</td> <td align="justify" rowspan="1" colspan="1">CDK3</td> <td align="justify" rowspan="1" colspan="1">37</td> <td align="justify" rowspan="1" colspan="1">alvocidib<a href="#t001fn004"><sup>b</sup></a></td> </tr> <tr> <td align="justify" rowspan="1" colspan="1">5605</td> <td align="justify" rowspan="1" colspan="1">MAP2K2</td> <td align="justify" rowspan="1" colspan="1">38</td> <td align="justify" rowspan="1" colspan="1">selumetinib, trametinib<a href="#t001fn002"><sup>†</sup></a></td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p><sup>§</sup>data are derived from siRNA screen,</p> </div> <div class="fn-type-" title=""> <p><sup>†</sup>drug is clinically approved in at least one country,</p> </div> <div class="fn-type-" title=""> <p><sup><i>a</i></sup>drug targets two genes listed in the table,</p> </div> <div class="fn-type-" title=""> <p><sup><i>b</i></sup>drug targets three genes listed in the table</p> </div> </div> </div> <p>Taken together, we identified two inhibitors targeting the cellular gene FLT4, which possess high antiviral activity at non-cytotoxic concentrations in A549 cells. Therefore, we decided to characterize the mode of action of FLT4 inhibitors regorafenib and sorafenib in more detail.</p> </div> <div class="urea-basedkinaseinhibitorsregorafenibandsorafenibdonotaffectinfluenzavirusentry" title="sec"> <div class="title" tagx="title" title="title">
Urea-based kinase inhibitors regorafenib and sorafenib do not affect influenza virus entry</div> <p>The gene FLT4 encodes the protein vascular endothelial growth factor receptor 3 (VEGFR3), a receptor tyrosine kinase (RTK). Both regorafenib and sorafenib belong to the class of urea-based kinase inhibitors (UBKIs), which are known to inhibit several other kinases as well as VEGFR3 [<a href="#ppat.1007601.ref015">15</a>, <a href="#ppat.1007601.ref016">16</a>, <a href="#ppat.1007601.ref023">23</a>]. Consequently, using the CRISPR/Cas9 technology, we generated knockout cell lines for the main targets of regorafenib/sorafenib and examined the effect on replication of the IV strain WSN. The strongest effect was observed with knockdown of FLT4, but silencing of FGFR1, FLT1 (encoding VEGFR1), and PDGFRB also markedly reduced viral replication (<a href="#ppat.1007601.s004">S4 Fig</a>). These observations indicate that regorafenib and sorafenib impair viral replication by blocking multiple relevant kinases in parallel–not solely by affecting FLT4.</p> <p>We wanted to identify the main step of the viral replication cycle that is blocked by regorafenib and sorafenib. The IV replication cycle starts with attachment of the virion by binding to sialic acid residues on the host cell surface [<a href="#ppat.1007601.ref024">24</a>]. Neither regorafenib nor sorafenib affected attachment, whereas <i>Clostridium perfringens</i> (<i>Cp</i>) neuraminidase, an enzyme removing sialic acid residues from the cell surface, impaired this process considerably, as indicated by fluorescence microscopy (<a href="#ppat.1007601.g004">Fig 4A</a>) and flow cytometry (<a href="#ppat.1007601.g004">Fig 4B</a>). Upon attachment, IV particles enter the cells mainly via clathrin-mediated endocytosis (CME) [<a href="#ppat.1007601.ref025">25</a>], although macropinocytosis as an alternative entry route has been described [<a href="#ppat.1007601.ref026">26</a>]. First, we assessed the effect of regorafenib and sorafenib on the uptake of two conventional substrates of CME: epidermal growth factor (EGF) and transferrin [<a href="#ppat.1007601.ref027">27</a>, <a href="#ppat.1007601.ref028">28</a>]. Neither of the two agents affected the internalization of EGF (<a href="#ppat.1007601.s005">S5A Fig</a>) or transferrin (<a href="#ppat.1007601.s005">S5B Fig</a>). Accordingly, the internalization of virus particles was not affected by regorafenib or sorafenib treatment, as indicated by fluorescence microscopy (<a href="#ppat.1007601.g004">Fig 4C</a>) and flow cytometry (<a href="#ppat.1007601.g004">Fig 4D</a>). Thus, we conclude that regorafenib and sorafenib do not affect the internalization of IV by host cells.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1007601.g004"><span class="label" tagx="label" title="label">Fig 4</span></a></div>   <div class="title" tagx="title" title="title">
UBKIs do not affect entry of IAV into host cells.</div> <p>(A and B) A549 cells were pre-treated with small molecules at 3 μM, an equivalent amount of DMSO, or <i>Cp</i> neuraminidase, for 30 min. Cells were incubated at 4°C with IAV strain PAN labeled with Alexa Fluor 647 (red), fixed and stained with DAPI (blue), and subjected to (A) microscopic analysis (scale bar = 10 μm) or (B) flow cytometry. Data in (B) represent DMSO-normalized mean ± SEM of n = 3 independent experiments (mean values are significantly different: one-way ANOVA: p = 0.0004; Dunnett´s multiple comparison test for comparison with DMSO control, ns: not significant (p-value &amp;gt; 0.05), ***: p-value ≤ 0.001). (C and D) A549 cells were pre-treated with small molecules at 3 μM, an equivalent amount of DMSO, or 100 μM dynasore for 30 min. After virus attachment for 1 h at 4°C, cells were cultivated in presence of the compounds for 45 min at 37°C to induce virus internalization, then stained for DNA (DAPI, blue) and (extracellular) IV hemagglutinin (HA) protein (green) and subjected to (C) microscopic analysis (scale bar = 10 μm) or (D) flow cytometry. Red signals represent internalized virus particles, yellow (sufficiently Alexa Fluor 647-labeled) and green (insufficiently Alexa Fluor 647-labeled) signals represent cell surface-bound virus particles. Data in (D) represent mean ± SEM of n = 3 independent experiments (mean values are significantly different: one-way ANOVA: p = 0.0011; Dunnett´s multiple comparison test for comparison with DMSO control, ns: not significant (p-value &amp;gt; 0.05), **: p-value ≤0.01.</p>   </div> </div> <div class="urea-basedkinaseinhibitorsregorafenibandsorafenibinhibitinfluenzavirusreplicationbyinterferingwithfusion" title="sec"> <div class="title" tagx="title" title="title">
Urea-based kinase inhibitors regorafenib and sorafenib inhibit influenza virus replication by interfering with fusion</div> <p>One of the first intracellular steps of the viral replication cycle is the fusion of the viral membrane with the endosomal membrane, a process that depends on endosomal acidification [<a href="#ppat.1007601.ref024">24</a>]. Again, we first tested the effect of regorafenib and sorafenib on the fate of the classical CME cargos EGF and transferrin, which also depends on endosomal acidification [<a href="#ppat.1007601.ref027">27</a>, <a href="#ppat.1007601.ref028">28</a>]. EGF is typically degraded in lysosomes, whereas transferrin is recycled to the plasma membrane. As expected, the intracellular level of EGF (<a href="#ppat.1007601.s006">S6A Fig</a>) and transferrin (<a href="#ppat.1007601.s006">S6B Fig</a>) gradually decreased over time in DMSO-treated cells. In contrast, treatment with regorafenib or sorafenib blocked the decrease in intracellular EGF and transferrin levels, most likely due to an impairment of endosomal acidification. Next, we directly characterized the effect of regorafenib and sorafenib on fusion of virus particles with the endosomal membrane. To this end, we simultaneously labeled IV particles with two lipophilic dyes generating a Förster resonance energy transfer (FRET) pair, DiOC18 (green) and Dil (red). Intact labeled virions appear red due to FRET from DiOC18 to DiI, whereas a shift towards green color occurs when the membranes fuse and the fluorophores become dispersed, reducing the FRET [<a href="#ppat.1007601.ref029">29</a>]. Thus, our data suggest that both regorafenib and sorafenib impair cellular processes relevant for the fusion process, as indicated by qualitative (<a href="#ppat.1007601.g005">Fig 5A</a>) and quantitative (<a href="#ppat.1007601.g005">Fig 5B</a>) fluorescence microscopic analysis. In line with these findings, the processes downstream of fusion, i.e. the uncoating of vRNPs, as indicated by M1 dispersal [<a href="#ppat.1007601.ref030">30</a>] (<a href="#ppat.1007601.g005">Fig 5C and 5D</a>), the nuclear import of vRNPs [<a href="#ppat.1007601.ref031">31</a>] (Figs <a href="#ppat.1007601.g005">5E</a> and <a href="#ppat.1007601.s007">S7</a>), the viral mRNA synthesis (<a href="#ppat.1007601.g005">Fig 5F</a>), and the viral protein expression (<a href="#ppat.1007601.g005">Fig 5G</a>) were strongly affected upon regorafenib and sorafenib treatment.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1007601.g005"><span class="label" tagx="label" title="label">Fig 5</span></a></div>   <div class="title" tagx="title" title="title">
UBKIs block cellular pathways important for fusion and downstream processes.</div> <p>(A and B) A549 cells were pre-treated with Hoechst to stain the nuclei (blue) and either small molecules (regorafenib and sorafenib: 3 μM, bafilomycin A1: 200 nM) or an equivalent amount of DMSO for 30 min before incubation at 4°C with IAV strain PAN labeled with DiOC18 (green) and DiI (red). Virus-containing medium was exchanged against media containing either small molecules or DMSO only, and cells imaged at 37°C for 65 min. In (A), representative images for the DiOC18 and the merge of the DiOC18 and the DiI signal acquired 60 min post infection are shown. Scale bar = 10 μm. In (B), micrographs were quantitatively analyzed and the change of the DiOC18 signal intensity is shown for individual time points. Data represent mean (n = 3) ± SEM of independent experiments. (C and D) A549 cells were pre-treated as described for A and B, incubated at 4°C with IAV strain PAN (MOI 5) and then cultivated for 3 h at 37°C in the presence of small molecules and cycloheximide before staining for DNA with DAPI (blue) and IAV M1 protein (green), and subjected to microscopic and flow cytometry analysis. (C) Representative images of three independent experiments. (D) Quantitative analysis. Data represent mean ± SEM of n = 3 independent experiments (mean values are significantly different: one-way ANOVA: p ≤ 0.0001; Dunnett´s multiple comparison test for comparison with DMSO control: **** indicates p<sub>adjusted</sub> = 0.0001). (E) A549 cells, pre-treated as described for (A and B), incubated at 4°C with IAV strain PAN (MOI 5) and cultivated for 4.5 h at 37°C in the presence of small molecules and cycloheximide before staining for DNA with DAPI (blue) and IAV NP protein (green), and subjected to microscopic analysis. Representative micrographs of the z-axis of individual cells are shown. (F) A549 cells were pre-treated with small molecules at 3 μM or an equivalent amount of DMSO for 2 h. Subsequently, the cells were infected with IAV strain WSN and cultivated for 4 h in presence of small molecules. RNA was extracted and the relative amount of the NP, M1, and M2 mRNAs determined by qRT-PCR. Data represent mean ± SEM of n = 3 independent experiments. Significance was calculated based on the one sample t-test for comparison with DMSO control: *: p-value ≤ 0.05, **: p-value ≤ 0.01, ****: p-value ≤ 0.0001. (G) A549 cells were pre-treated with small molecules at non-toxic concentrations (regorafenib &amp;amp; sorafenib: 3 μM, bafilomycinA1: 1 μM) for 2 h. Cells were infected and cultivated for 6 h in presence of small molecules before staining for viral nucleoprotein (NP). Data represent mean ± SEM of n = 3 independent experiments of the fraction of NP-positive cells relative to the vehicle control. Significance was calculated based on the one sample t-test for comparison with DMSO control: *: p-value ≤ 0.05, **: p-value ≤ 0.01.</p>   </div> </div> <div class="regorafenibandsorafenibpreventinfectionbyvariousviruses" title="sec"> <div class="title" tagx="title" title="title">
Regorafenib and sorafenib prevent infection by various viruses</div> <p>Next, we assessed the breadth of regorafenib’s and sorafenib’s antiviral activity. First, we tested the compounds with a panel of six human and three avian IAVs as well as two IBVs (<a href="#ppat.1007601.g006">Fig 6A</a>). Both drugs were active against all strains tested. However, regorafenib only had a moderate, statistically not significant effect on the avian IAVs. We speculated that the differences between the avian and the human IVs were due to differences in the acid stability of the hemagglutinin (HA) proteins: the less stable the HA protein, the higher the pH at which fusion takes place (and the less acidification of the endosomes is required for fusion) [<a href="#ppat.1007601.ref024">24</a>]. Therefore, we determined the fusion pH for a human IAV (PAN), an IBV (THW), and an avian IAV (A/Mallard/Germany/439/2004(H3N2), MAL) (<a href="#ppat.1007601.s008">S8 Fig</a>). To our surprise, the fusion pH for THW and MAL was almost identical and the fusion pH of PAN was even higher. Thus, the HA protein of MAL is not more acid sensitive than that of PAN and THW. The molecular mechanism underlying the low sensitivity of avian IVs against regorafenib thus remains to be illuminated.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.ppat.1007601.g006"><span class="label" tagx="label" title="label">Fig 6</span></a></div>   <div class="title" tagx="title" title="title">
UBKIs possess broad antiviral activity, are effective in primary cells, and impose a barrier against resistance development.</div> <p>(A) A549 cells were pre-treated with 3 μM of small molecules or an equivalent amount of DMSO for 2 h, infected and cultivated for 6 h in presence of small molecules before staining for viral NP protein. Data represent mean ± SEM of n = 3 independent experiments of the fraction of NP-positive cells relative to DMSO. (B) A549 (CPXV, HSV1, VSV) or HEL cell-derived megakaryocyte (HTV) cells were pre-treated as in (A). Cells were infected for 6 h (VSV), 18 h (CHIKV) or 24 h (CPXV, HSV1, HTV) prior to fixation. Presence of viral antigens (HTV: N protein; VSV: NP protein; all other viruses: green fluorescent protein (GFP) was detected by fluorescence microscopy. Data represent mean ± SEM of n = 3 independent experiments of the fraction of viral antigen-positive cells relative to DMSO. Significance in (A) and (B) was calculated based on the one sample t-test for comparison with DMSO control. *: p-value ≤ 0.05, **: p-value ≤ 0.01, ***: p-value ≤ 0.001. (C) Experiments were conducted as in (A) but with hAECB. Zanamivir was applied at 1 μM. (A, B, C) Data represent DMSO-normalized mean ± SEM of n = 3 independent experiments. Indicated p-values based on the one sample t-test describe the difference compared to the DMSO control. (D) hAECB were treated with small molecules at 5 μM. Infected cells were cultivated for 48 h in presence of small molecules and virus load was assessed every six hours by plaque assay. Data represent mean ± SEM of n = 3 technical replicates. Virus titers were analysed by mixed effects models for clustered data where treatment group and time were fixed effects. In the random intercept model residuals associated with different plates were specified to have different residuals. The calculation of p-values revealed a significant decrease of log-transformed virus-titers in the regorafenib and sorafenib-treated groups relative to the DMSO control at the indicated time points: *: p-value ≤ 0.05. (E) MDCK cells were pre-treated with sorafenib (3 μM), zanamivir (0.3 μM), amantadine (3 μM) or an equivalent amount of DMSO for 2 h, infected with MOI 0.001 of IV strain A/England/195/2009–M2<sub>N31S</sub> and cultivated for 24 h in presence of reagents. Virus titer in tissue culture supernatants was determined and the supernatants were used for inoculation of the next passage. The experiment was terminated after 8 passages. Data represent mean ± SD of technical replicates.</p>   </div> <p>Next, we tested the effect of the UBKIs on the non-related viruses cowpox virus (CPXV), herpes simplex virus 1 (HSV1), hantavirus (HTV), and vesicular stomatitis virus (VSV) (<a href="#ppat.1007601.g006">Fig 6B</a>). We observed strong effects on HTV, and VSV but only mild effects on CPXV and HSV1. This difference is likely due to the different mechanisms of entry between the viruses: HTV and VSV enter host cells exclusively via the endocytic route [<a href="#ppat.1007601.ref032">32</a>–<a href="#ppat.1007601.ref034">34</a>], whereas CPXV and HSV1 can enter by direct fusion with the cellular plasma membrane, as well as via the endocytic route [<a href="#ppat.1007601.ref035">35</a>, <a href="#ppat.1007601.ref036">36</a>]. Of note, we applied both inhibitors at 3 μM, as this dose was neither cytotoxic in A549 cells (<a href="#ppat.1007601.g003">Fig 3A and 3B</a>), used with CPXV, HSV1, and VSV, nor in HEL-derived megakaryocytes (<a href="#ppat.1007601.s009">S9A Fig</a>), used with HTV. In conclusion, regorafenib and sorafenib display antiviral activity against all viruses tested, particularly those that rely exclusively on the endocytic route.</p> </div> <div class="regorafenibandsorafenibinhibitinfluenzavirusreplicationinmurinelungandhumanprimarybronchialepithelialcellsandinmice" title="sec"> <div class="title" tagx="title" title="title">
Regorafenib and sorafenib inhibit influenza virus replication in murine lung and human primary bronchial epithelial cells and in mice</div> <p>We next tested if regorafenib and sorafenib are also active in more physiological systems, such as mouse and human primary cells. Both UBKIs showed antiviral activity in mouse lung epithelial cells, however with lowered efficacy and increased cytotoxicity compared to human cells (<a href="#ppat.1007601.s010">S10A and S10B Fig</a>). These results indicate that mice may not be an optimal system to investigate the antiviral activity of regorafenib and sorafenib. Still, we assessed the effect of regorafenib on influenza virus replication <i>in vivo</i> in BALB/c mice. Regorafenib or vehicle was administered once daily via the oral route. Mice were infected intranasally with 4.4 x 10<sup>4</sup> PFU/animal, 2 h after the first administration of regorafenib or vehicle, respectively. Two days post-infection, mice were culled, and the viral load in the lungs was determined by plaque assay. In line with the results obtained with mouse lung epithelial cells, we observed a reduced, albeit not significantly changed viral load in the lungs of regorafenib-treated mice (<a href="#ppat.1007601.s010">S10C Fig</a>).</p> <p>Next, we tested the effect of regorafenib and sorafenib on susceptibility of primary human airway epithelial cells from bronchi (hAECB). We used the same dose as with A549 cells, 3 μM, a dose not cytotoxic for hAECB cells (<a href="#ppat.1007601.s009">S9B Fig</a>). Also in primary cells, UBKIs efficiently blocked viral protein expression (<a href="#ppat.1007601.g006">Fig 6C</a>). Then, we assessed both UBKIs effects on viral replication in hAECB cells. To this end, we employed the highest dose without detectable cytotoxicity to hAECB cells, 5 μM (<a href="#ppat.1007601.s009">S9B and S9D Fig</a>). Again, both inhibitors revealed a significant reduction of virus replication, although the effect induced by regorafenib was found to be more pronounced, especially at later time points (<a href="#ppat.1007601.g006">Fig 6D</a>).</p> </div> <div class="regorafenibandsorafenibimposeahighbarriertoresistancedevelopmentincaninecells" title="sec"> <div class="title" tagx="title" title="title">
Regorafenib and sorafenib impose a high barrier to resistance development in canine cells</div> <p>Finally, we wanted to test the sensitivity of UBKIs to the development of resistant viruses, and we selected regorafenib and sorafenib as representative UBKIs. To this end, we passaged the IAV strain A/England/195/2009(H1N1) serially in MDCK cells in the presence of 3 μM of sorafenib. In addition, we included amantadine, an inhibitor of the viral M2 protein, and zanamivir, an FDA approved neuraminidase (NA) inhibitor as positive controls within this study. Since nearly all circulating IAV strains are already resistant to the group of adamantanes, we used a recombinant virus mutant that contains a mutation (N31S) within the viral M2 gene that makes the virus susceptible to amantadine treatment [<a href="#ppat.1007601.ref037">37</a>]. We chose the canine cell line MDCK for the resistance assay as it is a highly efficient IV producer with a very weak IFN response [<a href="#ppat.1007601.ref038">38</a>], a very important feature for serial passaging experiments. In congruence with previous studies that demonstrated full resistance to DAAs of the group of adamantanes within a few rounds of passaging, we observed resistance against amantadine after 3 passages. However, even after 8 passages no resistance development could be observed when virus replication was inhibited using sorafenib or zanamivir (<a href="#ppat.1007601.g006">Fig 6E</a>). To further confirm that resistance to UBKIs is difficult to induce, we also carried out the experiment in MDCK cells infected with A/WSN/1933(H1N1) in the presence of the non-cytotoxic dose (<a href="#ppat.1007601.s009">S9C Fig</a>) of 3 μM of regorafenib. Again, after five passages, a time when full resistance to DAAs of the group of adamantanes has normally occurred [<a href="#ppat.1007601.ref039">39</a>, <a href="#ppat.1007601.ref040">40</a>] (and see also <a href="#ppat.1007601.g006">Fig 6E</a>), no resistance to regorafenib was observed (<a href="#ppat.1007601.s011">S11 Fig</a>).</p> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Discussion</div> <p>We present here a large-scale comparative loss-of-function screen for IV replication. We also included an avian IAV strain and an IBV strain. An avian strain has only been tested in one other published screen [<a href="#ppat.1007601.ref013">13</a>], whereas no IBV strain was in the focus of previous loss-of-function screens. Due to the comparative nature of the present screen, we were able to detect strain-specific differences in host factor requirement. The number identified, several hundred strain-specific genes and gene groups, was unexpectedly large. The impact of these genes and gene groups on viral replication differed in many instances either between the three human IVs versus the avian IV strain and between the three IAVs versus the IBV strains. These differences likely reflect evolutionary divergence (IAV vs. IBV) or adaptation to divergent hosts (human IV vs. avian IV).</p> <p>Apart from identifying differences between IVs, the present study was designed to identify genes and cellular processes which are conserved parts of the viral replication cycle across the different types of IVs. We identified 133 host cell genes that are strictly required for replication of all four IV strains tested. Among these strain-independent genes, those involved in nucleocytoplasmic transport, cell cycle regulation, RNA splicing, translation, and tRNA charging were highly enriched.</p> <p>While several of these gene requirements have been previously noted [<a href="#ppat.1007601.ref012">12</a>], tRNA charging has not been seen in previous screens of human IAVs [<a href="#ppat.1007601.ref012">12</a>]. The exact implication of tRNA charging in IV replication remains to be characterized. Yet, it might affect translation since charged tRNAs constitute the substrates for protein synthesis. According to our results, the cellular process most strictly required for IV replication across all strains is the nucleocytoplasmic transport. Therefore, agents blocking nucleocytoplasmic transport represent attractive future candidates for a host-directed therapy of influenza. In accordance with this, promising results have been published with the exportin 1 inhibitor verdinexor [<a href="#ppat.1007601.ref041">41</a>]. However, a recent study demonstrated that IAVs quickly become resistant to depletion of importin-α7, another factor involved in nucleocytoplasmic trafficking, when an in vivo model is used and that the resistant virus even gained higher virulence [<a href="#ppat.1007601.ref042">42</a>]. Therefore, in vitro and in vivo experiments are required to further explore the therapeutic potential of cellular factors in general as antiviral drug target.</p> <p>The <i>de novo</i> development of drugs is a long and expensive process, making the repurposing of pharmaceuticals for new indications an attractive strategy. By drug database screening, we identified several approved or experimental drugs targeting strain-independent host factors. Most of these drugs were screened in A549 cells for antiviral activity and cytotoxicity rendering fourteen with a potential for therapeutic application. We focused on the FLT4 inhibitors regorafenib and sorafenib and found that in addition to VEGFR3 at least three other RTKs (FGFR1, VEGFR1, PDGFRB) targeted by these drugs are relevant to IV replication. Therefore, the antiviral activity of regorafenib and sorafenib is likely to be due to the inhibition of several target proteins in parallel, and may even include further kinases. Both drugs interfered with fusion of the viral and the endosomal membranes. Impairment of an early step in the IV life cycle, such as fusion, necessarily would affect all downstream steps. Consequently, it remains an open question if regorafenib and sorafenib affect additional (post-fusion) steps of viral replication. Interestingly, the broad-range RTK inhibitor genistein as well as a broad-range RTK inhibitor mix (containing the VEGFR/PDGFR inhibitor SU4312), have been shown to inhibit the uptake of IV virions [<a href="#ppat.1007601.ref043">43</a>], indicating that while some RTKs are required for internalization, others are required for fusion. It is possible that regorafenib and sorafenib inhibit kinases involved in endosomal maturation or block kinases that phosphorylate the V-ATPase subunits, since their phosphorylation status regulates their activity and thus endosomal acidification [<a href="#ppat.1007601.ref044">44</a>, <a href="#ppat.1007601.ref045">45</a>]. Future studies will shed more light on the mechanisms involved in the UBKI-mediated block of fusion.</p> <p>Regorafenib and sorafenib not only affected IV but also HTV, VSV, and HSV1. The effects on viruses using the endocytic route were stronger than the effect on those mainly fusing directly with the plasma membrane. The effect of regorafenib on avian IV was milder than on human IV. This was unexpected, since we incorporated regorafenib in our small molecule screen because one of its targets, FLT4, was universally required in the siRNA screen. Possibly, other targets of regorafenib (s. above) are strain-specific. The molecular mechanism underlying the differential effect of regorafenib on human and avian IVs is not clear but appears to be independent of the fusion pH of the HA protein of the different viruses. Recent studies showed that prior to fusion, pH-dependent conformational rearrangements–especially of the M1 protein–take place within the lumen of the virion, and these rearrangements are important for efficient uncoating [<a href="#ppat.1007601.ref046">46</a>, <a href="#ppat.1007601.ref047">47</a>]. As these processes require luminal acidification, they are dependent on the activity of the viral M2 ion channel. Thus, the IV strain-specific effect of regorafenib might be due to differences in the M1 or M2 proteins. Another possibility is that, as discussed above, regorafenib impairs steps in the viral replication cycle downstream of fusion, and in a strain-specific manner. Nevertheless, due to the broad antiviral activity, regorafenib and sorafenib might be suited for a host-directed therapy of various viral diseases. This indicates a clear benefit compared to current influenza treatments, which are only effective against IV. In addition, both drugs efficiently inhibited IV replication in primary human respiratory cells, implying that our findings might indeed have a physiological relevance.</p> <p>A recent study investigated the impact of 273 kinase inhibitors on IAV replication and reported only intermediate effects for regorafenib and sorafenib [<a href="#ppat.1007601.ref048">48</a>]. The cause of this discrepancy is not clear but might be due to differences in the assay setup. Furthermore, the previous study used an avian IV strain, and avian IVs are less sensitive to regorafenib treatment according to our results. In line with our findings, sorafenib mono-treatment has previously been found to impair replication of various viruses such as adenovirus, coxsackievirus, enterovirus 71, hepatitis C virus, human cytomegalovirus, mumps virus, polyomavirus BK, Rift Valley fever virus, and West Nile virus, respectively, and combinatorial treatment with sildenafil led also to inhibition of dengue virus, rabies virus, and yellow fever virus [<a href="#ppat.1007601.ref049">49</a>–<a href="#ppat.1007601.ref055">55</a>]. Furthermore, regorafenib treatment has been reported to affect adenovirus, coxsackievirus, and mumps virus, as well [<a href="#ppat.1007601.ref054">54</a>]. Interestingly, different mechanisms of antiviral activity have been described, including inhibition of entry, viral protein translation, and viral egress [<a href="#ppat.1007601.ref050">50</a>, <a href="#ppat.1007601.ref051">51</a>, <a href="#ppat.1007601.ref055">55</a>]. An effect of these drugs on viral fusion has, to our knowledge, not been reported so far. In conclusion, our study and the existing literature support the notion that regorafenib and sorafenib possess (A) broad antiviral activity and (B) different mechanisms of antiviral activity.</p> <p>Surprisingly, the efficacy of regorafenib and sorafenib in primary cells from bronchi was much lower than in the pneumocyte-derived cell line A549. This difference might be related to the different origin of the cell types (bronchi versus alveoli) but also to the different degrees of nativeness (primary cells versus cell line). To explore this further, the effect of these drugs on IV replication should be tested in additional relevant primary cell types (from upper respiratory tract and alveoli), followed by a suitable <i>in vivo</i> model of IV replication. In an initial animal experiment, regorafenib showed a rather weak antiviral effect in mice. However, regorafenib also exhibits a considerably lower efficacy <i>in vitro</i> in murine compared to human cells. Thus, the preliminary <i>in vivo</i> data might underestimate regorafenib’s true <i>in vivo</i> efficacy in a system which resembles the human host more closely, e.g. the ferret [<a href="#ppat.1007601.ref056">56</a>]. Consequently, experiments with such highly authentic animal models need to be performed in a future study to better judge the clinical applicability of regorafenib and sorafenib.</p> <p>Importantly, we did not observe development of resistance against regorafenib and sorafenib even after up to eight passages in cell culture. This indicates that UBKIs impose a strong barrier towards the development of viral resistance, exceeding the threshold of the viral M2 ion channel-directed drug amantadine, which occurs after three to five passages shown by us and others [<a href="#ppat.1007601.ref039">39</a>, <a href="#ppat.1007601.ref040">40</a>, <a href="#ppat.1007601.ref057">57</a>]. Interestingly, the passaged virus, that gained resistance to amantadine, now replicates at a higher rate compared to the initial virus. The mutation probably increases viral fitness slightly, in line with the fact that all currently circulating pH1N1 viruses are resistant to amantadine, despite a lack of selection pressure since amantadine is no longer used for therapy [<a href="#ppat.1007601.ref058">58</a>]. In contrast, even after eight passages development of resistance to approved viral NA inhibitors, such as zanamivir, was neither observed by us nor are has it been reported by others, to our knowledge. To compare the resistance profiles of viral NA inhibitors and regorafenib/sorafenib, long-term studies need to be performed.</p> <p>The current gold standard in influenza therapy are the DAAs oseltamivir and zanamivir, which target the viral neuraminidase. The IC<sub>50</sub> of these drugs is in the low nanomolar range [<a href="#ppat.1007601.ref059">59</a>]. We observed an IC<sub>50</sub> of 3 nM for zanamivir in our experimental system, which outperforms regorafenib and sorafenib. However, oseltamivir and zanamivir lack activity against unrelated viruses. Furthermore, resistance against oseltamivir and zanamivir has been reported in the clinic [<a href="#ppat.1007601.ref003">3</a>–<a href="#ppat.1007601.ref005">5</a>]. Thus, regorafenib and sorafenib might represent a valuable alternative to the current standard.</p> <p>An existing uncertainty of sorafenib as host-directed therapeutic against influenza is its immune modulatory effects. An impairment of dendritic cell differentiation and function has been observed [<a href="#ppat.1007601.ref060">60</a>, <a href="#ppat.1007601.ref061">61</a>]. In case of T cells and natural killer cells, inhibitory but also stimulatory effects have been reported [<a href="#ppat.1007601.ref062">62</a>–<a href="#ppat.1007601.ref065">65</a>]. Nevertheless, the seroprotection rates induced by influenza vaccination do not differ between cancer patients undergoing sorafenib treatment and controls [<a href="#ppat.1007601.ref066">66</a>]. The effects of regorafenib on the immune response have not been thoroughly characterized yet. Although interference with the immune response represents a potential disadvantage for an antiviral drug, its use might nonetheless be beneficial as long as the antiviral effects significantly overcompensate for the immunomodulatory effects. This, however, can only be addressed in suitable animal models.</p> <p>For approval of cancer therapeutics, such as regorafenib and sorafenib, more severe adverse events are tolerable than for influenza drugs considering influenza is usually not lethal. Although the therapeutic margin of regorafenib/sorafenib is indeed lower compared to standard anti-influenza drugs, such as zanamivir, it is still larger than that of ribavirin, a drug approved for treatment of several virus infections [<a href="#ppat.1007601.ref067">67</a>]. The most prominent side effects of regorafenib and sorafenib are rash and hypertension. However, both UBKIs are usually administered over weeks or months during cancer therapy, whereas a few days might be sufficient in case of influenza treatment. Although short-term treatment would not necessarily prevent any occurrence of adverse events, the duration of side effects would be shortened and the impairment of patients’ quality of lives lowered compared to cancer treatment. In addition, it might be possible to apply both UBKIs locally by inhalation, in contrast to the systemic administration used for cancer therapy, which would allow lower dosing (relative to the bodyweight) and thus potentially reduce adverse events. Furthermore, especially in cases of severe influenza or acute respiratory distress syndrome, the potential side effects of regorafenib/sorafenib may be acceptable. With the standard dosing regimen (160 mg, once daily), the plasma concentration of regorafenib reaches a maximum of 3.6 μM and stays above the IC<sub>50</sub> for IAV replication in A549 cells (i.e. &amp;gt; 0.89 μM) for at least 48 h [<a href="#ppat.1007601.ref068">68</a>]. Standard administration of sorafenib (400 mg, twice daily) achieves a plasma concentration of approximately 4 μM, which is stable for at least 24 h [<a href="#ppat.1007601.ref069">69</a>]. Thus, there is reason for confidence that regorafenib and sorafenib are viable treatment options.</p> <p>In summary, our siRNA screening campaign identified nucleocytoplasmic transport as the host cellular process that is most strictly required for replication across all tested IV strains. Subsequent small molecule screening showed that the two UBKIs regorafenib and sorafenib have powerful broad range antiviral activity, are also efficacious in primary cells, and impose a barrier towards the development of resistant virus strains.</p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials &amp;amp; methods</div> <div class="ethicalpermissions" title="sec"> <div class="title" tagx="title" title="title">
Ethical permissions</div> <p>All animal work was conducted in accordance with European regulations and approved by the Berlin state authorities, Landesamt für Gesundheit und Soziales (Reg No: G0206/12 and 0321/08).</p> </div> <div class="cells" title="sec"> <div class="title" tagx="title" title="title">
Cells</div> <p>A549 adenocarcinomic human alveolar basal epithelials (ATCC CCL-185) were normally grown in DMEM media (Thermo Fischer) supplemented with 2 mM L-glutamine, 1 mM sodium pyruvate, 100 U/ml penicillin/streptomycin (P/S) and 10% heat-inactivated fetal calf serum (hiFCS, Biochrom;) (DMEM complete medium). A549 cells to be infected with CPXV or HSV1 were grown in absence of sodium pyruvate. A549-IFN-Luc cells were generated by lentiviral transduction using the Cignal Lenti ISRE Reporter (luc) Kit CLS-008L (Qiagen) and grown in DMEM complete medium supplemented with 5 μg/ml puromycin dihydrochloride (Thermo Fischer). A549-CRISPR/Cas9 knockout cells were generated as described below and grown in DMEM complete medium supplemented with 10 μg/ml blasticidine and 2.5 μg/ml puromycin. The primary human airway epithelial cells from bronchi (hAECB, Epithelix Sarl) were propagated using NIH/3T3 cells gamma-irradiated with 30 Gy as feeders, using a 3:1 mixture of Ham’s F-12 nutrient mix (Invitrogen) and DMEM supplemented with 5% hiFCS, 0.4 μg/mL hydrocortisone (Sigma-Aldrich), 5 μg/mL recombinant human insulin (Sigma-Aldrich), 8.4 ng/mL cholera toxin (Sigma-Aldrich), 24 μg/mL adenine (Sigma-Aldrich), 10 ng/mL recombinant human EGF (Thermo Fischer) and 9 μM Y27632 (Miltenyi Biotec). When hAECB reached 80 to 90% confluence, they were separated from feeders by differential trypsinization and passaged to new culture vessels with CnT-Prime Airway medium (CELLnTEC). HEL 92.1.7 human erythroleukemia cells (HEL cells, ATCC TIB-180) were grown in RPMI 1640 media (Thermo Fischer) supplemented with 2 mM L-glutamine, 25 mM HEPES, 100 U/ml P/S and 10% hiFCS and differentiated into megakaryocytes at infection using stimulation with phorbol myristate acetate [<a href="#ppat.1007601.ref070">70</a>]. MDCK Madin-Darby canine kidney epithelial cells (ATCC CCL-34) and Vero E6 African green monkey kidney epithelial cells (ATCC CRL-1586) were grown in DMEM complete medium without sodium pyruvate. MLE 12 SV40-transformed mouse lung epithelial cells (ATCC CRL-2110) were grown in DMEM/Ham’s F10 media (Thermo Fischer) supplemented with 2 mM L-glutamine, 50 nM hydrocortisone, 5 μg/ml insulin, and 2% hiFCS. NIH/3T3 mouse fibroblast cells (ATCC CRL-1658) were grown in DMEM complete medium lacking antibiotics. All cells were grown at 37°C under humidity and 5% CO<sub>2</sub> on cell culture-treated plastic ware unless stated otherwise.</p> </div> <div class="generationofa549-crispr/cas9knockoutcelllines" title="sec"> <div class="title" tagx="title" title="title">
Generation of A549-CRISPR/Cas9 knockout cell lines</div> <p>First, A549 were transduced with lentiviruses based on the plasmid lentiCas9-Blast (Addgene number: 52962). After selection with blasticidine for 10 days, cells were transduced with lentiviruses derived from the plasmid lentiGuide-Puro [<a href="#ppat.1007601.ref071">71</a>] (Addgene number: 52963) that leads to the expression of a guideRNA with gene dependent targeting sequences (<a href="#ppat.1007601.s024">S12 Table</a>). Finally, single cell clones were kept under selection medium containing 10 μg/ml blasticidine and 2.5 μg/ml puromycin for an additional 10 days.</p> </div> <div class="viruses" title="sec"> <div class="title" tagx="title" title="title">
Viruses</div> <p>Influenza virus strains A/Puerto Rico/8/1934(H1N1), A/Panama/2007/1999(H3N2), A/Mallard/Germany/439/2004(H3N2) (all provided by T. Wolff, Robert Koch Institute, Germany), A/Hamburg/04/2009(H1N1) (provided by S. Becker, University of Marburg, Germany), A/Victoria/3/1975(H3N2) (described recently in [<a href="#ppat.1007601.ref072">72</a>]), A/FPV/Bratislava/1979(H7N7) (provided by S. Ludwig, University of Münster, Germany), B/Thuringia/02/2006xB/Vienna/33/2006 (provided by Avir Greenhills Biotechnology, Austria, [<a href="#ppat.1007601.ref073">73</a>]), and B/Brisbane/60/2008 (provided by T. Frensing, Max Planck Institute for Dynamics of Complex Technical Systems, Germany) were grown in the allantoic cavities of 9- or 10-day-old embryonated chicken eggs. Seeds of influenza virus strains A/WSN/1933(H1N1) (provided by St. Jude Children’s Research Hospital, USA) and A/Vietnam/1203/2004(H5N1) (provided by Y. Kawaoka, University of Wisconsin-Madison, USA) were produced by reverse genetics as described in [<a href="#ppat.1007601.ref074">74</a>] and [<a href="#ppat.1007601.ref075">75</a>], respectively. Virus seeds were subsequently propagated in the allantoic cavities of 9- or 10-day-old embryonated chicken eggs and used for experimentation. The amantadine sensitive A/England/195/2009–M2<sub>N31S</sub>(H1N1) virus was generated using reverse genetics. A plasmid-based system that allows rescue of a prototypic first wave pH1N1 virus A/England/195/2009 [<a href="#ppat.1007601.ref076">76</a>] was employed. The plasmid encoding segment 7 vRNA was mutated to generate an amino acid change in the M2 coding region at amino acid 31 from N to S. Influenza virus strain A/England/691/2010(H3N2) was obtained through the MOSAIC consortium [<a href="#ppat.1007601.ref076">76</a>]. A/England/195/2009(H1N1) were described recently. Vesicular stomatitis virus strain Indiana (provided T. Wolff) was grown in MDCK cells. Recombinant, GFP expressing viruses cowpox virus strain Brighton Red (provided by K. Tischer, Freie Universität Berlin, Germany), herpes simplex virus 1 strain KOS K26GFP (provided by P. Desai, John Hopkins University, USA), and hantavirus strain Hantaan 76–118 were grown in Vero E6 cells. For experiments, virus stocks were diluted to the desired MOI in PBS containing 9 μM bovine serum albumin (BSA), 492 μM CaCl<sub>2</sub>, and 901 μM MgCl<sub>2</sub> (infection buffer), unless stated otherwise.</p> </div> <div class="sirnascreenexecution" title="sec"> <div class="title" tagx="title" title="title">
siRNA screen execution</div> <p>A549 wildtype or A549-IFN-Luc cells were transfected with siRNAs in 384-well format as previously described for A549 wildtype cells [<a href="#ppat.1007601.ref011">11</a>]. After 24 h, the A549-IFN-Luc cells were subjected to IFN induction assay as described below. After 48 h, the A549 wildtype cells were either subjected to cell viability assays (WST-1 assay, cell number determination assay) or virus replication assay as described below. All multi-well pipetting steps were performed using a Biomek FX<sup>P</sup> Laboratory Automation Workstation (Beckman Coulter). The siRNA library was composed of a validated human kinome library and a custom designed siRNA library (Qiagen) in total containing 3,482 siRNAs targeting 1,208 human genes. On each plate, control RNAs were included: non-targeting siRNA Allstars (14 wells) as negative control for all assays, polyinosinic-polycytidylic acid (poly I:C, final concentration: 2 μg/ml, 6 wells, poly(I:C)-LMW, Invivogen) as positive control for the IFN induction assay, an siRNA targeting the cellular gene PLK1 (siPLK1, 4 wells, [<a href="#ppat.1007601.ref011">11</a>]) as positive control for cell viability assays, and an siRNA targeting the IAV NP gene in case of IAV (8 wells, [<a href="#ppat.1007601.ref077">77</a>]) or an siRNA targeting the cellular gene XPO1 (8 wells, target sequence: 5’-CTGTGTCAGTTTGTAATGGAA-3’) in case of IBV, respectively, as positive control for virus replication assays. All siRNAs were purchased from Qiagen.</p> </div> <div class="ifninductionassay" title="sec"> <div class="title" tagx="title" title="title">
IFN induction assay</div> <p>Twenty-four hours post-transfection, tissue culture supernatants of the A549-IFN-Luc cells, an A549 derivative cell line stably expressing <i>photinus pyralis</i> luciferase under control of tandem repeats of the IFN stimulated response element, were replaced with Beetle Lysis-Juice luciferase substrate (p.j.k.) and luciferase activity was quantified using the EnVision Multilabel Reader (Perkin Elmer).</p> </div> <div class="wst-1assay" title="sec"> <div class="title" tagx="title" title="title">
WST-1 assay</div> <p>In siRNA-transfected A549 cells, WST-1 assay was performed 48 h post transfection. To assess cytotoxicity of small molecules, cells were washed with PBS and incubated for 36 h (A549) or 48 h (HEL-derived megakaryocytes, hAECB, MDCK, MLE 12,). Cells were washed with infection buffer and cultured for 30 min in presence of DMEM supplemented with 9 μM BSA, 2 mM L-glutamine, 1 mM sodium pyruvate, 100 U/ml P/S, 25 mM HEPES (cell culture grade) (infection medium) containing 10% of WST-1 reagent (Roche). Finally, light absorbance at 460 nm and 590 nm was quantified using the EnVision Multilabel Reader and cell viability was defined as the difference of absorbance at 460 nm and 590 nm.</p> </div> <div class="cellnumberdeterminationassay" title="sec"> <div class="title" tagx="title" title="title">
Cell number determination assay</div> <p>To determine the number of cells per well, cells subjected to WST-1 assay as described above were immediately fixed and nuclei stained by incubation with 3.7% formaldehyde containing 10 μg/ml of Hoechst for 1 h. The solution was replaced with PBS and the number of nuclei per well was quantified using the Acumen eX3 Cytometer (TTP Labtech).</p> </div> <div class="virusreplicationassay(fluorescentfocusassay)" title="sec"> <div class="title" tagx="title" title="title">
Virus replication assay (fluorescent focus assay)</div> <p>To test the effect of siRNAs, virus replication assay was started 48 h post-transfection. To test the effect of small molecules, cells were washed with PBS and incubated for 2 h with small molecules under normal culture conditions. To test the effect of CRISPR/Cas9-mediated gene deletion, A549-CRISPR/Cas9 cells were subjected to the assay 10 days post selection. Then, cells were inoculated with virus diluted in infection buffer. Transfected A549 cells were inoculated with an MOI of 0.01 (WSN, PAN), 0.04 (VN), or 2.6 (THW). Small molecule-treated A549 and MLE 12 cells were inoculated with WSN with an MOI of 0.02, A549-CRISPR/Cas9 cells were inoculated with WSN with an MOI of 0.005 and MDCK cells were inoculated with WSN with an MOI of 0.001. After 40 min, plain infection medium or infection medium containing small molecules, respectively, containing 1 μg/ml TPCK-treated trypsin (Sigma-Aldrich) were added, and the cells were incubated under normal culture conditions for 24 h (WSN in MDCK), 36 h (WSN in all A549-derived cells, PAN, VN), 48 h (WSN in MLE 12), or 96 h (THW). Virus load was determined by inoculating (new) MDCK cells with the supernatants followed by fluorescence microscopic analysis of infection. To this end, MDCK cells were washed and inoculated with undiluted supernatant for 1 h. Virus was removed and MDCK cells were cultivated for 6 h under normal culture conditions with infection medium. Cells were stained for viral NP protein and nuclei and the infection rate (i.e. the rate of NP positive cells) determined as described previously [<a href="#ppat.1007601.ref011">11</a>]. The final read-out of the assay was the virus titer. This was extrapolated from the infection rate based on the following formula: multiplicity of infection = -ln(1- infection rate).</p> </div> <div class="sirnascreendataanalysis" title="sec"> <div class="title" tagx="title" title="title">
siRNA screen data analysis</div> <p>Raw data from the individual assays were analyzed with R (<a href="http://www.r-project.org/">http://www.R-project.org/</a>) using the cellHTS2 package using either "normalized percent inhibition�? (NPI; used for analysis of IFN induction and cell viability screens and virus replication screens to identify strain-independent genes and processes) or "percent of control�? (POC; used for analysis virus replication screens to identify strain-specific genes and processes) for raw data normalization [<a href="#ppat.1007601.ref078">78</a>]. For NPI normalization, raw data were scaled relative to an inactive and an active siRNA as described in ‘siRNA screen execution’. For POC normalization, raw data were scaled only relative to non-targeting siRNA Allstars. Screen results were analyzed based on three replicates. To ensure high data quality, only replicate plates with a Pearson product-moment correlation coefficient ≥ 0.45 and a background-to-signal ratio’ of ≥ 5 were analyzed. The background-to-signal ratio’ was calculated by dividing the plate’s average raw data value for the inactive control siRNA (Allstars) by the value of the respective active control (i.e., poly I:C, or anti-PLK1, -NP, or -XPO1 siRNA). IFN induction and cell viability assays were only analyzed on siRNA level. For analysis of virus replication screens, siRNA results were summarized on gene level using the collective mean and cSSMD estimated by the method of moments [<a href="#ppat.1007601.ref018">18</a>].</p> <p>As we did not observe a correlation of signals in the IFN induction assay and the virus replication assay (using the siRNA level data of the WSN screen, <a href="#ppat.1007601.s012">S12 Fig</a>), we regarded all sample siRNAs as non-inducers of IFN. The cut-off for cytotoxic siRNAs according to WST-1 assay was selected based on the point closest to (1,1) of a receiver operating characteristic curve using the siRNA level data of the WST-1 and the virus replication assay with IV strain WSN for Allstars and siPLK1. The cut-off for cytotoxic siRNAs according to the cell number determination assay was set as the mean—3 x SD of the cell count for Allstars.</p> <p>In contrast to the IFN induction, WST-1, and cell number determination assays, virus replication screens were not analyzed on siRNA but on gene level. For identification of genes required by all four IVs, NPI-normalized input data were used. Genes with collective mean &amp;gt; 0.5 and cSSMD ≥ 1 were regarded as proviral. Genes with collective mean &amp;lt; -1 and cSSMD ≤ 1 were regarded as antiviral. All other genes were regarded as neutral. Genes identified as proviral with all four IV strains tested are designated "strain-independent genes�?.</p> </div> <div class="genegroupenrichmentanalyses" title="sec"> <div class="title" tagx="title" title="title">
Gene group enrichment analyses</div> <p>All strain-independent genes were subjected to gene group enrichment analysis using IPA (Ingenuity H Systems) and DAVID [<a href="#ppat.1007601.ref019">19</a>]. In DAVID, databases GOTERM_BP_ALL, GOTERM_CC_ALL, GOTERM_MF_ALL, BIOCARTA, KEGG_PATHWAY, and REACTOME_PATHWAY were collectively analyzed by Functional Annotation Clustering (highest stringency).</p> </div> <div class="identificationofstrain-specificgenesandpathways" title="sec"> <div class="title" tagx="title" title="title">
Identification of strain-specific genes and pathways</div> <p>POC-normalized data were analyzed for host cell genes and pathways with virus strain-specific relevance with a mixed effects model using R with the library _nlme (<a href="http://cran.r-project.org/paclage=nlme%3e">http://CRAN.R-project.org/paclage=nlme&amp;gt;</a>). Genes were assigned gene groups according to databases Gene Ontology database [<a href="#ppat.1007601.ref020">20</a>], the Kyoto Encyclopedia of Genes and Genomes database [<a href="#ppat.1007601.ref021">21</a>], and the Reactome database [<a href="#ppat.1007601.ref022">22</a>], and only groups represented by ≥ 2 genes were analyzed. Genes/gene groups with a false discovery rate-adjusted p-value ≤ 0.05 for the interaction between the factors "virus�? and "treatment�? were regarded to be differentially required between the tested viruses. Differentially required entities with a POC-score ≥ 2 for all four and differentially required entities not classified as pro- or antiviral for any of the four viruses, respectively, were excluded from further analyses and the remaining genes and gene groups were designated "strain-specific genes�? and "strain-specific pathways�?, respectively. Strain-specific genes were furthermore analyzed with the clustering algorithm CLICK [<a href="#ppat.1007601.ref079">79</a>] embedded in EXPANDER [<a href="#ppat.1007601.ref080">80</a>].</p> </div> <div class="insilicoscreenforcandidatedrugsviaadrugrepositioningapproach" title="sec"> <div class="title" tagx="title" title="title">
In silico screen for candidate drugs via a drug repositioning approach</div> <p>All information about drugs targeting any of the 133 strain-independent genes was retrieved with Ingenuity Pathway Analysis. No drug combinations were investigated.</p> </div> <div class="smallmolecules" title="sec"> <div class="title" tagx="title" title="title">
Small molecules</div> <p>Small molecules tested as IV inhibitors were dissolved in DMSO if not stated otherwise. Abiraterone acetate, ketoconazole, vemurafenib, axitinib, cabozantinib, pazopanib, pazopanib HCl, vandetanib, ruxolitinib, tofacitinib, tacrolimus, temsirolimus, clopidogrel, ticagrelor, and ticlopidine were gifts from J. von Kries (Leibniz-Institut für Molekulare Pharmakologie, Germany). Alvocidib, selumetinib, trametinib, talmapimod, everolimus, OSI-027, pimecrolimus, sirolimus, epoprostenol, pictrelisib, SF 1126, and XL147 were purchased from Santa Cruz Biotechnology. Abiraterone, motesanib, regorafenib, tivozanib, and buparsilip were purchased from TRC. Dasatinib, prasugrel, treprostinil, and idealisib were purchased from Cayman Chemical. Sorafenib, sunitinib, and PX-866 were purchased from US Biological. Amantadine hydrochloride and zanamivir (Relenza), both dissolved in water, were purchased from Sigma-Aldrich and GlaxoSmithKline, respectively. Bafilomycin A1 was purchased from Adipogen. Ridaforolimus was purchased from BioAustralis. Dactolisib was purchased from Biovision. Zanamivir was purchased from GlaxoSmithKline and dissolved in water. Dynasore (dissolved in DMSO) and importazole (dissolved in water) were purchased from Sigma-Aldrich. For experimental analysis, small molecules were diluted to final concentration in infection medium (WST-1 assay, mRNA expression analysis, analysis of primary infection (IVs, VSV), virus replication assay), infection medium supplemented with 0.1% hiFCS (analysis of attachment, internalization, fusion, uncoating, and vRNP import), DMEM complete medium (analysis of primary infection of CPXV and HSV1), RPMI 1640 medium supplemented with 2 mM L-glutamine, 25 mM HEPES, 100 U/ml P/S and 10% hiFCS (analysis of primary infection of HTV), or DMEM complete medium containing only 0.5% hiFCS (starvation medium) (analysis of intracellular EGF and transferrin levels).</p> </div> <div class="determinationofantiviralpotencyandcytotoxicityofsmallmolecules" title="sec"> <div class="title" tagx="title" title="title">
Determination of antiviral potency and cytotoxicity of small molecules</div> <p>The antiviral potency and cytotoxicity of small molecules in A549 cells were tested in 384-well format, and all multiwell pipetting steps were performed using a Biomek FX<sup>P</sup> Laboratory Automation Workstation. The antiviral potency and cytotoxicity of small molecules in MLE 12 cells were tested manually in 96-well format. Small molecules were three-fold serially diluted in infection medium generating eight different concentrations. Then, small molecules were applied to the cells and the assays were performed as described above. EC<sub>50</sub> values for A549 cells were calculated using the R software package drc [<a href="#ppat.1007601.ref081">81</a>]. EC<sub>50</sub> values for MLE 12 cells were calculated using Prism (GraphPad Software; <a href="http://www.graphpad.com/">http://www.graphpad.com/</a>). To determine cytotoxicity of small molecules in hAECB, cells were treated as described for MLE 12 cells.</p> </div> <div class="quantificationofviralloadincellculturesupernatantsbyplaqueassay" title="sec"> <div class="title" tagx="title" title="title">
Quantification of viral load in cell culture supernatants by plaque assay</div> <p>To test the effect of small molecules on the viral load of cell culture supernatants, cells were washed with PBS and incubated for 2 h with small molecules under normal culture conditions. Then, cells were inoculated with an MOI of 0.001 of WSN diluted in infection buffer. After 40 min, infection medium containing small molecules containing 1 μg/ml TPCK-treated trypsin (Sigma-Aldrich) were added, and the cells were incubated under normal culture conditions for up to 48 h. In case of therapeutic small molecule application, cells were not pre-treated with small molecules prior to infection, but small molecules were added 4 h post infection. Virus load was determined by plaque assay on MDCK cells using an agar overlay medium [<a href="#ppat.1007601.ref082">82</a>].</p> </div> <div class="analysisofprimaryinfection" title="sec"> <div class="title" tagx="title" title="title">
Analysis of primary infection</div> <p>To analyze primary infection, A549, and HEL cells as well as hAECB were washed with PBS and incubated for 2 h with small molecules (3 μM) or an equivalent of DMSO under normal culture conditions. Then, cells were inoculated with an MOI of 0.1 (CPXV, HSV1, VSV), 0.3 (IVs), or 1.5 (HTV), respectively, for 30 min at 4°C. Unbound virus was removed and the cells cultivated for 6 h (IV, VSV) or 24 h (CPXV, HSV1, HTV) in presence of small molecules. Then, the rate of infected cells was determined by microscopically quantifying the number of cells positive for viral antigen. In case of IVs and VSV, presence of the NP protein was investigated by indirect immunofluorescence analysis using primary antibodies MCA400 (IAVs, AbD Serotec), MCA403 (IBVs, AbD Serotec), polyclonal-anti hantavirus N protein and V5507 (VSV, Sigma-Aldrich). In case of CPXV and HSV1, presence of GFP was investigated by fluorescence analysis. Microscopic analysis was performed as described previously [<a href="#ppat.1007601.ref011">11</a>].</p> </div> <div class="labelingofivwithalexafluor647" title="sec"> <div class="title" tagx="title" title="title">
Labeling of IV with Alexa Fluor 647</div> <p>500 μg/ml concentrate of IV strain PAN were mixed with PBS up to 50 μl and 2 ng AlexaFluor647-NHS-Ester (Thermo Fischer) and incubated over night at 4°C on a shaker at 300 rpm. To separate virus and dye, virus was precipitated by centrifugation (25,000 x g, 15 min, 4°C) and the pellet was resuspended in 100 μl PBS. Finally, virus was filtered using a 0.45 μm Filter Unit (EMD Millipore).</p> </div> <div class="analysisofvirusattachment" title="sec"> <div class="title" tagx="title" title="title">
Analysis of virus attachment</div> <p>A549 cells were washed with PBS and pre-incubated for 30 min with small molecules (3 μM), an equivalent of DMSO, or with 1.5 U/ml of neuraminidase from <i>Clostridium perfringens</i> (Sigma-Aldrich), respectively, under normal culture conditions. Cells were inoculated for 1 h at 4°C with 0.002 hemagglutination units per cell of IAV strain PAN labeled with Alexa Fluor 647 dye as described in ‘Labeling of IV with Alexa Fluor 647’. After removing unbound virus by washing with ice-cold PBS twice, cells were analyzed for virus attachment either qualitatively by fluorescence microscopy or quantitatively by flow cytometry. For quantitative analysis, cells were harvested by scraping and precipitated by centrifugation (1200 x g, 5 min, 4°C). For both analyses, cells were fixed by incubation with 4% paraformaldehyde. For image acquisition, nuclei were stained with DAPI and mounted in Mowiol (Sigma-Aldrich), whereas samples for flow cytometry analyses were precipitated and resuspended in PBS again. Finally, either representative images were acquired on an inverted FV-1000 MPE laser scanning microscope (Olympus) using a 60x UPlanSApo water objective (numerical aperture 1.2) (Olympus) or Alexa Fluor 647 intensity was determined using a FACSAria II flow cytometer (Becton Dickinson). Three independent experiments were performed in duplicates and for each duplicate measurement 10,000 cells were analyzed by flow cytometry.</p> </div> <div class="analysisofvirusinternalization" title="sec"> <div class="title" tagx="title" title="title">
Analysis of virus internalization</div> <p>A549 cells were pre-treated with small molecules and infected as described in ‘Analysis of virus attachment’. Dynasore was applied at 100 μM. One hour after inoculation, cells were incubated in presence of small molecules for an additional 45 min under normal culture conditions. Then, cells were analyzed for virus internalization either qualitatively by fluorescence microscopy or quantitatively by flow cytometry. For quantitative analysis, cells were harvested by scraping, precipitated by centrifugation (1200 x g, 5 min, 4°C) and fixed with 4% paraformaldehyde. For both analyses, cells were blocked with PBS containing 3% BSA for 2 h at room temperature. Subsequently, the cells were incubated with an anti-H3N2 HA protein antiserum (ViroStat) diluted 1:500 in PBS containing 1% BSA at 4°C overnight, followed by incubation with an Alexa Fluor 488-labeled secondary antibody (Thermo Fischer) diluted 1:1000 in PBS containing 1% BSA at room temperature for 3 h. Finally, either representative images were acquired as described for ‘Analysis of virus attachment’ (qualitative analysis) or Alexa Fluor 647 and Alexa Fluor 488 intensities were determined using a FACSAria II flow cytometer (Becton Dickinson) (quantitative analysis). Three independent experiments were performed in duplicates and per duplicate measurement, 10,000 cells were analyzed by flow cytometry.</p> </div> <div class="analysisofintracellularepidermalgrowthfactor(egf)andtransferrinlevels" title="sec"> <div class="title" tagx="title" title="title">
Analysis of intracellular epidermal growth factor (EGF) and transferrin levels</div> <p>A549 cells were serum-starved for 3 h by incubation with starvation medium. Then, cells were incubated for 30 min with small molecules (dynasore: 100 μM, regorafenib/sorafenib: 3 μM) or an equivalent of DMSO, respectively, under normal culture conditions. Afterwards, cells were transferred to 4°C and washed twice with cold PBS and incubated with 100 ng/ml Alexa Fluor 647-labeled EGF (Thermo Fischer) and 25 μg/ml Alexa Fluor 488-labeled transferrin (Thermo Fischer) diluted in starvation medium for 1 h at 4°C. Cells were pulsed for 10 min at 37°C, washed three times with PBS and chased for different periods of time (0, 30, 60, and 120 min) in starvation medium containing small molecules but lacking EGF and transferrin. After the chase period, cells were transferred to 4°C and washed once with cold acid buffer (150 mM NaCl, 25 mM NaOH, pH 3.5) to remove surface-bound EGF and transferrin. Then, cells were washed twice with cold PBS, harvested by scraping and precipitated by centrifugation (1200 x g, 5 min, 4°C). Finally, cells were fixed with 4% paraformaldehyde and analyzed for Alexa Fluor 647 and Alexa Fluor 488 intensities as described for ‘Analysis of virus attachment’. Three independent experiments were performed in duplicates and per duplicate measurement, 10,000 cells were analyzed by flow cytometry.</p> </div> <div class="analysisoffusion" title="sec"> <div class="title" tagx="title" title="title">
Analysis of fusion</div> <p>A549 cells grown in glass-bottom dishes (35 mm, uncoated, MatTek Cooperation) were washed once with PBS and pre-incubated with small molecules for 30 min (bafilomycin A1: 200 nM; regorafenib and sorafenib: 3 μM) or an equivalent of DMSO, respectively, and Hoechst under normal culture conditions. Then, cells were inoculated at 4°C with 62.5 μg/ml of IAV strain PAN simultaneously labeled with DiOC18 and DiI as described previously [<a href="#ppat.1007601.ref029">29</a>]. After 15 min, the cells were washed three times with cold PBS and subsequently cultivated under normal culture conditions in the presence of small molecules. Cells were live-imaged for 65 min as described for ‘Analysis of virus attachment’. For quantification, images of three independent experiments at indicated time points were summarized on the z-axis and particle recognition was performed using ImageJ (<a href="http://imagej.nih.gov/">http://imagej.nih.gov/</a>). Per image, 60 to 80 cells, containing 300 to 500 particles in total, were analyzed for changes in DiOC18 intensity over time.</p> </div> <div class="analysisofuncoatingandvrnpimport" title="sec"> <div class="title" tagx="title" title="title">
Analysis of uncoating and vRNP import</div> <p>A549 cells grown on glass slides were washed with PBS and pre-incubated for 30 min with small molecules (3 μM) or an equivalent of DMSO, under normal culture conditions. Then, cells were transferred to 4°C, washed twice with cold PBS, and inoculated with an MOI of 5 of IAV strain PAN. After 1 h, cells were washed three times with cold PBS and incubated for 3 h (uncoating) or 4.5 h (vRNP import) in the presence of small molecule inhibitors and 100 μM cycloheximide (Sigma-Aldrich) under normal culture conditions. Then, cells were fixed by incubation with 4% paraformaldehyde for 20 min at room temperature, permeabilized with PBS containing 0.5% Triton X-100 for 15 min, and blocked with PBS containing 3% BSA for 4 h. Cells were incubated overnight at 4°C with primary antibodies targeting either the IAV M1 (MCA401, AbD Serotec) or NP (EMD Millipore) proteins, respectively, diluted 1:500 in PBS containing 1% BSA, followed by a 5 h incubation step with an Alexa Fluor 488-labeled secondary antibody at room temperature. Finally, nuclei were stained by 30 min incubation with DAPI at room temperature and the slides mounted in Mowiol (Sigma-Aldrich). Fluorescence microscopic analysis was conducted as described for ‘Analysis of virus attachment’. Images of uncoating assay were segmented with CellProfiler 2.1.1 (<a href="http://cellprofiler.org/">http://cellprofiler.org/</a>) and classification of the subcellular distribution of the M1 protein (i.e. dispersed vs. dot-like) was performed using the CellProfiler Analyst 2.0 (<a href="http://cellprofiler.org/">http://cellprofiler.org/</a>). Three independent experiments, performed in duplicates, were subjected to analysis, and per experiment approx. 400 cells were investigated.</p> </div> <div class="mrnaexpressionanalysis" title="sec"> <div class="title" tagx="title" title="title">
mRNA expression analysis</div> <p>A549 cells were washed with PBS and pre-incubated for 2 h with small molecules (3 μM) or an equivalent of DMSO under normal culture conditions. Then, cells were inoculated with an MOI of 0.3 of IAV strain WSN for 30 min at 4°C. Subsequently, unbound virus was removed and the cells were cultivated for 4 h in the presence of small molecules. Total RNA was isolated from the cells using the GeneJET RNA purification kit (Thermo Fischer) according to the manufacturer’s instructions. For reverse transcription, oligo(dT) primer was used. Resulting cDNAs were quantified by quantitative PCR with oligonucleotides specific for viral NP mRNA (5’-CTCGTCGCTTATGACAAAGAAG-3’ and 5’-AGATCATCATGTGAGTCAGAC-3’), viral M1 mRNA (5’-GACCAATCCTGTCACCTC-3’ and 5’-GATCTCCGTTCCCATTAAGAG-3’), viral M2 mRNA (5’-GAGGTCGAAACGCCTAT-3’ and 5’-CTCCAGCTCTATGTTGACAAA-3’), and cellular GAPDH mRNA (5’-GGTATCGTGGAAGGACTCATGAC-3’ and 5’-ATGCCAGTGAGCTTCCCGTTCAG-3’). Data were analyzed as described previously [<a href="#ppat.1007601.ref083">83</a>] and levels of GAPDH were used for normalization.</p> </div> <div class="analysisoffusionph" title="sec"> <div class="title" tagx="title" title="title">
Analysis of fusion pH</div> <p>Fusion pH was determined as described previously [<a href="#ppat.1007601.ref084">84</a>]. In brief, human red blood cell ghosts were incubated with R18-labeled virus of strains PAN, THW, and MAL. Unbound virus was removed and the virus-ghost suspension incubated at different pH values. Finally, the fluorescence dequenching of R18 was recorded and used to determine the EC<sub>50</sub>, which specifies the fusion pH, and the Hill coefficient.</p> </div> <div class="mouseexperimentation" title="sec"> <div class="title" tagx="title" title="title">
Mouse experimentation</div> <p>Female BALB/c mice (Charles River, Sulzfeld, Germany) were housed under pathogen-free conditions in biosafety level 2, according to the German Animal Protection Law (Tierschutzgesetz TierSchG). The animal experiments were approved by the local authorities (Landesamt für Gesundheit und Soziales Berlin LAGeSo, Reg No.: G0206/12). Eight-week-old female mice orally received regorafenib (100 mg/kg, a dose shown to be not toxic for mice [<a href="#ppat.1007601.ref015">15</a>]) and vehicle control, respectively, formulated as solution in PEG400/125 mM aqueous methanesulfonic acid (80/20) with seven mice per group. Four h later, mice were intranasally infected with 4.4 x 10<sup>4</sup> PFU of IAV strain A/England/195/2009(H1N1). Twenty-four h after the first dose, regorafenib and vehicle, respectively, were administered again. The mice were euthanized 48 h p.i. and the viral load in the lungs was determined. To this end, the lungs were homogenized, cellular debris was removed by centrifugation, and the amount of infectious virus progeny was determined by plaque assay as described above.</p> </div> <div class="resistanceassay" title="sec"> <div class="title" tagx="title" title="title">
Resistance assay</div> <p>MDCK cells were washed with PBS and pre-incubated for 2 h with the different chemical compounds at the indicated concentrations or an equivalent of DMSO under normal culture conditions. Then, cells were inoculated with an MOI of 0.001 of IAV strain A/England/195/2009–M2<sub>N31S</sub> or WSN for 30 min at 4°C. Unbound virus was removed and the cells cultivated for 24 h in the presence of small molecules and TPCK-treated trypsin. Tissue culture supernatants were harvested, cells and debris removed by centrifugation (5 min, 4°C, 1,400 x g), and the supernatants stored at 4°C. Virus load in cleared supernatants was quantified by plaque assay. This procedure was performed five (WSN) or eight (A/England/195/2009–M2<sub>N31S</sub>) times, always using the supernatant of the latest passage for inoculation.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting information</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Fig</span> <div class="title" tagx="title" title="title">
Graphical representation of strain-independent genes and processes.</div> <p>The top ten ranking clusters according to DAVID functional annotation clustering (<a href="#ppat.1007601.g001">Fig 1D</a>) (light blue) and the top ten ranking canonical pathways according to IPA (<a href="#ppat.1007601.g001">Fig 1E</a>) (dark blue), the corresponding strain-independent genes (red) and the higher order cellular processes (green) have been illustrated as interaction map.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="ppat.1007601.s001.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S2 Fig</span> <div class="title" tagx="title" title="title">
Strain-specifically required genes can be assigned to one of four clusters.</div> <p>(A) Strain-specific genes were identified by mixed effects analysis and clustered using the CLICK algorithm [<a href="#ppat.1007601.ref079">79</a>]. Data represent mean and standard deviation (SD) of the normalized viral load upon knockdown of the genes in the individual clusters. A.u.: arbitrary units. Twenty-one genes (B) could not be assigned to any cluster. Data represent the mean of the normalized viral load for the siRNAs targeting the individual genes. Data analyzed are from the screen outlined in <a href="#ppat.1007601.g001">Fig 1A</a>.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="ppat.1007601.s002.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S3 Fig</span> <div class="title" tagx="title" title="title">
Several gene categories are strain-specifically required.</div> <p>Strain-specific gene categories were identified by mixed effects analysis. Exemplary gene categories are shown. Data represent average virus titers upon knockdown of genes of the respective categories relative to negative control. Number of tested genes and siRNAs associated with the respective category as well as p-value of mixed effects analysis are specified in boxes. (A) Gene Ontology (GO) category nucleotide-binding domain, leucine rich repeat containing receptor signaling pathway. This pathway activates NF-кB [<a href="#ppat.1007601.ref085">85</a>]. (B) GO category regulation of RNA splicing. (C) GO category RNA polymerase II transcription cofactor activity.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="ppat.1007601.s003.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S4 Fig</span> <div class="title" tagx="title" title="title">
CRISPR/Cas9-mediated knockout effects on viral replication for target genes of regorafenib/sorafenib.</div> <p>A549-CRISPR/Cas9 cells were infected with WSN for 36 h. Virus load was assessed by fluorescent focus assay. Genes selected are major targets of regorafenib/sorafenib [<a href="#ppat.1007601.ref015">15</a>, <a href="#ppat.1007601.ref016">16</a>, <a href="#ppat.1007601.ref023">23</a>]. Data represent average virus titers ± SEM of technical replicates (n = 3).</p> <p>(PDF)</p>  <div class="media" title="media"><a href="ppat.1007601.s004.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S5 Fig</span> <div class="title" tagx="title" title="title">
UBKIs do not affect internalization of CME cargos.</div> <p>(A) A549 cells were serum-starved for 3 h and subsequently pre-treated with small molecules (dynasore: 100 μM, regorafenib/sorafenib: 3 μM) or an equivalent amount of DMSO for 30 min. Cells were incubated at 4°C with Alexa Fluor 647-labeled epidermal growth factor (EGF) for 1 h. To induce internalization of EGF, cells were incubated at 37°C for 10 min. The amount of internalized EGF was quantified by flow cytometry. Data represent mean ± SEM of n = 3 independent experiments specified in arbitrary units (a.u). The one-way ANOVA of the log-transformed data provided evidence for different mean values (p = 0.052). Unadjusted post-tests led to a significant difference between DMSO and dynasore (p = 0.024). The adjusted p-value for comparison with DMSO was 0.071 for dynasore and non-significant (ns) for regorafenib and sorafenib. (B) Cells treated as in (A) but using Alexa Fluor 488-labeled transferrin. One-way ANOVA of the log-transformed data suggests significantly different mean values (p = 0.028). In contrast to regorafenib and sorafenib, adjusted post-tests for multiple testing led to a significant difference between DMSO and dynasore (p = 0.037).</p> <p>(PDF)</p>  <div class="media" title="media"><a href="ppat.1007601.s005.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S6 Fig</span> <div class="title" tagx="title" title="title">
UBKIs impair post-internalization processing of CME cargos.</div> <p>(A) A549 cells were pre-treated with small molecules or DMSO as described for <a href="#ppat.1007601.g004">Fig 4</a> before incubation at 4°C with EGF-A647. After a 10 min pulse, cells were further incubated at 37°C for 30, 60, or 120 min with EGF-free medium before fixation. The amount of internalized EGF-A647 was quantified by flow cytometry. Data represent mean (n = 3) ± SEM of independent experiments relative to obtained values after 10 min. (B) Same experimental setup as in (A) but using transferrin-Alexa-488. Two-way ANOVA for (A) and (B) suggests that time and group are significant factors, whereas the interaction is not significant. Comparison with the DMSO control at the respective time point was adjusted for multiple testing: *: p-value ≤ 0.05, **: p-value ≤ 0.01.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="ppat.1007601.s006.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S7 Fig</span> <div class="title" tagx="title" title="title">
UBKIs impair vRNP nuclear import.</div> <p>Data were acquired as described in the legend of <a href="#ppat.1007601.g005">Fig 5F</a>. Representative micrographs of the x-y plane (large) and the z-axis (narrow) of individual cells are shown. The horizontal z-stacks are identical to those shown in <a href="#ppat.1007601.g005">Fig 5F</a>.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="ppat.1007601.s007.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S8 Fig</span> <div class="title" tagx="title" title="title">
Fusion pH of representative IV strains.</div> <p>(A) Virus of strains PAN, THW, and MAL were labeled with the lipophilic dye R18. Labeled viruses were incubated with human red blood cell ghosts followed by incubation at different pH values. Finally, fluorescence dequenching (FDQ) of R18 was recorded. A.u.: arbitrary units (B) The EC<sub>50</sub> (which defines the fusion pH) and the Hill coefficient of the curves depicted in (A) are shown. EC<sub>50</sub>: pH at which FDQ is half maxima. SEM of EC<sub>50</sub> and Hill coefficient, respectively, are standard errors determined by nonlinear regression.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="ppat.1007601.s008.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S9 Fig</span> <div class="title" tagx="title" title="title">
Cell viability dose-response curves in different cell types.</div> <p>Cells were cultivated for 48 h in presence of small molecules at different concentrations prior to conduction of WST-1 assay. Data represent signal in WST-1 assay relative to the vehicle control expressed as mean ± SEM of n = 3 technical replicates. (A) HEL cell-derived megakaryocytes. (B) hAECB. (C) MDCK cells. (D) To test for potential cytotoxicity of FLT4 inhibitors at the concentration used in experiments shown in <a href="#ppat.1007601.g006">Fig 6</a>, both inhibitors were added to cultures for 48 h at a concentration of 6.25 μM. Data represent mean ± SEM of n = 3 independent experiments (mean values are not significantly different: one-way ANOVA: p = 0.697).</p> <p>(PDF)</p>  <div class="media" title="media"><a href="ppat.1007601.s009.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S10 Fig</span> <div class="title" tagx="title" title="title">
Antiviral efficacy of UBKIs in murine cells and mice.</div> <p>(A and B) Dose-response curves in MLE 12 cells. For determination of virus replication, MLE 12 cells were pre-treated with regorafenib (A) or sorafenib (B) at different concentrations for 2 h, infected with IV strain WSN and cultivated for 36 h in presence of small molecules. Virus titers in tissue culture supernatants were determined as described in ‘Materials and Methods’. Data represent mean ± SEM of n = 3 technical replicates. For determination of cell viability, MLE 12 cells were cultivated for 36 h in the presence of small molecules at different concentrations prior to WST-1 assay. Data indicate signal intensity in WST-1 assay relative to vehicle control and represent mean (n = 2) ± SEM. (C) Eight-week-old female mice (n = 7) were orally treated with regorafenib and vehicle control, respectively, with seven mice per group. Four h later, mice were intranasally infected with 4.4 x 10<sup>4</sup> PFU of IAV strain A/England/195/2009(H1N1). Twenty-four h after the first dose, regorafenib and vehicle, respectively, were administered again. The mice were euthanized 48 h p.i. and the viral load in the lung homogenates determined. Individual values, mean and SEM are presented. The virus load of treated animals was not significantly different to controls (p = 0.317, Wilcoxon rank-sum (Mann-Whitney) test).</p> <p>(PDF)</p>  <div class="media" title="media"><a href="ppat.1007601.s010.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S11 Fig</span> <div class="title" tagx="title" title="title">
UBKIs impose a barrier against resistance development.</div> <p>MDCK cells were pre-treated with 3 μM regorafenib or an equivalent amount of DMSO for 2 h, infected with MOI 0.001 of IV strain WSN and cultivated for 24 h in presence of reagents. Virus titer in tissue culture supernatants was determined and the supernatants were used for inoculation of the next passage. The experiment was terminated after 5 passages. Data represent mean ± SD of technical replicates.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="ppat.1007601.s011.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S12 Fig</span> <div class="title" tagx="title" title="title">
Missing correlation of IFN induction with virus replication.</div> <p>Based on the siRNA level data of the WSN screen the IFN induction data were plotted vs. the virus replication data. Since no correlation of signals in the IFN induction assay and the virus replication assay was observed, all sample siRNAs were treated as non-inducers of IFN.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="ppat.1007601.s012.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S1 Table</span> <div class="title" tagx="title" title="title">
Composition of the siRNA library.</div> <p>(DOCX)</p>  <div class="media" title="media"><a href="ppat.1007601.s013.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S2 Table</span> <div class="title" tagx="title" title="title">
Result of virus replication siRNA screen on siRNA level.</div> <p>(XLSX)</p>  <div class="media" title="media"><a href="ppat.1007601.s014.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S3 Table</span> <div class="title" tagx="title" title="title">
Result of virus replication siRNA screen on gene level.</div> <p>(XLSX)</p>  <div class="media" title="media"><a href="ppat.1007601.s015.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S4 Table</span> <div class="title" tagx="title" title="title">
Gene enrichment analysis with DAVID.</div> <p>(XLSX)</p>  <div class="media" title="media"><a href="ppat.1007601.s016.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S5 Table</span> <div class="title" tagx="title" title="title">
Gene enrichment analysis with IPA.</div> <p>(XLSX)</p>  <div class="media" title="media"><a href="ppat.1007601.s017.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S6 Table</span> <div class="title" tagx="title" title="title">
Differentially required gene categories associated with NF-kB activation.</div> <p>(XLSX)</p>  <div class="media" title="media"><a href="ppat.1007601.s018.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S7 Table</span> <div class="title" tagx="title" title="title">
Differentially required gene categories associated with regulation of RNA splicing.</div> <p>(XLSX)</p>  <div class="media" title="media"><a href="ppat.1007601.s019.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S8 Table</span> <div class="title" tagx="title" title="title">
Differentially required gene categories associated with cellular RNA polymerase activity.</div> <p>(XLSX)</p>  <div class="media" title="media"><a href="ppat.1007601.s020.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S9 Table</span> <div class="title" tagx="title" title="title">
Genes significantly different required between the four virus strains.</div> <p>(XLSX)</p>  <div class="media" title="media"><a href="ppat.1007601.s021.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S10 Table</span> <div class="title" tagx="title" title="title">
Gene groups significantly different required between the four virus strains.</div> <p>(XLSX)</p>  <div class="media" title="media"><a href="ppat.1007601.s022.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S11 Table</span> <div class="title" tagx="title" title="title">
Result of small molecule screen.</div> <p>(DOCX)</p>  <div class="media" title="media"><a href="ppat.1007601.s023.docx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">S12 Table</span> <div class="title" tagx="title" title="title">
GuideRNA sequences for CRISPR/Cas9-mediated gene deletion.</div> <p>(XLSX)</p>  <div class="media" title="media"><a href="ppat.1007601.s024.xlsx">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>We are grateful to Pedro Alves for support with creation of the A549-IFN-Luc cells. We thank Thorsten Wolff for providing IAV strains A/Puerto Rico/8/1934(H1N1), A/Panama/2007/1999(H3N2), and A/Mallard/Germany/439/2004(H3N2) as well as vesicular stomatitis virus strain Indiana, Stefan Becker for providing IAV strain A/Hamburg/04/2009(H1N1), Stephan Ludwig for providing IAV strain A/FPV/Bratislava/1979(H7N7), Avir Greenhills Biotechnology for providing IBV strain B/Thuringia/02/2006xB/Vienna/33/2006, Timo Frensing for providing IBV strain B/Brisbane/60/2008, Karsten Tischer for providing cowpox virus strain Brighton Red, and Prashant Desai for providing herpes simplex virus 1 strain KOS K26GFP. We also thank the St. Jude Children’s Research Hospital and Yoshirio Kawaoka for the gift of plasmids and Jens von Kries for the gift of small molecules. Furthermore, we thank Jan-David Manntz and Marina Drabkina for excellent technical support and Rike Zietlow for editing the manuscript.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="ppat.1007601.ref001" /><span class="label" tagx="label" title="label">1</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">WHO. Influenza (Seasonal), Fact sheet N°211. 2014.</span></li> <li tag="ref"><a name="ppat.1007601.ref002" /><span class="label" tagx="label" title="label">2</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salomon</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webster</span><span class="given-names" tagx="given-names" title="given-names">RG</span></span>. <span class="mixed-article-title" title="mixed-article-title">The influenza virus enigma</span>. <span class="source" tagx="source" title="source">Cell</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">136</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">402</span>–<span class="lpage" tagx="lpage" title="lpage">10</span>. Epub 2009/02/11. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cell.2009.01.029">10.1016/j.cell.2009.01.029</a></span> ; PubMed Central PMCID: PMCPmc2971533.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19203576">19203576</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref003" /><span class="label" tagx="label" title="label">3</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hayden</span><span class="given-names" tagx="given-names" title="given-names">FG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Jong</span><span class="given-names" tagx="given-names" title="given-names">MD</span></span>. <span class="mixed-article-title" title="mixed-article-title">Emerging influenza antiviral resistance threats</span>. <span class="source" tagx="source" title="source">J Infect Dis</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">203</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">6</span>–<span class="lpage" tagx="lpage" title="lpage">10</span>. Epub 2010/12/15. <span class="pub-id"><a href="https://dx.doi.org/10.1093/infdis/jiq012">10.1093/infdis/jiq012</a></span> ; PubMed Central PMCID: PMCPmc3086431.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21148489">21148489</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref004" /><span class="label" tagx="label" title="label">4</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hurt</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holien</span><span class="given-names" tagx="given-names" title="given-names">JK</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parker</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kelso</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barr</span><span class="given-names" tagx="given-names" title="given-names">IG</span></span>. <span class="mixed-article-title" title="mixed-article-title">Zanamivir-resistant influenza viruses with a novel neuraminidase mutation</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">83</span>(<span class="issue" tagx="issue" title="issue">20</span>):<span class="fpage" tagx="fpage" title="fpage">10366</span>–<span class="lpage" tagx="lpage" title="lpage">73</span>. Epub 2009/07/31. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.01200-09">10.1128/JVI.01200-09</a></span> ; PubMed Central PMCID: PMCPmc2753113.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19641000">19641000</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref005" /><span class="label" tagx="label" title="label">5</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Medina</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garcia-Sastre</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>. <span class="mixed-article-title" title="mixed-article-title">Influenza A viruses: new research developments</span>. <span class="source" tagx="source" title="source">Nature reviews Microbiology</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">9</span>(<span class="issue" tagx="issue" title="issue">8</span>):<span class="fpage" tagx="fpage" title="fpage">590</span>–<span class="lpage" tagx="lpage" title="lpage">603</span>. Epub 2011/07/13. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nrmicro2613">10.1038/nrmicro2613</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21747392">21747392</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref006" /><span class="label" tagx="label" title="label">6</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Der Ryst</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>. <span class="mixed-article-title" title="mixed-article-title">Maraviroc—A CCR5 Antagonist for the Treatment of HIV-1 Infection</span>. <span class="source" tagx="source" title="source">Front Immunol</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">6</span>:<span class="fpage" tagx="fpage" title="fpage">277</span> Epub 2015/06/23. <span class="pub-id"><a href="https://dx.doi.org/10.3389/fimmu.2015.00277">10.3389/fimmu.2015.00277</a></span> ; PubMed Central PMCID: PMCPmc4456946.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26097475">26097475</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref007" /><span class="label" tagx="label" title="label">7</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gabriel</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klingel</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Otte</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thiele</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hudjetz</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arman-Kalcek</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Differential use of importin-alpha isoforms governs cell tropism and host adaptation of influenza virus</span>. <span class="source" tagx="source" title="source">Nat Commun</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">2</span>:<span class="fpage" tagx="fpage" title="fpage">156</span> Epub 2011/01/20. <span class="pub-id"><a href="https://dx.doi.org/10.1038/ncomms1158">10.1038/ncomms1158</a></span> ; PubMed Central PMCID: PMCPmc3105303.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21245837">21245837</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref008" /><span class="label" tagx="label" title="label">8</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hudjetz</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gabriel</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>. <span class="mixed-article-title" title="mixed-article-title">Human-like PB2 627K influenza virus polymerase activity is regulated by importin-alpha1 and -alpha7</span>. <span class="source" tagx="source" title="source">PLoS Pathog</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">8</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">e1002488</span> Epub 2012/01/26. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1002488">10.1371/journal.ppat.1002488</a></span> ; PubMed Central PMCID: PMCPmc3262014.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22275867">22275867</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref009" /><span class="label" tagx="label" title="label">9</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Janssen</span><span class="given-names" tagx="given-names" title="given-names">HL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reesink</span><span class="given-names" tagx="given-names" title="given-names">HW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lawitz</span><span class="given-names" tagx="given-names" title="given-names">EJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zeuzem</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rodriguez-Torres</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patel</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Treatment of HCV infection by targeting microRNA</span>. <span class="source" tagx="source" title="source">N Engl J Med</span>. <span class="year" tagx="year" title="year">2013</span>;<span class="volume" tagx="volume" title="volume">368</span>(<span class="issue" tagx="issue" title="issue">18</span>):<span class="fpage" tagx="fpage" title="fpage">1685</span>–<span class="lpage" tagx="lpage" title="lpage">94</span>. Epub 2013/03/29. <span class="pub-id"><a href="https://dx.doi.org/10.1056/NEJMoa1209026">10.1056/NEJMoa1209026</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23534542">23534542</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref010" /><span class="label" tagx="label" title="label">10</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ottosen</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parsley</span><span class="given-names" tagx="given-names" title="given-names">TB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zeh</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Doorn</span><span class="given-names" tagx="given-names" title="given-names">LJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van der Veer</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">In vitro antiviral activity and preclinical and clinical resistance profile of miravirsen, a novel anti-hepatitis C virus therapeutic targeting the human factor miR-122</span>. <span class="source" tagx="source" title="source">Antimicrob Agents Chemother</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">59</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">599</span>–<span class="lpage" tagx="lpage" title="lpage">608</span>. Epub 2014/11/12. <span class="pub-id"><a href="https://dx.doi.org/10.1128/AAC.04220-14">10.1128/AAC.04220-14</a></span> ; PubMed Central PMCID: PMCPmc4291405.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25385103">25385103</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref011" /><span class="label" tagx="label" title="label">11</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karlas</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Machuy</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shin</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pleissner</span><span class="given-names" tagx="given-names" title="given-names">KP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Artarini</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heuer</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Genome-wide RNAi screen identifies human host factors crucial for influenza virus replication</span>. <span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">463</span>(<span class="issue" tagx="issue" title="issue">7282</span>):<span class="fpage" tagx="fpage" title="fpage">818</span>–<span class="lpage" tagx="lpage" title="lpage">22</span>. Epub 2010/01/19. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nature08760">10.1038/nature08760</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20081832">20081832</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref012" /><span class="label" tagx="label" title="label">12</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Watanabe</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Watanabe</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawaoka</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>. <span class="mixed-article-title" title="mixed-article-title">Cellular networks involved in the influenza virus life cycle</span>. <span class="source" tagx="source" title="source">Cell Host Microbe</span>. <span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">7</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">427</span>–<span class="lpage" tagx="lpage" title="lpage">39</span>. Epub 2010/06/15. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.chom.2010.05.008">10.1016/j.chom.2010.05.008</a></span> ; PubMed Central PMCID: PMCPmc3167038.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20542247">20542247</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref013" /><span class="label" tagx="label" title="label">13</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perez</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benitez</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kandasamy</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Genome-wide CRISPR/Cas9 Screen Identifies Host Factors Essential for Influenza Virus Replication</span>. <span class="source" tagx="source" title="source">Cell reports</span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">23</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">596</span>–<span class="lpage" tagx="lpage" title="lpage">607</span>. Epub 2018/04/12. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.celrep.2018.03.045">10.1016/j.celrep.2018.03.045</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29642015">29642015</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref014" /><span class="label" tagx="label" title="label">14</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ashburn</span><span class="given-names" tagx="given-names" title="given-names">TT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thor</span><span class="given-names" tagx="given-names" title="given-names">KB</span></span>. <span class="mixed-article-title" title="mixed-article-title">Drug repositioning: identifying and developing new uses for existing drugs</span>. <span class="source" tagx="source" title="source">Nat Rev Drug Discov</span>. <span class="year" tagx="year" title="year">2004</span>;<span class="volume" tagx="volume" title="volume">3</span>(<span class="issue" tagx="issue" title="issue">8</span>):<span class="fpage" tagx="fpage" title="fpage">673</span>–<span class="lpage" tagx="lpage" title="lpage">83</span>. Epub 2004/08/03. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nrd1468">10.1038/nrd1468</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15286734">15286734</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref015" /><span class="label" tagx="label" title="label">15</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilhelm</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dumas</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adnane</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lynch</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carter</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schutz</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Regorafenib (BAY 73–4506): a new oral multikinase inhibitor of angiogenic, stromal and oncogenic receptor tyrosine kinases with potent preclinical antitumor activity</span>. <span class="source" tagx="source" title="source">Int J Cancer</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">129</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">245</span>–<span class="lpage" tagx="lpage" title="lpage">55</span>. Epub 2010/12/21. <span class="pub-id"><a href="https://dx.doi.org/10.1002/ijc.25864">10.1002/ijc.25864</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21170960">21170960</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref016" /><span class="label" tagx="label" title="label">16</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hotte</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hirte</span><span class="given-names" tagx="given-names" title="given-names">HW</span></span>. <span class="mixed-article-title" title="mixed-article-title">BAY 43–9006: early clinical data in patients with advanced solid malignancies</span>. <span class="source" tagx="source" title="source">Curr Pharm Des</span>. <span class="year" tagx="year" title="year">2002</span>;<span class="volume" tagx="volume" title="volume">8</span>(<span class="issue" tagx="issue" title="issue">25</span>):<span class="fpage" tagx="fpage" title="fpage">2249</span>–<span class="lpage" tagx="lpage" title="lpage">53</span>. Epub 2002/10/09. .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12369852">12369852</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref017" /><span class="label" tagx="label" title="label">17</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shaw</span><span class="given-names" tagx="given-names" title="given-names">ML</span></span>. <span class="mixed-article-title" title="mixed-article-title">The host interactome of influenza virus presents new potential targets for antiviral drugs</span>. <span class="source" tagx="source" title="source">Rev Med Virol</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">21</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">358</span>–<span class="lpage" tagx="lpage" title="lpage">69</span>. Epub 2011/08/09. <span class="pub-id"><a href="https://dx.doi.org/10.1002/rmv.703">10.1002/rmv.703</a></span> ; PubMed Central PMCID: PMCPmc3207218.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21823192">21823192</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref018" /><span class="label" tagx="label" title="label">18</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">XD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Santini</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lacson</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marine</span><span class="given-names" tagx="given-names" title="given-names">SD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benetti</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">cSSMD: assessing collective activity for addressing off-target effects in genome-scale RNA interference screens</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">27</span>(<span class="issue" tagx="issue" title="issue">20</span>):<span class="fpage" tagx="fpage" title="fpage">2775</span>–<span class="lpage" tagx="lpage" title="lpage">81</span>. Epub 2011/08/19. <span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btr474">10.1093/bioinformatics/btr474</a></span> ; PubMed Central PMCID: PMCPmc3202303.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21846737">21846737</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref019" /><span class="label" tagx="label" title="label">19</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang da</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sherman</span><span class="given-names" tagx="given-names" title="given-names">BT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lempicki</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span>. <span class="mixed-article-title" title="mixed-article-title">Bioinformatics enrichment tools: paths toward the comprehensive functional analysis of large gene lists</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">37</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">1</span>–<span class="lpage" tagx="lpage" title="lpage">13</span>. Epub 2008/11/27. <span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkn923">10.1093/nar/gkn923</a></span> ; PubMed Central PMCID: PMCPmc2615629.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19033363">19033363</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref020" /><span class="label" tagx="label" title="label">20</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="mixed-article-title" title="mixed-article-title">Gene Ontology Consortium. Gene Ontology Consortium: going forward</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">43</span>(Database issue):<span class="fpage" tagx="fpage" title="fpage">D1049</span>–<span class="lpage" tagx="lpage" title="lpage">56</span>. Epub 2014/11/28. <span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gku1179">10.1093/nar/gku1179</a></span> ; PubMed Central PMCID: PMCPmc4383973.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25428369">25428369</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref021" /><span class="label" tagx="label" title="label">21</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kanehisa</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goto</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>. <span class="mixed-article-title" title="mixed-article-title">KEGG: kyoto encyclopedia of genes and genomes</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="year" tagx="year" title="year">2000</span>;<span class="volume" tagx="volume" title="volume">28</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">27</span>–<span class="lpage" tagx="lpage" title="lpage">30</span>. Epub 1999/12/11. ; PubMed Central PMCID: PMCPmc102409.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10592173">10592173</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref022" /><span class="label" tagx="label" title="label">22</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Croft</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Kelly</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haw</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gillespie</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matthews</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Reactome: a database of reactions, pathways and biological processes</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">39</span>(Database issue):<span class="fpage" tagx="fpage" title="fpage">D691</span>–<span class="lpage" tagx="lpage" title="lpage">7</span>. Epub 2010/11/12. <span class="pub-id"><a href="https://dx.doi.org/10.1093/nar/gkq1018">10.1093/nar/gkq1018</a></span> ; PubMed Central PMCID: PMCPmc3013646.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21067998">21067998</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref023" /><span class="label" tagx="label" title="label">23</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anastassiadis</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deacon</span><span class="given-names" tagx="given-names" title="given-names">SW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Devarajan</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ma</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peterson</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span>. <span class="mixed-article-title" title="mixed-article-title">Comprehensive assay of kinase catalytic activity reveals features of kinase inhibitor selectivity</span>. <span class="source" tagx="source" title="source">Nat Biotechnol</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">29</span>(<span class="issue" tagx="issue" title="issue">11</span>):<span class="fpage" tagx="fpage" title="fpage">1039</span>–<span class="lpage" tagx="lpage" title="lpage">45</span>. Epub 2011/11/01. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nbt.2017">10.1038/nbt.2017</a></span> ; PubMed Central PMCID: PMCPmc3230241.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22037377">22037377</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref024" /><span class="label" tagx="label" title="label">24</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>. <span class="mixed-article-title" title="mixed-article-title">Influenza virus entry</span>. <span class="source" tagx="source" title="source">Adv Exp Med Biol</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">726</span>:<span class="fpage" tagx="fpage" title="fpage">201</span>–<span class="lpage" tagx="lpage" title="lpage">21</span>. Epub 2012/02/03. <span class="pub-id"><a href="https://dx.doi.org/10.1007/978-1-4614-0980-9_9">10.1007/978-1-4614-0980-9_9</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22297515">22297515</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref025" /><span class="label" tagx="label" title="label">25</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matlin</span><span class="given-names" tagx="given-names" title="given-names">KS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reggio</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Helenius</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simons</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>. <span class="mixed-article-title" title="mixed-article-title">Infectious entry pathway of influenza virus in a canine kidney cell line</span>. <span class="source" tagx="source" title="source">J Cell Biol</span>. <span class="year" tagx="year" title="year">1981</span>;<span class="volume" tagx="volume" title="volume">91</span>(<span class="issue" tagx="issue" title="issue">3</span> Pt 1):<span class="fpage" tagx="fpage" title="fpage">601</span>–<span class="lpage" tagx="lpage" title="lpage">13</span>. Epub 1981/12/01. ; PubMed Central PMCID: PMCPmc2112819.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7328111">7328111</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref026" /><span class="label" tagx="label" title="label">26</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Vries</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tscherne</span><span class="given-names" tagx="given-names" title="given-names">DM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wienholts</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cobos-Jimenez</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scholte</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garcia-Sastre</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Dissection of the influenza A virus endocytic routes reveals macropinocytosis as an alternative entry pathway</span>. <span class="source" tagx="source" title="source">PLoS Pathog</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">7</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">e1001329</span> Epub 2011/04/13. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1001329">10.1371/journal.ppat.1001329</a></span> ; PubMed Central PMCID: PMCPmc3068995.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21483486">21483486</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref027" /><span class="label" tagx="label" title="label">27</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tomas</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Futter</span><span class="given-names" tagx="given-names" title="given-names">CE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eden</span><span class="given-names" tagx="given-names" title="given-names">ER</span></span>. <span class="mixed-article-title" title="mixed-article-title">EGF receptor trafficking: consequences for signaling and cancer</span>. <span class="source" tagx="source" title="source">Trends Cell Biol</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">24</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">26</span>–<span class="lpage" tagx="lpage" title="lpage">34</span>. Epub 2013/12/04. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.tcb.2013.11.002">10.1016/j.tcb.2013.11.002</a></span> ; PubMed Central PMCID: PMCPmc3884125.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24295852">24295852</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref028" /><span class="label" tagx="label" title="label">28</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tortorella</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karagiannis</span><span class="given-names" tagx="given-names" title="given-names">TC</span></span>. <span class="mixed-article-title" title="mixed-article-title">Transferrin receptor-mediated endocytosis: a useful target for cancer therapy</span>. <span class="source" tagx="source" title="source">J Membr Biol</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">247</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">291</span>–<span class="lpage" tagx="lpage" title="lpage">307</span>. Epub 2014/02/28. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s00232-014-9637-0">10.1007/s00232-014-9637-0</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24573305">24573305</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref029" /><span class="label" tagx="label" title="label">29</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sakai</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ohuchi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Imai</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mizuno</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawasaki</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuroda</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Dual wavelength imaging allows analysis of membrane fusion of influenza virus inside cells</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">80</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">2013</span>–<span class="lpage" tagx="lpage" title="lpage">8</span>. Epub 2006/01/28. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.80.4.2013-2018.2006">10.1128/JVI.80.4.2013-2018.2006</a></span> ; PubMed Central PMCID: PMCPmc1367152.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16439557">16439557</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref030" /><span class="label" tagx="label" title="label">30</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Banerjee</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miyake</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nobs</span><span class="given-names" tagx="given-names" title="given-names">SP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schneider</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Horvath</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kopf</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Influenza A virus uses the aggresome processing machinery for host cell entry</span>. <span class="source" tagx="source" title="source">Science</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">346</span>(<span class="issue" tagx="issue" title="issue">6208</span>):<span class="fpage" tagx="fpage" title="fpage">473</span>–<span class="lpage" tagx="lpage" title="lpage">7</span>. Epub 2014/10/25. <span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1257037">10.1126/science.1257037</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25342804">25342804</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref031" /><span class="label" tagx="label" title="label">31</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cros</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palese</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>. <span class="mixed-article-title" title="mixed-article-title">Trafficking of viral genomic RNA into and out of the nucleus: influenza, Thogoto and Borna disease viruses</span>. <span class="source" tagx="source" title="source">Virus Res</span>. <span class="year" tagx="year" title="year">2003</span>;<span class="volume" tagx="volume" title="volume">95</span>(<span class="issue" tagx="issue" title="issue">1–2</span>):<span class="fpage" tagx="fpage" title="fpage">3</span>–<span class="lpage" tagx="lpage" title="lpage">12</span>. Epub 2003/08/19. .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12921991">12921991</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref032" /><span class="label" tagx="label" title="label">32</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bernard</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Solignat</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gay</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chazal</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Higgs</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Devaux</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Endocytosis of chikungunya virus into mammalian cells: role of clathrin and early endosomal compartments</span>. <span class="source" tagx="source" title="source">PLoS One</span>. <span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">5</span>(<span class="issue" tagx="issue" title="issue">7</span>):<span class="fpage" tagx="fpage" title="fpage">e11479</span> Epub 2010/07/16. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0011479">10.1371/journal.pone.0011479</a></span> ; PubMed Central PMCID: PMCPmc2900206.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20628602">20628602</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref033" /><span class="label" tagx="label" title="label">33</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cifuentes-Munoz</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salazar-Quiroz</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tischler</span><span class="given-names" tagx="given-names" title="given-names">ND</span></span>. <span class="mixed-article-title" title="mixed-article-title">Hantavirus Gn and Gc envelope glycoproteins: key structural units for virus cell entry and virus assembly</span>. <span class="source" tagx="source" title="source">Viruses</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">6</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">1801</span>–<span class="lpage" tagx="lpage" title="lpage">22</span>. Epub 2014/04/24. <span class="pub-id"><a href="https://dx.doi.org/10.3390/v6041801">10.3390/v6041801</a></span> ; PubMed Central PMCID: PMCPmc4014721.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24755564">24755564</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref034" /><span class="label" tagx="label" title="label">34</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marsh</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Helenius</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>. <span class="mixed-article-title" title="mixed-article-title">Virus entry into animal cells</span>. <span class="source" tagx="source" title="source">Adv Virus Res</span>. <span class="year" tagx="year" title="year">1989</span>;<span class="volume" tagx="volume" title="volume">36</span>:<span class="fpage" tagx="fpage" title="fpage">107</span>–<span class="lpage" tagx="lpage" title="lpage">51</span>. Epub 1989/01/01. .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2500008">2500008</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref035" /><span class="label" tagx="label" title="label">35</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Milne</span><span class="given-names" tagx="given-names" title="given-names">RS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nicola</span><span class="given-names" tagx="given-names" title="given-names">AV</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Whitbeck</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eisenberg</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cohen</span><span class="given-names" tagx="given-names" title="given-names">GH</span></span>. <span class="mixed-article-title" title="mixed-article-title">Glycoprotein D receptor-dependent, low-pH-independent endocytic entry of herpes simplex virus type 1</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2005</span>;<span class="volume" tagx="volume" title="volume">79</span>(<span class="issue" tagx="issue" title="issue">11</span>):<span class="fpage" tagx="fpage" title="fpage">6655</span>–<span class="lpage" tagx="lpage" title="lpage">63</span>. Epub 2005/05/14. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.79.11.6655-6663.2005">10.1128/JVI.79.11.6655-6663.2005</a></span> ; PubMed Central PMCID: PMCPmc1112142.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15890903">15890903</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref036" /><span class="label" tagx="label" title="label">36</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moss</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>. <span class="mixed-article-title" title="mixed-article-title">Poxvirus entry and membrane fusion</span>. <span class="source" tagx="source" title="source">Virology</span>. <span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">344</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">48</span>–<span class="lpage" tagx="lpage" title="lpage">54</span>. Epub 2005/12/21. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virol.2005.09.037">10.1016/j.virol.2005.09.037</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16364735">16364735</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref037" /><span class="label" tagx="label" title="label">37</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takeda</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pekosz</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shuck</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pinto</span><span class="given-names" tagx="given-names" title="given-names">LH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lamb</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span>. <span class="mixed-article-title" title="mixed-article-title">Influenza a virus M2 ion channel activity is essential for efficient replication in tissue culture</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2002</span>;<span class="volume" tagx="volume" title="volume">76</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">1391</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. Epub 2002/01/05. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.76.3.1391-1399.2002">10.1128/JVI.76.3.1391-1399.2002</a></span> ; PubMed Central PMCID: PMCPmc135863.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11773413">11773413</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref038" /><span class="label" tagx="label" title="label">38</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seitz</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frensing</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoper</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kochs</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reichl</span><span class="given-names" tagx="given-names" title="given-names">U</span></span>. <span class="mixed-article-title" title="mixed-article-title">High yields of influenza A virus in Madin-Darby canine kidney cells are promoted by an insufficient interferon-induced antiviral state</span>. <span class="source" tagx="source" title="source">J Gen Virol</span>. <span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">91</span>(<span class="issue" tagx="issue" title="issue">Pt 7</span>):<span class="fpage" tagx="fpage" title="fpage">1754</span>–<span class="lpage" tagx="lpage" title="lpage">63</span>. Epub 2010/04/02. <span class="pub-id"><a href="https://dx.doi.org/10.1099/vir.0.020370-0">10.1099/vir.0.020370-0</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20357039">20357039</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref039" /><span class="label" tagx="label" title="label">39</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ludwig</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wolff</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ehrhardt</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wurzer</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reinhardt</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Planz</span><span class="given-names" tagx="given-names" title="given-names">O</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">MEK inhibition impairs influenza B virus propagation without emergence of resistant variants</span>. <span class="source" tagx="source" title="source">FEBS Lett</span>. <span class="year" tagx="year" title="year">2004</span>;<span class="volume" tagx="volume" title="volume">561</span>(<span class="issue" tagx="issue" title="issue">1–3</span>):<span class="fpage" tagx="fpage" title="fpage">37</span>–<span class="lpage" tagx="lpage" title="lpage">43</span>. Epub 2004/03/12. <span class="pub-id"><a href="https://dx.doi.org/10.1016/S0014-5793(04)00108-5">10.1016/S0014-5793(04)00108-5</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15013748">15013748</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref040" /><span class="label" tagx="label" title="label">40</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mazur</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wurzer</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ehrhardt</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pleschka</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Puthavathana</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Silberzahn</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Acetylsalicylic acid (ASA) blocks influenza virus propagation via its NF-kappaB-inhibiting activity</span>. <span class="source" tagx="source" title="source">Cell Microbiol</span>. <span class="year" tagx="year" title="year">2007</span>;<span class="volume" tagx="volume" title="volume">9</span>(<span class="issue" tagx="issue" title="issue">7</span>):<span class="fpage" tagx="fpage" title="fpage">1683</span>–<span class="lpage" tagx="lpage" title="lpage">94</span>. Epub 2007/02/28. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1462-5822.2007.00902.x">10.1111/j.1462-5822.2007.00902.x</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17324159">17324159</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref041" /><span class="label" tagx="label" title="label">41</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perwitasari</span><span class="given-names" tagx="given-names" title="given-names">O</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnson</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yan</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Howerth</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shacham</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Landesman</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Verdinexor, a novel selective inhibitor of nuclear export, reduces influenza a virus replication in vitro and in vivo</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">88</span>(<span class="issue" tagx="issue" title="issue">17</span>):<span class="fpage" tagx="fpage" title="fpage">10228</span>–<span class="lpage" tagx="lpage" title="lpage">43</span>. Epub 2014/06/27. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.01774-14">10.1128/JVI.01774-14</a></span> ; PubMed Central PMCID: PMCPmc4136318.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24965445">24965445</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref042" /><span class="label" tagx="label" title="label">42</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Resa-Infante</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paterson</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bonet</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Otte</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oliva</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fodor</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Targeting Importin-alpha7 as a Therapeutic Approach against Pandemic Influenza Viruses</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">89</span>(<span class="issue" tagx="issue" title="issue">17</span>):<span class="fpage" tagx="fpage" title="fpage">9010</span>–<span class="lpage" tagx="lpage" title="lpage">20</span>. Epub 2015/06/19. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.00583-15">10.1128/JVI.00583-15</a></span> ; PubMed Central PMCID: PMCPmc4524051.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26085167">26085167</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref043" /><span class="label" tagx="label" title="label">43</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eierhoff</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hrincius</span><span class="given-names" tagx="given-names" title="given-names">ER</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rescher</span><span class="given-names" tagx="given-names" title="given-names">U</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ludwig</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ehrhardt</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>. <span class="mixed-article-title" title="mixed-article-title">The epidermal growth factor receptor (EGFR) promotes uptake of influenza A viruses (IAV) into host cells</span>. <span class="source" tagx="source" title="source">PLoS Pathog</span>. <span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">6</span>(<span class="issue" tagx="issue" title="issue">9</span>):<span class="fpage" tagx="fpage" title="fpage">e1001099</span> Epub 2010/09/17. <span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1001099">10.1371/journal.ppat.1001099</a></span> ; PubMed Central PMCID: PMCPmc2936548.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20844577">20844577</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref044" /><span class="label" tagx="label" title="label">44</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alzamora</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thali</span><span class="given-names" tagx="given-names" title="given-names">RF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gong</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smolak</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baty</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">PKA regulates vacuolar H+-ATPase localization and activity via direct phosphorylation of the a subunit in kidney cells</span>. <span class="source" tagx="source" title="source">J Biol Chem</span>. <span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">285</span>(<span class="issue" tagx="issue" title="issue">32</span>):<span class="fpage" tagx="fpage" title="fpage">24676</span>–<span class="lpage" tagx="lpage" title="lpage">85</span>. Epub 2010/06/08. <span class="pub-id"><a href="https://dx.doi.org/10.1074/jbc.M110.106278">10.1074/jbc.M110.106278</a></span> ; PubMed Central PMCID: PMCPmc2915704.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20525692">20525692</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref045" /><span class="label" tagx="label" title="label">45</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hong-Hermesdorf</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brux</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gruber</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gruber</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schumacher</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>. <span class="mixed-article-title" title="mixed-article-title">A WNK kinase binds and phosphorylates V-ATPase subunit C</span>. <span class="source" tagx="source" title="source">FEBS Lett</span>. <span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">580</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">932</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. Epub 2006/01/24. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.febslet.2006.01.018">10.1016/j.febslet.2006.01.018</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16427632">16427632</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref046" /><span class="label" tagx="label" title="label">46</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stauffer</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nebioglu</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Heilig</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Picotti</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Helenius</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>. <span class="mixed-article-title" title="mixed-article-title">Stepwise priming by acidic pH and a high K+ concentration is required for efficient uncoating of influenza A virus cores after penetration</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">88</span>(<span class="issue" tagx="issue" title="issue">22</span>):<span class="fpage" tagx="fpage" title="fpage">13029</span>–<span class="lpage" tagx="lpage" title="lpage">46</span>. Epub 2014/08/29. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.01430-14">10.1128/JVI.01430-14</a></span> ; PubMed Central PMCID: PMCPmc4249060.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25165113">25165113</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref047" /><span class="label" tagx="label" title="label">47</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sieben</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ludwig</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hofer</span><span class="given-names" tagx="given-names" title="given-names">CT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chiantia</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herrmann</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">pH-Controlled two-step uncoating of influenza virus</span>. <span class="source" tagx="source" title="source">Biophys J</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">106</span>(<span class="issue" tagx="issue" title="issue">7</span>):<span class="fpage" tagx="fpage" title="fpage">1447</span>–<span class="lpage" tagx="lpage" title="lpage">56</span>. Epub 2014/04/08. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.bpj.2014.02.018">10.1016/j.bpj.2014.02.018</a></span> ; PubMed Central PMCID: PMCPmc3976520.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24703306">24703306</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref048" /><span class="label" tagx="label" title="label">48</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perwitasari</span><span class="given-names" tagx="given-names" title="given-names">O</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yan</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Donnell</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnson</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tripp</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span>. <span class="mixed-article-title" title="mixed-article-title">Repurposing Kinase Inhibitors as Antiviral Agents to Control Influenza A Virus Replication</span>. <span class="source" tagx="source" title="source">Assay Drug Dev Technol</span>. <span class="year" tagx="year" title="year">2015</span> Epub 2015/07/21. <span class="pub-id"><a href="https://dx.doi.org/10.1089/adt.2015.0003.drrr">10.1089/adt.2015.0003.drrr</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26192013">26192013</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref049" /><span class="label" tagx="label" title="label">49</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benedict</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bansal</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Senina</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hooper</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lundberg</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de la Fuente</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Repurposing FDA-approved drugs as therapeutics to treat Rift Valley fever virus infection</span>. <span class="source" tagx="source" title="source">Front Microbiol</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">6</span>:<span class="fpage" tagx="fpage" title="fpage">676</span> Epub 2015/07/29. <span class="pub-id"><a href="https://dx.doi.org/10.3389/fmicb.2015.00676">10.3389/fmicb.2015.00676</a></span> ; PubMed Central PMCID: PMCPmc4495339.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26217313">26217313</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref050" /><span class="label" tagx="label" title="label">50</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Descamps</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Helle</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Louandre</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brochot</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Izquierdo</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">The kinase-inhibitor sorafenib inhibits multiple steps of the Hepatitis C Virus infectious cycle in vitro</span>. <span class="source" tagx="source" title="source">Antiviral Res</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">118</span>:<span class="fpage" tagx="fpage" title="fpage">93</span>–<span class="lpage" tagx="lpage" title="lpage">102</span>. Epub 2015/04/01. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.antiviral.2015.03.012">10.1016/j.antiviral.2015.03.012</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25823619">25823619</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref051" /><span class="label" tagx="label" title="label">51</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duan</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">The multi-targeted kinase inhibitor sorafenib inhibits enterovirus 71 replication by regulating IRES-dependent translation of viral proteins</span>. <span class="source" tagx="source" title="source">Antiviral Res</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">106</span>:<span class="fpage" tagx="fpage" title="fpage">80</span>–<span class="lpage" tagx="lpage" title="lpage">5</span>. Epub 2014/04/01. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.antiviral.2014.03.009">10.1016/j.antiviral.2014.03.009</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24680956">24680956</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref052" /><span class="label" tagx="label" title="label">52</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Michaelis</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paulus</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Loschmann</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dauth</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stange</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doerr</span><span class="given-names" tagx="given-names" title="given-names">HW</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">The multi-targeted kinase inhibitor sorafenib inhibits human cytomegalovirus replication</span>. <span class="source" tagx="source" title="source">Cell Mol Life Sci</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">68</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">1079</span>–<span class="lpage" tagx="lpage" title="lpage">90</span>. Epub 2010/08/31. <span class="pub-id"><a href="https://dx.doi.org/10.1007/s00018-010-0510-8">10.1007/s00018-010-0510-8</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20803231">20803231</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref053" /><span class="label" tagx="label" title="label">53</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Randhawa</span><span class="given-names" tagx="given-names" title="given-names">PS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Farasati</span><span class="given-names" tagx="given-names" title="given-names">NA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mapara</span><span class="given-names" tagx="given-names" title="given-names">MY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shapiro</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>. <span class="mixed-article-title" title="mixed-article-title">Viral drug sensitivity testing using quantitative PCR: effect of tyrosine kinase inhibitors on polyomavirus BK replication</span>. <span class="source" tagx="source" title="source">Am J Clin Pathol</span>. <span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">134</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">916</span>–<span class="lpage" tagx="lpage" title="lpage">20</span>. Epub 2010/11/23. <span class="pub-id"><a href="https://dx.doi.org/10.1309/AJCP7JYHJN1PGQVC">10.1309/AJCP7JYHJN1PGQVC</a></span> ; PubMed Central PMCID: PMCPmc3075568.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21088155">21088155</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref054" /><span class="label" tagx="label" title="label">54</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roberts</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tavallai</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nourbakhsh</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fidanza</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cruz-Luna</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">GRP78/Dna K Is a Target for Nexavar/Stivarga/Votrient in the Treatment of Human Malignancies, Viral Infections and Bacterial Diseases</span>. <span class="source" tagx="source" title="source">J Cell Physiol</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">230</span>(<span class="issue" tagx="issue" title="issue">10</span>):<span class="fpage" tagx="fpage" title="fpage">2552</span>–<span class="lpage" tagx="lpage" title="lpage">78</span>. Epub 2015/04/11. <span class="pub-id"><a href="https://dx.doi.org/10.1002/jcp.25014">10.1002/jcp.25014</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25858032">25858032</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref055" /><span class="label" tagx="label" title="label">55</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brahms</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mudhasani</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pinkham</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kota</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nasar</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zamani</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Sorafenib Impedes Rift Valley Fever Virus Egress by Inhibiting Valosin-Containing Protein Function in the Cellular Secretory Pathway</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">91</span>(<span class="issue" tagx="issue" title="issue">21</span>). Epub 2017/08/11. <span class="pub-id"><a href="https://dx.doi.org/10.1128/jvi.00968-17">10.1128/jvi.00968-17</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28794043">28794043</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref056" /><span class="label" tagx="label" title="label">56</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuiken</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van den Brand</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Riel</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pantin-Jackwood</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Swayne</span><span class="given-names" tagx="given-names" title="given-names">DE</span></span>. <span class="mixed-article-title" title="mixed-article-title">Comparative pathology of select agent influenza a virus infections</span>. <span class="source" tagx="source" title="source">Vet Pathol</span>. <span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">47</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">893</span>–<span class="lpage" tagx="lpage" title="lpage">914</span>. Epub 2010/08/05. <span class="pub-id"><a href="https://dx.doi.org/10.1177/0300985810378651">10.1177/0300985810378651</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20682805">20682805</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref057" /><span class="label" tagx="label" title="label">57</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Triana-Baltzer</span><span class="given-names" tagx="given-names" title="given-names">GB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanders</span><span class="given-names" tagx="given-names" title="given-names">RL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hedlund</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jensen</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aschenbrenner</span><span class="given-names" tagx="given-names" title="given-names">LM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larson</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Phenotypic and genotypic characterization of influenza virus mutants selected with the sialidase fusion protein DAS181</span>. <span class="source" tagx="source" title="source">J Antimicrob Chemother</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">66</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">15</span>–<span class="lpage" tagx="lpage" title="lpage">28</span>. Epub 2010/11/26. <span class="pub-id"><a href="https://dx.doi.org/10.1093/jac/dkq387">10.1093/jac/dkq387</a></span> ; PubMed Central PMCID: PMCPmc3001847.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21097900">21097900</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref058" /><span class="label" tagx="label" title="label">58</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hurt</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span>. <span class="mixed-article-title" title="mixed-article-title">The epidemiology and spread of drug resistant human influenza viruses</span>. <span class="source" tagx="source" title="source">Curr Opin Virol</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">8</span>:<span class="fpage" tagx="fpage" title="fpage">22</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. Epub 2014/05/29. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.coviro.2014.04.009">10.1016/j.coviro.2014.04.009</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24866471">24866471</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref059" /><span class="label" tagx="label" title="label">59</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ferraris</span><span class="given-names" tagx="given-names" title="given-names">O</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kessler</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lina</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>. <span class="mixed-article-title" title="mixed-article-title">Sensitivity of influenza viruses to zanamivir and oseltamivir: a study performed on viruses circulating in France prior to the introduction of neuraminidase inhibitors in clinical practice</span>. <span class="source" tagx="source" title="source">Antiviral Res</span>. <span class="year" tagx="year" title="year">2005</span>;<span class="volume" tagx="volume" title="volume">68</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">43</span>–<span class="lpage" tagx="lpage" title="lpage">8</span>. Epub 2005/08/30. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.antiviral.2005.07.004">10.1016/j.antiviral.2005.07.004</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16125799">16125799</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref060" /><span class="label" tagx="label" title="label">60</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alfaro</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suarez</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gonzalez</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Solano</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Erro</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dubrot</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Influence of bevacizumab, sunitinib and sorafenib as single agents or in combination on the inhibitory effects of VEGF on human dendritic cell differentiation from monocytes</span>. <span class="source" tagx="source" title="source">Br J Cancer</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">100</span>(<span class="issue" tagx="issue" title="issue">7</span>):<span class="fpage" tagx="fpage" title="fpage">1111</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. Epub 2009/03/12. <span class="pub-id"><a href="https://dx.doi.org/10.1038/sj.bjc.6604965">10.1038/sj.bjc.6604965</a></span> ; PubMed Central PMCID: PMCPmc2670006.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19277038">19277038</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref061" /><span class="label" tagx="label" title="label">61</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hipp</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hilf</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walter</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Werth</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brauer</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Radsak</span><span class="given-names" tagx="given-names" title="given-names">MP</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Sorafenib, but not sunitinib, affects function of dendritic cells and induction of primary immune responses</span>. <span class="source" tagx="source" title="source">Blood</span>. <span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">111</span>(<span class="issue" tagx="issue" title="issue">12</span>):<span class="fpage" tagx="fpage" title="fpage">5610</span>–<span class="lpage" tagx="lpage" title="lpage">20</span>. Epub 2008/03/04. <span class="pub-id"><a href="https://dx.doi.org/10.1182/blood-2007-02-075945">10.1182/blood-2007-02-075945</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18310500">18310500</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref062" /><span class="label" tagx="label" title="label">62</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krusch</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salih</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schlicke</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baessler</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kampa</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mayer</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">The kinase inhibitors sunitinib and sorafenib differentially affect NK cell antitumor reactivity in vitro</span>. <span class="source" tagx="source" title="source">J Immunol</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">183</span>(<span class="issue" tagx="issue" title="issue">12</span>):<span class="fpage" tagx="fpage" title="fpage">8286</span>–<span class="lpage" tagx="lpage" title="lpage">94</span>. Epub 2009/12/17. <span class="pub-id"><a href="https://dx.doi.org/10.4049/jimmunol.0902404">10.4049/jimmunol.0902404</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20007592">20007592</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref063" /><span class="label" tagx="label" title="label">63</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gu</span><span class="given-names" tagx="given-names" title="given-names">YH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qu</span><span class="given-names" tagx="given-names" title="given-names">BQ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>. <span class="mixed-article-title" title="mixed-article-title">Sorafenib inhibits activation of human peripheral blood T cells by targeting LCK phosphorylation</span>. <span class="source" tagx="source" title="source">Leukemia</span>. <span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">22</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">1226</span>–<span class="lpage" tagx="lpage" title="lpage">33</span>. Epub 2008/03/14. <span class="pub-id"><a href="https://dx.doi.org/10.1038/leu.2008.58">10.1038/leu.2008.58</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18337760">18337760</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref064" /><span class="label" tagx="label" title="label">64</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lohmeyer</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nerreter</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dotterweich</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Einsele</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Seggewiss-Bernhardt</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>. <span class="mixed-article-title" title="mixed-article-title">Sorafenib paradoxically activates the RAS/RAF/ERK pathway in polyclonal human NK cells during expansion and thereby enhances effector functions in a dose- and time-dependent manner</span>. <span class="source" tagx="source" title="source">Clin Exp Immunol</span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">193</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">64</span>–<span class="lpage" tagx="lpage" title="lpage">72</span>. Epub 2018/03/25. <span class="pub-id"><a href="https://dx.doi.org/10.1111/cei.13128">10.1111/cei.13128</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29573266">29573266</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref065" /><span class="label" tagx="label" title="label">65</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sunay</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foote</span><span class="given-names" tagx="given-names" title="given-names">JB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leatherman</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edwards</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Armstrong</span><span class="given-names" tagx="given-names" title="given-names">TD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nirschl</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Sorafenib combined with HER-2 targeted vaccination can promote effective T cell immunity in vivo</span>. <span class="source" tagx="source" title="source">Int Immunopharmacol</span>. <span class="year" tagx="year" title="year">2017</span>;<span class="volume" tagx="volume" title="volume">46</span>:<span class="fpage" tagx="fpage" title="fpage">112</span>–<span class="lpage" tagx="lpage" title="lpage">23</span>. Epub 2017/03/11. <span class="pub-id"><a href="https://dx.doi.org/10.1016/j.intimp.2017.02.028">10.1016/j.intimp.2017.02.028</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28282575">28282575</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref066" /><span class="label" tagx="label" title="label">66</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mulder</span><span class="given-names" tagx="given-names" title="given-names">SF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jacobs</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olde Nordkamp</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Galama</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Desar</span><span class="given-names" tagx="given-names" title="given-names">IM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Torensma</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Cancer patients treated with sunitinib or sorafenib have sufficient antibody and cellular immune responses to warrant influenza vaccination</span>. <span class="source" tagx="source" title="source">Clin Cancer Res</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">17</span>(<span class="issue" tagx="issue" title="issue">13</span>):<span class="fpage" tagx="fpage" title="fpage">4541</span>–<span class="lpage" tagx="lpage" title="lpage">9</span>. Epub 2011/06/30. <span class="pub-id"><a href="https://dx.doi.org/10.1158/1078-0432.CCR-11-0253">10.1158/1078-0432.CCR-11-0253</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21712444">21712444</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref067" /><span class="label" tagx="label" title="label">67</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Furuta</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Takahashi</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fukuda</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuno</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kamiyama</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kozaki</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">In vitro and in vivo activities of anti-influenza virus compound T-705</span>. <span class="source" tagx="source" title="source">Antimicrob Agents Chemother</span>. <span class="year" tagx="year" title="year">2002</span>;<span class="volume" tagx="volume" title="volume">46</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">977</span>–<span class="lpage" tagx="lpage" title="lpage">81</span>. Epub 2002/03/19. <span class="pub-id"><a href="https://dx.doi.org/10.1128/AAC.46.4.977-981.2002">10.1128/AAC.46.4.977-981.2002</a></span> ; PubMed Central PMCID: PMCPmc127093.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11897578">11897578</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref068" /><span class="label" tagx="label" title="label">68</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mross</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frost</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steinbild</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hedbom</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buchert</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fasol</span><span class="given-names" tagx="given-names" title="given-names">U</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">A phase I dose-escalation study of regorafenib (BAY 73–4506), an inhibitor of oncogenic, angiogenic, and stromal kinases, in patients with advanced solid tumors</span>. <span class="source" tagx="source" title="source">Clin Cancer Res</span>. <span class="year" tagx="year" title="year">2012</span>;<span class="volume" tagx="volume" title="volume">18</span>(<span class="issue" tagx="issue" title="issue">9</span>):<span class="fpage" tagx="fpage" title="fpage">2658</span>–<span class="lpage" tagx="lpage" title="lpage">67</span>. Epub 2012/03/17. <span class="pub-id"><a href="https://dx.doi.org/10.1158/1078-0432.CCR-11-1900">10.1158/1078-0432.CCR-11-1900</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22421192">22421192</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref069" /><span class="label" tagx="label" title="label">69</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clark</span><span class="given-names" tagx="given-names" title="given-names">JW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eder</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ryan</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lathia</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lenz</span><span class="given-names" tagx="given-names" title="given-names">HJ</span></span>. <span class="mixed-article-title" title="mixed-article-title">Safety and pharmacokinetics of the dual action Raf kinase and vascular endothelial growth factor receptor inhibitor, BAY 43–9006, in patients with advanced, refractory solid tumors</span>. <span class="source" tagx="source" title="source">Clin Cancer Res</span>. <span class="year" tagx="year" title="year">2005</span>;<span class="volume" tagx="volume" title="volume">11</span>(<span class="issue" tagx="issue" title="issue">15</span>):<span class="fpage" tagx="fpage" title="fpage">5472</span>–<span class="lpage" tagx="lpage" title="lpage">80</span>. Epub 2005/08/03. <span class="pub-id"><a href="https://dx.doi.org/10.1158/1078-0432.CCR-04-2658">10.1158/1078-0432.CCR-04-2658</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16061863">16061863</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref070" /><span class="label" tagx="label" title="label">70</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zauli</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bassini</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Catani</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibellini</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Celeghini</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Borgatti</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">PMA-induced megakaryocytic differentiation of HEL cells is accompanied by striking modifications of protein kinase C catalytic activity and isoform composition at the nuclear level</span>. <span class="source" tagx="source" title="source">Br J Haematol</span>. <span class="year" tagx="year" title="year">1996</span>;<span class="volume" tagx="volume" title="volume">92</span>(<span class="issue" tagx="issue" title="issue">3</span>):<span class="fpage" tagx="fpage" title="fpage">530</span>–<span class="lpage" tagx="lpage" title="lpage">6</span>. Epub 1996/03/01. .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8616013">8616013</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref071" /><span class="label" tagx="label" title="label">71</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanjana</span><span class="given-names" tagx="given-names" title="given-names">NE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shalem</span><span class="given-names" tagx="given-names" title="given-names">O</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>. <span class="mixed-article-title" title="mixed-article-title">Improved vectors and genome-wide libraries for CRISPR screening</span>. <span class="source" tagx="source" title="source">Nat Methods</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">11</span>(<span class="issue" tagx="issue" title="issue">8</span>):<span class="fpage" tagx="fpage" title="fpage">783</span>–<span class="lpage" tagx="lpage" title="lpage">4</span>. Epub 2014/07/31. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nmeth.3047">10.1038/nmeth.3047</a></span> ; PubMed Central PMCID: PMCPmc4486245.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25075903">25075903</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref072" /><span class="label" tagx="label" title="label">72</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roberts</span><span class="given-names" tagx="given-names" title="given-names">KL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shelton</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scull</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pickles</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barclay</span><span class="given-names" tagx="given-names" title="given-names">WS</span></span>. <span class="mixed-article-title" title="mixed-article-title">Lack of transmission of a human influenza virus with avian receptor specificity between ferrets is not due to decreased virus shedding but rather a lower infectivity in vivo</span>. <span class="source" tagx="source" title="source">J Gen Virol</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">92</span>(<span class="issue" tagx="issue" title="issue">Pt 8</span>):<span class="fpage" tagx="fpage" title="fpage">1822</span>–<span class="lpage" tagx="lpage" title="lpage">31</span>. Epub 2011/04/22. <span class="pub-id"><a href="https://dx.doi.org/10.1099/vir.0.031203-0">10.1099/vir.0.031203-0</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21508186">21508186</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref073" /><span class="label" tagx="label" title="label">73</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wressnigg</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shurygina</span><span class="given-names" tagx="given-names" title="given-names">AP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wolff</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Redlberger-Fritz</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Popow-Kraupp</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muster</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Influenza B mutant viruses with truncated NS1 proteins grow efficiently in Vero cells and are immunogenic in mice</span>. <span class="source" tagx="source" title="source">J Gen Virol</span>. <span class="year" tagx="year" title="year">2009</span>;<span class="volume" tagx="volume" title="volume">90</span>(<span class="issue" tagx="issue" title="issue">Pt 2</span>):<span class="fpage" tagx="fpage" title="fpage">366</span>–<span class="lpage" tagx="lpage" title="lpage">74</span>. Epub 2009/01/15. <span class="pub-id"><a href="https://dx.doi.org/10.1099/vir.0.006122-0">10.1099/vir.0.006122-0</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19141445">19141445</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref074" /><span class="label" tagx="label" title="label">74</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hoffmann</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neumann</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawaoka</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hobom</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Webster</span><span class="given-names" tagx="given-names" title="given-names">RG</span></span>. <span class="mixed-article-title" title="mixed-article-title">A DNA transfection system for generation of influenza A virus from eight plasmids</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span>. <span class="year" tagx="year" title="year">2000</span>;<span class="volume" tagx="volume" title="volume">97</span>(<span class="issue" tagx="issue" title="issue">11</span>):<span class="fpage" tagx="fpage" title="fpage">6108</span>–<span class="lpage" tagx="lpage" title="lpage">13</span>. Epub 2000/05/10. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.100133697">10.1073/pnas.100133697</a></span> ; PubMed Central PMCID: PMCPmc18566.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10801978">10801978</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref075" /><span class="label" tagx="label" title="label">75</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Watanabe</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Watanabe</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kim</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hatta</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kawaoka</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>. <span class="mixed-article-title" title="mixed-article-title">Novel approach to the development of effective H5N1 influenza A virus vaccines: use of M2 cytoplasmic tail mutants</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">82</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">2486</span>–<span class="lpage" tagx="lpage" title="lpage">92</span>. Epub 2007/12/28. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.01899-07">10.1128/JVI.01899-07</a></span> ; PubMed Central PMCID: PMCPmc2258945.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18160446">18160446</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref076" /><span class="label" tagx="label" title="label">76</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elderfield</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Watson</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Godlee</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adamson</span><span class="given-names" tagx="given-names" title="given-names">WE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thompson</span><span class="given-names" tagx="given-names" title="given-names">CI</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dunning</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Accumulation of human-adapting mutations during circulation of A(H1N1)pdm09 influenza virus in humans in the United Kingdom</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">88</span>(<span class="issue" tagx="issue" title="issue">22</span>):<span class="fpage" tagx="fpage" title="fpage">13269</span>–<span class="lpage" tagx="lpage" title="lpage">83</span>. Epub 2014/09/12. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.01636-14">10.1128/JVI.01636-14</a></span> ; PubMed Central PMCID: PMCPmc4249111.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25210166">25210166</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref077" /><span class="label" tagx="label" title="label">77</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ge</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McManus</span><span class="given-names" tagx="given-names" title="given-names">MT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguyen</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">CH</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharp</span><span class="given-names" tagx="given-names" title="given-names">PA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eisen</span><span class="given-names" tagx="given-names" title="given-names">HN</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">RNA interference of influenza virus production by directly targeting mRNA for degradation and indirectly inhibiting all viral RNA transcription</span>. <span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span>. <span class="year" tagx="year" title="year">2003</span>;<span class="volume" tagx="volume" title="volume">100</span>(<span class="issue" tagx="issue" title="issue">5</span>):<span class="fpage" tagx="fpage" title="fpage">2718</span>–<span class="lpage" tagx="lpage" title="lpage">23</span>. Epub 2003/02/21. <span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0437841100">10.1073/pnas.0437841100</a></span> ; PubMed Central PMCID: PMCPmc151407.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12594334">12594334</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref078" /><span class="label" tagx="label" title="label">78</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boutros</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bras</span><span class="given-names" tagx="given-names" title="given-names">LP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huber</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>. <span class="mixed-article-title" title="mixed-article-title">Analysis of cell-based RNAi screens</span>. <span class="source" tagx="source" title="source">Genome Biol</span>. <span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">7</span>(<span class="issue" tagx="issue" title="issue">7</span>):<span class="fpage" tagx="fpage" title="fpage">R66</span> Epub 2006/07/28. <span class="pub-id"><a href="https://dx.doi.org/10.1186/gb-2006-7-7-r66">10.1186/gb-2006-7-7-r66</a></span> ; PubMed Central PMCID: PMCPmc1779553.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16869968">16869968</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref079" /><span class="label" tagx="label" title="label">79</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharan</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maron-Katz</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shamir</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>. <span class="mixed-article-title" title="mixed-article-title">CLICK and EXPANDER: a system for clustering and visualizing gene expression data</span>. <span class="source" tagx="source" title="source">Bioinformatics</span>. <span class="year" tagx="year" title="year">2003</span>;<span class="volume" tagx="volume" title="volume">19</span>(<span class="issue" tagx="issue" title="issue">14</span>):<span class="fpage" tagx="fpage" title="fpage">1787</span>–<span class="lpage" tagx="lpage" title="lpage">99</span>. Epub 2003/09/27. .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14512350">14512350</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref080" /><span class="label" tagx="label" title="label">80</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shamir</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maron-Katz</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tanay</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Linhart</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steinfeld</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharan</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">EXPANDER—an integrative program suite for microarray data analysis</span>. <span class="source" tagx="source" title="source">BMC Bioinformatics</span>. <span class="year" tagx="year" title="year">2005</span>;<span class="volume" tagx="volume" title="volume">6</span>:<span class="fpage" tagx="fpage" title="fpage">232</span> Epub 2005/09/24. <span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2105-6-232">10.1186/1471-2105-6-232</a></span> ; PubMed Central PMCID: PMCPmc1261157.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16176576">16176576</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref081" /><span class="label" tagx="label" title="label">81</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ritz</span><span class="given-names" tagx="given-names" title="given-names">CS</span></span>, J. C. <span class="mixed-article-title" title="mixed-article-title">Bioassay Analysis using R</span>. <span class="source" tagx="source" title="source">J Statist Software</span>. <span class="year" tagx="year" title="year">2005</span>;<span class="volume" tagx="volume" title="volume">12</span>(<span class="issue" tagx="issue" title="issue">5</span>).</span></li> <li tag="ref"><a name="ppat.1007601.ref082" /><span class="label" tagx="label" title="label">82</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Opitz</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rejaibi</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dauber</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eckhard</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vinzing</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmeck</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">IFNbeta induction by influenza A virus is mediated by RIG-I which is regulated by the viral NS1 protein</span>. <span class="source" tagx="source" title="source">Cell Microbiol</span>. <span class="year" tagx="year" title="year">2007</span>;<span class="volume" tagx="volume" title="volume">9</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">930</span>–<span class="lpage" tagx="lpage" title="lpage">8</span>. Epub 2006/12/05. <span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1462-5822.2006.00841.x">10.1111/j.1462-5822.2006.00841.x</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17140406">17140406</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref083" /><span class="label" tagx="label" title="label">83</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Livak</span><span class="given-names" tagx="given-names" title="given-names">KJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schmittgen</span><span class="given-names" tagx="given-names" title="given-names">TD</span></span>. <span class="mixed-article-title" title="mixed-article-title">Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method</span>. <span class="source" tagx="source" title="source">Methods</span>. <span class="year" tagx="year" title="year">2001</span>;<span class="volume" tagx="volume" title="volume">25</span>(<span class="issue" tagx="issue" title="issue">4</span>):<span class="fpage" tagx="fpage" title="fpage">402</span>–<span class="lpage" tagx="lpage" title="lpage">8</span>. Epub 2002/02/16. <span class="pub-id"><a href="https://dx.doi.org/10.1006/meth.2001.1262">10.1006/meth.2001.1262</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11846609">11846609</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref084" /><span class="label" tagx="label" title="label">84</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Papp</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sieben</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sisson</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kostka</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bottcher</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ludwig</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Inhibition of influenza virus activity by multivalent glycoarchitectures with matched sizes</span>. <span class="source" tagx="source" title="source">Chembiochem</span>. <span class="year" tagx="year" title="year">2011</span>;<span class="volume" tagx="volume" title="volume">12</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">887</span>–<span class="lpage" tagx="lpage" title="lpage">95</span>. Epub 2011/03/09. <span class="pub-id"><a href="https://dx.doi.org/10.1002/cbic.201000776">10.1002/cbic.201000776</a></span> .<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21384484">21384484</a></span></span></li> <li tag="ref"><a name="ppat.1007601.ref085" /><span class="label" tagx="label" title="label">85</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kanneganti</span><span class="given-names" tagx="given-names" title="given-names">TD</span></span>. <span class="mixed-article-title" title="mixed-article-title">Central roles of NLRs and inflammasomes in viral infection</span>. <span class="source" tagx="source" title="source">Nat Rev Immunol</span>. <span class="year" tagx="year" title="year">2010</span>;<span class="volume" tagx="volume" title="volume">10</span>(<span class="issue" tagx="issue" title="issue">10</span>):<span class="fpage" tagx="fpage" title="fpage">688</span>–<span class="lpage" tagx="lpage" title="lpage">98</span>. Epub 2010/09/18. <span class="pub-id"><a href="https://dx.doi.org/10.1038/nri2851">10.1038/nri2851</a></span> ; PubMed Central PMCID: PMCPmc3909537.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20847744">20847744</a></span></span></li> </ul> </div> </div>  </body></html>