<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">PLoS One</span><span class="iso-abbrev" title="iso-abbrev">PLoS ONE</span><span class="publisher-id" title="publisher-id">plos</span><span class="pmc" title="pmc">plosone</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">PLoS ONE</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1932-6203</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, USA</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/3815201">3815201</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24223766">24223766</a></span><span class="publisher-id" title="publisher-id">PONE-D-13-14216</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pone.0078122">10.1371/journal.pone.0078122</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Mapping the Shapes of Phylogenetic Trees from Human and Zoonotic RNA Viruses</div> <div class="alt-title" title="alt-title">
Shapes of Human and Zoonotic RNA Virus Phylogenies</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poon</span><span class="given-names" tagx="given-names" title="given-names">Art F. Y.</span></span><a href="#aff1"><sup>1</sup></a><a href="#aff2"><sup>2</sup></a><a href="#cor1"><sup>*</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walker</span><span class="given-names" tagx="given-names" title="given-names">Lorne W.</span></span><a href="#aff3"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murray</span><span class="given-names" tagx="given-names" title="given-names">Heather</span></span><a href="#aff1"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCloskey</span><span class="given-names" tagx="given-names" title="given-names">Rosemary M.</span></span><a href="#aff1"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harrigan</span><span class="given-names" tagx="given-names" title="given-names">P. Richard</span></span><a href="#aff1"><sup>1</sup></a><a href="#aff2"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">Richard H.</span></span><a href="#aff1"><sup>1</sup></a></span></div><span class="citation_author_institution" id="aff1">[1], <span class="addr-line" title="addr-line">British Columbia Centre for Excellence in HIV/AIDS, Vancouver, British Columbia, Canada</span></span><span class="citation_author_institution" id="aff2">[2], <span class="addr-line" title="addr-line">Department of Medicine, University of British Columbia, Vancouver, British Columbia, Canada</span></span><span class="citation_author_institution" id="aff3">[3], <span class="addr-line" title="addr-line">School of Medicine, University of California San Diego, La Jolla, California, United States of America</span></span><div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crandall</span><span class="given-names" tagx="given-names" title="given-names">Keith A.</span></span><span class="role" tagx="role" title="role">Editor</span><a href="#edit1" /></span></div><span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">George Washington University, United States of America</span></span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">apoon@cfenet.ubc.ca</span></div> <div class="fn-type-COI-statement" title="COI-statement"> <p><b>Competing Interests: </b>AFYP is a member of the PLOS ONE Editorial Board, and that this does not alter the authors' adherence to all the PLOS ONE policies on sharing data and materials.</p> </div> <div class="fn-type-con" title="con"> <p>Conceived and designed the experiments: AFYP. Performed the experiments: AFYP RL HM RM. Analyzed the data: AFYP. Contributed reagents/materials/analysis tools: LW. Wrote the paper: AFYP PRH.</p> </div> </div><span class="pub-date-collection" title="pub-date-collection">collection: <span>2013</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2013-11-11</span></span><span class="volume" tagx="volume" title="volume">8</span><span class="issue" tagx="issue" title="issue">11</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e78122</span><span class="history" title="history"><span class="received" title="received">received: 2013-4-5</span><span class="accepted" title="accepted">accepted: 2013-9-9</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Poon et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2013</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are properly credited.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>A phylogeny is a tree-based model of common ancestry that is an indispensable tool for studying biological variation. Phylogenies play a special role in the study of rapidly evolving populations such as viruses, where the proliferation of lineages is constantly being shaped by the mode of virus transmission, by adaptation to immune systems, and by patterns of human migration and contact. These processes may leave an imprint on the shapes of virus phylogenies that can be extracted for comparative study; however, tree shapes are intrinsically difficult to quantify. Here we present a comprehensive study of phylogenies reconstructed from 38 different RNA viruses from 12 taxonomic families that are associated with human pathologies. To accomplish this, we have developed a new procedure for studying phylogenetic tree shapes based on the â€˜kernel trickâ€™, a technique that maps complex objects into a statistically convenient space. We show that our kernel method outperforms nine different tree balance statistics at correctly classifying phylogenies that were simulated under different evolutionary scenarios. Using the kernel method, we observe patterns in the distribution of RNA virus phylogenies in this space that reflect modes of transmission and pathogenesis. For example, viruses that can establish persistent chronic infections (such as HIV and hepatitis C virus) form a distinct cluster. Although the visibly â€˜star-likeâ€™ shape characteristic of trees from these viruses has been well-documented, we show that established methods for quantifying tree shape fail to distinguish these trees from those of other viruses. The kernel approach presented here potentially represents an important new tool for characterizing the evolution and epidemiology of RNA viruses.</p> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="funding-statement" title="funding-statement">
This study was supported by a Canadian Institutes of Health Research (CIHR) operating grant (HOP-111406) awarded to AFYP. AFYP is supported by a Scholar Award from the Michael Smith Foundation for Health Research (MSFHR)/St. âˆ¼Paulâ€™s Hospital Foundation-Providence Health Care Research Institute (SPHF-PHCRI) Career Investigator program, and by a New Investigator Award from the CIHR HIV Vaccine Initiative (CHVI) for Vaccine Discovery and Social Research. RMM was supported by a Vice President Research Undergraduate Student Research Award (VPR-USRA) from Simon Fraser University. HM was supported by a Centre for Blood Research Summer Studentship from the University of British Columbia. PRH is supported by a CIHR/GSK Research Chair in Clinical Virology. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</div> </div> <div class="counts" title="counts"><span class="page-count" title="page-count">page-count: </span></div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Trees have long been used as a metaphor for making sense of a diverse living world. The modern phylogenetic tree is a model of the evolutionary relatedness of populations or individuals, which is increasingly based on the similarity of their genetic makeup. Phylogenies have become an indispensable tool for the study of biological variation by providing a framework for modelling evolutionary processes along lineages of descent from common ancestors. In particular, phylogenies are used extensively in the study of viruses. Viruses, especially those with RNA genomes, exhibit some of the fastest rates of molecular evolution in the natural world, driven by high rates of mutation <a href="#pone.0078122-Drake1">[1]</a>, large population sizes and short generation times. Many viral lineages can proliferate on time scales measured in days, as opposed to the millions of years it would require for many branches on the macroscopic â€˜Tree of Lifeâ€™ to grow to the same extent. The substantial variation in the shapes of RNA virus phylogenies (<a href="#pone-0078122-g001">Figure 1</a>) may be determined by epidemiological processes that operate at the scale of the host population. These processes may in turn be shaped by the ongoing evolution of the virus. The study of this multi-level interaction is known as â€˜phylodynamicsâ€™ <a href="#pone.0078122-Grenfell1">[2]</a>.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0078122.g001"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <div class="title" tagx="title" title="title">
Diversity in phylogenetic tree shapes for animal RNA viruses.</div> <p>These phylogenies were generated from samples of genetic sequences from HIV-1 subtype B (HIV1-B), dengue virus serotype 1II (DEN-1II), influenza A virus serotype H3N2 (IAV-H3), and coxsackievirus A24 variant (CA24v).</p>   </div> <p>Identifying the reproducible shape characteristics among virus phylogenies and the epidemiological and evolutionary determinants of these shapes are key research questions of phylodynamics <a href="#pone.0078122-Holmes1">[3]</a>. For instance, the stark contrast between phylogenies generated respectively from influenza A virus (IAV) and human immunodeficiency virus (HIV) type 1 sequences is a prototypical example of how virus phylogenies can be shaped by phylodynamic processes <a href="#pone.0078122-Korber1">[4]</a>, <a href="#pone.0078122-Sharp1">[5]</a>. A phylogeny reconstructed from sequences of IAV hemagglutinin, a receptor-binding glycoprotein which is a primary target of the host immune system, has a distinctively pectinate (â€˜comb-likeâ€™) shape (<a href="#pone-0078122-g001">Figure 1</a>) that is hypothesized to be driven by the short infectious period and antigenic drift of IAV, in which very few lineages persist between successive epidemics <a href="#pone.0078122-Nelson1">[6]</a>. In contrast, the life-long nature of HIV infection and ongoing transmission among hosts has enabled rapidly diverging lineages to proliferate and persist over time, leading to â€˜star-likeâ€™ phylogenies with short branches near the root and long branches at the tips (<a href="#pone-0078122-g001">Figure 1</a>).</p> <p>Until recently, our understanding of phylodynamics through the comparative study of virus phylogenies has been generally limited to conspicuous differences in tree shape, such as that between HIV and IAV phylogenies, because tree shapes are inherently difficult to quantify. A phylogenetic tree is an acyclic graph in which the nodes represent either observed entities (leaf nodes) or their common ancestors (internal nodes) which must be reconstructed from the variation observed at the leaf nodes. These nodes are linked by branches whose lengths represent either the passage of time or the amount of evolutionary change that has occurred between nodes. As a result, the shape of a phylogenetic tree is a complex object that incorporates both the lengths and distribution of branches among nodes. There is an abundance of descriptive statistics of tree shape that focus on the concept of â€˜tree balanceâ€™, which refers to the skewed distribution of internal nodes in a tree due to variation in rates of branching <a href="#pone.0078122-Blum1">[7]</a>. For example, the Sackin index <a href="#pone.0078122-Sackin1">[8]</a> counts the number of internal nodes along the path from each leaf node to the root of the tree. When branching events occur preferentially along specific lineages, the corresponding phylogeny should tend to have a higher Sackin index. However, tree balance statistics are notoriously sensitive to sample size <a href="#pone.0078122-Pompei1">[9]</a> and capture only one abstraction of tree shape; for example, differences in branch lengths are generally not taken into consideration.</p> <p>Here, we present a new approach to the study of phylodynamics using a versatile method from machine learning <a href="#pone.0078122-Aizerman1">[10]</a>. We begin with a function <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> that maps an object <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> into a representation space <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> that is more amenable to analysis than <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. A well-conceived <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> can be extremely useful for dealing with structured data such as trees, which otherwise have no direct numerical representation. For example, Sackinâ€™s index is a function that maps from the space of all possible tree shapes to the set of all positive integers. While integer representations of tree shape are far more convenient for analysis, this choice for <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> is not adequate for our purposes because it captures only a small fraction of the variation among tree shapes. When dealing with complex data, it is more appropriate to use a function that maps objects into a representation space with many dimensions. For example, one could devise a function to be applied to blocks of text that counted the occurrence of substrings of any length. Every possible substring adds a new dimension to the feature space, as <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> is referred to in the field of pattern recognition. As the data become larger and more complex, the number of possible features (and thereby the dimensionality of <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>) may become enormous. Thus, while <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> can provide a detailed representation of complex objects, it also becomes infeasible to operate within. Because we are interested specifically in finding linear relations between objects that have been mapped to <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, we bypass the difficult task of explicitly mapping each object to <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> by defining a function that efficiently evaluates the inner product between two representations of objects in <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>: <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> where <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. This procedure is known as the <i>kernel trick</i> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> is referred to as the kernel function <a href="#pone.0078122-Vert1">[11]</a>. As kernel functions are generally designed to take larger values when <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> have a greater number of features in common, they can each be interpreted as a non-linear measure of similarity.</p> <p>To provide an intuition to this use of the kernel trick, imagine a person who has encountered three texts written in English â€" specifically, the texts contained in Agapow and Purvis <a href="#pone.0078122-Agapow1">[12]</a>, Huelsenbeck <i>et al.</i><a href="#pone.0078122-Huelsenbeck1">[13]</a>, and Robinson and Foulds <a href="#pone.0078122-Robinson1">[14]</a>. Moreover, suppose this person has absolutely no knowledge of the English language, let alone phylogenetics. Realizing that there are discrete shapes (words) and that many of them appear repeatedly, he or she carries out the lengthy task of cataloguing the number of times that each shape appears in each text. (For the sake of example, we ignore such complications as punctuation and typography.) While there are many shapes that are unique to a given text, there is a much smaller subset of shapes that appear in two or more texts. Therefore, in order to compare two of these lists, it is convenient to multiply the counts of shapes that appear in both lists (knowing that unique shapes would multiply to zero) and sum these products to generate a similarity score. Multiplying the counts has the useful effect of emphasizing shapes that appear in both texts at roughly equal frequencies. For example, â€˜treeâ€™ appears 37 times in both <a href="#pone.0078122-Agapow1">[12]</a> and <a href="#pone.0078122-Robinson1">[14]</a>, which contributes a score of 1369. In contrast, if all but one of the 74 occurrences of â€˜treeâ€™ appeared in one of the two texts, then the score would have increased by only 73. This exercise yields three numbers corresponding to the pairwise comparisons: 264630 <a href="#pone.0078122-Agapow1">[12]</a>, <a href="#pone.0078122-Huelsenbeck1">[13]</a>, 141470 <a href="#pone.0078122-Agapow1">[12]</a>, <a href="#pone.0078122-Robinson1">[14]</a>, and 277008 <a href="#pone.0078122-Huelsenbeck1">[13]</a>, <a href="#pone.0078122-Robinson1">[14]</a>. However, <a href="#pone.0078122-Huelsenbeck1">[13]</a> is a much lengthier text that the other two, which has inflated the scores for pairwise comparisons that included this text. It is necessary to normalize each score for the size of the corresponding texts <a href="#pone.0078122-Collins1">[15]</a>; upon doing so, our imagined person is able to conclude that <a href="#pone.0078122-Agapow1">[12]</a> and <a href="#pone.0078122-Robinson1">[14]</a> are the most similar (0.84) without knowing that these papers are, in fact, both on the subject of phylogenetic tree shape, whereas <a href="#pone.0078122-Huelsenbeck1">[13]</a> (with normalized scores of 0.82 and 0.78, respectively) concerns the problem of inferring the root of a phylogeny.</p> <p>To construct a kernel function on phylogenetic tree shapes, we adapted a natural language processing kernel function <a href="#pone.0078122-Moschitti1">[16]</a> that was originally designed to classify text on the basis of its syntactic structure (a generative tree in which words descend from linguistic precursors <a href="#pone.0078122-Collins1">[15]</a>). Our modified kernel function <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> extracts all the subset trees that are the common features of two phylogenetic trees <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> (<a href="#pone-0078122-g002">Figure 2</a>). A subset tree is a contiguous collection of descendants of a specific node, but unlike a subtree, it does not necessarily include all of the descendants. Thus, our approach is similar to the Robinson-Foulds metric <a href="#pone.0078122-Robinson1">[14]</a> which compares alternative trees for a given set of taxa by counting the number of subtrees in common. Unlike the Robinson-Foulds metric, however, our kernel function not only allows us to compare trees relating different numbers and kinds of taxa, but it also accounts for differences in the branch lengths between matching subset trees. We assess the performance of our kernel function against nine measures of tree shape at classifying simulated phylogenies with varying rates of speciation, and then apply our function to a large collection of human and zoonotic RNA virus phylogenies.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0078122.g002"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <div class="title" tagx="title" title="title">
Kernel-assisted comparison of two tree shapes.</div> <p>For trees comprising <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> nodes, respectively, there are <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> pairs of nodes to evaluate. (A) Starting from a given pair of nodes (indicated in figure by circles with double-outlines), the algorithm finds the largest common subset tree rooted at these nodes. First, we find that for both nodes, neither of the branches terminate at a â€˜leaf nodeâ€™ (marked with â€˜<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>â€˜). This match contributes a relatively small amount to our kernel score, not only because the matching subset trees (highlighted in thick blue lines) comprise only one node each, but also because their discordant branch lengths lead to a substantial penalty. (B) Next, we descend down the left branch in both trees. The current nodes (open circles) in both trees spawn one leaf node and one internal node; therefore, the subset trees continue to match. In addition, their branch lengths are similar, so their contribution to the cumulative kernel score is given greater weight. (C) Finally, we descend down the right branch in both trees and find that the subset trees no longer match beyond this point. We also proceed down the right branch of the reference nodes and find no match, so our traversal of the two trees from these nodes is complete and we restart our search at the next pair of nodes.</p>   </div> </div> <div class="methods" title="methods"> <div class="title" tagx="title" title="title">
Methods</div> <div class="datacollection" title="sec"> <div class="title" tagx="title" title="title">
Data Collection</div> <p>A working list of RNA viruses associated with human pathologies was obtained from the ViralZone web resource (<a href="http://viralzone.expasy.org/all_by_species/678.html">http://viralzone.expasy.org/all_by_species/678.html</a>) <a href="#pone.0078122-Hulo1">[17]</a>. For each virus, we attempted to gather as many sequences as available covering a region of the genome with extensive genetic variation, based on prior knowledge or phylogenetic studies in the literature. When available, we used curated virus-specific sequence databases. HIV-1 <i>nef</i> gene sequences were obtained from the Los Alamos National Laboratory (LANL) HIV Sequence database (<a href="http://www.hiv.lanl.gov/">http://www.hiv.lanl.gov/</a>) using subtype-specific search queries excluding sequences annotated as recombinants, and restricting the result to one sequence per patient. Further screening for HIV-1 recombinant sequences was carried out using SCUEAL (Subtype Classification Using Evolutionary ALgorithms) <a href="#pone.0078122-KosakovskyPond1">[18]</a>. Hepatitis C virus E1 sequences were obtained from the LANL HCV sequence database (<a href="http://hcv.lanl.gov">http://hcv.lanl.gov</a>) <a href="#pone.0078122-Kuiken1">[19]</a> using genotype-specific search queries excluding recombinants. Search results were reduced to one sequence per patient based on patient clone annotation in the sequence headers using a custom Python script, as this functionality was not available for the LANL HCV sequence database. Influenza A virus hemagglutinin (HA) sequences were obtained from the National Institute of Allergy and Infectious Diseases (NIAID) Influenza Research Database (<a href="http://www.fludb.org">http://www.fludb.org</a>), using subtype-specific search queries excluding isolates from non-human hosts and laboratory strains.</p> <p>Gene sequences from all other viruses were obtained from the GenBank sequence database by primary organism taxonomy identifier (â€˜txidâ€™). Queries yielding fewer than 100 records irrespective of gene were omitted at this stage. In all cases, sequence data were downloaded as FASTA formatted files; these published data were already anonymized and did not include any information about patientsâ€™ medical records. Sequences annotated as multiple clonal isolates from the same patient, sequences from patents or laboratory strains were excluded from these data. For all viruses other than HCV, HIV and IAV, we retrieved collection dates and host organism data, when available, for each sequence by its Genbank accession number using an automated query algorithm implemented in Python. By convention, Genbank records are annotated with this information in a â€˜sourceâ€™ field; however, this field is not always included in data submissions. For HCV, HIV and IAV sequences, which were already screened for human hosts, collection dates were parsed from sequence headers that were generated by the respective database engines.</p> </div> <div class="sequencealignment" title="sec"> <div class="title" tagx="title" title="title">
Sequence Alignment</div> <p>For each FASTA file, one sequence was selected as a reference for initial pairwise alignment of all other sequences using an implementation of the modified Gotoh algorithm in HyPhy <a href="#pone.0078122-Gotoh1">[20]</a>, <a href="#pone.0078122-KosakovskyPond2">[21]</a>. Sequences were trimmed to the aligned portion and converted to a multiple sequence alignment using MUSCLE <a href="#pone.0078122-Edgar1">[22]</a> version 3.8 under default settings. All alignments were visually inspected and adjusted using Se-Al (Andrew Rambaut, <a href="http://tree.bio.ed.ac.uk/software/seal">http://tree.bio.ed.ac.uk/software/seal</a>). Columns in the alignment comprising over 50% gaps were excluded. A table of all viruses, subtypes and genomic regions used for phylogenetic reconstruction is provided as Supporting Information (<a href="#pone.0078122.s005">Table S1</a>).</p> <p>Initial phylogenies were generated using FastTree (version 2.1.4) <a href="#pone.0078122-Price1">[23]</a> under the general time-reversible model of nucleotide substitution. These phylogenies were used to identify clusters of sequences that could represent multiple isolates from the same individual, which we determined by evaluating the publications linked with sequences within putative clusters. Furthermore, the initial phylogenies were used to partition or filter alignments by genotype or subtype, because the Genbank sequence records were not consistently annotated with this information. The data sets that were partitioned or filtered using the initial phylogenies were: astrovirus, dengue virus type 1, hepatitis delta virus, hepatitis E virus, influenza A virus H1, Japanese encephalitis virus, Norwalk virus genotype II, poliovirus (wild and vaccine variants), rabies virus, rotavirus A, rubella virus, sapovirus, tick-borne encephalitis virus, and West Nile virus.</p> </div> <div class="phylogeneticreconstruction" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic Reconstruction</div> <p>Although it is possible to apply our kernel method (described below) to an unrooted tree by counting subtrees from the deepest node outwards, phylodynamic studies of tree shapes have conventionally been carried out on rooted trees which can be more informative about evolutionary processes. The root of a phylogeny is a hypothesis on the location of most recent common ancestor (the earliest point in time) on the tree. We used an outgroup criterion <a href="#pone.0078122-Huelsenbeck1">[13]</a> to infer rooted phylogenies from the alignments. This method relies on selecting an outgroup sequence that is sufficiently closely related to the true ancestor. For each alignment, we selected an outgroup sequence based on previous phylogenetic studies on the respective viruses in the literature, either a representative sequence from a closely-related virus genotype or subtype (<i>e.g.</i>, HIV-1 subtype D), or the earliest/prototype virus isolate (<i>e.g.</i>, West Nile virus isolate B956 <a href="#pone.0078122-Smithburn1">[24]</a>). A list of outgroup sequence accession numbers and literature citations is provided in <a href="#pone.0078122.s005">Table S1</a>. We assessed the suitability of the respective outgroup sequences by generating preliminary rooted trees using FastTree2. While results based on tree balance statistics were sensitive to outgroup rooting, our results based on the kernel analysis were robust to choice of outgroup (data not shown).</p> <p>For alignments with 200 or more sequences, we generated 100 replicate subsets of 100 sequences sampled at random without replacement from each data set for phylogenetic reconstructions. This resampling controlled for the effect of excessive variation in tree size (number of sequences) on comparisons of tree shape. Otherwise, a single phylogeny was generated for each alignment with fewer than 200 sequences (Crimean-Congo hemorrhagic fever virus; Chikungunya virus; encephalomyocarditis virus; HIV-1 subtype A; Hantaan virus; human rotavirus C; influenza A virus H2; influenza C virus; Murray valley encephalitis virus; mumps virus; Oropouche virus; human parainfluenza virus; Rift valley fever virus; rhinovirus A; rotavirus A genotypes 3, 4, and 12; rubella virus clade 2; sapovirus; tick-borne encephalitis virus, European and Far-Eastern strains; yellow fever virus). For data sets in which 200 or more sequences were annotated with collection dates, we generated 10 additional samples of 100 sequences drawn uniformly at random with respect to the year of collection. Phylogenies reconstructed from these latter samples provided a means to assess the sensitivity of our analyses to biased sample collection with respect to time.</p> <p>Phylogenetic reconstruction for every sample was carried out using a multithreaded implementation of RAxML (version 7.3.0) <a href="#pone.0078122-Stamatakis1">[25]</a> under a general time-reversible model of nucleotide substitution with 25 rate categories across sites, followed by optimization under a discretized gamma distribution of rate categories (GTRCAT). All phylogenies contained a number of â€˜softâ€™ polytomies, in which the phylogenetic reconstruction was unable to assign a non-zero branch length at an internal node due to inadequate genetic divergence. Because RAxML is limited to strictly bifurcating representations of phylogenetic trees (with the exception of the deepest node from which all branches are interpreted as descendants), soft polytomies were automatically resolved by the software into an arbitrary binary tree with branch lengths set to a small non-zero value. By randomizing the resolution of soft polytomies into binary trees, we found that our results were robust to this source of uncertainty in phylogenetic reconstruction (data not shown).</p> <p>For every phylogeny, we calculated two popular summary statistics of tree balance, Sackinâ€™s index and Collessâ€™ index following the formulae in <a href="#pone.0078122-Kirkpatrick1">[26]</a>. Next, we generated a kernel matrix for all phylogenies using an implementation of our phylogenetic tree shape kernel as described in the following section.</p> </div> <div class="kernelanalysis" title="sec"> <div class="title" tagx="title" title="title">
Kernel Analysis</div> <p>To apply the kernel function to these data, we needed to iterate through all internal nodes in two phylogenies and count all subset trees with the same topology (branching order) in both phylogenies (<a href="#pone-0078122-g002">Figure 2</a>). This is akin to quantifying the similarity of two manuscripts by counting the number of times that various words appeared in both texts, as illustrated by our example in the Introduction. Unlike a text, however, the shape of a tree is an ambiguous characteristic of a phylogeny. Rotating the branches around a node can produce a tree that has a different shape, despite remaining evolutionarily equivalent to the original. The number of different shapes for a tree of <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> tips (where we distinguish left from right and, for the moment, ignore branch lengths) is given by the Catalan numbers <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. For example, a tree of 100 tips has nearly <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> shapes. This presents a problem: how do we know which resolutions of shapes are being compared between two trees? We resolved this issue by â€˜ladderizingâ€™ the trees: rotating branches around every node so that branches leading to the larger number of descendants (tips) were always on the same side. This procedure, conventionally used to produce a more aesthetically pleasing visualization of a phylogeny, greatly increases the chance of finding matching subset trees between two phylogenies. Furthermore, we rotated branches around all â€˜cherriesâ€™ (nodes that are the direct ancestor of two tips) such that the longer branch was always on the same side. Based on simulations (see below), we determined that this combination of ladderization and rotating cherries significantly improved our ability to correctly classify simulated trees (<a href="#pone.0078122.s006">Table S2</a>), indicating that these steps resolved different trees into shapes which could be compared in a consistent and meaningful way.</p> <p>Following Moschittiâ€™s notation <a href="#pone.0078122-Moschitti1">[16]</a>, we defined a phylogenetic tree shape kernel by: </p><div class="disp-formula" title="disp-formula"> <div class="graphic" position="anchor" orientation="portrait" /><span class="label" tagx="label" title="label">(1)</span></div>where <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> is the set of all nodes from the <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> tree <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> is defined as follows:<p /> <ul> <li> <p>If <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> are leaf nodes, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, where <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> is a constant decay factor.</p> </li> <li> <p>If the numbers of children descending from <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> are the same (always two for bifurcating trees) and, of these, the numbers of leaf nodes are also the same, then:</p> </li> </ul> <p> </p><div class="disp-formula" title="disp-formula"> <div class="graphic" position="anchor" orientation="portrait" /> </div>where <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> is the number of children descending from node <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> is the <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>-th child of node <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> is a Gaussian radial basis function on the vectors of branch lengths descending from nodes <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, denoted as <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> respectively:<p /> <p> </p><div class="disp-formula" title="disp-formula"> <div class="graphic" position="anchor" orientation="portrait" /> </div>where <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> is the variance parameter that determines how strongly subset trees are penalized for discordant branch lengths.<p /> <ul> <li> <p>Otherwise, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>.</p> </li> </ul> <p>Note that matching subset trees were weighted in proportion to their size (number of nodes). Because a tree will match its own shape exactly, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> can take overwhelmingly large values. This â€˜large diagonalâ€™ effect <a href="#pone.0078122-Weston1">[27]</a> can severely curtail the effectiveness of subsequent analyses of the kernel matrix (<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>), which is a symmetric matrix that is obtained by applying the kernel function to all pairs of <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> trees, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. Therefore, the constant decay factor <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> served to penalize larger subset trees because it declined exponentially with tree size <a href="#pone.0078122-Collins1">[15]</a>. The Gaussian radial basis function <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> further penalized subset trees by the discordance in branch lengths for pairs of branches originating from the matching nodes (<a href="#pone-0078122-g002">Figure 2</a>). Although this incorporation of branch lengths is a significant departure from the original tree kernel for natural language processing, we prove that <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> is a valid positive semi-definite kernel function (<a href="#pone.0078122.s007">Text S1</a>).</p> <p>Phylogenies were imported from the Newick tree strings produced by RAxML into Python using the Phylo module from the Biopython package <a href="#pone.0078122-Cock1">[28]</a>, <a href="#pone.0078122-Talevich1">[29]</a>. Branch lengths in each phylogeny were normalized by the mean branch length to facilitate comparisons between viruses with different overall rates of evolution. A kernel matrix was computed from this collection of phylogenies using a custom Python module, which we have made publicly available on our webserver (<a href="http://bioinfo.cfenet.ubc.ca/pub/phylokernel">http://bioinfo.cfenet.ubc.ca/pub/phylokernel</a>). Although the kernel function can be computed quickly <a href="#pone.0078122-Moschitti1">[16]</a>, we needed to compute the kernel matrix for 4153 phylogenies (<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> replicates + <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> samples by collection year + <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>) with over 8 million pairwise comparisons. To speed up this calculation, we performed parallel computation on a Linux cluster with a message-passing interface (MPI) environment <a href="#pone.0078122-Dalcn1">[30]</a>. The matrix was confirmed to be positive semi-definite by Cholesky decomposition. Following Collins and Duffy <a href="#pone.0078122-Collins1">[15]</a>, all entries in the matrix were normalized by the formula </p><div class="disp-formula" title="disp-formula"> <div class="graphic" position="anchor" orientation="portrait" /> </div>so that the resulting kernel matrix was less sensitive to differences in the sizes of the respective trees. Values of <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> for replicate sets of phylogenies generated from random samples were averaged for every pairwise comparison of viruses or virus clades, and the resulting group mean distance matrix was renormalized. Kernel principal components analysis and support vector machine classification were performed on this normalized kernel matrix using the <i>R</i> package <i>kernlab</i><a href="#pone.0078122-Karatzoglou1">[31]</a>.<p /> </div> <div class="simulatingphylogenies" title="sec"> <div class="title" tagx="title" title="title">
Simulating Phylogenies</div> <p>We used the R package <i>diversitree</i><a href="#pone.0078122-FitzJohn1">[32]</a> to simulate the growth of phylogenies under different evolutionary scenarios of branching or â€˜speciationâ€™ rates. Trees were generated under a Quantitative State Speciation and Extinction (QuaSSE) model <a href="#pone.0078122-FitzJohn2">[33]</a> whereby speciation rates were determined by a continuous trait whose evolution was simulated by a Brownian motion (with zero drift and variance per unit time <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>) under two different mutation rates (<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>â€Š=â€Š0.1 and 0.01). This continuous trait was mapped to a speciation rate by a sigmoidal function with a zero midpoint, exponential decay rate of 0.25, and minimum and maximum values of 0.05 and 0.25, respectively. We performed two sets of 100 replicate simulations under the different mutation rate scenarios. The simulated data were converted into Newick tree strings using a Python script derived from the Biopython Phylo module <a href="#pone.0078122-Talevich1">[29]</a>. These strings are publicly available at <a href="http://bioinfo.cfenet.ubc.ca/pub/phylokernel">http://bioinfo.cfenet.ubc.ca/pub/phylokernel</a>. We calculated nine different tree balance statistics â€" Collessâ€™ index, Sackinâ€™s index, mean path length, variation in path length, Shao and Sokalâ€™s <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, and the sum, total mean, and mean of the 10 earliest nodes of Fusco and Cronkâ€™s imbalance statistic <a href="#pone.0078122-Purvis1">[34]</a> â€" for all trees using functions written in Python. A kernel matrix was computed for all trees using the methods described above. We trained a <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>-support vector machine classifier <a href="#pone.0078122-Schlkopf1">[35]</a> using the R package <i>kernlab</i> on a random subset of half of the phylogenies, and then measured the sensitivity and specificity of classification by mutation rate on the remaining half. Likewise, the sensitivity and specificity of classifying trees by tree balance statistics was evaluated at varying cutoffs using the R package <i>ROCR</i><a href="#pone.0078122-Sing1">[36]</a>.</p> </div> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="performanceonsimulatedtrees" title="sec"> <div class="title" tagx="title" title="title">
Performance on Simulated Trees</div> <p>We simulated two sets of 100 replicate phylogenies under different mutation rates (200 trees total) that influenced the branching rates in the phylogeny in an autocorrelated fashion <a href="#pone.0078122-FitzJohn1">[32]</a>. In other words, the branching rate was allowed to evolve as each phylogeny grew from a common ancestor to 100 descendants. These simulations provide a reasonable approximation of the kind of evolutionary scenarios between which tree balance statistics were meant to differentiate <a href="#pone.0078122-Agapow1">[12]</a>, since preferential branching along fewer lineages will tend to produce an imbalanced tree. For all simulated phylogenies, we computed nine different balance statistics (the eight statistics assessed in <a href="#pone.0078122-Agapow1">[12]</a> and Sackinâ€™s index). In addition, we computed the kernel matrix for all simulated phylogenies using our phylogenetic tree shape kernel. Note that the kernel method does not function as a summary statistic; it cannot assign a value to a given tree in the absence of any other information. Consequently, we compared the kernel method to balance statistics by evaluating their performance at correctly classifying trees that had been generated under different models of speciation, which is analogous to the transmission of an infectious disease between hosts.</p> <p>The sensitivity and specificity of classifying the simulated phylogenies by mutation rates (<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>â€Š=â€Š0.01 and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>â€Š=â€Š0.1) is summarized in <a href="#pone-0078122-g003">Figure 3</a>. Using our kernel method, we obtained a median sensitivity of 97.7% (interquartile range, IQRâ€Š=â€Š95.6%, 98.0%) and specificity of 90.8% (IQRâ€Š=â€Š89.1%, 92.8%) when averaged over 100 replicate training sets. These results were unambiguously superior to all nine balance statistics that were evaluated over the same simulated data. For instance, the sum of Fusco and Cronkâ€™s imbalance statistic was the most effective among the balance statistics, but none could exceed a sensitivity of <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>80% without a corresponding drop in specificity below <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>80%. Thus, the kernel method is capable of providing a substantial advantage for recognizing virus trees that have evolved under different evolutionary and epidemiological scenarios.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0078122.g003"><span class="label" tagx="label" title="label">Figure 3</span></a></div>   <div class="title" tagx="title" title="title">
Classification of simulated phylogenies using nine balance statistics and the kernel function.</div> <p>We simulated the growth of two sets of 100 phylogenies relating 100 taxa under different scenarios in which rates of speciation (branching) evolved at different rates. Greater variation in speciation rates tended to produce more imbalanced trees. Nine different balance statistics, including eight from <a href="#pone.0078122-Agapow1">[12]</a>, were computed for all phylogenies: Collessâ€™ index, Sackinâ€™s index, the mean and variance in path lengths from tips to the root, Shao and Sokalâ€™s <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> statistics, and the imbalance value (<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>) for the sum, total mean, and the mean of the earliest 10 internal nodes of the tree. This plot illustrates the trade-off between sensitivity and specificity of classifying phylogenies by applying a cutoff value each of these balance statistics. A single point (star) indicates the sensitivity and specificity attained by applying the phylogenetic kernel function (with <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>) to train a support vector machine (SVM) on a random subset (50%) of the phylogenies, and classifying the remaining half.</p>   </div> <p>To determine the sensitivity of the kernel function to <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, we evaluated our ability to classify simulated phylogenies under varying parameter settings. We found that both sensitivity and specificity were robust to different settings of <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> (<a href="#pone.0078122.s001">Figure S1</a>). For example, mean sensitivities varied by less than 1% (0.97 to 0.98) for values of <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> ranging from 1.0 to 20.0, given <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>; setting <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> conferred a slight advantage over this range in the classification of simulated phylogenies. These tests indicated that results from applying our kernel function to RNA virus phylogenies would be robust to varying parameter settings. This conjecture was supported in a comparison of kernel matrix projections under different settings (<a href="#pone.0078122.s002">Figure S2</a>; see below).</p> </div> <div class="treebalanceinrnaviruses" title="sec"> <div class="title" tagx="title" title="title">
Tree Balance in RNA Viruses</div> <p>We used maximum likelihood-based heuristics to reconstruct phylogenies from 62 sequence alignments representing 38 different human and zoonotic RNA viruses from 12 different taxonomic families (<a href="#pone.0078122.s005">Table S1</a>). Genetic sequences from viruses with highly divergent clades, such as the human immunodeficiency virus type 1 (HIV-1) subtypes or influenza A virus serotypes, were grouped accordingly into separate multiple sequence alignments to allow for clade-specific epidemiological or evolutionary dynamics. Replicate phylogenies were generated from random samples of 100 sequences (given adequate sample size) to control for the well-documented confounding effect of variation in sample size on conventional tree shape statistics <a href="#pone.0078122-Pompei1">[9]</a>.</p> <p>For each alignment, we calculated the means across replicates of two tree balance statistics (Collessâ€™ index and Sackinâ€™s index), which quantify the asymmetry in rates of branching among ancestral lineages in each tree. Specifically, Sackinâ€™s index <a href="#pone.0078122-Sackin1">[8]</a> counts the number of ancestral nodes separating each tip to the root, while Collessâ€™ index <a href="#pone.0078122-Colless1">[37]</a> evaluates the difference in the number of tips that descend from the left and right branches of each bifurcating ancestral node. The distribution of mean Collessâ€™ indices is displayed in <a href="#pone-0078122-g004">Figure 4</a>. As expected, influenza A virus (IAV) serotype H3 assumed one of the highest mean Collessâ€™ indices (0.544) indicating that the replicate trees were severely imbalanced; only Murray valley encephalitis virus (MVEV) presented a higher index (0.548). Only 39 published MVEV envelope sequences were available after culling 7 identical sequences; however, Collessâ€™ index is expected to be biased downwards with decreasing sample size, suggesting that the mean index for 100-sequence MVEV trees could have been even greater still. MVEV is endemic to Australia and Papua New Guinea and comprised almost entirely of mosquito isolates sampled over six decades (1956â€"2008). The IAV H3 sequences were strictly human isolates from around the globe but also sampled over a period of several decades (1968â€"2012).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0078122.g004"><span class="label" tagx="label" title="label">Figure 4</span></a></div>   <div class="title" tagx="title" title="title">
Distribution of mean normalized Collessâ€™ indices.</div> <p>Each label represents the mean index of a virus or virus clade. The vertical axis is used to elucidate the clustering of points by forcing overlapping labels (phylogenies with similar indices) to â€˜pile upâ€™ like a histogram. A higher Collessâ€™ index corresponds to a less â€˜balancedâ€™ tree in which branching events tend to occur along the same lineage. A conventional histogram is displayed in the background. Labels are defined as follows: AstVâ€Š=â€Š<i>astrovirus</i>; CCHFâ€Š=â€Š<i>Crimean-Congo hemorrhagic fever virus</i>; ChikVâ€Š=â€Š<i>chikungunya virus</i>; CA24vâ€Š=â€Š<i>coxsackievirus A24</i>; DENâ€Š=â€Š<i>dengue virus</i>; E30â€Š=â€Š<i>echovirus 30</i>; EMCVâ€Š=â€Š<i>encephalomyocarditis virus</i>; EV71â€Š=â€Š<i>enterovirus 71</i>; GBVCâ€Š=â€Š<i>GB virus C</i>; HTNVâ€Š=â€Š<i>Hantaan virus</i>; H[A-E]Vâ€Š=â€Š<i>hepatitis [A-E] virus</i>; HIVâ€Š=â€Š<i>human immunodeficiency virus type 1</i>; I[A-C]Vâ€Š=â€Š<i>influenza [A-C] virus</i>; JEVâ€Š=â€Š<i>Japanese encephalitis virus</i>; MeVâ€Š=â€Š<i>measles virus</i>; MuVâ€Š=â€Š<i>mumps virus</i>; MVEVâ€Š=â€Š<i>Murray valley encephalitis virus</i>; NVâ€Š=â€Š<i>Norwalk virus</i>; OROVâ€Š=â€Š<i>Oropouche virus</i>; hPIV-1â€Š=â€Š<i>human parainfluenza virus</i>; PVâ€Š=â€Š<i>poliovirus</i>; Rabâ€Š=â€Š<i>rabies virus</i>; Rotâ€Š=â€Š<i>human rotavirus</i>; RhiVâ€Š=â€Š<i>human rhinovirus</i>; RSVâ€Š=â€Š<i>human respiratory syncytical virus</i>; Rubâ€Š=â€Š<i>rubella virus</i>; RVFâ€Š=â€Š<i>Rift valley fever virus</i>; SapVâ€Š=â€Š<i>sapovirus</i>; SeoVâ€Š=â€Š<i>Seoul virus</i>; TBEVâ€Š=â€Š<i>tick-borne encephalitis virus</i>; WNVâ€Š=â€Š<i>West Nile virus</i>; YFVâ€Š=â€Š<i>yellow fever virus</i>.</p>   </div> <p>This raises a concern of whether tree balance is an artefact of biased sampling of virus isolates over time, such that the overrepresentation of isolates from a narrow range of years might tend to yield balanced trees. To address this, we calculated the mean Collessâ€™ indices for phylogenies generated from samples that were uniform with respect to sample collection years for alignments in which an adequate number of sequences were annotated with collection dates (<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>). Using Linâ€™s concordance correlation coefficient (<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>) <a href="#pone.0078122-Lin1">[38]</a> to quantify the agreement between indices, where <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> indicates perfect agreement and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> indicates no agreement, we found that the mean Collessâ€™ indices for phylogenies whose tips were uniformly distributed over years were highly concordant with indices from random samples (<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, 95% C.I.<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>). In other words, constraining the phylogenies of other viruses such as HIV to have evenly distributed tips over time did not make their Collessâ€™ indices more IAV-like.</p> <p>In sum, the distribution of mean Collessâ€™ indices was highly peaked in the interval between 0.15 to 0.2, such that this summary statistic did not discriminate substantially among the phylogenies of most RNA viruses in our study (<a href="#pone-0078122-g004">Figure 4</a>). For example, the third-highest mean index (<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>) derived from IAV serotype H1 phylogenies was similar to the index of a hepatitis C virus (HCV) genotype 2a phylogeny (<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>), which would not seem to be consistent with the observation that phylogenies from HCV and IAV have conspicuously different shapes <a href="#pone.0078122-Grenfell1">[2]</a>. Specifically, we found that 6 out of 100 replicate samples of IAV-H1 sequences led to Collessâ€™ indices that were less than the value observed for HCV-2a. Similarly, the relatively low Collessâ€™ indices of phylogenies of influenza B virus (IBV, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>), which together with IAV serotypes H1 and H3 is responsible for the majority of seasonal influenza outbreaks, implies that the imbalance of IBV-derived trees is no different from the bulk of animal RNA viruses (<a href="#pone-0078122-g004">Figure 4</a>). Furthermore, the mean Collessâ€™ index did not differentiate the phylogenies of HIV and HCV from other RNA viruses. We obtained the same qualitative results from an evaluation of mean Sackinâ€™s indices (<a href="#pone.0078122.s003">Figure S3</a>), which were highly correlated with the respective mean Collessâ€™ indices (Pearsonâ€™s <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>). There is much more to the conspicuously star-like shape of HIV and HCV phylogenies that remains unquantifiable by balance statistics. Consequently, relying too heavily on tree balance statistics potentially limits our ability to measure the diversity of human and zoonotic RNA virus phylogenies, and to link recurring shapes to epidemiological or evolutionary processes.</p> </div> <div class="kernelanalysisofrnavirustrees" title="sec"> <div class="title" tagx="title" title="title">
Kernel Analysis of RNA Virus Trees</div> <p>Here, we develop and apply a new approach to comparing RNA virus tree shapes by using a method from machine learning known as the â€˜kernel trickâ€™ <a href="#pone.0078122-Aizerman1">[10]</a>. Rather than attempting to reduce a complex tree shape down to one or more summary statistics, like the tree balance indices examined above, we defined a kernel function (<a href="#pone-0078122-g002">Figure 2</a>) that efficiently computes inner products between phylogenies with respect to their tree shapes. By this approach, a treeâ€™s shape is quantified only by its comparison to other trees. We have shown above that this method can be highly effective at discriminating between the shapes of phylogenies that were simulated under different evolutionary scenarios. We applied this phylogenetic tree kernel to our collection of RNA virus phylogenies to generate a kernel matrix that is akin to a pairwise similarity matrix. Entries in the kernel matrix were averaged across replicate phylogenies from the same alignments, and renormalized to yield a smaller kernel matrix with rows or columns corresponding to different viruses or virus clades and sampling schemes (completely at random or uniformly with respect to collection year). According to the eigenvalues of this projection, about 90% of the variance was explained by the first 2 principal components alone, with about 70% of the variance explained by the first component; these percentages varied slightly with different parameter settings of the kernel function (<a href="#pone.0078122.s002">Figure S2</a>). In all cases, the first component distinctly separated the phylogenies derived from HIV and HCV sequences from the rest of the RNA viruses. However, none of the remaining principal components were as readily interpretable. Furthermore, neither Sackinâ€™s nor Collessâ€™ indices were significantly correlated with any of the principal components based on a simple Pearson correlation test after adjusting for multiple comparisons. Additionally, we trained a support vector regression model using the kernel matrix and Collessâ€™ indices for a random subset of the data. The mean concordance correlation coefficient <a href="#pone.0078122-Lawrence1">[39]</a> between the observed Collessâ€™ indices and those predicted by this model for the remaining phylogenies was only 0.12 (interquartile range 0.07, 0.18), which is considered poor. These results, which were robust to varying <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, suggested that the kernel was manifesting aspects of phylogenetic tree shapes other than those captured by tree balance statistics.</p> <p>A scatterplot of the largest 2 principal components from the preceding analysis described a single arc in which HIV and HCV phylogenies comprised a distinct cluster from the other RNA viruses (<a href="#pone.0078122.s002">Figure S2</a>). However, the majority of the other viruses were agglomerated at the base of the arc, making it difficult to discern patterns from this visualization. This is a known issue in PCA in which the largest components are assumed to represent the important structure in the data whereas smaller components represent noise. To provide a clearer visualization, we generated another scatterplot (<a href="#pone-0078122-g005">Figure 5</a>) using a t-distributed stochastic neighbor embedding algorithm (t-SNE) which attempts to preserve both global and local structure in a low-dimensional visualization of high-dimensional data <a href="#pone.0078122-vanderMaaten1">[40]</a>. Again, HIV and HCV comprised a distinct cluster in this visualization. Phylogenies derived from these viruses tend to feature long terminal branches and relatively short internal branches (<a href="#pone-0078122-g001">Figure 1</a>), which has been attributed to the exponential spread of these viruses and their propensity to establish persistent infections. This outcome was robust to partitioning sequences by clade; for example, a phylogeny comprised of all three HIV subtypes in our study mapped adjacent to the HIV subtype-specific phylogenies in both types of projections (data not shown).</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0078122.g005"><span class="label" tagx="label" title="label">Figure 5</span></a></div>   <div class="title" tagx="title" title="title">
A visualization of RNA virus phylogenies in the tree shape kernel space (<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>) using t-distributed stochastic neighbor embedding (t-SNE).</div> <p>The t-SNE algorithm attempts to find the optimal map of high-dimensional data into a low-dimensional space while preserving the distances among points as much as possible. Thus, the distance between pair of viruses or virus clades (labelled by the same abbreviations as <a href="#pone-0078122-g004">Figure 4</a>) is approximately proportional to their mean kernel distance. Groups of virus clades of particular interest are highlighted with the corresponding colours: HIV, red; HCV, yellow; Dengue (DEN), green; IAV-H3, IAV-H1, and IBV (blue).</p>   </div> <p>GB virus C (GBVC), which is not known to be associated with any human disease, was consistently located within the cluster of HIV and HCV phylogenies in the kernel PCA projections. (Due to the stochastic nature of the <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>-SNE algorithm, GBVC was not always embedded in the vicinity of HIV and HCV.) A close relative of HCV, GBVC can establish persistent infections in its human hosts and may inhibit HIV replication when an individual has become co-infected by both viruses <a href="#pone.0078122-Williams1">[41]</a>. Thus, the ability to establish persistent infections appears to be a common characteristic of viruses whose phylogenies map to this region of tree shape space. To test this hypothesis, we trained a support vector machine (SVM) classifier to differentiate between viruses having the ability to establish persistent chronic infections (HIV, HCV, and GBVC) and those without. We randomly selected half of the members from each category and used the corresponding rows and columns in the kernel matrix as training data. Validating the classifier on the remaining half of the kernel matrix, we consistently obtained a sensitivity and specificity of 1.0 irrespective of varying the kernel parameters <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, indicating that viruses that can establish persistent chronic infections were unambiguously separated from the other RNA viruses in the space defined by the tree kernel.</p> <p>The influenza viruses predominantly responsible for the seasonal epidemics around the world (IAV-H1, IAV-H3, and IBV) were located close together in both projections of the kernel matrix. In contrast, the Sackinâ€™s and Collessâ€™ indices for IBV were substantially lower than those of IAV-H1 and IAV-H3. This suggests that the kernel was more successful in identifying commonalities in the shapes of the phylogenies of these influenza viruses. On the other hand, these influenza viruses were located in the midst of many other RNA viruses in both projections, despite the strikingly pectinate shape of IAV-H3 (<a href="#pone-0078122-g001">Figure 1</a>). Phylogenies from the other influenza viruses in this study (IAV-H2, IAV-H5, and influenza C virus) mapped further away from the seasonal epidemic influenza viruses. These other phylogenies tended to have longer internal branches (relative to terminal branches) than the phylogenies of IAV-H1, IAV-H3, and IBV. Specifically, the ratio of the median branch lengths of internal and terminal branches, respectively, was significantly greater for the other influenza viruses than the seasonal epidemic influenza viruses (Wilcoxon rank sum test <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>). A possible explanation for this pattern is that the H2 and H5 subtypes are infrequently re-introduced into the human population from under-sampled avian host populations; consequently, the evolving diversity of these viruses would be represented by only a small number of lineages in phylogenies.</p> <p>Our survey of animal RNA viruses included several examples of zoonotic viruses. One might anticipate that these viruses could map to one or more distinct regions in projections of the kernel matrix. We observed that mosquito-transmitted zoonotic viruses such as the highly divergent dengue serotypes (DEN-1 to -4), the arboviral encephalitides (Japanese encephalitis virus and West Nile virus), and yellow fever virus tended to map to a similar region in projections of the kernel matrix, although this region was not exclusive of other viruses for which humans are the only known natural hosts, such as rubella virus and poliovirus (<a href="#pone-0078122-g005">Figure 5</a>). Another interesting feature of this projection was that phylogenies representing clades of the zoonotic rabies virus (RabV-1 to -3) mapped to a similar region as the mosquito-transmitted zoonotic viruses. The majority of the rabies virus sequences were isolated from non-human hosts predominated by canine (numbered here as clades 1 and 3) and bat species (clade 2), with about 2%â€"9% sequences isolated from human hosts (<a href="#pone.0078122.s005">Table S1</a>).</p> <p>Finally, we observed that phylogenies from certain viruses shared a distinctive rapid proliferation of short terminal branches within one or more specific clades which may be a signature of sporadic outbreaks, as illustrated by the coxsackievirus A24 variant phylogeny (CA24v) depicted in <a href="#pone-0078122-g001">Figure 1</a>. For example, the majority of CA24v sequences within the highly unresolved clade of its phylogenies were sampled during outbreaks of acute hemorrhagic conjunctivitis in Brazil and China. The proximity of the zoonotic viruses Hantaan virus (HTNV) and Murray Valley encephalitis virus (MVEV) to CA24v in projections of the kernel matrices (<a href="#pone.0078122.s002">Figure S2</a>) suggested that the shapes of the respective phylogenies may have also been shaped by sporadic outbreaks.</p> <p>To assess the potential bias of uneven sampling over time, we incorporated replicate phylogenies into the kernel matrix that were generated from samples of virus sequences drawn at random with respect to collection year. We found that the mean distances between phylogenies sampled completely at random and phylogenies sampled uniformly by collection year from the same virus or virus clade were significantly shorter (larger kernel scores) than the mean distances between phylogenies from different viruses or virus clades (Wilcoxon rank sum test, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>). Specifically, the mean normalized kernel score between phylogenies of the same virus obtained by different sampling schemes was 0.98, whereas the mean among phylogenies from different viruses was 0.84, where a normalized score of 1.0 indicates an exact match with respect to tree shape. On visually examining a PCA projection of this kernel matrix, we found that sampling uniformly by collection year tended to shift phylogenies along the same trajectory in the space of tree shapes (<a href="#pone.0078122.s004">Figure S4</a>). With the exception of HCV genotype 6a (for which about 50% of isolates were obtained in 2006 alone), the cluster of HIV and HCV viruses remained distinct from other RNA viruses including the influenza viruses.</p> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>In this study, we have proposed a new framework for comparing phylogenetic tree shapes by extending a parse tree kernel from computational linguistics <a href="#pone.0078122-Moschitti1">[16]</a>. Our kernel function compares all subset trees shared by two phylogenies and weights this comparison by the concordance of branch lengths that comprise the respective subset trees (<a href="#pone-0078122-g002">Figure 2</a>). It therefore bears some similarity to the weighted Robinson-Foulds metric that is based on comparing the lengths of branches that descend from each ancestral node <a href="#pone.0078122-Robinson1">[14]</a>. However, the Robinson-Foulds metric is restricted to comparing alternative trees relating the same taxa; in other words, the terminal branches must have the same labels. This requirement makes it impossible to use the Robinson-Foulds metric to compare phylogenies from different viruses. In contrast, our kernel is designed to compare trees from completely different sets of taxa because it discriminates among subset trees primarily by the number of terminal branches that descend from each ancestral node, regardless of what taxa occupy the tips of those branches. Moreover, if two phylogenies had a subset tree in common, that subset tree would have to occur in the same location of the trees to be counted towards the Robinson-Foulds metric. In contrast, since the tree shape kernel iterates over all internal nodes in both phylogenies, subset trees in different locations can contribute towards the kernel score. This is an important distinction because we are primarily interested in recurring motifs in tree shape, irrespective of where they appear in either tree, because this commonality may represent the imprint of a common epidemiological or evolutionary process acting on both viruses.</p> <p>Using conventional tree balance statistics, we found limited variation among phylogenies from different RNA viruses, with most viruses falling into a narrow range of Collessâ€™ or Sackinâ€™s indices after controlling for sample size variation. In contrast, our kernel function was able to separate the distinctively star-like phylogenies of HIV and hepatitis C virus. However, it was unable to distinguish the conspicuously pectinate shapes of IAV-H1 and -H3 phylogenies to the same extent as the tree balance statistics. This is most likely due to our use of a decay factor <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> in computing the kernel function to penalize larger subset tree matches. Unless we penalize subset trees by <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, large phylogenies would result in enormous values along the diagonal of the kernel matrix because a phylogeny will always match itself exactly, washing out any patterns from comparisons between different phylogenies. In other words, our use of a decay factor places greater emphasis on local than global similarities between the shapes of two phylogenies. Using a decay factor evidently prevents the full extent of the imbalance in the IAV-H1 or -H3 phylogenies from being expressed in the kernel matrix, since the longer subset trees that would represent this large-scale pattern are penalized more heavily.</p> <p>A common concern directed at studies of phylogenetic tree shapes is that variation in tree shapes may be an artefact of differences in sampling. For example, influenza A virus sequences in our data were sampled over a wide range of years from 1925 to 2012, with 72% of sequences sampled after 2001. It is possible that differences among viruses in the distributions of sample dates may introduce a bias to tree balance statistics or the kernel method. However, when we constrained samples across viruses to be sampled uniformly with respect to collection years, we found no significant effect on the distributions of RNA viruses for either tree balance or kernel-based methods. Analyses of tree shapes can be robust to temporal sampling biases because the internal structures of the phylogenies are often comprised of ancestors that are substantially further back in time. For example, there was very little clustering of HIV-1 subtype B sequences in the phylogeny with respect to collection year, implying that a phylogeny reconstructed a tree from sequences collected in a specific year will retain most of its deeper structure.</p> <p>The tree shape kernel presented here potentially represents an important new exploratory tool for studying RNA virus phylogenies. By extracting recurring local motifs in the internal structures of different phylogenies, it can complement the established tree balance statistics to provide a more complete description of the diversity of tree shapes among RNA viruses <a href="#pone.0078122-Holmes1">[3]</a>. Our kernel is not a parametric model in itself; unlike recent advances in viral phylodynamics <a href="#pone.0078122-Frost1">[42]</a>, it does not attempt to directly estimate epidemiological model parameters from tree shapes. Unlike a tree balance statistic, the kernel function cannot assign a numerical value to a single phylogeny in the absence of any other data. Without summary statistics to compare to reported values in the literature, it is necessary to have at hand all the phylogenies that one wishes to compare in order to compute the entire kernel matrix. On the other hand, the use of a kernel method presents opportunities to utilize other techniques from machine learning such as support vector machines (SVMs) to extract and analyze patterns in phylogenetic tree shapes. For example, we simulated phylogenies under two different evolutionary scenarios that influenced the asymmetry in branching rates among lineages, and we found that an SVM assisted by our kernel function clearly outperformed nine different tree balance statistics at predicting which phylogeny was generated under which scenario (<a href="#pone-0078122-g003">Figure 3</a>). Furthermore, we trained another SVM classifier on a random subset of phylogenies labelled by whether the respective RNA viruses could establish persistent chronic infections. The trained model consistently predicted these labels for the remaining phylogenies with very high sensitivity and specificity (see Results section).</p> <p>It is possible that a similar procedure could be used, for example, to determine the natural animal reservoir of an incompletely characterized zoonotic virus. This may not only require further development of kernel methods such as those presented here, but also greater coverage and more detailed annotation of published sequence data for RNA viruses that are under-represented in public databases. Additionally, the tree shape kernel may provide an effective distance measure for fitting complex phylodynamic models by approximate Bayesian computation <a href="#pone.0078122-Ratmann1">[43]</a>, a rapidly emerging class of highly versatile and efficient computational methods in which a distance measure (such as the difference in summary statistics) is used to assess the congruence between model simulations and observed data.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting Information</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure S1</span> <p><b>Sensitivities (solid lines) and specificities (dashed lines) of classifying simulated phylogenies where the kernel matrix was generated under different settings of </b><span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span><b> and </b><span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span><b>.</b> Classification was performed using a kernel <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>-support vector machine implemented in the <i>R</i> package <i>kernlab</i><a href="#pone.0078122-Karatzoglou1">[31]</a>. For a given kernel matrix, sensitivity and specificity were averaged over 1000 replicate cross-validations on random samples of trees stratified by evolutionary scenario. Note that the <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>-axis is <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>-transformed and the <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>-axis is scaled to the range of the observed values, which all exceeded the performance of classifications based on tree balance statistics. Varying the decay factor <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> had little effect on either sensitivity or specificity; setting <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> conferred a very slight advantage in sensitivity over <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> or <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. Both sensitivity and specificity were fairly robust to a wide range of values for the Gaussian radial basis function variance parameter <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. For example, mean sensitivities varied by less than 1% for values of <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> ranging from 1 to 20.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pone.0078122.s001.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure S2</span> <p><b>PCA projections for kernel matrices generated from the RNA virus phylogenies under different settings for kernel function parameters </b><span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span><b> and </b><span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span><b>.</b> Labels corresponding to virus species and clades are the same in <a href="#pone-0078122-g004">Figure 4</a>. Viruses of particular interest (such as HIV and the common pandemic influenza A viruses) are highlighted with coloured convex hulls as in <a href="#pone-0078122-g005">Figure 5</a>. The proportion of variance explained by the first two principal components (roughly 90% to 95%), as determined by eigenvalues, are reported as percentages in the axes labels.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pone.0078122.s002.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure S3</span> <p><b>Distribution of mean normalized Sackinâ€™s indices.</b> Each label represents the mean index of a virus or virus clade. The vertical axis is used to elucidate the clustering of points by forcing overlapping labels (phylogenies with similar indices) to â€˜pile upâ€™ like a histogram. A higher Sackin index corresponds to a less â€˜balancedâ€™ tree in which branching events tend to occur along the same lineage. Label annotations are identical to <a href="#pone-0078122-g004">Figure 4</a>.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pone.0078122.s003.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure S4</span> <p><b>Visualizing the effect of resampling virus sequences uniformly at random with respect to year of collection.</b> The kernel matrix calculated from the original sample phylogenies (black) and the additional phylogenies from uniform sampling by year (red) is projected onto the first two principal components. Arrows are drawn between phylogenies generated by different sampling schemes from the same set of virus sequences. Note that this PCA projection is slightly different from projection generated under the same settings (<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>) depicted in <a href="#pone.0078122.s002">Figure S2</a> because it incorporates the uniform sample points.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pone.0078122.s004.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Table S1</span> <p><b>A list of all animal RNA viruses in this study, grouped by taxonomic family (bold face).</b> Abbrev.â€Š=â€Šabbreviation used in figures. <i>N</i>â€Š=â€Šnumber of sequences. %humanâ€Š=â€Špercentage of sequences isolated from human hosts. <i>L</i>â€Š=â€Šlength of alignment in nucleotides. Outgroup accessionâ€Š=â€ŠGenbank accession number of sequence used as outgroup. Referenceâ€Š=â€Šcitation of article in peer-reviewed literature on which choice of outgroup is based; or a short note on the method used for outgroup selection.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pone.0078122.s005.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Table S2</span> <p><b>Performance of kernel support vector machine classifier on 200 simulated phylogenies.</b> Trees were simulated with the R package <i>diversitree</i> by evolving a latent character state (with mutation rates <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>) which controlled the rate of speciation (branching). All trees were modified by rotating branches around nodes according to some predefined scheme, resulting in different but evolutionarily invariant shapes. Kernel matrices were generated with the parameter settings <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. Sensitivity and specificity values were generated by R package <i>ROCR</i> and averaged across 1000 cross-validations (using a random subset of 100 trees to train the kernel classifier and validating on the remaining 100). Empirical 95% confidence intervals (C.I.) were derived from the 25th and 975th-ranked cross validations. Randomly rotating branches in the trees did not significantly affect our ability to classify them by evolutionary scenario (Studentâ€™s <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>-test, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>). Ladderizing the trees, such that the most prolific branches were rotated to the same side, conferred a substantial and significant gain in sensitivity and specificity of classification relative to the unmodified trees (<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>). Rotating â€˜cherriesâ€™ (pairs of tips that descend directly from their common ancestor) of a ladderized tree, so that the longest branch was always to the same side, conferred a slight but significant advantage in classification (<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>). However, we found no advantage to rotating branches around internal nodes according to any of the schemes we evaluated. â€˜Tiesâ€™ refer to nodes that are not cherries and have the same number of descendant tips to the left and right, making them ambiguous to ladderization. Rotating branches â€˜by subtreeâ€™ indicates that the branch with the largest total branch length in the descendant subtree was rotated to the same side.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pone.0078122.s006.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Text S1</span> <p><b>Proof that </b><span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span><b> is a positive semidefinite kernel.</b></p> <p>(PDF)</p>  <div class="media" title="media"><a href="pone.0078122.s007.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>The authors thank Chelsea Himsworth for providing valuable feedback on interpreting patterns among RNA viruses with respect to animal reservoirs; Jonathan Carlson for suggesting the use of t-distributed stochastic neighbor embedding to visualize the kernel matrix; and Paul-Michael Agapow for advice on simulating phylogenies.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="pone.0078122-Drake1" /><span class="label" tagx="label" title="label">1</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drake</span><span class="given-names" tagx="given-names" title="given-names">JW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Charlesworth</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Charlesworth</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crow</span><span class="given-names" tagx="given-names" title="given-names">JF</span></span> (<span class="year" tagx="year" title="year">1998</span>) <span class="mixed-article-title" title="mixed-article-title">Rates of spontaneous mutation</span>. <span class="source" tagx="source" title="source">Genetics</span><span class="volume" tagx="volume" title="volume">148</span>: <span class="fpage" tagx="fpage" title="fpage">1667</span>â€"<span class="lpage" tagx="lpage" title="lpage">86</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9560386">9560386</a></span></span></li> <li tag="ref"><a name="pone.0078122-Grenfell1" /><span class="label" tagx="label" title="label">2</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grenfell</span><span class="given-names" tagx="given-names" title="given-names">BT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">OG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gog</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wood</span><span class="given-names" tagx="given-names" title="given-names">JLN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daly</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2004</span>) <span class="mixed-article-title" title="mixed-article-title">Unifying the epidemio-logical and evolutionary dynamics of pathogens</span>. <span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">303</span>: <span class="fpage" tagx="fpage" title="fpage">327</span>â€"<span class="lpage" tagx="lpage" title="lpage">32</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14726583">14726583</a></span></span></li> <li tag="ref"><a name="pone.0078122-Holmes1" /><span class="label" tagx="label" title="label">3</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grenfell</span><span class="given-names" tagx="given-names" title="given-names">BT</span></span> (<span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Discovering the phylodynamics of RNA viruses</span>. <span class="source" tagx="source" title="source">PLoS Com-put Biol</span><span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">e1000505</span>.</span></li> <li tag="ref"><a name="pone.0078122-Korber1" /><span class="label" tagx="label" title="label">4</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Korber</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gaschen</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yusim</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thakallapally</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kesmir</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2001</span>) <span class="mixed-article-title" title="mixed-article-title">Evolutionary and immunological implications of contemporary HIV-1 variation</span>. <span class="source" tagx="source" title="source">Br Med Bull</span><span class="volume" tagx="volume" title="volume">58</span>: <span class="fpage" tagx="fpage" title="fpage">19</span>â€"<span class="lpage" tagx="lpage" title="lpage">42</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11714622">11714622</a></span></span></li> <li tag="ref"><a name="pone.0078122-Sharp1" /><span class="label" tagx="label" title="label">5</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharp</span><span class="given-names" tagx="given-names" title="given-names">PM</span></span> (<span class="year" tagx="year" title="year">2002</span>) <span class="mixed-article-title" title="mixed-article-title">Origins of human virus diversity</span>. <span class="source" tagx="source" title="source">Cell</span><span class="volume" tagx="volume" title="volume">108</span>: <span class="fpage" tagx="fpage" title="fpage">305</span>â€"<span class="lpage" tagx="lpage" title="lpage">12</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11853665">11853665</a></span></span></li> <li tag="ref"><a name="pone.0078122-Nelson1" /><span class="label" tagx="label" title="label">6</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nelson</span><span class="given-names" tagx="given-names" title="given-names">MI</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span> (<span class="year" tagx="year" title="year">2007</span>) <span class="mixed-article-title" title="mixed-article-title">The evolution of epidemic influenza</span>. <span class="source" tagx="source" title="source">Nat Rev Genet</span><span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">196</span>â€"<span class="lpage" tagx="lpage" title="lpage">205</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17262054">17262054</a></span></span></li> <li tag="ref"><a name="pone.0078122-Blum1" /><span class="label" tagx="label" title="label">7</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blum</span><span class="given-names" tagx="given-names" title="given-names">MGB</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">FranÃ§ois</span><span class="given-names" tagx="given-names" title="given-names">O</span></span> (<span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">On statistical tests of phylogenetic tree imbalance: the sackin and other indices revisited</span>. <span class="source" tagx="source" title="source">Math Biosci</span><span class="volume" tagx="volume" title="volume">195</span>: <span class="fpage" tagx="fpage" title="fpage">141</span>â€"<span class="lpage" tagx="lpage" title="lpage">53</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15893336">15893336</a></span></span></li> <li tag="ref"><a name="pone.0078122-Sackin1" /><span class="label" tagx="label" title="label">8</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sackin</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span> (<span class="year" tagx="year" title="year">1972</span>) <span class="mixed-article-title" title="mixed-article-title">â€œGoodâ€� and â€œbadâ€� phenograms</span>. <span class="source" tagx="source" title="source">Sys Zool</span><span class="volume" tagx="volume" title="volume">21</span>: <span class="fpage" tagx="fpage" title="fpage">225</span>â€"<span class="lpage" tagx="lpage" title="lpage">226</span>.</span></li> <li tag="ref"><a name="pone.0078122-Pompei1" /><span class="label" tagx="label" title="label">9</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pompei</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Loreto</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tria</span><span class="given-names" tagx="given-names" title="given-names">F</span></span> (<span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Phylogenetic properties of rna viruses</span>. <span class="source" tagx="source" title="source">PLoS One</span><span class="volume" tagx="volume" title="volume">7</span>: <span class="fpage" tagx="fpage" title="fpage">e44849</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23028645">23028645</a></span></span></li> <li tag="ref"><a name="pone.0078122-Aizerman1" /><span class="label" tagx="label" title="label">10</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aizerman</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Braverman</span><span class="given-names" tagx="given-names" title="given-names">EM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rozoner</span><span class="given-names" tagx="given-names" title="given-names">LI</span></span> (<span class="year" tagx="year" title="year">1964</span>) <span class="mixed-article-title" title="mixed-article-title">Theoretical foundations of 1 the potential function method in pattern recognition learning</span>. <span class="source" tagx="source" title="source">Automation and Remote Control</span><span class="volume" tagx="volume" title="volume">25</span>: <span class="fpage" tagx="fpage" title="fpage">821</span>â€"<span class="lpage" tagx="lpage" title="lpage">837</span>.</span></li> <li tag="ref"><a name="pone.0078122-Vert1" /><span class="label" tagx="label" title="label">11</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vert</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsuda</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SchÃ¶lkopf</span><span class="given-names" tagx="given-names" title="given-names">B</span></span> (<span class="year" tagx="year" title="year">2004</span>) <span class="mixed-article-title" title="mixed-article-title">Kernel methods in computational biology, MIT Press, chapter</span>. <span class="volume" tagx="volume" title="volume">2</span>: <span class="fpage" tagx="fpage" title="fpage">35</span>â€"<span class="lpage" tagx="lpage" title="lpage">70</span>.</span></li> <li tag="ref"><a name="pone.0078122-Agapow1" /><span class="label" tagx="label" title="label">12</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Agapow</span><span class="given-names" tagx="given-names" title="given-names">PM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Purvis</span><span class="given-names" tagx="given-names" title="given-names">A</span></span> (<span class="year" tagx="year" title="year">2002</span>) <span class="mixed-article-title" title="mixed-article-title">Power of eight tree shape statistics to detect nonrandom diversification: a comparison by simulation of two models of cladogenesis</span>. <span class="source" tagx="source" title="source">Syst Biol</span><span class="volume" tagx="volume" title="volume">51</span>: <span class="fpage" tagx="fpage" title="fpage">866</span>â€"<span class="lpage" tagx="lpage" title="lpage">72</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12554452">12554452</a></span></span></li> <li tag="ref"><a name="pone.0078122-Huelsenbeck1" /><span class="label" tagx="label" title="label">13</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huelsenbeck</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bollback</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levine</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span> (<span class="year" tagx="year" title="year">2002</span>) <span class="mixed-article-title" title="mixed-article-title">Inferring the root of a phylogenetic tree</span>. <span class="source" tagx="source" title="source">Syst Biol</span><span class="volume" tagx="volume" title="volume">51</span>: <span class="fpage" tagx="fpage" title="fpage">32</span>â€"<span class="lpage" tagx="lpage" title="lpage">43</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11943091">11943091</a></span></span></li> <li tag="ref"><a name="pone.0078122-Robinson1" /><span class="label" tagx="label" title="label">14</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foulds</span><span class="given-names" tagx="given-names" title="given-names">L</span></span> (<span class="year" tagx="year" title="year">1981</span>) <span class="mixed-article-title" title="mixed-article-title">Comparison of phylogenetic trees</span>. <span class="source" tagx="source" title="source">Math Biosci</span><span class="volume" tagx="volume" title="volume">53</span>: <span class="fpage" tagx="fpage" title="fpage">131</span>â€"<span class="lpage" tagx="lpage" title="lpage">147</span>.</span></li> <li tag="ref"><a name="pone.0078122-Collins1" /><span class="label" tagx="label" title="label">15</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Collins M, Duffy N (2002) Convolution kernels for natural language. In: Dietterich TG, Becker S, Ghahramani Z, editors, Advances in Neural Information Processing Systems 14. Cambridge, MA: MIT Press.</span></li> <li tag="ref"><a name="pone.0078122-Moschitti1" /><span class="label" tagx="label" title="label">16</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Moschitti A (2006) Making tree kernels practical for natural language learning. In: Proceedings of the 11th Conference of the European Chapter of the Association for Computational Linguistics.</span></li> <li tag="ref"><a name="pone.0078122-Hulo1" /><span class="label" tagx="label" title="label">17</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hulo</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Castro</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Masson</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bougueleret</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bairoch</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2011</span>) <span class="mixed-article-title" title="mixed-article-title">ViralZone: a knowledge resource to understand virus diversity</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="volume" tagx="volume" title="volume">39</span>: <span class="fpage" tagx="fpage" title="fpage">D576</span>â€"<span class="lpage" tagx="lpage" title="lpage">82</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20947564">20947564</a></span></span></li> <li tag="ref"><a name="pone.0078122-KosakovskyPond1" /><span class="label" tagx="label" title="label">18</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kosakovsky Pond</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Posada</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stawiski</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chappey</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poon</span><span class="given-names" tagx="given-names" title="given-names">AFY</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">An evo- lutionary model-based algorithm for accurate phylogenetic breakpoint mapping and subtype prediction in HIV-1</span>. <span class="source" tagx="source" title="source">PLoS Comput Biol</span><span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">e1000581</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19956739">19956739</a></span></span></li> <li tag="ref"><a name="pone.0078122-Kuiken1" /><span class="label" tagx="label" title="label">19</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuiken</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yusim</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boykin</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Richardson</span><span class="given-names" tagx="given-names" title="given-names">R</span></span> (<span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">The Los Alamos hepatitis C sequence database</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">21</span>: <span class="fpage" tagx="fpage" title="fpage">379</span>â€"<span class="lpage" tagx="lpage" title="lpage">84</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15377502">15377502</a></span></span></li> <li tag="ref"><a name="pone.0078122-Gotoh1" /><span class="label" tagx="label" title="label">20</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gotoh</span><span class="given-names" tagx="given-names" title="given-names">O</span></span> (<span class="year" tagx="year" title="year">1982</span>) <span class="mixed-article-title" title="mixed-article-title">An improved algorithm for matching biological sequences</span>. <span class="source" tagx="source" title="source">J Mol Biol</span><span class="volume" tagx="volume" title="volume">162</span>: <span class="fpage" tagx="fpage" title="fpage">705</span>â€"<span class="lpage" tagx="lpage" title="lpage">8</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7166760">7166760</a></span></span></li> <li tag="ref"><a name="pone.0078122-KosakovskyPond2" /><span class="label" tagx="label" title="label">21</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kosakovsky Pond</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frost</span><span class="given-names" tagx="given-names" title="given-names">SDW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muse</span><span class="given-names" tagx="given-names" title="given-names">SV</span></span> (<span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">HyPhy: hypothesis testing 1 using phylogenies</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">21</span>: <span class="fpage" tagx="fpage" title="fpage">676</span>â€"<span class="lpage" tagx="lpage" title="lpage">679</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15509596">15509596</a></span></span></li> <li tag="ref"><a name="pone.0078122-Edgar1" /><span class="label" tagx="label" title="label">22</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edgar</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span> (<span class="year" tagx="year" title="year">2004</span>) <span class="mixed-article-title" title="mixed-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</span>. <span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="volume" tagx="volume" title="volume">32</span>: <span class="fpage" tagx="fpage" title="fpage">1792</span>â€"<span class="lpage" tagx="lpage" title="lpage">7</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15034147">15034147</a></span></span></li> <li tag="ref"><a name="pone.0078122-Price1" /><span class="label" tagx="label" title="label">23</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Price</span><span class="given-names" tagx="given-names" title="given-names">MN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dehal</span><span class="given-names" tagx="given-names" title="given-names">PS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arkin</span><span class="given-names" tagx="given-names" title="given-names">AP</span></span> (<span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Fasttree 2â€"approximately maximum-likelihood trees for large alignments</span>. <span class="source" tagx="source" title="source">PLoS One</span><span class="volume" tagx="volume" title="volume">5</span>: <span class="fpage" tagx="fpage" title="fpage">e9490</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20224823">20224823</a></span></span></li> <li tag="ref"><a name="pone.0078122-Smithburn1" /><span class="label" tagx="label" title="label">24</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smithburn</span><span class="given-names" tagx="given-names" title="given-names">KC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hughes</span><span class="given-names" tagx="given-names" title="given-names">TP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burke</span><span class="given-names" tagx="given-names" title="given-names">AV</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paul</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span> (<span class="year" tagx="year" title="year">1940</span>) <span class="mixed-article-title" title="mixed-article-title">A neurotropic virus isolated from the blood of a native of uganda</span>. <span class="source" tagx="source" title="source">Am J Trop Med Hyg</span><span class="volume" tagx="volume" title="volume">20</span>: <span class="fpage" tagx="fpage" title="fpage">471</span>â€"<span class="lpage" tagx="lpage" title="lpage">492</span>.</span></li> <li tag="ref"><a name="pone.0078122-Stamatakis1" /><span class="label" tagx="label" title="label">25</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stamatakis</span><span class="given-names" tagx="given-names" title="given-names">A</span></span> (<span class="year" tagx="year" title="year">2006</span>) <span class="mixed-article-title" title="mixed-article-title">RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">22</span>: <span class="fpage" tagx="fpage" title="fpage">2688</span>â€"<span class="lpage" tagx="lpage" title="lpage">90</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16928733">16928733</a></span></span></li> <li tag="ref"><a name="pone.0078122-Kirkpatrick1" /><span class="label" tagx="label" title="label">26</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kirkpatrick</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Slatkin</span><span class="given-names" tagx="given-names" title="given-names">M</span></span> (<span class="year" tagx="year" title="year">1993</span>) <span class="mixed-article-title" title="mixed-article-title">Searching for evolutionary patterns in the shape of a phylogenetic tree</span>. <span class="source" tagx="source" title="source">Evolution</span><span class="volume" tagx="volume" title="volume">47</span>: <span class="fpage" tagx="fpage" title="fpage">1171</span>â€"<span class="lpage" tagx="lpage" title="lpage">1181</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28564277">28564277</a></span></span></li> <li tag="ref"><a name="pone.0078122-Weston1" /><span class="label" tagx="label" title="label">27</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Weston</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SchÃ¶lkopf</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eskin</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leslie</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Noble</span><span class="given-names" tagx="given-names" title="given-names">W</span></span> (<span class="year" tagx="year" title="year">2003</span>) <span class="mixed-article-title" title="mixed-article-title">Dealing with large diagonals in kernel matrices</span>. <span class="source" tagx="source" title="source">Annals of the Institute of Statistical Mathematics</span><span class="volume" tagx="volume" title="volume">55</span>: <span class="fpage" tagx="fpage" title="fpage">391</span>â€"<span class="lpage" tagx="lpage" title="lpage">408</span>.</span></li> <li tag="ref"><a name="pone.0078122-Cock1" /><span class="label" tagx="label" title="label">28</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cock</span><span class="given-names" tagx="given-names" title="given-names">PJA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antao</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chapman</span><span class="given-names" tagx="given-names" title="given-names">BA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cox</span><span class="given-names" tagx="given-names" title="given-names">CJ</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2009</span>) <span class="mixed-article-title" title="mixed-article-title">Biopython: freely available python tools for computational molecular biology and bioinformatics</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">25</span>: <span class="fpage" tagx="fpage" title="fpage">1422</span>â€"<span class="lpage" tagx="lpage" title="lpage">3</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19304878">19304878</a></span></span></li> <li tag="ref"><a name="pone.0078122-Talevich1" /><span class="label" tagx="label" title="label">29</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Talevich</span><span class="given-names" tagx="given-names" title="given-names">E</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Invergo</span><span class="given-names" tagx="given-names" title="given-names">BM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cock</span><span class="given-names" tagx="given-names" title="given-names">PJA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chapman</span><span class="given-names" tagx="given-names" title="given-names">BA</span></span> (<span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Bio.phylo: a unified toolkit for processing, analyzing and visualizing phylogenetic trees in biopython</span>. <span class="source" tagx="source" title="source">BMC Bioinformatics</span><span class="volume" tagx="volume" title="volume">13</span>: <span class="fpage" tagx="fpage" title="fpage">209</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22909249">22909249</a></span></span></li> <li tag="ref"><a name="pone.0078122-Dalcn1" /><span class="label" tagx="label" title="label">30</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">DalcÃ­n</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Paz</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Storti</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dâ€™ElÃ­a</span><span class="given-names" tagx="given-names" title="given-names">J</span></span> (<span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Mpi for python: Performance improvements and mpi-2 extensions</span>. <span class="source" tagx="source" title="source">Journal of Parallel and Distributed Computing</span><span class="volume" tagx="volume" title="volume">68</span>: <span class="fpage" tagx="fpage" title="fpage">655</span>â€"<span class="lpage" tagx="lpage" title="lpage">662</span>.</span></li> <li tag="ref"><a name="pone.0078122-Karatzoglou1" /><span class="label" tagx="label" title="label">31</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karatzoglou</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smola</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hornik</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zeileis</span><span class="given-names" tagx="given-names" title="given-names">A</span></span> (<span class="year" tagx="year" title="year">2004</span>) <span class="mixed-article-title" title="mixed-article-title">kernlab â€" an S4 package for kernel methods in R. Journal of Statistical Software</span>. <span class="volume" tagx="volume" title="volume">11</span>: <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">20</span>.</span></li> <li tag="ref"><a name="pone.0078122-FitzJohn1" /><span class="label" tagx="label" title="label">32</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">FitzJohn</span><span class="given-names" tagx="given-names" title="given-names">RG</span></span> (<span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Diversitree: comparative phylogenetic analyses of diversification 1 in r</span>. <span class="source" tagx="source" title="source">Methods in Ecology and Evolution</span><span class="volume" tagx="volume" title="volume">3</span>: <span class="fpage" tagx="fpage" title="fpage">1084</span>â€"<span class="lpage" tagx="lpage" title="lpage">1092</span>.</span></li> <li tag="ref"><a name="pone.0078122-FitzJohn2" /><span class="label" tagx="label" title="label">33</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">FitzJohn</span><span class="given-names" tagx="given-names" title="given-names">RG</span></span> (<span class="year" tagx="year" title="year">2010</span>) <span class="mixed-article-title" title="mixed-article-title">Quantitative traits and diversification</span>. <span class="source" tagx="source" title="source">Systematic Biology</span><span class="volume" tagx="volume" title="volume">59</span>: <span class="fpage" tagx="fpage" title="fpage">619</span>â€"<span class="lpage" tagx="lpage" title="lpage">633</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20884813">20884813</a></span></span></li> <li tag="ref"><a name="pone.0078122-Purvis1" /><span class="label" tagx="label" title="label">34</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Purvis</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Katzourakis</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Agapow</span><span class="given-names" tagx="given-names" title="given-names">PM</span></span> (<span class="year" tagx="year" title="year">2002</span>) <span class="mixed-article-title" title="mixed-article-title">Evaluating phylogenetic tree shape: two modifications to fusco &amp;amp; cronkâ€™s method</span>. <span class="source" tagx="source" title="source">J Theor Biol</span><span class="volume" tagx="volume" title="volume">214</span>: <span class="fpage" tagx="fpage" title="fpage">99</span>â€"<span class="lpage" tagx="lpage" title="lpage">103</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11786035">11786035</a></span></span></li> <li tag="ref"><a name="pone.0078122-Schlkopf1" /><span class="label" tagx="label" title="label">35</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">SchÃ¶lkopf</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smola</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Williamson</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bartlett</span><span class="given-names" tagx="given-names" title="given-names">PL</span></span> (<span class="year" tagx="year" title="year">2000</span>) <span class="mixed-article-title" title="mixed-article-title">New support vector algorithms</span>. <span class="source" tagx="source" title="source">Neural computation</span><span class="volume" tagx="volume" title="volume">12</span>: <span class="fpage" tagx="fpage" title="fpage">1207</span>â€"<span class="lpage" tagx="lpage" title="lpage">1245</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10905814">10905814</a></span></span></li> <li tag="ref"><a name="pone.0078122-Sing1" /><span class="label" tagx="label" title="label">36</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sing</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sander</span><span class="given-names" tagx="given-names" title="given-names">O</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beerenwinkel</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lengauer</span><span class="given-names" tagx="given-names" title="given-names">T</span></span> (<span class="year" tagx="year" title="year">2005</span>) <span class="mixed-article-title" title="mixed-article-title">Rocr: visualizing classifier performance in r</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">21</span>: <span class="fpage" tagx="fpage" title="fpage">3940</span>â€"<span class="lpage" tagx="lpage" title="lpage">3941</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16096348">16096348</a></span></span></li> <li tag="ref"><a name="pone.0078122-Colless1" /><span class="label" tagx="label" title="label">37</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colless</span><span class="given-names" tagx="given-names" title="given-names">DH</span></span> (<span class="year" tagx="year" title="year">1982</span>) <span class="mixed-article-title" title="mixed-article-title">Review of â€œphylogenetics: the theory and practice of phylogenetic system- aticsâ€�</span>. <span class="source" tagx="source" title="source">Sys Zool</span><span class="volume" tagx="volume" title="volume">31</span>: <span class="fpage" tagx="fpage" title="fpage">100</span>â€"<span class="lpage" tagx="lpage" title="lpage">104</span>.</span></li> <li tag="ref"><a name="pone.0078122-Lin1" /><span class="label" tagx="label" title="label">38</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">LIK</span></span> (<span class="year" tagx="year" title="year">1989</span>) <span class="mixed-article-title" title="mixed-article-title">A concordance correlation coefficient to evaluate reproducibility</span>. <span class="source" tagx="source" title="source">Biometrics</span><span class="volume" tagx="volume" title="volume">45</span>: <span class="fpage" tagx="fpage" title="fpage">255</span>â€"<span class="lpage" tagx="lpage" title="lpage">268</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2720055">2720055</a></span></span></li> <li tag="ref"><a name="pone.0078122-Lawrence1" /><span class="label" tagx="label" title="label">39</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Lawrence I, Lin K (1989) A concordance correlation coefficient to evaluate reproducibility. Biometrics : 255â€"268.</span></li> <li tag="ref"><a name="pone.0078122-vanderMaaten1" /><span class="label" tagx="label" title="label">40</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van der Maaten</span><span class="given-names" tagx="given-names" title="given-names">L</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hinton</span><span class="given-names" tagx="given-names" title="given-names">G</span></span> (<span class="year" tagx="year" title="year">2008</span>) <span class="mixed-article-title" title="mixed-article-title">Visualizing data using t-sne</span>. <span class="source" tagx="source" title="source">J Mach Learn Res</span><span class="volume" tagx="volume" title="volume">9</span>: <span class="fpage" tagx="fpage" title="fpage">2579</span>â€"<span class="lpage" tagx="lpage" title="lpage">2605</span>.</span></li> <li tag="ref"><a name="pone.0078122-Williams1" /><span class="label" tagx="label" title="label">41</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Williams</span><span class="given-names" tagx="given-names" title="given-names">CF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Klinzman</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yamashita</span><span class="given-names" tagx="given-names" title="given-names">TE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Polgreen</span><span class="given-names" tagx="given-names" title="given-names">PM</span></span>, <span class="etal" title="etal"><i>et al.</i></span> (<span class="year" tagx="year" title="year">2004</span>) <span class="mixed-article-title" title="mixed-article-title">Persistent GB virus C infection and survival in HIV-infected men</span>. <span class="source" tagx="source" title="source">N Engl J Med</span><span class="volume" tagx="volume" title="volume">350</span>: <span class="fpage" tagx="fpage" title="fpage">981</span>â€"<span class="lpage" tagx="lpage" title="lpage">90</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14999110">14999110</a></span></span></li> <li tag="ref"><a name="pone.0078122-Frost1" /><span class="label" tagx="label" title="label">42</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frost</span><span class="given-names" tagx="given-names" title="given-names">SDW</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Volz</span><span class="given-names" tagx="given-names" title="given-names">EM</span></span> (<span class="year" tagx="year" title="year">2013</span>) <span class="mixed-article-title" title="mixed-article-title">Modelling tree shape and structure in viral phylodynamics</span>. <span class="source" tagx="source" title="source">Philos Trans R Soc Lond B Biol Sci</span><span class="volume" tagx="volume" title="volume">368</span>: <span class="fpage" tagx="fpage" title="fpage">20120208</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23382430">23382430</a></span></span></li> <li tag="ref"><a name="pone.0078122-Ratmann1" /><span class="label" tagx="label" title="label">43</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ratmann</span><span class="given-names" tagx="given-names" title="given-names">O</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Donker</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meijer</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fraser</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koelle</span><span class="given-names" tagx="given-names" title="given-names">K</span></span> (<span class="year" tagx="year" title="year">2012</span>) <span class="mixed-article-title" title="mixed-article-title">Ph1 ylodynamic inference and model assessment with approximate bayesian computation: influenza as a case study</span>. <span class="source" tagx="source" title="source">PLoS Comput Biol</span><span class="volume" tagx="volume" title="volume">8</span>: <span class="fpage" tagx="fpage" title="fpage">e1002835</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23300420">23300420</a></span></span></li> </ul> </div> </div>  </body></html>