<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="iso-abbrev" title="iso-abbrev">Trans Indian Natl. Acad. Eng.</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Transactions of the Indian National Academy of Engineering</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">2662-5415</span><span class="issn-epub" tagx="issn" title="issn-epub">2662-5423</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer Singapore</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Singapore</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7306100">7306100</a></span><span class="publisher-id" title="publisher-id">132</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1007/s41403-020-00132-8">10.1007/s41403-020-00132-8</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Original Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
En route to Peptide Therapeutics for COVID 19: Harnessing Potential Antigenic Mimicry Between Viral and Human Proteins</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Madhavan</span><span class="given-names" tagx="given-names" title="given-names">Maya</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-5929-9065</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mustafa</span><span class="given-names" tagx="given-names" title="given-names">Sabeena</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">msabeena@gmail.com</span></span><a href="#Aff2">2</a></span><span class="citation_author_institution" id="Aff1">[1], </span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.416641.0</span><span class="institution-id" title="institution-id">0000 0004 0607 2419</span><span class="institution" title="institution">Department of Biostatistics and Bioinformatics, King Abdullah International Medical Research Center (KAIMRC), </span><span class="institution" title="institution">Ministry of National Guard Health Affairs (MNGHA), </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-6-6</span></span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">5</span><span class="history" title="history"><span class="received" title="received">received: 2020-4-28</span><span class="rev-recd" title="rev-recd">rev-recd: 2020-6-5</span><span class="accepted" title="accepted">accepted: 2020-6-12</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p">This article is made available via the PMC Open Access Subset for unrestricted research re-use and secondary analysis in any form or by any means with acknowledgement of the original source. These permissions are granted for the duration of the World Health Organization (WHO) declaration of COVID-19 as a global pandemic.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p id="Par1">Molecular mimicry is a general strategy used by pathogens to infect the host cells. The emergence of SARS-CoV-2 virus has resulted in more than 6,700,000 infections and 390,000 deaths worldwide. Coronavirus disease (COVID-19) is an infectious disease caused by this virus. In this project concept, we aim to focus on the peptide-protein interaction analysis using two important drug targets in SARS-CoV-2 such as spike (S) protein and nucleocapsid (N) protein. These proteins play an important role in the virus entry and encapsidation of the viral particles. Motifs or functional regions in these two proteins must be sharing sequence homology with human protein (ACE2) which may be involved in the binding mechanism. The results will show a set of motif regions which can disrupt the viral infection. Once we identify these sets of antigenic determinant regions, antibody binding activity studies can be performed by in vitro methods. Our results from this study may suggest the existence of antigenic mimicry between SARS-CoV-2 and host proteins. The hit peptide components will have therapeutic applications to be developed into a wide variety of medicinal formulations against SARS-CoV-2 such as vaccine, intranasal and inhalation formulations. Also, the choice of conserved regions will lead to development of cross protective therapeutics against wide range of coronaviruses.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
SARS-CoV-2</div> <div class="kwd" title="kwd">
COVID-19</div> <div class="kwd" title="kwd">
Spike</div> <div class="kwd" title="kwd">
Nucleocapsid</div> <div class="kwd" title="kwd">
Immunogenicity</div> <div class="kwd" title="kwd">
Peptide-protein interaction</div> </div> </div> </div> <div class="body" title="body"> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p id="Par2">There is no approved vaccine or therapeutic option developed yet for COVID-19. So, the availability of various therapeutic options against COVID-19 will provide benefits for the control of the present pandemic situation as well as a control for reemergence of the virus in future. The effort of vaccine development is an important research area. Several strategies such as small molecules, repurposing of FDA approved compounds, and different classes of vaccines candidates are being developed in the labs and some are even under trials.</p> <p id="Par3">Peptide-based vaccines are designed to elicit immunity against particular pathogens by selectively stimulating antigens specific for B and T cells. Peptide vaccines have a whole lot of advantages though they suffer from certain challenges such as low immunogenicity, susceptibility to enzymatic digestion etc. (Skwarczynski and Toth <a href="#CR17">2016</a>). Advanced bioinformatics tools and databases help to speed up and design peptide-based vaccines for SARS-CoV-2 where peptides could be used as ligands (Abdelmageed et al. <a href="#CR1">2020</a>). The viable approach in designing peptide vaccines should be to identify peptide epitopes on the immunogens and to use synthetic versions of these peptides. Since these vaccines are totally synthetic, they would not carry the risk of reversion or of complete inactivation, and in principle; epitopes could be selected to avoid components that give rise to unwanted side effects (<a href="https://www.who.int/biologicals/publications/trs/areas/vaccines/peptide/WHO_TRS_889_A1.pdf?ua=1">https://www.who.int/biologicals/publications/trs/areas/vaccines/peptide/WHO_TRS_889_A1.pdf?ua=1</a>). Also, peptide-based vaccines do not need in vitro culture making them biologically safe, and their selectivity allows accurate activation of immune responses (Dudek et al. <a href="#CR7">2010</a>). Studies shows that effective vaccines can be developed using T cell and B cell epitopes comprising the immunome, which constitutes the immune system (Groot et al. <a href="#CR6">2008</a>; Tahir ul Qamar et al. <a href="#CR20">2019</a>). Epitopes are short amino acid sequences of a protein which can induce an immune response and epitope finding is quite expensive and laborious. The use ofÂ in silicoÂ methods are an alternative and effective strategy due to the importance in biomedical applications such as vaccine design, disease diagnostics, biomarker studies etc. (Bahrami et al. <a href="#CR3">2019</a>; Backert and Kohlbacher <a href="#CR2">2015</a>). There are many peptide vaccines in development, such as vaccine for AIDS by the virus human immunodeficiency virus (HIV) (Liu et al. <a href="#CR15">2007</a>), malaria (Epstein et al. <a href="#CR8">2007</a>), anthrax (Oscherwitz et al. <a href="#CR16">2010</a>), human papilloma virus (HPV) (Solares et al. <a href="#CR18">2011</a>). Our hypothesis is based on the fact that, epitope region of the drug targets which share the sequence identity and similarity to human proteins may be involved in their binding and fusion to human cells (Hwa et al. <a href="#CR11">2008</a>).</p> </div> <div class="selectionofsandnproteindrugtargetsofsars-cov-2" title="sec"> <div class="title" tagx="title" title="title">
Selection of S and N Protein Drug Targets of SARS-CoV-2</div> <div class="keypropertiesoftwodrugtargets,sandnproteins" title="sec"> <div class="title" tagx="title" title="title">
Key Properties of Two Drug Targets, S and N Proteins</div> <p id="Par4">SARS-Co-V-2 possesses many druggable targets. Coronavirus entry into host cell is mediated by the transmembrane Spike (S) glycoprotein that forms homotrimers protruding from the viral surface (Tortorici and Veesler <a href="#CR21">2019</a>). S comprises two functional subunits responsible for binding toÂ the host cell receptor (S<sub>1</sub>Â subunit) and fusion of the viral andÂ cellular membranes (S<sub>2</sub>Â subunit).Â SARS-CoV-2 and several SARS-related coronaviruses (SARS-CoV) interact directly with angiotensin-converting enzyme 2 (ACE2) via SÂ to enter target cells (Song et al. <a href="#CR19">2018</a>; Kirchdoerfer et al. <a href="#CR12">2018</a>; Ge et al. <a href="#CR9">2013</a>). As the coronavirus S glycoprotein is surface-exposed and mediates entry into host cells, it is the main target of neutralizing antibodies (Abs) upon infection and the focus of therapeutic andÂ vaccine design. After binding of RBD in S1 subunit of S protein on the virion to the ACE2 receptor on the target cell, the heptad repeat 1 (HR1) and 2 (HR2) domains in its S2 subunit interact with each other to form a six-helix bundle (6-HB) fusion core, bringing viral and cellular membranes into close proximity for fusion and infection (Bosch et al. <a href="#CR4">2004</a>).</p> <p id="Par5">The nucleocapsid (N) protein is one of the major structural proteins of the virus and is involved in the transcription and replication of viral RNA, packaging of the encapsidated genome into virions (Chang et al. <a href="#CR5">2006</a>; Hurst et al. <a href="#CR10">2009</a>) and interference with cell cycle processes of host cells. Moreover, in many coronaviruses, including SARS -CoV-2, the N protein has high immunogenic activity and is abundantly expressed during infection. Thus, both S and N proteins may be potential targets for design of peptide vaccines against COVID-19.</p> </div> <div class="sars-covandsars-cov-2sequencehomology" title="sec"> <div class="title" tagx="title" title="title">
SARS-CoV and SARS-CoV-2 Sequence Homology</div> <p id="Par6">SARS-CoV and SARS-CoV-2 have 89.8% sequence identity in their S protein S2 subunits, which mediate the membrane fusion process, and both of their S1 subunits utilize human angiotensin-converting enzyme 2 (hACE2) as the receptor to infect human cells (Zhou et al. <a href="#CR24">2020</a>). Also, the ACE2-binding affinity of the receptor-binding domain (RBD) in S1 subunit of S protein of SARS-CoV-2 is 10- to 20-fold higher than that of SARS-CoV which may contribute to the higher infectivity and transmissibility of SARS-CoV-2 compared to SARS-CoV (Wrapp et al. <a href="#CR22">2020</a>). Sequence alignment studies also show that S2 subunits of SARS-CoV-2 and SARS-CoV share consensus regions with 92.6% and 100% in HR1 and HR2 domains, respectively (Xia et al. <a href="#CR23">2020</a>). The presence of large amount of sequence identity between SARS-CoV and SARS-CoV-2 enables designing of vaccines which are not only cross-protective across a wide spectrum of betacoronavirusesÂ but also relatively resistant to the ongoing virus evolution.</p> </div> </div> <div class="methodology" title="sec"> <div class="title" tagx="title" title="title">
Methodology</div> <div class="identificationandretrievalofsandnproteindrugtargetsfromrepositories" title="sec"> <div class="title" tagx="title" title="title">
Identification and Retrieval of S And N Protein Drug Targets from Repositories</div> <p id="Par7">We searched the online repositories namely Genbank, NCBI protein and UniProtKb to retrieve the suitable sequence entries of the drug targets. The search resulted in many sequence entries and we selected two ids for further analysis as shown in Table <a href="#Tab1">1</a>. <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">Table 1</span> </div></p><p>Spike protein and nucleocapsid protein sequences information from repositories</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left">Sl. no</th> <th align="left">Sequence</th> <th align="left">Sequence repository and corresponding id</th> <th align="left">Length of residues (aa)</th> </tr> </thead> <tbody> <tr> <td align="left">1</td> <td align="left">Spike</td> <td align="left"> <p>Genbank-NC_045512.2,</p> <p>NCBI protein: QHR63290.2, QJS57231.1</p> <p>UniProtKB-P0DTC2</p> </td> <td char="." align="char">1273</td> </tr> <tr> <td align="left">2</td> <td align="left">Nucleocapsid</td> <td align="left"> <p>Genbank-NC_004718.3,</p> <p>NCBI protein: BCF79921.1, BCF79911.1</p> <p>UniProtKB- P0DTC9</p> </td> <td char="." align="char">419</td> </tr> </tbody> </table>  <p /> <p id="Par8">As the first step in our analysis, S and N protein sequences will be analyzed to identify their immunogenic regions. Algorithms which can predict immunogenicity, secondary structure analysis, protein topology features and hydrophobicity will be studied to design peptides for further testing. Immunogenic viral peptides could be identified based on the algorithm developed by Kolaskar and Tongaonkar. This method makes use of the physicochemical properties of amino acid residues and its frequencies of occurrence in experimentally identified epitopes which are developed to identify antigenic determinants in proteins (Kolaskar and Tongaonkar <a href="#CR13">1990</a>). These epitopes are the short residue set of peptides with antigenicity. IEDB (<a href="https://tools.iedb.org">https://tools.iedb.org</a>) is one of the tools using this algorithm, which is mainly used to determine the antigenic sites with a default threshold value of 1.024. The prediction algorithm works as follows based on the paper published by Kolaskar and Tongaonkar in 1990.The algorithm is as follows. </p><ol> <li> <p id="Par9">Calculate the average propensity values (Ap) for each peptides from N terminus to C terminus of the protein and assign the result to the fourth (iâ€‰+â€‰3) residue in the segment.</p> </li> <li> <p id="Par10">The average antigenic propensity of the protein will be determined</p> </li> <li> <p id="Par11">If the Ap of the protein is above 1.0 then all residues are potentially antigenic.</p> </li> <li> <p id="Par12">If the average for the protein is below 1.0 then all residues having above the average for the protein are potentially antigenic.</p> </li> <li> <p id="Par13">To select the antigenic determinant as a condition, six consecutive residues must satisfy step 3</p> </li> </ol> <p /> <p id="Par14">The above step will help to identify the region of viral proteins (S and N) that are homologous to human protein which will be shortlisted based on the score. The peptides which display high immunogenicity, location on the protein surface and low hydrophobicity are to be selected, following which these peptides will be synthesized and tested by ELISA for binding to antibodies present in the convalescent plasma of COVID 19 patients. This is in line with the theory that the simplest and most direct approach to combating SARS-CoV-2 during the outbreak would be to use plasma from the convalescent patients (Kruse <a href="#CR14">2020</a>). Patients who have been diagnosed with the disease, ascertained by RT-PCR and who are in their convalescent stage (&amp;gt;â€‰20Â days of incubation) should be selected for the study. The peptides which show good amount of binding and with negligible auto immune response can be selected for the design of peptide vaccines. Table <a href="#Tab2">2</a> shows the timeline, major resources and the milestones of the project on a quarterly basis. The detailed methodology is represented in the flowchart (Fig.Â <a href="#Fig1">1</a>). <div class="table-wrap_UNKNOWN" id="Tab2"><span class="label" tagx="label" title="label">Table 2</span> </div></p><p>Timeline, major resources and the milestones of the project on a quarterly basis</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left" /> <th align="left">Q1</th> <th align="left">Q2</th> <th align="left">Q3</th> </tr> </thead> <tbody> <tr> <td align="left">Timeline</td> <td align="left">Literature search, retrieval of S and N protein sequences from repositories. Query sequences will be analyzed to find immunogenic regions using peptide-protein interaction studies</td> <td align="left">Identify the regions of S and N which are homologous to human protein. A list of peptides with high immunogenicity, location on the protein surface and low hydrophobicity are to be selected</td> <td align="left"> <p>Peptide synthesis and wetlab validation using sera from patients. Evaluate the antibody formation in response to SARS-CoV-2 infection</p> <p>ELISA test and evaluation</p> </td> </tr> <tr> <td align="left">Resources</td> <td align="left">PUBMED, GENBANK, UniProtKb, PDB, Piper (Schrodinger)</td> <td align="left">NCBI BLAST, MSA, IEDB analysis resource database, NetMHC server, EpiJen, MHCpred, EpiSearch, PepSurf, BIMAS, Pep3D</td> <td align="left">Peptide Synthesizer, RT-PCR, ELISA system</td> </tr> </tbody> </table>  <p /> </div> </div> <div class="summaryandconclusion" title="sec"> <div class="title" tagx="title" title="title">
Summary and Conclusion</div> <p id="Par15">Our results may suggest the molecular mimicry between SARS-CoV-2 drug target protein sequences (spike and nucleocapsid) and human proteins. Our study will use immunoinformatics tools to identify the possible epitopes of spike and nucleocapsid proteins of SARS-CoV-2. These epitopes can be the ideal candidates to formulate the initial component to develop a therapy against COVID-19 either in the form of peptide and sera based vaccine, based on the selected antigenic determinants. However, our analysis will require several in vitro and in vivo validations before formulating the therapeutic products to resist COVIDâ€�19. Antibody binding activity of these motifs will be performed in vitro to evaluate the neutralization activity against SARS-CoV-2 infection or participate in the immunopathogenesis induced by SARS-CoV-2. Our studies on the mimicry motifs of S and N protein, may provide approaches to inhibit or disrupt the infection of SARS-CoV-2. The peptides selected from the study after testing for toxicity in vivo, can have wide range of applications- be made into vaccines or intranasal and inhalation formulations. Also, since these peptides are selected on the basis of their binding specificity to regions of the viral proteins that are highly conserved across various coronaviruses, the execution of such therapeutic approaches could lead to cross-protective antibody responses against future coronavirus epidemics and pandemics and help us be better prepared for the next novel coronavirus outbreak. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig.Â 1.</span></a></div>   <p>The detailed methodology is represented in the flowchart below</p>   </div> <p /> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisher's Note</b></p> <p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Abdelmageed MI, Abdelmoneim AH, Mustafa MI, et al (2020) Design of multi epitope-based peptide vaccine against E protein of human COVID-19: an immunoinformatics approach. bioRxiv 2020.02.04.934232. 10.1101/2020.02.04.934232</span></li> <li tag="ref"><a name="CR2" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Backert</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kohlbacher</span><span class="given-names" tagx="given-names" title="given-names">O</span></span></span><span class="mixed-article-title" title="mixed-article-title">Immunoinformatics and epitope prediction in the age of genomic medicine</span><span class="source" tagx="source" title="source">Genome Med</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">119</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s13073-015-0245-0">10.1186/s13073-015-0245-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26589500">26589500</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bahrami</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Payandeh</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khalili</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Immunoinformatics. In silico approaches and computational design of a multi-epitope</span><span class="source" tagx="source" title="source">Immun Protein Int Rev Immunol</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">38</span><span class="fpage" tagx="fpage" title="fpage">307</span><span class="lpage" tagx="lpage" title="lpage">322</span><span class="pub-id"><a href="https://dx.doi.org/10.1080/08830185.2019.1657426">10.1080/08830185.2019.1657426</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bosch</span><span class="given-names" tagx="given-names" title="given-names">BJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martina</span><span class="given-names" tagx="given-names" title="given-names">BEE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Van Der Zee</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Severe acute respiratory syndrome coronavirus (SARS-CoV) infection inhibition using spike protein heptad repeat-derived peptides</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci USA</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">101</span><span class="fpage" tagx="fpage" title="fpage">8455</span><span class="lpage" tagx="lpage" title="lpage">8460</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0400576101">10.1073/pnas.0400576101</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15150417">15150417</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chang</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sue</span><span class="given-names" tagx="given-names" title="given-names">S-C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Modular organization of SARS coronavirus nucleocapsid protein</span><span class="source" tagx="source" title="source">J Biomed Sci</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">59</span><span class="lpage" tagx="lpage" title="lpage">72</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s11373-005-9035-9">10.1007/s11373-005-9035-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16228284">16228284</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Groot</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moise</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McMurry</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">W</span></span></span><span class="mixed-article-title" title="mixed-article-title">Epitope-based immunome-derived vaccines: a strategy for improved design and safety</span><span class="source" tagx="source" title="source">Clin Appl Immun</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">2</span><span class="fpage" tagx="fpage" title="fpage">39</span><span class="lpage" tagx="lpage" title="lpage">69</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/978-0-387-79208-8_3">10.1007/978-0-387-79208-8_3</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dudek</span><span class="given-names" tagx="given-names" title="given-names">NL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Perlmutter</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aguilar</span><span class="given-names" tagx="given-names" title="given-names">M-I</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Epitope discovery and their use in peptide based vaccines</span><span class="source" tagx="source" title="source">Curr Pharm Des</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">3149</span><span class="lpage" tagx="lpage" title="lpage">3157</span><span class="pub-id"><a href="https://dx.doi.org/10.2174/138161210793292447">10.2174/138161210793292447</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20687873">20687873</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Epstein</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giersing</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mullen</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Malaria vaccines: are we getting closer?</span><span class="source" tagx="source" title="source">Curr Opin Mol Ther</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">12</span><span class="lpage" tagx="lpage" title="lpage">24</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17330398">17330398</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ge</span><span class="given-names" tagx="given-names" title="given-names">X-Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">J-L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">X-L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Isolation and characterization of a bat SARS-like coronavirus that uses the ACE2 receptor</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">503</span><span class="fpage" tagx="fpage" title="fpage">535</span><span class="lpage" tagx="lpage" title="lpage">538</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature12711">10.1038/nature12711</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24172901">24172901</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hurst</span><span class="given-names" tagx="given-names" title="given-names">KR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koetzner</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Masters</span><span class="given-names" tagx="given-names" title="given-names">PS</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of in vivo-interacting domains of the murine coronavirus nucleocapsid protein</span><span class="source" tagx="source" title="source">J Virol</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">83</span><span class="fpage" tagx="fpage" title="fpage">7221</span><span class="lpage" tagx="lpage" title="lpage">7234</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.00440-09">10.1128/JVI.00440-09</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19420077">19420077</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hwa</span><span class="given-names" tagx="given-names" title="given-names">K-Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lin</span><span class="given-names" tagx="given-names" title="given-names">WM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hou</span><span class="given-names" tagx="given-names" title="given-names">Y-I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yeh</span><span class="given-names" tagx="given-names" title="given-names">T-M</span></span></span><span class="mixed-article-title" title="mixed-article-title">Peptide mimicrying between SARS coronavirus spike protein and human proteins reacts with SARS patient serum</span><span class="source" tagx="source" title="source">J Biomed Biotechnol</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">2008</span><span class="fpage" tagx="fpage" title="fpage">326464</span><span class="pub-id"><a href="https://dx.doi.org/10.1155/2008/326464">10.1155/2008/326464</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18320019">18320019</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kirchdoerfer</span><span class="given-names" tagx="given-names" title="given-names">RN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pallesen</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Stabilized coronavirus spikes are resistant to conformational changes induced by receptor recognition or proteolysis</span><span class="source" tagx="source" title="source">Sci Rep</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">15701</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41598-018-34171-7">10.1038/s41598-018-34171-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30356097">30356097</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kolaskar</span><span class="given-names" tagx="given-names" title="given-names">AS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tongaonkar</span><span class="given-names" tagx="given-names" title="given-names">PC</span></span></span><span class="mixed-article-title" title="mixed-article-title">A semi-empirical method for prediction of antigenic determinants on protein antigens</span><span class="source" tagx="source" title="source">FEBS Lett</span><span class="year" tagx="year" title="year">1990</span><span class="volume" tagx="volume" title="volume">276</span><span class="fpage" tagx="fpage" title="fpage">172</span><span class="lpage" tagx="lpage" title="lpage">174</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0014-5793(90)80535-q">10.1016/0014-5793(90)80535-q</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1702393">1702393</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kruse</span><span class="given-names" tagx="given-names" title="given-names">RL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Therapeutic strategies in an outbreak scenario to treat the novel coronavirus originating in Wuhan</span><span class="source" tagx="source" title="source">China. F1000Research</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">72</span><span class="pub-id"><a href="https://dx.doi.org/10.12688/f1000research.22211.2">10.12688/f1000research.22211.2</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32117569">32117569</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McNevin</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Evolution of human immunodeficiency virus type 1 cytotoxic T-lymphocyte epitopes: fitness-balanced escape</span><span class="source" tagx="source" title="source">J Virol</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">81</span><span class="fpage" tagx="fpage" title="fpage">12179</span><span class="lpage" tagx="lpage" title="lpage">12188</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.01277-07">10.1128/JVI.01277-07</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17728222">17728222</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Oscherwitz</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cease</span><span class="given-names" tagx="given-names" title="given-names">KB</span></span></span><span class="mixed-article-title" title="mixed-article-title">A synthetic peptide vaccine directed against the 2ÃŸ2-2ÃŸ3 loop of domain 2 of protective antigen protects rabbits from inhalation anthrax</span><span class="source" tagx="source" title="source">J Immunol</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">185</span><span class="fpage" tagx="fpage" title="fpage">3661</span><span class="lpage" tagx="lpage" title="lpage">3668</span><span class="pub-id"><a href="https://dx.doi.org/10.4049/jimmunol.1001749">10.4049/jimmunol.1001749</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20696862">20696862</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Skwarczynski</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Toth</span><span class="given-names" tagx="given-names" title="given-names">I</span></span></span><span class="mixed-article-title" title="mixed-article-title">Peptide-based synthetic vaccines</span><span class="source" tagx="source" title="source">Chem Sci</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">842</span><span class="lpage" tagx="lpage" title="lpage">854</span><span class="pub-id"><a href="https://dx.doi.org/10.1039/c5sc03892h">10.1039/c5sc03892h</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28791117">28791117</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Solares</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Baladron</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ramos</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Safety and immunogenicity of a human papillomavirus peptide vaccine (CIGB-228) in Women with high-grade cervical intraepithelial neoplasia: first-in-human proof-of-concept trial</span><span class="source" tagx="source" title="source">ISRN Obstet Gynecol</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">2011</span><span class="fpage" tagx="fpage" title="fpage">292951</span><span class="pub-id"><a href="https://dx.doi.org/10.5402/2011/292951">10.5402/2011/292951</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21748025">21748025</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Song</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gui</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">Cryo-EM structure of the SARS coronavirus spike glycoprotein in complex with its host cell receptor ACE2</span><span class="source" tagx="source" title="source">PLoS Pathog</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">e1007236</span><span class="lpage" tagx="lpage" title="lpage">e1007236</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1007236">10.1371/journal.ppat.1007236</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30102747">30102747</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tahir ul Qamar</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saleem</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ashfaq</span><span class="given-names" tagx="given-names" title="given-names">UA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Epitope-based peptide vaccine design and target site depiction against Middle East Respiratory Syndrome Coronavirus: an immune-informatics study</span><span class="source" tagx="source" title="source">J Transl Med</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">17</span><span class="fpage" tagx="fpage" title="fpage">362</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12967-019-2116-8">10.1186/s12967-019-2116-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31703698">31703698</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tortorici</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Veesler</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="mixed-article-title" title="mixed-article-title">Structural insights into coronavirus entry</span><span class="source" tagx="source" title="source">Adv Virus Res</span><span class="year" tagx="year" title="year">2019</span><span class="volume" tagx="volume" title="volume">105</span><span class="fpage" tagx="fpage" title="fpage">93</span><span class="lpage" tagx="lpage" title="lpage">116</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/bs.aivir.2019.08.002">10.1016/bs.aivir.2019.08.002</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/31522710">31522710</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wrapp</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Corbett</span><span class="given-names" tagx="given-names" title="given-names">KS</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation</span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">367</span><span class="fpage" tagx="fpage" title="fpage">1260</span><span class="lpage" tagx="lpage" title="lpage">1263</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.abb2507">10.1126/science.abb2507</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32075877">32075877</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Inhibition of SARS-CoV-2 (previously 2019-nCoV) infection by a highly potent pan-coronavirus fusion inhibitor targeting its spike protein that harbors a high capacity to mediate membrane fusion</span><span class="source" tagx="source" title="source">Cell Res</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">30</span><span class="fpage" tagx="fpage" title="fpage">343</span><span class="lpage" tagx="lpage" title="lpage">355</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41422-020-0305-x">10.1038/s41422-020-0305-x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32231345">32231345</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yang</span><span class="given-names" tagx="given-names" title="given-names">X-L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">X-G</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2020</span><span class="volume" tagx="volume" title="volume">579</span><span class="fpage" tagx="fpage" title="fpage">270</span><span class="lpage" tagx="lpage" title="lpage">273</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41586-020-2012-7">10.1038/s41586-020-2012-7</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32015507">32015507</a></span></span></li> </ul> </div> </div>  </body></html>