<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2018/04/07/296996.atom" />
<meta name="HW.pisa" content="biorxiv;296996v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The transcriptional and translational landscape of equine torovirus" />
<meta name="DC.Identifier" content="10.1101/296996" />
<meta name="DC.Date" content="2018-04-07" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2018, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="The genus Torovirus (subfamily Torovirinae , family Coronaviridae , order Nidovirales ) encompasses a range of species that infect domestic ungulates including cattle, sheep, goats, pigs and horses, causing an acute self-limiting gastroenteritis. Using the prototype species equine torovirus (EToV) we performed parallel RNA sequencing (RNA-seq) and ribosome profiling (Ribo-seq) to analyse the relative expression levels of the known torovirus proteins and transcripts, chimaeric sequences produced via discontinuous RNA synthesis (a characteristic of the nidovirus replication cycle) and changes in host transcription and translation as a result of EToV infection. RNA sequencing confirmed that EToV utilises a unique combination of discontinuous and non-discontinuous RNA synthesis to produce its subgenomic RNAs; indeed, we identified transcripts arising from both mechanisms that would result in sgRNAs encoding the nucleocapsid. Our ribosome profiling analysis revealed that ribosomes efficiently translate two novel CUG-initiated ORFs, located within the so-called 5’ UTR. We have termed the resulting proteins U1 and U2. Comparative genomic analysis confirmed that these ORFs are conserved across all available torovirus sequences and the inferred amino acid sequences are subject to purifying selection, indicating that U1 and U2 are functionally relevant. This study provides the first high-resolution analysis of transcription and translation in this neglected group of livestock pathogens.

Importance Toroviruses infect cattle, goats, pigs and horses worldwide and can cause gastrointestinal disease. There is no treatment or vaccine and their ability to spill over into humans has not been assessed. These viruses are related to important human pathogens including severe acute respiratory syndrome (SARS) coronavirus and they share some common features, however the mechanism that they use to produce subgenomic RNA molecules differs. Here we performed deep sequencing to determine how equine torovirus produces subgenomic RNAs. In doing so, we also identified two previously unknown open reading frames “hidden” within the genome. Together these results highlight the similarities and differences between this domestic animal virus and related pathogens of humans and livestock." />
<meta name="DC.Contributor" content="Hazel Stewart" />
<meta name="DC.Contributor" content="Katherine Brown" />
<meta name="DC.Contributor" content="Adam M. Dinan" />
<meta name="DC.Contributor" content="Nerea Irigoyen" />
<meta name="DC.Contributor" content="Eric J. Snijder" />
<meta name="DC.Contributor" content="Andrew E. Firth" />
<meta name="article:published_time" content="2018-04-07" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="The transcriptional and translational landscape of equine torovirus" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;The genus &lt;i&gt;Torovirus&lt;/i&gt; (subfamily &lt;i&gt;Torovirinae&lt;/i&gt;, family &lt;i&gt;Coronaviridae&lt;/i&gt;, order &lt;i&gt;Nidovirales&lt;/i&gt;) encompasses a range of species that infect domestic ungulates including cattle, sheep, goats, pigs and horses, causing an acute self-limiting gastroenteritis. Using the prototype species equine torovirus (EToV) we performed parallel RNA sequencing (RNA-seq) and ribosome profiling (Ribo-seq) to analyse the relative expression levels of the known torovirus proteins and transcripts, chimaeric sequences produced via discontinuous RNA synthesis (a characteristic of the nidovirus replication cycle) and changes in host transcription and translation as a result of EToV infection. RNA sequencing confirmed that EToV utilises a unique combination of discontinuous and non-discontinuous RNA synthesis to produce its subgenomic RNAs; indeed, we identified transcripts arising from both mechanisms that would result in sgRNAs encoding the nucleocapsid. Our ribosome profiling analysis revealed that ribosomes efficiently translate two novel CUG-initiated ORFs, located within the so-called 5’ UTR. We have termed the resulting proteins U1 and U2. Comparative genomic analysis confirmed that these ORFs are conserved across all available torovirus sequences and the inferred amino acid sequences are subject to purifying selection, indicating that U1 and U2 are functionally relevant. This study provides the first high-resolution analysis of transcription and translation in this neglected group of livestock pathogens.&lt;/p&gt;&lt;h3&gt;Importance&lt;/h3&gt;
&lt;p&gt;Toroviruses infect cattle, goats, pigs and horses worldwide and can cause gastrointestinal disease. There is no treatment or vaccine and their ability to spill over into humans has not been assessed. These viruses are related to important human pathogens including severe acute respiratory syndrome (SARS) coronavirus and they share some common features, however the mechanism that they use to produce subgenomic RNA molecules differs. Here we performed deep sequencing to determine how equine torovirus produces subgenomic RNAs. In doing so, we also identified two previously unknown open reading frames “hidden” within the genome. Together these results highlight the similarities and differences between this domestic animal virus and related pathogens of humans and livestock.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2018/01/01" />
<meta name="citation_mjid" content="biorxiv;296996v1" />
<meta name="citation_id" content="296996v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/296996v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/296996v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/296996v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996.full.pdf" />
<meta name="citation_doi" content="10.1101/296996" />
<meta name="citation_num_pages" content="50" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="296996" />
<meta name="citation_author" content="Hazel Stewart" />
<meta name="citation_author_institution" content="Division of Virology, Department of Pathology, University of Cambridge" />
<meta name="citation_author" content="Katherine Brown" />
<meta name="citation_author_institution" content="Division of Virology, Department of Pathology, University of Cambridge" />
<meta name="citation_author" content="Adam M. Dinan" />
<meta name="citation_author_institution" content="Division of Virology, Department of Pathology, University of Cambridge" />
<meta name="citation_author" content="Nerea Irigoyen" />
<meta name="citation_author_institution" content="Division of Virology, Department of Pathology, University of Cambridge" />
<meta name="citation_author" content="Eric J. Snijder" />
<meta name="citation_author_institution" content="Molecular Virology Laboratory, Department of Medical Microbiology, Leiden University Medical Center" />
<meta name="citation_author" content="Andrew E. Firth" />
<meta name="citation_author_institution" content="Division of Virology, Department of Pathology, University of Cambridge" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7986-9520" />
<meta name="citation_reference" content="Lauber C, Ziebuhr J, Junglen S, Drosten C, Zirkel F, Nga PT, Morita K, Snijder EJ, Gorbalenya AE. 2012. Mesoniviridae: a proposed new family in the order Nidovirales formed by a single species of mosquito-borne viruses. Arch Virol 157:1623–8." />
<meta name="citation_reference" content="Brown DW, Beards GM, Flewett TH. 1987. Detection of Breda virus antigen and antibody in humans and animals by enzyme immunoassay. J Clin Microbiol 25:637–40." />
<meta name="citation_reference" content="Hoet AE, Saif LJ. 2004. Bovine torovirus (Breda virus) revisited. Anim Health Res Rev 5:157–71." />
<meta name="citation_reference" content="Hanke D, Pohlmann A, Sauter-Louis C, Hoper D, Stadler J, Ritzmann M, Steinrigl A, Schwarz BA, Akimkin V, Fux R, Blome S, Beer M. 2017. Porcine Epidemic Diarrhea in Europe: In-Detail Analyses of Disease Dynamics and Molecular Epidemiology. Viruses 9." />
<meta name="citation_reference" content="Pignatelli J, Grau-Roma L, Jimenez M, Segales J, Rodriguez D. 2010. Longitudinal serological and virological study on porcine torovirus (PToV) in piglets from Spanish farms. Vet Microbiol 146:260–8." />
<meta name="citation_reference" content="Alonso-Padilla J, Pignatelli J, Simon-Grife M, Plazuelo S, Casal J, Rodriguez D. 2012. Seroprevalence of porcine torovirus (PToV) in Spanish farms. BMC Res Notes 5:675." />
<meta name="citation_reference" content="Smits SL, Lavazza A, Matiz K, Horzinek MC, Koopmans MP, de Groot RJ. 2003. Phylogenetic and evolutionary relationships among torovirus field variants: evidence for multiple intertypic recombination events. J Virol 77:9567–77." />
<meta name="citation_reference" content="Pasternak AO, Spaan WJ, Snijder EJ. 2006. Nidovirus transcription: how to make sense…? J Gen Virol 87:1403–21." />
<meta name="citation_reference" content="van Marle G, Dobbe JC, Gultyaev AP, Luytjes W, Spaan WJ, Snijder EJ. 1999. Arterivirus discontinuous mRNA transcription is guided by base pairing between sense and antisense transcription-regulating sequences. Proc Natl Acad Sci U S A 96:12056–61." />
<meta name="citation_reference" content="Cowley JA, Dimmock CM, Walker PJ. 2002. Gill-associated nidovirus of Penaeus monodon prawns transcribes 3’-coterminal subgenomic mRNAs that do not possess 5’-leader sequences. J Gen Virol 83:927–35." />
<meta name="citation_reference" content="Zirkel F, Roth H, Kurth A, Drosten C, Ziebuhr J, Junglen S. 2013. Identification and characterization of genetically divergent members of the newly established family Mesoniviridae. J Virol 87:6346–58." />
<meta name="citation_reference" content="van Vliet AL, Smits SL, Rottier PJ, de Groot RJ. 2002. Discontinuous and non-discontinuous subgenomic RNA transcription in a nidovirus. EMBO J 21:6571–80." />
<meta name="citation_reference" content="Weiss M, Steck F, Horzinek MC. 1983. Purification and partial characterization of a new enveloped RNA virus (Berne virus). J Gen Virol 64 (Pt 9):1849–58." />
<meta name="citation_reference" content="Kuwabara M, Wada K, Maeda Y, Miyazaki A, Tsunemitsu H. 2007. First isolation of cytopathogenic bovine torovirus in cell culture from a calf with diarrhea. Clin Vaccine Immunol 14:998–1004." />
<meta name="citation_reference" content="Irigoyen N, Firth AE, Jones JD, Chung BY, Siddell SG, Brierley I. 2016. High-Resolution Analysis of Coronavirus Gene Expression by RNA Sequencing and Ribosome Profiling. PLoS Pathog 12:e1005473." />
<meta name="citation_reference" content="Snijder EJ, den Boon JA, Horzinek MC, Spaan WJ. 1991. Comparison of the genome organization of toro- and coronaviruses: evidence for two nonhomologous RNA recombination events during Berne virus evolution. Virology 180:448–52." />
<meta name="citation_reference" content="Snijder EJ, den Boon JA, Bredenbeek PJ, Horzinek MC, Rijnbrand R, Spaan WJ. 1990. The carboxyl-terminal part of the putative Berne virus polymerase is expressed by ribosomal frameshifting and contains sequence motifs which indicate that toro- and coronaviruses are evolutionarily related. Nucleic Acids Res 18:4535–42." />
<meta name="citation_reference" content="Yang D, Leibowitz JL. 2015. The structure and functions of coronavirus genomic 3’ and 5’ ends. Virus Res 206:120–33." />
<meta name="citation_reference" content="Snijder EJ, den Boon JA, Horzinek MC, Spaan WJ. 1991. Characterization of defective interfering RNAs of Berne virus. J Gen Virol 72 (Pt 7):1635–43." />
<meta name="citation_reference" content="Gerashchenko MV, Gladyshev VN. 2014. Translation inhibitors cause abnormalities in ribosome profiling experiments. Nucleic Acids Res 42:e134." />
<meta name="citation_reference" content="Andreev DE, O’Connor PB, Loughran G, Dmitriev SE, Baranov PV, Shatsky IN. 2017. Insights into the mechanisms of eukaryotic translation gained with ribosome profiling. Nucleic Acids Res 45:513–526." />
<meta name="citation_reference" content="Yang Z. 2007. PAML 4: phylogenetic analysis by maximum likelihood. Mol Biol Evol 24:1586–91." />
<meta name="citation_reference" content="Firth AE, Brown CM. 2006. Detecting overlapping coding sequences in virus genomes. BMC Bioinformatics 7:75." />
<meta name="citation_reference" content="Firth AE, Atkins JF. 2009. A case for a CUG-initiated coding sequence overlapping torovirus ORF1a and encoding a novel 30 kDa product. Virol J 6:136." />
<meta name="citation_reference" content="Keese PK, Gibbs A. 1992. Origins of genes: “big bang” or continuous creation? Proc Natl Acad Sci U S A 89:9489–93." />
<meta name="citation_reference" content="Rancurel C, Khosravi M, Dunker AK, Romero PR, Karlin D. 2009. Overlapping genes produce proteins with unusual sequence properties and offer insight into de novo protein creation. J Virol 83:10719–36." />
<meta name="citation_reference" content="Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ. 1990. Basic local alignment search tool. J Mol Biol 215:403–10." />
<meta name="citation_reference" content="Soding J, Biegert A, Lupas AN. 2005. The HHpred interactive server for protein homology detection and structure prediction. Nucleic Acids Res 33:W244–8." />
<meta name="citation_reference" content="Firth AE. 2014. Mapping overlapping functional elements embedded within the protein-coding regions of RNA viruses. Nucleic Acids Res 42:12425–39." />
<meta name="citation_reference" content="Kozak M. 1986. Point mutations define a sequence flanking the AUG initiator codon that modulates translation by eukaryotic ribosomes. Cell 44:283–92." />
<meta name="citation_reference" content="Kelley LA, Mezulis S, Yates CM, Wass MN, Sternberg MJ. 2015. The Phyre2 web portal for protein modeling, prediction and analysis. Nat Protoc 10:845–58." />
<meta name="citation_reference" content="Haas J, Roth S, Arnold K, Kiefer F, Schmidt T, Bordoli L, Schwede T. 2013. The Protein Model Portal--a comprehensive resource for protein structure and model information. Database (Oxford) 2013:bat031." />
<meta name="citation_reference" content="McGuffin LJ, Bryson K, Jones DT. 2000. The PSIPRED protein structure prediction server. Bioinformatics 16:404–5." />
<meta name="citation_reference" content="Smits SL, van Vliet AL, Segeren K, el Azzouzi H, van Essen M, de Groot RJ. 2005. Torovirus non-discontinuous transcription: mutational analysis of a subgenomic mRNA promoter. J Virol 79:8275–81." />
<meta name="citation_reference" content="Schutze H, Ulferts R, Schelle B, Bayer S, Granzow H, Hoffmann B, Mettenleiter TC, Ziebuhr J. 2006. Characterization of White bream virus reveals a novel genetic cluster of nidoviruses. J Virol 80:11598–609." />
<meta name="citation_reference" content="Di H, Madden JC, Jr., Morantz EK, Tang HY, Graham RL, Baric RS, Brinton MA. 2017. Expanded subgenomic mRNA transcriptome and coding capacity of a nidovirus. Proc Natl Acad Sci U S A 114:E8895–E8904." />
<meta name="citation_reference" content="Cong F, Liu X, Han Z, Shao Y, Kong X, Liu S. 2013. Transcriptome analysis of chicken kidney tissues following coronavirus avian infectious bronchitis virus infection. BMC Genomics 14:743." />
<meta name="citation_reference" content="Raaben M, Groot Koerkamp MJ, Rottier PJ, de Haan CA. 2007. Mouse hepatitis coronavirus replication induces host translational shutoff and mRNA decay, with concomitant formation of stress granules and processing bodies. Cell Microbiol 9:2218–29." />
<meta name="citation_reference" content="Zhang C, Xue C, Li Y, Kong Q, Ren X, Li X, Shu D, Bi Y, Cao Y. 2010. Profiling of cellular proteins in porcine reproductive and respiratory syndrome virus virions by proteomics analysis. Virol J 7:242." />
<meta name="citation_reference" content="VanLeuven JT, Ridenhour BJ, Gonzalez AJ, Miller CR, Miura TA. 2017. Lung epithelial cells have virus-specific and shared gene expression responses to infection by diverse respiratory viruses. PLoS One 12:e0178408." />
<meta name="citation_reference" content="Beura LK, Dinh PX, Osorio FA, Pattnaik AK. 2011. Cellular poly(c) binding proteins 1 and 2 interact with porcine reproductive and respiratory syndrome virus nonstructural protein 1beta and support viral replication. J Virol 85:12939–49." />
<meta name="citation_reference" content="Maestre AM, Garzon A, Rodriguez D. 2011. Equine torovirus (BEV) induces caspase-mediated apoptosis in infected cells. PLoS One 6:e20972." />
<meta name="citation_reference" content="Touriol C, Bornes S, Bonnal S, Audigier S, Prats H, Prats AC, Vagner S. 2003. Generation of protein isoform diversity by alternative initiation of translation at non-AUG codons. Biol Cell 95:169–78." />
<meta name="citation_reference" content="Firth AE, Brierley I. 2012. Non-canonical translation in RNA viruses. J Gen Virol 93:1385–409." />
<meta name="citation_reference" content="Draker R, Roper RL, Petric M, Tellier R. 2006. The complete sequence of the bovine torovirus genome. Virus Res 115:56–68." />
<meta name="citation_reference" content="Ito M, Tsuchiaka S, Naoi Y, Otomaru K, Sato M, Masuda T, Haga K, Oka T, Yamasato H, Omatsu T, Sugimura S, Aoki H, Furuya T, Katayama Y, Oba M, Shirai J, Katayama K, Mizutani T, Nagai M. 2016. Whole genome analysis of Japanese bovine toroviruses reveals natural recombination between porcine and bovine toroviruses. Infect Genet Evol 38:90–95." />
<meta name="citation_reference" content="Sun H, Lan D, Lu L, Chen M, Wang C, Hua X. 2014. Molecular characterization and phylogenetic analysis of the genome of porcine torovirus. Arch Virol 159:773–8." />
<meta name="citation_reference" content="Ruch TR, Machamer CE. 2012. The coronavirus E protein: assembly and beyond. Viruses 4:363–82." />
<meta name="citation_reference" content="Senanayake SD, Hofmann MA, Maki JL, Brian DA. 1992. The nucleocapsid protein gene of bovine coronavirus is bicistronic. J Virol 66:5277–83." />
<meta name="citation_reference" content="Fischer F, Peng D, Hingley ST, Weiss SR, Masters PS. 1997. The internal open reading frame within the nucleocapsid gene of mouse hepatitis virus encodes a structural protein that is not essential for viral replication. J Virol 71:996–1003." />
<meta name="citation_reference" content="Irigoyen N, Dinan AM, Brierley I, Firth AE. 2018. Ribosome profiling of the retrovirus murine leukemia virus. Retrovirology 15:10." />
<meta name="citation_reference" content="Coordinators NR. 2016. Database resources of the National Center for Biotechnology Information. Nucleic Acids Res 44:D7–19." />
<meta name="citation_reference" content="Pruitt KD, Tatusova T, Brown GR, Maglott DR. 2012. NCBI Reference Sequences (RefSeq): current status, new features and genome annotation policy. Nucleic Acids Res 40:D130–5." />
<meta name="citation_reference" content="Flicek P, Amode MR, Barrell D, Beal K, Brent S, Carvalho-Silva D, Clapham P, Coates G, Fairley S, Fitzgerald S, Gil L, Gordon L, Hendrix M, Hourlier T, Johnson N, Kahari AK, Keefe D, Keenan S, Kinsella R, Komorowska M, Koscielny G, Kulesha E, Larsson P, Longden I, McLaren W, Muffato M, Overduin B, Pignatelli M, Pritchard B, Riat HS, Ritchie GR, Ruffier M, Schuster M, Sobral D, Tang YA, Taylor K, Trevanion S, Vandrovcova J, White S, Wilson M, Wilder SP, Aken BL, Birney E, Cunningham F, Dunham I, Durbin R, Fernandez-Suarez XM, Harrow J, Herrero J, Hubbard TJ, et al. 2012. Ensembl 2012. Nucleic Acids Res 40:D84–90." />
<meta name="citation_reference" content="Chan PP, Lowe TM. 2016. GtRNAdb 2.0: an expanded database of transfer RNA genes identified in complete and draft genomes. Nucleic Acids Res 44:D184–9." />
<meta name="citation_reference" content="Langmead B, Trapnell C, Pop M, Salzberg SL. 2009. Ultrafast and memory-efficient alignment of short DNA sequences to the human genome. Genome Biol 10:R25." />
<meta name="citation_reference" content="Dobin A, Davis CA, Schlesinger F, Drenkow J, Zaleski C, Jha S, Batut P, Chaisson M, Gingeras TR. 2013. STAR: ultrafast universal RNA-seq aligner. Bioinformatics 29:15–21." />
<meta name="citation_reference" content="Grabherr MG, Haas BJ, Yassour M, Levin JZ, Thompson DA, Amit I, Adiconis X, Fan L, Raychowdhury R, Zeng Q, Chen Z, Mauceli E, Hacohen N, Gnirke A, Rhind N, di Palma F, Birren BW, Nusbaum C, Lindblad-Toh K, Friedman N, Regev A. 2011. Full-length transcriptome assembly from RNA-Seq data without a reference genome. Nat Biotechnol 29:644–52." />
<meta name="citation_reference" content="Katoh K, Standley DM. 2013. MAFFT multiple sequence alignment software version 7: improvements in performance and usability. Mol Biol Evol 30:772–80." />
<meta name="citation_reference" content="Sievers F, Higgins DG. 2014. Clustal omega. Curr Protoc Bioinformatics 48:3 13 1–16." />
<meta name="citation_reference" content="Bernhart SH, Hofacker IL, Will S, Gruber AR, Stadler PF. 2008. RNAalifold: improved consensus structure prediction for RNA alignments. BMC Bioinformatics 9:474." />
<meta name="citation_reference" content="Darty K, Denise A, Ponty Y. 2009. VARNA: Interactive drawing and editing of the RNA secondary structure. Bioinformatics 25:1974–5." />
<meta name="citation_reference" content="Anders S, Pyl PT, Huber W. 2015. HTSeq--a Python framework to work with high-throughput sequencing data. Bioinformatics 31:166–9." />
<meta name="citation_reference" content="Love MI, Huber W, Anders S. 2014. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol 15:550." />
<meta name="citation_reference" content="Strimmer K. 2008. fdrtool: a versatile R package for estimating local and tail area-based false discovery rates. Bioinformatics 24:1461–2." />
<meta name="citation_reference" content="Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, Cherry JM, Davis AP, Dolinski K, Dwight SS, Eppig JT, Harris MA, Hill DP, Issel-Tarver L, Kasarskis A, Lewis S, Matese JC, Richardson JE, Ringwald M, Rubin GM, Sherlock G. 2000. Gene ontology: tool for the unification of biology. The Gene Ontology Consortium. Nat Genet 25:25–9." />
<meta name="citation_reference" content="Durinck S, Spellman PT, Birney E, Huber W. 2009. Mapping identifiers for the integration of genomic datasets with the R/Bioconductor package biomaRt. Nat Protoc 4:1184–91." />
<meta name="citation_reference" content="Xiao Z, Zou Q, Liu Y, Yang X. 2016. Genome-wide assessment of differential translations with ribosome profiling data. Nat Commun 7:11194." />
<meta name="citation_reference" content="Anbalagan S, Peterson J, Wassman B, Elston J, Schwartz K. 2014. Genome sequence of torovirus identified from a pig with porcine epidemic diarrhea virus from the United States. Genome Announc 2." />
<meta name="citation_reference" content="Edgar RC. 2004. MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Res 32:1792–7." />
<meta name="citation_reference" content="Rice P, Longden I, Bleasby A. 2000. EMBOSS: the European Molecular Biology Open Software Suite. Trends Genet 16:276–7." />
<meta name="citation_reference" content="Guindon S, Gascuel O. 2003. A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood. Syst Biol 52:696–704." />
<meta name="citation_reference" content="Stocsits RR, Hofacker IL, Fried C, Stadler PF. 2005. Multiple sequence alignments of partially coding nucleic acid sequences. BMC Bioinformatics 6:160." />
<meta name="twitter:title" content="The transcriptional and translational landscape of equine torovirus" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="The genus Torovirus (subfamily Torovirinae , family Coronaviridae , order Nidovirales ) encompasses a range of species that infect domestic ungulates including cattle, sheep, goats, pigs and horses, causing an acute self-limiting gastroenteritis. Using the prototype species equine torovirus (EToV) we performed parallel RNA sequencing (RNA-seq) and ribosome profiling (Ribo-seq) to analyse the relative expression levels of the known torovirus proteins and transcripts, chimaeric sequences produced via discontinuous RNA synthesis (a characteristic of the nidovirus replication cycle) and changes in host transcription and translation as a result of EToV infection. RNA sequencing confirmed that EToV utilises a unique combination of discontinuous and non-discontinuous RNA synthesis to produce its subgenomic RNAs; indeed, we identified transcripts arising from both mechanisms that would result in sgRNAs encoding the nucleocapsid. Our ribosome profiling analysis revealed that ribosomes efficiently translate two novel CUG-initiated ORFs, located within the so-called 5’ UTR. We have termed the resulting proteins U1 and U2. Comparative genomic analysis confirmed that these ORFs are conserved across all available torovirus sequences and the inferred amino acid sequences are subject to purifying selection, indicating that U1 and U2 are functionally relevant. This study provides the first high-resolution analysis of transcription and translation in this neglected group of livestock pathogens.

Importance Toroviruses infect cattle, goats, pigs and horses worldwide and can cause gastrointestinal disease. There is no treatment or vaccine and their ability to spill over into humans has not been assessed. These viruses are related to important human pathogens including severe acute respiratory syndrome (SARS) coronavirus and they share some common features, however the mechanism that they use to produce subgenomic RNA molecules differs. Here we performed deep sequencing to determine how equine torovirus produces subgenomic RNAs. In doing so, we also identified two previously unknown open reading frames “hidden” within the genome. Together these results highlight the similarities and differences between this domestic animal virus and related pathogens of humans and livestock." />
<meta name="og-title" property="og:title" content="The transcriptional and translational landscape of equine torovirus" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/296996v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="The genus Torovirus (subfamily Torovirinae , family Coronaviridae , order Nidovirales ) encompasses a range of species that infect domestic ungulates including cattle, sheep, goats, pigs and horses, causing an acute self-limiting gastroenteritis. Using the prototype species equine torovirus (EToV) we performed parallel RNA sequencing (RNA-seq) and ribosome profiling (Ribo-seq) to analyse the relative expression levels of the known torovirus proteins and transcripts, chimaeric sequences produced via discontinuous RNA synthesis (a characteristic of the nidovirus replication cycle) and changes in host transcription and translation as a result of EToV infection. RNA sequencing confirmed that EToV utilises a unique combination of discontinuous and non-discontinuous RNA synthesis to produce its subgenomic RNAs; indeed, we identified transcripts arising from both mechanisms that would result in sgRNAs encoding the nucleocapsid. Our ribosome profiling analysis revealed that ribosomes efficiently translate two novel CUG-initiated ORFs, located within the so-called 5’ UTR. We have termed the resulting proteins U1 and U2. Comparative genomic analysis confirmed that these ORFs are conserved across all available torovirus sequences and the inferred amino acid sequences are subject to purifying selection, indicating that U1 and U2 are functionally relevant. This study provides the first high-resolution analysis of transcription and translation in this neglected group of livestock pathogens.

Importance Toroviruses infect cattle, goats, pigs and horses worldwide and can cause gastrointestinal disease. There is no treatment or vaccine and their ability to spill over into humans has not been assessed. These viruses are related to important human pathogens including severe acute respiratory syndrome (SARS) coronavirus and they share some common features, however the mechanism that they use to produce subgenomic RNA molecules differs. Here we performed deep sequencing to determine how equine torovirus produces subgenomic RNAs. In doing so, we also identified two previously unknown open reading frames “hidden” within the genome. Together these results highlight the similarities and differences between this domestic animal virus and related pathogens of humans and livestock." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2018-04-07" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/296996v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/296996v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/296996v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/296996v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/91424" />
    <title>The transcriptional and translational landscape of equine torovirus | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"HZ88pD4kcw9YnHWMk1ONjgDjZA1SPr7ZO7aQZTNJ2Yc","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"91424","apath":"\/biorxiv\/early\/2018\/04\/07\/296996.atom","pisa":"biorxiv;296996v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;296996v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/296996v1","title":"The transcriptional and translational landscape of equine torovirus","identifier":"node\/91424"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bi4T0Hie+qf48Tki+EH4nXjElU22ZTUiCCVr3FkJGfZGdq1fKl3DnRiO12q8IDJfDpUcpltVTZlgpgppeYR9FtZU4G3c8\\\\\\\/vdSREkIlnbW5jvlYFotfueBQ+UD1Rvq2XkffH05HHmhQHQiAE8bfwxXO21LuP5z+sFvLY56KHIp1o8vGyV\\\\\\\/Cdmr5YSSeT\\\\\\\/tJLskiqhNzGgDm8NgFoHH9SyUGv\\\\\\\/tQUHPhytEUuI9HyEn7pDOCZUkjJ\\\\\\\/b56nl0yBDiIp90kpZljL\\\\\\\/49uXCNmkg+KfczNtD31pamEogral9N2aBKIUyd5DJukIsM3Y9m5ZNKVwRWBrkqCw7bDnygxfdxCDDCW3n4+luRLUTtFtRWYeeN6Tdxk23CvRzvU5aa2Z\\\\\\\/buRcsuRgQSwr0qHZkm86QmQxUD969cBr+OmvoDxHI53woxfNbSpEFVrzjAnnJ\\\\\\\/u7Vhogwt6bNXcy\\\\\\\/4jofeIOjXq7DgQ21SAhUaO0gYHKQINGnWAOXjdkgJTX0FWzLiLaJSYmKX4jbab0VMgdaOBPAMbhuF+ro\\\\\\\/DadYcXvSCsQoJrnB1xadmGKK8DV1Gowox1RK2WbmNnl2NXu1jRNWITgJIklK65vwtfLxJvGtBCtfcHSZhFAV6UnVv6DPxx\\\\\\\/maJ73L2IAWjQMCNigNF67h0dPqLwgMNX9WiCiCG+Tp3CSwjK3bqoAw5sSA3MeIt1S\\\\\\\/okWxsMFvPpiv9Vk5D01VYit8wnZcZ6t94EK5exmgINHAjX4bJs6L\\\\\\\/WtbTePVLHSbKiMIijHq47IDDc9YOEDxruv7xPC4fKKNYwRr4zWmPafxUZJW\\\\\\\/GfjdGSWEvxvB3MxoSj\\\\\\\/zDpsmcL1XlYkqc6EjRsaheqv9x40MSZktI2jG8M60uR85Q6kqMrgy6b4bfwI+4GQdf3bP6PNgyjZTZzns0s40Tn9AgjJAMU9Avn3DVubpc\\\\\\\/DZSsOK4FBitfHoLtXOTN1Baet3VRuAgVrJMK33Z2U9hls6LmgJYAPpqh8wTzkoBi\\\\\\\/peON1mgbDNeOJ51+VLH9nr9xv2wgWmtCOva8kWzTkmB1L4rxHWxSu5B9FI\\\\\\\/4W1Ttz5SGOqKgew5xmTs3fde+KxdXiwWLLYmc7oj8Zb53BnOSkKlDGyHvkhkNh8\\\\\\\/+RvFw1i+J76YXej1Im3LdN\\\\\\\/GMaUWFJJj5Mxy5+Ga4\\\\\\\/\\\\\\\/Jkf+IgI2EQw2e2HcEERU5WB0Ey7LfJ6tu0ycPRm7L7vxRJb3LF3s\\\\\\\/drIThw4eQZ7jEOqG6L2uZzZ4zVszlNRPqY01Y58+NRPFHIBZZumTZ2Mj7MXUUT0hWK9j+lQWNPxj3EFqEop6E47DZxVtmTG\\\\\\\/aX8=\\\u0022,\\\u0022iv\\\u0022:\\\u0022Ql8UgQpBmZIJ2SQ0Qx\\\\\\\/eEg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022786d86c180ec7f61a78e41f8bcaeac177610dbe3e0a76590c2386f4ad7946806\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/296996v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-91424 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/296996v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-KTCh045ogIqDIq9bM9MkbA2BTg41rJtvQby4FO8c0to" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="91424" id="node91424" data-pisa="biorxiv;296996v1" data-pisa-master="biorxiv;296996" data-apath="/biorxiv/early/2018/04/07/296996.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">The transcriptional and translational landscape of equine torovirus</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Hazel</span> <span class="nlm-surname">Stewart</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Katherine</span> <span class="nlm-surname">Brown</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Adam M.</span> <span class="nlm-surname">Dinan</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Nerea</span> <span class="nlm-surname">Irigoyen</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Eric J.</span> <span class="nlm-surname">Snijder</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-7986-9520" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Andrew E.</span> <span class="nlm-surname">Firth</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/296996 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node91424" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hazel Stewart </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Division of Virology, Department of Pathology, University of Cambridge</span>, Cambridge, <span class='nlm-country'>United Kingdom</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hazel%2BStewart%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stewart%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHazel%2BStewart%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Katherine Brown </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Division of Virology, Department of Pathology, University of Cambridge</span>, Cambridge, <span class='nlm-country'>United Kingdom</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katherine%2BBrown%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brown%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKatherine%2BBrown%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Adam M. Dinan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Division of Virology, Department of Pathology, University of Cambridge</span>, Cambridge, <span class='nlm-country'>United Kingdom</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adam%2BM.%2BDinan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dinan%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAdam%2BM.%2BDinan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Nerea Irigoyen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Division of Virology, Department of Pathology, University of Cambridge</span>, Cambridge, <span class='nlm-country'>United Kingdom</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nerea%2BIrigoyen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Irigoyen%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANerea%2BIrigoyen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Eric J. Snijder </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Molecular Virology Laboratory, Department of Medical Microbiology, Leiden University Medical Center</span>, Leiden, <span class='nlm-country'>The Netherlands</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eric%2BJ.%2BSnijder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Snijder%20EJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEric%2BJ.%2BSnijder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Andrew E. Firth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Division of Virology, Department of Pathology, University of Cambridge</span>, Cambridge, <span class='nlm-country'>United Kingdom</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BE.%2BFirth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Firth%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BE.%2BFirth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7986-9520" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrew E. Firth</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/296996v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:91424" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:91424/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/296996v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:91424" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:91424/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/296996v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:91424" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:91424/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/296996v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:91424" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:91424/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/296996v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:91424" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:91424/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-4">The genus <em>Torovirus</em> (subfamily <em>Torovirinae</em>, family <em>Coronaviridae</em>, order <em>Nidovirales</em>) encompasses a range of species that infect domestic ungulates including cattle, sheep, goats, pigs and horses, causing an acute self-limiting gastroenteritis. Using the prototype species equine torovirus (EToV) we performed parallel RNA sequencing (RNA-seq) and ribosome profiling (Ribo-seq) to analyse the relative expression levels of the known torovirus proteins and transcripts, chimaeric sequences produced via discontinuous RNA synthesis (a characteristic of the nidovirus replication cycle) and changes in host transcription and translation as a result of EToV infection. RNA sequencing confirmed that EToV utilises a unique combination of discontinuous and non-discontinuous RNA synthesis to produce its subgenomic RNAs; indeed, we identified transcripts arising from both mechanisms that would result in sgRNAs encoding the nucleocapsid. Our ribosome profiling analysis revealed that ribosomes efficiently translate two novel CUG-initiated ORFs, located within the so-called 5’ UTR. We have termed the resulting proteins U1 and U2. Comparative genomic analysis confirmed that these ORFs are conserved across all available torovirus sequences and the inferred amino acid sequences are subject to purifying selection, indicating that U1 and U2 are functionally relevant. This study provides the first high-resolution analysis of transcription and translation in this neglected group of livestock pathogens.</p><div id="sec-1" class="subsection"><p id="p-5"><strong>Importance</strong> Toroviruses infect cattle, goats, pigs and horses worldwide and can cause gastrointestinal disease. There is no treatment or vaccine and their ability to spill over into humans has not been assessed. These viruses are related to important human pathogens including severe acute respiratory syndrome (SARS) coronavirus and they share some common features, however the mechanism that they use to produce subgenomic RNA molecules differs. Here we performed deep sequencing to determine how equine torovirus produces subgenomic RNAs. In doing so, we also identified two previously unknown open reading frames “hidden” within the genome. Together these results highlight the similarities and differences between this domestic animal virus and related pathogens of humans and livestock.</p></div></div><div class="section" id="sec-2"><h2 class="">Introduction</h2><p id="p-6">The order <em>Nidovirales</em> currently contains four families of positive-sense, single-stranded RNA viruses: the <em>Coronaviridae</em>, <em>Arteriviridae</em>, <em>Roniviridae</em> and <em>Mesoniviridae</em> (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>). Their grouping into the one taxonomic order is based upon replicase protein conservation, genome organisation and replication strategy. However these viral families are nonetheless very diverse with respect to their virion structure, host range, pathogenic potential and genome size.</p><p id="p-7">The genus <em>Torovirus</em> (family <em>Coronaviridae</em>, subfamily <em>Torovirinae</em>) encompasses a range of species with worldwide distribution that infect domestic ungulates including cattle, goats, sheep, pigs and horses, causing an acute self-limiting gastroenteritis. Approximately 55 % of cattle within the United Kingdom are seropositive for bovine torovirus and this pathogen represents a significant burden to the industry (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>). Similarly porcine torovirus is endemic in Europe and causes disease in production herds (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>–<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>). Despite this, limited research has been conducted upon these pathogens and neither specific antiviral treatments nor vaccines are available. The prevalence of toroviruses in non-domestic reservoirs and potential for cross-species transmission has not been assessed, although they are known to undergo recombination events (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>). The extensive research conducted upon the related coronaviruses would not necessarily be relevant in the event of an emerging torovirus infection, due to the divergent nature of these viruses.</p><p id="p-8">The genomes of <em>Nidovirales</em> are positive-sense, polycistronic RNAs. One of the hallmarks of this virus order is the utilisation of an unusual transcription mechanism to express the genes encoding structural and accessory proteins, which reside downstream of the large replicase open reading frames (ORFs) 1a and 1b (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). These proteins are typically translated from a nested set of 3’ coterminal subgenomic mRNAs (sg mRNAs). Although, with the exception of the smallest species, these sgRNAs are structurally polycistronic, translation is normally limited to the 5’ ORF of each mRNA. Studies of coronaviruses and arteriviruses have revealed that they produce negative-sense subgenome-sized RNAs via a mechanism of “discontinuous” extension (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>). This process resembles homology-assisted copy-choice recombination (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>) and requires the presence of multiple copies of a species-specific short motif, the transcription regulatory sequence (TRS). TRS motifs are located immediately upstream of the structural protein ORFs (body TRSs) and within the 5’ UTR (leader TRS).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic of the equine torovirus genome (EToV). Open reading frames (ORFs) are coloured according to their respective reading frames (pink: phase 0 yellow: phase &#x2212;1; blue: phase +1). Polyproteins pp1a and pp1ab are translated from genomic RNA, with pp1ab generated via &#x2212;1 programmed ribosomal frameshifting. Structural proteins are translated from a series of subgenomic RNAs. Untranslated regions of subgenomic RNAs are represented by black bars. The leader transcription regulatory sequence (TRS) (green) and putative body TRSs (blue) are displayed below the viral genome. The frameshift site and a putative RNA hairpin involved in S sgRNA synthesis are indicated above the genome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-880035169" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Schematic of the equine torovirus genome (EToV). Open reading frames (ORFs) are coloured according to their respective reading frames (pink: phase 0 yellow: phase &#x2212;1; blue: phase +1). Polyproteins pp1a and pp1ab are translated from genomic RNA, with pp1ab generated via &#x2212;1 programmed ribosomal frameshifting. Structural proteins are translated from a series of subgenomic RNAs. Untranslated regions of subgenomic RNAs are represented by black bars. The leader transcription regulatory sequence (TRS) (green) and putative body TRSs (blue) are displayed below the viral genome. The frameshift site and a putative RNA hairpin involved in S sgRNA synthesis are indicated above the genome.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F1.medium.gif" width="440" height="163"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F1.medium.gif" width="440" height="163"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-9" class="first-child">Schematic of the equine torovirus genome (EToV). Open reading frames (ORFs) are coloured according to their respective reading frames (pink: phase 0 yellow: phase −1; blue: phase +1). Polyproteins pp1a and pp1ab are translated from genomic RNA, with pp1ab generated via −1 programmed ribosomal frameshifting. Structural proteins are translated from a series of subgenomic RNAs. Untranslated regions of subgenomic RNAs are represented by black bars. The leader transcription regulatory sequence (TRS) (green) and putative body TRSs (blue) are displayed below the viral genome. The frameshift site and a putative RNA hairpin involved in S sgRNA synthesis are indicated above the genome.</p><div class="sb-div caption-clear"></div></div></div><p id="p-10">Negative strand RNA synthesis initiates at the 3’ end of the positive-sense viral genome. When the RNA-dependent RNA polymerase (RdRp) has copied a TRS sequence, a translocation event may occur during which the anti-TRS at the 3’ end of the nascent strand basepairs with the leader TRS within the 5’ UTR. Transcription reinitiates and continues to the 5’ end of the genomic template. The resulting “anti-leader” sequence that is added ranges from 55 – 92 nt in coronaviruses to ~200 nt in arteriviruses. These negative-sense transcripts are therefore 5’- and 3’-coterminal with the full length negative RNA strand and are identifiable as chimaeras with distinct leader-body junctions. The anti-leader sequence in each of the negative-sense templates then functions as a promoter, to drive synthesis of a mirror set of positive-sense sgRNAs that are translated to produce the structural proteins.</p><p id="p-11">However not all details of the mechanism outlined above are wholly conserved across the <em>Nidovirales</em>. Specifically, the two sg mRNAs of roniviruses (pathogens of shrimp) do not possess conserved 5’ leader sequences, indicative of the lack of a discontinuous step during their production (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>). Despite the presence of a conserved body TRS in each sg mRNA, an equivalent leader TRS is not readily identifiable in the 5’ UTR. It may therefore be reasoned that the ronivirus body TRSs stimulate termination of RNA synthesis without RdRp translocation and reinitiation. Mesoniviruses (a branch of <em>Nidovirales</em> recently identified in insects) are thought to produce two major sgRNAs possessing leader sequences of different lengths, indicating the nidoviral mechanism for discontinuous RNA synthesis may allow two very different leader/body TRS pairs to be utilised in a single viral species (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>).</p><p id="p-12">Toroviruses appear to represent a nidovirus subgroup with a remarkably flexible transcription strategy: equine torovirus (EToV) possesses a leader TRS-like sequence (CUUUAGA) but it is only involved in the synthesis of the mRNA used for expression of the spike (S) protein gene (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>). Despite similarities to the corona- and arteriviral mechanism, the preceding leader sequence incorporated into this mRNA is merely 6 nt in length (ACGUAU). Additionally, this case is unusual in that the translocation event is thought to be prompted by an RNA structure - a predicted RNA hairpin upstream of the S protein gene, rather than a body TRS (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>). Body TRSs are located upstream of the three remaining structural protein genes, yet a non-discontinuous mechanism is utilised for their production, as is the case for roniviruses. As a result, the sg mRNAs for membrane (M), nucleocapsid (N) and haemagglutinin-esterase (HE) do not normally possess a conserved 5’ leader sequence; they each possess a variable and unique extended version of the TRS at their 5’ end. It is clear there is significant difference between how the various <em>Nidovirales</em> families synthesise their sgRNAs.</p><p id="p-13">Here we describe the first high-resolution analysis of viral transcription during infection by EToV, which is one of the few toroviruses that can be propagated in cell culture (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>). RNA sequencing (RNA-seq) confirmed previous reports that EToV utilises a unique combination of both discontinuous and non-discontinuous RNA synthesis to generate its repertoire of sgRNAs. Strikingly, we also identified a small proportion of chimaeric transcripts spanning from the leader to the body TRS of the N protein gene, indicating that discontinuous and non-discontinuous mechanisms compete in this location. We also identified numerous locations across the genome where non-canonical RdRp translocation occurs, leading to a vast array of (presumably mostly non-functional) chimaeric transcripts.</p><p id="p-14">Ribosome profiling (Ribo-seq) conducted in tandem with the RNA-seq indicated ribosomes were actively translating within the so-called 5’ UTR. Further analysis confirmed the existence of two novel ORFs in this region, which are conserved in all torovirus genome sequences analysed to date. The specific function(s) of these proteins will be the topic of future work. Together, these results provide an overview of the transcriptional and translational events that accompany infection by this wide-ranging pathogen.</p></div><div class="section" id="sec-3"><h2 class="">Results</h2><div id="sec-4" class="subsection"><h3>Tandem RNA-seq and Ribo-seq of EToV infected cells</h3><p id="p-15">We conducted tandem RNA-seq and Ribo-seq of EToV infected equine dermal (ED) cells. Two biological replicates of virus-infected and mock-infected cells were analysed, generating 25 to 53 million reads per sample. For RNA-seq, 77-92 % of reads mapped to the host genome, of which a mean of 1.5 % mapped to rRNA, 19 % to mRNA, 32 % to ncRNA and 47 % elsewhere in the genome. For Ribo-seq, 46-60 % of reads mapped to the host genome, of which a mean of 56 % mapped to rRNA, 13 % to mRNA, 4.9 % to ncRNA and 26 % elsewhere in the genome (Supplementary Table 1). 1.3 % and 2.3 % of reads mapped to the virus genome in the two EToV-infected RNA-seq replicates and 0.41 % and 0.21 % in the two virus-infected Ribo-seq replicates.</p><p id="p-16">The viral genome was assembled <em>de novo</em> from RNA-seq reads and confirmed as EToV, Berne isolate. A single 27694-nt contig was assembled representing almost the entire viral genome. Only 18 nt at the 5’ terminus and 300 nt at the 3’ terminus of this contig failed to assemble automatically; however these regions were clearly covered by reads consistent with the reference sequence on inspection and so were added manually to the consensus sequence. Four single nucleotide changes were present in all reads but not the reference sequence compiled from previous sequencing data, at positions 18078 (ORF 1b, C &gt; U), 21429 (ORF S, A &gt; U), 21814 (ORF S, C &gt; A) and 25596 (ORF S, C &gt; U). The full-length virus sequence has been deposited in GenBank (Accession <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MG996765&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom">MG996765</a>).</p><p id="p-17">The distribution of reads on the virus genome and the phasing of these reads are shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. There was good coverage across the viral genome for both RNA-seq and Ribo-seq. The Ribo-seq/RNA-seq ratio along the genome was calculated (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2C</a>) to estimate translation efficiency (note that this simple estimate is naive since it does not account for the fact that the genomic RNA and different sgRNA species overlap one another). Ribo-seq density, RNA-seq density and translational efficiency were also calculated separately for each ORF (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>), based on the density of Ribo-seq reads in each ORF divided by the density of the RNA-seq reads for either the same region (for subgenomic RNAs) or the region of the genome which does not overlap the subgenomic RNAs (for genomic RNA). RNA-seq density was adjusted based on the “decumulation” methodology described previously (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>) (see Materials and Methods) to account for the fact that not all of the RNA-seq density in the 3’ ORFs derives from transcripts from which the ORFs can be expressed. Ribo-seq coverage is much higher towards the 3’ end of the genome, particularly across the M and N genes, reflecting the translation of abundant subgenomic RNAs in this region (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). ORFs 1a and 1b contain a considerably lower density of Ribo-seq reads. The relatively low translation efficiencies calculated for ORFs 1a and 1b may be partly due to some gRNA being packaged (or destined for packaging) and unavailable for translation but still contributing to the estimate of gRNA RNA-seq density. ORF1a has a higher Ribo-seq density and a higher translational efficiency than ORF1b, reflecting the proportion of ribosomes terminating at the ORF1a stop codon and not undergoing the −1 frameshift into ORF1b (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). As expected, RNA-seq density is similar across ORF1a and ORF1b, as both are present only on the full-length genomic RNA (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>). The region covering the HE ORF also has low ribosomal coverage (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>), which may be due to the fact that the EToV HE gene is nonfunctional due to a large deletion including the canonical AUG (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>). HE is not shown in <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a> as the HE transcript is much less abundant than the “upstream” M transcript which makes the decumulation procedure susceptible to noise (see Irigoyen et al., 2016). Translational efficiency appears highest for the M and S subgenomic RNAs. The high RNA-seq density in the 5’ UTR may be indicative of one or more defective interfering (DI) RNAs in the sample (see below). Ribosome protected fragments (RPFs) were also identified mapping to the second half of the 5’ UTR, mostly in the +2/-1 frame with respect to ORF1a (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2A</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Read density of (A) Ribo-seq and (B) RNA-seq reads across the viral genome from EToV infected cells. Red lines represent total reads per million mapped reads at each position; pink: reads in phase 0; yellow: phase &#x2212;1; blue: phase +1. Densities are smoothed with a 15-nt running mean filter and plotted on a log10(1+x) scale. Negative-sense reads (grey) are displayed below the x-axis for total reads only. Each line represents a single replicate. For Ribo-seq reads, a +12 nt offset has been applied to read 5&#x2019; end positions to map approximate P-site positions. (C) The positive sense Ribo-seq/RNA-seq ratio after applying a 100-nt running mean filter to each distribution. Each line represents one of the two paired Ribo-seq and RNA-seq replicates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-880035169" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Read density of (A) Ribo-seq and (B) RNA-seq reads across the viral genome from EToV infected cells. Red lines represent total reads per million mapped reads at each position; pink: reads in phase 0; yellow: phase &#x2212;1; blue: phase +1. Densities are smoothed with a 15-nt running mean filter and plotted on a log&lt;sub&gt;10&lt;/sub&gt;(1+x) scale. Negative-sense reads (grey) are displayed below the x-axis for total reads only. Each line represents a single replicate. For Ribo-seq reads, a +12 nt offset has been applied to read 5&#x2019; end positions to map approximate P-site positions. (C) The positive sense Ribo-seq/RNA-seq ratio after applying a 100-nt running mean filter to each distribution. Each line represents one of the two paired Ribo-seq and RNA-seq replicates.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F2.medium.gif" width="370" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F2.medium.gif" width="370" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-18" class="first-child">Read density of (A) Ribo-seq and (B) RNA-seq reads across the viral genome from EToV infected cells. Red lines represent total reads per million mapped reads at each position; pink: reads in phase 0; yellow: phase −1; blue: phase +1. Densities are smoothed with a 15-nt running mean filter and plotted on a log<sub>10</sub>(1+x) scale. Negative-sense reads (grey) are displayed below the x-axis for total reads only. Each line represents a single replicate. For Ribo-seq reads, a +12 nt offset has been applied to read 5’ end positions to map approximate P-site positions. (C) The positive sense Ribo-seq/RNA-seq ratio after applying a 100-nt running mean filter to each distribution. Each line represents one of the two paired Ribo-seq and RNA-seq replicates.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relative gene expression levels. (A) Ribo-seq density in reads per kilobase per million mapped reads (RPKM) for each ORF in the EToV genome. For each ORF, only reads mapping in the predominant phase (i.e. mapping to first positions of codons) were included. (B) &#x201C;Decumulated&#x201D; RNA-seq density in RPKM for each ORF. For subgenomic RNAs, density was calculated across the regions used for Ribo-seq in A; for genomic RNAs the regions for ORF1a and ORF1b were combined, as these ORFs are both translated from gRNA. A decumulation strategy was used to correct for the fact that the measured RNA density in 3&#x2019; ORFs derives from multiple 3&#x2019;-coterminal transcripts (see Materials and Methods). (C) Translation efficiency for each gene in the EToV genome, calculated as Ribo-seq density / decumulated RNA-seq density. For each ORF, the two bars represent two repeats." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-880035169" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Relative gene expression levels. (A) Ribo-seq density in reads per kilobase per million mapped reads (RPKM) for each ORF in the EToV genome. For each ORF, only reads mapping in the predominant phase (i.e. mapping to first positions of codons) were included. (B) &#x201C;Decumulated&#x201D; RNA-seq density in RPKM for each ORF. For subgenomic RNAs, density was calculated across the regions used for Ribo-seq in A; for genomic RNAs the regions for ORF1a and ORF1b were combined, as these ORFs are both translated from gRNA. A decumulation strategy was used to correct for the fact that the measured RNA density in 3&#x2019; ORFs derives from multiple 3&#x2019;-coterminal transcripts (see Materials and Methods). (C) Translation efficiency for each gene in the EToV genome, calculated as Ribo-seq density / decumulated RNA-seq density. For each ORF, the two bars represent two repeats.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F3.medium.gif" width="440" height="217"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F3.medium.gif" width="440" height="217"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <p id="p-19" class="first-child">Relative gene expression levels. (A) Ribo-seq density in reads per kilobase per million mapped reads (RPKM) for each ORF in the EToV genome. For each ORF, only reads mapping in the predominant phase (i.e. mapping to first positions of codons) were included. (B) “Decumulated” RNA-seq density in RPKM for each ORF. For subgenomic RNAs, density was calculated across the regions used for Ribo-seq in A; for genomic RNAs the regions for ORF1a and ORF1b were combined, as these ORFs are both translated from gRNA. A decumulation strategy was used to correct for the fact that the measured RNA density in 3’ ORFs derives from multiple 3’-coterminal transcripts (see Materials and Methods). (C) Translation efficiency for each gene in the EToV genome, calculated as Ribo-seq density / decumulated RNA-seq density. For each ORF, the two bars represent two repeats.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">To calculate the length distributions of host- and virus-mapped RPFs, we used reads mapping within coding regions. After adaptor trimming, the majority (75 %) of Ribo-seq reads were 27 – 29 nt in length, which is consistent with the expected size of mammalian ribosome footprints. As expected, the distribution of read lengths for RNA-seq was much broader, peaking between 60 and 70 nt (<a id="xref-fig-10-1" class="xref-fig" href="#F10">Supplementary Figure 1</a>). For quality control, histograms of the 5’ end positions of host mRNA Ribo-seq and RNA-seq reads relative to initiation and termination codons were constructed (<a id="xref-fig-11-1" class="xref-fig" href="#F11">Supplementary Figures 2</a>, <a id="xref-fig-12-1" class="xref-fig" href="#F12">3</a>). This confirmed we had high quality RPFs arising from host transcripts, with strong triplet periodicity (“phasing”) and very few reads mapping to 3’ UTRs. As in other datasets, a ramp effect of decreased RPF density was seen over a region of ~30 codons following initiation sites; but, unusually, in this dataset we did not observe a density peak at the initiation site itself (cf. Irigoyen et al 2016). This may be due to the flash freezing without cycloheximide pretreatment used for these samples, as for a later cycloheximide-treated sample this peak is present (<a id="xref-fig-11-2" class="xref-fig" href="#F11">Supplementary Figure 2</a>). Within coding sequences, the 5’ ends of the majority of reads from the host (65-81 %) and virus (60-75 %) mapped to the first positions of codons (<a id="xref-fig-13-1" class="xref-fig" href="#F13">Supplementary Figure 4</a>).</p><p id="p-21">The relative RPF density allowed us to estimate the efficiency of ribosomal frameshifting in the context of virus infection. After translating ORF1a, a proportion of ribosomes undergo a −1 ribosomal frameshift to translate ORF1b (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>). This is (presumably) required to produce a specific ratio of pp1a to pp1ab, thereby controlling the ratio of RNA-synthesing enzymes such as RdRp and helicase to other components of the replicase complex, including the proteinases and trans-membrane subunits encoded in ORF1a. The ORF1a/1b −1 ribosomal frameshifting event is stimulated by a pseudoknot structure 3’-adjacent to the U_UUA_AAC slippery heptanucleotide frameshift site. The efficiency of −1 ribosomal frameshifting (measured by dividing the mean RPF density in ORF1b by the mean density in ORF1a) was estimated to be 29.9 % for replicate one and 27.5 % for replicate 2, which is in accordance with the rates measured previously outside of the context of virus infection (20 – 30 %) (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>).</p></div><div id="sec-5" class="subsection"><h3>RNA sequencing indicates both discontinuous and non-discontinuous mechanisms are utilised for N protein gene sgRNA synthesis</h3><p id="p-22">RNA sequencing reads that did not map to either the viral genome or host databases were analysed for containing potential viral chimaeric junctions, indicative of leader-to-body joining during discontinuous sgRNA synthesis (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Relative abundances were calculated by normalising read counts to the number of non-chimaeric reads spanning each junction. Between the two replicates combined, 8330 reads were identified as chimaeras, mapping to 2837 putative junction sites. Of these, 213 were considered to be highly supported by the data, either due to being identified in at least 10 chimaeric reads or containing the full 5’ leader and TRS sequence. Adjacent donor or acceptor sites were then merged (see Materials and Methods), leaving 70 unique junctions (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of chimaeric viral reads. (A) Sashimi plot showing junctions in the EToV genome across which chimaeric RNA-seq reads were identified in EToV infected, non-drug treated samples. Chimaeric reads were defined as reads for which the intact read could not be mapped but for which the 5&#x2019; and 3&#x2019; ends could be uniquely mapped to non-contiguous regions of the EToV genome. Junctions that were either covered by at least 10 chimaeric reads (grey) and/or for which the 5&#x2019; section of the read contained the full 5&#x2019; leader sequence and leader TRS (red) were identified and adjacent positions merged. These junctions are shown as curved lines connecting the position of the 3&#x2019; end of the 5&#x2019; mapped segment of the read and the 5&#x2019; end of the 3&#x2019; mapped segment of the read. The apical height of each curved line shows the absolute number of reads spanning this junction on a log10(1+x) scale. (B) Inverted bar chart showing, for the 5&#x2019; (orange) and 3&#x2019; (blue) breakpoints for each junction, the number of chimaeric reads as a fraction of the total number of chimaeric and non-chimaeric reads at each site." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-880035169" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Analysis of chimaeric viral reads. (A) Sashimi plot showing junctions in the EToV genome across which chimaeric RNA-seq reads were identified in EToV infected, non-drug treated samples. Chimaeric reads were defined as reads for which the intact read could not be mapped but for which the 5&#x2019; and 3&#x2019; ends could be uniquely mapped to non-contiguous regions of the EToV genome. Junctions that were either covered by at least 10 chimaeric reads (grey) and/or for which the 5&#x2019; section of the read contained the full 5&#x2019; leader sequence and leader TRS (red) were identified and adjacent positions merged. These junctions are shown as curved lines connecting the position of the 3&#x2019; end of the 5&#x2019; mapped segment of the read and the 5&#x2019; end of the 3&#x2019; mapped segment of the read. The apical height of each curved line shows the absolute number of reads spanning this junction on a log&lt;sub&gt;10&lt;/sub&gt;(1+x) scale. (B) Inverted bar chart showing, for the 5&#x2019; (orange) and 3&#x2019; (blue) breakpoints for each junction, the number of chimaeric reads as a fraction of the total number of chimaeric and non-chimaeric reads at each site.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F4.medium.gif" width="440" height="286"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F4.medium.gif" width="440" height="286"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <p id="p-23" class="first-child">Analysis of chimaeric viral reads. (A) Sashimi plot showing junctions in the EToV genome across which chimaeric RNA-seq reads were identified in EToV infected, non-drug treated samples. Chimaeric reads were defined as reads for which the intact read could not be mapped but for which the 5’ and 3’ ends could be uniquely mapped to non-contiguous regions of the EToV genome. Junctions that were either covered by at least 10 chimaeric reads (grey) and/or for which the 5’ section of the read contained the full 5’ leader sequence and leader TRS (red) were identified and adjacent positions merged. These junctions are shown as curved lines connecting the position of the 3’ end of the 5’ mapped segment of the read and the 5’ end of the 3’ mapped segment of the read. The apical height of each curved line shows the absolute number of reads spanning this junction on a log<sub>10</sub>(1+x) scale. (B) Inverted bar chart showing, for the 5’ (orange) and 3’ (blue) breakpoints for each junction, the number of chimaeric reads as a fraction of the total number of chimaeric and non-chimaeric reads at each site.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">Three chimaeric junctions were identified where the first nucleotide of the corresponding read mapped to the first nucleotide of the viral genome. Of these, one junction was consistent with the previously characterised sgRNA produced via discontinuous RNA synthesis encoding the S gene (280 reads, or 3 % of total chimaeric reads) (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">12</a>). These reads spanned the entire leader-body junction of the S gene, possessing 14 - 18 nt of the 5’ UTR (i.e. the actual 5’-derived sequence is at least 14 nt, ACGUAUCUUUAGAA, comprising the so-called 6-nt leader, the leader TRS CUUUAGA, and an additional A), followed by the stretch of ORF1b just upstream of the S gene. A second set of transcripts containing 5’ leader sequence was identified by four unique reads starting with the 5’ leader (ACGUAU) and TRS sequence (CUUUAGA), where the remainder of the read mapped to the start of the N gene. This indicates that, contrary to previous reports, low levels of discontinuous RNA synthesis are used during production of the N gene negative-strand RNA. The final chimaera which included the 6 nt leader was represented by three reads. These reads included 44 - 46 nt of the 5’ UTR (i.e., significantly more than the predicted leader-TRS) followed by a sequence mapping to position 19987-19989 which is within ORF1b.</p><p id="p-25">A substantial number of additional chimaeric reads were identified, indicative of non-TRS-driven cases of discontinuous RNA synthesis, although formally it is possible that some of these are template-switching artefacts introduced during library preparation and/or sequencing. Additionally, a large number of reads spanning from the 5’ UTR to either within the N protein gene or the 3’ UTR were identified. Indeed, the only junction represented by over 1000 reads spanned nucleotides 673 to 27649; similarly the second most commonly identified junction spanned 687 to 27550 (642 reads). If chimaeric reads were predominantly a sequencing artefact, the abundance of any particular chimaera would be approximately proportional to the product of the abundances of the sequences from which the 5’ and 3’ ends of the chimaera are derived (with some variation due to sequence-specific biases), and thus a high density of chimaeras would be expected to fall entirely within the N transcript. In contrast, most of the observed chimaeric reads were between N and the 5’ UTR. The relative paucity of reads mapping to generic locations in the ORF1ab region also argues against the majority of chimaeras being simply artefactual. The 5’ UTR preference may be due to genome circularisation during negative-sense synthesis as has been proposed for coronaviruses (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>). Alternatively these may derive from autonomously replicating defective interfering RNAs, rather than multiple independent RNA translocation and reinitiation events. Such defective interfering RNAs have been extensively analysed previously and are a common complication of EToV studies (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>). Consistent with the high level of 5’UTR:N chimaeric sequences, there was high RNA-seq density throughout much of the 5’ UTR, with the 3’ extent of the region of high density coinciding approximately with the region to which a large number of the chimaeric 5’ ends mapped (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>).</p></div><div id="sec-6" class="subsection"><h3>Gene expression analysis indicates multiple pathways are perturbed by EToV infection</h3><p id="p-26">The RNA-seq data were analysed to identify genes that were differentially expressed between virus-infected and mock-infected ED cells. We identified 61 genes that were upregulated in virus-infected cells; amongst which eight gene ontology (GO) terms were overrepresented, mostly related to the nucleosome or immune responses (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). We found 24 genes that were downregulated in infected cells, amongst which four GO terms were overrepresented, two of which were related to the ribosome. We also analysed differential translational efficiency (based on the RPF to mRNA ratio) between mock- and virus-infected cells. We identified 22 genes that were translated more efficiently in infected cells; GO analysis indicated that these genes tend to encode proteins that are involved in RNA binding. Only two genes were found to be translated less efficiently in infected cells compared to mock (Supplementary Table 2 and <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>). Note that these analyses measure changes in individual genes relative to the global mean and do not inform on global changes in host transcription or translation as a result of virus infection.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Volcano plots showing the results of (A) differential transcription analysis performed using DESeq2 (64) and (B) differential translation efficiency analysis performed using Xtail (68), between cells infected with EToV (infected) and uninfected cells (mock). Genes which were expressed at significantly higher levels (FDR &#x2264; 0.05 and absolute log2(fold change) &#x2265; 1) in infected cells are highlighted in pink (transcription, A) and blue (translational efficiency, B). Genes which were expressed at significantly higher levels in mock infected cells are highlighted in green (transcription, A) and orange (translational efficiency, B). The five most significant genes in each category are labelled with the gene symbol where available and otherwise with the Ensembl gene ID. (C) Absolute log2(fold change) for all gene ontology (GO) terms which were significantly overrepresented compared to a background of all horse protein-coding genes for genes significantly more transcribed in infected cells (pink), genes significantly more efficiently translated in infected cells (blue) and genes significantly more transcribed in mock cells (green). No terms were identified for genes significantly more efficiently translated in mock cells." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-880035169" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Volcano plots showing the results of (A) differential transcription analysis performed using DESeq2 (64) and (B) differential translation efficiency analysis performed using Xtail (68), between cells infected with EToV (infected) and uninfected cells (mock). Genes which were expressed at significantly higher levels (FDR &#x2264; 0.05 and absolute log&lt;sub&gt;2&lt;/sub&gt;(fold change) &#x2265; 1) in infected cells are highlighted in pink (transcription, A) and blue (translational efficiency, B). Genes which were expressed at significantly higher levels in mock infected cells are highlighted in green (transcription, A) and orange (translational efficiency, B). The five most significant genes in each category are labelled with the gene symbol where available and otherwise with the Ensembl gene ID. (C) Absolute log&lt;sub&gt;2&lt;/sub&gt;(fold change) for all gene ontology (GO) terms which were significantly overrepresented compared to a background of all horse protein-coding genes for genes significantly more transcribed in infected cells (pink), genes significantly more efficiently translated in infected cells (blue) and genes significantly more transcribed in mock cells (green). No terms were identified for genes significantly more efficiently translated in mock cells.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F5.medium.gif" width="440" height="346"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F5.medium.gif" width="440" height="346"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <p id="p-27" class="first-child">Volcano plots showing the results of (A) differential transcription analysis performed using DESeq2 (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a>) and (B) differential translation efficiency analysis performed using Xtail (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">68</a>), between cells infected with EToV (infected) and uninfected cells (mock). Genes which were expressed at significantly higher levels (FDR ≤ 0.05 and absolute log<sub>2</sub>(fold change) ≥ 1) in infected cells are highlighted in pink (transcription, A) and blue (translational efficiency, B). Genes which were expressed at significantly higher levels in mock infected cells are highlighted in green (transcription, A) and orange (translational efficiency, B). The five most significant genes in each category are labelled with the gene symbol where available and otherwise with the Ensembl gene ID. (C) Absolute log<sub>2</sub>(fold change) for all gene ontology (GO) terms which were significantly overrepresented compared to a background of all horse protein-coding genes for genes significantly more transcribed in infected cells (pink), genes significantly more efficiently translated in infected cells (blue) and genes significantly more transcribed in mock cells (green). No terms were identified for genes significantly more efficiently translated in mock cells.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>Two additional proteins are translated from 5’ CUG-initiated ORFs</h3><p id="p-28">Our initial dataset indicated an excess density of ribosomes translating within the +2/-1 frame upstream of ORF1a and overlapping the 5’ end of ORF1a (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2A</a>). To further investigate this, we repeated the ribosome profiling using infected cells treated with translation inhibitors prior to flash freezing (harringtonine, HAR, and/or cycloheximide, CHX). HAR specifically arrests initiating ribosomes whilst allowing “run-off” of elongating ribosomes; conversely CHX stalls elongating ribosomes whilst allowing on-going accumulation at initiation sites. Our quality control analysis confirmed the datasets were of similar quality to our previous experiment (Supplementary Figures 1, 2 and 4) and mapping of the RPFs provided good coverage of the EToV genome (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Read density of Ribo-seq reads along the viral genome for EToV infected cells pretreated with (A) cycloheximide or (B) harringtonine. Red lines represent total reads per million mapped reads (RPM) at each position. Densities are smoothed with a 15-nt running mean filter and plotted on a log10(1+x) scale. Negative-sense reads (grey) are displayed below the x-axis. Each line represents a single replicate. A +12 nt offset has been applied to read 5&#x2019; end positions to map approximate P-site positions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-880035169" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Read density of Ribo-seq reads along the viral genome for EToV infected cells pretreated with (A) cycloheximide or (B) harringtonine. Red lines represent total reads per million mapped reads (RPM) at each position. Densities are smoothed with a 15-nt running mean filter and plotted on a log&lt;sub&gt;10&lt;/sub&gt;(1+x) scale. Negative-sense reads (grey) are displayed below the x-axis. Each line represents a single replicate. A +12 nt offset has been applied to read 5&#x2019; end positions to map approximate P-site positions.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F6.medium.gif" width="440" height="249"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F6.medium.gif" width="440" height="249"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6.</span> <p id="p-29" class="first-child">Read density of Ribo-seq reads along the viral genome for EToV infected cells pretreated with (A) cycloheximide or (B) harringtonine. Red lines represent total reads per million mapped reads (RPM) at each position. Densities are smoothed with a 15-nt running mean filter and plotted on a log<sub>10</sub>(1+x) scale. Negative-sense reads (grey) are displayed below the x-axis. Each line represents a single replicate. A +12 nt offset has been applied to read 5’ end positions to map approximate P-site positions.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">This Ribo-seq data confirmed translation of two ORFs located within the so-called 5’ UTR and overlapping the 5’ end of ORF1a. We have termed these U1 (80 codons) and U2 (258 codon). We predict that translation of both U1 and U2 is initiated from CUG codons, as a close inspection indicated that ribosomes accumulated at these two sites (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). It must be noted that pretreatment with CHX or HAR can introduce artefacts into ribosome profiling data: CHX can lead to an excess of RPF density over ~30 codons following initiation sites when cells are stressed (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>). It has also been suggested that both drugs can promote upstream initiation due to scanning pre-initiation complexes stacking behind ribosomes paused at canonical initiation sites (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>). However, the distance between the U1 CUG, the U2 CUG and the ORF1a initiation site, besides observation of efficient translation of U2 downstream of the ORF1a initiation site makes these artefacts unlikely to be significant confounding factors in the case of U1 and U2.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Read density of Ribo-seq reads across (A) U1, U2 and ORF1a; and (B) the U1 ORF and surrounding regions, for EToV infected cells with no drug treatment or with cycloheximide or harringtonine pretreatment. Pink: reads in phase 0; yellow: phase - 1; blue: phase +1. Graphs show total reads per million mapped reads (RPM) at each position. In (A) densities are smoothed with a 15-nt running mean filter while (B) shows the RPM counts at single-nt resolution. Each plot represents a single replicate. A +12 nt offset has been applied to read 5&#x2019; end positions to map approximate P-site positions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-880035169" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Read density of Ribo-seq reads across (A) U1, U2 and ORF1a; and (B) the U1 ORF and surrounding regions, for EToV infected cells with no drug treatment or with cycloheximide or harringtonine pretreatment. Pink: reads in phase 0; yellow: phase - 1; blue: phase +1. Graphs show total reads per million mapped reads (RPM) at each position. In (A) densities are smoothed with a 15-nt running mean filter while (B) shows the RPM counts at single-nt resolution. Each plot represents a single replicate. A +12 nt offset has been applied to read 5&#x2019; end positions to map approximate P-site positions.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F7.medium.gif" width="440" height="412"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7." src="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F7.medium.gif" width="440" height="412"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7.</span> <p id="p-31" class="first-child">Read density of Ribo-seq reads across (A) U1, U2 and ORF1a; and (B) the U1 ORF and surrounding regions, for EToV infected cells with no drug treatment or with cycloheximide or harringtonine pretreatment. Pink: reads in phase 0; yellow: phase - 1; blue: phase +1. Graphs show total reads per million mapped reads (RPM) at each position. In (A) densities are smoothed with a 15-nt running mean filter while (B) shows the RPM counts at single-nt resolution. Each plot represents a single replicate. A +12 nt offset has been applied to read 5’ end positions to map approximate P-site positions.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">Revisiting our first non-drug-treated dataset, we calculated the RPF densities and translational efficiencies within the U1 and U2 ORFs (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>). U1 has a higher translational efficiency than any of the other ORFs translated from genomic RNA, whereas U2 has a translational efficiency similar to that of ORF1a.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relative translation efficiencies for U1, U2, ORF1a and ORF1b. To reduce misassignment of reads in the U2/ORF1a overlap region, for all ORFs only reads mapping in the predominant phase (i.e. mapping to first positions of codons) were included. Ribo-seq densities were divided by the ORF1ab RNA-seq densities for the corresponding paired sample. For each ORF, the two bars represent two repeats." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-880035169" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Relative translation efficiencies for U1, U2, ORF1a and ORF1b. To reduce misassignment of reads in the U2/ORF1a overlap region, for all ORFs only reads mapping in the predominant phase (i.e. mapping to first positions of codons) were included. Ribo-seq densities were divided by the ORF1ab RNA-seq densities for the corresponding paired sample. For each ORF, the two bars represent two repeats.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F8.medium.gif" width="440" height="436"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8." src="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F8.medium.gif" width="440" height="436"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8.</span> <p id="p-33" class="first-child">Relative translation efficiencies for U1, U2, ORF1a and ORF1b. To reduce misassignment of reads in the U2/ORF1a overlap region, for all ORFs only reads mapping in the predominant phase (i.e. mapping to first positions of codons) were included. Ribo-seq densities were divided by the ORF1ab RNA-seq densities for the corresponding paired sample. For each ORF, the two bars represent two repeats.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">To assess the coding potential of U1, we calculated the ratio of non-synonymous to synonymous substitutions (dN/dS), where dN/dS &lt; 1 indicates selection against non-synonymous substitutions which is a strong indicator that a sequence encodes a functional protein. Application of codeml (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>) to a codon alignment of eight torovirus U1 nucleotide sequences resulted in a dN/dS estimate of 0.31 ± 0.08, indicating that the U1 ORF is likely to encode a functional protein. MLOGD (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>) uses a principle similar to the dN/dS statistic but also accounts for conservative amino acid substitutions (i.e. similar physico-chemical properties) being more probable than non-conservative substitutions in biologically functional polypeptides. MLOGD 3-frame “sliding window” analysis of a full-genome alignment revealed a strong coding signature in the known protein-coding ORFs (as expected) and also in the U1 ORF (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a>).</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Coding potential statistics for the torovirus genome. A map of the torovirus genome is shown at top. Breda virus (AY427798.1) was used as the reference genome for this analysis since EToV has a deletion in the HE gene. In Breda virus, U1 is in-frame with ORF1a due to a 2-nt insertion relative to EToV in the short non-coding region between U1 and U2. The next four panels show an analysis of synonymous site conservation in the concatenated coding ORFs (with the reading frame of the longer ORF being used wherever two ORFs overlap). Red lines show the probability that the degree of conservation within a given window (25- or 65-codons as indicated) could be obtained under a null model of neutral evolution at synonymous sites, whereas brown lines depict the absolute amount of conservation as represented by the ratio of the observed number of substitutions within a given window to the number expected under the null model. Greatly enhanced synonymous site conservation is seen in the region of ORF1a that is overlapped by the U2 ORF. The next three panels show MLOGD coding potential scores and stop codon plots for each of the three reading frames. The positions of stop codons are shown for each of the eight torovirus sequences mapped onto the Breda virus reference sequence coordinates. Note the conserved absence of stop codons in the U1 and U2 ORFs. MLOGD was applied in a 40-codon sliding-window (5-codon step size). Positive scores indicate that the sequence is likely to be coding in the given reading frame. Note the positive scores within the U1 and U2 ORFs besides the previously known ORFs. The bottom panel (green line) indicates the total amount of phylogenetic divergence contributing to the analyses at each alignment position (regions containing alignment gaps have reduced summed divergence leading to reduced statistical power). Pink regions in the stop codon plots (e.g. EToV sequence in the HE region) indicate regions excluded from the analyses due to poor or locally out-of-frame mapping to the Breda reference sequence (see Firth, 2014 for details)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-880035169" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Coding potential statistics for the torovirus genome. A map of the torovirus genome is shown at top. Breda virus (AY427798.1) was used as the reference genome for this analysis since EToV has a deletion in the HE gene. In Breda virus, U1 is in-frame with ORF1a due to a 2-nt insertion relative to EToV in the short non-coding region between U1 and U2. The next four panels show an analysis of synonymous site conservation in the concatenated coding ORFs (with the reading frame of the longer ORF being used wherever two ORFs overlap). Red lines show the probability that the degree of conservation within a given window (25- or 65-codons as indicated) could be obtained under a null model of neutral evolution at synonymous sites, whereas brown lines depict the absolute amount of conservation as represented by the ratio of the observed number of substitutions within a given window to the number expected under the null model. Greatly enhanced synonymous site conservation is seen in the region of ORF1a that is overlapped by the U2 ORF. The next three panels show MLOGD coding potential scores and stop codon plots for each of the three reading frames. The positions of stop codons are shown for each of the eight torovirus sequences mapped onto the Breda virus reference sequence coordinates. Note the conserved absence of stop codons in the U1 and U2 ORFs. MLOGD was applied in a 40-codon sliding-window (5-codon step size). Positive scores indicate that the sequence is likely to be coding in the given reading frame. Note the positive scores within the U1 and U2 ORFs besides the previously known ORFs. The bottom panel (green line) indicates the total amount of phylogenetic divergence contributing to the analyses at each alignment position (regions containing alignment gaps have reduced summed divergence leading to reduced statistical power). Pink regions in the stop codon plots (e.g. EToV sequence in the HE region) indicate regions excluded from the analyses due to poor or locally out-of-frame mapping to the Breda reference sequence (see Firth, 2014 for details).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F9.medium.gif" width="440" height="319"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9." src="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F9.medium.gif" width="440" height="319"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/07/296996/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9.</span> <p id="p-35" class="first-child">Coding potential statistics for the torovirus genome. A map of the torovirus genome is shown at top. Breda virus (AY427798.1) was used as the reference genome for this analysis since EToV has a deletion in the HE gene. In Breda virus, U1 is in-frame with ORF1a due to a 2-nt insertion relative to EToV in the short non-coding region between U1 and U2. The next four panels show an analysis of synonymous site conservation in the concatenated coding ORFs (with the reading frame of the longer ORF being used wherever two ORFs overlap). Red lines show the probability that the degree of conservation within a given window (25- or 65-codons as indicated) could be obtained under a null model of neutral evolution at synonymous sites, whereas brown lines depict the absolute amount of conservation as represented by the ratio of the observed number of substitutions within a given window to the number expected under the null model. Greatly enhanced synonymous site conservation is seen in the region of ORF1a that is overlapped by the U2 ORF. The next three panels show MLOGD coding potential scores and stop codon plots for each of the three reading frames. The positions of stop codons are shown for each of the eight torovirus sequences mapped onto the Breda virus reference sequence coordinates. Note the conserved absence of stop codons in the U1 and U2 ORFs. MLOGD was applied in a 40-codon sliding-window (5-codon step size). Positive scores indicate that the sequence is likely to be coding in the given reading frame. Note the positive scores within the U1 and U2 ORFs besides the previously known ORFs. The bottom panel (green line) indicates the total amount of phylogenetic divergence contributing to the analyses at each alignment position (regions containing alignment gaps have reduced summed divergence leading to reduced statistical power). Pink regions in the stop codon plots (e.g. EToV sequence in the HE region) indicate regions excluded from the analyses due to poor or locally out-of-frame mapping to the Breda reference sequence (see Firth, 2014 for details).</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">We previously predicted the existence of U2 via an analysis of coding potential and synonymous site conservation across the two torovirus genomes available at that time (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>). Six additional torovirus genome sequences have now become available. We therefore extended the bioinformatics analysis using all eight currently available torovirus genome sequences (<a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure 9</a>). Since the U2 ORF overlaps ORF1a, leading to constraint on dS, the dN/dS analysis is not appropriate for U2. MLOGD analysis indicated that the U2 ORF has a higher coding potential than the corresponding part of ORF1a (<a id="xref-fig-9-3" class="xref-fig" href="#F9">Figure 9</a>). Overlapping genes are thought mainly to evolve through “overprinting” of an ancestral gene by the <em>de novo</em> gene (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>). The <em>de novo</em> gene product is often an accessory protein and often disordered (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>). Interestingly, the fragment of pp1a encoded by the region of ORF1a that is overlapped by U2 has no tblastn (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>) nor HHpred (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>) homologues outside of the <em>Torovirus</em> genus. Thus, it is unclear which of U2 and the N-terminal domain of pp1a is ancestral. To provide further comparative genomic evidence for the functionality of U2, we used synplot2 to assess conservation at synonymous sites in the ORF1a reading frame, since overlapping functional elements are expected to place extra constraints on synonymous site evolution (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>). Consistent with the earlier 2-sequence analysis (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>), synplot2 revealed greatly enhanced ORF1a-frame synonymous-site conservation in a region coinciding precisely with the conserved absence of stop codons that defines the U2 ORF (<a id="xref-fig-9-4" class="xref-fig" href="#F9">Figure 9</a>), with the mean rate of synonymous substitutions in that region being 0.20 of the genome average. Summed over the 230-codon overlap region, the probability that the observed level of conservation would occur by chance is <em>p</em> = 6.5 × 10<sup>−40</sup>.</p><p id="p-37">Both U1 and U2 are conserved in all eight torovirus sequences with no variation in length or initiation or termination position (<a id="xref-fig-14-1" class="xref-fig" href="#F14">Supplementary Figure 5</a>). In all sequences, U1 and U2 begin with a CUG codon in a strong initiation context (‘A’ at −3 for U1, and ‘A’ at −3 and ‘G’ at +4 for U2) (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>). The U1 protein is predicted to contain two central transmembrane domains and has a C-terminus containing many charged amino acids. The U2 protein is predicted to form alternating α helix and antiparallel β sheet domains, however no structural homologs were found through searches of public databases (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>–<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>). Their function(s) will be the topic of future work.</p></div></div><div class="section" id="sec-8"><h2 class="">Discussion</h2><div id="sec-9" class="subsection"><h3>RNA-seq reveals the complexity of torovirus transcription mechanisms</h3><p id="p-38">The factors influencing which transcriptional mechanism is utilised for the synthesis of each sgRNA during torovirus replication have not been elucidated. The EToV genome contains seven occurrences of the canonical TRS motif (CUUUAGA): within the 5’ UTR (leader TRS), the end of U1, central ORF1a, central ORF1b, and immediately before the M, HE and N ORFs (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Consistent with experimental evidence (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">12</a>), we did not identify any chimaeric transcripts encompassing the body TRS of M or HE, or those within ORF1b or ORF1a. It appears that these sites do not stimulate interruption of negative strand RNA synthesis followed by subsequent re-pairing and reinitiation. The nucleotides flanking the N, M and HE TRSs are semi-conserved (<a id="xref-fig-15-1" class="xref-fig" href="#F15">Supplementary Figure 6</a>) and it has been suggested previously that the motif definition should be extended to cACN<sub>3-4</sub>CUUUAGA to reflect this (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>). It is likely that these flanking nucleotides contribute to the degree of utilisation.</p><p id="p-39">For the S gene, the chimaeric junction occurs within the run of uridines 3’-adjacent to the hairpin (<a id="xref-fig-15-2" class="xref-fig" href="#F15">Figure S6I</a>). Our results lend support to the hypothesis suggested previously that a short conserved RNA hairpin, 174 nt upstream of the AUG start codon of the EToV S protein gene, mediates discontinuous extension of negative strand RNA synthesis to produce this sgRNA (<a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">12</a>) (<a id="xref-fig-15-3" class="xref-fig" href="#F15">Supplementary Figure 6</a>). The predicted hairpin structure was not present in S gene chimaeric reads, indicating that translocation may indeed be prompted by the RdRp encountering a physical block after synthesising the reverse complement of the S ORF. This is in contrast to the coronaviral and arteriviral mechanism, wherein RNA structures are insufficient and an accompanying body TRS is required to act as a transcriptional attenuation signal, prompting translocation and re-pairing of the nascent RNA. We cannot unambiguously identify which nucleotides are templated before or after the translocation event, as a GUUU sequence maps to genomic RNA on either side of the breakpoint.</p><p id="p-40">The leader-TRS chimaeric reads mapping to the N protein gene initially appear consistent with the coronaviral and arteriviral mechanism of TRS-driven discontinuous RNA synthesis. However close inspection indicated that the homologous motif mediating copy-choice recombination-like translocation and re-pairing of RNA strands was actually a short AGAA sequence, not the true TRS (tetranucleotides underlined in <a id="xref-fig-15-4" class="xref-fig" href="#F15">Figures S6A</a> and <a id="xref-fig-15-5" class="xref-fig" href="#F15">S6G</a>). This would result in the nascent anti-TRS mispairing with the leader TRS; two nucleotides are “skipped” once reinitiation occurs. This may explain why the discontinuous mechanism is utilised so rarely for this mRNA.</p><p id="p-41">This leads to the suggestion that homology between any two sites may be sufficient to induce discontinuous RNA synthesis, i.e. that provided adequate sequence homology exists, the nascent RNA strand may re-pair with upstream sites within the genomic RNA regardless of the presence of a predefined TRS. This is consistent with the 5’ UTR-ORF1b chimaeric transcripts, which again revealed a particular sequence that could be templated from either region, in this case AACCUUA rather than the TRS.</p><p id="p-42">If TRS sequence-specificity is not required to stimulate EToV discontinuous RNA synthesis, it is presumably constrained by alternative roles. The highly conserved nature of the canonical leader, M, HE and N TRS (CUUUAG[A/U]) across all torovirus genomes (<a id="xref-fig-15-6" class="xref-fig" href="#F15">Supplementary Figure 6</a>) suggests it is not tolerant to mutations, however this has not been formally confirmed. Lack of conservation of the EToV U1, ORF1a and ORF1b TRS sequences is consistent with them not being functionally relevant. Our results indicate this essential nature is likely due to a role in transcriptional termination, as we did not identify a significant role of this motif in the generation of chimaeric transcripts. Conversely, the upstream region of the “extended” TRS (cACN<sub>3-4</sub>CUUUAGA) is tolerant to modifications, reflecting the variable nature within sequences; even when this spacer is extended to six nucleotides, transcripts are still detectable at 20 % of WT levels (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a>). Again, this is consistent with a role in termination rather than a requirement for re-pairing with upstream sequences. The canonical TRS sequences also presumably contribute to subgenomic promoter recognition, as the initial CAC is essential though the adenylate is the first nucleotide on all positive-strand subgenomic transcripts (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">34</a>). Initiation of sgRNA transcription at AC dinucleotides is also found in the roniviruses (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>). It may be that in these <em>Nidovirales</em> families, the conserved TRS is utilised primarily for signalling transcriptional termination followed by promoter recognition, and any use for discontinuous RNA synthesis is merely a byproduct of RdRp promiscuity.</p><p id="p-43">The unique combination of discontinuous and non-discontinuous mechanisms within the one virus so far appears unique to the mammalian toroviruses. The one bafinivirus isolated to date (white bream virus, family <em>Coronaviridae</em>, subfamily <em>Torovirinae</em>, genus <em>Bafinivirus</em>) has an extended TRS sequence (CA[G/A]CACUAC) which is not conserved with the mammalian toroviruses analysed in this study. Bafinivirus replication produces three sgRNAs which share an identical 42-nt leader also found at the far 5’ terminus of the genome, indicating this species utilises discontinuous RNA synthesis in a manner similar to the corona- and arteriviruses (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>). However there was preliminary evidence that two of the three sgRNAs exhibit diversity in their junction sites, suggesting the anti-TRS may bind to multiple sites within the 5’ leader during strand transfer, consistent with our suggestion that whilst a threshold level of homology is required this is not limited to particular primary sequences. This is reflected in the fact that the bafinivirus leader-TRS is not fully identical to the body TRSs.</p><p id="p-44">It is not known which mechanism was utilised by the last common ancestor of nidovirids, and thus which represents divergence from the original model. It has been suggested that convergent evolution has resulted in the mechanism for discontinuous negative strand synthesis arising multiple times within the <em>Nidovirales</em>. Similarly, whether the initial role of the TRS motif was to merely stimulate the attenuation of RNA synthesis or to direct the discontinuous mechanism is not known. Our data suggests that transcription mechanisms in the <em>Nidovirales</em> fall into multiple categories, each requiring a distinct role of the TRS: (i) homology-driven reinitiation (canonical discontinuous RNA synthesis, as seen in coronaviruses and arteriviruses and to a low extent, EToV N protein-coding mRNAs); (ii) structure-driven discontinuous transcription (EToV S <span class="underline">protein</span> gene); and (iii) transcription termination (EToV M, HE and the majority of N protein-coding transcripts). These mechanisms all require a RdRp which is prone to translocating when even relatively short homologous sequences are present, potentially leading to a large number of irrelevant transcripts being produced (as previously observed in an arterivirus (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>)) and also facilitating the production of defective interfering RNAs (<a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">34</a>) and recombinant strains (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>).</p></div><div id="sec-10" class="subsection"><h3>Effects upon the host: transcriptional and translational differential expression</h3><p id="p-45">The differential transcription analysis indicated that infection with EToV induces increased transcription of multiple genes, the products of which are significantly more likely than random to be involved in (i) nucleosome function and DNA binding, and (ii) immune responses to infection than genes which were not differentially transcribed. Some of the identified GO categories, including cytokine signalling, innate immune responses and ribosome biogenesis have been identified in previous RNA-seq analyses of various coronaviruses (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>). Similarly, although differential translational analyses or proteomic studies have not been conducted upon toroviruses, some of the identified proteins have been recognised as being incorporated into nidovirid virions (for example, TCP-1 and multiple heat shock proteins within arterivirus particles) (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>). Others have been identified as being upregulated upon infection with coronaviruses, such as the solute carrier family 25 members (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>). Notably, both poly(C) and poly(A) binding proteins were preferentially translated in infected cells; these have been previously identified as interaction partners of arteriviral non-structural protein 1β and contribute to viral RNA replication (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>). It therefore appears that torovirus infection induces a similar host response to many nidovirids.</p><p id="p-46">To the best of our knowledge, this is the first analysis of differential gene expression following infection with a torovirus. It would be of interest to repeat this analysis at later time points, as a previous study found that EToV-mediated global inhibition of host protein synthesis was only detectable at 16 h.p.i. (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">38</a>). The same study found induction of both the intrinsic and extrinsic apoptotic pathways was evident only by 24 h.p.i. (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>). It is clear that the transcriptional and translational profile of the host cell may differ significantly throughout the course of infection. Additionally, it must be noted that the horse (<em>Equus caballus</em>) genome is not highly annotated and thus many Ensembl gene identifications do not possess an annotated orthologue, a limiting factor in our analysis.</p></div><div id="sec-11" class="subsection"><h3>What is the function of U1 and U2?</h3><p id="p-47">The current lack of a published reverse genetics system to study torovirus replication means we are unable to perform targeted mutagenesis. This would enable definitive experimental confirmation that U1 and U2 are translated from their respective CUG codons, followed by phenotypic analysis of knock-out mutants. However the comparative genomic analysis together with the accumulation of ribosomes on both CUG codons is highly suggestive of this being the site of initiation; CUG has previously been reported as the most commonly utilised non-AUG initiation codon in mammalian systems (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>). In the case of U1, the coding sequence contains no AUG codons (in any frame), a situation that would facilitate pre-initiation ribosomes to continue scanning to the U2 CUG and the ORF1a AUG initiation sites (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>). It remains a possibility that U2 translation initiates at a downstream AUG, however the only in-frame AUG is located 336 nt downstream of our presumed start site and is in a poor initiation context (‘C’ at −3) and 3’ of the ORF1a AUG. We are therefore confident that the CUG codons that were identified in the ribosome profiling data represent the genuine translational start sites.</p><p id="p-48">The ORFs of both U1 and U2 are intact in all torovirus genomic sequences that we have analysed to date, including bovine (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>, <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>), caprine and porcine isolates (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>). Most of the U2 ORF is constrained by the fact that the sequence must also retain ORF1a coding capacity in another frame. U1 is not under such limitations, although it is likely that the viral genome must maintain specific 5’ UTR structures to facilitate viral replication. Previous investigations utilising defective interfering RNAs have confirmed that no more than the first 604 nt of the 5’ UTR and the entirety of the 3’ UTR are sufficient to allow both positive and minus strand RNA synthesis (<a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">34</a>); it is notable that this region only includes one-third of the U1 ORF (which starts at nucleotide 524) and hence only this subdomain would be constrained by maintaining two distinct functional roles. We suggest that the so-called 5’ UTR is actually limited to 523 nt preceding the CUG of U1, and the remainder of U1 and U2 is not under pressure to maintain <em>cis-</em>replication elements.</p><p id="p-49">Neither ORF could be identified within the white bream virus genome, a bafinivirus that constitutes another genus within the subfamily <em>Torovirinae</em> (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">35</a>), although the lack of multiple bafinivirus sequences makes comparative genomic analysis impossible.</p><p id="p-50">The function(s) of the proteins encoded by both U1 and U2 remain to be elucidated. Despite the relatively large size of the U2 protein (~30 kDa), after extensive database searches no structural homologs were identified. By comparison, the U1 protein is small (~10 kDa), highly basic (pI = 10.4) and possesses many of the predicted features of a double-spanning transmembrane protein, including two hydrophobic stretches separated by a ‘hinge’ and a predicted coiled-coil tertiary topology. Based on structural similarity to known proteins, one potential function might be a virally encoded ion channel (viroporin) embedded in either intracellular or plasma membranes. It is possible that U1 plays a similar role in toroviruses to that of the coronaviral and arteriviral E proteins, which have no known toroviral homologue. The coronavirus E protein is a small transmembrane protein (~10 kDa) which possesses ion channel activity and is required for virion assembly, forming a pentamer that traverses the viral envelope (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>). E proteins also possess a membrane-proximal palmitoylated cysteine residue, which is a predicted (and conserved) posttranslational modification for U1 (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a>).</p><p id="p-51">Alternatively viroporin activity may be mediated by a small, basic double-transmembrane protein, the ORF of which is embedded within the EToV N gene in the +1 frame (with respect to N). An analogous “N+1” protein has been identified in some group II coronaviruses and is postulated to play a structural role, however it is not essential for replication (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>, <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>). Neither our ribosome profiling nor comparative genomic analysis provides evidence that this ORF is utilised in toroviruses. We did not observe ribosomes translating in this frame in either the initial dataset or the drug-treated samples (although Ribo-seq may not always detect poorly translated overlapping genes); further, the ORF is not preserved in all torovirus genomes.</p><p id="p-52">Our data has revealed that the transcriptional landscape of a prototypic torovirus is complex and driven by many factors beyond the canonical “multi-loci TRS” model of coronaviruses. The development of a torovirus reverse genetics system would allow manipulation of potential translocation-inducing sequences and allow us to elucidate which features of the toroviral TRS cause them to act as terminators of RNA synthesis, rather than consistently inducing homology-assisted recombination. Our accompanying translational analysis has revealed two conserved novel ORFs, and has shortened the EToV 5’ UTR to a mere 523 nt. Together these data provide an insight into the molecular biology of the replication cycle of this neglected pathogen and highlight the disparities between the families of the <em>Nidovirales</em>.</p></div></div><div class="section" id="sec-12"><h2 class="">Materials and Methods</h2><div id="sec-13" class="subsection"><h3>Virus isolates</h3><p id="p-53">A plaque-purified isolate of equine torovirus, Berne strain (isolate P138/72) (EToV) was kindly provided by Raoul de Groot (Utrecht University) and cultured in equine dermis (ED) cells. This virus was initially isolated from a symptomatic horse in 1972 (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>). ED cells were maintained in Dulbecco’s modified Eagle’s medium (Invitrogen), supplemented with 10 % foetal calf serum, 100 IU/mL penicillin, 100 µg/mL streptomycin, 1 mM non-essential amino acids, 25 mM HEPES and 1 % L-glutamine in a humidified incubator at 37°C with 5% CO<sub>2</sub>.</p></div><div id="sec-14" class="subsection"><h3>RNA sequencing and ribosome profiling</h3><p id="p-54">ED cells were infected with EToV for 1 hour (h) in serum-free media (MOI = 0.1) and flash-frozen in liquid nitrogen at 8 h post infection (h.p.i.) prior to either RNA isolation or ribosome purification for profiling. Cells were either not pretreated or, where stated, were treated with a final concentration of 100 μg/mL cycloheximide (CHX) for 2 minutes (Sigma-Aldrich) or 2 μg/mL of harringtonine for 3 minutes (LKT Laboratories) followed by CHX for 2 minutes, before flash-freezing. RNA and ribosomes were harvested according to previously published protocols (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">15</a>, <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>) with minor modifications. Following either RPF or RNA isolation, duplex-specific nuclease was not utilised but instead rRNA was depleted with the RiboZero [human/mouse/rat] kit (Illumina). Libraries were prepared and sequenced using the NextSeq500 platform (Illumina).</p></div><div id="sec-15" class="subsection"><h3>Bioinformatic analysis of Ribo-seq and RNA-seq data</h3><p id="p-55">Both Ribo-seq and RNA-seq reads were demultiplexed and adaptor sequences trimmed using the FASTX-Toolkit (hannonlab.cshl.edu/fastx_toolkit/). Reads shorter than 25 nt after trimming were discarded. Bowtie (version 1.2.1.1) databases were generated as follows. Horse ribosomal RNA (rRNA) sequences were downloaded from the National Center for Biotechnology Information (NCBI) Entrez Nucleotide database (accessions <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=EU081775.1&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom">EU081775.1</a>, NR_046271.1, NR_046309.2, EU554425.1, XM_014728542.1 and FN402126.1) (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>). As the full-length virus RNA (vRNA) reference genome was not available for EToV, a reference was constructed from the following overlapping segments available from Entrez Nucleotide: DQ310701.1 (positions 1-14531), X52374.1 (13475–21394), X52506.1 (21250–26086), X52505.1 (26054–26850), X52375.1 (26784–27316) and D00563.1 (27264–279923). Horse messenger RNA (mRNA) sequences from EquCab2.0 (GCF_000002305.2) were downloaded from NCBI RefSeq (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>). Horse non-coding RNA (ncRNA) sequences were obtained from Ensembl release 89 (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>) and combined with horse transfer RNA (tRNA) sequences from GtRNADB (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>). Horse genomic DNA (gDNA) was obtained from Ensembl release 89. All horse sequences were from the EquCab2.0 genome build. Trimmed reads were then mapped sequentially to the rRNA, vRNA, mRNA and ncRNA databases using bowtie version 1.2.1.1 (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>), with parameters -v 2 --best (i.e. maximum 2 mismatches, report best match), with only unmapped reads passed to each following stage. Reads that did not align to any of the aforementioned databases were then mapped to the host gDNA using STAR version 2.5.4a (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>), again allowing a maximum of 2 mismatches per alignment. Remaining reads were classified as unmapped.</p><p id="p-56">Ribo-seq density and RNA-seq density were calculated for each gene in the EToV genome (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8</a>). To normalise for different library sizes, reads per million mapped reads (RPM) values were calculated using the sum of positive-sense virus RNA reads and host RefSeq mRNA reads as the denominator. In order to standardise the regions used to calculate RNA-seq and Ribo-seq density, the following regions were selected: ORF1a, start codon (position 882) to 5’ end of frameshift site (position 14518); ORF1b, 3’ end of frameshift site (position 14525) to 5’ end of the S gene hairpin (position 21118); all other ORFs, initiation codon to termination codon. For U2, a region overlapping with ORF1a was used because only 46 bases are unique to U2 and, for <a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8</a>, the ORF1a coordinates were updated to exclude the region which overlaps with U2, giving a range from 1552 to 21394. In addition, for all ORFs, only Ribo-seq reads mapping to the predominant phase (i.e. reads mapping to the first positions of codons) were used, as this should greatly diminish misassignment of ORF1a-translating ribosomes to U2 or <em>vice versa</em>. Reads mapping to the first five codons at the 5’ end of each region or the last six codons at the 3’ end of each region were excluded. For subgenomic RNAs, RNA-seq density was calculated for the same regions as described for Ribo-seq. For the genomic RNA the regions for ORF1a and ORF1b were combined into the interval from the start codon of ORF1a (position 882) to the 5’ end of the S gene hairpin (position 21118). Ribo-seq and RNA-seq densities were calculated as the number of reads per million mapped reads for which the 5’ end maps to each region, divided by the length of the region in nt, multiplied by 1000 (i.e. RPKM). For RNA-seq, a decumulation strategy was used to subtract the estimated RNA-seq density for longer overlapping genomic and subgenomic transcripts that would contribute to the RNA-seq density measured for each of the 3’ ORFs: the genomic RNA-seq density was subtracted from all subgenomic densities, and then the RNA-seq densities of overlapping “upstream” subgenomic transcripts were iteratively subtracted from “downstream” regions (e.g. RNA-seq density in the unique region of M was subtracted from HE, and this was subtracted from N). Translation efficiency for each gene was calculated as Ribo-seq density / decumulated RNA-seq density. Translational efficiencies for HE could not be accurately estimated as the low expression of the HE transcript made the decumulation procedure for HE susceptible to noise.</p><p id="p-57">Read length distributions were calculated for Ribo-seq and RNA-seq reads mapping to positive-sense host mRNA annotated CDSs or to the positive- or negative-sense EToV genome (<a id="xref-fig-10-2" class="xref-fig" href="#F10">Supplementary Figure 1</a>). Histograms of host mRNA Ribo-seq and RNA-seq 5’ end positions relative to initiation and termination codons (<a id="xref-fig-11-3" class="xref-fig" href="#F11">Supplementary Figure 2</a>, <a id="xref-fig-12-2" class="xref-fig" href="#F12">Supplementary Figure 3</a>) were derived from reads mapping to mRNAs with annotated CDSs ≥ 450 nt in length and annotated 5’ and 3’ UTRs ≥ 60 nt in length. Host mRNA Ribo-seq and RNA-seq phasing distributions (<a id="xref-fig-13-2" class="xref-fig" href="#F13">Supplementary Figure 4</a>) were calculated taking into account interior regions of annotated coding ORFs only (specifically, reads for which the 5’ end mapped between the first nucleotide of the initiation codon and 30 nt 5’ of the termination codon) in order to exclude reads on or near initiation or termination codons. For viral genome coverage plots, but not for meta-analyses of host RefSeq mRNA coverage, mapping positions of RPF 5’ ends were offset + 12 nt to approximate the location of the ribosomal P-site (<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">15</a>).</p></div><div id="sec-16" class="subsection"><h3>Analysis of viral transcripts</h3><p id="p-58">The EToV (Berne isolate) genome sequence was confirmed by <em>de novo</em> assembly of unmapped and vRNA reads from the infected RNA-seq samples. Assembly was performed using Trinity (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>) with the default settings for stranded single ended (--SS_lib type “F”) data. Viral contigs were identified using BLASTN (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a>) against a database of EToV reference sequences based on the NCBI records listed above. The viral contig was aligned to the reference using the MAFFT L-INS-i method (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>).</p><p id="p-59">Chimaeric reads were classified as reads for which the entire read mapped uniquely to the viral genome, with no mismatches, after adding a single breakpoint, with a minimum of 12 nt mapping on either side of the breakpoint, at least 5 nt apart. To identify such reads, all unmapped reads were split into two sub-reads at every possible position ≥12 nt from either end and these sub-reads were mapped to the viral genome using bowtie with no mismatches and no multimapping permitted. Transcription junctions were defined as “donor/acceptor” pairs that were either supported by at least 10 chimaeric reads or contained the entire 5’ leader and TRS sequence in the 5’ segment of the read. At some positions single nucleotide resolution for the chimaeric break-point could not be established; where reads were found to break at adjacent possible positions these positions were merged to give a short region containing the breakpoint. The number of non-chimaeric reads spanning each donor and acceptor site was calculated as the number of reads which overlapped the site by at least 12 nt in either direction (as chimaeric reads overlapping the site by &lt; 12 nt are not detectable). The proportion of chimaeric reads at each “donor” or “acceptor” site is therefore the number of chimaeric reads with a breakpoint at the site divided by this number plus the number of non-chimaeric reads spanning the site (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4B</a>).</p><p id="p-60">To visualise TRS conservation, multiple sequence alignments were generated using Clustal Omega with default parameters (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a>). RNA structure was predicted using RNA-Alifold (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a>) and visualised using VARNA (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a>).</p></div><div id="sec-17" class="subsection"><h3>Differential gene expression analysis</h3><p id="p-61">For analysis of host differential expression between non-drug treated infected and mock-infected cells, all reads which did not map to rRNA or vRNA were mapped to the EquCab2.0 reference genome and annotations (Ensembl release 89) using STAR (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">57</a>) with a maximum of two mismatches and removal of non-canonical, non-annotated splice junctions. Read counts were generated using HTSeq 0.8.0 (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a>). For differential transcription analysis, gene level counts were generated across the Ensembl release 89 EquCab2.0 gtf file, filtered to include only protein-coding genes. For differential translation efficiency analysis only coding regions (CDS) were considered: both RNA-seq and Ribo-seq counts were generated at CDS level using intersection-strict mode, based on the same annotation set. Multimapping reads were excluded from both analyses. Differential transcript abundance analysis was performed using the standard DESeq2 (<a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">64</a>) pipeline described in the vignette. Genes to which &lt;10 reads mapped were discarded and shrinkage of log<sub>2</sub> fold changes for lowly expressed genes was performed using the lfcshrink method of DESeq2. All recommended quality control plots were inspected, and no major biases were identified in the data. False discovery rate (FDR) values were calculated using the R fdrtool package (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a>). Genes with a log<sub>2</sub> fold change &gt;1 and an FDR less than 0.1 were considered to be differentially expressed. Gene ontology (GO) term enrichment analysis (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">66</a>) was performed against a background of all horse protein-coding genes in the Ensembl gtf using a Fisher Exact Test and corrected for multiple testing with a Bonferroni correction. GO annotations for horse genes were downloaded from BiomaRt (Ensembl release 90) (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">67</a>). Differential translational efficiency analysis was carried out using the CDS counts table, normalised using the DESeq2 “sizeFactors” technique. Similar to the differential transcription analysis, genes to which &lt;10 reads mapped were discarded. Again all recommended quality control plots for DESeq2 were inspected and no major biases were identified in the data. Differential translation efficiency analysis was performed using Xtail (<a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">68</a>), following the standard pipeline described in the vignette. <em>P</em>-values were adjusted automatically within Xtail using the Benjamini–Hochberg method. Genes with a log<sub>2</sub> fold change &gt;1 and an adjusted <em>p</em>-value less than 0.1 were considered to be differentially translated. GO enrichment analysis was performed as described for the differential transcript abundance analysis.</p></div><div id="sec-18" class="subsection"><h3>Comparative genomics</h3><p id="p-62">The Genbank accession numbers utilised for comparative genomic analysis were as follows: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=DQ310701.1&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom">DQ310701.1</a> (Berne virus), AY427798.1 (Breda virus) (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">45</a>), KR527150.1 (goat torovirus), JQ860350.1 (porcine torovirus) (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">47</a>), KM403390.1 (porcine torovirus) (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">69</a>), LT900503.1 (porcine torovirus), LC088094.1 (bovine torovirus) and LC088095.1 (bovine torovirus) (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">46</a>). The ratio of nonsynonymous to synonymous substitution rates (dN/dS) was estimated using the codeml program in the PAML package (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a>). The eight torovirus U1 nucleotide sequences were translated, aligned as amino acids with MUSCLE (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">70</a>), and the amino acid alignment used to guide a codon-based nucleotide alignment (EMBOSS tranalign) (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">71</a>). Alignment columns with gap characters in any sequence were removed, resulting in a reduction from 81 to 79 codon positions. PhyML (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">72</a>) was used to produce a nucleotide phylogenetic tree for the U1 alignment and, using this tree topology, dN/dS was calculated with codeml. The standard deviation for the codeml dN/dS value was estimated via a bootstrapping procedure, in which codon columns of the alignment were randomly resampled (with replacement); 100 randomized alignments were generated, and their dN/dS values calculated with codeml.</p><p id="p-63">Coding potential within each reading frame was analysed using MLOGD (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>) and synonymous site conservation was analysed with synplot2 (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a>). For these analyses we generated a codon-respecting alignment of the eight torovirus full-genome sequences using a procedure described previously (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">29</a>). In brief, each individual genome sequence was aligned to a reference sequence using code2aln version 1.2 (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">73</a>). Breda virus (GenBank accession <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=AY427798&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom">AY427798</a>) was used as reference, since unlike Berne virus it contains an intact HE gene. Genomes were then mapped to reference sequence coordinates by removing alignment positions that contained a gap character in the reference sequence, and these pairwise alignments were combined to give the multiple sequence alignment. This was analysed with MLOGD using a 40-codon sliding window and a 5-codon step size. For each of the three reading frames, within each window the null model is that the sequence is non-coding whereas the alternative model is that the sequence is coding in the given reading frame. Positive/negative values indicate that the sequences in the alignment are likely/unlikely to be coding in the given reading frame. To assess conservation at synonymous sites, the concatenated coding regions were extracted from the alignment and analysed with synplot2.</p></div></div><div class="section" id="sec-19"><h2 class="">Data availability</h2><p id="p-64">The sequencing data reported in this paper have been deposited in ArrayExpress (<a href="http://www.ebi.ac.uk/arrayexpress">http://www.ebi.ac.uk/arrayexpress</a>) under the accession number E-MTAB-6656.</p></div><div class="section" id="sec-20"><h2 class="">Funding Information</h2><p id="p-66">This work was supported by Wellcome Trust grant [106207] and European Research Council grant [646891] to A.E.F. and NWO-CW ECHO grant 711.014.004 from the Netherlands Organisation for Scientific Research to E.J.S.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-65">We thank Raoul de Groot and Arno van Vliet (Utrecht University) for providing the virus isolates and helpful advice, and Polly Roy (London School of Hygiene and Tropical Medicine) for ED cells.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.1" data-doi="10.1007/s00705-012-1295-x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lauber</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Junglen</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drosten</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zirkel</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nga</span>  <span class="cit-name-given-names">PT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morita</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorbalenya</span>  <span class="cit-name-given-names">AE.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Mesoniviridae: a proposed new family in the order Nidovirales formed by a single species of mosquito-borne viruses</span>. <abbr class="cit-jnl-abbrev">Arch Virol</abbr> <span class="cit-vol">157</span>:<span class="cit-fpage">1623</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArchives%2Bof%2Bvirology%26rft.stitle%253DArch%2BVirol%26rft.aulast%253DLauber%26rft.auinit1%253DC.%26rft.volume%253D157%26rft.issue%253D8%26rft.spage%253D1623%26rft.epage%253D1628%26rft.atitle%253DMesoniviridae%253A%2Ba%2Bproposed%2Bnew%2Bfamily%2Bin%2Bthe%2Border%2BNidovirales%2Bformed%2Bby%2Ba%2Bsingle%2Bspecies%2Bof%2Bmosquito-borne%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00705-012-1295-x%26rft_id%253Dinfo%253Apmid%252F22527862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00705-012-1295-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22527862&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">DW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beards</span>  <span class="cit-name-given-names">GM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Flewett</span>  <span class="cit-name-given-names">TH.</span></span> <span class="cit-pub-date">1987</span>. <span class="cit-article-title">Detection of Breda virus antigen and antibody in humans and animals by enzyme immunoassay</span>. <abbr class="cit-jnl-abbrev">J Clin Microbiol</abbr> <span class="cit-vol">25</span>:<span class="cit-fpage">637</span>–<span class="cit-lpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BMicrobiology%26rft.stitle%253DJ.%2BClin.%2BMicrobiol.%26rft.aulast%253DBrown%26rft.auinit1%253DD%2BW%26rft.volume%253D25%26rft.issue%253D4%26rft.spage%253D637%26rft.epage%253D640%26rft.atitle%253DDetection%2Bof%2BBreda%2Bvirus%2Bantigen%2Band%2Bantibody%2Bin%2Bhumans%2Band%2Banimals%2Bby%2Benzyme%2Bimmunoassay.%26rft_id%253Dinfo%253Apmid%252F3571473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjg6IjI1LzQvNjM3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDQvMDcvMjk2OTk2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.3" data-doi="10.1079/AHR200498"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoet</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saif</span>  <span class="cit-name-given-names">LJ.</span></span> <span class="cit-pub-date">2004</span>. <span class="cit-article-title">Bovine torovirus (Breda virus) revisited</span>. <abbr class="cit-jnl-abbrev">Anim Health Res Rev</abbr> <span class="cit-vol">5</span>:<span class="cit-fpage">157</span>–<span class="cit-lpage">71</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bhealth%2Bresearch%2Breviews%2B%252F%2BConference%2Bof%2BResearch%2BWorkers%2Bin%2BAnimal%2BDiseases%26rft.stitle%253DAnim%2BHealth%2BRes%2BRev%26rft.aulast%253DHoet%26rft.auinit1%253DA.%2BE.%26rft.volume%253D5%26rft.issue%253D2%26rft.spage%253D157%26rft.epage%253D171%26rft.atitle%253DBovine%2Btorovirus%2B%2528Breda%2Bvirus%2529%2Brevisited.%26rft_id%253Dinfo%253Adoi%252F10.1079%252FAHR200498%26rft_id%253Dinfo%253Apmid%252F15984322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1079/AHR200498&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15984322&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hanke</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pohlmann</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sauter-Louis</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoper</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stadler</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ritzmann</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Steinrigl</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schwarz</span>  <span class="cit-name-given-names">BA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Akimkin</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fux</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blome</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beer</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Porcine Epidemic Diarrhea in Europe: In-Detail Analyses of Disease Dynamics and Molecular Epidemiology</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-fpage">9</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">5.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-296996v1.5" data-doi="10.1016/j.vetmic.2010.05.023"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pignatelli</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grau-Roma</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jimenez</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Segales</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Longitudinal serological and virological study on porcine torovirus (PToV) in piglets from Spanish farms</span>. <abbr class="cit-jnl-abbrev">Vet Microbiol</abbr> <span class="cit-vol">146</span>:<span class="cit-fpage">260</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVeterinary%2Bmicrobiology%26rft.stitle%253DVet%2BMicrobiol%26rft.aulast%253DPignatelli%26rft.auinit1%253DJ.%26rft.volume%253D146%26rft.issue%253D3-4%26rft.spage%253D260%26rft.epage%253D268%26rft.atitle%253DLongitudinal%2Bserological%2Band%2Bvirological%2Bstudy%2Bon%2Bporcine%2Btorovirus%2B%2528PToV%2529%2Bin%2Bpiglets%2Bfrom%2BSpanish%2Bfarms.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.vetmic.2010.05.023%26rft_id%253Dinfo%253Apmid%252F20542392%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.vetmic.2010.05.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20542392&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.6" data-doi="10.1186/1756-0500-5-675"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Alonso-Padilla</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pignatelli</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simon-Grife</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Plazuelo</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Casal</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Seroprevalence of porcine torovirus (PToV) in Spanish farms</span>. <abbr class="cit-jnl-abbrev">BMC Res Notes</abbr> <span class="cit-vol">5</span>:<span class="cit-fpage">675</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BRes%2BNotes%26rft.volume%253D5%26rft.spage%253D675%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1756-0500-5-675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1756-0500-5-675&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.7" data-doi="10.1128/JVI.77.17.9567-9577.2003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Smits</span>  <span class="cit-name-given-names">SL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lavazza</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matiz</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horzinek</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koopmans</span>  <span class="cit-name-given-names">MP</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Groot</span>  <span class="cit-name-given-names">RJ.</span></span> <span class="cit-pub-date">2003</span>. <span class="cit-article-title">Phylogenetic and evolutionary relationships among torovirus field variants: evidence for multiple intertypic recombination events</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">77</span>:<span class="cit-fpage">9567</span>–<span class="cit-lpage">77</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DSmits%26rft.auinit1%253DS.%2BL.%26rft.volume%253D77%26rft.issue%253D17%26rft.spage%253D9567%26rft.epage%253D9577%26rft.atitle%253DPhylogenetic%2Band%2BEvolutionary%2BRelationships%2Bamong%2BTorovirus%2BField%2BVariants%253A%2BEvidence%2Bfor%2BMultiple%2BIntertypic%2BRecombination%2BEvents%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.77.17.9567-9577.2003%26rft_id%253Dinfo%253Apmid%252F12915570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI3Ny8xNy85NTY3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDQvMDcvMjk2OTk2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.8" data-doi="10.1099/vir.0.81611-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pasternak</span>  <span class="cit-name-given-names">AO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spaan</span>  <span class="cit-name-given-names">WJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ.</span></span> <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Nidovirus transcription: how to make sense…?</span> <abbr class="cit-jnl-abbrev">J Gen Virol</abbr> <span class="cit-vol">87</span>:<span class="cit-fpage">1403</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DPasternak%26rft.auinit1%253DA.%2BO.%26rft.volume%253D87%26rft.issue%253D6%26rft.spage%253D1403%26rft.epage%253D1421%26rft.atitle%253DNidovirus%2Btranscription%253A%2Bhow%2Bto%2Bmake%2Bsense...%253F%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.81611-0%26rft_id%253Dinfo%253Apmid%252F16690906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.81611-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16690906&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238167400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.9" data-doi="10.1073/pnas.96.21.12056"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van Marle</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dobbe</span>  <span class="cit-name-given-names">JC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gultyaev</span>  <span class="cit-name-given-names">AP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luytjes</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spaan</span>  <span class="cit-name-given-names">WJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ.</span></span> <span class="cit-pub-date">1999</span>. <span class="cit-article-title">Arterivirus discontinuous mRNA transcription is guided by base pairing between sense and antisense transcription-regulating sequences</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">96</span>:<span class="cit-fpage">12056</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253Dvan%2BMarle%26rft.auinit1%253DG.%26rft.volume%253D96%26rft.issue%253D21%26rft.spage%253D12056%26rft.epage%253D12061%26rft.atitle%253DArterivirus%2Bdiscontinuous%2BmRNA%2Btranscription%2Bis%2Bguided%2Bby%2Bbase%2Bpairing%2Bbetween%2Bsense%2Band%2Bantisense%2Btranscription-regulating%2Bsequences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.96.21.12056%26rft_id%253Dinfo%253Apmid%252F10518575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTYvMjEvMTIwNTYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNC8wNy8yOTY5OTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.10" data-doi="10.1099/0022-1317-83-4-927"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cowley</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dimmock</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">PJ.</span></span> <span class="cit-pub-date">2002</span>. <span class="cit-article-title">Gill-associated nidovirus of Penaeus monodon prawns transcribes 3’-coterminal subgenomic mRNAs that do not possess 5’-leader sequences</span>. <abbr class="cit-jnl-abbrev">J Gen Virol</abbr> <span class="cit-vol">83</span>:<span class="cit-fpage">927</span>–<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DCowley%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D83%26rft.issue%253D4%26rft.spage%253D927%26rft.epage%253D935%26rft.atitle%253DGill-associated%2Bnidovirus%2Bof%2BPenaeus%2Bmonodon%2Bprawns%2Btranscribes%2B3%2527-coterminal%2Bsubgenomic%2BmRNAs%2Bthat%2Bdo%2Bnot%2Bpossess%2B5%2527-leader%2Bsequences%26rft_id%253Dinfo%253Adoi%252F10.1099%252F0022-1317-83-4-927%26rft_id%253Dinfo%253Apmid%252F11907343%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/0022-1317-83-4-927&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11907343&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.11" data-doi="10.1128/JVI.00416-13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zirkel</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roth</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kurth</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drosten</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Junglen</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Identification and characterization of genetically divergent members of the newly established family Mesoniviridae</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">87</span>:<span class="cit-fpage">6346</span>–<span class="cit-lpage">58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00416-13%26rft_id%253Dinfo%253Apmid%252F23536661%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Ny8xMS82MzQ2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDQvMDcvMjk2OTk2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.12" data-doi="10.1093/emboj/cdf635"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van Vliet</span>  <span class="cit-name-given-names">AL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smits</span>  <span class="cit-name-given-names">SL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rottier</span>  <span class="cit-name-given-names">PJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Groot</span>  <span class="cit-name-given-names">RJ.</span></span> <span class="cit-pub-date">2002</span>. <span class="cit-article-title">Discontinuous and non-discontinuous subgenomic RNA transcription in a nidovirus</span>. <abbr class="cit-jnl-abbrev">EMBO J</abbr> <span class="cit-vol">21</span>:<span class="cit-fpage">6571</span>–<span class="cit-lpage">80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253Dvan%2BVliet%26rft.auinit1%253DA.%2BL.%2BW.%26rft.volume%253D21%26rft.issue%253D23%26rft.spage%253D6571%26rft.epage%253D6580%26rft.atitle%253DDiscontinuous%2Band%2Bnon-discontinuous%2Bsubgenomic%2BRNA%2Btranscription%2Bin%2Ba%2Bnidovirus%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252Fcdf635%26rft_id%253Dinfo%253Apmid%252F12456663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjEvMjMvNjU3MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA0LzA3LzI5Njk5Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.13" data-doi="10.1099/0022-1317-64-9-1849"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weiss</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Steck</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horzinek</span>  <span class="cit-name-given-names">MC.</span></span> <span class="cit-pub-date">1983</span>. <span class="cit-article-title">Purification and partial characterization of a new enveloped RNA virus (Berne virus)</span>. <abbr class="cit-jnl-abbrev">J Gen Virol</abbr> <span class="cit-vol">64</span> (<span class="cit-issue">Pt 9</span>):<span class="cit-fpage">1849</span>–<span class="cit-lpage">58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DWeiss%26rft.auinit1%253DM.%26rft.volume%253D64%26rft.issue%253D9%26rft.spage%253D1849%26rft.epage%253D1858%26rft.atitle%253DPurification%2Band%2BPartial%2BCharacterization%2Bof%2Ba%2BNew%2BEnveloped%2BRNA%2BVirus%2B%2528Berne%2BVirus%2529%26rft_id%253Dinfo%253Adoi%252F10.1099%252F0022-1317-64-9-1849%26rft_id%253Dinfo%253Apmid%252F6886677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/0022-1317-64-9-1849&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6886677&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1983RG41800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.14" data-doi="10.1128/CVI.00475-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kuwabara</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wada</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maeda</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miyazaki</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tsunemitsu</span>  <span class="cit-name-given-names">H.</span></span> <span class="cit-pub-date">2007</span>. <span class="cit-article-title">First isolation of cytopathogenic bovine torovirus in cell culture from a calf with diarrhea</span>. <abbr class="cit-jnl-abbrev">Clin Vaccine Immunol</abbr> <span class="cit-vol">14</span>:<span class="cit-fpage">998</span>–<span class="cit-lpage">1004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin%2BVaccine%2BImmunol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FCVI.00475-06%26rft_id%253Dinfo%253Apmid%252F17567770%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiY2RsaSI7czo1OiJyZXNpZCI7czo4OiIxNC84Lzk5OCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA0LzA3LzI5Njk5Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.15" data-doi="10.1371/journal.ppat.1005473"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Irigoyen</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Firth</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">JD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chung</span>  <span class="cit-name-given-names">BY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Siddell</span>  <span class="cit-name-given-names">SG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brierley</span>  <span class="cit-name-given-names">I.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">High-Resolution Analysis of Coronavirus Gene Expression by RNA Sequencing and Ribosome Profiling</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-vol">12</span>:<span class="cit-fpage">e1005473</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D12%26rft.spage%253De1005473%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1005473%26rft_id%253Dinfo%253Apmid%252F26919232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1005473&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26919232&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.16" data-doi="10.1016/0042-6822(91)90056-H"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">den Boon</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horzinek</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spaan</span>  <span class="cit-name-given-names">WJ.</span></span> <span class="cit-pub-date">1991</span>. <span class="cit-article-title">Comparison of the genome organization of toro- and coronaviruses: evidence for two nonhomologous RNA recombination events during Berne virus evolution</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">180</span>:<span class="cit-fpage">448</span>–<span class="cit-lpage">52</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DSnijder%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D180%26rft.issue%253D1%26rft.spage%253D448%26rft.epage%253D452%26rft.atitle%253DComparison%2Bof%2Bthe%2Bgenome%2Borganization%2Bof%2Btoro-%2Band%2Bcoronaviruses%253A%2Bevidence%2Bfor%2Btwo%2Bnonhomologous%2BRNA%2Brecombination%2Bevents%2Bduring%2BBerne%2Bvirus%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0042-6822%252891%252990056-H%26rft_id%253Dinfo%253Apmid%252F1984666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0042-6822(91)90056-H&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1984666&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991EP90800055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.17" data-doi="10.1093/nar/18.15.4535"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">den Boon</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bredenbeek</span>  <span class="cit-name-given-names">PJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horzinek</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rijnbrand</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spaan</span>  <span class="cit-name-given-names">WJ.</span></span> <span class="cit-pub-date">1990</span>. <span class="cit-article-title">The carboxyl-terminal part of the putative Berne virus polymerase is expressed by ribosomal frameshifting and contains sequence motifs which indicate that toro- and coronaviruses are evolutionarily related</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">18</span>:<span class="cit-fpage">4535</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSnijder%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D18%26rft.issue%253D15%26rft.spage%253D4535%26rft.epage%253D4542%26rft.atitle%253DThe%2Bcarboxyl-terminal%2Bpart%2Bof%2Bthe%2Bputative%2BBerne%2Bvirus%2Bpolymerase%2Bis%2Bexpressed%2Bby%2Bribosomal%2Bframeshifting%2Band%2Bcontains%2Bsequence%2Bmotifs%2Bwhich%2Bindicate%2Bthat%2Btoro-%2Band%2Bcoronaviruses%2Bare%2Bevolutionary%2Brelated%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F18.15.4535%26rft_id%253Dinfo%253Apmid%252F2388833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/18.15.4535&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2388833&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.18" data-doi="10.1016/j.virusres.2015.02.025"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leibowitz</span>  <span class="cit-name-given-names">JL.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">The structure and functions of coronavirus genomic 3’ and 5’ ends</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr> <span class="cit-vol">206</span>:<span class="cit-fpage">120</span>–<span class="cit-lpage">33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BRes%26rft.volume%253D206%26rft.spage%253D120%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2015.02.025%26rft_id%253Dinfo%253Apmid%252F25736566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2015.02.025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25736566&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.19" data-doi="10.1099/0022-1317-72-7-1635"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">EJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">den Boon</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horzinek</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spaan</span>  <span class="cit-name-given-names">WJ.</span></span> <span class="cit-pub-date">1991</span>. <span class="cit-article-title">Characterization of defective interfering RNAs of Berne virus</span>. <abbr class="cit-jnl-abbrev">J Gen Virol</abbr> <span class="cit-vol">72</span> (<span class="cit-issue">Pt 7</span>):<span class="cit-fpage">1635</span>–<span class="cit-lpage">43</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DSnijder%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D72%26rft.issue%253D7%26rft.spage%253D1635%26rft.epage%253D1643%26rft.atitle%253DCharacterization%2Bof%2Bdefective%2Binterfering%2BRNAs%2Bof%2BBerne%2Bvirus%26rft_id%253Dinfo%253Adoi%252F10.1099%252F0022-1317-72-7-1635%26rft_id%253Dinfo%253Apmid%252F1856694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/0022-1317-72-7-1635&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1856694&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.20" data-doi="10.1093/nar/gku671"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gerashchenko</span>  <span class="cit-name-given-names">MV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gladyshev</span>  <span class="cit-name-given-names">VN.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Translation inhibitors cause abnormalities in ribosome profiling experiments</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">42</span>:<span class="cit-fpage">e134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku671%26rft_id%253Dinfo%253Apmid%252F25056308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku671&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25056308&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.21" data-doi="10.1093/nar/gkw1190"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Andreev</span>  <span class="cit-name-given-names">DE</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Connor</span>  <span class="cit-name-given-names">PB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Loughran</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dmitriev</span>  <span class="cit-name-given-names">SE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baranov</span>  <span class="cit-name-given-names">PV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shatsky</span>  <span class="cit-name-given-names">IN.</span></span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Insights into the mechanisms of eukaryotic translation gained with ribosome profiling</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">45</span>:<span class="cit-fpage">513</span>–<span class="cit-lpage">526</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw1190%26rft_id%253Dinfo%253Apmid%252F27923997%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw1190&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27923997&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.22" data-doi="10.1093/molbev/msm088"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span> <span class="cit-pub-date">2007</span>. <span class="cit-article-title">PAML 4: phylogenetic analysis by maximum likelihood</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">24</span>:<span class="cit-fpage">1586</span>–<span class="cit-lpage">91</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm088%26rft_id%253Dinfo%253Apmid%252F17483113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msm088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17483113&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248848400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.23" data-doi="10.1186/1471-2105-7-75"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Firth</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">CM.</span></span> <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Detecting overlapping coding sequences in virus genomes</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">7</span>:<span class="cit-fpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DFirth%26rft.auinit1%253DA.%2BE.%26rft.volume%253D7%26rft.spage%253D75%26rft.epage%253D75%26rft.atitle%253DDetecting%2Boverlapping%2Bcoding%2Bsequences%2Bin%2Bvirus%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-7-75%26rft_id%253Dinfo%253Apmid%252F16483358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-7-75&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16483358&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.24" data-doi="10.1186/1743-422X-6-136"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Firth</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Atkins</span>  <span class="cit-name-given-names">JF.</span></span> <span class="cit-pub-date">2009</span>. <span class="cit-article-title">A case for a CUG-initiated coding sequence overlapping torovirus ORF1a and encoding a novel 30 kDa product</span>. <abbr class="cit-jnl-abbrev">Virol J</abbr> <span class="cit-vol">6</span>:<span class="cit-fpage">136</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DVirol%2BJ%26rft.aulast%253DFirth%26rft.auinit1%253DA.%2BE.%26rft.volume%253D6%26rft.spage%253D136%26rft.epage%253D136%26rft.atitle%253DA%2Bcase%2Bfor%2Ba%2BCUG-initiated%2Bcoding%2Bsequence%2Boverlapping%2Btorovirus%2BORF1a%2Band%2Bencoding%2Ba%2Bnovel%2B30%2BkDa%2Bproduct.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1743-422X-6-136%26rft_id%253Dinfo%253Apmid%252F19737402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1743-422X-6-136&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19737402&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.25" data-doi="10.1073/pnas.89.20.9489"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Keese</span>  <span class="cit-name-given-names">PK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gibbs</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">1992</span>. <span class="cit-article-title">Origins of genes: “big bang” or continuous creation?</span> <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">89</span>:<span class="cit-fpage">9489</span>–<span class="cit-lpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DKeese%26rft.auinit1%253DP.%2BK.%26rft.volume%253D89%26rft.issue%253D20%26rft.spage%253D9489%26rft.epage%253D9493%26rft.atitle%253DOrigins%2Bof%2Bgenes%253A%2B%2522big%2Bbang%2522%2Bor%2Bcontinuous%2Bcreation%253F%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.89.20.9489%26rft_id%253Dinfo%253Apmid%252F1329098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODkvMjAvOTQ4OSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA0LzA3LzI5Njk5Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.26" data-doi="10.1128/JVI.00595-09"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rancurel</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Khosravi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dunker</span>  <span class="cit-name-given-names">AK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Romero</span>  <span class="cit-name-given-names">PR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Karlin</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Overlapping genes produce proteins with unusual sequence properties and offer insight into de novo protein creation</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">83</span>:<span class="cit-fpage">10719</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00595-09%26rft_id%253Dinfo%253Apmid%252F19640978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4My8yMC8xMDcxOSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA0LzA3LzI5Njk5Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.27" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">SF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">EW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">DJ.</span></span> <span class="cit-pub-date">1990</span>. <span class="cit-article-title">Basic local alignment search tool</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr> <span class="cit-vol">215</span>:<span class="cit-fpage">403</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.28" data-doi="10.1093/nar/gki408"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Soding</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Biegert</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lupas</span>  <span class="cit-name-given-names">AN.</span></span> <span class="cit-pub-date">2005</span>. <span class="cit-article-title">The HHpred interactive server for protein homology detection and structure prediction</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">33</span>:<span class="cit-fpage">W244</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSoding%26rft.auinit1%253DJ.%26rft.volume%253D33%26rft.issue%253Dsuppl_2%26rft.spage%253DW244%26rft.epage%253DW248%26rft.atitle%253DThe%2BHHpred%2Binteractive%2Bserver%2Bfor%2Bprotein%2Bhomology%2Bdetection%2Band%2Bstructure%2Bprediction%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki408%26rft_id%253Dinfo%253Apmid%252F15980461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gki408&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15980461&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230271400046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.29" data-doi="10.1093/nar/gku981"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Firth</span>  <span class="cit-name-given-names">AE.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Mapping overlapping functional elements embedded within the protein-coding regions of RNA viruses</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">42</span>:<span class="cit-fpage">12425</span>–<span class="cit-lpage">39</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku981%26rft_id%253Dinfo%253Apmid%252F25326325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku981&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25326325&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.30" data-doi="10.1016/0092-8674(86)90762-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kozak</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">1986</span>. <span class="cit-article-title">Point mutations define a sequence flanking the AUG initiator codon that modulates translation by eukaryotic ribosomes</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">44</span>:<span class="cit-fpage">283</span>–<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DKozak%26rft.auinit1%253DM.%26rft.volume%253D44%26rft.issue%253D2%26rft.spage%253D283%26rft.epage%253D292%26rft.atitle%253DPoint%2Bmutations%2Bdefine%2Ba%2Bsequence%2Bflanking%2Bthe%2BAUG%2Binitiator%2Bcodon%2Bthat%2Bmodulates%2Btranslation%2Bby%2Beukaryotic%2Bribosomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252886%252990762-2%26rft_id%253Dinfo%253Apmid%252F3943125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(86)90762-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3943125&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1986AZC2900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.31" data-doi="10.1038/nprot.2015.053"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">LA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mezulis</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yates</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wass</span>  <span class="cit-name-given-names">MN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sternberg</span>  <span class="cit-name-given-names">MJ.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">The Phyre2 web portal for protein modeling, prediction and analysis</span>. <abbr class="cit-jnl-abbrev">Nat Protoc</abbr> <span class="cit-vol">10</span>:<span class="cit-fpage">845</span>–<span class="cit-lpage">58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BProtoc%26rft.volume%253D10%26rft.spage%253D845%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2015.053%26rft_id%253Dinfo%253Apmid%252F25950237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2015.053&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25950237&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">32.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-296996v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roth</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kiefer</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bordoli</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schwede</span>  <span class="cit-name-given-names">T.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">The Protein Model Portal--a comprehensive resource for protein structure and model information</span>. <abbr class="cit-jnl-abbrev">Database (Oxford)</abbr> 2013:<span class="cit-fpage">bat031</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.33" data-doi="10.1093/bioinformatics/16.4.404"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McGuffin</span>  <span class="cit-name-given-names">LJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bryson</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">DT.</span></span> <span class="cit-pub-date">2000</span>. <span class="cit-article-title">The PSIPRED protein structure prediction server</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">16</span>:<span class="cit-fpage">404</span>–<span class="cit-lpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DMcGuffin%26rft.auinit1%253DL.%2BJ.%26rft.volume%253D16%26rft.issue%253D4%26rft.spage%253D404%26rft.epage%253D405%26rft.atitle%253DThe%2BPSIPRED%2Bprotein%2Bstructure%2Bprediction%2Bserver%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F16.4.404%26rft_id%253Dinfo%253Apmid%252F10869041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/16.4.404&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10869041&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087933900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.34" data-doi="10.1128/JVI.79.13.8275-8281.2005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Smits</span>  <span class="cit-name-given-names">SL</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Vliet</span>  <span class="cit-name-given-names">AL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Segeren</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">el Azzouzi</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Essen</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Groot</span>  <span class="cit-name-given-names">RJ.</span></span> <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Torovirus non-discontinuous transcription: mutational analysis of a subgenomic mRNA promoter</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">79</span>:<span class="cit-fpage">8275</span>–<span class="cit-lpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DSmits%26rft.auinit1%253DS.%2BL.%26rft.volume%253D79%26rft.issue%253D13%26rft.spage%253D8275%26rft.epage%253D8281%26rft.atitle%253DTorovirus%2BNon-Discontinuous%2BTranscription%253A%2BMutational%2BAnalysis%2Bof%2Ba%2BSubgenomic%2BmRNA%2BPromoter%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.79.13.8275-8281.2005%26rft_id%253Dinfo%253Apmid%252F15956573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI3OS8xMy84Mjc1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDQvMDcvMjk2OTk2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.35" data-doi="10.1128/JVI.01758-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schutze</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ulferts</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schelle</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bayer</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Granzow</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mettenleiter</span>  <span class="cit-name-given-names">TC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Characterization of White bream virus reveals a novel genetic cluster of nidoviruses</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">80</span>:<span class="cit-fpage">11598</span>–<span class="cit-lpage">609</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01758-06%26rft_id%253Dinfo%253Apmid%252F16987966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4MC8yMy8xMTU5OCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA0LzA3LzI5Njk5Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.36" data-doi="10.1073/pnas.1706696114"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Di</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Madden</span>  <span class="cit-name-given-names">JC</span></span>, Jr., <span class="cit-auth"><span class="cit-name-surname">Morantz</span>  <span class="cit-name-given-names">EK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">HY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">RL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>  <span class="cit-name-given-names">RS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brinton</span>  <span class="cit-name-given-names">MA.</span></span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Expanded subgenomic mRNA transcriptome and coding capacity of a nidovirus</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">114</span>:<span class="cit-fpage">E8895</span>–<span class="cit-lpage">E8904</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1706696114%26rft_id%253Dinfo%253Apmid%252F29073030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE0LzQyL0U4ODk1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDQvMDcvMjk2OTk2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.37" data-doi="10.1186/1471-2164-14-743"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cong</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shao</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Transcriptome analysis of chicken kidney tissues following coronavirus avian infectious bronchitis virus infection</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>:<span class="cit-fpage">743</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D743%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-743%26rft_id%253Dinfo%253Apmid%252F24168272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-743&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24168272&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.38" data-doi="10.1111/j.1462-5822.2007.00951.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Raaben</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Groot Koerkamp</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rottier</span>  <span class="cit-name-given-names">PJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Haan</span>  <span class="cit-name-given-names">CA.</span></span> <span class="cit-pub-date">2007</span>. <span class="cit-article-title">Mouse hepatitis coronavirus replication induces host translational shutoff and mRNA decay, with concomitant formation of stress granules and processing bodies</span>. <abbr class="cit-jnl-abbrev">Cell Microbiol</abbr> <span class="cit-vol">9</span>:<span class="cit-fpage">2218</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCellular%2Bmicrobiology%26rft.stitle%253DCell%2BMicrobiol%26rft.aulast%253DRaaben%26rft.auinit1%253DM.%26rft.volume%253D9%26rft.issue%253D9%26rft.spage%253D2218%26rft.epage%253D2229%26rft.atitle%253DMouse%2Bhepatitis%2Bcoronavirus%2Breplication%2Binduces%2Bhost%2Btranslational%2Bshutoff%2Band%2BmRNA%2Bdecay%252C%2Bwith%2Bconcomitant%2Bformation%2Bof%2Bstress%2Bgranules%2Band%2Bprocessing%2Bbodies.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1462-5822.2007.00951.x%26rft_id%253Dinfo%253Apmid%252F17490409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1462-5822.2007.00951.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17490409&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.39" data-doi="10.1186/1743-422X-7-242"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xue</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shu</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bi</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">Y.</span></span> <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Profiling of cellular proteins in porcine reproductive and respiratory syndrome virus virions by proteomics analysis</span>. <abbr class="cit-jnl-abbrev">Virol J</abbr> <span class="cit-vol">7</span>:<span class="cit-fpage">242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DVirol%2BJ%26rft.aulast%253DZhang%26rft.auinit1%253DC.%26rft.volume%253D7%26rft.spage%253D242%26rft.epage%253D242%26rft.atitle%253DProfiling%2Bof%2Bcellular%2Bproteins%2Bin%2Bporcine%2Breproductive%2Band%2Brespiratory%2Bsyndrome%2Bvirus%2Bvirions%2Bby%2Bproteomics%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1743-422X-7-242%26rft_id%253Dinfo%253Apmid%252F20849641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1743-422X-7-242&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20849641&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.40" data-doi="10.1371/journal.pone.0178408"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">VanLeuven</span>  <span class="cit-name-given-names">JT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ridenhour</span>  <span class="cit-name-given-names">BJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gonzalez</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">CR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miura</span>  <span class="cit-name-given-names">TA.</span></span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Lung epithelial cells have virus-specific and shared gene expression responses to infection by diverse respiratory viruses</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">12</span>:<span class="cit-fpage">e0178408</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D12%26rft.spage%253De0178408%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0178408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0178408&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.41" data-doi="10.1128/JVI.05177-11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Beura</span>  <span class="cit-name-given-names">LK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dinh</span>  <span class="cit-name-given-names">PX</span></span>, <span class="cit-auth"><span class="cit-name-surname">Osorio</span>  <span class="cit-name-given-names">FA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pattnaik</span>  <span class="cit-name-given-names">AK.</span></span> <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Cellular poly(c) binding proteins 1 and 2 interact with porcine reproductive and respiratory syndrome virus nonstructural protein 1beta and support viral replication</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">85</span>:<span class="cit-fpage">12939</span>–<span class="cit-lpage">49</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.05177-11%26rft_id%253Dinfo%253Apmid%252F21976648%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4NS8yNC8xMjkzOSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA0LzA3LzI5Njk5Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.42" data-doi="10.1371/journal.pone.0020972"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maestre</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garzon</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Equine torovirus (BEV) induces caspase-mediated apoptosis in infected cells</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>:<span class="cit-fpage">e20972</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DMaestre%26rft.auinit1%253DA.%2BM.%26rft.volume%253D6%26rft.issue%253D6%26rft.spage%253De20972%26rft.epage%253De20972%26rft.atitle%253DEquine%2Btorovirus%2B%2528BEV%2529%2Binduces%2Bcaspase-mediated%2Bapoptosis%2Bin%2Binfected%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0020972%26rft_id%253Dinfo%253Apmid%252F21698249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0020972&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21698249&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.43" data-doi="10.1016/S0248-4900(03)00033-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Touriol</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bornes</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bonnal</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Audigier</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Prats</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Prats</span>  <span class="cit-name-given-names">AC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vagner</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-pub-date">2003</span>. <span class="cit-article-title">Generation of protein isoform diversity by alternative initiation of translation at non-AUG codons</span>. <abbr class="cit-jnl-abbrev">Biol Cell</abbr> <span class="cit-vol">95</span>:<span class="cit-fpage">169</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiology%2Bof%2Bthe%2Bcell%2B%252F%2Bunder%2Bthe%2Bauspices%2Bof%2Bthe%2BEuropean%2BCell%2BBiology%2BOrganization%26rft.stitle%253DBiol%2BCell%26rft.aulast%253DTouriol%26rft.auinit1%253DC.%26rft.volume%253D95%26rft.issue%253D3-4%26rft.spage%253D169%26rft.epage%253D178%26rft.atitle%253DGeneration%2Bof%2Bprotein%2Bisoform%2Bdiversity%2Bby%2Balternative%2Binitiation%2Bof%2Btranslation%2Bat%2Bnon-AUG%2Bcodons.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0248-4900%252803%252900033-9%26rft_id%253Dinfo%253Apmid%252F12867081%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0248-4900(03)00033-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12867081&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000184483100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.44" data-doi="10.1099/vir.0.042499-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Firth</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brierley</span>  <span class="cit-name-given-names">I.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Non-canonical translation in RNA viruses</span>. <abbr class="cit-jnl-abbrev">J Gen Virol</abbr> <span class="cit-vol">93</span>:<span class="cit-fpage">1385</span>–<span class="cit-lpage">409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BGen%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.042499-0%26rft_id%253Dinfo%253Apmid%252F22535777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.042499-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22535777&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306348900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.45" data-doi="10.1016/j.virusres.2005.07.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Draker</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roper</span>  <span class="cit-name-given-names">RL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Petric</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tellier</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-pub-date">2006</span>. <span class="cit-article-title">The complete sequence of the bovine torovirus genome</span>. <abbr class="cit-jnl-abbrev">Virus Res</abbr> <span class="cit-vol">115</span>:<span class="cit-fpage">56</span>–<span class="cit-lpage">68</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2Bresearch%26rft.stitle%253DVirus%2BRes%26rft.aulast%253DDraker%26rft.auinit1%253DR.%26rft.volume%253D115%26rft.issue%253D1%26rft.spage%253D56%26rft.epage%253D68%26rft.atitle%253DThe%2Bcomplete%2Bsequence%2Bof%2Bthe%2Bbovine%2Btorovirus%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virusres.2005.07.005%26rft_id%253Dinfo%253Apmid%252F16137782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virusres.2005.07.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16137782&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.46" data-doi="10.1016/j.meegid.2015.12.013"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ito</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tsuchiaka</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Naoi</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Otomaru</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Masuda</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haga</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oka</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yamasato</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Omatsu</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sugimura</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aoki</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Furuya</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Katayama</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oba</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shirai</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Katayama</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mizutani</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nagai</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Whole genome analysis of Japanese bovine toroviruses reveals natural recombination between porcine and bovine toroviruses</span>. <abbr class="cit-jnl-abbrev">Infect Genet Evol</abbr> <span class="cit-vol">38</span>:<span class="cit-fpage">90</span>–<span class="cit-lpage">95</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfect%2BGenet%2BEvol%26rft.volume%253D38%26rft.spage%253D90%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.meegid.2015.12.013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.meegid.2015.12.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.47" data-doi="10.1007/s00705-013-1861-x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lan</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hua</span>  <span class="cit-name-given-names">X.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Molecular characterization and phylogenetic analysis of the genome of porcine torovirus</span>. <abbr class="cit-jnl-abbrev">Arch Virol</abbr> <span class="cit-vol">159</span>:<span class="cit-fpage">773</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArch%2BVirol%26rft.volume%253D159%26rft.spage%253D773%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00705-013-1861-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00705-013-1861-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.48" data-doi="10.3390/v4030363"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ruch</span>  <span class="cit-name-given-names">TR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Machamer</span>  <span class="cit-name-given-names">CE.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">The coronavirus E protein: assembly and beyond</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">4</span>:<span class="cit-fpage">363</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DViruses%26rft.aulast%253DRuch%26rft.auinit1%253DT.%2BR.%26rft.volume%253D4%26rft.issue%253D3%26rft.spage%253D363%26rft.epage%253D382%26rft.atitle%253DThe%2Bcoronavirus%2BE%2Bprotein%253A%2Bassembly%2Band%2Bbeyond.%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv4030363%26rft_id%253Dinfo%253Apmid%252F22590676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v4030363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22590676&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Senanayake</span>  <span class="cit-name-given-names">SD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hofmann</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maki</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brian</span>  <span class="cit-name-given-names">DA.</span></span> <span class="cit-pub-date">1992</span>. <span class="cit-article-title">The nucleocapsid protein gene of bovine coronavirus is bicistronic</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">66</span>:<span class="cit-fpage">5277</span>–<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DSenanayake%26rft.auinit1%253DS%2BD%26rft.volume%253D66%26rft.issue%253D9%26rft.spage%253D5277%26rft.epage%253D5283%26rft.atitle%253DThe%2Bnucleocapsid%2Bprotein%2Bgene%2Bof%2Bbovine%2Bcoronavirus%2Bis%2Bbicistronic.%26rft_id%253Dinfo%253Apmid%252F1501275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjY2LzkvNTI3NyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA0LzA3LzI5Njk5Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hingley</span>  <span class="cit-name-given-names">ST</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weiss</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Masters</span>  <span class="cit-name-given-names">PS.</span></span> <span class="cit-pub-date">1997</span>. <span class="cit-article-title">The internal open reading frame within the nucleocapsid gene of mouse hepatitis virus encodes a structural protein that is not essential for viral replication</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">71</span>:<span class="cit-fpage">996</span>–<span class="cit-lpage">1003</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DFischer%26rft.auinit1%253DF.%26rft.volume%253D71%26rft.issue%253D2%26rft.spage%253D996%26rft.epage%253D1003%26rft.atitle%253DThe%2Binternal%2Bopen%2Breading%2Bframe%2Bwithin%2Bthe%2Bnucleocapsid%2Bgene%2Bof%2Bmouse%2Bhepatitis%2Bvirus%2Bencodes%2Ba%2Bstructural%2Bprotein%2Bthat%2Bis%2Bnot%2Bessential%2Bfor%2Bviral%2Breplication%26rft_id%253Dinfo%253Apmid%252F8995618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjcxLzIvOTk2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDQvMDcvMjk2OTk2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.51" data-doi="10.1186/s12977-018-0394-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Irigoyen</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dinan</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brierley</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Firth</span>  <span class="cit-name-given-names">AE.</span></span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Ribosome profiling of the retrovirus murine leukemia virus</span>. <abbr class="cit-jnl-abbrev">Retrovirology</abbr> <span class="cit-vol">15</span>:<span class="cit-fpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRetrovirology%26rft.volume%253D15%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12977-018-0394-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12977-018-0394-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.52" data-doi="10.1093/nar/gkv1290"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Coordinators</span>  <span class="cit-name-given-names">NR.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Database resources of the National Center for Biotechnology Information</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">44</span>:<span class="cit-fpage">D7</span>–<span class="cit-lpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1290%26rft_id%253Dinfo%253Apmid%252F26615191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1290&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26615191&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.53" data-doi="10.1093/nar/gkr1079"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pruitt</span>  <span class="cit-name-given-names">KD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tatusova</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">GR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maglott</span>  <span class="cit-name-given-names">DR.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">NCBI Reference Sequences (RefSeq): current status, new features and genome annotation policy</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">40</span>:<span class="cit-fpage">D130</span>–<span class="cit-lpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr1079%26rft_id%253Dinfo%253Apmid%252F22121212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr1079&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22121212&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298601300020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.54" data-doi="10.1093/nar/gkr991"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Flicek</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Amode</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barrell</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beal</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brent</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carvalho-Silva</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clapham</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coates</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fairley</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fitzgerald</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gil</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hendrix</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hourlier</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kahari</span>  <span class="cit-name-given-names">AK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keefe</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keenan</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kinsella</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Komorowska</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koscielny</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kulesha</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Larsson</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Longden</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">McLaren</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muffato</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Overduin</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pignatelli</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Riat</span>  <span class="cit-name-given-names">HS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ritchie</span>  <span class="cit-name-given-names">GR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ruffier</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schuster</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sobral</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">YA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Trevanion</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vandrovcova</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilder</span>  <span class="cit-name-given-names">SP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aken</span>  <span class="cit-name-given-names">BL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Birney</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cunningham</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fernandez-Suarez</span>  <span class="cit-name-given-names">XM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harrow</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Herrero</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hubbard</span>  <span class="cit-name-given-names">TJ</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Ensembl 2012</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">40</span>:<span class="cit-fpage">D84</span>–<span class="cit-lpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr991%26rft_id%253Dinfo%253Apmid%252F22086963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr991&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22086963&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298601300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.55" data-doi="10.1093/nar/gkv1309"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">PP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">TM.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">GtRNAdb 2.0: an expanded database of transfer RNA genes identified in complete and draft genomes</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">44</span>:<span class="cit-fpage">D184</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1309%26rft_id%253Dinfo%253Apmid%252F26673694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1309&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26673694&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.56" data-doi="10.1186/gb-2009-10-3-r25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">SL.</span></span> <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</span>. <abbr class="cit-jnl-abbrev">Genome Biol</abbr> <span class="cit-vol">10</span>:<span class="cit-fpage">R25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253DR25%26rft.epage%253DR25%26rft.atitle%253DUltrafast%2Band%2Bmemory-efficient%2Balignment%2Bof%2Bshort%2BDNA%2Bsequences%2Bto%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-3-r25%26rft_id%253Dinfo%253Apmid%252F19261174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-3-r25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19261174&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.57" data-doi="10.1093/bioinformatics/bts635"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dobin</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schlesinger</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drenkow</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zaleski</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jha</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Batut</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chaisson</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gingeras</span>  <span class="cit-name-given-names">TR.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">STAR: ultrafast universal RNA-seq aligner</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>:<span class="cit-fpage">15</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts635%26rft_id%253Dinfo%253Apmid%252F23104886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts635&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23104886&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312654600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.58" data-doi="10.1038/nbt.1883"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">MG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">BJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">JZ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Amit</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Adiconis</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Raychowdhury</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mauceli</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hacohen</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gnirke</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rhind</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">di Palma</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Birren</span>  <span class="cit-name-given-names">BW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nusbaum</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lindblad-Toh</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Regev</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome</span>. <abbr class="cit-jnl-abbrev">Nat Biotechnol</abbr> <span class="cit-vol">29</span>:<span class="cit-fpage">644</span>–<span class="cit-lpage">52</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DGrabherr%26rft.auinit1%253DM.%2BG.%26rft.volume%253D29%26rft.issue%253D7%26rft.spage%253D644%26rft.epage%253D652%26rft.atitle%253DFull-length%2Btranscriptome%2Bassembly%2Bfrom%2BRNA-Seq%2Bdata%2Bwithout%2Ba%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1883%26rft_id%253Dinfo%253Apmid%252F21572440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1883&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21572440&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.59" data-doi="10.1093/molbev/mst010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Katoh</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Standley</span>  <span class="cit-name-given-names">DM.</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">MAFFT multiple sequence alignment software version 7: improvements in performance and usability</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">30</span>:<span class="cit-fpage">772</span>–<span class="cit-lpage">80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst010%26rft_id%253Dinfo%253Apmid%252F23329690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23329690&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000317002300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.60"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sievers</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">DG.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Clustal omega</span>. <abbr class="cit-jnl-abbrev">Curr Protoc Bioinformatics</abbr> <span class="cit-vol">48</span>:<span class="cit-issue">3</span> <span class="cit-fpage">13 1</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr%2BProtoc%2BBioinformatics%26rft.volume%253D48%26rft.spage%253D131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.61" data-doi="10.1186/1471-2105-9-474"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bernhart</span>  <span class="cit-name-given-names">SH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hofacker</span>  <span class="cit-name-given-names">IL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Will</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gruber</span>  <span class="cit-name-given-names">AR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stadler</span>  <span class="cit-name-given-names">PF.</span></span> <span class="cit-pub-date">2008</span>. <span class="cit-article-title">RNAalifold: improved consensus structure prediction for RNA alignments</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">9</span>:<span class="cit-fpage">474</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DBernhart%26rft.auinit1%253DS.%2BH.%26rft.volume%253D9%26rft.spage%253D474%26rft.epage%253D474%26rft.atitle%253DRNAalifold%253A%2Bimproved%2Bconsensus%2Bstructure%2Bprediction%2Bfor%2BRNA%2Balignments.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-9-474%26rft_id%253Dinfo%253Apmid%252F19014431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-9-474&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19014431&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">62.</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.62" data-doi="10.1093/bioinformatics/btp250"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Darty</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Denise</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ponty</span>  <span class="cit-name-given-names">Y.</span></span> <span class="cit-pub-date">2009</span>. <span class="cit-article-title">VARNA: Interactive drawing and editing of the RNA secondary structure</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>:<span class="cit-fpage">1974</span>–<span class="cit-lpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp250%26rft_id%253Dinfo%253Apmid%252F19398448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp250&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19398448&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268107100023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">63.</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.63" data-doi="10.1093/bioinformatics/btu638"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pyl</span>  <span class="cit-name-given-names">PT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">HTSeq--a Python framework to work with high-throughput sequencing data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>:<span class="cit-fpage">166</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu638%26rft_id%253Dinfo%253Apmid%252F25260700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu638&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25260700&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000347832300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">64.</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.64" data-doi="10.1186/s13059-014-0550-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Love</span>  <span class="cit-name-given-names">MI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</span>. <abbr class="cit-jnl-abbrev">Genome Biol</abbr> <span class="cit-vol">15</span>:<span class="cit-fpage">550</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D15%26rft.spage%253D550%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0550-8%26rft_id%253Dinfo%253Apmid%252F25516281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0550-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25516281&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">65.</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.65" data-doi="10.1093/bioinformatics/btn209"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Strimmer</span>  <span class="cit-name-given-names">K.</span></span> <span class="cit-pub-date">2008</span>. <span class="cit-article-title">fdrtool: a versatile R package for estimating local and tail area-based false discovery rates</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span>:<span class="cit-fpage">1461</span>–<span class="cit-lpage">2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn209%26rft_id%253Dinfo%253Apmid%252F18441000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btn209&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18441000&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256756800008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">66.</span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 66. in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.66" data-doi="10.1038/75556"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ashburner</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ball</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blake</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cherry</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">AP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dolinski</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dwight</span>  <span class="cit-name-given-names">SS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eppig</span>  <span class="cit-name-given-names">JT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">DP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Issel-Tarver</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kasarskis</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matese</span>  <span class="cit-name-given-names">JC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Richardson</span>  <span class="cit-name-given-names">JE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ringwald</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">GM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sherlock</span>  <span class="cit-name-given-names">G.</span></span> <span class="cit-pub-date">2000</span>. <span class="cit-article-title">Gene ontology: tool for the unification of biology. The Gene Ontology Consortium</span>. <abbr class="cit-jnl-abbrev">Nat Genet</abbr> <span class="cit-vol">25</span>:<span class="cit-fpage">25</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DAshburner%26rft.auinit1%253DM.%26rft.volume%253D25%26rft.issue%253D1%26rft.spage%253D25%26rft.epage%253D29%26rft.atitle%253DGene%2Bontology%253A%2Btool%2Bfor%2Bthe%2Bunification%2Bof%2Bbiology.%2BThe%2BGene%2BOntology%2BConsortium.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F75556%26rft_id%253Dinfo%253Apmid%252F10802651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/75556&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10802651&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086884000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">67.</span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference 67. in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.67" data-doi="10.1038/nprot.2009.97"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Durinck</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spellman</span>  <span class="cit-name-given-names">PT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Birney</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span> <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Mapping identifiers for the integration of genomic datasets with the R/Bioconductor package biomaRt</span>. <abbr class="cit-jnl-abbrev">Nat Protoc</abbr> <span class="cit-vol">4</span>:<span class="cit-fpage">1184</span>–<span class="cit-lpage">91</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DDurinck%26rft.auinit1%253DS.%26rft.volume%253D4%26rft.issue%253D8%26rft.spage%253D1184%26rft.epage%253D1191%26rft.atitle%253DMapping%2Bidentifiers%2Bfor%2Bthe%2Bintegration%2Bof%2Bgenomic%2Bdatasets%2Bwith%2Bthe%2BR%252FBioconductor%2Bpackage%2BbiomaRt.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2009.97%26rft_id%253Dinfo%253Apmid%252F19617889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2009.97&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19617889&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268858800008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">68.</span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference 68. in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.68" data-doi="10.1038/ncomms11194"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">X.</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Genome-wide assessment of differential translations with ribosome profiling data</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-vol">7</span>:<span class="cit-fpage">11194</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D7%26rft.spage%253D11194%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms11194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms11194&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">69.</span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference 69. in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.69"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anbalagan</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wassman</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elston</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">K.</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Genome sequence of torovirus identified from a pig with porcine epidemic diarrhea virus from the United States</span>. <abbr class="cit-jnl-abbrev">Genome Announc</abbr> <span class="cit-fpage">2</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">70.</span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference 70. in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.70" data-doi="10.1093/nar/gkh340"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">RC.</span></span> <span class="cit-pub-date">2004</span>. <span class="cit-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">32</span>:<span class="cit-fpage">1792</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkh340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15034147&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220487200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">71.</span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference 71. in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.71" data-doi="10.1016/S0168-9525(00)02024-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Longden</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bleasby</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">2000</span>. <span class="cit-article-title">EMBOSS: the European Molecular Biology Open Software Suite</span>. <abbr class="cit-jnl-abbrev">Trends Genet</abbr> <span class="cit-vol">16</span>:<span class="cit-fpage">276</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DRice%26rft.auinit1%253DP.%26rft.volume%253D16%26rft.issue%253D6%26rft.spage%253D276%26rft.epage%253D277%26rft.atitle%253DEMBOSS%253A%2Bthe%2BEuropean%2BMolecular%2BBiology%2BOpen%2BSoftware%2BSuite.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252800%252902024-2%26rft_id%253Dinfo%253Apmid%252F10827456%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(00)02024-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10827456&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087457200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">72.</span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference 72. in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.72" data-doi="10.1080/10635150390235520"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guindon</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gascuel</span>  <span class="cit-name-given-names">O.</span></span> <span class="cit-pub-date">2003</span>. <span class="cit-article-title">A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood</span>. <abbr class="cit-jnl-abbrev">Syst Biol</abbr> <span class="cit-vol">52</span>:<span class="cit-fpage">696</span>–<span class="cit-lpage">704</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150390235520%26rft_id%253Dinfo%253Apmid%252F14530136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10635150390235520&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14530136&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185732500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">73.</span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference 73. in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-296996v1.73" data-doi="10.1186/1471-2105-6-160"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stocsits</span>  <span class="cit-name-given-names">RR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hofacker</span>  <span class="cit-name-given-names">IL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fried</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stadler</span>  <span class="cit-name-given-names">PF.</span></span> <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Multiple sequence alignments of partially coding nucleic acid sequences</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">6</span>:<span class="cit-fpage">160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DStocsits%26rft.auinit1%253DR.%2BR.%26rft.volume%253D6%26rft.spage%253D160%26rft.epage%253D160%26rft.atitle%253DMultiple%2Bsequence%2Balignments%2Bof%2Bpartially%2Bcoding%2Bnucleic%2Bacid%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-6-160%26rft_id%253Dinfo%253Apmid%252F15985156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-6-160&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15985156&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F07%2F296996.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/296996v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F296996v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/297218v1" title="Born to run? Quantifying the balance of prior bias and new information in prey escape decisions" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;April 07, 2018.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/296996v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/296996v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/296996v1" class="active" data-icon-position="" data-hide-link-title="0">The transcriptional and translational landscape of equine torovirus</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/91424" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-2Ukucb8OrsrUtx9vjDFDe9N-qcdhr2SXE_cxgNLIJX0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="91424" id="node91424--2" data-pisa="biorxiv;296996v1" data-pisa-master="biorxiv;296996" data-seqnum="91424" data-apath="/biorxiv/early/2018/04/07/296996.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">The transcriptional and translational landscape of equine torovirus</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Hazel</span> <span class="nlm-surname">Stewart</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Katherine</span> <span class="nlm-surname">Brown</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Adam M.</span> <span class="nlm-surname">Dinan</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Nerea</span> <span class="nlm-surname">Irigoyen</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Eric J.</span> <span class="nlm-surname">Snijder</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Andrew E.</span> <span class="nlm-surname">Firth</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">296996; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/296996 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/296996v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/296996v1&amp;title=The%20transcriptional%20and%20translational%20landscape%20of%20equine%20torovirus" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/296996v1&amp;title=The%20transcriptional%20and%20translational%20landscape%20of%20equine%20torovirus" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/296996v1&amp;text=The%20transcriptional%20and%20translational%20landscape%20of%20equine%20torovirus" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/296996v1&amp;title=The%20transcriptional%20and%20translational%20landscape%20of%20equine%20torovirus" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/296996v1&amp;t=The%20transcriptional%20and%20translational%20landscape%20of%20equine%20torovirus" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/296996v1&amp;title=The%20transcriptional%20and%20translational%20landscape%20of%20equine%20torovirus" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/296996v1&amp;title=The%20transcriptional%20and%20translational%20landscape%20of%20equine%20torovirus" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="91424" id="node91424--3" data-pisa="biorxiv;296996v1" data-pisa-master="biorxiv;296996" data-seqnum="91424" data-apath="/biorxiv/early/2018/04/07/296996.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">The transcriptional and translational landscape of equine torovirus</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Hazel</span> <span class="nlm-surname">Stewart</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Katherine</span> <span class="nlm-surname">Brown</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Adam M.</span> <span class="nlm-surname">Dinan</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Nerea</span> <span class="nlm-surname">Irigoyen</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Eric J.</span> <span class="nlm-surname">Snijder</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Andrew E.</span> <span class="nlm-surname">Firth</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">296996; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/296996 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=The%20transcriptional%20and%20translational%20landscape%20of%20equine%20torovirus&amp;rft.volume&amp;rft.issue&amp;rft.date=2018-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Stewart%2C+Hazel&amp;rft.au=Brown%2C+Katherine&amp;rft.au=Dinan%2C+Adam+M.&amp;rft.au=Irigoyen%2C+Nerea&amp;rft.au=Snijder%2C+Eric+J.&amp;rft.au=Firth%2C+Andrew+E."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/91424/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/91424/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/91424/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/91424/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/91424/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/91424/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/91424/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/91424/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/91424/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/91424/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/91424/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/91424/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/296996v1&amp;count=horizontal&amp;via=&amp;text=The%20transcriptional%20and%20translational%20landscape%20of%20equine%20torovirus&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/296996v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/296996v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/296996v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">Microbiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
