<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/228692/embed/graphic-9.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2019/12/13/228692.atom"/>
  <meta name="HW.pisa" content="biorxiv;228692v4"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Structurally informed evolutionary models improve phylogenetic reconstruction for emerging, seasonal, and pandemic influenza viruses"/>
  <meta name="DC.Identifier" content="10.1101/228692"/>
  <meta name="DC.Date" content="2019-12-13"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Precise estimation of genetic substitution patterns is critical for accurate reconstruction of pathogen phylogenies. Few studies of viral evolution account for variations of mutation rate across a single gene. This is especially true when considering evolution of segmented viruses where individual segments are short, encoding for few proteins. However, the structural and functional partitions of these proteins could provide valuable information for more accurate inference of viral evolution, due to the disparate immune selection pressure on different functional domains. Accurately reconstructed evolutionary features on specific functional domains can in turn provide biological information on viral protein and immune targets for vaccine design. In this study we developed and evaluated a structurally informed partitioning scheme that accounts for rate variation among immunogenic head and stalk domains of the surface protein hemagglutinin (HA) of influenza viruses. We evaluated the model fit and performance of four different models - HKY, SRD06 codon, HKY with a structurally informed partitioning scheme, SRD06 with a structurally informed partitioning scheme on pandemic A/H1N1pdm09, seasonal A/H1N1postpdm, A/H3N2, B-Yamagata-like and Victoria-like lineages, and two highly pathogenic avian influenza A viruses H5Nx and H7N9. Results showed that structurally informed partitioning with SRD06 performed better for all datasets with decisively statistical support. Significantly faster nucleotide substitution rates for head domain, compared to stalk domain was observed and may provide insight for stalk derived broadly-reactive vaccine design. Taken together, integrating a functionally informed partitioning scheme based on protein structures of immune targets allows for significant improvement of phylogenetic analysis and providing important biological insights."/>
  <meta name="DC.Contributor" content="Xueting Qiu"/>
  <meta name="DC.Contributor" content="Justin Bahl"/>
  <meta name="article:published_time" content="2019-12-13"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Structurally informed evolutionary models improve phylogenetic reconstruction for emerging, seasonal, and pandemic influenza viruses"/>
  <meta name="citation_abstract" lang="en" content="&lt;;h3&gt;;Abstract&lt;;/h3&gt;; &lt;;p&gt;;Precise estimation of genetic substitution patterns is critical for accurate reconstruction of pathogen phylogenies. Few studies of viral evolution account for variations of mutation rate across a single gene. This is especially true when considering evolution of segmented viruses where individual segments are short, encoding for few proteins. However, the structural and functional partitions of these proteins could provide valuable information for more accurate inference of viral evolution, due to the disparate immune selection pressure on different functional domains. Accurately reconstructed evolutionary features on specific functional domains can in turn provide biological information on viral protein and immune targets for vaccine design. In this study we developed and evaluated a structurally informed partitioning scheme that accounts for rate variation among immunogenic head and stalk domains of the surface protein hemagglutinin (HA) of influenza viruses. We evaluated the model fit and performance of four different models - HKY, SRD06 codon, HKY with a structurally informed partitioning scheme, SRD06 with a structurally informed partitioning scheme on pandemic A/H1N1pdm09, seasonal A/H1N1postpdm, A/H3N2, B-Yamagata-like and Victoria-like lineages, and two highly pathogenic avian influenza A viruses H5Nx and H7N9. Results showed that structurally informed partitioning with SRD06 performed better for all datasets with decisively statistical support. Significantly faster nucleotide substitution rates for head domain, compared to stalk domain was observed and may provide insight for stalk derived broadly-reactive vaccine design. Taken together, integrating a functionally informed partitioning scheme based on protein structures of immune targets allows for significant improvement of phylogenetic analysis and providing important biological insights.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2019/01/01"/>
  <meta name="citation_mjid" content="biorxiv;228692v4"/>
  <meta name="citation_id" content="228692v4"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/228692v4"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/228692v4.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/228692v4.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/228692.full.pdf"/>
  <meta name="citation_doi" content="10.1101/228692"/>
  <meta name="citation_num_pages" content="41"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="228692"/>
  <meta name="citation_author" content="Xueting Qiu"/>
  <meta name="citation_author_institution" content="Center for Ecology of Infectious Diseases, Department of Infectious Diseases, College of Veterinary Medicine, University of Georgia"/>
  <meta name="citation_author_institution" content="Center for Communicable Disease Dynamics, Harvard T.H. Chan School of Public Health"/>
  <meta name="citation_author_email" content="xuetingqiu@hsph.harvard.edu"/>
  <meta name="citation_author_email" content="Justin.Bahl@uga.edu"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6810-7304"/>
  <meta name="citation_author" content="Justin Bahl"/>
  <meta name="citation_author_institution" content="Center for Ecology of Infectious Diseases, Department of Infectious Diseases, College of Veterinary Medicine, University of Georgia"/>
  <meta name="citation_author_institution" content="Department of Epidemiology and Biostatistics, College of Public Health, University of Georgia"/>
  <meta name="citation_author_institution" content="Program in Emerging Infectious Diseases, Duke-National University of Singapore Graduate Medical School"/>
  <meta name="citation_author_email" content="xuetingqiu@hsph.harvard.edu"/>
  <meta name="citation_author_email" content="Justin.Bahl@uga.edu"/>
  <meta name="citation_reference" content="Campbell F, Strang C, Ferguson N, Cori A, Jombart T. When are pathogen genome sequences informative of transmission events? Parrish C, editor. PLOS Pathog [Internet]. 2018 Feb 8 [cited 2019 Jun 29];14(2):e1006885. Available from: https://dx.plos.org/10.1371/journal.ppat.1006885"/>
  <meta name="citation_reference" content="Biek R, Pybus OG, Lloyd-Smith JO, Didelot X. Measurably evolving pathogens in the genomic era. Trends Ecol Evol [Internet]. 2015 Jun [cited 2019 Jan 9];30(6):306–13. Available from: http://www.ncbi.nlm.nih.gov/pubmed/25887947"/>
  <meta name="citation_reference" content="Lanfear R, Calcott B, Ho SYW, Guindon S. PartitionFinder: Combined Selection of Partitioning Schemes and Substitution Models for Phylogenetic Analyses. Mol Biol Evol [Internet]. 2012 Jun 1 [cited 2019 Feb 20];29(6):1695–701. Available from: https://academic.oup.com/mbe/article-lookup/doi/10.1093/molbev/mss020"/>
  <meta name="citation_reference" content="Hasegawa M, Kishino H, Yano T. Dating of the human-ape splitting by a molecular clock of mitochondrial DNA. J Mol Evol [Internet]. 1985 [cited 2019 Feb 20];22(2):160–74. Available from: http://www.ncbi.nlm.nih.gov/pubmed/3934395"/>
  <meta name="citation_reference" content="Lanave C, Preparata G, Sacone C, Serio G. A new method for calculating evolutionary substitution rates. J Mol Evol [Internet]. 1984;20(1):86–93. Available from: https://doi.org/10.1007/BF02101990"/>
  <meta name="citation_reference" content="Tavaré S. Some Probabilistic and Statistical Problems in the Analysis of DNA Sequences. Am Math Soc Lect Math Life Sci [Internet]. 1986;17:57–86. Available from: citeulike-article-id:6732633"/>
  <meta name="citation_reference" content="Webster RG, Bean WJ, Gorman OT, Chambers TM, Kawaoka Y. Evolution and ecology of influenza A viruses. Microbiol Rev [Internet]. 1992 Mar 1 [cited 2019 Feb 12];56(1):152–79. Available from: http://www.ncbi.nlm.nih.gov/pubmed/1579108"/>
  <meta name="citation_reference" content="Baele G, Suchard MA, Rambaut A, Lemey P. Emerging Concepts of Data Integration in Pathogen Phylodynamics. Syst Biol [Internet]. 2017 Jun 1 [cited 2018 Dec 2];66(1):e47–65. Available from: https://academic.oup.com/sysbio/article/doi/10.1093/sysbio/syw054/2670001/Emerging-concepts-of-data-integration-in-pathogen"/>
  <meta name="citation_reference" content="Koonin E, Galperin M. Evolutionary Concept in Genetics and Genomics. In: Sequence - Evolution - Function: Computational Approaches in Comparative Genomics. Boston: Kluwer Academic; 2003."/>
  <meta name="citation_reference" content="Shapiro B, Rambaut A, Drummond AJ. Choosing Appropriate Substitution Models for the Phylogenetic Analysis of Protein-Coding Sequences. Mol Biol Evol [Internet]. 2006 Jan 1;23(1):7–9. Available from: http://dx.doi.org/10.1093/molbev/msj021"/>
  <meta name="citation_reference" content="Riwilaijaroen BNS, Uzuki YS. Review Molecular basis of the structure and function of H1 hemagglutinin of in fl uenza virus. 2012;88:226–49."/>
  <meta name="citation_reference" content="Wiley DC, Skehel JJ. The Structure and Function of the Hemagglutinin Membrane Glycoprotein of Influenza Virus. Annu Rev Biochem [Internet]. 1987 Jun 1 [cited 2019 Feb 13];56(1):365–94. Available from: http://www.ncbi.nlm.nih.gov/pubmed/3304138"/>
  <meta name="citation_reference" content="Nachbagauer R, Miller MS, Hai R, Ryder AB, Rose JK, Palese P, et al. Hemagglutinin Stalk Immunity Reduces Influenza Virus Replication and Transmission in Ferrets. 2016;90(6):3268–73."/>
  <meta name="citation_reference" content="Wang TT, Palese P. Catching a Moving Target. Science (80-) [Internet]. 2011 Aug 12;333(6044):834 LP – 835. Available from: http://science.sciencemag.org/content/333/6044/834.abstract"/>
  <meta name="citation_reference" content="Krammer F, Hai R, Yondola M, Tan GS, Leyva-Grado VH, Ryder AB, et al. Assessment of influenza virus hemagglutinin stalk-based immunity in ferrets. J Virol [Internet]. 2014 Mar 15 [cited 2017 Nov 16];88(6):3432–42. Available from: http://www.ncbi.nlm.nih.gov/pubmed/24403585"/>
  <meta name="citation_reference" content="Krammer F, Palese P. Influenza virus hemagglutinin stalk-based antibodies and vaccines. Curr Opin Virol [Internet]. 2013;3(5):521–30. Available from: http://www.sciencedirect.com/science/article/pii/S187962571300134X"/>
  <meta name="citation_reference" content="Subbarao K, Matsuoka Y. The prospects and challenges of universal vaccines for influenza. Trends Microbiol [Internet]. 2013 Nov 15;21(7):350–8. Available from: http://dx.doi.org/10.1016/j.tim.2013.04.003"/>
  <meta name="citation_reference" content="Shapiro B, Rambaut A, Drummond AJ. Choosing Appropriate Substitution Models for the Phylogenetic Analysis of Protein-Coding Sequences. Mol Biol Evol [Internet]. 2006 Jan 1 [cited 2019 Feb 20];23(1):7–9. Available from: http://academic.oup.com/mbe/article/23/1/7/1193608/Choosing-Appropriate-Substitution-Models-for-the"/>
  <meta name="citation_reference" content="Mintaev RR, Alexeevski A V, Kordyukova L V. Co-evolution analysis to predict protein– protein interactions within influenza virus envelope. J Bioinform Comput Biol [Internet]. 2014 Mar 6;12(02):1441008. Available from: https://doi.org/10.1142/S021972001441008X"/>
  <meta name="citation_reference" content="Li C, Lu G, Ortí G. Optimal Data Partitioning and a Test Case for Ray-Finned Fishes (Actinopterygii) Based on Ten Nuclear Loci. Buckley T, editor. Syst Biol [Internet]. 2008 Aug 1 [cited 2019 Apr 18];57(4):519–39. Available from: http://www.ncbi.nlm.nih.gov/pubmed/18622808"/>
  <meta name="citation_reference" content="Brandley MC, Schmitz A, Reeder TW. Partitioned Bayesian Analyses, Partition Choice, and the Phylogenetic Relationships of Scincid Lizards. Anderson F, editor. Syst Biol [Internet]. 2005 Jun 1 [cited 2019 Apr 18];54(3):373–90. Available from: http://www.ncbi.nlm.nih.gov/pubmed/16012105"/>
  <meta name="citation_reference" content="McGuire JA, Witt CC, Altshuler DL, Remsen JV. Phylogenetic Systematics and Biogeography of Hummingbirds: Bayesian and Maximum Likelihood Analyses of Partitioned Data and Selection of an Appropriate Partitioning Strategy. Zamudio K, Sullivan J, editors. Syst Biol [Internet]. 2007 Oct 1 [cited 2019 Apr 18];56(5):837–56. Available from: http://www.ncbi.nlm.nih.gov/pubmed/17934998"/>
  <meta name="citation_reference" content="Swanson KA, Settembre EC, Shaw CA, Dey AK, Rappuoli R, Mandl CW, et al. Structural basis for immunization with postfusion respiratory syncytial virus fusion F glycoprotein (RSV F) to elicit high neutralizing antibody titers. Proc Natl Acad Sci U S A [Internet]. 2011 Jun 7 [cited 2019 Mar 12];108(23):9619–24. Available from: http://www.ncbi.nlm.nih.gov/pubmed/21586636"/>
  <meta name="citation_reference" content="Kordyukova L. Structural and functional specificity of Influenza virus haemagglutinin and paramyxovirus fusion protein anchoring peptides. Virus Res [Internet]. 2017;227(Supplement C):183–99. Available from: http://www.sciencedirect.com/science/article/pii/S0168170216304063"/>
  <meta name="citation_reference" content="Lu X, Shi Y, Gao F, Xiao H, Wang M, Qi J, et al. Insights into Avian Influenza Virus Pathogenicity: the Hemagglutinin Precursor HA0 of Subtype H16 Has an Alpha-Helix Structure in Its Cleavage Site with Inefficient HA1/HA2 Cleavage. J Virol [Internet]. 2012 Dec 25;86(23):12861–70. Available from: http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3497694/"/>
  <meta name="citation_reference" content="Skehel JJ, Wiley DC. Receptor Binding and Membrane Fusion in Virus Entry: The Influenza Hemagglutinin. Annu Rev Biochem [Internet]. 2000 Jun 1;69(1):531–69. Available from: https://doi.org/10.1146/annurev.biochem.69.1.531"/>
  <meta name="citation_reference" content="Baele G, Lemey P, Bedford T, Rambaut A, Suchard MA, Alekseyenko A V. Improving the Accuracy of Demographic and Molecular Clock Model Comparison While Accommodating Phylogenetic Uncertainty Research article. 2012;29(9):2157–67."/>
  <meta name="citation_reference" content="Raftery A, Newton M, Satagopan J, Krivitsky P. Estimating the Integrated Likelihood via Posterior Simulation Using the Harmonic Mean Identity. Meml Sloan-Kettering Cancer Center, Dept Epidemiol Biostat Work Pap Ser [Internet]. 2006 Apr 14 [cited 2017 Nov 16]; Available from: http://biostats.bepress.com/mskccbiostat/paper6"/>
  <meta name="citation_reference" content="Lartillot N, Philippe H, Lewis P. Computing Bayes Factors Using Thermodynamic Integration. Syst Biol [Internet]. 2006 Apr 1;55(2):195–207. Available from: http://dx.doi.org/10.1080/10635150500433722"/>
  <meta name="citation_reference" content="Xie W, Lewis PO, Fan Y, Kuo L, Chen M-H. Improving Marginal Likelihood Estimation for Bayesian Phylogenetic Model Selection. Syst Biol [Internet]. 2011 Mar 1 [cited 2019 May 5];60(2):150–60. Available from: http://www.ncbi.nlm.nih.gov/pubmed/21187451"/>
  <meta name="citation_reference" content="Baele G, Lemey P, Bedford T, Rambaut A, Suchard MA, Alekseyenko A V. Improving the accuracy of demographic and molecular clock model comparison while accomodating phylogenetic uncertainty. Mol Biol Evol [Internet]. 2012 Jan; Available from: http://www.ncbi.nlm.nih.gov/pubmed/17065594"/>
  <meta name="citation_reference" content="Baele G, Lok W, Li S, Drummond AJ, Suchard MA, Lemey P. Accurate Model Selection of Relaxed Molecular Clocks in Bayesian Phylogenetics Letter Fast Track. 2012;30(2):239–43."/>
  <meta name="citation_reference" content="Kandeil A, Sabir JSM, Abdelaal A, Mattar EH, El-Taweel AN, Sabir MJ, et al. Efficacy of commercial vaccines against newly emerging avian influenza H5N8 virus in Egypt. Sci Rep. 2018 Dec 1;8(1)."/>
  <meta name="citation_reference" content="Hamid Samaha MP. Avian influenza vaccination in Egypt: Limitations of the current strategy. J Mol Genet Med. 2009;03(02)."/>
  <meta name="citation_reference" content="Spielman SJ, Wilke CO. The relationship between dN/dS and scaled selection coefficients. Mol Biol Evol [Internet]. 2015 Apr [cited 2019 Jul 19];32(4):1097–108. Available from: http://www.ncbi.nlm.nih.gov/pubmed/25576365"/>
  <meta name="citation_reference" content="Elbe S, Buckland-Merrett G. Data, disease and diplomacy: GISAID’s innovative contribution to global health. Glob Challenges [Internet]. 2017 Jan 1 [cited 2019 Apr 29];1(1):33–46. Available from: http://doi.wiley.com/10.1002/gch2.1018"/>
  <meta name="citation_reference" content="Shu Y, McCauley J. GISAID: Global initiative on sharing all influenza data – from vision to reality. Eurosurveillance [Internet]. 2017 Mar 30 [cited 2019 Apr 28];22(13):30494. Available from: http://www.ncbi.nlm.nih.gov/pubmed/28382917"/>
  <meta name="citation_reference" content="Bedford T, Riley S, Barr IG, Broor S, Chadha M, Cox NJ, et al. Global circulation patterns of seasonal influenza viruses vary with antigenic drift. Nature [Internet]. 2015 Jun 8 [cited 2019 Feb 7];523(7559):217. Available from: http://www.nature.com/articles/nature14460"/>
  <meta name="citation_reference" content="Qiu X, Duvvuri VR, Gubbay JB, Webby RJ, Kayali G, Bahl J. specific epitope profiles for HPAI H5 pre-• pandemic vaccine selection and evaluation. 2017;(July):445–56."/>
  <meta name="citation_reference" content="Vijaykrishna D, Holmes EC, Joseph U, Fourment M, Su YCF, Halpin R, et al. The contrasting phylodynamics of human influenza B viruses. Elife. 2015;4:1–23."/>
  <meta name="citation_reference" content="Drummond AJ, Suchard MA, Xie D, Rambaut A. Bayesian Phylogenetics with BEAUti and the BEAST 1.7. Mol Biol Evol [Internet]. 2012 Aug [cited 2019 Feb 5];29(8):1969–73. Available from: http://www.ncbi.nlm.nih.gov/pubmed/22367748"/>
  <meta name="citation_reference" content="Minin VN, Bloomquist EW, Suchard MA. Smooth Skyride through a Rough Skyline: Bayesian Coalescent-Based Inference of Population Dynamics. Mol Biol Evol [Internet]. 2008 Apr 3 [cited 2019 Feb 20];25(7):1459–71. Available from: https://academic.oup.com/mbe/article-lookup/doi/10.1093/molbev/msn090"/>
  <meta name="citation_reference" content="Kass RE, Raftery AE. Bayes Factors. J Am Stat Assoc [Internet]. 1995 Jun 1;90(430):773–95. Available from: http://www.tandfonline.com/doi/abs/10.1080/01621459.1995.10476572"/>
  <meta name="twitter:title" content="Structurally informed evolutionary models improve phylogenetic reconstruction for emerging, seasonal, and pandemic influenza viruses"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Precise estimation of genetic substitution patterns is critical for accurate reconstruction of pathogen phylogenies. Few studies of viral evolution account for variations of mutation rate across a single gene. This is especially true when considering evolution of segmented viruses where individual segments are short, encoding for few proteins. However, the structural and functional partitions of these proteins could provide valuable information for more accurate inference of viral evolution, due to the disparate immune selection pressure on different functional domains. Accurately reconstructed evolutionary features on specific functional domains can in turn provide biological information on viral protein and immune targets for vaccine design. In this study we developed and evaluated a structurally informed partitioning scheme that accounts for rate variation among immunogenic head and stalk domains of the surface protein hemagglutinin (HA) of influenza viruses. We evaluated the model fit and performance of four different models - HKY, SRD06 codon, HKY with a structurally informed partitioning scheme, SRD06 with a structurally informed partitioning scheme on pandemic A/H1N1pdm09, seasonal A/H1N1postpdm, A/H3N2, B-Yamagata-like and Victoria-like lineages, and two highly pathogenic avian influenza A viruses H5Nx and H7N9. Results showed that structurally informed partitioning with SRD06 performed better for all datasets with decisively statistical support. Significantly faster nucleotide substitution rates for head domain, compared to stalk domain was observed and may provide insight for stalk derived broadly-reactive vaccine design. Taken together, integrating a functionally informed partitioning scheme based on protein structures of immune targets allows for significant improvement of phylogenetic analysis and providing important biological insights."/>
  <meta name="og-title" property="og:title" content="Structurally informed evolutionary models improve phylogenetic reconstruction for emerging, seasonal, and pandemic influenza viruses"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/228692v4"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Precise estimation of genetic substitution patterns is critical for accurate reconstruction of pathogen phylogenies. Few studies of viral evolution account for variations of mutation rate across a single gene. This is especially true when considering evolution of segmented viruses where individual segments are short, encoding for few proteins. However, the structural and functional partitions of these proteins could provide valuable information for more accurate inference of viral evolution, due to the disparate immune selection pressure on different functional domains. Accurately reconstructed evolutionary features on specific functional domains can in turn provide biological information on viral protein and immune targets for vaccine design. In this study we developed and evaluated a structurally informed partitioning scheme that accounts for rate variation among immunogenic head and stalk domains of the surface protein hemagglutinin (HA) of influenza viruses. We evaluated the model fit and performance of four different models - HKY, SRD06 codon, HKY with a structurally informed partitioning scheme, SRD06 with a structurally informed partitioning scheme on pandemic A/H1N1pdm09, seasonal A/H1N1postpdm, A/H3N2, B-Yamagata-like and Victoria-like lineages, and two highly pathogenic avian influenza A viruses H5Nx and H7N9. Results showed that structurally informed partitioning with SRD06 performed better for all datasets with decisively statistical support. Significantly faster nucleotide substitution rates for head domain, compared to stalk domain was observed and may provide insight for stalk derived broadly-reactive vaccine design. Taken together, integrating a functionally informed partitioning scheme based on protein structures of immune targets allows for significant improvement of phylogenetic analysis and providing important biological insights."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2019-12-13"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Structurally informed evolutionary models improve phylogenetic reconstruction for emerging, seasonal, and pandemic influenza viruses | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1036469 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-2">Precise estimation of genetic substitution patterns is critical for accurate reconstruction of pathogen phylogenies. Few studies of viral evolution account for variations of mutation rate across a single gene. This is especially true when considering evolution of segmented viruses where individual segments are short, encoding for few proteins. However, the structural and functional partitions of these proteins could provide valuable information for more accurate inference of viral evolution, due to the disparate immune selection pressure on different functional domains. Accurately reconstructed evolutionary features on specific functional domains can in turn provide biological information on viral protein and immune targets for vaccine design. In this study we developed and evaluated a structurally informed partitioning scheme that accounts for rate variation among immunogenic head and stalk domains of the surface protein hemagglutinin (HA) of influenza viruses. We evaluated the model fit and performance of four different models - HKY, SRD06 codon, HKY with a structurally informed partitioning scheme, SRD06 with a structurally informed partitioning scheme on pandemic A/H1N1pdm09, seasonal A/H1N1postpdm, A/H3N2, B-Yamagata-like and Victoria-like lineages, and two highly pathogenic avian influenza A viruses H5Nx and H7N9. Results showed that structurally informed partitioning with SRD06 performed better for all datasets with decisively statistical support. Significantly faster nucleotide substitution rates for head domain, compared to stalk domain was observed and may provide insight for stalk derived broadly-reactive vaccine design. Taken together, integrating a functionally informed partitioning scheme based on protein structures of immune targets allows for significant improvement of phylogenetic analysis and providing important biological insights.</p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Introduction</h2>
    <p id="p-4">The importance of statistical phylogenetic methods to study the epidemiology, evolution and ecology of rapidly evolving viral pathogens has been driven by the growing availability of whole genome sequences (
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>). Precisely estimating the pattern of genetic variations is critical to reconstruct the accurate pathogen phylogenies (
     <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>, 
     <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>). Nucleotide substitution models have been developed to describe the process of change from one nucleic state to another among viral isolates, where they often allow for rate variations between transitions and transversions (
     <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>), or incorporate nucleotide base frequencies with substitution rate parameters (
     <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>, 
     <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>). Segmented viruses, such as influenza A virus (IAV), contain short gene segments that encode for few proteins (
     <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>). Consequently, the literature regarding model development of phylogeny reconstruction is dominated by the analysis on complete protein coding regions. For example, the complete viral surface glycoprotein hemagglutinin (HA) is used the most when studying influenza viral evolution and diffusion (
     <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>). However, assuming natural selection acts uniformly across the protein domains may not be justified (
     <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>). Some partitioning strategies have been developed to account for variations inside one gene segment (
     <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>, 
     <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>). For example, approximate codon-models, such as the SRD06, incorporate information about the genetic code by allowing for a rate variation by defining substitution models for codon positions 1 and 2 and an independent model for codon position 3. This model accounts for the rapid accumulation of synonymous mutations in the third codon position (
     <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>).
    </p>
    <p id="p-5">HA glycoprotein with a spike-shaped structure binds to the receptors on the targeted host cell membrane when the virus begins the infection (
     <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>). This protein has two main domains – the globular head domain and stalk domain. Functionally different, the globular head domain contains the receptor binding sites, while the stalk domain is a main structure responsible for membrane fusion machinery (
     <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>, 
     <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>). Even though immune selection strongly drives antigenic drift, allowing for accumulation of mutations in the head domain, the stalk domain is functionally conserved across viral subtypes. Current seasonal influenza virus vaccines induce humoral immune responses primarily targeting the immunodominant globular head domain (
     <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>, 
     <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>), which can provide strong protection from an infecting virus that is similar with vaccine candidates, but have weak cross-reactivity to antigenic drift variants. Conversely, studies in the ferret experimental model demonstrate that the stalk domain is highly conserved across IAV subtypes and stalk specific antibodies could provide cross-reactive protection to a high diversity of IAV subtypes (
     <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>, 
     <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>). Despite the importance of understanding how substitution rates and patterns vary across functionally conserved domains in universal vaccine design (
     <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>, 
     <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>), few studies have incorporated protein structure in phylogenetic models.
    </p>
    <p id="p-6">In this study we proposed a novel phylogenetic model that incorporates a protein structure informed partitioning scheme to account for variable evolutionary rates resulting from immune selection. We aimed to evaluate the appropriateness of the novel model to reliably to estimate biologically informative parameters from available genetic data. We evaluated and compared four different models: HKY, SRD06 codon, HKY with a structurally informed partitioning scheme, SRD06 with a structurally informed partitioning scheme. We also evaluated the model performance across multiple viral subtypes, including pandemic H1N1pdm09, seasonal A/H1N1postpdm, A/H3N2, the two seasonal Influenza B virus lineages – Yamagata-like and Victoria-like, and two highly pathogenic avian Influenza (HPAI) H5Nx and H7N9 viruses. We further conducted sensitivity analysis to determine whether the new model was sensitive to sample size, data distribution and epidemic stage resulting in biased estimation. Separate estimation of viral evolutionary rates for the head and stalk domain may be informative for vaccine design. We therefore formally tested the hypothesis that evolutionary rates in each domain were significantly different and further estimated the relative selection pressure for each functional domain of HA protein.</p>
    <div id="sec-2" class="subsection">
     <h3>New Approaches</h3>
     <p id="p-7">This study proposed a new model to incorporate a structurally informed partitioning scheme on a single protein into phylogenetic reconstruction. Decisive statistical support from the model selection procedure and stably supported across multiple influenza subtypes validated the superiority of the new model. The model can provide biological insights for viral evolution, with reporting domain-specific evolution rates and approximate selection pressure. The tree branches with domain-specific rate ratio 
      <em>d</em>C
      <sub>1+2</sub>/
      <em>d</em>C
      <sub>3</sub> can inform the approximate selection pressure on each strain, suggesting some biological explanations related to antigenic drift and emerging strains. The structurally informed phylogenetic model may reveal novel biological insights of viral evolution and have the potential to reveal more biological realism without over-parameterization.
     </p>
    </div>
   </div>
   <div class="section" id="sec-3">
    <h2 class="">Results</h2>
    <div id="sec-4" class="subsection">
     <h3>Model selection</h3>
     <p id="p-8">To compare the four models, the log scale Bayes Factor (BF) was used, which was the difference of log marginal likelihood estimations from two models. Marginal likelihood estimation was conducted with two approaches – path-sampling and stepping-stone sampling, where both generated consistent results. Based on the model selection procedure for all subtypes (
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a>), the model with mixed partitioning on both codon positions and HA protein structure (CP model in 
      <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1c</a>) had decisive BF support for better model fitting in all datasets compared to HKY and HKY with protein structure partitioning model (HKY model and P model in 
      <a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1c</a>). Compared to SRD06 codon model (C model in 
      <a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1c</a>), CP model performed significantly better for all datasets, with different levels of BF supports. HKY with protein structure model (P model) performed significantly better than HKY model for human influenza A and B viruses, but it was not significant in highly pathogenic avian influenza H7N9 and H5Nx datasets. This may indicate that these emerging viruses with avian-dominated host populations have a different immune selection pressure pattern compared to human influenza viruses.
     </p>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/228692/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Conceptual diagram for influenza virus Hemagglutinins (HA) domains and four models tested 1a. HA protein 3D structure: Stalk and Head domains. Head domain is the immunodominant with high plasticity, which contains the receptor binding sites. Stalk domain is immune-subdominant and more conserved. 1b. Linear diagram for functional domains on HA nucleotide sequence. The stalk domain has two parts separated by head domain on the linear sequence. To compensate for short length, signal peptide, transmembrane domain, and cytoplasmic tail are combined into one partition, referred to as [[#x201C]]STC[[#x201D]] in the analysis. 1c. Model conceptual design. Four models are applied and compared for each dataset. HKY model is a substitution model that considers different base frequencies and assigns different rates for transitions v.s. transversions. C model represents SRD06 codon position model. The partitioning strategy is to analyze codon positions 1 + 2 and codon position 3 separately. P model takes the protein structure partitions into account based on the amino acid positions for each domain on the linear diagram. CP model combines both C and P models, which estimates codon positions in protein structural partitions. All models use HKY substitution model." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-865403216" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Conceptual diagram for influenza virus Hemagglutinins &lt;;strong&gt;;(&lt;;/strong&gt;;HA) domains and four models tested &lt;;strong&gt;;&lt;;em&gt;;1a. HA protein 3D structure: Stalk and Head domains.&lt;;/em&gt;;&lt;;/strong&gt;; Head domain is the immunodominant with high plasticity, which contains the receptor binding sites. Stalk domain is immune-subdominant and more conserved. &lt;;strong&gt;;&lt;;em&gt;;1b. Linear diagram for functional domains on HA nucleotide sequence&lt;;/em&gt;;&lt;;/strong&gt;;. The stalk domain has two parts separated by head domain on the linear sequence. To compensate for short length, signal peptide, transmembrane domain, and cytoplasmic tail are combined into one partition, referred to as [[#x201C]]STC[[#x201D]] in the analysis. &lt;;strong&gt;;&lt;;em&gt;;1c. Model conceptual design.&lt;;/em&gt;;&lt;;/strong&gt;; Four models are applied and compared for each dataset. &lt;;strong&gt;;HKY model&lt;;/strong&gt;; is a substitution model that considers different base frequencies and assigns different rates for transitions v.s. transversions. &lt;;strong&gt;;C model&lt;;/strong&gt;; represents SRD06 codon position model. The partitioning strategy is to analyze codon positions 1 + 2 and codon position 3 separately. &lt;;strong&gt;;P model&lt;;/strong&gt;; takes the protein structure partitions into account based on the amino acid positions for each domain on the linear diagram. &lt;;strong&gt;;CP model&lt;;/strong&gt;; combines both C and P models, which estimates codon positions in protein structural partitions. All models use HKY substitution model.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/228692/F1.medium.gif" width="412" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/228692/F1.medium.gif" width="412" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/228692/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/228692/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 1.</span>
       <p id="p-9" class="first-child">Conceptual diagram for influenza virus Hemagglutinins 
        <strong>(</strong>HA) domains and four models tested
       </p>
       <p id="p-10">
        <strong>
         <em>1a. HA protein 3D structure: Stalk and Head domains.</em>
        </strong> Head domain is the immunodominant with high plasticity, which contains the receptor binding sites. Stalk domain is immune-subdominant and more conserved. 
        <strong>
         <em>1b. Linear diagram for functional domains on HA nucleotide sequence</em>
        </strong>. The stalk domain has two parts separated by head domain on the linear sequence. To compensate for short length, signal peptide, transmembrane domain, and cytoplasmic tail are combined into one partition, referred to as “STC” in the analysis. 
        <strong>
         <em>1c. Model conceptual design.</em>
        </strong> Four models are applied and compared for each dataset. 
        <strong>HKY model</strong> is a substitution model that considers different base frequencies and assigns different rates for transitions v.s. transversions. 
        <strong>C model</strong> represents SRD06 codon position model. The partitioning strategy is to analyze codon positions 1 + 2 and codon position 3 separately. 
        <strong>P model</strong> takes the protein structure partitions into account based on the amino acid positions for each domain on the linear diagram. 
        <strong>CP model</strong> combines both C and P models, which estimates codon positions in protein structural partitions. All models use HKY substitution model.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Tree root height and substitution rates</h3>
     <p id="p-11">To evaluate the accuracy and stability of model performance, tree parameters and estimated substitution rates were used to compare different models. Results from four models showed very similar tree root height and 95% Bayesian Credible Interval (BCI) for each influenza subtype (
      <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The estimated mean substitution rates (
      <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3</a>) estimated from all models were similar for each dataset suggesting that the CP model generated reliable estimations for important evolutionary parameters that were comparable to other models. Furthermore, since both CP and P model incorporated partitioning scheme on protein structure, these two models provided domain-specific substitution rates for STC (including signal peptide, transmembrane domain and cytoplasmic tail), stalk and head domains, respectively. However, the domain-specific rates reported by P model were underestimated, compared to CP model (results shown in Supplemental Table 2). Such an observation was unsurprising since the HKY substitution model on each protein structure partition cannot account for more realistic variations among codon positions. One thing to note, though STC domain was not partitioned on codon positions, its small amount of information from short nucleotide length (about 150 nucleotides) resulted in high uncertainty of the STC substitution rates, showing long tails in the violin plots.
     </p>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1041383/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1041383/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1041383" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 1.</span>
       <span class="caption-title">Tree root height estimation and its 95% Bayesian credible interval (BCI) from each model for each influenza subtype.</span>
       <p id="p-12" class="first-child">Results from four models show very similar tree root height and 95% BCI for each influenza subtype. The unit of the root height is in years.</p>
       <p id="p-13">
        <strong>Models: HKY model</strong> is a substitution model that considers different base frequencies and assigns different rates for transitions v.s. transversions. The 
        <strong>c model</strong> represents SRD06 codon position model. The partitioning strategy is to analyze codon positions 1 + 2 and codon position 3 separately. The 
        <strong>p model</strong> takes the protein structure partitions into account based on the amino acid positions for each domain on the linear diagram. The 
        <strong>cp model</strong> combines both c and p models, which estimates codon positions in protein structural partitions. All models use HKY substitution model.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/228692/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Log scale Bayes Factors for model selection procedure for each influenza subtype. All these Bayes Factors (BF) are calculated to compare with HKY via path-sampling (upper panel) and stepping-stone sampling (lower panel) approaches to compute the marginal likelihood estimates for each model. The asterisk * means a significant supportive BF for the structurally informed codon model (cp), compared to SRD06 codon only model (c model). Both cp and c models perform better than HKY model with decisive BF supports in all datasets. The HKY plus structurally informed model (p model) perform better in all datasets except for H7N9. Log scale BF criteria: BF&lt;; 5 represents no significance; 5 [[#x2264]] BF&lt;; 10 means substantial support; 10 [[#x2264]] BF&lt;; 15 means strong support; 15 [[#x2264]] BF &lt;; 20 means very strong support; BF [[#x2265]] 20 means decisive support. Models: HKY model is a substitution model that considers different base frequencies and assigns different rates for transitions v.s. transversions. The c model represents SRD06 codon position model. The partitioning strategy is to analyze codon positions 1 + 2 and codon position 3 separately. The p model takes the protein structure partitions into account based on the amino acid positions for each domain on the linear diagram. The cp model combines both c and p models, which estimates codon positions in protein structural partitions. All models use HKY substitution model." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-865403216" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Log scale Bayes Factors for model selection procedure for each influenza subtype.&lt;;/span&gt;; All these Bayes Factors (BF) are calculated to compare with HKY via path-sampling (upper panel) and stepping-stone sampling (lower panel) approaches to compute the marginal likelihood estimates for each model. The asterisk * means a significant supportive BF for the structurally informed codon model (cp), compared to SRD06 codon only model (c model). Both cp and c models perform better than HKY model with decisive BF supports in all datasets. The HKY plus structurally informed model (p model) perform better in all datasets except for H7N9. Log scale BF criteria: BF&lt;; 5 represents no significance; 5 [[#x2264]] BF&lt;; 10 means substantial support; 10 [[#x2264]] BF&lt;; 15 means strong support; 15 [[#x2264]] BF &lt;; 20 means very strong support; BF [[#x2265]] 20 means decisive support. &lt;;strong&gt;;Models: HKY model&lt;;/strong&gt;; is a substitution model that considers different base frequencies and assigns different rates for transitions v.s. transversions. The &lt;;strong&gt;;c model&lt;;/strong&gt;; represents SRD06 codon position model. The partitioning strategy is to analyze codon positions 1 + 2 and codon position 3 separately. The &lt;;strong&gt;;p model&lt;;/strong&gt;; takes the protein structure partitions into account based on the amino acid positions for each domain on the linear diagram. The &lt;;strong&gt;;cp model&lt;;/strong&gt;; combines both c and p models, which estimates codon positions in protein structural partitions. All models use HKY substitution model.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/228692/F2.medium.gif" width="440" height="314"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/228692/F2.medium.gif" width="440" height="314"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/228692/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/228692/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 2.</span>
       <span class="caption-title">Log scale Bayes Factors for model selection procedure for each influenza subtype.</span>
       <p id="p-14" class="first-child">All these Bayes Factors (BF) are calculated to compare with HKY via path-sampling (upper panel) and stepping-stone sampling (lower panel) approaches to compute the marginal likelihood estimates for each model. The asterisk * means a significant supportive BF for the structurally informed codon model (cp), compared to SRD06 codon only model (c model). Both cp and c models perform better than HKY model with decisive BF supports in all datasets. The HKY plus structurally informed model (p model) perform better in all datasets except for H7N9. Log scale BF criteria: BF&lt;; 5 represents no significance; 5 ≤ BF&lt;; 10 means substantial support; 10 ≤ BF&lt;; 15 means strong support; 15 ≤ BF &lt;; 20 means very strong support; BF ≥ 20 means decisive support.</p>
       <p id="p-15">
        <strong>Models: HKY model</strong> is a substitution model that considers different base frequencies and assigns different rates for transitions v.s. transversions. The 
        <strong>c model</strong> represents SRD06 codon position model. The partitioning strategy is to analyze codon positions 1 + 2 and codon position 3 separately. The 
        <strong>p model</strong> takes the protein structure partitions into account based on the amino acid positions for each domain on the linear diagram. The 
        <strong>cp model</strong> combines both c and p models, which estimates codon positions in protein structural partitions. All models use HKY substitution model.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/228692/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Overall mean substitution rates estimated from four models and domain-specific substitution rates from structurally informed partitioning model. Violin plots show the substitution rates from each step of Bayesian simulation. The results show overall mean substitution rates from four models for each dataset are similar, but the head domain has a significantly faster rate than that of stalk domain based on the new model (cp model). Generally, STC domain has more uncertain estimation with long tails of the violin plot, which is probably due to short length of nucleotides. Bayes Factors for the statistical tests on the differences of head and stalk domain substitution rates are 1,137 for H1N1pdm09, 6 for H1N1postpdm, 577 for A/H3N2, 197 for B-Victoria-like, 191 for B-Yamagata-like, 303 for H7N9 and +Infinity for H5Nx. Models: HKY model is a substitution model that considers different base frequencies and assigns different rates for transitions v.s. transversions. The c model represents SRD06 codon position model. The partitioning strategy is to analyze codon positions 1 + 2 and codon position 3 separately. The p model takes the protein structure partitions into account based on the amino acid positions for each domain on the linear diagram. The cp model combines both c and p models, which estimates codon positions in protein structural partitions. All models use HKY substitution model." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-865403216" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Overall mean substitution rates estimated from four models and domain-specific substitution rates from structurally informed partitioning model.&lt;;/span&gt;; Violin plots show the substitution rates from each step of Bayesian simulation. The results show overall mean substitution rates from four models for each dataset are similar, but the head domain has a significantly faster rate than that of stalk domain based on the new model (cp model). Generally, STC domain has more uncertain estimation with long tails of the violin plot, which is probably due to short length of nucleotides. Bayes Factors for the statistical tests on the differences of head and stalk domain substitution rates are 1,137 for H1N1pdm09, 6 for H1N1postpdm, 577 for A/H3N2, 197 for B-Victoria-like, 191 for B-Yamagata-like, 303 for H7N9 and +Infinity for H5Nx. &lt;;strong&gt;;Models: HKY model&lt;;/strong&gt;; is a substitution model that considers different base frequencies and assigns different rates for transitions v.s. transversions. The &lt;;strong&gt;;c model&lt;;/strong&gt;; represents SRD06 codon position model. The partitioning strategy is to analyze codon positions 1 + 2 and codon position 3 separately. The &lt;;strong&gt;;p model&lt;;/strong&gt;; takes the protein structure partitions into account based on the amino acid positions for each domain on the linear diagram. The &lt;;strong&gt;;cp model&lt;;/strong&gt;; combines both c and p models, which estimates codon positions in protein structural partitions. All models use HKY substitution model.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/228692/F3.medium.gif" width="289" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/228692/F3.medium.gif" width="289" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/228692/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/228692/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 3.</span>
       <span class="caption-title">Overall mean substitution rates estimated from four models and domain-specific substitution rates from structurally informed partitioning model.</span>
       <p id="p-16" class="first-child">Violin plots show the substitution rates from each step of Bayesian simulation. The results show overall mean substitution rates from four models for each dataset are similar, but the head domain has a significantly faster rate than that of stalk domain based on the new model (cp model). Generally, STC domain has more uncertain estimation with long tails of the violin plot, which is probably due to short length of nucleotides. Bayes Factors for the statistical tests on the differences of head and stalk domain substitution rates are 1,137 for H1N1pdm09, 6 for H1N1postpdm, 577 for A/H3N2, 197 for B-Victoria-like, 191 for B-Yamagata-like, 303 for H7N9 and +Infinity for H5Nx.</p>
       <p id="p-17">
        <strong>Models: HKY model</strong> is a substitution model that considers different base frequencies and assigns different rates for transitions v.s. transversions. The 
        <strong>c model</strong> represents SRD06 codon position model. The partitioning strategy is to analyze codon positions 1 + 2 and codon position 3 separately. The 
        <strong>p model</strong> takes the protein structure partitions into account based on the amino acid positions for each domain on the linear diagram. The 
        <strong>cp model</strong> combines both c and p models, which estimates codon positions in protein structural partitions. All models use HKY substitution model.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-18">Comparing the overall substitution rates for each influenza subtype, we observed that the pandemic H1N1pdm09 had the highest substitution rate (∼7.5 × 10-3 substitution/site/year). HPAI H7N9 and H5Nx had the substitution rates around 4.0 – 4.8 × 10-3 substitution/site/year. Seasonal influenza viruses had the range as 2.0 – 3.5 × 10-3 substitution/site/year, where the two type B lineages had lower rates than the two A subtypes.</p>
     <p id="p-19">Furthermore, we compared the separate substitution rates for stalk and head domains from the CP model. The absolute rates calculated directly from each step of the Bayesian Markov Chain Monte Carlo (MCMC) simulation were plotted into violin plot (
      <a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3</a>). Results showed that the head domain had a significantly higher substitution rate than stalk domain for all influenza subtypes, under the unit of substitutions per site per year. The Bayes Factors (BF) for the statistical tests between the substitution rates of head and stalk domains were 1137 (H1N1pdm09), 6 (H1Npostpdm), 577 (A/H3N2), 197 (B-Victoria), 191 (B-Yamagata), 303 (H7N9), +infinity (H5Nx), respectively, with the significant criteria as BF&gt;;3.
     </p>
    </div>
    <div id="sec-6" class="subsection">
     <h3>Approximate selection pressure</h3>
     <p id="p-20">With partitioning on codon positions, the CP model can estimate the ratio of substitution rate of codon positions 1 and 2 (majority of non-synonymous changes occur in these codons) over the rate of codon position 3 (majority changes are synonymous), that is, 
      <em>d</em>C
      <sub>1+2</sub>/
      <em>d</em>C
      <sub>3</sub>, for stalk domain and head domain separately. The relative rate of C
      <sub>3</sub> to C
      <sub>1+2</sub> is a good predictor of the ω parameter of selection pressure measure (
      <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>) and thus could reflect the domain specific selection pressure. As shown in 
      <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2.2</a>, the range of 
      <em>d</em>C
      <sub>1+2</sub>/
      <em>d</em>C
      <sub>3</sub> was 0.10 – 0.32 for stalk domain and 0.25 – 0.87 for head domain from all subtypes of influenza viruses included in the study. The overall 
      <em>d</em>C
      <sub>1+2</sub>/
      <em>d</em>C
      <sub>3</sub> for the whole HA protein calculated from the c model was lower than head domain but higher than stalk domain for each influenza subtype. We also observed that the stalk domain had a significantly lower 
      <em>d</em>C
      <sub>1+2</sub>/
      <em>d</em>C
      <sub>3</sub> than head domain for each influenza subtype (Bayes Factors shown in 
      <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>), which indicated that stalk domain experienced stronger purifying selection to maintain its conservation. Further to compare pandemic H1N1pdm09 and seasonal A/H1N1 post pandemic, we found that the seasonal H1N1 post pandemic had lower 
      <em>d</em>C
      <sub>1+2</sub>/
      <em>d</em>C
      <sub>3</sub> in both stalk and head domains, indicating pandemic H1N1pdm09 experienced less purifying selection.
     </p>
     <div id="T2" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1041380/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1041380/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1041380" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 2.</span>
       <span class="caption-title">Substitution rate and dC1+2/dC3 of stalk and head domain for each influenza subtype.</span>
       <p id="p-21" class="first-child">Compared to the head domain, the stalk domain with lower value of 
        <em>d</em>C
        <sub>1+2</sub>/
        <em>d</em>C
        <sub>3</sub> experiences stronger purifying selection to maintain its conserved functionality.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-22">To observe different patterns of selection pressure on the viral populations, phylogenetic tree branch-specific stalk 
      <em>d</em>C
      <sub>1+2</sub>/
      <em>d</em>C
      <sub>3</sub> and head 
      <em>d</em>C
      <sub>1+2</sub>/
      <em>d</em>C
      <sub>3</sub> were calculated and visualized via mapping the upper and lower quantiles of the ratio values onto the phylogenetic tree. We used seasonal H3N2 (Supplemental Fig. 1) and Egypt H5Nx (Supplemental Fig. 2) as examples. The 
      <em>d</em>C
      <sub>1+2</sub>/
      <em>d</em>C
      <sub>3</sub> had a range of 0.63 – 0.75 for H3N2 head domain, 0.21 – 0.23 for H3N2 stalk domain, 0.12 – 1.87 for Egypt H5Nx head domain and 0.09 – 0.32 for Egypt H5Nx stalk domain. Again, results from the branch-specific 
      <em>d</em>C
      <sub>1+2</sub>/
      <em>d</em>C
      <sub>3</sub> demonstrated that the head domain had significantly higher 
      <em>d</em>C
      <sub>1+2</sub>/
      <em>d</em>C
      <sub>3</sub> compared to stalk domain, which indicated that stalk domain was under stronger purifying selection pressure to maintain its conservation. To note, the majority of the 
      <em>d</em>C
      <sub>1+2</sub>/
      <em>d</em>C
      <sub>3</sub> values were less than 1, which indicated purifying selection, but occasionally some diversifying selection pressure (
      <em>d</em>C
      <sub>1+2</sub>/
      <em>d</em>C
      <sub>3</sub> &gt;;1) was observed in Egypt H5Nx head domain in chicken-dominated population. This high 
      <em>d</em>C
      <sub>1+2</sub>/
      <em>d</em>C
      <sub>3</sub> in H5Nx chicken population may indicate that the viruses circulating in chicken may experience less purifying selection pressure to generate higher diversity with keeping more non-synonymous changes compared to that in human population. This suggests that avian population may generate more variants with pandemic potential.
     </p>
    </div>
    <div id="sec-7" class="subsection">
     <h3>Sensitivity analysis and model validation</h3>
     <p id="p-23">With the largest dataset HPAI H5Nx in this study, sensitivity analysis was performed to estimate the stability of model performance under different sample sizes. The same model fitting and selection procedure was tested on randomly selected 80%, 60%, 40% and 20% of the H5Nx-full dataset. The model selection procedure (
      <a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4</a>) showed that all these datasets had decisive BFs to favor the structurally informed partitioning model with SRD06 (CP model). Tree root height estimation (
      <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>) from these datasets had similar estimates and 95% BCI, where it was only slightly overestimated in the 20% of H5Nx dataset. The substitution rates of protein structure partitions (
      <a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5</a>) also showed stable estimations across different sample sizes but only had slight overestimation with the 20% H5Nx dataset. Taken together, though the new model with partitioning on both codon positions and protein functional domains introduced more parameters, it performed stably with different amount of data and thus was not sensitive to sample size.
     </p>
     <div id="F4" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/228692/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Log scale Bayes Factors for sensitivity analysis on full and subsampled HPAI H5Nx datasets. All these Bayes Factors (BF) are calculated to compare with HKY via path-sampling (upper panel) and stepping-stone sampling (lower panel) approaches to compute the marginal likelihood estimates for each model. The asterisk * represents a significant supportive BF for the structurally informed codon model (cp), compared to SRD06 codon only model (c model). Both cp and c models perform better than HKY model with decisive BF supports. The HKY plus structurally informed model (p model) performs better with supportive BFs in all datasets. Log scale BF criteria: BF&lt;; 5 represents no significance; 5 [[#x2264]] BF&lt;; 10 means substantial support; 10 [[#x2264]] BF&lt;; 15 means strong support; 15 [[#x2264]] BF &lt;; 20 means very strong support; BF [[#x2265]] 20 means decisive support. Models: HKY model is a substitution model that considers different base frequencies and assigns different rates for transitions v.s. transversions. The c model represents SRD06 codon position model. The partitioning strategy is to analyze codon positions 1 + 2 and codon position 3 separately. The p model takes the protein structure partitions into account based on the amino acid positions for each domain on the linear diagram. The cp model combines both c and p models, which estimates codon positions in protein structural partitions. All models use HKY substitution model." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-865403216" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Log scale Bayes Factors for sensitivity analysis on full and subsampled HPAI H5Nx datasets.&lt;;/span&gt;; All these Bayes Factors (BF) are calculated to compare with HKY via path-sampling (upper panel) and stepping-stone sampling (lower panel) approaches to compute the marginal likelihood estimates for each model. The asterisk * represents a significant supportive BF for the structurally informed codon model (cp), compared to SRD06 codon only model (c model). Both cp and c models perform better than HKY model with decisive BF supports. The HKY plus structurally informed model (p model) performs better with supportive BFs in all datasets. Log scale BF criteria: BF&lt;; 5 represents no significance; 5 [[#x2264]] BF&lt;; 10 means substantial support; 10 [[#x2264]] BF&lt;; 15 means strong support; 15 [[#x2264]] BF &lt;; 20 means very strong support; BF [[#x2265]] 20 means decisive support. &lt;;strong&gt;;Models: HKY model&lt;;/strong&gt;; is a substitution model that considers different base frequencies and assigns different rates for transitions v.s. transversions. The &lt;;strong&gt;;c model&lt;;/strong&gt;; represents SRD06 codon position model. The partitioning strategy is to analyze codon positions 1 + 2 and codon position 3 separately. The &lt;;strong&gt;;p model&lt;;/strong&gt;; takes the protein structure partitions into account based on the amino acid positions for each domain on the linear diagram. The &lt;;strong&gt;;cp model&lt;;/strong&gt;; combines both c and p models, which estimates codon positions in protein structural partitions. All models use HKY substitution model.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/228692/F4.medium.gif" width="440" height="344"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/228692/F4.medium.gif" width="440" height="344"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/228692/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/228692/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 4.</span>
       <span class="caption-title">Log scale Bayes Factors for sensitivity analysis on full and subsampled HPAI H5Nx datasets.</span>
       <p id="p-24" class="first-child">All these Bayes Factors (BF) are calculated to compare with HKY via path-sampling (upper panel) and stepping-stone sampling (lower panel) approaches to compute the marginal likelihood estimates for each model. The asterisk * represents a significant supportive BF for the structurally informed codon model (cp), compared to SRD06 codon only model (c model). Both cp and c models perform better than HKY model with decisive BF supports. The HKY plus structurally informed model (p model) performs better with supportive BFs in all datasets. Log scale BF criteria: BF&lt;; 5 represents no significance; 5 ≤ BF&lt;; 10 means substantial support; 10 ≤ BF&lt;; 15 means strong support; 15 ≤ BF &lt;; 20 means very strong support; BF ≥ 20 means decisive support.</p>
       <p id="p-25">
        <strong>Models: HKY model</strong> is a substitution model that considers different base frequencies and assigns different rates for transitions v.s. transversions. The 
        <strong>c model</strong> represents SRD06 codon position model. The partitioning strategy is to analyze codon positions 1 + 2 and codon position 3 separately. The 
        <strong>p model</strong> takes the protein structure partitions into account based on the amino acid positions for each domain on the linear diagram. The 
        <strong>cp model</strong> combines both c and p models, which estimates codon positions in protein structural partitions. All models use HKY substitution model.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F5" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/228692/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Sensitivity analysis: overall mean substitution rates and domain-specific substitution rates from structurally informed partitioning model for HPAI H5Nx subsampled datasets. Violin plots show the substitution rates from each step of the Bayesian simulations. Only 20% dataset has the trend to slightly exaggerate the rates and would not report the precise evolution history of H5Nx. The model superiority generally holds in the sensitivity analysis and the model is not sensitive to sample size. Models: The cp model combines both c and p models, which estimates codon positions in protein structural partitions. It uses HKY substitution model." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-865403216" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Sensitivity analysis: overall mean substitution rates and domain-specific substitution rates from structurally informed partitioning model for HPAI H5Nx subsampled datasets.&lt;;/span&gt;; Violin plots show the substitution rates from each step of the Bayesian simulations. Only 20% dataset has the trend to slightly exaggerate the rates and would not report the precise evolution history of H5Nx. The model superiority generally holds in the sensitivity analysis and the model is not sensitive to sample size. &lt;;strong&gt;;Models:&lt;;/strong&gt;; The &lt;;strong&gt;;cp model&lt;;/strong&gt;; combines both c and p models, which estimates codon positions in protein structural partitions. It uses HKY substitution model.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/228692/F5.medium.gif" width="440" height="165"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/228692/F5.medium.gif" width="440" height="165"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/228692/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/228692/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 5.</span>
       <span class="caption-title">Sensitivity analysis: overall mean substitution rates and domain-specific substitution rates from structurally informed partitioning model for HPAI H5Nx subsampled datasets.</span>
       <p id="p-26" class="first-child">Violin plots show the substitution rates from each step of the Bayesian simulations. Only 20% dataset has the trend to slightly exaggerate the rates and would not report the precise evolution history of H5Nx. The model superiority generally holds in the sensitivity analysis and the model is not sensitive to sample size.</p>
       <p id="p-27">
        <strong>Models:</strong> The 
        <strong>cp model</strong> combines both c and p models, which estimates codon positions in protein structural partitions. It uses HKY substitution model.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="T3" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1041385/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1041385/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1041385" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 3.</span>
       <span class="caption-title">Tree root height estimation and its 95% Bayesian credible interval (BCI) from each model for all H5Nx datasets.</span>
       <p id="p-28" class="first-child">The statistical significance of the new model holds across different subsampling datasets. Only 20% dataset has the trend to slightly overestimate the root height and would not report the precise evolution history of H5Nx. In sum, the new model is not sensitive to sample size. The unit of root height is in years.</p>
       <p id="p-29">
        <strong>Models: HKY model</strong> is a substitution model that considers different base frequencies and assigns different rates for transitions v.s. transversions. The 
        <strong>c model</strong> represents SRD06 codon position model. The partitioning strategy is to analyze codon positions 1 + 2 and codon position 3 separately. The 
        <strong>p model</strong> takes the protein structure partitions into account based on the amino acid positions for each domain on the linear diagram. The 
        <strong>cp model</strong> combines both c and p models, which estimates codon positions in protein structural partitions. All models use HKY substitution model.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-30">With two subsets of random selection from seasonal H3N2, model selection procedure and evolutionary parameter estimations generated very similar results. It indicated that the new model validated through different subsets and was not sensitive to the different distributions of epidemiological factors, including geographic location and isolation time (shown in Supplemental Table 3).</p>
     <p id="p-31">With dividing the H1N1pdm09 dataset into the Early stage and Later stage of an epidemic outbreak, model selection procedure reported that the CP model was significantly better than all other models in both datasets. The overall and domain-specific substitution rates in the Early stage was slightly higher compared to the Later stage of the outbreak (shown in Supplemental Table 4). And the Early stage had a higher 
      <em>d</em>C
      <sub>1+2</sub>/
      <em>d</em>C
      <sub>3</sub>, indicating it experienced weaker purifying selection pressure to be relatively diverse. This may be related to host adaptation in the Early stage, where it generated more variants for the advantage of widely spreading in the host population.
     </p>
    </div>
   </div>
   <div class="section" id="sec-8">
    <h2 class="">Discussion</h2>
    <p id="p-32">In this study a novel phylogenetic model with integrating both codon positions and protein structure was developed and tested with HA protein of influenza viruses. Results showed that the new model was statistically supported to fit the data better and provided novel biological information that can reveal new insights into viral evolution. The substitution rates of stalk and head domains of HA protein were significantly different, with a higher rate in head domains for all the tested influenza types. Approximate selection pressure constructed from codon position partitions showed that stalk domain experienced higher purifying selection pressure to maintain its conservation. Model validation from subsets of multiple influenza subtype reported the model performed stably and was not sensitive to sample size, data distribution and epidemic stage. Taken together, this new model could provide potential biological explanations related to antigenic drift, vaccine escape and pandemic emergence to understand viral evolution and risk assessment.</p>
    <p id="p-33">Partitioning scheme on genetic data allows for capturing similar evolutionary features inside one partition of data and comparing different partitions to generate new insights on viral evolution. For example, SRD06 codon model categories the first and second codon positions together since they behave similarly to mainly generate non-synonymous changes, while the third codon position is treated as a separate partition due to mostly generating synonymous changes (
     <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">3</a>, 
     <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">10</a>). Some models conduct partitioning scheme on multiple proteins to reconstruct the co-evolution and estimate protein-specific rates under one model (
     <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>). These models function well for some purposes of observing the co-evolution of multiple proteins, however, it assumes evolutionary processes act uniformly across one protein, which may not be the biological reality due to distinctively functional domains on one protein.
    </p>
    <p id="p-34">When developing the new model in this study, the primary tasks are to efficiently capture the information regarding HA protein structural and functional domains and to properly incorporate this information into the evolutionary models. Therefore, how to partition on the protein structure with genetic data is critical. One solution is to compare all possible partitioning schemes for a given genetic sequence dataset to find the best partitioning strategy (
     <a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">3</a>). However, this approach is computationally inflexible because the combination of possible partitioning schemes is tremendous even for a very small number of data blocks (
     <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>). Consequently, many previous studies ended up either arbitrarily choosing a single partitioning scheme or just selecting the best scheme based on statistical model selection (
     <a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">3</a>,
     <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>,
     <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>). With these limitations on proper partitioning, the accuracy of the inferences from partitioned phylogenetic analysis remains uncertain, even though advanced developments in phylogenetic analysis have been achieved in recent decades (
     <a id="xref-ref-3-6" class="xref-bibr" href="#ref-3">3</a>). Due to advanced progress in viral protein structure (
     <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>), partitioning scheme in this study is based on biological functions of different protein domains, where HA protein has been defined to distinguish the immunogenic roles and other molecular-level functions during viral infection. Molecular mechanism of influenza viruses, for example, the initiation of viral fusion or antigenic-immune stimulation, is determined by the head domain of the protein. Nucleic acid changes observed in this domain usually result from a balance between immune selection and conserved functionality (
     <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>–
     <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>). The new partitioning model considers the distinctive functions of head domain to mainly bind to host-cell receptors and the stalk domain to perform membrane fusion.
    </p>
    <p id="p-35">Another advance in this study is that a formal model selection procedure with reliable approaches (
     <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>) to quantitatively evaluate the model performance and accuracy of inferences for pandemic, all seasonal and two emerging influenza viruses. Previously the harmonic mean estimate (HME) and Akaike’s information criterion through MCMC (AICM) (
     <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>) have been commonly used. In recent years, several new approaches to perform model selection in the field of phylogenetics, such as path-sampling (PS) (
     <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>) and stepping-stone sampling (SS) (
     <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>). Baele et al. (
     <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>) have tested HME, AICM, PS, and SS approaches for demographic and molecular clock model comparison, which “confirmed that HME systematically overestimates the marginal likelihood and fails to yield reliable model classification, and PS and SS substantially outperform HME estimators” (
     <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>). Therefore, in this study both PS and SS approaches have been used to perform the model selection procedure to generate reliable results of model comparison.
    </p>
    <p id="p-36">The new model with both partitioning on codon positions and protein structure incorporates biological functional domains inside one protein to allow for a quantitative exportation of the evolutionary process as a function of approximate host immune selection. The model did capture the different evolutionary process of these functional domains, which showed that the stalk domain had a significantly lower substitution rates and lower 
     <em>d</em>C
     <sub>1+2</sub>/
     <em>d</em>C
     <sub>3</sub>, indicating that the stalk domain experienced stronger purifying selection and maintained its conserved functionality. The estimated domain-specific 
     <em>d</em>C
     <sub>1+2</sub>/
     <em>d</em>C
     <sub>3</sub> can reveal insights of selection pressure on different viral isolates in different host populations. For example, diversifying selection pressure on the head domain has been observed only in chicken population in Egypt. One possible explanation could be that the failed efficacy of poultry influenza H5 vaccines implemented in Egypt (
     <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>, 
     <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>) may drive the genetic diversity of the viruses in the chicken population.
    </p>
    <p id="p-37">Even though 
     <em>d</em>C
     <sub>1+2</sub>/
     <em>d</em>C
     <sub>3</sub> estimated by the model is not an exact measurement of the ω parameter, it is a good predictor of the ratio of non-synonymous changes over synonymous changes (
     <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">10</a>). The new model can provide an informative approximation and avoid heavy computational demands of a full codon or amino acid models (
     <a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">10</a>). When assuming synonymous changes do not affect viral fitness, the 
     <em>d</em>C
     <sub>1+2</sub>/
     <em>d</em>C
     <sub>3</sub> value can provide corresponding estimation for selection pressure compared to mutation-selection models which assess the strength of natural selection on specific mutations (
     <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>). Furthermore, the partitioning scheme and additional parameter estimation did not significantly increase the analysis run time compared to other models in this study under the same computing environment.
    </p>
    <p id="p-38">The structurally informed evolutionary model may provide insight for universal vaccine design. The higher rates observed in the head domain are probably due to strong immune selection targeted on the globular head, where some sites are mostly targeted by the neutralizing immunity induced by seasonal influenza vaccine (
     <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">13</a>). Supported by many laboratory studies, stalk domain could be used to induce more broad-reactive vaccines (
     <a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">13</a>,
     <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>,
     <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>). This model can more accurately reconstruct the ancestral sequence of the stalk domain with estimating domain-specific rates and substitution patterns under a shared viral phylogeny. Therefore, if this model was used to reconstruct common ancestral sequence for all viral types in influenza A virus Group 1 (H1, H2, H5, H6, H8, H9, H11, H12, H13, H16, and H17) and Group 2 (H3, H4, H7, H10, H14 and H15) (
     <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>), respectively, it is probably amongst the most promising approaches to design a universal influenza vaccine.
    </p>
    <p id="p-39">While this model has been developed with influenza in mind, there is potential to apply this approach to other viruses. Proteins, such as the respiratory syncytial virus F and G protein or the coronavirus spike protein that have multiple functional domains including immune stimulators, may be well suited to the novel approach that incorporates protein structural and functional domains into the nucleotide substitution models. Incorporating this type of information into comparative genomic analyses has the potential to provide important biological information and improve both vaccine design and evaluation.</p>
   </div>
   <div class="section" id="sec-9">
    <h2 class="">Methods</h2>
    <div id="sec-10" class="subsection">
     <h3>Datasets</h3>
     <p id="p-40">Multiple datasets representing pandemic, seasonal and emerging influenza viruses were used to develop and broadly test the proposed structurally informed partitioning models, including pandemic H1N1pdm09, seasonal influenza viruses A/H1N1postpdm, A/H3N2, B/Yamagata-like and B/Victoria-like, and two HPAI H5Nx and H7N9. Except for HPAI H7N9, H1N1pdm09 and seasonal A/H1N1postpdm that have been directly downloaded from the Global Initiative on Sharing All Influenza Data (GISAID, 
      <a href="https://www.gisaid.org/">https://www.gisaid.org/</a>) (
      <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>, 
      <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>), all other datasets were from published research, including seasonal A/H3N2 (
      <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>), HPAI H5Nx (
      <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>), B/Yamagata-like and B/Victoria-like (
      <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>). To avoid improper weights of duplicate isolates on the partitioning models, sequences with 100% similarity were removed for all datasets (final datasets can be found via the GitHub link: 
      <a href="https://github.com/XuetingQiu/FluPartitioningModels">https://github.com/XuetingQiu/FluPartitioningModels</a>).
     </p>
     <p id="p-41">Briefly, the final datasets contained 505 global isolates for H1N1pdm09 during 2009/04 – 2009/12, 635 global isolates for HPAI H7N9 during 2013-2018, and 554 global isolates for seasonal A/H1N1postpdm during 2012-2018. HPAI H5Nx contained 1,095 isolates during 1996-2016, after removing the vaccine candidates (
      <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">39</a>). The two influenza B virus lineages were full genome datasets sampled during 2002-2013 in eastern Australia and New Zealand (
      <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">40</a>). For B-Victoria, the dataset finally retained 214 isolates after randomly sampling down to 50% of the original dataset and removing the duplicates. For B-Yamagata, 241 isolates were included after removing the duplicates. The global dataset of seasonal A/H3N2 (
      <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">38</a>) contained 906 isolates during 2000-2012 after the duplicates were removed and further randomly subsampled.
     </p>
    </div>
    <div id="sec-11" class="subsection">
     <h3>Definition of HA protein structural partitions</h3>
     <p id="p-42">HA glycoprotein has been defined as different functional domains, including signal peptide, stalk domain, globular head domain, transmembrane domain and cytoplasmic domain (
      <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">11</a>, 
      <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>). These protein domains labeled with sites of amino acid were translated into the linear sites on the nucleotide sequences of HA gene (
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2.1a</a> and 
      <a id="xref-fig-2-2" class="xref-fig" href="#F2">2.1b</a>). To use the full length of HA sequences and avoid some partitions being too short and lack of information, signal peptide, transmembrane domain, and cytoplasmic domain were combined as one partition named STC, given the similarly evolutionary characteristics of these functional domains. Though the stalk domain on the linear nucleotide sequence was separated by the head domain, the two gene regions of stalk domain were merged together for analysis (supplemental table I-1 and xml files in GitHub 
      <a href="https://github.com/XuetingQiu/FluPartitioningModels">https://github.com/XuetingQiu/FluPartitioningModels</a>).
     </p>
    </div>
    <div id="sec-12" class="subsection">
     <h3>Structurally informed phylogenetic models</h3>
     <p id="p-43">As shown in 
      <a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2.1b</a>, four different Bayesian phylogenetic models were conducted using the same molecular clock model and coalescent model for each dataset. Only the partitioning strategies on the nucleotide sequences were different. The base model applied HKY substitution model (
      <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>) with Gamma invariant distribution, where this model accounts for base frequencies and allows for rate variations between transitions and transversions. SRD06 codon model (C model in 
      <a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2.1c</a>) (
      <a id="xref-ref-10-6" class="xref-bibr" href="#ref-10">10</a>) partitioned all codons of the HA gene reading frame into codon positions 1+2 and codon position 3, where these two partitions also used HKY substitution model. Protein structure partitioning model (P model in 
      <a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig. 2.1c</a>) contained STC, stalk and head functional domains with HKY substitution model for each domain. The last model (CP model in 
      <a id="xref-fig-2-6" class="xref-fig" href="#F2">Fig. 2.1c</a>) combined the partitioning strategy in both C and P models, that is, codon positions in each protein functional domain were further partitioned into codon positions 1 + 2 and codon position 3.
     </p>
    </div>
    <div id="sec-13" class="subsection">
     <h3>Tree likelihood construction in the partitioning models</h3>
     <p id="p-44">After partitioning with codon positions or/and protein structural domains, the overall tree likelihood calculations need to be explained. The general Bayesian inferences are based on the posterior probability of a hypothesis, for example, a given tree 𝜏. Then the posterior probability of the tree based on the data can be obtained using Bayes theorem, 
      <span class="disp-formula" id="disp-formula-1">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/228692/embed/graphic-9.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/228692/embed/graphic-9.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-45">In which, 
      <em>Pr(</em>𝜏
      <em>)</em> is the prior probability of the tree hypothesis 𝜏 based on data x; 
      <em>Pr(x|</em>𝜏
      <em>)</em> is the probability of observing the data at a specific site.
     </p>
     <p id="p-46">Then assuming independence of substitution across sites, the general likelihood based on a specific phylogenetic model, i.e. the probability of observing the aligned matrix X of all j sequences is, 
      <span class="disp-formula" id="disp-formula-2">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/228692/embed/graphic-10.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/228692/embed/graphic-10.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-47">In which, c is the number of total sites along the sequences; 𝜏 represents the tree hypothesis, 𝑣 represents the branch lengths, 𝜃 is the substitution model parameters, and 𝜙 represents all other model parameters.</p>
     <p id="p-48">The proposed structurally informed partitioning model based on HA protein functional domains extends the formula [2] of the tree likelihood estimation. Each of the defined partition has its tree likelihood estimated independently, therefore, the overall tree likelihood is the products of each partition-specific likelihood, that is, 
      <span class="disp-formula" id="disp-formula-3">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/228692/embed/graphic-11.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/228692/embed/graphic-11.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-49">In which, the sum of c
      <sub>1</sub>, c
      <sub>2</sub>, …, c
      <sub>n</sub> is equal to c, the total number of nucleotide sites along the sequences; and n is the number of partitions in the model.
     </p>
    </div>
    <div id="sec-14" class="subsection">
     <h3>Phylogenetic model simulations</h3>
     <p id="p-50">All models were simulated in the package of Bayesian Evolutionary Analysis Sampling Trees (BEAST) v1.8.4 (
      <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>). To allow for rate variations across lineages, the uncorrelated lognormal relaxed molecular clock was used with an initial mean of 0.0033 with a uniform prior ranging from 0.0 to 1.0. Based on prior knowledge, a smooth and time-aware Gaussian Markov random field (GMRF) process prior on the population sizes was applied in the Skyride calescent model (
      <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>). Each model was performed at least four independent runs of 100 million Markov Chain Monte Carlo (MCMC) generations. To report the substitution rates and phylogenetic trees, four runs for each model were combined after removal of burn-in to achieve an Effective Sample Size (ESS) of &gt;;200 as diagnosed in Tracer v 1.5. Visualized violin plots for substitution rates were generated via personal R scripts (GitHub 
      <a href="https://github.com/XuetingQiu/FluPartitioningModels">https://github.com/XuetingQiu/FluPartitioningModels</a>
      <span class="underline">)</span>.
     </p>
    </div>
    <div id="sec-15" class="subsection">
     <h3>Model selection procedure and criteria</h3>
     <p id="p-51">The goal of model selection procedure is to compare model superiority and identify a model that is sufficiently complex to capture the biological realism and evolutionary processes that have occurred but to avoid overparameterized models with more parameters than these can be reliably estimated from the available data (
      <a id="xref-ref-3-7" class="xref-bibr" href="#ref-3">3</a>, 
      <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a>). In this study, both path-sampling (PS) (
      <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a>) and stepping-stone sampling (SS) (
      <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a>) were used to compute the marginal likelihood estimation to perform the model selection procedure. PS and SS approaches account for both the number of parameters and the appropriateness of prior distributions for these parameters.
     </p>
     <p id="p-52">The marginal likelihood is the probability of the data (that is, likelihood) given the model type, not assuming any particular model parameters (
      <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">31</a>). The marginal likelihood estimation for both PS and SS with Beta path step distributions is set to run 100 path steps and length of chains as 1 million. Bayes Factors (BF), the ratio of marginal likelihood estimations from two models, are used to evaluate the significance of the model comparison. When converted to a log scale, BF becomes the difference between two log marginal likelihood estimations. The mathematical conversion is below (
      <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>):
     </p>
     <p id="p-53">For a model with parameters Θ, the marginal likelihood for the model M with data X is, 
      <span class="disp-formula" id="disp-formula-4">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/228692/embed/graphic-12.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/228692/embed/graphic-12.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-54">Then the Bayes factor can be calculated for model M1 against model M2, 
      <span class="disp-formula" id="disp-formula-5">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/228692/embed/graphic-13.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/228692/embed/graphic-13.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-55">The log scale of BF is derived as below, 
      <span class="disp-formula" id="disp-formula-6">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/228692/embed/graphic-14.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/228692/embed/graphic-14.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-56">With log scale BF as dB, the criteria for model selection are: dB &lt;;0 means the selection supports M2; 0 ≤ dB &lt;;5 means no evidence for supporting M1; 5 ≤ dB &lt;; 10 means substantial strength of evidence for supporting M1; 10 ≤ dB &lt;;15 means strong evidence of supporting M1; 15 ≤ dB &lt;; 20 means very strong evidence of supporting M1; dB ≥ 20 means decisive evidence of supporting M1.</p>
    </div>
    <div id="sec-16" class="subsection">
     <h3>Model performance, validation and sensitivity analysis</h3>
     <p id="p-57">To compare model performance among four models for each dataset, two basic evolutionary parameters were estimated and reported: the root heights and overall substitution rates. If the parameters for each dataset were similar across these four models, then it indicated that the model can provide accurate and stable estimation no matter how many partitions it has included.</p>
     <p id="p-58">To evaluate whether the structurally informed partitioning model was sensitive to sample size, sensitivity analysis was performed on HPAI H5Nx from 1996-2017, the largest final dataset with 1,095 isolates in this study. With retaining the oldest isolate of H5Nx, the rest was randomly subsampled to keep 80%, 60%, 40% and 20% of the dataset, respectively. Four runs of 100 million MCMC chain length of the four models for the full dataset and these subsampled datasets were conducted with marginal likelihood estimations for model selection. Tree height parameters, domain-specific substitution rates, and model selection BFs were reported to evaluate the data sufficiency for model stability.</p>
     <p id="p-59">To further conduct model validation, two more approaches were applied. Firstly, to evaluate whether the structurally informed partitioning model was sensitive to data distribution, the full seasonal A/H3N2 dataset was randomly and repeated sampled 40% of the full dataset into two subsets. These two subsets had different geographical and temporal distributions of A/H3N2 isolates. Secondly, to assess whether the new model was sensitive to the epidemic stage. With the H1N1pdm09 dataset, the Early stage of the pandemic was defined as April to middle of July, 2009, and the Later stage of the pandemic was July-December, 2009. The same model variations and model selection procedure were conducted to all these subsets.</p>
    </div>
    <div id="sec-17" class="subsection">
     <h3>Domain-specific d
      <em>C
       <sub>1+2</sub>/
      </em>d
      <em>C
       <sub>3</sub>
      </em> recorded to tree branches
     </h3>
     <p id="p-60">The new model can report domain-specific rates and 
      <em>d</em>C
      <sub>1+2</sub>/
      <em>d</em>C
      <sub>3</sub> values. To visualize these specific rates or ratios on the tree branches, the BEAST xml files (GitHub link: 
      <a href="https://github.com/XuetingQiu/FluPartitioningModels">https://github.com/XuetingQiu/FluPartitioningModels</a>) were coded to compute and load the specific branch rates for stalk domain codon positions 1+2, stalk domain codon position 3, head domain codon positions 1+2, and head domain codon position 3. Python script (GitHub link: 
      <a href="https://github.com/XuetingQiu/FluPartitioningModels">https://github.com/XuetingQiu/FluPartitioningModels</a>) was constructed to compute and write the 
      <em>d</em>C
      <sub>1+2</sub>/
      <em>d</em>C
      <sub>3</sub> for stalk and head domain from the median rate of each partition to the final summarized tree branches. Seasonal A/H3N2 and all Egypt H5Nx data were used as examples to show the domain-specific 
      <em>d</em>C
      <sub>1+2</sub>/
      <em>d</em>C
      <sub>3</sub> changes in A/H3N2 with different vaccine isolates introduced and the changes in different hosts (human v.s. avian) in Egypt H5Nx.
     </p>
    </div>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgements</h2>
    <p id="p-61">This study was partially funded by NIAID Centers of Excellence for Influenza Virus Research and Surveillance (CEIRS, HHSN272201400006C). These funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript. We gratefully acknowledge the authors, originating and submitting laboratories of the sequences from GISAID’s EpiFluTM Database, for making tremendous data publicly available and largely benefiting the science community.</p>
   </div>
   <div class="section fn-group" id="fn-group-1">
    <h2>Footnotes</h2>
    <ul>
     <li class="fn-update fn-group-summary-of-updates" id="fn-1">
      <p id="p-3">Updates in this version include model validation on sample size, data distribution and epidemic stages; corrections on the measure of approximate selection pressure; and refining the figures. Further added new references.</p>
     </li>
    </ul>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.1" data-doi="10.1371/journal.ppat.1006885">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Campbell</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Strang</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ferguson</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cori</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jombart</span>
          <span class="cit-name-given-names">T.</span>
         </span>
         <span class="cit-article-title">When are pathogen genome sequences informative of transmission events? Parrish C, editor</span>. 
         <abbr class="cit-jnl-abbrev">PLOS Pathog</abbr> [Internet]. 
         <span class="cit-pub-date">2018</span> Feb 8 [cited 2019 Jun 29];
         <span class="cit-vol">14</span>(
         <span class="cit-issue">2</span>):
         <span class="cit-fpage">e1006885</span>. Available from: 
         <a href="https://dx.plos.org/10.1371/journal.ppat.1006885">https://dx.plos.org/10.1371/journal.ppat.1006885</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLOS%2BPathog%26rft.volume%253D14%26rft.spage%253De1006885%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1006885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1006885&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Biek</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pybus</span>
          <span class="cit-name-given-names">OG</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lloyd-Smith</span>
          <span class="cit-name-given-names">JO</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Didelot</span>
          <span class="cit-name-given-names">X</span>
         </span>. 
         <span class="cit-article-title">Measurably evolving pathogens in the genomic era</span>. 
         <abbr class="cit-jnl-abbrev">Trends Ecol Evol [Internet</abbr>]. 
         <span class="cit-pub-date">2015 Jun</span> [cited 2019 Jan 9];
         <span class="cit-vol">30</span>(
         <span class="cit-issue">6</span>):
         <span class="cit-fpage">306</span>–
         <span class="cit-lpage">13</span>. Available from: 
         <a href="http://www.ncbi.nlm.nih.gov/pubmed/25887947">http://www.ncbi.nlm.nih.gov/pubmed/25887947</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol%2BEvol%2B%255BInternet%26rft.volume%253D30%26rft.spage%253D306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lanfear</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Calcott</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ho</span>
          <span class="cit-name-given-names">SYW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Guindon</span>
          <span class="cit-name-given-names">S</span>
         </span>. 
         <span class="cit-article-title">PartitionFinder: Combined Selection of Partitioning Schemes and Substitution Models for Phylogenetic Analyses</span>. 
         <abbr class="cit-jnl-abbrev">Mol Biol Evol [Internet</abbr>]. 
         <span class="cit-pub-date">2012 Jun 1</span> [cited 2019 Feb 20];
         <span class="cit-vol">29</span>(
         <span class="cit-issue">6</span>):
         <span class="cit-fpage">1695</span>–
         <span class="cit-lpage">701</span>. Available from: 
         <a href="https://academic.oup.com/mbe/article-lookup/doi/10.1093/molbev/mss020">https://academic.oup.com/mbe/article-lookup/doi/10.1093/molbev/mss020</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%2B%255BInternet%26rft.volume%253D29%26rft.spage%253D1695%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hasegawa</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kishino</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yano</span>
          <span class="cit-name-given-names">T</span>
         </span>. 
         <span class="cit-article-title">Dating of the human-ape splitting by a molecular clock of mitochondrial DNA</span>. 
         <abbr class="cit-jnl-abbrev">J Mol Evol [Internet</abbr>]. 
         <span class="cit-pub-date">1985</span> [cited 2019 Feb 20];
         <span class="cit-vol">22</span>(
         <span class="cit-issue">2</span>):
         <span class="cit-fpage">160</span>–
         <span class="cit-lpage">74</span>. Available from: 
         <a href="http://www.ncbi.nlm.nih.gov/pubmed/3934395">http://www.ncbi.nlm.nih.gov/pubmed/3934395</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BMol%2BEvol%2B%255BInternet%26rft.volume%253D22%26rft.spage%253D160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lanave</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Preparata</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sacone</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Serio</span>
          <span class="cit-name-given-names">G</span>
         </span>. 
         <span class="cit-article-title">A new method for calculating evolutionary substitution rates</span>. 
         <abbr class="cit-jnl-abbrev">J Mol Evol [Internet</abbr>]. 
         <span class="cit-pub-date">1984</span>;
         <span class="cit-vol">20</span>(
         <span class="cit-issue">1</span>):
         <span class="cit-fpage">86</span>–
         <span class="cit-lpage">93</span>. Available from: 
         <a href="https://doi.org/10.1007/BF02101990">https://doi.org/10.1007/BF02101990</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BMol%2BEvol%2B%255BInternet%26rft.volume%253D20%26rft.spage%253D86%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Tavaré</span>
          <span class="cit-name-given-names">S</span>
         </span>. 
         <span class="cit-article-title">Some Probabilistic and Statistical Problems in the Analysis of DNA Sequences</span>. 
         <abbr class="cit-jnl-abbrev">Am Math Soc Lect Math Life Sci [Internet</abbr>]. 
         <span class="cit-pub-date">1986</span>;
         <span class="cit-vol">17</span>:
         <span class="cit-fpage">57</span>–
         <span class="cit-lpage">86</span>. Available from: citeulike-article-id:6732633
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAm%2BMath%2BSoc%2BLect%2BMath%2BLife%2BSci%2B%255BInternet%26rft.volume%253D17%26rft.spage%253D57%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Webster</span>
          <span class="cit-name-given-names">RG</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bean</span>
          <span class="cit-name-given-names">WJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gorman</span>
          <span class="cit-name-given-names">OT</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chambers</span>
          <span class="cit-name-given-names">TM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kawaoka</span>
          <span class="cit-name-given-names">Y</span>
         </span>. 
         <span class="cit-article-title">Evolution and ecology of influenza A viruses</span>. 
         <abbr class="cit-jnl-abbrev">Microbiol Rev [Internet</abbr>]. 
         <span class="cit-pub-date">1992 Mar 1</span> [cited 2019 Feb 12];
         <span class="cit-vol">56</span>(
         <span class="cit-issue">1</span>):
         <span class="cit-fpage">152</span>–
         <span class="cit-lpage">79</span>. Available from: 
         <a href="http://www.ncbi.nlm.nih.gov/pubmed/1579108">http://www.ncbi.nlm.nih.gov/pubmed/1579108</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMicrobiol%2BRev%2B%255BInternet%26rft.volume%253D56%26rft.spage%253D152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.8">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Baele</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Suchard</span>
          <span class="cit-name-given-names">MA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rambaut</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lemey</span>
          <span class="cit-name-given-names">P</span>
         </span>. 
         <span class="cit-article-title">Emerging Concepts of Data Integration in Pathogen Phylodynamics</span>. 
         <abbr class="cit-jnl-abbrev">Syst Biol [Internet</abbr>]. 
         <span class="cit-pub-date">2017 Jun 1</span> [cited 2018 Dec 2];
         <span class="cit-vol">66</span>(
         <span class="cit-issue">1</span>):
         <span class="cit-fpage">e47</span>–
         <span class="cit-lpage">65</span>. Available from: 
         <a href="https://academic.oup.com/sysbio/article/doi/10.1093/sysbio/syw054/2670001/Emerging-concepts-of-data-integration-in-pathogen">https://academic.oup.com/sysbio/article/doi/10.1093/sysbio/syw054/2670001/Emerging-concepts-of-data-integration-in-pathogen</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSyst%2BBiol%2B%255BInternet%26rft.volume%253D66%26rft.spage%253D47e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-book" id="cit-228692v4.9">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Koonin</span>
          <span class="cit-name-given-names">E</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Galperin</span>
          <span class="cit-name-given-names">M</span>
         </span>. 
         <span class="cit-source">Evolutionary Concept in Genetics and Genomics. In: Sequence - Evolution - Function: Computational Approaches in Comparative Genomics</span>. 
         <span class="cit-publ-loc">Boston</span>: 
         <span class="cit-publ-name">Kluwer Academic</span>; 
         <span class="cit-pub-date">2003</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.10">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Shapiro</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rambaut</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Drummond</span>
          <span class="cit-name-given-names">AJ</span>
         </span>. 
         <span class="cit-article-title">Choosing Appropriate Substitution Models for the Phylogenetic Analysis of Protein-Coding Sequences</span>. 
         <abbr class="cit-jnl-abbrev">Mol Biol Evol [Internet</abbr>]. 
         <span class="cit-pub-date">2006 Jan 1</span>;
         <span class="cit-vol">23</span>(
         <span class="cit-issue">1</span>):
         <span class="cit-fpage">7</span>–
         <span class="cit-lpage">9</span>. Available from: 
         <a href="http://dx.doi.org/10.1093/molbev/msj021">http://dx.doi.org/10.1093/molbev/msj021</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%2B%255BInternet%26rft.volume%253D23%26rft.spage%253D7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.11">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Riwilaijaroen</span>
          <span class="cit-name-given-names">BNS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Uzuki</span>
          <span class="cit-name-given-names">YS</span>
         </span>. 
         <abbr class="cit-jnl-abbrev">Review Molecular basis of the structure and function of H1 hemagglutinin of in fl uenza virus</abbr>. 
         <span class="cit-pub-date">2012</span>;
         <span class="cit-vol">88</span>:
         <span class="cit-fpage">226</span>–
         <span class="cit-lpage">49</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DReview%2BMolecular%2Bbasis%2Bof%2Bthe%2Bstructure%2Band%2Bfunction%2Bof%2BH1%2Bhemagglutinin%2Bof%2Bin%2Bfl%2Buenza%2Bvirus%26rft.volume%253D88%26rft.spage%253D226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.12">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wiley</span>
          <span class="cit-name-given-names">DC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Skehel</span>
          <span class="cit-name-given-names">JJ</span>
         </span>. 
         <span class="cit-article-title">The Structure and Function of the Hemagglutinin Membrane Glycoprotein of Influenza Virus</span>. 
         <abbr class="cit-jnl-abbrev">Annu Rev Biochem [Internet</abbr>]. 
         <span class="cit-pub-date">1987 Jun 1</span> [cited 2019 Feb 13];
         <span class="cit-vol">56</span>(
         <span class="cit-issue">1</span>):
         <span class="cit-fpage">365</span>–
         <span class="cit-lpage">94</span>. Available from: 
         <a href="http://www.ncbi.nlm.nih.gov/pubmed/3304138">http://www.ncbi.nlm.nih.gov/pubmed/3304138</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAnnu%2BRev%2BBiochem%2B%255BInternet%26rft.volume%253D56%26rft.spage%253D365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.13">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Nachbagauer</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Miller</span>
          <span class="cit-name-given-names">MS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hai</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ryder</span>
          <span class="cit-name-given-names">AB</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rose</span>
          <span class="cit-name-given-names">JK</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Palese</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <abbr class="cit-jnl-abbrev">Hemagglutinin Stalk Immunity Reduces Influenza Virus Replication and Transmission in Ferrets</abbr>. 
         <span class="cit-pub-date">2016</span>;
         <span class="cit-vol">90</span>(
         <span class="cit-issue">6</span>):
         <span class="cit-fpage">3268</span>–
         <span class="cit-lpage">73</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DHemagglutinin%2BStalk%2BImmunity%2BReduces%2BInfluenza%2BVirus%2BReplication%2Band%2BTransmission%2Bin%2BFerrets%26rft.volume%253D90%26rft.spage%253D3268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">TT</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Palese</span>
          <span class="cit-name-given-names">P</span>
         </span>. 
         <span class="cit-article-title">Catching a Moving Target</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr> (80-) [Internet]. 
         <span class="cit-pub-date">2011</span> Aug 12;
         <span class="cit-vol">333</span>(
         <span class="cit-issue">6044</span>):
         <span class="cit-fpage">834 LP</span> – 
         <span class="cit-lpage">835</span>. Available from: 
         <a href="http://science.sciencemag.org/content/333/6044/834.abstract">http://science.sciencemag.org/content/333/6044/834.abstract</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft.volume%253D333%26rft.spage%253D834LP%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.15">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Krammer</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hai</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yondola</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tan</span>
          <span class="cit-name-given-names">GS</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Leyva-Grado</span>
          <span class="cit-name-given-names">VH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ryder</span>
          <span class="cit-name-given-names">AB</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Assessment of influenza virus hemagglutinin stalk-based immunity in ferrets</span>. 
         <abbr class="cit-jnl-abbrev">J Virol [Internet</abbr>]. 
         <span class="cit-pub-date">2014 Mar 15</span> [cited 2017 Nov 16];
         <span class="cit-vol">88</span>(
         <span class="cit-issue">6</span>):
         <span class="cit-fpage">3432</span>–
         <span class="cit-lpage">42</span>. Available from: 
         <a href="http://www.ncbi.nlm.nih.gov/pubmed/24403585">http://www.ncbi.nlm.nih.gov/pubmed/24403585</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%2B%255BInternet%26rft.volume%253D88%26rft.spage%253D3432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.16">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Krammer</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Palese</span>
          <span class="cit-name-given-names">P</span>
         </span>. 
         <span class="cit-article-title">Influenza virus hemagglutinin stalk-based antibodies and vaccines</span>. 
         <abbr class="cit-jnl-abbrev">Curr Opin Virol [Internet</abbr>]. 
         <span class="cit-pub-date">2013</span>;
         <span class="cit-vol">3</span>(
         <span class="cit-issue">5</span>):
         <span class="cit-fpage">521</span>–
         <span class="cit-lpage">30</span>. Available from: 
         <a href="http://www.sciencedirect.com/science/article/pii/S187962571300134X">http://www.sciencedirect.com/science/article/pii/S187962571300134X</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCurr%2BOpin%2BVirol%2B%255BInternet%26rft.volume%253D3%26rft.spage%253D521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">17.</span>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.17">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Subbarao</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Matsuoka</span>
          <span class="cit-name-given-names">Y</span>
         </span>. 
         <span class="cit-article-title">The prospects and challenges of universal vaccines for influenza</span>. 
         <abbr class="cit-jnl-abbrev">Trends Microbiol [Internet</abbr>]. 
         <span class="cit-pub-date">2013 Nov 15</span>;
         <span class="cit-vol">21</span>(
         <span class="cit-issue">7</span>):
         <span class="cit-fpage">350</span>–
         <span class="cit-lpage">8</span>. Available from: 
         <a href="http://dx.doi.org/10.1016/j.tim.2013.04.003">http://dx.doi.org/10.1016/j.tim.2013.04.003</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DTrends%2BMicrobiol%2B%255BInternet%26rft.volume%253D21%26rft.spage%253D350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">18.</span>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.18">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Shapiro</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rambaut</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Drummond</span>
          <span class="cit-name-given-names">AJ</span>
         </span>. 
         <span class="cit-article-title">Choosing Appropriate Substitution Models for the Phylogenetic Analysis of Protein-Coding Sequences</span>. 
         <abbr class="cit-jnl-abbrev">Mol Biol Evol [Internet</abbr>]. 
         <span class="cit-pub-date">2006 Jan 1</span> [cited 2019 Feb 20];
         <span class="cit-vol">23</span>(
         <span class="cit-issue">1</span>):
         <span class="cit-fpage">7</span>–
         <span class="cit-lpage">9</span>. Available from: 
         <a href="http://academic.oup.com/mbe/article/23/1/7/1193608/Choosing-Appropriate-Substitution-Models-for-the">http://academic.oup.com/mbe/article/23/1/7/1193608/Choosing-Appropriate-Substitution-Models-for-the</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%2B%255BInternet%26rft.volume%253D23%26rft.spage%253D7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">19.</span>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.19">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Mintaev</span>
          <span class="cit-name-given-names">RR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Alexeevski A</span>
          <span class="cit-name-given-names">V</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kordyukova L</span>
          <span class="cit-name-given-names">V</span>
         </span>. 
         <span class="cit-article-title">Co-evolution analysis to predict protein– protein interactions within influenza virus envelope</span>. 
         <abbr class="cit-jnl-abbrev">J Bioinform Comput Biol [Internet</abbr>]. 
         <span class="cit-pub-date">2014 Mar 6</span>;
         <span class="cit-vol">12</span>(
         <span class="cit-issue">02</span>):
         <span class="cit-fpage">1441008</span>. Available from: 
         <a href="https://doi.org/10.1142/S021972001441008X">https://doi.org/10.1142/S021972001441008X</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BBioinform%2BComput%2BBiol%2B%255BInternet%26rft.volume%253D12%26rft.spage%253D1441008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">20.</span>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.20" data-doi="10.1080/10635150802206883">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lu</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ortí</span>
          <span class="cit-name-given-names">G.</span>
         </span>
         <span class="cit-article-title">Optimal Data Partitioning and a Test Case for Ray-Finned Fishes (Actinopterygii) Based on Ten Nuclear Loci. Buckley T, editor</span>. 
         <abbr class="cit-jnl-abbrev">Syst Biol</abbr> [Internet]. 
         <span class="cit-pub-date">2008</span> Aug 1 [cited 2019 Apr 18];
         <span class="cit-vol">57</span>(
         <span class="cit-issue">4</span>):
         <span class="cit-fpage">519</span>–
         <span class="cit-lpage">39</span>. Available from: 
         <a href="http://www.ncbi.nlm.nih.gov/pubmed/18622808">http://www.ncbi.nlm.nih.gov/pubmed/18622808</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSyst%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150802206883%26rft_id%253Dinfo%253Apmid%252F18622808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1080/10635150802206883&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18622808&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F228692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000257544000001&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">21.</span>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.21">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Brandley</span>
          <span class="cit-name-given-names">MC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schmitz</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Reeder</span>
          <span class="cit-name-given-names">TW.</span>
         </span>
         <span class="cit-article-title">Partitioned Bayesian Analyses, Partition Choice, and the Phylogenetic Relationships of Scincid Lizards</span>. 
         <abbr class="cit-jnl-abbrev">Anderson F, editor. Syst Biol</abbr> [Internet]. 
         <span class="cit-pub-date">2005</span> Jun 1 [cited 2019 Apr 18];
         <span class="cit-vol">54</span>(
         <span class="cit-issue">3</span>):
         <span class="cit-fpage">373</span>–
         <span class="cit-lpage">90</span>. Available from: 
         <a href="http://www.ncbi.nlm.nih.gov/pubmed/16012105">http://www.ncbi.nlm.nih.gov/pubmed/16012105</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAnderson%2BF%252C%2Beditor.%2BSyst%2BBiol%26rft.volume%253D54%26rft.spage%253D373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">22.</span>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.22" data-doi="10.1080/10635150701656360">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">McGuire</span>
          <span class="cit-name-given-names">JA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Witt</span>
          <span class="cit-name-given-names">CC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Altshuler</span>
          <span class="cit-name-given-names">DL</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Remsen</span>
          <span class="cit-name-given-names">JV.</span>
         </span>
         <span class="cit-article-title">Phylogenetic Systematics and Biogeography of Hummingbirds: Bayesian and Maximum Likelihood Analyses of Partitioned Data and Selection of an Appropriate Partitioning Strategy. Zamudio K, Sullivan J, editors</span>. 
         <abbr class="cit-jnl-abbrev">Syst Biol</abbr> [Internet]. 
         <span class="cit-pub-date">2007</span> Oct 1 [cited 2019 Apr 18];
         <span class="cit-vol">56</span>(
         <span class="cit-issue">5</span>):
         <span class="cit-fpage">837</span>–
         <span class="cit-lpage">56</span>. Available from: 
         <a href="http://www.ncbi.nlm.nih.gov/pubmed/17934998">http://www.ncbi.nlm.nih.gov/pubmed/17934998</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSyst%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150701656360%26rft_id%253Dinfo%253Apmid%252F17934998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1080/10635150701656360&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17934998&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F228692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000250959000010&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">23.</span>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.23">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Swanson</span>
          <span class="cit-name-given-names">KA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Settembre</span>
          <span class="cit-name-given-names">EC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shaw</span>
          <span class="cit-name-given-names">CA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dey</span>
          <span class="cit-name-given-names">AK</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rappuoli</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mandl</span>
          <span class="cit-name-given-names">CW</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Structural basis for immunization with postfusion respiratory syncytial virus fusion F glycoprotein (RSV F) to elicit high neutralizing antibody titers</span>. 
         <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A [Internet</abbr>]. 
         <span class="cit-pub-date">2011 Jun 7</span> [cited 2019 Mar 12];108(23):9619–24. Available from: 
         <a href="http://www.ncbi.nlm.nih.gov/pubmed/21586636">http://www.ncbi.nlm.nih.gov/pubmed/21586636</a>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">24.</span>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.24">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kordyukova</span>
          <span class="cit-name-given-names">L</span>
         </span>. 
         <span class="cit-article-title">Structural and functional specificity of Influenza virus haemagglutinin and paramyxovirus fusion protein anchoring peptides</span>. 
         <abbr class="cit-jnl-abbrev">Virus Res [Internet</abbr>]. 
         <span class="cit-pub-date">2017</span>;
         <span class="cit-vol">227</span>(Supplement 
         <span class="cit-issue">C</span>):
         <span class="cit-fpage">183</span>–
         <span class="cit-lpage">99</span>. Available from: 
         <a href="http://www.sciencedirect.com/science/article/pii/S0168170216304063">http://www.sciencedirect.com/science/article/pii/S0168170216304063</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVirus%2BRes%2B%255BInternet%26rft.volume%253D227%26rft.spage%253D183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">25.</span>
      <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.25">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lu</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shi</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gao</span>
          <span class="cit-name-given-names">F</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xiao</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Qi</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Insights into Avian Influenza Virus Pathogenicity: the Hemagglutinin Precursor HA0 of Subtype H16 Has an Alpha-Helix Structure in Its Cleavage Site with Inefficient HA1/HA2 Cleavage</span>. 
         <abbr class="cit-jnl-abbrev">J Virol [Internet</abbr>]. 
         <span class="cit-pub-date">2012 Dec 25</span>;
         <span class="cit-vol">86</span>(
         <span class="cit-issue">23</span>):
         <span class="cit-fpage">12861</span>–
         <span class="cit-lpage">70</span>. Available from: 
         <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3497694/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3497694/</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%2B%255BInternet%26rft.volume%253D86%26rft.spage%253D12861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">26.</span>
      <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.26">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Skehel</span>
          <span class="cit-name-given-names">JJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wiley</span>
          <span class="cit-name-given-names">DC</span>
         </span>. 
         <span class="cit-article-title">Receptor Binding and Membrane Fusion in Virus Entry: The Influenza Hemagglutinin</span>. 
         <abbr class="cit-jnl-abbrev">Annu Rev Biochem [Internet</abbr>]. 
         <span class="cit-pub-date">2000 Jun 1</span>;
         <span class="cit-vol">69</span>(
         <span class="cit-issue">1</span>):
         <span class="cit-fpage">531</span>–
         <span class="cit-lpage">69</span>. Available from: 
         <a href="https://doi.org/10.1146/annurev.biochem.69.1.531">https://doi.org/10.1146/annurev.biochem.69.1.531</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAnnu%2BRev%2BBiochem%2B%255BInternet%26rft.volume%253D69%26rft.spage%253D531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">27.</span>
      <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.27">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Baele</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lemey</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bedford</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rambaut</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Suchard</span>
          <span class="cit-name-given-names">MA</span>
         </span>, 
         <span class="cit-article-title">Alekseyenko A V</span>. 
         <abbr class="cit-jnl-abbrev">Improving the Accuracy of Demographic and Molecular Clock Model Comparison While Accommodating Phylogenetic Uncertainty Research article</abbr>. 
         <span class="cit-pub-date">2012</span>;
         <span class="cit-vol">29</span>(
         <span class="cit-issue">9</span>):
         <span class="cit-fpage">2157</span>–
         <span class="cit-lpage">67</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DImproving%2Bthe%2BAccuracy%2Bof%2BDemographic%2Band%2BMolecular%2BClock%2BModel%2BComparison%2BWhile%2BAccommodating%2BPhylogenetic%2BUncertainty%2BResearch%2Barticle%26rft.volume%253D29%26rft.spage%253D2157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">28.</span>
      <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.28">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Raftery</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Newton</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Satagopan</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Krivitsky</span>
          <span class="cit-name-given-names">P.</span>
         </span>
         <span class="cit-article-title">Estimating the Integrated Likelihood via Posterior Simulation Using the Harmonic Mean Identity</span>. 
         <abbr class="cit-jnl-abbrev">Meml Sloan-Kettering Cancer Center, Dept Epidemiol Biostat Work Pap Ser</abbr> [Internet]. 
         <span class="cit-pub-date">2006</span> Apr 14 [cited 2017 Nov 16]; Available from: 
         <a href="http://biostats.bepress.com/mskccbiostat/paper6">http://biostats.bepress.com/mskccbiostat/paper6</a>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">29.</span>
      <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.29">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lartillot</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Philippe</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lewis</span>
          <span class="cit-name-given-names">P</span>
         </span>. 
         <span class="cit-article-title">Computing Bayes Factors Using Thermodynamic Integration</span>. 
         <abbr class="cit-jnl-abbrev">Syst Biol [Internet</abbr>]. 
         <span class="cit-pub-date">2006 Apr 1</span>;
         <span class="cit-vol">55</span>(
         <span class="cit-issue">2</span>):
         <span class="cit-fpage">195</span>–
         <span class="cit-lpage">207</span>. Available from: 
         <a href="http://dx.doi.org/10.1080/10635150500433722">http://dx.doi.org/10.1080/10635150500433722</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSyst%2BBiol%2B%255BInternet%26rft.volume%253D55%26rft.spage%253D195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">30.</span>
      <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.30">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Xie</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lewis</span>
          <span class="cit-name-given-names">PO</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fan</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kuo</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">M-H</span>
         </span>. 
         <span class="cit-article-title">Improving Marginal Likelihood Estimation for Bayesian Phylogenetic Model Selection</span>. 
         <abbr class="cit-jnl-abbrev">Syst Biol [Internet</abbr>]. 
         <span class="cit-pub-date">2011 Mar 1</span> [cited 2019 May 5];
         <span class="cit-vol">60</span>(
         <span class="cit-issue">2</span>):
         <span class="cit-fpage">150</span>–
         <span class="cit-lpage">60</span>. Available from: 
         <a href="http://www.ncbi.nlm.nih.gov/pubmed/21187451">http://www.ncbi.nlm.nih.gov/pubmed/21187451</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSyst%2BBiol%2B%255BInternet%26rft.volume%253D60%26rft.spage%253D150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">31.</span>
      <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.31">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Baele</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lemey</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bedford</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rambaut</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Suchard</span>
          <span class="cit-name-given-names">MA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Alekseyenko A</span>
          <span class="cit-name-given-names">V</span>
         </span>. 
         <span class="cit-article-title">Improving the accuracy of demographic and molecular clock model comparison while accomodating phylogenetic uncertainty</span>. 
         <abbr class="cit-jnl-abbrev">Mol Biol Evol [Internet</abbr>]. 
         <span class="cit-pub-date">2012 Jan</span>; Available from: 
         <a href="http://www.ncbi.nlm.nih.gov/pubmed/17065594">http://www.ncbi.nlm.nih.gov/pubmed/17065594</a>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">32.</span>
      <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.32">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Baele</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lok</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Drummond</span>
          <span class="cit-name-given-names">AJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Suchard</span>
          <span class="cit-name-given-names">MA</span>
         </span>, 
         <span class="cit-article-title">Lemey P</span>. 
         <abbr class="cit-jnl-abbrev">Accurate Model Selection of Relaxed Molecular Clocks in Bayesian Phylogenetics Letter Fast Track</abbr>. 
         <span class="cit-pub-date">2012</span>;
         <span class="cit-vol">30</span>(
         <span class="cit-issue">2</span>):
         <span class="cit-fpage">239</span>–
         <span class="cit-lpage">43</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAccurate%2BModel%2BSelection%2Bof%2BRelaxed%2BMolecular%2BClocks%2Bin%2BBayesian%2BPhylogenetics%2BLetter%2BFast%2BTrack%26rft.volume%253D30%26rft.spage%253D239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">33.</span>
      <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.33">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kandeil</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sabir</span>
          <span class="cit-name-given-names">JSM</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Abdelaal</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mattar</span>
          <span class="cit-name-given-names">EH</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">El-Taweel</span>
          <span class="cit-name-given-names">AN</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sabir</span>
          <span class="cit-name-given-names">MJ</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Efficacy of commercial vaccines against newly emerging avian influenza H5N8 virus in Egypt</span>. 
         <abbr class="cit-jnl-abbrev">Sci Rep</abbr>. 
         <span class="cit-pub-date">2018 Dec 1</span>;
         <span class="cit-vol">8</span>(
         <span class="cit-issue">1</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">34.</span>
      <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.34">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hamid Samaha</span>
          <span class="cit-name-given-names">MP.</span>
         </span>
         <span class="cit-article-title">Avian influenza vaccination in Egypt: Limitations of the current strategy</span>. 
         <abbr class="cit-jnl-abbrev">J Mol Genet Med</abbr>. 
         <span class="cit-pub-date">2009</span>;
         <span class="cit-vol">03</span>(
         <span class="cit-issue">02</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">35.</span>
      <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.35">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Spielman</span>
          <span class="cit-name-given-names">SJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wilke</span>
          <span class="cit-name-given-names">CO</span>
         </span>. 
         <span class="cit-article-title">The relationship between dN/dS and scaled selection coefficients</span>. 
         <abbr class="cit-jnl-abbrev">Mol Biol Evol [Internet</abbr>]. 
         <span class="cit-pub-date">2015 Apr</span> [cited 2019 Jul 19];
         <span class="cit-vol">32</span>(
         <span class="cit-issue">4</span>):
         <span class="cit-fpage">1097</span>–
         <span class="cit-lpage">108</span>. Available from: 
         <a href="http://www.ncbi.nlm.nih.gov/pubmed/25576365">http://www.ncbi.nlm.nih.gov/pubmed/25576365</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%2B%255BInternet%26rft.volume%253D32%26rft.spage%253D1097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">36.</span>
      <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.36">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Elbe</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Buckland-Merrett</span>
          <span class="cit-name-given-names">G</span>
         </span>. 
         <span class="cit-article-title">Data, disease and diplomacy: GISAID’s innovative contribution to global health</span>. 
         <abbr class="cit-jnl-abbrev">Glob Challenges [Internet</abbr>]. 
         <span class="cit-pub-date">2017 Jan 1</span> [cited 2019 Apr 29];
         <span class="cit-vol">1</span>(
         <span class="cit-issue">1</span>):
         <span class="cit-fpage">33</span>–
         <span class="cit-lpage">46</span>. Available from: 
         <a href="http://doi.wiley.com/10.1002/gch2.1018">http://doi.wiley.com/10.1002/gch2.1018</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGlob%2BChallenges%2B%255BInternet%26rft.volume%253D1%26rft.spage%253D33%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">37.</span>
      <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.37">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Shu</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">McCauley</span>
          <span class="cit-name-given-names">J</span>
         </span>. 
         <span class="cit-article-title">GISAID: Global initiative on sharing all influenza data – from vision to reality</span>. 
         <abbr class="cit-jnl-abbrev">Eurosurveillance [Internet</abbr>]. 
         <span class="cit-pub-date">2017 Mar 30</span> [cited 2019 Apr 28];
         <span class="cit-vol">22</span>(
         <span class="cit-issue">13</span>):
         <span class="cit-fpage">30494</span>. Available from: 
         <a href="http://www.ncbi.nlm.nih.gov/pubmed/28382917">http://www.ncbi.nlm.nih.gov/pubmed/28382917</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEurosurveillance%2B%255BInternet%26rft.volume%253D22%26rft.spage%253D30494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">38.</span>
      <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.38" data-doi="10.1038/nature14460">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bedford</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Riley</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Barr</span>
          <span class="cit-name-given-names">IG</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Broor</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chadha</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cox</span>
          <span class="cit-name-given-names">NJ</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Global circulation patterns of seasonal influenza viruses vary with antigenic drift</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr> [Internet]. 
         <span class="cit-pub-date">2015</span> Jun 8 [cited 2019 Feb 7];
         <span class="cit-vol">523</span>(
         <span class="cit-issue">7559</span>):
         <span class="cit-fpage">217</span>. Available from: 
         <a href="http://www.nature.com/articles/nature14460">http://www.nature.com/articles/nature14460</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D523%26rft.spage%253D217%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14460%26rft_id%253Dinfo%253Apmid%252F26053121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature14460&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26053121&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F228692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">39.</span>
      <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.39">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Qiu</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Duvvuri</span>
          <span class="cit-name-given-names">VR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gubbay</span>
          <span class="cit-name-given-names">JB</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Webby</span>
          <span class="cit-name-given-names">RJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kayali</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bahl</span>
          <span class="cit-name-given-names">J.</span>
         </span>
         <abbr class="cit-jnl-abbrev">specific epitope profiles for HPAI H5 pre-• pandemic vaccine selection and evaluation</abbr>. 
         <span class="cit-pub-date">2017</span>;(July):
         <span class="cit-fpage">445</span>–
         <span class="cit-lpage">56</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">40.</span>
      <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.40" data-doi="10.7554/eLife.06633">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Vijaykrishna</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Holmes</span>
          <span class="cit-name-given-names">EC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Joseph</span>
          <span class="cit-name-given-names">U</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fourment</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Su</span>
          <span class="cit-name-given-names">YCF</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Halpin</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">The contrasting phylodynamics of human influenza B viruses</span>. 
         <abbr class="cit-jnl-abbrev">Elife</abbr>. 
         <span class="cit-pub-date">2015</span>;
         <span class="cit-vol">4</span>:
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">23</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DElife%26rft.volume%253D4%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.06633%26rft_id%253Dinfo%253Apmid%252F25748136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.7554/eLife.06633&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25748136&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F228692.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">41.</span>
      <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.41">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Drummond</span>
          <span class="cit-name-given-names">AJ</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Suchard</span>
          <span class="cit-name-given-names">MA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xie</span>
          <span class="cit-name-given-names">D</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rambaut</span>
          <span class="cit-name-given-names">A</span>
         </span>. 
         <span class="cit-article-title">Bayesian Phylogenetics with BEAUti and the BEAST 1.7</span>. 
         <abbr class="cit-jnl-abbrev">Mol Biol Evol [Internet</abbr>]. 
         <span class="cit-pub-date">2012 Aug</span> [cited 2019 Feb 5];
         <span class="cit-vol">29</span>(
         <span class="cit-issue">8</span>):
         <span class="cit-fpage">1969</span>–
         <span class="cit-lpage">73</span>. Available from: 
         <a href="http://www.ncbi.nlm.nih.gov/pubmed/22367748">http://www.ncbi.nlm.nih.gov/pubmed/22367748</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%2B%255BInternet%26rft.volume%253D29%26rft.spage%253D1969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">42.</span>
      <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.42">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Minin</span>
          <span class="cit-name-given-names">VN</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bloomquist</span>
          <span class="cit-name-given-names">EW</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Suchard</span>
          <span class="cit-name-given-names">MA</span>
         </span>. 
         <span class="cit-article-title">Smooth Skyride through a Rough Skyline: Bayesian Coalescent-Based Inference of Population Dynamics</span>. 
         <abbr class="cit-jnl-abbrev">Mol Biol Evol [Internet</abbr>]. 
         <span class="cit-pub-date">2008 Apr 3</span> [cited 2019 Feb 20];
         <span class="cit-vol">25</span>(
         <span class="cit-issue">7</span>):
         <span class="cit-fpage">1459</span>–
         <span class="cit-lpage">71</span>. Available from: 
         <a href="https://academic.oup.com/mbe/article-lookup/doi/10.1093/molbev/msn090">https://academic.oup.com/mbe/article-lookup/doi/10.1093/molbev/msn090</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%2B%255BInternet%26rft.volume%253D25%26rft.spage%253D1459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">43.</span>
      <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-228692v4.43">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kass</span>
          <span class="cit-name-given-names">RE</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Raftery</span>
          <span class="cit-name-given-names">AE</span>
         </span>. 
         <span class="cit-article-title">Bayes Factors</span>. 
         <abbr class="cit-jnl-abbrev">J Am Stat Assoc [Internet</abbr>]. 
         <span class="cit-pub-date">1995 Jun 1</span>;
         <span class="cit-vol">90</span>(
         <span class="cit-issue">430</span>):
         <span class="cit-fpage">773</span>–
         <span class="cit-lpage">95</span>. Available from: 
         <a href="http://www.tandfonline.com/doi/abs/10.1080/01621459.1995.10476572">http://www.tandfonline.com/doi/abs/10.1080/01621459.1995.10476572</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BAm%2BStat%2BAssoc%2B%255BInternet%26rft.volume%253D90%26rft.spage%253D773%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
