<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">EClinicalMedicine</span><span class="iso-abbrev" title="iso-abbrev">EClinicalMedicine</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">EClinicalMedicine</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2589-5370</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Elsevier</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6610774">6610774</a></span><span class="publisher-id" title="publisher-id">S2589-5370(19)30096-3</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1016/j.eclinm.2019.06.003">10.1016/j.eclinm.2019.06.003</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Paper</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Machine-learning Prognostic Models from the 2014–16 Ebola Outbreak: Data-harmonization Challenges, Validation Strategies, and mHealth Applications</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colubri</span><span class="given-names" tagx="given-names" title="given-names">Andres</span></span><span class="email" tagx="email" title="email">acolubri@fas.harvard.edu</span><a href="#af0005">a</a><a href="#af0010">b</a><a href="#af0015">c</a><a href="#cr0005">�?�</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hartley</span><span class="given-names" tagx="given-names" title="given-names">Mary-Anne</span></span><a href="#af0020">d</a><a href="#af0025">e</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Siakor</span><span class="given-names" tagx="given-names" title="given-names">Matthew</span></span><a href="#af0030">f</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wolfman</span><span class="given-names" tagx="given-names" title="given-names">Vanessa</span></span><a href="#af0030">f</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Felix</span><span class="given-names" tagx="given-names" title="given-names">August</span></span><a href="#af0010">b</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sesay</span><span class="given-names" tagx="given-names" title="given-names">Tom</span></span><a href="#af0035">g</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shaffer</span><span class="given-names" tagx="given-names" title="given-names">Jeffrey G.</span></span><a href="#af0040">h</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garry</span><span class="given-names" tagx="given-names" title="given-names">Robert F.</span></span><a href="#af0045">i</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grant</span><span class="given-names" tagx="given-names" title="given-names">Donald S.</span></span><a href="#af0050">j</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levine</span><span class="given-names" tagx="given-names" title="given-names">Adam C.</span></span><span class="email" tagx="email" title="email">alevine@internationalmedicalcorps.org</span><a href="#af0030">f</a><a href="#af0055">k</a><a href="#cr0010">�?��?�</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sabeti</span><span class="given-names" tagx="given-names" title="given-names">Pardis C.</span></span><a href="#af0005">a</a><a href="#af0010">b</a><a href="#af0015">c</a><a href="#af0060">l</a></span></div><span class="citation_author_institution" id="af0005">[a], </span><span class="citation_author_institution" id="af0010">[b], </span><span class="citation_author_institution" id="af0015">[c], </span><span class="citation_author_institution" id="af0020">[d], </span><span class="citation_author_institution" id="af0025">[e], </span><span class="citation_author_institution" id="af0030">[f], </span><span class="citation_author_institution" id="af0035">[g], </span><span class="citation_author_institution" id="af0040">[h], </span><span class="citation_author_institution" id="af0045">[i], </span><span class="citation_author_institution" id="af0050">[j], </span><span class="citation_author_institution" id="af0055">[k], </span><span class="citation_author_institution" id="af0060">[l], </span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">�?�</span>Correspondence to: A. Colubri, Harvard University, Department of Organismic and Evolutionary Biology, Cambridge, USA. <span class="email" tagx="email" title="email">acolubri@fas.harvard.edu</span></div> <div class="corresp" title="corresp"><span class="label" tagx="label" title="label">�?��?�</span>Correspondence to: A. C. Levine, International Medical Corps, Los Angeles, USA. <span class="email" tagx="email" title="email">alevine@internationalmedicalcorps.org</span></div> </div><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2019-6-6</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2019</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-6-6</span></span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">54</span><span class="lpage" tagx="lpage" title="lpage">64</span><span class="history" title="history"><span class="received" title="received">received: 2018-9-17</span><span class="rev-recd" title="rev-recd">rev-recd: 2019-5-2</span><span class="accepted" title="accepted">accepted: 2019-6-7</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder" />, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2019</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article under the CC BY license (http://creativecommons.org/licenses/by/4.0/).</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p>Ebola virus disease (EVD) plagues low-resource and difficult-to-access settings. Machine learning prognostic models and mHealth tools could improve the understanding and use of evidence-based care guidelines in such settings. However, data incompleteness and lack of interoperability limit model generalizability. This study harmonizes diverse datasets from the 2014–16 EVD epidemic and generates several prognostic models incorporated into the novel <i>Ebola Care Guidelines</i> app that provides informed access to recommended evidence-based guidelines.</p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <p>Multivariate logistic regression was applied to investigate survival outcomes in 470 patients admitted to five Ebola treatment units in Liberia and Sierra Leone at various timepoints during 2014–16. We generated a parsimonious model (viral load, age, temperature, bleeding, jaundice, dyspnea, dysphagia, and time-to-presentation) and several fallback models for when these variables are unavailable. All were externally validated against two independent datasets and compared to further models including expert observational wellness assessments. Models were incorporated into an app highlighting the signs/symptoms with the largest contribution to prognosis.</p> </div> <div class="findings" title="sec"> <div class="title" tagx="title" title="title">
Findings</div> <p>The parsimonious model approached the predictive power of observational assessments by experienced clinicians (Area-Under-the-Curve, AUC = 0.70–0.79, accuracy = 0.64–0.74) and maintained its performance across subcohorts with different healthcare seeking behaviors. Age and viral load contributed &amp;gt; 5-fold the weighting of other features and including them in a minimal model had a similar AUC, albeit at the cost of specificity.</p> </div> <div class="interpretation" title="sec"> <div class="title" tagx="title" title="title">
Interpretation</div> <p>Clinically guided prognostic models can recapitulate clinical expertise and be useful when such expertise is unavailable. Incorporating these models into mHealth tools may facilitate their interpretation and provide informed access to comprehensive clinical guidelines.</p> </div> <div class="funding" title="sec"> <div class="title" tagx="title" title="title">
Funding</div> <p>Howard Hughes Medical Institute, US National Institutes of Health, Bill &amp;amp; Melinda Gates Foundation, International Medical Corps, UK Department for International Development, and GOAL Global.</p> </div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
Ebola virus disease</div> <div class="kwd" title="kwd">
Prognostic models</div> <div class="kwd" title="kwd">
Machine learning</div> <div class="kwd" title="kwd">
Data visualization</div> <div class="kwd" title="kwd">
Severity score</div> <div class="kwd" title="kwd">
mHealth</div> <div class="kwd" title="kwd">
Supportive care guidelines</div> <div class="kwd" title="kwd">
Clinical intuition</div> </div> </div> </div> <div class="body" title="body"> <p id="p0005"> </p><div class="boxed-text" tagx="boxed-text" title="boxed-text"> <div class="researchincontext" title="sec"> <div class="title" tagx="title" title="title">
Research in context</div> <div class="evidencebeforethisstudy" title="sec"> <div class="title" tagx="title" title="title">
Evidence before this study</div> <p id="p0010">The recent Ebola virus disease (EVD) outbreaks have highlighted the need for field-deployable patient management tools adaptable to its heterogeneous spectrum of pathology across widely varying environments and resources. In 2015, a WHO-led assessment of shortcomings in the initial response to the unfolding epidemic noted that "better information was needed to understand best practices in clinical management�? and "innovations in data collection should be introduced, including mHealth communications�?. Despite the availability of numerous clinical guidelines, their digital accessibility is poorly adapted to field conditions and difficult to navigate or read on mobile phones. Further, information is distributed across various guidelines, quickly outdated, and undirected reading is often overly detailed or too generic for the patient at hand. On the Google Play store, the search term "Ebola�? returned just three EVD-specific applications directed at clinicians, only one of which assimilated guidelines from 2014 within their current structure. Machine learning prognostic models could improve understanding and personalized use of evidence-based guidelines by helping to prioritize recommended interventions according to the prognostic importance of each clinical feature present in the individual patient. We searched PubMed without date limits for various permutations of terms representing "prognostic�?, "machine learning�?, "mHealth�?, "guidelines�? and "Ebola�?. Less than ten EVD symptomatic prognostic models were found, with most constructed on single-site cohorts without external validation. While one provided a paper scorecard, none were integrated in an mHealth tool or made use of machine learning.</p> </div> <div class="addedvalueofthisstudy" title="sec"> <div class="title" tagx="title" title="title">
Added value of this study</div> <p id="p0015">This study is based on the largest and most diverse clinical EVD dataset available to date, comprising 470 confirmed EVD cases from five different locations in Sierra Leone and Liberia, and a further 264 cases from two independent datasets for external validation. It demonstrates how interoperability between diverse datasets can be achieved through data harmonization approaches and constructs a family of flexible externally validated prognostic models that are able to approximate observational wellness assessments made by experienced clinicians. We further integrate these models in the first field-deployable mobile app for EVD prognostication, which enables informed access to recommended guidelines.</p> </div> <div class="implicationsofalltheavailableevidence" title="sec"> <div class="title" tagx="title" title="title">
Implications of all the available evidence</div> <p id="p0020">The <i>Ebola Care Guidelines</i> app benchmarks an approach to generating evidence-based mobile clinical management tools to improve training and informed adherence to protocol, where a personalized predictive support system organizes clinical procedures more effectively around patient data.</p> </div> </div> <p>Alt-text: Unlabelled Box</p> </div> <p /> <div class="introduction" title="sec"><span class="label" tagx="label" title="label">1</span><div class="title" tagx="title" title="title">
Introduction</div> <p id="p0025">The 2014–2016 outbreak of EVD caused a worldwide health crisis with more than 28,000 cases and 11,000 deaths, the vast majority of which occurred in the West African countries of Liberia, Sierra Leone, and Guinea. The recent ongoing outbreak in the Democratic Republic of the Congo <a href="#bb0005">[1]</a> is evidence of the threat posed by EVD, even with the availability of experimental vaccines. Of particular concern, is the presence of outbreaks in regions with limited medical coverage such as the active conflict zone affected by the current outbreak <a href="#bb0010">[2]</a>. Despite its notoriety as a deadly disease, the pathology of EVD includes a range of outcomes, spanning from asymptomatic infection to complex organ failure. On the few cases treated in high-income countries, case fatality ratios (CFRs) of under 20% were achievable, revealing the importance of resources in determining prognosis <a href="#bb0015">[3]</a>. Prioritizing time and material resources for high-risk patients in remote and low-resource settings is one approach to decrease overall mortality when subject to such constraints <a href="#bb0020">[4]</a>. A complementary approach is to use tools providing clinical instructions for management, training, and improved protocol adherence <a href="#bb0025">[5]</a>, <a href="#bb0030">[6]</a>.</p> <p id="p0030">We previously introduced the use of prognostic models that can be deployed as mobile apps for the purpose of risk stratification in EVD <a href="#bb0035">[7]</a>. Our original models were developed on the single publicly available dataset at that time by Schieffelin et al. <a href="#bb0040">[8]</a>, which includes 106 Ebola-positive patients admitted at Kenema Government Hospital (KGH). These models outperformed simpler risk scores and allowed users to choose from various sets of predictors depending on the available clinical data. While this study showed the potential for such an approach, the models were limited by the narrow geographical and temporal scope of the relatively small and incomplete dataset. Further, the prototype app in which the models were packaged displayed only the severity score of the patient without further guidance. We thus sought to create models with greatly expanded geographic relevance packaged in a new app that could provide risk-based guidance to health workers particularly in limited-resource settings. The models are derived from the largest and most diverse EVD dataset published to date, comprised of 470 confirmed cases from five treatment centers spread across Sierra Leone and Liberia at various timepoints during the 2014–16 epidemic (provided by International Medical Corps, IMC) and further externally validated on two datasets of 264 cases (comprising the 106 KGH dataset and an additional 158 provided by GOAL Global).</p> <p id="p0035">Despite the availability of numerous clinical guidelines for EVD, their digital accessibility is poorly adapted to field conditions where their book-like formatting makes for awkward navigation and reading on mobile devices. Further, the static nature of these documents makes them quickly outdated, especially in the rapidly evolving context of an epidemic where new recommendations are often fragmented across research papers and field reports that are challenging and sometimes impossible for health workers to access. Our goal is not only to provide an updatable platform on which care guidelines could be centralized, but also to personalize the prioritization of recommendations based on the severity score of the individual patient, as predicted by validated prognostic models. Such tailored guidance can be achieved by objectively highlighting the symptom-based interventions that are most relevant given all the available information about the patient during triage.</p> </div> <div class="methods" title="sec"><span class="label" tagx="label" title="label">2</span><div class="title" tagx="title" title="title">
Methods</div> <div class="imcpatientcohortanddatacollection" title="sec"><span class="label" tagx="label" title="label">2.1</span><div class="title" tagx="title" title="title">
IMC Patient Cohort and Data Collection</div> <p id="p0040">Prognostic models were based on data collected from 470 patients at five ETUs operated by IMC in Liberia (n = 178, 38%) and Sierra Leone (n = 292, 62%) between September 15, 2014 and September 15, 2015. The ETUs were located at Lunsar (Port Loko District), Kambia (Kambia District), and Makeni (Bombali District) in Sierra Leone, and at Suakoko (Bong County) and Kakata (Margibi County) in Liberia. The majority of the patients presented directly to these centers without secondary referrals from holding units. The overall Case Fatality Ratio (CFR) across the 5 ETUs was 58%.</p> <p id="p0045">Trained health workers recorded patient demographic, clinical, and support data at least daily from admission to discharge on standardized paper forms. Collection and archival protocols are detailed in Roshania et al. <a href="#bb0045">[9]</a>. The clinical and lab protocols were mostly consistent across the five ETUs, making it possible to aggregate individuals into a single cohort. Notable exceptions were wellness scale (WS) and body temperature, both of which were only recorded in Sierra Leonean ETUs. WS is an observational assessment assigned by experienced clinicians on a scale ranging from 0 (cured) to 5 (very sick) as seen in <a href="#t0005">Table 1</a>. WS was recorded for 223 of the 292 patients treated at the three ETUs in Sierra Leone and was imputed for the 89 Sierra Leonean patients without it. The categorical variable fever, available for all ETUs, was used to guide imputation of body temperatures. <div class="table-wrap_UNKNOWN" position="float" id="t0005"><span class="label" tagx="label" title="label">Table 1</span> </div></p><p>Wellness scale. Interpretation of the 0-to-5 observational scale of patient wellness at the Sierra Leonean ETUs.</p> [alt-text] Table 1 <table frame="hsides" rules="groups"> <thead> <tr> <th>Wellness scale</th> <th>Interpretation</th> </tr> </thead> <tbody> <tr> <td align="center">0</td> <td>Cured</td> </tr> <tr> <td align="center">1</td> <td>Well: no symptoms: drinks and eats okay</td> </tr> <tr> <td align="center">2</td> <td>Few symptoms: drinks and eats okay</td> </tr> <tr> <td align="center">3</td> <td>Moderate symptoms: can walk, sit, and feed independently</td> </tr> <tr> <td align="center">4</td> <td>Sick: needs help to be fed, drink, and take medications</td> </tr> <tr> <td align="center">5</td> <td>Very sick: needs IV fluids and medications, lots of assistance</td> </tr> </tbody> </table>  <p /> <p id="p0050">The cycle threshold (CT) value is an inversely proportional proxy of viral load, with a cut-off of 40 cycles considered as negative. These values were calculated from PCRs performed on admission or from the second PCR when the first was missing, performed no later than two days after admission and affecting a total of 28 patients. The mean CT value of these patients <a href="#bb0125">[25]</a> was not significantly different from the rest of the cohort <a href="#bb0120">[24]</a> in crude analyses or when stratified by outcome. PCR data was normalized across sites to adjust for analytical bias between laboratories (described below).</p> </div> <div class="externalvalidationcohorts" title="sec"><span class="label" tagx="label" title="label">2.2</span><div class="title" tagx="title" title="title">
External Validation Cohorts</div> <p id="p0055">External validation was performed on two independently collected datasets from Sierra Leone. The KGH dataset described by Schieffelin et al. <a href="#bb0040">[8]</a> is the only such database to be made publicly available at the time of this study (<a href="https://dataverse.harvard.edu/dataverse/ebola">https://dataverse.harvard.edu/dataverse/ebola</a>). It includes 106 EVD-positive cases treated at KGH between 25 May and 18 June 2014, where CFR was 73%. Signs and symptoms at triage were available for 44 patients and viral load was determined in 58 cases. Missingness across the dataset was 78%. The GOAL dataset described by Hartley et al. <a href="#bb0050">[10]</a>, <a href="#bb0055">[11]</a> includes 158 EVD-positive cases treated at the GOAL-Mathaska ETU in Port Loko between December 2014 and June 2015, where the CFR was 60%. RT-PCR results and detailed sign and symptom data were available for all 158 patients. Missingness across the dataset was 1%.</p> </div> <div class="acquisition,transformationandnormalizationofcyclethresholdvalues" title="sec"><span class="label" tagx="label" title="label">2.3</span><div class="title" tagx="title" title="title">
Acquisition, Transformation and Normalization of Cycle Threshold Values</div> <p id="p0060">Further details on the assays and processing methodologies are provided in the supplementary materials (PCR Lab Notes section). RT-PCR data were obtained from four laboratories. Liberian ETUs were served by the United States Naval Medical Research Center (NMRC) Mobile Laboratory in Bong County. In Sierra Leone, the Lunsar, Makeni and GOAL ETU's were served by Public Health England (PHE), while a Nigerian Lab handled samples from the Kambia ETU. KGH had on-site laboratory support. Inconsistent with other sites, the KGH PCR data was reported as viral load (VL, in copies/ml) only. To harmonize measures, VL was transformed to CT according to the standard qPCR curve log(VL) = m × CT + c0, such that the minimum VL in the KGH dataset corresponded to the maximum CT in the aggregated IMC dataset, and vice versa. The accuracy of the transformation was supported by the observations that its limit of detection aligned to the IMC dataset, <a href="#bb0060">[12]</a>, <a href="#bb0065">[13]</a>, <a href="#bb0070">[14]</a> and also that the curve corresponded to its expected form, where a ≈ 10-fold increase in Ebola VL corresponded to a 3-point decrease in CT <a href="#bb0075">[15]</a>. Based on this relationship, − 3/m in our formula should be close to 1, which is indeed the case (− 3/m = 0.976 using the m and c0 constants derived from the KGH and IMC data). Transformation results were not dependent on the geographical origin of the IMC data.</p> <p id="p0065">Geographical origin did, however, have a significant (P &amp;lt; 0.0001) impact on CT distribution across sites (Suppl. Fig. S1), where IMC ETU's had a mean of 21.82 ± 5.16 in Sierra Leone and 27.67 ± 5.45 in Liberia. In the KGH and GOAL external validation cohorts, these values were 26.05 ± 6.00 and 22.20 ± 4.31, respectively. A possible reason for this discrepancy is the differing methods of RT-PCR in each site (TaqMan in the Liberian ETUs, commercial Altona and in-house "Trombley�? in the Sierra Leonean ETUs). To correct for such analytical bias, and ensure the results were internally consistent at each site and thus comparable between sites, we normalized the CT values from each site/group of sites by feature-scaling (subtracting the mean and dividing by the standard deviation). As the differences in CT could also arise from differences in care-seeking behaviors, the normalization of CT values within sites would also work towards reducing the effect of this potential confounder.</p> </div> <div class="ethicalapproval" title="sec"><span class="label" tagx="label" title="label">2.4</span><div class="title" tagx="title" title="title">
Ethical Approval</div> <p id="p0070">The Sierra Leone Ethics and Scientific Review Committee, the University of Liberia – Pacific Institute for Research &amp;amp; Evaluation Institutional Review Board, the Lifespan (Rhode Island Hospital) Institutional Review Board, and the Harvard Committee on the Use of Human Subjects provided ethical approval for this study and exemption from informed consent. A data sharing agreement was approved by IMC and the Broad Institute, following IMC's Research Review Committee Guidelines (<a href="https://internationalmedicalcorps.org/document.doc?id=800">https://internationalmedicalcorps.org/document.doc?id=800</a>).</p> </div> <div class="exploratoryandbivariateanalysis" title="sec"><span class="label" tagx="label" title="label">2.5</span><div class="title" tagx="title" title="title">
Exploratory and Bivariate Analysis</div> <p id="p0075">Analyses were undertaken in R version 3.5.1. The primary outcome was final disposition (death or survival). The outcome of five patients was missing due to being transferred to another facility. We carried out an initial bivariate analysis of all factors against outcome, using the χ<sup>2</sup> test with Yates correction for the binary variables, and the point biserial correlation test for numerical variables.</p> </div> <div class="multipleimputation" title="sec"><span class="label" tagx="label" title="label">2.6</span><div class="title" tagx="title" title="title">
Multiple imputation</div> <p id="p0080">Across all the potential predictors in the dataset, 22% of fields had missing values. Predictor exclusion was limited to those having more than 50% missingness so as to minimize bias in the regression coefficients <a href="#bb0080">[16]</a>. The remaining missing values were assessed for their suitability for imputation, using Little's MCAR test (R package BaylorEdPsych), which tests for bias in the missing values or whether they are "missing completely at random�? (MCAR) <a href="#bb0085">[17]</a>. Multiple imputation was undertaken with the aregImpute function from the R package Hmisc <a href="#bb0090">[18]</a>, which generates a Bayesian predictive distribution from the known data, and outputs a number N of imputed datasets. Each missing value in the i<sup>th</sup> imputation is predicted from an additive model fitted on a bootstrap sample with replacement from the original data. We set N = 100, well above standard imputation guidelines <a href="#bb0095">[19]</a>.</p> </div> <div class="machinelearningfeatureselectionandmultivariatemodeling" title="sec"><span class="label" tagx="label" title="label">2.7</span><div class="title" tagx="title" title="title">
Machine learning Feature Selection and Multivariate Modeling</div> <p id="p0085">Several logistic regression models were constructed to predict the binary outcome death/survival from the demographic, clinical (signs and symptoms), and laboratory data (viral load and malaria rapid test results) collected at triage. The predictors for the models were selected in a machine learning variable-selection process, where an initial set of candidate factors associated with death in EVD were submitted to penalized logistic regression using the R package Glmnet <a href="#bb0100">[20]</a> with Elastic Net regularization. To define a non-redundant, parsimonious subset of predictors, variables with a coefficient &amp;gt; 0 in at least half of the penalized models were retained. As marginal associations can become less significant when accounting for confounding dependencies in such multivariate models, the process tended to include variables with weak correlations to the outcome in bivariate analysis, such as bleeding and dyspnea, and elimination of variables that had a low P-value, like conjunctivitis. Using these variables, we constructed a family of non-penalized logistic regression models using the lrm function from the R package rms <a href="#bb0105">[21]</a>. This included a parsimonious model (using all the variables obtained from the selection process), and several fallback models that can be applied on smaller subsets of demographic information, clinical features and laboratory results, for use when more limited data is available at triage. This approach has been shown to outperform predictive value imputation, which consists of having only one full model and imputing missing values at prediction time using the data distribution from the training set <a href="#bb0110">[22]</a>. Each final model in the family was obtained by fitting N copies of the model on each imputed dataset, and then averaging those copies into a single model using the fit.mult.impute function in Hmisc. Internal validation was performed using bootstrap resampling and model performance is presented by the area under the curve (AUC), McFadden's pseudo-R<sup>2</sup>, Brier score, accuracy, sensitivity and specificity. A prediction was classified as death when the score from the model was over the 0.5 threshold. Confidence intervals (CI) of all the performance estimates were calculated using Fisher's transformation <a href="#bb0115">[23]</a>. Odds ratios (OR) from logistic regression were converted to risk ratios (RRs) according to Zhang and Yu <a href="#bb0120">[24]</a>.</p> </div> </div> <div class="results" title="sec"><span class="label" tagx="label" title="label">3</span><div class="title" tagx="title" title="title">
Results</div> <div class="prognosticpotentialandprevalenceofsignsandsymptomsrecordedattriage" title="sec"><span class="label" tagx="label" title="label">3.1</span><div class="title" tagx="title" title="title">
Prognostic Potential and Prevalence of Signs and Symptoms Recorded at Triage</div> <p id="p0090">Triage symptoms reported by over 50% of fatal Ebola patients were anorexia/loss of appetite, fever, asthenia/weakness, musculoskeletal pain, headache and diarrhea (<a href="#t0010">Table 2</a>A). Prevalence of several triage symptoms was notably different between fatal and non-fatal outcomes, as can be seen by comparing their ranking (Suppl. Fig. 2A) or their differential prevalence (Suppl. Fig. 2B). Few variables were significantly associated with patient outcome. Only CT, age (<a href="#t0010">Table 2</a>B), and jaundice (<a href="#t0010">Table 2</a>A) were associated with death at a level of P &amp;lt; 0.05, while conjunctivitis, confusion, dyspnea, headache, and bleeding were weakly associated at P &amp;lt; 0.15. <div class="table-wrap_UNKNOWN" position="float" id="t0010"><span class="label" tagx="label" title="label">Table 2</span> </div></p><p>Bivariate analysis. Correlation between binary (A) and continuous (B) clinical variables and the outcome of death. For binary variables, crude marginal risk-ratios (RR) were obtained from the 2 × 2 contingency table. For continuous variables, the odds ratios (OR) correspond to inter-quartile range changes in the variables when used as the only predictor of death (and 10-years increase in the case of age).</p> [alt-text] Table 2 <table frame="hsides" rules="groups"> <thead> <tr> <th colspan="7">A <hr /> </th> </tr> <tr> <th>Variable</th> <th>Total (%)</th> <th>Non-fatal (%)</th> <th>Fatal (%)</th> <th>Missing (%)</th> <th>RR (95% CI)</th> <th>P-value</th> </tr> </thead> <tbody> <tr> <td>Jaundice</td> <td align="center">24/464 (5)</td> <td align="center">4/197 (2)</td> <td align="center">20/267 (7)</td> <td align="center">1/470 (0)</td> <td align="center">1.06 (1.02, 1.10)</td> <td align="center">0.016</td> </tr> <tr> <td>Conjunctivitis</td> <td align="center">128/464 (27)</td> <td align="center">64/197 (32)</td> <td align="center">64/267 (23)</td> <td align="center">1/470 (0)</td> <td align="center">0.89 (0.79, 1.00)</td> <td align="center">0.054</td> </tr> <tr> <td>Coma<a href="#tf0005">a</a></td> <td align="center">5/178 (2)</td> <td align="center">0/83 (0)</td> <td align="center">5/95 (5)</td> <td align="center">292/470 (62)</td> <td align="center">1.05 (1.00, 1.11)</td> <td align="center">0.096</td> </tr> <tr> <td>Confusion<a href="#tf0005">a</a></td> <td align="center">16/178 (8)</td> <td align="center">4/83 (4)</td> <td align="center">12/95 (12)</td> <td align="center">292/470 (62)</td> <td align="center">1.09 (1.00, 1.19)</td> <td align="center">0.120</td> </tr> <tr> <td>Dyspnea</td> <td align="center">109/464 (23)</td> <td align="center">39/197 (19)</td> <td align="center">70/267 (26)</td> <td align="center">1/470 (0)</td> <td align="center">1.09 (0.98, 1.20)</td> <td align="center">0.133</td> </tr> <tr> <td>Headache</td> <td align="center">268/464 (57)</td> <td align="center">122/197 (61)</td> <td align="center">146/267 (54)</td> <td align="center">1/470 (0)</td> <td align="center">0.84 (0.67, 1.05)</td> <td align="center">0.142</td> </tr> <tr> <td>Bleeding</td> <td align="center">26/464 (5)</td> <td align="center">7/197 (3)</td> <td align="center">19/267 (7)</td> <td align="center">1/470 (0)</td> <td align="center">1.04 (0.99, 1.08)</td> <td align="center">0.148</td> </tr> <tr> <td>Asthenia/weakness</td> <td align="center">334/464 (71)</td> <td align="center">135/197 (68)</td> <td align="center">199/267 (74)</td> <td align="center">1/470 (0)</td> <td align="center">1.24 (0.92, 1.65)</td> <td align="center">0.187</td> </tr> <tr> <td>Diarrhea</td> <td align="center">234/430 (54)</td> <td align="center">96/187 (51)</td> <td align="center">138/243 (56)</td> <td align="center">35/470 (7)</td> <td align="center">1.13 (0.92, 1.38)</td> <td align="center">0.304</td> </tr> <tr> <td>Malaria<a href="#tf0010">b</a></td> <td align="center">49/225 (21)</td> <td align="center">17/94 (18)</td> <td align="center">32/131 (24)</td> <td align="center">241/470 (51)</td> <td align="center">1.08 (0.95, 1.24)</td> <td align="center">0.331</td> </tr> <tr> <td>Dysphagia</td> <td align="center">112/464 (24)</td> <td align="center">43/197 (21)</td> <td align="center">69/267 (25)</td> <td align="center">1/470 (0)</td> <td align="center">1.05 (0.95, 1.17)</td> <td align="center">0.374</td> </tr> <tr> <td>Vomiting</td> <td align="center">197/464 (42)</td> <td align="center">87/197 (44)</td> <td align="center">110/267 (41)</td> <td align="center">1/470 (0)</td> <td align="center">0.95 (0.81, 1.11)</td> <td align="center">0.587</td> </tr> <tr> <td>Nausea<a href="#tf0010">b</a></td> <td align="center">94/286 (32)</td> <td align="center">35/114 (30)</td> <td align="center">59/172 (34)</td> <td align="center">179/470 (38)</td> <td align="center">1.05 (0.90, 1.24)</td> <td align="center">0.613</td> </tr> <tr> <td>Abdominal pain</td> <td align="center">203/464 (43)</td> <td align="center">89/197 (45)</td> <td align="center">114/267 (42)</td> <td align="center">1/470 (0)</td> <td align="center">0.96 (0.81, 1.13)</td> <td align="center">0.662</td> </tr> <tr> <td>Bone/muscle/joint pain</td> <td align="center">272/465 (58)</td> <td align="center">118/197 (59)</td> <td align="center">154/268 (57)</td> <td align="center">0/470 (0)</td> <td align="center">0.94 (0.76, 1.17)</td> <td align="center">0.666</td> </tr> <tr> <td>Throat pain<a href="#tf0005">a</a></td> <td align="center">55/178 (30)</td> <td align="center">24/83 (28)</td> <td align="center">31/95 (32)</td> <td align="center">292/470 (62)</td> <td align="center">1.06 (0.87, 1.28)</td> <td align="center">0.709</td> </tr> <tr> <td>Cough<a href="#tf0005">a</a></td> <td align="center">61/178 (34)</td> <td align="center">30/83 (36)</td> <td align="center">31/95 (32)</td> <td align="center">292/470 (62)</td> <td align="center">0.95 (0.77, 1.17)</td> <td align="center">0.738</td> </tr> <tr> <td>Hiccups</td> <td align="center">55/464 (11)</td> <td align="center">22/197 (11)</td> <td align="center">33/267 (12)</td> <td align="center">1/470 (0)</td> <td align="center">1.01 (0.95, 1.08)</td> <td align="center">0.805</td> </tr> <tr> <td>Rash<a href="#tf0005">a</a></td> <td align="center">8/178 (4)</td> <td align="center">3/83 (3)</td> <td align="center">5/95 (5)</td> <td align="center">292/470 (62)</td> <td align="center">1.02 (0.96, 1.08)</td> <td align="center">0.867</td> </tr> <tr> <td>Chest pain<a href="#tf0005">a</a></td> <td align="center">88/178 (49)</td> <td align="center">41/83 (49)</td> <td align="center">47/95 (49)</td> <td align="center">292/470 (62)</td> <td align="center">1 (0.75, 1.34)</td> <td align="center">0.889</td> </tr> <tr> <td>Photophobia<a href="#tf0005">a</a></td> <td align="center">24/178 (13)</td> <td align="center">11/83 (13)</td> <td align="center">13/95 (13)</td> <td align="center">292/470 (62)</td> <td align="center">1 (0.89, 1.13)</td> <td align="center">0.892</td> </tr> <tr> <td>Anorexia/poor appetite</td> <td align="center">316/464 (68)</td> <td align="center">135/197 (68)</td> <td align="center">181/267 (67)</td> <td align="center">1/470 (0)</td> <td align="center">0.98 (0.75, 1.28)</td> <td align="center">0.946</td> </tr> <tr> <td>Fever</td> <td align="center">349/464 (75)</td> <td align="center">148/197 (75)</td> <td align="center">201/267 (75)</td> <td align="center">1/470 (0)</td> <td align="center">1.01 (0.73, 1.39)</td> <td align="center">0.944</td> </tr> <tr> <td colspan="7">  </td> </tr> </tbody> </table> <table frame="hsides" rules="groups"> <thead> <tr> <th colspan="7">B <hr /> </th> </tr> <tr> <th>Variable</th> <th>Non-fatal (mean, 95% CI)</th> <th>Fatal (mean, 95% CI)</th> <th>Missing fraction (%)</th> <th>Pearson's R</th> <th>OR (95% CI)</th> <th>P-value</th> </tr> </thead> <tbody> <tr> <td>Cycle threshold</td> <td align="center">26.72 (15.92, 37.52)</td> <td align="center">22.23 (11.18, 33.28)</td> <td align="center">137/470 (29)</td> <td align="center">− 0.37</td> <td align="center">0.33 (0.23, 0.47)</td> <td align="center">&amp;lt; 0.0001</td> </tr> <tr> <td>Wellness scale<a href="#tf0010">b</a></td> <td align="center">2.49 (0.82, 4.17)</td> <td align="center">3.20 (1.17, 5.22)</td> <td align="center">247/470 (52)</td> <td align="center">0.34</td> <td align="center">4.57 (2.44, 8.58)</td> <td align="center">&amp;lt; 0.0001</td> </tr> <tr> <td>Patient age</td> <td align="center">28.49 (0.00, 58.72)</td> <td align="center">32.03 (0.00, 72.10)</td> <td align="center">4/470 (1)</td> <td align="center">0.09</td> <td align="center">1.33 (1.01, 1.74)</td> <td align="center">0.043</td> </tr> <tr> <td>Body temperature<a href="#tf0010">b</a></td> <td align="center">37.41 (35.50, 39.32)<a href="#tf0005">a</a></td> <td align="center">37.67 (35.34, 40.01)</td> <td align="center">265/470 (56)</td> <td align="center">0.12</td> <td align="center">1.39 (0.94, 2.06)</td> <td align="center">0.099</td> </tr> <tr> <td>Time-to-presentation</td> <td align="center">4.28 (0.00, 12.03)</td> <td align="center">4.20 (0.00, 9.90)</td> <td align="center">108/470 (22)</td> <td align="center">− 0.01</td> <td align="center">0.97 (0.76, 1.25)</td> <td align="center">0.83</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">a</span><p id="np0005">Variables not recorded at the Sierra Leonean ETUs</p> </div> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""><span class="label" tagx="label" title="label">b</span><p id="np0010">Variables not recorded at the Liberian ETUs.</p> </div> </div>  <p /> </div> <div class="multivariatelogisticregressionmodels" title="sec"><span class="label" tagx="label" title="label">3.2</span><div class="title" tagx="title" title="title">
Multivariate Logistic Regression Models</div> <div class="variableselection" title="sec"><span class="label" tagx="label" title="label">3.2.1</span><div class="title" tagx="title" title="title">
Variable Selection</div> <p id="p0095">Applying a machine learning selection model described in the methods, we obtained a set of variables including patient age, Ebola-specific PCR result (CT), and several clinical features recorded at triage (body temperature, bleeding, jaundice, dyspnea, dysphagia, and time-to-presentation (TTP)).</p> </div> <div class="multipleimputation" title="sec"><span class="label" tagx="label" title="label">3.2.2</span><div class="title" tagx="title" title="title">
Multiple Imputation</div> <p id="p0100">Before procedure, selected variables were assessed for bias. We applied restricted cubic splines to model the non-linear relationships between CFR and age (Suppl. Fig. S2A) and CFR and body temperature (Suppl. Fig. S2B). The actual values of age and temperature were the input of the spline fitting procedure, as implemented in the Hmisc package. The lack of a saddle in the CFR vs CT plot (Suppl. Fig. S3) suggested that cubic splines were not required to model CT. Following Hartley's <a href="#bb0050">[10]</a> observation that viral load can act as a confounding factor for outcome and TTP (where patterns in early viral load directs care-seeking behavior as well as determines outcome), we added a CT × TTP interaction. While non-random patterns of missing values were found when considering all data, Little's MCAR test was satisfied at P = 0.05 when considering the Sierra Leonean and Liberian records separately, thus identifying geographical location as the main source of non-randomness. The remaining weak non-random patterns for missing fields in the TTP variable in Sierra Leonean records (P = 0.03) could be explained by low CT values. Inclusion of CT in the model controlled for this effect and ensured randomness within each subset of CT (high versus low).</p> </div> <div class="modelperformance" title="sec"><span class="label" tagx="label" title="label">3.2.3</span><div class="title" tagx="title" title="title">
Model Performance</div> <p id="p0105">In addition to the parsimonious model (using all features described above), we constructed three fallback models for use when certain features were unavailable. The first "parsimonious without temperature�? model was identical to the parsimonious but with backwards elimination of body temperature. A second "clinical-only�? model selected only clinical features after omitting CT from the initial list of variables before the machine learning selection step. This model included jaundice, bleeding, dyspnea, dysphagia, asthenia/weakness, and diarrhea. A final "minimal�? model only incorporated CT and age, which were the strongest individual predictors of outcome, as observed in our data and reported by other researchers <a href="#bb0125">[25]</a>. <a href="#t0015">Table 3</a> contains the validation indices and their 95% CIs. Most indices are very similar between the parsimonious and minimal models, with largely overlapping CIs. The specificity and R<sup>2</sup> are higher in the parsimonious models, while the sensitivity is slightly higher in the minimal model. The addition of temperature consistently increases performance across all indices, but only by a minor amount. The clinical-only model shows consistently lower performance. Detailed description of all the models is provided in Suppl. Table S1. <div class="table-wrap_UNKNOWN" position="float" id="t0015"><span class="label" tagx="label" title="label">Table 3</span> </div></p><p>Validation indices for the prognostic models. These indices include AUC, McFadden's pseudo R<sup>2</sup> goodness-of-fit index, Brier score, overall accuracy, sensitivity, and specificity. The means and 95% confidence intervals were obtained with 200 iterations of bootstrap resampling.</p> [alt-text] Table 3 <table frame="hsides" rules="groups"> <thead> <tr> <th /> <th>Parsimonious (95% CI)</th> <th>Parsimonious w/out temp. (95% CI)</th> <th>Clinical-only (95% CI)</th> <th>Minimal (95% CI)</th> </tr> </thead> <tbody> <tr> <td>AUC</td> <td align="center">0.75 (0.70, 0.79)</td> <td align="center">0.74 (0.69, 0.78)</td> <td align="center">0.64 (0.58, 0.69)</td> <td align="center">0.76 (0.71, 0.80)</td> </tr> <tr> <td>R<sup>2</sup></td> <td align="center">0.22 (0.17, 0.27)</td> <td align="center">0.21 (0.16, 0.25)</td> <td align="center">0.12 (0.09, 0.15)</td> <td align="center">0.16 (0.13, 0.21)</td> </tr> <tr> <td>Brier</td> <td align="center">0.21 (0.16, 0.25)</td> <td align="center">0.21 (0.16, 0.26)</td> <td align="center">0.24 (0.19, 0.29)</td> <td align="center">0.21 (0.16, 0.25)</td> </tr> <tr> <td>Accuracy</td> <td align="center">0.69 (0.64, 0.74)</td> <td align="center">0.68 (0.63, 0.73)</td> <td align="center">0.60 (0.54, 0.66)</td> <td align="center">0.68 (0.63, 0.73)</td> </tr> <tr> <td>Sensitivity</td> <td align="center">0.80 (0.75, 0.84)</td> <td align="center">0.79 (0.74, 0.83)</td> <td align="center">0.70 (0.64, 0.75)</td> <td align="center">0.81 (0.76, 0.85)</td> </tr> <tr> <td>Specificity</td> <td align="center">0.57 (0.51, 0.63)</td> <td align="center">0.55 (0.49, 0.61)</td> <td align="center">0.51 (0.44, 0.57)</td> <td align="center">0.52 (0.46, 0.58)</td> </tr> </tbody> </table>  <p /> <p id="p0110">The calibration curves comparing the predicted and actual probabilities of death (<a href="#f0005">Fig. 1</a>) suggest that the two parsimonious models represent the actual probabilities well, with some underestimation for low risk patients. The minimal and clinical-only models are less calibrated, with the former underestimating the actual probabilities at both the low- and high-risk ends, and the later overestimating the actual probabilities for a wide range of risks above 60%. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <p>Bootstrap overfitting-corrected calibration curve. Estimated for the four prognostic models: parsimonious (A), parsimonious without body temperature (B), clinical-only (C), and minimal (D). Each plot contains the rug chart at the top showing the distribution of predicted risks. The dotted line represents the apparent calibration curve and the solid line shows the optimism-corrected calibration. A perfectly-calibrated model will fall along the diagonal. Generated with the calibrate function in the rms package.</p>   </div> <p /> <p id="p0115">The ranking of all of the variables by their importance in the parsimonious model, as measured by the Wald χ<sup>2</sup> statistic, reveals that the most important variables are CT and patient age, with jaundice and bleeding coming in at a distant third and fourth place respectively, followed by body temperature and the CT-TTP interaction (<a href="#f0010">Fig. 2</a>A). The odds ratios (<a href="#f0010">Fig. 2</a>B) indicate that presentation of either jaundice or bleeding are associated with more than a doubling of the odds of death, although their prevalence is low at 5% (<a href="#t0010">Table 2</a>). Transformation of the odds risks into risk ratios (Suppl. Table S1) yields a more reasonable estimate of 4% increase in risk associated with the occurrence of those symptoms at presentation. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <p>Evaluation of predictor variables in the parsimonious model. Ranking of the variables according to their predictive importance in the model, as measured by the χ<sup>2</sup>-d.f. (degrees of freedom) statistic (A). Odds ratios for all the variables, using interquartile-range odds ratios for continuous features, and simple odds ratios for categorical features (B). Generated with the anova.rms and summary functions in the rms and base packages in R.</p>   </div> <p /> </div> <div class="temporalflexibilityofmodels" title="sec"><span class="label" tagx="label" title="label">3.2.4</span><div class="title" tagx="title" title="title">
Temporal Flexibility of Models</div> <p id="p0120">To test the temporal flexibility of models we compared the predictions from the parsimonious model when using the CT value from day 0 versus day 2, for those patients for whom these CT values where available. There were 84 patients in the d0 subcohort (CFR: 63%) and 14 patients in the d2 subcohort (CFR: 64%). The model performed similarly on these cohorts (Suppl. Table S2A) with 15 misclassifications using d0 CT reads subcohort, and 3 using d2 CT reads. Further, comparing patients with slow (&amp;gt; 3 d) or rapid (≤ 3 d) times to presentation, we see little difference (Suppl. Table S2B) in performance parameters for the parsimonious model.</p> </div> </div> <div class="externalvalidation" title="sec"><span class="label" tagx="label" title="label">3.3</span><div class="title" tagx="title" title="title">
External Validation</div> <p id="p0125">External validation on the 158 EVD-positive patients in the GOAL dataset shows that the four models described earlier improve their performance with respect to the internal validation. CT was missing in 37 patients, and both CT and TTP were missing in 53, so the parsimonious models were validated on 105 patients and the minimal model on 121. The clinical-only model was validated on all the 158 patients. We obtained AUCs of 0.82, 0.84, 0.73, and 0.82 for the parsimonious, parsimonious without temperature, clinical-only, and minimal (<a href="#t0020">Table 4</a>A). In terms of the accuracy, sensitivity, and specificity, all the models, with the exception of the clinical-only, performed similarly well with accuracies around 74%. Excluding again the clinical-only model, sensitivity ranged between 78% and 82%, while specificity was lowest for the minimal model at 61% and highest for the parsimonious model at 68%. Therefore, it is not possible to pick an overall best model between the three including CT but depending on what is the priority in the predictive task, higher sensitivity or specificity, one could favor either the parsimonious or the minimal. <div class="table-wrap_UNKNOWN" position="float" id="t0020"><span class="label" tagx="label" title="label">Table 4</span> </div></p><p>External validation on the GOAL (A) and KGH (B) datasets. The AUC, Brier, accuracy, sensitivity, specificity, positive predictive values (PPV) and negative predictive values (NPV) indices were calculated (A) on all the records from the GOAL dataset with complete data and (B) all the records from the KGH dataset that had enough data to evaluate the clinical-only and minimal models. The results for the parsimonious models were obtained after imputing missing values in the KGH patients, and the performance indices include the mean and standard deviation over 100 multiple imputations.</p> [alt-text] Table 4 <table frame="hsides" rules="groups"> <thead> <tr> <th colspan="5">A <hr /> </th> </tr> <tr> <th /> <th>Parsimonious</th> <th>Parsimonious w/out temp.</th> <th>Clinical-only</th> <th>Minimal</th> </tr> </thead> <tbody> <tr> <td>AUC</td> <td align="center">0.82</td> <td align="center">0.84</td> <td align="center">0.73</td> <td align="center">0.82</td> </tr> <tr> <td>Brier</td> <td align="center">0.17</td> <td align="center">0.16</td> <td align="center">0.21</td> <td align="center">0.17</td> </tr> <tr> <td>Accuracy</td> <td align="center">0.74</td> <td align="center">0.75</td> <td align="center">0.66</td> <td align="center">0.74</td> </tr> <tr> <td>Sensitivity</td> <td align="center">0.78</td> <td align="center">0.82</td> <td align="center">0.77</td> <td align="center">0.82</td> </tr> <tr> <td>Specificity</td> <td align="center">0.68</td> <td align="center">0.63</td> <td align="center">0.52</td> <td align="center">0.61</td> </tr> <tr> <td>PPV</td> <td align="center">0.81</td> <td align="center">0.80</td> <td align="center">0.66</td> <td align="center">0.79</td> </tr> <tr> <td>NPV</td> <td align="center">0.63</td> <td align="center">0.67</td> <td align="center">0.65</td> <td align="center">0.66</td> </tr> </tbody> </table> <table frame="hsides" rules="groups"> <thead> <tr> <th colspan="5">B <hr /> </th> </tr> <tr> <th /> <th>Parsimonious (SD)</th> <th>Parsimonious w/out temp. (SD)</th> <th>Clinical-only</th> <th>Minimal</th> </tr> </thead> <tbody> <tr> <td>AUC</td> <td align="center">0.78 (0.04)</td> <td align="center">0.78 (0.04)</td> <td align="center">0.72</td> <td align="center">0.82</td> </tr> <tr> <td>Brier</td> <td align="center">0.19 (0.01)</td> <td align="center">0.18 (0.01)</td> <td align="center">0.22</td> <td align="center">0.17</td> </tr> <tr> <td>Accuracy</td> <td align="center">0.73 (0.03)</td> <td align="center">0.72 (0.03)</td> <td align="center">0.59</td> <td align="center">0.74</td> </tr> <tr> <td>Sensitivity</td> <td align="center">0.73 (0.03)</td> <td align="center">0.73 (0.03)</td> <td align="center">0.56</td> <td align="center">0.82</td> </tr> <tr> <td>Specificity</td> <td align="center">0.88 (0.03)</td> <td align="center">0.86 (0.02)</td> <td align="center">0.75</td> <td align="center">0.61</td> </tr> <tr> <td>PPV</td> <td align="center">0.88 (0.03)</td> <td align="center">0.86 (0.02)</td> <td align="center">0.91</td> <td align="center">0.81</td> </tr> <tr> <td>NPV</td> <td align="center">0.48 (0.05)</td> <td align="center">0.48 (0.05)</td> <td align="center">0.27</td> <td align="center">0.50</td> </tr> </tbody> </table>  <p /> <p id="p0130">For the KGH dataset (<a href="#t0020">Table 4</a>B), only 11 patients had enough complete records to validate the parsimonious models, but after multiple imputation, all the 106 KGH cases could be used. As it was observed in the first validation, all models with the exception of the clinical-only, exhibited consistent performance in terms of AUC and overall accuracy, with analogous differences in terms of specificity and sensitivity. The parsimonious models have higher specificity, while the minimal is more sensitive. The impact of imputation was inconsequential.</p> </div> <div class="wellnessscalemodels" title="sec"><span class="label" tagx="label" title="label">3.4</span><div class="title" tagx="title" title="title">
Wellness Scale Models</div> <p id="p0135">Four additional models were constructed using the wellness scale (WS) variable in place of the detailed clinical signs and symptoms: wellness parsimonious (including CT, patient age, body temperature, WS, TTP and CT x TTP), wellness parsimonious without temperature, wellness clinical-only (including patient age, body temperature, and WS), and wellness minimal (including CT, patient age, and WS). The performance indices of these models, obtained from internal validation on the Sierra Leonean patients from the IMC dataset, are shown in <a href="#t0025">Table 5</a>. <div class="table-wrap_UNKNOWN" position="float" id="t0025"><span class="label" tagx="label" title="label">Table 5</span> </div></p><p>Validation of the wellness scale models. These models were evaluated using the same indices of performance as the previous models: AUC, R<sup>2</sup>, Brier, accuracy, sensitivity, and specificity. The means and 95% confidence intervals were obtained with 200 iterations of bootstrap resampling.</p> [alt-text] Table 5 <table frame="hsides" rules="groups"> <thead> <tr> <th /> <th>Wellness parsimonious (95% CI)</th> <th>Wellness parsimonious w/out temp. (95% CI)</th> <th>Wellness clinical-only (95% CI)</th> <th>Wellness minimal (95% CI)</th> </tr> </thead> <tbody> <tr> <td>AUC</td> <td align="center">0.80 (0.74, 0.84)</td> <td align="center">0.81 (0.76, 0.85)</td> <td align="center">0.74 (0.68, 0.80)</td> <td align="center">0.80 (0.74, 0.84)</td> </tr> <tr> <td>R<sup>2</sup></td> <td align="center">0.31 (0.24, 0.38)</td> <td align="center">0.30 (0.23, 0.37)</td> <td align="center">0.19 (0.14, 0.25)</td> <td align="center">0.25 (0.19, 0.32)</td> </tr> <tr> <td>Brier</td> <td align="center">0.19 (0.13, 0.24)</td> <td align="center">0.18 (0.13, 0.23)</td> <td align="center">0.20 (0.15, 0.26)</td> <td align="center">0.19 (0.13, 0.24)</td> </tr> <tr> <td>Accuracy</td> <td align="center">0.73 (0.66, 0.79)</td> <td align="center">0.73 (0.67, 0.79)</td> <td align="center">0.69 (0.62, 0.75)</td> <td align="center">0.72 (0.65, 0.77)</td> </tr> <tr> <td>Sensitivity</td> <td align="center">0.82 (0.76, 0.86)</td> <td align="center">0.82 (0.77, 0.86)</td> <td align="center">0.79 (0.72, 0.83)</td> <td align="center">0.81 (0.75, 0.85)</td> </tr> <tr> <td>Specificity</td> <td align="center">0.63 (0.55, 0.70)</td> <td align="center">0.63 (0.55, 0.70)</td> <td align="center">0.57 (0.48, 0.64)</td> <td align="center">0.61 (0.53, 0.68)</td> </tr> </tbody> </table>  <p /> <p id="p0140">All the performance indices of these new models are higher than the corresponding original models. In fact, the performance of the worst performing model in this set, the wellness clinical-only, is comparable to that of the parsimonious model including the detailed signs and symptoms. The calibration curves for the parsimonious models with and without wellness scale are almost indistinguishable, but it is possible to note an improvement for the clinical-only and minimal models with the addition of the wellness scale (<a href="#f0015">Fig. 3</a>). To evaluate the effect of imputation of wellness score, we fitted the four wellness models only on those patients with known WS (Suppl. Table S3) and found imputed variance inflation to be minimal. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3</span></a></div>   <p>Bootstrap overfitting-corrected calibration curves for the wellness models. Estimated for the four prognostic models using the wellness scale as predictor: wellness parsimonious (A), wellness parsimonious without body temperature (B), wellness clinical-only (C), and wellness minimal (D). Each plot contains the rug chart at the top showing the distribution of predicted risks. The dotted line represents the apparent calibration curve and the solid line shows the optimism-corrected calibration. A perfectly-calibrated model will fall along the diagonal. Generated with the calibrate function in the rms package.</p>   </div> <p /> </div> <div class="ebolacareguidelinesapp" title="sec"><span class="label" tagx="label" title="label">3.5</span><div class="title" tagx="title" title="title">
Ebola Care Guidelines App</div> <p id="p0145">We created a multilingual (English and French) Android app for health workers, named <i>Ebola Care Guidelines</i>, that integrates available patient care and management guidelines for EVD patients with the prognostic models described earlier. To adapt to the needs of rural or remote locations with limited internet access, the app only requires internet connectivity to be installed the first time, and all subsequent use can be performed offline. The home screen shows a list of supportive care recommendations for Ebola patients, compiled from Lamontagne et al. <a href="#bb0130">[26]</a>, and care and management guidelines for hemorrhagic fevers from WHO <a href="#bb0135">[27]</a> and MSF <a href="#bb0140">[28]</a> (<a href="#f0020">Fig. 4</a>A). The list is categorized by intervention type and selecting a guideline from this list provides a summary description and specific interventions related to that guideline (<a href="#f0020">Fig. 4</a>B). When users select a specific intervention, the app redirects to the corresponding page in the reference document (<a href="#f0020">Fig. 4</a>C). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 4</span></a></div>   <p>Ebola Care Guidelines app. The home screen presents the list of recommendations (A), which can be selected to access specific interventions associated with each recommendation (B). Selecting a specific intervention or guideline redirects the user to the corresponding section in the WHO's manuals for care and management of hemorrhagic fever patients (C). The app allows the users to enter basic demographic information (age, weight), vitals, signs &amp;amp; symptoms at presentation, lab data (CT value from first RT-PCR and malaria test result), and wellness scale (D). Based on the available data, the app calculates the severity score of the patient using the suitable prognostic model and presents a customized risk visualization (E). The recommendations that are associated with the presentation signs and symptoms are highlighted in the home screen (E).</p>   </div> <p /> <p id="p0150">Users can also input patient information (age, sex, pregnancy status, weight, height), clinical signs and symptoms recorded at triage, laboratory results, and the wellness scale (WS) from the first clinical rounds after admission (<a href="#f0020">Fig. 4</a>D), through a built-in form, or via a separate CommCare (<a href="https://dimagi.com/commcare/">https://dimagi.com/commcare/</a>) app. After all, or some, of this data is recorded, the app computes the severity score of the patient by selecting the model most appropriate for the available data. It then offers a visualization of the score and the magnitude of patient-specific contributions for each feature included in their score, where the score value is shown at the top in a color-graded scale and patient-specific feature contributions to that score are depicted in a bar chart summary page (<a href="#f0020">Fig. 4</a>E). Each clinical feature is linked to one or more of the care guidelines according to their appearance in the guideline, so that when that feature is present in the data, the corresponding guidelines are highlighted (<a href="#f0020">Fig. 4</a>F). The total severity score can also be linked to specific guidelines when it is over a threshold defined in the app's settings. Those guidelines will also be highlighted when the score is higher than a user-defined threshold. This feature is designed to bring the user's attention to the resource-consuming recommendations that could be most relevant given the clinical manifestation of the patient (e.g. intensive monitoring). It is important to note that the app does not make any choices for the user nor does it deviate or omit information from the guidelines, it simply directs the user to the section in the guidelines that advises them in that decision. The trigger signs/symptoms linking to the guidelines are straightforward, since are precisely those listed in the guidelines themselves (e.g. vomiting and diarrhea for the ORS/IV fluids). The app's source code is available on GitHub under the MIT license. An online wiki provides detailed instructions on how to update the models and the guidelines included in the app (<a href="https://github.com/broadinstitute/ebola-imc-public/wiki">https://github.com/broadinstitute/ebola-imc-public/wiki</a>).</p> </div> </div> <div class="discussion" title="sec"><span class="label" tagx="label" title="label">4</span><div class="title" tagx="title" title="title">
Discussion</div> <p id="p0155">The purpose of this study was two-fold: first, to present generalizable machine learning prognostic models for EVD derived from the largest multi-center clinical dataset available to date, externally validated across diverse sites representing various periods of the largest Ebola epidemic on record; and second, to show how these models could guide clinical decisions by organizing existing knowledge of patient care and management more efficiently and making it easily available as a mobile app.</p> <div class="prognosticmodelsrecapitulatepriorfindingsandhighlighttheimportanceofclinicalintuition" title="sec"><span class="label" tagx="label" title="label">4.1</span><div class="title" tagx="title" title="title">
Prognostic Models Recapitulate Prior Findings and Highlight the Importance of Clinical Intuition</div> <p id="p0160">In order to account for different levels of clinical detail collected at the ETUs, we constructed a family of prognostic models that range from models requiring only clinical signs/symptoms or age and viral load, to more complex models incorporating a mixture of laboratory data, signs/symptoms, and observational assessments from experienced health providers. The discriminative capacity of the models is robust across the training set and two independent validation sets, with AUCs ranging from 0.75 up to 0.80. Furthermore, these models recapitulate several findings reported earlier in the literature and also reveal further associations between mortality and clinical signs/symptoms. The most informative predictors of EVD outcome are patient age and viral load, and models incorporating just these two variables exhibit good performance. Occurrence of jaundice or bleeding at initial presentation are important predictors of death, but both have low incidence at triage among the patients in the IMC cohort of only 5%. In contrast, more widespread EVD manifestations such as dyspnea, dysphagia, and asthenia at triage have a much weaker correlation with mortality. Even though fever is a non-specific symptom with no predictive power, body temperature at triage is informative by increasing specificity in the predictions. The performance of models incorporating the clinical wellness score is at least comparable or superior to the more detailed models including individual clinical features deemed as the most predictive in our variable selection process. This is a particularly interesting result in our study, since it suggests that machine learning approaches, when properly designed and implemented, and applied on rich-enough data, could approximate the clinical intuition that physicians acquire through their experience in the field. Conversely, this also provides evidence that good bedside intuition can very effectively integrate the individual indicators of the patient's clinical status into an overall health assessment of great predictive power. These models may be useful in emergency situations when the appropriate experience is unavailable or under-developed.</p> </div> <div class="updatingthemodelsandapp" title="sec"><span class="label" tagx="label" title="label">4.2</span><div class="title" tagx="title" title="title">
Updating the Models and App</div> <p id="p0165">Our prognostic models and guidelines app can be easily updated with new data. With increasing knowledge on the management of Ebola, we expect the mortality rate to decrease over time or to be more reliably estimated by earlier symptoms. New information from the ongoing outbreak in DRC or future epidemics would enable us to train and update the models to more appropriate calculations that are adapted to improving standards of care, and geographically divergent variables. For example: health-care seeking behavior in the conflict-afflicted DRC would be undoubtedly more limited than that in West Africa in 2014–16, and perhaps other factors will be more predictive of death. Therefore, not only are the models adapted to the newly available information but the new models would also automatically link up to the easily-updatable guidelines as they become available.</p> </div> <div class="data-harmonizationchallenges" title="sec"><span class="label" tagx="label" title="label">4.3</span><div class="title" tagx="title" title="title">
Data-harmonization Challenges</div> <p id="p0170">When aggregating data from several sources, care must be taken to search and correct for systematic differences between datasets and best ensure interoperability. For example, in order to aggregate CT values and control for analytical bias generated by the different PCR labs, we applied an intra-site normalization to ensure that values were comparable across sites. We had to discard several potentially informative clinical variables such as confusion and coma since they were not available in all the IMC ETUs. Other features could be imputed, such as body temperature by using fever as a proxy variable.</p> </div> <div class="validationstrategies" title="sec"><span class="label" tagx="label" title="label">4.4</span><div class="title" tagx="title" title="title">
Validation Strategies</div> <p id="p0175">Our prognostic models, particularly those incorporating the parsimonious sets of predictors, perform well on two independent datasets used for external validation. These datasets have a wide temporal, geographic and clinical scope. A major difference between these datasets was the time during which they were collected, with the KGH data representing an earlier time point, with less refined treatment protocols, increased patient volume and admission intensity with a larger number of patients delayed during transfers from holding centers. On the other hand, the GOAL dataset includes patients from the final months of the epidemic with a 13% lower CFR. Thus, as may be expected, the models underestimated the observed risk for patients of the KGH cohort, while observed risk was slightly overestimated in the GOAL cohort. The IMC training dataset covers a much broader temporal window of the epidemic as well as a wider catchment area, spanning several districts across two countries, which may explain its consistent performance in these disparate populations. We also investigated the temporal flexibility of the models by testing them on CT values obtained two days after triage and comparing model performance on patients with short or long times to presentation at the ETU. The results show that the parsimonious models could maintain their performance parameters when deployed in contexts with systematically different referral times or laboratory turnaround times, and further justifies the use of time-to-presentation within the model to control for time-dependent effects.</p> </div> <div class="mhealthapplications" title="sec"><span class="label" tagx="label" title="label">4.5</span><div class="title" tagx="title" title="title">
mHealth Applications</div> <p id="p0180">Finally, with the <i>Ebola Care Guidelines</i> app we benchmark a robust and flexible mHealth platform for clinicians that could be extended to other diseases affecting rural and low-resource areas. By linking evidence-based prognostication with trusted clinical care information, we propose the app both as a reference tool to improve training and adherence to protocol, as well as a support system that better tailors the prioritization of clinical interventions to the patient's data. The integration of mHealth platforms with rapid point of care diagnostic kits <a href="#bb0145">[29]</a>, <a href="#bb0150">[30]</a> has the potential to realize the concept of a "pocket lab�? <a href="#bb0155">[31]</a>, which could be used outside laboratory settings and during health emergencies. The ultimate goal of these platforms is to aid clinical management decisions by personalizing the understanding of individual prognostic predictions and better organizing access to the trusted, updateable medical knowledge. This platform also allows updating of the prognostic models, where new weightings may be trained on more geographically or temporally relevant data as it becomes available. The current update process simply involves bundling the specifications of the new models (predictor variables and coefficients), rebuilding the app, and releasing it through Android's app store. Future versions of the app may not even require to be updated, as they could retrieve new models directly from a central server. We believe that if clinical staff can obtain actionable information from the data they collect, they may be incentivized to generate more and higher-quality data: creating a positive feedback loop which drives precision. Further, the visualization of the clinical basis of prediction models (such as the charts provided in our app) provides a learning platform that builds informed clinical experience rather than simply replacing it.</p> </div> <div class="limitations" title="sec"><span class="label" tagx="label" title="label">4.6</span><div class="title" tagx="title" title="title">
Limitations</div> <p id="p0185">Despite being the largest EVD prognosis modeling study to date, the amount and quality of available clinical data is still limited. We accounted for these limitations by harmonizing the data from different ETUs and applying various statistical techniques recommended for prognosis modeling (multiple imputation, bootstrap sampling, external validation). Even with the help of these approaches, data might be affected by variations in clinical assessments from clinicians with varying levels of experience, errors in data collection (including patient symptom recall or history taking skills), and differences in lab protocols. Ultimately, future predictive models will require larger and better datasets, and consensus mechanisms to ensure that data is consistent across sites.</p> </div> </div> <div class="conclusions" title="sec"><span class="label" tagx="label" title="label">5</span><div class="title" tagx="title" title="title">
Conclusions</div> <p id="p0190">It is possible to generate generalizable machine learning prognostic models if data from representative cohorts can be harmonized and externally validated. The use of low-cost mHealth tools on the ground incorporating the insights gained from such models, in combination with effective data collection and sharing among all stakeholders, will be key elements in the early detection and containment of future outbreaks of Ebola and other emerging infectious diseases.</p> </div> <div class="availabilityofsourcecode,data,andapp" title="sec"> <div class="title" tagx="title" title="title">
Availability of source code, data, and app</div> <p id="p0195">The full source code is openly available at <a href="https://github.com/broadinstitute/ebola-imc-public">https://github.com/broadinstitute/ebola-imc-public</a>. Refer to IMC's Ebola Response page (<a href="https://internationalmedicalcorps.org/ebola-response">https://internationalmedicalcorps.org/ebola-response</a>), for instructions on how external researchers can access the data. The app is freely available on Google Play: <a href="https://play.google.com/store/apps/details?id=org.broadinstitute.ebola_care_guidelines">https://play.google.com/store/apps/details?id=org.broadinstitute.ebola_care_guidelines</a>.</p> </div> <div class="contributors" title="sec"> <div class="title" tagx="title" title="title">
Contributors</div> <p id="p0200">Andres Colubri: Project conception, literature search, data analysis, data interpretation, writing, figures, tool development.</p> <p id="p0205">Mary-Anne Hartley: Literature search, data collection, data interpretation, writing, figures.</p> <p id="p0210">Mathew Siakor: Data collection, editing.</p> <p id="p0215">Vanessa Wolfman: Data collection, editing.</p> <p id="p0220">August Felix: Protocol compliance, editing.</p> <p id="p0225">Tom Sesay: Data collection.</p> <p id="p0230">Jeffrey G. Shaffer: Data collection.</p> <p id="p0235">Robert F. Garry: Data collection.</p> <p id="p0240">Donald S. Grant: Data collection.</p> <p id="p0245">Adam C. Levine: Project supervision, data collection, data interpretation, writing.</p> <p id="p0250">Pardis C. Sabeti: Project supervision, data interpretation, writing.</p> </div> </div> <div class="back" title="back"> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="bb0005" /><span class="label" tagx="label" title="label">1</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barry</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ahuka-Mundeke</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ali Ahmed</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Allarangar</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anoko</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Archer</span><span class="given-names" tagx="given-names" title="given-names">B.N.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Outbreak of Ebola virus disease in the Democratic Republic of the Congo, April–May, 2018: an epidemiological study</span><span class="source" tagx="source" title="source">The Lancet</span><span class="volume" tagx="volume" title="volume">392</span><span class="issue" tagx="issue" title="issue">10143</span><span class="year" tagx="year" title="year">2018</span><span class="fpage" tagx="fpage" title="fpage">213</span><span class="lpage" tagx="lpage" title="lpage">221</span></span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">.Barry A, Ahuka-Mundeke S, Ali Ahmed Y, Allarangar Y, Anoko J, Archer BN, et al. Outbreak of Ebola virus disease in the Democratic Republic of the Congo, April–May, 2018: an epidemiological study. The Lancet. 2018;392(10143):213-21.</span></li> <li tag="ref"><a name="bb0010" /><span class="label" tagx="label" title="label">2</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moran</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Fighting Ebola in conflict in the DR Congo</span><span class="source" tagx="source" title="source">The Lancet</span><span class="volume" tagx="volume" title="volume">392</span><span class="issue" tagx="issue" title="issue">10155</span><span class="year" tagx="year" title="year">2018</span><span class="fpage" tagx="fpage" title="fpage">1295</span><span class="lpage" tagx="lpage" title="lpage">1296</span></span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Moran B. Fighting Ebola in conflict in the DR Congo. The Lancet. 2018;392(10155):1295-6.</span></li> <li tag="ref"><a name="bb0015" /><span class="label" tagx="label" title="label">3</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fischer</span><span class="given-names" tagx="given-names" title="given-names">W.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crozier</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bausch</span><span class="given-names" tagx="given-names" title="given-names">D.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muyembe</span><span class="given-names" tagx="given-names" title="given-names">J.-J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mulangu</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diaz</span><span class="given-names" tagx="given-names" title="given-names">J.V.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Shifting the paradigm — applying universal standards of care to Ebola virus disease</span><span class="source" tagx="source" title="source">N Engl J Med</span><span class="volume" tagx="volume" title="volume">380</span><span class="issue" tagx="issue" title="issue">15</span><span class="year" tagx="year" title="year">2019</span><span class="fpage" tagx="fpage" title="fpage">1389</span><span class="lpage" tagx="lpage" title="lpage">1391</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30970184">30970184</a></span></span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Fischer WA, Crozier I, Bausch DG, Muyembe J-J, Mulangu S, Diaz JV, et al. Shifting the Paradigm — Applying Universal Standards of Care to Ebola Virus Disease. New England Journal of Medicine. 2019;380(15):1389-91.</span></li> <li tag="ref"><a name="bb0020" /><span class="label" tagx="label" title="label">4</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Amarasingham</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patel</span><span class="given-names" tagx="given-names" title="given-names">P.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Toto</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nelson</span><span class="given-names" tagx="given-names" title="given-names">L.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Swanson</span><span class="given-names" tagx="given-names" title="given-names">T.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moore</span><span class="given-names" tagx="given-names" title="given-names">B.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Allocating scarce resources in real-time to reduce heart failure readmissions: a prospective, controlled study</span><span class="source" tagx="source" title="source">BMJ Quality &amp;amp; Safety</span><span class="volume" tagx="volume" title="volume">22</span><span class="issue" tagx="issue" title="issue">12</span><span class="year" tagx="year" title="year">2013</span><span class="fpage" tagx="fpage" title="fpage">998</span><span class="lpage" tagx="lpage" title="lpage">1005</span></span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Amarasingham R, Patel PC, Toto K, Nelson LL, Swanson TS, Moore BJ, et al. Allocating scarce resources in real-time to reduce heart failure readmissions: a prospective, controlled study. BMJ Quality &amp;amp; Safety. 2013;22(12):998-1005.</span></li> <li tag="ref"><a name="bb0025" /><span class="label" tagx="label" title="label">5</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mitchell</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hedt-Gauthier</span><span class="given-names" tagx="given-names" title="given-names">B.L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Msellemu</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nkaka</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lesh</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Using electronic technology to improve clinical care – results from a before-after cluster trial to evaluate assessment and classification of sick children according to Integrated Management of Childhood Illness (IMCI) protocol in Tanzania</span><span class="source" tagx="source" title="source">BMC Med Inform Decis Mak</span><span class="volume" tagx="volume" title="volume">13</span><span class="issue" tagx="issue" title="issue">1</span><span class="year" tagx="year" title="year">2013</span><span class="fpage" tagx="fpage" title="fpage">95</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23981292">23981292</a></span></span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Mitchell M, Hedt-Gauthier BL, Msellemu D, Nkaka M, Lesh N. Using electronic technology to improve clinical care – results from a before-after cluster trial to evaluate assessment and classification of sick children according to Integrated Management of Childhood Illness (IMCI) protocol in Tanzania. BMC Medical Informatics and Decision Making. 2013;13(1):95.</span></li> <li tag="ref"><a name="bb0030" /><span class="label" tagx="label" title="label">6</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lund</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boas</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bedesa</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fekede</span><span class="given-names" tagx="given-names" title="given-names">W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nielsen</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sørensen</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Association between the safe delivery app and quality of care and perinatal survival in Ethiopia: a randomized clinical trial</span><span class="source" tagx="source" title="source">JAMA Pediatr</span><span class="volume" tagx="volume" title="volume">170</span><span class="issue" tagx="issue" title="issue">8</span><span class="year" tagx="year" title="year">2016</span><span class="fpage" tagx="fpage" title="fpage">765</span><span class="lpage" tagx="lpage" title="lpage">771</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27322089">27322089</a></span></span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Lund S, Boas I, Bedesa T, Fekede W, Nielsen H, Sørensen B. Association between the safe delivery app and quality of care and perinatal survival in ethiopia: A randomized clinical trial. JAMA Pediatrics. 2016;170(8):765-71.</span></li> <li tag="ref"><a name="bb0035" /><span class="label" tagx="label" title="label">7</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colubri</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Silver</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fradet</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Retzepi</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fry</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sabeti</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Transforming clinical data into actionable prognosis models: machine-learning framework and field-deployable app to predict outcome of Ebola patients</span><span class="source" tagx="source" title="source">PLoS Negl Trop Dis</span><span class="volume" tagx="volume" title="volume">10</span><span class="issue" tagx="issue" title="issue">3</span><span class="year" tagx="year" title="year">2016</span>e0004549</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Colubri A, Silver T, Fradet T, Retzepi K, Fry B, Sabeti P. Transforming Clinical Data into Actionable Prognosis Models: Machine-Learning Framework and Field-Deployable App to Predict Outcome of Ebola Patients. PLoS neglected tropical diseases. 2016;10(3):e0004549.</span></li> <li tag="ref"><a name="bb0040" /><span class="label" tagx="label" title="label">8</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schieffelin</span><span class="given-names" tagx="given-names" title="given-names">J.S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shaffer</span><span class="given-names" tagx="given-names" title="given-names">J.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goba</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gbakie</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gire</span><span class="given-names" tagx="given-names" title="given-names">S.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colubri</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Clinical illness and outcomes in patients with Ebola in Sierra Leone</span><span class="source" tagx="source" title="source">N Engl J Med</span><span class="volume" tagx="volume" title="volume">371</span><span class="issue" tagx="issue" title="issue">22</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">2092</span><span class="lpage" tagx="lpage" title="lpage">2100</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25353969">25353969</a></span></span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Schieffelin JS, Shaffer JG, Goba A, Gbakie M, Gire SK, Colubri A, et al. Clinical illness and outcomes in patients with Ebola in Sierra Leone. N Engl J Med. 2014;371(22):2092-100.</span></li> <li tag="ref"><a name="bb0045" /><span class="label" tagx="label" title="label">9</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roshania</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mallow</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dunbar</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mansary</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shetty</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lyon</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Successful implementation of a multicountry clinical surveillance and data collection system for Ebola virus disease in West Africa: findings and lessons learned</span><span class="source" tagx="source" title="source">Global Health: Science and Practice</span><span class="volume" tagx="volume" title="volume">4</span><span class="issue" tagx="issue" title="issue">3</span><span class="year" tagx="year" title="year">2016</span><span class="fpage" tagx="fpage" title="fpage">394</span><span class="lpage" tagx="lpage" title="lpage">409</span></span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Roshania R, Mallow M, Dunbar N, Mansary D, Shetty P, Lyon T, et al. Successful Implementation of a Multicountry Clinical Surveillance and Data Collection System for Ebola Virus Disease in West Africa: Findings and Lessons Learned. Global Health: Science and Practice. 2016;4(3):394-409.</span></li> <li tag="ref"><a name="bb0050" /><span class="label" tagx="label" title="label">10</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hartley</span><span class="given-names" tagx="given-names" title="given-names">M.-A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Young</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tran</span><span class="given-names" tagx="given-names" title="given-names">A.-M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Okoni-Williams</span><span class="given-names" tagx="given-names" title="given-names">H.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suma</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mancuso</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Predicting Ebola severity: a clinical prioritization score for Ebola virus disease</span><span class="source" tagx="source" title="source">PLoS Negl Trop Dis</span><span class="volume" tagx="volume" title="volume">11</span><span class="issue" tagx="issue" title="issue">2</span><span class="year" tagx="year" title="year">2017</span>e0005265</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Hartley M-A, Young A, Tran A-M, Okoni-Williams HH, Suma M, Mancuso B, et al. Predicting Ebola Severity: A Clinical Prioritization Score for Ebola Virus Disease. PLoS neglected tropical diseases. 2017;11(2):e0005265.</span></li> <li tag="ref"><a name="bb0055" /><span class="label" tagx="label" title="label">11</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hartley</span><span class="given-names" tagx="given-names" title="given-names">M.-A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Young</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tran</span><span class="given-names" tagx="given-names" title="given-names">A.-M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Okoni-Williams</span><span class="given-names" tagx="given-names" title="given-names">H.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suma</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mancuso</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Predicting Ebola infection: a malaria-sensitive triage score for Ebola virus disease</span><span class="source" tagx="source" title="source">PLoS Negl Trop Dis</span><span class="volume" tagx="volume" title="volume">11</span><span class="issue" tagx="issue" title="issue">2</span><span class="year" tagx="year" title="year">2017</span>e0005356</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Hartley M-A, Young A, Tran A-M, Okoni-Williams HH, Suma M, Mancuso B, et al. Predicting Ebola infection: A malaria-sensitive triage score for Ebola virus disease. PLoS neglected tropical diseases. 2017;11(2):e0005356.</span></li> <li tag="ref"><a name="bb0060" /><span class="label" tagx="label" title="label">12</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gire</span><span class="given-names" tagx="given-names" title="given-names">S.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goba</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andersen</span><span class="given-names" tagx="given-names" title="given-names">K.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sealfon</span><span class="given-names" tagx="given-names" title="given-names">R.S.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">D.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kanneh</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genomic surveillance elucidates Ebola virus origin and transmission during the 2014 outbreak</span><span class="source" tagx="source" title="source">Science (New York, NY)</span><span class="volume" tagx="volume" title="volume">345</span><span class="issue" tagx="issue" title="issue">6202</span><span class="year" tagx="year" title="year">2014</span><span class="fpage" tagx="fpage" title="fpage">1369</span><span class="lpage" tagx="lpage" title="lpage">1372</span></span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">.Gire SK, Goba A, Andersen KG, Sealfon RSG, Park DJ, Kanneh L, et al. Genomic surveillance elucidates Ebola virus origin and transmission during the 2014 outbreak. Science (New York, NY). 2014;345(6202):1369-72.</span></li> <li tag="ref"><a name="bb0065" /><span class="label" tagx="label" title="label">13</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Center USNMR</span></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Ebola Zaire (EZ1) rRT-PCR (TaqMan) assay</span><span class="year" tagx="year" title="year">2014</span></span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Center USNMR. Ebola Zaire (EZ1) rRT-PCR (TaqMan) Assay. 2014.</span></li> <li tag="ref"><a name="bb0070" /><span class="label" tagx="label" title="label">14</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Trombley</span><span class="given-names" tagx="given-names" title="given-names">A.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wachter</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garrison</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buckley-Beason</span><span class="given-names" tagx="given-names" title="given-names">V.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jahrling</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hensley</span><span class="given-names" tagx="given-names" title="given-names">L.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Comprehensive panel of real-time TaqMan™ polymerase chain reaction assays for detection and absolute quantification of filoviruses, arenaviruses, and New World hantaviruses</span><span class="source" tagx="source" title="source">Am J Trop Med Hyg</span><span class="volume" tagx="volume" title="volume">82</span><span class="issue" tagx="issue" title="issue">5</span><span class="year" tagx="year" title="year">2010</span><span class="fpage" tagx="fpage" title="fpage">954</span><span class="lpage" tagx="lpage" title="lpage">960</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20439981">20439981</a></span></span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">.Trombley AR, Wachter L, Garrison J, Buckley-Beason VA, Jahrling J, Hensley LE, et al. Comprehensive Panel of Real-Time TaqMan™ Polymerase Chain Reaction Assays for Detection and Absolute Quantification of Filoviruses, Arenaviruses, and New World Hantaviruses. The American journal of tropical medicine and hygiene. 2010;82(5):954-60.</span></li> <li tag="ref"><a name="bb0075" /><span class="label" tagx="label" title="label">15</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prevention CfDCa</span></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Ebola Virus VP40 Real-Time RT-PCR Assay</span><span class="year" tagx="year" title="year">2014</span></span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Prevention CfDCa. Ebola Virus VP40 Real-Time RT-PCR Assay. 2014.</span></li> <li tag="ref"><a name="bb0080" /><span class="label" tagx="label" title="label">16</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Janssen</span><span class="given-names" tagx="given-names" title="given-names">K.J.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Donders</span><span class="given-names" tagx="given-names" title="given-names">A.R.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harrell</span><span class="given-names" tagx="given-names" title="given-names">F.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vergouwe</span><span class="given-names" tagx="given-names" title="given-names">Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Q.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grobbee</span><span class="given-names" tagx="given-names" title="given-names">D.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Missing covariate data in medical research: to impute is better than to ignore</span><span class="source" tagx="source" title="source">J Clin Epidemiol</span><span class="volume" tagx="volume" title="volume">63</span><span class="issue" tagx="issue" title="issue">7</span><span class="year" tagx="year" title="year">2010</span><span class="fpage" tagx="fpage" title="fpage">721</span><span class="lpage" tagx="lpage" title="lpage">727</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20338724">20338724</a></span></span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Janssen KJM, Donders ART, Harrell FE, Vergouwe Y, Chen Q, Grobbee DE, et al. Missing covariate data in medical research: To impute is better than to ignore. Journal of Clinical Epidemiology. 2010;63(7):721-7.</span></li> <li tag="ref"><a name="bb0085" /><span class="label" tagx="label" title="label">17</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beaujean</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">BaylorEdPsych: R package for Baylor University Educational Psychology quantitative courses. R package version 0.5</span><span class="year" tagx="year" title="year">2012</span></span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">.Beaujean A. BaylorEdPsych: R Package for Baylor University Educational Psychology Quantitative Courses. R package version 0.5. 2012.</span></li> <li tag="ref"><a name="bb0090" /><span class="label" tagx="label" title="label">18</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harrell</span><span class="given-names" tagx="given-names" title="given-names">F.E.</span></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Hmisc: Harrell miscellaneous. R package version 4.2</span><span class="year" tagx="year" title="year">2019</span></span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Harrell FE. Hmisc: Harrell Miscellaneous. R package version 4.2. 2019.</span></li> <li tag="ref"><a name="bb0095" /><span class="label" tagx="label" title="label">19</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harrell</span><span class="given-names" tagx="given-names" title="given-names">F.E.</span></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Regression modeling strategies: with applications to linear models, logistic and ordinal regression, and survival analysis</span><span class="edition" tagx="edition" title="edition">2nd ed.</span><span class="volume" tagx="volume" title="volume">vol. xxv</span><span class="year" tagx="year" title="year">2015</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Cham</span><span class="comment">1 online resource. 582 pages</span></span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Harrell FE. Regression modeling strategies: with applications to linear models, logistic and ordinal regression, and survival analysis. Second edition. ed. Cham: Springer; 2015. 1 online resource (xxv, 582 pages) p.</span></li> <li tag="ref"><a name="bb0100" /><span class="label" tagx="label" title="label">20</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Friedman</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hastie</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tibshirani</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simon</span><span class="given-names" tagx="given-names" title="given-names">N.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Narasimhan</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">glmnet</span><span class="given-names" tagx="given-names" title="given-names">Qian J.</span></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Lasso and elastic-net regularized generalized linear models. R package version 2.0</span><span class="year" tagx="year" title="year">2018</span></span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Friedman J, Hastie T, Tibshirani R, Simon N, Narasimhan B, Qian J. glmnet: Lasso and Elastic-Net Regularized Generalized Linear Models. R package version 2.0. 2018.</span></li> <li tag="ref"><a name="bb0105" /><span class="label" tagx="label" title="label">21</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harrell</span><span class="given-names" tagx="given-names" title="given-names">F.E.</span></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">rms: Regression Modeling Strategies</span><span class="year" tagx="year" title="year">2019</span></span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Harrell FE. rms: Regression Modeling Strategies. 2019.</span></li> <li tag="ref"><a name="bb0110" /><span class="label" tagx="label" title="label">22</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saar-Tsechansky</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Provost</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Handling missing values when applying classification models</span><span class="source" tagx="source" title="source">J Mach Learn Res</span><span class="volume" tagx="volume" title="volume">8</span><span class="year" tagx="year" title="year">2007</span><span class="fpage" tagx="fpage" title="fpage">1623</span><span class="lpage" tagx="lpage" title="lpage">1657</span></span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Saar-Tsechansky M, Provost F. Handling Missing Values when Applying Classification Models. J Mach Learn Res. 2007;8:1623-57.</span></li> <li tag="ref"><a name="bb0115" /><span class="label" tagx="label" title="label">23</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harel</span><span class="given-names" tagx="given-names" title="given-names">O.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The estimation of R 2 and adjusted R 2 in incomplete data sets using multiple imputation</span><span class="source" tagx="source" title="source">Journal of Applied Statistics</span><span class="volume" tagx="volume" title="volume">36</span><span class="issue" tagx="issue" title="issue">10</span><span class="year" tagx="year" title="year">2009</span><span class="fpage" tagx="fpage" title="fpage">1109</span><span class="lpage" tagx="lpage" title="lpage">1118</span></span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Harel O. The estimation of R 2 and adjusted R 2 in incomplete data sets using multiple imputation. Journal of Applied Statistics. 2009;36(10):1109-18.</span></li> <li tag="ref"><a name="bb0120" /><span class="label" tagx="label" title="label">24</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">K.F.</span></span></span><span class="mixed-article-title" title="mixed-article-title">What's the relative risk?A method of correcting the odds ratio in cohort studies of common outcomes</span><span class="source" tagx="source" title="source">JAMA</span><span class="volume" tagx="volume" title="volume">280</span><span class="issue" tagx="issue" title="issue">19</span><span class="year" tagx="year" title="year">1998</span><span class="fpage" tagx="fpage" title="fpage">1690</span><span class="lpage" tagx="lpage" title="lpage">1691</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9832001">9832001</a></span></span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Zhang J, Yu KF. What's the Relative Risk?A Method of Correcting the Odds Ratio in Cohort Studies of Common Outcomes. JAMA. 1998;280(19):1690-1.</span></li> <li tag="ref"><a name="bb0125" /><span class="label" tagx="label" title="label">25</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crowe</span><span class="given-names" tagx="given-names" title="given-names">S.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maenner</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuah</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Erickson</span><span class="given-names" tagx="given-names" title="given-names">B.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coffee</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knust</span><span class="given-names" tagx="given-names" title="given-names">B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Prognostic indicators for Ebola patient survival</span><span class="source" tagx="source" title="source">Emerging Infectious Disease journal</span><span class="volume" tagx="volume" title="volume">22</span><span class="issue" tagx="issue" title="issue">2</span><span class="year" tagx="year" title="year">2016</span><span class="fpage" tagx="fpage" title="fpage">217</span></span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Crowe SJ, Maenner MJ, Kuah S, Erickson BR, Coffee M, Knust B, et al. Prognostic Indicators for Ebola Patient Survival. Emerging Infectious Disease journal. 2016;22(2):217.</span></li> <li tag="ref"><a name="bb0130" /><span class="label" tagx="label" title="label">26</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lamontagne</span><span class="given-names" tagx="given-names" title="given-names">F.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fowler</span><span class="given-names" tagx="given-names" title="given-names">R.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adhikari</span><span class="given-names" tagx="given-names" title="given-names">N.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murthy</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brett-Major</span><span class="given-names" tagx="given-names" title="given-names">D.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jacobs</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Evidence-based guidelines for supportive care of patients with Ebola virus disease</span><span class="source" tagx="source" title="source">The Lancet</span><span class="volume" tagx="volume" title="volume">391</span><span class="issue" tagx="issue" title="issue">10121</span><span class="year" tagx="year" title="year">2018</span><span class="fpage" tagx="fpage" title="fpage">700</span><span class="lpage" tagx="lpage" title="lpage">708</span></span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Lamontagne F, Fowler RA, Adhikari NK, Murthy S, Brett-Major DM, Jacobs M, et al. Evidence-based guidelines for supportive care of patients with Ebola virus disease. The Lancet. 2018;391(10121):700-8.</span></li> <li tag="ref"><a name="bb0135" /><span class="label" tagx="label" title="label">27</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">WHO</span></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Manual for the care and management of patients in Ebola care units/community care Centres: Interim emergency guidance, January 2015</span><span class="year" tagx="year" title="year">2015</span></span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">WHO. Manual for the care and management of patients in Ebola care units/Community Care Centres: interim emergency guidance, January 2015. 2015.</span></li> <li tag="ref"><a name="bb0140" /><span class="label" tagx="label" title="label">28</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">WHO</span></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Clinical management of patients with viral haemorrhagic fever</span><span class="source" tagx="source" title="source">A pocket guide for front-line health workers</span><span class="year" tagx="year" title="year">2016</span></span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">.WHO. Clinical management of patients with viral haemorrhagic fever. A pocket guide for front-line health workers. 2016.</span></li> <li tag="ref"><a name="bb0145" /><span class="label" tagx="label" title="label">29</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Broadhurst MJ, Kelly JD, Miller A, Semper A, Bailey D, Groppelli E, et al. ReEBOV antigen rapid test kit for point-of-care and laboratory-based testing for Ebola virus disease: a field validation study. The Lancet.386(9996):867–74.</span></li> <li tag="ref"><a name="bb0150" /><span class="label" tagx="label" title="label">30</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yen</span><span class="given-names" tagx="given-names" title="given-names">C.-W.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Puig</span><span class="given-names" tagx="given-names" title="given-names">H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tam</span><span class="given-names" tagx="given-names" title="given-names">J.O.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gomez-Marquez</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bosch</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hamad-Schifferli</span><span class="given-names" tagx="given-names" title="given-names">K.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Multicolored silver nanoparticles for multiplexed disease diagnostics: distinguishing dengue, yellow fever, and Ebola viruses</span><span class="source" tagx="source" title="source">Lab Chip</span><span class="volume" tagx="volume" title="volume">15</span><span class="issue" tagx="issue" title="issue">7</span><span class="year" tagx="year" title="year">2015</span><span class="fpage" tagx="fpage" title="fpage">1638</span><span class="lpage" tagx="lpage" title="lpage">1641</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25672590">25672590</a></span></span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Yen C-W, de Puig H, Tam JO, Gomez-Marquez J, Bosch I, Hamad-Schifferli K, et al. Multicolored silver nanoparticles for multiplexed disease diagnostics: distinguishing dengue, yellow fever, and Ebola viruses. Lab on a Chip. 2015;15(7):1638-41.</span></li> <li tag="ref"><a name="bb0155" /><span class="label" tagx="label" title="label">31</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Radin</span><span class="given-names" tagx="given-names" title="given-names">J.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Topol</span><span class="given-names" tagx="given-names" title="given-names">E.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Andersen</span><span class="given-names" tagx="given-names" title="given-names">K.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steinhubl</span><span class="given-names" tagx="given-names" title="given-names">S.R.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A laboratory in your pocket</span><span class="source" tagx="source" title="source">The Lancet</span><span class="volume" tagx="volume" title="volume">388</span><span class="issue" tagx="issue" title="issue">10054</span><span class="year" tagx="year" title="year">1875</span></span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Radin JM, Topol EJ, Andersen KG, Steinhubl SR. A laboratory in your pocket. The Lancet.388(10054):1875.</span></li> </ul> </div> <div class="supplementary-material" title="supplementary-material"><span class="label" tagx="label" title="label">Appendix A</span><div class="title" tagx="title" title="title">
Supplementary data</div> <p id="p0260"> </p><div class="local-data" title="local-data">  <p>Supplementary material</p>  <div class="media" title="media"><a href="mmc1.docx">LINK</a>[alt-text] Image 1</div> </div> <p /> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>We would like to thank the governments of Liberia, Sierra Leone, and Guinea for contributing to the International Medical Corps' humanitarian response. We would also like to thank all of our generous institutional, corporate, foundation, and individual donors who placed their confidence and trust in International Medical Corps and made our work during the Ebola epidemic possible. We would also like to thank the United States Naval Medical Research Center, Public Health England, the European Union Mobile Laboratory, and the Nigerian Laboratory for providing laboratory data to our Ebola Treatment Units. We would like to further acknowledge all members of our Research Review Committee and other technical teams that contributed to this research. Finally, we would also like to thank our clinical, WASH, and psychosocial teams as well as all of our monitoring and evaluation staff, including the data collection officers at each of our ETUs. AC would like to thank Mary Lynn Baniecki and Christian Matranga for insightful discussions on the EBOV qPCR assays, and Christopher Moxon for critical feedback on the manuscript. Finally, we thank the patients included in this study whose data has and will continue to make invaluable contribution to improving future Ebola care. PCS, AF, and AC received support from </p><div class="funding-source" title="funding-source">
Howard Hughes Medical Institute</div>, AC was supported by NI grants R01AI114855, U01HG007480, and U19AI110818, and <div class="funding-source" title="funding-source">
Bill &amp;amp; Melinda Gates Foundation</div> grant OPP1195122. ACL, MS, and VW were fully supported by IMC for this study. MAH and TS were supported by GOAL Global and U.K. Department for International Development grant 04890. RFG was supported by <div class="funding-source" title="funding-source">
NIH</div> grant U19AI115589. JGS was supported by <div class="funding-source" title="funding-source">
NIH</div> grant U19AI135995. DSG was supported by <div class="funding-source" title="funding-source">
NIH</div> contract number HHSN272201400048C.<p /> <div class="conflictofintereststatements" title="sec"> <div class="title" tagx="title" title="title">
Conflict of interest statements</div> <p id="p3165">RFG discloses conflicts of interest from receiving NIH grants and from the development of medical countermeasures by members of the Viral Hemorrhagic Fever Consortium that may result in royalties. The rest of the authors declare that there are no conflicts of interest associated with this research manuscript.</p> </div> <div class="roleoffundingsource" title="sec"> <div class="title" tagx="title" title="title">
Role of funding source</div> <p id="p2465">The funders of the research had no role in study design, data collection, data analysis, data interpretation, or writing of the manuscript. The corresponding authors had full access to all the data in the study and had final responsibility for the decision to submit for publication.</p> </div> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-supplementary-material" title="supplementary-material"><span class="label" tagx="label" title="label">Appendix A</span><p id="p0265">Supplementary data to this article can be found online at <a href="https://doi.org/10.1016/j.eclinm.2019.06.003">https://doi.org/10.1016/j.eclinm.2019.06.003</a>.</p> </div> </div> </div>  </body></html>