<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/embed/graphic-1.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2018/04/06/294488.atom"/>
  <meta name="HW.pisa" content="biorxiv;294488v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Isolation and sequencing of AGO-bound RNAs reveals characteristics of mammalian stem-loop processing in vivo"/>
  <meta name="DC.Identifier" content="10.1101/294488"/>
  <meta name="DC.Date" content="2018-04-06"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2018, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="MicroRNA precursors (pre-miRNAs) are short hairpin RNAs that are rapidly processed into mature microRNAs (miRNAs) in the cytoplasm. Due to their low abundance in cells, sequencing-based studies of pre-miRNAs have been limited. We successfully enriched for and deep sequenced pre-miRNAs in human cells by capturing these RNAs during their interaction with Argonaute (AGO) proteins. Using this approach, we detected &gt;; 350 pre-miRNAs in human cells and &gt;; 250 pre-miRNAs in a reanalysis of a similar study in mouse cells. We uncovered widespread trimming and non-templated additions to the 3’ ends of pre- and mature miRNAs. Additionally, we created an index for microRNA precursor processing efficiency. This analysis revealed a subset of pre-miRNAs that produce low levels of mature miRNAs despite abundant precursors, including an annotated miRNA in the 5’ UTR of the DiGeorge syndrome critical region 8 mRNA transcript. This led us to search for AGO-associated stem-loops originating from other mRNA species, which identified hundreds of putative pre-miRNAs derived from human and mouse mRNAs. In summary, we provide a wealth of information on mammalian pre-miRNAs, and identify novel microRNA and microRNA-like elements localized in mRNAs."/>
  <meta name="DC.Contributor" content="Ian M. Silverman"/>
  <meta name="DC.Contributor" content="Sager J. Gosai"/>
  <meta name="DC.Contributor" content="Nicholas Vrettos"/>
  <meta name="DC.Contributor" content="Shawn W. Foley"/>
  <meta name="DC.Contributor" content="Nathan D. Berkowitz"/>
  <meta name="DC.Contributor" content="Zissimos Mourelatos"/>
  <meta name="DC.Contributor" content="Brian D. Gregory"/>
  <meta name="article:published_time" content="2018-04-06"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Isolation and sequencing of AGO-bound RNAs reveals characteristics of mammalian stem-loop processing in vivo"/>
  <meta name="citation_abstract" lang="en" content="&lt;;h3&gt;;ABSTRACT&lt;;/h3&gt;; &lt;;p&gt;;MicroRNA precursors (pre-miRNAs) are short hairpin RNAs that are rapidly processed into mature microRNAs (miRNAs) in the cytoplasm. Due to their low abundance in cells, sequencing-based studies of pre-miRNAs have been limited. We successfully enriched for and deep sequenced pre-miRNAs in human cells by capturing these RNAs during their interaction with Argonaute (AGO) proteins. Using this approach, we detected &amp;;gt; 350 pre-miRNAs in human cells and &amp;;gt; 250 pre-miRNAs in a reanalysis of a similar study in mouse cells. We uncovered widespread trimming and non-templated additions to the 3’ ends of pre- and mature miRNAs. Additionally, we created an index for microRNA precursor processing efficiency. This analysis revealed a subset of pre-miRNAs that produce low levels of mature miRNAs despite abundant precursors, including an annotated miRNA in the 5’ UTR of the DiGeorge syndrome critical region 8 mRNA transcript. This led us to search for AGO-associated stem-loops originating from other mRNA species, which identified hundreds of putative pre-miRNAs derived from human and mouse mRNAs. In summary, we provide a wealth of information on mammalian pre-miRNAs, and identify novel microRNA and microRNA-like elements localized in mRNAs.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2018/01/01"/>
  <meta name="citation_mjid" content="biorxiv;294488v1"/>
  <meta name="citation_id" content="294488v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/294488v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/294488v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/294488v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488.full.pdf"/>
  <meta name="citation_doi" content="10.1101/294488"/>
  <meta name="citation_num_pages" content="56"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="294488"/>
  <meta name="citation_author" content="Ian M. Silverman"/>
  <meta name="citation_author_institution" content="Department of Biology, University of Pennsylvania, Philadelphia"/>
  <meta name="citation_author_institution" content="Cell and Molecular Biology Graduate Group, Perelman School of Medicine, University of Pennsylvania"/>
  <meta name="citation_author" content="Sager J. Gosai"/>
  <meta name="citation_author_institution" content="Department of Biology, University of Pennsylvania, Philadelphia"/>
  <meta name="citation_author" content="Nicholas Vrettos"/>
  <meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, Division of Neuropathology, Perelman School of Medicine, University of Pennsylvania"/>
  <meta name="citation_author" content="Shawn W. Foley"/>
  <meta name="citation_author_institution" content="Department of Biology, University of Pennsylvania, Philadelphia"/>
  <meta name="citation_author_institution" content="Cell and Molecular Biology Graduate Group, Perelman School of Medicine, University of Pennsylvania"/>
  <meta name="citation_author" content="Nathan D. Berkowitz"/>
  <meta name="citation_author_institution" content="Department of Biology, University of Pennsylvania, Philadelphia"/>
  <meta name="citation_author_institution" content="Genomics and Computational Biology Graduate Group, Perelman School of Medicine, University of Pennsylvania"/>
  <meta name="citation_author" content="Zissimos Mourelatos"/>
  <meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, Division of Neuropathology, Perelman School of Medicine, University of Pennsylvania"/>
  <meta name="citation_author" content="Brian D. Gregory"/>
  <meta name="citation_author_institution" content="Department of Biology, University of Pennsylvania, Philadelphia"/>
  <meta name="citation_author_institution" content="Cell and Molecular Biology Graduate Group, Perelman School of Medicine, University of Pennsylvania"/>
  <meta name="citation_author_institution" content="Genomics and Computational Biology Graduate Group, Perelman School of Medicine, University of Pennsylvania"/>
  <meta name="citation_author_email" content="bdgregor@sas.upenn.edu"/>
  <meta name="citation_reference" content="Humphreys, D.T., Westman, B.J., Martin, D.I. and Preiss, T. (2005) MicroRNAs control translation initiation by inhibiting eukaryotic initiation factor 4E/cap and poly(A) tail function. Proc Natl Acad Sci U S A, 102, 16961–16966."/>
  <meta name="citation_reference" content="Djuranovic, S., Nahvi, A. and Green, R. (2012) miRNA-mediated gene silencing by translational repression followed by mRNA deadenylation and decay. Science, 336, 237–240."/>
  <meta name="citation_reference" content="Gregory, R.I., Yan, K.P., Amuthan, G., Chendrimada, T., Doratotaj, B., Cooch, N. and Shiekhattar, R. (2004) The Microprocessor complex mediates the genesis of microRNAs. Nature, 432, 235–240."/>
  <meta name="citation_reference" content="Lee, Y., Ahn, C., Han, J., Choi, H., Kim, J., Yim, J., Lee, J., Provost, P., Radmark, O., Kim, S. et al. (2003) The nuclear RNase III Drosha initiates microRNA processing. Nature, 425, 415–419."/>
  <meta name="citation_reference" content="Okamura, K., Hagen, J.W., Duan, H., Tyler, D.M. and Lai, E.C. (2007) The mirtron pathway generates microRNA-class regulatory RNAs in Drosophila. Cell, 130, 89–100."/>
  <meta name="citation_reference" content="Ruby, J.G., Jan, C.H. and Bartel, D.P. (2007) Intronic microRNA precursors that bypass Drosha processing. Nature, 448, 83–86."/>
  <meta name="citation_reference" content="Yi, R., Qin, Y., Macara, I.G. and Cullen, B.R. (2003) Exportin-5 mediates the nuclear export of pre-microRNAs and short hairpin RNAs. Genes &amp;; development, 17, 3011–3016."/>
  <meta name="citation_reference" content="Lund, E., Guttinger, S., Calado, A., Dahlberg, J.E. and Kutay, U. (2004) Nuclear export of microRNA precursors. Science, 303, 95–98."/>
  <meta name="citation_reference" content="Chendrimada, T.P., Gregory, R.I., Kumaraswamy, E., Norman, J., Cooch, N., Nishikura, K. and Shiekhattar, R. (2005) TRBP recruits the Dicer complex to Ago2 for microRNA processing and gene silencing. Nature, 436, 740–744."/>
  <meta name="citation_reference" content="Gregory, R.I., Chendrimada, T.P., Cooch, N. and Shiekhattar, R. (2005) Human RISC couples microRNA biogenesis and posttranscriptional gene silencing. Cell, 123, 631–640."/>
  <meta name="citation_reference" content="Maniataki, E. and Mourelatos, Z. (2005) A human, ATP-independent, RISC assembly machine fueled by pre-miRNA. Genes &amp;; development, 19, 2979–2990."/>
  <meta name="citation_reference" content="Cifuentes, D., Xue, H., Taylor, D.W., Patnode, H., Mishima, Y., Cheloufi, S., Ma, E., Mane, S., Hannon, G.J., Lawson, N.D. et al. (2010) A novel miRNA processing pathway independent of Dicer requires Argonaute2 catalytic activity. Science, 328, 1694–1698."/>
  <meta name="citation_reference" content="Diederichs, S. and Haber, D.A. (2007) Dual role for argonautes in microRNA processing and posttranscriptional regulation of microRNA expression. Cell, 131, 1097–1108."/>
  <meta name="citation_reference" content="Cheloufi, S., Dos Santos, C.O., Chong, M.M. and Hannon, G.J. (2010) A dicer-independent miRNA biogenesis pathway that requires Ago catalysis. Nature, 465, 584–589."/>
  <meta name="citation_reference" content="Yoda, M., Cifuentes, D., Izumi, N., Sakaguchi, Y., Suzuki, T., Giraldez, A.J. and Tomari, Y. (2013) Poly(A)-specific ribonuclease mediates 3’-end trimming of Argonaute2-cleaved precursor microRNAs. Cell reports, 5, 715–726."/>
  <meta name="citation_reference" content="Liu, X., Jin, D.-Y., McManus, M.T. and Mourelatos, Z. (2012) Precursor microRNA-programmed silencing complex assembly pathways in mammals. Mol. Cell, 46, 507–517."/>
  <meta name="citation_reference" content="Heo, I., Joo, C., Cho, J., Ha, M., Han, J. and Kim, V.N. (2008) Lin28 mediates the terminal uridylation of let-7 precursor MicroRNA. Mol Cell, 32, 276–284."/>
  <meta name="citation_reference" content="Heo, I., Ha, M., Lim, J., Yoon, M.-J., Park, J.-E., Kwon, S.C., Chang, H. and Kim, V.N. (2012) Mono-uridylation of pre-microRNA as a key step in the biogenesis of group II let-7 microRNAs. Cell, 151, 521–532."/>
  <meta name="citation_reference" content="Liu, X., Zheng, Q., Vrettos, N., Maragkakis, M., Alexiou, P., Gregory, B.D. and Mourelatos, Z. (2014) A MicroRNA precursor surveillance system in quality control of MicroRNA synthesis. Mol. Cell, 55, 868–879."/>
  <meta name="citation_reference" content="Kim, B., Ha, M., Loeff, L., Chang, H., Simanshu, D.K., Li, S., Fareh, M., Patel, D.J., Joo, C. and Kim, V.N. (2015) TUT7 controls the fate of precursor microRNAs by using three different uridylation mechanisms. The EMBO journal, 34, 1801–1815."/>
  <meta name="citation_reference" content="Newman, M.A., Mani, V. and Hammond, S.M. (2011) Deep sequencing of microRNA precursors reveals extensive 3’ end modification. RNA, 17, 1795–1803."/>
  <meta name="citation_reference" content="Wyman, S.K., Knouf, E.C., Parkin, R.K., Fritz, B.R., Lin, D.W., Dennis, L.M., Krouse, M.A., Webster, P.J. and Tewari, M. (2011) Post-transcriptional generation of miRNA variants by multiple nucleotidyl transferases contributes to miRNA transcriptome complexity. Genome research, 21, 1450–1461."/>
  <meta name="citation_reference" content="Dueck, A., Ziegler, C., Eichner, A., Berezikov, E. and Meister, G. (2012) microRNAs associated with the different human Argonaute proteins. Nucleic acids research, 40, 9850–9862."/>
  <meta name="citation_reference" content="Burroughs, A.M., Ando, Y., de Hoon, M.J., Tomaru, Y., Nishibu, T., Ukekawa, R., Funakoshi, T., Kurokawa, T., Suzuki, H., Hayashizaki, Y. et al. (2010) A comprehensive survey of 3’ animal miRNA modification events and a possible role for 3’ adenylation in modulating miRNA targeting effectiveness. Genome research, 20, 1398–1410."/>
  <meta name="citation_reference" content="Friedlander, M.R., Chen, W., Adamidi, C., Maaskola, J., Einspanier, R., Knespel, S. and Rajewsky, N. (2008) Discovering microRNAs from deep sequencing data using miRDeep. Nature biotechnology, 26, 407–415."/>
  <meta name="citation_reference" content="Londin, E., Loher, P., Telonis, A.G., Quann, K., Clark, P., Jing, Y., Hatzimichael, E., Kirino, Y., Honda, S., Lally, M. et al. (2015) Analysis of 13 cell types reveals evidence for the expression of numerous novel primate- and tissue-specific microRNAs. Proceedings of the National Academy of Sciences of the United States of America, 112, E1106–1115."/>
  <meta name="citation_reference" content="Li, N., You, X., Chen, T., Mackowiak, S.D., Friedländer, M.R., Weigt, M., Du, H., Gogol-Döring, A., Chang, Z., Dieterich, C. et al. (2013) Global profiling of miRNAs and the hairpin precursors: insights into miRNA processing and novel miRNA discovery. Nucleic acids research, 41, 3619–3634."/>
  <meta name="citation_reference" content="Burroughs, A.M., Kawano, M., Ando, Y., Daub, C.O. and Hayashizaki, Y. (2012) pre-miRNA profiles obtained through application of locked nucleic acids and deep sequencing reveals complex 5’/3’ arm variation including concomitant cleavage and polyuridylation patterns. Nucleic acids research, 40, 1424–1437."/>
  <meta name="citation_reference" content="Ingolia, N.T., Ghaemmaghami, S., Newman, J.R. and Weissman, J.S. (2009) Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling. Science, 324, 218–223."/>
  <meta name="citation_reference" content="Kirino, Y., Vourekas, A., Khandros, E. and Mourelatos, Z. (2011) Immunoprecipitation of piRNPs and directional, next generation sequencing of piRNAs. Methods in molecular biology, 725, 281–293."/>
  <meta name="citation_reference" content="Schmittgen, T.D., Lee, E.J., Jiang, J., Sarkar, A., Yang, L., Elton, T.S. and Chen, C. (2008) Real-time PCR quantification of precursor and mature microRNA. Methods, 44, 31–38."/>
  <meta name="citation_reference" content="Moll, E.K., Donoghue, A.J., Alpern, E.R., Kleppel, J., Durbin, D.R. and Winston, F.K. (2002) Child bicyclist injuries: are we obtaining enough information in the emergency department chart? Inj Prev, 8, 165–169."/>
  <meta name="citation_reference" content="Huber, W., von Heydebreck, A., Sultmann, H., Poustka, A. and Vingron, M. (2002) Variance stabilization applied to microarray data calibration and to the quantification of differential expression. Bioinformatics, 18 Suppl 1, S96–104."/>
  <meta name="citation_reference" content="Li, F., Zheng, Q., Ryvkin, P., Dragomir, I., Desai, Y., Aiyer, S., Valladares, O., Yang, J., Bambina, S., Sabin, L.R. et al. (2012) Global analysis of RNA secondary structure in two metazoans. Cell reports, 1, 69–82."/>
  <meta name="citation_reference" content="Althammer, S., Gonzalez-Vallinas, J., Ballare, C., Beato, M. and Eyras, E. (2011) Pyicos: a versatile toolkit for the analysis of high-throughput sequencing data. Bioinformatics, 27, 3333–3340."/>
  <meta name="citation_reference" content="Lorenz, R., Bernhart, S.H., Honer Zu Siederdissen, C., Tafer, H., Flamm, C., Stadler, P.F. and Hofacker, I.L. (2011) ViennaRNA Package 2.0. Algorithms for molecular biology: AMB, 6, 26."/>
  <meta name="citation_reference" content="Elliott, R., Li, F., Dragomir, I., Chua, M.M., Gregory, B.D. and Weiss, S.R. (2013) Analysis of the host transcriptome from demyelinating spinal cord of murine coronavirus-infected mice. PLoS One, 8, e75346."/>
  <meta name="citation_reference" content="Anders, S., Pyl, P.T. and Huber, W. (2015) HTSeq--a Python framework to work with high-throughput sequencing data. Bioinformatics, 31, 166–169."/>
  <meta name="citation_reference" content="Trapnell, C., Pachter, L. and Salzberg, S.L. (2009) TopHat: discovering splice junctions with RNA-Seq. Bioinformatics, 25, 1105–1111."/>
  <meta name="citation_reference" content="Martin, M. (2011) Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet.journal, 17, 10–12."/>
  <meta name="citation_reference" content="Love, M.I., Huber, W. and Anders, S. (2014) Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome biology, 15, 550."/>
  <meta name="citation_reference" content="Nelson, P.T., De Planell-Saguer, M., Lamprinaki, S., Kiriakidou, M., Zhang, P., O’Doherty, U. and Mourelatos, Z. (2007) A novel monoclonal antibody against human Argonaute proteins reveals unexpected characteristics of miRNAs in human blood cells. RNA, 13, 1787–1792."/>
  <meta name="citation_reference" content="Smith, T.F. and Waterman, M.S. (1981) Identification of common molecular subsequences. Journal of molecular biology, 147, 195–197."/>
  <meta name="citation_reference" content="Langmead, B. and Salzberg, S.L. (2012) Fast gapped-read alignment with Bowtie 2. Nat Methods, 9, 357–359."/>
  <meta name="citation_reference" content="Kozomara, A. and Griffiths-Jones, S. (2014) miRBase: annotating high confidence microRNAs using deep sequencing data. Nucleic acids research, 42, D68–73."/>
  <meta name="citation_reference" content="Heo, I., Joo, C., Kim, Y.K., Ha, M., Yoon, M.J., Cho, J., Yeom, K.H., Han, J. and Kim, V.N. (2009) TUT4 in concert with Lin28 suppresses microRNA biogenesis through pre-microRNA uridylation. Cell, 138, 696–708."/>
  <meta name="citation_reference" content="Thornton, J.E., Du, P., Jing, L., Sjekloca, L., Lin, S., Grossi, E., Sliz, P., Zon, L.I. and Gregory, R.I. (2014) Selective microRNA uridylation by Zcchc6 (TUT7) and Zcchc11 (TUT4). Nucleic acids research, 42, 11777–11791."/>
  <meta name="citation_reference" content="Liu, J., Carmell, M.A., Rivas, F.V., Marsden, C.G., Thomson, J.M., Song, J.J., Hammond, S.M., Joshua-Tor, L. and Hannon, G.J. (2004) Argonaute2 is the catalytic engine of mammalian RNAi. Science, 305, 1437–1441."/>
  <meta name="citation_reference" content="Meister, G., Landthaler, M., Patkaniowska, A., Dorsett, Y., Teng, G. and Tuschl, T. (2004) Human Argonaute2 mediates RNA cleavage targeted by miRNAs and siRNAs. Molecular cell, 15, 185–197."/>
  <meta name="citation_reference" content="Yekta, S., Shih, I.H. and Bartel, D.P. (2004) MicroRNA-directed cleavage of HOXB8 mRNA. Science, 304, 594–596."/>
  <meta name="citation_reference" content="Yang, J.S., Maurin, T., Robine, N., Rasmussen, K.D., Jeffrey, K.L., Chandwani, R., Papapetrou, E.P., Sadelain, M., O’Carroll, D. and Lai, E.C. (2010) Conserved vertebrate mir-451 provides a platform for Dicer-independent, Ago2-mediated microRNA biogenesis. Proc Natl Acad Sci U S A, 107, 15163–15168."/>
  <meta name="citation_reference" content="Han, J., Pedersen, J.S., Kwon, S.C., Belair, C.D., Kim, Y.K., Yeom, K.H., Yang, W.Y., Haussler, D., Blelloch, R. and Kim, V.N. (2009) Posttranscriptional crossregulation between Drosha and DGCR8. Cell, 136, 75–84."/>
  <meta name="citation_reference" content="Kadener, S., Rodriguez, J., Abruzzi, K.C., Khodor, Y.L., Sugino, K., Marr, M.T., 2nd, Nelson, S. and Rosbash, M. (2009) Genome-wide identification of targets of the drosha-pasha/DGCR8 complex. RNA, 15, 537–545."/>
  <meta name="citation_reference" content="Triboulet, R., Chang, H.M., Lapierre, R.J. and Gregory, R.I. (2009) Post-transcriptional control of DGCR8 expression by the Microprocessor. RNA, 15, 1005–1011."/>
  <meta name="citation_reference" content="Rybak-Wolf, A., Jens, M., Murakawa, Y., Herzog, M., Landthaler, M. and Rajewsky, N. (2014) A variety of dicer substrates in human and C. elegans. Cell, 159, 1153–1167."/>
  <meta name="citation_reference" content="Macias, S., Plass, M., Stajuda, A., Michlewski, G., Eyras, E. and Caceres, J.F. (2012) DGCR8 HITS-CLIP reveals novel functions for the Microprocessor. Nature structural &amp;; molecular biology, 19, 760–766."/>
  <meta name="citation_reference" content="Spitale, R.C., Flynn, R.A., Zhang, Q.C., Crisalli, P., Lee, B., Jung, J.W., Kuchelmeister, H.Y., Batista, P.J., Torre, E.A., Kool, E.T. et al. (2015) Structural imprints in vivo decode RNA regulatory mechanisms. Nature, 519, 486–490."/>
  <meta name="citation_reference" content="Hentze, M.W., Rouault, T.A., Caughman, S.W., Dancis, A., Harford, J.B. and Klausner, R.D. (1987) A cis-acting element is necessary and sufficient for translational regulation of human ferritin expression in response to iron. Proc Natl Acad Sci U S A, 84, 6730–6734."/>
  <meta name="twitter:title" content="Isolation and sequencing of AGO-bound RNAs reveals characteristics of mammalian stem-loop processing in vivo"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="MicroRNA precursors (pre-miRNAs) are short hairpin RNAs that are rapidly processed into mature microRNAs (miRNAs) in the cytoplasm. Due to their low abundance in cells, sequencing-based studies of pre-miRNAs have been limited. We successfully enriched for and deep sequenced pre-miRNAs in human cells by capturing these RNAs during their interaction with Argonaute (AGO) proteins. Using this approach, we detected &gt;; 350 pre-miRNAs in human cells and &gt;; 250 pre-miRNAs in a reanalysis of a similar study in mouse cells. We uncovered widespread trimming and non-templated additions to the 3’ ends of pre- and mature miRNAs. Additionally, we created an index for microRNA precursor processing efficiency. This analysis revealed a subset of pre-miRNAs that produce low levels of mature miRNAs despite abundant precursors, including an annotated miRNA in the 5’ UTR of the DiGeorge syndrome critical region 8 mRNA transcript. This led us to search for AGO-associated stem-loops originating from other mRNA species, which identified hundreds of putative pre-miRNAs derived from human and mouse mRNAs. In summary, we provide a wealth of information on mammalian pre-miRNAs, and identify novel microRNA and microRNA-like elements localized in mRNAs."/>
  <meta name="og-title" property="og:title" content="Isolation and sequencing of AGO-bound RNAs reveals characteristics of mammalian stem-loop processing in vivo"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/294488v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="MicroRNA precursors (pre-miRNAs) are short hairpin RNAs that are rapidly processed into mature microRNAs (miRNAs) in the cytoplasm. Due to their low abundance in cells, sequencing-based studies of pre-miRNAs have been limited. We successfully enriched for and deep sequenced pre-miRNAs in human cells by capturing these RNAs during their interaction with Argonaute (AGO) proteins. Using this approach, we detected &gt;; 350 pre-miRNAs in human cells and &gt;; 250 pre-miRNAs in a reanalysis of a similar study in mouse cells. We uncovered widespread trimming and non-templated additions to the 3’ ends of pre- and mature miRNAs. Additionally, we created an index for microRNA precursor processing efficiency. This analysis revealed a subset of pre-miRNAs that produce low levels of mature miRNAs despite abundant precursors, including an annotated miRNA in the 5’ UTR of the DiGeorge syndrome critical region 8 mRNA transcript. This led us to search for AGO-associated stem-loops originating from other mRNA species, which identified hundreds of putative pre-miRNAs derived from human and mouse mRNAs. In summary, we provide a wealth of information on mammalian pre-miRNAs, and identify novel microRNA and microRNA-like elements localized in mRNAs."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2018-04-06"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Isolation and sequencing of AGO-bound RNAs reveals characteristics of mammalian stem-loop processing in vivo | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-91282 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">ABSTRACT</h2>
    <p id="p-4">MicroRNA precursors (pre-miRNAs) are short hairpin RNAs that are rapidly processed into mature microRNAs (miRNAs) in the cytoplasm. Due to their low abundance in cells, sequencing-based studies of pre-miRNAs have been limited. We successfully enriched for and deep sequenced pre-miRNAs in human cells by capturing these RNAs during their interaction with Argonaute (AGO) proteins. Using this approach, we detected &gt;; 350 pre-miRNAs in human cells and &gt;; 250 pre-miRNAs in a reanalysis of a similar study in mouse cells. We uncovered widespread trimming and non-templated additions to the 3’ ends of pre- and mature miRNAs. Additionally, we created an index for microRNA precursor processing efficiency. This analysis revealed a subset of pre-miRNAs that produce low levels of mature miRNAs despite abundant precursors, including an annotated miRNA in the 5’ UTR of the DiGeorge syndrome critical region 8 mRNA transcript. This led us to search for AGO-associated stem-loops originating from other mRNA species, which identified hundreds of putative pre-miRNAs derived from human and mouse mRNAs. In summary, we provide a wealth of information on mammalian pre-miRNAs, and identify novel microRNA and microRNA-like elements localized in mRNAs.</p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">INTRODUCTION</h2>
    <p id="p-5">MicroRNAs (miRNAs) are ~22 nucleotide (nt) small RNAs (smRNAs) that negatively regulate gene expression at the post-transcriptional level by repressing translation and/or promoting degradation of target messenger RNAs (mRNAs) (
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>,
     <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>). Animal miRNAs are generated in a two-step process, whereby miRNA precursors (pre-miRNAs) are first cleaved from their primary transcripts (pri-miRNAs) by the action of the DGCR8/DROSHA microprocessor complex (
     <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>,
     <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>). Alternative pre-miRNA biogenesis pathways have been described that bypass the microprocessor. For example, mirtron loci generate pre-miRNAs in a splicing-dependent and DROSHA-independent fashion (
     <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>,
     <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>). Pre-miRNAs are then transported into the cytoplasm by Exportin-5 for further processing (
     <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>,
     <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>).
    </p>
    <p id="p-6">Cytoplasmic pre-miRNAs are matured by the miRNA loading complex (miRLC) which is composed of the type III endonuclease DICER, the double-stranded RNA-binding protein TRBP, and the miRNA effector protein Argonaute (AGO) (
     <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>–
     <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>). DICER cleaves the pre-miRNA to reveal a ~22 nt miRNA duplex, consisting of the upstream miRNA (denoted as the 5p miRNA) and downstream miRNA (denoted as the 3p miRNA). One of these strands is selectively loaded into AGO to form the RNA-induced silencing complex (RISC). Alternatively, AGO2 has been shown to directly cleave several pre-miRNAs, which are then further processed by the poly(A)-specific ribonuclease (PARN) to give rise to mature miRNAs. This class of miRNAs is known as the AGO2-cleaved pre-miRNAs (ac-pre-miRNAs) (
     <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>–
     <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>). Such DICER-independent processing involves the pre-miRNA deposit complex (miPDC), which is composed of just AGO2 and a pre-miRNA (
     <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>).
    </p>
    <p id="p-7">The miRNA biogenesis mechanism is further complicated by the fact that both pre- and mature miRNAs can be post-transcriptionally modified at the 3’ end by trimming or by non-templated addition of ribonucleotides, especially uridine and adenine (
     <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>–
     <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>). In general, mono-uridylation is thought to re-establish a 3’ 2 nt overhang, which is required for efficient DICER cleavage. In contrast oligo-uridylation has been shown to be a signal for degradation and usually occurs after AGO2-mediated slicing of pre-miRNAs. However, most detailed studies of pre-miRNAs have been performed on a small number of pre-miRNA sequences, thus our understanding of the overall landscape, and global function of these modifications remains elusive.
    </p>
    <p id="p-8">High-throughput sequencing of total or AGO-bound smRNAs in numerous cells, tissues, and organisms has provided a wealth of information about miRNAs. In concert with bioinformatic approaches, these datasets have been expanded to identify thousands of novel miRNAs (
     <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>,
     <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>). In contrast, sequencing of pre-miRNAs has been challenging, due to the presence of other RNA species, including the highly abundant transfer RNAs (tRNAs) and small nucleolar RNAs (snoRNAs), that exist in the same size range as pre-miRNAs. Attempts to use size selection to sequence pre-miRNAs have achieved &lt;; 1% of total sequenced clones corresponding to pre-miRNAs even after selective depletion of abundant species (
     <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>). Alternatively, primer-based approaches have been applied, but these methods restrict analysis to known miRNA species and thus cannot be used for discovery (
     <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>,
     <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>,
     <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>). Thus, there is a need for unbiased, sequencing based approaches to gain a more comprehensive understanding of pre-miRNA expression and sequence content.
    </p>
    <p id="p-9">Leveraging the knowledge that AGO is an integral component of the pre-miRNA processing complexes (miRLC and miPDC), we recently developed an approach to enrich for pre-miRNAs by immunoprecipitating AGO proteins and subsequently cloning RNAs from 50-80 nucleotides (nts). Using this approach, pre-miRNA libraries from mouse embryonic fibroblasts (MEFs) were generated with &gt;; 40% of reads mapping to miRNA loci (
     <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>). Thus, this strategy can be used to efficiently study pre-miRNAs globally without the need for primer-based approaches or depletion of abundant RNA species. Here, we applied this approach to isolate and sequence pre-miRNAs and mature miRNAs from human embryonic kidney (HEK293T) cells. We developed a bioinformatic pipeline to capture post-transcriptional modifications, which we applied to data generated in this study from HEK293T cells, as well as to the previous data generated in MEFs. Our results provide global insights into pre-miRNA processing and provide an alternative strategy for identifying pre-miRNAs and other AGO-associated stem-loops in transcriptomes of interest.
    </p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">MATERIALS AND METHODS</h2>
    <div id="sec-3" class="subsection">
     <h3>Cell culture</h3>
     <p id="p-10">HEK293T cells were grown to 70-80% confluence in 15 cm tissue culture plates with DMEM media supplemented with 10% FBS and 1X pen/strep at 37°C and 5% CO
      <sub>2</sub>.
     </p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>AGO-IP-sequencing</h3>
     <p id="p-11">Pre-miRNA-seq and miRNA-seq preparation was based on previous work (
      <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a>,
      <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>,
      <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>). HEK293T cells were lysed in RSB 200 (20 mM Tris-HCl pH 7.4, 200 mM NaCl, 2.5 mM MgCl
      <sub>2</sub>, 0.5% NP-40) supplemented with 0.2 U/μl RNasein (Promega) and protease inhibitor cocktail (Roche). Cleared lysates were incubated with Protein G agarose beads (Life Technologies) conjugated to AGO 2A8 antibody or non-immune serum (NMS), for 1.5 h at 4°C. After four 5 min washes in lysis buffer, a sample of beads was drawn for western blot analysis while the rest was subjected to RNA extraction with Trizol (Life Technologies). RNA was dephosphorylated with antarctic phosphatase (NEB) and P
      <sup>32</sup>-γ-ATP labeled with T4 polynucleotide kinase (NEB).
     </p>
     <p id="p-12">5’ end radiolabeled RNAs were resolved on a 7 M urea, 15% polyacrylamide gel. Gel slices from 20-25 nt (miRNA-seq) and 50-80 nt (pre-miRNA-seq) were recovered and ligated to miRCat 3’ Linker-1 (IDT) with T4 RNA Ligase 2 Truncated (NEB), in the presence of 15% PEG8000. Ligation products were PAGE purified and reverse transcribed by a 5’ phosphorylated RT primer containing 3’ and 5’ adaptor complementary sequences. To do this, RT primer was mixed with RNA and heated at 80°C for 5 min, then incubated at 60°C for 5 min and finally cooled down to room temperature before the remaining reagents were added (Affinityscipt, Stratagene) (
      <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>). cDNAs were resolved and purified from a 7 M urea, 6% polyacrylamide gel, then circularized with CircLigase I (Epicentre) and PCR amplified with Phusion DNA polymerase (NEB). The amplicon yield was boosted with a second PCR amplification round, then size selected on a 3% metaphor gel (Lonza), purified, and sent for deep sequencing at the Next-Generation Sequencing Core at the University of Pennsylvania. All oligonucleotide sequences used in this study can be found in the attached document (
      <strong>Table S9</strong>).
     </p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Mapping Pipeline</h3>
     <p id="p-13">The first 20 nt of the 3’ adapter sequence 
      <em>CTGTAGGCACCATCAATAGA</em> was used to trim adapter sequence from the raw reads using cutadapt (v1.4.2). Identical reads were collapsed but clone information was retained to reduce computational time. Trimmed reads were sequentially mapped to miRBase (v20) and RefSeq annotated spliced transcript models (hg19 or mm10, downloaded on 06082015), with a two-stage alignment strategy with Bowtie2 and EMBOSS-WATER. First, Bowtie2 was used to map the 5’ regions of reads to either miRbase (v20) primary miRNA sequences with a 50 nt extension on the 3’ end or RefSeq annotated spliced transcript models. For pre-miRNA-seq, the first 35 nt were used in the initial alignment step, whereas 18 nt were used for miRNA- and smRNA-seq. Following Bowtie2 alignment, reads were extended by local alignment with EMBOSS-WATER (with parameters: -gapopen 10.0 -gapextend 0.5). Multimapped reads were partially resolved by selecting the longest, highest scoring alignments. Mismatches detected at the 3’ ends of reads were considered as non-templated additions and analyzed separately. We filtered unmapped reads from pre-miRNA-seq for known rRNA, snoRNA, snRNA, tRNA, mitochondrial transcripts, and repeat-masked sequences before mapping to RefSeq.
     </p>
    </div>
    <div id="sec-6" class="subsection">
     <h3>Analysis of miRNA trimming and non-templated tailing</h3>
     <p id="p-14">We tabulated non-templated additions revealed by our mapping pipeline for reads, which mapped to high-confidence miRBase annotations. Additionally, we calculated templated-extension and trimming by comparing mapped ends of pre- miRNA-seq, miRNA-seq, and smRNA-seq reads against annotated 5p and 3p miRNA ends.</p>
    </div>
    <div id="sec-7" class="subsection">
     <h3>Identification of AGO-2 cleaved pre-miRNAs</h3>
     <p id="p-15">For each miRBase miRNA, we calculated the percentage of trimmed pre-miRNA-seq reads that ended between 8-15 nt upstream of the 3’ end of the 3p miRNA or downstream of the 5’ end of the 5p miRNA. We identified pre-miRNAs with &gt;;1% of such cleavage events in the 3p arm as putative ac-pre-miRNAs. For cleavage events in the 5p arm, we took a more conservative approach, requiring 5% of reads to terminate in this region and &gt;; 50% of mature miRNA-seq reads in the 3p arm compared to 5p arm.</p>
    </div>
    <div id="sec-8" class="subsection">
     <h3>Determination of MPAR</h3>
     <p id="p-16">We calculated the mature to precursor abundance ratio (MPAR) for miRBase miRNAs or AGO-associated stem-loops as the generalized log ratio (glog) (
      <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>–
      <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>) of miRNA-seq (smRNA-seq) to pre-miRNA-seq RPM coverage (nmi, npr, respectively) as follows:
     </p>
     <span class="disp-formula" id="disp-formula-1">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/04/06/294488/embed/graphic-1.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/04/06/294488/embed/graphic-1.gif"/>
       </noscript>
      </span>
     </span>
     <p id="p-17">Thus MPAR can be calculated for loci that are not represented in either the pre-miR-seq or miR-seq library. Loci with no coverage in either sequencing library were omitted from this analysis.</p>
    </div>
    <div id="sec-9" class="subsection">
     <h3>Identification of AGO-associated stem-loops derived from mRNAs</h3>
     <p id="p-18">Pre-miRNA-seq reads that failed to map to miRBase and were not removed by mapping to known rRNA, snoRNA, snRNA, tRNA, mitochondrial transcripts, and repeat-masked sequences were aligned to RefSeq transcripts with Bowtie2. After filtering for best matches for reads with less than 4 mismatches, we used Pycioclip to call significant peaks with a modified False Discovery Rate &lt;; 0.01 (
      <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>). To remove abundant genes with high numbers of mappings but no local peaks, we filtered out peaks that were greater than 200 nt in length. We then chose the most abundant clone and predicted its secondary structure with RNAfold using standard parameters (
      <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>). We again filtered clusters, requiring that they had greater than 15 bp in the longest hairpin and a total MFE of less than −0.3 kcal/mol/nt. We analyzed non-templated tailing for reads mapping to these hairpins as described above for our miRNA analysis.
     </p>
    </div>
    <div id="sec-10" class="subsection">
     <h3>icSHAPE analysis</h3>
     <p id="p-19">
      <em>In vivo</em> icSHAPE data from mouse ES cells was downloaded from GEO using the accession number GSE60034. Only scores in the top or bottom 10
      <sup>th</sup> percentile were used for analysis. icSHAPE reactivity scores were overlaid on RNAfold diagrams using RNAplot (
      <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">36</a>).
     </p>
    </div>
    <div id="sec-11" class="subsection">
     <h3>Small RNA Northern blotting</h3>
     <p id="p-20">20-30 μg of total RNA was resolved on 8 M urea 12.5% PAGE and transferred to Hybond-N+ membranes (GE Healthcare). Blotted RNA was hybridized with γ-
      <sup>32</sup>P-ATP labeled LNA oligonucleotides (
      <strong>Table S9</strong>) in UltraHyb-Oligo Buffer (Ambion) at 50ºC overnight. Membranes were washed once in 2x SSC, 0.1% SDS at 50ºC and once in 0.1x SSC, 0.1% SDS at 50ºC before exposing to autoradiography film for 1 week at −80ºC.
     </p>
    </div>
    <div id="sec-12" class="subsection">
     <h3>siRNA-mediated knockdown of DGCR8 and DROSHA</h3>
     <p id="p-21">ON-TARGET plus siRNAs against Human 
      <em>DGCR8 and DROSHA</em> were obtained from Dharmacon (J-015713-05-0002, J-015713-06-0002, J-016996-05-0002, J-016996-06-0002). siRNAs against Luciferase were generated by IDT. We performed two sequential siRNA transfections 48 hours apart with 45 pmol of siRNAs (22.5 pmol siRNA-1 and 22.5 pmol siRNA-2), 125 μL Opti-MEM (Life Technologies), and 6 μL Lipofectamine 2000 (Life Technologies) per reaction. Reactions were incubated at room temperature for 20 minutes. During this incubation, we seeded 6.0×10
      <sup>5</sup> HEK293T cells per replicate in two wells of a 6-well plate (3.0×10
      <sup>5</sup> cells/well) in 2 mL media. We then added the siRNA mixture to each well dropwise, and allowed cells to incubate at 37ºC in 5% CO
      <sub>2</sub> for 48 hours. After 48 hours we repeated the transfection, harvesting the two wells per replicate (~1.2×10
      <sup>5</sup> cells) and dividing them into four wells in a 6-well plate. We treated each well with 45 pmol of siRNAs, and allowed them to incubate at 37ºC in 5% CO
      <sub>2</sub> for another 48 hours. Cells were then pooled and washed with PBS prior to storage at −80ºC.
     </p>
    </div>
    <div id="sec-13" class="subsection">
     <h3>mRNA-seq</h3>
     <p id="p-22">mRNA-seq was performed as previously described (
      <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>). Briefly, total RNA was purified from the HEK293T cell cultures (miRNeasy; Qiagen). Poly(A)
      <sup>+</sup> RNA was isolated using oligo(dT) beads (Life Technologies). RNA was fragmented for 7 minutes using Fragmentation Reagent (Life Technologies). mRNA-seq libraries were then generated using the Illumina mRNA-seq kit (Illumina). Reads were trimmed with Cutadapt, mapped with Tophat2, and gene expression was quantified using HTseq (
      <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>–
      <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>). DEseq2 was used to perform differential expression analysis (
      <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>)
     </p>
    </div>
   </div>
   <div class="section" id="sec-14">
    <h2 class="">RESULTS</h2>
    <div id="sec-15" class="subsection">
     <h3>Isolation and sequencing of pre-miRNAs</h3>
     <p id="p-23">To isolate and sequence pre-miRNAs we first immunoprecipitated AGO proteins from HEK293T cells with the pan-AGO-2A8 antibody (
      <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>) (
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1A</a>). RNA was purified from AGO immunoprecipitates, dephosphorylated and labeled with P
      <sup>32</sup>-γ-ATP. Autoradiography of the RNA gel showed a major band at 20-25 nt, representing mature miRNAs, and several other prominent bands between 50-80 nt, corresponding to the size range of pre-miRNAs (
      <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1B</a>). We excised gel slices from both of these regions and generated high-throughput sequencing libraries, referred to herein as miRNA-seq and pre-miRNA-seq libraries, respectively. Previous attempts to sequence pre-miRNAs have been unsuccessful due to the inaccessibility of the pre-miRNA 5’ ends. Therefore, we used a method that attaches the 5’ linker through a CircLigase-mediated cDNA circularization step (see 
      <strong>Methods</strong>) (
      <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">19</a>,
      <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a>).
     </p>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Isolation and sequencing of AGO interacting pre-miRNAs and mature miRNAs A) Western blot of AGO-IP from HEK293T cells. AGO-2A8 and non-immune mouse serum (NMS) were used for immunoprecipitation. AGO-2A8 antibody was used for detection. *Radixin is known to cross-react with the 2A8 antibody. Immunoglobulin heavy chain (HC) and light chain (LC) are denoted. B) Autoradiography of RNA co-immunoprecipitated with AGO. Pre-miRNAs were excised from 50-80 nt size range and mature miRNAs were excised from 20-25 nt size range. C) Bioinformatics pipeline: Adapter sequences were trimmed and PCR duplicates were collapsed for efficiency. The first 35 nt of pre-miRNAs (18 nt of mature miRNAs) were mapped to miRBase (v20). Remaining bases were mapped with a Smith-Waterman aligner. D) Percent of reads mapping to human miRBase for HEK293T pre-miRNA-seq and miRNA-seq. E-F) Size distribution of pre-miRNA-seq (E) and miRNA-seq (F) reads mapping to miRBase. G-H) Coverage plot of pre-miRNA-seq (G) and miRNA-seq (H) reads mapping to hsa-miR-16-2 locus. White bars indicate templated nucleotides. Colored bars indicate non-templated additions. Dashed red and blue lines indicate boundaries of annotated 5p (blue) and 3p (red) mature miRNAs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-814844358" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Isolation and sequencing of AGO interacting pre-miRNAs and mature miRNAs&lt;;/span&gt;; A) Western blot of AGO-IP from HEK293T cells. AGO-2A8 and non-immune mouse serum (NMS) were used for immunoprecipitation. AGO-2A8 antibody was used for detection. *Radixin is known to cross-react with the 2A8 antibody. Immunoglobulin heavy chain (HC) and light chain (LC) are denoted. B) Autoradiography of RNA co-immunoprecipitated with AGO. Pre-miRNAs were excised from 50-80 nt size range and mature miRNAs were excised from 20-25 nt size range. C) Bioinformatics pipeline: Adapter sequences were trimmed and PCR duplicates were collapsed for efficiency. The first 35 nt of pre-miRNAs (18 nt of mature miRNAs) were mapped to miRBase (v20). Remaining bases were mapped with a Smith-Waterman aligner. D) Percent of reads mapping to human miRBase for HEK293T pre-miRNA-seq and miRNA-seq. E-F) Size distribution of pre-miRNA-seq (E) and miRNA-seq (F) reads mapping to miRBase. G-H) Coverage plot of pre-miRNA-seq (G) and miRNA-seq (H) reads mapping to hsa-miR-16-2 locus. White bars indicate templated nucleotides. Colored bars indicate non-templated additions. Dashed red and blue lines indicate boundaries of annotated 5p (blue) and 3p (red) mature miRNAs.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F1.medium.gif" width="440" height="225"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F1.medium.gif" width="440" height="225"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 1.</span>
       <span class="caption-title">Isolation and sequencing of AGO interacting pre-miRNAs and mature miRNAs</span>
       <p id="p-24" class="first-child">A) Western blot of AGO-IP from HEK293T cells. AGO-2A8 and non-immune mouse serum (NMS) were used for immunoprecipitation. AGO-2A8 antibody was used for detection. *Radixin is known to cross-react with the 2A8 antibody. Immunoglobulin heavy chain (HC) and light chain (LC) are denoted. B) Autoradiography of RNA co-immunoprecipitated with AGO. Pre-miRNAs were excised from 50-80 nt size range and mature miRNAs were excised from 20-25 nt size range. C) Bioinformatics pipeline: Adapter sequences were trimmed and PCR duplicates were collapsed for efficiency. The first 35 nt of pre-miRNAs (18 nt of mature miRNAs) were mapped to miRBase (v20). Remaining bases were mapped with a Smith-Waterman aligner. D) Percent of reads mapping to human miRBase for HEK293T pre-miRNA-seq and miRNA-seq. E-F) Size distribution of pre-miRNA-seq (E) and miRNA-seq (F) reads mapping to miRBase. G-H) Coverage plot of pre-miRNA-seq (G) and miRNA-seq (H) reads mapping to hsa-miR-16-2 locus. White bars indicate templated nucleotides. Colored bars indicate non-templated additions. Dashed red and blue lines indicate boundaries of annotated 5p (blue) and 3p (red) mature miRNAs.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-25">Using this approach, we successfully generated high-throughput sequencing libraries for both pre-miRNAs and mature miRNAs. We obtained 27.8 and 9.8 million reads with sufficient adapter sequence from the pre-miRNA-seq and mature miRNA-seq libraries, respectively (
      <strong>Table S1</strong>). Given that pre-miRNAs and mature miRNAs contain non-templated modifications at their 3’ ends, we reasoned that standard alignment pipelines would be limited in their ability to align these sequences to miRBase. Therefore, we developed an alignment pipeline that utilized the first 35 nt of the pre-miRNA-seq (18 nt for miRNA-seq) reads for alignment (
      <a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1C</a>), followed by Smith-Waterman local alignment to extend the read as far as possible along the mapped miRNA sequence (see 
      <strong>Methods</strong>) (
      <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>–
      <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>). After all possible nucleotide matches were made, we selected alignments with the lowest mismatch rate and captured non-templated modifications at the 3’ end.
     </p>
     <p id="p-26">We performed this analysis on pre-miRNA-seq, miRNA-seq, and smRNA-seq (without IP) from human HEK293T cells as well as pre-miRNA-seq and mature miRNA-seq data previously generated using the same technique on MEFs (
      <a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">19</a>). We successfully aligned 10.8% of pre-miRNA-seq, 98.7% of miRNA-seq, and 52.4% of smRNA-seq reads to the human miRBase v20 annotation set (
      <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figs. 1D</a>, 
      <strong>S1A, and Table S1</strong>). We were not surprised to find such high rates of mapping for mature miRNAs, however, a 10% mapping rate for pre-miRNAs is significantly higher than previous attempts to sequence pre-miRNAs without pre-miRNA specific primers (0.8%) (
      <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a>). Using the datasets previously generated from MEFs, we had a much higher rate of pre-miRNA-seq reads mapping (44%), but mapped fewer miRNA-seq reads (90.3%) (
      <a id="xref-fig-9-1" class="xref-fig" href="#F9">Fig. S2A</a>
      <strong>and Table S1</strong>). These differences likely represent variable experimental conditions and amounts of starting material and/or biological differences in the smRNA populations between these two mammals.
     </p>
     <p id="p-27">Overall, we obtained pre-miRNA-seq reads mapping to 367 annotated human microRNAs and 267 annotated mouse microRNAs (
      <strong>Table S2</strong>). For libraries prepared from smaller RNA species, we mapped reads to 931 (HEK239T miRNA-seq), 567 (MEF miRNAs-seq), and 1,364 (HEK293T smRNA-seq) miRBase miRNAs. We examined the size distribution of mapped pre-miRNA-seq reads and found that they were broadly distributed between 55-65 nt in both cell types (
      <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figs. 1E</a> and 
      <a id="xref-fig-9-2" class="xref-fig" href="#F9">S2B</a>). In contrast, miRNA-seq reads, were tightly distributed between 21-24 nt, in both HEK293T AGO-bound and total cellular fractions, as well as in MEFs (
      <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figs. 1F</a>, 
      <a id="xref-fig-8-1" class="xref-fig" href="#F8">S1B</a>, and 
      <a id="xref-fig-9-3" class="xref-fig" href="#F9">S2C</a>). We determined the abundance, end concordance, and non-templated additions for all mapped miRNAs and generated coverage plots to represent these data, which are available for download at 
      <a href="http://gregorylab.bio.upenn.edu/AGO_IP_Seq/">http://gregorylab.bio.upenn.edu/AGO_IP_Seq/</a> (
      <strong>Table S2; examples in</strong>
      <a id="xref-fig-1-7" class="xref-fig" href="#F1">Figs. 1G-H</a>). Together, our biochemical and bioinformatic approaches provide a data rich resource for the global and unbiased analysis of pre-miRNAs in two mammals.
     </p>
     <div id="F8" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F8.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-814844358" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F8.medium.gif" width="440" height="343"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure8" src="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F8.medium.gif" width="440" height="343"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure8" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
     </div>
     <div id="F9" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F9.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-814844358" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F9.medium.gif" width="440" height="412"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure9" src="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F9.medium.gif" width="440" height="412"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure9" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
     </div>
    </div>
    <div id="sec-16" class="subsection">
     <h3>Diverse ends of AGO-bound pre-miRNAs</h3>
     <p id="p-28">It is well established that pre-miRNA trimming and non-templated tailing (uridylation) is a mechanism of regulation (
      <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>,
      <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>,
      <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a>). However, most studies have investigated individual miRNAs or used targeted approaches to examine a predetermined subset of miRNAs. Using our novel datasets, we examined the end concordance of sequenced pre-miRNA-seq (
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figs. 2A</a> and 
      <a id="xref-fig-9-4" class="xref-fig" href="#F9">S2D</a>) and miRNA-seq reads (
      <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figs. 2B</a> and 
      <a id="xref-fig-9-5" class="xref-fig" href="#F9">S2E</a>), relative to high confidence human and mouse miRBase miRNA ends. We found that the majority (&gt;;90%) of 5’ read ends of pre-miRNAs coincided with annotated 5’ ends of 5p miRNAs for both human and mouse (
      <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figs. 2A</a> and 
      <a id="xref-fig-9-6" class="xref-fig" href="#F9">S2D</a>). In contrast 3’ read ends of pre-miRNAs were highly variable, with only 40% and 20% of reads ending precisely at the annotated 3’ end of 3p miRNAs in human and mouse cells, respectively (
      <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figs. 2A</a> and 
      <a id="xref-fig-9-7" class="xref-fig" href="#F9">S2D</a>). Relatedly, mature miRNAs (
      <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figs. 2B</a> and 
      <a id="xref-fig-9-8" class="xref-fig" href="#F9">S2E</a>) and smRNAs (
      <a id="xref-fig-8-2" class="xref-fig" href="#F8">Fig. S1C</a>) displayed higher variation in their 3’ ends relative to 5’ ends for both 5p and 3p miRNAs, but not nearly to the extent that was observed for pre-miRNAs (
      <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figs. 2A</a> and 
      <a id="xref-fig-9-9" class="xref-fig" href="#F9">S2D</a>). Overall, these results reveal that 3’ end variation is more common in pre-miRNAs than in mature miRNAs.
     </p>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="pre-miRNA-seq identifies diverse ends of pre-miRNAs A) Distribution of pre-miRNA-seq read ends relative to annotated 5[[#x2019]] end of 5p and 3[[#x2019]] end of 3p miRNAs for high confidence human miRNAs. Positive and negative values indicate trimming and extension of reads, respectively. B) Distribution of miRNA-seq read ends relative to annotated ends of 5p and 3p miRNAs for high confidence human miRNAs. Positive and negative values indicate trimming and extension of reads, respectively. C) Percentage of reads with mono-tails and oligo-tails for HEK293T pre-miRNA-seq and miRNA-seq reads mapped to high confidence human miRBase miRNAs. D-E) Non-templated 3[[#x2019]] end tail length and sequence content for pre-miRNA-seq (D) and miRNA-seq (E) reads mapping to miRBase high confidence human miRNAs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-814844358" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;pre-miRNA-seq identifies diverse ends of pre-miRNAs&lt;;/span&gt;; A) Distribution of pre-miRNA-seq read ends relative to annotated 5[[#x2019]] end of 5p and 3[[#x2019]] end of 3p miRNAs for high confidence human miRNAs. Positive and negative values indicate trimming and extension of reads, respectively. B) Distribution of miRNA-seq read ends relative to annotated ends of 5p and 3p miRNAs for high confidence human miRNAs. Positive and negative values indicate trimming and extension of reads, respectively. C) Percentage of reads with mono-tails and oligo-tails for HEK293T pre-miRNA-seq and miRNA-seq reads mapped to high confidence human miRBase miRNAs. D-E) Non-templated 3[[#x2019]] end tail length and sequence content for pre-miRNA-seq (D) and miRNA-seq (E) reads mapping to miRBase high confidence human miRNAs.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F2.medium.gif" width="440" height="289"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F2.medium.gif" width="440" height="289"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 2.</span>
       <span class="caption-title">pre-miRNA-seq identifies diverse ends of pre-miRNAs</span>
       <p id="p-29" class="first-child">A) Distribution of pre-miRNA-seq read ends relative to annotated 5’ end of 5p and 3’ end of 3p miRNAs for high confidence human miRNAs. Positive and negative values indicate trimming and extension of reads, respectively. B) Distribution of miRNA-seq read ends relative to annotated ends of 5p and 3p miRNAs for high confidence human miRNAs. Positive and negative values indicate trimming and extension of reads, respectively. C) Percentage of reads with mono-tails and oligo-tails for HEK293T pre-miRNA-seq and miRNA-seq reads mapped to high confidence human miRBase miRNAs. D-E) Non-templated 3’ end tail length and sequence content for pre-miRNA-seq (D) and miRNA-seq (E) reads mapping to miRBase high confidence human miRNAs.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-30">Pre-miRNAs and miRNAs are post-transcriptionally modified at their 3’ ends through the action of TUT4 and TUT7 terminal uriydyl transferases (TUTases) (
      <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">17</a>,
      <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>,
      <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>,
      <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>). We found that 14.5% of human and 17.4% of mouse pre-miRNAs contained single nucleotide additions to their 3’ ends, whereas 4.9% and 7.1% had more than two non-templated additions on their 3’ ends in human and mouse high confidence miRBase-annotated miRNAs, respectively (
      <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figs. 2C</a> and 
      <a id="xref-fig-9-10" class="xref-fig" href="#F9">S2F</a>). For mature miRNAs, we also found a large number of single nucleotide additions at the 3’ end; 8.4% of human and 12.5% of mouse mature miRNAs. However, tails greater than one nucleotide were much less frequent, with only 1.1% of human and 2.2% of mouse mature miRNAs containing long tails. The lower overall modification level of mature miRNAs is consistent with the hypothesis that oligo-uridylation is a signal for degradation, whereas mono-uridylation establishes the 3’ 2 nt overhang, required for efficient DICER cleavage.
     </p>
     <p id="p-31">For cellular smRNA-seq from HEK293T cells, we observed lower levels of single nucleotide additions (4.0%) and higher levels of multiple non-templated 3’ additions (3.3%), compared to AGO-bound miRNAs (8.4% and 1.1% respectively in human). (
      <a id="xref-fig-8-3" class="xref-fig" href="#F8">Fig. S1D</a>). These differences may indicate that short oligo-nucleotide additions, which in some cases can promote pre-miRNA processing, could in fact prevent proper AGO loading of the 3’ mature miRNA. Collectively, these data demonstrate that non-templated additions are widespread in both pre-miRNAs and mature miRNAs, and that extended tails are more frequent in pre-miRNAs compared to mature miRNAs.
     </p>
     <p id="p-32">We also generated metaplots to analyze the sequence content of non-templated additions to the 3’ end of pre-miRNAs (
      <a id="xref-fig-2-8" class="xref-fig" href="#F2">Figs. 2D</a> and 
      <a id="xref-fig-9-11" class="xref-fig" href="#F9">S2G</a>) and mature miRNAs (
      <a id="xref-fig-2-9" class="xref-fig" href="#F2">Figs. 2E</a>, 
      <a id="xref-fig-8-4" class="xref-fig" href="#F8">S1E</a>, and 
      <a id="xref-fig-9-12" class="xref-fig" href="#F9">S2H</a>). Uridine (denoted as T) was by far the most common addition to human and mouse pre-miRNAs, and was even more prevalent in positions past the first non-templated nucleotide (
      <a id="xref-fig-2-10" class="xref-fig" href="#F2">Figs. 2D</a> and 
      <a id="xref-fig-9-13" class="xref-fig" href="#F9">S2G</a>). For mature miRNAs, adenosine was the most common non-templated addition, with much lower levels of uridine as compared to pre-miRNAs (
      <a id="xref-fig-2-11" class="xref-fig" href="#F2">Figs 2E</a>, 
      <a id="xref-fig-8-5" class="xref-fig" href="#F8">S1E</a>, and 
      <a id="xref-fig-9-14" class="xref-fig" href="#F9">S2H</a>). When examining non-templated additions in total smRNA-seq data, we found that 2 nucleotide tails were much more common than in AGO-bound miRNAs (
      <a id="xref-fig-8-6" class="xref-fig" href="#F8">Fig. S1E</a>). Collectively, these data show that mono- and oligo-tailing are widespread in human and mouse pre- and mature miRNAs. Furthermore, they reveal that uridylation is more common in pre-miRNAs, especially after the first nucleotide, whereas adenylation is more common in mature miRNAs.
     </p>
    </div>
    <div id="sec-17" class="subsection">
     <h3>Identification of AGO2-cleaved pre-miRNAs</h3>
     <p id="p-33">AGO2, is unique amongst the mammalian AGO proteins in that it has slicing activity (
      <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>–
      <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>). In fact, AGO2 has been demonstrated to cleave a subset of pre-miRNAs, named AGO-cleaved pre-miRNAs (ac-pre-miRNAs) (
      <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>). The best characterized of the ac-pre-miRNAs, pre-miR-451, is trimmed by PARN following AGO2 cleavage to generate the mature miRNA sequence (
      <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>,
      <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>,
      <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>,
      <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>). However, the extent to which this process occurs in mammalian pre-miRNA populations has not been determined. AGO2 cleavage events are known to occur around 10 nucleotide upstream of the 3’ end of the pre-miRNA and give rise to 5p miRNAs (
      <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>). To search for these events, we calculated the percentage of pre-miRNA-seq reads that terminated 8-15 nucleotide from the 3’ end of the 3p miRNA for each miRNA. In mouse, for which ac-pre-miRNAs are best characterized, we observed all previously identified ac-pre-miRNAs with the exception of miR-451, which is not expressed in MEFs (
      <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). In our human dataset, we also observed previously identified ac-pre-miRNAs, including 9 members of the let-7 family and miR-9-2 (
      <strong>Table S3</strong>). From this approach, we identified 7 putative ac-pre-miRNA candidates in humans and 37 in mouse, including hsa-miR-455 and mmu-miR-335 (
      <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figs. 3A-B</a> and 
      <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Tables 1</a> and S3).
     </p>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="De novo identification of AGO2-cleaved pre-miRNAs A-D) Examples of novel ac-pre-miRNAs identified from pre-miRNA-seq. hsa-miR-455 (A) and mmu-miR-335 (B) are novel ac-pre-miRNAs with cleavage in the 3p miRNA. hsa-miR-140 (C) and mmu-miR-22 (D) are novel ac-pre-miRNAs with cleavage in the 5p miRNA. White bars indicate templated nucleotides, colored bars indicate non-templated additions. Dashed red and blue lines indicate boundaries of annotated 5p (blue) and 3p (red) miRNAs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-814844358" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;&lt;;em&gt;;De novo&lt;;/em&gt;; identification of AGO2-cleaved pre-miRNAs&lt;;/span&gt;; A-D) Examples of novel ac-pre-miRNAs identified from pre-miRNA-seq. hsa-miR-455 (A) and mmu-miR-335 (B) are novel ac-pre-miRNAs with cleavage in the 3p miRNA. hsa-miR-140 (C) and mmu-miR-22 (D) are novel ac-pre-miRNAs with cleavage in the 5p miRNA. White bars indicate templated nucleotides, colored bars indicate non-templated additions. Dashed red and blue lines indicate boundaries of annotated 5p (blue) and 3p (red) miRNAs.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F3.medium.gif" width="440" height="389"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F3.medium.gif" width="440" height="389"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 3.</span>
       <span class="caption-title">
        <em>De novo</em> identification of AGO2-cleaved pre-miRNAs
       </span>
       <p id="p-34" class="first-child">A-D) Examples of novel ac-pre-miRNAs identified from pre-miRNA-seq. hsa-miR-455 (A) and mmu-miR-335 (B) are novel ac-pre-miRNAs with cleavage in the 3p miRNA. hsa-miR-140 (C) and mmu-miR-22 (D) are novel ac-pre-miRNAs with cleavage in the 5p miRNA. White bars indicate templated nucleotides, colored bars indicate non-templated additions. Dashed red and blue lines indicate boundaries of annotated 5p (blue) and 3p (red) miRNAs.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/459822/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup last">
          <a href="/highwire/markup/459822/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 1:</span>
       <p id="p-35" class="first-child">Known and putative ac-pre-miRNAs identified in MEFs</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-36">All previous examples of ac-pre-miRNAs have been shown to be cleaved within the 3p miRNA sequence and generate 5p miRNAs. To identify putative ac-pre-miRNAs that are processed with the opposite directionality, we performed a parallel search for pre-miRNA-seq reads that began 8-15 nucleotide from the 5’ end of the 5p miRNA. From this analysis, we found 3 candidates in humans and 5 candidates in mouse including hsa-miR-140 and mmu-miR-22 (
      <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figs. 3C-D</a>
      <strong>and Table S3</strong>). Importantly, these 5p cleaved ac-pre-miRNAs give rise predominantly to 3p miRNAs suggesting that cleavage in this region is not a degradation byproduct and is likely a competent mechanism for generating mature miRNAs (
      <strong>Table S3</strong>). These results reveal a greater collection of ac-pre-miRNAs than previously appreciated and uncover a likely novel class of 5p cleaved ac-pre-miRNAs.
     </p>
    </div>
    <div id="sec-18" class="subsection">
     <h3>Relating pre-miRNA and mature miRNA abundance</h3>
     <p id="p-37">Almost nothing is known about the relationship between pre-miRNA and mature miRNA abundance. In order to assess this, we first grouped miRNAs into their families and merged the mapped reads (
      <strong>Table S4</strong>). This was necessary to avoid artifacts, given that multiple distinct precursors give rise to identical, or nearly identical mature miRNAs. We further refined our analysis by only focusing on high confidence miRNAs, which left us with 158 human and 159 mouse miRNAs with reads mapping from pre-miRNA-seq and/or miRNA-seq libraries (
      <strong>Table S5</strong>). We analyzed the correlation between pre-miRNAs and mature miRNA levels expressed from these families (
      <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figs. 4A-B</a>). We found a positive but modest correlation between pre-miRNA and mature miRNA expression in both humans (Spearman correlation R = 0.53, p-value &lt;; 6.635e
      <sup>−08</sup>) and mouse (Spearman correlation R = 0.52, p-value &lt;; 2.047e
      <sup>−07</sup>). We also assessed the relationship between smRNA-seq and pre-miRNA-seq in HEK293T cells (
      <a id="xref-fig-10-1" class="xref-fig" href="#F10">Fig. S3A</a>). We found a strikingly similar correlation to that of miRNA-seq (Spearman correlation; R = 0.53, p-value &lt;; 4.331e
      <sup>−08</sup>), which is explained by the high correlation of smRNA-seq to miRNA-seq (Spearman correlation; R = 0.90, p-value &lt;; 2.2e
      <sup>−16</sup>) (
      <a id="xref-fig-10-2" class="xref-fig" href="#F10">Fig. S3B</a>). Together, these results reveal that in general pre-miRNA and mature miRNA levels have a modest positive correlation in both mouse and humans.
     </p>
     <div id="F10" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F10.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-814844358" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure10" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F10.medium.gif" width="440" height="141"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure10" src="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F10.medium.gif" width="440" height="141"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure10" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
     </div>
     <div id="F4" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="An index for miRNA precursor processing efficiency A) Correlation of pre-miRNA-seq and miRNA-seq for high confidence human (A) and mouse (B) miRNA loci (human: R = 0.53, p-value &lt;; 6.635e[[#x2212]]08, mouse: R = 0.52, p-value &lt;; 2.047e[[#x2212]]07; Spearman correlation coefficient). C-D) Rank ordered list of mature to precursor abundance ratio (MPAR) scores for high confidence human (C) and mouse (D) miRNAs. E-G) Small RNA Northern blots of select miRNAs with low MPAR scores from human (HEK293T) or mouse (MEF) lysates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-814844358" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;An index for miRNA precursor processing efficiency&lt;;/span&gt;; A) Correlation of pre-miRNA-seq and miRNA-seq for high confidence human (A) and mouse (B) miRNA loci (human: R = 0.53, p-value &lt;; 6.635e&lt;;sup&gt;;[[#x2212]]08&lt;;/sup&gt;;, mouse: R = 0.52, p-value &lt;; 2.047e&lt;;sup&gt;;[[#x2212]]07&lt;;/sup&gt;;; Spearman correlation coefficient). C-D) Rank ordered list of mature to precursor abundance ratio (MPAR) scores for high confidence human (C) and mouse (D) miRNAs. E-G) Small RNA Northern blots of select miRNAs with low MPAR scores from human (HEK293T) or mouse (MEF) lysates.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F4.medium.gif" width="322" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F4.medium.gif" width="322" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 4.</span>
       <span class="caption-title">An index for miRNA precursor processing efficiency</span>
       <p id="p-38" class="first-child">A) Correlation of pre-miRNA-seq and miRNA-seq for high confidence human (A) and mouse (B) miRNA loci (human: R = 0.53, p-value &lt;; 6.635e
        <sup>−08</sup>, mouse: R = 0.52, p-value &lt;; 2.047e
        <sup>−07</sup>; Spearman correlation coefficient). C-D) Rank ordered list of mature to precursor abundance ratio (MPAR) scores for high confidence human (C) and mouse (D) miRNAs. E-G) Small RNA Northern blots of select miRNAs with low MPAR scores from human (HEK293T) or mouse (MEF) lysates.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-19" class="subsection">
     <h3>An index for miRNA precursor processing</h3>
     <p id="p-39">We reasoned that the ratio of mature miRNAs to pre-miRNAs represents a reasonable estimate for 
      <em>in vivo</em> miRNA processing efficiency. As some miRNAs had no detectable pre-miRNA reads or vice versa, we used a generalized log odds ratio of miRNA-seq to pre-miRNA-seq reads (
      <strong>see METHODS</strong>) to compute a mature to precursor abundance ratio (MPAR) for miRNAs expressed in human HEK293T and mouse MEF cells (
      <strong>Tables S4 and S5</strong>). As expected, the majority of high confidence miRNAs from human and mouse cells exhibited MPAR values &gt;; 0 (
      <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figs. 4C-D</a> and 
      <a id="xref-fig-10-3" class="xref-fig" href="#F10">S3C</a>), suggesting they are in general efficiently processed. Among the maximum scoring high confidence miRNAs in humans was hsa-miR-338, which had no detectable pre-miRNA-seq reads and 1,358 reads per million (RPM) in miRNA-seq libraries (MPAR = 11.4) (
      <strong>Table S5</strong>). In contrast, we found a number of miRNAs that had many more pre-miRNA-seq reads than mature miRNA-seq reads. To validate that our MPAR predictions are representative, we performed northern blotting on one human (hsa-miR-1296) and two mouse (mmu-miR-671 and mmu-miR-503) miRNAs with low MPAR scores (
      <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figs. 4E-G</a>). In each case, we observed bands corresponding to both the mature and precursor miRNA at similar levels. In comparison to the overall levels of pre-miRNA to mature miRNA (
      <a id="xref-fig-1-8" class="xref-fig" href="#F1">Fig. 1B</a>) these miRNAs show a relative enrichment towards the pre-miRNA, indicating that they are inefficiently processed to mature miRNAs. Thus, examining the ratio of mature to pre-miRNAs allows us to approximate the efficiency of miRNA processing for hundreds of miRNAs in a single experiment.
     </p>
    </div>
    <div id="sec-20" class="subsection">
     <h3>Distinct processing of two pre-miRNAs from Dgcr8 mRNA</h3>
     <p id="p-40">The microprocessor complex (DGCR8/DROSHA) auto-regulates the expression of the 
      <em>Dgcr8</em> transcript by binding and cleaving two hairpins near the 5’ end of its mRNA (
      <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>–
      <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>). We re-examined the abundance, modification status, and MPAR scores for the two annotated miRNAs in this region (
      <a id="xref-fig-11-1" class="xref-fig" href="#F11">Fig. S4</a>). We determined that hsa-miR-3618, which is encoded in the 5’ untranslated region (UTR) of 
      <em>Dgcr8</em>, is inefficiently processed into mature miRNAs (MPAR = −5.22), whereas hsa-miR-1306, which lies in the coding sequence (CDS), had a high MPAR score (MPAR = 3.09) (
      <a id="xref-fig-11-2" class="xref-fig" href="#F11">Figs. S4A-D</a>
      <strong>and Table S4</strong>). Furthermore, we found that less than 10% of hsa-miR-3618 pre-miRNAs contained non-templated tails, whereas 100% of the hsa-miR-1306 clones were mono-uridylated, a signal that has previously been linked to efficient processing in some miRNAs (
      <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">18</a>). Thus, divergent processing of two miRNAs from the same primary transcript underscores the selectivity of DICER processing, and suggests that these two hairpins likely have evolved distinct functions in post-transcriptional regulation.
     </p>
     <div id="F11" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F11.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-814844358" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure11" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F11.medium.gif" width="440" height="393"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure11" src="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F11.medium.gif" width="440" height="393"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure11" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
     </div>
    </div>
    <div id="sec-21" class="subsection">
     <h3>Identification of AGO-associated stem-loops derived from mRNAs</h3>
     <p id="p-41">Given this observation, we examined our dataset for novel pre-miRNAs embedded in other mRNAs. To do this, we took a conservative approach, using pre-miRNA-seq reads that failed to map to miRBase, and filtering them by mapping to small nuclear RNAs (snRNAs), snoRNAs, tRNAs, ribosomal RNAs (rRNAs), and repeat-masked sequences (
      <a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5A</a>, 
      <strong>see Methods</strong>). Approximately 1.2 and 0.37 million clones passed our stringent filtering steps and mapped to human and mouse mRNAs, respectively (
      <strong>Table S1</strong>). To identify pre-miRNA-like AGO-associated stem-loops derived from mRNAs, we used a CLIP-seq peak calling approach to identify significant (modified false discovery rate (mFDR) &lt;; 0.01) read clusters in mRNAs (
      <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">35</a>). We found a number of significant peaks that corresponded to nearly the full length of some highly expressed genes (e.g 
      <em>ACTB</em>). Therefore, we further filtered significant clusters based on their size (&lt;; 200 nt), then chose the top clone from each cluster, folded it using RNAfold, and captured clusters that had a minimum free energy (MFE) &lt;; −0.3 kcal/mol/nt and a minimum of 15 base pairs (bp) in the longest hairpin (
      <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">36</a>).
     </p>
     <div id="F5" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="AGO-associated stem-loops derived from mRNAs A) Bioinformatics pipeline for identification of AGO-associated stem-loops derived from mRNAs. Pre-miRNA-seq reads that did not map to miRBase were filtered on ncRNA and repeat-masker (RMSK), trimmed to 35 nt and mapped to spliced mRNA sequences. These alignments were then extended to find all matching bases. AGO-associated stem-loops derived from mRNAs were identified using a CLIP-seq peak caller (mFDR &lt;; 0.01) followed by length filtering (&lt;; 200 nt), MFE (&lt;; [[#x2212]]0.3 kcal/mol/nt), and paired bases (&gt;; 15 bp/hairpin). B) Distribution of AGO-associated stem-loops derived from mRNAs. C) Enrichment of AGO-associated stem-loops derived from mRNAs within mRNA regions relative to their genomic coverage. D-E) Size distribution of pre-miRNA-seq reads from human (D) and mouse experiments (E) mapping to AGO-associated stem-loops derived from mRNAs. F-G) Non-templated 3[[#x2019]] end tail length and sequence content for pre-miRNA-seq reads mapping to human (F) and mouse (G) AGO-associated stem-loops derived from mRNAs. H) Western blot of DGCR8, DROSHA, and TUBB in HEK293T cells following siRNA knockdown. I) RT-qPCR analysis of mRNAs hosting AGO-associated stem-loops following knockdown of DGCR8 or DROSHA in HEK293T cells. ** = p-value &lt;; 0.01, *** = p-value &lt;; 0.001; Students[[#x2019]] t-test." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-814844358" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;AGO-associated stem-loops derived from mRNAs&lt;;/span&gt;; A) Bioinformatics pipeline for identification of AGO-associated stem-loops derived from mRNAs. Pre-miRNA-seq reads that did not map to miRBase were filtered on ncRNA and repeat-masker (RMSK), trimmed to 35 nt and mapped to spliced mRNA sequences. These alignments were then extended to find all matching bases. AGO-associated stem-loops derived from mRNAs were identified using a CLIP-seq peak caller (mFDR &lt;; 0.01) followed by length filtering (&lt;; 200 nt), MFE (&lt;; [[#x2212]]0.3 kcal/mol/nt), and paired bases (&gt;; 15 bp/hairpin). B) Distribution of AGO-associated stem-loops derived from mRNAs. C) Enrichment of AGO-associated stem-loops derived from mRNAs within mRNA regions relative to their genomic coverage. D-E) Size distribution of pre-miRNA-seq reads from human (D) and mouse experiments (E) mapping to AGO-associated stem-loops derived from mRNAs. F-G) Non-templated 3[[#x2019]] end tail length and sequence content for pre-miRNA-seq reads mapping to human (F) and mouse (G) AGO-associated stem-loops derived from mRNAs. H) Western blot of &lt;;em&gt;;DGCR8&lt;;/em&gt;;, &lt;;em&gt;;DROSHA&lt;;/em&gt;;, and &lt;;em&gt;;TUBB&lt;;/em&gt;; in HEK293T cells following siRNA knockdown. I) RT-qPCR analysis of mRNAs hosting AGO-associated stem-loops following knockdown of &lt;;em&gt;;DGCR8&lt;;/em&gt;; or &lt;;em&gt;;DROSHA&lt;;/em&gt;; in HEK293T cells. ** = p-value &lt;; 0.01, *** = p-value &lt;; 0.001; Students[[#x2019]] t-test.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F5.medium.gif" width="353" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F5.medium.gif" width="353" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 5.</span>
       <span class="caption-title">AGO-associated stem-loops derived from mRNAs</span>
       <p id="p-42" class="first-child">A) Bioinformatics pipeline for identification of AGO-associated stem-loops derived from mRNAs. Pre-miRNA-seq reads that did not map to miRBase were filtered on ncRNA and repeat-masker (RMSK), trimmed to 35 nt and mapped to spliced mRNA sequences. These alignments were then extended to find all matching bases. AGO-associated stem-loops derived from mRNAs were identified using a CLIP-seq peak caller (mFDR &lt;; 0.01) followed by length filtering (&lt;; 200 nt), MFE (&lt;; −0.3 kcal/mol/nt), and paired bases (&gt;; 15 bp/hairpin). B) Distribution of AGO-associated stem-loops derived from mRNAs. C) Enrichment of AGO-associated stem-loops derived from mRNAs within mRNA regions relative to their genomic coverage. D-E) Size distribution of pre-miRNA-seq reads from human (D) and mouse experiments (E) mapping to AGO-associated stem-loops derived from mRNAs. F-G) Non-templated 3’ end tail length and sequence content for pre-miRNA-seq reads mapping to human (F) and mouse (G) AGO-associated stem-loops derived from mRNAs. H) Western blot of 
        <em>DGCR8</em>, 
        <em>DROSHA</em>, and 
        <em>TUBB</em> in HEK293T cells following siRNA knockdown. I) RT-qPCR analysis of mRNAs hosting AGO-associated stem-loops following knockdown of 
        <em>DGCR8</em> or 
        <em>DROSHA</em> in HEK293T cells. ** = p-value &lt;; 0.01, *** = p-value &lt;; 0.001; Students’ t-test.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-43">This resulted in 403 and 373 AGO-associated stem-loops derived from human and mouse mRNAs, respectively (
      <strong>Table S6</strong>). We intersected our list of AGO-associated stem-loops derived from human mRNAs with recently identified miRNAs in humans from extensive smRNA-seq or from DICER photoactivatable-ribonucleoside-enhanced crosslinking and immunoprecipitation (PAR-CLIP) (
      <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a>,
      <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>). In fact, 12 of our AGO-associated stem-loops were annotated as novel miRNAs in these lists supporting the ability of our approach to identify novel miRNAs (
      <a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5A</a>). Furthermore, 34 and 37 AGO-associated stem-loops overlapped with DICER and DGCR8 binding sites respectively, revealing that a number of the AGO-associated stem-loops derived from mRNAs interact with other components of the canonical miRNA processing pathway (
      <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">55</a>,
      <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>).
     </p>
     <p id="p-44">We next examined the distribution of AGO-associated stem-loops derived from mRNAs across mRNA regions and found that they were equally present in all regions of mRNAs and similarly distributed between human and mouse (
      <a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig. 5B</a>). When we normalized the distribution of AGO-associated stem-loops derived from mRNAs by the relative genomic coverage of each mRNA region, we found that the CDS was underrepresented, whereas the 5’ UTR was enriched 2.5 fold for AGO-associated stem-loops in both organisms (
      <a id="xref-fig-5-4" class="xref-fig" href="#F5">Fig. 5C</a>). We also examined the size distribution of reads mapping to AGO-associated stem-loops derived from mRNAs and found them to be similar in size in both mammals, between 55-75 nt in humans and 52-80 nt in mouse (
      <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figs. 5D-E</a>). This size range was slightly broader than pre-miRNA-seq reads that mapped to miRBase (
      <a id="xref-fig-1-9" class="xref-fig" href="#F1">Figs. 1E-F</a>).
     </p>
     <p id="p-45">We also analyzed non-templated 3’ additions to pre-miRNA-seq reads mapped to AGO-associated stem-loops derived from human and mouse mRNAs (
      <a id="xref-fig-5-6" class="xref-fig" href="#F5">Figs. 5F-G</a>
      <strong>and Table S7</strong>). We found that pre-miRNA-seq reads which mapped to these elements were enriched for uridylation events, and in fact, a higher percentage were oligo-tailed (13.0% in human and 24.3% in mouse), compared to reads mapping to miRBase (
      <a id="xref-fig-2-12" class="xref-fig" href="#F2">Figs. 2C</a> and 
      <a id="xref-fig-9-15" class="xref-fig" href="#F9">S2F</a>). This result suggests that the AGO-associated stem-loops derived from mRNAs identified by our approach undergo similar modifications to known pre-miRNAs.
     </p>
     <p id="p-46">Finally, we overlaid 
      <em>in vivo</em> RNA structure-probing data from mouse embryonic stem (ES) cells generated with 
      <em>in vivo</em> click selective 2’-hydroxyl acylation and profiling (icSHAPE) onto our AGO-associated stem-loops derived from mouse mRNAs (
      <a id="xref-fig-12-1" class="xref-fig" href="#F12">Fig. S5</a>). icSHAPE, chemically modifies the backbone of unpaired nucleotides, causing early termination of reverse transcriptase (
      <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>). Based on our RNAfold predictions, we examined the icSHAPE reactivity at paired or unpaired bases and found that icSHAPE reactivity was significantly higher at unpaired bases (Kruskal-Wallis one-way ANOVA; p-value &lt;; 8.18e
      <sup>−133</sup>) (
      <a id="xref-fig-12-2" class="xref-fig" href="#F12">Fig. S5A</a>). This can been seen clearly when examining the RNAfold structure diagrams of AGO-associated stem-loops with icSHAPE reactivity overlaid (
      <a id="xref-fig-12-3" class="xref-fig" href="#F12">Figs. S5B-D</a>). Together, our analyses provide numerous candidate pre-miRNA-like elements that are processed into 50-80 nt AGO-associated stem-loops from mRNAs in mammalian cells.
     </p>
     <div id="F12" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F12.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-814844358" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure12" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F12.medium.gif" width="440" height="218"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure12" src="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F12.medium.gif" width="440" height="218"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure12" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
     </div>
    </div>
    <div id="sec-22" class="subsection">
     <h3>AGO-associated stem-loop host mRNAs are generally not regulated by the microprocessor complex</h3>
     <p id="p-47">To assess whether AGO-associated stem-loop containing mRNAs are regulated by the microprocessor complex, we performed siRNA-mediated knockdown of 
      <em>DROSHA</em> and 
      <em>DGCR8</em> in HEK293T cells and assessed changes in gene expression (
      <a id="xref-fig-5-7" class="xref-fig" href="#F5">Figs. 5H-J</a>). We observed robust knockdown of both DROSHA and DGCR8 protein levels (
      <a id="xref-fig-5-8" class="xref-fig" href="#F5">Fig. 5H</a>). Consistent with the role of DROSHA in regulating DGCR8 expression, we found that knockdown of 
      <em>DROSHA</em> increased DGCR8 protein and RNA expression (
      <a id="xref-fig-5-9" class="xref-fig" href="#F5">Fig. 5H-I</a>). To determine whether DGCR8 and/or DROSHA participate in the regulation of AGO-associated stem-loop containing mRNAs, we performed mRNA-seq and RT-qPCR on knockdown RNA samples. To our surprise, the majority of mRNAs hosting AGO-associated stem-loops were unaffected by knockdown of components of the microprocessor. (
      <strong>Table S7 and</strong>
      <a id="xref-fig-5-10" class="xref-fig" href="#F5">Fig. 5I</a>). In fact, the only gene tested by RT-qPCR that was significantly affected was 
      <em>GLUL,</em> but knockdown of either DGCR8 or DROSHA decreased the level of 
      <em>GLUL</em> expression, in contrast to the expected role of the microprocessor. These data suggest that cleavage of AGO-associated stem-loops does not have a significant impact on the expression of the host mRNA because cleavage events are rare, or because this process is independent of the microprocessor, with neither model being mutually exclusive.
     </p>
    </div>
    <div id="sec-23" class="subsection">
     <h3>Most AGO-associated stem-loops derived from mRNAs do not produce AGO-bound smRNAs</h3>
     <p id="p-48">We next calculated the miRNA-seq and smRNA-seq coverage at AGO-associated stem-loops derived from mRNAs. We found that miRNA-seq and smRNA-seq reads mapping to AGO-associated stem-loops derived from mRNAs were of a similar size as miRBase miRNAs and had non-templated additions to their 3’ ends (
      <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figs. 6A-B</a> and 
      <a id="xref-fig-13-1" class="xref-fig" href="#F13">S6A-D</a>). However, when we calculated the MPAR scores for AGO-associated stem-loops derived from mRNAs, we found that they had overall very low scores, indicating that they are poor substrates for producing mature AGO-bound smRNAs (
      <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figs. 6C</a> and 
      <a id="xref-fig-13-2" class="xref-fig" href="#F13">S6E-F</a>
      <strong>and Table S8</strong>). This is in agreement with the finding that these elements are more commonly oligo-tailed compared to canonical pre-miRNAs. In fact, we only found a total of 171 miRNA-seq reads mapping to AGO-associated stem-loops derived from human mRNAs and the majority of these reads mapped to the recently identified miRNAs embedded within the mRNAs; 
      <em>GNAS, GLUL</em>, and 
      <em>E2F1</em> (
      <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">26</a>,
      <a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">55</a>) (example of E2F1 in 
      <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figs. 6D-E</a>). Additionally, we found a total of 2,240 smRNA-seq reads mapping to AGO-associated mRNA stem-loops derived from human mRNAs, which in part reflects the higher sequencing depth of these libraries. Some of stem-loops with the most smRNA-seq reads, corresponded to previously identified novel miRNAs, including 
      <em>BRD2</em>, 
      <em>GLUL</em> and 
      <em>E2F1</em> (
      <strong>example in</strong>
      <a id="xref-fig-6-4" class="xref-fig" href="#F6">Fig. 6F</a>). However, we also noticed numerous reads mapping to mRNA stem-loops that did not have binding evidence from miRNA-seq, including 
      <em>FTH1</em>, 
      <em>FTL</em>, 
      <em>SOX4</em>, and 
      <em>KMT2C</em>.
     </p>
     <div id="F13" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F13.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-814844358" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure13" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F13.medium.gif" width="314" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure13" src="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F13.medium.gif" width="314" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure13" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
     </div>
     <div id="F6" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Few AGO-associated stem-loops derived from mRNAs produce smRNAs A) Size distribution of miRNA-seq reads mapping to human AGO-associated stem-loops derived from mRNAs. B) Non-templated 3[[#x2019]] end tail length and sequence content for miRNA-seq reads mapping to human AGO-associated stem-loops derived from mRNAs. C) Rank order of MPAR score for AGO-associated stem-loops derived from human mRNAs. D-F) Coverage plot of pre-miRNA-seq (D), miRNA-seq (E), and smRNA-seq (F) reads mapping to a human AGO-associated stem-loop in the 3[[#x2019]] UTR of E2F1. White bars indicate templated nucleotides, colored bars indicate non-templated additions. G-H) Coverage plot of pre-miRNA-seq (G) and miRNA-seq (H) reads mapping to a mouse AGO-associated stem-loop in the CDS of Rpl9. White bars indicate templated nucleotides, colored bars indicate non-templated additions. I) RNAfold predicted structure and icSHAPE reactivity for mouse AGO-associated stem-loop in the CDS of Rpl9. J) Small RNA Northern blot of mouse AGO-associated stem-loop in the CDS of Rpl9 from mouse (MEF) cells. * indicates band at the expected size of miRNA-like product." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-814844358" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Few AGO-associated stem-loops derived from mRNAs produce smRNAs&lt;;/span&gt;; A) Size distribution of miRNA-seq reads mapping to human AGO-associated stem-loops derived from mRNAs. B) Non-templated 3[[#x2019]] end tail length and sequence content for miRNA-seq reads mapping to human AGO-associated stem-loops derived from mRNAs. C) Rank order of MPAR score for AGO-associated stem-loops derived from human mRNAs. D-F) Coverage plot of pre-miRNA-seq (D), miRNA-seq (E), and smRNA-seq (F) reads mapping to a human AGO-associated stem-loop in the 3[[#x2019]] UTR of &lt;;em&gt;;E2F1&lt;;/em&gt;;. White bars indicate templated nucleotides, colored bars indicate non-templated additions. G-H) Coverage plot of pre-miRNA-seq (G) and miRNA-seq (H) reads mapping to a mouse AGO-associated stem-loop in the CDS of &lt;;em&gt;;Rpl9&lt;;/em&gt;;. White bars indicate templated nucleotides, colored bars indicate non-templated additions. I) RNAfold predicted structure and icSHAPE reactivity for mouse AGO-associated stem-loop in the CDS of &lt;;em&gt;;Rpl9&lt;;/em&gt;;. J) Small RNA Northern blot of mouse AGO-associated stem-loop in the CDS of &lt;;em&gt;;Rpl9&lt;;/em&gt;; from mouse (MEF) cells. * indicates band at the expected size of miRNA-like product.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F6.medium.gif" width="440" height="433"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F6.medium.gif" width="440" height="433"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 6.</span>
       <span class="caption-title">Few AGO-associated stem-loops derived from mRNAs produce smRNAs</span>
       <p id="p-49" class="first-child">A) Size distribution of miRNA-seq reads mapping to human AGO-associated stem-loops derived from mRNAs. B) Non-templated 3’ end tail length and sequence content for miRNA-seq reads mapping to human AGO-associated stem-loops derived from mRNAs. C) Rank order of MPAR score for AGO-associated stem-loops derived from human mRNAs. D-F) Coverage plot of pre-miRNA-seq (D), miRNA-seq (E), and smRNA-seq (F) reads mapping to a human AGO-associated stem-loop in the 3’ UTR of 
        <em>E2F1</em>. White bars indicate templated nucleotides, colored bars indicate non-templated additions. G-H) Coverage plot of pre-miRNA-seq (G) and miRNA-seq (H) reads mapping to a mouse AGO-associated stem-loop in the CDS of 
        <em>Rpl9</em>. White bars indicate templated nucleotides, colored bars indicate non-templated additions. I) RNAfold predicted structure and icSHAPE reactivity for mouse AGO-associated stem-loop in the CDS of 
        <em>Rpl9</em>. J) Small RNA Northern blot of mouse AGO-associated stem-loop in the CDS of 
        <em>Rpl9</em> from mouse (MEF) cells. * indicates band at the expected size of miRNA-like product.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-50">For AGO-associated stem-loops derived from mouse mRNAs we mapped 2,698 miRNA-seq reads, however 71% of these mapped to a single locus in the CDS of 
      <em>Rpl9</em> (
      <a id="xref-fig-6-5" class="xref-fig" href="#F6">Figs. 6G-I</a>). Northern blotting with a probe complementary to this sequence revealed a strong signal at ~70 nt, corresponding to the stem-loop, and a weak signal at 25 nt, corresponding to the expected size of the novel miRNA. (
      <a id="xref-fig-6-6" class="xref-fig" href="#F6">Fig. 6J</a>). Therefore, we provide evidence that 
      <em>Rpl9</em> is processed into a stem-loop like structure and further processed, albeit with low efficiency, into a miRNA-like smRNA. Other AGO-bound smRNA producing stem-loops were regions of the 
      <em>Cyr61</em> 5’ UTR, 
      <em>Asf1b</em> 5’ UTR, and 
      <em>Klf9</em> 3’ UTR (
      <a id="xref-fig-14-1" class="xref-fig" href="#F14">Fig. S7</a>). Collectively, these data demonstrate that pre-miRNA-seq uncovers AGO-associated mRNA stem-loops, some of which may represent novel miRNAs, but most of which are poorly processed into AGO-bound smRNAs in the cell lines investigated in this study.
     </p>
     <div id="F14" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F14.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-814844358" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure14" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F14.medium.gif" width="398" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure14" src="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F14.medium.gif" width="398" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure14" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
     </div>
    </div>
    <div id="sec-24" class="subsection">
     <h3>Iron response elements are processed into AGO-associated stem-loops</h3>
     <p id="p-51">The top AGO-associated stem-loop candidate in humans is derived from the 5’ UTR of Ferritin heavy chain (
      <em>FTH1</em>) mRNA (
      <a id="xref-fig-7-1" class="xref-fig" href="#F7">Fig. 7A</a>). We identified over 5,000 pre-miRNA-seq reads in this region, which had a strong predicted hairpin structure, clearly defined ends, and significant mono-tailing on the 3’ end. Intriguingly, this region of the 
      <em>FTH1</em> transcripts corresponds precisely to the iron responsive element (IRE), which is a well-studied structural element that regulates translation in an iron-dependent fashion (
      <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>). We scanned our list of AGO-associated stem-loops for other IRE containing genes, and found Ferritin light chain (
      <em>FTL</em>) mRNA was also producing AGO-associated stem-loops from its IRE region (
      <a id="xref-fig-7-2" class="xref-fig" href="#F7">Fig. 7E</a>). Furthermore, we identified AGO-associated stem-loops supported by icSHAPE data from the mouse homologs of both of these genes, 
      <em>Fth1</em> and 
      <em>Ftl</em> (
      <a id="xref-fig-15-1" class="xref-fig" href="#F15">Figs. S8A-E</a>). Finally, we confirmed the presence of the 
      <em>FTH1</em> pre-miRNA-like structure by northern blotting (
      <a id="xref-fig-7-3" class="xref-fig" href="#F7">Fig. 7D</a>). Therefore, processing of IREs from ferritin genes into AGO-associated stem-loops is conserved in mammals.
     </p>
     <div id="F15" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F15.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-814844358" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure15" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F15.medium.gif" width="440" height="293"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure15" src="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F15.medium.gif" width="440" height="293"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F15.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure15" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F15.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
     </div>
     <div id="F7" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F7.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="The IREs of human FTH1 and FTL are processed into AGO-associated stem-loops A-B) Coverage plot of pre-miRNA-seq (A) and smRNA-seq (B) reads mapping to a human AGO-associated stem loop in the 5[[#x2019]] UTR of the FTH1 gene. White bars indicate templated nucleotides, colored bars indicate non-templated additions. C) Size distribution of smRNA-seq reads mapping to the human AGO-associated stem-loop in the 5[[#x2019]] UTR of FTH1. D) Small RNA Northern blot of human AGO-associated stem-loop in the 5[[#x2019]]UTR of FTH1 from HEK293T cells. * indicates band at the expected size of miRNA-like product. E-G) Coverage plot of pre-miRNA-seq (E) and smRNA-seq (F) reads mapping to a human AGO-associated stem loop in the 5[[#x2019]] UTR of the FTL gene. White bars indicate templated nucleotides, colored bars indicate non-templated additions. G) Size distribution of smRNA-seq reads mapping to the human AGO-associated stem-loop in the 5[[#x2019]] UTR of FTL." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-814844358" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;The IREs of human &lt;;em&gt;;FTH1&lt;;/em&gt;; and &lt;;em&gt;;FTL&lt;;/em&gt;; are processed into AGO-associated stem-loops&lt;;/span&gt;; A-B) Coverage plot of pre-miRNA-seq (A) and smRNA-seq (B) reads mapping to a human AGO-associated stem loop in the 5[[#x2019]] UTR of the &lt;;em&gt;;FTH1&lt;;/em&gt;; gene. White bars indicate templated nucleotides, colored bars indicate non-templated additions. C) Size distribution of smRNA-seq reads mapping to the human AGO-associated stem-loop in the 5[[#x2019]] UTR of &lt;;em&gt;;FTH1&lt;;/em&gt;;. D) Small RNA Northern blot of human AGO-associated stem-loop in the 5[[#x2019]]UTR of &lt;;em&gt;;FTH1&lt;;/em&gt;; from HEK293T cells. * indicates band at the expected size of miRNA-like product. E-G) Coverage plot of pre-miRNA-seq (E) and smRNA-seq (F) reads mapping to a human AGO-associated stem loop in the 5[[#x2019]] UTR of the &lt;;em&gt;;FTL&lt;;/em&gt;; gene. White bars indicate templated nucleotides, colored bars indicate non-templated additions. G) Size distribution of smRNA-seq reads mapping to the human AGO-associated stem-loop in the 5[[#x2019]] UTR of &lt;;em&gt;;FTL&lt;;/em&gt;;.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F7.medium.gif" width="440" height="254"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 7." src="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F7.medium.gif" width="440" height="254"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/06/294488/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 7.</span>
       <span class="caption-title">The IREs of human 
        <em>FTH1</em> and 
        <em>FTL</em> are processed into AGO-associated stem-loops
       </span>
       <p id="p-52" class="first-child">A-B) Coverage plot of pre-miRNA-seq (A) and smRNA-seq (B) reads mapping to a human AGO-associated stem loop in the 5’ UTR of the 
        <em>FTH1</em> gene. White bars indicate templated nucleotides, colored bars indicate non-templated additions. C) Size distribution of smRNA-seq reads mapping to the human AGO-associated stem-loop in the 5’ UTR of 
        <em>FTH1</em>. D) Small RNA Northern blot of human AGO-associated stem-loop in the 5’UTR of 
        <em>FTH1</em> from HEK293T cells. * indicates band at the expected size of miRNA-like product. E-G) Coverage plot of pre-miRNA-seq (E) and smRNA-seq (F) reads mapping to a human AGO-associated stem loop in the 5’ UTR of the 
        <em>FTL</em> gene. White bars indicate templated nucleotides, colored bars indicate non-templated additions. G) Size distribution of smRNA-seq reads mapping to the human AGO-associated stem-loop in the 5’ UTR of 
        <em>FTL</em>.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-53">To examine whether IRE-processed hairpins produce functional small RNAs, we examined miRNA-seq and smRNA-seq data from these regions. We did not find any AGO-interacting small RNAs from human 
      <em>FTH1</em> or 
      <em>FTL</em>. Nevertheless, we did observe cellular smRNAs from these regions in our smRNA-seq data (
      <a id="xref-fig-7-4" class="xref-fig" href="#F7">Figs. 7B,F</a>). However, they were heterogeneous in size and only loosely reflective of DICER processing (
      <a id="xref-fig-7-5" class="xref-fig" href="#F7">Figs. 7C,G</a>). Furthermore, smRNAs corresponding to the FTH1 were detectable by northern blotting but with much lower levels than the pre-miRNA-like structure. (
      <a id="xref-fig-7-6" class="xref-fig" href="#F7">Fig. 7D</a>). Therefore, these smRNAs appear to be rare and likely the consequence of subsequent degradation of the processed stem-loops that are not loaded into AGO to make a functional RISC complex. In mouse, miRNA-seq only demonstrated a small number of clones originating from the 
      <em>Fth</em> pre-miRNA (
      <a id="xref-fig-15-2" class="xref-fig" href="#F15">Figs. S8D</a>), further corroborating our results from human cells. Intriguingly, siRNA-mediated knockdown of 
      <em>DROSHA</em> or 
      <em>DGCR8</em> in HEK23T cells had no effect on 
      <em>FTH1</em> or 
      <em>FTL</em> mRNA expression levels (
      <a id="xref-fig-5-11" class="xref-fig" href="#F5">Fig. 5I</a>). This suggests that processing of IREs into stem-loops is DROSHA-independent and may work through a different endonuclease. In total, these results demonstrate that mammalian IREs are processed into AGO-bound stem-loops through a microprocessor-independent mechanism, and that these stem-loops are unlikely to be substrates for DICER. Whether cleaved IRE stem-loops are functional molecules remains to be determined.
     </p>
    </div>
   </div>
   <div class="section" id="sec-25">
    <h2 class="">DISCUSSION</h2>
    <p id="p-54">Here, we describe the application and further development of a methodology to enrich for and sequence AGO-associated pre-miRNAs in both human and mouse cells. This biochemical approach combined with custom bioinformatics pipelines successfully enriches for and maps pre-miRNAs in mammalian genomes (
     <a id="xref-fig-1-10" class="xref-fig" href="#F1">Fig. 1</a>). Using this approach, we detected 367 pre-miRNAs in human and 267 in mouse cell lines, with ~ 28 and ~ 17 million raw sequencing reads in each experiment, respectively (
     <strong>Tables S1 and S2</strong>). This gave us specific insights into the exact sequence and abundance of pre-miRNAs and miRNAs expressed in cells of two different mammalian organisms. We generated profiles to visualize coverage, trimming, and non-templated tailing at each annotated miRNA expressed in either cell type, which is available for download at 
     <a href="http://gregorylab.bio.upenn.edu/AGO_IP_Seq/">http://gregorylab.bio.upenn.edu/AGO_IP_Seq/</a>.
    </p>
    <p id="p-55">Using these unique datasets, we uncovered widespread trimming and non-templated tailing in both pre- and mature miRNAs from human and mouse cells (
     <a id="xref-fig-2-13" class="xref-fig" href="#F2">Figs. 2</a>, 
     <a id="xref-fig-8-7" class="xref-fig" href="#F8">S1C-D</a> and 
     <a id="xref-fig-9-16" class="xref-fig" href="#F9">S2D-G</a>). We also identified known and putative ac-pre-miRNAs (
     <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figs. 3A-B</a> and 
     <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Tables 1</a>
     <strong>and S3</strong>). The large number of ac-pre-miRNAs identified suggests that DICER-independent pre-miRNA processing may be a more commonly used mechanism than previously appreciated (
     <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">13</a>,
     <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>,
     <a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">19</a>). Furthermore, we identified putative ac-pre-miRNAs that cleave in the 5p arm of the pre-miRNA (
     <strong>Table S3 and</strong>
     <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figs 3C-D</a>), and thus are processed in the opposite direction of the currently known members of this pre-miRNA class. This potentially novel pre-miRNA processing mechanism would require an alternative maturation process, with processive exonucleolytic nucleotide removal occurring step-wise from the 5’ end.
    </p>
    <p id="p-56">Given the unique nature of our datasets, we were able to make the first comprehensive analysis of the relationship between pre-miRNA and mature miRNA abundance in an unbiased fashion (
     <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figs. 4A-B</a> and 
     <a id="xref-fig-10-4" class="xref-fig" href="#F10">S3A</a>). Remarkably, we found very consistent relationships between mature miRNA and pre-miRNA abundance in human and mouse cells. We determined a mature to precursor abundance ratio (MPAR), allowing us to investigate productive and unproductive miRNA maturation (
     <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figs. 4C-D</a> and 
     <a id="xref-fig-10-5" class="xref-fig" href="#F10">S3C</a>). We uncovered some pre-miRNAs that make surprisingly few mature species, despite abundant precursors. We also found that two miRNAs in 
     <em>DGCR8</em> mRNA are processed with highly divergent efficiencies, suggesting distinct functionalities (
     <a id="xref-fig-11-3" class="xref-fig" href="#F11">Fig. S4</a>).
    </p>
    <p id="p-57">From a further examination of pre-miRNA-seq reads that did not map to miRBase, we identified AGO-associated stem-loops derived from mRNAs (
     <a id="xref-fig-5-12" class="xref-fig" href="#F5">Fig. 5A</a>). We found that these were enriched in the 5’ UTR of these transcripts, which suggests they may play a similar role to miR-3618 in the 5’ UTR of human 
     <em>DGCR8</em> (
     <a id="xref-fig-5-13" class="xref-fig" href="#F5">Figs. 5B-C</a>). Furthermore, we found that these stem-loops had a broader size distribution than known miRNAs and were oligo-uridylated, suggesting they are processed by TUTases (
     <a id="xref-fig-5-14" class="xref-fig" href="#F5">Figs. 5D-G</a>). RNA structure prediction algorithms and 
     <em>in vivo</em> structure probing data from mouse ES cells provide strong evidence that these regions of mRNAs form stem-loops (
     <a id="xref-fig-12-4" class="xref-fig" href="#F12">Fig. S5</a>). Surprisingly, AGO-bound stem-loop containing mRNAs were not impacted following siRNA-mediated knockdown of components of the microprocessor complex (
     <a id="xref-fig-5-15" class="xref-fig" href="#F5">Fig. 5I</a>). Our results strongly support the presence of AGO-associated stem-loops from mRNAs, however their biogenesis pathway remains undetermined. Furthermore, we found limited evidence for further processing into mature miRNA-like smRNAs, which may be cell-type or contextdependent and thus should be the target of future investigations.
    </p>
    <p id="p-58">Interestingly, we found very few mature AGO-bound sequences coming from these regions and overall low MPAR scores (
     <a id="xref-fig-6-7" class="xref-fig" href="#F6">Figs. 6A-C</a> and 
     <a id="xref-fig-13-3" class="xref-fig" href="#F13">S6</a>). In human cells, the vast majority of AGO-bound smRNAs from these regions can be explained by recently identified novel miRNAs (
     <a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">26</a>,
     <a id="xref-ref-55-4" class="xref-bibr" href="#ref-55">55</a>). In mouse cells, we uncovered a region in the 
     <em>Rpl9</em> CDS and a few other putative miRNAs that account for most of the AGO-bound smRNAs from these regions (
     <a id="xref-fig-6-8" class="xref-fig" href="#F6">Figs. 6G-I</a> and 
     <a id="xref-fig-14-2" class="xref-fig" href="#F14">S7</a>). These findings indicate that our methodology uncovers novel miRNAs (
     <a id="xref-fig-14-3" class="xref-fig" href="#F14">Fig. S7</a>), but the function of most of these stem-loops, which produce few AGO-bound miRNAs, remains a mystery.
    </p>
    <p id="p-59">We also found that the IRE elements of human and mouse ferritin mRNAs are processed into pre-miRNA-like molecules, but rarely into mature AGO-bound smRNA species (
     <a id="xref-fig-7-7" class="xref-fig" href="#F7">Figs. 7</a> and 
     <a id="xref-fig-15-3" class="xref-fig" href="#F15">S8</a>). Further processing of theses hairpins may only occur in specific cell types (i.e. blood cells) under stress conditions. Moreover, knockdown of 
     <em>DROSHA</em> or 
     <em>DGCR8</em> had no effect on the expression of IRE host genes (
     <a id="xref-fig-5-16" class="xref-fig" href="#F5">Fig. 5I</a>). Then, what is the function of these pre-miRNA-like molecules? IRE hairpins may represent stable remnants of normal degradation of IRE containing mRNAs. However, given the conservation of AGO-associated stem-loops from IREs in both human and mouse cell, this seems unlikely. Alternatively, they may be processed from ferritin host genes by endonucleases other than the DGCR8/DROSHA microprocessor. This would also be consistent with the inability of these pre-miRNA-like molecules to serve as substrates for DICER processing. However, these stem-loops could serve alternative roles, such as acting as an RNA-binding protein sink for IRE-binding proteins or other factors. Undoubtedly, the biogenesis and biological relevance of the IRE and other processed stem-loops will be the subject of further investigation.
    </p>
   </div>
   <div class="section" id="sec-26">
    <h2 class="">DATA AVAILABILITY</h2>
    <p id="p-60">All sequencing data generated in this study has been deposited in GEO under the accession number GSE71710 (available at 
     <a href="http://www.ncbi.nlm.nih.gov/geo/">http://www.ncbi.nlm.nih.gov/geo/</a>). RNAfold diagrams and coverage plots are available for download at 
     <a href="http://gregorylab.bio.upenn.edu/AGO_IP_Seq/">http://gregorylab.bio.upenn.edu/AGO_IP_Seq/</a>. MEF pre-miRNA-seq and miRNA-seq data were downloaded from European Nucleotide Archive (
     <a href="http://www.ebi.ac.uk/ena/">http://www.ebi.ac.uk/ena/</a>) under the accession number PRJEB6756. HEK293T smRNA-seq data was obtained from GSE66224.
    </p>
   </div>
   <div class="section" id="sec-27">
    <h2 class="">FUNDING</h2>
    <p id="p-61">This work was supported by NSF Career Award MCB-1053846 to BDG and NIH Grant GM-072777 to ZM.</p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">ACKNOWLEDGMENTS</h2>
    <p id="p-62">The authors thank members of the Gregory and Mourelatos laboratories for helpful discussions.</p>
   </div>
   <div class="section fn-group" id="fn-group-1">
    <h2>Footnotes</h2>
    <ul>
     <li class="fn-other" id="fn-2">
      <p id="p-2">
       <em>Authors Note</em>: We have deposited raw and processed sequencing data to GEO under the accession GSE71710, which is currently available to reviewers at 
       <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?token=mbursywspvkjxoh&amp;;acc=GSE71710/">http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?token=mbursywspvkjxoh&amp;;acc=GSE71710</a>. We have also created a website to house coverage profiles and RNAfold diagrams generated in this study at 
       <a href="http://gregorylab.bio.upenn.edu/AGO_IP_Seq/">http://gregorylab.bio.upenn.edu/AGO_IP_Seq/</a>
      </p>
     </li>
    </ul>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">REFERENCES</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.1" data-doi="10.1073/pnas.0506482102">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Humphreys</span>, 
          <span class="cit-name-given-names">D.T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Westman</span>, 
          <span class="cit-name-given-names">B.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Martin</span>, 
          <span class="cit-name-given-names">D.I.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Preiss</span>, 
          <span class="cit-name-given-names">T.</span>
         </span> (
         <span class="cit-pub-date">2005</span>) 
         <span class="cit-article-title">MicroRNAs control translation initiation by inhibiting eukaryotic initiation factor 4E/cap and poly(A) tail function</span>. 
         <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>, 
         <span class="cit-vol">102</span>, 
         <span class="cit-fpage">16961</span>–
         <span class="cit-lpage">16966</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0506482102%26rft_id%253Dinfo%253Apmid%252F16287976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAyLzQ3LzE2OTYxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDQvMDYvMjk0NDg4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.2" data-doi="10.1126/science.1215691">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Djuranovic</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nahvi</span>, 
          <span class="cit-name-given-names">A.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Green</span>, 
          <span class="cit-name-given-names">R.</span>
         </span> (
         <span class="cit-pub-date">2012</span>) 
         <span class="cit-article-title">miRNA-mediated gene silencing by translational repression followed by mRNA deadenylation and decay</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>, 
         <span class="cit-vol">336</span>, 
         <span class="cit-fpage">237</span>–
         <span class="cit-lpage">240</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDjuranovic%26rft.auinit1%253DS.%26rft.volume%253D336%26rft.issue%253D6078%26rft.spage%253D237%26rft.epage%253D240%26rft.atitle%253DmiRNA-Mediated%2BGene%2BSilencing%2Bby%2BTranslational%2BRepression%2BFollowed%2Bby%2BmRNA%2BDeadenylation%2Band%2BDecay%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1215691%26rft_id%253Dinfo%253Apmid%252F22499947%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzYvNjA3OC8yMzciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNC8wNi8yOTQ0ODguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.3" data-doi="10.1038/nature03120">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gregory</span>, 
          <span class="cit-name-given-names">R.I.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yan</span>, 
          <span class="cit-name-given-names">K.P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Amuthan</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chendrimada</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Doratotaj</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cooch</span>, 
          <span class="cit-name-given-names">N.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Shiekhattar</span>, 
          <span class="cit-name-given-names">R.</span>
         </span> (
         <span class="cit-pub-date">2004</span>) 
         <span class="cit-article-title">The Microprocessor complex mediates the genesis of microRNAs</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>, 
         <span class="cit-vol">432</span>, 
         <span class="cit-fpage">235</span>–
         <span class="cit-lpage">240</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGregory%26rft.auinit1%253DR.%2BI.%26rft.volume%253D432%26rft.issue%253D7014%26rft.spage%253D235%26rft.epage%253D240%26rft.atitle%253DThe%2BMicroprocessor%2Bcomplex%2Bmediates%2Bthe%2Bgenesis%2Bof%2BmicroRNAs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03120%26rft_id%253Dinfo%253Apmid%252F15531877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature03120&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15531877&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000225020200050&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.4" data-doi="10.1038/nature01957">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lee</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ahn</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Han</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Choi</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kim</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yim</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lee</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Provost</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Radmark</span>, 
          <span class="cit-name-given-names">O.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kim</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2003</span>) 
         <span class="cit-article-title">The nuclear RNase III Drosha initiates microRNA processing</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>, 
         <span class="cit-vol">425</span>, 
         <span class="cit-fpage">415</span>–
         <span class="cit-lpage">419</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLee%26rft.auinit1%253DY.%26rft.volume%253D425%26rft.issue%253D6956%26rft.spage%253D415%26rft.epage%253D419%26rft.atitle%253DThe%2Bnuclear%2BRNase%2BIII%2BDrosha%2Binitiates%2BmicroRNA%2Bprocessing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01957%26rft_id%253Dinfo%253Apmid%252F14508493%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature01957&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=14508493&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000185502300045&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.5" data-doi="10.1016/j.cell.2007.06.028">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Okamura</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hagen</span>, 
          <span class="cit-name-given-names">J.W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Duan</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tyler</span>, 
          <span class="cit-name-given-names">D.M.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Lai</span>, 
          <span class="cit-name-given-names">E.C.</span>
         </span> (
         <span class="cit-pub-date">2007</span>) 
         <span class="cit-article-title">The mirtron pathway generates microRNA-class regulatory RNAs in Drosophila</span>. 
         <abbr class="cit-jnl-abbrev">Cell</abbr>, 
         <span class="cit-vol">130</span>, 
         <span class="cit-fpage">89</span>–
         <span class="cit-lpage">100</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DOkamura%26rft.auinit1%253DK.%26rft.volume%253D130%26rft.issue%253D1%26rft.spage%253D89%26rft.epage%253D100%26rft.atitle%253DThe%2Bmirtron%2Bpathway%2Bgenerates%2BmicroRNA-class%2Bregulatory%2BRNAs%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2007.06.028%26rft_id%253Dinfo%253Apmid%252F17599402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.cell.2007.06.028&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17599402&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000248587000013&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.6" data-doi="10.1038/nature05983">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ruby</span>, 
          <span class="cit-name-given-names">J.G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jan</span>, 
          <span class="cit-name-given-names">C.H.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Bartel</span>, 
          <span class="cit-name-given-names">D.P.</span>
         </span> (
         <span class="cit-pub-date">2007</span>) 
         <span class="cit-article-title">Intronic microRNA precursors that bypass Drosha processing</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>, 
         <span class="cit-vol">448</span>, 
         <span class="cit-fpage">83</span>–
         <span class="cit-lpage">86</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRuby%26rft.auinit1%253DJ.%2BG.%26rft.volume%253D448%26rft.issue%253D7149%26rft.spage%253D83%26rft.epage%253D86%26rft.atitle%253DIntronic%2BmicroRNA%2Bprecursors%2Bthat%2Bbypass%2BDrosha%2Bprocessing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05983%26rft_id%253Dinfo%253Apmid%252F17589500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature05983&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17589500&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000247720900042&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.7" data-doi="10.1101/gad.1158803">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yi</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Qin</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Macara</span>, 
          <span class="cit-name-given-names">I.G.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Cullen</span>, 
          <span class="cit-name-given-names">B.R.</span>
         </span> (
         <span class="cit-pub-date">2003</span>) 
         <span class="cit-article-title">Exportin-5 mediates the nuclear export of pre-microRNAs and short hairpin RNAs</span>. 
         <abbr class="cit-jnl-abbrev">Genes &amp;; development</abbr>, 
         <span class="cit-vol">17</span>, 
         <span class="cit-fpage">3011</span>–
         <span class="cit-lpage">3016</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2Bdevelopment%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1158803%26rft_id%253Dinfo%253Apmid%252F14681208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE3LzI0LzMwMTEiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNC8wNi8yOTQ0ODguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.8" data-doi="10.1126/science.1090599">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lund</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Guttinger</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Calado</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dahlberg</span>, 
          <span class="cit-name-given-names">J.E.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Kutay</span>, 
          <span class="cit-name-given-names">U.</span>
         </span> (
         <span class="cit-pub-date">2004</span>) 
         <span class="cit-article-title">Nuclear export of microRNA precursors</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>, 
         <span class="cit-vol">303</span>, 
         <span class="cit-fpage">95</span>–
         <span class="cit-lpage">98</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1090599%26rft_id%253Dinfo%253Apmid%252F14631048%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMDMvNTY1NC85NSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA0LzA2LzI5NDQ4OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.9" data-doi="10.1038/nature03868">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chendrimada</span>, 
          <span class="cit-name-given-names">T.P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gregory</span>, 
          <span class="cit-name-given-names">R.I.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kumaraswamy</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Norman</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cooch</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nishikura</span>, 
          <span class="cit-name-given-names">K.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Shiekhattar</span>, 
          <span class="cit-name-given-names">R.</span>
         </span> (
         <span class="cit-pub-date">2005</span>) 
         <span class="cit-article-title">TRBP recruits the Dicer complex to Ago2 for microRNA processing and gene silencing</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>, 
         <span class="cit-vol">436</span>, 
         <span class="cit-fpage">740</span>–
         <span class="cit-lpage">744</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DChendrimada%26rft.auinit1%253DT.%2BP.%26rft.volume%253D436%26rft.issue%253D7051%26rft.spage%253D740%26rft.epage%253D744%26rft.atitle%253DTRBP%2Brecruits%2Bthe%2BDicer%2Bcomplex%2Bto%2BAgo2%2Bfor%2BmicroRNA%2Bprocessing%2Band%2Bgene%2Bsilencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03868%26rft_id%253Dinfo%253Apmid%252F15973356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature03868&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15973356&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000230964500052&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-294488v1.10" data-doi="10.1016/j.cell.2005.10.022">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gregory</span>, 
          <span class="cit-name-given-names">R.I.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chendrimada</span>, 
          <span class="cit-name-given-names">T.P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cooch</span>, 
          <span class="cit-name-given-names">N.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Shiekhattar</span>, 
          <span class="cit-name-given-names">R.</span>
         </span> (
         <span class="cit-pub-date">2005</span>) 
         <span class="cit-article-title">Human RISC couples microRNA biogenesis and posttranscriptional gene silencing</span>. 
         <abbr class="cit-jnl-abbrev">Cell</abbr>, 
         <span class="cit-vol">123</span>, 
         <span class="cit-fpage">631</span>–
         <span class="cit-lpage">640</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DGregory%26rft.auinit1%253DR.%2BI.%26rft.volume%253D123%26rft.issue%253D4%26rft.spage%253D631%26rft.epage%253D640%26rft.atitle%253DHuman%2BRISC%2Bcouples%2BmicroRNA%2Bbiogenesis%2Band%2Bposttranscriptional%2Bgene%2Bsilencing.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2005.10.022%26rft_id%253Dinfo%253Apmid%252F16271387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.cell.2005.10.022&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16271387&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000233497400012&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.11" data-doi="10.1101/gad.1384005">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Maniataki</span>, 
          <span class="cit-name-given-names">E.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Mourelatos</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span> (
         <span class="cit-pub-date">2005</span>) 
         <span class="cit-article-title">A human, ATP-independent, RISC assembly machine fueled by pre-miRNA</span>. 
         <abbr class="cit-jnl-abbrev">Genes &amp;; development</abbr>, 
         <span class="cit-vol">19</span>, 
         <span class="cit-fpage">2979</span>–
         <span class="cit-lpage">2990</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DManiataki%26rft.auinit1%253DE.%26rft.volume%253D19%26rft.issue%253D24%26rft.spage%253D2979%26rft.epage%253D2990%26rft.atitle%253DA%2Bhuman%252C%2BATP-independent%252C%2BRISC%2Bassembly%2Bmachine%2Bfueled%2Bby%2Bpre-miRNA%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1384005%26rft_id%253Dinfo%253Apmid%252F16357216%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE5LzI0LzI5NzkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNC8wNi8yOTQ0ODguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.12" data-doi="10.1126/science.1190809">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Cifuentes</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xue</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Taylor</span>, 
          <span class="cit-name-given-names">D.W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Patnode</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mishima</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cheloufi</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ma</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mane</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hannon</span>, 
          <span class="cit-name-given-names">G.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lawson</span>, 
          <span class="cit-name-given-names">N.D.</span>
         </span>
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2010</span>) 
         <span class="cit-article-title">A novel miRNA processing pathway independent of Dicer requires Argonaute2 catalytic activity</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>, 
         <span class="cit-vol">328</span>, 
         <span class="cit-fpage">1694</span>–
         <span class="cit-lpage">1698</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1190809%26rft_id%253Dinfo%253Apmid%252F20448148%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjgvNTk4Ni8xNjk0IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDQvMDYvMjk0NDg4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.13" data-doi="10.1016/j.cell.2007.10.032">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Diederichs</span>, 
          <span class="cit-name-given-names">S.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Haber</span>, 
          <span class="cit-name-given-names">D.A.</span>
         </span> (
         <span class="cit-pub-date">2007</span>) 
         <span class="cit-article-title">Dual role for argonautes in microRNA processing and posttranscriptional regulation of microRNA expression</span>. 
         <abbr class="cit-jnl-abbrev">Cell</abbr>, 
         <span class="cit-vol">131</span>, 
         <span class="cit-fpage">1097</span>–
         <span class="cit-lpage">1108</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DDiederichs%26rft.auinit1%253DS.%26rft.volume%253D131%26rft.issue%253D6%26rft.spage%253D1097%26rft.epage%253D1108%26rft.atitle%253DDual%2Brole%2Bfor%2Bargonautes%2Bin%2BmicroRNA%2Bprocessing%2Band%2Bposttranscriptional%2Bregulation%2Bof%2BmicroRNA%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2007.10.032%26rft_id%253Dinfo%253Apmid%252F18083100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.cell.2007.10.032&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18083100&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000252217100018&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.14" data-doi="10.1038/nature09092">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Cheloufi</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dos Santos</span>, 
          <span class="cit-name-given-names">C.O.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chong</span>, 
          <span class="cit-name-given-names">M.M.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Hannon</span>, 
          <span class="cit-name-given-names">G.J.</span>
         </span> (
         <span class="cit-pub-date">2010</span>) 
         <span class="cit-article-title">A dicer-independent miRNA biogenesis pathway that requires Ago catalysis</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>, 
         <span class="cit-vol">465</span>, 
         <span class="cit-fpage">584</span>–
         <span class="cit-lpage">589</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DCheloufi%26rft.auinit1%253DS.%26rft.volume%253D465%26rft.issue%253D7298%26rft.spage%253D584%26rft.epage%253D589%26rft.atitle%253DA%2Bdicer-independent%2BmiRNA%2Bbiogenesis%2Bpathway%2Bthat%2Brequires%2BAgo%2Bcatalysis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09092%26rft_id%253Dinfo%253Apmid%252F20424607%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature09092&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20424607&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000278249000034&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.15">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yoda</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cifuentes</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Izumi</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sakaguchi</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Suzuki</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Giraldez</span>, 
          <span class="cit-name-given-names">A.J.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Tomari</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span> (
         <span class="cit-pub-date">2013</span>) 
         <span class="cit-article-title">Poly(A)-specific ribonuclease mediates 3’-end trimming of Argonaute2-cleaved precursor microRNAs</span>. 
         <abbr class="cit-jnl-abbrev">Cell reports</abbr>, 
         <span class="cit-vol">5</span>, 
         <span class="cit-fpage">715</span>–
         <span class="cit-lpage">726</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%2Breports%26rft.volume%253D5%26rft.spage%253D715%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.16" data-doi="10.1016/j.molcel.2012.03.010">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jin</span>, 
          <span class="cit-name-given-names">D.-Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">McManus</span>, 
          <span class="cit-name-given-names">M.T.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Mourelatos</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span> (
         <span class="cit-pub-date">2012</span>) 
         <span class="cit-article-title">Precursor microRNA-programmed silencing complex assembly pathways in mammals</span>. 
         <abbr class="cit-jnl-abbrev">Mol. Cell</abbr>, 
         <span class="cit-vol">46</span>, 
         <span class="cit-fpage">507</span>–
         <span class="cit-lpage">517</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DLiu%26rft.auinit1%253DX.%26rft.volume%253D46%26rft.issue%253D4%26rft.spage%253D507%26rft.epage%253D517%26rft.atitle%253DPrecursor%2BmicroRNA-programmed%2Bsilencing%2Bcomplex%2Bassembly%2Bpathways%2Bin%2Bmammals.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2012.03.010%26rft_id%253Dinfo%253Apmid%252F22503104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.molcel.2012.03.010&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22503104&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000304518900014&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">17.</span>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.17" data-doi="10.1016/j.molcel.2008.09.014">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Heo</span>, 
          <span class="cit-name-given-names">I.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Joo</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cho</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ha</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Han</span>, 
          <span class="cit-name-given-names">J.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Kim</span>, 
          <span class="cit-name-given-names">V.N.</span>
         </span> (
         <span class="cit-pub-date">2008</span>) 
         <span class="cit-article-title">Lin28 mediates the terminal uridylation of let-7 precursor MicroRNA</span>. 
         <abbr class="cit-jnl-abbrev">Mol Cell</abbr>, 
         <span class="cit-vol">32</span>, 
         <span class="cit-fpage">276</span>–
         <span class="cit-lpage">284</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DHeo%26rft.auinit1%253DI.%26rft.volume%253D32%26rft.issue%253D2%26rft.spage%253D276%26rft.epage%253D284%26rft.atitle%253DLin28%2Bmediates%2Bthe%2Bterminal%2Buridylation%2Bof%2Blet-7%2Bprecursor%2BMicroRNA.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2008.09.014%26rft_id%253Dinfo%253Apmid%252F18951094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.molcel.2008.09.014&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18951094&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000260546800015&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">18.</span>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.18" data-doi="10.1016/j.cell.2012.09.022">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Heo</span>, 
          <span class="cit-name-given-names">I.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ha</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lim</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yoon</span>, 
          <span class="cit-name-given-names">M.-J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Park</span>, 
          <span class="cit-name-given-names">J.-E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kwon</span>, 
          <span class="cit-name-given-names">S.C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chang</span>, 
          <span class="cit-name-given-names">H.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Kim</span>, 
          <span class="cit-name-given-names">V.N.</span>
         </span> (
         <span class="cit-pub-date">2012</span>) 
         <span class="cit-article-title">Mono-uridylation of pre-microRNA as a key step in the biogenesis of group II let-7 microRNAs</span>. 
         <abbr class="cit-jnl-abbrev">Cell</abbr>, 
         <span class="cit-vol">151</span>, 
         <span class="cit-fpage">521</span>–
         <span class="cit-lpage">532</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHeo%26rft.auinit1%253DI.%26rft.volume%253D151%26rft.issue%253D3%26rft.spage%253D521%26rft.epage%253D532%26rft.atitle%253DMono-Uridylation%2Bof%2BPre-MicroRNA%2Bas%2Ba%2BKey%2BStep%2Bin%2Bthe%2BBiogenesis%2Bof%2BGroup%2BII%2Blet-7%2BMicroRNAs.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2012.09.022%26rft_id%253Dinfo%253Apmid%252F23063654%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.cell.2012.09.022&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23063654&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000310529300010&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">19.</span>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.19" data-doi="10.1016/j.molcel.2014.07.017">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zheng</span>, 
          <span class="cit-name-given-names">Q.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vrettos</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Maragkakis</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Alexiou</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gregory</span>, 
          <span class="cit-name-given-names">B.D.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Mourelatos</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span> (
         <span class="cit-pub-date">2014</span>) 
         <span class="cit-article-title">A MicroRNA precursor surveillance system in quality control of MicroRNA synthesis</span>. 
         <abbr class="cit-jnl-abbrev">Mol. Cell</abbr>, 
         <span class="cit-vol">55</span>, 
         <span class="cit-fpage">868</span>–
         <span class="cit-lpage">879</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D55%26rft.spage%253D868%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2014.07.017%26rft_id%253Dinfo%253Apmid%252F25175028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.molcel.2014.07.017&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25175028&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">20.</span>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.20" data-doi="10.15252/embj.201590931">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kim</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ha</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Loeff</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chang</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Simanshu</span>, 
          <span class="cit-name-given-names">D.K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fareh</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Patel</span>, 
          <span class="cit-name-given-names">D.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Joo</span>, 
          <span class="cit-name-given-names">C.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Kim</span>, 
          <span class="cit-name-given-names">V.N.</span>
         </span> (
         <span class="cit-pub-date">2015</span>) 
         <span class="cit-article-title">TUT7 controls the fate of precursor microRNAs by using three different uridylation mechanisms</span>. 
         <abbr class="cit-jnl-abbrev">The EMBO journal</abbr>, 
         <span class="cit-vol">34</span>, 
         <span class="cit-fpage">1801</span>–
         <span class="cit-lpage">1815</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2Bjournal%26rft_id%253Dinfo%253Adoi%252F10.15252%252Fembj.201590931%26rft_id%253Dinfo%253Apmid%252F25979828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMzQvMTMvMTgwMSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA0LzA2LzI5NDQ4OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">21.</span>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.21" data-doi="10.1261/rna.2713611">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Newman</span>, 
          <span class="cit-name-given-names">M.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mani</span>, 
          <span class="cit-name-given-names">V.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Hammond</span>, 
          <span class="cit-name-given-names">S.M.</span>
         </span> (
         <span class="cit-pub-date">2011</span>) 
         <span class="cit-article-title">Deep sequencing of microRNA precursors reveals extensive 3’ end modification</span>. 
         <abbr class="cit-jnl-abbrev">RNA</abbr>, 
         <span class="cit-vol">17</span>, 
         <span class="cit-fpage">1795</span>–
         <span class="cit-lpage">1803</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.2713611%26rft_id%253Dinfo%253Apmid%252F21849429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjEwOiIxNy8xMC8xNzk1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDQvMDYvMjk0NDg4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">22.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-294488v1.22" data-doi="10.1101/gr.118059.110">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wyman</span>, 
          <span class="cit-name-given-names">S.K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Knouf</span>, 
          <span class="cit-name-given-names">E.C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Parkin</span>, 
          <span class="cit-name-given-names">R.K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fritz</span>, 
          <span class="cit-name-given-names">B.R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lin</span>, 
          <span class="cit-name-given-names">D.W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dennis</span>, 
          <span class="cit-name-given-names">L.M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Krouse</span>, 
          <span class="cit-name-given-names">M.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Webster</span>, 
          <span class="cit-name-given-names">P.J.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Tewari</span>, 
          <span class="cit-name-given-names">M.</span>
         </span> (
         <span class="cit-pub-date">2011</span>) 
         <span class="cit-article-title">Post-transcriptional generation of miRNA variants by multiple nucleotidyl transferases contributes to miRNA transcriptome complexity</span>. 
         <abbr class="cit-jnl-abbrev">Genome research</abbr>, 
         <span class="cit-vol">21</span>, 
         <span class="cit-fpage">1450</span>–
         <span class="cit-lpage">1461</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.118059.110%26rft_id%253Dinfo%253Apmid%252F21813625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzkvMTQ1MCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA0LzA2LzI5NDQ4OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">23.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-294488v1.23" data-doi="10.1093/nar/gks705">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Dueck</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ziegler</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Eichner</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Berezikov</span>, 
          <span class="cit-name-given-names">E.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Meister</span>, 
          <span class="cit-name-given-names">G.</span>
         </span> (
         <span class="cit-pub-date">2012</span>) 
         <span class="cit-article-title">microRNAs associated with the different human Argonaute proteins</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr>, 
         <span class="cit-vol">40</span>, 
         <span class="cit-fpage">9850</span>–
         <span class="cit-lpage">9862</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks705%26rft_id%253Dinfo%253Apmid%252F22844086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gks705&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22844086&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000310377200046&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">24.</span>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.24" data-doi="10.1101/gr.106054.110">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Burroughs</span>, 
          <span class="cit-name-given-names">A.M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ando</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">de Hoon</span>, 
          <span class="cit-name-given-names">M.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tomaru</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nishibu</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ukekawa</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Funakoshi</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kurokawa</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Suzuki</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hayashizaki</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2010</span>) 
         <span class="cit-article-title">A comprehensive survey of 3’ animal miRNA modification events and a possible role for 3’ adenylation in modulating miRNA targeting effectiveness</span>. 
         <abbr class="cit-jnl-abbrev">Genome research</abbr>, 
         <span class="cit-vol">20</span>, 
         <span class="cit-fpage">1398</span>–
         <span class="cit-lpage">1410</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.106054.110%26rft_id%253Dinfo%253Apmid%252F20719920%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMC8xMC8xMzk4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDQvMDYvMjk0NDg4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">25.</span>
      <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.25" data-doi="10.1038/nbt1394">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Friedlander</span>, 
          <span class="cit-name-given-names">M.R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Adamidi</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Maaskola</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Einspanier</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Knespel</span>, 
          <span class="cit-name-given-names">S.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Rajewsky</span>, 
          <span class="cit-name-given-names">N.</span>
         </span> (
         <span class="cit-pub-date">2008</span>) 
         <span class="cit-article-title">Discovering microRNAs from deep sequencing data using miRDeep</span>. 
         <abbr class="cit-jnl-abbrev">Nature biotechnology</abbr>, 
         <span class="cit-vol">26</span>, 
         <span class="cit-fpage">407</span>–
         <span class="cit-lpage">415</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DFriedlander%26rft.auinit1%253DM.%2BR.%26rft.volume%253D26%26rft.issue%253D4%26rft.spage%253D407%26rft.epage%253D415%26rft.atitle%253DDiscovering%2BmicroRNAs%2Bfrom%2Bdeep%2Bsequencing%2Bdata%2Busing%2BmiRDeep.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt1394%26rft_id%253Dinfo%253Apmid%252F18392026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nbt1394&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18392026&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000254782500023&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">26.</span>
      <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.26" data-doi="10.1073/pnas.1420955112">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Londin</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Loher</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Telonis</span>, 
          <span class="cit-name-given-names">A.G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Quann</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Clark</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jing</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hatzimichael</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kirino</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Honda</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lally</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2015</span>) 
         <span class="cit-article-title">Analysis of 13 cell types reveals evidence for the expression of numerous novel primate- and tissue-specific microRNAs</span>. 
         <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr>, 
         <span class="cit-vol">112</span>, 
         <span class="cit-fpage">E1106</span>–
         <span class="cit-lpage">1115</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1420955112%26rft_id%253Dinfo%253Apmid%252F25713380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEyLzEwL0UxMTA2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDQvMDYvMjk0NDg4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">27.</span>
      <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.27" data-doi="10.1093/nar/gkt072">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">You</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mackowiak</span>, 
          <span class="cit-name-given-names">S.D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Friedländer</span>, 
          <span class="cit-name-given-names">M.R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Weigt</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Du</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gogol-Döring</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chang</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dieterich</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2013</span>) 
         <span class="cit-article-title">Global profiling of miRNAs and the hairpin precursors: insights into miRNA processing and novel miRNA discovery</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr>, 
         <span class="cit-vol">41</span>, 
         <span class="cit-fpage">3619</span>–
         <span class="cit-lpage">3634</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt072%26rft_id%253Dinfo%253Apmid%252F23396444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkt072&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23396444&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000318063400020&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">28.</span>
      <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.28" data-doi="10.1093/nar/gkr903">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Burroughs</span>, 
          <span class="cit-name-given-names">A.M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kawano</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ando</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Daub</span>, 
          <span class="cit-name-given-names">C.O.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Hayashizaki</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span> (
         <span class="cit-pub-date">2012</span>) 
         <span class="cit-article-title">pre-miRNA profiles obtained through application of locked nucleic acids and deep sequencing reveals complex 5’/3’ arm variation including concomitant cleavage and polyuridylation patterns</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr>, 
         <span class="cit-vol">40</span>, 
         <span class="cit-fpage">1424</span>–
         <span class="cit-lpage">1437</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr903%26rft_id%253Dinfo%253Apmid%252F22058130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkr903&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22058130&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000301069400001&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">29.</span>
      <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.29" data-doi="10.1126/science.1168978">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ingolia</span>, 
          <span class="cit-name-given-names">N.T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ghaemmaghami</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Newman</span>, 
          <span class="cit-name-given-names">J.R.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Weissman</span>, 
          <span class="cit-name-given-names">J.S.</span>
         </span> (
         <span class="cit-pub-date">2009</span>) 
         <span class="cit-article-title">Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>, 
         <span class="cit-vol">324</span>, 
         <span class="cit-fpage">218</span>–
         <span class="cit-lpage">223</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1168978%26rft_id%253Dinfo%253Apmid%252F19213877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjQvNTkyNC8yMTgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNC8wNi8yOTQ0ODguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">30.</span>
      <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.30">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kirino</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vourekas</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Khandros</span>, 
          <span class="cit-name-given-names">E.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Mourelatos</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span> (
         <span class="cit-pub-date">2011</span>) 
         <span class="cit-article-title">Immunoprecipitation of piRNPs and directional, next generation sequencing of piRNAs</span>. 
         <abbr class="cit-jnl-abbrev">Methods in molecular biology</abbr>, 
         <span class="cit-vol">725</span>, 
         <span class="cit-fpage">281</span>–
         <span class="cit-lpage">293</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%26rft.volume%253D725%26rft.spage%253D281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">31.</span>
      <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.31" data-doi="10.1016/j.ymeth.2007.09.006">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Schmittgen</span>, 
          <span class="cit-name-given-names">T.D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lee</span>, 
          <span class="cit-name-given-names">E.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jiang</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sarkar</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Elton</span>, 
          <span class="cit-name-given-names">T.S.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">C.</span>
         </span> (
         <span class="cit-pub-date">2008</span>) 
         <span class="cit-article-title">Real-time PCR quantification of precursor and mature microRNA</span>. 
         <abbr class="cit-jnl-abbrev">Methods</abbr>, 
         <span class="cit-vol">44</span>, 
         <span class="cit-fpage">31</span>–
         <span class="cit-lpage">38</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMethods%2B%2528San%2BDiego%252C%2BCalif.%2529%26rft.stitle%253DMethods%26rft.aulast%253DSchmittgen%26rft.auinit1%253DT.%2BD.%26rft.volume%253D44%26rft.issue%253D1%26rft.spage%253D31%26rft.epage%253D38%26rft.atitle%253DReal-time%2BPCR%2Bquantification%2Bof%2Bprecursor%2Band%2Bmature%2BmicroRNA.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ymeth.2007.09.006%26rft_id%253Dinfo%253Apmid%252F18158130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.ymeth.2007.09.006&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18158130&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000253189300005&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">32.</span>
      <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.32" data-doi="10.1136/ip.8.2.165">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Moll</span>, 
          <span class="cit-name-given-names">E.K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Donoghue</span>, 
          <span class="cit-name-given-names">A.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Alpern</span>, 
          <span class="cit-name-given-names">E.R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kleppel</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Durbin</span>, 
          <span class="cit-name-given-names">D.R.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Winston</span>, 
          <span class="cit-name-given-names">F.K.</span>
         </span> (
         <span class="cit-pub-date">2002</span>) 
         <span class="cit-article-title">Child bicyclist injuries: are we obtaining enough information in the emergency department chart?</span>
         <abbr class="cit-jnl-abbrev">Inj Prev</abbr>, 
         <span class="cit-vol">8</span>, 
         <span class="cit-fpage">165</span>–
         <span class="cit-lpage">169</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DInjury%2BPrevention%26rft.stitle%253DInj.%2BPrev.%26rft.aulast%253DMoll%26rft.auinit1%253DE%2BK%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253D165%26rft.epage%253D169%26rft.atitle%253DChild%2Bbicyclist%2Binjuries%253A%2Bare%2Bwe%2Bobtaining%2Benough%2Binformation%2Bin%2Bthe%2Bemergency%2Bdepartment%2Bchart%253F%26rft_id%253Dinfo%253Adoi%252F10.1136%252Fip.8.2.165%26rft_id%253Dinfo%253Apmid%252F12120839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6ImluanVyeXByZXYiO3M6NToicmVzaWQiO3M6NzoiOC8yLzE2NSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA0LzA2LzI5NDQ4OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">33.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-294488v1.33" data-doi="10.1093/bioinformatics/18.suppl_1.S96">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Huber</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">von Heydebreck</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sultmann</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Poustka</span>, 
          <span class="cit-name-given-names">A.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Vingron</span>, 
          <span class="cit-name-given-names">M.</span>
         </span> (
         <span class="cit-pub-date">2002</span>) 
         <span class="cit-article-title">Variance stabilization applied to microarray data calibration and to the quantification of differential expression</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, 
         <span class="cit-vol">18</span>
         <span class="cit-issue">Suppl 1</span>, 
         <span class="cit-fpage">S96</span>–
         <span class="cit-lpage">104</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DHuber%26rft.auinit1%253DW.%26rft.volume%253D18%26rft.issue%253D90001%26rft.spage%253DS96%26rft.epage%253D104%26rft.atitle%253DVariance%2Bstabilization%2Bapplied%2Bto%2Bmicroarray%2Bdata%2Bcalibration%2Band%2Bto%2Bthe%2Bquantification%2Bof%2Bdifferential%2Bexpression%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F18.suppl_1.S96%26rft_id%253Dinfo%253Apmid%252F12169536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/18.suppl_1.S96&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=12169536&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">34.</span>
      <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.34">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zheng</span>, 
          <span class="cit-name-given-names">Q.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ryvkin</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dragomir</span>, 
          <span class="cit-name-given-names">I.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Desai</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Aiyer</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Valladares</span>, 
          <span class="cit-name-given-names">O.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bambina</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sabin</span>, 
          <span class="cit-name-given-names">L.R.</span>
         </span>
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2012</span>) 
         <span class="cit-article-title">Global analysis of RNA secondary structure in two metazoans</span>. 
         <abbr class="cit-jnl-abbrev">Cell reports</abbr>, 
         <span class="cit-vol">1</span>, 
         <span class="cit-fpage">69</span>–
         <span class="cit-lpage">82</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%2Breports%26rft.volume%253D1%26rft.spage%253D69%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">35.</span>
      <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.35" data-doi="10.1093/bioinformatics/btr570">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Althammer</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gonzalez-Vallinas</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ballare</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Beato</span>, 
          <span class="cit-name-given-names">M.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Eyras</span>, 
          <span class="cit-name-given-names">E.</span>
         </span> (
         <span class="cit-pub-date">2011</span>) 
         <span class="cit-article-title">Pyicos: a versatile toolkit for the analysis of high-throughput sequencing data</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, 
         <span class="cit-vol">27</span>, 
         <span class="cit-fpage">3333</span>–
         <span class="cit-lpage">3340</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr570%26rft_id%253Dinfo%253Apmid%252F21994224%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr570&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21994224&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000297860000002&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">36.</span>
      <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.36" data-doi="10.1186/1748-7188-6- 26">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lorenz</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bernhart</span>, 
          <span class="cit-name-given-names">S.H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Honer Zu Siederdissen</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tafer</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Flamm</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Stadler</span>, 
          <span class="cit-name-given-names">P.F.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Hofacker</span>, 
          <span class="cit-name-given-names">I.L.</span>
         </span> (
         <span class="cit-pub-date">2011</span>) 
         <span class="cit-article-title">ViennaRNA Package 2.0</span>. 
         <abbr class="cit-jnl-abbrev">Algorithms for molecular biology: AMB</abbr>, 
         <span class="cit-vol">6</span>, 
         <span class="cit-fpage">26</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAlgorithms%2Bfor%2Bmolecular%2Bbiology%253A%2BAMB%26rft.volume%253D6%26rft.spage%253D26%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1748-7188-6-%2B26%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/1748-7188-6- 26&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">37.</span>
      <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.37" data-doi="10.1371/journal.pone.0075346">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Elliott</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dragomir</span>, 
          <span class="cit-name-given-names">I.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chua</span>, 
          <span class="cit-name-given-names">M.M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gregory</span>, 
          <span class="cit-name-given-names">B.D.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Weiss</span>, 
          <span class="cit-name-given-names">S.R.</span>
         </span> (
         <span class="cit-pub-date">2013</span>) 
         <span class="cit-article-title">Analysis of the host transcriptome from demyelinating spinal cord of murine coronavirus-infected mice</span>. 
         <abbr class="cit-jnl-abbrev">PLoS One</abbr>, 
         <span class="cit-vol">8</span>, 
         <span class="cit-fpage">e75346</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De75346%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0075346%26rft_id%253Dinfo%253Apmid%252F24058676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0075346&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24058676&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">38.</span>
      <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.38" data-doi="10.1093/bioinformatics/btu638">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Anders</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pyl</span>, 
          <span class="cit-name-given-names">P.T.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Huber</span>, 
          <span class="cit-name-given-names">W.</span>
         </span> (
         <span class="cit-pub-date">2015</span>) 
         <span class="cit-article-title">HTSeq--a Python framework to work with high-throughput sequencing data</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, 
         <span class="cit-vol">31</span>, 
         <span class="cit-fpage">166</span>–
         <span class="cit-lpage">169</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu638%26rft_id%253Dinfo%253Apmid%252F25260700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu638&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25260700&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000347832300003&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">39.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-294488v1.39" data-doi="10.1093/bioinformatics/btp120">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Trapnell</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pachter</span>, 
          <span class="cit-name-given-names">L.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Salzberg</span>, 
          <span class="cit-name-given-names">S.L.</span>
         </span> (
         <span class="cit-pub-date">2009</span>) 
         <span class="cit-article-title">TopHat: discovering splice junctions with RNA-Seq</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, 
         <span class="cit-vol">25</span>, 
         <span class="cit-fpage">1105</span>–
         <span class="cit-lpage">1111</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp120%26rft_id%253Dinfo%253Apmid%252F19289445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp120&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19289445&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000265523300002&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">40.</span>
      <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.40" data-doi="10.14806.17.1.200">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Martin</span>, 
          <span class="cit-name-given-names">M.</span>
         </span> (
         <span class="cit-pub-date">2011</span>) 
         <span class="cit-article-title">Cutadapt removes adapter sequences from high-throughput sequencing reads</span>. 
         <abbr class="cit-jnl-abbrev">EMBnet.journal</abbr>, 
         <span class="cit-vol">17</span>, 
         <span class="cit-fpage">10</span>–
         <span class="cit-lpage">12</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEMBnet.journal%26rft.volume%253D17%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.14806.17.1.200%26rft_id%253Dinfo%253Apmid%252F22955987%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.14806.17.1.200&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22955987&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">41.</span>
      <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.41" data-doi="10.1186/s13059-014-0550-8">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Love</span>, 
          <span class="cit-name-given-names">M.I.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huber</span>, 
          <span class="cit-name-given-names">W.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Anders</span>, 
          <span class="cit-name-given-names">S.</span>
         </span> (
         <span class="cit-pub-date">2014</span>) 
         <span class="cit-article-title">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</span>. 
         <abbr class="cit-jnl-abbrev">Genome biology</abbr>, 
         <span class="cit-vol">15</span>, 
         <span class="cit-fpage">550</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.volume%253D15%26rft.spage%253D550%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0550-8%26rft_id%253Dinfo%253Apmid%252F25516281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/s13059-014-0550-8&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25516281&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">42.</span>
      <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.42" data-doi="10.1261/rna.646007">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Nelson</span>, 
          <span class="cit-name-given-names">P.T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">De Planell-Saguer</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lamprinaki</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kiriakidou</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">O’Doherty</span>, 
          <span class="cit-name-given-names">U.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Mourelatos</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span> (
         <span class="cit-pub-date">2007</span>) 
         <span class="cit-article-title">A novel monoclonal antibody against human Argonaute proteins reveals unexpected characteristics of miRNAs in human blood cells</span>. 
         <abbr class="cit-jnl-abbrev">RNA</abbr>, 
         <span class="cit-vol">13</span>, 
         <span class="cit-fpage">1787</span>–
         <span class="cit-lpage">1792</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.646007%26rft_id%253Dinfo%253Apmid%252F17720879%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjEwOiIxMy8xMC8xNzg3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDQvMDYvMjk0NDg4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">43.</span>
      <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.43" data-doi="10.1016/0022-2836(81)90087-5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Smith</span>, 
          <span class="cit-name-given-names">T.F.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Waterman</span>, 
          <span class="cit-name-given-names">M.S.</span>
         </span> (
         <span class="cit-pub-date">1981</span>) 
         <span class="cit-article-title">Identification of common molecular subsequences</span>. 
         <abbr class="cit-jnl-abbrev">Journal of molecular biology</abbr>, 
         <span class="cit-vol">147</span>, 
         <span class="cit-fpage">195</span>–
         <span class="cit-lpage">197</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DSmith%26rft.auinit1%253DT.%2BF.%26rft.volume%253D147%26rft.issue%253D1%26rft.spage%253D195%26rft.epage%253D197%26rft.atitle%253DIdentification%2Bof%2Bcommon%2Bmolecular%2Bsubsequences.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252881%252990087-5%26rft_id%253Dinfo%253Apmid%252F7265238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/0022-2836(81)90087-5&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=7265238&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1981LJ17700013&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">44.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-294488v1.44" data-doi="10.1038/nmeth.1923">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Langmead</span>, 
          <span class="cit-name-given-names">B.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Salzberg</span>, 
          <span class="cit-name-given-names">S.L.</span>
         </span> (
         <span class="cit-pub-date">2012</span>) 
         <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2</span>. 
         <abbr class="cit-jnl-abbrev">Nat Methods</abbr>, 
         <span class="cit-vol">9</span>, 
         <span class="cit-fpage">357</span>–
         <span class="cit-lpage">359</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22388286&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000302218500017&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">45.</span>
      <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.45" data-doi="10.1093/nar/gkt1181">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kozomara</span>, 
          <span class="cit-name-given-names">A.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Griffiths-Jones</span>, 
          <span class="cit-name-given-names">S.</span>
         </span> (
         <span class="cit-pub-date">2014</span>) 
         <span class="cit-article-title">miRBase: annotating high confidence microRNAs using deep sequencing data</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr>, 
         <span class="cit-vol">42</span>, 
         <span class="cit-fpage">D68</span>–
         <span class="cit-lpage">73</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1181%26rft_id%253Dinfo%253Apmid%252F24275495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkt1181&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24275495&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000331139800011&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">46.</span>
      <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.46" data-doi="10.1016/j.cell.2009.08.002">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Heo</span>, 
          <span class="cit-name-given-names">I.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Joo</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kim</span>, 
          <span class="cit-name-given-names">Y.K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ha</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yoon</span>, 
          <span class="cit-name-given-names">M.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cho</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yeom</span>, 
          <span class="cit-name-given-names">K.H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Han</span>, 
          <span class="cit-name-given-names">J.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Kim</span>, 
          <span class="cit-name-given-names">V.N.</span>
         </span> (
         <span class="cit-pub-date">2009</span>) 
         <span class="cit-article-title">TUT4 in concert with Lin28 suppresses microRNA biogenesis through pre-microRNA uridylation</span>. 
         <abbr class="cit-jnl-abbrev">Cell</abbr>, 
         <span class="cit-vol">138</span>, 
         <span class="cit-fpage">696</span>–
         <span class="cit-lpage">708</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHeo%26rft.auinit1%253DI.%26rft.volume%253D138%26rft.issue%253D4%26rft.spage%253D696%26rft.epage%253D708%26rft.atitle%253DTUT4%2Bin%2Bconcert%2Bwith%2BLin28%2Bsuppresses%2BmicroRNA%2Bbiogenesis%2Bthrough%2Bpre-microRNA%2Buridylation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2009.08.002%26rft_id%253Dinfo%253Apmid%252F19703396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.cell.2009.08.002&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19703396&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000269156100012&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">47.</span>
      <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.47" data-doi="10.1093/nar/gku805">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Thornton</span>, 
          <span class="cit-name-given-names">J.E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Du</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jing</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sjekloca</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lin</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Grossi</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sliz</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zon</span>, 
          <span class="cit-name-given-names">L.I.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Gregory</span>, 
          <span class="cit-name-given-names">R.I.</span>
         </span> (
         <span class="cit-pub-date">2014</span>) 
         <span class="cit-article-title">Selective microRNA uridylation by Zcchc6 (TUT7) and Zcchc11 (TUT4)</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr>, 
         <span class="cit-vol">42</span>, 
         <span class="cit-fpage">11777</span>–
         <span class="cit-lpage">11791</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku805%26rft_id%253Dinfo%253Apmid%252F25223788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gku805&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25223788&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">48.</span>
      <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.48" data-doi="10.1126/science.1102513">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Carmell</span>, 
          <span class="cit-name-given-names">M.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rivas</span>, 
          <span class="cit-name-given-names">F.V.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Marsden</span>, 
          <span class="cit-name-given-names">C.G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Thomson</span>, 
          <span class="cit-name-given-names">J.M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Song</span>, 
          <span class="cit-name-given-names">J.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hammond</span>, 
          <span class="cit-name-given-names">S.M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Joshua-Tor</span>, 
          <span class="cit-name-given-names">L.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Hannon</span>, 
          <span class="cit-name-given-names">G.J.</span>
         </span> (
         <span class="cit-pub-date">2004</span>) 
         <span class="cit-article-title">Argonaute2 is the catalytic engine of mammalian RNAi</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>, 
         <span class="cit-vol">305</span>, 
         <span class="cit-fpage">1437</span>–
         <span class="cit-lpage">1441</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1102513%26rft_id%253Dinfo%253Apmid%252F15284456%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDUvNTY4OS8xNDM3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDQvMDYvMjk0NDg4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">49.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-294488v1.49" data-doi="10.1016/j.molcel.2004.07.007">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Meister</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Landthaler</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Patkaniowska</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dorsett</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Teng</span>, 
          <span class="cit-name-given-names">G.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Tuschl</span>, 
          <span class="cit-name-given-names">T.</span>
         </span> (
         <span class="cit-pub-date">2004</span>) 
         <span class="cit-article-title">Human Argonaute2 mediates RNA cleavage targeted by miRNAs and siRNAs</span>. 
         <abbr class="cit-jnl-abbrev">Molecular cell</abbr>, 
         <span class="cit-vol">15</span>, 
         <span class="cit-fpage">185</span>–
         <span class="cit-lpage">197</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DMeister%26rft.auinit1%253DG.%26rft.volume%253D15%26rft.issue%253D2%26rft.spage%253D185%26rft.epage%253D197%26rft.atitle%253DHuman%2BArgonaute2%2Bmediates%2BRNA%2Bcleavage%2Btargeted%2Bby%2BmiRNAs%2Band%2BsiRNAs.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2004.07.007%26rft_id%253Dinfo%253Apmid%252F15260970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.molcel.2004.07.007&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15260970&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000222958300007&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">50.</span>
      <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.50" data-doi="10.1126/science.1097434">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yekta</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shih</span>, 
          <span class="cit-name-given-names">I.H.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Bartel</span>, 
          <span class="cit-name-given-names">D.P.</span>
         </span> (
         <span class="cit-pub-date">2004</span>) 
         <span class="cit-article-title">MicroRNA-directed cleavage of HOXB8 mRNA</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>, 
         <span class="cit-vol">304</span>, 
         <span class="cit-fpage">594</span>–
         <span class="cit-lpage">596</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DYekta%26rft.auinit1%253DS.%26rft.volume%253D304%26rft.issue%253D5670%26rft.spage%253D594%26rft.epage%253D596%26rft.atitle%253DMicroRNA-Directed%2BCleavage%2Bof%2BHOXB8%2BmRNA%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1097434%26rft_id%253Dinfo%253Apmid%252F15105502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDQvNTY3MC81OTQiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNC8wNi8yOTQ0ODguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">51.</span>
      <a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.51" data-doi="10.1073/pnas.1006432107">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">J.S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Maurin</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Robine</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rasmussen</span>, 
          <span class="cit-name-given-names">K.D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jeffrey</span>, 
          <span class="cit-name-given-names">K.L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chandwani</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Papapetrou</span>, 
          <span class="cit-name-given-names">E.P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sadelain</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">O’Carroll</span>, 
          <span class="cit-name-given-names">D.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Lai</span>, 
          <span class="cit-name-given-names">E.C.</span>
         </span> (
         <span class="cit-pub-date">2010</span>) 
         <span class="cit-article-title">Conserved vertebrate mir-451 provides a platform for Dicer-independent, Ago2-mediated microRNA biogenesis</span>. 
         <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>, 
         <span class="cit-vol">107</span>, 
         <span class="cit-fpage">15163</span>–
         <span class="cit-lpage">15168</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1006432107%26rft_id%253Dinfo%253Apmid%252F20699384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzM0LzE1MTYzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDQvMDYvMjk0NDg4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">52.</span>
      <a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.52" data-doi="10.1016/j.cell.2008.10.053">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Han</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pedersen</span>, 
          <span class="cit-name-given-names">J.S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kwon</span>, 
          <span class="cit-name-given-names">S.C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Belair</span>, 
          <span class="cit-name-given-names">C.D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kim</span>, 
          <span class="cit-name-given-names">Y.K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yeom</span>, 
          <span class="cit-name-given-names">K.H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">W.Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Haussler</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Blelloch</span>, 
          <span class="cit-name-given-names">R.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Kim</span>, 
          <span class="cit-name-given-names">V.N.</span>
         </span> (
         <span class="cit-pub-date">2009</span>) 
         <span class="cit-article-title">Posttranscriptional crossregulation between Drosha and DGCR8</span>. 
         <abbr class="cit-jnl-abbrev">Cell</abbr>, 
         <span class="cit-vol">136</span>, 
         <span class="cit-fpage">75</span>–
         <span class="cit-lpage">84</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHan%26rft.auinit1%253DJ.%26rft.volume%253D136%26rft.issue%253D1%26rft.spage%253D75%26rft.epage%253D84%26rft.atitle%253DPosttranscriptional%2Bcrossregulation%2Bbetween%2BDrosha%2Band%2BDGCR8.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.10.053%26rft_id%253Dinfo%253Apmid%252F19135890%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.10.053&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19135890&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000262318400016&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">53.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-294488v1.53" data-doi="10.1261/rna.1319309">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kadener</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rodriguez</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Abruzzi</span>, 
          <span class="cit-name-given-names">K.C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Khodor</span>, 
          <span class="cit-name-given-names">Y.L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sugino</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Marr</span>, 
          <span class="cit-name-given-names">M.T.</span>
         </span>, 2nd, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nelson</span>, 
          <span class="cit-name-given-names">S.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Rosbash</span>, 
          <span class="cit-name-given-names">M.</span>
         </span> (
         <span class="cit-pub-date">2009</span>) 
         <span class="cit-article-title">Genome-wide identification of targets of the drosha-pasha/DGCR8 complex</span>. 
         <abbr class="cit-jnl-abbrev">RNA</abbr>, 
         <span class="cit-vol">15</span>, 
         <span class="cit-fpage">537</span>–
         <span class="cit-lpage">545</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.1319309%26rft_id%253Dinfo%253Apmid%252F19223442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjE1LzQvNTM3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDQvMDYvMjk0NDg4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">54.</span>
      <a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.54" data-doi="10.1261/rna.1591709">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Triboulet</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chang</span>, 
          <span class="cit-name-given-names">H.M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lapierre</span>, 
          <span class="cit-name-given-names">R.J.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Gregory</span>, 
          <span class="cit-name-given-names">R.I.</span>
         </span> (
         <span class="cit-pub-date">2009</span>) 
         <span class="cit-article-title">Post-transcriptional control of DGCR8 expression by the Microprocessor</span>. 
         <abbr class="cit-jnl-abbrev">RNA</abbr>, 
         <span class="cit-vol">15</span>, 
         <span class="cit-fpage">1005</span>–
         <span class="cit-lpage">1011</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.1591709%26rft_id%253Dinfo%253Apmid%252F19383765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjk6IjE1LzYvMTAwNSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA0LzA2LzI5NDQ4OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">55.</span>
      <a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.55" data-doi="10.1016/j.cell.2014.10.040">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rybak-Wolf</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jens</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Murakawa</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Herzog</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Landthaler</span>, 
          <span class="cit-name-given-names">M.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Rajewsky</span>, 
          <span class="cit-name-given-names">N.</span>
         </span> (
         <span class="cit-pub-date">2014</span>) 
         <span class="cit-article-title">A variety of dicer substrates in human and C. elegans</span>. 
         <abbr class="cit-jnl-abbrev">Cell</abbr>, 
         <span class="cit-vol">159</span>, 
         <span class="cit-fpage">1153</span>–
         <span class="cit-lpage">1167</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D159%26rft.spage%253D1153%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2014.10.040%26rft_id%253Dinfo%253Apmid%252F25416952%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.cell.2014.10.040&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25416952&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">56.</span>
      <a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.56">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Macias</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Plass</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Stajuda</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Michlewski</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Eyras</span>, 
          <span class="cit-name-given-names">E.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Caceres</span>, 
          <span class="cit-name-given-names">J.F.</span>
         </span> (
         <span class="cit-pub-date">2012</span>) 
         <span class="cit-article-title">DGCR8 HITS-CLIP reveals novel functions for the Microprocessor</span>. 
         <abbr class="cit-jnl-abbrev">Nature structural &amp;; molecular biology</abbr>, 
         <span class="cit-vol">19</span>, 
         <span class="cit-fpage">760</span>–
         <span class="cit-lpage">766</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.volume%253D19%26rft.spage%253D760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">57.</span>
      <a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.57" data-doi="10.1038/nature14263">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Spitale</span>, 
          <span class="cit-name-given-names">R.C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Flynn</span>, 
          <span class="cit-name-given-names">R.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">Q.C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Crisalli</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lee</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jung</span>, 
          <span class="cit-name-given-names">J.W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kuchelmeister</span>, 
          <span class="cit-name-given-names">H.Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Batista</span>, 
          <span class="cit-name-given-names">P.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Torre</span>, 
          <span class="cit-name-given-names">E.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kool</span>, 
          <span class="cit-name-given-names">E.T.</span>
         </span>
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2015</span>) 
         <span class="cit-article-title">Structural imprints in vivo decode RNA regulatory mechanisms</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>, 
         <span class="cit-vol">519</span>, 
         <span class="cit-fpage">486</span>–
         <span class="cit-lpage">490</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D519%26rft.spage%253D486%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14263%26rft_id%253Dinfo%253Apmid%252F25799993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature14263&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25799993&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F06%2F294488.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">58.</span>
      <a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-294488v1.58" data-doi="10.1073/pnas.84.19.6730">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hentze</span>, 
          <span class="cit-name-given-names">M.W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rouault</span>, 
          <span class="cit-name-given-names">T.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Caughman</span>, 
          <span class="cit-name-given-names">S.W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dancis</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Harford</span>, 
          <span class="cit-name-given-names">J.B.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Klausner</span>, 
          <span class="cit-name-given-names">R.D.</span>
         </span> (
         <span class="cit-pub-date">1987</span>) 
         <span class="cit-article-title">A cis-acting element is necessary and sufficient for translational regulation of human ferritin expression in response to iron</span>. 
         <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>, 
         <span class="cit-vol">84</span>, 
         <span class="cit-fpage">6730</span>–
         <span class="cit-lpage">6734</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DHentze%26rft.auinit1%253DM.%2BW.%26rft.volume%253D84%26rft.issue%253D19%26rft.spage%253D6730%26rft.epage%253D6734%26rft.atitle%253DA%2Bcis-acting%2Belement%2Bis%2Bnecessary%2Band%2Bsufficient%2Bfor%2Btranslational%2Bregulation%2Bof%2Bhuman%2Bferritin%2Bexpression%2Bin%2Bresponse%2Bto%2Biron.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.84.19.6730%26rft_id%253Dinfo%253Apmid%252F3477805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODQvMTkvNjczMCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzA0LzA2LzI5NDQ4OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
