<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/04/03/2020.03.05.976167.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.05.976167v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Direct RNA sequencing and early evolution of SARS-CoV-2" />
<meta name="DC.Identifier" content="10.1101/2020.03.05.976167" />
<meta name="DC.Date" content="2020-04-03" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Fundamental aspects of SARS-CoV-2 biology remain to be described, having the potential to provide insight to the response effort for this high-priority pathogen. Here we describe the first native RNA sequence of SARS-CoV-2, detailing the coronaviral transcriptome and epitranscriptome, and share these data publicly. A data-driven inference of viral genetic features and evolutionary rate is also made. The rapid sharing of sequence information throughout the SARS-CoV-2 pandemic represents an inflection point for public health and genomic epidemiology, providing early insights into the biology and evolution of this emerging pathogen." />
<meta name="DC.Contributor" content="George Taiaroa" />
<meta name="DC.Contributor" content="Daniel Rawlinson" />
<meta name="DC.Contributor" content="Leo Featherstone" />
<meta name="DC.Contributor" content="Miranda Pitt" />
<meta name="DC.Contributor" content="Leon Caly" />
<meta name="DC.Contributor" content="Julian Druce" />
<meta name="DC.Contributor" content="Damian Purcell" />
<meta name="DC.Contributor" content="Leigh Harty" />
<meta name="DC.Contributor" content="Thomas Tran" />
<meta name="DC.Contributor" content="Jason Roberts" />
<meta name="DC.Contributor" content="Nichollas Scott" />
<meta name="DC.Contributor" content="Mike Catton" />
<meta name="DC.Contributor" content="Deborah Williamson" />
<meta name="DC.Contributor" content="Lachlan Coin" />
<meta name="DC.Contributor" content="Sebastian Duchene" />
<meta name="article:published_time" content="2020-04-03" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Direct RNA sequencing and early evolution of SARS-CoV-2" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Fundamental aspects of SARS-CoV-2 biology remain to be described, having the potential to provide insight to the response effort for this high-priority pathogen. Here we describe the first native RNA sequence of SARS-CoV-2, detailing the coronaviral transcriptome and epitranscriptome, and share these data publicly. A data-driven inference of viral genetic features and evolutionary rate is also made. The rapid sharing of sequence information throughout the SARS-CoV-2 pandemic represents an inflection point for public health and genomic epidemiology, providing early insights into the biology and evolution of this emerging pathogen.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.05.976167v2" />
<meta name="citation_id" content="2020.03.05.976167v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.05.976167v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.05.976167v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.05.976167v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.05.976167" />
<meta name="citation_num_pages" content="31" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.05.976167" />
<meta name="citation_author" content="George Taiaroa" />
<meta name="citation_author_institution" content="Department of Microbiology and Immunology, University of Melbourne at The Peter Doherty Institute for Infection and Immunity" />
<meta name="citation_author_institution" content="Victorian Infectious Diseases Reference Laboratory, Royal Melbourne Hospital, at the Peter Doherty Institute for Infection and Immunity" />
<meta name="citation_author_email" content="george.taiaroa@unimelb.edu.au" />
<meta name="citation_author" content="Daniel Rawlinson" />
<meta name="citation_author_institution" content="Department of Microbiology and Immunology, University of Melbourne at The Peter Doherty Institute for Infection and Immunity" />
<meta name="citation_author" content="Leo Featherstone" />
<meta name="citation_author_institution" content="Department of Microbiology and Immunology, University of Melbourne at The Peter Doherty Institute for Infection and Immunity" />
<meta name="citation_author" content="Miranda Pitt" />
<meta name="citation_author_institution" content="Department of Microbiology and Immunology, University of Melbourne at The Peter Doherty Institute for Infection and Immunity" />
<meta name="citation_author" content="Leon Caly" />
<meta name="citation_author_institution" content="Victorian Infectious Diseases Reference Laboratory, Royal Melbourne Hospital, at the Peter Doherty Institute for Infection and Immunity" />
<meta name="citation_author" content="Julian Druce" />
<meta name="citation_author_institution" content="Victorian Infectious Diseases Reference Laboratory, Royal Melbourne Hospital, at the Peter Doherty Institute for Infection and Immunity" />
<meta name="citation_author" content="Damian Purcell" />
<meta name="citation_author_institution" content="Department of Microbiology and Immunology, University of Melbourne at The Peter Doherty Institute for Infection and Immunity" />
<meta name="citation_author" content="Leigh Harty" />
<meta name="citation_author_institution" content="Department of Microbiology and Immunology, University of Melbourne at The Peter Doherty Institute for Infection and Immunity" />
<meta name="citation_author" content="Thomas Tran" />
<meta name="citation_author_institution" content="Victorian Infectious Diseases Reference Laboratory, Royal Melbourne Hospital, at the Peter Doherty Institute for Infection and Immunity" />
<meta name="citation_author" content="Jason Roberts" />
<meta name="citation_author_institution" content="Victorian Infectious Diseases Reference Laboratory, Royal Melbourne Hospital, at the Peter Doherty Institute for Infection and Immunity" />
<meta name="citation_author" content="Nichollas Scott" />
<meta name="citation_author_institution" content="Department of Microbiology and Immunology, University of Melbourne at The Peter Doherty Institute for Infection and Immunity" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2556-8316" />
<meta name="citation_author" content="Mike Catton" />
<meta name="citation_author_institution" content="Victorian Infectious Diseases Reference Laboratory, Royal Melbourne Hospital, at the Peter Doherty Institute for Infection and Immunity" />
<meta name="citation_author" content="Deborah Williamson" />
<meta name="citation_author_institution" content="Department of Microbiology and Immunology, University of Melbourne at The Peter Doherty Institute for Infection and Immunity" />
<meta name="citation_author_institution" content="Department of Microbiology, Royal Melbourne Hospital" />
<meta name="citation_author" content="Lachlan Coin" />
<meta name="citation_author_institution" content="Department of Microbiology and Immunology, University of Melbourne at The Peter Doherty Institute for Infection and Immunity" />
<meta name="citation_author" content="Sebastian Duchene" />
<meta name="citation_author_institution" content="Department of Microbiology and Immunology, University of Melbourne at The Peter Doherty Institute for Infection and Immunity" />
<meta name="citation_reference" content="World Health Organization. Pneumonia of unknown cause — China. 2020 (https://www.who.int/csr/don/05-january-2020-pneumonia-of-unkown-cause-china/en/)." />
<meta name="citation_reference" content="United Nations Development Programme – March 2020. UNDP support for coronavirus-affected countries goes beyond health. https://www.undp.org/content/undp/en/home/blog/2020/undp-support-for-coronavirus-affected-countries-goes-beyond-heal.html" />
<meta name="citation_reference" content="Dong, Ensheng, Hongru Du, and Lauren Gardner. “An interactive web-based dashboard to track COVID-19 in real time.” The Lancet Infectious Diseases (2020). DOI: 10.1016/S1473-3099(20)30120-1" />
<meta name="citation_reference" content="World Health Organization. Statement on the second meeting of the International Health Regulations (2005) Emergency Committee regarding the outbreak of novel coronavirus (2019-nCoV). January 30, 2020 (https://www.who.int/news-room/detail/30-01-2020-statement-on-the-second-meeting-of-the-international-health-regulations-(2005)-emergency-committee-regarding-the-outbreak-of-novel-coronavirus-(2019-ncov)." />
<meta name="citation_reference" content="Lu, Roujian, et al. “Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding.” The Lancet (2020). DOI: 10.1016/S0140-6736(20)30251-8" />
<meta name="citation_reference" content="Peiris JS, Yuen KY, Osterhaus AD, Stöhr K. The severe acute respiratory syndrome. New England Journal of Medicine. 2003 Dec 18;349(25):2431–41. DOI: 10.1056/NEJMra032498" />
<meta name="citation_reference" content="Assiri A, McGeer A, Perl TM, Price CS, Al Rabeeah AA, Cummings DA, Alabdullatif ZN, Assad M, Almulhim A, Makhdoom H, Madani H. Hospital outbreak of Middle East respiratory syndrome coronavirus. New England Journal of Medicine. 2013 Aug 1;369(5):407–16. DOI: 10.1056/NEJMoa1306742" />
<meta name="citation_reference" content="Perlman, S. Another decade, another coronavirus. New England Journal of Medicine. 2020 February 20; 382:760–762. DOI: 10.1056/NEJMe2001126" />
<meta name="citation_reference" content="Shu Y, McCauley J. GISAID: Global initiative on sharing all influenza data–from vision to reality. Eurosurveillance. 2017 Mar 30;22(13). https://www.gisaid.org/" />
<meta name="citation_reference" content="Hadfield J, Megill C, Bell SM, Huddleston J, Potter B, Callender C, Sagulenko P, Bedford T, Neher RA. Nextstrain: real-time tracking of pathogen evolution. Bioinformatics. 2018 Dec 1;34(23):4121-3. https://nextstrain.org/ncov" />
<meta name="citation_reference" content="Andersen KG, Rambaut A, Lipkin WI, Holmes EC, Garry RF. The Proximal Origin of SARS-CoV-2. Virological, accessed on 27/02/2020. http://virological.org/t/the-proximal-origin-of-sars-cov-2/398" />
<meta name="citation_reference" content="Rambaut A. Phylodynamic Analysis 129 genomes 24 Feb 2020. Virological, accessed 27/02/2020. http://virological.org/t/phylodynamic-analysis-129-genomes-24-feb-2020/356" />
<meta name="citation_reference" content="Yount B, Curtis KM, Fritz EA, Hensley LE, Jahrling PB, Prentice E, Denison MR, Geisbert TW, Baric RS. Reverse genetics with a full-length infectious cDNA of severe acute respiratory syndrome coronavirus. Proceedings of the National Academy of Sciences. 2003 Oct 28;100(22):12995–3000. DOI: 10.1073/pnas.1735582100" />
<meta name="citation_reference" content="Brian DA, Baric RS. Coronavirus genome structure and replication. InCoronavirus replication and reverse genetics 2005 (pp. 1-30). Springer, Berlin, Heidelberg." />
<meta name="citation_reference" content="Chen Y, Cai H, Xiang N, Tien P, Ahola T, Guo D. Functional screen reveals SARS coronavirus nonstructural protein nsp14 as a novel cap N7 methyltransferase. Proceedings of the National Academy of Sciences. 2009 Mar 3;106(9):3484–9. DOI: 10.1073/pnas.0808790106" />
<meta name="citation_reference" content="Garalde DR, Snell EA, Jachimowicz D, Sipos B, Lloyd JH, Bruce M, Pantic N, Admassu T, James P, Warland A, Jordan M. Highly parallel direct RNA sequencing on an array of nanopores. Nature methods. 2018 Mar;15(3):201. DOI: 10.1038/nmeth.4577" />
<meta name="citation_reference" content="Schaecher SR, Mackenzie JM, Pekosz A (2007). The ORF7b Protein of Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) Is Expressed in Virus-Infected Cells and Incorporated into SARS-CoV Particles. Journal of Virology, 81(2), 718–731. DOI: 10.1128/JVI.01691-06" />
<meta name="citation_reference" content="Bojkova D, Klann K, Koch B, Widera M, Krause D, Ciesek S, Cinatl J, Münch C (2020) SARS-CoV-2 infected host cell proteomics reveal potential therapy targets. DOI: 10.21203/rs.3.rs-17218/v1" />
<meta name="citation_reference" content="Davidson AD, Williamson MK, Lewis S, Shoemark D, Carroll MW, Heesom K, Zambon M, Ellis J, Lewis PA, Hiscox JA, Matthews DA (2020) Characterisation of the transcriptome and proteome of SARS-CoV-2 using direct RNA sequencing and tandem mass spectrometry reveals evidence for a cell passage induced in-frame deletion in the spike glycoprotein that removes the furin-like cleavage site. DOI: 10.1101/2020.03.22.002204" />
<meta name="citation_reference" content="He R, Dobie F, Ballantine M, Leeson A, Li Y, Bastien N, Cutts T, Andonov A, Cao J, Booth TF, Plummer FA, Tyler S, Baker L, Li X (2004). Analysis of multimerization of the SARS coronavirus nucleocapsid protein. Biochemical and Biophysical Research Communications, 316(2), 476–483. DOI: 10.1016/j.bbrc.2004.02.074" />
<meta name="citation_reference" content="Goebel SJ, Hsue B, Dombrowski TF, Masters PS. Characterization of the RNA components of a putative molecular switch in the 3’ untranslated region of the murine coronavirus genome. J Virol. 2004 Jan;78(2):669–82. DOI: 10.1128/jvi.78.2.669-682.2004" />
<meta name="citation_reference" content="Tengs T, Kristoffersen AB, Bachvaroff TR, Jonassen CM. A mobile genetic element with unknown function found in distantly related viruses. Virol J. 2013 Apr 25;10:132. DOI: 10.1186/1743-422X-10-132" />
<meta name="citation_reference" content="Rangan R, Zheludev IN, Das R. RNA genome conservation and secondary structure in SARS-CoV-2 and SARS-related viruses. bioRxiv preprint 2020 DOI: 10.1101/2020.03.27.012906." />
<meta name="citation_reference" content="Robertson MP, Igel H, Baertsch R, Haussler D, Ares M, Scott WG. The Structure of a Rigorously Conserved RNA Element within the SARS Virus Genome. PLoS Biol. 2005 Jan; 3(1): e5. DOI: 10.1371/journal.pbio.0030005" />
<meta name="citation_reference" content="Viehweger A, Krautwurst S, Lamkiewicz K, Madhugiri R, Ziebuhr J, Hölzer M, Marz M. Direct RNA nanopore sequencing of full-length coronavirus genomes provides novel insights into structural variants and enables modification analysis. Genome research. 2019 Sep 1;29(9):1545–54. DOI: 10.1101/gr.247064.118" />
<meta name="citation_reference" content="Lichinchi G, Zhao BS, Wu Y, Lu Z, Qin Y, He C, Rana TM. Dynamics of human and viral RNA methylation during Zika virus infection. Cell host &amp; microbe. 2016 Nov 9;20(5):666–73. DOI: 10.1016/j.chom.2016.10.002" />
<meta name="citation_reference" content="Denison MR, Graham RL, Donaldson EF, Eckerle LD, Baric RS. Coronaviruses: an RNA proofreading machine regulates replication fidelity and diversity. RNA biology. 2011 Mar 1;8(2):270–9. DOI: 10.4161/rna.8.2.15013" />
<meta name="citation_reference" content="Chinese SARS Molecular Epidemiology Consortium. Molecular evolution of the SARS coronavirus during the course of the SARS epidemic in China. Science. 2004 Mar 12;303(5664):1666–9. DOI: 10.1126/science.1092002" />
<meta name="citation_reference" content="Dudas G, Carvalho LM, Rambaut A, Bedford T. MERS-CoV spillover at the camel-human interface. Elife. 2018 Jan 16;7:e31257. DOI: 10.7554/eLife.31257" />
<meta name="citation_reference" content="Cotten M, Watson SJ, Kellam P, Al-Rabeeah AA, Makhdoom HQ, Assiri A, Al-Tawfiq JA, Alhakeem RF, Madani H, AlRabiah FA, Al Hajjar S. Transmission and evolution of the Middle East respiratory syndrome coronavirus in Saudi Arabia: a descriptive genomic study. The Lancet. 2013 Dec 14;382(9909):1993–2002. DOI: 10.1016/S0140-6736(13)61887-5" />
<meta name="citation_reference" content="Andersen K. Clock and TMRCA based on 27 genomes. Virological, accessed on 27/02/2020. http://virological.org/t/clock-and-tmrca-based-on-27-genomes/347" />
<meta name="citation_reference" content="Bedford, T. Phylodynamic estimation of incidence and prevalence of novel coronavirus (nCoV) infections through time. Virological, accessed on 27/02/2020. http://virological.org/t/phylodynamic-estimation-of-incidence-and-prevalence-of-novel-coronavirus-ncov-infections-through-time/391" />
<meta name="citation_reference" content="Kearse M, Moir R, Wilson A, Stones-Havas S, Cheung M, Sturrock S, Buxton S, Cooper A, Markowitz S, Duran C, Thierer T. Geneious Basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data. Bioinformatics. 2012 Jun 15;28(12):1647–9." />
<meta name="citation_reference" content="Li H. Minimap2: pairwise alignment for nucleotide sequences. Bioinformatics. 2018 Sep 15;34(18):3094–100." />
<meta name="citation_reference" content="Milne I, Bayer M, Stephen G, Cardle L, Marshall D. Tablet: visualizing next-generation sequence assemblies and mappings. Bioinformatics. 2016 (pp. 253-268). Humana Press, New York, NY." />
<meta name="citation_reference" content="Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, Marth G, Abecasis G, Durbin R. The sequence alignment/map format and SAMtools. Bioinformatics. 2009 Aug 15;25(16):2078–9." />
<meta name="citation_reference" content="Wickham H. ggplot2: elegant graphics for data analysis. Springer; 2016 Jun 8" />
<meta name="citation_reference" content="R Core Team (2018). R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria." />
<meta name="citation_reference" content="Sato, Kengo, et al. “IPknot: fast and accurate prediction of RNA secondary structures with pseudoknots using integer programming.” Bioinformatics. 2011 27.13: i85-i93." />
<meta name="citation_reference" content="Darty, Kévin, Alain Denise, and Yann Ponty. “VARNA: Interactive drawing and editing of the RNA secondary structure.” Bioinformatics 2009 25.15: 1974." />
<meta name="citation_reference" content="Grant CE, Bailey TL, Noble WS. FIMO: scanning for occurrences of a given motif. Bioinformatics 2011 27: 1017–1018" />
<meta name="citation_reference" content="Vizcaino JA, Csordas A, del-Toro N, Dianes JA, Griss J, Lavidas I, et al. Update of the PRIDE database and its related tools. Nucleic Acids Res. 2016;44(D1):D447–56." />
<meta name="citation_reference" content="Cox J, Mann M. MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification. Nat Biotechnol. 2008;26(12):1367–72. DOI: 10.1038/nbt.1511." />
<meta name="citation_reference" content="Ferguson JM, Smith MA. SquiggleKit: A toolkit for manipulating nanopore signal data. Bioinformatics. 2019 Dec 15;35(24):5372–3." />
<meta name="citation_reference" content="Drummond AJ, Rambaut A. BEAST: Bayesian evolutionary analysis by sampling trees. BMC evolutionary biology. 2007 Dec;7(1):214." />
<meta name="citation_reference" content="Menardo F, Loiseau C, Brites D, Coscolla M, Gygli SM, Rutaihwa LK, Trauner A, Beisel C, Borrell S, Gagneux S. Treemmer: a tool to reduce large phylogenetic datasets with minimal loss of diversity. BMC Bioinformatics 2018 19(1), 164. DOI: 10.1186/s12859-018-2164-8" />
<meta name="citation_reference" content="Duchene S, Stadler T, Ho SY, Duchene DA, Dhanasekaran V, Baele G. Bayesian Evaluation of Temporal Signal in Measurably Evolving Populations. bioRxiv. 2019 Jan 1:810697." />
<meta name="citation_reference" content="Rambaut A, Drummond AJ, Xie D, Baele G, Suchard MA. Posterior summarization in Bayesian phylogenetics using Tracer 1.7. Systematic biology. 2018 Sep;67(5):901." />
<meta name="citation_reference" content="Cotten M, Watson SJ, Kellam P, Al-Rabeeah AA, Makhdoom HQ, Assiri A, Al-Tawfiq JA, Alhakeem RF, Madani H, AlRabiah FA, Al Hajjar S. Transmission and evolution of the Middle East respiratory syndrome coronavirus in Saudi Arabia: a descriptive genomic study. The Lancet. 2013 Dec 14;382(9909):1993–2002." />
<meta name="citation_reference" content="Cotten M, Watson SJ, Zumla AI, Makhdoom HQ, Palser AL, Ong SH, Al Rabeeah AA, Alhakeem RF, Assiri A, Al-Tawfiq JA, Albarrak A. Spread, circulation, and evolution of the Middle East respiratory syndrome coronavirus. MBio. 2014 Feb 28;5(1):e01062–13." />
<meta name="citation_reference" content="Dudas G, Carvalho LM, Rambaut A, Bedford T. MERS-CoV spillover at the camel-human interface. Elife. 2018 Jan 16;7:e31257." />
<meta name="citation_reference" content="Zhao Z, Li H, Wu X, Zhong Y, Zhang K, Zhang YP, Boerwinkle E, Fu YX. Moderate mutation rate in the SARS coronavirus genome and its implications. BMC evolutionary biology. 2004 Dec 1;4(1):21." />
<meta name="citation_reference" content="Salemi M, Fitch WM, Ciccozzi M, Ruiz-Alvarez MJ, Rezza G, Lewis MJ. Severe acute respiratory syndrome coronavirus sequence characteristics and evolutionary rate estimate from maximum likelihood analysis. Journal of virology. 2004 Feb 1;78(3):1602–3." />
<meta name="citation_reference" content="Wu SF D. CJ, Wan P, Chen TG, Li JQ, Li D, Zeng YJ, Zhu YP, He FC. The genome comparison of SARS-CoV and other coronaviruses. Yi chuan= Hereditas. 2003 Jul;25(4):373–82." />
<meta name="citation_reference" content="Chinese SARS Molecular Epidemiology Consortium. Molecular evolution of the SARS coronavirus during the course of the SARS epidemic in China. Science. 2004 Mar 12;303(5664):1666–9." />
<meta name="citation_reference" content="Sanjuán R. From molecular genetics to phylodynamics: evolutionary relevance of mutation rates across viruses. PLoS pathogens. 2012 May;8(5)." />
<meta name="citation_reference" content="Duchêne S, Geoghegan JL, Holmes EC, Ho SY. Estimating evolutionary rates using time-structured data: a general comparison of phylogenetic methods. Bioinformatics. 2016 32(22), 3375–3379." />
<meta name="twitter:title" content="Direct RNA sequencing and early evolution of SARS-CoV-2" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Fundamental aspects of SARS-CoV-2 biology remain to be described, having the potential to provide insight to the response effort for this high-priority pathogen. Here we describe the first native RNA sequence of SARS-CoV-2, detailing the coronaviral transcriptome and epitranscriptome, and share these data publicly. A data-driven inference of viral genetic features and evolutionary rate is also made. The rapid sharing of sequence information throughout the SARS-CoV-2 pandemic represents an inflection point for public health and genomic epidemiology, providing early insights into the biology and evolution of this emerging pathogen." />
<meta name="og-title" property="og:title" content="Direct RNA sequencing and early evolution of SARS-CoV-2" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.05.976167v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Fundamental aspects of SARS-CoV-2 biology remain to be described, having the potential to provide insight to the response effort for this high-priority pathogen. Here we describe the first native RNA sequence of SARS-CoV-2, detailing the coronaviral transcriptome and epitranscriptome, and share these data publicly. A data-driven inference of viral genetic features and evolutionary rate is also made. The rapid sharing of sequence information throughout the SARS-CoV-2 pandemic represents an inflection point for public health and genomic epidemiology, providing early insights into the biology and evolution of this emerging pathogen." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-04-03" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.05.976167v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.05.976167v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.05.976167v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.05.976167v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1225704" />
    <title>Direct RNA sequencing and early evolution of SARS-CoV-2 | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"RuD16r6uBc3iVQwM1a1i_PNZkEiXs5nIDwUMPcgPnkA","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1225704","apath":"\/biorxiv\/early\/2020\/04\/03\/2020.03.05.976167.atom","pisa":"biorxiv;2020.03.05.976167v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.05.976167v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.05.976167v2","title":"Direct RNA sequencing and early evolution of SARS-CoV-2","identifier":"node\/1225704"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022wIBHWG4h\\\\\\\/EHg2ISfZae7hJ\\\\\\\/KwVzT6EZD5k5QBRqhD4PhW4hPjBWkrvwHH1xz9P5ngfD2z5IQW1nsTOG0Vt7sgh332Dwt66+HHh2EK41fShPI4ZfUHbdMUtiBw27k3McfByPTHDnkJQiSJUBWgnlX+yBI+LRgiYe9mBnpUze3h1Net2SAAS987BhpSLRroSx0+VEZ9axVWekOzcnkgW\\\\\\\/VM5N1VuBi2pd+c39xj0WBDdiDhAEwB07GH5EujPaHMlb89IA+9DvW9VCT9fdbP\\\\\\\/I1KUBIvtZslK9HKKXqNe1o9zAnyHg3VvumjGaEZG6kYRDRhHXhqGElG3puoTOk7uMIdVjUbPwRoDPDzIenrhrl6wZlF6yIxZTIrpa0JGPc13i8xLkIAqaW4GPUsgEKuGmEXrivQ4cr2XdGZoFT16AMr8oGqhZA8TqierBQiwY9v7\\\\\\\/D7j8\\\\\\\/5dn9u6VVeitO8Qv0Uh\\\\\\\/sS6UT73xuaFDKfz2EJHCZFTxP8fRatn1HbrYo2v9fu\\\\\\\/VBB2b+\\\\\\\/15Qa7ALiRArI9F9hGyKX5nJlnbOYBfQbFMd73cmhcANJmk4rFDJQk+ma4TwtxVrxdBVzO\\\\\\\/9ZarlSNScFHEHzfjdIh\\\\\\\/adupnk0vz0h8vNKG24LkE7vYhnM+Sj2tL2ooTCd+yV7b01UN2a6QjXx9xczOs6oe8tnxGe7HplfVHgASb\\\\\\\/w3fuFagMppqZvubZL5DmWw42qU1D9gkB24r15kio0HxpVuUnLFHTeztBnUb6FV1igrU782kdZMYLeSwj3\\\\\\\/tkVPZ9zt6gaVHRhHaNtxyPQS6PocxjFS\\\\\\\/1J6LTlMguGWG\\\\\\\/piVPAVAc3TYKwK6SbkrTHvk8VU7sAio+sP\\\\\\\/jzH5zrV8pDlpggIMt3DyBeYeB7c8npIYf15zlqIRV6lbI8HyKS2b3yWEVVXVMzIBagwKYIKuiupLwlHwRB2xjUxhFSOP\\\\\\\/71RH062y+MCtBL5YUXf29I\\\\\\\/1WkG6jXGQug9pjtX0\\\\\\\/Rgtt9oEFhPfm7ZMfcZAM9pdxQPrE2vPPjDJF+zzZD5SDUmFpoANCPH6QpctGyzwUQRGGbOBx6Mk1dDTkE5TFjUdh\\\\\\\/bmXUEktv+ulaoyhJAjXW7MecqY14b6qGVqUUag70GyPBU9FN\\\\\\\/FMqiYSMXK00lhGHlN2gtNhTbilt0Y3pBIgU4IlhgLhj9wVjW4J\\\\\\\/aWgHbskc5ektBVdV8fN1VMGVHU90T\\\\\\\/1EfGJxdvFhTQNNJhD\\\\\\\/oRkoOLnqbItP8ORP5hQ\\\\\\\/VW385cQTc2mw+r1EfO2OOICv8IKqa+NfpnHDbWTEX3pvI+tlOrAZ4WN3cCWYRmzaPeNcOuxVUuvQDNFY7M4FfmhdpF39BSbVu\\\u0022,\\\u0022iv\\\u0022:\\\u0022FJPLLBjzov5dNnadWdJc\\\\\\\/g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022e9085d6beff282dca25511fdc201ebd4114183dc4fdac08b670f5f490efde9a5\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.05.976167v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1225704 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.05.976167v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-H11QhRF7zMXGbnXM9Fz4S1xqa0HvRfLYS5lQJhchURg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1225704" id="node1225704" data-pisa="biorxiv;2020.03.05.976167v2" data-pisa-master="biorxiv;2020.03.05.976167" data-apath="/biorxiv/early/2020/04/03/2020.03.05.976167.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Direct RNA sequencing and early evolution of SARS-CoV-2</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">George</span> <span class="nlm-surname">Taiaroa</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Daniel</span> <span class="nlm-surname">Rawlinson</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Leo</span> <span class="nlm-surname">Featherstone</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Miranda</span> <span class="nlm-surname">Pitt</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Leon</span> <span class="nlm-surname">Caly</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Julian</span> <span class="nlm-surname">Druce</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Damian</span> <span class="nlm-surname">Purcell</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Leigh</span> <span class="nlm-surname">Harty</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Thomas</span> <span class="nlm-surname">Tran</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Jason</span> <span class="nlm-surname">Roberts</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0003-2556-8316" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Nichollas</span> <span class="nlm-surname">Scott</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Mike</span> <span class="nlm-surname">Catton</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Deborah</span> <span class="nlm-surname">Williamson</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Lachlan</span> <span class="nlm-surname">Coin</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Sebastian</span> <span class="nlm-surname">Duchene</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.05.976167 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1225704" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">George Taiaroa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Microbiology and Immunology, University of Melbourne at The Peter Doherty Institute for Infection and Immunity</span>, Melbourne, <span class='nlm-country'>Australia</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Victorian Infectious Diseases Reference Laboratory, Royal Melbourne Hospital, at the Peter Doherty Institute for Infection and Immunity</span>, Victoria, <span class='nlm-country'>Australia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=George%2BTaiaroa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Taiaroa%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGeorge%2BTaiaroa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:george.taiaroa@unimelb.edu.au" class="" data-icon-position="" data-hide-link-title="0">george.taiaroa@unimelb.edu.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Daniel Rawlinson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Microbiology and Immunology, University of Melbourne at The Peter Doherty Institute for Infection and Immunity</span>, Melbourne, <span class='nlm-country'>Australia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BRawlinson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rawlinson%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BRawlinson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Leo Featherstone </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Microbiology and Immunology, University of Melbourne at The Peter Doherty Institute for Infection and Immunity</span>, Melbourne, <span class='nlm-country'>Australia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leo%2BFeatherstone%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Featherstone%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeo%2BFeatherstone%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Miranda Pitt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Microbiology and Immunology, University of Melbourne at The Peter Doherty Institute for Infection and Immunity</span>, Melbourne, <span class='nlm-country'>Australia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miranda%2BPitt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pitt%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMiranda%2BPitt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Leon Caly </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Victorian Infectious Diseases Reference Laboratory, Royal Melbourne Hospital, at the Peter Doherty Institute for Infection and Immunity</span>, Victoria, <span class='nlm-country'>Australia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leon%2BCaly%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Caly%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeon%2BCaly%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Julian Druce </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Victorian Infectious Diseases Reference Laboratory, Royal Melbourne Hospital, at the Peter Doherty Institute for Infection and Immunity</span>, Victoria, <span class='nlm-country'>Australia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julian%2BDruce%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Druce%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJulian%2BDruce%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Damian Purcell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Microbiology and Immunology, University of Melbourne at The Peter Doherty Institute for Infection and Immunity</span>, Melbourne, <span class='nlm-country'>Australia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Damian%2BPurcell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Purcell%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADamian%2BPurcell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Leigh Harty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Microbiology and Immunology, University of Melbourne at The Peter Doherty Institute for Infection and Immunity</span>, Melbourne, <span class='nlm-country'>Australia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leigh%2BHarty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Harty%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALeigh%2BHarty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Thomas Tran </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Victorian Infectious Diseases Reference Laboratory, Royal Melbourne Hospital, at the Peter Doherty Institute for Infection and Immunity</span>, Victoria, <span class='nlm-country'>Australia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BTran%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tran%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BTran%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Jason Roberts </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Victorian Infectious Diseases Reference Laboratory, Royal Melbourne Hospital, at the Peter Doherty Institute for Infection and Immunity</span>, Victoria, <span class='nlm-country'>Australia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BRoberts%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Roberts%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJason%2BRoberts%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Nichollas Scott </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Microbiology and Immunology, University of Melbourne at The Peter Doherty Institute for Infection and Immunity</span>, Melbourne, <span class='nlm-country'>Australia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nichollas%2BScott%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Scott%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANichollas%2BScott%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2556-8316" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nichollas Scott</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Mike Catton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Victorian Infectious Diseases Reference Laboratory, Royal Melbourne Hospital, at the Peter Doherty Institute for Infection and Immunity</span>, Victoria, <span class='nlm-country'>Australia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mike%2BCatton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Catton%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMike%2BCatton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Deborah Williamson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Microbiology and Immunology, University of Melbourne at The Peter Doherty Institute for Infection and Immunity</span>, Melbourne, <span class='nlm-country'>Australia</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Microbiology, Royal Melbourne Hospital</span>, Victoria, <span class='nlm-country'>Australia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Deborah%2BWilliamson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Williamson%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADeborah%2BWilliamson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Lachlan Coin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Microbiology and Immunology, University of Melbourne at The Peter Doherty Institute for Infection and Immunity</span>, Melbourne, <span class='nlm-country'>Australia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lachlan%2BCoin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Coin%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALachlan%2BCoin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Sebastian Duchene </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Microbiology and Immunology, University of Melbourne at The Peter Doherty Institute for Infection and Immunity</span>, Melbourne, <span class='nlm-country'>Australia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sebastian%2BDuchene%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Duchene%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASebastian%2BDuchene%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.05.976167v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1225704" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1225704/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.05.976167v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1225704" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1225704/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.05.976167v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1225704" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1225704/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.05.976167v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1225704" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1225704/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.05.976167v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1225704" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1225704/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.05.976167v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1225704" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1225704/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">Fundamental aspects of SARS-CoV-2 biology remain to be described, having the potential to provide insight to the response effort for this high-priority pathogen. Here we describe the first native RNA sequence of SARS-CoV-2, detailing the coronaviral transcriptome and epitranscriptome, and share these data publicly. A data-driven inference of viral genetic features and evolutionary rate is also made. The rapid sharing of sequence information throughout the SARS-CoV-2 pandemic represents an inflection point for public health and genomic epidemiology, providing early insights into the biology and evolution of this emerging pathogen.</p></div><div id="sec-1" class="subsection"><p id="p-4">The pandemic of severe acute respiratory syndrome 2 (SARS-CoV-2), causing the disease COVID-19 and originating in Wuhan, China, has spread to more than 200 countries and territories, and has caused more than 1,000,000 cases globally [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>-<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>]. SARS-CoV-2 is a positive-sense single-stranded RNA ((+)ssRNA) virus, belonging to the <em>Coronaviridae</em> family and <em>betacoronavirus</em> genus [<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>]. Related betacoronaviruses are capable of infection and ongoing transmission in mammalian and avian hosts, resulting in illness in humans such as Middle East respiratory syndrome (MERS) and the original severe acute respiratory syndrome (SARS) as examples [<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>-<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>]. Based on the limited sampling of potential reservoir species, SARS-CoV-2 has been found to be most similar to bat betacoronaviruses on a genomic level, potentially indicating that bats are a natural reservoir [<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>,<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>].</p><p id="p-5">The genome sequence of SARS-CoV-2 was rapidly determined and shared on January 5<sup>th</sup> of 2020, being 29,903 nucleotides in length, and annotated based on sequence similarity to other coronaviruses (GenBank: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MN908947.3&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.05.976167.atom">MN908947.3</a>). As the emergence of SARS-CoV-2 has escalated, genomic analyses have played a key role in public health responses, including in the design of appropriate molecular diagnostics and supporting epidemiological efforts to track and contain the outbreak [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>,<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>]. Taken together, publicly available sequence data suggest a recently occurring, point-source outbreak, as described in online sources [<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>-<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>].</p><p id="p-6">Aspects of the response assume that the biology of SARS-CoV-2 is comparable with previously characterised coronaviruses, including the annotation of genes and the estimation of molecular evolutionary rates [<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>-<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>]. It remains highly relevant to determine these features experimentally with SARS-CoV-2-specific data, potentially revealing other insights into the biology of this emergent pathogen. To address this, here we describe (i) the first native RNA sequence of SARS-CoV-2, detailing the coronaviral transcriptome and epitranscriptome, and (ii) estimates of coronaviral evolutionary rates and related timescales, based on data available at this stage of the outbreak.</p><p id="p-7">Characterised coronaviruses have some of the largest genomes among RNA viruses, and express their genetic content as a nested set of polyadenylated mRNA transcripts (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>), with lengths corresponding to each encoded open reading frame (ORF). These include two large ORFs, ORF1a and ORF1ab, encoded by the complete viral genome and expressed upon cell entry. Other subgenomic mRNAs are generated through a mechanism termed discontinuous extension of minus strands, encoding structural proteins (spike protein (S), envelope protein (E), membrane protein (M) and nucleocapsid protein (N)) and accessory proteins (3a, 6, 7a, 7b, 8 and 10). The subgenomic mRNAs have a common 5′ leader sequence, near-identical to that located in the 5′-UTR of the viral genome; the transcription mechanism repositions the 5′ leader sequence upstream of ORFs, with each translation start site being located at the primary position for ribosome scanning. These ORFs, although annotated, are yet to be shown as expressed experimentally. Standard sequencing technologies are unable to produce reads representing (i) complete RNA viral genomes or (ii) subgenomic mRNAs needed to verify annotated ORFs, as these methods generate short reads and have a reliance on amplification to generate complementary DNA (cDNA) sequences.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SARS-CoV-2 genetics and transcriptome architecture. A) Schematic of the early stages of SARS-CoV-2 cell entry and transcript production, including in vivo synthesis of positive sense genome-length RNA molecules and subgenomic mRNAs. B) Read coverage of direct RNA reads from cell-culture material, aligned to the local SARS-CoV-2 genome (29,893 bases), showing a bias towards the 3&#x2019; polyadenylated end. C) Read length histogram, showing subgenomic mRNAs attributed to coding sequences." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-943205205" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;SARS-CoV-2 genetics and transcriptome architecture.&lt;/span&gt; A) Schematic of the early stages of SARS-CoV-2 cell entry and transcript production, including &lt;em&gt;in vivo&lt;/em&gt; synthesis of positive sense genome-length RNA molecules and subgenomic mRNAs. B) Read coverage of direct RNA reads from cell-culture material, aligned to the local SARS-CoV-2 genome (29,893 bases), showing a bias towards the 3&#x2019; polyadenylated end. C) Read length histogram, showing subgenomic mRNAs attributed to coding sequences.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F1.medium.gif" width="440" height="427"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F1.medium.gif" width="440" height="427"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">SARS-CoV-2 genetics and transcriptome architecture.</span><p id="p-8" class="first-child">A) Schematic of the early stages of SARS-CoV-2 cell entry and transcript production, including <em>in vivo</em> synthesis of positive sense genome-length RNA molecules and subgenomic mRNAs. B) Read coverage of direct RNA reads from cell-culture material, aligned to the local SARS-CoV-2 genome (29,893 bases), showing a bias towards the 3’ polyadenylated end. C) Read length histogram, showing subgenomic mRNAs attributed to coding sequences.</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">To define the architecture of the coronaviral transcriptome, a recently established direct RNA sequencing approach was applied, using a highly parallel array of nanopores [<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>]. In brief, nucleic acids were prepared from cell culture material with high levels of SARS-CoV-2 growth, this being expected to include examples of both genomic mRNA and transcripts corresponding to each ORF. These were sequenced with Oxford Nanopore Technologies, including poly(T) adaptors and an R9.4 flowcell on a GridION platform (Oxford Nanopore Technologies). Through this approach, the electronic current is measured as individual strands of RNA translocate through a nanopore, with derived signal-space data basecalled to infer the corresponding nucleobases. As a comparator, virion material of SARS-CoV-2 was also prepared and sequenced through this approach, with complete viral genome sequences expected to predominate rather than subgenomic transcripts.</p><p id="p-10">The cellular-derived material was used to generate 680,347 reads, comprising 860Mb of sequence information (BioProject PRJNA608224). Aligning to the genome of the cultured SARS-CoV-2 isolate (MT007544.1), a subset of reads were attributed to coronavirus sequences (28.9%), comprising 367Mb of sequence distributed across the 29,893 base genome. Of these, a number had lengths &gt;20,000 bases, capturing the majority of the SARS-CoV-2 genome on a single molecule. This direct RNA sequencing approach generated an average 12,230 fold coverage of the coronaviral genome, biased towards sequences proximal to the polyadenylated 3’ tail; coverage ranged from 34 fold to &gt;160,000 fold (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>), reflecting the higher abundance of subgenomic mRNAs carrying these sequences, as well as the directional sequencing from the polyadenylated 3’ tail. The virion material generated fewer reads, and included a calibration standard added during library preparation (430,923 reads, BioProject PRJNA608224).</p><p id="p-11">Many features of SARS-CoV-2 biology are captured in these direct RNA sequence data, including the transcriptome, as well as RNA base modifications or ‘epitranscriptome’. To define the transcriptome, the shared 5’ leader sequence was used as a marker to identify intact transcripts, these corresponding to subgenomic mRNAs and having a low abundance in the virion-derived data (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Supplementary Figures 1</a> and <a id="xref-fig-5-1" class="xref-fig" href="#F5">2</a>). In SARS-CoV-2, we identify eight major viral mRNAs in addition to the viral genome; each annotated gene was observed as a distinct subgenomic mRNA, outside of ORF7b and ORF10 (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>, Supplementary Table 1). In SARS, ORF7a and ORF7b are encoded on a shared subgenomic mRNA, with translation of ORF7b being achieved through ribosome leaky scanning, explaining the absence of a dedicated ORF7b-encoding subgenomic mRNA [<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>]. There is however no satisfactory explanation for the absence of an ORF10-encoding subgenomic mRNA.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Native RNA sequence coverage of the SARS-CoV-2 genome for cell-culture and virion-derived material. A) Coverage of the SARS-CoV-2 genome for the cell-culture dataset, for all reads and for those predicted to be intact mRNA transcripts or &#x2018;leader reads&#x2019;, showing an abundance of such transcripts. B) Coverage of the SARS-CoV-2 genome for the virion-derived dataset, showing a relative paucity of intact mRNA transcripts." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-943205205" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Native RNA sequence coverage of the SARS-CoV-2 genome for cell-culture and virion-derived material. A) Coverage of the SARS-CoV-2 genome for the cell-culture dataset, for all reads and for those predicted to be intact mRNA transcripts or &#x2018;leader reads&#x2019;, showing an abundance of such transcripts. B) Coverage of the SARS-CoV-2 genome for the virion-derived dataset, showing a relative paucity of intact mRNA transcripts.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F4.medium.gif" width="440" height="283"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 1" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F4.medium.gif" width="440" height="283"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 1</span> <p id="p-24" class="first-child">Native RNA sequence coverage of the SARS-CoV-2 genome for cell-culture and virion-derived material. A) Coverage of the SARS-CoV-2 genome for the cell-culture dataset, for all reads and for those predicted to be intact mRNA transcripts or ‘leader reads’, showing an abundance of such transcripts. B) Coverage of the SARS-CoV-2 genome for the virion-derived dataset, showing a relative paucity of intact mRNA transcripts.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of native RNA reads between intact transcripts (&#x2018;leader reads&#x2019;) and other partial transcripts and genomic sequences. Intact transcripts include the leader sequence at the 5&#x2019; and a polyadenylated 3&#x2019; end (A and E, for cell-culture material and virion material respectively), while other partial transcripts or genomes either contain a leader sequence and lack an appropriate 3&#x2019; sequence (B and F), vice versa (C and D), or lack both a leader sequence and an expected 3&#x2019; end." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-943205205" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of native RNA reads between intact transcripts (&#x2018;leader reads&#x2019;) and other partial transcripts and genomic sequences. Intact transcripts include the leader sequence at the 5&#x2019; and a polyadenylated 3&#x2019; end (A and E, for cell-culture material and virion material respectively), while other partial transcripts or genomes either contain a leader sequence and lack an appropriate 3&#x2019; sequence (B and F), vice versa (C and D), or lack both a leader sequence and an expected 3&#x2019; end.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F5.medium.gif" width="440" height="435"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 2" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F5.medium.gif" width="440" height="435"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 2</span> <p id="p-25" class="first-child">Distribution of native RNA reads between intact transcripts (‘leader reads’) and other partial transcripts and genomic sequences. Intact transcripts include the leader sequence at the 5’ and a polyadenylated 3’ end (A and E, for cell-culture material and virion material respectively), while other partial transcripts or genomes either contain a leader sequence and lack an appropriate 3’ sequence (B and F), vice versa (C and D), or lack both a leader sequence and an expected 3’ end.</p><div class="sb-div caption-clear"></div></div></div><p id="p-12">ORF10 is the last predicted coding sequence upstream of the poly-A sequence, and the shortest of the predicted coding sequences at 117 bases in length. ORF10 also has no annotated function, and the putative encoded peptide does not appear in SARS-CoV-2 proteomes [<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>,<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>] or have a homolog in the SARS-CoV-1 proteome (Proteome ID: UP000000354 [<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>]). Subgenomic mRNAs corresponding to ORF10 are not identifiable in our reads (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Supplementary Figure 3</a>). These data suggest that the sequence currently annotated as ORF10 does not have a protein coding function in SARS-CoV-2. Ongoing molecular evolution at this locus should be considered in light of this finding.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Absence of observed coding potential for ORF10 in SARS-CoV-2. A) Read length histogram, showing subgenomic mRNAs attributed to coding sequences, with the area highlighted shown in detail in a second panel. B) Read length histogram, showing read counts of lengths corresponding to those of the ORF10 subgenomic mRNA (&#x223C;360 bases), if present in the dataset. Of the " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-943205205" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Absence of observed coding potential for ORF10 in SARS-CoV-2. A) Read length histogram, showing subgenomic mRNAs attributed to coding sequences, with the area highlighted shown in detail in a second panel. B) Read length histogram, showing read counts of lengths corresponding to those of the ORF10 subgenomic mRNA (&#x223C;360 bases), if present in the dataset. Of the &lt;500 base reads shown, none align to ORF10.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F6.medium.gif" width="440" height="209"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 3" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F6.medium.gif" width="440" height="209"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 3</span> <p id="p-26" class="first-child">Absence of observed coding potential for ORF10 in SARS-CoV-2. A) Read length histogram, showing subgenomic mRNAs attributed to coding sequences, with the area highlighted shown in detail in a second panel. B) Read length histogram, showing read counts of lengths corresponding to those of the ORF10 subgenomic mRNA (∼360 bases), if present in the dataset. Of the &lt;500 base reads shown, none align to ORF10.</p><div class="sb-div caption-clear"></div></div></div><p id="p-13">Instead of encoding a protein sequence, the locus annotated as ORF10 immediately upstream of the 3’ UTR may act itself or as a precursor of other RNAs in the regulation of gene expression, replication or modulating translation efficiency or cellular antiviral pathways; the 3’ UTR of coronaviruses contains domains critical for regulating viral RNA synthesis and other aspects of viral biology [<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>]. An initial region of the 3’ UTR appears essential for viral replication, and an area further 3’ includes the stem-loop II-like motif (s2m) a feature conserved in SARS-CoV-2 and other coronaviruses [<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>,<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>], the s2m having a proposed role in recruiting host translational machinery [<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>]. A small number of cell culture-derived SARS-CoV-2 genomes carry a shared deletion at an area of the 3’ UTR including an aspect of the s2m (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Supplementary Figure 4</a>), this parallel molecular evolution further suggesting the region may have functional roles <em>in vivo</em>.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Structured RNAs in the SARS-CoV-2 3&#x2019; UTR. A) An alignment of SARS-CoV-2 3&#x2019; UTR sequences, including the original Wuhan-Hu-1 sourced from Wuhan, China and considered the reference genome for the outbreak, and two examples of cultured SARS-CoV-2 isolates exhibiting deletions in a shared 3&#x2019; UTR region predicted to form a pseudoknot structure. B) Predicted pseudoknot structure of the SARS-CoV-2 3&#x2019;UTR affected by the above culture-derived deletions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-943205205" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Structured RNAs in the SARS-CoV-2 3&#x2019; UTR. A) An alignment of SARS-CoV-2 3&#x2019; UTR sequences, including the original Wuhan-Hu-1 sourced from Wuhan, China and considered the reference genome for the outbreak, and two examples of cultured SARS-CoV-2 isolates exhibiting deletions in a shared 3&#x2019; UTR region predicted to form a pseudoknot structure. B) Predicted pseudoknot structure of the SARS-CoV-2 3&#x2019;UTR affected by the above culture-derived deletions.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F7.medium.gif" width="440" height="435"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 4" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F7.medium.gif" width="440" height="435"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 4</span> <p id="p-27" class="first-child">Structured RNAs in the SARS-CoV-2 3’ UTR. A) An alignment of SARS-CoV-2 3’ UTR sequences, including the original Wuhan-Hu-1 sourced from Wuhan, China and considered the reference genome for the outbreak, and two examples of cultured SARS-CoV-2 isolates exhibiting deletions in a shared 3’ UTR region predicted to form a pseudoknot structure. B) Predicted pseudoknot structure of the SARS-CoV-2 3’UTR affected by the above culture-derived deletions.</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">An analysis of transcript breakpoints further illustrates the potential for 5’ UTR positions outside of the canonical leader sequence to enable transcript production, with low-frequency non-canonical variations in mRNA splice co-ordinates (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-fig-8-1" class="xref-fig" href="#F8">Supplementary Figures 5</a> and <a id="xref-fig-9-1" class="xref-fig" href="#F9">6</a>). Low frequency variants may be generated during the preparation of nucleic acids for sequencing, with the rate of chimeric read formation being unknown; this could be explored through analysis of <em>in vitro</em> transcribed RNA control material.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Extended breakpoint analysis of the SARS-CoV-2 transcriptome. The genome coordinates 3&#x2019; of the breakpoint are extended to include potential 3&#x2019; sequences positioned between 1-10,001 of the genome. This highlights low frequency breakpoints, increasing in frequency near the sequence annotated as ORF10 and the 3&#x2019; end of the genome. The key shows the distribution of transcript breakpoints. Colour is matched to a &#x2018;value&#x2019; measuring the number of reads with break points at that position, log10-scaled. The histogram component illustrates the number of transcripts with a given abundance value." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-943205205" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Extended breakpoint analysis of the SARS-CoV-2 transcriptome. The genome coordinates 3&#x2019; of the breakpoint are extended to include potential 3&#x2019; sequences positioned between 1-10,001 of the genome. This highlights low frequency breakpoints, increasing in frequency near the sequence annotated as ORF10 and the 3&#x2019; end of the genome. The key shows the distribution of transcript breakpoints. Colour is matched to a &#x2018;value&#x2019; measuring the number of reads with break points at that position, log10-scaled. The histogram component illustrates the number of transcripts with a given abundance value.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F8.medium.gif" width="346" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 5" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F8.medium.gif" width="346" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 5</span> <p id="p-28" class="first-child">Extended breakpoint analysis of the SARS-CoV-2 transcriptome. The genome coordinates 3’ of the breakpoint are extended to include potential 3’ sequences positioned between 1-10,001 of the genome. This highlights low frequency breakpoints, increasing in frequency near the sequence annotated as ORF10 and the 3’ end of the genome. The key shows the distribution of transcript breakpoints. Colour is matched to a ‘value’ measuring the number of reads with break points at that position, log10-scaled. The histogram component illustrates the number of transcripts with a given abundance value.</p><div class="sb-div caption-clear"></div></div></div><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="ORF-specific breakpoint analyses. The corresponding breakpoints for each currently annotated ORF in the SARS-CoV-2 genome are shown (A-I), highlighting a canonical breakpoint for ORFs with a corresponding subgenome mRNA, and a low frequency of non-canonical splice sites often centred on a canonical site. Of note, low frequency splice sites can be seen for an area between ORF 7a and ORF8, likely corresponding to ORF7b (G). There is an absence of splice sites for ORF in this dataset (I). The key shows the distribution of transcript breakpoints. Colour is matched to a &#x2018;value&#x2019; measuring the number of reads with break points at that position, log10-scaled. The histogram component illustrates the number of transcripts with a given abundance value." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-943205205" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;ORF-specific breakpoint analyses. The corresponding breakpoints for each currently annotated ORF in the SARS-CoV-2 genome are shown (A-I), highlighting a canonical breakpoint for ORFs with a corresponding subgenome mRNA, and a low frequency of non-canonical splice sites often centred on a canonical site. Of note, low frequency splice sites can be seen for an area between ORF 7a and ORF8, likely corresponding to ORF7b (G). There is an absence of splice sites for ORF in this dataset (I). The key shows the distribution of transcript breakpoints. Colour is matched to a &#x2018;value&#x2019; measuring the number of reads with break points at that position, log10-scaled. The histogram component illustrates the number of transcripts with a given abundance value.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F9.medium.gif" width="320" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 6" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F9.medium.gif" width="320" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 6</span> <p id="p-29" class="first-child">ORF-specific breakpoint analyses. The corresponding breakpoints for each currently annotated ORF in the SARS-CoV-2 genome are shown (A-I), highlighting a canonical breakpoint for ORFs with a corresponding subgenome mRNA, and a low frequency of non-canonical splice sites often centred on a canonical site. Of note, low frequency splice sites can be seen for an area between ORF 7a and ORF8, likely corresponding to ORF7b (G). There is an absence of splice sites for ORF in this dataset (I). The key shows the distribution of transcript breakpoints. Colour is matched to a ‘value’ measuring the number of reads with break points at that position, log10-scaled. The histogram component illustrates the number of transcripts with a given abundance value.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Breakpoint analysis of the SARS-CoV-2 transcriptome. Direct RNA reads carrying a breakpoint relative to the 5&#x2019; leader sequence are shown, representing potentially viable transcripts. These breakpoints are localised at the same position on the leader sequence (positions 62-68), and on the 3&#x2019; to predicted transcription regulating sequences in the body of the genome (TRS-Bs, highlighted by vertical weight lines), generating common subgenomic mRNAs. Of note, many low frequency breakpoints are detected, although few near the sequence currently annotated as ORF10. The key shows the distribution of transcript breakpoints. Colour is matched to a &#x2018;value&#x2019; measuring the number of reads with break points at that position, log10-scaled. The histogram component illustrates the number of transcripts with a given abundance value." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-943205205" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Breakpoint analysis of the SARS-CoV-2 transcriptome.&lt;/span&gt; Direct RNA reads carrying a breakpoint relative to the 5&#x2019; leader sequence are shown, representing potentially viable transcripts. These breakpoints are localised at the same position on the leader sequence (positions 62-68), and on the 3&#x2019; to predicted transcription regulating sequences in the body of the genome (TRS-Bs, highlighted by vertical weight lines), generating common subgenomic mRNAs. Of note, many low frequency breakpoints are detected, although few near the sequence currently annotated as ORF10. The key shows the distribution of transcript breakpoints. Colour is matched to a &#x2018;value&#x2019; measuring the number of reads with break points at that position, log10-scaled. The histogram component illustrates the number of transcripts with a given abundance value.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F2.medium.gif" width="332" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F2.medium.gif" width="332" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Breakpoint analysis of the SARS-CoV-2 transcriptome.</span><p id="p-15" class="first-child">Direct RNA reads carrying a breakpoint relative to the 5’ leader sequence are shown, representing potentially viable transcripts. These breakpoints are localised at the same position on the leader sequence (positions 62-68), and on the 3’ to predicted transcription regulating sequences in the body of the genome (TRS-Bs, highlighted by vertical weight lines), generating common subgenomic mRNAs. Of note, many low frequency breakpoints are detected, although few near the sequence currently annotated as ORF10. The key shows the distribution of transcript breakpoints. Colour is matched to a ‘value’ measuring the number of reads with break points at that position, log10-scaled. The histogram component illustrates the number of transcripts with a given abundance value.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">In addition to RNA modifications such as the methylation of the 5’ cap structure and polyadenylation of the 3’ terminus needed for efficient translation of coding sequences, other RNA modifications may have functional roles in SARS-CoV-2 [<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>]. A range of modifications are identifiable using direct RNA sequence data [<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>,<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>]; our available SARS-CoV-2 direct RNA sequence data providing adequate coverage to confidently call specific modifications. Through analysis of signal-space data, we identified 42 positions with predicted 5-methylcytosine modifications, appearing at consistent positions between subgenomic mRNAs (<a id="xref-fig-10-1" class="xref-fig" href="#F10">Supplementary Figure 7</a>, and Supplementary Table 2). In other positive ssRNA viruses, RNA methylation can change dynamically during the course of infection [<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>], influencing host-pathogen interaction and viral replication. Other modifications may become apparent once training datasets are available for direct RNA sequence data, with little known of the epitranscriptomic landscape of coronaviruses [<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">25</a>,<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>].</p><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Subgenomic mRNA abundance and predicted sites of modification. A) Coverage of relevant coding sequences achieved by alignment of subgenomic mRNAs to the SARS-CoV-2 genome (log scale). Red lines indicate the first base of each coding sequence from ORF2-10. B) Schematic of relevant annotated coding sequences. C) Position of predicted m5C positions in subgenomic mRNAs. Dark blue lines indicate positions predicted to have &gt;90% base modification; light blue lines indicate positions predicted to have between 50% and 90% base modification." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-943205205" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Subgenomic mRNA abundance and predicted sites of modification. A) Coverage of relevant coding sequences achieved by alignment of subgenomic mRNAs to the SARS-CoV-2 genome (log scale). Red lines indicate the first base of each coding sequence from ORF2-10. B) Schematic of relevant annotated coding sequences. C) Position of predicted m5C positions in subgenomic mRNAs. Dark blue lines indicate positions predicted to have &gt;90% base modification; light blue lines indicate positions predicted to have between 50% and 90% base modification.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F10.medium.gif" width="440" height="257"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 7" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F10.medium.gif" width="440" height="257"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 7</span> <p id="p-30" class="first-child">Subgenomic mRNA abundance and predicted sites of modification. A) Coverage of relevant coding sequences achieved by alignment of subgenomic mRNAs to the SARS-CoV-2 genome (log scale). Red lines indicate the first base of each coding sequence from ORF2-10. B) Schematic of relevant annotated coding sequences. C) Position of predicted m5C positions in subgenomic mRNAs. Dark blue lines indicate positions predicted to have &gt;90% base modification; light blue lines indicate positions predicted to have between 50% and 90% base modification.</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">As well as investigating the above assumed features of SARS-CoV-2 genetics, sequence data also enable an estimate of the molecular evolutionary rate, with globally sourced genome sequences being shared and publicly available. Evolutionary rate estimates from other coronaviruses such as Middle East Respiratory Syndrome (MERS) are not necessarily applicable here, particularly because MERS had multiple independent introductions into humans [<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>-<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>]. To estimate the evolutionary rate and time of origin of the SARS-CoV-2 outbreak, we carried out Bayesian phylogenetic analyses using a curated set of 122 high quality publicly available SARS-CoV-2 genome sequences, each having a known collection date (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, Supplementary Table 3). The sampling times were sufficient to calibrate a molecular clock and infer the evolutionary rate and timescale of the outbreak using a Bayesian approach; the evolutionary rate of SARS-CoV-2 was estimated to be 1.20 × 10<sup>−3</sup> substitutions/site/year (95% HPD 8.91×10<sup>−4</sup> - 1.52×10<sup>−3</sup>), and the of time of origin to be late November 2019 (95% HPD August 2019, December 2019), which is in agreement with epidemiological evidence and other recent analyses (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>) [<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>-<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>].</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Assessment of viral evolutionary rate and outbreak timing with SARS-CoV-2-specific data. A) A timed highest clade-credibility phylogenetic tree of curated SARS-CoV-2 genomes as inferred in BEAST. B) Comparison of SARS-CoV-2 rate estimates with varying datasets, and previously published estimates of other coronaviruses." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-943205205" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Assessment of viral evolutionary rate and outbreak timing with SARS-CoV-2-specific data.&lt;/span&gt; A) A timed highest clade-credibility phylogenetic tree of curated SARS-CoV-2 genomes as inferred in BEAST. B) Comparison of SARS-CoV-2 rate estimates with varying datasets, and previously published estimates of other coronaviruses.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F3.medium.gif" width="320" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F3.medium.gif" width="320" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">Assessment of viral evolutionary rate and outbreak timing with SARS-CoV-2-specific data.</span><p id="p-18" class="first-child">A) A timed highest clade-credibility phylogenetic tree of curated SARS-CoV-2 genomes as inferred in BEAST. B) Comparison of SARS-CoV-2 rate estimates with varying datasets, and previously published estimates of other coronaviruses.</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">A further set of 66 high quality genomes collected earlier in the outbreak (Supplementary Table 4), and maximal diversity data set from all data available in GISAID to March 28th to show the utility of capturing varying degrees of genetic diversity (Supplementary Table 5). This Bayesian approach demonstrated improved precision in estimates of evolutionary rates using our dataset with highest genetic diversity. This may be explained by the stochastic variation typical in data from early in an outbreak having a smaller impact as the virus accumulates genetic variation. These results are also supported by root-to-tip regression, a visual assessment of the degree of clocklike evolution in the data (<a id="xref-fig-11-1" class="xref-fig" href="#F11">Supplementary Figures 8</a> and <a id="xref-fig-12-1" class="xref-fig" href="#F12">9</a>). The evolutionary rate generated the high diversity set of 100 genomes was 1.56 × 10<sup>−3</sup> substitutions/site/year (95% HPD 1.09×10<sup>−3</sup> - 2.05×10<sup>−3</sup>), whereas that based on 66 genomes was 1.16 × 10<sup>−3</sup> substitutions/site/year (95% HPD 6.32×10<sup>−4</sup> - 1.69×10<sup>−3</sup>). Our estimate of the evolutionary rate of SARS-CoV-2 is in line with those of other coronaviruses (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>), and the low genomic diversity and recent timescale of the outbreak support a recently occurring, point-source transfer to humans.</p><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Assessment of SARS-CoV-2 phylogenetics and viral evolutionary rate based on 66 early genomes made publicly available. A) A timed highest clade-credibility phylogenetic tree of curated SARS-CoV-2 66 genomes as inferred in BEAST. B) Comparison of the SARS-CoV-2 rate estimate for the n=66 set and previously published estimates of other coronaviruses." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-943205205" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Assessment of SARS-CoV-2 phylogenetics and viral evolutionary rate based on 66 early genomes made publicly available. A) A timed highest clade-credibility phylogenetic tree of curated SARS-CoV-2 66 genomes as inferred in BEAST. B) Comparison of the SARS-CoV-2 rate estimate for the n=66 set and previously published estimates of other coronaviruses.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F11.medium.gif" width="298" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 8" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F11.medium.gif" width="298" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 8</span> <p id="p-31" class="first-child">Assessment of SARS-CoV-2 phylogenetics and viral evolutionary rate based on 66 early genomes made publicly available. A) A timed highest clade-credibility phylogenetic tree of curated SARS-CoV-2 66 genomes as inferred in BEAST. B) Comparison of the SARS-CoV-2 rate estimate for the n=66 set and previously published estimates of other coronaviruses.</p><div class="sb-div caption-clear"></div></div></div><div id="F12" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F12.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Time-to-most-common-recent-ancestor (TMRCA)and root-to-tip regression of both early and maximally diverse SARS-CoV-2 genome datasets. A) TMRCA and root-to-tip regression of 122 high quality complete SARS-CoV-2 genomes made available early in the pandemic. B) TMRCA and root-to-tip regression of 100 maximally diverse SARS-CoV-2 genomes, selected from the first 700 genomes made publicly available." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-943205205" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Time-to-most-common-recent-ancestor (TMRCA)and root-to-tip regression of both early and maximally diverse SARS-CoV-2 genome datasets. A) TMRCA and root-to-tip regression of 122 high quality complete SARS-CoV-2 genomes made available early in the pandemic. B) TMRCA and root-to-tip regression of 100 maximally diverse SARS-CoV-2 genomes, selected from the first 700 genomes made publicly available.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F12.medium.gif" width="415" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 9" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F12.medium.gif" width="415" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 9</span> <p id="p-32" class="first-child">Time-to-most-common-recent-ancestor (TMRCA)and root-to-tip regression of both early and maximally diverse SARS-CoV-2 genome datasets. A) TMRCA and root-to-tip regression of 122 high quality complete SARS-CoV-2 genomes made available early in the pandemic. B) TMRCA and root-to-tip regression of 100 maximally diverse SARS-CoV-2 genomes, selected from the first 700 genomes made publicly available.</p><div class="sb-div caption-clear"></div></div></div><div id="F13" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F13.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of SARS-CoV-2 sampling times used to generate publicly available genomes. The distribution has notable deviations from an expected exponential growth in the number of genomes available, such as in mid-February, with constant sampling being an underlying assumption for many phylodynamic skyline approaches inferring effective population size." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-943205205" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of SARS-CoV-2 sampling times used to generate publicly available genomes. The distribution has notable deviations from an expected exponential growth in the number of genomes available, such as in mid-February, with constant sampling being an underlying assumption for many phylodynamic skyline approaches inferring effective population size.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 10" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F13.medium.gif" width="440" height="423"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 10" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F13.medium.gif" width="440" height="423"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 10" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/03/2020.03.05.976167/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 10</span> <p id="p-33" class="first-child">Distribution of SARS-CoV-2 sampling times used to generate publicly available genomes. The distribution has notable deviations from an expected exponential growth in the number of genomes available, such as in mid-February, with constant sampling being an underlying assumption for many phylodynamic skyline approaches inferring effective population size.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">Other phylodynamic inferences may soon become possible for SARS-CoV-2, as further genomic data becomes available and the sampling rate becomes more consistent. The current distribution of sampling times (<a id="xref-fig-11-2" class="xref-fig" href="#F11">Supplementary Figure 8</a>) appears to be prohibitive to phylodynamic inference of the SARS-CoV-2 effective population size (N<sub>e</sub>, not included here). Although a required threshold of genomes to allow such phylodynamic investigation may have been crossed, the temporal spread of these isolates may differ too much to satisfy constant sampling assumptions underlying many phylodynamic skyline approaches inferring N<sub>e</sub> over time. Again, as sampling continues a more consistent rate of sampling is likely to emerge, allowing such analyses.</p><p id="p-21">Insights are provided on the molecular biology of SARS-CoV-2, revealed through the use of direct RNA sequence and publicly available data. The rapid sharing of these and other genetic data support the global response effort and represents an inflection point for communicable diseases and genomic epidemiology, with complete data shared openly and rapidly between academic and public health groups.</p></div><div class="section" id="sec-4"><h2 class="">Methods</h2><div class="section" id="sec-5"><h3>Samples for direct RNA sequencing</h3><p id="p-34">The SARS-CoV-2 material was prepared from the first Australian case of COVID-2019 (Australia/VIC01/2020), maintained in cell culture. In brief, African green monkey kidney cells expressing the human signalling lymphocytic activation molecule (SLAM; termed Vero/hSLAM cells accordingly) with associated SARS-CoV-2 infection were grown at 37°C at 5% CO<sub>2</sub> in media consisting of 10 mL Earle’s minimum essential medium, 7% FBS (Bovogen Biologicals, Keilor East, Aus), 2 mM L-Glutamine, 1 mM Sodium pyruvate, 1500 mg/L sodium bicarbonate, 15 mM HEPES and 0.4 mg/ml geneticin in 25cm<sup>2</sup> flasks. This isolate is to the best of our knowledge typical for SARS-CoV-2 isolates, with the genome of the cultured isolate (MT007544.1) having three single nucleotide variants (T19065C, T22303G, G26144T) relative to the SARS-CoV-2 Wuhan-Hu-1 reference genome (MN908947.3), and a 10 base deletion in the 3’ UTR. Both the T22303G and 3’ UTR variants have been confirmed as culture-derived through Sanger sequencing of clinical and culture material, and do not appear in the earlier virion-derived data.</p><p id="p-35">Nucleic acids were prepared from clarified cell-free supernatant (reflecting virion material) and infected cell culture material (representing actively transcribed and translated viral material), following inactivation with linear acrylamide and ethanol. RNA was extracted from 100µl of supernatant and a modest pellet for the cell-culture material (∼200mg) respectively, using manually prepared wide-bore pipette tips and minimal steps to maintain RNA length for long read sequencing, and a QIAamp Viral RNA Mini Kit (Qiagen, Hilden, Germany). Carrier RNA was not added to Buffer AVL, with 1% linear acrylamide (Life Technologies, Carlsbad, CA, USA) added instead. Wash buffer AW1 was omitted from the purification stage, with RNA eluted in 50 μl of nuclease free water, followed by DNase treatment with Turbo DNase (Thermo Fisher Scientific, Waltham, MA, USA) 37°C for 30 min. RNA was cleaned and concentrated to 10 μl using the RNA Clean &amp; Concentrator-5 kit (Zymo Research, Irvine, CA, USA), as per manufacturer’s instructions.</p></div><div class="section" id="sec-6"><h3>Nanopore sequencing of direct RNA</h3><p id="p-36">Prepared RNA (∼1µg) was carried into a direct RNA sequence library preparation with the Oxford Nanopore DRS protocol (SQK-RNA002, Oxford Nanopore Technologies) following the manufacturer’s specifications, with addition of the control RNA in the virion sample. Libraries were loaded on R9.4 flow cells and sequenced on a GridION device for the cell-derived material and a MinION device for the virion-derived material (Oxford Nanopore Technologies), and sequenced for 40 hours. Signal-space data was used to generate nucleobase sequences (‘basecalled’) using Guppy, either as a standalone program or as ont-guppy-for-gridion 3.0.6. Both signal-space and basecalled read data are available at BioProject PRJNA608224. It should be noted that non-polyadenylated RNAs are not expected to be detected with this approach.</p></div><div class="section" id="sec-7"><h3>Characterisation of SARS-CoV-2 transcriptome architecture</h3><p id="p-37">Direct RNA reads passing the above given quality thresholds were aligned to the genome of the cultured Australian SARS-COV-2 isolate (MT007544.1), with parallel and concordant analyses in Geneious Prime (2019.2.1, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">[M1]</a>) and minimap2 v 2.11 using the “spliced” preset <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">[M2]</a>. Coverage statistics were determined from the resulting read alignments. To identify complete subgenomic mRNAs, reads were aligned to a 62 base SARS-COV-2 leader sequence (5’ACCUUCCCAGGUAACAAACCAACCAACUUUCGAUCUCUUGUAGAU CUGUUCUCUAAACGAAC), with reads aligning to the leader sequence being pooled and visualized in a length histogram. Significant peaks were identified visually and confirmed with a smoothed z-score algorithm. Reads captured in this binning-by-length strategy were re-aligned to the reference genome using the above methods and visualized in Tablet <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">[M3]</a>. Subgenome bins were refined to remove reads which did not originate at the 3’ poly-A tail as expected for intact subgenomic mRNAs, or which had leader sequences at least 10bp longer than expected. Subgenome bins were re-aligned, with coverage calculated in SAMtools <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">[M4]</a>, and plotted using ggplot2 <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">[M5]</a> in R <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">[M6]</a>. Breakpoints in mRNAs were determined with CIGAR string manipulation; any given spliced region longer than 100bp (represented by Ns in the CIGAR string after aligning with minimap2) was regarded as a spliced transcript and the 5’ and 3’ genome co-ordinates of the breakpoint were recorded for analysis. The IPKnot webserver <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">[M7]</a> was used to predict the RNA secondary structures, and the VARNA visualization applet <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">[M8]</a> to produce schematics.</p><p id="p-38">As an alternate method of defining the SARS-CoV-2 transcriptome, reads carrying a breakpoint relative to the 5’ leader sequence are shown, representing potentially viable transcripts. This was determined through CIGAR string manipulation. Any spliced region longer than 100bp (represented by Ns in the CIGAR string after aligning with minimap2) was regarded as a spliced transcript and the 5’ and 3’ genome co-ordinates of the breakpoint were recorded for analysis. Locations of Transcription Regulating Sequence in the body of the genome (TRS-B) were determined with a Position Weight Matrix (PWM) search. Portions of reads aligning to the conserved TRS in the leader sequence (TRS-L) were transformed into a count matrix, which was then passed into the FIMO program version 5.5.1 for motif detection with NRDB as the background distribution <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">[M9]</a>. Detected TRS-B sites are plotted alongside breakpoint heatmaps.</p></div><div class="section" id="sec-8"><h3>Data availability</h3><p id="p-39">All signal-space (fast5) and basecalled data (fastq) generated in this work are publicly available on the sequence read archive (SRA), as part of the BioProject PRJNA608224 (See Supplementary Table 6 for relevant accession numbers).</p></div><div class="section" id="sec-9"><h3>Identification of 5mC methylation</h3><p id="p-40">Nanopore sequencing preserves <em>in vivo</em> base modifications and enables their detection from raw voltage signal information. In brief, the signal-space fast5 files corresponding to identified subgenomic mRNAs were assessed to identify signal changes corresponding to 5mC methylation. These were first retrieved using the fast5_fetcher_multi function in SquiggleKit <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">[M10]</a>. Reads were processed to align raw signal with basecalled sequence data using Tombo v1.5 [<a href="https://github.com/nanoporetech/tombo">https://github.com/nanoporetech/tombo</a>]. Canonical reference sequences were made for each subgenomic mRNAs, with the binned fast5 files input into the detect_modifications function, with 5mC as the alternate-model parameter. Outputs were converted to dampened_fraction wiggle files and exported for visualization and analysis.</p></div><div class="section" id="sec-10"><h3>Assessment of publicly available proteomes</h3><p id="p-41">Proteomic datasets were downloaded from the PRIDE proteomic database <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">[M11]</a> (Pride accession: PXD017710) and processes using Maxquant (1.6.3.4 <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">[M12]</a>) allowing semi-specific free-N-terminus tryptic as a protease specificity. Quantitation was set to TMT11 plex labelling and human proteome (Uniprot: UP000005640) and SARS-CoV2 (build in house) databases were used. Additional searches were made of the SARS-CoV reference proteome (Proteome ID: UP000000354), and recently available SARS-CoV-2 proteomic data [<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a>].</p></div><div class="section" id="sec-11"><h3>SARS-CoV-2 Phylogenetics</h3><p id="p-42">In order to estimate the evolutionary rate and time of origin of SARS-CoV-2, we carried out phylogenetic analyses in BEAST v1.101 [M13 on three datasets. The first dataset included 66 high quality genomes available up to February 10th 2020, the second consisted of 122 available up to February 24th 2020, both from GISAID and GenBank (Supplementary Table 3). A third maximal diversity dataset (n=100) was included to demonstrate the utility of capturing varying degrees of genetic diversity, these genomes being selected from the first 700 genomes available on GISAID and maximised phylogenetic diversity achieved using Treemer <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">[M14]</a> (Supplementary Table 4). Temporal signal was assessed using BETS <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">[M15]</a>. Initially we determined whether the evolutionary signal and time over which the genome data were collected was sufficient to calibrate the molecular clock, allowing for the evolutionary rate and timescale of the outbreak to be inferred. The model selection approach from BETS supported a strict molecular clock model with genome sampling times for calibration and a coalescent exponential tree prior, which posits that the number of infected individuals grows exponentially over time. We used the HKY+Γ substitution model, and set the following priors for key parameters:</p><ul class="list-unord " id="list-1"><li id="list-item-1"><p id="p-43">A continuous time Markov chain for the evolutionary rate</p></li><li id="list-item-2"><p id="p-44">A Laplace distribution with mean of 0 and scale of 100 for the growth rate</p></li><li id="list-item-3"><p id="p-45">An exponential distribution with mean of 1 for the effective population size.</p></li></ul><p id="p-46">A Markov chain Monte Carlo of length 10<sup>7</sup> was set, sampling every 10<sup>3</sup> steps, and assessed sufficient sampling by verifying that the effective sample size for all parameters was at least 200 as determined in Tracer <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">[M16]</a>, automatically discarding 10% of the burn in. We summarised the posterior distribution of phylogenetic trees by selecting the highest clade credibility tree alongside calculating posterior node probabilities and the distribution of node ages. Comparison to other coronaviral evolutionary rates included studies [M17-24].</p><p id="p-47">A root-to-tip regression usually produces lower evolutionary rate estimates than explicit phylogenetic methods <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">[M25]</a>, although is commonly used to inspect temporal signal in the data. In our analyses, the data set that maximised phylogenetic diversity had a higher R<sup>2</sup> than that with 122 samples collected earlier, and an evolutionary rate that was more similar to that obtained in BEAST. Although, both data sets had temporal signal according to BETS, the root-to-tip regressions demonstrate that including more genetic diversity can produce improved estimates, probably because stochastic variation has a stronger impact in smaller data sets that are collected early in the outbreak.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-22">The authors gratefully acknowledge the traditional peoples of the land on which the work was carried out, the Wurundjeri Woi-wurrung people of the Kulin nation.</p><p id="p-23">The authors also acknowledge the broader staff of the Victorian Infectious Disease Reference Laboratory (VIDRL), their public health partners, and VIDRL’s major funder, the Victorian Department of Health and Human Services. The authors also thank David Matthews and Andrew Davidson of the University of Bristol for useful comments on the generated data, and Allison Hicks of Harvard T. H. Chan School of Public Health, for critical review of the manuscript. Lastly, the authors gratefully acknowledge the public health and academic groups contributing to the preparation, release and analysis of SARS-CoV-2 sequence data, including Nextstrain, GISAID and Virological.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">[1].</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference [1] in text" id="ref-1">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.05.976167v2.1"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">World Health Organization</span>. <span class="cit-source">Pneumonia of unknown cause — China. 2020 (</span><a href="https://www.who.int/csr/don/05-january-2020-pneumonia-of-unkown-cause-china/en/">https://www.who.int/csr/don/05-january-2020-pneumonia-of-unkown-cause-china/en/</a>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[2].</span><div class="cit ref-cit ref-web no-rev-xref" id="cit-2020.03.05.976167v2.2"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">United Nations Development Programme – March 2020</span>. <span class="cit-source">UNDP support for coronavirus-affected countries goes beyond health</span>. <a href="https://www.undp.org/content/undp/en/home/blog/2020/undp-support-for-coronavirus-affected-countries-goes-beyond-heal.html">https://www.undp.org/content/undp/en/home/blog/2020/undp-support-for-coronavirus-affected-countries-goes-beyond-heal.html</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[3].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.05.976167v2.3" data-doi="10.1016/S1473-3099(20)30120-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dong</span>,  <span class="cit-name-given-names">Ensheng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Hongru</span> <span class="cit-name-surname">Du</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Lauren</span> <span class="cit-name-surname">Gardner</span></span>. “<span class="cit-article-title">An interactive web-based dashboard to track COVID-19 in real time</span>.” <abbr class="cit-jnl-abbrev">The Lancet Infectious Diseases</abbr> (<span class="cit-pub-date">2020</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/S1473-3099(20)30120-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BLancet%2BInfectious%2BDiseases%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1473-3099%252820%252930120-1%26rft_id%253Dinfo%253Apmid%252F32087114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1473-3099(20)30120-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32087114&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.05.976167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[4].</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference [4] in text" id="ref-4">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.05.976167v2.4"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">World Health Organization. Statement on the second meeting of the International Health Regulations</span> (<span class="cit-pub-date">2005</span>) <span class="cit-source">Emergency Committee regarding the outbreak of novel coronavirus (2019-nCoV). January 30, 2020</span> (https://www.who.int/news-room/detail/30-01-2020-statement-on-the-second-meeting-of-the-international-health-regulations-(2005)-emergency-committee-regarding-the-outbreak-of-novel-coronavirus-(2019-ncov).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[5].</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference [5] in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.5" data-doi="10.1016/S0140-6736(20)30251-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">Roujian</span></span>, <span class="cit-etal">et al.</span> “<span class="cit-article-title">Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</span>.” <abbr class="cit-jnl-abbrev">The Lancet</abbr> (<span class="cit-pub-date">2020</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/S0140-6736(20)30251-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BLancet%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252820%252930251-8%26rft_id%253Dinfo%253Apmid%252F32007145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(20)30251-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32007145&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.05.976167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[6].</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference [6] in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.6" data-doi="10.1056/NEJMra032498"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peiris</span>  <span class="cit-name-given-names">JS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yuen</span>  <span class="cit-name-given-names">KY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Osterhaus</span>  <span class="cit-name-given-names">AD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stöhr</span>  <span class="cit-name-given-names">K.</span></span> <span class="cit-article-title">The severe acute respiratory syndrome</span>. <abbr class="cit-jnl-abbrev">New England Journal of Medicine</abbr>. <span class="cit-pub-date">2003</span> <span class="cit-month">Dec</span> 18;<span class="cit-vol">349</span>(<span class="cit-issue">25</span>):<span class="cit-fpage">2431</span>–<span class="cit-lpage">41</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1056/NEJMra032498</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.stitle%253DNEJM%26rft.aulast%253DPeiris%26rft.auinit1%253DJ.%2BS.M.%26rft.volume%253D349%26rft.issue%253D25%26rft.spage%253D2431%26rft.epage%253D2441%26rft.atitle%253DThe%2BSevere%2BAcute%2BRespiratory%2BSyndrome%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMra032498%26rft_id%253Dinfo%253Apmid%252F14681510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMra032498&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14681510&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.05.976167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187326200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[7].</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference [7] in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.7" data-doi="10.1056/NEJMoa1306742"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Assiri</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">McGeer</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Perl</span>  <span class="cit-name-given-names">TM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">CS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Al Rabeeah</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cummings</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alabdullatif</span>  <span class="cit-name-given-names">ZN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Assad</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Almulhim</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Makhdoom</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Madani</span>  <span class="cit-name-given-names">H.</span></span> <span class="cit-article-title">Hospital outbreak of Middle East respiratory syndrome coronavirus</span>. <abbr class="cit-jnl-abbrev">New England Journal of Medicine</abbr>. <span class="cit-pub-date">2013</span> <span class="cit-month">Aug</span> 1;<span class="cit-vol">369</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">407</span>–<span class="cit-lpage">16</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1056/NEJMoa1306742</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1306742%26rft_id%253Dinfo%253Apmid%252F23782161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa1306742&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23782161&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.05.976167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[8].</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference [8] in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.8" data-doi="10.1056/NEJMe2001126"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Perlman</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Another decade, another coronavirus</span>. <abbr class="cit-jnl-abbrev">New England Journal of Medicine</abbr>. <span class="cit-pub-date">2020</span> <span class="cit-month">February</span> 20; <span class="cit-vol">382</span>:<span class="cit-fpage">760</span>–<span class="cit-lpage">762</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1056/NEJMe2001126</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMe2001126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMe2001126&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[9].</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference [9] in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">McCauley</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">GISAID: Global initiative on sharing all influenza data–from vision to reality</span>. <abbr class="cit-jnl-abbrev">Eurosurveillance</abbr>. <span class="cit-pub-date">2017</span> <span class="cit-month">Mar</span> 30;<span class="cit-vol">22</span>(<span class="cit-issue">13</span>). <a href="https://www.gisaid.org/">https://www.gisaid.org/</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[10].</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference [10] in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hadfield</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Megill</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">SM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huddleston</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Potter</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Callender</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sagulenko</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bedford</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neher</span>  <span class="cit-name-given-names">RA</span></span>. <span class="cit-article-title">Nextstrain: real-time tracking of pathogen evolution</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2018</span> <span class="cit-month">Dec</span> 1;<span class="cit-vol">34</span>(<span class="cit-issue">23</span>):<span class="cit-fpage">4121</span>-<span class="cit-lpage">3</span>. <a href="https://nextstrain.org/ncov">https://nextstrain.org/ncov</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D4121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[11].</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference [11] in text" id="ref-11">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.05.976167v2.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">KG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lipkin</span>  <span class="cit-name-given-names">WI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">EC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garry</span>  <span class="cit-name-given-names">RF</span></span>. <span class="cit-article-title">The Proximal Origin of SARS-CoV-2</span>. <span class="cit-source">Virological, accessed on 27/02/2020</span>. <a href="http://virological.org/t/the-proximal-origin-of-sars-cov-2/398">http://virological.org/t/the-proximal-origin-of-sars-cov-2/398</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[12].</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference [12] in text" id="ref-12">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.05.976167v2.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-source">Phylodynamic Analysis 129 genomes 24 Feb 2020</span>. Virological, accessed 27/02/2020. <a href="http://virological.org/t/phylodynamic-analysis-129-genomes-24-feb-2020/356">http://virological.org/t/phylodynamic-analysis-129-genomes-24-feb-2020/356</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[13].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.05.976167v2.13" data-doi="10.1073/pnas.1735582100"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yount</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Curtis</span>  <span class="cit-name-given-names">KM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fritz</span>  <span class="cit-name-given-names">EA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hensley</span>  <span class="cit-name-given-names">LE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jahrling</span>  <span class="cit-name-given-names">PB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Prentice</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Denison</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Geisbert</span>  <span class="cit-name-given-names">TW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>  <span class="cit-name-given-names">RS</span></span>. <span class="cit-article-title">Reverse genetics with a full-length infectious cDNA of severe acute respiratory syndrome coronavirus</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr>. <span class="cit-pub-date">2003</span> <span class="cit-month">Oct</span> 28;<span class="cit-vol">100</span>(<span class="cit-issue">22</span>):<span class="cit-fpage">12995</span>–<span class="cit-lpage">3000</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1735582100</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1735582100%26rft_id%253Dinfo%253Apmid%252F14569023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAwLzIyLzEyOTk1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDMvMjAyMC4wMy4wNS45NzYxNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[14].</span><div class="cit ref-cit ref-book no-rev-xref" id="cit-2020.03.05.976167v2.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brian</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>  <span class="cit-name-given-names">RS</span></span>. <span class="cit-chapter-title">Coronavirus genome structure and replication</span>. <span class="cit-source">InCoronavirus replication and reverse genetics</span> <span class="cit-pub-date">2005</span> (pp. <span class="cit-fpage">1</span>-<span class="cit-lpage">30</span>). <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin, Heidelberg</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[15].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.05.976167v2.15" data-doi="10.1073/pnas.0808790106"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiang</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tien</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ahola</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">Functional screen reveals SARS coronavirus nonstructural protein nsp14 as a novel cap N7 methyltransferase</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr>. <span class="cit-pub-date">2009</span> <span class="cit-month">Mar</span> 3;<span class="cit-vol">106</span>(<span class="cit-issue">9</span>):<span class="cit-fpage">3484</span>–<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.0808790106</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0808790106%26rft_id%253Dinfo%253Apmid%252F19208801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA2LzkvMzQ4NCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAzLzIwMjAuMDMuMDUuOTc2MTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[16].</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference [16] in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.16" data-doi="10.1038/nmeth.4577"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Garalde</span>  <span class="cit-name-given-names">DR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snell</span>  <span class="cit-name-given-names">EA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jachimowicz</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sipos</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lloyd</span>  <span class="cit-name-given-names">JH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bruce</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pantic</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Admassu</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Warland</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jordan</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Highly parallel direct RNA sequencing on an array of nanopores</span>. <abbr class="cit-jnl-abbrev">Nature methods</abbr>. <span class="cit-pub-date">2018</span> <span class="cit-month">Mar</span>;<span class="cit-vol">15</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">201</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/nmeth.4577</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bmethods%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.4577%26rft_id%253Dinfo%253Apmid%252F29334379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.4577&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29334379&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.05.976167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[17].</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference [17] in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.17" data-doi="10.1128/JVI.01691-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schaecher</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mackenzie</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pekosz</span>  <span class="cit-name-given-names">A</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">The ORF7b Protein of Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) Is Expressed in Virus-Infected Cells and Incorporated into SARS-CoV Particles</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>, <span class="cit-vol">81</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">718</span>–<span class="cit-lpage">731</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.01691-06</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01691-06%26rft_id%253Dinfo%253Apmid%252F17079322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjgxLzIvNzE4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDMvMjAyMC4wMy4wNS45NzYxNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[18].</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference [18] in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.18" data-doi="10.21203/rs.3.rs-17218/v1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bojkova</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klann</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koch</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Widera</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krause</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ciesek</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cinatl</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Münch</span>  <span class="cit-name-given-names">C</span></span> (<span class="cit-pub-date">2020</span>) <abbr class="cit-jnl-abbrev">SARS-CoV-2 infected host cell proteomics reveal potential therapy targets</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.21203/rs.3.rs-17218/v1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSARS-CoV-2%2Binfected%2Bhost%2Bcell%2Bproteomics%2Breveal%2Bpotential%2Btherapy%2Btargets%26rft_id%253Dinfo%253Adoi%252F10.21203%252Frs.3.rs-17218%252Fv1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.21203/rs.3.rs-17218/v1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[19].</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference [19] in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.19" data-doi="10.1101/2020.03.22.002204"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">AD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">MK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shoemark</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carroll</span>  <span class="cit-name-given-names">MW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heesom</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zambon</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ellis</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">PA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hiscox</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matthews</span>  <span class="cit-name-given-names">DA</span></span> (<span class="cit-pub-date">2020</span>) <abbr class="cit-jnl-abbrev">Characterisation of the transcriptome and proteome of SARS-CoV-2 using direct RNA sequencing and tandem mass spectrometry reveals evidence for a cell passage induced in-frame deletion in the spike glycoprotein that removes the furin-like cleavage site</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1101/2020.03.22.002204</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCharacterisation%2Bof%2Bthe%2Btranscriptome%2Band%2Bproteome%2Bof%2BSARS-CoV-2%2Busing%2Bdirect%2BRNA%2Bsequencing%2Band%2Btandem%2Bmass%2Bspectrometry%2Breveals%2Bevidence%2Bfor%2Ba%2Bcell%2Bpassage%2Binduced%2Bin-frame%2Bdeletion%2Bin%2Bthe%2Bspike%2Bglycoprotein%2Bthat%2Bremoves%2Bthe%2Bfurin-like%2Bcleavage%2Bsite%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.03.22.002204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wMy4yMi4wMDIyMDR2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAzLzIwMjAuMDMuMDUuOTc2MTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[20].</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference [20] in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.20" data-doi="10.1016/j.bbrc.2004.02.074"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dobie</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ballantine</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leeson</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bastien</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cutts</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andonov</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Booth</span>  <span class="cit-name-given-names">TF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Plummer</span>  <span class="cit-name-given-names">FA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tyler</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X</span></span> (<span class="cit-pub-date">2004</span>). <span class="cit-article-title">Analysis of multimerization of the SARS coronavirus nucleocapsid protein</span>. <abbr class="cit-jnl-abbrev">Biochemical and Biophysical Research Communications</abbr>, <span class="cit-vol">316</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">476</span>–<span class="cit-lpage">483</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.bbrc.2004.02.074</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2Band%2Bbiophysical%2Bresearch%2Bcommunications%26rft.stitle%253DBiochem%2BBiophys%2BRes%2BCommun%26rft.aulast%253DHe%26rft.auinit1%253DR.%26rft.volume%253D316%26rft.issue%253D2%26rft.spage%253D476%26rft.epage%253D483%26rft.atitle%253DAnalysis%2Bof%2Bmultimerization%2Bof%2Bthe%2BSARS%2Bcoronavirus%2Bnucleocapsid%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbrc.2004.02.074%26rft_id%253Dinfo%253Apmid%252F15020242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbrc.2004.02.074&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15020242&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.05.976167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220333200027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[21].</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference [21] in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.21" data-doi="10.1128/JVI.78.2.669-682.2004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goebel</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hsue</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dombrowski</span>  <span class="cit-name-given-names">TF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Masters</span>  <span class="cit-name-given-names">PS</span></span>. <span class="cit-article-title">Characterization of the RNA components of a putative molecular switch in the 3’ untranslated region of the murine coronavirus genome</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr>. <span class="cit-pub-date">2004</span> <span class="cit-month">Jan</span>;<span class="cit-vol">78</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">669</span>–<span class="cit-lpage">82</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1128/jvi.78.2.669-682.2004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DGoebel%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D78%26rft.issue%253D2%26rft.spage%253D669%26rft.epage%253D682%26rft.atitle%253DCharacterization%2Bof%2Bthe%2BRNA%2BComponents%2Bof%2Ba%2BPutative%2BMolecular%2BSwitch%2Bin%2Bthe%2B3%2527%2BUntranslated%2BRegion%2Bof%2Bthe%2BMurine%2BCoronavirus%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.78.2.669-682.2004%26rft_id%253Dinfo%253Apmid%252F14694098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6Ijc4LzIvNjY5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDMvMjAyMC4wMy4wNS45NzYxNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[22].</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference [22] in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.22" data-doi="10.1186/1743-422X-10-132"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tengs</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kristoffersen</span>  <span class="cit-name-given-names">AB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bachvaroff</span>  <span class="cit-name-given-names">TR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jonassen</span>  <span class="cit-name-given-names">CM</span></span>. <span class="cit-article-title">A mobile genetic element with unknown function found in distantly related viruses</span>. <abbr class="cit-jnl-abbrev">Virol J</abbr>. <span class="cit-pub-date">2013</span> <span class="cit-month">Apr</span> 25;<span class="cit-vol">10</span>:<span class="cit-fpage">132</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1186/1743-422X-10-132</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirol%2BJ%26rft.volume%253D10%26rft.spage%253D132%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1743-422X-10-132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1743-422X-10-132&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[23].</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference [23] in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.23" data-doi="10.1101/2020.03.27.012906"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rangan</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zheludev</span>  <span class="cit-name-given-names">IN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Das</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">RNA genome conservation and secondary structure in SARS-CoV-2 and SARS-related viruses</span>. <abbr class="cit-jnl-abbrev">bioRxiv preprint 2020</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1101/2020.03.27.012906</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%2Bpreprint%2B2020%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.03.27.012906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wMy4yNy4wMTI5MDZ2MiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAzLzIwMjAuMDMuMDUuOTc2MTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[24].</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference [24] in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.24" data-doi="10.1371/journal.pbio.0030005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">MP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Igel</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baertsch</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haussler</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ares</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">WG</span></span>. <span class="cit-article-title">The Structure of a Rigorously Conserved RNA Element within the SARS Virus Genome</span>. <abbr class="cit-jnl-abbrev">PLoS Biol</abbr>. <span class="cit-pub-date">2005</span> <span class="cit-month">Jan</span>; <span class="cit-vol">3</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">e5</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pbio.0030005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DRobertson%26rft.auinit1%253DM.%2BP.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253De5%26rft.epage%253De5%26rft.atitle%253DThe%2Bstructure%2Bof%2Ba%2Brigorously%2Bconserved%2BRNA%2Belement%2Bwithin%2Bthe%2BSARS%2Bvirus%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0030005%26rft_id%253Dinfo%253Apmid%252F15630477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0030005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15630477&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.05.976167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[25].</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference [25] in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.25" data-doi="10.1101/gr.247064.118"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Viehweger</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krautwurst</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lamkiewicz</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Madhugiri</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziebuhr</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hölzer</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marz</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Direct RNA nanopore sequencing of full-length coronavirus genomes provides novel insights into structural variants and enables modification analysis</span>. <abbr class="cit-jnl-abbrev">Genome research</abbr>. <span class="cit-pub-date">2019</span> <span class="cit-month">Sep</span> 1;<span class="cit-vol">29</span>(<span class="cit-issue">9</span>):<span class="cit-fpage">1545</span>–<span class="cit-lpage">54</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1101/gr.247064.118</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.247064.118%26rft_id%253Dinfo%253Apmid%252F31439691%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI5LzkvMTU0NSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAzLzIwMjAuMDMuMDUuOTc2MTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[26].</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference [26] in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.26" data-doi="10.1016/j.chom.2016.10.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lichinchi</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">BS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qin</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rana</span>  <span class="cit-name-given-names">TM</span></span>. <span class="cit-article-title">Dynamics of human and viral RNA methylation during Zika virus infection</span>. <abbr class="cit-jnl-abbrev">Cell host &amp; microbe</abbr>. <span class="cit-pub-date">2016</span> <span class="cit-month">Nov</span> 9;<span class="cit-vol">20</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">666</span>–<span class="cit-lpage">73</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.chom.2016.10.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Bhost%2B%2526%2Bmicrobe%26rft.volume%253D20%26rft.spage%253D666%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2016.10.002%26rft_id%253Dinfo%253Apmid%252F27773536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2016.10.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27773536&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.05.976167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[27].</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference [27] in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.27" data-doi="10.4161/rna.8.2.15013"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Denison</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">RL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Donaldson</span>  <span class="cit-name-given-names">EF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eckerle</span>  <span class="cit-name-given-names">LD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>  <span class="cit-name-given-names">RS</span></span>. <span class="cit-article-title">Coronaviruses: an RNA proofreading machine regulates replication fidelity and diversity</span>. <abbr class="cit-jnl-abbrev">RNA biology</abbr>. <span class="cit-pub-date">2011</span> <span class="cit-month">Mar</span> 1;<span class="cit-vol">8</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">270</span>–<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.4161/rna.8.2.15013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DRNA%2BBiol%26rft.aulast%253DDenison%26rft.auinit1%253DM.%2BR.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253D270%26rft.epage%253D279%26rft.atitle%253DCoronaviruses%253A%2Ban%2BRNA%2Bproofreading%2Bmachine%2Bregulates%2Breplication%2Bfidelity%2Band%2Bdiversity.%26rft_id%253Dinfo%253Adoi%252F10.4161%252Frna.8.2.15013%26rft_id%253Dinfo%253Apmid%252F21593585%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/rna.8.2.15013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21593585&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.05.976167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289934300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[28].</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference [28] in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.28" data-doi="10.1126/science.1092002"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Chinese SARS Molecular Epidemiology Consortium</span>. <span class="cit-article-title">Molecular evolution of the SARS coronavirus during the course of the SARS epidemic in China</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-pub-date">2004</span> <span class="cit-month">Mar</span> 12;<span class="cit-vol">303</span>(<span class="cit-issue">5664</span>):<span class="cit-fpage">1666</span>–<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.1092002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1092002%26rft_id%253Dinfo%253Apmid%252F14752165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDMvNTY2NC8xNjY2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDMvMjAyMC4wMy4wNS45NzYxNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[29].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.05.976167v2.29" data-doi="10.7554/eLife.31257"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dudas</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carvalho</span>  <span class="cit-name-given-names">LM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bedford</span>  <span class="cit-name-given-names">T.</span></span> <span class="cit-article-title">MERS-CoV spillover at the camel-human interface</span>. <abbr class="cit-jnl-abbrev">Elife</abbr>. <span class="cit-pub-date">2018</span> <span class="cit-month">Jan</span> 16;<span class="cit-vol">7</span>:<span class="cit-fpage">e31257</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.7554/eLife.31257</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.31257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.31257&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[30].</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference [30] in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.30" data-doi="10.1016/S0140-6736(13)61887-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cotten</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Watson</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kellam</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Al-Rabeeah</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Makhdoom</span>  <span class="cit-name-given-names">HQ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Assiri</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Al-Tawfiq</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alhakeem</span>  <span class="cit-name-given-names">RF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Madani</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">AlRabiah</span>  <span class="cit-name-given-names">FA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Al Hajjar</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Transmission and evolution of the Middle East respiratory syndrome coronavirus in Saudi Arabia: a descriptive genomic study</span>. <abbr class="cit-jnl-abbrev">The Lancet</abbr>. <span class="cit-pub-date">2013</span> <span class="cit-month">Dec</span> 14;<span class="cit-vol">382</span>(<span class="cit-issue">9909</span>):<span class="cit-fpage">1993</span>–<span class="cit-lpage">2002</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/S0140-6736(13)61887-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BLancet%26rft.volume%253D382%26rft.spage%253D1993%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252813%252961887-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(13)61887-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[31].</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference [31] in text" id="ref-31">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.05.976167v2.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">K.</span></span> <span class="cit-source">Clock and TMRCA based on 27 genomes. Virological, accessed on 27/02/2020</span>. <a href="http://virological.org/t/clock-and-tmrca-based-on-27-genomes/347">http://virological.org/t/clock-and-tmrca-based-on-27-genomes/347</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[32].</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference [32] in text" id="ref-32">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.05.976167v2.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bedford</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-source">Phylodynamic estimation of incidence and prevalence of novel coronavirus (nCoV) infections through time. Virological, accessed on 27/02/2020</span>. <a href="http://virological.org/t/phylodynamic-estimation-of-incidence-and-prevalence-of-novel-coronavirus-ncov-infections-through-time/391">http://virological.org/t/phylodynamic-estimation-of-incidence-and-prevalence-of-novel-coronavirus-ncov-infections-through-time/391</a></cite></div><div class="cit-extra"></div></div></li></ol></div><div class="section ref-list" id="ref-list-2"><ol class="cit-list ref-use-labels"><li><span class="ref-label">[M1].</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference [M1] in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.33" data-doi="10.1093/bioinformatics/bts199"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kearse</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moir</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stones-Havas</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sturrock</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buxton</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Markowitz</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duran</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thierer</span>  <span class="cit-name-given-names">T.</span></span> <span class="cit-article-title">Geneious Basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2012</span> <span class="cit-month">Jun</span> 15;<span class="cit-vol">28</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">1647</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts199%26rft_id%253Dinfo%253Apmid%252F22543367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts199&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22543367&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.05.976167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305419800052&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[M2].</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference [M2] in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.34" data-doi="10.1093/bioinformatics/bty191"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span> <span class="cit-article-title">Minimap2: pairwise alignment for nucleotide sequences</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2018</span> <span class="cit-month">Sep</span> 15;<span class="cit-vol">34</span>(<span class="cit-issue">18</span>):<span class="cit-fpage">3094</span>–<span class="cit-lpage">100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D3094%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty191%26rft_id%253Dinfo%253Apmid%252F29750242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty191&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29750242&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.05.976167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[M3].</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference [M3] in text" id="ref-35">↵</a><div class="cit ref-cit ref-book" id="cit-2020.03.05.976167v2.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Milne</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bayer</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stephen</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cardle</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-chapter-title">Tablet: visualizing next-generation sequence assemblies and mappings</span>. <span class="cit-source">Bioinformatics</span>. <span class="cit-pub-date">2016</span> (pp. <span class="cit-fpage">253</span>-<span class="cit-lpage">268</span>). <span class="cit-publ-name">Humana Press</span>, <span class="cit-publ-loc">New York, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[M4].</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference [M4] in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.36" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Homer</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marth</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">The sequence alignment/map format and SAMtools</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2009</span> <span class="cit-month">Aug</span> 15;<span class="cit-vol">25</span>(<span class="cit-issue">16</span>):<span class="cit-fpage">2078</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.05.976167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[M5].</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference [M5] in text" id="ref-37">↵</a><div class="cit ref-cit ref-book" id="cit-2020.03.05.976167v2.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wickham</span>  <span class="cit-name-given-names">H.</span></span> <span class="cit-source">ggplot2: elegant graphics for data analysis</span>. <span class="cit-publ-name">Springer</span>; <span class="cit-pub-date">2016</span> Jun 8</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[M6].</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference [M6] in text" id="ref-38">↵</a><div class="cit ref-cit ref-book" id="cit-2020.03.05.976167v2.38"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">R Core Team</span> (<span class="cit-pub-date">2018</span>). <span class="cit-source">R: A language and environment for statistical computing. R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[M7].</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference [M7] in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sato</span>,  <span class="cit-name-given-names">Kengo</span></span>, <span class="cit-etal">et al.</span> “<span class="cit-article-title">IPknot: fast and accurate prediction of RNA secondary structures with pseudoknots using integer programming</span>.” <abbr class="cit-jnl-abbrev">Bioinformatics. 2011 27.13: i85-i93</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[M8].</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference [M8] in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Darty</span>,  <span class="cit-name-given-names">Kévin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Alain</span> <span class="cit-name-surname">Denise</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Yann</span> <span class="cit-name-surname">Ponty</span></span>. “<span class="cit-article-title">VARNA: Interactive drawing and editing of the RNA secondary structure</span>.” <abbr class="cit-jnl-abbrev">Bioinformatics 2009 25.15</abbr>: <span class="cit-pub-date">1974</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[M9].</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference [M9] in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.41" data-doi="10.1093/bioinformatics/btr064"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grant</span>  <span class="cit-name-given-names">CE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">TL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Noble</span>  <span class="cit-name-given-names">WS</span></span>. <span class="cit-article-title">FIMO: scanning for occurrences of a given motif</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-pub-date">2011</span> <span class="cit-vol">27</span>: <span class="cit-fpage">1017</span>–<span class="cit-lpage">1018</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr064%26rft_id%253Dinfo%253Apmid%252F21330290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21330290&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.05.976167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289162000022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[M10].</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference [M10] in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.42" data-doi="10.1093/nar/gkv1145"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vizcaino</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Csordas</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">del-Toro</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dianes</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Griss</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lavidas</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Update of the PRIDE database and its related tools</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">44</span>(<span class="cit-issue">D1</span>):<span class="cit-fpage">D447</span>–<span class="cit-lpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1145%26rft_id%253Dinfo%253Apmid%252F26527722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1145&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26527722&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.05.976167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[M11].</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference [M11] in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.43" data-doi="10.1038/nbt.1511"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mann</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification</span>. <abbr class="cit-jnl-abbrev">Nat Biotechnol</abbr>. <span class="cit-pub-date">2008</span>;<span class="cit-vol">26</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">1367</span>–<span class="cit-lpage">72</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1038/nbt.1511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DCox%26rft.auinit1%253DJ.%26rft.volume%253D26%26rft.issue%253D12%26rft.spage%253D1367%26rft.epage%253D1372%26rft.atitle%253DMaxQuant%2Benables%2Bhigh%2Bpeptide%2Bidentification%2Brates%252C%2Bindividualized%2Bp.p.b.-range%2Bmass%2Baccuracies%2Band%2Bproteome-wide%2Bprotein%2Bquantification.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1511%26rft_id%253Dinfo%253Apmid%252F19029910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1511&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19029910&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.05.976167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261591300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[M12].</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference [M12] in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">MA</span></span>. <span class="cit-article-title">SquiggleKit: A toolkit for manipulating nanopore signal data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2019</span> <span class="cit-month">Dec</span> 15;<span class="cit-vol">35</span>(<span class="cit-issue">24</span>):<span class="cit-fpage">5372</span>–<span class="cit-lpage">3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D35%26rft.spage%253D5372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[M13].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.05.976167v2.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">BEAST: Bayesian evolutionary analysis by sampling trees</span>. <abbr class="cit-jnl-abbrev">BMC evolutionary biology</abbr>. <span class="cit-pub-date">2007</span> <span class="cit-month">Dec</span>;<span class="cit-vol">7</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%26rft.volume%253D7%26rft.spage%253D214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[M14].</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference [M14] in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.46" data-doi="10.1186/s12859-018-2164-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Menardo</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Loiseau</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brites</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coscolla</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gygli</span>  <span class="cit-name-given-names">SM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rutaihwa</span>  <span class="cit-name-given-names">LK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Trauner</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beisel</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Borrell</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gagneux</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Treemmer: a tool to reduce large phylogenetic datasets with minimal loss of diversity</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-pub-date">2018</span> <span class="cit-vol">19</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">164</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1186/s12859-018-2164-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D19%26rft.spage%253D164%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-018-2164-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-018-2164-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[M15].</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference [M15] in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Duchene</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stadler</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">SY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duchene</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dhanasekaran</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baele</span>  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">Bayesian Evaluation of Temporal Signal in Measurably Evolving Populations</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>. <span class="cit-pub-date">2019</span> <span class="cit-month">Jan</span> 1:<span class="cit-fpage">810697</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[M16].</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference [M16] in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.48" data-doi="10.1093/sysbio/syy032"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baele</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suchard</span>  <span class="cit-name-given-names">MA</span></span>. <span class="cit-article-title">Posterior summarization in Bayesian phylogenetics using Tracer 1.7</span>. <abbr class="cit-jnl-abbrev">Systematic biology</abbr>. <span class="cit-pub-date">2018</span> <span class="cit-month">Sep</span>;<span class="cit-vol">67</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">901</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2Bbiology%26rft.volume%253D67%26rft.spage%253D901%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyy032%26rft_id%253Dinfo%253Apmid%252F29718447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syy032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29718447&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.05.976167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[M17].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.05.976167v2.49" data-doi="10.1016/S0140-6736(13)61887-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cotten</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Watson</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kellam</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Al-Rabeeah</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Makhdoom</span>  <span class="cit-name-given-names">HQ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Assiri</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Al-Tawfiq</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alhakeem</span>  <span class="cit-name-given-names">RF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Madani</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">AlRabiah</span>  <span class="cit-name-given-names">FA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Al Hajjar</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Transmission and evolution of the Middle East respiratory syndrome coronavirus in Saudi Arabia: a descriptive genomic study</span>. <abbr class="cit-jnl-abbrev">The Lancet</abbr>. <span class="cit-pub-date">2013</span> <span class="cit-month">Dec</span> 14;<span class="cit-vol">382</span>(<span class="cit-issue">9909</span>):<span class="cit-fpage">1993</span>–<span class="cit-lpage">2002</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BLancet%26rft.volume%253D382%26rft.spage%253D1993%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252813%252961887-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(13)61887-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[M18].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.05.976167v2.50" data-doi="10.1128/mBio.01062-13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cotten</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Watson</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zumla</span>  <span class="cit-name-given-names">AI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Makhdoom</span>  <span class="cit-name-given-names">HQ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Palser</span>  <span class="cit-name-given-names">AL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ong</span>  <span class="cit-name-given-names">SH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Al Rabeeah</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alhakeem</span>  <span class="cit-name-given-names">RF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Assiri</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Al-Tawfiq</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Albarrak</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Spread, circulation, and evolution of the Middle East respiratory syndrome coronavirus</span>. <abbr class="cit-jnl-abbrev">MBio</abbr>. <span class="cit-pub-date">2014</span> <span class="cit-month">Feb</span> 28;<span class="cit-vol">5</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">e01062</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMBio%26rft.volume%253D5%26rft.spage%253De01062%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.01062-13%26rft_id%253Dinfo%253Apmid%252F24549846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/mBio.01062-13&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24549846&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.05.976167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[M19].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.05.976167v2.51"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dudas</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carvalho</span>  <span class="cit-name-given-names">LM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bedford</span>  <span class="cit-name-given-names">T.</span></span> <span class="cit-article-title">MERS-CoV spillover at the camel-human interface</span>. <abbr class="cit-jnl-abbrev">Elife</abbr>. <span class="cit-pub-date">2018</span> <span class="cit-month">Jan</span> 16;<span class="cit-vol">7</span>:<span class="cit-fpage">e31257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElife%26rft.volume%253D7%26rft.spage%253De31257%26rft_id%253Dinfo%253Apmid%252F29336306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=29336306&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.05.976167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[M20].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.05.976167v2.52"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">YP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boerwinkle</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">YX</span></span>. <span class="cit-article-title">Moderate mutation rate in the SARS coronavirus genome and its implications</span>. <abbr class="cit-jnl-abbrev">BMC evolutionary biology</abbr>. <span class="cit-pub-date">2004</span> <span class="cit-month">Dec</span> 1;<span class="cit-vol">4</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%26rft.volume%253D4%26rft.spage%253D21%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[M21].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.05.976167v2.53" data-doi="10.1128/JVI.78.3.1602-1603.2004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Salemi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fitch</span>  <span class="cit-name-given-names">WM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ciccozzi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ruiz-Alvarez</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rezza</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">MJ</span></span>. <span class="cit-article-title">Severe acute respiratory syndrome coronavirus sequence characteristics and evolutionary rate estimate from maximum likelihood analysis</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr>. <span class="cit-pub-date">2004</span> <span class="cit-month">Feb</span> 1;<span class="cit-vol">78</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">1602</span>–<span class="cit-lpage">3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DSalemi%26rft.auinit1%253DM.%26rft.volume%253D78%26rft.issue%253D3%26rft.spage%253D1602%26rft.epage%253D1603%26rft.atitle%253DSevere%2BAcute%2BRespiratory%2BSyndrome%2BCoronavirus%2BSequence%2BCharacteristics%2Band%2BEvolutionary%2BRate%2BEstimate%2Bfrom%2BMaximum%2BLikelihood%2BAnalysis%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.78.3.1602-1603.2004%26rft_id%253Dinfo%253Apmid%252F14722315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijc4LzMvMTYwMiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzAzLzIwMjAuMDMuMDUuOTc2MTY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[M22].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.05.976167v2.54"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">SF D. CJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">TG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">JQ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">YJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">YP</span></span>, <span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">FC</span></span>. <span class="cit-article-title">The genome comparison of SARS-CoV and other coronaviruses</span>. <abbr class="cit-jnl-abbrev">Yi chuan= Hereditas</abbr>. <span class="cit-pub-date">2003</span> <span class="cit-month">Jul</span>;<span class="cit-vol">25</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">373</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYi%2Bchuan%253D%2BHereditas%26rft.volume%253D25%26rft.spage%253D373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[M23].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.05.976167v2.55" data-doi="10.1126/science.1092002"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Chinese SARS Molecular Epidemiology Consortium. Molecular evolution of the SARS coronavirus during the course of the SARS epidemic in China</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-pub-date">2004</span> <span class="cit-month">Mar</span> 12;<span class="cit-vol">303</span>(<span class="cit-issue">5664</span>):<span class="cit-fpage">1666</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1092002%26rft_id%253Dinfo%253Apmid%252F14752165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDMvNTY2NC8xNjY2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDMvMjAyMC4wMy4wNS45NzYxNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[M24].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.05.976167v2.56"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sanjuán</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">From molecular genetics to phylodynamics: evolutionary relevance of mutation rates across viruses</span>. <abbr class="cit-jnl-abbrev">PLoS pathogens</abbr>. <span class="cit-pub-date">2012</span> <span class="cit-month">May</span>;<span class="cit-vol">8</span>(<span class="cit-issue">5</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[M25].</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference [M25] in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.05.976167v2.57" data-doi="10.1093/bioinformatics/btw421"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Duchêne</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Geoghegan</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">EC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">SY</span></span>. <span class="cit-article-title">Estimating evolutionary rates using time-structured data: a general comparison of phylogenetic methods</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2016</span> <span class="cit-vol">32</span>(<span class="cit-issue">22</span>), <span class="cit-fpage">3375</span>–<span class="cit-lpage">3379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw421%26rft_id%253Dinfo%253Apmid%252F27412094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw421&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27412094&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F03%2F2020.03.05.976167.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.05.976167v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.05.976167v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/869263v9" title="Transcriptome analysis reveals salinity responses in four Tartary buckwheat cultivars" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.13.990598v2" title="When Darkness Becomes a Ray of Light in the Dark Times: Understanding the COVID-19 via the Comparative Analysis of the Dark Proteomes of SARS-CoV-2, Human SARS and Bat SARS-Like Coronaviruses" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;April 03, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.05.976167v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.03.05.976167v2.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.05.976167v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.05.976167v2" class="active" data-icon-position="" data-hide-link-title="0">Direct RNA sequencing and early evolution of SARS-CoV-2</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1225704" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-dMwcLS3kO6XO0qX47XLg0Q_Ubqu5fnQ2jD2Z-b-jfyg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1225704" id="node1225704" data-pisa="biorxiv;2020.03.05.976167v2" data-pisa-master="biorxiv;2020.03.05.976167" data-seqnum="1225704" data-apath="/biorxiv/early/2020/04/03/2020.03.05.976167.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Direct RNA sequencing and early evolution of SARS-CoV-2</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">George</span> <span class="nlm-surname">Taiaroa</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Daniel</span> <span class="nlm-surname">Rawlinson</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Leo</span> <span class="nlm-surname">Featherstone</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Miranda</span> <span class="nlm-surname">Pitt</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Leon</span> <span class="nlm-surname">Caly</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Julian</span> <span class="nlm-surname">Druce</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Damian</span> <span class="nlm-surname">Purcell</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Leigh</span> <span class="nlm-surname">Harty</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Thomas</span> <span class="nlm-surname">Tran</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Jason</span> <span class="nlm-surname">Roberts</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Nichollas</span> <span class="nlm-surname">Scott</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Mike</span> <span class="nlm-surname">Catton</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Deborah</span> <span class="nlm-surname">Williamson</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Lachlan</span> <span class="nlm-surname">Coin</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Sebastian</span> <span class="nlm-surname">Duchene</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.05.976167; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.05.976167 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.05.976167v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.05.976167v2&amp;title=Direct%20RNA%20sequencing%20and%20early%20evolution%20of%20SARS-CoV-2" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.05.976167v2&amp;title=Direct%20RNA%20sequencing%20and%20early%20evolution%20of%20SARS-CoV-2" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.05.976167v2&amp;text=Direct%20RNA%20sequencing%20and%20early%20evolution%20of%20SARS-CoV-2" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.05.976167v2&amp;title=Direct%20RNA%20sequencing%20and%20early%20evolution%20of%20SARS-CoV-2" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.05.976167v2&amp;t=Direct%20RNA%20sequencing%20and%20early%20evolution%20of%20SARS-CoV-2" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.05.976167v2&amp;title=Direct%20RNA%20sequencing%20and%20early%20evolution%20of%20SARS-CoV-2" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.05.976167v2&amp;title=Direct%20RNA%20sequencing%20and%20early%20evolution%20of%20SARS-CoV-2" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1225704" id="node1225704--2" data-pisa="biorxiv;2020.03.05.976167v2" data-pisa-master="biorxiv;2020.03.05.976167" data-seqnum="1225704" data-apath="/biorxiv/early/2020/04/03/2020.03.05.976167.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Direct RNA sequencing and early evolution of SARS-CoV-2</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">George</span> <span class="nlm-surname">Taiaroa</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Daniel</span> <span class="nlm-surname">Rawlinson</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Leo</span> <span class="nlm-surname">Featherstone</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Miranda</span> <span class="nlm-surname">Pitt</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Leon</span> <span class="nlm-surname">Caly</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Julian</span> <span class="nlm-surname">Druce</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Damian</span> <span class="nlm-surname">Purcell</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Leigh</span> <span class="nlm-surname">Harty</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Thomas</span> <span class="nlm-surname">Tran</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Jason</span> <span class="nlm-surname">Roberts</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Nichollas</span> <span class="nlm-surname">Scott</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Mike</span> <span class="nlm-surname">Catton</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Deborah</span> <span class="nlm-surname">Williamson</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Lachlan</span> <span class="nlm-surname">Coin</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Sebastian</span> <span class="nlm-surname">Duchene</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.05.976167; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.05.976167 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Direct%20RNA%20sequencing%20and%20early%20evolution%20of%20SARS-CoV-2&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Taiaroa%2C+George&amp;rft.au=Rawlinson%2C+Daniel&amp;rft.au=Featherstone%2C+Leo&amp;rft.au=Pitt%2C+Miranda&amp;rft.au=Caly%2C+Leon&amp;rft.au=Druce%2C+Julian&amp;rft.au=Purcell%2C+Damian&amp;rft.au=Harty%2C+Leigh&amp;rft.au=Tran%2C+Thomas&amp;rft.au=Roberts%2C+Jason&amp;rft.au=Scott%2C+Nichollas&amp;rft.au=Catton%2C+Mike&amp;rft.au=Williamson%2C+Deborah&amp;rft.au=Coin%2C+Lachlan&amp;rft.au=Duchene%2C+Sebastian"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1225704/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1225704/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1225704/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1225704/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1225704/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1225704/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1225704/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1225704/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1225704/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1225704/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1225704/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1225704/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.05.976167v2&amp;count=horizontal&amp;via=&amp;text=Direct%20RNA%20sequencing%20and%20early%20evolution%20of%20SARS-CoV-2&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.05.976167v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.05.976167v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.05.976167v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">Microbiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
