<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/04/06/2020.03.20.000794.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.20.000794v4" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="COVID-19 Vaccine Candidates: Prediction and Validation of 174 SARS-CoV-2 Epitopes" />
<meta name="DC.Identifier" content="10.1101/2020.03.20.000794" />
<meta name="DC.Date" content="2020-04-06" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="The recent outbreak of SARS-CoV-2 (2019-nCoV) virus has highlighted the need for fast and efficacious vaccine development. Stimulation of a proper immune response that leads to protection is highly dependent on presentation of epitopes to circulating T-cells via the HLA complex. SARS-CoV-2 is a large RNA virus and testing of all overlapping peptides in vitro to deconvolute an immune response is not feasible. Therefore HLA-binding prediction tools are often used to narrow down the number of peptides to test. We tested 19 epitope-HLA-binding prediction tools, and using an in vitro peptide MHC stability assay, we assessed 777 peptides that were predicted to be good binders across 11 MHC allotypes. In this investigation of potential SARS-CoV-2 epitopes we found that current prediction tools vary in performance when assessing binding stability, and they are highly dependent on the MHC allotype in question. Designing a COVID-19 vaccine where only a few epitope targets are included is therefore a very challenging task. Here, we present 174 SARS-CoV-2 epitopes with high prediction binding scores, validated to bind stably to 11 HLA allotypes. Our findings may contribute to the design of an efficacious vaccine against COVID-19." />
<meta name="DC.Contributor" content="Marek Prachar" />
<meta name="DC.Contributor" content="Sune Justesen" />
<meta name="DC.Contributor" content="Daniel Bisgaard Steen-Jensen" />
<meta name="DC.Contributor" content="Stephan Thorgrimsen" />
<meta name="DC.Contributor" content="Erik Jurgons" />
<meta name="DC.Contributor" content="Ole Winther" />
<meta name="DC.Contributor" content="Frederik Otzen Bagger" />
<meta name="article:published_time" content="2020-04-06" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="COVID-19 Vaccine Candidates: Prediction and Validation of 174 SARS-CoV-2 Epitopes" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The recent outbreak of SARS-CoV-2 (2019-nCoV) virus has highlighted the need for fast and efficacious vaccine development. Stimulation of a proper immune response that leads to protection is highly dependent on presentation of epitopes to circulating T-cells via the HLA complex. SARS-CoV-2 is a large RNA virus and testing of all overlapping peptides in vitro to deconvolute an immune response is not feasible. Therefore HLA-binding prediction tools are often used to narrow down the number of peptides to test. We tested 15 epitope-HLA-binding prediction tools, and using an in vitro peptide MHC stability assay, we assessed 777 peptides that were predicted to be good binders across 11 MHC allotypes. In this investigation of potential SARS-CoV-2 epitopes we found that current prediction tools vary in performance when assessing binding stability, and they are highly dependent on the MHC allotype in question. Designing a COVID-19 vaccine where only a few epitope targets are included is therefore a very challenging task. Here, we present 174 SARS-CoV-2 epitopes with high prediction binding scores, validated to bind stably to 11 HLA allotypes. Our findings may contribute to the design of an efficacious vaccine against COVID-19.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.20.000794v4" />
<meta name="citation_id" content="2020.03.20.000794v4" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.20.000794v4" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.20.000794v4.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.20.000794v4.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.20.000794.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.20.000794" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.20.000794" />
<meta name="citation_author" content="Marek Prachar" />
<meta name="citation_author_institution" content="Center for Genomic Medicine, Copenhagen University Hospital," />
<meta name="citation_author_institution" content="Bioinformatics Centre, Department of Biology, University of Copenhagen" />
<meta name="citation_author_institution" content="Immunitrack ApS" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1797-7147" />
<meta name="citation_author" content="Sune Justesen" />
<meta name="citation_author_institution" content="Immunitrack ApS" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3039-2178" />
<meta name="citation_author" content="Daniel Bisgaard Steen-Jensen" />
<meta name="citation_author_institution" content="Immunitrack ApS" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7891-4502" />
<meta name="citation_author" content="Stephan Thorgrimsen" />
<meta name="citation_author_institution" content="Immunitrack ApS" />
<meta name="citation_author" content="Erik Jurgons" />
<meta name="citation_author_institution" content="INTAVIS Peptide Services GmbH &amp; Co.KG" />
<meta name="citation_author" content="Ole Winther" />
<meta name="citation_author_institution" content="Center for Genomic Medicine, Copenhagen University Hospital," />
<meta name="citation_author_institution" content="Bioinformatics Centre, Department of Biology, University of Copenhagen" />
<meta name="citation_author_institution" content="Department of Applied Mathematics and Computer Science, Technical University of Denmark" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1966-3205" />
<meta name="citation_author" content="Frederik Otzen Bagger" />
<meta name="citation_author_institution" content="Center for Genomic Medicine, Copenhagen University Hospital," />
<meta name="citation_author_institution" content="UKBB Universitats-Kinderspital, Department of Biomedicine" />
<meta name="citation_author_institution" content="Swiss Institute of Bioinformatics" />
<meta name="citation_author_email" content="frederik.otzen.bagger@regionh.dk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0636-8845" />
<meta name="citation_reference" content="Abdelmageed, Miyssa I., Abdelrahman H. Abdelmoneim, Mujahed I. Mustafa, Nafisa M. Elfadol, Naseem S. Murshed, Shaza W. Shantier, and Abdelrafie M. Makhawi. 2020. “Design of Multi Epitope-Based Peptide Vaccine against E Protein of Human COVID-19: An Immunoinformatics Approach.” bioRxiv. https://doi.org/10.1101/2020.02.04.934232." />
<meta name="citation_reference" content="Bassani-Sternberg, Michal, Eva Bräunlein, Richard Klar, Thomas Engleitner, Pavel Sinitcyn, Stefan Audehm, Melanie Straub, et al. 2016. “Direct Identification of Clinically Relevant Neoepitopes Presented on Native Human Melanoma Tissue by Mass Spectrometry.” Nature Communications 7 (November): 13404." />
<meta name="citation_reference" content="Bassani-Sternberg, Michal, Chloé Chong, Philippe Guillaume, Marthe Solleder, Huisong Pak, Philippe O. Gannon, Lana E. Kandalaft, George Coukos, and David Gfeller. 2017. “Deciphering HLA-I Motifs across HLA Peptidomes Improves Neo-Antigen Predictions and Identifies Allostery Regulating HLA Specificity.” PLoS Computational Biology 13 (8): e1005725." />
<meta name="citation_reference" content="Bhattacharya, Rohit, Ashok Sivakumar, Collin Tokheim, Violeta Beleva Guthrie, Valsamo Anagnostou, Victor E. Velculescu, and Rachel Karchin. 2017. “Evaluation of Machine Learning Methods to Predict Peptide Binding to MHC Class I Proteins.” https://doi.org/10.1101/154757." />
<meta name="citation_reference" content="Blicher, Thomas, Jette Sandholm Kastrup, Søren Buus, and Michael Gajhede. 2005. “High-Resolution Structure of HLA-A*1101 in Complex with SARS Nucleocapsid Peptide.” Acta Crystallographica. Section D, Biological Crystallography 61 (Pt 8): 1031–40." />
<meta name="citation_reference" content="Chen, Wen-Hsiang, Ulrich Strych, Peter J. Hotez, and Maria Elena Bottazzi. 2020. “The SARS-CoV-2 Vaccine Pipeline: An Overview.” Current Tropical Medicine Reports. https://doi.org/10.1007/s40475-020-00201-6." />
<meta name="citation_reference" content="Fast, Ethan, Russ B. Altman, and Binbin Chen. 2020. “Potential T-Cell and B-Cell Epitopes of 2019-nCoV.” bioRxiv. https://doi.org/10.1101/2020.02.19.955484." />
<meta name="citation_reference" content="Grifoni, Alba, John Sidney, Yun Zhang, Richard H. Scheuermann, Bjoern Peters, and Alessandro Sette. 2020. “A Sequence Homology and Bioinformatic Approach Can Predict Candidate Targets for Immune Responses to SARS-CoV-2.” Cell Host &amp; Microbe, March. https://doi.org/10.1016/j.chom.2020.03.002." />
<meta name="citation_reference" content="Han, Youngmahn, and Dongsup Kim. 2017. “Deep Convolutional Neural Networks for Pan-Specific Peptide-MHC Class I Binding Prediction.” BMC Bioinformatics 18 (1): 585." />
<meta name="citation_reference" content="Harndahl, Mikkel, Sune Justesen, Kasper Lamberth, Gustav Røder, Morten Nielsen, and Søren Buus. 2009. “Peptide Binding to HLA Class I Molecules: Homogenous, High-Throughput Screening, and Affinity Assays.” Journal of Biomolecular Screening 14 (2): 173–80." />
<meta name="citation_reference" content="Harndahl, Mikkel, Kasper Lamberth, Sune Justesen, Gustav Røder, Michael Madsen, Christina Sylvester-Hvid, Morten Nielsen, et al. 2006. “Large Scale Analysis of Peptide-HLA Class I Interactions.” 2006. https://www.iedb.org/reference/1000945." />
<meta name="citation_reference" content="Ishizuka, Jeffrey, Kristie Grebe, Eugene Shenderov, Bjoern Peters, Qiongyu Chen, Yanchun Peng, Lili Wang, et al. 2009. “Quantitating T Cell Cross-Reactivity for Unrelated Peptide Antigens.” Journal of Immunology 183 (7): 4337–45." />
<meta name="citation_reference" content="Jensen, Kamilla Kjaergaard, Massimo Andreatta, Paolo Marcatili, Søren Buus, Jason A. Greenbaum, Zhen Yan, Alessandro Sette, Bjoern Peters, and Morten Nielsen. 2018. “Improved Methods for Predicting Peptide Binding Affinity to MHC Class II Molecules.” Immunology 154 (3): 394–406." />
<meta name="citation_reference" content="Jørgensen, Kasper W., Michael Rasmussen, Søren Buus, and Morten Nielsen. 2014. “NetMHCstab - Predicting Stability of Peptide-MHC-I Complexes; Impacts for Cytotoxic T Lymphocyte Epitope Discovery.” Immunology 141 (1): 18–26." />
<meta name="citation_reference" content="Jurtz, Vanessa, Sinu Paul, Massimo Andreatta, Paolo Marcatili, Bjoern Peters, and Morten Nielsen. 2017. “NetMHCpan-4.0: Improved Peptide-MHC Class I Interaction Predictions Integrating Eluted Ligand and Peptide Binding Affinity Data.” Journal of Immunology 199 (9): 3360–68." />
<meta name="citation_reference" content="Justesen, Sune, Mikkel Harndahl, Kasper Lamberth, Lise-Lotte B. Nielsen, and Søren Buus. 2009. “Functional Recombinant MHC Class II Molecules and High-Throughput Peptide-Binding Assays.” Immunome Research 5 (May): 2." />
<meta name="citation_reference" content="Karosiene, Edita, Claus Lundegaard, Ole Lund, and Morten Nielsen. 2012. “NetMHCcons: A Consensus Method for the Major Histocompatibility Complex Class I Predictions.” Immunogenetics 64 (3): 177–86." />
<meta name="citation_reference" content="Kim, Yohan, John Sidney, Søren Buus, Alessandro Sette, Morten Nielsen, and Bjoern Peters. 2014. “Dataset Size and Composition Impact the Reliability of Performance Benchmarks for Peptide-MHC Binding Predictions.” BMC Bioinformatics 15 (July): 241." />
<meta name="citation_reference" content="Kim, Yohan, John Sidney, Clemencia Pinilla, Alessandro Sette, and Bjoern Peters. 2009. “Derivation of an Amino Acid Similarity Matrix for Peptide: MHC Binding and Its Application as a Bayesian Prior.” BMC Bioinformatics 10 (November): 394." />
<meta name="citation_reference" content="Liu, Geng, Dongli Li, Zhang Li, Si Qiu, Wenhui Li, Cheng-Chi Chao, Naibo Yang, et al. 2017. “PSSMHCpan: A Novel PSSM-Based Software for Predicting Class I Peptide-HLA Binding Affinity.” GigaScience 6 (5): 1–11." />
<meta name="citation_reference" content="Mei, Shutao, Fuyi Li, André Leier, Tatiana T. Marquez-Lago, Kailin Giam, Nathan P. Croft, Tatsuya Akutsu, et al. 2019. “A Comprehensive Review and Performance Evaluation of Bioinformatics Tools for HLA Class I Peptide-Binding Prediction.” Briefings in Bioinformatics, June. https://doi.org/10.1093/bib/bbz051." />
<meta name="citation_reference" content="Moutaftsi, Magdalini, Bjoern Peters, Valerie Pasquetto, David C. Tscharke, John Sidney, Huynh-Hoa Bui, Howard Grey, and Alessandro Sette. 2006. “A Consensus Epitope Prediction Approach Identifies the Breadth of Murine T(CD8+)-Cell Responses to Vaccinia Virus.” Nature Biotechnology 24 (7): 817–19." />
<meta name="citation_reference" content="Nielsen, Morten, Claus Lundegaard, and Ole Lund. 2007. “Prediction of MHC Class II Binding Affinity Using SMM-Align, a Novel Stabilization Matrix Alignment Method.” BMC Bioinformatics 8 (July): 238." />
<meta name="citation_reference" content="Nielsen, Morten, Claus Lundegaard, Peder Worning, Sanne Lise Lauemøller, Kasper Lamberth, Søren Buus, Søren Brunak, and Ole Lund. 2003. “Reliable Prediction of T-Cell Epitopes Using Neural Networks with Novel Sequence Representations.” Protein Science: A Publication of the Protein Society 12 (5): 1007–17." />
<meta name="citation_reference" content="O’Donnell, Timothy J., Alex Rubinsteyn, Maria Bonsack, Angelika B. Riemer, Uri Laserson, and Jeff Hammerbacher. 2018. “MHCflurry: Open-Source Class I MHC Binding Affinity Prediction.” Cell Systems 7 (1): 129–32.e4." />
<meta name="citation_reference" content="Paszke, Adam, Sam Gross, Soumith Chintala, Gregory Chanan, Edward Yang, Zachary Devito, Zeming Lin, Alban Desmaison, Luca Antiga, and Adam Lerer. 2017. “Automatic Differentiation in PyTorch.” https://pdfs.semanticscholar.org/b36a/5bb1707bb9c70025294b3a310138aae8327a.pdf." />
<meta name="citation_reference" content="Peters, Bjoern, Morten Nielsen, and Alessandro Sette. 2020. “T Cell Epitope Predictions.” Annual Review of Immunology, February. https://doi.org/10.1146/annurev-immunol-082119-124838." />
<meta name="citation_reference" content="Peters, Bjoern, and Alessandro Sette. 2005. “10.1186/1471-2105-6-132.” BMC Bioinformatics. https://doi.org/10.1186/1471-2105-6-132." />
<meta name="citation_reference" content="Qu, Zehui, Zibin Li, Lizhen Ma, Xiaohui Wei, Lijie Zhang, Ruiying Liang, Geng Meng, Nianzhi Zhang, and Chun Xia. 2019. “Structure and Peptidome of the Bat MHC Class I Molecule Reveal a Novel Mechanism Leading to High-Affinity Peptide Binding.” Journal of Immunology 202 (12): 3493–3506." />
<meta name="citation_reference" content="Rammensee, Hans-Georg. 1995. “Chemistry of Peptides Associated with MHC Class I and Class II Molecules.” Current Opinion in Immunology. https://doi.org/10.1016/0952-7915(95)80033-6." />
<meta name="citation_reference" content="Rasmussen, Michael, Mikkel Nors Harndahl, Anne Bregnballe Kristensen, Ida Kallehauge Nielsen, Kasper W. Jorgensen, Anette Stryhn, Morten Nielsen, and Sören Buus Buus. 2014. “Large Scale Analysis of Peptide - HLA-I Stability.” 2014. https://www.iedb.org/reference/1028288." />
<meta name="citation_reference" content="Saethang, Thammakorn, Osamu Hirose, Ingorn Kimkong, Vu Anh Tran, Xuan Tho Dang, Lan Anh T. Nguyen, Tu Kien T. Le, Mamoru Kubo, Yoichi Yamada, and Kenji Satou. 2012. “EpicCapo: Epitope Prediction Using Combined Information of Amino Acid Pairwise Contact Potentials and HLA-Peptide Contact Site Information.” BMC Bioinformatics 13 (November): 313." />
<meta name="citation_reference" content="Sarkizova, Siranush, Susan Klaeger, Phuong M. Le, Letitia W. Li, Giacomo Oliveira, Hasmik Keshishian, Christina R. Hartigan, et al. 2020. “A Large Peptidome Dataset Improves HLA Class I Epitope Prediction across Most of the Human Population.” Nature Biotechnology 38 (2): 199–209." />
<meta name="citation_reference" content="Sidney, John, Jason Botten, Benjamin Neuman, Michael Buchmeier, and Alessandro Sette. 2006. “Epitopes Described in - Immune Epitope Database (IEDB).” 2006. https://www.iedb.org/reference/1000425." />
<meta name="citation_reference" content="Sturniolo, T., E. Bono, J. Ding, L. Raddrizzani, O. Tuereci, U. Sahin, M. Braxenthaler, et al. 1999. “Generation of Tissue-Specific and Promiscuous HLA Ligand Databases Using DNA Microarrays and Virtual HLA Class II Matrices.” Nature Biotechnology 17 (6): 555–61." />
<meta name="citation_reference" content="Sylvester-Hvid, C., N. Kristensen, T. Blicher, H. Ferré, S. L. Lauemøller, X. A. Wolf, K. Lamberth, M. H. Nissen, L. Ø. Pedersen, and S. Buus. 2002. “Establishment of a Quantitative ELISA Capable of Determining Peptide - MHC Class I Interaction.” Tissue Antigens 59 (4): 251–58." />
<meta name="citation_reference" content="Sylvester-Hvid, C., M. Nielsen, K. Lamberth, G. Røder, S. Justesen, C. Lundegaard, P. Worning, et al. 2004. “SARS CTL Vaccine Candidates; HLA Supertype-, Genome-Wide Scanning and Biochemical Validation.” Tissue Antigens 63 (5): 395–400." />
<meta name="citation_reference" content="Wang, Peng, John Sidney, Courtney Dow, Bianca Mothé, Alessandro Sette, and Bjoern Peters. 2008. “A Systematic Assessment of MHC Class II Peptide Binding Predictions and Evaluation of a Consensus Approach.” PLoS Computational Biology 4 (4): e1000048." />
<meta name="citation_reference" content="WHO. 2020a. “Novel Coronavirus (2019-nCoV) SITUATION REPORT - 1 21 JANUARY 2020.” WHO. https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf?sfvrsn=20a99c10_4." />
<meta name="citation_reference" content="WHO. 2020b. “Coronavirus Disease 2019 (COVID-19) Situation Report – 46.” WHO. https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200307-sitrep-47-covid-19.pdf?sfvrsn=27c364a4_2." />
<meta name="citation_reference" content="WHO. 2020c. “Coronavirus Disease 2019 (COVID-19) Situation Report – 52.” WHO. https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200312-sitrep-52-covid-19.pdf?sfvrsn=e2bfc9c0_4." />
<meta name="citation_reference" content="Wu, Fan, Su Zhao, Bin Yu, Yan-Mei Chen, Wen Wang, Zhi-Gang Song, Yi Hu, et al. 2020. “A New Coronavirus Associated with Human Respiratory Disease in China.” Nature 579 (7798): 265–69." />
<meta name="citation_reference" content="Wu, Jingcheng, Wenzhe Wang, Jiucheng Zhang, Binbin Zhou, Wenyi Zhao, Zhixi Su, Xun Gu, Jian Wu, Zhan Zhou, and Shuqing Chen. 2019. “DeepHLApan: A Deep Learning Approach for Neoantigen Prediction Considering Both HLA-Peptide Binding and Immunogenicity.” Frontiers in Immunology 10 (November): 2559." />
<meta name="citation_reference" content="Zhang, Hao, Ole Lund, and Morten Nielsen. 2009. “The PickPocket Method for Predicting Binding Specificities for Receptors Based on Receptor Pocket Similarities: Application to MHC-Peptide Binding.” Bioinformatics 25 (10): 1293–99." />
<meta name="twitter:title" content="COVID-19 Vaccine Candidates: Prediction and Validation of 174 SARS-CoV-2 Epitopes" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="The recent outbreak of SARS-CoV-2 (2019-nCoV) virus has highlighted the need for fast and efficacious vaccine development. Stimulation of a proper immune response that leads to protection is highly dependent on presentation of epitopes to circulating T-cells via the HLA complex. SARS-CoV-2 is a large RNA virus and testing of all overlapping peptides in vitro to deconvolute an immune response is not feasible. Therefore HLA-binding prediction tools are often used to narrow down the number of peptides to test. We tested 19 epitope-HLA-binding prediction tools, and using an in vitro peptide MHC stability assay, we assessed 777 peptides that were predicted to be good binders across 11 MHC allotypes. In this investigation of potential SARS-CoV-2 epitopes we found that current prediction tools vary in performance when assessing binding stability, and they are highly dependent on the MHC allotype in question. Designing a COVID-19 vaccine where only a few epitope targets are included is therefore a very challenging task. Here, we present 174 SARS-CoV-2 epitopes with high prediction binding scores, validated to bind stably to 11 HLA allotypes. Our findings may contribute to the design of an efficacious vaccine against COVID-19." />
<meta name="og-title" property="og:title" content="COVID-19 Vaccine Candidates: Prediction and Validation of 174 SARS-CoV-2 Epitopes" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.20.000794v4" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="The recent outbreak of SARS-CoV-2 (2019-nCoV) virus has highlighted the need for fast and efficacious vaccine development. Stimulation of a proper immune response that leads to protection is highly dependent on presentation of epitopes to circulating T-cells via the HLA complex. SARS-CoV-2 is a large RNA virus and testing of all overlapping peptides in vitro to deconvolute an immune response is not feasible. Therefore HLA-binding prediction tools are often used to narrow down the number of peptides to test. We tested 19 epitope-HLA-binding prediction tools, and using an in vitro peptide MHC stability assay, we assessed 777 peptides that were predicted to be good binders across 11 MHC allotypes. In this investigation of potential SARS-CoV-2 epitopes we found that current prediction tools vary in performance when assessing binding stability, and they are highly dependent on the MHC allotype in question. Designing a COVID-19 vaccine where only a few epitope targets are included is therefore a very challenging task. Here, we present 174 SARS-CoV-2 epitopes with high prediction binding scores, validated to bind stably to 11 HLA allotypes. Our findings may contribute to the design of an efficacious vaccine against COVID-19." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-04-06" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.20.000794v4.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.20.000794v4.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.20.000794v4.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.20.000794v4" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1231850" />
    <title>COVID-19 Vaccine Candidates: Prediction and Validation of 174 SARS-CoV-2 Epitopes | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"fg_Kx2QYvEutmmez7iqwu0RUuFnlkEeqls80YaPAZMA","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1231850","apath":"\/biorxiv\/early\/2020\/04\/06\/2020.03.20.000794.atom","pisa":"biorxiv;2020.03.20.000794v4","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.20.000794v4"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.20.000794v4","title":"COVID-19 Vaccine Candidates: Prediction and Validation of 174 SARS-CoV-2 Epitopes","identifier":"node\/1231850"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022q4XqLJ5f6Coyly3RYiPMtkQdRAdqhySh3tPjAazE8If8pRcxaStNkXnVSJriSrwZe4WcEWDYsaAIubB77KQH1JXO4cwYq0sn8m\\\\\\\/Sn29330DpqCf8VhRykLgql5QxcdHUiSsLFhuQG+tzPb3trO68IC5uW1GNikdy1ZaHhfy9N44wuukmwrgApzyAj7tXGrUOSZXWi9GOCAXZM\\\\\\\/I78kjeO+1RppH1KnEkaA7KsGW50Cz8e55SbM\\\\\\\/EwvyBiYJr6TN\\\\\\\/kWRsxBaUtyjdIUs529Cp1u5kfVw\\\\\\\/\\\\\\\/qSgOtShNH3SQMO6TM0luVT\\\\\\\/nfGtlUJyzIUd6FjjBXq6JtIlJB7WuHdT26Fs1a241i26PuYcAj5jC7VZF2132uPLSFRcwj\\\\\\\/nlNivSjrMy\\\\\\\/UFzT6TzkCRTffxEhrT8311cDngK\\\\\\\/fTX9b7FQaFrrp+2lKIIDxSeQ1WqMFJsAc69XNOlNItYmCvl5Zv\\\\\\\/5F0zYAC1y8f1RfXo4W+rMRX1P3MZ\\\\\\\/E8NNLdPfrS3UPWjDRpBX0GxstM6554SiBcDhFkXPWhO4MF0ELxHSG0fsF4ViHOBEjTfWdx5BNZa0aKaypznuWMpaNeAbzn1TvP45NpcDXlwhifVOT6d67btSduGUc1AHUdrGM92+hnStchcJHP7svWRJSw\\\\\\\/DcBWZTF2dYrSyHp29PP\\\\\\\/eeHhz48gpmSW7UsxcYKmaR7XlMTN8cP3Ehd8TNiXJ7p8APtgrrL48d5HbHbzj+0XpusV+G1IqCTLTZ9mjQfRdqoqa+jvYWbQ0Uc4atsGd6X19tbWf17VlGSRgMLVFOJ2sZqJtW6TDDpm6OOAJ3gTUycucxJO0o\\\\\\\/JpHh973ae5qcpr5UoIuvQw2TzyEot0zfUOPRJFySWwTa53cfd7s6aTMIJe\\\\\\\/aZMXY4fjvVXDPxYNkgL+EC2OeB2n0sRW1UBZVQZQmn2UbPj+Q4ultLGFsuE6Xs0g0AGaBKZqTvYHJkz2NaoWKk1uQ5ceKWmN5HJTaXC2l9bagX\\\\\\\/aYcm+l7MVoHQbgx1F9FwQEq46eeU0xuxFrjY8HLfUcp7POf+TLBhQY8kBZ3FySCmTz4hJW2Ly6zbRFIPzy3swg\\\\\\\/F8wwsKt0tCUAi92P2pw9kLgo41StYxnfJbnoUgc2VaiuExobQhYYJ3ZBPqn\\\\\\\/mtLiX18qYVgDCvBDVgPEb2pbgzQtsd6V+NMQo6J7RKzcLXGzlXHc5d4PUt0OhFJ19BICbAw6yXhzof6A6q54QNH6YzlxVRyj3IMvTZmwxHbFyQmqm9hSLskjJSn0GWNrqP+z0m+G1kSq7+bX+Tw5cX+W8aFp6n63ugwy7IQcFaQVXYvjamJi07dqBSDWNYKk3fH\\\u0022,\\\u0022iv\\\u0022:\\\u0022ECS4t8SZhGo9qcTKPgxe5g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022e147e1dc5fe69e3a8113fd39db718f0cbf8763eb5da0bf28d821ec11872b512b\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.20.000794v4.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1231850 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.20.000794v4.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-SZCUwWE6lPjOdExWsL4Tael-tC1mqohJvkXkEvsWwtg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1231850" id="node1231850" data-pisa="biorxiv;2020.03.20.000794v4" data-pisa-master="biorxiv;2020.03.20.000794" data-apath="/biorxiv/early/2020/04/06/2020.03.20.000794.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">COVID-19 Vaccine Candidates: Prediction and Validation of 174 SARS-CoV-2 Epitopes</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1797-7147" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Marek</span> <span class="nlm-surname">Prachar</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-3039-2178" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Sune</span> <span class="nlm-surname">Justesen</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7891-4502" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Daniel Bisgaard</span> <span class="nlm-surname">Steen-Jensen</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Stephan</span> <span class="nlm-surname">Thorgrimsen</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Erik</span> <span class="nlm-surname">Jurgons</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-1966-3205" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Ole</span> <span class="nlm-surname">Winther</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-0636-8845" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Frederik Otzen</span> <span class="nlm-surname">Bagger</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.20.000794 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1231850" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marek Prachar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for Genomic Medicine, Copenhagen University Hospital,</span> Copenhagen, <span class='nlm-country'>Denmark</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Bioinformatics Centre, Department of Biology, University of Copenhagen</span>, Copenhagen, <span class='nlm-country'>Denmark</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Immunitrack ApS</span>, Copenhagen, <span class='nlm-country'>Denmark</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marek%2BPrachar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Prachar%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarek%2BPrachar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1797-7147" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marek Prachar</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sune Justesen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Immunitrack ApS</span>, Copenhagen, <span class='nlm-country'>Denmark</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sune%2BJustesen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Justesen%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASune%2BJustesen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3039-2178" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sune Justesen</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daniel Bisgaard Steen-Jensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Immunitrack ApS</span>, Copenhagen, <span class='nlm-country'>Denmark</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BBisgaard%2BSteen-Jensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Steen-Jensen%20DB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BBisgaard%2BSteen-Jensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7891-4502" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel Bisgaard Steen-Jensen</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Stephan Thorgrimsen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Immunitrack ApS</span>, Copenhagen, <span class='nlm-country'>Denmark</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephan%2BThorgrimsen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thorgrimsen%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephan%2BThorgrimsen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Erik Jurgons </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>INTAVIS Peptide Services GmbH &#38; Co.KG</span>, Waldhäuser Str. 64, 72076 Tübingen, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Erik%2BJurgons%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jurgons%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AErik%2BJurgons%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Ole Winther </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for Genomic Medicine, Copenhagen University Hospital,</span> Copenhagen, <span class='nlm-country'>Denmark</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Bioinformatics Centre, Department of Biology, University of Copenhagen</span>, Copenhagen, <span class='nlm-country'>Denmark</span></div><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Department of Applied Mathematics and Computer Science, Technical University of Denmark</span>, 2800 Kgs. Lyngby, <span class='nlm-country'>Denmark</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ole%2BWinther%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Winther%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOle%2BWinther%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1966-3205" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ole Winther</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Frederik Otzen Bagger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for Genomic Medicine, Copenhagen University Hospital,</span> Copenhagen, <span class='nlm-country'>Denmark</span></div><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>UKBB Universitats-Kinderspital, Department of Biomedicine</span>, Basel, 4031 Basel, <span class='nlm-country'>Switzerland</span></div><div class='author-affiliation'><span class='nlm-sup'>7</span><span class='nlm-institution'>Swiss Institute of Bioinformatics</span>, Basel, 4053 Basel, <span class='nlm-country'>Switzerland</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frederik%2BOtzen%2BBagger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bagger%20FO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrederik%2BOtzen%2BBagger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0636-8845" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Frederik Otzen Bagger</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:frederik.otzen.bagger@regionh.dk" class="" data-icon-position="" data-hide-link-title="0">frederik.otzen.bagger@regionh.dk</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.20.000794v4" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1231850" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1231850/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.20.000794v4.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1231850" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1231850/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.20.000794v4.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1231850" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1231850/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.20.000794v4.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1231850" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1231850/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.20.000794v4.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1231850" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1231850/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.20.000794v4.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1231850" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1231850/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">The recent outbreak of SARS-CoV-2 (2019-nCoV) virus has highlighted the need for fast and efficacious vaccine development. Stimulation of a proper immune response that leads to protection is highly dependent on presentation of epitopes to circulating T-cells via the HLA complex. SARS-CoV-2 is a large RNA virus and testing of all overlapping peptides <em>in vitro</em> to deconvolute an immune response is not feasible. Therefore HLA-binding prediction tools are often used to narrow down the number of peptides to test. We tested 19 epitope-HLA-binding prediction tools, and using an <em>in vitro</em> peptide MHC stability assay, we assessed 777 peptides that were predicted to be good binders across 11 MHC allotypes. In this investigation of potential SARS-CoV-2 epitopes we found that current prediction tools vary in performance when assessing binding stability, and they are highly dependent on the MHC allotype in question. Designing a COVID-19 vaccine where only a few epitope targets are included is therefore a very challenging task. Here, we present 174 SARS-CoV-2 epitopes with high prediction binding scores, validated to bind stably to 11 HLA allotypes. Our findings may contribute to the design of an efficacious vaccine against COVID-19.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-4">2019-nCoV (SARS-CoV-2) was first reported in Wuhan, China, on 31 December 2019, following a series of unexplained pneumonia cases (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">WHO 2020a</a>). Currently, the disease is intensifying with case reports over a continuously growing geographical area. WHO now gives the risk assessment ‘Very High’ on a global level and classifies the situation as pandemic (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">WHO 2020b</a>, <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">[c] 2020</a>). Vaccine development is of high priority at this stage, and a number of public and private initiatives are focused on this task (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chen et al. 2020</a>). Most, if not all, ongoing vaccine development efforts are focused on raising an immune response against the spike protein. However, the spike protein only makes up 1/8 of the COVID-19 genome, so this vaccine strategy may inadvertently miss a lot of potential immune reactivity. SARS-CoV-2 has a large proteome (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">F. Wu et al. 2020</a>). Immune deconvolution to identify T cell epitopes will require initial filtering to assess which SARS-CoV-2-derived peptides are likely to bind a given HLA allotype and to be presented on the surface of infected cells from where it can activate passing T cells. The core binding groove of most MHC molecules can accommodate 9 amino acid residues, with some variation or suspected impact of flanking positions (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bassani-Sternberg et al. 2016</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Rammensee 1995</a>).</p><p id="p-5">Several tools (a selection is presented in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) have been developed that can predict the binding of peptides to HLA. Traditionally, these tools were trained using data from affinity assays (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Harndahl et al. 2009</a>), but more recently many of them also incorporate data from peptides identified by HLA ligandome analysis. Most tools rely on small neural networks (NN) or variations of position-specific weight matrices (PSSM), to calculate the probability of a peptide matching a consensus motif or model.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1239323/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1239323/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1239323" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-6" class="first-child">Current best-performing or novel HLA prediction tools (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Mei et al. 2019</a>). Webservers checked on 2 March 2020, NN: Neural network, Cons: Consensus, PSSM: Position specific scoring matrix.</p><p id="p-7">*availability as a fraction of allotypes included in study (10 HLA class I and 1 of HLA class II)</p><div class="sb-div caption-clear"></div></div></div><p id="p-8">NetMHC tools (such as netMHC, netMHCII, netMHCpan, netMHCIIpan and others) have been under constant development and have consistently performed well throughout the last decade, (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Peters, Nielsen, and Sette 2020</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Mei et al. 2019</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Saethang et al. 2012</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bhattacharya et al. 2017</a>). Several tools are restricted in terms of which allotypes are available for prediction, in particular for MHC class II. This restriction is primarily determined by the availability of training data, for which the large public collection is currently the Immune Epitope Database (IEDB) (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Sidney et al. 2006</a>). Attempts to overcome this limitation have been made via sequence-to-sequence predictions, most notably for netMHCpan (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Jurtz et al. 2017</a>). A number of recent publications makes use of prediction tools to suggest vaccine candidate epitopes for SARS-CoV-2 (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Fast, Altman, and Chen 2020</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Grifoni et al. 2020</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abdelmageed et al. 2020</a>).</p><p id="p-9">To assess whether current peptide-HLA prediction tools could be suitable for identification of epitopes relevant in a vaccine against SARS-CoV-2, we tested predicted binders from the netMHC tools, using a new MHC:peptide complex stability assay NeoScreen<sup>®</sup>. We found that algorithmically predicting binding between epitopes from SARS-CoV-2 and HLA outputs many complexes that turned out to exhibit low stability. Such peptides are thus very unlikely to elicit an immune response against SARS-CoV-2 and are therefore unsuitable for vaccine development. To investigate if this finding was a result of the quality of available training data, we constructed a proof-of-concept prediction model, which we trained on 2,193 historic in-house stability data, and found that it outperforms other tools. Training data was primarily human cancer-derived or based on random sequences. SARS-CoV-2 peptides that we validated as binding or non-binding in this study are freely available for use to assist in vaccine design against COVID-19.</p></div><div class="section" id="sec-2"><h2 class="">Results</h2><p id="p-10">We set out to identify peptides with epitope potential in a future COVID-19 vaccine. We commenced by translating the reference sequence of SARS-CoV-2 (ACCESSION <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MN908947&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.20.000794.atom">MN908947</a>, VERSION <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MN908947.3&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.20.000794.atom">MN908947.3</a>) to protein-coding sequence and predicted potential epitopes in a sliding window of 9 using netMHC tools (netMHC/II and “-pan” versions, when allotype was not available). We identified the top 94 predicted peptides for 11 HLA allotypes (94 x 11 = 1034), and went further to validate the binding of these 94 peptides to each allotype in an <em>in vitro</em> MHC:peptide complex stability assay (NeoScreen<sup>®</sup>). We removed eight peptides that were synthetically introduced when translating the DNA sequence to protein sequence. Of the remaining 1,026 peptides we observed a high degree of overlap between different allotypes, resulting in 777 unique peptides. We found that 174 of the 777 unique peptides formed a stable peptide-HLA complex. Of these 174 peptides, 48 were previously measured and deposited in IEDB in relation to SARS and the remaining 126 peptides are novel. The full list of predicted binders (excluding synthetic peptides) can be found in the supplementary material (Supplementary Data S1).</p><p id="p-11">In order to first assess potential variability across the stability measurements we made replicate measurements (<em>n</em>= 4) of 120 peptides on 8 HLA alleles. Each peptide was measured with urea in 4 different concentrations (0M, 2M, 4M, 6M), and we observed an average standard deviation between replicates of 0.10 with an average mean of 0.56. All remaining experiments were performed in duplicate for all concentrations.</p><p id="p-12">To further address whether alternative prediction tools would have higher concordance with measured stability, we performed predictions for all tools listed in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>. Predictions for the 19 different tools were performed either through their web server or a stand-alone version, (see Materials and methods section for details). Furthermore, using in-house stability data, we developed PrdX 1.0, a prediction tool for a single allele HLA-A*02:01, where all other tools performed most poorly.</p><p id="p-13">We assessed the false positive rate for each tool via Receiver Operating Characteristic (ROC) curves, and their Area under curve (AUC) for all allotypes that had more than 10 binders. The analysis revealed that ANN 4.0 achieved the highest score for allotype HLA-A*01:01 (AUC = 97,47; <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>), closely followed by NetMHCcons 1.1, NetMHCpan_BA 4.0 and IEDB-AR Consensus. PrdX 1.0 scored highest for HLA-A*02:01 (AUC = 85.54; <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>), NetMHCcons 1.1 scored highest for HLA-A*03:01 (AUC = 79.25; <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>), and MHCflurry 1.3.0 performed best for HLA-B*40:01 (AUC = 91.06; <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1F</a>). NetMHCstab 1.0 was the only tool that achieved the highest score for more than 1 allotype: HLA-A*11:01 and HLA-A*24:02 (AUC = 89.80; 86.03; <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1D, E</a>, respectively). Out of the tools tested for HLA class II, IEDB-AR Consensus achieved the highest score for HLA-DRB1*04:01 (AUC = 81,31; <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1G</a>). <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> provides all AUC values, and the result obtained for each allotype is marked in bold. Notably, in the case of HLA-A*02:01 we observed particularly poor performance among all tested tools despite the extensive amount of data available for this allotype.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1239324/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1239324/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1239324" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2.</span> <p id="p-14" class="first-child">AUC values for ROC curves from <a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a> for allotypes with more than 10 stable complexes. First 6 six columns contain HLA class I, last column contains HLA class II. Only four tools are tested for HLA class II. PrdX 1.0 is only available for allotype A*02:01. Highest value for each allotype is marked in bold.</p><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.20.000794/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="ROC curves for each allotype that bound more than 10 peptides stably (subplots A, B, C, D, E, F, G), H) tools used in the benchmark, I) precision-recall curves for HLA-A*02:01. Corresponding area under curve (AUC) values are listed in Table 2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1073220954" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;ROC curves for each allotype that bound more than 10 peptides stably (subplots A, B, C, D, E, F, G), H) tools used in the benchmark, I) precision-recall curves for HLA-A*02:01. Corresponding area under curve (AUC) values are listed in Table 2.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.20.000794/F1.medium.gif" width="403" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.20.000794/F1.medium.gif" width="403" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.20.000794/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.20.000794/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-15" class="first-child">ROC curves for each allotype that bound more than 10 peptides stably (subplots A, B, C, D, E, F, G), H) tools used in the benchmark, I) precision-recall curves for HLA-A*02:01. Corresponding area under curve (AUC) values are listed in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">To assess the correlation between the predicted and measured peptide-HLA complexes, Spearman correlation was calculated for all allotypes. This revealed significant inconsistencies in performance depending on the predicted allotype. PSSMHCpan 1.0 displayed the highest consistency, taking into account its coverage (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>), but the correlation median scored lower than other tools such as IEDB-AR Consensus, MixMHCpred 2.0.2, NetMHCpan_EL 4.0 or PrdX 1.0. The results of the Spearman correlations are summarised in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.20.000794/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plot of Spearman correlation between predicted values and results of Neoscreen&#xAE; stability assay for each available allotype. Each colour represents an individual allotype. Whiskers accord for 1.5 distance between the median and quartile hinges. Data points beyond the end of the whiskers are outliers." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1073220954" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Plot of Spearman correlation between predicted values and results of Neoscreen&lt;sup&gt;&#xAE;&lt;/sup&gt; stability assay for each available allotype. Each colour represents an individual allotype. Whiskers accord for 1.5 distance between the median and quartile hinges. Data points beyond the end of the whiskers are outliers.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.20.000794/F2.medium.gif" width="440" height="308"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.20.000794/F2.medium.gif" width="440" height="308"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.20.000794/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.03.20.000794/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-17" class="first-child">Plot of Spearman correlation between predicted values and results of Neoscreen<sup>®</sup> stability assay for each available allotype. Each colour represents an individual allotype. Whiskers accord for 1.5 distance between the median and quartile hinges. Data points beyond the end of the whiskers are outliers.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-3"><h2 class="">Discussion</h2><p id="p-18">Here we benchmark a number of tools to identify epitopes for SARS-CoV-2 virus, and validate via stability assay the binding of candidate epitopes to 10 allotypes of HLA class I and one allotype of HLA class II. We find that the false positive rate is high for all tested tools when testing for predicted HLA-binding peptides from SARS-CoV-2 virus. This creates a challenge for vaccine development efforts, especially for the design of epitope vaccines, where only a limited number of epitopes may be included. Furthermore, it highlights the risk for failed vaccine design (for any pathogen or disease) if predicted HLA-binding protein regions in reality do not bind and allow immune presentation and response.</p><p id="p-19">We observed, remarkably, that all tools tested performed poorly for HLA-A*02:01, which is the allotype with most training data available (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kim et al. 2014</a>). Based on our observations we hypothesise that publicly available training data is not of high enough quality. This is supported by the fact that AUC and Spearman correlations indicate that performance seems to correlate with the allotypes and not the tools; thus suggesting that either the training data or the difficulty of modelling the allotype is responsible for poor predictions. To test the hypothesis that training data is limiting for tool performance, we trained a vanilla NN on only 2,193 historic in-house stability measurements, and found that our model outperformed all tested prediction tools in this setting. This observation could also be explained by more similar data distributions between test and training data for PrdX 1.0.</p><p id="p-20">We identified 174 potential SARS-CoV-2 vaccine candidate peptides, out of which 48 have been previously deposited in IEDB following various studies (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Qu et al. 2019</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Blicher et al. 2005</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Sylvester-Hvid et al. 2004</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Harndahl et al. 2006</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Sidney et al. 2006</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Ishizuka et al. 2009</a>). The majority of the previously deposited peptides were measured in one or multiple affinity assays and reached low Kd (&lt;50 nM) values, indicating strong affinity. Additionally, 9 of these peptides were previously measured in another stability assay and were recognised as stable binders (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Rasmussen et al. 2014</a>), independently confirming our approach and measurements. Of the 48 peptides deposited in IEDB, two (FLLPSLATV <a href="https://www.iedb.org/epitope/16743">https://www.iedb.org/epitope/16743</a>, FLNRFTTTL <a href="https://www.iedb.org/epitope/16786">https://www.iedb.org/epitope/16786</a>) have previously been assessed for their ability to provoke T cell responses, with negative T cell responses for both.</p><p id="p-21">To further assess the potential of the SARS-CoV-2 epitopes we explored their overlap with T cell-confirmed epitopes from SARS that are deposited in the IEDB. We identified 4 peptides within our candidates that are represented within these confirmed epitopes as partial motifs.</p><p id="p-22">In conclusion, we make freely available the identities of 174 COVID-19 epitopes that we have predicted and validated <em>in vitro</em> to be HLA-binding. We hope that this contribution will aid the development of a vaccine against SARS-CoV-2. We performed a benchmark analysis of 20 tools on 777 peptides that were predicted by state-of-the-art prediction tools from netMHC to be binders, and revealed high false positive rates for all benchmarked tools. We observed improved performance after training our own prediction tool PrdX 1.0 on allotype A*02:01 using in-house generated stability data. Our findings suggest that the performance of current state-of-the-art epitope prediction tools are impacted by the varying quality of publicly available data.</p></div><div class="section" id="sec-4"><h2 class="">Data availability</h2><p id="p-23">All epitopes are available at the vendor webpage (<a href="http://www.immunitrack.com">www.immunitrack.com</a>) and in supplemental materials (Data S2.).</p></div><div class="section" id="sec-5"><h2 class="">Materials and methods</h2><p id="p-24">Fifteen prediction tools tested on a relevant dataset of peptides from the SARS-CoV-2 genome (assembly MN908947.3). The genome sequence was downloaded from the NCBI database (<a href="https://www.ncbi.nlm.nih.gov/nuccore/MN908947.3">https://www.ncbi.nlm.nih.gov/nuccore/MN908947.3</a>) (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">F. Wu et al. 2020</a>). Using NetMHC tools we predicted the top 94 peptides for HLA-A*01:01, HLA-A*02:01, HLA-A*03:01, HLA-A*24:02, HLA-B*40:01, HLA-C*04:01, HLA-C*07:01, HLA-C*07:02 (netMHC 4.0), HLA-C*01:02 (netMHCpan 4.0) and HLA-DRB1*04:01 (netMHCII 2.3). Subsequently, the peptides were analysed for binding stability to the respective HLA allotype. Taking into account the cross-reactivity between the two allotypes, peptides predicted to bind HLA-A*03:01 were also measured on HLA-A*11:01.</p><p id="p-25">Peptides were synthesised using standard Fmoc solid-phase synthesis on a modified cellulose support as solid support according to the SPOT synthesis protocol, starting with the acid labile Ramage linker.</p><p id="p-26">After synthesis, peptides were cleaved off the membranes using 95% trifluoroacetic acid (TFA),3% triisopropylsilane (TIS) &amp; 2% H2O.Peptides were then precipitated with diethylether and washed with methyl-tert-butylether.</p><p id="p-27">Peptides were subsequently dissolved in a proprietary mixture and dried under vacuum using a speed vac. Finally, 5% of all peptides were analysed by MALDI-TOF to confirm correct molecular weight. The anticipated yield per spot was 50 μg.</p><div id="sec-6" class="subsection"><h3>NeoScreen<sup>®</sup> assay</h3><p id="p-28">The NeoScreen<sup>®</sup> stability assay utilises urea denaturation to assess peptide:MHC complex stability. Briefly, peptides were dissolved in 200 µl DMSO with 1 mM β-mercaptoethanol and subsequently diluted into an assay buffer in 96 well plates at a final concentration of 2 µM. Positions A1 and H12 were reserved for a mixture of reference peptides with known stable binding to the MHC of interest. MHC I was diluted into an assay buffer with beta 2 microglobulin and added at a 1:1 ratio to diluted peptides. For MHC II, the urea-denatured alpha and beta chains were diluted into an assay buffer and added at a 1:1 ratio to diluted peptides. The concentration of MHC depended on the actual chain but final concentrations were in the range of 2-10 nM (hence peptide was added in excess). Upon folding, peptide:MHC complexes were transferred to 384 well plates where they were challenged with 4 different urea concentrations. Following the period of urea-induced stress the plates were developed in a conventional ELISA as described previously (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Justesen et al. 2009</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Sylvester-Hvid et al. 2002</a>). The ABS450 nm signals from the 4 different wells were averaged and normalised to the included reference to the included reference peptides in wells A1 and H12.</p></div><div id="sec-7" class="subsection"><h3>Benchmarking of tools</h3><p id="p-29"><a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a> provides a summary of tools tested in this benchmark analysis. It features the year of their development, the algorithm used, web server availability and a reference. Most of the tested tools are available at the IEDB Analysis Resource web page (<a href="http://tools.iedb.org/main/">http://tools.iedb.org/main/</a>) and were run through their web interface (<a href="http://tools.iedb.org/mhci/">http://tools.iedb.org/mhci/</a> or <a href="http://tools.iedb.org/mhcii/">http://tools.iedb.org/mhcii/</a>). MixMHCpred, MHCflurry and PSSMHCpan were downloaded from their respective github pages (<a href="https://github.com/GfellerLab/MixMHCpred">https://github.com/GfellerLab/MixMHCpred</a>,<a href="https://github.com/openvax/mhcflurry">https://github.com/openvax/mhcflurry</a>, <a href="https://github.com/BGI2016/PSSMHCpan">https://github.com/BGI2016/PSSMHCpan</a>, respectively). ConvMHC, DeepHLApan and HLAthena were used from their privately hosted web servers (<a href="http://jumong.kaist.ac.kr:8080/convmhc">http://jumong.kaist.ac.kr:8080/convmhc</a>, <a href="http://biopharm.zju.edu.cn/deephlapan/">http://biopharm.zju.edu.cn/deephlapan/</a>, <a href="http://hlathena.tools/">http://hlathena.tools/</a>, respectively).</p><p id="p-30">All tested peptides were subjected to in silico predictions (with each prediction tool) regarding their available allotype. Predictions were compared against measured stability determinations obtained through the NeoScreen<sup>®</sup> assay. Measurements were normalised to an allotype-specific reference peptide (stability = 100). The list of reference peptides used is available in supplementary material (Table S1.). The threshold for a stable binder was set to 60. Predictions were subsequently evaluated according to commonly-used metrics such as the Receiver Operating Characteristic (ROC) and its Area Under Curve (AUC) to visualise the relationship between sensitivity and specificity. Spearman correlation was also used to compare the ranked correlation of predicted and measured data.</p></div><div id="sec-8" class="subsection"><h3>PrdX</h3><p id="p-31">To assess the performance of predictors trained on stability data we used PyTorch (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Paszke et al. 2017</a>) to train a fully connected, feed-forward neural network with 64 and 32 hidden units on historic in-house stability data from allotype A*02:01. This data contains a mixture of human cancer-related stability measurements and measurements made on synthetic random peptides. We used BLOSUM62 matrix for encoding, simple network architecture, train-test split and early stopping for training.</p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-update fn-group-summary-of-updates" id="fn-1"><p id="p-3">Corrections in Figure 1 and Table 2.</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.20.000794v4.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Abdelmageed</span>,  <span class="cit-name-given-names">Miyssa I.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Abdelrahman H.</span> <span class="cit-name-surname">Abdelmoneim</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Mujahed I.</span> <span class="cit-name-surname">Mustafa</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Nafisa M.</span> <span class="cit-name-surname">Elfadol</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Naseem S.</span> <span class="cit-name-surname">Murshed</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Shaza W.</span> <span class="cit-name-surname">Shantier</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Abdelrafie M.</span> <span class="cit-name-surname">Makhawi</span></span>. <span class="cit-pub-date">2020</span>. “<span class="cit-article-title">Design of Multi Epitope-Based Peptide Vaccine against E Protein of Human COVID-19: An Immunoinformatics Approach</span>.” <span class="cit-source">bioRxiv</span>. <a href="https://doi.org/10.1101/2020.02.04.934232">https://doi.org/10.1101/2020.02.04.934232</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000794v4.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bassani-Sternberg</span>,  <span class="cit-name-given-names">Michal</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Eva</span> <span class="cit-name-surname">Bräunlein</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Richard</span> <span class="cit-name-surname">Klar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Thomas</span> <span class="cit-name-surname">Engleitner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Pavel</span> <span class="cit-name-surname">Sinitcyn</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Stefan</span> <span class="cit-name-surname">Audehm</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Melanie</span> <span class="cit-name-surname">Straub</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2016</span>. “<span class="cit-article-title">Direct Identification of Clinically Relevant Neoepitopes Presented on Native Human Melanoma Tissue by Mass Spectrometry</span>.” <abbr class="cit-jnl-abbrev">Nature Communications</abbr> <span class="cit-vol">7</span> (<span class="cit-month">November</span>): <span class="cit-fpage">13404</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D7%26rft.spage%253D13404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.20.000794v4.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bassani-Sternberg</span>,  <span class="cit-name-given-names">Michal</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chloé</span>  <span class="cit-name-given-names">Chong</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Philippe</span> <span class="cit-name-surname">Guillaume</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Marthe</span> <span class="cit-name-surname">Solleder</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Huisong</span> <span class="cit-name-surname">Pak</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Philippe O.</span> <span class="cit-name-surname">Gannon</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Lana E.</span> <span class="cit-name-surname">Kandalaft</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">George</span> <span class="cit-name-surname">Coukos</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">David</span> <span class="cit-name-surname">Gfeller</span></span>. <span class="cit-pub-date">2017</span>. “<span class="cit-article-title">Deciphering HLA-I Motifs across HLA Peptidomes Improves Neo-Antigen Predictions and Identifies Allostery Regulating HLA Specificity</span>.” <abbr class="cit-jnl-abbrev">PLoS Computational Biology</abbr> <span class="cit-vol">13</span> (<span class="cit-issue">8</span>): <span class="cit-fpage">e1005725</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BComputational%2BBiology%26rft.volume%253D13%26rft.spage%253D1005725e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.20.000794v4.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bhattacharya</span>,  <span class="cit-name-given-names">Rohit</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Ashok</span> <span class="cit-name-surname">Sivakumar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Collin</span> <span class="cit-name-surname">Tokheim</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Violeta Beleva</span> <span class="cit-name-surname">Guthrie</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Valsamo</span> <span class="cit-name-surname">Anagnostou</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Victor E.</span> <span class="cit-name-surname">Velculescu</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Rachel</span> <span class="cit-name-surname">Karchin</span></span>. <span class="cit-pub-date">2017</span>. “<span class="cit-source">Evaluation of Machine Learning Methods to Predict Peptide Binding to MHC Class I Proteins</span>.” <a href="https://doi.org/10.1101/154757">https://doi.org/10.1101/154757</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000794v4.5" data-doi="10.1107/S0907444905013090"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Blicher</span>,  <span class="cit-name-given-names">Thomas</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Jette Sandholm</span> <span class="cit-name-surname">Kastrup</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Søren</span> <span class="cit-name-surname">Buus</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Michael</span> <span class="cit-name-surname">Gajhede</span></span>. <span class="cit-pub-date">2005</span>. “<span class="cit-article-title">High-Resolution Structure of HLA-A*1101 in Complex with SARS Nucleocapsid Peptide</span>.” <abbr class="cit-jnl-abbrev">Acta Crystallographica. Section D, Biological Crystallography</abbr> <span class="cit-vol">61</span> (<span class="cit-issue">Pt 8</span>): <span class="cit-fpage">1031</span>–<span class="cit-lpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BCrystallographica%252C%2BSection%2BD%253A%2BBiological%2BCrystallography%26rft.stitle%253DActa%2BCrystallographica%252C%2BSection%2BD%253A%2BBiological%2BCrystallography%26rft.aulast%253DBlicher%26rft.auinit1%253DT.%26rft.volume%253D61%26rft.issue%253DPt%2B8%26rft.spage%253D1031%26rft.epage%253D1040%26rft.atitle%253DHigh-resolution%2Bstructure%2Bof%2BHLA-A%2A1101%2Bin%2Bcomplex%2Bwith%2BSARS%2Bnucleocapsid%2Bpeptide.%26rft_id%253Dinfo%253Adoi%252F10.1107%252FS0907444905013090%26rft_id%253Dinfo%253Apmid%252F16041067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1107/S0907444905013090&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16041067&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.20.000794.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.20.000794v4.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">Wen-Hsiang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Ulrich</span> <span class="cit-name-surname">Strych</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Peter J.</span> <span class="cit-name-surname">Hotez</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Maria Elena</span> <span class="cit-name-surname">Bottazzi</span></span>. <span class="cit-pub-date">2020</span>. “<span class="cit-article-title">The SARS-CoV-2 Vaccine Pipeline: An Overview</span>.” <span class="cit-source">Current Tropical Medicine Reports</span>. <a href="https://doi.org/10.1007/s40475-020-00201-6">https://doi.org/10.1007/s40475-020-00201-6</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.20.000794v4.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fast</span>,  <span class="cit-name-given-names">Ethan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Russ B.</span> <span class="cit-name-surname">Altman</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Binbin</span> <span class="cit-name-surname">Chen</span></span>. <span class="cit-pub-date">2020</span>. “<span class="cit-article-title">Potential T-Cell and B-Cell Epitopes of 2019-nCoV</span>.” <span class="cit-source">bioRxiv</span>. <a href="https://doi.org/10.1101/2020.02.19.955484">https://doi.org/10.1101/2020.02.19.955484</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.20.000794v4.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grifoni</span>,  <span class="cit-name-given-names">Alba</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">John</span> <span class="cit-name-surname">Sidney</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Yun</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Richard H.</span> <span class="cit-name-surname">Scheuermann</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Bjoern</span> <span class="cit-name-surname">Peters</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Alessandro</span> <span class="cit-name-surname">Sette</span></span>. <span class="cit-pub-date">2020</span>. “<span class="cit-article-title">A Sequence Homology and Bioinformatic Approach Can Predict Candidate Targets for Immune Responses to SARS-CoV-2</span>.” <span class="cit-source">Cell Host &amp; Microbe</span>, <span class="cit-month">March</span>. <a href="https://doi.org/10.1016/j.chom.2020.03.002">https://doi.org/10.1016/j.chom.2020.03.002</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.20.000794v4.9" data-doi="10.1186/s12859-017-1997-x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">Youngmahn</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Dongsup</span> <span class="cit-name-surname">Kim</span></span>. <span class="cit-pub-date">2017</span>. “<span class="cit-article-title">Deep Convolutional Neural Networks for Pan-Specific Peptide-MHC Class I Binding Prediction</span>.” <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">18</span> (<span class="cit-issue">1</span>): <span class="cit-fpage">585</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D18%26rft.spage%253D585%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-017-1997-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-017-1997-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000794v4.10" data-doi="10.1177/1087057108329453"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Harndahl</span>,  <span class="cit-name-given-names">Mikkel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Sune</span> <span class="cit-name-surname">Justesen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Kasper</span> <span class="cit-name-surname">Lamberth</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gustav</span>  <span class="cit-name-given-names">Røder</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Morten</span> <span class="cit-name-surname">Nielsen</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Søren</span> <span class="cit-name-surname">Buus</span></span>. <span class="cit-pub-date">2009</span>. “<span class="cit-article-title">Peptide Binding to HLA Class I Molecules: Homogenous, High-Throughput Screening, and Affinity Assays</span>.” <abbr class="cit-jnl-abbrev">Journal of Biomolecular Screening</abbr> <span class="cit-vol">14</span> (<span class="cit-issue">2</span>): <span class="cit-fpage">173</span>–<span class="cit-lpage">80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiomolecular%2BScreening%26rft_id%253Dinfo%253Adoi%252F10.1177%252F1087057108329453%26rft_id%253Dinfo%253Apmid%252F19196700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1177/1087057108329453&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19196700&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.20.000794.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263990000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.20.000794v4.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Harndahl</span>,  <span class="cit-name-given-names">Mikkel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Kasper</span> <span class="cit-name-surname">Lamberth</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Sune</span> <span class="cit-name-surname">Justesen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Gustav</span> <span class="cit-name-surname">Røder</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Michael</span> <span class="cit-name-surname">Madsen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Christina</span> <span class="cit-name-surname">Sylvester-Hvid</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Morten</span> <span class="cit-name-surname">Nielsen</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2006</span>. “<span class="cit-source">Large Scale Analysis of Peptide-HLA Class I Interactions.” 2006</span>. <a href="https://www.iedb.org/reference/1000945">https://www.iedb.org/reference/1000945</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000794v4.12" data-doi="10.4049/jimmunol.0901607"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ishizuka</span>,  <span class="cit-name-given-names">Jeffrey</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Kristie</span> <span class="cit-name-surname">Grebe</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Eugene</span> <span class="cit-name-surname">Shenderov</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Bjoern</span> <span class="cit-name-surname">Peters</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Qiongyu</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Yanchun</span> <span class="cit-name-surname">Peng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Lili</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2009</span>. “<span class="cit-article-title">Quantitating T Cell Cross-Reactivity for Unrelated Peptide Antigens</span>.” <abbr class="cit-jnl-abbrev">Journal of Immunology</abbr> <span class="cit-vol">183</span> (<span class="cit-issue">7</span>): <span class="cit-fpage">4337</span>–<span class="cit-lpage">45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BImmunology%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.0901607%26rft_id%253Dinfo%253Apmid%252F19734234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTA6IjE4My83LzQzMzciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wNi8yMDIwLjAzLjIwLjAwMDc5NC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.20.000794v4.13" data-doi="10.1111/imm.12889"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jensen</span>,  <span class="cit-name-given-names">Kamilla Kjaergaard</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Massimo</span> <span class="cit-name-surname">Andreatta</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Paolo</span> <span class="cit-name-surname">Marcatili</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Søren</span> <span class="cit-name-surname">Buus</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Jason A.</span> <span class="cit-name-surname">Greenbaum</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Zhen</span> <span class="cit-name-surname">Yan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Alessandro</span> <span class="cit-name-surname">Sette</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Bjoern</span> <span class="cit-name-surname">Peters</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Morten</span> <span class="cit-name-surname">Nielsen</span></span>. <span class="cit-pub-date">2018</span>. “<span class="cit-article-title">Improved Methods for Predicting Peptide Binding Affinity to MHC Class II Molecules</span>.” <abbr class="cit-jnl-abbrev">Immunology</abbr> <span class="cit-vol">154</span> (<span class="cit-issue">3</span>): <span class="cit-fpage">394</span>–<span class="cit-lpage">406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunology%26rft.volume%253D154%26rft.spage%253D394%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fimm.12889%26rft_id%253Dinfo%253Apmid%252F29315598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/imm.12889&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29315598&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.20.000794.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.20.000794v4.14" data-doi="10.1111/imm.12160"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jørgensen</span>,  <span class="cit-name-given-names">Kasper W.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Michael</span> <span class="cit-name-surname">Rasmussen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Søren</span> <span class="cit-name-surname">Buus</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Morten</span> <span class="cit-name-surname">Nielsen</span></span>. <span class="cit-pub-date">2014</span>. “<span class="cit-article-title">NetMHCstab - Predicting Stability of Peptide-MHC-I Complexes; Impacts for Cytotoxic T Lymphocyte Epitope Discovery</span>.” <abbr class="cit-jnl-abbrev">Immunology</abbr> <span class="cit-vol">141</span> (<span class="cit-issue">1</span>): <span class="cit-fpage">18</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunology%26rft.volume%253D141%26rft.spage%253D18%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fimm.12160%26rft_id%253Dinfo%253Apmid%252F23927693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/imm.12160&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23927693&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.20.000794.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000794v4.15" data-doi="10.4049/jimmunol.1700893"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jurtz</span>,  <span class="cit-name-given-names">Vanessa</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Sinu</span> <span class="cit-name-surname">Paul</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Massimo</span> <span class="cit-name-surname">Andreatta</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Paolo</span> <span class="cit-name-surname">Marcatili</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Bjoern</span> <span class="cit-name-surname">Peters</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Morten</span> <span class="cit-name-surname">Nielsen</span></span>. <span class="cit-pub-date">2017</span>. “<span class="cit-article-title">NetMHCpan-4.0: Improved Peptide-MHC Class I Interaction Predictions Integrating Eluted Ligand and Peptide Binding Affinity Data</span>.” <abbr class="cit-jnl-abbrev">Journal of Immunology</abbr> <span class="cit-vol">199</span> (<span class="cit-issue">9</span>): <span class="cit-fpage">3360</span>–<span class="cit-lpage">68</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BImmunology%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.1700893%26rft_id%253Dinfo%253Apmid%252F28978689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTA6IjE5OS85LzMzNjAiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wNi8yMDIwLjAzLjIwLjAwMDc5NC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000794v4.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Justesen</span>,  <span class="cit-name-given-names">Sune</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Mikkel</span> <span class="cit-name-surname">Harndahl</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Kasper</span> <span class="cit-name-surname">Lamberth</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Lise-Lotte B.</span> <span class="cit-name-surname">Nielsen</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Søren</span> <span class="cit-name-surname">Buus</span></span>. <span class="cit-pub-date">2009</span>. “<span class="cit-article-title">Functional Recombinant MHC Class II Molecules and High-Throughput Peptide-Binding Assays</span>.” <abbr class="cit-jnl-abbrev">Immunome Research</abbr> <span class="cit-vol">5</span> (<span class="cit-month">May</span>): <span class="cit-fpage">2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunome%2BResearch%26rft.volume%253D5%26rft.spage%253D2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.20.000794v4.17" data-doi="10.1007/s00251-011-0579-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Karosiene</span>,  <span class="cit-name-given-names">Edita</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Claus</span> <span class="cit-name-surname">Lundegaard</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Ole</span> <span class="cit-name-surname">Lund</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Morten</span> <span class="cit-name-surname">Nielsen</span></span>. <span class="cit-pub-date">2012</span>. “<span class="cit-article-title">NetMHCcons: A Consensus Method for the Major Histocompatibility Complex Class I Predictions</span>.” <abbr class="cit-jnl-abbrev">Immunogenetics</abbr> <span class="cit-vol">64</span> (<span class="cit-issue">3</span>): <span class="cit-fpage">177</span>–<span class="cit-lpage">86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunogenetics%26rft.volume%253D64%26rft.spage%253D177%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00251-011-0579-8%26rft_id%253Dinfo%253Apmid%252F22009319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00251-011-0579-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22009319&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.20.000794.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000300831600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000794v4.18" data-doi="10.1186/1471-2105-15-241"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">Yohan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">John</span> <span class="cit-name-surname">Sidney</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Søren</span> <span class="cit-name-surname">Buus</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Alessandro</span> <span class="cit-name-surname">Sette</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Morten</span> <span class="cit-name-surname">Nielsen</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Bjoern</span> <span class="cit-name-surname">Peters</span></span>. <span class="cit-pub-date">2014</span>. “<span class="cit-article-title">Dataset Size and Composition Impact the Reliability of Performance Benchmarks for Peptide-MHC Binding Predictions</span>.” <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">15</span> (<span class="cit-month">July</span>): <span class="cit-fpage">241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D15%26rft.spage%253D241%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-15-241%26rft_id%253Dinfo%253Apmid%252F25017736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-15-241&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25017736&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.20.000794.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.20.000794v4.19" data-doi="10.1186/1471-2105-10-394"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">Yohan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">John</span> <span class="cit-name-surname">Sidney</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Clemencia</span> <span class="cit-name-surname">Pinilla</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Alessandro</span> <span class="cit-name-surname">Sette</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Bjoern</span> <span class="cit-name-surname">Peters</span></span>. <span class="cit-pub-date">2009</span>. “<span class="cit-article-title">Derivation of an Amino Acid Similarity Matrix for Peptide: MHC Binding and Its Application as a Bayesian Prior</span>.” <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span> (<span class="cit-month">November</span>): <span class="cit-fpage">394</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DKim%26rft.auinit1%253DY.%26rft.volume%253D10%26rft.spage%253D394%26rft.epage%253D394%26rft.atitle%253DDerivation%2Bof%2Ban%2Bamino%2Bacid%2Bsimilarity%2Bmatrix%2Bfor%2Bpeptide%253A%2BMHC%2Bbinding%2Band%2Bits%2Bapplication%2Bas%2Ba%2BBayesian%2Bprior.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-394%26rft_id%253Dinfo%253Apmid%252F19948066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-394&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19948066&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.20.000794.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.20.000794v4.20" data-doi="10.1093/gigascience/giw015"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Geng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Dongli</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Zhang</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Si</span> <span class="cit-name-surname">Qiu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Wenhui</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Cheng-Chi</span> <span class="cit-name-surname">Chao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Naibo</span> <span class="cit-name-surname">Yang</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2017</span>. “<span class="cit-article-title">PSSMHCpan: A Novel PSSM-Based Software for Predicting Class I Peptide-HLA Binding Affinity</span>.” <abbr class="cit-jnl-abbrev">GigaScience</abbr> <span class="cit-vol">6</span> (<span class="cit-issue">5</span>): <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigaScience%26rft.volume%253D6%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgigascience%252Fgiw015%26rft_id%253Dinfo%253Apmid%252F29092041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gigascience/giw015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29092041&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.20.000794.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.20.000794v4.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mei</span>,  <span class="cit-name-given-names">Shutao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Fuyi</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"><span class="cit-name-surname">André</span>  <span class="cit-name-given-names">Leier</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Tatiana T.</span> <span class="cit-name-surname">Marquez-Lago</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Kailin</span> <span class="cit-name-surname">Giam</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Nathan P.</span> <span class="cit-name-surname">Croft</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Tatsuya</span> <span class="cit-name-surname">Akutsu</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2019</span>. “<span class="cit-article-title">A Comprehensive Review and Performance Evaluation of Bioinformatics Tools for HLA Class I Peptide-Binding Prediction</span>.” <span class="cit-source">Briefings in Bioinformatics</span>, <span class="cit-month">June</span>. <a href="https://doi.org/10.1093/bib/bbz051">https://doi.org/10.1093/bib/bbz051</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.20.000794v4.22" data-doi="10.1038/nbt1215"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Moutaftsi</span>,  <span class="cit-name-given-names">Magdalini</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Bjoern</span> <span class="cit-name-surname">Peters</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Valerie</span> <span class="cit-name-surname">Pasquetto</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">David C.</span> <span class="cit-name-surname">Tscharke</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">John</span> <span class="cit-name-surname">Sidney</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Huynh-Hoa</span> <span class="cit-name-surname">Bui</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Howard</span> <span class="cit-name-surname">Grey</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Alessandro</span> <span class="cit-name-surname">Sette</span></span>. <span class="cit-pub-date">2006</span>. “<span class="cit-article-title">A Consensus Epitope Prediction Approach Identifies the Breadth of Murine T(CD8+)-Cell Responses to Vaccinia Virus</span>.” <abbr class="cit-jnl-abbrev">Nature Biotechnology</abbr> <span class="cit-vol">24</span> (<span class="cit-issue">7</span>): <span class="cit-fpage">817</span>–<span class="cit-lpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DMoutaftsi%26rft.auinit1%253DM.%26rft.volume%253D24%26rft.issue%253D7%26rft.spage%253D817%26rft.epage%253D819%26rft.atitle%253DA%2Bconsensus%2Bepitope%2Bprediction%2Bapproach%2Bidentifies%2Bthe%2Bbreadth%2Bof%2Bmurine%2BT%2528CD8%252B%2529-cell%2Bresponses%2Bto%2Bvaccinia%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt1215%26rft_id%253Dinfo%253Apmid%252F16767078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt1215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16767078&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.20.000794.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239025100031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.20.000794v4.23" data-doi="10.1186/1471-2105-8-238"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>,  <span class="cit-name-given-names">Morten</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Claus</span> <span class="cit-name-surname">Lundegaard</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Ole</span> <span class="cit-name-surname">Lund</span></span>. <span class="cit-pub-date">2007</span>. “<span class="cit-article-title">Prediction of MHC Class II Binding Affinity Using SMM-Align, a Novel Stabilization Matrix Alignment Method</span>.” <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">8</span> (<span class="cit-month">July</span>): <span class="cit-fpage">238</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DNielsen%26rft.auinit1%253DM.%26rft.volume%253D8%26rft.spage%253D238%26rft.epage%253D238%26rft.atitle%253DPrediction%2Bof%2BMHC%2Bclass%2BII%2Bbinding%2Baffinity%2Busing%2BSMM-align%252C%2Ba%2Bnovel%2Bstabilization%2Bmatrix%2Balignment%2Bmethod.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-8-238%26rft_id%253Dinfo%253Apmid%252F17608956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-8-238&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17608956&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.20.000794.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.20.000794v4.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>,  <span class="cit-name-given-names">Morten</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Claus</span> <span class="cit-name-surname">Lundegaard</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Peder</span> <span class="cit-name-surname">Worning</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Sanne</span> <span class="cit-name-surname">Lise Lauemøller</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Kasper</span> <span class="cit-name-surname">Lamberth</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Søren</span> <span class="cit-name-surname">Buus</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Søren</span> <span class="cit-name-surname">Brunak</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Ole</span> <span class="cit-name-surname">Lund</span></span>. <span class="cit-pub-date">2003</span>. “<span class="cit-article-title">Reliable Prediction of T-Cell Epitopes Using Neural Networks with Novel Sequence Representations</span>.” <abbr class="cit-jnl-abbrev">Protein Science: A Publication of the Protein Society</abbr> <span class="cit-vol">12</span> (<span class="cit-issue">5</span>): <span class="cit-fpage">1007</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtein%2BScience%253A%2BA%2BPublication%2Bof%2Bthe%2BProtein%2BSociety%26rft.volume%253D12%26rft.spage%253D1007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.20.000794v4.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">O’Donnell</span>,  <span class="cit-name-given-names">Timothy J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Alex</span> <span class="cit-name-surname">Rubinsteyn</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Maria</span> <span class="cit-name-surname">Bonsack</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Angelika B.</span> <span class="cit-name-surname">Riemer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Uri</span> <span class="cit-name-surname">Laserson</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Jeff</span> <span class="cit-name-surname">Hammerbacher</span></span>. <span class="cit-pub-date">2018</span>. “<span class="cit-article-title">MHCflurry: Open-Source Class I MHC Binding Affinity Prediction</span>.” <abbr class="cit-jnl-abbrev">Cell Systems</abbr> <span class="cit-vol">7</span> (<span class="cit-issue">1</span>): <span class="cit-fpage">129</span>–<span class="cit-lpage">32</span>.e4.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BSystems%26rft.volume%253D7%26rft.spage%253D129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.20.000794v4.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Paszke</span>,  <span class="cit-name-given-names">Adam</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Sam</span> <span class="cit-name-surname">Gross</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Soumith</span> <span class="cit-name-surname">Chintala</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Gregory</span> <span class="cit-name-surname">Chanan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Edward</span> <span class="cit-name-surname">Yang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Zachary</span> <span class="cit-name-surname">Devito</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Zeming</span> <span class="cit-name-surname">Lin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Alban</span> <span class="cit-name-surname">Desmaison</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Luca</span> <span class="cit-name-surname">Antiga</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Adam</span> <span class="cit-name-surname">Lerer</span></span>. <span class="cit-pub-date">2017</span>. “<span class="cit-source">Automatic Differentiation in PyTorch</span>.” <a href="https://pdfs.semanticscholar.org/b36a/5bb1707bb9c70025294b3a310138aae8327a.pdf">https://pdfs.semanticscholar.org/b36a/5bb1707bb9c70025294b3a310138aae8327a.pdf</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.20.000794v4.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peters</span>,  <span class="cit-name-given-names">Bjoern</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Morten</span> <span class="cit-name-surname">Nielsen</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Alessandro</span> <span class="cit-name-surname">Sette</span></span>. <span class="cit-pub-date">2020</span>. “<span class="cit-article-title">T Cell Epitope Predictions</span>.” <span class="cit-source">Annual Review of Immunology</span>, <span class="cit-month">February</span>. <a href="https://doi.org/10.1146/annurev-immunol-082119-124838">https://doi.org/10.1146/annurev-immunol-082119-124838</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-web no-rev-xref" id="cit-2020.03.20.000794v4.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peters</span>,  <span class="cit-name-given-names">Bjoern</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Alessandro</span> <span class="cit-name-surname">Sette</span></span>. <span class="cit-pub-date">2005</span>. “<span class="cit-article-title">10.1186/1471-2105-6-132</span>.” <span class="cit-source">BMC Bioinformatics</span>. <a href="https://doi.org/10.1186/1471-2105-6-132">https://doi.org/10.1186/1471-2105-6-132</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000794v4.29" data-doi="10.4049/jimmunol.1900001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Qu</span>,  <span class="cit-name-given-names">Zehui</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Zibin</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Lizhen</span> <span class="cit-name-surname">Ma</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Xiaohui</span> <span class="cit-name-surname">Wei</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Lijie</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Ruiying</span> <span class="cit-name-surname">Liang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Geng</span> <span class="cit-name-surname">Meng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Nianzhi</span> <span class="cit-name-surname">Zhang</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Chun</span> <span class="cit-name-surname">Xia</span></span>. <span class="cit-pub-date">2019</span>. “<span class="cit-article-title">Structure and Peptidome of the Bat MHC Class I Molecule Reveal a Novel Mechanism Leading to High-Affinity Peptide Binding</span>.” <abbr class="cit-jnl-abbrev">Journal of Immunology</abbr> <span class="cit-vol">202</span> (<span class="cit-issue">12</span>): <span class="cit-fpage">3493</span>–<span class="cit-lpage">3506</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BImmunology%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.1900001%26rft_id%253Dinfo%253Apmid%252F31076531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTE6IjIwMi8xMi8zNDkzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDYvMjAyMC4wMy4yMC4wMDA3OTQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.20.000794v4.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rammensee</span>,  <span class="cit-name-given-names">Hans-Georg</span></span>. <span class="cit-pub-date">1995</span>. “<span class="cit-article-title">Chemistry of Peptides Associated with MHC Class I and Class II Molecules</span>.” <span class="cit-source">Current Opinion in Immunology</span>. <a href="https://doi.org/10.1016/0952-7915(95)80033-6">https://doi.org/10.1016/0952-7915(95)80033-6</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.20.000794v4.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rasmussen</span>,  <span class="cit-name-given-names">Michael</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Mikkel Nors</span> <span class="cit-name-surname">Harndahl</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Anne Bregnballe</span> <span class="cit-name-surname">Kristensen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Ida Kallehauge</span> <span class="cit-name-surname">Nielsen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Kasper W.</span> <span class="cit-name-surname">Jorgensen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Anette</span> <span class="cit-name-surname">Stryhn</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Morten</span> <span class="cit-name-surname">Nielsen</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Sören Buus</span> <span class="cit-name-surname">Buus</span></span>. <span class="cit-pub-date">2014</span>. “<span class="cit-source">Large Scale Analysis of Peptide - HLA-I Stability.” 2014</span>. <a href="https://www.iedb.org/reference/1028288">https://www.iedb.org/reference/1028288</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000794v4.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Saethang</span>,  <span class="cit-name-given-names">Thammakorn</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Osamu</span> <span class="cit-name-surname">Hirose</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Ingorn</span> <span class="cit-name-surname">Kimkong</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Vu Anh</span> <span class="cit-name-surname">Tran</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Xuan Tho</span> <span class="cit-name-surname">Dang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Lan Anh T.</span> <span class="cit-name-surname">Nguyen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Tu Kien T.</span> <span class="cit-name-surname">Le</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Mamoru</span> <span class="cit-name-surname">Kubo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Yoichi</span> <span class="cit-name-surname">Yamada</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Kenji</span> <span class="cit-name-surname">Satou</span></span>. <span class="cit-pub-date">2012</span>. “<span class="cit-article-title">EpicCapo: Epitope Prediction Using Combined Information of Amino Acid Pairwise Contact Potentials and HLA-Peptide Contact Site Information</span>.” <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">13</span> (<span class="cit-month">November</span>): <span class="cit-fpage">313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DSaethang%26rft.auinit1%253DT.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D313%26rft.epage%253D313%26rft.atitle%253DEpicCapo%253A%2Bepitope%2Bprediction%2Busing%2Bcombined%2Binformation%2Bof%2Bamino%2Bacid%2Bpairwise%2Bcontact%2Bpotentials%2Band%2BHLA-peptide%2Bcontact%2Bsite%2Binformation.%26rft_id%253Dinfo%253Apmid%252F23176036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23176036&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.20.000794.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.20.000794v4.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sarkizova</span>,  <span class="cit-name-given-names">Siranush</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Susan</span> <span class="cit-name-surname">Klaeger</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Phuong M.</span> <span class="cit-name-surname">Le</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Letitia W.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Giacomo</span> <span class="cit-name-surname">Oliveira</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Hasmik</span> <span class="cit-name-surname">Keshishian</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Christina R.</span> <span class="cit-name-surname">Hartigan</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2020</span>. “<span class="cit-article-title">A Large Peptidome Dataset Improves HLA Class I Epitope Prediction across Most of the Human Population</span>.” <abbr class="cit-jnl-abbrev">Nature Biotechnology</abbr> <span class="cit-vol">38</span> (<span class="cit-issue">2</span>): <span class="cit-fpage">199</span>–<span class="cit-lpage">209</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BBiotechnology%26rft.volume%253D38%26rft.spage%253D199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.20.000794v4.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sidney</span>,  <span class="cit-name-given-names">John</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Jason</span> <span class="cit-name-surname">Botten</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Benjamin</span> <span class="cit-name-surname">Neuman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Michael</span> <span class="cit-name-surname">Buchmeier</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Alessandro</span> <span class="cit-name-surname">Sette</span></span>. <span class="cit-pub-date">2006</span>. “<span class="cit-source">Epitopes Described in - Immune Epitope Database (IEDB).” 2006</span>. <a href="https://www.iedb.org/reference/1000425">https://www.iedb.org/reference/1000425</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.20.000794v4.35" data-doi="10.1038/9858"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sturniolo</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Bono</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ding</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Raddrizzani</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Tuereci</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Sahin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Braxenthaler</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">1999</span>. “<span class="cit-article-title">Generation of Tissue-Specific and Promiscuous HLA Ligand Databases Using DNA Microarrays and Virtual HLA Class II Matrices</span>.” <abbr class="cit-jnl-abbrev">Nature Biotechnology</abbr> <span class="cit-vol">17</span> (<span class="cit-issue">6</span>): <span class="cit-fpage">555</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DSturniolo%26rft.auinit1%253DT.%26rft.volume%253D17%26rft.issue%253D6%26rft.spage%253D555%26rft.epage%253D561%26rft.atitle%253DGeneration%2Bof%2Btissue-specific%2Band%2Bpromiscuous%2BHLA%2Bligand%2Bdatabases%2Busing%2BDNA%2Bmicroarrays%2Band%2Bvirtual%2BHLA%2Bclass%2BII%2Bmatrices.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F9858%26rft_id%253Dinfo%253Apmid%252F10385319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/9858&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10385319&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.20.000794.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000080716500026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000794v4.36" data-doi="10.1034/j.1399-0039.2002.590402.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sylvester-Hvid</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Kristensen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Blicher</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Ferré</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Lauemøller</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X. A.</span> <span class="cit-name-surname">Wolf</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Lamberth</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. H.</span> <span class="cit-name-surname">Nissen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. Ø.</span> <span class="cit-name-surname">Pedersen</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Buus</span></span>. <span class="cit-pub-date">2002</span>. “<span class="cit-article-title">Establishment of a Quantitative ELISA Capable of Determining Peptide - MHC Class I Interaction</span>.” <abbr class="cit-jnl-abbrev">Tissue Antigens</abbr> <span class="cit-vol">59</span> (<span class="cit-issue">4</span>): <span class="cit-fpage">251</span>–<span class="cit-lpage">58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTissue%2Bantigens%26rft.stitle%253DTissue%2BAntigens%26rft.aulast%253DSylvester-Hvid%26rft.auinit1%253DC.%26rft.volume%253D59%26rft.issue%253D4%26rft.spage%253D251%26rft.epage%253D258%26rft.atitle%253DEstablishment%2Bof%2Ba%2Bquantitative%2BELISA%2Bcapable%2Bof%2Bdetermining%2Bpeptide%2B-%2BMHC%2Bclass%2BI%2Binteraction.%26rft_id%253Dinfo%253Adoi%252F10.1034%252Fj.1399-0039.2002.590402.x%26rft_id%253Dinfo%253Apmid%252F12135423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1034/j.1399-0039.2002.590402.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12135423&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.20.000794.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176924200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000794v4.37" data-doi="10.1111/j.0001-2815.2004.00221.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sylvester-Hvid</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Nielsen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Lamberth</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Røder</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Justesen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Lundegaard</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Worning</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2004</span>. “<span class="cit-article-title">SARS CTL Vaccine Candidates; HLA Supertype-, Genome-Wide Scanning and Biochemical Validation</span>.” <abbr class="cit-jnl-abbrev">Tissue Antigens</abbr> <span class="cit-vol">63</span> (<span class="cit-issue">5</span>): <span class="cit-fpage">395</span>–<span class="cit-lpage">400</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTissue%2Bantigens%26rft.stitle%253DTissue%2BAntigens%26rft.aulast%253DSylvester-Hvid%26rft.auinit1%253DC.%26rft.volume%253D63%26rft.issue%253D5%26rft.spage%253D395%26rft.epage%253D400%26rft.atitle%253DSARS%2BCTL%2Bvaccine%2Bcandidates%253B%2BHLA%2Bsupertype-%252C%2Bgenome-wide%2Bscanning%2Band%2Bbiochemical%2Bvalidation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0001-2815.2004.00221.x%26rft_id%253Dinfo%253Apmid%252F15104671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0001-2815.2004.00221.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15104671&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.20.000794.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221620300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.20.000794v4.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Peng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">John</span> <span class="cit-name-surname">Sidney</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Courtney</span> <span class="cit-name-surname">Dow</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Bianca</span> <span class="cit-name-surname">Mothé</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Alessandro</span> <span class="cit-name-surname">Sette</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Bjoern</span> <span class="cit-name-surname">Peters</span></span>. <span class="cit-pub-date">2008</span>. “<span class="cit-article-title">A Systematic Assessment of MHC Class II Peptide Binding Predictions and Evaluation of a Consensus Approach</span>.” <abbr class="cit-jnl-abbrev">PLoS Computational Biology</abbr> <span class="cit-vol">4</span> (<span class="cit-issue">4</span>): <span class="cit-fpage">e1000048</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BComputational%2BBiology%26rft.volume%253D4%26rft.spage%253D1000048e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-book" id="cit-2020.03.20.000794v4.39"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">WHO</span>. <span class="cit-pub-date">2020a</span>. “<span class="cit-source">Novel Coronavirus (2019-nCoV) SITUATION REPORT - 1 21 JANUARY 2020</span>.” <span class="cit-publ-name">WHO</span>. <a href="https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf?sfvrsn=20a99c10_4">https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf?sfvrsn=20a99c10_4</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-book" id="cit-2020.03.20.000794v4.40"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">WHO</span>. <span class="cit-pub-date">2020b</span>. “<span class="cit-source">Coronavirus Disease 2019 (COVID-19) Situation Report – 46</span>.” <span class="cit-publ-name">WHO</span>. <a href="https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200307-sitrep-47-covid-19.pdf?sfvrsn=27c364a4_2">https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200307-sitrep-47-covid-19.pdf?sfvrsn=27c364a4_2</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-book" id="cit-2020.03.20.000794v4.41"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">WHO</span>. <span class="cit-pub-date">2020c</span>. “<span class="cit-source">Coronavirus Disease 2019 (COVID-19) Situation Report – 52</span>.” <span class="cit-publ-name">WHO</span>. <a href="https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200312-sitrep-52-covid-19.pdf?sfvrsn=e2bfc9c0_4">https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200312-sitrep-52-covid-19.pdf?sfvrsn=e2bfc9c0_4</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.20.000794v4.42" data-doi="10.1038/s41586-020-2008-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">Fan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Su</span> <span class="cit-name-surname">Zhao</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bin</span>  <span class="cit-name-given-names">Yu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Yan-Mei</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Wen</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Zhi-Gang</span> <span class="cit-name-surname">Song</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Yi</span> <span class="cit-name-surname">Hu</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2020</span>. “<span class="cit-article-title">A New Coronavirus Associated with Human Respiratory Disease in China</span>.” <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">579</span> (<span class="cit-issue">7798</span>): <span class="cit-fpage">265</span>–<span class="cit-lpage">69</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D579%26rft.spage%253D265%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2008-3%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-020-2008-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=http://www.n&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.20.000794.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.20.000794v4.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">Jingcheng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Wenzhe</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Jiucheng</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Binbin</span> <span class="cit-name-surname">Zhou</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Wenyi</span> <span class="cit-name-surname">Zhao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Zhixi</span> <span class="cit-name-surname">Su</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Xun</span> <span class="cit-name-surname">Gu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Jian</span> <span class="cit-name-surname">Wu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Zhan</span> <span class="cit-name-surname">Zhou</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Shuqing</span> <span class="cit-name-surname">Chen</span></span>. <span class="cit-pub-date">2019</span>. “<span class="cit-article-title">DeepHLApan: A Deep Learning Approach for Neoantigen Prediction Considering Both HLA-Peptide Binding and Immunogenicity</span>.” <abbr class="cit-jnl-abbrev">Frontiers in Immunology</abbr> <span class="cit-vol">10</span> (<span class="cit-month">November</span>): <span class="cit-fpage">2559</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2BImmunology%26rft.volume%253D10%26rft.spage%253D2559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.20.000794v4.44" data-doi="10.1093/bioinformatics/btp137"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Hao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Ole</span> <span class="cit-name-surname">Lund</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Morten</span> <span class="cit-name-surname">Nielsen</span></span>. <span class="cit-pub-date">2009</span>. “<span class="cit-article-title">The PickPocket Method for Predicting Binding Specificities for Receptors Based on Receptor Pocket Similarities: Application to MHC-Peptide Binding</span>.” <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span> (<span class="cit-issue">10</span>): <span class="cit-fpage">1293</span>–<span class="cit-lpage">99</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp137%26rft_id%253Dinfo%253Apmid%252F19297351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp137&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19297351&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.03.20.000794.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265950600013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.20.000794v4.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.20.000794v4" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.04.06.027391v1" title="Situation of antibiotic resistance in Bangladesh and its association with resistance genes for horizontal transfer" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.04.01.020693v2" title="Caenorhabditis elegans dauers vary recovery in response to bacteria from natural habitat" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;April 06, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.20.000794v4.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.03.20.000794v4.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.20.000794v4.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.20.000794v4" class="active" data-icon-position="" data-hide-link-title="0">COVID-19 Vaccine Candidates: Prediction and Validation of 174 SARS-CoV-2 Epitopes</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1231850" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-WBDp4vMf8NT8rjPxTtsCwQn_WABnm7wAaj4XHQOB2sc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1231850" id="node1231850--2" data-pisa="biorxiv;2020.03.20.000794v4" data-pisa-master="biorxiv;2020.03.20.000794" data-seqnum="1231850" data-apath="/biorxiv/early/2020/04/06/2020.03.20.000794.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">COVID-19 Vaccine Candidates: Prediction and Validation of 174 SARS-CoV-2 Epitopes</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Marek</span> <span class="nlm-surname">Prachar</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Sune</span> <span class="nlm-surname">Justesen</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Daniel Bisgaard</span> <span class="nlm-surname">Steen-Jensen</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Stephan</span> <span class="nlm-surname">Thorgrimsen</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Erik</span> <span class="nlm-surname">Jurgons</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Ole</span> <span class="nlm-surname">Winther</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Frederik Otzen</span> <span class="nlm-surname">Bagger</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.20.000794; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.20.000794 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.20.000794v4" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.20.000794v4&amp;title=COVID-19%20Vaccine%20Candidates%3A%20Prediction%20and%20Validation%20of%20174%20SARS-CoV-2%20Epitopes" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.20.000794v4&amp;title=COVID-19%20Vaccine%20Candidates%3A%20Prediction%20and%20Validation%20of%20174%20SARS-CoV-2%20Epitopes" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.20.000794v4&amp;text=COVID-19%20Vaccine%20Candidates%3A%20Prediction%20and%20Validation%20of%20174%20SARS-CoV-2%20Epitopes" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.20.000794v4&amp;title=COVID-19%20Vaccine%20Candidates%3A%20Prediction%20and%20Validation%20of%20174%20SARS-CoV-2%20Epitopes" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.20.000794v4&amp;t=COVID-19%20Vaccine%20Candidates%3A%20Prediction%20and%20Validation%20of%20174%20SARS-CoV-2%20Epitopes" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.20.000794v4&amp;title=COVID-19%20Vaccine%20Candidates%3A%20Prediction%20and%20Validation%20of%20174%20SARS-CoV-2%20Epitopes" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.20.000794v4&amp;title=COVID-19%20Vaccine%20Candidates%3A%20Prediction%20and%20Validation%20of%20174%20SARS-CoV-2%20Epitopes" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1231850" id="node1231850--3" data-pisa="biorxiv;2020.03.20.000794v4" data-pisa-master="biorxiv;2020.03.20.000794" data-seqnum="1231850" data-apath="/biorxiv/early/2020/04/06/2020.03.20.000794.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">COVID-19 Vaccine Candidates: Prediction and Validation of 174 SARS-CoV-2 Epitopes</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Marek</span> <span class="nlm-surname">Prachar</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Sune</span> <span class="nlm-surname">Justesen</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Daniel Bisgaard</span> <span class="nlm-surname">Steen-Jensen</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Stephan</span> <span class="nlm-surname">Thorgrimsen</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Erik</span> <span class="nlm-surname">Jurgons</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Ole</span> <span class="nlm-surname">Winther</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Frederik Otzen</span> <span class="nlm-surname">Bagger</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.20.000794; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.20.000794 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=COVID-19%20Vaccine%20Candidates%3A%20Prediction%20and%20Validation%20of%20174%20SARS-CoV-2%20Epitopes&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Prachar%2C+Marek&amp;rft.au=Justesen%2C+Sune&amp;rft.au=Steen-Jensen%2C+Daniel+Bisgaard&amp;rft.au=Thorgrimsen%2C+Stephan&amp;rft.au=Jurgons%2C+Erik&amp;rft.au=Winther%2C+Ole&amp;rft.au=Bagger%2C+Frederik+Otzen"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1231850/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1231850/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1231850/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1231850/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1231850/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1231850/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1231850/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1231850/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1231850/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1231850/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1231850/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1231850/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.20.000794v4&amp;count=horizontal&amp;via=&amp;text=COVID-19%20Vaccine%20Candidates%3A%20Prediction%20and%20Validation%20of%20174%20SARS-CoV-2%20Epitopes&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.20.000794v4" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.20.000794v4&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.20.000794v4" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/immunology" class="highlight" data-icon-position="" data-hide-link-title="0">Immunology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
