<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/02/27/2020.02.22.951178.atom" />
<meta name="HW.pisa" content="biorxiv;2020.02.22.951178v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Spike protein binding prediction with neutralizing antibodies of SARS-CoV-2" />
<meta name="DC.Identifier" content="10.1101/2020.02.22.951178" />
<meta name="DC.Date" content="2020-02-27" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Coronavirus disease 2019 (COVID-19) is a new emerging human infectious disease caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2, also previously known as 2019-nCoV), originated in Wuhan seafood and animal market, China. Since December 2019, more than 69,000 cases of COVID-19 have been confirmed in China and quickly spreads to other counties. Currently, researchers put their best efforts to identify effective drugs for COVID-19. The neutralizing antibody, which binds to viral capsid in a manner that inhibits cellular entry of virus and uncoating of the genome, is the specific defense against viral invaders. In this study, we investigate to identify neutralizing antibodies that can bind to SARS-CoV-2 Sipke (S) protein and interfere with the interaction between viral S protein and a host receptor by bioinformatic methods. The sequence analysis of S protein showed two major differences in the RBD region of the SARS-CoV-2 S protein compared to SARS-CoV and SARS-CoV related bat viruses (btSARS-CoV). The insertion regions were close to interacting residues with the human ACE2 receptor. Epitope analysis of neutralizing antibodies revealed that SARS-CoV neutralizing antibodies used conformational epitopes, whereas MERS-CoV neutralizing antibodies used a common linear epitope region, which contributes to form the β-sheet structure in MERS-CoV S protein and deleted in SARS-CoV-2 S protein. To identify effective neutralizing antibodies for SARS-CoV-2, the binding affinities of neutralizing antibodies with SARS-CoV-2 S protein were predicted and compared by antibody-antigen docking simulation. The result showed that CR3022 neutralizing antibody from human may have higher binding affinity with SARS-CoV-2 S protein than SARS-CoV S protein. We also found that F26G19 and D12 mouse antibodies could bind to SARS-CoV S protein with high affinity. Our findings provide crucial clues towards the development of antigen diagnosis, therapeutic antibody, and the vaccine against SARS-CoV-2." />
<meta name="DC.Contributor" content="Tamina Park" />
<meta name="DC.Contributor" content="Sang-Yeop Lee" />
<meta name="DC.Contributor" content="Seil Kim" />
<meta name="DC.Contributor" content="Mi Jeong Kim" />
<meta name="DC.Contributor" content="Hong Gi Kim" />
<meta name="DC.Contributor" content="Sangmi Jun" />
<meta name="DC.Contributor" content="Seung Il Kim" />
<meta name="DC.Contributor" content="Bum Tae Kim" />
<meta name="DC.Contributor" content="Edmond Changkyun Park" />
<meta name="DC.Contributor" content="Daeui Park" />
<meta name="article:published_time" content="2020-02-27" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Spike protein binding prediction with neutralizing antibodies of SARS-CoV-2" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Coronavirus disease 2019 (COVID-19) is a new emerging human infectious disease caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2, also previously known as 2019-nCoV), originated in Wuhan seafood and animal market, China. Since December 2019, more than 69,000 cases of COVID-19 have been confirmed in China and quickly spreads to other counties. Currently, researchers put their best efforts to identify effective drugs for COVID-19. The neutralizing antibody, which binds to viral capsid in a manner that inhibits cellular entry of virus and uncoating of the genome, is the specific defense against viral invaders. In this study, we investigate to identify neutralizing antibodies that can bind to SARS-CoV-2 Sipke (S) protein and interfere with the interaction between viral S protein and a host receptor by bioinformatic methods. The sequence analysis of S protein showed two major differences in the RBD region of the SARS-CoV-2 S protein compared to SARS-CoV and SARS-CoV related bat viruses (btSARS-CoV). The insertion regions were close to interacting residues with the human ACE2 receptor. Epitope analysis of neutralizing antibodies revealed that SARS-CoV neutralizing antibodies used conformational epitopes, whereas MERS-CoV neutralizing antibodies used a common linear epitope region, which contributes to form the β-sheet structure in MERS-CoV S protein and deleted in SARS-CoV-2 S protein. To identify effective neutralizing antibodies for SARS-CoV-2, the binding affinities of neutralizing antibodies with SARS-CoV-2 S protein were predicted and compared by antibody-antigen docking simulation. The result showed that CR3022 neutralizing antibody from human may have higher binding affinity with SARS-CoV-2 S protein than SARS-CoV S protein. We also found that F26G19 and D12 mouse antibodies could bind to SARS-CoV S protein with high affinity. Our findings provide crucial clues towards the development of antigen diagnosis, therapeutic antibody, and the vaccine against SARS-CoV-2.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.02.22.951178v1" />
<meta name="citation_id" content="2020.02.22.951178v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.22.951178v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.22.951178v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.22.951178v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.951178.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.02.22.951178" />
<meta name="citation_num_pages" content="22" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.02.22.951178" />
<meta name="citation_author" content="Tamina Park" />
<meta name="citation_author_institution" content="Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology" />
<meta name="citation_author_institution" content="Department of Predictive Toxicology, Korea Institute of Toxicology" />
<meta name="citation_author_institution" content="Department of Human and Environmental Toxicology, University of Science and Technology" />
<meta name="citation_author" content="Sang-Yeop Lee" />
<meta name="citation_author_institution" content="Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology" />
<meta name="citation_author_institution" content="Research Center for Bioconvergence Analysis, Korea Basic Science Institute" />
<meta name="citation_author" content="Seil Kim" />
<meta name="citation_author_institution" content="Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology" />
<meta name="citation_author_institution" content="Division of Chemical and Medical Metrology, Center for Bioanalysis, Korea Research Institute of Standards and Science" />
<meta name="citation_author_institution" content="Department of Bio-Analysis Science, University of Science &amp; Technology (UST)" />
<meta name="citation_author" content="Mi Jeong Kim" />
<meta name="citation_author_institution" content="Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology" />
<meta name="citation_author_institution" content="Research Center for Bioconvergence Analysis, Korea Basic Science Institute" />
<meta name="citation_author" content="Hong Gi Kim" />
<meta name="citation_author_institution" content="Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology" />
<meta name="citation_author" content="Sangmi Jun" />
<meta name="citation_author_institution" content="Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology" />
<meta name="citation_author_institution" content="Center for Research Equipment, Korea Basic Science Institute" />
<meta name="citation_author" content="Seung Il Kim" />
<meta name="citation_author_institution" content="Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology" />
<meta name="citation_author_institution" content="Research Center for Bioconvergence Analysis, Korea Basic Science Institute" />
<meta name="citation_author_institution" content="Department of Bio-Analysis Science, University of Science &amp; Technology (UST)" />
<meta name="citation_author" content="Bum Tae Kim" />
<meta name="citation_author_institution" content="Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology" />
<meta name="citation_author" content="Edmond Changkyun Park" />
<meta name="citation_author_institution" content="Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology" />
<meta name="citation_author_institution" content="Research Center for Bioconvergence Analysis, Korea Basic Science Institute" />
<meta name="citation_author_institution" content="Department of Bio-Analysis Science, University of Science &amp; Technology (UST)" />
<meta name="citation_author_email" content="daeui.park@kitox.re.kr" />
<meta name="citation_author_email" content="edpark@kbsi.re.kr" />
<meta name="citation_author" content="Daeui Park" />
<meta name="citation_author_institution" content="Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology" />
<meta name="citation_author_institution" content="Department of Predictive Toxicology, Korea Institute of Toxicology" />
<meta name="citation_author_institution" content="Department of Human and Environmental Toxicology, University of Science and Technology" />
<meta name="citation_author_email" content="daeui.park@kitox.re.kr" />
<meta name="citation_author_email" content="edpark@kbsi.re.kr" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9452-5849" />
<meta name="citation_reference" content="Li G, Fan Y, Lai Y, Han T, Li Z, Zhou P, Pan P, Wang W, Hu D, Liu X et al: Coronavirus infections and immune responses. J Med Virol 2020." />
<meta name="citation_reference" content="Summary of probable SARS cases with onset of illness form 1 November 2002 to 31 July 2003 [https://www.who.int/csr/sars/country/table2004_04_21/en/]" />
<meta name="citation_reference" content="MERS situation update. November 2019 [https://www.who.int/emergencies/mers-cov/en/]" />
<meta name="citation_reference" content="Novel Coronavirus (2019-nCoV) Situation Report-1 (21 January 2020) [https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf?sfvrsn=20a99c10_4]" />
<meta name="citation_reference" content="Fan Wu, Su Zhao, Bin Yu, Yan-Mei Chen, Wen Wang, Yi Hu, Zhi-Gang Song, Zhao-Wu Tao, Jun-Hua Tian, Yuan-Yuan Pei et al: Complete genome characterisation of a novel coronavirus associated with severe human respiratory disease in Wuhan, China. In: BioRxiv. 2020." />
<meta name="citation_reference" content="Alexander E. Gorbalenya, Susan C. Baker, Ralph S. Baric, Raoul J. de Groot, Christian Drosten, Anastasia A. Gulyaeva, Bart L. Haagmans, Chris Lauber, Andrey M Leontovich, Benjamin W. Neuman, et al: Severe acute respiratory syndrome-related coronavirus: The species and its viruses – a statement of the Coronavirus Study Group. In: BioRxiv. 2020." />
<meta name="citation_reference" content="K BA, Madhavan A, T RR, Thomas S, Nisha P: Short chain fatty acids enriched fermentation metabolites of soluble dietary fibre from Musa paradisiaca drives HT29 colon cancer cells to apoptosis. PLoS One 2019, 14(5):e0216604." />
<meta name="citation_reference" content="Huang C, Wang Y, Li X, Ren L, Zhao J, Hu Y, Zhang L, Fan G, Xu J, Gu X et al: Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China. Lancet 2020." />
<meta name="citation_reference" content="Chen N, Zhou M, Dong X, Qu J, Gong F, Han Y, Qiu Y, Wang J, Liu Y, Wei Y et al: Epidemiological and clinical characteristics of 99 cases of 2019 novel coronavirus pneumonia in Wuhan, China: a descriptive study. Lancet 2020." />
<meta name="citation_reference" content="Coronavirus disease 2019 (COVID-19) Situation Report – 24 (13 February 2020) [https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200213-sitrep-24-covid-19.pdf?sfvrsn=9a7406a4_4]" />
<meta name="citation_reference" content="Lu R, Zhao X, Li J, Niu P, Yang B, Wu H, Wang W, Song H, Huang B, Zhu N et al: Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding. Lancet 2020." />
<meta name="citation_reference" content="Li W, Moore MJ, Vasilieva N, Sui J, Wong SK, Berne MA, Somasundaran M, Sullivan JL, Luzuriaga K, Greenough TC et al: Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus. Nature 2003, 426(6965):450–454." />
<meta name="citation_reference" content="Zhou P, Yang XL, Wang XG, Hu B, Zhang L, Zhang W, Si HR, Zhu Y, Li B, Huang CL et al: A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature 2020." />
<meta name="citation_reference" content="Xiaolong Tian, Cheng Li, Ailing Huang, Shuai Xia, Sicong Lu, Zhengli Shi, Lu Lu, Shibo Jiang, Zhenlin Yang, Yanling Wu et al: Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody. In: BioRxiv. 2020." />
<meta name="citation_reference" content="Wrapp D, Wang N, Corbett KS, Goldsmith JA, Hsieh CL, Abiona O, Graham BS, McLellan JS: Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation. Science 2020." />
<meta name="citation_reference" content="Wan Y, Shang J, Graham R, Baric RS, Li F: Receptor recognition by novel coronavirus from Wuhan: An analysis based on decade-long structural studies of SARS. J Virol 2020." />
<meta name="citation_reference" content="Coughlin MM, Prabhakar BS: Neutralizing human monoclonal antibodies to severe acute respiratory syndrome coronavirus: target, mechanism of action, and therapeutic potential. Rev Med Virol 2012, 22(1):2–17." />
<meta name="citation_reference" content="Han HJ, Liu JW, Yu H, Yu XJ: Neutralizing Monoclonal Antibodies as Promising Therapeutics against Middle East Respiratory Syndrome Coronavirus Infection. Viruses 2018, 10(12)." />
<meta name="citation_reference" content="Shu Y, McCauley J: GISAID: Global initiative on sharing all influenza data - from vision to reality. Euro Surveill 2017, 22(13)." />
<meta name="citation_reference" content="Pickett BE, Greer DS, Zhang Y, Stewart L, Zhou L, Sun G, Gu Z, Kumar S, Zaremba S, Larsen CN et al: Virus pathogen database and analysis resource (ViPR): a comprehensive bioinformatics database and analysis resource for the coronavirus research community. Viruses 2012, 4(11):3209–3226." />
<meta name="citation_reference" content="Kumar S, Stecher G, Li M, Knyaz C, Tamura K: MEGA X: Molecular Evolutionary Genetics Analysis across Computing Platforms. Mol Biol Evol 2018, 35(6):1547–1549." />
<meta name="citation_reference" content="Saitou N, Nei M: The neighbor-joining method: a new method for reconstructing phylogenetic trees. Mol Biol Evol 1987, 4(4):406–425." />
<meta name="citation_reference" content="Felsenstein J: Evolutionary trees from DNA sequences: a maximum likelihood approach. J Mol Evol 1981, 17(6):368–376." />
<meta name="citation_reference" content="Fitch WM: Toward Defining the Course of Evolution: Minimum Change for a Specific Tree Topology. Systematic Zoology 1971, 20(4):406–416." />
<meta name="citation_reference" content="citation_title=Mammalian Protein Metabolism;citation_pages=21-132;citation_year=1969" />
<meta name="citation_reference" content="Felsenstein J: Confidence Limits on Phylogenies: An Approach Using the Bootstrap. Evolution 1985, 39(4):783–791." />
<meta name="citation_reference" content="Landau M, Mayrose I, Rosenberg Y, Glaser F, Martz E, Pupko T, Ben-Tal N: ConSurf 2005: the projection of evolutionary conservation scores of residues on protein structures. Nucleic Acids Res 2005, 33(Web Server issue):W299–302." />
<meta name="citation_reference" content="Biasini M, Bienert S, Waterhouse A, Arnold K, Studer G, Schmidt T, Kiefer F, Gallo Cassarino T, Bertoni M, Bordoli L et al: SWISS-MODEL: modelling protein tertiary and quaternary structure using evolutionary information. Nucleic Acids Res 2014, 42(Web Server issue):W252–258." />
<meta name="citation_reference" content="Meng EC, Pettersen EF, Couch GS, Huang CC, Ferrin TE: Tools for integrated sequence-structure analysis with UCSF Chimera. BMC Bioinformatics 2006, 7:339." />
<meta name="citation_reference" content="ter Meulen J, van den Brink EN, Poon LL, Marissen WE, Leung CS, Cox F, Cheung CY, Bakker AQ, Bogaards JA, van Deventer E et al: Human monoclonal antibody combination against SARS coronavirus: synergy and coverage of escape mutants. PLoS Med 2006, 3(7):e237." />
<meta name="citation_reference" content="Leaver-Fay A, Tyka M, Lewis SM, Lange OF, Thompson J, Jacak R, Kaufman K, Renfrew PD, Smith CA, Sheffler W et al: ROSETTA3: an object-oriented software suite for the simulation and design of macromolecules. Methods Enzymol 2011, 487:545–574." />
<meta name="citation_reference" content="Weitzner BD, Jeliazkov JR, Lyskov S, Marze N, Kuroda D, Frick R, Adolf-Bryfogle J, Biswas N, Dunbrack RL, Jr., Gray JJ: Modeling and docking of antibody structures with Rosetta. Nat Protoc 2017, 12(2):401–416." />
<meta name="citation_reference" content="Sircar A, Gray JJ: SnugDock: paratope structural optimization during antibody-antigen docking compensates for errors in antibody homology models. PLoS Comput Biol 2010, 6(1):e1000644." />
<meta name="citation_reference" content="Tengs T, Jonassen CM: Distribution and Evolutionary History of the Mobile Genetic Element s2m in Coronaviruses. Diseases 2016, 4(3)." />
<meta name="citation_reference" content="Wu K, Peng G, Wilken M, Geraghty RJ, Li F: Mechanisms of host receptor adaptation by severe acute respiratory syndrome coronavirus. J Biol Chem 2012, 287(12):8904–8911." />
<meta name="citation_reference" content="Ge XY, Li JL, Yang XL, Chmura AA, Zhu G, Epstein JH, Mazet JK, Hu B, Zhang W, Peng C et al: Isolation and characterization of a bat SARS-like coronavirus that uses the ACE2 receptor. Nature 2013, 503(7477):535–538." />
<meta name="citation_reference" content="Wang N, Shi X, Jiang L, Zhang S, Wang D, Tong P, Guo D, Fu L, Cui Y, Liu X et al: Structure of MERS-CoV spike receptor-binding domain complexed with human receptor DPP4. Cell Res 2013, 23(8):986–993." />
<meta name="citation_reference" content="Qiang Huang, Herrmann A: Fast assessment of human receptor-binding capability of 2019 novel coronavirus (2019-nCoV). In: BioRxiv. 2020." />
<meta name="citation_reference" content="Zhu Z, Chakraborti S, He Y, Roberts A, Sheahan T, Xiao X, Hensley LE, Prabakaran P, Rockx B, Sidorov IA et al: Potent cross-reactive neutralization of SARS coronavirus isolates by human monoclonal antibodies. Proc Natl Acad Sci U S A 2007, 104(29):12123–12128." />
<meta name="citation_reference" content="Hwang WC, Lin Y, Santelli E, Sui J, Jaroszewski L, Stec B, Farzan M, Marasco WA, Liddington RC: Structural basis of neutralization by a human anti-severe acute respiratory syndrome spike protein antibody, 80R. J Biol Chem 2006, 281(45):34610–34616." />
<meta name="citation_reference" content="Walls AC, Xiong X, Park YJ, Tortorici MA, Snijder J, Quispe J, Cameroni E, Gopal R, Dai M, Lanzavecchia A et al: Unexpected Receptor Functional Mimicry Elucidates Activation of Coronavirus Fusion. Cell 2019, 176(5):1026–1039 e1015." />
<meta name="citation_reference" content="Pak JE, Sharon C, Satkunarajah M, Auperin TC, Cameron CM, Kelvin DJ, Seetharaman J, Cochrane A, Plummer FA, Berry JD et al: Structural insights into immune recognition of the severe acute respiratory syndrome coronavirus S protein receptor binding domain. J Mol Biol 2009, 388(4):815–823." />
<meta name="citation_reference" content="Wang L, Shi W, Chappell JD, Joyce MG, Zhang Y, Kanekiyo M, Becker MM, van Doremalen N, Fischer R, Wang N et al: Importance of Neutralizing Monoclonal Antibodies Targeting Multiple Antigenic Sites on the Middle East Respiratory Syndrome Coronavirus Spike Glycoprotein To Avoid Neutralization Escape. J Virol 2018, 92(10)." />
<meta name="citation_reference" content="Yu X, Zhang S, Jiang L, Cui Y, Li D, Wang D, Wang N, Fu L, Shi X, Li Z et al: Structural basis for the neutralization of MERS-CoV by a human monoclonal antibody MERS-27. Sci Rep 2015, 5:13133." />
<meta name="citation_reference" content="Ying T, Prabakaran P, Du L, Shi W, Feng Y, Wang Y, Wang L, Li W, Jiang S, Dimitrov DS et al: Junctional and allele-specific residues are critical for MERS-CoV neutralization by an exceptionally potent germline-like antibody. Nat Commun 2015, 6:8223." />
<meta name="citation_reference" content="Chen Z, Bao L, Chen C, Zou T, Xue Y, Li F, Lv Q, Gu S, Gao X, Cui S et al: Human Neutralizing Monoclonal Antibody Inhibition of Middle East Respiratory Syndrome Coronavirus Replication in the Common Marmoset. J Infect Dis 2017, 215(12):1807–1815." />
<meta name="citation_reference" content="Li Y, Wan Y, Liu P, Zhao J, Lu G, Qi J, Wang Q, Lu X, Wu Y, Liu W et al: A humanized neutralizing antibody against MERS-CoV targeting the receptor-binding domain of the spike protein. Cell Res 2015, 25(11):1237–1249." />
<meta name="citation_reference" content="Wang L, Shi W, Joyce MG, Modjarrad K, Zhang Y, Leung K, Lees CR, Zhou T, Yassine HM, Kanekiyo M et al: Evaluation of candidate vaccine approaches for MERS-CoV. Nat Commun 2015, 6:7712." />
<meta name="twitter:title" content="Spike protein binding prediction with neutralizing antibodies of SARS-CoV-2" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Coronavirus disease 2019 (COVID-19) is a new emerging human infectious disease caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2, also previously known as 2019-nCoV), originated in Wuhan seafood and animal market, China. Since December 2019, more than 69,000 cases of COVID-19 have been confirmed in China and quickly spreads to other counties. Currently, researchers put their best efforts to identify effective drugs for COVID-19. The neutralizing antibody, which binds to viral capsid in a manner that inhibits cellular entry of virus and uncoating of the genome, is the specific defense against viral invaders. In this study, we investigate to identify neutralizing antibodies that can bind to SARS-CoV-2 Sipke (S) protein and interfere with the interaction between viral S protein and a host receptor by bioinformatic methods. The sequence analysis of S protein showed two major differences in the RBD region of the SARS-CoV-2 S protein compared to SARS-CoV and SARS-CoV related bat viruses (btSARS-CoV). The insertion regions were close to interacting residues with the human ACE2 receptor. Epitope analysis of neutralizing antibodies revealed that SARS-CoV neutralizing antibodies used conformational epitopes, whereas MERS-CoV neutralizing antibodies used a common linear epitope region, which contributes to form the β-sheet structure in MERS-CoV S protein and deleted in SARS-CoV-2 S protein. To identify effective neutralizing antibodies for SARS-CoV-2, the binding affinities of neutralizing antibodies with SARS-CoV-2 S protein were predicted and compared by antibody-antigen docking simulation. The result showed that CR3022 neutralizing antibody from human may have higher binding affinity with SARS-CoV-2 S protein than SARS-CoV S protein. We also found that F26G19 and D12 mouse antibodies could bind to SARS-CoV S protein with high affinity. Our findings provide crucial clues towards the development of antigen diagnosis, therapeutic antibody, and the vaccine against SARS-CoV-2." />
<meta name="og-title" property="og:title" content="Spike protein binding prediction with neutralizing antibodies of SARS-CoV-2" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.22.951178v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Coronavirus disease 2019 (COVID-19) is a new emerging human infectious disease caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2, also previously known as 2019-nCoV), originated in Wuhan seafood and animal market, China. Since December 2019, more than 69,000 cases of COVID-19 have been confirmed in China and quickly spreads to other counties. Currently, researchers put their best efforts to identify effective drugs for COVID-19. The neutralizing antibody, which binds to viral capsid in a manner that inhibits cellular entry of virus and uncoating of the genome, is the specific defense against viral invaders. In this study, we investigate to identify neutralizing antibodies that can bind to SARS-CoV-2 Sipke (S) protein and interfere with the interaction between viral S protein and a host receptor by bioinformatic methods. The sequence analysis of S protein showed two major differences in the RBD region of the SARS-CoV-2 S protein compared to SARS-CoV and SARS-CoV related bat viruses (btSARS-CoV). The insertion regions were close to interacting residues with the human ACE2 receptor. Epitope analysis of neutralizing antibodies revealed that SARS-CoV neutralizing antibodies used conformational epitopes, whereas MERS-CoV neutralizing antibodies used a common linear epitope region, which contributes to form the β-sheet structure in MERS-CoV S protein and deleted in SARS-CoV-2 S protein. To identify effective neutralizing antibodies for SARS-CoV-2, the binding affinities of neutralizing antibodies with SARS-CoV-2 S protein were predicted and compared by antibody-antigen docking simulation. The result showed that CR3022 neutralizing antibody from human may have higher binding affinity with SARS-CoV-2 S protein than SARS-CoV S protein. We also found that F26G19 and D12 mouse antibodies could bind to SARS-CoV S protein with high affinity. Our findings provide crucial clues towards the development of antigen diagnosis, therapeutic antibody, and the vaccine against SARS-CoV-2." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-02-27" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.02.22.951178v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.02.22.951178v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.02.22.951178v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.02.22.951178v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1169701" />
    <title>Spike protein binding prediction with neutralizing antibodies of SARS-CoV-2 | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ZPDN9uaiYIVNo6NzWj5l3nPFbX5Xi9W4KNQtJr_QuBo","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1169701","apath":"\/biorxiv\/early\/2020\/02\/27\/2020.02.22.951178.atom","pisa":"biorxiv;2020.02.22.951178v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.02.22.951178v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.02.22.951178v1","title":"Spike protein binding prediction with neutralizing antibodies of SARS-CoV-2","identifier":"node\/1169701"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022PEfjK9ojSet9l2o7G5xrCRAlvnQFA0Lj9xTBTh8xib0khQRm7Kp\\\\\\\/wIs41kD2i1L4DP\\\\\\\/SzNTbWPcfg6LzQ9T9Zh5kftFroWuosb3Oow6PotWPyJEwjKV8MfZMeZUo2IxiFbNvXvELu6mVZo9VhS74Wij2FSkLBL2rKAgpoXVfN4tj+KaPrTEDs5JKAABOirCzCZZvIr+3\\\\\\\/Mj29MShy4GgFd1tFnxY+V8RhmMRTpvdtgQaMDxUH0VLVU\\\\\\\/CAorzY2dCEWzarv1e5j\\\\\\\/XgsNBD4l2xt9Hv732py\\\\\\\/XqDekp+xPIraa0EOKfLvd\\\\\\\/5M4\\\\\\\/bDiDmjvdLaXaes9r9+3kvsjcYOFMeDrOmHneehAu\\\\\\\/tUJtdX4Sn2kfTUnZuHQFJpOX8QSj\\\\\\\/7EQxS5c33EnBhDyKP2TrbJ\\\\\\\/bBT26e8duS45I5K\\\\\\\/ftnGRM8PKzAlwUaCkYub4VxNMVifYoKhOnQ16hRM2DbifUI5hl\\\\\\\/GR5wVPtSGnAExuxllHWUECkS+2fNXKa17c46cMsI2b+4O29v8mfPBlYANK2wq61rLHs2c4XBXH1i8oQMdGYEBFygAvKX4QTIwDqZrUGYuqml4KboSPlS5MJfpqqPcbwfXo0dp7tw6\\\\\\\/aad5jgN5cNVaKYPOVCGlI2X83P1IP1mnPWcsJL4eV7UjkTfeRoaSZB2FNIkmXwo+r28Z+Bk0kCrJSN2eTGmMmZqAzZv+I4Ma7ATGe6yZYZplcMLqDQbZsGkg9GCgBg8tQUW6b9jpxc3DxEvpVB0YLYELpRYoIZCo2fQsPsAnOTUx9ZHeZbHkSs3WF1fGLB5B+xJzgH0Q+rzB1AaXmY4phHeYRGV\\\\\\\/wrmy40RySLHxiYoDlE8OC3cArKI7LpCkws5ViJQV+0hAEBwcIiFPnW2TQHEnlgibJTgg3ANHyeJtaIN6wZ5njOhxF7NIzSeVY80tO5yqDjJtwLpvOW4+aeoxH9qgL0YaZHHalgTuCGtQicXRgbknnjbwtUqXKDsnoW3A3lgqEGyZmolXL\\\\\\\/DpadZH+8HYfwlKJCgGl3ajjLozrdjecn9mZRpP4zxS2sescyzWPW7ZfDT6Zy+jh3dcsozh248shDm1zpRnctEY2f69bRwrbi0C54IbHGSTfWkz4k+uLRXmLQkgjyQfqtk2fEs1b\\\\\\\/kkrZ\\\\\\\/O9WMic8nct6ELFCVxspLqbbXI0KlDsKBanqvbHnQYdU7dqWLoU6\\\\\\\/SkfSQ4kS62b5YOe3ZJ0nSs6knVNvN8YCEJ9hM4uuvU2Kk1XajLHED2D2sEzRe+9bXmsiP2\\\\\\\/mm0rSVbR5qryqyKqdpIZw5omqqexXfmUhnccWN0TZxKcsfGr5cxG3OegkMb3m1aDc8di0SK\\\u0022,\\\u0022iv\\\u0022:\\\u0022XA7FTskuhlZjRDDvZl9Hkw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022fef86c6fc7bb9dd220f5a4599f970ac44cac11c67f9d64b20bd790b31645a85a\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.02.22.951178v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1169701 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.02.22.951178v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-RzCXa-tG5-SgdqPusMLGioLjT-VFpmUPPL47neMIeMY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1169701" id="node1169701" data-pisa="biorxiv;2020.02.22.951178v1" data-pisa-master="biorxiv;2020.02.22.951178" data-apath="/biorxiv/early/2020/02/27/2020.02.22.951178.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Spike protein binding prediction with neutralizing antibodies of SARS-CoV-2</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Tamina</span> <span class="nlm-surname">Park</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Sang-Yeop</span> <span class="nlm-surname">Lee</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Seil</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Mi Jeong</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Hong Gi</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Sangmi</span> <span class="nlm-surname">Jun</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Seung Il</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Bum Tae</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Edmond Changkyun</span> <span class="nlm-surname">Park</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-9452-5849" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Daeui</span> <span class="nlm-surname">Park</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.02.22.951178 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1169701" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Tamina Park </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology</span>, Daejeon 34114, <span class='nlm-country'>Republic of Korea</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Predictive Toxicology, Korea Institute of Toxicology</span>, Daejeon 34114, <span class='nlm-country'>Republic of Korea</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Human and Environmental Toxicology, University of Science and Technology</span>, Daejeon 34113, <span class='nlm-country'>Republic of Korea</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tamina%2BPark%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Park%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATamina%2BPark%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sang-Yeop Lee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology</span>, Daejeon 34114, <span class='nlm-country'>Republic of Korea</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Research Center for Bioconvergence Analysis, Korea Basic Science Institute</span>, Cheongju 28119, <span class='nlm-country'>Republic of Korea</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sang-Yeop%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lee%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASang-Yeop%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Seil Kim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology</span>, Daejeon 34114, <span class='nlm-country'>Republic of Korea</span></div><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Division of Chemical and Medical Metrology, Center for Bioanalysis, Korea Research Institute of Standards and Science</span>, Daejeon 34113, <span class='nlm-country'>Republic of Korea</span></div><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Department of Bio-Analysis Science, University of Science &#38; Technology (UST)</span>, Daejeon 34113, <span class='nlm-country'>Republic of Korea</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Seil%2BKim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kim%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASeil%2BKim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Mi Jeong Kim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology</span>, Daejeon 34114, <span class='nlm-country'>Republic of Korea</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Research Center for Bioconvergence Analysis, Korea Basic Science Institute</span>, Cheongju 28119, <span class='nlm-country'>Republic of Korea</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mi%2BJeong%2BKim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kim%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMi%2BJeong%2BKim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Hong Gi Kim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology</span>, Daejeon 34114, <span class='nlm-country'>Republic of Korea</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hong%2BGi%2BKim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kim%20HG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHong%2BGi%2BKim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sangmi Jun </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology</span>, Daejeon 34114, <span class='nlm-country'>Republic of Korea</span></div><div class='author-affiliation'><span class='nlm-sup'>7</span><span class='nlm-institution'>Center for Research Equipment, Korea Basic Science Institute</span>, Cheongju 28119, <span class='nlm-country'>Republic of Korea</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sangmi%2BJun%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jun%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASangmi%2BJun%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Seung Il Kim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology</span>, Daejeon 34114, <span class='nlm-country'>Republic of Korea</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Research Center for Bioconvergence Analysis, Korea Basic Science Institute</span>, Cheongju 28119, <span class='nlm-country'>Republic of Korea</span></div><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Department of Bio-Analysis Science, University of Science &#38; Technology (UST)</span>, Daejeon 34113, <span class='nlm-country'>Republic of Korea</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Seung%2BIl%2BKim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kim%20SI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASeung%2BIl%2BKim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Bum Tae Kim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology</span>, Daejeon 34114, <span class='nlm-country'>Republic of Korea</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bum%2BTae%2BKim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kim%20BT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABum%2BTae%2BKim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Edmond Changkyun Park </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology</span>, Daejeon 34114, <span class='nlm-country'>Republic of Korea</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Research Center for Bioconvergence Analysis, Korea Basic Science Institute</span>, Cheongju 28119, <span class='nlm-country'>Republic of Korea</span></div><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Department of Bio-Analysis Science, University of Science &#38; Technology (UST)</span>, Daejeon 34113, <span class='nlm-country'>Republic of Korea</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edmond%2BChangkyun%2BPark%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Park%20EC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEdmond%2BChangkyun%2BPark%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:daeui.park@kitox.re.kr" class="" data-icon-position="" data-hide-link-title="0">daeui.park@kitox.re.kr</a>
<a href="mailto:edpark@kbsi.re.kr" class="" data-icon-position="" data-hide-link-title="0">edpark@kbsi.re.kr</a></span></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Daeui Park </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Center for Convergent Research of Emerging Virus Infection, Korea Research Institute of Chemical Technology</span>, Daejeon 34114, <span class='nlm-country'>Republic of Korea</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Predictive Toxicology, Korea Institute of Toxicology</span>, Daejeon 34114, <span class='nlm-country'>Republic of Korea</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Human and Environmental Toxicology, University of Science and Technology</span>, Daejeon 34113, <span class='nlm-country'>Republic of Korea</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daeui%2BPark%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Park%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaeui%2BPark%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9452-5849" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daeui Park</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:daeui.park@kitox.re.kr" class="" data-icon-position="" data-hide-link-title="0">daeui.park@kitox.re.kr</a>
<a href="mailto:edpark@kbsi.re.kr" class="" data-icon-position="" data-hide-link-title="0">edpark@kbsi.re.kr</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.02.22.951178v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1169701" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1169701/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.22.951178v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1169701" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1169701/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.22.951178v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1169701" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1169701/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.22.951178v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1169701" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1169701/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.02.22.951178v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1169701" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1169701/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.02.22.951178v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1169701" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1169701/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">Coronavirus disease 2019 (COVID-19) is a new emerging human infectious disease caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2, also previously known as 2019-nCoV), originated in Wuhan seafood and animal market, China. Since December 2019, more than 69,000 cases of COVID-19 have been confirmed in China and quickly spreads to other counties. Currently, researchers put their best efforts to identify effective drugs for COVID-19. The neutralizing antibody, which binds to viral capsid in a manner that inhibits cellular entry of virus and uncoating of the genome, is the specific defense against viral invaders. In this study, we investigate to identify neutralizing antibodies that can bind to SARS-CoV-2 Sipke (S) protein and interfere with the interaction between viral S protein and a host receptor by bioinformatic methods. The sequence analysis of S protein showed two major differences in the RBD region of the SARS-CoV-2 S protein compared to SARS-CoV and SARS-CoV related bat viruses (btSARS-CoV). The insertion regions were close to interacting residues with the human ACE2 receptor. Epitope analysis of neutralizing antibodies revealed that SARS-CoV neutralizing antibodies used conformational epitopes, whereas MERS-CoV neutralizing antibodies used a common linear epitope region, which contributes to form the β-sheet structure in MERS-CoV S protein and deleted in SARS-CoV-2 S protein. To identify effective neutralizing antibodies for SARS-CoV-2, the binding affinities of neutralizing antibodies with SARS-CoV-2 S protein were predicted and compared by antibody-antigen docking simulation. The result showed that CR3022 neutralizing antibody from human may have higher binding affinity with SARS-CoV-2 S protein than SARS-CoV S protein. We also found that F26G19 and D12 mouse antibodies could bind to SARS-CoV S protein with high affinity. Our findings provide crucial clues towards the development of antigen diagnosis, therapeutic antibody, and the vaccine against SARS-CoV-2.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-4"><em>Coronaviridae</em> is a family of enveloped viruses which have a single strand, positive-stranded RNA genome and classified into four genera: ɑ, β, γ, and δ. Coronavirus (CoV) has been identified in human and animals including bats, camels, zpigs, cats, and mice. The viruses usually cause mild to moderate upper-respiratory tract illnesses in human [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>]. Two of betacoronaviruses, severe acute respiratory syndrome (SARS) coronavirus (SARS-CoV) and Middle East respiratory syndrome (MERS) coronavirus (MERS-CoV), caused severe epidemics during the last two decades. SARS-CoV emerged in November 2002 in Guangdong province, China and affected 29 countries. The epidemic of SARS-CoV resulted in 8,096 human infections and 774 deaths (9.6% fatality rate) by July 2003 [<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]. MERS-CoV was first reported in Saudi Arabia in September 2012 and spread to 28 countries. The epidemic of MERS-CoV resulted in 2,494 human infections and 858 deaths (34.4% fatality rate) by November 2019 [<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>].</p><p id="p-5">Coronavirus disease 2019 (COVID-19) is newly emerging human infectious disease caused by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2, also previous known as 2019-nCoV) originated in Wuhan seafood and animal market. In December 2019, a series of pneumonia cases of unknown cause have been reported in Wuhan, Hubei province, China [<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>]. Later, on January 7, a novel CoV was identified from the bronchoalveolar lavage fluid of a patient [<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>], and named SARS-CoV-2 by International Committee on Taxonomy of Viruses [<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>]. The early COVID-19 patients show symptoms of severe acute respiratory infection, such as fever, cough, sore throat, nasal congestion, headache, muscle pain or malaise, and severe patients develop to acute respiratory distress syndrome, sepsis or septic shock [<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>–<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>]. As of February 16, 2020, more than 69,200 cases of COVID-19 have been confirmed in China and quickly spreads to other counties [<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>].</p><p id="p-6">The SARS-CoV-2 is a member of betacoronavirus and shows 79% and 50% sequence identity with SARS-CoV and MERS-CoV, respectively. Phylogenetic analysis revealed that SARS-CoV-2 is most similar (88% sequence identity) to SARS-like CoVs previously collected from bats in China [<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>]. Although the viral pathogenesis of SARS-CoV-2 is unknown, most recent reports suggest that SARS-CoV-2 may use angiotensin-converting enzyme II (ACE2) as a cellular entry receptor. ACE2 is a well known host cell receptor for SARS-CoV [<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>]. Shi and colleges showed that SARS-CoV-2 uses ACE2 as a cellular entry receptor but not other CoV receptors, aminopeptidase N (APN) and dipeptidyl peptidase 4 (DPP4) [<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>]. Ying and colleges showed the receptor-binding domain (RBD) of SARS-CoV-2 spike glycoprotein (S protein) interacts with ACE2 [<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>]. McLaellen and colleges showed that ACE2 binds to SARS-CoV-2 S protein with much higher affinity than to SARS-CoV S protein [<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>]. In addition, bioinformatic analysis proposed binding structure of RBD of S protein (S-RBD) and ACE2 [<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>]. Thus, it is of great interest to identify neutralizing antibodies that can interact with SARS-CoV-2 S-RBD and interfere with the binding between viral S protein and host receptor ACE2.</p><p id="p-7">After the severe epidemic events of SARS and MERS, researchers have been made great efforts to discover neutralizing antibodies for CoVs [<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>]. The neutralizing antibodies for CoVs mainly targeted to S-RBD. S protein of SARS-CoV-2 shows 76.2% and 34.1% amino acid sequence identity to those of SARS-CoV and MERS-CoV, respectively. Therefore, the neutralizing antibodies of SARS-CoV and MERS-CoV S proteins may have a possibility to interact with SARS-CoV-2 S protein and show similar viral neutralization effect. In the present study, we employed a antibody-antigen docking approach to predict the interaction between SARS-CoV-2 S-RBD and previously reported neutralization antibodies for SARS-CoV and MERS-CoV.</p></div><div class="section" id="sec-2"><h2 class="">Methods</h2><div id="sec-3" class="subsection"><h3>Phylogenetic analysis of SARS-CoV-2 S protein</h3><p id="p-8">To comparing of S gene containing S protein among SARS-CoV-2, SARS-CoV, and MERS-CoV strains, the nucleotide sequences of S gene were retrieved from GISAID [<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>] and ViPR [<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>]. The S genes of SARS-CoV-2 were retrieved from initially sequenced 62 genomes of SARS-CoV-2 strains. After removal of identical S gene sequences, 16 genes of S protein were used in the study. The sequence from SARS-CoV-2 Wuhan-Hu-1 (Genbank <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=MN908947.3&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.22.951178.atom">MN908947.3</a> or GISAID EPI ISL 402125) was used as a representative sequence of SARS-CoV-2 strains throughout this study. The closely related strains of SARS-CoV-2 were selected from preliminary and extensive phylogenetic analaysis of SARS-CoV related strains including btSARS-CoV, SARS-CoV and a MERS-CoV strain. More detail information of the sequences used in this study were listed in Supplementary Table S1. The sequence alignments and phylogenetic analysis were done using MEGA X [<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>]. The nucleotide sequence were codon aligned using ClustalW with default parameters and the phylogenetic tree was inferred using neighbor-joining [<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>], maximum-likelihood [<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>], and maximum-parsimony methods [<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>]. The distance matrix was calculated based on the Jukes-Cantor methods [<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>]. The bootstrap values of the phylogenetic tree were derived from 1,000 replicates [<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>].</p></div><div id="sec-4" class="subsection"><h3>Conservation score and epitope mapping of SARS-CoV-2 S protein</h3><p id="p-9">The conservation score of amino acid positions on S protein in SARS-CoV-2 was calculated by ConSurf program [<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>]. The multiple sequence alignment of SARS-CoV-2 strain Wuhan-Hu-1and 12 related strains (Supplementary Table S1) was used as an input for ConSurf. In ConSurf, conservation scores and confidence intervals for the conservation scores were calculated using the empirical Bayesian method. The scores were normalized using the number of inputted sequences. Also, the highest score of ConSurf program means the most conserved position among sequences. We additionally checked the epitope positions on the SARS-CoV-2 S protein based on the known epitope information of 11 neutralizing antibodies developing for SARS-CoV and MERS-CoV. Each information of epitope positions was acquired from literatures (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1173947/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1173947/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1173947" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-10" class="first-child">Neutralizing antibodies and their epitopes analyzed in the study.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Structure of SARS-CoV-2 S protein</h3><p id="p-11">S-RBD protein structure was used cryo-EM structure (Protein Data Bank ID: 6VSB) [<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>]. To predict the missing region of cryo-EM structure in SARS-CoV-2 S-RBD, we performed homology modeling based on known the three dimensional structure of SARS-CoV (PDB ID: 6NB7) using SWISS-MODEL (<a href="https://swissmodel.expasy.org/">https://swissmodel.expasy.org/</a>) [<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>]. Then, the best homology models were selected according to Qualitative Model Energy ANalysis (QMEAN) statistical parameter. The structures were visualized with UCSF’s Chimera (<a href="https://www.cgl.ucsf.edu/chimera/">https://www.cgl.ucsf.edu/chimera/</a>).</p></div><div id="sec-6" class="subsection"><h3>Neutralizing antibody candidates</h3><p id="p-12">As neutralizing antibody candidates of SARS-CoV-2, the five antibodies against SARS-CoV and the six antibodies to prevent MERS-CoV were selected in the study (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The complex structure of RBD and ten neutralizing antibodies was retrieved from PDB. The complex structures were superimposed to the RBD structure of SARS-CoV-2 which were built by homology modeling. The procedures were performed that the RBD structures of SARS-CoV2 and SARS-CoV were aligned by pairwise sequence alignment. And then the structures were superimposed according to those pairwise alignments using MatchMaker program [<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>]. Finally, we suceesfully predicted the complex structures of neutralizaing antibody candidates and RBD of SARS-CoV-2. About the antibody such as CR3022 [<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>] that the structure was not revealed, we performed the antibody strcture modeling with Rosetta program [<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>].</p></div><div id="sec-7" class="subsection"><h3>Antibody-antigen docking simulation</h3><p id="p-13">Docking simlutation between the RBD of SARS-CoV-2 and certain SARS-CoV and MERS-CoV antibodies were implemented with Rosetta antibody-antigen docking protocols [<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>]. Rosetta SnugDock program can refine homology models with the flexible and uncertain region, because the program simulates most of conformation space available to antibody paratopes [<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>]. With the complex structures of RBD and antibody candidates, all-atom relax protocol, docking prepack protocol, and antibody-antigen docking simulation were carried out to calculate the free energy of low-energy binding conformations. The distiribution of docking scores displayed as funnel plots using interface RMD (interface RMS) versus the binding score (dG binding) between antibody and antigen (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4</a>). The binding score was used Rosetta’s docking interface score (based on the Talaris2013 force field) to rank the complexes. Rosetta interface score is defined as Isc = <em>E</em>bound − <em>E</em>unbound, where <em>E</em>bound is the score of the bound complex and <em>E</em>unbound is the sum of the scores of the individual protein partners in isolation. In addition, 1000 independent docking runs were performed to generate the antibody-antigen models. To predict possible neutralizing antibody candidates of SARS-CoV-2, the docking results were compared between interface binding scores of SARS-CoV-2 S-RBD (homology modeling) and interface binding scores of SARS-CoV or MERS-CoV S-RBD (crystal structure) with 11 antibodies for SARS-CoV and MERS-CoV have been developed. The statistical significance was tested using student’s <em>t-test</em>.</p></div></div><div class="section" id="sec-8"><h2 class="">Results and Discussion</h2><div id="sec-9" class="subsection"><h3>Phylogenetic analysis and amino acid variation of S protein</h3><p id="p-14">The phylogenetic tree showed that the protein gene sequences were clearly clustered into three groups; SARS-CoV-2 related, SARS-CoV related and HKU3 related groups (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1A</a>). SARS-CoV and SARS-CoV-2 groups formed a rigid monophyletic group with their own closest bat SARS-CoV related strains, respectively. The result suggested that these two human-pathogenic CoV strains were derived from common ancestral bat CoV. The sequence alignments showed that there were insertions and deletion during the divergence among the strains (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1B</a> and <a id="xref-fig-1-3" class="xref-fig" href="#F1">1C</a>). Various deletions were observed in SARS-CoV related group. The NTD region (position 71-77, GTNGTKR) of S protein in the strain Wuhan-Hu-1 was mostly conserved in SARS-CoV-2 group but not in SARS-CoV related group. The NTD region was also conserved in other btSARS-CoV strains but the sequence similarity was low.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.951178/F1/graphic-2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Neighbor-joining tree based on spike protein gene sequences showing the relationship between 2019-nCoV and related SARS-related viruses. The bootstrap value greater than 50 are shown at the branch nodes. The filled circles indicated that the corresponding nodes are conserved in all tree-drawing methods. MERS-CoV KNIH002/05/2015 is used as the outgroup. Bar, 0.1 substitutions per nucleotide position. (B) The upper panel is structure of spike gene of SARS-CoV-2. The middle panel is the result of the Sim plot analysis. The sequence similarity of NTD domain region was the lowest among spike genes. Nevertheless, SARS-CoV-2_bat-RaTG13 had highly conserved in NTD domain than other SARS-CoVs. On the other hand, in the RBD region, not only SARS-CoV but also SARS-CoV-2_bat-RaTG13 showed low sequence similarity. The lower panel indicates the conservation score of the protein sequences of 27 SARS-CoV species. (C) Intersertional regions in SARS-CoV-2 S protein. The amino acid position 445-449 (VGGNY) and 470-486 (TEIYQAGSTPCNGVEGF) were conserved in SARS-CoV-2 related group except bat-SL-CoVZC45 strains (red color), and the corresponding sequences in SARS-related groups were &#x2018;STGNY&#x2019; and &#x2018;NVPFSPDGKPCTPPAL&#x2019; (blue color)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1592864756" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) Neighbor-joining tree based on spike protein gene sequences showing the relationship between 2019-nCoV and related SARS-related viruses. The bootstrap value greater than 50 are shown at the branch nodes. The filled circles indicated that the corresponding nodes are conserved in all tree-drawing methods. MERS-CoV KNIH002/05/2015 is used as the outgroup. Bar, 0.1 substitutions per nucleotide position. (B) The upper panel is structure of spike gene of SARS-CoV-2. The middle panel is the result of the Sim plot analysis. The sequence similarity of NTD domain region was the lowest among spike genes. Nevertheless, SARS-CoV-2_bat-RaTG13 had highly conserved in NTD domain than other SARS-CoVs. On the other hand, in the RBD region, not only SARS-CoV but also SARS-CoV-2_bat-RaTG13 showed low sequence similarity. The lower panel indicates the conservation score of the protein sequences of 27 SARS-CoV species. (C) Intersertional regions in SARS-CoV-2 S protein. The amino acid position 445-449 (VGGNY) and 470-486 (TEIYQAGSTPCNGVEGF) were conserved in SARS-CoV-2 related group except bat-SL-CoVZC45 strains (red color), and the corresponding sequences in SARS-related groups were &#x2018;STGNY&#x2019; and &#x2018;NVPFSPDGKPCTPPAL&#x2019; (blue color).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.951178/F1/graphic-2.medium.gif" width="440" height="336"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.951178/F1/graphic-2.medium.gif" width="440" height="336"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.951178/F1/graphic-2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.951178/F1/graphic-2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.951178/F1/graphic-3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Neighbor-joining tree based on spike protein gene sequences showing the relationship between 2019-nCoV and related SARS-related viruses. The bootstrap value greater than 50 are shown at the branch nodes. The filled circles indicated that the corresponding nodes are conserved in all tree-drawing methods. MERS-CoV KNIH002/05/2015 is used as the outgroup. Bar, 0.1 substitutions per nucleotide position. (B) The upper panel is structure of spike gene of SARS-CoV-2. The middle panel is the result of the Sim plot analysis. The sequence similarity of NTD domain region was the lowest among spike genes. Nevertheless, SARS-CoV-2_bat-RaTG13 had highly conserved in NTD domain than other SARS-CoVs. On the other hand, in the RBD region, not only SARS-CoV but also SARS-CoV-2_bat-RaTG13 showed low sequence similarity. The lower panel indicates the conservation score of the protein sequences of 27 SARS-CoV species. (C) Intersertional regions in SARS-CoV-2 S protein. The amino acid position 445-449 (VGGNY) and 470-486 (TEIYQAGSTPCNGVEGF) were conserved in SARS-CoV-2 related group except bat-SL-CoVZC45 strains (red color), and the corresponding sequences in SARS-related groups were &#x2018;STGNY&#x2019; and &#x2018;NVPFSPDGKPCTPPAL&#x2019; (blue color)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1592864756" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) Neighbor-joining tree based on spike protein gene sequences showing the relationship between 2019-nCoV and related SARS-related viruses. The bootstrap value greater than 50 are shown at the branch nodes. The filled circles indicated that the corresponding nodes are conserved in all tree-drawing methods. MERS-CoV KNIH002/05/2015 is used as the outgroup. Bar, 0.1 substitutions per nucleotide position. (B) The upper panel is structure of spike gene of SARS-CoV-2. The middle panel is the result of the Sim plot analysis. The sequence similarity of NTD domain region was the lowest among spike genes. Nevertheless, SARS-CoV-2_bat-RaTG13 had highly conserved in NTD domain than other SARS-CoVs. On the other hand, in the RBD region, not only SARS-CoV but also SARS-CoV-2_bat-RaTG13 showed low sequence similarity. The lower panel indicates the conservation score of the protein sequences of 27 SARS-CoV species. (C) Intersertional regions in SARS-CoV-2 S protein. The amino acid position 445-449 (VGGNY) and 470-486 (TEIYQAGSTPCNGVEGF) were conserved in SARS-CoV-2 related group except bat-SL-CoVZC45 strains (red color), and the corresponding sequences in SARS-related groups were &#x2018;STGNY&#x2019; and &#x2018;NVPFSPDGKPCTPPAL&#x2019; (blue color).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.951178/F1/graphic-3.medium.gif" width="348" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.951178/F1/graphic-3.medium.gif" width="348" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.951178/F1/graphic-3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.951178/F1/graphic-3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-15" class="first-child">(A) Neighbor-joining tree based on spike protein gene sequences showing the relationship between 2019-nCoV and related SARS-related viruses. The bootstrap value greater than 50 are shown at the branch nodes. The filled circles indicated that the corresponding nodes are conserved in all tree-drawing methods. MERS-CoV KNIH002/05/2015 is used as the outgroup. Bar, 0.1 substitutions per nucleotide position. (B) The upper panel is structure of spike gene of SARS-CoV-2. The middle panel is the result of the Sim plot analysis. The sequence similarity of NTD domain region was the lowest among spike genes. Nevertheless, SARS-CoV-2_bat-RaTG13 had highly conserved in NTD domain than other SARS-CoVs. On the other hand, in the RBD region, not only SARS-CoV but also SARS-CoV-2_bat-RaTG13 showed low sequence similarity. The lower panel indicates the conservation score of the protein sequences of 27 SARS-CoV species. (C) Intersertional regions in SARS-CoV-2 S protein. The amino acid position 445-449 (VGGNY) and 470-486 (TEIYQAGSTPCNGVEGF) were conserved in SARS-CoV-2 related group except bat-SL-CoVZC45 strains (red color), and the corresponding sequences in SARS-related groups were ‘STGNY’ and ‘NVPFSPDGKPCTPPAL’ (blue color).</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">Interestingly, human pathogenic strains and their closest strains had two insertion sequences in RBD region of SARS-CoV-2. The amino acid position 445-449 (VGGNY) and 470-486 (TEIYQAGSTPCNGVEGF) were conserved in SARS-CoV-2 related group except bat-SL-CoVZC45 strain and the corresponding sequences in SARS-related groups were ‘STGNY’ and ‘NVPFSPDGKPCTPPAL’ (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1C</a>). The results could not give clear answers that these insertion sequences had directly diverged from the common ancestor of SARS-CoV and SARS-CoV-2 or that the sequences in SARS-CoV-2 were derived from SARS-CoV related group by mobile genetic elements [<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>]. Nevertheless, the insertion sequences have several antibody epitope regions (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>) and the two key residues (amino acid position 455 and 486) interacting with human ACE2 [<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>] which is used as a cellular receptor of btSARS-CoV strain WIV1 [<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>]. This suggested that these sequence were might be related with human susceptibility and virulence.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.951178/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Epitope map and conversation score of the RBD region in SARS-CoV and MERS-CoV S protein. The blue bar chart shows the conservation score. The numbers are the amino acid positions in the S protein. Color boxes indicate binding epitopes for SARS-CoV (red color) and MERS-CoV (green color) antibodies. The amino acid residues in red color indicate binding epitopes for host receptors." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1592864756" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Epitope map and conversation score of the RBD region in SARS-CoV and MERS-CoV S protein. The blue bar chart shows the conservation score. The numbers are the amino acid positions in the S protein. Color boxes indicate binding epitopes for SARS-CoV (red color) and MERS-CoV (green color) antibodies. The amino acid residues in red color indicate binding epitopes for host receptors.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.951178/F2.medium.gif" width="440" height="234"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.951178/F2.medium.gif" width="440" height="234"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.951178/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.951178/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-17" class="first-child">Epitope map and conversation score of the RBD region in SARS-CoV and MERS-CoV S protein. The blue bar chart shows the conservation score. The numbers are the amino acid positions in the S protein. Color boxes indicate binding epitopes for SARS-CoV (red color) and MERS-CoV (green color) antibodies. The amino acid residues in red color indicate binding epitopes for host receptors.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Identification and analysis of the neutralizing antibody epitopes</h3><p id="p-18">Previously, numbers of neutralizing antibodies for SARS-CoV and MERS-CoV have been developed [<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>]. To suggest possible SARS-CoV-2 neutralizing antibodies, monoclonal antibodies were selected from the literature and PDB (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Epitope map showed that the antibody-binding residues of S protein are located within RDB region (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2</a>). Four SARS-CoV neutralizing antibodies had the epitopes about 5 to 14 residues (total 34 residues, average 9.5 residues) of S-RBD and six MERS-CoV neutralizing antibodies bound to 22 to 33 (total 52 residues, average 25 residues) residues. Distribution of the antibody-binding residues indicates that SARS-CoV neutralizing antibodies might be bind to mainly conformational epitopes of S-RDB, whereas MERS-CoV neutralizing antibodies bound to linear epitopes of S-RBD (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2</a>). Interestingly, the major linear epitope region (EDGDYYRKQL) for MERS-CoV neutralizing antibodies was specific insertion of MERS-CoV S protein. MERS-CoV neutralizing antibodies interacted with three receptor binding residues (E536, D537, D539) in the linear epitope region, which results in the neutralizing activity of antibodies by directly interferes the binding between S protein and dipeptidyl peptidase 4 of human . In addition, the difference of binding aspect of neutralizing antibodies might be caused by the difference of subdomain structure of receptor binding motif (RBM). The RBM of SARS-CoV S protein is made of mainly coiled structure with two short β-sheets, whereas the RBM of MERS-CoV S protein consists of four long β-sheets [<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>]. Sequence alignment revealed that RBD of SARS-CoV-2 was more similar to that of SARS-CoV than MERS-CoV (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2</a>). Therefore, this suggested that SARS-CoV neutralizing antibodies could be effective for SARS-CoV-2.</p></div><div id="sec-11" class="subsection"><h3>S-RBD structure modeling and superimposition of neutralizing antibodies</h3><p id="p-19">Human infection of SARS-CoV-2 was firstly reported in Wuhan, Hubei province, China last December [<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>]. Previous studies have reported several results for the interaction between S protein of SARS-CoV-2 and ACE2 as a receptor [<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>, <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>]. However, any interaction of SARS-CoV-2 S protein with developed neutralizing antibodies for SARS-CoV and MERS-CoV has not been reported yet. The structure of SARS-CoV-2 S protein was used S protein which revealed by cryo electron microscopy structure. Subsequently, the missing region of SARS-CoV-2 S-RBD region comprising of 181 amino acids were built from SARS-CoV S proteins (PDB ID: 6NB7) which were good structural templates (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3</a> box). In the S-RBD structure, we also displayed experimentally defined epitope information based on position specific aligment with SARS-CoV or MERS-CoV antibody binding epitopes.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.951178/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The three dimensional structure of SARS-CoV-2 S protein. (A) The three dimensional structure of RBD domain in S protein was colored gray. The RBD structure shown from various sides. On the surface representation, the SARS-CoV antibodies, MERS-CoV antibodies, and both SARS-CoV and MERS-CoV antibodies binding epitopes are colored red, green, and purple, respectively. The sky blue color represents SARS-CoV-2 S protein and the RBD domain were highlighted with orange color. The red box indicates the RBD region, which is containing SARS-CoV or MERS-CoV antibody binding epitope. The predicted RBD structure of SARS-CoV-2 S protein in complex with five SARS-CoV antibodies (B) and six MERS-CoV antibodies (C). The complex structure was predicted by integrating the previously known complex structures of SARS-CoV or MERS-CoV with antibodies using the superimposition of structures. Each colored structure in surface representation indicates antibody labeled with the same color. More detail information about antibodies were described in Table 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1592864756" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The three dimensional structure of SARS-CoV-2 S protein. (A) The three dimensional structure of RBD domain in S protein was colored gray. The RBD structure shown from various sides. On the surface representation, the SARS-CoV antibodies, MERS-CoV antibodies, and both SARS-CoV and MERS-CoV antibodies binding epitopes are colored red, green, and purple, respectively. The sky blue color represents SARS-CoV-2 S protein and the RBD domain were highlighted with orange color. The red box indicates the RBD region, which is containing SARS-CoV or MERS-CoV antibody binding epitope. The predicted RBD structure of SARS-CoV-2 S protein in complex with five SARS-CoV antibodies (B) and six MERS-CoV antibodies (C). The complex structure was predicted by integrating the previously known complex structures of SARS-CoV or MERS-CoV with antibodies using the superimposition of structures. Each colored structure in surface representation indicates antibody labeled with the same color. More detail information about antibodies were described in Table 1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.951178/F3.medium.gif" width="440" height="300"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.951178/F3.medium.gif" width="440" height="300"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.951178/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.951178/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <p id="p-20" class="first-child">The three dimensional structure of SARS-CoV-2 S protein. (A) The three dimensional structure of RBD domain in S protein was colored gray. The RBD structure shown from various sides. On the surface representation, the SARS-CoV antibodies, MERS-CoV antibodies, and both SARS-CoV and MERS-CoV antibodies binding epitopes are colored red, green, and purple, respectively. The sky blue color represents SARS-CoV-2 S protein and the RBD domain were highlighted with orange color. The red box indicates the RBD region, which is containing SARS-CoV or MERS-CoV antibody binding epitope. The predicted RBD structure of SARS-CoV-2 S protein in complex with five SARS-CoV antibodies (B) and six MERS-CoV antibodies (C). The complex structure was predicted by integrating the previously known complex structures of SARS-CoV or MERS-CoV with antibodies using the superimposition of structures. Each colored structure in surface representation indicates antibody labeled with the same color. More detail information about antibodies were described in <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">To visualize the overall antibody binding region to SARS-CoV-2, we superimposed the predicted structure of SARS-CoV-2 RBD protein at the X-ray crystal structure of known antibody-antigen complex from SARS and MERS (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3</a>). The structures of five antibodies including m396, 80R, F26G19, S230, and CR3022 developing to prevent SARS-CoV were aligned on SARS-CoV-2 S-RBD successfully (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3A</a>). The six MERS-CoV antibodies such as MERS-27, CDC2-C2, m336, 4C2, D12, and MCA1 were also aligned on SARS-CoV-2 S-RBD (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3B</a>). Because the X-ray crystal structure of CR3022 was not revealed, the optimized structure was predicted using antibody homology modeling by 1000 structures generated using Rosetta program. As a results, two SARS-CoV antibodies including CR3022 (−13.91 dG score) and F26G19 (−15.98 dG score) and MERS-CoV D12 (−14.01 dG score) antibody had higher binding score than other antibodies with SARS-CoV-2 S-RBD region. However, various MERS-CoV antibodies did not match SARS-CoV-2 because MERS-CoV antibodies were interacted with the outter regon of S-RBD which was located in major linear epitope region (EDGDYYRKQL) (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig. 2</a>).</p></div><div id="sec-12" class="subsection"><h3>Comparison of Antibody-RBD protein binding interaction</h3><p id="p-22">Based on antibody-antigen docking simulation, we calculated the binding scores between 11 antibodies and S-RBD structures. The antibody-antigen docking simulation generated not only the crystal structures of SARS-CoV and MERS-CoV S-RBD proteins, but also the high-quaility homology models with SARS-CoV-2 S-RBD. To suggest S-RBD binding antibody, antibody-RBD docking comparisons were performed using the mean value of caculated scores from the generated models. The mean scores of the docking simulation are shown in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1173955/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1173955/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1173955" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2.</span> <p id="p-23" class="first-child">Antibody-antigen docking score and experimental affinity</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">Among the SARS-CoV antibodies, only CR3022 showed that the binding affinity of SARS-CoV-2 was higher than SARS-CoV. In addition, the docking score distribution of CR3022 was significantly changed between SARS-CoV-2 and SARS-CoV-2 (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4</a>). For the CR3022 antibody, the mean score of binding affinity was increased from −11.21 dG score (SARS-CoV, crystal structure) to −13.91 dG score (SARS-CoV-2, cryo-EM structure) with a <em>p-value</em> of 0.00367. The binding affinity of all antibody-antibgen docking was tested using 1000 generated structures. Interestingly, the CR3022 was experimentally performed for the binding effect of SARS-CoV-2 S-RBD [<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">14</a>]. The researchers found that the CR3022 had the binding effect anainst SARS-CoV-2 S-RBD and that m396 and m336 antibodies did not bind to SARS-CoV-2 S-RBD. The researchers also reported that the binding affinity of CR3022 was increased to 6.28nM with SARS-CoV-2 from 0.125nM with SARS-CoV, The results of the docking simulation were consistent with the evidence although more research was needed to prevent effects, including an experiment using live viruses.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.951178/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The distiribution of docking scores between antibody and antigen using interface RMD (interface RMS) versus the binding energy (dG binding). The docking simulation were performed by 1000 generated antibody-antigen structures. The statistical significance was tested using student&#x2019;s t-test. (A) Mean score of docking simulation between CR3022 and SARS-CoV S-RBD was &#x2212;11.21 dG score, and mean score of CR-3022 and SARS-CoV2 S-RBD was &#x2212;13.61 dG score (p-value = 0.00367)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1592864756" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The distiribution of docking scores between antibody and antigen using interface RMD (interface RMS) versus the binding energy (dG binding). The docking simulation were performed by 1000 generated antibody-antigen structures. The statistical significance was tested using student&#x2019;s &lt;em&gt;t-test&lt;/em&gt;. (A) Mean score of docking simulation between CR3022 and SARS-CoV S-RBD was &#x2212;11.21 dG score, and mean score of CR-3022 and SARS-CoV2 S-RBD was &#x2212;13.61 dG score (&lt;em&gt;p-value&lt;/em&gt; = 0.00367).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.951178/F4.medium.gif" width="440" height="175"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.951178/F4.medium.gif" width="440" height="175"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.951178/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/27/2020.02.22.951178/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <p id="p-25" class="first-child">The distiribution of docking scores between antibody and antigen using interface RMD (interface RMS) versus the binding energy (dG binding). The docking simulation were performed by 1000 generated antibody-antigen structures. The statistical significance was tested using student’s <em>t-test</em>. (A) Mean score of docking simulation between CR3022 and SARS-CoV S-RBD was −11.21 dG score, and mean score of CR-3022 and SARS-CoV2 S-RBD was −13.61 dG score (<em>p-value</em> = 0.00367).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-13"><h2 class="">Conclusions</h2><p id="p-26">The fact that CoVs similar to SARS in Chinese bats is most identical to SARS-CoV-2 suggests that SARS-CoV may have been originated from a common ancestral bat CoV. Comparing the sequences among the three groups, various deletions were observed in the SARS-CoV related group. Especially, amino acid positions 71-77 (GTNGTKR) in the NTD region of the S protein, 445-449 (VGGNY) and 470-486 (TEIYQAGSTPCNGVEGF) were noteworthy. The regions were highly conserved in SARS-CoV-2, unlike other SARS-CoVs.</p><p id="p-27">Among the neutralizing antibodies for SARS-CoV and MERS-CoV, CR3022 was predicted to have better binding affinity to the S-RBD region of SARS-CoV-2 than other antibodies. The comparison of antibody binding region between SARS-CoV-2 and other coronaviruses, such as SARS-CoV and MERS-CoV, was conducted to apply the suitable diagnostic or therapeutic antibodies and vaccines that are mimetics of extremely infectious SARS-CoV-2.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgment</h2><p id="p-28">We thanks to Prof. Jason S. McLellan for providing cyro-EM structure of of SARS-CoV-2 S protein, Korea Institute of Science and Technology Information’s 5th supercomputer NURION, and bio bigdata center of Clinomics Inc.. This work was supported by National Research Council of Science and Technology grant by the Ministry of Science and ICT (Grant No. CRC-16-01-KRICT).</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">Coronavirus infections and immune responses</span>. <abbr class="cit-jnl-abbrev">J Med Virol</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-web" id="cit-2020.02.22.951178v1.2"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Summary of probable SARS cases with onset of illness form 1 November 2002 to 31 July 2003</span> [<a href="https://www.who.int/csr/sars/country/table2004_04_21/en/">https://www.who.int/csr/sars/country/table2004_04_21/en/</a>]</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-web" id="cit-2020.02.22.951178v1.3"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">MERS situation update. November 2019</span> [<a href="https://www.who.int/emergencies/mers-cov/en/">https://www.who.int/emergencies/mers-cov/en/</a>]</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-web" id="cit-2020.02.22.951178v1.4"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Novel Coronavirus (2019-nCoV) Situation Report-1 (21 January 2020)</span> [<a href="https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf?sfvrsn=20a99c10_4">https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf?sfvrsn=20a99c10_4</a>]</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.5"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Fan</span> <span class="cit-name-surname">Wu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Su</span> <span class="cit-name-surname">Zhao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Bin</span> <span class="cit-name-surname">Yu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Yan-Mei</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Wen</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Yi</span> <span class="cit-name-surname">Hu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Zhi-Gang</span> <span class="cit-name-surname">Song</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Zhao-Wu</span> <span class="cit-name-surname">Tao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Jun-Hua</span> <span class="cit-name-surname">Tian</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Yuan-Yuan</span> <span class="cit-name-surname">Pei</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">Complete genome characterisation of a novel coronavirus associated with severe human respiratory disease in Wuhan, China</span>. In: <abbr class="cit-jnl-abbrev">BioRxiv.</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.6"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Alexander E.</span> <span class="cit-name-surname">Gorbalenya</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Susan C.</span> <span class="cit-name-surname">Baker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Ralph S.</span> <span class="cit-name-surname">Baric</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Raoul J.</span> <span class="cit-name-surname">de Groot</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Christian</span> <span class="cit-name-surname">Drosten</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Anastasia A.</span> <span class="cit-name-surname">Gulyaeva</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Bart L.</span> <span class="cit-name-surname">Haagmans</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Chris</span> <span class="cit-name-surname">Lauber</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Andrey M</span> <span class="cit-name-surname">Leontovich</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Benjamin W.</span> <span class="cit-name-surname">Neuman</span></span>, <span class="cit-etal">et al</span>: <span class="cit-article-title">Severe acute respiratory syndrome-related coronavirus: The species and its viruses – a statement of the Coronavirus Study Group</span>. In: <abbr class="cit-jnl-abbrev">BioRxiv.</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.7"><div class="cit-metadata"><cite>K BA, <span class="cit-auth"><span class="cit-name-surname">Madhavan</span>  <span class="cit-name-given-names">A</span></span>, T RR, <span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nisha</span>  <span class="cit-name-given-names">P</span></span>: <span class="cit-article-title">Short chain fatty acids enriched fermentation metabolites of soluble dietary fibre from Musa paradisiaca drives HT29 colon cancer cells to apoptosis</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-pub-date">2019</span>, <span class="cit-vol">14</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">e0216604</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D14%26rft.spage%253De0216604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">8.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.22.951178v1.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">X</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gong</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qiu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">Y</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">Epidemiological and clinical characteristics of 99 cases of 2019 novel coronavirus pneumonia in Wuhan, China: a descriptive study</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-web" id="cit-2020.02.22.951178v1.10"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Coronavirus disease 2019 (COVID-19) Situation Report – 24 (13 February 2020)</span> [<a href="https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200213-sitrep-24-covid-19.pdf?sfvrsn=9a7406a4_4">https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200213-sitrep-24-covid-19.pdf?sfvrsn=9a7406a4_4</a>]</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">N</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.12" data-doi="10.1038/nature02145"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vasilieva</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sui</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">SK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berne</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Somasundaran</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sullivan</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luzuriaga</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Greenough</span>  <span class="cit-name-given-names">TC</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-pub-date">2003</span>, <span class="cit-vol">426</span>(<span class="cit-issue">6965</span>):<span class="cit-fpage">450</span>–<span class="cit-lpage">454</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLi%26rft.auinit1%253DW.%26rft.volume%253D426%26rft.issue%253D6965%26rft.spage%253D450%26rft.epage%253D454%26rft.atitle%253DAngiotensin-converting%2Benzyme%2B2%2Bis%2Ba%2Bfunctional%2Breceptor%2Bfor%2Bthe%2BSARS%2Bcoronavirus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02145%26rft_id%253Dinfo%253Apmid%252F14647384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02145&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14647384&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.22.951178.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">XL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">XG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Si</span>  <span class="cit-name-given-names">HR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">CL</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.14"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Xiaolong</span> <span class="cit-name-surname">Tian</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Cheng</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Ailing</span> <span class="cit-name-surname">Huang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Shuai</span> <span class="cit-name-surname">Xia</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Sicong</span> <span class="cit-name-surname">Lu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Zhengli</span> <span class="cit-name-surname">Shi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Lu</span> <span class="cit-name-surname">Lu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Shibo</span> <span class="cit-name-surname">Jiang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Zhenlin</span> <span class="cit-name-surname">Yang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Yanling</span> <span class="cit-name-surname">Wu</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody</span>. In: <abbr class="cit-jnl-abbrev">BioRxiv.</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wrapp</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Corbett</span>  <span class="cit-name-given-names">KS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goldsmith</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hsieh</span>  <span class="cit-name-given-names">CL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Abiona</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">BS</span></span>, <span class="cit-auth"><span class="cit-name-surname">McLellan</span>  <span class="cit-name-given-names">JS</span></span>: <span class="cit-article-title">Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baric</span>  <span class="cit-name-given-names">RS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F</span></span>: <span class="cit-article-title">Receptor recognition by novel coronavirus from Wuhan: An analysis based on decade-long structural studies of SARS</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.17" data-doi="10.1002/rmv.706"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Coughlin</span>  <span class="cit-name-given-names">MM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Prabhakar</span>  <span class="cit-name-given-names">BS</span></span>: <span class="cit-article-title">Neutralizing human monoclonal antibodies to severe acute respiratory syndrome coronavirus: target, mechanism of action, and therapeutic potential</span>. <abbr class="cit-jnl-abbrev">Rev Med Virol</abbr> <span class="cit-pub-date">2012</span>, <span class="cit-vol">22</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">2</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DReviews%2Bin%2Bmedical%2Bvirology%26rft.stitle%253DRev%2BMed%2BVirol%26rft.aulast%253DCoughlin%26rft.auinit1%253DM.%2BM.%26rft.volume%253D22%26rft.issue%253D1%26rft.spage%253D2%26rft.epage%253D17%26rft.atitle%253DNeutralizing%2Bhuman%2Bmonoclonal%2Bantibodies%2Bto%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome%2Bcoronavirus%253A%2Btarget%252C%2Bmechanism%2Bof%2Baction%252C%2Band%2Btherapeutic%2Bpotential.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Frmv.706%26rft_id%253Dinfo%253Apmid%252F21905149%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/rmv.706&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21905149&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.22.951178.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">HJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">JW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">XJ</span></span>: <span class="cit-article-title">Neutralizing Monoclonal Antibodies as Promising Therapeutics against Middle East Respiratory Syndrome Coronavirus Infection</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-pub-date">2018</span>, <span class="cit-vol">10</span>(<span class="cit-issue">12</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">McCauley</span>  <span class="cit-name-given-names">J</span></span>: <span class="cit-article-title">GISAID: Global initiative on sharing all influenza data - from vision to reality</span>. <abbr class="cit-jnl-abbrev">Euro Surveill</abbr> <span class="cit-pub-date">2017</span>, <span class="cit-vol">22</span>(<span class="cit-issue">13</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.20" data-doi="10.3390/v4113209"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pickett</span>  <span class="cit-name-given-names">BE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Greer</span>  <span class="cit-name-given-names">DS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zaremba</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Larsen</span>  <span class="cit-name-given-names">CN</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">Virus pathogen database and analysis resource (ViPR): a comprehensive bioinformatics database and analysis resource for the coronavirus research community</span>. <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-pub-date">2012</span>, <span class="cit-vol">4</span>(<span class="cit-issue">11</span>):<span class="cit-fpage">3209</span>–<span class="cit-lpage">3226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DViruses%26rft.aulast%253DPickett%26rft.auinit1%253DB.%2BE.%26rft.volume%253D4%26rft.issue%253D11%26rft.spage%253D3209%26rft.epage%253D3226%26rft.atitle%253DVirus%2BPathogen%2BDatabase%2Band%2BAnalysis%2BResource%2B%2528ViPR%2529%253A%2B%2BA%2BComprehensive%2BBioinformatics%2BDatabase%2Band%2BAnalysis%2BResource%2Bfor%2Bthe%2BCoronavirus%2BResearch%2BCommunity.%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv4113209%26rft_id%253Dinfo%253Apmid%252F23202522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v4113209&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23202522&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.22.951178.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.21" data-doi="10.1093/molbev/msy096"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stecher</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knyaz</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K</span></span>: <span class="cit-article-title">MEGA X: Molecular Evolutionary Genetics Analysis across Computing Platforms</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-pub-date">2018</span>, <span class="cit-vol">35</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">1547</span>–<span class="cit-lpage">1549</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft.volume%253D35%26rft.spage%253D1547%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsy096%26rft_id%253Dinfo%253Apmid%252F29722887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msy096&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29722887&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.22.951178.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.22" data-doi="10.1093/oxfordjournals.molbev.a040454"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Saitou</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M</span></span>: <span class="cit-article-title">The neighbor-joining method: a new method for reconstructing phylogenetic trees</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-pub-date">1987</span>, <span class="cit-vol">4</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">406</span>–<span class="cit-lpage">425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DSaitou%26rft.auinit1%253DN%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253D406%26rft.epage%253D425%26rft.atitle%253DThe%2Bneighbor-joining%2Bmethod%253A%2Ba%2Bnew%2Bmethod%2Bfor%2Breconstructing%2Bphylogenetic%2Btrees%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a040454%26rft_id%253Dinfo%253Apmid%252F3447015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a040454&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3447015&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.22.951178.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987J406700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.23" data-doi="10.1007/BF01734359"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>  <span class="cit-name-given-names">J</span></span>: <span class="cit-article-title">Evolutionary trees from DNA sequences: a maximum likelihood approach</span>. <abbr class="cit-jnl-abbrev">J Mol Evol</abbr> <span class="cit-pub-date">1981</span>, <span class="cit-vol">17</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">368</span>–<span class="cit-lpage">376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DFelsenstein%26rft.auinit1%253DJ.%26rft.volume%253D17%26rft.issue%253D6%26rft.spage%253D368%26rft.epage%253D376%26rft.atitle%253DEvolutionary%2Btrees%2Bfrom%2BDNA%2Bsequences%253A%2Ba%2Bmaximum%2Blikelihood%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01734359%26rft_id%253Dinfo%253Apmid%252F7288891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01734359&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7288891&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.22.951178.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1981MG91100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.24" data-doi="10.1093/sysbio/20.4.406"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fitch</span>  <span class="cit-name-given-names">WM</span></span>: <span class="cit-article-title">Toward Defining the Course of Evolution: Minimum Change for a Specific Tree Topology</span>. <abbr class="cit-jnl-abbrev">Systematic Zoology</abbr> <span class="cit-pub-date">1971</span>, <span class="cit-vol">20</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">406</span>–<span class="cit-lpage">416</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2BZoology%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252F20.4.406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/20.4.406&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-book" id="cit-2020.02.22.951178v1.25"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Munro</span>  <span class="cit-name-given-names">H</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Jukes</span>  <span class="cit-name-given-names">TH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cantor</span>  <span class="cit-name-given-names">CR</span></span>: <span class="cit-chapter-title">Evolution of protein molecules. In</span>: <span class="cit-source">Mammalian Protein Metabolism</span>. Edited by <span class="cit-ed"><span class="cit-name-surname">Munro</span>  <span class="cit-name-given-names">H</span></span>: <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">New York</span>; <span class="cit-pub-date">1969</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">132</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.26" data-doi="10.2307/2408678"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>  <span class="cit-name-given-names">J</span></span>: <span class="cit-article-title">Confidence Limits on Phylogenies: An Approach Using the Bootstrap</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-pub-date">1985</span>, <span class="cit-vol">39</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">783</span>–<span class="cit-lpage">791</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.volume%253D39%26rft.issue%253D4%26rft.spage%253D783%26rft.epage%253D791%26rft.atitle%253DConfidence%2Blimits%2Bon%2Bphylogenies%253B%2Ban%2Bapproach%2Busing%2Bthe%2Bbootstrap%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2408678%26rft_id%253Dinfo%253Apmid%252F7288891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2408678&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1996015960&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=7288891&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.22.951178.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1985APJ8100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.27" data-doi="10.1093/nar/gki370"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Landau</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mayrose</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Glaser</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martz</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pupko</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ben-Tal</span>  <span class="cit-name-given-names">N</span></span>: <span class="cit-article-title">ConSurf 2005: the projection of evolutionary conservation scores of residues on protein structures</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-pub-date">2005</span>, <span class="cit-vol">33</span>(<span class="cit-issue">Web Server issue</span>):<span class="cit-fpage">W299</span>–<span class="cit-lpage">302</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DLandau%26rft.auinit1%253DM.%26rft.volume%253D33%26rft.issue%253Dsuppl_2%26rft.spage%253DW299%26rft.epage%253DW302%26rft.atitle%253DConSurf%2B2005%253A%2Bthe%2Bprojection%2Bof%2Bevolutionary%2Bconservation%2Bscores%2Bof%2Bresidues%2Bon%2Bprotein%2Bstructures%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki370%26rft_id%253Dinfo%253Apmid%252F15980475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gki370&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15980475&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.22.951178.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230271400059&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.28" data-doi="10.1093/nar/gku340"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Biasini</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bienert</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Waterhouse</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arnold</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Studer</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kiefer</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gallo Cassarino</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bertoni</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bordoli</span>  <span class="cit-name-given-names">L</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">SWISS-MODEL: modelling protein tertiary and quaternary structure using evolutionary information</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-pub-date">2014</span>, <span class="cit-vol">42</span>(<span class="cit-issue">Web Server issue</span>):<span class="cit-fpage">W252</span>–<span class="cit-lpage">258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku340%26rft_id%253Dinfo%253Apmid%252F24782522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24782522&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.22.951178.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000339715000042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.29" data-doi="10.1186/1471-2105-7-339"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Meng</span>  <span class="cit-name-given-names">EC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pettersen</span>  <span class="cit-name-given-names">EF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Couch</span>  <span class="cit-name-given-names">GS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">CC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ferrin</span>  <span class="cit-name-given-names">TE</span></span>: <span class="cit-article-title">Tools for integrated sequence-structure analysis with UCSF Chimera</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-pub-date">2006</span>, <span class="cit-vol">7</span>:<span class="cit-fpage">339</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DMeng%26rft.auinit1%253DE.%2BC.%26rft.volume%253D7%26rft.spage%253D339%26rft.epage%253D339%26rft.atitle%253DTools%2Bfor%2Bintegrated%2Bsequence-structure%2Banalysis%2Bwith%2BUCSF%2BChimera.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-7-339%26rft_id%253Dinfo%253Apmid%252F16836757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-7-339&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16836757&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.22.951178.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.30" data-doi="10.1371/journal.pmed.0030237"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">ter Meulen</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">van den Brink</span>  <span class="cit-name-given-names">EN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poon</span>  <span class="cit-name-given-names">LL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marissen</span>  <span class="cit-name-given-names">WE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leung</span>  <span class="cit-name-given-names">CS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">CY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bakker</span>  <span class="cit-name-given-names">AQ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bogaards</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Deventer</span>  <span class="cit-name-given-names">E</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">Human monoclonal antibody combination against SARS coronavirus: synergy and coverage of escape mutants</span>. <abbr class="cit-jnl-abbrev">PLoS Med</abbr> <span class="cit-pub-date">2006</span>, <span class="cit-vol">3</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">e237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BMed%26rft.aulast%253DTer%2BMeulen%26rft.auinit1%253DJ.%26rft.volume%253D3%26rft.issue%253D7%26rft.spage%253De237%26rft.epage%253De237%26rft.atitle%253DHuman%2Bmonoclonal%2Bantibody%2Bcombination%2Bagainst%2BSARS%2Bcoronavirus%253A%2Bsynergy%2Band%2Bcoverage%2Bof%2Bescape%2Bmutants.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pmed.0030237%26rft_id%253Dinfo%253Apmid%252F16796401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pmed.0030237&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16796401&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.22.951178.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.31" data-doi="10.1016/B978-0-12-381270-4.00019-6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Leaver-Fay</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tyka</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">SM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">OF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jacak</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kaufman</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Renfrew</span>  <span class="cit-name-given-names">PD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sheffler</span>  <span class="cit-name-given-names">W</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">ROSETTA3: an object-oriented software suite for the simulation and design of macromolecules</span>. <abbr class="cit-jnl-abbrev">Methods Enzymol</abbr> <span class="cit-pub-date">2011</span>, <span class="cit-vol">487</span>:<span class="cit-fpage">545</span>–<span class="cit-lpage">574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Benzymology%26rft.stitle%253DMethods%2BEnzymol%26rft.aulast%253DLeaver-Fay%26rft.auinit1%253DA.%26rft.volume%253D487%26rft.spage%253D545%26rft.epage%253D574%26rft.atitle%253DROSETTA3%253A%2Ban%2Bobject-oriented%2Bsoftware%2Bsuite%2Bfor%2Bthe%2Bsimulation%2Band%2Bdesign%2Bof%2Bmacromolecules.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FB978-0-12-381270-4.00019-6%26rft_id%253Dinfo%253Apmid%252F21187238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/B978-0-12-381270-4.00019-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21187238&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.22.951178.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weitzner</span>  <span class="cit-name-given-names">BD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jeliazkov</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lyskov</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marze</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuroda</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frick</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Adolf-Bryfogle</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Biswas</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dunbrack</span>  <span class="cit-name-given-names">RL</span>,   <span class="cit-name-suffix">Jr.</span>.</span>, <span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">JJ</span></span>: <span class="cit-article-title">Modeling and docking of antibody structures with Rosetta</span>. <abbr class="cit-jnl-abbrev">Nat Protoc</abbr> <span class="cit-pub-date">2017</span>, <span class="cit-vol">12</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">401</span>–<span class="cit-lpage">416</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BProtoc%26rft.volume%253D12%26rft.spage%253D401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.33" data-doi="10.1371/journal.pcbi.1000644"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sircar</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">JJ</span></span>: <span class="cit-article-title">SnugDock: paratope structural optimization during antibody-antigen docking compensates for errors in antibody homology models</span>. <abbr class="cit-jnl-abbrev">PLoS Comput Biol</abbr> <span class="cit-pub-date">2010</span>, <span class="cit-vol">6</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">e1000644</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DSircar%26rft.auinit1%253DA.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253De1000644%26rft.epage%253De1000644%26rft.atitle%253DSnugDock%253A%2Bparatope%2Bstructural%2Boptimization%2Bduring%2Bantibody-antigen%2Bdocking%2Bcompensates%2Bfor%2Berrors%2Bin%2Bantibody%2Bhomology%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1000644%26rft_id%253Dinfo%253Apmid%252F20098500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1000644&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20098500&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.22.951178.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tengs</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jonassen</span>  <span class="cit-name-given-names">CM</span></span>: <span class="cit-article-title">Distribution and Evolutionary History of the Mobile Genetic Element s2m in Coronaviruses</span>. <abbr class="cit-jnl-abbrev">Diseases</abbr> <span class="cit-pub-date">2016</span>, <span class="cit-vol">4</span>(<span class="cit-issue">3</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.35" data-doi="10.1074/jbc.M111.325803"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilken</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Geraghty</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F</span></span>: <span class="cit-article-title">Mechanisms of host receptor adaptation by severe acute respiratory syndrome coronavirus</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-pub-date">2012</span>, <span class="cit-vol">287</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">8904</span>–<span class="cit-lpage">8911</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M111.325803%26rft_id%253Dinfo%253Apmid%252F22291007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjExOiIyODcvMTIvODkwNCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzI3LzIwMjAuMDIuMjIuOTUxMTc4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.36" data-doi="10.1038/nature12711"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ge</span>  <span class="cit-name-given-names">XY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">XL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chmura</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Epstein</span>  <span class="cit-name-given-names">JH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mazet</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">C</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">Isolation and characterization of a bat SARS-like coronavirus that uses the ACE2 receptor</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-pub-date">2013</span>, <span class="cit-vol">503</span>(<span class="cit-issue">7477</span>):<span class="cit-fpage">535</span>–<span class="cit-lpage">538</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D503%26rft.spage%253D535%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12711%26rft_id%253Dinfo%253Apmid%252F24172901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24172901&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.22.951178.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327464200045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.37" data-doi="10.1038/cr.2013.92"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">Structure of MERS-CoV spike receptor-binding domain complexed with human receptor DPP4</span>. <abbr class="cit-jnl-abbrev">Cell Res</abbr> <span class="cit-pub-date">2013</span>, <span class="cit-vol">23</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">986</span>–<span class="cit-lpage">993</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRes%26rft.volume%253D23%26rft.spage%253D986%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2013.92%26rft_id%253Dinfo%253Apmid%252F23835475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/cr.2013.92&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23835475&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.22.951178.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000322574900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.02.22.951178v1.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Qiang</span>  <span class="cit-name-given-names">Huang</span></span>, <span class="cit-auth"><span class="cit-name-surname">Herrmann</span>  <span class="cit-name-given-names">A</span></span>: <span class="cit-article-title">Fast assessment of human receptor-binding capability of 2019 novel coronavirus (2019-nCoV)</span>. In: <abbr class="cit-jnl-abbrev">BioRxiv.</abbr> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">39.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.22.951178v1.39" data-doi="10.1073/pnas.0701000104"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chakraborti</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sheahan</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hensley</span>  <span class="cit-name-given-names">LE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Prabakaran</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rockx</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sidorov</span>  <span class="cit-name-given-names">IA</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">Potent cross-reactive neutralization of SARS coronavirus isolates by human monoclonal antibodies</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-pub-date">2007</span>, <span class="cit-vol">104</span>(<span class="cit-issue">29</span>):<span class="cit-fpage">12123</span>–<span class="cit-lpage">12128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0701000104%26rft_id%253Dinfo%253Apmid%252F17620608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzI5LzEyMTIzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMjcvMjAyMC4wMi4yMi45NTExNzguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">40.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.22.951178v1.40" data-doi="10.1074/jbc.M603275200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hwang</span>  <span class="cit-name-given-names">WC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Santelli</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sui</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jaroszewski</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stec</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Farzan</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marasco</span>  <span class="cit-name-given-names">WA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liddington</span>  <span class="cit-name-given-names">RC</span></span>: <span class="cit-article-title">Structural basis of neutralization by a human anti-severe acute respiratory syndrome spike protein antibody, 80R</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-pub-date">2006</span>, <span class="cit-vol">281</span>(<span class="cit-issue">45</span>):<span class="cit-fpage">34610</span>–<span class="cit-lpage">34616</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M603275200%26rft_id%253Dinfo%253Apmid%252F16954221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODEvNDUvMzQ2MTAiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8yNy8yMDIwLjAyLjIyLjk1MTE3OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">41.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.22.951178v1.41" data-doi="10.1016/j.cell.2018.12.028"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Walls</span>  <span class="cit-name-given-names">AC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiong</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">YJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tortorici</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snijder</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Quispe</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameroni</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gopal</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dai</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lanzavecchia</span>  <span class="cit-name-given-names">A</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">Unexpected Receptor Functional Mimicry Elucidates Activation of Coronavirus Fusion</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-pub-date">2019</span>, <span class="cit-vol">176</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">1026</span>–<span class="cit-lpage">1039 e1015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D176%26rft.spage%253D1026%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2018.12.028%26rft_id%253Dinfo%253Apmid%252F30712865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2018.12.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30712865&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.22.951178.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">42.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.22.951178v1.42" data-doi="10.1016/j.jmb.2009.03.042"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pak</span>  <span class="cit-name-given-names">JE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sharon</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Satkunarajah</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Auperin</span>  <span class="cit-name-given-names">TC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kelvin</span>  <span class="cit-name-given-names">DJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Seetharaman</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cochrane</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Plummer</span>  <span class="cit-name-given-names">FA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berry</span>  <span class="cit-name-given-names">JD</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">Structural insights into immune recognition of the severe acute respiratory syndrome coronavirus S protein receptor binding domain</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr> <span class="cit-pub-date">2009</span>, <span class="cit-vol">388</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">815</span>–<span class="cit-lpage">823</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DPak%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D388%26rft.issue%253D4%26rft.spage%253D815%26rft.epage%253D823%26rft.atitle%253DStructural%2Binsights%2Binto%2Bimmune%2Brecognition%2Bof%2Bthe%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome%2Bcoronavirus%2BS%2Bprotein%2Breceptor%2Bbinding%2Bdomain.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2009.03.042%26rft_id%253Dinfo%253Apmid%252F19324051%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2009.03.042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19324051&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.22.951178.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266302500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">43.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.22.951178v1.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chappell</span>  <span class="cit-name-given-names">JD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joyce</span>  <span class="cit-name-given-names">MG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kanekiyo</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Becker</span>  <span class="cit-name-given-names">MM</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Doremalen</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">N</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">Importance of Neutralizing Monoclonal Antibodies Targeting Multiple Antigenic Sites on the Middle East Respiratory Syndrome Coronavirus Spike Glycoprotein To Avoid Neutralization Escape</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-pub-date">2018</span>, <span class="cit-vol">92</span>(<span class="cit-issue">10</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">44.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.22.951178v1.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">Structural basis for the neutralization of MERS-CoV by a human monoclonal antibody MERS-27</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-pub-date">2015</span>, <span class="cit-vol">5</span>:<span class="cit-fpage">13133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D5%26rft.spage%253D13133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">45.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.22.951178v1.45" data-doi="10.1038/ncomms9223"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ying</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Prabakaran</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dimitrov</span>  <span class="cit-name-given-names">DS</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">Junctional and allele-specific residues are critical for MERS-CoV neutralization by an exceptionally potent germline-like antibody</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-pub-date">2015</span>, <span class="cit-vol">6</span>:<span class="cit-fpage">8223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D6%26rft.spage%253D8223%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms9223%26rft_id%253Dinfo%253Apmid%252F26370782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms9223&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26370782&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.22.951178.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">46.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.22.951178v1.46" data-doi="10.1093/infdis/jix209"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bao</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xue</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lv</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">S</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">Human Neutralizing Monoclonal Antibody Inhibition of Middle East Respiratory Syndrome Coronavirus Replication in the Common Marmoset</span>. <abbr class="cit-jnl-abbrev">J Infect Dis</abbr> <span class="cit-pub-date">2017</span>, <span class="cit-vol">215</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">1807</span>–<span class="cit-lpage">1815</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BInfect%2BDis%26rft.volume%253D215%26rft.spage%253D1807%26rft_id%253Dinfo%253Adoi%252F10.1093%252Finfdis%252Fjix209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/infdis/jix209&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">47.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.22.951178v1.47" data-doi="10.1038/cr.2015.113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">W</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">A humanized neutralizing antibody against MERS-CoV targeting the receptor-binding domain of the spike protein</span>. <abbr class="cit-jnl-abbrev">Cell Res</abbr> <span class="cit-pub-date">2015</span>, <span class="cit-vol">25</span>(<span class="cit-issue">11</span>):<span class="cit-fpage">1237</span>–<span class="cit-lpage">1249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRes%26rft.volume%253D25%26rft.spage%253D1237%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2015.113%26rft_id%253Dinfo%253Apmid%252F26391698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/cr.2015.113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26391698&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.22.951178.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">48.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.22.951178v1.48" data-doi="10.1038/ncomms8712"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joyce</span>  <span class="cit-name-given-names">MG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Modjarrad</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leung</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lees</span>  <span class="cit-name-given-names">CR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yassine</span>  <span class="cit-name-given-names">HM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kanekiyo</span>  <span class="cit-name-given-names">M</span></span> <span class="cit-etal">et al</span>: <span class="cit-article-title">Evaluation of candidate vaccine approaches for MERS-CoV</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-pub-date">2015</span>, <span class="cit-vol">6</span>:<span class="cit-fpage">7712</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D6%26rft.spage%253D7712%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms8712%26rft_id%253Dinfo%253Apmid%252F26218507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms8712&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26218507&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F27%2F2020.02.22.951178.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.02.22.951178v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.02.22.951178v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/689547v2" title="Poincaré Maps for Analyzing Complex Hierarchies in Single-Cell Data" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.02.22.961268v1" title="A simple magnetic nanoparticles-based viral RNA extraction method for efficient detection of SARS-CoV-2" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;February 27, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.02.22.951178v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.02.22.951178v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.02.22.951178v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.02.22.951178v1" class="active" data-icon-position="" data-hide-link-title="0">Spike protein binding prediction with neutralizing antibodies of SARS-CoV-2</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1169701" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-8OiWlDpNwoxrUE-xfna0kPUerNe0ys3mQtuqCCFmgM0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1169701" id="node1169701" data-pisa="biorxiv;2020.02.22.951178v1" data-pisa-master="biorxiv;2020.02.22.951178" data-seqnum="1169701" data-apath="/biorxiv/early/2020/02/27/2020.02.22.951178.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Spike protein binding prediction with neutralizing antibodies of SARS-CoV-2</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Tamina</span> <span class="nlm-surname">Park</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Sang-Yeop</span> <span class="nlm-surname">Lee</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Seil</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Mi Jeong</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Hong Gi</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Sangmi</span> <span class="nlm-surname">Jun</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Seung Il</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Bum Tae</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Edmond Changkyun</span> <span class="nlm-surname">Park</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Daeui</span> <span class="nlm-surname">Park</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.02.22.951178; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.02.22.951178 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.02.22.951178v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.02.22.951178v1&amp;title=Spike%20protein%20binding%20prediction%20with%20neutralizing%20antibodies%20of%20SARS-CoV-2" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.22.951178v1&amp;title=Spike%20protein%20binding%20prediction%20with%20neutralizing%20antibodies%20of%20SARS-CoV-2" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.22.951178v1&amp;text=Spike%20protein%20binding%20prediction%20with%20neutralizing%20antibodies%20of%20SARS-CoV-2" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.22.951178v1&amp;title=Spike%20protein%20binding%20prediction%20with%20neutralizing%20antibodies%20of%20SARS-CoV-2" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.02.22.951178v1&amp;t=Spike%20protein%20binding%20prediction%20with%20neutralizing%20antibodies%20of%20SARS-CoV-2" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.02.22.951178v1&amp;title=Spike%20protein%20binding%20prediction%20with%20neutralizing%20antibodies%20of%20SARS-CoV-2" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.22.951178v1&amp;title=Spike%20protein%20binding%20prediction%20with%20neutralizing%20antibodies%20of%20SARS-CoV-2" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1169701" id="node1169701--2" data-pisa="biorxiv;2020.02.22.951178v1" data-pisa-master="biorxiv;2020.02.22.951178" data-seqnum="1169701" data-apath="/biorxiv/early/2020/02/27/2020.02.22.951178.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Spike protein binding prediction with neutralizing antibodies of SARS-CoV-2</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Tamina</span> <span class="nlm-surname">Park</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Sang-Yeop</span> <span class="nlm-surname">Lee</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Seil</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Mi Jeong</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Hong Gi</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Sangmi</span> <span class="nlm-surname">Jun</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Seung Il</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Bum Tae</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Edmond Changkyun</span> <span class="nlm-surname">Park</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Daeui</span> <span class="nlm-surname">Park</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.02.22.951178; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.02.22.951178 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Spike%20protein%20binding%20prediction%20with%20neutralizing%20antibodies%20of%20SARS-CoV-2&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Park%2C+Tamina&amp;rft.au=Lee%2C+Sang-Yeop&amp;rft.au=Kim%2C+Seil&amp;rft.au=Kim%2C+Mi+Jeong&amp;rft.au=Kim%2C+Hong+Gi&amp;rft.au=Jun%2C+Sangmi&amp;rft.au=Kim%2C+Seung+Il&amp;rft.au=Kim%2C+Bum+Tae&amp;rft.au=Park%2C+Edmond+Changkyun&amp;rft.au=Park%2C+Daeui"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1169701/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1169701/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1169701/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1169701/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1169701/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1169701/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1169701/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1169701/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1169701/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1169701/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1169701/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1169701/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.22.951178v1&amp;count=horizontal&amp;via=&amp;text=Spike%20protein%20binding%20prediction%20with%20neutralizing%20antibodies%20of%20SARS-CoV-2&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.02.22.951178v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.02.22.951178v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.02.22.951178v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">Bioinformatics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
