<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/embed/graphic-1.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/03/08/2020.03.03.972133.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.03.03.972133v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="AI-aided design of novel targeted covalent inhibitors against SARS-CoV-2"/>
  <meta name="DC.Identifier" content="10.1101/2020.03.03.972133"/>
  <meta name="DC.Date" content="2020-03-08"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="The focused drug repurposing of known approved drugs (such as lopinavir/ritonavir) has been reported failed for curing SARS-CoV-2 infected patients. It is urgent to generate new chemical entities against this virus. As a key enzyme in the life-cycle of coronavirus, the 3C-like main protease (3CLpro or Mpro) is the most attractive for antiviral drug design. Based on a recently solved structure (PDB ID: 6LU7), we developed a novel advanced deep Q-learning network with the fragment-based drug design (ADQN-FBDD) for generating potential lead compounds targeting SARS-CoV-2 3CLpro. We obtained a series of derivatives from those lead compounds by our structure-based optimization policy (SBOP). All the 47 lead compounds directly from our AI-model and related derivatives based on SBOP are accessible in our molecular library at &lt;;https://github.com/tbwxmu/2019-nCov&gt;;. These compounds can be used as potential candidates for researchers in their development of drugs against SARS-CoV-2."/>
  <meta name="DC.Contributor" content="Bowen Tang"/>
  <meta name="DC.Contributor" content="Fengming He"/>
  <meta name="DC.Contributor" content="Dongpeng Liu"/>
  <meta name="DC.Contributor" content="Meijuan Fang"/>
  <meta name="DC.Contributor" content="Zhen Wu"/>
  <meta name="DC.Contributor" content="Dong Xu"/>
  <meta name="article:published_time" content="2020-03-08"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="AI-aided design of novel targeted covalent inhibitors against SARS-CoV-2"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;The focused drug repurposing of known approved drugs (such as lopinavir/ritonavir) has been reported failed for curing SARS-CoV-2 infected patients. It is urgent to generate new chemical entities against this virus. As a key enzyme in the life-cycle of coronavirus, the 3C-like main protease (3CLpro or Mpro) is the most attractive for antiviral drug design. Based on a recently solved structure (PDB ID: 6LU7), we developed a novel advanced deep Q-learning network with the fragment-based drug design (ADQN-FBDD) for generating potential lead compounds targeting SARS-CoV-2 3CLpro. We obtained a series of derivatives from those lead compounds by our structure-based optimization policy (SBOP). All the 47 lead compounds directly from our AI-model and related derivatives based on SBOP are accessible in our molecular library at https://github.com/tbwxmu/2019-nCov. These compounds can be used as potential candidates for researchers in their development of drugs against SARS-CoV-2.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.03.03.972133v1"/>
  <meta name="citation_id" content="2020.03.03.972133v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.03.972133v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.03.972133v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.03.972133v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.03.03.972133"/>
  <meta name="citation_num_pages" content="26"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.03.03.972133"/>
  <meta name="citation_author" content="Bowen Tang"/>
  <meta name="citation_author_institution" content="Fujian Provincial Key Laboratory of Innovative Drug Target Research, School of Pharmaceutical Sciences, Xiamen University"/>
  <meta name="citation_author_institution" content="Department of Electrical Engineering and Computer Science, Informatics Institute, and Christopher S. Bond Life Sciences Center, University of Missouri"/>
  <meta name="citation_author" content="Fengming He"/>
  <meta name="citation_author_institution" content="Fujian Provincial Key Laboratory of Innovative Drug Target Research, School of Pharmaceutical Sciences, Xiamen University"/>
  <meta name="citation_author" content="Dongpeng Liu"/>
  <meta name="citation_author_institution" content="Department of Electrical Engineering and Computer Science, Informatics Institute, and Christopher S. Bond Life Sciences Center, University of Missouri"/>
  <meta name="citation_author" content="Meijuan Fang"/>
  <meta name="citation_author_institution" content="Fujian Provincial Key Laboratory of Innovative Drug Target Research, School of Pharmaceutical Sciences, Xiamen University"/>
  <meta name="citation_author" content="Zhen Wu"/>
  <meta name="citation_author_institution" content="Fujian Provincial Key Laboratory of Innovative Drug Target Research, School of Pharmaceutical Sciences, Xiamen University"/>
  <meta name="citation_author_email" content="wuzhen@xmu.edu.cn"/>
  <meta name="citation_author_email" content="xudong@missouri.edu"/>
  <meta name="citation_author" content="Dong Xu"/>
  <meta name="citation_author_institution" content="Department of Electrical Engineering and Computer Science, Informatics Institute, and Christopher S. Bond Life Sciences Center, University of Missouri"/>
  <meta name="citation_author_email" content="wuzhen@xmu.edu.cn"/>
  <meta name="citation_author_email" content="xudong@missouri.edu"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4809-0514"/>
  <meta name="citation_reference" content="Gorbalenya AE. Severe acute respiratory syndrome-related coronavirus–The species and its viruses, a statement of the Coronavirus Study Group. BioRxiv, (2020)."/>
  <meta name="citation_reference" content="Coronavirus COVID-19 Global Cases by Johns Hopkins CSSE, available from: https://forum.truckersmp.com/index.php?/topic/93003-coronavirus-covid-19-global-cases-by-johns-hopkins-csse/&amp;;tab=comments."/>
  <meta name="citation_reference" content="Zhang L, et al. Alpha-ketoamides as broad-spectrum inhibitors of coronavirus and enterovirus replication Structure-based design, synthesis, and activity assessment. Journal of Medicinal Chemistry, (2020)."/>
  <meta name="citation_reference" content="卢洪洲 陈凌席刘李李尚王沈. 洛匹那韦利托那韦和阿比多尔用于治疗新型冠状病毒肺炎的有效性研究. 中华传染病杂志 38, (2020)."/>
  <meta name="citation_reference" content="Liu X, Wang X-J. Potential inhibitors for 2019-nCoV coronavirus M protease from clinically approved medicines. bioRxiv, (2020)."/>
  <meta name="citation_reference" content="Yang H, et al. Design of wide-spectrum inhibitors targeting coronavirus main proteases. PLoS biology 3, (2005)."/>
  <meta name="citation_reference" content="Singh J, Petter RC, Baillie TA, Whitty A. The resurgence of covalent drugs. Nature reviews Drug discovery 10, 307–317 (2011)."/>
  <meta name="citation_reference" content="Tuley A, Fast W. The taxonomy of covalent inhibitors. Biochemistry 57, 3326–3337 (2018)."/>
  <meta name="citation_reference" content="Jain RP, et al. Synthesis and evaluation of keto-glutamine analogues as potent inhibitors of severe acute respiratory syndrome 3CLpro. Journal of medicinal chemistry 47, 6113–6116 (2004)."/>
  <meta name="citation_reference" content="Wu C-Y, et al. Small molecules targeting severe acute respiratory syndrome human coronavirus. Proceedings of the National Academy of Sciences 101, 10012–10017 (2004)."/>
  <meta name="citation_reference" content="Ghosh AK, et al. Design and synthesis of peptidomimetic severe acute respiratory syndrome chymotrypsin-like protease inhibitors. Journal of medicinal chemistry 48, 6767–6771 (2005)."/>
  <meta name="citation_reference" content="Shie J-J, et al. Discovery of potent anilide inhibitors against the severe acute respiratory syndrome 3CL protease. Journal of medicinal chemistry 48, 4469–4473 (2005)."/>
  <meta name="citation_reference" content="Shie J-J, et al. Inhibition of the severe acute respiratory syndrome 3CL protease by peptidomimetic α, β-unsaturated esters. Bioorganic &amp;; medicinal chemistry 13, 5240–5252 (2005)."/>
  <meta name="citation_reference" content="Al-Gharabli SI, et al. An efficient method for the synthesis of peptide aldehyde libraries employed in the discovery of reversible SARS coronavirus main protease (SARS-CoV Mpro) inhibitors. ChemBioChem 7, 1048–1055 (2006)."/>
  <meta name="citation_reference" content="Lu I-L, et al. Structure-based drug design and structural biology study of novel nonpeptide inhibitors of severe acute respiratory syndrome coronavirus main protease. Journal of medicinal chemistry 49, 5154–5161 (2006)."/>
  <meta name="citation_reference" content="Tsai K-C, et al. Discovery of a novel family of SARS-CoV protease inhibitors by virtual screening and 3D-QSAR studies. Journal of medicinal chemistry 49, 3485–3495 (2006)."/>
  <meta name="citation_reference" content="Wu C-Y, et al. Stable benzotriazole esters as mechanism-based inactivators of the severe acute respiratory syndrome 3CL protease. Chemistry &amp;; biology 13, 261–268 (2006)."/>
  <meta name="citation_reference" content="Akaji K, Konno H, Onozuka M, Makino A, Saito H, Nosaka K. Evaluation of peptide-aldehyde inhibitors using R188I mutant of SARS 3CL protease as a proteolysis-resistant mutant. Bioorganic &amp;; medicinal chemistry 16, 9400–9408 (2008)."/>
  <meta name="citation_reference" content="Ghosh AK, et al. Design, synthesis and antiviral efficacy of a series of potent chloropyridyl ester-derived SARS-CoV 3CLpro inhibitors. Bioorganic &amp;; medicinal chemistry letters 18, 5684–5688 (2008)."/>
  <meta name="citation_reference" content="Shao Y-M, et al. Design, synthesis, and evaluation of trifluoromethyl ketones as inhibitors of SARS-CoV 3CL protease. Bioorganic &amp;; medicinal chemistry 16, 4652–4660 (2008)."/>
  <meta name="citation_reference" content="Kuo C-J, et al. Individual and common inhibitors of coronavirus and picornavirus main proteases. FEBS letters 583, 549–555 (2009)."/>
  <meta name="citation_reference" content="Ramajayam R, Tan K-P, Liu H-G, Liang P-H. Synthesis and evaluation of pyrazolone compounds as SARS-coronavirus 3C-like protease inhibitors. Bioorganic &amp;; medicinal chemistry 18, 7849–7854 (2010)."/>
  <meta name="citation_reference" content="Ryu YB, et al. Biflavonoids from Torreya nucifera displaying SARS-CoV 3CLpro inhibition. Bioorganic &amp;; medicinal chemistry 18, 7940–7947 (2010)."/>
  <meta name="citation_reference" content="Akaji K, et al. Structure-based design, synthesis, and evaluation of peptide-mimetic SARS 3CL protease inhibitors. Journal of medicinal chemistry 54, 7962–7973 (2011)."/>
  <meta name="citation_reference" content="Jacobs J, et al. Discovery, synthesis, and structure-based optimization of a series of N-(tert-butyl)-2-(N-arylamido)-2-(pyridin-3-yl) acetamides (ML188) as potent noncovalent small molecule inhibitors of the severe acute respiratory syndrome coronavirus (SARS-CoV) 3CL protease. Journal of medicinal chemistry 56, 534–546 (2013)."/>
  <meta name="citation_reference" content="Ren Z, et al. The newly emerged SARS-like coronavirus HCoV-EMC also has an&quot;; Achilles' heel&quot;;: current effective inhibitor targeting a 3C-like protease. Protein &amp;; cell 4, 248 (2013)."/>
  <meta name="citation_reference" content="Thanigaimalai P, et al. Development of potent dipeptide-type SARS-CoV 3CL protease inhibitors with novel P3 scaffolds: Design, synthesis, biological evaluation, and docking studies. European journal of medicinal chemistry 68, 372–384 (2013)."/>
  <meta name="citation_reference" content="Turlington M, et al. Discovery of N-(benzo [1, 2, 3] triazol-1-yl)-N-(benzyl) acetamido) phenyl) carboxamides as severe acute respiratory syndrome coronavirus (SARS-CoV) 3CLpro inhibitors: identification of ML300 and noncovalent nanomolar inhibitors with an induced-fit binding. Bioorganic &amp;; medicinal chemistry letters 23, 6172–6177 (2013)."/>
  <meta name="citation_reference" content="Kumar V, et al. Identification and evaluation of potent Middle East respiratory syndrome coronavirus (MERS-CoV) 3CLPro inhibitors. Antiviral research 141, 101–106 (2017)."/>
  <meta name="citation_reference" content="Tang B, Kramer ST, Fang M, Qiu Y, Wu Z, Xu D. A self-attention based message passing neural network for predicting molecular lipophilicity and aqueous solubility. Journal of Cheminformatics 12, 1–9 (2020)."/>
  <meta name="citation_reference" content="Liu K, et al. Chemi-Net: a molecular graph convolutional network for accurate drug property prediction. International journal of molecular sciences 20, 3389 (2019)."/>
  <meta name="citation_reference" content="Wang X, Li Z, Jiang M, Wang S, Zhang S, Wei Z. Molecule Property Prediction Based on Spatial Graph Embedding. Journal of chemical information and modeling 59, 3817–3828 (2019)."/>
  <meta name="citation_reference" content="Wu Z, et al. MoleculeNet: a benchmark for molecular machine learning. Chemical science 9, 513–530 (2018)."/>
  <meta name="citation_reference" content="You J, Liu B, Ying Z, Pande V, Leskovec J. Graph convolutional policy network for goal-directed molecular graph generation. In: Advances in neural information processing systems (2018)."/>
  <meta name="citation_reference" content="Zhu K, et al. Docking Covalent Inhibitors: A Parameter Free Approach To Pose Prediction and Scoring. Journal of Chemical Information &amp;; Modeling 54, 1932–1940 (2014)."/>
  <meta name="citation_reference" content="Walters WP, Murcko M. Assessing the impact of generative AI on medicinal chemistry. Nature Biotechnology, 1–3 (2020)."/>
  <meta name="citation_reference" content="Olivecrona M, Blaschke T, Engkvist O, Chen H. Molecular de-novo design through deep reinforcement learning. Journal of cheminformatics 9, 48 (2017)."/>
  <meta name="citation_reference" content="Zhou Z, Kearnes S, Li L, Zare RN, Riley P. Optimization of molecules via deep reinforcement learning. Scientific reports 9, 1–10 (2019)."/>
  <meta name="citation_reference" content="Popova M, Isayev O, Tropsha A. Deep reinforcement learning for de novo drug design. Science advances 4, eaap7885 (2018)."/>
  <meta name="citation_reference" content="Zhavoronkov A, et al. Deep learning enables rapid identification of potent DDR1 kinase inhibitors. Nature biotechnology 37, 1038–1040 (2019)."/>
  <meta name="citation_reference" content="Van Hasselt H, Guez A, Silver D. Deep reinforcement learning with double q-learning. In: Thirtieth AAAI conference on artificial intelligence (2016)."/>
  <meta name="citation_reference" content="Simonini T. Improvements in Deep Q Learning: Dueling Double DQN, Prioritized Experience Replay, and fixed Q-targets. Červenec (2018)."/>
  <meta name="citation_reference" content="Speck-Planche A. Recent advances in fragment-based computational drug design: tackling simultaneous targets/biological effects. Future Science (2018)."/>
  <meta name="citation_reference" content="Degen J, Wegscheid-Gerlach C, Zaliani A, Rarey M. On the Art of Compiling and Using‘Drug-Like’Chemical Fragment Spaces. ChemMedChem: Chemistry Enabling Drug Discovery 3, 1503–1507 (2008)."/>
  <meta name="citation_reference" content="Varin T, Schuffenhauer A, Ertl P, Renner S. Mining for bioactive scaffolds with scaffold networks: improved compound set enrichment from primary screening data. Journal of chemical information and modeling 51, 1528–1538 (2011)."/>
  <meta name="citation_reference" content="Schuffenhauer A, Ertl P, Roggo S, Wetzel S, Koch MA, Waldmann H. The scaffold tree-visualization of the scaffold universe by hierarchical scaffold classification. Journal of chemical information and modeling 47, 47–58 (2007)."/>
  <meta name="citation_reference" content="Reis J, Gaspar A, Milhazes N, Borges F. Chromone as a Privileged Scaffold in Drug Discovery: Recent Advances: Miniperspective. Journal of medicinal chemistry 60, 7941–7957 (2017)."/>
  <meta name="citation_reference" content="Pillaiyar T, Manickam M, Namasivayam V, Hayashi Y, Jung S-H. An Overview of Severe Acute Respiratory Syndrome-Coronavirus (SARS-CoV) 3CL Protease Inhibitors: Peptidomimetics and Small Molecule Chemotherapy. Journal of Medicinal Chemistry, (2016)."/>
  <meta name="citation_reference" content="Bickerton GR, Paolini GV, Besnard J, Muresan S, Hopkins AL. Quantifying the chemical beauty of drugs. Nature chemistry 4, 90 (2012)."/>
  <meta name="citation_reference" content="Elton DC, Boukouvalas Z, Fuge MD, Chung PW. Deep learning for molecular design—a review of the state of the art. Molecular Systems Design &amp;; Engineering 4, 828–849 (2019)."/>
  <meta name="citation_reference" content="Yang X, Wang Y, Byrne R, Schneider G, Yang S. Concepts of artificial intelligence for computer-assisted drug discovery. Chemical reviews 119, 10520–10594 (2019)."/>
  <meta name="citation_reference" content="Friesner RA, et al. Extra precision glide: Docking and scoring incorporating a model of hydrophobic enclosure for protein− ligand complexes. Journal of medicinal chemistry 49, 6177–6196 (2006)."/>
  <meta name="citation_reference" content="Jin Z, et al. Structure-based drug design, virtual screening and high-throughput screening rapidly identify antiviral leads targeting COVID-19. bioRxiv, 2020.2002.2026.964882 (2020)."/>
  <meta name="twitter:title" content="AI-aided design of novel targeted covalent inhibitors against SARS-CoV-2"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="The focused drug repurposing of known approved drugs (such as lopinavir/ritonavir) has been reported failed for curing SARS-CoV-2 infected patients. It is urgent to generate new chemical entities against this virus. As a key enzyme in the life-cycle of coronavirus, the 3C-like main protease (3CLpro or Mpro) is the most attractive for antiviral drug design. Based on a recently solved structure (PDB ID: 6LU7), we developed a novel advanced deep Q-learning network with the fragment-based drug design (ADQN-FBDD) for generating potential lead compounds targeting SARS-CoV-2 3CLpro. We obtained a series of derivatives from those lead compounds by our structure-based optimization policy (SBOP). All the 47 lead compounds directly from our AI-model and related derivatives based on SBOP are accessible in our molecular library at &lt;;https://github.com/tbwxmu/2019-nCov&gt;;. These compounds can be used as potential candidates for researchers in their development of drugs against SARS-CoV-2."/>
  <meta name="og-title" property="og:title" content="AI-aided design of novel targeted covalent inhibitors against SARS-CoV-2"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.03.972133v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="The focused drug repurposing of known approved drugs (such as lopinavir/ritonavir) has been reported failed for curing SARS-CoV-2 infected patients. It is urgent to generate new chemical entities against this virus. As a key enzyme in the life-cycle of coronavirus, the 3C-like main protease (3CLpro or Mpro) is the most attractive for antiviral drug design. Based on a recently solved structure (PDB ID: 6LU7), we developed a novel advanced deep Q-learning network with the fragment-based drug design (ADQN-FBDD) for generating potential lead compounds targeting SARS-CoV-2 3CLpro. We obtained a series of derivatives from those lead compounds by our structure-based optimization policy (SBOP). All the 47 lead compounds directly from our AI-model and related derivatives based on SBOP are accessible in our molecular library at &lt;;https://github.com/tbwxmu/2019-nCov&gt;;. These compounds can be used as potential candidates for researchers in their development of drugs against SARS-CoV-2."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-03-08"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>AI-aided design of novel targeted covalent inhibitors against SARS-CoV-2 | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1186830 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-2">The focused drug repurposing of known approved drugs (such as lopinavir/ritonavir) has been reported failed for curing SARS-CoV-2 infected patients. It is urgent to generate new chemical entities against this virus. As a key enzyme in the life-cycle of coronavirus, the 3C-like main protease (3CL
     <sup>pro</sup> or M
     <sup>pro</sup>) is the most attractive for antiviral drug design. Based on a recently solved structure (PDB ID: 6LU7), we developed a novel advanced deep Q-learning network with the fragment-based drug design (ADQN-FBDD) for generating potential lead compounds targeting SARS-CoV-2 3CL
     <sup>pro</sup>. We obtained a series of derivatives from those lead compounds by our structure-based optimization policy (SBOP). All the 47 lead compounds directly from our AI-model and related derivatives based on SBOP are accessible in our molecular library at 
     <a href="https://github.com/tbwxmu/2019-nCov">https://github.com/tbwxmu/2019-nCov</a>. These compounds can be used as potential candidates for researchers in their development of drugs against SARS-CoV-2.
    </p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Introduction</h2>
    <p id="p-4">The emerging coronavirus SARS-CoV-2 has caused an outbreak of coronavirus disease (COVID-19) worldwide.
     <sup>
      <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>
     </sup> As of March 2, 2020, more than 90,000 people have been infected by SARS-CoV-2 and more than 3000 people have been reported dead according to Johns Hopkins Coronavirus map tracker.
     <sup>
      <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>
     </sup> The numbers of infection and death are still increasing. To face the considerable threat of SARS-CoV-2, it is urgent to develop new inhibitors or drugs against this deadly virus. Unfortunately, since the outbreak of severe acute respiratory syndrome (SARS) eighteen years ago, there has been no approved treatment against the SARS coronavirus (SARS-CoV),
     <sup>
      <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>
     </sup> which is similar to SARS-CoV-2. Repurposing potential drugs such as lopinavir and ritonavir also failed to SARS-CoV-2 injected patients.
     <sup>
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>
     </sup> Structure-based antiviral drug design with a new artificial intelligence algorithm may represent a more helpful approach to get the SARS-CoV-2 targeted inhibitors or drugs. Thanks to the prompt efforts of many researchers, we have several pieces of important information about this vital virus genome and protein structures. We now know that the non-structural protein 5 (Nsp5) is the main protease (M
     <sup>pro</sup>) of SARS-CoV-2 and it is a cysteine protease, which also been called “3C-like protease” (3CL
     <sup>pro</sup>). Moreover, we know that the 3D structure of 3CL
     <sup>pro</sup> is very similar to SARS-CoV with a sequence identity of &gt;;96% and 3D structure superposition RMSD
     <sub>Cα</sub> of 0.44 Å as shown in Figures S1 and S2.
    </p>
    <p id="p-5">3CL
     <sup>pro</sup> has been reported as an attractive target for developing anti-coronaviral drugs: 1) this protease is highly conserved in both sequences and 3D structures;
     <sup>
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>
     </sup> 2) 3CL
     <sup>pro</sup> is a key enzyme for related virus (including SARS and SARS-CoV-2) replication; 3) it only exists in the virus, not in humans. Developing specific antiviral drugs targeting 3CL
     <sup>pro</sup> of the specific virus has shown significant success; for example, both approved drugs lopinavir and ritonavir can completely occupy the substrate-binding site of 3CL
     <sup>pro</sup> to break down the replication of human immunodeficiency virus (HIV). However, due to the large difference between HIV and SARS-CoV-2 3CL
     <sup>pro</sup>, lopinavir and ritonavir were validated ineffective for inhibiting SARS-Cov-2.
     <sup>
      <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>
     </sup> On the other hand, the substrate-binding site of 3CL
     <sup>pro</sup> is almost the same between the SARS-CoV-2 and SARS as Figure S3 presents. The developed potential inhibitors and drug-design experience targeting SARS-3CL
     <sup>pro</sup> may also be applicable to SARS-CoV-2. For example, the recently solved structure of SARS-CoV-2 3CL
     <sup>pro</sup> (PDB ID: 6LU7) indicates that the developed inhibitor N3,
     <sup>
      <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>
     </sup> which is a covalent inhibitor derived from non-covalent inhibitors against SARS can also bind SARS-CoV-2 3CL
     <sup>pro</sup> with a similar binding conformation (Figure S4).
    </p>
    <p id="p-6">All the above available information paved a way to design new targeted covalent inhibitors (TCI)
     <sup>
      <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>
     </sup> against SARS-CoV-2. A successful TCI against 3CL
     <sup>pro</sup> must first be able to fit in the binding site of 3CL
     <sup>pro</sup> with an appropriate pose that keeps its reactive groups close enough to the Cys145, which then undergoes a chemical step (nucleophilic attack by Cys145) leading to the formation of a stable covalent bond as presented in the scheme below: 
     <span class="disp-formula" id="disp-formula-1">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/graphic-1.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/graphic-1.gif"/>
       </noscript>
      </span>
     </span>
    </p>
    <p id="p-7">TCIs usually have a longer target residence time than the relative non-covalent inhibitors in theory given the following: 1) For the inhibition, k
     <sub>1</sub> must be larger than k
     <sub>2</sub>, and thus the non-covalent binding is determined by the equilibrium constant k
     <sub>1</sub>/k
     <sub>2</sub>; 2) TCIs have the chemical reaction step with the target, where usually k
     <sub>3</sub> ≫ k
     <sub>4</sub>; and 3) for TCIs, the binding process is controlled by k
     <sub>1</sub> k
     <sub>3</sub>/(k
     <sub>2</sub> k
     <sub>4</sub>), which is bigger than k
     <sub>1</sub>/k
     <sub>2</sub> in non-covalent inhibitors. In some extreme cases k
     <sub>4</sub> = 0, and hence, these irreversible inhibitors covalently bind the target until the target disappears.
     <sup>
      <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>, 
      <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>
     </sup>
    </p>
    <p id="p-8">Considering the inhibitors of SARS 3CL
     <sup>pro</sup> may be also bio-active to SARS-CoV-2, we have created a molecular library including all the reported SARS-3CL
     <sup>pro</sup> inhibitors (284 molecules)
     <sup>
      <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>, 
      <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>, 
      <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>, 
      <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>, 
      <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>, 
      <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>, 
      <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>, 
      <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>, 
      <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>, 
      <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>, 
      <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>, 
      <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>, 
      <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>, 
      <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>, 
      <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>, 
      <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>, 
      <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>
      <span class="xref-sep">,</span>
      <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>, 
      <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>, 
      <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>, 
      <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>
     </sup> and we will also add new validated molecular structures into this library with our research progresses. To date, there are no clinically approved vaccines or drugs specifically targeting SARS-CoV-2. Thus, with the hope to discover novel candidate drugs targeting SARS-CoV-2, we combine artificial intelligence (AI) with the structure-based drug design (SBDD) to accelerate generating potential lead compounds and design TCIs.
    </p>
    <p id="p-9">AI, especially deep learning, has been applied in predicting molecular properties
     <sup>
      <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>, 
      <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>, 
      <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>, 
      <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>
     </sup> and designing novel molecules
     <sup>
      <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>
     </sup>. Unlike earlier deep-learning molecular design by adding single atom one at a time
     <sup>
      <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a>, 
      <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>
     </sup>, our approach explores new molecules by adding a meaningful molecular fragment one by one, which is not only computationally more efficient but also chemically more reasonable. To make our AI model work well, the first step is to prepare the molecular fragment library as shown in 
     <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. We used our collected SARS-CoV 3CL
     <sup>pro</sup> inhibitors (284 molecules) as the initial molecule database targeting SARS-CoV-2 3CL
     <sup>pro</sup>. Then, we split this set of molecules into fragments with a molecule weight no more than 200 daltons. Both of the collected inhibitors and the fragments are supplied in 
     <a href="https://github.com/tbwxmu/2019-nCov">https://github.com/tbwxmu/2019-nCov</a>. Then we applied an advanced deep Q-learning network with the fragment-based drug design (ADQN-FBDD) for generating potential lead compounds. Noted, if researchers have enough experience and internal lead compounds or biased fragments, they can inject all such information into ADQN-FBDD by manually adding the lead compounds and biased fragments to the corresponding files. By using the same fragments directly from existing bioactivate molecules, our ADQN-FBDD agent can easily access the potential chemical space focused on 3CL
     <sup>pro</sup> of SARS-CoV-2.
    </p>
    <div id="F1" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Flowchart for our SARS-CoV-2 3CLpro lead compounds development." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1961549559" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Flowchart for our SARS-CoV-2 3CL&lt;;sup&gt;;pro&lt;;/sup&gt;; lead compounds development.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F1.medium.gif" width="440" height="236"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F1.medium.gif" width="440" height="236"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 1.</span>
      <p id="p-10" class="first-child">Flowchart for our SARS-CoV-2 3CL
       <sup>pro</sup> lead compounds development.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-11">After ADQN-FBDD automatically generated novel compounds targeting this new virus 3CL
     <sup>pro</sup>, we obtained a covalent lead compound library with 4,922 unique valid structures. In total, 47 of these compounds were selected with high scores from our AI-model’s reward function. Then these molecules were further evaluated by docking and covalent docking studies. The lead compound #
     <strong>46</strong> with a high covalent docking score attracts our attention, which also has a low difference between non-covalent and covalent docking pose among the 47 lead compounds. After carefully checking the lead #
     <strong>46</strong>’s interaction mode with 3CL
     <sup>pro</sup>, we believe there is still much space to optimize lead it. Then we designed a series of derivatives from compound #
     <strong>46</strong> based on our chemical biology knowledge and the structure-based optimization policy. All the generated molecular structures are published in our code library 
     <a href="https://github.com/tbwxmu/2019-nCov">https://github.com/tbwxmu/2019-nCov</a>. We encourage researchers who are interested in finding a potential treatment for this viral infection to synthesize and evaluate some of these molecules for treating COVID-19.
    </p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Results</h2>
    <p id="p-12">Integrating the double dueling deep Q learning with fixed q-targets and prioritized experience replay enables our agent ADQN stable and efficient during learning from the chemical environment. Combining the state-of-the-art AI algorithm with the idea of FBDD as presented in 
     <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, ADQN-FBDD is flexible and efficient to access the focused chemical space targeting the SARS-CoV-2 3CL
     <sup>pro</sup>. Based on the configurations targeting the SARS-CoV-2 3CL
     <sup>pro</sup>, ADQN-FBDD generated a potential lead library containing 4,922 unique molecular structures (Supplementary dataset in 
     <a href="https://github.com/tbwxmu/2019-nCov">https://github.com/tbwxmu/2019-nCov</a>). To narrow our focus to a smaller set of molecules for analysis, we elaborately defined filter rules (QED&gt;;0.1 and DRL-reward&gt;;=0.6) and the detailed information of the rules can be found in the Methods section. And then 47 unique molecules (Table S1) were kept for the next non-covalent docking and covalent-docking evaluation. These 47 virtual leads display suitable 3D-complexity with common characteristics of peptidomimetics and protein-protein interaction (PPI) inhibitors. They are mainly ranked by covalent-docking scores, considering covalent-docking also contains the scores of non-covalent docking.
     <sup>
      <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>
     </sup> And we also paid attention on the RMSD difference between the covalent binding and non-covalent binding poses based on all heavy atoms. Finally, we selected the lead molecule #
     <strong>46</strong> as displayed in 
     <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> with both a good covalent docking score and a small RMSD value (Table S1). We further optimized it and get a series of derivatives based on the SBDD approach.
    </p>
    <div id="F2" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="A: Framework of ADQN-FBDD. B: Examples for explaining the fragment-based actions. C: The solid lines represent taken actions including the addition/deletion of a different fragment or no modification during each step. The dashed lines represent actions that our reinforcement learning agent considered but did not make. After the first three actions to S1, S2, and S3, the fourth action to S4 was an exploratory action, meaning that it was taken even though another sibling action, the one colored in red dashed line leading to , was ranked higher. Exploratory action does not result in any learning, but each of our other actions does, resulting in updates as suggested by the purple curved arrow in which estimated values are moved up the tree from later nodes to earlier ones." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1961549559" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;A: Framework of ADQN-FBDD. B: Examples for explaining the fragment-based actions. C: The solid lines represent taken actions including the addition/deletion of a different fragment or no modification during each step. The dashed lines represent actions that our reinforcement learning agent considered but did not make. After the first three actions to S&lt;;sub&gt;;1&lt;;/sub&gt;;, S&lt;;sub&gt;;2&lt;;/sub&gt;;, and S&lt;;sub&gt;;3&lt;;/sub&gt;;, the fourth action to S&lt;;sub&gt;;4&lt;;/sub&gt;; was an exploratory action, meaning that it was taken even though another sibling action, the one colored in red dashed line leading to , was ranked higher. Exploratory action does not result in any learning, but each of our other actions does, resulting in updates as suggested by the purple curved arrow in which estimated values are moved up the tree from later nodes to earlier ones.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F2.medium.gif" width="440" height="205"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F2.medium.gif" width="440" height="205"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 2.</span>
      <p id="p-13" class="first-child">A: Framework of ADQN-FBDD. B: Examples for explaining the fragment-based actions. C: The solid lines represent taken actions including the addition/deletion of a different fragment or no modification during each step. The dashed lines represent actions that our reinforcement learning agent considered but did not make. After the first three actions to S
       <sub>1</sub>, S
       <sub>2</sub>, and S
       <sub>3</sub>, the fourth action to S
       <sub>4</sub> was an exploratory action, meaning that it was taken even though another sibling action, the one colored in red dashed line leading to 
       <span class="inline-formula" id="inline-formula-1">
        <span class="highwire-responsive-lazyload">
         <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/F2/embed/inline-graphic-1.gif"/>
         <noscript>
          <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/F2/embed/inline-graphic-1.gif"/>
         </noscript>
        </span>
       </span>, was ranked higher. Exploratory action does not result in any learning, but each of our other actions does, resulting in updates as suggested by the purple curved arrow in which estimated values are moved up the tree from later nodes to earlier ones.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F3" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Lead compound #46 generated by AI. A: Non-covalent molecular docking model of SARS-CoV-2 3CLpro (brown surface) with the bound lead compound #46 (magenta sticks). The triazole ring binds to the S1 subsite of the catalytic active center, the covalent fragment of [[#x3B1]], [[#x3B2]]-unsaturated aldehyde binds to S1[[#x2019]] subsite, [[#x3B2]]-lactam ring binds to S2 subsite and 5,7-dihydroxy chromone binds to S3 subsite. B: A two-dimensional (2D) view of non-bonding interaction of lead compound #46 in complex with 3CL protease based on non-covalent docking. The triazole ring, i.e., keto-amide, phenolic hydroxy forms hydrogen bond (H-bond) with His163, Glu166, and Thr190, respectively. C: Covalent docking model between compound #46 (green sticks) and 3CL protease (brown surface) exhibits a similar docking model as non-covalent docking. D: A 2D view of ligand interaction between compound #46 and protease under covalent docking. The triazole ring forms an H-bond with His163; the fragment of [[#x3B1]], [[#x3B2]]-unsaturated aldehyde forms a covalent bond with Cys145, i.e., the key residue in the catalytic center of the protease, resulting in covalent inhibition; aldehyde carbonyl forms an H-bond with His41; and besides, the hydroxyl of chromone at position 7 forms an H-bond with Thr190." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1961549559" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Lead compound #&lt;;strong&gt;;46&lt;;/strong&gt;; generated by AI. A: Non-covalent molecular docking model of SARS-CoV-2 3CL&lt;;sup&gt;;pro&lt;;/sup&gt;; (brown surface) with the bound lead compound #&lt;;strong&gt;;46 (&lt;;/strong&gt;;magenta sticks&lt;;strong&gt;;)&lt;;/strong&gt;;. The triazole ring binds to the S1 subsite of the catalytic active center, the covalent fragment of [[#x3B1]], [[#x3B2]]-unsaturated aldehyde binds to S1[[#x2019]] subsite, [[#x3B2]]-lactam ring binds to S2 subsite and 5,7-dihydroxy chromone binds to S3 subsite. B: A two-dimensional (2D) view of non-bonding interaction of lead compound #&lt;;strong&gt;;46&lt;;/strong&gt;; in complex with 3CL protease based on non-covalent docking. The triazole ring, i.e., keto-amide, phenolic hydroxy forms hydrogen bond (H-bond) with His163, Glu166, and Thr190, respectively. C: Covalent docking model between compound #&lt;;strong&gt;;46 (&lt;;/strong&gt;;green sticks&lt;;strong&gt;;)&lt;;/strong&gt;; and 3CL protease (brown surface) exhibits a similar docking model as non-covalent docking. D: A 2D view of ligand interaction between compound #&lt;;strong&gt;;46&lt;;/strong&gt;; and protease under covalent docking. The triazole ring forms an H-bond with His163; the fragment of [[#x3B1]], [[#x3B2]]-unsaturated aldehyde forms a covalent bond with Cys145, i.e., the key residue in the catalytic center of the protease, resulting in covalent inhibition; aldehyde carbonyl forms an H-bond with His41; and besides, the hydroxyl of chromone at position 7 forms an H-bond with Thr190.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F3.medium.gif" width="394" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F3.medium.gif" width="394" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 3.</span>
      <p id="p-14" class="first-child">Lead compound #
       <strong>46</strong> generated by AI. A: Non-covalent molecular docking model of SARS-CoV-2 3CL
       <sup>pro</sup> (brown surface) with the bound lead compound #
       <strong>46 (</strong>magenta sticks
       <strong>)</strong>. The triazole ring binds to the S1 subsite of the catalytic active center, the covalent fragment of α, β-unsaturated aldehyde binds to S1’ subsite, β-lactam ring binds to S2 subsite and 5,7-dihydroxy chromone binds to S3 subsite. B: A two-dimensional (2D) view of non-bonding interaction of lead compound #
       <strong>46</strong> in complex with 3CL protease based on non-covalent docking. The triazole ring, i.e., keto-amide, phenolic hydroxy forms hydrogen bond (H-bond) with His163, Glu166, and Thr190, respectively. C: Covalent docking model between compound #
       <strong>46 (</strong>green sticks
       <strong>)</strong> and 3CL protease (brown surface) exhibits a similar docking model as non-covalent docking. D: A 2D view of ligand interaction between compound #
       <strong>46</strong> and protease under covalent docking. The triazole ring forms an H-bond with His163; the fragment of α, β-unsaturated aldehyde forms a covalent bond with Cys145, i.e., the key residue in the catalytic center of the protease, resulting in covalent inhibition; aldehyde carbonyl forms an H-bond with His41; and besides, the hydroxyl of chromone at position 7 forms an H-bond with Thr190.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-15">Molecule #
     <strong>46</strong> was ranked number 1 based on the covalent docking score and its interaction model with the binding site was carefully checked as shown in 
     <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>. Although compound #
     <strong>46</strong> has the best covalent docking score, it has the alerting group aldehyde. Considering there is still much space for compound #
     <strong>46</strong> to fill in the S1’ subsite and α-ketoamides may be good to fit the oxyanion hole (
     <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>) of 3CL
     <sup>pro</sup>,
     <sup>
      <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>
     </sup> we replaced the aldehyde by formamide and also replaced the 1,4 Michael acceptors by alpha-ketoamides. Thus, we optimized compound #
     <strong>46</strong> to compound 
     <strong>46-14-1</strong>(
     <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4 A and B</a>).
    </p>
    <div id="F4" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="The covalent binding models of compound 46-14-1 and 46-14-2 in complex with the SARS-CoV-2 3CLpro. A: The covalent docking model between compound 46-14-1 (green sticks) with 3CL protease (yellow-orange surface). The oxyanion hole formed by the segment of [[#x3B1]]-ketoamides is shown in the red circle. B: The detailed view of the interactions between the compound 46-14-1 and 3CLpro. C: The covalent docking model between compound 46-14-2 with 3CL protease. The molecule 46-14-2 is shown as green sticks and the protein is shown as brown surface. D: The 2D view of the interactions between the compound 46-14-1 and 3CLpro." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1961549559" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;The covalent binding models of compound &lt;;strong&gt;;46-14-1&lt;;/strong&gt;; and &lt;;strong&gt;;46-14-2&lt;;/strong&gt;; in complex with the SARS-CoV-2 3CL&lt;;sup&gt;;pro&lt;;/sup&gt;;. A: The covalent docking model between compound &lt;;strong&gt;;46-14-1&lt;;/strong&gt;; (green sticks) with 3CL protease (yellow-orange surface). The oxyanion hole formed by the segment of [[#x3B1]]-ketoamides is shown in the red circle. B: The detailed view of the interactions between the compound &lt;;strong&gt;;46-14-1&lt;;/strong&gt;; and 3CL&lt;;sup&gt;;pro&lt;;/sup&gt;;. C: The covalent docking model between compound &lt;;strong&gt;;46-14-2&lt;;/strong&gt;; with 3CL protease. The molecule &lt;;strong&gt;;46-14-2&lt;;/strong&gt;; is shown as green sticks and the protein is shown as brown surface. D: The 2D view of the interactions between the compound &lt;;strong&gt;;46-14-1&lt;;/strong&gt;; and 3CL&lt;;sup&gt;;pro&lt;;/sup&gt;;.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F4.medium.gif" width="403" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F4.medium.gif" width="403" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 4.</span>
      <p id="p-16" class="first-child">The covalent binding models of compound 
       <strong>46-14-1</strong> and 
       <strong>46-14-2</strong> in complex with the SARS-CoV-2 3CL
       <sup>pro</sup>. A: The covalent docking model between compound 
       <strong>46-14-1</strong> (green sticks) with 3CL protease (yellow-orange surface). The oxyanion hole formed by the segment of α-ketoamides is shown in the red circle. B: The detailed view of the interactions between the compound 
       <strong>46-14-1</strong> and 3CL
       <sup>pro</sup>. C: The covalent docking model between compound 
       <strong>46-14-2</strong> with 3CL protease. The molecule 
       <strong>46-14-2</strong> is shown as green sticks and the protein is shown as brown surface. D: The 2D view of the interactions between the compound 
       <strong>46-14-1</strong> and 3CL
       <sup>pro</sup>.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-17">The non-bonding interaction between compound 
     <strong>46-14-1</strong> and SARS-CoV-2 3CL
     <sup>pro</sup> is mainly hydrogen bond. The carbonyl group of covalent scaffold α-ketoamide forms H-bonds with Leu141 and Gly143 as a hydrogen acceptor or a hydrogen donor. The hydrogen on the nitrogen atom of the triazole ring forms an H-bond with Glu166, while Glu166 also forms an H-bond with the carbonyl group on the main chain. The oxygen in the β-lactam ring forms an H-bond with His41. In order to enhance the polarity of the compounds, sulfonic groups were introduced to replace the ketone carbonyl groups on the main chain and sulfonamides 
     <strong>46-14-2</strong> were obtained. The covalent docking model of compound 
     <strong>46-14-2</strong> with SARS-CoV-2 3CL
     <sup>pro</sup> is shown in 
     <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4 C and D</a>.
    </p>
    <p id="p-18">In order to make the compound 
     <strong>46-14-2</strong> fit the active pocket with a higher affinity, we added a carbon atom to the sulfonic acid group of the original molecule, extending the carbon chain to increase the flexibility of the molecule, and obtained another optimized compound 
     <strong>46-14-3</strong>. The mode of covalent docking with SARS-CoV-2 3CL
     <sup>pro</sup> is shown in 
     <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>. Due to the introduction of carbon atoms and the enhancement of molecular flexibility, the β-lactam ring can be inserted deeper into the S2 pocket and the other fragments of the compound can better adapt to the S1, S1’ and S3 subsites. The α-carbonyl carbon on the α-ketoamide of compound 
     <strong>46-14-3</strong> forms a covalent bond with the key residue Cys145 on the protease, but the main non-bond interaction is still hydrogen bond (indicated by a yellow dash). The triazole ring mainly forms H-bonds with Phe140 and Glu166 residues in the S1 pocket; α-ketoamide covalent binding fragment mainly forms H-bonds with key amino acid residues Cys145, Gly143 and Ser144 in the S1’ subsite, which forms an oxyanion hole in the red circle in 
     <a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a>; the β-lactam side chain mainly forms H-bonds with residues Tyr54 and AsS187 in the S2 pocket; the chromone scaffold mainly forms H-bonds with key residues Thr190 and Gln192 in the S3 pocket. In addition, the oxygen on the sulfonyl group of the main chain forms an H-bond with Glu166 in the S1 pocket.
    </p>
    <div id="F5" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="The covalent binding model of Compound 46-14-3 in SARS-CoV-2 3CLpro. A: Surface representation of SARS-CoV-2 3CLpro (brown) complexed with 46-14-3 (green sticks). B: A stereo view showing 43-14-3 bound into the substrate-binding pocket of the SARS-CoV-2 3CLpro at 4 [[#xC5]]. The molecule 43-14-3 is shown as green sticks. Residues forming the H-bond are shown as yellow sticks. And the yellow dashes represent the H-bonds, as well as the oxyanion hole, is in the red circle region." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1961549559" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;The covalent binding model of Compound &lt;;strong&gt;;46-14-3&lt;;/strong&gt;; in SARS-CoV-2 3CL&lt;;sup&gt;;pro&lt;;/sup&gt;;. A: Surface representation of SARS-CoV-2 3CL&lt;;sup&gt;;pro&lt;;/sup&gt;; (brown) complexed with &lt;;strong&gt;;46-14-3&lt;;/strong&gt;; (green sticks). B: A stereo view showing &lt;;strong&gt;;43-14-3&lt;;/strong&gt;; bound into the substrate-binding pocket of the SARS-CoV-2 3CL&lt;;sup&gt;;pro&lt;;/sup&gt;; at 4 [[#xC5]]. The molecule &lt;;strong&gt;;43-14-3&lt;;/strong&gt;; is shown as green sticks. Residues forming the H-bond are shown as yellow sticks. And the yellow dashes represent the H-bonds, as well as the oxyanion hole, is in the red circle region.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F5.medium.gif" width="440" height="229"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F5.medium.gif" width="440" height="229"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 5.</span>
      <p id="p-19" class="first-child">The covalent binding model of Compound 
       <strong>46-14-3</strong> in SARS-CoV-2 3CL
       <sup>pro</sup>. A: Surface representation of SARS-CoV-2 3CL
       <sup>pro</sup> (brown) complexed with 
       <strong>46-14-3</strong> (green sticks). B: A stereo view showing 
       <strong>43-14-3</strong> bound into the substrate-binding pocket of the SARS-CoV-2 3CL
       <sup>pro</sup> at 4 Å. The molecule 
       <strong>43-14-3</strong> is shown as green sticks. Residues forming the H-bond are shown as yellow sticks. And the yellow dashes represent the H-bonds, as well as the oxyanion hole, is in the red circle region.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-20">The representatives were selected (
     <strong>46-14-1</strong>, 
     <strong>46-14-2</strong> and 
     <strong>46-14-3</strong> as 
     <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> displays), which may be further evaluated by molecular dynamics simulation to get the binding free energy and by quantum chemical calculation to get the reaction energy barrier. Meanwhile, 
     <strong>46-14-1</strong>, 
     <strong>46-14-2</strong> and 
     <strong>46-14-3</strong> are chosen as our candidates for chemical synthesis and anti-SARS-CoV-2 activity testing, which is ongoing.
    </p>
    <div id="F6" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Structures of optimized compounds." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1961549559" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Structures of optimized compounds.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F6.medium.gif" width="440" height="334"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F6.medium.gif" width="440" height="334"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 6.</span>
      <p id="p-21" class="first-child">Structures of optimized compounds.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
   </div>
   <div class="section" id="sec-3">
    <h2 class="">Discussions</h2>
    <p id="p-22">Computational approaches are particularly important for emerging diseases given the urgent need to provide timely solutions. In this work, our robust and efficient computational method and pipeline for designing compounds can provide useful drug candidates for treating SARS-CoV-2 infections. For more information about our AI model generated leads and SBDD optimized derivatives, please go to the library 
     <a href="https://github.com/tbwxmu/2019-nCov">https://github.com/tbwxmu/2019-nCov</a>. These candidates or their variants have a good chance to produce valid leads of anti-COVID-19 drugs. It is understood that the computational design requires experimental validations. While we are exploring experimental validations ourselves, we like to release these candidates promptly for other researchers to accelerate the development of anti-COVID-19 drugs given the emergency of seeking treatments for the disease.
    </p>
    <p id="p-23">Comparing with other deep RL methods, ADQN-FBDD has several highlights: 1) It directly modifies and generate molecular structures without format conversion problems while some other tools such as Insilico Medicine’s GENTRL (
     <a href="https://insilico.com">https://insilico.com</a>) may generate invalid SMILES output. 2) Most of the generative models require pre-training on a specific dataset and then produce the molecules with high similarities to a given training set. For example, the best molecule from Insilico Medicine for their target DDR1 was actually very similar to the kinase inhibitor Iclusig (ponatinib) on the market.
     <sup>
      <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>
     </sup> ADQN-FBDD does not need pre-training at all and has a capacity to generate novel molecules. 3) The process of generating molecules is very efficient and effective as ADQN-FBBD is molecular fragment-based growing with the knowledge of chemical reactions, while other models are all atom-based with no rules of chemical reactions at all
     <sup>
      <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">34</a>, 
      <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>, 
      <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>
     </sup>. 4) ADQN-FBDD is highly flexible and user-friendly for the medicinal chemists, who can easily inject their drug discovery experience into the reward function to guide the novel molecule generation. Our ADQN-FBDD and related pipeline can be used not only for designing anti-COVID-19 drugs but also other structure-based drug discoveries, especially for emerging infetious diseases that require treatments timely.
    </p>
   </div>
   <div class="section" id="sec-4">
    <h2 class="">Methods</h2>
    <div id="sec-5" class="subsection">
     <h3>Markov Decision Process for Molecule Generation</h3>
     <p id="p-24">Intuitively, the problem of chemical structure graph generation is formulated as learning a reinforced agent, which performs discrete actions of chemical reaction-based fragment addition or removal in a chemistry-aware Markov Decision Process (MDP). Formally, the components of MDP) include 
      <em>M</em> = {
      <em>S, A, P, R, γ</em>}, where each term is defined as follows:
     </p>
     <p id="p-25">
      <em>S</em> = {
      <em>S</em>
      <sub>
       <em>t</em>
      </sub>} denotes the state space containing all possible intermediate and final generated molecular graphs. Each 
      <em>s</em>
      <sub>
       <em>t</em>
      </sub> is a tuple of (
      <em>s</em>, 
      <em>t</em>). 
      <em>s</em> stands for a valid molecule structure and 
      <em>t</em> is the time step. For the initial state 
      <em>s</em>
      <sub>0</sub>, the structure can be represented as a specific core as 
      <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2C</a> indicates or randomly chosen from the prepared fragment library at time 
      <em>t</em> = 0. We also limit the maximum number of time steps 
      <em>T</em> in our molecule fragment based MDP, which defines the set of terminal states as {
      <em>s</em>
      <sub>
       <em>t</em>
      </sub>|
      <em>t</em>=
      <em>T</em>} containing all the states with the number of steps reaching the maximum allowed value 
      <em>T</em>.
     </p>
     <p id="p-26">Ac={
      <em>A</em>
      <sub>
       <em>t</em>
      </sub>} denotes a set of actions that describe the modification made on the current molecule structure at each time step 
      <em>t</em>. Here, each action can be classified into three categories: fragment addition, fragment deletion, and no modification.
     </p>
     <p id="p-27">
      <em>P</em> =
      <em>p</em>(
      <em>s</em>
      <sub>
       <em>t</em>+1
      </sub> |
      <em>s</em>
      <sub>
       <em>t</em>
      </sub> … 
      <em>s</em>
      <sub>0</sub>) =
      <em>p(s</em>
      <sub>
       <em>t+1</em>
      </sub>|
      <em>s</em>
      <sub>
       <em>t</em>
      </sub>, 
      <em>a
       <sub>t</sub>)
      </em> is the basic assumption in MDP. The state transition probability, which specifies the next possible state given the current state and action at time step t. Here, we define the state transition to be deterministic. For example, as 
      <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a> indicates S
      <sub>0</sub> to S
      <sub>1</sub>, by adding a 1
      <em>H</em>-1,2,3-triazol-4-yl fragment on 
      <em>S</em>
      <sub>
       <em>0</em>
      </sub>, the next state 
      <em>S</em>
      <sub>
       <em>1</em>
      </sub> will be the new structure of added 1
      <em>H</em>-1,2,3-triazol-4-yl with a probability of 1.
     </p>
     <p id="p-28">
      <em>R</em> is the reward function that specifies the reward after reaching state 
      <em>S</em>
      <sub>
       <em>t</em>
      </sub>
      <em>γ∊</em>(0,1] is the discount factor and typically γ = 0.9 in our study. In our framework, the state always has a valid and complete chemical structure at each step as 
      <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a> indicates. A reward is given not just at the terminal states, but at each step. Both intermediate rewards and final rewards are used to guide the behavior of the reinforcement learning (RL) agent. So, there is no delayed or sparse reward issue as many other reinforced frameworks suffered.
      <sup>
       <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>, 
       <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>
      </sup> Furthermore, to ensure that the last state is rewarded the most, we use 
      <em>γ</em>
      <sup>
       <em>T</em>−
       <em>t</em>
      </sup> to discount the value of the rewards at state 
      <em>s</em>
      <sub>
       <em>t</em>
      </sub>. In addition, our reward function can directly integrate the experience of medicinal chemists. For example, given a core of interest, medicinal chemists may add biased fragments of their interests and their input can be used to design a reward function that gives high reward signal values to biased fragments so that ADQN-FBDD may have a better chance to generate the desired structures.
     </p>
    </div>
    <div id="sec-6" class="subsection">
     <h3>Chemical Environment Design</h3>
     <p id="p-29">In our RL framework, the chemical environment receives action 
      <em>a</em>
      <sub>
       <em>t</em>
      </sub> from the agent and emits scalar reward r
      <sub>t</sub> and state 
      <span class="inline-formula" id="inline-formula-2">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/inline-graphic-2.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/inline-graphic-2.gif"/>
        </noscript>
       </span>
      </span> to the agent as 
      <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2A</a> shows. Note that the definition of the environment state is different from the general approach that the environment state is only the environment’s private representation invisible to the agent. We define the state of the chemical environment 
      <span class="inline-formula" id="inline-formula-3">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/inline-graphic-3.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/inline-graphic-3.gif"/>
        </noscript>
       </span>
      </span> as the intermediate generated molecule structure at time step t, which is fully observable by the RL agent. Simply, the environment’s state 
      <span class="inline-formula" id="inline-formula-4">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/inline-graphic-4.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/inline-graphic-4.gif"/>
        </noscript>
       </span>
      </span> is the agent’s state 
      <em>s</em>
      <sub>
       <em>t</em>+1
      </sub>. For the task of molecule generation, the environment incorporates rules of chemistry. In our study, chemistry rules are not only the basic chemical valency, but also the rules about adding and removing the fragments derived from known inhibitors based on chemical reactions. The detail information of 45 defined chemical reaction rules is presented in Table S2.
     </p>
    </div>
    <div id="sec-7" class="subsection">
     <h3>Agent Design</h3>
     <p id="p-30">As indicated in 
      <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2A</a>, the basic model of our ADQN-FBDD is an advanced Q-network. The goal of molecule generation is equally to fit a Q function 
      <em>Q</em>(
      <em>s</em>
      <sub>
       <em>t</em>
      </sub>, 
      <em>a</em>
      <sub>
       <em>t</em>
      </sub>) to make the agent choose the action 
      <em>a</em>
      <sub>
       <em>t</em>
      </sub> at state 
      <em>s</em>
      <sub>
       <em>t</em>
      </sub> that maximizes the future expected γ-discounted cumulative rewards with policy π. Mathematically, given the agent’s policy π, the value of the state-action pair Q
      <sup>π</sup>(
      <em>s</em>
      <sub>
       <em>t</em>
      </sub>, 
      <em>a</em>
      <sub>
       <em>t</em>
      </sub>) and the value of state V
      <sup>π</sup>(
      <em>s</em>
      <sub>
       <em>t</em>
      </sub>) are defined as, respectively: 
      <span class="disp-formula" id="disp-formula-2">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/graphic-8.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/graphic-8.gif"/>
        </noscript>
       </span>
      </span>
      <span class="disp-formula" id="disp-formula-3">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/graphic-9.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/graphic-9.gif"/>
        </noscript>
       </span>
      </span> where 
      <span class="inline-formula" id="inline-formula-5">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/inline-graphic-5.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/inline-graphic-5.gif"/>
        </noscript>
       </span>
      </span> is the expectation within policy π on state 
      <em>s</em>
      <sub>
       <em>t</em>
      </sub> taken 
      <em>a</em>
      <sub>
       <em>t</em>
      </sub> and R(
      <em>s</em>
      <sub>
       <em>n</em>
      </sub>, 
      <em>a</em>
      <sub>
       <em>n</em>
      </sub>) denotes the reward at step 
      <em>n</em>. Value function Q
      <sup>π</sup>(
      <em>s</em>
      <sub>
       <em>t</em>
      </sub>, 
      <em>a</em>
      <sub>
       <em>t</em>
      </sub>) measures the value of taking action 
      <em>a</em>
      <sub>
       <em>t</em>
      </sub> on state 
      <em>s</em>
      <sub>
       <em>t</em>
      </sub>. V
      <sup>π</sup>(
      <em>s</em>
      <sub>
       <em>t</em>
      </sub>) is the value of being at state 
      <em>s</em>
      <sub>
       <em>t</em>
      </sub> means how good to be in this state. Obviously, V
      <sup>π</sup>(s
      <sub>t</sub>) can be seen as a part of Q
      <sup>π</sup>(
      <em>s</em>
      <sub>
       <em>t</em>
      </sub>, 
      <em>a</em>
      <sub>
       <em>t</em>
      </sub>). Then, the rest part from Q
      <sup>π</sup>(
      <em>s</em>
      <sub>
       <em>t</em>
      </sub>, 
      <em>a</em>
      <sub>
       <em>t</em>
      </sub>) can be defined as the so-called advantage function A
      <sup>π</sup>(
      <em>s</em>
      <sub>
       <em>t</em>
      </sub>, 
      <em>a</em>
      <sub>
       <em>t</em>
      </sub>)
      <sup>
       <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>
      </sup> as: 
      <span class="disp-formula" id="disp-formula-4">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/graphic-10.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/graphic-10.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-31">Intuitively, the advantage value shows how advantageous selecting the action is relative to the others at the same given state. Then 
      <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">Eqn. (2)</a> can be rewritten according to 
      <a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">Eqn. (4)</a>: 
      <span class="disp-formula" id="disp-formula-5">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/graphic-11.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/graphic-11.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-32">Obviously, 
      <span class="inline-formula" id="inline-formula-6">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/inline-graphic-6.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/inline-graphic-6.gif"/>
        </noscript>
       </span>
      </span>. To avoid the issue of identifiability, we deduct the mean value from the prediction and the Q-function of dueling DQN can be defined as: 
      <span class="disp-formula" id="disp-formula-6">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/graphic-12.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/graphic-12.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-33">Note that 
      <em>Θ, α</em> and 
      <em>β</em> come from the dueling Q-network as 
      <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a> indicates. |Ac| is the size of action space and 
      <span class="inline-formula" id="inline-formula-7">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/inline-graphic-7.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/inline-graphic-7.gif"/>
        </noscript>
       </span>
      </span>. To make our RL agent more robust to be more stable learning and to handle the problem of the overestimation of Q-values, double Q-network
      <sup>
       <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>
      </sup> and fixed Q-targets
      <sup>
       <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>
      </sup> are also incorporated: 
      <span class="disp-formula" id="disp-formula-7">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/graphic-13.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/graphic-13.gif"/>
        </noscript>
       </span>
      </span> where, TD is the temporal-difference; 
      <span class="inline-formula" id="inline-formula-8">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/inline-graphic-8.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/inline-graphic-8.gif"/>
        </noscript>
       </span>
      </span> is another dueling DQN network as the target network and its parameters (
      <em>Θ′, α′, β′</em>) keep fixed and copy from the dueling DQN Q
      <sup>π</sup> every 
      <em>m</em> steps (
      <em>m</em>=20 as we used). To update the parameters (
      <em>Θ, α, β</em>) from the dueling DQN as 
      <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a> displays, we can train our RL agent by minimizing the loss function: 
      <span class="disp-formula" id="disp-formula-8">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/graphic-14.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/graphic-14.gif"/>
        </noscript>
       </span>
      </span> where 
      <strong>
       <em>E</em>
      </strong> is the expectation. As the L2 loss has a disadvantage of the tendency to be dominated by outliers, we use the Huber loss as the loss function 
      <em>f</em>
      <sub>
       <em>l</em>
      </sub>: 
      <span class="disp-formula" id="disp-formula-9">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/graphic-15.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/graphic-15.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <div id="F7" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F7.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Architecture of used advance deep Q-learning network (ADQN). TD is the temporal difference error." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1961549559" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Architecture of used advance deep Q-learning network (ADQN). TD is the temporal difference error.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F7.medium.gif" width="440" height="218"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 7." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F7.medium.gif" width="440" height="218"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 7.</span>
       <p id="p-34" class="first-child">Architecture of used advance deep Q-learning network (ADQN). TD is the temporal difference error.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-8" class="subsection">
     <h3>Prioritized Experience Replay</h3>
     <p id="p-35">Prioritized Experience Replay (PER)
      <sup>
       <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>
      </sup> is a technique to enable reinforcement learning agents remember and reuse experience from the past, and to replay important transitions more frequently. PER is very useful for replaying some less frequent experiences. Here, we use the same code of “Prioritized Replay Buffer” from open AI’s gym with version 0.15.4.
      <sup>
       <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>
      </sup> Finally, our RL agent is the double dueling deep Q learning with fixed q-targets and prioritized experience replay.
     </p>
    </div>
    <div id="sec-9" class="subsection">
     <h3>Fragment Library Design</h3>
     <p id="p-36">The fragment-based approach to drug discovery (FBDD) has been established as an efficient tool in the search for new drugs.
      <sup>
       <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>
      </sup> The idea of FBDD is that proper optimization of each unique interaction in the binding site and subsequent incorporation into a single molecular entity should produce a compound with a binding affinity that is the sum of the individual interactions. However, the widely used fragment libraries only consider the diversity of fragments, such as the ZINC fragment database. They have a very low probability to exhibit the desired bioactivity for a given protein.
     </p>
     <p id="p-37">To combine the idea of FBDD with our RL framework, we first collected and built a SARS-CoV-2 3CL
      <sup>pro</sup> inhibitor dataset containing 284 reported molecules. Then using the improved BRICS algorithm
      <sup>
       <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>
      </sup> to split those molecules to get the fragment library target on SARS-CoV-2 3CL
      <sup>pro</sup> as the flowchart displayed in 
      <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>’s yellow box. An elaborate filtering cascade accompanied by manual inspection and the rules can be changed based on the needs in different studies. Our fragment library contains 316 fragments with molecular weight &lt;;200 daltons, the minimum number non-hydrogen atom &gt;;1 and the maximum ≤ 25. The fragments directly from the existing inhibitors based on the chemical retrosynthetic rules are always true substructures and may have a high quality of bioactivity targeting 3CL
      <sup>pro</sup>. It is worth noting that the quality of the designed fragment library directly affects the behavior of the chemical environment of ADQN-FBDD.
     </p>
    </div>
    <div id="sec-10" class="subsection">
     <h3>Core Selection</h3>
     <p id="p-38">Studies have identified various scaffolds or core structures that have privileged characteristics in terms of the activity of a certain target.
      <sup>
       <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>, 
       <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>, 
       <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>
      </sup> Core structure selection is the starting point in a scaffold-based drug discovery. Choosing or designing a proper initial scaffold is never trivial, and medicinal chemists may need enough experience to get such a skill. Luckily, there are serval reported privileged core structures targeting SARS M
      <sup>pro</sup>.
      <sup>
       <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">3</a>, 
       <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>
      </sup> Here, we chose 4-aminopent-2-enal and 3-amino-2-oxobutanal as the starting cores as 
      <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a> displays, because both cores have been validated to generate covalent bonds with the Cys145 of SARS or SARS-CoV-2 3CL
      <sup>pro</sup>.
     </p>
     <div id="F8" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F8.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Structures of the chosen cores" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1961549559" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Structures of the chosen cores&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F8.medium.gif" width="440" height="227"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 8." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F8.medium.gif" width="440" height="227"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 8.</span>
       <p id="p-39" class="first-child">Structures of the chosen cores</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-11" class="subsection">
     <h3>Reward Design</h3>
     <p id="p-40">Most reported RL methods using the complete structure information of a positive drug or inhibitor as the template,
      <sup>
       <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">38</a>, 
       <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">39</a>
      </sup> and they design a reward function for the RL agent to learn to regenerate the template structure or generate highly similar structures to the template. This way may be useful in testing the performance of RL methods but not suitable in a real-world drug design because no one knows the complete structural information of the novel molecule. A more practical approach is to learn the structure features of existing drugs or inhibitors to local, focused chemical space for a specific protein target. Instead of simply putting attention on the diversity of molecule structure, we explore the possibility of generating novel molecules based on the existing knowledge. Here, we designed a deep reinforcement learning reward (DRL-reward) function that consists of the final property score, containing specific fragments (CSF) score and pharmacophore score as: 
      <span class="disp-formula" id="disp-formula-10">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/graphic-18.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/graphic-18.gif"/>
        </noscript>
       </span>
      </span>
      <span class="disp-formula" id="disp-formula-11">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/graphic-19.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/graphic-19.gif"/>
        </noscript>
       </span>
      </span> where 
      <em>w</em>
      <sub>
       <em>pro</em>
      </sub> represents weight for the quantitative estimate of drug-likeness (QED) property and its default value is 0.1. 
      <em>f</em>
      <sub>
       <em>pro</em>
      </sub> stands for the QED. QED values can range from 0 (all properties are unfavourable) to 1 (all properties are favourable), which are calculated by eight molecular properties.
      <sup>
       <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>
      </sup> The score function 
      <em>f</em>
      <sub>
       <em>csf</em>
      </sub> of containing specific fragments (csf) is: 
      <span class="disp-formula" id="disp-formula-12">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/graphic-20.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/graphic-20.gif"/>
        </noscript>
       </span>
      </span>
      <span class="disp-formula" id="disp-formula-13">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/graphic-21.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/graphic-21.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-41">The binding site of SARS-CoV-2 3CL
      <sup>pro</sup> (
      <a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure 9</a>) is commonly divided into the catalytic activity center (His41 and Cys145, specified as S1’) and several subsites, defined as S1 (His163, Glu166, Phe140, Leu141 and Asn142), S1’ (His41, Cys145, Gly143 and Ser144), S2 (Tyr54, Asp187, His41, Arg188, His164 and Met49), and S3 (Thr190, Gln192, Glu166, Met165, Leu167 and Gln189). Each subsite may have its own favorable binding fragment. When generated structures (including the intermediates) containing those favorable fragments, 
      <em>f</em>
      <sub>
       <em>csf</em>
      </sub> is equivalent to give an additional reward to our RL agent. 
      <em>w</em>
      <sub>
       <em>con</em>
      </sub> controls the contribution of the biased fragment to the reward signal and the default value is 0.6. 
      <em>n</em>
      <sub>
       <em>match</em>
      </sub> stands for how many biased fragments have been matched in one generated structure. 
      <em>N</em>
      <sub>
       <em>total</em>
      </sub> is the number of biased fragments defined based on our knowledge learned from related work. 
      <em>f</em>
      <sub>
       <em>pha</em>
      </sub> represents the score function of pharmacophores, which mainly depends on the ligand-protein interaction mode from the crystal structure (PDB ID: 6LU7): 
      <span class="disp-formula" id="disp-formula-14">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/graphic-22.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/03/08/2020.03.03.972133/embed/graphic-22.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <div id="F9" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F9.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Binding site of SARS-CoV-2 3CLpro. A: The subsites that complement the substrate-binding pocket are shown as surface, which are assigned as S1 (green), S1[[#x2019]] (red), S2 (magenta), and S3(cyan). B: The key residues of the binding site are displayed by green, red, magenta and cyan lines (S1, S1[[#x2019]], S2 and S3). The pictures of the binding site are generated using PyMol ((http://www.pymol.org/)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1961549559" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Binding site of SARS-CoV-2 3CL&lt;;sup&gt;;pro&lt;;/sup&gt;;. A: The subsites that complement the substrate-binding pocket are shown as surface, which are assigned as S1 (green), S1[[#x2019]] (red), S2 (magenta), and S3(cyan). B: The key residues of the binding site are displayed by green, red, magenta and cyan lines (S1, S1[[#x2019]], S2 and S3). The pictures of the binding site are generated using PyMol ((http://www.pymol.org/).&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 9." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F9.medium.gif" width="440" height="224"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 9." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F9.medium.gif" width="440" height="224"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/08/2020.03.03.972133/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 9.</span>
       <p id="p-42" class="first-child">Binding site of SARS-CoV-2 3CL
        <sup>pro</sup>. A: The subsites that complement the substrate-binding pocket are shown as surface, which are assigned as S1 (green), S1’ (red), S2 (magenta), and S3(cyan). B: The key residues of the binding site are displayed by green, red, magenta and cyan lines (S1, S1’, S2 and S3). The pictures of the binding site are generated using PyMol ((
        <a href="http://www.pymol.org/">http://www.pymol.org/</a>).
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-43">The pharmacophores plot was added in Figure S5. 
      <em>w</em>
      <sub>
       <em>pha</em>
      </sub> controls the contribution of the pharmacophore score to the reward and the default value is 0.4.
     </p>
    </div>
    <div id="sec-12" class="subsection">
     <h3>Molecular Generation and Selection</h3>
     <p id="p-44">As discussed in the above reward design part, our reward function considers the molecular descriptor thresholds (QED&gt;;0.1), the defined pharmacophores and biased fragments. In total, 4,922 unique valid structures were automatically generated and all matched the defined rules by using ADQN-FBDD without any pre-training as many other methods need.
      <sup>
       <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">35</a>, 
       <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">40</a>, 
       <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">41</a>, 
       <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>, 
       <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>
      </sup> Next, All the molecules with high deep reinforcement learning scores (DRL score: R(S)&gt;;0.6) were kept (47 molecules). Then, these 47 unique molecules were prepared to generate at least 1 conformation with the local energy minimization using the OPLS-2005 force field by the “ligand prepare” module of Schrödinger 2015 software. The 47 unique molecules generated a total of 163 3D conformations before docking into the substrate-binding site of SARS-CoV-2 3CL
      <sup>pro</sup>. Considering the balance between precision and calculation time, the stand-precision (SP) Glide
      <sup>
       <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>
      </sup> was firstly used to predict the possible non-covalent binding poses in this binding site and the binding site grid centered on the original ligand N3
      <sup>
       <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>
      </sup> with 20 Å buffer dimensions. Following this non-covalent docking, we also calculated the covalent docking poses and scores for those 47 molecules. We reordered the docking results mainly based on the covalent docking score and the RMSD value from covalent docking pose to non-covalent (Table S1).
     </p>
    </div>
   </div>
   <div class="section" id="sec-14">
    <h2 class="">Supplementary information</h2>
    <p id="p-45">Supplementary dataset and additional information of this paper can be found at 
     <a href="https://github.com/tbwxmu/2019-nCov">https://github.com/tbwxmu/2019-nCov</a>.
    </p>
   </div>
   <div class="section" id="sec-15">
    <h2 class="">Authors’ contributions</h2>
    <p id="p-47">BT and DX designed the study. BT developed the AI-aided methods and wrote the manuscript. All authors contributed to the interpretation of results. All authors reviewed and edited the manuscript. All authors read and approved the final manuscript.</p>
   </div>
   <div class="section" id="sec-16">
    <h2 class="">Data and Code availability</h2>
    <p id="p-48">The code for the ADQN-FBDD and related data in this paper will be available at 
     <a href="https://github.com/tbwxmu/2019-nCov">https://github.com/tbwxmu/2019-nCov</a> upon acceptance of this paper for journal publication.
    </p>
   </div>
   <div class="section COI-statement" id="sec-17">
    <h2 class="">Competing interests</h2>
    <p id="p-49">The authors declare that they have no competing interests.</p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgements</h2>
    <p id="p-46">We would like to thank Prof. Dongqing Wei’s group and the PCL lab for their generous support of high-performance computing resources. BT’s work was funded by the program of China Scholarships Council No. 201806310017. DX’s effort was supported by the US National Institutes of Health grant R35-GM126985.</p>
   </div>
   <div class="section fn-group" id="fn-group-1">
    <h2>Footnotes</h2>
    <ul>
     <li class="fn-dataset fn-group-external-links" id="fn-1">
      <p id="p-3">
       <a href="https://github.com/tbwxmu/2019-nCov">https://github.com/tbwxmu/2019-nCov</a>
      </p>
     </li>
    </ul>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References:</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gorbalenya</span>
          <span class="cit-name-given-names">AE.</span>
         </span>
         <span class="cit-article-title">Severe acute respiratory syndrome-related coronavirus–The species and its viruses, a statement of the Coronavirus Study Group</span>. 
         <abbr class="cit-jnl-abbrev">BioRxiv</abbr>, (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.03.03.972133v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">Coronavirus COVID-19 Global Cases by Johns Hopkins CSSE</span>, available from: 
         <a href="https://forum.truckersmp.com/index.php?/topic/93003-coronavirus-covid-19-global-cases-by-johns-hopkins-csse/&amp;;tab=comments">https://forum.truckersmp.com/index.php?/topic/93003-coronavirus-covid-19-global-cases-by-johns-hopkins-csse/&amp;;tab=comments</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Alpha-ketoamides as broad-spectrum inhibitors of coronavirus and enterovirus replication Structure-based design, synthesis, and activity assessment</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Medicinal Chemistry</abbr>, (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">卢洪洲</span>
          <span class="cit-name-given-names">陈凌席刘李李尚王沈</span>
         </span>. 
         <span class="cit-article-title">洛匹那韦利托那韦和阿比多尔用于治疗新型冠状病毒肺炎的有效性研究</span>. 
         <abbr class="cit-jnl-abbrev">中华传染病杂志</abbr>
         <span class="cit-vol">38</span>, (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">X-J.</span>
         </span>
         <span class="cit-article-title">Potential inhibitors for 2019-nCoV coronavirus M protease from clinically approved medicines</span>. 
         <abbr class="cit-jnl-abbrev">bioRxiv</abbr>, (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Design of wide-spectrum inhibitors targeting coronavirus main proteases</span>. 
         <abbr class="cit-jnl-abbrev">PLoS biology</abbr>
         <span class="cit-vol">3</span>, (
         <span class="cit-pub-date">2005</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.7" data-doi="10.1038/nrd3410">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Singh</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Petter</span>
          <span class="cit-name-given-names">RC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Baillie</span>
          <span class="cit-name-given-names">TA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Whitty</span>
          <span class="cit-name-given-names">A.</span>
         </span>
         <span class="cit-article-title">The resurgence of covalent drugs</span>. 
         <abbr class="cit-jnl-abbrev">Nature reviews Drug discovery</abbr>
         <span class="cit-vol">10</span>, 
         <span class="cit-fpage">307</span>–
         <span class="cit-lpage">317</span> (
         <span class="cit-pub-date">2011</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BDrug%2Bdiscovery%26rft.stitle%253DNat%2BRev%2BDrug%2BDiscov%26rft.aulast%253DSingh%26rft.auinit1%253DJ.%26rft.volume%253D10%26rft.issue%253D4%26rft.spage%253D307%26rft.epage%253D317%26rft.atitle%253DThe%2Bresurgence%2Bof%2Bcovalent%2Bdrugs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrd3410%26rft_id%253Dinfo%253Apmid%252F21455239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nrd3410&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21455239&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F08%2F2020.03.03.972133.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.8">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Tuley</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fast</span>
          <span class="cit-name-given-names">W.</span>
         </span>
         <span class="cit-article-title">The taxonomy of covalent inhibitors</span>. 
         <abbr class="cit-jnl-abbrev">Biochemistry</abbr>
         <span class="cit-vol">57</span>, 
         <span class="cit-fpage">3326</span>–
         <span class="cit-lpage">3337</span> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBiochemistry%26rft.volume%253D57%26rft.spage%253D3326%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.9" data-doi="10.1021/jm0494873">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jain</span>
          <span class="cit-name-given-names">RP</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Synthesis and evaluation of keto-glutamine analogues as potent inhibitors of severe acute respiratory syndrome 3CLpro</span>. 
         <abbr class="cit-jnl-abbrev">Journal of medicinal chemistry</abbr>
         <span class="cit-vol">47</span>, 
         <span class="cit-fpage">6113</span>–
         <span class="cit-lpage">6116</span> (
         <span class="cit-pub-date">2004</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmedicinal%2Bchemistry%26rft.stitle%253DJ%2BMed%2BChem%26rft.aulast%253DJain%26rft.auinit1%253DR.%2BP.%26rft.volume%253D47%26rft.issue%253D25%26rft.spage%253D6113%26rft.epage%253D6116%26rft.atitle%253DSynthesis%2Band%2Bevaluation%2Bof%2Bketo-glutamine%2Banalogues%2Bas%2Bpotent%2Binhibitors%2Bof%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome%2B3CLpro.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fjm0494873%26rft_id%253Dinfo%253Apmid%252F15566280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1021/jm0494873&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15566280&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F08%2F2020.03.03.972133.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.10" data-doi="10.1073/pnas.0403596101">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>
          <span class="cit-name-given-names">C-Y</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Small molecules targeting severe acute respiratory syndrome human coronavirus</span>. 
         <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr>
         <span class="cit-vol">101</span>, 
         <span class="cit-fpage">10012</span>–
         <span class="cit-lpage">10017</span> (
         <span class="cit-pub-date">2004</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0403596101%26rft_id%253Dinfo%253Apmid%252F15226499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzI3LzEwMDEyIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMDgvMjAyMC4wMy4wMy45NzIxMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.11" data-doi="10.1021/jm050548m">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ghosh</span>
          <span class="cit-name-given-names">AK</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Design and synthesis of peptidomimetic severe acute respiratory syndrome chymotrypsin-like protease inhibitors</span>. 
         <abbr class="cit-jnl-abbrev">Journal of medicinal chemistry</abbr>
         <span class="cit-vol">48</span>, 
         <span class="cit-fpage">6767</span>–
         <span class="cit-lpage">6771</span> (
         <span class="cit-pub-date">2005</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmedicinal%2Bchemistry%26rft.stitle%253DJ%2BMed%2BChem%26rft.aulast%253DGhosh%26rft.auinit1%253DA.%2BK.%26rft.volume%253D48%26rft.issue%253D22%26rft.spage%253D6767%26rft.epage%253D6771%26rft.atitle%253DDesign%2Band%2Bsynthesis%2Bof%2Bpeptidomimetic%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome%2Bchymotrypsin-like%2Bprotease%2Binhibitors.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fjm050548m%26rft_id%253Dinfo%253Apmid%252F16250632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1021/jm050548m&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16250632&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F08%2F2020.03.03.972133.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000233017300001&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.12" data-doi="10.1021/jm050184y">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Shie</span>
          <span class="cit-name-given-names">J-J</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Discovery of potent anilide inhibitors against the severe acute respiratory syndrome 3CL protease</span>. 
         <abbr class="cit-jnl-abbrev">Journal of medicinal chemistry</abbr>
         <span class="cit-vol">48</span>, 
         <span class="cit-fpage">4469</span>–
         <span class="cit-lpage">4473</span> (
         <span class="cit-pub-date">2005</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmedicinal%2Bchemistry%26rft.stitle%253DJ%2BMed%2BChem%26rft.aulast%253DShie%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D48%26rft.issue%253D13%26rft.spage%253D4469%26rft.epage%253D4473%26rft.atitle%253DDiscovery%2Bof%2Bpotent%2Banilide%2Binhibitors%2Bagainst%2Bthe%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome%2B3CL%2Bprotease.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fjm050184y%26rft_id%253Dinfo%253Apmid%252F15974598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1021/jm050184y&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15974598&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F08%2F2020.03.03.972133.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.13">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Shie</span>
          <span class="cit-name-given-names">J-J</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Inhibition of the severe acute respiratory syndrome 3CL protease by peptidomimetic α, β-unsaturated esters</span>. 
         <abbr class="cit-jnl-abbrev">Bioorganic &amp;; medicinal chemistry</abbr>
         <span class="cit-vol">13</span>, 
         <span class="cit-fpage">5240</span>–
         <span class="cit-lpage">5252</span> (
         <span class="cit-pub-date">2005</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioorganic%2B%2526%2Bmedicinal%2Bchemistry%26rft.volume%253D13%26rft.spage%253D5240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Al-Gharabli</span>
          <span class="cit-name-given-names">SI</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">An efficient method for the synthesis of peptide aldehyde libraries employed in the discovery of reversible SARS coronavirus main protease (SARS-CoV Mpro) inhibitors</span>. 
         <abbr class="cit-jnl-abbrev">ChemBioChem</abbr>
         <span class="cit-vol">7</span>, 
         <span class="cit-fpage">1048</span>–
         <span class="cit-lpage">1055</span> (
         <span class="cit-pub-date">2006</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DChembiochem%2B%253A%2B%2Ba%2BEuropean%2Bjournal%2Bof%2Bchemical%2Bbiology%26rft.stitle%253DChembiochem%26rft.aulast%253DAl-Gharabli%26rft.auinit1%253DS.%2BI.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253D1048%26rft.epage%253D1055%26rft.atitle%253DAn%2Befficient%2Bmethod%2Bfor%2Bthe%2Bsynthesis%2Bof%2Bpeptide%2Baldehyde%2Blibraries%2Bemployed%2Bin%2Bthe%2Bdiscovery%2Bof%2Breversible%2BSARS%2Bcoronavirus%2Bmain%2Bprotease%2B%2528SARS-CoV%2BMpro%2529%2Binhibitors.%26rft_id%253Dinfo%253Apmid%252F16688706%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=16688706&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F08%2F2020.03.03.972133.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.15" data-doi="10.1021/jm060207o">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lu</span>
          <span class="cit-name-given-names">I-L</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Structure-based drug design and structural biology study of novel nonpeptide inhibitors of severe acute respiratory syndrome coronavirus main protease</span>. 
         <abbr class="cit-jnl-abbrev">Journal of medicinal chemistry</abbr>
         <span class="cit-vol">49</span>, 
         <span class="cit-fpage">5154</span>–
         <span class="cit-lpage">5161</span> (
         <span class="cit-pub-date">2006</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmedicinal%2Bchemistry%26rft.stitle%253DJ%2BMed%2BChem%26rft.aulast%253DLu%26rft.auinit1%253DI.%2BL.%26rft.volume%253D49%26rft.issue%253D17%26rft.spage%253D5154%26rft.epage%253D5161%26rft.atitle%253DStructure-based%2Bdrug%2Bdesign%2Band%2Bstructural%2Bbiology%2Bstudy%2Bof%2Bnovel%2Bnonpeptide%2Binhibitors%2Bof%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome%2Bcoronavirus%2Bmain%2Bprotease.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fjm060207o%26rft_id%253Dinfo%253Apmid%252F16913704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1021/jm060207o&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16913704&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F08%2F2020.03.03.972133.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.16" data-doi="10.1021/jm050852f">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Tsai</span>
          <span class="cit-name-given-names">K-C</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Discovery of a novel family of SARS-CoV protease inhibitors by virtual screening and 3D-QSAR studies</span>. 
         <abbr class="cit-jnl-abbrev">Journal of medicinal chemistry</abbr>
         <span class="cit-vol">49</span>, 
         <span class="cit-fpage">3485</span>–
         <span class="cit-lpage">3495</span> (
         <span class="cit-pub-date">2006</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmedicinal%2Bchemistry%26rft.stitle%253DJ%2BMed%2BChem%26rft.aulast%253DTsai%26rft.auinit1%253DK.%2BC.%26rft.volume%253D49%26rft.issue%253D12%26rft.spage%253D3485%26rft.epage%253D3495%26rft.atitle%253DDiscovery%2Bof%2Ba%2Bnovel%2Bfamily%2Bof%2BSARS-CoV%2Bprotease%2Binhibitors%2Bby%2Bvirtual%2Bscreening%2Band%2B3D-QSAR%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fjm050852f%26rft_id%253Dinfo%253Apmid%252F16759091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1021/jm050852f&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16759091&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F08%2F2020.03.03.972133.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">17.</span>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.17" data-doi="10.1016/j.chembiol.2005.12.008">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>
          <span class="cit-name-given-names">C-Y</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Stable benzotriazole esters as mechanism-based inactivators of the severe acute respiratory syndrome 3CL protease</span>. 
         <abbr class="cit-jnl-abbrev">Chemistry &amp;; biology</abbr>
         <span class="cit-vol">13</span>, 
         <span class="cit-fpage">261</span>–
         <span class="cit-lpage">268</span> (
         <span class="cit-pub-date">2006</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DChemistry%2B%2526%2Bbiology%26rft.stitle%253DChem%2BBiol%26rft.aulast%253DWu%26rft.auinit1%253DC.%2BY.%26rft.volume%253D13%26rft.issue%253D3%26rft.spage%253D261%26rft.epage%253D268%26rft.atitle%253DStable%2Bbenzotriazole%2Besters%2Bas%2Bmechanism-based%2Binactivators%2Bof%2Bthe%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome%2B3CL%2Bprotease.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chembiol.2005.12.008%26rft_id%253Dinfo%253Apmid%252F16638531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.chembiol.2005.12.008&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16638531&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F08%2F2020.03.03.972133.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">18.</span>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.18">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Akaji</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Konno</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Onozuka</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Makino</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Saito</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nosaka</span>
          <span class="cit-name-given-names">K.</span>
         </span>
         <span class="cit-article-title">Evaluation of peptide-aldehyde inhibitors using R188I mutant of SARS 3CL protease as a proteolysis-resistant mutant</span>. 
         <abbr class="cit-jnl-abbrev">Bioorganic &amp;; medicinal chemistry</abbr>
         <span class="cit-vol">16</span>, 
         <span class="cit-fpage">9400</span>–
         <span class="cit-lpage">9408</span> (
         <span class="cit-pub-date">2008</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioorganic%2B%2526%2Bmedicinal%2Bchemistry%26rft.volume%253D16%26rft.spage%253D9400%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">19.</span>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.19">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ghosh</span>
          <span class="cit-name-given-names">AK</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Design, synthesis and antiviral efficacy of a series of potent chloropyridyl ester-derived SARS-CoV 3CLpro inhibitors</span>. 
         <abbr class="cit-jnl-abbrev">Bioorganic &amp;; medicinal chemistry letters</abbr>
         <span class="cit-vol">18</span>, 
         <span class="cit-fpage">5684</span>–
         <span class="cit-lpage">5688</span> (
         <span class="cit-pub-date">2008</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioorganic%2B%2526%2Bmedicinal%2Bchemistry%2Bletters%26rft.volume%253D18%26rft.spage%253D5684%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">20.</span>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.20">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Shao</span>
          <span class="cit-name-given-names">Y-M</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Design, synthesis, and evaluation of trifluoromethyl ketones as inhibitors of SARS-CoV 3CL protease</span>. 
         <abbr class="cit-jnl-abbrev">Bioorganic &amp;; medicinal chemistry</abbr>
         <span class="cit-vol">16</span>, 
         <span class="cit-fpage">4652</span>–
         <span class="cit-lpage">4660</span> (
         <span class="cit-pub-date">2008</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioorganic%2B%2526%2Bmedicinal%2Bchemistry%26rft.volume%253D16%26rft.spage%253D4652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">21.</span>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.21" data-doi="10.1016/j.febslet.2008.12.059">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kuo</span>
          <span class="cit-name-given-names">C-J</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Individual and common inhibitors of coronavirus and picornavirus main proteases</span>. 
         <abbr class="cit-jnl-abbrev">FEBS letters</abbr>
         <span class="cit-vol">583</span>, 
         <span class="cit-fpage">549</span>–
         <span class="cit-lpage">555</span> (
         <span class="cit-pub-date">2009</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DKuo%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D583%26rft.issue%253D3%26rft.spage%253D549%26rft.epage%253D555%26rft.atitle%253DIndividual%2Band%2Bcommon%2Binhibitors%2Bof%2Bcoronavirus%2Band%2Bpicornavirus%2Bmain%2Bproteases.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.febslet.2008.12.059%26rft_id%253Dinfo%253Apmid%252F19166843%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.febslet.2008.12.059&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19166843&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F08%2F2020.03.03.972133.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">22.</span>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.22">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ramajayam</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tan</span>
          <span class="cit-name-given-names">K-P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>
          <span class="cit-name-given-names">H-G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Liang</span>
          <span class="cit-name-given-names">P-H.</span>
         </span>
         <span class="cit-article-title">Synthesis and evaluation of pyrazolone compounds as SARS-coronavirus 3C-like protease inhibitors</span>. 
         <abbr class="cit-jnl-abbrev">Bioorganic &amp;; medicinal chemistry</abbr>
         <span class="cit-vol">18</span>, 
         <span class="cit-fpage">7849</span>–
         <span class="cit-lpage">7854</span> (
         <span class="cit-pub-date">2010</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioorganic%2B%2526%2Bmedicinal%2Bchemistry%26rft.volume%253D18%26rft.spage%253D7849%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">23.</span>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.23">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ryu</span>
          <span class="cit-name-given-names">YB</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Biflavonoids from Torreya nucifera displaying SARS-CoV 3CLpro inhibition</span>. 
         <abbr class="cit-jnl-abbrev">Bioorganic &amp;; medicinal chemistry</abbr>
         <span class="cit-vol">18</span>, 
         <span class="cit-fpage">7940</span>–
         <span class="cit-lpage">7947</span> (
         <span class="cit-pub-date">2010</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioorganic%2B%2526%2Bmedicinal%2Bchemistry%26rft.volume%253D18%26rft.spage%253D7940%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">24.</span>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.24">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Akaji</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Structure-based design, synthesis, and evaluation of peptide-mimetic SARS 3CL protease inhibitors</span>. 
         <abbr class="cit-jnl-abbrev">Journal of medicinal chemistry</abbr>
         <span class="cit-vol">54</span>, 
         <span class="cit-fpage">7962</span>–
         <span class="cit-lpage">7973</span> (
         <span class="cit-pub-date">2011</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmedicinal%2Bchemistry%26rft.stitle%253DJ%2BMed%2BChem%26rft.aulast%253DAkaji%26rft.auinit1%253DK.%26rft.volume%253D54%26rft.issue%253D23%26rft.spage%253D7962%26rft.epage%253D7973%26rft.atitle%253DStructure-based%2Bdesign%252C%2Bsynthesis%252C%2Band%2Bevaluation%2Bof%2Bpeptide-mimetic%2BSARS%2B3CL%2Bprotease%2Binhibitors.%26rft_id%253Dinfo%253Apmid%252F22014094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=22014094&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F08%2F2020.03.03.972133.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">25.</span>
      <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.25" data-doi="10.1021/jm301580n">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jacobs</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Discovery, synthesis, and structure-based optimization of a series of N-(tert-butyl)-2-(N-arylamido)-2-(pyridin-3-yl) acetamides (ML188) as potent noncovalent small molecule inhibitors of the severe acute respiratory syndrome coronavirus (SARS-CoV) 3CL protease</span>. 
         <abbr class="cit-jnl-abbrev">Journal of medicinal chemistry</abbr>
         <span class="cit-vol">56</span>, 
         <span class="cit-fpage">534</span>–
         <span class="cit-lpage">546</span> (
         <span class="cit-pub-date">2013</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmedicinal%2Bchemistry%26rft.volume%253D56%26rft.spage%253D534%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fjm301580n%26rft_id%253Dinfo%253Apmid%252F23231439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1021/jm301580n&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23231439&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F08%2F2020.03.03.972133.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">26.</span>
      <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.26" data-doi="10.1007/s13238-013-2841-3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ren</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">The newly emerged SARS-like coronavirus HCoV-EMC also has an" Achilles' heel": current effective inhibitor targeting a 3C-like protease</span>. 
         <abbr class="cit-jnl-abbrev">Protein &amp;; cell</abbr>
         <span class="cit-vol">4</span>, 
         <span class="cit-fpage">248</span> (
         <span class="cit-pub-date">2013</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProtein%2B%2526%2Bcell%26rft.volume%253D4%26rft.spage%253D248%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs13238-013-2841-3%26rft_id%253Dinfo%253Apmid%252F23549610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1007/s13238-013-2841-3&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23549610&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F08%2F2020.03.03.972133.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">27.</span>
      <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.27">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Thanigaimalai</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Development of potent dipeptide-type SARS-CoV 3CL protease inhibitors with novel P3 scaffolds: Design, synthesis, biological evaluation, and docking studies</span>. 
         <abbr class="cit-jnl-abbrev">European journal of medicinal chemistry</abbr>
         <span class="cit-vol">68</span>, 
         <span class="cit-fpage">372</span>–
         <span class="cit-lpage">384</span> (
         <span class="cit-pub-date">2013</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEuropean%2Bjournal%2Bof%2Bmedicinal%2Bchemistry%26rft.volume%253D68%26rft.spage%253D372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">28.</span>
      <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.28">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Turlington</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Discovery of N-(benzo [1, 2, 3] triazol-1-yl)-N-(benzyl) acetamido) phenyl) carboxamides as severe acute respiratory syndrome coronavirus (SARS-CoV) 3CLpro inhibitors: identification of ML300 and noncovalent nanomolar inhibitors with an induced-fit binding</span>. 
         <abbr class="cit-jnl-abbrev">Bioorganic &amp;; medicinal chemistry letters</abbr>
         <span class="cit-vol">23</span>, 
         <span class="cit-fpage">6172</span>–
         <span class="cit-lpage">6177</span> (
         <span class="cit-pub-date">2013</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioorganic%2B%2526%2Bmedicinal%2Bchemistry%2Bletters%26rft.volume%253D23%26rft.spage%253D6172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">29.</span>
      <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.29">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kumar</span>
          <span class="cit-name-given-names">V</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Identification and evaluation of potent Middle East respiratory syndrome coronavirus (MERS-CoV) 3CLPro inhibitors</span>. 
         <abbr class="cit-jnl-abbrev">Antiviral research</abbr>
         <span class="cit-vol">141</span>, 
         <span class="cit-fpage">101</span>–
         <span class="cit-lpage">106</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAntiviral%2Bresearch%26rft.volume%253D141%26rft.spage%253D101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">30.</span>
      <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.30">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Tang</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kramer</span>
          <span class="cit-name-given-names">ST</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fang</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Qiu</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>
          <span class="cit-name-given-names">D.</span>
         </span>
         <span class="cit-article-title">A self-attention based message passing neural network for predicting molecular lipophilicity and aqueous solubility</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Cheminformatics</abbr>
         <span class="cit-vol">12</span>, 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">9</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BCheminformatics%26rft.volume%253D12%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">31.</span>
      <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.31">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Chemi-Net: a molecular graph convolutional network for accurate drug property prediction</span>. 
         <abbr class="cit-jnl-abbrev">International journal of molecular sciences</abbr>
         <span class="cit-vol">20</span>, 
         <span class="cit-fpage">3389</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DInternational%2Bjournal%2Bof%2Bmolecular%2Bsciences%26rft.volume%253D20%26rft.spage%253D3389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">32.</span>
      <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.32">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jiang</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wei</span>
          <span class="cit-name-given-names">Z.</span>
         </span>
         <span class="cit-article-title">Molecule Property Prediction Based on Spatial Graph Embedding</span>. 
         <abbr class="cit-jnl-abbrev">Journal of chemical information and modeling</abbr>
         <span class="cit-vol">59</span>, 
         <span class="cit-fpage">3817</span>–
         <span class="cit-lpage">3828</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bchemical%2Binformation%2Band%2Bmodeling%26rft.volume%253D59%26rft.spage%253D3817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">33.</span>
      <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.33">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">MoleculeNet: a benchmark for molecular machine learning</span>. 
         <abbr class="cit-jnl-abbrev">Chemical science</abbr>
         <span class="cit-vol">9</span>, 
         <span class="cit-fpage">513</span>–
         <span class="cit-lpage">530</span> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DChemical%2Bscience%26rft.volume%253D9%26rft.spage%253D513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">34.</span>
      <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.34">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jin</span>
          <span class="cit-name-given-names">W</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Barzilay</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jaakkola</span>
          <span class="cit-name-given-names">T.</span>
         </span>
         <span class="cit-article-title">Junction tree variational autoencoder for molecular graph generation</span>. arXiv preprint
         <span class="cit-pub-id-sep cit-pub-id-arxiv-sep"> </span>
         <span class="cit-pub-id-scheme">arXiv:</span>
         <span class="cit-pub-id cit-pub-id-arxiv">180204364</span>, (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">35.</span>
      <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.35">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">You</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>
          <span class="cit-name-given-names">B</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ying</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pande</span>
          <span class="cit-name-given-names">V</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Leskovec</span>
          <span class="cit-name-given-names">J.</span>
         </span>
         <span class="cit-article-title">Graph convolutional policy network for goal-directed molecular graph generation</span>. In: 
         <abbr class="cit-jnl-abbrev">Advances in neural information processing systems</abbr> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">36.</span>
      <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.36">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>
          <span class="cit-name-given-names">K</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Docking Covalent Inhibitors: A Parameter Free Approach To Pose Prediction and Scoring</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Chemical Information &amp;; Modeling</abbr>
         <span class="cit-vol">54</span>, 
         <span class="cit-fpage">1932</span>–
         <span class="cit-lpage">1940</span> (
         <span class="cit-pub-date">2014</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BChemical%2BInformation%2B%2526%2BModeling%26rft.volume%253D54%26rft.spage%253D1932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">37.</span>
      <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.37">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Walters</span>
          <span class="cit-name-given-names">WP</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Murcko</span>
          <span class="cit-name-given-names">M.</span>
         </span>
         <span class="cit-article-title">Assessing the impact of generative AI on medicinal chemistry</span>. 
         <abbr class="cit-jnl-abbrev">Nature Biotechnology</abbr>, 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">3</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">38.</span>
      <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.38">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Olivecrona</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Blaschke</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Engkvist</span>
          <span class="cit-name-given-names">O</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>
          <span class="cit-name-given-names">H.</span>
         </span>
         <span class="cit-article-title">Molecular de-novo design through deep reinforcement learning</span>. 
         <abbr class="cit-jnl-abbrev">Journal of cheminformatics</abbr>
         <span class="cit-vol">9</span>, 
         <span class="cit-fpage">48</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcheminformatics%26rft.volume%253D9%26rft.spage%253D48%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">39.</span>
      <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.39">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kearnes</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>
          <span class="cit-name-given-names">L</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zare</span>
          <span class="cit-name-given-names">RN</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Riley</span>
          <span class="cit-name-given-names">P.</span>
         </span>
         <span class="cit-article-title">Optimization of molecules via deep reinforcement learning</span>. 
         <abbr class="cit-jnl-abbrev">Scientific reports</abbr>
         <span class="cit-vol">9</span>, 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">10</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScientific%2Breports%26rft.volume%253D9%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">40.</span>
      <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.40" data-doi="10.1126/sciadv.aap7885">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Popova</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Isayev</span>
          <span class="cit-name-given-names">O</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tropsha</span>
          <span class="cit-name-given-names">A.</span>
         </span>
         <span class="cit-article-title">Deep reinforcement learning for de novo drug design</span>. 
         <abbr class="cit-jnl-abbrev">Science advances</abbr>
         <span class="cit-vol">4</span>, 
         <span class="cit-fpage">eaap7885</span> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft.stitle%253DSci%2BAdv%26rft.aulast%253DPopova%26rft.auinit1%253DM.%26rft.volume%253D4%26rft.issue%253D7%26rft.spage%253Deaap7885%26rft.epage%253Deaap7885%26rft.atitle%253DDeep%2Breinforcement%2Blearning%2Bfor%2Bde%2Bnovo%2Bdrug%2Bdesign%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsciadv.aap7885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJhZHZhbmNlcyI7czo1OiJyZXNpZCI7czoxMjoiNC83L2VhYXA3ODg1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMDgvMjAyMC4wMy4wMy45NzIxMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">41.</span>
      <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.41" data-doi="10.1038/s41587-019-0224-x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhavoronkov</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Deep learning enables rapid identification of potent DDR1 kinase inhibitors</span>. 
         <abbr class="cit-jnl-abbrev">Nature biotechnology</abbr>
         <span class="cit-vol">37</span>, 
         <span class="cit-fpage">1038</span>–
         <span class="cit-lpage">1040</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.volume%253D37%26rft.spage%253D1038%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41587-019-0224-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41587-019-0224-x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">42.</span>
      <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.42">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schaul</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hessel</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Van Hasselt</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lanctot</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">De Freitas</span>
          <span class="cit-name-given-names">N.</span>
         </span>
         <span class="cit-article-title">Dueling network architectures for deep reinforcement learning</span>. arXiv preprint
         <span class="cit-pub-id-sep cit-pub-id-arxiv-sep"> </span>
         <span class="cit-pub-id-scheme">arXiv:</span>
         <span class="cit-pub-id cit-pub-id-arxiv">151106581</span>, (
         <span class="cit-pub-date">2015</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">43.</span>
      <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a>
      <div class="cit ref-cit ref-confproc" id="cit-2020.03.03.972133v1.43">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Van Hasselt</span>
          <span class="cit-name-given-names">H</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Guez</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Silver</span>
          <span class="cit-name-given-names">D.</span>
         </span>
         <span class="cit-article-title">Deep reinforcement learning with double q-learning</span>. In: 
         <span class="cit-conf-name">Thirtieth AAAI conference on artificial intelligence</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">44.</span>
      <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.44">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Simonini</span>
          <span class="cit-name-given-names">T.</span>
         </span>
         <span class="cit-article-title">Improvements in Deep Q Learning: Dueling Double DQN, Prioritized Experience Replay, and fixed Q-targets</span>. 
         <abbr class="cit-jnl-abbrev">Červenec</abbr> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">45.</span>
      <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.45">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Schaul</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Quan</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Antonoglou</span>
          <span class="cit-name-given-names">I</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Silver</span>
          <span class="cit-name-given-names">D.</span>
         </span>
         <span class="cit-article-title">Prioritized experience replay</span>. arXiv preprint
         <span class="cit-pub-id-sep cit-pub-id-arxiv-sep"> </span>
         <span class="cit-pub-id-scheme">arXiv:</span>
         <span class="cit-pub-id cit-pub-id-arxiv">151105952</span>, (
         <span class="cit-pub-date">2015</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">46.</span>
      <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.46">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Brockman</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Openai gym</span>. arXiv preprint
         <span class="cit-pub-id-sep cit-pub-id-arxiv-sep"> </span>
         <span class="cit-pub-id-scheme">arXiv:</span>
         <span class="cit-pub-id cit-pub-id-arxiv">160601540</span>, (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">47.</span>
      <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.47">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Speck-Planche</span>
          <span class="cit-name-given-names">A.</span>
         </span>
         <span class="cit-article-title">Recent advances in fragment-based computational drug design: tackling simultaneous targets/biological effects</span>. 
         <abbr class="cit-jnl-abbrev">Future Science</abbr> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">48.</span>
      <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.48">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Degen</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wegscheid-Gerlach</span>
          <span class="cit-name-given-names">C</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zaliani</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rarey</span>
          <span class="cit-name-given-names">M.</span>
         </span>
         <span class="cit-article-title">On the Art of Compiling and Using‘Drug-Like’Chemical Fragment Spaces</span>. 
         <abbr class="cit-jnl-abbrev">ChemMedChem: Chemistry Enabling Drug Discovery</abbr>
         <span class="cit-vol">3</span>, 
         <span class="cit-fpage">1503</span>–
         <span class="cit-lpage">1507</span> (
         <span class="cit-pub-date">2008</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DChemMedChem%253A%2BChemistry%2BEnabling%2BDrug%2BDiscovery%26rft.volume%253D3%26rft.spage%253D1503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">49.</span>
      <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.49" data-doi="10.1021/ci2000924">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Varin</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schuffenhauer</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ertl</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Renner</span>
          <span class="cit-name-given-names">S.</span>
         </span>
         <span class="cit-article-title">Mining for bioactive scaffolds with scaffold networks: improved compound set enrichment from primary screening data</span>. 
         <abbr class="cit-jnl-abbrev">Journal of chemical information and modeling</abbr>
         <span class="cit-vol">51</span>, 
         <span class="cit-fpage">1528</span>–
         <span class="cit-lpage">1538</span> (
         <span class="cit-pub-date">2011</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bchemical%2Binformation%2Band%2Bmodeling%26rft.stitle%253DJ%2BChem%2BInf%2BModel%26rft.aulast%253DVarin%26rft.auinit1%253DT.%26rft.volume%253D51%26rft.issue%253D7%26rft.spage%253D1528%26rft.epage%253D1538%26rft.atitle%253DMining%2Bfor%2Bbioactive%2Bscaffolds%2Bwith%2Bscaffold%2Bnetworks%253A%2Bimproved%2Bcompound%2Bset%2Benrichment%2Bfrom%2Bprimary%2Bscreening%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fci2000924%26rft_id%253Dinfo%253Apmid%252F21615076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1021/ci2000924&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21615076&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F08%2F2020.03.03.972133.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">50.</span>
      <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.50" data-doi="10.1021/ci600338x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Schuffenhauer</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ertl</span>
          <span class="cit-name-given-names">P</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Roggo</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wetzel</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Koch</span>
          <span class="cit-name-given-names">MA</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Waldmann</span>
          <span class="cit-name-given-names">H.</span>
         </span>
         <span class="cit-article-title">The scaffold tree-visualization of the scaffold universe by hierarchical scaffold classification</span>. 
         <abbr class="cit-jnl-abbrev">Journal of chemical information and modeling</abbr>
         <span class="cit-vol">47</span>, 
         <span class="cit-fpage">47</span>–
         <span class="cit-lpage">58</span> (
         <span class="cit-pub-date">2007</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bchemical%2Binformation%2Band%2Bmodeling%26rft.stitle%253DJ%2BChem%2BInf%2BModel%26rft.aulast%253DSchuffenhauer%26rft.auinit1%253DA.%26rft.volume%253D47%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D58%26rft.atitle%253DThe%2Bscaffold%2Btree--visualization%2Bof%2Bthe%2Bscaffold%2Buniverse%2Bby%2Bhierarchical%2Bscaffold%2Bclassification.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fci600338x%26rft_id%253Dinfo%253Apmid%252F17238248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1021/ci600338x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17238248&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F08%2F2020.03.03.972133.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">51.</span>
      <a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.51">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Reis</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gaspar</span>
          <span class="cit-name-given-names">A</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Milhazes</span>
          <span class="cit-name-given-names">N</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Borges</span>
          <span class="cit-name-given-names">F.</span>
         </span>
         <span class="cit-article-title">Chromone as a Privileged Scaffold in Drug Discovery: Recent Advances: Miniperspective</span>. 
         <abbr class="cit-jnl-abbrev">Journal of medicinal chemistry</abbr>
         <span class="cit-vol">60</span>, 
         <span class="cit-fpage">7941</span>–
         <span class="cit-lpage">7957</span> (
         <span class="cit-pub-date">2017</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmedicinal%2Bchemistry%26rft.volume%253D60%26rft.spage%253D7941%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">52.</span>
      <a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.52">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Pillaiyar</span>
          <span class="cit-name-given-names">T</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Manickam</span>
          <span class="cit-name-given-names">M</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Namasivayam</span>
          <span class="cit-name-given-names">V</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hayashi</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jung</span>
          <span class="cit-name-given-names">S-H.</span>
         </span>
         <span class="cit-article-title">An Overview of Severe Acute Respiratory Syndrome-Coronavirus (SARS-CoV) 3CL Protease Inhibitors: Peptidomimetics and Small Molecule Chemotherapy</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Medicinal Chemistry</abbr>, (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">53.</span>
      <a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.53">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bickerton</span>
          <span class="cit-name-given-names">GR</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Paolini</span>
          <span class="cit-name-given-names">GV</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Besnard</span>
          <span class="cit-name-given-names">J</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Muresan</span>
          <span class="cit-name-given-names">S</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hopkins</span>
          <span class="cit-name-given-names">AL.</span>
         </span>
         <span class="cit-article-title">Quantifying the chemical beauty of drugs</span>. 
         <abbr class="cit-jnl-abbrev">Nature chemistry</abbr>
         <span class="cit-vol">4</span>, 
         <span class="cit-fpage">90</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Bchemistry%26rft.volume%253D4%26rft.spage%253D90%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">54.</span>
      <a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.54" data-doi="10.1039/c9me00039a">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Elton</span>
          <span class="cit-name-given-names">DC</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Boukouvalas</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fuge</span>
          <span class="cit-name-given-names">MD</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chung</span>
          <span class="cit-name-given-names">PW.</span>
         </span>
         <span class="cit-article-title">Deep learning for molecular design—a review of the state of the art</span>. 
         <abbr class="cit-jnl-abbrev">Molecular Systems Design &amp;; Engineering</abbr>
         <span class="cit-vol">4</span>, 
         <span class="cit-fpage">828</span>–
         <span class="cit-lpage">849</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BDesign%2B%2526%2BEngineering%26rft.volume%253D4%26rft.spage%253D828%26rft_id%253Dinfo%253Adoi%252F10.1039%252Fc9me00039a%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1039/c9me00039a&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">55.</span>
      <a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.55">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">X</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>
          <span class="cit-name-given-names">Y</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Byrne</span>
          <span class="cit-name-given-names">R</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schneider</span>
          <span class="cit-name-given-names">G</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>
          <span class="cit-name-given-names">S.</span>
         </span>
         <span class="cit-article-title">Concepts of artificial intelligence for computer-assisted drug discovery</span>. 
         <abbr class="cit-jnl-abbrev">Chemical reviews</abbr>
         <span class="cit-vol">119</span>, 
         <span class="cit-fpage">10520</span>–
         <span class="cit-lpage">10594</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DChemical%2Breviews%26rft.volume%253D119%26rft.spage%253D10520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">56.</span>
      <a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.56" data-doi="10.1021/jm051256o">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Friesner</span>
          <span class="cit-name-given-names">RA</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Extra precision glide: Docking and scoring incorporating a model of hydrophobic enclosure for protein− ligand complexes</span>. 
         <abbr class="cit-jnl-abbrev">Journal of medicinal chemistry</abbr>
         <span class="cit-vol">49</span>, 
         <span class="cit-fpage">6177</span>–
         <span class="cit-lpage">6196</span> (
         <span class="cit-pub-date">2006</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmedicinal%2Bchemistry%26rft.stitle%253DJ%2BMed%2BChem%26rft.aulast%253DFriesner%26rft.auinit1%253DR.%2BA.%26rft.volume%253D49%26rft.issue%253D21%26rft.spage%253D6177%26rft.epage%253D6196%26rft.atitle%253DExtra%2Bprecision%2Bglide%253A%2Bdocking%2Band%2Bscoring%2Bincorporating%2Ba%2Bmodel%2Bof%2Bhydrophobic%2Benclosure%2Bfor%2Bprotein-ligand%2Bcomplexes.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fjm051256o%26rft_id%253Dinfo%253Apmid%252F17034125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1021/jm051256o&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17034125&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F08%2F2020.03.03.972133.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000241192400010&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">57.</span>
      <a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.03.972133v1.57">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jin</span>
          <span class="cit-name-given-names">Z</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Structure-based drug design, virtual screening and high-throughput screening rapidly identify antiviral leads targeting COVID-19</span>. 
         <abbr class="cit-jnl-abbrev">bioRxiv</abbr>, 
         <span class="cit-fpage">2020.2002.2026.964882</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
