<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/embed/inline-graphic-2.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2018/01/22/251322.atom"/>
  <meta name="HW.pisa" content="biorxiv;251322v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="The Evolutionary History and Impact of Bacterial tRNA Modifications"/>
  <meta name="DC.Identifier" content="10.1101/251322"/>
  <meta name="DC.Date" content="2018-01-22"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2018, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Along with tRNAs, enzymes that modify anticodon bases are a key aspect of translation across the tree of life. tRNA modifications extend wobble pairing, allowing specific (“target”) tRNAs to recognize multiple codons and cover for other (“non-target”) tRNAs, often improving translation efficiency and accuracy. However, the detailed evolutionary history and impact of tRNA modifying enzymes has not been analyzed. Using ancestral reconstruction of five tRNA modifications across 1093 bacteria, we show that most modifications were ancestral to eubacteria, but were repeatedly lost in many lineages. Most modification losses coincided with evolutionary shifts in non-target tRNAs, often driven by increased bias in genomic GC and associated codon use, or by genome reduction. In turn, the loss of tRNA modifications stabilized otherwise highly dynamic tRNA gene repertoires. Our work thus traces the complex history of bacterial tRNA modifications, providing the first clear evidence for their role in the evolution of bacterial translation."/>
  <meta name="DC.Contributor" content="Gaurav D Diwan"/>
  <meta name="DC.Contributor" content="Deepa Agashe"/>
  <meta name="article:published_time" content="2018-01-22"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="The Evolutionary History and Impact of Bacterial tRNA Modifications"/>
  <meta name="citation_abstract" lang="en" content="&lt;;h3&gt;;ABSTRACT&lt;;/h3&gt;; &lt;;p&gt;;Along with tRNAs, enzymes that modify anticodon bases are a key aspect of translation across the tree of life. tRNA modifications extend wobble pairing, allowing specific (“target”) tRNAs to recognize multiple codons and cover for other (“non-target”) tRNAs, often improving translation efficiency and accuracy. However, the detailed evolutionary history and impact of tRNA modifying enzymes has not been analyzed. Using ancestral reconstruction of five tRNA modifications across 1093 bacteria, we show that most modifications were ancestral to eubacteria, but were repeatedly lost in many lineages. Most modification losses coincided with evolutionary shifts in non-target tRNAs, often driven by increased bias in genomic GC and associated codon use, or by genome reduction. In turn, the loss of tRNA modifications stabilized otherwise highly dynamic tRNA gene repertoires. Our work thus traces the complex history of bacterial tRNA modifications, providing the first clear evidence for their role in the evolution of bacterial translation.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2018/01/01"/>
  <meta name="citation_mjid" content="biorxiv;251322v1"/>
  <meta name="citation_id" content="251322v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/251322v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/251322v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/251322v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322.full.pdf"/>
  <meta name="citation_doi" content="10.1101/251322"/>
  <meta name="citation_num_pages" content="31"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="251322"/>
  <meta name="citation_author" content="Gaurav D Diwan"/>
  <meta name="citation_author_institution" content="National Centre for Biological Sciences, Tata Institute of Fundamental Research"/>
  <meta name="citation_author_institution" content="SASTRA University"/>
  <meta name="citation_author" content="Deepa Agashe"/>
  <meta name="citation_author_institution" content="National Centre for Biological Sciences, Tata Institute of Fundamental Research"/>
  <meta name="citation_author_email" content="dagashe@ncbs.res.in"/>
  <meta name="citation_reference" content="Björk, G. R., &amp;; Hagervall, T. G. (2014). Transfer RNA Modification: Presence, Synthesis, and Function. EcoSal Plus, 6(1). http://doi.org/10.1128/ecosalplus.ESP-0007–2013"/>
  <meta name="citation_reference" content="Boccaletto, P., Machnicka, M. A., Purta, E., Piątkowski, P., Bagiński, B., Wirecki, T. K., … Bujnicki, J. M. (2018). MODOMICS: a database of RNA modification pathways. 2017 update. Nucleic Acids Research, 46(D1), D303–D307. http://doi.org/10.1093/nar/gkx1030"/>
  <meta name="citation_reference" content="Bollback, J. P. (2006). SIMMAP: stochastic character mapping of discrete traits on phylogenies. BMC Bioinformatics, 7(1), 88. http://doi.org/10.1186/1471–2105-7–88"/>
  <meta name="citation_reference" content="Brierley, I., Meredith, M. R., Bloys, A. J., &amp;; Hagervall, T. G. (1997). Expression of a coronavirus ribosomal frameshift signal in Escherichia coli: influence of tRNA anticodon modification on frameshifting. Journal of Molecular Biology, 270(3), 360–373. http://doi.org/10.1006/jmbi.1997.1134"/>
  <meta name="citation_reference" content="Chan, P. P., &amp;; Lowe, T. M. (2009). GtRNAdb: A database of transfer RNA genes detected in genomic sequence. Nucleic Acids Research, 37(SUPPL. 1), 93–97. http://doi.org/10.1093/nar/gkn787"/>
  <meta name="citation_reference" content="Charif, D., Thioulouse, J., Lobry, J. R., &amp;; Perrière, G. (2005). Online synonymous codon usage analyses with the ade4 and seqinR packages. Bioinformatics, 21(4), 545–547. http://doi.org/10.1093/bioinformatics/bti037"/>
  <meta name="citation_reference" content="Chen, S. L., Lee, W., Hottes, A. K., Shapiro, L., &amp;; McAdams, H. H. (2004). Codon usage between genomes is constrained by genome-wide mutational processes. Proceedings of the National Academy of Sciences of the United States of America, 101(10), 3480–3485. http://doi.org/10.1073/pnas.0307827100"/>
  <meta name="citation_reference" content="Chionh, Y. H., McBee, M., Babu, I. R., Hia, F., Lin, W., Zhao, W., … Dedon, P. C. (2016). tRNA-mediated codon-biased translation in mycobacterial hypoxic persistence. Nature Communications, 7, 13302. http://doi.org/10.1038/ncomms13302"/>
  <meta name="citation_reference" content="Crick, F. H. (1966). Codon--anticodon pairing: the wobble hypothesis. Journal of Molecular Biology, 19(2), 548–55. http://doi.org/10.1016/S0022-2836(66)80022-0"/>
  <meta name="citation_reference" content="Eddy, S. (1998). Profile hidden Markov models. Bioinformatics, 14(9), 755–763. http://doi.org/btb114 [pii]"/>
  <meta name="citation_reference" content="Eddy, S. R. (2009). A New Generation of Homology Search Tools Based on Probabilistic Inference. Genome Informatics, 23(1), 205–211. http://doi.org/10.1142/9781848165632_0019"/>
  <meta name="citation_reference" content="Elliot, M. G., &amp;; Mooers, A. Ø. (2014). Inferring ancestral states without assuming neutrality or gradualism using a stable model of continuous character evolution. BMC Evolutionary Biology, 14(1), 226. http://doi.org/10.1186/s12862–014-0226–8"/>
  <meta name="citation_reference" content="Gao, T., Tan, M., Liu, W., Zhang, C., Zhang, T., Zheng, L., … Zhou, R. (2016). GidA, a tRNA Modification Enzyme, Contributes to the Growth, and Virulence of Streptococcus suis Serotype 2. Frontiers in Cellular and Infection Microbiology, 6(April), 1–11. http://doi.org/10.3389/fcimb.2016.00044"/>
  <meta name="citation_reference" content="Grosjean, H., Breton, M., Sirand-Pugnet, P., Tardy, F., Thiaucourt, F., Citti, C., … Blanchard, A. (2014). Predicting the minimal translation apparatus: lessons from the reductive evolution of Mollicutes. PLoS Genetics, 10(5), e1004363. http://doi.org/10.1371/journal.pgen.1004363"/>
  <meta name="citation_reference" content="Grosjean, H., de Crécy-Lagard, V., &amp;; Marck, C. (2010). Deciphering synonymous codons in the three domains of life: Co-evolution with specific tRNA modification enzymes. FEBS Letters, 584(2), 252–264. http://doi.org/10.1016/j.febslet.2009.11.052"/>
  <meta name="citation_reference" content="Hershberg, R., &amp;; Petrov, D. A. (2009). General Rules for Optimal Codon Choice. PLoS Genetics, 5(7), e1000556. http://doi.org/10.1371/journal.pgen.1000556"/>
  <meta name="citation_reference" content="Ikemura, T. (1985). Codon usage and tRNA content in unicellular and multicellular organisms. Molecular Biology and Evolution, 2(1), 13–34."/>
  <meta name="citation_reference" content="Kanaya, S., Yamada, Y., Kudo, Y., &amp;; Ikemura, T. (1999). Studies of codon usage and tRNA genes of 18 unicellular organisms and quantification of Bacillus subtilis tRNAs: Gene expression level and species-specific diversity of codon usage based on multivariate analysis. Gene, 238(1), 143–155. http://doi.org/10.1016/S0378–1119(99)00225–5"/>
  <meta name="citation_reference" content="Lowe, T. M., &amp;; Eddy, S. R. (1996). TRNAscan-SE: A program for improved detection of transfer RNA genes in genomic sequence. Nucleic Acids Research, 25(5), 955–964. http://doi.org/10.1093/nar/25.5.0955"/>
  <meta name="citation_reference" content="Manickam, N., Joshi, K., Bhatt, M. J., &amp;; Farabaugh, P. J. (2015). Effects of tRNA modification on translational accuracy depend on intrinsic codon–anticodon strength. Nucleic Acids Research, 2 gkv1506. http://doi.org/10.1093/nar/gkv1506"/>
  <meta name="citation_reference" content="Marck, C., &amp;; Grosjean, H. (2002). tRNomics: analysis of tRNA genes from 50 genomes of Eukarya, Archaea, and Bacteria reveals anticodon-sparing strategies and domain-specific features. RNA (New York, N.Y.), 8(10), 1189–232. http://doi.org/10.1017/S1355838202022021"/>
  <meta name="citation_reference" content="Murao, K., Hasegawa, T., &amp;; Ishikura, H. (1976). 5-Methoxyuridine: A New Minor Constituent Located in the First Position of the Anticodon of tRNAAla, tRNAThr, and tRNAVal from Bacillus Subtilis. Nucleic Acids Research, 3(10), 2851–2860. http://doi.org/10.1093/nar/3.10.2851"/>
  <meta name="citation_reference" content="Nasvall, S. J., Chen, P., &amp;; Bjork, G. R. (2004). The modified wobble nucleoside uridine-5-oxyacetic acid in tRNAPro(cmo5UGG) promotes reading of all four proline codons in vivo. RNA (New York, N.Y.), 10(10), 1662–73. http://doi.org/10.1261/rna.7106404"/>
  <meta name="citation_reference" content="Näsvall, S. J., Chen, P., &amp;; Björk, G. R. (2007). The wobble hypothesis revisited: uridine-5-oxyacetic acid is critical for reading of G-ending codons. RNA (New York, N.Y.), 13(Iii), 2151–2164. http://doi.org/10.1261/rna.731007"/>
  <meta name="citation_reference" content="Noguchi, S., Nishimura, Y., Hirota, Y., &amp;; Nishimura, S. (1982). Isolation and characterization of an Escherichia coli mutant lacking tRNA-guanine transglycosylase. Function and biosynthesis of queuosine in tRNA. The Journal of Biological Chemistry, 257(11), 6544–50."/>
  <meta name="citation_reference" content="Notredame, C., Higgins, D., &amp;; Heringa, J. (2000). T-Coffee: A novel method for fast and accurate multiple sequence alignment. Journal of Molecular Biology, 302(1), 205–217. http://doi.org/10.1006/jmbi.2000.4042"/>
  <meta name="citation_reference" content="Novoa, E. M., Pavon-Eternod, M., Pan, T., &amp;; Ribas De Pouplana, L. (2012). A role for tRNA modifications in genome structure and codon usage. Cell, 149(1), 202–213. http://doi.org/10.1016/j.cell.2012.01.050"/>
  <meta name="citation_reference" content="Overbeek, R., Begley, T., Butler, R. M., Choudhuri, J. V., Chuang, H.-Y., Cohoon, M., … Vonstein, V. (2005). The subsystems approach to genome annotation and its use in the project to annotate 1000 genomes. Nucleic Acids Research, 33(17), 5691–702. http://doi.org/10.1093/nar/gki866"/>
  <meta name="citation_reference" content="R Development Core Team. (2015). R: A Language and Environment for Statistical Computing. http://doi.org/10.1038/sj.hdy.6800737"/>
  <meta name="citation_reference" content="Revell, L. J. (2012). phytools: An R package for phylogenetic comparative biology (and other things). Methods in Ecology and Evolution, 3(2), 217–223. http://doi.org/10.1111/j.2041–210X.2011.00169.x"/>
  <meta name="citation_reference" content="Segata, N., Börnigen, D., Morgan, X. C., &amp;; Huttenhower, C. (2013). PhyloPhlAn is a new method for improved phylogenetic and taxonomic placement of microbes. Nature Communications, 4. http://doi.org/10.1038/ncomms3304"/>
  <meta name="citation_reference" content="Shippy, D. C., Eakley, N. M., Lauhon, C. T., Bochsler, P. N., &amp;; Fadl, A. A. (2013). Virulence characteristics of Salmonella following deletion of genes encoding the tRNA modification enzymes GidA and MnmE. Microbial Pathogenesis, 57, 1–9. http://doi.org/10.1016/j.micpath.2013.01.004"/>
  <meta name="citation_reference" content="Wald, N., &amp;; Margalit, H. (2014). Auxiliary tRNAs: Large-scale analysis of tRNA genes reveals patterns of tRNA repertoire dynamics. Nucleic Acids Research, 42(10), 6552–6566. http://doi.org/10.1093/nar/gku245"/>
  <meta name="citation_reference" content="Wolf, J., Gerber, A. P., &amp;; Keller, W. (2002). tad A, an essential tRNA-specific adenosine deaminase from Escherichia coli. The EMBO Journal, 21(14), 3841–51. http://doi.org/10.1093/emboj/cdf362"/>
  <meta name="citation_reference" content="Yamada, Y., Matsugi, J., Ishikura, H., &amp;; Murao, K. (2005). Bacillus subtilis tRNAPro with the anticodon mo5UGG can recognize the codon CCC. Biochimica et Biophysica Acta - Gene Structure and Expression, 1728(3), 143–149. http://doi.org/10.1016/j.bbaexp.2005.02.011"/>
  <meta name="citation_reference" content="Yokobori, S. I., Kitamura, A., Grosjean, H., &amp;; Bessho, Y. (2013). Life without tRNAArg-adenosine deaminase TadA: Evolutionary consequences of decoding the four CGN codons as arginine in Mycoplasmas and other Mollicutes. Nucleic Acids Research, 41(13), 6531–6543. http://doi.org/10.1093/nar/gkt356"/>
  <meta name="citation_reference" content="Yokoyama, S., Watanabe, T., Murao, K., Ishikura, H., Yamaizumi, Z., Nishimura, S., &amp;; Miyazawa, T. (1985). Molecular mechanism of codon recognition by tRNA species with modified uridine in the first position of the anticodon. Proceedings of the National Academy of Sciences, 82(15), 4905–4909. http://doi.org/10.1073/pnas.82.15.4905"/>
  <meta name="citation_reference" content="Zallot, R., Yuan, Y., &amp;; de Crécy-Lagard, V. (2017). The Escherichia coli COG1738 Member YhhQ Is Involved in 7-Cyanodeazaguanine (preQ₀) Transport. Biomolecules, 7(1), 1–13. http://doi.org/10.3390/biom7010012"/>
  <meta name="twitter:title" content="The Evolutionary History and Impact of Bacterial tRNA Modifications"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Along with tRNAs, enzymes that modify anticodon bases are a key aspect of translation across the tree of life. tRNA modifications extend wobble pairing, allowing specific (“target”) tRNAs to recognize multiple codons and cover for other (“non-target”) tRNAs, often improving translation efficiency and accuracy. However, the detailed evolutionary history and impact of tRNA modifying enzymes has not been analyzed. Using ancestral reconstruction of five tRNA modifications across 1093 bacteria, we show that most modifications were ancestral to eubacteria, but were repeatedly lost in many lineages. Most modification losses coincided with evolutionary shifts in non-target tRNAs, often driven by increased bias in genomic GC and associated codon use, or by genome reduction. In turn, the loss of tRNA modifications stabilized otherwise highly dynamic tRNA gene repertoires. Our work thus traces the complex history of bacterial tRNA modifications, providing the first clear evidence for their role in the evolution of bacterial translation."/>
  <meta name="og-title" property="og:title" content="The Evolutionary History and Impact of Bacterial tRNA Modifications"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/251322v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Along with tRNAs, enzymes that modify anticodon bases are a key aspect of translation across the tree of life. tRNA modifications extend wobble pairing, allowing specific (“target”) tRNAs to recognize multiple codons and cover for other (“non-target”) tRNAs, often improving translation efficiency and accuracy. However, the detailed evolutionary history and impact of tRNA modifying enzymes has not been analyzed. Using ancestral reconstruction of five tRNA modifications across 1093 bacteria, we show that most modifications were ancestral to eubacteria, but were repeatedly lost in many lineages. Most modification losses coincided with evolutionary shifts in non-target tRNAs, often driven by increased bias in genomic GC and associated codon use, or by genome reduction. In turn, the loss of tRNA modifications stabilized otherwise highly dynamic tRNA gene repertoires. Our work thus traces the complex history of bacterial tRNA modifications, providing the first clear evidence for their role in the evolution of bacterial translation."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2018-01-22"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>The Evolutionary History and Impact of Bacterial tRNA Modifications | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-77162 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">ABSTRACT</h2>
    <p id="p-2">Along with tRNAs, enzymes that modify anticodon bases are a key aspect of translation across the tree of life. tRNA modifications extend wobble pairing, allowing specific (“target”) tRNAs to recognize multiple codons and cover for other (“non-target”) tRNAs, often improving translation efficiency and accuracy. However, the detailed evolutionary history and impact of tRNA modifying enzymes has not been analyzed. Using ancestral reconstruction of five tRNA modifications across 1093 bacteria, we show that most modifications were ancestral to eubacteria, but were repeatedly lost in many lineages. Most modification losses coincided with evolutionary shifts in non-target tRNAs, often driven by increased bias in genomic GC and associated codon use, or by genome reduction. In turn, the loss of tRNA modifications stabilized otherwise highly dynamic tRNA gene repertoires. Our work thus traces the complex history of bacterial tRNA modifications, providing the first clear evidence for their role in the evolution of bacterial translation.</p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">INTRODUCTION</h2>
    <p id="p-3">Compared to the total complement of 61 sense codons, most bacterial genomes contain only 25 – 46 unique tRNA species (
     <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Grosjean, de Crécy-Lagard, &amp;; Marck, 2010</a>). This apparent shortfall in decoding ability is mitigated by G::U wobble base pairing that allows a single tRNA to decode non–complementary codons (
     <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Crick, 1966</a>). Additionally, specific modifications to the first anticodon base of the tRNA also contribute to wobble base pairing, sometimes reducing translational errors (
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Björk &amp;; Hagervall, 2014</a>; 
     <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Brierley, Meredith, Bloys, &amp;; Hagervall, 1997</a>; 
     <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Manickam, Joshi, Bhatt, &amp;; Farabaugh, 2015</a>; 
     <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Marck &amp;; Grosjean, 2002</a>; 
     <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Nasvall, Chen, &amp;; Bjork, 2004</a>; 
     <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Näsvall, Chen, &amp;; Björk, 2007</a>; 
     <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Yokoyama et al., 1985</a>). Base modifications are carried out by several tRNA modifying enzyme (ME) pathways, modifying the wobble position of a large fraction (42%) of all tRNA species (
     <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Boccaletto et al., 2018</a>; 
     <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Grosjean et al., 2010</a>; 
     <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Machnicka, Olchowik, Grosjean, &amp;; Bujnicki, 2015</a>). Given their potential impact on the effective tRNA pool and on translation dynamics, MEs should generally face strong selection, especially in species with a restricted tRNA repertoire. Indeed, in several bacteria, deleting modifying enzymes is lethal or otherwise deleterious for growth and translation (
     <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Björk &amp;; Hagervall, 2014</a>; 
     <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gao et al., 2016</a>; 
     <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Shippy, Eakley, Lauhon, Bochsler, &amp;; Fadl, 2013</a>; 
     <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Wolf, Gerber, &amp;; Keller, 2002</a>). However, in some cases MEs do not appear to be essential (
     <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Noguchi, Nishimura, Hirota, &amp;; Nishimura, 1982</a>), suggesting strong yet variable evolutionary impact of tRNA modifications. Recently, Novoa and colleagues showed that MEs unique to the three kingdoms of life have shaped kingdom-specific tRNA pools (
     <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Novoa, Pavon-Eternod, Pan, &amp;; Ribas De Pouplana, 2012</a>). However, the detailed evolutionary history of diverse bacterial tRNA modifications and their impact on key components of bacterial translation remain unclear.
    </p>
    <p id="p-4">To address these gaps, we traced the evolutionary history of the five known bacterial tRNA modifications at the first wobble base of the anticodon: (c)mnm5(s2)U (5-carboxymethylaminomethyl 2-thiouridine and its variants), cmo5U (uridine-5-oxyacetic acid), k2C (2-lysyl-cytidine), Q (Queuosine) and I (Inosine) (
     <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Björk &amp;; Hagervall, 2014</a>). We first determined the occurrence of each modification pathway in 1093 sequenced bacteria that represent the diversity of eubacteria, and then used ancestral reconstruction to infer the major evolutionary gain or loss events in the bacterial phylogeny. To determine the causes and impacts of change in tRNA modification, we analyzed two other key components of translation: tRNA gene pools and codon bias, thought to coevolve under translational selection (
     <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Ikemura, 1985</a>; 
     <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kanaya, Yamada, Kudo, &amp;; Ikemura, 1999</a>). Finally, since both of these factors are in turn correlated with genomic GC content (
     <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chen, Lee, Hottes, Shapiro, &amp;; McAdams, 2004</a>; 
     <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hershberg &amp;; Petrov, 2009</a>; 
     <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Wald &amp;; Margalit, 2014</a>), we also mapped genomic GC content across the bacterial phylogeny. Thus, we aimed to generate a comprehensive model of the joint evolutionary history of important components of bacterial translation – tRNA genes and modifying enzymes – and their relationship with genomic GC content.
    </p>
    <p id="p-5">Each modified tRNA can recognize between 2 to 4 codons. Although there are no direct measurements of the effect of tRNA modification on translation rate, the presence of a multifunctional tRNA should amplify the effective tRNA pool, reducing ribosomal waiting times for the correct tRNA and contributing to an increase in global translation speed. Modification also prevents tRNA carrying amino acids coded by two synonymous codons from recognizing other non-synonymous codons, potentially reducing mistranslation (
     <a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Björk &amp;; Hagervall, 2014</a>; 
     <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Grosjean et al., 2010</a>). Thus, tRNA modifications may affect both translational speed and accuracy. Specifically, we hypothesized that the presence of a tRNA modification should selectively favor the tRNAs that it modifies (henceforth “target” RNA). Concomitantly, selection on tRNA that are not modified by the ME (henceforth “non-target” tRNA; encoding the same amino acid) should be weakened, since their function would be redundant with the modified target tRNA. As such, we predicted that the evolutionary loss of non-target tRNA should be permissible only in the presence of a modification that allows target tRNA to decode multiple sense codons. In contrast, the evolutionary loss of a modification should be associated with a generally irreversible expansion of the tRNA repertoire to include both target and non-target tRNA. Finally, we expected that these changes in tRNA gene content should be correlated with changes in codon use as well as genomic GC content.
    </p>
    <p id="p-6">Our analyses indicate that the eubacterial ancestor had a large tRNA repertoire as well as most of the known bacterial tRNA modifications. However, some types of wobble were lost in multiple bacterial lineages. As predicted, ME loss was strongly associated with an expansion of the tRNA repertoire; either in conjunction with large shifts in genomic GC and associated changes in codon use, or in a GC-independent manner. We suggest that the expanded tRNA set weakened selection for MEs, allowing their repeated loss via drift. Conversely, in rare cases, a shrinking of tRNA diversity may have favored the evolution of a novel modification; alternatively, the chance evolution of a new modification may have relaxed selection on the tRNA pool. Our work represents the first systematic analysis of the joint evolutionary history of tRNA modifications, genome GC content and tRNA genes, and elucidates the role of tRNA modifications in the evolution of the bacterial translation machinery.</p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">RESULTS</h2>
    <div id="sec-3" class="subsection">
     <h3>The evolutionary history of bacterial tRNA modifications is highly dynamic</h3>
     <p id="p-7">We determined the occurrence of tRNA modifications across 1093 bacteria using homology searches for complete protein sequences of previously described MEs (see Methods). We assume that detected homologs are functional, and have the same modification function as that of the previously described enzymes. Within each modification pathway, we focused on the following enzymes that directly modify the tRNA and complete the modification (see Methods), and whose absence should effectively prevent the modification. i) MnmE-MnmG for the (c)mnm5(s2)U modification ii) CmoA-CmoB for the cmo5U modification iii) Tgt-QueA-QueG for the Q modification iv) TadA for the I modification and v) TilS for the k2C modification. Note that in our analysis, we did not consider G::U wobble pairing, which is not mediated by specific modification enzymes. We observed that except for TilS, most MEs are not found in all bacteria (
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1A</a>), suggesting that different modification pathways were gained or lost in specific lineages. The essential genes RpoB and RplA (positive controls) were respectively present in 99.5% and 99.7% of analyzed bacteria, indicating that our homology detection method resulted in very few false negatives. When multiple enzymes are necessary for a modification, they tend to co-occur in bacterial lineages (
      <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1A</a>). For instance, both MnmE and MnmG are absent only in Actinobacteria. Similarly, both CmoA and CmoB are primarily observed in γ-proteobacteria and ∊-proteobacteria. However, this contradicts previous reports showing that the cmo5U modification is also active in 
      <em>Bacillus subtilis</em> and 
      <em>Mycobacterium bovis</em> str. BCG (
      <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chionh et al., 2016</a>; 
      <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Murao, Hasegawa, &amp;; Ishikura, 1976</a>; 
      <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Yamada, Matsugi, Ishikura, &amp;; Murao, 2005</a>). Despite an extensive search, we could not reliably identify the reported enzymes in the sequences of any other organisms (see Methods for details). Hence, here we focus on the proteobacterial cmo5U modification. In contrast to the co-occurrence of CmoA and CmoB, for the Queuosine modification, we found that QueG is often missing even when Tgt and QueA are present. This relative evolutionary instability of QueG corroborates a recent report suggesting that Tgt alone may be enough to generate the Q modification (
      <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Zallot, Yuan, &amp;; de Crécy-Lagard, 2017</a>). 
     </p>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="The evolutionary history of bacterial tRNA modifications. (A) Columns indicate the presence (gray) or absence (white) of various tRNA modifying enzymes as noted above each column. The leftmost column shows two essential housekeeping proteins (RpoB-RNA polymerase beta subunit and RplA-50S Ribosomal protein L1) that serve as positive controls for homology detection. (B) Bacterial phylogenetic tree (Segata et al., 2013) pruned to show the evolutionary relationships between the 1093 bacterial species used in our study. Branch tips are colored by taxonomic group. Filled and open circles respectively mark the major gain and loss of tRNA modifications (with at least 5 descendent species in the relevant lineage), as inferred by ancestral reconstruction (see Methods). Circles are colored according to the modification, as indicated in panel A. Values in parentheses indicate the posterior probability of each gain or loss event for each ME involved in the modification pathway. All other minor gain and loss events (with &lt;; 5 descendent species in the pruned phylogeny) are shown in Fig. S1. Asterisks at the root of the tree indicate the presence of the respective modification. (C) Genetic code table showing the target (solid boxes) and non-target tRNA (dashed boxes) for each modification, colored as in panel A. tRNA with a yellow border show G::U wobble base pairing and were not considered in subsequent 16 analyses. Serine tRNA are highlighted with a black box to indicate the ability of to decode all Serine codons. The column titled [[#x201C]]Anc[[#x201D]] indicates the presence (grey cells) or absence (white cells) of each tRNA in the eubacterial ancestor, inferred from ancestral reconstruction (see Methods). Value in each cell indicates the posterior probability of the inferred state for each tRNA." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-449755067" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;The evolutionary history of bacterial tRNA modifications.&lt;;/span&gt;; (A) Columns indicate the presence (gray) or absence (white) of various tRNA modifying enzymes as noted above each column. The leftmost column shows two essential housekeeping proteins (RpoB-RNA polymerase beta subunit and RplA-50S Ribosomal protein L1) that serve as positive controls for homology detection. (B) Bacterial phylogenetic tree (Segata et al., 2013) pruned to show the evolutionary relationships between the 1093 bacterial species used in our study. Branch tips are colored by taxonomic group. Filled and open circles respectively mark the major gain and loss of tRNA modifications (with at least 5 descendent species in the relevant lineage), as inferred by ancestral reconstruction (see Methods). Circles are colored according to the modification, as indicated in panel A. Values in parentheses indicate the posterior probability of each gain or loss event for each ME involved in the modification pathway. All other minor gain and loss events (with &lt;; 5 descendent species in the pruned phylogeny) are shown in Fig. S1. Asterisks at the root of the tree indicate the presence of the respective modification. (C) Genetic code table showing the target (solid boxes) and non-target tRNA (dashed boxes) for each modification, colored as in panel A. tRNA with a yellow border show G::U wobble base pairing and were not considered in subsequent &lt;;sup&gt;;16&lt;;/sup&gt;; analyses. Serine tRNA are highlighted with a black box to indicate the ability of to decode all Serine codons. The column titled [[#x201C]]Anc[[#x201D]] indicates the presence (grey cells) or absence (white cells) of each tRNA in the eubacterial ancestor, inferred from ancestral reconstruction (see Methods). Value in each cell indicates the posterior probability of the inferred state for each tRNA.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F1.medium.gif" width="266" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F1.medium.gif" width="266" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 1:</span>
       <span class="caption-title">The evolutionary history of bacterial tRNA modifications.</span>
       <p id="p-8" class="first-child">(A) Columns indicate the presence (gray) or absence (white) of various tRNA modifying enzymes as noted above each column. The leftmost column shows two essential housekeeping proteins (RpoB-RNA polymerase beta subunit and RplA-50S Ribosomal protein L1) that serve as positive controls for homology detection. (B) Bacterial phylogenetic tree (
        <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Segata et al., 2013</a>) pruned to show the evolutionary relationships between the 1093 bacterial species used in our study. Branch tips are colored by taxonomic group. Filled and open circles respectively mark the major gain and loss of tRNA modifications (with at least 5 descendent species in the relevant lineage), as inferred by ancestral reconstruction (see Methods). Circles are colored according to the modification, as indicated in panel A. Values in parentheses indicate the posterior probability of each gain or loss event for each ME involved in the modification pathway. All other minor gain and loss events (with &lt;; 5 descendent species in the pruned phylogeny) are shown in Fig. S1. Asterisks at the root of the tree indicate the presence of the respective modification. (C) Genetic code table showing the target (solid boxes) and non-target tRNA (dashed boxes) for each modification, colored as in panel A. tRNA with a yellow border show G::U wobble base pairing and were not considered in subsequent 
        <sup>16</sup> analyses. Serine tRNA are highlighted with a black box to indicate the ability of 
        <span class="inline-formula" id="inline-formula-1">
         <span class="highwire-responsive-lazyload">
          <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/22/251322/F1/embed/inline-graphic-1.gif"/>
          <noscript>
           <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/22/251322/F1/embed/inline-graphic-1.gif"/>
          </noscript>
         </span>
        </span> to decode all Serine codons. The column titled “Anc” indicates the presence (grey cells) or absence (white cells) of each tRNA in the eubacterial ancestor, inferred from ancestral reconstruction (see Methods). Value in each cell indicates the posterior probability of the inferred state for each tRNA.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-9">To determine the key evolutionary changes in major tRNA modification pathways across bacteria, we performed ancestral reconstruction with stochastic character mapping for each ME (
      <a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1A</a>; Table S1). We used a posterior probability value of ≥ 0.7 to assign the ME state at each internal node in the bacterial phylogeny. Four of the five modification systems – MnmE-MnmG, Tgt-QueA-QueG, TadA and TilS – were predicted to already occur at the root of the phylogenetic tree, i.e. were present in the eubacterial ancestor (
      <a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1B</a>). In contrast, CmoA-CmoB were gained/evolved much later: once at the root of γ-proteobacteria, once in ∊-proteobacteria, and once in some species of δ-proteobacteria (
      <a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1B</a>). Thus, except for the cmo5U modification, most bacterial tRNA modifications are ancient. These ancient modifications were then independently lost multiple times, with a total of 12 major losses (on branches with at least 5 descendant bacterial species in our phylogeny; 
      <a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig. 1B</a>) and 19 minor loss events (in lineages with fewer than five descendants; Fig. S1). The only exception to this pattern of repeated ME gains/losses is the k2C modification (TilS), which appears to be evolutionarily stable across bacteria. Overall, our analysis demonstrates a highly dynamic evolutionary history of tRNA modifications that is dominated by repeated secondary losses and only a few gains (a total of 44 loss events and 13 gains across the phylogeny; Fig. S1).
     </p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>tRNA modification loss is associated with more non-target tRNAs</h3>
     <p id="p-10">The early evolution of nearly all tRNA modifications suggests that the eubacterial ancestor should have been able to decode most sense codons using only a subset of all possible tRNAs. Indeed, ancestral reconstruction of tRNA genes suggests that the eubacterial ancestor did have all target 
      <sup>14</sup> tRNAs 
      <span class="inline-formula" id="inline-formula-2">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/22/251322/embed/inline-graphic-2.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/22/251322/embed/inline-graphic-2.gif"/>
        </noscript>
       </span>
      </span> but did not possess non-target tRNAs for the Q, k2C and I modifications (
      <a id="xref-fig-1-7" class="xref-fig" href="#F1">Fig. 1C</a>; note that 1C; note that the state of 
      <span class="inline-formula" id="inline-formula-3">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/22/251322/embed/inline-graphic-3.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/22/251322/embed/inline-graphic-3.gif"/>
        </noscript>
       </span>
      </span> is ambiguous). Interestingly, for the (c)mnm5(s2)U modification, both target and non-target tRNA were present in the eubacterial ancestor. Thus, a subsequent loss of any of these modifications should be strongly associated with the gain (or retention, in case of (c)mnm5(s2)U) of the respective non-target tRNA. Both predictions were supported when we observed that the proportion of species that have non-target tRNA is significantly different between bacteria with vs. without a specific modification (p &lt;; 0.05, 2-sample proportions test; Fig. S2). Overall, these patterns suggest that the loss of tRNA modifications was associated with the retention or secondary gain of non-target tRNAs, and the gain of MEs was associated with the loss of non-target tRNAs.
     </p>
     <p id="p-11">To account for phylogenetic relationships between analysed bacteria, we compared the tRNA gene content of sister clades with contrasting modification status; i.e. following a major modification gain or loss event (see Methods; 
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>). For instance, since the (c)mnm5(s2)U modification was lost at the root of Actinobacteria (MnmE/MnmG enzymes, 
      <a id="xref-fig-1-8" class="xref-fig" href="#F1">Fig. 1B</a>), we compared the tRNA gene content of species within the Actinobacteria clade with that of species in the sister group: the Cyanobacteria-Deinococcus-Thermus (CDT) clade. Similarly, we compared the target and non-target tRNA gene copy numbers for descendent sister clades for 14 of the 15 major modification gain and loss events highlighted in 
      <a id="xref-fig-1-9" class="xref-fig" href="#F1">Fig. 1B</a> (we did not analyze Thermotogae, because its sister clade would consist of all other bacteria). Of these, 4 cases involving the Q modification are not informative with regard to non-target tRNAs because these are absent in all bacteria (
      <a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2F–2I</a>). However, we describe these comparisons below since they are useful for understanding the evolution of target tRNAs. 
     </p>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="tRNA gene content associated with modification gain or loss in sister clades. Gene copy number of target and non-target tRNA within sister clades with contrasting modification status (marked in Fig. 1B). Stacked bar plots show the proportion of species within each sister clade that lack a specific tRNA (blue bars), have a single copy of the gene (light red bars), or have multiple copies (dark red bars). The total number of species analyzed in each clade are given in parentheses above the bar plots. Each panel shows data for a specific sister clade comparison; panels are grouped by modification pathway. Gray shading represents anticodons (i.e. tRNA) for which the proportion of species without the tRNA is significantly different across groups, with respect to modification presence vs. absence (p &lt;; 0.05; 2-sample test for equality of proportions with continuity correction). CDT [[#x2013]] Cyanobacteria-Deinococcus-Thermus, FCB [[#x2013]] Fibrobacteres-Chlorobi-Bacteroidetes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-449755067" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;tRNA gene content associated with modification gain or loss in sister clades.&lt;;/span&gt;; Gene copy number of target and non-target tRNA within sister clades with contrasting modification status (marked in Fig. 1B). Stacked bar plots show the proportion of species within each sister clade that lack a specific tRNA (blue bars), have a single copy of the gene (light red bars), or have multiple copies (dark red bars). The total number of species analyzed in each clade are given in parentheses above the bar plots. Each panel shows data for a specific sister clade comparison; panels are grouped by modification pathway. Gray shading represents anticodons (i.e. tRNA) for which the proportion of species without the tRNA is significantly different across groups, with respect to modification presence vs. absence (p &lt;; 0.05; 2-sample test for equality of proportions with continuity correction). CDT [[#x2013]] Cyanobacteria-Deinococcus-Thermus, FCB [[#x2013]] Fibrobacteres-Chlorobi-Bacteroidetes.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F2.medium.gif" width="346" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F2.medium.gif" width="346" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 2:</span>
       <span class="caption-title">tRNA gene content associated with modification gain or loss in sister clades.</span>
       <p id="p-12" class="first-child">Gene copy number of target and non-target tRNA within sister clades with contrasting modification status (marked in 
        <a id="xref-fig-1-10" class="xref-fig" href="#F1">Fig. 1B</a>). Stacked bar plots show the proportion of species within each sister clade that lack a specific tRNA (blue bars), have a single copy of the gene (light red bars), or have multiple copies (dark red bars). The total number of species analyzed in each clade are given in parentheses above the bar plots. Each panel shows data for a specific sister clade comparison; panels are grouped by modification pathway. Gray shading represents anticodons (i.e. tRNA) for which the proportion of species without the tRNA is significantly different across groups, with respect to modification presence vs. absence (p &lt;; 0.05; 2-sample test for equality of proportions with continuity correction). CDT – Cyanobacteria-Deinococcus-Thermus, FCB – Fibrobacteres-Chlorobi-Bacteroidetes.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-13">In most of the informative sister clade comparisons (8 out of 10 cases), species were more likely to have non-target tRNAs if the modification was absent (compare bottom left vs. bottom right bar plots within each panel of 
      <a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2</a>), supporting our hypothesis that modification loss (gain) is associated with the presence (absence) of non-target tRNAs. For instance, in three out of the five comparisons made for the (c)mnm5(s2)U and cmo5U modifications, at least 5 of the 6 non-target tRNAs were present in a significantly higher proportion of species without the modification (
      <a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2A, 2C</a> and 
      <a id="xref-fig-2-5" class="xref-fig" href="#F2">2D</a>; p &lt;; 0.05, 2-sample proportions test). The two exceptions to this pattern occur in clades with relatively low sample sizes (
      <a id="xref-fig-2-6" class="xref-fig" href="#F2">Fig. 2B</a> and 
      <a id="xref-fig-2-7" class="xref-fig" href="#F2">2E</a>), and may potentially reflect low statistical power. Alternatively, these may be biologically interesting exceptions that require further examination. Additionally, in all five comparisons made for the I modification, at least one non-target tRNA (in most cases both) was present in significantly more species without the modification, compared to species with the modification (
      <a id="xref-fig-2-8" class="xref-fig" href="#F2">Fig. 2J – 2N</a>; p &lt;; 0.05, 2-sample proportions test).
     </p>
     <p id="p-14">In contrast to the wide variation in the occurrence of non-target tRNAs, we found that target tRNA genes were largely stable irrespective of species’ modification status (top row in each panel of 
      <a id="xref-fig-2-9" class="xref-fig" href="#F2">Fig. 2</a>). Contrary to expectation, the gain or loss of the (c)mnm5(s2)U and cmo5U modifications does not appear to have any impact on target tRNAs (
      <a id="xref-fig-2-10" class="xref-fig" href="#F2">Fig. 2A–2E</a>; p &gt;; 0.05, 2-sample proportions test); likely because the corresponding codons are still used in these species and cannot be decoded by non-target tRNAs. In the case of the Q modification, target tRNAs were always present; but in 2 of 4 comparisons their copy number was lower in clades that had lost the ME (
      <a id="xref-fig-2-11" class="xref-fig" href="#F2">Fig. 2F–2I</a>; p &lt;; 0.05, 2-sample proportions test). Inosine presents the only evidence supporting weakened selection on target tRNAs upon ME loss. In 3 of 5 comparisons, target tRNAs were absent in nearly all species without the modification (
      <a id="xref-fig-2-12" class="xref-fig" href="#F2">Fig. 2J–2L</a>). In the remaining 2 comparisons, target tRNAs were present in fewer species without modifications (
      <a id="xref-fig-2-13" class="xref-fig" href="#F2">Fig. 2M</a> and 
      <a id="xref-fig-2-14" class="xref-fig" href="#F2">2N</a>; p &lt;; 0.05, 2-sample proportions test). Together, these results suggest that except for the I modification, the evolution of modifications had no impact on the evolution of target tRNA genes.
     </p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Modification status is associated with tRNA diversity, GC content and genome size</h3>
     <p id="p-15">To identify the cause and impact of evolutionary changes in tRNA modifications, we compared key translation-associated genomic features of sister clades with contrasting modification status. As shown above, we found that in 11 of 14 comparisons, the tRNA diversity of sister clades was significantly different (p &lt;; 0.05, Wilcoxon rank sum test; 
      <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3</a> Fig. S3), indicating a strong association between the evolution of modifications and the tRNA gene pool. However, in four of these cases the direction of association opposed our initial expectation, and clades without the modification had lower tRNA diversity (
      <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figs. 3G, 3I, 3M</a> and 
      <a id="xref-fig-3-3" class="xref-fig" href="#F3">3N</a>). In three of these cases involving Tenericutes and the endosymbiotic species from Enterobacterales (
      <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figs. 3G, 3I</a> and 
      <a id="xref-fig-3-5" class="xref-fig" href="#F3">3M</a>), we also observed a significant reduction in the total number of tRNA genes (Fig. S3G, S3I, S3M), suggesting wide-ranging gene loss that is potentially a result of genome reduction in these clades. Indeed, previous reports have shown that reductive genome evolution in Mollicutes has led to the loss of several tRNA modifying enzymes (
      <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Grosjean et al., 2014</a>; 
      <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Yokobori, Kitamura, Grosjean, &amp;; Bessho, 2013</a>). Thus, overall half the sister clade comparisons supported our initial prediction that the lack of modifications should be associated with greater tRNA diversity. Next, we tested whether changes in modification status and tRNA genes were also associated with changes in GC content and codon use. 
     </p>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Differences in key genomic features across sister clades with contrasting modification status. Box plots show the range of values of three genomic traits, across all species within a sister clade (see Fig. 3; red boxes = modification absent; green boxes = modification present). For the Relative Synonymous Codon Usage (RSCU) panel, asterisks indicate that the difference between the distributions represented by green boxplots is significantly different than the difference between the distributions represented by the red boxplots (i.e., codon usage is more biased in one case; p &lt;; 0.05; Wilcoxon rank sum test with p-values correction for multiple comparisons). For tRNA types, GC3 (GC content of the third codon base) and genome size, asterisks indicate a significant difference in distributions across the sister clades (p &lt;; 0.05; Wilcoxon rank sum test with p-values correction for multiple comparisons)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-449755067" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Differences in key genomic features across sister clades with contrasting modification status.&lt;;/span&gt;; Box plots show the range of values of three genomic traits, across all species within a sister clade (see Fig. 3; red boxes = modification absent; green boxes = modification present). For the Relative Synonymous Codon Usage (RSCU) panel, asterisks indicate that the difference between the distributions represented by green boxplots is significantly different than the difference between the distributions represented by the red boxplots (i.e., codon usage is more biased in one case; p &lt;; 0.05; Wilcoxon rank sum test with p-values correction for multiple comparisons). For tRNA types, GC3 (GC content of the third codon base) and genome size, asterisks indicate a significant difference in distributions across the sister clades (p &lt;; 0.05; Wilcoxon rank sum test with p-values correction for multiple comparisons).&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F3.medium.gif" width="440" height="382"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F3.medium.gif" width="440" height="382"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 3:</span>
       <span class="caption-title">Differences in key genomic features across sister clades with contrasting modification status.</span>
       <p id="p-16" class="first-child">Box plots show the range of values of three genomic traits, across all species within a sister clade (see 
        <a id="xref-fig-3-6" class="xref-fig" href="#F3">Fig. 3</a>; red boxes = modification absent; green boxes = modification present). For the Relative Synonymous Codon Usage (RSCU) panel, asterisks indicate that the difference between the distributions represented by green boxplots is significantly different than the difference between the distributions represented by the red boxplots (i.e., codon usage is more biased in one case; p &lt;; 0.05; Wilcoxon rank sum test with p-values correction for multiple comparisons). For tRNA types, GC3 (GC content of the third codon base) and genome size, asterisks indicate a significant difference in distributions across the sister clades (p &lt;; 0.05; Wilcoxon rank sum test with p-values correction for multiple comparisons).
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-17">In most cases showing altered tRNA diversity (8 out of 11), we found that the change in diversity was also associated with significant shifts in GC content, so that species in clades without the tRNA modification typically had more extreme GC3 values (i.e. GC content in the third base of codons, 
      <a id="xref-fig-3-7" class="xref-fig" href="#F3">Fig. 3</a>; p &lt;; 0.05, Wilcoxon rank sum test; we observed similar results with genomic GC content, Fig. S3). In seven of these eight cases, species without the modification also had significantly more biased codon use (measured as relative synonymous codon use RSCU for all proteins; p &lt;; 0.05, Wilcoxon rank sum test; we observed similar results for RSCU of highly expressed ribosomal proteins; Fig. S3). Finally, in five of the eight cases we observed significant differences in genome size across sister clades (
      <a id="xref-fig-3-8" class="xref-fig" href="#F3">Fig. 3</a>). Of these, in the two comparisons that involved Actinobacteria (
      <a id="xref-fig-3-9" class="xref-fig" href="#F3">Figs. 3A</a> and 
      <a id="xref-fig-3-10" class="xref-fig" href="#F3">3F</a>) we observed that larger genomes were associated with the absence of the modification. As mentioned above, in the other three cases involving Tenericutes and Enterobacterales (
      <a id="xref-fig-3-11" class="xref-fig" href="#F3">Figs. 3G, 3I</a> and 
      <a id="xref-fig-3-12" class="xref-fig" href="#F3">3M</a>), the lack of modification may reflect overall genome reduction. Note that in every comparison where codon use and genome size were different, GC content was also skewed significantly. Thus, to test the overall effect of genome size and GC content on modification status, we carried out a phylogenetic regression. We found that both GC content and genome size had a significant effect on the presence/absence of the Q modification alone (Fig S4). These results suggest that except for the Q modification, modifications were lost in only a few instances of shifts in GC content. Thus, GC content may be the major ultimate driver of changes in tRNA gene content and the subsequent loss of modification may generate selection to fix these changes. Altogether, three of the 14 major changes in modification status were not associated with significant shifts in either tRNA or GC content, and remain unexplained. These included one instance of the loss of the (c)mnm5(s2)U modification (
      <a id="xref-fig-3-13" class="xref-fig" href="#F3">Fig. 3B</a>) and two losses of the I modification (
      <a id="xref-fig-3-14" class="xref-fig" href="#F3">Figs. 3K–L</a>).
     </p>
     <p id="p-18">Finally, to quantify the evolutionary impact of tRNA modifications on tRNA gene pools, we simultaneously mapped changes in tRNA genes copies, MEs, and GC content on the bacterial phylogeny. As expected, following major GC shifts, non-target tRNA showed several gain/loss events whereas target tRNA were more stable (
      <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figs. 4</a>–
      <a id="xref-fig-5-1" class="xref-down-link" href="#F5">
       <span>⇓</span>
      </a>
      <a id="xref-fig-6-1" class="xref-fig" href="#F6">6</a>). As suggested by our sister clade comparison, in some cases modification gain or loss was also preceded by a GC shift (8 out of 14 independent lineages). Moreover, a phylogenetic regression to test the impact of GC and MEs on tRNA genes revealed that GC content had the strongest impact on the dynamics of non-target tRNA genes (
      <a id="xref-fig-7-1" class="xref-fig" href="#F7">Fig. 7</a>). However, as observed in the sister clade comparison, the phylogenetic regression and mapping also reveal clear instances of GC-independent impacts of modification status on tRNA genes (
      <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figs. 4</a>–
      <a id="xref-fig-6-2" class="xref-down-link" href="#F6">
       <span>⇓</span>
      </a>
      <a id="xref-fig-7-2" class="xref-fig" href="#F7">7</a>). The strongest impact of tRNA modification is observed the case of the Inosine, where the target tRNA was present in all species when the modification was present (
      <a id="xref-fig-6-3" class="xref-fig" href="#F6">Fig. 6</a>); and the non-target tRNA were absent in most of these species (
      <a id="xref-fig-2-15" class="xref-fig" href="#F2">Fig. 2D</a>). On the other hand, the loss of target tRNAs and presence of non-target tRNAs is strongly associated with the loss of the Inosine modification (
      <a id="xref-fig-2-16" class="xref-fig" href="#F2">Fig. 2D</a>, 
      <a id="xref-fig-6-4" class="xref-fig" href="#F6">Fig. 6</a>, 
      <a id="xref-fig-7-3" class="xref-fig" href="#F7">Fig. 7</a>). Thus, the bacterial tRNA repertoire is strongly affected by evolutionary changes in GC content as well as tRNA modifying enzymes. 
     </p>
     <div id="F4" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Evolutionary dynamics of tRNA genes, MEs and GC content for the (c)mnm5(s2)U modification. Branches are colored according to the inferred GC content for each branch, with white indicating low GC% and black indicating high GC%. Nodes that showed the largest GC shifts are indicated with black circles. Colored open and closed circles respectively indicate the loss and gain of each ME involved in the modification. Open and closed triangles indicate the loss and gain of tRNA genes relevant to the (c)mnm5(s2)U modification, with target tRNA indicated on the tree on the left and non-target tRNA indicated on the mirror tree on the right." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-449755067" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Evolutionary dynamics of tRNA genes, MEs and GC content for the (c)mnm5(s2)U modification.&lt;;/span&gt;; Branches are colored according to the inferred GC content for each branch, with white indicating low GC% and black indicating high GC%. Nodes that showed the largest GC shifts are indicated with black circles. Colored open and closed circles respectively indicate the loss and gain of each ME involved in the modification. Open and closed triangles indicate the loss and gain of tRNA genes relevant to the (c)mnm5(s2)U modification, with target tRNA indicated on the tree on the left and non-target tRNA indicated on the mirror tree on the right.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F4.medium.gif" width="350" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F4.medium.gif" width="350" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 4:</span>
       <span class="caption-title">Evolutionary dynamics of tRNA genes, MEs and GC content for the (c)mnm5(s2)U modification.</span>
       <p id="p-19" class="first-child">Branches are colored according to the inferred GC content for each branch, with white indicating low GC% and black indicating high GC%. Nodes that showed the largest GC shifts are indicated with black circles. Colored open and closed circles respectively indicate the loss and gain of each ME involved in the modification. Open and closed triangles indicate the loss and gain of tRNA genes relevant to the (c)mnm5(s2)U modification, with target tRNA indicated on the tree on the left and non-target tRNA indicated on the mirror tree on the right.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F5" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Evolutionary dynamics of tRNA genes, MEs and GC content for the cmo5U modification Branches are colored according to the inferred GC content for each branch, with white indicating low GC% and black indicating high GC%. Nodes that showed the largest GC shifts are indicated with black circles. Colored open and closed circles respectively indicate the loss and gain of each ME involved in the modification. Open and closed triangles indicate the loss and gain of tRNA genes relevant to the cmo5U modification, with target tRNA indicated on the tree on the left and non-target tRNA indicated on the mirror tree on the right." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-449755067" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Evolutionary dynamics of tRNA genes, MEs and GC content for the cmo5U modification&lt;;/span&gt;; Branches are colored according to the inferred GC content for each branch, with white indicating low GC% and black indicating high GC%. Nodes that showed the largest GC shifts are indicated with black circles. Colored open and closed circles respectively indicate the loss and gain of each ME involved in the modification. Open and closed triangles indicate the loss and gain of tRNA genes relevant to the cmo5U modification, with target tRNA indicated on the tree on the left and non-target tRNA indicated on the mirror tree on the right.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F5.medium.gif" width="355" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F5.medium.gif" width="355" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 5:</span>
       <span class="caption-title">Evolutionary dynamics of tRNA genes, MEs and GC content for the cmo5U modification</span>
       <p id="p-20" class="first-child">Branches are colored according to the inferred GC content for each branch, with white indicating low GC% and black indicating high GC%. Nodes that showed the largest GC shifts are indicated with black circles. Colored open and closed circles respectively indicate the loss and gain of each ME involved in the modification. Open and closed triangles indicate the loss and gain of tRNA genes relevant to the cmo5U modification, with target tRNA indicated on the tree on the left and non-target tRNA indicated on the mirror tree on the right.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F6" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Evolutionary dynamics of tRNA genes, MEs and GC content for the I modification Branches are colored according to the inferred GC content for each branch, with white indicating low GC% and black indicating high GC%. Nodes that showed the largest GC shifts are indicated with black circles. Colored open and closed circles respectively indicate the loss and gain of each ME involved in the modification. Open and closed triangles indicate the loss and gain of tRNA genes relevant to the I modification, with target tRNA indicated on the tree on the left and non target tRNA indicated on the mirror tree on the right." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-449755067" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Evolutionary dynamics of tRNA genes, MEs and GC content for the I modification&lt;;/span&gt;; Branches are colored according to the inferred GC content for each branch, with white indicating low GC% and black indicating high GC%. Nodes that showed the largest GC shifts are indicated with black circles. Colored open and closed circles respectively indicate the loss and gain of each ME involved in the modification. Open and closed triangles indicate the loss and gain of tRNA genes relevant to the I modification, with target tRNA indicated on the tree on the left and non target tRNA indicated on the mirror tree on the right.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F6.medium.gif" width="354" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 6:" src="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F6.medium.gif" width="354" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6:" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 6:</span>
       <span class="caption-title">Evolutionary dynamics of tRNA genes, MEs and GC content for the I modification</span>
       <p id="p-21" class="first-child">Branches are colored according to the inferred GC content for each branch, with white indicating low GC% and black indicating high GC%. Nodes that showed the largest GC shifts are indicated with black circles. Colored open and closed circles respectively indicate the loss and gain of each ME involved in the modification. Open and closed triangles indicate the loss and gain of tRNA genes relevant to the I modification, with target tRNA indicated on the tree on the left and non target tRNA indicated on the mirror tree on the right.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F7" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F7.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Results of phylogenetic regressions for factors affecting tRNA gene presence or absence. Bar plots show the R2 value for the phylogenetic regression for presence/absence of target and non-target tRNAs for the (c)mnm5(s2)U, cmo5U and I modifications. Results are shown for three different models: (A) Full model, tRNA presence/absence ~ Modification status (i.e. presence/absence) * GC content. (B) tRNA presence/absence ~ GC content (C) tRNA presence/absence ~ Modification status (i.e. presence/absence). Asterisks indicate a significant impact of the focal trait on the presence of the tRNA gene (see Methods). Target tRNAs are highlighted with a solid underline, and non-target tRNAs with a dashed underline. Note the different y-axis scale in panel C." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-449755067" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Results of phylogenetic regressions for factors affecting tRNA gene presence or absence.&lt;;/span&gt;; Bar plots show the R&lt;;sup&gt;;2&lt;;/sup&gt;; value for the phylogenetic regression for presence/absence of target and non-target tRNAs for the (c)mnm5(s2)U, cmo5U and I modifications. Results are shown for three different models: (A) Full model, tRNA presence/absence ~ Modification status (i.e. presence/absence) * GC content. (B) tRNA presence/absence ~ GC content (C) tRNA presence/absence ~ Modification status (i.e. presence/absence). Asterisks indicate a significant impact of the focal trait on the presence of the tRNA gene (see Methods). Target tRNAs are highlighted with a solid underline, and non-target tRNAs with a dashed underline. Note the different y-axis scale in panel C.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 7:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F7.medium.gif" width="440" height="331"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 7:" src="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F7.medium.gif" width="440" height="331"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7:" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/22/251322/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 7:</span>
       <span class="caption-title">Results of phylogenetic regressions for factors affecting tRNA gene presence or absence.</span>
       <p id="p-22" class="first-child">Bar plots show the R
        <sup>2</sup> value for the phylogenetic regression for presence/absence of target and non-target tRNAs for the (c)mnm5(s2)U, cmo5U and I modifications. Results are shown for three different models: (A) Full model, tRNA presence/absence ~ Modification status (i.e. presence/absence) * GC content. (B) tRNA presence/absence ~ GC content (C) tRNA presence/absence ~ Modification status (i.e. presence/absence). Asterisks indicate a significant impact of the focal trait on the presence of the tRNA gene (see Methods). Target tRNAs are highlighted with a solid underline, and non-target tRNAs with a dashed underline. Note the different y-axis scale in panel C.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
   </div>
   <div class="section" id="sec-6">
    <h2 class="">DISCUSSION</h2>
    <p id="p-23">Over 50 years ago, Francis Crick described the phenomenon of wobble base pairing where one tRNA molecule can recognize multiple codons (
     <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Crick, 1966</a>). Since then, a large body of work has developed a detailed understanding of the enzymatic pathways that generate extended wobble pairing via base modifications of tRNA anticodons (reviewed in (
     <a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">Björk &amp;; Hagervall, 2014</a>)). Given their critical role in translation, modifying enzymes are expected to evolve under strong selection, and in turn influence the evolution of other components of the translation machinery (
     <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Grosjean et al., 2010</a>). Here, we trace the evolutionary history of tRNA modifications across the bacterial phylogeny, showing that the eubacterial ancestor already had nearly all known modification pathways. Several major bacterial clades subsequently lost the modifications, often accompanied by increasing tRNA diversity and shifts in genome GC content and codon use. These results are consistent with the hypothesis that an expanded tRNA pool gradually weakens selection on MEs, allowing their loss through drift. However, once the modification is lost, bacteria likely face strong selection to retain the expanded tRNA pool. In contrast, if GC shifts shrink the tRNA gene pool, selection would strongly favor the innovation or retention of tRNA modifications. We speculate that in Proteobacteria, such a process may have led to the evolution of the novel cmo5U modification. Alternatively, Proteobacteria may have evolved the new modification by chance, secondarily weakening selection on the non-target tRNAs. Interestingly, we also uncovered instances where changes in tRNA or GC content could not explain the evolution of tRNA modifying enzymes. In some of these cases, we suspect that overall genome reduction may have led to the loss of the modification; but three major cases of modification loss remain unexplained (loss of (c)mnm5(s2)U in Leuconostoc and I in Spirochaetes and ∊-proteobacteria). Nonetheless, our results strongly support the conclusion that ME evolution was dominated by frequent losses driven by weakened selection. Note that reversing the direction of causality in our arguments would not make logical sense: the loss of non-target tRNAs cannot generate selection favoring the loss of tRNA modifications, and there is no reason to expect that the gain of a modification enzyme should selectively favor the retention on non-target tRNAs. Similarly, it is difficult to imagine how evolutionary gains or losses of tRNA modifications could precipitate genome-scale changes in GC content or genome size. Thus, our systematic phylogenetic analysis clarifies the contrasting roles of strong vs. weak selection acting on tRNA modifications, and on the evolutionary history of key components of bacterial translation.
    </p>
    <p id="p-24">Our results were generally consistent with the prediction that a lack of specific tRNA modifications should result in strong positive selection favoring the relevant non-target tRNAs. However, we also observed some instructive exceptions to this pattern. For instance, comparing across sister clades, we found that ~33% of the species that do not have the cmo5U modification also lack at least one non-target tRNA. This should pose a problem because codons recognized by the missing non-target tRNAs cannot be decoded efficiently. However, we observed that these codons were significantly depleted in these genomes, compared to species that retained the non-target tRNA (Fig. S5). Thus, these relatively rare codons may not pose a major problem even if they are inefficiently translated. It is also possible that these codons are decoded by the respective U-starting target tRNAs that can pair with A, G or U ending codons (
     <a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">Grosjean et al., 2010</a>). This scenario is especially likely in species where the Q modification is absent. Recall that all bacteria lack the non-target tRNA for this modification; hence species without the Q modification should be unable to translate many codons. We hypothesize that in species where both the Q modification as well as the non-target tRNA are missing, there is an as yet undiscovered G::U wobble base pairing between the target tRNA and codons complementary to the non-target tRNA in these species.
    </p>
    <p id="p-25">Our analysis quantifies the role of specific tRNAs in the evolutionary dynamics of bacterial translation, both as drivers of selection on tRNA modifications and as a means to rapidly respond to shifts in genomic GC content. Specifically, we found that target tRNAs are evolutionarily largely stable (except for those modified by Inosine), whereas non-target tRNAs are often gained or lost in lineages following a GC shift. The strong phylogenetic correlation between non-target tRNAs and genomic GC content echoes the previously reported association between GC content and “auxiliary” tRNAs (tRNAs with a high number of gains and losses) defined by Wald and Margalit (
     <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Wald &amp;; Margalit, 2014</a>). The nearly ubiquitous presence of target tRNAs may reflect strong positive selection, since the function of target tRNA cannot be carried out by any other tRNA via wobble base pairing rules. Thus, target tRNA may be essential even if their complementary codons are used relatively rarely.
    </p>
    <p id="p-26">In summary, we propose that the evolution of diverse bacterial tRNA modification enzymes has been strongly influenced by changes in tRNA gene pools and genomic GC content, as well as by genome reduction in specific lineages. Our results support the hypothesis that these changes weakened selection on MEs and allowed their loss through drift. Conversely, we hypothesize that the loss of non-target tRNA may have generated strong selection for the innovation of the cmo5U modification in γ-proteobacteria and ε-proteobacteria. It is also possible that the tRNA loss succeeded only in the wake of the chance innovation of the cmo5U modification. These patterns generate a number of broad, testable predictions about the future evolutionary trajectory of translational components in various bacterial lineages. For instance, MEs may be lost in GC-rich bacteria without major consequences for translation; such losses should evolutionarily fix the expanded tRNA repertoire; and MEs should face strong positive selection in AT-rich clades with low tRNA diversity. Specifically, in AT rich bacteria, the combination of MEs and largely target tRNAs should confer higher fitness than a full tRNA set without MEs. Our analysis also shows that multiple key components of bacterial translation have evolved in close association with genomic GC content. Although the evolutionary and ecological forces driving GC shifts and GC-independent changes in tRNA diversity remain unclear, our analyses highlight the rich evolutionary dynamics of wobble pairing, and more broadly, bacterial translation.</p>
   </div>
   <div class="section" id="sec-7">
    <h2 class="">METHODS</h2>
    <div id="sec-8" class="subsection">
     <h3>Phylogenetic tree, genomes and tRNA gene data</h3>
     <p id="p-27">All analyses were carried out in R (
      <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">R Development Core Team, 2015</a>) using the phytools (
      <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Revell, 2012</a>) and seqinr (
      <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Charif, Thioulouse, Lobry, &amp;; Perrière, 2005</a>) packages, unless otherwise specified. The R scripts for all analyses are available at 
      <a href="https://github.com/gauravdiwan89/me_evolution_project">https://github.com/gauravdiwan89/me_evolution_project</a>. We used the bacterial phylogeny generated by Segata and co-workers (
      <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Segata, Börnigen, Morgan, &amp;; Huttenhower, 2013</a>). This phylogeny was generated using an alignment of the 400 most conserved protein sequences across all prokaryotes. We downloaded the phylogeny (
      <a href="https://bitbucket.org/nsegata/phylophlan/wiki/bs_tree.reroot.nwk">https://bitbucket.org/nsegata/phylophlan/wiki/bs_tree.reroot.nwk</a>) and removed all archaea, as well as bacteria whose genomes have not been fully sequenced. We pruned the tree further using a distance based pruning function. Briefly, when closely related species had a pairwise phylogenetic distance ≤0.03, we randomly sampled only one of these species. This allowed us to remove the excessive representation of some highly sequenced genomes, such as several 
      <em>Escherichia coli</em> strains. This led to the final set of 1093 bacterial genomes that we used in our subsequent analyses.
     </p>
     <p id="p-28">We downloaded the tRNA gene copy numbers for each of these genomes from the Genomic tRNA database (
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chan &amp;; Lowe, 2009</a>). For genomes where this information was not available, we determined the tRNA gene copy numbers by running the program tRNAscan-SE (
      <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Lowe &amp;; Eddy, 1996</a>) using the default parameters for bacterial genomes. The information for each of these genomes was represented using different identifiers in the various databases that we used. This information is consolidated in Table S1, consisting of: i) species name according to NCBI taxonomy, ii) NCBI accession number, iii) Phylogenetic tree tip label in the tree, iv) IMG database identifier and v) Genomic tRNA database abbreviation.
     </p>
    </div>
    <div id="sec-9" class="subsection">
     <h3>Choice of crucial tRNA MEs and non-target tRNA</h3>
     <p id="p-29">The (c)mnm5(s2)U and Q modifications are created by elaborate ME pathways. For instance, our definition of the (c)mnm5(s2)U modification includes three different modifications: mnm5U, cmnm5U and cmnm5s2U, each of them acting on different isoaccepting tRNA. These modifications have unique MEs that are involved in the last steps of the pathway. MnmE and MnmG are the common enzymes for each of these modifications and they carry out the first step of the modifying the Uridine base (
      <a id="xref-ref-1-6" class="xref-bibr" href="#ref-1">Björk &amp;; Hagervall, 2014</a>) and references therein. Thus, we considered MnmE-MnmG to be crucial, and other enzymes such as MnmC, MnmA, MnmH and trmL as auxiliary. The Q base is first biochemically synthesized as Q
      <sub>1</sub> in the bacterial cell using the enzymes FolE, QueC, QueD, QueE and QueF. Subsequently, the Tgt enzyme replaces Guanosine in the anticodon stem loop with Q
      <sub>1</sub> and QueA and QueG convert it to Q ((
      <a id="xref-ref-1-7" class="xref-bibr" href="#ref-1">Björk &amp;; Hagervall, 2014</a>) and references within). A recent report has suggested the possibility of Tgt alone carrying out the Q modification in the absence of QueA and QueG, assuming the existence of a Q salvage pathway (
      <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Zallot et al., 2017</a>). However, there is no evidence for this phenomenon across most bacteria from our analysis; hence, we considered Tgt, QueA and QueG as the crucial enzymes for the Q modification.
     </p>
     <p id="p-30">The cmo5U modification exists in tRNA molecules for four box codons (Leucine, Valine, Serine, Proline, Threonine and Alanine) and therefore there were three possible non-target tRNA in each case. However, tRNA with GNN anticodons (where N is any nucleotide) in each of these codon boxes can also recognize NNU codons via G::U wobble pairing. Thus, we only focused on the non-target tRNA that have CNN anticodons (indicated in 
      <a id="xref-fig-1-11" class="xref-fig" href="#F1">Fig. 1C</a>).
     </p>
    </div>
    <div id="sec-10" class="subsection">
     <h3>Homology search for tRNA MEs</h3>
     <p id="p-31">We determined the presence/absence of all known tRNA MEs in the 1093 bacterial genomes using Hidden Markov Model (HMM) searches (S. 
      <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Eddy, 1998</a>). First, we downloaded all available reviewed protein sequences for each of the MEs from the UniProt-KB database. We created a multiple sequence alignment using the t-coffee program (
      <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Notredame, Higgins, &amp;; Heringa, 2000</a>) with default parameters. We used this alignment to build an HMM profile, and then used the “hmmsearch” command in the HMMER suite (S. R. 
      <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Eddy, 2009</a>) to detect the presence/absence of each ME. We used the amino acid sequences of all proteins from the 1093 genomes (downloaded in November 2016 from the NCBI ftp site) as the sequence search space, and set all other parameters to default. Each search generated a table with accession numbers of potential homologous hits with a corresponding e-value and bit-score. There is no consensus in the field with respect to setting an e-value cut-off to detect true homologs. Here, we present a slightly informative method of deciding e-value cut-offs for detecting true homologs. To infer presence/absence with precision, we implemented a dynamic e-value cut-off to eliminate spurious hits. We did this by plotting the number of species in which the ME would be detected as we increased the e-value cut-off (Fig. S6). We observed stable plateaus where the number of species in which an ME was detected did not change despite changing the e-value cut-off, suggesting that these represent robust e-value thresholds. Thus, for each enzyme we set an enzyme-specific e-value cut-off as the first e-value at the beginning of the most stable plateau (see Fig. S6). For each ME, homologous hits that had e-values lower than this cut-off were considered true homologues.
     </p>
     <p id="p-32">Previous reports have experimentally shown that the cmo5U modification occurs in 
      <em>Bacillus subtilis</em> and 
      <em>Mycobacterium bovis</em> str. BCG (
      <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Chionh et al., 2016</a>; 
      <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Murao et al., 1976</a>; 
      <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Yamada et al., 2005</a>). However, in our initial analysis we did not find CmoA and CmoB homologs in these or related species. A closer inspection of the 
      <em>Bacillus subtilis</em> genome also did not reveal these enzymes, as indicated in a previous report (
      <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Grosjean et al., 2014</a>). Instead, there were a few entries for these enzymes in the UniProt database (each with a poor annotation score). We therefore tested whether these enzymes are true homologs of the proteobacterial CmoA and CmoB, using homology detection (using ‘jackhmmer’). We observed that only the query organisms showed the presence of these enzymes (data not shown). A survey of the SEED database (
      <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Overbeek et al., 2005</a>) also revealed the absence of CmoA and CmoB in all but one 
      <em>Bacillus</em> species. When we carried out a blastp analysis using CmoA and CmoB enzymes from 
      <em>Escherichia coli</em> in 
      <em>Mycobacterium bovis</em> str. BCG, we observed no significant hits for CmoA and no hits for CmoB with e-values &lt;; 5e-3. Thus, although the modification may exist in several species, we could not reliably identify homologs of the proteobacterial CmoA and CmoB in other bacteria.
     </p>
    </div>
    <div id="sec-11" class="subsection">
     <h3>Ancestral reconstruction of tRNA modifications, genomic GC content, and tRNA genes</h3>
     <p id="p-33">We built a presence-absence matrix for each ME and used the binary state of each ME to infer gain or loss events along the phylogeny. We used the stochastic character mapping (
      <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bollback, 2006</a>) function in the phytools package in R, with the following parameters: transition rate matrix determination method – Bayesian MCMC, prior distribution on the root node of the tree – estimated, number of simulations – 500. We fixed the number of simulations such that the posterior probability that an enzyme is present for 100 randomly sampled nodes did not change (Fig. S7). We used the enzyme with the most variable presence/absence state (queG) for this analysis, and assumed that calculations for all other enzymes would be more stable. The mapping resulted in 500 enzyme states at each node of the tree. We determined probabilities of each state at each node and assigned the state that had a posterior probability of ≥ 0.7 (Fig. S8) as the state of the enzyme at that node. In case the probability was &lt;; 0.7, we assumed the same state at the node as the preceding node. To determine gain and loss of tRNA modification pathways, we marked the branch that succeeded the gain or loss of all MEs for a given modification as the focal branch. Only these events are described in this study; however, all data for each enzyme are shown in Fig. S1.
     </p>
     <p id="p-34">We determined the GC content at each ancestral node in the bacterial phylogeny using the StableTraits program (
      <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Elliot &amp;; Mooers, 2014</a>) with 10 million iterations and all other parameters set to default. We then determined 20 nodes whose two immediate descendants had the largest difference in GC. Each of these nodes had at least 10 descendant species. These nodes representing major GC shifts are shown on the phylogeny along with the ancestral GC content (
      <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figs. 4</a>–
      <a id="xref-fig-5-2" class="xref-up-link" href="#F5">
       <span>⇑</span>
      </a>
      <a id="xref-fig-6-5" class="xref-fig" href="#F6">6</a>).
     </p>
     <p id="p-35">We determined the evolutionary history of tRNA gains and losses using a similar method as described for MEs. However, in this case we carried out the ancestral reconstruction separately for each bacterial family, which allowed us to account for family-specific rates of evolution of tRNA genes. We used the ancestral state of each tRNA at the root of each bacterial family to infer the tRNA gene content of the eubacterial ancestor. Briefly, we converted tRNA gene copy numbers for all species into a binary vector by changing the state of tRNA genes with multiple copies to “1” i.e. presence. We then carried out ancestral reconstruction using stochastic character mapping and noted evolutionary transitions on the phylogenetic tree as before.</p>
    </div>
    <div id="sec-12" class="subsection">
     <h3>Identifying sister clades with contrasting modifying enzyme status</h3>
     <p id="p-36">We identified clades where crucial enzymes for each tRNA modification were gained or lost, and determined the closest sister clade where the state of modification was opposite to that of the focal clade. We determined the sister clade agnostic of its taxonomic classification. We checked if there were at least 5 descendant extant species in this clade for each gain or loss event. If this criterion was not met, we picked the next closest sister clade neighboring the common ancestor of the above two clades. Within each clade, we excluded species that had an opposite state for any of the MEs involved in each modification. We assigned the identity of the comparison group using the lowest level of taxonomic classification that encompassed the focal species.</p>
    </div>
    <div id="sec-13" class="subsection">
     <h3>Phylogenetic regression</h3>
     <p id="p-37">We carried out phylogenetic regression to test the impact of GC content and modifcations on tRNA presence/absence, using the Maximum Likelihood Continuous Regression program in BayesTraits, similar to the method used by Wald and Margalit (
      <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Wald &amp;; Margalit, 2014</a>). We estimated the lambda parameter for each case and used default values for all other parameters. For each tRNA, we first tested the full model: tRNA presence/absence ~ GC content*tRNA modification status. To determine the impact of each explanatory variable, we compared the likelihood of the full regression model to the likelihood of the same regression when the R value was set to zero. For reduced models containing only GC content or modification status as the explanatory variable, we compared the likelihood of the regression of the reduced model with the likelihood of the regression of the full model. When these likelihoods were comparable (p &gt;; 0.05; Likelihood Ratio test), we inferred that the reduced model was sufficient to explain tRNA presence/absence, and hence had a significant impact on the evolution of the tRNA. We used the same method for testing the impact of GC content and genome size on modification status (Fig. S4).
     </p>
    </div>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">ACKNOWLEDGEMENTS</h2>
    <p id="p-38">We thank Saurabh Mahajan for help with analyses; and Aparna Agarwal, Shyam Buddh, Vrinda Ravi Kumar, Saurabh Mahajan and Laasya Samhita for constructive comments on the manuscript. This work was funded by the National Centre for Biological Sciences (NCBS-TIFR) and grants from the Department of Science and Technology, India (INSPIRE Faculty award, IFA-1 LSBM-64) and the Council for Scientific and Industrial Research, India (37(1629)/14/EMR-II). GDD acknowledges a Senior Research Fellowship from the University Grants Commission, India.</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">REFERENCES</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Björk</span>, 
          <span class="cit-name-given-names">G. R.</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Hagervall</span>, 
          <span class="cit-name-given-names">T. G.</span>
         </span> (
         <span class="cit-pub-date">2014</span>). 
         <span class="cit-article-title">Transfer RNA Modification: Presence, Synthesis, and Function</span>. 
         <span class="cit-source">EcoSal Plus</span>, 
         <span class="cit-vol">6</span>(
         <span class="cit-issue">1</span>). 
         <a href="http://doi.org/10.1128/ecosalplus.ESP-0007%E2%80%932013">http://doi.org/10.1128/ecosalplus.ESP-0007–2013</a>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.2" data-doi="10.1093/nar/gkx1030">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Boccaletto</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Machnicka</span>, 
          <span class="cit-name-given-names">M. A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Purta</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Piątkowski</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bagiński</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wirecki</span>, 
          <span class="cit-name-given-names">T. K.</span>
         </span>, … 
         <span class="cit-auth">
          <span class="cit-name-surname">Bujnicki</span>, 
          <span class="cit-name-given-names">J. M.</span>
         </span> (
         <span class="cit-pub-date">2018</span>). 
         <span class="cit-article-title">MODOMICS: a database of RNA modification pathways. 2017 update</span>. 
         <span class="cit-source">Nucleic Acids Research</span>, 
         <span class="cit-vol">46</span>(
         <span class="cit-issue">D1</span>), 
         <span class="cit-fpage">D303</span>–
         <span class="cit-lpage">D307</span>. 
         <a href="http://doi.org/10.1093/nar/gkx1030">http://doi.org/10.1093/nar/gkx1030</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.volume%253D46%26rft.spage%253DD303%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx1030%26rft_id%253Dinfo%253Apmid%252F29106616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkx1030&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=29106616&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F22%2F251322.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.3" data-doi="10.1186/1471-2105-7-88">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bollback</span>, 
          <span class="cit-name-given-names">J. P.</span>
         </span> (
         <span class="cit-pub-date">2006</span>). 
         <span class="cit-article-title">SIMMAP: stochastic character mapping of discrete traits on phylogenies</span>. 
         <span class="cit-source">BMC Bioinformatics</span>, 
         <span class="cit-vol">7</span>(
         <span class="cit-issue">1</span>), 
         <span class="cit-fpage">88</span>. 
         <a href="http://doi.org/10.1186/1471%E2%80%932105-7%E2%80%9388">http://doi.org/10.1186/1471–2105-7–88</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DBollback%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D7%26rft.spage%253D88%26rft.epage%253D88%26rft.atitle%253DSIMMAP%253A%2Bstochastic%2Bcharacter%2Bmapping%2Bof%2Bdiscrete%2Btraits%2Bon%2Bphylogenies.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-7-88%26rft_id%253Dinfo%253Apmid%252F16504105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/1471-2105-7-88&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16504105&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F22%2F251322.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.4" data-doi="10.1006/jmbi.1997.1134">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Brierley</span>, 
          <span class="cit-name-given-names">I.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Meredith</span>, 
          <span class="cit-name-given-names">M. R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bloys</span>, 
          <span class="cit-name-given-names">A. J.</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Hagervall</span>, 
          <span class="cit-name-given-names">T. G.</span>
         </span> (
         <span class="cit-pub-date">1997</span>). 
         <span class="cit-article-title">Expression of a coronavirus ribosomal frameshift signal in Escherichia coli: influence of tRNA anticodon modification on frameshifting</span>. 
         <span class="cit-source">Journal of Molecular Biology</span>, 
         <span class="cit-vol">270</span>(
         <span class="cit-issue">3</span>), 
         <span class="cit-fpage">360</span>–
         <span class="cit-lpage">373</span>. 
         <a href="http://doi.org/10.1006/jmbi.1997.1134">http://doi.org/10.1006/jmbi.1997.1134</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DBrierley%26rft.auinit1%253DI.%26rft.volume%253D270%26rft.issue%253D3%26rft.spage%253D360%26rft.epage%253D373%26rft.atitle%253DExpression%2Bof%2Ba%2Bcoronavirus%2Bribosomal%2Bframeshift%2Bsignal%2Bin%2BEscherichia%2Bcoli%253A%2Binfluence%2Bof%2BtRNA%2Banticodon%2Bmodification%2Bon%2Bframeshifting.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1997.1134%26rft_id%253Dinfo%253Apmid%252F9237903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1006/jmbi.1997.1134&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=9237903&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F22%2F251322.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1997XM19300005&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chan</span>, 
          <span class="cit-name-given-names">P. P.</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Lowe</span>, 
          <span class="cit-name-given-names">T. M.</span>
         </span> (
         <span class="cit-pub-date">2009</span>). 
         <span class="cit-article-title">GtRNAdb: A database of transfer RNA genes detected in genomic sequence</span>. 
         <span class="cit-source">Nucleic Acids Research</span>, 
         <span class="cit-vol">37</span>(
         <span class="cit-issue">SUPPL. 1</span>), 
         <span class="cit-fpage">93</span>–
         <span class="cit-lpage">97</span>. 
         <a href="http://doi.org/10.1093/nar/gkn787">http://doi.org/10.1093/nar/gkn787</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.volume%253D37%26rft.spage%253D93%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.6" data-doi="10.1093/bioinformatics/bti037">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Charif</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Thioulouse</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lobry</span>, 
          <span class="cit-name-given-names">J. R.</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Perrière</span>, 
          <span class="cit-name-given-names">G.</span>
         </span> (
         <span class="cit-pub-date">2005</span>). 
         <span class="cit-article-title">Online synonymous codon usage analyses with the ade4 and seqinR packages</span>. 
         <span class="cit-source">Bioinformatics</span>, 
         <span class="cit-vol">21</span>(
         <span class="cit-issue">4</span>), 
         <span class="cit-fpage">545</span>–
         <span class="cit-lpage">547</span>. 
         <a href="http://doi.org/10.1093/bioinformatics/bti037">http://doi.org/10.1093/bioinformatics/bti037</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti037%26rft_id%253Dinfo%253Apmid%252F15374859%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti037&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15374859&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F22%2F251322.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000227127000018&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.7" data-doi="10.1073/pnas.0307827100">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">S. L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lee</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hottes</span>, 
          <span class="cit-name-given-names">A. K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shapiro</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">McAdams</span>, 
          <span class="cit-name-given-names">H. H.</span>
         </span> (
         <span class="cit-pub-date">2004</span>). 
         <span class="cit-article-title">Codon usage between genomes is constrained by genome-wide mutational processes</span>. 
         <span class="cit-source">Proceedings of the National Academy of Sciences of the United States of America</span>, 
         <span class="cit-vol">101</span>(
         <span class="cit-issue">10</span>), 
         <span class="cit-fpage">3480</span>–
         <span class="cit-lpage">3485</span>. 
         <a href="http://doi.org/10.1073/pnas.0307827100">http://doi.org/10.1073/pnas.0307827100</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0307827100%26rft_id%253Dinfo%253Apmid%252F14990797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzEwLzM0ODAiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wMS8yMi8yNTEzMjIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.8">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chionh</span>, 
          <span class="cit-name-given-names">Y. H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">McBee</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Babu</span>, 
          <span class="cit-name-given-names">I. R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hia</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lin</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, … 
         <span class="cit-auth">
          <span class="cit-name-surname">Dedon</span>, 
          <span class="cit-name-given-names">P. C.</span>
         </span> (
         <span class="cit-pub-date">2016</span>). 
         <span class="cit-article-title">tRNA-mediated codon-biased translation in mycobacterial hypoxic persistence</span>. 
         <span class="cit-source">Nature Communications</span>, 
         <span class="cit-vol">7</span>, 
         <span class="cit-fpage">13302</span>. 
         <a href="http://doi.org/10.1038/ncomms13302">http://doi.org/10.1038/ncomms13302</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D7%26rft.spage%253D13302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.9" data-doi="10.1016/S0022-2836(66)80022-0">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Crick</span>, 
          <span class="cit-name-given-names">F. H.</span>
         </span> (
         <span class="cit-pub-date">1966</span>). 
         <span class="cit-article-title">Codon--anticodon pairing: the wobble hypothesis</span>. 
         <span class="cit-source">Journal of Molecular Biology</span>, 
         <span class="cit-vol">19</span>(
         <span class="cit-issue">2</span>), 
         <span class="cit-fpage">548</span>–
         <span class="cit-lpage">55</span>. 
         <a href="http://doi.org/10.1016/S0022%E2%80%932836">http://doi.org/10.1016/S0022-2836(66)80022-0</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DCrick%26rft.auinit1%253DF.%2BH.%26rft.volume%253D19%26rft.issue%253D2%26rft.spage%253D548%26rft.epage%253D555%26rft.atitle%253DCodon--anticodon%2Bpairing%253A%2Bthe%2Bwobble%2Bhypothesis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0022-2836%252866%252980022-0%26rft_id%253Dinfo%253Apmid%252F5969078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0022-2836(66)80022-0&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=5969078&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F22%2F251322.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A19668268700022&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.10" data-doi="10.1093/bioinformatics/14.9.755">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Eddy</span>, 
          <span class="cit-name-given-names">S.</span>
         </span> (
         <span class="cit-pub-date">1998</span>). 
         <span class="cit-article-title">Profile hidden Markov models</span>. 
         <span class="cit-source">Bioinformatics</span>, 
         <span class="cit-vol">14</span>(
         <span class="cit-issue">9</span>), 
         <span class="cit-fpage">755</span>–
         <span class="cit-lpage">763</span>. 
         <a href="http://doi.org/btb114">http://doi.org/btb114</a> [pii]
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DEddy%26rft.auinit1%253DS.%2BR.%26rft.volume%253D14%26rft.issue%253D9%26rft.spage%253D755%26rft.epage%253D763%26rft.atitle%253DProfile%2Bhidden%2BMarkov%2Bmodels%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F14.9.755%26rft_id%253Dinfo%253Apmid%252F9918945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/14.9.755&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=9918945&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F22%2F251322.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000077489900002&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.11">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Eddy</span>, 
          <span class="cit-name-given-names">S. R.</span>
         </span> (
         <span class="cit-pub-date">2009</span>). 
         <span class="cit-article-title">A New Generation of Homology Search Tools Based on Probabilistic Inference</span>. 
         <span class="cit-source">Genome Informatics</span>, 
         <span class="cit-vol">23</span>(
         <span class="cit-issue">1</span>), 
         <span class="cit-fpage">205</span>–
         <span class="cit-lpage">211</span>. 
         <a href="http://doi.org/10.1142/9781848165632_0019">http://doi.org/10.1142/9781848165632_0019</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2BInformatics%26rft.volume%253D23%26rft.spage%253D205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.12">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Elliot</span>, 
          <span class="cit-name-given-names">M. G.</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Mooers</span>, 
          <span class="cit-name-given-names">A. Ø.</span>
         </span> (
         <span class="cit-pub-date">2014</span>). 
         <span class="cit-article-title">Inferring ancestral states without assuming neutrality or gradualism using a stable model of continuous character evolution</span>. 
         <span class="cit-source">BMC Evolutionary Biology</span>, 
         <span class="cit-vol">14</span>(
         <span class="cit-issue">1</span>), 
         <span class="cit-fpage">226</span>. 
         <a href="http://doi.org/10.1186/s12862%E2%80%93014-0226%E2%80%938">http://doi.org/10.1186/s12862–014-0226–8</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2BEvolutionary%2BBiology%26rft.volume%253D14%26rft.spage%253D226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.13">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gao</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tan</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zheng</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, … 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhou</span>, 
          <span class="cit-name-given-names">R.</span>
         </span> (
         <span class="cit-pub-date">2016</span>). 
         <span class="cit-article-title">GidA, a tRNA Modification Enzyme, Contributes to the Growth, and Virulence of 
          <em>Streptococcus suis</em> Serotype 2
         </span>. 
         <span class="cit-source">Frontiers in Cellular and Infection Microbiology</span>, 
         <span class="cit-vol">6</span>(
         <span class="cit-month">April</span>), 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">11</span>. 
         <a href="http://doi.org/10.3389/fcimb.2016.00044">http://doi.org/10.3389/fcimb.2016.00044</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2BCellular%2Band%2BInfection%2BMicrobiology%26rft.volume%253D6%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Grosjean</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Breton</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sirand-Pugnet</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tardy</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Thiaucourt</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Citti</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, … 
         <span class="cit-auth">
          <span class="cit-name-surname">Blanchard</span>, 
          <span class="cit-name-given-names">A.</span>
         </span> (
         <span class="cit-pub-date">2014</span>). 
         <span class="cit-article-title">Predicting the minimal translation apparatus: lessons from the reductive evolution of Mollicutes</span>. 
         <span class="cit-source">PLoS Genetics</span>, 
         <span class="cit-vol">10</span>(
         <span class="cit-issue">5</span>), 
         <span class="cit-fpage">e1004363</span>. 
         <a href="http://doi.org/10.1371/journal.pgen.1004363">http://doi.org/10.1371/journal.pgen.1004363</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenetics%26rft.volume%253D10%26rft.spage%253D1004363e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.15" data-doi="10.1016/j.febslet.2009.11.052">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Grosjean</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">de Crécy-Lagard</span>, 
          <span class="cit-name-given-names">V.</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Marck</span>, 
          <span class="cit-name-given-names">C.</span>
         </span> (
         <span class="cit-pub-date">2010</span>). 
         <span class="cit-article-title">Deciphering synonymous codons in the three domains of life: Co-evolution with specific tRNA modification enzymes</span>. 
         <span class="cit-source">FEBS Letters</span>, 
         <span class="cit-vol">584</span>(
         <span class="cit-issue">2</span>), 
         <span class="cit-fpage">252</span>–
         <span class="cit-lpage">264</span>. 
         <a href="http://doi.org/10.1016/j.febslet.2009.11.052">http://doi.org/10.1016/j.febslet.2009.11.052</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DGrosjean%26rft.auinit1%253DH.%26rft.volume%253D584%26rft.issue%253D2%26rft.spage%253D252%26rft.epage%253D264%26rft.atitle%253DDeciphering%2Bsynonymous%2Bcodons%2Bin%2Bthe%2Bthree%2Bdomains%2Bof%2Blife%253A%2Bco-evolution%2Bwith%2Bspecific%2BtRNA%2Bmodification%2Benzymes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.febslet.2009.11.052%26rft_id%253Dinfo%253Apmid%252F19931533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.febslet.2009.11.052&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19931533&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F22%2F251322.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000273209600002&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.16">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hershberg</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Petrov</span>, 
          <span class="cit-name-given-names">D. A.</span>
         </span> (
         <span class="cit-pub-date">2009</span>). 
         <span class="cit-article-title">General Rules for Optimal Codon Choice</span>. 
         <span class="cit-source">PLoS Genetics</span>, 
         <span class="cit-vol">5</span>(
         <span class="cit-issue">7</span>), 
         <span class="cit-fpage">e1000556</span>. 
         <a href="http://doi.org/10.1371/journal.pgen.1000556">http://doi.org/10.1371/journal.pgen.1000556</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenetics%26rft.volume%253D5%26rft.spage%253De1000556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-251322v1.17">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ikemura</span>, 
          <span class="cit-name-given-names">T.</span>
         </span> (
         <span class="cit-pub-date">1985</span>). 
         <span class="cit-article-title">Codon usage and tRNA content in unicellular and multicellular organisms</span>. 
         <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr>, 
         <span class="cit-vol">2</span>(
         <span class="cit-issue">1</span>), 
         <span class="cit-fpage">13</span>–
         <span class="cit-lpage">34</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DIkemura%26rft.auinit1%253DT%26rft.volume%253D2%26rft.issue%253D1%26rft.spage%253D13%26rft.epage%253D34%26rft.atitle%253DCodon%2Busage%2Band%2BtRNA%2Bcontent%2Bin%2Bunicellular%2Band%2Bmulticellular%2Borganisms%26rft_id%253Dinfo%253Apmid%252F3916708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=3916708&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F22%2F251322.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1985AAA8800002&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.18" data-doi="10.1016/S0378-1119(99)00225-5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kanaya</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yamada</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kudo</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Ikemura</span>, 
          <span class="cit-name-given-names">T.</span>
         </span> (
         <span class="cit-pub-date">1999</span>). 
         <span class="cit-article-title">Studies of codon usage and tRNA genes of 18 unicellular organisms and quantification of Bacillus subtilis tRNAs: Gene expression level and species-specific diversity of codon usage based on multivariate analysis</span>. 
         <span class="cit-source">Gene</span>, 
         <span class="cit-vol">238</span>(
         <span class="cit-issue">1</span>), 
         <span class="cit-fpage">143</span>–
         <span class="cit-lpage">155</span>. 
         <a href="http://doi.org/10.1016/S0378%E2%80%931119(99)00225%E2%80%935">http://doi.org/10.1016/S0378–1119(99)00225–5</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DKanaya%26rft.auinit1%253DS.%26rft.volume%253D238%26rft.issue%253D1%26rft.spage%253D143%26rft.epage%253D155%26rft.atitle%253DStudies%2Bof%2Bcodon%2Busage%2Band%2BtRNA%2Bgenes%2Bof%2B18%2Bunicellular%2Borganisms%2Band%2Bquantification%2Bof%2BBacillus%2Bsubtilis%2BtRNAs%253A%2Bgene%2Bexpression%2Blevel%2Band%2Bspecies-specific%2Bdiversity%2Bof%2Bcodon%2Busage%2Bbased%2Bon%2Bmultivariate%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1119%252899%252900225-5%26rft_id%253Dinfo%253Apmid%252F10570992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0378-1119(99)00225-5&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=10570992&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F22%2F251322.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000083005400015&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.19">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lowe</span>, 
          <span class="cit-name-given-names">T. M.</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Eddy</span>, 
          <span class="cit-name-given-names">S. R.</span>
         </span> (
         <span class="cit-pub-date">1996</span>). 
         <span class="cit-article-title">TRNAscan-SE: A program for improved detection of transfer RNA genes in genomic sequence</span>. 
         <span class="cit-source">Nucleic Acids Research</span>, 
         <span class="cit-vol">25</span>(
         <span class="cit-issue">5</span>), 
         <span class="cit-fpage">955</span>–
         <span class="cit-lpage">964</span>. 
         <a href="http://doi.org/10.1093/nar/25.5.0955">http://doi.org/10.1093/nar/25.5.0955</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.volume%253D25%26rft.spage%253D955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.20">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Machnicka</span>, 
          <span class="cit-name-given-names">M. a</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Olchowik</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Grosjean</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Bujnicki</span>, 
          <span class="cit-name-given-names">J. M.</span>
         </span> (
         <span class="cit-pub-date">2015</span>). 
         <span class="cit-article-title">Distribution and frequencies of post-transcriptional modifications in transfer RNAs</span>. 
         <span class="cit-source">RNA Biology</span>, 
         <span class="cit-vol">6286</span>(
         <span class="cit-month">December</span>), 
         <span class="cit-fpage">0</span>. 
         <a href="http://doi.org/10.4161/15476286.2014.992273">http://doi.org/10.4161/15476286.2014.992273</a>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.21">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Manickam</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Joshi</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bhatt</span>, 
          <span class="cit-name-given-names">M. J.</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Farabaugh</span>, 
          <span class="cit-name-given-names">P. J.</span>
         </span> (
         <span class="cit-pub-date">2015</span>). 
         <span class="cit-article-title">Effects of tRNA modification on translational accuracy depend on intrinsic codon–anticodon strength</span>. 
         <span class="cit-source">Nucleic Acids Research, 2 gkv1506</span>. 
         <a href="http://doi.org/10.1093/nar/gkv1506">http://doi.org/10.1093/nar/gkv1506</a>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.22" data-doi="10.1017/S1355838202022021">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Marck</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Grosjean</span>, 
          <span class="cit-name-given-names">H.</span>
         </span> (
         <span class="cit-pub-date">2002</span>). 
         <span class="cit-article-title">tRNomics: analysis of tRNA genes from 50 genomes of Eukarya, Archaea, and Bacteria reveals anticodon-sparing strategies and domain-specific features</span>. 
         <span class="cit-source">RNA (New York, N.Y.)</span>, 
         <span class="cit-vol">8</span>(
         <span class="cit-issue">10</span>), 
         <span class="cit-fpage">1189</span>–
         <span class="cit-lpage">232</span>. 
         <a href="http://doi.org/10.1017/S1355838202022021">http://doi.org/10.1017/S1355838202022021</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DRNA%26rft.stitle%253DRNA%26rft.aulast%253DMarck%26rft.auinit1%253DC.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253D1189%26rft.epage%253D1232%26rft.atitle%253DtRNomics%253A%2Banalysis%2Bof%2BtRNA%2Bgenes%2Bfrom%2B50%2Bgenomes%2Bof%2BEukarya%252C%2BArchaea%252C%2Band%2BBacteria%2Breveals%2Banticodon-sparing%2Bstrategies%2Band%2Bdomain-specific%2Bfeatures%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS1355838202022021%26rft_id%253Dinfo%253Apmid%252F12403461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjk6IjgvMTAvMTE4OSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzAxLzIyLzI1MTMyMi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.23" data-doi="10.1093/nar/3.10.2851">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Murao</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hasegawa</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Ishikura</span>, 
          <span class="cit-name-given-names">H.</span>
         </span> (
         <span class="cit-pub-date">1976</span>). 
         <span class="cit-article-title">5-Methoxyuridine: A New Minor Constituent Located in the First Position of the Anticodon of tRNAAla, tRNAThr, and tRNAVal from Bacillus Subtilis</span>. 
         <span class="cit-source">Nucleic Acids Research</span>, 
         <span class="cit-vol">3</span>(
         <span class="cit-issue">10</span>), 
         <span class="cit-fpage">2851</span>–
         <span class="cit-lpage">2860</span>. 
         <a href="http://doi.org/10.1093/nar/3.10.2851">http://doi.org/10.1093/nar/3.10.2851</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F3.10.2851%26rft_id%253Dinfo%253Apmid%252F825836%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/3.10.2851&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=825836&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F22%2F251322.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.24" data-doi="10.1261/rna.7106404">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Nasvall</span>, 
          <span class="cit-name-given-names">S. J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Bjork</span>, 
          <span class="cit-name-given-names">G. R.</span>
         </span> (
         <span class="cit-pub-date">2004</span>). 
         <span class="cit-article-title">The modified wobble nucleoside uridine-5-oxyacetic acid in tRNAPro(cmo5UGG) promotes reading of all four proline codons in vivo</span>. 
         <span class="cit-source">RNA (New York, N.Y.)</span>, 
         <span class="cit-vol">10</span>(
         <span class="cit-issue">10</span>), 
         <span class="cit-fpage">1662</span>–
         <span class="cit-lpage">73</span>. 
         <a href="http://doi.org/10.1261/rna.7106404">http://doi.org/10.1261/rna.7106404</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DRNA%26rft.stitle%253DRNA%26rft.aulast%253DNASVALL%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D10%26rft.issue%253D10%26rft.spage%253D1662%26rft.epage%253D1673%26rft.atitle%253DThe%2Bmodified%2Bwobble%2Bnucleoside%2Buridine-5-oxyacetic%2Bacid%2Bin%2BtRNAProcmo5UGG%2Bpromotes%2Breading%2Bof%2Ball%2Bfour%2Bproline%2Bcodons%2Bin%2Bvivo%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.7106404%26rft_id%253Dinfo%253Apmid%252F15383682%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjEwOiIxMC8xMC8xNjYyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDEvMjIvMjUxMzIyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference in text" id="ref-25">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.25" data-doi="10.1261/rna.731007">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Näsvall</span>, 
          <span class="cit-name-given-names">S. J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Björk</span>, 
          <span class="cit-name-given-names">G. R.</span>
         </span> (
         <span class="cit-pub-date">2007</span>). 
         <span class="cit-article-title">The wobble hypothesis revisited: uridine-5-oxyacetic acid is critical for reading of G-ending codons</span>. 
         <span class="cit-source">RNA (New York, N.Y.)</span>, 
         <span class="cit-vol">13</span>(
         <span class="cit-issue">Iii</span>), 
         <span class="cit-fpage">2151</span>–
         <span class="cit-lpage">2164</span>. 
         <a href="http://doi.org/10.1261/rna.731007">http://doi.org/10.1261/rna.731007</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DRNA%2B%2528New%2BYork%252C%2BN.Y.%2529%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.731007%26rft_id%253Dinfo%253Apmid%252F17942742%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjEwOiIxMy8xMi8yMTUxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDEvMjIvMjUxMzIyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference in text" id="ref-26">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-251322v1.26">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Noguchi</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nishimura</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hirota</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Nishimura</span>, 
          <span class="cit-name-given-names">S.</span>
         </span> (
         <span class="cit-pub-date">1982</span>). 
         <span class="cit-article-title">Isolation and characterization of an Escherichia coli mutant lacking tRNA-guanine transglycosylase. Function and biosynthesis of queuosine in tRNA</span>. 
         <abbr class="cit-jnl-abbrev">The Journal of Biological Chemistry</abbr>, 
         <span class="cit-vol">257</span>(
         <span class="cit-issue">11</span>), 
         <span class="cit-fpage">6544</span>–
         <span class="cit-lpage">50</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DNoguchi%26rft.auinit1%253DS.%26rft.volume%253D257%26rft.issue%253D11%26rft.spage%253D6544%26rft.epage%253D6550%26rft.atitle%253DIsolation%2Band%2Bcharacterization%2Bof%2Ban%2BEscherichia%2Bcoli%2Bmutant%2Blacking%2BtRNA-guanine%2Btransglycosylase.%2BFunction%2Band%2Bbiosynthesis%2Bof%2Bqueuosine%2Bin%2BtRNA.%26rft_id%253Dinfo%253Apmid%252F6804468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjExOiIyNTcvMTEvNjU0NCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzAxLzIyLzI1MTMyMi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference in text" id="ref-27">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.27" data-doi="10.1006/jmbi.2000.4042">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Notredame</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Higgins</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Heringa</span>, 
          <span class="cit-name-given-names">J.</span>
         </span> (
         <span class="cit-pub-date">2000</span>). 
         <span class="cit-article-title">T-Coffee: A novel method for fast and accurate multiple sequence alignment</span>. 
         <span class="cit-source">Journal of Molecular Biology</span>, 
         <span class="cit-vol">302</span>(
         <span class="cit-issue">1</span>), 
         <span class="cit-fpage">205</span>–
         <span class="cit-lpage">217</span>. 
         <a href="http://doi.org/10.1006/jmbi.2000.4042">http://doi.org/10.1006/jmbi.2000.4042</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DNotredame%26rft.auinit1%253DC.%26rft.volume%253D302%26rft.issue%253D1%26rft.spage%253D205%26rft.epage%253D217%26rft.atitle%253DT-Coffee%253A%2BA%2Bnovel%2Bmethod%2Bfor%2Bfast%2Band%2Baccurate%2Bmultiple%2Bsequence%2Balignment.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.2000.4042%26rft_id%253Dinfo%253Apmid%252F10964570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1006/jmbi.2000.4042&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=10964570&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F22%2F251322.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000089352000015&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference in text" id="ref-28">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.28" data-doi="10.1016/j.cell.2012.01.050">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Novoa</span>, 
          <span class="cit-name-given-names">E. M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pavon-Eternod</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pan</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Ribas De Pouplana</span>, 
          <span class="cit-name-given-names">L.</span>
         </span> (
         <span class="cit-pub-date">2012</span>). 
         <span class="cit-article-title">A role for tRNA modifications in genome structure and codon usage</span>. 
         <span class="cit-source">Cell</span>, 
         <span class="cit-vol">149</span>(
         <span class="cit-issue">1</span>), 
         <span class="cit-fpage">202</span>–
         <span class="cit-lpage">213</span>. 
         <a href="http://doi.org/10.1016/j.cell.2012.01.050">http://doi.org/10.1016/j.cell.2012.01.050</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DNovoa%26rft.auinit1%253DE.%2BM.%26rft.volume%253D149%26rft.issue%253D1%26rft.spage%253D202%26rft.epage%253D213%26rft.atitle%253DA%2Brole%2Bfor%2BtRNA%2Bmodifications%2Bin%2Bgenome%2Bstructure%2Band%2Bcodon%2Busage.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2012.01.050%26rft_id%253Dinfo%253Apmid%252F22464330%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.cell.2012.01.050&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22464330&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F22%2F251322.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000302235400021&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference in text" id="ref-29">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.29" data-doi="10.1093/nar/gki866">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Overbeek</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Begley</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Butler</span>, 
          <span class="cit-name-given-names">R. M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Choudhuri</span>, 
          <span class="cit-name-given-names">J. V.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chuang</span>, 
          <span class="cit-name-given-names">H.-Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cohoon</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, … 
         <span class="cit-auth">
          <span class="cit-name-surname">Vonstein</span>, 
          <span class="cit-name-given-names">V.</span>
         </span> (
         <span class="cit-pub-date">2005</span>). 
         <span class="cit-article-title">The subsystems approach to genome annotation and its use in the project to annotate 1000 genomes</span>. 
         <span class="cit-source">Nucleic Acids Research</span>, 
         <span class="cit-vol">33</span>(
         <span class="cit-issue">17</span>), 
         <span class="cit-fpage">5691</span>–
         <span class="cit-lpage">702</span>. 
         <a href="http://doi.org/10.1093/nar/gki866">http://doi.org/10.1093/nar/gki866</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DOverbeek%26rft.auinit1%253DR.%26rft.volume%253D33%26rft.issue%253D17%26rft.spage%253D5691%26rft.epage%253D5702%26rft.atitle%253DThe%2BSubsystems%2BApproach%2Bto%2BGenome%2BAnnotation%2Band%2Bits%2BUse%2Bin%2Bthe%2BProject%2Bto%2BAnnotate%2B1000%2BGenomes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki866%26rft_id%253Dinfo%253Apmid%252F16214803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gki866&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16214803&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F22%2F251322.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000232593700041&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference in text" id="ref-30">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.30">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">R Development Core Team</span>. (
         <span class="cit-pub-date">2015</span>). 
         <span class="cit-source">R: A Language and Environment for Statistical Computing</span>. 
         <a href="http://doi.org/10.1038/sj.hdy.6800737">http://doi.org/10.1038/sj.hdy.6800737</a>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference in text" id="ref-31">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.31">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Revell</span>, 
          <span class="cit-name-given-names">L. J.</span>
         </span> (
         <span class="cit-pub-date">2012</span>). 
         <span class="cit-article-title">phytools: An R package for phylogenetic comparative biology (and other things)</span>. 
         <span class="cit-source">Methods in Ecology and Evolution</span>, 
         <span class="cit-vol">3</span>(
         <span class="cit-issue">2</span>), 
         <span class="cit-fpage">217</span>–
         <span class="cit-lpage">223</span>. 
         <a href="http://doi.org/10.1111/j.2041%E2%80%93210X.2011.00169.x">http://doi.org/10.1111/j.2041–210X.2011.00169.x</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D3%26rft.spage%253D217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference in text" id="ref-32">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.32">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Segata</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Börnigen</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Morgan</span>, 
          <span class="cit-name-given-names">X. C.</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Huttenhower</span>, 
          <span class="cit-name-given-names">C.</span>
         </span> (
         <span class="cit-pub-date">2013</span>). 
         <span class="cit-article-title">PhyloPhlAn is a new method for improved phylogenetic and taxonomic placement of microbes</span>. 
         <span class="cit-source">Nature Communications</span>, 
         <span class="cit-vol">4</span>. 
         <a href="http://doi.org/10.1038/ncomms3304">http://doi.org/10.1038/ncomms3304</a>
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference in text" id="ref-33">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.33" data-doi="10.1016/j.micpath.2013.01.004">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Shippy</span>, 
          <span class="cit-name-given-names">D. C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Eakley</span>, 
          <span class="cit-name-given-names">N. M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lauhon</span>, 
          <span class="cit-name-given-names">C. T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bochsler</span>, 
          <span class="cit-name-given-names">P. N.</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Fadl</span>, 
          <span class="cit-name-given-names">A. A.</span>
         </span> (
         <span class="cit-pub-date">2013</span>). 
         <span class="cit-article-title">Virulence characteristics of Salmonella following deletion of genes encoding the tRNA modification enzymes GidA and MnmE</span>. 
         <span class="cit-source">Microbial Pathogenesis</span>, 
         <span class="cit-vol">57</span>, 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">9</span>. 
         <a href="http://doi.org/10.1016/j.micpath.2013.01.004">http://doi.org/10.1016/j.micpath.2013.01.004</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMicrobial%2BPathogenesis%26rft.volume%253D57%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.micpath.2013.01.004%26rft_id%253Dinfo%253Apmid%252F23375888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.micpath.2013.01.004&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23375888&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F22%2F251322.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference in text" id="ref-34">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.34" data-doi="10.1093/nar/gku245">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wald</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Margalit</span>, 
          <span class="cit-name-given-names">H.</span>
         </span> (
         <span class="cit-pub-date">2014</span>). 
         <span class="cit-article-title">Auxiliary tRNAs: Large-scale analysis of tRNA genes reveals patterns of tRNA repertoire dynamics</span>. 
         <span class="cit-source">Nucleic Acids Research</span>, 
         <span class="cit-vol">42</span>(
         <span class="cit-issue">10</span>), 
         <span class="cit-fpage">6552</span>–
         <span class="cit-lpage">6566</span>. 
         <a href="http://doi.org/10.1093/nar/gku245">http://doi.org/10.1093/nar/gku245</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku245%26rft_id%253Dinfo%253Apmid%252F24782525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gku245&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24782525&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F22%2F251322.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference in text" id="ref-35">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.35" data-doi="10.1093/emboj/cdf362">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wolf</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gerber</span>, 
          <span class="cit-name-given-names">A. P.</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Keller</span>, 
          <span class="cit-name-given-names">W.</span>
         </span> (
         <span class="cit-pub-date">2002</span>). 
         <span class="cit-article-title">tad A, an essential tRNA-specific adenosine deaminase from Escherichia coli</span>. 
         <span class="cit-source">The EMBO Journal</span>, 
         <span class="cit-vol">21</span>(
         <span class="cit-issue">14</span>), 
         <span class="cit-fpage">3841</span>–
         <span class="cit-lpage">51</span>. 
         <a href="http://doi.org/10.1093/emboj/cdf362">http://doi.org/10.1093/emboj/cdf362</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DWolf%26rft.auinit1%253DJ.%26rft.volume%253D21%26rft.issue%253D14%26rft.spage%253D3841%26rft.epage%253D3851%26rft.atitle%253DtadA%252C%2Ban%2Bessential%2BtRNA-specific%2Badenosine%2Bdeaminase%2Bfrom%2BEscherichia%2Bcoli%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252Fcdf362%26rft_id%253Dinfo%253Apmid%252F12110595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjEvMTQvMzg0MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzAxLzIyLzI1MTMyMi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference in text" id="ref-36">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.36">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yamada</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Matsugi</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ishikura</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Murao</span>, 
          <span class="cit-name-given-names">K.</span>
         </span> (
         <span class="cit-pub-date">2005</span>). 
         <span class="cit-article-title">Bacillus subtilis tRNAPro with the anticodon mo5UGG can recognize the codon CCC</span>. 
         <span class="cit-source">Biochimica et Biophysica Acta - Gene Structure and Expression</span>, 
         <span class="cit-vol">1728</span>(
         <span class="cit-issue">3</span>), 
         <span class="cit-fpage">143</span>–
         <span class="cit-lpage">149</span>. 
         <a href="http://doi.org/10.1016/j.bbaexp.2005.02.011">http://doi.org/10.1016/j.bbaexp.2005.02.011</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa%2B-%2BGene%2BStructure%2Band%2BExpression%26rft.volume%253D1728%26rft.spage%253D143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference in text" id="ref-37">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.37" data-doi="10.1093/nar/gkt356">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yokobori</span>, 
          <span class="cit-name-given-names">S. I.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kitamura</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Grosjean</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Bessho</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span> (
         <span class="cit-pub-date">2013</span>). 
         <span class="cit-article-title">Life without tRNAArg-adenosine deaminase TadA: Evolutionary consequences of decoding the four CGN codons as arginine in Mycoplasmas and other Mollicutes</span>. 
         <span class="cit-source">Nucleic Acids Research</span>, 
         <span class="cit-vol">41</span>(
         <span class="cit-issue">13</span>), 
         <span class="cit-fpage">6531</span>–
         <span class="cit-lpage">6543</span>. 
         <a href="http://doi.org/10.1093/nar/gkt356">http://doi.org/10.1093/nar/gkt356</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt356%26rft_id%253Dinfo%253Apmid%252F23658230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkt356&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23658230&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F22%2F251322.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000322458800022&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference in text" id="ref-38">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.38" data-doi="10.1073/pnas.82.15.4905">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yokoyama</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Watanabe</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Murao</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ishikura</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yamaizumi</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nishimura</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Miyazawa</span>, 
          <span class="cit-name-given-names">T.</span>
         </span> (
         <span class="cit-pub-date">1985</span>). 
         <span class="cit-article-title">Molecular mechanism of codon recognition by tRNA species with modified uridine in the first position of the anticodon</span>. 
         <span class="cit-source">Proceedings of the National Academy of Sciences</span>, 
         <span class="cit-vol">82</span>(
         <span class="cit-issue">15</span>), 
         <span class="cit-fpage">4905</span>–
         <span class="cit-lpage">4909</span>. 
         <a href="http://doi.org/10.1073/pnas.82.15.4905">http://doi.org/10.1073/pnas.82.15.4905</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DYokoyama%26rft.auinit1%253DS.%26rft.volume%253D82%26rft.issue%253D15%26rft.spage%253D4905%26rft.epage%253D4909%26rft.atitle%253DMolecular%2Bmechanism%2Bof%2Bcodon%2Brecognition%2Bby%2BtRNA%2Bspecies%2Bwith%2Bmodified%2Buridine%2Bin%2Bthe%2Bfirst%2Bposition%2Bof%2Bthe%2Banticodon.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.82.15.4905%26rft_id%253Dinfo%253Apmid%252F3860833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODIvMTUvNDkwNSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzAxLzIyLzI1MTMyMi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference in text" id="ref-39">↵</a>
      <div class="cit ref-cit ref-web" id="cit-251322v1.39" data-doi="10.3390/biom7030053">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zallot</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yuan</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">de Crécy-Lagard</span>, 
          <span class="cit-name-given-names">V.</span>
         </span> (
         <span class="cit-pub-date">2017</span>). 
         <span class="cit-article-title">The 
          <em>Escherichia coli</em> COG1738 Member YhhQ Is Involved in 7-Cyanodeazaguanine (preQ₀) Transport
         </span>. 
         <span class="cit-source">Biomolecules</span>, 
         <span class="cit-vol">7</span>(
         <span class="cit-issue">1</span>), 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">13</span>. 
         <a href="http://doi.org/10.3390/biom7010012">http://doi.org/10.3390/biom7010012</a>
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBiomolecules%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fbiom7030053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.3390/biom7030053&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
