<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="publisher-id" title="publisher-id">978-1-4614-5474-8</span><span class="doi" title="doi">10.1007/978-1-4614-5474-8</span><span class="nlm-ta" title="nlm-ta">Modeling the Interplay Between Human Behavior and the Spread of Infectious Diseases</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Modeling the Interplay Between Human Behavior and the Spread of Infectious Diseases</span></div><span class="isbn" tagx="isbn" title="isbn">978-1-4614-5473-1</span><span class="isbn" tagx="isbn" title="isbn">978-1-4614-5474-8</span></div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7121241">7121241</a></span><span class="publisher-id" title="publisher-id">9</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1007/978-1-4614-5474-8_9">10.1007/978-1-4614-5474-8_9</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Risk Perception, Heuristics and Epidemic Spread</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Manfredi</span><span class="given-names" tagx="given-names" title="given-names">Piero</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">manfredi@ec.unipi.it</span></span><a href="#AffID1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">D'Onofrio</span><span class="given-names" tagx="given-names" title="given-names">Alberto</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">alberto.donofrio@ifom-ieo-campus.it</span></span><a href="#AffID2">2</a></span><span class="citation_author_institution" id="AffID1">[ID1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.5395.a</span><span class="institution-id" title="institution-id">0000000417573729</span><span class="institution" title="institution">, Department of Economics and Management, </span><span class="institution" title="institution">University of Pisa, </span></div></span><span class="citation_author_institution" id="AffID2">[ID2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.15667.33</span><span class="institution-id" title="institution-id">0000000417570843</span><span class="institution" title="institution">, Department of Experimental Oncology, </span><span class="institution" title="institution">European Institute of Oncology, </span></div></span></div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">LiÃ²</span><span class="given-names" tagx="given-names" title="given-names">Pietro</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">pl219@cam.ac.uk</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lucia</span><span class="given-names" tagx="given-names" title="given-names">Bianchi</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">luciagbianchi@gmail.com</span></span><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguyen</span><span class="given-names" tagx="given-names" title="given-names">Viet-Anh</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">van25@cam.ac.uk</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kitchovitch</span><span class="given-names" tagx="given-names" title="given-names">Stephan</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">Stephan.Dinkov@gmail.com</span></span><a href="#Aff1">1</a></span><span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.5335.0</span><span class="institution-id" title="institution-id">0000000121885934</span><span class="institution" title="institution">Computer Laboratory, </span><span class="institution" title="institution">University of Cambridge, </span></div></span><span class="citation_author_institution" id="Aff2">[2], </span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2012-10-10</span></span><span class="fpage" tagx="fpage" title="fpage">139</span><span class="lpage" tagx="lpage" title="lpage">152</span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p">This article is made available via the PMC Open Access Subset for unrestricted research re-use and secondary analysis in any form or by any means with acknowledgement of the original source. These permissions are granted for the duration of the World Health Organization (WHO) declaration of COVID-19 as a global pandemic.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>During an outbreak of an infectious disease, people often change their behaviour to reduce their risk of infection. In a given population, the levels of perceived risk of infection vary greatly among individuals. The difference in perception could be due to a number of different factors including varying levels of information regarding the pathogen, quality of local healthcare, availability of preventive measures, individual and group usage of heuristics in the decision-making process. First we discuss the rigorous assessment of the risk, then we describe how our brain assesses the risk through the use of heuristics that are still rooted in animal evolution. Then we discuss the impact and the role of mass media and social networks in modulating risk perception. Next, we show how mathematical modelling is challenged by multi-scale epidemiological problems where the risk perception level is coupled with all the other microscopic and macroscopic levels. Finally, we draw future scenarios of personal risk evaluation through self-monitoring devices and personal genomics. The aim of this chapter is to discuss the importance of risk perception related to the spreading of a disease and to present a variety of ideas that could be fruitfully explored through modelling.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
Risk Perception</div> <div class="kwd" title="kwd">
Mirror Neuron</div> <div class="kwd" title="kwd">
Boundary Node</div> <div class="kwd" title="kwd">
Personal Genomic</div> <div class="kwd" title="kwd">
Scarlet Fever</div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="Â© Springer Science+Business Media New York 2013" /> </div> </div> </div> <div class="body" title="body"> <div class="therelationshipbetweenepidemicriskandawareness" title="sec"> <div class="title" tagx="title" title="title">
The Relationship Between Epidemic Risk and Awareness</div> <p>Current and future public health hazards do not depend only on the biological characteristics of the infectious organism and on our immune system capability of response but also on the responsible behaviour of individuals, particularly of those who are more likely to be infected. We find difficult to accurately evaluate infection risk because of the lack of data and multi-scale and multi-factorial conditions of infection events. We are able to estimate different health threats of different viral strains by comparing the binding energies of viral proteins with host membrane receptors, but it is still difficult to estimate all interaction combinatorics at the whole cell proteomics level. Recently animal models have provided important experimental data on influenza spreading. Lowen and co-authors [<a href="#CR22">22</a>] have found that the influenza virus can pass between guinea pigs by means of droplet spread and was transmitted from infected guinea pigs to healthy guinea pigs housed in the same cage, an adjacent cage, or a cage placed 91â€‰cm away. Coughing, sneezing, talking, and breathing generate a cloud of airborne particles with diameters ranging from a few mm to less than 1â€‰Î¼m. Alford et al. [<a href="#CR2">2</a>] revealed that humans could contract influenza by inhaling an experimental small-particle aerosol containing low levels of influenza virus. We trust hospitals as safe places but they are known to originate the spread of infectious diseases in patients. Even isolation rooms equipped with advanced heating ventilation air conditioning (HVAC) systems may not be completely safe as shown in [<a href="#CR4">4</a>]. Here, 3D models of the room consider different, most typical, positions of the patients. Results indicate the best conditions for high induction air inlet diffuser and the scheme of pressures imposed in the room to provide the effective means of controlling flows containing virus droplets. The authors used finite element modelling software and CAD techniques to model the isolation room under different operating conditions (negative and positive pressure of the isolation room, airflow patterns associated with different cough conditions, switch on off of the HVAC). The authors found difficult to obtain technical details of surgery theatres, probably because results could change patients awareness and highlight structural deficiencies during economic crisis. The seasonal flu infection follows an age pattern: Brownstein [<a href="#CR5">5</a>] showed that influenza spread firstly in children aged 3â€"4 years old, then the infection involves their brothers and sisters who may have more hygienic behaviour, then their parents and with the start of the winter it spreads among the elderly who may be at greater risk. People affected by severe immune pathologies could have different infectious risks and different awareness. With age, both frailty and risk perception usually increase. A challenge task is to evaluate the probability of the emergence of new variety of pathogen strains. Although about 3,000 different viruses are nowadays known, next generation sequencing and metagenomic approaches have revealed the vast majority of the viruses on Earth is not yet characterised. Statistical bioinformatics models will be increasingly used to calculate the number of viral mutations between safety and pandemics with little population awareness due to scarce education about DNAÂ sequences.</p> </div> <div class="puttingriskevidencestogether" title="sec"> <div class="title" tagx="title" title="title">
Putting Risk Evidences Together</div> <p>Expensive and complex epidemiological data are gathered and being analysed in a rather simple way that runs the risk of missing the opportunity to uncover combinations of predictive and meaningful profiles among the data. The challenge is to establish whether any causal conclusion suggested by the best fitting model is warranted by the data. Formal approaches to causal inference are needed in order to guide the researcher specifying the underlying causal assumptions, formulating the causal hypothesis in a non-ambiguous way, determining whether the data provide information to address that hypothesis. The complexly structured data, embracing molecular, subclinical and clinical, epidemiological information, social network require novel methodology to elucidate epidemics and health-related mechanisms. The multilevel data point at a huge space of causal models. The problem is not simply one of defining a suitable class of models and, within this class, to search for those models which provide the best fit but to combine the multilevel data within a study, with one type of data across studies, into multilevel data across studies. There is an increasing need to use all the available information, even with missing values, from different datasets (meta-analysis) and different dimensionality, which may be called super meta methodology.</p> <p>An important approach in evidence synthesis is that of back-calculation, which is based on convolution equations; it uses reported numbers of diagnoses of a disease (say AIDS) in each calendar period, together with estimates of the distribution of time from infection to diagnosis of AIDS (the incubation period) to generate estimates of the underlying incidence of infections. Combined with information on survival, this approach allows to estimate current HIV prevalence. However, cheap diagnostic tools and medical treatments would change the actual incubation period (information ignorance). Another approach is the direct method where the general population is subdivided into mutually exclusive groups, indexed by <i>g</i>, at risk for HIV and information on the proportion of the population in each risk group, Î³<sub><i>g</i></sub>, is derived from routinely collected statistics or the census [<a href="#CR12">12</a>]. The task is then to estimate the number of prevalent diagnosed and undiagnosed infections in each risk group, as well as the risk group totals <i>RG</i>, which can be expressed as </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">1</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M1">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document} $$RG = \sum\limits_{g}N(\pi _{g}\gamma _{g}\delta _{g} + {\pi _{g}}_{g}(1 - \delta _{g})).$$ \end{document}</div> </div> </div> The direct method is not easy to implement in practice because it requires complete data from all the risk groups to compute confidence intervals for each group and the relationships between the various items of evidence. Interesting example of software is the Estimation and Projection Package [<a href="#CR16">16</a>] that fits a simple infectious transmission model to the prevalence data via sampling importance resampling and generates a cluster of epidemic curves for each urban/rural and subgroup-specific sub-epidemic. A novel promising approach is that of multi-parameter evidence synthesis which offers a coherent analytical framework designed to make rational and exhaustive use of the whole body of information available [<a href="#CR1">1</a>]. It constructs a formal specification of the relationships between data and parameters, which dictates how direct and indirect evidences on the parameters of interest can be integrated [<a href="#CR15">15</a>].<p /> <p>Recent advances in cognitive science have given insights into the human brain as a rational Bayesian machine. Risk perception of a particular disease is shaped by both past events experienced by the individual and available hints from living environment at global and local scales. Given an incomplete picture <i>Y</i><sub><i>obs</i></sub> of the situation, the brain performs a filling-in process of the missing details <i>Y</i><sub><i>miss</i></sub> so it could come up with decisions of rational action. The inference is facilitated by assuming a single model with the relevant variables, specified by the set of parameters <i>Î˜</i> with prior belief <i>p</i>(<b>Î¸</b>). The predictive distribution of missing details can be obtained by integrating out the parameter posterior space </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">2</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M2">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document} $$p(Y _{miss}\vert Y _{obs}) = \int p(Y _{miss}\vert \Theta )p(\Theta \vert Y _{obs})d\Theta $$ \end{document}</div> </div> </div> where parameter posterior is in turn estimated from probable values of missing portion <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">3</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M3">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document} $$p(\Theta \vert Y _{obs}) \propto p(\Theta )\int p(Y _{obs},Y _{miss}\vert \Theta )dY _{miss}$$ \end{document}</div> </div> </div> An approximation of this iterated scheme [<a href="#CR23">23</a>] has been demonstrated to successfully uncover missing feature and relational data in the biological domain.<p /> </div> <div class="perceptionashumanquicksynthesisofevidences" title="sec"> <div class="title" tagx="title" title="title">
Perception as Human Quick Synthesis of Evidences</div> <p>Since the individual decision-making process is affected by the risk perception, during evolution we have developed fast decision shortcuts, called heuristics, in order to increase our survival. Heuristics are efficient cognitive processes that ignore information and enable fast decisions. Because of their cognitive limitations, humans are often unable to perform rational calculations and instead rely on error-prone heuristics; moreover, even when people could optimise, that is, to compute the best decision, they often rely on heuristics to save efforts at the price of sacrificing some accuracy. These concepts are based on the principle of an accuracy-effort trade-off: the less information, computation or time one uses, the less accurate oneâ€™s judgments will be. This trade-off is believed to be one of the few general laws of the mind. It is important to evaluate the heuristics in terms of its ecological rationality environment. How do people make decisions when optimisation is out of reach? Examples of the embedded heuristics toolbox we use at individual or social levels are the recognition heuristic, which states that if one of the two alternatives is recognised, one will infer that it has the higher value on the criterion (less-is-more effect is often detected); the 1â€‰âˆ•â€‰<i>N</i> equality heuristic, which allocates resources equally to each of N alternatives; and <i>titfortat</i>, in which one cooperates first and then imitates her/his partnerâ€™s last behaviour; other widely used heuristics are the imitation of the behaviour of majority and the imitation of the successful person. The last two heuristics are recognised as a driving force in bonding and group identification and therefore play an important role in our choices [<a href="#CR11">11</a>]. In 1950 Herbert Simon [<a href="#CR27">27</a>] first proposed that the people satisfice rather than maximise. Maximisation means optimisation, the process of finding the best solution for a problem, whereas satisficing (satisfying and sacrificing at the same time) means finding a good enough solution. This corresponds to a well-known heuristics: in order to select a good alternative (e.g. a house or a spouse) from a series of options encountered sequentially, a person sets an aspiration level, chooses the first one that meets the aspiration and then terminates the search. The aspiration level can be fixed or adjusted following experience. For Simon, humans rely on heuristics not simply because of their cognitive limitation but also because of the task environment. We often trust a person we know well as non-infected while we may judge as potentially infected a person we do not know; sometimes these decision-making processes take less than one minute. Gigerenzer has shown that simple rules, that is, decision-making tree based on fast and frugal heuristics, behave nearly the same as more complex diagnostic procedures [<a href="#CR32">32</a>]. The self-diagnosis, which may correlate with the time you decide a medical visit is opportune, the choice of a doctor or of a therapy could be biased by the underlying presence of the recognition heuristics. There is a growing activity of mathematical modelling of the impact of heuristics in the theory of mind and in the ICT. Recent approaches are based on so-called ACTR (adaptive control of thought-rational) and establish analogies with an ecological model for strategy selection. In Parma during 1980s and 1990s Rizzolatti and colleagues demonstrated the existence of a mirror-matching system in the human brain [<a href="#CR25">25</a>]. It is nowadays accepted that the mirror neurons system mediates the automatic imitation and empathy; in some sense they provide basis for connecting individualsâ€™ emotions. The neurophysiology of mirror neurons could possibly suggest the origin of heuristics in the imitation and processing capabilities of mirror neurons and in the spread of risk perception levels by imitation. The ancient structure of the mirror neurons could provide some explanation for some ancestral fears and overestimated risk perception such as that for snakes. It is noteworthy that the link between heuristics and neurophysiology and the link between epidemics spreading and heuristics generate a connection between epidemics and the functioning of brain regions affected by fear and conscience. This connection is also affected by reading newspapers, twitter and blogs, watching television and Youtube.</p> </div> <div class="riskperceptionmodulationbymassmediaandsocialmedia" title="sec"> <div class="title" tagx="title" title="title">
Risk Perception Modulation by Mass Media and Social Media</div> <p>Our risk perception is shaped by heuristics and more influenced by friendsâ€™ recommendations than mass media channels such as newspapers and television. Nevertheless, mass media have a tremendous importance in influencing individual behaviour probably because of the concept of the wisdom of the crowds, that is, the many are smarter than the few. Mass media is influenced by commercial trends, politics and the journalistic practice, for example, an article could appear in the first page in large characters or towards the end of the newspaper in a minor section; moreover, there is a journalistic tendency to draw attention to certain features of an issue while minimising attention to others. Social media is strongly influenced by our social constrains. Robin Dunbar and collaborators have found that social networks have a cognitive limit for emotional closeness of about 150 members (albeit with significant variance around this value); within a 150-people layer, there are those you know as persons. Beyond 150, we know people only as categories; interactions are defined by rules, not personal knowledge. Kin are given preference in the network where individuals from large families have fewer friends, and there are strong same-sex preferences. Kin networks are usually more dense than friend networks, suggesting that family links are less fragile than friends (reflecting the say that blood is not water) [<a href="#CR8">8</a>]. There is a widespread belief that social media have increased the amount and quality of information but also background rumours. Facebook, Twitter and other social websites have boosted the public awareness of disease outbreaks but also make it more difficult to separate facts from fiction. Many WHO experts have pointed out that social media are mixed blessing in epidemics. WHO officials reported that during the H1N1 swine flu that swept the world in 2009/2010, one of the Internet rumours was increasing your salt intake can help, consequently boosting a counteraction by WHO about the dangers of taking too much salt. Two important differences with respect to mass media are represented by specific information social networks for patients such as the website <i>PatientsLikeMe</i> and flu tracker phone or Internet-based networks which act as a quick alarm network and continuous coverage statistics on the evolution of an epidemic front. The website <i>PatientsLikeMe</i> acts as a quick self-diagnose based on symptomatic similarity and drug performance evaluation social network. The drawbacks could be mis-diagnoses and less personalised treatment (an individual takes the same drugs that have well performed for other patients disregarding his/her special conditions and past history of treatments). Although observational studies cannot meet the rigid standard of randomised clinical trials, they provide an opportunity to collect possibly useful data by capturing patientsâ€™ conditions. There is more trust in people with similar conditions and the heuristics of imitating likewise people. A recent work reported that a personâ€™s overall web literacy predicts their behaviour to a significant extent. Especially for people with lower web literacy, the extent to which a web pageâ€™s declared ideological perspective was consistent with their own was very important for whether a person decided to believe the information posted on that web page. In [<a href="#CR31">31</a>] the authors have proposed a susceptible (S)-infected (I)-hospitalised (H)-recovered (R) model where the media function is incorporated into the model using an exponentially decreasing function:</p> <p> </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">4</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M4">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document} $$\frac{dS} {dt} = \Lambda - {e}^{-M\left (t\right )}\lambda S - \mu S + \delta R$$ \end{document}</div> </div> </div> where the rate of susceptibles depends on the recruited into the population <i>Î›</i> and on the media coverage over time <i>M</i>(<i>t</i>) times the constant recruitment rate, Î»; Î¼ is the natural death rate and Î´ is the recovery rate. This model is able to highlight different levels of beneficial effects of media coverage. M(t) depends on both the number of cases and their rate. Their model is successful in fitting some relevant data but also opens the possibility for parameter estimation. Recent work by Durham and Casman [<a href="#CR9">9</a>] has taken a step towards constructing a model for SARS that is based on time series of media coverage regarding the disease and on the proportion of the population wearing face masks during the outbreak. The approach of [<a href="#CR9">9</a>] demonstrates the possibility of identifying parameters of human behaviour using various data sources such as news, surveys or even search engine query data [<a href="#CR13">13</a>]. Models are at the level of the intensity of single source of data acquisition; we still need to approach the variety of data acquisition.<p /> </div> <div class="heuristicsandundervaccination" title="sec"> <div class="title" tagx="title" title="title">
Heuristics and Undervaccination</div> <p>Undervaccination is observed in both poor and wealthy communities [<a href="#CR30">30</a>]. In poor countries, intensive international programmes aiming at familiarising adult primary care providers (APCPs) with vaccine-preventable diseases and the importance of using vaccines are needed in order to improve vaccination rates. Recent work shows poor familiarisation of APCPs to inform hospitalised individuals about vaccinations [<a href="#CR30">30</a>]. Noteworthy, on Facebook (2011), there are more than 40,000 pages on vaccinations and more than 1200 groups; 95â€‰% of them are negative towards vaccination; on Youtube 90â€‰% of the 10,000 movies about vaccination are against. So the background noises for rumours have become much louder, making it so much harder to detect the really important segments. Typical reasons found in blogs are the following: <i>mom knows best;if vaccines caused autism, I would probably opt out of most vaccines, because most kids donâ€™t die of whooping cough or scarlet fever, but autism is forever; donâ€™t want my kids to get autism. So I will risk a deadly disease instead</i>. Clearly the decision is based on a heuristic approach and not on a careful reasoning which should have considered that for highly contagious viruses, like measles, about 95â€‰% of the population need to be immunised to effectively prevent spread. In some sense the mother had not made a choice for her son only, but for everyone with whom he had come in contact. Is it reasonable that this motherâ€™s rights should include the right to have her child catch and transmit a potentially fatal infection? Peadiatricians must also take proactive precautions with the growing number of unvaccinated, potentially biohazardous children and immediately place them in an examination room away from office traffic flow. It is totally unfair to expose the rest of your patients who are trying their best to protect their children with recommended vaccines. Mathematical models have highlighted the different values of vaccination strategies and information (see for instance [<a href="#CR19">19</a>, <a href="#CR21">21</a>, <a href="#CR24">24</a>]). A model with pulse vaccination has shown that the media can trigger a vaccinating panic if the vaccine is imperfect and simplified messages may result in the vaccinated mixing with the infectives without regard to disease risk [<a href="#CR31">31</a>]. The perception of the individual responsibility in different context is an important factor that has not been incorporated in models.</p> </div> <div class="buildingmodelsofriskperception" title="sec"> <div class="title" tagx="title" title="title">
Building Models of Risk Perception</div> <p>A study on the psychological responses to the 2009 H1N1 virus, [<a href="#CR14">14</a>], reported significant reduction in the use of public transport, a high number of flight cancellations and a considerable amount of investment in preventative goods. Individuals undertaking such precautionary measures may succeed in reducing their susceptibility to the disease and thus potentially reduce the size of an epidemic outbreak. Despite the potentially considerable impact of behavioural changes in the population, the majority of mathematical models have only considered changes in human behaviour resulting from various public health interventions. The aim of such models is to study the effectiveness of interventions such as social distancing and the provision of treatment and prophylaxis in containing an emerging pandemic. [<a href="#CR33">33</a>] have also considered the compliance with suggested interventions, mentioning that the compliance of individuals may be closely related to various demographics and that levels of compliance may vary over the course of the disease outbreak. Even in the absence of institutionally enforced interventions, individuals may occasionally take the initiative and change their personal behaviour in response to their perceived risk of becoming infected. The risk perception framework was applied by [<a href="#CR3">3</a>] to study the effect of human behaviour on both homogeneous and directed scale-free networks [<a href="#CR3">3</a>]. In this framework, as a result of alertness to the disease, the probability of transmission of the disease due to contact between an infectious and a susceptible individual is multiplied by a factor of </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">5</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M5">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document} $$A(s,k) = exp\left [-\left (H + J \frac{s} {k}\right )\right ]$$ \end{document}</div> </div> </div> where <i>s</i> is the number of the individualâ€™s infected connections and <i>k</i> is the degree of the connectivity of the network. The parameter <i>J</i> represents the â€˜localâ€™ or personal perception and determines how strongly the individual reacts to observing disease symptoms in his close contacts. The global awareness parameter <i>H</i> determines the awareness that an individual has gained from publicly available information or due to access to treatment and preventative measures. The probability that an individual becomes infected following contact with at least one of its infectious neighbours is modelled as <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">6</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M6">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document} $$\lambda \left (s,k\right ) = 1 -{\left [1 - A\left (s,k\right )\tau \right ]}^{s}.$$ \end{document}</div> </div> </div> [<a href="#CR3">3</a>] presented an additional approach to examining the effects of risk perception on a population. They showed that, for random networks and infectivity, there exists a critical value of the individual awareness <i>J</i>, above which it can prevent an epidemic outbreak.<p /> </div> <div class="combiningcommunitystructureandriskperception" title="sec"> <div class="title" tagx="title" title="title">
Combining Community Structure and Risk Perception</div> <p>Mean-field approach shows that, in the presence of several communities, a value of <i>J</i> higher than the critical value for the homogeneous case, is necessary to prevent a community from experiencing an epidemic. Since in a realistic situation the amount of precautions an individual can take are limited, a consequence of this result is that, in the presence of community interactions, even high awareness of the disease may be insufficient to prevent an epidemic. Different ethnic communities or communities in geographical regions (e.g., countryside versus city) may respond differently to an epidemic, so community models will provide a better fit to real situations [<a href="#CR18">18</a>, <a href="#CR19">19</a>]. In [<a href="#CR19">19</a>] the authors developed an epidemic prediction tool that uses a combination of geographical, demographic and housing information. We foresee that similar tools will have automatic update of the relevant information. Following [<a href="#CR18">18</a>] we consider a network of N individuals described by five parameters: C is the number of communities; <i>n</i>(<i>X</i>) is the size of community X; <i>p</i><sub><i>i</i></sub>(<i>X</i>) is the probability of connection between nodes in community X; <i>p</i><sub><i>e</i></sub>(<i>X</i>) is the probability of connection between nodes in community X and the nodes of any other community; and <i>H</i>(<i>X</i>) the awareness of the disease in community X. We use Ï‚ to denote the set of all communities in the network. The first four parameters allow for variation in size and connectivity per community and are necessary for constructing a network of heterogeneous communities. The parameters <i>p</i><sub><i>i</i></sub>(<i>X</i>) and <i>p</i><sub><i>e</i></sub>(<i>X</i>) are chosen based on the required average internal and external connectivity per individual in each community. The <i>H</i>(<i>X</i>) parameter is only necessary when the communities modeled also have varying levels of risk perception. A common approach for generating networks to test community detection algorithms is the use of the planted l-partition model [<a href="#CR7">7</a>, <a href="#CR10">10</a>]. The algorithm divides a set of N nodes into l equally sized groups according to two probabilities: <i>p</i><sub><i>in</i></sub> is the probability of connection between nodes in the same group; <i>p</i><sub><i>out</i></sub> is the probability of connection between nodes belonging to separate groups. Links are generated between all pairs of nodes according to these probabilities and the result is an Erdos-Renyi-like (ER) random network of <i>l</i> communities, provided that <i>p</i><sub><i>in</i></sub>â€‰&amp;gt;â€‰<i>p</i><sub><i>out</i></sub>. This approach generates a network of communities of varying sizes. The average connectivity in the resulting network is homogeneous for individuals within the same community but varies across communities. The communities exhibit the small-world property. Across communities, the average shortest path L is likely to be larger than within communities, due to the lower density of edges between communities.</p> <p>The investigation of the effects of boundary nodes (i.e. nodes within a community with at least one external connection) is a common procedure when examining community structure in networks [<a href="#CR10">10</a>]. In the case of non-overlapping communities, these nodes have a high betweenness centrality and represent the only means by which infection may travel between communities. In this model the probability of connection between two nodes, members of the communities X and V, respectively, is <i>p</i><sub><i>e</i></sub>(<i>X</i>)<i>p</i><sub><i>e</i></sub>(<i>V</i> ). The probability of a node in V not having a connection to any node in X is </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">7</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M7">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document} $$[1 - p_{e}(V )p_{e}(X)]n(X).$$ \end{document}</div> </div> </div> <p /> <p>Thus, the expected number of boundary nodes in V, considering all communities in Ï‚, is given by </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">8</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M8">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document} $$B_{V } = n(V )[1 -\prod\limits_{X\in \varsigma,X\neq V }{[1 - p_{e}(V )p_{e}(X)]}^{n(X)}]$$ \end{document}</div> </div> </div> <p /> <p>An isolated community is simply an ER random graph. Let us consider for simplicity the case of the arrival of the infection from outside the community and not its subsequent spread. The set of parameters we need to consider are thus the community size n (the size of the infected outside world is <i>N</i>â€‰âˆ’â€‰<i>n</i>), the probability of external connectivity <i>p</i><sub><i>e</i></sub> (<i>p</i><sub><i>e</i></sub>â€‰=â€‰1 for the outside world) and the awareness of the community H. The expected number of infections entering a susceptible community is simply the number of boundary nodes which would become infected. The probability of an individual becomes infected following contact with at least one of its infectious neighbours is given by equation above.</p> <p>If B is the number of boundary nodes of the community, then the expected number of infections, for an individual with s infectious out of k total contacts, is simply Î»(<i>s</i>,â€‰<i>k</i>)<i>B</i>. If we do not consider the local awareness and the outside world is completely infected we can simplify Î»(<i>s</i>,â€‰<i>k</i>) to <span class="inline-formula" title="inline-formula"> </span></p><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M9">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document} $$1 - {[1 - {e}^{-H}\tau ]}^{{k}^{out} }$$ \end{document}</div> </div> and we obtain<p /> <p> </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">9</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M10">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document} $$y_{X}(k,i) = \sum\limits_{s=0}^{k}\left({k\atop s}\right)\lambda (s,&amp;lt; k_{ X} &amp;gt;){i}^{s}{(1 - i)}^{k-s}$$ \end{document}</div> </div> </div> <p /> <p>where <i>y</i><sub><i>X</i></sub>(<i>k</i>,â€‰<i>i</i>) is the prevalence of the disease and <span class="inline-formula" title="inline-formula"> </span></p><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M11">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document} $$&amp;lt; k_{X} &amp;gt;=&amp;lt; k_{X}^{in} &amp;gt;+ &amp;lt; k_{X}^{out} &amp;gt;$$ \end{document}</div> </div>. Note that the force of infection <i>y</i><sub><i>X</i></sub>(<i>k</i>,â€‰<i>i</i>) considers the expected rate of infection over all possible neighbourhoods of any node in community X according to the current prevalence. Thus <i>y</i><sub><i>X</i></sub>(<i>k</i>,â€‰<i>i</i>) serves to replace the uniform mixing assumption with an approximation of the mixing between infective and susceptible individuals occurring across different parts of the network. Next, we define the fraction of infected individuals both within and outside of community X. The prevalence of the disease within the community X is<p /> <p> </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">10</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M12">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document} $$i_{X}^{in} = \frac{I_{X}} {n(X)}$$ \end{document}</div> </div> </div> <p /> <p>where <i>i</i><sub><i>X</i></sub> denotes the number of infected individuals in X. The expected number of infected external acquaintances to the boundary nodes of community X is givenÂ by</p> <p> </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">11</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M13">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document} $$i_{X}^{out} = \frac{p_{e}(X)} {&amp;lt; k_{X}^{out} &amp;gt;}\sum\limits_{Y \in \varsigma,Y \neq X}p_{e}(Y )I_{Y }$$ \end{document}</div> </div> </div> <p /> <p>Using these definitions we can write expressions for the force of infection experienced by both boundary and non-boundary nodes. If <i>S</i><sub><i>X</i></sub> is the number of susceptible individuals of community X, the number of susceptible non-boundary nodes and the force of infection they experience are, respectively, </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">12</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M14">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document} $$\begin{array}{rcl} \left (1 - \frac{B_{X}} {n(X)}\right )S_{X},\qquad y(&amp;lt; k_{X}^{in} &amp;gt;,i_{ X}^{in})&amp;amp; &amp;amp;\end{array}$$ \end{document}</div> </div> </div> and the number of susceptible boundary nodes which can acquire infection from either outside or inside the community, and the force of infection they experience are, respectively, <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">13</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M15">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document} $$\begin{array}{rcl} \frac{B_{x}} {n(X)}S_{X},\qquad y(&amp;lt; k_{X}^{in} &amp;gt;,i_{ X}^{in}) + y(&amp;lt; k_{ X}^{out} &amp;gt;,i_{ X}^{out}) - y(&amp;lt; k_{ X}^{in} &amp;gt;,i_{ X}^{in})y(&amp;lt; k_{ X}^{out} &amp;gt;,i_{ X}^{out})&amp;amp; &amp;amp;\end{array}$$ \end{document}</div> </div> </div> <p /> <p>Using the expressions defined above, the dynamics of community X can be described by the following ODEs: </p><div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">14</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M16">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document} $$\begin{array}{rcl} \frac{dS_{X}} {dt} &amp;amp; =&amp;amp; -\left [S_{X}\left (y\left (&amp;lt; k_{X}^{in} &amp;gt;,i_{ X}^{in}\right ) + \frac{B_{X}} {n(X)}\left [y(&amp;lt; k_{X}^{out} &amp;gt;,i_{ X}^{out})\right.\right.\right. \\ &amp;amp; &amp;amp; \quad \left.\left.\left.-y(&amp;lt; k_{X}^{in} &amp;gt;,i_{ X}^{in})y(&amp;lt; k_{ X}^{out} &amp;gt;,i_{ X}^{out})\right ]\right )\right ] \end{array}$$ \end{document}</div> </div> </div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">15</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M17">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document} $$\begin{array}{rcl} \frac{dI_{X}} {dt} &amp;amp; =&amp;amp; -S_{X}\left (y\left (&amp;lt; k_{X}^{in} &amp;gt;,i_{ X}^{in}\right ) + \frac{B_{X}} {n(X)}\left [y(&amp;lt; k_{X}^{out} &amp;gt;,i_{ X}^{out})\right.\right. \\ &amp;amp; &amp;amp; \quad \left.\left.-y(&amp;lt; k_{X}^{in} &amp;gt;,i_{ X}^{in})y\left (&amp;lt; k_{ X}^{out} &amp;gt;,i_{ X}^{out}\right )\right ]\right ) - \alpha I_{ X} \end{array}$$ \end{document}</div> </div> </div> <div class="disp-formula" title="disp-formula"><span class="label" tagx="label" title="label">16</span><div class="tex" title="tex"> <div class="tex-math_UNKNOWN" id="M18">
\documentclass[12pt]{minimal} \usepackage{amsmath} \usepackage{wasysym} \usepackage{amsfonts} \usepackage{amssymb} \usepackage{amsbsy} \usepackage{mathrsfs} \usepackage{upgreek} \setlength{\oddsidemargin}{-69pt} \begin{document} $$\begin{array}{rcl} \frac{dR_{X}} {dt} &amp;amp; =&amp;amp; \alpha I_{X}\end{array}$$ \end{document}</div> </div> </div> <p /> <p>For a population of C communities, C sets of three equations need to be solved simultaneously to approximate the disease dynamics, as the equations all depend on <i>i</i><sub><i>X</i></sub><sup><i>out</i></sup> which is defined according to the prevalence across all communities. We can remove this dependency by setting <i>i</i><sub><i>X</i></sub><sup><i>out</i></sup>â€‰=â€‰0 for all communities, so that the external force of infection is <i>y</i>(â€‰<div class="â€‰_UNKNOWN"><i>k</i><sub><i>X</i></sub><sup><i>in</i></sup>â€‰&amp;gt;â€‰)â€‰=â€‰0, thus obtaining the dynamics of the disease if each community is isolated from the rest of the population.</div></p> <p>This model has been implemented in an open software available from the authors that allow testing for different community structure, topologies of connections (variable boundaries), individual and community awareness <i>J</i>,â€‰<i>H</i> and modelling interventions. Using the mean-field approximation, [<a href="#CR18">18</a>] showed that in the presence of interaction between communities, values of J higher than the critical value are necessary to prevent a community from experiencing an epidemic. Since in a realistic situation the amount of precautions an individual can take are limited, a consequence of this result is that, in the presence of community interactions, even high awareness of the disease may be insufficient to prevent an epidemic.</p> <p>Applying a different transmission model to this framework is also possible. For example, if we wished to implement a susceptible-exposed-infected-recovered (SEIR) model we could easily do so by defining the mean period of time Î± for the exposed period, so that at every time step an individual in the exposed state becomes infected with probability 1â€‰âˆ•â€‰Î±. An important consideration might be to allow for the modelling of diseases with asymptomatic infectious cases, such as influenza. Asymptomatic individuals may still be infectious, although potentially less so than symptomatic cases.</p> <p>At the microscale modelling, more efforts are needed to estimate better the effective reproduction number of seasonal versus pandemic disease and how the magnitude of this number is affected by both the presence of risky conditions because of health (e.g. the genetic structure) or behaviour of a community and the genetic diversity and the accumulation of mutations of the pathogen, particularly its antigenic evolution. At the macroscopic scale, the current economic crisis seems to produce relevant effects on the possibility of pandemic and seasonal diseases due to the toughened conditions for certain high-risk groups, including migrants, homeless persons, and prison populations that are particularly vulnerable to reductions in treatment access, quality of care due to drops in government spending [<a href="#CR28">28</a>]. These groups show stress with increased high-risk behaviour, consumption of alcohol, tobacco, substance abuse and worse nutrition. Clearly a better description of the information acquisition dynamics and of heuristics involved in the decision making process for various communities would lead to better prediction.</p> <p>The combining analysis of micro- and macroscales of an epidemic could provide an accurate estimate of the effective reproduction number of the disease. The aim of mitigation strategies is to reduce the effective reproduction number of the disease to a value close to or less than the epidemic threshold and thus prevent the disease from causing a large epidemic outbreak.</p> <p>Recently Lio et al. [<a href="#CR20">20</a>] have combined formal languages and hybrid modelling to model the use of different therapies between therapies following the spread of an epidemic. Important efforts are focused on developing multi-scale approaches of epidemics: from the intracellular signalling and the communication between cells which form the levels of the immune response, to the tissue and organs levels that form the level of the structural response variation induced by the disease and are at the core of therapies application, to the population levels where parameters are related to social contacts, economy, psychology, for instance the cost of the therapies and vaccination. Note that all scales of description are dependent on each other. Furthermore, usage of personal technology tools will increase the individual and community differences of awareness.</p> </div> <div class="futureriskandawarenessmanagementthroughselfmonitoringdevicesandpersonalgenomics" title="sec"> <div class="title" tagx="title" title="title">
Future Risk and Awareness Management Through Self Monitoring Devices and Personal Genomics</div> <p>There is growing interest in using technological devices to continuously monitor oneâ€™s own health. In the past these tools were used by people trying to lose weight or improve their fitness. Athletes and their coaches commonly make detailed notes on nutrition, training sessions, sleep and other variables. New technologies (e.g. wristbands for heart rate, sleep patterns, skin conductance) make it simpler than ever to gather and analyse personal data. Sensors have shrunk and become cheaper. In USA and Europe, quantified self conferences are showing people new ways to deal with medical problems or improve their quality of life with technology. For instance together with various signals such as heart rate, posture, motion and temperature, co-ordination, reaction times, memory and emotions could be monitored. Personal genomics will provide an estimate of the personal risk in an epidemics. Ultra-high-throughput sequencing strategies have now been used to sequence more than 10,000 full, individual human genomes. One of the clinical implication is the possibility to obtain an almost complete map of the antigenic coverage of each of us against existing and predicted (on genomic basis) epitopes of pathogens. The availability of personal genome sequences and antigenic response data will require the development of models that will incorporate such information. It is noteworthy that the price of sequencing is dropping quickly. This suggests that samples could be collected from mining sewage systems and spots like toilets in pubs, trains, airports or emergency rooms to monitor public health concerns, for example, flu outbreaks that can then be sent off for sequencing, allowing us to build an early alarm map over time of the arrival of pathogens. We can imagine this map to be consulted using a phone in the same way we now watch the weather forecasting map. The combination of personal genomics would provide useful information on the personal threat posed by the arising pathogen strain.</p> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>This study was supported by the following project: RECOGNITION: Relevance and Cognition for Self-Awareness in a Content-Centric Internet (257756), which is funded by the European Commission within the 7th Framework Programme (FP7).</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Ades, A.E. Sutton, A.J.: J. R. Stat. Soc. Ser. A Stat. Soc. <b>169</b>, 5 (2006)</span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alford</span><span class="given-names" tagx="given-names" title="given-names">RH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kasel</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gerone</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knight</span><span class="given-names" tagx="given-names" title="given-names">V</span></span></span><span class="source" tagx="source" title="source">Proc. Soc. Exp. Biol. Med.</span><span class="year" tagx="year" title="year">1966</span><span class="volume" tagx="volume" title="volume">122</span><span class="fpage" tagx="fpage" title="fpage">800</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5918954">5918954</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Bagnoli, F., LiÃ², P., Sguanci, L.: Phys. Rev. E. Stat. Nonlin. Soft. Matter Phys. <b>76</b>, 061904 (2007)</span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balocco</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lio</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="source" tagx="source" title="source">Energ. Buildings</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">43</span><span class="fpage" tagx="fpage" title="fpage">246</span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brownstein</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kleinman</span><span class="given-names" tagx="given-names" title="given-names">KP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mandl</span><span class="given-names" tagx="given-names" title="given-names">KD</span></span></span><span class="source" tagx="source" title="source">Am. J. Epi.</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">162</span><span class="fpage" tagx="fpage" title="fpage">686</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/aje/kwi257">10.1093/aje/kwi257</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coelho</span><span class="given-names" tagx="given-names" title="given-names">FC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Codeo</span><span class="given-names" tagx="given-names" title="given-names">CT</span></span></span><span class="source" tagx="source" title="source">PLoS Comput. Biol.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">5</span><span class="issue" tagx="issue" title="issue">7</span><span class="fpage" tagx="fpage" title="fpage">e1000425</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pcbi.1000425">10.1371/journal.pcbi.1000425</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19593365">19593365</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Condon</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karp</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span></span><span class="source" tagx="source" title="source">Random Struct. Algor.</span><span class="year" tagx="year" title="year">2001</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">116</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/1098-2418(200103)18:2&lt;116::AID-RSA1001&gt;3.0.CO;2-2">10.1002/1098-2418(200103)18:2&amp;lt;116::AID-RSA1001&amp;gt;3.0.CO;2-2</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dunbar</span><span class="given-names" tagx="given-names" title="given-names">RI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shultz</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">317</span><span class="fpage" tagx="fpage" title="fpage">1344</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1145463">10.1126/science.1145463</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17823343">17823343</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Durham</span><span class="given-names" tagx="given-names" title="given-names">D.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Casman</span><span class="given-names" tagx="given-names" title="given-names">E.A.</span></span></span><span class="source" tagx="source" title="source">J. Roy. Soc., Interface</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">562</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21775324">21775324</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fortunato</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="source" tagx="source" title="source">Phy. Rep.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">486</span><span class="fpage" tagx="fpage" title="fpage">75</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.physrep.2009.11.002">10.1016/j.physrep.2009.11.002</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gigerenzer</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hertwig</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pachur</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="source" tagx="source" title="source">Heuristics: The Foundations of Adaptive Behavior</span><span class="year" tagx="year" title="year">2011</span><span class="edition" tagx="edition" title="edition">1</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">USA</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford University Press</span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Giesecke</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Johnson</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hawkins</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Noone</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nicoll</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span></span><span class="source" tagx="source" title="source">Roy. Statist. Soc. Ser. A</span><span class="year" tagx="year" title="year">1994</span><span class="volume" tagx="volume" title="volume">157</span><span class="fpage" tagx="fpage" title="fpage">89</span><span class="pub-id"><a href="https://dx.doi.org/10.2307/2983507">10.2307/2983507</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ginsberg</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mohebbi</span><span class="given-names" tagx="given-names" title="given-names">MH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patel</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brammer</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smolinski</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brilliant</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="source" tagx="source" title="source">Nature</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">457</span><span class="fpage" tagx="fpage" title="fpage">1012</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature07634">10.1038/nature07634</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19020500">19020500</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goodwin</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haque</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Neto</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Myers</span><span class="given-names" tagx="given-names" title="given-names">LB</span></span></span><span class="source" tagx="source" title="source">BMC Infect. Dis.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">166</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2334-9-166">10.1186/1471-2334-9-166</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19807908">19807908</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goubar</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ades</span><span class="given-names" tagx="given-names" title="given-names">AE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Angelis</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McGarrigle</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mercer</span><span class="given-names" tagx="given-names" title="given-names">CH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tookey</span><span class="given-names" tagx="given-names" title="given-names">PA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fenton</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gill</span><span class="given-names" tagx="given-names" title="given-names">ON</span></span></span><span class="source" tagx="source" title="source">J. Roy. Statist. Soc. Ser. A</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">171</span><span class="issue" tagx="issue" title="issue">3</span><span class="fpage" tagx="fpage" title="fpage">541</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1467-985X.2007.00537.x">10.1111/j.1467-985X.2007.00537.x</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghys</span><span class="given-names" tagx="given-names" title="given-names">PD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grassly</span><span class="given-names" tagx="given-names" title="given-names">NC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garnett</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stanecki</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stover</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walker</span><span class="given-names" tagx="given-names" title="given-names">N</span></span></span><span class="source" tagx="source" title="source">Brit. Med. J.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">80</span><span class="fpage" tagx="fpage" title="fpage">i5</span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grenfell</span><span class="given-names" tagx="given-names" title="given-names">BT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pybus</span><span class="given-names" tagx="given-names" title="given-names">OG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gog</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wood</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Daly</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mumford</span><span class="given-names" tagx="given-names" title="given-names">JA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span></span><span class="source" tagx="source" title="source">Science</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">303</span><span class="fpage" tagx="fpage" title="fpage">327</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1090727">10.1126/science.1090727</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14726583">14726583</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Kitchovitch, S., LiÃ², P.: PLoS ONE <b>6</b>(7), e22220. doi:10.1371/ (2011)</span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leung</span><span class="given-names" tagx="given-names" title="given-names">I.X.Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibbs</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bagnoli</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sorathiya</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">LiÃ²</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="mixed-article-title" title="mixed-article-title">Contact Network Modeling of Flu Epidemics. In: Umeo, H., Morishita, S., Nishinari, K., Komatsuzaki, T., Bandini, S. (eds.) Cellular Automata</span><span class="source" tagx="source" title="source">Proceedings</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">5191</span><span class="fpage" tagx="fpage" title="fpage">354</span><span class="lpage" tagx="lpage" title="lpage">361</span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">LiÃ², P., Merell, E., Paoletti, N.: Disease processes as hybrid dynamical systems. Proceedings of the First International Workshop on Hybrid Systems and Biology. Newcasle upon Tyne, UK, September 3, (2012)</span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cui</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="source" tagx="source" title="source">Commun. Nonlinear Sc. Numer. Simulat.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">2353</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.cnsns.2008.06.024">10.1016/j.cnsns.2008.06.024</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lowen</span><span class="given-names" tagx="given-names" title="given-names">AC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mubareka</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tumpey</span><span class="given-names" tagx="given-names" title="given-names">TM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Garca-Sastre</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Palese</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="source" tagx="source" title="source">PNAS</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">103</span><span class="fpage" tagx="fpage" title="fpage">26</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0604157103">10.1073/pnas.0604157103</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Nguyen, V.A., KoukolÃ­kovÃ¡-Nicola, Z., Bagnoli, F., LiÃ³, P.: Bayesian Inference on Hidden Knowledge in High-Throughput Molecular Biology Data. PRICAI 2008: Trends in Artificial Intelligence, pp. 829â€"38 (2008)</span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">dOnofrio</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Manfredi</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salinelli</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span><span class="source" tagx="source" title="source">Theor. Pop. Biol.</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">71</span><span class="fpage" tagx="fpage" title="fpage">301</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17335862">17335862</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rizzolatti</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Craighero</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="source" tagx="source" title="source">Annu. Rev. Neurosci.</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">27</span><span class="fpage" tagx="fpage" title="fpage">169</span><span class="pub-id"><a href="https://dx.doi.org/10.1146/annurev.neuro.27.070203.144230">10.1146/annurev.neuro.27.070203.144230</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15217330">15217330</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rosenquist</span><span class="given-names" tagx="given-names" title="given-names">JN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Murabito</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fowler</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Christakis</span><span class="given-names" tagx="given-names" title="given-names">NA</span></span></span><span class="source" tagx="source" title="source">Ann. Intern. Med.</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">152</span><span class="fpage" tagx="fpage" title="fpage">426</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20368648">20368648</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Simon, H.A.: Models of Bounded Rationality. Economic Analysis and Public Policy, vol. 1, pp. 235. MIT Press, Cambridge (1955)</span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suhrcke</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stuckler</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suk</span><span class="given-names" tagx="given-names" title="given-names">JE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Desai</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Senek</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McKee</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsolova</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Basu</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Abubakar</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hunter</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rechel</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Semenza</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span></span><span class="source" tagx="source" title="source">PLoS One.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">e20724</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0020724">10.1371/journal.pone.0020724</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21695209">21695209</a></span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sjoberg</span><span class="given-names" tagx="given-names" title="given-names">L</span></span></span><span class="source" tagx="source" title="source">Risk Anal.</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">20</span><span class="issue" tagx="issue" title="issue">11</span><span class="fpage" tagx="fpage" title="fpage">40</span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tan</span><span class="given-names" tagx="given-names" title="given-names">TQ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhattacharya</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gerbie</span><span class="given-names" tagx="given-names" title="given-names">MV</span></span></span><span class="source" tagx="source" title="source">J. Reprod. Med.</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">56</span><span class="fpage" tagx="fpage" title="fpage">301</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21838159">21838159</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tchuenche</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dube</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bhunu</span><span class="given-names" tagx="given-names" title="given-names">CP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">RJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bauch</span><span class="given-names" tagx="given-names" title="given-names">CT</span></span></span><span class="source" tagx="source" title="source">BMC Public Health</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">11</span><span class="issue" tagx="issue" title="issue">Suppl 1</span><span class="fpage" tagx="fpage" title="fpage">S5</span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wegwarth</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gaissmaier</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gigerenzer</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="source" tagx="source" title="source">Med. Educ.</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">43</span><span class="fpage" tagx="fpage" title="fpage">721</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/j.1365-2923.2009.03359.x">10.1111/j.1365-2923.2009.03359.x</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19573016">19573016</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">JT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Riley</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fraser</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leung</span><span class="given-names" tagx="given-names" title="given-names">GM</span></span></span><span class="source" tagx="source" title="source">Plos Med.</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">3</span><span class="issue" tagx="issue" title="issue">9</span><span class="fpage" tagx="fpage" title="fpage">e3614057</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pmed.0030361">10.1371/journal.pmed.0030361</a></span></span></li> </ul> </div> </div>  </body></html>