<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Virol J</span><span class="iso-abbrev" title="iso-abbrev">Virol. J</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Virology Journal</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1743-422X</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">BioMed Central</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7359424">7359424</a></span><span class="publisher-id" title="publisher-id">1382</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1186/s12985-020-01382-2">10.1186/s12985-020-01382-2</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Origin and evolution of emerging <i>Liao ning Virus</i> (genus <i>Seadornavirus</i>, family <i>Reoviridae</i>)</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Jun</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-5182-4750</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Hong</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">liuhongseminar@hotmail.com</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Jiahui</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Jiheng</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Jianming</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Jiayue</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Xin</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ji</span><span class="given-names" tagx="given-names" title="given-names">Hongfang</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ding</span><span class="given-names" tagx="given-names" title="given-names">Zhongfeng</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">Han</span></span><a href="#Aff2">2</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">Chunyang</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">Qian</span></span><a href="#Aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">Guodong</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">gdliang@hotmail.com</span></span><a href="#Aff3">3</a></span><span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.412509.b</span><span class="institution-id" title="institution-id">0000 0004 1808 3414</span><span class="institution" title="institution">Shandong Provincial Research Center for Bioinformatic Engineering and Technique, School of Life Sciences, </span><span class="institution" title="institution">Shandong University of Technology, </span></div></span><span class="citation_author_institution" id="Aff2">[2], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.9227.e</span><span class="institution-id" title="institution-id">0000000119573309</span><span class="institution" title="institution">Key Laboratory of Special Pathogens and Biosafety, Wuhan Institute of Virology, </span><span class="institution" title="institution">Chinese Academy of Sciences, </span></div></span><span class="citation_author_institution" id="Aff3">[3], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">grid.198530.6</span><span class="institution-id" title="institution-id">0000 0000 8803 2373</span><span class="institution" title="institution">State Key Laboratory of Infectious Disease Prevention and Control, National Institute for Viral Disease Control and Prevention, </span><span class="institution" title="institution">Chinese Center for Disease Control and Prevention, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-7-7</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2020-7-7</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020</span></span><span class="volume" tagx="volume" title="volume">17</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">105</span><span class="history" title="history"><span class="received" title="received">received: 2020-1-23</span><span class="accepted" title="accepted">accepted: 2020-7-3</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b>This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>. The Creative Commons Public Domain Dedication waiver (<a href="http://creativecommons.org/publicdomain/zero/1.0/">http://creativecommons.org/publicdomain/zero/1.0/</a>) applies to the data made available in this article, unless otherwise stated in a credit line to the data.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p id="Par1"><i>Liao ning virus</i> (LNV) is a member of the genus <i>Seadornavirus</i>, family <i>Reoviridae</i> and has been isolated from kinds of vectors in Asia and Australia. However, there are no systematic studies describe the molecular genetic evolution and migration of LNVs. With the development of bioinformatics, viral genetic data combining the information of virus isolation time and locations could be integrated to infer the virus evolution and spread in nature.</p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <p id="Par2">Here, a phylogenetic and phylogeographic analysis using Bayesian Markov chain Monte Carlo simulations was conducted on the LNVs isolated from a variety of vectors during 1990â€"2014 to identify the evolution and migration patterns of LNVs.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p id="Par3">The results demonstrated that the LNV could be divided into 3 genotypes, of which genotype 1 mainly composed of LNVs isolated from Australia during 1990 to 2014 and the original LNV strain (LNV-NE97â€"31) isolated from Liaoning province in northern China in 1997, genotype 2 comprised of the isolates all from Xinjiang province in western China and genotype 3 consisted the isolates from Qinghai and Shanxi province of central China. LNVs emerged about 272â€‰years ago and gradually evolved into three lineages in the order genotype 1, genotype 2 and genotype 3. Following phylogeographic analysis, it shows genotype 1 LNVs transmitted from Australia (113Â°E-153Â°E,10Â°S-42Â°S) to Liaoning province (118Â°E-125Â°E,38Â°N-43Â°N) in Northeast Asian continent then further spread across the central part of China to western China (75Â°E-95Â°E,35Â°N-50Â°N).</p> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p id="Par4">LNVs were initially isolated from Liaoning province of China in the Northeast Asia, however, the present study revealed that LNVs were first appeared in Australia in the South Pacific region and transmitted to mainland China then rapidly spread across China and evolved three different genotypes. The above results suggested that LNV had the characteristics of long-distance transmission and there were great genetic diversity existed in the LNV population. Notably, current information of 80 strains of LNVs are limited. It is of great importance to strengthen the surveillance of LNVs to explore its real origin in nature and monitoring of the LNVsâ€™ population variation and maintain vigilance to avoid LNV breaking through the species barrier and further clarify its relationship to human and animal infection.</p> </div> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd"><i>Liao ning virus</i></div> <div class="kwd" title="kwd">
LNV</div> <div class="kwd" title="kwd"><i>Seadornavirus</i></div> <div class="kwd" title="kwd">
Evolution</div> <div class="kwd" title="kwd">
Migration</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">http://dx.doi.org/10.13039/501100001809</span><span class="institution" title="institution">National Natural Science Foundation of China</span></div> </div><span class="award-id" tagx="award-id" title="award-id">81301479</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Hong</span></span></span></div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">Development Grant of the State Key Laboratory of Infectious Disease Prevention and Control</span></div><span class="award-id" tagx="award-id" title="award-id">2014SKLID103</span><span class="award-id" tagx="award-id" title="award-id">2018SKLID309</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Hong</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">Guodong</span></span></span></div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution"> Natural Science Foundation of Shandong Province, China </span></div><span class="award-id" tagx="award-id" title="award-id">ZR2013HQ008</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Hong</span></span></span></div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">Natural Science Foundation of Shandong Province, China </span></div><span class="award-id" tagx="award-id" title="award-id">ZR2019MH020</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ji</span><span class="given-names" tagx="given-names" title="given-names">Hongfang</span></span></span></div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">HealthÂ CommissionÂ ofÂ HubeiÂ Province</span></div><span class="award-id" tagx="award-id" title="award-id">WJ2019Q060</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">Han</span></span></span></div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="Â© The Author(s) 2020" /> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="Par27"><i>Liao ning virus</i> (LNV) belongs to the genus <i>Seadornavirus</i> of the family <i>Reoviridae</i> [<a href="#CR1">1</a>]. The genus <i>Seadornavirus</i> comprises 3 species, <i>Banna virus</i> (BAV), <i>Kadipiro virus</i> (KDV) and LNV [<a href="#CR1">1</a>â€"<a href="#CR3">3</a>]. The <i>Seadornaviruses</i> have a genome composed of 12 segments of double-stranded RNA, which decreases with the relative molecular weight during gel electrophoresis, and is named 1â€"12 segments [<a href="#CR2">2</a>]. The full length of the LNV genome is about 21,000â€‰bp, with segment lengths that range from 3747â€‰bp (segment 1) to 759â€‰bp (segment 12). Each segment contains a open reading frame encodes a viral protein (Viral protein, VP), of which 5 are non-structural proteins (VP5, VP6, VP7, VP11, VP12), 7 are structural proteins (VP1, VP2, VP3, VP4, VP8, VP9, VP10).VP10 protein is an outer capsid structural protein which directly interacts with the receptor of host cells and is the main region that determines the antigenicity of the virus [<a href="#CR3">3</a>].</p> <p id="Par28">It is reported that the <i>Seadornaviruses</i> could be carried by a variety of blood-sucking insects including mosquitoes, ticks, and midges and even have been isolated from pigs, cattle and patients with fever and encephalitis [<a href="#CR4">4</a>â€"<a href="#CR10">10</a>]. Studies have also shown that BAV, which is the prototype of genus <i>Seadornavirus</i>, is an emerging pathogen that causes human viral encephalitis [<a href="#CR2">2</a>]. Therefore, <i>Seadornaviruses</i> may well be a group of newly discovered viruses close related to human and animal diseases [<a href="#CR2">2</a>, <a href="#CR4">4</a>].</p> <p id="Par29">LNV was initially isolated from mosquitoes (<i>Aedes dorsalis</i>) in Liaoning province, northeast China in 1997 [<a href="#CR10">10</a>], since then additional isolates have been isolated from mosquitoes (genus <i>culex</i> and <i>Aedes</i>) in Shanxi province and Qinghai province in China [<a href="#CR5">5</a>, <a href="#CR7">7</a>, <a href="#CR8">8</a>].Interestingly,it is observed that all of the LNVs have only been isolated from the northern part of China and no LNV has been reported in other provinces in China and abroad [<a href="#CR6">6</a>].Once, the LNV has been considered to be a virus specie only restricted in northern part of China. Until a recent research reported that number of LNV strains have been isolated from mosquitoes of four genera (<i>Culex</i>, <i>Anopheles</i>, <i>Mansonia</i> and <i>Aedes</i>) in Australia in 2016 [<a href="#CR11">11</a>, <a href="#CR12">12</a>]. This result demonstrated that LNV was not only limited in mainland China but also widely distributed in Australia in the South Pacific region. All of these research suggested that LNV was a widely distributed virus that could be transmitted by kinds of blood-sucking vectors and its geographical distribution even exceeds that of the other two <i>Seadornaviruses</i> (BAV, KDV) [<a href="#CR1">1</a>, <a href="#CR4">4</a>].</p> <p id="Par31">Previous researches have reported on the molecular genetic evolution of LNV isolated in China and Australia [<a href="#CR5">5</a>, <a href="#CR7">7</a>, <a href="#CR11">11</a>]. However, the data used for analysis were only limited to the local LNV isolates [<a href="#CR5">5</a>, <a href="#CR7">7</a>, <a href="#CR11">11</a>]. China in Asia and Australia in the South Pacific are located in the northern and southern hemispheres, respectively. The natural environment in the two regions were significant different from each other. These observations raise several questions. What is the genetic relationship between the LNVs isolated from different regions? What are the characteristics of the molecular evolution of the entire LNV population? Where is the origin of LNVs? Recent revolutionary developments in virology bioinformatics provide unprecedented opportunities for analyzing the viral genetic data (nucleotide or amino acid) to model the evolutionary relationships between virus samples and help to explain epidemiological patterns and uncover processes of transmission [<a href="#CR13">13</a>â€"<a href="#CR15">15</a>]. Examples of such analyses include the estimation of origin and divergence time of <i>Japanese encephalitis virus</i> (JEV) [<a href="#CR16">16</a>, <a href="#CR17">17</a>],identification of phylogenetic relationships among viral lineages in <i>Zika virus</i> (ZIKV) [<a href="#CR18">18</a>], inference of phylogeographic history and migration patterns of recent epidemics of <i>Middle East respiratory syndrome coronavirus</i> (MERS-CoV) [<a href="#CR13">13</a>] and <i>Ebola virus</i> (EBOV) [<a href="#CR19">19</a>]. Therefore, in the present study a comprehensive phylogeographic study of all the LNVs isolated worldwide were conducted to explore the molecular genetic evolution and migration patterns of LNVs isolated from different vectors since 1990 in China and Australia.</p> </div> <div class="materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
Materials and methods</div> <div class="datasetconstructionoftheÂ 10thsegmentgeneoflnv" title="sec"> <div class="title" tagx="title" title="title">
Data set construction of theÂ 10th segment gene of LNV</div> <p id="Par32">LNV is a 12-segment double-stranded RNA virus. The 10th segment gene encodes the viral cell attachment protein, which directly interacts with the receptor on host cells [<a href="#CR3">3</a>]. Thus, the phylogenetic analysis based on this segment can best demonstrated the evolutionary and dispersal trends of viral strains from different isolation sites and vectors. Thus,we downloaded all the 10th segment gene sequences of LNV from GenBank as of June 2019 (TableÂ <a href="#Tab1">1</a>). The data set contained 80 sequences, representing the samples isolated from kinds of mosquitoes:genus <i>Culex</i> (<i>n</i>â€‰=â€‰64), genus <i>Aedes</i> (<i>n</i>â€‰=â€‰10), genus <i>Ochlerotatus</i> (<i>n</i>â€‰=â€‰3),unidentified mosquitoes (<i>n</i>Â =â€‰3). The LNV isolation sites included the Liaoning province, Shanxi province, Qinghai province and Xinjiang province in China as well as New South Wales, Northern Territory, Queensland and Western Australia in Australia. The geographical distribution extends from Australia (113Â°E-153Â°E, 10Â°S-42Â°S) to mainland China (73Â°E-135Â°E, 3Â°N-53Â°N) (Fig.Â <a href="#Fig1">1</a>). <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">Table 1</span> </div></p><p>The background information of LNVs used in this study</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th>NO.</th> <th>Strain</th> <th>Location</th> <th>Host</th> <th>Date</th> <th>Accession no.</th> </tr> </thead> <tbody> <tr> <td>1</td> <td>LN-SW10194</td> <td>Australia: Western Australia</td> <td>mosquito</td> <td>1990</td> <td>MG725066.1</td> </tr> <tr> <td>2</td> <td>LNV-NE9712</td> <td>China:Liaoning Province</td> <td><i>Aedes dorsalis mosquitoes</i></td> <td>1997</td> <td>NC_007745.1</td> </tr> <tr> <td>3</td> <td>LNSV-NE97â€"31</td> <td>China:Liaoning Province</td> <td><i>Aedes dorsalis mosquitoes</i></td> <td>1997</td> <td>AY317108.1</td> </tr> <tr> <td>4</td> <td>LNV QLD2005</td> <td>Australia: Brisbane</td> <td><i>Culex annulirostris</i></td> <td>2005</td> <td>MG725847.1</td> </tr> <tr> <td>5</td> <td>J60</td> <td>China:Xinjiang Province</td> <td><i>Culex mosquitoes</i></td> <td>2005</td> <td>HM745532.1</td> </tr> <tr> <td>6</td> <td>J59</td> <td>China:Xinjiang Province</td> <td><i>Culex mosquitoes</i></td> <td>2005</td> <td>HM745531.1</td> </tr> <tr> <td>7</td> <td>J55</td> <td>China:Xinjiang Province</td> <td><i>Culex mosquitoes</i></td> <td>2005</td> <td>HM745530.1</td> </tr> <tr> <td>8</td> <td>J54</td> <td>China:Xinjiang Province</td> <td><i>Culex mosquitoes</i></td> <td>2005</td> <td>HM745529.1</td> </tr> <tr> <td>9</td> <td>Jâ€™49</td> <td>China:Xinjiang Province</td> <td><i>Culex mosquitoes</i></td> <td>2005</td> <td>HM745528.1</td> </tr> <tr> <td>10</td> <td>Jâ€™46</td> <td>China:Xinjiang Province</td> <td><i>Culex mosquitoes</i></td> <td>2005</td> <td>HM745527.1</td> </tr> <tr> <td>11</td> <td>J44</td> <td>China:Xinjiang Province</td> <td><i>Culex mosquitoes</i></td> <td>2005</td> <td>HM745525.1</td> </tr> <tr> <td>12</td> <td>Jâ€™44</td> <td>China:Xinjiang Province</td> <td><i>Culex mosquitoes</i></td> <td>2005</td> <td>HM745526.1</td> </tr> <tr> <td>13</td> <td>Jâ€™41</td> <td>China:Xinjiang Province</td> <td><i>Culex mosquitoes</i></td> <td>2005</td> <td>HM745524.1</td> </tr> <tr> <td>14</td> <td>Jâ€™37</td> <td>China:Xinjiang Province</td> <td><i>Culex mosquitoes</i></td> <td>2005</td> <td>HM745523.1</td> </tr> <tr> <td>15</td> <td>Jâ€™36</td> <td>China:Xinjiang Province</td> <td><i>Culex mosquitoes</i></td> <td>2005</td> <td>HM745522.1</td> </tr> <tr> <td>16</td> <td>J35</td> <td>China:Xinjiang Province</td> <td><i>Culex mosquitoes</i></td> <td>2005</td> <td>HM745521.1</td> </tr> <tr> <td>17</td> <td>J32</td> <td>China:Xinjiang Province</td> <td><i>Culex mosquitoes</i></td> <td>2005</td> <td>HM745520.1</td> </tr> <tr> <td>18</td> <td>J29</td> <td>China:Xinjiang Province</td> <td><i>Culex mosquitoes</i></td> <td>2005</td> <td>HM745519.1</td> </tr> <tr> <td>19</td> <td>J27</td> <td>China:Xinjiang Province</td> <td><i>Culex mosquitoes</i></td> <td>2005</td> <td>HM745518.1</td> </tr> <tr> <td>20</td> <td>J24</td> <td>China:Xinjiang Province</td> <td><i>Culex mosquitoes</i></td> <td>2005</td> <td>HM745517.1</td> </tr> <tr> <td>21</td> <td>Jâ€™18</td> <td>China:Xinjiang Province</td> <td><i>Culex mosquitoes</i></td> <td>2005</td> <td>HM745516.1</td> </tr> <tr> <td>22</td> <td>Jâ€™13</td> <td>China:Xinjiang Province</td> <td><i>Culex mosquitoes</i></td> <td>2005</td> <td>HM745515.1</td> </tr> <tr> <td>23</td> <td>J4</td> <td>China:Xinjiang Province</td> <td><i>Culex mosquitoes</i></td> <td>2005</td> <td>HM745514.1</td> </tr> <tr> <td>24</td> <td>J2</td> <td>China:Xinjiang Province</td> <td><i>Culex mosquitoes</i></td> <td>2005</td> <td>HM745513.1</td> </tr> <tr> <td>25</td> <td>J1</td> <td>China:Xinjiang Province</td> <td><i>Culex mosquitoes</i></td> <td>2005</td> <td>HM745512.1</td> </tr> <tr> <td>26</td> <td>B10</td> <td>China:Xinjiang Province</td> <td><i>Culex mosquitoes</i></td> <td>2005</td> <td>HM745511.1</td> </tr> <tr> <td>27</td> <td>B9</td> <td>China:Xinjiang Province</td> <td><i>Culex mosquitoes</i></td> <td>2005</td> <td>HM745510.1</td> </tr> <tr> <td>28</td> <td>B8</td> <td>China:Xinjiang Province</td> <td><i>Culex mosquitoes</i></td> <td>2005</td> <td>HM745509.1</td> </tr> <tr> <td>29</td> <td>B7</td> <td>China:Xinjiang Province</td> <td><i>Culex mosquitoes</i></td> <td>2005</td> <td>HM745508.1</td> </tr> <tr> <td>30</td> <td>B6</td> <td>China:Xinjiang Province</td> <td><i>Culex mosquitoes</i></td> <td>2005</td> <td>HM745507.1</td> </tr> <tr> <td>31</td> <td>B5</td> <td>China:Xinjiang Province</td> <td><i>Culex mosquitoes</i></td> <td>2005</td> <td>HM745506.1</td> </tr> <tr> <td>32</td> <td>XJ06121</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2006</td> <td>HM136838.1</td> </tr> <tr> <td>33</td> <td>XJ0615</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2006</td> <td>HM136869.1</td> </tr> <tr> <td>34</td> <td>XJ0642</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2006</td> <td>HM136867.1</td> </tr> <tr> <td>35</td> <td>XJ0640</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2006</td> <td>HM136866.1</td> </tr> <tr> <td>36</td> <td>XJ0656</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2006</td> <td>HM136863.1</td> </tr> <tr> <td>37</td> <td>XJ0672</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2006</td> <td>HM136862.1</td> </tr> <tr> <td>38</td> <td>XJ0610</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2006</td> <td>HM136860.1</td> </tr> <tr> <td>39</td> <td>XJ0635</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2006</td> <td>HM136859.1</td> </tr> <tr> <td>40</td> <td>XJ0621</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2006</td> <td>HM136858.1</td> </tr> <tr> <td>41</td> <td>XJ0626</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2006</td> <td>HM136857.1</td> </tr> <tr> <td>42</td> <td>XJ0623</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2006</td> <td>HM136854.1</td> </tr> <tr> <td>43</td> <td>XJ0603</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2006</td> <td>HM136853.1</td> </tr> <tr> <td>44</td> <td>XJ0658</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2006</td> <td>HM136852.1</td> </tr> <tr> <td>45</td> <td>XJ0670</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2006</td> <td>HM136851.1</td> </tr> <tr> <td>46</td> <td>XJ0624</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2006</td> <td>HM136850.1</td> </tr> <tr> <td>47</td> <td>XJ0659</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2006</td> <td>HM136849.1</td> </tr> <tr> <td>48</td> <td>XJ0665</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2006</td> <td>HM136846.1</td> </tr> <tr> <td>49</td> <td>XJ0654</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2006</td> <td>HM136845.1</td> </tr> <tr> <td>50</td> <td>XJ0632</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2006</td> <td>HM136844.1</td> </tr> <tr> <td>51</td> <td>XJ0611</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2006</td> <td>HM136843.1</td> </tr> <tr> <td>52</td> <td>XJ0627</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2006</td> <td>HM136842.1</td> </tr> <tr> <td>53</td> <td>XJ0653</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2006</td> <td>HM136841.1</td> </tr> <tr> <td>54</td> <td>XJ0620</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2006</td> <td>HM136840.1</td> </tr> <tr> <td>55</td> <td>XJ0657</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2006</td> <td>HM136837.1</td> </tr> <tr> <td>56</td> <td>XJ0661</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2006</td> <td>HM136836.1</td> </tr> <tr> <td>57</td> <td>XJ0606</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2006</td> <td>HM136835.1</td> </tr> <tr> <td>58</td> <td>XJ0648</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2006</td> <td>HM136834.1</td> </tr> <tr> <td>59</td> <td>XJ0614</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2006</td> <td>HM136833.1</td> </tr> <tr> <td>60</td> <td>XJ0662</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2006</td> <td>HM136832.1</td> </tr> <tr> <td>61</td> <td>XJ0616</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2006</td> <td>HM136831.1</td> </tr> <tr> <td>62</td> <td>XJ0602</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2006</td> <td>HM136830.1</td> </tr> <tr> <td>63</td> <td>XJ0737</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2007</td> <td>HM136848.1</td> </tr> <tr> <td>64</td> <td>XJ0735</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2007</td> <td>HM136847.1</td> </tr> <tr> <td>65</td> <td>XJ0719</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2007</td> <td>HM136856.1</td> </tr> <tr> <td>66</td> <td>XJ0727</td> <td>China:Xinjiang Province</td> <td><i>Culex spp.</i></td> <td>2007</td> <td>HM136855.1</td> </tr> <tr> <td>67</td> <td>XJ0740</td> <td>China:Xinjiang Province</td> <td><i>Aedes dorsalis</i></td> <td>2007</td> <td>HM136861.1</td> </tr> <tr> <td>68</td> <td>XJ0753</td> <td>China:Xinjiang Province</td> <td><i>Aedes dorsalis</i></td> <td>2007</td> <td>HM136865.1</td> </tr> <tr> <td>69</td> <td>XJ0746</td> <td>China:Xinjiang Province</td> <td><i>Aedes dorsalis</i></td> <td>2007</td> <td>HM136864.1</td> </tr> <tr> <td>70</td> <td>LNV NSW B115745</td> <td>Australia: Sydney</td> <td><i>Ochlerotatus vigilax</i></td> <td>2007</td> <td>MG725857.1</td> </tr> <tr> <td>71</td> <td>LN-5724</td> <td>Australia: New South Wales</td> <td>mosquito</td> <td>2007</td> <td>MG725042.1</td> </tr> <tr> <td>72</td> <td>LN-5798</td> <td>Australia: New South Wales</td> <td>mosquito</td> <td>2007</td> <td>MG725054.1</td> </tr> <tr> <td>73</td> <td>QH07130</td> <td>China:Qinghai Province</td> <td><i>Culex modestus</i></td> <td>2007</td> <td>HM230645.1</td> </tr> <tr> <td>74</td> <td>SX0794</td> <td>China:Shanxi Province</td> <td><i>Aedes vexans</i></td> <td>2007</td> <td>HM243144.1</td> </tr> <tr> <td>75</td> <td>SX0771</td> <td>China:Shanxi Province</td> <td><i>Aedes vexans</i></td> <td>2007</td> <td>HM243143.1</td> </tr> <tr> <td>76</td> <td>XJ0822</td> <td>China:Xinjiang Province</td> <td><i>Aedes dorsalis</i></td> <td>2008</td> <td>HM136870.1</td> </tr> <tr> <td>77</td> <td>XJ0833</td> <td>China:Xinjiang Province</td> <td><i>Aedes dorsalis</i></td> <td>2008</td> <td>HM136868.1</td> </tr> <tr> <td>78</td> <td>XJ0815</td> <td>China:Xinjiang Province</td> <td><i>Aedes dorsalis</i></td> <td>2008</td> <td>HM136839.1</td> </tr> <tr> <td>79</td> <td>LNV NT2013</td> <td>Australia: Bradshaw</td> <td><i>Ochlerotatus vigilax</i></td> <td>2013</td> <td>MG725842.1</td> </tr> <tr> <td>80</td> <td>LNV WA 60042</td> <td>Australia: Peel region</td> <td><i>Ochlerotatus camptorhynchus</i></td> <td>2014</td> <td>MG725869.1</td> </tr> </tbody> </table>  <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <p>The distribution of LNV worldwide. LN: Liaoning province, SX: Shanxi province, QH: Qinghai province, XJ: Xinjiang province, WA: Western Australia, BB: Brisbane, SY: Sydney, NSW: New South Wales, BW: Bradshaw, PR: Peel region</p>   </div> <p /> </div> <div class="timescaledphylogeneticandphylogeographicanalysisoflnv" title="sec"> <div class="title" tagx="title" title="title">
Time scaled phylogenetic and phylogeographic analysis of LNV</div> <p id="Par33">The 10th segment sequence database of LNV was analyzed using Bayesian Markov chain Monte Carlo (MCMC) method [<a href="#CR20">20</a>]. The GTRâ€‰+â€‰Iâ€‰+â€‰G substitution model was selected to be the optimal model by MrModelTest [<a href="#CR21">21</a>]. Bayesian time scaled phylogenetic analysis and nucleotide substitution rate and most recent common ancestor (tMRCA) of LNVs were coestimated using the BEAST software package [<a href="#CR22">22</a>]. The relaxed clock model with different demographic models was tested [<a href="#CR23">23</a>], and the best models were selected by means of a Bayes factor (BF) test using marginal likelihoods values (2lnBFâ€‰&amp;gt;â€‰2) and 95% highest posterior density (HPD) intervals. The chain length was 1,000,000,000 generations. Convergence of parameters was checked using TRACER (<a href="http://beast.community/tracer">http://beast.community/tracer</a>) and was indicated as effective sample size (ESS&amp;gt;200), and the maximum clade credibility (MCC) tree was constructed using TreeAnnotator (<a href="http://beast.community/treeannotator">http://beast.community/treeannotator</a>) with 10% burn-in.</p> <p id="Par34">In order to clarify the geographical dispersal history of LNVs, the bayesian stochastic search variable selection (BSSVS) was used to provide evidence for statistically supported diffusion between state variables under BEAST software package [<a href="#CR24">24</a>]. This method estimates the most probable state at each node in the MCC trees, allowing us to reconstruct ancestral positions for ancestral viral lineages along the tree. For phylogeographic reconstructions, each region was coded as a discrete trait. BSSVS output and surfaces representing uncertainty for continuous diffusion processes were formatted as KML using the SPREAD software [<a href="#CR25">25</a>]. Determination of each locality was coordinated and performed using Google Earth. ArcGis was finally used to display the dispersal pattern of LNV based on the phylogeographic analysis.</p> </div> <div class="thesequenceanalysisofthe10thsegmentoflnv" title="sec"> <div class="title" tagx="title" title="title">
The sequence analysis of the 10th segment of LNV</div> <p id="Par35">Sequence similarity and nucleotide base composition analysis were performed using the CLUSTALX software [<a href="#CR26">26</a>], BioEdit software [<a href="#CR27">27</a>] MEGA-X [<a href="#CR28">28</a>] and MegAlign (DNASTAR, Madison, WI, USA). Hemi (version1.0) [<a href="#CR29">29</a>] software visualized the similarity comparison results of LNV and Rstudio made the nucleotide base composition analysis map of LNV.</p> </div> <div class="three-dimensionalstructuralanalysisofvp10oflnvs" title="sec"> <div class="title" tagx="title" title="title">
Three-dimensional structural analysis of VP10 of LNVs</div> <p id="Par36">The crystal structure of the cell attachment protein VP9 of BAV (PDB 1w9z) was selected as the best template for the homology modeling. The YASARA software [<a href="#CR30">30</a>], PyMol software [<a href="#CR31">31</a>] and VMD (version 1.9.2) [<a href="#CR32">32</a>] were used to analyze the structures and surface charge density of VP10 among LNV isolates.</p> </div> </div> <div class="result" title="sec"> <div class="title" tagx="title" title="title">
Result</div> <div class="theevolutionoflnvisolatedworldwidebasedonthetime-scaledphylogeneticanalysis" title="sec"> <div class="title" tagx="title" title="title">
The evolution of LNV isolated worldwide based on the time-scaled phylogenetic analysis</div> <p id="Par37">According to the 95% HPD intervals and bayes factor, the Bayesian skyline model with a relaxed molecular clock was selected to be the best fit model. The MCC tree was established using the Bayesian Markov chain Monte Carlo (MCMC) approach (Fig.Â <a href="#Fig2">2</a>). The posterior probability values of each branch node were all greater than 0.7, showing the robustness of the result. The mean nucleotide substitution rate for the entire LNV population was estimated to be 1.6â€‰Ã—â€‰10<sup>âˆ’â€‰3</sup>â€‰s/s/y (95%HPD:2.8â€‰Ã—â€‰10<sup>âˆ’â€‰4</sup>,3.3â€‰Ã—â€‰10<sup>âˆ’â€‰3</sup>) (TableÂ <a href="#Tab2">2</a>). At this evolution rate, the tMRCA of the LNV was calculated to be 272.6â€‰years ago (95%HPD:58.9,790.3). The LNV evolved chronologically into three major evolutionary populations, genotype 1, genotype 2 and genotype 3.Genotype1, which included the initial isolate (LNV-NE97â€"31) from Liaoning province of China and seven strains from Australian, emerged 73.0â€‰years ago (95%HPD:28.9,193.0) and demonstrated to be the oldest lineage. Genotype 2 emerged about 46.5â€‰years ago (95%HPD:18.8,113.5) and was composed of LNV strains isolated from Xinjiang province in western China. Genotype 3, which appeared approximately 25.3â€‰years ago (95%HPD:17.4,46.6), included isolates from Qinghai province, Shanxi province and Liaoning province (another initial isolate LNV-NE97â€"12), and was the youngest LNV lineage. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <p>Evolution analysis of LNV isolated worldwide from 1990 to 2014. Maximum clade credibility tree for the 10th segment of LNVs isolated from 1990 to 2014 worldwide. The tree identified 3 distinct lineages, genotype 1, genotype 2 and genotype 3.Genotype 1 contained the initial LNV isolate (LNSV-NE97â€"31) and seven Australian isolates (Blue). Genotype 2 consisted of isolates all from Xingjiang province in western China (Green). Genotype 3 comprised LNV isolates from Qinghai province, Shanxi province and Liaoning province (another initial LNV isolate LNSV-NE97â€"12) (Red). Estimated tMRCAs of these lineages (with their 95% HPD values in parentheses) are presented. The posterior probability values of main branch nodes were indicated. The letters G1, G2 and G3 represent genotypes 1,2 and 3, respectively</p>   </div> <div class="table-wrap_UNKNOWN" id="Tab2"><span class="label" tagx="label" title="label">Table 2</span> <p>Sequence similarity and Bayesian Markov chain Monte Carlo analysis of LNVs</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th rowspan="2">Genotypes</th> <th colspan="2">Nucleotide similarity</th> <th colspan="2">Amino acid similarity</th> <th colspan="2">tMRCA y (95% HPD)</th> <th>Substitution rate s/s/y (95% HPD)</th> </tr> <tr> <th>Over all</th> <th>Each group</th> <th>Over all</th> <th>Each group</th> <th>Over all</th> <th>Each group</th> <th>Over all</th> </tr> </thead> <tbody> <tr> <td>1</td> <td rowspan="3">70.4%â€‰~â€‰100% (Î¼â€‰=â€‰92.6%)</td> <td>87.5%â€‰~â€‰100% (Î¼â€‰=â€‰94.3%)</td> <td rowspan="3">75.0%â€‰~â€‰100% (Î¼â€‰=â€‰94.6%)</td> <td>91.2%â€‰~â€‰100% (Î¼â€‰=â€‰96.4%)</td> <td char="(" align="char" rowspan="3">272.6 (58.9,790.3)</td> <td>73.0 (28.9,193.0)</td> <td rowspan="3">1.6â€‰Ã—â€‰10<sup>âˆ’3</sup> (2.8â€‰Ã—â€‰10<sup>âˆ’4</sup>,3.3â€‰Ã—â€‰10<sup>âˆ’3</sup>)</td> </tr> <tr> <td>2</td> <td>92.5%â€‰~â€‰100% (Î¼â€‰=â€‰97.8%)</td> <td>92.5%â€‰~â€‰100% (Î¼â€‰=â€‰98.9%)</td> <td>46.5 (18.8,113.5)</td> </tr> <tr> <td>3</td> <td>97.5%â€‰~â€‰100% (Î¼â€‰=â€‰99.2%)</td> <td>98.1%â€‰~â€‰100% (Î¼â€‰=â€‰99.4%)</td> <td>25.3 (17.4,46.6)</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>Note: <i>Î¼</i> average nucleotide (amino acid) similarity, <i>y</i> years; <i>s/s/y</i> Substitutions per site per year</p> </div> </div> <p /> </div> <div class="populationdynamicsoflnv" title="sec"> <div class="title" tagx="title" title="title">
Population dynamics of LNV</div> <p id="Par38">The skyline plot of the LNV population dynamics is shown in Fig.Â <a href="#Fig3">3</a>. The population dynamic of LNV kept stable from 1990 to 1997.Since 1997, the population diversity began to decrease and reached the lowest point at the year of 2003.From 2003 the LNV population diversity increased, peaking around 2005.After a minor decrease during around 2005 to 2006, the virus population was stable from 2006 to 2014. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3</span></a></div>   <p>Bayesian skyline plot for LNV. Highlighted areas correspond to 95% HPD intervals</p>   </div> <p /> </div> <div class="thesequencesimilarityanalysisoflnv" title="sec"> <div class="title" tagx="title" title="title">
The sequence similarity analysis of LNV</div> <p id="Par39">The Sequence analyses of 80 strains of LNV isolated from different vectors and locations during 1990â€"2014 revealed that the level of nucleotide similarity varied from 70.4 to 100% with an average value of Î¼â€‰=â€‰92.6% while the amino acid similarity ranged from 75.0 to 100% with an average value Î¼â€‰=â€‰94.6% (Fig.Â <a href="#Fig4">4</a>). Genotype 1 exhibited nucleotide similarity between 87.5â€"100% (Î¼â€‰=â€‰94.3%) and amino acid similarity range from 91.2 to 100% (Î¼â€‰=â€‰96.4%).Genotype 2 exhibited nucleotide similarity from 92.5 to 100% (Î¼â€‰=â€‰97.8%) and amino acid similarity from 92.5 to 100% (Î¼â€‰=â€‰98.9%). Genotype3 exhibited nucleotide similarity from 97.5 to 100% (Î¼â€‰=â€‰99.2%) and amino acid similarity from 98.1 toÂ 100% (Î¼â€‰=â€‰99.4%). The results demonstrated that the genotype 1 population possessed the highest degree of nucleotide and amino acid variation among all the three LNV genotypes. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 4</span></a></div>   <p>The Nucleotide and amino acid similarity of the 10th segment of LNV. <b>a</b>: Nucleotide similarity of LNV. <b>b</b>: Amino acid similarity of LNV. The horizontal and vertical axes represent the names of different genotypes of LNV isolates. The names of the isolates colored blue, green and red represent the genotypes 1,2 and 3, respectively</p>   </div> <p /> </div> <div class="nucleotidecompositionofthe10thsegmentgenomeoflnv" title="sec"> <div class="title" tagx="title" title="title">
Nucleotide composition of the 10th segment genome of LNV</div> <p id="Par40">The values of nucleotide contents in the entire LNV population and each genotype were analyzed (Fig.Â <a href="#Fig5">5</a>). The results showed that VP10 genome of LNV was rich in A and U (Aâ€‰+â€‰Uâ€‰&amp;gt;â€‰Câ€‰+â€‰G) and the percentage of AU was reaching 59.5%. The A%, U%, G%, C% are 26.0% Â±0.6 (meanâ€‰Â±â€‰SD), 32.3%â€‰Â±â€‰1.3, 18.7%â€‰Â±â€‰0.4, 23.1%â€‰Â±â€‰0.4, respectively. There was a common trend in the usage of A,U,G,C between the three genotypes that was Uâ€‰&amp;gt;â€‰Aâ€‰&amp;gt;â€‰0.25â€‰&amp;gt;â€‰Câ€‰&amp;gt;â€‰G (Fig. <a href="#Fig5">5</a>). As for each genotype, the average contents of A,U,G,C for genotype 1 were 27.6%â€‰Â±â€‰0.5, 29.0%â€‰Â±â€‰0.5, 19.8%â€‰Â±â€‰0.2, 23.5%â€‰Â±â€‰0.3,for genotypes 2 were 25.8%â€‰Â±â€‰0.3, 32.8%â€‰Â±â€‰0.5,18.5%â€‰Â±â€‰0.2, 23.0%â€‰Â±â€‰0.4 and these values for genotypesÂ 3 were 26.3%â€‰Â±â€‰0.2, 30.7%â€‰Â±â€‰0.3, 19.4%â€‰Â±â€‰0.1, 23.7%â€‰Â±â€‰0.2. The value of the A content in genotypeÂ 1 was the highest while that in genotype 2 was the lowest. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 5</span></a></div>   <p>The A, U, G and C contents of different genotypes of the 10th segment of LNV. The vertical axes indicates the percentage of nucleotide contents. The cross bar represents the different isolates in the same genotype. The red bar identifies adenine (A), green identifies cytosine (C), cyan identifies guanine (G) and violet identifies uracil (U)</p>   </div> <p /> </div> <div class="thedispersalrouteoflnvbaseonphylogeographicanalysis" title="sec"> <div class="title" tagx="title" title="title">
The dispersal route of LNV base on phylogeographic analysis</div> <p id="Par41">The estimated history of LNV dispersal route over time is shown in detail in Fig.Â <a href="#Fig6">6</a>. According to our results of the phylogeographic analysis, LNV was originated in Australia in the south pacific region and then initially introduced to Liaoning province in China in the Northeast Asia around 1980s.Subsequently, the virus was further westwarded spread to Shanxi province, Qinghai province and Xingjiang province of China during 1990s. </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 6</span></a></div>   <p>Dispersal route of LNVs based on phylogeographic analysis. Gray shadow presents the areas with LNV isolated. Line and arrow represents the transmission route and direction of LNV. Red line: Spread from Australia to Liaoning province of China in 1980s; Green line: Spread from Liaoning province to Shanxi province, Qinghai province and Xinjiang province in 1990s. LN: Liaoning province, SX: Shanxi province, QH: Qinghai province, XJ: Xinjiang province, AU: Australia</p>   </div> <p /> </div> <div class="theaminoacidcomparisonandthethreedimensionalstructuralanalysisofthecellattachmentprotein(vp10)oflnv" title="sec"> <div class="title" tagx="title" title="title">
The amino acid comparison and the three dimensional structural analysis of the cell attachment protein (VP10) of LNV</div> <p id="Par42">To characterize the mutations in the cell attachment protein (VP10) between the three genotypes of LNV, we analyzed the amino acids encoded by the VP10 genes derived from the previously mentioned LNV isolates. The results revealed that the ORF of VP10 was 753 nucleotides in length and encoded 250 amino acids. The amino acid sequences of genotype 1 were significantly different from those of the rest two genotypes while the genotype 2 and 3 shared a great similarity in amino acid sequences. The detailed results were that a total of 20 common amino acid differences identified between genotype 1 to genotype 2 and 3 (TableÂ <a href="#Tab3">3</a>). The amino acid sequences were highly similar between genotype2 and 3 and there were only 3 amino acid differences were identified (Table <a href="#Tab3">3</a>). When compared the amino acid sequences of genotype 1 to those of genotype 2 and 3, twenty common amino acid differences were identified as follows: S44A, A60G, D64G, S65N, H68V, N82A, P99A, V122T, M131S, A132G, S160N, I167S, Q171S, A181L, E207D, S209V, T217N, S219N, N223H, N238S.The amino acid sequences of genotype 2 and 3 were highly similar, there were only three amino acid mutations: S58A, A130N, L210H. <div class="table-wrap_UNKNOWN" id="Tab3"><span class="label" tagx="label" title="label">Table 3</span> </div></p><p>Comparison of VP10 amino acid sequences between different LNV genotypes</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th rowspan="2">Genotypes</th> <th colspan="23">Positions of amino acid</th> </tr> <tr> <th>44</th> <th>58</th> <th>60</th> <th>64</th> <th>65</th> <th>68</th> <th>82</th> <th>99</th> <th>122</th> <th>130</th> <th>131</th> <th>132</th> <th>160</th> <th>167</th> <th>171</th> <th>181</th> <th>207</th> <th>209</th> <th>210</th> <th>217</th> <th>219</th> <th>223</th> <th>238</th> </tr> </thead> <tbody> <tr> <td>1</td> <td>S</td> <td>A</td> <td>A</td> <td>D</td> <td>S</td> <td>H</td> <td>N</td> <td>P</td> <td>V</td> <td>N</td> <td>M</td> <td>A</td> <td>S</td> <td>I</td> <td>Q</td> <td>A</td> <td>E</td> <td>S</td> <td>Q</td> <td>T</td> <td>S</td> <td>N</td> <td>N</td> </tr> <tr> <td>2</td> <td>A</td> <td>S</td> <td>G</td> <td>G</td> <td>N</td> <td>V</td> <td>A</td> <td>A</td> <td>T</td> <td>A</td> <td>S</td> <td>G</td> <td>N</td> <td>S</td> <td>S</td> <td>L</td> <td>D</td> <td>V</td> <td>L</td> <td>N</td> <td>N</td> <td>H</td> <td>S</td> </tr> <tr> <td>3</td> <td>A</td> <td>A</td> <td>G</td> <td>G</td> <td>N</td> <td>V</td> <td>A</td> <td>A</td> <td>T</td> <td>N</td> <td>S</td> <td>G</td> <td>N</td> <td>S</td> <td>S</td> <td>L</td> <td>D</td> <td>V</td> <td>H</td> <td>N</td> <td>N</td> <td>H</td> <td>S</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p>Note: Only the different amino acids among three genotypes of LNV are shown in the table</p> </div>  <p /> <p id="Par43">In order to further explore whether these mutations affected the three shape and the surface charge density of the cell attachment protein of LNV, the three dimensional structures and electrostatic potential analysis have been conducted between different genotypes of LNV. Seven of the twenty common amino acid differences were located in the Î²-sheet (residues 82, 122, 131, 132, 181, 223, and 238), while the remaining thirteen were located in the loop region (residues 44, 60, 64, 65, 68, 99, 160, 167, 171, 207, 209, 217 and 219). These differences have some effect on the structure and electrostatic properties of VP10 between the different genotypes. In particular, the residue at position 64, 65, 131, 132, 167, 171, 207, 209 and 219.There were great differences existed the R group of the amino acid and the electrical polarity of the mentioned 9 mutations, resulting in significant differences in structure and charge at these sites between the LNVs (Fig.Â <a href="#Fig7">7</a>aâ€™, bâ€™, câ€™). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 7</span></a></div>   <p>The theoretical three-dimensional structure and surface charge of different genotypes of LNV. <b>a</b>, <b>b</b>, <b>c</b>: The theoretical three-dimensional structure of LNV of genotype1,2,3 and the differences sites of amino acid within 3 genotypes. <b>a</b>â€™, <b>b</b>â€™, <b>c</b>â€™: The theoretical three-dimensional structure as well as the surface charge distribution of LNV of genotype 1,2,3. def and dâ€™eâ€™fâ€™ are the angles of view of abc and aâ€™bâ€™câ€™ rotated the Y axis by 180 degrees, respectively</p>   </div> <p /> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p id="Par44">During the arboviruses survey in China in 1997, two virus strains (LNSV-NE9731 and LNV-NE9712) were obtained from <i>Aedes dorsalis</i> mosquitoes in Liaoning province in North-East of China which were found to cause cytopathic effect in C6/36 cells. The virus was designated <i>Liao ning virus</i> (LNV) [<a href="#CR3">3</a>, <a href="#CR10">10</a>]. Since then, additional strains of LNV were isolated from Shanxi province, Qinhai province and Xinjiang province [<a href="#CR7">7</a>, <a href="#CR8">8</a>]. A more than 30 years national arbovirus surveillance in mainland China revealed the LNV strains had only been isolated in a long and narrow region covering 73â€"125 Â° E and 31â€"48 Â° N in the northwest to northeast part of the country. No strains of LNV have been reported isolated in other regions of China or abroad [<a href="#CR6">6</a>]. However, a recent research reported a total of 35 strains of LNV were isolated from mosquitoes belonging to four genera (<i>Culex</i>, <i>Anopheles, Mansonia and Aedes</i>) collected from 1988 to 2014 in Australia in the southern hemisphere. The isolation sites included New South Wales, Northern Territory, Queensland and Western Australia, almost across the entire Australian continent. Thus, it is obvious that the geographical distribution and the vectors of LNV are wide and variable in Australia. What is more, the initial LNV isolate in Australian was obtained from mosquitoes collected in 1988, predating the first Chinese LNV isolate which was obtained in 1997 [<a href="#CR10">10</a>, <a href="#CR11">11</a>].</p> <p id="Par45">Several phylogenetic analysis have been conducted on the LNVâ€™s nucleotide sequences, previously. The LNVs isolated in China could be divided into 3 evolutionary branches. The LNSV-NE9731 strain isolated in Liaoning province in 1997 formed an independent evolutionary branch while another Liaoning isolates named LNSV-NE9712 clustered together with the isolates from Qinghai province and Shanxi province that formed a branch. All of the Xinjiang isolates grouped together formed the largest evolutionary branch [<a href="#CR5">5</a>, <a href="#CR33">33</a>]. All of the Australian LNV isolates were divided into two disparate lineage, one composed the isolates from eastern and northern Australia and the other included the isolates from western and southern Australia [<a href="#CR11">11</a>, <a href="#CR12">12</a>]. In this study, a comprehensive molecular phylogenetic analysis of all the LNVs isolated from China and Austrila demonstrated that the LNVs could be divide into three genotypes (Fig. <a href="#Fig2">2</a>). Genotype 1 included LNSV-NE9731 (isolated in Liaoning provinve of China in 1997) and 7 strains from Austrila (the mosquito collection time was 1990,2005,2007,2013 and 2014, respectively). A total of 68 strains of LNV isolated from Xinjiang province of western China clustered together that formed an independent branch. And the strain LNV-NE9712 isolated in 1997 together with the Qinghai and Shanxi isolates formed the genotype 3 evolutionary branch. This result suggested that the Australia LNV isolates were not only restricted and circulated in Australia and even this virus population transmitted a long way to northern China in Asia then evolved into new LNV populations with local circulation characteristics.</p> <p id="Par46">The time-scaled evolutionary analysis of LNV revealed the branching of the lineages occurred in the following order:genotype 1 emerged 73â€‰years ago (95%HPD:28.9, 193.0), the genotypeÂ 2 at 46.5â€‰years ago (95%HPD:18.8, 113.5) and genotypeÂ 3 at 25.3â€‰years ago (95%HPD:17.4, 46.6.) Thus, genotype 1 is the oldest lineage and the genotype 3 is the youngest one (Table <a href="#Tab2">2</a>). The current results lead to an estimate that the most recent common ancestor of LNV appeared about 277â€‰years ago which similar to that of another member of <i>seadornaviruses</i> BAV, whose tMRCA is 105â€‰years ago [<a href="#CR34">34</a>], indicating that LNV is an emerging virus population.</p> <p id="Par47">Although the LNVs were initially isolated from several inland provinces in China, the phylogeopgraphic analysis of our study showed that the LNV likely originated from Australia in the South Pacific region and the genotype 1 viruses spread northward from Australia to Liaoning province in northeast China. The reasons that LNV originated in Australia and spread to the Asian continent may be related to the following factors: First, Australia is located in the South Pacific region and contains a variety of geographic climate types, including the subtropical humid climate in the east, the savanna climate in the northwest, the tropical rain forest climate in the northeast, and Mediterranean climate in the southwest [<a href="#CR35">35</a>]. The climate types in Australia facilitate the local host vectors diverse and can breed LNV population with a strong adaptability. Second, the nucleotide similarity of genotype 1 was the most divergent between all of the three genotypes, indicating greater population variability exists in Australian LNVs.</p> <p id="Par48">Additionally, there was a very interesting finding that the AU content was relative high in VP10 of LNV, whereas the AU content was also high within the LNVâ€™s whole viral genome (data not shown). These results were consistent with the prior studies wherein A and U frequencies were higher than C and G frequencies for avian <i>rotaviruses</i> and some <i>flaviviruses</i> including <i>dengue virus</i> (DENV), <i>West Nile virus</i> (WNV), y<i>ellow fever virus</i> (YFV) and JEV [<a href="#CR36">36</a>â€"<a href="#CR39">39</a>]. Besides, the nucleotide composition of the BAV, which is the prototype species of genus <i>Seadornavirus</i>, was also observed a higher AU content [<a href="#CR40">40</a>]. It has been reported that the AU-rich genome structure facilitate the <i>Human Immunodeficiency virus</i> (HIV) to avoid recognition by the innate immune system of host cell [<a href="#CR41">41</a>]. However, the biological causes and the consequence for increased A and U within the LNV and other mentioned arboviruses genome are still unknown, so enhanced experimental studies are required to explore the AU rich molecular function in LNV.</p> <p id="Par49">This may be the reasons why LNVs have been continuously isolated from 12 species of mosquitoes belonging to four genera in different geographical and climatic regions in Australia, during the last 30 years [<a href="#CR11">11</a>, <a href="#CR12">12</a>]. However, in China the LNV has only been isolated from a long and narrow region restricted from 31Â°N to 48Â°N, where belongs to the north temperate zone with cold climate, less rain and low species diversity [<a href="#CR5">5</a>, <a href="#CR7">7</a>, <a href="#CR10">10</a>]. What is more, after transmission from Australia to China, the genotype 1 LNV population gradually adapted to the local natural environment and evolved new strains with regional genetic characteristics. The newly evolved LNVs (genotypes 2 and 3) contain 20 amino acid mutations compared with the initial LNVs (genotype 1) in the cell attachment protein (VP10), which altered the structure and electrostatic presentation influencing the binding properties to host vectors. This might be one of the reasons why the newly evolved LNVs (genotyp 2 and 3) were restricted to a relative narrow range of vectors and habitat. Compared with genotype 1 and genotype 2, the genotype 3 which located in the central part of China contains 2 unique amino acids, which were identical with genotype 1 but different from genotype 2,indicating that this lineage is at evolutionary transitional position which preserved the genetic information of the original LNV population and also evolved novel genetic information sites during spread to new locations. When compared with original LNV population, the genotype 2 LNVs in Xinjiang province contained the the maximum numbers of amino acid mutations thus it was the the most divergent lineage and formed a independed evolutionary branch. The genetic informative sites of the entire LNVs population confirmed its transmission path from north to south, and then from east to west.</p> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p id="Par50">In this study, the evolution analysis of LNVs revealed that the virus belongs to the emerging virus group. In particular, it was suggested that the genotype 1 LNVs were a group of segmented double-strand RNA virus with extremely adaptability, which have a large group of vectors, a high rate of genetic variation and apparent active transmissibility that can adapt to different geographical environments over a quite long distance. Recently, the genome information of LNV was reported to be detected in <i>Aedes aegypti</i> of African origin, reminding us that LNVs was not only limited to China in Asia and Australia in the South Pacific region but may well be extended to Africa and even posing a high risk of spreading to new areas such as Central Asia and Europe. It is well known that genetic mutation such as recombination or reassortment can easily occur in segmented RNA virus. For example, BAV which also belongs to the <i>Seadornavirus</i>, was originally discovered in southern China. However, several novel variants of BAV have been found, such as the BALV isolate from Hungary [<a href="#CR33">33</a>] and the <i>Mangshi virus</i> from southern China [<a href="#CR42">42</a>]. Considering LNV was the only specie in <i>Seadornavirus</i> that can replicate in mammalian cell lines and cause fatal haemorrhagic symptoms in mice [<a href="#CR3">3</a>], it might be a pathogen that has great potential to cause disease in human and (or) animals. Therefore, to strengthen the research on genetic variation of LNV and to clarify the relationship between LNV and zoonosis is not only a research topic for virologists, but also a scientific issues for public health communities.</p> </div> </div> <div class="back" title="back"> <div class="glossary" title="glossary"> <div class="title" tagx="title" title="title">
Abbreviations</div> <div class="def-list" title="def-list"> <div class="def-item" title="def-item"> <div class="term" title="term">
LNV</div> <div class="def" title="def"> <p id="Par5"><i>Liao ning virus</i></p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
LNVs</div> <div class="def" title="def"> <p id="Par6"><i>Liao ning viruses</i></p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
BAV</div> <div class="def" title="def"> <p id="Par7"><i>Banna virus</i></p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
KDV</div> <div class="def" title="def"> <p id="Par8"><i>Kadipiro virus</i></p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
RNA</div> <div class="def" title="def"> <p id="Par9">RibonucleicAcid</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
bp</div> <div class="def" title="def"> <p id="Par10">basepair</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
VP</div> <div class="def" title="def"> <p id="Par11">Viral protein</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
VP10</div> <div class="def" title="def"> <p id="Par12">The tenth segnment of viral protein</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
MCMC</div> <div class="def" title="def"> <p id="Par13">The Bayesian Markov chain Monte Carlo</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
tMRCA</div> <div class="def" title="def"> <p id="Par14">The time of most recent ancestor</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
BEAST</div> <div class="def" title="def"> <p id="Par15">Bayesian Evolutionary Analysis Sampling Trees</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
BF</div> <div class="def" title="def"> <p id="Par16">Bayes factor</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
HPD</div> <div class="def" title="def"> <p id="Par17">Highest posterior density</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
MCC</div> <div class="def" title="def"> <p id="Par18">The maximum clade credibility</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
ESS</div> <div class="def" title="def"> <p id="Par19">The effective sample size</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
BSSVS</div> <div class="def" title="def"> <p id="Par20">The bayesian stochastic search variable selection</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
KML</div> <div class="def" title="def"> <p id="Par21">Keyhole Markup Language</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SPREAD</div> <div class="def" title="def"> <p id="Par22">Spatial phylogenetic reconstruction of evolutionary dynamics</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
SD</div> <div class="def" title="def"> <p id="Par23">Standard Deviation</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
A,U,G,C</div> <div class="def" title="def"> <p id="Par24">Adenine,Uracil,Guanine,Cytosine</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
ORF</div> <div class="def" title="def"> <p id="Par25">Open reading frame</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
BALV</div> <div class="def" title="def"> <p id="Par26"><i>Banna-like virus</i></p> </div> </div> </div> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p><b>Publisherâ€™s Note</b></p> <p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> <div class="fn-type-" title=""> <p>Jun Zhang, Hong Liu and Jiahui Wang contributed equally to this work.</p> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>Not applicable.</p> </div> <div class="title" tagx="title" title="title">
Authorsâ€™ contributions</div> <p>GDL and HL conceived and designed the study and wrote the paper. GDL, HL and JZ revised the manuscript. JZ Contributed materials/analysis tools, performed the study, analyzed the data. JZ, JHW, JhW, JMZ, JYW and XZ performed the study. ZFD, HX, HFJ, CYZ, QZ and HL analysis and revised data. All authors read and approved the final manuscript.</p> <div class="title" tagx="title" title="title">
Funding</div> <p>This work was supported by grants from the National Natural Science Foundation of China (81301479), a Development Grant of the State Key Laboratory of Infectious Disease Prevention and Control (2014SKLID103,2018SKLID309), the Natural Science Foundation of Shandong Province, China (ZR2013HQ008, ZR2019MH020) and the Health Commission of Hubei Province [WJ2019Q060] the young scientist supportive program of Shandong University of Technology. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p> <div class="title" tagx="title" title="title">
Availability of data and materials</div> <p>All data and materials are included in this published article.</p> <div class="title" tagx="title" title="title">
Ethics approval and consent to participate</div> <p id="Par51">Not applicable.</p> <div class="title" tagx="title" title="title">
Consent for publication</div> <p id="Par52">Not applicable.</p> <div class="title" tagx="title" title="title">
Competing interests</div> <p id="Par53">The authors declare that there are no conflicts of interest.</p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Attoui</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Billoir</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Biagini</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Micco</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Lamballerie</span><span class="given-names" tagx="given-names" title="given-names">X</span></span></span><span class="mixed-article-title" title="mixed-article-title">Complete sequence determination and genetic analysis of Banna virus and Kadipiro virus: proposal for assignment to a new genus (Seadornavirus) within the family Reoviridae</span><span class="source" tagx="source" title="source">J Gen Virol</span><span class="year" tagx="year" title="year">2000</span><span class="volume" tagx="volume" title="volume">81</span><span class="fpage" tagx="fpage" title="fpage">1507</span><span class="lpage" tagx="lpage" title="lpage">1515</span><span class="pub-id"><a href="https://dx.doi.org/10.1099/0022-1317-81-6-1507">10.1099/0022-1317-81-6-1507</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10811934">10811934</a></span></span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Attoui</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mohd Jaafar</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Micco</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Lamballerie</span><span class="given-names" tagx="given-names" title="given-names">X</span></span></span><span class="mixed-article-title" title="mixed-article-title">Coltiviruses and seadornaviruses in North America, Europe, and Asia</span><span class="source" tagx="source" title="source">Emerg Infect Dis</span><span class="year" tagx="year" title="year">2005</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">1673</span><span class="lpage" tagx="lpage" title="lpage">1679</span><span class="pub-id"><a href="https://dx.doi.org/10.3201/eid1111.050868">10.3201/eid1111.050868</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16318717">16318717</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Attoui</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mohd Jaafar</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belhouchet</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tao</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Liao ning virus, a new Chinese seadornavirus that replicates in transformed and embryonic mammalian cells</span><span class="source" tagx="source" title="source">J Gen Virol</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">87</span><span class="fpage" tagx="fpage" title="fpage">199</span><span class="lpage" tagx="lpage" title="lpage">208</span><span class="pub-id"><a href="https://dx.doi.org/10.1099/vir.0.81294-0">10.1099/vir.0.81294-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16361432">16361432</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">MH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhai</span><span class="given-names" tagx="given-names" title="given-names">YG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meng</span><span class="given-names" tagx="given-names" title="given-names">WS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">XH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">YX</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Banna virus, China, 1987-2007</span><span class="source" tagx="source" title="source">Emerg Infect Dis</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">514</span><span class="lpage" tagx="lpage" title="lpage">517</span><span class="pub-id"><a href="https://dx.doi.org/10.3201/eid1603.091160">10.3201/eid1603.091160</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20202434">20202434</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dong</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Liao ning virus in China</span><span class="source" tagx="source" title="source">Virol J</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">282</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1743-422X-8-282">10.1186/1743-422X-8-282</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21649929">21649929</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Arboviruses and their related infections in China: a comprehensive field and laboratory investigation over the last 3 decades</span><span class="source" tagx="source" title="source">Rev Med Virol</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">28</span><span class="fpage" tagx="fpage" title="fpage">e1959</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/rmv.1959">10.1002/rmv.1959</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lv</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mohd Jaafar</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Belhouchet</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Isolates of Liao ning virus from wild-caught mosquitoes in the Xinjiang province of China in 2005</span><span class="source" tagx="source" title="source">PLoS One</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">e37732</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0037732">10.1371/journal.pone.0037732</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22649554">22649554</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">WJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">MH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lv</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">YX</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">ZL</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Isolation and identification of Liao ning virus from Culex pipiens mosquitoes in Qinghai Province</span><span class="source" tagx="source" title="source">Chinese J Vector Biology and Control</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">12</span><span class="lpage" tagx="lpage" title="lpage">14</span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sun</span><span class="given-names" tagx="given-names" title="given-names">XH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meng</span><span class="given-names" tagx="given-names" title="given-names">WS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">SH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Feng</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhai</span><span class="given-names" tagx="given-names" title="given-names">YG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The first report of Kadipiro virus isolation in China</span><span class="source" tagx="source" title="source">Bing Du Xue Bao</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">173</span><span class="lpage" tagx="lpage" title="lpage">177</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19634758">19634758</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tao</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cai</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span></span><span class="mixed-article-title" title="mixed-article-title">Yang D: [new subtype of coltivirus isolated from mosquitoes in the northeast part of China]</span><span class="source" tagx="source" title="source">Zhonghua Shi Yan He Lin Chuang Bing Du Xue Za Zhi</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">228</span><span class="lpage" tagx="lpage" title="lpage">230</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12569751">12569751</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Prow</span><span class="given-names" tagx="given-names" title="given-names">NA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mah</span><span class="given-names" tagx="given-names" title="given-names">MG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deerain</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Warrilow</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colmant</span><span class="given-names" tagx="given-names" title="given-names">AMG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Brien</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">New genotypes of Liao ning virus (LNV) in Australia exhibit an insect-specific phenotype</span><span class="source" tagx="source" title="source">J Gen Virol</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">99</span><span class="fpage" tagx="fpage" title="fpage">596</span><span class="lpage" tagx="lpage" title="lpage">609</span><span class="pub-id"><a href="https://dx.doi.org/10.1099/jgv.0.001038">10.1099/jgv.0.001038</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29533743">29533743</a></span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Coffey</span><span class="given-names" tagx="given-names" title="given-names">LL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Page</span><span class="given-names" tagx="given-names" title="given-names">BL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greninger</span><span class="given-names" tagx="given-names" title="given-names">AL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Herring</span><span class="given-names" tagx="given-names" title="given-names">BL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Russell</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doggett</span><span class="given-names" tagx="given-names" title="given-names">SL</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Enhanced arbovirus surveillance with deep sequencing: identification of novel rhabdoviruses and bunyaviruses in Australian mosquitoes</span><span class="source" tagx="source" title="source">Virology</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">448</span><span class="fpage" tagx="fpage" title="fpage">146</span><span class="lpage" tagx="lpage" title="lpage">158</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virol.2013.09.026">10.1016/j.virol.2013.09.026</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24314645">24314645</a></span></span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ibrahim</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McMahon</span><span class="given-names" tagx="given-names" title="given-names">DP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hufsky</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beer</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deng</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mercier</span><span class="given-names" tagx="given-names" title="given-names">PL</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A new era of virus bioinformatics</span><span class="source" tagx="source" title="source">Virus Res</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">251</span><span class="fpage" tagx="fpage" title="fpage">86</span><span class="lpage" tagx="lpage" title="lpage">90</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virusres.2018.05.009">10.1016/j.virusres.2018.05.009</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29751021">29751021</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Geoghegan</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holmes</span><span class="given-names" tagx="given-names" title="given-names">EC</span></span></span><span class="mixed-article-title" title="mixed-article-title">Evolutionary virology at 40</span><span class="source" tagx="source" title="source">Genetics</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">210</span><span class="fpage" tagx="fpage" title="fpage">1151</span><span class="lpage" tagx="lpage" title="lpage">1162</span><span class="pub-id"><a href="https://dx.doi.org/10.1534/genetics.118.301556">10.1534/genetics.118.301556</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30523166">30523166</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wolf</span><span class="given-names" tagx="given-names" title="given-names">YI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kazlauskas</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iranzo</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lucia-Sanz</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kuhn</span><span class="given-names" tagx="given-names" title="given-names">JH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Krupovic</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Origins and evolution of the global RNA Virome</span><span class="source" tagx="source" title="source">mBio</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/mBio.02329-18">10.1128/mBio.02329-18</a></span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pan</span><span class="given-names" tagx="given-names" title="given-names">XL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">HY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">SH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">HZ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">HL</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Emergence of genotype I of Japanese encephalitis virus as the dominant genotype in Asia</span><span class="source" tagx="source" title="source">J Virol</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">85</span><span class="fpage" tagx="fpage" title="fpage">9847</span><span class="lpage" tagx="lpage" title="lpage">9853</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.00825-11">10.1128/JVI.00825-11</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21697481">21697481</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="mixed-article-title" title="mixed-article-title">Insights into the evolutionary history of Japanese encephalitis virus (JEV) based on whole-genome sequences comprising the five genotypes</span><span class="source" tagx="source" title="source">Virol J</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">43</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12985-015-0270-z">10.1186/s12985-015-0270-z</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25884184">25884184</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Liu H, Shen L, Zhang XL, Li XL, Liang GD, Ji HF. From discovery to outbreak: the genetic evolution of the emerging Zika virus. Emerg Microbes Infect. 2016;5:e111.</span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Dudas G, Carvalho LM, Bedford T, Tatem AJ, Baele G, Faria NR, Park DJ, Ladner JT, Arias A, Asogun D, et al. Virus genomes reveal factors that spread and sustained the Ebola epidemic. Nature. 2017;544:309â€"15.</span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Ravenzwaaij</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cassey</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">SD</span></span></span><span class="mixed-article-title" title="mixed-article-title">A simple introduction to Markov chain Monte-Carlo sampling</span><span class="source" tagx="source" title="source">Psychon Bull Rev</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">143</span><span class="lpage" tagx="lpage" title="lpage">154</span><span class="pub-id"><a href="https://dx.doi.org/10.3758/s13423-016-1015-8">10.3758/s13423-016-1015-8</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26968853">26968853</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Darriba</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taboada</span><span class="given-names" tagx="given-names" title="given-names">GL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Doallo</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Posada</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="mixed-article-title" title="mixed-article-title">jModelTest 2: more models, new heuristics and parallel computing</span><span class="source" tagx="source" title="source">Nat Methods</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">772</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nmeth.2109">10.1038/nmeth.2109</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bayesian phylogenetics with BEAUti and the BEAST 1.7</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">29</span><span class="fpage" tagx="fpage" title="fpage">1969</span><span class="lpage" tagx="lpage" title="lpage">1973</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/mss075">10.1093/molbev/mss075</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22367748">22367748</a></span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ho</span><span class="given-names" tagx="given-names" title="given-names">SY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duchene</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular-clock methods for estimating evolutionary rates and timescales</span><span class="source" tagx="source" title="source">Mol Ecol</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">5947</span><span class="lpage" tagx="lpage" title="lpage">5965</span><span class="pub-id"><a href="https://dx.doi.org/10.1111/mec.12953">10.1111/mec.12953</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25290107">25290107</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemey</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span></span><span class="mixed-article-title" title="mixed-article-title">Bayesian phylogeography finds its roots</span><span class="source" tagx="source" title="source">PLoS Comput Biol</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">e1000520</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pcbi.1000520">10.1371/journal.pcbi.1000520</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19779555">19779555</a></span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bielejec</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rambaut</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suchard</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lemey</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="mixed-article-title" title="mixed-article-title">SPREAD: spatial phylogenetic reconstruction of evolutionary dynamics</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">27</span><span class="fpage" tagx="fpage" title="fpage">2910</span><span class="lpage" tagx="lpage" title="lpage">2912</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btr481">10.1093/bioinformatics/btr481</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21911333">21911333</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Larkin</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Blackshields</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brown</span><span class="given-names" tagx="given-names" title="given-names">NP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chenna</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McGettigan</span><span class="given-names" tagx="given-names" title="given-names">PA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McWilliam</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Clustal W and Clustal X version 2.0</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">2947</span><span class="lpage" tagx="lpage" title="lpage">2948</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/btm404">10.1093/bioinformatics/btm404</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17846036">17846036</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tippmann</span><span class="given-names" tagx="given-names" title="given-names">HF</span></span></span><span class="mixed-article-title" title="mixed-article-title">Analysis for free: comparing programs for sequence analysis</span><span class="source" tagx="source" title="source">Brief Bioinform</span><span class="year" tagx="year" title="year">2004</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">82</span><span class="lpage" tagx="lpage" title="lpage">87</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/bib/5.1.82">10.1093/bib/5.1.82</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15153308">15153308</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stecher</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Knyaz</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamura</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="mixed-article-title" title="mixed-article-title">MEGA X: molecular evolutionary genetics analysis across computing platforms</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">35</span><span class="fpage" tagx="fpage" title="fpage">1547</span><span class="lpage" tagx="lpage" title="lpage">1549</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msy096">10.1093/molbev/msy096</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29722887">29722887</a></span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deng</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheng</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xue</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="mixed-article-title" title="mixed-article-title">HemI: a toolkit for illustrating heatmaps</span><span class="source" tagx="source" title="source">PLoS One</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">e111988</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0111988">10.1371/journal.pone.0111988</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25372567">25372567</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Land</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Humble</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span></span><span class="mixed-article-title" title="mixed-article-title">YASARA: a tool to obtain structural guidance in biocatalytic investigations</span><span class="source" tagx="source" title="source">Methods Mol Biol</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">1685</span><span class="fpage" tagx="fpage" title="fpage">43</span><span class="lpage" tagx="lpage" title="lpage">67</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/978-1-4939-7366-8_4">10.1007/978-1-4939-7366-8_4</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29086303">29086303</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Grell</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parkin</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Slatest</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Craig</span><span class="given-names" tagx="given-names" title="given-names">PA</span></span></span><span class="mixed-article-title" title="mixed-article-title">EZ-Viz, a tool for simplifying molecular viewing in PyMOL</span><span class="source" tagx="source" title="source">Biochem Mol Biol Educ</span><span class="year" tagx="year" title="year">2006</span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">402</span><span class="lpage" tagx="lpage" title="lpage">407</span><span class="pub-id"><a href="https://dx.doi.org/10.1002/bmb.2006.494034062672">10.1002/bmb.2006.494034062672</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21638731">21638731</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Humphrey</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dalke</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schulten</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="mixed-article-title" title="mixed-article-title">VMD: visual molecular dynamics</span><span class="source" tagx="source" title="source">J Mol Graph</span><span class="year" tagx="year" title="year">1996</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">33</span><span class="lpage" tagx="lpage" title="lpage">38</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/0263-7855(96)00018-5">10.1016/0263-7855(96)00018-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8744570">8744570</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Reuter</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boros</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delwart</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pankovics</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="mixed-article-title" title="mixed-article-title">Novel seadornavirus (family Reoviridae) related to Banna virus in Europe</span><span class="source" tagx="source" title="source">Arch Virol</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">158</span><span class="fpage" tagx="fpage" title="fpage">2163</span><span class="lpage" tagx="lpage" title="lpage">2167</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s00705-013-1712-9">10.1007/s00705-013-1712-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23624680">23624680</a></span></span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gao</span><span class="given-names" tagx="given-names" title="given-names">XY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">SH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">MH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhai</span><span class="given-names" tagx="given-names" title="given-names">YG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meng</span><span class="given-names" tagx="given-names" title="given-names">WS</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular evolution of emerging Banna virus</span><span class="source" tagx="source" title="source">Infect Genet Evol</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">45</span><span class="fpage" tagx="fpage" title="fpage">250</span><span class="lpage" tagx="lpage" title="lpage">255</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.meegid.2016.08.034">10.1016/j.meegid.2016.08.034</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27590713">27590713</a></span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haines</span><span class="given-names" tagx="given-names" title="given-names">HA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olley</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kemp</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">English</span><span class="given-names" tagx="given-names" title="given-names">NB</span></span></span><span class="mixed-article-title" title="mixed-article-title">Progress in Australian dendroclimatology: identifying growth limiting factors in four climate zones</span><span class="source" tagx="source" title="source">Sci Total Environ</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">572</span><span class="fpage" tagx="fpage" title="fpage">412</span><span class="lpage" tagx="lpage" title="lpage">421</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.scitotenv.2016.08.096">10.1016/j.scitotenv.2016.08.096</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27543945">27543945</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kattoor</span><span class="given-names" tagx="given-names" title="given-names">JJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malik</span><span class="given-names" tagx="given-names" title="given-names">YS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sasidharan</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rajan</span><span class="given-names" tagx="given-names" title="given-names">VM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dhama</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ghosh</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Analysis of codon usage pattern evolution in avian rotaviruses and their preferred host</span><span class="source" tagx="source" title="source">Infect Genet Evol</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">17</span><span class="lpage" tagx="lpage" title="lpage">25</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.meegid.2015.06.018">10.1016/j.meegid.2015.06.018</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26086995">26086995</a></span></span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lara-Ramirez</span><span class="given-names" tagx="given-names" title="given-names">EE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salazar</span><span class="given-names" tagx="given-names" title="given-names">MI</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lopez-Lopez Mde</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Salas-Benito</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanchez-Varela</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">X</span></span></span><span class="mixed-article-title" title="mixed-article-title">Large-scale genomic analysis of codon usage in dengue virus and evaluation of its phylogenetic dependence</span><span class="source" tagx="source" title="source">Biomed Res Int</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">2014</span><span class="fpage" tagx="fpage" title="fpage">851425</span><span class="pub-id"><a href="https://dx.doi.org/10.1155/2014/851425">10.1155/2014/851425</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25136631">25136631</a></span></span></li> <li tag="ref"><a name="CR38" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moratorio</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Iriarte</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moreno</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Musto</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cristina</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">A detailed comparative analysis on the overall codon usage patterns in West Nile virus</span><span class="source" tagx="source" title="source">Infect Genet Evol</span><span class="year" tagx="year" title="year">2013</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">396</span><span class="lpage" tagx="lpage" title="lpage">400</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.meegid.2013.01.001">10.1016/j.meegid.2013.01.001</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23333335">23333335</a></span></span></li> <li tag="ref"><a name="CR39" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Hemert</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Berkhout</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="mixed-article-title" title="mixed-article-title">Nucleotide composition of the Zika virus RNA genome and its codon usage</span><span class="source" tagx="source" title="source">Virol J</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">13</span><span class="fpage" tagx="fpage" title="fpage">95</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12985-016-0551-1">10.1186/s12985-016-0551-1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27278486">27278486</a></span></span></li> <li tag="ref"><a name="CR40" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Long</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yao</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="mixed-article-title" title="mixed-article-title">Analysis of compositional bias and codon usage pattern of the coding sequence in Banna virus genome</span><span class="source" tagx="source" title="source">Virus Res</span><span class="year" tagx="year" title="year">2018</span><span class="volume" tagx="volume" title="volume">258</span><span class="fpage" tagx="fpage" title="fpage">68</span><span class="lpage" tagx="lpage" title="lpage">72</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.virusres.2018.10.006">10.1016/j.virusres.2018.10.006</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30316801">30316801</a></span></span></li> <li tag="ref"><a name="CR41" /><span class="label" tagx="label" title="label">41.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vabret</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bailly-Bechet</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Najburg</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Muller-Trutwin</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Verrier</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tangy</span><span class="given-names" tagx="given-names" title="given-names">F</span></span></span><span class="mixed-article-title" title="mixed-article-title">The biased nucleotide composition of HIV-1 triggers type I interferon response and correlates with subtype D increased pathogenicity</span><span class="source" tagx="source" title="source">PLoS One</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">e33502</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0033502">10.1371/journal.pone.0033502</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22529893">22529893</a></span></span></li> <li tag="ref"><a name="CR42" /><span class="label" tagx="label" title="label">42.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">He</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhou</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meng</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Isolation and genetic characterization of Mangshi virus: a newly discovered Seadornavirus of the Reoviridae family found in Yunnan Province, China</span><span class="source" tagx="source" title="source">PLoS One</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">e0143601</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0143601">10.1371/journal.pone.0143601</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26630378">26630378</a></span></span></li> </ul> </div> </div>  </body></html>