<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/embed/graphic-2.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/02/01/696195.atom" />
<meta name="HW.pisa" content="biorxiv;696195v3" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Accelerated viral dynamics in bat cell lines, with implications for zoonotic emergence" />
<meta name="DC.Identifier" content="10.1101/696195" />
<meta name="DC.Date" content="2020-02-01" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Bats host virulent zoonotic viruses without experiencing disease. A mechanistic understanding of the impact of bats’ virus hosting capacities, including uniquely constitutive immune pathways, on cellular-scale viral dynamics is needed to elucidate zoonotic emergence. We carried out virus infectivity assays on bat cell lines expressing induced and constitutive immune phenotypes, then developed a theoretical model of our in vitro system, which we fit to empirical data. Best fit models recapitulated expected immune phenotypes for representative cell lines, supporting robust antiviral defenses in bat cells that correlated with higher estimates for within-host viral propagation rates. In general, heightened immune responses limit pathogen-induced cellular morbidity, which can facilitate the establishment of rapidly-propagating persistent infections within-host. Rapidly-replicating viruses that have evolved with bat immune systems will likely cause enhanced virulence following emergence into secondary hosts with immune systems that diverge from those unique to bats." />
<meta name="DC.Contributor" content="Cara E. Brook" />
<meta name="DC.Contributor" content="Mike Boots" />
<meta name="DC.Contributor" content="Kartik Chandran" />
<meta name="DC.Contributor" content="Andrew P. Dobson" />
<meta name="DC.Contributor" content="Christian Drosten" />
<meta name="DC.Contributor" content="Andrea L. Graham" />
<meta name="DC.Contributor" content="Bryan T. Grenfell" />
<meta name="DC.Contributor" content="Marcel A. Müller" />
<meta name="DC.Contributor" content="Melinda Ng" />
<meta name="DC.Contributor" content="Lin-Fa Wang" />
<meta name="DC.Contributor" content="Anieke van Leeuwen" />
<meta name="article:published_time" content="2020-02-01" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Accelerated viral dynamics in bat cell lines, with implications for zoonotic emergence" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Bats host virulent zoonotic viruses without experiencing disease. A mechanistic understanding of the impact of bats’ virus hosting capacities, including uniquely constitutive immune pathways, on cellular-scale viral dynamics is needed to elucidate zoonotic emergence. We carried out virus infectivity assays on bat cell lines expressing induced and constitutive immune phenotypes, then developed a theoretical model of our &lt;i&gt;in vitro&lt;/i&gt; system, which we fit to empirical data. Best fit models recapitulated expected immune phenotypes for representative cell lines, supporting robust antiviral defenses in bat cells that correlated with higher estimates for within-host viral propagation rates. In general, heightened immune responses limit pathogen-induced cellular morbidity, which can facilitate the establishment of rapidly-propagating persistent infections within-host. Rapidly-replicating viruses that have evolved with bat immune systems will likely cause enhanced virulence following emergence into secondary hosts with immune systems that diverge from those unique to bats.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;696195v3" />
<meta name="citation_id" content="696195v3" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/696195v3" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/696195v3.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/696195v3.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195.full.pdf" />
<meta name="citation_doi" content="10.1101/696195" />
<meta name="citation_num_pages" content="80" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="696195" />
<meta name="citation_author" content="Cara E. Brook" />
<meta name="citation_author_institution" content="Department of Integrative Biology, University of California" />
<meta name="citation_author_institution" content="Department of Ecology &amp; Evolutionary Biology, Princeton University" />
<meta name="citation_author_email" content="cbrook@berkeley.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4276-073X" />
<meta name="citation_author" content="Mike Boots" />
<meta name="citation_author_institution" content="Department of Integrative Biology, University of California" />
<meta name="citation_author" content="Kartik Chandran" />
<meta name="citation_author_institution" content="Department of Microbiology and Immunology, Albert Einstein College of Medicine" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0232-7077" />
<meta name="citation_author" content="Andrew P. Dobson" />
<meta name="citation_author_institution" content="Department of Ecology &amp; Evolutionary Biology, Princeton University" />
<meta name="citation_author" content="Christian Drosten" />
<meta name="citation_author_institution" content="Institute of Virology, Charité-Universitätsmedizin Berlin, corporate member of Freie Universität Berlin, Humboldt-Universität zu Berlin, and Berlin Institute of Health" />
<meta name="citation_author" content="Andrea L. Graham" />
<meta name="citation_author_institution" content="Department of Ecology &amp; Evolutionary Biology, Princeton University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6580-2755" />
<meta name="citation_author" content="Bryan T. Grenfell" />
<meta name="citation_author_institution" content="Department of Ecology &amp; Evolutionary Biology, Princeton University" />
<meta name="citation_author_institution" content="Fogarty International Center, National Institutes of Health" />
<meta name="citation_author" content="Marcel A. Müller" />
<meta name="citation_author_institution" content="Institute of Virology, Charité-Universitätsmedizin Berlin, corporate member of Freie Universität Berlin, Humboldt-Universität zu Berlin, and Berlin Institute of Health" />
<meta name="citation_author_institution" content="Martsinovsky Institute of Medical Parasitology, Tropical and Vector Borne Diseases, Sechenov University" />
<meta name="citation_author" content="Melinda Ng" />
<meta name="citation_author_institution" content="Department of Microbiology and Immunology, Albert Einstein College of Medicine" />
<meta name="citation_author" content="Lin-Fa Wang" />
<meta name="citation_author_institution" content="Emerging Infectious Diseases Program, Duke-National University of Singapore Medical School" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2752-0535" />
<meta name="citation_author" content="Anieke van Leeuwen" />
<meta name="citation_author_institution" content="Royal Netherlands Institute for Sea Research, Department of Coastal Systems, and Utrecht University" />
<meta name="citation_author_institution" content="Department of Ecology &amp; Evolutionary Biology, Princeton University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1987-1458" />
<meta name="citation_reference" content="Ahn, Matae, Danielle E. Anderson, Qian Zhang, Chee Wah Tan, Beng Lee Lim, Katarina Luko, Ming Wen, et al. 2019. “Dampened NLRP3-Mediated Inflammation in Bats and Implications for a Special Viral Reservoir Host.” Nature Microbiology 4: 789–99. https://doi.org/10.1038/s41564-019-0371-3." />
<meta name="citation_reference" content="Arnold, Catherine E, Jonathan C Guito, Louis A Altamura, Sean P Lovett, Elyse R Nagle, Gustavo F Palacios, and Mariano Sanchez-lockhart. 2018. “Transcriptomics Reveal Antiviral Gene Induction in the Egyptian Rousette Bat Is Antagonized in Vitro by Marburg Virus Infection.” Viruses 10 (607). https://doi.org/10.3390/v10110607." />
<meta name="citation_reference" content="Baccam, Prasith, Catherine Beauchemin, Catherine A Macken, Frederick G Hayden, and Alan S Perelson. 2006. “Kinetics of Influenza A Virus Infection in Humans.” Journal of Virology 80 (15): 7590–99. https://doi.org/10.1128/JVI.01623-05." />
<meta name="citation_reference" content="Biesold, Susanne E., Daniel Ritz, Florian Gloza-Rausch, Robert Wollny, Jan Felix Drexler, Victor M. Corman, Elisabeth K V Kalko, Samuel Oppong, Christian Drosten, and Marcel A. Müller. 2011. “Type I Interferon Reaction to Viral Infection in Interferon-Competent, Immortalized Cell Lines from the African Fruit Bat Eidolon Helvum.” PLoS ONE 6 (11). https://doi.org/10.1371/journal.pone.0028131." />
<meta name="citation_reference" content="Bonhoeffer, S., R. M. May, G. M. Shaw, and M. A. Nowak. 1997. “Virus Dynamics and Drug Therapy.” Proceedings of the National Academy of Sciences 94 (13): 6971–76. https://doi.org/10.1073/pnas.94.13.6971." />
<meta name="citation_reference" content="Brook, Cara E., and Andrew P. Dobson. 2015. “Bats as ‘special’ Reservoirs for Emerging Zoonotic Pathogens.” Trends in Microbiology 23 (3): 172–80. https://doi.org/10.1016/j.tim.2014.12.004." />
<meta name="citation_reference" content="Brook, Cara E., Hafaliana Christian Ranaivoson, Christopher C. Broder, Andrew A. Cunningham, Jean-Michel Héraud, Alison J. Peel, Louise Gibson, James L.N. Wood, C. Jessica Metcalf, and Andrew P. Dobson. 2019. “Disentangling Serology to Elucidate Henipa- and Filovirus Transmission in Madagascar Fruit Bats.” Journal of Animal Ecology 00: 1–16. https://doi.org/10.1111/1365-2656.12985." />
<meta name="citation_reference" content="Calisher, Charles H, James E Childs, Hume E Field, Kathryn V Holmes, and Tony Schountz. 2006. “Bats: Important Reservoir Hosts of Emerging Viruses.” Clinical Microbiology Reviews 19 (3): 531–45. https://doi.org/10.1128/CMR.00017-06." />
<meta name="citation_reference" content="Coffin, John M. 1995. “HIV Population Dynamics in Vivo: Implications for Genetic Variation, Pathogenesis, and Therapy.” Science (New York, N.Y.) 267 (5197): 483–89." />
<meta name="citation_reference" content="Crameri, Gary, Shawn Todd, Samantha Grimley, Jennifer A. McEachern, Glenn A. Marsh, Craig Smith, Mary Tachedjian, et al. 2009. “Establishment, Immortalisation and Characterisation of Pteropid Bat Cell Lines.” PLoS ONE 4 (12): e8266. https://doi.org/10.1371/journal.pone.0008266." />
<meta name="citation_reference" content="Desmyter, J, J L Melnick, and W E Rawls. 1968. “Defectiveness of Interferon Production and of Rubella Virus Interference in a Line of African Green Monkey Kidney Cells (Vero).” Journal of Virology 2 (10): 955–61. https://doi.org/J.Virol.19682:1955." />
<meta name="citation_reference" content="Dhooge, A., W. Govaerts, Yu A. Kuznetsov, H. G E Meijer, and B. Sautois. 2008. “New Features of the Software MatCont for Bifurcation Analysis of Dynamical Systems.” Mathematical and Computer Modelling of Dynamical Systems 14 (2): 147–75. https://doi.org/10.1080/13873950701742754." />
<meta name="citation_reference" content="Diekmann, O., J.A.P. Heesterbeek, and J.A.J Metz. 1990. “On the Definition and Computation of the Basic Reproduction Ratio R0 in Models for Infectious Diseases in Heterogenous Populations.” Journal of Mathematical Biology 28: 365–82." />
<meta name="citation_reference" content="Eddelbuettel, Dirk, and Romain Francois. 2011. “Rcpp: Seamless R and C++ Integration.” Journal of Statistical Software 40: 1–18. https://doi.org/10.1007/978-1-4614-6868-4." />
<meta name="citation_reference" content="Eddelbuettel, Dirk, and Conrad Sanderson. 2017. “RcppArmadillo: Accelerating R with High-Performance C++ Linear Algebra.” Computational Statistics and Data Analysis 71 (2014): 1–16." />
<meta name="citation_reference" content="Emeny, J. M., and M. J. Morgan. 1979. “Regulation of the Interferon System: Evidence That Vero Cells Have a Genetic Defect in Interferon Production.” Journal of General Virology 43 (1): 247–52. https://doi.org/10.1099/0022-1317-43-1-247." />
<meta name="citation_reference" content="Emery, V.C., A.V. Cope, E.F. Bowen, D. Gor, and P.D. Griffiths. 1999. “The Dynamics of Human Cytomegalovirus Replication in Vivo.” The Journal of Experimental Medicine 190 (2): 177–82. https://doi.org/10.1084/jem.190.2.177." />
<meta name="citation_reference" content="Haydon, Daniel T, Sarah Cleaveland, Louise H Taylor, and M Karen Laurenson. 2002. “Identifying Reservoirs of Infection: A Conceptual and Practical Challenge.” Emerging Infectious Diseases 8 (12): 1468–73. https://doi.org/10.3201/eid0812.010317." />
<meta name="citation_reference" content="Heffernan, J.M., R.J. Smith, and L.M. Wahl. 2005. “Perspectives on the Basic Reproductive Ratio.” Journal of The Royal Society Interface 2 (4): 281–93. https://doi.org/10.1098/rsif.2005.0042." />
<meta name="citation_reference" content="Ho, David D., Avidan U. Neumann, Alan S. Perelson, Wen Chen, John M. Leonard, and Martin Markowitz. 1995. “Rapid Turnover of Plasma Virions and CD4 Lymphocytes in HIV-1 Infection.” Nature 373: 123–26. https://doi.org/10.1038/373123a0." />
<meta name="citation_reference" content="Hooper, Peter, Sherif Zaki, Peter Daniels, and Deborah Middleton. 2001. “Comparative Pathology of the Diseases Caused by Hendra and Nipah Viruses.” Microbes and Infection 3 (4): 315–22. https://doi.org/10.1016/S1286-4579(01)01385-5." />
<meta name="citation_reference" content="Howat, Tom J, Cristina Barreca, Peter O’Hare, Julia R Gog, and Bryan T Grenfell. 2006. “Modelling Dynamics of the Type I Interferon Response to in Vitro Viral Infection.” Journal of the Royal Society, Interface 3 (10): 699–709. https://doi.org/10.1098/rsif.2006.0136." />
<meta name="citation_reference" content="Igarashi, Manabu, Heinz Feldmann, Yoshihiro Takadate, Tatsunari Kondoh, Manabu Igarashi, Junki Maruyama, Rashid Manzoor, and Hirohito Ogawa. 2020. “Niemann-Pick C1 Heterogeneity of Bat Cells Controls Filovirus Tropism.” Cell Reports 30 (2): 308–319.e5. https://doi.org/10.1016/j.celrep.2019.12.042." />
<meta name="citation_reference" content="Kacprzyk, Joanna, Graham M. Hughes, Eva M. Palsson-McDermott, Susan R. Quinn, Sébastien J. Puechmaille, Luke A. J. O’Neill, and Emma C. Teeling. 2017. “A Potent Anti­Inflammatory Response in Bat Macrophages May Be Linked to Extended Longevity and Viral Tolerance.” Acta Chiropterologica 19 (2): 219–28. https://doi.org/10.3161/15081109ACC2017.19.2.001." />
<meta name="citation_reference" content="Keeling, Matt J., and Pejman Rohani. 2008. Modeling Infectious Diseases in Humans and Animals. Princeton, NJ: Princeton University Press." />
<meta name="citation_reference" content="Kühl, Annika, Markus Hoffmann, Marcel A. Müller, Vincent J. Munster, Kerstin Gnirß, Miriam Kiene, Theodros Solomon Tsegaye, et al. 2011. “Comparative Analysis of Ebola Virus Glycoprotein Interactions with Human and Bat Cells.” Journal of Infectious Diseases 204: S840–S849. https://doi.org/10.1093/infdis/jir306." />
<meta name="citation_reference" content="Kuzmin, I.V., Toni M. Schwarz, Philipp A. Ilinykh, Ingo Jordan, Thamas G. Ksiazek, Ravi Sachidanandam, Christopher F. Basler, and Alexander Bukreyev. 2017. “Innate Immune Response of Bat and Human Cells to Filoviruses: Commonalities and Distinctions.” Journal of Virology 91 (8): e02471–16. https://doi.org/10.1128/JVI.02471-16." />
<meta name="citation_reference" content="Livak, K J, and T D Schmittgen. 2001. “Analysis of Relative Gene Expression Data Using Real­Time Quantitative PCR and the 2Λ(-ΔΔCT) Method.” Methods 25 (4): 402–8. https://doi.org/10.1006/meth.2001.1262." />
<meta name="citation_reference" content="Mahanty, Siddhartha, and Mike Bray. 2004. “Pathogenesis of Filoviral Haemorrhagic Fevers.” Lancet Infectious Diseases 4 (8): 487–98. https://doi.org/10.1016/S1473-3099(04)01103-X." />
<meta name="citation_reference" content="Miller, Emily Happy, Gregor Obernosterer, Matthijs Raaben, Andrew S Herbert, Maika S Deffieu, Anuja Krishnan, Esther Ndungo, et al. 2012. “Ebola Virus Entry Requires the Host-Programmed Recognition of an Intracellular Receptor.” The EMBO Journal 31 (8): 1947–60. https://doi.org/10.1038/emboj.2012.53." />
<meta name="citation_reference" content="Morris, Sinead E., Andrew J. Yates, Rik L. de Stwart, Rory D. de Vries, Michael J. Mina, Ashley N. Nelson, Wen-Hsuan W. Lin, Roger D. Kouyos, Diane E. Griffin, and Bryan T. Grenfell. 2018. “Modeling the Measles Paradox Reveals the Importance of Cellular Immunity in Regulating Viral Clearance.” PLoS Pathogens 14 (12): e1007493." />
<meta name="citation_reference" content="Neumann, AU, NP Lam, H Dahari, DR Gretch, TE Wiley, TJ Layden, and AS Perelson. 1998. “Hepatitis C Viral Dynamics in Vivo and the Antiviral Efficacy of Interferon-Alpha Therapy.” Science 282: 103–7." />
<meta name="citation_reference" content="Ng, Melinda, and Kartik Chandran. 2018. “Unpublished Results.”" />
<meta name="citation_reference" content="Ng, Melinda, Esther Ndungo, Maryska Kaczmarek, Andrew S. Herbert, Tabea Binger, Rebekah James, Rohit K. Jangra, et al. 2015. “NPC1 Contributes to Species-Specific Patterns of Ebola Virus Infection in Bats.” ELife 4: e11785. https://doi.org/10.7554/eLife.11785." />
<meta name="citation_reference" content="Nicholls, J M, L L Poon, K C Lee, W F Ng, S T Lai, C Y Leung, C M Chu, et al. 2003. “Lung Pathogy of Fatal Severe Acute Respiratory Syndrom.” Lancet 361: 1773–78. https://doi.org/10.1016/S0140-6736(03)13413-7." />
<meta name="citation_reference" content="Nowak, M. A., and R. M. May. 2000. Virus Dynamics: Mathematical Principles of Immunology and Virology. Oxford, UK: Oxford University Press." />
<meta name="citation_reference" content="Nowak, Martin A., Robert M. May, Rodney E. Phillips, Sarah Rowland-Jones, David G. Lalloo, Steven McAdam, Paul Klenerman, et al. 1995. “Antigenic Oscillations and Shifting Immunodominance in HIV-1 Infections.” Nature 375: 606–11." />
<meta name="citation_reference" content="Nowak, Martin A, Sebastian Bonhoeffer, Andrew M Hill, Richard Boehme, Howard C Thomas, and Hugh McDade. 1996. “Viral Dynamics in Hepatitis B Virus Infection.” Proceedings of the National Academy of Sciences 93: 4398–4402. http://www.pnas.org/content/pnas/93/9/4398.full.pdf." />
<meta name="citation_reference" content="Pau, Gregoire, Florian Fuchs, Oleg Sklyar, Michael Boutros, and Wolfgang Huber. 2010. “EBImage-an R Package for Image Processing with Applications to Cellular Phenotypes.” Bioinformatics 26 (7): 979–81. https://doi.org/10.1093/bioinformatics/btq046." />
<meta name="citation_reference" content="Pavlovich, Stephanie S., Sean P. Lovett, Galina Koroleva, Jonathan C. Guito, Catherine E. Arnold, Elyse R. Nagle, Kirsten Kulcsar, et al. 2018. “The Egyptian Rousette Genome Reveals Unexpected Features of Bat Antiviral Immunity.” Cell 173 (5): 1098–1110. https://doi.org/10.1016/j.cell.2018.03.070." />
<meta name="citation_reference" content="Pawelek, Kasia A., Giao T. Huynh, Michelle Quinlivan, Ann Cullinane, Libin Rong, and Alan S. Perelson. 2012. “Modeling Within-Host Dynamics of Influenza Virus Infection Including Immune Responses.” PLoS Computational Biology 8 (6): e1002588. https://doi.org/10.1371/journal.pcbi.1002588." />
<meta name="citation_reference" content="Peel, Alison J., Kate S. Baker, David T. S. Hayman, Christopher C. Broder, Andrew A. Cunningham, Anthony R. Fooks, Romain Garnier, James L. N. Wood, and Olivier Restif. 2018. “Support for Viral Persistence in Bats from Age-Specific Serology and Models of Maternal Immunity.” Scientific Reports 8 (1): 3859. https://doi.org/10.1038/s41598-018-22236-6." />
<meta name="citation_reference" content="Peel, Alison J, Kate S Baker, Gary Crameri, Jennifer A Barr, David TS Hayman, Edward Wright, Christopher C Broder, et al. 2012. “Henipavirus Neutralising Antibodies in an Isolated Island Population of African Fruit Bats.” PloS One 7 (1): e30346. https://doi.org/10.1371/journal.pone.0030346." />
<meta name="citation_reference" content="Perelson, A S, A U Neumann, M Markowitz, J M Leonard, and D D Ho. 1996. “HIV-1 Dynamics in Vivo: Virion Clearance Rate, Infected Cell Life-Span, and Viral Generation Time.” Science (New York, N.Y.) 271 (5255): 1582–86. https://doi.org/10.1126/science.271.5255.1582." />
<meta name="citation_reference" content="Perelson, Alan S. 2002. “Modelling Viral and Immune System Dynamics.” Nature Reviews Immunology 2 (1): 28–36. https://doi.org/10.1038/nri700." />
<meta name="citation_reference" content="Plowright, Raina K., Alison J. Peel, Daniel G. Streicker, Amy Gilbert, Hamish McCallum, James Wood, Michelle L. Baker, and Olivier Restif. 2016. “Transmission or Within-Host Dynamics Driving Pulses of Zoonotic Viruses in Reservoir-Host Populations.” PLoS Neglected Tropical Diseases 10 (8): e0004796. https://doi.org/10.1371/journal.pntd.0004796." />
<meta name="citation_reference" content="Radke, K L, C Colby, J R Kates, H M Krider, and D M Prescott. 1974. “Establishment and Maintenance of the Interferon-Induced Antiviral State: Studies in Enucleated Cells.” Journal of Virology 13 (3): 623–30. http://www.ncbi.nlm.nih.gov/pubmed/4362865http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC355347." />
<meta name="citation_reference" content="Rasmussen, L., and L. B. Farley. 1975. “Inhibition of Herpesvirus Hominis Replication by Human Interferon.” Infection and Immunity 12 (1): 104–8." />
<meta name="citation_reference" content="Rhim, J. S., K. Schell, B. Creasy, and W. Case. 1969. “Biological Characteristics and Viral Susceptibility of an African Green Monkey Kidney Cell Line (Vero).” Proceedings of the Society for Experimental Biology and Medicine 132 (2): 670–678." />
<meta name="citation_reference" content="Saenz, Roberto A, Michelle Quinlivan, Debra Elton, Shona MacRae, Anthony S Blunden, Jennifer A Mumford, Janet M Daly, et al. 2010. “Dynamics of Influenza Virus Infection and Pathology.” Journal of Virology 84 (8): 3974–83. https://doi.org/10.1128/JVI.02078-09." />
<meta name="citation_reference" content="Samuel, C. E., and G. S. Knutson. 1982. “Mechanism of Interferon Action.” Journal of Biological Chemistry 257 (19): 11791–95." />
<meta name="citation_reference" content="Schountz, Tony, Michelle L Baker, John Butler, and Vincent Munster. 2017. “Immunological Control of Viral Infections in Bats and the Emergence of Viruses Highly Pathogenic to Humans.” Frontiers in Immunology 8 (September): Article 1098. https://doi.org/10.3389/fimmu.2017.01098." />
<meta name="citation_reference" content="Soetaert, Karline, Thomas Petzoldt, and R Woodrow Setzer. 2010. “Package DeSolve: Solving Initial Value Differential Equations in R.” Journal of Statistical Software 33 (9): 1–25. https://doi.org/10.18637/jss.v033.i09." />
<meta name="citation_reference" content="Stetson, Daniel B., and Ruslan Medzhitov. 2006. “Type I Interferons in Host Defense.” Immunity 25 (3): 373–81. https://doi.org/10.1016/j.immuni.2006.08.007." />
<meta name="citation_reference" content="Team, R Core. 2019. “R: A Language and Environment for Statistical Computing.” R Foundation for Statistical Computing. Vienna, Austria." />
<meta name="citation_reference" content="Wang, Lin-Fa, and Danielle E Anderson. 2019. “Viruses in Bats and Potential Spillover to Animals and Humans.” Current Opinion in Virology 34: 79–89. https://doi.org/10.1016/j.coviro.2018.12.007." />
<meta name="citation_reference" content="Webb, Steven D., Matt J. Keeling, and Mike Boots. 2007. “Host-Parasite Interactions between the Local and the Mean-Field: How and When Does Spatial Population Structure Matter?” Journal of Theoretical Biology 249 (1): 140–52. https://doi.org/10.1016/j.jtbi.2007.06.013." />
<meta name="citation_reference" content="Wong, Anthony C, Rohini G Sandesara, Nirupama Mulherkar, Sean P Whelan, and Kartik Chandran. 2010. “A Forward Genetic Strategy Reveals Destabilizing Mutations in the Ebolavirus Glycoprotein That Alter Its Protease Dependence during Cell Entry.” Journal of Virology 84 (1): 163–75. https://doi.org/10.1128/JVI.01832-09." />
<meta name="citation_reference" content="Xie, Jiazheng, Yang Li, Xurui Shen, Jiazheng Xie, Yang Li, Xurui Shen, Geraldine Goh, et al. 2018. “Dampened STING-Dependent Interferon Activation in Bats.” Cell Host and Microbe 23: 297–301." />
<meta name="citation_reference" content="Zhang, Guojie, Christopher Cowled, Zhengli Shi, Zhiyong Huang, Kimberly a Bishop-Lilly, Xiaodong Fang, James W Wynne, et al. 2013. “Comparative Analysis of Bat Genomes Provides Insight into the Evolution of Flight and Immunity.” Science 339 (6118): 456–60. https://doi.org/10.1126/science.1230835." />
<meta name="citation_reference" content="Zhou, Peng, Mary Tachedjian, James W Wynne, Victoria Boyd, Jie Cui, Ina Smith, Christopher Cowled, et al. 2016. “Contraction of the Type I IFN Locus and Unusual Constitutive Expression of IFN-α in Bats.” Proceedings of the National Academy of Sciences 113 (10): 2696–2701. https://doi.org/10.1073/pnas.1518240113." />
<meta name="twitter:title" content="Accelerated viral dynamics in bat cell lines, with implications for zoonotic emergence" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Bats host virulent zoonotic viruses without experiencing disease. A mechanistic understanding of the impact of bats’ virus hosting capacities, including uniquely constitutive immune pathways, on cellular-scale viral dynamics is needed to elucidate zoonotic emergence. We carried out virus infectivity assays on bat cell lines expressing induced and constitutive immune phenotypes, then developed a theoretical model of our in vitro system, which we fit to empirical data. Best fit models recapitulated expected immune phenotypes for representative cell lines, supporting robust antiviral defenses in bat cells that correlated with higher estimates for within-host viral propagation rates. In general, heightened immune responses limit pathogen-induced cellular morbidity, which can facilitate the establishment of rapidly-propagating persistent infections within-host. Rapidly-replicating viruses that have evolved with bat immune systems will likely cause enhanced virulence following emergence into secondary hosts with immune systems that diverge from those unique to bats." />
<meta name="og-title" property="og:title" content="Accelerated viral dynamics in bat cell lines, with implications for zoonotic emergence" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/696195v3" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Bats host virulent zoonotic viruses without experiencing disease. A mechanistic understanding of the impact of bats’ virus hosting capacities, including uniquely constitutive immune pathways, on cellular-scale viral dynamics is needed to elucidate zoonotic emergence. We carried out virus infectivity assays on bat cell lines expressing induced and constitutive immune phenotypes, then developed a theoretical model of our in vitro system, which we fit to empirical data. Best fit models recapitulated expected immune phenotypes for representative cell lines, supporting robust antiviral defenses in bat cells that correlated with higher estimates for within-host viral propagation rates. In general, heightened immune responses limit pathogen-induced cellular morbidity, which can facilitate the establishment of rapidly-propagating persistent infections within-host. Rapidly-replicating viruses that have evolved with bat immune systems will likely cause enhanced virulence following emergence into secondary hosts with immune systems that diverge from those unique to bats." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-02-01" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/696195v3.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/696195v3.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/696195v3.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/696195v3" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1130755" />
    <title>Accelerated viral dynamics in bat cell lines, with implications for zoonotic emergence | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"OS9gc2tLCfoTg8CQKbkRtJF2DX1TW9oyP6wvWkh5uu8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1130755","apath":"\/biorxiv\/early\/2020\/02\/01\/696195.atom","pisa":"biorxiv;696195v3","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;696195v3"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/696195v3","title":"Accelerated viral dynamics in bat cell lines, with implications for zoonotic emergence","identifier":"node\/1130755"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022aNM+T2kr4dxtSevb855x+pRv0O0THCwEAqJ1K3wEnL+ah9dg\\\\\\\/ZqBJPBRNzcV2opB0dOrV6MU\\\\\\\/CFggSfq0Y6Gaz2H2JKHebwt8kpfGlM36hottDL7qLBVyRRrqxtBTElNk1M6hVPLdvkDRXACy7ESahM8BXVY5vrpgwhgFr7fpWu0AHUlIv+zZWNpuAOuk5DmPHP5XEQtxaBkzzbJg+ifZpjk\\\\\\\/3EN+dkyfSdCDnUmsDNWHlIwI76LBDU9IUuhrHXbQK9eJ5AJFHiRh18v0URGxm0FZEi9+Pg80W6px1rmqicS47MSAeQPhqjlA4WmayX6\\\\\\\/Fnhn4X2yd6nXTx7zftgxNK3Lty2DkMBfjkd4QLMQeXe6e5PXtQcJylRCt5x7eSwG38j62jaPb2luoMZVWf4MvqZFf1kp+0LdWfxCcS4ObNP\\\\\\\/AH200xhVf8sgkdIPVbFNaekr67Fkltuokbz1EqnZJWh44iDfMwRPds0sC7pUGgvnBx1BZLOb7JWrf1txbM75OI0KgAvNWpkaPEGLFjOrC1dUGCOry6afTpuLtyh4fOicSntaiYY+Drn35LSYTxAcvqfR5jDGyeNznHTp3\\\\\\\/Zv+m56Wxlnu8ayIN3dFf\\\\\\\/YU4moWZM3+4S4ohbjz+Fg5xD+3JHAfDVPptMcIF2PZRgpvOVAiKO\\\\\\\/Sm3wL6I097LK9+3m6zQvZDZ77kg9BN7E1EJ2o29Kws34EiHkgQCnomMqzDnf6JPsPDVySbtiBMo\\\\\\\/0qy\\\\\\\/gvEcy52NS97cslF3aoseT3T8YJm6fPRMe7fNqRVv2So1dk+zCIzoR+CQfCK4788xhSJkqJJ5G5kmK4OvjtJW0Sm34TdI1AHrp7u1uI4NEPwXLVUgSI8oS6UprYDOLhuyLH1pQbLL7mOD3PDusl1o1fNBQDaGOmpZPTmVcLp6+Ez00XRUI5QtMO9dCRBY6S5Fs3xfmNX3wceDrVFCGf4rb19oIbZOXYDD2aOLsGT37SXQBKhaiqiKPpnI44U1E0u3yhCOG80Z26+YiXMxck\\\\\\\/IxYmopZjBD\\\\\\\/OLP5OP0UyIEhFo\\\\\\\/uit\\\\\\\/ypNUELvHjsrIfp4tzLNfUEwlMNk+yCdWDzaiz5fhDy60HkVx32I8k1dstnyT2Chg5X27k1mAbYugl9+P7Uzy0bpQPUgwU6jQQd2EvOSY5RSCyu27ODkT0FVr1LaCU6ItO0qIXOrPISAQPvI53nKKSBCGYxb5fo2yP0UNwwO4FVApnfJoWATQ5gSdHbX3Nv3C5u\\\\\\\/BNee+iTWFyMJEaaUTDYd8SnwFCBP+kzBUBzIFVLmcMP\\\\\\\/4mip\\\\\\\/uSU44LvEIU\\\\\\\/TLx+b4LRUitjjI6cLFZOzn\\\\\\\/K7UkTMWbAlms\\\u0022,\\\u0022iv\\\u0022:\\\u0022xhuW+OHCSVBBW3mQi8uxbQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002245bf669445f0edb18bf839a90317d9f45f2b895d658d81ff32e4163e67957135\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/696195v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1130755 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/696195v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YoFkv5-UJR3VYvQY4eE9A47WkgMmMxV3f7d3ufuuEb8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1130755" id="node1130755" data-pisa="biorxiv;696195v3" data-pisa-master="biorxiv;696195" data-apath="/biorxiv/early/2020/02/01/696195.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Accelerated viral dynamics in bat cell lines, with implications for zoonotic emergence</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4276-073X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Cara E.</span> <span class="nlm-surname">Brook</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Mike</span> <span class="nlm-surname">Boots</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-0232-7077" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Kartik</span> <span class="nlm-surname">Chandran</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Andrew P.</span> <span class="nlm-surname">Dobson</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Christian</span> <span class="nlm-surname">Drosten</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-6580-2755" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Andrea L.</span> <span class="nlm-surname">Graham</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Bryan T.</span> <span class="nlm-surname">Grenfell</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Marcel A.</span> <span class="nlm-surname">Müller</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Melinda</span> <span class="nlm-surname">Ng</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0003-2752-0535" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Lin-Fa</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0003-1987-1458" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Anieke</span> <span class="nlm-surname">van Leeuwen</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/696195 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1130755" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Cara E. Brook </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Integrative Biology, University of California</span>, Berkeley, Berkeley, CA, <span class='nlm-country'>USA</span>.</div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Ecology &#38; Evolutionary Biology, Princeton University</span>, Princeton, NJ, <span class='nlm-country'>USA</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cara%2BE.%2BBrook%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brook%20CE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACara%2BE.%2BBrook%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4276-073X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Cara E. Brook</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cbrook@berkeley.edu" class="" data-icon-position="" data-hide-link-title="0">cbrook@berkeley.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mike Boots </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Integrative Biology, University of California</span>, Berkeley, Berkeley, CA, <span class='nlm-country'>USA</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mike%2BBoots%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boots%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMike%2BBoots%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kartik Chandran </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Microbiology and Immunology, Albert Einstein College of Medicine</span>, Bronx, NY, <span class='nlm-country'>USA</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kartik%2BChandran%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chandran%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKartik%2BChandran%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0232-7077" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kartik Chandran</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Andrew P. Dobson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Ecology &#38; Evolutionary Biology, Princeton University</span>, Princeton, NJ, <span class='nlm-country'>USA</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BP.%2BDobson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dobson%20AP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrew%2BP.%2BDobson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Christian Drosten </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Institute of Virology, Charité-Universitätsmedizin Berlin, corporate member of Freie Universität Berlin, Humboldt-Universität zu Berlin, and Berlin Institute of Health</span>, Berlin, <span class='nlm-country'>Germany</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BDrosten%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Drosten%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristian%2BDrosten%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Andrea L. Graham </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Ecology &#38; Evolutionary Biology, Princeton University</span>, Princeton, NJ, <span class='nlm-country'>USA</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrea%2BL.%2BGraham%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Graham%20AL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrea%2BL.%2BGraham%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6580-2755" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrea L. Graham</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Bryan T. Grenfell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Ecology &#38; Evolutionary Biology, Princeton University</span>, Princeton, NJ, <span class='nlm-country'>USA</span>.</div><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Fogarty International Center, National Institutes of Health</span>, Bethesda, Maryland, <span class='nlm-country'>USA</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bryan%2BT.%2BGrenfell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grenfell%20BT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABryan%2BT.%2BGrenfell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Marcel A. Müller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Institute of Virology, Charité-Universitätsmedizin Berlin, corporate member of Freie Universität Berlin, Humboldt-Universität zu Berlin, and Berlin Institute of Health</span>, Berlin, <span class='nlm-country'>Germany</span>.</div><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Martsinovsky Institute of Medical Parasitology, Tropical and Vector Borne Diseases, Sechenov University</span>, Moscow, <span class='nlm-country'>Russia</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcel%2BA.%2BM%C3%BCller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=M%C3%BCller%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarcel%2BA.%2BM%25C3%25BCller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Melinda Ng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Microbiology and Immunology, Albert Einstein College of Medicine</span>, Bronx, NY, <span class='nlm-country'>USA</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Melinda%2BNg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ng%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMelinda%2BNg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Lin-Fa Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>7</span><span class='nlm-institution'>Emerging Infectious Diseases Program, Duke-National University of Singapore Medical School</span>, <span class='nlm-country'>Singapore</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lin-Fa%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALin-Fa%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2752-0535" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lin-Fa Wang</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Anieke van Leeuwen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>8</span><span class='nlm-institution'>Royal Netherlands Institute for Sea Research, Department of Coastal Systems, and Utrecht University</span>, the <span class='nlm-country'>Netherlands</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Ecology &#38; Evolutionary Biology, Princeton University</span>, Princeton, NJ, <span class='nlm-country'>USA</span>.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anieke%2Bvan%2BLeeuwen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=van%20Leeuwen%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnieke%2Bvan%2BLeeuwen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1987-1458" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Anieke van Leeuwen</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/696195v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1130755" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1130755/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/696195v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1130755" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1130755/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/696195v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1130755" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1130755/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/696195v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1130755" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1130755/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/696195v3.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:1130755" data-trigger="external-links" data-url-enabled="1">Data/Code</a><a href="/panels_ajax_tab/article_tab_data_code/node:1130755/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/696195v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1130755" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1130755/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Bats host virulent zoonotic viruses without experiencing disease. A mechanistic understanding of the impact of bats’ virus hosting capacities, including uniquely constitutive immune pathways, on cellular-scale viral dynamics is needed to elucidate zoonotic emergence. We carried out virus infectivity assays on bat cell lines expressing induced and constitutive immune phenotypes, then developed a theoretical model of our <em>in vitro</em> system, which we fit to empirical data. Best fit models recapitulated expected immune phenotypes for representative cell lines, supporting robust antiviral defenses in bat cells that correlated with higher estimates for within-host viral propagation rates. In general, heightened immune responses limit pathogen-induced cellular morbidity, which can facilitate the establishment of rapidly-propagating persistent infections within-host. Rapidly-replicating viruses that have evolved with bat immune systems will likely cause enhanced virulence following emergence into secondary hosts with immune systems that diverge from those unique to bats.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-5">Bats have received much attention in recent years for their role as reservoir hosts for emerging viral zoonoses, including rabies and related lyssaviruses, Hendra and Nipah henipaviruses, Ebola and Marburg filoviruses, and SARS coronavirus (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Calisher et al. 2006</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Wang and Anderson 2019</a>). In most non-Chiropteran mammals, henipaviruses, filoviruses, and coronaviruses induce substantial morbidity and mortality, display short durations of infection, and elicit robust, long-term immunity in hosts surviving infection (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Nicholls et al. 2003</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hooper et al. 2001</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Mahanty and Bray 2004</a>). Bats, by contrast, demonstrate no obvious disease symptoms upon infection with pathogens that are highly virulent in non-volant mammals (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Schountz et al. 2017</a>) but may, instead, support viruses as long-term persistent infections, rather than transient, immunizing pathologies (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Plowright et al. 2016</a>).</p><p id="p-6">Recent research advances are beginning to shed light on the molecular mechanisms by which bats avoid pathology from these otherwise virulent pathogens (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Brook and Dobson 2015</a>). Bats leverage a suite of species-specific mechanisms to limit viral load, which include host receptor sequence incompatibilities for some bat-virus combinations (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Ng et al. 2015</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Igarashi et al. 2020</a>) and constitutive expression of the antiviral cytokine, IFN-α, for others (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Zhou et al. 2016</a>). Typically, the presence of viral RNA or DNA in the cytoplasm of mammalian cells will induce secretion of type I interferon proteins (IFN-α and IFN-β), which promote expression and translation of interferon-stimulated genes (ISGs) in neighboring cells and render them effectively antiviral (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Stetson and Medzhitov 2006</a>). In some bat cells, the transcriptomic blueprints for this IFN response are expressed constitutively, even in the absence of stimulation by viral RNA or DNA (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Zhou et al. 2016</a>). In non-flying mammals, constitutive IFN expression would likely elicit widespread inflammation and concomitant immunopathology upon viral infection, but bats support unique adaptations to combat inflammation (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Zhang et al. 2013</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ahn et al. 2019</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Xie et al. 2018</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Pavlovich et al. 2018</a>) that may have evolved to mitigate metabolic damage induced during flight (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kacprzyk et al. 2017</a>). The extent to which constitutive IFN-α expression signifies constitutive antiviral defense in the form of functional IFN-α protein remains unresolved. In bat cells constitutively expressing IFN-α, some protein-stimulated, downstream ISGs appear to be also constitutively expressed, but additional ISG induction is nonetheless possible following viral challenge and stimulation of IFN-β (<a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Zhou et al. 2016</a>; <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Xie et al. 2018</a>). Despite recent advances in molecular understanding of bat viral tolerance, the consequences of this unique bat immunity on within-host virus dynamics—and its implications for understanding zoonotic emergence— have yet to be elucidated.</p><p id="p-7">The field of ‘virus dynamics’ was first developed to describe the mechanistic underpinnings of long-term patterns of steady-state viral load exhibited by patients in chronic phase infections with HIV, who appeared to produce and clear virus at equivalent rates (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Nowak and May 2000</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Ho et al. 1995</a>). Models of simple target cell depletion, in which viral load is dictated by a bottom-up resource supply of infection-susceptible host cells, were first developed for HIV (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Perelson 2002</a>) but have since been applied to other chronic infections, including hepatitis-C virus (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Neumann et al. 1998</a>), hepatitis-B virus (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Nowak et al. 1996</a>) and cytomegalovirus (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Emery et al. 1999</a>). Recent work has adopted similar techniques to model the within-host dynamics of acute infections, such as influenza A and measles, inspiring debate over the extent to which explicit modeling of top-down immune control can improve inference beyond the basic resource limitation assumptions of the target cell model (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baccam et al. 2006</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Pawelek et al. 2012</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Saenz et al. 2010</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Morris et al. 2018</a>).</p><p id="p-8">To investigate the impact of unique bat immune processes on <em>in vitro</em> viral kinetics, we first undertook a series of virus infection experiments on bat cell lines expressing divergent interferon phenotypes, then developed a theoretical model elucidating the dynamics of within-host viral spread. We evaluated our theoretical model analytically independent of the data, then fit the model to data recovered from <em>in vitro</em> experimental trials in order to estimate rates of within-host virus transmission and cellular progression to antiviral status under diverse assumptions of absent, induced, and constitutive immunity. Finally, we confirmed our findings in spatially-explicit stochastic simulations of fitted time series from our mean field model. We hypothesized that top-down immune processes would overrule classical resource-limitation in bat cell lines described as constitutively antiviral in the literature, offering a testable prediction for models fit to empirical data. We further predicted that the most robust antiviral responses would be associated with the most rapid within-host virus propagation rates but also protect cells against virus-induced mortality to support the longest enduring infections in tissue culture.</p></div><div class="section" id="sec-2"><h2 class="">Results</h2><div id="sec-3" class="subsection"><h3>Virus infection experiments in antiviral bat cell cultures yield reduced cell mortality and elongated epidemics</h3><p id="p-9">We first explored the influence of innate immune phenotype on within-host viral propagation in a series of infection experiments in cell culture. We conducted plaque assays on six-well plate monolayers of three immortalized mammalian kidney cell lines: [1] Vero (African green monkey) cells, which are IFN-defective and thus limited in antiviral capacity (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Desmyter, Melnick, and Rawls 1968</a>); [2] RoNi/7.1 (<em>Rousettus aegyptiacus</em>) cells which demonstrate idiosyncratic induced interferon responses upon viral challenge (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kuzmin et al. 2017</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Arnold et al. 2018</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Biesold et al. 2011</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Pavlovich et al. 2018</a>); and [3] PaKiT01 (<em>Pteropus alecto</em>) cells which constitutively express IFN-α (<a id="xref-ref-61-4" class="xref-bibr" href="#ref-61">Zhou et al. 2016</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Crameri et al. 2009</a>). To intensify cell line-specific differences in constitutive immunity, we carried out infectivity assays with GFP-tagged, replication-competent vesicular stomatitis Indiana viruses: rVSV-G, rVSV-EBOV, and rVSV-MARV, which have been previously described (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Miller et al. 2012</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Wong et al. 2010</a>). Two of these viruses, rVSV-EBOV and rVSV-MARV, are recombinants for which cell entry is mediated by the glycoprotein of the bat-evolved filoviruses, Ebola (EBOV) and Marburg (MARV), thus allowing us to modulate the extent of structural, as well as immunological, antiviral defense at play in each infection. Previous work in this lab has demonstrated incompatibilities in the NPC1 filovirus receptor which render PaKiT01 cells refractory to infection with rVSV-MARV (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Ng and Chandran 2018</a>), making them structurally antiviral, over and above their constitutive expression of IFN-α. All three cell lines were challenged with all three viruses at two multiplicities of infection (MOI): 0.001 and 0.0001. Between 18-39 trials were run at each cell-virus-MOI combination, excepting rVSV-MARV infections on PaKiT01 cells at MOI=0.001, for which only 8 trials were run (see Materials and Methods; <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 1-figure supplement 1-3</a>, Supplementary File 1).</p><p id="p-10">Because plaque assays restrict viral transmission neighbor-to-neighbor in two-dimensional cellular space (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Howat et al. 2006</a>), we were able to track the spread of GFP-expressing virus-infected cells across tissue monolayers via inverted fluorescence microscopy. For each infection trial, we monitored and re-imaged plates for up to 200 hours of observations or until total monolayer destruction, processed resulting images, and generated a time series of the proportion of infectious-cell occupied plate space across the duration of each trial (see Materials and Methods). We used generalized additive models to infer the time course of all cell culture replicates and construct the multi-trial dataset to which we eventually fit our mechanistic transmission model for each cell line-virus-specific combination (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>; <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 1-figure supplement 1-5</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Fitted time series of infectious cell proportions from mean field model for rVSV-G, rVSV-EBOV, and rVSV-MARV infections (columns) on Vero, RoNi/7.1, and PaKiT01 cell lines (rows) at MOI=0.001. Results are shown for the best fit immune absent model on Vero cells, induced immunity model on RoNi/7.1 cells, and constitutive (for rVSV-VSVG and rVSV-EBOV) and induced (for rVSV-MARV) immunity models on PaKiT01 cells. Raw data across all trials are shown as open circles (statistical smoothers from each trial used for fitting are available in Figure 1-figure supplement 2-3). Model output is shown as a solid crimson line (95% confidence intervals by standard error = red shading). Panel background corresponds to empirical outcome of the average stochastic cell culture trial (persistent infection = white; virus-induced epidemic extinction = gray; immune-mediated epidemic extinction = black). Parameter values are listed in Table 1 and Supplementary File 4. Results for absent/induced/constitutive fitted models across all cell lines are shown in Figure 1-figure supplement 4 (MOI=0.001) and Figure 1-figure supplement 5 (MOI=0.0001)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1488400151" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Fitted time series of infectious cell proportions from mean field model for rVSV-G, rVSV-EBOV, and rVSV-MARV infections (columns) on Vero, RoNi/7.1, and PaKiT01 cell lines (rows) at MOI=0.001. Results are shown for the best fit immune absent model on Vero cells, induced immunity model on RoNi/7.1 cells, and constitutive (for rVSV-VSVG and rVSV-EBOV) and induced (for rVSV-MARV) immunity models on PaKiT01 cells. Raw data across all trials are shown as open circles (statistical smoothers from each trial used for fitting are available in Figure 1-figure supplement 2-3). Model output is shown as a solid crimson line (95% confidence intervals by standard error = red shading). Panel background corresponds to empirical outcome of the average stochastic cell culture trial (persistent infection = white; virus-induced epidemic extinction = gray; immune-mediated epidemic extinction = black). Parameter values are listed in Table 1 and Supplementary File 4. Results for absent/induced/constitutive fitted models across all cell lines are shown in Figure 1-figure supplement 4 (MOI=0.001) and Figure 1-figure supplement 5 (MOI=0.0001).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F1.medium.gif" width="440" height="323"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F1.medium.gif" width="440" height="323"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-11" class="first-child">Fitted time series of infectious cell proportions from mean field model for rVSV-G, rVSV-EBOV, and rVSV-MARV infections (columns) on Vero, RoNi/7.1, and PaKiT01 cell lines (rows) at MOI=0.001. Results are shown for the best fit immune absent model on Vero cells, induced immunity model on RoNi/7.1 cells, and constitutive (for rVSV-VSVG and rVSV-EBOV) and induced (for rVSV-MARV) immunity models on PaKiT01 cells. Raw data across all trials are shown as open circles (statistical smoothers from each trial used for fitting are available in <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 1-figure supplement 2-3</a>). Model output is shown as a solid crimson line (95% confidence intervals by standard error = red shading). Panel background corresponds to empirical outcome of the average stochastic cell culture trial (persistent infection = white; virus-induced epidemic extinction = gray; immune-mediated epidemic extinction = black). Parameter values are listed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and Supplementary File 4. Results for absent/induced/constitutive fitted models across all cell lines are shown in <a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure 1-figure supplement 4</a> (MOI=0.001) and <a id="xref-fig-10-2" class="xref-fig" href="#F10">Figure 1-figure supplement 5</a> (MOI=0.0001).</p><div class="sb-div caption-clear"></div></div></div><p id="p-12">All three recombinant vesicular stomatitis viruses (rVSV-G, rVSV-EBOV, and rVSV-MARV) infected Vero, RoNi/7.1, and PaKiT01 tissue cultures at both focal MOIs. Post­invasion, virus spread rapidly across most cell monolayers, resulting in virus-induced epidemic extinction. Epidemics were less severe in bat cell cultures, especially when infected with the recombinant filoviruses, rVSV-EBOV and rVSV-MARV. Monolayer destruction was avoided in the case of rVSV-EBOV and rVSV-MARV infections on PaKiT01 cells: in the former, persistent viral infection was maintained throughout the 200-hour duration of each experiment, while, in the latter, infection was eliminated early in the time series, preserving a large proportion of live, uninfectious cells across the duration of the experiment. We assumed this pattern to be the result of immune-mediated epidemic extinction (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Patterns from MOI=0.001 were largely recapitulated at MOI = 0.0001, though at somewhat reduced total proportions (<a id="xref-fig-10-3" class="xref-fig" href="#F10">Figure 1-figure supplement 5</a>).</p></div><div id="sec-4" class="subsection"><h3>A theoretical model fit to <em>in vitro</em> data recapitulates expected immune phenotypes for bat cells</h3><p id="p-13">We next developed a within-host model to fit to these data in order to elucidate the effects of induced and constitutive immunity on the dynamics of viral spread in host tissue (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). The compartmental within-host system mimicked our two-dimensional cell culture monolayer, with cells occupying five distinct infection states: susceptible (S), antiviral (A), exposed (E), infectious (I), and dead (D). We modeled exposed cells as infected but not yet infectious, capturing the ‘eclipse phase’ of viral integration into a host cell which precedes viral replication. Antiviral cells were immune to viral infection, in accordance with the “antiviral state” induced from interferon stimulation of ISGs in tissues adjacent to infection (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Stetson and Medzhitov 2006</a>). Because we aimed to translate available data into modeled processes, we did not explicitly model interferon dynamics but instead scaled the rate of cell progression from susceptible to antiviral (ρ) by the proportion of exposed cells (globally) in the system. In systems permitting constitutive immunity, a second rate of cellular acquisition of antiviral status (ε) additionally scaled with the global proportion of susceptible cells in the model. Compared with virus, IFN particles are small and highly diffusive, justifying this global signaling assumption at the limited spatial extent of a six well plate and maintaining consistency with previous modeling approximations of IFN signaling in plaque assay (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Howat et al. 2006</a>).</p><p id="p-14">To best represent our empirical monolayer system, we expressed our state variables as proportions <em>(P<sub>s</sub></em>, <em>P<sub>A</sub></em>, <em>P<sub>E</sub></em>, <em>P<sub>I</sub></em>, and <em>P<sub>D</sub></em>), under assumptions of frequency-dependent transmission in a well-mixed population (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Keeling and Rohani 2008</a>), though note that the inclusion of <em>P<sub>D</sub></em> (representing the proportion of dead space in the modeled tissue) had the functional effect of varying transmission with infectious cell density. This resulted in the following system of ordinary differential equations:
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-2.gif"/></noscript></span></span>
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-3.gif"/></noscript></span></span>
<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-4.gif"/></noscript></span></span>
<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-5.gif"/></noscript></span></span>
<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-6.gif"/></noscript></span></span></p><p id="p-15">We defined “induced immunity” as complete, modeling all cells as susceptible to viral invasion at disease free equilibrium, with defenses induced subsequent to viral exposure through the term p. By contrast, we allowed the extent of constitutive immunity to vary across the parameter range of <em>ε</em> &gt; 0, defining a “constitutive” system as one containing <em>any</em> antiviral cells at disease free equilibrium. In fitting this model to tissue culture data, we independently estimated both ρ and <em>ε,</em> as well as the cell-to-cell transmission rate, β, for each cell-virus combination. Since the extent to which constitutively-expressed IFN-α is constitutively translated into functional protein is not yet known for bat hosts (<a id="xref-ref-61-5" class="xref-bibr" href="#ref-61">Zhou et al. 2016</a>), this approach permitted our tissue culture data to drive modeling inference: even in PaKiT01 cell lines known to constitutively express IFN-α, the true constitutive extent of the system (i.e. the quantity of antiviral cells present at disease free equilibrium) was allowed to vary through estimation of <em>ε.</em> For the purposes of model-fitting, we fixed the value of <em>c,</em> the return rate of antiviral cells to susceptible status, at 0. The small spatial scale and short time course (max 200 hours) of our experiments likely prohibited any return of antiviral cells to susceptible status in our empirical system; nonetheless, we retained the term <em>c</em> in analytical evaluations of our model because regression from antiviral to susceptible status is possible over long time periods <em>in vitro</em> and at the scale of a complete organism (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Samuel and Knutson 1982</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Rasmussen and Farley 1975</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Radke et al. 1974</a>).</p><p id="p-16">Before fitting to empirical time series, we undertook bifurcation analysis of our theoretical model and generated testable hypotheses on the basis of model outcomes. From our 9 within-host model system (<a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">equations 1</a>-<a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">5</a>), we derived the following expression for <em>R<sub>0</sub></em>, the pathogen basic reproduction number (Supplementary File 2):
<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-7.gif"/></noscript></span></span></p><p id="p-17">Pathogens can invade a host tissue culture when <em>R<sub>0</sub> &gt;</em> 1. Rapid rates of constitutive antiviral acquisition (<em>ε</em>) will drive <em>R<sub>0</sub></em> &lt;1: tissue cultures with highly constitutive antiviral immunity will be therefore resistant to virus invasion from the outset. Since, by definition, induced immunity is stimulated following initial virus invasion, the rate of induced antiviral acquisition (ρ) is not incorporated into the equation for <em>R<sub>0</sub>;</em> while induced immune processes can control virus after initial invasion, they cannot prevent it from occurring to begin with. In cases of fully induced or absent immunity (<em>ε</em> = 0), the <em>R<sub>0</sub></em> equation thus reduces to a form typical of the classic SEIR model:
<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-8.gif"/></noscript></span></span></p><p id="p-18">At equilibrium, the theoretical, mean field model demonstrates one of three infection states: endemic equilibrium, stable limit cycles, or no infection (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Respectively, these states approximate the persistent infection, virus-induced epidemic extinction, and immune-mediated epidemic extinction phenotypes previously witnessed in tissue culture experiments (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). Theoretically, endemic equilibrium is maintained when new infections are generated at the same rate at which infections are lost, while limit cycles represent parameter space under which infectious and susceptible populations are locked in predictable oscillations. Endemic equilibria resulting from cellular regeneration (i.e. births) have been described <em>in vivo</em> for HIV (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Coffin 1995</a>) and <em>in vitro</em> for herpesvirus plaque assays (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Howat et al. 2006</a>), but, because they so closely approach zero, true limit cycles likely only occur theoretically, instead yielding stochastic extinctions in empirical time series.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Two parameter bifurcations of the mean field model, showing variation in the transmission rate, &#x3B2;, against variation in the pathogen-induced mortality rate, &#x3B1;, under diverse immune assumptions. Panel (A) depicts dynamics under variably constitutive immunity, ranging from absent (left: &#x3B5; = 0) to high (right: &#x3B5; = .0025). In all panel (A) plots, the rate of induced immune antiviral acquisition (&#x3C1;) was fixed at 0.01. Panel (B) depicts dynamics under variably induced immunity, ranging from absent (left: &#x3C1;=0) to high (right: &#x3C1;=1). In all panel (B) plots, the rate of constitutive antiviral acquisition (&#x3B5;) was fixed at 0.0001. Branch point curves are represented as solid lines and Hopf curves as dashed lines. White space indicates endemic equilibrium (persistence), gray space indicates limit cycles, and black space indicates no infection (extinction). Other parameter values for equilibrium analysis were fixed at: b = .025, &#x3BC; = .001, &#x3C3; = 1/6, c = 0. Special points from bifurcations analyses are listed in Supplementary File 3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1488400151" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Two parameter bifurcations of the mean field model, showing variation in the transmission rate, &#x3B2;, against variation in the pathogen-induced mortality rate, &#x3B1;, under diverse immune assumptions. Panel (A) depicts dynamics under variably constitutive immunity, ranging from absent (left: &lt;em&gt;&#x3B5; =&lt;/em&gt; 0) to high (right: &lt;em&gt;&#x3B5; =&lt;/em&gt; .0025). In all panel (A) plots, the rate of induced immune antiviral acquisition (&#x3C1;) was fixed at 0.01. Panel (B) depicts dynamics under variably induced immunity, ranging from absent (left: &#x3C1;=0) to high (right: &#x3C1;=1). In all panel (B) plots, the rate of constitutive antiviral acquisition (&lt;em&gt;&#x3B5;&lt;/em&gt;) was fixed at 0.0001. Branch point curves are represented as solid lines and Hopf curves as dashed lines. White space indicates endemic equilibrium (persistence), gray space indicates limit cycles, and black space indicates no infection (extinction). Other parameter values for equilibrium analysis were fixed at: &lt;em&gt;b&lt;/em&gt; = .025, &#x3BC; = .001, &#x3C3; = 1/6, &lt;em&gt;c&lt;/em&gt; = 0. Special points from bifurcations analyses are listed in Supplementary File 3.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F2.medium.gif" width="440" height="307"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F2.medium.gif" width="440" height="307"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-19" class="first-child">Two parameter bifurcations of the mean field model, showing variation in the transmission rate, β, against variation in the pathogen-induced mortality rate, α, under diverse immune assumptions. Panel (A) depicts dynamics under variably constitutive immunity, ranging from absent (left: <em>ε =</em> 0) to high (right: <em>ε =</em> .0025). In all panel (A) plots, the rate of induced immune antiviral acquisition (ρ) was fixed at 0.01. Panel (B) depicts dynamics under variably induced immunity, ranging from absent (left: ρ=0) to high (right: ρ=1). In all panel (B) plots, the rate of constitutive antiviral acquisition (<em>ε</em>) was fixed at 0.0001. Branch point curves are represented as solid lines and Hopf curves as dashed lines. White space indicates endemic equilibrium (persistence), gray space indicates limit cycles, and black space indicates no infection (extinction). Other parameter values for equilibrium analysis were fixed at: <em>b</em> = .025, μ = .001, σ = 1/6, <em>c</em> = 0. Special points from bifurcations analyses are listed in Supplementary File 3.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">Bifurcation analysis of our mean field model revealed that regions of no infection (pathogen extinction) were bounded at lower threshold (Branch point) values for β, below which the pathogen was unable to invade. We found no upper threshold to invasion for β under any circumstances (i.e. β high enough to drive pathogen-induced extinction), but high β values resulted in Hopf bifurcations, which delineate regions of parameter space characterized by limit cycles. Since limit cycles so closely approach zero, high βs recovered in this range would likely produce virus-induced epidemic extinctions under experimental conditions. Under more robust representations of immunity, with higher values for either or both induced (ρ) and constitutive (<em>ε</em>) rates of antiviral acquisition, Hopf bifurcations occurred at increasingly higher values for β, meaning that persistent infections could establish at higher viral replication rates (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Consistent with our derivation for <em>R<sub>0</sub>,</em> we found that the Branch point threshold for viral invasion was independent of changes to the induced immune parameter (ρ) but saturated at high values of <em>ε</em> that characterize highly constitutive immunity (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Two parameter bifurcations of the mean field model, showing variation in the transmission rate, &#x3B2;, against variation in: (A) the induced immunity rate of antiviral acquisition (&#x3C1;) and (B) the constitutive immunity rate of antiviral acquisition (&#x3B5;). Panels show variation in the extent of immunity, from absent (left) to high (right). Branch point curves are represented as solid lines and Hopf curves as dashed lines. White space indicates endemic equilibrium (persistence), gray space indicates limit cycling, and black space indicates no infection (extinction). Other parameter values for equilibrium analysis were fixed at: b = .025, &#x3BC; = .001, &#x3C3; = 1/6, &#x3B1; = 1/6, c = 0. Special points from bifurcations analyses are listed in Supplementary File 3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1488400151" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Two parameter bifurcations of the mean field model, showing variation in the transmission rate, &#x3B2;, against variation in: (A) the induced immunity rate of antiviral acquisition (&#x3C1;) and (B) the constitutive immunity rate of antiviral acquisition &lt;em&gt;(&#x3B5;).&lt;/em&gt; Panels show variation in the extent of immunity, from absent (left) to high (right). Branch point curves are represented as solid lines and Hopf curves as dashed lines. White space indicates endemic equilibrium (persistence), gray space indicates limit cycling, and black space indicates no infection (extinction). Other parameter values for equilibrium analysis were fixed at: &lt;em&gt;b&lt;/em&gt; = .025, &#x3BC; = .001, &#x3C3; = 1/6, &#x3B1; = 1/6, &lt;em&gt;c&lt;/em&gt; = 0. Special points from bifurcations analyses are listed in Supplementary File 3.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F3.medium.gif" width="440" height="307"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F3.medium.gif" width="440" height="307"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <p id="p-21" class="first-child">Two parameter bifurcations of the mean field model, showing variation in the transmission rate, β, against variation in: (A) the induced immunity rate of antiviral acquisition (ρ) and (B) the constitutive immunity rate of antiviral acquisition <em>(ε).</em> Panels show variation in the extent of immunity, from absent (left) to high (right). Branch point curves are represented as solid lines and Hopf curves as dashed lines. White space indicates endemic equilibrium (persistence), gray space indicates limit cycling, and black space indicates no infection (extinction). Other parameter values for equilibrium analysis were fixed at: <em>b</em> = .025, μ = .001, σ = 1/6, α = 1/6, <em>c</em> = 0. Special points from bifurcations analyses are listed in Supplementary File 3.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">We next fit our theoretical model by least squares to each cell line-virus combination, under absent, induced, and constitutive assumptions of immunity. In general, best fit models recapitulated expected outcomes based on the immune phenotype of the cell line in question, as described in the general literature (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>; Supplementary File 4). The absent immune model offered the most accurate approximation of IFN-deficient Vero cell time series, the induced immune model best recovered the RoNi/7.1 cell trials, and, in most cases, the constitutive immune model most closely recaptured infection dynamics across constitutively IFN-a-expressing PaKiT01 cell lines (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>; <a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 1-figure supplement 4-5</a>, Supplementary File 4). Ironically, the induced immune model offered a slightly better fit than the constitutive to rVSV-MARV infections on the PaKiT01 cell line (the one cell line-virus combination for which we know a constitutively antiviral cell-receptor incompatibility to be at play). Because constitutive immune assumptions can prohibit pathogen invasion <em>(R<sub>0</sub> &lt;</em> 1), model fits to this time series under constitutive assumptions were handicapped by overestimations of <em>ε</em>, which prohibited pathogen invasion. Only by incorporating an exceedingly rapid rate of induced antiviral acquisition could the model guarantee that initial infection would be permitted and then rapidly controlled.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1135245/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1135245/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1135245" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <span class="caption-title">Optimized parameters from best fit deterministic model and spatial approximation at MOI=.001</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Robust immunity is linked to rapid within-host virus transmission rates in fitted models</h3><p id="p-23">In fitting our theoretical model to <em>in vitro</em> data, we estimated the within-host virus transmission rate (β) and the rate(s) of cellular acquisition to antiviral status (ρ or ρ + <em>ε)</em> (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>; Supplementary File 4). Under absent immune assumptions, ρ and <em>ε</em> were fixed at 0 while β was estimated; under induced immune assumptions, <em>ε</em> was fixed at 0 while ρ and β were estimated; and under constitutive immune assumptions, all three parameters (ρ, <em>ε</em>, and β) were simultaneously estimated for each cell-virus combination. Best fit parameter estimates for MOI=0.001 data are visualized in conjunction with β - ρ and β - <em>ε</em> bifurcations in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>; all general patterns were recapitulated at lower values for β on MOI=0.0001 trials (<a id="xref-fig-13-1" class="xref-fig" href="#F13">Figure 4-figure supplement 1</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Best fit parameter estimates for &#x3B2; and &#x3C1; or &#x3B5; from mean-field model fits to MOI=0.001 time series data, atop (A,B) &#x3B2; - &#x3C1; and (C) &#x3B2; - &#x3B5; bifurcation. Fits and bifurcations are grouped by immune phenotype: (A) absent; (B) induced; (C) constitutive immunity, with cell lines differentiated by shape (Vero=circles; RoNi/7.1 = triangles; PaKiT01=squares) and viral infections by color (rVSV-G = green, rVSV-EBOV = magenta, rVSV-MARV = blue). Note that y-axis values are ten&#xAD;fold higher in panel (C). Branch point curves (solid lines) and Hopf curves (dashed lines) are reproduced from Figure 3. White space indicates endemic equilibrium (pathogen persistence), gray space indicates limit cycling (virus-induced epidemic extinction), and black space indicates no infection (immune-mediated pathogen extinction). In panel (A) and (B), &#x3B5; is fixed at 0; in panel (C), &#x3C1; is fixed at 5&#xD7;10-8 for bifurcation curves and estimated at 4&#xD7;10-8 and 8&#xD7;10-8 for rVSV-EBOV and rVSV-G parameter points, respectively. Other parameter values were fixed at: b = .025, &#x3BC; = 0.001, &#x3C3; = 1/6, &#x3B1; = 1/6, and c = 0 across all panels. Raw fitted values and corresponding 95% confidence intervals for &#x3B2;, &#x3C1;, and &#x3B5;, background parameter values, and AIC recovered from model fit, are reported in Supplementary File 4. Parameter fits at MOI=0.0001 are visualized in Figure 4-figure supplement 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1488400151" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Best fit parameter estimates for &#x3B2; and &#x3C1; or &lt;em&gt;&#x3B5;&lt;/em&gt; from mean-field model fits to MOI=0.001 time series data, atop (A,B) &#x3B2; - &#x3C1; and (C) &#x3B2; - &lt;em&gt;&#x3B5;&lt;/em&gt; bifurcation. Fits and bifurcations are grouped by immune phenotype: (A) absent; (B) induced; (C) constitutive immunity, with cell lines differentiated by shape (Vero=circles; RoNi/7.1 = triangles; PaKiT01=squares) and viral infections by color (rVSV-G = green, rVSV-EBOV = magenta, rVSV-MARV = blue). Note that y-axis values are ten&#xAD;fold higher in panel (C). Branch point curves (solid lines) and Hopf curves (dashed lines) are reproduced from Figure 3. White space indicates endemic equilibrium (pathogen persistence), gray space indicates limit cycling (virus-induced epidemic extinction), and black space indicates no infection (immune-mediated pathogen extinction). In panel (A) and (B), &lt;em&gt;&#x3B5;&lt;/em&gt; is fixed at 0; in panel (C), &#x3C1; is fixed at 5&#xD7;10&lt;sup&gt;-8&lt;/sup&gt; for bifurcation curves and estimated at 4&#xD7;10&lt;sup&gt;-8&lt;/sup&gt; and 8&#xD7;10&lt;sup&gt;-8&lt;/sup&gt; for rVSV-EBOV and rVSV-G parameter points, respectively. Other parameter values were fixed at: &lt;em&gt;b&lt;/em&gt; = .025, &#x3BC; = 0.001, &#x3C3; = 1/6, &#x3B1; = 1/6, and c = 0 across all panels. Raw fitted values and corresponding 95% confidence intervals for &#x3B2;, &#x3C1;, and &lt;em&gt;&#x3B5;&lt;/em&gt;, background parameter values, and AIC recovered from model fit, are reported in Supplementary File 4. Parameter fits at MOI=0.0001 are visualized in Figure 4-figure supplement 1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F4.medium.gif" width="440" height="326"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F4.medium.gif" width="440" height="326"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <p id="p-24" class="first-child">Best fit parameter estimates for β and ρ or <em>ε</em> from mean-field model fits to MOI=0.001 time series data, atop (A,B) β - ρ and (C) β - <em>ε</em> bifurcation. Fits and bifurcations are grouped by immune phenotype: (A) absent; (B) induced; (C) constitutive immunity, with cell lines differentiated by shape (Vero=circles; RoNi/7.1 = triangles; PaKiT01=squares) and viral infections by color (rVSV-G = green, rVSV-EBOV = magenta, rVSV-MARV = blue). Note that y-axis values are ten­fold higher in panel (C). Branch point curves (solid lines) and Hopf curves (dashed lines) are reproduced from <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>. White space indicates endemic equilibrium (pathogen persistence), gray space indicates limit cycling (virus-induced epidemic extinction), and black space indicates no infection (immune-mediated pathogen extinction). In panel (A) and (B), <em>ε</em> is fixed at 0; in panel (C), ρ is fixed at 5×10<sup>-8</sup> for bifurcation curves and estimated at 4×10<sup>-8</sup> and 8×10<sup>-8</sup> for rVSV-EBOV and rVSV-G parameter points, respectively. Other parameter values were fixed at: <em>b</em> = .025, μ = 0.001, σ = 1/6, α = 1/6, and c = 0 across all panels. Raw fitted values and corresponding 95% confidence intervals for β, ρ, and <em>ε</em>, background parameter values, and AIC recovered from model fit, are reported in Supplementary File 4. Parameter fits at MOI=0.0001 are visualized in <a id="xref-fig-13-2" class="xref-fig" href="#F13">Figure 4-figure supplement 1</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">As anticipated, the immune absent model (a simple target cell model) offered the best fit to IFN-deficient Vero cell infections (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>; <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>; Supplementary File 4). Among Vero cell trials, infections with rVSV-G produced the highest β estimates, followed by infections with rVSV-EBOV and rVSV-MARV. Best fit parameter estimates on Vero cell lines localized in the region of parameter space corresponding to theoretical limit cycles, consistent with observed virus-induced epidemic extinctions in stochastic tissue cultures.</p><p id="p-26">In contrast to Vero cells, the induced immunity model offered the best fit to all RoNi/7.1 data, consistent with reported patterns in the literature and our own validation by qPCR (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>; <a id="xref-fig-11-1" class="xref-fig" href="#F11">Figure 1-figure supplement 6</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Biesold et al. 2011</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Kuzmin et al. 2017</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Arnold et al. 2018</a>; <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Pavlovich et al. 2018</a>). As in Vero cell trials, we estimated highest β values for rVSV-G infections on RoNi/7.1 cell lines but here recovered higher β estimates for rVSV-MARV than for rVSV-EBOV. This reversal was balanced by a higher estimated rate of acquisition to antiviral status (ρ) for rVSV-EBOV versus rVSV-MARV. In general, we observed that more rapid rates of antiviral acquisition (either induced, ρ, constitutive, <em>ε</em>, or both) correlated with higher transmission rates (β). When offset by ρ, β values estimated for RoNi/7.1 infections maintained the same amplitude as those estimated for immune-absent Vero cell lines but caused gentler epidemics and reduced cellular mortality (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a>). RoNi/7.1 parameter estimates localized in the region corresponding to endemic equilibrium for the deterministic, theoretical model (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>), yielding less acute epidemics which nonetheless went extinct in stochastic experiments.</p><p id="p-27">Finally, rVSV-G and rVSV-EBOV trials on PaKiT01 cells were best fit by models assuming constitutive immunity, while rVSV-MARV infections on PaKiT01 were matched equivalently by models assuming either induced or constitutive immunity—with induced models favored over constitutive in AIC comparisons because one fewer parameter was estimated (<a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure 1-figure supplement 4-5</a>; Supplementary File 4). For all virus infections, PaKiT01 cell lines yielded β estimates a full order of magnitude higher than Vero or RoNi/7.1 cells, with each β balanced by an immune response (either ρ, or ρ combined with ε) also an order of magnitude higher than that recovered for the other cell lines (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>; <a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>). As in RoNi/7.1 cells, PaKiT01 parameter fits localized in the region corresponding to endemic equilibrium for the deterministic theoretical model. Because constitutive immune processes can actually prohibit initial pathogen invasion, constitutive immune fits to rVSV-MARV infections on PaKiT01 cell lines consistently localized at or below the Branch point threshold for virus invasion (<em>R<sub>0</sub> =</em> 1). During model fitting for optimization of <em>ε</em>, any parameter tests of <em>ε</em> values producing <em>R<sub>0</sub> &lt; 1</em> resulted in no infection and, consequently, produced an exceedingly poor fit to infectious time series data. In all model fits assuming constitutive immunity, across all cell lines, parameter estimates for ρ and <em>ε</em> traded off, with one parameter optimized at values approximating zero, such that the immune response was modeled as almost entirely induced or entirely constitutive (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>; Supplementary File 4). For RoNi/7.1 cells, even when constitutive immunity was allowed, the immune response was estimated as almost entirely induced, while for rVSV-G and rVSV-EBOV fits on PaKiT01 cells, the immune response optimized as almost entirely constitutive. For rVSV-MARV on PaKiT01 cells, however, estimation of ρ was high under all assumptions, such that any additional antiviral contributions from <em>ε</em> prohibited virus from invading at all. The induced immune model thus produced a more parsimonious recapitulation of these data because virus invasion was always permitted, then rapidly controlled.</p></div><div id="sec-6" class="subsection"><h3>Antiviral cells safeguard live cells against rapid cell mortality to elongate epidemic duration <em>in vitro</em></h3><p id="p-28">In order to compare the relative contributions of each cell line’s disparate immune processes to epidemic dynamics, we next used our mean field parameter estimates to calculate the initial ‘antiviral rate’—the initial accumulation rate of antiviral cells upon virus invasion for each cell-virus-MOI combination—based on the following equation:
<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-13.gif"/></noscript></span></span>
where P<sub>E</sub> was calculated from the initial infectious dose (MOI) of each infection experiment and P<sub>S</sub> was estimated at disease free equilibrium:
<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-14.gif"/></noscript></span></span>
<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-15.gif"/></noscript></span></span></p><p id="p-29">Because ρ and <em>ε</em> both contribute to this initial antiviral rate, induced and constitutive immune assumptions are capable of yielding equally rapid rates, depending on parameter fits. Indeed, under fully induced immune assumptions, the induced antiviral acquisition rate (ρ) estimated for rVSV-MARV infection on PaKiT01 cells was so high that the initial antiviral rate exceeded even that estimated under constitutive assumptions for this cell-virus combination (Supplementary File 4). In reality, we know that NPC1 receptor incompatibilities make PaKiT01 cell lines constitutively refractory to rVSV-MARV infection (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Ng and Chandran 2018</a>) and that PaKiT01 cells also constitutively express the antiviral cytokine, IFN-α. Model fitting results suggest that this constitutive expression of IFN-α may act more as a rapidly inducible immune response following virus invasion than as a constitutive secretion of functional IFN-α protein. Nonetheless, as hypothesized, PaKiT01 cell lines were by far the most antiviral of any in our study—with initial antiviral rates estimated several orders of magnitude higher than any others in our study, under either induced or constitutive assumptions (<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>; Supplementary File 4). RoNi/7.1 cells displayed the second-most-pronounced signature of immunity, followed by Vero cells, for which the initial antiviral rate was essentially zero even under forced assumptions of induced or constitutive immunity (<a id="xref-table-wrap-1-9" class="xref-table" href="#T1">Table 1</a>; Supplementary File 4).</p><p id="p-30">Using fitted parameters for β and <em>ε</em>, we additionally calculated R<sub>0</sub>, the basic reproduction number for the virus, for each cell line-virus-MOI combination (<a id="xref-table-wrap-1-10" class="xref-table" href="#T1">Table 1</a>; Supplementary File 4). We found that R<sub>0</sub> was essentially unchanged across differing immune assumptions for RoNi/7.1 and Vero cells, for which the initial antiviral rate was low. In the case of PaKiT01 cells, a high initial antiviral rate under either induced or constitutive immunity resulted in a correspondingly high estimation of β (and, consequently, R<sub>0</sub>) which still produced the same epidemic curve that resulted from the much lower estimates for β and R<sub>0</sub> paired with absent immunity. These findings suggest that antiviral immune responses protect host tissues against virus-induced cell mortality and may facilitate the establishment of more rapid within-host transmission rates.</p><p id="p-31">Total monolayer destruction occurred in all cell-virus combinations excepting rVSV-EBOV infections on RoNi/7.1 cells and rVSV-EBOV and rVSV-MARV infections on PaKiT01 cells. Monolayer destruction corresponded to susceptible cell depletion and epidemic turnover where R-effective (the product of <em>R<sub>0</sub></em> and the proportion susceptible) was reduced below one (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). For rVSV-EBOV infections on RoNi/7.1, induced antiviral cells safeguarded remnant live cells, which birthed new susceptible cells late in the time series. In rVSV-EBOV and rVSV-MARV infections on PaKiT01 cells, this antiviral protection halted the epidemic (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>; R-effective &lt;1) before susceptibles fully declined. In the case of rVSV-EBOV on PaKiT01, the birth of new susceptibles from remnant live cells protected by antiviral status maintained late-stage transmission to facilitate long-term epidemic persistence. Importantly, under fixed parameter values for the infection incubation rate (σ) and infection-induced mortality rate (a), models were unable to reproduce the longer-term infectious time series captured in data from rVSV-EBOV infections on PaKiT01 cell lines without incorporation of cell births, an assumption adopted in previous modeling representations of IFN-mediated viral dynamics in tissue culture (<a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Howat et al. 2006</a>). In our experiments, we observed that cellular reproduction took place as plaque assays achieved confluency.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Fitted time series of susceptible (green shading) and antiviral (blue shading) cell proportions from the mean field model for rVSV-G, rVSV-EBOV, and rVSV-MARV infections (columns) on Vero, RoNi/7.1, and PaKiT01 cell lines (rows) at MOI=0.001. Results are shown for the best fit immune absent model on Vero cells, induced immunity model on RoNi/7.1 cells and constitutive (rVSV-G and rVSV-EBOV) and induced (rVSV-MARV) immune models on PaKiT01 cells. Combined live, uninfectious cell populations (S + A + E) are shown in tan shading, with raw live, uninfectious cell data from Hoechst stains visualized as open circles. The right-hand y-axis corresponds to R-effective (pink solid line) across each time series; R-effective =1 is a pink dashed, horizontal line. Panel background corresponds to empirical outcome of the average stochastic cell culture trial (persistent infection = white; virus-induced epidemic extinction = gray; immune-mediated epidemic extinction = black). Parameter values are listed in Supplementary File 4 and results for absent/induced/constitutive fitted models across all cell lines in Figure 5-figure supplement 1 (MOI=0.001) and Figure 5-figure supplement 2 (MOI=0.0001)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1488400151" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Fitted time series of susceptible (green shading) and antiviral (blue shading) cell proportions from the mean field model for rVSV-G, rVSV-EBOV, and rVSV-MARV infections (columns) on Vero, RoNi/7.1, and PaKiT01 cell lines (rows) at MOI=0.001. Results are shown for the best fit immune absent model on Vero cells, induced immunity model on RoNi/7.1 cells and constitutive (rVSV-G and rVSV-EBOV) and induced (rVSV-MARV) immune models on PaKiT01 cells. Combined live, uninfectious cell populations (S + A + E) are shown in tan shading, with raw live, uninfectious cell data from Hoechst stains visualized as open circles. The right-hand y-axis corresponds to R-effective (pink solid line) across each time series; R-effective =1 is a pink dashed, horizontal line. Panel background corresponds to empirical outcome of the average stochastic cell culture trial (persistent infection = white; virus-induced epidemic extinction = gray; immune-mediated epidemic extinction = black). Parameter values are listed in Supplementary File 4 and results for absent/induced/constitutive fitted models across all cell lines in Figure 5-figure supplement 1 (MOI=0.001) and Figure 5-figure supplement 2 (MOI=0.0001).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F5.medium.gif" width="440" height="329"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F5.medium.gif" width="440" height="329"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <p id="p-32" class="first-child">Fitted time series of susceptible (green shading) and antiviral (blue shading) cell proportions from the mean field model for rVSV-G, rVSV-EBOV, and rVSV-MARV infections (columns) on Vero, RoNi/7.1, and PaKiT01 cell lines (rows) at MOI=0.001. Results are shown for the best fit immune absent model on Vero cells, induced immunity model on RoNi/7.1 cells and constitutive (rVSV-G and rVSV-EBOV) and induced (rVSV-MARV) immune models on PaKiT01 cells. Combined live, uninfectious cell populations (S + A + E) are shown in tan shading, with raw live, uninfectious cell data from Hoechst stains visualized as open circles. The right-hand y-axis corresponds to R-effective (pink solid line) across each time series; R-effective =1 is a pink dashed, horizontal line. Panel background corresponds to empirical outcome of the average stochastic cell culture trial (persistent infection = white; virus-induced epidemic extinction = gray; immune-mediated epidemic extinction = black). Parameter values are listed in Supplementary File 4 and results for absent/induced/constitutive fitted models across all cell lines in <a id="xref-fig-14-1" class="xref-fig" href="#F14">Figure 5-figure supplement 1</a> (MOI=0.001) and <a id="xref-fig-15-1" class="xref-fig" href="#F15">Figure 5-figure supplement 2</a> (MOI=0.0001).</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">Finally, because the protective effect of antiviral cells is more clearly observable spatially, we confirmed our results by simulating fitted time series in a spatially-explicit, stochastic reconstruction of our mean field model. In spatial simulations, rates of antiviral acquisition were fixed at fitted values for ρ and <em>ε</em> derived from mean field estimates, while transmission rates (β) were fixed at values ten times greater than those estimated under mean field conditions, accounting for the intensification of parameter thresholds permitting pathogen invasion in local spatial interactions (see Materials and Methods; Videos 1-3; <a id="xref-fig-16-1" class="xref-fig" href="#F16">Figure 5-figure supplement 3</a>; Supplementary File 5; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Webb, Keeling, and Boots 2007</a>). In immune capable time series, spatial antiviral cells acted as ‘refugia’ which protected live cells from infection as each initial epidemic wave ‘washed’ across a cell monolayer. Eventual birth of new susceptibles from these living refugia allowed for sustained epidemic transmission in cases where some infectious cells persisted at later timepoints in simulation (Videos 1-3; <a id="xref-fig-16-2" class="xref-fig" href="#F16">Figure 5-figure supplement 3</a>).</p></div></div><div class="section" id="sec-7"><h2 class="">Discussion</h2><p id="p-34">Bats are reservoirs for several important emerging zoonoses but appear not to experience disease from otherwise virulent viral pathogens. Though the molecular biological literature has made great progress in elucidating the mechanisms by which bats tolerate viral infections (<a id="xref-ref-61-6" class="xref-bibr" href="#ref-61">Zhou et al. 2016</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Ahn et al. 2019</a>; <a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Xie et al. 2018</a>; <a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Pavlovich et al. 2018</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Zhang et al. 2013</a>), the impact of unique bat immunity on virus dynamics within-host has not been well-elucidated. We used an innovative combination of <em>in vitro</em> experimentation and within-host modeling to explore the impact of unique bat immunity on virus dynamics. Critically, we found that bat cell lines demonstrated a signature of enhanced interferon-mediated immune response, of either constitutive or induced form, which allowed for establishment of rapid within-host, cell-to-cell virus transmission rates (β). These results were supported by both data-independent bifurcation analysis of our mean field theoretical model, as well as fitting of this model to viral infection time series established in bat cell culture. Additionally, we demonstrated that the antiviral state induced by the interferon pathway protects live cells from mortality in tissue culture, resulting in <em>in vitro</em> epidemics of extended duration that enhance that probability of establishing a long-term persistent infection. Our findings suggest that viruses evolved in bat reservoirs possessing enhanced IFN capabilities could achieve more rapid within-host transmission rates without causing pathology to their hosts. Such rapidly-reproducing viruses would likely generate extreme virulence upon spillover to hosts lacking similar immune capacities to bats.</p><p id="p-35">To achieve these results, we first developed a novel, within-host, theoretical model elucidating the effects of unique bat immunity, then undertook bifurcation analysis of the model’s equilibrium properties under immune absent, induced, and constitutive assumptions. We considered a cell line to be constitutively immune if possessing any number of antiviral cells at disease free equilibrium but allowed the extent of constitutive immunity to vary across the parameter range for <em>ε</em>, the constitutive rate of antiviral acquisition. In deriving the equation for <em>R<sub>0</sub>,</em> the basic reproduction number, which defines threshold conditions for virus invasion of a tissue <em>(R<sub>0</sub> &gt;</em> 1), we demonstrated how the invasion threshold is elevated at high values of constitutive antiviral acquisition, <em>ε.</em> Constitutive immune processes can thus prohibit pathogen invasion, while induced responses, by definition, can only control infections <em>post-hoc</em>. Once thresholds for pathogen invasion have been met, assumptions of constitutive immunity will limit the cellular mortality (virulence) incurred at high transmission rates. Regardless of mechanism (induced or constitutive), interferon-stimulated antiviral cells appear to play a key role in maintaining longer term or persistent infections by safeguarding susceptible cells from rapid infection and concomitant cell death.</p><p id="p-36">Fitting of our model to <em>in vitro</em> data supported expected immune phenotypes for different bat cell lines as described in the literature. Simple target cell models that ignore the effects of immunity best recapitulated infectious time series derived from IFN-deficient Vero cells, while models assuming induced immune processes most accurately reproduced trials derived from RoNi/7.1 (<em>Rousettus aegyptiacus</em>) cells, which possesses a standard virus-induced IFN-response. In most cases, models assuming constitutive immune processes best recreated virus epidemics produced on PaKiT01 (<em>Pteropus alecto</em>) cells, which are known to constitutively express the antiviral cytokine, IFN-α (<a id="xref-ref-61-7" class="xref-bibr" href="#ref-61">Zhou et al. 2016</a>). Model support for induced immune assumptions in fits to rVSV-MARV infections on PaKiT01cells suggests that the constitutive IFN-α expression characteristic of <em>P. alecto</em> cells may represent more of a constitutive immune priming process than a perpetual, functional, antiviral defense. Results from mean field model fitting were additionally confirmed in spatially explicit stochastic simulations of each time series.</p><p id="p-37">As previously demonstrated in within-host models for HIV (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Coffin 1995</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Perelson et al. 1996</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Nowak et al. 1995</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bonhoeffer et al. 1997</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Ho et al. 1995</a>), assumptions of simple target-cell depletion can often provide satisfactory approximations of viral dynamics, especially those reproduced in simple <em>in vitro</em> systems. Critically, our model fitting emphasizes the need for incorporation of top-down effects of immune control in order to accurately reproduce infectious time series derived from bat cell tissue cultures, especially those resulting from the robustly antiviral PaKiT01 <em>P. alecto</em> cell line. These findings indicate that enhanced IFN-mediated immune pathways in bat reservoirs may promote elevated within-host virus replication rates prior to cross-species emergence. We nonetheless acknowledge the limitations imposed by <em>in vitro</em> experiments in tissue culture, especially involving recombinant viruses and immortalized cell lines. Future work should extend these cell culture studies to include measurements of multiple state variables (i.e. antiviral cells) to enhance epidemiological inference.</p><p id="p-38">The continued recurrence of Ebola epidemics across central Africa highlights the importance of understanding bats’ roles as reservoirs for virulent zoonotic disease. The past decade has born witness to emerging consensus regarding the unique pathways by which bats resist and tolerate highly virulent infections (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Brook and Dobson 2015</a>; <a id="xref-ref-59-4" class="xref-bibr" href="#ref-59">Xie et al. 2018</a>; <a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Zhang et al. 2013</a>; <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Ahn et al. 2019</a>; <a id="xref-ref-61-8" class="xref-bibr" href="#ref-61">Zhou et al. 2016</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Ng et al. 2015</a>; <a id="xref-ref-40-5" class="xref-bibr" href="#ref-40">Pavlovich et al. 2018</a>). Nonetheless, an understanding of the mechanisms by which bats support endemic pathogens at the population level, or promote the evolution of virulent pathogens at the individual level, remains elusive. Endemic maintenance of infection is a defining characteristic of a pathogen reservoir (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Haydon et al. 2002</a>), and bats appear to merit such a title, supporting long-term persistence of highly transmissible viral infections in isolated island populations well below expected critical community sizes (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Peel et al. 2012</a>). Researchers debate the relative influence of population-level and within-host mechanisms which might explain these trends (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Plowright et al. 2016</a>), but increasingly, field data are difficult to reconcile without acknowledgement of a role for persistent infections (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Peel et al. 2018</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Brook et al. 2019</a>). We present general methods to study cross-scale viral dynamics, which suggest that within-host persistence is supported by robust antiviral responses characteristic of bat immune processes. Viruses which evolve rapid replication rates under these robust antiviral defenses may pose the greatest hazard for cross-species pathogen emergence into spillover hosts with immune systems that differ from those unique to bats.</p></div><div class="section" id="sec-8"><h2 class="">Materials and Methods</h2><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1135251/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/1135251/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div></div><div id="sec-9" class="subsection"><h3>Cell Culture Experiments</h3><div id="sec-10" class="subsection"><h4>Cells</h4><p id="p-39">All experiments were carried out on three immortalized mammalian kidney cell lines: Vero (African green monkey), RoNi/7.1 (<em>Rousettus aegyptiacus</em>) (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kühl et al. 2011</a>; <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Biesold et al. 2011</a>) and PaKiT01 (<em>Pteropus alecto</em>) (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Crameri et al. 2009</a>). The species identification of all bat cell lines were confirmed morphologically and genetically in the publications in which they were originally described (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Kühl et al. 2011</a>; <a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Biesold et al. 2011</a>; <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Crameri et al. 2009</a>). Vero cells were obtained from ATCC.</p><p id="p-40">Monolayers of each cell line were grown to 90% confluency (∼9 x 10<sup>5</sup> cells) in 6-well plates. Cells were maintained in a humidified 37°C, 5% CO<sub>2</sub> incubator and cultured in Dulbecco’s modified Eagle medium (DMEM) (Life Technologies, Grand Island, NY), supplemented with 2% fetal bovine serum (FBS) (Gemini Bio Products, West Sacramento, CA), and 1% penicillin-streptomycin (Life Technologies). Cells were tested monthly for mycoplasma contamination while experiments were taking place; all cells assayed negative for contamination at every testing.</p><p id="p-41">Previous work has demonstrated that all cell lines used are capable of mounting a type I IFN response upon viral challenge, with the exception of Vero cells, which possess an IFN-β deficiency (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Desmyter, Melnick, and Rawls 1968</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Rhim et al. 1969</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Emeny and Morgan 1979</a>). RoNi/7.1 cells have been shown to mount idiosyncratic induced IFN defenses upon viral infection (<a id="xref-ref-40-6" class="xref-bibr" href="#ref-40">Pavlovich et al. 2018</a>; <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Kuzmin et al. 2017</a>; <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Arnold et al. 2018</a>; <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Kühl et al. 2011</a>; <a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Biesold et al. 2011</a>), while PaKiT01 cells are known to constitutively express the antiviral cytokine, IFN-α (<a id="xref-ref-61-9" class="xref-bibr" href="#ref-61">Zhou et al. 2016</a>). This work is the first documentation of IFN signaling induced upon challenge with the particular recombinant VSVs outlined below. We verified known antiviral immune phenotypes via qPCR. Results were consistent with the literature, indicating a less pronounced role for interferon defense against viral infection in RoNi/7.1 versus PaKiT01 cells.</p></div><div id="sec-11" class="subsection"><h4>Viruses</h4><p id="p-42">Replication-capable recombinant vesicular stomatitis Indiana viruses, expressing filovirus glycoproteins in place of wild type G (rVSV-G, rVSV-EBOV, and rVSV-MARV) have been previously described (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Wong et al. 2010</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Miller et al. 2012</a>). Viruses were selected to represent a broad range of anticipated antiviral responses from host cells, based on a range of past evolutionary histories between the virus glycoprotein mediating cell entry and the host cell’s entry receptor. These interactions ranged from the total absence of evolutionary history in the case of rVSV-G infections on all cell lines to a known receptor-level cell entry incompatibility in the case of rVSV-MARV infections on PaKiT01 cell lines.</p><p id="p-43">To measure infectivities of rVSVs on each of the cell lines outlined above, so as to calculate the correct viral dose for each MOI, NH<sub>4</sub>Cl (20 mM) was added to infected cell cultures at 1-2 hours post-infection to block viral spread, and individual eGFP-positive cells were manually counted at 12-14 hours post-infection.</p></div><div id="sec-12" class="subsection"><h4>Innate Immune Phenotypes via qPCR of IFN Genes</h4><p id="p-44">Previously published work indicates that immortalized kidney cell lines of <em>Rousettus aegyptiacus</em> (RoNi/7.1) and <em>Pteropus alecto</em> (PaKiT01) exhibit different innate antiviral immune phenotypes through, respectively, induced (<a id="xref-ref-4-6" class="xref-bibr" href="#ref-4">Biesold et al. 2011</a>; <a id="xref-ref-40-7" class="xref-bibr" href="#ref-40">Pavlovich et al. 2018</a>; <a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Kühl et al. 2011</a>; <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Arnold et al. 2018</a>) and constitutive (<a id="xref-ref-61-10" class="xref-bibr" href="#ref-61">Zhou et al. 2016</a>) expression of type I interferon genes. We verified these published phenotypes on our own cell lines infected with rVSV-G, rVSV-EBOV, and rVSV-MARV via qPCR of IFN-α and IFN-β genes across a longitudinal time series of infection.</p><p id="p-45">Specifically, we carried out multiple time series of infection of each cell line with each of the viruses described above, under mock infection conditions and at MOIs of 0.0001 and 0.001—with the exception of rVSV-MARV on PaKiT01 cell lines, for which infection was only performed at MOI=0.0001 due to limited viral stocks and the extremely low infectivity of this virus on this cell line (thus requiring high viral loads for initial infection). All experiments were run in duplicate on 6-well plates, such that a typical plate for any of the three viruses had two control (mock) wells, two MOI=0.0001 wells and two MOI=0.001 wells, excepting PaKiT01 plates, which had two control and four MOI=0.0001 wells at a given time. We justify this PaKiT01 exemption through the expectation that IFN-α expression is constitutive for these cells, and by the assumption that any expression exhibited at the lower MOI should also be present at the higher MOI.</p><p id="p-46">For these gene expression time series, four 6-well plates for each cell line-virus combination were incubated with virus for one hour at 37°C. Following incubation, virus was aspirated off, and cell monolayers were washed in PBS, then covered with an agar plaque assay overlay to mimic conditions under which infection trials were run. Plates were then harvested sequentially at timepoints of roughly 5, 10, 15, and 20 hours post-infection (exact timing varied as multiple trials were running simultaneously). Upon harvest of each plate, agar overlay was removed, and virus was lysed and RNA extracted from cells using the Zymo Quick RNA Mini Prep kit, according to the manufacturer’s instructions and including the step for cellular DNA digestion. Post-extraction, RNA quality was verified via nanodrop, and RNA was converted to cDNA using the Invitrogen Superscript III cDNA synthesis kit, according to the manufacturer’s instructions. cDNA was then stored at 4°C and as a frozen stock at -20°C to await qPCR.</p><p id="p-47">We undertook qPCR of cDNA to assess expression of the type I interferon genes, IFN-α and IFN-β, and the housekeeping gene, β-Actin, using primers previously reported in the literature (Supplementary File 6). For qPCR, 2ul of each cDNA sample was incubated with 7ul of deionized water, 1ul of 5UM forward/reverse primer mix and 10ul of iTaq Universal SYBR Green, then cycled on a QuantStudio3 Real-Time PCR machine under the following conditions: initial denaturation at 94°C for 2 min followed by 40 cycles of: denaturation at 95°C (5 sec), annealing at 58°C (15 sec), and extension at 72°C (10 sec).</p><p id="p-48">We report simple δ-Ct values for each run, with raw Ct of the target gene of interest (IFN-α or IFN-β) subtracted from raw Ct of the β-Actin housekeeping gene in <a id="xref-fig-11-2" class="xref-fig" href="#F11">Figure 1-figure supplement 6</a>. Calculation of fold change upon viral infection in comparison to mock using the δ-δ-Ct method (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Livak and Schmittgen 2001</a>) was inappropriate in this case, as we wished to demonstrate constitutive expression of IFN-α in PaKiT01 cells, whereby data from mock cells was identical to that produced from infected cells.</p></div><div id="sec-13" class="subsection"><h4>Plaque Assays and Time Series Imaging</h4><p id="p-49">After being grown to ∼90% confluency, cells were incubated with pelleted rVSVs expressing eGFP (rVSV-G, rVSV-EBOV, rVSV-MARV). Cell lines were challenged with both a low (0.0001) and high (0.001) multiplicity of infection (MOI) for each virus. In a cell monolayer infected at a given MOI (m), the proportion of cells (P), infected by k viral particles can be described by the Poisson distribution: <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/inline-graphic-1.gif"/></noscript></span></span>, such that the number of initially infected cells in an experiment equals: <em>1 — e<sup>-m</sup></em>. We assumed that a ∼90% confluent culture at each trial’s origin was comprised of ∼9×10<sup>5</sup> cells and conducted all experiments at MOIs of 0.0001 and 0.001, meaning that we began each trial by introducing virus to, respectively, ∼81 or 810 cells, representing the state variable ‘E’ in our theoretical model. Low MOIs were selected to best approximate the dynamics of mean field infection and limit artifacts of spatial structuring, such as premature epidemic extinction when growing plaques collide with plate walls in cell culture.</p><p id="p-50">Six well plates were prepared with each infection in duplicate or triplicate, such that a control well (no virus) and 2-3 wells each at MOI 0.001 and 0.0001 were incubated simultaneously on the same plate. In total, we ran between 18-39 trials at each cell-virus-MOI combination, excepting r-VSV-MARV infections on PaKiT01 cells at MOI=0.001, for which we ran only 8 trials due to the low infectivity of this virus on this cell line, which required high viral loads for initial infection. Cells were incubated with virus for one hour at <em>37°C.</em> Following incubation, virus was aspirated off, and cell monolayers were washed in PBS, then covered with a molten viscous overlay (50% 2X MEM/L-glutamine; 5% FBS; 3% HEPES; 42% agarose), cooled for 20 minutes, and re-incubated in their original humidified 37°C, 5% CO<sub>2</sub> environment.</p><p id="p-51">After application of the overlay, plates were monitored periodically using an inverted fluorescence microscope until the first signs of GFP expression were witnessed (∼6-9.5 hours post-infection, depending on the cell line and virus under investigation). From that time forward, a square subset of the center of each well (comprised of either 64- or 36-subframes and corresponding to roughly 60 and 40% of the entire well space) was imaged periodically, using a CellInsight CX5 High Content Screening (HCS) Platform with a 4X air objective (ThermoFisher, Inc., Waltham, MA). Microscope settings were held standard across all trials, with exposure time fixed at 0.0006 sec for each image. One color channel was imaged, such that images produced show GFP-expressing cells in white and non-GFP-expressing cells in black (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 1-figure supplement 1</a>).</p><p id="p-52">Wells were photographed in rotation, as frequently as possible, from the onset of GFP expression until the time that the majority of cells in the well were surmised to be dead, GFP expression could no longer be detected, or early termination was desired to permit Hoechst staining.</p><p id="p-53">In the case of PaKiT01 cells infected with rVSV-EBOV, where an apparently persistent infection established, the assay was terminated after 200+ hours (8+ days) of continuous observation. Upon termination of all trials, cells were fixed in formaldehyde (4% for 15 min), incubated with Hoechst stain (0.0005% for 15 min) (ThermoFisher, Inc., Waltham, MA), then imaged at 4X on the CellInsight CX5 High Content Screening (HCS) Platform. The machine was allowed to find optimal focus for each Hoechst stain image. One color channel was permitted such that images produced showed live nuclei in white and dead cells in black.</p></div><div id="sec-14" class="subsection"><h4>Hoechst Staining</h4><p id="p-54">Hoechst stain colors cellular DNA, and viral infection is thought to interfere with the clarity of the stain (Dembowski and DeLuca, 2015). As such, infection termination, cell fixation, and Hoechst staining enables generation of a rough time series of uninfectious live cells (i.e. susceptible + antiviral cells) to complement the images which produced time series of proportions infectious. Due to uncertainty over the exact epidemic state of Hoechst-stained cells (<em>i.e.</em> exposed but not yet infectious cells may still stain), we elected to fit our models only to the infectious time series derived from GFP-expressing images and used Hoechst stain images as a <em>post hoc</em> visual check on our fit only (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>; <a id="xref-fig-14-2" class="xref-fig" href="#F14">Figure 5-figure supplement 1-2</a>).</p></div><div id="sec-15" class="subsection"><h4>Image Processing</h4><p id="p-55">Images recovered from the time series above were processed into binary (‘infectious’ vs. ‘non-infectious’ or, for Hoechst-stained images, ‘live’ vs. ‘dead’) form using the EBImage package (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Pau et al. 2010</a>) in R version 3.6 for MacIntosh, after methods further detailed in Supplementary File 7. Binary images were then further processed into time series of infectious or, for Hoechst-stained images, live cells using a series of cell counting scripts. Because of logistical constraints (i.e. many plates of simultaneously running infection trials and only one available imaging microscope), the time course of imaging across the duration of each trial was quite variable. As such, we fitted a series of statistical models to our processed image data to reconstruct reliable values of the infectious proportion of each well per hour for each distinct trial in all cell line-virus-MOI combinations (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 1-figure supplement 2-3</a>), as well as for declining live cell counts from control well data derived from the Hoestch time series (Supplementary File 1; Supplementary File 7; <a id="xref-fig-12-1" class="xref-fig" href="#F12">Figure 1-figure supplement 7</a>). All original and processed images, image processing and counting code, and resulting time series data are freely available for download at the following FigShare repository: DOI: 10.6084/m9.figshare.8312807.</p></div></div><div id="sec-16" class="subsection"><h3>Mean Field Model</h3><div id="sec-17" class="subsection"><h4>Theoretical Model Details</h4><p id="p-56">To derive the expression for <em>R<sub>0</sub></em>, the basic pathogen reproductive number <em>in vitro</em>, we used Next Generation Matrix (NGM) techniques (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Diekmann, Heesterbeek, and Metz 1990</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Heffernan, Smith, and Wahl 2005</a>), employing Wolfram Mathematica (version 11.2) as an analytical tool. <em>R</em><sub>0</sub> describes the number of new infections generated by an existing infection in a completely susceptible host population; a pathogen will invade a population when <em>R</em><sub>0</sub> &gt; 1 (Supplementary File 2). We then analyzed stability properties of the system, exploring dynamics across a range of parameter spaces, using MatCont (version 2.2) (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dhooge et al. 2008</a>) for Matlab (version R2018a) (Supplementary File 3).</p></div><div id="sec-18" class="subsection"><h4>Theoretical Model Fitting</h4><p id="p-57">The birth rate, <em>b</em>, and natural mortality rate, μ, balance to yield a population-level growth rate, such that it is impossible to estimate both <em>b</em> and μ simultaneously from total population size data alone. As such, we fixed <em>b</em> at .025 and estimated μ by fitting an infection-absent version of our mean field model to the susceptible time series derived via Hoechst staining of control wells for each of the three cell lines (<a id="xref-fig-12-2" class="xref-fig" href="#F12">Figure 1-figure supplement 7</a>). This yielded a natural mortality rate, μ, corresponding to a lifespan of approximately 121, 191, and 84 hours, respectively, for Vero, RoNi/7.1, and PaKiT01 cell lines (<a id="xref-fig-12-3" class="xref-fig" href="#F12">Figure 1-figure supplement 7</a>). We then fixed the virus incubation rate, σ, as the inverse of the shortest observed duration of time from initial infection to the observation of the first infectious cells via fluorescent microscope for all nine cell line - virus combinations (ranging 6 to 9.5 hours). We fixed a, the infection-induced mortality rate, at <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/inline-graphic-2.gif"/></noscript></span></span>, an accepted standard for general viral kinetics (<a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">Howat et al. 2006</a>), and held <em>c</em>, the rate of antiviral cell regression to susceptible status, at 0 for the timespan (&lt;200 hours) of the experimental cell line infection trials.</p><p id="p-58">We estimated cell line-virus-MOI-specific values for β, ρ, and ε by fitting the deterministic output of infectious proportions in our mean field model to the full suite of statistical outputs of all trials for each infected cell culture time series (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 1-figure supplement 2-3</a>). Fitting was performed by minimizing the sum of squared differences between the deterministic model output and cell line-virus-MOI-specific infectious proportion of the data at each timestep. We optimized parameters for MOI = 0.001 and 0.0001 simultaneously to leverage statistical power across the two datasets, estimating a different transmission rate, β, for trials run at each infectious dose but, where applicable, estimating the same rates of ρ and ε across the two time series. We used the differential equation solver lsoda() in the R package deSolve (Soetaert, Petzoldt, and Setzer 2010) to obtain numerical solutions for the mean field model and carried out minimization using the ‘Nelder-Mead’ algorithm of the optim() function in base R. All model fits were conducted using consistent starting guesses for the parameters, β (β= 3), and where applicable, ρ (ρ = 0.001) and ε (ε = 0.001). In the case of failed fits or indefinite hessians, we generated a series of random guesses around the starting conditions and continued estimation until successful fits were achieved.</p><p id="p-59">All eighteen cell line-virus-MOI combinations of data were fit by an immune absent (ε = ρ = 0) version of the theoretical model and, subsequently, an induced immunity (ε = 0; ρ &gt; 0) and constitutive immunity (ε &gt; 0; ρ &gt; 0) version of the model. Finally, we compared fits across each cell line-virus-MOI combination via AIC. In calculating AIC, the number of fitted parameters in each model (<em>k</em>) varied across the immune phenotypes, with one parameter (β) estimated for absent immune assumptions, two (β and ρ) for induced immune assumptions, and three (β, ρ, and ε) for constitutive immune assumptions. The sample size <em>(n)</em> corresponded to the number of discrete time steps across all empirical infectious trials to which the model was fitted for each cell-line virus combination. All fitting and model comparison script is freely available for download at the following FigShare repository: DOI: 685 10.6084/m9.figshare.8312807.</p></div><div id="sec-19" class="subsection"><h4>Spatial Model Simulations</h4><p id="p-60">Finally, we verified all mean field fits in a spatial context, in order to more thoroughly elucidate the role of antiviral cells in each time series. We constructed our spatial model in C++ implemented in R using the packages Rcpp and RcppArmadillo (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Eddelbuettel and Francois 2011</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Eddelbuettel and Sanderson 2017</a>). Following Nagai and Honda (2001) and Howat et al. (2006), we modeled this system on a two-dimensional hexagonal lattice, using a ten-minute epidemic timestep for cell state transitions. At the initialization of each simulation, we randomly assigned a duration of natural lifespan, incubation period, infectivity period, and time from antiviral to susceptible status to all cells in a theoretical monolayer. Parameter durations were drawn from a normal distribution centered at the inverse of the respective fixed rates of μ, σ, α, and <em>c</em>, as reported with our mean field model. Transitions involving the induced (ρ) and constitutive (ε) rates of antiviral acquisition were governed probabilistically and adjusted dynamically at each timestep based on the global environment. As such, we fixed these parameters at the same values estimated in the mean field model, and multiplied both ρ and ε by the global proportion of, respectively, exposed and susceptible cells at a given timestep.</p><p id="p-61">In contrast to antiviral acquisition rates, transitions involving the birth rate (<em>b</em>) and the transmission rate (β) occurred probabilistically based on each cell’s local environment. The birth rate, <em>b</em>, was multiplied by the proportion of susceptible cells within a six-neighbor circumference of a focal dead cell, while β was multiplied by the proportion of infectious cells within a thirty-six neighbor vicinity of a focal susceptible cell, thus allowing viral transmission to extend beyond the immediate nearest-neighbor boundaries of an infectious cell. To compensate for higher thresholds to cellular persistence and virus invasion which occur under local spatial conditions (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Webb, Keeling, and Boots 2007</a>), we increased the birth rate, <em>b</em>, and the cell-to-cell transmission rate, β, respectively, to six and ten times the values used in the mean field model (Supplementary File 4). We derived these increases based on the assumption that births took place exclusively based on pairwise nearest-neighbor interactions (the six immediately adjacent cells to a focal dead cell), while viral transmission was locally concentrated but included a small (7.5%) global contribution, representing the thirty-six cell surrounding vicinity of a focal susceptible. We justify these increases and derive their origins further in Supplementary File 5.</p><p id="p-62">We simulated ten stochastic spatial time series for all cell-virus combinations under all three immune assumptions at a population size of 10,000 cells and compared model output with data in <a id="xref-fig-16-3" class="xref-fig" href="#F16">Figure 5-figure supplement 3</a>. Spatial model code is available for public access at the following FigShare repository: DOI: 10.6084/m9.figshare.8312807.</p></div></div></div><div class="section" id="sec-20"><h2 class="">Figure Supplement Captions</h2><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Cell culture models of viral propagation. (A), (B), and (C) show raw, original images of rVSV-EBOV propagation across Vero cell lines at, respectively, 17, 21, and 28 hours post-infection (timesteps 2, 3, and 5 from trial Ver6_B1). (D), (E), and (F) show corresponding, binary images processed in the R package, EBImage. Cells expressing viral eGFP are depicted in white and uninfected/dead cells in black." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1488400151" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Cell culture models of viral propagation. (A), (B), and (C) show raw, original images of rVSV-EBOV propagation across Vero cell lines at, respectively, 17, 21, and 28 hours post-infection (timesteps 2, 3, and 5 from trial Ver6_B1). (D), (E), and (F) show corresponding, binary images processed in the R package, EBImage. Cells expressing viral eGFP are depicted in white and uninfected/dead cells in black.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1 - figure supplement 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F6.medium.gif" width="440" height="297"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1 - figure supplement 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F6.medium.gif" width="440" height="297"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1 - figure supplement 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 1 - figure supplement 1.</span> <p id="p-64" class="first-child">Cell culture models of viral propagation. (A), (B), and (C) show raw, original images of rVSV-EBOV propagation across Vero cell lines at, respectively, 17, 21, and 28 hours post-infection (timesteps 2, 3, and 5 from trial Ver6_B1). (D), (E), and (F) show corresponding, binary images processed in the R package, EBImage. Cells expressing viral eGFP are depicted in white and uninfected/dead cells in black.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Time series data to which mean field mechanistic models were fit, across rVSV-G (left), rVSV-EBOV (middle), and rVSV-MARV (right) infections on Vero, RoNi/7.1, and PaKiT01 cell lines, at MOI=0.001. Open circles show raw data across all trials, while red, dashed line gives the statistical mean of each trials, established from GAM model incorporating random effects per trial. Results for MOI=0.0001 are shown in Figure 1 - figure supplement 3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1488400151" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Time series data to which mean field mechanistic models were fit, across rVSV-G (left), rVSV-EBOV (middle), and rVSV-MARV (right) infections on Vero, RoNi/7.1, and PaKiT01 cell lines, at MOI=0.001. Open circles show raw data across all trials, while red, dashed line gives the statistical mean of each trials, established from GAM model incorporating random effects per trial. Results for MOI=0.0001 are shown in Figure 1 - figure supplement 3.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1 - figure supplement 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F7.medium.gif" width="440" height="328"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1 - figure supplement 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F7.medium.gif" width="440" height="328"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1 - figure supplement 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 1 - figure supplement 2.</span> <p id="p-65" class="first-child">Time series data to which mean field mechanistic models were fit, across rVSV-G (left), rVSV-EBOV (middle), and rVSV-MARV (right) infections on Vero, RoNi/7.1, and PaKiT01 cell lines, at MOI=0.001. Open circles show raw data across all trials, while red, dashed line gives the statistical mean of each trials, established from GAM model incorporating random effects per trial. Results for MOI=0.0001 are shown in <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 1 - figure supplement 3</a>.</p><div class="sb-div caption-clear"></div></div></div><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Time series data to which mean field mechanistic models were fit, across rVSV-G (left), rVSV-EBOV (middle), and rVSV-MARV (right) infections on Vero, oNi/7.1, and PaKiT01 cell lines, at MOI=0.0001. Open circles show raw data across all trials, while red, dashed line gives the statistical mean of each trials, established from GAM model incorporating random effects per trial. Results for MOI=0.001 are shown in Figure 1 - figure supplement 2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1488400151" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Time series data to which mean field mechanistic models were fit, across rVSV-G (left), rVSV-EBOV (middle), and rVSV-MARV (right) infections on Vero, oNi/7.1, and PaKiT01 cell lines, at MOI=0.0001. Open circles show raw data across all trials, while red, dashed line gives the statistical mean of each trials, established from GAM model incorporating random effects per trial. Results for MOI=0.001 are shown in Figure 1 - figure supplement 2.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1 - figure supplement 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F8.medium.gif" width="440" height="328"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1 - figure supplement 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F8.medium.gif" width="440" height="328"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1 - figure supplement 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 1 - figure supplement 3.</span> <p id="p-66" class="first-child">Time series data to which mean field mechanistic models were fit, across rVSV-G (left), rVSV-EBOV (middle), and rVSV-MARV (right) infections on Vero, oNi/7.1, and PaKiT01 cell lines, at MOI=0.0001. Open circles show raw data across all trials, while red, dashed line gives the statistical mean of each trials, established from GAM model incorporating random effects per trial. Results for MOI=0.001 are shown in <a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 1 - figure supplement 2</a>.</p><div class="sb-div caption-clear"></div></div></div><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Figure replicates Figure 1 (main text) but includes all output across mean field model fits assuming (A) absent immunity, (B) induced immunity, and (C) constitutive immunity. Figure shows fitted time series of infectious cell proportions for rVSV-G, rVSV-EBOV, and rVSV-MARV infections (columns) on Vero, RoNi/7.1, and PaKiT01 cell lines (rows) at MOI=0.001. Raw data across all trials are shown as open circles and model output as the solid crimson line (95% confidence intervals by standard error = red shading). Panel background corresponds to empirical outcome of the average stochastic cell culture trial (persistent infection = white; virus-induced epidemic extinction = gray; immune-mediated epidemic extinction = black)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1488400151" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Figure replicates Figure 1 (main text) but includes all output across mean field model fits assuming (A) absent immunity, (B) induced immunity, and (C) constitutive immunity. Figure shows fitted time series of infectious cell proportions for rVSV-G, rVSV-EBOV, and rVSV-MARV infections (columns) on Vero, RoNi/7.1, and PaKiT01 cell lines (rows) at MOI=0.001. Raw data across all trials are shown as open circles and model output as the solid crimson line (95% confidence intervals by standard error = red shading). Panel background corresponds to empirical outcome of the average stochastic cell culture trial (persistent infection = white; virus-induced epidemic extinction = gray; immune-mediated epidemic extinction = black).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1 - figure supplement 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F9.medium.gif" width="368" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1 - figure supplement 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F9.medium.gif" width="368" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1 - figure supplement 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 1 - figure supplement 4.</span> <p id="p-67" class="first-child">Figure replicates <a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a> (main text) but includes all output across mean field model fits assuming (A) absent immunity, (B) induced immunity, and (C) constitutive immunity. Figure shows fitted time series of infectious cell proportions for rVSV-G, rVSV-EBOV, and rVSV-MARV infections (columns) on Vero, RoNi/7.1, and PaKiT01 cell lines (rows) at MOI=0.001. Raw data across all trials are shown as open circles and model output as the solid crimson line (95% confidence intervals by standard error = red shading). Panel background corresponds to empirical outcome of the average stochastic cell culture trial (persistent infection = white; virus-induced epidemic extinction = gray; immune-mediated epidemic extinction = black).</p><div class="sb-div caption-clear"></div></div></div><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Figure replicates Figure 1 - figure supplement 4 exactly but shows model fits and data for all cell-virus combinations at MOI=0.0001." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1488400151" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Figure replicates Figure 1 - figure supplement 4 exactly but shows model fits and data for all cell-virus combinations at MOI=0.0001.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1 - figure supplement 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F10.medium.gif" width="368" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1 - figure supplement 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F10.medium.gif" width="368" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1 - figure supplement 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 1 - figure supplement 5.</span> <p id="p-68" class="first-child">Figure replicates <a id="xref-fig-9-3" class="xref-fig" href="#F9">Figure 1 - figure supplement 4</a> exactly but shows model fits and data for all cell-virus combinations at MOI=0.0001.</p><div class="sb-div caption-clear"></div></div></div><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) IFN-&#x3B1; and (B) IFN-&#x3B2; gene expression profiles from qPCR for rVSV infections on RoNi/7.1 and PaKiT01 cell lines. Panels show &#x3B4;-Ct (raw Ct of IFN gene assay subtracted from raw Ct of &#x3B2;-Actin housekeeping gene assay) across a time series for mock (left), MOI=0.0001 (middle) and MOI=0.001 (right) infections across a time series. Viruses are represented by color (rVSV-G=green, rVSV-EBOV=magenta, rVSV-MARV=blue). The red dashed line at &#x3B4;-Ct=37 corresponds to no expression; higher expression is indicated at lower values for &#x3B4;-Ct. qPCR was carried out using primers summarized in Supplementary File 6." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1488400151" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) IFN-&#x3B1; and (B) IFN-&#x3B2; gene expression profiles from qPCR for rVSV infections on RoNi/7.1 and PaKiT01 cell lines. Panels show &#x3B4;-Ct (raw Ct of IFN gene assay subtracted from raw Ct of &#x3B2;-Actin housekeeping gene assay) across a time series for mock (left), MOI=0.0001 (middle) and MOI=0.001 (right) infections across a time series. Viruses are represented by color (rVSV-G=green, rVSV-EBOV=magenta, rVSV-MARV=blue). The red dashed line at &#x3B4;-Ct=37 corresponds to no expression; higher expression is indicated at lower values for &#x3B4;-Ct. qPCR was carried out using primers summarized in Supplementary File 6.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1 - figure supplement 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F11.medium.gif" width="318" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1 - figure supplement 6." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F11.medium.gif" width="318" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1 - figure supplement 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 1 - figure supplement 6.</span> <p id="p-69" class="first-child">(A) IFN-α and (B) IFN-β gene expression profiles from qPCR for rVSV infections on RoNi/7.1 and PaKiT01 cell lines. Panels show δ-Ct (raw Ct of IFN gene assay subtracted from raw Ct of β-Actin housekeeping gene assay) across a time series for mock (left), MOI=0.0001 (middle) and MOI=0.001 (right) infections across a time series. Viruses are represented by color (rVSV-G=green, rVSV-EBOV=magenta, rVSV-MARV=blue). The red dashed line at δ-Ct=37 corresponds to no expression; higher expression is indicated at lower values for δ-Ct. qPCR was carried out using primers summarized in Supplementary File 6.</p><div class="sb-div caption-clear"></div></div></div><div id="F12" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F12.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Curve fits to control data for standard birth (b =.025) and natural mortality (&#x3BC; =  hours for, respectively, Vero, RoNi/7.1, and PaKiT01 cell lines) rates across all three cell lines. Raw data from multiple trials are shown as open circles, statistical means as dashed black lines, with the output from the mean field model, using the fixed birth rate and estimated mortality rate, in solid green." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1488400151" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Curve fits to control data for standard birth (&lt;em&gt;b&lt;/em&gt; =.025) and natural mortality (&lt;em&gt;&#x3BC;&lt;/em&gt; =  hours for, respectively, Vero, RoNi/7.1, and PaKiT01 cell lines) rates across all three cell lines. Raw data from multiple trials are shown as open circles, statistical means as dashed black lines, with the output from the mean field model, using the fixed birth rate and estimated mortality rate, in solid green.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1 - figure supplement 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F12.medium.gif" width="343" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1 - figure supplement 7." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F12.medium.gif" width="343" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1 - figure supplement 7." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 1 - figure supplement 7.</span> <p id="p-70" class="first-child">Curve fits to control data for standard birth (<em>b</em> =.025) and natural mortality (<em>μ</em> = <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/F12/embed/inline-graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/F12/embed/inline-graphic-3.gif"/></noscript></span></span> hours for, respectively, Vero, RoNi/7.1, and PaKiT01 cell lines) rates across all three cell lines. Raw data from multiple trials are shown as open circles, statistical means as dashed black lines, with the output from the mean field model, using the fixed birth rate and estimated mortality rate, in solid green.</p><div class="sb-div caption-clear"></div></div></div><div id="F13" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F13.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Best fit parameter estimates for &#x3B2; and &#x3C1; or &#x3B5; from mean-field model fits to MOI=0.0001 time series data, atop (A,B) &#x3B2; - &#x3C1; and (C) &#x3B2; - &#x3B5; bifurcation. Fits and bifurcations are grouped by immune phenotype: (A) absent; (B) induced; (C) constitutive immunity, with cell lines differentiated by shape (Vero=circles; RoNi/7.1 = triangles; PaKiT01=squares) and viral infections by color (rVSV-G = green, rVSV-EBOV = magenta, rVSV-MARV = blue). Note that y-axis values are ten-fold higher in panel (C). Branch point curves (solid lines) and Hopf curves (dashed lines) are reproduced from Figure 3 (main text). White space indicates endemic equilibrium (pathogen persistence), gray space indicates limit cycling (virus-induced epidemic extinction), and black space indicates no infection (immune- mediated pathogen extinction). In panel (A) and (B), &#x3B5; is fixed at 0; in panel (C), &#x3B5; is fixed at 5&#xD7;10-8 for bifurcation curves and estimated at 4&#xD7;10-8 and 8&#xD7;10-8 for rVSV-EBOV and rVSV-G parameter points, respectively. To construct bifurcation curves, other parameter values were fixed at: b = 0.025, &#x3BC; = 0.001, , and c = 0 across all panels. Raw fitted values and corresponding 95% confidence intervals for &#x3B2;, &#x3C1;, and &#x3B5;, background parameter values, and AIC recovered from model fit, are reported in Supplementary File 4. Parameter fits at MOI=0.0001 are visualized in Figure 4 of the main text." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1488400151" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Best fit parameter estimates for &#x3B2; and &#x3C1; or &#x3B5; from mean-field model fits to MOI=0.0001 time series data, atop (A,B) &#x3B2; - &#x3C1; and (C) &#x3B2; - &#x3B5; bifurcation. Fits and bifurcations are grouped by immune phenotype: (A) absent; (B) induced; (C) constitutive immunity, with cell lines differentiated by shape (Vero=circles; RoNi/7.1 = triangles; PaKiT01=squares) and viral infections by color (rVSV-G = green, rVSV-EBOV = magenta, rVSV-MARV = blue). Note that y-axis values are ten-fold higher in panel (C). Branch point curves (solid lines) and Hopf curves (dashed lines) are reproduced from Figure 3 (main text). White space indicates endemic equilibrium (pathogen persistence), gray space indicates limit cycling (virus-induced epidemic extinction), and black space indicates no infection (immune- mediated pathogen extinction). In panel (A) and (B), &#x3B5; is fixed at 0; in panel (C), &#x3B5; is fixed at 5&#xD7;10&lt;sup&gt;-8&lt;/sup&gt; for bifurcation curves and estimated at 4&#xD7;10&lt;sup&gt;-8&lt;/sup&gt; and 8&#xD7;10&lt;sup&gt;-8&lt;/sup&gt; for rVSV-EBOV and rVSV-G parameter points, respectively. To construct bifurcation curves, other parameter values were fixed at: &lt;em&gt;b =&lt;/em&gt; 0.025, &lt;em&gt;&#x3BC; =&lt;/em&gt; 0.001, , and &lt;em&gt;c =&lt;/em&gt; 0 across all panels. Raw fitted values and corresponding 95% confidence intervals for &#x3B2;, &#x3C1;, and &#x3B5;, background parameter values, and AIC recovered from model fit, are reported in Supplementary File 4. Parameter fits at MOI=0.0001 are visualized in Figure 4 of the main text.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4 - figure supplement 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F13.medium.gif" width="440" height="326"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4 - figure supplement 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F13.medium.gif" width="440" height="326"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4 - figure supplement 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 - figure supplement 1.</span> <p id="p-71" class="first-child">Best fit parameter estimates for β and ρ or ε from mean-field model fits to MOI=0.0001 time series data, atop (A,B) β - ρ and (C) β - ε bifurcation. Fits and bifurcations are grouped by immune phenotype: (A) absent; (B) induced; (C) constitutive immunity, with cell lines differentiated by shape (Vero=circles; RoNi/7.1 = triangles; PaKiT01=squares) and viral infections by color (rVSV-G = green, rVSV-EBOV = magenta, rVSV-MARV = blue). Note that y-axis values are ten-fold higher in panel (C). Branch point curves (solid lines) and Hopf curves (dashed lines) are reproduced from <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a> (main text). White space indicates endemic equilibrium (pathogen persistence), gray space indicates limit cycling (virus-induced epidemic extinction), and black space indicates no infection (immune- mediated pathogen extinction). In panel (A) and (B), ε is fixed at 0; in panel (C), ε is fixed at 5×10<sup>-8</sup> for bifurcation curves and estimated at 4×10<sup>-8</sup> and 8×10<sup>-8</sup> for rVSV-EBOV and rVSV-G parameter points, respectively. To construct bifurcation curves, other parameter values were fixed at: <em>b =</em> 0.025, <em>μ =</em> 0.001, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/F13/embed/inline-graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/F13/embed/inline-graphic-4.gif"/></noscript></span></span>, and <em>c =</em> 0 across all panels. Raw fitted values and corresponding 95% confidence intervals for β, ρ, and ε, background parameter values, and AIC recovered from model fit, are reported in Supplementary File 4. Parameter fits at MOI=0.0001 are visualized in <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a> of the main text.</p><div class="sb-div caption-clear"></div></div></div><div id="F14" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F14.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Figure replicates Figure 5 (main text) but includes all output across mean field model fits assuming (A) absent immunity, (B) induced immunity, and (C) constitutive immunity. Figure shows fitted time series of susceptible (green shading) and antiviral (blue shading) cell proportions from the mean field model for rVSV-G, rVSV-EBOV, and rVSV-MARV infections (columns) on Vero, RoNi/7.1, and PaKiT01 cell lines (rows) at MOI=0.001. Combined live, uninfectious cell populations (S + A + E, summed across the time series) is shown in tan shading, with raw live, uninfectious cell data from Hoechst stains of terminal time series visualized as open circles. The right-hand y-axis corresponds to R-effective (pink solid line) across each time series; R-effective = 1 is given as a pink dashed, horizontal line. Panel background corresponds to empirical outcome of the average stochastic cell culture trial (persistent infection = white; virus-induced epidemic extinction = gray; immune-mediated epidemic extinction = black)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1488400151" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Figure replicates Figure 5 (main text) but includes all output across mean field model fits assuming (A) absent immunity, (B) induced immunity, and (C) constitutive immunity. Figure shows fitted time series of susceptible (green shading) and antiviral (blue shading) cell proportions from the mean field model for rVSV-G, rVSV-EBOV, and rVSV-MARV infections (columns) on Vero, RoNi/7.1, and PaKiT01 cell lines (rows) at MOI=0.001. Combined live, uninfectious cell populations (S + A + E, summed across the time series) is shown in tan shading, with raw live, uninfectious cell data from Hoechst stains of terminal time series visualized as open circles. The right-hand y-axis corresponds to R-effective (pink solid line) across each time series; R-effective = 1 is given as a pink dashed, horizontal line. Panel background corresponds to empirical outcome of the average stochastic cell culture trial (persistent infection = white; virus-induced epidemic extinction = gray; immune-mediated epidemic extinction = black).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5 - figure supplement 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F14.medium.gif" width="347" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5 - figure supplement 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F14.medium.gif" width="347" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5 - figure supplement 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 - figure supplement 1.</span> <p id="p-72" class="first-child">Figure replicates <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a> (main text) but includes all output across mean field model fits assuming (A) absent immunity, (B) induced immunity, and (C) constitutive immunity. Figure shows fitted time series of susceptible (green shading) and antiviral (blue shading) cell proportions from the mean field model for rVSV-G, rVSV-EBOV, and rVSV-MARV infections (columns) on Vero, RoNi/7.1, and PaKiT01 cell lines (rows) at MOI=0.001. Combined live, uninfectious cell populations (S + A + E, summed across the time series) is shown in tan shading, with raw live, uninfectious cell data from Hoechst stains of terminal time series visualized as open circles. The right-hand y-axis corresponds to R-effective (pink solid line) across each time series; R-effective = 1 is given as a pink dashed, horizontal line. Panel background corresponds to empirical outcome of the average stochastic cell culture trial (persistent infection = white; virus-induced epidemic extinction = gray; immune-mediated epidemic extinction = black).</p><div class="sb-div caption-clear"></div></div></div><div id="F15" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F15.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Figure replicates Figure 5 - figure supplement 1 exactly but shows model fits and data for all cell-virus combinations at MOI=0.0001." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1488400151" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Figure replicates Figure 5 - figure supplement 1 exactly but shows model fits and data for all cell-virus combinations at MOI=0.0001.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5 - figure supplement 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F15.medium.gif" width="347" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5 - figure supplement 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F15.medium.gif" width="347" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F15.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5 - figure supplement 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F15.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 - figure supplement 2.</span> <p id="p-73" class="first-child">Figure replicates <a id="xref-fig-14-3" class="xref-fig" href="#F14">Figure 5 - figure supplement 1</a> exactly but shows model fits and data for all cell-virus combinations at MOI=0.0001.</p><div class="sb-div caption-clear"></div></div></div><div id="F16" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F16.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Spatial model state variable outputs, fit to MOI=0.001 data only, for all 27 unique cell line - virus - immune assumption combinations: (A) absent immunity, (B) induced immunity, and (C) constitutive immunity. Values for &#x3C1; and &#x3B5; were fixed at equivalent values to those optimized in mean field trials and &#x3B2; fixed at ten times the value estimated under mean field conditions. Figure shows mean output from 10 runs of the spatial stochastic model, on a 10,000 cell lattice for MOI=0.001 infections of rVSV-G, rVSV-EBOV, and rVSV-MARV (columns) on Vero, RoNi/7.1, and PaKiT01 (rows) cell lines. Mean state variable outputs are plotted as colored lines with 95% confidence intervals by standard error shown in corresponding shading (infectious = red; susceptible = green; antiviral = blue). Raw infectious cell data across all time trials are plotted as open red circles, with the Hoechst-stained live cell population as open black circles. Modeled live, uninfectious cell populations (S+A+E) are shown in tan shading in the background. Panel background shading corresponds to the mean spatial model outcome for each cell line - virus combination (persistent infection = white; virus- induced epidemic extinction = gray; immune-mediated epidemic extinction = black). All parameter values are reported in Supplementary File 4." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1488400151" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Spatial model state variable outputs, fit to MOI=0.001 data only, for all 27 unique cell line - virus - immune assumption combinations: (A) absent immunity, (B) induced immunity, and (C) constitutive immunity. Values for &#x3C1; and &#x3B5; were fixed at equivalent values to those optimized in mean field trials and &#x3B2; fixed at ten times the value estimated under mean field conditions. Figure shows mean output from 10 runs of the spatial stochastic model, on a 10,000 cell lattice for MOI=0.001 infections of rVSV-G, rVSV-EBOV, and rVSV-MARV (columns) on Vero, RoNi/7.1, and PaKiT01 (rows) cell lines. Mean state variable outputs are plotted as colored lines with 95% confidence intervals by standard error shown in corresponding shading (infectious = red; susceptible = green; antiviral = blue). Raw infectious cell data across all time trials are plotted as open red circles, with the Hoechst-stained live cell population as open black circles. Modeled live, uninfectious cell populations (S+A+E) are shown in tan shading in the background. Panel background shading corresponds to the mean spatial model outcome for each cell line - virus combination (persistent infection = white; virus- induced epidemic extinction = gray; immune-mediated epidemic extinction = black). All parameter values are reported in Supplementary File 4.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5 - figure supplement 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F16.medium.gif" width="377" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5 - figure supplement 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F16.medium.gif" width="377" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F16.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5 - figure supplement 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F16.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 - figure supplement 3.</span> <p id="p-74" class="first-child">Spatial model state variable outputs, fit to MOI=0.001 data only, for all 27 unique cell line - virus - immune assumption combinations: (A) absent immunity, (B) induced immunity, and (C) constitutive immunity. Values for ρ and ε were fixed at equivalent values to those optimized in mean field trials and β fixed at ten times the value estimated under mean field conditions. Figure shows mean output from 10 runs of the spatial stochastic model, on a 10,000 cell lattice for MOI=0.001 infections of rVSV-G, rVSV-EBOV, and rVSV-MARV (columns) on Vero, RoNi/7.1, and PaKiT01 (rows) cell lines. Mean state variable outputs are plotted as colored lines with 95% confidence intervals by standard error shown in corresponding shading (infectious = red; susceptible = green; antiviral = blue). Raw infectious cell data across all time trials are plotted as open red circles, with the Hoechst-stained live cell population as open black circles. Modeled live, uninfectious cell populations (S+A+E) are shown in tan shading in the background. Panel background shading corresponds to the mean spatial model outcome for each cell line - virus combination (persistent infection = white; virus- induced epidemic extinction = gray; immune-mediated epidemic extinction = black). All parameter values are reported in Supplementary File 4.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-21"><h2 class="">Video Captions</h2><p id="p-75"><strong>Video 1.</strong> Two hundred hour time series of spatial stochastic model for rVSV-EBOV infection on 10,000 cell grid for PaKiT01, assuming conditions of <em>absent immunity</em>: (A) spatial spread of infection, (B) time series of state variables. Parameter values are listed in Supplementary File 4.</p><p id="p-76">Video 2. Two hundred hour time series of spatial stochastic model for rVSV-EBOV infection on 10,000 cell grid for PaKiT01, assuming conditions of <em>induced immunity</em>: (A) spatial spread of infection, (B) time series of state variables. Parameter values are listed in Supplementary File 4.</p><p id="p-77">Video 3. Two hundred hour time series of spatial stochastic model for rVSV-EBOV infection on 10,000 cell grid for PaKiT01, assuming conditions of <em>constitutive immunity</em>: (A) spatial spread of infection, (B) time series of state variables. Parameter values are listed in Supplementary File 4.</p></div><div class="section" id="sec-22"><h2 class="">Supplementary File Captions</h2><p id="p-78"><strong>Supplementary File 1.</strong> (A) Raw proportion infectious from cell culture images. Dataset gives raw proportion of infectious cells and time elapsed for all trials of all cell line/virus/MOI combinations, derived from image processing of binary images. (B) Raw proportion uninfectious from cell culture images. Dataset gives raw proportion of uninfectious cells and time elapsed for all trials of all cell line/virus/MOI combinations, derived from image processing of binary Hoechst-stained images. (C) Statistical mean of infectious time series for all trials of each cell line/virus/MOI experiment, from GAM fitted model incorporating random effects by trial. Data were smoothed to yield the proportion infectious per hourly timestep for each trial, and mean field mechanistic models were fit to the smoothed mean of all compiled trials for each cell line/virus/MOI combination. (D) Statistical mean of uninfectious time series for all eighteen cell line/virus/MOI experiments, from generalized linear model fit to Hoechst stain data reported on tab B. Note that these means were not used in epidemic model fitting but natural mortality rates for each cell line were derived from fitting an infection-absent model to the trajectory of susceptible decline for control trials for each cell line, as shown in <a id="xref-fig-12-4" class="xref-fig" href="#F12">Figure 1 - figure supplement 7</a>. All original raw image files, processed binary images, and image processing code are available freely for download at the following FigShare repository: DOI: 10.6084/m9.figshare.8312807.</p><p id="p-79"><strong>Supplementary Files 2-7</strong> are all supplementary tables or text which do not require captions.</p><p id="p-80"><strong>Supplementary File 2.</strong> Derivation of R<sub>0</sub></p><p id="p-81">From the main text, we recall that our mean field model takes on the following form:
<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-30.gif"/></noscript></span></span>
<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-31.gif"/></noscript></span></span>
<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-32.gif"/></noscript></span></span>
<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-33.gif"/></noscript></span></span>
<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-34.gif"/></noscript></span></span></p><p id="p-82">Parameters are defined according to:</p><ul class="list-unord " id="list-1"><li id="list-item-1"><p id="p-83"><em>b</em> = natural birth rate; fixed at .025 to yield stable population sizes in <em>in vitro</em> models (<a id="xref-fig-10-4" class="xref-fig" href="#F10">Figure 1-figure supplement 5</a>)</p></li><li id="list-item-2"><p id="p-84"><em>μ</em> = natural mortality rate; estimated for each cell line by fitting a simple birth-death model to susceptible declines from control trials; values at <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/inline-graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/inline-graphic-5.gif"/></noscript></span></span>, hours for, respectively, Vero, RoNi/7.1, and PaKiT01 cell lines</p></li><li id="list-item-3"><p id="p-85"><em>β</em> = transmission coefficient; estimated for each cell line/virus/MOI combination by fitting model to data</p></li><li id="list-item-4"><p id="p-86">α = infection-induced mortality rate (virulence); fixed at <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/inline-graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/inline-graphic-6.gif"/></noscript></span></span> hours</p></li><li id="list-item-5"><p id="p-87"><em>σ</em> = incubation rate of cell transitioning from an exposed (E) to an infectious (I) state; measured for each cell line-virus combination as the inverse of the time from initial infection to first observation of infectious virus by GFP under the fluorescent micropscope. Respectively, fixed at the following rates: <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/inline-graphic-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/inline-graphic-7.gif"/></noscript></span></span> hours for rVSV-G, rVSV-EBOV, and rVSV-MARV infections on Vero cells; <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/inline-graphic-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/inline-graphic-8.gif"/></noscript></span></span> hours for rVSV-G, rVSV-EBOV, and rVSV-MARV infections on RoNi/7.1 cells; and <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/inline-graphic-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/inline-graphic-9.gif"/></noscript></span></span> hours for rVSV-G, rVSVEBOV, and rVSV-MARV infections on PaKiT01 cells</p></li><li id="list-item-6"><p id="p-88">ρ = rate of infection-induced acquisition to antiviral state, based on the global proportion of exposed cells in the tissue culture; fixed at 0 under conditions of absent immunity and estimated under conditions of induced and constitutive immunity for all cell line/virus combinations by fitting model to joint data over both MOI-0.001 and 0.0001 data.</p></li><li id="list-item-7"><p id="p-89"><em>ε</em> = rate of constitutive acquisition to antiviral state, based on the global proportion of susceptible cells in the tissue culture; fixed at 0 under assumptions of absent and induced immunity; estimated under conditions of constitutive immunity for all cell line/virus combinations by fitting model to joint data over both MOI=0.001 and 0.0001 data.</p></li><li id="list-item-8"><p id="p-90"><em>c</em> = rate of return to susceptibility from antiviral status; fixed at 0 in all model fits under all assumptions of immunity. Regression from antiviral status to susceptibility occurs on longer time scales than the 200 hour duration of our cell culture trials</p></li><li id="list-item-9"><p id="p-91">We assume that <em>P<sub>S</sub> + P<sub>A</sub> + P<sub>E</sub> +P<sub>I</sub> + P<sub>D</sub> =</em> 1</p></li></ul><p id="p-92">We rewrote the equations to model explicit cell densities within a defined well volume. Note that these two systems of equations are equivalent and that parameter values can be used interchangeably across both forms. In the following system, N indicates the total cell capacity of the area modeled:
<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-35.gif"/></noscript></span></span>
<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-36.gif"/></noscript></span></span>
<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-37.gif"/></noscript></span></span>
<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-38.gif"/></noscript></span></span>
<span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-39.gif"/></noscript></span></span></p><p id="p-93">To solve for R<sub>0</sub>, we adopted a next generation matrix approach, after <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Heffernan, Smith, and Wahl 2005</a>, which we applied to the proportional system of <a id="xref-disp-formula-1-2" class="xref-disp-formula" href="#disp-formula-1">equations (1</a>-<a id="xref-disp-formula-5-2" class="xref-disp-formula" href="#disp-formula-5">5</a>, above). Step one of this process necessitated constructing two transition matrices: the F matrix, which describes how new infections are created and the V matrix which represents host transitions between exposed and infected states.</p><p id="p-94">To build these F and V matrices, we first calculated the value of P<sub>S</sub> at the disease free equilibrium (DFE, whereby P<sub>I</sub> and P<sub>E</sub> are equal to zero. We allow for the presence of antiviral\ cells (P<sub>A</sub>) at disease free equilibrium, though their equilibrium quantity (and, by consequence, the extent of constitutive immunity at play in the system) depends on certain parameter values. When <em>ε</em> = 0, P<sub>A</sub> will always be 0 at disease free equilibrium. By allowing the extent of constitutive immunity to vary with parameter values, we maintain consistency with the published literature, which suggests that, in constitutively antiviral bat cell lines (i.e. PaKiT01), some interferon-stimulated genes (ISGs) are perpetually expressed, like their IFN-α precursor, while others still require induction upon infection. At DFE, our antiviral population takes on the following form:
<span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-40.gif"/></noscript></span></span></p><p id="p-95">And, by extension, our susceptible population can be represented as:
<span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-41.gif"/></noscript></span></span></p><p id="p-96">For cells with either absent or fully induced immunity, for which <em>ε</em> = 0, this contracts to:
<span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-42.gif"/></noscript></span></span></p><p id="p-97">Because we cannot have negative cell proportions at equilibrium, we impose the following constraint on the above parameters:
<span class="disp-formula" id="disp-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-43.gif"/></noscript></span></span></p><p id="p-98">When we substitute the above value for P<sub>S</sub>* into the infectious equations, we find:
<span class="disp-formula" id="disp-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-44.gif"/></noscript></span></span>
<span class="disp-formula" id="disp-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-45.gif"/></noscript></span></span></p><p id="p-99">Our F matrix then takes on the following form:
<span class="disp-formula" id="disp-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-46.gif"/></noscript></span></span></p><p id="p-100">And the V matrix takes on the following form:
<span class="disp-formula" id="disp-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-47.gif"/></noscript></span></span></p><p id="p-101">After <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Heffernan, Smith, and Wahl 2005</a>, it follows that <em>G = -FV<sup>-1</sup></em>. Thus, it follows that:
<span class="disp-formula" id="disp-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-48.gif"/></noscript></span></span></p><p id="p-102">When this F matrix is multiplied by matrix -<em>V</em><sup>-1</sup>, we find:
<span class="disp-formula" id="disp-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-49.gif"/></noscript></span></span> which, after matrix algebra, yields the following equation for R<sub>0</sub>:
<span class="disp-formula" id="disp-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-50.gif"/></noscript></span></span></p><p id="p-103">Pathogens can invade a host tissue under conditions of <em>R</em><sub>0</sub> &gt; 1, or when the system satisfies the following inequality:
<span class="disp-formula" id="disp-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-51.gif"/></noscript></span></span></p><p id="p-104">Finally, we note that for all cells lacking in constitutive antiviral properties (<em>ε</em> = 0), R<sub>0</sub> reduces to:
<span class="disp-formula" id="disp-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-52.gif"/></noscript></span></span></p><p id="p-105"><strong>Supplementary File 3.</strong> Special Points from Bifurcation Analysis</p><div id="F17" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F17.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1488400151" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure17" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F17.medium.gif" width="440" height="317"/><noscript><img class="highwire-fragment fragment-image" alt="Figure17" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F17.medium.gif" width="440" height="317"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F17.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure17" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F17.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div><p id="p-106"><strong>Supplementary File 4.</strong> Optimized parameters from all deterministic model outputs and spatial approximations</p><div id="F18" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F18/graphic-54.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1488400151" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F18/graphic-54.medium.gif" width="373" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F18/graphic-54.medium.gif" width="373" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F18/graphic-54.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F18/graphic-54.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F18/graphic-55.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1488400151" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F18/graphic-55.medium.gif" width="440" height="325"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F18/graphic-55.medium.gif" width="440" height="325"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F18/graphic-55.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F18/graphic-55.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div></div></div><p id="p-107"><strong>Supplementary File 5.</strong> Justification for parameter increase from mean field to spatial model.</p><p id="p-108">Because spatial configurations elevate thresholds for pathogen invasion, as well for host cell persistence (<a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Webb, Keeling, and Boots 2007</a>), we were forced to elevate both birth rates (<em>b</em>) and transmission rates (<em>β</em>) above those used in the mean field model to apply to the spatial context. We here justify our chosen values for parameter increase:</p><p id="p-109">In <a id="xref-ref-57-4" class="xref-bibr" href="#ref-57">Webb, Keeling, and Boots 2007</a>, the authors approximate spatial dynamics using a series of differential equations tracking the pairwise neighbor-neighbor interactions of a regular square lattice with a Von Neumann neighborhood. Webb <em>et al</em>. represent local reproduction as <em>b</em>(1 - <em>L<sub>b</sub></em>)/<em>z</em> and local transmission as <em>β</em>(1 - <em>L<sub>I</sub></em>)/<em>z</em> where z is the number of near-neighbor pairs, <em>L<sub>b</sub></em><sub>’</sub> is the proportion of global reproduction (births), and <em>L<sub>I</sub></em> is the proportion of global transmission in the system. Note that we altered the reproduction parameter <em>r</em> to <em>b</em> to reflect our own nomenclature.</p><p id="p-110">Cumulatively, we can represent total reproduction (<em>b</em>) and total transmission (<em>β<sub>tot</sub></em>) in a system with both local and global effects as:
<span class="disp-formula" id="disp-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-56.gif"/></noscript></span></span>
<span class="disp-formula" id="disp-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-57.gif"/></noscript></span></span></p><p id="p-111">Our goal is to find a scalar (<em>q</em>) by which to multiply the mean field birth (<em>q<sub>b</sub></em>) and transmission (<em>q<sub>β</sub></em>) rates to values appropriate for the spatial context, such that:
<span class="disp-formula" id="disp-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-58.gif"/></noscript></span></span>
<span class="disp-formula" id="disp-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-59.gif"/></noscript></span></span></p><p id="p-112">Since we know that <em>L<sub>b</sub></em> and <em>L</em><sub>I</sub> both equal 1 in the mean field model, we can represent these interactions as:
<span class="disp-formula" id="disp-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-60.gif"/></noscript></span></span>
<span class="disp-formula" id="disp-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-61.gif"/></noscript></span></span></p><p id="p-113">And then solve for (<em>q<sub>b</sub></em>) and (<em>q<sub>β</sub></em>):
<span class="disp-formula" id="disp-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-62.gif"/></noscript></span></span>
<span class="disp-formula" id="disp-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-63.gif"/></noscript></span></span></p><p id="p-114">It is easy to see how, in the perfect pair approximation, when <em>L<sub>b</sub></em> or <em>L<sub>I</sub></em> = 0, <em>q</em> = z meaning that the mean field transmission rate is simply multiplied by the number of nearest neighbors.</p><p id="p-115">Equally, in the mean field context, when <em>L<sub>b</sub></em> or <em>L<sub>I</sub></em> = 1, <em>q</em> = 1.</p><p id="p-116">From <a id="xref-disp-formula-7-1" class="xref-disp-formula" href="#disp-formula-7">equation (7)</a>, it is straightforward to calculate (<em>q<sub>b</sub></em>) for our spatial model, since births are permitted only at the nearest-neighbor interface, and each cell has six adjacent neighbors. As such:
<span class="disp-formula" id="disp-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-64.gif"/></noscript></span></span></p><p id="p-117">To this end, we multiplied all mean field birth rates (b=0.025) by 6 to equal .15 in the spatial model.</p><p id="p-118">The dynamics are slightly more complicated in the case of the transmission rate modifier (<em>q<sub>β</sub></em>) because our spatial model allows cells to influence infection up to three ‘rings’ out from the nearest neighbor, such that each cell is affected by the proportion infectious in a 36-cell vicinity. These dynamics accurately reflect viral transmission, which, even in a plaque assay, can diffuse beyond the immediate neighbor-neighbor boundary, especially in a ten-minute timestep.</p><p id="p-119">As such, we chose to represent transmission processes in the spatial model under assumptions of <em>L<sub>I</sub></em> = .075, allowing ∼7.5% of transmission to be modeled globally and leading to a less extreme multiplication of the mean field transmission rate than assumed under the perfect pair approximation (<em>q</em> = z) illustrated in <a id="xref-disp-formula-9-1" class="xref-disp-formula" href="#disp-formula-9">equation (9)</a> above. Under these new assumptions, we modify the mean field transmission rate for the spatial context as follows:
<span class="disp-formula" id="disp-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/01/696195/embed/graphic-65.gif"/></noscript></span></span></p><p id="p-120">Since this is an approximation and all transitions occur stochastically, we round <em>q<sub>β</sub></em> up to multiply all mean field transmission rates by ten in the spatial context.</p><p id="p-121"><strong>Supplementary File 6.</strong> Primers for qPCR</p><div id="F19" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F19.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1488400151" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure19" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F19.medium.gif" width="440" height="263"/><noscript><img class="highwire-fragment fragment-image" alt="Figure19" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F19.medium.gif" width="440" height="263"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F19.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure19" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/01/696195/F19.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div><p id="p-122"><strong>Supplementary File 7.</strong> Detailed methods for image and image data processing.</p><p id="p-123"><em>Image Processing</em>.</p><p id="p-124">All image processing and data analysis was carried out in R version 3.6 for MacIntosh (R Core Team 2019). Original images were imported into R and processed via the package EBImage (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Pau et al. 2010</a>). Composite images of each well were first split into the 36 or 64-subframes from which they were composed (each subframe represents the visual region of focus for the microscope at the time of imaging). Each subframe was trimmed (to remove border effects), processed individually, and recompiled post-processing into binary form, such that light-colored regions of the original image were assigned a value of 1 (white), and dark regions were assigned a value of 0 (black). In the case of images of GFP-expressing cells, these white regions corresponded to “infectious” cells, while in images of Hoechst-stained nuclei, they indicated live, “uninfected” cells.</p><p id="p-125">Microscope focus was poor for control wells and for subframes early in the time series of each trial before GFP expression became apparent, and the original versions of any such subframes were light gray and grainy. Our image processing code identified these subframes as any which possessed a mean pixel value greater than .25 (a value substantially higher than any subframes in which true GFP-expressing or Hoechst-stained cells were visible) and subsequently converted the entire frame to 0 (black).</p><p id="p-126">All other subframes were processed following thresholding methods that have been previously described by the authors of EBImage (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Pau et al. 2010</a>). As a first pass, all pixels excepting the top 25% lightest pixels tallied were converted to 0 (black). Next, each image frame was walked through a series of secondary thresholding steps using if-else statements in R, such that the lightness threshold for “infectious” status was elevated in frames which were lighter overall due to manual variation in imaging and focusing. Processed subframes were then reconstructed into composite binary images, which were manually checked against original images to ensure consistent and reliable results.</p><p id="p-127">Post-processing into binary form, the number of discrete shapes with value of 1 were tabulated within each image, using the max(bwlabel(X)) function in EBimage, to determine a cell count per image, again corresponding to a count of infectious cells for GFP-expressing images and to a count of uninfected cells for Hoechst stain images. All image processing and counting scripts, in addition to the resulting data, are freely available for download at the following FigShare repository: DOI: 10.6084/m9.figshare.8312807.</p><p id="p-128"><em>Image Data processing</em>.</p><p id="p-129">GFP-expressing images were processed and cells counted across the duration of each infection trial, thus generating a time series of infectious cells. For a subset of plates, infection was terminated, and cells were fixed, Hoechst stained, and imaged at periodic intervals across the duration of the time series. Processing of these images thus allowed for construction of a corresponding time series of live, uninfected cells. Because of logistical constraints (i.e. many plates of simultaneously running infection trials and only one available imaging microscope), the time course of imaging across the duration of each trial was quite variable. As such, we fitted a series of statistical models to our raw image data to reconstruct reliable values of the infectious proportion of each well per hour for each distinct trial in all cell line-virus-MOI combinations (<a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 1-figure supplement 2-3</a>).</p><p id="p-130">There was considerable natural variation in initial cell counts across each trial, resulting from subtle differences in the seeding density and growth duration of time until the trial was initiated (when wells were subjectively deemed to have reached “90% confluency”). Baseline cell counts were also different across our three cell lines, which varied substantially in natural size. To correct for this variation, we opted to model the proportion of infectious cell spaces per hour for each well, rather than rely on the raw count data. To this end, we collected the maximum number of live cells counted in susceptible control wells at timepoint 0 and set this count to a rough measure of 100% well occupancy for the cell line in question. Using this method, maximum cell counts were, respectively, 103712, 82308, and 92233 for Vero, RoNi/7.1, and PaKiT01 cell lines, reflecting innate variation in cell sizes. We then converted all cell counts tabulated via our image processing code across the infectious time trials into proportions by dividing the counts by the total number of possible cell spaces for the cell line in question. Though clearly subject to some error, these methods nonetheless maintained internal consistency in our counting methods and generated reasonable time series. We originally experimented with directly tabulating the proportion of infected versus uninfected space in our binary images; however, this approach impaired our ability to generalize across more or less densely seeded trials, as well as trials on cell lines of disparate sizes. As such, we adopted the count-to-proportion methods described here.</p><p id="p-131">To generate an infectious time series of evenly distributed time steps against which to fit our mean field mechanistic model, we next fit a series of statistical models to the proportional data produced from the image processing methods described above. For the GFP-expressing data, we used the mgcv package in R (Wood 2001) to fit generalized additive models (GAMs) in the Gaussian family, with time elapsed (in hours) post infection as a predictor variable for proportion of infectious cells (the response variable). We fit a separate GAM model to each unique cell - virus - MOI combination, incorporating a random effect of well ID (such that each trial was modeled individually), and we fixed the smoothing parameter at k=7 for all trials, as recommended by the package author (Wood 2001). The gam.predict() function was used to return an estimate of infectious proportions per hour across the duration of each time series for each cell-virus-MOI combination.</p><p id="p-132">The uninfected counts from the Hoechst stain data were much less numerous since each count required termination of the infection trial and fixation of cells; by definition, only one data point could be produced per trial. Due to this relative scarcity, we opted to fit a more standard linear regression model, again in the Gaussian family, to these data, rather than using the data-hungry GAM methods employed above. As before, we set time elapsed post infection as the predictor for the Hoechst stain data and produced a unique estimate of the proportion of uninfected cells per hour across the duration of the longest-observed trial. No random effects were included in this model, and the resulting time series were used to estimate natural mortality rates for each cell line, when fit to control well data depicting natural susceptible decline (<a id="xref-fig-12-5" class="xref-fig" href="#F12">Figure 1-figure supplement 7</a>).</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-63">CEB was supported by a National Science Foundation Graduate Research Fellowship at Princeton University and a Miller Institute for Basic Research Fellowship at UC Berkeley. Tissue culture experiments were funded by an NIH grant R01 AI134824 to KC. Work in LFW’s lab was funded by the Singapore National Research Foundation grants (NRF2012NRF-CRP001-056 and NRF2016NRF-NSFC002-013). CD was supported by the German Research Council (DFG) grant DFG SPP1596 (DR 772/10-2), the Federal Ministry of Education and Research (BMBF) grant RAPID (#01KI1723A) and the EU Horizon 2020 grant EVAg (#653316). The authors thank the Chandran lab at Albert Einstein College of Medicine - in particular, Cecilia Harold, Megan Slough, Rohit Jangra, and Tanwee Alkutkar - for technical support during tissue culture experiments. The authors further thank Jessica Metcalf and the Graham lab at Princeton for conceptual guidance throughout the project’s development.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-update fn-group-summary-of-updates" id="fn-1"><p id="p-3">We here present an updated version of our manuscript, including edits to the basic model structure and corresponding bifurcation analyses, fits to data, and textual explanation.</p></li><li class="fn-dataset fn-group-external-links" id="fn-2"><p id="p-4">
<a href="https://doi.org/10.6084/m9.figshare.8312807.v3">https://doi.org/10.6084/m9.figshare.8312807.v3</a>
</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ahn</span>,  <span class="cit-name-given-names">Matae</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Danielle E.</span> <span class="cit-name-surname">Anderson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Qian</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Chee Wah</span> <span class="cit-name-surname">Tan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Beng Lee</span> <span class="cit-name-surname">Lim</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Katarina</span> <span class="cit-name-surname">Luko</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Ming</span> <span class="cit-name-surname">Wen</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2019</span>. “<span class="cit-article-title">Dampened NLRP3-Mediated Inflammation in Bats and Implications for a Special Viral Reservoir Host</span>.” <abbr class="cit-jnl-abbrev">Nature Microbiology</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">789</span>–<span class="cit-lpage">99</span>. <a href="https://doi.org/10.1038/s41564-019-0371-3">https://doi.org/10.1038/s41564-019-0371-3</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BMicrobiology%26rft.volume%253D4%26rft.spage%253D789%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Arnold</span>,  <span class="cit-name-given-names">Catherine E</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Jonathan C</span> <span class="cit-name-surname">Guito</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Louis A</span> <span class="cit-name-surname">Altamura</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Sean P</span> <span class="cit-name-surname">Lovett</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Elyse R</span> <span class="cit-name-surname">Nagle</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Gustavo F</span> <span class="cit-name-surname">Palacios</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Mariano</span> <span class="cit-name-surname">Sanchez-lockhart</span></span>. <span class="cit-pub-date">2018</span>. “<span class="cit-article-title">Transcriptomics Reveal Antiviral Gene Induction in the Egyptian Rousette Bat Is Antagonized in Vitro by Marburg Virus Infection</span>.” <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">10</span> (<span class="cit-fpage">607</span>). <a href="https://doi.org/10.3390/v10110607">https://doi.org/10.3390/v10110607</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D10%26rft.spage%253D607%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.3" data-doi="10.1128/JVI.01623-05"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Baccam</span>,  <span class="cit-name-given-names">Prasith</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Catherine</span> <span class="cit-name-surname">Beauchemin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Catherine A</span> <span class="cit-name-surname">Macken</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Frederick G</span> <span class="cit-name-surname">Hayden</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Alan S</span> <span class="cit-name-surname">Perelson</span></span>. <span class="cit-pub-date">2006</span>. “<span class="cit-article-title">Kinetics of Influenza A Virus Infection in Humans</span>.” <abbr class="cit-jnl-abbrev">Journal of Virology</abbr> <span class="cit-vol">80</span> (<span class="cit-issue">15</span>): <span class="cit-fpage">7590</span>–<span class="cit-lpage">99</span>. <a href="https://doi.org/10.1128/JVI.01623-05">https://doi.org/10.1128/JVI.01623-05</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DBaccam%26rft.auinit1%253DP.%26rft.volume%253D80%26rft.issue%253D15%26rft.spage%253D7590%26rft.epage%253D7599%26rft.atitle%253DKinetics%2Bof%2BInfluenza%2BA%2BVirus%2BInfection%2Bin%2BHumans%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01623-05%26rft_id%253Dinfo%253Apmid%252F16840338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4MC8xNS83NTkwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMDEvNjk2MTk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Biesold</span>,  <span class="cit-name-given-names">Susanne E.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Daniel</span> <span class="cit-name-surname">Ritz</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Florian</span> <span class="cit-name-surname">Gloza-Rausch</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Robert</span> <span class="cit-name-surname">Wollny</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Jan Felix</span> <span class="cit-name-surname">Drexler</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Victor M.</span> <span class="cit-name-surname">Corman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Elisabeth K V</span> <span class="cit-name-surname">Kalko</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Samuel</span> <span class="cit-name-surname">Oppong</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Christian</span> <span class="cit-name-surname">Drosten</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Marcel A.</span> <span class="cit-name-surname">Müller</span></span>. <span class="cit-pub-date">2011</span>. “<span class="cit-article-title">Type I Interferon Reaction to Viral Infection in Interferon-Competent, Immortalized Cell Lines from the African Fruit Bat Eidolon Helvum</span>.” <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span> (<span class="cit-issue">11</span>). <a href="https://doi.org/10.1371/journal.pone.0028131">https://doi.org/10.1371/journal.pone.0028131</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.5" data-doi="10.1073/pnas.94.13.6971"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bonhoeffer</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">May</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. M.</span> <span class="cit-name-surname">Shaw</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Nowak</span></span>. <span class="cit-pub-date">1997</span>. “<span class="cit-article-title">Virus Dynamics and Drug Therapy</span>.” <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr> <span class="cit-vol">94</span> (<span class="cit-issue">13</span>): <span class="cit-fpage">6971</span>–<span class="cit-lpage">76</span>. <a href="https://doi.org/10.1073/pnas.94.13.6971">https://doi.org/10.1073/pnas.94.13.6971</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DBonhoeffer%26rft.auinit1%253DS.%26rft.volume%253D94%26rft.issue%253D13%26rft.spage%253D6971%26rft.epage%253D6976%26rft.atitle%253DVirus%2Bdynamics%2Band%2Bdrug%2Btherapy%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.94.13.6971%26rft_id%253Dinfo%253Apmid%252F9192676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTQvMTMvNjk3MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzAxLzY5NjE5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.6" data-doi="10.1016/j.tim.2014.12.004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brook</span>,  <span class="cit-name-given-names">Cara E.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Andrew P.</span> <span class="cit-name-surname">Dobson</span></span>. <span class="cit-pub-date">2015</span>. “<span class="cit-article-title">Bats as ‘special’ Reservoirs for Emerging Zoonotic Pathogens</span>.” <abbr class="cit-jnl-abbrev">Trends in Microbiology</abbr> <span class="cit-vol">23</span> (<span class="cit-issue">3</span>): <span class="cit-fpage">172</span>–<span class="cit-lpage">80</span>. <a href="https://doi.org/10.1016/j.tim.2014.12.004">https://doi.org/10.1016/j.tim.2014.12.004</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BMicrobiology%26rft.volume%253D23%26rft.spage%253D172%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2014.12.004%26rft_id%253Dinfo%253Apmid%252F25572882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2014.12.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25572882&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F01%2F696195.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brook</span>,  <span class="cit-name-given-names">Cara E.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Hafaliana Christian</span> <span class="cit-name-surname">Ranaivoson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Christopher C.</span> <span class="cit-name-surname">Broder</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Andrew A.</span> <span class="cit-name-surname">Cunningham</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Jean-Michel</span> <span class="cit-name-surname">Héraud</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Alison J.</span> <span class="cit-name-surname">Peel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Louise</span> <span class="cit-name-surname">Gibson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">James L.N.</span> <span class="cit-name-surname">Wood</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Jessica Metcalf</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Andrew P.</span> <span class="cit-name-surname">Dobson</span></span>. <span class="cit-pub-date">2019</span>. “<span class="cit-article-title">Disentangling Serology to Elucidate Henipa- and Filovirus Transmission in Madagascar Fruit Bats</span>.” <abbr class="cit-jnl-abbrev">Journal of Animal Ecology</abbr> <span class="cit-vol">00</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">16</span>. <a href="https://doi.org/10.1111/1365-2656.12985">https://doi.org/10.1111/1365-2656.12985</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BEcology%26rft.volume%253D00%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.8" data-doi="10.1128/CMR.00017-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Calisher</span>,  <span class="cit-name-given-names">Charles H</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">James E</span> <span class="cit-name-surname">Childs</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Hume E</span> <span class="cit-name-surname">Field</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Kathryn V</span> <span class="cit-name-surname">Holmes</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Tony</span> <span class="cit-name-surname">Schountz</span></span>. <span class="cit-pub-date">2006</span>. “<span class="cit-article-title">Bats: Important Reservoir Hosts of Emerging Viruses</span>.” <abbr class="cit-jnl-abbrev">Clinical Microbiology Reviews</abbr> <span class="cit-vol">19</span> (<span class="cit-issue">3</span>): <span class="cit-fpage">531</span>–<span class="cit-lpage">45</span>. <a href="https://doi.org/10.1128/CMR.00017-06">https://doi.org/10.1128/CMR.00017-06</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2BMicrobiology%2BReviews%26rft.stitle%253DClin.%2BMicrobiol.%2BRev.%26rft.aulast%253DCalisher%26rft.auinit1%253DC.%2BH.%26rft.volume%253D19%26rft.issue%253D3%26rft.spage%253D531%26rft.epage%253D545%26rft.atitle%253DBats%253A%2BImportant%2BReservoir%2BHosts%2Bof%2BEmerging%2BViruses%26rft_id%253Dinfo%253Adoi%252F10.1128%252FCMR.00017-06%26rft_id%253Dinfo%253Apmid%252F16847084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiY21yIjtzOjU6InJlc2lkIjtzOjg6IjE5LzMvNTMxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMDEvNjk2MTk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Coffin</span>,  <span class="cit-name-given-names">John M</span></span>. <span class="cit-pub-date">1995</span>. <span class="cit-article-title">“HIV Population Dynamics in Vivo: Implications for Genetic Variation, Pathogenesis, and Therapy.” <em>Science (New York</em></span>, <abbr class="cit-jnl-abbrev">N.Y</abbr><em>.)</em> <span class="cit-vol">267</span> (<span class="cit-issue">5197</span>): <span class="cit-fpage">483</span>–<span class="cit-lpage">89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.Y%26rft.volume%253D267%26rft.spage%253D483%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.10" data-doi="10.1371/journal.pone.0008266"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Crameri</span>,  <span class="cit-name-given-names">Gary</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Shawn</span> <span class="cit-name-surname">Todd</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Samantha</span> <span class="cit-name-surname">Grimley</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Jennifer A.</span> <span class="cit-name-surname">McEachern</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Glenn A.</span> <span class="cit-name-surname">Marsh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Craig</span> <span class="cit-name-surname">Smith</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Mary</span> <span class="cit-name-surname">Tachedjian</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2009</span>. “<span class="cit-article-title">Establishment, Immortalisation and Characterisation of Pteropid Bat Cell Lines</span>.” <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">4</span> (<span class="cit-issue">12</span>): <span class="cit-fpage">e8266</span>. <a href="https://doi.org/10.1371/journal.pone.0008266">https://doi.org/10.1371/journal.pone.0008266</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DCrameri%26rft.auinit1%253DG.%26rft.volume%253D4%26rft.issue%253D12%26rft.spage%253De8266%26rft.epage%253De8266%26rft.atitle%253DEstablishment%252C%2Bimmortalisation%2Band%2Bcharacterisation%2Bof%2Bpteropid%2Bbat%2Bcell%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0008266%26rft_id%253Dinfo%253Apmid%252F20011515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0008266&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20011515&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F01%2F696195.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Desmyter</span>,  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J L</span> <span class="cit-name-surname">Melnick</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">W E</span> <span class="cit-name-surname">Rawls</span></span>. <span class="cit-pub-date">1968</span>. “<span class="cit-article-title">Defectiveness of Interferon Production and of Rubella Virus Interference in a Line of African Green Monkey Kidney Cells (Vero)</span>.” <abbr class="cit-jnl-abbrev">Journal of Virology</abbr> <span class="cit-vol">2</span> (<span class="cit-issue">10</span>): <span class="cit-fpage">955</span>–<span class="cit-lpage">61</span>. <a href="https://doi.org/J.Virol.19682:1955">https://doi.org/J.Virol.19682:1955</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DDesmyter%26rft.auinit1%253DJ.%26rft.volume%253D2%26rft.issue%253D10%26rft.spage%253D955%26rft.epage%253D961%26rft.atitle%253DDefectiveness%2Bof%2BInterferon%2BProduction%2Band%2Bof%2BRubella%2BVirus%2BInterference%2Bin%2Ba%2BLine%2Bof%2BAfrican%2BGreen%2BMonkey%2BKidney%2BCells%2B%2528Vero%2529%26rft_id%253Dinfo%253Apmid%252F4302013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjIvMTAvOTU1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMDEvNjk2MTk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dhooge</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Govaerts</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Yu A.</span> <span class="cit-name-surname">Kuznetsov</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H. G E</span> <span class="cit-name-surname">Meijer</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Sautois</span></span>. <span class="cit-pub-date">2008</span>. “<span class="cit-article-title">New Features of the Software MatCont for Bifurcation Analysis of Dynamical Systems</span>.” <abbr class="cit-jnl-abbrev">Mathematical and Computer Modelling of Dynamical Systems</abbr> <span class="cit-vol">14</span> (<span class="cit-issue">2</span>): <span class="cit-fpage">147</span>–<span class="cit-lpage">75</span>. <a href="https://doi.org/10.1080/13873950701742754">https://doi.org/10.1080/13873950701742754</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMathematical%2Band%2BComputer%2BModelling%2Bof%2BDynamical%2BSystems%26rft.volume%253D14%26rft.spage%253D147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.13" data-doi="10.1007/BF00178324"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Diekmann</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.A.P.</span> <span class="cit-name-surname">Heesterbeek</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.A.J</span> <span class="cit-name-surname">Metz</span></span>. <span class="cit-pub-date">1990</span>. “<span class="cit-article-title">On the Definition and Computation of the Basic Reproduction Ratio R0 in Models for Infectious Diseases in Heterogenous Populations</span>.” <abbr class="cit-jnl-abbrev">Journal of Mathematical Biology</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">365</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmathematical%2Bbiology%26rft.stitle%253DJ%2BMath%2BBiol%26rft.aulast%253DDiekmann%26rft.auinit1%253DO.%26rft.volume%253D28%26rft.issue%253D4%26rft.spage%253D365%26rft.epage%253D382%26rft.atitle%253DOn%2Bthe%2Bdefinition%2Band%2Bthe%2Bcomputation%2Bof%2Bthe%2Bbasic%2Breproduction%2Bratio%2BR0%2Bin%2Bmodels%2Bfor%2Binfectious%2Bdiseases%2Bin%2Bheterogeneous%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00178324%26rft_id%253Dinfo%253Apmid%252F2117040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00178324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2117040&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F01%2F696195.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990DG35000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eddelbuettel</span>,  <span class="cit-name-given-names">Dirk</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Romain</span> <span class="cit-name-surname">Francois</span></span>. <span class="cit-pub-date">2011</span>. “<span class="cit-article-title">Rcpp: Seamless R and C++ Integration</span>.” <abbr class="cit-jnl-abbrev">Journal of Statistical Software</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">18</span>. <a href="https://doi.org/10.1007/978-1-4614-6868-4">https://doi.org/10.1007/978-1-4614-6868-4</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BStatistical%2BSoftware%26rft.volume%253D40%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eddelbuettel</span>,  <span class="cit-name-given-names">Dirk</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Conrad</span> <span class="cit-name-surname">Sanderson</span></span>. <span class="cit-pub-date">2017</span>. “<span class="cit-article-title">RcppArmadillo: Accelerating R with High-Performance C++ Linear Algebra</span>.” <abbr class="cit-jnl-abbrev">Computational Statistics and Data Analysis</abbr> <span class="cit-vol">71</span> (<span class="cit-issue">2014</span>): <span class="cit-fpage">1</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComputational%2BStatistics%2Band%2BData%2BAnalysis%26rft.volume%253D71%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.16" data-doi="10.1099/0022-1317-43-1-247"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Emeny</span>,  <span class="cit-name-given-names">J. M.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Morgan</span></span>. <span class="cit-pub-date">1979</span>. “<span class="cit-article-title">Regulation of the Interferon System: Evidence That Vero Cells Have a Genetic Defect in Interferon Production</span>.” <abbr class="cit-jnl-abbrev">Journal of General Virology</abbr> <span class="cit-vol">43</span> (<span class="cit-issue">1</span>): <span class="cit-fpage">247</span>–<span class="cit-lpage">52</span>. <a href="https://doi.org/10.1099/0022-1317-43-1-247">https://doi.org/10.1099/0022-1317-43-1-247</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DEmeny%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D43%26rft.issue%253D1%26rft.spage%253D247%26rft.epage%253D252%26rft.atitle%253DRegulation%2Bof%2Bthe%2BInterferon%2BSystem%253A%2BEvidence%2Bthat%2BVero%2BCells%2Bhave%2Ba%2BGenetic%2BDefect%2Bin%2BInterferon%2BProduction%26rft_id%253Dinfo%253Adoi%252F10.1099%252F0022-1317-43-1-247%26rft_id%253Dinfo%253Apmid%252F113494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/0022-1317-43-1-247&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=113494&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F01%2F696195.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1979GZ36600024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.17" data-doi="10.1084/jem.190.2.177"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Emery</span>,  <span class="cit-name-given-names">V.C.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.V.</span> <span class="cit-name-surname">Cope</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.F.</span> <span class="cit-name-surname">Bowen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gor</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">P.D.</span> <span class="cit-name-surname">Griffiths</span></span>. <span class="cit-pub-date">1999</span>. “<span class="cit-article-title">The Dynamics of Human Cytomegalovirus Replication in Vivo</span>.” <abbr class="cit-jnl-abbrev">The Journal of Experimental Medicine</abbr> <span class="cit-vol">190</span> (<span class="cit-issue">2</span>): <span class="cit-fpage">177</span>–<span class="cit-lpage">82</span>. <a href="https://doi.org/10.1084/jem.190.2.177">https://doi.org/10.1084/jem.190.2.177</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BExperimental%2BMedicine%26rft_id%253Dinfo%253Adoi%252F10.1084%252Fjem.190.2.177%26rft_id%253Dinfo%253Apmid%252F10432281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamVtIjtzOjU6InJlc2lkIjtzOjk6IjE5MC8yLzE3NyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzAxLzY5NjE5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.18" data-doi="10.3201/eid0812.010317"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Haydon</span>,  <span class="cit-name-given-names">Daniel T</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Sarah</span> <span class="cit-name-surname">Cleaveland</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Louise H</span> <span class="cit-name-surname">Taylor</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M</span> <span class="cit-name-surname">Karen Laurenson</span></span>. <span class="cit-pub-date">2002</span>. “<span class="cit-article-title">Identifying Reservoirs of Infection: A Conceptual and Practical Challenge</span>.” <abbr class="cit-jnl-abbrev">Emerging Infectious Diseases</abbr> <span class="cit-vol">8</span> (<span class="cit-issue">12</span>): <span class="cit-fpage">1468</span>–<span class="cit-lpage">73</span>. <a href="https://doi.org/10.3201/eid0812.010317">https://doi.org/10.3201/eid0812.010317</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2Binfectious%2Bdiseases%26rft.stitle%253DEmerg%2BInfect%2BDis%26rft.aulast%253DHaydon%26rft.auinit1%253DD.%2BT.%26rft.volume%253D8%26rft.issue%253D12%26rft.spage%253D1468%26rft.epage%253D1473%26rft.atitle%253DIdentifying%2Breservoirs%2Bof%2Binfection%253A%2Ba%2Bconceptual%2Band%2Bpractical%2Bchallenge.%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid0812.010317%26rft_id%253Dinfo%253Apmid%252F12498665%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid0812.010317&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12498665&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F01%2F696195.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179834300017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Heffernan</span>,  <span class="cit-name-given-names">J.M.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.J.</span> <span class="cit-name-surname">Smith</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">L.M.</span> <span class="cit-name-surname">Wahl</span></span>. <span class="cit-pub-date">2005</span>. “<span class="cit-article-title">Perspectives on the Basic Reproductive Ratio</span>.” <abbr class="cit-jnl-abbrev">Journal of The Royal Society Interface</abbr> <span class="cit-vol">2</span> (<span class="cit-issue">4</span>): <span class="cit-fpage">281</span>–<span class="cit-lpage">93</span>. <a href="https://doi.org/10.1098/rsif.2005.0042">https://doi.org/10.1098/rsif.2005.0042</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BThe%2BRoyal%2BSociety%2BInterface%26rft.volume%253D2%26rft.spage%253D281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.20" data-doi="10.1038/373123a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ho</span>,  <span class="cit-name-given-names">David D.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Avidan U.</span> <span class="cit-name-surname">Neumann</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Alan S.</span> <span class="cit-name-surname">Perelson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Wen</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">John M.</span> <span class="cit-name-surname">Leonard</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Martin</span> <span class="cit-name-surname">Markowitz</span></span>. <span class="cit-pub-date">1995</span>. “<span class="cit-article-title">Rapid Turnover of Plasma Virions and CD4 Lymphocytes in HIV-1 Infection</span>.” <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">373</span>: <span class="cit-fpage">123</span>–<span class="cit-lpage">26</span>. <a href="https://doi.org/10.1038/373123a0">https://doi.org/10.1038/373123a0</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHo%26rft.auinit1%253DD.%2BD.%26rft.volume%253D373%26rft.issue%253D6510%26rft.spage%253D123%26rft.epage%253D126%26rft.atitle%253DRapid%2Bturnover%2Bof%2Bplasma%2Bvirions%2Band%2BCD4%2Blymphocytes%2Bin%2BHIV-1%2Binfection.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F373123a0%26rft_id%253Dinfo%253Apmid%252F7816094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/373123a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7816094&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F01%2F696195.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QB06300049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.21" data-doi="10.1016/S1286-4579(01)01385-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hooper</span>,  <span class="cit-name-given-names">Peter</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Sherif</span> <span class="cit-name-surname">Zaki</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Peter</span> <span class="cit-name-surname">Daniels</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Deborah</span> <span class="cit-name-surname">Middleton</span></span>. <span class="cit-pub-date">2001</span>. “<span class="cit-article-title">Comparative Pathology of the Diseases Caused by Hendra and Nipah Viruses</span>.” <abbr class="cit-jnl-abbrev">Microbes and Infection</abbr> <span class="cit-vol">3</span> (<span class="cit-issue">4</span>): <span class="cit-fpage">315</span>–<span class="cit-lpage">22</span>. <a href="https://doi.org/10.1016/S1286-4579(01)01385-5">https://doi.org/10.1016/S1286-4579(01)01385-5</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobes%2Band%2Binfection%2B%252F%2BInstitut%2BPasteur%26rft.stitle%253DMicrobes%2BInfect%26rft.aulast%253DHooper%26rft.auinit1%253DP.%26rft.volume%253D3%26rft.issue%253D4%26rft.spage%253D315%26rft.epage%253D322%26rft.atitle%253DComparative%2Bpathology%2Bof%2Bthe%2Bdiseases%2Bcaused%2Bby%2BHendra%2Band%2BNipah%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1286-4579%252801%252901385-5%26rft_id%253Dinfo%253Apmid%252F11334749%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1286-4579(01)01385-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11334749&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F01%2F696195.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168143300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Howat</span>,  <span class="cit-name-given-names">Tom J</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Cristina</span> <span class="cit-name-surname">Barreca</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Peter</span> <span class="cit-name-surname">O’Hare</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Julia R</span> <span class="cit-name-surname">Gog</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Bryan T</span> <span class="cit-name-surname">Grenfell</span></span>. <span class="cit-pub-date">2006</span>. <span class="cit-article-title">“Modelling Dynamics of the Type I Interferon Response to in Vitro Viral Infection.” <em>Journal of the Royal Society</em></span>, <abbr class="cit-jnl-abbrev">Interface</abbr> <span class="cit-vol">3</span> (<span class="cit-issue">10</span>): <span class="cit-fpage">699</span>–<span class="cit-lpage">709</span>. <a href="https://doi.org/10.1098/rsif.2006.0136">https://doi.org/10.1098/rsif.2006.0136</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInterface%26rft.volume%253D3%26rft.spage%253D699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Igarashi</span>,  <span class="cit-name-given-names">Manabu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Heinz</span> <span class="cit-name-surname">Feldmann</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Yoshihiro</span> <span class="cit-name-surname">Takadate</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Tatsunari</span> <span class="cit-name-surname">Kondoh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Manabu</span> <span class="cit-name-surname">Igarashi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Junki</span> <span class="cit-name-surname">Maruyama</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Rashid</span> <span class="cit-name-surname">Manzoor</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Hirohito</span> <span class="cit-name-surname">Ogawa</span></span>. <span class="cit-pub-date">2020</span>. “<span class="cit-article-title">Niemann-Pick C1 Heterogeneity of Bat Cells Controls Filovirus Tropism</span>.” <abbr class="cit-jnl-abbrev">Cell Reports</abbr> <span class="cit-vol">30</span> (<span class="cit-issue">2</span>): <span class="cit-fpage">308</span>–<span class="cit-lpage">319</span>.e5. <a href="https://doi.org/10.1016/j.celrep.2019.12.042">https://doi.org/10.1016/j.celrep.2019.12.042</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BReports%26rft.volume%253D30%26rft.spage%253D308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kacprzyk</span>,  <span class="cit-name-given-names">Joanna</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Graham M.</span> <span class="cit-name-surname">Hughes</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Eva M.</span> <span class="cit-name-surname">Palsson-McDermott</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Susan R.</span> <span class="cit-name-surname">Quinn</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Sébastien J.</span> <span class="cit-name-surname">Puechmaille</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Luke A. J.</span> <span class="cit-name-surname">O’Neill</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Emma C.</span> <span class="cit-name-surname">Teeling</span></span>. <span class="cit-pub-date">2017</span>. “<span class="cit-article-title">A Potent Anti­Inflammatory Response in Bat Macrophages May Be Linked to Extended Longevity and Viral Tolerance</span>.” <abbr class="cit-jnl-abbrev">Acta Chiropterologica</abbr> <span class="cit-vol">19</span> (<span class="cit-issue">2</span>): <span class="cit-fpage">219</span>–<span class="cit-lpage">28</span>. <a href="https://doi.org/10.3161/15081109ACC2017.19.2.001">https://doi.org/10.3161/15081109ACC2017.19.2.001</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BChiropterologica%26rft.volume%253D19%26rft.spage%253D219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-book" id="cit-696195v3.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Keeling</span>,  <span class="cit-name-given-names">Matt J.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Pejman</span> <span class="cit-name-surname">Rohani</span></span>. <span class="cit-pub-date">2008</span>. <span class="cit-source">Modeling Infectious Diseases in Humans and Animals</span>. <span class="cit-publ-loc">Princeton, NJ</span>: <span class="cit-publ-name">Princeton University Press</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.26" data-doi="10.1093/infdis/jir306"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kühl</span>,  <span class="cit-name-given-names">Annika</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Markus</span> <span class="cit-name-surname">Hoffmann</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Marcel A.</span> <span class="cit-name-surname">Müller</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Vincent J.</span> <span class="cit-name-surname">Munster</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Kerstin</span> <span class="cit-name-surname">Gnirß</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Miriam</span> <span class="cit-name-surname">Kiene</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Theodros Solomon</span> <span class="cit-name-surname">Tsegaye</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2011</span>. “<span class="cit-article-title">Comparative Analysis of Ebola Virus Glycoprotein Interactions with Human and Bat Cells</span>.” <abbr class="cit-jnl-abbrev">Journal of Infectious Diseases</abbr> <span class="cit-vol">204</span>: <span class="cit-fpage">S840</span>–<span class="cit-lpage">S849</span>. <a href="https://doi.org/10.1093/infdis/jir306">https://doi.org/10.1093/infdis/jir306</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BInfectious%2BDiseases%26rft_id%253Dinfo%253Adoi%252F10.1093%252Finfdis%252Fjir306%26rft_id%253Dinfo%253Apmid%252F21987760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/infdis/jir306&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21987760&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F01%2F696195.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kuzmin</span>,  <span class="cit-name-given-names">I.V.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Toni M.</span> <span class="cit-name-surname">Schwarz</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Philipp A.</span> <span class="cit-name-surname">Ilinykh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Ingo</span> <span class="cit-name-surname">Jordan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Thamas G.</span> <span class="cit-name-surname">Ksiazek</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Ravi</span> <span class="cit-name-surname">Sachidanandam</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Christopher F.</span> <span class="cit-name-surname">Basler</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Alexander</span> <span class="cit-name-surname">Bukreyev</span></span>. <span class="cit-pub-date">2017</span>. “<span class="cit-article-title">Innate Immune Response of Bat and Human Cells to Filoviruses: Commonalities and Distinctions</span>.” <abbr class="cit-jnl-abbrev">Journal of Virology</abbr> <span class="cit-vol">91</span> (<span class="cit-issue">8</span>): <span class="cit-fpage">e02471</span>–<span class="cit-lpage">16</span>. <a href="https://doi.org/10.1128/JVI.02471-16">https://doi.org/10.1128/JVI.02471-16</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.volume%253D91%26rft.spage%253De02471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.28" data-doi="10.1006/meth.2001.1262"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Livak</span>,  <span class="cit-name-given-names">K J</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">T D</span> <span class="cit-name-surname">Schmittgen</span></span>. <span class="cit-pub-date">2001</span>. “<span class="cit-article-title">Analysis of Relative Gene Expression Data Using Real­Time Quantitative PCR and the 2<sup>Λ</sup>(-ΔΔCT) Method</span>.” <abbr class="cit-jnl-abbrev">Methods</abbr> <span class="cit-vol">25</span> (<span class="cit-issue">4</span>): <span class="cit-fpage">402</span>–<span class="cit-lpage">8</span>. <a href="https://doi.org/10.1006/meth.2001.1262">https://doi.org/10.1006/meth.2001.1262</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2B%2528San%2BDiego%252C%2BCalif.%2529%26rft.stitle%253DMethods%26rft.aulast%253DLivak%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D25%26rft.issue%253D4%26rft.spage%253D402%26rft.epage%253D408%26rft.atitle%253DAnalysis%2Bof%2Brelative%2Bgene%2Bexpression%2Bdata%2Busing%2Breal-time%2Bquantitative%2BPCR%2Band%2Bthe%2B2%2528-Delta%2BDelta%2BC%2528T%2529%2529%2BMethod.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fmeth.2001.1262%26rft_id%253Dinfo%253Apmid%252F11846609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/meth.2001.1262&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11846609&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F01%2F696195.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173949500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.29" data-doi="10.1016/S1473-3099(04)01103-X"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mahanty</span>,  <span class="cit-name-given-names">Siddhartha</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Mike</span> <span class="cit-name-surname">Bray</span></span>. <span class="cit-pub-date">2004</span>. “<span class="cit-article-title">Pathogenesis of Filoviral Haemorrhagic Fevers</span>.” <abbr class="cit-jnl-abbrev">Lancet Infectious Diseases</abbr> <span class="cit-vol">4</span> (<span class="cit-issue">8</span>): <span class="cit-fpage">487</span>–<span class="cit-lpage">98</span>. <a href="https://doi.org/10.1016/S1473-3099(04)01103-X">https://doi.org/10.1016/S1473-3099(04)01103-X</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BLancet%2Binfectious%2Bdiseases%26rft.stitle%253DLancet%2BInfect%2BDis%26rft.aulast%253DMahanty%26rft.auinit1%253DS.%26rft.volume%253D4%26rft.issue%253D8%26rft.spage%253D487%26rft.epage%253D498%26rft.atitle%253DPathogenesis%2Bof%2Bfiloviral%2Bhaemorrhagic%2Bfevers.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1473-3099%252804%252901103-X%26rft_id%253Dinfo%253Apmid%252F15288821%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1473-3099(04)01103-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15288821&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F01%2F696195.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222995900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.30" data-doi="10.1038/emboj.2012.53"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Miller</span>,  <span class="cit-name-given-names">Emily Happy</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Gregor</span> <span class="cit-name-surname">Obernosterer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Matthijs</span> <span class="cit-name-surname">Raaben</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Andrew S</span> <span class="cit-name-surname">Herbert</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Maika S</span> <span class="cit-name-surname">Deffieu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Anuja</span> <span class="cit-name-surname">Krishnan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Esther</span> <span class="cit-name-surname">Ndungo</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2012</span>. “<span class="cit-article-title">Ebola Virus Entry Requires the Host-Programmed Recognition of an Intracellular Receptor</span>.” <abbr class="cit-jnl-abbrev">The EMBO Journal</abbr> <span class="cit-vol">31</span> (<span class="cit-issue">8</span>): <span class="cit-fpage">1947</span>–<span class="cit-lpage">60</span>. <a href="https://doi.org/10.1038/emboj.2012.53">https://doi.org/10.1038/emboj.2012.53</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft_id%253Dinfo%253Adoi%252F10.1038%252Femboj.2012.53%26rft_id%253Dinfo%253Apmid%252F22395071%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo5OiIzMS84LzE5NDciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8wMS82OTYxOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Morris</span>,  <span class="cit-name-given-names">Sinead E.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Andrew J.</span> <span class="cit-name-surname">Yates</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Rik L.</span> <span class="cit-name-surname">de Stwart</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Rory D.</span> <span class="cit-name-surname">de Vries</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Michael J.</span> <span class="cit-name-surname">Mina</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Ashley N.</span> <span class="cit-name-surname">Nelson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Wen-Hsuan W.</span> <span class="cit-name-surname">Lin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Roger D.</span> <span class="cit-name-surname">Kouyos</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Diane E.</span> <span class="cit-name-surname">Griffin</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Bryan T.</span> <span class="cit-name-surname">Grenfell</span></span>. <span class="cit-pub-date">2018</span>. “<span class="cit-article-title">Modeling the Measles Paradox Reveals the Importance of Cellular Immunity in Regulating Viral Clearance</span>.” <abbr class="cit-jnl-abbrev">PLoS Pathogens</abbr> <span class="cit-vol">14</span> (<span class="cit-issue">12</span>): <span class="cit-fpage">e1007493</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathogens%26rft.volume%253D14%26rft.spage%253De1007493%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.32" data-doi="10.1126/science.282.5386.103"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Neumann</span>,  <span class="cit-name-given-names">AU</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">NP</span> <span class="cit-name-surname">Lam</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H</span> <span class="cit-name-surname">Dahari</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">DR</span> <span class="cit-name-surname">Gretch</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">TE</span> <span class="cit-name-surname">Wiley</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">TJ</span> <span class="cit-name-surname">Layden</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">AS</span> <span class="cit-name-surname">Perelson</span></span>. <span class="cit-pub-date">1998</span>. “<span class="cit-article-title">Hepatitis C Viral Dynamics in Vivo and the Antiviral Efficacy of Interferon-Alpha Therapy</span>.” <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DNeumann%26rft.auinit1%253DA.%2BU.%26rft.volume%253D282%26rft.issue%253D5386%26rft.spage%253D103%26rft.epage%253D107%26rft.atitle%253DHepatitis%2BC%2BViral%2BDynamics%2Bin%2BVivo%2Band%2Bthe%2BAntiviral%2BEfficacy%2Bof%2BInterferon-%2BTherapy%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.282.5386.103%26rft_id%253Dinfo%253Apmid%252F9756471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODIvNTM4Ni8xMDMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8wMS82OTYxOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ng</span>,  <span class="cit-name-given-names">Melinda</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Kartik</span> <span class="cit-name-surname">Chandran</span></span>. <span class="cit-pub-date">2018</span>. “<span class="cit-article-title">Unpublished Results</span>.”</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.34" data-doi="10.7554/eLife.11785"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ng</span>,  <span class="cit-name-given-names">Melinda</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Esther</span> <span class="cit-name-surname">Ndungo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Maryska</span> <span class="cit-name-surname">Kaczmarek</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Andrew S.</span> <span class="cit-name-surname">Herbert</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Tabea</span> <span class="cit-name-surname">Binger</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Rebekah</span> <span class="cit-name-surname">James</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Rohit K.</span> <span class="cit-name-surname">Jangra</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2015</span>. “<span class="cit-article-title">NPC1 Contributes to Species-Specific Patterns of Ebola Virus Infection in Bats</span>.” <abbr class="cit-jnl-abbrev">ELife</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e11785</span>. <a href="https://doi.org/10.7554/eLife.11785">https://doi.org/10.7554/eLife.11785</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DELife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.11785%26rft_id%253Dinfo%253Apmid%252F26698106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.11785&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26698106&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F01%2F696195.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.35" data-doi="10.1016/S0140-6736(03)13413-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nicholls</span>,  <span class="cit-name-given-names">J M</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L L</span> <span class="cit-name-surname">Poon</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K C</span> <span class="cit-name-surname">Lee</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W F</span> <span class="cit-name-surname">Ng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S T</span> <span class="cit-name-surname">Lai</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C Y</span> <span class="cit-name-surname">Leung</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C M</span> <span class="cit-name-surname">Chu</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2003</span>. “<span class="cit-article-title">Lung Pathogy of Fatal Severe Acute Respiratory Syndrom</span>.” <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-vol">361</span>: <span class="cit-fpage">1773</span>–<span class="cit-lpage">78</span>. <a href="https://doi.org/10.1016/S0140-6736(03)13413-7">https://doi.org/10.1016/S0140-6736(03)13413-7</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.stitle%253DLancet%26rft.aulast%253DNicholls%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D361%26rft.issue%253D9371%26rft.spage%253D1773%26rft.epage%253D1778%26rft.atitle%253DLung%2Bpathology%2Bof%2Bfatal%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252803%252913413-7%26rft_id%253Dinfo%253Apmid%252F12781536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(03)13413-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12781536&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F01%2F696195.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183074800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-book" id="cit-696195v3.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nowak</span>,  <span class="cit-name-given-names">M. A.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">May</span></span>. <span class="cit-pub-date">2000</span>. <span class="cit-source">Virus Dynamics: Mathematical Principles of Immunology and Virology</span>. <span class="cit-publ-loc">Oxford, UK</span>: <span class="cit-publ-name">Oxford University Press</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.37" data-doi="10.1038/375606a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nowak</span>,  <span class="cit-name-given-names">Martin A.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Robert M.</span> <span class="cit-name-surname">May</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Rodney E.</span> <span class="cit-name-surname">Phillips</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Sarah</span> <span class="cit-name-surname">Rowland-Jones</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">David G.</span> <span class="cit-name-surname">Lalloo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Steven</span> <span class="cit-name-surname">McAdam</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Paul</span> <span class="cit-name-surname">Klenerman</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">1995</span>. “<span class="cit-article-title">Antigenic Oscillations and Shifting Immunodominance in HIV-1 Infections</span>.” <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">375</span>: <span class="cit-fpage">606</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DNowak%26rft.auinit1%253DM.%2BA.%26rft.volume%253D375%26rft.issue%253D6532%26rft.spage%253D606%26rft.epage%253D611%26rft.atitle%253DAntigenic%2Boscillations%2Band%2Bshifting%2Bimmunodominance%2Bin%2BHIV-1%2Binfections.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F375606a0%26rft_id%253Dinfo%253Apmid%252F7791879%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/375606a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7791879&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F01%2F696195.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.38" data-doi="10.1073/pnas.93.9.4398"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nowak</span>,  <span class="cit-name-given-names">Martin A</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Sebastian</span> <span class="cit-name-surname">Bonhoeffer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Andrew M</span> <span class="cit-name-surname">Hill</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Richard</span> <span class="cit-name-surname">Boehme</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Howard C</span> <span class="cit-name-surname">Thomas</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Hugh</span> <span class="cit-name-surname">McDade</span></span>. <span class="cit-pub-date">1996</span>. “<span class="cit-article-title">Viral Dynamics in Hepatitis B Virus Infection</span>.” <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">4398</span>–<span class="cit-lpage">4402</span>. <a href="http://www.pnas.org/content/pnas/93/9/4398.full.pdf">http://www.pnas.org/content/pnas/93/9/4398.full.pdf</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DNowak%26rft.auinit1%253DM.%2BA.%26rft.volume%253D93%26rft.issue%253D9%26rft.spage%253D4398%26rft.epage%253D4402%26rft.atitle%253DViral%2Bdynamics%2Bin%2Bhepatitis%2BB%2Bvirus%2Binfection.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.93.9.4398%26rft_id%253Dinfo%253Apmid%252F8633078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5My85LzQzOTgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8wMS82OTYxOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.39" data-doi="10.1093/bioinformatics/btq046"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pau</span>,  <span class="cit-name-given-names">Gregoire</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Florian</span> <span class="cit-name-surname">Fuchs</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Oleg</span> <span class="cit-name-surname">Sklyar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Michael</span> <span class="cit-name-surname">Boutros</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Wolfgang</span> <span class="cit-name-surname">Huber</span></span>. <span class="cit-pub-date">2010</span>. “<span class="cit-article-title">EBImage-an R Package for Image Processing with Applications to Cellular Phenotypes</span>.” <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span> (<span class="cit-issue">7</span>): <span class="cit-fpage">979</span>–<span class="cit-lpage">81</span>. <a href="https://doi.org/10.1093/bioinformatics/btq046">https://doi.org/10.1093/bioinformatics/btq046</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq046%26rft_id%253Dinfo%253Apmid%252F20338898%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20338898&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F01%2F696195.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276045800024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.40" data-doi="10.1016/j.cell.2018.03.070"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pavlovich</span>,  <span class="cit-name-given-names">Stephanie S.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Sean P.</span> <span class="cit-name-surname">Lovett</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Galina</span> <span class="cit-name-surname">Koroleva</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Jonathan C.</span> <span class="cit-name-surname">Guito</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Catherine E.</span> <span class="cit-name-surname">Arnold</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Elyse R.</span> <span class="cit-name-surname">Nagle</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Kirsten</span> <span class="cit-name-surname">Kulcsar</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2018</span>. “<span class="cit-article-title">The Egyptian Rousette Genome Reveals Unexpected Features of Bat Antiviral Immunity</span>.” <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">173</span> (<span class="cit-issue">5</span>): <span class="cit-fpage">1098</span>–<span class="cit-lpage">1110</span>. <a href="https://doi.org/10.1016/j.cell.2018.03.070">https://doi.org/10.1016/j.cell.2018.03.070</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D173%26rft.spage%253D1098%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2018.03.070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2018.03.070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pawelek</span>,  <span class="cit-name-given-names">Kasia A.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Giao T.</span> <span class="cit-name-surname">Huynh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Michelle</span> <span class="cit-name-surname">Quinlivan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Ann</span> <span class="cit-name-surname">Cullinane</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Libin</span> <span class="cit-name-surname">Rong</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Alan S.</span> <span class="cit-name-surname">Perelson</span></span>. <span class="cit-pub-date">2012</span>. “<span class="cit-article-title">Modeling Within-Host Dynamics of Influenza Virus Infection Including Immune Responses</span>.” <abbr class="cit-jnl-abbrev">PLoS Computational Biology</abbr> <span class="cit-vol">8</span> (<span class="cit-issue">6</span>): <span class="cit-fpage">e1002588</span>. <a href="https://doi.org/10.1371/journal.pcbi.1002588">https://doi.org/10.1371/journal.pcbi.1002588</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BComputational%2BBiology%26rft.volume%253D8%26rft.spage%253D1002588e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peel</span>,  <span class="cit-name-given-names">Alison J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Kate S.</span> <span class="cit-name-surname">Baker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">David T. S.</span> <span class="cit-name-surname">Hayman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Christopher C.</span> <span class="cit-name-surname">Broder</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Andrew A.</span> <span class="cit-name-surname">Cunningham</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Anthony R.</span> <span class="cit-name-surname">Fooks</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Romain</span> <span class="cit-name-surname">Garnier</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">James L. N.</span> <span class="cit-name-surname">Wood</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Olivier</span> <span class="cit-name-surname">Restif</span></span>. <span class="cit-pub-date">2018</span>. “<span class="cit-article-title">Support for Viral Persistence in Bats from Age-Specific Serology and Models of Maternal Immunity</span>.” <abbr class="cit-jnl-abbrev">Scientific Reports</abbr> <span class="cit-vol">8</span> (<span class="cit-issue">1</span>): <span class="cit-fpage">3859</span>. <a href="https://doi.org/10.1038/s41598-018-22236-6">https://doi.org/10.1038/s41598-018-22236-6</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScientific%2BReports%26rft.volume%253D8%26rft.spage%253D3859%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.43" data-doi="10.1371/journal.pone.0030346"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peel</span>,  <span class="cit-name-given-names">Alison J</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Kate S</span> <span class="cit-name-surname">Baker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Gary</span> <span class="cit-name-surname">Crameri</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Jennifer A</span> <span class="cit-name-surname">Barr</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">David TS</span> <span class="cit-name-surname">Hayman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Edward</span> <span class="cit-name-surname">Wright</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Christopher C</span> <span class="cit-name-surname">Broder</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2012</span>. “<span class="cit-article-title">Henipavirus Neutralising Antibodies in an Isolated Island Population of African Fruit Bats</span>.” <abbr class="cit-jnl-abbrev">PloS One</abbr> <span class="cit-vol">7</span> (<span class="cit-issue">1</span>): <span class="cit-fpage">e30346</span>. <a href="https://doi.org/10.1371/journal.pone.0030346">https://doi.org/10.1371/journal.pone.0030346</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPeel%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D7%26rft.issue%253D1%26rft.spage%253De30346%26rft.epage%253De30346%26rft.atitle%253DHenipavirus%2Bneutralising%2Bantibodies%2Bin%2Ban%2Bisolated%2Bisland%2Bpopulation%2Bof%2BAfrican%2Bfruit%2Bbats.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0030346%26rft_id%253Dinfo%253Apmid%252F22253928%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0030346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22253928&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F01%2F696195.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Perelson</span>,  <span class="cit-name-given-names">A S</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A U</span> <span class="cit-name-surname">Neumann</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M</span> <span class="cit-name-surname">Markowitz</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J M</span> <span class="cit-name-surname">Leonard</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D D</span> <span class="cit-name-surname">Ho</span></span>. <span class="cit-pub-date">1996</span>. <span class="cit-article-title">“HIV-1 Dynamics in Vivo: Virion Clearance Rate, Infected Cell Life-Span, and Viral Generation Time.” <em>Science (New York</em></span>, <abbr class="cit-jnl-abbrev">N.Y</abbr><em>.)</em> <span class="cit-vol">271</span> (<span class="cit-issue">5255</span>): <span class="cit-fpage">1582</span>–<span class="cit-lpage">86</span>. <a href="https://doi.org/10.1126/science.271.5255.1582">https://doi.org/10.1126/science.271.5255.1582</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.Y%26rft.volume%253D271%26rft.spage%253D1582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.45" data-doi="10.1038/nri700"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Perelson</span>,  <span class="cit-name-given-names">Alan S</span></span>. <span class="cit-pub-date">2002</span>. “<span class="cit-article-title">Modelling Viral and Immune System Dynamics</span>.” <abbr class="cit-jnl-abbrev">Nature Reviews Immunology</abbr> <span class="cit-vol">2</span> (<span class="cit-issue">1</span>): <span class="cit-fpage">28</span>–<span class="cit-lpage">36</span>. <a href="https://doi.org/10.1038/nri700">https://doi.org/10.1038/nri700</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BImmunology%26rft.stitle%253DNat%2BRev%2BImmunol%26rft.aulast%253DPerelson%26rft.auinit1%253DA.%2BS.%26rft.volume%253D2%26rft.issue%253D1%26rft.spage%253D28%26rft.epage%253D36%26rft.atitle%253DModelling%2Bviral%2Band%2Bimmune%2Bsystem%2Bdynamics.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnri700%26rft_id%253Dinfo%253Apmid%252F11905835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nri700&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11905835&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F01%2F696195.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180398300021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Plowright</span>,  <span class="cit-name-given-names">Raina K.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Alison J.</span> <span class="cit-name-surname">Peel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Daniel G.</span> <span class="cit-name-surname">Streicker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Amy</span> <span class="cit-name-surname">Gilbert</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Hamish</span> <span class="cit-name-surname">McCallum</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">James</span> <span class="cit-name-surname">Wood</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Michelle L.</span> <span class="cit-name-surname">Baker</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Olivier</span> <span class="cit-name-surname">Restif</span></span>. <span class="cit-pub-date">2016</span>. “<span class="cit-article-title">Transmission or Within-Host Dynamics Driving Pulses of Zoonotic Viruses in Reservoir-Host Populations</span>.” <abbr class="cit-jnl-abbrev">PLoS Neglected Tropical Diseases</abbr> <span class="cit-vol">10</span> (<span class="cit-issue">8</span>): <span class="cit-fpage">e0004796</span>. <a href="https://doi.org/10.1371/journal.pntd.0004796">https://doi.org/10.1371/journal.pntd.0004796</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BNeglected%2BTropical%2BDiseases%26rft.volume%253D10%26rft.spage%253D0004796e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Radke</span>,  <span class="cit-name-given-names">K L</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C</span> <span class="cit-name-surname">Colby</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J R</span> <span class="cit-name-surname">Kates</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H M</span> <span class="cit-name-surname">Krider</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D M</span> <span class="cit-name-surname">Prescott</span></span>. <span class="cit-pub-date">1974</span>. “<span class="cit-article-title">Establishment and Maintenance of the Interferon-Induced Antiviral State: Studies in Enucleated Cells</span>.” <abbr class="cit-jnl-abbrev">Journal of Virology</abbr> <span class="cit-vol">13</span> (<span class="cit-issue">3</span>): <span class="cit-fpage">623</span>–<span class="cit-lpage">30</span>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/4362865">http://www.ncbi.nlm.nih.gov/pubmed/4362865</a><a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC355347">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC355347</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DRadke%26rft.auinit1%253DK.%2BL.%26rft.volume%253D13%26rft.issue%253D3%26rft.spage%253D623%26rft.epage%253D630%26rft.atitle%253DEstablishment%2Band%2BMaintenance%2Bof%2Bthe%2BInterferon-Induced%2BAntiviral%2BState%253A%2BStudies%2Bin%2BEnucleated%2BCells%26rft_id%253Dinfo%253Apmid%252F4362865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjEzLzMvNjIzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMDEvNjk2MTk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.48"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rasmussen</span>,  <span class="cit-name-given-names">L.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">L. B.</span> <span class="cit-name-surname">Farley</span></span>. <span class="cit-pub-date">1975</span>. “<span class="cit-article-title">Inhibition of <em>Herpesvirus Hominis</em> Replication by Human Interferon</span>.” <abbr class="cit-jnl-abbrev">Infection and Immunity</abbr> <span class="cit-vol">12</span> (<span class="cit-issue">1</span>): <span class="cit-fpage">104</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%2Band%2BImmunity%26rft.stitle%253DInfect.%2BImmun.%26rft.aulast%253DRasmussen%26rft.auinit1%253DL%26rft.volume%253D12%26rft.issue%253D1%26rft.spage%253D104%26rft.epage%253D108%26rft.atitle%253DInhibition%2Bof%2BHerpesvirus%2Bhominis%2Breplication%2Bby%2Bhuman%2Binterferon.%26rft_id%253Dinfo%253Apmid%252F166922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaWFpIjtzOjU6InJlc2lkIjtzOjg6IjEyLzEvMTA0IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMDEvNjk2MTk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.49" data-doi="10.3181/00379727-132-34285"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rhim</span>,  <span class="cit-name-given-names">J. S.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Schell</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Creasy</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Case</span></span>. <span class="cit-pub-date">1969</span>. “<span class="cit-article-title">Biological Characteristics and Viral Susceptibility of an African Green Monkey Kidney Cell Line (Vero)</span>.” <abbr class="cit-jnl-abbrev">Proceedings of the Society for Experimental Biology and Medicine</abbr> <span class="cit-vol">132</span> (<span class="cit-issue">2</span>): <span class="cit-fpage">670</span>–<span class="cit-lpage">678</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExperimental%2BBiology%2Band%2BMedicine%26rft.stitle%253DExp%2BBiol%2BMed%2B%2528Maywood%2529%26rft.aulast%253DRhim%26rft.auinit1%253DJ.%2BS.%26rft.volume%253D132%26rft.issue%253D2%26rft.spage%253D670%26rft.epage%253D678%26rft.atitle%253DBiological%2BCharacteristics%2Band%2BViral%2BSusceptibility%2Bof%2Ban%2BAfrican%2BGreen%2BMonkey%2BKidney%2BCell%2BLine%2B%2528Vero%2529%26rft_id%253Dinfo%253Adoi%252F10.3181%252F00379727-132-34285%26rft_id%253Dinfo%253Apmid%252F4982209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3181/00379727-132-34285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4982209&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F01%2F696195.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.50" data-doi="10.1128/JVI.02078-09"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Saenz</span>,  <span class="cit-name-given-names">Roberto A</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Michelle</span> <span class="cit-name-surname">Quinlivan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Debra</span> <span class="cit-name-surname">Elton</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Shona</span> <span class="cit-name-surname">MacRae</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Anthony S</span> <span class="cit-name-surname">Blunden</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Jennifer A</span> <span class="cit-name-surname">Mumford</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Janet M</span> <span class="cit-name-surname">Daly</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2010</span>. “<span class="cit-article-title">Dynamics of Influenza Virus Infection and Pathology</span>.” <abbr class="cit-jnl-abbrev">Journal of Virology</abbr> <span class="cit-vol">84</span> (<span class="cit-issue">8</span>): <span class="cit-fpage">3974</span>–<span class="cit-lpage">83</span>. <a href="https://doi.org/10.1128/JVI.02078-09">https://doi.org/10.1128/JVI.02078-09</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02078-09%26rft_id%253Dinfo%253Apmid%252F20130053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg0LzgvMzk3NCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzAxLzY5NjE5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.51"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Samuel</span>,  <span class="cit-name-given-names">C. E.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">G. S.</span> <span class="cit-name-surname">Knutson</span></span>. <span class="cit-pub-date">1982</span>. “<span class="cit-article-title">Mechanism of Interferon Action</span>.” <abbr class="cit-jnl-abbrev">Journal of Biological Chemistry</abbr> <span class="cit-vol">257</span> (<span class="cit-issue">19</span>): <span class="cit-fpage">11791</span>–<span class="cit-lpage">95</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DSamuel%26rft.auinit1%253DC.%2BE.%26rft.volume%253D257%26rft.issue%253D19%26rft.spage%253D11791%26rft.epage%253D11795%26rft.atitle%253DMechanism%2Bof%2Binterferon%2Baction.%2BKinetics%2Bof%2Binduction%2Bof%2Bthe%2Bantiviral%2Bstate%2Band%2Bprotein%2Bphosphorylation%2Bin%2Bmouse%2Bfibroblasts%2Btreated%2Bwith%2Bnatural%2Band%2Bcloned%2Binterferons.%26rft_id%253Dinfo%253Apmid%252F6181059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNTcvMTkvMTE3OTEiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8wMS82OTYxOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.52"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schountz</span>,  <span class="cit-name-given-names">Tony</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Michelle L</span> <span class="cit-name-surname">Baker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">John</span> <span class="cit-name-surname">Butler</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Vincent</span> <span class="cit-name-surname">Munster</span></span>. <span class="cit-pub-date">2017</span>. “<span class="cit-article-title">Immunological Control of Viral Infections in Bats and the Emergence of Viruses Highly Pathogenic to Humans</span>.” <abbr class="cit-jnl-abbrev">Frontiers in Immunology</abbr> 8 (September): Article 1098. <a href="https://doi.org/10.3389/fimmu.2017.01098">https://doi.org/10.3389/fimmu.2017.01098</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">53.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-696195v3.53" data-doi="10.18637/jss.v033.i01"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Soetaert</span>,  <span class="cit-name-given-names">Karline</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Thomas</span> <span class="cit-name-surname">Petzoldt</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">R</span> <span class="cit-name-surname">Woodrow Setzer</span></span>. <span class="cit-pub-date">2010</span>. “<span class="cit-article-title">Package DeSolve: Solving Initial Value Differential Equations in R</span>.” <abbr class="cit-jnl-abbrev">Journal of Statistical Software</abbr> <span class="cit-vol">33</span> (<span class="cit-issue">9</span>): <span class="cit-fpage">1</span>–<span class="cit-lpage">25</span>. <a href="https://doi.org/10.18637/jss.v033.i09">https://doi.org/10.18637/jss.v033.i09</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BStatistical%2BSoftware%26rft.volume%253D33%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.18637%252Fjss.v033.i01%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.18637/jss.v033.i01&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.54" data-doi="10.1016/j.immuni.2006.08.007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stetson</span>,  <span class="cit-name-given-names">Daniel B.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Ruslan</span> <span class="cit-name-surname">Medzhitov</span></span>. <span class="cit-pub-date">2006</span>. “<span class="cit-article-title">Type I Interferons in Host Defense</span>.” <abbr class="cit-jnl-abbrev">Immunity</abbr> <span class="cit-vol">25</span> (<span class="cit-issue">3</span>): <span class="cit-fpage">373</span>–<span class="cit-lpage">81</span>. <a href="https://doi.org/10.1016/j.immuni.2006.08.007">https://doi.org/10.1016/j.immuni.2006.08.007</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunity%26rft.stitle%253DImmunity%26rft.aulast%253DStetson%26rft.auinit1%253DD.%2BB.%26rft.volume%253D25%26rft.issue%253D3%26rft.spage%253D373%26rft.epage%253D381%26rft.atitle%253DType%2BI%2Binterferons%2Bin%2Bhost%2Bdefense.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.immuni.2006.08.007%26rft_id%253Dinfo%253Apmid%252F16979569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.immuni.2006.08.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16979569&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F01%2F696195.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240862200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">55.</span><div class="cit ref-cit ref-book no-rev-xref" id="cit-696195v3.55"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Team, R Core</span>. <span class="cit-pub-date">2019</span>. “<span class="cit-chapter-title">R: A Language and Environment for Statistical Computing</span>.” <span class="cit-source">R Foundation for Statistical Computing</span>. <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.56"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Lin-Fa</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Danielle E</span> <span class="cit-name-surname">Anderson</span></span>. <span class="cit-pub-date">2019</span>. “<span class="cit-article-title">Viruses in Bats and Potential Spillover to Animals and Humans</span>.” <abbr class="cit-jnl-abbrev">Current Opinion in Virology</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">79</span>–<span class="cit-lpage">89</span>. <a href="https://doi.org/10.1016/j.coviro.2018.12.007">https://doi.org/10.1016/j.coviro.2018.12.007</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2BOpinion%2Bin%2BVirology%26rft.volume%253D34%26rft.spage%253D79%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.57" data-doi="10.1016/j.jtbi.2007.06.013"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Webb</span>,  <span class="cit-name-given-names">Steven D.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Matt J.</span> <span class="cit-name-surname">Keeling</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Mike</span> <span class="cit-name-surname">Boots</span></span>. <span class="cit-pub-date">2007</span>. “<span class="cit-article-title">Host-Parasite Interactions between the Local and the Mean-Field: How and When Does Spatial Population Structure Matter?</span>” <abbr class="cit-jnl-abbrev">Journal of Theoretical Biology</abbr> <span class="cit-vol">249</span> (<span class="cit-issue">1</span>): <span class="cit-fpage">140</span>–<span class="cit-lpage">52</span>. <a href="https://doi.org/10.1016/j.jtbi.2007.06.013">https://doi.org/10.1016/j.jtbi.2007.06.013</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DWebb%26rft.auinit1%253DS.%2BD.%26rft.volume%253D249%26rft.issue%253D1%26rft.spage%253D140%26rft.epage%253D152%26rft.atitle%253DHost-parasite%2Binteractions%2Bbetween%2Bthe%2Blocal%2Band%2Bthe%2Bmean-field%253A%2Bhow%2Band%2Bwhen%2Bdoes%2Bspatial%2Bpopulation%2Bstructure%2Bmatter%253F%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jtbi.2007.06.013%26rft_id%253Dinfo%253Apmid%252F17719608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jtbi.2007.06.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17719608&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F01%2F696195.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250847700012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.58" data-doi="10.1128/JVI.01832-09"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wong</span>,  <span class="cit-name-given-names">Anthony C</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Rohini G</span> <span class="cit-name-surname">Sandesara</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Nirupama</span> <span class="cit-name-surname">Mulherkar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Sean P</span> <span class="cit-name-surname">Whelan</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Kartik</span> <span class="cit-name-surname">Chandran</span></span>. <span class="cit-pub-date">2010</span>. “<span class="cit-article-title">A Forward Genetic Strategy Reveals Destabilizing Mutations in the Ebolavirus Glycoprotein That Alter Its Protease Dependence during Cell Entry</span>.” <abbr class="cit-jnl-abbrev">Journal of Virology</abbr> <span class="cit-vol">84</span> (<span class="cit-issue">1</span>): <span class="cit-fpage">163</span>–<span class="cit-lpage">75</span>. <a href="https://doi.org/10.1128/JVI.01832-09">https://doi.org/10.1128/JVI.01832-09</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01832-09%26rft_id%253Dinfo%253Apmid%252F19846533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6Ijg0LzEvMTYzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMDEvNjk2MTk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.59"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xie</span>,  <span class="cit-name-given-names">Jiazheng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Yang</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Xurui</span> <span class="cit-name-surname">Shen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Jiazheng</span> <span class="cit-name-surname">Xie</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Yang</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Xurui</span> <span class="cit-name-surname">Shen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Geraldine</span> <span class="cit-name-surname">Goh</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2018</span>. “<span class="cit-article-title">Dampened STING-Dependent Interferon Activation in Bats</span>.” <abbr class="cit-jnl-abbrev">Cell Host and Microbe</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">297</span>–<span class="cit-lpage">301</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2Band%2BMicrobe%26rft.volume%253D23%26rft.spage%253D297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.60" data-doi="10.1126/science.1230835"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Guojie</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Christopher</span> <span class="cit-name-surname">Cowled</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Zhengli</span> <span class="cit-name-surname">Shi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Zhiyong</span> <span class="cit-name-surname">Huang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Kimberly a</span> <span class="cit-name-surname">Bishop-Lilly</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Xiaodong</span> <span class="cit-name-surname">Fang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">James W</span> <span class="cit-name-surname">Wynne</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2013</span>. “<span class="cit-article-title">Comparative Analysis of Bat Genomes Provides Insight into the Evolution of Flight and Immunity</span>.” <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">339</span> (<span class="cit-issue">6118</span>): <span class="cit-fpage">456</span>–<span class="cit-lpage">60</span>. <a href="https://doi.org/10.1126/science.1230835">https://doi.org/10.1126/science.1230835</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1230835%26rft_id%253Dinfo%253Apmid%252F23258410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzkvNjExOC80NTYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8wMS82OTYxOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-696195v3.61" data-doi="10.1073/pnas.1518240113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">Peng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Mary</span> <span class="cit-name-surname">Tachedjian</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">James W</span> <span class="cit-name-surname">Wynne</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Victoria</span> <span class="cit-name-surname">Boyd</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Jie</span> <span class="cit-name-surname">Cui</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Ina</span> <span class="cit-name-surname">Smith</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Christopher</span> <span class="cit-name-surname">Cowled</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2016</span>. “<span class="cit-article-title">Contraction of the Type I IFN Locus and Unusual Constitutive Expression of IFN-α in Bats</span>.” <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr> <span class="cit-vol">113</span> (<span class="cit-issue">10</span>): <span class="cit-fpage">2696</span>–<span class="cit-lpage">2701</span>. <a href="https://doi.org/10.1073/pnas.1518240113">https://doi.org/10.1073/pnas.1518240113</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1518240113%26rft_id%253Dinfo%253Apmid%252F26903655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEzLzEwLzI2OTYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8wMS82OTYxOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/696195v3.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F696195v3" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.01.28.923037v3" title="Ultrafiltration Segregates Tissue Regenerative Stimuli Harboured Within and Independent of Extracellular Vesicles" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/855320v2" title="The impact of global selection on local adaptation and reproductive isolation" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;February 01, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/696195v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/early/2020/02/01/696195.external-links" target="_self" class="link-icon"><i class="icon-file"></i> <span class="title">Data/Code</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/696195v3.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/696195v3" class="active" data-icon-position="" data-hide-link-title="0">Accelerated viral dynamics in bat cell lines, with implications for zoonotic emergence</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1130755" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-51zbAG9Z8fD8Sd04xkff7BpDYkIfYvJKPXfoX63vsSE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1130755" id="node1130755--2" data-pisa="biorxiv;696195v3" data-pisa-master="biorxiv;696195" data-seqnum="1130755" data-apath="/biorxiv/early/2020/02/01/696195.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Accelerated viral dynamics in bat cell lines, with implications for zoonotic emergence</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Cara E.</span> <span class="nlm-surname">Brook</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Mike</span> <span class="nlm-surname">Boots</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Kartik</span> <span class="nlm-surname">Chandran</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Andrew P.</span> <span class="nlm-surname">Dobson</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Christian</span> <span class="nlm-surname">Drosten</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Andrea L.</span> <span class="nlm-surname">Graham</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Bryan T.</span> <span class="nlm-surname">Grenfell</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Marcel A.</span> <span class="nlm-surname">Müller</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Melinda</span> <span class="nlm-surname">Ng</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Lin-Fa</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Anieke</span> <span class="nlm-surname">van Leeuwen</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">696195; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/696195 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/696195v3" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/696195v3&amp;title=Accelerated%20viral%20dynamics%20in%20bat%20cell%20lines%2C%20with%20implications%20for%20zoonotic%20emergence" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/696195v3&amp;title=Accelerated%20viral%20dynamics%20in%20bat%20cell%20lines%2C%20with%20implications%20for%20zoonotic%20emergence" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/696195v3&amp;text=Accelerated%20viral%20dynamics%20in%20bat%20cell%20lines%2C%20with%20implications%20for%20zoonotic%20emergence" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/696195v3&amp;title=Accelerated%20viral%20dynamics%20in%20bat%20cell%20lines%2C%20with%20implications%20for%20zoonotic%20emergence" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/696195v3&amp;t=Accelerated%20viral%20dynamics%20in%20bat%20cell%20lines%2C%20with%20implications%20for%20zoonotic%20emergence" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/696195v3&amp;title=Accelerated%20viral%20dynamics%20in%20bat%20cell%20lines%2C%20with%20implications%20for%20zoonotic%20emergence" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/696195v3&amp;title=Accelerated%20viral%20dynamics%20in%20bat%20cell%20lines%2C%20with%20implications%20for%20zoonotic%20emergence" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1130755" id="node1130755--3" data-pisa="biorxiv;696195v3" data-pisa-master="biorxiv;696195" data-seqnum="1130755" data-apath="/biorxiv/early/2020/02/01/696195.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Accelerated viral dynamics in bat cell lines, with implications for zoonotic emergence</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Cara E.</span> <span class="nlm-surname">Brook</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Mike</span> <span class="nlm-surname">Boots</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Kartik</span> <span class="nlm-surname">Chandran</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Andrew P.</span> <span class="nlm-surname">Dobson</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Christian</span> <span class="nlm-surname">Drosten</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Andrea L.</span> <span class="nlm-surname">Graham</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Bryan T.</span> <span class="nlm-surname">Grenfell</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Marcel A.</span> <span class="nlm-surname">Müller</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Melinda</span> <span class="nlm-surname">Ng</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Lin-Fa</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Anieke</span> <span class="nlm-surname">van Leeuwen</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">696195; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/696195 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Accelerated%20viral%20dynamics%20in%20bat%20cell%20lines%2C%20with%20implications%20for%20zoonotic%20emergence&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Brook%2C+Cara+E.&amp;rft.au=Boots%2C+Mike&amp;rft.au=Chandran%2C+Kartik&amp;rft.au=Dobson%2C+Andrew+P.&amp;rft.au=Drosten%2C+Christian&amp;rft.au=Graham%2C+Andrea+L.&amp;rft.au=Grenfell%2C+Bryan+T.&amp;rft.au=M%C3%BCller%2C+Marcel+A.&amp;rft.au=Ng%2C+Melinda&amp;rft.au=Wang%2C+Lin-Fa&amp;rft.au=van+Leeuwen%2C+Anieke"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1130755/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1130755/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1130755/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1130755/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1130755/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1130755/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1130755/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1130755/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1130755/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1130755/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1130755/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1130755/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/696195v3&amp;count=horizontal&amp;via=&amp;text=Accelerated%20viral%20dynamics%20in%20bat%20cell%20lines%2C%20with%20implications%20for%20zoonotic%20emergence&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/696195v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/696195v3&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/696195v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/ecology" class="highlight" data-icon-position="" data-hide-link-title="0">Ecology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
