<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/03/30/2020.03.29.014209.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.29.014209v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Orthogonal genome-wide screenings in bat cells identify MTHFD1 as a target of broad antiviral therapy" />
<meta name="DC.Identifier" content="10.1101/2020.03.29.014209" />
<meta name="DC.Date" content="2020-03-30" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Bats are responsible for the zoonotic transmission of several major viral diseases including the 2003 SARS outbreak and the ongoing COVID-19 pandemic. While bat genomic sequencing studies have revealed characteristic adaptations of the innate immune system, functional genomic studies are urgently needed to provide a foundation for the molecular dissection of the tolerance of viral infections in bats. Here we report the establishment and screening of genome-wide RNAi library and CRISPR library for the model megabat, Pteropus Alecto . We used the complementary RNAi and CRISPR libraries to interrogate Pteropus Alecto cells for infection with two different viruses, mumps virus and Influenza A virus, respectively. Screening results converged on the endocytosis pathway and the protein secretory pathway as required for both viral infections. Additionally, we revealed a general dependence of the C-1-tetrahydrofolate synthase gene, MTHFD1, for viral replication in bat cells as well as in human cells. MTHFD1 inhibitor carolacton potently blocked replication of several RNA viruses including SARS-CoV-2. Our studies provide a resource for systematic inquiry into the genetic underpinnings of bat biology and a potential target for developing broad spectrum antiviral therapy." />
<meta name="DC.Contributor" content="Danielle E Anderson" />
<meta name="DC.Contributor" content="Jin Cui" />
<meta name="DC.Contributor" content="Qian Ye" />
<meta name="DC.Contributor" content="Baoying Huang" />
<meta name="DC.Contributor" content="Wenhong Zu" />
<meta name="DC.Contributor" content="Jing Gong" />
<meta name="DC.Contributor" content="Weiqiang Liu" />
<meta name="DC.Contributor" content="So Young Kim" />
<meta name="DC.Contributor" content="Biao Guo Yan" />
<meta name="DC.Contributor" content="Kristmundur Sigmundsson" />
<meta name="DC.Contributor" content="Xiao Fang Lim" />
<meta name="DC.Contributor" content="Fei Ye" />
<meta name="DC.Contributor" content="Peihua Niu" />
<meta name="DC.Contributor" content="Xuming Zhou" />
<meta name="DC.Contributor" content="Wenjie Tan" />
<meta name="DC.Contributor" content="Lin-Fa Wang" />
<meta name="DC.Contributor" content="Xu Tan" />
<meta name="article:published_time" content="2020-03-30" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Orthogonal genome-wide screenings in bat cells identify MTHFD1 as a target of broad antiviral therapy" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Bats are responsible for the zoonotic transmission of several major viral diseases including the 2003 SARS outbreak and the ongoing COVID-19 pandemic. While bat genomic sequencing studies have revealed characteristic adaptations of the innate immune system, functional genomic studies are urgently needed to provide a foundation for the molecular dissection of the tolerance of viral infections in bats. Here we report the establishment and screening of genome-wide RNAi library and CRISPR library for the model megabat, Pteropus Alecto. We used the complementary RNAi and CRISPR libraries to interrogate Pteropus Alecto cells for infection with two different viruses, mumps virus and Influenza A virus, respectively. Screening results converged on the endocytosis pathway and the protein secretory pathway as required for both viral infections. Additionally, we revealed a general dependence of the C-1-tetrahydrofolate synthase gene, MTHFD1, for viral replication in bat cells as well as in human cells. MTHFD1 inhibitor carolacton potently blocked replication of several RNA viruses including SARS-CoV-2. Our studies provide a resource for systematic inquiry into the genetic underpinnings of bat biology and a potential target for developing broad spectrum antiviral therapy.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.29.014209v1" />
<meta name="citation_id" content="2020.03.29.014209v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.29.014209v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.29.014209v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.29.014209v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.29.014209" />
<meta name="citation_num_pages" content="28" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.29.014209" />
<meta name="citation_author" content="Danielle E Anderson" />
<meta name="citation_author_institution" content="Programme in Emerging Infectious Diseases, Duke–NUS Medical School" />
<meta name="citation_author" content="Jin Cui" />
<meta name="citation_author_institution" content="MOE Key Laboratory of Bioorganic Phosphorus Chemistry &amp; Chemical Biology, Beijing Advanced Innovation Center for Structural Biology, School of Pharmaceutical Sciences, Tsinghua-Peking Center for Life Sciences, Center for Infectious Disease Research, School of Medicine, Tsinghua University" />
<meta name="citation_author" content="Qian Ye" />
<meta name="citation_author_institution" content="MOE Key Laboratory of Bioorganic Phosphorus Chemistry &amp; Chemical Biology, Beijing Advanced Innovation Center for Structural Biology, School of Pharmaceutical Sciences, Tsinghua-Peking Center for Life Sciences, Center for Infectious Disease Research, School of Medicine, Tsinghua University" />
<meta name="citation_author" content="Baoying Huang" />
<meta name="citation_author_institution" content="NHC Key Laboratory of Biosafety, Ministry of Health, National Institute for Viral Disease Control and Prevention" />
<meta name="citation_author" content="Wenhong Zu" />
<meta name="citation_author_institution" content="MOE Key Laboratory of Bioorganic Phosphorus Chemistry &amp; Chemical Biology, Beijing Advanced Innovation Center for Structural Biology, School of Pharmaceutical Sciences, Tsinghua-Peking Center for Life Sciences, Center for Infectious Disease Research, School of Medicine, Tsinghua University" />
<meta name="citation_author" content="Jing Gong" />
<meta name="citation_author_institution" content="MOE Key Laboratory of Bioorganic Phosphorus Chemistry &amp; Chemical Biology, Beijing Advanced Innovation Center for Structural Biology, School of Pharmaceutical Sciences, Tsinghua-Peking Center for Life Sciences, Center for Infectious Disease Research, School of Medicine, Tsinghua University" />
<meta name="citation_author" content="Weiqiang Liu" />
<meta name="citation_author_institution" content="Institute of Zoology, Chinese Academy of Science" />
<meta name="citation_author" content="So Young Kim" />
<meta name="citation_author_institution" content="Department of Molecular Genetics and Microbiology, Duke University" />
<meta name="citation_author" content="Biao Guo Yan" />
<meta name="citation_author_institution" content="Programme in Emerging Infectious Diseases, Duke–NUS Medical School" />
<meta name="citation_author" content="Kristmundur Sigmundsson" />
<meta name="citation_author_institution" content="Programme in Emerging Infectious Diseases, Duke–NUS Medical School" />
<meta name="citation_author" content="Xiao Fang Lim" />
<meta name="citation_author_institution" content="Programme in Emerging Infectious Diseases, Duke–NUS Medical School" />
<meta name="citation_author" content="Fei Ye" />
<meta name="citation_author_institution" content="NHC Key Laboratory of Biosafety, Ministry of Health, National Institute for Viral Disease Control and Prevention" />
<meta name="citation_author" content="Peihua Niu" />
<meta name="citation_author_institution" content="NHC Key Laboratory of Biosafety, Ministry of Health, National Institute for Viral Disease Control and Prevention" />
<meta name="citation_author" content="Xuming Zhou" />
<meta name="citation_author_institution" content="Institute of Zoology, Chinese Academy of Science" />
<meta name="citation_author" content="Wenjie Tan" />
<meta name="citation_author_institution" content="NHC Key Laboratory of Biosafety, Ministry of Health, National Institute for Viral Disease Control and Prevention" />
<meta name="citation_author" content="Lin-Fa Wang" />
<meta name="citation_author_institution" content="Programme in Emerging Infectious Diseases, Duke–NUS Medical School" />
<meta name="citation_author_email" content="xutan@tsinghua.edu.cn" />
<meta name="citation_author_email" content="linfa.wang@duke-nus.edu.sg" />
<meta name="citation_author" content="Xu Tan" />
<meta name="citation_author_institution" content="MOE Key Laboratory of Bioorganic Phosphorus Chemistry &amp; Chemical Biology, Beijing Advanced Innovation Center for Structural Biology, School of Pharmaceutical Sciences, Tsinghua-Peking Center for Life Sciences, Center for Infectious Disease Research, School of Medicine, Tsinghua University" />
<meta name="citation_author_email" content="xutan@tsinghua.edu.cn" />
<meta name="citation_author_email" content="linfa.wang@duke-nus.edu.sg" />
<meta name="citation_reference" content="Leroy, E. M. et al. Fruit bats as reservoirs of Ebola virus. Nature, 438, 575–576, doi:10.1038/438575a (2005)." />
<meta name="citation_reference" content="Li, W. et al. Bats are natural reservoirs of SARS-like coronaviruses. Science 310, 676–679, doi:10.1126/science.1118391 (2005)." />
<meta name="citation_reference" content="Clayton, B. A., Wang, L. F. &amp; Marsh, G. A. Henipaviruses: an updated review focusing on the pteropid reservoir and features of transmission. Zoonoses Public Health 60, 69–83, doi:10.1111/j.1863-2378.2012.01501.x (2013)." />
<meta name="citation_reference" content="Mohd, H. A., Al-Tawfiq, J. A. &amp; Memish, Z. A. Middle East Respiratory Syndrome Coronavirus (MERS-CoV) origin and animal reservoir. Virol J 13, 87, doi:10.1186/s12985-016-0544-0 (2016)." />
<meta name="citation_reference" content="Wu, F. et al. A new coronavirus associated with human respiratory disease in China. Nature, doi:10.1038/s41586-020-2008-3 (2020)." />
<meta name="citation_reference" content="Zhou, P. et al. A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature, doi:10.1038/s41586-020-2012-7 (2020)." />
<meta name="citation_reference" content="Olival, K. J. et al. Host and viral traits predict zoonotic spillover from mammals. Nature 546, 646–650, doi:10.1038/nature22975 (2017)." />
<meta name="citation_reference" content="Wang, L. F. &amp; Anderson, D. E. Viruses in bats and potential spillover to animals and humans. Curr Opin Virol 34, 79–89, doi:10.1016/j.coviro.2018.12.007 (2019)." />
<meta name="citation_reference" content="Zhang, G. et al. Comparative analysis of bat genomes provides insight into the evolution of flight and immunity. Science 339, 456–460, doi:10.1126/science.1230835 (2013)." />
<meta name="citation_reference" content="Pavlovich, S. S. et al. The Egyptian Rousette Genome Reveals Unexpected Features of Bat Antiviral Immunity. Cell 173, 1098–1110 e1018, doi:10.1016/j.cell.2018.03.070 (2018)." />
<meta name="citation_reference" content="Mohr, S., Bakal, C. &amp; Perrimon, N. Genomic screening with RNAi: results and challenges. Annu Rev Biochem 79, 37–64, doi:10.1146/annurev-biochem-060408-092949 (2010)." />
<meta name="citation_reference" content="Shalem, O., Sanjana, N. E. &amp; Zhang, F. High-throughput functional genomics using CRISPR-Cas9. Nat Rev Genet 16, 299–311, doi:10.1038/nrg3899 (2015)." />
<meta name="citation_reference" content="McDougall, W. M., Perreira, J. M., Reynolds, E. C. &amp; Brass, A. L. CRISPR genetic screens to discover host-virus interactions. Curr Opin Virol 29, 87–100, doi:10.1016/j.coviro.2018.03.007 (2018)." />
<meta name="citation_reference" content="Panda, D. &amp; Cherry, S. Cell-based genomic screening: elucidating virus-host interactions. Curr Opin Virol 2, 784–792, doi:10.1016/j.coviro.2012.10.007 (2012)." />
<meta name="citation_reference" content="Puschnik, A. S., Majzoub, K., Ooi, Y. S. &amp; Carette, J. E. A CRISPR toolbox to study virushost interactions. Nat Rev Microbiol 15, 351–364, doi:10.1038/nrmicro.2017.29 (2017)." />
<meta name="citation_reference" content="Schuster, A. et al. RNAi/CRISPR Screens: from a Pool to a Valid Hit. Trends Biotechnol 37, 38–55, doi:10.1016/j.tibtech.2018.08.002 (2019)." />
<meta name="citation_reference" content="Perreira, J. M., Meraner, P. &amp; Brass, A. L. Functional Genomic Strategies for Elucidating Human-Virus Interactions: Will CRISPR Knockout RNAi and Haploid Cells? Adv Virus Res 94, 1–51, doi:10.1016/bs.aivir.2015.11.001 (2016)." />
<meta name="citation_reference" content="Taylor, J. &amp; Woodcock, S. A Perspective on the Future of High-Throughput RNAi Screening: Will CRISPR Cut Out the Competition or Can RNAi Help Guide the Way? J Biomol Screen 20, 1040–1051, doi:10.1177/1087057115590069 (2015)." />
<meta name="citation_reference" content="Shalem, O. et al. Genome-scale CRISPR-Cas9 knockout screening in human cells. Science 343, 84–87, doi:10.1126/science.1247005 (2014)." />
<meta name="citation_reference" content="Luo, B. et al. Highly parallel identification of essential genes in cancer cells. Proc Natl AcadSci USA 105, 20380–20385, doi:10.1073/pnas.0810485105 (2008)." />
<meta name="citation_reference" content="Watanabe, T., Watanabe, S. &amp; Kawaoka, Y. Cellular networks involved in the influenza virus life cycle. Cell Host Microbe 7, 427–439, doi:10.1016/j.chom.2010.05.008 (2010)." />
<meta name="citation_reference" content="Zhu, P. et al. Host Cellular Protein TRAPPC6ADelta Interacts with Influenza A Virus M2 Protein and Regulates Viral Propagation by Modulating M2 Trafficking. J Virol 91, doi:10.1128/JVI.01757-16 (2017)." />
<meta name="citation_reference" content="Shah, P. S. et al. Comparative Flavivirus-Host Protein Interaction Mapping Reveals Mechanisms of Dengue and Zika Virus Pathogenesis. Cell 175, 1931–1945 e1918, doi:10.1016/j.cell.2018.11.028 (2018)." />
<meta name="citation_reference" content="Sanjana, N. E., Shalem, O. &amp; Zhang, F. Improved vectors and genome-wide libraries for CRISPR screening. Nat Methods 11, 783–784, doi:10.1038/nmeth.3047 (2014)." />
<meta name="citation_reference" content="Anderson, D. E. et al. Comparative Loss-of-Function Screens Reveal ABCE1 as an Essential Cellular Host Factor for Efficient Translation of Paramyxoviridae and Pneumoviridae. mBio 10, doi:10.1128/mBio.00826-19 (2019)." />
<meta name="citation_reference" content="van Gent, M., Sparrer, K. M. J. &amp; Gack, M. U. TRIM Proteins and Their Roles in Antiviral Host Defenses. Annu Rev Virol 5, 385–405, doi:10.1146/annurev-virology-092917-043323 (2018)." />
<meta name="citation_reference" content="Schoggins, J. W. et al. A diverse range of gene products are effectors of the type I interferon antiviral response. Nature 472, 481–485, doi:10.1038/nature09907 (2011)." />
<meta name="citation_reference" content="Chua, K. B. et al. A previously unknown reovirus of bat origin is associated with an acute respiratory disease in humans. Proc Natl Acad Sci USA 104, 11424–11429, doi:10.1073/pnas.0701372104 (2007)." />
<meta name="citation_reference" content="Ducker, G. S. &amp; Rabinowitz, J. D. One-Carbon Metabolism in Health and Disease. Cell Metab 25, 27–42, doi:10.1016/j.cmet.2016.08.009 (2017)." />
<meta name="citation_reference" content="Field, M. S., Kamynina, E., Watkins, D., Rosenblatt, D. S. &amp; Stover, P. J. Human mutations in methylenetetrahydrofolate dehydrogenase 1 impair nuclear de novo thymidylate biosynthesis. Proc Natl Acad Sci US A 112, 400–405, doi:10.1073/pnas.1414555112 (2015)." />
<meta name="citation_reference" content="Sundararajan, S. &amp; MacKenzie, R. E. Residues involved in the mechanism of the bifunctional methylenetetrahydrofolate dehydrogenase-cyclohydrolase: the roles of glutamine 100 and aspartate 125. J Biol Chem 277, 18703–18709, doi:10.1074/jbc.M200127200 (2002)." />
<meta name="citation_reference" content="Celeste, L. R. et al. Mechanism of N10-formyltetrahydrofolate synthetase derived from complexes with intermediates and inhibitors. Protein Sci 21, 219–228, doi:10.1002/pro.2005 (2012)." />
<meta name="citation_reference" content="Fu, C. et al. The natural product carolacton inhibits folate-dependent C1 metabolism by targeting FolD/MTHFD. Nat Commun 8, 1529, doi:10.1038/s41467-017-01671-5 (2017)." />
<meta name="citation_reference" content="Martyn, G. E., Quinlan, K. G. R. &amp; Crossley, M. The regulation of human globin promoters by CCAAT box elements and the recruitment of NF-Y. Biochim Biophys Acta Gene Regul Mech 1860, 525–536, doi:10.1016/j.bbagrm.2016.10.002 (2017)." />
<meta name="citation_reference" content="Luo, J., Solimini, N. L. &amp; Elledge, S. J. Principles of cancer therapy: oncogene and nononcogene addiction. Cell 136, 823–837, doi:10.1016/j.cell.2009.02.024 (2009)." />
<meta name="citation_reference" content="Deans, R. M. et al. Parallel shRNA and CRISPR-Cas9 screens enable antiviral drug target identification. Nat Chem Biol 12, 361–366, doi:10.1038/nchembio.2050 (2016)." />
<meta name="citation_reference" content="Luo, B. et al. Highly parallel identification of essential genes in cancer cells. Proc Natl Acad Sci USA 105, 20380–20385, doi:10.1073/pnas.0810485105 (2008)." />
<meta name="citation_reference" content="https://doi.org/10.14806/ej.17.1.200" />
<meta name="citation_reference" content="Langmead, B. &amp; Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. Nat Methods 9, 357–359, doi:10.1038/nmeth.1923 (2012)." />
<meta name="citation_reference" content="Li, H. et al. The Sequence Alignment/Map format and SAMtools. Bioinformatics 25, 2078–2079, doi:10.1093/bioinformatics/btp352 (2009)." />
<meta name="citation_reference" content="Anderson, D. E. et al. Comparative Loss-of-Function Screens Reveal ABCE1 as an Essential Cellular Host Factor for Efficient Translation of Paramyxoviridae and Pneumoviridae. mBio 10, doi:10.1128/mBio.00826-19 (2019)." />
<meta name="citation_reference" content="Malo, N., Hanley, J. A., Cerquozzi, S., Pelletier, J. &amp; Nadon, R. Statistical practice in high-throughput screening data analysis. Nat Biotechnol 24, 167–175, doi:10.1038/nbt1186 (2006)" />
<meta name="citation_reference" content="Reed, LJ, Muench, H. A simple method of estimating fifty percent endpoints. Am. J. Hygiene 27: 493–497 (1938)." />
<meta name="twitter:title" content="Orthogonal genome-wide screenings in bat cells identify MTHFD1 as a target of broad antiviral therapy" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Bats are responsible for the zoonotic transmission of several major viral diseases including the 2003 SARS outbreak and the ongoing COVID-19 pandemic. While bat genomic sequencing studies have revealed characteristic adaptations of the innate immune system, functional genomic studies are urgently needed to provide a foundation for the molecular dissection of the tolerance of viral infections in bats. Here we report the establishment and screening of genome-wide RNAi library and CRISPR library for the model megabat, Pteropus Alecto . We used the complementary RNAi and CRISPR libraries to interrogate Pteropus Alecto cells for infection with two different viruses, mumps virus and Influenza A virus, respectively. Screening results converged on the endocytosis pathway and the protein secretory pathway as required for both viral infections. Additionally, we revealed a general dependence of the C-1-tetrahydrofolate synthase gene, MTHFD1, for viral replication in bat cells as well as in human cells. MTHFD1 inhibitor carolacton potently blocked replication of several RNA viruses including SARS-CoV-2. Our studies provide a resource for systematic inquiry into the genetic underpinnings of bat biology and a potential target for developing broad spectrum antiviral therapy." />
<meta name="og-title" property="og:title" content="Orthogonal genome-wide screenings in bat cells identify MTHFD1 as a target of broad antiviral therapy" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.29.014209v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Bats are responsible for the zoonotic transmission of several major viral diseases including the 2003 SARS outbreak and the ongoing COVID-19 pandemic. While bat genomic sequencing studies have revealed characteristic adaptations of the innate immune system, functional genomic studies are urgently needed to provide a foundation for the molecular dissection of the tolerance of viral infections in bats. Here we report the establishment and screening of genome-wide RNAi library and CRISPR library for the model megabat, Pteropus Alecto . We used the complementary RNAi and CRISPR libraries to interrogate Pteropus Alecto cells for infection with two different viruses, mumps virus and Influenza A virus, respectively. Screening results converged on the endocytosis pathway and the protein secretory pathway as required for both viral infections. Additionally, we revealed a general dependence of the C-1-tetrahydrofolate synthase gene, MTHFD1, for viral replication in bat cells as well as in human cells. MTHFD1 inhibitor carolacton potently blocked replication of several RNA viruses including SARS-CoV-2. Our studies provide a resource for systematic inquiry into the genetic underpinnings of bat biology and a potential target for developing broad spectrum antiviral therapy." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-03-30" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.29.014209v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.29.014209v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.29.014209v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.29.014209v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1217863" />
    <title>Orthogonal genome-wide screenings in bat cells identify MTHFD1 as a target of broad antiviral therapy | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"GD1HA3-HtLhmWaXaVcH2tlMvhS4HAU5KEGPyU4-US80","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1217863","apath":"\/biorxiv\/early\/2020\/03\/30\/2020.03.29.014209.atom","pisa":"biorxiv;2020.03.29.014209v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.29.014209v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.29.014209v1","title":"Orthogonal genome-wide screenings in bat cells identify MTHFD1 as a target of broad antiviral therapy","identifier":"node\/1217863"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ssBhBOFzeJzcjArvEwSbsM9nB\\\\\\\/QrRfRd37cNlM+mjwnfY+Yl4lFTCtNYCSc2oPrhmfBfppbjktoNDYEPWpChlNKe5YGmT0Sbw6vi6o6AhqWpIbx4XpFOPGF10NR8hF2ZJnc8ypZUnQ3CPNVvOz2G7h+43xdoTyrpMMdhNePQJ7RH5q1ZQCZMKdMq4MG++uMQlUBr+bz5XAlJELcdl3KM4xIgt9fEdABipRwNflNe0dr3EmEsZU8TWHbC6tNKUAIuWfCdFrHyYVWCQM6nJiICYVpwwj0XdrpQD22hfSDWPq0H9zoY3vr7RDbtbeRwoE\\\\\\\/lARc8n1+Zyd0QPSSgJkFyLYWSUK6pkIdaqI6jAWAy11JD5\\\\\\\/tm2K18V6oeqmxZ5upcKyYXCvHtm0a3rlPjUJPn3CkaBkX\\\\\\\/wSBoueI5IEgoc0\\\\\\\/oWc1blILQ3kcEzzuwG1A0sAeMsYw2wFrynB42+twOOJfPZrL7fd6jarJxuNsSO72FxH79iiIWrwdbQPZR410R\\\\\\\/8Lm7h3Xb\\\\\\\/af84b5DdM+CS3Yn59mLvXWzd3r5\\\\\\\/R3srzSAIzD92jzigz+d5m84h6XVqOXdVY+RIffbEMvMF2yfo9s6+2IhDFabp9NMEN5ZwX96qewMEPw10ckmqOwHWEBQlnJlt87iNJdwxpNmrZuu3\\\\\\\/PNeSV2b1QMb+mKW5fbFHVyyjLUVNwIoFM9Jos5CTN+MoOxMySaciOojQ8BayUPbMWvzHoM2iJHP\\\\\\\/sAheW\\\\\\\/w7KryfHDUOHLAdNB3cz2WRoqhwuL9TcEUfdcFiCuLde+ipNa1k+23hNORaS0dPnWDaHx4WOhAEh4dEItr24w4HSD+Hpwe8eeQDsdg5CiuNx9eJGqbbL5aZI7r2dumyc9me1W7p3ekAGE6c2gMtCBsyjuay2u8\\\\\\\/g4xLMuAFtQTcL6144ILkQZhp6GcQVQ95c8z6AOEAL4yv3asTk7vBqcX2027qhqZi9d6u0T8H1xlcrJC+sJNVXoy6uC1MdQjdm61zEzeR0EimUs6eBc661J0kpk3l8OM8Zbh8BAnh2SgjxZKYFrYM8OtElrFBokS0hTOc8xib+nHk1fSHICK5MJECMdVTkB+c7tLY+chZnxetJtvVXgoMzT9dkfVUm3\\\\\\\/04BiFLrpnpdCcOfQ71h9MtHvXBs7GLWBOR7hPLvQUltAbJLS9xuX1p+DVeLPHT5oflc6kvQE9HMPC79MWqAB8\\\\\\\/Ca6cP3KBoltQVyqxUm47\\\\\\\/GXlLV3+5RbnAfM6W3SyhLsILH7N0cHtYYC5rNb3Fc5\\\\\\\/5MsIC\\\\\\\/+KrCZg44R2tHw4K7KYeLTV4MT8JnBmvLOHij0lHjKxFOO1aSsM1EQmqFbk5vHZ\\\u0022,\\\u0022iv\\\u0022:\\\u002206u+TG\\\\\\\/scHpGBZ1ZDk7lWg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00227d1191449abba34d98bf21ac462ce34e708c3f89169a292fb5d95c7cc558bd6f\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.29.014209v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1217863 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.29.014209v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-_a2Ya4XJTDJSbeD8K-n_5mssTiXNLJN9EsDK6hUikoQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1217863" id="node1217863" data-pisa="biorxiv;2020.03.29.014209v1" data-pisa-master="biorxiv;2020.03.29.014209" data-apath="/biorxiv/early/2020/03/30/2020.03.29.014209.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Orthogonal genome-wide screenings in bat cells identify MTHFD1 as a target of broad antiviral therapy</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Danielle E</span> <span class="nlm-surname">Anderson</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Jin</span> <span class="nlm-surname">Cui</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Qian</span> <span class="nlm-surname">Ye</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Baoying</span> <span class="nlm-surname">Huang</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Wenhong</span> <span class="nlm-surname">Zu</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Jing</span> <span class="nlm-surname">Gong</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Weiqiang</span> <span class="nlm-surname">Liu</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">So Young</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Biao Guo</span> <span class="nlm-surname">Yan</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Kristmundur</span> <span class="nlm-surname">Sigmundsson</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Xiao Fang</span> <span class="nlm-surname">Lim</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Fei</span> <span class="nlm-surname">Ye</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Peihua</span> <span class="nlm-surname">Niu</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Xuming</span> <span class="nlm-surname">Zhou</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Wenjie</span> <span class="nlm-surname">Tan</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Lin-Fa</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">Xu</span> <span class="nlm-surname">Tan</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.29.014209 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1217863" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Danielle E Anderson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Programme in Emerging Infectious Diseases, Duke–NUS Medical School</span>, Singapore, <span class='nlm-country'>Singapore</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Danielle%2BE%2BAnderson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Anderson%20DE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADanielle%2BE%2BAnderson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jin Cui </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>MOE Key Laboratory of Bioorganic Phosphorus Chemistry &#38; Chemical Biology, Beijing Advanced Innovation Center for Structural Biology, School of Pharmaceutical Sciences, Tsinghua-Peking Center for Life Sciences, Center for Infectious Disease Research, School of Medicine, Tsinghua University</span>, Beijing, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jin%2BCui%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cui%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJin%2BCui%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Qian Ye </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>MOE Key Laboratory of Bioorganic Phosphorus Chemistry &#38; Chemical Biology, Beijing Advanced Innovation Center for Structural Biology, School of Pharmaceutical Sciences, Tsinghua-Peking Center for Life Sciences, Center for Infectious Disease Research, School of Medicine, Tsinghua University</span>, Beijing, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qian%2BYe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ye%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQian%2BYe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Baoying Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>NHC Key Laboratory of Biosafety, Ministry of Health, National Institute for Viral Disease Control and Prevention</span>, China CDC, Beijing, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Baoying%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABaoying%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Wenhong Zu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>MOE Key Laboratory of Bioorganic Phosphorus Chemistry &#38; Chemical Biology, Beijing Advanced Innovation Center for Structural Biology, School of Pharmaceutical Sciences, Tsinghua-Peking Center for Life Sciences, Center for Infectious Disease Research, School of Medicine, Tsinghua University</span>, Beijing, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wenhong%2BZu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zu%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWenhong%2BZu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jing Gong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>MOE Key Laboratory of Bioorganic Phosphorus Chemistry &#38; Chemical Biology, Beijing Advanced Innovation Center for Structural Biology, School of Pharmaceutical Sciences, Tsinghua-Peking Center for Life Sciences, Center for Infectious Disease Research, School of Medicine, Tsinghua University</span>, Beijing, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jing%2BGong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gong%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJing%2BGong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Weiqiang Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Institute of Zoology, Chinese Academy of Science</span>, Beijing, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Weiqiang%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWeiqiang%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">So Young Kim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Department of Molecular Genetics and Microbiology, Duke University</span>, Durham, North Carolina, 27708, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=So%2BYoung%2BKim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kim%20SY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASo%2BYoung%2BKim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Biao Guo Yan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Programme in Emerging Infectious Diseases, Duke–NUS Medical School</span>, Singapore, <span class='nlm-country'>Singapore</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Biao%2BGuo%2BYan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yan%20BG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABiao%2BGuo%2BYan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Kristmundur Sigmundsson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Programme in Emerging Infectious Diseases, Duke–NUS Medical School</span>, Singapore, <span class='nlm-country'>Singapore</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kristmundur%2BSigmundsson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sigmundsson%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKristmundur%2BSigmundsson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Xiao Fang Lim </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Programme in Emerging Infectious Diseases, Duke–NUS Medical School</span>, Singapore, <span class='nlm-country'>Singapore</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiao%2BFang%2BLim%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lim%20XF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiao%2BFang%2BLim%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Fei Ye </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>NHC Key Laboratory of Biosafety, Ministry of Health, National Institute for Viral Disease Control and Prevention</span>, China CDC, Beijing, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fei%2BYe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ye%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFei%2BYe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Peihua Niu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>NHC Key Laboratory of Biosafety, Ministry of Health, National Institute for Viral Disease Control and Prevention</span>, China CDC, Beijing, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peihua%2BNiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Niu%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeihua%2BNiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Xuming Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Institute of Zoology, Chinese Academy of Science</span>, Beijing, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xuming%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXuming%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Wenjie Tan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>NHC Key Laboratory of Biosafety, Ministry of Health, National Institute for Viral Disease Control and Prevention</span>, China CDC, Beijing, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wenjie%2BTan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tan%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWenjie%2BTan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Lin-Fa Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Programme in Emerging Infectious Diseases, Duke–NUS Medical School</span>, Singapore, <span class='nlm-country'>Singapore</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lin-Fa%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALin-Fa%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:xutan@tsinghua.edu.cn" class="" data-icon-position="" data-hide-link-title="0">xutan@tsinghua.edu.cn</a>
<a href="mailto:linfa.wang@duke-nus.edu.sg" class="" data-icon-position="" data-hide-link-title="0">linfa.wang@duke-nus.edu.sg</a></span></li></ul></div><div class="author-tooltip-16"><div class="author-tooltip-name">Xu Tan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>MOE Key Laboratory of Bioorganic Phosphorus Chemistry &#38; Chemical Biology, Beijing Advanced Innovation Center for Structural Biology, School of Pharmaceutical Sciences, Tsinghua-Peking Center for Life Sciences, Center for Infectious Disease Research, School of Medicine, Tsinghua University</span>, Beijing, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xu%2BTan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tan%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXu%2BTan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:xutan@tsinghua.edu.cn" class="" data-icon-position="" data-hide-link-title="0">xutan@tsinghua.edu.cn</a>
<a href="mailto:linfa.wang@duke-nus.edu.sg" class="" data-icon-position="" data-hide-link-title="0">linfa.wang@duke-nus.edu.sg</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.29.014209v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1217863" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1217863/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.29.014209v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1217863" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1217863/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.29.014209v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1217863" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1217863/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.29.014209v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1217863" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1217863/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.29.014209v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1217863" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1217863/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">SUMMARY</h2><p id="p-3">Bats are responsible for the zoonotic transmission of several major viral diseases including the 2003 SARS outbreak and the ongoing COVID-19 pandemic. While bat genomic sequencing studies have revealed characteristic adaptations of the innate immune system, functional genomic studies are urgently needed to provide a foundation for the molecular dissection of the tolerance of viral infections in bats. Here we report the establishment and screening of genome-wide RNAi library and CRISPR library for the model megabat, <em>Pteropus Alecto</em>. We used the complementary RNAi and CRISPR libraries to interrogate <em>Pteropus Alecto</em> cells for infection with two different viruses, mumps virus and Influenza A virus, respectively. Screening results converged on the endocytosis pathway and the protein secretory pathway as required for both viral infections. Additionally, we revealed a general dependence of the C-1-tetrahydrofolate synthase gene, MTHFD1, for viral replication in bat cells as well as in human cells. MTHFD1 inhibitor carolacton potently blocked replication of several RNA viruses including SARS-CoV-2. Our studies provide a resource for systematic inquiry into the genetic underpinnings of bat biology and a potential target for developing broad spectrum antiviral therapy.</p></div><p id="p-4">As the only mammal capable of sustained flying, bats are intriguing animals with unique behavioral and physiological characteristics. Bats are notorious for transmission of deadly viruses, causing the epidemics of the Ebola<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a></sup>, SARS<sup><a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a></sup>, Henipavirus<sup><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a></sup>, MERS<sup><a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a></sup> and COVID-19<sup><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>,<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a></sup>. Bats are widely considered the largest reservoir of numerous virus species but are resilient to the diseases associated with many of these viruses<sup><a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>,<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a></sup>. How bats tolerate the infections with these viruses is of intense interest due to the scientific and public health relevance. Genomic sequencing studies have revealed several genetic adaptions underlying this tolerance of viral infections. For example, the DNA damage checkpoint pathway and the NF-κB pathway are known to be positively selected in the bat genomes<sup><a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a></sup>. In addition, in the type I interferon gene family, the MHC class I genes and natural killer cell receptor family have been reported to be highly expanded in the bat genomes<sup><a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a></sup>. These studies provided evidence to support the hypothesis that bats have evolved a highly specific genetic configuration, especially in the innate immunity pathways. A major caveat is the lack of systematic examination of the functions of these genes due to shortage of functional genomic tools.</p><p id="p-5">CRISPR and RNAi technologies have enabled genome-wide screening of gene functions for a variety of cell types, yielding an unprecedented wealth of information revealing the complexity of biological processes<sup><a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>,<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a></sup>. This is especially true in the field of virus-host interactions, where a plethora of host factors have been identified for a variety of viruses<sup><a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>–<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a></sup>. Both technologies have advantages and disadvantages and provide complementary approaches to achieve comprehensive genetic perturbations<sup><a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>–<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a></sup>. CRISPR permits complete knockout of a gene, thus affords generation of truly null genotype. A caveat is that the lethality associated with knockout of essential genes prevents functional parsing of these genes in different biological processes. In addition, CRISPR screening is usually used a pool-based format that requires the generation of stable cell lines through long term culturing. Therefore, CRISPR is suited for studying phenotypes that require a longer time to manifest. Based on previous studies, CRISPR screens demonstrated excellent specificity but limited sensitivity<sup><a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></sup>. In contrast, siRNA screening relies on temporary knockdown of genes in an individual well format, a methodology suited for examining short-term phenotypic changes. Due to the transient nature of the assay, siRNA screening can tolerate growth defects due to gene knockdown and can elucidate hypomorphic phenotypes induced by siRNAs with varying degrees of knockdown. In general, RNAi screens allow comprehensive identification of genes in a biological process at the cost of more false positives due to pervasive off-target effects<sup><a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a></sup>. To fully realize the power of CRISPR and RNAi screening as complementary strategies to interrogate gene function in bats, here we developed a pool-based CRISPR library and a plate-based siRNA library for the whole genome of the model megabat <em>Pteropus Alecto.</em> We screened two viruses, influenza A virus and mumps virus, using the CRISPR and RNAi libraries, respectively, in the hope of uncovering common host factors for different viruses. From these two screens, common pathways such as the endocytosis pathway and protein secretory pathway have been identified, demonstrating conserved pathways between bat and other mammals in hosting viral infections. We extensively characterized a common hit from the two screens, MTHFD1, and identified it as a critical factor for viral replication in both bats and humans. We further demonstrate that MTHFD1 is a potential target for developing broad-spectrum antiviral drugs. MTHFD1 inhibitor carolacton demonstrated broad antiviral activities against Zika virus, mumps virus, and importantly, SARS-CoV-2. In summary, we developed two orthogonal libraries for genome-wide loss-of-function screening in bats and demonstrated their utility in discovery of important host factors for viral infections.</p><div class="section" id="sec-1"><h2 class="">Design and validation of genome-wide CRISPR of <em>P. Alecto</em></h2><p id="p-6">We designed and synthesized the CRISPR and RNAi libraries based on the genome of <em>P. alecto.</em> For the CRISPR library, we adopted the algorithm developed for the construction of the popular GeCKO library to select single guide RNAs (sgRNAs)<sup><a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a></sup>. We designed 85,275 sgRNAs targeting 21,334 genes annotated in the <em>P. alecto</em> genome with 21,330 genes with 4 sgRNAs for each gene. The other 34 genes have 1-3 sgRNAs due to the constraints of the gene sequences for designing sgRNAs. Selection of sgRNAs was based on the calculations of off-target scores to lower potential off-target effects of the sgRNAs. The sgRNAs designs were validated for 21 sgRNAs targeting 11 genes using T7E1 endonuclease cleavage assay or Western blotting. We observed 90.5% of effectiveness in gene editing or gene knockdown, supporting the quality of the designed library (<strong><a id="xref-fig-5-1" class="xref-fig" href="#F5">Supplementary Figure 1</a></strong>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Validation of sgRNA design of ten genes in PaKi cell line using CRISPR-Cas9. a. T7E1 assay to confirm indels in the target genes. NT: nontargeting control, g1,g2: designed guide RNA targeting each gene. b. Western blot to confirm the decrease of protein level of HSP70 by two guide RNAs g1 and g2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1758073080" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Validation of sgRNA design of ten genes in PaKi cell line using CRISPR-Cas9.&lt;/span&gt; &lt;strong&gt;a.&lt;/strong&gt; T7E1 assay to confirm indels in the target genes. NT: nontargeting control, g1,g2: designed guide RNA targeting each gene. &lt;strong&gt;b.&lt;/strong&gt; Western blot to confirm the decrease of protein level of HSP70 by two guide RNAs g1 and g2.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F5.medium.gif" width="440" height="397"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F5.medium.gif" width="440" height="397"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Supplementary Figure 1.</span> <span class="caption-title">Validation of sgRNA design of ten genes in PaKi cell line using CRISPR-Cas9.</span><p id="p-37" class="first-child"><strong>a.</strong> T7E1 assay to confirm indels in the target genes. NT: nontargeting control, g1,g2: designed guide RNA targeting each gene. <strong>b.</strong> Western blot to confirm the decrease of protein level of HSP70 by two guide RNAs g1 and g2.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-2"><h2 class="">CRISPR screening identified influenza A virus host factors</h2><p id="p-7">We performed a CRISPR screen with influenza A virus in PaKi cells to identify viral host factors (<strong><a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1a</a></strong>). PaKi cells stably expressing SgCas9 and sgRNA library were infected with the H1N1 PR8 strain of influenza A virus, in duplicate. We used a high multiplicity-of-infection (MOI) so that naïve PaKi cells all died before the 9th day post infection. We then collected the surviving cells and compared sgRNA abundance with mock-infected cells. The two replicates demonstrated a high correlation, indicating high reproducibility (<strong><a id="xref-fig-6-1" class="xref-fig" href="#F6">Supplementary Fig. 2</a></strong>). We calculated the results using the RIGER algorithm and identified 21 host factors required for the viral infection or pathogenesis<sup><a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a></sup> (<strong><a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1b</a></strong>). Three of the top four hits belong to the V-type proton ATPase subunits (ATP6V1B2, ATP6V1F and ATP6V0D1), which are involved in the acidification of endosome and have previously been previous identified as key host factors for influenza virus entry in several screens performed in human cells (<strong><a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1b</a></strong>)<sup><a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a></sup>. Among the other hits were two genes, COPZ1 and SEC23B, involved in the protein secretory pathway, which are also known to be required for the replication of many viruses including influenza virus and flavivirus (<strong><a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1b</a></strong>)<sup><a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>–<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></sup>. The enrichment of previously known host factors in the identified hits in the CRISPR screen validated our library and the screening methodology.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correlation between the read counts of each sgRNA in two biological replicates of the CRISPR screen." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1758073080" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Correlation between the read counts of each sgRNA in two biological replicates of the CRISPR screen.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F6.medium.gif" width="440" height="424"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F6.medium.gif" width="440" height="424"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 2.</span> <p id="p-38" class="first-child">Correlation between the read counts of each sgRNA in two biological replicates of the CRISPR screen.</p><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Work flows and results of CRISPR and RNAi screens in PaKi cells. a. For CRISPR Cas9 knock out screen, PaKi cells stably expressing Cas9 were transduced with lentiviral sgRNA library targeting 21300 genes of P. alecto. The transduced cells were selected with puromycin to generate the sgRNA cell library. Then the library cells were infected with influenza A virus (IAV) PR8 strain (A/PuertoRico/8/1934) at lethal dose or mock infected. The surviving cells were subjected to two more rounds of infection. The surviving cells at day 9 post-infection were harvested and the genomic DNA were prepared and amplified for the sgRNA region for deep sequencing. b. The sgRNA abundance from sequencing was analyzed by RIGER algorithm, the top hits representing host factors required for cell deaths from influenza infection were labeled. c. For siRNA screen, siRNAs targeting 18328 genes of P. alecto were arrayed in the 96-well plates. The siRNAs were reverse transfected into PaKi cells and 48 h after transfection, the cells were infected with mumps virus. The percentage of infected cells was calculated at 48 h post infection using high content imaging. d. The siRNA targeted genes were ranked by robust Z score, and the potential host factors or restriction factors were shown in the bottom or the top, respectively. e. PaKi cells expressing an shRNA targeting ATP6V0D1 or a non-targeting control shRNA were infected with IAV and the virus infection rate was measured by immunofluorescence using the hemagglutinin antibody at 12h post infection (left). The shRNA knockdown efficiency was detected by real-time PCR (right). For immunofluorescence experiments, n=3, and for real-time PCR data, n=2. Data shown as mean &#xB1; standard deviation. f. PaKi cells expressing an shRNA targeting SEC23B or a non-targeting control shRNA were infected with IAV PR8 and the virus infection rate was measured by immunofluorescence using the hemagglutinin antibody at 12h post infection (left). The shRNA knockdown efficiency was detected by real-time PCR (right). For immunofluorescence experiments, n=3, and for real-time PCR data, n=2. Data shown as mean &#xB1; standard deviation. g. PaKi cells were treated with indicated doses of V-ATPase inhibitor, bafilomycin A1, and infected with IAV. The infection rate was measured by immunofluorescence using the hemagglutinin antibody at 12h post infection. Data shown as the mean &#xB1; standard deviation (n=3). *: P" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1758073080" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Work flows and results of CRISPR and RNAi screens in PaKi cells.&lt;/span&gt; &lt;strong&gt;a&lt;/strong&gt;. For CRISPR Cas9 knock out screen, PaKi cells stably expressing Cas9 were transduced with lentiviral sgRNA library targeting 21300 genes of &lt;em&gt;P. alecto.&lt;/em&gt; The transduced cells were selected with puromycin to generate the sgRNA cell library. Then the library cells were infected with influenza A virus (IAV) PR8 strain (A/PuertoRico/8/1934) at lethal dose or mock infected. The surviving cells were subjected to two more rounds of infection. The surviving cells at day 9 post-infection were harvested and the genomic DNA were prepared and amplified for the sgRNA region for deep sequencing. &lt;strong&gt;b&lt;/strong&gt;. The sgRNA abundance from sequencing was analyzed by RIGER algorithm, the top hits representing host factors required for cell deaths from influenza infection were labeled. &lt;strong&gt;c&lt;/strong&gt;. For siRNA screen, siRNAs targeting 18328 genes of &lt;em&gt;P. alecto&lt;/em&gt; were arrayed in the 96-well plates. The siRNAs were reverse transfected into PaKi cells and 48 h after transfection, the cells were infected with mumps virus. The percentage of infected cells was calculated at 48 h post infection using high content imaging. &lt;strong&gt;d.&lt;/strong&gt; The siRNA targeted genes were ranked by robust Z score, and the potential host factors or restriction factors were shown in the bottom or the top, respectively. &lt;strong&gt;e&lt;/strong&gt;. PaKi cells expressing an shRNA targeting ATP6V0D1 or a non-targeting control shRNA were infected with IAV and the virus infection rate was measured by immunofluorescence using the hemagglutinin antibody at 12h post infection (left). The shRNA knockdown efficiency was detected by real-time PCR (right). For immunofluorescence experiments, n=3, and for real-time PCR data, n=2. Data shown as mean &#xB1; standard deviation. f. PaKi cells expressing an shRNA targeting SEC23B or a non-targeting control shRNA were infected with IAV PR8 and the virus infection rate was measured by immunofluorescence using the hemagglutinin antibody at 12h post infection (left). The shRNA knockdown efficiency was detected by real-time PCR (right). For immunofluorescence experiments, n=3, and for real-time PCR data, n=2. Data shown as mean &#xB1; standard deviation. &lt;strong&gt;g.&lt;/strong&gt; PaKi cells were treated with indicated doses of V-ATPase inhibitor, bafilomycin A1, and infected with IAV. The infection rate was measured by immunofluorescence using the hemagglutinin antibody at 12h post infection. Data shown as the mean &#xB1; standard deviation (n=3). *: P&lt;0.05, **: P&lt;0.01, ***: P&lt;0.001.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F1.medium.gif" width="323" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F1.medium.gif" width="323" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 1.</span> <span class="caption-title">Work flows and results of CRISPR and RNAi screens in PaKi cells.</span><p id="p-8" class="first-child"><strong>a</strong>. For CRISPR Cas9 knock out screen, PaKi cells stably expressing Cas9 were transduced with lentiviral sgRNA library targeting 21300 genes of <em>P. alecto.</em> The transduced cells were selected with puromycin to generate the sgRNA cell library. Then the library cells were infected with influenza A virus (IAV) PR8 strain (A/PuertoRico/8/1934) at lethal dose or mock infected. The surviving cells were subjected to two more rounds of infection. The surviving cells at day 9 post-infection were harvested and the genomic DNA were prepared and amplified for the sgRNA region for deep sequencing. <strong>b</strong>. The sgRNA abundance from sequencing was analyzed by RIGER algorithm, the top hits representing host factors required for cell deaths from influenza infection were labeled. <strong>c</strong>. For siRNA screen, siRNAs targeting 18328 genes of <em>P. alecto</em> were arrayed in the 96-well plates. The siRNAs were reverse transfected into PaKi cells and 48 h after transfection, the cells were infected with mumps virus. The percentage of infected cells was calculated at 48 h post infection using high content imaging. <strong>d.</strong> The siRNA targeted genes were ranked by robust Z score, and the potential host factors or restriction factors were shown in the bottom or the top, respectively. <strong>e</strong>. PaKi cells expressing an shRNA targeting ATP6V0D1 or a non-targeting control shRNA were infected with IAV and the virus infection rate was measured by immunofluorescence using the hemagglutinin antibody at 12h post infection (left). The shRNA knockdown efficiency was detected by real-time PCR (right). For immunofluorescence experiments, n=3, and for real-time PCR data, n=2. Data shown as mean ± standard deviation. f. PaKi cells expressing an shRNA targeting SEC23B or a non-targeting control shRNA were infected with IAV PR8 and the virus infection rate was measured by immunofluorescence using the hemagglutinin antibody at 12h post infection (left). The shRNA knockdown efficiency was detected by real-time PCR (right). For immunofluorescence experiments, n=3, and for real-time PCR data, n=2. Data shown as mean ± standard deviation. <strong>g.</strong> PaKi cells were treated with indicated doses of V-ATPase inhibitor, bafilomycin A1, and infected with IAV. The infection rate was measured by immunofluorescence using the hemagglutinin antibody at 12h post infection. Data shown as the mean ± standard deviation (n=3). *: P&lt;0.05, **: P&lt;0.01, ***: P&lt;0.001.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-3"><h2 class="">RNAi screen results converge with CRISPR screening on key pathways</h2><p id="p-9">We synthesized the sgRNA library in an array-based oligonucleotide library and cloned them into the lentiGuide-Puro lentivirus expression vector<sup><a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a></sup>. A custom bat siRNA library was designed to target 18,328 genes using 40,016 siRNA duplexes. We performed RNAi screening in PaKi cells for identification of host factors for the infection of mumps virus, a paramyxovirus (<strong><a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1c</a></strong>). The screen was performed in 384-well format with a mumps virus strain expressing EGFP from an additional open reading frame<sup><a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a></sup>. PaKi cells were infected 48 hours after transfection of the siRNA library. High-content imaging analysis was utilized to quantitate the infection rate as well as the total cell number each well, indicating cytotoxicity of the siRNA. Excluding siRNAs with significant cytotoxicity, we selected 45 host dependency factors whose knockdown resulted in significantly reduced viral infection and 45 host restriction factors whose knockdown promoted viral infection based on the Z score of infection rate (<strong><a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig. 1d</a></strong>). We selected a number of genes for individual validation with more replicates and obtained a high validation rate (11/12 of host dependency factors and 7/12 of the host restriction factors validated with at least one siRNA) (<strong><a id="xref-fig-7-1" class="xref-fig" href="#F7">Supplementary Fig. 3</a></strong>), supporting the reproducibility of our screening methodology.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Validation of siRNA screen hits. PaKi cells were transfected with the siRNAs followed by mumps-GFP virus infection. The virus infection levels were measured by high content imaging." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1758073080" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Validation of siRNA screen hits.&lt;/span&gt; PaKi cells were transfected with the siRNAs followed by mumps-GFP virus infection. The virus infection levels were measured by high content imaging.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F7.medium.gif" width="386" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F7.medium.gif" width="386" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 3.</span> <span class="caption-title">Validation of siRNA screen hits.</span><p id="p-39" class="first-child">PaKi cells were transfected with the siRNAs followed by mumps-GFP virus infection. The virus infection levels were measured by high content imaging.</p><div class="sb-div caption-clear"></div></div></div><p id="p-10">Although there is only one gene overlap between the host factors identified in both CRISPR and RNAi screens, we observed that the endocytosis pathway and protein secretory pathway were enriched in both screens, albeit represented by different genes (<strong><a id="xref-fig-1-7" class="xref-fig" href="#F1">Fig. 1b</a> and <a id="xref-fig-1-8" class="xref-fig" href="#F1">1d</a></strong>). For the endocytosis pathway, two genes encoding components of clathrin complex, CLTA and CLTC, were among the top three hits (<strong><a id="xref-fig-1-9" class="xref-fig" href="#F1">Fig. 1d</a></strong>). Four genes involved in the protein secretory, namely ARF1, ARF4, COPG1 and SEC61B, were identified (<strong><a id="xref-fig-1-10" class="xref-fig" href="#F1">Fig. 1d</a></strong>). This is consistent with the idea that both influenza A virus and mumps require the two pathways for viral replication. In addition, protein translation initiation, which was previously reported to be required for influenza A infection<sup><a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">21</a></sup>, was also represented in the top hits by three genes: EIF3B, EIF3G and EIF4A3. Conversely, RNAi screening revealed genes that restrict mumps infection in PaKi cells. Among the top hits were several known antiviral genes, TRIM39, TRIM43<sup><a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a></sup> and NAMPT<sup><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a></sup>. Overall, we identified many candidate genes that might play a role in viral infection in bats.</p><p id="p-11">The two pathways identified by both CRISPR screen and siRNA screen, endocytosis and protein secretion, have previously been implicated in human host response to a number of viral<sup><a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>,<a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">21</a>,<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a></sup> pathogens To validate the role of these pathways in the bat response to viral infection, we established PaKi stable cell lines expressing shRNAs targeting several hits from the two screens. Stable knockdown of a V-type proton ATPase subunit and SEC23B led to a significant decrease of influenza A virus infection in PaKi cells (<strong><a id="xref-fig-1-11" class="xref-fig" href="#F1">Fig. 1e</a> and <a id="xref-fig-1-12" class="xref-fig" href="#F1">1f</a></strong>). In addition, the small molecule inhibitor of V-type proton ATPase, bafilomycin A (<strong><a id="xref-fig-1-13" class="xref-fig" href="#F1">Fig. 1g</a></strong>), potently blocked influenza infection. Taken together, these results demonstrate that bat and human cells share key genes and pathways in hosting viral infection.</p></div><div class="section" id="sec-4"><h2 class="">MTHFD1 is a key host factor for a broad spectrum of RNA viruses</h2><p id="p-12">The gene identified as a host factor from both the CRISPR and RNAi screen was MTHFD1 (<strong><a id="xref-fig-1-14" class="xref-fig" href="#F1">Fig. 1c</a> and <a id="xref-fig-1-15" class="xref-fig" href="#F1">1d</a></strong>), which we validated by siRNA knockdown in the context of mumps infection (<strong><a id="xref-fig-7-2" class="xref-fig" href="#F7">Supplemental Fig. 3</a></strong>). We validated its function in influenza A virus replication in PaKi cells with two different sgRNAs targeting MTHFD1 (<strong><a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2a</a>–<a id="xref-fig-2-2" class="xref-fig" href="#F2">2b</a></strong>). Similarly, in HEK293T cells with MTHFD1 knocked out by two sgRNAs, influenza replication is also inhibited, suggesting a conserved requirement for this gene by influenza virus in human cells (<strong><a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2c</a></strong>). Further virus titration assay demonstrated that MTHFD1 knockdown significantly inhibits viral replication for mumps virus in PaKi cells (<strong><a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2d</a> and <a id="xref-fig-2-5" class="xref-fig" href="#F2">2e</a></strong>). The effect is similar to the knockdown of SEC61B, a well-studied host protein required for viral replication and a top hit in our RNAi screen (<strong><a id="xref-fig-2-6" class="xref-fig" href="#F2">Fig. 2d</a> and <a id="xref-fig-2-7" class="xref-fig" href="#F2">2e</a></strong>)<sup><a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">23</a></sup>. We additionally tested Melaka virus (PRV3M), a pteropine orthoreovirus that causes acute respiratory disease when transmitted by bats to humans<sup><a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a></sup>. PRV3M titer was also significantly reduced by MTHFD1 knockdown (<strong><a id="xref-fig-2-8" class="xref-fig" href="#F2">Fig. 2e</a> and <a id="xref-fig-2-9" class="xref-fig" href="#F2">2f</a></strong>). Infection of Zika virus, a member of the flavivirus family, was also inhibited by MTHFD1 knockdown using siRNA (<strong><a id="xref-fig-2-10" class="xref-fig" href="#F2">Fig. 2g</a></strong>). Alternative knockdown with stable expression of two shRNAs validated the requirement of MTHFD1 for the replication of mumps virus and Zika virus (<strong><a id="xref-fig-2-11" class="xref-fig" href="#F2">Fig. 2h</a>, <a id="xref-fig-2-12" class="xref-fig" href="#F2">2i</a> and <a id="xref-fig-2-13" class="xref-fig" href="#F2">2j</a></strong>). Importantly, we also show that this inhibition is completely rescued by overexpression of MTHFD1 from a construct resistant to the shRNA, ruling out potential off-target effects (<strong><a id="xref-fig-2-14" class="xref-fig" href="#F2">Fig. 2k</a></strong>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="MTHFD1 is host factor for replication of multiple RNA viruses. a. MTHFD1 knockout clonal PaKi cells were generated from two independent sgRNAs and infected with Influenza A virus (IAV) for 12h. The infection rates were quantified with immunofluorescence staining of HA. Data shown as the mean &#xB1; standard deviation (n=3). b. Representative images from a. (scale bar=100 &#x3BC;m). c. MTHFD1 knock out 293T cells were generated from two independent sgRNAs and infected with IAV for another 12h. Then the cells were collected and analyzed by Western blotting. Data shown are representative of three independent experiments. d-f. PaKi cells were transfected with non-target control siRNA (siNT) or siRNAs targeting MTHFD1 and SEC61B followed by mumps virus(d) and Melaka virus (f) infection. The virus in the supernatant were tittered by limiting dilution. The knockdown efficiency was determined by realtime PCR and shown in (e). For d and e n=3, and for f, n=4. Data shown as the mean &#xB1; standard deviation. g. PaKi cells were transfected with siNT or siRNA targeting MTHFD1, 48 h later the cells were infected with Zika virus. The Zika virus RNA level was measured by real-time PCR. For g, n=4. Data shown as the mean &#xB1; standard deviation. h-j. PaKi shRNA control cells (shNT) and shMTHFD1 cells were infected with mumps virus (h) or Zika virus (i), the virus infection level was measured by immunofluorescence. For h, n=2 and for i, n=3. Data shown as the mean &#xB1; standard deviation. The shRNA knockdown efficiency was tested by Western blotting as showing in (j). Data shown are representative of three independent experiments. k. Luciferase or shRNA resistant MTHFD1 construct were stably expressed in shMTHFD1-2 PaKi cells, then the cells were infected with Zika virus. The rescue level of Zika virus was analyzed by Western blotting. Data shown are representative of three independent experiments. For all bar graphs, the mean &#xB1; standard deviation values are shown. *: P" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1758073080" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;MTHFD1 is host factor for replication of multiple RNA viruses.&lt;/span&gt; &lt;strong&gt;a.&lt;/strong&gt; MTHFD1 knockout clonal PaKi cells were generated from two independent sgRNAs and infected with Influenza A virus (IAV) for 12h. The infection rates were quantified with immunofluorescence staining of HA. Data shown as the mean &#xB1; standard deviation (n=3). &lt;strong&gt;b.&lt;/strong&gt; Representative images from &lt;strong&gt;a.&lt;/strong&gt; (scale bar=100 &#x3BC;m). &lt;strong&gt;c&lt;/strong&gt;. MTHFD1 knock out 293T cells were generated from two independent sgRNAs and infected with IAV for another 12h. Then the cells were collected and analyzed by Western blotting. Data shown are representative of three independent experiments. &lt;strong&gt;d-f.&lt;/strong&gt; PaKi cells were transfected with non-target control siRNA (siNT) or siRNAs targeting MTHFD1 and SEC61B followed by mumps virus(&lt;strong&gt;d&lt;/strong&gt;) and Melaka virus &lt;strong&gt;(f)&lt;/strong&gt; infection. The virus in the supernatant were tittered by limiting dilution. The knockdown efficiency was determined by realtime PCR and shown in &lt;strong&gt;(e).&lt;/strong&gt; For &lt;strong&gt;d&lt;/strong&gt; and &lt;strong&gt;e&lt;/strong&gt; n=3, and for &lt;strong&gt;f&lt;/strong&gt;, n=4. Data shown as the mean &#xB1; standard deviation. &lt;strong&gt;g&lt;/strong&gt;. PaKi cells were transfected with siNT or siRNA targeting MTHFD1, 48 h later the cells were infected with Zika virus. The Zika virus RNA level was measured by real-time PCR. For &lt;strong&gt;g,&lt;/strong&gt; n=4. Data shown as the mean &#xB1; standard deviation. &lt;strong&gt;h-j&lt;/strong&gt;. PaKi shRNA control cells (shNT) and shMTHFD1 cells were infected with mumps virus &lt;strong&gt;(h)&lt;/strong&gt; or Zika virus &lt;strong&gt;(i)&lt;/strong&gt;, the virus infection level was measured by immunofluorescence. For &lt;strong&gt;h&lt;/strong&gt;, n=2 and for &lt;strong&gt;i&lt;/strong&gt;, n=3. Data shown as the mean &#xB1; standard deviation. The shRNA knockdown efficiency was tested by Western blotting as showing in (&lt;strong&gt;j&lt;/strong&gt;). Data shown are representative of three independent experiments. &lt;strong&gt;k&lt;/strong&gt;. Luciferase or shRNA resistant MTHFD1 construct were stably expressed in shMTHFD1-2 PaKi cells, then the cells were infected with Zika virus. The rescue level of Zika virus was analyzed by Western blotting. Data shown are representative of three independent experiments. For all bar graphs, the mean &#xB1; standard deviation values are shown. *: P&lt;0.05, **: P&lt;0.01, ***: P&lt;0.001.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F2.medium.gif" width="422" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F2.medium.gif" width="422" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">MTHFD1 is host factor for replication of multiple RNA viruses.</span><p id="p-13" class="first-child"><strong>a.</strong> MTHFD1 knockout clonal PaKi cells were generated from two independent sgRNAs and infected with Influenza A virus (IAV) for 12h. The infection rates were quantified with immunofluorescence staining of HA. Data shown as the mean ± standard deviation (n=3). <strong>b.</strong> Representative images from <strong>a.</strong> (scale bar=100 μm). <strong>c</strong>. MTHFD1 knock out 293T cells were generated from two independent sgRNAs and infected with IAV for another 12h. Then the cells were collected and analyzed by Western blotting. Data shown are representative of three independent experiments. <strong>d-f.</strong> PaKi cells were transfected with non-target control siRNA (siNT) or siRNAs targeting MTHFD1 and SEC61B followed by mumps virus(<strong>d</strong>) and Melaka virus <strong>(f)</strong> infection. The virus in the supernatant were tittered by limiting dilution. The knockdown efficiency was determined by realtime PCR and shown in <strong>(e).</strong> For <strong>d</strong> and <strong>e</strong> n=3, and for <strong>f</strong>, n=4. Data shown as the mean ± standard deviation. <strong>g</strong>. PaKi cells were transfected with siNT or siRNA targeting MTHFD1, 48 h later the cells were infected with Zika virus. The Zika virus RNA level was measured by real-time PCR. For <strong>g,</strong> n=4. Data shown as the mean ± standard deviation. <strong>h-j</strong>. PaKi shRNA control cells (shNT) and shMTHFD1 cells were infected with mumps virus <strong>(h)</strong> or Zika virus <strong>(i)</strong>, the virus infection level was measured by immunofluorescence. For <strong>h</strong>, n=2 and for <strong>i</strong>, n=3. Data shown as the mean ± standard deviation. The shRNA knockdown efficiency was tested by Western blotting as showing in (<strong>j</strong>). Data shown are representative of three independent experiments. <strong>k</strong>. Luciferase or shRNA resistant MTHFD1 construct were stably expressed in shMTHFD1-2 PaKi cells, then the cells were infected with Zika virus. The rescue level of Zika virus was analyzed by Western blotting. Data shown are representative of three independent experiments. For all bar graphs, the mean ± standard deviation values are shown. *: P&lt;0.05, **: P&lt;0.01, ***: P&lt;0.001.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-5"><h2 class="">MTHFD1’s formyl tetrahydrofolate synthetase activity is essential for RNA virus replication</h2><p id="p-14">MTHFD1 is a trifunctional enzyme involved in the one carbon (C1) metabolism pathway, which is responsible for cellular production of purine, dTMP and methyl groups<sup><a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a></sup>. MTHFD1 has three enzymatic functions including dehydrogenase and cyclohydrolase activities encoded in the N-terminal domain and formyl tetrahydrofolate (Formyl THF) synthetase activity encoded in the C-terminal domain (<strong><a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3a</a> and <a id="xref-fig-3-2" class="xref-fig" href="#F3">3b</a></strong>). We suspected that MTHFD1 knockdown would lead to the deficiency of cellular purine levels thereby reducing RNA replication. Indeed, supplementation of purine analogs, either hypoxanthine or inosine in the media almost completely rescued the replication of influenza A and Zika viruses in MTHFD1-knockout or knockdown PaKi cells as shown by both immunofluorescence staining and virus tittering assay (<strong><a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3c</a>, <a id="xref-fig-3-4" class="xref-fig" href="#F3">3d</a> and <a id="xref-fig-3-5" class="xref-fig" href="#F3">3e</a></strong>). In contrast, supplementation of methyl-group donor or thymidine had little effect on viral replication, evidence that it is the purine synthesis activity of MTHFD1 that is essential for the viral replication (<strong><a id="xref-fig-3-6" class="xref-fig" href="#F3">Fig. 3d</a></strong>). To further pinpoint the enzymatic function of MTHFD1 in viral replication, we performed gene complementation experiments in an shRNA cell line with over 90% knockdown of MTHFD1 (<strong><a id="xref-fig-2-15" class="xref-fig" href="#F2">Fig. 2i</a></strong>). The block of Zika virus replication in this cell line was completely rescued by overexpression of an shRNA-resistant MTHFD1 construct or bat homolog (<strong><a id="xref-fig-3-7" class="xref-fig" href="#F3">Fig. 3f</a> and <a id="xref-fig-3-8" class="xref-fig" href="#F3">3g</a></strong>). Mutations of MTHFD1 that are known to disrupt its dehydrogenase and cyclohydrolase activities (D125A and R173C)<sup><a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>,<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a></sup> did not affect this rescue (<strong><a id="xref-fig-3-9" class="xref-fig" href="#F3">Fig. 3b</a>, <a id="xref-fig-3-10" class="xref-fig" href="#F3">3f</a> and <a id="xref-fig-3-11" class="xref-fig" href="#F3">3g</a></strong>). However, mutations in the Formyl THF synthetase active site (F759A/W788A, or FW/AA)<sup><a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></sup> completely abolished this rescue, further supporting that the purine synthesis activity of MTHFD1 is an essential activity for viral replication (<strong><a id="xref-fig-3-12" class="xref-fig" href="#F3">Fig. 3b</a>, <a id="xref-fig-3-13" class="xref-fig" href="#F3">3f</a> and <a id="xref-fig-3-14" class="xref-fig" href="#F3">3g</a></strong>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="MTHFD1&#x2019;s formyl tetrahydrofolate synthetase activity is essential for RNA virus replication, a. MTHFD1 is a trifunctional enzyme in the one carbon metabolism which is required for purines, dTMP and methionine synthesis. b. Domain structures and key amino acids of MTHFD1 with key residues for the three enzymatic functions indicated. c. PaKi MTHFD1 knock out cells were infected with IAV with or without supplementation of hypoxanthine or thymidine for 12 h and the virus infection level was detected by immunofluorescence. Data shown as the mean &#xB1; standard deviation (n=3). d. PaKi shMTHFD1-2 cells or control cells with a non-targeting shRNA were infected with Zika virus in regular medium or in medium supplemented with inosine, thymidine or 5-CH3THF. Data shown as the mean &#xB1; standard deviation (n=3). e. PaKi shMTHFD1 cells were infected with Zika virus in regular medium or in medium supplemented with inosine. At 48 h post infection, the virus in the supernatant was harvested and the titre was determined by limiting dilution in PaKi cells. Data shown as the mean &#xB1; standard deviation (n=2). f. PaKi shMTHFD1-2 cells or control cells with a non-targeting shRNA were transfected with luciferase (negative control), human wild type MTHFD1, bat wildtype MTHFD1 or bat MTHFD mutants, followed by Zika virus infection. The virus infection rate was measured by immunofluorescence. Data shown as the mean &#xB1; standard deviation (n=3). g. Representative images from (f). Scale bar: 100 &#x3BC;m. h. HEK293T MTHFD1 knock out cells or control HEK293T cells with a non-targeting sgRNA (sgNT) were infected with PR8 with or without the supplementation of inosine, thymidine or 5-CH3THF. At 12 h post infection, the cells were collected and analyzed by Western blotting. Data shown are representative of three independent experiments. i. HEK293T MTHFD1 knockout cells (sgMTHFD1-2) or control cells (sgNT) were infected with IAV. The vRNA level was determined by real-time PCR at the indicated time points. The results shown representative of three independent experiments j. HEK293T MTHFD1 knockout cells (sgMTHFD1-2) or control cells (sgNT) were transfected with plasmids expressing the influenza virus mini-genome including polymerase subunits PB1, PB2 and PA, and nucleoprotein together with pPOLI-HA, which can transcribe the hemagglutinin vRNA of influenza virus with or without inosine supplementation. At 16h, 28h, 40h and 52h post transfection, the cells were collected and analyzed by Western blotting. Data shown are representative of three independent experiments. For all bar graphs, the mean &#xB1; standard deviation values are shown. *P" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1758073080" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;MTHFD1&#x2019;s formyl tetrahydrofolate synthetase activity is essential for RNA virus replication,&lt;/span&gt; &lt;strong&gt;a.&lt;/strong&gt; MTHFD1 is a trifunctional enzyme in the one carbon metabolism which is required for purines, dTMP and methionine synthesis. &lt;strong&gt;b&lt;/strong&gt;. Domain structures and key amino acids of MTHFD1 with key residues for the three enzymatic functions indicated. &lt;strong&gt;c&lt;/strong&gt;. PaKi MTHFD1 knock out cells were infected with IAV with or without supplementation of hypoxanthine or thymidine for 12 h and the virus infection level was detected by immunofluorescence. Data shown as the mean &#xB1; standard deviation (n=3). &lt;strong&gt;d&lt;/strong&gt;. PaKi shMTHFD1-2 cells or control cells with a non-targeting shRNA were infected with Zika virus in regular medium or in medium supplemented with inosine, thymidine or 5-CH&lt;sub&gt;3&lt;/sub&gt;THF. Data shown as the mean &#xB1; standard deviation (n=3). &lt;strong&gt;e&lt;/strong&gt;. PaKi shMTHFD1 cells were infected with Zika virus in regular medium or in medium supplemented with inosine. At 48 h post infection, the virus in the supernatant was harvested and the titre was determined by limiting dilution in PaKi cells. Data shown as the mean &#xB1; standard deviation (n=2). f. PaKi shMTHFD1-2 cells or control cells with a non-targeting shRNA were transfected with luciferase (negative control), human wild type MTHFD1, bat wildtype MTHFD1 or bat MTHFD mutants, followed by Zika virus infection. The virus infection rate was measured by immunofluorescence. Data shown as the mean &#xB1; standard deviation (n=3). &lt;strong&gt;g.&lt;/strong&gt; Representative images from (&lt;strong&gt;f&lt;/strong&gt;). Scale bar: 100 &#x3BC;m. &lt;strong&gt;h.&lt;/strong&gt; HEK293T MTHFD1 knock out cells or control HEK293T cells with a non-targeting sgRNA (sgNT) were infected with PR8 with or without the supplementation of inosine, thymidine or 5-CH3THF. At 12 h post infection, the cells were collected and analyzed by Western blotting. Data shown are representative of three independent experiments. &lt;strong&gt;i&lt;/strong&gt;. HEK293T MTHFD1 knockout cells (sgMTHFD1-2) or control cells (sgNT) were infected with IAV. The vRNA level was determined by real-time PCR at the indicated time points. The results shown representative of three independent experiments &lt;strong&gt;j.&lt;/strong&gt; HEK293T MTHFD1 knockout cells (sgMTHFD1-2) or control cells (sgNT) were transfected with plasmids expressing the influenza virus mini-genome including polymerase subunits PB1, PB2 and PA, and nucleoprotein together with pPOLI-HA, which can transcribe the hemagglutinin vRNA of influenza virus with or without inosine supplementation. At 16h, 28h, 40h and 52h post transfection, the cells were collected and analyzed by Western blotting. Data shown are representative of three independent experiments. For all bar graphs, the mean &#xB1; standard deviation values are shown. *P&lt;0.05.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F3.medium.gif" width="218" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F3.medium.gif" width="218" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">MTHFD1’s formyl tetrahydrofolate synthetase activity is essential for RNA virus replication,</span><p id="p-15" class="first-child"><strong>a.</strong> MTHFD1 is a trifunctional enzyme in the one carbon metabolism which is required for purines, dTMP and methionine synthesis. <strong>b</strong>. Domain structures and key amino acids of MTHFD1 with key residues for the three enzymatic functions indicated. <strong>c</strong>. PaKi MTHFD1 knock out cells were infected with IAV with or without supplementation of hypoxanthine or thymidine for 12 h and the virus infection level was detected by immunofluorescence. Data shown as the mean ± standard deviation (n=3). <strong>d</strong>. PaKi shMTHFD1-2 cells or control cells with a non-targeting shRNA were infected with Zika virus in regular medium or in medium supplemented with inosine, thymidine or 5-CH<sub>3</sub>THF. Data shown as the mean ± standard deviation (n=3). <strong>e</strong>. PaKi shMTHFD1 cells were infected with Zika virus in regular medium or in medium supplemented with inosine. At 48 h post infection, the virus in the supernatant was harvested and the titre was determined by limiting dilution in PaKi cells. Data shown as the mean ± standard deviation (n=2). f. PaKi shMTHFD1-2 cells or control cells with a non-targeting shRNA were transfected with luciferase (negative control), human wild type MTHFD1, bat wildtype MTHFD1 or bat MTHFD mutants, followed by Zika virus infection. The virus infection rate was measured by immunofluorescence. Data shown as the mean ± standard deviation (n=3). <strong>g.</strong> Representative images from (<strong>f</strong>). Scale bar: 100 μm. <strong>h.</strong> HEK293T MTHFD1 knock out cells or control HEK293T cells with a non-targeting sgRNA (sgNT) were infected with PR8 with or without the supplementation of inosine, thymidine or 5-CH3THF. At 12 h post infection, the cells were collected and analyzed by Western blotting. Data shown are representative of three independent experiments. <strong>i</strong>. HEK293T MTHFD1 knockout cells (sgMTHFD1-2) or control cells (sgNT) were infected with IAV. The vRNA level was determined by real-time PCR at the indicated time points. The results shown representative of three independent experiments <strong>j.</strong> HEK293T MTHFD1 knockout cells (sgMTHFD1-2) or control cells (sgNT) were transfected with plasmids expressing the influenza virus mini-genome including polymerase subunits PB1, PB2 and PA, and nucleoprotein together with pPOLI-HA, which can transcribe the hemagglutinin vRNA of influenza virus with or without inosine supplementation. At 16h, 28h, 40h and 52h post transfection, the cells were collected and analyzed by Western blotting. Data shown are representative of three independent experiments. For all bar graphs, the mean ± standard deviation values are shown. *P&lt;0.05.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">We observed a similar dependence on MTHFD1 function for influenza A virus infection in HEK293T cells, a human kidney epithelial cell line (<strong><a id="xref-fig-3-15" class="xref-fig" href="#F3">Fig. 3h</a></strong>). Time course experiments demonstrated a viral replication following MTHFD1 knockdown steadily decreased from four hours post-infection with influenza A virus, consistent with a RNA replication defect (<strong><a id="xref-fig-3-16" class="xref-fig" href="#F3">Fig. 3i</a></strong>). This defect in replication was completely rescued by supplementing purine analog inosine in the media, supporting a conserved function of MTHFD1 in viral infection in bats and humans (<strong><a id="xref-fig-3-17" class="xref-fig" href="#F3">Fig. 3h</a> and <a id="xref-fig-3-18" class="xref-fig" href="#F3">3j</a>, <a id="xref-fig-8-1" class="xref-fig" href="#F8">Supplementary Fig. 4</a></strong>).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Supplementation of inosine rescues IAV replication in MTHFD1 knockout 293T cells. MTHFD1 knockout cells (sgMTHFD1-1) were starved in the medium without HT for 8h and then infected with PR8 with or without inosine supplementation. At 16h, 28h, 40h and 52h post infection, the cells were collected and analyzed by Western blotting." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1758073080" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Supplementation of inosine rescues IAV replication in MTHFD1 knockout 293T cells.&lt;/span&gt; MTHFD1 knockout cells (sgMTHFD1-1) were starved in the medium without HT for 8h and then infected with PR8 with or without inosine supplementation. At 16h, 28h, 40h and 52h post infection, the cells were collected and analyzed by Western blotting.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F8.medium.gif" width="440" height="242"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F8.medium.gif" width="440" height="242"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 4.</span> <span class="caption-title">Supplementation of inosine rescues IAV replication in MTHFD1 knockout 293T cells.</span><p id="p-40" class="first-child">MTHFD1 knockout cells (sgMTHFD1-1) were starved in the medium without HT for 8h and then infected with PR8 with or without inosine supplementation. At 16h, 28h, 40h and 52h post infection, the cells were collected and analyzed by Western blotting.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-6"><h2 class="">MTHFD1 inhibitor carolacton inhibits RNA virus replication</h2><p id="p-17">Carolacton, a natural product derived from bacteria, has recently been identified as a potent inhibitor of MTHFD1<sup><a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a></sup>. We tested the effect of this compound on viral replication. Carolacton inhibited Zika virus and mumps virus replication in PaKi cells in a dose dependent fashion (<strong><a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4a</a>, <a id="xref-fig-4-2" class="xref-fig" href="#F4">4b</a> and <a id="xref-fig-4-3" class="xref-fig" href="#F4">4c</a></strong>). The inhibition was achieved at doses with little cytotoxicity (<strong><a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig. 4a</a></strong>). Mechanistically, this inhibition was completely rescued by supplementation of inosine, providing support that carolacton indeed exerts antiviral activities by inhibiting MTHFD1 (<strong><a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig. 4b</a> and <a id="xref-fig-4-6" class="xref-fig" href="#F4">4c</a></strong>). We next tested the effect of carolacton on the infection of SARS-CoV-2. The viral infection in Vero E6 cells was effectively inhibited at sub-micromolar concentrations by carolacton (IC<sub>50</sub>=0.14 μM) (<strong><a id="xref-fig-4-7" class="xref-fig" href="#F4">Fig. 4d</a></strong>). The cytotoxic effect is very moderate in comparison (<strong><a id="xref-fig-4-8" class="xref-fig" href="#F4">Fig. 4d</a></strong>), supporting a therapeutic window of this compound in potential clinical applications.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="MTHFD1 inhibitor carolacton can inhibit a number of RNA viruses including SARS-CoV-2. a. PaKi cells were treated with different concentrations of carolacton and infected with mumps virus expressing EGFP or Zika virus. Virus infection level was detected by EGFP or immunofluorescence staining of Zika E protein. The cell viability was measured by MTT assay. n=3. Data shown as the mean &#xB1; standard deviation. b. Representative images of immunofluorescence staining of mumps infection from (a). c. PaKi cells were treated with carolacton and infected with Zika virus in the presence of inosine or mock treated. Representative images of immunofluorescence staining of Zika virus E protein were shown. Infection rate shown as the mean &#xB1; standard deviation (n=3). d. PaKi cells were treated with carolacton, and 1h later infected with SARS-CoV-2. At 48 h post infection, the SARS-CoV-2 in the supernatant was measured by real-time PCR. Data shown as the mean &#xB1; standard deviation (n=3)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1758073080" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;MTHFD1 inhibitor carolacton can inhibit a number of RNA viruses including SARS-CoV-2.&lt;/span&gt; &lt;strong&gt;a.&lt;/strong&gt; PaKi cells were treated with different concentrations of carolacton and infected with mumps virus expressing EGFP or Zika virus. Virus infection level was detected by EGFP or immunofluorescence staining of Zika E protein. The cell viability was measured by MTT assay. n=3. Data shown as the mean &#xB1; standard deviation. &lt;strong&gt;b.&lt;/strong&gt; Representative images of immunofluorescence staining of mumps infection from (&lt;strong&gt;a&lt;/strong&gt;). &lt;strong&gt;c.&lt;/strong&gt; PaKi cells were treated with carolacton and infected with Zika virus in the presence of inosine or mock treated. Representative images of immunofluorescence staining of Zika virus E protein were shown. Infection rate shown as the mean &#xB1; standard deviation (n=3). &lt;strong&gt;d.&lt;/strong&gt; PaKi cells were treated with carolacton, and 1h later infected with SARS-CoV-2. At 48 h post infection, the SARS-CoV-2 in the supernatant was measured by real-time PCR. Data shown as the mean &#xB1; standard deviation (n=3).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F4.medium.gif" width="440" height="315"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F4.medium.gif" width="440" height="315"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">MTHFD1 inhibitor carolacton can inhibit a number of RNA viruses including SARS-CoV-2.</span><p id="p-18" class="first-child"><strong>a.</strong> PaKi cells were treated with different concentrations of carolacton and infected with mumps virus expressing EGFP or Zika virus. Virus infection level was detected by EGFP or immunofluorescence staining of Zika E protein. The cell viability was measured by MTT assay. n=3. Data shown as the mean ± standard deviation. <strong>b.</strong> Representative images of immunofluorescence staining of mumps infection from (<strong>a</strong>). <strong>c.</strong> PaKi cells were treated with carolacton and infected with Zika virus in the presence of inosine or mock treated. Representative images of immunofluorescence staining of Zika virus E protein were shown. Infection rate shown as the mean ± standard deviation (n=3). <strong>d.</strong> PaKi cells were treated with carolacton, and 1h later infected with SARS-CoV-2. At 48 h post infection, the SARS-CoV-2 in the supernatant was measured by real-time PCR. Data shown as the mean ± standard deviation (n=3).</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-7"><h2 class="">Discussion</h2><p id="p-19">We developed the first bat genome-wide RNAi and CRISPR libraries and performed two screens with two different RNA viruses. Although the screening methodologies are quite different, we uncovered similar pathways essential for viral replication, namely the endocytosis pathway and protein secretory pathways. The different genes in these two pathways identified from the screens reflect the complementarity of the methodologies, with RNAi screening useful for observing acute effects on viral infection and CRISPR screening for long-term effects on cell survival. Other than technical differences between screening methods, viral kinetics, sensitivity of the viruses to loss-of-function of genes, differences in genome coverage within the two libraries likely contributed to the different hits observed in our studies. Both libraries provide valuable and complementary sources that meet the urgent need to address the biological questions of bats. Similar to human genomic libraries, the utility of these bat libraries in screening different cell types of bats, especially those of the immune systems, would unveil more secrets of this enigmatic species.</p><p id="p-20">Both screens identified MTHFD1 as an essential viral host factor. MTHFD1 is a key enzyme in the C1 metabolism responsible for production of purine, dTMP and methyl group, the first two of which are building blocks for DNA and RNA synthesis. Interestingly, all previous RNAi and CRISPR screens using human cells have not identified MTHFD1 as a host factor. Moreover, we found bats cells and tissues have lower MTHFD1 expression levels than their human counterparts (<strong><a id="xref-fig-9-1" class="xref-fig" href="#F9">Supplementary Fig. 5a</a>, <a id="xref-fig-9-2" class="xref-fig" href="#F9">5b</a> and <a id="xref-fig-9-3" class="xref-fig" href="#F9">5c</a></strong>). This might be related to the difference in the promoter regions between bats and humans, notably a lack of CCAAT box<sup><a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a></sup> in the promoter of two bat species (<strong><a id="xref-fig-9-4" class="xref-fig" href="#F9">Supplementary Fig. 5d</a>).</strong> We show that MTHFD1 knockdown blocks replication of multiple RNA viruses, including influenza virus, mumps virus, Melaka virus and Zika virus. Importantly, host cells have a higher tolerance for MTHFD1 inhibition than viruses, potentially providing a therapeutic window for targeting MTHFD1 with antiviral drugs. Virus infection converts the whole cellular machinery into a factory to massively produce progeny viruses, which explains the vital dependence on MTHFD1. This is analogous to cancer cell’s addiction to non-oncogenic gene functions such as many metabolic enzymes<sup><a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup>. Similar to the therapeutic targeting of these non-oncogenic addictions for developing cancer drugs, these viral addictions to metabolic enzymes such as MTHFD1 could open doors to many new opportunities for developing broadspectrum antiviral drugs<sup><a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a></sup>. These drugs could be readily applied in emerging epidemics, as demonstrated by our evidence here that MTHFD1 inhibitor carolacton can suppress the infection of SARS-CoV-2. We envision the functional genomics tools we provided here will be instrumental to further discoveries by better understanding the bat-virus interaction.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bats have a lower MTHFD1 expression level compared humans. a. livers and spleens of human and two bat species were applied to RNAseq, and MTHFD1 mRNA level was analyzed. b. MTHFD1 mRNA levels in difference cell lines were compared using qRTPCR with GAPDH as a reference gene. c. MTHFD1 protein levels in different cell lines was compared by Western blotting. d. promoter analysis of MTHFD1 between human and two bat species (P. alecto and E. spelaea). The transcription factor binding sites and structure of promotors were characterized by the help of GPMiner (http://gpminer.mbc.nctu.edu.tw/) and the related information of the regulatory features such as transcription factor binding site (TFBS), G + C content, TATA box, CCAAT box, GC box and DNA stability are shown. Blue and red boxes indicate positions of those putative elements and related transcription factors predicted by GPMiner. A lack of CCAAT box was observed in both bats." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1758073080" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Bats have a lower MTHFD1 expression level compared humans.&lt;/span&gt; &lt;strong&gt;a.&lt;/strong&gt; livers and spleens of human and two bat species were applied to RNAseq, and MTHFD1 mRNA level was analyzed. &lt;strong&gt;b.&lt;/strong&gt; MTHFD1 mRNA levels in difference cell lines were compared using qRTPCR with GAPDH as a reference gene. &lt;strong&gt;c.&lt;/strong&gt; MTHFD1 protein levels in different cell lines was compared by Western blotting. &lt;strong&gt;d.&lt;/strong&gt; promoter analysis of MTHFD1 between human and two bat species &lt;em&gt;(P. alecto&lt;/em&gt; and &lt;em&gt;E. spelaea).&lt;/em&gt; The transcription factor binding sites and structure of promotors were characterized by the help of GPMiner (http://gpminer.mbc.nctu.edu.tw/) and the related information of the regulatory features such as transcription factor binding site (TFBS), G + C content, TATA box, CCAAT box, GC box and DNA stability are shown. Blue and red boxes indicate positions of those putative elements and related transcription factors predicted by GPMiner. A lack of CCAAT box was observed in both bats.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F9.medium.gif" width="344" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Supplementary Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F9.medium.gif" width="344" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/30/2020.03.29.014209/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Supplementary Figure 5.</span> <span class="caption-title">Bats have a lower MTHFD1 expression level compared humans.</span><p id="p-41" class="first-child"><strong>a.</strong> livers and spleens of human and two bat species were applied to RNAseq, and MTHFD1 mRNA level was analyzed. <strong>b.</strong> MTHFD1 mRNA levels in difference cell lines were compared using qRTPCR with GAPDH as a reference gene. <strong>c.</strong> MTHFD1 protein levels in different cell lines was compared by Western blotting. <strong>d.</strong> promoter analysis of MTHFD1 between human and two bat species <em>(P. alecto</em> and <em>E. spelaea).</em> The transcription factor binding sites and structure of promotors were characterized by the help of GPMiner (<a href="http://gpminer.mbc.nctu.edu.tw/">http://gpminer.mbc.nctu.edu.tw/</a>) and the related information of the regulatory features such as transcription factor binding site (TFBS), G + C content, TATA box, CCAAT box, GC box and DNA stability are shown. Blue and red boxes indicate positions of those putative elements and related transcription factors predicted by GPMiner. A lack of CCAAT box was observed in both bats.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-8"><h2 class="">Author contributions</h2><p id="p-21">XT, DEA and L-FW conceived and guided the study. DEA performed and analyzed RNAi screen and RNAi hit validation with assistance from SYK, BGY, KS and XFL. JC and QY performed CRISPR screen and hits validation. JC performed most experiments on MTHFD1. BH performed SARS-CoV-2 infection assay with help from FY and PFY and under the guidance of WT. WZ, JG, WL and XZ performed bioinformatics analysis of the study. XT wrote the manuscript with input from all authors.</p></div><div class="section" id="sec-9"><h2 class="">Methods</h2><div class="section" id="sec-10"><h3>Cells and virus</h3><p id="p-23">PaKi, Vero and 293T cells were cultured in DMEM supplemented with 10% heat inactivated FBS. Influenza H1N1 PR8 (A/Puerto Rico/8/1934) was propagated in embryonated eggs. Mumps virus was propagated in PaKi or Vero cells.</p></div><div class="section" id="sec-11"><h3>Genome-wide CRISPR-Cas9 knockout screen</h3><p id="p-24">Cas9 stably expressed bat kidney derived PaKi cells were transduced with lentivirus of sgRNA library targeting 21300 genes in the genome of <em>P. alecto.</em> The transduced cells were selected with puromycin to generate the library cells. Then the sgRNA library cells were mock infected (negative control) or changed with influenza A H1N1 PR8 strain (A/Puerto Rico/8/1934) at lethal dose. The surviving cells were subjected to two more round infection. The final surviving cells were applying to deep sequencing. And the sgRNA abundance was analyzed by RIGER algorithm<sup><a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">1</a></sup>.</p></div><div class="section" id="sec-12"><h3>Deep sequencing data analysis</h3><p id="p-25">De-multiplexed reads were trimmed by Cutadapt<sup><a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">2</a></sup> (version 1.18) using 22 bp flanking sequences around the guide sequence. Trimmed reads were mapped to the index of bat sgRNA guide library using Bowtie2<sup><a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">3</a></sup>. Samtools<sup><a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">4</a></sup> (version 1.9) was used to process the alignment files to retain perfectly mapped reads only, and a custom-made Python script was used to count the reads mapped to each guide. The reads count of guide for each condition were then normalized using DEseq2. The log2 ratio of counts between conditions were calculated. The log2 fold change of reads count of infected samples to mock infected samples were then used as the input for RIGER algorithm as previously described (RNAi Gene Enrichment Ranking, GENE-E)<sup><a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">1</a></sup>. The log-fold-change metric for ranking sgRNAs and the weighted sum method (weighted sum of first two ranks of sgRNAs for a gene) with 1 × 10<sup>6</sup> permutations were used to convert individual sgRNA to genes.</p><p id="p-26">In the output of RIGER analysis, a given gene was assigned a normalized enrichment score (NES) to give the ranked gene list. Hits were defined as those owning p-value smaller than 0.001.</p></div><div class="section" id="sec-13"><h3>siRNA screen</h3><p id="p-27">For siRNA screen, siRNAs targeting 18,328 genes of <em>P. alecto</em> were arrayed in the 96-well plates. The siRNAs were reverse transfected into PaKi cells, 48 h later, the cells were infected with mumps virus. The virus infection level was measured at 48 h post infection by high content imaging (Operetta) as previously described<sup><a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">5</a></sup>. The siRNA targeted genes were ranked by robust Z scores<sup><a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">6</a></sup>, and the potential host factors (Z score &lt; −3) or restriction factors (Z score &gt; 6) were identified.</p></div><div class="section" id="sec-14"><h3>Validation of sgRNA</h3><p id="p-28">Single sgRNA was cloned into lentiGuide-puro and packaged in lentivirus, which was transduced into Cas9 stably expressing PaKi cells, followed by puromycin and blasticidin selection. The clonal cells were generated by limiting dilution. The knock out or knock down efficiency was tested by Western blotting. To facilitate the growth of single clone MTHFD1 knock out cells, the cells were cultured with HT medium (DMEM supplemented with 10% FBS, 100μM hypoxanthine and 16μM thymidine).The knock out or knock down efficiency was tested by Western blotting. Validated knockout cells were applied to virus infection assays. For MTHFD1 knock out cells, the HT was removed 8h before virus infection.</p></div><div class="section" id="sec-15"><h3>Validation of shRNA</h3><p id="p-29">Individual shRNAs were designed using the BLOC-iT<sup>TM</sup> RNAi Designer (Thermo Fisher Scientific). The specificity was tested by BLAST (NCBI). The shRNA primes were synthesized as oligos and cloned into PLKO.1.</p><p id="p-30">PaKi shMTHFD1-1 forward primer: 5’-ccgg GCACATGGGAATTCCTCTACCctcgag GGTAGAGGAATTCCCATGTGC tttttg-3’ PaKi shMTHFD1-1 reverse primer: 5’-AATTCAAAAA GCACATGGGAATTCCTCTACC CTCGAG GGTAGAGGAATTCCCATGTGC-3’ PaKi shMTHFD1-2 forward primer: 5’-ccgg GCCTGCTGTCACTTAGGAAATctcgag ATTTCCTAAGTGACAGCAGGC tttttg −3’ PaKi shMTHFD1-2 reverse primer: 5’-AATTCAAAAA GCCTGCTGTCACTTAGGAAAT CTCGAG ATTTCCTAAGTGACAGCAGGC −3’</p></div><div class="section" id="sec-16"><h3>Western blot</h3><p id="p-31">Cells were harvested and lysed in RIPA lysis buffer (Beyotime, P0013C) with a cocktail of protease inhibitors (Biomake, B14001). Samples were applied to 10% or 12% SDS/PAGE gels(BioRad) and transferred to PVDF membranes (millipore). Membranes were blocked with 5% nonfat milk in PBS with 0.1% tween 20. Then probed with primary antibodies. The following antibodies were used in this study: anti-MTHFD1 (Proteintech, 10794-1-AP), anti-beta actin (Easybio, BE0022), anti PR8 M1 (Genetex, GTX125928-S), anti HA (H1N1) (Genetex, GTX117951-S), anti-ZIKV NS1 (Genetex, GTX133307), anti flavivirus group antigen antibody (Millipore, MAB10216). The blots were developed by HRP reaction and imaged with a ProteinSimple FluorChem imaging system.</p></div><div class="section" id="sec-17"><h3>RNA isolation and real-time PCR</h3><p id="p-32">Total RNA from the cells were isolated with a HiPure Total RNA Mini Kit (Magen, R4111-03). The cDNA was generated by RevertAid RT Reverse Transcription Kit (thermos scientific, K1691). Influenza vRNA specific primer was used for vRNA transcription. The quantitative realtime PCR was performed with SYBR Green qPCR master mix (Vazyme, Q311-02) and data were analyzed by Bio-Rad CFX96 system. The primers for target genes were as follows: Human GAPDH forward primer: 5’-ACAACTTTGGTATCGTGGAAGG-3’ Human GAPDH reverse primer: 5’ – GCCATCACGCCACAGTTTC-3’ P. alecto ACTIN forward primer: 5’ – gccagtctacaccgtctgcag −3’ P. alecto ACTIN reverse primer: 5’ – cgtaggaatccttctggcccatg-3’ P. alecto MTHFD1 forward primer: 5’- gggagcgactgaagaaccaag-3’ P. alecto MTHFD1 reverse primer: 5’- tcttcagcagccttcagcttcac-3’ P. alecto SEC23B forward primer: 5’- cagcgtttgaccaggaggcc-3’ P. alecto SEC23B reverse primer: 5’- gggtcagatcctgtcgggc −3’ P. alecto GAPDH forward primer: 5’- ATACTTCTCATGGTTCACAC −3’ P. alecto GAPDH reverse primer: 5’ – TCATTGACCTCAACTACATG-3’ P. alecto ATP6V0D1 forward primer: 5’ – GTGGTAGAGTTCCGCCACAT-3’ P. alecto ATP6V0D1 reverse primer: 5’ – CTCAAAGCTGCCTAGTGGGT-3’ PR8 M1 forward primer: 5’ – TTCTAACCGAGGTCGAAACGTACG-3’ PR8 M1 reserve primer: 5’- ACAAAGCGTCTACGCTGCAG-3’ PR8 vRNA reserve transcription primer: 5’-AGCRAAAGCAGG-3’ ZIKV NS5 forward primer: 5’- GGTCAGCGTCCTCTCTAATAAACG-3’ ZIKV NS5 reserve primer: 5’- GCACCCTAGTGTCCACTTTTTCC-3’</p></div><div class="section" id="sec-18"><h3>Immunofluorescence</h3><p id="p-33">Virus infected cells were fixed with 4% paraformaldehyde (PFA) for 10min at room temperature (RT), and permeated with 0.2% Triton X-100 for another 10min at RT, the cells were washed with PBS for 3 times and incubated with PR8 HA antibody or ZIKV E protein antibody for 2h at RT or 4 °C overnight, then washed with PBS for 3 times and incubated with second antibody AF488 (goat anti mouse). The nuclei were stained with DAPI. Images were acquired with Cellomic ArrayScan VTI HCS (Thermo Scientific)</p></div><div class="section" id="sec-19"><h3>Virus titration</h3><p id="p-34">Approximately 1×10<sup>4</sup> PaKi cells were plated into each well of 96-well plates. The supernatant containing virus harvested from previously infected cell was subjected to ten-fold serial dilution. Then dilutions were added to the cells and cytopathic effect was observed 96 h after infection. Titres were determined using standard methodology<sup><a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">7</a></sup>.</p></div><div class="section" id="sec-20"><h3>Influenza virus mini-genome assay</h3><p id="p-35">Approximately 2×10<sup>5</sup> of 293T control cells or 293T MTHFD1 knock out cells were plated into each well of 12-well plates and the knockout cells were cultured with HT (100 μM hypoxanthine and 16 μM thymidine, Gibco) medium. Then the cells were transfected with the plasmids expressing viral polymerase subunits PB1, PB2 and PA, and nucleoprotein, together with pPOLI–HA, transcribing hemagglutinin vRNA segment of A/WSN/33(H1N1) (WSN). At the time of transfection, HT was removed from the cells and supplied with fresh medium with or without inosine (100 μM). The HA protein level were analyzed at 16 h, 28 h, 40 h and 52 h post transfection by Western blot.</p></div><div class="section" id="sec-21"><h3>MTT assay</h3><p id="p-36">MTT cell viability assay (Solarbio) was used to analyze cell viability according to the manufacturer’s instructions. Briefly, 10 μl MTT (0.5%) mixed with 90 μl fresh medium was added into the well. The cells were incubated at 37°C for 4 h, after the formazan crystals were formed, the medium was removed and 110 μl DMSO was used to solubilize the formazan crystals. The absorbance at 490 nm was measured with a multimode plate reader victory x5 (Perkin Elmer).</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-22">We thank Fu Chengzhang and Rolf Müller for providing carolacton. We thank Jianzhong Xi for oligonucleotide synthesis. XT is supported by the the China National Funds for Excellent Young Scientists (31722030) and was supported by the Beijing Advanced Innovation Center for Structural Biology and the Tsinghua-Peking Joint Center for Life Sciences. DEA and L-FW are supported by grants NRF2012NRF-CRP001–056 and NRF2016NRF-NSFC002-013 from the Singapore National Research Foundation and National Medical Research Council Cooperative Basic Research Grant NMRC/BNIG/2030/2015.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Reference</h2><ol class="cit-list"><li><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1 in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.1" data-doi="10.1038/438575a"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Leroy</span>,  <span class="cit-name-given-names">E. M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Fruit bats as reservoirs of Ebola virus</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">438</span>, <span class="cit-fpage">575</span>–<span class="cit-lpage">576</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/438575a</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLeroy%26rft.auinit1%253DE.%2BM.%26rft.volume%253D438%26rft.issue%253D7068%26rft.spage%253D575%26rft.epage%253D576%26rft.atitle%253DFruit%2Bbats%2Bas%2Breservoirs%2Bof%2BEbola%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F438575a%26rft_id%253Dinfo%253Apmid%252F16319873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/438575a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16319873&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.29.014209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233593100030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2 in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.2" data-doi="10.1126/science.1118391"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Bats are natural reservoirs of SARS-like coronaviruses</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">310</span>, <span class="cit-fpage">676</span>–<span class="cit-lpage">679</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.1118391</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1118391%26rft_id%253Dinfo%253Apmid%252F16195424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTAvNTc0OC82NzYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8zMC8yMDIwLjAzLjI5LjAxNDIwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3 in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.3" data-doi="10.1111/j.1863-2378.2012.01501.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Clayton</span>,  <span class="cit-name-given-names">B. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">L. F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Marsh</span>,  <span class="cit-name-given-names">G. A.</span></span> <span class="cit-article-title">Henipaviruses: an updated review focusing on the pteropid reservoir and features of transmission</span>. <abbr class="cit-jnl-abbrev">Zoonoses Public Health</abbr> <span class="cit-vol">60</span>, <span class="cit-fpage">69</span>–<span class="cit-lpage">83</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1863-2378.2012.01501.x</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZoonoses%2BPublic%2BHealth%26rft.volume%253D60%26rft.spage%253D69%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1863-2378.2012.01501.x%26rft_id%253Dinfo%253Apmid%252F22709528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1863-2378.2012.01501.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22709528&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.29.014209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4 in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.4" data-doi="10.1186/s12985-016-0544-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mohd</span>,  <span class="cit-name-given-names">H. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Al-Tawfiq</span>,  <span class="cit-name-given-names">J. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Memish</span>,  <span class="cit-name-given-names">Z. A.</span></span> <span class="cit-article-title">Middle East Respiratory Syndrome Coronavirus (MERS-CoV) origin and animal reservoir</span>. <abbr class="cit-jnl-abbrev">Virol J</abbr> <span class="cit-vol">13</span>, <span class="cit-fpage">87</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/s12985-016-0544-0</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirol%2BJ%26rft.volume%253D13%26rft.spage%253D87%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12985-016-0544-0%26rft_id%253Dinfo%253Apmid%252F27255185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12985-016-0544-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27255185&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.29.014209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5 in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.5" data-doi="10.1038/s41586-020-2008-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A new coronavirus associated with human respiratory disease in China</span>. <abbr class="cit-jnl-abbrev">Nature</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41586-020-2008-3</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2008-3%26rft_id%253Dinfo%253Apmid%252F32015508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-020-2008-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32015508&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.29.014209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6 in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.6" data-doi="10.1038/s41586-020-2012-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. <abbr class="cit-jnl-abbrev">Nature</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41586-020-2012-7</span> (<span class="cit-pub-date">2020</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2012-7%26rft_id%253Dinfo%253Apmid%252F32015507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-020-2012-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=32015507&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.29.014209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7 in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.7" data-doi="10.1038/nature22975"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Olival</span>,  <span class="cit-name-given-names">K. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Host and viral traits predict zoonotic spillover from mammals</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">546</span>, <span class="cit-fpage">646</span>–<span class="cit-lpage">650</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature22975</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D546%26rft.spage%253D646%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature22975%26rft_id%253Dinfo%253Apmid%252F28636590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature22975&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28636590&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.29.014209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8 in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.8" data-doi="10.1016/j.coviro.2018.12.007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">L. F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">D. E.</span></span> <span class="cit-article-title">Viruses in bats and potential spillover to animals and humans</span>. <abbr class="cit-jnl-abbrev">Curr Opin Virol</abbr> <span class="cit-vol">34</span>, <span class="cit-fpage">79</span>–<span class="cit-lpage">89</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.coviro.2018.12.007</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr%2BOpin%2BVirol%26rft.volume%253D34%26rft.spage%253D79%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.coviro.2018.12.007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.coviro.2018.12.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9 in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.9" data-doi="10.1126/science.1230835"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Comparative analysis of bat genomes provides insight into the evolution of flight and immunity</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">339</span>, <span class="cit-fpage">456</span>–<span class="cit-lpage">460</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.1230835</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1230835%26rft_id%253Dinfo%253Apmid%252F23258410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzkvNjExOC80NTYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8zMC8yMDIwLjAzLjI5LjAxNDIwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10 in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.10" data-doi="10.1016/j.cell.2018.03.070"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pavlovich</span>,  <span class="cit-name-given-names">S. S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The Egyptian Rousette Genome Reveals Unexpected Features of Bat Antiviral Immunity</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">173</span>, <span class="cit-fpage">1098</span>–<span class="cit-lpage">1110</span> <span class="cit-page-range">e1018</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cell.2018.03.070</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D173%26rft.spage%253D1098%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2018.03.070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2018.03.070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11 in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.11" data-doi="10.1146/annurev-biochem-060408-092949"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mohr</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bakal</span>,  <span class="cit-name-given-names">C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Perrimon</span>,  <span class="cit-name-given-names">N.</span></span> <span class="cit-article-title">Genomic screening with RNAi: results and challenges</span>. <abbr class="cit-jnl-abbrev">Annu Rev Biochem</abbr> <span class="cit-vol">79</span>, <span class="cit-fpage">37</span>–<span class="cit-lpage">64</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1146/annurev-biochem-060408-092949</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.aulast%253DMohr%26rft.auinit1%253DS.%26rft.volume%253D79%26rft.spage%253D37%26rft.epage%253D64%26rft.atitle%253DGenomic%2Bscreening%2Bwith%2BRNAi%253A%2Bresults%2Band%2Bchallenges.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-biochem-060408-092949%26rft_id%253Dinfo%253Apmid%252F20367032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-biochem-060408-092949&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20367032&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.29.014209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280225300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12 in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.12" data-doi="10.1038/nrg3899"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shalem</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sanjana</span>,  <span class="cit-name-given-names">N. E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">High-throughput functional genomics using CRISPR-Cas9</span>. <abbr class="cit-jnl-abbrev">Nat Rev Genet</abbr> <span class="cit-vol">16</span>, <span class="cit-fpage">299</span>–<span class="cit-lpage">311</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nrg3899</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BRev%2BGenet%26rft.volume%253D16%26rft.spage%253D299%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3899%26rft_id%253Dinfo%253Apmid%252F25854182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3899&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25854182&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.29.014209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13 in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.13" data-doi="10.1016/j.coviro.2018.03.007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McDougall</span>,  <span class="cit-name-given-names">W. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Perreira</span>,  <span class="cit-name-given-names">J. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reynolds</span>,  <span class="cit-name-given-names">E. C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Brass</span>,  <span class="cit-name-given-names">A. L.</span></span> <span class="cit-article-title">CRISPR genetic screens to discover host-virus interactions</span>. <abbr class="cit-jnl-abbrev">Curr Opin Virol</abbr> <span class="cit-vol">29</span>, <span class="cit-fpage">87</span>–<span class="cit-lpage">100</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.coviro.2018.03.007</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr%2BOpin%2BVirol%26rft.volume%253D29%26rft.spage%253D87%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.coviro.2018.03.007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.coviro.2018.03.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.29.014209v1.14" data-doi="10.1016/j.coviro.2012.10.007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Panda</span>,  <span class="cit-name-given-names">D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Cherry</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Cell-based genomic screening: elucidating virus-host interactions</span>. <abbr class="cit-jnl-abbrev">Curr Opin Virol</abbr> <span class="cit-vol">2</span>, <span class="cit-fpage">784</span>–<span class="cit-lpage">792</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.coviro.2012.10.007</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr%2BOpin%2BVirol%26rft.volume%253D2%26rft.spage%253D784%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.coviro.2012.10.007%26rft_id%253Dinfo%253Apmid%252F23122855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.coviro.2012.10.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23122855&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.29.014209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15 in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.15" data-doi="10.1038/nrmicro.2017.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Puschnik</span>,  <span class="cit-name-given-names">A. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Majzoub</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ooi</span>,  <span class="cit-name-given-names">Y. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Carette</span>,  <span class="cit-name-given-names">J. E.</span></span> <span class="cit-article-title">A CRISPR toolbox to study virushost interactions</span>. <abbr class="cit-jnl-abbrev">Nat Rev Microbiol</abbr> <span class="cit-vol">15</span>, <span class="cit-fpage">351</span>–<span class="cit-lpage">364</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nrmicro.2017.29</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BRev%2BMicrobiol%26rft.volume%253D15%26rft.spage%253D351%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro.2017.29%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro.2017.29&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16 in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.16" data-doi="10.1016/j.tibtech.2018.08.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schuster</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">RNAi/CRISPR Screens: from a Pool to a Valid Hit</span>. <abbr class="cit-jnl-abbrev">Trends Biotechnol</abbr> <span class="cit-vol">37</span>, <span class="cit-fpage">38</span>–<span class="cit-lpage">55</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.tibtech.2018.08.002</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BBiotechnol%26rft.volume%253D37%26rft.spage%253D38%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibtech.2018.08.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibtech.2018.08.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17 in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.17" data-doi="10.1016/bs.aivir.2015.11.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Perreira</span>,  <span class="cit-name-given-names">J. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meraner</span>,  <span class="cit-name-given-names">P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Brass</span>,  <span class="cit-name-given-names">A. L.</span></span> <span class="cit-article-title">Functional Genomic Strategies for Elucidating Human-Virus Interactions: Will CRISPR Knockout RNAi and Haploid Cells?</span> <abbr class="cit-jnl-abbrev">Adv Virus Res</abbr> <span class="cit-vol">94</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">51</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/bs.aivir.2015.11.001</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv%2BVirus%2BRes%26rft.volume%253D94%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fbs.aivir.2015.11.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/bs.aivir.2015.11.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18 in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.18" data-doi="10.1177/1087057115590069"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Taylor</span>,  <span class="cit-name-given-names">J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Woodcock</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">A Perspective on the Future of High-Throughput RNAi Screening: Will CRISPR Cut Out the Competition or Can RNAi Help Guide the Way?</span> <abbr class="cit-jnl-abbrev">J Biomol Screen</abbr> <span class="cit-vol">20</span>, <span class="cit-fpage">1040</span>–<span class="cit-lpage">1051</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1177/1087057115590069</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiomol%2BScreen%26rft_id%253Dinfo%253Adoi%252F10.1177%252F1087057115590069%26rft_id%253Dinfo%253Apmid%252F26048892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1177/1087057115590069&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26048892&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.29.014209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19 in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.19" data-doi="10.1126/science.1247005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shalem</span>,  <span class="cit-name-given-names">O.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Genome-scale CRISPR-Cas9 knockout screening in human cells</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">343</span>, <span class="cit-fpage">84</span>–<span class="cit-lpage">87</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.1247005</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1247005%26rft_id%253Dinfo%253Apmid%252F24336571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzNDMvNjE2Ni84NCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzMwLzIwMjAuMDMuMjkuMDE0MjA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20 in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.20" data-doi="10.1073/pnas.0810485105"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Luo</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Highly parallel identification of essential genes in cancer cells</span>. <abbr class="cit-jnl-abbrev">Proc Natl AcadSci USA</abbr> <span class="cit-vol">105</span>, <span class="cit-fpage">20380</span>–<span class="cit-lpage">20385</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.0810485105</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcadSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0810485105%26rft_id%253Dinfo%253Apmid%252F19091943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzUxLzIwMzgwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMzAvMjAyMC4wMy4yOS4wMTQyMDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21 in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.21" data-doi="10.1016/j.chom.2010.05.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Watanabe</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Kawaoka</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-article-title">Cellular networks involved in the influenza virus life cycle</span>. <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr> <span class="cit-vol">7</span>, <span class="cit-fpage">427</span>–<span class="cit-lpage">439</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.chom.2010.05.008</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCell%2BHost%2BMicrobe%26rft.aulast%253DWatanabe%26rft.auinit1%253DT.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253D427%26rft.epage%253D439%26rft.atitle%253DCellular%2Bnetworks%2Binvolved%2Bin%2Bthe%2Binfluenza%2Bvirus%2Blife%2Bcycle.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2010.05.008%26rft_id%253Dinfo%253Apmid%252F20542247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2010.05.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20542247&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.29.014209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279209200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.29.014209v1.22" data-doi="10.1128/JVI.01757-16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Host Cellular Protein TRAPPC6ADelta Interacts with Influenza A Virus M2 Protein and Regulates Viral Propagation by Modulating M2 Trafficking</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">91</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.01757-16</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01757-16%26rft_id%253Dinfo%253Apmid%252F27795429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjE0OiI5MS8xL2UwMTc1Ny0xNiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzMwLzIwMjAuMDMuMjkuMDE0MjA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23 in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.23" data-doi="10.1016/j.cell.2018.11.028"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shah</span>,  <span class="cit-name-given-names">P. S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Comparative Flavivirus-Host Protein Interaction Mapping Reveals Mechanisms of Dengue and Zika Virus Pathogenesis</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">175</span>, <span class="cit-fpage">1931</span>–<span class="cit-lpage">1945</span> <span class="cit-page-range">e1918</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cell.2018.11.028</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D175%26rft.spage%253D1931%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2018.11.028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2018.11.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24 in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.24" data-doi="10.1038/nmeth.3047"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sanjana</span>,  <span class="cit-name-given-names">N. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shalem</span>,  <span class="cit-name-given-names">O.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">Improved vectors and genome-wide libraries for CRISPR screening</span>. <abbr class="cit-jnl-abbrev">Nat Methods</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">783</span>–<span class="cit-lpage">784</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nmeth.3047</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D11%26rft.spage%253D783%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3047%26rft_id%253Dinfo%253Apmid%252F25075903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3047&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25075903&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.29.014209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340075600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25 in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.25" data-doi="10.1128/mBio.00826-19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">D. E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Comparative Loss-of-Function Screens Reveal ABCE1 as an Essential Cellular Host Factor for Efficient Translation of Paramyxoviridae and Pneumoviridae</span>. <abbr class="cit-jnl-abbrev">mBio</abbr> <span class="cit-vol">10</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/mBio.00826-19</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DmBio%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.00826-19%26rft_id%253Dinfo%253Apmid%252F31088929%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibWJpbyI7czo1OiJyZXNpZCI7czoxNDoiMTAvMy9lMDA4MjYtMTkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8zMC8yMDIwLjAzLjI5LjAxNDIwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26 in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.26" data-doi="10.1146/annurev-virology-092917-043323"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van Gent</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sparrer</span>,  <span class="cit-name-given-names">K. M. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Gack</span>,  <span class="cit-name-given-names">M. U.</span></span> <span class="cit-article-title">TRIM Proteins and Their Roles in Antiviral Host Defenses</span>. <abbr class="cit-jnl-abbrev">Annu Rev Virol</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">385</span>–<span class="cit-lpage">405</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1146/annurev-virology-092917-043323</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu%2BRev%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-virology-092917-043323%26rft_id%253Dinfo%253Apmid%252F29949725%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-virology-092917-043323&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29949725&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.29.014209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27 in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.27" data-doi="10.1038/nature09907"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schoggins</span>,  <span class="cit-name-given-names">J. W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A diverse range of gene products are effectors of the type I interferon antiviral response</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">472</span>, <span class="cit-fpage">481</span>–<span class="cit-lpage">485</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature09907</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSchoggins%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D472%26rft.issue%253D7344%26rft.spage%253D481%26rft.epage%253D485%26rft.atitle%253DA%2Bdiverse%2Brange%2Bof%2Bgene%2Bproducts%2Bare%2Beffectors%2Bof%2Bthe%2Btype%2BI%2Binterferon%2Bantiviral%2Bresponse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09907%26rft_id%253Dinfo%253Apmid%252F21478870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09907&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478870&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.29.014209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289949600036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28 in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.28" data-doi="10.1073/pnas.0701372104"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chua</span>,  <span class="cit-name-given-names">K. B.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A previously unknown reovirus of bat origin is associated with an acute respiratory disease in humans</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr> <span class="cit-vol">104</span>, <span class="cit-fpage">11424</span>–<span class="cit-lpage">11429</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.0701372104</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0701372104%26rft_id%253Dinfo%253Apmid%252F17592121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzI3LzExNDI0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMzAvMjAyMC4wMy4yOS4wMTQyMDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29 in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.29" data-doi="10.1016/j.cmet.2016.08.009"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ducker</span>,  <span class="cit-name-given-names">G. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Rabinowitz</span>,  <span class="cit-name-given-names">J. D.</span></span> <span class="cit-article-title">One-Carbon Metabolism in Health and Disease</span>. <abbr class="cit-jnl-abbrev">Cell Metab</abbr> <span class="cit-vol">25</span>, <span class="cit-fpage">27</span>–<span class="cit-lpage">42</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cmet.2016.08.009</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BMetab%26rft.volume%253D25%26rft.spage%253D27%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cmet.2016.08.009%26rft_id%253Dinfo%253Apmid%252F27641100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cmet.2016.08.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27641100&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.29.014209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30 in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.30" data-doi="10.1073/pnas.1414555112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Field</span>,  <span class="cit-name-given-names">M. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kamynina</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Watkins</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rosenblatt</span>,  <span class="cit-name-given-names">D. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Stover</span>,  <span class="cit-name-given-names">P. J.</span></span> <span class="cit-article-title">Human mutations in methylenetetrahydrofolate dehydrogenase 1 impair nuclear de novo thymidylate biosynthesis</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci US A</abbr> <span class="cit-vol">112</span>, <span class="cit-fpage">400</span>–<span class="cit-lpage">405</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1414555112</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1414555112%26rft_id%253Dinfo%253Apmid%252F25548164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMTIvMi80MDAiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8zMC8yMDIwLjAzLjI5LjAxNDIwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31 in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.31" data-doi="10.1074/jbc.M200127200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sundararajan</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">MacKenzie</span>,  <span class="cit-name-given-names">R. E.</span></span> <span class="cit-article-title">Residues involved in the mechanism of the bifunctional methylenetetrahydrofolate dehydrogenase-cyclohydrolase: the roles of glutamine 100 and aspartate 125</span>. <abbr class="cit-jnl-abbrev">J Biol Chem</abbr> <span class="cit-vol">277</span>, <span class="cit-fpage">18703</span>–<span class="cit-lpage">18709</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1074/jbc.M200127200</span> (<span class="cit-pub-date">2002</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiol%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M200127200%26rft_id%253Dinfo%253Apmid%252F11904299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzcvMjEvMTg3MDMiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8zMC8yMDIwLjAzLjI5LjAxNDIwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32 in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.32" data-doi="10.1002/pro.2005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Celeste</span>,  <span class="cit-name-given-names">L. R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Mechanism of N10-formyltetrahydrofolate synthetase derived from complexes with intermediates and inhibitors</span>. <abbr class="cit-jnl-abbrev">Protein Sci</abbr> <span class="cit-vol">21</span>, <span class="cit-fpage">219</span>–<span class="cit-lpage">228</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1002/pro.2005</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtein%2BScience%26rft.stitle%253DProtein%2BSci.%26rft.aulast%253DCeleste%26rft.auinit1%253DL.%2BR.%26rft.volume%253D21%26rft.issue%253D2%26rft.spage%253D219%26rft.epage%253D228%26rft.atitle%253DMechanism%2Bof%2BN10-formyltetrahydrofolate%2Bsynthetase%2Bderived%2Bfrom%2Bcomplexes%2Bwith%2Bintermediates%2Band%2Binhibitors.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpro.2005%26rft_id%253Dinfo%253Apmid%252F22109967%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/pro.2005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22109967&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.29.014209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33 in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.33" data-doi="10.1038/s41467-017-01671-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fu</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The natural product carolacton inhibits folate-dependent C1 metabolism by targeting FolD/MTHFD</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">1529</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41467-017-01671-5</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D8%26rft.spage%253D1529%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-017-01671-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-017-01671-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34 in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.34" data-doi="10.1016/j.bbagrm.2016.10.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martyn</span>,  <span class="cit-name-given-names">G. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Quinlan</span>,  <span class="cit-name-given-names">K. G. R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Crossley</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">The regulation of human globin promoters by CCAAT box elements and the recruitment of NF-Y</span>. <abbr class="cit-jnl-abbrev">Biochim Biophys Acta Gene Regul Mech</abbr> <span class="cit-vol">1860</span>, <span class="cit-fpage">525</span>–<span class="cit-lpage">536</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.bbagrm.2016.10.002</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochim%2BBiophys%2BActa%2BGene%2BRegul%2BMech%26rft.volume%253D1860%26rft.spage%253D525%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbagrm.2016.10.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbagrm.2016.10.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35 in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.35" data-doi="10.1016/j.cell.2009.02.024"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Luo</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Solimini</span>,  <span class="cit-name-given-names">N. L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Elledge</span>,  <span class="cit-name-given-names">S. J.</span></span> <span class="cit-article-title">Principles of cancer therapy: oncogene and nononcogene addiction</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">136</span>, <span class="cit-fpage">823</span>–<span class="cit-lpage">837</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cell.2009.02.024</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLuo%26rft.auinit1%253DJ.%26rft.volume%253D136%26rft.issue%253D5%26rft.spage%253D823%26rft.epage%253D837%26rft.atitle%253DPrinciples%2Bof%2Bcancer%2Btherapy%253A%2Boncogene%2Band%2Bnon-oncogene%2Baddiction.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2009.02.024%26rft_id%253Dinfo%253Apmid%252F19269363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2009.02.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19269363&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.29.014209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263930900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36 in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.36" data-doi="10.1038/nchembio.2050"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Deans</span>,  <span class="cit-name-given-names">R. M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Parallel shRNA and CRISPR-Cas9 screens enable antiviral drug target identification</span>. <abbr class="cit-jnl-abbrev">Nat Chem Biol</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">361</span>–<span class="cit-lpage">366</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nchembio.2050</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BChem%2BBiol%26rft.volume%253D12%26rft.spage%253D361%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnchembio.2050%26rft_id%253Dinfo%253Apmid%252F27018887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nchembio.2050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27018887&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.29.014209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><div class="section ref-list" id="ref-list-2"><ol class="cit-list"><li><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 1 in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.37" data-doi="10.1073/pnas.0810485105"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Luo</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Highly parallel identification of essential genes in cancer cells</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr> <span class="cit-vol">105</span>, <span class="cit-fpage">20380</span>–<span class="cit-lpage">20385</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.0810485105</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0810485105%26rft_id%253Dinfo%253Apmid%252F19091943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzUxLzIwMzgwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMzAvMjAyMC4wMy4yOS4wMTQyMDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 2 in text" id="ref-38">↵</a><div class="cit ref-cit ref-web" id="cit-2020.03.29.014209v1.38"><div class="cit-metadata unstructured"><a href="https://doi.org/10.14806/ej.17.1.200">https://doi.org/10.14806/ej.17.1.200</a></div><div class="cit-extra"></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 3 in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.39" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Langmead</span>,  <span class="cit-name-given-names">B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Salzberg</span>,  <span class="cit-name-given-names">S. L.</span></span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2</span>. <abbr class="cit-jnl-abbrev">Nat Methods</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nmeth.1923</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.29.014209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 4 in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.40" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>, <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btp352</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.29.014209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 5 in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.41" data-doi="10.1128/mBio.00826-19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">D. E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Comparative Loss-of-Function Screens Reveal ABCE1 as an Essential Cellular Host Factor for Efficient Translation of Paramyxoviridae and Pneumoviridae</span>. <abbr class="cit-jnl-abbrev">mBio</abbr> <span class="cit-vol">10</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1128/mBio.00826-19</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DmBio%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.00826-19%26rft_id%253Dinfo%253Apmid%252F31088929%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibWJpbyI7czo1OiJyZXNpZCI7czoxNDoiMTAvMy9lMDA4MjYtMTkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8zMC8yMDIwLjAzLjI5LjAxNDIwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 6 in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.42" data-doi="10.1038/nbt1186"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Malo</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hanley</span>,  <span class="cit-name-given-names">J. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cerquozzi</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pelletier</span>,  <span class="cit-name-given-names">J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Nadon</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Statistical practice in high-throughput screening data analysis</span>. <abbr class="cit-jnl-abbrev">Nat Biotechnol</abbr> <span class="cit-vol">24</span>, <span class="cit-fpage">167</span>–<span class="cit-lpage">175</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nbt1186</span> (<span class="cit-pub-date">2006</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DMalo%26rft.auinit1%253DN.%26rft.volume%253D24%26rft.issue%253D2%26rft.spage%253D167%26rft.epage%253D175%26rft.atitle%253DStatistical%2Bpractice%2Bin%2Bhigh-throughput%2Bscreening%2Bdata%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt1186%26rft_id%253Dinfo%253Apmid%252F16465162%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt1186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16465162&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F30%2F2020.03.29.014209.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235232300032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 7 in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.29.014209v1.43" data-doi="10.1002/prot.22879"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reed</span>,  <span class="cit-name-given-names">LJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muench</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-article-title">A simple method of estimating fifty percent endpoints</span>. <abbr class="cit-jnl-abbrev">Am. J. Hygiene</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">493</span>–<span class="cit-lpage">497</span> (<span class="cit-pub-date">1938</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHygiene%26rft.volume%253D27%26rft.spage%253D493%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fprot.22879%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/prot.22879&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.29.014209v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.29.014209v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.03.28.013508v1" title="SARS-CoV-2 detection from nasopharyngeal swab samples without RNA extraction" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.02.973412v2" title="A Histoplasma capsulatum lipid metabolic map identifies antifungal targets and virulence factors" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 30, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.29.014209v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.29.014209v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.29.014209v1" class="active" data-icon-position="" data-hide-link-title="0">Orthogonal genome-wide screenings in bat cells identify MTHFD1 as a target of broad antiviral therapy</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1217863" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-fr1qX30flT0TnrzpwWxe7OwNc4l2bfc72FauKcix2YU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1217863" id="node1217863--2" data-pisa="biorxiv;2020.03.29.014209v1" data-pisa-master="biorxiv;2020.03.29.014209" data-seqnum="1217863" data-apath="/biorxiv/early/2020/03/30/2020.03.29.014209.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Orthogonal genome-wide screenings in bat cells identify MTHFD1 as a target of broad antiviral therapy</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Danielle E</span> <span class="nlm-surname">Anderson</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Jin</span> <span class="nlm-surname">Cui</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Qian</span> <span class="nlm-surname">Ye</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Baoying</span> <span class="nlm-surname">Huang</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Wenhong</span> <span class="nlm-surname">Zu</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Jing</span> <span class="nlm-surname">Gong</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Weiqiang</span> <span class="nlm-surname">Liu</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">So Young</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Biao Guo</span> <span class="nlm-surname">Yan</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Kristmundur</span> <span class="nlm-surname">Sigmundsson</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Xiao Fang</span> <span class="nlm-surname">Lim</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Fei</span> <span class="nlm-surname">Ye</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Peihua</span> <span class="nlm-surname">Niu</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Xuming</span> <span class="nlm-surname">Zhou</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Wenjie</span> <span class="nlm-surname">Tan</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Lin-Fa</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">Xu</span> <span class="nlm-surname">Tan</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.29.014209; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.29.014209 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.29.014209v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.29.014209v1&amp;title=Orthogonal%20genome-wide%20screenings%20in%20bat%20cells%20identify%20MTHFD1%20as%20a%20target%20of%20broad%20antiviral%20therapy" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.29.014209v1&amp;title=Orthogonal%20genome-wide%20screenings%20in%20bat%20cells%20identify%20MTHFD1%20as%20a%20target%20of%20broad%20antiviral%20therapy" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.29.014209v1&amp;text=Orthogonal%20genome-wide%20screenings%20in%20bat%20cells%20identify%20MTHFD1%20as%20a%20target%20of%20broad%20antiviral%20therapy" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.29.014209v1&amp;title=Orthogonal%20genome-wide%20screenings%20in%20bat%20cells%20identify%20MTHFD1%20as%20a%20target%20of%20broad%20antiviral%20therapy" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.29.014209v1&amp;t=Orthogonal%20genome-wide%20screenings%20in%20bat%20cells%20identify%20MTHFD1%20as%20a%20target%20of%20broad%20antiviral%20therapy" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.29.014209v1&amp;title=Orthogonal%20genome-wide%20screenings%20in%20bat%20cells%20identify%20MTHFD1%20as%20a%20target%20of%20broad%20antiviral%20therapy" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.29.014209v1&amp;title=Orthogonal%20genome-wide%20screenings%20in%20bat%20cells%20identify%20MTHFD1%20as%20a%20target%20of%20broad%20antiviral%20therapy" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1217863" id="node1217863--3" data-pisa="biorxiv;2020.03.29.014209v1" data-pisa-master="biorxiv;2020.03.29.014209" data-seqnum="1217863" data-apath="/biorxiv/early/2020/03/30/2020.03.29.014209.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Orthogonal genome-wide screenings in bat cells identify MTHFD1 as a target of broad antiviral therapy</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Danielle E</span> <span class="nlm-surname">Anderson</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Jin</span> <span class="nlm-surname">Cui</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Qian</span> <span class="nlm-surname">Ye</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Baoying</span> <span class="nlm-surname">Huang</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Wenhong</span> <span class="nlm-surname">Zu</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Jing</span> <span class="nlm-surname">Gong</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Weiqiang</span> <span class="nlm-surname">Liu</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">So Young</span> <span class="nlm-surname">Kim</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Biao Guo</span> <span class="nlm-surname">Yan</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Kristmundur</span> <span class="nlm-surname">Sigmundsson</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Xiao Fang</span> <span class="nlm-surname">Lim</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Fei</span> <span class="nlm-surname">Ye</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Peihua</span> <span class="nlm-surname">Niu</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Xuming</span> <span class="nlm-surname">Zhou</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Wenjie</span> <span class="nlm-surname">Tan</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Lin-Fa</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">Xu</span> <span class="nlm-surname">Tan</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.29.014209; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.29.014209 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Orthogonal%20genome-wide%20screenings%20in%20bat%20cells%20identify%20MTHFD1%20as%20a%20target%20of%20broad%20antiviral%20therapy&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Anderson%2C+Danielle+E&amp;rft.au=Cui%2C+Jin&amp;rft.au=Ye%2C+Qian&amp;rft.au=Huang%2C+Baoying&amp;rft.au=Zu%2C+Wenhong&amp;rft.au=Gong%2C+Jing&amp;rft.au=Liu%2C+Weiqiang&amp;rft.au=Kim%2C+So+Young&amp;rft.au=Yan%2C+Biao+Guo&amp;rft.au=Sigmundsson%2C+Kristmundur&amp;rft.au=Lim%2C+Xiao+Fang&amp;rft.au=Ye%2C+Fei&amp;rft.au=Niu%2C+Peihua&amp;rft.au=Zhou%2C+Xuming&amp;rft.au=Tan%2C+Wenjie&amp;rft.au=Wang%2C+Lin-Fa&amp;rft.au=Tan%2C+Xu"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1217863/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1217863/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1217863/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1217863/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1217863/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1217863/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1217863/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1217863/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1217863/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1217863/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1217863/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1217863/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.29.014209v1&amp;count=horizontal&amp;via=&amp;text=Orthogonal%20genome-wide%20screenings%20in%20bat%20cells%20identify%20MTHFD1%20as%20a%20target%20of%20broad%20antiviral%20therapy&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.29.014209v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.29.014209v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.29.014209v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">Microbiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
