<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2017/07/24/167999.atom" />
<meta name="HW.pisa" content="biorxiv;167999v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The IFN response in bat cells consists of canonical and non-canonical ISGs with unique temporal expression kinetics" />
<meta name="DC.Identifier" content="10.1101/167999" />
<meta name="DC.Date" content="2017-07-24" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2017, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Bats are reservoirs for a number of highly pathogenic zoonotic viruses, yet they remain relatively asymptomatic during infection. Whether this viral resistance is due to a unique innate immune system is unknown. An evolutionarily conserved feature of vertebrate antiviral immunity is the interferon (IFN) response, which triggers cellular defenses through interferon-stimulated gene (ISG) expression. While bats encode an intact IFN system, global ISG expression patterns in bat cells are not well characterized. Here, we used RNA-Seq to assess the transcriptional response to IFNα in cells derived from the bat Pteropus alecto (black flying fox). We show induction of more than 100 transcripts, most of which are canonical ISGs observed in other species. Kinetic gene profiling revealed that P. alecto ISGs fall into two unique temporal subclusters with similar early induction kinetics but distinct late-phase declines. In contrast to bat ISGs, human ISGs generally remained elevated for longer periods following IFN treatment, suggesting host-based differences in gene regulatory mechanisms. Notably, we also identified a small group of non-canonical bat ISGs, including an enzymatically active RNASEL that plays a role in controlling viral infection. These studies provide insight into the innate immune response of an important viral reservoir and lay a foundation for studies into the immunological features that may underlie unique virus-host relationship in bats.

Significance Statement Bats are considered unique in their ability to resist disease caused by viruses that are often pathogenic in humans. While the nature of this viral resistance is unknown, genomic data suggest bat innate immune systems may be specialized in controlling these disease-causing viruses. A critical cell intrinsic antiviral defense system in vertebrates is the interferon response, which suppresses viral infection through induction of hundreds of interferon-stimulated genes (ISGs). In this study, we report the repertoire of ISGs and several unique features of ISG induction kinetics in bat cells. We also characterize induction and antiviral activity of bat RNASEL, which is induced by IFN in bat, but not human cells. These studies lay the foundation for discovery of potentially new antiviral mechanisms in bats, which may spur research into development of therapies to combat viral infection." />
<meta name="DC.Contributor" content="Pamela C. De La Cruz-Rivera" />
<meta name="DC.Contributor" content="Mohammed Kanchwala" />
<meta name="DC.Contributor" content="Hanquan Liang" />
<meta name="DC.Contributor" content="Ashwani Kumar" />
<meta name="DC.Contributor" content="Lin-Fa Wang" />
<meta name="DC.Contributor" content="Chao Xing" />
<meta name="DC.Contributor" content="John W. Schoggins" />
<meta name="article:published_time" content="2017-07-24" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="The IFN response in bat cells consists of canonical and non-canonical ISGs with unique temporal expression kinetics" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Bats are reservoirs for a number of highly pathogenic zoonotic viruses, yet they remain relatively asymptomatic during infection. Whether this viral resistance is due to a unique innate immune system is unknown. An evolutionarily conserved feature of vertebrate antiviral immunity is the interferon (IFN) response, which triggers cellular defenses through interferon-stimulated gene (ISG) expression. While bats encode an intact IFN system, global ISG expression patterns in bat cells are not well characterized. Here, we used RNA-Seq to assess the transcriptional response to IFNα in cells derived from the bat &lt;i&gt;Pteropus alecto&lt;/i&gt; (black flying fox). We show induction of more than 100 transcripts, most of which are canonical ISGs observed in other species. Kinetic gene profiling revealed that &lt;i&gt;P. alecto&lt;/i&gt; ISGs fall into two unique temporal subclusters with similar early induction kinetics but distinct late-phase declines. In contrast to bat ISGs, human ISGs generally remained elevated for longer periods following IFN treatment, suggesting host-based differences in gene regulatory mechanisms. Notably, we also identified a small group of non-canonical bat ISGs, including an enzymatically active RNASEL that plays a role in controlling viral infection. These studies provide insight into the innate immune response of an important viral reservoir and lay a foundation for studies into the immunological features that may underlie unique virus-host relationship in bats.&lt;/p&gt;&lt;h3&gt;Significance Statement&lt;/h3&gt;
&lt;p&gt;Bats are considered unique in their ability to resist disease caused by viruses that are often pathogenic in humans. While the nature of this viral resistance is unknown, genomic data suggest bat innate immune systems may be specialized in controlling these disease-causing viruses. A critical cell intrinsic antiviral defense system in vertebrates is the interferon response, which suppresses viral infection through induction of hundreds of interferon-stimulated genes (ISGs). In this study, we report the repertoire of ISGs and several unique features of ISG induction kinetics in bat cells. We also characterize induction and antiviral activity of bat RNASEL, which is induced by IFN in bat, but not human cells. These studies lay the foundation for discovery of potentially new antiviral mechanisms in bats, which may spur research into development of therapies to combat viral infection.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2017/01/01" />
<meta name="citation_mjid" content="biorxiv;167999v1" />
<meta name="citation_id" content="167999v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/167999v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/167999v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/167999v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2017/07/24/167999.full.pdf" />
<meta name="citation_doi" content="10.1101/167999" />
<meta name="citation_num_pages" content="26" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="167999" />
<meta name="citation_author" content="Pamela C. De La Cruz-Rivera" />
<meta name="citation_author_institution" content="Department of Microbiology, UT Southwestern Medical Center" />
<meta name="citation_author" content="Mohammed Kanchwala" />
<meta name="citation_author_institution" content="McDermott Center Bioinformatics Core, UT Southwestern Medical Center" />
<meta name="citation_author" content="Hanquan Liang" />
<meta name="citation_author_institution" content="McDermott Center Bioinformatics Core, UT Southwestern Medical Center" />
<meta name="citation_author" content="Ashwani Kumar" />
<meta name="citation_author_institution" content="McDermott Center Bioinformatics Core, UT Southwestern Medical Center" />
<meta name="citation_author" content="Lin-Fa Wang" />
<meta name="citation_author_institution" content="Programme in Emerging Infectious Diseases, Duke-National University of Singapore Medical School" />
<meta name="citation_author" content="Chao Xing" />
<meta name="citation_author_institution" content="McDermott Center Bioinformatics Core, UT Southwestern Medical Center" />
<meta name="citation_author_institution" content="Department of Bioinformatics, UT Southwestern Medical Center" />
<meta name="citation_author_institution" content="Department of Clinical Sciences, UT Southwestern Medical Center" />
<meta name="citation_author" content="John W. Schoggins" />
<meta name="citation_author_institution" content="Department of Microbiology, UT Southwestern Medical Center" />
<meta name="citation_author_email" content="john.schoggins@utsouthwestern.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7944-6800" />
<meta name="citation_reference" content="Chua KB, et al. (2002) Isolation of Nipah virus from Malaysian Island flying-foxes. Microbes Infect 4(2):145-151." />
<meta name="citation_reference" content="Halpin K, Young PL, Field HE, &amp; Mackenzie JS (2000) Isolation of Hendra virus from pteropid bats: a natural reservoir of Hendra virus. J Gen Virol 81(Pt 8):1927-1932." />
<meta name="citation_reference" content="Towner JS, et al. (2007) Marburg virus infection detected in a common African bat. PLoS One 2(8):e764." />
<meta name="citation_reference" content="Li W, et al. (2005) Bats are natural reservoirs of SARS-like coronaviruses. Science 310(5748):676-679." />
<meta name="citation_reference" content="Leroy EM, et al. (2005) Fruit bats as reservoirs of Ebola virus. Nature 438(7068):575-576." />
<meta name="citation_reference" content="Williamson MM, Hooper PT, Selleck PW, Westbury HA, &amp; Slocombe RF (2000) Experimental hendra virus infectionin pregnant guinea-pigs and fruit Bats (Pteropus poliocephalus). J Comp Pathol 122(2-3):201-207." />
<meta name="citation_reference" content="Lau SK, et al. (2010) Ecoepidemiology and complete genome comparison of different strains of severe acute respiratory syndrome-related Rhinolophus bat coronavirus in China reveal bats as a reservoir for acute, self-limiting infection that allows recombination events. J Virol 84(6):2808-2819." />
<meta name="citation_reference" content="Calisher CH, Childs JE, Field HE, Holmes KV, &amp; Schountz T (2006) Bats: important reservoir hosts of emerging viruses. Clin Microbiol Rev 19(3):531-545." />
<meta name="citation_reference" content="Brook CE &amp; Dobson AP (2015) Bats as ‘special’ reservoirs for emerging zoonotic pathogens. Trends Microbiol 23(3):172-180." />
<meta name="citation_reference" content="Olival KJ, et al. (2017) Host and viral traits predict zoonotic spillover from mammals. Nature." />
<meta name="citation_reference" content="Zhang G, et al. (2013) Comparative analysis of bat genomes provides insight into the evolution of flight and immunity. Science 339(6118):456-460." />
<meta name="citation_reference" content="Schoggins JW, et al. (2011) A diverse range of gene products are effectors of the type I interferon antiviral response. Nature 472(7344):481-485." />
<meta name="citation_reference" content="Field H, et al. (2001) The natural history of Hendra and Nipah viruses. Microbes Infect 3(4):307-314." />
<meta name="citation_reference" content="Wang L-F, Mackenzie JS, &amp; Broder CC (2013) Henipaviruses. Fields Virology, eds Knipe DM &amp; Howley PM (Lippincott Williams &amp; Wilkins, Philadelphia), 6th Ed Vol 2, pp 268-313." />
<meta name="citation_reference" content="Cowled C, et al. (2011) Molecular characterisation of Toll-like receptors in the black flying fox Pteropus alecto. Dev Comp Immunol 35(1):7-18." />
<meta name="citation_reference" content="Papenfuss AT, et al. (2012) The immune gene repertoire of an important viral reservoir, the Australian black flying fox. BMC Genomics 13:261." />
<meta name="citation_reference" content="Holzer M, et al. (2016) Differential transcriptional responses to Ebola and Marburg virus infection in bat and human cells. Sci Rep 6:34589." />
<meta name="citation_reference" content="Wu L, et al. (2013) Deep RNA sequencing reveals complex transcriptional landscape of a bat adenovirus. J Virol 87(1):503-511." />
<meta name="citation_reference" content="Wynne JW, et al. (2014) Proteomics informed by transcriptomics reveals Hendra virus sensitizes bat cells to TRAIL-mediated apoptosis. Genome Biol 15(11):532." />
<meta name="citation_reference" content="Glennon NB, Jabado O, Lo MK, &amp; Shaw ML (2015) Transcriptome Profiling of the Virus-Induced Innate Immune Response in Pteropus vampyrus and Its Attenuation by Nipah Virus Interferon Antagonist Functions. J Virol 89(15):7550-7566." />
<meta name="citation_reference" content="Zhou P, et al. (2016) Contraction of the type I IFN locus and unusual constitutive expression of IFN-alpha in bats. Proc Natl Acad Sci U S A 113(10):2696-2701." />
<meta name="citation_reference" content="Janardhana V, et al. (2012) Cloning, expression and antiviral activity of IFNgamma from the Australian fruit bat, Pteropus alecto. Dev Comp Immunol 36(3):610-618." />
<meta name="citation_reference" content="Zhou P, Cowled C, Wang LF, &amp; Baker ML (2013) Bat Mx1 and Oas1, but not Pkr are highly induced by bat interferon and viral infection. Dev Comp Immunol 40(3-4):240-247." />
<meta name="citation_reference" content="Mi H, Poudel S, Muruganujan A, Casagrande JT, &amp; Thomas PD (2016) PANTHER version 10: expanded protein families and functions, and analysis tools. Nucleic Acids Res 44(D1):D336-342." />
<meta name="citation_reference" content="Rusinova I, et al. (2013) Interferome v2.0: an updated database of annotated interferon-regulated genes. Nucleic Acids Res 41(Database issue):D1040-1046." />
<meta name="citation_reference" content="Tai YCaS, Terrence P. (2006) A multivariate empirical Bayes statistic for replicated microarray time course data. The Annals of Statistics 34(5):2387-2412." />
<meta name="citation_reference" content="Silverman RH (2007) Viral encounters with 2′,5′-oligoadenylate synthetase and RNase L during the interferon antiviral response. J Virol 81(23):12720-12729." />
<meta name="citation_reference" content="Zhou A, Molinaro RJ, Malathi K, &amp; Silverman RH (2005) Mapping of the human RNASEL promoter and expression in cancer and normal cells. J Interferon Cytokine Res 25(10):595-603." />
<meta name="citation_reference" content="Mali P, et al. (2013) RNA-guided human genome engineering via Cas9. Science 339(6121):823-826." />
<meta name="citation_reference" content="Wreschner DH, James TC, Silverman RH, &amp; Kerr IM (1981) Ribosomal RNA cleavage, nuclease activation and 2-5A(ppp(A2’p)nA) in interferon-treated cells. Nucleic Acids Res 9(7): 1571-1581." />
<meta name="citation_reference" content="Li Y, et al. (2016) Activation of RNase L is dependent on OAS3 expression during infection with diverse human viruses. Proc Natl Acad Sci U S A 113(8):2241-2246." />
<meta name="citation_reference" content="Mostafavi S, et al. (2016) Parsing the Interferon Transcriptional Network and Its Disease Associations. Cell 164(3):564-578." />
<meta name="citation_reference" content="Malathi K, Dong B, Gale M, Jr., &amp; Silverman RH (2007) Small self-RNA generated by RNase L amplifies antiviral innate immunity. Nature 448(7155):816-819." />
<meta name="citation_reference" content="Sorgeloos F, Jha BK, Silverman RH, &amp; Michiels T (2013) Evasion of antiviral innate immunity by Theiler’s virus L* protein through direct inhibition of RNase L. PLoS Pathog 9(6):e1003474." />
<meta name="citation_reference" content="Martinand C, et al. (1999) RNase L inhibitor is induced during human immunodeficiency virus type 1 infection and down regulates the 2-5A/RNase L pathway in human T cells. J Virol 73(1):290-296." />
<meta name="citation_reference" content="Martinand C, Salehzada T, Silhol M, Lebleu B, &amp; Bisbal C (1998) RNase L inhibitor (RLI) antisense constructions block partially the down regulation of the 2-5A/RNase L pathway in encephalomyocarditis-virus-(EMCV)-infected cells. Eur J Biochem 254(2):248-255." />
<meta name="citation_reference" content="Crameri G, et al. (2009) Establishment, immortalisation and characterisation of pteropid bat cell lines. PLoS One 4(12):e8266." />
<meta name="citation_reference" content="Kim D, et al. (2013) TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions. Genome Biol 14(4):R36." />
<meta name="citation_reference" content="Roberts A, Pimentel H, Trapnell C, &amp; Pachter L (2011) Identification of novel transcripts in annotated genomes using RNA-Seq. Bioinformatics 27(17):2325-2329." />
<meta name="citation_reference" content="Liao Y, Smyth GK, &amp; Shi W (2014) featureCounts: an efficient general purpose program for assigning sequence reads to genomic features. Bioinformatics 30(7):923-930." />
<meta name="citation_reference" content="Robinson MD, McCarthy DJ, &amp; Smyth GK (2010) edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics 26(1):139-140." />
<meta name="citation_reference" content="Law CW, Chen Y, Shi W, &amp; Smyth GK (2014) voom: Precision weights unlock linear model analysis tools for RNA-seq read counts. Genome Biol 15(2):R29." />
<meta name="citation_reference" content="Altschul SF, et al. (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res 25(17):3389-3402." />
<meta name="citation_reference" content="Sanjana NE, Shalem O, &amp; Zhang F (2014) Improved vectors and genome-wide libraries for CRISPR screening. Nat Methods 11(8):783-784." />
<meta name="citation_reference" content="Naito Y, Hino K, Bono H, &amp; Ui-Tei K (2015) CRISPRdirect: software for designing CRISPR/Cas guide RNA with reduced off-target sites. Bioinformatics 31(7):1120-1123." />
<meta name="citation_reference" content="Shalem O, et al. (2014) Genome-scale CRISPR-Cas9 knockout screening in human cells. Science 343(6166):84-87." />
<meta name="twitter:title" content="The IFN response in bat cells consists of canonical and non-canonical ISGs with unique temporal expression kinetics" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Bats are reservoirs for a number of highly pathogenic zoonotic viruses, yet they remain relatively asymptomatic during infection. Whether this viral resistance is due to a unique innate immune system is unknown. An evolutionarily conserved feature of vertebrate antiviral immunity is the interferon (IFN) response, which triggers cellular defenses through interferon-stimulated gene (ISG) expression. While bats encode an intact IFN system, global ISG expression patterns in bat cells are not well characterized. Here, we used RNA-Seq to assess the transcriptional response to IFNα in cells derived from the bat Pteropus alecto (black flying fox). We show induction of more than 100 transcripts, most of which are canonical ISGs observed in other species. Kinetic gene profiling revealed that P. alecto ISGs fall into two unique temporal subclusters with similar early induction kinetics but distinct late-phase declines. In contrast to bat ISGs, human ISGs generally remained elevated for longer periods following IFN treatment, suggesting host-based differences in gene regulatory mechanisms. Notably, we also identified a small group of non-canonical bat ISGs, including an enzymatically active RNASEL that plays a role in controlling viral infection. These studies provide insight into the innate immune response of an important viral reservoir and lay a foundation for studies into the immunological features that may underlie unique virus-host relationship in bats.

Significance Statement Bats are considered unique in their ability to resist disease caused by viruses that are often pathogenic in humans. While the nature of this viral resistance is unknown, genomic data suggest bat innate immune systems may be specialized in controlling these disease-causing viruses. A critical cell intrinsic antiviral defense system in vertebrates is the interferon response, which suppresses viral infection through induction of hundreds of interferon-stimulated genes (ISGs). In this study, we report the repertoire of ISGs and several unique features of ISG induction kinetics in bat cells. We also characterize induction and antiviral activity of bat RNASEL, which is induced by IFN in bat, but not human cells. These studies lay the foundation for discovery of potentially new antiviral mechanisms in bats, which may spur research into development of therapies to combat viral infection." />
<meta name="og-title" property="og:title" content="The IFN response in bat cells consists of canonical and non-canonical ISGs with unique temporal expression kinetics" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/167999v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Bats are reservoirs for a number of highly pathogenic zoonotic viruses, yet they remain relatively asymptomatic during infection. Whether this viral resistance is due to a unique innate immune system is unknown. An evolutionarily conserved feature of vertebrate antiviral immunity is the interferon (IFN) response, which triggers cellular defenses through interferon-stimulated gene (ISG) expression. While bats encode an intact IFN system, global ISG expression patterns in bat cells are not well characterized. Here, we used RNA-Seq to assess the transcriptional response to IFNα in cells derived from the bat Pteropus alecto (black flying fox). We show induction of more than 100 transcripts, most of which are canonical ISGs observed in other species. Kinetic gene profiling revealed that P. alecto ISGs fall into two unique temporal subclusters with similar early induction kinetics but distinct late-phase declines. In contrast to bat ISGs, human ISGs generally remained elevated for longer periods following IFN treatment, suggesting host-based differences in gene regulatory mechanisms. Notably, we also identified a small group of non-canonical bat ISGs, including an enzymatically active RNASEL that plays a role in controlling viral infection. These studies provide insight into the innate immune response of an important viral reservoir and lay a foundation for studies into the immunological features that may underlie unique virus-host relationship in bats.

Significance Statement Bats are considered unique in their ability to resist disease caused by viruses that are often pathogenic in humans. While the nature of this viral resistance is unknown, genomic data suggest bat innate immune systems may be specialized in controlling these disease-causing viruses. A critical cell intrinsic antiviral defense system in vertebrates is the interferon response, which suppresses viral infection through induction of hundreds of interferon-stimulated genes (ISGs). In this study, we report the repertoire of ISGs and several unique features of ISG induction kinetics in bat cells. We also characterize induction and antiviral activity of bat RNASEL, which is induced by IFN in bat, but not human cells. These studies lay the foundation for discovery of potentially new antiviral mechanisms in bats, which may spur research into development of therapies to combat viral infection." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2017-07-24" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/167999v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/167999v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/167999v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/167999v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/50065" />
    <title>The IFN response in bat cells consists of canonical and non-canonical ISGs with unique temporal expression kinetics | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__vup9YXfxWOBOGbthJCUtxdPS6xRY2OQ_vXqbojfxPAk__ziC21uj-cnpEf3LG0kjRMEQHSEtjX5ljVuosklz2DZ4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"U77wU102UaCMGnhnBHR9fqtQte-binaEsIWe9BwdbcI","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"50065","apath":"\/biorxiv\/early\/2017\/07\/24\/167999.atom","pisa":"biorxiv;167999v1","markup":[{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;167999\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;167999\/DC1"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;167999v1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;167999\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;167999\/DC1"}],"processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/167999v1","title":"The IFN response in bat cells consists of canonical and non-canonical ISGs with unique temporal expression kinetics","identifier":"node\/50065"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022aD2wrPWXL0Wu56fqjWP0tXwt3AWL3q4ZsdNXI5Ovw6+v1QAw+RrmxfBdmHDIyH3t45ss1h\\\\\\\/\\\\\\\/S60p2CV0RwRfpiOWIjuWaKccs6dLHENiaxnfffIBDghEu\\\\\\\/eL9Lf4X4bhYqvIRfo59SkgQd0TwUxip9x2Wbkpn4NpkczvfvogO52n7sfe5RoVmKekt4fHR6Bqeyt07nK1PsO49E1SvYwYmkYuE8Sy8Ca5F9eYSTzjPoS+30AnIBCrjk3nHlaynlwVblcASegY+YgKkC5tYj+rWDkblDkcmvmJvK34jukCqVKssjPlucUv868+KtM8Vnys15Syg2KdO2+m2NonD+tkn80Ve9HJHQ62HHbHn5oGvBZiUebtdWTZTuI7HYOd9a6wfg460isMUKOuzBWwvrQEuPvHTAT7wNIO\\\\\\\/hSUDS+XRcR6HBKrzLoa\\\\\\\/xzLrRXruqqggKEFfzfy8Al08wd5EVhJIFGnrHMPlDVn\\\\\\\/YPjPKq4aiuDQ5RuIGW0pQYHyHUvpJKi\\\\\\\/zDKI2aK3RH6URTN34yJCOPIhrVNtpriZm3z3NDGs6bOULc7SzJcYXlIq80zSs2flWhnVO49yfi3Ni5kyMvmczJ0tWbatJEz2xuVv7ijDLUVMWHH98rSBmdjIr1GWfVqwVs2thmY0S9qq0ZtLnge7vi7V77e+US\\\\\\\/vTm9CqWqAKB8\\\\\\\/vpQbU+FBjJrqn1c4OhQi9LqIC02\\\\\\\/\\\\\\\/rSjtThhn+zjlAuvqCCS2RkkkUTnApMZscMVhqdapmujY3WAU0eln0tdkUYqyngN7RcCIFrxsXBFNbNz51fM01ddxTpkDs0R3Z76ovMzCgQ\\\\\\\/Th68CmHyTyZrNCVYYzmYvIdDz3QoisggwBE9McoxirMi3KcU769EqUvOLK+81BzjVx9vKcC\\\\\\\/wQlsrcg0\\\\\\\/P9MtY5RkhpAHrr5dg6GWWgGQ4oof\\\\\\\/deEqhAJzdDqBumhNovKDmOuzI46MYZrFsFcwpp6GIGe2LKidyIRTcry\\\\\\\/Y9NcljmWoirnWjihtBOwWonqst0JjXraQathjRtzkfX4aYTCdgtp8B3RP3PiCl2bWkxKc0yLdYgOo+\\\\\\\/cJgwRjoGsTZ9kTIYayNWqYGfxGdq85OvcEzcacYGgo8Yewr+WMYhh03FMBmG1IqUaFZyWBgNDpPXhVBzkajrVLAAqt+BTNJC12xogBWpRtrsGCmN8l6Uc0obysTuHk3zL\\\\\\\/zDF\\\\\\\/nNwbIqQaY069NuZrcMVvZswoQkdRCTEQ5hKdvJXbpTCP7se+bakUsvq4kVP+SHe+ljmtekz\\\\\\\/h2D9UDAzlRonasxpCojzN52LBWE570pD2DuGQkbdS7z+asIyj0M=\\\u0022,\\\u0022iv\\\u0022:\\\u0022N3AHfTmiXsYQ0aPBjH2W1g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002256ab493f03442d2d4765c932a716efee441f53ba9307cfafb2fd83041bd8d116\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/167999v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-50065 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/167999v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-GWpzRKj58tDLQYLLU3YikLFa-kVdTtzD4OnVkn2qvxU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="50065" id="node50065" data-pisa="biorxiv;167999v1" data-pisa-master="biorxiv;167999" data-apath="/biorxiv/early/2017/07/24/167999.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">The IFN response in bat cells consists of canonical and non-canonical ISGs with unique temporal expression kinetics</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Pamela C.</span> <span class="nlm-surname">De La Cruz-Rivera</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Mohammed</span> <span class="nlm-surname">Kanchwala</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Hanquan</span> <span class="nlm-surname">Liang</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Ashwani</span> <span class="nlm-surname">Kumar</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Lin-Fa</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Chao</span> <span class="nlm-surname">Xing</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-7944-6800" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">John W.</span> <span class="nlm-surname">Schoggins</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/167999 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node50065" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Pamela C. De La Cruz-Rivera </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Microbiology, UT Southwestern Medical Center</span>, Dallas, TX 75390</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pamela%2BC.%2BDe%2BLa%2BCruz-Rivera%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=De%20La%20Cruz-Rivera%20PC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APamela%2BC.%2BDe%2BLa%2BCruz-Rivera%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mohammed Kanchwala </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>McDermott Center Bioinformatics Core, UT Southwestern Medical Center</span>, Dallas, TX 75390</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mohammed%2BKanchwala%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kanchwala%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMohammed%2BKanchwala%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Hanquan Liang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>McDermott Center Bioinformatics Core, UT Southwestern Medical Center</span>, Dallas, TX 75390</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hanquan%2BLiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHanquan%2BLiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ashwani Kumar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>McDermott Center Bioinformatics Core, UT Southwestern Medical Center</span>, Dallas, TX 75390</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ashwani%2BKumar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kumar%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAshwani%2BKumar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Lin-Fa Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Programme in Emerging Infectious Diseases, Duke-National University of Singapore Medical School</span>, Singapore 169857</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lin-Fa%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALin-Fa%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Chao Xing </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>McDermott Center Bioinformatics Core, UT Southwestern Medical Center</span>, Dallas, TX 75390</div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Department of Bioinformatics, UT Southwestern Medical Center</span>, Dallas, TX 75390</div><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Department of Clinical Sciences, UT Southwestern Medical Center</span>, Dallas, TX 75390</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chao%2BXing%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xing%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChao%2BXing%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">John W. Schoggins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Microbiology, UT Southwestern Medical Center</span>, Dallas, TX 75390</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BW.%2BSchoggins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schoggins%20JW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BW.%2BSchoggins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7944-6800" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for John W. Schoggins</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:john.schoggins@utsouthwestern.edu" class="" data-icon-position="" data-hide-link-title="0">john.schoggins@utsouthwestern.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/167999v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:50065" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:50065/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/167999v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:50065" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:50065/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/167999v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:50065" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:50065/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/167999v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:50065" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:50065/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/167999v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:50065" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:50065/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/167999v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:50065" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:50065/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Bats are reservoirs for a number of highly pathogenic zoonotic viruses, yet they remain relatively asymptomatic during infection. Whether this viral resistance is due to a unique innate immune system is unknown. An evolutionarily conserved feature of vertebrate antiviral immunity is the interferon (IFN) response, which triggers cellular defenses through interferon-stimulated gene (ISG) expression. While bats encode an intact IFN system, global ISG expression patterns in bat cells are not well characterized. Here, we used RNA-Seq to assess the transcriptional response to IFNα in cells derived from the bat <em>Pteropus alecto</em> (black flying fox). We show induction of more than 100 transcripts, most of which are canonical ISGs observed in other species. Kinetic gene profiling revealed that <em>P. alecto</em> ISGs fall into two unique temporal subclusters with similar early induction kinetics but distinct late-phase declines. In contrast to bat ISGs, human ISGs generally remained elevated for longer periods following IFN treatment, suggesting host-based differences in gene regulatory mechanisms. Notably, we also identified a small group of non-canonical bat ISGs, including an enzymatically active RNASEL that plays a role in controlling viral infection. These studies provide insight into the innate immune response of an important viral reservoir and lay a foundation for studies into the immunological features that may underlie unique virus-host relationship in bats.</p><div id="sec-1" class="subsection"><p id="p-3"><strong>Significance Statement</strong> Bats are considered unique in their ability to resist disease caused by viruses that are often pathogenic in humans. While the nature of this viral resistance is unknown, genomic data suggest bat innate immune systems may be specialized in controlling these disease-causing viruses. A critical cell intrinsic antiviral defense system in vertebrates is the interferon response, which suppresses viral infection through induction of hundreds of interferon-stimulated genes (ISGs). In this study, we report the repertoire of ISGs and several unique features of ISG induction kinetics in bat cells. We also characterize induction and antiviral activity of bat RNASEL, which is induced by IFN in bat, but not human cells. These studies lay the foundation for discovery of potentially new antiviral mechanisms in bats, which may spur research into development of therapies to combat viral infection.</p></div></div><div class="section" id="sec-2"><h2 class="">Introduction</h2><p id="p-4">Bats are recognized as important viral reservoirs due to their ability to harbor a number of highly pathogenic viruses, including Nipah virus (NiV) (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>), Hendra virus (HeV) (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>), Marburg virus (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>), SARS coronavirus (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>), and Ebolavirus (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>) without suffering from overt disease symptoms (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>-<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>). A recent study identified bats as hosts for a greater proportion of zoonotic viruses than all other mammalian orders tested, with the highest viral richness found in Flavi-, Bunya- and Rhabdoviruses (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>). While the mechanisms underlying disease resistance are not known, it has been speculated that bats possess unique immune components that confer innate antiviral protection (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>). In vertebrates, one of the first lines of defense against viral pathogens is the interferon (IFN) response. Upon viral infection, pattern-recognition receptors (PRRs) sense viral components and initiate a signaling cascade that results in the secretion of IFNs. These IFNs bind their cognate receptors to activate the JAK-STAT signaling pathway, leading to the transcriptional induction of hundreds of interferon-stimulated genes (ISGs), many of which have antiviral activity (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>).</p><p id="p-5"><em>Pteropus alecto</em> (also known as the black flying fox or black fruit bat) is an asymptomatic natural reservoir for the highly lethal henipaviruses (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>). Studies of the recently sequenced <em>P. alecto</em> genome revealed that genes for key components of antiviral immunity are conserved in bats, such as pathogen sensors (including toll-like receptors (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>), RIG-I-like helicases, and NOD-like receptors), IFNs and their receptors, and ISGs (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>). Previous efforts to study bat-virus interactions have mainly focused on the host response to viral infection (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>-<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>), but global transcriptional responses to type I IFN remain uncharacterized.</p><p id="p-6">Since <em>P. alecto</em> harbors pathogenic human viruses and has an annotated genome, we sought to characterize the IFN-induced transcriptional response in this species. Gene expression analyses revealed that bat cells induce a pool of common ISGs. However, they also induced a small number of apparent novel ISGs, including 2-5A-dependent endoribonuclease (RNASEL). Kinetic analyses revealed that bat ISGs can be categorized into distinct groups depending on their temporal gene expression patterns. Additionally, maintenance of ISG expression over time differed between bat and human cells, suggesting distinct mechanisms of gene regulation.</p></div><div class="section" id="sec-3"><h2 class="">Results</h2><div id="sec-4" class="subsection"><h3><em>Pteropus alecto</em>-derived cell lines respond to exogenous type I interferon</h3><p id="p-7">Previous studies have shown that exogenous IFNα and IFNγ treatment of cells from <em>P. alecto</em> can suppress <em>Pteropine orthoreovirus</em> Pulau virus (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>) and Hendra virus (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>), respectively. We treated immortalized <em>P. alecto</em> kidney-derived (PaKi) cells for 24h with universal IFNα, which is designed for activity across multiple species. We then infected the cells with two model GFP-expressing reporter viruses: a negative-sense single-stranded RNA rhabdovirus, vesicular stomatitis virus (VSV), and a positive-sense single stranded RNA flavivirus, yellow fever virus (YFV). We observed a dose-dependent inhibition of both viruses with IFN treatment (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig 1A, B</a>). VSV infection was maximally inhibited by only 50%, whereas YFV infection was suppressed completely at the highest IFN dose. In addition, we confirmed IFN-mediated dose-dependent and time-dependent induction of the canonical ISG <em>MX1</em> (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig 1C, D</a>) (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>). These data confirm that PaKi cells are capable of mounting an antiviral response and highlight virus-specific sensitivities to IFN in this cellular background.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/07/24/167999/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bat-derived cells respond to Type I IFN. (A) PaKi cells were treated with increasing doses of IFN&#x3B1; for 24h and then infected with VSV-GFP at an MOI of 1. The percentage of infected cells was quantified using flow cytometry and normalized to a mock-treated control. Data are represented as mean &#xB1; SD for three independent experiments. (B) Same as (A), using YFV-17D Venus. (C) PaKi cells were treated with increasing doses of IFN&#x3B1; and RNA was harvested at 8h. MX1 mRNA levels were quantified using qRT-PCR and normalized to RPS11. Data are represented as mean &#xB1; SD for three independent experiments. (D) PaKi cells were treated with IFN&#x3B1; (50U/mL) and RNA was harvested at several time points. MX1 mRNA levels were measured using qRT-PCR and normalized to RPS11. Data are represented as mean &#xB1; SD for three independent experiments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1948760313" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Bat-derived cells respond to Type I IFN.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; PaKi cells were treated with increasing doses of IFN&#x3B1; for 24h and then infected with VSV-GFP at an MOI of 1. The percentage of infected cells was quantified using flow cytometry and normalized to a mock-treated control. Data are represented as mean &#xB1; SD for three independent experiments. &lt;strong&gt;(B)&lt;/strong&gt; Same as (A), using YFV-17D Venus. &lt;strong&gt;(C)&lt;/strong&gt; PaKi cells were treated with increasing doses of IFN&#x3B1; and RNA was harvested at 8h. MX1 mRNA levels were quantified using qRT-PCR and normalized to RPS11. Data are represented as mean &#xB1; SD for three independent experiments. &lt;strong&gt;(D)&lt;/strong&gt; PaKi cells were treated with IFN&#x3B1; (50U/mL) and RNA was harvested at several time points. MX1 mRNA levels were measured using qRT-PCR and normalized to RPS11. Data are represented as mean &#xB1; SD for three independent experiments.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/07/24/167999/F1.medium.gif" width="440" height="325"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2017/07/24/167999/F1.medium.gif" width="440" height="325"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/07/24/167999/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/07/24/167999/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">Bat-derived cells respond to Type I IFN.</span><p id="p-8" class="first-child"><strong>(A)</strong> PaKi cells were treated with increasing doses of IFNα for 24h and then infected with VSV-GFP at an MOI of 1. The percentage of infected cells was quantified using flow cytometry and normalized to a mock-treated control. Data are represented as mean ± SD for three independent experiments. <strong>(B)</strong> Same as (A), using YFV-17D Venus. <strong>(C)</strong> PaKi cells were treated with increasing doses of IFNα and RNA was harvested at 8h. MX1 mRNA levels were quantified using qRT-PCR and normalized to RPS11. Data are represented as mean ± SD for three independent experiments. <strong>(D)</strong> PaKi cells were treated with IFNα (50U/mL) and RNA was harvested at several time points. MX1 mRNA levels were measured using qRT-PCR and normalized to RPS11. Data are represented as mean ± SD for three independent experiments.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>IFN induces a classical ISG signature in PaKi cells</h3><p id="p-9">We next used total RNA sequencing (RNA-Seq) to profile the global transcriptional response of PaKi cells treated with IFNα over time (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig 2A</a>). Transcriptome assembly analysis across all experimental conditions returned approximately 30,000 genes, of which 11,559 met a minimal read count threshold (mean log<sub>2</sub>CPM≥0). Differential gene expression analysis revealed that IFN induced 93 genes at 4h, 104 at 8h, 103 at 12h, and 103 at 24h (fold change (FC)≥1.5, FDR≤0.05) (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig 2B</a>). There were no downregulated genes at 4h, 2 at 8h, 105 at 12h, and 279 at 24h. However, statistical significance of upregulated genes was more robust than the statistical significance of the downregulated genes. Overall, IFN treatment of PaKi cells produced a positive gene induction signature.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/07/24/167999/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="PaKi transcriptome response to IFN. (A) Experimental schematic. (B) Volcano plots for all time points. Differentially-expressed genes are quantified on the top left. Grey bars indicate log2FC&#x2265;1.5 (vertical) or FDR&#x2264;0.05 (horizontal). Data is a result of three independent experiments (C) Heatmap including genes that meet the cutoffs mean log2CPM&#x2265;0, log2FC&#x2265;2, FDR&#x2264;0.05 for at least 2 time points. XLOC genes represent unannotated genes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1948760313" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;PaKi transcriptome response to IFN.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; Experimental schematic. &lt;strong&gt;(B)&lt;/strong&gt; Volcano plots for all time points. Differentially-expressed genes are quantified on the top left. Grey bars indicate log&lt;sub&gt;2&lt;/sub&gt;FC&#x2265;1.5 (vertical) or FDR&#x2264;0.05 (horizontal). Data is a result of three independent experiments &lt;strong&gt;(C)&lt;/strong&gt; Heatmap including genes that meet the cutoffs mean log&lt;sub&gt;2&lt;/sub&gt;CPM&#x2265;0, log&lt;sub&gt;2&lt;/sub&gt;FC&#x2265;2, FDR&#x2264;0.05 for at least 2 time points. XLOC genes represent unannotated genes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/07/24/167999/F2.medium.gif" width="440" height="356"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2017/07/24/167999/F2.medium.gif" width="440" height="356"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/07/24/167999/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/07/24/167999/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">PaKi transcriptome response to IFN.</span><p id="p-10" class="first-child"><strong>(A)</strong> Experimental schematic. <strong>(B)</strong> Volcano plots for all time points. Differentially-expressed genes are quantified on the top left. Grey bars indicate log<sub>2</sub>FC≥1.5 (vertical) or FDR≤0.05 (horizontal). Data is a result of three independent experiments <strong>(C)</strong> Heatmap including genes that meet the cutoffs mean log<sub>2</sub>CPM≥0, log<sub>2</sub>FC≥2, FDR≤0.05 for at least 2 time points. XLOC genes represent unannotated genes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">Next, heat maps were generated to assess individual gene induction, using a cutoff of FDR≤0.05 and FC≥4 for at least 2 time points (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig 2C</a>). Many genes in this list have known roles in innate immunity, including well-known ISGs <em>(IFIT1, MX1, OAS2, RSAD2</em>/viperin, <em>USP18</em>), members of the JAK-STAT signaling cascade (<em>STAT1, STAT2, IRF9</em>), pattern recognition receptors (<em>DDX58</em>/RIG-I, <em>IFIH1</em>/MDA5, <em>ZBP1</em>) and transcription factors (<em>ETV7, IRF7, SP110</em>) (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>). Notably, we detected induction of <em>GVIN1</em> (interferon-induced very large GTPase), which is predicted to encode a protein in bats but is annotated as a pseudogene in humans. This list also included transcripts predicted to encode an endogenous retrovirus <em>(ERVK25)</em> and several transcripts predicted to be long non-coding RNAs.</p><p id="p-12">Differentially-expressed genes were cross-referenced to the INTERFEROME v2.0 database (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>) to determine if they had previously been reported as IFN-induced genes. At the 4h and 8h time points, more than 80% of the genes in our data set overlapped with INTERFEROME v2.0. Since the INTEFEROME database consists predominantly of human and mouse data sets, this result suggests that antiviral responses in bat cells include a conserved repertoire of IFN-inducible genes commonly found in other mammalian species.</p></div><div id="sec-6" class="subsection"><h3>Differential temporal regulation of <em>P. alecto</em> ISGs</h3><p id="p-13">We next used a clustering algorithm to group genes in the RNA-Seq data set based on induction kinetics, without imposing fold change or FDR cutoffs (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>). This analysis revealed that genes were organized into eight subclusters (SC) based on changes in expression levels throughout the IFN time course (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig 3A</a>). Genes in SC1 and SC2 increased in expression after 8h. Genes in SC3 and SC4 were induced by 4h and peaked at 4-8h, with genes in SC3 returning close to baseline by 12h and SC4 genes remaining elevated for at least 24h. These two SCs were highly-enriched for genes found in the INTERFEROME v2.0 database. Genes in SC5 increased slightly and peaked at 8h, returning to baseline levels by 12h. Levels of SC6 genes either increased or decreased by 4h, followed by decreased levels at 12-24h. SC7 gene expression levels decreased sharply by 8h, followed by a partial recovery by 12h and further decrease in expression by 24h. Finally, genes in SC8 exhibited a sustained decrease in expression levels starting at 8-12h. These data demonstrate that interferon induces distinct subsets of genes which are characterized by differing temporal expression patterns.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/07/24/167999/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Temporal clustering analysis. (A) Temporal subclusters. The number of genes in each subcluster is indicated below the title. Blue lines represent mean log2(fold change) over time for a single gene. Black lines represent median of all genes in the subcluster. (B) RNA-seq validation using Nanostring. RNA was isolated from PaKi cells and gene expression over time was compared between RNA-seq data (left) and Nanostring data (right). Data presented is the mean of three independent experiments. HK = housekeeping genes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1948760313" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Temporal clustering analysis.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; Temporal subclusters. The number of genes in each subcluster is indicated below the title. Blue lines represent mean log&lt;sub&gt;2&lt;/sub&gt;(fold change) over time for a single gene. Black lines represent median of all genes in the subcluster. &lt;strong&gt;(B)&lt;/strong&gt; RNA-seq validation using Nanostring. RNA was isolated from PaKi cells and gene expression over time was compared between RNA-seq data (left) and Nanostring data (right). Data presented is the mean of three independent experiments. HK = housekeeping genes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/07/24/167999/F3.medium.gif" width="388" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2017/07/24/167999/F3.medium.gif" width="388" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/07/24/167999/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/07/24/167999/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">Temporal clustering analysis.</span><p id="p-14" class="first-child"><strong>(A)</strong> Temporal subclusters. The number of genes in each subcluster is indicated below the title. Blue lines represent mean log<sub>2</sub>(fold change) over time for a single gene. Black lines represent median of all genes in the subcluster. <strong>(B)</strong> RNA-seq validation using Nanostring. RNA was isolated from PaKi cells and gene expression over time was compared between RNA-seq data (left) and Nanostring data (right). Data presented is the mean of three independent experiments. HK = housekeeping genes.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>Orthogonal validation of RNA-Seq data</h3><p id="p-15">To validate our RNA-seq results, we used Nanostring nCounter technology, which uses colorimetrically-barcoded DNA probes for direct detection of mRNA without the use of nucleic acid amplification. A customized nCounter codeset was designed containing 50 genes from several temporal SCs, with a focus on the ISG-rich SC3 and SC4 (Fig S2). Temporal expression profiles using Nanostring were generally similar to the RNA-Seq data (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig 3B</a>). SC1 and SC2 contained genes with increased expression levels at the later time points. SC3 and SC4 had genes with peak expression levels at 8h, with genes in SC4 exhibiting expression levels that remained elevated at 24h. Genes in SC8 decreased in expression over time, with the lowest expression levels observed at 24h.</p></div><div id="sec-8" class="subsection"><h3>Human vs Bat Temporal ISG Regulation</h3><p id="p-16">We next used Nanostring to compare temporal regulation of ISGs between bat and human cell lines. We first compared gene expression between PaKi and HEK293 as both cell lines are kidney-derived, but HEK293 cells responded poorly to type I IFN (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig S1</a>). After screening for robust IFN responses in other human cell lines, we chose human A549 cells for comparative studies (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig 4A</a>). A striking difference was observed between expression profiles of SC1 and SC2 when comparing PaKi and A549 cells. Of the 5 selected genes in SC1 and SC2, none were significantly upregulated in IFN-treated A549 cells. Similarly, the decreased expression levels observed for bat genes in SC8 were not observed in A549 cells.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/07/24/167999/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="ISG expression over time in bat and human cells. (A) Nanostring gene expression for PaKi (left) and A549 cells (right). PaKi data is the same shown in Fig 3B, with additional time points. Data presented is the mean of three independent experiments. HK = housekeeping genes. (B) Expression over time for PaKi and A549 genes in SC3 and SC4. Each line represents the median log2(FC) for all genes in a SC. (C) Time at peak induction for PaKi and A549 genes in SC3 and SC4. (D) Percent genes induced to levels &#x2265;50% of maximum induction at specified time points in SC3 and SC4. (E) Percent genes that recovered to levels &#x2264;50% of maximum induction at specified time points in SC3 and SC4. (F) Baseline mRNA counts for PaKi and A549 genes in SC3 and SC4. Data are represented as mean + SD for three independent experiments. Darker symbols represent mean counts for each subcluster. * = p-value " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1948760313" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;ISG expression over time in bat and human cells.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; Nanostring gene expression for PaKi (left) and A549 cells (right). PaKi data is the same shown in Fig 3B, with additional time points. Data presented is the mean of three independent experiments. HK = housekeeping genes. &lt;strong&gt;(B)&lt;/strong&gt; Expression over time for PaKi and A549 genes in SC3 and SC4. Each line represents the median log&lt;sub&gt;2&lt;/sub&gt;(FC) for all genes in a SC. &lt;strong&gt;(C)&lt;/strong&gt; Time at peak induction for PaKi and A549 genes in SC3 and SC4. &lt;strong&gt;(D)&lt;/strong&gt; Percent genes induced to levels &#x2265;50% of maximum induction at specified time points in SC3 and SC4. &lt;strong&gt;(E)&lt;/strong&gt; Percent genes that recovered to levels &#x2264;50% of maximum induction at specified time points in SC3 and SC4. &lt;strong&gt;(F)&lt;/strong&gt; Baseline mRNA counts for PaKi and A549 genes in SC3 and SC4. Data are represented as mean + SD for three independent experiments. Darker symbols represent mean counts for each subcluster. * = p-value &lt;0.05 using Student&#x2019;s t-test. &lt;strong&gt;(G)&lt;/strong&gt; Maximum mRNA counts for PaKi and A549 genes in SC3 and SC4. Data are represented as mean + SD for three independent experiments. Darker symbols represent mean counts for each subcluster. * = p-value &lt;0.05 using Student&#x2019;s t-test.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/07/24/167999/F4.medium.gif" width="403" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2017/07/24/167999/F4.medium.gif" width="403" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/07/24/167999/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/07/24/167999/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">ISG expression over time in bat and human cells.</span><p id="p-17" class="first-child"><strong>(A)</strong> Nanostring gene expression for PaKi (left) and A549 cells (right). PaKi data is the same shown in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig 3B</a>, with additional time points. Data presented is the mean of three independent experiments. HK = housekeeping genes. <strong>(B)</strong> Expression over time for PaKi and A549 genes in SC3 and SC4. Each line represents the median log<sub>2</sub>(FC) for all genes in a SC. <strong>(C)</strong> Time at peak induction for PaKi and A549 genes in SC3 and SC4. <strong>(D)</strong> Percent genes induced to levels ≥50% of maximum induction at specified time points in SC3 and SC4. <strong>(E)</strong> Percent genes that recovered to levels ≤50% of maximum induction at specified time points in SC3 and SC4. <strong>(F)</strong> Baseline mRNA counts for PaKi and A549 genes in SC3 and SC4. Data are represented as mean + SD for three independent experiments. Darker symbols represent mean counts for each subcluster. * = p-value &lt;0.05 using Student’s t-test. <strong>(G)</strong> Maximum mRNA counts for PaKi and A549 genes in SC3 and SC4. Data are represented as mean + SD for three independent experiments. Darker symbols represent mean counts for each subcluster. * = p-value &lt;0.05 using Student’s t-test.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">For further analysis, we chose to focus on previously-reported ISGs, particularly those found in SC3 and SC4. When comparing changes in gene expression in SC3, the median fold-change of all genes in SC3 followed similar trends between bat and human cell lines (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig 4B</a>). In SC4, however, genes from A549 cells exhibited higher fold induction and remained elevated at later time points when compared to PaKi cells.</p><p id="p-19">We next determined the time point at which we observed maximum gene induction (t<sub>max</sub>). For both SC3 and SC4, approximately 80% of IFN-induced PaKi genes peaked at 8h (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig 4C</a>). However, A549 genes had a bimodal pattern within both SCs, with most genes peaking at 4-8h and a smaller subset peaking at 12h or later.</p><p id="p-20">To identify potential differences in induction kinetics, we calculated the percentage of genes that were induced to at least 50% of maximum expression for each time point. Greater than 80% of PaKi genes in both SCs were induced by 4h (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig 4D</a>). A549 genes in SC3 behaved similarly, although a small subset of genes was induced by 2h. However, approximately 40% of A549 genes in SC4 were induced by 2h, indicating faster induction of this subset of genes. Genes in this list include <em>HERC5, IFI6, IFIH1, MX1, NLRC5, OAS2, OASL</em>, and <em>PARP12</em>. Notably, PaKi cells express higher baseline levels of ISGs such that by full induction at 8h, PaKi cells express similar or greater mRNA counts compared to A549 cells, despite faster induction in A549 cells (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig 4D</a>, <a id="xref-fig-4-6" class="xref-fig" href="#F4">Fig 4G</a>, and <a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig S1</a>).</p><p id="p-21">Next, we calculated the percentage of genes that had recovered to below 50% of maximum expression for each time point. In both SCs, recovery occurred earlier in PaKi cells, with most genes recovering by 16h (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Fig 4E</a>). In contrast, the expression levels for most A549 genes remained elevated throughout the time course. Together, these data suggest that the regulatory mechanisms governing IFN-mediated gene induction and maintenance of gene expression differs in each cell type, particularly with genes in SC4.</p><p id="p-22">It was recently reported that <em>P. alecto</em> tissues have constitutive and ubiquitous expression of IFNα, suggesting that bat cells may be primed to inhibit viral infection due to constitutive expression of certain ISGs (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>). Indeed, we observed overall higher ISG mRNA levels in unstimulated PaKi cells, particularly the SC4 genes (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Fig 4F</a>). In addition, we found that over half of SC4 PaKi ISGs were induced to higher maximum counts than corresponding A549 ISGs (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Fig 4G</a>).</p><p id="p-23">Bat cells express multiple non-canonical ISGs, including an active RNASEL. Our gene expression profiling revealed several genes not previously reported to be ISGs. These included <em>EMC2, FILIP1, IL17RC, OTOGL, SLC10A2</em>, and <em>SLC24A1</em> (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig 2A</a>). In addition, we observed IFN-mediated induction of <em>RNASEL</em>, which encodes a 2’-5’-oligoadenylate synthetase-dependent ribonuclease. This protein exerts its antiviral effect by degrading viral RNA in response to 2’-5’-linked oligoadenylates, which are generated by the IFN-inducible oligoadenylate synthase (OAS) family of enzymes upon stimulation by dsRNA in the cytosol (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>). In humans, RNASEL is a constitutively-expressed latent enzyme and is not IFN-inducible (<a id="xref-fig-4-10" class="xref-fig" href="#F4">Fig 4A</a>, <a id="xref-fig-6-3" class="xref-fig" href="#F6">Fig S1</a>) (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>). Interestingly, we observed a dose-dependent induction of <em>RNASEL</em> in IFN-treated PaKi cells (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig 5A</a>). Of note, similar mRNA expression levels of <em>RNASEL</em> were observed in unstimulated human and bat cell lines (<a id="xref-fig-4-11" class="xref-fig" href="#F4">Fig 4F</a> and <a id="xref-fig-6-4" class="xref-fig" href="#F6">Fig S1</a>). In addition, we observed IFN-mediated <em>RNASEL</em> induction in brain-derived (PaBr) and lung-derived (PaLu) <em>P. alecto</em> cell lines (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig 5B</a>), suggesting IFN-mediated induction of RNASEL is not cell type-specific.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/07/24/167999/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="RNASEL is IFN-induced in P. alecto. (A) PaKi cells were treated with increasing doses of IFN&#x3B1; and RNA was harvested at 8h. RNASEL mRNA levels were measured using qRT-PCR and normalized to untreated control using RPS11 as a housekeeping control. Data are represented as mean &#xB1; SD for two independent experiments. (B) RNASEL induction in P. alecto brain, lung, and kidney cells. Cells were treated with IFN&#x3B1; (100U/mL) and RNA was harvested at 8h. RNASEL levels were quantified using qRT-PCR and normalized to untreated control using RPS11 as housekeeping control. Data are represented as mean &#xB1; SD for three independent experiments. (C) PaKi cells were treated with IFN&#x3B1; (100U/mL) or mock-treated for 24h, followed by transfection with poly(I:C) (100ng/mL) or mock-transfected for 4h and RNA integrity was monitored using a bionalyzer. 100 ng total RNA was run per lane. Data is representative of two independent experiments. (D) WT or RNASEL KO bulk PaKi cells were infected with YFV-17D Venus for 24h and viral infection was quantified using flow cytometry. Data are represented as mean &#xB1; SD for three independent experiments. * = p-value &lt; 0.05 using Student&#x2019;s t-test. (E) WT or RNASEL KO bulk PaKi cells were treated with increasing doses of IFN&#x3B1; for 24h, then infected with YFV-17D Venus at an MOI of 0.5 for 24h. Viral infection was quantified using flow cytometry. Data are represented as mean &#xB1; SD for three independent experiments. * = p-value &lt; 0.05 using Student&#x2019;s t-test." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1948760313" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;RNASEL is IFN-induced in &lt;em&gt;P. alecto&lt;/em&gt;.&lt;/span&gt; &lt;strong&gt;(A)&lt;/strong&gt; PaKi cells were treated with increasing doses of IFN&#x3B1; and RNA was harvested at 8h. RNASEL mRNA levels were measured using qRT-PCR and normalized to untreated control using RPS11 as a housekeeping control. Data are represented as mean &#xB1; SD for two independent experiments. &lt;strong&gt;(B)&lt;/strong&gt; RNASEL induction in &lt;em&gt;P. alecto&lt;/em&gt; brain, lung, and kidney cells. Cells were treated with IFN&#x3B1; (100U/mL) and RNA was harvested at 8h. RNASEL levels were quantified using qRT-PCR and normalized to untreated control using RPS11 as housekeeping control. Data are represented as mean &#xB1; SD for three independent experiments. &lt;strong&gt;(C)&lt;/strong&gt; PaKi cells were treated with IFN&#x3B1; (100U/mL) or mock-treated for 24h, followed by transfection with poly(I:C) (100ng/mL) or mock-transfected for 4h and RNA integrity was monitored using a bionalyzer. 100 ng total RNA was run per lane. Data is representative of two independent experiments. &lt;strong&gt;(D)&lt;/strong&gt; WT or RNASEL KO bulk PaKi cells were infected with YFV-17D Venus for 24h and viral infection was quantified using flow cytometry. Data are represented as mean &#xB1; SD for three independent experiments. * = p-value &lt; 0.05 using Student&#x2019;s t-test. &lt;strong&gt;(E)&lt;/strong&gt; WT or RNASEL KO bulk PaKi cells were treated with increasing doses of IFN&#x3B1; for 24h, then infected with YFV-17D Venus at an MOI of 0.5 for 24h. Viral infection was quantified using flow cytometry. Data are represented as mean &#xB1; SD for three independent experiments. * = p-value &lt; 0.05 using Student&#x2019;s t-test.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/07/24/167999/F5.medium.gif" width="440" height="229"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2017/07/24/167999/F5.medium.gif" width="440" height="229"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/07/24/167999/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/07/24/167999/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <span class="caption-title">RNASEL is IFN-induced in <em>P. alecto</em>.</span><p id="p-24" class="first-child"><strong>(A)</strong> PaKi cells were treated with increasing doses of IFNα and RNA was harvested at 8h. RNASEL mRNA levels were measured using qRT-PCR and normalized to untreated control using RPS11 as a housekeeping control. Data are represented as mean ± SD for two independent experiments. <strong>(B)</strong> RNASEL induction in <em>P. alecto</em> brain, lung, and kidney cells. Cells were treated with IFNα (100U/mL) and RNA was harvested at 8h. RNASEL levels were quantified using qRT-PCR and normalized to untreated control using RPS11 as housekeeping control. Data are represented as mean ± SD for three independent experiments. <strong>(C)</strong> PaKi cells were treated with IFNα (100U/mL) or mock-treated for 24h, followed by transfection with poly(I:C) (100ng/mL) or mock-transfected for 4h and RNA integrity was monitored using a bionalyzer. 100 ng total RNA was run per lane. Data is representative of two independent experiments. <strong>(D)</strong> WT or RNASEL KO bulk PaKi cells were infected with YFV-17D Venus for 24h and viral infection was quantified using flow cytometry. Data are represented as mean ± SD for three independent experiments. * = p-value &lt; 0.05 using Student’s t-test. <strong>(E)</strong> WT or RNASEL KO bulk PaKi cells were treated with increasing doses of IFNα for 24h, then infected with YFV-17D Venus at an MOI of 0.5 for 24h. Viral infection was quantified using flow cytometry. Data are represented as mean ± SD for three independent experiments. * = p-value &lt; 0.05 using Student’s t-test.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">Next, we constructed RNASEL-deficient PaKi cells using CRISPR (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>). Due to lack of a bat-specific RNASEL antibody, we were not able to monitor RNASEL protein levels. However, we detected modifications to the <em>RNASEL</em> gene in the PaKi knock-out bulk population (KO bulk) as compared to the parental wild-type (WT) population (Table S1). To test if RNASEL protein was functional, we activated RNASEL with poly (I:C) transfection after IFN or mock treatment and monitored RNA integrity (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig 5C</a>) (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>). We observed rRNA degradation when cells were treated with poly(I:C), but not with IFN alone. Treating cells with IFN for 24h to induce RNASEL expression before poly (I:C) transfection resulted in increased RNA degradation and accumulation of smaller products, suggesting increased RNASEL activity. The RNA degradation observed in the WT cells was reduced in the bulk KO cells. Together, these data indicate that bat RNASEL is a functional ribonuclease that, unlike its human ortholog, is IFN-inducible.</p><p id="p-26">To test if the presence of RNASEL is important in the context of viral infection, we infected both WT and bulk KO cells with YFV17D-Venus and quantified infectivity after one viral life cycle (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Fig 5D</a>). RNASEL KO cells were more permissive to infection at all doses used, suggesting that RNASEL is important for suppression of initial viral infection. To test if RNASEL induction played a role in the protective effect of the IFN response, we treated PaKi WT and KO bulk cells with increasing doses of IFNα for 24h, then infected with YFV17D-Venus (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Fig 5E</a>). Consistent with previous results, KO bulk cells were more permissive to infection than WT cells. In addition, KO bulk cells were resistant to the protective activity of IFN. IFNα (100U/mL) pre-treatment resulted in 80% reduction of infection in WT cells, but only 50% reduction in bulk KO cells. Together, these data suggest RNASEL contributes to inhibition of viral infection in bat cells, particularly in the context of the IFN response.</p></div></div><div class="section" id="sec-9"><h2 class="">Discussion</h2><p id="p-27">This study aimed to identify IFN-stimulated transcripts in the viral reservoir <em>P. alecto</em>. Transcriptional analysis revealed over 100 genes induced in response to IFNα. Most of these genes have been previously described as ISGs, suggesting strong evolutionary conservation of the ISG pool, as would be predicted by previous genomic studies of immune genes in <em>P. alecto</em> (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>).</p><p id="p-28">We have provided a framework of <em>P. alecto</em> ISG induction, organized by early, mid, and late responses to type I IFN. Our data set reveal a previously unappreciated subtlety in the temporal induction and regulation of ISG expression. A recent study of IFN-inducible gene kinetics in mouse cells reported a profile that resembles a combination of our SC3 and SC4 data sets (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>). Our study further refines this kinetic analysis by delineating two separate ISG pools based on unique temporal induction profiles. While both SCs are characterized by similar peak mRNA levels and a subsequent decline by 12-24 hours, SC4 contained some genes that remained elevated. These genes may offer residual antiviral protection, even when IFN signaling has returned to basal levels. In addition, many genes in SC4 have both higher baseline and higher maximal induction levels in bat cells compared to human cells, which could contribute to species-specific differences in susceptibility to viral infection. Additional studies are needed to explore the mechanisms underlying unique expression of certain ISGs in bats cells. Compared to human cells, bat cells have a more rapid decline in ISG levels, suggesting tightly-regulated expression kinetics. The reason for this strict transcriptional regulation remains unclear, but such a mechanism may exist to prevent excessive inflammation in a highly metabolically active host (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">11</a>).</p><p id="p-29">We also identified several previously unrecognized, or non-canonical ISGs, including RNASEL. The OAS/RNASEL pathway consists of OAS proteins that recognize viral dsRNA and subsequently catalyze the formation of short oligoadenylates that act as second messengers to activate the constitutively-expressed latent enzyme RNASEL, which degrades viral genetic material. Due to cleavage of both viral and cellular RNA, RNASEL activation can also lead to apoptosis of infected cells. In addition, the short RNA fragments created by RNASEL can potentiate the IFN response by activating the cytosolic RNA sensor RIG-I (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>). The induction of RNASEL in response to IFN in bats may provide an additional layer of antiviral protection. Indeed, knockout of RNASEL increased viral susceptibility of <em>P. alecto-derived</em> cells. Although RNASEL induction itself does not result in significant nuclease activity, stimulation with poly (I:C) is sufficient to cause degradation of total RNA in the cell. Unlike in humans, where only the upstream OAS proteins are IFN-induced, bat cells co-induce both components of the OAS/RNASEL pathway, likely creating a more rapid and potent effect that would inhibit viral replication before extensive viral spread could occur. RNASEL induction may also be a counterdefense against viruses that subvert the pathway through through various immune evasion mechanisms, such as degradation of RNASEL by L* protein of murine Theiler’s virus(<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>), or increased expression of an RNASEL inhibitor by HIV(<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>) and EMCV(<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>).</p><p id="p-30">Additional studies will be needed to determine whether these non-canonical bat ISGs contribute to unique viral resistance in bats. It is also possible that canonical bat ISGs have evolved unique effector functions that contribute to reduced viral susceptibility. Uncovering mechanisms of bat ISGs will provide insight into the innate immune responses of an important viral reservoir and may inform research and development of antiviral therapies.</p></div><div class="section" id="sec-10"><h2 class="">Methods &amp; Materials</h2><div id="sec-11" class="subsection"><h3>Cell lines</h3><p id="p-31"><em>P. alecto</em>-derived PaBr (brain), PaLu (lung), and PaKi (kidney) immortalized cell lines (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>) were grown at 37°C and 5% CO<sub>2</sub> and passaged in DMEM/F-12 media (GIBCO) supplemented with 10% FBS. Human A549 lung adenocarcinoma and HEK293 cells were grown at 37°C and 5% CO<sub>2</sub> and passaged in DMEM media (GIBCO) supplemented with 10% FBS and 1X non-essential amino acids (NEAA) (GIBCO #11140076).</p></div><div id="sec-12" class="subsection"><h3>Viruses</h3><p id="p-32">YFV-Venus (derived from YF17D-5C25Venus2AUbi) stocks were generated by electroporation of in vitro-transcribed RNA into <em>STAT1</em><sup>-/-</sup> fibroblasts as previously described (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>). VSV-GFP (provided by Jack Rose) was generated by passage in BHK-J cells. For both viruses, virus-containing supernatant was centrifuged to remove cellular debris and stored at -80C until use.</p></div><div id="sec-13" class="subsection"><h3>Viral infection</h3><p id="p-33">Cells were seeded into 24-well plates at a density of 1x10<sup>5</sup> cells per well. Viral stocks were diluted into DMEM/F-12 media supplemented with 1% FBS to make infection media. Media was aspirated and replaced with 200 μL of infection media. Infections were carried out at 37°C for 1h, then 800 μL DMEM/F-12 media supplemented with 10% FBS was added back to each well. After approximately one viral life cycle, cells were harvested for flow cytometry.</p></div><div id="sec-14" class="subsection"><h3>Flow cytometry</h3><p id="p-34">Cells were detached using Accumax, fixed in 1% PFA for 10 min at room temperature, and pelleted by centrifugation at 800xg. Fixed cell pellets were resuspended in 200 μL FACS buffer (1X PBS supplemented with 3% FBS). Samples were run on a Stratedigm S1000 instrument using CellCapTure software and gated based on GFP fluorescence. Data analysis was done using FlowJo software (v9.7.6).</p></div><div id="sec-15" class="subsection"><h3>Interferon treatment and RNA isolation</h3><p id="p-35">Cells were seeded into 6-well plates at 3x10<sup>5</sup> cells per well. The following day, cells were treated with 2 mL of DMEM F-12 media supplemented with 10% FBS and 50 units/mL of Universal Type I IFN Alpha (PBL Assay Science Cat. #11200). The cells were harvested by aspirating the media, washing twice with 2 mL of sterile 1X PBS, then lysing with 350 μL RLT buffer from the RNeasy Mini kit (Qiagen). The cell lysate was stored at -80C until RNA isolation was completed using the RNeasy kit following the manufacturer’s protocol.</p></div><div id="sec-16" class="subsection"><h3>RNA-Seq</h3><p id="p-36">Total RNA samples for each time point were prepared in three independent experiments as described above. The Agilent 2100 Bioanalyzer was used to determine RNA quality; only samples with a RIN Score &gt;9 were used. A Qubit fluorometer was used to determine RNA concentration. One μg of DNAse-treated total RNA was prepared with the TruSeq Stranded Total RNA LT Sample Prep Kit (Illumina). rRNA-depleted total RNA was fragmented and used for strand-specific cDNA synthesis. cDNA was then A-tailed and ligated with indexed adapters. Samples were then PCR amplified, purified with AmpureXP beads, and re-validated on the Agilent 2100 Bioanalyzer and Qubit.</p><p id="p-37">Normalized and pooled samples were run on the Illumina HiSeq 2500 using SBS v3 reagents. Paired-end 100 bp read length Fastq files were checked for quality using fastqc (<a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc">http://www.bioinformatics.babraham.ac.uk/projects/fastqc</a>) and fastq_screen (<a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqscreen/">http://www.bioinformatics.babraham.ac.uk/projects/fastqscreen/</a>) and were quality trimmed using fastq-mcf (<a href="https://github.com/ExpressionAnalysis/ea-utils/blob/wiki/FastqMcf.md">https://github.com/ExpressionAnalysis/ea-utils/blob/wiki/FastqMcf.md</a>). Trimmed fastq files were mapped to Pteropus alecto (black flying fox) assembly ASM32557v1 (<a href="ftp://ftp.ncbi.nih.gov/genomes/Pteropusalecto">ftp://ftp.ncbi.nih.gov/genomes/Pteropusalecto</a>) using TopHat (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>). Duplicates were marked using picard-tools (<a href="https://broadinstitute.github.io/picard/">https://broadinstitute.github.io/picard/</a>). Reference annotation based transcript assembly was done using cufflinks (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>), and read counts were generated using featureCounts (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>). Pairwise differential expression analysis was performed using edgeR (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>), and longitudinal analysis was performed using time course (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a>) after data transformation by voom (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>). Differentially-expressed unannotated genes were manually annotated using a nucleotide BLAST (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>) search for homologous genes (indicated with an asterisk).</p></div><div id="sec-17" class="subsection"><h3>Heatmaps</h3><p id="p-38">Heatmaps were constructed using GENE-E and Morpheus, both available at <a href="https://software.broadinstitute.org">https://software.broadinstitute.org</a>.</p></div><div id="sec-18" class="subsection"><h3>Nanostring analysis</h3><p id="p-39">Total RNA was isolated from IFN-treated samples as described above. 100 ng of total RNA in 5 μL was used as input in each reaction for NanoString assay. A customized panel containing both <em>P. alecto</em> and <em>H. sapiens</em> genes (Schoggins_1_C4066, NanoString Technologies, Seattle, WA, USA) was used to measure the expression of 100 genes in the RNA samples by following the NanoString nCounter XT Codeset Gene Expression Assay manufacturer’s protocol. Following hybridization, transcripts were quantitated using the nCounter Digital Analyzer.</p></div><div id="sec-19" class="subsection"><h3>qRT-PCR</h3><p id="p-40">Total RNA was prepared as described above. Reactions were prepared with the QuantiFast SYBR Green RT-PCR kit (Qiagen #204154), using 50 ng total RNA per reaction. Samples were run on the Applied Biosystems 7500 Fast Real-Time PCR System using 7500 Software v2.0.6. Primers used to amplify RNASEL are 5’-CCACCCTGGGGAAAATGTGA-3’ and 5’-GGAGGATCCTGCTTGCTTGT-3’. Primers used to amplify housekeeping gene RPS11 are 5’-ATCCGCCGAGACTATCTCCA-3’ and 5’-GGACATCTCTGAAGCAGGGT-3’.</p></div><div id="sec-20" class="subsection"><h3>DNA constructs and plasmid propagation</h3><p id="p-41">lentiCRISPR v2 was a gift from Feng Zhang (Addgene plasmid # 52961). RNASEL targeting guides were generated by cloning annealed, complementary 20-bp oligos with <em>Esp3l</em>-compatible overhangs (5’-caccgAGACCCACACCCTCCAGCAG-3’ and 5’-aaacCTGCTGGAGGGTGTGGGTCTc-3’) targeting the <em>P. alecto RNASEL</em> gene into the lentiCRISPRv2 backbone as described in (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>). CRISPR guide oligos were designed using CRISPRdirect (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>). Proper assembly was confirmed using Sanger sequencing.</p></div><div id="sec-21" class="subsection"><h3>Lentiviral pseudoparticles</h3><p id="p-42">Lentiviral pseudoparticles were made as described in (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>), with some modifications. Briefly, 2x10<sup>6</sup> HEK293T cells were seeded on a poly-lysine coated 10 cm plate. The following day, media was changed to 7.5 mL DMEM with 3%FBS and 1x NEAA. Cells were co-transfected with 5 μg lentiCRIPSRv2, 2.5 μg pCMV-VSVg and 3.5 μg pGag-pol. 4h post-transfection, media was changed to 7.5 mL fresh DMEM with 3% FBS and 1x NEAA. Supernatant was collected 48h post-transfection, filtered through 0.45 μm to remove debris, and stored at -80°C until use.</p></div><div id="sec-22" class="subsection"><h3><em>RNASEL</em> KO bulk PaKi cell lines</h3><p id="p-43">3x10<sup>5</sup> PaKi cells were seeded on 6-well plates. The following day, media was changed to DMEM/F-12 supplemented with 3% FBS, 4 μg/mL polybrene and 20 mM HEPES. LentiCRISPRv2 lentiviral pseudoparticles were added and cells were spinoculated at 800xg for 45 min at 37°C. Media was changed to DMEM/F-12 with 10% FBS immediately following spinoculation. 48h after transduction, cells were pooled into a 10-cm dish and selected in DMEM/F-12 with 10% FBS and 5 μg/mL puromycin.</p></div><div id="sec-23" class="subsection"><h3>Genomic characterization of PaKi <em>RNASEL</em> KO bulk cell lines</h3><p id="p-44">Genomic DNA was isolated from wild-type or RNASEL KO bulk populations using the DNeasy Blood and Tissue Kit (Qiagen). The region falnking the CRISPR-targeted sequence was amplified via PCR using primers (5′-ATGGAGACCAAGAGCCACAACA-3′) and (5’-CGTCCTCGTCCTGGAAATTGA-3’). PCR products were gel-purified using the QIAquick Gel Extraction Kit (Qiagen) and subsequently used in TOPO cloning reactions using the TOPO TA Cloning Kit (Thermo Fisher). Several colonies were selected for each cell background and colony PCR was used to amplify the CRISPR-targeted region. Samples were then analyzed using Sanger sequencing.</p></div><div id="sec-24" class="subsection"><h3>rRNA degradation assay</h3><p id="p-45">3x10<sup>5</sup> PaKi cells were seeded on 6-well plates. The following day, universal IFN (100 U/mL) was added to the media. After 24h, cells were transfected with 100 ng/mL poly(I:C) in Optimem using Lipofectamine 3000. After 4h, RNA was harvested using the RNeasy Mini Kit (Qiagen) and RNA integrity was measured on a Total RNA Nano chip using an Agilent 2100 Bioanalyzer.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-46">We would like to thank Jeanine Baisch and Cynthia Silverman from the Genomics Core at Baylor Research Institute for assistance with Nanostring samples, and the McDermott Center Sequencing Core and Bioinformatics Core for sequencing and analysis. This work was supported in part by NIH grant AI117922 (J.W.S.), the UT Southwestern Endowed Scholars program (J.W.S.), the UT Southwestern High Impact / High Risk Grant Program (J.W.S.), the William F. and Grace H. Kirkpatrick Award (P.C.D.) and the NRF-CRP grant NRF2012NRF-CRP001-056 (L-F.W). C.X. was partially supported by NIH grant UL1TR001105.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.1" data-doi="10.1016/S1286-4579(01)01522-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chua</span>  <span class="cit-name-given-names">KB</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2002</span>) <span class="cit-article-title">Isolation of Nipah virus from Malaysian Island flying-foxes</span>. <abbr class="cit-jnl-abbrev">Microbes Infect</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">145</span>-<span class="cit-lpage">151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobes%2Band%2Binfection%2B%252F%2BInstitut%2BPasteur%26rft.stitle%253DMicrobes%2BInfect%26rft.aulast%253DChua%26rft.auinit1%253DK.%2BB.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253D145%26rft.epage%253D151%26rft.atitle%253DIsolation%2Bof%2BNipah%2Bvirus%2Bfrom%2BMalaysian%2BIsland%2Bflying-foxes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1286-4579%252801%252901522-2%26rft_id%253Dinfo%253Apmid%252F11880045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1286-4579(01)01522-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11880045&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F07%2F24%2F167999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174578300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.2" data-doi="10.1099/vir.0.17031-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Halpin</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">PL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Field</span>  <span class="cit-name-given-names">HE</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Mackenzie</span>  <span class="cit-name-given-names">JS</span></span> (<span class="cit-pub-date">2000</span>) <span class="cit-article-title">Isolation of Hendra virus from pteropid bats: a natural reservoir of Hendra virus</span>. <abbr class="cit-jnl-abbrev">J Gen Virol</abbr> <span class="cit-vol">81</span>(Pt <span class="cit-issue">8</span>):<span class="cit-fpage">1927</span>-<span class="cit-lpage">1932</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DHalpin%26rft.auinit1%253DK.%26rft.volume%253D81%26rft.issue%253D8%26rft.spage%253D1927%26rft.epage%253D1932%26rft.atitle%253DIsolation%2Bof%2BHendra%2Bvirus%2Bfrom%2Bpteropid%2Bbats%253A%2Ba%2Bnatural%2Breservoir%2Bof%2BHendra%2Bvirus%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.17031-0%26rft_id%253Dinfo%253Apmid%252F10900029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.17031-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10900029&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F07%2F24%2F167999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000088509900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.3" data-doi="10.1371/journal.pone.0000764"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Towner</span>  <span class="cit-name-given-names">JS</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">Marburg virus infection detected in a common African bat</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">2</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">e764</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DTowner%26rft.auinit1%253DJ.%2BS.%26rft.volume%253D2%26rft.issue%253D8%26rft.spage%253De764%26rft.epage%253De764%26rft.atitle%253DMarburg%2Bvirus%2Binfection%2Bdetected%2Bin%2Ba%2Bcommon%2BAfrican%2Bbat.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000764%26rft_id%253Dinfo%253Apmid%252F17712412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0000764&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17712412&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F07%2F24%2F167999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.4" data-doi="10.1126/science.1118391"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">Bats are natural reservoirs of SARS-like coronaviruses</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">310</span>(<span class="cit-issue">5748</span>):<span class="cit-fpage">676</span>-<span class="cit-lpage">679</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1118391%26rft_id%253Dinfo%253Apmid%252F16195424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTAvNTc0OC82NzYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNy8wNy8yNC8xNjc5OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.5" data-doi="10.1038/438575a"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Leroy</span>  <span class="cit-name-given-names">EM</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">Fruit bats as reservoirs of Ebola virus</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">438</span>(<span class="cit-issue">7068</span>):<span class="cit-fpage">575</span>-<span class="cit-lpage">576</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLeroy%26rft.auinit1%253DE.%2BM.%26rft.volume%253D438%26rft.issue%253D7068%26rft.spage%253D575%26rft.epage%253D576%26rft.atitle%253DFruit%2Bbats%2Bas%2Breservoirs%2Bof%2BEbola%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F438575a%26rft_id%253Dinfo%253Apmid%252F16319873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/438575a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16319873&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F07%2F24%2F167999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233593100030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.6" data-doi="10.1053/jcpa.1999.0364"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">MM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hooper</span>  <span class="cit-name-given-names">PT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Selleck</span>  <span class="cit-name-given-names">PW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Westbury</span>  <span class="cit-name-given-names">HA</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Slocombe</span>  <span class="cit-name-given-names">RF</span></span> (<span class="cit-pub-date">2000</span>) <span class="cit-article-title">Experimental hendra virus infectionin pregnant guinea-pigs and fruit Bats (Pteropus poliocephalus)</span>. <abbr class="cit-jnl-abbrev">J Comp Pathol</abbr> <span class="cit-vol">122</span>(<span class="cit-issue">2-3</span>):<span class="cit-fpage">201</span>-<span class="cit-lpage">207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomparative%2Bpathology%26rft.stitle%253DJ%2BComp%2BPathol%26rft.aulast%253DWilliamson%26rft.auinit1%253DM.%2BM.%26rft.volume%253D122%26rft.issue%253D2-3%26rft.spage%253D201%26rft.epage%253D207%26rft.atitle%253DExperimental%2Bhendra%2Bvirus%2Binfectionin%2Bpregnant%2Bguinea-pigs%2Band%2Bfruit%2BBats%2B%2528Pteropus%2Bpoliocephalus%2529.%26rft_id%253Dinfo%253Adoi%252F10.1053%252Fjcpa.1999.0364%26rft_id%253Dinfo%253Apmid%252F10684689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1053/jcpa.1999.0364&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10684689&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F07%2F24%2F167999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085511500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">7.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-167999v1.7" data-doi="10.1128/JVI.02219-09"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">SK</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Ecoepidemiology and complete genome comparison of different strains of severe acute respiratory syndrome-related Rhinolophus bat coronavirus in China reveal bats as a reservoir for acute, self-limiting infection that allows recombination events</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">84</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">2808</span>-<span class="cit-lpage">2819</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02219-09%26rft_id%253Dinfo%253Apmid%252F20071579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg0LzYvMjgwOCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA3LzI0LzE2Nzk5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">8.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-167999v1.8" data-doi="10.1128/CMR.00017-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Calisher</span>  <span class="cit-name-given-names">CH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Childs</span>  <span class="cit-name-given-names">JE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Field</span>  <span class="cit-name-given-names">HE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">KV</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Schountz</span>  <span class="cit-name-given-names">T</span></span> (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">Bats: important reservoir hosts of emerging viruses</span>. <abbr class="cit-jnl-abbrev">Clin Microbiol Rev</abbr> <span class="cit-vol">19</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">531</span>-<span class="cit-lpage">545</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2BMicrobiology%2BReviews%26rft.stitle%253DClin.%2BMicrobiol.%2BRev.%26rft.aulast%253DCalisher%26rft.auinit1%253DC.%2BH.%26rft.volume%253D19%26rft.issue%253D3%26rft.spage%253D531%26rft.epage%253D545%26rft.atitle%253DBats%253A%2BImportant%2BReservoir%2BHosts%2Bof%2BEmerging%2BViruses%26rft_id%253Dinfo%253Adoi%252F10.1128%252FCMR.00017-06%26rft_id%253Dinfo%253Apmid%252F16847084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiY21yIjtzOjU6InJlc2lkIjtzOjg6IjE5LzMvNTMxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDcvMjQvMTY3OTk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.9" data-doi="10.1016/j.tim.2014.12.004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brook</span>  <span class="cit-name-given-names">CE</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Dobson</span>  <span class="cit-name-given-names">AP</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Bats as ‘special’ reservoirs for emerging zoonotic pathogens</span>. <abbr class="cit-jnl-abbrev">Trends Microbiol</abbr> <span class="cit-vol">23</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">172</span>-<span class="cit-lpage">180</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BMicrobiol%26rft.volume%253D23%26rft.spage%253D172%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2014.12.004%26rft_id%253Dinfo%253Apmid%252F25572882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2014.12.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25572882&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F07%2F24%2F167999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Olival</span>  <span class="cit-name-given-names">KJ</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Host and viral traits predict zoonotic spillover from mammals</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.11" data-doi="10.1126/science.1230835"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Comparative analysis of bat genomes provides insight into the evolution of flight and immunity</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">339</span>(<span class="cit-issue">6118</span>):<span class="cit-fpage">456</span>-<span class="cit-lpage">460</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1230835%26rft_id%253Dinfo%253Apmid%252F23258410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzkvNjExOC80NTYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNy8wNy8yNC8xNjc5OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.12" data-doi="10.1038/nature09907"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schoggins</span>  <span class="cit-name-given-names">JW</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">A diverse range of gene products are effectors of the type I interferon antiviral response</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">472</span>(<span class="cit-issue">7344</span>):<span class="cit-fpage">481</span>-<span class="cit-lpage">485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSchoggins%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D472%26rft.issue%253D7344%26rft.spage%253D481%26rft.epage%253D485%26rft.atitle%253DA%2Bdiverse%2Brange%2Bof%2Bgene%2Bproducts%2Bare%2Beffectors%2Bof%2Bthe%2Btype%2BI%2Binterferon%2Bantiviral%2Bresponse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09907%26rft_id%253Dinfo%253Apmid%252F21478870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09907&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478870&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F07%2F24%2F167999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289949600036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.13" data-doi="10.1016/S1286-4579(01)01384-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Field</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2001</span>) <span class="cit-article-title">The natural history of Hendra and Nipah viruses</span>. <abbr class="cit-jnl-abbrev">Microbes Infect</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">307</span>-<span class="cit-lpage">314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobes%2Band%2Binfection%2B%252F%2BInstitut%2BPasteur%26rft.stitle%253DMicrobes%2BInfect%26rft.aulast%253DField%26rft.auinit1%253DH.%26rft.volume%253D3%26rft.issue%253D4%26rft.spage%253D307%26rft.epage%253D314%26rft.atitle%253DThe%2Bnatural%2Bhistory%2Bof%2BHendra%2Band%2BNipah%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1286-4579%252801%252901384-3%26rft_id%253Dinfo%253Apmid%252F11334748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1286-4579(01)01384-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11334748&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F07%2F24%2F167999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168143300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-167999v1.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L-F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mackenzie</span>  <span class="cit-name-given-names">JS</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Broder</span>  <span class="cit-name-given-names">CC</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-chapter-title">Henipaviruses</span>. <span class="cit-source">Fields Virology</span>, eds <span class="cit-auth"><span class="cit-name-surname">Knipe</span>  <span class="cit-name-given-names">DM</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Howley</span>  <span class="cit-name-given-names">PM</span></span> (<span class="cit-publ-name">Lippincott Williams</span> &amp; <span class="cit-publ-loc">Wilkins, Philadelphia</span>), <span class="cit-edition">6th Ed</span> Vol <span class="cit-vol">2</span>, pp <span class="cit-fpage">268</span>-<span class="cit-lpage">313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFields%2BVirology%26rft.volume%253D2%26rft.spage%253D268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.15" data-doi="10.1016/j.dci.2010.07.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cowled</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Molecular characterisation of Toll-like receptors in the black flying fox Pteropus alecto</span>. <abbr class="cit-jnl-abbrev">Dev Comp Immunol</abbr> <span class="cit-vol">35</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">7</span>-<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Band%2Bcomparative%2Bimmunology%26rft.stitle%253DDev%2BComp%2BImmunol%26rft.aulast%253DCowled%26rft.auinit1%253DC.%26rft.volume%253D35%26rft.issue%253D1%26rft.spage%253D7%26rft.epage%253D18%26rft.atitle%253DMolecular%2Bcharacterisation%2Bof%2BToll-like%2Breceptors%2Bin%2Bthe%2Bblack%2Bflying%2Bfox%2BPteropus%2Balecto.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.dci.2010.07.006%26rft_id%253Dinfo%253Apmid%252F20692287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.dci.2010.07.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20692287&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F07%2F24%2F167999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.16" data-doi="10.1186/1471-2164-13-261"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Papenfuss</span>  <span class="cit-name-given-names">AT</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">The immune gene repertoire of an important viral reservoir, the Australian black flying fox</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>:<span class="cit-fpage">261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DPapenfuss%26rft.auinit1%253DA.%2BT.%26rft.volume%253D13%26rft.spage%253D261%26rft.epage%253D261%26rft.atitle%253DThe%2Bimmune%2Bgene%2Brepertoire%2Bof%2Ban%2Bimportant%2Bviral%2Breservoir%252C%2Bthe%2BAustralian%2Bblack%2Bflying%2Bfox.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-261%26rft_id%253Dinfo%253Apmid%252F22716473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-261&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22716473&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F07%2F24%2F167999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.17" data-doi="10.1038/srep34589"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Holzer</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Differential transcriptional responses to Ebola and Marburg virus infection in bat and human cells</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-vol">6</span>:<span class="cit-fpage">34589</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D6%26rft.spage%253D34589%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep34589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep34589&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">18.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-167999v1.18" data-doi="10.1128/JVI.02332-12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Deep RNA sequencing reveals complex transcriptional landscape of a bat adenovirus</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">87</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">503</span>-<span class="cit-lpage">511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02332-12%26rft_id%253Dinfo%253Apmid%252F23097437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6Ijg3LzEvNTAzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDcvMjQvMTY3OTk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">19.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-167999v1.19" data-doi="10.1186/s13059-014-0532-x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wynne</span>  <span class="cit-name-given-names">JW</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Proteomics informed by transcriptomics reveals Hendra virus sensitizes bat cells to TRAIL-mediated apoptosis</span>. <abbr class="cit-jnl-abbrev">Genome Biol</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">11</span>):<span class="cit-fpage">532</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D15%26rft.spage%253D532%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0532-x%26rft_id%253Dinfo%253Apmid%252F25398248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0532-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25398248&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F07%2F24%2F167999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.20" data-doi="10.1128/JVI.00302-15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Glennon</span>  <span class="cit-name-given-names">NB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jabado</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lo</span>  <span class="cit-name-given-names">MK</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">ML</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Transcriptome Profiling of the Virus-Induced Innate Immune Response in Pteropus vampyrus and Its Attenuation by Nipah Virus Interferon Antagonist Functions</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">89</span>(<span class="cit-issue">15</span>):<span class="cit-fpage">7550</span>-<span class="cit-lpage">7566</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00302-15%26rft_id%253Dinfo%253Apmid%252F25972557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4OS8xNS83NTUwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDcvMjQvMTY3OTk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.21" data-doi="10.1073/pnas.1518240113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Contraction of the type I IFN locus and unusual constitutive expression of IFN-alpha in bats</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">113</span>(<span class="cit-issue">10</span>):<span class="cit-fpage">2696</span>-<span class="cit-lpage">2701</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1518240113%26rft_id%253Dinfo%253Apmid%252F26903655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEzLzEwLzI2OTYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNy8wNy8yNC8xNjc5OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.22" data-doi="10.1016/j.dci.2011.11.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Janardhana</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Cloning, expression and antiviral activity of IFNgamma from the Australian fruit bat, Pteropus alecto</span>. <abbr class="cit-jnl-abbrev">Dev Comp Immunol</abbr> <span class="cit-vol">36</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">610</span>-<span class="cit-lpage">618</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Band%2Bcomparative%2Bimmunology%26rft.stitle%253DDev%2BComp%2BImmunol%26rft.aulast%253DJanardhana%26rft.auinit1%253DV.%26rft.volume%253D36%26rft.issue%253D3%26rft.spage%253D610%26rft.epage%253D618%26rft.atitle%253DCloning%252C%2Bexpression%2Band%2Bantiviral%2Bactivity%2Bof%2BIFN%25CE%25B3%2Bfrom%2Bthe%2BAustralian%2Bfruit%2Bbat%252C%2BPteropus%2Balecto.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.dci.2011.11.001%26rft_id%253Dinfo%253Apmid%252F22093696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.dci.2011.11.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22093696&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F07%2F24%2F167999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.23" data-doi="10.1016/j.dci.2013.03.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cowled</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">LF</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">ML</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Bat Mx1 and Oas1, but not Pkr are highly induced by bat interferon and viral infection</span>. <abbr class="cit-jnl-abbrev">Dev Comp Immunol</abbr> <span class="cit-vol">40</span>(<span class="cit-issue">3-4</span>):<span class="cit-fpage">240</span>-<span class="cit-lpage">247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDev%2BComp%2BImmunol%26rft.volume%253D40%26rft.spage%253D240%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.dci.2013.03.006%26rft_id%253Dinfo%253Apmid%252F23541614%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.dci.2013.03.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23541614&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F07%2F24%2F167999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.24" data-doi="10.1093/nar/gkv1194"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mi</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poudel</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muruganujan</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Casagrande</span>  <span class="cit-name-given-names">JT</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">PD</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">PANTHER version 10: expanded protein families and functions, and analysis tools</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">44</span>(<span class="cit-issue">D1</span>):<span class="cit-fpage">D336</span>-<span class="cit-lpage">342</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1194%26rft_id%253Dinfo%253Apmid%252F26578592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1194&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26578592&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F07%2F24%2F167999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.25" data-doi="10.1093/nar/gks1215"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rusinova</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Interferome v2.0: an updated database of annotated interferon-regulated genes</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">41</span>(Database issue):<span class="cit-fpage">D1040</span>-<span class="cit-lpage">1046</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks1215%26rft_id%253Dinfo%253Apmid%252F23203888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks1215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23203888&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F07%2F24%2F167999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312893300148&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tai</span>  <span class="cit-name-given-names">YCaS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Terrence</span>  <span class="cit-name-given-names">P</span></span>. (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">A multivariate empirical Bayes statistic for replicated microarray time course data</span>. <abbr class="cit-jnl-abbrev">The Annals of Statistics</abbr> <span class="cit-vol">34</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">2387</span>-<span class="cit-lpage">2412</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAnnals%2Bof%2BStatistics%26rft.volume%253D34%26rft.spage%253D2387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.27" data-doi="10.1128/JVI.01471-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Silverman</span>  <span class="cit-name-given-names">RH</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">Viral encounters with 2′,5′-oligoadenylate synthetase and RNase L during the interferon antiviral response</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">81</span>(<span class="cit-issue">23</span>):<span class="cit-fpage">12720</span>-<span class="cit-lpage">12729</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01471-07%26rft_id%253Dinfo%253Apmid%252F17804500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4MS8yMy8xMjcyMCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA3LzI0LzE2Nzk5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.28" data-doi="10.1089/jir.2005.25.595"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Molinaro</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Malathi</span>  <span class="cit-name-given-names">K</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Silverman</span>  <span class="cit-name-given-names">RH</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">Mapping of the human RNASEL promoter and expression in cancer and normal cells</span>. <abbr class="cit-jnl-abbrev">J Interferon Cytokine Res</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">10</span>):<span class="cit-fpage">595</span>-<span class="cit-lpage">603</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Binterferon%2B%2526%2Bcytokine%2Bresearch%2B%253A%2B%2Bthe%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BSociety%2Bfor%2BInterferon%2Band%2BCytokine%2BResearch%26rft.stitle%253DJ%2BInterferon%2BCytokine%2BRes%26rft.aulast%253DReitblat%26rft.auinit1%253DT.%26rft.volume%253D25%26rft.issue%253D10%26rft.spage%253D595%26rft.epage%253D603%26rft.atitle%253DMapping%2Bof%2Bthe%2Bhuman%2BRNASEL%2Bpromoter%2Band%2Bexpression%2Bin%2Bcancer%2Band%2Bnormal%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fjir.2005.25.595%26rft_id%253Dinfo%253Apmid%252F16241858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/jir.2005.25.595&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16241858&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F07%2F24%2F167999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232784700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.29" data-doi="10.1126/science.1232033"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mali</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">RNA-guided human genome engineering via Cas9</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">339</span>(<span class="cit-issue">6121</span>):<span class="cit-fpage">823</span>-<span class="cit-lpage">826</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1232033%26rft_id%253Dinfo%253Apmid%252F23287722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzkvNjEyMS84MjMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNy8wNy8yNC8xNjc5OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.30" data-doi="10.1093/nar/9.7.1571"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wreschner</span>  <span class="cit-name-given-names">DH</span></span>, <span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">TC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Silverman</span>  <span class="cit-name-given-names">RH</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Kerr</span>  <span class="cit-name-given-names">IM</span></span> (<span class="cit-pub-date">1981</span>) <span class="cit-article-title">Ribosomal RNA cleavage, nuclease activation and 2-5A(ppp(A2’p)nA) in interferon-treated cells</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">1571</span>-<span class="cit-lpage">1581</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DWreschner%26rft.auinit1%253DD.H.%26rft.volume%253D9%26rft.issue%253D7%26rft.spage%253D1571%26rft.epage%253D1581%26rft.atitle%253DRibosomal%2BRNA%2Bcleavage%252C%2Bnuclease%2Bactivation%2Band%2B2-5A%2B%2528ppp%2528A2%2527p%2529nA%2529%2Bin%2Binterferon-treated%2Bcells%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F9.7.1571%26rft_id%253Dinfo%253Apmid%252F6164990%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/9.7.1571&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6164990&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F07%2F24%2F167999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1981LL09200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.31" data-doi="10.1073/pnas.1519657113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Activation of RNase L is dependent on OAS3 expression during infection with diverse human viruses</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">113</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">2241</span>-<span class="cit-lpage">2246</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1519657113%26rft_id%253Dinfo%253Apmid%252F26858407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTEzLzgvMjI0MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA3LzI0LzE2Nzk5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.32" data-doi="10.1016/j.cell.2015.12.032"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mostafavi</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Parsing the Interferon Transcriptional Network and Its Disease Associations</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">164</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">564</span>-<span class="cit-lpage">578</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D164%26rft.spage%253D564%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2015.12.032%26rft_id%253Dinfo%253Apmid%252F26824662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2015.12.032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26824662&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F07%2F24%2F167999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.33" data-doi="10.1038/nature06042"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Malathi</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gale</span>  <span class="cit-name-given-names">M</span>,   <span class="cit-name-suffix">Jr.</span>.</span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Silverman</span>  <span class="cit-name-given-names">RH</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">Small self-RNA generated by RNase L amplifies antiviral innate immunity</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">448</span>(<span class="cit-issue">7155</span>):<span class="cit-fpage">816</span>-<span class="cit-lpage">819</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMalathi%26rft.auinit1%253DK.%26rft.volume%253D448%26rft.issue%253D7155%26rft.spage%253D816%26rft.epage%253D819%26rft.atitle%253DSmall%2Bself-RNA%2Bgenerated%2Bby%2BRNase%2BL%2Bamplifies%2Bantiviral%2Binnate%2Bimmunity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06042%26rft_id%253Dinfo%253Apmid%252F17653195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17653195&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F07%2F24%2F167999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248754200042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.34" data-doi="10.1371/journal.ppat.1003474"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sorgeloos</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jha</span>  <span class="cit-name-given-names">BK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Silverman</span>  <span class="cit-name-given-names">RH</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Michiels</span>  <span class="cit-name-given-names">T</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Evasion of antiviral innate immunity by Theiler’s virus L* protein through direct inhibition of RNase L</span>. <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">e1003474</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D9%26rft.spage%253De1003474%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003474%26rft_id%253Dinfo%253Apmid%252F23825954%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003474&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23825954&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F07%2F24%2F167999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martinand</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">1999</span>) <span class="cit-article-title">RNase L inhibitor is induced during human immunodeficiency virus type 1 infection and down regulates the 2-5A/RNase L pathway in human T cells</span>. <abbr class="cit-jnl-abbrev">J Virol</abbr> <span class="cit-vol">73</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">290</span>-<span class="cit-lpage">296</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DMartinand%26rft.auinit1%253DC.%26rft.volume%253D73%26rft.issue%253D1%26rft.spage%253D290%26rft.epage%253D296%26rft.atitle%253DRNase%2BL%2BInhibitor%2BIs%2BInduced%2Bduring%2BHuman%2BImmunodeficiency%2BVirus%2BType%2B1%25A0Infection%2Band%2BDown%2BRegulates%2Bthe%2B2-5A%252FRNase%2BL%2BPathway%2Bin%2BHuman%2BT%2BCells%26rft_id%253Dinfo%253Apmid%252F9847332%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjczLzEvMjkwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDcvMjQvMTY3OTk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.36" data-doi="10.1046/j.1432-1327.1998.2540248.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martinand</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Salehzada</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Silhol</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lebleu</span>  <span class="cit-name-given-names">B</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Bisbal</span>  <span class="cit-name-given-names">C</span></span> (<span class="cit-pub-date">1998</span>) <span class="cit-article-title">RNase L inhibitor (RLI) antisense constructions block partially the down regulation of the 2-5A/RNase L pathway in encephalomyocarditis-virus-(EMCV)-infected cells</span>. <abbr class="cit-jnl-abbrev">Eur J Biochem</abbr> <span class="cit-vol">254</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">248</span>-<span class="cit-lpage">255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BJournal%26rft.stitle%253DFEBS%2BJ%26rft.aulast%253DMartinand%26rft.auinit1%253DC%26rft.volume%253D254%26rft.issue%253D2%26rft.spage%253D248%26rft.epage%253D255%26rft.atitle%253DRNase%2BL%2Binhibitor%2B%2528RLI%2529%2Bantisense%2Bconstructions%2Bblock%2Bpartially%2Bthe%2Bdown%2Bregulation%2Bof%2Bthe%2B2-5A%252FRNase%2BL%2Bpathway%2Bin%2Bencephalomyocarditis-virus-%2528EMCV%2529-infected%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1432-1327.1998.2540248.x%26rft_id%253Dinfo%253Apmid%252F9660177%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1432-1327.1998.2540248.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9660177&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F07%2F24%2F167999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000074158900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.37" data-doi="10.1371/journal.pone.0008266"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Crameri</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Establishment, immortalisation and characterisation of pteropid bat cell lines</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">e8266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DCrameri%26rft.auinit1%253DG.%26rft.volume%253D4%26rft.issue%253D12%26rft.spage%253De8266%26rft.epage%253De8266%26rft.atitle%253DEstablishment%252C%2Bimmortalisation%2Band%2Bcharacterisation%2Bof%2Bpteropid%2Bbat%2Bcell%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0008266%26rft_id%253Dinfo%253Apmid%252F20011515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0008266&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20011515&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F07%2F24%2F167999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.38" data-doi="10.1186/gb-2013-14-4-r36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions</span>. <abbr class="cit-jnl-abbrev">Genome Biol</abbr> <span class="cit-vol">14</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">R36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D14%26rft.spage%253DR36%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-4-r36%26rft_id%253Dinfo%253Apmid%252F23618408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-4-r36&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23618408&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F07%2F24%2F167999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.39" data-doi="10.1093/bioinformatics/btr355"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pimentel</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Pachter</span>  <span class="cit-name-given-names">L</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Identification of novel transcripts in annotated genomes using RNA-Seq</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>(<span class="cit-issue">17</span>):<span class="cit-fpage">2325</span>-<span class="cit-lpage">2329</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr355%26rft_id%253Dinfo%253Apmid%252F21697122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr355&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21697122&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F07%2F24%2F167999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294067300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.40" data-doi="10.1093/bioinformatics/btt656"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liao</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">GK</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">W</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">featureCounts: an efficient general purpose program for assigning sequence reads to genomic features</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">923</span>-<span class="cit-lpage">930</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt656%26rft_id%253Dinfo%253Apmid%252F24227677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt656&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24227677&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F07%2F24%2F167999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000334078300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.41" data-doi="10.1093/bioinformatics/btp616"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-auth"><span class="cit-name-surname">McCarthy</span>  <span class="cit-name-given-names">DJ</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">GK</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">edgeR: a Bioconductor package for differential expression analysis of digital gene expression data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">139</span>-<span class="cit-lpage">140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp616%26rft_id%253Dinfo%253Apmid%252F19910308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp616&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19910308&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F07%2F24%2F167999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273116100025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.42" data-doi="10.1186/gb-2014-15-2-r29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Law</span>  <span class="cit-name-given-names">CW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">W</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">GK</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">voom: Precision weights unlock linear model analysis tools for RNA-seq read counts</span>. <abbr class="cit-jnl-abbrev">Genome Biol</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">R29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D15%26rft.spage%253DR29%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2014-15-2-r29%26rft_id%253Dinfo%253Apmid%252F24485249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2014-15-2-r29&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24485249&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F07%2F24%2F167999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.43" data-doi="10.1093/nar/25.17.3389"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">SF</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">1997</span>) <span class="cit-article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">17</span>):<span class="cit-fpage">3389</span>-<span class="cit-lpage">3402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.17.3389%26rft_id%253Dinfo%253Apmid%252F9254694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/25.17.3389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9254694&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F07%2F24%2F167999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XU79300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.44" data-doi="10.1038/nmeth.3047"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sanjana</span>  <span class="cit-name-given-names">NE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shalem</span>  <span class="cit-name-given-names">O</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">F</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Improved vectors and genome-wide libraries for CRISPR screening</span>. <abbr class="cit-jnl-abbrev">Nat Methods</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">783</span>-<span class="cit-lpage">784</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D11%26rft.spage%253D783%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3047%26rft_id%253Dinfo%253Apmid%252F25075903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3047&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25075903&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F07%2F24%2F167999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340075600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.45" data-doi="10.1093/bioinformatics/btu743"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Naito</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hino</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bono</span>  <span class="cit-name-given-names">H</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Ui-Tei</span>  <span class="cit-name-given-names">K</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">CRISPRdirect: software for designing CRISPR/Cas guide RNA with reduced off-target sites</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">1120</span>-<span class="cit-lpage">1123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu743%26rft_id%253Dinfo%253Apmid%252F25414360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu743&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25414360&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F07%2F24%2F167999.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-167999v1.46" data-doi="10.1126/science.1247005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shalem</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Genome-scale CRISPR-Cas9 knockout screening in human cells</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">343</span>(<span class="cit-issue">6166</span>):<span class="cit-fpage">84</span>-<span class="cit-lpage">87</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1247005%26rft_id%253Dinfo%253Apmid%252F24336571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzNDMvNjE2Ni84NCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzA3LzI0LzE2Nzk5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/167999v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F167999v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/167619v1" title="Best Practice Data Life Cycle Approaches for the Life Sciences" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/167445v1" title="beachmat: a Bioconductor C++ API for accessing single-cell genomics data from a variety of R matrix types" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;July 24, 2017.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/167999v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/167999v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/167999v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/167999v1" class="active" data-icon-position="" data-hide-link-title="0">The IFN response in bat cells consists of canonical and non-canonical ISGs with unique temporal expression kinetics</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/50065" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Wje2EkSZ0T_pJ6uUoueENH5GQ8Y5StfXcKGdUZcE5pM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="50065" id="node50065--2" data-pisa="biorxiv;167999v1" data-pisa-master="biorxiv;167999" data-seqnum="50065" data-apath="/biorxiv/early/2017/07/24/167999.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">The IFN response in bat cells consists of canonical and non-canonical ISGs with unique temporal expression kinetics</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Pamela C.</span> <span class="nlm-surname">De La Cruz-Rivera</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Mohammed</span> <span class="nlm-surname">Kanchwala</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Hanquan</span> <span class="nlm-surname">Liang</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Ashwani</span> <span class="nlm-surname">Kumar</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Lin-Fa</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Chao</span> <span class="nlm-surname">Xing</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">John W.</span> <span class="nlm-surname">Schoggins</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">167999; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/167999 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/167999v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/167999v1&amp;title=The%20IFN%20response%20in%20bat%20cells%20consists%20of%20canonical%20and%20non-canonical%20ISGs%20with%20unique%20temporal%20expression%20kinetics" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/167999v1&amp;title=The%20IFN%20response%20in%20bat%20cells%20consists%20of%20canonical%20and%20non-canonical%20ISGs%20with%20unique%20temporal%20expression%20kinetics" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/167999v1&amp;text=The%20IFN%20response%20in%20bat%20cells%20consists%20of%20canonical%20and%20non-canonical%20ISGs%20with%20unique%20temporal%20expression%20kinetics" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/167999v1&amp;title=The%20IFN%20response%20in%20bat%20cells%20consists%20of%20canonical%20and%20non-canonical%20ISGs%20with%20unique%20temporal%20expression%20kinetics" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/167999v1&amp;t=The%20IFN%20response%20in%20bat%20cells%20consists%20of%20canonical%20and%20non-canonical%20ISGs%20with%20unique%20temporal%20expression%20kinetics" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/167999v1&amp;title=The%20IFN%20response%20in%20bat%20cells%20consists%20of%20canonical%20and%20non-canonical%20ISGs%20with%20unique%20temporal%20expression%20kinetics" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/167999v1&amp;title=The%20IFN%20response%20in%20bat%20cells%20consists%20of%20canonical%20and%20non-canonical%20ISGs%20with%20unique%20temporal%20expression%20kinetics" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="50065" id="node50065--3" data-pisa="biorxiv;167999v1" data-pisa-master="biorxiv;167999" data-seqnum="50065" data-apath="/biorxiv/early/2017/07/24/167999.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">The IFN response in bat cells consists of canonical and non-canonical ISGs with unique temporal expression kinetics</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Pamela C.</span> <span class="nlm-surname">De La Cruz-Rivera</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Mohammed</span> <span class="nlm-surname">Kanchwala</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Hanquan</span> <span class="nlm-surname">Liang</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Ashwani</span> <span class="nlm-surname">Kumar</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Lin-Fa</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Chao</span> <span class="nlm-surname">Xing</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">John W.</span> <span class="nlm-surname">Schoggins</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">167999; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/167999 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=The%20IFN%20response%20in%20bat%20cells%20consists%20of%20canonical%20and%20non-canonical%20ISGs%20with%20unique%20temporal%20expression%20kinetics&amp;rft.volume&amp;rft.issue&amp;rft.date=2017-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=De+La+Cruz-Rivera%2C+Pamela+C.&amp;rft.au=Kanchwala%2C+Mohammed&amp;rft.au=Liang%2C+Hanquan&amp;rft.au=Kumar%2C+Ashwani&amp;rft.au=Wang%2C+Lin-Fa&amp;rft.au=Xing%2C+Chao&amp;rft.au=Schoggins%2C+John+W."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/50065/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/50065/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/50065/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/50065/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/50065/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/50065/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/50065/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/50065/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/50065/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/50065/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/50065/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/50065/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/167999v1&amp;count=horizontal&amp;via=&amp;text=The%20IFN%20response%20in%20bat%20cells%20consists%20of%20canonical%20and%20non-canonical%20ISGs%20with%20unique%20temporal%20expression%20kinetics&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/167999v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/167999v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/167999v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/immunology" class="highlight" data-icon-position="" data-hide-link-title="0">Immunology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
