<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Gut Pathog</span><span class="iso-abbrev" title="iso-abbrev">Gut Pathog</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Gut Pathogens</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1757-4749</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">BioMed Central</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/5824483">5824483</a></span><span class="publisher-id" title="publisher-id">233</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1186/s13099-018-0233-1">10.1186/s13099-018-0233-1</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Molecular epidemiology of genogroup II norovirus infections in acute gastroenteritis patients during 2014â€“2016 in Pudong New Area, Shanghai, China</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-0877-572X</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xue</span><span class="given-names" tagx="given-names" title="given-names">Caoyi</span></span><a href="#Aff1">1</a><a href="#Aff2">2</a><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pan</span><span class="given-names" tagx="given-names" title="given-names">Lifeng</span></span><a href="#Aff2">2</a><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">Weiping</span></span><a href="#Aff2">2</a><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Yuanping</span></span><a href="#Aff2">2</a><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">Huiqin</span></span><a href="#Aff2">2</a><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cui</span><span class="given-names" tagx="given-names" title="given-names">Chang</span></span><a href="#Aff2">2</a><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">Lan</span></span><a href="#Aff2">2</a><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qiao</span><span class="given-names" tagx="given-names" title="given-names">Sun</span></span><a href="#Aff2">2</a><a href="#Aff3">3</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">Biao</span></span><span class="address" title="address"><span class="email" tagx="email" title="email">bxu@shmu.edu.cn</span></span><a href="#Aff1">1</a></span><span class="citation_author_institution" id="Aff1">[1], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 0125 2443</span><span class="institution-id" title="institution-id">grid.8547.e</span><span class="institution" title="institution">School of Public Health, </span><span class="institution" title="institution">Fudan University, </span></div></span><span class="citation_author_institution" id="Aff2">[2], </span><span class="citation_author_institution" id="Aff3">[3], <div class="institution-wrap" title="institution-wrap"><span class="institution-id" title="institution-id">0000 0001 0125 2443</span><span class="institution-id" title="institution-id">grid.8547.e</span><span class="institution" title="institution">Fudan University Pudong Institute of Preventive Medicine, </span></div></span></div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-2-2</span></span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2018-2-2</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2018</span></span><span class="volume" tagx="volume" title="volume">10</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">7</span><span class="history" title="history"><span class="received" title="received">received: 2017-12-27</span><span class="accepted" title="accepted">accepted: 2018-2-19</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , </span><span class="license" title="license"><span class="license-p" title="license-p"><b>Open Access</b>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<a href="http://creativecommons.org/publicdomain/zero/1.0/">http://creativecommons.org/publicdomain/zero/1.0/</a>) applies to the data made available in this article, unless otherwise stated.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p id="Par1">Norovirus (NoV), a member of the Caliciviridae, is now recognized as the leading cause of acute gastroenteritis (AGE) worldwide. Globally, the GII.4 Sydney_2012 variant has predominated in NoV-related AGE since 2012, although the novel variant GII.17 has also been reported as responsible for gastroenteritis outbreaks in East Asia since 2014. This study aimed to disclose the recent genotype patterns of NoV genogroup II (GII) presenting in AGE patients in Pudong New Area of Shanghai through a laboratory-based syndromic surveillance system. The study further aimed to delineate the predominant strains circulating in the population.</p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <p id="Par2">Pudong New Area is located in eastern Shanghai and covers 20.89% of the Shanghai population. The laboratory-based syndromic surveillance system is composed of 12 sentinel hospitals among the 68 general hospitals in this area. AGE patients who sought medical care were sampled following an AGE surveillance protocol. Stool samples were collected from participating patients, and a standardized questionnaire was given to each patient by trained nurses to gain information on the disease profiles and demographics of the patients. Real-time reverse transcription polymerase chain reaction (qRT-PCR) was used to screen the GI nd GII NoV and RT-PCR was used to amplify NoV GII partial capsid protein open reading frame 2 (ORF2). NoV Genotyping Tool (version 1.0, RIVM, MA Bilthoven, Netherlands) was used for genotyping, and a phylogenetic analysis was conducted by MEGA 7.0.</p> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <p id="Par3">During 2014â€“2016, among the 2069 virus-infected AGE cases, 65.88% were caused by NoV. NoV-AGE occurred most frequently in the periods from October to March. The patients with more severe diarrheal symptoms and vomiting were more likely to be infected by NoV. The main genotypes were GII.17 (44.69%) and GII.4 (39.26%), which dominated the NoV-AGE epidemics jointly or in turn, whereas a slight increase in GII.2 was observed beginning in May 2016. The GII.17 strains tended to cluster more with the Hu/JP/2014/GII.P17_GII.17/Kawasaki323 variants, representing novel prevalent strains. Among the GII.4 strains, the GII.4 Sydney_2012 variant was still the predominant strain.</p> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p id="Par4">NoV GII has become the main cause of virus-infected AGE in Pudong New Area, Shanghai. The predominant genotypes of NoV GII were GII.17 and GII.4. Comprehensive laboratory-based surveillance is important for clinical diagnosis and treatment. Identification of emerging new genotypes is also crucial for the prevention and control of NoV-infected AGE.</p> </div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"><span class="institution" title="institution">Young Medical Talent Training Program of Health System of Pudong New Area,Shanghai</span></div><span class="award-id" tagx="award-id" title="award-id">PWRq2016-17</span><span class="principal-award-recipient" title="principal-award-recipient"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xue</span><span class="given-names" tagx="given-names" title="given-names">Caoyi</span></span></span></div> </div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="issue-copyright-statement" value="Â© The Author(s) 2018" /> </div> </div> </div> <div class="body" title="body"> <div class="background" title="sec"> <div class="title" tagx="title" title="title">
Background</div> <p id="Par11">Acute gastroenteritis (AGE) is a major cause of morbidity and mortality affecting both developing and developed countries [<a href="#CR1">1</a>, <a href="#CR2">2</a>]. AGE is ranked second among all infectious diseases, causing an estimated 89.5 million disability adjusted life years (DALYs) and 1.45 million deaths worldwide per year [<a href="#CR2">2</a>].</p> <p id="Par12">Pudong New Area is located in eastern Shanghai with the population of 5.5 million, covering 20.89% of the Shanghai. To support early detection of and response to AGE epidemics, a laboratory-based syndromic surveillance system was established in 2010 in Pudong New Area of Shanghai. Based on service load, laboratory capacity, and geographic location, 12 of the 68 general hospitals in this area were selected as sentinel hospitals for AGE syndromic surveillance. The surveillance system includes three tertiary general hospitals, seven secondary general hospitals and three community health care centers covering about 71.2% AGE patients per year.</p> <p id="Par13">Norovirus (NoV), a member of Caliciviridae, is now recognized as the leading cause of AGE worldwide [<a href="#CR3">3</a>]. The first report of a NoV-AGE outbreak was in 1968 in the United States. Since then, NoV-AGE has become epidemic both in developed and developing countries, especially among children and the elderly. In Shanghai, according to the surveillance data, outbreaks occurred in schools and workplaces every year [<a href="#CR4">4</a>, <a href="#CR5">5</a>]. These outbreaks could involve tens or even hundreds of patients. Thus, understanding the epidemiology of NoV-AGE and enhancing pathogen-guided intervention is of great importance.</p> <p id="Par14">Norovirus is currently classified into five genogroups, among which only NoV GI, G GII and GIV are associated with human gastroenteritis [<a href="#CR6">6</a>]. The genome contains 3 open reading frames (ORFs). ORF1 encodes non-structural proteins that include an RNA-dependent RNA polymerase (RdRp); ORF2 encodes a capsid protein (VP1); and ORF3 encodes a minor capsid protein (VP2). NoV G GII is known to have a wider circulation than that of NoV GI, playing a major role in AGE. Based on differences in capsid protein VP1, NoV G GII can be further classified into 23 genotypes, namely, GII.1â€“GII.23 [<a href="#CR7">7</a>].</p> <p id="Par15">Norovirus circulate widely and hold considerable genetic diversity. NoV GII.4 has become the most common genotype circulating globally [<a href="#CR8">8</a>] and represents a typical example of NoV diversity. However, since the autumn of 2014, a new GII.17 strain has emerged as the major cause of AGE outbreaks. The emergence of new NoV genotypes suggests further threats of outbreaks all over the world. Moreover, some studies have found that the newly emerging strains might have stronger invasion abilities and target people with impaired immunity through displacing the previous predominant genotype [<a href="#CR9">9</a>].</p> <p id="Par16">The average annual incidence for NoV-associated outpatient visits was estimated to be 1.5/100 person-years with diarrhea in the community who sought medical care and a community incidence of 8.9/100 person-years by conducting the age-stratified Hospital Utilization Attitudes Survey among respondents residing in Pudong. NoV was a substantial burden on the community and healthcare system of Pudong [<a href="#CR10">10</a>]. Thus, understanding the dominant genotypes of NoVs and identifying longitudinal variation in circulating NoVs are key to NoV-AGE diagnosis, vaccine development, and outbreak prevention and control. This study aimed to study the molecular epidemiology of GII NoV presenting in AGE patients in Pudong New Area of Shanghai through a laboratory-based syndromic surveillance system and, furthermore, to delineate the dominant strains circulating in the population in recent years. Findings from this study could be useful in the early detection of and rapid response to potential NoV-AGE outbreaks.</p> </div> <div class="methods" title="sec"> <div class="title" tagx="title" title="title">
Methods</div> <div class="studyparticipantsandspecimencollection" title="sec"> <div class="title" tagx="title" title="title">
Study participants and specimen collection</div> <p id="Par17">From January 1, 2014, to December 12, 2016, in each sentinel hospital every week, the first two to five AGE patients who sought medical care at the diarrhea outpatient department were identified for pathogenic-based surveillance. According to the definition given by the World Health Organization [<a href="#CR11">11</a>] and Diarrheal Syndrome Surveillance Protocol [<a href="#CR12">12</a>], the requirements for AGE reporting are those visiting diarrhea outpatient department with 3 or more loose or liquid stools per day. Each participant should have a 2-week symptom-free period before the episode. The AGE cases of suspected or confirmed outbreaks were excluded. Each patient was requested to submit one stool sample for bacterial culture and viral detection following the routine diagnosis procedure of AGE. The specimens were sent to the Pudong New Area Center for Disease Control and Prevention (PDCDC) within 24 h.</p> </div> <div class="rnaextraction" title="sec"> <div class="title" tagx="title" title="title">
RNA extraction</div> <p id="Par18">Nucleic acids were extracted from a 200 Î¼L suspension using a QIAxtractor TM workstation (Qiagen, Hilden, Germany) with QIAxtractor Virus Reagents Qiagen, Maryland, USA) according to the manufacturerâ€™s instructions. The elution volume used for the nucleic acids was 50 Î¼L.</p> </div> <div class="primersandprobes" title="sec"> <div class="title" tagx="title" title="title">
Primers and probes</div> <p id="Par19">Real-time reverse transcription polymerase chain reaction (qRT-PCR) was used for initial sample testing for NoV according to the method of previous literature [<a href="#CR13">13</a>]. Genes for sequencing were amplified by RT-PCR from those qRT-PCR positive samples. The primer pairs and probes used for sample screening and sequencing of NoV are listed in Table <a href="#Tab1">1</a>. <div class="table-wrap_UNKNOWN" id="Tab1"><span class="label" tagx="label" title="label">Table 1</span> </div></p><p>Primers and probes used to detect and sequence NoV GII</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left" /> <th align="left" colspan="2">Primer/probe sequence<sup>a,b</sup></th> <th align="left">Location of 5â€² end</th> <th align="left">References</th> </tr> </thead> <tbody> <tr> <td align="left" rowspan="3">qRT-PCR</td> <td align="left">F</td> <td align="left">CARGARBCNATGTTYAGR TGGATGAG</td> <td align="left">5003</td> <td align="left" rowspan="3">[<a href="#CR13">13</a>]</td> </tr> <tr> <td align="left">R</td> <td align="left">TCGACGCCATCTTCATTCACA</td> <td char="." align="char">5100</td> </tr> <tr> <td align="left">P</td> <td align="left">TGGGAGGGCGATCGCAATCT</td> <td char="." align="char">5048</td> </tr> <tr> <td align="left" rowspan="2">RT-PCR</td> <td align="left">F</td> <td align="left">CNTGGGAGGGCGATCGCAA</td> <td char="." align="char">5058</td> <td align="left" rowspan="2">[<a href="#CR14">14</a>]</td> </tr> <tr> <td align="left">R</td> <td align="left">CCRCCNGCATRHCCRTTRTACAT</td> <td char="." align="char">5401</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <p><sup>a</sup>Mixed bases are as follows: Y, C or T; R, A or G; B, not A; N, any</p> <p><sup>b</sup>F, R and P represent forward primer, reverse primer and oligonucleotide capture probe, respectively</p> </div>  <p /> </div> <div class="screeingforgiandgiinov" title="sec"> <div class="title" tagx="title" title="title">
Screeing for GI and GII NoV</div> <p id="Par20">NoV GI and GII positive samples were screened with QuantiTect<sup>Â®</sup> Probe RT-PCR Kit (Qiagen, Hilden, Germany). Each 20 Î¼L reaction volume consisted of 5.7 Î¼L RNase-free water, 12.5 Î¼L 2Ã— QuantiTect Probe RT-PCR Master Mix, 0.3 Î¼L 10 mM TaqMan prober, 0.3 Î¼L RT mix, 0.6 Î¼Lâ€“10 mM e forward and reverse primers, respectively. qPCR was performed with a LightCycler<sup>Â®</sup> 480II system (Roche, Switzerland) with the thermal profile of 50 Â°C for 30 min; 95 Â°C for 15 min, 40 cycles of 95 Â°C for 10 s, 56 Â°C for 1 min.</p> </div> <div class="genotypingofgiinov" title="sec"> <div class="title" tagx="title" title="title">
Genotyping of GII NoV</div> <p id="Par21">The partial capsid protein open reading frame 2 (ORF2) of NoV GII positive sample was amplified by RT-PCR according to the method of previous literature [<a href="#CR14">14</a>]. RT-PCR was performed with an ABI9700 GeneAmp PCR system (ABI, Marsiling, Singapore) The reaction were carried out in a 20 Î¼L reaction tube containing 5 Î¼L 5Ã— Qiagen OneStep RT-PCR Buffer, 1 Î¼L Qiagen Onestep RT-PCR Enzyme Mix (Qiagen, Hilden, Germany) 12 Î¼L of RNase-free water, 1 Î¼L Dntp Mix, 0.5 Î¼Lâ€“10 Mm forward and reverse primers, respectively. The thermal profile consisted of 50 Â°C for 30 min; 95 Â°C for 15 min, 40 cycles of 95 Â°C for 10 s, 52 Â°C for 1 min, 72 Â°C for 1 min; 72 Â°C for 10 min. PCR products were sized using QIAxcel DNA Screening Gel Cartridges (Qiagen, Hilden, Germany) on the QIAxcel system (Qiagen, Zurich, Switzerland), The expected PCR product size was 343 bp.</p> <p id="Par22">The amplicons were purified and sequenced by Shanghai Yiyue Biotechnology Co., Ltd. NoV genotypes were determined with NoV Genotyping Tool Version 1.0 (NGTV1, <a href="http://www.rivm.nl/mpf/norovirus/typingtool%2cRIVM%2cMA">http://www.rivm.nl/mpf/norovirus/typingtool,RIVM,MA</a> Bilthoven, Netherlands). Sequences of the strains that were isolated in the study were deposited in the GenBank database. The accession numbers are listed in Table <a href="#Tab3">3</a> according to their genotypes based on the capsed region of ORF2. Alignment of the sequences was performed using BioEdit Sequence Alignment Editor software v7.0.9.0 [<a href="#CR15">15</a>]. The dendrogram was graphed by using the neighbour-joining method in MEGA version 7 [<a href="#CR16">16</a>]. Bootstrap resampling (1000 replications) was used, and bootstrap values â‰¥ 50% are shown.</p> </div> <div class="statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
Statistical analysis</div> <p id="Par23">The Statistical Product and Service Solutions (SPSS v19.0) software (IBM, USA) was used for all analyses. Differences in discrete variable levels were examined using the Chi square test or the Fisherâ€™s exact with two-tailed. A value of P &amp;lt; 0.05 was considered statistically significant.</p> </div> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="detectiontrendsofnovinfectioninagepatients" title="sec"> <div class="title" tagx="title" title="title">
Detection trends of NoV infection in AGE patients</div> <p id="Par24">In total, 5927 AGE patients were reported to the surveillance system during 2014â€“2016. The total detection rate of any intestinal virus was 34.91%, while 1363 NoV-AGE was detected, accounting for 65.88% of viral AGE cases. NoV-AGE occurred most frequently in the periods from October to March (Fig. <a href="#Fig1">1</a>). Among the AGE cases caused by NoV, the main genogroup was GII throughout these 3 years, while GI infection and double infection were rare (Table <a href="#Tab2">2</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 1</span></a></div>   <p>Detection rates of pathogens in AGE patients during 2014â€“16, Pudong New Area, Shanghai</p>   </div> <div class="table-wrap_UNKNOWN" id="Tab2"><span class="label" tagx="label" title="label">Table 2</span> <p>Genogroups of NoV detected in AGE patients from 2014 to 2016 in Pudong New Area, Shanghai</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left" rowspan="2">Date</th> <th align="left">AGE cases</th> <th align="left" rowspan="2">AGE NoV infections</th> <th align="left" colspan="2">GI</th> <th align="left" colspan="2">GII</th> <th align="left" colspan="2">Double infection with GI and GII</th> </tr> <tr> <th align="left">n</th> <th align="left">n</th> <th align="left">%</th> <th align="left">n</th> <th align="left">%</th> <th align="left">n</th> <th align="left">%</th> </tr> </thead> <tbody> <tr> <td align="left">14-Jan</td> <td char="." align="char">113</td> <td char="." align="char">22</td> <td char="." align="char">3</td> <td char="." align="char">2.65</td> <td char="." align="char">18</td> <td char="." align="char">15.93</td> <td char="." align="char">1</td> <td char="." align="char">0.01</td> </tr> <tr> <td align="left">14-Feb</td> <td char="." align="char">77</td> <td char="." align="char">9</td> <td char="." align="char">1</td> <td char="." align="char">1.30</td> <td char="." align="char">6</td> <td char="." align="char">7.79</td> <td char="." align="char">2</td> <td char="." align="char">0.03</td> </tr> <tr> <td align="left">14-Mar</td> <td char="." align="char">115</td> <td char="." align="char">23</td> <td char="." align="char">3</td> <td char="." align="char">2.61</td> <td char="." align="char">19</td> <td char="." align="char">16.52</td> <td char="." align="char">1</td> <td char="." align="char">0.01</td> </tr> <tr> <td align="left">14-Apr</td> <td char="." align="char">79</td> <td char="." align="char">23</td> <td char="." align="char">9</td> <td char="." align="char">11.39</td> <td char="." align="char">13</td> <td char="." align="char">16.46</td> <td char="." align="char">1</td> <td char="." align="char">0.01</td> </tr> <tr> <td align="left">14-May</td> <td char="." align="char">124</td> <td char="." align="char">17</td> <td char="." align="char">3</td> <td char="." align="char">2.42</td> <td char="." align="char">14</td> <td char="." align="char">11.29</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> </tr> <tr> <td align="left">14-Jun</td> <td char="." align="char">123</td> <td char="." align="char">9</td> <td char="." align="char">1</td> <td char="." align="char">0.81</td> <td char="." align="char">7</td> <td char="." align="char">5.69</td> <td char="." align="char">1</td> <td char="." align="char">0.01</td> </tr> <tr> <td align="left">14-Jul</td> <td char="." align="char">214</td> <td char="." align="char">13</td> <td char="." align="char">1</td> <td char="." align="char">0.47</td> <td char="." align="char">12</td> <td char="." align="char">5.61</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> </tr> <tr> <td align="left">14-Aug</td> <td char="." align="char">172</td> <td char="." align="char">19</td> <td char="." align="char">1</td> <td char="." align="char">0.58</td> <td char="." align="char">18</td> <td char="." align="char">10.47</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> </tr> <tr> <td align="left">14-Sep</td> <td char="." align="char">214</td> <td char="." align="char">27</td> <td char="." align="char">1</td> <td char="." align="char">0.47</td> <td char="." align="char">26</td> <td char="." align="char">12.15</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> </tr> <tr> <td align="left">14-Oct</td> <td char="." align="char">180</td> <td char="." align="char">65</td> <td char="." align="char">1</td> <td char="." align="char">0.56</td> <td char="." align="char">64</td> <td char="." align="char">35.56</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> </tr> <tr> <td align="left">14-Nov</td> <td char="." align="char">222</td> <td char="." align="char">53</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> <td char="." align="char">53</td> <td char="." align="char">23.87</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> </tr> <tr> <td align="left">14-Dec</td> <td char="." align="char">205</td> <td char="." align="char">60</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> <td char="." align="char">60</td> <td char="." align="char">29.27</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> </tr> <tr> <td align="left">15-Jan</td> <td char="." align="char">153</td> <td char="." align="char">52</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> <td char="." align="char">52</td> <td char="." align="char">33.99</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> </tr> <tr> <td align="left">15-Feb</td> <td char="." align="char">99</td> <td char="." align="char">44</td> <td char="." align="char">2</td> <td char="." align="char">2.02</td> <td char="." align="char">42</td> <td char="." align="char">42.42</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> </tr> <tr> <td align="left">15-Mar</td> <td char="." align="char">162</td> <td char="." align="char">93</td> <td char="." align="char">6</td> <td char="." align="char">3.70</td> <td char="." align="char">86</td> <td char="." align="char">53.09</td> <td char="." align="char">1</td> <td char="." align="char">0.01</td> </tr> <tr> <td align="left">15-Apr</td> <td char="." align="char">190</td> <td char="." align="char">65</td> <td char="." align="char">1</td> <td char="." align="char">0.53</td> <td char="." align="char">62</td> <td char="." align="char">32.63</td> <td char="." align="char">2</td> <td char="." align="char">0.01</td> </tr> <tr> <td align="left">15-May</td> <td char="." align="char">195</td> <td char="." align="char">30</td> <td char="." align="char">2</td> <td char="." align="char">1.03</td> <td char="." align="char">28</td> <td char="." align="char">14.36</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> </tr> <tr> <td align="left">15-Jun</td> <td char="." align="char">171</td> <td char="." align="char">16</td> <td char="." align="char">1</td> <td char="." align="char">0.58</td> <td char="." align="char">14</td> <td char="." align="char">8.19</td> <td char="." align="char">1</td> <td char="." align="char">0.01</td> </tr> <tr> <td align="left">15-Jul</td> <td char="." align="char">197</td> <td char="." align="char">15</td> <td char="." align="char">2</td> <td char="." align="char">1.02</td> <td char="." align="char">13</td> <td char="." align="char">6.60</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> </tr> <tr> <td align="left">15-Aug</td> <td char="." align="char">230</td> <td char="." align="char">16</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> <td char="." align="char">16</td> <td char="." align="char">6.96</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> </tr> <tr> <td align="left">15-Sep</td> <td char="." align="char">158</td> <td char="." align="char">18</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> <td char="." align="char">18</td> <td char="." align="char">11.39</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> </tr> <tr> <td align="left">15-Oct</td> <td char="." align="char">209</td> <td char="." align="char">74</td> <td char="." align="char">2</td> <td char="." align="char">0.96</td> <td char="." align="char">72</td> <td char="." align="char">34.45</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> </tr> <tr> <td align="left">15-Nov</td> <td char="." align="char">174</td> <td char="." align="char">61</td> <td char="." align="char">7</td> <td char="." align="char">4.02</td> <td char="." align="char">52</td> <td char="." align="char">29.89</td> <td char="." align="char">2</td> <td char="." align="char">0.01</td> </tr> <tr> <td align="left">15-Dec</td> <td char="." align="char">165</td> <td char="." align="char">26</td> <td char="." align="char">4</td> <td char="." align="char">2.42</td> <td char="." align="char">21</td> <td char="." align="char">12.73</td> <td char="." align="char">1</td> <td char="." align="char">0.01</td> </tr> <tr> <td align="left">16-Jan</td> <td char="." align="char">149</td> <td char="." align="char">20</td> <td char="." align="char">2</td> <td char="." align="char">1.34</td> <td char="." align="char">18</td> <td char="." align="char">12.08</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> </tr> <tr> <td align="left">16-Feb</td> <td char="." align="char">143</td> <td char="." align="char">34</td> <td char="." align="char">4</td> <td char="." align="char">2.80</td> <td char="." align="char">30</td> <td char="." align="char">20.98</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> </tr> <tr> <td align="left">16-Mar</td> <td char="." align="char">147</td> <td char="." align="char">55</td> <td char="." align="char">8</td> <td char="." align="char">5.44</td> <td char="." align="char">47</td> <td char="." align="char">31.97</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> </tr> <tr> <td align="left">16-Apr</td> <td char="." align="char">138</td> <td char="." align="char">48</td> <td char="." align="char">4</td> <td char="." align="char">2.90</td> <td char="." align="char">44</td> <td char="." align="char">31.88</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> </tr> <tr> <td align="left">16-May</td> <td char="." align="char">152</td> <td char="." align="char">37</td> <td char="." align="char">1</td> <td char="." align="char">0.66</td> <td char="." align="char">36</td> <td char="." align="char">23.68</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> </tr> <tr> <td align="left">16-Jun</td> <td char="." align="char">166</td> <td char="." align="char">24</td> <td char="." align="char">3</td> <td char="." align="char">1.81</td> <td char="." align="char">21</td> <td char="." align="char">12.65</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> </tr> <tr> <td align="left">16-Jul</td> <td char="." align="char">215</td> <td char="." align="char">20</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> <td char="." align="char">20</td> <td char="." align="char">9.30</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> </tr> <tr> <td align="left">16-Aug</td> <td char="." align="char">226</td> <td char="." align="char">37</td> <td char="." align="char">3</td> <td char="." align="char">1.33</td> <td char="." align="char">34</td> <td char="." align="char">15.04</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> </tr> <tr> <td align="left">16-Sep</td> <td char="." align="char">157</td> <td char="." align="char">70</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> <td char="." align="char">70</td> <td char="." align="char">44.59</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> </tr> <tr> <td align="left">16-Oct</td> <td char="." align="char">195</td> <td char="." align="char">69</td> <td char="." align="char">1</td> <td char="." align="char">0.51</td> <td char="." align="char">68</td> <td char="." align="char">34.87</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> </tr> <tr> <td align="left">16-Nov</td> <td char="." align="char">142</td> <td char="." align="char">39</td> <td char="." align="char">4</td> <td char="." align="char">2.82</td> <td char="." align="char">35</td> <td char="." align="char">24.65</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> </tr> <tr> <td align="left">16-Dec</td> <td char="." align="char">156</td> <td char="." align="char">60</td> <td char="." align="char">6</td> <td char="." align="char">3.85</td> <td char="." align="char">54</td> <td char="." align="char">34.62</td> <td char="." align="char">0</td> <td char="." align="char">0.00</td> </tr> </tbody> </table> </div> <p /> </div> <div class="compositionofgiinovinfectioninagepatients" title="sec"> <div class="title" tagx="title" title="title">
Composition of GII NoV infection in AGE patients</div> <p id="Par25">A total of 866 NoV GII strains among 1363 GII NoV-AGE were further genotyped by sequencing the partial nucleotide sequence of ORF2. Twelve genotypes of NoV GII were determined by NoV Genotyping Tool Version 1.0 (Fig. <a href="#Fig2">2</a>). During the years from 2014â€“2016, the main genotypes were GII.17 (44.69%, 387/866) and GII.4 (39.26%, 340/866). These strains dominated the NoV-AGE epidemics jointly or in turn. The GII.17 strain was first detected in a case reported in Aug 2014, and it had replaced GII.4 as the dominant NoV genotype by Aug 2015. After that time, GII.4 and GII.17 predominated in turns from Sep 2015 to Dec 2016. Since June 2016, GII.2 has become more prevalent (Fig. <a href="#Fig2">2</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 2</span></a></div>   <p>The composition of genogroup II NoV infection in the AGE patients (2014â€“2016)</p>   </div> <p /> </div> <div class="epidemiological,clinicalcharacteristicsofnovinfectionintheagepatients" title="sec"> <div class="title" tagx="title" title="title">
Epidemiological, clinical characteristics of NoV infection in the AGE patients</div> <p id="Par26">In the study, AGE cases were divided into five age groups, where 922 (52.6%), 530 (8.9%), 451 (7.6%), 3282 (55.4%), and 742 (12.5%) AGE patients fell into the age ranges of 0â€“5, 5â€“14, 15â€“64 and over 65 years, respectively. Among all the age groups, the detection rate of adults (25â€“64 years old) was highest (Ï‡<sup>2</sup> = 41.93, P &amp;lt; 0.001). The patients with more severe diarrheal symptoms and vomiting were more likely to be infected by NoV norovirus. The detection rates of NoV varies between different severity of diarrhea (Ï‡<sup>2</sup> = 15.68, P &amp;lt; 0.001). The patients with vomiting seems more likely to be infected by NoV (Ï‡<sup>2</sup> = 148.33, P &amp;lt; 0.001 (Tables <a href="#Tab3">3</a>, <a href="#Tab4">4</a>). <div class="table-wrap_UNKNOWN" id="Tab3"><span class="label" tagx="label" title="label">Table 3</span> </div></p><p>Demographic characteristics of the NoV-AGE patients</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left" rowspan="2" /> <th align="left">AGE patients</th> <th align="left">NoV-AGE patients</th> <th align="left" rowspan="2">Detection rate [%, (95% <i>CI</i>)]</th> <th align="left" rowspan="2">Ï‡<sup>2</sup></th> <th align="left" rowspan="2">P value</th> </tr> <tr> <th align="left">N (%)</th> <th align="left">N (%)</th> </tr> </thead> <tbody> <tr> <td align="left" colspan="6">Gender</td> </tr> <tr> <td align="left"> Male</td> <td align="left">3084 (52.0)</td> <td align="left">743 (54.5)</td> <td align="left">24.1 (22.6â€“25.6)</td> <td char="." align="char">4.36</td> <td char="." align="char">0.037</td> </tr> <tr> <td align="left"> Female</td> <td align="left">2843 (48.0)</td> <td align="left">620 (45.5)</td> <td align="left">21.8 (20.3â€“23.3)</td> <td align="left" /> <td align="left" /> </tr> <tr> <td align="left" colspan="6">Age group</td> </tr> <tr> <td align="left"> &amp;lt; 5 year</td> <td align="left">922 (15.6)</td> <td align="left">189 (13.9)</td> <td align="left">20.5 (17.9â€“23.1)</td> <td char="." align="char">41.93</td> <td char="." align="char">&amp;lt; 0.001</td> </tr> <tr> <td align="left"> 5â€“14 year</td> <td align="left">530 (8.9)</td> <td align="left">78 (5.7)</td> <td align="left">14.7 (11.7â€“17.7)</td> <td align="left" /> <td align="left" /> </tr> <tr> <td align="left"> 15â€“24 year</td> <td align="left">451 (7.6)</td> <td align="left">99 (7.3)</td> <td align="left">22.0 (18.1â€“25.8)</td> <td align="left" /> <td align="left" /> </tr> <tr> <td align="left"> 25â€“64 year</td> <td align="left">3282 (55.4)</td> <td align="left">847 (62.1)</td> <td align="left">25.8 (24.3â€“27.3)</td> <td align="left" /> <td align="left" /> </tr> <tr> <td align="left"> â‰¥ 65 year</td> <td align="left">742 (12.5)</td> <td align="left">150 (11.0)</td> <td align="left">20.2 (17.3â€“23.1)</td> <td align="left" /> <td align="left" /> </tr> <tr> <td align="left"> Total</td> <td align="left">5927</td> <td align="left">1363</td> <td align="left">22.3</td> <td align="left" /> <td align="left" /> </tr> </tbody> </table>  <div class="table-wrap_UNKNOWN" id="Tab4"><span class="label" tagx="label" title="label">Table 4</span> <p>Clinical manifestations of the NoV-AGE patients</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left" rowspan="2" /> <th align="left">AGE patients</th> <th align="left">NoV-AGE patients</th> <th align="left" rowspan="2">Detection rate [%, (95% <i>CI</i>)]</th> <th align="left" rowspan="2">Ï‡<sup>2</sup></th> <th align="left" rowspan="2">P value</th> </tr> <tr> <th align="left">N (%)</th> <th align="left">N (%)</th> </tr> </thead> <tbody> <tr> <td align="left" colspan="6">Severity of diarrhea</td> </tr> <tr> <td align="left"> 3â€“5 times per day</td> <td char="(" align="char">3752 (63.3)</td> <td char="(" align="char">801 (58.8)</td> <td char="(" align="char">21.3 (20.0â€“22.7)</td> <td char="." align="char">15.68</td> <td char="." align="char">&amp;lt; 0.001</td> </tr> <tr> <td align="left"> &amp;gt; 5 times per day</td> <td char="(" align="char">2175 (36.7)</td> <td char="(" align="char">562 (41.2)</td> <td char="(" align="char">25.8 (24.0â€“27.7)</td> <td align="left" /> <td align="left" /> </tr> <tr> <td align="left" colspan="6">Vomiting</td> </tr> <tr> <td align="left"> Yes</td> <td char="(" align="char">1398 (23.6)</td> <td char="(" align="char">489 (35.9)</td> <td char="(" align="char">35.0 (32.5â€“37.5)</td> <td char="." align="char">148.33</td> <td char="." align="char">&amp;lt; 0.001</td> </tr> <tr> <td align="left"> No</td> <td char="(" align="char">4529 (76.4)</td> <td char="(" align="char">874 (64.1)</td> <td char="(" align="char">19.3 (18.1â€“20.4)</td> <td align="left" /> <td align="left" /> </tr> <tr> <td align="left" colspan="6">Fever ( &amp;gt; 38 Â°C)</td> </tr> <tr> <td align="left"> Yes</td> <td char="(" align="char">833 (14.1)</td> <td char="(" align="char">196 (14.4)</td> <td char="(" align="char">23.5 (20.6â€“26.4)</td> <td char="." align="char">0.15</td> <td char="." align="char">0.693</td> </tr> <tr> <td align="left"> No</td> <td char="(" align="char">5094 (85.9)</td> <td char="(" align="char">1167 (85.6)</td> <td char="(" align="char">22.9 (21.8â€“24.1)</td> <td align="left" /> <td align="left" /> </tr> <tr> <td align="left" colspan="6">Dehydration</td> </tr> <tr> <td align="left"> Yes</td> <td char="(" align="char">43 (0.7)</td> <td char="(" align="char">9 (0.6)</td> <td char="(" align="char">20.9 (8.3â€“33.6)</td> <td char="." align="char">0.10</td> <td char="." align="char">0.747</td> </tr> <tr> <td align="left"> No</td> <td char="(" align="char">5884 (99.3)</td> <td char="(" align="char">1354 (99.4)</td> <td char="(" align="char">23.0 (21.9â€“24.1)</td> <td align="left" /> <td align="left" /> </tr> </tbody> </table> </div> <p /> <p id="Par27">The genotype seems to have a role in the severity of diarrhea among different groups. GII.4 infection may caused less severe diarrhea symptoms (Ï‡<sup>2</sup> = 23.40, P &amp;lt; 0.001). Slight seasonal trend was observed in NoV GII.4 had a peak in prevalence in the autumn while GII.17 in the spring (Ï‡<sup>2</sup> = 308.71, P &amp;lt; 0.001). The time distribution showed minor seasonal variations across 3 years, respectively (Ï‡<sup>2</sup> = 151.73, P &amp;lt; 0.001) (Tables <a href="#Tab5">5</a>, <a href="#Tab6">6</a>, <a href="#Tab7">7</a>). <div class="table-wrap_UNKNOWN" id="Tab5"><span class="label" tagx="label" title="label">Table 5</span> </div></p><p>Demographic characteristics of GII NoV-AGE patients with different genogroups</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left" /> <th align="left">GII.4 (n (%))</th> <th align="left">GII.17 (n (%))</th> <th align="left">Other GII (n (%))</th> <th align="left">Ï‡<sup>2</sup></th> <th align="left">P value</th> </tr> </thead> <tbody> <tr> <td align="left" colspan="6">Age group (year)</td> </tr> <tr> <td align="left"> &amp;lt; 5</td> <td align="left">82 (24.1)</td> <td align="left">20 (6.2)</td> <td align="left">21 (15.1)</td> <td char="." align="char">151.73</td> <td char="." align="char">&amp;lt; 0.001</td> </tr> <tr> <td align="left"> 5â€“14</td> <td align="left">18 (5.3)</td> <td align="left">14 (3.6)</td> <td align="left">8 (5.8)</td> <td align="left" /> <td align="left" /> </tr> <tr> <td align="left"> 15â€“24</td> <td align="left">18 (5.3)</td> <td align="left">20 (5.2)</td> <td align="left">13 (9.4)</td> <td align="left" /> <td align="left" /> </tr> <tr> <td align="left"> 25â€“64</td> <td align="left">181 (53.2)</td> <td align="left">296 (76.5)</td> <td align="left">43 (30.9)</td> <td align="left" /> <td align="left" /> </tr> <tr> <td align="left"> &amp;gt; 65</td> <td align="left">41 (12.1)</td> <td align="left">37 (9.5)</td> <td align="left">54 (38.8)</td> <td align="left" /> <td align="left" /> </tr> <tr> <td align="left" colspan="6">Gender</td> </tr> <tr> <td align="left"> Male</td> <td align="left">188 (55.3)</td> <td align="left">208 (53.7)</td> <td align="left">77 (55.4)</td> <td char="." align="char">0.22</td> <td char="." align="char">0.898</td> </tr> <tr> <td align="left"> Female</td> <td align="left">152 (44.7)</td> <td align="left">179 (46.3)</td> <td align="left">62 (44.6)</td> <td align="left" /> <td align="left" /> </tr> <tr> <td align="left"> Total</td> <td align="left">340</td> <td align="left">387</td> <td align="left">139</td> <td align="left" /> <td align="left" /> </tr> </tbody> </table>  <div class="table-wrap_UNKNOWN" id="Tab6"><span class="label" tagx="label" title="label">Table 6</span> <p>Seasonal distribution of GII NoV-AGE patients with different genogroups</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left" /> <th align="left">GII.4 (n (%))</th> <th align="left">GII.17 (n (%))</th> <th align="left">Other GII (n (%))</th> <th align="left">Ï‡<sup>2</sup></th> <th align="left">P value</th> </tr> </thead> <tbody> <tr> <td align="left" colspan="6">Year</td> </tr> <tr> <td align="left"> 2014</td> <td align="left">148 (43.5)</td> <td align="left">35 (9.0)</td> <td align="left">36 (25.9)</td> <td char="." align="char">259.95</td> <td char="." align="char">&amp;lt;0.001</td> </tr> <tr> <td align="left"> 2015</td> <td align="left">46 (13.5)</td> <td align="left">267 (67.0)</td> <td align="left">43 (30.9)</td> <td align="left" /> <td align="left" /> </tr> <tr> <td align="left"> 2016</td> <td align="left">146 (43.0)</td> <td align="left">85 (22.0)</td> <td align="left">60 (43.2)</td> <td align="left" /> <td align="left" /> </tr> <tr> <td align="left" colspan="6">Season</td> </tr> <tr> <td align="left"> Spring (Marâ€“May)</td> <td align="left">27 (7.9)</td> <td align="left">204 (52.7)</td> <td align="left">47 (33.9)</td> <td char="." align="char">308.71</td> <td char="." align="char">&amp;lt; 0.001</td> </tr> <tr> <td align="left"> Summer (Junâ€“Aug)</td> <td align="left">33 (9.7)</td> <td align="left">38 (9.8)</td> <td align="left">26 (18.7)</td> <td align="left" /> <td align="left" /> </tr> <tr> <td align="left"> Autumn (Sepâ€“Nov)</td> <td align="left">204 (60.0)</td> <td align="left">22 (5.7)</td> <td align="left">33 (23.7)</td> <td align="left" /> <td align="left" /> </tr> <tr> <td align="left"> Winter (Decâ€“Feb)</td> <td align="left">76 (22.4)</td> <td align="left">123 (31.8)</td> <td align="left">33 (23.7)</td> <td align="left" /> <td align="left" /> </tr> <tr> <td align="left"> Total</td> <td align="left">340</td> <td align="left">387</td> <td align="left">139</td> <td align="left" /> <td align="left" /> </tr> </tbody> </table> </div> <div class="table-wrap_UNKNOWN" id="Tab7"><span class="label" tagx="label" title="label">Table 7</span> <p>Clinical manifestation of GII NoV-AGE patients with different genogroups</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left" /> <th align="left">GII.4 (n (%))</th> <th align="left">GII.17 (n (%))</th> <th align="left">Other GII (n (%))</th> <th align="left">Ï‡<sup>2</sup></th> <th align="left">P value</th> </tr> </thead> <tbody> <tr> <td align="left" colspan="6">Severity of diarrhea</td> </tr> <tr> <td align="left"> 3â€“5 times per day</td> <td align="left">224 (65.9)</td> <td align="left">201 (51.9)</td> <td align="left">62 (44.6)</td> <td char="." align="char">23.40</td> <td char="." align="char">&amp;lt; 0.001</td> </tr> <tr> <td align="left"> &amp;gt; 5 times per day</td> <td align="left">116 (34.1)</td> <td align="left">186 (48.1)</td> <td align="left">77 (55.4)</td> <td align="left" /> <td align="left" /> </tr> <tr> <td align="left" colspan="6">Vomiting</td> </tr> <tr> <td align="left"> Yes</td> <td align="left">129 (37.9)</td> <td align="left">163 (42.1)</td> <td align="left">57 (41.0)</td> <td char="." align="char">1.34</td> <td char="." align="char">0.510</td> </tr> <tr> <td align="left"> No</td> <td align="left">211 (62.1)</td> <td align="left">224 (57.9)</td> <td align="left">82 (59.0)</td> <td align="left" /> <td align="left" /> </tr> <tr> <td align="left" colspan="6">Fever ( &amp;gt; 38 Â°C)</td> </tr> <tr> <td align="left"> Yes</td> <td align="left">48 (14.1)</td> <td align="left">55 (14.2)</td> <td align="left">19 (13.7)</td> <td char="." align="char">0.025</td> <td char="." align="char">0.987</td> </tr> <tr> <td align="left"> No</td> <td align="left">292 (85.9)</td> <td align="left">332 (85.8)</td> <td align="left">120 (86.3)</td> <td align="left" /> <td align="left" /> </tr> <tr> <td align="left" colspan="6">Dehydration</td> </tr> <tr> <td align="left"> Yes</td> <td align="left">1 (0.3)</td> <td align="left">7 (1.8)</td> <td align="left">1 (0.7)</td> <td char="." align="char">3.798</td> <td char="." align="char">0.129</td> </tr> <tr> <td align="left"> No</td> <td align="left">339 (99.7)</td> <td align="left">380 (98.2)</td> <td align="left">138 (99.3)</td> <td align="left" /> <td align="left" /> </tr> <tr> <td align="left">Total</td> <td align="left">340</td> <td align="left">387</td> <td align="left">139</td> <td align="left" /> <td align="left" /> </tr> </tbody> </table> </div> <p /> </div> <div class="nucleotidesequenceaccessionnumbers" title="sec"> <div class="title" tagx="title" title="title">
Nucleotide sequence accession numbers</div> <p id="Par28">The partial genomic NoV ORF2 nucleotide sequences of the strains that were isolated in this study were deposited in the GenBank database. The accession numbers are listed in Table <a href="#Tab8">8</a> according to their genotypes based on the capsid region of ORF2. <div class="table-wrap_UNKNOWN" id="Tab8"><span class="label" tagx="label" title="label">Table 8</span> </div></p><p>NoV GII strains genotyped with the capsid region of ORF2 from NoV-AGE patients in Pudong, Shanghai in 2014â€“2016</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th align="left">Genotype</th> <th align="left">Accession number of strains in this study</th> <th align="left">Quantity of strains</th> <th align="left">Strains in phylogenetic tree</th> </tr> </thead> <tbody> <tr> <td align="left">GII.1</td> <td align="left">KU672081</td> <td char="." align="char">1</td> <td align="left">KU672081</td> </tr> <tr> <td align="left">GII.2</td> <td align="left">KU672082, KU672083, KU672087, KY797735, KY983635-49</td> <td char="." align="char">19</td> <td align="left">KU672087, U672082, KU672083, KY983635, KY983636, KY983637, KY983641, KY983643, KY983644, KY983647, KY983649, KY797735</td> </tr> <tr> <td align="left">GII.3</td> <td align="left">KU672088-92, KU672103-109, KY797764, KY983651-79</td> <td char="." align="char">42</td> <td align="left">KU672103, KU672106, KU672107, KU672108, KU672088, KU672089, KU672090, KU672092, KY983651, KY983653, KY983654, KY983661, KY983668, KY983670, KY983671, KY983675, KY983677, KY983679, KY797764</td> </tr> <tr> <td align="left">GII.4</td> <td align="left">KU532332, KU532354-455, KU532676, KU532705-66, KY797698-725, KY983683-832</td> <td char="." align="char">340</td> <td align="left">KU532440, KU532363, KU532368, KU532373, KU532376, KU532384<br /><br />KU532386, KU532705, KU532709, KU532734, KU532758, KU532439, KU532441, KU532448, KU532454,<br /><br />KU532455, KY983692, KY983695,<br /><br />KY983704, KY983719, KY983728, KY983733, KY983737, KY983742, KY983747, KY983748, KY983750, KY983765, KY983798, KY983689, KY797711</td> </tr> <tr> <td align="left">GII.5</td> <td align="left">KU672135, KY797775</td> <td char="." align="char">2</td> <td align="left">KU672135, KY797775</td> </tr> <tr> <td align="left">GII.6</td> <td align="left">KU672136, KU672149-59, KY797777, KY983833-39</td> <td char="." align="char">20</td> <td align="left">KU672149, KU672151, KU672153, KU672156, KU672157, KU672158, KU672159, KU672136, KY983833, KY983835, KY983836, KY983838, KY983839, KY797777</td> </tr> <tr> <td align="left">GII.8</td> <td align="left">KU672163, KU672164, KY838340</td> <td char="." align="char">3</td> <td align="left">KU672163, KU672164, KY983840</td> </tr> <tr> <td align="left">GII.12</td> <td align="left">KU672167, KU672168</td> <td char="." align="char">2</td> <td align="left">KU672167</td> </tr> <tr> <td align="left">GII.13</td> <td align="left">KU672193-200, KU672170-177, KY983842-57, KY797778-81</td> <td char="." align="char">36</td> <td align="left">KU672193, KU672194, KU672197, KU672200, KU672170, KU672171, KU672172, KU672173, KU672177, KY983842, KY983843, KY983845, KY983847, KY983850, KY983851, KY983853, KY983854, KY983856, KY797778</td> </tr> <tr> <td align="left">GII.14</td> <td align="left">KU532778, KU532779, KY983858</td> <td char="." align="char">3</td> <td align="left">KU532778, KU532779, KY983858</td> </tr> <tr> <td align="left">GII.17</td> <td align="left">KU516833-966, KU516970-93, KU672203-331, KY797681-2, KY983864-961</td> <td char="." align="char">387</td> <td align="left">KU516882, KU516947, KU516961, KU516979, KU516980, KU516881, KU672221, KU516883, KU672240, KU516922, KU516942, KU672318, KY983864, KY983868, KY983874, KY983875, KY983876, KY983879, KY983882, KY983887, KY983896, KY983908, KY983927, KY983941, KY983951, KY983956, KY983958, KY983961, KY797685</td> </tr> <tr> <td align="left">GII.21</td> <td align="left">KU532767-76, KY983962</td> <td char="." align="char">11</td> <td align="left">KU532767, KU532768, KU532772, KU532776, KY983962</td> </tr> </tbody> </table>  <p /> </div> <div class="phylogeneticrelationshipsamonggiistrainsofnov" title="sec"> <div class="title" tagx="title" title="title">
Phylogenetic relationships among GII strains of NoV</div> <p id="Par29">A phylogenetic tree was constructed with 139 sequences from the strains collected and 24 additional worldwide reference sequences. The nucleotide sequences of the NoV ORF2 gene revealed that the GII.17 strains tended to cluster more with the strain Hu/JP/2014/GII.P17_GII.17/Kawasaki323 than with the strain Hu/US/2002/GII.17/CSE1, which suggested that the Kawasaki323 strain became the novel prevalent strain in Shanghai [<a href="#CR17">17</a>]. In the GII.4 group, the strain Hu/AU/2012/GII.4/Sydney/NSW0514 was more frequently observed than Hu/JP/2005/GII.4/Sakai was, indicating that the Sydney strain was still the predominant GII.4 strain in Shanghai (Fig. <a href="#Fig3">3</a>). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Fig. 3</span></a></div>   <p>Phylogenetic tree based on partial capsid region (ORF2) of NoV strains. Sequences from sporadic acute gastroenteritis patients in Shanghai, 2014â€“2016 are shown with black triangles, time and Genbank accession numbers in the phylogenetic trees. Reference sequences, obtained from GenBank, display in the phylogenetic trees with black square, time, genotype, city or country, and accession number</p>   </div> <p /> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p id="Par30">NoV infection is a leading cause of AGE among all age groups [<a href="#CR18">18</a>]. According to a systematic review of global NoV infection, the pooled prevalence of NoV infection in patients with AGE varied geographically, from approximately 19% (95% CI 16â€“22) in low-mortality developing countries to 20% (95% CI 17â€“22) in developed countries [<a href="#CR19">19</a>].</p> <p id="Par31">Given the high prevalence of NoV, representing a substantial disease burden to residents of all age groups, this study aimed to type the recent patterns of GII NoV presenting in AGE patients in Pudong through a laboratory-based syndromic surveillance system and to delineate the dominant strains circulating in the population. This 3-year data analysis and strain collection study could delineate the genotype circulation of NoV infection in AGE patients reported in Shanghai. The combination of epidemiological, etiological and molecular data analysis could provide comprehensive evidence to support NoV infection control.</p> <p id="Par32">In this study, the proportion of NoV in AGE patients was 21.59% in Pudong, close to that of developed countries. This result could be the result of a reduced number of bacterially-caused AGE cases, along with improvements in environmental and personal hygiene [<a href="#CR20">20</a>]. NoV-AGE was prevalent throughout the year and peaked in the cold season, as previously reported. In Pudong New Area, the seasonal increase of NoV-AGE started in October and usually lasted until March of the following year, consistent with the results of Xue et al. [<a href="#CR21">21</a>] in other districts in Shanghai and observations in other provinces in China [<a href="#CR22">22</a>, <a href="#CR23">23</a>]. In the study, the adult aged from 25 to 64 years old seemed vulnerable to get NoV infection, which was not consistent with previous studies [<a href="#CR19">19</a>]. Since health seeking behaviour is voluntary related to the severity of the illness and accessibility of health service, most mild patients may appeal to Over-The-Counter Drug. On the other hand, vomiting is considered as the most frequent symptoms in NoV infected kids and infants. The frequency of vomiting without concurrent diarrhea suggests that epidemiology studies that enroll subjects based on the presence of diarrhea may be significantly underestimating the true burden of NoV infection, especially in the young people [<a href="#CR24">24</a>]. The patients with more severe diarrhea symptoms and vomiting were more likely to be infected by NoV. However, vomiting with concurrent diarrhea should be regarded as the predictors of NoV infection for more efficient and direct testing and treatment [<a href="#CR25">25</a>]. The circulating genotypes of NoV could vary geographically and longitudinally, which might influence the size and severity of NoV-AGE epidemics. Chen et al. [<a href="#CR26">26</a>] reported that, during the winter of 2014â€“2015, 6 GII genotypes circulated among adult patients in Shanghai, i.e., GII.2, GII.3, GII.4, GII.6, GII.13, and GII.17. Despite covering only a part of Shanghai, this study found 12 GII genotypes in Pudong New Area from 2014 to 2016. NoV GII.4 is the most common genotype globally, due to its high diversity, and a new strain of GII.4 emerges every 2â€“3 years, since it has a 1.7-fold higher rate of evolution on average within the capsid sequence and a greater number of non-synonymous changes compared to other NoVs [<a href="#CR27">27</a>]. A study by Lindesmith and colleagues found that the NoV GII.4 strains alter their histo-blood group antigen (HBGA) carbohydrate-binding targets over time, which allows for not only escape from high-penetrance host susceptibility alleles but also immune-driven selection in the receptor-binding region to facilitate avoidance of protective herd immunity [<a href="#CR28">28</a>]. This locus could represent a focal point for vaccine research to control NoV infection.</p> <p id="Par33">For example, the GII.4 strain US95/96 was responsible for a majority of the norovirus outbreaks in the US and Europe from 1995 to 1996. Between 2000 and 2004, US95/96 was replaced by two new GII.4 variants: Farmington Hills in the US and a new GII.4 variant, GII.4b, in Europe. In 2004, the Hunter GII.4 variant was detected in Australia, Europe, and Asia, and then this variant was replaced in early 2006 by two new cocirculating GII.4 variants, Sakai and Minerva, in the US, Europe, and Asia [<a href="#CR28">28</a>]. In 2009, the New Orleans strain was reported circulating worldwide [<a href="#CR29">29</a>].</p> <p id="Par34">In March 2012, a new GII.4 norovirus strain was identified in Australia and named GII.4 Sydney. As the predominant strain of the GII.4 genotype, Sydney_2012 and its variants have been widely reported since 2012 [<a href="#CR30">30</a>â€“<a href="#CR32">32</a>]. This emerging strain has caused AGE outbreaks in multiple countries [<a href="#CR33">33</a>]. GII.4 NoVs remain the predominant cause of outbreaks, and the GII.4 Sydney strain appears to have replaced the previously predominant strain, GII.4 New Orleans. Compared with other genotypes, GII.4 NoVs were associated with increased rates of hospitalization and death during outbreaks [<a href="#CR34">34</a>]. Similar to the results of Han et al., Sydney_2012, together with Den_Haag_2006b and New_Orleans_2009, were the main strains of GII.4 causing AGE in Shanghai, with Sydney_2012 and its variants being the predominant strains [<a href="#CR35">35</a>]. In this study, strain Hu/AU/2012/GII.4/Sydney/NSW0514 was more frequently observed than Hu/JP/2005/GII.4/Sakai was, indicating that the Sydney strain was still the predominant strain. Public health practitioners should remain alert to the potential for increased norovirus activity in future seasons related to variations in this strain. During the study period, we observed circulation of ORF2 genotypes that showed the trends in norovirus genomic diversity, recombination, and NoV reporting.</p> <p id="Par35">Since the winter of 2014, Jiangsu [<a href="#CR36">36</a>] and Zhejiang Province [<a href="#CR35">35</a>] in China have reported AGE outbreaks caused by a novel NoV genotype, GII.17. NoV GII.17 has a polymerase sequence and amino acid substitution in the capsid region, according to a study by Matesushiam et al. [<a href="#CR37">37</a>]. The novel GII.17 genotype was first detected in AGE patients in Aug 2014 in Pudong New Area [<a href="#CR17">17</a>], before the first report of an outbreak in Jiangsu. AGE outbreaks caused by the novel GII.17 genotype have also been reported from nurseries, schools and a military unit [<a href="#CR38">38</a>, <a href="#CR39">39</a>]. NoV GII.17 could be detected in AGE patients, their close contacts and environmental surfaces such as trash cans and toilet bowls [<a href="#CR5">5</a>].</p> <p id="Par36">In this study, GII.2 infection was found to have increased slightly since June 2016. In late 2016, AGE outbreaks increased substantially in Pudong, greatly surpassing the numbers during the same season in previous years. Some studies reported that the majority of the outbreaks were associated with a new GII.2 strain that was likely to infect preschool children [<a href="#CR40">40</a>, <a href="#CR41">41</a>]. From 2012 to 2016, a total of 313 NoV outbreaks in 24 provinces and 96 cities in China were reported to the National Emergent Public Health Event Information Management System; 109 (35%) were reported in 2016, mostly in winter. In November and December 2016, NoV outbreaks increased sharply, accounting for 56 (51%) of the 109 NoV outbreaks, and 78% of the GII.2 outbreaks occurred in kindergartens. Further analyses should be conducted to combine sporadic cases with outbreak data to explore the impact of the circulation of this new strain.</p> <p id="Par37">Although this study was surveillance-based, the findings of this study can only present the circulating and predominant NoV strains in the local area. Information on the epidemiological, clinical and molecular characteristics of the circulating and emerging strains was not linked. Additional efforts should be given to the study of NoV-AGE outbreaks and the transmission mechanism of this virus.</p> </div> <div class="limitation" title="sec"> <div class="title" tagx="title" title="title">
Limitation</div> <p id="Par38">There were several limitations on our research. Firstly, the subjects were mainly selected from 12 sentinel hospitals selected through Modeling Spatial means of surfaces with stratified Non-homogeneity. The first two to five AGE patients who sought medical care at the diarrhea outpatient department were enrolled which would bring selection bias because of the convenient sampling. Secondly, the people under 14-year-old always visit the pediatric department while NoV usually cause vomiting among them, which would under-estimate the prevalence of the NoV infection among the under 14-year-old age group. Thirdly, genetic characteristics analysis of NoV was only based on partial polymerase regions (ORF2) in this study. Genotyping of RNA-dependent RNA polymerase regions should be conducted in order to enhance the accuracy and completeness of NoV GII genotyping which is our next research plan of the NoV infection surveillance. Meanwhile, the 3-year data of molecular epidemiological analysis of NoV infection has initiated in the year of 2014, which could only tracing out the circulating pattens rather than capturing the genomic diversity. This would also become our research focus in the future.</p> </div> <div class="conclusions" title="sec"> <div class="title" tagx="title" title="title">
Conclusions</div> <p id="Par39">NoV GII has become the main cause of virus-infected AGE in Pudong New Area, Shanghai. NoV-AGE accounted for 65.88% of viral AGE during 2014â€“2016, with GII as the main group; GII.17 and GII.4 dominated the NoV-AGE epidemics. GII.17 strains tended to cluster more with the strain Hu/JP/2014/GII.P17_GII.17/Kawasaki323, and GII.4 with Hu/AU/2012/GII.4/Sydney/NSW0514, as shown by the phylogenetic tree. A Comprehensive laboratory-based surveillance is important for clinical diagnosis and treatment. Identification of emerging new genotypes is also crucial for the prevention and control of NoV-infected AGE.</p> </div> </div> <div class="back" title="back"> <div class="glossary" title="glossary"> <div class="title" tagx="title" title="title">
Abbreviations</div> <div class="def-list" title="def-list"> <div class="def-item" title="def-item"> <div class="term" title="term">
AGE</div> <div class="def" title="def"> <p id="Par5">acute gastroenteritis</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
CI</div> <div class="def" title="def"> <p id="Par6">confidence interval</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
NoV</div> <div class="def" title="def"> <p id="Par7">norovirus</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
GI</div> <div class="def" title="def"> <p id="Par8">genogroup I</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
GII</div> <div class="def" title="def"> <p id="Par9">genogroup II</p> </div> </div> <div class="def-item" title="def-item"> <div class="term" title="term">
RT-PCR</div> <div class="def" title="def"> <p id="Par10">reverse transcriptase polymerase chain reaction</p> </div> </div> </div> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p>Caoyi Xue and Lifeng Pan contributed equally to this work</p> </div> </div> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Authorsâ€™ contributions</div> <p>CX designed the study, performed the analysis, and drafted and revised the manuscript. LP conceived and designed the experiments, performed the analysis and revised the manuscript. YW and CC conducted the investigation, collected the data, performed the analysis and commented on the manuscript. HF and LL collected the specimens, prepared the experiments and commented on the manuscript. WZ and QS designed the study, supervised the study and commented on the manuscript. BX designed and supervised the overall study, as well as conceptualized, drafted and revised the manuscript. All authors ensured the accuracy and integrity of this article. All authors read and approved the final manuscript.</p> <div class="acknowledgements" title="sec"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p id="Par40">Appreciation for the 12 sentinel hospitals of AGE surveillance of Pudong New Area, Shanghai, China.</p> </div> <div class="competinginterests" title="sec"> <div class="title" tagx="title" title="title">
Competing interests</div> <p id="Par41">The authors declare that they have no competing interests.</p> </div> <div class="availabilityofdataandmaterials" title="sec"> <div class="title" tagx="title" title="title">
Availability of data and materials</div> <p id="Par42">The dataset used and analyzed during the current study is available from the corresponding author on reasonable request.</p> </div> <div class="consentforpublications" title="sec"> <div class="title" tagx="title" title="title">
Consent for publications</div> <p id="Par43">Not applicable.</p> </div> <div class="ethicsapprovalandconsenttoparticipate" title="sec"> <div class="title" tagx="title" title="title">
Ethics approval and consent to participate</div> <p id="Par44">The study protocol was approved by the Ethics Review Board of Pudong Center for Disease Control and Prevention, Shanghai, China.</p> </div> <div class="funding" title="sec"> <div class="title" tagx="title" title="title">
Funding</div> <p id="Par45">This work was supported by Young Medical Talent Training Program of Health System of Pudong New Area, Shanghai (PWRq2016-17). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p> </div> <div class="publisherâ€™snote" title="sec"> <div class="title" tagx="title" title="title">
Publisherâ€™s Note</div> <p id="Par46">Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="CR1" /><span class="label" tagx="label" title="label">1.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Eurosurveillance editorial team Collective. WHO launches the World Health Statistics 2012. Euro Surveill. 2012;17(20):20175.</span></li> <li tag="ref"><a name="CR2" /><span class="label" tagx="label" title="label">2.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lozano</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naghavi</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Foreman</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Global and regional mortality from 235 causes of death for 20 age groups in 1990 and 2010: a systematic analysis for the Global Burden of Disease Study 2010</span><span class="source" tagx="source" title="source">Lancet</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">380</span><span class="fpage" tagx="fpage" title="fpage">2095</span><span class="lpage" tagx="lpage" title="lpage">2128</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0140-6736(12)61728-0">10.1016/S0140-6736(12)61728-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23245604">23245604</a></span></span></li> <li tag="ref"><a name="CR3" /><span class="label" tagx="label" title="label">3.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patel</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hall</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vinje</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Noroviruses: a comprehensive review</span><span class="source" tagx="source" title="source">J Clin Virol</span><span class="year" tagx="year" title="year">2009</span><span class="volume" tagx="volume" title="volume">44</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">8</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jcv.2008.10.009">10.1016/j.jcv.2008.10.009</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19084472">19084472</a></span></span></li> <li tag="ref"><a name="CR4" /><span class="label" tagx="label" title="label">4.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xue</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">An outbreak of acute norovirus gastroenteritis in a boarding school in Shanghai: a retrospective cohort study</span><span class="source" tagx="source" title="source">BMC Public Health</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">1092</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2458-14-1092">10.1186/1471-2458-14-1092</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25335780">25335780</a></span></span></li> <li tag="ref"><a name="CR5" /><span class="label" tagx="label" title="label">5.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cui</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pan</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">An outbreak of acute GII. 17 norovirus gastroenteritis in a long-term care facility in China: the role of nursing assistants</span><span class="source" tagx="source" title="source">J Infect Public Health</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">725</span><span class="lpage" tagx="lpage" title="lpage">729</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jiph.2016.10.007">10.1016/j.jiph.2016.10.007</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28196637">28196637</a></span></span></li> <li tag="ref"><a name="CR6" /><span class="label" tagx="label" title="label">6.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">La Rosa</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fontana</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Di Grazia</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular identification and genetic analysis of norovirus genogroups I and II in water environments: comparative analysis of different reverse transcription-PCR assays</span><span class="source" tagx="source" title="source">Appl Environ Microbiol</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">73</span><span class="fpage" tagx="fpage" title="fpage">4152</span><span class="lpage" tagx="lpage" title="lpage">4161</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/AEM.00222-07">10.1128/AEM.00222-07</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17483265">17483265</a></span></span></li> <li tag="ref"><a name="CR7" /><span class="label" tagx="label" title="label">7.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kroneman</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vennema</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Deforche</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">An automated genotyping tool for enteroviruses and noroviruses</span><span class="source" tagx="source" title="source">J Clin Virol</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">51</span><span class="fpage" tagx="fpage" title="fpage">121</span><span class="lpage" tagx="lpage" title="lpage">125</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jcv.2011.03.006">10.1016/j.jcv.2011.03.006</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21514213">21514213</a></span></span></li> <li tag="ref"><a name="CR8" /><span class="label" tagx="label" title="label">8.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vinje</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="mixed-article-title" title="mixed-article-title">Advances in laboratory methods for detection and typing of norovirus</span><span class="source" tagx="source" title="source">J Clin Microbiol</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">53</span><span class="fpage" tagx="fpage" title="fpage">373</span><span class="lpage" tagx="lpage" title="lpage">381</span><span class="pub-id"><a href="https://dx.doi.org/10.1128/JCM.01535-14">10.1128/JCM.01535-14</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24989606">24989606</a></span></span></li> <li tag="ref"><a name="CR9" /><span class="label" tagx="label" title="label">9.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hong-ying</span><span class="given-names" tagx="given-names" title="given-names">Z</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li-min</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wei</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular epidemiology of genogroup II noroviruses infection in outpatients with acute gastroenteritis in Nanjing, China (2010â€“2013)</span><span class="source" tagx="source" title="source">Biomed Res Int</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">2014</span><span class="fpage" tagx="fpage" title="fpage">620740</span><span class="pub-id"><a href="https://dx.doi.org/10.1155/2014/620740">10.1155/2014/620740</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25133173">25133173</a></span></span></li> <li tag="ref"><a name="CR10" /><span class="label" tagx="label" title="label">10.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ye</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lai</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Incidence of norovirus-associated diarrhea, Shanghai, China, 2012â€“2013</span><span class="source" tagx="source" title="source">Emerg Infect Dis</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">312</span><span class="lpage" tagx="lpage" title="lpage">315</span><span class="pub-id"><a href="https://dx.doi.org/10.3201/eid2302.161153">10.3201/eid2302.161153</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28098539">28098539</a></span></span></li> <li tag="ref"><a name="CR11" /><span class="label" tagx="label" title="label">11.</span><span class="element-citation'"><span class="person-group'"><span class="collab'">collab: World Health Organization</span></span><span class="source" tagx="source" title="source">The treatment of diarrohea: a manual for physicians and other senior health workers</span><span class="year" tagx="year" title="year">2005</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Geneva</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">World Health Organization</span></span></li> <li tag="ref"><a name="CR12" /><span class="label" tagx="label" title="label">12.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Management Office of National Science and Technology Major Project of China CCFD, prevention. Diarrheal Syndrome Surveillance Protocol (2012 version). 2012.</span></li> <li tag="ref"><a name="CR13" /><span class="label" tagx="label" title="label">13.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kibiki</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Maro</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Multiplex reverse transcription PCR Luminex assay for detection and quantitation of viral agents of gastroenteritis</span><span class="source" tagx="source" title="source">J Clin Virol</span><span class="year" tagx="year" title="year">2011</span><span class="volume" tagx="volume" title="volume">50</span><span class="fpage" tagx="fpage" title="fpage">308</span><span class="lpage" tagx="lpage" title="lpage">313</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.jcv.2010.12.009">10.1016/j.jcv.2010.12.009</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21256076">21256076</a></span></span></li> <li tag="ref"><a name="CR14" /><span class="label" tagx="label" title="label">14.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kojima</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kageyama</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fukushi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Genogroup-specific PCR primers for detection of Norwalk-like viruses</span><span class="source" tagx="source" title="source">J Virol Methods</span><span class="year" tagx="year" title="year">2002</span><span class="volume" tagx="volume" title="volume">100</span><span class="fpage" tagx="fpage" title="fpage">107</span><span class="lpage" tagx="lpage" title="lpage">114</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0166-0934(01)00404-9">10.1016/S0166-0934(01)00404-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11742657">11742657</a></span></span></li> <li tag="ref"><a name="CR15" /><span class="label" tagx="label" title="label">15.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hall</span><span class="given-names" tagx="given-names" title="given-names">TA</span></span></span><span class="mixed-article-title" title="mixed-article-title">BioEdit: a user-friendly biological sequence alignment editor and analysis program for Windows 95/98/NT</span><span class="source" tagx="source" title="source">Nucleic Acids Symp Ser</span><span class="year" tagx="year" title="year">1999</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">95</span><span class="lpage" tagx="lpage" title="lpage">98</span></span></li> <li tag="ref"><a name="CR16" /><span class="label" tagx="label" title="label">16.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stecher</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamura</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="mixed-article-title" title="mixed-article-title">MEGA7: molecular evolutionary genetics analysis version 7.0 for bigger datasets</span><span class="source" tagx="source" title="source">Mol Biol Evol</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">33</span><span class="fpage" tagx="fpage" title="fpage">1870</span><span class="lpage" tagx="lpage" title="lpage">1874</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/molbev/msw054">10.1093/molbev/msw054</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27004904">27004904</a></span></span></li> <li tag="ref"><a name="CR17" /><span class="label" tagx="label" title="label">17.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pan</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xue</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The novel norovirus genotype GII. 17 is the predominant strain in diarrheal patients in Shanghai, China</span><span class="source" tagx="source" title="source">Gut Pathog</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">8</span><span class="fpage" tagx="fpage" title="fpage">49</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s13099-016-0131-3">10.1186/s13099-016-0131-3</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27800029">27800029</a></span></span></li> <li tag="ref"><a name="CR18" /><span class="label" tagx="label" title="label">18.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patel</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Widdowson</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glass</span><span class="given-names" tagx="given-names" title="given-names">RI</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Systematic literature review of role of noroviruses in sporadic gastroenteritis</span><span class="source" tagx="source" title="source">Emerg Infect Dis</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">1224</span><span class="lpage" tagx="lpage" title="lpage">1231</span><span class="pub-id"><a href="https://dx.doi.org/10.3201/eid1408.071114">10.3201/eid1408.071114</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18680645">18680645</a></span></span></li> <li tag="ref"><a name="CR19" /><span class="label" tagx="label" title="label">19.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ahmed</span><span class="given-names" tagx="given-names" title="given-names">SM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hall</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Robinson</span><span class="given-names" tagx="given-names" title="given-names">AE</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Global prevalence of norovirus in cases of gastroenteritis: a systematic review and meta-analysis</span><span class="source" tagx="source" title="source">Lancet Infect Dis</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">725</span><span class="lpage" tagx="lpage" title="lpage">730</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S1473-3099(14)70767-4">10.1016/S1473-3099(14)70767-4</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24981041">24981041</a></span></span></li> <li tag="ref"><a name="CR20" /><span class="label" tagx="label" title="label">20.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cohen</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Colford</span><span class="given-names" tagx="given-names" title="given-names">JM</span></span></span><span class="mixed-article-title" title="mixed-article-title">Effects of boiling drinking water on diarrhea and pathogen-specific infections in low- and middle-income countries: a systematic review and meta-analysis</span><span class="source" tagx="source" title="source">Am J Trop Med Hyg</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">97</span><span class="fpage" tagx="fpage" title="fpage">1362</span><span class="lpage" tagx="lpage" title="lpage">1377</span><span class="pub-id"><a href="https://dx.doi.org/10.4269/ajtmh.17-0190">10.4269/ajtmh.17-0190</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29016318">29016318</a></span></span></li> <li tag="ref"><a name="CR21" /><span class="label" tagx="label" title="label">21.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xue</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pan</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Epidemiology of norovirus infections among diarrhea outpatients in a diarrhea surveillance system in Shanghai, China: a cross-sectional study</span><span class="source" tagx="source" title="source">BMC Infect Dis</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">15</span><span class="fpage" tagx="fpage" title="fpage">183</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12879-015-0922-z">10.1186/s12879-015-0922-z</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25884557">25884557</a></span></span></li> <li tag="ref"><a name="CR22" /><span class="label" tagx="label" title="label">22.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fang</span><span class="given-names" tagx="given-names" title="given-names">ZY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xie</span><span class="given-names" tagx="given-names" title="given-names">HP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lv</span><span class="given-names" tagx="given-names" title="given-names">HX</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The study on the infant diarrhea caused by human calicivirus in China, 1999â€“2005</span><span class="source" tagx="source" title="source">Chin J Virol</span><span class="year" tagx="year" title="year">2007</span><span class="volume" tagx="volume" title="volume">1</span><span class="fpage" tagx="fpage" title="fpage">9</span><span class="lpage" tagx="lpage" title="lpage">15</span></span></li> <li tag="ref"><a name="CR23" /><span class="label" tagx="label" title="label">23.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">SX</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yin</span><span class="given-names" tagx="given-names" title="given-names">JW</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Emergence of human caliciviruses among diarrhea cases insouthwest China</span><span class="source" tagx="source" title="source">BMC Infect Dis</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">511</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12879-016-1831-5">10.1186/s12879-016-1831-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27663519">27663519</a></span></span></li> <li tag="ref"><a name="CR24" /><span class="label" tagx="label" title="label">24.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kirby</span><span class="given-names" tagx="given-names" title="given-names">AE</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Streby</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moe</span><span class="given-names" tagx="given-names" title="given-names">CL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Vomiting as a symptom and transmission risk in norovirus illness: evidence from human challenge studies</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">11</span><span class="fpage" tagx="fpage" title="fpage">e143759</span></span></li> <li tag="ref"><a name="CR25" /><span class="label" tagx="label" title="label">25.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bresee</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marcus</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Venezia</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">The etiology of severe acute gastroenteritis among adults visiting emergency departments in the United States</span><span class="source" tagx="source" title="source">J Infect Dis</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">205</span><span class="fpage" tagx="fpage" title="fpage">1374</span><span class="lpage" tagx="lpage" title="lpage">1381</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/infdis/jis206">10.1093/infdis/jis206</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22454468">22454468</a></span></span></li> <li tag="ref"><a name="CR26" /><span class="label" tagx="label" title="label">26.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qian</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A novel norovirus GII. 17 lineage contributed to adult gastroenteritis in Shanghai, China, during the winter of 2014â€“2015</span><span class="source" tagx="source" title="source">Emerg Microbes Infect</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">e67</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/emi.2015.67">10.1038/emi.2015.67</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26975060">26975060</a></span></span></li> <li tag="ref"><a name="CR27" /><span class="label" tagx="label" title="label">27.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bull</span><span class="given-names" tagx="given-names" title="given-names">RA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eden</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rawlinson</span><span class="given-names" tagx="given-names" title="given-names">WD</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Rapid evolution of pandemic noroviruses of the GII. 4 lineage</span><span class="source" tagx="source" title="source">PLoS Pathog</span><span class="year" tagx="year" title="year">2010</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">e1000831</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1000831">10.1371/journal.ppat.1000831</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20360972">20360972</a></span></span></li> <li tag="ref"><a name="CR28" /><span class="label" tagx="label" title="label">28.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lindesmith</span><span class="given-names" tagx="given-names" title="given-names">LC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Donaldson</span><span class="given-names" tagx="given-names" title="given-names">EF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lobue</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Mechanisms of GII. 4 norovirus persistence in human populations</span><span class="source" tagx="source" title="source">PLoS Med</span><span class="year" tagx="year" title="year">2008</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">e31</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pmed.0050031">10.1371/journal.pmed.0050031</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18271619">18271619</a></span></span></li> <li tag="ref"><a name="CR29" /><span class="label" tagx="label" title="label">29.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Medici</span><span class="given-names" tagx="given-names" title="given-names">MC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tummolo</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Grazia</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Epidemiological dynamics of norovirus GII.4 variant New Orleans 2009</span><span class="source" tagx="source" title="source">J Gen Virol</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">96</span><span class="fpage" tagx="fpage" title="fpage">2919</span><span class="lpage" tagx="lpage" title="lpage">2927</span><span class="pub-id"><a href="https://dx.doi.org/10.1099/vir.0.000204">10.1099/vir.0.000204</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26025873">26025873</a></span></span></li> <li tag="ref"><a name="CR30" /><span class="label" tagx="label" title="label">30.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Manso</span><span class="given-names" tagx="given-names" title="given-names">CF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Romalde</span><span class="given-names" tagx="given-names" title="given-names">JL</span></span></span><span class="mixed-article-title" title="mixed-article-title">Molecular epidemiology of norovirus from patients with acute gastroenteritis in northwestern Spain</span><span class="source" tagx="source" title="source">Epidemiol Infect</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">143</span><span class="fpage" tagx="fpage" title="fpage">316</span><span class="lpage" tagx="lpage" title="lpage">324</span><span class="pub-id"><a href="https://dx.doi.org/10.1017/S0950268814000740">10.1017/S0950268814000740</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24739188">24739188</a></span></span></li> <li tag="ref"><a name="CR31" /><span class="label" tagx="label" title="label">31.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jung</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hwang</span><span class="given-names" tagx="given-names" title="given-names">BM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jeong</span><span class="given-names" tagx="given-names" title="given-names">HJ</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Occurrence of norovirus GII. 4 Sydney variant-related outbreaks in Korea</span><span class="source" tagx="source" title="source">Osong Public Health Res Perspect</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">322</span><span class="lpage" tagx="lpage" title="lpage">326</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/j.phrp.2015.10.004">10.1016/j.phrp.2015.10.004</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26929914">26929914</a></span></span></li> <li tag="ref"><a name="CR32" /><span class="label" tagx="label" title="label">32.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Polkowska</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ronnqvist</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lepisto</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Outbreak of gastroenteritis caused by norovirus GII. 4 Sydney variant after a wedding reception at a resort/activity centre, Finland, August 2012</span><span class="source" tagx="source" title="source">Epidemiol Infect</span><span class="year" tagx="year" title="year">2014</span><span class="volume" tagx="volume" title="volume">142</span><span class="fpage" tagx="fpage" title="fpage">1877</span><span class="lpage" tagx="lpage" title="lpage">1883</span><span class="pub-id"><a href="https://dx.doi.org/10.1017/S0950268813002847">10.1017/S0950268813002847</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24229743">24229743</a></span></span></li> <li tag="ref"><a name="CR33" /><span class="label" tagx="label" title="label">33.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van Beek</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Graaf</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xia</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Comparison of norovirus genogroup I, II and IV seroprevalence among children in the Netherlands, 1963, 1983 and 2006</span><span class="source" tagx="source" title="source">J Gen Virol</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">97</span><span class="fpage" tagx="fpage" title="fpage">2255</span><span class="lpage" tagx="lpage" title="lpage">2264</span><span class="pub-id"><a href="https://dx.doi.org/10.1099/jgv.0.000533">10.1099/jgv.0.000533</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27365054">27365054</a></span></span></li> <li tag="ref"><a name="CR34" /><span class="label" tagx="label" title="label">34.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Desai</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hembree</span><span class="given-names" tagx="given-names" title="given-names">CD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Handel</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Severe outcomes are associated with genogroup 2 genotype 4 norovirus outbreaks: a systematic literature review</span><span class="source" tagx="source" title="source">Clin Infect Dis</span><span class="year" tagx="year" title="year">2012</span><span class="volume" tagx="volume" title="volume">55</span><span class="fpage" tagx="fpage" title="fpage">189</span><span class="lpage" tagx="lpage" title="lpage">193</span><span class="pub-id"><a href="https://dx.doi.org/10.1093/cid/cis372">10.1093/cid/cis372</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22491335">22491335</a></span></span></li> <li tag="ref"><a name="CR35" /><span class="label" tagx="label" title="label">35.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Han</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ji</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shen</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Emergence and predominance of norovirus GII. 17 in Huzhou, China, 2014â€“2015</span><span class="source" tagx="source" title="source">Virol J</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">12</span><span class="fpage" tagx="fpage" title="fpage">139</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s12985-015-0370-9">10.1186/s12985-015-0370-9</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26362650">26362650</a></span></span></li> <li tag="ref"><a name="CR36" /><span class="label" tagx="label" title="label">36.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ai</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jin</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Emergence of a new GII.17 norovirus variant in patients with acute gastroenteritis in Jiangsu, China, September 2014 to March 2015</span><span class="source" tagx="source" title="source">Euro Surveill</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">21157</span><span class="pub-id"><a href="https://dx.doi.org/10.2807/1560-7917.ES2015.20.24.21157">10.2807/1560-7917.ES2015.20.24.21157</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26111236">26111236</a></span></span></li> <li tag="ref"><a name="CR37" /><span class="label" tagx="label" title="label">37.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Matsushima</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ishikawa</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shimizu</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Genetic analyses of GII. 17 norovirus strains in diarrheal disease outbreaks from December 2014 to March 2015 in Japan reveal a novel polymerase sequence and amino acid substitutions in the capsid region</span><span class="source" tagx="source" title="source">Euro Surveill</span><span class="year" tagx="year" title="year">2015</span><span class="volume" tagx="volume" title="volume">20</span><span class="fpage" tagx="fpage" title="fpage">21173</span><span class="pub-id"><a href="https://dx.doi.org/10.2807/1560-7917.ES2015.20.26.21173">10.2807/1560-7917.ES2015.20.26.21173</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26159307">26159307</a></span></span></li> <li tag="ref"><a name="CR38" /><span class="label" tagx="label" title="label">38.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanchez</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Corcostegui</span><span class="given-names" tagx="given-names" title="given-names">SP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Broucker</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Norovirus GII. 17 outbreak linked to an infected post-symptomatic food worker in a French military unit located in France</span><span class="source" tagx="source" title="source">Food Environ Virol</span><span class="year" tagx="year" title="year">2016</span><span class="volume" tagx="volume" title="volume">9</span><span class="fpage" tagx="fpage" title="fpage">234</span><span class="lpage" tagx="lpage" title="lpage">237</span><span class="pub-id"><a href="https://dx.doi.org/10.1007/s12560-016-9274-0">10.1007/s12560-016-9274-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27909946">27909946</a></span></span></li> <li tag="ref"><a name="CR39" /><span class="label" tagx="label" title="label">39.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">XY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Su</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lu</span><span class="given-names" tagx="given-names" title="given-names">QC</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">A large outbreak of acute gastroenteritis caused by the human norovirus GII. 17 strain at a university in Henan Province, China</span><span class="source" tagx="source" title="source">Infect Dis Poverty</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">6</span><span class="fpage" tagx="fpage" title="fpage">6</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/s40249-017-0236-z">10.1186/s40249-017-0236-z</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28143569">28143569</a></span></span></li> <li tag="ref"><a name="CR40" /><span class="label" tagx="label" title="label">40.</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ao</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ling</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="mixed-article-title" title="mixed-article-title">Norovirus GII. P16/GII. 2-associated gastroenteritis, China, 2016</span><span class="source" tagx="source" title="source">Emerg Infect Dis</span><span class="year" tagx="year" title="year">2017</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">1172</span><span class="lpage" tagx="lpage" title="lpage">1175</span><span class="pub-id"><a href="https://dx.doi.org/10.3201/eid2307.170034">10.3201/eid2307.170034</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28430563">28430563</a></span></span></li> <li tag="ref"><a name="CR41" /><span class="label" tagx="label" title="label">41.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation">Niendorf S, Jacobsen S, Faber M, Eis-HÃ¼binger A M, Hofmann J, Zimmermann O, HÃ¶hne M, Bock C T. Steep rise in norovirus cases and emergence of a new recombinant strain GII. P16-GII. 2, Germany, winter 2016. Euro Surveill. 2017;22(4):30447.</span></li> </ul> </div> </div>  </body></html>