<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2017/03/07/114819.atom" />
<meta name="HW.pisa" content="biorxiv;114819v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Prophage genomics reveals patterns in phage genome organization and replication" />
<meta name="DC.Identifier" content="10.1101/114819" />
<meta name="DC.Date" content="2017-03-07" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2017, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Temperate phage genomes are highly variable mosaic collections of genes that infect a bacterial host, integrate into the host’s genome or replicate as low copy number plasmids, and are regulated to switch from the lysogenic to lytic cycles to generate new virions and escape their host. Genomes from most Bacterial phyla contain at least one or more prophages. We updated our PhiSpy algorithm to improve detection of prophages and to provide a web-based framework for PhiSpy. We have used this algorithm to identify 36,488 prophage regions from 11,941 bacterial genomes, including almost 600 prophages with no known homology to any proteins. Transfer RNA genes were abundant in the prophages, many of which alleviate the limits of translation efficiency due to host codon bias and presumably enable phages to surpass the normal capacity of the hosts’ translation machinery. We identified integrase genes in 15,765 prophages (43% of the prophages). The integrase was routinely located at either end of the integrated phage genome, and was used to orient and align prophage genomes to reveal their underlying organization. The conserved genome alignments of phages recapitulate early, middle, and late gene order in transcriptional control of phage genes, and demonstrate that gene order, presumably selected by transcription timing and/or coordination among functional modules has been stably conserved throughout phage evolution." />
<meta name="DC.Contributor" content="Han Suh Kang" />
<meta name="DC.Contributor" content="Katelyn McNair" />
<meta name="DC.Contributor" content="Daniel A. Cuevas" />
<meta name="DC.Contributor" content="Barbara A. Bailey" />
<meta name="DC.Contributor" content="Anca M. Segall" />
<meta name="DC.Contributor" content="Robert A. Edwards" />
<meta name="article:published_time" content="2017-03-07" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Prophage genomics reveals patterns in phage genome organization and replication" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Temperate phage genomes are highly variable mosaic collections of genes that infect a bacterial host, integrate into the host’s genome or replicate as low copy number plasmids, and are regulated to switch from the lysogenic to lytic cycles to generate new virions and escape their host. Genomes from most Bacterial phyla contain at least one or more prophages. We updated our PhiSpy algorithm to improve detection of prophages and to provide a web-based framework for PhiSpy. We have used this algorithm to identify 36,488 prophage regions from 11,941 bacterial genomes, including almost 600 prophages with no known homology to any proteins. Transfer RNA genes were abundant in the prophages, many of which alleviate the limits of translation efficiency due to host codon bias and presumably enable phages to surpass the normal capacity of the hosts’ translation machinery. We identified integrase genes in 15,765 prophages (43% of the prophages). The integrase was routinely located at either end of the integrated phage genome, and was used to orient and align prophage genomes to reveal their underlying organization. The conserved genome alignments of phages recapitulate early, middle, and late gene order in transcriptional control of phage genes, and demonstrate that gene order, presumably selected by transcription timing and/or coordination among functional modules has been stably conserved throughout phage evolution.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2017/01/01" />
<meta name="citation_mjid" content="biorxiv;114819v1" />
<meta name="citation_id" content="114819v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/114819v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/114819v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/114819v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819.full.pdf" />
<meta name="citation_doi" content="10.1101/114819" />
<meta name="citation_num_pages" content="28" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="114819" />
<meta name="citation_author" content="Han Suh Kang" />
<meta name="citation_author_institution" content="Department of Biology, San Diego State University" />
<meta name="citation_author" content="Katelyn McNair" />
<meta name="citation_author_institution" content="Department of Computer Science, San Diego State University" />
<meta name="citation_author" content="Daniel A. Cuevas" />
<meta name="citation_author_institution" content="Computational Sciences Research Center, San Diego State University" />
<meta name="citation_author" content="Barbara A. Bailey" />
<meta name="citation_author_institution" content="Department of Mathematics and Statistics, San Diego State University" />
<meta name="citation_author" content="Anca M. Segall" />
<meta name="citation_author_institution" content="Department of Biology, San Diego State University" />
<meta name="citation_author" content="Robert A. Edwards" />
<meta name="citation_author_institution" content="Department of Biology, San Diego State University" />
<meta name="citation_author_institution" content="Department of Computer Science, San Diego State University" />
<meta name="citation_author_institution" content="Computational Sciences Research Center, San Diego State University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8383-8949" />
<meta name="citation_reference" content="Prentki,P., Chandler,M. and Caro,L. (1977) Replication of the prophage P1 during the cell cycle of Escherichia coli. Mol. Gen. Genet. MGG, 152, 71–76." />
<meta name="citation_reference" content="Campbell,A.M. (1992) Chromosomal insertion sites for phages and plasmids. J. Bacteriol., 174, 7495–7499." />
<meta name="citation_reference" content="Campbell,A. (2007) Phage integration and chromosome structure. A personal history. Annu. Rev. Genet., 41, 1–11." />
<meta name="citation_reference" content="Maloy,S.R., Stewart,V.J. and Taylor,R.K. (1996) Genetic Analysis of Pathogenic Bacteria: A Laboratory Manual Cold Spring Harbor Laboratory Press." />
<meta name="citation_reference" content="Brüssow,H., Canchaya,C. and Hardt,W.-D. (2004) Phages and the evolution of bacterial pathogens: from genomic rearrangements to lysogenic conversion. Microbiol. Mol. Biol. Rev. MMBR, 68, 560–602, table of contents." />
<meta name="citation_reference" content="Bushman,F. (2001) Lateral DNA Transfer: Mechanisms and Consequences CSHL Press, New York, NY." />
<meta name="citation_reference" content="Heidelberg,J.F., Eisen,J.A., Nelson,W.C., Clayton,R.A., Gwinn,M.L., Dodson,R.J., Haft,D.H., Hickey,E.K., Peterson,J.D., Umayam,L., et al. (2000) DNA sequence of both chromosomes of the cholera pathogen Vibrio cholerae. Nature, 406, 477–83." />
<meta name="citation_reference" content="citation_title=Streptococcus pyogenes: Basic Biology to Clinical Manifestations;citation_year=2016" />
<meta name="citation_reference" content="Canchaya,C., Fournous,G., Chibani-Chennoufi,S., Dillmann,M.L. and Brüssow,H. (2003) Phage as agents of lateral gene transfer. Curr. Opin. Microbiol., 6, 417–424." />
<meta name="citation_reference" content="Modi,S.R., Lee,H.H., Spina,C.S. and Collins,J.J. (2013) Antibiotic treatment expands the resistance reservoir and ecological network of the phage metagenome. Nature, 499, 219–222." />
<meta name="citation_reference" content="Mann,N.H., Cook,A., Millard,A., Bailey,S. and Clokie,M. (2003) Marine ecosystems: Bacterial photosynthesis genes in a virus. Nature, 424, 741–741." />
<meta name="citation_reference" content="Knowles,B., Silveira,C.B., Bailey,B.A., Barott,K., Cantu,V.A., Cobián-Güemes,A.G., Coutinho,F.H., Dinsdale,E.A., Felts,B., Furby,K.A., et al. (2016) Lytic to temperate switching of viral communities. Nature, 531, 466–470." />
<meta name="citation_reference" content="Hatfull,G.F. (2008) Bacteriophage genomics. Curr. Opin. Microbiol., 11, 447–453." />
<meta name="citation_reference" content="Aziz,R.K., Bartels,D., Best,A.A., DeJongh,M., Disz,T., Edwards,R.A., Formsma,K., Gerdes,S., Glass,E.M., Kubal,M., et al. (2008) The RAST Server: rapid annotations using subsystems technology. BMC Genomics, 9, 75." />
<meta name="citation_reference" content="Overbeek,R., Olson,R., Pusch,G.D., Olsen,G.J., Davis,J.J., Disz,T., Edwards,R.A., Gerdes,S., Parrello,B., Shukla,M., et al. (2014) The SEED and the Rapid Annotation of microbial genomes using Subsystems Technology (RAST). Nucleic Acids Res., 42, D206–214." />
<meta name="citation_reference" content="Zhou,Y., Liang,Y., Lynch,K.H., Dennis,J.J. and Wishart,D.S. (2011) PHAST: A Fast Phage Search Tool. Nucleic Acids Res., 39, W347–W352." />
<meta name="citation_reference" content="Bose,M. and Barber,R.D. (2006) Prophage Finder: a prophage loci prediction tool for prokaryotic genome sequences. Silico Biol, 6, 223–7." />
<meta name="citation_reference" content="Fouts,D.E. (2006) Phage_Finder: automated identification and classification of prophage regions in complete bacterial genome sequences. Nucleic Acids Res, 34, 5839–51." />
<meta name="citation_reference" content="Lima-Mendez,G., Van Helden,J., Toussaint,A. and Leplae,R. (2008) Prophinder: a computational tool for prophage prediction in prokaryotic genomes. Bioinforma. Oxf Engl., 24, 863–865." />
<meta name="citation_reference" content="Roux,S., Enault,F., Hurwitz,B.L. and Sullivan,M.B. (2015) VirSorter: mining viral signal from microbial genomic data. PeerJ, 3, e985." />
<meta name="citation_reference" content="Akhter,S., Aziz,R.K. and Edwards,R.A. (2012) PhiSpy: a novel algorithm for finding prophages in bacterial genomes that combines similarity-and composition-based strategies. Nucleic Acids Res., 40, e126–e126." />
<meta name="citation_reference" content="Lowe,T.M. and Eddy,S.R. (1997) tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence. Nucleic Acids Res, 25, 955–64." />
<meta name="citation_reference" content="Davis,J.J. and Olsen,G.J. (2010) Modal Codon Usage: Assessing the Typical Codon Usage of a Genome. Mol. Biol. Evol., 27, 800–810." />
<meta name="citation_reference" content="Huang,Y., Niu,B., Gao,Y., Fu,L. and Li,W. (2010) CD-HIT Suite: a web server for clustering and comparing biological sequences. Bioinformatics, 26, 680–2." />
<meta name="citation_reference" content="Sievers,F., Wilm,A., Dineen,D., Gibson,T.J., Karplus,K., Li,W., Lopez,R., McWilliam,H., Remmert,M., Söding,J., et al. (2011) Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega. Mol. Syst. Biol., 7, 539." />
<meta name="citation_reference" content="Goujon,M., McWilliam,H., Li,W., Valentin,F., Squizzato,S., Paern,J. and Lopez,R. (2010) A new bioinformatics analysis tools framework at EMBL-EBI. Nucleic Acids Res., 38, W695–699." />
<meta name="citation_reference" content="Holmes,E.C. and Rambaut,A. (2004) Viral evolution and the emergence of SARS coronavirus. Philos. Trans. R. Soc. B Biol. Sci., 359, 1059–1065." />
<meta name="citation_reference" content="NCBI Resource Coordinators (2016) Database resources of the National Center for Biotechnology Information. Nucleic Acids Res., 44, D7–19." />
<meta name="citation_reference" content="Sanchez,S.E., Cuevas,D.A., Rostron,J.E., Liang,T.Y., Pivaroff,C.G., Haynes,M.R., Nulton,J., Felts,B., Bailey,B.A., Salamon,P., et al. (2015) Phage Phenomics: Physiological Approaches to Characterize Novel Viral Proteins. J. Vis. Exp., 10.3791/52854." />
<meta name="citation_reference" content="Seguritan,V., Feng,I.-W., Rohwer,F., Swift,M. and Segall,A.M. (2003) Genome sequences of two closely related Vibrio parahaemolyticus phages, VP16T and VP16C. J. Bacteriol., 185, 6434–6447." />
<meta name="citation_reference" content="Rohwer,F., Segall,A., Steward,G., Seguritan,V., Breitbart,M., Wolven,F. and Azam,F. (2000) The complete genomic sequence of the marine phage Roseophage SIOI shares homology with nonmarine phages. Limnol. Oceanogr., 45, 408–418." />
<meta name="citation_reference" content="Hendrix,R.W., Smith,M.C.M., Burns,R.N., Ford,M.E. and Hatfull,G.F. (1999) Evolutionary relationships among diverse bacteriophages and prophages: All the world’s a phage. Proc. Natl. Acad. Sci., 96, 2192–2197." />
<meta name="citation_reference" content="citation_title=Structure and Physics of Viruses, Subcellular Biochemistry;citation_pages=553-595;citation_year=2013" />
<meta name="citation_reference" content="Roux,S., Hallam,S.J., Woyke,T. and Sullivan,M.B. (2015) Viral dark matter and virus-host interactions resolved from publicly available microbial genomes. eLife, 4, e08490." />
<meta name="citation_reference" content="Edwards,R.A., McNair,K., Faust,K., Raes,J. and Dutilh,B.E. (2015) Computational approaches to predict bacteriophage-host relationships. FEMS Microbiol. Rev., 10.1093/femsre/fuv048." />
<meta name="citation_reference" content="Rajeev,L., Malanowska,K. and Gardner,J.F. (2009) Challenging a paradigm: the role of DNA homology in tyrosine recombinase reactions. Microbiol. Mol. Biol. Rev. MMBR, 73, 300–309." />
<meta name="citation_reference" content="Hirano,N., Muroi,T., Takahashi,H. and Haruki,M. (2011) Site-specific recombinases as tools for heterologous gene integration. Appl. Microbiol. Biotechnol., 92, 227–239." />
<meta name="citation_reference" content="Das,B., Martinez,E., Midonet,C. and Barre,F.-X. (2013) Integrative mobile elements exploiting Xer recombination. Trends Microbiol., 21, 23–30." />
<meta name="citation_reference" content="Askora,A., Abdel-Haliem,M.E.F. and Yamada,T. (2012) Site-specific recombination systems in filamentous phages. Mol. Genet. Genomics, 287, 525–530." />
<meta name="citation_reference" content="Krupovic,M. and Forterre,P. (2015) Single-stranded DNA viruses employ a variety of mechanisms for integration into host genomes. Ann. N. Y. Acad. Sci., 1341, 41–53." />
<meta name="citation_reference" content="Jayaram,M., Ma,C.-H., Kachroo,A.H., Rowley,P.A., Guga,P., Fan,H.-F. and Voziyanov,Y. (2015) An Overview of Tyrosine Site-specific Recombination: From an Flp Perspective. Microbiol. Spectr., 3." />
<meta name="citation_reference" content="Smith,M.C.M., Brown,W.R.A., McEwan,A.R. and Rowley,P.A. (2010) Site-specific recombination by phiC31 integrase and other large serine recombinases. Biochem. Soc. Trans., 38, 388–394." />
<meta name="citation_reference" content="Ghosh,P., Kim,A.I. and Hatfull,G.F. (2003) The orientation of mycobacteriophage Bxb1 integration is solely dependent on the central dinucleotide of attP and attB. Mol. Cell, 12, 1101–1111." />
<meta name="citation_reference" content="Argos,P., Landy,A., Abremski,K., Egan,J.B., Haggard-Ljungquist,E., Hoess,R.H., Kahn,M.L., Kalionis,B., Narayana,S.V. and Pierson,L.S. (1986) The integrase family of site-specific recombinases: regional similarities and global diversity. EMBO J., 5, 433–440." />
<meta name="citation_reference" content="Esposito,D. and ScoccaJ.J. (1997) The integrase family of tyrosine recombinases: evolution of a conserved active site domain. Nucleic Acids Res., 25, 3605–3614." />
<meta name="citation_reference" content="Nunes-Düby,S.E., Kwon,H.J., Tirumalai,R.S., Ellenberger,T. and Landy,A. (1998) Similarities and differences among 105 members of the Int family of site-specific recombinases. Nucleic Acids Res., 26, 391–406." />
<meta name="citation_reference" content="WilsonJ.H. (1973) Function of the bacteriophage T4 transfer RNA’s. J. Mol. Biol., 74, 753–757." />
<meta name="citation_reference" content="Weiss,S.B., Hsu,W.T., FoftJ.W. and Scherberg,N.H. (1968) Transfer RNA coded by the T4 bacteriophage genome. Proc. Natl. Acad. Sci. U. S. A., 61, 114–121." />
<meta name="citation_reference" content="Kunisawa,T. (1992) Synonymous codon preferences in bacteriophage T4: a distinctive use of transfer RNAs from T4 and from its host Escherichia coli. J. Theor. Biol., 159, 287–298." />
<meta name="citation_reference" content="Bailly-Bechet,M., Vergassola,M. and Rocha,E. (2007) Causes for the intriguing presence of tRNAs in phages. Genome Res., 17, 1486–1495." />
<meta name="citation_reference" content="Canchaya,C., Fournous,G. and Brüssow,H. (2004) The impact of prophages on bacterial chromosomes. Mol. Microbiol., 53, 9–18." />
<meta name="citation_reference" content="Casjens,S. (2003) Prophages and bacterial genomics: what have we learned so far? Mol Microbiol, 49, 277–300." />
<meta name="citation_reference" content="Frank,J.A., Lorimer,D., Youle,M., Witte,P., Craig,T., Abendroth,J., Rohwer,F., Edwards,R.A., Segall,A.M. and Burgin,A.B. (2013) Structure and function of a cyanophage-encoded peptide deformylase. ISME J., 7, 1150–1160." />
<meta name="citation_reference" content="Sharon,I., Battchikova,N., Aro,E.-M., Giglione,C., Meinnel,T., Glaser,F., Pinter,R.Y., Breitbart,M., Rohwer,F. and Béjà,O. (2011) Comparative metagenomics of microbial traits within oceanic viral communities. ISME J., 5, 1178–1190." />
<meta name="citation_reference" content="Muniesa,M., Recktenwald,J., Bielaszewska,M., Karch,H. and Schmidt,H. (2000) Characterization of a shiga toxin 2e-converting bacteriophage from an Escherichia coli strain of human origin. Infect. Immun., 68, 4850–4855." />
<meta name="citation_reference" content="Pope,W.H., Bowman,C.A., Russell,D.A., Jacobs-Sera,D., Asai,D.J., Cresawn,S.G., Jr,W.R.J., Hendrix,R.W., Lawrence,J.G., Hatfull,G.F., et al. (2015) Whole genome comparison of a large collection of mycobacteriophages reveals a continuum of phage genetic diversity. eLife, 4, e06416." />
<meta name="citation_reference" content="Miller,E.S., Kutter,E., Mosig,G., Arisaka,F., Kunisawa,T. and Ruger,W. (2003) Bacteriophage T4 Genome. Microbiol. Mol. Biol. Rev., 67, 86–156." />
<meta name="citation_reference" content="Marrs,C.F. and Howe,M.M. (1990) Kinetics and regulation of transcription of bacteriophage Mu. Virology, 174, 192–203." />
<meta name="citation_reference" content="Pero,J., Hannett,N.M. and Talkington,C. (1979) Restriction cleavage map of SP01 DNA: general location of early, middle, and late genes. J. Virol., 31, 156–171." />
<meta name="citation_reference" content="Ventura,M. and Brüssow,H. (2004) Temporal Transcription Map of the Virulent Streptococcus thermophilus Bacteriophage Sfi19. Appl. Environ. Microbiol., 70, 5041–5046." />
<meta name="twitter:title" content="Prophage genomics reveals patterns in phage genome organization and replication" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Temperate phage genomes are highly variable mosaic collections of genes that infect a bacterial host, integrate into the host’s genome or replicate as low copy number plasmids, and are regulated to switch from the lysogenic to lytic cycles to generate new virions and escape their host. Genomes from most Bacterial phyla contain at least one or more prophages. We updated our PhiSpy algorithm to improve detection of prophages and to provide a web-based framework for PhiSpy. We have used this algorithm to identify 36,488 prophage regions from 11,941 bacterial genomes, including almost 600 prophages with no known homology to any proteins. Transfer RNA genes were abundant in the prophages, many of which alleviate the limits of translation efficiency due to host codon bias and presumably enable phages to surpass the normal capacity of the hosts’ translation machinery. We identified integrase genes in 15,765 prophages (43% of the prophages). The integrase was routinely located at either end of the integrated phage genome, and was used to orient and align prophage genomes to reveal their underlying organization. The conserved genome alignments of phages recapitulate early, middle, and late gene order in transcriptional control of phage genes, and demonstrate that gene order, presumably selected by transcription timing and/or coordination among functional modules has been stably conserved throughout phage evolution." />
<meta name="og-title" property="og:title" content="Prophage genomics reveals patterns in phage genome organization and replication" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/114819v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Temperate phage genomes are highly variable mosaic collections of genes that infect a bacterial host, integrate into the host’s genome or replicate as low copy number plasmids, and are regulated to switch from the lysogenic to lytic cycles to generate new virions and escape their host. Genomes from most Bacterial phyla contain at least one or more prophages. We updated our PhiSpy algorithm to improve detection of prophages and to provide a web-based framework for PhiSpy. We have used this algorithm to identify 36,488 prophage regions from 11,941 bacterial genomes, including almost 600 prophages with no known homology to any proteins. Transfer RNA genes were abundant in the prophages, many of which alleviate the limits of translation efficiency due to host codon bias and presumably enable phages to surpass the normal capacity of the hosts’ translation machinery. We identified integrase genes in 15,765 prophages (43% of the prophages). The integrase was routinely located at either end of the integrated phage genome, and was used to orient and align prophage genomes to reveal their underlying organization. The conserved genome alignments of phages recapitulate early, middle, and late gene order in transcriptional control of phage genes, and demonstrate that gene order, presumably selected by transcription timing and/or coordination among functional modules has been stably conserved throughout phage evolution." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2017-03-07" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/114819v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/114819v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/114819v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/114819v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/33254" />
    <title>Prophage genomics reveals patterns in phage genome organization and replication | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"pqsdYBlQ-SfcOEH68lv55KZ2NUG5UbxXSNfwvbxtQ_k","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"33254","apath":"\/biorxiv\/early\/2017\/03\/07\/114819.atom","pisa":"biorxiv;114819v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;114819v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/114819v1","title":"Prophage genomics reveals patterns in phage genome organization and replication","identifier":"node\/33254"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022fRSdhVPeQ8ejHIQPJmR39XaJY3q1XlzxUrksKZvaNnTa5H2JF2FHwjp\\\\\\\/\\\\\\\/uMko5xBLCzeOPuuy\\\\\\\/XJWkStmQ1Akt2VTdQVt3ZIew\\\\\\\/nItKWLFxSHOm3COt87dg3Rb+gY6imPItAoFQE80EI2DWpQZ+Rw2YyxBWRU2Pmumyk2HeLqFoaWtUDYeBAdMXFTngFfaiIZV4NnC9CqRD47pWk2IATbYK5AssT+o1mexGZ02NokGEr2d1AViB\\\\\\\/JTLYs26Lzt3LrmaMZPDrTpwgEjivpRRY7S6bZAfdjF6GgCfMsPAB8DIFh5tf319jxvNnj7qBuOghrdHp95NfF1r0SixejU9874+06A\\\\\\\/X5PRvg2kEAUVbsDZ2n9yDnmqJsdUq8S91xui8Lk++rnJANvtFlxm3MBOJppJHqQcXh5mEr8PRGXJ5Dy0QR+DKfO9JU1n2JSUX\\\\\\\/1A406SZYvLTyUk0r+d83XS6rLFkXohSmJDwgTCfLX2VstCcuTzJvszBIWxf+8uq5i3brbmOK+2fBv2zKOR\\\\\\\/g\\\\\\\/CTfKLZKPLkZbZY0iEeTVQD5faJkHnGKM2HJZC31KwoPzPtY69+cZS350hkeWx7eqiokUeomj34oA8KAb1Lccwe8KKm2ClkRjfB1QlpgWhTX\\\\\\\/DusQi3DikFZXYnnJcrcja8kWJJOtyWQPoza\\\\\\\/QyqL8Yc0OQpcODWoRJFpouR3oVfXiqPtMwMoYlB\\\\\\\/BoDFm+IKlF\\\\\\\/cC45eHbIkVWFDDrL6icStgrdXTDc8R6sY9ZHQWwiyCKxGEHSzcc8fEthvlAxR466TotqLcoO\\\\\\\/XdQTo04lGugpIIM\\\\\\\/hSAgbbRRTjTL9Dx1AouLzH0v1wQGURt0TJGCPEfTNpblmykk7KZeF5gy6bIzcQxSHRDBi4IcFGsYiWaeD4+ujVbdisXZGEbGxT33RxGWBG2tOv3zIYR5XKyAcgZEf+56gc4f+QDZJe4+mYTSBplfYVaNGEb5wJYYGNfknUcL\\\\\\\/aGZgp72zTVHuaPkF6OqroZE2mTxk8XROZJCvu9DP+rEbgeHM7almwu\\\\\\\/qOghoeUizWIGG2r67D6eN0P6GJhfkpoZME5B5cxWaJS3et+xX6g68CoIpbSrPRYYJlbAOO0NlhuTg9TPBsRZXWeVNUfvqFEvdrHhOgW5r05gFD5sW+G2L7BcJ9bFN7Pcd59cWHbo3XPbN8OpFJLEvLjoBxYpBKI7D9I8eguaEZynxebYvXyHnAJyPlBQnzGMvbfdBOk\\\\\\\/O4mUbmdVxYXiFOMIZaSb90ShilVyAmcRg18vV60PVyoH\\\\\\\/x4udLzPW++8CsIKyL0T4vvHa+e42ypno6uq\\\\\\\/AlF4=\\\u0022,\\\u0022iv\\\u0022:\\\u0022mEOXgdohVrREg1PXZJN\\\\\\\/dw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002285457e07cd6b0d8282f1862a26797972469ad334499b9b770e2bb20c32b53507\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/114819v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-33254 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/114819v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Mhyhqm5oeX6AI1tUmoIO98RmZzUbHiax3TBk4TUEYDk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="33254" id="node33254" data-pisa="biorxiv;114819v1" data-pisa-master="biorxiv;114819" data-apath="/biorxiv/early/2017/03/07/114819.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Prophage genomics reveals patterns in phage genome organization and replication</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Han Suh</span> <span class="nlm-surname">Kang</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Katelyn</span> <span class="nlm-surname">McNair</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Daniel A.</span> <span class="nlm-surname">Cuevas</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Barbara A.</span> <span class="nlm-surname">Bailey</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Anca M.</span> <span class="nlm-surname">Segall</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-8383-8949" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Robert A.</span> <span class="nlm-surname">Edwards</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/114819 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node33254" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Han Suh Kang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Biology, San Diego State University</span>, 5500 Campanile Dr, San Diego, CA 92182</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Han%2BSuh%2BKang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kang%20HS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHan%2BSuh%2BKang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Katelyn McNair </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Computer Science, San Diego State University</span>, 5500 Campanile Dr, San Diego, CA 92182</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katelyn%2BMcNair%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McNair%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKatelyn%2BMcNair%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daniel A. Cuevas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Computational Sciences Research Center, San Diego State University</span>, 5500 Campanile Dr, San Diego, CA 92182</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BA.%2BCuevas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cuevas%20DA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BA.%2BCuevas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Barbara A. Bailey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Department of Mathematics and Statistics, San Diego State University</span>, 5500 Campanile Dr, San Diego, CA 92182</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Barbara%2BA.%2BBailey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bailey%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABarbara%2BA.%2BBailey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Anca M. Segall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Biology, San Diego State University</span>, 5500 Campanile Dr, San Diego, CA 92182</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anca%2BM.%2BSegall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Segall%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnca%2BM.%2BSegall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Robert A. Edwards </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Biology, San Diego State University</span>, 5500 Campanile Dr, San Diego, CA 92182</div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Computer Science, San Diego State University</span>, 5500 Campanile Dr, San Diego, CA 92182</div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Computational Sciences Research Center, San Diego State University</span>, 5500 Campanile Dr, San Diego, CA 92182</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BA.%2BEdwards%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Edwards%20RA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobert%2BA.%2BEdwards%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8383-8949" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Robert A. Edwards</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/114819v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:33254" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:33254/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/114819v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:33254" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:33254/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/114819v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:33254" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:33254/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/114819v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:33254" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:33254/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/114819v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:33254" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:33254/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Temperate phage genomes are highly variable mosaic collections of genes that infect a bacterial host, integrate into the host’s genome or replicate as low copy number plasmids, and are regulated to switch from the lysogenic to lytic cycles to generate new virions and escape their host. Genomes from most Bacterial phyla contain at least one or more prophages. We updated our PhiSpy algorithm to improve detection of prophages and to provide a web-based framework for PhiSpy. We have used this algorithm to identify 36,488 prophage regions from 11,941 bacterial genomes, including almost 600 prophages with no known homology to any proteins. Transfer RNA genes were abundant in the prophages, many of which alleviate the limits of translation efficiency due to host codon bias and presumably enable phages to surpass the normal capacity of the hosts’ translation machinery. We identified integrase genes in 15,765 prophages (43% of the prophages). The integrase was routinely located at either end of the integrated phage genome, and was used to orient and align prophage genomes to reveal their underlying organization. The conserved genome alignments of phages recapitulate early, middle, and late gene order in transcriptional control of phage genes, and demonstrate that gene order, presumably selected by transcription timing and/or coordination among functional modules has been stably conserved throughout phage evolution.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">Phages are major drivers in global biogeochemical cycles through the control they exert on bacterial populations. Despite their abundance in the environment they are under-represented in sequence databases and knowledge-bases. Phages’ two life cycles, the lytic and lysogenic cycles, provide a unique window into phage genomics. During the lysogenic cycle, temperate phages integrate their genomes into their host’s genome in a non-lethal fashion to generate a region of DNA known as a prophage – either as a plasmid (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>) or by directly recombining with the host’s genome (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>). The phage replicates in the host genome until external signals promote a transition to the lytic cycle, at which point the phage hijacks the host’s cellular DNA replication machinery in order to replicate the phage genome, and adopts the host’s protein synthesis machinery to create new phage particles. Eventually the phage lyse releases infective particles to find new hosts to attack.</p><p id="p-4">The activation and movement of a prophage from one host to another enables phages to act as mediators of horizontal gene transfer, via either generalized or specialized transduction (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>). This phage-mediated horizontal gene transfer may account for up to 10<sup>15</sup> transduction events per second (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>). Phage-mediated horizontal gene transfer events are responsible for the spread of novel functions including the development of pathogenicity in bacterial strains such as <em>Escherichia coli</em> O157:H7 (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>), <em>Vibrio cholerae</em> El Tor N16961 (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>), <em>Streptococcus pyogenes</em> (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>) and many <em>Staphylococcus aureus</em> strains (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>). Phages may mediate the spread of antibiotic resistance in the mammalian intestine (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>), and also transfer more benign genes such as those involved in photosynthesis (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>).</p><p id="p-5">We recently proposed a new model for phage host interactions that suggests at higher host densities ecological considerations favor the temperate lifestyle (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>). The increased energy available at high host densities allows bacteria to support the replication of prophages, which in turn provide beneficial protection to the host via super-infection exclusion. The phages are using their host for replication, but are also providing protection for their host from other invading phages.</p><p id="p-6">Identification of prophages in bacterial genomes is complicated by the problem that many phage genes do not share sequence homology with other proteins. In some cases a newly discovered phage does not carry any genes that share sequence similarity to any other gene in the available databases, highlighting how little we know about these viruses (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>). The underrepresentation of well-annotated phage genes in available databases makes it difficult to assign functions to newly sequenced genomes. This is exacerbated as genome sequencing increases at a near-exponential rate, requiring reliance on automated annotation software that is dependent on primary sequence similarity to assign functions to new genes (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>). Most prophage identification algorithms rely on sequence similarity to known phage genes to identify prophage regions (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>-<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>). This leads to a paradoxical problem of not being able to identify a prophage that does not contain any regions homologous to known phage proteins. Our prophage-finding program, PhiSpy, uses a novel algorithm that is capable of identifying a prophage even if there is no similarity to any of the proteins in the databases by analyzing a suite of genomic characteristics that separate phage-encoding regions from the bacterial core of the genome (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>).</p><p id="p-7">Here we present an analysis of over 11,000 bacterial genomes from which we identified 36,488 prophages. We used this data to identify a core phage genome organizational profile and demonstrate that a significant fraction of phage genomes adhere to this model. This genome structure allows us to predict the function of genes with no known homology solely based on their location in the genome. We identified the integrase genes in 15,162 prophages, and aligned those genes to infer phylogenetic relationships between prophages. Furthermore, we explored the connection between phage and host genomic characteristics to enable predictions of the bacteria that phages are infecting.</p></div><div class="section" id="sec-2"><h2 class="">Materials and Methods</h2><div id="sec-3" class="subsection"><h3>PhiSpy version 3.2 Improvements</h3><p id="p-8">Several improvements were made to the PhiSpy program, and we have released a new version of the software and a web site to provide an accessible interface to the program. The most important change we made was to allow PhiSpy to process genomes that were composed of multiple contigs, as the original version of PhiSpy would treat proteins from different contigs as adjoining proteins. The previous version of PhiSpy used a scanning window that looked 40 proteins ahead of the current protein being classified, but to remove the inherent direction bias, we changed the window to be 20 proteins on either side of the protein being classified. For proteins near the edges of contigs, which had fewer than 20 proteins on a side, we included all possible proteins. To update the list of prophage protein annotation keywords, all the phage genomes from PhAnToMe (<a href="http://www.phantome.org">http://www.phantome.org</a>), and all the complete prokaryotic genomes from GenBank were downloaded. We compared the annotations of each set, and found 12 keywords unique to phage protein annotations: endolysin, terminase, baseplate, base plate, virion, antirepressor, excisionase, Cro-like repressor, cI-like repressor, rIIIA lysis, rI lysis, and rIIB lysis. These keywords were incorporated into PhiSpy for protein classification. In addition to these major changes, many other minor bug fixes were applied. A website was also built so that users can run PhiSpy online, without having to download and install the program and its dependencies: <a href="http://edwards.sdsu.edu/PhiSpy/">http://edwards.sdsu.edu/PhiSpy/</a>. The source code for the updated version of PhiSpy is available from that website and from GitHub: <a href="https://github.com/linsalrob/PhiSpy">https://github.com/linsalrob/PhiSpy</a>.</p></div><div id="sec-4" class="subsection"><h3>Dataset generation</h3><p id="p-9">All bacterial genomes were extracted from the SEED database, which contains both sequence and annotation information that is used by PhiSpy in making prophage designations (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>). The bacterial genomes contained a representative of every genus currently known. The updated version of PhiSpy (version 3.2) was run on all bacterial genomes using the generic bacterial training set option, and the outputs were parsed to identify all predicted prophage regions (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a>). All PhiSpy predictions are available from <a href="https://edwards.sdsu.edu/PhiSpy">https://edwards.sdsu.edu/PhiSpy</a> and <a href="https://figshare.com/articles/PhiSpy_prophage_predictions/3146656">https://figshare.com/articles/PhiSpy_prophage_predictions/3146656</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Workflow of the methodology used to identify prophages in complete microbial genomes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-977106862" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Workflow of the methodology used to identify prophages in complete microbial genomes.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F1.medium.gif" width="440" height="259"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 1." src="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F1.medium.gif" width="440" height="259"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Fig. 1.</span> <p id="p-10" class="first-child">Workflow of the methodology used to identify prophages in complete microbial genomes.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Raw sequence phage-host correlation studies</h3><p id="p-11">Prophage coordinates from PhiSpy were used to extract nucleotide sequence data from the original bacterial genome. The nucleotide sequences are available as FASTA files from <a href="https://edwards.sdsu.edu/PhiSpy">https://edwards.sdsu.edu/PhiSpy</a> and <a href="https://figshare.com/articles/PhiSpy_prophage_predictions/3146656">https://figshare.com/articles/PhiSpy_prophage_predictions/3146656</a>. Putative tRNA-encoding genes were identified from those genomes using tRNAscan-SE (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>). Modal codon usage was calculated from both the predicted prophage and the host genomes as described previously (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>), and tRNA use and modal codon use were compared using the χ<sup>2</sup>-test. The G+C content of the phage and host nucleotide sequences were calculated using custom-written Python scripts.</p></div><div id="sec-6" class="subsection"><h3>Genome heatmaps</h3><p id="p-12">High-quality prophage genome predictions were defined as those where at least one protein was annotated as an integrase, a capsid, a portal, a terminase, and a tail protein. The annotated genes in all the prophages were tabulated to identify a suite of genes that occur frequently in phage genome annotations (integrase, portal, tail, holin, transposase, terminase, endonuclease, capsid, antitermination, Holliday junction resolvase, baseplate assembly protein, primosomal protein, protease-like, lysin, toxin-encoding, and lysozyme).</p><p id="p-13">To generate the genome alignment, each gene was assigned a unique letter indicating its function. A modified Clustal Omega alignment algorithm that scores +1 for matches and 0 for mismatches was used to align the annotated genomes.</p></div><div id="sec-7" class="subsection"><h3>Integrase phylogenomics</h3><p id="p-14">A total of 24,936 integrase (Int) gene sequences were extracted from the high-quality manually curated dataset and clustered using CD-HIT (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>) at a 90% 24,936 similarity threshold (an approximate virus “species” demarcation (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>) to produce 8,536 protein clusters. A representative of each cluster was used in the phylogenetic assessment of the 24,936 genes. In addition, short motifs were identified in the Int genes by using <em>k-mer</em> profiles for <em>k</em> from 4 to 7 amino acids long.</p><p id="p-15">This approach allowed us to identify shared amino acid motifs including the eight-residue motif at the active site of the tyrosine recombinases. One hundred randomly selected integrases were aligned using Clustal Omega (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>) and visualized using FigTree (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>). To compare the evolution of phage and bacterial proteins, all 24,936 phage Int proteins were aligned in a pair-wise fashion using the Smith-Waterman algorithm and the percent identity between each pair of proteins was calculated. Bacterial taxonomy was taken from the NCBI taxonomy database (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>).</p></div><div id="sec-8" class="subsection"><h3>Computational materials</h3><p id="p-16">Computational work was conducted on the Edwards Bioinformatics Lab compute cluster. Standard Python libraries were used for file parsing and Biopython version 1.65 was used to parse GenBank files. Scripts written for file parsing and data analysis are available in the github repository: <a href="https://github.com/hkang408/ProphageGenomics">https://github.com/hkang408/ProphageGenomics</a> and from <a href="https://github.com/EdwardsLab">https://github.com/EdwardsLab</a>.</p></div></div><div class="section" id="sec-9"><h2 class="">Results and Discussion</h2><div id="sec-10" class="subsection"><h3>Prophage genometrics</h3><p id="p-17">At the time of analysis, the SEED contained 11,941 bacterial genomes spanning 34 different phyla of Archaea and Bacteria (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>). A total of 36,465 prophages were identified in 9,883 bacterial hosts (82.76% of the bacteria have at least one prophage). Slightly fewer than one half of the prophages that were identified (15,765 prophages) contained an annotated Int protein, suggesting that they are phages with the potential to integrate. The mechanism of integration of the remaining phages could not be easily identified (see below). A total of 1,511,644 prophage ORFs were identified – of these, 45.7% or 798,235 ORFs had no known function. In addition, 567 genomic regions had no similarity to any of the proteins in the SEED non-redundant database (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">15</a>) although these putative prophages had many other hallmarks of prophages such as shorter genes, many consecutive genes on the same DNA strand, and %GC compositional skews. A total of 7,341 portal genes, 6,228 holin genes, 14,771 capsid genes, and 59,010 tail-related genes were identified in the prophage dataset (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bacterial genomes and the number of prophage regions per genome (blue bars). The phylogenetic tree is based on the NCBI taxonomy and was visualized using iTOL (http://itol.embl.de/). Selected genera have been annotated on the tree." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-977106862" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Bacterial genomes and the number of prophage regions per genome (blue bars). The phylogenetic tree is based on the NCBI taxonomy and was visualized using iTOL (http://itol.embl.de/). Selected genera have been annotated on the tree.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F2.medium.gif" width="440" height="401"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 2." src="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F2.medium.gif" width="440" height="401"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 2.</span> <p id="p-18" class="first-child">Bacterial genomes and the number of prophage regions per genome (blue bars). The phylogenetic tree is based on the NCBI taxonomy and was visualized using iTOL (<a href="http://itol.embl.de/">http://itol.embl.de/</a>). Selected genera have been annotated on the tree.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/275056/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/275056/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/275056" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-19" class="first-child">Frequency of identification of different genes among the prophages. ORFs were identified from 36,465 prophages.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">As we have shown previously, some families of genes are easier to identify than others, and gaps remain in our understanding of families of core phage proteins that we cannot readily identify from sequence homology alone (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>–<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>).</p><p id="p-21">Across the most highly represented phyla, there were an average of between 2.7 (Actinobacteria) and 3.8 (Proteobacteria) prophages found per genome. Within phyla there were also differences (although not statistically significant) in the prophage genome content. For example, <em>Sodalis glossinidius,</em> a proteobacterium, had the most prophages at 29, while several phyla (Caldiserica, Deinococcus–Thermus, Dictyoglomi, Elusimicrobia, and Fibrobacteres) had no prophages, Spirochetes had the shortest prophage genomes on average (25,595 bp), and Bacteroidetes had the longest (46,059 bp) (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3</a>). Across all of the prophages, the mean and median prophage genome length was 36,804 bp and 32,352 bp, respectively. Shorter prophage genomes may indicate that the phage is no longer viable and is in the process of being degraded or removed from the host’s genome (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>). Alternatively, different genome lengths may represent different packaging strategies used by prophages associated with different phyla (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Box and whisker plot denoting number of prophages identified per host, grouped by six phyla with most prophages. (B) Length (bp) of prophages identified grouped by six phyla with most prophages. In both A and B plots, the center line denotes the median of the distribution while box edges denote one standard deviation above or below the median." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-977106862" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;strong&gt;(A)&lt;/strong&gt; Box and whisker plot denoting number of prophages identified per host, grouped by six phyla with most prophages. &lt;strong&gt;(B)&lt;/strong&gt; Length (bp) of prophages identified grouped by six phyla with most prophages. In both A and B plots, the center line denotes the median of the distribution while box edges denote one standard deviation above or below the median.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F3.medium.gif" width="272" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 3." src="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F3.medium.gif" width="272" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 3.</span> <p id="p-22" class="first-child"><strong>(A)</strong> Box and whisker plot denoting number of prophages identified per host, grouped by six phyla with most prophages. <strong>(B)</strong> Length (bp) of prophages identified grouped by six phyla with most prophages. In both A and B plots, the center line denotes the median of the distribution while box edges denote one standard deviation above or below the median.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">Several prophage genome characteristics correlate with the host's characteristics (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>). For example, the prophage’s and host's GC content correlated strongly (R<sup>2</sup> = 0.95; <a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4</a>). As we have shown previously, the GC content of phage genomes can be accurately used to predict the phyla of the host associated with a phage (to &gt;95% accuracy), however GC content can not be accurately used to predict the phages’ host at lower taxonomic levels (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>). Even though %GC content correlates at the phyla level, there are not enough degrees of freedom in %GC calculations to separate thousands of phages and their hosts. Percent GC is basically a <em>k</em>-mer DNA profile with <em>k</em> = 1. Higher <em>k</em>-mer DNA profiles increase the correlation between phage and host, but when <em>k</em> exceeds 7 nucleotides the data become sparse and over fit the predictions (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">35</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The %GC content of prophages shows strong correlation to %GC content of their hosts (R2 = 0.94735). There is no correlation between prophage %GC content and prophage length (upper panel)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-977106862" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The %GC content of prophages shows strong correlation to %GC content of their hosts (R&lt;sup&gt;2&lt;/sup&gt; = 0.94735). There is no correlation between prophage %GC content and prophage length (upper panel).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F4.medium.gif" width="440" height="401"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 4." src="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F4.medium.gif" width="440" height="401"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 4.</span> <p id="p-24" class="first-child">The %GC content of prophages shows strong correlation to %GC content of their hosts (R<sup>2</sup> = 0.94735). There is no correlation between prophage %GC content and prophage length (upper panel).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Phage integrase</h3><p id="p-25">Phage integrases are responsible for catalyzing the site-specific recombination event that inserts the phages into the hosts’ genome (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>–<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>), as well as during replication to help in phage and plasmid genome maintenance. A total of 18,352 prophages (43%) contained an integrase-like protein from one of the two main families – the tyrosine and serine recombinases – named after the catalytic residue involved in recombination. The tyrosine recombinases include the well-studied integrase employed by phage λ to insert its own genome into the bacterial chromosome (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>), while the serine recombinases are exemplified by the integrases of <em>Streptomyces</em> phage phiC31 (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>) and <em>Mycobacteria</em> phage Bxbl (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>). The 24,936 integrase sequences were clustered at a 90% sequence identity threshold using CD-HIT, which resulted in 8,536 clusters. Representatives from each cluster were used to generate a smaller dataset with which to work.</p><p id="p-26">Five-mer profiles of the amino acid sequences were used to identify an eight-residue motif in one of the catalytic region of tyrosine recombinases with a consensus sequence of <strong>HDLRHTFA</strong> (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5A</a>). The very strong conservation of five of the amino acids (bolded) suggests that they participate in the catalytic activity of these recombinases. Several of these amino acids were identified earlier as invariant among tyrosine integrases or nearly so (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>–<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>), and both of the histidines, the leucine, and the arginine have been experimentally demonstrated to participate in DNA recognition, binding, and/or catalysis (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">41</a>). Alignment of one hundred randomly selected tyrosine integrases from the condensed integrase set shows that the proteins that contained the motif were grouped together (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5B</a>), offering a potential route to identify tyrosine recombinases.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Residue frequency of the eight-mer motif contained in the catalytic domain of phage HP1-like integrases. The bars represent the frequency with which each amino acid was found at each of the eight positions in the motif. The consensus sequence is HDLRHTFA. (B) A phylogenetic tree built from one hundred randomly selected integrases shows clusterings of integrase containing the eight-mer motif (highlighted in red)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-977106862" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;strong&gt;(A)&lt;/strong&gt; Residue frequency of the eight-mer motif contained in the catalytic domain of phage HP1-like integrases. The bars represent the frequency with which each amino acid was found at each of the eight positions in the motif. The consensus sequence is HDLRHTFA. &lt;strong&gt;(B)&lt;/strong&gt; A phylogenetic tree built from one hundred randomly selected integrases shows clusterings of integrase containing the eight-mer motif (highlighted in red).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F5.medium.gif" width="312" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 5." src="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F5.medium.gif" width="312" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 5.</span> <p id="p-27" class="first-child"><strong>(A)</strong> Residue frequency of the eight-mer motif contained in the catalytic domain of phage HP1-like integrases. The bars represent the frequency with which each amino acid was found at each of the eight positions in the motif. The consensus sequence is HDLRHTFA. <strong>(B)</strong> A phylogenetic tree built from one hundred randomly selected integrases shows clusterings of integrase containing the eight-mer motif (highlighted in red).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>tRNA presence in prophages compared to tRNA integration occurrences</h3><p id="p-28">Many phages encode tRNA genes, which enhance phage fecundity (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>, <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>). Several hypotheses have been proposed to explain the presence of tRNA genes in phage genomes, including providing tRNAs to compensate for missing or weakly expressed genes in the host (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>). A previous study of 37 phage genomes from 15 bacterial genomes demonstrated that phage tRNA genes compensate for codon bias between host and phage, especially when a codon is common in the phage and rare in the host (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>). In addition, tRNA genes carried by the phage may replace those interrupted during phage insertion into the genome at tRNA genes (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>, <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>). It is also possible that encoding multiple tRNA genes provides the phage genome alternative attachment sites to integrate into the host's genome in case the preferred attachment site is mutated and/or occupied by another phage.</p><p id="p-29">A set of 17,539 tRNA genes were identified from the phages in our total dataset and condensed to a subset of 3,833 tRNA genes (as described for the integrase genes). As expected, the tRNA gene profiles in the phages reflect the number of synonymous codons for each tRNA (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig. 6</a>). However, we found that tRNA-Met is one of the most commonly carried tRNA genes among the prophages we studied. Phages often encode a peptide deformylase enzyme that removes the formyl group from newly synthesized proteins (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>, <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>). We propose that the methionine tRNA genes encoded by the prophages are included to increase the overall rate of translation when the phage enters the lytic cycle. Together, the peptide deformylase and tRNA-Met suggest that phage protein synthesis saturates the host's protein synthesis machinery and that, in many cases, optimal production of phage progeny requires protein translation capacity which exceeds that provided by the host.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The frequency of different tRNAs identified within prophage regions. For each amino acid, the total number of phage-encoded tRNA is shown in parentheses above each graph. The percentage of those tRNAs for each of the possible anticodons is shown in the figure." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-977106862" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The frequency of different tRNAs identified within prophage regions. For each amino acid, the total number of phage-encoded tRNA is shown in parentheses above each graph. The percentage of those tRNAs for each of the possible anticodons is shown in the figure.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F6.medium.gif" width="440" height="308"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 6." src="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F6.medium.gif" width="440" height="308"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 6.</span> <p id="p-30" class="first-child">The frequency of different tRNAs identified within prophage regions. For each amino acid, the total number of phage-encoded tRNA is shown in parentheses above each graph. The percentage of those tRNAs for each of the possible anticodons is shown in the figure.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">We explored the hypothesis that the tRNA genes compensate for codon bias between the host and the phage. For a tRNA gene found in a prophage, the anticodon was converted to the genomic DNA codon it’s associated with. This codon’s modal usage was calculated for the prophage’s genome, as well as for the host’s genome without the prophage sequence. The codon’s modal usage in the phage and host was compared using the χ<sup>2</sup>-test. We found that 41.3% of the codons associated with the tRNA found in a prophage had statistically higher modal usage in the phage than in the host (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig. 6</a>).</p><p id="p-32">We applied the same approach to compare the phage’s modal usage of the same codon with an expected usage assuming an even distribution given the number of synonymous codons available. This comparison revealed that 31% of the codon associated with the tRNA found in a prophage had a higher usage in the phage than expected by chance. Together, these results suggest that phage-encoded tRNA are supplementing limitations in the hosts translational machinery to ensure that phage particles are produced with high efficiency upon induction.</p></div><div id="sec-13" class="subsection"><h3>Genome Architecture</h3><p id="p-33">Phages have a mosaic genome architecture (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>, <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>), in which the genome is organized into groups or clusters of genes. Mosaicism has been proposed to be driven by illegitimate recombination, but it is not clear from previous genomic analyses whether the mosaic boundaries are between clusters of related genes or between individual genes (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">32</a>, <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">53</a>, <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>).</p><p id="p-34">We identified the fourteen most commonly occurring and well-annotated genes within the prophage dataset (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The majority (66.9%; 10,542/15,765) of these predicted prophages had the integrase gene located near one of the ends of the prophage genome. We generated a genome data set where all the genomes were organized with the integrase at the same end of the genome, in some cases by reverse-complementing the genome to place the integrase gene consistently at the left end of every genome, and in other cases by recircularizing the genome <em>in silico</em> and introducing a break that placed the integrase at the left end.</p><p id="p-35">The resulting alignment allowed us to compare genome architecture across all of the prophages, and revealed that gene order is conserved across phage genomes (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Fig. 7a</a>), suggesting that there is an advantageous order of clusters of related function within the genomes of prophages. To highlight the clusters, we used a modified Smith-Waterman approach, essentially ascribing a single letter to each of the functions and then aligning the genomes based on the order of the functions with a +1 score for a match and no penalty for a mismatch (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Fig. 7b</a>). This shows that the gene order across 1,537 phage genomes is highly conserved, reminiscent of the early—middle—late gene organization of many well characterized phages and eukaryotic viruses (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>–<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a>). We also visualized the presence of each gene at its location in the genome using a gene density plot (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Fig. 8</a>), which highlights the separation of the genes based on location in the genome. The combined results from these alignments strongly indicate that the mosaicism in phage genomes occurs at the gene-cluster level, and that phage genomes are not comprised of random hybrids of many genomes. Instead, recombination (illegitimate or otherwise) appears to be constrained to result in a highly conserved gene order where the early genes, including those encoding entry and DNA replication functions, precede those encoding packaging functions, which in turn precede tail formation and host lysis functions.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Conserved prophage genome organization. Proteins in the phage genomes are colored by their annotation (integrase, portal, tail, holin, transposase, terminase, endonuclease, capsid, antitermination, Holliday junction resolvase, baseplate assembly protein, primosomal, protease-like, lysin, toxin-encoding genes, and lysozyme), and the phage genomes were normalized to the largest prophage. (B) Phage gene heatmap aligned using a modified Clustal alignment algorithm, as described in Materials and Methods." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-977106862" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;strong&gt;(A)&lt;/strong&gt; Conserved prophage genome organization. Proteins in the phage genomes are colored by their annotation (integrase, portal, tail, holin, transposase, terminase, endonuclease, capsid, antitermination, Holliday junction resolvase, baseplate assembly protein, primosomal, protease-like, lysin, toxin-encoding genes, and lysozyme), and the phage genomes were normalized to the largest prophage. &lt;strong&gt;(B)&lt;/strong&gt; Phage gene heatmap aligned using a modified Clustal alignment algorithm, as described in Materials and Methods.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F7.medium.gif" width="228" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 7." src="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F7.medium.gif" width="228" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 7." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 7.</span> <p id="p-36" class="first-child"><strong>(A)</strong> Conserved prophage genome organization. Proteins in the phage genomes are colored by their annotation (integrase, portal, tail, holin, transposase, terminase, endonuclease, capsid, antitermination, Holliday junction resolvase, baseplate assembly protein, primosomal, protease-like, lysin, toxin-encoding genes, and lysozyme), and the phage genomes were normalized to the largest prophage.</p><p id="p-37"><strong>(B)</strong> Phage gene heatmap aligned using a modified Clustal alignment algorithm, as described in Materials and Methods.</p><div class="sb-div caption-clear"></div></div></div><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Gene density plots of the occurrence of phage marker genes along the genomes of 1,537 phage genomes. The genomes were rearranged as described in the text to place the integrase gene at the 5&#x2019; end of the phage genome, and then genes were displayed as ordered by the modal frequency of each gene." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-977106862" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Gene density plots of the occurrence of phage marker genes along the genomes of 1,537 phage genomes. The genomes were rearranged as described in the text to place the integrase gene at the 5&#x2019; end of the phage genome, and then genes were displayed as ordered by the modal frequency of each gene.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F8.medium.gif" width="233" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 8." src="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F8.medium.gif" width="233" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 8." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/07/114819/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 8.</span> <p id="p-38" class="first-child">Gene density plots of the occurrence of phage marker genes along the genomes of 1,537 phage genomes. The genomes were rearranged as described in the text to place the integrase gene at the 5’ end of the phage genome, and then genes were displayed as ordered by the modal frequency of each gene.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">The phage genome is therefore not a collection of genes that have been acquired at random by recombination, but rather analogous to a computer motherboard, where modules may be replaced provided they are slotted into the appropriate gene expression framework. Thus, a viable phage is more likely to be produced if genes are placed in the appropriate modules.</p></div></div><div class="section" id="sec-14"><h2 class="">Conclusions</h2><p id="p-40">Here we presented an analysis of over 11,000 bacterial genomes from which we identified 36,488 prophages. Many phages appear to be limited by initiation of translation by the host's machinery, and may increase translation rates by carrying their own tRNA genes, effectively increasing the availability of both tRNAs loaded with methionine and of peptide deformylase. We have also demonstrated that phages maintain a highly conserved gene order that suggests phage genome mosaicism is limited to clusters of conserved genes rather than individual genes.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.1" data-doi="10.1007/BF00264942"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Prentki</span>, <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chandler</span>, <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Caro</span>, <span class="cit-name-given-names">L.</span></span> (<span class="cit-pub-date">1977</span>) <span class="cit-article-title">Replication of the prophage P1 during the cell cycle of Escherichia coli</span>. <abbr class="cit-jnl-abbrev">Mol. Gen. Genet. MGG</abbr>, <span class="cit-vol">152</span>, <span class="cit-fpage">71</span>–<span class="cit-lpage">76</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BGen.%2BGenet.%2BMGG%26rft.volume%253D152%26rft.spage%253D71%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00264942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00264942&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.2" data-doi="10.1128/jb.174.23.7495-7499.1992"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Campbell</span>, <span class="cit-name-given-names">A.M.</span></span> (<span class="cit-pub-date">1992</span>) <span class="cit-article-title">Chromosomal insertion sites for phages and plasmids</span>. <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr>, <span class="cit-vol">174</span>, <span class="cit-fpage">7495</span>–<span class="cit-lpage">7499</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DCampbell%26rft.auinit1%253DA.%2BM.%26rft.volume%253D174%26rft.issue%253D23%26rft.spage%253D7495%26rft.epage%253D7499%26rft.atitle%253DChromosomal%2Binsertion%2Bsites%2Bfor%2Bphages%2Band%2Bplasmids.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.174.23.7495-7499.1992%26rft_id%253Dinfo%253Apmid%252F1447124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czoyOiJqYiI7czo1OiJyZXNpZCI7czoxMToiMTc0LzIzLzc0OTUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNy8wMy8wNy8xMTQ4MTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Campbell</span>, <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">Phage integration and chromosome structure. A personal history</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr>, <span class="cit-vol">41</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DCampbell%26rft.auinit1%253DA.%26rft.volume%253D41%26rft.spage%253D1%26rft.epage%253D11%26rft.atitle%253DPhage%2Bintegration%2Band%2Bchromosome%2Bstructure.%2BA%2Bpersonal%2Bhistory.%26rft_id%253Dinfo%253Apmid%252F17474874%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17474874&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-114819v1.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maloy</span>, <span class="cit-name-given-names">S.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stewart</span>, <span class="cit-name-given-names">V.J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Taylor</span>, <span class="cit-name-given-names">R.K.</span></span> (<span class="cit-pub-date">1996</span>) <span class="cit-source">Genetic Analysis of Pathogenic Bacteria</span>: <span class="cit-publ-name">A Laboratory Manual Cold Spring Harbor Laboratory Press</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brüssow</span>, <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Canchaya</span>, <span class="cit-name-given-names">C.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Hardt</span>, <span class="cit-name-given-names">W.-D.</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">Phages and the evolution of bacterial pathogens: from genomic rearrangements to lysogenic conversion</span>. <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev. MMBR</abbr>, <span class="cit-vol">68</span>, <span class="cit-fpage">560</span>–<span class="cit-lpage">602</span>, <span class="cit-comment">table of contents</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%2BMMBR%26rft.volume%253D68%26rft.spage%253D560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-book" id="cit-114819v1.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bushman</span>, <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">2001</span>) <span class="cit-source">Lateral DNA Transfer</span>: <span class="cit-publ-name">Mechanisms and Consequences CSHL Press</span>, <span class="cit-publ-loc">New York, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.7" data-doi="10.1038/35020000"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Heidelberg</span>, <span class="cit-name-given-names">J.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eisen</span>, <span class="cit-name-given-names">J.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nelson</span>, <span class="cit-name-given-names">W.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clayton</span>, <span class="cit-name-given-names">R.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gwinn</span>, <span class="cit-name-given-names">M.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dodson</span>, <span class="cit-name-given-names">R.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haft</span>, <span class="cit-name-given-names">D.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hickey</span>, <span class="cit-name-given-names">E.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peterson</span>, <span class="cit-name-given-names">J.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Umayam</span>, <span class="cit-name-given-names">L.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2000</span>) <span class="cit-article-title">DNA sequence of both chromosomes of the cholera pathogen Vibrio cholerae</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">406</span>, <span class="cit-fpage">477</span>–<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHeidelberg%26rft.auinit1%253DJ.%2BF.%26rft.volume%253D406%26rft.issue%253D6795%26rft.spage%253D477%26rft.epage%253D483%26rft.atitle%253DDNA%2Bsequence%2Bof%2Bboth%2Bchromosomes%2Bof%2Bthe%2Bcholera%2Bpathogen%2BVibrio%2Bcholerae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35020000%26rft_id%253Dinfo%253Apmid%252F10952301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35020000&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10952301&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000088538000035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-book" id="cit-114819v1.8"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Ferretti</span>, <span class="cit-name-given-names">J.J.</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Stevens</span>, <span class="cit-name-given-names">D.L.</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Fischetti</span>, <span class="cit-name-given-names">V.A.</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">McShan</span>, <span class="cit-name-given-names">W.M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Nguyen</span>, <span class="cit-name-given-names">S.V.</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-chapter-title">The Bacteriophages of Streptococcus pyogenes</span>. In <span class="cit-ed"><span class="cit-name-surname">Ferretti</span>, <span class="cit-name-given-names">J.J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Stevens</span>, <span class="cit-name-given-names">D.L.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Fischetti</span>, <span class="cit-name-given-names">V.A.</span></span> (eds), <span class="cit-source">Streptococcus pyogenes: Basic Biology to Clinical Manifestations</span>. <span class="cit-publ-name">University of Oklahoma Health Sciences Center</span>, <span class="cit-publ-loc">Oklahoma City (OK)</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.9" data-doi="10.1016/S1369-5274(03)00086-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Canchaya</span>, <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fournous</span>, <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chibani-Chennoufi</span>, <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dillmann</span>, <span class="cit-name-given-names">M.L.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Brüssow</span>, <span class="cit-name-given-names">H</span></span>. (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">Phage as agents of lateral gene transfer</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Microbiol.</abbr>, <span class="cit-vol">6</span>, <span class="cit-fpage">417</span>–<span class="cit-lpage">424</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bmicrobiology%26rft.stitle%253DCurr%2BOpin%2BMicrobiol%26rft.aulast%253DCanchaya%26rft.auinit1%253DC.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253D417%26rft.epage%253D424%26rft.atitle%253DPhage%2Bas%2Bagents%2Bof%2Blateral%2Bgene%2Btransfer.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1369-5274%252803%252900086-9%26rft_id%253Dinfo%253Apmid%252F12941415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1369-5274(03)00086-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12941415&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185173000018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.10" data-doi="10.1038/nature12212"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Modi</span>, <span class="cit-name-given-names">S.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>, <span class="cit-name-given-names">H.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spina</span>, <span class="cit-name-given-names">C.S.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Collins</span>, <span class="cit-name-given-names">J.J.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Antibiotic treatment expands the resistance reservoir and ecological network of the phage metagenome</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">499</span>, <span class="cit-fpage">219</span>–<span class="cit-lpage">222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D499%26rft.spage%253D219%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12212%26rft_id%253Dinfo%253Apmid%252F23748443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12212&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23748443&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321557600064&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.11" data-doi="10.1038/424741a"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mann</span>, <span class="cit-name-given-names">N.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cook</span>, <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Millard</span>, <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bailey</span>, <span class="cit-name-given-names">S.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Clokie</span>, <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">Marine ecosystems: Bacterial photosynthesis genes in a virus</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">424</span>, <span class="cit-fpage">741</span>–<span class="cit-lpage">741</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAmanullah%26rft.auinit1%253DA.%26rft.volume%253D424%26rft.issue%253D6950%26rft.spage%253D741%26rft.epage%253D741%26rft.atitle%253DCell%2Bsignalling%253A%2Bcell%2Bsurvival%2Band%2Ba%2BGadd45-factor%2Bdeficiency.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F424741a%26rft_id%253Dinfo%253Apmid%252F12917673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/424741a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12917673&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.12" data-doi="10.1038/nature17193"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Knowles</span>, <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Silveira</span>, <span class="cit-name-given-names">C.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bailey</span>, <span class="cit-name-given-names">B.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barott</span>, <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cantu</span>, <span class="cit-name-given-names">V.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cobián-Güemes</span>, <span class="cit-name-given-names">A.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coutinho</span>, <span class="cit-name-given-names">F.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dinsdale</span>, <span class="cit-name-given-names">E.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Felts</span>, <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Furby</span>, <span class="cit-name-given-names">K.A.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Lytic to temperate switching of viral communities</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">531</span>, <span class="cit-fpage">466</span>–<span class="cit-lpage">470</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D531%26rft.spage%253D466%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature17193%26rft_id%253Dinfo%253Apmid%252F26982729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature17193&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26982729&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.13" data-doi="10.1016/j.mib.2008.09.004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hatfull</span>, <span class="cit-name-given-names">G.F.</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">Bacteriophage genomics</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Microbiol.</abbr>, <span class="cit-vol">11</span>, <span class="cit-fpage">447</span>–<span class="cit-lpage">453</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bmicrobiology%26rft.stitle%253DCurr%2BOpin%2BMicrobiol%26rft.aulast%253DHatfull%26rft.auinit1%253DG.%2BF.%26rft.volume%253D11%26rft.issue%253D5%26rft.spage%253D447%26rft.epage%253D453%26rft.atitle%253DBacteriophage%2Bgenomics.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mib.2008.09.004%26rft_id%253Dinfo%253Apmid%252F18824125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mib.2008.09.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18824125&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261246900012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.14" data-doi="10.1186/1471-2164-9-75"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aziz</span>, <span class="cit-name-given-names">R.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bartels</span>, <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Best</span>, <span class="cit-name-given-names">A.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">DeJongh</span>, <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Disz</span>, <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Edwards</span>, <span class="cit-name-given-names">R.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Formsma</span>, <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gerdes</span>, <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Glass</span>, <span class="cit-name-given-names">E.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kubal</span>, <span class="cit-name-given-names">M.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">The RAST Server: rapid annotations using subsystems technology</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr>, <span class="cit-vol">9</span>, <span class="cit-fpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DAziz%26rft.auinit1%253DR.%2BK.%26rft.volume%253D9%26rft.spage%253D75%26rft.epage%253D75%26rft.atitle%253DThe%2BRAST%2BServer%253A%2Brapid%2Bannotations%2Busing%2Bsubsystems%2Btechnology.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-75%26rft_id%253Dinfo%253Apmid%252F18261238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-75&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18261238&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.15" data-doi="10.1093/nar/gkt1226"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Overbeek</span>, <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Olson</span>, <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pusch</span>, <span class="cit-name-given-names">G.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Olsen</span>, <span class="cit-name-given-names">G.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davis</span>, <span class="cit-name-given-names">J.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Disz</span>, <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Edwards</span>, <span class="cit-name-given-names">R.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gerdes</span>, <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Parrello</span>, <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shukla</span>, <span class="cit-name-given-names">M.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">The SEED and the Rapid Annotation of microbial genomes using Subsystems Technology (RAST)</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>, <span class="cit-vol">42</span>, <span class="cit-fpage">D206</span>–<span class="cit-lpage">214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1226%26rft_id%253Dinfo%253Apmid%252F24293654%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt1226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24293654&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000331139800032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.16" data-doi="10.1093/nar/gkr485"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>, <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liang</span>, <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lynch</span>, <span class="cit-name-given-names">K.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dennis</span>, <span class="cit-name-given-names">J.J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Wishart</span>, <span class="cit-name-given-names">D.S.</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">PHAST: A Fast Phage Search Tool</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>, <span class="cit-vol">39</span>, <span class="cit-fpage">W347</span>–<span class="cit-lpage">W352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr485%26rft_id%253Dinfo%253Apmid%252F21672955%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr485&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21672955&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292325300056&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">17.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-114819v1.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bose</span>, <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Barber</span>, <span class="cit-name-given-names">R.D.</span></span> (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">Prophage Finder: a prophage loci prediction tool for prokaryotic genome sequences</span>. <abbr class="cit-jnl-abbrev">Silico Biol</abbr>, <span class="cit-vol">6</span>, <span class="cit-fpage">223</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSilico%2BBiol%26rft.volume%253D6%26rft.spage%253D223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.18" data-doi="10.1093/nar/gkl732"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fouts</span>, <span class="cit-name-given-names">D.E.</span></span> (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">Phage_Finder: automated identification and classification of prophage regions in complete bacterial genome sequences</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">34</span>, <span class="cit-fpage">5839</span>–<span class="cit-lpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl732%26rft_id%253Dinfo%253Apmid%252F17062630%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkl732&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17062630&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242474800018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">19.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-114819v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lima-Mendez</span>, <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van Helden</span>, <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Toussaint</span>, <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Leplae</span>, <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">Prophinder: a computational tool for prophage prediction in prokaryotic genomes</span>. <abbr class="cit-jnl-abbrev">Bioinforma. Oxf Engl.</abbr>, <span class="cit-vol">24</span>, <span class="cit-fpage">863</span>–<span class="cit-lpage">865</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinforma.%2BOxf%2BEngl.%26rft.volume%253D24%26rft.spage%253D863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.20" data-doi="10.7717/peerj.985"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Roux</span>, <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Enault</span>, <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hurwitz</span>, <span class="cit-name-given-names">B.L.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Sullivan</span>, <span class="cit-name-given-names">M.B.</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">VirSorter: mining viral signal from microbial genomic data</span>. <abbr class="cit-jnl-abbrev">PeerJ</abbr>, <span class="cit-vol">3</span>, <span class="cit-fpage">e985</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPeerJ%26rft.volume%253D3%26rft.spage%253D985e%26rft_id%253Dinfo%253Adoi%252F10.7717%252Fpeerj.985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7717/peerj.985&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.21" data-doi="10.1093/nar/gks406"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Akhter</span>, <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aziz</span>, <span class="cit-name-given-names">R.K.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Edwards</span>, <span class="cit-name-given-names">R.A.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">PhiSpy: a novel algorithm for finding prophages in bacterial genomes that combines similarity-and composition-based strategies</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>, <span class="cit-vol">40</span>, <span class="cit-fpage">e126</span>–<span class="cit-lpage">e126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks406%26rft_id%253Dinfo%253Apmid%252F22584627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks406&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22584627&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.22" data-doi="10.1093/nar/25.5.955"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lowe</span>, <span class="cit-name-given-names">T.M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Eddy</span>, <span class="cit-name-given-names">S.R.</span></span> (<span class="cit-pub-date">1997</span>) <span class="cit-article-title">tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-vol">25</span>, <span class="cit-fpage">955</span>–<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.5.955%26rft_id%253Dinfo%253Apmid%252F9023104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/25.5.955&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9023104&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.23" data-doi="10.1093/molbev/msp281"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Davis</span>, <span class="cit-name-given-names">J.J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Olsen</span>, <span class="cit-name-given-names">G.J.</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Modal Codon Usage: Assessing the Typical Codon Usage of a Genome</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr>, <span class="cit-vol">27</span>, <span class="cit-fpage">800</span>–<span class="cit-lpage">810</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp281%26rft_id%253Dinfo%253Apmid%252F20018979%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msp281&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20018979&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276639800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.24" data-doi="10.1093/bioinformatics/btq003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>, <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Niu</span>, <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>, <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fu</span>, <span class="cit-name-given-names">L.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Li</span>, <span class="cit-name-given-names">W.</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">CD-HIT Suite: a web server for clustering and comparing biological sequences</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">26</span>, <span class="cit-fpage">680</span>–<span class="cit-lpage">2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq003%26rft_id%253Dinfo%253Apmid%252F20053844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20053844&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274973800015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.25" data-doi="10.1038/msb.2011.75"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sievers</span>, <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilm</span>, <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dineen</span>, <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gibson</span>, <span class="cit-name-given-names">T.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Karplus</span>, <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>, <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lopez</span>, <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McWilliam</span>, <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Remmert</span>, <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Söding</span>, <span class="cit-name-given-names">J.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega</span>. <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr>, <span class="cit-vol">7</span>, <span class="cit-fpage">539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DSievers%26rft.auinit1%253DF.%26rft.volume%253D7%26rft.issue%253D1%26rft.spage%253D539%26rft.epage%253D539%26rft.atitle%253DFast%252C%2Bscalable%2Bgeneration%2Bof%2Bhigh-quality%2Bprotein%2Bmultiple%2Bsequence%2Balignments%2Busing%2BClustal%2BOmega%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2011.75%26rft_id%253Dinfo%253Apmid%252F21988835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjc6IjcvMS81MzkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNy8wMy8wNy8xMTQ4MTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.26" data-doi="10.1093/nar/gkq313"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goujon</span>, <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McWilliam</span>, <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>, <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Valentin</span>, <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Squizzato</span>, <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Paern</span>, <span class="cit-name-given-names">J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Lopez</span>, <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">A new bioinformatics analysis tools framework at EMBL-EBI</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>, <span class="cit-vol">38</span>, <span class="cit-fpage">W695</span>–<span class="cit-lpage">699</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq313%26rft_id%253Dinfo%253Apmid%252F20439314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq313&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20439314&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284148900113&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.27" data-doi="10.1098/rstb.2004.1478"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Holmes</span>, <span class="cit-name-given-names">E.C.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>, <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">Viral evolution and the emergence of SARS coronavirus</span>. <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. B Biol. Sci.</abbr>, <span class="cit-vol">359</span>, <span class="cit-fpage">1059</span>–<span class="cit-lpage">1065</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DHolmes%26rft.auinit1%253DE.%2BC.%26rft.volume%253D359%26rft.issue%253D1447%26rft.spage%253D1059%26rft.epage%253D1065%26rft.atitle%253DViral%2Bevolution%2Band%2Bthe%2Bemergence%2Bof%2BSARS%2Bcoronavirus%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2004.1478%26rft_id%253Dinfo%253Apmid%252F15306390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2004.1478&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15306390&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222688200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.28" data-doi="10.1093/nar/gkv1290"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">NCBI Resource Coordinators</span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Database resources of the National Center for Biotechnology Information</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>, <span class="cit-vol">44</span>, <span class="cit-fpage">D7</span>–<span class="cit-lpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1290%26rft_id%253Dinfo%253Apmid%252F26615191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1290&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26615191&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sanchez</span>, <span class="cit-name-given-names">S.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cuevas</span>, <span class="cit-name-given-names">D.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rostron</span>, <span class="cit-name-given-names">J.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liang</span>, <span class="cit-name-given-names">T.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pivaroff</span>, <span class="cit-name-given-names">C.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haynes</span>, <span class="cit-name-given-names">M.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nulton</span>, <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Felts</span>, <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bailey</span>, <span class="cit-name-given-names">B.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Salamon</span>, <span class="cit-name-given-names">P.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Phage Phenomics: Physiological Approaches to Characterize Novel Viral Proteins</span>. <abbr class="cit-jnl-abbrev">J. Vis. Exp.</abbr>, <span class="cit-comment">10.3791/52854</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">30.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-114819v1.30" data-doi="10.1128/JB.185.21.6434-6447.2003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Seguritan</span>, <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feng</span>, <span class="cit-name-given-names">I.-W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rohwer</span>, <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Swift</span>, <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Segall</span>, <span class="cit-name-given-names">A.M.</span></span> (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">Genome sequences of two closely related Vibrio parahaemolyticus phages, VP16T and VP16C</span>. <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr>, <span class="cit-vol">185</span>, <span class="cit-fpage">6434</span>–<span class="cit-lpage">6447</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DSeguritan%26rft.auinit1%253DV.%26rft.volume%253D185%26rft.issue%253D21%26rft.spage%253D6434%26rft.epage%253D6447%26rft.atitle%253DGenome%2BSequences%2Bof%2BTwo%2BClosely%2BRelated%2BVibrio%2Bparahaemolyticus%2BPhages%252C%2BVP16T%2Band%2BVP16C%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.185.21.6434-6447.2003%26rft_id%253Dinfo%253Apmid%252F14563879%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE4NS8yMS82NDM0IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDMvMDcvMTE0ODE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rohwer</span>, <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Segall</span>, <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Steward</span>, <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Seguritan</span>, <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Breitbart</span>, <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wolven</span>, <span class="cit-name-given-names">F.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Azam</span>, <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">2000</span>) <span class="cit-article-title">The complete genomic sequence of the marine phage Roseophage SIOI shares homology with nonmarine phages</span>. <abbr class="cit-jnl-abbrev">Limnol. Oceanogr.</abbr>, <span class="cit-vol">45</span>, <span class="cit-fpage">408</span>–<span class="cit-lpage">418</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLimnol.%2BOceanogr.%26rft.volume%253D45%26rft.spage%253D408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.32" data-doi="10.1073/pnas.96.5.2192"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hendrix</span>, <span class="cit-name-given-names">R.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>, <span class="cit-name-given-names">M.C.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burns</span>, <span class="cit-name-given-names">R.N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ford</span>, <span class="cit-name-given-names">M.E.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Hatfull</span>, <span class="cit-name-given-names">G.F.</span></span> (<span class="cit-pub-date">1999</span>) <span class="cit-article-title">Evolutionary relationships among diverse bacteriophages and prophages: All the world’s a phage</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci.</abbr>, <span class="cit-vol">96</span>, <span class="cit-fpage">2192</span>–<span class="cit-lpage">2197</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DHendrix%26rft.auinit1%253DR.%2BW.%26rft.volume%253D96%26rft.issue%253D5%26rft.spage%253D2192%26rft.epage%253D2197%26rft.atitle%253DEvolutionary%2Brelationships%2Bamong%2Bdiverse%2Bbacteriophages%2Band%2Bprophages%253A%2BAll%2Bthe%2Bworld%2527s%2Ba%2Bphage%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.96.5.2192%26rft_id%253Dinfo%253Apmid%252F10051617%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5Ni81LzIxOTIiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNy8wMy8wNy8xMTQ4MTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-book" id="cit-114819v1.33"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Mateu</span>, <span class="cit-name-given-names">M.G.</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Luque</span>, <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Reguera</span>, <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-chapter-title">Theoretical Studies on Assembly, Physical Stability and Dynamics of Viruses</span>. In <span class="cit-ed"><span class="cit-name-surname">Mateu</span>, <span class="cit-name-given-names">M.G.</span></span> (ed), <span class="cit-source">Structure and Physics of Viruses, Subcellular Biochemistry</span>. <span class="cit-publ-name">Springer Netherlands</span>, pp. <span class="cit-fpage">553</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.34" data-doi="10.7554/eLife.08490"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Roux</span>, <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hallam</span>, <span class="cit-name-given-names">S.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Woyke</span>, <span class="cit-name-given-names">T.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Sullivan</span>, <span class="cit-name-given-names">M.B.</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Viral dark matter and virus-host interactions resolved from publicly available microbial genomes</span>. <abbr class="cit-jnl-abbrev">eLife</abbr>, <span class="cit-vol">4</span>, <span class="cit-fpage">e08490</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.08490%26rft_id%253Dinfo%253Apmid%252F26200428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.08490&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26200428&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Edwards</span>, <span class="cit-name-given-names">R.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McNair</span>, <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Faust</span>, <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Raes</span>, <span class="cit-name-given-names">J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Dutilh</span>, <span class="cit-name-given-names">B.E.</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Computational approaches to predict bacteriophage-host relationships</span>. <abbr class="cit-jnl-abbrev">FEMS Microbiol. Rev.</abbr>, <span class="cit-comment">10.1093/femsre/fuv048</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rajeev</span>, <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Malanowska</span>, <span class="cit-name-given-names">K.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Gardner</span>, <span class="cit-name-given-names">J.F.</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Challenging a paradigm: the role of DNA homology in tyrosine recombinase reactions</span>. <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev. MMBR</abbr>, <span class="cit-vol">73</span>, <span class="cit-fpage">300</span>–<span class="cit-lpage">309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%2BMMBR%26rft.volume%253D73%26rft.spage%253D300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">37.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-114819v1.37" data-doi="10.1007/s00253-011-3519-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hirano</span>, <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muroi</span>, <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Takahashi</span>, <span class="cit-name-given-names">H.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Haruki</span>, <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Site-specific recombinases as tools for heterologous gene integration</span>. <abbr class="cit-jnl-abbrev">Appl. Microbiol. Biotechnol.</abbr>, <span class="cit-vol">92</span>, <span class="cit-fpage">227</span>–<span class="cit-lpage">239</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2BMicrobiology%2Band%2BBiotechnology%26rft.stitle%253DApplied%2BMicrobiology%2Band%2BBiotechnology%26rft.aulast%253DHirano%26rft.auinit1%253DN.%26rft.volume%253D92%26rft.issue%253D2%26rft.spage%253D227%26rft.epage%253D239%26rft.atitle%253DSite-specific%2Brecombinases%2Bas%2Btools%2Bfor%2Bheterologous%2Bgene%2Bintegration.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00253-011-3519-5%26rft_id%253Dinfo%253Apmid%252F21822899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00253-011-3519-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21822899&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295325300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">38.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-114819v1.38" data-doi="10.1016/j.tim.2012.10.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Das</span>, <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martinez</span>, <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Midonet</span>, <span class="cit-name-given-names">C.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Barre</span>, <span class="cit-name-given-names">F.-X.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Integrative mobile elements exploiting Xer recombination</span>. <abbr class="cit-jnl-abbrev">Trends Microbiol.</abbr>, <span class="cit-vol">21</span>, <span class="cit-fpage">23</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BMicrobiol.%26rft.volume%253D21%26rft.spage%253D23%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2012.10.003%26rft_id%253Dinfo%253Apmid%252F23127381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2012.10.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23127381&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000314012600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">39.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-114819v1.39" data-doi="10.1007/s00438-012-0700-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Askora</span>, <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Abdel-Haliem</span>, <span class="cit-name-given-names">M.E.F.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Yamada</span>, <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Site-specific recombination systems in filamentous phages</span>. <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr>, <span class="cit-vol">287</span>, <span class="cit-fpage">525</span>–<span class="cit-lpage">530</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DAskora%26rft.auinit1%253DA.%26rft.volume%253D287%26rft.issue%253D7%26rft.spage%253D525%26rft.epage%253D530%26rft.atitle%253DSite-specific%2Brecombination%2Bsystems%2Bin%2Bfilamentous%2Bphages.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-012-0700-1%26rft_id%253Dinfo%253Apmid%252F22661259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-012-0700-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22661259&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">40.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-114819v1.40" data-doi="10.1111/nyas.12675"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Krupovic</span>, <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Forterre</span>, <span class="cit-name-given-names">P.</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Single-stranded DNA viruses employ a variety of mechanisms for integration into host genomes</span>. <abbr class="cit-jnl-abbrev">Ann. N. Y. Acad. Sci.</abbr>, <span class="cit-vol">1341</span>, <span class="cit-fpage">41</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BN.%2BY.%2BAcad.%2BSci.%26rft.volume%253D1341%26rft.spage%253D41%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnyas.12675%26rft_id%253Dinfo%253Apmid%252F25675979%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nyas.12675&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25675979&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jayaram</span>, <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ma</span>, <span class="cit-name-given-names">C.-H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kachroo</span>, <span class="cit-name-given-names">A.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rowley</span>, <span class="cit-name-given-names">P.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guga</span>, <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fan</span>, <span class="cit-name-given-names">H.-F.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Voziyanov</span>, <span class="cit-name-given-names">Y.</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">An Overview of Tyrosine Site-specific Recombination: From an Flp Perspective</span>. <abbr class="cit-jnl-abbrev">Microbiol. Spectr.</abbr>, <span class="cit-vol">3</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.42" data-doi="10.1042/BST0380388"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Smith</span>, <span class="cit-name-given-names">M.C.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brown</span>, <span class="cit-name-given-names">W.R.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McEwan</span>, <span class="cit-name-given-names">A.R.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Rowley</span>, <span class="cit-name-given-names">P.A.</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Site-specific recombination by phiC31 integrase and other large serine recombinases</span>. <abbr class="cit-jnl-abbrev">Biochem. Soc. Trans.</abbr>, <span class="cit-vol">38</span>, <span class="cit-fpage">388</span>–<span class="cit-lpage">394</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochem.%2BSoc.%2BTrans.%26rft_id%253Dinfo%253Adoi%252F10.1042%252FBST0380388%26rft_id%253Dinfo%253Apmid%252F20298189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoicHBiaW9zdCI7czo1OiJyZXNpZCI7czo4OiIzOC8yLzM4OCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzAzLzA3LzExNDgxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.43" data-doi="10.1016/S1097-2765(03)00444-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ghosh</span>, <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kim</span>, <span class="cit-name-given-names">A.I.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Hatfull</span>, <span class="cit-name-given-names">G.F.</span></span> (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">The orientation of mycobacteriophage Bxb1 integration is solely dependent on the central dinucleotide of attP and attB</span>. <abbr class="cit-jnl-abbrev">Mol. Cell</abbr>, <span class="cit-vol">12</span>, <span class="cit-fpage">1101</span>–<span class="cit-lpage">1111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DGhosh%26rft.auinit1%253DP.%26rft.volume%253D12%26rft.issue%253D5%26rft.spage%253D1101%26rft.epage%253D1111%26rft.atitle%253DThe%2Borientation%2Bof%2Bmycobacteriophage%2BBxb1%2Bintegration%2Bis%2Bsolely%2Bdependent%2Bon%2Bthe%2Bcentral%2Bdinucleotide%2Bof%2BattP%2Band%2BattB.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252803%252900444-1%26rft_id%253Dinfo%253Apmid%252F14636570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(03)00444-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14636570&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186764700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Argos</span>, <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Landy</span>, <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Abremski</span>, <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Egan</span>, <span class="cit-name-given-names">J.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haggard-Ljungquist</span>, <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoess</span>, <span class="cit-name-given-names">R.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kahn</span>, <span class="cit-name-given-names">M.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kalionis</span>, <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Narayana</span>, <span class="cit-name-given-names">S.V.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Pierson</span>, <span class="cit-name-given-names">L.S.</span></span> (<span class="cit-pub-date">1986</span>) <span class="cit-article-title">The integrase family of site-specific recombinases: regional similarities and global diversity</span>. <abbr class="cit-jnl-abbrev">EMBO J.</abbr>, <span class="cit-vol">5</span>, <span class="cit-fpage">433</span>–<span class="cit-lpage">440</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft.volume%253D5%26rft.spage%253D433%26rft_id%253Dinfo%253Apmid%252F3011407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=3011407&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1986A347600029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">45.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-114819v1.45" data-doi="10.1093/nar/25.18.3605"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Esposito</span>, <span class="cit-name-given-names">D.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Scocca</span> <span class="cit-name-given-names">J.J.</span></span> (<span class="cit-pub-date">1997</span>) <span class="cit-article-title">The integrase family of tyrosine recombinases: evolution of a conserved active site domain</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>, <span class="cit-vol">25</span>, <span class="cit-fpage">3605</span>–<span class="cit-lpage">3614</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.18.3605%26rft_id%253Dinfo%253Apmid%252F9278480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/25.18.3605&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9278480&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XW78800008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.46" data-doi="10.1093/nar/26.2.391"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nunes-Düby</span>, <span class="cit-name-given-names">S.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kwon</span>, <span class="cit-name-given-names">H.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tirumalai</span>, <span class="cit-name-given-names">R.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ellenberger</span>, <span class="cit-name-given-names">T.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Landy</span>, <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">1998</span>) <span class="cit-article-title">Similarities and differences among 105 members of the Int family of site-specific recombinases</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr>, <span class="cit-vol">26</span>, <span class="cit-fpage">391</span>–<span class="cit-lpage">406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F26.2.391%26rft_id%253Dinfo%253Apmid%252F9421491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/26.2.391&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9421491&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000071779000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.47" data-doi="10.1016/0022-2836(73)90065-X"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wilson</span> <span class="cit-name-given-names">J.H.</span></span> (<span class="cit-pub-date">1973</span>) <span class="cit-article-title">Function of the bacteriophage T4 transfer RNA’s</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr>, <span class="cit-vol">74</span>, <span class="cit-fpage">753</span>–<span class="cit-lpage">757</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DWilson%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D74%26rft.issue%253D4%26rft.spage%253D753%26rft.epage%253D757%26rft.atitle%253DFunction%2Bof%2Bthe%2Bbacteriophage%2BT4%2Btransfer%2BRNA%2527s.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252873%252990065-X%26rft_id%253Dinfo%253Apmid%252F4729526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(73)90065-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4729526&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1973P034000029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.48" data-doi="10.1073/pnas.61.1.114"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weiss</span>, <span class="cit-name-given-names">S.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hsu</span>, <span class="cit-name-given-names">W.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Foft</span> <span class="cit-name-given-names">J.W.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Scherberg</span>, <span class="cit-name-given-names">N.H.</span></span> (<span class="cit-pub-date">1968</span>) <span class="cit-article-title">Transfer RNA coded by the T4 bacteriophage genome</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A.</abbr>, <span class="cit-vol">61</span>, <span class="cit-fpage">114</span>–<span class="cit-lpage">121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DWeiss%26rft.auinit1%253DS.%2BB.%26rft.volume%253D61%26rft.issue%253D1%26rft.spage%253D114%26rft.epage%253D121%26rft.atitle%253DTransfer%2BRNA%2Bcoded%2Bby%2Bthe%2BT4%2Bbacteriophage%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.61.1.114%26rft_id%253Dinfo%253Apmid%252F4880604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI2MS8xLzExNCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzAzLzA3LzExNDgxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.49" data-doi="10.1016/S0022-5193(05)80725-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kunisawa</span>, <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">1992</span>) <span class="cit-article-title">Synonymous codon preferences in bacteriophage T4: a distinctive use of transfer RNAs from T4 and from its host Escherichia coli</span>. <abbr class="cit-jnl-abbrev">J. Theor. Biol.</abbr>, <span class="cit-vol">159</span>, <span class="cit-fpage">287</span>–<span class="cit-lpage">298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DKunisawa%26rft.auinit1%253DT.%26rft.volume%253D159%26rft.issue%253D3%26rft.spage%253D287%26rft.epage%253D298%26rft.atitle%253DSynonymous%2Bcodon%2Bpreferences%2Bin%2Bbacteriophage%2BT4%253A%2Ba%2Bdistinctive%2Buse%2Bof%2Btransfer%2BRNAs%2Bfrom%2BT4%2Band%2Bfrom%2Bits%2Bhost%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0022-5193%252805%252980725-8%26rft_id%253Dinfo%253Apmid%252F1296091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0022-5193(05)80725-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1296091&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992KH22900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.50" data-doi="10.1101/gr.6649807"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bailly-Bechet</span>, <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vergassola</span>, <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Rocha</span>, <span class="cit-name-given-names">E.</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">Causes for the intriguing presence of tRNAs in phages</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr>, <span class="cit-vol">17</span>, <span class="cit-fpage">1486</span>–<span class="cit-lpage">1495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.6649807%26rft_id%253Dinfo%253Apmid%252F17785533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNy8xMC8xNDg2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDMvMDcvMTE0ODE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.51" data-doi="10.1111/j.1365-2958.2004.04113.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Canchaya</span>, <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fournous</span>, <span class="cit-name-given-names">G.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Brüssow</span>, <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">The impact of prophages on bacterial chromosomes</span>. <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr>, <span class="cit-vol">53</span>, <span class="cit-fpage">9</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DCanchaya%26rft.auinit1%253DC.%26rft.volume%253D53%26rft.issue%253D1%26rft.spage%253D9%26rft.epage%253D18%26rft.atitle%253DThe%2Bimpact%2Bof%2Bprophages%2Bon%2Bbacterial%2Bchromosomes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2004.04113.x%26rft_id%253Dinfo%253Apmid%252F15225299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2004.04113.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15225299&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222208100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.52" data-doi="10.1046/j.1365-2958.2003.03580.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Casjens</span>, <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">Prophages and bacterial genomics: what have we learned so far</span>? <abbr class="cit-jnl-abbrev">Mol Microbiol</abbr>, <span class="cit-vol">49</span>, <span class="cit-fpage">277</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DCasjens%26rft.auinit1%253DS.%26rft.volume%253D49%26rft.issue%253D2%26rft.spage%253D277%26rft.epage%253D300%26rft.atitle%253DProphages%2Band%2Bbacterial%2Bgenomics%253A%2Bwhat%2Bhave%2Bwe%2Blearned%2Bso%2Bfar%253F%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.2003.03580.x%26rft_id%253Dinfo%253Apmid%252F12886937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.2003.03580.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12886937&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000184224700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.53" data-doi="10.1038/ismej.2013.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Frank</span>, <span class="cit-name-given-names">J.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lorimer</span>, <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Youle</span>, <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Witte</span>, <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Craig</span>, <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Abendroth</span>, <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rohwer</span>, <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Edwards</span>, <span class="cit-name-given-names">R.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Segall</span>, <span class="cit-name-given-names">A.M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Burgin</span>, <span class="cit-name-given-names">A.B.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Structure and function of a cyanophage-encoded peptide deformylase</span>. <abbr class="cit-jnl-abbrev">ISME J.</abbr>, <span class="cit-vol">7</span>, <span class="cit-fpage">1150</span>–<span class="cit-lpage">1160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DISME%2BJ.%26rft.volume%253D7%26rft.spage%253D1150%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fismej.2013.4%26rft_id%253Dinfo%253Apmid%252F23407310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ismej.2013.4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23407310&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.54" data-doi="10.1038/ismej.2011.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sharon</span>, <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Battchikova</span>, <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aro</span>, <span class="cit-name-given-names">E.-M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Giglione</span>, <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meinnel</span>, <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Glaser</span>, <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pinter</span>, <span class="cit-name-given-names">R.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Breitbart</span>, <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rohwer</span>, <span class="cit-name-given-names">F.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Béjà</span>, <span class="cit-name-given-names">O.</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Comparative metagenomics of microbial traits within oceanic viral communities</span>. <abbr class="cit-jnl-abbrev">ISME J.</abbr>, <span class="cit-vol">5</span>, <span class="cit-fpage">1178</span>–<span class="cit-lpage">1190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DISME%2BJ%26rft.aulast%253DSharon%26rft.auinit1%253DI.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253D1178%26rft.epage%253D1190%26rft.atitle%253DComparative%2Bmetagenomics%2Bof%2Bmicrobial%2Btraits%2Bwithin%2Boceanic%2Bviral%2Bcommunities.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fismej.2011.2%26rft_id%253Dinfo%253Apmid%252F21307954%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ismej.2011.2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21307954&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295688500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.55" data-doi="10.1128/IAI.68.9.4850-4855.2000"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Muniesa</span>, <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Recktenwald</span>, <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bielaszewska</span>, <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Karch</span>, <span class="cit-name-given-names">H.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Schmidt</span>, <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">2000</span>) <span class="cit-article-title">Characterization of a shiga toxin 2e-converting bacteriophage from an Escherichia coli strain of human origin</span>. <abbr class="cit-jnl-abbrev">Infect. Immun.</abbr>, <span class="cit-vol">68</span>, <span class="cit-fpage">4850</span>–<span class="cit-lpage">4855</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%2Band%2BImmunity%26rft.stitle%253DInfect.%2BImmun.%26rft.aulast%253DMuniesa%26rft.auinit1%253DM.%26rft.volume%253D68%26rft.issue%253D9%26rft.spage%253D4850%26rft.epage%253D4855%26rft.atitle%253DCharacterization%2Bof%2Ba%2BShiga%2BToxin%2B2e-Converting%2BBacteriophage%2Bfrom%2Ban%2BEscherichia%2Bcoli%2BStrain%2Bof%2BHuman%2BOrigin%26rft_id%253Dinfo%253Adoi%252F10.1128%252FIAI.68.9.4850-4855.2000%26rft_id%253Dinfo%253Apmid%252F10948096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaWFpIjtzOjU6InJlc2lkIjtzOjk6IjY4LzkvNDg1MCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzAzLzA3LzExNDgxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.56" data-doi="10.7554/eLife.06416"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pope</span>, <span class="cit-name-given-names">W.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bowman</span>, <span class="cit-name-given-names">C.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Russell</span>, <span class="cit-name-given-names">D.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jacobs-Sera</span>, <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Asai</span>, <span class="cit-name-given-names">D.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cresawn</span>, <span class="cit-name-given-names">S.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jr</span>, <span class="cit-name-given-names">W.R.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hendrix</span>, <span class="cit-name-given-names">R.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lawrence</span>, <span class="cit-name-given-names">J.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hatfull</span>, <span class="cit-name-given-names">G.F.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Whole genome comparison of a large collection of mycobacteriophages reveals a continuum of phage genetic diversity</span>. <abbr class="cit-jnl-abbrev">eLife</abbr>, <span class="cit-vol">4</span>, <span class="cit-fpage">e06416</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.06416%26rft_id%253Dinfo%253Apmid%252F25919952%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.06416&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25919952&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.57"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Miller</span>, <span class="cit-name-given-names">E.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kutter</span>, <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mosig</span>, <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arisaka</span>, <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kunisawa</span>, <span class="cit-name-given-names">T.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Ruger</span>, <span class="cit-name-given-names">W.</span></span> (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">Bacteriophage T4</span> <abbr class="cit-jnl-abbrev">Genome. Microbiol. Mol. Biol. Rev.</abbr>, <span class="cit-vol">67</span>, <span class="cit-fpage">86</span>–<span class="cit-lpage">156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome.%2BMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.volume%253D67%26rft.spage%253D86%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.58" data-doi="10.1016/0042-6822(90)90068-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Marrs</span>, <span class="cit-name-given-names">C.F.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Howe</span>, <span class="cit-name-given-names">M.M.</span></span> (<span class="cit-pub-date">1990</span>) <span class="cit-article-title">Kinetics and regulation of transcription of bacteriophage Mu</span>. <abbr class="cit-jnl-abbrev">Virology</abbr>, <span class="cit-vol">174</span>, <span class="cit-fpage">192</span>–<span class="cit-lpage">203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DMarrs%26rft.auinit1%253DC.%2BF.%26rft.volume%253D174%26rft.issue%253D1%26rft.spage%253D192%26rft.epage%253D203%26rft.atitle%253DKinetics%2Band%2Bregulation%2Bof%2Btranscription%2Bof%2Bbacteriophage%2BMu.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0042-6822%252890%252990068-3%26rft_id%253Dinfo%253Apmid%252F2136777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0042-6822(90)90068-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2136777&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F07%2F114819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">59.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-114819v1.59"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pero</span>, <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hannett</span>, <span class="cit-name-given-names">N.M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Talkington</span>, <span class="cit-name-given-names">C.</span></span> (<span class="cit-pub-date">1979</span>) <span class="cit-article-title">Restriction cleavage map of SP01 DNA: general location of early, middle, and late genes</span>. <abbr class="cit-jnl-abbrev">J. Virol.</abbr>, <span class="cit-vol">31</span>, <span class="cit-fpage">156</span>–<span class="cit-lpage">171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DPero%26rft.auinit1%253DJ%26rft.volume%253D31%26rft.issue%253D1%26rft.spage%253D156%26rft.epage%253D171%26rft.atitle%253DRestriction%2Bcleavage%2Bmap%2Bof%2BSP01%2BDNA%253A%2Bgeneral%2Blocation%2Bof%2Bearly%252C%2Bmiddle%252C%2Band%2Blate%2Bgenes.%26rft_id%253Dinfo%253Apmid%252F116010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjMxLzEvMTU2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDMvMDcvMTE0ODE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-114819v1.60" data-doi="10.1128/AEM.70.8.5041-5046.2004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ventura</span>, <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Brüssow</span>, <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">Temporal Transcription Map of the Virulent Streptococcus thermophilus Bacteriophage Sfi19</span>. <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr>, <span class="cit-vol">70</span>, <span class="cit-fpage">5041</span>–<span class="cit-lpage">5046</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft.stitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft.aulast%253DVentura%26rft.auinit1%253DM.%26rft.volume%253D70%26rft.issue%253D8%26rft.spage%253D5041%26rft.epage%253D5046%26rft.atitle%253DTemporal%2BTranscription%2BMap%2Bof%2Bthe%2BVirulent%2BStreptococcus%2Bthermophilus%2BBacteriophage%2BSfi19%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.70.8.5041-5046.2004%26rft_id%253Dinfo%253Apmid%252F15294848%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjk6IjcwLzgvNTA0MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzAzLzA3LzExNDgxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/114819v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F114819v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/114272v1" title="Can malaria parasites manipulate the odour-mediated host preference of their mosquito vectors?" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/114587v1" title="Soft sweeps and beyond: Understanding the patterns and probabilities of selection footprints under rapid adaptation" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 07, 2017.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/114819v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/114819v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/114819v1" class="active" data-icon-position="" data-hide-link-title="0">Prophage genomics reveals patterns in phage genome organization and replication</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/33254" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-B_wwwyRmK8jrgKiSnequE41viK1plK4Hq7AYI_E-RdQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="33254" id="node33254--2" data-pisa="biorxiv;114819v1" data-pisa-master="biorxiv;114819" data-seqnum="33254" data-apath="/biorxiv/early/2017/03/07/114819.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Prophage genomics reveals patterns in phage genome organization and replication</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Han Suh</span> <span class="nlm-surname">Kang</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Katelyn</span> <span class="nlm-surname">McNair</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Daniel A.</span> <span class="nlm-surname">Cuevas</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Barbara A.</span> <span class="nlm-surname">Bailey</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Anca M.</span> <span class="nlm-surname">Segall</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Robert A.</span> <span class="nlm-surname">Edwards</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">114819; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/114819 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/114819v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/114819v1&amp;title=Prophage%20genomics%20reveals%20patterns%20in%20phage%20genome%20organization%20and%20replication" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/114819v1&amp;title=Prophage%20genomics%20reveals%20patterns%20in%20phage%20genome%20organization%20and%20replication" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/114819v1&amp;text=Prophage%20genomics%20reveals%20patterns%20in%20phage%20genome%20organization%20and%20replication" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/114819v1&amp;title=Prophage%20genomics%20reveals%20patterns%20in%20phage%20genome%20organization%20and%20replication" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/114819v1&amp;t=Prophage%20genomics%20reveals%20patterns%20in%20phage%20genome%20organization%20and%20replication" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/114819v1&amp;title=Prophage%20genomics%20reveals%20patterns%20in%20phage%20genome%20organization%20and%20replication" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/114819v1&amp;title=Prophage%20genomics%20reveals%20patterns%20in%20phage%20genome%20organization%20and%20replication" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="33254" id="node33254--3" data-pisa="biorxiv;114819v1" data-pisa-master="biorxiv;114819" data-seqnum="33254" data-apath="/biorxiv/early/2017/03/07/114819.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Prophage genomics reveals patterns in phage genome organization and replication</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Han Suh</span> <span class="nlm-surname">Kang</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Katelyn</span> <span class="nlm-surname">McNair</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Daniel A.</span> <span class="nlm-surname">Cuevas</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Barbara A.</span> <span class="nlm-surname">Bailey</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Anca M.</span> <span class="nlm-surname">Segall</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Robert A.</span> <span class="nlm-surname">Edwards</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">114819; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/114819 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Prophage%20genomics%20reveals%20patterns%20in%20phage%20genome%20organization%20and%20replication&amp;rft.volume&amp;rft.issue&amp;rft.date=2017-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Kang%2C+Han+Suh&amp;rft.au=McNair%2C+Katelyn&amp;rft.au=Cuevas%2C+Daniel+A.&amp;rft.au=Bailey%2C+Barbara+A.&amp;rft.au=Segall%2C+Anca+M.&amp;rft.au=Edwards%2C+Robert+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/33254/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/33254/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/33254/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/33254/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/33254/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/33254/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/33254/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/33254/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/33254/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/33254/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/33254/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/33254/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/114819v1&amp;count=horizontal&amp;via=&amp;text=Prophage%20genomics%20reveals%20patterns%20in%20phage%20genome%20organization%20and%20replication&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/114819v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/114819v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/114819v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/biochemistry" class="highlight" data-icon-position="" data-hide-link-title="0">Biochemistry<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20277)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
