<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/03/28/2020.03.23.004176.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.23.004176v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Structure-based modeling of SARS-CoV-2 peptide/HLA-A02 antigens" />
<meta name="DC.Identifier" content="10.1101/2020.03.23.004176" />
<meta name="DC.Date" content="2020-03-28" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="As a first step toward the development of diagnostic and therapeutic tools to fight the Coronavirus disease (COVID-19), it is important to characterize CD8+ T cell epitopes in the SARS-CoV-2 peptidome that can trigger adaptive immune responses. Here, we use RosettaMHC, a comparative modeling approach which leverages existing high-resolution X-ray structures from peptide/MHC complexes available in the Protein Data Bank, to derive physically realistic 3D models for high-affinity SARS-CoV-2 epitopes. We outline an application of our method to model 439 9mer and 279 10mer predicted epitopes displayed by the common allele HLA-A*02:01, and we make our models publicly available through an online database (&lt;https://rosettamhc.chemistry.ucsc.edu&gt;). As more detailed studies on antigen-specific T cell recognition become available, RosettaMHC models of antigens from different strains and HLA alleles can be used as a basis to understand the link between peptide/HLA complex structure and surface chemistry with immunogenicity, in the context of SARS-CoV-2 infection." />
<meta name="DC.Contributor" content="Santrupti Nerli" />
<meta name="DC.Contributor" content="Nikolaos G. Sgourakis" />
<meta name="article:published_time" content="2020-03-28" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Structure-based modeling of SARS-CoV-2 peptide/HLA-A02 antigens" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;As a first step toward the development of diagnostic and therapeutic tools to fight the Coronavirus disease (COVID-19), it is important to characterize CD8+ T cell epitopes in the SARS-CoV-2 peptidome that can trigger adaptive immune responses. Here, we use RosettaMHC, a comparative modeling approach which leverages existing high-resolution X-ray structures from peptide/MHC complexes available in the Protein Data Bank, to derive physically realistic 3D models for high-affinity SARS-CoV-2 epitopes. We outline an application of our method to model 439 9mer and 279 10mer predicted epitopes displayed by the common allele HLA-A*02:01, and we make our models publicly available through an online database (https://rosettamhc.chemistry.ucsc.edu). As more detailed studies on antigen-specific T cell recognition become available, RosettaMHC models of  antigens from different strains and HLA alleles can be used as a basis to understand the link between peptide/HLA complex structure and surface chemistry with immunogenicity, in the context of SARS-CoV-2 infection.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.23.004176v2" />
<meta name="citation_id" content="2020.03.23.004176v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.23.004176v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.23.004176v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.23.004176v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.23.004176.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.23.004176" />
<meta name="citation_num_pages" content="15" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.23.004176" />
<meta name="citation_author" content="Santrupti Nerli" />
<meta name="citation_author_institution" content="Department of Biomolecular Engineering, University of California Santa Cruz" />
<meta name="citation_author" content="Nikolaos G. Sgourakis" />
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, University of California Santa Cruz" />
<meta name="citation_author_email" content="nsgourak@ucsc.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3655-3902" />
<meta name="citation_reference" content="Liu, C., Q. Zhou, Y. Li, L. V. Garner, S. P. Watkins, L. J. Carter, J. Smoot, A. C. Gregg, A. D. Daniels, S. Jervey, and D. Albaiu. 2020. Research and Development on Therapeutic Agents and Vaccines for COVID-19 and Related Human Coronavirus Diseases. ACS Cent. Sci." />
<meta name="citation_reference" content="Kumaki, Y., J. Ennis, R. Rahbar, J. D. Turner, M. K. Wandersee, A. J. Smith, K. W. Bailey, Z. G. Vest, J. R. Madsen, J. K.-K. Li, and D. L. Barnard. 2011. Single-dose intranasal administration with mDEF201 (adenovirus vectored mouse interferon-alpha) confers protection from mortality in a lethal SARS-CoV BALB/c mouse model. Antiviral Res. 89: 75–82." />
<meta name="citation_reference" content="Kishimoto, T. 2006. Interleukin-6: discovery of a pleiotropic cytokine. Arthritis Res. Ther. 8: S2." />
<meta name="citation_reference" content="Rock, K. L., E. Reits, and J. Neefjes. 2016. Present Yourself! By MHC Class I and MHC Class II Molecules. Trends Immunol. 37: 724–737." />
<meta name="citation_reference" content="Kaufman, J. 2018. Unfinished Business: Evolution of the MHC and the Adaptive Immune System of Jawed Vertebrates. Annu. Rev. Immunol. 36: 383–409." />
<meta name="citation_reference" content="Thevarajan, I., T. H. O. Nguyen, M. Koutsakos, J. Druce, L. Caly, C. E. van de Sandt, X. Jia, S. Nicholson, M. Catton, B. Cowie, S. Y. C. Tong, S. R. Lewin, and K. Kedzierska. 2020. Breadth of concomitant immune responses prior to patient recovery: a case report of non-severe COVID-19. Nat. Med. 1–3." />
<meta name="citation_reference" content="Goronzy, J. J., F. Fang, M. M. Cavanagh, Q. Qi, and C. M. Weyand. 2015. Naïve T cell maintenance and function in human aging. J. Immunol. Baltim. Md 1950 194: 4073–4080." />
<meta name="citation_reference" content="Oyarzun, P., and B. Kobe. 2015. Computer-aided design of T-cell epitope-based vaccines: addressing population coverage. Int. J. Immunogenet. 42: 313–321." />
<meta name="citation_reference" content="Birnbaum, M. E., J. L. Mendoza, D. K. Sethi, S. Dong, J. Glanville, J. Dobbins, E. Ozkan, M. M. Davis, K. W. Wucherpfennig, and K. C. Garcia. 2014. Deconstructing the peptide-MHC specificity of T cell recognition. Cell 157: 1073–1087." />
<meta name="citation_reference" content="Wooldridge, L., J. Ekeruche-Makinde, H. A. van den Berg, A. Skowera, J. J. Miles, M. P. Tan, G. Dolton, M. Clement, S. Llewellyn-Lacey, D. A. Price, M. Peakman, and A. K. Sewell. 2012. A single autoimmune T cell receptor recognizes more than a million different peptides. J. Biol. Chem. 287: 1168–1177." />
<meta name="citation_reference" content="Peters, B., M. Nielsen, and A. Sette. 2020. T Cell Epitope Predictions. Annu. Rev. Immunol." />
<meta name="citation_reference" content="Vita, R., S. Mahajan, J. A. Overton, S. K. Dhanda, S. Martini, J. R. Cantrell, D. K. Wheeler, A. Sette, and B. Peters. 2019. The Immune Epitope Database (IEDB): 2018 update. Nucleic Acids Res. 47: D339–D343." />
<meta name="citation_reference" content="Antunes, D. A., J. R. Abella, D. Devaurs, M. M. Rigo, and L. E. Kavraki. 2018. Structure-based Methods for Binding Mode and Binding Affinity Prediction for Peptide-MHC Complexes. Curr. Top. Med. Chem. 18: 2239–2255." />
<meta name="citation_reference" content="Yanover, C., and P. Bradley. 2011. Large-scale characterization of peptide-MHC binding landscapes with structural simulations. Proc. Natl. Acad. Sci. 108: 6981–6986." />
<meta name="citation_reference" content="King, C., E. N. Garza, R. Mazor, J. L. Linehan, I. Pastan, M. Pepper, and D. Baker. 2014. Removing T-cell epitopes with computational protein design. Proc. Natl. Acad. Sci. U. S. A. 111: 8577–8582." />
<meta name="citation_reference" content="Liu, T., X. Pan, L. Chao, W. Tan, S. Qu, L. Yang, B. Wang, and H. Mei. 2014. Subangstrom accuracy in pHLA-I modeling by Rosetta FlexPepDock refinement protocol. J. Chem. Inf. Model. 54: 2233–2242." />
<meta name="citation_reference" content="Kyeong, H.-H., Y. Choi, and H.-S. Kim. 2018. GradDock: rapid simulation and tailored ranking functions for peptide-MHC Class I docking. Bioinformatics 34: 469–476." />
<meta name="citation_reference" content="Rubenstein, A. B., M. A. Pethe, and S. D. Khare. 2017. MFPred: Rapid and accurate prediction of protein-peptide recognition multispecificity using self-consistent mean field theory. PLoS Comput. Biol. 13: e1005614." />
<meta name="citation_reference" content="Song, Y., F. DiMaio, R. Y.-R. Wang, D. Kim, C. Miles, T. Brunette, J. Thompson, and D. Baker. 2013. High-Resolution Comparative Modeling with RosettaCM. Structure 21: 1735–1742." />
<meta name="citation_reference" content="Frontiers | A Recurrent Mutation in Anaplastic Lymphoma Kinase with Distinct Neoepitope Conformations | Immunology." />
<meta name="citation_reference" content="Robinson, J., L. A. Guethlein, N. Cereb, S. Y. Yang, P. J. Norman, S. G. E. Marsh, and P. Parham. 2017. Distinguishing functional polymorphism from random variation in the sequences of &gt;10,000 HLA-A, -B and -C alleles. PLoS Genet. 13." />
<meta name="citation_reference" content="Berman, H. M., J. Westbrook, Z. Feng, G. Gilliland, T. N. Bhat, H. Weissig, I. N. Shindyalov, and P. E. Bourne. 2000. The Protein Data Bank. Nucleic Acids Res. 28: 235–242." />
<meta name="citation_reference" content="Bentzen, A. K., A. M. Marquard, R. Lyngaa, S. K. Saini, S. Ramskov, M. Donia, L. Such, A. J. S. Furness, N. McGranahan, R. Rosenthal, P. T. Straten, Z. Szallasi, I. M. Svane, C. Swanton, S. A. Quezada, S. N. Jakobsen, A. C. Eklund, and S. R. Hadrup. 2016. Large-scale detection of antigen-specific T cells using peptide-MHC-I multimers labeled with DNA barcodes. Nat. Biotechnol. 34: 1037–1045." />
<meta name="citation_reference" content="Saini, S. K., T. Tamhane, R. Anjanappa, A. Saikia, S. Ramskov, M. Donia, I. M. Svane, S. N. Jakobsen, M. Garcia-Alai, M. Zacharias, R. Meijers, S. Springer, and S. R. Hadrup. 2019. Empty peptide-receptive MHC class I molecules for efficient detection of antigen-specific T cells. Sci. Immunol. 4." />
<meta name="citation_reference" content="Overall, S. A., J. S. Toor, S. Hao, M. Yarmarkovich, S. M. O’Rourke, G. I. Morozov, S. Nguyen, A. S. Japp, N. Gonzalez, D. Moschidi, M. R. Betts, J. M. Maris, P. Smibert, and N. G. Sgourakis. High Throughput pMHC-I Tetramer Library Production Using Chaperone-Mediated Peptide Exchange. Nat. Commun. Press." />
<meta name="citation_reference" content="Ishizuka, J., K. Grebe, E. Shenderov, B. Peters, Q. Chen, Y. Peng, L. Wang, T. Dong, V. Pasquetto, C. Oseroff, J. Sidney, H. Hickman, V. Cerundolo, A. Sette, J. R. Bennink, A. McMichael, and J. W. Yewdell. 2009. Quantitating T Cell Cross-Reactivity for Unrelated Peptide Antigens. J. Immunol. 183: 4337–4345." />
<meta name="citation_reference" content="Jurtz, V., S. Paul, M. Andreatta, P. Marcatili, B. Peters, and M. Nielsen. 2017. NetMHCpan-4.0: Improved Peptide-MHC Class I Interaction Predictions Integrating Eluted Ligand and Peptide Binding Affinity Data. J. Immunol. Baltim. Md 1950 199: 3360–3368." />
<meta name="citation_reference" content="Henikoff, S., and J. G. Henikoff. 1992. Amino acid substitution matrices from protein blocks. Proc. Natl. Acad. Sci. U. S. A. 89: 10915–10919." />
<meta name="citation_reference" content="Chaudhury, S., S. Lyskov, and J. J. Gray. 2010. PyRosetta: a script-based interface for implementing molecular modeling algorithms using Rosetta. Bioinformatics 26: 689–691." />
<meta name="citation_reference" content="Sievers, F., A. Wilm, D. Dineen, T. J. Gibson, K. Karplus, W. Li, R. Lopez, H. McWilliam, M. Remmert, J. Söding, J. D. Thompson, and D. G. Higgins. 2011. Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega. Mol. Syst. Biol. 7: 539." />
<meta name="citation_reference" content="Tyka, M. D., D. A. Keedy, I. André, F. Dimaio, Y. Song, D. C. Richardson, J. S. Richardson, and D. Baker. 2011. Alternate states of proteins revealed by detailed energy landscape mapping. J. Mol. Biol. 405: 607–618." />
<meta name="citation_reference" content="Wu, F., S. Zhao, B. Yu, Y.-M. Chen, W. Wang, Z.-G. Song, Y. Hu, Z.-W. Tao, J.-H. Tian, Y.-Y. Pei, M.-L. Yuan, Y.-L. Zhang, F.-H. Dai, Y. Liu, Q.-M. Wang, J.-J. Zheng, L. Xu, E. C. Holmes, and Y.-Z. Zhang. 2020. A new coronavirus associated with human respiratory disease in China. Nature 579: 265–269." />
<meta name="citation_reference" content="Kent, W. J., C. W. Sugnet, T. S. Furey, K. M. Roskin, T. H. Pringle, A. M. Zahler, and and D. Haussler. 2002. The Human Genome Browser at UCSC. Genome Res. 12: 996–1006." />
<meta name="citation_reference" content="Trolle, T., C. P. McMurtrey, J. Sidney, W. Bardet, S. C. Osborn, T. Kaever, A. Sette, W. H. Hildebrand, M. Nielsen, and B. Peters. 2016. The length distribution of class I restricted T cell epitopes is determined by both peptide supply and MHC allele specific binding preference. J. Immunol. Baltim. Md 1950 196: 1480–1487." />
<meta name="citation_reference" content="Mishto, M., and J. Liepe. 2017. Post-Translational Peptide Splicing and T Cell Responses. Trends Immunol. 38: 904–915." />
<meta name="citation_reference" content="Alford, R. F., A. Leaver-Fay, J. R. Jeliazkov, M. O’Meara, F. P. DiMaio, H. Park, M. V. Shapovalov, P. D. Renfrew, V. K. Mulligan, K. Kappel, J. W. Labonte, M. S. Pacella, R. Bonneau, P. Bradley, R. L. Dunbrack Jr., R. Das, D. Baker, B. Kuhlman, T. Kortemme, and J. J. Gray. 2017. The Rosetta All-Atom Energy Function for Macromolecular Modeling and Design. J. Chem. Theory Comput. 13: 3031–3048." />
<meta name="citation_reference" content="Grifoni, A., J. Sidney, Y. Zhang, R. H. Scheuermann, B. Peters, and A. Sette. 2020. A Sequence Homology and Bioinformatic Approach Can Predict Candidate Targets for Immune Responses to SARS-CoV-2. Cell Host Microbe." />
<meta name="citation_reference" content="Pickett, B. E., E. L. Sadat, Y. Zhang, J. M. Noronha, R. B. Squires, V. Hunt, M. Liu, S. Kumar, S. Zaremba, Z. Gu, L. Zhou, C. N. Larson, J. Dietrich, E. B. Klem, and R. H. Scheuermann. 2012. ViPR: an open bioinformatics database and analysis resource for virology research. Nucleic Acids Res. 40: D593–D598." />
<meta name="citation_reference" content="Rossjohn, J., S. Gras, J. J. Miles, S. J. Turner, D. I. Godfrey, and J. McCluskey. 2015. T Cell Antigen Receptor Recognition of Antigen-Presenting Molecules. Annu. Rev. Immunol. 33: 169–200." />
<meta name="citation_reference" content="Rudolph, M. G., R. L. Stanfield, and I. A. Wilson. 2006. How TCRs bind MHCs, peptides, and coreceptors. Annu. Rev. Immunol. 24: 419–466." />
<meta name="citation_reference" content="Borrman, T., J. Cimons, M. Cosiano, M. Purcaro, B. G. Pierce, B. M. Baker, and Z. Weng. 2017. ATLAS: A database linking binding affinities with structures for wild-type and mutant TCR-pMHC complexes. Proteins Struct. Funct. Bioinforma. 85: 908–916." />
<meta name="citation_reference" content="Riley, T. P., G. L. J. Keller, A. R. Smith, L. M. Davancaze, A. G. Arbuiso, J. R. Devlin, and B. M. Baker. 2019. Structure Based Prediction of Neoantigen Immunogenicity. Front. Immunol. 10." />
<meta name="citation_reference" content="Mallon, D. H., C. Kling, M. Robb, E. Ellinghaus, J. A. Bradley, C. J. Taylor, D. Kabelitz, and V. Kosmoliaptsis. 2018. Predicting Humoral Alloimmunity from Differences in Donor and Recipient HLA Surface Electrostatic Potential. J. Immunol. 201: 3780–3792." />
<meta name="citation_reference" content="Baker, N. A., D. Sept, S. Joseph, M. J. Holst, and J. A. McCammon. 2001. Electrostatics of nanosystems: application to microtubules and the ribosome. Proc. Natl. Acad. Sci. U. S. A. 98: 10037–10041." />
<meta name="citation_reference" content="Jurrus, E., D. Engel, K. Star, K. Monson, J. Brandi, L. E. Felberg, D. H. Brookes, L. Wilson, J. Chen, K. Liles, M. Chun, P. Li, D. W. Gohara, T. Dolinsky, R. Konecny, D. R. Koes, J. E. Nielsen, T. Head-Gordon, W. Geng, R. Krasny, G.-W. Wei, M. J. Holst, J. A. McCammon, and N. A. Baker. 2018. Improvements to the APBS biomolecular solvation software suite. Protein Sci. 27: 112–128." />
<meta name="citation_reference" content="The PyMOL Molecular Graphics System,. Schrödinger, LLC." />
<meta name="twitter:title" content="Structure-based modeling of SARS-CoV-2 peptide/HLA-A02 antigens" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="As a first step toward the development of diagnostic and therapeutic tools to fight the Coronavirus disease (COVID-19), it is important to characterize CD8+ T cell epitopes in the SARS-CoV-2 peptidome that can trigger adaptive immune responses. Here, we use RosettaMHC, a comparative modeling approach which leverages existing high-resolution X-ray structures from peptide/MHC complexes available in the Protein Data Bank, to derive physically realistic 3D models for high-affinity SARS-CoV-2 epitopes. We outline an application of our method to model 439 9mer and 279 10mer predicted epitopes displayed by the common allele HLA-A*02:01, and we make our models publicly available through an online database (&lt;https://rosettamhc.chemistry.ucsc.edu&gt;). As more detailed studies on antigen-specific T cell recognition become available, RosettaMHC models of antigens from different strains and HLA alleles can be used as a basis to understand the link between peptide/HLA complex structure and surface chemistry with immunogenicity, in the context of SARS-CoV-2 infection." />
<meta name="og-title" property="og:title" content="Structure-based modeling of SARS-CoV-2 peptide/HLA-A02 antigens" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.23.004176v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="As a first step toward the development of diagnostic and therapeutic tools to fight the Coronavirus disease (COVID-19), it is important to characterize CD8+ T cell epitopes in the SARS-CoV-2 peptidome that can trigger adaptive immune responses. Here, we use RosettaMHC, a comparative modeling approach which leverages existing high-resolution X-ray structures from peptide/MHC complexes available in the Protein Data Bank, to derive physically realistic 3D models for high-affinity SARS-CoV-2 epitopes. We outline an application of our method to model 439 9mer and 279 10mer predicted epitopes displayed by the common allele HLA-A*02:01, and we make our models publicly available through an online database (&lt;https://rosettamhc.chemistry.ucsc.edu&gt;). As more detailed studies on antigen-specific T cell recognition become available, RosettaMHC models of antigens from different strains and HLA alleles can be used as a basis to understand the link between peptide/HLA complex structure and surface chemistry with immunogenicity, in the context of SARS-CoV-2 infection." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-03-28" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.23.004176v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.23.004176v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.23.004176v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.23.004176v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1215884" />
    <title>Structure-based modeling of SARS-CoV-2 peptide/HLA-A02 antigens | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Taaap4LB7-8JE04M0Gi3AjIlCXYAhUhAfHi4urxc_5Y","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1215884","apath":"\/biorxiv\/early\/2020\/03\/28\/2020.03.23.004176.atom","pisa":"biorxiv;2020.03.23.004176v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.23.004176v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.23.004176v2","title":"Structure-based modeling of SARS-CoV-2 peptide\/HLA-A02 antigens","identifier":"node\/1215884"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022J\\\\\\\/0+jYkW9YEUj2HZMSw+s8u16Y+KSWSK85UHGkFfE7qEN6qN5pkmhBkAj4KY0Yx1jsmzmYhYup0fgVWHg2witQmEdEIr\\\\\\\/u2yR6FIHJ\\\\\\\/Au\\\\\\\/wp\\\\\\\/5h0\\\\\\\/VT8OwxqT48JaYNf6pt5uGjriq2IFuNFRQunoqU+WLppw7ty6EPn6fk18cUyisDAH+AEsH9v7dUzHWOAlMVyCRcAhZWw5y+ect0LZvvabJvjjtBuXfwlOetNCPRjxv8WT1fz1uxpGyvk8Hq41\\\\\\\/nPmVl1iKCPiWWcnmPJuf+myRXdT9SYFUxiPg0qLMircXK6xdGD9IRbEnyKpquTZra0VyAguywDdYHLsOP4VeuQmF+W8bq++RaPF+Z98nG9p0x57S1nHwOvT04HSGt\\\\\\\/hHMpExarwoKas7JpoxmLca\\\\\\\/1jv0c1sP5srrp0bnZxjeM805gjAxEnCaOCjGNt4eACRkmQQKGqVXa4nRZ29N9CsqpoMcRd16TYlcwuN2zhtX\\\\\\\/oSVHYfnfsnVpcVfoUTA3hgjJFd275NLMRKlnZS8KVhf8DUHrECL3w82xCBbtUHZ8C+qWzH7mFucltQtWHi4bIIyfYxcpDfiJa8BAd7L6sWWvLLEp7GxpGDG2\\\\\\\/IA3QXSbIXuCtE1d440iTY6NjAm\\\\\\\/pzDKHXq9UeY3QWZ1h8bPNV9FlV99dUgHN00C36wWFN5aFjpAHHZK9uw26Yd2No1oLEZPC4OK9hTINDw58ft5S8QE8Qau8akinJ51kTH119uz3U85EyDDLqD1LQmGQk5h6utK3aKzAyGet42c85s8EJoJ8O03DItFHOH3Qkp1k1mPmz7UBsWbihOl4S6MDbVhoEbInGshDVi4rEiDxWDEc2T+kMnlSxcdfH5ubPMIju3lxeiY3WQzcftHwO8cl0jrwM\\\\\\\/xXONtElt5\\\\\\\/VDrh+J2J5Hy0SXma4UPvxA4X6XuSE9Isaix0ctSdFbsLilKU2YIfJ1KilFluXK94rA6HgTotkgEkeRAeAJ92Mb3PYwF9EF\\\\\\\/6D1y5lPO6h\\\\\\\/kjqcvk9lv2rxc88g+AdzMnBvRWyJ31RaAisW3QqiomlMdKkndczeiKYwElmetpECgP93LD3e\\\\\\\/HbyKWSyHnQ1qKKxv8j711mGH7Ha3pc\\\\\\\/wGFO\\\\\\\/p\\\\\\\/xMhU1z8N8x5wiiq148DdHOf+NrZD8LNqnmJr5azsd1SZayno5WJzo+FHusfMpar8xI8ERbPRKxgTMg4aslAeHupV+QvicicPV39PEmVYP941mxj6Wkv4RGteLugjB+HpCrZtu41pb6xocWBNd5pUybMokbVtlVd0ZqYcVLHdozt8k4c+a6Quuk+Ffpbu6UiLOC0gXxo\\\\\\\/7fnQWJGoWSnNW\\\\\\\/F49K\\\u0022,\\\u0022iv\\\u0022:\\\u002215jXqKubjoxDfRWgQph+7Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00223a856c29bccf64989c5a6b96e266cbc3feff4bb196429e662fb8a424260d1a77\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.23.004176v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1215884 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.23.004176v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-MWbz7tBMMhiA_LTY9Gvg_11XYfq6wWLAbHSnsgZGgx0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1215884" id="node1215884" data-pisa="biorxiv;2020.03.23.004176v2" data-pisa-master="biorxiv;2020.03.23.004176" data-apath="/biorxiv/early/2020/03/28/2020.03.23.004176.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Structure-based modeling of SARS-CoV-2 peptide/HLA-A02 antigens</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Santrupti</span> <span class="nlm-surname">Nerli</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-3655-3902" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Nikolaos G.</span> <span class="nlm-surname">Sgourakis</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.23.004176 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1215884" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Santrupti Nerli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Biomolecular Engineering, University of California Santa Cruz</span>, Santa Cruz, CA 95064, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Santrupti%2BNerli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nerli%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASantrupti%2BNerli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Nikolaos G. Sgourakis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Chemistry and Biochemistry, University of California Santa Cruz</span>, Santa Cruz, CA 95064, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nikolaos%2BG.%2BSgourakis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sgourakis%20NG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANikolaos%2BG.%2BSgourakis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3655-3902" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nikolaos G. Sgourakis</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nsgourak@ucsc.edu" class="" data-icon-position="" data-hide-link-title="0">nsgourak@ucsc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.23.004176v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1215884" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1215884/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.23.004176v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1215884" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1215884/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.23.004176v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1215884" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1215884/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.23.004176v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1215884" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1215884/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.23.004176v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1215884" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1215884/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.23.004176v2.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:1215884" data-trigger="external-links" data-url-enabled="1">Data/Code</a><a href="/panels_ajax_tab/article_tab_data_code/node:1215884/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.23.004176v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1215884" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1215884/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">ABSTRACT</h2><p id="p-2">As a first step toward the development of diagnostic and therapeutic tools to fight the Coronavirus disease (COVID-19), it is important to characterize CD8+ T cell epitopes in the SARS-CoV-2 peptidome that can trigger adaptive immune responses. Here, we use RosettaMHC, a comparative modeling approach which leverages existing high-resolution X-ray structures from peptide/MHC complexes available in the Protein Data Bank, to derive physically realistic 3D models for high-affinity SARS-CoV-2 epitopes. We outline an application of our method to model 439 9mer and 279 10mer predicted epitopes displayed by the common allele HLA-A*02:01, and we make our models publicly available through an online database (<a href="https://rosettamhc.chemistry.ucsc.edu">https://rosettamhc.chemistry.ucsc.edu</a>). As more detailed studies on antigen-specific T cell recognition become available, RosettaMHC models of antigens from different strains and HLA alleles can be used as a basis to understand the link between peptide/HLA complex structure and surface chemistry with immunogenicity, in the context of SARS-CoV-2 infection.</p></div><p id="p-6">An ongoing pandemic caused by the novel SARS coronavirus (SARS-CoV-2) has become the focus of extensive efforts to develop vaccines and antiviral therapies (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>). Immune modulatory interferons, which promote a widespread antiviral reaction in infected cells, and inhibition of pro-inflammatory cytokine function through anti-IL-6/IL-6R antibodies, have been proposed as possible COVID-19 therapies (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>). However, stimulating a targeted T cell response against specific viral antigens is hampered by a lack of detailed knowledge of the immunodominant epitopes displayed by common Human Leukocyte Antigen (HLA) alleles across individuals (public epitopes). The molecules of the class I major histocompatibility complex (MHC-I, or HLA in humans) display on the cell surface a diverse pool of 8 to 15 amino acid peptides derived from the endogenous processing of proteins expressed inside the cell (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>). This MHC-I restriction of peptide antigens provides jawed vertebrates with an essential mechanism for adaptive immunity: surveillance of the displayed peptide/MHC-I (pMHC-I) molecules by CD8+ cytotoxic T-lymphocytes allows detection of aberrant protein expression patterns, which signify viral infection and can trigger an adaptive immune response (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>). A recent study has shown important changes in T cell compartments during the acute phase of SARS-CoV-2 infection (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>), suggesting that the ability to quantify antigen-specific T cells would provide new avenues for understanding the expansion and contraction of the TCR repertoire in different disease cohorts and clinical settings. Given the reduction in breadth and functionality of the naïve T cell repertoire during aging (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>), identifying a minimal set of viral antigens that can elicit a protective response will enable the design of diagnostic tools to monitor critical gaps in the T cell repertoire of high-risk cohorts, which can be addressed using peptide or epitope string DNA vaccines (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>).</p><p id="p-7">Human MHC-I molecules are extremely polymorphic, with thousands of known alleles in the classical HLA-A, -B and -C loci. Specific amino acid polymorphisms along the peptide-binding groove (termed A-F pockets) define a repertoire of 10<sup>4</sup>-10<sup>6</sup> peptide antigens that can be recognized by each HLA allotype (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>). Several machine-learning methods have been developed to predict the likelihood that a target peptide will bind to a given allele (reviewed in (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>)). Generally these methods make use of available data sets in the Immune Epitope Database (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>) to train artificial neural networks that predict peptide processing, binding and display, and their performance varies depending on peptide length and HLA allele representation in the database. Structure-based approaches have also been proposed to model the bound peptide conformation <em>de novo</em> (reviewed in (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>)). These approaches utilize various algorithms to optimize the backbone and side chain degrees of freedom of the peptide/MHC structure according to an all-atom scoring function, derived from physical principles (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>–<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>), that can be further enhanced using modified scoring terms (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>) or mean field theory (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>). While these methods do not rely on large training data sets, their performance is affected by bottlenecks in sampling of different backbone conformations, and any possible structural adaptations of the HLA peptide-binding groove.</p><p id="p-8">Predicting the bound peptide conformation whose N- and C-termini are anchored within a fixed-length groove is a tractable modeling problem that can be addressed using standard comparative modeling approaches (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>). In previous work focusing on the HLA-B*15:01 and HLA-A*01:01 alleles in the context of neuroblastoma neoantigens, we have found that a combined backbone and side chain optimization approach can yield accurate pMHC-I models for a pool of target peptides, provided that a reliable template of the same allele and peptide length can be identified in the database (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>). In this approach (RosettaMHC), a local optimization of the backbone degrees of freedom is sufficient to capture minor (within 0.5 Å heavy atom RMSD) changes of the target peptide backbone relative to the conformation of the peptide in the template, used as a starting point. For HLA-A*02:01, the most common HLA allele among disease-relevant population cohorts (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>), there is a large number of high-resolution X-ray structures available in the PDB (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>), suggesting that a similar principle can be applied to produce models of candidate epitopes directly from the proteome of a pathogen of interest. Here, we apply RosettaMHC to all HLA-A*02:01 epitopes predicted directly from the ~30 kbp SARS-CoV-2 genome, and make our models publicly available through an online database. The computed binding energies of our models can be used as an additional validation layer to select high-affinity epitopes from large peptide sets. As detailed epitope mapping data from high-throughput tetramer staining (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>–<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>) and T cell functional screens (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>) become available, the models presented here can provide a toehold for understanding links between pMHC-I antigen structure and immunogenicity, with actionable value for the development of peptide vaccines to combat the disease.</p><div class="section" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Identification of SARS-CoV-2 peptide epitopes</h3><p id="p-9">The SARS-CoV-2 protein sequences (<a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_045512.2">https://www.ncbi.nlm.nih.gov/nuccore/NC_045512.2</a>) were obtained from NCBI and used to generate all possible peptides of lengths 9 and 10 (9,621 9mer and 9,611 10mer peptides). We used NetMHCpan-4.0 (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>) to derive binding scores to HLA-A*02:01, and retained only peptides classified as strong or weak binders (selected using the default percentile rank cut-off values). The binding classification was performed using eluted ligand likelihood predictions. While in this study we use NetMHCpan-4.0 predictions as inputs to select candidate epitopes for structure modeling, our workflow is fully compatible with any alternative epitope prediction method.</p></div><div id="sec-3" class="subsection"><h3>Selection of PDB templates</h3><p id="p-10">To model SARS-CoV-2 / HLA-A*02:01 antigens, we identified 3D structures from the PDB that can be used as templates for comparative modeling. First, we selected all HLA-A02 X-ray structures that are below 3.5 Å resolution and retained only those that have 100% identity to the HLA-A*02:01 heavy chain sequence (residues 1-180). We found 241 template structures bound to epitopes of lengths from 8 to 15 residues (of which 170 are 9mers and 61 are 10mers). For each SARS-CoV-2 target peptide, we selected a set of candidate templates of the same length by matching the target peptide anchor positions (P2 and P9/P10) to each peptide in the template structures. Then, we used the BLOSUM62 (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>) substitution matrix to score all remaining positions in the pairwise alignment of the target/template sequences, and the PDB template with the top alignment score was selected for modeling. For target peptides where we found no templates which match both peptide anchors, we scored all positions in the pairwise alignment and selected the top scoring template for modeling.</p></div><div id="sec-4" class="subsection"><h3>RosettaMHC modeling framework and database</h3><p id="p-11">RosettaMHC (manuscript in preparation) is a comparative modeling protocol developed using PyRosetta (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>) to model pMHC-I complexes. The program accepts as input a list of peptide sequences, an HLA allele definition and a template PDB file (selected as described in the previous step). To minimize “noise” in the simulation from parts of the MHC-I fold that do not contribute to peptide binding, only the α<sub>1</sub> and α<sub>2</sub> domains are considered in all steps. For each peptide, a full alignment between the target and template peptide/MHC sequences is performed using clustal omega (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>). The alignment is used as input to Rosetta’s threading protocol (<em>partial_thread.&lt;ext&gt;</em>). From the threaded model, all residues in the MHC-I groove that are within a heavy-atom distance of 3.5 Å from the peptide are subjected to 10 independent all-atom refinement simulations using the FastRelax method (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>) and a custom movemap file. Binding energies are extracted from the refined structures using interface analyzer protocol (<em>InterfaceAnalyzer.&lt;ext&gt;</em>). The top three models are selected based on the binding energies, and used to compute an average energy for each peptide in the input list. RosettaMHC models of SARS-CoV-2/HLA-A*02:01 epitopes are made available through an online database (see data availability). The website that hosts our database was constructed using the Django web framework.</p></div></div><div class="section" id="sec-5"><h2 class="">Results and Discussion</h2><div id="sec-6" class="subsection"><h3>Template identification for structure modeling using RosettaMHC</h3><p id="p-12">Our full workflow for template identification and structure modeling is outlined in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1a</a>, with a flowchart shown in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1b</a>. To identify all possible regular peptide binders to HLA-A*02:01 that are expressed by SARS-CoV-2, we used a recently annotated version of all open reading frames (ORFs) in the viral genome from NCBI (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>), made available through the UCSC genome browser (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>). We used 9- and 10-residue sliding windows to scan all protein sequences, since these are the optimum peptide lengths for binding to the HLA-A*02:01 groove (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>). While spliced peptide epitopes (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>) are not considered in the current study, this functionality can be added to our method at a later stage. Using NetMHCpan-4.0 (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a>), we identified all 439 9mer and 279 10mer epitopes that are predicted to yield positive (classified as both weak and strong) binders. To further validate this set and derive plausible 3D models of the peptide/HLA-A*02:01 complexes, we used a structure-guided approach, RosettaMHC, which aims to derive a physically realistic fitness score for each peptide in the HLA-A*02:01 binding groove using an annotated database of high-resolution structures and Rosetta’s all-atom energy function (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>). RosettaMHC leverages a database of 241 HLA-A*02:01 X-ray structures encompassing a range of bound peptides, to find the closest match to each target epitope predicted from the SARS-CoV-2 proteome. To identify the best template for structure modeling, we use sequence matching criteria which first consider the peptide anchors (positions P2 and P9/P10 for 9mer/10mer epitopes), followed by a sequence similarity metric calculated from the full alignment between the template and target peptide sequences. The template assignment statistics for the four different classes of SARS-CoV-2 epitopes in our set are shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2a</a>. We find that we can cover the entire set of 718 predicted binders using a subset of 114 HLA-A*02:01 templates in our annotated database of PDB-derived structures (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2b</a>). Each target peptide sequence is then threaded onto the backbone of its best identified template, followed by all-atom refinement of the side chain and backbone degrees of freedom using Rosetta’s Ref2015 energy function (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">36</a>), and binding energy calculation.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.23.004176/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Structure-guided modeling of T cell epitopes in the SARS-CoV-2 proteome (a) General workflow of our pipeline for structure-guided epitope ranking. (b) Protein sequences from the annotated SARS-CoV-2 proteome are used to generate peptide epitopes with a sliding window covering all frames of a fixed length (9,621 9mer and 9,611 10mer possible peptides). Candidate peptides are first filtered by NetMHCpan-4.0 (27) to identify all predicted strong and weak binders (439 9mer and 279 10mer epitopes). For rapid template matching and structure modeling, we use a local database of 241 HLA-A*02:01 X-ray structures with resolution below 3.5 &#xC5; from the Protein Data Bank (22). Each candidate peptide is scanned against all peptide sequences of the same length in the database, and the top-scoring template is used to guide the RosettaMHC comparative modeling protocol and to compute a binding energy." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-720689072" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Structure-guided modeling of T cell epitopes in the SARS-CoV-2 proteome&lt;/span&gt; &lt;strong&gt;(a)&lt;/strong&gt; General workflow of our pipeline for structure-guided epitope ranking. &lt;strong&gt;(b)&lt;/strong&gt; Protein sequences from the annotated SARS-CoV-2 proteome are used to generate peptide epitopes with a sliding window covering all frames of a fixed length (9,621 9mer and 9,611 10mer possible peptides). Candidate peptides are first filtered by NetMHCpan-4.0 (27) to identify all predicted strong and weak binders (439 9mer and 279 10mer epitopes). For rapid template matching and structure modeling, we use a local database of 241 HLA-A*02:01 X-ray structures with resolution below 3.5 &#xC5; from the Protein Data Bank (22). Each candidate peptide is scanned against all peptide sequences of the same length in the database, and the top-scoring template is used to guide the RosettaMHC comparative modeling protocol and to compute a binding energy.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="FIGURE 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.23.004176/F1.medium.gif" width="440" height="398"/><noscript><img class="highwire-fragment fragment-image" alt="FIGURE 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.23.004176/F1.medium.gif" width="440" height="398"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.23.004176/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIGURE 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.23.004176/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">FIGURE 1.</span> <span class="caption-title">Structure-guided modeling of T cell epitopes in the SARS-CoV-2 proteome</span><p id="p-13" class="first-child"><strong>(a)</strong> General workflow of our pipeline for structure-guided epitope ranking. <strong>(b)</strong> Protein sequences from the annotated SARS-CoV-2 proteome are used to generate peptide epitopes with a sliding window covering all frames of a fixed length (9,621 9mer and 9,611 10mer possible peptides). Candidate peptides are first filtered by NetMHCpan-4.0 (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">27</a>) to identify all predicted strong and weak binders (439 9mer and 279 10mer epitopes). For rapid template matching and structure modeling, we use a local database of 241 HLA-A*02:01 X-ray structures with resolution below 3.5 Å from the Protein Data Bank (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a>). Each candidate peptide is scanned against all peptide sequences of the same length in the database, and the top-scoring template is used to guide the RosettaMHC comparative modeling protocol and to compute a binding energy.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.23.004176/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Coverage of predicted HLA-A02 epitopes by structural templates in the PDB (a) Peptide anchor matching statistics of all predicted SARS-CoV-2 strong (SB) and weak binders (WB) of lengths 9 and 10 to a database of 241 high-resolution HLA-A*02:01 X-ray structures (b) Plot showing cumulative distribution (CDF) of strong and weak binder peptides of lengths 9 and 10, as a function of the total number of matching templates from the Protein Data Bank (22)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-720689072" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Coverage of predicted HLA-A02 epitopes by structural templates in the PDB&lt;/span&gt; &lt;strong&gt;(a)&lt;/strong&gt; Peptide anchor matching statistics of all predicted SARS-CoV-2 strong (SB) and weak binders (WB) of lengths 9 and 10 to a database of 241 high-resolution HLA-A*02:01 X-ray structures &lt;strong&gt;(b)&lt;/strong&gt; Plot showing cumulative distribution (CDF) of strong and weak binder peptides of lengths 9 and 10, as a function of the total number of matching templates from the Protein Data Bank (22).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="FIGURE 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.23.004176/F2.medium.gif" width="440" height="164"/><noscript><img class="highwire-fragment fragment-image" alt="FIGURE 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.23.004176/F2.medium.gif" width="440" height="164"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.23.004176/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIGURE 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.23.004176/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">FIGURE 2.</span> <span class="caption-title">Coverage of predicted HLA-A02 epitopes by structural templates in the PDB</span><p id="p-14" class="first-child"><strong>(a)</strong> Peptide anchor matching statistics of all predicted SARS-CoV-2 strong (SB) and weak binders (WB) of lengths 9 and 10 to a database of 241 high-resolution HLA-A*02:01 X-ray structures <strong>(b)</strong> Plot showing cumulative distribution (CDF) of strong and weak binder peptides of lengths 9 and 10, as a function of the total number of matching templates from the Protein Data Bank (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">22</a>).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>RosettaMHC models recapitulate features of high-resolution X-ray structures</h3><p id="p-15">The sequence logos derived from 9mer and 10mer peptides with good structural complementarity to the HLA-A*02:01 groove according to Rosetta’s binding energy (see below) adhere to the canonical motif, with a preference for hydrophobic, methyl-bearing side chains at the peptide anchor residues P2 and P9 (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3a</a>). The anchor residue preferences are recapitulated in representative 9mer and 10mer models of the two top binders in our set as ranked by Rosetta’s energy (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3c</a> and <a id="xref-fig-3-3" class="xref-fig" href="#F3">3d</a>), corresponding to epitopes TMADLVYAL and FLFVAAIFYL derived from the RNA polymerase and nsp3 proteins, respectively, which are both encoded by <em>orf1ab</em> in the viral genome (NCBI Reference YP_009724389.1). In accordance with features seen in high-resolution structures of HLA-A*02:01-restricted epitopes, the peptides adopt an extended, bulged backbone conformation. The free N-terminus of both peptides is stabilized by a network of polar contacts with Tyr 7, Tyr 159, Tyr 171 and Glu 63 in the A- and B- pockets of the HLA-A*02:01 groove. The Met (9mer) or Leu (10mer) side chain of P2 is buried in a B-pocket hydrophobic cleft formed by Met 45 and Val 67. Equivalently, the C-terminus is coordinated through polar contacts with Asp 77 and Lys 145 from opposite sides of the groove, with the Leu P9/P10 anchor nestled in the F-pocket defined by the side chains of Leu 81, Tyr 116, Tyr 123 and Trp 147. Residues P3-P8 form a series of backbone and side chain contacts with pockets C, D and E, while most backbone amide and carbonyl groups form hydrogen bonds with the side chains of residues lining the MHC-I groove. These high-resolution structural features are consistent across low-energy models of unrelated target peptides in our input set, suggesting that, when provided with a large set of input templates, a combined threading and side chain optimization protocol can derive physically realistic models.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.23.004176/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summary of RosettaMHC modeling results for SARS-CoV-2 peptide epitopes Sequence logos from the n top ranking epitopes in the SARS-CoV-2 genome, predicted by NetMHCpan-4.0 (27) and further refined using RosettaMHC binding simulations are shown for: (a) 9mers (n=154) and (b) 10mers (n=72). The top 9mer and 10mer epitopes in our refined set are shown: (c) TMADLVYAL, from RNA polymerase and (d) FLFVAAIFYL, from nsp3. Dotted lines indicate polar contacts between peptide and heavy chain residues, with peptide residues labelled. The template PDB IDs and original peptides used for modeling the target peptides are indicated below each model. (e) Density plots showing distribution of average Rosetta binding energies (kcal/mol) for all epitopes of length 9. Distributions reflect 93 PDB templates (green), 164 strong binder epitopes (according to NetMHCpan-4.0 (27)) (blue), and 93 poly alanine peptides modeled using the same PDB templates and used as a reference set for sub-optimal binders (polyA; pink). The binding energies of models generated for 28 confirmed SARS T cell epitopes from the IEDB and ViPR (37, 38) are indicated by circles at the bottom of the plot. Red circles (19/28) indicate epitopes that lie within the distribution of refined PDB templates and black circles (9/28) indicate epitopes that fall within the distribution of polyA (sub-optimal binders). (f) Box plots showing distribution of average binding energies for 93 PDB templates, 93 poly alanine peptides, 28 confirmed epitopes (37, 38) and RosettaMHC models for 164 strong (SB) and 275 weak (WB) binder 9mer epitopes predicted from the SARS-CoV-2 proteome using NetMHCpan-4.0 (27)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-720689072" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Summary of RosettaMHC modeling results for SARS-CoV-2 peptide epitopes&lt;/span&gt; Sequence logos from the &lt;em&gt;n&lt;/em&gt; top ranking epitopes in the SARS-CoV-2 genome, predicted by NetMHCpan-4.0 (27) and further refined using RosettaMHC binding simulations are shown for: &lt;strong&gt;(a)&lt;/strong&gt; 9mers (&lt;em&gt;n&lt;/em&gt;=154) and &lt;strong&gt;(b)&lt;/strong&gt; 10mers (&lt;em&gt;n&lt;/em&gt;=72). The top 9mer and 10mer epitopes in our refined set are shown: &lt;strong&gt;(c)&lt;/strong&gt; TMADLVYAL, from RNA polymerase and &lt;strong&gt;(d)&lt;/strong&gt; FLFVAAIFYL, from nsp3. Dotted lines indicate polar contacts between peptide and heavy chain residues, with peptide residues labelled. The template PDB IDs and original peptides used for modeling the target peptides are indicated below each model. &lt;strong&gt;(e)&lt;/strong&gt; Density plots showing distribution of average Rosetta binding energies (kcal/mol) for all epitopes of length 9. Distributions reflect 93 PDB templates (green), 164 strong binder epitopes (according to NetMHCpan-4.0 (27)) (blue), and 93 poly alanine peptides modeled using the same PDB templates and used as a reference set for sub-optimal binders (polyA; pink). The binding energies of models generated for 28 confirmed SARS T cell epitopes from the IEDB and ViPR (37, 38) are indicated by circles at the bottom of the plot. Red circles (19/28) indicate epitopes that lie within the distribution of refined PDB templates and black circles (9/28) indicate epitopes that fall within the distribution of polyA (sub-optimal binders). &lt;strong&gt;(f)&lt;/strong&gt; Box plots showing distribution of average binding energies for 93 PDB templates, 93 poly alanine peptides, 28 confirmed epitopes (37, 38) and RosettaMHC models for 164 strong (SB) and 275 weak (WB) binder 9mer epitopes predicted from the SARS-CoV-2 proteome using NetMHCpan-4.0 (27).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="FIGURE 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.23.004176/F3.medium.gif" width="374" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="FIGURE 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.23.004176/F3.medium.gif" width="374" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.23.004176/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIGURE 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.23.004176/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">FIGURE 3.</span> <span class="caption-title">Summary of RosettaMHC modeling results for SARS-CoV-2 peptide epitopes</span><p id="p-16" class="first-child">Sequence logos from the <em>n</em> top ranking epitopes in the SARS-CoV-2 genome, predicted by NetMHCpan-4.0 (<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">27</a>) and further refined using RosettaMHC binding simulations are shown for: <strong>(a)</strong> 9mers (<em>n</em>=154) and <strong>(b)</strong> 10mers (<em>n</em>=72). The top 9mer and 10mer epitopes in our refined set are shown: <strong>(c)</strong> TMADLVYAL, from RNA polymerase and <strong>(d)</strong> FLFVAAIFYL, from nsp3. Dotted lines indicate polar contacts between peptide and heavy chain residues, with peptide residues labelled. The template PDB IDs and original peptides used for modeling the target peptides are indicated below each model. <strong>(e)</strong> Density plots showing distribution of average Rosetta binding energies (kcal/mol) for all epitopes of length 9. Distributions reflect 93 PDB templates (green), 164 strong binder epitopes (according to NetMHCpan-4.0 (<a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">27</a>)) (blue), and 93 poly alanine peptides modeled using the same PDB templates and used as a reference set for sub-optimal binders (polyA; pink). The binding energies of models generated for 28 confirmed SARS T cell epitopes from the IEDB and ViPR (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>) are indicated by circles at the bottom of the plot. Red circles (19/28) indicate epitopes that lie within the distribution of refined PDB templates and black circles (9/28) indicate epitopes that fall within the distribution of polyA (sub-optimal binders). <strong>(f)</strong> Box plots showing distribution of average binding energies for 93 PDB templates, 93 poly alanine peptides, 28 confirmed epitopes (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">37</a>, <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">38</a>) and RosettaMHC models for 164 strong (SB) and 275 weak (WB) binder 9mer epitopes predicted from the SARS-CoV-2 proteome using NetMHCpan-4.0 (<a id="xref-ref-27-6" class="xref-bibr" href="#ref-27">27</a>).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>Selection of high-affinity peptide epitopes using a structure-based score</h3><p id="p-17">To evaluate the accuracy of our models and fitness of each peptide within the HLA-A*02:01 binding groove, we computed Rosetta all-atom binding energies across all complexes modeled for different peptide sets. High binding energies can be used as an additional metric to filter low-affinity peptides in the NetMHCpan-4.0 predictions, with the caveat that high energies can be also due to incomplete optimization of the Rosetta energy function as a result of significant deviations between the target and template backbone conformations, not captured by our protocol. We performed 10 independent calculations for each peptide, and the 3 lower-energy models were selected as the final ensemble and used to compute an average binding energy. The results for all 9mer peptides are summarized in <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figures 3e, f</a>, while additional results for 10mers are provided through our web-interface and outlined in Supplemental Table 1. As a positive reference, we used the binding energies of the idealized and relaxed PDB templates, which are at a local minimum of the Rosetta scoring function. As a reference set for sub-optimal binders, we modeled decoy structures of poly alanine (polyA) peptide sequences (predicted by NetMHCpan-4.0 to be a top 9th percentile binder for HLA-A*02:01), threaded onto the same PDB templates.</p><p id="p-18">We observe a significant, negative (−26 kcal/mol) energy gap between the average binding energies for PDB templates and poly alanine models. The binding energies for all modeled 9mers from the SARS-CoV-2 genome fall between the average energies of the optimal PDB templates and sub-optimal polyA binders, and show a bimodal distribution with significant overlap with the refined PDB template energies (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1e</a>). Comparison of the distributions between epitopes that are classified as strong versus weak binders by NetMHCpan-4.0 shows a moderate bias towards lower binding energies for the strong binders and a larger spread in energies for weak binders, likely due to suboptimal residues at the P2 and P9 anchor positions (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3f</a>). As an intendent positive set, we also modeled 28 9mer peptides that are homologous to peptides in the SARS viral genome and have been previously reported to bind HLA-A*02:01 in the IEDB and ViPR (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">37</a>, <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">38</a>) databases (Supplemental Table 2). Inspection of Rosetta binding energies derived from models in this set shows a similar distribution to the epitopes classified by NetMHCpan-4.0 as strong binders, with the energies of 19/28 peptides falling well within the distribution of the refined PDB templates (red dots in <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3e</a>).</p><p id="p-19">Based on these observations, we further classified all epitopes in the original set provided by NetMHCpan-4.0 as strong or weak binders according to the Rosetta binding energy. Peptides with binding energies that fall well within the PDB template distribution (green curve and red dots in <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3e</a>) are classified as strong binders. We obtained 154 9mer and 72 10mer strong binders which show optimal complementarity within the HLA-A*02:01 peptide-binding groove according to our modeling simulations. These results suggest that the high-resolution features seen in our models (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3c, d</a>) yield optimal binding energies for a significant fraction of the epitopes predicted by NetMHCpan-4.0 (45/33% of strong binders and 30/25% of weak binders for 9mers/10mers, respectively), which are comparable to locally refined PDB structures. The average binding energies for all peptides are provided in our web-interface and in Supplemental Table 1.</p></div><div id="sec-9" class="subsection"><h3>Surface features of peptide/HLA-A*02:01 models for T cell recognition</h3><p id="p-20">Visualization of our models through an interactive online interface provides direct information on SARS-CoV-2 peptide residues that are bulging out of the MHC-I groove, and are therefore accessible to interactions with complementarity-determining regions (CDRs) of T cell receptors (TCRs). Given that αβ TCRs generally employ a diagonal binding mode to engage pMHC-I antigens where the CDR3α and CDR3β TCR loops form direct contacts with key peptide residues (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>), knowledge of the surface features for different epitopes adds an extra layer of information to interpret sequence variability between different viral strains. For other important antigens with known structures in the PDB, such features can be derived from an annotated database connecting pMHC-I/TCR co-crystal structures with biophysical binding data (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>), and were recently employed in an artificial neural network approach to predict the immunogenicity of different HLA-A*02:01 bound peptides in the context of tumor neoantigen display (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>). A separate study has shown that the electrostatic compatibility between self vs foreign HLA surfaces can be used to determine antibody alloimmune responses (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>). Given that antibodies and TCRs use a common fold and similar principles to engage pMHC-I molecules (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">40</a>), it is likely that surface electrostatic features play an important role in recognition of peptide/HLA surfaces by their cognate TCRs in the context of SARS-CoV-2 infection.</p><p id="p-21">Electrostatic surface potentials calculated using a numerical solution to the Poisson-Boltzmann Equation (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>) for our modeled peptide/HLA-A*02:01 complexes allow us to compare important features for TCR recognition between different high-affinity epitopes (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). We observe a moderate electropositive character of the HLA-A*02:01 α<sub>1</sub> helix, and a moderate negative potential on the α2 helix, which is consistent between complexes with different bound peptides. However, due to substantial sequence variability in surface-exposed residues at the P2-P8 positions, we observe a range of electrostatic features ranging from negative (epitope TMADLVYAL), to neutral (NLIDSYFVV) or positively charged (KLWAQCVQL). Further classification and ranking of the top binders in our set on the basis of their molecular surface features would enable the selection of the most diverse panel of peptides for high-throughput pMHC tetramer library generation (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>–<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>). Tetramer screening of T cells from COVID-19 patients, recovered individuals and healthy donors can be used to identify critical gaps in the T cell repertoire of high-risk groups, and to design epitope DNA strings for vaccine development.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.23.004176/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Variability in TCR recognition features of HLA-A02 with different high-affinity peptides. Molecular surfaces of SARS-CoV-2/HLA-A*02:01 RosettaMHC models are shown for four top-scoring epitopes (ranked by Rosetta binding energy from left to right) captured in the A, B, C, D, E and F pockets of the MHC-I groove (top panel). The origins of the peptide epitopes in the ~30 kbp SARS-CoV-2 genome are noted. Electrostatic surfaces computed for the same models are shown in the bottom panel. Solvent-accessible surface representation with electrostatic potential in the indicated ranges (down to &#x2212;60 kcal/(mol&#xB7;e) in red and up to +61 kcal/(mol&#xB7;e) in blue) were calculated using the APBS solver (45) in Pymol (46). All calculations were performed at 150 mM ionic strength, 298.15 Kelvin, pH 7.2, protein dielectric 2.0, and solvent dielectric 78.54. Electrostatic potentials are given in units of kT/e. A 1.4 &#xC5; solvent (probe) radius and 10.0 points/&#xC5;2 density was used to calculate molecular surfaces." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-720689072" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Variability in TCR recognition features of HLA-A02 with different high-affinity peptides.&lt;/span&gt; Molecular surfaces of SARS-CoV-2/HLA-A*02:01 RosettaMHC models are shown for four top-scoring epitopes (ranked by Rosetta binding energy from left to right) captured in the A, B, C, D, E and F pockets of the MHC-I groove (top panel). The origins of the peptide epitopes in the ~30 kbp SARS-CoV-2 genome are noted. Electrostatic surfaces computed for the same models are shown in the bottom panel. Solvent-accessible surface representation with electrostatic potential in the indicated ranges (down to &#x2212;60 kcal/(mol&#xB7;&lt;em&gt;e&lt;/em&gt;) in red and up to +61 kcal/(mol&#xB7;&lt;em&gt;e&lt;/em&gt;) in blue) were calculated using the APBS solver (45) in Pymol (46). All calculations were performed at 150 mM ionic strength, 298.15 Kelvin, pH 7.2, protein dielectric 2.0, and solvent dielectric 78.54. Electrostatic potentials are given in units of kT/e. A 1.4 &#xC5; solvent (probe) radius and 10.0 points/&#xC5;&lt;sup&gt;2&lt;/sup&gt; density was used to calculate molecular surfaces.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="FIGURE 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.23.004176/F4.medium.gif" width="440" height="322"/><noscript><img class="highwire-fragment fragment-image" alt="FIGURE 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.23.004176/F4.medium.gif" width="440" height="322"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.23.004176/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIGURE 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/28/2020.03.23.004176/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">FIGURE 4.</span> <span class="caption-title">Variability in TCR recognition features of HLA-A02 with different high-affinity peptides.</span><p id="p-22" class="first-child">Molecular surfaces of SARS-CoV-2/HLA-A*02:01 RosettaMHC models are shown for four top-scoring epitopes (ranked by Rosetta binding energy from left to right) captured in the A, B, C, D, E and F pockets of the MHC-I groove (top panel). The origins of the peptide epitopes in the ~30 kbp SARS-CoV-2 genome are noted. Electrostatic surfaces computed for the same models are shown in the bottom panel. Solvent-accessible surface representation with electrostatic potential in the indicated ranges (down to −60 kcal/(mol·<em>e</em>) in red and up to +61 kcal/(mol·<em>e</em>) in blue) were calculated using the APBS solver (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>) in Pymol (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>). All calculations were performed at 150 mM ionic strength, 298.15 Kelvin, pH 7.2, protein dielectric 2.0, and solvent dielectric 78.54. Electrostatic potentials are given in units of kT/e. A 1.4 Å solvent (probe) radius and 10.0 points/Å<sup>2</sup> density was used to calculate molecular surfaces.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-11"><h2 class="">Code and Data availability</h2><p id="p-24">An online web-interface for visualization and download of all models is available at: <a href="https://rosettamhc.chemistry.ucsc.edu">https://rosettamhc.chemistry.ucsc.edu</a>. The RosettaMHC source code is available at <a href="https://github.com/snerligit/mhc-pep-threader">https://github.com/snerligit/mhc-pep-threader</a>. Rosetta binding energies for all 718 HLA-A*02:01-restricted peptides in our set are provided in Supplemental Table 1.</p></div><div class="section" id="sec-12"><h2 class="">Disclosures</h2><p id="p-25">The authors have no financial conflicts of interest.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-23">The authors are grateful to Alison Lindberg and ITS/ADC staff at UCSC for assistance in setting up the web server for the database, and Hiram Clawson (UCSC Genome Browser) for providing the SARS-CoV-2 protein sequence data. We thank Andrew McShan, Hailey Wallace (UCSC) for assistance in the development of the RosettaMHC protocol, and David Haussler (UCSC), Michael Betts (University of Pennsylvania) and David Margulies (NIH) for helpful discussions. This work was supported through Grants from NIAID (5R01AI143997) and NIGMS (5R35GM125034).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-update fn-group-summary-of-updates" id="fn-1"><p id="p-3">Main text revised for style and minor edits; Supplementary data revised; Figure 3 revised</p></li><li class="fn-dataset fn-group-external-links" id="fn-2"><p id="p-4">
<a href="https://rosettamhc.chemistry.ucsc.edu">https://rosettamhc.chemistry.ucsc.edu</a>
</p></li><li class="fn-dataset fn-group-external-links" id="fn-3"><p id="p-5">
<a href="https://github.com/snerligit/mhc-pep-threader">https://github.com/snerligit/mhc-pep-threader</a>
</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Zhou</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. V.</span> <span class="cit-name-surname">Garner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. P.</span> <span class="cit-name-surname">Watkins</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. J.</span> <span class="cit-name-surname">Carter</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Smoot</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Gregg</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Daniels</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Jervey</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Albaiu</span></span>. <span class="cit-pub-date">2020</span>. <span class="cit-article-title">Research and Development on Therapeutic Agents and Vaccines for COVID-19 and Related Human Coronavirus Diseases</span>. <abbr class="cit-jnl-abbrev">ACS Cent. Sci</abbr>..</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.2" data-doi="10.1016/j.antiviral.2010.11.007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kumaki</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ennis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Rahbar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Turner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. K.</span> <span class="cit-name-surname">Wandersee</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Smith</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. W.</span> <span class="cit-name-surname">Bailey</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z. G.</span> <span class="cit-name-surname">Vest</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Madsen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. K.-K.</span> <span class="cit-name-surname">Li</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D. L.</span> <span class="cit-name-surname">Barnard</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Single-dose intranasal administration with mDEF201 (adenovirus vectored mouse interferon-alpha) confers protection from mortality in a lethal SARS-CoV BALB/c mouse model</span>. <abbr class="cit-jnl-abbrev">Antiviral Res</abbr>. <span class="cit-vol">89</span>: <span class="cit-fpage">75</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntiviral%2Bresearch%26rft.stitle%253DAntiviral%2BRes%26rft.aulast%253DKumaki%26rft.auinit1%253DY.%26rft.volume%253D89%26rft.issue%253D1%26rft.spage%253D75%26rft.epage%253D82%26rft.atitle%253DSingle-dose%2Bintranasal%2Badministration%2Bwith%2BmDEF201%2B%2528adenovirus%2Bvectored%2Bmouse%2Binterferon-alpha%2529%2Bconfers%2Bprotection%2Bfrom%2Bmortality%2Bin%2Ba%2Blethal%2BSARS-CoV%2BBALB%252Fc%2Bmouse%2Bmodel.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.antiviral.2010.11.007%26rft_id%253Dinfo%253Apmid%252F21093489%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.antiviral.2010.11.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21093489&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.23.004176.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kishimoto</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Interleukin-6: discovery of a pleiotropic cytokine</span>. <abbr class="cit-jnl-abbrev">Arthritis Res. Ther</abbr>. <span class="cit-vol">8</span>: <span class="cit-fpage">S2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArthritis%2BRes.%2BTher%26rft.volume%253D8%26rft.spage%253DS2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.4" data-doi="10.1016/j.it.2016.08.010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rock</span>,  <span class="cit-name-given-names">K. L.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Reits</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Neefjes</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Present Yourself! By MHC Class I and MHC Class II Molecules</span>. <abbr class="cit-jnl-abbrev">Trends Immunol</abbr>. <span class="cit-vol">37</span>: <span class="cit-fpage">724</span>–<span class="cit-lpage">737</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BImmunol%26rft.volume%253D37%26rft.spage%253D724%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.it.2016.08.010%26rft_id%253Dinfo%253Apmid%252F27614798%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.it.2016.08.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27614798&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.23.004176.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kaufman</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Unfinished Business: Evolution of the MHC and the Adaptive Immune System of Jawed Vertebrates</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Immunol</abbr>. <span class="cit-vol">36</span>: <span class="cit-fpage">383</span>–<span class="cit-lpage">409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BImmunol%26rft.volume%253D36%26rft.spage%253D383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thevarajan</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. H. O.</span> <span class="cit-name-surname">Nguyen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Koutsakos</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Druce</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Caly</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. E.</span> <span class="cit-name-surname">van de Sandt</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Jia</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Nicholson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Catton</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Cowie</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. Y. C.</span> <span class="cit-name-surname">Tong</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Lewin</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kedzierska</span></span>. <span class="cit-pub-date">2020</span>. <span class="cit-article-title">Breadth of concomitant immune responses prior to patient recovery: a case report of non-severe COVID-19</span>. <abbr class="cit-jnl-abbrev">Nat. Med</abbr>. <span class="cit-fpage">1</span>–<span class="cit-lpage">3</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goronzy</span>,  <span class="cit-name-given-names">J. J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Fang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. M.</span> <span class="cit-name-surname">Cavanagh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Qi</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Weyand</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Naïve T cell maintenance and function in human aging</span>. <abbr class="cit-jnl-abbrev">J. Immunol. Baltim. Md 1950</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">4073</span>–<span class="cit-lpage">4080</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BImmunol.%2BBaltim.%2BMd%2B1950%26rft.volume%253D194%26rft.spage%253D4073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Oyarzun</span>,  <span class="cit-name-given-names">P.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Kobe</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Computer-aided design of T-cell epitope-based vaccines: addressing population coverage</span>. <abbr class="cit-jnl-abbrev">Int. J. Immunogenet</abbr>. <span class="cit-vol">42</span>: <span class="cit-fpage">313</span>–<span class="cit-lpage">321</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BImmunogenet%26rft.volume%253D42%26rft.spage%253D313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.9" data-doi="10.1016/j.cell.2014.03.047"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Birnbaum</span>,  <span class="cit-name-given-names">M. E.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Mendoza</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. K.</span> <span class="cit-name-surname">Sethi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Dong</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Glanville</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Dobbins</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Ozkan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. M.</span> <span class="cit-name-surname">Davis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. W.</span> <span class="cit-name-surname">Wucherpfennig</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">K. C.</span> <span class="cit-name-surname">Garcia</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Deconstructing the peptide-MHC specificity of T cell recognition</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1073</span>–<span class="cit-lpage">1087</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D157%26rft.spage%253D1073%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2014.03.047%26rft_id%253Dinfo%253Apmid%252F24855945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2014.03.047&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24855945&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.23.004176.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000336437200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.10" data-doi="10.1074/jbc.M111.289488"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wooldridge</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ekeruche-Makinde</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H. A.</span> <span class="cit-name-surname">van den Berg</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Skowera</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Miles</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. P.</span> <span class="cit-name-surname">Tan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Dolton</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Clement</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Llewellyn-Lacey</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Price</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Peakman</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A. K.</span> <span class="cit-name-surname">Sewell</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">A single autoimmune T cell receptor recognizes more than a million different peptides</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem</abbr>. <span class="cit-vol">287</span>: <span class="cit-fpage">1168</span>–<span class="cit-lpage">1177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M111.289488%26rft_id%253Dinfo%253Apmid%252F22102287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyODcvMi8xMTY4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjgvMjAyMC4wMy4yMy4wMDQxNzYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peters</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Nielsen</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sette</span></span>. <span class="cit-pub-date">2020</span>. <span class="cit-article-title">T Cell Epitope Predictions</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Immunol</abbr>..</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.12" data-doi="10.1093/nar/gky1006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vita</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Mahajan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Overton</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. K.</span> <span class="cit-name-surname">Dhanda</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Martini</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Cantrell</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. K.</span> <span class="cit-name-surname">Wheeler</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sette</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Peters</span></span>. <span class="cit-pub-date">2019</span>. <span class="cit-article-title">The Immune Epitope Database (IEDB): 2018 update</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">47</span>: <span class="cit-fpage">D339</span>–<span class="cit-lpage">D343</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D47%26rft.spage%253DD339%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky1006%26rft_id%253Dinfo%253Apmid%252F30357391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gky1006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30357391&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.23.004176.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Antunes</span>,  <span class="cit-name-given-names">D. A.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Abella</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Devaurs</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. M.</span> <span class="cit-name-surname">Rigo</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">L. E.</span> <span class="cit-name-surname">Kavraki</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Structure-based Methods for Binding Mode and Binding Affinity Prediction for Peptide-MHC Complexes</span>. <abbr class="cit-jnl-abbrev">Curr. Top. Med. Chem</abbr>. <span class="cit-vol">18</span>: <span class="cit-fpage">2239</span>–<span class="cit-lpage">2255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BTop.%2BMed.%2BChem%26rft.volume%253D18%26rft.spage%253D2239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.14" data-doi="10.1073/pnas.1018165108"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yanover</span>,  <span class="cit-name-given-names">C.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Bradley</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Large-scale characterization of peptide-MHC binding landscapes with structural simulations</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci</abbr>. <span class="cit-vol">108</span>: <span class="cit-fpage">6981</span>–<span class="cit-lpage">6986</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1018165108%26rft_id%253Dinfo%253Apmid%252F21478437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA4LzE3LzY5ODEiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yOC8yMDIwLjAzLjIzLjAwNDE3Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">15.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.23.004176v2.15" data-doi="10.1073/pnas.1321126111"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">King</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. N.</span> <span class="cit-name-surname">Garza</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Mazor</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Linehan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Pastan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Pepper</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Baker</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Removing T-cell epitopes with computational protein design</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>. <span class="cit-vol">111</span>: <span class="cit-fpage">8577</span>–<span class="cit-lpage">8582</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1321126111%26rft_id%253Dinfo%253Apmid%252F24843166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTExLzIzLzg1NzciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yOC8yMDIwLjAzLjIzLjAwNDE3Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.16" data-doi="10.1021/ci500393h"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Pan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Chao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Tan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Qu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Yang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Wang</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Mei</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Subangstrom accuracy in pHLA-I modeling by Rosetta FlexPepDock refinement protocol</span>. <abbr class="cit-jnl-abbrev">J. Chem. Inf. Model</abbr>. <span class="cit-vol">54</span>: <span class="cit-fpage">2233</span>–<span class="cit-lpage">2242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BChem.%2BInf.%2BModel%26rft.volume%253D54%26rft.spage%253D2233%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fci500393h%26rft_id%253Dinfo%253Apmid%252F25050981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ci500393h&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25050981&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.23.004176.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kyeong</span>,  <span class="cit-name-given-names">H.-H.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Choi</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">H.-S.</span> <span class="cit-name-surname">Kim</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">GradDock: rapid simulation and tailored ranking functions for peptide-MHC Class I docking</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">469</span>–<span class="cit-lpage">476</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rubenstein</span>,  <span class="cit-name-given-names">A. B.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Pethe</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S. D.</span> <span class="cit-name-surname">Khare</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">MFPred: Rapid and accurate prediction of protein-peptide recognition multispecificity using self-consistent mean field theory</span>. <abbr class="cit-jnl-abbrev">PLoS Comput. Biol</abbr>. <span class="cit-vol">13</span>: <span class="cit-fpage">e1005614</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BComput.%2BBiol%26rft.volume%253D13%26rft.spage%253De1005614%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.19" data-doi="10.1016/j.str.2013.08.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">DiMaio</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. Y.-R.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Kim</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Miles</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Brunette</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Thompson</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Baker</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">High-Resolution Comparative Modeling with RosettaCM</span>. <abbr class="cit-jnl-abbrev">Structure</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1735</span>–<span class="cit-lpage">1742</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStructure%26rft.volume%253D21%26rft.spage%253D1735%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.str.2013.08.005%26rft_id%253Dinfo%253Apmid%252F24035711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.str.2013.08.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24035711&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.23.004176.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.20"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Frontiers</span> | <span class="cit-article-title">A Recurrent Mutation in Anaplastic Lymphoma Kinase with Distinct Neoepitope Conformations</span> | <abbr class="cit-jnl-abbrev">Immunology</abbr>..</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Robinson</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. A.</span> <span class="cit-name-surname">Guethlein</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Cereb</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. Y.</span> <span class="cit-name-surname">Yang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Norman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. G. E.</span> <span class="cit-name-surname">Marsh</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Parham</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Distinguishing functional polymorphism from random variation in the sequences of &gt;10,000 HLA-A, -B and -C alleles</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr>. <span class="cit-vol">13</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.22" data-doi="10.1093/nar/28.1.235"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Berman</span>,  <span class="cit-name-given-names">H. M.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Westbrook</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Feng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Gilliland</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. N.</span> <span class="cit-name-surname">Bhat</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Weissig</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I. N.</span> <span class="cit-name-surname">Shindyalov</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">P. E.</span> <span class="cit-name-surname">Bourne</span></span>. <span class="cit-pub-date">2000</span>. <span class="cit-article-title">The Protein Data Bank</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">28</span>: <span class="cit-fpage">235</span>–<span class="cit-lpage">242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DBerman%26rft.auinit1%253DH.%2BM.%26rft.volume%253D28%26rft.issue%253D1%26rft.spage%253D235%26rft.epage%253D242%26rft.atitle%253DThe%2BProtein%2BData%2BBank%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F28.1.235%26rft_id%253Dinfo%253Apmid%252F10592235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/28.1.235&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10592235&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.23.004176.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084896300069&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.23" data-doi="10.1038/nbt.3662"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bentzen</span>,  <span class="cit-name-given-names">A. K.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Marquard</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Lyngaa</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. K.</span> <span class="cit-name-surname">Saini</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Ramskov</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Donia</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Such</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. J. S.</span> <span class="cit-name-surname">Furness</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">McGranahan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Rosenthal</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. T.</span> <span class="cit-name-surname">Straten</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Szallasi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I. M.</span> <span class="cit-name-surname">Svane</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Swanton</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Quezada</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. N.</span> <span class="cit-name-surname">Jakobsen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Eklund</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Hadrup</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Large-scale detection of antigen-specific T cells using peptide-MHC-I multimers labeled with DNA barcodes</span>. <abbr class="cit-jnl-abbrev">Nat. Biotechnol</abbr>. <span class="cit-vol">34</span>: <span class="cit-fpage">1037</span>–<span class="cit-lpage">1045</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol%26rft.volume%253D34%26rft.spage%253D1037%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3662%26rft_id%253Dinfo%253Apmid%252F27571370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3662&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27571370&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.23.004176.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">24.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.23.004176v2.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Saini</span>,  <span class="cit-name-given-names">S. K.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Tamhane</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Anjanappa</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Saikia</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Ramskov</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Donia</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I. M.</span> <span class="cit-name-surname">Svane</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. N.</span> <span class="cit-name-surname">Jakobsen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Garcia-Alai</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Zacharias</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Meijers</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Springer</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Hadrup</span></span>. <span class="cit-pub-date">2019</span>. <span class="cit-article-title">Empty peptide-receptive MHC class I molecules for efficient detection of antigen-specific T cells</span>. <abbr class="cit-jnl-abbrev">Sci. Immunol</abbr>. <span class="cit-vol">4</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Overall</span>,  <span class="cit-name-given-names">S. A.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Toor</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Hao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Yarmarkovich</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">O’Rourke</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. I.</span> <span class="cit-name-surname">Morozov</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Nguyen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. S.</span> <span class="cit-name-surname">Japp</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Gonzalez</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Moschidi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Betts</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Maris</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Smibert</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">N. G.</span> <span class="cit-name-surname">Sgourakis</span></span>. <span class="cit-article-title">High Throughput pMHC-I Tetramer Library Production Using Chaperone-Mediated Peptide Exchange</span>. <abbr class="cit-jnl-abbrev">Nat. Commun. Press</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.26" data-doi="10.4049/jimmunol.0901607"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ishizuka</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Grebe</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Shenderov</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Peters</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Peng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Dong</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Pasquetto</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Oseroff</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Sidney</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Hickman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Cerundolo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sette</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Bennink</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">McMichael</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">Yewdell</span></span>. <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Quantitating T Cell Cross-Reactivity for Unrelated Peptide Antigens</span>. <abbr class="cit-jnl-abbrev">J. Immunol</abbr>. <span class="cit-vol">183</span>: <span class="cit-fpage">4337</span>–<span class="cit-lpage">4345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BImmunol%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.0901607%26rft_id%253Dinfo%253Apmid%252F19734234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTA6IjE4My83LzQzMzciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yOC8yMDIwLjAzLjIzLjAwNDE3Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jurtz</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Paul</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Andreatta</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Marcatili</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Peters</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Nielsen</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">NetMHCpan-4.0: Improved Peptide-MHC Class I Interaction Predictions Integrating Eluted Ligand and Peptide Binding Affinity Data</span>. <abbr class="cit-jnl-abbrev">J. Immunol. Baltim. Md 1950</abbr> <span class="cit-vol">199</span>: <span class="cit-fpage">3360</span>–<span class="cit-lpage">3368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BImmunol.%2BBaltim.%2BMd%2B1950%26rft.volume%253D199%26rft.spage%253D3360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.28" data-doi="10.1073/pnas.89.22.10915"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>,  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J. G.</span> <span class="cit-name-surname">Henikoff</span></span>. <span class="cit-pub-date">1992</span>. <span class="cit-article-title">Amino acid substitution matrices from protein blocks</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>. <span class="cit-vol">89</span>: <span class="cit-fpage">10915</span>–<span class="cit-lpage">10919</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DHenikoff%26rft.auinit1%253DS.%26rft.volume%253D89%26rft.issue%253D22%26rft.spage%253D10915%26rft.epage%253D10919%26rft.atitle%253DAmino%2Bacid%2Bsubstitution%2Bmatrices%2Bfrom%2Bprotein%2Bblocks.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.89.22.10915%26rft_id%253Dinfo%253Apmid%252F1438297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiODkvMjIvMTA5MTUiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yOC8yMDIwLjAzLjIzLjAwNDE3Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.29" data-doi="10.1093/bioinformatics/btq007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chaudhury</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Lyskov</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Gray</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">PyRosetta: a script-based interface for implementing molecular modeling algorithms using Rosetta</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">689</span>–<span class="cit-lpage">691</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq007%26rft_id%253Dinfo%253Apmid%252F20061306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20061306&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.23.004176.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274973800018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.30" data-doi="10.1038/msb.2011.75"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sievers</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wilm</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Dineen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. J.</span> <span class="cit-name-surname">Gibson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Karplus</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Lopez</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">McWilliam</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Remmert</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Söding</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Thompson</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D. G.</span> <span class="cit-name-surname">Higgins</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega</span>. <abbr class="cit-jnl-abbrev">Mol. Syst. Biol</abbr>. <span class="cit-vol">7</span>: <span class="cit-fpage">539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BSyst.%2BBiol%26rft.volume%253D7%26rft.spage%253D539%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2011.75%26rft_id%253Dinfo%253Apmid%252F21988835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/msb.2011.75&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21988835&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.23.004176.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.31" data-doi="10.1016/j.jmb.2010.11.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tyka</span>,  <span class="cit-name-given-names">M. D.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Keedy</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">André</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Dimaio</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Song</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. C.</span> <span class="cit-name-surname">Richardson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Richardson</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Baker</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Alternate states of proteins revealed by detailed energy landscape mapping</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol</abbr>. <span class="cit-vol">405</span>: <span class="cit-fpage">607</span>–<span class="cit-lpage">618</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DTyka%26rft.auinit1%253DM.%2BD.%26rft.volume%253D405%26rft.issue%253D2%26rft.spage%253D607%26rft.epage%253D618%26rft.atitle%253DAlternate%2Bstates%2Bof%2Bproteins%2Brevealed%2Bby%2Bdetailed%2Benergy%2Blandscape%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2010.11.008%26rft_id%253Dinfo%253Apmid%252F21073878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2010.11.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21073878&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.23.004176.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Zhao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Yu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.-M.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.-G.</span> <span class="cit-name-surname">Song</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Hu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.-W.</span> <span class="cit-name-surname">Tao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.-H.</span> <span class="cit-name-surname">Tian</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.-Y.</span> <span class="cit-name-surname">Pei</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.-L.</span> <span class="cit-name-surname">Yuan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.-L.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.-H.</span> <span class="cit-name-surname">Dai</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Q.-M.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.-J.</span> <span class="cit-name-surname">Zheng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Xu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. C.</span> <span class="cit-name-surname">Holmes</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Y.-Z.</span> <span class="cit-name-surname">Zhang</span></span>. <span class="cit-pub-date">2020</span>. <span class="cit-article-title">A new coronavirus associated with human respiratory disease in China</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">579</span>: <span class="cit-fpage">265</span>–<span class="cit-lpage">269</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D579%26rft.spage%253D265%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=http://www.n&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.23.004176.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.33" data-doi="10.1101/gr.229102. Article published online before print in May 2002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kent</span>,  <span class="cit-name-given-names">W. J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. W.</span> <span class="cit-name-surname">Sugnet</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. S.</span> <span class="cit-name-surname">Furey</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Roskin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. H.</span> <span class="cit-name-surname">Pringle</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Zahler</span></span>, and and <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Haussler</span></span>. <span class="cit-pub-date">2002</span>. <span class="cit-article-title">The Human Genome Browser at UCSC</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr>. <span class="cit-vol">12</span>: <span class="cit-fpage">996</span>–<span class="cit-lpage">1006</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.229102.%2BArticle%2Bpublished%2Bonline%2Bbefore%2Bprint%2Bin%2BMay%2B2002%26rft_id%253Dinfo%253Apmid%252F12045153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEyLzYvOTk2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjgvMjAyMC4wMy4yMy4wMDQxNzYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Trolle</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. P.</span> <span class="cit-name-surname">McMurtrey</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Sidney</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Bardet</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. C.</span> <span class="cit-name-surname">Osborn</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Kaever</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sette</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W. H.</span> <span class="cit-name-surname">Hildebrand</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Nielsen</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Peters</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">The length distribution of class I restricted T cell epitopes is determined by both peptide supply and MHC allele specific binding preference</span>. <abbr class="cit-jnl-abbrev">J. Immunol. Baltim. Md 1950</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">1480</span>–<span class="cit-lpage">1487</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BImmunol.%2BBaltim.%2BMd%2B1950%26rft.volume%253D196%26rft.spage%253D1480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.35" data-doi="10.1016/j.it.2017.07.011"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mishto</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Liepe</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Post-Translational Peptide Splicing and T Cell Responses</span>. <abbr class="cit-jnl-abbrev">Trends Immunol</abbr>. <span class="cit-vol">38</span>: <span class="cit-fpage">904</span>–<span class="cit-lpage">915</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BImmunol%26rft.volume%253D38%26rft.spage%253D904%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.it.2017.07.011%26rft_id%253Dinfo%253Apmid%252F28830734%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.it.2017.07.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28830734&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.23.004176.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.36" data-doi="doi:10.1101/106054"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Alford</span>,  <span class="cit-name-given-names">R. F.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Leaver-Fay</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Jeliazkov</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">O’Meara</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F. P.</span> <span class="cit-name-surname">DiMaio</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Park</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. V.</span> <span class="cit-name-surname">Shapovalov</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. D.</span> <span class="cit-name-surname">Renfrew</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V. K.</span> <span class="cit-name-surname">Mulligan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kappel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">Labonte</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. S.</span> <span class="cit-name-surname">Pacella</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Bonneau</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Bradley</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Dunbrack</span>   <span class="cit-name-suffix">Jr.</span>.</span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Das</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Baker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Kuhlman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Kortemme</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Gray</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">The Rosetta All-Atom Energy Function for Macromolecular Modeling and Design</span>. <abbr class="cit-jnl-abbrev">J. Chem. Theory Comput</abbr>. <span class="cit-vol">13</span>: <span class="cit-fpage">3031</span>–<span class="cit-lpage">3048</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BChem.%2BTheory%2BComput%26rft.volume%253D13%26rft.spage%253D3031%26rft_id%253Dinfo%253Adoi%252Fdoi%253A10.1101%252F106054%26rft_id%253Dinfo%253Apmid%252F28430426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=doi:10.1101/106054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28430426&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.23.004176.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grifoni</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Sidney</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. H.</span> <span class="cit-name-surname">Scheuermann</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Peters</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sette</span></span>. <span class="cit-pub-date">2020</span>. <span class="cit-article-title">A Sequence Homology and Bioinformatic Approach Can Predict Candidate Targets for Immune Responses to SARS-CoV-2</span>. <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.38" data-doi="10.1093/nar/gkr859"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pickett</span>,  <span class="cit-name-given-names">B. E.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. L.</span> <span class="cit-name-surname">Sadat</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Noronha</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. B.</span> <span class="cit-name-surname">Squires</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Hunt</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kumar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Zaremba</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Gu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Zhou</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. N.</span> <span class="cit-name-surname">Larson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Dietrich</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. B.</span> <span class="cit-name-surname">Klem</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">R. H.</span> <span class="cit-name-surname">Scheuermann</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">ViPR: an open bioinformatics database and analysis resource for virology research</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">40</span>: <span class="cit-fpage">D593</span>–<span class="cit-lpage">D598</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr859%26rft_id%253Dinfo%253Apmid%252F22006842%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr859&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22006842&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.23.004176.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298601300089&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.39" data-doi="10.1146/annurev-immunol-032414-112334"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rossjohn</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Gras</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Miles</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Turner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. I.</span> <span class="cit-name-surname">Godfrey</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">McCluskey</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">T Cell Antigen Receptor Recognition of Antigen-Presenting Molecules</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Immunol</abbr>. <span class="cit-vol">33</span>: <span class="cit-fpage">169</span>–<span class="cit-lpage">200</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BImmunol%26rft.volume%253D33%26rft.spage%253D169%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-immunol-032414-112334%26rft_id%253Dinfo%253Apmid%252F25493333%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-immunol-032414-112334&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25493333&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.23.004176.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.40" data-doi="10.1146/annurev.immunol.23.021704.115658"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rudolph</span>,  <span class="cit-name-given-names">M. G.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Stanfield</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">I. A.</span> <span class="cit-name-surname">Wilson</span></span>. <span class="cit-pub-date">2006</span>. <span class="cit-article-title">How TCRs bind MHCs, peptides, and coreceptors</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Immunol</abbr>. <span class="cit-vol">24</span>: <span class="cit-fpage">419</span>–<span class="cit-lpage">466</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bimmunology%26rft.stitle%253DAnnu%2BRev%2BImmunol%26rft.aulast%253DRudolph%26rft.auinit1%253DM.%2BG.%26rft.volume%253D24%26rft.spage%253D419%26rft.epage%253D466%26rft.atitle%253DHow%2BTCRs%2Bbind%2BMHCs%252C%2Bpeptides%252C%2Band%2Bcoreceptors.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.immunol.23.021704.115658%26rft_id%253Dinfo%253Apmid%252F16551255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.immunol.23.021704.115658&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16551255&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.23.004176.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237583300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Borrman</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Cimons</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Cosiano</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Purcaro</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. G.</span> <span class="cit-name-surname">Pierce</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. M.</span> <span class="cit-name-surname">Baker</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Weng</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">ATLAS: A database linking binding affinities with structures for wild-type and mutant TCR-pMHC complexes</span>. <abbr class="cit-jnl-abbrev">Proteins Struct. Funct. Bioinforma</abbr>. <span class="cit-vol">85</span>: <span class="cit-fpage">908</span>–<span class="cit-lpage">916</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteins%2BStruct.%2BFunct.%2BBioinforma%26rft.volume%253D85%26rft.spage%253D908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Riley</span>,  <span class="cit-name-given-names">T. P.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. L. J.</span> <span class="cit-name-surname">Keller</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Smith</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. M.</span> <span class="cit-name-surname">Davancaze</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. G.</span> <span class="cit-name-surname">Arbuiso</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Devlin</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">B. M.</span> <span class="cit-name-surname">Baker</span></span>. <span class="cit-pub-date">2019</span>. <span class="cit-article-title">Structure Based Prediction of Neoantigen Immunogenicity</span>. <abbr class="cit-jnl-abbrev">Front. Immunol</abbr>. <span class="cit-vol">10</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.43" data-doi="10.4049/jimmunol.1800683"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mallon</span>,  <span class="cit-name-given-names">D. H.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Kling</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Robb</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Ellinghaus</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Bradley</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Taylor</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Kabelitz</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Kosmoliaptsis</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Predicting Humoral Alloimmunity from Differences in Donor and Recipient HLA Surface Electrostatic Potential</span>. <abbr class="cit-jnl-abbrev">J. Immunol</abbr>. <span class="cit-vol">201</span>: <span class="cit-fpage">3780</span>–<span class="cit-lpage">3792</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BImmunol%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.1800683%26rft_id%253Dinfo%253Apmid%252F30429288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTE6IjIwMS8xMi8zNzgwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjgvMjAyMC4wMy4yMy4wMDQxNzYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.44" data-doi="10.1073/pnas.181342398"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Baker</span>,  <span class="cit-name-given-names">N. A.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Sept</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Joseph</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Holst</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">McCammon</span></span>. <span class="cit-pub-date">2001</span>. <span class="cit-article-title">Electrostatics of nanosystems: application to microtubules and the ribosome</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>. <span class="cit-vol">98</span>: <span class="cit-fpage">10037</span>–<span class="cit-lpage">10041</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.181342398%26rft_id%253Dinfo%253Apmid%252F11517324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTgvMTgvMTAwMzciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yOC8yMDIwLjAzLjIzLjAwNDE3Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.23.004176v2.45" data-doi="10.1002/pro.3280"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jurrus</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Engel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Star</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Monson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Brandi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. E.</span> <span class="cit-name-surname">Felberg</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. H.</span> <span class="cit-name-surname">Brookes</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Wilson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Liles</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Chun</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. W.</span> <span class="cit-name-surname">Gohara</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Dolinsky</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Konecny</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Koes</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Nielsen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Head-Gordon</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Geng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Krasny</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.-W.</span> <span class="cit-name-surname">Wei</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Holst</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">McCammon</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Baker</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Improvements to the APBS biomolecular solvation software suite</span>. <abbr class="cit-jnl-abbrev">Protein Sci</abbr>. <span class="cit-vol">27</span>: <span class="cit-fpage">112</span>–<span class="cit-lpage">128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtein%2BSci%26rft.volume%253D27%26rft.spage%253D112%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpro.3280%26rft_id%253Dinfo%253Apmid%252F28836357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/pro.3280&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28836357&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F28%2F2020.03.23.004176.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-book" id="cit-2020.03.23.004176v2.46"><div class="cit-metadata"><cite><span class="cit-source">The PyMOL Molecular Graphics System</span>,. <span class="cit-publ-name">Schrödinger, LLC</span>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.23.004176v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.23.004176v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.03.20.999730v3" title="Identification of antiviral drug candidates against SARS-CoV-2 from FDA-approved drugs" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.26.010165v1" title="Structural Basis for Potent Neutralization of Betacoronaviruses by Single-domain Camelid Antibodies" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 28, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.23.004176v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.03.23.004176v2.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/early/2020/03/28/2020.03.23.004176.external-links" target="_self" class="link-icon"><i class="icon-file"></i> <span class="title">Data/Code</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.23.004176v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.23.004176v2" class="active" data-icon-position="" data-hide-link-title="0">Structure-based modeling of SARS-CoV-2 peptide/HLA-A02 antigens</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1215884" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-PO3yqpxum5GTnwnVrHYU3x7caxmvAf7LfkC_MUjPg1I" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1215884" id="node1215884--2" data-pisa="biorxiv;2020.03.23.004176v2" data-pisa-master="biorxiv;2020.03.23.004176" data-seqnum="1215884" data-apath="/biorxiv/early/2020/03/28/2020.03.23.004176.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Structure-based modeling of SARS-CoV-2 peptide/HLA-A02 antigens</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Santrupti</span> <span class="nlm-surname">Nerli</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Nikolaos G.</span> <span class="nlm-surname">Sgourakis</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.23.004176; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.23.004176 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.23.004176v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.23.004176v2&amp;title=Structure-based%20modeling%20of%20SARS-CoV-2%20peptide/HLA-A02%20antigens" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.23.004176v2&amp;title=Structure-based%20modeling%20of%20SARS-CoV-2%20peptide/HLA-A02%20antigens" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.23.004176v2&amp;text=Structure-based%20modeling%20of%20SARS-CoV-2%20peptide/HLA-A02%20antigens" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.23.004176v2&amp;title=Structure-based%20modeling%20of%20SARS-CoV-2%20peptide/HLA-A02%20antigens" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.23.004176v2&amp;t=Structure-based%20modeling%20of%20SARS-CoV-2%20peptide/HLA-A02%20antigens" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.23.004176v2&amp;title=Structure-based%20modeling%20of%20SARS-CoV-2%20peptide/HLA-A02%20antigens" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.23.004176v2&amp;title=Structure-based%20modeling%20of%20SARS-CoV-2%20peptide/HLA-A02%20antigens" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1215884" id="node1215884--3" data-pisa="biorxiv;2020.03.23.004176v2" data-pisa-master="biorxiv;2020.03.23.004176" data-seqnum="1215884" data-apath="/biorxiv/early/2020/03/28/2020.03.23.004176.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Structure-based modeling of SARS-CoV-2 peptide/HLA-A02 antigens</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Santrupti</span> <span class="nlm-surname">Nerli</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Nikolaos G.</span> <span class="nlm-surname">Sgourakis</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.23.004176; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.23.004176 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Structure-based%20modeling%20of%20SARS-CoV-2%20peptide/HLA-A02%20antigens&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Nerli%2C+Santrupti&amp;rft.au=Sgourakis%2C+Nikolaos+G."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1215884/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1215884/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1215884/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1215884/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1215884/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1215884/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1215884/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1215884/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1215884/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1215884/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1215884/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1215884/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.23.004176v2&amp;count=horizontal&amp;via=&amp;text=Structure-based%20modeling%20of%20SARS-CoV-2%20peptide/HLA-A02%20antigens&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.23.004176v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.23.004176v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.23.004176v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/immunology" class="highlight" data-icon-position="" data-hide-link-title="0">Immunology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
