<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/03/06/2020.02.17.952895.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.02.17.952895v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Functional pangenome analysis suggests inhibition of the protein E as a readily available therapy for COVID-2019"/>
  <meta name="DC.Identifier" content="10.1101/2020.02.17.952895"/>
  <meta name="DC.Date" content="2020-03-06"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="The spread of the novel coronavirus (SARS-CoV-2) has triggered a global emergency, that demands urgent solutions for detection and therapy to prevent escalating health, social and economic impacts. The spike protein (S) of this virus enables binding to the human receptor ACE2, and hence presents a prime target for vaccines preventing viral entry into host cells[1][1]. The S proteins from SARS-CoV-1 and SARS-CoV-2 are similar[2][2], but structural differences in the receptor binding domain (RBD) preclude the use of SARS-CoV-1–specific neutralizing antibodies to inhibit SARS-CoV-2[3][3]. Here we used comparative pangenomic analysis of all sequenced Betacoronaviruses to reveal that, among all core gene clusters present in these viruses, the envelope protein E shows a variant shared by SARS and SARS-Cov2 with two completely-conserved key functional features, an ion-channel and a PDZ-binding Motif (PBM). These features trigger a cytokine storm that activates the inflammasome, leading to increased edema in lungs causing the acute respiratory distress syndrome (ARDS)[4][4]-[6][5], the leading cause of death in SARS-CoV-1 and SARS-CoV-2 infection[7][6],[8][7]. However, three drugs approved for human use may inhibit SARS-CoV-1 and SARS-CoV-2 Protein E, either acting upon the ion channel (Amantadine and Hexamethylene amiloride[9][8],[10][9]) or the PBM (SB203580[5][10]), thereby potentially increasing the survival of the host, as already demonstrated for SARS-CoV-1in animal models. Hence, blocking the SARS protein E inhibits development of ARDS in vivo . Given that our results demonstrate that the protein E subcluster for the SARS clade is quasi -identical for the key functional regions of SARS-CoV-1 and SARS-CoV-2, we conclude that use of approved drugs shown to act as SARS E protein inhibitors can help prevent further casualties from COVID-2019 while vaccines and other preventive measures are being developed. [1]: #ref-1 [2]: #ref-2 [3]: #ref-3 [4]: #ref-4 [5]: #ref-6 [6]: #ref-7 [7]: #ref-8 [8]: #ref-9 [9]: #ref-10 [10]: #ref-5"/>
  <meta name="DC.Contributor" content="Intikhab alam"/>
  <meta name="DC.Contributor" content="Allan Kamau"/>
  <meta name="DC.Contributor" content="Maxat Kulmanov"/>
  <meta name="DC.Contributor" content="Stefan T. Arold"/>
  <meta name="DC.Contributor" content="Arnab Pain"/>
  <meta name="DC.Contributor" content="Takashi Gojobori"/>
  <meta name="DC.Contributor" content="Carlos M. Duarte"/>
  <meta name="article:published_time" content="2020-03-06"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Functional pangenome analysis suggests inhibition of the protein E as a readily available therapy for COVID-2019"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;The spread of the novel coronavirus (SARS-CoV-2) has triggered a global emergency, that demands urgent solutions for detection and therapy to prevent escalating health, social and economic impacts. The spike protein (S) of this virus enables binding to the human receptor ACE2, and hence presents a prime target for vaccines preventing viral entry into host cells. The S proteins from SARS-CoV-1 and SARS-CoV-2 are similar, but structural differences in the receptor binding domain (RBD) preclude the use of SARS-CoV-1-specific neutralizing antibodies to inhibit SARS-CoV-2. Here we used comparative pangenomic analysis of all sequenced Betacoronaviruses to reveal that, among all core gene clusters present in these viruses, the envelope protein E shows a variant shared by SARS and SARS-Cov2 with two completely-conserved key functional features, an ion-channel and a PDZ-binding Motif (PBM). These features trigger a cytokine storm that activates the inflammasome, leading to increased edema in lungs causing the acute respiratory distress syndrome (ARDS), the leading cause of death in SARS-CoV-1 and SARS-CoV-2 infection. However, three drugs approved for human use may inhibit SARS-CoV-1 and SARS-CoV-2 Protein E, either acting upon the ion channel (Amantadine and Hexamethylene amiloride) or the PBM (SB203580), thereby potentially increasing the survival of the host, as already demonstrated for SARS-CoV-1in animal models. Hence, blocking the SARS protein E inhibits development of ARDS in vivo. Given that our results demonstrate that the protein E subcluster for the SARS clade is quasi-identical for the key functional regions of SARS-CoV-1 and SARS-CoV-2, we conclude that use of approved drugs shown to act as SARS E protein inhibitors can help prevent further casualties from COVID-2019 while vaccines and other preventive measures are being developed.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.02.17.952895v2"/>
  <meta name="citation_id" content="2020.02.17.952895v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.17.952895v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.17.952895v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.17.952895v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.02.17.952895"/>
  <meta name="citation_num_pages" content="17"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.02.17.952895"/>
  <meta name="citation_author" content="Intikhab alam"/>
  <meta name="citation_author_institution" content="King Abdullah University of Science and Technology (KAUST), Computational Bioscience Research Center (CBRC), Biological and Environmental Science and Engineering (BESE)"/>
  <meta name="citation_author_email" content="Intikhab.Alam@kaust.edu.sa"/>
  <meta name="citation_author_email" content="Carlos.Duarte@kaust.edu.sa"/>
  <meta name="citation_author" content="Allan Kamau"/>
  <meta name="citation_author_institution" content="King Abdullah University of Science and Technology (KAUST), Computational Bioscience Research Center (CBRC), Biological and Environmental Science and Engineering (BESE)"/>
  <meta name="citation_author" content="Maxat Kulmanov"/>
  <meta name="citation_author_institution" content="King Abdullah University of Science and Technology (KAUST), Computational Bioscience Research Center (CBRC), Biological and Environmental Science and Engineering (BESE)"/>
  <meta name="citation_author" content="Stefan T. Arold"/>
  <meta name="citation_author_institution" content="King Abdullah University of Science and Technology (KAUST), Computational Bioscience Research Center (CBRC), Biological and Environmental Science and Engineering (BESE)"/>
  <meta name="citation_author_institution" content="Centre de Biochimie Structurale, CNRS, INSERM, Université de Montpellier"/>
  <meta name="citation_author" content="Arnab Pain"/>
  <meta name="citation_author_institution" content="Biological and Environmental Science and Engineering (BESE)"/>
  <meta name="citation_author_institution" content="Research Center for Zoonosis Control, Hokkaido University"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1755-2819"/>
  <meta name="citation_author" content="Takashi Gojobori"/>
  <meta name="citation_author_institution" content="King Abdullah University of Science and Technology (KAUST), Computational Bioscience Research Center (CBRC), Biological and Environmental Science and Engineering (BESE)"/>
  <meta name="citation_author" content="Carlos M. Duarte"/>
  <meta name="citation_author_institution" content="King Abdullah University of Science and Technology (KAUST), Computational Bioscience Research Center (CBRC), Biological and Environmental Science and Engineering (BESE)"/>
  <meta name="citation_author_email" content="Intikhab.Alam@kaust.edu.sa"/>
  <meta name="citation_author_email" content="Carlos.Duarte@kaust.edu.sa"/>
  <meta name="citation_reference" content="Li, F. Structure, Function, and Evolution of Coronavirus Spike Proteins. Annu Rev Virol 3, 237–261, doi: 10.1146/annurev-virology-110615-042301 (2016)."/>
  <meta name="citation_reference" content="Wan, Y., Shang, J., Graham, R., Baric, R. S. &amp;; Li, F. Receptor recognition by novel coronavirus from Wuhan: An analysis based on decade-long structural studies of SARS. J Virol, doi: 10.1128/JVI.00127-20 (2020)."/>
  <meta name="citation_reference" content="Wrapp, D. et al. Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation. Science, doi: 10.1126/science.abb2507 (2020)."/>
  <meta name="citation_reference" content="Nieto-Torres, J. L. et al. Severe acute respiratory syndrome coronavirus envelope protein ion channel activity promotes virus fitness and pathogenesis. PLoS Pathog 10, e1004077, doi: 10.1371/journal.ppat.1004077 (2014)."/>
  <meta name="citation_reference" content="Jimenez-Guardeno, J. M. et al. The PDZ-binding motif of severe acute respiratory syndrome coronavirus envelope protein is a determinant of viral pathogenesis. PLoS Pathog 10, e1004320, doi: 10.1371/journal.ppat.1004320 (2014)."/>
  <meta name="citation_reference" content="Torres, J., Surya, W., Li, Y. &amp;; Liu, D. X. Protein-Protein Interactions of Viroporins in Coronaviruses and Paramyxoviruses: New Targets for Antivirals? Viruses 7, 2858–2883, doi: 10.3390/v7062750 (2015)."/>
  <meta name="citation_reference" content="Huang, C. et al. Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China. Lancet 395, 497–506, doi: 10.1016/S0140-6736(20)30183-5 (2020)."/>
  <meta name="citation_reference" content="Xu, Z. et al. Pathological findings of COVID-19 associated with acute respiratory distress syndrome. Lancet Respir Med, doi: 10.1016/S2213-2600(20)30076-X (2020)."/>
  <meta name="citation_reference" content="Wilson, L., Gage, P. &amp;; Ewart, G. Hexamethylene amiloride blocks E protein ion channels and inhibits coronavirus replication. Virology 353, 294–306, doi: 10.1016/j.virol.2006.05.028 (2006)."/>
  <meta name="citation_reference" content="Behmard, E., Abdolmaleki, P. &amp;; Taghdir, M. Understanding the inhibitory mechanism of BIT225 drug against p7 viroporin using computational study. Biophys Chem 233, 47–54, doi: 10.1016/j.bpc.2017.11.002 (2018)."/>
  <meta name="citation_reference" content="Wu, A. et al. Genome Composition and Divergence of the Novel Coronavirus (2019-nCoV) Originating in China. Cell Host Microbe, doi: 10.1016/j.chom.2020.02.001 (2020)."/>
  <meta name="citation_reference" content="Ding, W., Baumdicker, F. &amp;; Neher, R. A. panX: pan-genome analysis and exploration. Nucleic Acids Res 46, e5, doi: 10.1093/nar/gkx977 (2018)."/>
  <meta name="citation_reference" content="Masters, P. S. The molecular biology of coronaviruses. Adv Virus Res 66, 193–292, doi: 10.1016/S0065-3527(06)66005-3 (2006)."/>
  <meta name="citation_reference" content="Castano-Rodriguez, C. et al. Role of Severe Acute Respiratory Syndrome Coronavirus Viroporins E, 3a, and 8a in Replication and Pathogenesis. mBio 9, doi: 10.1128/mBio.02325-17 (2018)."/>
  <meta name="citation_reference" content="Minakshi, R. et al. The SARS Coronavirus 3a protein causes endoplasmic reticulum stress and induces ligand-independent downregulation of the type 1 interferon receptor. PLoS One 4, e8342, doi: 10.1371/journal.pone.0008342 (2009)."/>
  <meta name="citation_reference" content="Hanel, K., Stangler, T., Stoldt, M. &amp;; Willbold, D. Solution structure of the X4 protein coded by the SARS related coronavirus reveals an immunoglobulin like fold and suggests a binding activity to integrin I domains. J Biomed Sci 13, 281–293, doi: 10.1007/s11373-005-9043-9 (2006)."/>
  <meta name="citation_reference" content="DeDiego, M. L. et al. Inhibition of NF-kappaB-mediated inflammation in severe acute respiratory syndrome coronavirus-infected mice increases survival. J Virol 88, 913–924, doi: 10.1128/JVI.02576-13 (2014)."/>
  <meta name="citation_reference" content="Kulmanov, M. &amp;; Hoehndorf, R. DeepGOPlus: improved protein function prediction from sequence. Bioinformatics 36, 422–429, doi: 10.1093/bioinformatics/btz595 (2020)."/>
  <meta name="citation_reference" content="Liao, Y., Tam, J. P. &amp;; Liu, D. X. Viroporin activity of SARS-CoV E protein. Adv Exp Med Biol 581, 199–202, doi: 10.1007/978-0-387-33012-9_34 (2006)."/>
  <meta name="citation_reference" content="DeDiego, M. L. et al. A severe acute respiratory syndrome coronavirus that lacks the E gene is attenuated in vitro and in vivo. Journal of Virology 81, 1701–1713, doi: 10.1128/Jvi.01467-06 (2007)."/>
  <meta name="citation_reference" content="Lamirande, E. W. et al. A live attenuated severe acute respiratory syndrome coronavirus is immunogenic and efficacious in golden Syrian hamsters. J Virol 82, 7721–7724, doi: 10.1128/JVI.00304-08 (2008)."/>
  <meta name="citation_reference" content="Regla-Nava, J. A. et al. Severe acute respiratory syndrome coronaviruses with mutations in the E protein are attenuated and promising vaccine candidates. J Virol 89, 3870–3887, doi: 10.1128/JVI.03566-14 (2015)."/>
  <meta name="citation_reference" content="Schoeman, D. &amp;; Fielding, B. C. Coronavirus envelope protein: current knowledge. Virol J 16, 69, doi: 10.1186/s12985-019-1182-0 (2019)."/>
  <meta name="citation_reference" content="Corman, V. M. et al. Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR. Euro Surveill 25, doi: 10.2807/1560-7917.ES.2020.25.3.2000045 (2020)."/>
  <meta name="citation_reference" content="Kruse, R. L. Therapeutic strategies in an outbreak scenario to treat the novel coronavirus originating in Wuhan, China. F1000Res 9, 72, doi: 10.12688/f1000research.22211.2 (2020)."/>
  <meta name="citation_reference" content="Liu, W., Morse, J. S., Lalonde, T. &amp;; Xu, S. Learning from the Past: Possible Urgent Prevention and Treatment Options for Severe Acute Respiratory Infections Caused by 2019-nCoV. Chembiochem, doi: 10.1002/cbic.202000047 (2020)."/>
  <meta name="citation_reference" content="Alam, I. et al. INDIGO - INtegrated data warehouse of microbial genomes with examples from the red sea extremophiles. PLoS One 8, e82210, doi: 10.1371/journal.pone.0082210 (2013)."/>
  <meta name="citation_reference" content="Almagro Armenteros, J. J. et al. SignalP 5.0 improves signal peptide predictions using deep neural networks. Nat Biotechnol 37, 420–423, doi: 10.1038/s41587-019-0036-z (2019)."/>
  <meta name="citation_reference" content="Waterhouse, A. et al. SWISS-MODEL: homology modelling of protein structures and complexes. Nucleic Acids Res 46, W296–W303, doi: 10.1093/nar/gky427 (2018)."/>
  <meta name="citation_reference" content="Xu, D. &amp;; Zhang, Y. Ab initio protein structure assembly using continuous structure fragments and optimized knowledge-based force field. Proteins 80, 1715–1735, doi: 10.1002/prot.24065 (2012)."/>
  <meta name="twitter:title" content="Functional pangenome analysis suggests inhibition of the protein E as a readily available therapy for COVID-2019"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="The spread of the novel coronavirus (SARS-CoV-2) has triggered a global emergency, that demands urgent solutions for detection and therapy to prevent escalating health, social and economic impacts. The spike protein (S) of this virus enables binding to the human receptor ACE2, and hence presents a prime target for vaccines preventing viral entry into host cells[1][1]. The S proteins from SARS-CoV-1 and SARS-CoV-2 are similar[2][2], but structural differences in the receptor binding domain (RBD) preclude the use of SARS-CoV-1–specific neutralizing antibodies to inhibit SARS-CoV-2[3][3]. Here we used comparative pangenomic analysis of all sequenced Betacoronaviruses to reveal that, among all core gene clusters present in these viruses, the envelope protein E shows a variant shared by SARS and SARS-Cov2 with two completely-conserved key functional features, an ion-channel and a PDZ-binding Motif (PBM). These features trigger a cytokine storm that activates the inflammasome, leading to increased edema in lungs causing the acute respiratory distress syndrome (ARDS)[4][4]-[6][5], the leading cause of death in SARS-CoV-1 and SARS-CoV-2 infection[7][6],[8][7]. However, three drugs approved for human use may inhibit SARS-CoV-1 and SARS-CoV-2 Protein E, either acting upon the ion channel (Amantadine and Hexamethylene amiloride[9][8],[10][9]) or the PBM (SB203580[5][10]), thereby potentially increasing the survival of the host, as already demonstrated for SARS-CoV-1in animal models. Hence, blocking the SARS protein E inhibits development of ARDS in vivo . Given that our results demonstrate that the protein E subcluster for the SARS clade is quasi -identical for the key functional regions of SARS-CoV-1 and SARS-CoV-2, we conclude that use of approved drugs shown to act as SARS E protein inhibitors can help prevent further casualties from COVID-2019 while vaccines and other preventive measures are being developed. [1]: #ref-1 [2]: #ref-2 [3]: #ref-3 [4]: #ref-4 [5]: #ref-6 [6]: #ref-7 [7]: #ref-8 [8]: #ref-9 [9]: #ref-10 [10]: #ref-5"/>
  <meta name="og-title" property="og:title" content="Functional pangenome analysis suggests inhibition of the protein E as a readily available therapy for COVID-2019"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.17.952895v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="The spread of the novel coronavirus (SARS-CoV-2) has triggered a global emergency, that demands urgent solutions for detection and therapy to prevent escalating health, social and economic impacts. The spike protein (S) of this virus enables binding to the human receptor ACE2, and hence presents a prime target for vaccines preventing viral entry into host cells[1][1]. The S proteins from SARS-CoV-1 and SARS-CoV-2 are similar[2][2], but structural differences in the receptor binding domain (RBD) preclude the use of SARS-CoV-1–specific neutralizing antibodies to inhibit SARS-CoV-2[3][3]. Here we used comparative pangenomic analysis of all sequenced Betacoronaviruses to reveal that, among all core gene clusters present in these viruses, the envelope protein E shows a variant shared by SARS and SARS-Cov2 with two completely-conserved key functional features, an ion-channel and a PDZ-binding Motif (PBM). These features trigger a cytokine storm that activates the inflammasome, leading to increased edema in lungs causing the acute respiratory distress syndrome (ARDS)[4][4]-[6][5], the leading cause of death in SARS-CoV-1 and SARS-CoV-2 infection[7][6],[8][7]. However, three drugs approved for human use may inhibit SARS-CoV-1 and SARS-CoV-2 Protein E, either acting upon the ion channel (Amantadine and Hexamethylene amiloride[9][8],[10][9]) or the PBM (SB203580[5][10]), thereby potentially increasing the survival of the host, as already demonstrated for SARS-CoV-1in animal models. Hence, blocking the SARS protein E inhibits development of ARDS in vivo . Given that our results demonstrate that the protein E subcluster for the SARS clade is quasi -identical for the key functional regions of SARS-CoV-1 and SARS-CoV-2, we conclude that use of approved drugs shown to act as SARS E protein inhibitors can help prevent further casualties from COVID-2019 while vaccines and other preventive measures are being developed. [1]: #ref-1 [2]: #ref-2 [3]: #ref-3 [4]: #ref-4 [5]: #ref-6 [6]: #ref-7 [7]: #ref-8 [8]: #ref-9 [9]: #ref-10 [10]: #ref-5"/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-03-06"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Functional pangenome analysis suggests inhibition of the protein E as a readily available therapy for COVID-2019 | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1183036 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-2">The spread of the novel coronavirus (SARS-CoV-2) has triggered a global emergency, that demands urgent solutions for detection and therapy to prevent escalating health, social and economic impacts. The spike protein (S) of this virus enables binding to the human receptor ACE2, and hence presents a prime target for vaccines preventing viral entry into host cells
     <sup>
      <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>
     </sup>. The S proteins from SARS-CoV-1 and SARS-CoV-2 are similar
     <sup>
      <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>
     </sup>, but structural differences in the receptor binding domain (RBD) preclude the use of SARS-CoV-1–specific neutralizing antibodies to inhibit SARS-CoV-2
     <sup>
      <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>
     </sup>. Here we used comparative pangenomic analysis of all sequenced 
     <em>Betacoronaviruses</em> to reveal that, among all core gene clusters present in these viruses, the envelope protein E shows a variant shared by SARS and SARS-Cov2 with two completely-conserved key functional features, an ion-channel and a PDZ-binding Motif (PBM). These features trigger a cytokine storm that activates the inflammasome, leading to increased edema in lungs causing the acute respiratory distress syndrome (ARDS)
     <sup>
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>-
      <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>
     </sup>, the leading cause of death in SARS-CoV-1 and SARS-CoV-2 infection
     <sup>
      <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>,
      <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>
     </sup>. However, three drugs approved for human use may inhibit SARS-CoV-1 and SARS-CoV-2 Protein E, either acting upon the ion channel (Amantadine and Hexamethylene amiloride
     <sup>
      <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>,
      <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>
     </sup>) or the PBM (SB203580
     <sup>
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>
     </sup>), thereby potentially increasing the survival of the host, as already demonstrated for SARS-CoV-1in animal models. Hence, blocking the SARS protein E inhibits development of ARDS 
     <em>in vivo</em>. Given that our results demonstrate that the protein E subcluster for the SARS clade is 
     <em>quasi</em>-identical for the key functional regions of SARS-CoV-1 and SARS-CoV-2, we conclude that use of approved drugs shown to act as SARS E protein inhibitors can help prevent further casualties from COVID-2019 while vaccines and other preventive measures are being developed.
    </p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Introduction</h2>
    <p id="p-4">The current epidemic of the novel coronavirus (SARS-CoV-2) has triggered a rising global health emergency (WHO 
     <a href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019">https://www.who.int/emergencies/diseases/novel-coronavirus-2019</a>) with disruptive health, social and economic repercussions. Three major actions, confinement, detection and therapy are required to contain this pandemic. Detection and therapy can be guided by virus sequence data, which was first made available in 17 December, 2019 at gsaid.org, soon after detection of the first cases. The SARS-CoV-2 genome sequence data are a pivotal resource on its own, but can gain greater value when embedded within those of other 
     <em>Betacoronavirus</em>, allowing a comparative pangenomic analysis. This approach can help identify the core genome of 
     <em>Betacoronaviruses</em> and extract accessory genomic features shared by a subset of these viruses or unique to SARS-CoV-2. Whereas core genomic features are required for the virus to be functional, accessory features are candidates to provide insights into the drivers of the unique capacities of SARS-CoV-2 explaining its spread and virulence. Genome annotation and structural modelling can then be used to assess the possible functions of these accessory features and guide approaches to detection and treatment.
    </p>
    <p id="p-5">Here we apply a comparative pangenomic approach of all 
     <em>Betacoronavirus</em> genomes sequenced thus far (
     <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>), to detect the core and accessory gene cluster of this genus, and then annotate the functions, further assessed through structural analysis. Furthermore, we provide insights into importance, key features and differences in the Envelope protein, E, conserved separately in SARS, MERS and coronaviruses from other animals.
    </p>
    <div id="T1" class="table pos-float">
     <div class="table-inline table-callout-links">
      <div class="callout">
       <span>View this table:</span>
       <ul class="callout-links">
        <li class="view-inline first">
         <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1188078/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
        </li>
        <li class="view-popup">
         <a href="/highwire/markup/1188078/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
        </li>
        <li class="download-ppt last">
         <a href="/highwire/powerpoint/1188078" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
        </li>
       </ul>
      </div>
     </div>
     <div class="table-caption">
      <span class="table-label">Table 1.</span>
      <p id="p-6" class="first-child">A list of 
       <em>Betacoronavirus</em> genomes and metadata included in this study
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Results and Discussion</h2>
    <p id="p-7">We used a pangenome approach to explore the genome of SARS-CoV-2 (MN985325 and related isolates) in comparison to other 
     <em>Betacornaviruses</em>, including SARS and MERS. Previous approaches
     <sup>
      <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>
     </sup> proceeded by extracting individual genes, aligning them, and then establishing a phylogeny. Our approach differs in that we first cluster all sequences to then calculate the alignment. This approach, based on using panX
     <sup>
      <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>
     </sup>, allows us to achieve a higher sensibility in the detection of gene clusters. The resulting pangenome including all core and accessory gene clusters, interactive trees and alignments from 24 
     <em>Betacoronavirus</em> genomes, including 4 isolates from SARS-CoV-2, is available at 
     <a href="https://pangenomedb.cbrc.kaust.edu.sa/">https://pangenomedb.cbrc.kaust.edu.sa/</a>.
    </p>
    <p id="p-8">There are five essential genes in coronaviruses, the Spike protein (S), Membrane glycoprotein (M), Nucleocapsid (N), envelope protein (E) and the Orf1ab (a large polyprotein known as replicase/protease), all required to produce a structurally complete viral particle
     <sup>
      <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>
     </sup>. These five genes define the core pan-genome. However, panX
     <sup>
      <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>
     </sup> only retrieved four of these (S, M, N and the ORF1ab) as components of the core pan-genome (
     <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). However, the fifth gene in the 
     <em>Betacoronavirus</em> core genome, the envelope protein (E), varied among genomes, defining three distinct subclusters within the envelope protein E of 
     <em>Betacoronaviruses</em> (
     <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). One of these E clusters comprises only SARS-CoV-1 (AY274119), SARS-CoV-2 and two Bat Coronaviruses (MG772933 and KF63652). Among the other two protein E gene clusters, one includes MERS (JX869059 and KC164505) and several coronaviruses from different animals, while the third cluster includes coronaviruses from two 
     <em>Rousettus</em> bats. To validate the conservation and specificity of the SARS E cluster, we compared the SARS-CoV-2 E sequence with all known sequences in NCBI’s non redundant (NR) protein database. This comparison showed, consistent with our pan-genome analysis, that the subcluster of the protein E gene appearing in SARS-CoV-2, SARS and two bat coronaviruses, is defined by the same essential functional features, which are exactly conserved (100% similarity) between bats and SARS-CoV-2, but differ slightly (95% similarity with one deletion and three substitutions) from SARS (
     <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a> and 
     <a id="xref-fig-9-1" class="xref-fig" href="#F9">supplementary figure 7</a>). A recent study compared genomes and gene families of all alpha, beta, delta and gamma coronaviruses, but did not explore the variability present within the protein E gene sequences, which we did through the clustering approach reported above.
    </p>
    <div id="F1" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Pangenomic gene clusters based on sequence comparison of genes from genomes of genus Betacoronavirus. A. 37 gene clusters are shown with a binary heatmap representing presence (black) or absence (white) of genes in Betacoronavirus clades 1, 2, 3 and 4. Clade 2 is expanded to show presence absence of genes for its members that include SARS-CoV-2, SARS and two bat coronaviruses (MG772933 and KF636752). One of the gene cluster, orf10, marked in yellow, is a case of annotation artifact as it appears to be unique in SARS-CoV-2 according to annotations from GenBank, this gene is not predicted in any other genomes, however a TBLASTN search of this protein against NCBI[[#x2019]]s Nucleotide database (NT) show sequence matches this gene with 100% coverage in other SARS and SARS-like coronaviruses. B. This panel shows a phylogenetic tree based on SNPs from core (M, N, orf1ab and S) gene clusters. Tree is labelled with Clade numbers to distinguish SARS-like and other coronaviruses. Coloring of the tree is obtained based on related host information." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-542279032" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Pangenomic gene clusters based on sequence comparison of genes from genomes of genus &lt;;em&gt;;Betacoronavirus&lt;;/em&gt;;.&lt;;/span&gt;; A. 37 gene clusters are shown with a binary heatmap representing presence (black) or absence (white) of genes in &lt;;em&gt;;Betacoronavirus&lt;;/em&gt;; clades 1, 2, 3 and 4. Clade 2 is expanded to show presence absence of genes for its members that include SARS-CoV-2, SARS and two bat coronaviruses (MG772933 and KF636752). One of the gene cluster, orf10, marked in yellow, is a case of annotation artifact as it appears to be unique in SARS-CoV-2 according to annotations from GenBank, this gene is not predicted in any other genomes, however a TBLASTN search of this protein against NCBI[[#x2019]]s Nucleotide database (NT) show sequence matches this gene with 100% coverage in other SARS and SARS-like coronaviruses. B. This panel shows a phylogenetic tree based on SNPs from core (M, N, orf1ab and S) gene clusters. Tree is labelled with Clade numbers to distinguish SARS-like and other coronaviruses. Coloring of the tree is obtained based on related host information.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F1.medium.gif" width="440" height="307"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F1.medium.gif" width="440" height="307"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 1.</span>
      <span class="caption-title">Pangenomic gene clusters based on sequence comparison of genes from genomes of genus 
       <em>Betacoronavirus</em>.
      </span>
      <p id="p-9" class="first-child">A. 37 gene clusters are shown with a binary heatmap representing presence (black) or absence (white) of genes in 
       <em>Betacoronavirus</em> clades 1, 2, 3 and 4. Clade 2 is expanded to show presence absence of genes for its members that include SARS-CoV-2, SARS and two bat coronaviruses (MG772933 and KF636752). One of the gene cluster, orf10, marked in yellow, is a case of annotation artifact as it appears to be unique in SARS-CoV-2 according to annotations from GenBank, this gene is not predicted in any other genomes, however a TBLASTN search of this protein against NCBI’s Nucleotide database (NT) show sequence matches this gene with 100% coverage in other SARS and SARS-like coronaviruses.
      </p>
      <p id="p-10">B. This panel shows a phylogenetic tree based on SNPs from core (M, N, orf1ab and S) gene clusters. Tree is labelled with Clade numbers to distinguish SARS-like and other coronaviruses. Coloring of the tree is obtained based on related host information.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F2" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Pangenome analysis of 3 clusters related E protein. A. The first E cluster, it shows much similar E proteins from SARS and SARS-like genomes, highlighted in the species tree (A left panel) alongside a gene tree (A right panel). B. Protein alignment of SARS and SARS-like E protein cluster. It includes SARS (AY274119), SARS-COV-2 (MN985325 and other isolates) and two bat coronaviruses (MG772933 and KF636752). Two features are important here, a set of ion-channel forming amino acids (N15 and V25) and the PBM class II motif (DLLV) completely conserved in SARS and SARS-CoV-2, potential targets for COVID-2019 complications such as Acute Respiratory Distress Syndrome (ARDS). C. Protein alignment from another E cluster that groups E sequences from clade 3 and 4, including MERS and coronaviruses from other animals. D. Protein alignment of 3rd E cluster that groups sequences from clade 1, related to two bat coronaviruses." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-542279032" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Pangenome analysis of 3 clusters related E protein.&lt;;/span&gt;; A. The first E cluster, it shows much similar E proteins from SARS and SARS-like genomes, highlighted in the species tree (A left panel) alongside a gene tree (A right panel). B. Protein alignment of SARS and SARS-like E protein cluster. It includes SARS (AY274119), SARS-COV-2 (MN985325 and other isolates) and two bat coronaviruses (MG772933 and KF636752). Two features are important here, a set of ion-channel forming amino acids (N15 and V25) and the PBM class II motif (DLLV) completely conserved in SARS and SARS-CoV-2, potential targets for COVID-2019 complications such as Acute Respiratory Distress Syndrome (ARDS). C. Protein alignment from another E cluster that groups E sequences from clade 3 and 4, including MERS and coronaviruses from other animals. D. Protein alignment of 3&lt;;sup&gt;;rd&lt;;/sup&gt;; E cluster that groups sequences from clade 1, related to two bat coronaviruses.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F2.medium.gif" width="407" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F2.medium.gif" width="407" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 2.</span>
      <span class="caption-title">Pangenome analysis of 3 clusters related E protein.</span>
      <p id="p-11" class="first-child">A. The first E cluster, it shows much similar E proteins from SARS and SARS-like genomes, highlighted in the species tree (A left panel) alongside a gene tree (A right panel).</p>
      <p id="p-12">B. Protein alignment of SARS and SARS-like E protein cluster. It includes SARS (AY274119), SARS-COV-2 (MN985325 and other isolates) and two bat coronaviruses (MG772933 and KF636752). Two features are important here, a set of ion-channel forming amino acids (N15 and V25) and the PBM class II motif (DLLV) completely conserved in SARS and SARS-CoV-2, potential targets for COVID-2019 complications such as Acute Respiratory Distress Syndrome (ARDS).</p>
      <p id="p-13">C. Protein alignment from another E cluster that groups E sequences from clade 3 and 4, including MERS and coronaviruses from other animals.</p>
      <p id="p-14">D. Protein alignment of 3
       <sup>rd</sup> E cluster that groups sequences from clade 1, related to two bat coronaviruses.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-15">We found five accessory gene clusters (15% of the accessory genome of genus 
     <em>Betacoronavirus</em>) common to SARS-like coronaviruses, including SARS-CoV-2, out of 33 total accessory gene clusters. Four of these gene clusters (ORF3a, ORF6, ORF7a, and ORF8, 
     <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1A</a>), appear only in SARS-CoV-2, bat-SL-CoVZC45 and the SARS virus. A gene cluster annotated as ORF10 appeared to be unique to SARS-CoV-2 (
     <a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1</a>). However, a blast-based search against the DNA of all sequences in NCBI’s NT database showed that the ORF10 sequence matches with 100% coverage in DNA of other SARS-like genomes, but there is no open-reading frame predicted for those genomes in the matching region (
     <a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig 1A</a>). Hence, the apparent uniqueness of ORF10 might simply be an artifact of the GeneBank annotation pipeline that did not predict an ORF for this gene in other matching genomes. However, for completeness, we report a functional analysis of ORF10 as Supplementary material (Supplementary material 1) since at present there is no experimental evidence available against or in favor of this being a real gene.
    </p>
    <p id="p-16">Compared to the essential genes S, M, N, E and ORF1ab, other genes such as ORF3a and ORF7a are poorly characterized, according to GenBank annotations (
     <a href="https://www.ncbi.nlm.nih.gov/genome/proteins/86693?genome_assembly_id=760344">https://www.ncbi.nlm.nih.gov/genome/proteins/86693?genome_assembly_id=760344</a>). Our bioinformatic protein structural analysis confirmed that SARS-CoV-2 ORF3a (cluster 12), also known as viroporin, and ORF7a (cluster 17), that is same as ORF8a
     <sup>
      <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>
     </sup> in SARS, retain the structural features observed in other SARS viruses, namely a multi-pass transmembrane domain and a cytoplasmic β-barrel or β-sandwich fold (ORF3a) and an N-terminal 
     <em>sec</em>-pathway signal peptide, cleaved after residue 15, an immunoglobulin-like β-sandwich fold stabilized by two cysteine di-sulfide bonds, and a C-terminal single-pass transmembrane helix (ORF7a; 
     <a id="xref-fig-7-1" class="xref-fig" href="#F7">Supplementary Figures 5</a>,
     <a id="xref-fig-8-1" class="xref-fig" href="#F8">6</a>). Hence these SARS-CoV-2 proteins are expected to act in the same way as they do in other SARS, namely as accessory proteins mostly localized in the endoplasmic reticulum-Golgi intermediate compartment, but also occurring on the cell membrane where they enhance viral pathogenicity and mortality through protein-protein interactions
     <sup>
      <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>-
      <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>
     </sup>.
    </p>
    <p id="p-17">The remaining two accessory gene clusters, ORF6, ORF8, present in SARS-CoV-2, SARS and bat-SL-CoVZC45 are functionally uncharacterized. All are very short polypeptides (61, 121 and 38 residues for ORF6 and 8, respectively) with a large percentage of hydrophobic residues (62%, 56%, respectively). None of the proteins have trans-membrane regions, but ORF8 has an N-terminal 
     <em>sec</em>-pathway signal peptide with a cleavage site after residues 15, suggesting that it is secreted into the extracellular space (
     <a id="xref-fig-7-2" class="xref-fig" href="#F7">Supplementary Figures 5</a>,
     <a id="xref-fig-8-2" class="xref-fig" href="#F8">6</a>). Following signal peptide cleavage, the ORF8 protein core is predicted to consist largely of β-strands and features 7 cysteines. We predict that this protein adopts a cysteine disulfide-bond stabilized β-sandwich structure similar to the soluble domain of ORF7a (
     <a id="xref-fig-7-3" class="xref-fig" href="#F7">Supplementary Figures 5</a>,
     <a id="xref-fig-8-3" class="xref-fig" href="#F8">6</a>), inferring that ORF8 also functions as ligand binding module. ORF6 consists of a long amphipathic helical region, followed by an acidic tail. Our deep-learning-based annotation (DeepGOPlus
     <sup>
      <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>
     </sup>) suggested that ORFs 6, 7, 8 are involved in the regulation of molecular functions, either in response to stimuli (ORFs 8), or in cellular component biogenesis and organization (ORF6) (
     <a id="xref-fig-4-1" class="xref-fig" href="#F4">Suppl. Figure 2</a>-
     <a id="xref-fig-6-1" class="xref-fig" href="#F6">4</a>). Collectively, our analysis suggested no major function or gene cluster to be unique to SARS-CoV-2. Moreover, our analysis is in agreement with ORFs 3a, 6, 7a, 8 and being accessory non-essential proteins, that would be inefficient targets for COVID-2019 therapy.
    </p>
    <p id="p-18">The S protein binds to the host receptor ACE2, and hence presents a prime target for preventing viral entry into host cells
     <sup>
      <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>
     </sup>. The S proteins from SARS-CoV-1 and SARS-CoV-2 are similar
     <sup>
      <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>
     </sup>, but structural differences in the receptor binding domain (RBD) preclude the use of SARS-CoV-1–specific neutralizing antibodies to inhibit SARS-CoV-2
     <sup>
      <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>
     </sup>. We therefore focused our attention on the protein E, which is highly similar in SARS-CoV. The E protein, also a viroporin
     <sup>
      <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>
     </sup>, was previously confirmed as a determinant of pathogenicity
     <sup>
      <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>,
      <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>
     </sup> in SARS-CoV-1. Protein E was used as target for SARS antivirals
     <sup>
      <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>
     </sup>, and studies using SARS-CoV with lacking or mutated protein E as vaccine candidates showed promising results
     <sup>
      <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>-
      <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>
     </sup>. Furthermore, the E gene is one of the key genes used in identification of SARS-CoV-2 through RT-PCR
     <sup>
      <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>,
      <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>
     </sup>.
    </p>
    <p id="p-19">The SARS-Cov-1 protein E has ion channel (IC) activity and features a postsynaptic density-95/discs large/zona occludens-1 (PDZ)-binding motif (PBM). Both IC and PBM were required for SARS-CoV to induce virulence in mice
     <sup>
      <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>
     </sup>. The protein E from SARS-CoV-2 differs from that of SARS-CoV-1 only by three substitutions and one deletion (
     <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A-B</a>). The PBM and IC are identical in all SARS E proteins in cluster 1, but distinct in orthologues from the other two clusters (
     <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B-D</a>). The substitutions and insertions are positioned in flexible cytoplasmic regions, where they are not expected to affect the protein structure, IC or function of the PBM (
     <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2E</a>).
    </p>
    <p id="p-20">Studies on SARS-CoV-1 demonstrated that the E protein uses the IC and PBM to trigger a cytokine storm that activates the inflammasome, leading to increased edema in lungs. Ultimately these events result in ARDS
     <sup>
      <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">4</a>-
      <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">6</a>
     </sup>, a leading cause of death in SARS-CoV-1 and SARS-CoV-2 infection
     <sup>
      <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">7</a>,
      <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>
     </sup>. However, the drugs Amantadine, Hexamethylene amiloride and also BIT225 (BIT225 is in clinical trials) completely block the IC activity of SARS-CoV-1
     <sup>
      <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>,
      <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>
     </sup> and restrict its reproduction, leading to better survival of the animal host. The PBM enables the E protein to interact with the cellular PDZ domains. Its association with the tandem PDZ domains of syntenin activates the MAP kinase p38, triggering the overexpression of inflammatory cytokines. Consequently, inhibition of p38 by SB203580 increases the survival of the host
     <sup>
      <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">5</a>
     </sup>.
    </p>
    <p id="p-21">The high level of conservation of the IC and PBM within clade 2, including SARS-CoV-1 and SARS-CoV-2, strongly suggests that the available inhibitors for the E protein of SARS-CoV-1 should also be active on SARS-CoV-2.</p>
   </div>
   <div class="section" id="sec-3">
    <h2 class="">Conclusion</h2>
    <p id="p-22">Therapies to combat the spread of SARS-CoV-2 and the lethality caused by the resulting COVID-2019 are currently focusing primarily on S, the viral spike protein
     <sup>
      <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">3</a>,
      <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>,
      <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>
     </sup>. However, despite high similarities between proteins S between SARS-CoV-1 and SARS-CoV-2 viruses, existing neutralizing antibodies are ineffective against SARS-CoV-2
     <sup>
      <a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">3</a>
     </sup>. Hence, new antibodies that bind specifically to the SARS-CoV-2 spike protein need to be developed, tested and approved for human use, which would be a time-consuming process.
    </p>
    <p id="p-23">Our pangenomic analysis suggests that the protein E of all SARS viruses preserves its critical motifs used for pathogenesis. Hence, we predict that the readily available and approved inhibitors (Amantadine, Hexamethylene amiloride and SB203580), that proved efficient for alleviating ARDS in SARS-CoV-1-infected animal models should also be effective against SARS-CoV-2. While not preventing future spreading of the virus, these inhibitors might reduce the mortality rate while effective vaccines are being developed.</p>
   </div>
   <div class="section" id="sec-4">
    <h2 class="">Methods</h2>
    <div id="sec-5" class="subsection">
     <h3>Collection of 
      <em>Betacoronavirus</em> strains
     </h3>
     <p id="p-24">The NCBI genome assemblies page was searched, on January 26, for taxonid of genus 
      <em>Betacoronavirus</em>, resulting in a list of 22 genomes, including four isolates of SARS-CoV-2. We downloaded in addition to this initial list two more assemblies one for a bat coronavirus (MG772933) and another for MERS (KF600630), see 
      <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>. For easy identification of genes and annotations from GenBank, we included unique locus_tag identifiers containing locus id and an index number. Metadata was collected to define user friendly strain names and to understand phylogenetic trees in the context of virus host, country, taxon id and data release date.
     </p>
    </div>
    <div id="sec-6" class="subsection">
     <h3>Creating a pangenome for 
      <em>Betacoronavirus</em>
     </h3>
     <p id="p-25">GenBank annotation and sequences were used as a starting point for this pangenomic analysis study. These GenBank annotations contain heterogeneous identifiers for genes so we included unique locus tags containing the strain accession number and gene index number for computational processing. In addition, we collected metadata for these coronaviruses for displaying on the interactive phylogenetic trees for analysis later on. Gene sequences were compared to each other for sequence similarity, followed by gene clustering. Phylogenetic analysis was carried out on the core gene clusters to produce a species tree and similarly for each gene cluster to produce gene trees. For this pangenome analysis we used panX
      <sup>
       <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">12</a>
      </sup> (Ding et al., 2018) that takes GenBank files as input and extracts gene coordinates, annotations from GenBank and sequences from these genomes on both nucleotide and protein levels. For interactive visualization of pangenome clusters, alignments and phylogenetic trees we used panX visualization module obtain from github, 
      <a href="https://github.com/neherlab/pan-genome-visualization/">https://github.com/neherlab/pan-genome-visualization/</a>. Other databases showing greater detail about many viruses, such as Virus Pathogen Resource (VIPR, 
      <a href="https://www.viprbrc.org">https://www.viprbrc.org</a>) and GSAID (
      <a href="https://www.gisaid.org">https://www.gisaid.org</a>) are very useful but these resources do not provide gene clustering or pan-genome analysis.
     </p>
    </div>
    <div id="sec-7" class="subsection">
     <h3>Re-annotation of proteins from 
      <em>Betacoronavirus</em> genomes using KAUST Metagenomic Analysis Platform (KMAP) annotation pipeline
     </h3>
     <p id="p-26">To annotate uncharacterized genes in 
      <em>Betacoronavirus</em>, we utilized all available protein sequences available from the clustering analysis as an input to our Automatic Annotation of Microbial Genomes (AAMG) pipeline
      <sup>
       <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>
      </sup>, available via KAUST Metagenomic Analysis Platform (
      <a href="http://www.cbrc.kaust.edu.sa/kmap">http://www.cbrc.kaust.edu.sa/kmap</a>). AAMG uses sequence-based BLAST to UniProtKB (
      <a href="http://www.uniprot.org">www.uniprot.org</a>), KEGG (
      <a href="http://www.kegg.jp">www.kegg.jp</a>) and also Protein Family (PFam) domain detection using InterPro (
      <a href="http://www.ebi.ac.uk/interpro">www.ebi.ac.uk/interpro</a>).
     </p>
    </div>
    <div id="sec-8" class="subsection">
     <h3>Prediction of function using DeepGOPlus</h3>
     <p id="p-27">We used all genes from 
      <em>Betacoronavirus</em> dataset in order to explore Gene Ontology predictions through our in-house DeepGOPlus 
      <sup>
       <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>
      </sup> tool that combines deep convolutional neural network (CNN) model with sequence similarity-based predictions to derive relevant functional classes from Gene Ontology alongside a confidence score. Results were included with a score 0.1 and filtered for class specificity. Predicted Gene Ontology (GO) terms with a confidence score from DeepGOPlus were visualized using QuickGO (
      <a href="https://www.ebi.ac.uk/QuickGO/">https://www.ebi.ac.uk/QuickGO/</a>) for SARS-CoV-2 specific gene clusters. DeepGOPlus is available on github, 
      <a href="https://github.com/bio-ontology-research-group/DeepGOPlus">https://github.com/bio-ontology-research-group/DeepGOPlus</a>
     </p>
    </div>
    <div id="sec-9" class="subsection">
     <h3>Structure based prediction of function</h3>
     <p id="p-28">ProtParam (
      <a href="https://web.expasy.org/cgi-bin/protparam/protparam">https://web.expasy.org/cgi-bin/protparam/protparam</a>) was used for calculating protein features. Phobius (
      <a href="http://phobius.sbc.su.se/">http://phobius.sbc.su.se/</a>) and SignalP-5.0 were used for prediction of transmembrane regions and signal peptides 
      <sup>
       <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>
      </sup>. Jpred4 was used to calculate secondary structure features (
      <a href="http://www.compbio.dundee.ac.uk/jpred4/">http://www.compbio.dundee.ac.uk/jpred4/</a>). 3D modelling was carried out using SwissModel (homology modelling)
      <sup>
       <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>
      </sup> or QUARK (ab initio structure predictions)
      <sup>
       <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>
      </sup>.
     </p>
    </div>
   </div>
   <div class="section" id="sec-10">
    <h2 class="">Author contributions</h2>
    <p id="p-29">IA, TG and CMD conceived and designed the research, IA led the study and conducted the data analysis, A.K. helped developed the web-based resource and led computational components of the study, MK and SA contributed to the functional structural analysis, and AP contributed with inferences on virulence. IA and CMD developed the first draft of the manuscript and all authors contributed to writing and improving the manuscript, and approved the submission.</p>
   </div>
   <div class="section" id="sec-11">
    <h2 class="">
     <em>Supplementary Information 1</em>, ORF10 gene in SARS-CoV-2
    </h2>
    <p id="p-31">A short gene predicted in in SARS-CoV-2, known as ORF10, appears unique only to SARS-CoV-2. A sequence comparison of this gene to DNA of all 
     <em>Betacoronavirus</em> genomes in NCBI shows matches in SARS and SARS-like genomes with 100% coverage, although no gene has been predicted in matching location of ORF10 in SARS or SARS-like genomes. This raises a question whether ORF10 is an artifact of annotation in SARS-CoV-2 where it has been predicted or an artifact in SARS and SARS-like genomes where it has not been predicted. There is no experimental data available to approve or disprove the reality of this gene. For completeness purposes we included ORF10 in our functional and structural analysis of SARS-CoV-2 but not included in the main text.
    </p>
    <p id="p-32">
     <a id="xref-fig-6-2" class="xref-fig" href="#F6">Supplementary Figure 4</a>-
     <a id="xref-fig-8-4" class="xref-fig" href="#F8">6</a> shows predicted Gene Ontology and structural information about ORF10 using deep learning Gene Ontology Prediction tool (DeepGOplus). It shows that, similar to ORF8, this gene regulates molecular functions and contributes to response to stimulus. It is predicted to be localized only in the extracellular region of the host cells. OF10 is predicted to harbor a long helix followed by a β-strand and appears to be localized only in the extracellular region of the host cells. It has been concluded that SARS-CoV-2 accessory genes, including ORF10, carry a helper function and do not serve as a major target for detection or therapy of COVID-2019.
    </p>
    <div id="F3" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="DeepGOPlus predicted Gene Ontology Graph for SARS-CoV-2 envelope protein, E. According to predictions, the protein can be localized in the host cell Golgi apparatus, vacuoles, endoplasmic reticulum, and membrane cell periphery of the host cell. It contributes to the process of disruption of the host cell and viral budding from Golgi membrane." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-542279032" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;DeepGOPlus predicted Gene Ontology Graph for SARS-CoV-2 envelope protein, E. According to predictions, the protein can be localized in the host cell Golgi apparatus, vacuoles, endoplasmic reticulum, and membrane cell periphery of the host cell. It contributes to the process of disruption of the host cell and viral budding from Golgi membrane.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F3.medium.gif" width="440" height="230"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Supplementary Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F3.medium.gif" width="440" height="230"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Supplementary Figure 1.</span>
      <p id="p-33" class="first-child">DeepGOPlus predicted Gene Ontology Graph for SARS-CoV-2 envelope protein, E. According to predictions, the protein can be localized in the host cell Golgi apparatus, vacuoles, endoplasmic reticulum, and membrane cell periphery of the host cell. It contributes to the process of disruption of the host cell and viral budding from Golgi membrane.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F4" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="DeepGOPlus predicted Gene Ontology Graph for SARS-CoV-2 ORF6. Predictions generally describe this protein to be localized in a membrane protein complex. It negatively regulates biological processes and contributes to cellular localization and biogenesis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-542279032" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;DeepGOPlus predicted Gene Ontology Graph for SARS-CoV-2 ORF6. Predictions generally describe this protein to be localized in a membrane protein complex. It negatively regulates biological processes and contributes to cellular localization and biogenesis.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F4.medium.gif" width="440" height="165"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Supplementary Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F4.medium.gif" width="440" height="165"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Supplementary Figure 2.</span>
      <p id="p-34" class="first-child">DeepGOPlus predicted Gene Ontology Graph for SARS-CoV-2 ORF6. Predictions generally describe this protein to be localized in a membrane protein complex. It negatively regulates biological processes and contributes to cellular localization and biogenesis.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F5" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="DeepGOPlus predicted Gene Ontology Graph for SARS-CoV-2 ORF8. This protein contributes to processes as response to stimulus along with biological regulation and potentially localized in vacuoles, cell periphery and extracellular regions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-542279032" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;DeepGOPlus predicted Gene Ontology Graph for SARS-CoV-2 ORF8. This protein contributes to processes as response to stimulus along with biological regulation and potentially localized in vacuoles, cell periphery and extracellular regions.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F5.medium.gif" width="440" height="252"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Supplementary Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F5.medium.gif" width="440" height="252"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Supplementary Figure 3.</span>
      <p id="p-35" class="first-child">DeepGOPlus predicted Gene Ontology Graph for SARS-CoV-2 ORF8. This protein contributes to processes as response to stimulus along with biological regulation and potentially localized in vacuoles, cell periphery and extracellular regions.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F6" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="DeepGOPlus Gene Ontology Graph for SARS-CoV-2 gene, ORF10. Similar to ORF8, this protein (if this predicted ORF is not an annotation artifact, see Figure 1) regulates molecular functions and contributes to response to stimulus. It is predicted to be localized only in the extracellular region of the host cells." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-542279032" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;DeepGOPlus Gene Ontology Graph for SARS-CoV-2 gene, ORF10. Similar to ORF8, this protein (if this predicted ORF is not an annotation artifact, see Figure 1) regulates molecular functions and contributes to response to stimulus. It is predicted to be localized only in the extracellular region of the host cells.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F6.medium.gif" width="440" height="149"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Supplementary Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F6.medium.gif" width="440" height="149"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Supplementary Figure 4.</span>
      <p id="p-36" class="first-child">DeepGOPlus Gene Ontology Graph for SARS-CoV-2 gene, ORF10. Similar to ORF8, this protein (if this predicted ORF is not an annotation artifact, see 
       <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>) regulates molecular functions and contributes to response to stimulus. It is predicted to be localized only in the extracellular region of the host cells.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F7" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F7.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Prediction of signal peptides and transmembrane regions. A) combined prediction of signal peptides and transmembrane regions, and (B) an additional analysis to identify the type of signal peptide and cleavage site for ORF7a and ORF10." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-542279032" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Prediction of signal peptides and transmembrane regions. A) combined prediction of signal peptides and transmembrane regions, and (B) an additional analysis to identify the type of signal peptide and cleavage site for ORF7a and ORF10.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F7.medium.gif" width="440" height="311"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Supplementary Figure 5" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F7.medium.gif" width="440" height="311"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Supplementary Figure 5</span>
      <p id="p-37" class="first-child">Prediction of signal peptides and transmembrane regions. A) combined prediction of signal peptides and transmembrane regions, and (B) an additional analysis to identify the type of signal peptide and cleavage site for ORF7a and ORF10.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F8" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F8.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Prediction of secondary structure characteristics and tertiary architecture (colour-ramped from N- (blue) to C-terminus (red)). E: homology model based on the pentameric SARS CoV E protein (PDB Id 5[[#xD7]]29, 26% sequence identity). Only one chain is colour-ramped. The membrane (pale pink) is indicated with cytoplasmic and luminal sides labelled. Note that the N-terminal 7 and C-terminal 15 residues are not included in the model. ORF3a and ORF7a: structural models for the soluble domains. The two di-sulphide bridges conserved in SARS ORF3a are shown in magenta (template: 1xak). 90 [[#xB0]] views of ab initio models are shown for ORF6 and ORF10. Hydrophobic and charged side chains are highlighted in ORF6, and hydrophobic side chains are highlighted in ORF10." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-542279032" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Prediction of secondary structure characteristics and tertiary architecture (colour-ramped from N- (blue) to C-terminus (red)). E: homology model based on the pentameric SARS CoV E protein (PDB Id 5[[#xD7]]29, 26% sequence identity). Only one chain is colour-ramped. The membrane (pale pink) is indicated with cytoplasmic and luminal sides labelled. Note that the N-terminal 7 and C-terminal 15 residues are not included in the model. ORF3a and ORF7a: structural models for the soluble domains. The two di-sulphide bridges conserved in SARS ORF3a are shown in magenta (template: 1xak). 90 [[#xB0]] views of &lt;;em&gt;;ab initio&lt;;/em&gt;; models are shown for ORF6 and ORF10. Hydrophobic and charged side chains are highlighted in ORF6, and hydrophobic side chains are highlighted in ORF10.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F8.medium.gif" width="407" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Supplementary Figure 6" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F8.medium.gif" width="407" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Supplementary Figure 6</span>
      <p id="p-38" class="first-child">Prediction of secondary structure characteristics and tertiary architecture (colour-ramped from N- (blue) to C-terminus (red)). E: homology model based on the pentameric SARS CoV E protein (PDB Id 5×29, 26% sequence identity). Only one chain is colour-ramped. The membrane (pale pink) is indicated with cytoplasmic and luminal sides labelled. Note that the N-terminal 7 and C-terminal 15 residues are not included in the model. ORF3a and ORF7a: structural models for the soluble domains. The two di-sulphide bridges conserved in SARS ORF3a are shown in magenta (template: 1xak). 90 ° views of 
       <em>ab initio</em> models are shown for ORF6 and ORF10. Hydrophobic and charged side chains are highlighted in ORF6, and hydrophobic side chains are highlighted in ORF10.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F9" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F9.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Alignment of SARS-CoV-2 E protein comparison to hits available in NCBI[[#x2019]]s NR database. The first two appear from SARS-CoV-2 and rest of the hits appear from SARS and SARS-like Betacoronaviruses. This result shows SARS-CoV-2 E protein is conserved only in SARS clade and E from MERS or other animal coronaviruses clusters in different clusters (see Figure 2)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-542279032" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Alignment of SARS-CoV-2 E protein comparison to hits available in NCBI[[#x2019]]s NR database. The first two appear from SARS-CoV-2 and rest of the hits appear from SARS and SARS-like &lt;;em&gt;;Betacoronaviruses&lt;;/em&gt;;. This result shows SARS-CoV-2 E protein is conserved only in SARS clade and E from MERS or other animal coronaviruses clusters in different clusters (see Figure 2).&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F9.medium.gif" width="440" height="321"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Supplementary Figure 7." src="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F9.medium.gif" width="440" height="321"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 7." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/06/2020.02.17.952895/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Supplementary Figure 7.</span>
      <p id="p-39" class="first-child">Alignment of SARS-CoV-2 E protein comparison to hits available in NCBI’s NR database. The first two appear from SARS-CoV-2 and rest of the hits appear from SARS and SARS-like 
       <em>Betacoronaviruses</em>. This result shows SARS-CoV-2 E protein is conserved only in SARS clade and E from MERS or other animal coronaviruses clusters in different clusters (see 
       <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>).
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgements</h2>
    <p id="p-30">We are thankful to KAUST Supercomputing Laboratory (KSL), KAUST Information Technology (IT) and Muhammad Saif from CBRC, KAUST for providing support in computing resources.</p>
   </div>
   <div class="section fn-group" id="fn-group-1">
    <h2>Footnotes</h2>
    <ul>
     <li class="fn-dataset fn-group-external-links" id="fn-1">
      <p id="p-3">
       <a href="http://pangenomedb.cbrc.kaust.edu.sa">http://pangenomedb.cbrc.kaust.edu.sa</a>
      </p>
     </li>
    </ul>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list">
     <li>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1 in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.17.952895v2.1" data-doi="10.1146/annurev-virology-110615-042301">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>
         <span class="cit-article-title">Structure, Function, and Evolution of Coronavirus Spike Proteins</span>. 
         <abbr class="cit-jnl-abbrev">Annu Rev Virol</abbr>
         <span class="cit-vol">3</span>, 
         <span class="cit-fpage">237</span>–
         <span class="cit-lpage">261</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1146/annurev-virology-110615-042301</span> (
         <span class="cit-pub-date">2016</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAnnu%2BRev%2BVirol%26rft.volume%253D3%26rft.spage%253D237%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-virology-110615-042301%26rft_id%253Dinfo%253Apmid%252F27578435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1146/annurev-virology-110615-042301&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=27578435&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F06%2F2020.02.17.952895.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2 in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.17.952895v2.2" data-doi="10.1128/JVI.00127-20">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wan</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shang</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Graham</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Baric</span>, 
          <span class="cit-name-given-names">R. S.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>
         <span class="cit-article-title">Receptor recognition by novel coronavirus from Wuhan: An analysis based on decade-long structural studies of SARS</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.00127-20</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00127-20%26rft_id%253Dinfo%253Apmid%252F31996437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjE0OiJKVkkuMDAxMjctMjB2MSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzA2LzIwMjAuMDIuMTcuOTUyODk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3 in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.17.952895v2.3" data-doi="10.1126/science.abb2507">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wrapp</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1126/science.abb2507</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.abb2507%26rft_id%253Dinfo%253Apmid%252F32075877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE3OiJzY2llbmNlLmFiYjI1MDd2MiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzA2LzIwMjAuMDIuMTcuOTUyODk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4 in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.17.952895v2.4" data-doi="10.1371/journal.ppat.1004077">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Nieto-Torres</span>, 
          <span class="cit-name-given-names">J. L.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Severe acute respiratory syndrome coronavirus envelope protein ion channel activity promotes virus fitness and pathogenesis</span>. 
         <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>
         <span class="cit-vol">10</span>, 
         <span class="cit-fpage">e1004077</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1371/journal.ppat.1004077</span> (
         <span class="cit-pub-date">2014</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D10%26rft.spage%253De1004077%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1004077%26rft_id%253Dinfo%253Apmid%252F24788150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1004077&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24788150&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F06%2F2020.02.17.952895.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5 in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.17.952895v2.5" data-doi="10.1371/journal.ppat.1004320">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jimenez-Guardeno</span>, 
          <span class="cit-name-given-names">J. M.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">The PDZ-binding motif of severe acute respiratory syndrome coronavirus envelope protein is a determinant of viral pathogenesis</span>. 
         <abbr class="cit-jnl-abbrev">PLoS Pathog</abbr>
         <span class="cit-vol">10</span>, 
         <span class="cit-fpage">e1004320</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1371/journal.ppat.1004320</span> (
         <span class="cit-pub-date">2014</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D10%26rft.spage%253De1004320%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1004320%26rft_id%253Dinfo%253Apmid%252F25122212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1004320&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25122212&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F06%2F2020.02.17.952895.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6 in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.17.952895v2.6" data-doi="10.3390/v7062750">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Torres</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Surya</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>, 
          <span class="cit-name-given-names">D. X.</span>
         </span>
         <span class="cit-article-title">Protein-Protein Interactions of Viroporins in Coronaviruses and Paramyxoviruses: New Targets for Antivirals?</span>
         <abbr class="cit-jnl-abbrev">Viruses</abbr>
         <span class="cit-vol">7</span>, 
         <span class="cit-fpage">2858</span>–
         <span class="cit-lpage">2883</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.3390/v7062750</span> (
         <span class="cit-pub-date">2015</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D7%26rft.spage%253D2858%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv7062750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.3390/v7062750&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7 in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.17.952895v2.7" data-doi="10.1016/S0140-6736(20)30183-5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</span>. 
         <abbr class="cit-jnl-abbrev">Lancet</abbr>
         <span class="cit-vol">395</span>, 
         <span class="cit-fpage">497</span>–
         <span class="cit-lpage">506</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1016/S0140-6736(20)30183-5</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DLancet%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252820%252930183-5%26rft_id%253Dinfo%253Apmid%252F31986264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0140-6736(20)30183-5&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=31986264&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F06%2F2020.02.17.952895.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8 in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.17.952895v2.8" data-doi="10.1016/S2213-2600(20)30076-X">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Pathological findings of COVID-19 associated with acute respiratory distress syndrome</span>. 
         <abbr class="cit-jnl-abbrev">Lancet Respir Med</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1016/S2213-2600(20)30076-X</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DLancet%2BRespir%2BMed%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS2213-2600%252820%252930076-X%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S2213-2600(20)30076-X&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9 in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.17.952895v2.9" data-doi="10.1016/j.virol.2006.05.028">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wilson</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gage</span>, 
          <span class="cit-name-given-names">P.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Ewart</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>
         <span class="cit-article-title">Hexamethylene amiloride blocks E protein ion channels and inhibits coronavirus replication</span>. 
         <abbr class="cit-jnl-abbrev">Virology</abbr>
         <span class="cit-vol">353</span>, 
         <span class="cit-fpage">294</span>–
         <span class="cit-lpage">306</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1016/j.virol.2006.05.028</span> (
         <span class="cit-pub-date">2006</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DWilson%26rft.auinit1%253DL.%26rft.volume%253D353%26rft.issue%253D2%26rft.spage%253D294%26rft.epage%253D306%26rft.atitle%253DHexamethylene%2Bamiloride%2Bblocks%2BE%2Bprotein%2Bion%2Bchannels%2Band%2Binhibits%2Bcoronavirus%2Breplication.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2006.05.028%26rft_id%253Dinfo%253Apmid%252F16815524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.virol.2006.05.028&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16815524&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F06%2F2020.02.17.952895.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10 in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.17.952895v2.10" data-doi="10.1016/j.bpc.2017.11.002">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Behmard</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Abdolmaleki</span>, 
          <span class="cit-name-given-names">P.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Taghdir</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>
         <span class="cit-article-title">Understanding the inhibitory mechanism of BIT225 drug against p7 viroporin using computational study</span>. 
         <abbr class="cit-jnl-abbrev">Biophys Chem</abbr>
         <span class="cit-vol">233</span>, 
         <span class="cit-fpage">47</span>–
         <span class="cit-lpage">54</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1016/j.bpc.2017.11.002</span> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBiophys%2BChem%26rft.volume%253D233%26rft.spage%253D47%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bpc.2017.11.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.bpc.2017.11.002&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11 in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.17.952895v2.11" data-doi="10.1016/j.chom.2020.02.001">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Genome Composition and Divergence of the Novel Coronavirus (2019-nCoV) Originating in China</span>. 
         <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1016/j.chom.2020.02.001</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2BMicrobe%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2020.02.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.chom.2020.02.001&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12 in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.17.952895v2.12" data-doi="10.1093/nar/gkx977">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ding</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Baumdicker</span>, 
          <span class="cit-name-given-names">F.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Neher</span>, 
          <span class="cit-name-given-names">R. A.</span>
         </span>
         <span class="cit-article-title">panX: pan-genome analysis and exploration</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>
         <span class="cit-vol">46</span>, 
         <span class="cit-fpage">e5</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gkx977</span> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D46%26rft.spage%253De5%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx977%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkx977&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13 in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.17.952895v2.13" data-doi="10.1016/S0065-3527(06)66005-3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Masters</span>, 
          <span class="cit-name-given-names">P. S.</span>
         </span>
         <span class="cit-article-title">The molecular biology of coronaviruses</span>. 
         <abbr class="cit-jnl-abbrev">Adv Virus Res</abbr>
         <span class="cit-vol">66</span>, 
         <span class="cit-fpage">193</span>–
         <span class="cit-lpage">292</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1016/S0065-3527(06)66005-3</span> (
         <span class="cit-pub-date">2006</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bvirus%2Bresearch%26rft.stitle%253DAdv%2BVirus%2BRes%26rft.aulast%253DMasters%26rft.auinit1%253DP.%2BS.%26rft.volume%253D66%26rft.spage%253D193%26rft.epage%253D292%26rft.atitle%253DThe%2Bmolecular%2Bbiology%2Bof%2Bcoronaviruses.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0065-3527%252806%252966005-3%26rft_id%253Dinfo%253Apmid%252F16877062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0065-3527(06)66005-3&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16877062&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F06%2F2020.02.17.952895.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000240137400005&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14 in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.17.952895v2.14" data-doi="10.1128/mBio.02325-17">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Castano-Rodriguez</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Role of Severe Acute Respiratory Syndrome Coronavirus Viroporins E, 3a, and 8a in Replication and Pathogenesis</span>. 
         <abbr class="cit-jnl-abbrev">mBio</abbr>
         <span class="cit-vol">9</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1128/mBio.02325-17</span> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DmBio%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.02325-17%26rft_id%253Dinfo%253Apmid%252F29789363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibWJpbyI7czo1OiJyZXNpZCI7czoxMzoiOS8zL2UwMjMyNS0xNyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzA2LzIwMjAuMDIuMTcuOTUyODk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15 in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.17.952895v2.15" data-doi="10.1371/journal.pone.0008342">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Minakshi</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">The SARS Coronavirus 3a protein causes endoplasmic reticulum stress and induces ligand-independent downregulation of the type 1 interferon receptor</span>. 
         <abbr class="cit-jnl-abbrev">PLoS One</abbr>
         <span class="cit-vol">4</span>, 
         <span class="cit-fpage">e8342</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0008342</span> (
         <span class="cit-pub-date">2009</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DMinakshi%26rft.auinit1%253DR.%26rft.volume%253D4%26rft.issue%253D12%26rft.spage%253De8342%26rft.epage%253De8342%26rft.atitle%253DThe%2BSARS%2BCoronavirus%2B3a%2Bprotein%2Bcauses%2Bendoplasmic%2Breticulum%2Bstress%2Band%2Binduces%2Bligand-independent%2Bdownregulation%2Bof%2Bthe%2Btype%2B1%2Binterferon%2Breceptor.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0008342%26rft_id%253Dinfo%253Apmid%252F20020050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0008342&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20020050&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F06%2F2020.02.17.952895.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.17.952895v2.16" data-doi="10.1007/s11373-005-9043-9">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hanel</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Stangler</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Stoldt</span>, 
          <span class="cit-name-given-names">M.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Willbold</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>
         <span class="cit-article-title">Solution structure of the X4 protein coded by the SARS related coronavirus reveals an immunoglobulin like fold and suggests a binding activity to integrin I domains</span>. 
         <abbr class="cit-jnl-abbrev">J Biomed Sci</abbr>
         <span class="cit-vol">13</span>, 
         <span class="cit-fpage">281</span>–
         <span class="cit-lpage">293</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1007/s11373-005-9043-9</span> (
         <span class="cit-pub-date">2006</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bbiomedical%2Bscience%26rft.stitle%253DJ%2BBiomed%2BSci%26rft.aulast%253DHanel%26rft.auinit1%253DK.%26rft.volume%253D13%26rft.issue%253D3%26rft.spage%253D281%26rft.epage%253D293%26rft.atitle%253DSolution%2Bstructure%2Bof%2Bthe%2BX4%2Bprotein%2Bcoded%2Bby%2Bthe%2BSARS%2Brelated%2Bcoronavirus%2Breveals%2Ban%2Bimmunoglobulin%2Blike%2Bfold%2Band%2Bsuggests%2Ba%2Bbinding%2Bactivity%2Bto%2Bintegrin%2BI%2Bdomains.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11373-005-9043-9%26rft_id%253Dinfo%253Apmid%252F16328780%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1007/s11373-005-9043-9&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16328780&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F06%2F2020.02.17.952895.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17 in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.17.952895v2.17" data-doi="10.1128/JVI.02576-13">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">DeDiego</span>, 
          <span class="cit-name-given-names">M. L.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Inhibition of NF-kappaB-mediated inflammation in severe acute respiratory syndrome coronavirus-infected mice increases survival</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>
         <span class="cit-vol">88</span>, 
         <span class="cit-fpage">913</span>–
         <span class="cit-lpage">924</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.02576-13</span> (
         <span class="cit-pub-date">2014</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02576-13%26rft_id%253Dinfo%253Apmid%252F24198408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6Ijg4LzIvOTEzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMDYvMjAyMC4wMi4xNy45NTI4OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18 in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.17.952895v2.18" data-doi="10.1093/bioinformatics/btz595">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kulmanov</span>, 
          <span class="cit-name-given-names">M.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Hoehndorf</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>
         <span class="cit-article-title">DeepGOPlus: improved protein function prediction from sequence</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">36</span>, 
         <span class="cit-fpage">422</span>–
         <span class="cit-lpage">429</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btz595</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D36%26rft.spage%253D422%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtz595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btz595&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19 in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.17.952895v2.19" data-doi="10.1007/978-0-387-33012-9_34">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Liao</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tam</span>, 
          <span class="cit-name-given-names">J. P.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>, 
          <span class="cit-name-given-names">D. X.</span>
         </span>
         <span class="cit-article-title">Viroporin activity of SARS-CoV E protein</span>. 
         <abbr class="cit-jnl-abbrev">Adv Exp Med Biol</abbr>
         <span class="cit-vol">581</span>, 
         <span class="cit-fpage">199</span>–
         <span class="cit-lpage">202</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1007/978-0-387-33012-9_34</span> (
         <span class="cit-pub-date">2006</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bexperimental%2Bmedicine%2Band%2Bbiology%26rft.stitle%253DAdv%2BExp%2BMed%2BBiol%26rft.aulast%253DLiao%26rft.auinit1%253DY.%26rft.volume%253D581%26rft.spage%253D199%26rft.epage%253D202%26rft.atitle%253DViroporin%2Bactivity%2Bof%2BSARS-CoV%2BE%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-0-387-33012-9_34%26rft_id%253Dinfo%253Apmid%252F17037530%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1007/978-0-387-33012-9_34&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17037530&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F06%2F2020.02.17.952895.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20 in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.17.952895v2.20" data-doi="10.1128/JVI.01467-06">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">DeDiego</span>, 
          <span class="cit-name-given-names">M. L.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A severe acute respiratory syndrome coronavirus that lacks the E gene is attenuated in vitro and in vivo</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Virology</abbr>
         <span class="cit-vol">81</span>, 
         <span class="cit-fpage">1701</span>–
         <span class="cit-lpage">1713</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1128/Jvi.01467-06</span> (
         <span class="cit-pub-date">2007</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01467-06%26rft_id%253Dinfo%253Apmid%252F17108030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgxLzQvMTcwMSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzA2LzIwMjAuMDIuMTcuOTUyODk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.17.952895v2.21" data-doi="10.1128/JVI.00304-08">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lamirande</span>, 
          <span class="cit-name-given-names">E. W.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A live attenuated severe acute respiratory syndrome coronavirus is immunogenic and efficacious in golden Syrian hamsters</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>
         <span class="cit-vol">82</span>, 
         <span class="cit-fpage">7721</span>–
         <span class="cit-lpage">7724</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.00304-08</span> (
         <span class="cit-pub-date">2008</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00304-08%26rft_id%253Dinfo%253Apmid%252F18463152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Mi8xNS83NzIxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMDYvMjAyMC4wMi4xNy45NTI4OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.17.952895v2.22" data-doi="10.1128/JVI.03566-14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Regla-Nava</span>, 
          <span class="cit-name-given-names">J. A.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Severe acute respiratory syndrome coronaviruses with mutations in the E protein are attenuated and promising vaccine candidates</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>
         <span class="cit-vol">89</span>, 
         <span class="cit-fpage">3870</span>–
         <span class="cit-lpage">3887</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1128/JVI.03566-14</span> (
         <span class="cit-pub-date">2015</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.03566-14%26rft_id%253Dinfo%253Apmid%252F25609816%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg5LzcvMzg3MCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzA2LzIwMjAuMDIuMTcuOTUyODk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23 in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.17.952895v2.23" data-doi="10.1186/s12985-019-1182-0">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Schoeman</span>, 
          <span class="cit-name-given-names">D.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Fielding</span>, 
          <span class="cit-name-given-names">B. C.</span>
         </span>
         <span class="cit-article-title">Coronavirus envelope protein: current knowledge</span>. 
         <abbr class="cit-jnl-abbrev">Virol J</abbr>
         <span class="cit-vol">16</span>, 
         <span class="cit-fpage">69</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1186/s12985-019-1182-0</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DVirol%2BJ%26rft.volume%253D16%26rft.spage%253D69%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12985-019-1182-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/s12985-019-1182-0&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24 in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.17.952895v2.24" data-doi="10.2807/1560-7917.ES.2020.25.3.2000045">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Corman</span>, 
          <span class="cit-name-given-names">V. M.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR</span>. 
         <abbr class="cit-jnl-abbrev">Euro Surveill</abbr>
         <span class="cit-vol">25</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.2807/1560-7917.ES.2020.25.3.2000045</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEuro%2BSurveill%26rft_id%253Dinfo%253Adoi%252F10.2807%252F1560-7917.ES.2020.25.3.2000045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.2807/1560-7917.ES.2020.25.3.2000045&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25 in text" id="ref-25">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.17.952895v2.25" data-doi="10.12688/f1000research.22211.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kruse</span>, 
          <span class="cit-name-given-names">R. L.</span>
         </span>
         <span class="cit-article-title">Therapeutic strategies in an outbreak scenario to treat the novel coronavirus originating in Wuhan, China</span>. 
         <abbr class="cit-jnl-abbrev">F1000Res</abbr>
         <span class="cit-vol">9</span>, 
         <span class="cit-fpage">72</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.12688/f1000research.22211.2</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DF1000Res%26rft.volume%253D9%26rft.spage%253D72%26rft_id%253Dinfo%253Adoi%252F10.12688%252Ff1000research.22211.2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.12688/f1000research.22211.2&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26 in text" id="ref-26">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.17.952895v2.26" data-doi="10.1002/cbic.202000047">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Morse</span>, 
          <span class="cit-name-given-names">J. S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lalonde</span>, 
          <span class="cit-name-given-names">T.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>
         <span class="cit-article-title">Learning from the Past: Possible Urgent Prevention and Treatment Options for Severe Acute Respiratory Infections Caused by 2019-nCoV</span>. 
         <abbr class="cit-jnl-abbrev">Chembiochem</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1002/cbic.202000047</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DChembiochem%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fcbic.202000047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1002/cbic.202000047&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27 in text" id="ref-27">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.17.952895v2.27" data-doi="10.1371/journal.pone.0082210">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Alam</span>, 
          <span class="cit-name-given-names">I.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">INDIGO - INtegrated data warehouse of microbial genomes with examples from the red sea extremophiles</span>. 
         <abbr class="cit-jnl-abbrev">PLoS One</abbr>
         <span class="cit-vol">8</span>, 
         <span class="cit-fpage">e82210</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0082210</span> (
         <span class="cit-pub-date">2013</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De82210%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0082210%26rft_id%253Dinfo%253Apmid%252F24324765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0082210&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24324765&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F06%2F2020.02.17.952895.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28 in text" id="ref-28">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.17.952895v2.28" data-doi="10.1038/s41587-019-0036-z">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Almagro Armenteros</span>, 
          <span class="cit-name-given-names">J. J.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">SignalP 5.0 improves signal peptide predictions using deep neural networks</span>. 
         <abbr class="cit-jnl-abbrev">Nat Biotechnol</abbr>
         <span class="cit-vol">37</span>, 
         <span class="cit-fpage">420</span>–
         <span class="cit-lpage">423</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1038/s41587-019-0036-z</span> (
         <span class="cit-pub-date">2019</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BBiotechnol%26rft.volume%253D37%26rft.spage%253D420%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41587-019-0036-z%26rft_id%253Dinfo%253Apmid%252F30778233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41587-019-0036-z&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=30778233&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F06%2F2020.02.17.952895.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29 in text" id="ref-29">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.17.952895v2.29" data-doi="10.1093/nar/gky427">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Waterhouse</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">SWISS-MODEL: homology modelling of protein structures and complexes</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>
         <span class="cit-vol">46</span>, 
         <span class="cit-fpage">W296</span>–
         <span class="cit-lpage">W303</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1093/nar/gky427</span> (
         <span class="cit-pub-date">2018</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D46%26rft.spage%253DW296%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky427%26rft_id%253Dinfo%253Apmid%252F29788355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gky427&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=29788355&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F06%2F2020.02.17.952895.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30 in text" id="ref-30">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.17.952895v2.30" data-doi="10.1002/prot.24065">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>, 
          <span class="cit-name-given-names">D.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>
         <span class="cit-article-title">Ab initio protein structure assembly using continuous structure fragments and optimized knowledge-based force field</span>. 
         <abbr class="cit-jnl-abbrev">Proteins</abbr>
         <span class="cit-vol">80</span>, 
         <span class="cit-fpage">1715</span>–
         <span class="cit-lpage">1735</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi: </span>
         <span class="cit-pub-id cit-pub-id-doi">10.1002/prot.24065</span> (
         <span class="cit-pub-date">2012</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProteins%26rft.stitle%253DProteins%26rft.aulast%253DXu%26rft.auinit1%253DD.%26rft.volume%253D80%26rft.issue%253D7%26rft.spage%253D1715%26rft.epage%253D1735%26rft.atitle%253DAb%2Binitio%2Bprotein%2Bstructure%2Bassembly%2Busing%2Bcontinuous%2Bstructure%2Bfragments%2Band%2Boptimized%2Bknowledge-based%2Bforce%2Bfield.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fprot.24065%26rft_id%253Dinfo%253Apmid%252F22411565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1002/prot.24065&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22411565&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F06%2F2020.02.17.952895.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000304866000001&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
