<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146/embed/graphic-2.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/04/06/2020.04.05.026146.atom" />
<meta name="HW.pisa" content="biorxiv;2020.04.05.026146v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="CovProfile: profiling the viral genome and gene expressions of SARS-COV2" />
<meta name="DC.Identifier" content="10.1101/2020.04.05.026146" />
<meta name="DC.Date" content="2020-04-06" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="SARS-COV2 is infecting more than one million people. Knowing its genome and gene expressions is essential to understand the virus. Here, we propose a computational tool to detect the viral genomic variations as well as viral gene expressions from the sequenced amplicons with Nanopore devices of the multiplex reverse transcription polymerase chain reactions. We applied our tool to 11 samples of terminally ill patients, and discovered that seven of the samples are infected by two viral strains. The expression of viral genes ORF1ab gene, S gene, M gene, and N gene are high among most of the samples. While performing our tests, we noticed the abundances of MUC5B transcript segments to be consistently higher than those of other genes in the host." />
<meta name="DC.Contributor" content="Yonghan Yu" />
<meta name="DC.Contributor" content="Zhengtu Li" />
<meta name="DC.Contributor" content="Yinhu Li" />
<meta name="DC.Contributor" content="Le Yu" />
<meta name="DC.Contributor" content="Wenlong Jia" />
<meta name="DC.Contributor" content="Feng Ye" />
<meta name="DC.Contributor" content="Shuai Cheng Li" />
<meta name="article:published_time" content="2020-04-06" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="CovProfile: profiling the viral genome and gene expressions of SARS-COV2" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;SARS-COV2 is infecting more than one million people. Knowing its genome and gene expressions is essential to understand the virus. Here, we propose a computational tool to detect the viral genomic variations as well as viral gene expressions from the sequenced amplicons with Nanopore devices of the multiplex reverse transcription polymerase chain reactions. We applied our tool to 11 samples of terminally ill patients, and discovered that seven of the samples are infected by two viral strains. The expression of viral genes ORF1ab gene, S gene, M gene, and N gene are high among most of the samples. While performing our tests, we noticed the abundances of MUC5B transcript segments to be consistently higher than those of other genes in the host.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.04.05.026146v1" />
<meta name="citation_id" content="2020.04.05.026146v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.04.05.026146v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.05.026146v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.05.026146v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.04.05.026146" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.04.05.026146" />
<meta name="citation_author" content="Yonghan Yu" />
<meta name="citation_author_institution" content="City University of Hong Kong Shenzhen Research Institute" />
<meta name="citation_author" content="Zhengtu Li" />
<meta name="citation_author_institution" content="State Key Laboratory of Respiratory Disease, National Clinical Research Center for Respiratory Disease, Guangzhou Institute of Respiratory Health, the First Affiliated Hospital of Guangzhou Medical University" />
<meta name="citation_author" content="Yinhu Li" />
<meta name="citation_author_institution" content="City University of Hong Kong Shenzhen Research Institute" />
<meta name="citation_author" content="Le Yu" />
<meta name="citation_author_institution" content="Beijing YuanShengKangTai (ProtoDNA) Genetech Co. Ltd" />
<meta name="citation_author" content="Wenlong Jia" />
<meta name="citation_author_institution" content="City University of Hong Kong Shenzhen Research Institute" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7136-9919" />
<meta name="citation_author" content="Feng Ye" />
<meta name="citation_author_institution" content="State Key Laboratory of Respiratory Disease, National Clinical Research Center for Respiratory Disease, Guangzhou Institute of Respiratory Health, the First Affiliated Hospital of Guangzhou Medical University" />
<meta name="citation_author_email" content="yefeng@gird.cn" />
<meta name="citation_author_email" content="shuaicli@gmail.com" />
<meta name="citation_author" content="Shuai Cheng Li" />
<meta name="citation_author_institution" content="City University of Hong Kong Shenzhen Research Institute" />
<meta name="citation_author_email" content="yefeng@gird.cn" />
<meta name="citation_author_email" content="shuaicli@gmail.com" />
<meta name="citation_reference" content="Novel coronavirus (2019-ncov) situation reports, https://www.who.int/emergencies/diseases/novelcoronavirus-2019/situation-reports, 2020." />
<meta name="citation_reference" content="A. Assiri, G. R. Abedi, M. A. Masri, A. B. Saeed, S. I. Gerber, and J. T. Watson. Middle east respiratory syndrome coronavirus infection during pregnancy: A report of 5 cases from saudi arabia: Table 1. Clinical Infectious Diseases, 63(7):951–953, June 2016." />
<meta name="citation_reference" content="M. T. Bolisetty, G. Rajadinakaran, and B. R. Graveley. Determining exon connectivity in complex mrnas by nanopore sequencing. Genome Biology, 16(1):204, 2015." />
<meta name="citation_reference" content="E. de Wit, N. van Doremalen, D. Falzarano, and V. J. Munster. Sars and mers: recent insights into emerging coronaviruses. Nature Reviews Microbiology, 14(8):523, 2016." />
<meta name="citation_reference" content="D. Gkika, L. Lemonnier, G. Shapovalov, D. Gordienko, C. Poux, M. Bernardini, A. Bokhobza, G. Bidaux, C. Degerny, K. Verreman, et al. Trp channel–associated factors are a novel protein family that regulates trpm8 trafficking and activityidentification of trpm8 partner proteins. The Journal of Cell Biology, 208(1):89–107, 2015." />
<meta name="citation_reference" content="A. Gorbalenya, S. Baker, R. Baric, et al. The species severe acute respiratory syndrome-related coronavirus: classifying 2019-ncov and naming it sars-cov-2. Nature Microbiology, 2020." />
<meta name="citation_reference" content="P. H. Guzzi, D. Mercatelli, C. Ceraolo, and F. M. Giorgi. Master regulator analysis of the sars-cov-2/human interactome. bioRxiv, 2020." />
<meta name="citation_reference" content="L. A. Hancock, C. E. Hennessy, G. M. Solomon, E. Dobrinskikh, A. Estrella, N. Hara, D. B. Hill, W. J. Kissner, M. R. Markovetz, D. E. G. Villalon, et al. Muc5b overexpression causes mucociliary dysfunction and enhances lung fibrosis in mice. Nature communications, 9(1):1–10, 2018." />
<meta name="citation_reference" content="M. Hoffmann, H. Kleine-Weber, S. Schroeder, N. Kruger, T. Herrler, S. Erichsen, T. S. Schiergens, G. Herrler, N.-H. Wu, A. Nitsche, et al. Sars-cov-2 cell entry depends on ace2 and tmprss2 and is blocked by a clinically proven protease inhibitor. Cell, 2020." />
<meta name="citation_reference" content="W. J. Kent. BLAT—the BLAST-like alignment tool. Genome Research, 12(4):656–664, Mar. 2002." />
<meta name="citation_reference" content="H. Li. Minimap2: pairwise alignment for nucleotide sequences. Bioinformatics, 34(18):3094–3100, 05 2018." />
<meta name="citation_reference" content="C. Liu, Q. Zhou, Y. Li, L. V. Garner, S. P. Watkins, L. J. Carter, J. Smoot, A. C. Gregg, A. D. Daniels, S. Jervey, et al. Research and development on therapeutic agents and vaccines for covid-19 and related human coronavirus diseases, 2020." />
<meta name="citation_reference" content="S. K. Mathai, S. Humphries, J. A. Kropski, T. S. Blackwell, J. Powers, A. D. Walts, C. Markin, J. Woodward, J. H. Chung, K. K. Brown, et al. Muc5b variant is associated with visually and quantitatively detected preclinical pulmonary fibrosis. Thorax, 74(12):1131–1139, 2019." />
<meta name="citation_reference" content="S. C. Moore, R. Penrice-Randal, M. Alruwaili, X. Dong, S. T. Pullan, D. Carter, K. Bewley, Q. Zhao, Y. Sun, C. Hartley, et al. Amplicon based minion sequencing of sars-cov-2 and metagenomic characterisation of nasopharyngeal swabs from patients with covid-19. medRxiv, 2020." />
<meta name="citation_reference" content="T. Phan. Genetic diversity and evolution of SARS-CoV-2. Infection, Genetics and Evolution, 81:104260, July 2020." />
<meta name="citation_reference" content="M. Prachar, S. Justesen, D. B. Steen-Jensen, S. P. Thorgrimsen, E. Jurgons, O. Winther, and F. O. Bagger. Covid-19 vaccine candidates: Prediction and validation of 174 sars-cov-2 epitopes. bioRxiv, 2020." />
<meta name="citation_reference" content="J. Quick, N. D. Grubaugh, S. T. Pullan, I. M. Claro, A. D. Smith, K. Gangavarapu, G. Oliveira, R. Robles-Sikisaka, T. F. Rogers, N. A. Beutler, et al. Multiplex pcr method for minion and illumina sequencing of zika and other virus genomes directly from clinical samples. nature protocols, 12(6):1261, 2017." />
<meta name="citation_reference" content="M. G. Roy, A. Livraghi-Butrico, A. A. Fletcher, M. M. McElwee, S. E. Evans, R. M. Boerner, S. N. Alexander, L. K. Bellinghausen, A. S. Song, Y. M. Petrova, et al. Muc5b is required for airway defence. Nature, 505(7483):412–416, 2014." />
<meta name="citation_reference" content="T. P. Sheahan, A. C. Sims, S. R. Leist, A. Schafer, J. Won, A. J. Brown, S. A. Montgomery, A. Hogg, D. Babusis, M. O. Clarke, et al. Comparative therapeutic efficacy of remdesivir and combination lopinavir, ritonavir, and interferon beta against mers-cov. Nature Communications, 11(1):1–14, 2020." />
<meta name="citation_reference" content="H. Shi, X. Han, N. Jiang, Y. Cao, O. Alwalid, J. Gu, Y. Fan, and C. Zheng. Radiological findings from 81 patients with COVID-19 pneumonia in wuhan, china: a descriptive study. The Lancet Infectious Diseases, 20(4):425–434, Apr. 2020." />
<meta name="citation_reference" content="S. C. Stubbs, B. A. Blacklaws, B. Yohan, F. A. Yudhaputri, R. F. Hayati, B. Schwem, E. M. Salvana, R. V. Destura, J. S. Lester, K. S. Myint, et al. Assessment of a multiplex pcr and nanopore-based method for dengue virus sequencing in indonesia. Virology Journal, 17(1):1–13, 2020." />
<meta name="citation_reference" content="G. M.-K. Tse. Pulmonary pathological features in coronavirus associated severe acute respiratory syndrome (SARS). Journal of Clinical Pathology, 57(3):260–265, Mar. 2004." />
<meta name="citation_reference" content="C. Wang, P. W. Horby, F. G. Hayden, and G. F. Gao. A novel coronavirus outbreak of global health concern. The Lancet, 395(10223):470–473, 2020." />
<meta name="citation_reference" content="R. White, C. Pellefigues, F. Ronchese, O. Lamiable, and D. Eccles. Investigation of chimeric reads using the minion. F1000Research, 6, 2017." />
<meta name="citation_reference" content="F. Wu, S. Zhao, B. Yu, Y.-M. Chen, W. Wang, Z.-G. Song, Y. Hu, Z.-W. Tao, J.-H. Tian, Y.-Y. Pei, et al. A new coronavirus associated with human respiratory disease in china. Nature, 579(7798):265–269, 2020." />
<meta name="citation_reference" content="Y. Xu, K. Lewandowski, S. Lumley, S. Pullan, R. Vipond, M. Carroll, D. Foster, P. C. Matthews, T. Peto, and D. Crook. Detection of viral pathogens with multiplex nanopore minion sequencing: be careful with cross-talk. Frontiers in microbiology, 9:2225, 2018." />
<meta name="citation_reference" content="P. Zhou, X.-L. Yang, X.-G. Wang, B. Hu, L. Zhang, W. Zhang, H.-R. Si, Y. Zhu, B. Li, C.-L. Huang, et al. A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature, pages 1–4, 2020." />
<meta name="citation_reference" content="N. Zhu, D. Zhang, W. Wang, X. Li, B. Yang, J. Song, X. Zhao, B. Huang, W. Shi, R. Lu, et al. China novel coronavirus investigating and research team. a novel coronavirus from patients with pneumonia in china, 2019. N Engl J Med, 382(8):727–733, 2020." />
<meta name="twitter:title" content="CovProfile: profiling the viral genome and gene expressions of SARS-COV2" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="SARS-COV2 is infecting more than one million people. Knowing its genome and gene expressions is essential to understand the virus. Here, we propose a computational tool to detect the viral genomic variations as well as viral gene expressions from the sequenced amplicons with Nanopore devices of the multiplex reverse transcription polymerase chain reactions. We applied our tool to 11 samples of terminally ill patients, and discovered that seven of the samples are infected by two viral strains. The expression of viral genes ORF1ab gene, S gene, M gene, and N gene are high among most of the samples. While performing our tests, we noticed the abundances of MUC5B transcript segments to be consistently higher than those of other genes in the host." />
<meta name="og-title" property="og:title" content="CovProfile: profiling the viral genome and gene expressions of SARS-COV2" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.04.05.026146v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="SARS-COV2 is infecting more than one million people. Knowing its genome and gene expressions is essential to understand the virus. Here, we propose a computational tool to detect the viral genomic variations as well as viral gene expressions from the sequenced amplicons with Nanopore devices of the multiplex reverse transcription polymerase chain reactions. We applied our tool to 11 samples of terminally ill patients, and discovered that seven of the samples are infected by two viral strains. The expression of viral genes ORF1ab gene, S gene, M gene, and N gene are high among most of the samples. While performing our tests, we noticed the abundances of MUC5B transcript segments to be consistently higher than those of other genes in the host." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-04-06" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.04.05.026146v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.04.05.026146v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.04.05.026146v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.04.05.026146v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1231122" />
    <title>CovProfile: profiling the viral genome and gene expressions of SARS-COV2 | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ImqIiNVg-aaEqmS-TFUlV5p5e3-NbfhzlzPpOgnGofQ","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1231122","apath":"\/biorxiv\/early\/2020\/04\/06\/2020.04.05.026146.atom","pisa":"biorxiv;2020.04.05.026146v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.04.05.026146v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.04.05.026146v1","title":"CovProfile: profiling the viral genome and gene expressions of SARS-COV2","identifier":"node\/1231122"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022fh\\\\\\\/udd58PL\\\\\\\/l1wT1OKwQlemex80+m+XaUMbkOSIDe4fsUUmzlBL7uQP7y2bNbFzKSgb7bINQAblT8Jytv5gTFij1Vcr6EtRDq1If0vjXWKU0thweEo\\\\\\\/3GjimLP6OPyUIWJYKGhos1Skkd2fl\\\\\\\/IylPrROKmXXWtVYHIlpV5FeA+U2OeUrXhDe97XscUQXeU+0f9J+NinjjF+KyMXZa2VXhmFOLXDOxE5gEJpCc1hQ+ni0vAnJLBYIbfpsjEfNy0iUMRRkivUTE9iS6VwZlMvhhLw5jhm8BBvBpixNvIMuX0eaTNU4oQaFwIgyoik5txnr59ZGfB\\\\\\\/EQ8pgROmVUzG+FBqdz7zuVylItwBPnI6o5II4ROANXMSUT7wPhLi3p97DN2UHeZylbkngsYW1iFXSXQks+LKwV1TiWVKgD1vkTIqSHJSgyPkoXdQNP6Gi9rYR90nYw\\\\\\\/L0DVGjmp18n5gK1lkNJYnlWKOK9rOsN6cVVN4f2KK4DvklK52YKuJTsBR83\\\\\\\/4vYQGf9F+Dsec86a6WeKRuviREPDCwWlk2Jd3jjYkNi\\\\\\\/MXGqfJQJQpqtv\\\\\\\/2h1\\\\\\\/C4RWOyJpPmkObgHVoO2liuDb59E\\\\\\\/5PqndFtfrSiNuJ8E+0Y4ddBjj\\\\\\\/ybJR3LAK3D1IIbjj+zR6Nc4S4XE+vXvJcPpPT25bgrOV6rm\\\\\\\/Fvnczb0AxK+QXs3rkAzU7T68djbqblMd4dIkbV\\\\\\\/Ms+AMikxi8u859Ix7h7q0HMWsZeEVInCixcxKVo2A9eaixvgvVIW0gIdyVa2WN7zcrUaBzYmRMP3g6ZjUp3GAoW5qCMKgDGnNzsUOaL2U3pW6wzB5Kcad5ZNwwLb8Pij9iHKGJSSQ7xOgmyIQEVzU8LgwM\\\\\\\/awm5Wuy+PaCBBevsLQnv5gUH7dOauuYSpaG+5DODdsATyGwTROq26Ye7wHtlvsyuAtboZceb7K\\\\\\\/ghqA7ZhsEvkHw06M9LzsAnVrU\\\\\\\/x3D4JKI15EfbpTtlZUm2ua4SUBUcZfnSgRiddKGdZJLyfGwND5xVu+lvrejtipfK+TdEKlj\\\\\\\/PG\\\\\\\/zZZLUXmx9XeGxghoiU5ODgT7cIDZROvKNECMcou6vFvcE4Tk\\\\\\\/bRbDA8GqgexSvEWS2sKxwolQKont5TCZq\\\\\\\/mWXFSfLF71l45DhzSxqWjE2zdNnhgh327qjO1KALK8ktAiOhq6His9X3dXN6N1RDbxBpCJ0wRLL69vadYSADiWHFTuB1Z+1vVjiKIAkhrsJXJwz614coDvXPHfDNFX0711HHSqgSTZFErOMneLTdrk7PHW+F8PW3HuPWyTNT7NoxV96CD53Xct1m2UoU4248txwPS7PIfzGeRQ6owdBMR\\\u0022,\\\u0022iv\\\u0022:\\\u0022zhoG7owfH83Dd0U9FYyruA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00227cee0a8687157d9573a08201e84c87ce2868878a3db2f0c0224ac94ad9ae6818\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.04.05.026146v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1231122 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.04.05.026146v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Xie4c_iStPoTJ6gS1M2ET1nL9d3zauG-adNZHV6sbGc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1231122" id="node1231122" data-pisa="biorxiv;2020.04.05.026146v1" data-pisa-master="biorxiv;2020.04.05.026146" data-apath="/biorxiv/early/2020/04/06/2020.04.05.026146.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">CovProfile: profiling the viral genome and gene expressions of SARS-COV2</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Yonghan</span> <span class="nlm-surname">Yu</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Zhengtu</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Yinhu</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Le</span> <span class="nlm-surname">Yu</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-7136-9919" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Wenlong</span> <span class="nlm-surname">Jia</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Feng</span> <span class="nlm-surname">Ye</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Shuai Cheng</span> <span class="nlm-surname">Li</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.04.05.026146 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1231122" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yonghan Yu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>City University of Hong Kong Shenzhen Research Institute</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yonghan%2BYu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYonghan%2BYu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Zhengtu Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>State Key Laboratory of Respiratory Disease, National Clinical Research Center for Respiratory Disease, Guangzhou Institute of Respiratory Health, the First Affiliated Hospital of Guangzhou Medical University</span>, Guangzhou 510120, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhengtu%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhengtu%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yinhu Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>City University of Hong Kong Shenzhen Research Institute</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yinhu%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYinhu%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Le Yu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Beijing YuanShengKangTai (ProtoDNA) Genetech Co. Ltd</span>., Beijing 100190, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Le%2BYu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yu%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALe%2BYu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Wenlong Jia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>City University of Hong Kong Shenzhen Research Institute</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wenlong%2BJia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jia%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWenlong%2BJia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7136-9919" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Wenlong Jia</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Feng Ye </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>State Key Laboratory of Respiratory Disease, National Clinical Research Center for Respiratory Disease, Guangzhou Institute of Respiratory Health, the First Affiliated Hospital of Guangzhou Medical University</span>, Guangzhou 510120, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Feng%2BYe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ye%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFeng%2BYe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:yefeng@gird.cn" class="" data-icon-position="" data-hide-link-title="0">yefeng@gird.cn</a>
<a href="mailto:shuaicli@gmail.com" class="" data-icon-position="" data-hide-link-title="0">shuaicli@gmail.com</a></span></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Shuai Cheng Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>City University of Hong Kong Shenzhen Research Institute</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shuai%2BCheng%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20SC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShuai%2BCheng%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:yefeng@gird.cn" class="" data-icon-position="" data-hide-link-title="0">yefeng@gird.cn</a>
<a href="mailto:shuaicli@gmail.com" class="" data-icon-position="" data-hide-link-title="0">shuaicli@gmail.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.04.05.026146v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1231122" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1231122/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.05.026146v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1231122" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1231122/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.05.026146v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1231122" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1231122/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.05.026146v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1231122" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1231122/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.04.05.026146v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1231122" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1231122/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">SARS-COV2 is infecting more than one million people. Knowing its genome and gene expressions is essential to understand the virus. Here, we propose a computational tool to detect the viral genomic variations as well as viral gene expressions from the sequenced amplicons with Nanopore devices of the multiplex reverse transcription polymerase chain reactions. We applied our tool to 11 samples of terminally ill patients, and discovered that seven of the samples are infected by two viral strains. The expression of viral genes ORF1ab gene, S gene, M gene, and N gene are high among most of the samples. While performing our tests, we noticed the abundances of MUC5B transcript segments to be consistently higher than those of other genes in the host.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-4">Severe acute respiratory syndrome coronavirus 2 (SARS-COV-2) has infected more than one million people, and has led to more than 80,000 deaths, at the time of preparation of this manuscript [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>,<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>,<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>,<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>]. The virus is an enveloped and single-stranded RNA betacoronavirus 30kb base-pairs (bps),, which belongs to the family Coronaviridae [<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>,<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>]. Since the year 2000, we have witnessed and experienced three highly widespread pathogenic coronaviruses in human populations — the other two viruses are SARS-COV in 2002-2003, and MERS-CoV in 2012 [<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>]. All three viruses can lead to Acute Respiratory Distress Syndrome (ARDS) in the human host, which may cause pulmonary fibrosis, and lead to permanent lung function reduction or death [<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>]. The mechanisms of these severe cases remain unclear.</p><p id="p-5">No drug is currently confirmed to be effective against SARS-COV-2. Some vaccine candidates are under intensive studies [<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>]. A nucleotide analog RNA-dependent RNA Polymerase (RdRP) inhibitor called remdesivir has demonstrated effects on SARS-COV-2 [<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>]. Hoffmann <em>et al.</em> suggested that TMPRSS2 inhibitor approved for clinical use can be adopted as a treatment option [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>]. <em>In silico</em> methods have as well produced lists of potential drugs [<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>] and epitopes [<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>] worthy of further exploration. However, short of details on how the viral molecule compromises the cells of hosts, it is hard to prioritize our exploration of the huge array of potential drugs and therapies. Analysis of the mechanism can provide us insights on the viral transmission as well as revealing therapeutic targets.</p><p id="p-6">However, there are many difficulties in analyzing the virus. Isolating the virus can be tedious and time-consuming. The samples usually carry low viral load, resulting in false negatives. To reduce the false negatives, one approach is to amplify the viral sequences with multiplex reverse transcription polymerase chain reaction (RT-PCR), followed by sequencing the amplicons with third generation sequencing technologies such as the Nanopore devices. The approach has been applied to sequence the dengue [<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>] and Zika viruses [<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>], as well as in the SARS-COV-2 viruses [<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>].</p><p id="p-7">Nanopore sequencing generate long reads, which enable the detection of large genomic structural changes, viral recombination and viral integration into host genome. They also supports alternative splicing analyses of microorganism studies [<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>]. However, The long reads suffers high error rates. Detection of the viral and host gene expressions using amplicons is further complicated by several factors. First, the sub-sequences can be amplified unevenly. Second, reads in the gene region amplified from the viral genome are indistinguishable from the gene transcripts. Third, chimeric reads are frequent.</p><p id="p-8">Here, we developed a computational tool, CovProfile, to detect viral genomic variations, and profile viral gene expressions through data from multiplex RT-PCR Amplicon Sequencing on Nanopore GridION. To detect the genomic variations, we mapped the reads to SARS-CoV-2 reference genome (MN908947.3), and to profile the viral gene expressions, we adopt a regularization method to find out the primer efficiencies, We have tested the tool on the data obtained from the sputum samples in eleven severe COVID-19 patients. We found seven out of eleven patients are infected by two strains, and discovered no structural variations. We found that the expression of viral genes ORF1ab gene, S gene, M gene, and N gene are high among most of the samples. While performing our tests, we noticed the abundances of MUC5B transcript segments to be consistently higher than those of other genes in host. The function of this gene may be associated with the severe symptom of pulmonary fibrosis.</p></div><div class="section" id="sec-2"><h2 class="">Method and Materials</h2><div id="sec-3" class="subsection"><h3>Data</h3><p id="p-9">In this study, we collected Nanopore sequencing data for SARS-CoV-2 virus from the sputum samples of 11 COVID-19 patients. A total of 1,804,043 clean reads were generated, with an average of 164,004.00±90,848.28 (Mean±SD) reads per sample (Supplementary Figure 1). Aligning the reads to SARS-CoV-2 and human genomes respectively, the alignment ratio ranged from 20.14% to 99.74% on SARS-CoV-2 genome, and ranged from 0.11% to 72.12% on the human genome. Furthermore, each sample contained 71.24±35.36M data on average, while 61.29±42.13M and 8.87±9.34M data were aligned to SARS-CoV-2 and host genomes, respectively.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Compositions of the data sets. The heights of the columns represented the composition of dsta. The data came from the host, SARS-CoV-2 and other genomes respectively, which were represented by blue, orange and grey colors." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-339047387" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Compositions of the data sets.&lt;/span&gt; The heights of the columns represented the composition of dsta. The data came from the host, SARS-CoV-2 and other genomes respectively, which were represented by blue, orange and grey colors.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146/F1.medium.gif" width="440" height="199"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146/F1.medium.gif" width="440" height="199"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">Compositions of the data sets.</span><p id="p-10" class="first-child">The heights of the columns represented the composition of dsta. The data came from the host, SARS-CoV-2 and other genomes respectively, which were represented by blue, orange and grey colors.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Remove false-positive chimeric reads</h3><p id="p-11">Chimeric nanopore reads formed by random connection of multiplex RT-PCR amplicons were split into multiple segments if the location of their bilateral ends matches to corresponding locations of PCR primer pair. These chimeric reads could be split into several segments with their bilateral ends exactly matching the sites where the multiplex PCR primers are located, suggesting that they should be processed accurately to avoid false positive identification of virus recombinations or host integrations. We position primers on both viral and host genome, and splits the identified chimeric reads into segments corresponding to PCR amplicons. This method rescues lots of sequencing data, providing more accurate alignment and higher coverage. Realignment were performed on the separated segments before analysis.</p></div><div id="sec-5" class="subsection"><h3>Mutation Detection</h3><p id="p-12">We aligned the clean data to SARS-CoV-2 genome (NCBI MN908947.3) with minimap2 [<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>] with default parameters for Oxford Nanopore reads. With the alignment results, the genomic variations with average quality larger than 10 and coverage depth larger than 20 were called with bcftools (version 1.8). To reduce the effects brought about by the PCR amplification, a variation is filtered if it was located within 10 bp upstream or downstream of the primer region, and occurred only in one sample.</p></div><div id="sec-6" class="subsection"><h3>Estimating viral load and primer efficiency</h3><p id="p-13">Assume that there are <em>n</em> primer pairs (<em>s<sub>i</sub></em>,<em>t<sub>i</sub></em>), with primer efficiency <em>e<sub>i</sub></em>, 1 ≤ <em>i</em> ≤ <em>n</em>. Let the viral gene boundaries be denoted (<em>l<sub>g</sub>,r<sub>g</sub></em>), where <em>g</em> indexes the genes. Denote the observed sequence depth captured by the primer pair (<em>s<sub>i</sub>, t<sub>i</sub></em>) as <em>d<sub>i</sub></em>.</p><p id="p-14">We want to estimate the viral gene expression. Let the number of viral copies be denoted <em>v</em>, the number of transcripts of gene <em>g</em> be denoted <em>t<sub>g</sub></em>, and the sequencing rate of amplified segment be denoted α. Then, we can have the following two approximations for <em>d<sub>i</sub></em>.</p><p id="p-15">If the primer <em>i</em> is fully covered by gene <em>g</em> (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig1. A</a>), then as the number of amplicons increases exponentially with the number of replication cycles we have
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/06/2020.04.05.026146/embed/graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/06/2020.04.05.026146/embed/graphic-2.gif"/></noscript></span>

</span>
where <em>k</em> is the number of replication cycles.</p><p id="p-16">Similarly, if primer pair <em>i</em> spans across gene boundaries; that is, it belongs to none of the gene transcripts (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig1. B</a>), then we have
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/06/2020.04.05.026146/embed/graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/06/2020.04.05.026146/embed/graphic-3.gif"/></noscript></span>

</span>
</p><p id="p-17">If some reads share two ends, where one end is a primer and the other end is at some gene boundary (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig 1D</a>, <a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig 1E</a>), it may indicate that these amplicons are the result of linear amplification. Denote the number of reads <em>r<sub>i,g</sub></em> with two endpoints as (<em>t<sub>i</sub></em>, <em>e<sub>g</sub></em>), and the number of reads <em>r<sub>g,i</sub></em> with two endpoints as (<em>l<sub>i</sub></em>, <em>s<sub>g</sub></em>), then
<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/06/2020.04.05.026146/embed/graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/06/2020.04.05.026146/embed/graphic-4.gif"/></noscript></span>

</span>
</p><p id="p-18">If a gene <em>g</em> is within some primer target region (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig2. G</a>), then the sequenced fragment can be expected to exactly cover the gene. Let the number of observed reads span exactly the whole gene <em>g</em> as <em>r<sub>g</sub></em>, then
<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/06/2020.04.05.026146/embed/graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/06/2020.04.05.026146/embed/graphic-5.gif"/></noscript></span>

</span>
</p><p id="p-19">To find the expressions of the viral genes, we want to minimize the following objective function based on the entities defined above
<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/06/2020.04.05.026146/embed/graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/04/06/2020.04.05.026146/embed/graphic-6.gif"/></noscript></span>

</span>
where λ<sub>1</sub> and λ<sub>2</sub> are two regularization parameters, and <em>k</em> is a weight factor (we set <em>k</em> to 21 according to our experiments). The viral load <em>v</em>, primer efficiencies <em>e<sub>i</sub></em>, and expressions <em>t<sub>g</sub></em> which minimize the objective equation are our estimation for these parameters.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relations between viral gene, primer pair and sequence. A: Primer pair spans fully on gene with sequence aligned with primers. B: Primer pair spans across gene boundaries at 5&#x2019; with sequence aligned with primers. C: Primer pair spans across gene boundaries at 3&#x2019; with sequence aligned with primers. D: Primer pair spans across two gene with sequence aligned with primers. E: Primer pair spans across gene boundaries at 5&#x2019; with sequence aligned with gene boundaries at 3&#x2019; and primer at 5&#x2019;. F: Primer pair spans across gene boundaries at 3&#x2019; with sequence aligned with gene boundaries at 5&#x2019; and primer at 3&#x2019;. G: Sequence aligned with gene boundaries at both 5&#x2019; and 3&#x2019;." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-339047387" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Relations between viral gene, primer pair and sequence.&lt;/span&gt; &lt;strong&gt;A&lt;/strong&gt;: Primer pair spans fully on gene with sequence aligned with primers. &lt;strong&gt;B&lt;/strong&gt;: Primer pair spans across gene boundaries at 5&#x2019; with sequence aligned with primers. &lt;strong&gt;C&lt;/strong&gt;: Primer pair spans across gene boundaries at 3&#x2019; with sequence aligned with primers. &lt;strong&gt;D&lt;/strong&gt;: Primer pair spans across two gene with sequence aligned with primers. &lt;strong&gt;E&lt;/strong&gt;: Primer pair spans across gene boundaries at 5&#x2019; with sequence aligned with gene boundaries at 3&#x2019; and primer at 5&#x2019;. &lt;strong&gt;F&lt;/strong&gt;: Primer pair spans across gene boundaries at 3&#x2019; with sequence aligned with gene boundaries at 5&#x2019; and primer at 3&#x2019;. &lt;strong&gt;G&lt;/strong&gt;: Sequence aligned with gene boundaries at both 5&#x2019; and 3&#x2019;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146/F2.medium.gif" width="440" height="327"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146/F2.medium.gif" width="440" height="327"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Relations between viral gene, primer pair and sequence.</span><p id="p-20" class="first-child"><strong>A</strong>: Primer pair spans fully on gene with sequence aligned with primers. <strong>B</strong>: Primer pair spans across gene boundaries at 5’ with sequence aligned with primers. <strong>C</strong>: Primer pair spans across gene boundaries at 3’ with sequence aligned with primers. <strong>D</strong>: Primer pair spans across two gene with sequence aligned with primers. <strong>E</strong>: Primer pair spans across gene boundaries at 5’ with sequence aligned with gene boundaries at 3’ and primer at 5’. <strong>F</strong>: Primer pair spans across gene boundaries at 3’ with sequence aligned with gene boundaries at 5’ and primer at 3’. <strong>G</strong>: Sequence aligned with gene boundaries at both 5’ and 3’.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>Calculate viral gene expression</h3><p id="p-21">To calculate the viral gene expression, the duplication generated during PCR were removed based on the estimated primer efficiencies. The gene expression can be calculated as the sum of the base depth of each gene divided by its length.</p></div><div id="sec-8" class="subsection"><h3>Capture gene product from host</h3><p id="p-22">To capture the gene products of the host, reads and primers were aligned to the hg38 transcript database derived from UCSC refGene annotation. Primer alignment were performed with BLAT [<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>]. Regions where the depth of reads alignment are equal or larger to 5 are screened. Reads with endpoints matching the screened primer alignment were treated as PCR duplications. Due to mismatches in primer to transcript alignment, primer efficiencies may change when capturing gene product. Thus, the recognized PCR duplication were removed and one copy of those reads were kept. The gene expression from host can then be estimated as the sum of the base depth of each transcript divided by its length.</p></div></div><div class="section" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Distribution of SNPs in SARS-CoV-2 genome</h3><p id="p-23">After aligning the sequencing data to the SARS-CoV-2 genome, we detected the mutations of SARS-CoV-2 in the 11 samples3. Based on the mutations of SARS-CoV-2 in 11 samples, we discovered 38 SNPs (Single Nucleotide Polymorphisms) in total; 35 of them are located on the genetic regions, including gene ORF1ab, S, N and M. Gene ORF1ab contained 23 SNPs in 11 samples, and 17 of them were non-synonymous mutations. Non-synonymous mutations also occupied a large part of SNPs in the gene S, N and M: 5 of 8 SNPs in gene S, 1 of 3 SNPs in gene N, and 1 of 1 SNPs in gene M were non-synonymous mutations. Since most of the SNPs were located in only 1 sample, we deduced that the functions of the discovered SNPs need to be further verified according to lab experiments. Surprisingly, we also found that 7 among 11 samples were heterogeneous at loci 8782. This may imply that the hosts have two different virus strains. However, we did not find creditable InDels (Insertions and Deletions), or structural variations, viral-host integration during the mutation analysis.</p></div><div id="sec-11" class="subsection"><h3>Evaluations on the amplification efficient for the primers</h3><p id="p-24">Since 98 pairs of primers from two pools were designed for the enrichment of SARS-CoV-2 genome before sequencing, we discovered that different parts of SARS-CoV-2 genome exhibited different coverage depths. We then calculated the amplification efficiencies for the primers with our proposed method. We discovered that the primers located at the 5’ and 3’ ends of the SARS-CoV-2 genome exhibited higher amplification efficiency (&gt;1.2, <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>), including primer01 and primer84-primer93. With the genomic locations of the primers, we know that the genomic sequences within the ranges 24-410 and 25,280-28,464 can be amplified with more sequencing data. On the other hand, the remaining primers exhibited slightly lower efficiencies, within the range of 1.0-1.2. For primers 64, 70, 83 and 85, their amplification efficiencies were as low as 1, which indicates that the target regions of these primers were not amplified. Based on the primer efficiencies, the effect of data bias which was induced by the PCR experiments can be attenuated from the original data, and they can be further applied for the normalization of viral gene expression under different primers.</p></div><div id="sec-12" class="subsection"><h3>Normalized coverage depth on SARS-CoV-2 genome</h3><p id="p-25">Due to the amplification discrepancy among the primers, the sequencing data was unevenly distributed along the SARS-CoV-2 genome. After aligning the sequencing data to the SARS-CoV-2 genome, we found that the coverage depths of some genomic regions reached over 8000x in some samples, but some regions have coverage depths less than 10x (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). In order to minimize the data imbalance induced by PCR amplification, we performed data normalization based on the primer efficiency and their locations on SARS-CoV-2 genome, so that the normalized data could reflect the actual virus reproduction in the host. Based on the algorithm illustrated in the method, we obtained the normalized sequencing depth of 11 sample data.</p><p id="p-26">After data normalization, the overall coverage depth along the SARS-CoV-2 genome was relatively even, and less than 300x. In addition, we discovered that the ORF1ab gene, S gene, M gene, and N gene exhibited higher coverage depth in the samples, which might indicate higher expression (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). These genes are related to host cell recognition, amplification and virus particle assembly, so we speculate that the results might explain the severe SARS-CoV-2 virus infection in the patients. Since gene E, ORF6, ORF7a, ORF7b, ORF8 and ORF10 are less than 400bp in length, their transcripts might not have been captured by the primers, and hence their expressions cannot be estimated.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distributions of High-frequency SNPs among the 11 COVID-19 samples. The locations of genes in SARS-CoV-2 were exhibited on top of the plot, and the distributions of SNPs were specified for the 11 samples." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-339047387" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Distributions of High-frequency SNPs among the 11 COVID-19 samples.&lt;/span&gt; The locations of genes in SARS-CoV-2 were exhibited on top of the plot, and the distributions of SNPs were specified for the 11 samples.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146/F3.medium.gif" width="351" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146/F3.medium.gif" width="351" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">Distributions of High-frequency SNPs among the 11 COVID-19 samples.</span><p id="p-27" class="first-child">The locations of genes in SARS-CoV-2 were exhibited on top of the plot, and the distributions of SNPs were specified for the 11 samples.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Amplification efficient for the 98 pairs of primers. The amplification efficient was calculated for the primers, and their distributions were detected in 11 samples. The primers in pool1 and pool2 were marked with red and blue colors respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-339047387" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Amplification efficient for the 98 pairs of primers.&lt;/span&gt; The amplification efficient was calculated for the primers, and their distributions were detected in 11 samples. The primers in pool1 and pool2 were marked with red and blue colors respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146/F4.medium.gif" width="386" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146/F4.medium.gif" width="386" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">Amplification efficient for the 98 pairs of primers.</span><p id="p-28" class="first-child">The amplification efficient was calculated for the primers, and their distributions were detected in 11 samples. The primers in pool1 and pool2 were marked with red and blue colors respectively.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Original and normalized coverage depth for the 11 samples. The locations of genes in SARS-CoV-2 were exhibited on top of the plot, and they were marked by different colors. Each sample contained two plots, which represented the original (left) and normalized coverage (right) depths for the bases along the SARS-CoV-2 genome respectively. The coverage depth were colored by the genes in SARS-CoV-2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-339047387" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Original and normalized coverage depth for the 11 samples.&lt;/span&gt; The locations of genes in SARS-CoV-2 were exhibited on top of the plot, and they were marked by different colors. Each sample contained two plots, which represented the original (left) and normalized coverage (right) depths for the bases along the SARS-CoV-2 genome respectively. The coverage depth were colored by the genes in SARS-CoV-2.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146/F5.medium.gif" width="370" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146/F5.medium.gif" width="370" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <span class="caption-title">Original and normalized coverage depth for the 11 samples.</span><p id="p-29" class="first-child">The locations of genes in SARS-CoV-2 were exhibited on top of the plot, and they were marked by different colors. Each sample contained two plots, which represented the original (left) and normalized coverage (right) depths for the bases along the SARS-CoV-2 genome respectively. The coverage depth were colored by the genes in SARS-CoV-2.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>High expression genes in host</h3><p id="p-30">Previous studies on the sequencing data have shown that it contains the human genome, so we detected the corresponding genes and their expression levels. Aligning these data to human transcript database, we identified many segments of human gene transcripts. Subsequently, the coverage depths of these genes were normalized based on the primer efficiencies. Only several genes are listed within the top 5% in more than five samples according to the normalized depth. They include the genes MUC5B, TCAF12, KRT12 and etc. MUC5B is the only gene consistently within the top 5% across all the samples. More than 50% of the MUC5B gene region is covered (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>). However, the region with high coverage in MUC5B gene is inconsistent among the samples, perhaps due to data scarcity.</p></div></div><div class="section" id="sec-14"><h2 class="">Conclusion and Discussion</h2><p id="p-31">In this study, we implemented a software tool, CovProfile, to analyze multiplex RT-PCRs on Nanopore sequencing data, and studied the sputum samples of COVID-19 patients. Our results show enrichment of PCR amplicons which totally cover 99.7% of SARS-CoV-2 virus genome, revealing that the multiplex RT-PCRs are a reliable method in identification of COVID-19 infection. Interestingly, the variant (26709:G&gt;A) is identified in half of samples, and shows consistent heterozygosity (alteration frequency: 19.36%-26.15%) in all relevant samples. This suggests at least two virus strains in the patients.</p><p id="p-32">The data contained large amount of chimeric reads formed by unintended random connection of multiplex PCR amplicons, making up 1.69% of total sequencing reads. Similar observation has been reported that these chimeric reads can be formed during library preparation and sequencing, and lead to the cross barcode assignment errors in multiplex samples [<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>]. We applied CovProfile on the data and recovered majority of chimeric reads.</p><p id="p-33">The sequencing reads could be used to evaluate the expression of SARS-CoV-2 genes. However, the coverage unevenness caused by efficiency differences in the multiplex primers impedes our calculation of pathogenic genes expression. To solve this problem, we propose a multi-parameter method which considers the primer efficiency, gene differential expressions, and RNA virus genome copy number simultaneously. Different locations on the viral genome have distinct composition of molecule sources, including RNA genome, expressed viral transcript, and PCR amplicons. According to the gene locations along the viral genome, we enumerate reads types based on aligned positions, and deduce the contributions of different sources to the local region depth. By applying this method on our data, we find that viral gene ORF1ab may maintain higher expression than other genes across most samples. This viral gene occupied a large region on SARS-CoV-2 genome, and it responsible for the amplification and assembly of the virus [<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>]. For the primer, the multiplex primers efficiency has wide distribution in single sample, and the distributions among samples demonstrate similar trend. This suggests that our estimation of primers efficiency comply to the reality.</p><p id="p-34">We utilize the sequencing data to evaluate host genome expression and find several genes consistently activated in the majority of samples after eliminating the PCR effects. The most abundant gene is the MUC5B, which encodes a member of the mucin protein family. The family proteins are the major components of mucus secretions in normal lung mucus [<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>]. MUC5B has been reported to be highly expressed in lung disease, such as chronic obstructive pulmonary disease (COPD). Its up-regulated transcription could cause mucociliary dysfunction and enhances lung fibrosis [<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>], thus might be relevant to the recent report that a lot of mucus are found in the lung of COVID-19 infected patients. Another activated host gene is the TCAF2, which encodes transient receptor potential cation channel-associated factor. It could bind to the TRPM8 channel and promote its trafficking to the cell surface. Previous study reported that TCAF2 significantly increases the migration speed of prostate cancer cells when it co-expresses with TRPM8 [<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>]. Other enriched genes might also break the cellular homeostasis, including the nucleoredoxin-like gene NXNL2, tyrosine kinases FER, and so on. Further studies of these genes could help researchers gain insights into the dysregulation of host cells after the SARS-CoV-2 infection. We should point out these genes are extracted based on a very limited amount of data, and the conclusions can be different with a large data sets.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Normalized coverage depth on MUC5B gene in the 11 samples. In each plot, the X and Y coordinates exhibited the length of MUC5B and the normalized coverage depth on along the MUC5B gene respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-339047387" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Normalized coverage depth on MUC5B gene in the 11 samples.&lt;/span&gt; In each plot, the X and Y coordinates exhibited the length of MUC5B and the normalized coverage depth on along the MUC5B gene respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146/F6.medium.gif" width="385" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146/F6.medium.gif" width="385" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/06/2020.04.05.026146/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6.</span> <span class="caption-title">Normalized coverage depth on MUC5B gene in the 11 samples.</span><p id="p-35" class="first-child">In each plot, the X and Y coordinates exhibited the length of MUC5B and the normalized coverage depth on along the MUC5B gene respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">Overall, we present a tool for genomic variation detection and the transcriptional profile of SARS-CoV-2 infection from multiplex Nanopore sequencing data. The tool decomposes and realign the chimeric reads that commonly exist in the multiplex Nanopore sequencing, and this will greatly promote the data usage. The method also provides a robust estimation of primer efficiency through a multi-parameter model. The viral and host gene expression we report could pave steps to the development of diagnosis and therapy strategies of SARS-CoV-2 infection candidates.</p></div><div class="section" id="sec-15"><h2 class="">Funding</h2><p id="p-38">Not applicable.</p></div><div class="section" id="sec-16"><h2 class="">Author’s contributions</h2><p id="p-39">SCL conceived the idea.</p><p id="p-40">SCL and FY supervised the project.</p><p id="p-41">SCL and YY designed the algorithms</p><p id="p-42">SCL, WJ, YY and HL discussed and designed the experiments.</p><p id="p-43">YY and JL implemented the code and conducted the analysis.</p><p id="p-44">ZL and LY prepared the data.</p><p id="p-45">SCL YY WJ and YL drafted the manuscript.</p><p id="p-46">SCL, FY revised the manuscript.</p><p id="p-47">All authors read and approved the final manuscript.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-37">We thank just about everybody.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.05.026146v1.1"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Novel coronavirus (2019-ncov)</span> <span class="cit-source">situation reports</span>, <a href="https://www.who.int/emergencies/diseases/novelcoronavirus-2019/situation-reports">https://www.who.int/emergencies/diseases/novelcoronavirus-2019/situation-reports</a>, <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026146v1.2" data-doi="10.1093/cid/ciw412"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Assiri</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. R.</span> <span class="cit-name-surname">Abedi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Masri</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. B.</span> <span class="cit-name-surname">Saeed</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. I.</span> <span class="cit-name-surname">Gerber</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J. T.</span> <span class="cit-name-surname">Watson</span></span>. <span class="cit-article-title">Middle east respiratory syndrome coronavirus infection during pregnancy: A report of 5 cases from saudi arabia: Table 1</span>. <abbr class="cit-jnl-abbrev">Clinical Infectious Diseases</abbr>, <span class="cit-vol">63</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">951</span>–<span class="cit-lpage">953</span>, <span class="cit-month">June</span> <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2BInfectious%2BDiseases%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fcid%252Fciw412%26rft_id%253Dinfo%253Apmid%252F27358348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/cid/ciw412&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27358348&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.04.05.026146.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026146v1.3" data-doi="10.1186/s13059-015-0777-z"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. T.</span> <span class="cit-name-surname">Bolisetty</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Rajadinakaran</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">B. R.</span> <span class="cit-name-surname">Graveley</span></span>. <span class="cit-article-title">Determining exon connectivity in complex mrnas by nanopore sequencing</span>. <abbr class="cit-jnl-abbrev">Genome Biology</abbr>, <span class="cit-vol">16</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">204</span>, <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiology%26rft.volume%253D16%26rft.spage%253D204%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0777-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0777-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026146v1.4" data-doi="10.1038/nrmicro.2016.81"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">de Wit</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">van Doremalen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Falzarano</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">V. J.</span> <span class="cit-name-surname">Munster</span></span>. <span class="cit-article-title">Sars and mers: recent insights into emerging coronaviruses</span>. <abbr class="cit-jnl-abbrev">Nature Reviews Microbiology</abbr>, <span class="cit-vol">14</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">523</span>, <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BReviews%2BMicrobiology%26rft.volume%253D14%26rft.spage%253D523%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro.2016.81%26rft_id%253Dinfo%253Apmid%252F27344959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro.2016.81&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27344959&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.04.05.026146.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026146v1.5" data-doi="10.1083/jcb.201402076"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gkika</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Lemonnier</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Shapovalov</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gordienko</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Poux</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Bernardini</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bokhobza</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Bidaux</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Degerny</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Verreman</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Trp channel–associated factors are a novel protein family that regulates trpm8 trafficking and activityidentification of trpm8 partner proteins</span>. <abbr class="cit-jnl-abbrev">The Journal of Cell Biology</abbr>, <span class="cit-vol">208</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">89</span>–<span class="cit-lpage">107</span>, <span class="cit-pub-date">2015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BCell%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.201402076%26rft_id%253Dinfo%253Apmid%252F25559186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjg6IjIwOC8xLzg5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDYvMjAyMC4wNC4wNS4wMjYxNDYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026146v1.6"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Gorbalenya</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Baker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Baric</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">The species severe acute respiratory syndrome-related coronavirus: classifying 2019-ncov and naming it sars-cov-2</span>. <abbr class="cit-jnl-abbrev">Nature Microbiology</abbr>, <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026146v1.7"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">P. H.</span> <span class="cit-name-surname">Guzzi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Mercatelli</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Ceraolo</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">F. M.</span> <span class="cit-name-surname">Giorgi</span></span>. <span class="cit-article-title">Master regulator analysis of the sars-cov-2/human interactome</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>, <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026146v1.8"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L. A.</span> <span class="cit-name-surname">Hancock</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. E.</span> <span class="cit-name-surname">Hennessy</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G. M.</span> <span class="cit-name-surname">Solomon</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Dobrinskikh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Estrella</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Hara</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. B.</span> <span class="cit-name-surname">Hill</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W. J.</span> <span class="cit-name-surname">Kissner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Markovetz</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. E. G.</span> <span class="cit-name-surname">Villalon</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Muc5b overexpression causes mucociliary dysfunction and enhances lung fibrosis in mice</span>. <abbr class="cit-jnl-abbrev">Nature communications</abbr>, <span class="cit-vol">9</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>, <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bcommunications%26rft.volume%253D9%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026146v1.9"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hoffmann</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Kleine-Weber</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Schroeder</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Kruger</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Herrler</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Erichsen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. S.</span> <span class="cit-name-surname">Schiergens</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Herrler</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.-H.</span> <span class="cit-name-surname">Wu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Nitsche</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Sars-cov-2 cell entry depends on ace2 and tmprss2 and is blocked by a clinically proven protease inhibitor</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>, <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026146v1.10" data-doi="10.1101/gr.229202. Article published online before March 2002"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">W. J.</span> <span class="cit-name-surname">Kent</span></span>. <span class="cit-article-title">BLAT—the BLAST-like alignment tool</span>. <abbr class="cit-jnl-abbrev">Genome Research</abbr>, <span class="cit-vol">12</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">656</span>–<span class="cit-lpage">664</span>, <span class="cit-month">Mar</span>. <span class="cit-pub-date">2002</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.229202.%2BArticle%2Bpublished%2Bonline%2Bbefore%2BMarch%2B2002%26rft_id%253Dinfo%253Apmid%252F11932250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEyLzQvNjU2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDYvMjAyMC4wNC4wNS4wMjYxNDYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026146v1.11" data-doi="10.1093/bioinformatics/bty191"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Li</span></span>. <span class="cit-article-title">Minimap2: pairwise alignment for nucleotide sequences</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">34</span>(<span class="cit-issue">18</span>):<span class="cit-fpage">3094</span>–<span class="cit-lpage">3100</span>, <span class="cit-day">05</span> <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D3094%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty191%26rft_id%253Dinfo%253Apmid%252F29750242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty191&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29750242&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.04.05.026146.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026146v1.12"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Zhou</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. V.</span> <span class="cit-name-surname">Garner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. P.</span> <span class="cit-name-surname">Watkins</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. J.</span> <span class="cit-name-surname">Carter</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Smoot</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Gregg</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Daniels</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Jervey</span></span>, <span class="cit-etal">et al.</span> <abbr class="cit-jnl-abbrev">Research and development on therapeutic agents and vaccines for covid-19 and related human coronavirus diseases</abbr>, <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026146v1.13" data-doi="10.1136/thoraxjnl-2018-212430"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. K.</span> <span class="cit-name-surname">Mathai</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Humphries</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Kropski</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. S.</span> <span class="cit-name-surname">Blackwell</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Powers</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Walts</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Markin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Woodward</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Chung</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. K.</span> <span class="cit-name-surname">Brown</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Muc5b variant is associated with visually and quantitatively detected preclinical pulmonary fibrosis</span>. <abbr class="cit-jnl-abbrev">Thorax</abbr>, <span class="cit-vol">74</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">1131</span>–<span class="cit-lpage">1139</span>, <span class="cit-pub-date">2019</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThorax%26rft_id%253Dinfo%253Adoi%252F10.1136%252Fthoraxjnl-2018-212430%26rft_id%253Dinfo%253Apmid%252F31558622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToidGhvcmF4am5sIjtzOjU6InJlc2lkIjtzOjEwOiI3NC8xMi8xMTMxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDYvMjAyMC4wNC4wNS4wMjYxNDYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026146v1.14"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. C.</span> <span class="cit-name-surname">Moore</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Penrice-Randal</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Alruwaili</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Dong</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. T.</span> <span class="cit-name-surname">Pullan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Carter</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Bewley</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Zhao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Sun</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Hartley</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Amplicon based minion sequencing of sars-cov-2 and metagenomic characterisation of nasopharyngeal swabs from patients with covid-19</span>. <abbr class="cit-jnl-abbrev">medRxiv</abbr>, <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026146v1.15"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Phan</span></span>. <span class="cit-article-title">Genetic diversity and evolution of SARS-CoV-2</span>. <abbr class="cit-jnl-abbrev">Infection, Genetics and Evolution</abbr>, <span class="cit-vol">81</span>:<span class="cit-fpage">104260</span>, <span class="cit-month">July</span> <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%252C%2BGenetics%2Band%2BEvolution%26rft.volume%253D81%26rft.spage%253D104260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026146v1.16"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Prachar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Justesen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. B.</span> <span class="cit-name-surname">Steen-Jensen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. P.</span> <span class="cit-name-surname">Thorgrimsen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Jurgons</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Winther</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">F. O.</span> <span class="cit-name-surname">Bagger</span></span>. <span class="cit-article-title">Covid-19 vaccine candidates: Prediction and validation of 174 sars-cov-2 epitopes</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>, <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026146v1.17"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Quick</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N. D.</span> <span class="cit-name-surname">Grubaugh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. T.</span> <span class="cit-name-surname">Pullan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I. M.</span> <span class="cit-name-surname">Claro</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Smith</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Gangavarapu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Oliveira</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Robles-Sikisaka</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. F.</span> <span class="cit-name-surname">Rogers</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Beutler</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Multiplex pcr method for minion and illumina sequencing of zika and other virus genomes directly from clinical samples</span>. <abbr class="cit-jnl-abbrev">nature protocols</abbr>, <span class="cit-vol">12</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">1261</span>, <span class="cit-pub-date">2017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253Dnature%2Bprotocols%26rft.volume%253D12%26rft.spage%253D1261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026146v1.18" data-doi="10.1038/nature12807"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. G.</span> <span class="cit-name-surname">Roy</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Livraghi-Butrico</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Fletcher</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. M.</span> <span class="cit-name-surname">McElwee</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. E.</span> <span class="cit-name-surname">Evans</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Boerner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. N.</span> <span class="cit-name-surname">Alexander</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. K.</span> <span class="cit-name-surname">Bellinghausen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. S.</span> <span class="cit-name-surname">Song</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y. M.</span> <span class="cit-name-surname">Petrova</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Muc5b is required for airway defence</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">505</span>(<span class="cit-issue">7483</span>):<span class="cit-fpage">412</span>–<span class="cit-lpage">416</span>, <span class="cit-pub-date">2014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D505%26rft.spage%253D412%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12807%26rft_id%253Dinfo%253Apmid%252F24317696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12807&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24317696&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.04.05.026146.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000329621800045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026146v1.19"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">T. P.</span> <span class="cit-name-surname">Sheahan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Sims</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Leist</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Schafer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Won</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Brown</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Montgomery</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Hogg</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Babusis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. O.</span> <span class="cit-name-surname">Clarke</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Comparative therapeutic efficacy of remdesivir and combination lopinavir, ritonavir, and interferon beta against mers-cov</span>. <abbr class="cit-jnl-abbrev">Nature Communications</abbr>, <span class="cit-vol">11</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>, <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D11%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026146v1.20"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Shi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Han</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Jiang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Cao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Alwalid</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Gu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Fan</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Zheng</span></span>. <span class="cit-article-title">Radiological findings from 81 patients with COVID-19 pneumonia in wuhan, china: a descriptive study</span>. <abbr class="cit-jnl-abbrev">The Lancet Infectious Diseases</abbr>, <span class="cit-vol">20</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">425</span>–<span class="cit-lpage">434</span>, <span class="cit-month">Apr</span>. <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BLancet%2BInfectious%2BDiseases%26rft.volume%253D20%26rft.spage%253D425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026146v1.21"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. C.</span> <span class="cit-name-surname">Stubbs</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. A.</span> <span class="cit-name-surname">Blacklaws</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Yohan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F. A.</span> <span class="cit-name-surname">Yudhaputri</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. F.</span> <span class="cit-name-surname">Hayati</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Schwem</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. M.</span> <span class="cit-name-surname">Salvana</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. V.</span> <span class="cit-name-surname">Destura</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Lester</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. S.</span> <span class="cit-name-surname">Myint</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Assessment of a multiplex pcr and nanopore-based method for dengue virus sequencing in indonesia</span>. <abbr class="cit-jnl-abbrev">Virology Journal</abbr>, <span class="cit-vol">17</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>, <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%2BJournal%26rft.volume%253D17%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026146v1.22" data-doi="10.1136/jcp.2003.013276"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">G. M.-K.</span> <span class="cit-name-surname">Tse</span></span>. <span class="cit-article-title">Pulmonary pathological features in coronavirus associated severe acute respiratory syndrome (SARS)</span>. <abbr class="cit-jnl-abbrev">Journal of Clinical Pathology</abbr>, <span class="cit-vol">57</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">260</span>–<span class="cit-lpage">265</span>, <span class="cit-month">Mar</span>. <span class="cit-pub-date">2004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BPathology%26rft.stitle%253DJ.%2BClin.%2BPathol.%26rft.aulast%253DTse%26rft.auinit1%253DG%2BM-K%26rft.volume%253D57%26rft.issue%253D3%26rft.spage%253D260%26rft.epage%253D265%26rft.atitle%253DPulmonary%2Bpathological%2Bfeatures%2Bin%2Bcoronavirus%2Bassociated%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome%2B%2528SARS%2529%26rft_id%253Dinfo%253Adoi%252F10.1136%252Fjcp.2003.013276%26rft_id%253Dinfo%253Apmid%252F14990596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToiamNsaW5wYXRoIjtzOjU6InJlc2lkIjtzOjg6IjU3LzMvMjYwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDYvMjAyMC4wNC4wNS4wMjYxNDYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026146v1.23"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. W.</span> <span class="cit-name-surname">Horby</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F. G.</span> <span class="cit-name-surname">Hayden</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">G. F.</span> <span class="cit-name-surname">Gao</span></span>. <span class="cit-article-title">A novel coronavirus outbreak of global health concern</span>. <abbr class="cit-jnl-abbrev">The Lancet</abbr>, <span class="cit-vol">395</span>(<span class="cit-issue">10223</span>):<span class="cit-fpage">470</span>–<span class="cit-lpage">473</span>, <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BLancet%26rft.volume%253D395%26rft.spage%253D470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026146v1.24"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">White</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Pellefigues</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Ronchese</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Lamiable</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Eccles</span></span>. <span class="cit-article-title">Investigation of chimeric reads using the minion</span>. <abbr class="cit-jnl-abbrev">F1000Research</abbr>, <span class="cit-fpage">6</span>, <span class="cit-pub-date">2017</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026146v1.25" data-doi="10.1038/s41586-020-2008-3"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Wu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Zhao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Yu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.-M.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.-G.</span> <span class="cit-name-surname">Song</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Hu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Z.-W.</span> <span class="cit-name-surname">Tao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.-H.</span> <span class="cit-name-surname">Tian</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.-Y.</span> <span class="cit-name-surname">Pei</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">A new coronavirus associated with human respiratory disease in china</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">579</span>(<span class="cit-issue">7798</span>):<span class="cit-fpage">265</span>–<span class="cit-lpage">269</span>, <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D579%26rft.spage%253D265%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2008-3%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-020-2008-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=http://www.n&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.04.05.026146.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026146v1.26"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Xu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Lewandowski</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Lumley</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Pullan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Vipond</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Carroll</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Foster</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. C.</span> <span class="cit-name-surname">Matthews</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Peto</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Crook</span></span>. <span class="cit-article-title">Detection of viral pathogens with multiplex nanopore minion sequencing: be careful with cross-talk</span>. <abbr class="cit-jnl-abbrev">Frontiers in microbiology</abbr>, <span class="cit-vol">9</span>:<span class="cit-fpage">2225</span>, <span class="cit-pub-date">2018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2Bmicrobiology%26rft.volume%253D9%26rft.spage%253D2225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026146v1.27"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Zhou</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.-L.</span> <span class="cit-name-surname">Yang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.-G.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Hu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H.-R.</span> <span class="cit-name-surname">Si</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.-L.</span> <span class="cit-name-surname">Huang</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, pages <span class="cit-fpage">1</span>–<span class="cit-lpage">4</span>, <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.05.026146v1.28" data-doi="10.1056/NEJMoa2001017"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Zhu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Zhang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Wang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Yang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Song</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhao</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Huang</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Shi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Lu</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">China novel coronavirus investigating and research team. a novel coronavirus from patients with pneumonia in china, 2019</span>. <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>, <span class="cit-vol">382</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">727</span>–<span class="cit-lpage">733</span>, <span class="cit-pub-date">2020</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft.volume%253D382%26rft.spage%253D727%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa2001017%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa2001017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=http://www.n&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F06%2F2020.04.05.026146.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.04.05.026146v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.04.05.026146v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.03.16.994236v2" title="The sequence of human ACE2 is suboptimal for binding the S spike protein of SARS coronavirus 2" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.13.990242v4" title="Accurate Identification of SARS-CoV-2 from Viral Genome Sequences using Deep Learning" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;April 06, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.04.05.026146v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.04.05.026146v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.04.05.026146v1" class="active" data-icon-position="" data-hide-link-title="0">CovProfile: profiling the viral genome and gene expressions of SARS-COV2</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1231122" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-gOozZyhmSDlgyBGpcDm_UXu0RbBXm9pt5Ok1JkmyHbs" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1231122" id="node1231122--2" data-pisa="biorxiv;2020.04.05.026146v1" data-pisa-master="biorxiv;2020.04.05.026146" data-seqnum="1231122" data-apath="/biorxiv/early/2020/04/06/2020.04.05.026146.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">CovProfile: profiling the viral genome and gene expressions of SARS-COV2</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Yonghan</span> <span class="nlm-surname">Yu</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Zhengtu</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Yinhu</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Le</span> <span class="nlm-surname">Yu</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Wenlong</span> <span class="nlm-surname">Jia</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Feng</span> <span class="nlm-surname">Ye</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Shuai Cheng</span> <span class="nlm-surname">Li</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.04.05.026146; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.04.05.026146 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.04.05.026146v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.04.05.026146v1&amp;title=CovProfile%3A%20profiling%20the%20viral%20genome%20and%20gene%20expressions%20of%20SARS-COV2" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.05.026146v1&amp;title=CovProfile%3A%20profiling%20the%20viral%20genome%20and%20gene%20expressions%20of%20SARS-COV2" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.05.026146v1&amp;text=CovProfile%3A%20profiling%20the%20viral%20genome%20and%20gene%20expressions%20of%20SARS-COV2" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.05.026146v1&amp;title=CovProfile%3A%20profiling%20the%20viral%20genome%20and%20gene%20expressions%20of%20SARS-COV2" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.04.05.026146v1&amp;t=CovProfile%3A%20profiling%20the%20viral%20genome%20and%20gene%20expressions%20of%20SARS-COV2" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.04.05.026146v1&amp;title=CovProfile%3A%20profiling%20the%20viral%20genome%20and%20gene%20expressions%20of%20SARS-COV2" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.05.026146v1&amp;title=CovProfile%3A%20profiling%20the%20viral%20genome%20and%20gene%20expressions%20of%20SARS-COV2" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1231122" id="node1231122--3" data-pisa="biorxiv;2020.04.05.026146v1" data-pisa-master="biorxiv;2020.04.05.026146" data-seqnum="1231122" data-apath="/biorxiv/early/2020/04/06/2020.04.05.026146.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">CovProfile: profiling the viral genome and gene expressions of SARS-COV2</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Yonghan</span> <span class="nlm-surname">Yu</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Zhengtu</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Yinhu</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Le</span> <span class="nlm-surname">Yu</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Wenlong</span> <span class="nlm-surname">Jia</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Feng</span> <span class="nlm-surname">Ye</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Shuai Cheng</span> <span class="nlm-surname">Li</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.04.05.026146; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.04.05.026146 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=CovProfile%3A%20profiling%20the%20viral%20genome%20and%20gene%20expressions%20of%20SARS-COV2&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Yu%2C+Yonghan&amp;rft.au=Li%2C+Zhengtu&amp;rft.au=Li%2C+Yinhu&amp;rft.au=Yu%2C+Le&amp;rft.au=Jia%2C+Wenlong&amp;rft.au=Ye%2C+Feng&amp;rft.au=Li%2C+Shuai+Cheng"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1231122/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1231122/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1231122/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1231122/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1231122/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1231122/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1231122/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1231122/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1231122/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1231122/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1231122/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1231122/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.05.026146v1&amp;count=horizontal&amp;via=&amp;text=CovProfile%3A%20profiling%20the%20viral%20genome%20and%20gene%20expressions%20of%20SARS-COV2&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.04.05.026146v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.04.05.026146v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.04.05.026146v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">Bioinformatics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
