<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/04/08/2020.04.06.028811.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.04.06.028811v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="The Potential Use of Unprocessed Sample for RT-qPCR Detection of COVID-19 without an RNA Extraction Step"/>
  <meta name="DC.Identifier" content="10.1101/2020.04.06.028811"/>
  <meta name="DC.Date" content="2020-04-08"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NoDerivs 4.0 International), CC BY-ND 4.0, as described at http://creativecommons.org/licenses/by-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Quantitative reverse transcription polymerase chain reaction (RT-qPCR) assay is the gold standard recommended to test for acute SARS-CoV-2 infection.[1][1]–[4][2] It has been used by the Centers for Disease Control and Prevention (CDC) and several other companies in their Emergency Use Authorization (EUA) assays. With many PCR-based molecular assays, an extraction step is routinely used as part of the protocol. This step can take up a significant amount of time and labor, especially if the extraction is performed manually. Long assay time, partly caused by slow sample preparation steps, has created a large backlog when testing patient samples suspected of COVID-19. Using flu and RSV clinical specimens, we have collected evidence that the RT-qPCR assay can be performed directly on patient sample material from a nasal swab immersed in virus transport medium (VTM) without an RNA extraction step. We have also used this approach to test for the direct detection of SARS-CoV-2 reference materials spiked in VTM. Our data, while preliminary, suggest that using a few microliters of these untreated samples still can lead to sensitive test results. If RNA extraction steps can be omitted without significantly affecting clinical sensitivity, the turn-around time of COVID-19 tests and the backlog we currently experience can be reduced drastically. Next, we will confirm our findings using patient samples. [1]: #ref-1 [2]: #ref-4"/>
  <meta name="DC.Contributor" content="Arunkumar Arumugam"/>
  <meta name="DC.Contributor" content="Season Wong"/>
  <meta name="article:published_time" content="2020-04-08"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="The Potential Use of Unprocessed Sample for RT-qPCR Detection of COVID-19 without an RNA Extraction Step"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;Quantitative reverse transcription polymerase chain reaction (RT-qPCR) assay is the gold standard recommended to test for acute SARS-CoV-2 infection. It has been used by the Centers for Disease Control and Prevention (CDC) and several other companies in their Emergency Use Authorization (EUA) assays. With many PCR-based molecular assays, an extraction step is routinely used as part of the protocol. This step can take up a significant amount of time and labor, especially if the extraction is performed manually. Long assay time, partly caused by slow sample preparation steps, has created a large backlog when testing patient samples suspected of COVID-19. Using flu and RSV clinical specimens, we have collected evidence that the RT-qPCR assay can be performed directly on patient sample material from a nasal swab immersed in virus transport medium (VTM) without an RNA extraction step. We have also used this approach to test for the direct detection of SARS-CoV-2 reference materials spiked in VTM. Our data, while preliminary, suggest that using a few microliters of these untreated samples still can lead to sensitive test results. If RNA extraction steps can be omitted without significantly affecting clinical sensitivity, the turn-around time of COVID-19 tests and the backlog we currently experience can be reduced drastically. Next, we will confirm our findings using patient samples.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.04.06.028811v1"/>
  <meta name="citation_id" content="2020.04.06.028811v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.04.06.028811v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.06.028811v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.06.028811v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.028811.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.04.06.028811"/>
  <meta name="citation_num_pages" content="10"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.04.06.028811"/>
  <meta name="citation_author" content="Arunkumar Arumugam"/>
  <meta name="citation_author_institution" content="AI Biosciences, Inc., College Station"/>
  <meta name="citation_author" content="Season Wong"/>
  <meta name="citation_author_institution" content="AI Biosciences, Inc., College Station"/>
  <meta name="citation_author_email" content="season.wong@aibiosciences.com"/>
  <meta name="citation_reference" content="Chu, D. K. W. et al. Molecular Diagnosis of a Novel Coronavirus (2019-nCoV) Causing an Outbreak of Pneumonia. Clinical Chemistry, doi:10.1093/clinchem/hvaa029 (2020)."/>
  <meta name="citation_reference" content="Zhu, N. et al. A Novel Coronavirus from Patients with Pneumonia in China, 2019. New England Journal of Medicine 382, 727–733, doi:10.1056/NEJMoa2001017 (2020)."/>
  <meta name="citation_reference" content="Xie, C. et al. Comparison of different samples for 2019 novel coronavirus detection by nucleic acid amplification tests. International Journal of Infectious Diseases, doi:https://doi.org/10.1016/j.ijid.2020.02.050 (2020)."/>
  <meta name="citation_reference" content="Corman, V. M. et al. Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR. Euro Surveill 25, 2000045, doi:10.2807/1560-7917.ES.2020.25.3.2000045 (2020)."/>
  <meta name="citation_reference" content="Zou, L. et al. SARS-CoV-2 Viral Load in Upper Respiratory Specimens of Infected Patients. New England Journal of Medicine, doi:10.1056/NEJMc2001737 (2020)."/>
  <meta name="citation_reference" content="Selvaraju, S. B. &amp;; Selvarangan, R. Evaluation of Three Influenza A and B Real-Time Reverse Transcription-PCR Assays and a New 2009 H1N1 Assay for Detection of Influenza Viruses. Journal of Clinical Microbiology 48, 3870–3875, doi:10.1128/JCM.02464-09 (2010)."/>
  <meta name="citation_reference" content="Fry, A. M. et al. The Burden of Hospitalized Lower Respiratory Tract Infection due to Respiratory Syncytial Virus in Rural Thailand. PLOS ONE 5, e15098, doi:10.1371/journal.pone.0015098 (2010)."/>
  <meta name="citation_reference" content="Centers for Disease, Control Prevention, Division of Viral Diseases, CDC 2019-Novel Coronavirus (2019-nCoV) Real-Time RT-PCR Diagnostic Panel, https://www.fda.gov/media/134922/download (Accessed 4/4/2020)."/>
  <meta name="twitter:title" content="The Potential Use of Unprocessed Sample for RT-qPCR Detection of COVID-19 without an RNA Extraction Step"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Quantitative reverse transcription polymerase chain reaction (RT-qPCR) assay is the gold standard recommended to test for acute SARS-CoV-2 infection.[1][1]–[4][2] It has been used by the Centers for Disease Control and Prevention (CDC) and several other companies in their Emergency Use Authorization (EUA) assays. With many PCR-based molecular assays, an extraction step is routinely used as part of the protocol. This step can take up a significant amount of time and labor, especially if the extraction is performed manually. Long assay time, partly caused by slow sample preparation steps, has created a large backlog when testing patient samples suspected of COVID-19. Using flu and RSV clinical specimens, we have collected evidence that the RT-qPCR assay can be performed directly on patient sample material from a nasal swab immersed in virus transport medium (VTM) without an RNA extraction step. We have also used this approach to test for the direct detection of SARS-CoV-2 reference materials spiked in VTM. Our data, while preliminary, suggest that using a few microliters of these untreated samples still can lead to sensitive test results. If RNA extraction steps can be omitted without significantly affecting clinical sensitivity, the turn-around time of COVID-19 tests and the backlog we currently experience can be reduced drastically. Next, we will confirm our findings using patient samples. [1]: #ref-1 [2]: #ref-4"/>
  <meta name="og-title" property="og:title" content="The Potential Use of Unprocessed Sample for RT-qPCR Detection of COVID-19 without an RNA Extraction Step"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.04.06.028811v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Quantitative reverse transcription polymerase chain reaction (RT-qPCR) assay is the gold standard recommended to test for acute SARS-CoV-2 infection.[1][1]–[4][2] It has been used by the Centers for Disease Control and Prevention (CDC) and several other companies in their Emergency Use Authorization (EUA) assays. With many PCR-based molecular assays, an extraction step is routinely used as part of the protocol. This step can take up a significant amount of time and labor, especially if the extraction is performed manually. Long assay time, partly caused by slow sample preparation steps, has created a large backlog when testing patient samples suspected of COVID-19. Using flu and RSV clinical specimens, we have collected evidence that the RT-qPCR assay can be performed directly on patient sample material from a nasal swab immersed in virus transport medium (VTM) without an RNA extraction step. We have also used this approach to test for the direct detection of SARS-CoV-2 reference materials spiked in VTM. Our data, while preliminary, suggest that using a few microliters of these untreated samples still can lead to sensitive test results. If RNA extraction steps can be omitted without significantly affecting clinical sensitivity, the turn-around time of COVID-19 tests and the backlog we currently experience can be reduced drastically. Next, we will confirm our findings using patient samples. [1]: #ref-1 [2]: #ref-4"/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-04-08"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>The Potential Use of Unprocessed Sample for RT-qPCR Detection of COVID-19 without an RNA Extraction Step | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1235182 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">ABSTRACT</h2>
    <p id="p-2">Quantitative reverse transcription polymerase chain reaction (RT-qPCR) assay is the gold standard recommended to test for acute SARS-CoV-2 infection.
     <sup>
      <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>–
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>
     </sup> It has been used by the Centers for Disease Control and Prevention (CDC) and several other companies in their Emergency Use Authorization (EUA) assays. With many PCR-based molecular assays, an extraction step is routinely used as part of the protocol. This step can take up a significant amount of time and labor, especially if the extraction is performed manually. Long assay time, partly caused by slow sample preparation steps, has created a large backlog when testing patient samples suspected of COVID-19. Using flu and RSV clinical specimens, we have collected evidence that the RT-qPCR assay can be performed directly on patient sample material from a nasal swab immersed in virus transport medium (VTM) without an RNA extraction step. We have also used this approach to test for the direct detection of SARS-CoV-2 reference materials spiked in VTM. Our data, while preliminary, suggest that using a few microliters of these untreated samples still can lead to sensitive test results. If RNA extraction steps can be omitted without significantly affecting clinical sensitivity, the turn-around time of COVID-19 tests and the backlog we currently experience can be reduced drastically. Next, we will confirm our findings using patient samples.
    </p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">RESULTS/DISCUSSION</h2>
    <p id="p-3">The sample preparation step is generally time-consuming, regardless of whether it is done manually or automated. In addition, there is a current shortage of the recommended viral RNA extraction kits needed for the Centers for Disease Control and Prevention (CDC) RT-qPCR assay to diagnose SARS-CoV-2. During a study in developing a rapid protocol for influenza (Inf) and respiratory syncytial virus (RSV) diagnostics, we investigated the feasibility of omitting the sample preparation steps to expedite the test without significantly impacting the test’s sensitivity. Using Inf and RSV clinical specimens, we successfully performed RT-qPCR reactions by simply adding a few microliters of the unprocessed sample in viral transport medium (VTM) directly into the RT-qPCR assay master mix. We then tested the approach using SARS-CoV-2 plasmid and SeraCare AccuPlex reference materials. The data presented below suggest that it is possible to skip the RNA extraction step in COVID-19 testing without a significant drop in assay sensitivity.</p>
    <div id="sec-2" class="subsection">
     <h3>RT-qPCR detection of Influenza and RSV from VTM without an RNA isolation step</h3>
     <p id="p-4">We first tested the feasibility using a very small amount of sample in RT-qPCR reactions by using aerosol generating vials to spray the samples over the uncapped PCR tubes. The material in vials containing influenza A (InfA), influenza B (InfB), or RSV clinical specimens (swabs in VTM) were sprayed into PCR tubes containing primer and probe sets targeting InfA, InfB, RSV and RNaseP (RP) in master mix prior to capping the tubes and performing RT-qPCR. 
      <strong>
       <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a>
      </strong> shows that there were positive PCR signals in the respective tubes that did not have any non-specific amplification. Though the unprocessed, sprayed samples have higher Ct values (36.5, 36.2 and 31 for InfA, InfB, and RSV, respectively) than the corresponding extracted RNA template samples (30.4, 32.6, 29.6 for InfA, InfB, and RSV). It is important to note that we were able to detect low viral load samples (Ct &gt;;30) when less than 1 μL of sample entered the tubes as measured by an analytical balance.
     </p>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.028811/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Direct detection of influenza A, B and RSV using unprocessed samples in VTM. Influenza A, B and RSV clinical specimens were diluted 1/100 times in VTM to mimic low viral load samples and sprayed into a row of four-well PCR tubes each containing InfA, InfB, RSV or RNaseP (RP) primers/probes. InfA, InfB and RSV probes were tagged with FAM dye, and the RP probe was tagged with CalfluorRed 610 (CalRed610) dye. A - NTC B - InfA clinical specimen sprayed, C - InfB clinical specimen sprayed, D-RSV clinical specimen sprayed. Inserts [[#x2013]] Images of PCR tubes taken under blue LED illumination with an orange lens filter to capture the FAM dye[[#x2019]]s fluorescence." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-888568694" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Direct detection of influenza A, B and RSV using unprocessed samples in VTM.&lt;;/span&gt;; Influenza A, B and RSV clinical specimens were diluted 1/100 times in VTM to mimic low viral load samples and sprayed into a row of four-well PCR tubes each containing InfA, InfB, RSV or RNaseP (RP) primers/probes. InfA, InfB and RSV probes were tagged with FAM dye, and the RP probe was tagged with CalfluorRed 610 (CalRed610) dye. &lt;;strong&gt;;A&lt;;/strong&gt;; - NTC &lt;;strong&gt;;B&lt;;/strong&gt;; - InfA clinical specimen sprayed, &lt;;strong&gt;;C&lt;;/strong&gt;; - InfB clinical specimen sprayed, &lt;;strong&gt;;D&lt;;/strong&gt;;-RSV clinical specimen sprayed. Inserts [[#x2013]] Images of PCR tubes taken under blue LED illumination with an orange lens filter to capture the FAM dye[[#x2019]]s fluorescence.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.028811/F1.medium.gif" width="440" height="323"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.028811/F1.medium.gif" width="440" height="323"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.028811/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.028811/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. 1.</span>
       <span class="caption-title">Direct detection of influenza A, B and RSV using unprocessed samples in VTM.</span>
       <p id="p-5" class="first-child">Influenza A, B and RSV clinical specimens were diluted 1/100 times in VTM to mimic low viral load samples and sprayed into a row of four-well PCR tubes each containing InfA, InfB, RSV or RNaseP (RP) primers/probes. InfA, InfB and RSV probes were tagged with FAM dye, and the RP probe was tagged with CalfluorRed 610 (CalRed610) dye. 
        <strong>A</strong> - NTC 
        <strong>B</strong> - InfA clinical specimen sprayed, 
        <strong>C</strong> - InfB clinical specimen sprayed, 
        <strong>D</strong>-RSV clinical specimen sprayed. Inserts – Images of PCR tubes taken under blue LED illumination with an orange lens filter to capture the FAM dye’s fluorescence.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-6">To test the extent of PCR inhibition exerted by flu specimens in VTM, we used clinical samples from Discovery Life Sciences, a Biospecimen repository. Between 0.1 μL to 6 μL of the unprocessed samples were directly spiked into the master mix with InfA primers in a 20 μL reaction mix (
      <strong>
       <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>
      </strong>). An extracted template (in Promega Maxwell device) from the same sample was also amplified along with unprocessed samples. Adding more unprocessed samples (up to 6 μL) improved (reduced) the Ct values. The Ct difference between the extracted template (100 μL input and 100 μL eluate) and the unprocessed sample is minimal (29.3 vs. 30.3, respectively). Adding more than 6 μL of untreated samples has resulted in high Ct (data not shown), meaning the inhibitory effect outweighed the benefits of having more copies of the target in a reaction.
     </p>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.028811/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Using clinical samples without RNA extraction steps did not inhibit the reaction. In a 20-[[#x3BC]]L RT-qPCR run, 0.1 to 6.0 [[#x3BC]]L of InfA positive clinical specimens were directly added. Purified nucleic acid template (4 [[#x3BC]]L) isolated from a Promega Maxwell device was also amplified. The Ct difference between the purified template and the directly spiked clinical samples (i.e., 4 [[#x3BC]]L, 5 [[#x3BC]]L and 6 [[#x3BC]]L) is minimal." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-888568694" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Using clinical samples without RNA extraction steps did not inhibit the reaction.&lt;;/span&gt;; In a 20-[[#x3BC]]L RT-qPCR run, 0.1 to 6.0 [[#x3BC]]L of InfA positive clinical specimens were directly added. Purified nucleic acid template (4 [[#x3BC]]L) isolated from a Promega Maxwell device was also amplified. The Ct difference between the purified template and the directly spiked clinical samples (i.e., 4 [[#x3BC]]L, 5 [[#x3BC]]L and 6 [[#x3BC]]L) is minimal.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.028811/F2.medium.gif" width="440" height="327"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.028811/F2.medium.gif" width="440" height="327"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.028811/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.028811/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. 2.</span>
       <span class="caption-title">Using clinical samples without RNA extraction steps did not inhibit the reaction.</span>
       <p id="p-7" class="first-child">In a 20-μL RT-qPCR run, 0.1 to 6.0 μL of InfA positive clinical specimens were directly added. Purified nucleic acid template (4 μL) isolated from a Promega Maxwell device was also amplified. The Ct difference between the purified template and the directly spiked clinical samples (i.e., 4 μL, 5 μL and 6 μL) is minimal.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-3" class="subsection">
     <h3>RT-qPCR detection of SARS-CoV-2 reference material without an RNA isolation step</h3>
     <p id="p-8">We next tested whether the RNA from SARS-CoV-2 can be detected by directly spiking samples of the non-replicative recombinant virus particles (SeraCare AccuPlex SARS-CoV-2 reference material) in VTM to master mix without an extraction step. The SARS-CoV-2 virus particles were mixed with VTM to get a final concentration of 2,500 copies per mL. Different amounts (2, 4, 6 and 8 μL) of these mock clinical samples were spiked into the master mix containing CDC recommended SARS-CoV-2 RT-qPCR diagnostic panel primers N1, N2 o N3 in 20 μL PCR reactions, though we note that the N3 primers were recently removed by the CDC. Then, 100 μL of AccuPlex SARS-CoV-2 reference material was processed in Promega Maxwell device (using an AS1520 cartridge) and eluted in 100 μL of elution buffer. The extracted template (4 μL) was added to master mix, and RT-qPCR amplification was carried out in a Bio-Rad (CFX-96) thermal cycler along with the reactions with unprocessed samples.</p>
     <p id="p-9">As shown in 
      <strong>
       <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3</a>
      </strong>, the SARS-CoV-2 RNA from directly spiked samples was successfully detected by the RT-qPCR reaction without a nucleic acid extraction step (N1 target shown). Up to 8 μL of VTM and AccuPlex SARS-CoV-2 mix was used in the reaction, which did not exert major inhibition on PCR amplification. The average Ct values are 38.5 (for 2 μL), 37.6 (for 4 μL), 38 (for 6 μL), and 39.1 (for 8 μL), still less than the 40 cycles that is considered to be a common cut-off for positive results. Though the theoretical copy number varied in each of the unprocessed samples (5, 10, 15 and 20 copies in 2, 4, 6 and 8 μL of input, respectively), all of them were detected. In this test, using 4 μL of the spiked sample displayed the lowest average (best) Ct. We speculate that in a 4 μL sample, the number of virus copies and PCR inhibitors in the reaction are balanced to offer maximum amplification efficiency. In comparison, the Ct value of the extracted template (20 copies/reaction) is 35.5, which is 2 cycles lower than the result from 4 μL of untreated sample (10 copies) in the reaction (
      <strong>
       <a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3</a>
      </strong>). We speculate that the SeraCare matrix, containing Tris-buffered saline, glycerol, anti-microbial agents and human proteins, could be the reason for the above noticed PCR inhibition.
     </p>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.028811/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Low concentrations of SARS-CoV-2 reference material in VTM are detected without an RNA extraction step. SARS-CoV-2 non-replicative, inactivated viral particles (5,000 viral particles/mL) were diluted and mixed in an equal amount of VTM to mimic clinical samples (2500 viral particles/mL). 2 [[#x3BC]]L, 4 [[#x3BC]]L, 6 [[#x3BC]]L and 8 [[#x3BC]]L of this sample were directly spiked into a 20 [[#x3BC]]L PCR reaction mix containing SARS-CoV-2, N1 targeting primer and probe set and amplified alongside templates extracted in Promega Maxwell device (100 [[#x3BC]]L input and 100 [[#x3BC]]L eluate). RT-qPCR can detect all the directly spiked samples without any sample preparation samples (n=2)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-888568694" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Low concentrations of SARS-CoV-2 reference material in VTM are detected without an RNA extraction step.&lt;;/span&gt;; SARS-CoV-2 non-replicative, inactivated viral particles (5,000 viral particles/mL) were diluted and mixed in an equal amount of VTM to mimic clinical samples (2500 viral particles/mL). 2 [[#x3BC]]L, 4 [[#x3BC]]L, 6 [[#x3BC]]L and 8 [[#x3BC]]L of this sample were directly spiked into a 20 [[#x3BC]]L PCR reaction mix containing SARS-CoV-2, N1 targeting primer and probe set and amplified alongside templates extracted in Promega Maxwell device (100 [[#x3BC]]L input and 100 [[#x3BC]]L eluate). RT-qPCR can detect all the directly spiked samples without any sample preparation samples (n=2).&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Fig. 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.028811/F3.medium.gif" width="440" height="323"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Fig. 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.028811/F3.medium.gif" width="440" height="323"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.028811/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/08/2020.04.06.028811/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Fig. 3.</span>
       <span class="caption-title">Low concentrations of SARS-CoV-2 reference material in VTM are detected without an RNA extraction step.</span>
       <p id="p-10" class="first-child">SARS-CoV-2 non-replicative, inactivated viral particles (5,000 viral particles/mL) were diluted and mixed in an equal amount of VTM to mimic clinical samples (2500 viral particles/mL). 2 μL, 4 μL, 6 μL and 8 μL of this sample were directly spiked into a 20 μL PCR reaction mix containing SARS-CoV-2, N1 targeting primer and probe set and amplified alongside templates extracted in Promega Maxwell device (100 μL input and 100 μL eluate). RT-qPCR can detect all the directly spiked samples without any sample preparation samples (n=2).</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-11">We also tested VTM mixed SARS-CoV-2 plasmid (purchased from Integrated DNA Technologies, Inc.) using RT-qPCR. This plasmid is used as a positive control for the CDC’s SARS-CoV-2 RT-qPCR assay. The positive control plasmid was mixed with VTM and 4 μL of this mix was used for the RT-qPCR reaction. The RT-qPCR results showed that the Ct values of control (without VTM) and VTM mixed reactions (all containing 400 copies/reaction) were very similar for all three (N1, N2 and N3) SARS-CoV-2 targets (
      <strong>
       <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>
      </strong>). We also tested 40 copies of SARS-CoV-2 plasmid using RT-qPCR, which gave a Ct value of ~38 (data not shown). This means direct spiking of a higher concentration of SARS-CoV-2 genome copies in VTM did not have a major PCR inhibitory effect in the reactions when the target concentration is high (
      <strong>
       <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>
      </strong>). Therefore, it is likely that specimens with higher viral load could also be detected by RT-qPCR without needing an RNA extraction step.
     </p>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1242395/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup last">
          <a href="/highwire/markup/1242395/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 1:</span>
       <span class="caption-title">Primer and probe sequences</span>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="T2" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1242394/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1242394/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1242394" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 2:</span>
       <span class="caption-title">RT-qPCR setup</span>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="T3" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1242393/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/1242393/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/1242393" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 3:</span>
       <span class="caption-title">RT-qPCR detection of SARS-CoV-2 plasmid in VTM.</span>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-12">SARS-CoV-2 plasmid in VTM did not show any PCR inhibition when compared to plasmid alone. The CDC positive control plasmid for SARS-CoV-2 (from IDT) was diluted in VTM or TE buffer (control) to get 100,000 copies per mL. 4 μL of SARS-CoV-2 plasmids in VTM and in TE buffer were added to a 20 μL PCR reaction mix targeting N1, N2 and N3 genes for the detection of SARS-CoV-2. The Ct values of both the control and plasmid in VTM are very similar, indicating that the sample preparation step can be omitted in high viral load samples.</p>
    </div>
   </div>
   <div class="section" id="sec-4">
    <h2 class="">CONCLUSIONS</h2>
    <p id="p-13">Our data using both high (Ct &lt;; 35) and low (Ct &gt;;35) target concentrations suggest that since RT-qPCR is highly sensitive, using raw samples or minimal sample preparation steps might not reduce the test sensitivity as most patients tend to have a higher viral load.
     <sup>
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>
     </sup> Also, the efficiency of extraction methods tends to drop significantly at very low target concentrations when processed through numerous washing steps. Therefore, nucleic acid loss during extraction steps may hurt the limit of detection. While further studies with patient specimens and a higher number of samples are needed to confirm our preliminary results, we report that the use of untreated samples can be a viable option during the COVID-19 pandemic.
    </p>
   </div>
   <div class="section" id="sec-5">
    <h2 class="">MATERIALS AND METHODS</h2>
    <div id="sec-6" class="subsection">
     <h3>Materials and Reagents</h3>
     <p id="p-14">The AccuPlex SARS-CoV-2 reference material kit (Cat. No. 0505-0126) was purchased from SeraCare (Milford, MA). Primer and probe sets for the SARS-CoV-2 RT-qPCR assay (Cat. No. 10006606) were purchased from Integrated DNA Technologies (Coralville, IA). The clinical specimens of Influenza A (DLS16-85584), Influenza B (DLS15-33890 and RSV (KH19-00715) were obtained from Discovery Life Sciences Inc. The TaqPath 1-step multiplex master mix (Cat. No. A28525) was purchased from Thermo Fisher Scientific (Waltham, MA). The viral transport media (Cat. No. R99) used for the dilution and spiking experiments was purchased from Hardy diagnostics (Santa Maria, CA).</p>
    </div>
    <div id="sec-7" class="subsection">
     <h3>Preparation of low viral load clinical specimen mimics</h3>
     <p id="p-15">AccuPlex SARS-CoV-2 reference material from SeraCare containing con-replicative viral particles (5,000 viral particles per mL) was mixed with equal volumes of VTM to get a final concentration of 2,500 viral particles per mL. Each microliter contained ~2.5 genomic material equivalents of SARS-CoV-2.</p>
    </div>
    <div id="sec-8" class="subsection">
     <h3>Preparation of high viral load clinical specimen mimics</h3>
     <p id="p-16">SARS-CoV-2 positive control plasmid (CDC recommended) was obtained from Integrated DNA Technologies (200,000 copies/μL). This was diluted to 10,000 copies/μL in TE buffer and 10 μL of this stock solution was added to 990 μL of VTM to get a final concentration of 100,000 copies/mL. A control was prepared by diluting 10 μL of plasmid stock solution in 990 μL of TE buffer. Both the VTM and TE buffer working solutions (4 μL or 400 copies in each reaction) were added into a 20 μL PCR reaction.</p>
    </div>
    <div id="sec-9" class="subsection">
     <h3>Promega Maxwell extraction</h3>
     <p id="p-17">The Promega Maxwell extraction was performed using AS1520 cartridge (Promega Corporation, Madison, WI). 100 μL of the sample (AccuPlex SARS-CoV-2 or Influenza A/B or RSV) was added to the cartridge and eluted in 100 μL elution buffer provided in the kit.</p>
    </div>
    <div id="sec-10" class="subsection">
     <h3>RT-qPCR reaction setup</h3>
     <p id="p-18">TaqPath 1-step multiplex master mix was used for the RT-qPCR reaction with specific primers and probes tagged with FAM. The final concentration of primers and probe sets for influenza A, B and RSV in the reaction is 250 nM in a 20 μL reaction.
      <sup>
       <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>,
       <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>
      </sup> For the detection of SARS-CoV-2 target genes, N1, N2 and N3, the final concentration of primers are 500 nM and probes are 125 nM as per the CDC protocol
      <sup>
       <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>
      </sup>.
     </p>
    </div>
   </div>
   <div class="section" id="sec-11">
    <h2 class="">ETHICAL STATEMENT</h2>
    <p id="p-19">Specimens were purchased from Discovery Life Science. These de-identified samples are not considered to be Human Subjects.</p>
   </div>
   <div class="section COI-statement" id="sec-12">
    <h2 class="">CONFLICT OF INTEREST</h2>
    <p id="p-20">The authors declare no conflict of interest.</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">REFERENCES</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.06.028811v1.1" data-doi="10.1093/clinchem/hvaa029">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chu</span>, 
          <span class="cit-name-given-names">D. K. W.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Molecular Diagnosis of a Novel Coronavirus (2019-nCoV) Causing an Outbreak of Pneumonia</span>. 
         <abbr class="cit-jnl-abbrev">Clinical Chemistry</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1093/clinchem/hvaa029</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DClinical%2BChemistry%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fclinchem%252Fhvaa029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/clinchem/hvaa029&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.06.028811v1.2" data-doi="10.1056/NEJMoa2001017">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">A Novel Coronavirus from Patients with Pneumonia in China, 2019</span>. 
         <abbr class="cit-jnl-abbrev">New England Journal of Medicine</abbr>
         <span class="cit-vol">382</span>, 
         <span class="cit-fpage">727</span>–
         <span class="cit-lpage">733</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1056/NEJMoa2001017</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa2001017%26rft_id%253Dinfo%253Apmid%252F31978945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1056/NEJMoa2001017&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=31978945&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F08%2F2020.04.06.028811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.06.028811v1.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Xie</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Comparison of different samples for 2019 novel coronavirus detection by nucleic acid amplification tests</span>. 
         <abbr class="cit-jnl-abbrev">International Journal of Infectious Diseases</abbr>, doi:
         <a href="https://doi.org/10.1016/j.ijid.2020.02.050">https://doi.org/10.1016/j.ijid.2020.02.050</a> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.06.028811v1.4" data-doi="10.2807/1560-7917.ES.2020.25.3.2000045">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Corman</span>, 
          <span class="cit-name-given-names">V. M.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR</span>. 
         <abbr class="cit-jnl-abbrev">Euro Surveill</abbr>
         <span class="cit-vol">25</span>, 
         <span class="cit-fpage">2000045</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.2807/1560-7917.ES.2020.25.3.2000045</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEuro%2BSurveill%26rft_id%253Dinfo%253Adoi%252F10.2807%252F1560-7917.ES.2020.25.3.2000045%26rft_id%253Dinfo%253Apmid%252F31992387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.2807/1560-7917.ES.2020.25.3.2000045&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=31992387&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F08%2F2020.04.06.028811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.06.028811v1.5" data-doi="10.1056/NEJMc2001737">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zou</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">SARS-CoV-2 Viral Load in Upper Respiratory Specimens of Infected Patients</span>. 
         <abbr class="cit-jnl-abbrev">New England Journal of Medicine</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1056/NEJMc2001737</span> (
         <span class="cit-pub-date">2020</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMc2001737%26rft_id%253Dinfo%253Apmid%252F32074444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1056/NEJMc2001737&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=32074444&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F08%2F2020.04.06.028811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.06.028811v1.6" data-doi="10.1128/JCM.02464-09">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Selvaraju</span>, 
          <span class="cit-name-given-names">S. B.</span>
         </span> &amp;; 
         <span class="cit-auth">
          <span class="cit-name-surname">Selvarangan</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>
         <span class="cit-article-title">Evaluation of Three Influenza A and B Real-Time Reverse Transcription-PCR Assays and a New 2009 H1N1 Assay for Detection of Influenza Viruses</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Clinical Microbiology</abbr>
         <span class="cit-vol">48</span>, 
         <span class="cit-fpage">3870</span>–
         <span class="cit-lpage">3875</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1128/JCM.02464-09</span> (
         <span class="cit-pub-date">2010</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BMicrobiology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.02464-09%26rft_id%253Dinfo%253Apmid%252F20844230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjEwOiI0OC8xMS8zODcwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDgvMjAyMC4wNC4wNi4wMjg4MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.04.06.028811v1.7" data-doi="10.1371/journal.pone.0015098">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Fry</span>, 
          <span class="cit-name-given-names">A. M.</span>
         </span>
         <span class="cit-etal">et al.</span>
         <span class="cit-article-title">The Burden of Hospitalized Lower Respiratory Tract Infection due to Respiratory Syncytial Virus in Rural Thailand</span>. 
         <abbr class="cit-jnl-abbrev">PLOS ONE</abbr>
         <span class="cit-vol">5</span>, 
         <span class="cit-fpage">e15098</span>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep">, </span>
         <span class="cit-pub-id-scheme">doi:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0015098</span> (
         <span class="cit-pub-date">2010</span>).
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DFry%26rft.auinit1%253DA.%2BM.%26rft.volume%253D5%26rft.issue%253D11%26rft.spage%253De15098%26rft.epage%253De15098%26rft.atitle%253DThe%2Bburden%2Bof%2Bhospitalized%2Blower%2Brespiratory%2Btract%2Binfection%2Bdue%2Bto%2Brespiratory%2Bsyncytial%2Bvirus%2Bin%2Brural%2BThailand.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0015098%26rft_id%253Dinfo%253Apmid%252F21152047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0015098&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21152047&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F08%2F2020.04.06.028811.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-web" id="cit-2020.04.06.028811v1.8">
       <div class="cit-metadata">
        <cite>
         <span class="cit-source">Centers for Disease, Control Prevention, Division of Viral Diseases, CDC 2019-Novel Coronavirus (2019-nCoV) Real-Time RT-PCR Diagnostic Panel</span>, 
         <a href="https://www.fda.gov/media/134922/download">https://www.fda.gov/media/134922/download</a> (Accessed 
         <span class="cit-date-in-citation">4/4/2020</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
