<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/03/29/2020.03.26.009993.atom" />
<meta name="HW.pisa" content="biorxiv;2020.03.26.009993v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Non-invasive surveys of mammalian viruses using environmental DNA" />
<meta name="DC.Identifier" content="10.1101/2020.03.26.009993" />
<meta name="DC.Date" content="2020-03-29" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Environmental DNA (eDNA) and its subdiscipline, invertebrate-derived DNA (iDNA) have been used to survey biodiversity non-invasively [[1][1],[2][2]]. Water is ubiquitous in most ecosystems, and, among invertebrates, terrestrial haematophagous leeches are abundant and can be easily collected in many tropical rainforests [[3][3],[4][4]]. Such non-invasive nucleic acid sources can mitigate difficulties of obtaining wildlife samples, particularly in remote areas or for rare species. Recently, eDNA/iDNA sources have been applied to monitoring specific wildlife pathogens [[5][5],[6][6]]. However, previous studies have focused on known pathogens, whereas most wildlife pathogens are uncharacterized and unknown. Non-invasive approaches to monitoring known and novel pathogens may be of particular benefit in ecosystems prone to viral emergence, many of which occur in areas where invasive sampling is challenging, such as tropical rainforests. Here, we show that both eDNA from natural waterholes, and iDNA from terrestrial haematophagous leeches, can be used to detect unknown viruses circulating in mammalian hosts ([Figure 1][7]). Using a curated set of RNA oligonucleotides based on the ViroChip microarray assay [[7][8]] as baits in a hybridization capture system, multiple mammalian RNA and DNA viruses were detected from both eDNA and iDNA samples. Congruence was found between host DNA assignment and viruses identified in leeches, and between animals observed visiting the waterholes and the viruses detected. Our results demonstrate that eDNA/iDNA samples may represent an effective non-invasive resource for studying wildlife viral diversity. Several of the detected viruses were novel, highlighting the potential of eDNA/iDNA for epidemiological analysis of emerging viruses prior to their emergence.

Highlights 

In brief Alfano, Dayaram, et al. demonstrate that environmental DNA from southeast Asian leech bloodmeals and waterholes from Africa and Mongolia can be used as to detect viruses circulating in wildlife. These nucleic acid sources may represent an effective non-invasive resource for studying wildlife viral diversity and emerging viruses pre-emergence.

 [1]: #ref-1
 [2]: #ref-2
 [3]: #ref-3
 [4]: #ref-4
 [5]: #ref-5
 [6]: #ref-6
 [7]: #F1
 [8]: #ref-7" />
<meta name="DC.Contributor" content="Niccolo Alfano" />
<meta name="DC.Contributor" content="Anisha Dayaram" />
<meta name="DC.Contributor" content="Jan Axtner" />
<meta name="DC.Contributor" content="Kyriakos Tsangaras" />
<meta name="DC.Contributor" content="Marie-Louise Kampmann" />
<meta name="DC.Contributor" content="Azlan Mohamed" />
<meta name="DC.Contributor" content="Seth T. Wong" />
<meta name="DC.Contributor" content="M. Thomas P. Gilbert" />
<meta name="DC.Contributor" content="Andreas Wilting" />
<meta name="DC.Contributor" content="Alex D. Greenwood" />
<meta name="article:published_time" content="2020-03-29" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Non-invasive surveys of mammalian viruses using environmental DNA" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Environmental DNA (eDNA) and its subdiscipline, invertebrate-derived DNA (iDNA) have been used to survey biodiversity non-invasively [1,2]. Water is ubiquitous in most ecosystems, and, among invertebrates, terrestrial haematophagous leeches are abundant and can be easily collected in many tropical rainforests [3,4]. Such non-invasive nucleic acid sources can mitigate difficulties of obtaining wildlife samples, particularly in remote areas or for rare species. Recently, eDNA/iDNA sources have been applied to monitoring specific wildlife pathogens [5,6]. However, previous studies have focused on known pathogens, whereas most wildlife pathogens are uncharacterized and unknown. Non-invasive approaches to monitoring known and novel pathogens may be of particular benefit in ecosystems prone to viral emergence, many of which occur in areas where invasive sampling is challenging, such as tropical rainforests. Here, we show that both eDNA from natural waterholes, and iDNA from terrestrial haematophagous leeches, can be used to detect unknown viruses circulating in mammalian hosts (Figure 1). Using a curated set of RNA oligonucleotides based on the ViroChip microarray assay [7] as baits in a hybridization capture system, multiple mammalian RNA and DNA viruses were detected from both eDNA and iDNA samples. Congruence was found between host DNA assignment and viruses identified in leeches, and between animals observed visiting the waterholes and the viruses detected. Our results demonstrate that eDNA/iDNA samples may represent an effective non-invasive resource for studying wildlife viral diversity. Several of the detected viruses were novel, highlighting the potential of eDNA/iDNA for epidemiological analysis of emerging viruses prior to their emergence.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.03.26.009993v1" />
<meta name="citation_id" content="2020.03.26.009993v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.26.009993v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.26.009993v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.26.009993v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.26.009993.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.03.26.009993" />
<meta name="citation_num_pages" content="18" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.03.26.009993" />
<meta name="citation_author" content="Niccolo Alfano" />
<meta name="citation_author_institution" content="Department of Ecological Dynamics, Leibniz Institute for Zoo and Wildlife Research" />
<meta name="citation_author_institution" content="Department of Biodiversity &amp; Molecular Ecology, Fondazione Edmund Mach, Research and Innovation Centre" />
<meta name="citation_author" content="Anisha Dayaram" />
<meta name="citation_author_institution" content="Department of Wildlife Diseases, Leibniz Institute for Zoo and Wildlife Research" />
<meta name="citation_author_institution" content="Charité-Universitätsmedizin Berlin, corporate member of Freie Universitäts Berlin and Humboldt-Universität of Berlin, Institut für Neurophysiologie" />
<meta name="citation_author" content="Jan Axtner" />
<meta name="citation_author_institution" content="Department of Ecological Dynamics, Leibniz Institute for Zoo and Wildlife Research" />
<meta name="citation_author" content="Kyriakos Tsangaras" />
<meta name="citation_author_institution" content="Department of Translational Genetics, The Cyprus Institute of Neurology and Genetics" />
<meta name="citation_author" content="Marie-Louise Kampmann" />
<meta name="citation_author_institution" content="Department of Ecological Dynamics, Leibniz Institute for Zoo and Wildlife Research" />
<meta name="citation_author_institution" content="Section of Forensic Genetics, Department of Forensic Medicine, Faculty of Health and Medical Sciences, University of Copenhagen" />
<meta name="citation_author" content="Azlan Mohamed" />
<meta name="citation_author_institution" content="Department of Ecological Dynamics, Leibniz Institute for Zoo and Wildlife Research" />
<meta name="citation_author_institution" content="WWF-Malaysia, PJCC" />
<meta name="citation_author" content="Seth T. Wong" />
<meta name="citation_author_institution" content="Department of Ecological Dynamics, Leibniz Institute for Zoo and Wildlife Research" />
<meta name="citation_author" content="M. Thomas P. Gilbert" />
<meta name="citation_author_institution" content="The GLOBE Institute, University of Copenhagen" />
<meta name="citation_author_institution" content="University Museum. NTNU" />
<meta name="citation_author" content="Andreas Wilting" />
<meta name="citation_author_institution" content="Department of Ecological Dynamics, Leibniz Institute for Zoo and Wildlife Research" />
<meta name="citation_author_email" content="wilting@izw-berlin.de" />
<meta name="citation_author_email" content="greenwood@izw-berlin.de" />
<meta name="citation_author" content="Alex D. Greenwood" />
<meta name="citation_author_institution" content="Department of Wildlife Diseases, Leibniz Institute for Zoo and Wildlife Research" />
<meta name="citation_author_institution" content="Department of Veterinary Medicine, Freie Universität Berlin" />
<meta name="citation_author_email" content="wilting@izw-berlin.de" />
<meta name="citation_author_email" content="greenwood@izw-berlin.de" />
<meta name="citation_reference" content="Bohmann, K., Evans, A., Gilbert, M.T.P., Carvalho, G.R., Creer, S., Knapp, M., Douglas, W.Y., and De Bruyn, M. (2014). Environmental DNA for wildlife biology and biodiversity monitoring. Trends Ecol. Evol. 29, 358–367." />
<meta name="citation_reference" content="Thomsen, P.F., and Willerslev, E. (2015). Environmental DNA–An emerging tool in conservation for monitoring past and present biodiversity. Biol. Conserv. 183, 4–18." />
<meta name="citation_reference" content="Seeber, P.A., McEwen, G.K., Löber, U., Förster, D.W., East, M.L., Melzheimer, J., and Greenwood, A.D. (2019). Terrestrial mammal surveillance using hybridization capture of environmental DNA from African waterholes. Mol. Ecol. Resour." />
<meta name="citation_reference" content="Schnell, I.B., Sollmann, R., Calvignac-Spencer, S., Siddall, M.E., Douglas, W.Y., Wilting, A., and Gilbert, M.T.P. (2015). iDNA from terrestrial haematophagous leeches as a wildlife surveying and monitoring tool–prospects, pitfalls and avenues to be developed. Front. Zool. 12, 24." />
<meta name="citation_reference" content="Gogarten, J.F., Düx, A., Mubemba, B., Pléh, K., Hoffmann, C., Mielke, A., Müller-Tiburtius, J., Sachse, A., Wittig, R.M., Calvignac-Spencer, S., et al. (2019). Tropical rainforest flies carrying pathogens form stable associations with social nonhuman primates. Mol. Ecol. 28, 4242–4258." />
<meta name="citation_reference" content="Mosher, B.A., Huyvaert, K.P., Chestnut, T., Kerby, J.L., Madison, J.D., and Bailey, L.L. (2017). Design-and model-based recommendations for detecting and quantifying an amphibian pathogen in environmental samples. Ecol. Evol. 7, 10952–10962." />
<meta name="citation_reference" content="Chen, E.C., Miller, S.A., DeRisi, J.L., and Chiu, C.Y. (2011). Using a Pan-Viral Microarray Assay (Virochip) to Screen Clinical Samples for Viral Pathogens." />
<meta name="citation_reference" content="Kampmann, M.-L., Schnell, I.B., Jensen, R.H., Axtner, J., Sander, A.F., Hansen, A.J., Bertelsen, M.F., Greenwood, A.D., Gilbert, M.T.P., and Wilting, A. (2017). Leeches as a source of mammalian viral DNA and RNA—a study in medicinal leeches. Eur. J. Wildl. Res. 63, 36." />
<meta name="citation_reference" content="Kaczensky, P., Lkhagvasuren, B., Pereladova, O., Hemami, M., and Bouskila, A. (2015). Equus hemionus. The IUCN Red List of Threatened Species 2015: e. T7951A45171204." />
<meta name="citation_reference" content="Löhr, C.V., Juan-Sallés, C., Rosas-Rosas, A., García, A.P., Garner, M.M., and Teifke, J.P. (2005). Sarcoids in captive zebras (Equus burchellii): association with bovine papillomavirus type 1 infection. J. Zoo Wildl. Med. 36, 74–81." />
<meta name="citation_reference" content="van Dyk, E., Oosthuizen, M.C., Bosman, A.-M., Nel, P.J., Zimmerman, D., and Venter, E.H. (2009). Detection of bovine papillomavirus DNA in sarcoid-affected and healthy free-roaming zebra (Equus zebra) populations in South Africa. J. Virol. Methods 158, 141–151." />
<meta name="citation_reference" content="Johnson, J., Howard, K., Wilson, A., Ward, M., Gilbert, G.L., and Degeling, C. (2019). Public preferences for One Health approaches to emerging infectious diseases: a discrete choice experiment. Soc. Sci. Med. 228, 164–171." />
<meta name="citation_reference" content="Jones, K.E., Patel, N.G., Levy, M.A., Storeygard, A., Balk, D., Gittleman, J.L., and Daszak, P. (2008). Global trends in emerging infectious diseases. Nature 451, 990–993." />
<meta name="citation_reference" content="Pruvot, M., Khammavong, K., Milavong, P., Philavong, C., Reinharz, D., Mayxay, M., Rattanavong, S., Horwood, P., Dussart, P., Douangngeun, B., et al. (2019). Toward a quantification of risks at the nexus of conservation and health: The case of bushmeat markets in Lao PDR. Sci. Total Environ. 676, 732–745." />
<meta name="citation_reference" content="Swift, L., Hunter, P.R., Lees, A.C., and Bell, D.J. (2007). Wildlife trade and the emergence of infectious diseases. EcoHealth 4, 25." />
<meta name="citation_reference" content="Zhou, P., Yang, X.-L., Wang, X.-G., Hu, B., Zhang, L., Zhang, W., Si, H.-R., Zhu, Y., Li, B., Huang, C.-L., et al. (2020). A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature, 1–4." />
<meta name="citation_reference" content="Peeri, N.C., Shrestha, N., Rahman, M.S., Zaki, R., Tan, Z., Bibi, S., Baghbanzadeh, M., Aghamohammadi, N., Zhang, W., and Haque, U. (2020). The SARS, MERS and novel coronavirus (COVID-19) epidemics, the newest and biggest global health threats: what lessons have we learned? Int. J. Epidemiol." />
<meta name="citation_reference" content="Drosten, C., Günther, S., Preiser, W., Van Der Werf, S., Brodt, H.-R., Becker, S., Rabenau, H., Panning, M., Kolesnikova, L., Fouchier, R.A., et al. (2003). Identification of a novel coronavirus in patients with severe acute respiratory syndrome. N. Engl. J. Med. 348, 1967–1976." />
<meta name="citation_reference" content="Baize, S., Pannetier, D., Oestereich, L., Rieger, T., Koivogui, L., Magassouba, N., Soropogui, B., Sow, M.S., Keïta, S., De Clerck, H., et al. (2014). Emergence of Zaire Ebola virus disease in Guinea. N. Engl. J. Med. 371, 1418–1425." />
<meta name="citation_reference" content="Sharp, P.M., and Hahn, B.H. (2011). Origins of HIV and the AIDS pandemic. Cold Spring Harb. Perspect. Med. 1, a006841." />
<meta name="citation_reference" content="Hoye, B.J., Munster, V.J., Nishiura, H., Klaassen, M., and Fouchier, R.A. (2010). Surveillance of wild birds for avian influenza virus. Emerg. Infect. Dis. 16, 1827." />
<meta name="citation_reference" content="Wilking, H., Ziller, M., Staubach, C., Globig, A., Harder, T.C., and Conraths, F.J. (2009). Chances and limitations of wild bird monitoring for the avian influenza virus H5N1—detection of pathogens highly mobile in time and space. PLoS One 4." />
<meta name="citation_reference" content="Śmietanka, K., Woźniakowski, G., Kozak, E., Niemczuk, K., Frączyk, M., Bocian, \Lukasz, Kowalczyk, A., and Pejsak, Z. (2016). African swine fever epidemic, Poland, 2014-2015. Emerg. Infect. Dis. 22, 1201." />
<meta name="citation_reference" content="Bitome-Essono, P.-Y., Ollomo, B., Arnathau, C., Durand, P., Mokoudoum, N.D., Yacka-Mouele, L., Okouga, A.-P., Boundenga, L., Mve-Ondo, B., Obame-Nkoghe, J., et al. (2017). Tracking zoonotic pathogens using blood-sucking flies as’ flying syringes’. elife 6, e22069." />
<meta name="citation_reference" content="Nasi, R., Taber, A., and Van Vliet, N. (2011). Empty forests, empty stomachs? Bushmeat and livelihoods in the Congo and Amazon Basins. Int. For. Rev. 13, 355–368." />
<meta name="citation_reference" content="Schnell, I.B., Bohmann, K., Schultze, S.E., Richter, S.R., Murray, D.C., Sinding, M.-H.S., Bass, D., Cadle, J.E., Campbell, M.J., Dolch, R., et al. (2018). Debugging diversity–a pan-continental exploration of the potential of terrestrial blood-feeding leeches as a vertebrate monitoring tool. Mol. Ecol. Resour. 18, 1282–1298." />
<meta name="citation_reference" content="Abrams, J.F., Hörig, L.A., Brozovic, R., Axtner, J., Crampton-Platt, A., Mohamed, A., Wong, S.T., Sollmann, R., Yu, D.W., and Wilting, A. (2019). Shifting up a gear with iDNA: From mammal detection events to standardised surveys. J. Appl. Ecol. 56, 1637–1648." />
<meta name="citation_reference" content="Ng, T.F.F., Willner, D.L., Lim, Y.W., Schmieder, R., Chau, B., Nilsson, C., Anthony, S., Ruan, Y., Rohwer, F., and Breitbart, M. (2011). Broad surveys of DNA viral diversity obtained through viral metagenomics of mosquitoes. PloS One 6." />
<meta name="citation_reference" content="Hoffmann, C., Stockhausen, M., Merkel, K., Calvignac-Spencer, S., and Leendertz, F.H. (2016). Assessing the feasibility of fly based surveillance of wildlife infectious diseases. Sci. Rep. 6, 1–9." />
<meta name="citation_reference" content="Farkas, K., McDonald, J.E., Malham, S.K., and Jones, D.L. (2018). Two-step concentration of complex water samples for the detection of viruses. Methods Protoc. 1, 35." />
<meta name="citation_reference" content="Axtner, J., Crampton-Platt, A., Hörig, L.A., Mohamed, A., Xu, C.C., Yu, D.W., and Wilting, A. (2019). An efficient and robust laboratory workflow and tetrapod database for larger scale environmental DNA studies. GigaScience 8, giz029." />
<meta name="citation_reference" content="Meyer, M., and Kircher, M. (2010). Illumina sequencing library preparation for highly multiplexed target capture and sequencing. Cold Spring Harb. Protoc. 2010, pdb–prot5448." />
<meta name="citation_reference" content="Alfano, N., Courtiol, A., Vielgrader, H., Timms, P., Roca, A.L., and Greenwood, A.D. (2015). Variation in koala microbiomes within and between individuals: effect of body region and captivity status. Sci. Rep. 5, 10189." />
<meta name="citation_reference" content="Wang, D., Coscoy, L., Zylberberg, M., Avila, P.C., Boushey, H.A., Ganem, D., and DeRisi, J.L. (2002). Microarray-based detection and genotyping of viral pathogens. Proc. Natl. Acad. Sci. 99, 15687–15692." />
<meta name="citation_reference" content="Yozwiak, N.L., Skewes-Cox, P., Stenglein, M.D., Balmaseda, A., Harris, E., and DeRisi, J.L. (2012). Virus identification in unknown tropical febrile illness cases using deep sequencing. PLoS Negl. Trop. Dis. 6." />
<meta name="citation_reference" content="Edgar, R., Domrachev, M., and Lash, A.E. (2002). Gene Expression Omnibus: NCBI gene expression and hybridization array data repository. Nucleic Acids Res. 30, 207–210." />
<meta name="citation_reference" content="Langmead, B., and Salzberg, S. (2012). Fast gapped-read alignment with bowtie 2 Nat Methods 9 (4): 357–359. pmid: 22388286 View Article PubMed (NCBI)." />
<meta name="citation_reference" content="Kopylova, E., Noé, L., and Touzet, H. (2012). SortMeRNA: fast and accurate filtering of ribosomal RNAs in metatranscriptomic data. Bioinformatics 28, 3211–3217." />
<meta name="citation_reference" content="Bankevich, A., Nurk, S., Antipov, D., Gurevich, A.A., Dvorkin, M., Kulikov, A.S., Lesin, V.M., Nikolenko, S.I., Pham, S., Prjibelski, A.D., et al. (2012). SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J. Comput. Biol. 19, 455–477." />
<meta name="citation_reference" content="Grabherr, M.G., Haas, B.J., Yassour, M., Levin, J.Z., Thompson, D.A., Amit, I., Adiconis, X., Fan, L., Raychowdhury, R., Zeng, Q., et al. (2011). Trinity: reconstructing a full-length transcriptome without a genome from RNA-Seq data. Nat. Biotechnol. 29, 644." />
<meta name="citation_reference" content="Edgar, R.C. (2010). Search and clustering orders of magnitude faster than BLAST. Bioinformatics 26, 2460–2461." />
<meta name="citation_reference" content="Vilsker, M., Moosa, Y., Nooij, S., Fonseca, V., Ghysens, Y., Dumon, K., Pauwels, R., Alcantara, L.C., Vanden Eynden, E., Vandamme, A.-M., et al. (2019). Genome Detective: an automated system for virus identification from high-throughput sequencing data. Bioinformatics 35, 871–873." />
<meta name="citation_reference" content="Kearse, M., Moir, R., Wilson, A., Stones-Havas, S., Cheung, M., Sturrock, S., Buxton, S., Cooper, A., Markowitz, S., Duran, C., et al. (2012). Geneious Basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data. Bioinformatics 28, 1647–1649." />
<meta name="citation_reference" content="Katoh, K., and Standley, D.M. (2013). MAFFT multiple sequence alignment software version 7: improvements in performance and usability. Mol. Biol. Evol. 30, 772–780." />
<meta name="citation_reference" content="Stamatakis, A. (2014). RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. Bioinformatics 30, 1312–1313." />
<meta name="citation_reference" content="Somervuo, P., Koskela, S., Pennanen, J., Henrik Nilsson, R., and Ovaskainen, O. (2016). Unbiased probabilistic taxonomic classification for DNA barcoding. Bioinformatics 32, 2920–2927." />
<meta name="citation_reference" content="Dayaram, A., Franz, M., Schattschneider, A., Damiani, A.M., Bischofberger, S., Osterrieder, N., and Greenwood, A.D. (2017). Long term stability and infectivity of herpesviruses in water. Sci. Rep. 7, 46559." />
<meta name="citation_reference" content="Palmarini, M., Datta, S., Omid, R., Murgia, C., and Fan, H. (2000). The long terminal repeat of Jaagsiekte sheep retrovirus is preferentially active in differentiated epithelial cells of the lungs. J. Virol. 74, 5776–5787." />
<meta name="twitter:title" content="Non-invasive surveys of mammalian viruses using environmental DNA" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Environmental DNA (eDNA) and its subdiscipline, invertebrate-derived DNA (iDNA) have been used to survey biodiversity non-invasively [[1][1],[2][2]]. Water is ubiquitous in most ecosystems, and, among invertebrates, terrestrial haematophagous leeches are abundant and can be easily collected in many tropical rainforests [[3][3],[4][4]]. Such non-invasive nucleic acid sources can mitigate difficulties of obtaining wildlife samples, particularly in remote areas or for rare species. Recently, eDNA/iDNA sources have been applied to monitoring specific wildlife pathogens [[5][5],[6][6]]. However, previous studies have focused on known pathogens, whereas most wildlife pathogens are uncharacterized and unknown. Non-invasive approaches to monitoring known and novel pathogens may be of particular benefit in ecosystems prone to viral emergence, many of which occur in areas where invasive sampling is challenging, such as tropical rainforests. Here, we show that both eDNA from natural waterholes, and iDNA from terrestrial haematophagous leeches, can be used to detect unknown viruses circulating in mammalian hosts ([Figure 1][7]). Using a curated set of RNA oligonucleotides based on the ViroChip microarray assay [[7][8]] as baits in a hybridization capture system, multiple mammalian RNA and DNA viruses were detected from both eDNA and iDNA samples. Congruence was found between host DNA assignment and viruses identified in leeches, and between animals observed visiting the waterholes and the viruses detected. Our results demonstrate that eDNA/iDNA samples may represent an effective non-invasive resource for studying wildlife viral diversity. Several of the detected viruses were novel, highlighting the potential of eDNA/iDNA for epidemiological analysis of emerging viruses prior to their emergence.

Highlights 

In brief Alfano, Dayaram, et al. demonstrate that environmental DNA from southeast Asian leech bloodmeals and waterholes from Africa and Mongolia can be used as to detect viruses circulating in wildlife. These nucleic acid sources may represent an effective non-invasive resource for studying wildlife viral diversity and emerging viruses pre-emergence.

 [1]: #ref-1
 [2]: #ref-2
 [3]: #ref-3
 [4]: #ref-4
 [5]: #ref-5
 [6]: #ref-6
 [7]: #F1
 [8]: #ref-7" />
<meta name="og-title" property="og:title" content="Non-invasive surveys of mammalian viruses using environmental DNA" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.26.009993v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Environmental DNA (eDNA) and its subdiscipline, invertebrate-derived DNA (iDNA) have been used to survey biodiversity non-invasively [[1][1],[2][2]]. Water is ubiquitous in most ecosystems, and, among invertebrates, terrestrial haematophagous leeches are abundant and can be easily collected in many tropical rainforests [[3][3],[4][4]]. Such non-invasive nucleic acid sources can mitigate difficulties of obtaining wildlife samples, particularly in remote areas or for rare species. Recently, eDNA/iDNA sources have been applied to monitoring specific wildlife pathogens [[5][5],[6][6]]. However, previous studies have focused on known pathogens, whereas most wildlife pathogens are uncharacterized and unknown. Non-invasive approaches to monitoring known and novel pathogens may be of particular benefit in ecosystems prone to viral emergence, many of which occur in areas where invasive sampling is challenging, such as tropical rainforests. Here, we show that both eDNA from natural waterholes, and iDNA from terrestrial haematophagous leeches, can be used to detect unknown viruses circulating in mammalian hosts ([Figure 1][7]). Using a curated set of RNA oligonucleotides based on the ViroChip microarray assay [[7][8]] as baits in a hybridization capture system, multiple mammalian RNA and DNA viruses were detected from both eDNA and iDNA samples. Congruence was found between host DNA assignment and viruses identified in leeches, and between animals observed visiting the waterholes and the viruses detected. Our results demonstrate that eDNA/iDNA samples may represent an effective non-invasive resource for studying wildlife viral diversity. Several of the detected viruses were novel, highlighting the potential of eDNA/iDNA for epidemiological analysis of emerging viruses prior to their emergence.

Highlights 

In brief Alfano, Dayaram, et al. demonstrate that environmental DNA from southeast Asian leech bloodmeals and waterholes from Africa and Mongolia can be used as to detect viruses circulating in wildlife. These nucleic acid sources may represent an effective non-invasive resource for studying wildlife viral diversity and emerging viruses pre-emergence.

 [1]: #ref-1
 [2]: #ref-2
 [3]: #ref-3
 [4]: #ref-4
 [5]: #ref-5
 [6]: #ref-6
 [7]: #F1
 [8]: #ref-7" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-03-29" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.03.26.009993v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.03.26.009993v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.03.26.009993v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.03.26.009993v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1216079" />
    <title>Non-invasive surveys of mammalian viruses using environmental DNA | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Sj5Os4QTSjCSP4O3COJuEkG7F9crUrF8aBujKgas6gk","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1216079","apath":"\/biorxiv\/early\/2020\/03\/29\/2020.03.26.009993.atom","pisa":"biorxiv;2020.03.26.009993v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.03.26.009993v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.03.26.009993v1","title":"Non-invasive surveys of mammalian viruses using environmental DNA","identifier":"node\/1216079"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022DRdFmK6MCsm5AM7cgKGu7ijd51Xpcg37AC6pFBmL+FW4ImQ+K5jmGefeVBq2EFv7V7Rk63+nXbP77RWN8dsjjmgk7mscIHDgU1VqjXQEl5+TZr3d+3unjjIkL97kJ9zaUYyxk4EfSbVbKT9kv+4ieK6lFW6oMKmy4UwC2gsl+eJFHYZGtIv9OZ\\\\\\\/J9FEKU4NR74PsfjOWRPt4qYo4ndByqZGLgKoa88zzDxK6Lh9xRSuWbSjnj56zdzxMyy6Lo1kBJ4nLIGRENK63Wu81tEoY+DvRkWmAI2U+olu0vTisDfOuOG0wKrdNUpHZAwHYo\\\\\\\/qSmROAd5KvmcdTIH5YvhAaFo7QKZMvDo7o+M2eaiTRME+RGnAtIb7Qo4D8TYwNZZvClDscpNrvyfFw6OyDVEodZJVxMgERN5cnhm26Fckqm5hnvm2rFx\\\\\\\/os9eziLQ1N1w1+qOnQliundcEeNIDLz4JbGzL81qZHDZO9o5mdKtAaTCtfBirgls0bbn12Ripdj\\\\\\\/DCgb4wRBO\\\\\\\/skXInqZAgPmab2YfEHBglIFKpoQ3ypFSboDb96ZB8Od1y0AM\\\\\\\/doZfprxvgQnxWbMouQyzIDmSeH3k4dq8tJWmIiKqSsPxu+RP3C\\\\\\\/+XFx\\\\\\\/xUa5edSoPyUF+yUTon4H0T6fT0QRsUztmX2jHbc7zl\\\\\\\/RQA70YCTh0PAmjkoTaj84UtmsKl92AF93g\\\\\\\/GL\\\\\\\/ZVEtuyIbDafkAfMfyog45tyXluu4OPwO0PfAKgYMIT4KT8\\\\\\\/7uGFctlIZuz3r47DFlyqPM8iv0VcG85kavU0S4OjrgKLjYQi\\\\\\\/3vdJbrP7KF+t9ulumxmDo04B1WyEvwWKYTXBvNhHwqF2FVmusXwYTRBR7O5KMQKRNzCUbq2cmmNyO60yQMnDb9s6jI9ucv2p3p069uWEi6oAJwoFIbf86bv+YMwP+WNtNvz7UpHFMVa88sRUXnJzrxrxuAkR9Q9bCMmfoJezA9qpxBlE8yG8ISAvDJyAP8TyOJe14qMsKlNkBrKUavDxjLKrtF2w75AcBn13W946jsp9odCTu2ffmAPWVHD7n9lZjIcVF5bHAzOhyPVO2jtq9UL8dfTkF6DOsrs8J1HNwP50YIc40ebX2Ji2h4t6jzU3RRbnH+8vbnlzVQgxAueZ0MCikZbiRXxywnZxYtSMpgfxoW8eXkcrx3XodRreh7wbgCO4hA2m5WiGXOmfT4CMdJDM4ONPT3C\\\\\\\/a78I9rIxQsoAO+BxoBgExvNj\\\\\\\/gJv+0mVIQ6r0USLccPNNiipYXorzxDGLAHWEWQ2\\\\\\\/snfzU\\\\\\\/ZtwSRdk1VNcrBEv1CdBW1hJKQO0UyCqfykaC7ZOYCpFS\\\\\\\/4aPP5GAGw\\\u0022,\\\u0022iv\\\u0022:\\\u0022B5swZ7ra9NGZvT3e3v6qIg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002232cb6b31c8bd854bd9cf8183cc423a448223037f29a97fb49991e636edd36157\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.03.26.009993v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1216079 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.03.26.009993v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-DHwHOwRn6XDs7Vz0supOqseadVJ5dKiqUkqnmKEXgSc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1216079" id="node1216079" data-pisa="biorxiv;2020.03.26.009993v1" data-pisa-master="biorxiv;2020.03.26.009993" data-apath="/biorxiv/early/2020/03/29/2020.03.26.009993.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Non-invasive surveys of mammalian viruses using environmental DNA</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Niccolo</span> <span class="nlm-surname">Alfano</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Anisha</span> <span class="nlm-surname">Dayaram</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jan</span> <span class="nlm-surname">Axtner</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Kyriakos</span> <span class="nlm-surname">Tsangaras</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Marie-Louise</span> <span class="nlm-surname">Kampmann</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Azlan</span> <span class="nlm-surname">Mohamed</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Seth T.</span> <span class="nlm-surname">Wong</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">M. Thomas P.</span> <span class="nlm-surname">Gilbert</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Andreas</span> <span class="nlm-surname">Wilting</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Alex D.</span> <span class="nlm-surname">Greenwood</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.03.26.009993 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1216079" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Niccolo Alfano </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Ecological Dynamics, Leibniz Institute for Zoo and Wildlife Research</span>, Alfred-Kowalke-Str. 17, 10315 Berlin, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Biodiversity &#38; Molecular Ecology, Fondazione Edmund Mach, Research and Innovation Centre</span>, Via Edmund Mach 1, 38010 San Michele All’adige (TN), <span class='nlm-country'>Italy</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Niccolo%2BAlfano%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Alfano%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANiccolo%2BAlfano%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anisha Dayaram </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Wildlife Diseases, Leibniz Institute for Zoo and Wildlife Research</span>, Alfred-Kowalke-Str. 17, 10315 Berlin, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Charité-Universitätsmedizin Berlin, corporate member of Freie Universitäts Berlin and Humboldt-Universität of Berlin, Institut für Neurophysiologie</span>, Berlin, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anisha%2BDayaram%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dayaram%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnisha%2BDayaram%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jan Axtner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Ecological Dynamics, Leibniz Institute for Zoo and Wildlife Research</span>, Alfred-Kowalke-Str. 17, 10315 Berlin, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jan%2BAxtner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Axtner%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJan%2BAxtner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Kyriakos Tsangaras </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Department of Translational Genetics, The Cyprus Institute of Neurology and Genetics</span>, Iroon Avenue 6, Agios Dometios 2371, <span class='nlm-country'>Cyprus</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kyriakos%2BTsangaras%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tsangaras%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKyriakos%2BTsangaras%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Marie-Louise Kampmann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Ecological Dynamics, Leibniz Institute for Zoo and Wildlife Research</span>, Alfred-Kowalke-Str. 17, 10315 Berlin, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>Section of Forensic Genetics, Department of Forensic Medicine, Faculty of Health and Medical Sciences, University of Copenhagen</span>, <span class='nlm-country'>Denmark</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie-Louise%2BKampmann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kampmann%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarie-Louise%2BKampmann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Azlan Mohamed </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Ecological Dynamics, Leibniz Institute for Zoo and Wildlife Research</span>, Alfred-Kowalke-Str. 17, 10315 Berlin, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>7</span><span class='nlm-institution'>WWF-Malaysia, PJCC</span>, 46150 Petaling Jaya, Selangor, <span class='nlm-country'>Malaysia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Azlan%2BMohamed%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mohamed%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAzlan%2BMohamed%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Seth T. Wong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Ecological Dynamics, Leibniz Institute for Zoo and Wildlife Research</span>, Alfred-Kowalke-Str. 17, 10315 Berlin, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Seth%2BT.%2BWong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wong%20ST&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASeth%2BT.%2BWong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">M. Thomas P. Gilbert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>8</span><span class='nlm-institution'>The GLOBE Institute, University of Copenhagen</span>, Øster Farimagsgade 5A, 1352 Copenhagen, <span class='nlm-country'>Denmark</span></div><div class='author-affiliation'><span class='nlm-sup'>9</span><span class='nlm-institution'>University Museum. NTNU</span>, 7491 Trondheim, <span class='nlm-country'>Norway</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BThomas%2BP.%2BGilbert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gilbert%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM.%2BThomas%2BP.%2BGilbert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Andreas Wilting </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Ecological Dynamics, Leibniz Institute for Zoo and Wildlife Research</span>, Alfred-Kowalke-Str. 17, 10315 Berlin, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andreas%2BWilting%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wilting%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndreas%2BWilting%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wilting@izw-berlin.de" class="" data-icon-position="" data-hide-link-title="0">wilting@izw-berlin.de</a>
<a href="mailto:greenwood@izw-berlin.de" class="" data-icon-position="" data-hide-link-title="0">greenwood@izw-berlin.de</a></span></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Alex D. Greenwood </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Department of Wildlife Diseases, Leibniz Institute for Zoo and Wildlife Research</span>, Alfred-Kowalke-Str. 17, 10315 Berlin, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-sup'>10</span><span class='nlm-institution'>Department of Veterinary Medicine, Freie Universität Berlin</span>, Robert-von-Ostertag-Str. 7-13, Berlin 14163, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alex%2BD.%2BGreenwood%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Greenwood%20AD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlex%2BD.%2BGreenwood%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wilting@izw-berlin.de" class="" data-icon-position="" data-hide-link-title="0">wilting@izw-berlin.de</a>
<a href="mailto:greenwood@izw-berlin.de" class="" data-icon-position="" data-hide-link-title="0">greenwood@izw-berlin.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.03.26.009993v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1216079" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1216079/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.26.009993v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1216079" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1216079/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.26.009993v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1216079" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1216079/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.26.009993v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1216079" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1216079/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.03.26.009993v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1216079" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1216079/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.03.26.009993v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1216079" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1216079/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Summary</h2><p id="p-4">Environmental DNA (eDNA) and its subdiscipline, invertebrate-derived DNA (iDNA) have been used to survey biodiversity non-invasively [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>,<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]. Water is ubiquitous in most ecosystems, and, among invertebrates, terrestrial haematophagous leeches are abundant and can be easily collected in many tropical rainforests [<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>,<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>]. Such non-invasive nucleic acid sources can mitigate difficulties of obtaining wildlife samples, particularly in remote areas or for rare species. Recently, eDNA/iDNA sources have been applied to monitoring specific wildlife pathogens [<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>,<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>]. However, previous studies have focused on known pathogens, whereas most wildlife pathogens are uncharacterized and unknown. Non-invasive approaches to monitoring known and novel pathogens may be of particular benefit in ecosystems prone to viral emergence, many of which occur in areas where invasive sampling is challenging, such as tropical rainforests. Here, we show that both eDNA from natural waterholes, and iDNA from terrestrial haematophagous leeches, can be used to detect unknown viruses circulating in mammalian hosts (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Using a curated set of RNA oligonucleotides based on the ViroChip microarray assay [<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>] as baits in a hybridization capture system, multiple mammalian RNA and DNA viruses were detected from both eDNA and iDNA samples. Congruence was found between host DNA assignment and viruses identified in leeches, and between animals observed visiting the waterholes and the viruses detected. Our results demonstrate that eDNA/iDNA samples may represent an effective non-invasive resource for studying wildlife viral diversity. Several of the detected viruses were novel, highlighting the potential of eDNA/iDNA for epidemiological analysis of emerging viruses prior to their emergence.</p><div id="sec-1" class="subsection"><p><strong>Highlights</strong></p><ul class="list-unord " id="list-1"><li id="list-item-1"><p id="p-5">Environmental DNA (water and blood-sucking leeches) provided a non-invasive method of screening wildlife for viruses</p></li><li id="list-item-2"><p id="p-6">A comprehensive viral RNA oligonucleotide bait set was developed to capture known and unknown mammalian virus diversity</p></li><li id="list-item-3"><p id="p-7">Leech blood meal host determination and viruses identified were congruent</p></li><li id="list-item-4"><p id="p-8">Viruses determined from water correlated with known and observed species visiting the water sources</p></li></ul></div><div id="sec-2" class="subsection"><p id="p-9"><strong>In brief</strong> Alfano, Dayaram, et al. demonstrate that environmental DNA from southeast Asian leech bloodmeals and waterholes from Africa and Mongolia can be used as to detect viruses circulating in wildlife. These nucleic acid sources may represent an effective non-invasive resource for studying wildlife viral diversity and emerging viruses pre-emergence.</p></div></div><div class="section" id="sec-3"><h2 class="">Results</h2><div id="sec-4" class="subsection"><h3>Positive and negative controls</h3><p id="p-10">In order to test the sensitivity of the viral capture in recovering vertebrate host viruses, the capture system was first applied to a positive control consisting of medical leeches fed with human blood spiked with two RNA viruses and two DNA viruses at different concentrations [<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>]. All four viruses were detected, even if enrichment efficiency (proportion of on-target viral reads) and target genome recovery varied among viruses (Suppl. Fig. 1). No viral contigs were identified in the negative controls included to monitor laboratory contaminations for either the leech or water experiments.</p></div><div id="sec-5" class="subsection"><h3>Leech viral identification</h3><p id="p-11">Tiger leeches (<em>Haemadipsa picta</em>) and brown leeches (<em>Haemadipsa zeylanica</em>) were collected in Malaysian Borneo and processed as pools (bulk samples) consisting of 1 to 77 individual leeches separated by leech species and sampling location. Viruses were identified in 40 of the 68 leech pools analysed (59%) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>; Suppl. Tab. 1). In 18 of these (45%), two to three viruses were identified. Sequence data from six vertebrate-infecting viral families were detected, including the <em>Anelloviridae</em>, <em>Circoviridae</em>, <em>Coronaviridae</em>, <em>Parvoviridae</em>, <em>Retroviridae</em> and <em>Rhabdoviridae</em>. The most common viral group detected was <em>Rhabdoviridae</em> which was found in 37% of samples (25 samples out of 68), followed by <em>Coronaviridae</em> which was identified in 24% of samples (16 samples). Members of the <em>Anelloviridae</em> were identified in 12% of samples (8 samples), <em>Retroviridae</em> in three samples (4%), and <em>Parvoviridae</em> and <em>Circoviridae</em> in two samples (3%) (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2</a>; Suppl. Tab. 1).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.26.009993/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Viral screening of vertebrate viruses from leech iDNA and waterholes eDNA using RNA olignonucleotide based hybridization capture. In the upper panel the left photo shows a leech feeding on a frog in a rainforest of Vietnam (courtesy Andrew Tilker; Leibniz-IZW) and the right photo shows an African waterhole in Tanzania (courtesy Peter Seeber; Leibniz-IZW). The middle panel depicts the hybridization capture protocol. Briefly, Illumina libraries were produced from reverse transcribed RNA and DNA from leech bloodmeals or from waterhole surface water and sediments. Biotinylated viral RNA baits were hybridized to the libraries and non-target DNA was washed away. The remaining DNA was sequenced, reads assembled into contigs and mapped to reference viral genomes. These contigs were further analyzed to determine viral identity. Viral identity was paired with host identity determined either by mammalian metabarcoding of the leech samples, or by observation of waterhole usage." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1059275928" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Viral screening of vertebrate viruses from leech iDNA and waterholes eDNA using RNA olignonucleotide based hybridization capture. In the upper panel the left photo shows a leech feeding on a frog in a rainforest of Vietnam (courtesy Andrew Tilker; Leibniz-IZW) and the right photo shows an African waterhole in Tanzania (courtesy Peter Seeber; Leibniz-IZW). The middle panel depicts the hybridization capture protocol. Briefly, Illumina libraries were produced from reverse transcribed RNA and DNA from leech bloodmeals or from waterhole surface water and sediments. Biotinylated viral RNA baits were hybridized to the libraries and non-target DNA was washed away. The remaining DNA was sequenced, reads assembled into contigs and mapped to reference viral genomes. These contigs were further analyzed to determine viral identity. Viral identity was paired with host identity determined either by mammalian metabarcoding of the leech samples, or by observation of waterhole usage.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.26.009993/F1.medium.gif" width="440" height="343"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.26.009993/F1.medium.gif" width="440" height="343"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.26.009993/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.26.009993/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <p id="p-12" class="first-child">Viral screening of vertebrate viruses from leech iDNA and waterholes eDNA using RNA olignonucleotide based hybridization capture. In the upper panel the left photo shows a leech feeding on a frog in a rainforest of Vietnam (courtesy Andrew Tilker; Leibniz-IZW) and the right photo shows an African waterhole in Tanzania (courtesy Peter Seeber; Leibniz-IZW). The middle panel depicts the hybridization capture protocol. Briefly, Illumina libraries were produced from reverse transcribed RNA and DNA from leech bloodmeals or from waterhole surface water and sediments. Biotinylated viral RNA baits were hybridized to the libraries and non-target DNA was washed away. The remaining DNA was sequenced, reads assembled into contigs and mapped to reference viral genomes. These contigs were further analyzed to determine viral identity. Viral identity was paired with host identity determined either by mammalian metabarcoding of the leech samples, or by observation of waterhole usage.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.26.009993/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relative abundance of viruses from each family, shown as the percentage of the total number of viral reads in each leech and waterhole sample. In the sample names, S stands for sediment, W for water, T for Tanzania, M for Mongolia and L for leeches. The leech host assignment for each leech sample is shown on the right (see Suppl. Tab. 1 for further details)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1059275928" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Relative abundance of viruses from each family, shown as the percentage of the total number of viral reads in each leech and waterhole sample. In the sample names, S stands for sediment, W for water, T for Tanzania, M for Mongolia and L for leeches. The leech host assignment for each leech sample is shown on the right (see Suppl. Tab. 1 for further details).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.26.009993/F2.medium.gif" width="329" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.26.009993/F2.medium.gif" width="329" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.26.009993/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/29/2020.03.26.009993/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <p id="p-13" class="first-child">Relative abundance of viruses from each family, shown as the percentage of the total number of viral reads in each leech and waterhole sample. In the sample names, S stands for sediment, W for water, T for Tanzania, M for Mongolia and L for leeches. The leech host assignment for each leech sample is shown on the right (see Suppl. Tab. 1 for further details).</p><div class="sb-div caption-clear"></div></div></div><p id="p-14"><em>Rhabdoviridae</em> contigs were genetically similar to three different viral genera (Suppl. Tab. 1). Five contigs were most similar (69-77%) to the Vesicular stomatitis Indiana virus (VSIV) (genus <em>Vesiculovirus</em>) as determined by BLAST searches. The limited similarity of the contigs to known rhabdoviruses suggest they may represent a new genus related to fish rhabodviruses (<em>Perhabdovirus</em> and <em>Sprivirus</em>) or <em>Vesiculovirus</em> (Suppl. Fig. 2). The other contigs clustered phylogenetically, suggesting they represent two new species of a rhabdovirus related to lyssaviruses (Suppl. Fig. 2). Although in most cases one contig per sample was observed, in five samples (L4, L12, L23, L58, L68) two different viruses were found. Several viral regions for <em>Rhabdoviridae</em> were represented in the baits. However, most of the oligonucleotides were specific for the L gene which encodes the RNA-dependent RNA polymerase. All the recovered contigs mapped to the L gene (Suppl. Fig. 3A-C). The viral contig sequences were confirmed by PCR and Sanger sequencing for L55 and L58 (Suppl. Fig. 3D).</p><p id="p-15">All <em>Coronaviridae</em> contigs matched a bat betacoronavirus as determined by BLAST searches with identities between 70-73% (Suppl. Tab. 1). The resulting sequence did not cluster in any of the four clades representing the known <em>Coronaviridae</em> genera, suggesting it may represent a novel coronavirus genus (Suppl. Fig. 4). Each contig overlapped with the coronavirus <em>RNA-dependent RNA polymerase</em> gene (orf1ab), the viral region mainly targeted by the RNA oligonucleotide baits (Suppl. Fig. 5).</p><p id="p-16"><em>Anelloviridae</em> contigs matched either porcine torque teno virus (PTTV) (95-96% identity), a giant panda anellovirus (GpAV) (81-92% identity) or a masked palm civet torque teno virus (Pl-TTV) (83-92% identity) (Suppl. Tab. 1). The PTTV contigs were found in two samples (L8 and L37), while the GpAV and Pl-TTV contigs were detected in six samples. GpAV was the best match in four samples (L7, L17, L36, L39) and Pl-TTV in three (L21, L25, L39). In sample L39 both were identified. Every <em>Anelloviridae</em> contig mapped to the non-coding region of the relative reference genome since all <em>Anelloviridae</em> baits targeted the same untranslated region (Suppl. Fig. 6A, C, E). The non-coding region sequenced is not phylogenetically informative and therefore, phylogenetic analysis could not be performed. Viral contigs were confirmed by PCR and Sanger sequencing for samples L7, L17, L25 and L37 (Suppl. Fig. 6B, D, F).</p><p id="p-17">Three <em>Circoviridae</em> contigs matching a porcine circovirus (PCV) (100% identity) were identified in L7 and L59 (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1</a>; Suppl. Tab. 1). Two non-overlapping but adjacent contigs were retrieved from L7. A single contig overlapping with one of the two contigs determined from L7 was recovered from L59 (Suppl. Fig. 7A). The contigs mapped to the PCV replication protein (Rep), targeted by the <em>Circoviridae</em> baits (Suppl. Fig. 7A). The two overlapping contigs of L7 and L59 were confirmed by PCR and Sanger sequencing (Suppl. Fig. 7B). Since the identity of the contigs with known viral sequences in GenBank was 100%, no phylogenetic analysis was performed.</p><p id="p-18"><em>Parvoviridae</em> contigs with the highest similarity to porcine parvovirus (PPV) were found in L8 (1 contig with 98% identity) and L14 (2 contigs with 74-77% identity) (Suppl. Tab. 1). The contig of L8 clustered within the <em>Tetraparvovirus</em> genus, close to ungulate parvoviruses (porcine, ovine and bovine PV), while the contigs of L14 within the <em>Copiparvovirus</em> genus, close to PPV4 (Suppl. Fig. 8). Two of the three contigs mapped to the <em>replicase</em> gene, while one from L14 mapped to an intergenic region (Suppl. Fig. 7C). Whereas the <em>replicase</em> region of PPV was covered by <em>Parvoviridae</em> baits, the intergenic region was not (Suppl. Fig. 7C). This portion of the virus may have been recovered by other non-<em>Parvoviridae</em> baits targeting that region non-specifically.</p><p id="p-19"><em>Retroviridae</em> contigs similar to the simian and feline foamy virus (<em>Spumaretrovirinae</em> subfamily, 79-82% identity) were detected in three samples (L7, L46, L64) (Suppl. Tab. 1). Phylogenetically the contigs clustered together as a sister group to the feline foamy viruses (<em>Felispumavirus</em> genus), potentially being a new genus within the <em>Spumaretrovirinae</em> (Suppl. Fig. 9). The contigs mapped to the <em>polymerase</em> gene, which the exogenous retrovirus baits were designed to target (Suppl. Fig. 7D).</p></div><div id="sec-6" class="subsection"><h3>Leech bloodmeal host assignments</h3><p id="p-20">The mammalian hosts of the leeches were determined by metabarcoding [<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>]. The bearded pig (<em>Sus barbatus</em>) was identified in samples yielding porcine viruses, such as porcine circovirus (L7), porcine parvovirus (L8) and porcine torque teno virus (L8 and L37). Four leech samples with giant panda anellovirus (L7, L17, L36, L39) sequences yielded sun bear (<em>Helarctos malayanus</em>) sequences. Malay civet (<em>Viverra tangalunga</em>) was identified in one of the three samples (L25) with masked palm civet torque teno virus sequences. Fourteen of the 16 samples with the potentially new coronavirus genus (87.5%) yielded deer sequences, specifically sambar (<em>Rusa unicolor</em>), indicating that the novel coronavirus might be a cervid virus. Similarly, the novel <em>Lyssavirus</em>-like <em>Rhabdoviridae</em> sequences were associated with cervid species (sambar or muntjac) (16 of 22 samples, 73%). However, due to the high prevalence of deer in the samples tested (70%) we could not reject that the occurrence of viruses and deer are independent variables (Chi<sup>2</sup><sub>Coronaviridae</sub> = 1.916, 1 df, p = 0.1663; Chi<sup>2</sup><sub>Rhabdoviridae</sub> = 1.046, 1 df, p = 0.3064).</p></div><div id="sec-7" class="subsection"><h3>Waterhole viral identification</h3><p id="p-21">Five waterholes from Tanzania and six from Mongolia were tested. From each waterhole, one water filtrate and one sediment sample were collected (except for one waterhole where only a sediment sample was collected), for a total of twenty-one samples. Five samples (two water and three sediment samples) in total were positive for viral sequences (23.8%). Four viral families were identified including: <em>Retroviridae, Herpesviridae, Adenoviridae</em> and <em>Papillomaviridae</em>. In filtered water and sediment samples collected from the same waterhole, only one virus per sample was generally identified and in one location (WM20 and SM20) contigs from different viral families were isolated based on sample type. Differences between sediment and water are not unexpected as the sediment likely represents a longer-term accumulation of biomaterial and the water represents more acute contamination at the surface and variable mixing throughout.</p><p id="p-22">Of the 11 water filtrate samples tested, two samples from Mongolia (WM3 and WM20) (18.2%) had viral contigs with 100% identity to the Equid herpesvirus 1 and 3 (EHV-1 and EHV-3). The contig of WM20 mapped to the membrane glycoprotein B, whereas the two contigs of WM3 to the DNA packaging protein and membrane glycoprotein G, all regions covered by the <em>Herpesviridae</em> baits (Suppl. Fig. 10A-D). A nested panherpes PCR targeting the <em>DNA polymerase</em> gene and the resulting Sanger sequences further confirmed EHV presence (Suppl. Fig. 10E). Several equine species including domestic horses inhabit the Gobi Desert [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>], which is consistent with the presence of these viruses.</p><p id="p-23">From the 12 sediment samples tested, two from Mongolia and one from Tanzania yielded viral sequences (25%) representing three viral families including: <em>Retroviridae</em>, <em>Adenoviridae</em> and <em>Papillomaviridae.</em> Mongolian sediment sample SM6 was positive for four contigs mapping to the <em>protease</em> (<em>pro</em>) gene of the Jaagsiekte sheep retrovirus (JSRV) with 100% identity (Suppl. Fig. 11A). JSRV from this sample was further confirmed by PCR (Suppl. Fig. 11A). Mongolia sediment sample SM20 was positive for Equine adenovirus (100% identity) with a contig mapping to a region comprised between the <em>pVI</em> and <em>hexon</em> capsid genes (Suppl. Fig. 11B). Given that multiple equine species are found in the Gobi Desert in Mongolia, it is likely that the water sources sampled may have been frequented by these species [<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>]. The sediment sample from Tanzania ST38 was positive for a Zetapapillomavirus related to the <em>Equus caballus</em> papillomavirus and <em>Equus asinus</em> papillomavirus (74% identity; <em>E1</em>-<em>E2</em> genes) (Suppl. Fig. 11C; Suppl. Fig. 12), consistent with the detection of Plains zebra’s (<em>Equus quagga</em>) DNA from this water source [<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>]. Given that both captive and wild zebras have been known to contract bovine papillomaviruses [<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>,<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>] it is likely that they are susceptible to different equine papillomaviruses.</p></div></div><div class="section" id="sec-8"><h2 class="">Discussion</h2><p id="p-24">Emerging infectious viruses increasingly threaten human, domestic animal and wildlife health [<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>]. Sixty percent of emerging infectious diseases in humans are of zoonotic origin [<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>]. Wildlife trade and consumption of bushmeat, especially in Africa and Asia, have increasingly played a role in pathogen spillovers into human populations [<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>,<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>]. Wildlife markets have recently facilitated the spillover of SARS-CoV-2 to humans [<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>] resulting in a pandemic [<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>]. The 2002–2003 SARS-CoV outbreak [<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>], the Ebola outbreak in West Africa [<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>] and the global emergence of HIV [<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>] have all been linked to wildlife trade and bushmeat consumption. Early detection of novel infectious agents in wildlife represent a key factor to prevent their emergence. However, identification, surveillance and monitoring of emerging viruses using currently broadly applied approaches based on direct sampling of wildlife requires enormous investment in sampling, particularly for viruses that have low prevalence [<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>]. For example, 25,000 wild birds were sampled in Germany to detect avian influenza prevalence below 1% [<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>]. Similarly, sampling of over 8,157 animals in Poland was required to determine an 0.12% prevalence of African swine fever virus (ASF) [<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>]. Sampling under remote field conditions or in developing countries present additional challenges.</p><p id="p-25">We provide evidence that environmental and invertebrate-derived DNA samples including waterhole water, sediment and wild haematophagous terrestrial leeches can be used to survey known and unknown viruses. DNA and RNA viruses could be detected in 59% and 20.8% of the iDNA (leech) and eDNA (waterhole) samples, respectively. The congruence of host DNA assignment for leeches and viral families identified suggests that bloodmeals are a useful resource for determining viral diversity. Similarly, the detection of primarily equine viruses from African and Mongolian waterholes, where intense wild equid visitation rates were directly observed, suggests eDNA viruses from this resource reflect host utilization of the water and do not derive from other environmental sources such as fomites distributed over long distances.</p><p id="p-26">PCR based approaches, as used in earlier studies to detect pathogens from flies [<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>,<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>] or, under laboratory conditions, in medicinal leeches [<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>], require prior knowledge about the expected pathogens in the samples. The unknown viral diversity in the wild, and the potential degradation of viral nucleic acids in bloodmeals or in the environment, may affect detection by PCR resulting in high false negative rates. RNA oligonucleotide based hybridization capture overcomes such limitations because the short baits can capture divergent and degraded DNA. The comprehensive viral group representation in the RNA bait set also allows for the determination of both viral presence and viral diversity with a relatively simple workflow. The ability of oligonucleotides with substantial divergence from the target sequence to capture more distantly related sequences is particularly useful in virology since most viruses are uncharacterized in wildlife and many evolve rapidly.</p><p id="p-27">Using short RNA baits to capture highly conserved sequences from every known vertebrate viral genome is a useful and relatively inexpensive approach for providing an initial viral identification. However, to fully characterize each virus, the RNA oligonucleotide bait set would need to be customized to retrieve full length viral genomes. Initial screening with full length genomes for all viruses is costly and may result in detection of host DNA in cases of spurious homology between viruses and host DNA sequence.</p><p id="p-28">Several novel viruses were identified with our short RNA bait approach, which is not unexpected as little is known about the virology of wildlife in Southeast Asia, where the leeches were collected. Several viral contigs were phylogenetically distinct from known viruses and may represent new genera. For example, the novel coronavirus identified in leech bloodmeals did not cluster with any of the known <em>Coronaviridae</em> clades. This finding highlights the ability of this method to detect unknown viruses. We could also associate the novel corona- and rhabdoviruses with mammal bloodmeals with limited evidence of a cervid association for both. Cervids are regularly sold as bushmeat in wildlife markets [<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>] and both recent coronavirus epidemics (SARS-CoV [<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>] and SARS-Cov-2 [<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>]) spilled over from wildlife. This suggests that eDNA/iDNA-based pathogen surveillance approaches may complement efforts to proactively identify viruses that could potentially spillover to humans or livestock.</p><p id="p-29">The collection of wild haematophagous invertebrates such as leeches or water and sediments has both advantages and disadvantages compared to invasively collected wildlife samples. Large amounts of DNA can be extracted from bloodmeals, in particular when leeches are processed in bulk. We pooled up to 77 leeches and many of our leech bulk samples contained a diverse mix of mammalian DNA. A disadvantage of leeches is that they cannot be found in all environments: for example haematophagous terrestrial species are restricted to tropical rainforests of Asia, Madagascar and Australia [<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>]. In addition, leech feeding biases could influence diversity surveys [<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>,<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>]. However, this disadvantage could be overcome in the future by employing additional invertebrates such mosquitoes [<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>] or carrion flies [<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>]. Waterholes are commonly found in almost all environments. In environments with seasonal water shortages, DNA from animals can become highly concentrated due to many animals utilizing rare water sources. The disadvantages are that the dilution factor of water, depending on water body size, can obscure rare DNA sequences and mixed host species sequences are generally the rule rather than the exception. Further experiments with field filtration and sample concentration such as methods used with pathogen detection in waste water may improve detection rates [<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>].</p><p id="p-30">Environmental DNA and in particular its subdiscipline invertebrate-derived DNA viral hybridization capture may be a useful and economical tool for identifying and characterizing major viral pathogens particularly in difficult to access sampling environments prior to viral emergence. Sampling in environments where direct access to animals is difficult or highly restricted, eDNA and iDNA may be the only option to detect viral pathogens in the wild. The current study suggests this approach will be successful in either complementing or replacing invasive approaches.</p></div><div class="section" id="sec-9"><h2 class="">Star methods</h2><div id="sec-10" class="subsection"><h3>Sample collection</h3><div id="sec-11" class="subsection"><h4>Leeches</h4><p id="p-31">Two types of leeches, tiger leeches (<em>Haemadipsa picta</em>) and brown leeches (<em>Haemadipsa zeylanica</em>) were collected from February to May 2015 in the Deramakot Forest Reserve in Sabah, Malaysian Borneo as described in <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Abrams et al. 2019</a> [<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">27</a>] <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Axtner and et al. 2019</a> [<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">31</a>]. All leeches of the same type (tiger or brown) from the same site and occasion were pooled and processed as one sample. Number of leeches ranged from 1 to 77 per pool (median= 7). Samples were stored in RNA fixating saturated ammonium sulfate solution and exported under the permit ‘JKM/MBS.1000-2/3 JLD.2 (8)’ issued by the Sabah Biodiversity Council. A total of 68 pools (L1-L68) were selected for viral capture to maximize representation of host wildlife species identified from bloodmeals [<a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">31</a>].</p></div><div id="sec-12" class="subsection"><h4>Sediment and water</h4><p id="p-32">In February, June, July and October 2016 samples were collected from the Serengeti National Park (ca. 2.2° S, 34.8° E) Tanzania from waterholes. In October 2015 samples were collected from South East Gobi (45.5905°N, 107.1596 °E) and in between June – July 2016 samples were collected from Gobi B (45.1882°N, 93.4288°E) in Mongolia. At each waterhole, 50 ml of water was passed through a 0.22 μm Sterivex filter unit (Millipore) using a disposable 50-ml syringe to remove debris from water. In addition, 25 g of the top 1-3 cm of sediment was collected at each waterhole. The samples were stored on ice packs during the respective field trip, and frozen at −20 °C. In total water filtrate and sediment samples were sampled at 12 waterholes, six respectively from Mongolia and Tanzania. For each sample, 32 ml of water filtrate was ultra-centrifuged at 28,000 rpm for 2 hours to pellet DNA and viral particles. The supernatant was then removed, the pellet re-suspended in 1 ml of cold phosphate-buffered saline (PBS) (pH 7.2) (Sigma-Aldrich) and left at 4 °C overnight.</p></div></div><div id="sec-13" class="subsection"><h3>Preparation of samples and nucleic acid extraction</h3><div id="sec-14" class="subsection"><h4>Leeches</h4><p id="p-33">Leeches were cut into small pieces with a new scalpel blade and lysed overnight (≥12 hours) at 55°C in proteinase K and ATL buffer at a ratio of 1:10; 0.2 ml per leech. Total nucleic acids were extracted from leech samples using the DNeasy 96 Blood &amp; Tissue kit (Qiagen) following the manufacturer’s protocol and finally eluted twice with 100μl 1x Tris-EDTA buffer. For more detailed information on laboratory protocol and samples please see <a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">Axtner et al. 2019</a> and its supporting data (doi: 10.5524/100570).</p></div><div id="sec-15" class="subsection"><h4>Water and sediment</h4><p id="p-34">500 μl of the centrifuged filtrate was used to extract viral nucleic acids using the RTP<sup>®</sup> DNA/RNA Virus Mini Kit (Stratec biomedical). The following modifications were made to the original protocol: 400 μl of lysis buffer, 400 μl of binding buffer and 20 μl of proteinase K and carrier RNA were used per sample. Samples were eluted in 60 μl. The NucleoSpin Soil kit (Macherey-Nagel) was used to extract DNA/RNA from sediment. 500 mg of soil was extracted according to the manufactures protocol using an elution volume of 100 μl.</p></div></div><div id="sec-16" class="subsection"><h3>Positive control</h3><p id="p-35">As a positive control medical leeches (<em>Hirudo</em> spp.) were fed human blood spiked with four viruses [<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">8</a>]. Two RNA viruses, influenza A and measles morbillivirus, and two DNA viruses, bovine herpesvirus and human adenovirus were used (see <a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Kampmann et al. 2017</a> [<a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">8</a>] for details).</p></div><div id="sec-17" class="subsection"><h3>Library Preparation</h3><p id="p-36">The RNA was reverse transcribed using SuperScript III and IV (Thermo Fisher Scientific) with random hexamers prior to second-strand synthesis with Klenow fragment (New England Biolabs). The resulting double-stranded cDNA/DNA mix was sheared to an average fragment size of 200 bp using a M220 focused ultrasonicator (Covaris). Sheared product was purified using the ZR-96 DNA Clean &amp; Concentrator-5 kit (Zymo). Dual-indexed Illumina sequencing libraries were constructed as described by <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Meyer and Kircher 2010</a> [<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">32</a>] with the modifications described in <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Alfano et al. 2015</a> [<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">33</a>]. Each library was amplified in three replicate reactions to minimize amplification bias in individual PCRs. The three replicate PCR products for each sample were pooled and purified using the MinElute PCR Purification Kit (Qiagen). Negative control libraries were also prepared from different stages of the experimental process (extraction, reverse transcription, library preparation and index PCR) and indexed separately to monitor any contamination introduced during the experiment. Amplified libraries were quantified using the 2200 TapeStation (Agilent Technologies) on D1000 ScreenTapes.</p></div><div id="sec-18" class="subsection"><h3>RNA oligonucleotide Bait Design</h3><p id="p-37">The targeted sequence capture panel was designed based on the oligonucleotide probes represented on the Virochip microarray [<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>]. The Virochip is a pan-viral DNA microarray comprising the most highly conserved 70 mer sequences from every fully sequenced reference viral genome present in GenBank, which was developed for the rapid identification and characterization of novel viruses and emerging infectious disease. We retrieved the viral oligonucleotides from the 5th generation Virochip (Viro5) [<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>], which are publicly available at NCBI’s Gene Expression Omnibus (GEO) repository [<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>], accession number GPL13323 (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL13323">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL13323</a>). This platform includes ∼17,500 oligonucleotides (70 mer nucleotides) derived from ∼2,000 viral species. We excluded sequences from bacteriophage, plant viruses, viral families infecting only invertebrates and endogenous retroviruses. We included viruses that could have both vertebrate and invertebrate hosts, such as vertebrate viruses with insect vectors. Exogenous retroviruses were represented but murine leukemia viruses (MLVs) were removed. MLVs sequences may interfere with the capture of other viruses, since MLVs can cross enrich endogenous retroviruses which can represent large portions of several vertebrate genomes and mask rarer viral sequences. Control oligonucleotides included in the Virochip, such as those from human genes, yeast intergenic sequences, and human papilloma virus sequences present in HeLa cells were also removed. Ninety-two 70-mer oligonucleotides covering (spaced end-to-end) the entire <em>pol</em> and <em>gB</em> genes of Equine herpesvirus 1 (EHV-1) were included as PCR screening of the water samples indicated they were positive for this virus (data not shown). The resulting 13,532 oligonucleotides were examined for repetitive elements, short repeats, and low complexity regions, which are problematic for probe design and capture, using RepeatMasker. Repetitive motifs were identified in 234 oligonucleotides, which were removed. The final targeted sequence capture panel consisted of 13,298 unique sequences which were synthesized (as a panel of biotinylated RNAs) at MYcroarray (Ann Arbor, USA).</p></div><div id="sec-19" class="subsection"><h3>Viral Enrichment Strategy and Sequencing</h3><p id="p-38">In-solution target enrichment via hybridization-based capture was performed according to the manufacturer’s protocol (MYbaits<sup>®</sup> custom targeted enrichment, MYcroarray, Ann Arbor, USA), with the following modifications for likely partially degraded samples with an expected low target viral content: 50uL Dynabeads<sup>®</sup> M-270 Streptavidin beads (Invitrogen) instead of 30 uL Dynabeads<sup>®</sup> MyOne™ Streptavidin C1 (Invitrogen); hybridization, bead-bait binding, and wash steps temperature set to 60°C; 48 hours hybridization time; 200 ng baits per reaction; 10 μL indexed library inputs. For capture, libraries generated from pooled leeches consisting of more than 16 individuals were captured individually, while libraries generated from pools of fewer individuals were combined to have a comparable number (15-20) of leeches per capture. This was done in order to ensure that each individual leech represented in each library was allocated enough bait for capture. For capture libraries generated from water and sediment samples. Samples were pooled in groups of two. Sediment and water cDNA and DNA were pooled separately. Per pooled sample, 5 μl of baits were used to ensure enough bait for each sample. The enriched libraries were re-amplified using Herculase II Fusion DNA polymerase (Agilent Technologies) with P5 and P7 Illumina library outer primers with the same cycling conditions described in Alfano et al. 2016. The re-amplified enriched libraries were purified using the MinElute PCR Purification Kit (Qiagen), quantified using the 2200 TapeStation (Agilent Technologies) on D1000 ScreenTapes and finally pooled in equimolar amounts for single-read sequencing on two lanes of an Illumina NextSeq 500 with the TG NextSeq<sup>®</sup> 500/550 High Output Kit v2 (300 cycles).</p></div><div id="sec-20" class="subsection"><h3>Data analysis and bioinformatics pipelines</h3><p>A total of 219,580,903 sequence reads 300 bp long were generated (average: 3,181,781 single reads per sample; standard deviation [SD]: 1,481,098) (Suppl. Tab. 1) and sorted by their dual index sequences. Cutadapt v1.16 and Trimmomatic v0.36 were used to remove adapter sequences and low-quality reads using a quality cutoff of 20 and a minimal read length of 30 nt. After trimming, 97% of the sequences were retained. Three different approaches (A, B, C) were used to analyse the viral capture data:
</p><ol class="list-alphaupper " id="list-2"><li id="list-item-5"><p id="p-40">Leech reads were removed from the dataset by alignment to the <em>Helobdella robusta</em> genome v1.0 (assembly GCA_000326865.1), which is the only complete genome of Hirudinea available in GenBank, and all leech sequences from GenBank (4,957 sequences resulting from “Hirudinea” search) using Bowtie2 v2.3.5.1 [<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>]. This left 81% of the original reads (Suppl. Tab. 1). Then, the filtered reads were searched by BLAST against a database generated from the capture bait sequences. The reads which matched with baits were then extracted and screened against the entire NCBI nucleotide database (nt) using BLASTn to find the best viral match. The filtered reads were mapped both to the corresponding bait sequence and the genome sequence of the best hit obtained by BLAST against the complete nt database, in order to generate a consensus sequence. This consensus sequence was again searched against the NCBI nt database using BLASTn to obtain a viral assignment.</p></li><li id="list-item-6"><p id="p-41">Leech reads were removed as in method A. In addition, rRNA reads were removed using SortMeRNA [<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>], leaving 75% of the original reads (Suppl. Tab. 1). The filtered reads were <em>de novo</em> assembled using both Spades v3.11.1 [<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>] and Trinity v2.6.6 [<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>] assemblers. The obtained contigs from Spades and Trinity were pooled and clustered to remove duplicated or highly similar sequences using USEARCH v11.0.667 [<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>] with a 90% threshold identity value. The centroids were then subjected to sequential BLAST searches against the NCBI nucleotide database and NCBI RefSeq viral protein database using BLASTn and BLASTx, respectively.</p></li><li id="list-item-7"><p id="p-42">The adaptor and quality trimmed data were uploaded to Genome Detective [<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>], a web base software that assembles viral genomes from NGS data. The software first groups reads into different buckets based on the proteins similarity to different viral hits. Genome detective then <em>de novo</em> assembles the reads of each bucket creating a longer consensus sequence that is then searched against the NCBI RefSeq viral database using BLASTx and BLASTn algorithms. The results of amino acid and nucleotide search are combined and viral hit is assigned based on the best combined score.</p></li></ol><p id="p-43">Bacteriophages, invertebrate viruses and retroviruses were excluded from subsequent steps, which only focused on eukaryotic, specifically vertebrate viruses. The results of the three methods were compared and the viral contigs obtained were manually inspected. If more than one method generated a contig with the same viral hit, the contigs from each method were compared. If they had the same sequence or were overlapping, the longest contig was selected. The filtered reads were mapped to the viral contigs to calculate the number of viral reads for each virus. Finally, the viral contigs were mapped to the reference genome of the virus corresponding to the best BLAST hit using Geneious v11.0.2 (Biomatters, Inc.) [<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>]. The baits were mapped to the same references to determine the genomic positions targeted by our bait panel for each virus.</p></div><div id="sec-21" class="subsection"><h3>Phylogenetic analyses</h3><p id="p-44">Viral contigs were assigned to viral families according to the best BLAST results. Comprehensive sets of representative sequences from these viral families were retrieved from GenBank and aligned with the contigs using MAFFT v7.450 [<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>]. Phylogenetic analysis was performed using the maximum-likelihood method based on the general time reversible substitution model with among-site rate heterogeneity modelled by the Γ distribution and estimation of proportion of invariable sites available in RAxML v8 [<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>], including 500 bootstrap replicates to determine node support. Phylogenetic analyses were performed only on viral contigs i) showing divergence from known viruses, i.e. with both BLAST identity and coverage to the best reference below 95%, to place them into a phylogenetic context, and ii) mapping to phylogenetically relevant genomic regions. Therefore, <em>Circoviridae</em> and <em>Anelloviridae</em> contigs were excluded as were those identified from water.</p></div><div id="sec-22" class="subsection"><h3>Leech vertebrate host assignments</h3><p id="p-45">Host identification of leeches followed an eDNA/iDNA workflow recently published [<a id="xref-ref-31-6" class="xref-bibr" href="#ref-31">31</a>]. In summary, leech samples were digested and short fragments of the mitochondrial markers 12S, 16S and cytochrome B were amplified in four PCR replicates each resulting in 12 PCR replicates per sample. We used a twin-tagging 2-step PCR protocol and PCR products were sequenced using an Illumina MiSeq (for details please see <a id="xref-ref-31-7" class="xref-bibr" href="#ref-31">Axtner et al. 2019</a> [<a id="xref-ref-31-8" class="xref-bibr" href="#ref-31">31</a>]). After demultiplexing and read processing, each haplotype was taxonomically assigned to a curated reference database using PROTAX [<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>]. Taxonomic assignments followed the criteria of <a id="xref-ref-31-9" class="xref-bibr" href="#ref-31">Axtner et al. 2019</a>.</p></div><div id="sec-23" class="subsection"><h3>Viral detection confirmation by PCR</h3><p id="p-46">The primers listed in Suppl. Tab. 2 were designed to confirm by PCR the viral contig sequences generated by the three approaches (A, B, C see above) from the leech samples. For PCRs targeting RNA viruses, 50 uL of extract were digested with rDNase I (Ambion) following the manufacturer's protocol. The DNAse-digested extracts were then purified using the RNeasy MinElute Cleanup Kit (Qiagen). RNA was reverse transcribed into cDNA using iScript™ Reverse Transcription Supermix (Bio Rad). Sediment and water samples that tested positive for EHV and JSRV were screened using a previously described pan-herpes PCR [<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>] and for JSRV [<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>], respectively. The resulting amplicons were Sanger sequenced.</p></div></div><div class="section" id="sec-25"><h2 class="">Author Contributions</h2><p id="p-48">ADG and AW designed the study; AM and STW collected the leeches in the field; JA performed the leech nucleic acids extractions and the PCRs on leeches; NA and AD performed the capture experiments; NA and KT performed the bioinformatics and phylogenetic analyses; NA, AD, ADG and AW wrote the manuscript; NA, AD, JA, KT, MLK, TPG, AW and ADG reviewed the manuscript; all authors read and approved the final manuscript.</p></div><div class="section COI-statement" id="sec-26"><h2 class="">Declaration of Interests</h2><p id="p-49">The authors declare no competing interests.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-47">We would like to thank Joseph DeRisi for guidance in using his oligonucleotide data. This project received financial support from the German Federal Ministry of Education and Research (BMBF FKZ: 01LN1301A) to NA, JA, AM and AW and was supported by funds from the Leibniz Gemeinschaft, SAW-2015-IZW-1 440 to AD and ADG. We thank Peter Seeber and Sanatana-Erini Soilemetzidou for collection of water and sediment samples in Africa and Mongolia respectively. We thank the Sabah Forestry Department, especially Johnny Kissing, Peter Lagan, and Datuk Sam Mannan, for supporting the fieldwork and the Sabah Biodiversity Council for providing research, collection, and export permits (JKM/MBS.1000-2/3 JLD.2) for the leech work.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-other" id="fn-1"><p id="p-1"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label">11</span> Lead contacts</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.1" data-doi="10.1016/j.tree.2014.04.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bohmann</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Evans</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gilbert</span>,  <span class="cit-name-given-names">M.T.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carvalho</span>,  <span class="cit-name-given-names">G.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Creer</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knapp</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Douglas</span>,  <span class="cit-name-given-names">W.Y.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">De Bruyn</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Environmental DNA for wildlife biology and biodiversity monitoring</span>. <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">29</span>, <span class="cit-fpage">358</span>–<span class="cit-lpage">367</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol.%2BEvol.%26rft.volume%253D29%26rft.spage%253D358%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2014.04.003%26rft_id%253Dinfo%253Apmid%252F24821515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2014.04.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24821515&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.26.009993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000336873800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thomsen</span>,  <span class="cit-name-given-names">P.F.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Willerslev</span>,  <span class="cit-name-given-names">E.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Environmental DNA–An emerging tool in conservation for monitoring past and present biodiversity</span>. <abbr class="cit-jnl-abbrev">Biol. Conserv.</abbr> <span class="cit-vol">183</span>, <span class="cit-fpage">4</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BConserv.%26rft.volume%253D183%26rft.spage%253D4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Seeber</span>,  <span class="cit-name-given-names">P.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McEwen</span>,  <span class="cit-name-given-names">G.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Löber</span>,  <span class="cit-name-given-names">U.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Förster</span>,  <span class="cit-name-given-names">D.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">East</span>,  <span class="cit-name-given-names">M.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Melzheimer</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Greenwood</span>,  <span class="cit-name-given-names">A.D.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Terrestrial mammal surveillance using hybridization capture of environmental DNA from African waterholes</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schnell</span>,  <span class="cit-name-given-names">I.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sollmann</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Calvignac-Spencer</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Siddall</span>,  <span class="cit-name-given-names">M.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Douglas</span>,  <span class="cit-name-given-names">W.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilting</span>,  <span class="cit-name-given-names">A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Gilbert</span>,  <span class="cit-name-given-names">M.T.P.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">iDNA from terrestrial haematophagous leeches as a wildlife surveying and monitoring tool–prospects, pitfalls and avenues to be developed</span>. <abbr class="cit-jnl-abbrev">Front. Zool.</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BZool.%26rft.volume%253D12%26rft.spage%253D24%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gogarten</span>,  <span class="cit-name-given-names">J.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Düx</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mubemba</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pléh</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mielke</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Müller-Tiburtius</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sachse</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wittig</span>,  <span class="cit-name-given-names">R.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Calvignac-Spencer</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Tropical rainforest flies carrying pathogens form stable associations with social nonhuman primates</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">28</span>, <span class="cit-fpage">4242</span>–<span class="cit-lpage">4258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D28%26rft.spage%253D4242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mosher</span>,  <span class="cit-name-given-names">B.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huyvaert</span>,  <span class="cit-name-given-names">K.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chestnut</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kerby</span>,  <span class="cit-name-given-names">J.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Madison</span>,  <span class="cit-name-given-names">J.D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Bailey</span>,  <span class="cit-name-given-names">L.L.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Design-and model-based recommendations for detecting and quantifying an amphibian pathogen in environmental samples</span>. <abbr class="cit-jnl-abbrev">Ecol. Evol.</abbr> <span class="cit-vol">7</span>, <span class="cit-fpage">10952</span>–<span class="cit-lpage">10962</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BEvol.%26rft.volume%253D7%26rft.spage%253D10952%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">E.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>,  <span class="cit-name-given-names">S.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">DeRisi</span>,  <span class="cit-name-given-names">J.L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Chiu</span>,  <span class="cit-name-given-names">C.Y.</span></span> (<span class="cit-pub-date">2011</span>). <abbr class="cit-jnl-abbrev">Using a Pan-Viral Microarray Assay (Virochip) to Screen Clinical Samples for Viral Pathogens</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kampmann</span>,  <span class="cit-name-given-names">M.-L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schnell</span>,  <span class="cit-name-given-names">I.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jensen</span>,  <span class="cit-name-given-names">R.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Axtner</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sander</span>,  <span class="cit-name-given-names">A.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hansen</span>,  <span class="cit-name-given-names">A.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bertelsen</span>,  <span class="cit-name-given-names">M.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Greenwood</span>,  <span class="cit-name-given-names">A.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gilbert</span>,  <span class="cit-name-given-names">M.T.P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Wilting</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Leeches as a source of mammalian viral DNA and RNA—a study in medicinal leeches</span>. <abbr class="cit-jnl-abbrev">Eur. J. Wildl. Res.</abbr> <span class="cit-vol">63</span>, <span class="cit-fpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEur.%2BJ.%2BWildl.%2BRes.%26rft.volume%253D63%26rft.spage%253D36%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kaczensky</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lkhagvasuren</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pereladova</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hemami</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Bouskila</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Equus hemionus</span>. <abbr class="cit-jnl-abbrev">The IUCN Red List of Threatened Species</abbr> <span class="cit-vol">2015</span>: e. T7951A45171204.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.10" data-doi="10.1638/03-126"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Löhr</span>,  <span class="cit-name-given-names">C.V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Juan-Sallés</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rosas-Rosas</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">García</span>,  <span class="cit-name-given-names">A.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garner</span>,  <span class="cit-name-given-names">M.M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Teifke</span>,  <span class="cit-name-given-names">J.P.</span></span> (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">Sarcoids in captive zebras (Equus burchellii): association with bovine papillomavirus type 1 infection</span>. <abbr class="cit-jnl-abbrev">J. Zoo Wildl. Med.</abbr> <span class="cit-vol">36</span>, <span class="cit-fpage">74</span>–<span class="cit-lpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bzoo%2Band%2Bwildlife%2Bmedicine%2B%253A%2B%2Bofficial%2Bpublication%2Bof%2Bthe%2BAmerican%2BAssociation%2Bof%2BZoo%2BVeterinarians%26rft.stitle%253DJ%2BZoo%2BWildl%2BMed%26rft.aulast%253DLohr%26rft.auinit1%253DC.%2BV.%26rft.volume%253D36%26rft.issue%253D1%26rft.spage%253D74%26rft.epage%253D81%26rft.atitle%253DSarcoids%2Bin%2Bcaptive%2Bzebras%2B%2528Equus%2Bburchellii%2529%253A%2Bassociation%2Bwith%2Bbovine%2Bpapillomavirus%2Btype%2B1%2Binfection.%26rft_id%253Dinfo%253Adoi%252F10.1638%252F03-126%26rft_id%253Dinfo%253Apmid%252F17315460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1638/03-126&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17315460&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.26.009993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229243400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.11" data-doi="10.1016/j.jviromet.2009.02.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van Dyk</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oosthuizen</span>,  <span class="cit-name-given-names">M.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bosman</span>,  <span class="cit-name-given-names">A.-M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nel</span>,  <span class="cit-name-given-names">P.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zimmerman</span>,  <span class="cit-name-given-names">D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Venter</span>,  <span class="cit-name-given-names">E.H.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Detection of bovine papillomavirus DNA in sarcoid-affected and healthy free-roaming zebra (Equus zebra) populations in South Africa</span>. <abbr class="cit-jnl-abbrev">J. Virol. Methods</abbr> <span class="cit-vol">158</span>, <span class="cit-fpage">141</span>–<span class="cit-lpage">151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirological%2Bmethods%26rft.stitle%253DJ%2BVirol%2BMethods%26rft.aulast%253Dvan%2BDyk%26rft.auinit1%253DE.%26rft.volume%253D158%26rft.issue%253D1-2%26rft.spage%253D141%26rft.epage%253D151%26rft.atitle%253DDetection%2Bof%2Bbovine%2Bpapillomavirus%2BDNA%2Bin%2Bsarcoid-affected%2Band%2Bhealthy%2Bfree-roaming%2Bzebra%2B%2528Equus%2Bzebra%2529%2Bpopulations%2Bin%2BSouth%2BAfrica.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jviromet.2009.02.008%26rft_id%253Dinfo%253Apmid%252F19428583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jviromet.2009.02.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19428583&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.26.009993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266349200024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Johnson</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Howard</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilson</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ward</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gilbert</span>,  <span class="cit-name-given-names">G.L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Degeling</span>,  <span class="cit-name-given-names">C.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Public preferences for One Health approaches to emerging infectious diseases: a discrete choice experiment</span>. <abbr class="cit-jnl-abbrev">Soc. Sci. Med.</abbr> <span class="cit-vol">228</span>, <span class="cit-fpage">164</span>–<span class="cit-lpage">171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSoc.%2BSci.%2BMed.%26rft.volume%253D228%26rft.spage%253D164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.13" data-doi="10.1038/nature06536"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">K.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Patel</span>,  <span class="cit-name-given-names">N.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Levy</span>,  <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Storeygard</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Balk</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gittleman</span>,  <span class="cit-name-given-names">J.L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Daszak</span>,  <span class="cit-name-given-names">P.</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">Global trends in emerging infectious diseases</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">451</span>, <span class="cit-fpage">990</span>–<span class="cit-lpage">993</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJones%26rft.auinit1%253DK.%2BE.%26rft.volume%253D451%26rft.issue%253D7181%26rft.spage%253D990%26rft.epage%253D993%26rft.atitle%253DGlobal%2Btrends%2Bin%2Bemerging%2Binfectious%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06536%26rft_id%253Dinfo%253Apmid%252F18288193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06536&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18288193&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.26.009993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253313100048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pruvot</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Khammavong</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Milavong</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Philavong</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reinharz</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mayxay</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rattanavong</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horwood</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dussart</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Douangngeun</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Toward a quantification of risks at the nexus of conservation and health: The case of bushmeat markets in Lao PDR</span>. <abbr class="cit-jnl-abbrev">Sci. Total Environ.</abbr> <span class="cit-vol">676</span>, <span class="cit-fpage">732</span>–<span class="cit-lpage">745</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BTotal%2BEnviron.%26rft.volume%253D676%26rft.spage%253D732%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Swift</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hunter</span>,  <span class="cit-name-given-names">P.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lees</span>,  <span class="cit-name-given-names">A.C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Bell</span>,  <span class="cit-name-given-names">D.J.</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">Wildlife trade and the emergence of infectious diseases</span>. <abbr class="cit-jnl-abbrev">EcoHealth</abbr> <span class="cit-vol">4</span>, <span class="cit-fpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcoHealth%26rft.volume%253D4%26rft.spage%253D25%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000246116500005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">X.-L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">X.-G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Si</span>,  <span class="cit-name-given-names">H.-R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">C.-L.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-fpage">1</span>–<span class="cit-lpage">4</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peeri</span>,  <span class="cit-name-given-names">N.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shrestha</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rahman</span>,  <span class="cit-name-given-names">M.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zaki</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tan</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bibi</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baghbanzadeh</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aghamohammadi</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">W.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Haque</span>,  <span class="cit-name-given-names">U.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">The SARS, MERS and novel coronavirus (COVID-19) epidemics, the newest and biggest global health threats: what lessons have we learned</span>? <abbr class="cit-jnl-abbrev">Int. J. Epidemiol.</abbr></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.18" data-doi="10.1056/NEJMoa030747"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drosten</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Günther</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Preiser</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van Der Werf</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brodt</span>,  <span class="cit-name-given-names">H.-R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Becker</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rabenau</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Panning</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kolesnikova</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fouchier</span>,  <span class="cit-name-given-names">R.A.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2003</span>). <span class="cit-article-title">Identification of a novel coronavirus in patients with severe acute respiratory syndrome</span>. <abbr class="cit-jnl-abbrev">N. Engl. J. Med.</abbr> <span class="cit-vol">348</span>, <span class="cit-fpage">1967</span>–<span class="cit-lpage">1976</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BEngl.%2BJ.%2BMed.%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa030747%26rft_id%253Dinfo%253Apmid%252F12690091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa030747&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12690091&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.26.009993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182823400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.19" data-doi="10.1056/NEJMoa1404505"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Baize</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pannetier</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oestereich</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rieger</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koivogui</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Magassouba</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Soropogui</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sow</span>,  <span class="cit-name-given-names">M.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keïta</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">De Clerck</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Emergence of Zaire Ebola virus disease in Guinea</span>. <abbr class="cit-jnl-abbrev">N. Engl. J. Med.</abbr> <span class="cit-vol">371</span>, <span class="cit-fpage">1418</span>–<span class="cit-lpage">1425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BEngl.%2BJ.%2BMed.%26rft.volume%253D371%26rft.spage%253D1418%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1404505%26rft_id%253Dinfo%253Apmid%252F24738640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa1404505&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24738640&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.26.009993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000342674800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-book" id="cit-2020.03.26.009993v1.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sharp</span>,  <span class="cit-name-given-names">P.M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Hahn</span>,  <span class="cit-name-given-names">B.H.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-chapter-title">Origins of HIV and the AIDS pandemic</span>. <span class="cit-publ-name">Cold Spring Harb</span>. <span class="cit-source">Perspect. Med.</span> <span class="cit-vol">1</span>, <span class="cit-fpage">a006841</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPerspect.%2BMed.%26rft.volume%253D1%26rft.spage%253D006841a%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.21" data-doi="10.3201/eid1612.100589"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoye</span>,  <span class="cit-name-given-names">B.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Munster</span>,  <span class="cit-name-given-names">V.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nishiura</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klaassen</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Fouchier</span>,  <span class="cit-name-given-names">R.A.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Surveillance of wild birds for avian influenza virus</span>. <abbr class="cit-jnl-abbrev">Emerg. Infect. Dis.</abbr> <span class="cit-vol">16</span>, <span class="cit-fpage">1827</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2Binfectious%2Bdiseases%26rft.stitle%253DEmerg%2BInfect%2BDis%26rft.aulast%253DHoye%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D16%26rft.issue%253D12%26rft.spage%253D1827%26rft.epage%253D1834%26rft.atitle%253DSurveillance%2Bof%2Bwild%2Bbirds%2Bfor%2Bavian%2Binfluenza%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid1612.100589%26rft_id%253Dinfo%253Apmid%252F21122209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid1612.100589&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21122209&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.26.009993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285031100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wilking</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziller</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Staubach</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Globig</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harder</span>,  <span class="cit-name-given-names">T.C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Conraths</span>,  <span class="cit-name-given-names">F.J.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Chances and limitations of wild bird monitoring for the avian influenza virus H5N1—detection of pathogens highly mobile in time and space</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.23" data-doi="10.3201/eid2207.151708"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Śmietanka</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Woźniakowski</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kozak</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Niemczuk</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frączyk</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bocian</span>,  <span class="cit-name-given-names">\Lukasz</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kowalczyk</span>,  <span class="cit-name-given-names">A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Pejsak</span>,  <span class="cit-name-given-names">Z.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">African swine fever epidemic, Poland, 2014-2015</span>. <abbr class="cit-jnl-abbrev">Emerg. Infect. Dis.</abbr> <span class="cit-vol">22</span>, <span class="cit-fpage">1201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerg.%2BInfect.%2BDis.%26rft.volume%253D22%26rft.spage%253D1201%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid2207.151708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid2207.151708&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bitome-Essono</span>,  <span class="cit-name-given-names">P.-Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ollomo</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arnathau</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Durand</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mokoudoum</span>,  <span class="cit-name-given-names">N.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yacka-Mouele</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Okouga</span>,  <span class="cit-name-given-names">A.-P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boundenga</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mve-Ondo</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Obame-Nkoghe</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Tracking zoonotic pathogens using blood-sucking flies as’ flying syringes’</span>. <abbr class="cit-jnl-abbrev">elife</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">e22069</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253Delife%26rft.volume%253D6%26rft.spage%253De22069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nasi</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taber</span>,  <span class="cit-name-given-names">A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Van Vliet</span>,  <span class="cit-name-given-names">N.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Empty forests, empty stomachs? Bushmeat and livelihoods in the Congo and Amazon Basins</span>. <abbr class="cit-jnl-abbrev">Int. For. Rev.</abbr> <span class="cit-vol">13</span>, <span class="cit-fpage">355</span>–<span class="cit-lpage">368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BFor.%2BRev.%26rft.volume%253D13%26rft.spage%253D355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000298123100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schnell</span>,  <span class="cit-name-given-names">I.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bohmann</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schultze</span>,  <span class="cit-name-given-names">S.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Richter</span>,  <span class="cit-name-given-names">S.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murray</span>,  <span class="cit-name-given-names">D.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sinding</span>,  <span class="cit-name-given-names">M.-H.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bass</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cadle</span>,  <span class="cit-name-given-names">J.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Campbell</span>,  <span class="cit-name-given-names">M.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dolch</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Debugging diversity–a pan-continental exploration of the potential of terrestrial blood-feeding leeches as a vertebrate monitoring tool</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">18</span>, <span class="cit-fpage">1282</span>–<span class="cit-lpage">1298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D18%26rft.spage%253D1282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Abrams</span>,  <span class="cit-name-given-names">J.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hörig</span>,  <span class="cit-name-given-names">L.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brozovic</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Axtner</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crampton-Platt</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mohamed</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wong</span>,  <span class="cit-name-given-names">S.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sollmann</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">D.W.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Wilting</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Shifting up a gear with iDNA: From mammal detection events to standardised surveys</span>. <abbr class="cit-jnl-abbrev">J. Appl. Ecol.</abbr> <span class="cit-vol">56</span>, <span class="cit-fpage">1637</span>–<span class="cit-lpage">1648</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAppl.%2BEcol.%26rft.volume%253D56%26rft.spage%253D1637%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ng</span>,  <span class="cit-name-given-names">T.F.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Willner</span>,  <span class="cit-name-given-names">D.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lim</span>,  <span class="cit-name-given-names">Y.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmieder</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chau</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nilsson</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anthony</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ruan</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rohwer</span>,  <span class="cit-name-given-names">F.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Breitbart</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Broad surveys of DNA viral diversity obtained through viral metagenomics of mosquitoes</span>. <abbr class="cit-jnl-abbrev">PloS One</abbr> <span class="cit-vol">6</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.29" data-doi="10.1038/s41598-016-0001-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stockhausen</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Merkel</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Calvignac-Spencer</span>,  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Leendertz</span>,  <span class="cit-name-given-names">F.H.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Assessing the feasibility of fly based surveillance of wildlife infectious diseases</span>. <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-016-0001-8%26rft_id%253Dinfo%253Apmid%252F27503664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-016-0001-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27503664&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.26.009993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Farkas</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McDonald</span>,  <span class="cit-name-given-names">J.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Malham</span>,  <span class="cit-name-given-names">S.K.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">D.L.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Two-step concentration of complex water samples for the detection of viruses</span>. <abbr class="cit-jnl-abbrev">Methods Protoc.</abbr> <span class="cit-vol">1</span>, <span class="cit-fpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BProtoc.%26rft.volume%253D1%26rft.spage%253D35%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Axtner</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crampton-Platt</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hörig</span>,  <span class="cit-name-given-names">L.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mohamed</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">C.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">D.W.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Wilting</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">An efficient and robust laboratory workflow and tetrapod database for larger scale environmental DNA studies</span>. <abbr class="cit-jnl-abbrev">GigaScience</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">giz029</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigaScience%26rft.volume%253D8%26rft.spage%253Dgiz029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-book" id="cit-2020.03.26.009993v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Meyer</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kircher</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-chapter-title">Illumina sequencing library preparation for highly multiplexed target capture and sequencing</span>. <span class="cit-publ-name">Cold Spring Harb</span>. <span class="cit-source">Protoc.</span> <span class="cit-vol">2010</span>, <span class="cit-fpage">pdb</span>–<span class="cit-lpage">prot5448</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtoc.%26rft.volume%253D2010%26rft.spage%253Dpdb%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.33" data-doi="10.1038/srep10189"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Alfano</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Courtiol</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vielgrader</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Timms</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roca</span>,  <span class="cit-name-given-names">A.L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Greenwood</span>,  <span class="cit-name-given-names">A.D.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Variation in koala microbiomes within and between individuals: effect of body region and captivity status</span>. <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">10189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D5%26rft.spage%253D10189%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep10189%26rft_id%253Dinfo%253Apmid%252F25960327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep10189&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25960327&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.26.009993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.34" data-doi="10.1073/pnas.242579699"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coscoy</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zylberberg</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Avila</span>,  <span class="cit-name-given-names">P.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boushey</span>,  <span class="cit-name-given-names">H.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ganem</span>,  <span class="cit-name-given-names">D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">DeRisi</span>,  <span class="cit-name-given-names">J.L.</span></span> (<span class="cit-pub-date">2002</span>). <span class="cit-article-title">Microarray-based detection and genotyping of viral pathogens</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci.</abbr> <span class="cit-vol">99</span>, <span class="cit-fpage">15687</span>–<span class="cit-lpage">15692</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.242579699%26rft_id%253Dinfo%253Apmid%252F12429852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTkvMjQvMTU2ODciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMy8yOS8yMDIwLjAzLjI2LjAwOTk5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yozwiak</span>,  <span class="cit-name-given-names">N.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Skewes-Cox</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stenglein</span>,  <span class="cit-name-given-names">M.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Balmaseda</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harris</span>,  <span class="cit-name-given-names">E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">DeRisi</span>,  <span class="cit-name-given-names">J.L.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Virus identification in unknown tropical febrile illness cases using deep sequencing</span>. <abbr class="cit-jnl-abbrev">PLoS Negl. Trop. Dis.</abbr> <span class="cit-fpage">6</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.36" data-doi="10.1093/nar/30.1.207"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Edgar</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Domrachev</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Lash</span>,  <span class="cit-name-given-names">A.E.</span></span> (<span class="cit-pub-date">2002</span>). <span class="cit-article-title">Gene Expression Omnibus: NCBI gene expression and hybridization array data repository</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">30</span>, <span class="cit-fpage">207</span>–<span class="cit-lpage">210</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%26rft.volume%253D30%26rft.issue%253D1%26rft.spage%253D207%26rft.epage%253D210%26rft.atitle%253DGene%2BExpression%2BOmnibus%253A%2BNCBI%2Bgene%2Bexpression%2Band%2Bhybridization%2Barray%2Bdata%2Brepository%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F30.1.207%26rft_id%253Dinfo%253Apmid%252F11752295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/30.1.207&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11752295&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.26.009993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173077100055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.37" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Langmead</span>,  <span class="cit-name-given-names">B.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Salzberg</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Fast gapped-read alignment with bowtie 2 Nat Methods</span> <span class="cit-vol">9</span> (<span class="cit-issue">4</span>): <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">pmid: </span><span class="cit-pub-id cit-pub-id-pmid">22388286</span> <abbr class="cit-jnl-abbrev">View Article PubMed (NCBI)</abbr>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.26.009993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.38" data-doi="10.1093/bioinformatics/bts611"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kopylova</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Noé</span>,  <span class="cit-name-given-names">L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Touzet</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">SortMeRNA: fast and accurate filtering of ribosomal RNAs in metatranscriptomic data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>, <span class="cit-fpage">3211</span>–<span class="cit-lpage">3217</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts611%26rft_id%253Dinfo%253Apmid%252F23071270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts611&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23071270&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.26.009993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312105300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.39" data-doi="10.1089/cmb.2012.0021"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bankevich</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nurk</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Antipov</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gurevich</span>,  <span class="cit-name-given-names">A.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dvorkin</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kulikov</span>,  <span class="cit-name-given-names">A.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lesin</span>,  <span class="cit-name-given-names">V.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nikolenko</span>,  <span class="cit-name-given-names">S.I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pham</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Prjibelski</span>,  <span class="cit-name-given-names">A.D.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing</span>. <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr> <span class="cit-vol">19</span>, <span class="cit-fpage">455</span>–<span class="cit-lpage">477</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DBankevich%26rft.auinit1%253DA.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D455%26rft.epage%253D477%26rft.atitle%253DSPAdes%253A%2Ba%2Bnew%2Bgenome%2Bassembly%2Balgorithm%2Band%2Bits%2Bapplications%2Bto%2Bsingle-cell%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2012.0021%26rft_id%253Dinfo%253Apmid%252F22506599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/cmb.2012.0021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22506599&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.26.009993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.40" data-doi="10.1038/nbt.1883"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>,  <span class="cit-name-given-names">M.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haas</span>,  <span class="cit-name-given-names">B.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yassour</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Levin</span>,  <span class="cit-name-given-names">J.Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thompson</span>,  <span class="cit-name-given-names">D.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Amit</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Adiconis</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fan</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Raychowdhury</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zeng</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Trinity: reconstructing a full-length transcriptome without a genome from RNA-Seq data</span>. <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>, <span class="cit-fpage">644</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DGrabherr%26rft.auinit1%253DM.%2BG.%26rft.volume%253D29%26rft.issue%253D7%26rft.spage%253D644%26rft.epage%253D652%26rft.atitle%253DFull-length%2Btranscriptome%2Bassembly%2Bfrom%2BRNA-Seq%2Bdata%2Bwithout%2Ba%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1883%26rft_id%253Dinfo%253Apmid%252F21572440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1883&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21572440&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.26.009993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.41" data-doi="10.1093/bioinformatics/btq461"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Edgar</span>,  <span class="cit-name-given-names">R.C.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Search and clustering orders of magnitude faster than BLAST</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>, <span class="cit-fpage">2460</span>–<span class="cit-lpage">2461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq461%26rft_id%253Dinfo%253Apmid%252F20709691%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq461&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20709691&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.26.009993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282170000016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.42" data-doi="10.1093/bioinformatics/bty695"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vilsker</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moosa</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nooij</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fonseca</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ghysens</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dumon</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pauwels</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alcantara</span>,  <span class="cit-name-given-names">L.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vanden Eynden</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vandamme</span>,  <span class="cit-name-given-names">A.-M.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Genome Detective: an automated system for virus identification from high-throughput sequencing data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">35</span>, <span class="cit-fpage">871</span>–<span class="cit-lpage">873</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D35%26rft.spage%253D871%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty695%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.43" data-doi="10.1093/bioinformatics/bts199"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kearse</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moir</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilson</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stones-Havas</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheung</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sturrock</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buxton</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cooper</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Markowitz</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duran</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Geneious Basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>, <span class="cit-fpage">1647</span>–<span class="cit-lpage">1649</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts199%26rft_id%253Dinfo%253Apmid%252F22543367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts199&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22543367&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.26.009993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305419800052&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.44" data-doi="10.1093/molbev/mst010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Katoh</span>,  <span class="cit-name-given-names">K.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Standley</span>,  <span class="cit-name-given-names">D.M.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">MAFFT multiple sequence alignment software version 7: improvements in performance and usability</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>, <span class="cit-fpage">772</span>–<span class="cit-lpage">780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst010%26rft_id%253Dinfo%253Apmid%252F23329690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23329690&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.26.009993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000317002300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.45" data-doi="10.1093/bioinformatics/btu033"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>, <span class="cit-fpage">1312</span>–<span class="cit-lpage">1313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu033%26rft_id%253Dinfo%253Apmid%252F24451623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24451623&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.26.009993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000336095100024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.46" data-doi="10.1093/bioinformatics/btw346"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Somervuo</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koskela</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pennanen</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Henrik Nilsson</span>,  <span class="cit-name-given-names">R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Ovaskainen</span>,  <span class="cit-name-given-names">O.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Unbiased probabilistic taxonomic classification for DNA barcoding</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>, <span class="cit-fpage">2920</span>–<span class="cit-lpage">2927</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw346%26rft_id%253Dinfo%253Apmid%252F27296980%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27296980&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F29%2F2020.03.26.009993.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47 in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.47" data-doi="10.1038/srep46559"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dayaram</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Franz</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schattschneider</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Damiani</span>,  <span class="cit-name-given-names">A.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bischofberger</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Osterrieder</span>,  <span class="cit-name-given-names">N.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Greenwood</span>,  <span class="cit-name-given-names">A.D.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Long term stability and infectivity of herpesviruses in water</span>. <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">7</span>, <span class="cit-fpage">46559</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D7%26rft.spage%253D46559%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep46559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep46559&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.03.26.009993v1.48" data-doi="10.1128/JVI.74.13.5776-5787.2000"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Palmarini</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Datta</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Omid</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murgia</span>,  <span class="cit-name-given-names">C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Fan</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">2000</span>). <span class="cit-article-title">The long terminal repeat of Jaagsiekte sheep retrovirus is preferentially active in differentiated epithelial cells of the lungs</span>. <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">74</span>, <span class="cit-fpage">5776</span>–<span class="cit-lpage">5787</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DPalmarini%26rft.auinit1%253DM.%26rft.volume%253D74%26rft.issue%253D13%26rft.spage%253D5776%26rft.epage%253D5787%26rft.atitle%253DThe%2BLong%2BTerminal%2BRepeat%2Bof%2BJaagsiekte%2BSheep%2BRetrovirus%2BIs%2BPreferentially%2BActive%2Bin%2BDifferentiated%2BEpithelial%2BCells%2Bof%2Bthe%2BLungs%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.74.13.5776-5787.2000%26rft_id%253Dinfo%253Apmid%252F10846056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI3NC8xMy81Nzc2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDMvMjkvMjAyMC4wMy4yNi4wMDk5OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.03.26.009993v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.03.26.009993v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.03.27.012260v1" title="Extended depth of focus multiphoton microscopy via incoherent pulse splitting" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.26.010066v1" title="Dissecting the toxicity and mitigating the impact of harmful Prymnesium blooms in the UK waters of the Norfolk Broads" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 29, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.03.26.009993v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.03.26.009993v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.03.26.009993v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.03.26.009993v1" class="active" data-icon-position="" data-hide-link-title="0">Non-invasive surveys of mammalian viruses using environmental DNA</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1216079" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-C49UtZnJyFiKEEmzrpxa2dAcbRsXW_4Wgv_7f1xrBI8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1216079" id="node1216079--2" data-pisa="biorxiv;2020.03.26.009993v1" data-pisa-master="biorxiv;2020.03.26.009993" data-seqnum="1216079" data-apath="/biorxiv/early/2020/03/29/2020.03.26.009993.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Non-invasive surveys of mammalian viruses using environmental DNA</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Niccolo</span> <span class="nlm-surname">Alfano</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Anisha</span> <span class="nlm-surname">Dayaram</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jan</span> <span class="nlm-surname">Axtner</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Kyriakos</span> <span class="nlm-surname">Tsangaras</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Marie-Louise</span> <span class="nlm-surname">Kampmann</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Azlan</span> <span class="nlm-surname">Mohamed</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Seth T.</span> <span class="nlm-surname">Wong</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">M. Thomas P.</span> <span class="nlm-surname">Gilbert</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Andreas</span> <span class="nlm-surname">Wilting</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Alex D.</span> <span class="nlm-surname">Greenwood</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.26.009993; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.26.009993 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.03.26.009993v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.03.26.009993v1&amp;title=Non-invasive%20surveys%20of%20mammalian%20viruses%20using%20environmental%20DNA" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.26.009993v1&amp;title=Non-invasive%20surveys%20of%20mammalian%20viruses%20using%20environmental%20DNA" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.26.009993v1&amp;text=Non-invasive%20surveys%20of%20mammalian%20viruses%20using%20environmental%20DNA" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.26.009993v1&amp;title=Non-invasive%20surveys%20of%20mammalian%20viruses%20using%20environmental%20DNA" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.03.26.009993v1&amp;t=Non-invasive%20surveys%20of%20mammalian%20viruses%20using%20environmental%20DNA" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.03.26.009993v1&amp;title=Non-invasive%20surveys%20of%20mammalian%20viruses%20using%20environmental%20DNA" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.26.009993v1&amp;title=Non-invasive%20surveys%20of%20mammalian%20viruses%20using%20environmental%20DNA" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1216079" id="node1216079--3" data-pisa="biorxiv;2020.03.26.009993v1" data-pisa-master="biorxiv;2020.03.26.009993" data-seqnum="1216079" data-apath="/biorxiv/early/2020/03/29/2020.03.26.009993.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Non-invasive surveys of mammalian viruses using environmental DNA</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Niccolo</span> <span class="nlm-surname">Alfano</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Anisha</span> <span class="nlm-surname">Dayaram</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jan</span> <span class="nlm-surname">Axtner</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Kyriakos</span> <span class="nlm-surname">Tsangaras</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Marie-Louise</span> <span class="nlm-surname">Kampmann</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Azlan</span> <span class="nlm-surname">Mohamed</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Seth T.</span> <span class="nlm-surname">Wong</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">M. Thomas P.</span> <span class="nlm-surname">Gilbert</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Andreas</span> <span class="nlm-surname">Wilting</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Alex D.</span> <span class="nlm-surname">Greenwood</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.03.26.009993; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.03.26.009993 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Non-invasive%20surveys%20of%20mammalian%20viruses%20using%20environmental%20DNA&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Alfano%2C+Niccolo&amp;rft.au=Dayaram%2C+Anisha&amp;rft.au=Axtner%2C+Jan&amp;rft.au=Tsangaras%2C+Kyriakos&amp;rft.au=Kampmann%2C+Marie-Louise&amp;rft.au=Mohamed%2C+Azlan&amp;rft.au=Wong%2C+Seth+T.&amp;rft.au=Gilbert%2C+M.+Thomas+P.&amp;rft.au=Wilting%2C+Andreas&amp;rft.au=Greenwood%2C+Alex+D."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1216079/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1216079/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1216079/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1216079/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1216079/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1216079/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1216079/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1216079/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1216079/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1216079/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1216079/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1216079/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.03.26.009993v1&amp;count=horizontal&amp;via=&amp;text=Non-invasive%20surveys%20of%20mammalian%20viruses%20using%20environmental%20DNA&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.03.26.009993v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.03.26.009993v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.03.26.009993v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/molecular-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Molecular Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
