<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">PLoS One</span><span class="iso-abbrev" title="iso-abbrev">PLoS ONE</span><span class="publisher-id" title="publisher-id">plos</span><span class="pmc" title="pmc">plosone</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">PLoS ONE</span></div><span class="issn-epub" tagx="issn" title="issn-epub">1932-6203</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, USA</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/3355188">3355188</a></span><span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22615729">22615729</a></span><span class="publisher-id" title="publisher-id">PONE-D-11-09309</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.pone.0029617">10.1371/journal.pone.0029617</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Research Article</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Computational Biology</span>: <span class="subject" title="subject">Sequence Analysis</span></div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Evolutionary Systematics</span>: <span class="subject" title="subject">Phylogenetics</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Population Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Epidemiology</span>: <span class="subject" title="subject">Infectious Disease Epidemiology</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Epidemiology</span>: <span class="subject" title="subject">Infectious Disease Epidemiology</span></div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Infectious diseases</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Zoonoses</span>: <span class="subject" title="subject">Avian influenza A viruses</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Social and Behavioral Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Geography</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Geoinformatics</span>: <span class="subject" title="subject">Environmental Systems Modeling</span><span class="subject" title="subject">Geostatistics</span><span class="subject" title="subject">Spatial Autocorrelation</span></div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Human Geography</span>: <span class="subject" title="subject">Spatial Analysis</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Veterinary Science</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Veterinary Diseases</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Zoonotic Diseases</span>: <span class="subject" title="subject">Animal Influenza</span></div> </div> </div> </div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Using Knowledge Fusion to Analyze Avian Influenza H5N1 in East and Southeast Asia</div> <div class="alt-title" title="alt-title">
Knowledge Fusion to Analyze H5N1</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ge</span><span class="given-names" tagx="given-names" title="given-names">Erjia</span></span><a href="#aff1"><sup>1</sup></a><a href="#aff2"><sup>2</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haining</span><span class="given-names" tagx="given-names" title="given-names">Robert</span></span><a href="#aff3"><sup>3</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">Chi Pang</span></span><a href="#aff4"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yu</span><span class="given-names" tagx="given-names" title="given-names">Zuguo</span></span><a href="#aff5"><sup>5</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waye</span><span class="given-names" tagx="given-names" title="given-names">Miu Yee</span></span><a href="#aff6"><sup>6</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chu</span><span class="given-names" tagx="given-names" title="given-names">Ka Hou</span></span><a href="#aff4"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leung</span><span class="given-names" tagx="given-names" title="given-names">Yee</span></span><a href="#aff1"><sup>1</sup></a><a href="#aff7"><sup>7</sup></a><a href="#cor1"><sup>*</sup></a></span></div><span class="citation_author_institution" id="aff1">[1], <span class="addr-line" title="addr-line">Department of Geography and Resource Management, The Chinese University of Hong Kong, Hong Kong, China</span></span><span class="citation_author_institution" id="aff2">[2], <span class="addr-line" title="addr-line">School of Public Health and Primary Care, The Chinese University of Hong Kong, Hong Kong, China</span></span><span class="citation_author_institution" id="aff3">[3], <span class="addr-line" title="addr-line">Department of Geography, University of Cambridge, Cambridge, United Kingdom</span></span><span class="citation_author_institution" id="aff4">[4], <span class="addr-line" title="addr-line">School of Life Sciences, The Chinese University of Hong Kong, Hong Kong, China</span></span><span class="citation_author_institution" id="aff5">[5], <span class="addr-line" title="addr-line">School of Mathematics and Computational Science, Xiangtan University, Hunan Province, China</span></span><span class="citation_author_institution" id="aff6">[6], <span class="addr-line" title="addr-line">School of Biomedical Sciences, The Chinese University of Hong Kong, Hong Kong, China</span></span><span class="citation_author_institution" id="aff7">[7], <span class="addr-line" title="addr-line">Institute of Environment, Energy and Sustainability, The Chinese University of Hong Kong, Hong Kong, China</span></span><div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">LÃ³pez-GalÃ­ndez</span><span class="given-names" tagx="given-names" title="given-names">Cecilio</span></span><span class="role" tagx="role" title="role">Editor</span><a href="#edit1" /></span></div><span class="citation_author_institution" id="edit1">[], </span><div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">yeeleung@cuhk.edu.hk</span></div> <div class="fn-type-con" title="con"> <p>Conceived and designed the experiments: YL ZGY KHC EJG. Performed the experiments: YL EJG CPL. Analyzed the data: YL EJG RH ZGY KHC MYW. Wrote the paper: YL EJG RH.</p> </div> </div><span class="pub-date-collection" title="pub-date-collection">collection: <span>2012</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2012-5-5</span></span><span class="volume" tagx="volume" title="volume">7</span><span class="issue" tagx="issue" title="issue">5</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e29617</span><span class="history" title="history"><span class="received" title="received">received: 2011-5-26</span><span class="accepted" title="accepted">accepted: 2011-12-1</span></span><div class="permissions"><span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2012</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are properly credited.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Highly pathogenic avian influenza (HPAI) H5N1, a disease associated with high rates of mortality in infected human populations, poses a serious threat to public health in many parts of the world. This article reports findings from a study aimed at improving our understanding of the spatial pattern of the highly pathogenic avian influenza, H5N1, risk in East-Southeast Asia where the disease is both persistent and devastating. Though many disciplines have made important contributions to our understanding of H5N1, it remains a challenge to integrate knowledge from different disciplines. This study applies genetic analysis that identifies the evolution of the H5N1 virus in space and time, epidemiological analysis that determines socio-ecological factors associated with H5N1 occurrence, and statistical analysis that identifies outbreak clusters, and then applies a methodology to formally integrate the findings of the three sets of methodologies. The present study is novel in two respects. First it makes the initiative attempt to use genetic sequences and space-time data to create a space-time phylogenetic tree to estimate and map the virus' ability to spread. Second, by integrating the results we are able to generate insights into the space-time occurrence and spread of H5N1 that we believe have a higher level of corroboration than is possible when analysis is based on only one methodology. Our research identifies links between the occurrence of H5N1 by area and a set of socio-ecological factors including altitude, population density, poultry density, and the shortest path distances to inland water, coastlines, migrating routes, railways, and roads. This study seeks to lay a solid foundation for the interdisciplinary study of this and other influenza outbreaks. It will provide substantive information for containing H5N1 outbreaks.</p> </div> <div class="counts" title="counts"><span class="page-count" title="page-count">page-count: </span></div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Highly pathogenic avian influenza (HPAI) H5N1 first isolated in Guangdong province, southern China in 1996 <a href="#pone.0029617-Xu1">[1]</a>. The HPAI H5N1 virus was redetected in East-Southeast Asia in late 2003 associated with large outbreaks among poultry in Thailand, Vietnam, Indonesia, and China <a href="#pone.0029617-Li1">[2]</a>. The HPAI H5N1 influenza virus has established multiple regional sub-lineages and shown long-term persistence in these countries <a href="#pone.0029617-Chen1">[3]</a>. Also, the virus has continued its geographical migration through Southeast Asia to Eurasia and Africa. Disease control measures resulted in large numbers of domestic and wild birds being killed or culled to prevent a possible avian-influenza pandemic that might cost still more human lives in addition to huge economic loss <a href="#pone.0029617-Enserink1">[4]</a>. HPAI H5N1 is considered a serious public health risk despite relatively low numbers of human infections. The public health risk arises through the high human mortality rates (to September 2010, a total of 505 patients from 15 countries were known to have been infected by H5N1, of which about <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> had died <a href="#pone.0029617-WHO1">[5]</a>) and the prospect that the H5N1 virus might reassort with another influenza A strain to produce a highly pathogenic and transmissible virus in humans <a href="#pone.0029617-Preiser1">[6]</a>. Moreover, the wide distribution of H5N1 in South-East Asia provides ample opportunity for reassortment <a href="#pone.0029617-Dawood1">[7]</a>. Identifying areas that are at risk of H5N1 is important for the development of strategies to control this disease.</p> <p>Traditional genetic analysis, which infers the phylogenetic relationships associated with the H5N1 virus from its genomic sequences, has enabled progress to be made in understanding the evolution of avian influenza viruses <a href="#pone.0029617-Li1">[2]</a>, <a href="#pone.0029617-Smith1">[8]</a>. By this means, possible sources and pathways that are associated with the spread of the H5N1 virus can be inferred <a href="#pone.0029617-Smith1">[8]</a>, <a href="#pone.0029617-Duan1">[9]</a>. Phylogeographic analysis <a href="#pone.0029617-Avise1">[10]</a> offers a method of tracking the migration of the H5N1 virus. It analyzes the topology of the phylogenetic tree and uses evolutionary models to statistically infer the resident localities of the H5N1 virus <a href="#pone.0029617-Wallace1">[11]</a>. Medical geography examines the spatial pattern of H5N1 looking for localized hot spots where outbreaks are significantly clustered <a href="#pone.0029617-Si1">[12]</a>. In spatial epidemiology, risk factor analysis focuses on the identification of factors associated with H5N1 occurrence and statistical modeling is used to predict the incidence of the disease <a href="#pone.0029617-Gilbert1">[13]</a>. All these different forms of analyses have a common objective which is to understand the distribution and the spread of avian influenza. The study of avian influenza H5N1 is multi-disciplinary across Virology, Molecular Biology, Evolutionary Biology, Medical Geography, and Spatial Epidemiology.</p> <p>However, any study that relies on only one kind of disciplinary knowledge may miss important connections. For example, not taking into account the effects caused by geographical scale (the so-called â€œmodifiable areal unit problemâ€� <a href="#pone.0029617-Openshaw1">[14]</a>) could lead to interpretation errors when reviewing results from statistical analysis and modeling in area-based epidemiology. The predictive mapping of H5N1 risk in China, for instance, in Fang et al. <a href="#pone.0029617-Fang1">[15]</a>, conflicts with empirical observations <a href="#pone.0029617-WHO1">[5]</a>, <a href="#pone.0029617-FAO1">[16]</a> and previous studies <a href="#pone.0029617-Smith2">[17]</a>, <a href="#pone.0029617-SmallmanRaynor1">[18]</a>. Also, phylogenetic analysis <a href="#pone.0029617-CavalliSforza1">[19]</a> has its own limitations when genetic data are incomplete, and evolutionary models inappropriate <a href="#pone.0029617-Penny1">[20]</a>. This form of analysis, which provides a micro-scale insight into the process of viral evolution, is insufficient for understanding macro-scale spread of avian influenza. In addition, uncertainty, perhaps arising from incomplete data, limited domain knowledge, or the application of an insufficiently sophisticated methodology could limit the value of these analyses. Wallace et al. states that limited sampling may lead to results that are not statistically significant <a href="#pone.0029617-Wallace1">[11]</a>.</p> <p>Although current research tends to integrate multi-disciplinary studies of avian influenza, it stops at the early stage of analytically integrating data, for example, on phylogenetic relationships between isolated occurrence of the virus, migratory bird movements, and trade in poultry and wild birds <a href="#pone.0029617-Kilpatrick1">[21]</a>, <a href="#pone.0029617-Liang1">[22]</a> or implementing only basic statistical analysis between genetic distance and geographic distance <a href="#pone.0029617-Carrel1">[23]</a>. Even though knowledge has been acquired in many different disciplines, adequate methods to quantify and integrate this knowledge for H5N1 remain poorly developed.</p> <p>This article proposes a novel approach to integrating the findings of phylogenetic analysis, which unravels H5N1 evolution in space and time, with modified local <i>K</i> function analysis, which identifies outbreak clusters in space, and also with spatial epidemiology, which determines socio-ecological factors associated with the occurrence of H5N1. In this study, Dempster-Shafer theory of evidence <a href="#pone.0029617-Dempster1">[24]</a>, <a href="#pone.0029617-Shafer1">[25]</a>, a mathematical method for making inferences based on multiple forms of evidence and which recognizes the uncertainties associated with the different sources of evidence, is used to formally integrate the three sets of findings. The present study is novel and significant in seeking to lay a solid foundation for the inter-disciplinary study of this and other relevant influenza epidemics. First, it uses genetic sequences and space-time data to create a phylogenetic tree to estimate the virus' capability of spreading. This is the first attempt to provide a mapping of H5N1 viruses derived from the phylogenetic tree. Second, by integrating the results obtained from the three analyses, we offer insights into the occurrence and space-time spread of H5N1 that have a higher level of correlation with empirical evidence than is found when analysis is based on only one methodology. In addition, we apply the methodology across multiple scales; that is to the whole of East-Southeast Asia as well as the individual countries of Thailand, Vietnam, Indonesia, and China, respectively. Our analysis results in a significant advance in findings over those reported in, for example, Gilbert et al. <a href="#pone.0029617-Gilbert1">[13]</a>, and we believe our findings are more precise and informative in representing the occurrence and the space-time dynamics associated with the spread of H5N1.</p> </div> <div class="resultsanddiscussion" title="sec"> <div class="title" tagx="title" title="title">
Results and Discussion</div> <div class="thailandandvietnam" title="sec"> <div class="title" tagx="title" title="title">
Thailand and Vietnam</div> <p><a href="#pone-0029617-g001">Figure 1</a> and <a href="#pone.0029617.s001">Figure S1</a> show the result for Thailand and Vietnam. <a href="#pone.0029617.s001">Figure S1</a>(c) and (f) demonstrate our estimate of the spatially varying â€˜degree of beliefâ€™ in the level of risk of H5N1 obtained by integrating the three forms of analysis (see <a href="#pone-0029617-g001">Figures 1(a) and (c)</a>, and <a href="#pone-0029617-g001">Figures 1(b) and (d)</a>, and <a href="#pone.0029617.s001">Figures S1</a>(b) and (e)) using the Dempster-Shafer theory of knowledge fusion. <a href="#pone.0029617.s001">Figures S1</a>(b) and (e) are the results from the epidemiological analysis of Gilbert. et al. <a href="#pone.0029617-Gilbert1">[13]</a>. In both cases the closer any area's value is to 1 (the redder it is) the greater the likelihood of an H5N1 outbreak in that area. <a href="#pone.0029617.s001">Figure S1</a> also reports Pearson correlation coefficients (R) and associated <i>p</i>-values which show that our experimental patterns ((c) and (f)) have a closer correspondence to the observed pattern of cases ((a) and (d)) than the results of Gilbert et al. <a href="#pone.0029617-Gilbert1">[13]</a> ((b) and (e)). This finding holds over a range of spatial scales from 60<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>60 cell aggregates to 30<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>30 cell aggregates.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0029617.g001"><span class="label" tagx="label" title="label">Figure 1</span></a></div>   <div class="title" tagx="title" title="title">
Probability maps predicting the occurrence of avian influenza (H5N1) in Thailand and Vietnam.</div> <p>(a) and (c) show the probabilities derived from the phylogenetic trees analyses (see <a href="#pone.0029617.s003">Figure S3</a>(b)); (b) and (d) show the results of the modified local <i>K</i> function analysis, depicting the spatial distribution of outbreak clusters. The experimental data covers the H5N1 outbreaks from late 2003 to 2009.</p>   </div> <p><a href="#pone.0029617.s001">Figure S1</a>(c) shows that the greatest risk of H5N1 is in the upper central region and the lower part of northern Thailand. It indicates less risk in central Thailand than predicted by Gilbert et al. <a href="#pone.0029617-Gilbert1">[13]</a> (<a href="#pone-0029617-t001">Table 1(b)</a>). This pattern corresponds with fewer observed outbreaks in this area of Thailand.</p> <div class="table-wrap_UNKNOWN" id="pone-0029617-t001" position="float"><a href="https://dx.doi.org/10.1371/journal.pone.0029617.t001">10.1371/journal.pone.0029617.t001</a><span class="label" tagx="label" title="label">Table 1</span> <div class="title" tagx="title" title="title">
Summary results of the logistic regression model for the avian influenza H5N1 epidemics in East-Southeast Asia, Indonesia, and China, 1996â€"2009.</div>  <div class="alternatives" title="alternatives"> <div class="graphic" id="pone-0029617-t001-1" /> <table frame="hsides" rules="groups"> <colgroup span="1"> <col align="left" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup> <colgroup><col align="center" span="1" /></colgroup>  <thead> <tr> <td align="left" rowspan="1" colspan="1">Regent or Country</td> <td align="left" rowspan="1" colspan="1">Con</td> <td align="left" rowspan="1" colspan="1">Alt</td> <td align="left" rowspan="1" colspan="1">PopDen</td> <td align="left" rowspan="1" colspan="1">PolDen</td> <td align="left" rowspan="1" colspan="1">D2Wat</td> <td align="left" rowspan="1" colspan="1">D2Coast</td> <td align="left" rowspan="1" colspan="1">D2Flyway</td> <td align="left" rowspan="1" colspan="1">D2Rail</td> <td align="left" rowspan="1" colspan="1">D2Road</td> </tr> </thead> <tbody> <tr> <td align="left" rowspan="1" colspan="1">Eastâ€"Southeast Asia</td> <td align="left" rowspan="1" colspan="1">.5371</td> <td align="left" rowspan="1" colspan="1">âˆ’.001</td> <td align="left" rowspan="1" colspan="1">.0007</td> <td align="left" rowspan="1" colspan="1">âˆ’4.01<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span></td> <td align="left" rowspan="1" colspan="1">âˆ’.0037</td> <td align="left" rowspan="1" colspan="1">âˆ’.0008</td> <td align="left" rowspan="1" colspan="1">âˆ’.0007</td> <td align="left" rowspan="1" colspan="1">âˆ’.0004</td> <td align="left" rowspan="1" colspan="1">âˆ’.249</td> </tr> <tr> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">p<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>.001</td> <td align="left" rowspan="1" colspan="1">p<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>.001</td> <td align="left" rowspan="1" colspan="1">pâ€Š=â€Š.142</td> <td align="left" rowspan="1" colspan="1">p<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>.001</td> <td align="left" rowspan="1" colspan="1">p<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>.001</td> <td align="left" rowspan="1" colspan="1">p<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>.001</td> <td align="left" rowspan="1" colspan="1">p<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>.001</td> <td align="left" rowspan="1" colspan="1">p<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>.001</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">Indonesia</td> <td align="left" rowspan="1" colspan="1">âˆ’1.841</td> <td align="left" rowspan="1" colspan="1">3.654<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span></td> <td align="left" rowspan="1" colspan="1">4.298<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span></td> <td align="left" rowspan="1" colspan="1">4.142<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span></td> <td align="left" rowspan="1" colspan="1">5.597<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span></td> <td align="left" rowspan="1" colspan="1">âˆ’4.383<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span></td> <td align="left" rowspan="1" colspan="1">3.904<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span></td> <td align="left" rowspan="1" colspan="1">âˆ’1.853<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span></td> <td align="left" rowspan="1" colspan="1">âˆ’.014</td> </tr> <tr> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">pâ€Š=â€Š.074</td> <td align="left" rowspan="1" colspan="1">p<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>.001</td> <td align="left" rowspan="1" colspan="1">p<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>.001</td> <td align="left" rowspan="1" colspan="1">pâ€Š=â€Š.030</td> <td align="left" rowspan="1" colspan="1">pâ€Š=â€Š.012</td> <td align="left" rowspan="1" colspan="1">pâ€Š=â€Š.011</td> <td align="left" rowspan="1" colspan="1">p<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>.001</td> <td align="left" rowspan="1" colspan="1">pâ€Š=â€Š.082</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">China (1996â€"2009)</td> <td align="left" rowspan="1" colspan="1">âˆ’1.637</td> <td align="left" rowspan="1" colspan="1">3.065<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span></td> <td align="left" rowspan="1" colspan="1">8.289<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span></td> <td align="left" rowspan="1" colspan="1">âˆ’4.74<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span></td> <td align="left" rowspan="1" colspan="1">âˆ’.012</td> <td align="left" rowspan="1" colspan="1">âˆ’2.458<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span></td> <td align="left" rowspan="1" colspan="1">7.951<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span></td> <td align="left" rowspan="1" colspan="1">âˆ’9.076<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span></td> <td align="left" rowspan="1" colspan="1">âˆ’.025</td> </tr> <tr> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">pâ€Š=â€Š.036</td> <td align="left" rowspan="1" colspan="1">pâ€Š=â€Š.004</td> <td align="left" rowspan="1" colspan="1">pâ€Š=â€Š.580</td> <td align="left" rowspan="1" colspan="1">p<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>.001</td> <td align="left" rowspan="1" colspan="1">pâ€Š=â€Š.094</td> <td align="left" rowspan="1" colspan="1">pâ€Š=â€Š.003</td> <td align="left" rowspan="1" colspan="1">pâ€Š=â€Š.166</td> <td align="left" rowspan="1" colspan="1">pâ€Š=â€Š.012</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">China (1996â€"2004)</td> <td align="left" rowspan="1" colspan="1">âˆ’1.367</td> <td align="left" rowspan="1" colspan="1">6.515<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span></td> <td align="left" rowspan="1" colspan="1">1.407<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span></td> <td align="left" rowspan="1" colspan="1">âˆ’4.947<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span></td> <td align="left" rowspan="1" colspan="1">âˆ’.034</td> <td align="left" rowspan="1" colspan="1">âˆ’3.718<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span></td> <td align="left" rowspan="1" colspan="1">1.353<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span></td> <td align="left" rowspan="1" colspan="1">âˆ’4.431<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span></td> <td align="left" rowspan="1" colspan="1">âˆ’.041</td> </tr> <tr> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">pâ€Š=â€Š.074</td> <td align="left" rowspan="1" colspan="1">pâ€Š=â€Š.078</td> <td align="left" rowspan="1" colspan="1">pâ€Š=â€Š.459</td> <td align="left" rowspan="1" colspan="1">pâ€Š=â€Š.0002</td> <td align="left" rowspan="1" colspan="1">pâ€Š=â€Š.288</td> <td align="left" rowspan="1" colspan="1">pâ€Š=â€Š.028</td> <td align="left" rowspan="1" colspan="1">pâ€Š=â€Š.049</td> <td align="left" rowspan="1" colspan="1">pâ€Š=â€Š.175</td> </tr> <tr> <td align="left" rowspan="1" colspan="1">China (2005â€"2009)</td> <td align="left" rowspan="1" colspan="1">âˆ’1.422</td> <td align="left" rowspan="1" colspan="1">2.547<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span></td> <td align="left" rowspan="1" colspan="1">6.517<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span></td> <td align="left" rowspan="1" colspan="1">âˆ’3.429<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span></td> <td align="left" rowspan="1" colspan="1">âˆ’.014</td> <td align="left" rowspan="1" colspan="1">âˆ’4.7<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span></td> <td align="left" rowspan="1" colspan="1">4.184<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span></td> <td align="left" rowspan="1" colspan="1">âˆ’7.944<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span></td> <td align="left" rowspan="1" colspan="1">âˆ’.027</td> </tr> <tr> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1" /> <td align="left" rowspan="1" colspan="1">pâ€Š=â€Š.145</td> <td align="left" rowspan="1" colspan="1">pâ€Š=â€Š.083</td> <td align="left" rowspan="1" colspan="1">pâ€Š=â€Š.621</td> <td align="left" rowspan="1" colspan="1">pâ€Š=â€Š.001</td> <td align="left" rowspan="1" colspan="1">pâ€Š=â€Š.721</td> <td align="left" rowspan="1" colspan="1">pâ€Š=â€Š.169</td> <td align="left" rowspan="1" colspan="1">pâ€Š=â€Š.289</td> <td align="left" rowspan="1" colspan="1">pâ€Š=â€Š.020</td> </tr> </tbody> </table> </div> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""><span class="label" tagx="label" title="label" /><p>These values are the average of 1000 bootstrap replicates of the logistic regression model. The meaning of the abbreviation shows as follow: Altâ€Š=â€Šaverage altitude; PopDenâ€Š=â€Špopulation density; PolDenâ€Š=â€Špoultry density, D2Waterâ€Š=â€Šminimal distance to inland water bodies; D2coastâ€Š=â€Šminimal distance to coastline; D2Flywayâ€Š=â€Šminimal distance to migratory bird pathways; D2Railâ€Š=â€Šminimal distance to railways; D2Roadâ€Š=â€Šminimal distance to roads. Con is the constant of the logistic regression models.</p> </div> </div> </div> <p>For Vietnam, the greatest risk occurs in the north and to a lesser extent the south of the country (<a href="#pone.0029617.s001">Figure S1</a>(f)). Unlike the work of Gilbert et al. <a href="#pone.0029617-Gilbert1">[13]</a>, our results not only model the spatial distribution of H5N1 outbreaks, but also the space-time dynamics of viral evolution. Our analysis combines real-world outbreak data with evidence on viral evolution. For instance, the H5N1 virus was first detected and became established around Hanoi, north Vietnam, in 2001 and consequently spread to the south around Ho Chi Minh city <a href="#pone.0029617-Wan1">[26]</a>. Phylogenetic analysis shows that the virus, isolated from the north, has multiple sublineages and shares a close phylogenetic relationship with the virus from Thailand, Malaysia, Laos, and provinces in southern China <a href="#pone.0029617-Chen1">[3]</a>, <a href="#pone.0029617-Smith1">[8]</a>. Furthermore, the northern H5N1 virus is associated with novel genetic subtypes, and these have facilitated the spread of the disease both within and outside the country <a href="#pone.0029617-Wan1">[26]</a>. Finally, it was reported that the number of H5N1 outbreaks decreased in the south in late 2005, but the disease still persists in causing outbreaks in northern Vietnam <a href="#pone.0029617-FAO1">[16]</a>, <a href="#pone.0029617-Wan1">[26]</a>. <a href="#pone.0029617.s001">Figure S1</a> again demonstrates that the risk estimates obtained from our integrated analysis correspond more closely than the results from Gilbert et al. to the empirical outbreak pattern at a range of different scales.</p> </div> <div class="indonesia,china,andeast-southeastasia" title="sec"> <div class="title" tagx="title" title="title">
Indonesia, China, and East-Southeast Asia</div> <p><a href="#pone-0029617-g002">Figure 2</a> shows the observed distribution of H5N1 outbreaks and our estimates of the risk of the disease in Indonesia, China, and the whole of East-Southeast Asia. <a href="#pone-0029617-t001">Table 1</a> summarizes the logistic regression analysis in the three areas.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/10.1371/journal.pone.0029617.g002"><span class="label" tagx="label" title="label">Figure 2</span></a></div>   <div class="title" tagx="title" title="title">
The spatial pattern of H5N1 in Indonesia, China, and East-Southeast Asia.</div> <p>(a), (c), (e) show the distribution of observed H5N1 outbreaks; (b), (d), and (f) show the probability maps integrating the findings of the phylogenetic analysis ( <a href="#pone.0029617.s002">Figures S2</a>(a), (d), and (g)), the modified local <i>K</i> function analysis (<a href="#pone.0029617.s002">Figures S2</a>(b), (e), and (h)), and the logistic regression analysis (<a href="#pone.0029617.s002">Figures S2</a>(c), (f), and (i)). The closer the probability is to 1, the greater is the probability of an H5N1 outbreaks.</p>   </div> <p>For Indonesia, the highest risk of H5N1 is in central Java (<a href="#pone-0029617-g002">Figure 2(b)</a>). It shows that the risk extends through the island to its surrounding archipelagos. The findings from the logistic regression analysis provide an interpretation for this pattern. <a href="#pone-0029617-t001">Table 1</a> illustrates that population density, poultry density, and the shortest path distance to railways are significantly (<i>p</i><span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>.001) associated with occurrences of the disease. The results indicate that the very high density of population (more than 940 people per km<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>) and the high concentration of poultry production (<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> of Indonesia's production takes places in Java) are underlying factors in the establishment of multiple H5N1 subgroups <a href="#pone.0029617-Smith1">[8]</a>. These factors greatly increase the risk of H5N1 occurrence in Java. In addition, the significant negative relationship between H5N1 and the shortest path distance to railways implies that avian influenza is spread to other surrounding archipelagos through the production and trading of poultry <a href="#pone.0029617-Smith1">[8]</a>.</p> <p>For China, our result shows that the highest risk of H5N1 is in Guangdong, Yunnan, Fujian, and areas close to Dongting Lake in Hunan province, south China. High risk also appears to extend along the coastline of southern China (see <a href="#pone-0029617-g002">Figure 2(d)</a>). The risk increases from the northwest to southeast. Qinghai Lake, where over 6000 migratory birds were infected and killed by HPAI H5N1 in early 2005 <a href="#pone.0029617-Chen2">[27]</a>, <a href="#pone.0029617-Liu1">[28]</a>, was also highlighted as a potential source for the disease. The logistic regression analysis identified population density and the shortest path distance to inland water bodies as significantly (<i>p</i><span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>.005) associated with the occurrence of H5N1 (<a href="#pone-0029617-t001">Table 1</a>). In eastern China, farming, free-grazing poultry, and economic trade and movement, have played significant roles in the maintenance and the transmission of avian influenza. The significant positive relationship (<i>p</i><span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>.005) between the shortest path distance to migratory bird pathways and outbreaks, however, appears to suggest that bird migration may not be a key factor triggering large outbreaks and viral transmission in China, particularly in eastern China between 1996 and 2009, even though bird migration has been widely thought as a cause of wide spread of the disease globally <a href="#pone.0029617-Chen2">[27]</a>â€"<a href="#pone.0029617-Olsen1">[29]</a>. The other socio-environmental variables (as shown in <a href="#pone-0029617-t001">Table 1</a>), including altitude, population density, poultry density, and the shortest distances to inland water bodies, coastlines, migratory bird pathways, railways, and roads, fail to show statistical significance within the two periods between 1996 and 2004, and 2005 and 2009. This might be due to data limitations associated with using such a short time span.</p> <p><a href="#pone-0029617-g002">Figure 2(f)</a> depicts the risk of occurrence of avian influenza H5N1 in East-Southeast Asia. The large-scale mapping shows that the highest risk is in central Thailand and the northern and southern parts of Vietnam. The central part of Indonesia also has high risk. Compared to these countries, China appears to have lower levels of risk, especially in the northwest including Tibet and Xinjiang autonomous regions, and the northern part of Qinghai province. This pattern can be regarded as a reflection of the relationship between the disease and heterogeneity of the local socio-ecological environment. The logistic regression analysis indicates that outbreaks significantly (<i>p</i><span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>.001) associated with altitude, population density, and the shortest path distance to inland water bodies, coastlines, railways, and roads (<a href="#pone-0029617-t001">Table 1</a>). In East-Southeast Asia, most cities and countries are usually located in areas where the environment is suitable for human habitation and agricultural production. Rice cropping and poultry rearing are popular in Thailand, Vietnam, Indonesia, and south China. These countries benefit from abundant hydrological resources (e.g., river deltas), but suffer from ecological-environmental problems caused by a rapidly increasing population which in turn facilitates the establishment of multiple H5N1 sublineages <a href="#pone.0029617-Chen2">[27]</a>. In addition, convenient transport networks like railways and roads become significant in spreading and triggering the re-occurrence of H5N1 in East-Southeast Asia.</p> </div> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <p>Three analyses are implemented on raster data frames. The probability of the occurrence of HPAI H5N1 is estimated for each lattice point (or small pixel). Specifically, this study consists of four parts. First, phylogenetic tree are built for the evolution of the H5N1 virus. The branches and the topology of the tree describe the processes of viral evolution. The ability of a virus to survive in nature (its â€œcapabilityâ€�) is a characteristic of a virus which goes through a long evolutionary process with wide spatial dispersion and persistence over time. Strong capability may lead to a high probability of the disease spreading widely. In this study, the years and localities from which the genetic sequences were sampled are collated for each H5N1 virus. Integrating in space and time, the quantification of the phylogenetic tree is a feasible way to measure and map the capability of the H5N1 virus. Second, the local <i>K</i> function, a spatial point pattern statistic <a href="#pone.0029617-Botts1">[30]</a>, is modified for the purpose of identifying the local pattern of outbreaks. The estimates obtained from the analysis are an indicator of outbreak clusters. Third spatial epidemiological analysis involves building a logistic regression model for analyzing the statistical association between the presence/absence of reported H5N1 outbreaks and eight socio-environmental variables. This model can be used to predict the probability of the occurrence of an H5N1 outbreak. The findings obtained from the three analyses provide evidence with which to explore the spatial distribution of H5N1. To present a powerful, robust, and unified result, Dempster-Shafer theory of evidence is applied to integrate the different forms of evidence.</p> <div class="data" title="sec"> <div class="title" tagx="title" title="title">
Data</div> <p>Three kinds of data were used in this study: genetic sequences, reported H5N1 outbreak records, and socio-environmental factors, including altitude, population density, poultry density, and the shortest path distances to inland water, coastlines, migratory bird pathways, railways, and roads (see <a href="#pone.0029617.s002">Figure S2</a>). All these data were collated for lattices based on different spatial resolutions: 8.4 km<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, 34.22 km<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, 0.94 km<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, 0.32 km<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, and 0.24 km<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> for East-Southeast Asia, China, Indonesia, Thailand, and Vietnam, respectively.</p> <p>First, from GenBank <a href="#pone.0029617-Benson1">[31]</a>, 888 genetic sequences of influenza A H5N1 hemagglutinin (HA) and neuraminidase (NA) genes were isolated from a variety of hosts between 1996 and 2009 across the areas of East-Southeast Asia covering Thailand, Vietnam, Laos, Cambodia, Indonesia, and China. With reported geographic localities, the sequences could be assigned to lattices by geocoding. Multiple alignment of all sequences from each HA and NA gene was carried out using MUSCLE <a href="#pone.0029617-Edgar1">[32]</a>, and the HA and NA sequences were combined using TaxonDNA <a href="#pone.0029617-Meier1">[33]</a>. A phylogenetic tree for the combined dataset was constructed using neighbor-joining (NJ) <a href="#pone.0029617-Saitou1">[34]</a> in PAUP* 4.0 <a href="#pone.0029617-Swofford1">[35]</a>(see <a href="#pone.0029617.s003">Figure S3</a>). The best-fit model of genomic RNA substitution for the NJ analysis was assessed by Model test version 3.7 <a href="#pone.0029617-Posada1">[36]</a>. Base composition and pairwise comparisons were examined using MEGA version 4.02 <a href="#pone.0029617-Tamura1">[37]</a>. The NJ algorithm was adopted in this study because this distance-based method of phylogenetic reconstruction gave the genetic distances among the sequences that needed to be combined with the results of the other analysis in the knowledge fusion step.</p> <p>Second, outbreaks of the disease were assigned to lattice points. Data on avian influenza H5N1 outbreaks in East-Southeast Asia include 2204 avian and 327 human H5N1-confirmed cases between May <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> 1997 to March <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> 2009, compiled by the World Organisation of Animal Health (OIE, <a href="http://www.oie.int">www.oie.int</a>) and the World Health Organization (WHO, <a href="http://www.who.int">www.who.int</a>), respectively. Each record contained the following attributes: country, province, location (latitude, longitude), start time, affected species, and the number of deaths. Using latitude and longitude, any outbreak could be assigned to the lattice. For each lattice point, the numbers of outbreaks of both human and avian influenza were recorded.</p> <p>Third, the socio-environmental data were compiled for the raster grids based on the spatial resolutions of East-Southeast Asia, Indonesia and China, respectively. Poultry census data in 2005 were obtained from Food and Agriculture Organization's Animal Production and Health Division (FAO-AGA).</p> <p>These poultry data, collected from sub-national livestock census data and corresponding to administrative boundaries, were converted into densities by excluding the areas unsuitable for livestock. The poultry density data were downloaded from GeoNetwork (<a href="http://www.fao.org/geonetwork/srv/en">www.fao.org/geonetwork/srv/en</a>), and collated to raster grids with each pixel value representing actual density per km<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. Population density was chosen as an indicator of the volume of viral traffic <a href="#pone.0029617-Gilbert1">[13]</a>. The 2010 estimate of population density, produced by the Center for International Earth Science Information Network (CIESIN), Columbia University and the United Nations Food and Agriculture Programme (FAO), were downloaded from the CIESIN (<a href="http://www.ciesin.org">www.ciesin.org</a>). Railways and roads were chosen because these two variables were used as surrogate indicator of long-range movements of human and poultry. The shape data for the two transport networks were provided by GIVA-GIS (<a href="http://www.diva-gis.org">www.diva-gis.org</a>), an open source for mapping and geographic data. The area and line-based data, including inland water bodies, coastline, and migratory bird pathways, downloaded from the GIVA-GIS, were used to determine the association between birds and the outbreaks. The migratory bird pathways were specified by 70 km buffers on each side because of uncertainty about the behavior of migrating birds. All shapes were converted to binary lattice data indicating presence or absence. In addition average elevation data [90_m resolution Digital Terrain Model from the Shuttle Radar Topography Mission data, STRM V3 (<a href="http://srtm.csi.cgiar.org">http://srtm.csi.cgiar.org</a>)] were used to capture topographic features that might be associated with the establishment of an H5N1 epidemic <a href="#pone.0029617-Gilbert1">[13]</a>.</p> </div> <div class="quantifyingphylogenetictree" title="sec"> <div class="title" tagx="title" title="title">
Quantifying Phylogenetic Tree</div> <p>In the phylogenetic trees of the H5N1 virus (<a href="#pone.0029617.s003">Figures S3</a> and <a href="#pone.0029617.s004">S4</a>), the process of viral evolution is composed of a set of branches, originating from a common ancestor at the root of the tree. The length of a branch describes an evolutionary stage which starts from a previous hypothesized ancestor (or a node). The taxa having a common node can be regarded as a subgroup (or a clade). By this means, a phylogenetic tree is usually divided and the taxa are grouped into different subgroups, with the members of each subgroup are phylogenetically close. In Virology, subgroups are usually determined by eye balling <a href="#pone.0029617-Li1">[2]</a>, <a href="#pone.0029617-Smith1">[8]</a>. A subgroup is believed to have a strong capability of surviving if its members show a pattern of wide spatial dispersion and extensive persistence over time <a href="#pone.0029617-Chen1">[3]</a>. This also indicates the capability of a subgroup to spread the disease. However, for a virus, this capability can be measured by both the process of the viral evolution and the capability of the subgroups which the virus belongs to. The longer is the evolutionary process the greater is the ability of a virus to survive and the higher is the possibility of the spread of the virus. Also, a virus is believed to have a large capability if it is a member of a strong subgroup of the strains showing extensive persistence in space and time. For evaluation, it is necessary to sum up all evolutionary stages that a virus goes through. Each stage contains a branch and a node which is shared by a subgroup of the taxa. As a first approximation, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, an estimate of the capability of a virus, can be measured by a linear sum, </p><div class="disp-formula" title="disp-formula"> <div class="graphic" mimetype="image" position="float" /><span class="label" tagx="label" title="label">(1)</span></div>where <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> are viral evolution, spatial dispersion, and time span at a stage <i>i</i>. <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> are weights for the three variables, assigned the values 0.4, 0.4, and 0.2, respectively. The lower weight for the time span is because a temporal scale of a year is coarse relative to the other two variables. The normalized values of <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> can be estimated from the hierarchical structure of the tree. First, the length of the branch at a stage <i>i</i> is used to measure viral evolution. Second, we measure how widely dispersed, geographically, members of the same subgroup are. Spatial dispersion is estimated by the total inertia (sum of the variances) of the 2<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>2 covariance matrix calculated using the locations (latitude, longitude) of the members at the same stage. Third, the length of time for a subgroup is measured by the time span from early to late occurrences of virus members. The time span <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> is a ratio, including the persistence of a subgroup at stage <i>i</i>. It is calculated as the time length <i>i</i> divided by the time length of the whole tree. Estimates of <i>e</i> and <i>s</i> are normalized to avoid the effect of large values. Mathematically, the capability of a virus can be evaluated by <div class="disp-formula" title="disp-formula"> <div class="graphic" mimetype="image" position="float" /><span class="label" tagx="label" title="label">(2)</span></div>where <i>n</i> is the number of evolutionary stages of the virus. Quantifying a phylogenetic tree enables us to estimate the capability of a virus.<p /> <p>To determine the range of a virus, we also need to examine the density of surrounding outbreaks at a series of scales from 10 to 250 km. Within this range, a scale <i>l</i> at which outbreak density is at a maximum was selected. The area affected by a virus is estimated using a spatial Gaussian kernel function: </p><div class="disp-formula" title="disp-formula"> <div class="graphic" mimetype="image" position="float" /><span class="label" tagx="label" title="label">(3)</span></div>If <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, the range of the affected surface is <i>l</i>; otherwise, if <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, the range is <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. <a href="#pone-0029617-g001">Figures 1(a) and (c)</a> show the capability of H5N1 in Thailand and Vietnam based on quantifying the phylogenetic trees of the H5N1 virus (<a href="#pone.0029617.s003">Figures S3</a>(a) and (b)). The quantification of the trees of Indonesia, China, and East-Southeast Asia (<a href="#pone.0029617.s003">Figures S3</a> and <a href="#pone.0029617.s004">S4</a>(c) and (d)) can be cross-referenced with to <a href="#pone.0029617.s005">Figure S5</a>(a), (d), and (g).<p /> </div> <div class="clusteranalysis" title="sec"> <div class="title" tagx="title" title="title">
Cluster Analysis</div> <p>The local <i>K</i> function, first prosed by Getis et al. <a href="#pone.0029617-Getis1">[38]</a>â€"<a href="#pone.0029617-Getis3">[40]</a>, is a statistic of spatial point patterns <a href="#pone.0029617-Botts1">[30]</a>. In this study, we apply a weighted local <i>K</i> function for the identification of clusters of outbreaks on lattices. The modified function can estimate the intensity of aggregation (for an example, see <a href="#pone.0029617.s006">Figure S6</a>) by taking into account the spatial effects arising from the locations of outbreaks. Specifically, this statistic weights the number of outbreaks by distance and can be expressed as <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> which equals the expected weighted number of outbreaks within distance <i>h</i> of a lattice point <i>i</i>. Mathematically, the estimate of the modified function is: </p><div class="disp-formula" title="disp-formula"> <div class="graphic" mimetype="image" position="float" /><span class="label" tagx="label" title="label">(4)</span></div>where <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> is the global density of outbreaks and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> is the distance between lattice points <i>i</i> and <i>j</i>. <i>I</i> is an indicator function, commonly defined as 1 for <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and 0 for <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. It indicates the outbreaks surrounding the lattice point <i>i</i> that are within a distance range <i>h</i>. Each such outbreak is weighted by the inverse Euclidean distance, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. In this study, the modified local <i>K</i> function analysis set <i>h</i>â€Š=â€Š113 km which is recommended by <a href="#pone.0029617-Viboud1">[41]</a> as the appropriate scale for exploring local patterns of influenza.<p /> <p>The estimates obtained from this analysis can be used as predictors of the occurrence of H5N1 by normalizing. <a href="#pone-0029617-g001">Figure 1(b) and (d)</a>, for instance, show the local patterns of the outbreak clusters in Thailand and Vietnam. The cluster analysis results of Indonesia, China, and East-Southeast Asia are shown in <a href="#pone.0029617.s005">Figures S5</a>(b), (e), and (h).</p> </div> <div class="riskfactoranalysis" title="sec"> <div class="title" tagx="title" title="title">
Risk Factor Analysis</div> <p>A logistic regression was fitted to model the association between the occurrence of H5N1 outbreaks and eight socio-environmental factors in East-Southeast Asia, Indonesia, and China. Data on the predictors were compiled for each lattice point, and the outbreaks converted to â€˜presence of outbreaksâ€™ (1) or â€˜absence of outbreaksâ€™ (0) for each lattice point.</p> <p>A very large majority of lattice points have no outbreaks which makes fitting the logistic model to all the lattice points unsatisfactory (see also <a href="#pone.0029617-Gilbert1">[13]</a>). We build models by selecting all outbreak-present lattice points and randomly selecting double the number of lattices points with no outbreaks. The number of H5N1 outbreaks in East-Southeast Asia including Indonesia and China are much lower than in Thailand and Vietnam. One thousand bootstrap replicates were implemented to ensure a satisfactory sample size for carrying out model inference. In each repeat, the coefficient and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> value of each predictor were estimated. Cohen's kappa index was used to evaluate the observed/model predicted misclassification matrix and Nagelkerke/Cragg &amp;amp; Uhler's psuedo-<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> was used as the goodness-of-fit statistic for the logistic regression models. In addition, the receiver-operating characteristic ROC provides a two-dimensional depiction of predictive performance <a href="#pone.0029617-Fawcett1">[42]</a>. The area under the ROC curve (AUC) measures the probability of a correct classification <a href="#pone.0029617-Hanely1">[43]</a>( <a href="#pone.0029617.s007">Figure S7</a>). The ROC, Cohen's kappa index, and Nagelkerke/Cragg &amp;amp; Uhler's psuedo-<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> were calculated for the purpose of assessing the predictive power of the model (<a href="#pone.0029617.s008">Table S1</a>). All these estimates were averaged over the 1000 bootstrap replicates. The predictions for the H5N1 in Indonesia, China, and East-Southeast Asia are shown in <a href="#pone.0029617.s005">Figures S5</a>(c), (f), and (i).</p> </div> <div class="knowledgefusionandthedempster-shafertheoryofevidence" title="sec"> <div class="title" tagx="title" title="title">
Knowledge Fusion and the Dempster-Shafer Theory of Evidence</div> <p>Knowledge fusion provides a framework for integrating information from different research domains. Uncertainty, perhaps arising from incomplete data, limited domain knowledge, or the application of an insufficiently sophisticated methodology, could limit the value of the above analyses. Dempster-Shafer (D-S) theory of evidence seeks to overcome the limitations associated with conventional probability theory when the researcher seeks to quantify and reason with imprecise, uncertain and/or weak information <a href="#pone.0029617-Dempster1">[24]</a>, <a href="#pone.0029617-Shafer1">[25]</a>. It has been widely used in applications involving the use of geographical information systems <a href="#pone.0029617-Malpica1">[44]</a> and image processing technologies <a href="#pone.0029617-Kaftandjian1">[45]</a>, <a href="#pone.0029617-Adamek1">[46]</a>; it has also been used in climate change research <a href="#pone.0029617-Luo1">[47]</a>, detecting credit card fraud <a href="#pone.0029617-Panigrahi1">[48]</a>, and in evaluating outcomes associated with medical intervention <a href="#pone.0029617-Jones1">[49]</a>.</p> <p>Recently, this method has been employed to identify areas at risk from rift valley fever in Africa <a href="#pone.0029617-Clements1">[50]</a>. To provide a formal description of D-S theory, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> defines a finite set of mutually exclusive and exhaustive elementary hypothesis <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> called the frame of discernment (<i>FOD</i>). The number of possible subsets of <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> is <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, including the full and null hypotheses. In this study, each lattice point is a binary frame of a discernment containing two elementary hypotheses: <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> for the presence of H5N1 risk and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> for absence, i.e., <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. The subsets of <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> are <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. In particular, the subset <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> stand for empty and unknown (or uncertainty).</p> <p>In D-S theory, each subset is assigned a belief value by the available evidence, called probability mass function, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. In particular, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, where <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> represents the subsets <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and 0<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>1. In this study, the results of the three analyses (the capability measure of the H5N1 virus, the clusters of outbreaks, and the results from the logistic regression analysis) provide evidence indicating the belief for the risk of occurrence of H5N1, denoted by <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. However, no evidence is directly provided by the first two studies regarding the absence of the disease. Therefore, the values of <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> cannot be determined and so the mass functions of unknown, i.e. <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, are assigned: <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. The logistic regression model results are different however because the predictive power of the model allows us to assign values to the mass functions. The average AUC is used to assess model predictions. The multiplication of the predicted value and the average AUC is the evidence for <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. The values of <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> are thus <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, respectively.</p> <p>The Dempster's rule of combination offers an approach to combining evidence from different sources. The joint probability mass function, for instance <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, can be obtained from the combination of the two mass functions, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>: </p><div class="disp-formula" title="disp-formula"> <div class="graphic" mimetype="image" position="float" /><span class="label" tagx="label" title="label">(5)</span></div>where <div class="disp-formula" title="disp-formula"> <div class="graphic" mimetype="image" position="float" /><span class="label" tagx="label" title="label">(6)</span></div>where <i>k</i> is a normalization factor, and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> and <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span> are subsets of <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. Dempster's rule is commutative and associative, and thus the joint mass function is independent of the order in which evidence are combined <a href="#pone.0029617-Dempster1">[24]</a>.<p /> <p>In this study, the three sources of evidence are combined via an iterative procedure in order to identify for each lattice point the â€˜degree of beliefâ€™ we have in the likely occurrence of H5N1. To simplify the text we have referred to this as the â€œriskâ€�. Thailand, for example, is represented by a matrix 3088<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>1738 lattice points. <a href="#pone.0029617.s001">Figure S1</a>(c) shows the integrated result of the H5N1 epidemic in Thailand. Dempster's combination procedure is demonstrated for Thailand in <a href="#pone.0029617.s009">Table S2</a>.</p> </div> <div class="spatialcorrespondenceanalysis" title="sec"> <div class="title" tagx="title" title="title">
Spatial Correspondence Analysis</div> <p>Spatial correspondence analysis using Pearson's correlation coefficient, <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>, <a href="#pone.0029617-Haining1">[51]</a>, <a href="#pone.0029617-Haining2">[52]</a> measures the association between observed avian-influenza outbreaks and model predictions. A sample size adjusted t-test was employed to test statistical significance of <span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>. The reduction in the degrees of freedom is a function of the level of spatial autocorrelation in the two maps that are being correlated. This is because spatial autocorrelation introduces redundancy into a set of data and the adjustment procedure identifies the â€œequivalentâ€� number of independent observations.</p> <p>For the implementation, lattices were aggregated into blocks varying in size from 60<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>60 to 30<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>30 because the number of cases is small relative to the original spatial resolution of the data which is extremely fine-grained. The observed outbreaks were converted to a rate by calculating for each areal unit the number of outbreak cases divided by the population at risk which is the human plus poultry population, (number of outbreak)/(population<span class="inline-formula" title="inline-formula"><span class="inline-graphic" title="inline-graphic" /></span>poultry). Model predictions were obtained by averaging results across the lattice points falling within any block. Results from the analysis of spatial correspondence using the sample size adjusted t-test for significance testing are shown in <a href="#pone.0029617.s001">Figure S1</a>.</p> </div> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supporting Information</div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure S1</span> <p><b>Test for spatial correspondence of H5N1 outbreaks and the empirical patterns.</b></p> <p>(TIF)</p>  <div class="media" title="media"><a href="pone.0029617.s001.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure S2</span> <p><b>Spatial distribution of H5N1: spatial distribution of reported H5N1 outbreaks, inland water bodies, migratory bird pathways, pathways, and roads in East-Southeast Asia.</b></p> <p>(TIFF)</p>  <div class="media" title="media"><a href="pone.0029617.s002.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure S3</span> <p><b>NJ tree of the 888 H5N1 concatenation of hemagglutinin (HA) and neuraminidase (NA) DNA sequences from East-Southeast Asia covering Thailand, Vietnam, Cambodia, Laos, Indonesia, and China.</b> The best model is TVM+I+G (transversional model incorporating invariable sites and rate variation among sites). The goose H5N1 DNA sequence from Guangdong in 1996 (A/Goose/Guangdong/1/96) is used to root the tree. The length of a unit is .02. The taxa are colored by locality with green for Thailand, blue for Vietnam, orange for Indonesia, and black for all others including China, Cambodia, Laos, and purple, especially, for Qinghai province, western China.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="pone.0029617.s003.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure S4</span> <p><b>NJ trees for the H5N1 virus from (a) Thailand, (b) Vietnam, (c) Indonesia, and (d) China.</b> The best model is GTR+I+G (General Time Reversible incorporating invariable sites and rate variation among sites.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="pone.0029617.s004.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure S5</span> <p><b>Results from phylogenetic tree analysis, modified local </b><b><i>K</i></b><b> function analysis, and logistic regression analysis for Indonesia, China, East-Southeast Asia.</b> The values shown in the color bar are the probability of outbreaks of H5N1. (a), (d), (g) are the outcomes from the phylogenetic tree analysis (Figure 3(b) and <a href="#pone.0029617.s003">Figures S3</a>(c) and (d)), showing the spatial profile of the capability of the H5N1 virus; (b), (e), and (h) are the results from the modified local <i>K</i> function analysis, depicting the spatial distribution of outbreak clusters; (c), (f), and (i) show the predictive results from the logistic regression models.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="pone.0029617.s005.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure S6</span> <p><b>Illustration of the original and modified local </b><b><i>K</i></b><b> function.</b><i>Note</i>: the original local <i>K</i> function based on the number of outbreaks cannot distinguish between the cluster with the red and blue circles. The modified local <i>K</i> function, by taking into account the distance between outbreaks, is able to distinguish the two patterns.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="pone.0029617.s006.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Figure S7</span> <p><b>ROC curves for the logistic regression model.</b> The area under the ROC curve (AUC) indicates the probability of a correct classification. (a) and (b) show the average AUC of the models for outbreaks of East-Southeast Asia and Indonesia from 1996 to 2009. (c), (d), (e) are the AUC of the models for China for the periods between 1996 and 2009, 1996 and 2004, and 2005 and 2009, respectively. The blue areas are the envelopes of the 1000 bootstrap replicates.</p> <p>(TIF)</p>  <div class="media" title="media"><a href="pone.0029617.s007.tif">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Table S1</span> <p>Logistic regression model assemssment for the H5N1 occurrences in East-Southeast Asia, Inodnesia, and China, 1996â€"2009 and the two epidemic waves between 1996â€"2004 and 2005â€"2009.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pone.0029617.s008.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"><span class="label" tagx="label" title="label">Table S2</span> <p>Dempster's combination for the three sources of evidence on H5N1 in Thailand.</p> <p>(PDF)</p>  <div class="media" title="media"><a href="pone.0029617.s009.pdf">LINK</a> <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <p>The authors would like to thank Dr. M. Gilbert for useful discussion and providing their results for our fusion task.</p> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-COI-statement" title="COI-statement"> <p><b>Competing Interests: </b>The authors have declared that no competing interests exist.</p> </div> <div class="fn-type-financial-disclosure" title="financial-disclosure"> <p><b>Funding: </b>This work was supported by an earmarked grant (CUHK447109) of the Hong Kong Research Grants Council. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="pone.0029617-Xu1" /><span class="label" tagx="label" title="label">1</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Subbarao</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cox</span><span class="given-names" tagx="given-names" title="given-names">NJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span></span><span class="year" tagx="year" title="year">1999</span><span class="mixed-article-title" title="mixed-article-title">Genetic characterization of the pathogenic influenza A/Goose/Guangdong/1/96 (H5N1) virus: Similarity of its Hemagglutinin gene to those of H5N1 viruses from the 1997 outbreaks in Hong Kong.</span><span class="source" tagx="source" title="source">Virology</span><span class="volume" tagx="volume" title="volume">261</span><span class="fpage" tagx="fpage" title="fpage">15</span><span class="lpage" tagx="lpage" title="lpage">19</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10484749">10484749</a></span></span></li> <li tag="ref"><a name="pone.0029617-Li1" /><span class="label" tagx="label" title="label">2</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">KS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guan</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">GJD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">KM</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">Genesis of a highly pathogenic and potentially pandemic H5N1 influenza virus in eastern Asia.</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">430</span><span class="fpage" tagx="fpage" title="fpage">209</span><span class="lpage" tagx="lpage" title="lpage">213</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15241415">15241415</a></span></span></li> <li tag="ref"><a name="pone.0029617-Chen1" /><span class="label" tagx="label" title="label">3</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">GJD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">KS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fan</span><span class="given-names" tagx="given-names" title="given-names">XH</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Establishment of multiple sublineages of H5N1 influenza virus in Asia: Implications for pandemic control.</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span><span class="volume" tagx="volume" title="volume">103</span><span class="fpage" tagx="fpage" title="fpage">2845</span><span class="lpage" tagx="lpage" title="lpage">2850</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16473931">16473931</a></span></span></li> <li tag="ref"><a name="pone.0029617-Enserink1" /><span class="label" tagx="label" title="label">4</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Enserink</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">H5N1 moves into Africa, European Union, deepening global crisis.</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">311</span><span class="fpage" tagx="fpage" title="fpage">932</span></span></li> <li tag="ref"><a name="pone.0029617-WHO1" /><span class="label" tagx="label" title="label">5</span><span class="element-citation'"><span class="collab'">collab: WHO</span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Cumulative number of confirmed human cases of avian influenza A/(H5N1).</span><span class="comment">Available: <a href="http://www.who.int/csr/disease/avian">http://www.who.int/csr/disease/avian</a> influenza/country/cases table 2010 08 31/en/index.html. Accessed 2010 Sep 17</span></span></li> <li tag="ref"><a name="pone.0029617-Preiser1" /><span class="label" tagx="label" title="label">6</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Preiser</span><span class="given-names" tagx="given-names" title="given-names">KH</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="source" tagx="source" title="source">Influenza Report 2006, Waye MY, editor</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Flying Publisher</span></span></li> <li tag="ref"><a name="pone.0029617-Dawood1" /><span class="label" tagx="label" title="label">7</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dawood</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jain</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Finelli</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shaw</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lindstrom</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Emergence of a novel swine-origin influenza A (H1N1) virus in humans.</span><span class="source" tagx="source" title="source">N Engl J Med</span><span class="volume" tagx="volume" title="volume">25</span><span class="fpage" tagx="fpage" title="fpage">2605</span><span class="lpage" tagx="lpage" title="lpage">2615</span></span></li> <li tag="ref"><a name="pone.0029617-Smith1" /><span class="label" tagx="label" title="label">8</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">GJD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Naipospos</span><span class="given-names" tagx="given-names" title="given-names">TSP</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguyen</span><span class="given-names" tagx="given-names" title="given-names">TD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Jong</span><span class="given-names" tagx="given-names" title="given-names">MD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vijaykrishna</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Evolution and adaptation of H5N1 influenza virus in avian and human hosts in Indonesia and Vietnam.</span><span class="source" tagx="source" title="source">Virology</span><span class="volume" tagx="volume" title="volume">350</span><span class="fpage" tagx="fpage" title="fpage">258</span><span class="lpage" tagx="lpage" title="lpage">268</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16713612">16713612</a></span></span></li> <li tag="ref"><a name="pone.0029617-Duan1" /><span class="label" tagx="label" title="label">9</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duan</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Campitelli</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fan</span><span class="given-names" tagx="given-names" title="given-names">XH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Leung</span><span class="given-names" tagx="given-names" title="given-names">YHC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vijaykrishna</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Characterization of low-pathogenic H5 subtype influenza viruses from eurasia: Implications for the origin of highly pathogenic H5N1 viruses.</span><span class="source" tagx="source" title="source">J Virol</span><span class="volume" tagx="volume" title="volume">81</span><span class="fpage" tagx="fpage" title="fpage">7529</span><span class="lpage" tagx="lpage" title="lpage">7539</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17507485">17507485</a></span></span></li> <li tag="ref"><a name="pone.0029617-Avise1" /><span class="label" tagx="label" title="label">10</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Avise</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arnold</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ball</span><span class="given-names" tagx="given-names" title="given-names">RM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bermingham</span><span class="given-names" tagx="given-names" title="given-names">E</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lamb</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">1987</span><span class="mixed-article-title" title="mixed-article-title">Intraspecific phylogeography: The mitochondrial DNA bridge between population genetics and systematics.</span><span class="source" tagx="source" title="source">Annu Rev Ecol Syst</span><span class="volume" tagx="volume" title="volume">18</span><span class="fpage" tagx="fpage" title="fpage">489</span><span class="lpage" tagx="lpage" title="lpage">522</span></span></li> <li tag="ref"><a name="pone.0029617-Wallace1" /><span class="label" tagx="label" title="label">11</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wallace</span><span class="given-names" tagx="given-names" title="given-names">RG</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hodac</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lathrop</span><span class="given-names" tagx="given-names" title="given-names">RH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fitch</span><span class="given-names" tagx="given-names" title="given-names">WM</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">A statistical phylogeography of influenza A H5N1.</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span><span class="volume" tagx="volume" title="volume">104</span><span class="fpage" tagx="fpage" title="fpage">4473</span><span class="lpage" tagx="lpage" title="lpage">4478</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17360548">17360548</a></span></span></li> <li tag="ref"><a name="pone.0029617-Si1" /><span class="label" tagx="label" title="label">12</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Si</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Skidmore</span><span class="given-names" tagx="given-names" title="given-names">AK</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Boer</span><span class="given-names" tagx="given-names" title="given-names">WF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Debba</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Spatio-temporal dynamics of global H5N1 outbreaks match bird migration patterns.</span><span class="source" tagx="source" title="source">Geospatial Health</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">65</span><span class="lpage" tagx="lpage" title="lpage">78</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19908191">19908191</a></span></span></li> <li tag="ref"><a name="pone.0029617-Gilbert1" /><span class="label" tagx="label" title="label">13</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gilbert</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiao</span><span class="given-names" tagx="given-names" title="given-names">X</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pfeiffer</span><span class="given-names" tagx="given-names" title="given-names">DU</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Epprecht</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boles</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Mapping H5N1 highly pathogenic avian influenza risk in southeast Asia.</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span><span class="volume" tagx="volume" title="volume">105</span><span class="fpage" tagx="fpage" title="fpage">4769</span><span class="lpage" tagx="lpage" title="lpage">4774</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18362346">18362346</a></span></span></li> <li tag="ref"><a name="pone.0029617-Openshaw1" /><span class="label" tagx="label" title="label">14</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Openshaw</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">1984</span><span class="source" tagx="source" title="source">The Modifiable Areal Unit Problem</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Norwich, UK</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Geo Books</span><span class="comment">Printed by Headley Brothers Ltd. Kent</span></span></li> <li tag="ref"><a name="pone.0029617-Fang1" /><span class="label" tagx="label" title="label">15</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fang</span><span class="given-names" tagx="given-names" title="given-names">LQ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Vlas</span><span class="given-names" tagx="given-names" title="given-names">SJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Looman</span><span class="given-names" tagx="given-names" title="given-names">CWN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gong</span><span class="given-names" tagx="given-names" title="given-names">P</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Environmental factors contributing to the spread of H5N1 avian influenza in mainland China.</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">e2268</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18509468">18509468</a></span></span></li> <li tag="ref"><a name="pone.0029617-FAO1" /><span class="label" tagx="label" title="label">16</span><span class="element-citation'"><span class="collab'">collab: FAO</span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Technical task force on avian influenza. Update on the avian influenza situation.</span><span class="comment">Available: <a href="http://www.fao.org/avianu/en/aidenews.html">http://www.fao.org/avianu/en/aidenews.html</a>. Accessed 2010 Aug 5</span></span></li> <li tag="ref"><a name="pone.0029617-Smith2" /><span class="label" tagx="label" title="label">17</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">GJD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fan</span><span class="given-names" tagx="given-names" title="given-names">XH</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">KS</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qin</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Emergence and predominance of an H5N1 influenza variant in China.</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span><span class="volume" tagx="volume" title="volume">103</span><span class="fpage" tagx="fpage" title="fpage">16936</span><span class="lpage" tagx="lpage" title="lpage">16941</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17075062">17075062</a></span></span></li> <li tag="ref"><a name="pone.0029617-SmallmanRaynor1" /><span class="label" tagx="label" title="label">18</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smallman-Raynor</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cliff</span><span class="given-names" tagx="given-names" title="given-names">AD</span></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">The geographical spread of avian influenza A (H5N1): Pan- zootic transmission (December 2003â€"May 2006), pandemic potential, and implications.</span><span class="source" tagx="source" title="source">Ann Assoc Am Geogr</span><span class="volume" tagx="volume" title="volume">98</span><span class="fpage" tagx="fpage" title="fpage">553</span><span class="lpage" tagx="lpage" title="lpage">582</span></span></li> <li tag="ref"><a name="pone.0029617-CavalliSforza1" /><span class="label" tagx="label" title="label">19</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cavalli-Sforza</span><span class="given-names" tagx="given-names" title="given-names">LL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edwards</span><span class="given-names" tagx="given-names" title="given-names">AWF</span></span></span><span class="year" tagx="year" title="year">1967</span><span class="mixed-article-title" title="mixed-article-title">Phylogenetic analysis: Models and estimation procedures.</span><span class="source" tagx="source" title="source">Am J Hum Genet</span><span class="volume" tagx="volume" title="volume">19</span><span class="fpage" tagx="fpage" title="fpage">233</span><span class="lpage" tagx="lpage" title="lpage">257</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6026583">6026583</a></span></span></li> <li tag="ref"><a name="pone.0029617-Penny1" /><span class="label" tagx="label" title="label">20</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Penny</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hendy</span><span class="given-names" tagx="given-names" title="given-names">MD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Steel</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span></span><span class="year" tagx="year" title="year">1992</span><span class="mixed-article-title" title="mixed-article-title">Progress with methods for constructing evolutionary trees.</span><span class="source" tagx="source" title="source">Trends Ecol Evol</span><span class="volume" tagx="volume" title="volume">7</span><span class="fpage" tagx="fpage" title="fpage">73</span><span class="lpage" tagx="lpage" title="lpage">79</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21235960">21235960</a></span></span></li> <li tag="ref"><a name="pone.0029617-Kilpatrick1" /><span class="label" tagx="label" title="label">21</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kilpatrick</span><span class="given-names" tagx="given-names" title="given-names">AM</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chmura</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gibbons</span><span class="given-names" tagx="given-names" title="given-names">DW</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fleischer</span><span class="given-names" tagx="given-names" title="given-names">RC</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marra</span><span class="given-names" tagx="given-names" title="given-names">PP</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Predicting the global spread of H5N1 avian influenza.</span><span class="source" tagx="source" title="source">Proc Natl Acad Sci U S A</span><span class="volume" tagx="volume" title="volume">103</span><span class="fpage" tagx="fpage" title="fpage">19368</span><span class="lpage" tagx="lpage" title="lpage">19373</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17158217">17158217</a></span></span></li> <li tag="ref"><a name="pone.0029617-Liang1" /><span class="label" tagx="label" title="label">22</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liang</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xu</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">W</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Combining spatial-temporal and phylogenetic analysis approaches for improved understanding on global H5N1 transmission.</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">e13575</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21042591">21042591</a></span></span></li> <li tag="ref"><a name="pone.0029617-Carrel1" /><span class="label" tagx="label" title="label">23</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Carrel</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Emch</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jobe</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moody</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wan</span><span class="given-names" tagx="given-names" title="given-names">X</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Spatiotemporal structure of molecular evolution of H5N1 highly pathogenic avian influenza in Vietnam.</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">e86311</span><span class="lpage" tagx="lpage" title="lpage">e86319</span></span></li> <li tag="ref"><a name="pone.0029617-Dempster1" /><span class="label" tagx="label" title="label">24</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dempster</span><span class="given-names" tagx="given-names" title="given-names">AP</span></span></span><span class="year" tagx="year" title="year">1967</span><span class="mixed-article-title" title="mixed-article-title">Upper and lower probabilities induced by a multivalued mapping.</span><span class="source" tagx="source" title="source">Ann Math Stat</span><span class="volume" tagx="volume" title="volume">38</span><span class="fpage" tagx="fpage" title="fpage">325</span><span class="lpage" tagx="lpage" title="lpage">339</span></span></li> <li tag="ref"><a name="pone.0029617-Shafer1" /><span class="label" tagx="label" title="label">25</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Shafer</span><span class="given-names" tagx="given-names" title="given-names">G</span></span></span><span class="year" tagx="year" title="year">1976</span><span class="source" tagx="source" title="source">A Mathematical Theory of Evidence</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Princeton, NJ</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Princeton University Press</span><span class="size" title="size">size: 298page</span></span></li> <li tag="ref"><a name="pone.0029617-Wan1" /><span class="label" tagx="label" title="label">26</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wan</span><span class="given-names" tagx="given-names" title="given-names">XF</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nguyen</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Davis</span><span class="given-names" tagx="given-names" title="given-names">CT</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">CB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhao</span><span class="given-names" tagx="given-names" title="given-names">ZM</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2008</span><span class="mixed-article-title" title="mixed-article-title">Evolution of highly pathogenic H5N1 avian influenza viruses in Vietnam between 2001 and 2007.</span><span class="source" tagx="source" title="source">PLoS ONE</span><span class="volume" tagx="volume" title="volume">3</span><span class="fpage" tagx="fpage" title="fpage">e3462</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18941631">18941631</a></span></span></li> <li tag="ref"><a name="pone.0029617-Chen2" /><span class="label" tagx="label" title="label">27</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">GJD</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">SY</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qin</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">Avian u: H5N1 virus outbreak in migratory waterfowl.</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">436</span><span class="fpage" tagx="fpage" title="fpage">191</span><span class="lpage" tagx="lpage" title="lpage">192</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16007072">16007072</a></span></span></li> <li tag="ref"><a name="pone.0029617-Liu1" /><span class="label" tagx="label" title="label">28</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Liu</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Xiao</span><span class="given-names" tagx="given-names" title="given-names">H</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lei</span><span class="given-names" tagx="given-names" title="given-names">F</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">Q</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Qin</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2005</span><span class="mixed-article-title" title="mixed-article-title">Highly pathogenic H5N1 influenza virus infection in migratory birds.</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">309</span><span class="fpage" tagx="fpage" title="fpage">1206</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16000410">16000410</a></span></span></li> <li tag="ref"><a name="pone.0029617-Olsen1" /><span class="label" tagx="label" title="label">29</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Olsen</span><span class="given-names" tagx="given-names" title="given-names">B</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Munster</span><span class="given-names" tagx="given-names" title="given-names">VJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wallensten</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Waldenstrm</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Osterhaus</span><span class="given-names" tagx="given-names" title="given-names">ADME</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Global patterns of influenza a virus in wild birds.</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">312</span><span class="fpage" tagx="fpage" title="fpage">384</span><span class="lpage" tagx="lpage" title="lpage">388</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16627734">16627734</a></span></span></li> <li tag="ref"><a name="pone.0029617-Botts1" /><span class="label" tagx="label" title="label">30</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Botts</span><span class="given-names" tagx="given-names" title="given-names">BN</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Getis</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">1988</span><span class="mixed-article-title" title="mixed-article-title">Point Pattern Analysis.</span><span class="source" tagx="source" title="source">Scientific geography series, Vol. 8</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Thousand Oaks, CA</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Sage publications, Inc</span><span class="fpage" tagx="fpage" title="fpage">1</span><span class="lpage" tagx="lpage" title="lpage">93</span></span></li> <li tag="ref"><a name="pone.0029617-Benson1" /><span class="label" tagx="label" title="label">31</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Benson</span><span class="given-names" tagx="given-names" title="given-names">DA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Karsch-Mizrachi</span><span class="given-names" tagx="given-names" title="given-names">I</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Lipman</span><span class="given-names" tagx="given-names" title="given-names">DJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ostell</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wheeler</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">GenBank.</span><span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="volume" tagx="volume" title="volume">34</span><span class="fpage" tagx="fpage" title="fpage">D16</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16381837">16381837</a></span></span></li> <li tag="ref"><a name="pone.0029617-Edgar1" /><span class="label" tagx="label" title="label">32</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Edgar</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">2004</span><span class="mixed-article-title" title="mixed-article-title">MUSCLE: Multiple sequence alignment with high accuracy and high throughput.</span><span class="source" tagx="source" title="source">Nucleic Acids Res</span><span class="volume" tagx="volume" title="volume">32</span><span class="fpage" tagx="fpage" title="fpage">1792</span><span class="lpage" tagx="lpage" title="lpage">1797</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15034147">15034147</a></span></span></li> <li tag="ref"><a name="pone.0029617-Meier1" /><span class="label" tagx="label" title="label">33</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meier</span><span class="given-names" tagx="given-names" title="given-names">R</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kwong</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vaidva</span><span class="given-names" tagx="given-names" title="given-names">G</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ng</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">DNA barcoding and taxonomy in Diptera: A tale of high intraspecific and low identification succeess.</span><span class="source" tagx="source" title="source">Syst Biol</span><span class="volume" tagx="volume" title="volume">55</span><span class="fpage" tagx="fpage" title="fpage">715</span><span class="lpage" tagx="lpage" title="lpage">728</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17060194">17060194</a></span></span></li> <li tag="ref"><a name="pone.0029617-Saitou1" /><span class="label" tagx="label" title="label">34</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Saitou</span><span class="given-names" tagx="given-names" title="given-names">N</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nei</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">1987</span><span class="mixed-article-title" title="mixed-article-title">The neighbor-joining method: A new method for reconstructing phyloge- netic trees.</span><span class="source" tagx="source" title="source">Mol Bio Evol</span><span class="volume" tagx="volume" title="volume">4</span><span class="fpage" tagx="fpage" title="fpage">406</span><span class="lpage" tagx="lpage" title="lpage">425</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3447015">3447015</a></span></span></li> <li tag="ref"><a name="pone.0029617-Swofford1" /><span class="label" tagx="label" title="label">35</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Swofford</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">2002</span><span class="source" tagx="source" title="source">PAUP*: Phylogenetic Analysis Using Parsimony (* and other methods)</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Sunderland, MA</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Sinauer Associates</span><span class="comment">Available: <a href="http://www.sinauer.com/detail.php?id=8060">http://www.sinauer.com/detail.php?id=8060</a>. Accessed 2012 May 2</span></span></li> <li tag="ref"><a name="pone.0029617-Posada1" /><span class="label" tagx="label" title="label">36</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Posada</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Crandall</span><span class="given-names" tagx="given-names" title="given-names">K</span></span></span><span class="year" tagx="year" title="year">1998</span><span class="mixed-article-title" title="mixed-article-title">MODELTEST: Testing the model of DNA substitution.</span><span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">14</span><span class="fpage" tagx="fpage" title="fpage">817</span><span class="lpage" tagx="lpage" title="lpage">818</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9918953">9918953</a></span></span></li> <li tag="ref"><a name="pone.0029617-Tamura1" /><span class="label" tagx="label" title="label">37</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tamura</span><span class="given-names" tagx="given-names" title="given-names">K</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dudley</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nei</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kumar</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">MEGA4: Molecular evolutionary genetics analysis (MEGA) software version 4.0.</span><span class="source" tagx="source" title="source">Mol Bio Evol</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">1596</span><span class="lpage" tagx="lpage" title="lpage">1599</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17488738">17488738</a></span></span></li> <li tag="ref"><a name="pone.0029617-Getis1" /><span class="label" tagx="label" title="label">38</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Getis</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">1984</span><span class="mixed-article-title" title="mixed-article-title">Interaction modeling using second-order analysis.</span><span class="source" tagx="source" title="source">Environ Plann A</span><span class="volume" tagx="volume" title="volume">16</span><span class="fpage" tagx="fpage" title="fpage">173</span><span class="lpage" tagx="lpage" title="lpage">183</span></span></li> <li tag="ref"><a name="pone.0029617-Getis2" /><span class="label" tagx="label" title="label">39</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Getis</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Franklin</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">1987</span><span class="mixed-article-title" title="mixed-article-title">Second-order neighborhood analysis of mapped point patterns.</span><span class="source" tagx="source" title="source">Ecology</span><span class="volume" tagx="volume" title="volume">68</span><span class="fpage" tagx="fpage" title="fpage">473</span><span class="lpage" tagx="lpage" title="lpage">477</span></span></li> <li tag="ref"><a name="pone.0029617-Getis3" /><span class="label" tagx="label" title="label">40</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Getis</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ord</span><span class="given-names" tagx="given-names" title="given-names">JK</span></span></span><span class="year" tagx="year" title="year">1996</span><span class="source" tagx="source" title="source">Local Spatial Statistics: An overview</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Cambridge</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Geoinformation International</span><span class="fpage" tagx="fpage" title="fpage">261</span><span class="lpage" tagx="lpage" title="lpage">277</span></span></li> <li tag="ref"><a name="pone.0029617-Viboud1" /><span class="label" tagx="label" title="label">41</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Viboud</span><span class="given-names" tagx="given-names" title="given-names">C</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bjornstad</span><span class="given-names" tagx="given-names" title="given-names">ON</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simonsen</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Miller</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span><span class="etal" title="etal"><i>et al.</i></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Synchrony, waves, and spatial hierarchies in the spread of influenza.</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">312</span><span class="fpage" tagx="fpage" title="fpage">447</span><span class="lpage" tagx="lpage" title="lpage">451</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16574822">16574822</a></span></span></li> <li tag="ref"><a name="pone.0029617-Fawcett1" /><span class="label" tagx="label" title="label">42</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Fawcett</span><span class="given-names" tagx="given-names" title="given-names">T</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">An introduction to ROC analysis.</span><span class="source" tagx="source" title="source">Pattern Recogn Lett</span><span class="volume" tagx="volume" title="volume">27</span><span class="fpage" tagx="fpage" title="fpage">861</span><span class="lpage" tagx="lpage" title="lpage">874</span></span></li> <li tag="ref"><a name="pone.0029617-Hanely1" /><span class="label" tagx="label" title="label">43</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hanely</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McNeil</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="year" tagx="year" title="year">1982</span><span class="mixed-article-title" title="mixed-article-title">The meaning and use of the area under a receiver operating characteristic (ROC) curve.</span><span class="source" tagx="source" title="source">Radiology</span><span class="volume" tagx="volume" title="volume">143</span><span class="fpage" tagx="fpage" title="fpage">29</span><span class="lpage" tagx="lpage" title="lpage">36</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7063747">7063747</a></span></span></li> <li tag="ref"><a name="pone.0029617-Malpica1" /><span class="label" tagx="label" title="label">44</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Malpica</span><span class="given-names" tagx="given-names" title="given-names">J</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Alonso</span><span class="given-names" tagx="given-names" title="given-names">M</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sanz</span><span class="given-names" tagx="given-names" title="given-names">M</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Dempster-Shafer theory in geographic information systems: A survey.</span><span class="source" tagx="source" title="source">Expert Syst Appl</span><span class="volume" tagx="volume" title="volume">32</span><span class="fpage" tagx="fpage" title="fpage">47</span><span class="lpage" tagx="lpage" title="lpage">55</span></span></li> <li tag="ref"><a name="pone.0029617-Kaftandjian1" /><span class="label" tagx="label" title="label">45</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kaftandjian</span><span class="given-names" tagx="given-names" title="given-names">V</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dupuis</span><span class="given-names" tagx="given-names" title="given-names">O</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Babot</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhu</span><span class="given-names" tagx="given-names" title="given-names">YM</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="mixed-article-title" title="mixed-article-title">Uncertainty modelling using Dempster-Shafer theory for improving detection of weld defects.</span><span class="source" tagx="source" title="source">Pattern Recogn Lett</span><span class="volume" tagx="volume" title="volume">24</span><span class="fpage" tagx="fpage" title="fpage">547</span><span class="lpage" tagx="lpage" title="lpage">564</span></span></li> <li tag="ref"><a name="pone.0029617-Adamek1" /><span class="label" tagx="label" title="label">46</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adamek</span><span class="given-names" tagx="given-names" title="given-names">T</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">O'Connor</span><span class="given-names" tagx="given-names" title="given-names">N</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">Using Dempster-Shafer theory to fuse multiple information sources in region-based segmentation.</span><span class="fpage" tagx="fpage" title="fpage">II-269</span><span class="lpage" tagx="lpage" title="lpage">II272</span><span class="comment">In: IEEE International Conference on Image Processing, 2007 (ICIP 2007). volume 2 of <i>IEEE International Conference on Image Processing, 2007 (ICIP 2007)</i></span><span class="comment">doi:<a href="http://dx.doi.org/10.1109/ICIP.2007.4379144">10.1109/ICIP.2007.4379144</a></span></span></li> <li tag="ref"><a name="pone.0029617-Luo1" /><span class="label" tagx="label" title="label">47</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Luo</span><span class="given-names" tagx="given-names" title="given-names">WB</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Caselton</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="year" tagx="year" title="year">1997</span><span class="mixed-article-title" title="mixed-article-title">Using Dempster-Shafer theory to represent climate change uncertainties.</span><span class="source" tagx="source" title="source">J Environ Manage</span><span class="volume" tagx="volume" title="volume">49</span><span class="fpage" tagx="fpage" title="fpage">73</span><span class="lpage" tagx="lpage" title="lpage">93</span></span></li> <li tag="ref"><a name="pone.0029617-Panigrahi1" /><span class="label" tagx="label" title="label">48</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Panigrahi</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kundu</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sural</span><span class="given-names" tagx="given-names" title="given-names">S</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Majumdar</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2009</span><span class="mixed-article-title" title="mixed-article-title">Credit card fraud detection: A fusion approach using Dempster-Shafer theory and Bayesian learning.</span><span class="source" tagx="source" title="source">Inform Fusion</span><span class="volume" tagx="volume" title="volume">10</span><span class="fpage" tagx="fpage" title="fpage">354</span><span class="lpage" tagx="lpage" title="lpage">363</span></span></li> <li tag="ref"><a name="pone.0029617-Jones1" /><span class="label" tagx="label" title="label">49</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">L</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beynon</span><span class="given-names" tagx="given-names" title="given-names">MJ</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Holt</span><span class="given-names" tagx="given-names" title="given-names">CA</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roy</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">An application of the Dempster-Shafer theory of evidence to the classification of knee function and detection of improvement due to total knee replacement surgery.</span><span class="source" tagx="source" title="source">J Biomec</span><span class="volume" tagx="volume" title="volume">39</span><span class="fpage" tagx="fpage" title="fpage">2512</span><span class="lpage" tagx="lpage" title="lpage">2520</span></span></li> <li tag="ref"><a name="pone.0029617-Clements1" /><span class="label" tagx="label" title="label">50</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Clements</span><span class="given-names" tagx="given-names" title="given-names">A</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pfeiffer</span><span class="given-names" tagx="given-names" title="given-names">D</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Martin</span><span class="given-names" tagx="given-names" title="given-names">V</span></span></span><span class="year" tagx="year" title="year">2006</span><span class="mixed-article-title" title="mixed-article-title">Application of knowledge-driven spatial modelling approaches and uncertainty management to a study of Rift Valley fever in Africa.</span><span class="source" tagx="source" title="source">Int J Health Geogr</span><span class="volume" tagx="volume" title="volume">5</span><span class="fpage" tagx="fpage" title="fpage">57</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17156467">17156467</a></span></span></li> <li tag="ref"><a name="pone.0029617-Haining1" /><span class="label" tagx="label" title="label">51</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haining</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">1991</span><span class="mixed-article-title" title="mixed-article-title">Bivariate correlation with spatial data.</span><span class="source" tagx="source" title="source">Geogr Anal</span><span class="volume" tagx="volume" title="volume">23</span><span class="fpage" tagx="fpage" title="fpage">210</span><span class="lpage" tagx="lpage" title="lpage">27</span></span></li> <li tag="ref"><a name="pone.0029617-Haining2" /><span class="label" tagx="label" title="label">52</span><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haining</span><span class="given-names" tagx="given-names" title="given-names">R</span></span></span><span class="year" tagx="year" title="year">2003</span><span class="source" tagx="source" title="source">Spatial Data Analysis: Theory and Practice</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Cambridge</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Cambridge University Press</span></span></li> </ul> </div> </div>  </body></html>