<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/03/12/2020.03.10.986190.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.03.10.986190v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Rigidity, normal modes and flexible motion of a SARS-CoV-2 (COVID-19) protease structure"/>
  <meta name="DC.Identifier" content="10.1101/2020.03.10.986190"/>
  <meta name="DC.Date" content="2020-03-12"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="The rigidity and flexibility of two recently reported crystal structures (PDB entries 6Y2E and 6LU7) of a protease from the SARS-CoV-2 virus, the infectious agent of the COVID-19 respiratory disease, has been investigated using pebble-game rigidity analysis, elastic network model normal mode analysis, and all-atom geometric simulations. This computational investigation of the viral protease follows protocols that have been effective in studying other homodimeric enzymes. The protease is predicted to display flexible motions in vivo which directly affect the geometry of a known inhibitor binding site and which open new potential binding sites elsewhere in the structure. A database of generated PDB files representing natural flexible variations on the crystal structures has been produced and made available for download from an institutional data archive. This information may inform structure-based drug design and fragment screening efforts aimed at identifying specific antiviral therapies for the treatment of COVID-19."/>
  <meta name="DC.Contributor" content="Stephen A. Wells"/>
  <meta name="article:published_time" content="2020-03-12"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Rigidity, normal modes and flexible motion of a SARS-CoV-2 (COVID-19) protease structure"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;The rigidity and flexibility of two recently reported crystal structures (PDB entries 6Y2E and 6LU7) of a protease from the SARS-CoV-2 virus, the infectious agent of the COVID-19 respiratory disease, has been investigated using pebble-game rigidity analysis, elastic network model normal mode analysis, and all-atom geometric simulations. This computational investigation of the viral protease follows protocols that have been effective in studying other homodimeric enzymes. The protease is predicted to display flexible motions in vivo which directly affect the geometry of a known inhibitor binding site and which open new potential binding sites elsewhere in the structure. A database of generated PDB files representing natural flexible variations on the crystal structures has been produced and made available for download from an institutional data archive. This information may inform structure-based drug design and fragment screening efforts aimed at identifying specific antiviral therapies for the treatment of COVID-19.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.03.10.986190v1"/>
  <meta name="citation_id" content="2020.03.10.986190v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.03.10.986190v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.10.986190v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.03.10.986190v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.03.10.986190"/>
  <meta name="citation_num_pages" content="13"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.03.10.986190"/>
  <meta name="citation_author" content="Stephen A. Wells"/>
  <meta name="citation_author_institution" content="Department of Chemical Engineering, University of Bath"/>
  <meta name="citation_author_email" content="saw42@bath.ac.uk"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3920-3644"/>
  <meta name="citation_reference" content="WHO, Naming the coronavirus disease (COVID-19) and the virus that causes it. 2020."/>
  <meta name="citation_reference" content="Henzler-Wildman, K. and D. Kern, Dynamic personalities of proteins. Nature, 2007. 450(7172): p. 964–72."/>
  <meta name="citation_reference" content="Jimenez-Roldan, J.E., et al., Rapid simulation of protein motion: merging flexibility, rigidity and normal mode analyses. Physical Biology, 2012. 9(1)."/>
  <meta name="citation_reference" content="Wells, S.A., et al., Structure and Function in Homodimeric Enzymes: Simulations of Cooperative and Independent Functional Motions. Plos One, 2015. 10(8)."/>
  <meta name="citation_reference" content="Jones, H.B.L., et al., A complete thermodynamic analysis of enzyme turnover links the free energy landscape to enzyme catalysis. Febs Journal, 2017. 284(17): p. 2829–2842."/>
  <meta name="citation_reference" content="Jones, H.B.L., et al., Exposing the Interplay Between Enzyme Turnover, Protein Dynamics, and the Membrane Environment in Monoamine Oxidase B. Biochemistry, 2019. 58(18): p. 2362–2372."/>
  <meta name="citation_reference" content="Li, H.L., et al., Protein flexibility is key to cisplatin crosslinking in calmodulin. Protein Science, 2012. 21(9): p. 1269–1279."/>
  <meta name="citation_reference" content="Adjogatse, E., et al., Structure and function of l-threonine-3-dehydrogenase from the parasitic protozoan Trypanosoma brucei revealed by X-ray crystallography and geometric simulations. Acta Crystallographica Section D-Structural Biology, 2018. 74: p. 861–876."/>
  <meta name="citation_reference" content="Romer, R.A., et al., The flexibility and dynamics of protein disulfide isomerase. Proteins-Structure Function and Bioinformatics, 2016. 84(12): p. 1776–1785."/>
  <meta name="citation_reference" content="Walsh, M., Main protease structure and XChem fragment screen. 2020."/>
  <meta name="citation_reference" content="Wells, S.A., Dataset for “Rigidity, normal modes and flexible motion of a SARS-CoV-2 (COVID19) protease structure”. 2020."/>
  <meta name="citation_reference" content="LLC, S., The Pymol molecular graphics system. 2010."/>
  <meta name="citation_reference" content="Williams, C.J., et al., MolProbity: More and better reference data for improved all-atom structure validation. Protein Science, 2018. 27(1): p. 293–315."/>
  <meta name="citation_reference" content="Jacobs, D.J., et al., Protein flexibility predictions using graph theory. Proteins-Structure Function and Genetics, 2001. 44(2): p. 150–165."/>
  <meta name="citation_reference" content="McManus, T.J., S.A. Wells, and A.B. Walker, Salt bridge impact on global rigidity and thermostability in thermophilic citrate synthase. Physical Biology, 2020. 17(1)."/>
  <meta name="citation_reference" content="Suhre, K. and Y.H. Sanejouand, ElNemo: a normal mode web server for protein movement analysis and the generation of templates for molecular replacement. Nucleic Acids Research, 2004. 32: p. W610–W614."/>
  <meta name="citation_reference" content="Wells, S.A., et al., Constrained geometric simulation of diffusive motion in proteins. Physical Biology, 2005. 2(4): p. S127–S136."/>
  <meta name="citation_reference" content="Wells, S.A., J.E. Jimenez-Roldan, and R.A. Romer, Comparative analysis of rigidity across protein families. Physical Biology, 2009. 6(4)."/>
  <meta name="citation_reference" content="Wang, F., et al., Crystal Structure of Feline Infectious Peritonitis Virus Main Protease in Complex with Synergetic Dual Inhibitors. J Virol, 2016. 90(4): p. 1910–7."/>
  <meta name="twitter:title" content="Rigidity, normal modes and flexible motion of a SARS-CoV-2 (COVID-19) protease structure"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="The rigidity and flexibility of two recently reported crystal structures (PDB entries 6Y2E and 6LU7) of a protease from the SARS-CoV-2 virus, the infectious agent of the COVID-19 respiratory disease, has been investigated using pebble-game rigidity analysis, elastic network model normal mode analysis, and all-atom geometric simulations. This computational investigation of the viral protease follows protocols that have been effective in studying other homodimeric enzymes. The protease is predicted to display flexible motions in vivo which directly affect the geometry of a known inhibitor binding site and which open new potential binding sites elsewhere in the structure. A database of generated PDB files representing natural flexible variations on the crystal structures has been produced and made available for download from an institutional data archive. This information may inform structure-based drug design and fragment screening efforts aimed at identifying specific antiviral therapies for the treatment of COVID-19."/>
  <meta name="og-title" property="og:title" content="Rigidity, normal modes and flexible motion of a SARS-CoV-2 (COVID-19) protease structure"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.03.10.986190v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="The rigidity and flexibility of two recently reported crystal structures (PDB entries 6Y2E and 6LU7) of a protease from the SARS-CoV-2 virus, the infectious agent of the COVID-19 respiratory disease, has been investigated using pebble-game rigidity analysis, elastic network model normal mode analysis, and all-atom geometric simulations. This computational investigation of the viral protease follows protocols that have been effective in studying other homodimeric enzymes. The protease is predicted to display flexible motions in vivo which directly affect the geometry of a known inhibitor binding site and which open new potential binding sites elsewhere in the structure. A database of generated PDB files representing natural flexible variations on the crystal structures has been produced and made available for download from an institutional data archive. This information may inform structure-based drug design and fragment screening efforts aimed at identifying specific antiviral therapies for the treatment of COVID-19."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-03-12"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Rigidity, normal modes and flexible motion of a SARS-CoV-2 (COVID-19) protease structure | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1192812 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-2">The rigidity and flexibility of two recently reported crystal structures (PDB entries 6Y2E and 6LU7) of a protease from the SARS-CoV-2 virus, the infectious agent of the COVID-19 respiratory disease, has been investigated using pebble-game rigidity analysis, elastic network model normal mode analysis, and all-atom geometric simulations. This computational investigation of the viral protease follows protocols that have been effective in studying other homodimeric enzymes. The protease is predicted to display flexible motions in vivo which directly affect the geometry of a known inhibitor binding site and which open new potential binding sites elsewhere in the structure. A database of generated PDB files representing natural flexible variations on the crystal structures has been produced and made available for download from an institutional data archive. This information may inform structure-based drug design and fragment screening efforts aimed at identifying specific antiviral therapies for the treatment of COVID-19.</p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Introduction</h2>
    <p id="p-5">As of late 2019, there has been global concern over a novel respiratory disease (designated COVID-19 by the World Health Organisation (WHO)) which originated in the Wuhan region of China and has subsequently spread to multiple other countries worldwide. The virus causing COVID-19 is a coronavirus with the taxonomic identifier SARS-CoV-2[
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>]. Thanks to the diligent efforts of structural biologists, several crystal structures of proteins from SARS-CoV-2 have already been obtained and made available through the Protein Data Bank (PDB) even in advance of journal publication.
    </p>
    <p id="p-6">Crystal structures provide us with detailed structural knowledge of the arrangement of atoms making up a protein or protein complex. A crystal structure is a static snapshot of the protein “frozen” into one of its many possible conformations. This static snapshot is rich in implicit information on the natural dynamics which the protein will explore 
     <em>in vivo</em>. These natural dynamics, described by Kern[
     <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>] as a protein’s “dynamic personality”, can be critical to a protein’s function, and can directly affect the geometry of, for example, an enzyme active site.
    </p>
    <p id="p-7">A judicious combination of simplified methodologies – rigidity analysis, elastic network modelling, and geometric simulation of flexible motion – can work together to extract valuable information on the large-amplitude, low-frequency motions of a protein structure, at a fraction of the computational cost (in resources and time) of molecular dynamics (MD) approaches[
     <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>]. These simplified methods are strongly complementary to both computational MD investigations and experimental biophysical/biochemical studies of protein behaviour.
    </p>
    <p id="p-8">Recent studies has established that the motions explored by geometric simulation correspond well to the essential dynamics extracted from MD trajectories[
     <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>]; that global flexible motions can couple to enzyme active site geometry[
     <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>-
     <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>]; and, crucially, that a structure generated from geometric simulation of a large amplitude motion can be used as input for further MD investigation[
     <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>], as the geometric simulation retains the local bonding geometry and constraint network of the input crystal structure and forbids major bonding distortions and steric clashes.
    </p>
    <p id="p-9">The information obtained here regarding the flexibility of the enzyme should be of use and interest in structure-based drug design and computational screening of drugs and drug-like molecules capable of binding to and inhibiting the action of the viral protease, such as the pioneering work of Walsh 
     <em>et al</em>. reported online within the past few days[
     <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>]. The simulations reported here suggest that the protease structure is capable of substantial flexible motions which alter domain orientations, open and close clefts, and affect the geometry of an inhibitor-binding site. The entirety of the computational work described here was carried out in less than one working day on a laptop computer. The inputs and outputs of the simulations, principally consisting of PDB files representing flexible variations on the protease crystal structures, are available for download from the University of Bath research data archive at 
     <a href="https://researchdata.bath.ac.uk/772/">https://researchdata.bath.ac.uk/772/</a> [
     <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>] with DOI 
     <a href="https://doi.org/10.15125/BATH-00772">https://doi.org/10.15125/BATH-00772</a>
    </p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Methods, software and data</h2>
    <p id="p-10">Input data: crystal structures of a viral protease were downloaded from PDB entries 6Y2E (free protease) and 6LU7 (inhibitor bound).</p>
    <p id="p-11">Visualisations: visualisation and some minor structural editing, 
     <em>viz</em>. generation of symmetry copies, removal of water and other heterogroups, removal of alternate sidechain conformations, and renumbering of entries after addition of hydrogens, were carried out with the PyMOL viewer, version 0.99[
     <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>]. All structures were aligned to the 6Y2E crystal structure for consistency of view.
    </p>
    <p id="p-12">Addition of hydrogens, sidechain flipping: the MolProbity[
     <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>] web server hosted at Duke University provides online tools carrying out these functions.
    </p>
    <p id="p-13">Rigidity analysis: carried out using the pebble-game constraint counting algorithm implemented in the software FIRST[
     <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>]. FIRST should be available from Arizona State University via flexweb.asu.edu; academic readers should contact the present author if there is difficulty in obtaining the software, as the site appears currently to be down. Recent research has shown that the assignment of energies to polar interactions in FIRST suffers from a bug that leads to some salt-bridge interactions with short donor-acceptor distances being treated as weak rather than strong. Therefore, interactions were identified and energies assigned using a recently developed corrected function, SBFIRST[
     <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>]. A copy of the SBFIRST software for constraint identification is included with the deposited dataset for this study.
    </p>
    <p id="p-14">Normal mode analysis: carried out using the Elnemo elastic network modelling software[
     <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>]. This method builds an elastic network model of the protein with one site per residue (using the alpha carbon _CA_ atom coordinates as the node positions), placing springs of uniform strength between each pair of residues lying less than 10 Å apart, and diagonalising the resulting Hessian matrix to identify eigenvectors.
    </p>
    <p id="p-15">Geometric simulations of flexible motion: carried out using the FRODA geometric simulation engine[
     <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>] incorporated into FIRST. The FRODA engine iteratively projects an all-atom model of the protein structure along a normal mode direction[
     <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>] while constraining the local geometry to match that of the original protein structure. Frames along each such trajectory are written out as numbered PDB files.
    </p>
    <p id="p-16">Workflow and protocol: for each protein structure, the complete workflow for this study is as follows.</p>
    <p id="p-17">Download structure in PDB or mmCIF format. In this case, the downloaded structure contains coordinates for a single protein chain, and symmetry operations to generate copies making up the crystal structure.</p>
    <p id="p-18">Visualise structure in PyMOL. Generate symmetry mates. Select the appropriate symmetry mate to make up the homodimer indicated in the PDB entry (under Global Stoichiometry) as the biological entity of interest. Delete other symmetry copies. Alter chain ID(s) in the symmetry copy to give each chain a unique chain ID.</p>
    <p id="p-19">Process structure through the MolProbity website, adding hydrogens at electron-cloud positions, accepting all recommendations for side chain flips. Download the hydrogenated structure (in which all hydrogens have a serial number of 0).</p>
    <p id="p-20">Visualise structure in PyMOL for final cleaning and preparation. Remove alternate conformations (i.e. those with an 
     <em>altloc</em> entry which is not a blank or an A). Remove water molecules and extraneous heterogroups such as glycerol, acetyl etc. Save the cleaned structure; in this step, PyMOL renumbers the atoms sequentially, so hydrogens now have appropriate serial numbers.
    </p>
    <p id="p-21">Identify covalent and noncovalent interactions from the atomic geometry of the protein structure using SBFIRST. This generates lists of covalent bonds, tagged as being either rotatable or non-rotatable (such as the C-N bond in the peptide main chain); hydrophobic tether interactions, wherever two nonpolar sidechains are closely adjacent; and polar interactions, including salt bridges and hydrogen bonds. Polar interactions are assigned energies between 0 and -10 kcal/mol based on the donor-hydrogen-acceptor geometry.</p>
    <p id="p-22">Analyse rigidity with FIRST as a function of hydrogen-bond energy cutoff. This analysis groups atoms into rigid clusters by matching degrees of freedom against constraints. FIRST can carry out a dilution, gradually lowering the cutoff to exclude polar interactions one by one from weakest to strongest. It is also possible to simply carry out “snapshots” of the rigidity at a series of cutoff values, which in this study were -0.0, -0.5, -1.0 … -3.5, -4.0 kcal/mol.</p>
    <p id="p-23">Review constraints based on rigidity information. At this stage, a small number of noncovalent constraints (one in 6Y2E, three in 6LU7) were identifiable as artefacts of crystal packing, and were excluded from the network when carrying out the geometric simulations. Based on the rigidity analysis, a cutoff of -3.0 kcal/mol appeared appropriate for the geometric simulations, in line with previous studies [
     <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>, 
     <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>].
    </p>
    <p id="p-24">Normal mode analysis. Extract only the alpha carbon positions and run Elnemo 
     <em>pdbmat</em> (generation of matrix) and 
     <em>diagstd</em> (diagonalise matrix) functions. The result is a series of normal modes of motion, reported as eigenvector/eigenvalue pairs. When these are sorted by eigenvalue from lowest to highest, the first six modes are trivial rigid-body motions with near-zero eigenvalues (frequencies), while the low-frequency modes from 7 upwards represent flexible motions of the protein itself. This study examines modes 7 to 16, that is, the ten lowest-frequency nontrivial modes, with modes 7 to 12 being shown in the figures.
    </p>
    <p id="p-25">Geometric simulation of flexible motion. For each normal mode of interest (7 to 16), run FIRST with a chosen energy cutoff (−3.0 kcal/mol) and the previously identified lists of covalent and noncovalent bonding constraints, providing the normal mode eigenvector and a direction of motion (parallel or antiparallel to the vector) along with a directed step size (in this case 0.01Å), and invoking the FRODA engine implemented in FIRST. FRODA carries out a series of steps. In each step, all atom positions are moved along the normal mode direction. Then, the local atomic geometry defined by the bonding constraints and steric exclusion is restored by a few cycles of iterative relaxation. After a user-defined number of steps (in this case 100), a numbered “frame” is written out as a PDB file. A series of such frames describes the trajectory of flexible motion. FRODA continues each run until a user-defined maximum number of steps (in this case 1500) or until “jamming” occurs where the constraints can no longer be satisfied, typically when the motion has led to severe steric clashes through the collision of two domains.</p>
    <p id="p-26">Output data: the inputs, scripts and outputs of the rigidity analysis, normal mode analysis, and geometric simulations for each protein have been deposited in the University of Bath’s data repository and are accessible from 
     <a href="https://researchdata.bath.ac.uk/772/">https://researchdata.bath.ac.uk/772/</a> [
     <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>]
    </p>
   </div>
   <div class="section" id="sec-3">
    <h2 class="">Results and discussion</h2>
    <p id="p-27">The 6Y2E and 6LU7 crystal structures each contain explicit coordinates for one chain (A) of the protease, and in the case of 6LU7, a chain (C) representing a bound inhibitor. The input structures for analysis and simulation consist of chain A and a symmetry copy (B), making up the biological homodimer. Chain C is likewise duplicated to chain D. The X-ray structure does not contain explicit hydrogens, which are added using MolProbity[
     <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>]. With hydrogens added, it is then possible to detect the covalent bonds, polar noncovalent interactions (hydrogen bonds and salt bridges) and hydrophobic-tether interactions using SBFIRST[
     <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>] (see Methods). The polar interactions are rated with an effective energy between 0 and -10 kcal/mol based on the donor-hydrogen-acceptor geometry.
    </p>
    <p id="p-28">Pebble-game rigidity analysis[
     <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>, 
     <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>] divides the structure into rigid clusters and flexible regions based on the distribution of degrees of freedom and constraints. This rigid cluster decomposition (RCD) depends on the set of polar interactions that are included, based on an energy cutoff that excludes weaker interactions. Information on the relative stability of different portions of the structure is typically obtained at cutoffs around a “room temperature” value of -1 kcal/mol. RCDs are shown in 
     <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. With the inclusion of weaker polar constraints, a single large rigid cluster extends across both chains of the protease dimer. As weaker constraints are eliminated, the large cluster extends across the N-terminal domains (roughly residues 1-200) of both chains, which consist largely of beta-sheet structure, while the largely alpha-helical C-terminal domain breaks up into several smaller rigid clusters, each one a single helix. With further elimination of constraints, the beta-sheet regions become fully flexible.
    </p>
    <div id="F1" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Rigid Cluster Decompositions as a function of hydrogen bond energy cutoff for the empty 6Y2E and inhibitor-bound 6LU7 structures. The twenty largest rigid clusters in each case are shown as spheres and colour-coded from red (largest) to blue; the flexible regions are shown as cartoon." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-807296253" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Rigid Cluster Decompositions as a function of hydrogen bond energy cutoff for the empty 6Y2E and inhibitor-bound 6LU7 structures. The twenty largest rigid clusters in each case are shown as spheres and colour-coded from red (largest) to blue; the flexible regions are shown as cartoon.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="FIGURE 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F1.medium.gif" width="440" height="211"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="FIGURE 1:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F1.medium.gif" width="440" height="211"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIGURE 1:" data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">FIGURE 1:</span>
      <p id="p-29" class="first-child">Rigid Cluster Decompositions as a function of hydrogen bond energy cutoff for the empty 6Y2E and inhibitor-bound 6LU7 structures. The twenty largest rigid clusters in each case are shown as spheres and colour-coded from red (largest) to blue; the flexible regions are shown as cartoon.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-30">Flexible motion of a protein structure is best carried out at with a rigidity cutoff such that the structure is fully flexible. The mobility studies that follow are carried out with a cutoff of -3.0 kcal/mol, similar to previous studies on enzymes[
     <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">4</a>, 
     <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">5</a>]. It is important to appreciate that even at this lower cutoff the structure is still thoroughly constrained by noncovalent interactions, as illustrated in 
     <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. The core of each folded domain is rich in hydrophobic interactions, as is the interface region between the two beta-sheet domains as the two chains form the homodimer, and arrays of strong polar interactions constrain secondary structure (for example, the backbone hydrogen bonds within alpha helices and beta sheets).
    </p>
    <div id="F2" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Noncovalent constraints in the 6LU7 structure, shown as green (hydrophobic) and red (polar) lines. The protein mainchain is shown in cartoon representation. The circled area highlights a small set of interdomain hydrophobic constraints, one between the ALA 285 residues of the two chains, and one each between residue THR 280 of one chain and LEU 286 of the other. In the 6Y2E structure only the ALA 285 interaction is present. These interactions may be considered an artefact of crystal packing." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-807296253" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Noncovalent constraints in the 6LU7 structure, shown as green (hydrophobic) and red (polar) lines. The protein mainchain is shown in cartoon representation. The circled area highlights a small set of interdomain hydrophobic constraints, one between the ALA 285 residues of the two chains, and one each between residue THR 280 of one chain and LEU 286 of the other. In the 6Y2E structure only the ALA 285 interaction is present. These interactions may be considered an artefact of crystal packing.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="FIGURE 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F2.medium.gif" width="440" height="371"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="FIGURE 2:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F2.medium.gif" width="440" height="371"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIGURE 2:" data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">FIGURE 2:</span>
      <p id="p-31" class="first-child">Noncovalent constraints in the 6LU7 structure, shown as green (hydrophobic) and red (polar) lines. The protein mainchain is shown in cartoon representation. The circled area highlights a small set of interdomain hydrophobic constraints, one between the ALA 285 residues of the two chains, and one each between residue THR 280 of one chain and LEU 286 of the other. In the 6Y2E structure only the ALA 285 interaction is present. These interactions may be considered an artefact of crystal packing.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-32">Also highlighted in 
     <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> are a small set of hydrophobic interactions assigned between the N-terminal domains of chains A and B. In the 6Y2E structure, a single tether connects the sidechains of the ALA 285 residues of both chains. In the 6LU7 structure illustrated, there are two additional tethers nearby, each connecting the sidechains of the THR 280 residue of one chain and the LEU 286 of the other. Since the two N-terminal domains are otherwise not connected by noncovalent interactions, and these tethers are not involved in forming a rigid cluster, it seems reasonable to consider these tethers an artefact of crystal packing, and eliminate them from further consideration. Their deletion does not affect the rigid cluster decompositions of the structures. If these tethers were retained, they would somewhat limit (but not entirely prevent) the central cleft-opening motion discussed below.
    </p>
    <p id="p-33">Normal modes, representing directions of motion intrinsic to the structure, are easily obtained from an elastic network model[
     <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>] (see Methods) in which the protein is represented as one site per residue and springs of uniform strength are placed between every pair of sites with a separation less than 10 Å. In the 6LU7 structure, the amino acid residues of the bound inhibitor are included in the elastic network. Of the 3N normal modes of a structure with N residues, six modes of near-zero frequency represent the trivial rigid body motions of the structure, while the low-frequency modes from 7 upwards represent modes of flexible motion. When the structure moves along a low-frequency mode direction, the global geometry changes with minimal change in the local geometry, making these directions of “easy” motion for the protein. The sign of a normal mode eigenvector is arbitrary; motions parallel and antiparallel to each bias direction must be investigated equally.
    </p>
    <p id="p-34">Linear projection of a protein structure along a normal mode direction would rapidly introduce unphysical distortions into the structure. The geometric simulation approach implemented in FRODA[
     <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">3</a>, 
     <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>] instead applies a bias, moving the structure along a normal mode direction, while maintaining steric exclusion and the bonding geometry and noncovalent constraints of the input structure. This approach rapidly generates physically realistic flexible variations on the input structure which retain all-atom steric and bonding detail.
    </p>
    <p id="p-35">Geometric simulations of the lowest-frequency nontrivial modes of motion (7 upwards) show that the protease structure is capable of substantial amplitudes of easy motion covering several Ångstroms distance in global root-mean-square displacement (RMSD). In particular, the orientations of the alpha-helical domains relative to the beta-sheet domains can change through flexing and rotation about the interdomain “hinges”, and the beta-sheet domains themselves are large enough to display bending and twisting motions within themselves. These variations are illustrated for the 6Y2E structure in 
     <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> and for 6LU7 in 
     <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>.
    </p>
    <div id="F3" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Flexible variations (outer circle) of the 6Y2E structure (centre), shown in cartoon representation and coloured from blue to red along each chain. Each flexible variation is the 1000th frame of a geometric simulation, lying at an all-atom RMSD of 3 to 4 [[#xC5]]ngstroms from the starting crystal structure. Note the substantial variations of domain orientation achieved. Each variation is labelled with the normal mode used to bias the geometric simulation, from 7 to 12, and with the direction of bias, parallel (+) or antiparallel (-) to the mode eigenvector, whose sign is arbitrary." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-807296253" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Flexible variations (outer circle) of the 6Y2E structure (centre), shown in cartoon representation and coloured from blue to red along each chain. Each flexible variation is the 1000&lt;;sup&gt;;th&lt;;/sup&gt;; frame of a geometric simulation, lying at an all-atom RMSD of 3 to 4 [[#xC5]]ngstroms from the starting crystal structure. Note the substantial variations of domain orientation achieved. Each variation is labelled with the normal mode used to bias the geometric simulation, from 7 to 12, and with the direction of bias, parallel (+) or antiparallel (-) to the mode eigenvector, whose sign is arbitrary.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="FIGURE 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F3.medium.gif" width="410" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="FIGURE 3:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F3.medium.gif" width="410" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIGURE 3:" data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">FIGURE 3:</span>
      <p id="p-36" class="first-child">Flexible variations (outer circle) of the 6Y2E structure (centre), shown in cartoon representation and coloured from blue to red along each chain. Each flexible variation is the 1000
       <sup>th</sup> frame of a geometric simulation, lying at an all-atom RMSD of 3 to 4 Ångstroms from the starting crystal structure. Note the substantial variations of domain orientation achieved. Each variation is labelled with the normal mode used to bias the geometric simulation, from 7 to 12, and with the direction of bias, parallel (+) or antiparallel (-) to the mode eigenvector, whose sign is arbitrary.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <div id="F4" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Flexible variations (outer circle) of the 6LU7 structure (centre), shown in cartoon representation and coloured from blue to red along each chain. The bound inhibitor is shown as spheres. Each flexible variation is the 1000th frame of a geometric simulation, lying at an all-atom RMSD of 3 to 4 [[#xC5]]ngstroms from the starting crystal structure. Note the substantial variations of domain orientation achieved. Each variation is labelled with the normal mode used to bias the geometric simulation, from 7 to 12, and with the direction of bias, parallel (+) or antiparallel (-) to the mode eigenvector, whose sign is arbitrary." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-807296253" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Flexible variations (outer circle) of the 6LU7 structure (centre), shown in cartoon representation and coloured from blue to red along each chain. The bound inhibitor is shown as spheres. Each flexible variation is the 1000&lt;;sup&gt;;th&lt;;/sup&gt;; frame of a geometric simulation, lying at an all-atom RMSD of 3 to 4 [[#xC5]]ngstroms from the starting crystal structure. Note the substantial variations of domain orientation achieved. Each variation is labelled with the normal mode used to bias the geometric simulation, from 7 to 12, and with the direction of bias, parallel (+) or antiparallel (-) to the mode eigenvector, whose sign is arbitrary.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="FIGURE 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F4.medium.gif" width="414" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="FIGURE 4:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F4.medium.gif" width="414" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIGURE 4:" data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">FIGURE 4:</span>
      <p id="p-37" class="first-child">Flexible variations (outer circle) of the 6LU7 structure (centre), shown in cartoon representation and coloured from blue to red along each chain. The bound inhibitor is shown as spheres. Each flexible variation is the 1000
       <sup>th</sup> frame of a geometric simulation, lying at an all-atom RMSD of 3 to 4 Ångstroms from the starting crystal structure. Note the substantial variations of domain orientation achieved. Each variation is labelled with the normal mode used to bias the geometric simulation, from 7 to 12, and with the direction of bias, parallel (+) or antiparallel (-) to the mode eigenvector, whose sign is arbitrary.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-38">Flexible variations on a protein crystal structure are potentially relevant to structure-based drug design in at least two ways. Firstly, flexible motion can open or expose clefts and potential binding sites not directly visible in the static crystal structure. Since the protein 
     <em>in vivo</em> will be exploring its flexible motion thanks to the Brownian-motion driving force of its solvent (e.g. cytosol), such latent sites may constitute valid target areas for inhibitors.
    </p>
    <p id="p-39">Secondly, the global low-frequency motion couples to variations in the binding site/active site geometry[
     <a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">5</a>, 
     <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>]. Knowledge of the range of flexible variation here is potentially useful for structure-based drug design and/or fragment screening, since attention can be focussed on candidate molecules that interact robustly with the binding site and tolerate its flexibility, in preference to molecules that interact well only with the crystal structure and not with its flexible variations.
    </p>
    <p id="p-40">
     <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> illustrates a case of cleft opening. In both the 6Y2E and 6LU7 structures, some of the modes of motion (9- and 11+ in 6Y2E; 8+ and 11+ in 6LU7) have the two alpha-helical domains moving apart from each other, exposing a cleft at the centre of the protein that is a largely covered, narrow tunnel in the crystal structures. If the hydrophobic tethers eliminated previously were retained, the amplitude of this opening motion would be limited, but the cleft opening could still occur in the form of a widening of the tunnel. The protein structures are in this case shown in a space-filling (sphere) representation, since it is the exposure of new surface that is of interest.
    </p>
    <div id="F5" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Opening of a cleft between the N-terminal domains during flexible motion. Top row: 6Y2E crystal structure (a) and cleft opening motion along modes 9- (b) and 11+(c). Bottom row: 6LU7 crystal structure (d) and cleft opening motion along modes 8+ (e) and 11+ (f). Protein structure is shown as spheres and coloured by amino acid type. Green, aliphatic or aromatic residues; yellow, cysteine or methionine; red, hydroxyl (serine and threonine); magenta, acidic; purple, amidic; blue, basic." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-807296253" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Opening of a cleft between the N-terminal domains during flexible motion. Top row: 6Y2E crystal structure (a) and cleft opening motion along modes 9- (b) and 11+(c). Bottom row: 6LU7 crystal structure (d) and cleft opening motion along modes 8+ (e) and 11+ (f). Protein structure is shown as spheres and coloured by amino acid type. Green, aliphatic or aromatic residues; yellow, cysteine or methionine; red, hydroxyl (serine and threonine); magenta, acidic; purple, amidic; blue, basic.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="FIGURE 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F5.medium.gif" width="440" height="300"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="FIGURE 5:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F5.medium.gif" width="440" height="300"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIGURE 5:" data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">FIGURE 5:</span>
      <p id="p-41" class="first-child">Opening of a cleft between the N-terminal domains during flexible motion. Top row: 6Y2E crystal structure (a) and cleft opening motion along modes 9- (b) and 11+(c). Bottom row: 6LU7 crystal structure (d) and cleft opening motion along modes 8+ (e) and 11+ (f). Protein structure is shown as spheres and coloured by amino acid type. Green, aliphatic or aromatic residues; yellow, cysteine or methionine; red, hydroxyl (serine and threonine); magenta, acidic; purple, amidic; blue, basic.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-42">A more detailed view of this opening cleft is shown in 
     <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>. The residues are coloured to bring out the chemical character of the amino acid side chains. The portions of the alpha-helical domain surfaces that move aside to open the cleft display largely hydrophobic residues (aliphatic or aromatic side chains). Intriguingly, the exposed surface area within the cleft is richly lined with a series of basic residues (lysine and arginine), and is flanked by acidic (aspartic and glutamic acids) and polar (threonine) residues. In the crystal structures the basic and acidic residues appear to be involved in a network of inter- and intra-domain salt bridge interactions which stabilise the core of the homodimer. An antagonist capable of targeting this zone of strongly polar surface geometry exposed by flexible motion could potentially disrupt the dynamics of the enzyme and interfere with its function.
    </p>
    <div id="F6" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Detail view of interdomain cleft opening (structure 6LU7, mode 11+, 1000th frame of geometric simulation). Protein structure is shown as spheres and coloured by amino acid type. Green, aliphatic or aromatic residues; yellow, cysteine or methionine; red, hydroxyl (serine and threonine); magenta, acidic; purple, amidic; blue, basic. One end of the bound inhibitor is visible as grey spheres in the lower left. The exposed cleft is rich in basic, acidic and polar residues." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-807296253" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;Detail view of interdomain cleft opening (structure 6LU7, mode 11+, 1000&lt;;sup&gt;;th&lt;;/sup&gt;; frame of geometric simulation). Protein structure is shown as spheres and coloured by amino acid type. Green, aliphatic or aromatic residues; yellow, cysteine or methionine; red, hydroxyl (serine and threonine); magenta, acidic; purple, amidic; blue, basic. One end of the bound inhibitor is visible as grey spheres in the lower left. The exposed cleft is rich in basic, acidic and polar residues.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="FIGURE 6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F6.medium.gif" width="440" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="FIGURE 6:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F6.medium.gif" width="440" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIGURE 6:" data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">FIGURE 6:</span>
      <p id="p-43" class="first-child">Detail view of interdomain cleft opening (structure 6LU7, mode 11+, 1000
       <sup>th</sup> frame of geometric simulation). Protein structure is shown as spheres and coloured by amino acid type. Green, aliphatic or aromatic residues; yellow, cysteine or methionine; red, hydroxyl (serine and threonine); magenta, acidic; purple, amidic; blue, basic. One end of the bound inhibitor is visible as grey spheres in the lower left. The exposed cleft is rich in basic, acidic and polar residues.
      </p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-44">In the 6LU7 structure, a binding site on the flank of each beta-sheet domain is occupied by the inhibitor N-((5-methylisoxazol-3-yl)carbonyl)alanyl-L-valyl-N∼1∼-((1R,2Z)-4-(benzyloxy)-4-oxo-1-(((3R)-2-oxopyrrolidin-3-yl)methyl)but-2-enyl)-L-leucinamide (chain C of the structure), hereinafter “N3”. This inhibitor is listed in the PDB as PRD_002214; it is a peptide-like inhibitor, with a central core of amino acids modified at either end with heterogroups, and was previously known to inhibit a protease of a feline coronavirus[
     <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>] (see PDB entry 5EU8). In the 6Y2E structure the same binding cleft is empty, which means that its geometry is likely to change in the course of flexible motion of the beta-sheet domain.
    </p>
    <p id="p-45">
     <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a> shows the N3 binding site of the 6LU7 structure and the corresponding region of the 6Y2E structure. As noted, all structures have been aligned to the 6Y2E crystal structure for ease of viewing and comparison. 
     <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a> also shows flexible variations of the 6Y2E site in the course of flexible motion. A set of residues forming a rough square around the binding site (GLU 166, GLN 189, ASN 142 and THR 25) are highlighted in white as a guide to the eye. It is immediately visible that openings, closings and flexible distortions of the site are an intrinsic feature of the flexible motion of the protease. Note in particular the effect of motion along the lowest-frequency nontrivial mode direction, mode 7. Motion biased antiparallel to the mode eigenvector (7-) includes an opening of the cleft, while motion biased parallel to it (7+) closes it. Since the N3 inhibitor is quite deeply embedded in the binding site of the 6LU7 structure, opening and closing of this cleft in the course of the protein’s natural motion may be necessary for molecules to access the cleft. On visual inspection, the site in the 6Y2E structure is clearly slightly more closed that in 6LU7, consistent with the proposition that such opening/closing motions are intrinsic to the protein.
    </p>
    <div id="F7" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F7.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Top row: the N3 inhibitor binding site of the 6LU7 structure, and the corresponding empty site of the 6Y2E structure. Rows below: flexible variations of the 6Y2E site from geometric simulations along normal modes as indicated. Residues 1-200 of a single chain of the protease are shown as spheres and coloured by amino acid type. Green, aliphatic or aromatic residues; yellow, cysteine or methionine; red, hydroxyl (serine and threonine); magenta, acidic; purple, amidic; blue, basic. To highlight the binding site, residues GLU 166, GLN 189, ASN 142 and THR 25 are coloured white. Substantial opening and closing of the binding site cleft occurs during the flexible motion." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-807296253" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Top row: the N3 inhibitor binding site of the 6LU7 structure, and the corresponding empty site of the 6Y2E structure. Rows below: flexible variations of the 6Y2E site from geometric simulations along normal modes as indicated. Residues 1-200 of a single chain of the protease are shown as spheres and coloured by amino acid type. Green, aliphatic or aromatic residues; yellow, cysteine or methionine; red, hydroxyl (serine and threonine); magenta, acidic; purple, amidic; blue, basic. To highlight the binding site, residues GLU 166, GLN 189, ASN 142 and THR 25 are coloured white. Substantial opening and closing of the binding site cleft occurs during the flexible motion.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="FIGURE 7:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F7.medium.gif" width="416" height="440"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="FIGURE 7:" src="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F7.medium.gif" width="416" height="440"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIGURE 7:" data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2020/03/12/2020.03.10.986190/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">FIGURE 7:</span>
      <p id="p-46" class="first-child">Top row: the N3 inhibitor binding site of the 6LU7 structure, and the corresponding empty site of the 6Y2E structure. Rows below: flexible variations of the 6Y2E site from geometric simulations along normal modes as indicated. Residues 1-200 of a single chain of the protease are shown as spheres and coloured by amino acid type. Green, aliphatic or aromatic residues; yellow, cysteine or methionine; red, hydroxyl (serine and threonine); magenta, acidic; purple, amidic; blue, basic. To highlight the binding site, residues GLU 166, GLN 189, ASN 142 and THR 25 are coloured white. Substantial opening and closing of the binding site cleft occurs during the flexible motion.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
   </div>
   <div class="section" id="sec-4">
    <h2 class="">Conclusions</h2>
    <p id="p-47">The rigidity and flexibility of two recently reported crystal structures (PDB entries 6Y2E and 6LU7) of a protease from the SARS-CoV-2 virus, the infectious agent of the COVID-19 respiratory disease, has been investigated using pebble-game rigidity analysis, elastic network model normal mode analysis, and all-atom geometric simulations. This computational investigation of the viral protease follows protocols that have been effective in studying other homodimeric enzymes. The protease is predicted to display flexible motions 
     <em>in vivo</em> which directly affect the geometry of a known inhibitor binding site, 
     <em>e</em>.
     <em>g</em>. through an opening/closing motion, and which open new potential binding sites elsewhere in the structure. A database of generated PDB files representing natural flexible variations on the crystal structures has been produced and made available for download from an institutional data archive. This information may inform structure-based drug design and fragment screening efforts aimed at identifying specific antiviral therapies for the treatment of COVID-19.
    </p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgements</h2>
    <p id="p-48">SAW has received funding from the European Research Council (ERC) under the European Union’s Horizon 2020 research and innovation programme (grant agreement No 648283 “GROWMOF”, PI: Prof. T. Düren).</p>
   </div>
   <div class="section fn-group" id="fn-group-1">
    <h2>Footnotes</h2>
    <ul>
     <li class="fn-dataset fn-group-external-links" id="fn-1">
      <p id="p-3">
       <a href="https://doi.org/10.15125/BATH-00772">https://doi.org/10.15125/BATH-00772</a>
      </p>
     </li>
     <li class="fn-dataset fn-group-external-links" id="fn-2">
      <p id="p-4">
       <a href="https://researchdata.bath.ac.uk/772/">https://researchdata.bath.ac.uk/772/</a>
      </p>
     </li>
    </ul>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label">1.</span>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.10.986190v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">WHO</span>, 
         <abbr class="cit-jnl-abbrev">Naming the coronavirus disease (COVID-19) and the virus that causes it</abbr>. 
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">2.</span>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.10.986190v1.2" data-doi="10.1038/nature06522">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Henzler-Wildman</span>, 
          <span class="cit-name-given-names">K.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Kern</span>
         </span>, 
         <span class="cit-article-title">Dynamic personalities of proteins</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>, 
         <span class="cit-pub-date">2007</span>. 
         <span class="cit-vol">450</span>(
         <span class="cit-issue">7172</span>): p. 
         <span class="cit-fpage">964</span>–
         <span class="cit-lpage">72</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHenzler-Wildman%26rft.auinit1%253DK.%26rft.volume%253D450%26rft.issue%253D7172%26rft.spage%253D964%26rft.epage%253D972%26rft.atitle%253DDynamic%2Bpersonalities%2Bof%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06522%26rft_id%253Dinfo%253Apmid%252F18075575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature06522&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18075575&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.10.986190.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000251579900066&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">3.</span>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.10.986190v1.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jimenez-Roldan</span>, 
          <span class="cit-name-given-names">J.E.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Rapid simulation of protein motion: merging flexibility, rigidity and normal mode analyses</span>. 
         <abbr class="cit-jnl-abbrev">Physical Biology</abbr>, 
         <span class="cit-pub-date">2012</span>. 
         <span class="cit-vol">9</span>(
         <span class="cit-issue">1</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">4.</span>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.10.986190v1.4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wells</span>, 
          <span class="cit-name-given-names">S.A.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Structure and Function in Homodimeric Enzymes: Simulations of Cooperative and Independent Functional Motions</span>. 
         <abbr class="cit-jnl-abbrev">Plos One</abbr>, 
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-vol">10</span>(
         <span class="cit-issue">8</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">5.</span>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.10.986190v1.5">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jones</span>, 
          <span class="cit-name-given-names">H.B.L.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">A complete thermodynamic analysis of enzyme turnover links the free energy landscape to enzyme catalysis</span>. 
         <abbr class="cit-jnl-abbrev">Febs Journal</abbr>, 
         <span class="cit-pub-date">2017</span>. 
         <span class="cit-vol">284</span>(
         <span class="cit-issue">17</span>): p. 
         <span class="cit-fpage">2829</span>–
         <span class="cit-lpage">2842</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DFebs%2BJournal%26rft.volume%253D284%26rft.spage%253D2829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">6.</span>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.10.986190v1.6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jones</span>, 
          <span class="cit-name-given-names">H.B.L.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Exposing the Interplay Between Enzyme Turnover, Protein Dynamics, and the Membrane Environment in Monoamine Oxidase B</span>. 
         <abbr class="cit-jnl-abbrev">Biochemistry</abbr>, 
         <span class="cit-pub-date">2019</span>. 
         <span class="cit-vol">58</span>(
         <span class="cit-issue">18</span>): p. 
         <span class="cit-fpage">2362</span>–
         <span class="cit-lpage">2372</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBiochemistry%26rft.volume%253D58%26rft.spage%253D2362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">7.</span>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.03.10.986190v1.7" data-doi="10.1002/pro.2111">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">H.L.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Protein flexibility is key to cisplatin crosslinking in calmodulin</span>. 
         <abbr class="cit-jnl-abbrev">Protein Science</abbr>, 
         <span class="cit-pub-date">2012</span>. 
         <span class="cit-vol">21</span>(
         <span class="cit-issue">9</span>): p. 
         <span class="cit-fpage">1269</span>–
         <span class="cit-lpage">1279</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProtein%2BScience%26rft.stitle%253DProtein%2BSci.%26rft.aulast%253DLi%26rft.auinit1%253DH.%26rft.volume%253D21%26rft.issue%253D9%26rft.spage%253D1269%26rft.epage%253D1279%26rft.atitle%253DProtein%2Bflexibility%2Bis%2Bkey%2Bto%2Bcisplatin%2Bcrosslinking%2Bin%2Bcalmodulin.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpro.2111%26rft_id%253Dinfo%253Apmid%252F22733664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1002/pro.2111&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22733664&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.10.986190.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">8.</span>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.10.986190v1.8">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Adjogatse</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Structure and function of l-threonine-3-dehydrogenase from the parasitic protozoan Trypanosoma brucei revealed by X-ray crystallography and geometric simulations</span>. 
         <abbr class="cit-jnl-abbrev">Acta Crystallographica Section D-Structural Biology</abbr>, 
         <span class="cit-pub-date">2018</span>. 
         <span class="cit-vol">74</span>: p. 
         <span class="cit-fpage">861</span>–
         <span class="cit-lpage">876</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DActa%2BCrystallographica%2BSection%2BD-Structural%2BBiology%26rft.volume%253D74%26rft.spage%253D861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">9.</span>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.10.986190v1.9">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Romer</span>, 
          <span class="cit-name-given-names">R.A.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">The flexibility and dynamics of protein disulfide isomerase</span>. 
         <abbr class="cit-jnl-abbrev">Proteins-Structure Function and Bioinformatics</abbr>, 
         <span class="cit-pub-date">2016</span>. 
         <span class="cit-vol">84</span>(
         <span class="cit-issue">12</span>): p. 
         <span class="cit-fpage">1776</span>–
         <span class="cit-lpage">1785</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProteins-Structure%2BFunction%2Band%2BBioinformatics%26rft.volume%253D84%26rft.spage%253D1776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">10.</span>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.10.986190v1.10">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Walsh</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <abbr class="cit-jnl-abbrev">Main protease structure and XChem fragment screen</abbr>. 
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">11.</span>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.10.986190v1.11">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wells</span>, 
          <span class="cit-name-given-names">S.A.</span>
         </span>, 
         <abbr class="cit-jnl-abbrev">Dataset for “Rigidity, normal modes and flexible motion of a SARS-CoV-2 (COVID19) protease structure”</abbr>. 
         <span class="cit-pub-date">2020</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">12.</span>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.10.986190v1.12">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth cit-collab">LLC, S</span>., 
         <abbr class="cit-jnl-abbrev">The Pymol molecular graphics system</abbr>. 
         <span class="cit-pub-date">2010</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">13.</span>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.10.986190v1.13">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Williams</span>, 
          <span class="cit-name-given-names">C.J.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <abbr class="cit-jnl-abbrev">MolProbity: More and better reference data for improved all-atom structure validation. Protein Science</abbr>, 
         <span class="cit-pub-date">2018</span>. 
         <span class="cit-vol">27</span>(
         <span class="cit-issue">1</span>): p. 
         <span class="cit-fpage">293</span>–
         <span class="cit-lpage">315</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMolProbity%253A%2BMore%2Band%2Bbetter%2Breference%2Bdata%2Bfor%2Bimproved%2Ball-atom%2Bstructure%2Bvalidation.%2BProtein%2BScience%26rft.volume%253D27%26rft.spage%253D293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">14.</span>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.10.986190v1.14" data-doi="10.1002/prot.1081">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jacobs</span>, 
          <span class="cit-name-given-names">D.J.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Protein flexibility predictions using graph theory</span>. 
         <abbr class="cit-jnl-abbrev">Proteins-Structure Function and Genetics</abbr>, 
         <span class="cit-pub-date">2001</span>. 
         <span class="cit-vol">44</span>(
         <span class="cit-issue">2</span>): p. 
         <span class="cit-fpage">150</span>–
         <span class="cit-lpage">165</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProteins%26rft.stitle%253DProteins%26rft.aulast%253DJacobs%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D44%26rft.issue%253D2%26rft.spage%253D150%26rft.epage%253D165%26rft.atitle%253DProtein%2Bflexibility%2Bpredictions%2Busing%2Bgraph%2Btheory.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fprot.1081%26rft_id%253Dinfo%253Apmid%252F11391777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1002/prot.1081&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=11391777&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.10.986190.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000169592300009&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">15.</span>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.10.986190v1.15">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">McManus</span>, 
          <span class="cit-name-given-names">T.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.A.</span>
          <span class="cit-name-surname">Wells</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.B.</span>
          <span class="cit-name-surname">Walker</span>
         </span>, 
         <span class="cit-article-title">Salt bridge impact on global rigidity and thermostability in thermophilic citrate synthase</span>. 
         <abbr class="cit-jnl-abbrev">Physical Biology</abbr>, 
         <span class="cit-pub-date">2020</span>. 
         <span class="cit-vol">17</span>(
         <span class="cit-issue">1</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">16.</span>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.10.986190v1.16" data-doi="10.1093/nar/gkh368">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Suhre</span>, 
          <span class="cit-name-given-names">K.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Y.H.</span>
          <span class="cit-name-surname">Sanejouand</span>
         </span>, 
         <span class="cit-article-title">ElNemo: a normal mode web server for protein movement analysis and the generation of templates for molecular replacement</span>. 
         <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr>, 
         <span class="cit-pub-date">2004</span>. 
         <span class="cit-vol">32</span>: p. 
         <span class="cit-fpage">W610</span>–
         <span class="cit-lpage">W614</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSuhre%26rft.auinit1%253DK.%26rft.volume%253D32%26rft.issue%253Dsuppl_2%26rft.spage%253DW610%26rft.epage%253DW614%26rft.atitle%253DElNemo%253A%2Ba%2Bnormal%2Bmode%2Bweb%2Bserver%2Bfor%2Bprotein%2Bmovement%2Banalysis%2Band%2Bthe%2Bgeneration%2Bof%2Btemplates%2Bfor%2Bmolecular%2Breplacement%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh368%26rft_id%253Dinfo%253Apmid%252F15215461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/nar/gkh368&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=15215461&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F03%2F12%2F2020.03.10.986190.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000222273100123&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">17.</span>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.10.986190v1.17">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wells</span>, 
          <span class="cit-name-given-names">S.A.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Constrained geometric simulation of diffusive motion in proteins</span>. 
         <abbr class="cit-jnl-abbrev">Physical Biology</abbr>, 
         <span class="cit-pub-date">2005</span>. 
         <span class="cit-vol">2</span>(
         <span class="cit-issue">4</span>): p. 
         <span class="cit-fpage">S127</span>–
         <span class="cit-lpage">S136</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPhysical%2BBiology%26rft.volume%253D2%26rft.spage%253D127S%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label">18.</span>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.10.986190v1.18">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wells</span>, 
          <span class="cit-name-given-names">S.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.E.</span>
          <span class="cit-name-surname">Jimenez-Roldan</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.A.</span>
          <span class="cit-name-surname">Romer</span>
         </span>, 
         <span class="cit-article-title">Comparative analysis of rigidity across protein families</span>. 
         <abbr class="cit-jnl-abbrev">Physical Biology</abbr>, 
         <span class="cit-pub-date">2009</span>. 
         <span class="cit-vol">6</span>(
         <span class="cit-issue">4</span>).
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label">19.</span>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.03.10.986190v1.19" data-doi="10.1128/JVI.02685-15">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>, 
         <span class="cit-etal">et al.</span>, 
         <span class="cit-article-title">Crystal Structure of Feline Infectious Peritonitis Virus Main Protease in Complex with Synergetic Dual Inhibitors</span>. 
         <abbr class="cit-jnl-abbrev">J Virol</abbr>, 
         <span class="cit-pub-date">2016</span>. 
         <span class="cit-vol">90</span>(
         <span class="cit-issue">4</span>): p. 
         <span class="cit-fpage">1910</span>–
         <span class="cit-lpage">7</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BVirol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02685-15%26rft_id%253Dinfo%253Apmid%252F26656689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjkwLzQvMTkxMCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAzLzEyLzIwMjAuMDMuMTAuOTg2MTkwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
