<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2018/03/08/271171.atom" />
<meta name="HW.pisa" content="biorxiv;271171v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Membrane proteins with high N-glycosylation, high expression, and multiple interaction partners were preferred by mammalian viruses as receptors" />
<meta name="DC.Identifier" content="10.1101/271171" />
<meta name="DC.Date" content="2018-03-08" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2018, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NoDerivs 4.0 International), CC BY-ND 4.0, as described at http://creativecommons.org/licenses/by-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Receptor mediated entry is the first step for viral infection. However, the relationship between viruses and receptors is still obscure. Here, by manually curating a high-quality database of 268 pairs of mammalian virus-host receptor interaction, which included 128 unique viral species or sub-species and 119 virus receptors, we found the viral receptors were structurally and functionally diverse, yet they had several common features when compared to other cell membrane proteins: more protein domains, higher level of N-glycosylation, higher ratio of self-interaction and more interaction partners, and higher expression in most tissues of the host. Additionally, the receptors used by the same virus tended to co-evolve. Further correlation analysis between viral receptors and the tissue and host specificity of the virus shows that the virus receptor similarity was a significant predictor for mammalian virus cross-species. This work could deepen our understanding towards the viral receptor selection and help evaluate the risk of viral zoonotic diseases." />
<meta name="DC.Contributor" content="Zheng Zhang" />
<meta name="DC.Contributor" content="Zhaozhong Zhu" />
<meta name="DC.Contributor" content="Wenjun Chen" />
<meta name="DC.Contributor" content="Zena Cai" />
<meta name="DC.Contributor" content="Beibei Xu" />
<meta name="DC.Contributor" content="Zhiying Tan" />
<meta name="DC.Contributor" content="Aiping Wu" />
<meta name="DC.Contributor" content="Xingyi Ge" />
<meta name="DC.Contributor" content="Xinhong Guo" />
<meta name="DC.Contributor" content="Zhongyang Tan" />
<meta name="DC.Contributor" content="Zanxian Xia" />
<meta name="DC.Contributor" content="Haizhen Zhu" />
<meta name="DC.Contributor" content="Taijiao Jiang" />
<meta name="DC.Contributor" content="Yousong Peng" />
<meta name="article:published_time" content="2018-03-08" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Membrane proteins with high N-glycosylation, high expression, and multiple interaction partners were preferred by mammalian viruses as receptors" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Receptor mediated entry is the first step for viral infection. However, the relationship between viruses and receptors is still obscure. Here, by manually curating a high-quality database of 268 pairs of mammalian virus-host receptor interaction, which included 128 unique viral species or sub-species and 119 virus receptors, we found the viral receptors were structurally and functionally diverse, yet they had several common features when compared to other cell membrane proteins: more protein domains, higher level of N-glycosylation, higher ratio of self-interaction and more interaction partners, and higher expression in most tissues of the host. Additionally, the receptors used by the same virus tended to co-evolve. Further correlation analysis between viral receptors and the tissue and host specificity of the virus shows that the virus receptor similarity was a significant predictor for mammalian virus cross-species. This work could deepen our understanding towards the viral receptor selection and help evaluate the risk of viral zoonotic diseases.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2018/01/01" />
<meta name="citation_mjid" content="biorxiv;271171v2" />
<meta name="citation_id" content="271171v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/271171v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/271171v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/271171v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2018/03/08/271171.full.pdf" />
<meta name="citation_doi" content="10.1101/271171" />
<meta name="citation_num_pages" content="36" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="271171" />
<meta name="citation_author" content="Zheng Zhang" />
<meta name="citation_author_institution" content="College of Biology, Hunan University" />
<meta name="citation_author" content="Zhaozhong Zhu" />
<meta name="citation_author_institution" content="College of Biology, Hunan University" />
<meta name="citation_author" content="Wenjun Chen" />
<meta name="citation_author_institution" content="College of Biology, Hunan University" />
<meta name="citation_author" content="Zena Cai" />
<meta name="citation_author_institution" content="College of Biology, Hunan University" />
<meta name="citation_author" content="Beibei Xu" />
<meta name="citation_author_institution" content="College of Computer Science and Electronic Engineering, Hunan University" />
<meta name="citation_author" content="Zhiying Tan" />
<meta name="citation_author_institution" content="College of Computer Science and Electronic Engineering, Hunan University" />
<meta name="citation_author" content="Aiping Wu" />
<meta name="citation_author_institution" content="Center of System Medicine, Institute of Basic Medical Sciences, Chinese Academy of Medical Sciences &amp; Peking Union Medical College" />
<meta name="citation_author_institution" content="Suzhou Institute of Systems Medicine" />
<meta name="citation_author" content="Xingyi Ge" />
<meta name="citation_author_institution" content="College of Biology, Hunan University" />
<meta name="citation_author" content="Xinhong Guo" />
<meta name="citation_author_institution" content="College of Biology, Hunan University" />
<meta name="citation_author" content="Zhongyang Tan" />
<meta name="citation_author_institution" content="College of Biology, Hunan University" />
<meta name="citation_author" content="Zanxian Xia" />
<meta name="citation_author_institution" content="School of Life Sciences, Central South University" />
<meta name="citation_author" content="Haizhen Zhu" />
<meta name="citation_author_institution" content="College of Biology, Hunan University" />
<meta name="citation_author_institution" content="State Key Laboratory of Chemo/Biosensing and Chemometrics, Hunan University" />
<meta name="citation_author_email" content="zhuhaizhen69@yahoo.com" />
<meta name="citation_author_email" content="taijiao@ibms.pumc.edu.cn" />
<meta name="citation_author_email" content="pys2013@hnu.edu.cn" />
<meta name="citation_author" content="Taijiao Jiang" />
<meta name="citation_author_institution" content="Center of System Medicine, Institute of Basic Medical Sciences, Chinese Academy of Medical Sciences &amp; Peking Union Medical College" />
<meta name="citation_author_institution" content="Suzhou Institute of Systems Medicine" />
<meta name="citation_author_email" content="zhuhaizhen69@yahoo.com" />
<meta name="citation_author_email" content="taijiao@ibms.pumc.edu.cn" />
<meta name="citation_author_email" content="pys2013@hnu.edu.cn" />
<meta name="citation_author" content="Yousong Peng" />
<meta name="citation_author_institution" content="College of Biology, Hunan University" />
<meta name="citation_author_email" content="zhuhaizhen69@yahoo.com" />
<meta name="citation_author_email" content="taijiao@ibms.pumc.edu.cn" />
<meta name="citation_author_email" content="pys2013@hnu.edu.cn" />
<meta name="citation_reference" content="Mlakar J, Korva M, Tul N, et al. Zika virus associated with microcephaly. New Engl J Med, 2016, 374: 951–958" />
<meta name="citation_reference" content="Maganga GD, Kapetshi J, Berthet N, et al. Ebola virus disease in the democratic republic of congo. New Engl J Med, 2014, 371: 2083–2091" />
<meta name="citation_reference" content="Breban R, Riou J, Fontanet A. Interhuman transmissibility of middle east respiratory syndrome coronavirus: Estimation of pandemic risk. Lancet, 2013, 382: 694–699" />
<meta name="citation_reference" content="Paez-Espino D, Eloe-Fadrosh EA, Pavlopoulos GA, et al. Uncovering earth’s virome. Nature, 2016, 536: 425-+" />
<meta name="citation_reference" content="Mihara T, Nishimura Y, Shimizu Y, et al. Linking virus genomes with host taxonomy. Viruses-Basel, 2016, 8:" />
<meta name="citation_reference" content="Geoghegan JL, Senior AM, Di Giallonardo F, et al. Virological factors that increase the transmissibility of emerging human viruses. Proceedings of the National Academy of Sciences of the United States of America, 2016, 113: 4170–4175" />
<meta name="citation_reference" content="Olival KJ, Hosseini PR, Zambrana-Torrelio C, et al. Host and viral traits predict zoonotic spillover from mammals. Nature, 2017, 546: 646-+" />
<meta name="citation_reference" content="Sharp PM, Hahn BH. Origins of hiv and the aids pandemic. Csh Perspect Med, 2011, 1:2" />
<meta name="citation_reference" content="Ge XY, Li JL, Yang XL, et al. Isolation and characterization of a bat sars-like coronavirus that uses the ace2 receptor. Nature, 2013, 503: 535-+" />
<meta name="citation_reference" content="Dimitrov DS. Virus entry: Molecular mechanisms and biomedical applications. Nat Rev Microbiol, 2004, 2: 109–122" />
<meta name="citation_reference" content="Li F. Receptor recognition mechanisms of coronaviruses: A decade of structural studies. Journal of virology, 2015, 89: 1954–1964" />
<meta name="citation_reference" content="Baranowski E, Ruiz-Jarabo CM, Domingo E. Evolution of cell recognition by viruses. Science, 2001, 292: 1102–1105" />
<meta name="citation_reference" content="Grove J, Marsh M. The cell biology of receptor-mediated virus entry. The Journal of cell biology, 2011, 195: 1071–1082" />
<meta name="citation_reference" content="Casasnovas JM. Virus-receptor interactions and receptor-mediated virus entry into host cells. 2013" />
<meta name="citation_reference" content="Wang JH. Protein recognition by cell surface receptors: Physiological receptors versus virus interactions. Trends in biochemical sciences, 2002, 27: 122–126" />
<meta name="citation_reference" content="Li F. Structure, function, and evolution of coronavirus spike proteins. Annu Rev Virol, 2016, 3: 237–261" />
<meta name="citation_reference" content="Yan H, Zhong G, Xu G, et al. Sodium taurocholate cotransporting polypeptide is a functional receptor for human hepatitis b and d virus. eLife, 2012, 1: e00049" />
<meta name="citation_reference" content="Peng WJ, de Vries RP, Grant OC, et al. Recent h3n2 viruses have evolved specificity for extended, branched human-type receptors, conferring potential for increased avidity. Cell host &amp; microbe, 2017, 21: 23–34" />
<meta name="citation_reference" content="Isa P, Arias CF, Lopez S. Role of sialic acids in rotavirus infection. Glycoconjugate journal, 2006, 23: 27–37" />
<meta name="citation_reference" content="Mazzon M, Mercer J. Lipid interactions during virus entry and infection. Cellular microbiology, 2014, 16: 1493–1502" />
<meta name="citation_reference" content="Marija Backovic FAR. Virus entry: Old viruses, new receptors. Current opinion in virology, 2012, 2: 10" />
<meta name="citation_reference" content="Coffin JM. Virions at the gates: Receptors and the host-virus arms race. PLoS Biology, 2013, 11:" />
<meta name="citation_reference" content="Kumlin U, Olofsson S, Dimock K, et al. Sialic acid tissue distribution and influenza virus tropism. Influenza and other respiratory viruses, 2008, 2: 147–154" />
<meta name="citation_reference" content="Harris HJ, Davis C, Mullins JG, et al. Claudin association with cd81 defines hepatitis c virus entry. The Journal of biological chemistry, 2010, 285: 21092–21102" />
<meta name="citation_reference" content="Ribeiro RM, Hazenberg MD, Perelson AS, et al. Naive and memory cell turnover as drivers of ccr5-to-cxcr4 tropism switch in human immunodeficiency virus type 1: Implications for therapy. Journal of virology, 2006, 80: 802–809" />
<meta name="citation_reference" content="Carter CC, McNamara LA, Onafuwa-Nuga A, et al. Hiv-1 utilizes the cxcr4 chemokine receptor to infect multipotent hematopoietic stem and progenitor cells. Cell host &amp; microbe, 2011, 9: 223–234" />
<meta name="citation_reference" content="Taubenberger JK, Kash JC. Influenza virus evolution, host adaptation, and pandemic formation. Cell host &amp; microbe, 2010, 7: 440–451" />
<meta name="citation_reference" content="Lu G, Wang Q, Gao GF. Bat-to-human: Spike features determining ‘host jump’of coronaviruses sars-cov, mers-cov, and beyond. Trends in microbiology, 2015, 23: 468–478" />
<meta name="citation_reference" content="Li F. Receptor recognition and cross-species infections of sars coronavirus. Antiviral research, 2013, 100: 246–254" />
<meta name="citation_reference" content="Maginnis MS, Haley SA, Gee GV, et al. Role of n-linked glycosylation of the 5-ht2a receptor in jc virus infection. Journal of virology, 2010, 84: 9677–9684" />
<meta name="citation_reference" content="Menche J, Sharma A, Kitsak M, et al. Uncovering disease-disease relationships through the incomplete interactome. Science, 2015, 347:" />
<meta name="citation_reference" content="Corfield A. Eukaryotic protein glycosylation: A primer for histochemists and cell biologists. Histochemistry and cell biology, 2017, 147: 119–147" />
<meta name="citation_reference" content="Krupovic M, Koonin EV. Multiple origins of viral capsid proteins from cellular ancestors. Proceedings of the National Academy of Sciences of the United States of America, 2017, 114: E2401–E2410" />
<meta name="citation_reference" content="Drayman N, Glick Y, Ben-nun-shaul O, et al. Pathogens use structural mimicry of native host ligands as a mechanism for host receptor engagement. Cell host &amp; microbe, 2013, 14: 63–73" />
<meta name="citation_reference" content="Boonarkart C, Champunot R, Uiprasertkul M, et al. Case report: Increased viral receptor expression associated with high viral load and severe pneumonia in a young patient infected with 2009 h1n1 influenza a with no pre-existing conditions. Journal of medical virology, 2012, 84: 380–385" />
<meta name="citation_reference" content="Nowakowski TJ, Pollen AA, Di Lullo E, et al. Expression analysis highlights axl as a candidate zika virus entry receptor in neural stem cells. Cell stem cell, 2016, 18: 591–596" />
<meta name="citation_reference" content="McNab F, Mayer-Barber K, Sher A, et al. Type i interferons in infectious disease. Nature reviews Immunology, 2015, 15: 87–103" />
<meta name="citation_reference" content="Jost S, Altfeld M. Control of human viral infections by natural killer cells. Annual review of immunology, 2013, 31: 163–194" />
<meta name="citation_reference" content="Randall G, Panis M, Cooper JD, et al. Cellular cofactors affecting hepatitis c virus infection and replication. Proceedings of the National Academy of Sciences of the United States of America, 2007, 104: 12884–12889" />
<meta name="citation_reference" content="Konig R, Zhou Y, Elleder D, et al. Global analysis of host-pathogen interactions that regulate early-stage hiv-1 replication. Cell, 2008, 135: 49–60" />
<meta name="citation_reference" content="Li W. The hepatitis b virus receptor. Annual review of cell and developmental biology, 2015, 31: 125–147" />
<meta name="citation_reference" content="Pillay S, Meyer NL, Puschnik AS, et al. An essential receptor for adeno-associated virus infection. Nature, 2016, 530: 108–112" />
<meta name="citation_reference" content="Agarwala R, Barrett T, Beck J, et al. Database resources of the national center for biotechnology information. Nucleic Acids Res, 2016, 44: D7–D19" />
<meta name="citation_reference" content="Masson P, Hulo C, De Castro E, et al. Viralzone: Recent updates to the virus knowledge resource. Nucleic Acids Res, 2013, 41: D579–D583" />
<meta name="citation_reference" content="Bateman A, Martin MJ, O’Donovan C, et al. Uniprot: The universal protein knowledgebase. Nucleic Acids Res, 2017, 45: D158–D169" />
<meta name="citation_reference" content="Hofmann K, Stoffel W. j Tmpred: Prediction of transmembrane regions and orientation. 2017, https://embnet.vital-it.ch/software/TMPRED_form.html" />
<meta name="citation_reference" content="Roy A, Kucukural A, Zhang Y. I-tasser: A unified platform for automated protein structure and function prediction. Nat Protoc, 2010, 5: 725–738" />
<meta name="citation_reference" content="Bernstein HJ. Rasmol 2.7.5. 2017, http://www.openrasmol.org/" />
<meta name="citation_reference" content="Yu GC, Wang LG, Han YY, et al. Clusterprofiler: An r package for comparing biological themes among gene clusters. Omics, 2012, 16: 284–287" />
<meta name="citation_reference" content="Team RC. R: A language and environment for statistical computing. R foundation for statistical computing, vienna, austria. 2016, http://www.R-project.org/" />
<meta name="citation_reference" content="G C, T N. The igraph software package for complex network research, interjournal, complex systems 1695. 2006, http://igraph.org" />
<meta name="citation_reference" content="Shannon P, Markiel A, Ozier O, et al. Cytoscape: A software environment for integrated models of biomolecular interaction networks. Genome research, 2003, 13: 2498–2504" />
<meta name="citation_reference" content="Szklarczyk D, Franceschini A, Wyder S, et al. String v10: Protein-protein interaction networks, integrated over the tree of life. Nucleic Acids Res, 2015, 43: D447–D452" />
<meta name="citation_reference" content="Pruitt KD, Tatusova T, Brown GR, et al. Ncbi reference sequences (refseq): Current status, new features and genome annotation policy. Nucleic Acids Res, 2012, 40: D130–D135" />
<meta name="citation_reference" content="Altschul SF, Gish W, Miller W, et al. Basic local alignment search tool. Journal of molecular biology, 1990, 215: 403–410" />
<meta name="citation_reference" content="Felsenstein J. Phylip - phylogeny inference package (version 3.2). Cladistics, 1989, 5: 3" />
<meta name="citation_reference" content="Eisenberg E, Levanon EY. Human housekeeping genes, revisited. Trends Genet, 2013, 29: 569–574" />
<meta name="citation_reference" content="Petryszak R, Keays M, Tang YA, et al. Expression atlas update—an integrated database of gene and protein expression in humans, animals and plants. Nucleic Acids Res, 2016, 44: D746–D752" />
<meta name="citation_reference" content="Robin X, Turck N, Hainard A, et al. Proc: An open-source package for r and s plus to analyze and compare roc curves. Bmc Bioinformatics, 2011, 12:" />
<meta name="twitter:title" content="Membrane proteins with high N-glycosylation, high expression, and multiple interaction partners were preferred by mammalian viruses as receptors" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Receptor mediated entry is the first step for viral infection. However, the relationship between viruses and receptors is still obscure. Here, by manually curating a high-quality database of 268 pairs of mammalian virus-host receptor interaction, which included 128 unique viral species or sub-species and 119 virus receptors, we found the viral receptors were structurally and functionally diverse, yet they had several common features when compared to other cell membrane proteins: more protein domains, higher level of N-glycosylation, higher ratio of self-interaction and more interaction partners, and higher expression in most tissues of the host. Additionally, the receptors used by the same virus tended to co-evolve. Further correlation analysis between viral receptors and the tissue and host specificity of the virus shows that the virus receptor similarity was a significant predictor for mammalian virus cross-species. This work could deepen our understanding towards the viral receptor selection and help evaluate the risk of viral zoonotic diseases." />
<meta name="og-title" property="og:title" content="Membrane proteins with high N-glycosylation, high expression, and multiple interaction partners were preferred by mammalian viruses as receptors" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/271171v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Receptor mediated entry is the first step for viral infection. However, the relationship between viruses and receptors is still obscure. Here, by manually curating a high-quality database of 268 pairs of mammalian virus-host receptor interaction, which included 128 unique viral species or sub-species and 119 virus receptors, we found the viral receptors were structurally and functionally diverse, yet they had several common features when compared to other cell membrane proteins: more protein domains, higher level of N-glycosylation, higher ratio of self-interaction and more interaction partners, and higher expression in most tissues of the host. Additionally, the receptors used by the same virus tended to co-evolve. Further correlation analysis between viral receptors and the tissue and host specificity of the virus shows that the virus receptor similarity was a significant predictor for mammalian virus cross-species. This work could deepen our understanding towards the viral receptor selection and help evaluate the risk of viral zoonotic diseases." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2018-03-08" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/271171v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/271171v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/271171v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/271171v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/85500" />
    <title>Membrane proteins with high N-glycosylation, high expression, and multiple interaction partners were preferred by mammalian viruses as receptors | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__vup9YXfxWOBOGbthJCUtxdPS6xRY2OQ_vXqbojfxPAk__ziC21uj-cnpEf3LG0kjRMEQHSEtjX5ljVuosklz2DZ4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"xAcjQm8Bs0-Q0i1r3NLYMt2ISPi9h3z_UFfvJ2aoYqk","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"85500","apath":"\/biorxiv\/early\/2018\/03\/08\/271171.atom","pisa":"biorxiv;271171v2","markup":[{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;271171\/DC2"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;271171\/DC2"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;271171\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;271171\/DC1"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;271171v2"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;271171\/DC2"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;271171\/DC2"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;271171\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;271171\/DC1"}],"processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/271171v2","title":"Membrane proteins with high N-glycosylation, high expression, and multiple interaction partners were preferred by mammalian viruses as receptors","identifier":"node\/85500"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002234yOpIS1YBpN3EnOk\\\\\\\/Ie9awY0fQastXQsH4NiJBAfBvmbISGxqGTrZcvSyiqy+vjJD043OxYrwQA2eU2+1a8Wun4zgGgc1KyIKCgW4O8LdSwRDIYzRgMFEHKa6yaOp0YyduqJ03BwLeC3oP+KmOvM\\\\\\\/+uxVU\\\\\\\/hfRwu3+0gq3pn7yQwUxgE7JykkjLXWXyTztHqnbCwze+kvOTasHRDy5eMLObtuRHbzeCy6XalScXPi4tgiUMCVxgtdn73CM3zUJt3tx4fGZdXkNzMpGvwh4gF1YUVpDmouKpg0MPwZt0k2Cg8ldhQSH9DvXeVMonQbMeMY+4+NkapcPPfyHO1hGdR72DJWdJ9lesPON0+fQIk6LW8czdjHCvQ3cvax2InxeYgTHZ5WoUVbQ+SP\\\\\\\/th1Vr2Xk9+P7GqQxqWXUE2CNRQYFDChe9vh5+4BX59zj9Y1SaTlJ6Z6j+zMfc\\\\\\\/4LbFqNQQHu0dKcSKmCNKPRxTCy1dyaFJbopTFXQonsp5p8nlPKDnrySLtUunk\\\\\\\/3fQZUw7UVec3+nlxCeArB6la9nsLnK5KgEhPMphF2U1sNz7\\\\\\\/ZHKgZ8kAp+neIzMg13\\\\\\\/8THVYm8RM6PSSgu8ta\\\\\\\/6gcUgwVO94LJgZpRJl8Ob6\\\\\\\/c35mcVQ8f2Er\\\\\\\/Awqh63n3vSbTHprzKzgIoT1ieGBjJcG4rr6wZpHvCtYqcgVFFDV8PX+KjQBaMKlLuKGIoP0N1EQ3lMs2L6r\\\\\\\/cPGNruW2kbyy1Puv\\\\\\\/w2y0NhncXO3A7mrMJ0wnJCEGxpwb2YbIHc+7R9ynlRSZ8r7sXTfKoghU6eY3yMO1qou+pZwvhFvIuBDBWQsB7jraLD6asU7NNY6niVpr6GlWhtw6nvol2vyvOz2OhOTGpi4JqtPKxc8pq0FUzWu\\\\\\\/7NFIj9cYP\\\\\\\/LOlYiwfC7g0SjwBAfnHh8vr\\\\\\\/jj2pZEkYN7EjbzN+olS5gjeC4dzaiNIi8D\\\\\\\/BiVdOFyEZuyDYTkXbtSAeTP6aBdFemC1tF4KCSjC2p7RZtclxBOZqtgWLyLvhhhbNHQzkxHo35sgwui6OUx8mv57Oc3GQ7EkVIkDdqMfS8\\\\\\\/YLlxn5l25cd0gT5OQFisZ4k1pBNQCJ5EiS7gUbVpxDn7tbKN+y+OUBPO7boMDMPIHFF3IWvr9bz13GR6xdcVNMzoY2Frt6zzz\\\\\\\/g5ZQac3sjfvIfn6skC3sLzglYC96Gmyh7CLcsFoXtYibPIHptWs\\\\\\\/3Fy85ecqh3s9aRqUCIKngaw6BQ5EFQIWDl3MXurkEdWBBOhyMlmKlzOXtcxTtuxm14LB3Su6xEdMUFBYDBu5pEliE5n0QxgD3riZVjQ=\\\u0022,\\\u0022iv\\\u0022:\\\u0022MOT3M6qoOGCgfE7PzEJvcQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00224774e8ee3eb89cef6b2fdfbab38d303d0facccd198cf42587fd2f663904f369d\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/271171v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-85500 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/271171v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-pCz0NJWmoHqpDJb7c2-9EJinx97oi5NL_TuocKhxrds" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="85500" id="node85500" data-pisa="biorxiv;271171v2" data-pisa-master="biorxiv;271171" data-apath="/biorxiv/early/2018/03/08/271171.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Membrane proteins with high N-glycosylation, high expression, and multiple interaction partners were preferred by mammalian viruses as receptors</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Zheng</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Zhaozhong</span> <span class="nlm-surname">Zhu</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Wenjun</span> <span class="nlm-surname">Chen</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Zena</span> <span class="nlm-surname">Cai</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Beibei</span> <span class="nlm-surname">Xu</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Zhiying</span> <span class="nlm-surname">Tan</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Aiping</span> <span class="nlm-surname">Wu</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Xingyi</span> <span class="nlm-surname">Ge</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Xinhong</span> <span class="nlm-surname">Guo</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Zhongyang</span> <span class="nlm-surname">Tan</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Zanxian</span> <span class="nlm-surname">Xia</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Haizhen</span> <span class="nlm-surname">Zhu</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Taijiao</span> <span class="nlm-surname">Jiang</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Yousong</span> <span class="nlm-surname">Peng</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/271171 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node85500" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zheng Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>College of Biology, Hunan University</span>, Changsha, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zheng%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZheng%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Zhaozhong Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>College of Biology, Hunan University</span>, Changsha, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhaozhong%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhaozhong%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Wenjun Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>College of Biology, Hunan University</span>, Changsha, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wenjun%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWenjun%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Zena Cai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>College of Biology, Hunan University</span>, Changsha, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zena%2BCai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cai%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZena%2BCai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Beibei Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>College of Computer Science and Electronic Engineering, Hunan University</span>, Changsha, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Beibei%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABeibei%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Zhiying Tan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>College of Computer Science and Electronic Engineering, Hunan University</span>, Changsha, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhiying%2BTan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tan%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhiying%2BTan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Aiping Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Center of System Medicine, Institute of Basic Medical Sciences, Chinese Academy of Medical Sciences &#38; Peking Union Medical College</span>, Beijing, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Suzhou Institute of Systems Medicine</span>, Suzhou, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aiping%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAiping%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Xingyi Ge </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>College of Biology, Hunan University</span>, Changsha, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xingyi%2BGe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ge%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXingyi%2BGe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Xinhong Guo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>College of Biology, Hunan University</span>, Changsha, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xinhong%2BGuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guo%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXinhong%2BGuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Zhongyang Tan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>College of Biology, Hunan University</span>, Changsha, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhongyang%2BTan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tan%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhongyang%2BTan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Zanxian Xia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>School of Life Sciences, Central South University</span>, Changsha, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zanxian%2BXia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xia%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZanxian%2BXia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Haizhen Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>College of Biology, Hunan University</span>, Changsha, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>6</span><span class='nlm-institution'>State Key Laboratory of Chemo/Biosensing and Chemometrics, Hunan University</span>, Changsha, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haizhen%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHaizhen%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zhuhaizhen69@yahoo.com" class="" data-icon-position="" data-hide-link-title="0">zhuhaizhen69@yahoo.com</a>
<a href="mailto:taijiao@ibms.pumc.edu.cn" class="" data-icon-position="" data-hide-link-title="0">taijiao@ibms.pumc.edu.cn</a>
<a href="mailto:pys2013@hnu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">pys2013@hnu.edu.cn</a></span></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Taijiao Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Center of System Medicine, Institute of Basic Medical Sciences, Chinese Academy of Medical Sciences &#38; Peking Union Medical College</span>, Beijing, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Suzhou Institute of Systems Medicine</span>, Suzhou, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Taijiao%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATaijiao%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zhuhaizhen69@yahoo.com" class="" data-icon-position="" data-hide-link-title="0">zhuhaizhen69@yahoo.com</a>
<a href="mailto:taijiao@ibms.pumc.edu.cn" class="" data-icon-position="" data-hide-link-title="0">taijiao@ibms.pumc.edu.cn</a>
<a href="mailto:pys2013@hnu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">pys2013@hnu.edu.cn</a></span></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Yousong Peng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>College of Biology, Hunan University</span>, Changsha, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yousong%2BPeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Peng%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYousong%2BPeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zhuhaizhen69@yahoo.com" class="" data-icon-position="" data-hide-link-title="0">zhuhaizhen69@yahoo.com</a>
<a href="mailto:taijiao@ibms.pumc.edu.cn" class="" data-icon-position="" data-hide-link-title="0">taijiao@ibms.pumc.edu.cn</a>
<a href="mailto:pys2013@hnu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">pys2013@hnu.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/271171v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:85500" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:85500/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/271171v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:85500" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:85500/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/271171v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:85500" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:85500/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/271171v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:85500" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:85500/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/271171v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:85500" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:85500/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/271171v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:85500" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:85500/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">Receptor mediated entry is the first step for viral infection. However, the relationship between viruses and receptors is still obscure. Here, by manually curating a high-quality database of 268 pairs of mammalian virus-host receptor interaction, which included 128 unique viral species or sub-species and 119 virus receptors, we found the viral receptors were structurally and functionally diverse, yet they had several common features when compared to other cell membrane proteins: more protein domains, higher level of N-glycosylation, higher ratio of self-interaction and more interaction partners, and higher expression in most tissues of the host. Additionally, the receptors used by the same virus tended to co-evolve. Further correlation analysis between viral receptors and the tissue and host specificity of the virus shows that the virus receptor similarity was a significant predictor for mammalian virus cross-species. This work could deepen our understanding towards the viral receptor selection and help evaluate the risk of viral zoonotic diseases.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-4">In the new century, much progress has been made in prevention and control of infectious diseases, but the recent serial outbreaks of Zika virus <sup>[<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>]</sup>, Ebola virus (EBOV) <sup>[<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]</sup> and Middle East Respiratory Syndrome Coronavirus (MERS-CoV) <sup>[<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>]</sup> indicate that the viral infectious diseases still pose a serious threat to human health and global security. The virus is the most abundant biological entity on Earth and exists in all habitats of the world <sup>[<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>]</sup>. Nearly all cellular organisms are prey to viral attack. Humans were reported to be infected by hundreds of viruses <sup>[<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>]</sup>. Most of the human emerging infectious diseases are zoonotic, with viruses that originate in mammals of particular concern <sup>[<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>]</sup>, such as the Human Immunodeficiency Virus (HIV) <sup>[<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>]</sup> and Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) <sup>[<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>]</sup>. Mammals are not only the most closely related animal to humans in phylogeny, but also contact with humans most frequently <sup>[<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>]</sup>, especially for the livestock and pet. For effective control of human viral diseases, much attention should be paid to the mammalian virus.</p><p id="p-5">Receptor-binding is the first step for viral infection of host cells <sup>[<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>-<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>]</sup>. Multiple types of molecules could be used as viral receptors <sup>[<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>]</sup>, including protein <sup>[<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>-<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>]</sup>, carbohydrate <sup>[<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>]</sup> and lipid <sup>[<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>]</sup>. How to select receptors by the virus is an important unsolved question <sup>[<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>, <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>]</sup>. Specificity and affinity are two most important factors for viral receptor selection <sup>[<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">14</a>]</sup>. Carbohydrates and lipids are widely distributed on host cell surfaces and easy targets for viruses to grab <sup>[<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>]</sup>. Compared to these molecules, proteins were reported to be more suitable receptors because of stronger affinity and higher specificity for viral attachment, which could increase the efficiency of viral entry and facilitate viruses to expand their host ranges and alter their tropisms <sup>[<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">10</a>-<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a></sup>, <sup><a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>]</sup> Previous studies have shown that proteins that were abundant in the host cell surface or had relatively low affinity for their natural ligands, were preferred by viruses as receptors, such as proteins involved in cell adhesion and recognition by reversible, multivalent avidity-determined interactions <sup>[<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">10</a>, <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">15</a>]</sup>. This suggests that the selection of proteins by viruses as receptors should not be a random process. A systematic analysis of the characteristics of the viral receptor could help understand the mechanisms under the receptor selection by viruses.</p><p id="p-6">The virus-receptor interaction was reported to be a principal determinant of viral host range, tissue tropism and cross-species infection <sup>[<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>]</sup>. The existence and expression of the virus receptor in a host (or tissue) should be a prerequisite for viral infection of the host (or tissue) <sup>[<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>]</sup>. Usually, a virus mainly infects some particular type of hosts or tissues. For example, the influenza virus mostly infects cells of the respiratory tract <sup>[<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>]</sup>. However, the virus-receptor interaction is a highly dynamic process. Some viruses can recognize one or more receptors <sup>[<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">13</a>, <a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>]</sup>, which can also differ among virus variants or during the course of infections <sup>[<a id="xref-ref-14-6" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>]</sup>. In some cases, a few amino acid mutations in the viral protein or the receptor could abolish or enhance viral infection <sup>[<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>-<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>]</sup>. Besides, the virus-receptor interaction is under continuous evolutionary pressure to increase the viral infection efficiency, which may result in the emergence of virus variants with altered host or tissue tropism. For example, the SARS-CoV and MERS-CoV, which belong to the same genus, <em>betacoronavirus</em>, have evolved to use different receptors (angiotensin I converting enzyme 2 (ACE2) and dipeptidyl peptidase 4 (DPP4) respectively) and also infect different hosts <sup>[<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">16</a>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>]</sup>. Despite of numerous studies about the tissue and host specificity of the virus and the viral receptor, the systematical correlation characteristics between them are still obscure.</p><p id="p-7">Here, by manually curating a high-quality database of 268 pairs of mammalian virus-receptor interaction, which included 128 unique viral species or sub-species and 119 virus receptors, we systematically analyzed the structural, functional, evolutionary and tissue-specific expression characteristics of mammalian virus receptors, which could not only deepen our understanding towards the mechanism behind the viral receptor selection, but also help to predict and identify viral receptors. Besides, we also investigated the associations between the tissue and host specificity of the virus and the viral receptor, and further evaluated the risk of viral cross-species based on viral receptors. It would help for early warning and prediction of viral zoonotic diseases.</p></div><div class="section" id="sec-2"><h2 class="">Results</h2><div id="sec-3" class="subsection"><h3>Database of mammalian virus-host receptor interaction</h3><p id="p-8">To understand how the virus selects receptors, we manually curated a high-quality database of 268 pairs of mammalian virus-host receptor interactions (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and Table S1), which included 128 unique viral species or sub-species from 21 viral families and 119 virus receptors from 13 mammal species. The viral receptor collected here belonged to 13 mammal species (Figure S1A), among which the human accounted for the most (74/119). The viruses included in the database covered all groups of viruses in the Baltimore classification (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Among them, the single-stranded RNA (ssRNA) virus accounted for over half of all viruses (76/128), while the double-stranded RNA (dsRNA) virus accounted for the least (3/128). On the level of family, the family of <em>Picornaviridae</em> of ssRNA virus, <em>Retroviridae</em> of Retro-transcribing viruses (RT) and <em>Herpesviridae</em> of double-stranded DNA (dsDNA) viruses were the most abundant ones in the database (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> and Table S1).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/08/271171/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The mammalian viruses and their related receptors in our database. The lines between the virus and their related receptors were colored according to the group of the virus in the Baltimore classification. The names of some viral families were presented in italic. Viral names were displayed in abbreviation (see Table S1 for the full name). The host names were given for the receptor of non-human mammal species. The receptor CD209 was marked with an asterisk. For more details about the mammalian virus and their receptors, please see the website http://www.computationalbiology.cn:5000/viralReceptor." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1039870262" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The mammalian viruses and their related receptors in our database. The lines between the virus and their related receptors were colored according to the group of the virus in the Baltimore classification. The names of some viral families were presented in italic. Viral names were displayed in abbreviation (see Table S1 for the full name). The host names were given for the receptor of non-human mammal species. The receptor CD209 was marked with an asterisk. For more details about the mammalian virus and their receptors, please see the website http://www.computationalbiology.cn:5000/viralReceptor.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/03/08/271171/F1.medium.gif" width="209" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2018/03/08/271171/F1.medium.gif" width="209" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/08/271171/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/08/271171/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-9" class="first-child">The mammalian viruses and their related receptors in our database. The lines between the virus and their related receptors were colored according to the group of the virus in the Baltimore classification. The names of some viral families were presented in italic. Viral names were displayed in abbreviation (see Table S1 for the full name). The host names were given for the receptor of non-human mammal species. The receptor CD209 was marked with an asterisk. For more details about the mammalian virus and their receptors, please see the website <a href="http://www.computationalbiology.cn:5000/viralReceptor">http://www.computationalbiology.cn:5000/viralReceptor</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Association between mammalian viruses and their receptors</h3><p id="p-10">Analysis of the association between the virus and their receptors showed that 60% of viruses (77/128) used only one receptor (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a> and Figure S1B), while the remaining viruses used two or more receptors. Surprisingly, some viruses, such as the Human alphaherpesvirus 1 (HSV-1) and Hepacivirus C (HCV), used more than five receptors. We next analyzed the receptor usage on the level of viral family. For fifteen viral families including two or more viruses in the database, all of them used two or more sets of receptors, suggesting that different viruses of the same family tend to use different receptors. For example, in the family of <em>Togaviridae</em>, the Chikungunya virus (CHIKV), the Rubella virus (RBV) and the Sindbis virus (SBV) used the receptor of prohibitin (PHB), myelin oligodendrocyte glycoprotein (MOG) and ribosomal protein SA (RPSA), respectively. On the other hand, some viruses of different families or even different groups used the same receptor (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>). For example, HIV-2 and EBOV, from the family of <em>Retroviridae</em> (RT group) and <em>Filoviridae</em> (ssRNA group) respectively, both took CD209 molecule (CD209) (marked with an asterisk in <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a>) as the receptor. On average, each receptor was used by more than two viruses. More specifically, among 119 virus receptors, forty-four of them were used by more than one virus (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a> and Figure S1C); twenty-one of them were used by viruses of more than one family and fifteen of them were used by viruses of more than one group (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1</a>).</p></div><div id="sec-5" class="subsection"><h3>Structural, functional, evolutionary and tissue-specific expression characteristics of mammalian virus receptors</h3><p id="p-11">To understand how the virus selects receptors, we systematically analyzed the structural, functional, evolutionary and tissue-specific expression characteristics of the mammalian virus receptor.</p><div id="sec-6" class="subsection"><h4>1) The mammalian virus receptor were structurally diverse</h4><p id="p-12">We firstly investigated the structural characteristics of mammalian virus receptor proteins. As expected, all the mammalian virus receptor protein belonged to the membrane protein which had at least one transmembrane alpha helix (Figure S2A). Twenty-four of them had more than five helixes, such as 5-hydroxytryptamine receptor 2A (HTR2A) and NPC intracellular cholesterol transporter 1 (NPC1). The receptor protein was mainly located in the cell membrane. Besides, more than one third (43/119) of them were also located in the cytoplasm, and thirteen of them were located in the nucleus.</p><p id="p-13">Then, the protein domain composition of the mammalian virus receptor protein was analyzed. The mammalian virus receptor proteins contained a total of 336 domains based on the Pfam database, with each viral receptor protein containing more than two domains on average (Figure S2B). This was significantly more than that of human proteins or human membrane proteins (p-values &lt; 0.001 in the Wilcoxon rank-sum test). Some viral receptor proteins may contain more than 10 domains, such as complement C3d receptor 2 (CR2) and low density lipoprotein receptor (LDLR). The protein domains of the mammalian virus receptor protein could be grouped into 77 families in the Pfam database, suggesting the structure diversity of the mammalian virus receptor protein. The most commonly observed Pfam families were Immunoglobulin V-set domain, Immunoglobulin C2-set domain, Integrin beta chain VWA domain, Integrin plexin domain, and so on (Figure S2C).</p></div><div id="sec-7" class="subsection"><h4>2) The mammalian virus receptor had high level of N-glycosylation</h4><p id="p-14">Glycosylation of protein is widespread in the eukaryote cell. We next characterized the glycosylation level of the mammalian virus receptor. N-glycosylation is the most common type of glycosylation. We found that 93 of 119 mammalian virus receptors were N-glycosylated with an average of 0.94 glycosylation sites per 100 amino acids (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). It increased to 0.97 glycosylation sites per 100 amino acids for the human viral receptor (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>), among which 62 were N-glycosylated. Twelve human viral receptors were observed to have ten or more N-glycosylation sites, such as complement C3b/C4b receptor 1 (CR1) and lysosomal associated membrane protein 1 (LAMP1). <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a> displayed the modeled 3D-structure of HTR2A, the receptor for JC polyomavirus (JCPyV). Five N-glycosylation sites were highlighted in red on the structure, which were reported to be important for viral infection <sup>[<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>]</sup>. For comparison, we also characterized the N-glycosylation level for the human cell membrane protein, human membrane proteins and all human proteins (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2A</a>). It was found they had a significantly lower level of N-glycosylation than that of human and mammalian virus receptors (p-values &lt; 0.001 in the Wilcoxon rank-sum test), which suggests the importance of N-glycosylation for the viral receptor.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/08/271171/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of N-glycosylation and protein-protein interactions of mammalian virus receptors. (A) Comparison of the N-glycosylation level between mammalian viral receptors, human viral receptors, human cell membrane proteins, human membrane proteins and all human proteins. For clarity, the outliers greater than 2.5 were removed. &#x201C;***&#x201D;, p-value &lt; 0.001. (B) The modeled 3D-structure of HTR2A. Five N-glycosylation sites were highlighted in red. Artificial glycans were manually added onto the site. (C) Comparison of the degree of proteins between human viral receptors, human cell membrane proteins, human membrane proteins and all human proteins in the human PPIN. For clarity, the node degree was logarithmically transformed. &#x201C;***&#x201D;, p-value &lt; 0.001. (D) Partial human PPI network composing of the PPIs which involved at least one viral receptor protein (colored in red)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1039870262" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Analysis of N-glycosylation and protein-protein interactions of mammalian virus receptors. (A) Comparison of the N-glycosylation level between mammalian viral receptors, human viral receptors, human cell membrane proteins, human membrane proteins and all human proteins. For clarity, the outliers greater than 2.5 were removed. &#x201C;***&#x201D;, p-value &lt; 0.001. (B) The modeled 3D-structure of HTR2A. Five N-glycosylation sites were highlighted in red. Artificial glycans were manually added onto the site. (C) Comparison of the degree of proteins between human viral receptors, human cell membrane proteins, human membrane proteins and all human proteins in the human PPIN. For clarity, the node degree was logarithmically transformed. &#x201C;***&#x201D;, p-value &lt; 0.001. (D) Partial human PPI network composing of the PPIs which involved at least one viral receptor protein (colored in red).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/03/08/271171/F2.medium.gif" width="320" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2018/03/08/271171/F2.medium.gif" width="320" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/08/271171/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/08/271171/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-15" class="first-child">Analysis of N-glycosylation and protein-protein interactions of mammalian virus receptors. (A) Comparison of the N-glycosylation level between mammalian viral receptors, human viral receptors, human cell membrane proteins, human membrane proteins and all human proteins. For clarity, the outliers greater than 2.5 were removed. “***”, p-value &lt; 0.001. (B) The modeled 3D-structure of HTR2A. Five N-glycosylation sites were highlighted in red. Artificial glycans were manually added onto the site. (C) Comparison of the degree of proteins between human viral receptors, human cell membrane proteins, human membrane proteins and all human proteins in the human PPIN. For clarity, the node degree was logarithmically transformed. “***”, p-value &lt; 0.001. (D) Partial human PPI network composing of the PPIs which involved at least one viral receptor protein (colored in red).</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">O-glycosylation is also a common type of glycosylation. We found there was only a small fraction of mammalian virus receptors (14/119) with O-glycosylation. Besides, no significant difference was observed between the O-glycosylation level of mammalian virus receptor proteins and that of human proteins (Figure S2D).</p></div><div id="sec-8" class="subsection"><h4>3) Functional enrichment analysis of the human virus receptor</h4><p id="p-17">We next attempted to identify the gene functions and pathways enriched in the mammalian virus receptor. As was mentioned above, 74 of 119 mammalian virus receptors belonged to the human. Besides, analysis showed that 36 of the remaining non-human mammalian virus receptors were homologs of the human virus receptor (Table S2). Therefore, we conducted the function enrichment analysis only for the human virus receptor based on the databases of Gene Ontology (GO) and KEGG. For the GO Cellular Component (Table S3), the human virus receptor was mainly enriched in the membranes and junctions, the latter of which included the adherens junction, cell-substrate junction, focal adhesion, and so on. For the GO Biological Process (Table S3), the human virus receptor was mainly enriched in the process of entry into the host. Besides, some terms related to the immune response were also enriched, such as “Regulation of leukocyte activation” and “Lymphocyte activation”. For the GO Molecular Function (Table S3), besides for the enrichment of terms related to the virus receptor activity, the human virus receptor was also enriched in terms of binding to integrin, glycoprotein, cytokine, and so on.</p><p id="p-18">Consistent with the enrichment analysis of GO Cellular Component, the KEGG pathways of “Cell adhesion molecules”, “Focal adhesion” and “ECM-receptor interaction” were also enriched. Besides, the pathway of “Phagosome” was enriched (Table S3), which may be associated with viral entry into the host cell. Interestingly, some pathways associated with heart diseases were enriched, including “Dilated cardiomyopathy”, “Hypertrophic cardiomyopathy”, “Arrhythmogenic right ventricular cardiomyopathy” and “Viral myocarditis”.</p></div><div id="sec-9" class="subsection"><h4>4) Human virus receptors had more interaction partners than other proteins</h4><p id="p-19">We next analyzed the protein-protein interactions (PPIs) which the mammalian virus receptor protein took part in. As the reason mentioned above, we only used the human virus receptor for PPI analysis. A human PPI network (PPIN) was constructed based on the work of Menche et al <sup>[<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>]</sup>. It included a total of 13,460 human proteins that are interconnected by 141,296 interactions. The degree and betweenness of each protein in the PPIN were calculated, which could measure the importance of a protein in the PPIN. It was found that the degrees for human membrane proteins and cell membrane proteins were significantly smaller than those of other human proteins (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2C</a> &amp; Figure S3A, p-value &lt; 0.001 in Wilcox rank-sum test) in the PPIN. Similar observations could be found for the node betweenness in the PPIN (Figure S3B&amp;C). However, the human virus receptor protein, a subset of the human cell membrane protein, was found to have significantly larger degrees and higher betweenness than other human proteins in the PPIN (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2C</a> and Figure S3, p-value &lt; 0.001 in Wilcox rank-sum test). The median degrees for human virus receptors was 13 (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2C</a>), which was nearly twice as much as that of all human proteins. Six viral receptors were observed to have degrees larger than 100 (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2D</a>), including epidermal growth factor receptor (EGFR), heat shock protein family A member 8 (HSPA8), PHB, RPSA, CD4 molecule (CD4) and integrin subunit beta 1 (ITGB1). Since the viral receptor (colored in red in <a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2D</a>) interacted with lots of other human proteins (colored in black in <a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2D</a>) in PPIN, we further investigated the functional enrichment of these proteins by GO enrichment analysis. Interestingly, six of top ten enriched terms in the domain of Biological Process were related to protein targeting or localization (Table S3).</p><p id="p-20">When looking at the interactions between viral receptors, we found that 38 of 74 viral receptors interacted with themselves. This ratio (38/74 = 51%) was much higher than that of human proteins (22%), membrane proteins (11%) and human cell membrane proteins (14%). However, we found the viral receptor tended not to interact with each other (Figure S3D). Among 74 human virus receptor proteins, 36 of them had no interactions with any other human virus receptor. There were only 50 PPIs between different human virus receptor proteins, with each viral receptor protein interacting with an average of only one other viral receptor protein.</p></div><div id="sec-10" class="subsection"><h4>5) The mammalian virus receptor was not more conserved than other genes</h4><p id="p-21">Large degree of the human viral receptor in the human PPIN suggests the importance of them in cellular activity. Analysis showed that 11 human viral receptors belonged to the housekeeping gene. This ratio (0.15 = 11/74) was a little lower than that of housekeeping genes in all human genes (0.19 = 3804/20243), suggesting that the human viral receptor was not enriched in the housekeeping gene.</p><p id="p-22">Then, we investigated the evolutionary conservation of mammalian virus receptors in 108 mammal species which were richly annotated in the NCBI Reference Sequences (RefSeq) database (see Methods). Over half of mammalian virus receptors had homologs (see Methods) in more than 100 mammal species (Figure S4A). We further calculated the pairwise sequence identities between the viral receptor and their homologs in mammal species. For nearly half of viral receptors, the average of pairwise sequence identities was higher than 0.8 (Figure S4B). For comparison, we also analyzed the evolutionary conservation of human proteins by randomly selecting 1000 human proteins from the NCBI RefSeq database. They were observed to have similar conservation level with that of human viral receptors (Figure S4C&amp;D and Table S4).</p></div><div id="sec-11" class="subsection"><h4>6) Viral receptors expressed higher than other proteins in 32 major human tissues</h4><p id="p-23">Since the virus has to compete with other proteins for binding to the receptor, proteins with high expression should be preferred by viruses as receptors. Thus we measured the average expression level of human viral receptors in 32 major human tissues (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). For comparison, those of human membrane genes, human cell membrane genes and all human genes were also displayed in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>. As was shown clearly, the expression level of human cell membrane genes (in cyan) was similar to that of all human genes (in black) in these tissues. Both of them were lower than that of human membrane genes (in blue). However, the human viral receptor (in red), which was part of the human cell membrane gene, expressed much higher than other sets of genes in nearly all these tissues. On average, they had an expression level of 24 transcripts per million (TPM) in these tissues, while this was 8, 4 and 4 TPM for the human membrane gene, the human cell membrane gene and all human genes (see the black arrow in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/08/271171/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The average expression level of human viral receptors (red), human cell membrane genes (cyan), human membrane genes (blue) and all human genes (black) in 32 major human tissues. The expression level was measured with transcripts per million (TPM). The black arrow refers to the average expression level of genes in all 32 tissues." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1039870262" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The average expression level of human viral receptors (red), human cell membrane genes (cyan), human membrane genes (blue) and all human genes (black) in 32 major human tissues. The expression level was measured with transcripts per million (TPM). The black arrow refers to the average expression level of genes in all 32 tissues.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/03/08/271171/F3.medium.gif" width="440" height="249"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2018/03/08/271171/F3.medium.gif" width="440" height="249"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/08/271171/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/08/271171/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <p id="p-24" class="first-child">The average expression level of human viral receptors (red), human cell membrane genes (cyan), human membrane genes (blue) and all human genes (black) in 32 major human tissues. The expression level was measured with transcripts per million (TPM). The black arrow refers to the average expression level of genes in all 32 tissues.</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-12" class="subsection"><h3>Viral receptors used by the same virus tended to co-evolve</h3><p id="p-25">The results mentioned above shows that a total of 51 viruses used more than one viral receptor. These viral receptors may work together or independently. We then analyzed the relationship between them. Structural analysis shows that except for integrins which generally work in heterodimer, few of viral receptors used by the same virus shared the same protein domain (data not shown). This suggests that when the virus expands the use of receptors, it tends to select structurally diverse proteins. We continued to analyze the co-evolution between mammalian viral receptors in 108 mammal species. We found that the average of Spearman Correlation Coefficients (SCCs, a measure of the extent of co-evolution) between viral receptors employed by the same virus was 0.54, which was significantly larger than that between other viral receptors (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>, p-value &lt; 0.001 in the Wilcoxon rank-sum test). For example, SARS-CoV used four receptors, i.e., ACE2, CD209, C-type lectin domain family 4 member G (CLEC4G) and member M (CLEC4M). The average of SCCs between these four receptors was as high as 0.86. In addition, we analyzed the extent of co-expression between human viral receptors in 32 tissues. It was found that the extent of co-expression between viral receptors employed by the same virus was a little larger than that between other viral receptors, yet this difference was not statistically significant (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>, p-value &gt; 0.1 in the Wilcoxon rank-sum test).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/08/271171/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The co-evolution and co-expression of viral receptors. (A) Comparing the extent of co-evolution between mammalian virus receptors in 108 mammal species in the set of receptors used by the same virus and all mammalian virus receptors. &#x201C;***&#x201D;, p-value &lt; 0.001 in the Wilcoxon rank-sum test. (B) Comparing the extent of co-expression between human viral receptors in 32 human tissues in the set of receptors used by the same virus and all human virus receptors." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1039870262" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The co-evolution and co-expression of viral receptors. (A) Comparing the extent of co-evolution between mammalian virus receptors in 108 mammal species in the set of receptors used by the same virus and all mammalian virus receptors. &#x201C;***&#x201D;, p-value &lt; 0.001 in the Wilcoxon rank-sum test. (B) Comparing the extent of co-expression between human viral receptors in 32 human tissues in the set of receptors used by the same virus and all human virus receptors.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/03/08/271171/F4.medium.gif" width="440" height="324"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2018/03/08/271171/F4.medium.gif" width="440" height="324"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/08/271171/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/08/271171/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <p id="p-26" class="first-child">The co-evolution and co-expression of viral receptors. (A) Comparing the extent of co-evolution between mammalian virus receptors in 108 mammal species in the set of receptors used by the same virus and all mammalian virus receptors. “***”, p-value &lt; 0.001 in the Wilcoxon rank-sum test. (B) Comparing the extent of co-expression between human viral receptors in 32 human tissues in the set of receptors used by the same virus and all human virus receptors.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Analysis of the association between the tissue and host specificity of the virus and the viral receptor</h3><p id="p-27">Although there were plenty of studies about the tissue and host specificity of the virus and the viral receptor, there was still a lack of systematic analysis towards the association between them. Besides, few studies quantify such associations. Therefore, we further investigated systematically the association between the tissue and host specificity of the virus and the viral receptor.</p><div id="sec-14" class="subsection"><h4>1) Viral receptor expressed higher in tissues infected by viruses than in those not infected</h4><p id="p-28">To investigate the association between the tissue specificity of the virus and tissue-specific expression of viral receptors, we manually compiled the tissue tropism of viruses from the literature or Wikipedia and obtained that in 32 human tissues for a total of 52 viruses (Table S5). Some viral receptors had high expression levels in most tissues, most of which were housekeeping genes, such as CD81 molecule (CD81) and ITGB1. While for most viral receptors, their expression levels varied much in different tissues. Analysis of the association between the tissue-specific expression of viral receptors and viral tissue tropism showed that the viral receptor expressed higher in the tissues infected by viruses (marked with asterisks in Table S5) than in those not infected, yet this difference was not statistically significant (p-value &gt; 0.1 in the Wilcoxon rank-sum test) (Figure S5). For example, the neural cell adhesion molecule 1 (NCAM1), which was employed by the Rabies lyssavirus (RabiesV) as the receptor, expressed much higher in the tissue of Cerebral cortex (infected by RabiesV) than in other tissues not infected by the virus (Table S5).</p></div><div id="sec-15" class="subsection"><h4>2) The viral receptor was a significant predictor in predicting viral cross-species in mammal species</h4><p id="p-29">Since the viral receptor determines the host specificity of the virus to a large extent, it is expected that the closer between the viral receptor and its homolog in a species, the more likely the virus which used the receptor would infect the species. To validate this hypothesis, we firstly calculated the sequence identities between the viral receptor and their homologs in 108 mammal species (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> and Table S6). For clarity, only 26 mammal species, which were frequently observed, were presented in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>. Then, we compared the sequence identities between viral receptor proteins and their homologs in the species infected by the virus which used the receptor (marked with asterisks and triangles), and in those not infected by the virus. As expected, the former was significantly higher than the latter (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>, p-value &lt; 0.001 in the Wilcoxon rank-sum test).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/08/271171/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of the association between the host specificity of the virus and the viral receptor. It listed the sequence identities (colored according to the legend) between the viral receptor and its homologs in 26 mammal species (at the bottom). The white referred to no homologs in the species. The viral receptor and the virus which used them were displayed in the left and right side of the figure respectively. For the non-human mammalian virus receptor, the species it belongs to was presented in brackets. The asterisk referred to the viral infection of the mammal species based on Olival&#x2019;s work, while the triangle referred to that based on our database. For more details, please see Table S6." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1039870262" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Analysis of the association between the host specificity of the virus and the viral receptor. It listed the sequence identities (colored according to the legend) between the viral receptor and its homologs in 26 mammal species (at the bottom). The white referred to no homologs in the species. The viral receptor and the virus which used them were displayed in the left and right side of the figure respectively. For the non-human mammalian virus receptor, the species it belongs to was presented in brackets. The asterisk referred to the viral infection of the mammal species based on Olival&#x2019;s work, while the triangle referred to that based on our database. For more details, please see Table S6.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/03/08/271171/F5.medium.gif" width="248" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2018/03/08/271171/F5.medium.gif" width="248" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/08/271171/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/08/271171/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <p id="p-30" class="first-child">Analysis of the association between the host specificity of the virus and the viral receptor. It listed the sequence identities (colored according to the legend) between the viral receptor and its homologs in 26 mammal species (at the bottom). The white referred to no homologs in the species. The viral receptor and the virus which used them were displayed in the left and right side of the figure respectively. For the non-human mammalian virus receptor, the species it belongs to was presented in brackets. The asterisk referred to the viral infection of the mammal species based on Olival’s work, while the triangle referred to that based on our database. For more details, please see Table S6.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/08/271171/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Quantify to what extent the viral receptor determine the host specificity of the virus. (A) Sequence identities between viral receptors and their homologs in mammal species infected by viruses and those not infected. &#x201C;***&#x201D;, p-value &lt; 0.001. (B) The Receiver Operating Characteristic (ROC) curve for models of predicting viral cross-species in mammal species based on host relatedness (in black) and receptor sequence identity (in red). AUC, Area Under the ROC Curve." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1039870262" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Quantify to what extent the viral receptor determine the host specificity of the virus. (A) Sequence identities between viral receptors and their homologs in mammal species infected by viruses and those not infected. &#x201C;***&#x201D;, p-value &lt; 0.001. (B) The Receiver Operating Characteristic (ROC) curve for models of predicting viral cross-species in mammal species based on host relatedness (in black) and receptor sequence identity (in red). AUC, Area Under the ROC Curve.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/03/08/271171/F6.medium.gif" width="440" height="283"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2018/03/08/271171/F6.medium.gif" width="440" height="283"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/08/271171/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/03/08/271171/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6.</span> <p id="p-31" class="first-child">Quantify to what extent the viral receptor determine the host specificity of the virus. (A) Sequence identities between viral receptors and their homologs in mammal species infected by viruses and those not infected. “***”, p-value &lt; 0.001. (B) The Receiver Operating Characteristic (ROC) curve for models of predicting viral cross-species in mammal species based on host relatedness (in black) and receptor sequence identity (in red). AUC, Area Under the ROC Curve.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">Previous work by Olival et al. showed that phylogenetic relatedness of host was a major factor which influenced the cross-species of mammalian viruses <sup>[<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">7</a>]</sup>. We compared the ability of the host relatedness and the viral receptor similarity in predicting viral cross-species in mammal species. The models based on the latter achieved an Area Under the ROC Curve (AUC) of 0.65 (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>), a little higher than that of the model based on the former (0.62), although this difference was not statistically significant (p-value = 0.13). This suggests that the viral receptor similarity should be a significant predictor in predicting viral cross-species in mammal species as well as the host relatedness.</p><p id="p-33">Based on the results mentioned above, we continued to evaluate the risk of viral cross-species transmission in 108 mammal species based on viral receptors. <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a> shows that more than 40% of viral receptors, such as insulin degrading enzyme (IDE) and PHB, had high sequence identities with their homologs in most mammal species, suggesting that the virus using them may have a high probability to infect these species. On the other hand, some mammal species had homologs which were highly similar to most viral receptors, such as the primates (Chimpanzee, Macaca mulatta and Lowland gorilla). They may have a high risk of infection by the virus which used these receptors.</p></div></div></div><div class="section" id="sec-16"><h2 class="">Discussion</h2><p id="p-34">The viral receptor is essential for viral infection. By collecting the largest dataset ever reported about the mammalian virus-host receptor interactions, we systematically analyzed the structural, functional, evolutionary and tissue-specific expression characteristics of mammalian virus receptors. We found that the viral receptors were a subset of structurally and functionally diverse cell membrane proteins. They were enriched in GO terms and KEGG pathways related to junctions, adhesion and binding, which were typical features of viral receptors reported by previous studies <sup>[<a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">10</a>, <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-14-7" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">15</a>, <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">21</a>]</sup>. Besides, our analysis identified several novel features of the viral receptor. Firstly, the viral receptor had a higher level of N-glycosylation than other proteins. Then, what’s the relationship between glycosylation and viral receptor selection? As we know, glycosylation of proteins is widely observed in eukaryote cells <sup>[<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>]</sup>. It plays an important role in multiple cellular activities, such as folding and stability of glycoprotein, immune response, cell-cell adhesion, and so on. Glycans are abundant on host cell surfaces. They were probably the primordial and fallback receptors for the virus <sup>[<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">11</a>]</sup>. To use glycans as their receptors, a large number of viruses have stolen a host galectin and employed it as a viral lectin <sup>[<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>]</sup>. For example, the SJR fold, which was mainly responsible for glycan recognition and binding in cellular proteins, was observed in viral capsid proteins of over one fourth of viruses <sup>[<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">33</a>]</sup>. Thus, during the process of searching for protein receptors, the protein with high level of glycosylation could provide a basal attachment ability for the virus, and should be the preferred receptor for the virus.</p><p id="p-35">Secondly, our analysis showed that the viral receptor protein had a tendency to interact with itself and had far more interaction partners than other membrane proteins. Besides the function of viral receptor, the receptor protein functions in the host cell by interacting with other proteins of the host, such as signal molecules and ligands. Therefore, the virus has to compete with these proteins for binding to the receptor <sup>[<a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">15</a>]</sup>. The protein with less interaction partners are expected to be preferred by the virus. Why did the virus select the proteins with multiple interaction partners as receptors? One possible reason is that the receptor proteins are closely related to the “door” of the cell, so that many proteins have to interact with them for in-and-out of the cell. This could be partly validated by the observation that for the interaction partners of human viral receptors, six of top ten enriched terms in the domain of GO Biological Process were related to protein targeting or localization (Table S3). For entry into the cell, the virus also selects these proteins as receptors. Another possible reason is that viral entry into the cell needs cooperation of multiple proteins which were not identified as viral receptors yet. Besides, previous studies show that the virus could structurally mimic native host ligands <sup>[<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>]</sup>, which help them bind to the host receptor. Thus, membrane proteins with multiple interaction partners have a larger probability to be used by viruses as receptors than other proteins.</p><p id="p-36">Thirdly, the viral receptor was observed to have a much higher level of expression than other genes in each of the 32 human tissues. This may be directly related to the above finding that the viral receptor generally had multiple interaction partners: on the one hand, the viral receptor needs multiple copies to interact with multiple proteins; on the other hand, since the virus has to compete with other proteins for binding to the receptor, high expression of the receptor will facilitate the virus’s binding to the viral receptor.</p><p id="p-37">The virus-receptor interaction is a major determinant of viral host range and tissue tropism. Previous case studies showed that the viral receptor expressed highly in the tissues infected by the virus <sup>[<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>, <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>]</sup>. Consistent with these studies, our systematic analysis found that the tissues with high expression of the viral receptor, and the mammal species with homologs highly similar to the viral receptor, were more possibly to be infected by the virus. However, the opposites were also observed. Some mammal species (or tissues) which had no receptor homolog (or low expression of the viral receptor) were also infected by the virus. These viruses may use other receptors not identified yet. Some mammal species (or tissues) with homologs highly similar to viral receptors (or high expression of the viral receptor) were observed to be not infected by the virus. This may be partly explained by the missed virus-host interactions in our data. Besides, it may also suggest that the host or tissue susceptibility to the virus is not solely determined by the viral receptor. More factors such as the host or tissue accessibility <sup>[<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>]</sup>, the cell defense system <sup>[<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>]</sup> and the complex interaction between viral and host proteins <sup>[<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>]</sup> may also influence viral infections.</p><p id="p-38">There were some limitations within the study. Firstly, the viral receptor was biased towards the human, due to the bias of studies towards human viruses. Fortunately, the viral receptor was conserved in mammal species to a large extent, which may reduce the influence of this bias on the diversity of viral receptors. Secondly, the virus-host interactions were not complete due to limited surveys <sup>[<a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">7</a>]</sup>. According to the risk analysis of viral cross-species based on viral receptors, much more mammal species may be infected by the mammalian virus analyzed in this study. High attention should be paid to this risk. Thirdly, due to the difficulties of identifying viral receptors <sup>[<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>, <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>]</sup> the database of mammalian virus-host receptor interaction was still limited in its size, which hindered us from a more comprehensive survey of the correlation characteristics between viruses and viral receptors. More effective methods, either experimental or computational <sup>[<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a>]</sup>, should be developed for identifying viral receptors, while the characteristics identified in this study may help such endeavors.</p><p id="p-39">Overall, the structural, functional, evolutionary and tissue-specific expression characteristics identified here should not only deepen our understanding of the viral receptor selection, but also help for development of more effective methods for identifying viral receptors. Besides, evaluating the risk of viral cross-species infection based on the viral receptor could also help for early warning and prediction of viral zoonotic diseases.</p></div><div class="section" id="sec-17"><h2 class="">Materials and Methods</h2><div id="sec-18" class="subsection"><h3>Database of mammalian virus-host receptor interaction</h3><p id="p-40">The data of mammalian virus-host receptor interaction were compiled from three sources: firstly, the literature related to viral receptors (a total of 1303 papers) were downloaded from NCBI Pubmed database <sup>[<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>]</sup> by searching “virus receptor” [TIAB] or “viral receptor”[TIAB] on August 14<sup>th</sup>, 2017. The mammalian virus and their related host receptors were manually extracted from the literature; secondly, part of viral receptors were directly obtained from the database of ViralZone <sup>[<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>]</sup> on September 9<sup>th</sup>, 2017; thirdly, proteins annotated with one of GO terms “virus receptor activity”, “viral entry into host cell” and “receptor activity” in UniprotKB database <sup>[<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>]</sup> were collected on August 14<sup>th</sup>, 2017, and manually checked later. In combination, a database was created with 268 pairs of mammalian virus-host receptor interaction, which included 128 unique viral species or sub-species and 119 viral receptors (Table S1).</p></div><div id="sec-19" class="subsection"><h3>Analysis of structural features of viral receptors</h3><p id="p-41">The number of transmembrane alpha helix of the mammalian virus receptor was derived from the database of UniprotKB and the web server TMpred <sup>[<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>]</sup>. The location for the viral receptor was inferred from the description of “Subcellular location” for the receptor protein provided by UniProtKB, or from the GO annotations for them: the viral receptors annotated with GO terms which included the words of “cell surface” or “plasma membrane” were considered to be located in the cell membrane; those annotated with GO terms which included the words of “cytoplasm”, “cytosol” or “cytoplasmic vesicle”, or shown to be in the cytoplasm in UniProtKB, were considered to be located in the cytoplasm; those annotated with GO terms “nucleus” (GO:0005634) or “nucleoplasm” (GO:0005654) were considered to be located in the nuclear. The Pfam family, the N-glycosylation and O-glycosylation sites for the protein were obtained from the database of UniprotKB.</p><p id="p-42">For comparison, the human proteins and their related structural characteristics were obtained from the database of UniProtKB/SwissProt on November 24<sup>th</sup>, 2017. The proteins which had at least one transmembrane alpha helix were considered as membrane proteins. The membrane proteins which were shown to be located in the cell membrane were considered as cell membrane proteins. In total, we obtained 20243 human proteins, 5187 human membrane proteins and 2208 human cell membrane proteins.</p><p id="p-43">The 3D structure for the viral receptor HTR2A were modeled with the help of I-TASSER <sup>[<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>]</sup> based on the protein sequence of HTR2A (accession number in the database of UniProt: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=P28223&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom">P28223</a>). The best model was selected, and visualized in RasMol (version 2.7.5) <sup>[<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>]</sup>.</p></div><div id="sec-20" class="subsection"><h3>Functional enrichment analysis</h3><p id="p-44">The GO function and KEGG pathway enrichment analysis for the human viral receptor were conducted with functions of <em>enrichGO()</em> and <em>enrichKEGG()</em> in the package “clusterProfiler” (version 3.4.4) <sup>[<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>]</sup> in R (version 3.4.2) <sup>[<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>]</sup>.</p></div><div id="sec-21" class="subsection"><h3>Protein-protein interaction (PPI) network analysis</h3><p id="p-45">The human PPI network (PPIN) was constructed based on the work of Menche et al [<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a>]. The degree and betweenness for each protein in the PPIN were calculated with functions of <em>degree()</em> and <em>betweenness()</em> in the package “igraph” (version 1.0.0) <sup>[<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>]</sup> in R (version 3.2.5). The network was displayed with Cytoscape (version 2.6.2) <sup>[<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>]</sup>.</p><p id="p-46">For robustness of the results, we also conducted PPI analysis based on the human PPIs derived from the database of STRING <sup>[<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>]</sup> on November 7, 2017. The human PPIN was built based on the PPIs with median confidence (combined score equal to or greater than 0.4). It included 710,188 PPIs and 17,487 proteins which could be mapped to NCBI gene ids. Similar to those mentioned above, the viral receptor protein was observed to have far more interaction partners and higher betweenness than other proteins in the human PPIN (Figure S3A&amp;C).</p></div><div id="sec-22" class="subsection"><h3>Evolutionary analysis</h3><p id="p-47">To identify the homolog of the mammalian virus receptor in other mammal species, the protein sequence of each viral receptor was searched against the database of mammalian protein sequences, which were downloaded from NCBI RefSeq database on <sup>[<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>]</sup>October 10<sup>th</sup>, 2017, with the help of BLAST (version 2.6.0) <sup>[<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>]</sup>. Analysis showed that in the database of mammalian protein sequences, there were 108 mammal species which were richly annotated and had far more protein sequences than other mammal species (Table S7). Therefore, only these 108 mammal species were considered in the evolutionary analysis. Based on the results of BLAST, the homolog for the viral receptor was defined as the hit with E-value small than 1E-10, coverage equal to or greater than 80% and sequence identity equal to or greater than 30%. Only the closest homolog, i.e., the best hit, in each mammal species was used for further analysis. For measuring the conservation level of viral receptors, two indicators were used. The first indicator was the number of mammal species with homolog of the viral receptor in 108 mammal species. The other indicator was the average of the pairwise sequence identities between the viral receptor and its homologs in 108 mammal species. For comparison, 1000 human protein sequences were randomly selected from the NCBI RefSeq database (Table S4). Similar methods as above were utilized to calculate the indicators of conservation level for these proteins.</p><p id="p-48">For analysis of co-evolution between viral receptors, firstly for each viral receptor, a phylogenetic tree was built based on the protein sequences of the receptor and its homologs in 108 mammal species with the help of phylip (version 3.68) <sup>[<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>]</sup>. The neighbor-joining method was used with the default parameter. Then, the genetic distances between the viral receptor and their homologs were extracted from the phylogenetic tree with a perl script. Finally, for a pair of viral receptors, the spearman correlation coefficient (SCC) was calculated between the pairwise genetic distances of viral receptors and their homologs, which was used to measure the extent of co-evolution between this pair of viral receptors.</p><p id="p-49">The set of housekeeping gene in human was adapted from the work of Eisenberg et al<sup>[<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>]</sup> A total of 3804 genes were identified as the housekeeping gene.</p></div><div id="sec-23" class="subsection"><h3>Analysis of the tissue-specific expression of human viral receptors</h3><p id="p-50">The expression level for human viral receptors and other human genes in 32 human tissues were derived from the database of Expression Atlas <sup>[<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>]</sup>. For analysis of the association between viral infection and tissue-specific expression of viral receptors, we manually compiled the tissue tropism of viruses from the literature or Wikipedia and obtained that in 32 human tissues for a total of 52 viruses which used a total of 46 receptors (Table S5). When comparing the expression level of human viral receptors and other set of human genes in 32 human tissues, to reduce the influence of extreme values, the median instead of the mean of the expression values was used to measure the average expression value of a gene set in a tissue.</p><p id="p-51">The SCCs between the expression values of viral receptors in 32 tissues were calculated to measure the extent of co-expression between viral receptors.</p></div><div id="sec-24" class="subsection"><h3>Analysis of the host specificity of the virus and the viral receptor</h3><p id="p-52">The mammalian virus-host interactions were primarily adapted from Olival’s work <sup>[<a id="xref-ref-7-6" class="xref-bibr" href="#ref-7">7</a>]</sup>. One hundred and fifteen viruses in our database and 61 of 108 richly annotated mammal species could be mapped to those in Olival’s work (Table S8). These 115 viruses used a total of 116 viral receptors. The sequence identities of these viral receptor proteins to their related homologs in the corresponding mammal species were presented in Table S6.</p><p id="p-53">For comparison, we also extracted genetic distances (host relatedness) between the mammal species and the viral host with reported receptors based on Olival’s work (Table S9). Then, the genetic distance of the mammal species to the viral host with reported receptors, and the sequence identity of the receptor homolog in the mammal species to the viral receptor protein, was respectively used to predict whether a mammal species could be infected by the virus which infected the host with reported receptors. The method of Receiver Operating Characteristic (ROC) curve was used to evaluate and compare their performance with the functions of <em>roc(), auc(), roc.test()</em> and <em>plot.roc()</em> in the package of “pROC” <sup>[<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>]</sup> in R (version 3.2.5).</p></div><div id="sec-25" class="subsection"><h3>Statistical analysis</h3><p id="p-54">All the statistical analysis was conducted in R (version 3.2.5) <sup>[<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">50</a>]</sup>. The wilcoxon rank-sum test was conducted with the function of <em>wilcox.test()</em>.</p></div></div><div class="section" id="sec-26"><h2 class="">Author contributions</h2><p id="p-57">HZ, TJ and YP conceived and designed the study. ZZ and ZZZ did the computational analysis. WC, ZC, ZT and BX compiled the database of mammalian viruses and their related receptors, and the tissue tropism of viruses. AW and XYG directed the computational analysis about the tissue and host specificity of viruses. YP and ZZ wrote the paper. AW, XYG, XHG, ZT, ZX, TJ and HZ reviewed and edited the manuscript. All authors read and approved the manuscript.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-55">This work was supported by the National Key Plan for Scientific Research and Development of China (2016YFD0500300 and 2016YFC1200200), the National Natural Science Foundation of China (31500126, 31671371, 81730064 and 81571985), National Science and Technology Major Project (2017ZX10202201) and the Chinese Academy of Medical Sciences (2016-I2M-1-005). The authors would like to thank Pro. Xiangjun Du in Sun Yat-sen University for helpful suggestions.</p><p id="p-56">The authors have declared that no competing interests exist.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">[1].</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference [1] in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.1" data-doi="10.1056/NEJMoa1600651"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mlakar</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Korva</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tul</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Zika virus associated with microcephaly</span>. <abbr class="cit-jnl-abbrev">New Engl J Med</abbr>, <span class="cit-pub-date">2016</span>, <span class="cit-vol">374</span>: <span class="cit-fpage">951</span>–<span class="cit-lpage">958</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngl%2BJ%2BMed%26rft.volume%253D374%26rft.spage%253D951%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1600651%26rft_id%253Dinfo%253Apmid%252F26862926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa1600651&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26862926&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[2].</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference [2] in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.2" data-doi="10.1056/NEJMoa1411099"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maganga</span>  <span class="cit-name-given-names">GD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kapetshi</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berthet</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Ebola virus disease in the democratic republic of congo</span>. <abbr class="cit-jnl-abbrev">New Engl J Med</abbr>, <span class="cit-pub-date">2014</span>, <span class="cit-vol">371</span>: <span class="cit-fpage">2083</span>–<span class="cit-lpage">2091</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngl%2BJ%2BMed%26rft.volume%253D371%26rft.spage%253D2083%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMoa1411099%26rft_id%253Dinfo%253Apmid%252F25317743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJMoa1411099&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25317743&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[3].</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference [3] in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.3" data-doi="10.1016/S0140-6736(13)61492-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Breban</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Riou</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fontanet</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Interhuman transmissibility of middle east respiratory syndrome coronavirus: Estimation of pandemic risk</span>. <abbr class="cit-jnl-abbrev">Lancet</abbr>, <span class="cit-pub-date">2013</span>, <span class="cit-vol">382</span>: <span class="cit-fpage">694</span>–<span class="cit-lpage">699</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.volume%253D382%26rft.spage%253D694%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252813%252961492-0%26rft_id%253Dinfo%253Apmid%252F23831141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(13)61492-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23831141&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000323542400027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[4].</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference [4] in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.4" data-doi="10.1038/nature19094"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Paez-Espino</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eloe-Fadrosh</span>  <span class="cit-name-given-names">EA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pavlopoulos</span>  <span class="cit-name-given-names">GA</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Uncovering earth’s virome</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-pub-date">2016</span>, <span class="cit-vol">536</span>: <span class="cit-fpage">425</span>-+</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D536%26rft.spage%253D425%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature19094%26rft_id%253Dinfo%253Apmid%252F27533034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature19094&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27533034&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[5].</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference [5] in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mihara</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nishimura</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shimizu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Linking virus genomes with host taxonomy</span>. <abbr class="cit-jnl-abbrev">Viruses-Basel</abbr>, <span class="cit-pub-date">2016</span>, <span class="cit-vol">8</span>:</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[6].</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference [6] in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.6" data-doi="10.1073/pnas.1521582113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Geoghegan</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Senior</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Di Giallonardo</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Virological factors that increase the transmissibility of emerging human viruses</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr>, <span class="cit-pub-date">2016</span>, <span class="cit-vol">113</span>: <span class="cit-fpage">4170</span>–<span class="cit-lpage">4175</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1521582113%26rft_id%253Dinfo%253Apmid%252F27001840%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEzLzE1LzQxNzAiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wMy8wOC8yNzExNzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[7].</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference [7] in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.7" data-doi="10.1038/nature22975"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Olival</span>  <span class="cit-name-given-names">KJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hosseini</span>  <span class="cit-name-given-names">PR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zambrana-Torrelio</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Host and viral traits predict zoonotic spillover from mammals</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-pub-date">2017</span>, <span class="cit-vol">546</span>: <span class="cit-fpage">646</span>-+</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D546%26rft.spage%253D646%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature22975%26rft_id%253Dinfo%253Apmid%252F28636590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature22975&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28636590&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[8].</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference [8] in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">PM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">BH</span></span>. <span class="cit-article-title">Origins of hiv and the aids pandemic</span>. <abbr class="cit-jnl-abbrev">Csh Perspect Med</abbr>, <span class="cit-pub-date">2011</span>, <span class="cit-fpage">1</span>:<span class="cit-lpage">2</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[9].</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference [9] in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.9" data-doi="10.1038/nature12711"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ge</span>  <span class="cit-name-given-names">XY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">XL</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Isolation and characterization of a bat sars-like coronavirus that uses the ace2 receptor</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-pub-date">2013</span>, <span class="cit-vol">503</span>: <span class="cit-fpage">535</span>-+</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D503%26rft.spage%253D535%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12711%26rft_id%253Dinfo%253Apmid%252F24172901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24172901&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327464200045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[10].</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference [10] in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.10" data-doi="10.1038/nrmicro817"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dimitrov</span>  <span class="cit-name-given-names">DS</span></span>. <span class="cit-article-title">Virus entry: Molecular mechanisms and biomedical applications</span>. <abbr class="cit-jnl-abbrev">Nat Rev Microbiol</abbr>, <span class="cit-pub-date">2004</span>, <span class="cit-vol">2</span>: <span class="cit-fpage">109</span>–<span class="cit-lpage">122</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DDimitrov%26rft.auinit1%253DD.%2BS.%26rft.volume%253D2%26rft.issue%253D2%26rft.spage%253D109%26rft.epage%253D122%26rft.atitle%253DVirus%2Bentry%253A%2Bmolecular%2Bmechanisms%2Band%2Bbiomedical%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro817%26rft_id%253Dinfo%253Apmid%252F15043007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro817&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15043007&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220431700014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[11].</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference [11] in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.11" data-doi="10.1128/JVI.02615-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">Receptor recognition mechanisms of coronaviruses: A decade of structural studies</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr>, <span class="cit-pub-date">2015</span>, <span class="cit-vol">89</span>: <span class="cit-fpage">1954</span>–<span class="cit-lpage">1964</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.02615-14%26rft_id%253Dinfo%253Apmid%252F25428871%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg5LzQvMTk1NCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE4LzAzLzA4LzI3MTE3MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[12].</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference [12] in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.12" data-doi="10.1126/science.1058613"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Baranowski</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ruiz-Jarabo</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Domingo</span>  <span class="cit-name-given-names">E.</span></span> <span class="cit-article-title">Evolution of cell recognition by viruses</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-pub-date">2001</span>, <span class="cit-vol">292</span>: <span class="cit-fpage">1102</span>–<span class="cit-lpage">1105</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBaranowski%26rft.auinit1%253DE.%26rft.volume%253D292%26rft.issue%253D5519%26rft.spage%253D1102%26rft.epage%253D1105%26rft.atitle%253DEvolution%2Bof%2BCell%2BRecognition%2Bby%2BViruses%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1058613%26rft_id%253Dinfo%253Apmid%252F11352064%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTIvNTUxOS8xMTAyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDMvMDgvMjcxMTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[13].</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference [13] in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.13" data-doi="10.1083/jcb.201108131"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grove</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marsh</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">The cell biology of receptor-mediated virus entry</span>. <abbr class="cit-jnl-abbrev">The Journal of cell biology</abbr>, <span class="cit-pub-date">2011</span>, <span class="cit-vol">195</span>: <span class="cit-fpage">1071</span>–<span class="cit-lpage">1082</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bcell%2Bbiology%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.201108131%26rft_id%253Dinfo%253Apmid%252F22123832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjEwOiIxOTUvNy8xMDcxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDMvMDgvMjcxMTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[14].</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference [14] in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Casasnovas</span>  <span class="cit-name-given-names">JM</span></span>. <abbr class="cit-jnl-abbrev">Virus-receptor interactions and receptor-mediated virus entry into host cells</abbr>. <span class="cit-pub-date">2013</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[15].</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference [15] in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.15" data-doi="10.1016/S0968-0004(01)02038-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">JH</span></span>. <span class="cit-article-title">Protein recognition by cell surface receptors: Physiological receptors versus virus interactions</span>. <abbr class="cit-jnl-abbrev">Trends in biochemical sciences</abbr>, <span class="cit-pub-date">2002</span>, <span class="cit-vol">27</span>: <span class="cit-fpage">122</span>–<span class="cit-lpage">126</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bbiochemical%2Bsciences%26rft.stitle%253DTrends%2BBiochem%2BSci%26rft.aulast%253DWang%26rft.auinit1%253DJ.%26rft.volume%253D27%26rft.issue%253D3%26rft.spage%253D122%26rft.epage%253D126%26rft.atitle%253DProtein%2Brecognition%2Bby%2Bcell%2Bsurface%2Breceptors%253A%2Bphysiological%2Breceptors%2Bversus%2Bvirus%2Binteractions.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0968-0004%252801%252902038-2%26rft_id%253Dinfo%253Apmid%252F11893508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0968-0004(01)02038-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11893508&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174416700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[16].</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference [16] in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.16" data-doi="10.1146/annurev-virology-110615-042301"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">Structure, function, and evolution of coronavirus spike proteins</span>. <abbr class="cit-jnl-abbrev">Annu Rev Virol</abbr>, <span class="cit-pub-date">2016</span>, <span class="cit-vol">3</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">261</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu%2BRev%2BVirol%26rft.volume%253D3%26rft.spage%253D237%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-virology-110615-042301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-virology-110615-042301&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[17].</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference [17] in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.17" data-doi="10.7554/eLife.00049"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Sodium taurocholate cotransporting polypeptide is a functional receptor for human hepatitis b and d virus</span>. <abbr class="cit-jnl-abbrev">eLife</abbr>, <span class="cit-pub-date">2012</span>, <span class="cit-vol">1</span>: <span class="cit-fpage">e00049</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.00049%26rft_id%253Dinfo%253Apmid%252F23150796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.00049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23150796&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[18].</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference [18] in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.18" data-doi="10.1016/j.chom.2016.11.004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">WJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Vries</span>  <span class="cit-name-given-names">RP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grant</span>  <span class="cit-name-given-names">OC</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Recent h3n2 viruses have evolved specificity for extended, branched human-type receptors, conferring potential for increased avidity</span>. <abbr class="cit-jnl-abbrev">Cell host &amp; microbe</abbr>, <span class="cit-pub-date">2017</span>, <span class="cit-vol">21</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">34</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Bhost%2B%2526%2Bmicrobe%26rft.volume%253D21%26rft.spage%253D23%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2016.11.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2016.11.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[19].</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference [19] in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.19" data-doi="10.1007/s10719-006-5435-y"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Isa</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arias</span>  <span class="cit-name-given-names">CF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lopez</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Role of sialic acids in rotavirus infection</span>. <abbr class="cit-jnl-abbrev">Glycoconjugate journal</abbr>, <span class="cit-pub-date">2006</span>, <span class="cit-vol">23</span>: <span class="cit-fpage">27</span>–<span class="cit-lpage">37</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlycoconjugate%2Bjournal%26rft.stitle%253DGlycoconj%2BJ%26rft.aulast%253DIsa%26rft.auinit1%253DP.%26rft.volume%253D23%26rft.issue%253D1-2%26rft.spage%253D27%26rft.epage%253D37%26rft.atitle%253DRole%2Bof%2Bsialic%2Bacids%2Bin%2Brotavirus%2Binfection.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10719-006-5435-y%26rft_id%253Dinfo%253Apmid%252F16575520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10719-006-5435-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16575520&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[20].</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference [20] in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.20" data-doi="10.1111/cmi.12340"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mazzon</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mercer</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Lipid interactions during virus entry and infection</span>. <abbr class="cit-jnl-abbrev">Cellular microbiology</abbr>, <span class="cit-pub-date">2014</span>, <span class="cit-vol">16</span>: <span class="cit-fpage">1493</span>–<span class="cit-lpage">1502</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCellular%2Bmicrobiology%26rft.volume%253D16%26rft.spage%253D1493%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fcmi.12340%26rft_id%253Dinfo%253Apmid%252F25131438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/cmi.12340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25131438&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[21].</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference [21] in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.21"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Marija Backovic FAR</span>. <span class="cit-article-title">Virus entry: Old viruses, new receptors</span>. <abbr class="cit-jnl-abbrev">Current opinion in virology</abbr>, <span class="cit-pub-date">2012</span>, <span class="cit-vol">2</span>: <span class="cit-fpage">10</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bvirology%26rft.volume%253D2%26rft.spage%253D10%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[22].</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference [22] in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Coffin</span>  <span class="cit-name-given-names">JM.</span></span> <span class="cit-article-title">Virions at the gates: Receptors and the host-virus arms race</span>. <abbr class="cit-jnl-abbrev">PLoS Biology</abbr>, <span class="cit-pub-date">2013</span>, <span class="cit-vol">11</span>:</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[23].</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference [23] in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kumlin</span>  <span class="cit-name-given-names">U</span></span>, <span class="cit-auth"><span class="cit-name-surname">Olofsson</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dimock</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Sialic acid tissue distribution and influenza virus tropism</span>. <abbr class="cit-jnl-abbrev">Influenza and other respiratory viruses</abbr>, <span class="cit-pub-date">2008</span>, <span class="cit-vol">2</span>: <span class="cit-fpage">147</span>–<span class="cit-lpage">154</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfluenza%2Band%2Bother%2Brespiratory%2Bviruses%26rft.volume%253D2%26rft.spage%253D147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[24].</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference [24] in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.24" data-doi="10.1074/jbc.M110.104836"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">HJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mullins</span>  <span class="cit-name-given-names">JG</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Claudin association with cd81 defines hepatitis c virus entry</span>. <abbr class="cit-jnl-abbrev">The Journal of biological chemistry</abbr>, <span class="cit-pub-date">2010</span>, <span class="cit-vol">285</span>: <span class="cit-fpage">21092</span>–<span class="cit-lpage">21102</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bbiological%2Bchemistry%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M110.104836%26rft_id%253Dinfo%253Apmid%252F20375010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODUvMjcvMjEwOTIiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wMy8wOC8yNzExNzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[25].</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference [25] in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.25" data-doi="10.1128/JVI.80.2.802-809.2006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ribeiro</span>  <span class="cit-name-given-names">RM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hazenberg</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Perelson</span>  <span class="cit-name-given-names">AS</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Naive and memory cell turnover as drivers of ccr5-to-cxcr4 tropism switch in human immunodeficiency virus type 1: Implications for therapy</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr>, <span class="cit-pub-date">2006</span>, <span class="cit-vol">80</span>: <span class="cit-fpage">802</span>–<span class="cit-lpage">809</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DRibeiro%26rft.auinit1%253DR.%2BM.%26rft.volume%253D80%26rft.issue%253D2%26rft.spage%253D802%26rft.epage%253D809%26rft.atitle%253DNaive%2Band%2BMemory%2BCell%2BTurnover%2Bas%2BDrivers%2Bof%2BCCR5-to-CXCR4%2BTropism%2BSwitch%2Bin%2BHuman%2BImmunodeficiency%2BVirus%2BType%2B1%253A%2BImplications%2Bfor%2BTherapy%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.80.2.802-809.2006%26rft_id%253Dinfo%253Apmid%252F16378982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjg6IjgwLzIvODAyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDMvMDgvMjcxMTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[26].</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference [26] in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.26" data-doi="10.1016/j.chom.2011.02.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">CC</span></span>, <span class="cit-auth"><span class="cit-name-surname">McNamara</span>  <span class="cit-name-given-names">LA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Onafuwa-Nuga</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Hiv-1 utilizes the cxcr4 chemokine receptor to infect multipotent hematopoietic stem and progenitor cells</span>. <abbr class="cit-jnl-abbrev">Cell host &amp; microbe</abbr>, <span class="cit-pub-date">2011</span>, <span class="cit-vol">9</span>: <span class="cit-fpage">223</span>–<span class="cit-lpage">234</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCell%2BHost%2BMicrobe%26rft.aulast%253DCarter%26rft.auinit1%253DC.%2BC.%26rft.volume%253D9%26rft.issue%253D3%26rft.spage%253D223%26rft.epage%253D234%26rft.atitle%253DHIV-1%2Butilizes%2Bthe%2BCXCR4%2Bchemokine%2Breceptor%2Bto%2Binfect%2Bmultipotent%2Bhematopoietic%2Bstem%2Band%2Bprogenitor%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2011.02.005%26rft_id%253Dinfo%253Apmid%252F21402361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2011.02.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21402361&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288826600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[27].</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference [27] in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.27" data-doi="10.1016/j.chom.2010.05.009"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Taubenberger</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kash</span>  <span class="cit-name-given-names">JC</span></span>. <span class="cit-article-title">Influenza virus evolution, host adaptation, and pandemic formation</span>. <abbr class="cit-jnl-abbrev">Cell host &amp; microbe</abbr>, <span class="cit-pub-date">2010</span>, <span class="cit-vol">7</span>: <span class="cit-fpage">440</span>–<span class="cit-lpage">451</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCell%2BHost%2BMicrobe%26rft.aulast%253DTaubenberger%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253D440%26rft.epage%253D451%26rft.atitle%253DInfluenza%2Bvirus%2Bevolution%252C%2Bhost%2Badaptation%252C%2Band%2Bpandemic%2Bformation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2010.05.009%26rft_id%253Dinfo%253Apmid%252F20542248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2010.05.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20542248&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279209200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[28].</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference [28] in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.28" data-doi="10.1016/j.tim.2015.06.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">GF</span></span>. <span class="cit-article-title">Bat-to-human: Spike features determining ‘host jump’of coronaviruses sars-cov, mers-cov, and beyond</span>. <abbr class="cit-jnl-abbrev">Trends in microbiology</abbr>, <span class="cit-pub-date">2015</span>, <span class="cit-vol">23</span>: <span class="cit-fpage">468</span>–<span class="cit-lpage">478</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bmicrobiology%26rft.volume%253D23%26rft.spage%253D468%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2015.06.003%26rft_id%253Dinfo%253Apmid%252F26206723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2015.06.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26206723&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[29].</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference [29] in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.29" data-doi="10.1016/j.antiviral.2013.08.014"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">Receptor recognition and cross-species infections of sars coronavirus</span>. <abbr class="cit-jnl-abbrev">Antiviral research</abbr>, <span class="cit-pub-date">2013</span>, <span class="cit-vol">100</span>: <span class="cit-fpage">246</span>–<span class="cit-lpage">254</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntiviral%2Bresearch%26rft.volume%253D100%26rft.spage%253D246%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.antiviral.2013.08.014%26rft_id%253Dinfo%253Apmid%252F23994189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.antiviral.2013.08.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23994189&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000328179300027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[30].</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference [30] in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.30" data-doi="10.1128/JVI.00978-10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maginnis</span>  <span class="cit-name-given-names">MS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">SA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gee</span>  <span class="cit-name-given-names">GV</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Role of n-linked glycosylation of the 5-ht2a receptor in jc virus infection</span>. <abbr class="cit-jnl-abbrev">Journal of virology</abbr>, <span class="cit-pub-date">2010</span>, <span class="cit-vol">84</span>: <span class="cit-fpage">9677</span>–<span class="cit-lpage">9684</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bvirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00978-10%26rft_id%253Dinfo%253Apmid%252F20660194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4NC8xOS85Njc3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDMvMDgvMjcxMTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[31].</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference [31] in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Menche</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kitsak</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Uncovering disease-disease relationships through the incomplete interactome</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-pub-date">2015</span>, <span class="cit-fpage">347</span>:</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[32].</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference [32] in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.32" data-doi="10.1007/s00418-016-1526-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Corfield</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Eukaryotic protein glycosylation: A primer for histochemists and cell biologists</span>. <abbr class="cit-jnl-abbrev">Histochemistry and cell biology</abbr>, <span class="cit-pub-date">2017</span>, <span class="cit-vol">147</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">147</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHistochemistry%2Band%2Bcell%2Bbiology%26rft.volume%253D147%26rft.spage%253D119%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00418-016-1526-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00418-016-1526-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[33].</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference [33] in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.33" data-doi="10.1073/pnas.1621061114"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Krupovic</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">EV</span></span>. <span class="cit-article-title">Multiple origins of viral capsid proteins from cellular ancestors</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr>, <span class="cit-pub-date">2017</span>, <span class="cit-vol">114</span>: <span class="cit-fpage">E2401</span>–<span class="cit-lpage">E2410</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1621061114%26rft_id%253Dinfo%253Apmid%252F28265094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE0LzEyL0UyNDAxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDMvMDgvMjcxMTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[34].</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference [34] in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.34" data-doi="10.1016/j.chom.2013.05.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drayman</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Glick</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ben-nun-shaul</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Pathogens use structural mimicry of native host ligands as a mechanism for host receptor engagement</span>. <abbr class="cit-jnl-abbrev">Cell host &amp; microbe</abbr>, <span class="cit-pub-date">2013</span>, <span class="cit-vol">14</span>: <span class="cit-fpage">63</span>–<span class="cit-lpage">73</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Bhost%2B%2526%2Bmicrobe%26rft.volume%253D14%26rft.spage%253D63%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2013.05.005%26rft_id%253Dinfo%253Apmid%252F23870314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2013.05.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23870314&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[35].</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference [35] in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Boonarkart</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Champunot</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Uiprasertkul</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Case report: Increased viral receptor expression associated with high viral load and severe pneumonia in a young patient infected with 2009 h1n1 influenza a with no pre-existing conditions</span>. <abbr class="cit-jnl-abbrev">Journal of medical virology</abbr>, <span class="cit-pub-date">2012</span>, <span class="cit-vol">84</span>: <span class="cit-fpage">380</span>–<span class="cit-lpage">385</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmedical%2Bvirology%26rft.stitle%253DJ%2BMed%2BVirol%26rft.aulast%253DBoonarkart%26rft.auinit1%253DC.%26rft.volume%253D84%26rft.issue%253D3%26rft.spage%253D380%26rft.epage%253D385%26rft.atitle%253DCase%2Breport%253A%2BIncreased%2Bviral%2Breceptor%2Bexpression%2Bassociated%2Bwith%2Bhigh%2Bviral%2Bload%2Band%2Bsevere%2Bpneumonia%2Bin%2Ba%2Byoung%2Bpatient%2Binfected%2Bwith%2B2009%2BH1N1%2Binfluenza%2Ba%2Bwith%2Bno%2Bpre-existing%2Bconditions.%26rft_id%253Dinfo%253Apmid%252F22246822%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22246822&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[36].</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference [36] in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.36" data-doi="10.1016/j.stem.2016.03.012"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nowakowski</span>  <span class="cit-name-given-names">TJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pollen</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Di Lullo</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Expression analysis highlights axl as a candidate zika virus entry receptor in neural stem cells</span>. <abbr class="cit-jnl-abbrev">Cell stem cell</abbr>, <span class="cit-pub-date">2016</span>, <span class="cit-vol">18</span>: <span class="cit-fpage">591</span>–<span class="cit-lpage">596</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Bstem%2Bcell%26rft.volume%253D18%26rft.spage%253D591%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.stem.2016.03.012%26rft_id%253Dinfo%253Apmid%252F27038591%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.stem.2016.03.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27038591&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[37].</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference [37] in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.37" data-doi="10.1038/nri3787"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McNab</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mayer-Barber</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sher</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Type i interferons in infectious disease</span>. <abbr class="cit-jnl-abbrev">Nature reviews Immunology</abbr>, <span class="cit-pub-date">2015</span>, <span class="cit-vol">15</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">103</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews%2BImmunology%26rft.volume%253D15%26rft.spage%253D87%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnri3787%26rft_id%253Dinfo%253Apmid%252F25614319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nri3787&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25614319&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[38].</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference [38] in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.38" data-doi="10.1146/annurev-immunol-032712-100001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jost</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Altfeld</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Control of human viral infections by natural killer cells</span>. <abbr class="cit-jnl-abbrev">Annual review of immunology</abbr>, <span class="cit-pub-date">2013</span>, <span class="cit-vol">31</span>: <span class="cit-fpage">163</span>–<span class="cit-lpage">194</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bimmunology%26rft.volume%253D31%26rft.spage%253D163%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-immunol-032712-100001%26rft_id%253Dinfo%253Apmid%252F23298212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-immunol-032712-100001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23298212&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321861600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[39].</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference [39] in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.39" data-doi="10.1073/pnas.0704894104"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Randall</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Panis</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">JD</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Cellular cofactors affecting hepatitis c virus infection and replication</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr>, <span class="cit-pub-date">2007</span>, <span class="cit-vol">104</span>: <span class="cit-fpage">12884</span>–<span class="cit-lpage">12889</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0704894104%26rft_id%253Dinfo%253Apmid%252F17616579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzMxLzEyODg0IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDMvMDgvMjcxMTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[40].</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference [40] in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.40" data-doi="10.1016/j.cell.2008.07.032"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Konig</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elleder</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Global analysis of host-pathogen interactions that regulate early-stage hiv-1 replication</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>, <span class="cit-pub-date">2008</span>, <span class="cit-vol">135</span>: <span class="cit-fpage">49</span>–<span class="cit-lpage">60</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DKonig%26rft.auinit1%253DR.%26rft.volume%253D135%26rft.issue%253D1%26rft.spage%253D49%26rft.epage%253D60%26rft.atitle%253DGlobal%2Banalysis%2Bof%2Bhost-pathogen%2Binteractions%2Bthat%2Bregulate%2Bearly-stage%2BHIV-1%2Breplication.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.07.032%26rft_id%253Dinfo%253Apmid%252F18854154%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.07.032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18854154&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259728100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[41].</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference [41] in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.41" data-doi="10.1146/annurev-cellbio-100814-125241"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span> <span class="cit-article-title">The hepatitis b virus receptor</span>. <abbr class="cit-jnl-abbrev">Annual review of cell and developmental biology</abbr>, <span class="cit-pub-date">2015</span>, <span class="cit-vol">31</span>: <span class="cit-fpage">125</span>–<span class="cit-lpage">147</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bcell%2Band%2Bdevelopmental%2Bbiology%26rft.volume%253D31%26rft.spage%253D125%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-cellbio-100814-125241%26rft_id%253Dinfo%253Apmid%252F26436705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-cellbio-100814-125241&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26436705&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[42].</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference [42] in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.42" data-doi="10.1038/nature16465"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pillay</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">NL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Puschnik</span>  <span class="cit-name-given-names">AS</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">An essential receptor for adeno-associated virus infection</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-pub-date">2016</span>, <span class="cit-vol">530</span>: <span class="cit-fpage">108</span>–<span class="cit-lpage">112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D530%26rft.spage%253D108%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature16465%26rft_id%253Dinfo%253Apmid%252F26814968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature16465&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26814968&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[43].</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference [43] in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.43" data-doi="10.1093/nar/gkv1290"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Agarwala</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beck</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Database resources of the national center for biotechnology information</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-pub-date">2016</span>, <span class="cit-vol">44</span>: <span class="cit-fpage">D7</span>–<span class="cit-lpage">D19</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1290%26rft_id%253Dinfo%253Apmid%252F26615191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1290&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26615191&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[44].</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference [44] in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.44" data-doi="10.1093/nar/gks1220"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Masson</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hulo</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">De Castro</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Viralzone: Recent updates to the virus knowledge resource</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-pub-date">2013</span>, <span class="cit-vol">41</span>: <span class="cit-fpage">D579</span>–<span class="cit-lpage">D583</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks1220%26rft_id%253Dinfo%253Apmid%252F23193299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks1220&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23193299&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312893300082&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[45].</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference [45] in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.45" data-doi="10.1093/nar/gkw1099"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bateman</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Donovan</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Uniprot: The universal protein knowledgebase</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-pub-date">2017</span>, <span class="cit-vol">45</span>: <span class="cit-fpage">D158</span>–<span class="cit-lpage">D169</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw1099%26rft_id%253Dinfo%253Apmid%252F27899622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw1099&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27899622&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[46].</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference [46] in text" id="ref-46">↵</a><div class="cit ref-cit ref-web" id="cit-271171v2.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hofmann</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stoffel</span>  <span class="cit-name-given-names">W. j</span></span> <span class="cit-source">Tmpred: Prediction of transmembrane regions and orientation</span>. <span class="cit-pub-date">2017</span>, <a href="https://embnet.vital-it.ch/software/TMPRED_form.html">https://embnet.vital-it.ch/software/TMPRED_form.html</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[47].</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference [47] in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.47" data-doi="10.1038/nprot.2010.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Roy</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kucukural</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span> <span class="cit-article-title">I-tasser: A unified platform for automated protein structure and function prediction</span>. <abbr class="cit-jnl-abbrev">Nat Protoc</abbr>, <span class="cit-pub-date">2010</span>, <span class="cit-vol">5</span>: <span class="cit-fpage">725</span>–<span class="cit-lpage">738</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DRoy%26rft.auinit1%253DA.%26rft.volume%253D5%26rft.issue%253D4%26rft.spage%253D725%26rft.epage%253D738%26rft.atitle%253DI-TASSER%253A%2Ba%2Bunified%2Bplatform%2Bfor%2Bautomated%2Bprotein%2Bstructure%2Band%2Bfunction%2Bprediction.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2010.5%26rft_id%253Dinfo%253Apmid%252F20360767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2010.5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20360767&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276196500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[48].</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference [48] in text" id="ref-48">↵</a><div class="cit ref-cit ref-web" id="cit-271171v2.48"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bernstein</span>  <span class="cit-name-given-names">HJ</span></span>. <span class="cit-source">Rasmol 2.7.5</span>. <span class="cit-pub-date">2017</span>, <a href="http://www.openrasmol.org/">http://www.openrasmol.org/</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[49].</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference [49] in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.49" data-doi="10.1089/omi.2011.0118"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">GC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">LG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">YY</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Clusterprofiler: An r package for comparing biological themes among gene clusters</span>. <abbr class="cit-jnl-abbrev">Omics</abbr>, <span class="cit-pub-date">2012</span>, <span class="cit-vol">16</span>: <span class="cit-fpage">284</span>–<span class="cit-lpage">287</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOmics%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bintegrative%2Bbiology%26rft.stitle%253DOMICS%26rft.aulast%253DYu%26rft.auinit1%253DG.%26rft.volume%253D16%26rft.issue%253D5%26rft.spage%253D284%26rft.epage%253D287%26rft.atitle%253DclusterProfiler%253A%2Ban%2BR%2Bpackage%2Bfor%2Bcomparing%2Bbiological%2Bthemes%2Bamong%2Bgene%2Bclusters.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fomi.2011.0118%26rft_id%253Dinfo%253Apmid%252F22455463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/omi.2011.0118&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22455463&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303653300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[50].</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference [50] in text" id="ref-50">↵</a><div class="cit ref-cit ref-web" id="cit-271171v2.50"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Team RC</span>. <span class="cit-article-title">R: A language and environment for statistical computing</span>. <span class="cit-source">R foundation for statistical computing</span>, <span class="cit-publ-name">vienna</span>, <span class="cit-publ-loc">austria</span>. <span class="cit-pub-date">2016</span>, <a href="http://www.R-project.org/">http://www.R-project.org/</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[51].</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference [51] in text" id="ref-51">↵</a><div class="cit ref-cit ref-web" id="cit-271171v2.51"><div class="cit-metadata"><cite>G C, T N. <span class="cit-source">The igraph software package for complex network research, interjournal, complex systems</span> <span class="cit-fpage">1695</span>. <span class="cit-pub-date">2006</span>, <a href="http://igraph.org">http://igraph.org</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[52].</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference [52] in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.52" data-doi="10.1101/gr.1239303"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shannon</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Markiel</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ozier</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Cytoscape: A software environment for integrated models of biomolecular interaction networks</span>. <abbr class="cit-jnl-abbrev">Genome research</abbr>, <span class="cit-pub-date">2003</span>, <span class="cit-vol">13</span>: <span class="cit-fpage">2498</span>–<span class="cit-lpage">2504</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DShannon%26rft.auinit1%253DP.%26rft.volume%253D13%26rft.issue%253D11%26rft.spage%253D2498%26rft.epage%253D2504%26rft.atitle%253DCytoscape%253A%2BA%2BSoftware%2BEnvironment%2Bfor%2BIntegrated%2BModels%2Bof%2BBiomolecular%2BInteraction%2BNetworks%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1239303%26rft_id%253Dinfo%253Apmid%252F14597658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMy8xMS8yNDk4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDMvMDgvMjcxMTcxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[53].</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference [53] in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.53" data-doi="10.1093/nar/gku1003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Szklarczyk</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Franceschini</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wyder</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">String v10: Protein-protein interaction networks, integrated over the tree of life</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-pub-date">2015</span>, <span class="cit-vol">43</span>: <span class="cit-fpage">D447</span>–<span class="cit-lpage">D452</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1003%26rft_id%253Dinfo%253Apmid%252F25352553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku1003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25352553&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[54].</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference [54] in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.54" data-doi="10.1093/nar/gkr1079"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pruitt</span>  <span class="cit-name-given-names">KD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tatusova</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">GR</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Ncbi reference sequences (refseq): Current status, new features and genome annotation policy</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-pub-date">2012</span>, <span class="cit-vol">40</span>: <span class="cit-fpage">D130</span>–<span class="cit-lpage">D135</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr1079%26rft_id%253Dinfo%253Apmid%252F22121212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr1079&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22121212&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298601300020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[55].</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference [55] in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.55" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">SF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Basic local alignment search tool</span>. <abbr class="cit-jnl-abbrev">Journal of molecular biology</abbr>, <span class="cit-pub-date">1990</span>, <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[56].</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference [56] in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.56" data-doi="10.1111/j.1096-0031.1989.tb00480.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Phylip - phylogeny inference package (version 3.2)</span>. <abbr class="cit-jnl-abbrev">Cladistics</abbr>, <span class="cit-pub-date">1989</span>, <span class="cit-vol">5</span>: <span class="cit-fpage">3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCladistics%26rft.volume%253D5%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1096-0031.1989.tb00480.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1096-0031.1989.tb00480.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[57].</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference [57] in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.57" data-doi="10.1016/j.tig.2013.05.010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eisenberg</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Levanon</span>  <span class="cit-name-given-names">EY</span></span>. <span class="cit-article-title">Human housekeeping genes, revisited</span>. <abbr class="cit-jnl-abbrev">Trends Genet</abbr>, <span class="cit-pub-date">2013</span>, <span class="cit-vol">29</span>: <span class="cit-fpage">569</span>–<span class="cit-lpage">574</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet%26rft.volume%253D29%26rft.spage%253D569%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2013.05.010%26rft_id%253Dinfo%253Apmid%252F23810203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2013.05.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23810203&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000325300800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[58].</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference [58] in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.58" data-doi="10.1093/nar/gkv1045"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Petryszak</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keays</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">YA</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Expression atlas update—an integrated database of gene and protein expression in humans, animals and plants</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>, <span class="cit-pub-date">2016</span>, <span class="cit-vol">44</span>: <span class="cit-fpage">D746</span>–<span class="cit-lpage">D752</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1045%26rft_id%253Dinfo%253Apmid%252F26481351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26481351&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F03%2F08%2F271171.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[59].</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference [59] in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-271171v2.59"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Robin</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Turck</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hainard</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Proc: An open-source package for r and s plus to analyze and compare roc curves</span>. <abbr class="cit-jnl-abbrev">Bmc Bioinformatics</abbr>, <span class="cit-pub-date">2011</span>, <span class="cit-fpage">12</span>:</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/271171v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F271171v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/278267v2" title="Finding Nemo’s Genes: A chromosome-scale reference assembly of the genome of the orange clownfish Amphiprion percula" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/278804v1" title="An expression atlas of chemosensory ionotropic glutamate receptors identifies a molecular basis of carbonation detection" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 08, 2018.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/271171v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/271171v2.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/271171v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/271171v2" class="active" data-icon-position="" data-hide-link-title="0">Membrane proteins with high N-glycosylation, high expression, and multiple interaction partners were preferred by mammalian viruses as receptors</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/85500" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-KbCD_Zd3nu6pkE7sG90n_utoq9T6A1V5dWZXR5Fw5Bs" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="85500" id="node85500--2" data-pisa="biorxiv;271171v2" data-pisa-master="biorxiv;271171" data-seqnum="85500" data-apath="/biorxiv/early/2018/03/08/271171.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Membrane proteins with high N-glycosylation, high expression, and multiple interaction partners were preferred by mammalian viruses as receptors</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Zheng</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Zhaozhong</span> <span class="nlm-surname">Zhu</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Wenjun</span> <span class="nlm-surname">Chen</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Zena</span> <span class="nlm-surname">Cai</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Beibei</span> <span class="nlm-surname">Xu</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Zhiying</span> <span class="nlm-surname">Tan</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Aiping</span> <span class="nlm-surname">Wu</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Xingyi</span> <span class="nlm-surname">Ge</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Xinhong</span> <span class="nlm-surname">Guo</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Zhongyang</span> <span class="nlm-surname">Tan</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Zanxian</span> <span class="nlm-surname">Xia</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Haizhen</span> <span class="nlm-surname">Zhu</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Taijiao</span> <span class="nlm-surname">Jiang</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Yousong</span> <span class="nlm-surname">Peng</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">271171; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/271171 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/271171v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/271171v2&amp;title=Membrane%20proteins%20with%20high%20N-glycosylation%2C%20high%20expression%2C%20and%20multiple%20interaction%20partners%20were%20preferred%20by%20mammalian%20viruses%20as%20receptors" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/271171v2&amp;title=Membrane%20proteins%20with%20high%20N-glycosylation%2C%20high%20expression%2C%20and%20multiple%20interaction%20partners%20were%20preferred%20by%20mammalian%20viruses%20as%20receptors" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/271171v2&amp;text=Membrane%20proteins%20with%20high%20N-glycosylation%2C%20high%20expression%2C%20and%20multiple%20interaction%20partners%20were%20preferred%20by%20mammalian%20viruses%20as%20receptors" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/271171v2&amp;title=Membrane%20proteins%20with%20high%20N-glycosylation%2C%20high%20expression%2C%20and%20multiple%20interaction%20partners%20were%20preferred%20by%20mammalian%20viruses%20as%20receptors" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/271171v2&amp;t=Membrane%20proteins%20with%20high%20N-glycosylation%2C%20high%20expression%2C%20and%20multiple%20interaction%20partners%20were%20preferred%20by%20mammalian%20viruses%20as%20receptors" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/271171v2&amp;title=Membrane%20proteins%20with%20high%20N-glycosylation%2C%20high%20expression%2C%20and%20multiple%20interaction%20partners%20were%20preferred%20by%20mammalian%20viruses%20as%20receptors" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/271171v2&amp;title=Membrane%20proteins%20with%20high%20N-glycosylation%2C%20high%20expression%2C%20and%20multiple%20interaction%20partners%20were%20preferred%20by%20mammalian%20viruses%20as%20receptors" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="85500" id="node85500--3" data-pisa="biorxiv;271171v2" data-pisa-master="biorxiv;271171" data-seqnum="85500" data-apath="/biorxiv/early/2018/03/08/271171.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Membrane proteins with high N-glycosylation, high expression, and multiple interaction partners were preferred by mammalian viruses as receptors</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Zheng</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Zhaozhong</span> <span class="nlm-surname">Zhu</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Wenjun</span> <span class="nlm-surname">Chen</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Zena</span> <span class="nlm-surname">Cai</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Beibei</span> <span class="nlm-surname">Xu</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Zhiying</span> <span class="nlm-surname">Tan</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Aiping</span> <span class="nlm-surname">Wu</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Xingyi</span> <span class="nlm-surname">Ge</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Xinhong</span> <span class="nlm-surname">Guo</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Zhongyang</span> <span class="nlm-surname">Tan</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Zanxian</span> <span class="nlm-surname">Xia</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Haizhen</span> <span class="nlm-surname">Zhu</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Taijiao</span> <span class="nlm-surname">Jiang</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Yousong</span> <span class="nlm-surname">Peng</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">271171; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/271171 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Membrane%20proteins%20with%20high%20N-glycosylation%2C%20high%20expression%2C%20and%20multiple%20interaction%20partners%20were%20preferred%20by%20mammalian%20viruses%20as%20receptors&amp;rft.volume&amp;rft.issue&amp;rft.date=2018-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Zhang%2C+Zheng&amp;rft.au=Zhu%2C+Zhaozhong&amp;rft.au=Chen%2C+Wenjun&amp;rft.au=Cai%2C+Zena&amp;rft.au=Xu%2C+Beibei&amp;rft.au=Tan%2C+Zhiying&amp;rft.au=Wu%2C+Aiping&amp;rft.au=Ge%2C+Xingyi&amp;rft.au=Guo%2C+Xinhong&amp;rft.au=Tan%2C+Zhongyang&amp;rft.au=Xia%2C+Zanxian&amp;rft.au=Zhu%2C+Haizhen&amp;rft.au=Jiang%2C+Taijiao&amp;rft.au=Peng%2C+Yousong"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/85500/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/85500/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/85500/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/85500/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/85500/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/85500/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/85500/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/85500/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/85500/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/85500/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/85500/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/85500/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/271171v2&amp;count=horizontal&amp;via=&amp;text=Membrane%20proteins%20with%20high%20N-glycosylation%2C%20high%20expression%2C%20and%20multiple%20interaction%20partners%20were%20preferred%20by%20mammalian%20viruses%20as%20receptors&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/271171v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/271171v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/271171v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/microbiology" class="highlight" data-icon-position="" data-hide-link-title="0">Microbiology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
