<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/04/09/2020.04.07.030551.1.atom" />
<meta name="HW.pisa" content="biorxiv;2020.04.07.030551v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="IDseq – An Open Source Cloud-based Pipeline and Analysis Service for Metagenomic Pathogen Detection and Monitoring" />
<meta name="DC.Identifier" content="10.1101/2020.04.07.030551" />
<meta name="DC.Date" content="2020-04-09" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Background Metagenomic next generation sequencing (mNGS) has enabled the rapid, unbiased detection and identification of microbes without pathogen-specific reagents, culturing, or a priori knowledge of the microbial landscape. mNGS data analysis requires a series of computationally intensive processing steps to accurately determine the microbial composition of a sample. Existing mNGS data analysis tools typically require bioinformatics expertise and access to local server-class hardware resources. For many research laboratories, this presents an obstacle, especially in resource limited environments.

Findings We present IDseq, an open source cloud-based metagenomics pipeline and service for global pathogen detection and monitoring (&lt;https://idseq.net&gt;). The IDseq Portal accepts raw mNGS data, performs host and quality filtration steps, then executes an assembly-based alignment pipeline which results in the assignment of reads and contigs to taxonomic categories. The taxonomic relative abundances are reported and visualized in an easy-to-use web application to facilitate data interpretation and hypothesis generation. Furthermore, IDseq supports environmental background model generation and automatic internal spike-in control recognition, providing statistics which are critical for data interpretation. IDseq was designed with the specific intent of detecting novel pathogens. Here, we benchmark novel virus detection capability using both synthetically evolved viral sequences, and real-world samples, including IDseq analysis of a nasopharyngeal swab sample acquired and processed locally in Cambodia from a tourist from Wuhan, China, infected with the recently emergent SARS-CoV-2.

Conclusion The IDseq Portal reduces the barrier to entry for mNGS data analysis and enables bench scientists, clinicians, and bioinformaticians to gain insight from mNGS datasets for both known and novel pathogens.

### Competing Interest Statement

The authors have declared no competing interest.

*   ASG
    :   Auto-scaling Groups
    SARS-CoV-2
    :   Severe Acute Respiratory Syndrome Coronavirus 2
    AUPR
    :   Area Under the Precision Recall Curve
    AWS
    :   Amazon Web Services
    CLI
    :   Command Line Interface
    CSF
    :   Cerebrospinal Fluid
    DAG
    :   Directed Acyclic Graph
    DCR
    :   Duplicate Compression Ratio
    EC2
    :   Elastic Compute Cloud
    ERCCs
    :   Externa RNA Controls Consortium
    mNGS
    :   metagenomic Next-Generation Sequencing
    NCBI
    :   National Center for Biotechnology Information
    QC
    :   Quality Control
    taxID
    :   Taxonomic identifier
    NT
    :   Nucleotide, NCBI nucleotide (nt) database
    NR
    :   Protein, NCBI non-redundant protein (nr) database
    rPM
    :   Reads per Million Reads Sequenced" />
<meta name="DC.Contributor" content="Katrina L. Kalantar" />
<meta name="DC.Contributor" content="Tiago Carvalho" />
<meta name="DC.Contributor" content="Charles F.A. de Bourcy" />
<meta name="DC.Contributor" content="Boris Dimitrov" />
<meta name="DC.Contributor" content="Greg Dingle" />
<meta name="DC.Contributor" content="Rebecca Egger" />
<meta name="DC.Contributor" content="Julie Han" />
<meta name="DC.Contributor" content="Olivia B. Holmes" />
<meta name="DC.Contributor" content="Yun-Fang Juan" />
<meta name="DC.Contributor" content="Ryan King" />
<meta name="DC.Contributor" content="Andrey Kislyuk" />
<meta name="DC.Contributor" content="Maria Mariano" />
<meta name="DC.Contributor" content="Lucia V. Reynoso" />
<meta name="DC.Contributor" content="David Rissato Cruz" />
<meta name="DC.Contributor" content="Jonathan Sheu" />
<meta name="DC.Contributor" content="Jennifer Tang" />
<meta name="DC.Contributor" content="James Wang" />
<meta name="DC.Contributor" content="Mark A. Zhang" />
<meta name="DC.Contributor" content="Emily Zhong" />
<meta name="DC.Contributor" content="Vida Ahyong" />
<meta name="DC.Contributor" content="Sreyngim Lay" />
<meta name="DC.Contributor" content="Sophana Chea" />
<meta name="DC.Contributor" content="Jennifer A. Bohl" />
<meta name="DC.Contributor" content="Jessica E. Manning" />
<meta name="DC.Contributor" content="Cristina M. Tato" />
<meta name="DC.Contributor" content="Joseph L. DeRisi" />
<meta name="article:published_time" content="2020-04-09" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="IDseq – An Open Source Cloud-based Pipeline and Analysis Service for Metagenomic Pathogen Detection and Monitoring" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Background: Metagenomic next generation sequencing (mNGS) has enabled the rapid, unbiased detection and identification of microbes without pathogen-specific reagents, culturing, or a priori knowledge of the microbial landscape. mNGS data analysis requires a series of computationally intensive processing steps to accurately determine the microbial composition of a sample. Existing mNGS data analysis tools typically require bioinformatics expertise and access to local server-class hardware resources. For many research laboratories, this presents an obstacle, especially in resource limited environments. Findings: We present IDseq, an open source cloud-based metagenomics pipeline and service for global pathogen detection and monitoring (https://idseq.net). The IDseq Portal accepts raw mNGS data, performs host and quality filtration steps, then executes an assembly-based alignment pipeline which results in the assignment of reads and contigs to taxonomic categories. The taxonomic relative abundances are reported and visualized in an easy-to-use web application to facilitate data interpretation and hypothesis generation. Furthermore, IDseq supports environmental background model generation and automatic internal spike-in control recognition, providing statistics which are critical for data interpretation. IDseq was designed with the specific intent of detecting novel pathogens. Here, we benchmark novel virus detection capability using both synthetically evolved viral sequences, and real-world samples, including IDseq analysis of a nasopharyngeal swab sample acquired and processed locally in Cambodia from a tourist from Wuhan, China, infected with the recently emergent SARS-CoV-2. Conclusion: The IDseq Portal reduces the barrier to entry for mNGS data analysis and enables bench scientists, clinicians, and bioinformaticians to gain insight from mNGS datasets for both known and novel pathogens.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.04.07.030551v2" />
<meta name="citation_id" content="2020.04.07.030551v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.04.07.030551v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.07.030551v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.04.07.030551v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.04.07.030551" />
<meta name="citation_num_pages" content="38" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.04.07.030551" />
<meta name="citation_author" content="Katrina L. Kalantar" />
<meta name="citation_author_institution" content="Chan Zuckerberg Initiative" />
<meta name="citation_author_email" content="katrina.kalantar@chanzuckerberg.com" />
<meta name="citation_author" content="Tiago Carvalho" />
<meta name="citation_author_institution" content="Chan Zuckerberg Initiative" />
<meta name="citation_author" content="Charles F.A. de Bourcy" />
<meta name="citation_author_institution" content="Chan Zuckerberg Initiative" />
<meta name="citation_author" content="Boris Dimitrov" />
<meta name="citation_author_institution" content="Chan Zuckerberg Initiative" />
<meta name="citation_author" content="Greg Dingle" />
<meta name="citation_author_institution" content="Chan Zuckerberg Initiative" />
<meta name="citation_author" content="Rebecca Egger" />
<meta name="citation_author_institution" content="Chan Zuckerberg Initiative" />
<meta name="citation_author" content="Julie Han" />
<meta name="citation_author_institution" content="Chan Zuckerberg Initiative" />
<meta name="citation_author" content="Olivia B. Holmes" />
<meta name="citation_author_institution" content="Chan Zuckerberg Initiative" />
<meta name="citation_author" content="Yun-Fang Juan" />
<meta name="citation_author_institution" content="Chan Zuckerberg Initiative" />
<meta name="citation_author" content="Ryan King" />
<meta name="citation_author_institution" content="Chan Zuckerberg Initiative" />
<meta name="citation_author" content="Andrey Kislyuk" />
<meta name="citation_author_institution" content="Chan Zuckerberg Initiative" />
<meta name="citation_author" content="Maria Mariano" />
<meta name="citation_author_institution" content="Chan Zuckerberg Initiative" />
<meta name="citation_author" content="Lucia V. Reynoso" />
<meta name="citation_author_institution" content="Chan Zuckerberg Initiative" />
<meta name="citation_author" content="David Rissato Cruz" />
<meta name="citation_author_institution" content="Chan Zuckerberg Initiative" />
<meta name="citation_author" content="Jonathan Sheu" />
<meta name="citation_author_institution" content="Chan Zuckerberg Initiative" />
<meta name="citation_author" content="Jennifer Tang" />
<meta name="citation_author_institution" content="Chan Zuckerberg Initiative" />
<meta name="citation_author" content="James Wang" />
<meta name="citation_author_institution" content="Chan Zuckerberg Initiative" />
<meta name="citation_author" content="Mark A. Zhang" />
<meta name="citation_author_institution" content="Chan Zuckerberg Initiative" />
<meta name="citation_author" content="Emily Zhong" />
<meta name="citation_author_institution" content="Chan Zuckerberg Initiative" />
<meta name="citation_author" content="Vida Ahyong" />
<meta name="citation_author_institution" content="Chan Zuckerberg Biohub" />
<meta name="citation_author" content="Sreyngim Lay" />
<meta name="citation_author_institution" content="Malaria and Vector Research Laboratory, National Institute of Allergy and Infectious Diseases" />
<meta name="citation_author" content="Sophana Chea" />
<meta name="citation_author_institution" content="Malaria and Vector Research Laboratory, National Institute of Allergy and Infectious Diseases" />
<meta name="citation_author" content="Jennifer A. Bohl" />
<meta name="citation_author_institution" content="Malaria and Vector Research Laboratory, National Institute of Allergy and Infectious Diseases" />
<meta name="citation_author" content="Jessica E. Manning" />
<meta name="citation_author_institution" content="Malaria and Vector Research Laboratory, National Institute of Allergy and Infectious Diseases" />
<meta name="citation_author" content="Cristina M. Tato" />
<meta name="citation_author_institution" content="Chan Zuckerberg Biohub" />
<meta name="citation_author" content="Joseph L. DeRisi" />
<meta name="citation_author_institution" content="Chan Zuckerberg Biohub" />
<meta name="citation_reference" content="Saha S, Ramesh A, Kalantar K, Malaker R, Hasanuzzaman M, Khan LM, et al. Unbiased metagenomic sequencing for pediatric meningitis in bangladesh reveals neuroinvasive chikungunya virus outbreak and other unrealized pathogens. MBio. 2019;10." />
<meta name="citation_reference" content="Simner PJ, Miller S, Carroll KC. Understanding the Promises and Hurdles of Metagenomic Next-Generation Sequencing as a Diagnostic Tool for Infectious Diseases. Clin Infect Dis. 2018; doi: 10.1093/cid/cix881." />
<meta name="citation_reference" content="Lu J, Breitwieser FP, Thielen P, Salzberg SL. Bracken: Estimating species abundance in metagenomics data. PeerJ Comput Sci. 2017; 2017:e104." />
<meta name="citation_reference" content="Kim D, Song L, Breitwieser FP, Salzberg SL. Centrifuge: Rapid and sensitive classification of metagenomic sequences. Genome Res. 2016;26:1721–9." />
<meta name="citation_reference" content="Walker MA, Pedamallu CS, Ojesina AI, Bullman S, Sharpe T, Whelan CW, et al. GATK PathSeq: a customizable computational tool for the discovery and identification of microbial sequences in libraries from eukaryotic hosts. Bioinformatics. 2018; doi: 10.1093/bioinformatics/bty501" />
<meta name="citation_reference" content="Břinda K, Salikhov K, Pignotti S, Kucherov G. karel-brinda/prophyle: ProPhyle 0.3.1.0. 2017;" />
<meta name="citation_reference" content="Corvelo A, Clarke WE, Robine N, Zody MC. taxMaps: Comprehensive and highly accurate taxonomic classification of short-read data in reasonable time. Genome Res. 2018;28:751–8." />
<meta name="citation_reference" content="Buchfink B, Xie C, Huson DH. Fast and sensitive protein alignment using DIAMOND. Nat. Methods. 2014. p. 59–60." />
<meta name="citation_reference" content="Menzel P, Ng KL, Krogh A. Fast and sensitive taxonomic classification for metagenomics with Kaiju. Nat Commun. 2016;7:1–9." />
<meta name="citation_reference" content="Hauser M, Steinegger M, Sö Ding J. MMseqs software suite for fast and deep clustering and searching of large protein sequence sets. 2016; doi: 10.1093/bioinformatics/btw006" />
<meta name="citation_reference" content="Truong DT, Franzosa EA, Tickle TL, Scholz M, Weingart G, Pasolli E, et al. MetaPhlAn2 for enhanced metagenomic taxonomic profiling. Nat. Methods. 2015. p. 902–3." />
<meta name="citation_reference" content="Milanese A, Mende DR, Paoli L, Salazar G, Ruscheweyh HJ, Cuenca M, et al. Microbial abundance, activity and population genomic profiling with mOTUs2. Nat Commun. 2019;10:1–11." />
<meta name="citation_reference" content="Ounit R, Wanamaker S, Close TJ, Lonardi S. CLARK: fast and accurate classification of metagenomic and genomic sequences using discriminative k-mers. BMC Genomics. 2015;16:236." />
<meta name="citation_reference" content="Ounit R, Lonardi S. Higher classification sensitivity of short metagenomic reads with CLARK-S. Bioinformatics. 2016;32." />
<meta name="citation_reference" content="Wood DE, Lu J, Langmead B. Improved metagenomic analysis with Kraken 2. Genome Biol. 2019;20:257." />
<meta name="citation_reference" content="Wood DE, Salzberg SL. Kraken: ultrafast metagenomic sequence classification using exact alignments. Genome Biology. 2014;304:66–74." />
<meta name="citation_reference" content="Breitwieser FP, Baker DN, Salzberg SL. KrakenUniq: confident and fast metagenomics classification using unique k-mer counts. Genome Biol. 2018;19:198." />
<meta name="citation_reference" content="Ainsworth D, Sternberg MJE, Come R, Butcher SA. k-SLAM: Accurate and Ultra-Fast Taxonomic Classification and Gene Identification for Large Metagenomic Data Sets - PubMed. Nucleic Acids Res. 2017;45." />
<meta name="citation_reference" content="Morgulis A, Coulouris G, Raytselis Y, Madden TL, Agarwala R, Schäffer AA. Database indexing for production MegaBLAST searches. Bioinformatics. 2008;24:1757–64." />
<meta name="citation_reference" content="Liu X, Yu Y, Liu J, Elliott CF, Qian C, Liu J. Sequence analysis A novel data structure to support ultra-fast taxonomic classification of metagenomic sequences with k-mer signatures. Bioinformatics. 2018;34." />
<meta name="citation_reference" content="Gu W, Miller S, Chiu CY. Clinical Metagenomic Next-Generation Sequencing for Pathogen Detection. Annu Rev Pathol Mech Dis. 2019;14:319–38." />
<meta name="citation_reference" content="One Codex [Internet]. [cited 2020 Mar 11]. Available from: https://www.onecodex.com/" />
<meta name="citation_reference" content="Clarke EL, Taylor LJ, Zhao C, Connell A, Lee JJ, Fett B, et al. Sunbeam: An extensible pipeline for analyzing metagenomic sequencing experiments. Microbiome. 2019;7:46" />
<meta name="citation_reference" content="Naccache SN, Federman S, Veeraraghavan N, Zaharia M, Lee D, Samayoa E, et al. A cloud-compatible bioinformatics pipeline for ultrarapid pathogen identification from next-generation sequencing of clinical samples. Genome Res. 2014;24:1180–92." />
<meta name="citation_reference" content="Yozwiak NL, Skewes-Cox P, Stenglein MD, Balmaseda A, Harris E, DeRisi JL. Virus Identification in Unknown Tropical Febrile Illness Cases Using Deep Sequencing. PLoS Negl Trop Dis. 2012;6:e1485." />
<meta name="citation_reference" content="Dobin A, Davis CA, Schlesinger F, Drenkow J, Zaleski C, Jha S, et al. Sequence analysis STAR: ultrafast universal RNA-seq aligner. Bioinformatics. 2013;29:15–21." />
<meta name="citation_reference" content="Langelier C, Kalantar KL, Moazed F, Wilson MR, Crawford ED, Deiss T, et al. Integrating host response and unbiased microbe detection for lower respiratory tract infection diagnosis in critically ill adults. Proc Natl Acad Sci. 2018;115:E12353–62." />
<meta name="citation_reference" content="Bolger AM, Lohse M, Usadel B. Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics. 2014;30:2114–20." />
<meta name="citation_reference" content="Ruby JG, Bellare P, DeRisi JL. PRICE: Software for the targeted assembly of components of (Meta) genomic sequence data. G3 Genes, Genomes, Genet. 2013;3:865–80." />
<meta name="citation_reference" content="Li Ã W, Godzik A. Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences. Bioinformatics. 2006;22:1658–9." />
<meta name="citation_reference" content="Langmead B, Salzberg SL. Fast gapped-read alignment with Bowtie 2. Nat Methods. 2012;9:357–9." />
<meta name="citation_reference" content="Wu TD, Nacu S. Sequence analysis Fast and SNP-tolerant detection of complex variants and splicing in short reads. Bioinformatics. 2010;26:873–81." />
<meta name="citation_reference" content="Zinter MS, Mayday MY, Ryckman KK, Jelliffe-Pawlowski LL, Derisi JL. Towards precision quantification of contamination in metagenomic sequencing experiments. Microbiome. 2019;7:1–5." />
<meta name="citation_reference" content="Davis NM, Proctor DiM, Holmes SP, Relman DA, Callahan BJ. Simple statistical identification and removal of contaminant sequences in marker-gene and metagenomics data. Microbiome. 2018;6:226." />
<meta name="citation_reference" content="Ye Y, Choi JH, Tang H. RAPSearch: A fast protein similarity search tool for short reads. BMC Bioinformatics. 2011;12:159." />
<meta name="citation_reference" content="Index of /pub/taxonomy/accession2taxid [Internet]. [cited 2020 Feb 26]. Available from: ftp://ftp.ncbi.nih.gov/pub/taxonomy/accession2taxid" />
<meta name="citation_reference" content="Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ. Basic local alignment search tool. J Mol Biol. 1990;215:403–10." />
<meta name="citation_reference" content="Kulikov AS, Prjibelski AD, Tesler G, Vyahhi N, Sirotkin A V., Pham S, et al. SPAdes: A New Genome Assembly Algorithm and Its Applications to Single-Cell Sequencing. J Comput Biol. 2012;19:455–77." />
<meta name="citation_reference" content="Ye SH, Siddle KJ, Park DJ, Sabeti PC. Benchmarking Metagenomics Tools for Taxonomic Classification. Cell. 2019. p. 779–94." />
<meta name="citation_reference" content="Luo C, Knight R, Siljander H, Knip M, Xavier RJ, Gevers D. ConStrains identifies microbial strains in metagenomic datasets. Nat Biotechnol. 2015;33:1045–52." />
<meta name="citation_reference" content="Scholz M, Ward D V., Pasolli E, Tolio T, Zolfo M, Asnicar F, et al. Strain-level microbial epidemiology and population genomics from shotgun metagenomics. Nat Methods. 2016;13:435–8." />
<meta name="citation_reference" content="Truong DT, Tett A, Pasolli E, Huttenhower C, Segata N. Microbial strain-level population structure &amp; genetic diversity from metagenomes. Genome Res. 2017;27:626–38." />
<meta name="citation_reference" content="Steinhauer DA, Holland JJ. Rapid Evolution of RNA Viruses. Annu Rev Microbiol. 1987;41:409–31." />
<meta name="citation_reference" content="Woolhouse MEJ, Brierley L, McCaffery C, Lycett S. Assessing the epidemic potential of RNA and DNA viruses. Emerg Infect Dis. Centers for Disease Control and Prevention (CDC); 2016;22:2037–44." />
<meta name="citation_reference" content="Schuffenecker I, Iteman I, Michault A, Murri S, Frangeul L, Vaney M-C, et al. Genome Microevolution of Chikungunya Viruses Causing the Indian Ocean Outbreak. PLoS Med [Internet]. 2006;3:e263." />
<meta name="citation_reference" content="Pu J, Wang S, Yin Y, Zhang G, Carter RA, Wang J, et al. Evolution of the H9N2 influenza genotype that facilitated the genesis of the novel H7N9 virus. Proc Natl Acad Sci. 2015;112:548–53." />
<meta name="citation_reference" content="Zhu N, Zhang D, Wang W, Li X, Yang B, Song J, et al. A Novel Coronavirus from Patients with Pneumonia in China, 2019. N Engl J Med. 2020;" />
<meta name="citation_reference" content="Chappell JG, Byaruhanga T, Tsoleridis T, Ball JK, Patrick McClure C. Identification of infectious agents in high-throughput sequencing data sets is easily achievable using free, cloud-based bioinformatics platforms. J. Clin. Microbiol 2019." />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=A. Ramesh;citation_author=S. Nakielny;citation_author=J. Hsu;citation_author=M. Kyohere;citation_author=O. Byaruhanga;citation_author=C. de Bourcy;citation_title=Metagenomic next-generation sequencing of samples from pediatric febrile illness in Tororo, Uganda;citation_pages=e0218318;citation_volume=14;citation_year=2019" />
<meta name="citation_reference" content="Crawford E, Kamm J, Miller S, Li LM, Caldera S, Lyden A, et al. Investigating Transfusion-related Sepsis Using Culture-Independent Metagenomic Sequencing. Clin Infect Dis. 2019. doi: 10.1093/cid/ciz960" />
<meta name="citation_reference" content="Hasan MR, Sundararaju S, Tang P, Tsui K-M, Lopez AP, Janahi M, et al. A Metagenomics – Based Diagnostic Approach for Central Nervous System Infections in Hospital Acute Care Setting. bioRxiv. 2019;752618." />
<meta name="citation_reference" content="Ruppé E, Schrenzel J. Messages from the second International Conference on Clinical Metagenomics (ICCMg2). Microbes Infect. 2018. p. 222–7." />
<meta name="citation_reference" content="Wilson MR, O’Donovan BD, Gelfand JM, Sample HA, Chow FC, Betjemann JP, et al. Chronic meningitis investigated via metagenomic next-generation sequencing. JAMA Neurol. 2018;75:947–55." />
<meta name="citation_reference" content="Mollerup S, Friis-Nielsen J, Vinner L, Hansen TA, Richter SR, Fridholm H, et al. Propionibacterium acnes: Disease-causing agent or common contaminant? detection in diverse patient samples by next-generation sequencing. J Clin Microbiol. 2016;54:980–7." />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. Laurence;citation_author=C. Hatzis;citation_author=DE. Brash;citation_title=Common Contaminants in Next-Generation Sequencing That Hinder Discovery of Low-Abundance Microbes;citation_pages=e97876;citation_volume=9;citation_year=2014;citation_pmid=24837716;citation_doi=10.1371/journal.pone.0097876" />
<meta name="citation_reference" content="Manning JE, Bohl JA, Lay S, Chea S, Sovann L, Sengdoeurn Y, et al. Rapid metagenomic characterization of a case of imported COVID-19 in Cambodia. Biorxiv. 2020. doi: 10.1101/2020.03.02.968818" />
<meta name="citation_reference" content="Retallack H, Okihiro MS, Britton E, Sommeran S Van, DeRisi JL. Metagenomic Next-Generation Sequencing Reveals Miamiensis avidus (Ciliophora: Scuticocilitida) in the 2017 Epizootic of Leopard Sharks (Triakis semifasciata) in San Francisco Bay, California, USA. J Wildl Dis. 2019;55:375." />
<meta name="citation_reference" content="Batson J, Dudas G, Haas-Stapleton E, Kistler AL, Li LM, Logan P, et al. Single mosquito metatranscriptomics recovers mosquito species, blood meal sources, and microbial cargo, including viral dark matter. bioRxiv. 2020;2020.02.10.942854." />
<meta name="citation_reference" content="Sczyrba A, Hofmann P, Belmann P, Koslicki D, Janssen S, Dröge J, et al. Critical Assessment of Metagenome Interpretation - A benchmark of metagenomics software. Nat Methods. 2017;14:1063–71." />
<meta name="citation_reference" content="McIntyre ABR, Ounit R, Afshinnekoo E, Prill RJ, Hénaff E, Alexander N, et al. Comprehensive benchmarking and ensemble approaches for metagenomic classifiers. Genome Biol. 2017;18:1–19." />
<meta name="citation_reference" content="Zinter MS, Dvorak CC, Mayday MY, Iwanaga K, Ly NP, Mcgarry ME, et al. Clinical Infectious Diseases Pulmonary Metagenomic Sequencing Suggests Missed Infections in Immunocompromised Children. Clin Infect Dis. 2019:1847." />
<meta name="citation_reference" content="GitHub - chanzuckerberg/idseq-bench: IDseq infectious disease benchmarking tools. [cited 2020 Mar 11]. Available from: https://github.com/chanzuckerberg/idseq-bench" />
<meta name="citation_reference" content="Gourlé H, Karlsson-Lindsjö O, Hayer J, Bongcam-Rudloff E. Simulating Illumina metagenomic data with InSilicoSeq. [cited 2020 Feb 26]; Available from: https://github.com/HadrienG/InSilicoSeq" />
<meta name="citation_reference" content="Petitjean M, Vanet A. VIRAPOPS2 supports the influenza virus reassortments. Biol Med. 2014;9:18." />
<meta name="citation_reference" content="O’leary NA, Wright MW, Brister JR, Ciufo S, Haddad D, Mcveigh R, et al. Reference sequence (RefSeq) database at NCBI: current status, taxonomic expansion, and functional annotation. Nucleic Acids Res. 2015;44:733–45." />
<meta name="citation_reference" content="Guentzel MN. Escherichia, Klebsiella, Enterobacter, Serratia, Citrobacter, and Proteus. Med. Microbiol. 1996." />
<meta name="citation_reference" content="Wood DE, Lu J, Langmead B. Improved metagenomic analysis with Kraken 2. Genome Biol. 2019;20:257." />
<meta name="citation_reference" content="Gourlé H, Karlsson-Lindsjö O, Hayer J, Bongcam-Rudloff E. Simulating Illumina metagenomic data with InSilicoSeq. [cited 2020 Feb 26]; Available from: https://github.com/HadrienG/InSilicoSeq" />
<meta name="citation_reference" content="Manter DK, Hunter WJ, Vivanco JM. Enterobacter soli sp. nov.: A lignin-degrading γ-Proteobacteria isolated from soil. Curr Microbiol. 2011;62:1044–9." />
<meta name="citation_reference" content="Ounit R, Lonardi S. Higher classification sensitivity of short metagenomic reads with CLARK-S. Bioinformatics. 2016;32:3823–5." />
<meta name="citation_reference" content="Bosi E, Fondi M, Orlandini V, Perrin E, Maida I, de Pascale D, et al. The pangenome of (Antarctic) Pseudoalteromonas bacteria: Evolutionary and functional insights. BMC Genomics. 2017;18:93." />
<meta name="citation_reference" content="Pellegrinelli L, Giardina F, Lunghi G, Renteria SCU, Greco L, Fratini A, et al. Emergence of divergent enterovirus (EV) D68 sub-clade D1 strains, northern Italy, September to October 2018. Eurosurveillance. 2019;24." />
<meta name="citation_reference" content="Ahamed SF, Rosario V, Britto C, Dias M, Nayak K, Chandele A, et al. Emergence of new genotypes and lineages of dengue viruses during the 2012–15 epidemics in southern India. Int J Infect Dis. 2019;84:S34–43." />
<meta name="citation_reference" content="Jesus JG de, Dutra KR, Salles FC da S, Claro IM, Terzian AC, Candido D da S, et al. Early identification of dengue virus lineage replacement in Brazil using portable genomic surveillance. bioRxiv. 2019;716159." />
<meta name="citation_reference" content="Fall G, Di Paola N, Faye M, Dia M, Freire CC de M, Loucoubar C, et al. Biological and phylogenetic characteristics of West African lineages of West Nile virus. PLoS Negl Trop Dis. 2017;11:e0006078." />
<meta name="citation_reference" content="Chang HH, Huber RG, Bond PJ, Grad YH, Camerini D, Maurer-Stroh S, et al. Systematic analysis of protein identity between Zika virus and other arthropod-borne viruses. World Health Organization; 2017;95:517–25." />
<meta name="citation_reference" content="Batson J, Dudas G, Haas-Stapleton E, Kistler AL, Li LM, Logan P, et al. Single mosquito metatranscriptomics recovers mosquito species, blood meal sources, and microbial cargo, including viral dark matter. bioRxiv. 2020;2020.02.10.942854." />
<meta name="citation_reference" content="Ribosomal RNA - an overview | ScienceDirect Topics [Internet]. [cited 2020 Feb 26]. Available from: https://www.sciencedirect.com/topics/neuroscience/ribosomal-rna" />
<meta name="citation_reference" content="Cottier F, Srinivasan KG, Yurieva M, Liao W, Poidinger M, Zolezzi F, et al. Advantages of meta-total RNA sequencing (MeTRS) over shotgun metagenomics and amplicon-based sequencing in the profiling of complex microbial communities. Biofilms Microbiomes. 2018;4:1–7." />
<meta name="twitter:title" content="IDseq – An Open Source Cloud-based Pipeline and Analysis Service for Metagenomic Pathogen Detection and Monitoring" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Background Metagenomic next generation sequencing (mNGS) has enabled the rapid, unbiased detection and identification of microbes without pathogen-specific reagents, culturing, or a priori knowledge of the microbial landscape. mNGS data analysis requires a series of computationally intensive processing steps to accurately determine the microbial composition of a sample. Existing mNGS data analysis tools typically require bioinformatics expertise and access to local server-class hardware resources. For many research laboratories, this presents an obstacle, especially in resource limited environments.

Findings We present IDseq, an open source cloud-based metagenomics pipeline and service for global pathogen detection and monitoring (&lt;https://idseq.net&gt;). The IDseq Portal accepts raw mNGS data, performs host and quality filtration steps, then executes an assembly-based alignment pipeline which results in the assignment of reads and contigs to taxonomic categories. The taxonomic relative abundances are reported and visualized in an easy-to-use web application to facilitate data interpretation and hypothesis generation. Furthermore, IDseq supports environmental background model generation and automatic internal spike-in control recognition, providing statistics which are critical for data interpretation. IDseq was designed with the specific intent of detecting novel pathogens. Here, we benchmark novel virus detection capability using both synthetically evolved viral sequences, and real-world samples, including IDseq analysis of a nasopharyngeal swab sample acquired and processed locally in Cambodia from a tourist from Wuhan, China, infected with the recently emergent SARS-CoV-2.

Conclusion The IDseq Portal reduces the barrier to entry for mNGS data analysis and enables bench scientists, clinicians, and bioinformaticians to gain insight from mNGS datasets for both known and novel pathogens.

### Competing Interest Statement

The authors have declared no competing interest.

*   ASG
    :   Auto-scaling Groups
    SARS-CoV-2
    :   Severe Acute Respiratory Syndrome Coronavirus 2
    AUPR
    :   Area Under the Precision Recall Curve
    AWS
    :   Amazon Web Services
    CLI
    :   Command Line Interface
    CSF
    :   Cerebrospinal Fluid
    DAG
    :   Directed Acyclic Graph
    DCR
    :   Duplicate Compression Ratio
    EC2
    :   Elastic Compute Cloud
    ERCCs
    :   Externa RNA Controls Consortium
    mNGS
    :   metagenomic Next-Generation Sequencing
    NCBI
    :   National Center for Biotechnology Information
    QC
    :   Quality Control
    taxID
    :   Taxonomic identifier
    NT
    :   Nucleotide, NCBI nucleotide (nt) database
    NR
    :   Protein, NCBI non-redundant protein (nr) database
    rPM
    :   Reads per Million Reads Sequenced" />
<meta name="og-title" property="og:title" content="IDseq – An Open Source Cloud-based Pipeline and Analysis Service for Metagenomic Pathogen Detection and Monitoring" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.04.07.030551v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Background Metagenomic next generation sequencing (mNGS) has enabled the rapid, unbiased detection and identification of microbes without pathogen-specific reagents, culturing, or a priori knowledge of the microbial landscape. mNGS data analysis requires a series of computationally intensive processing steps to accurately determine the microbial composition of a sample. Existing mNGS data analysis tools typically require bioinformatics expertise and access to local server-class hardware resources. For many research laboratories, this presents an obstacle, especially in resource limited environments.

Findings We present IDseq, an open source cloud-based metagenomics pipeline and service for global pathogen detection and monitoring (&lt;https://idseq.net&gt;). The IDseq Portal accepts raw mNGS data, performs host and quality filtration steps, then executes an assembly-based alignment pipeline which results in the assignment of reads and contigs to taxonomic categories. The taxonomic relative abundances are reported and visualized in an easy-to-use web application to facilitate data interpretation and hypothesis generation. Furthermore, IDseq supports environmental background model generation and automatic internal spike-in control recognition, providing statistics which are critical for data interpretation. IDseq was designed with the specific intent of detecting novel pathogens. Here, we benchmark novel virus detection capability using both synthetically evolved viral sequences, and real-world samples, including IDseq analysis of a nasopharyngeal swab sample acquired and processed locally in Cambodia from a tourist from Wuhan, China, infected with the recently emergent SARS-CoV-2.

Conclusion The IDseq Portal reduces the barrier to entry for mNGS data analysis and enables bench scientists, clinicians, and bioinformaticians to gain insight from mNGS datasets for both known and novel pathogens.

### Competing Interest Statement

The authors have declared no competing interest.

*   ASG
    :   Auto-scaling Groups
    SARS-CoV-2
    :   Severe Acute Respiratory Syndrome Coronavirus 2
    AUPR
    :   Area Under the Precision Recall Curve
    AWS
    :   Amazon Web Services
    CLI
    :   Command Line Interface
    CSF
    :   Cerebrospinal Fluid
    DAG
    :   Directed Acyclic Graph
    DCR
    :   Duplicate Compression Ratio
    EC2
    :   Elastic Compute Cloud
    ERCCs
    :   Externa RNA Controls Consortium
    mNGS
    :   metagenomic Next-Generation Sequencing
    NCBI
    :   National Center for Biotechnology Information
    QC
    :   Quality Control
    taxID
    :   Taxonomic identifier
    NT
    :   Nucleotide, NCBI nucleotide (nt) database
    NR
    :   Protein, NCBI non-redundant protein (nr) database
    rPM
    :   Reads per Million Reads Sequenced" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-04-09" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.04.07.030551v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.04.07.030551v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.04.07.030551v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.04.07.030551v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1237378" />
    <title>IDseq – An Open Source Cloud-based Pipeline and Analysis Service for Metagenomic Pathogen Detection and Monitoring | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__J9zYrFgVU9w_9DRO_7M268JyEPb2If_fHKWgrFly-Tw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"eyOKBzponyQmd4hv6_W1neIzaeu96JFJdfF1PmBe6Ro","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1237378","apath":"\/biorxiv\/early\/2020\/04\/09\/2020.04.07.030551.1.atom","pisa":"biorxiv;2020.04.07.030551v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.04.07.030551v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.04.07.030551v2","title":"IDseq \u2013 An Open Source Cloud-based Pipeline and Analysis Service for Metagenomic Pathogen Detection and Monitoring","identifier":"node\/1237378"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022nblgPWvBdJSFto50H29FLv\\\\\\\/Gdx9h\\\\\\\/u53CfMFpnL8DPnv+aQo7egHvXr6Tj4mTkItzm4VtPLdESJsNvzQYw4ThDyvypSJF8\\\\\\\/GkYSpsrr+fP7PSMrHtEqIjYfG+iUNliF1vqMcTVRySmeIJwrnMgAKJCMKMVdC0A9cVs79915syy9BcNs1COvTPJ2aso8eBcOVYx\\\\\\\/gHFwefQ1FjSIU0\\\\\\\/MfhKypci8l82ScOALrxle\\\\\\\/PICy+d31y\\\\\\\/ykxX6lblSqDZu6qnMz+f+AEP1FNbmjOCOyZycWhgJkQA\\\\\\\/ybaZ1\\\\\\\/KDin5gymJOfup7+rdSrWbOAQj8MZoqsgOd8Zh\\\\\\\/hWyaVhat+H2T7RwN7KYom1ny8jyDSANY2F9QBo27z5hjI3rrZRxm55UHajLNzPVWeZJGoE4JDRT1YTQbY9QTe+IxKnstTOCJtqXNZf30YrqaquOnPc0XOl3bbyFwoW3wSj\\\\\\\/2Iz2iRP6ZEA15c647F6PnIlpxRIdGTFbRfdamiVPJGDOUo3Ju1vl4RGimxgJlMV0tre9ra9Inv86f5Qh4DI3nxAP+ApWUZmaqF+4auDv1BetX+RkMOAT1GFOwIUJ3NcqR7uRFrIW8gDRccMZUnR310ufNXJmaUxz5uDLtKPZp9IlDNYsuRnnH9ZWHVxTUbrM0YWdBv37yhPwa8tAGqonU9y9TD8ppLjFS1pGNW5nywR6mzct76nSYix21ScMzskatI3rQi2XYWOlwniJ49ourakwm7Dv0hojFzB\\\\\\\/U1AAAHHpRZ1yC3zQVs4e+9tDELwr2BctcxBrudx6\\\\\\\/\\\\\\\/wYMwYfTDGS2OO\\\\\\\/aEyxGcut6pPnTx3tnepOqVmpYAUjcja+Oc0DFPSaF+4Pt9as+EakZ0RG7V5zXAtH1AQ\\\\\\\/Lw1XAJlPAQyNJSlggRAhoja50CageH144F3Cm72hyMUAjGU37tnI7uMQ7FwJ0OsLGmDXEJzL3qP+4iWGwaCXit9UGO3WaEfazjf0G8vW3DGXmDYCo6o4CDbWjT5Kzob9nw\\\\\\\/DHoEmq9WJ8rlHd2di5xEIkeubfkojuhatM9U+ADr1m2S5lawyUsP2qF1EzLFMgkpHYgPBQRXEunqAB7q1t8qTCwGK7shrQIuUdUjcbR62FdF2azb7eXq9Ed\\\\\\\/RT88\\\\\\\/aB4MPwcUMQXSAjx58sp6pYnAgq3FCFhUCT\\\\\\\/GpQrOXramV0yKaBX5afhUwmUqd2kqzGl6vsmKJq5U2kW2oAu\\\\\\\/TCh0NFxSJqyO+0eYJ2j8UxPT\\\\\\\/Oi\\\\\\\/5augsgKb5pJdR0pgFefnR77hT9QsPjZQiRrNNppwBf6uR\\\\\\\/bSGm9X\\\\\\\/qDrZVPpwH1WYctwxWORrdXzMLPkgWR6vj1qznR3awTKET\\\u0022,\\\u0022iv\\\u0022:\\\u00223q+exr6ab0JicCkFpVzyVQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00220f7857232d4b61efdace7ae7cf0098392b08bdcfb4e11704354625f91cf757cc\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.04.07.030551v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1237378 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.04.07.030551v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Sg_uoCjN1TGB4A7fym2aBrqOBixJIeWPS6_FmzzQR3o" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1237378" id="node1237378" data-pisa="biorxiv;2020.04.07.030551v2" data-pisa-master="biorxiv;2020.04.07.030551" data-apath="/biorxiv/early/2020/04/09/2020.04.07.030551.1.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">IDseq – An Open Source Cloud-based Pipeline and Analysis Service for Metagenomic Pathogen Detection and Monitoring</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Katrina L.</span> <span class="nlm-surname">Kalantar</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Tiago</span> <span class="nlm-surname">Carvalho</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Charles F.A.</span> <span class="nlm-surname">de Bourcy</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Boris</span> <span class="nlm-surname">Dimitrov</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Greg</span> <span class="nlm-surname">Dingle</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Rebecca</span> <span class="nlm-surname">Egger</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Julie</span> <span class="nlm-surname">Han</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Olivia B.</span> <span class="nlm-surname">Holmes</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Yun-Fang</span> <span class="nlm-surname">Juan</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Ryan</span> <span class="nlm-surname">King</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Andrey</span> <span class="nlm-surname">Kislyuk</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Maria</span> <span class="nlm-surname">Mariano</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Lucia V.</span> <span class="nlm-surname">Reynoso</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">David Rissato</span> <span class="nlm-surname">Cruz</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Jonathan</span> <span class="nlm-surname">Sheu</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Jennifer</span> <span class="nlm-surname">Tang</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">James</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="17"><span class="nlm-given-names">Mark A.</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="18"><span class="nlm-given-names">Emily</span> <span class="nlm-surname">Zhong</span></span>, <span class="highwire-citation-author" data-delta="19"><span class="nlm-given-names">Vida</span> <span class="nlm-surname">Ahyong</span></span>, <span class="highwire-citation-author" data-delta="20"><span class="nlm-given-names">Sreyngim</span> <span class="nlm-surname">Lay</span></span>, <span class="highwire-citation-author" data-delta="21"><span class="nlm-given-names">Sophana</span> <span class="nlm-surname">Chea</span></span>, <span class="highwire-citation-author" data-delta="22"><span class="nlm-given-names">Jennifer A.</span> <span class="nlm-surname">Bohl</span></span>, <span class="highwire-citation-author" data-delta="23"><span class="nlm-given-names">Jessica E.</span> <span class="nlm-surname">Manning</span></span>, <span class="highwire-citation-author" data-delta="24"><span class="nlm-given-names">Cristina M.</span> <span class="nlm-surname">Tato</span></span>, <span class="highwire-citation-author" data-delta="25"><span class="nlm-given-names">Joseph L.</span> <span class="nlm-surname">DeRisi</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.04.07.030551 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1237378" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Katrina L. Kalantar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Chan Zuckerberg Initiative</span>, Redwood City, California, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katrina%2BL.%2BKalantar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kalantar%20KL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKatrina%2BL.%2BKalantar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:katrina.kalantar@chanzuckerberg.com" class="" data-icon-position="" data-hide-link-title="0">katrina.kalantar@chanzuckerberg.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Tiago Carvalho </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Chan Zuckerberg Initiative</span>, Redwood City, California, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tiago%2BCarvalho%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carvalho%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATiago%2BCarvalho%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Charles F.A. de Bourcy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Chan Zuckerberg Initiative</span>, Redwood City, California, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BF.A.%2Bde%2BBourcy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Bourcy%20CF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BF.A.%2Bde%2BBourcy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Boris Dimitrov </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Chan Zuckerberg Initiative</span>, Redwood City, California, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Boris%2BDimitrov%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dimitrov%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABoris%2BDimitrov%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Greg Dingle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Chan Zuckerberg Initiative</span>, Redwood City, California, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Greg%2BDingle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dingle%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGreg%2BDingle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Rebecca Egger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Chan Zuckerberg Initiative</span>, Redwood City, California, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rebecca%2BEgger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Egger%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARebecca%2BEgger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Julie Han </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Chan Zuckerberg Initiative</span>, Redwood City, California, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julie%2BHan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Han%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJulie%2BHan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Olivia B. Holmes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Chan Zuckerberg Initiative</span>, Redwood City, California, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Olivia%2BB.%2BHolmes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Holmes%20OB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOlivia%2BB.%2BHolmes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Yun-Fang Juan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Chan Zuckerberg Initiative</span>, Redwood City, California, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yun-Fang%2BJuan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Juan%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYun-Fang%2BJuan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Ryan King </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Chan Zuckerberg Initiative</span>, Redwood City, California, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ryan%2BKing%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=King%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARyan%2BKing%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Andrey Kislyuk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Chan Zuckerberg Initiative</span>, Redwood City, California, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrey%2BKislyuk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kislyuk%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrey%2BKislyuk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Maria Mariano </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Chan Zuckerberg Initiative</span>, Redwood City, California, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BMariano%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mariano%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria%2BMariano%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Lucia V. Reynoso </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Chan Zuckerberg Initiative</span>, Redwood City, California, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lucia%2BV.%2BReynoso%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reynoso%20LV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALucia%2BV.%2BReynoso%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">David Rissato Cruz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Chan Zuckerberg Initiative</span>, Redwood City, California, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BRissato%2BCruz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cruz%20DR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BRissato%2BCruz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Jonathan Sheu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Chan Zuckerberg Initiative</span>, Redwood City, California, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BSheu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sheu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BSheu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Jennifer Tang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Chan Zuckerberg Initiative</span>, Redwood City, California, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BTang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BTang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-16"><div class="author-tooltip-name">James Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Chan Zuckerberg Initiative</span>, Redwood City, California, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-17"><div class="author-tooltip-name">Mark A. Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Chan Zuckerberg Initiative</span>, Redwood City, California, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BA.%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BA.%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-18"><div class="author-tooltip-name">Emily Zhong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Chan Zuckerberg Initiative</span>, Redwood City, California, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emily%2BZhong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhong%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEmily%2BZhong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-19"><div class="author-tooltip-name">Vida Ahyong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Chan Zuckerberg Biohub</span>, San Francisco, California, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vida%2BAhyong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ahyong%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVida%2BAhyong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-20"><div class="author-tooltip-name">Sreyngim Lay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Malaria and Vector Research Laboratory, National Institute of Allergy and Infectious Diseases</span>, Phnom Penh, <span class='nlm-country'>Cambodia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sreyngim%2BLay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lay%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASreyngim%2BLay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-21"><div class="author-tooltip-name">Sophana Chea </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Malaria and Vector Research Laboratory, National Institute of Allergy and Infectious Diseases</span>, Phnom Penh, <span class='nlm-country'>Cambodia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sophana%2BChea%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chea%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASophana%2BChea%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-22"><div class="author-tooltip-name">Jennifer A. Bohl </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Malaria and Vector Research Laboratory, National Institute of Allergy and Infectious Diseases</span>, Phnom Penh, <span class='nlm-country'>Cambodia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BA.%2BBohl%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bohl%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BA.%2BBohl%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-23"><div class="author-tooltip-name">Jessica E. Manning </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Malaria and Vector Research Laboratory, National Institute of Allergy and Infectious Diseases</span>, Phnom Penh, <span class='nlm-country'>Cambodia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jessica%2BE.%2BManning%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Manning%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJessica%2BE.%2BManning%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-24"><div class="author-tooltip-name">Cristina M. Tato </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Chan Zuckerberg Biohub</span>, San Francisco, California, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cristina%2BM.%2BTato%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tato%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACristina%2BM.%2BTato%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-25"><div class="author-tooltip-name">Joseph L. DeRisi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Chan Zuckerberg Biohub</span>, San Francisco, California, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BL.%2BDeRisi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=DeRisi%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoseph%2BL.%2BDeRisi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.04.07.030551v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1237378" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1237378/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.07.030551v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1237378" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1237378/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.07.030551v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1237378" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1237378/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.04.07.030551v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1237378" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1237378/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.04.07.030551v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1237378" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1237378/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">ABSTRACT</h2><div id="sec-1" class="subsection"><p id="p-2"><strong>Background</strong> Metagenomic next generation sequencing (mNGS) has enabled the rapid, unbiased detection and identification of microbes without pathogen-specific reagents, culturing, or <em>a priori</em> knowledge of the microbial landscape. mNGS data analysis requires a series of computationally intensive processing steps to accurately determine the microbial composition of a sample. Existing mNGS data analysis tools typically require bioinformatics expertise and access to local server-class hardware resources. For many research laboratories, this presents an obstacle, especially in resource limited environments.</p></div><div id="sec-2" class="subsection"><p id="p-3"><strong>Findings</strong> We present IDseq, an open source cloud-based metagenomics pipeline and service for global pathogen detection and monitoring (<a href="https://idseq.net">https://idseq.net</a>). The IDseq Portal accepts raw mNGS data, performs host and quality filtration steps, then executes an assembly-based alignment pipeline which results in the assignment of reads and contigs to taxonomic categories. The taxonomic relative abundances are reported and visualized in an easy-to-use web application to facilitate data interpretation and hypothesis generation. Furthermore, IDseq supports environmental background model generation and automatic internal spike-in control recognition, providing statistics which are critical for data interpretation. IDseq was designed with the specific intent of detecting novel pathogens. Here, we benchmark novel virus detection capability using both synthetically evolved viral sequences, and real-world samples, including IDseq analysis of a nasopharyngeal swab sample acquired and processed locally in Cambodia from a tourist from Wuhan, China, infected with the recently emergent SARS-CoV-2.</p></div><div id="sec-3" class="subsection"><p id="p-4"><strong>Conclusion</strong> The IDseq Portal reduces the barrier to entry for mNGS data analysis and enables bench scientists, clinicians, and bioinformaticians to gain insight from mNGS datasets for both known and novel pathogens.</p></div></div><div class="section" id="sec-4"><h2 class="">BACKGROUND</h2><p id="p-7">Infectious diseases remain a leading cause of morbidity and mortality worldwide. Despite significant advancement in our understanding of infectious disease biology, existing microbiological tests often fail to identify etiologic pathogens in cases of suspected infection. This can be due to a number of causes - failure to isolate an appropriate sample type, preemptive antibiotic exposure precluding growth in culture, lack of suspicion of a particular infection precluding the ordering of an appropriate test, or lack of available specific diagnostic tests due, in part, to limited knowledge of circulating pathogens. This is compounded further by the fact that novel, previously uncharacterized pathogens may also be present. This fact was illustrated vividly by the recent emergence of COVID-19 in Wuhan, China, in early December 2019. Metagenomic next-generation sequencing (mNGS) of nucleic acid from biological samples offers the potential for a universal pathogen detection method, including the detection of novel species. mNGS has great potential as a broad spectrum surveillance or patient monitoring tool, especially in low and middle income countries where the infectious disease burden remains high [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>]. While the expense of sequencing continues to drop, the challenge of mNGS data analysis, the lack of bioinformatics expertise, and the access to sufficient compute and storage remains a major obstacle.</p><p id="p-8">mNGS experiments result in millions of sequencing reads generated from the nucleic acid present within a biological sample, which may include complex microbial populations. A primary goal of mNGS data analysis is to determine what nucleic acid derives from the host (for example, a patient), and what cannot be attributed to the host or environmental contaminants. Further analysis of the non-host sequence may then attempt to determine the relative abundances of different taxa present in a particular sample, as this may provide insight into the presence and relevance of potentially pathogenic microbes. This is typically done via alignment of sequencing reads to a reference database. In the context of infectious diseases, identification of pathogens via this approach obviates the need for pathogen-specific reagents or the ability to culture the microbe. This is especially important for microbes that are difficult, or impossible to culture, including many viruses, fungal species, eukaryotic parasites, and bacteria [<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]. Additional downstream analysis may then be employed to understand trends in the abundances and relatedness of organisms across samples.</p><p id="p-9">There are several tools available for estimating relative abundance of microbial populations from mNGS data [<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>–<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>]. However, running these tools requires bioinformatics expertise and fluency with command-line tools. Additionally, pathogen detection in the context of a host organism presents unique informatics challenges beyond microbial abundance estimation. As noted, a substantial fraction of the sample may consist of host sequences that are secondary to the goal of pathogen detection [<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>]. Existing tools do not perform sensitive removal of host sequences or quality control (QC) steps, thus requiring the use of separate QC and alignment tools, and therefore additional computational experience in pipelining. A number of tools exist to incorporate multiple pipeline steps alongside reporting capabilities, including OneCodex [<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>], Sunbeam [<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>], and SURPI [<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>]. However, these tools require paid subscription or significant computational resources to build the underlying databases and run the analyses. Consequently, existing tools are not sufficient to support new applications of mNGS in poorly resourced settings where the detection of infectious agents could make a major impact on population health.</p><p id="p-10">Here, we describe IDseq - an open source cloud-based service for pathogen detection and monitoring. IDseq is a continuously evolving service that enables robust and reproducible analysis of mNGS data for microbial identification, regardless of sample type or host species. We first describe the technical aspects of the IDseq pipeline implementation, including host filtration and QC, assembly-based alignment, and downstream reporting and visualization tools. We then evaluate the performance of the IDseq pipeline, first on a set of standard mNGS benchmark samples as compared to other tools aimed at providing taxonomic abundance estimates from mNGS data, and secondly on a simulated dataset to evaluate the ability to detect divergent viruses. Finally, we provide two case studies to demonstrate the application of IDseq. First, in a subset of samples from a previously published report which sought to investigate unknown etiologies of pediatric meningitis [<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>]. Secondly, we describe the performance of IDseq in the context of a real-world nasopharyngeal swab, processed and uploaded to IDseq from Phnom Penh, Cambodia, with respect to an emerging viral pathogen, SARS-CoV-2. By combining an intuitive web application, a cloud-based pipeline, and downstream visualization tools, IDseq enables investigation of mNGS data for pathogen detection and monitoring, especially suited for researchers with limited computational resources. Importantly, IDseq also enables facile collaboration and data sharing, while enhancing data analysis reproducibility across organizations and countries.</p></div><div class="section" id="sec-5"><h2 class="">IMPLEMENTATION</h2><div id="sec-6" class="subsection"><h3>IDseq Bioinformatics Pipeline</h3><p id="p-11">The IDseq Portal <a href="https://idseq.net">https://idseq.net</a> is a cloud-based, open-source bioinformatics platform that enables detection of microbial pathogens from raw next-generation sequencing (NGS) reads. The IDseq pipeline is conceptually based on previously implemented pipelines [<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">1</a>,<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>], but is optimized for scalable Amazon Web Services (AWS) cloud deployment (<strong><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a></strong>). Here, we describe v3.13 of the IDseq pipeline. Up-to-date pipeline documentation can be found at <a href="https://help.idseq.net">https://help.idseq.net</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A) Overview of the IDseq pipeline steps and data analysis workflow. The IDseq pipeline for pathogen discovery is composed of several steps, including host filtering and QC, assembly-based alignment, and taxonomic aggregation and reporting. Each step is comprised of a number of existing bioinformatics tools. B) The IDseq pipeline is optimized for AWS cloud computational infrastructure. Each of the core pipeline steps (host filtering and qc, assembly-based alignment, and taxonomic aggregation and reporting) is managed by EC2 Autoscaling Groups." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1081500886" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;strong&gt;A)&lt;/strong&gt; Overview of the IDseq pipeline steps and data analysis workflow. The IDseq pipeline for pathogen discovery is composed of several steps, including host filtering and QC, assembly-based alignment, and taxonomic aggregation and reporting. Each step is comprised of a number of existing bioinformatics tools. &lt;strong&gt;B)&lt;/strong&gt; The IDseq pipeline is optimized for AWS cloud computational infrastructure. Each of the core pipeline steps (host filtering and qc, assembly-based alignment, and taxonomic aggregation and reporting) is managed by EC2 Autoscaling Groups.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F1.medium.gif" width="440" height="327"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F1.medium.gif" width="440" height="327"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <p id="p-12" class="first-child"><strong>A)</strong> Overview of the IDseq pipeline steps and data analysis workflow. The IDseq pipeline for pathogen discovery is composed of several steps, including host filtering and QC, assembly-based alignment, and taxonomic aggregation and reporting. Each step is comprised of a number of existing bioinformatics tools. <strong>B)</strong> The IDseq pipeline is optimized for AWS cloud computational infrastructure. Each of the core pipeline steps (host filtering and qc, assembly-based alignment, and taxonomic aggregation and reporting) is managed by EC2 Autoscaling Groups.</p><div class="sb-div caption-clear"></div></div></div><p id="p-13">The IDseq pipeline ingests raw, short-read sequencing data (either RNA- or DNA-seq from any sample type), which can be uploaded from local sources via the web interface or the command line interface (CLI) or directly from Illumina’s BaseSpace platform. Sequence analysis proceeds through three main phases: 1) Host filtering and QC, 2) Assembly-based alignment, 3) Reporting and visualization (<strong><a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a></strong>).</p><div id="sec-7" class="subsection"><h4>Host Filtering and QC</h4><p id="p-14">The first phase of the pipeline begins with validation of input files (single- or paired-end .fastq or .fasta files). Currently, raw read files are arbitrarily capped at 150 million reads, a threshold that is, according to our experience, larger than most single metagenomic samples. Most mNGS samples processed for pathogen detection are sampled from a potentially infected host organism and thus the majority of sequencing reads derive from the host organism itself [<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>]. IDseq performs <em>a priori</em> subtraction of host sequences via STAR (Spliced Transcripts Alignment to a Reference) alignment of raw reads to a host-specific database [<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>]. IDseq is host-agnostic and allows researchers to select from several available hosts including human, mouse, pig, ticks, and mosquito, among others. For example, human host samples are aligned to the HG38 reference database (GCA_000001405.15), while mosquito samples are aligned to a combined collection of reference genomes from <em>Culex</em> and <em>Aedes</em> species as well as other diptera. Reads that align to the selected host genome are removed from the analysis. For hosts with well-annotated genomes, individual gene counts may be saved for offline transcriptome analysis, provided appropriate consent in the case of human subject research. Such host-based analyses have been shown to complement metagenomic analysis for pathogen detection [<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>]. For all host organisms, sequences for optional spike-in RNA controls developed by the External RNA Controls Consortium (ERCCs) are automatically recognized for downstream steps.</p><p id="p-15">Next, IDseq performs a series of Quality Control (QC) steps, as outlined in <strong><a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a></strong>. First, Trimmomatic [<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>] trims Illumina adapters. Low-quality reads, duplicates, and low-complexity reads are then removed using the Paired-Read Iterative Contig Extension (PRICE) computational package [<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>], the CD-HIT-DUP tool (v 4.6.8) [<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>], and a filter based on the Lempel-Ziv-Welch (LZW) compression score, respectively. Regardless of the host genome, the data is scoured to remove all remaining human sequences using Bowtie2 against the HG38 reference database [<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>] and gmap-gsnap against a more stringent database including sequences combining both HG38 and Chimpanzees (<em>Pan troglodytes</em>) [<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>]. This step is especially important in the case of vector research, where blood meals may contain human sequences. At each step, the total number of reads remaining in the analysis is computed and these basic QC metrics (including total non-host reads, % passing QC, and duplicate compression ratio) are provided both in the user interface, as well as via download.</p><p id="p-16">While the host filtering and QC steps performed by the IDseq pipeline serve primarily to reduce the computational burden and noise in downstream alignment steps, these metrics can also provide a resource for evaluating and troubleshooting sample preparation steps. The proportion of reads lost at each step may provide insight into possible sample degradation, fragment size, sequencing quality, or library complexity. IDseq’s automatic estimation of ERCC abundances enables back-calculation of the total input nucleic acid content, estimation of the lower limit of detection, and increases the ability to distinguish contaminants [<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>]. ERCC spike-ins are increasingly recognized as a best-practice for addressing the challenges associated with distinguishing background contamination from true microbial populations (Methods) [<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>].</p></div><div id="sec-8" class="subsection"><h4>Assembly-based Alignment</h4><p id="p-17">To assign taxonomic identities to each read, an assembly-based alignment procedure is used. First, filtered short-read sequences are aligned to the NCBI nucleotide (nt) and non-redundant protein (nr) databases (<a href="ftp://ftp.ncbi.nlm.nih.gov/blast/db/FASTA/">ftp://ftp.ncbi.nlm.nih.gov/blast/db/FASTA/</a>) using GSNAPL [<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">32</a>] and RAPsearch2 [<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>], respectively (<strong><a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1A</a></strong>). GSNAPL is a specialized instance of the gmap-gsnap package written by Tom Wu, intended for very large genome databases.</p><p id="p-18">The NCBI database indices are updated biannually, or as needed, via direct pull from NCBI. The index version is tracked for each pipeline run providing for versioned results. Putative accessions are assigned to each read using the NCBI accession2taxid database [<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>] and a BLAST+ (v 2.6.0) [<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>] database is constructed on-the-fly from the set of putative accessions (one database for each, nt and nr). In parallel, short reads are <em>de novo</em> assembled into contigs using SPADES [<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>]. Raw reads are mapped back to the resulting contigs using Bowtie2, in order to identify the contig to which each raw read belongs. Finally, each contig is aligned to the set of possible accessions represented by the BLAST database generated in the previous step, thereby improving the specificity of alignments to all the underlying reads, especially for homologous regions where short reads may align equally well to multiple different accessions.</p></div></div><div id="sec-9" class="subsection"><h3>AWS Cloud Infrastructure</h3><p id="p-19">IDseq is optimized for scalable Amazon Web Services (AWS) cloud deployment (<strong><a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1B</a></strong>). Bioinformatics data processing jobs are orchestrated by the IDseq pipeline directed acyclic graph (DAG, <a href="https://github.com/chanzuckerberg/idseq-dag">https://github.com/chanzuckerberg/idseq-dag</a>) and carried out on demand as Docker containers using AWS Batch. Alignments to the National Center for Biotechnology Information (NCBI) database are executed on dedicated auto scaling groups (ASG) of Amazon Elastic Compute Cloud (EC2) instances, with the number of server instances varied with job load. Fast downloads of the NCBI database from the Amazon Simple Storage Service to each new server instance are enabled by the open-source tool s3mi (<a href="https://github.com/chanzuckerberg/s3mi">https://github.com/chanzuckerberg/s3mi</a>).</p></div><div id="sec-10" class="subsection"><h3>Reporting and Visualization</h3><p id="p-20">Where alignments exist, taxonomic identifiers (taxID) for each of nt and nr, are assigned to each read. If there exist alignments with equivalent scores to multiple species taxIDs, then a single taxID is selected at random. If a read was incorporated into a contig, it is assigned the taxID belonging to the NCBI accession to whom its parent contig was assigned, as described above. If the read does not assemble into a contig, it is assigned the taxID of the NCBI nt and nr accessions it mapped to in the initial short-read alignment phase. The results are then aggregated to produce NT and NR counts for each taxID at both the species and genus level. Reads matching GenBank records in the superphylum Deuterostomia are removed, given the high likelihood that such residual reads are of host origin.</p><p id="p-21">The IDseq Portal provides a number of different methods for interpretation of the pipeline results (<strong><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a></strong>). First, relevant QC metrics and pipeline run information, including the number of reads remaining at each step of the host and quality filtering steps as well as estimates of internal control abundances are provided for each sample (<strong><a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2AB</a>, Methods</strong>). The single-sample report tables provide key metrics for each taxon identified in the sample, including the total number of reads aligning to the taxon (in both NT and NR) as well as contig stats from the assembly-based alignment step (<strong><a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a></strong>). The tree view enables rapid assessment of taxonomic relatedness of microbes identified in the sample (<strong><a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2D</a></strong>). For all views of the data, a wide range of user-selectable compound query and filtering tools are made available, enabling facile investigation of the data. For each taxonomic category, IDseq also provides one-click downloads of the corresponding underlying reads and contigs. Furthermore, coverage plots for contigs relative to all corresponding accessions to which they map are automatically generated (<strong><a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2F</a></strong>). To assist with distinguishing microbial signal from reagent and environmental contamination, IDseq supports background model generation, which allows researchers to evaluate the significance (reported in z-scores) of relative abundance estimates for taxons in samples of interest as compared to water-only or other environmental control sample collections. Altogether, the single sample report and associated filtering functionality enables evaluation of taxonomic hits. More documentation on specific metrics can be found at <a href="https://help.idseq.net">https://help.idseq.net</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The IDseq web application provides multiple easy-to-use visualizations to help the user assess the quality and content of their sample. Screenshots taken from the IDseq Portal correspond to the re-analysis of samples from a study of etiologies of pediatric meningitis originally published in Saha et al. 2019 (see section: Application I). CHRF_0002 and CHRF_0094 are CSF samples from pediatric patients with meningitis due to Streptococcus pneumonia and chikungunya virus, respectively. CHRF_0000 is a water control. A) Table of reads remaining during each step of the host filtration step (for CHRF_0094) - interpretation of the relative loss at each step in can provide insight into the quality of the library preparation and sequencing run. B) Automatic quantification of ERCC counts from sample CHRF_0094; ERCC quantification enables back-calculation of input RNA concentration. C) The results for a single sample (CHRF_0094) are presented as a table, with key metrics for interpreting taxon alignment quality. D) The tree view indicates the relative abundance of sequences and their taxonomic relationship within a particular sample, shown is the relative abundance of chikungunya virus reads in CHRF_0094. E) The results from multiple samples can be compared using the IDseq heatmap view, with associated metadata (purple = CSF, blue = water control). The interactive heatmap visualization can be viewed at https://idseq.net/zlfl1. The heatmap is especially powerful when analyzing trends across a larger number of samples. F) Coverage of chikungunya virus in CHRF_0094; the coverage visualization enables rapid interrogation of genome coverage." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1081500886" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The IDseq web application provides multiple easy-to-use visualizations to help the user assess the quality and content of their sample. Screenshots taken from the IDseq Portal correspond to the re-analysis of samples from a study of etiologies of pediatric meningitis originally published in Saha &lt;em&gt;et al&lt;/em&gt;. 2019 (see section: Application I). CHRF_0002 and CHRF_0094 are CSF samples from pediatric patients with meningitis due to &lt;em&gt;Streptococcus pneumonia&lt;/em&gt; and &lt;em&gt;chikungunya virus&lt;/em&gt;, respectively. CHRF_0000 is a water control. &lt;strong&gt;A)&lt;/strong&gt; Table of reads remaining during each step of the host filtration step (for CHRF_0094) - interpretation of the relative loss at each step in can provide insight into the quality of the library preparation and sequencing run. &lt;strong&gt;B)&lt;/strong&gt; Automatic quantification of ERCC counts from sample CHRF_0094; ERCC quantification enables back-calculation of input RNA concentration. &lt;strong&gt;C)&lt;/strong&gt; The results for a single sample (CHRF_0094) are presented as a table, with key metrics for interpreting taxon alignment quality. &lt;strong&gt;D)&lt;/strong&gt; The tree view indicates the relative abundance of sequences and their taxonomic relationship within a particular sample, shown is the relative abundance of &lt;em&gt;chikungunya virus&lt;/em&gt; reads in CHRF_0094. &lt;strong&gt;E)&lt;/strong&gt; The results from multiple samples can be compared using the IDseq heatmap view, with associated metadata (purple = CSF, blue = water control). The interactive heatmap visualization can be viewed at https://idseq.net/zlfl1. The heatmap is especially powerful when analyzing trends across a larger number of samples. &lt;strong&gt;F)&lt;/strong&gt; Coverage of &lt;em&gt;chikungunya virus&lt;/em&gt; in CHRF_0094; the coverage visualization enables rapid interrogation of genome coverage.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F2.medium.gif" width="440" height="404"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F2.medium.gif" width="440" height="404"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <p id="p-22" class="first-child">The IDseq web application provides multiple easy-to-use visualizations to help the user assess the quality and content of their sample. Screenshots taken from the IDseq Portal correspond to the re-analysis of samples from a study of etiologies of pediatric meningitis originally published in Saha <em>et al</em>. 2019 (see section: Application I). CHRF_0002 and CHRF_0094 are CSF samples from pediatric patients with meningitis due to <em>Streptococcus pneumonia</em> and <em>chikungunya virus</em>, respectively. CHRF_0000 is a water control. <strong>A)</strong> Table of reads remaining during each step of the host filtration step (for CHRF_0094) - interpretation of the relative loss at each step in can provide insight into the quality of the library preparation and sequencing run. <strong>B)</strong> Automatic quantification of ERCC counts from sample CHRF_0094; ERCC quantification enables back-calculation of input RNA concentration. <strong>C)</strong> The results for a single sample (CHRF_0094) are presented as a table, with key metrics for interpreting taxon alignment quality. <strong>D)</strong> The tree view indicates the relative abundance of sequences and their taxonomic relationship within a particular sample, shown is the relative abundance of <em>chikungunya virus</em> reads in CHRF_0094. <strong>E)</strong> The results from multiple samples can be compared using the IDseq heatmap view, with associated metadata (purple = CSF, blue = water control). The interactive heatmap visualization can be viewed at <a href="https://idseq.net/zlfl1">https://idseq.net/zlfl1</a>. The heatmap is especially powerful when analyzing trends across a larger number of samples. <strong>F)</strong> Coverage of <em>chikungunya virus</em> in CHRF_0094; the coverage visualization enables rapid interrogation of genome coverage.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">To facilitate visualization and hypothesis generation across multiple samples, the IDseq portal provides user customizable taxon heatmaps (<strong><a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2E</a></strong>). For advanced users, the pipeline visualization tool clearly documents the input parameters at each step of the analysis pipeline and provides download access to the input and output files at each step so data can be made available for offline analysis (<strong><a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure S1</a></strong>), such as phylogenetics.</p></div><div id="sec-11" class="subsection"><h3>Versioning and Development</h3><p id="p-24">IDseq is an open source software tool under continued development across two GitHub repositories – one which hosts the web interface (<a href="https://github.com/chanzuckerberg/idseq-web">https://github.com/chanzuckerberg/idseq-web</a>), and one which hosts the pipeline code (<a href="https://github.com/chanzuckerberg/idseq-dag">https://github.com/chanzuckerberg/idseq-dag</a>). Modifications to the web interface, which are deployed twice-weekly, do not affect the analysis results. To provide a record of features and how to use them, full documentation is provided at <a href="https://help.idseq.net">https://help.idseq.net</a>.</p><p id="p-25">Updates to the pipeline code may impact analysis results. Therefore, IDseq has adopted a semantic versioning system. Changes implemented to each version are listed in the README file. For each pipeline run, the pipeline and NCBI database versions are also tracked. Major changes to the pipeline outputs result in a major version number update (2.x to 3.x) and are communicated broadly to researchers via email updates. The change from IDseq v 2.x to v. 3.x involved the incorporation of the current assembly steps to refine alignment results, which improved the ability to resolve taxonomic identities in potentially homologous regions. Small changes to the pipeline that may still affect downstream results are indicated by an increased minor version number. For example, addition of a minimum alignment length filter to improve specificity of NT alignments caused a version change from 3.9.4 to 3.10.0. Changes to the pipeline which do not affect the results are indicated by incremental minor version (i.e. 3.13.1 to 3.13.2).</p><p id="p-26">Continued development on IDseq aims to 1) improve the computational efficiency and accuracy of the results; 2) expand the integration with other tools to enable researchers’ flexibility in the downstream analysis of their processed results; 3) support the expanding number of mNGS sequencing platforms that will be used by researchers for pathogen detection globally. A suite of benchmarking samples are used for analysis of additional pipeline updates as discussed below.</p></div><div id="sec-12" class="subsection"><h3>Software and Data Availability</h3><p id="p-27">Additional documentation and guides for getting started with IDseq can be found at <a href="https://help.idseq.net">https://help.idseq.net</a>. The code is open source and available in the GitHub repositories listed in <strong>Table S1</strong>.</p></div></div><div class="section" id="sec-13"><h2 class="">RESULTS</h2><div id="sec-14" class="subsection"><h3>Evaluation of IDseq on External Benchmark Datasets</h3><div id="sec-15" class="subsection"><h4>IDseq Analysis of Unambiguously Mapped Datasets</h4><p id="p-28">A recent study evaluated the performance of 20 taxonomic classifiers for mNGS data on ten reference datasets that are commonly used for benchmarking, containing computationally simulated reads from between 12 and 525 bacterial species [<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>]. It evaluated performance using two metrics - the area under the precision recall curve (AUPR) and the L2 distance. The AUPR evaluates the ability to detect the presence of microbes (binary presence/absence) above a relative abundance threshold, taking into consideration the precision and recall rates as said threshold is adjusted. A species-level AUPR of 1.0 indicates that there is a threshold (proportion of reads) above which all true positive species can be identified with no false positive species. The L2 distance provides a complementary metric that considers the similarity in relative abundances between the results and the ground truth.</p><p id="p-29">We evaluated the performance of the IDseq pipeline on these same datasets (<strong>Methods, Table S2</strong>). Samples took an average of 3 hours (min = 1.6 hours, max = 10 hours) to process on IDseq pipeline version 3.13, with the NCBI database version from September 2019. Performance metrics (AUPR and L2 distance) were computed separately for the NCBI nt and nr results and compared to those published recently by Ye <em>et al</em>. (idseq_nt and idseq_nr, <strong><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a></strong>) [<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">39</a>]. IDseq provides an automated pipeline, but at the cost of inability to easily swap in new databases. Therefore, we compared our results against those reported by Ye <em>et al</em>. for the “default database” of other tools. The performance metrics may inherit biases due to differences in the reference database contents as well as recency of input sequences.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Performance metrics calculated for IDseq (NT and NR), as compared to the values recently published by Ye et al. [39] A) Area under the precision recall curve (AUPR) and L2 distance values for 22 tools, as evaluated against their default databases. B) The AUPR values for specific benchmark datasets evaluated for three tools (Kraken2, IDseq NT, and IDseq NR), including metrics obtained when evaluating basic threshold filters integrating both IDseq NT and NR (idseq_ntnr). C) The precision and recall of the same three tools for detecting known taxa." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1081500886" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Performance metrics calculated for IDseq (NT and NR), as compared to the values recently published by &lt;em&gt;Ye et al&lt;/em&gt;. [39] &lt;strong&gt;A)&lt;/strong&gt; Area under the precision recall curve (AUPR) and L2 distance values for 22 tools, as evaluated against their default databases. &lt;strong&gt;B)&lt;/strong&gt; The AUPR values for specific benchmark datasets evaluated for three tools (Kraken2, IDseq NT, and IDseq NR), including metrics obtained when evaluating basic threshold filters integrating both IDseq NT and NR (idseq_ntnr). &lt;strong&gt;C)&lt;/strong&gt; The precision and recall of the same three tools for detecting known taxa.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F3.medium.gif" width="440" height="312"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F3.medium.gif" width="440" height="312"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <p id="p-30" class="first-child">Performance metrics calculated for IDseq (NT and NR), as compared to the values recently published by <em>Ye et al</em>. [<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">39</a>] <strong>A)</strong> Area under the precision recall curve (AUPR) and L2 distance values for 22 tools, as evaluated against their default databases. <strong>B)</strong> The AUPR values for specific benchmark datasets evaluated for three tools (Kraken2, IDseq NT, and IDseq NR), including metrics obtained when evaluating basic threshold filters integrating both IDseq NT and NR (idseq_ntnr). <strong>C)</strong> The precision and recall of the same three tools for detecting known taxa.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h4>Deep Dive of Unambiguously Mapped Datasets Results</h4><p id="p-31">The IDseq pipeline demonstrated comparable performance to the other mNGS tools tested (<strong><a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a></strong>). The unambiguously mapped datasets demonstrated limited resolution for distinguishing the tools when evaluated by AUPR and L2, as most tools show relatively high performance (with AUPR scores above 0.8 at the species level, <strong><a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a></strong>). Consistent with Ye <em>et al</em>, we observed that the greatest differences between tools was in the reduced precision at high recall. IDseq protein alignments (NR) demonstrated greater AUPR than IDseq nucleotide (NT) across most datasets, but consistently identified more taxa at low abundance (less than 1%), therefore resulting in reduced precision (<strong><a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a></strong>). Meanwhile, IDseq NT exhibited increased specificity. IDseq NT and NR had a mean AUPR across all the datasets of 0.9627 and 0.9633, respectively. The top mean AUPR of any single tool was achieved by metaothello (0.9661), followed by Kraken2 (0.9635). Given Kraken2’s performance on the unambiguous benchmark datasets and its wide adoption for relative microbial abundance estimation, additional analyses focused on comparison against Kraken2 (<strong><a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3BC</a></strong>). Another distinguishing factor between the tools was in the number of reads that were “unclassified” across multiple datasets. mmseq2, metaothello, kaiju, and bracken consistently left &gt; 10% of reads “unclassified”. IDseq (NT and NR) removed an average of 10% of the reads during host filtering and QC steps, but of the remaining sequences, an average of less than 1% of reads were unmapped across the ten datasets. This can be attributed in part to IDseq always assigning reads to a species when an alignment exists (increasing sensitivity at the expense of specificity) and secondly to the use of assembled contigs to refine alignments where short reads may have been unmapped.</p><p id="p-32">To further investigate differences between the tools, we evaluated the results for each dataset independently (<strong><a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3B</a></strong>). IDseq NR demonstrated lower precision across all datasets than many other tools, including IDseq NT and Kraken2 (<strong><a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3C</a>, <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure S2</a></strong>). The ATCC Staggered dataset, which includes several microbes present at very low abundance, yields the lowest AUPR of all samples tested via IDseq NR, consistent with findings in Ye <em>et al</em>. that protein-based classifiers consistently struggled to identify the low-abundance taxa amongst other low-abundance false positives. Meanwhile, IDseq NT demonstrated reduced performance on the NYCSM dataset (Supplemental Text). IDseq’s usage of the full NCBI nt and nr databases resulted in relatively high performance for the Buccal dataset. Ye <em>et al</em>. discuss that the Buccal dataset was a low-performing outlier for most evaluated classifiers due to inclusion of reads from a species with only contig-quality reference, which is not included in most default databases.</p><p id="p-33">The IDseq web portal is designed to provide researchers with the choice of utilizing either NT or NR results, or both in conjunction with each other. For example, the impact of spurious NR alignments can be mitigated by requiring a corresponding alignment with IDseq NT. Using this strategy, the performance of IDseq was evaluated, considering the NT relative abundances reported for taxa with both NT r &gt; 0 and NR r &gt; 0 (idseq_ntnr, <strong><a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure S2</a></strong>). We observed that requiring concordance resulted in the greatest mean AUPR across all other tested tools (0.9673) and increased the precision of IDseq above that of either NT or NR alone.</p><p id="p-34">Altogether, these results highlight some key trade-offs with respect to relative abundance estimation of bacterial species. IDseq is capable of identifying organisms with respect to the latest versions of NCBI and demonstrates relatively high recall (<strong><a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3D</a></strong>). But use of the full NCBI database may result in false-positive alignments at low abundance which can reduce precision (<strong><a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3C</a></strong>). In the context of pathogen-identification, it has been observed that infecting agents may comprise the majority of sequencing reads in certain circumstances [<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a>]. For such data sets, the reduced precision for abundance estimation at low levels is less impactful. Meanwhile, researchers interested in evaluating highly complex microbiome composition at the species- and strain-level may need to bring in other tools to supplement their analyses [<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>–<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>] or rely on genus-level estimates provided by IDseq.</p></div></div><div id="sec-17" class="subsection"><h3>Evaluation of IDseq on Internal Benchmark Datasets</h3><p id="p-35">To address the gaps between the existing benchmark datasets and the IDseq pipeline’s primary use-case for pathogen detection, we tested IDseq’s performance on three additional datasets specifically designed to evaluate detection of divergent viruses (<strong>Methods</strong>) and common clinical microbes (<strong>Supplemental Text</strong>). For each dataset, we evaluated the performance of IDseq (NT and NR), as compared to Kraken2 [<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>], using per-species recall.</p><div id="sec-18" class="subsection"><h4>Detection of Divergent and Novel Viruses</h4><p id="p-36">Viruses are known to evolve rapidly and therefore their sequences may diverge from sequences in the known NCBI database over relatively short timescales [<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>]. Maintaining the ability to detect divergent viruses is of paramount concern, given their role in numerous recent outbreaks, including the recent emergence of SARS-CoV-2, the coronavirus responsible for the COVID-19 outbreak [<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>–<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>]. The idseq-bench tool was used to generate 17 simulated NGS samples from Rhinovirus C genomes at varying levels of divergence (after in-silico forward evolution from a reference sequence obtained from the NCBI database), ranging from 100% identical to the reference sequence to 25% similar (at the nucleotide level) (<strong>Methods, <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>, Table S3</strong>). The resulting samples were uploaded to IDseq (Project HRhinoC Simulation). Meanwhile, the same raw .fastq files (prior to host filtering), were analyzed using Kraken2 (<strong>Methods</strong>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A) Graphic representation of genomic similarity for simulated divergent Rhinovirus C genomes, at 95%, 75%, and 50% similarity to reference sequence NC_009996.1. Mutations are shown in dark blue. B) Performance of IDseq (NT and NR) as compared to Kraken2 for recovery of reads from simulated divergent Rhinovirus C genomes at varying levels of divergence. The dotted yellow line indicates the theoretical limit for detection of Rhinovirus C achieved by manual BLASTx of IDseq-produced contigs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1081500886" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;strong&gt;A)&lt;/strong&gt; Graphic representation of genomic similarity for simulated divergent Rhinovirus C genomes, at 95%, 75%, and 50% similarity to reference sequence NC_009996.1. Mutations are shown in dark blue. &lt;strong&gt;B)&lt;/strong&gt; Performance of IDseq (NT and NR) as compared to Kraken2 for recovery of reads from simulated divergent Rhinovirus C genomes at varying levels of divergence. The dotted yellow line indicates the theoretical limit for detection of Rhinovirus C achieved by manual BLASTx of IDseq-produced contigs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F4.medium.gif" width="440" height="290"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F4.medium.gif" width="440" height="290"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4:</span> <p id="p-37" class="first-child"><strong>A)</strong> Graphic representation of genomic similarity for simulated divergent Rhinovirus C genomes, at 95%, 75%, and 50% similarity to reference sequence NC_009996.1. Mutations are shown in dark blue. <strong>B)</strong> Performance of IDseq (NT and NR) as compared to Kraken2 for recovery of reads from simulated divergent Rhinovirus C genomes at varying levels of divergence. The dotted yellow line indicates the theoretical limit for detection of Rhinovirus C achieved by manual BLASTx of IDseq-produced contigs.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">Both IDseq NT and Kraken2 identified reads aligning to Rhinovirus C down to 75% sequence divergence (<strong><a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a></strong>). Meanwhile, IDseq NR recalled Rhinovirus C alignments down to 70% sequence divergence, demonstrating a greater sensitivity for divergent virus detection. We note that IDseq NR experienced a rapid drop in total recall (8,558 reads correctly mapping to Rhinovirus C, of 10,000 total and 8,558 passing QC steps at 70% sequence similarity vs. 0 reads detected at 65% sequence similarity). This highlights an artifact of the computational cost-saving mechanisms employed by IDseq - whereby a BLAST database is constructed from only the subset of accessions identified in the initial short-read GSNAP and Rapsearch2 alignments to the NCBI database. In cases where the highly divergent short-read sequences don’t match to NT or NR in the initial alignment, the BLAST database will be empty and none of the reads or contigs will map. However, IDseq does provide the ability to download all assembled contigs, enabling offline interrogation of this divergent “dark matter”. Manual BLASTx of contigs assembled by SPADES in IDseq to the full NCBI database, was able to recover the Rhinovirus C identity down to 55% sequence identity. Future iterations of the IDseq pipeline may aim to automate the manual follow-up steps for divergent viral contigs as well as incorporating other tools for dark matter investigation to probe for pathogen motifs.</p><p id="p-39">Further comparison of the IDseq (NT and NR) results to Kraken2 shows that Kraken2 initially recovered more of the simulated reads than IDseq (9,964 of 10,000 vs. 8,582 for both IDseq NT and NR). This is explained by the QC steps in the IDseq pipeline, which removed ∼15% of reads at the PriceSeq filtering step due to low quality - an expected outcome given that the simulated reads mimic error models of Illumina sequencers (<strong>Methods</strong>). Of the reads remaining after host filtering, IDseq identified 100% as aligning to Rhinovirus C. This pattern persists down to 95% sequence similarity, at which point Kraken2 begins to identify fewer reads. While some Rhinovirus C reads are identified down to 75% sequence similarity (same as IDseq NT), IDseq NT identified a significantly greater number of reads mapping to Rhinovirus C at increasing levels of divergence. Specifically, at 80% divergence, 8,544 reads mapped by IDseq NT while only 1,042 reads mapped by Kraken2. Altogether, these benchmark results are consistent with existing reports of the utility of IDseq NR in detecting divergent viruses [<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>] and are within the ranges of nucleotide divergence associated with emerging human pathogens (<strong>Supplemental Text</strong>).</p></div></div><div id="sec-19" class="subsection"><h3>APPLICATION I. IDseq for Pathogen Discovery in Cases of Pediatric Meningitis</h3><p id="p-40">The IDseq pipeline is sample-type agnostic, allowing researchers interested in a broad range of scientific questions across a diverse array of host organisms (humans, mice, mosquitos, ticks, plants, environmental, etc.) to obtain relevant microbial information from any sample type (blood, CSF, respiratory fluids, tissue, etc.) [<a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">1</a>,<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>–<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>]. There are many challenges for data interpretation that are common across mNGS applications, such as impact of PCR amplification on samples with low amounts of input RNA, background contamination, and genomic similarity between short regions of related organisms. Here, through a re-analysis of the IDseq results for three cerebrospinal fluid (CSF) samples from a recent study investigating etiologies of pediatric meningitis in Bangladesh [<a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">1</a>], we highlight specific IDseq features to address these challenges. The original study, conducted by Saha <em>et al</em>. included 91 CSF samples (36 positive, 30 negative, and 25 idiopathic) and 6 water controls, processed on IDseq v3.1. We focus on one known infection (<em>Streptococcus pneumoniae</em>, CHRF_0002), one idiopathic sample that was later confirmed to have chikungunya virus (CHRF_0094), and one water control (CHRF_0000) (<strong><a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a></strong>). These samples, for demonstrative purposes, were re-run on IDseq v3.13 and are available in IDseq project CHRF RR007 Example. Key pipeline run metrics for these three samples are provided in <strong><a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a></strong>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1243634/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1243634/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1243634" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1:</span> <p id="p-41" class="first-child">IDseq provides key metrics enabling The Host filtering and QC stage of the IDseq pipeline is composed of several individual steps. The proportion of reads lost at each step can provide insight into sample quality and library preparation. Interpretation of these metrics may be valuable for labs evaluating new sample storage techniques, library preparation protocols, etc. These three samples, provided as an example, can be investigated in the IDseq portal in Project <span class="underline">CHRF RR007 Example</span>. CSF: Cerebral Spinal Fluid, DCR: duplicate compression ratio, “number of rows”: total number of species and genus-level rows in the IDseq sample report, NT: results based on NCBI nucleotide (nt) database, NR: results based on NCBI non-redundant protein (nr) database, rPM: reads per million, L: average alignment length across all reads and contigs mapping to that taxon.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-20" class="subsection"><h4>Sample 0094: A case of neuroinvasive chikungunya virus</h4><p id="p-42">CHRF_0094 was a pediatric encephalitis case of unknown etiology that was later determined to be a case of neuroinvasive chikungunya virus. <strong><a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2A</a></strong> shows the number of reads removed by each host filtration and QC step. One challenge for mNGS-based pathogen detection is that host sequences dominate the mNGS library. Notably, in CHRF_0094, chikungunya virus reads in sample CHRF_0094 represented less than 1% of the total sequencing reads. However, after IDseq’s host filtering and QC steps, it represented 63% of the remaining non-host reads. A second, widely acknowledged challenge for mNGS data interpretation is the presence of environmental contaminants. Best-practices suggest including at least one water control with every sequencing experiment [<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a>,<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>]. To assist with interpretation of results with respect to control samples, IDseq implements a z-score approach (<strong>Methods</strong>) first described in Wilson <em>et al</em>. [<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>]. Z-score statistics computed by IDseq indicate the significance of relative abundance estimates in a sample as compared to the user-selected background controls - which may include water controls or healthy control samples. Z-score thresholds can be imposed to remove taxa that are prevalent in the water or healthy controls. In sample CHRF_0094, 8 rows (4 species from 4 genera) were reported with NT reads per million (rPM) greater than 10, NR rPM &gt; 10, and z-score &gt; 1 (a relatively stringent threshold employed to remove many of the low-abundance taxa for first-pass evaluation). 7,876.2 rPM were associated with chikungunya virus, of which many were associated with the 4 contigs aligning to chikungunya virus. By using the IDseq portal coverage visualization, which displays reads and contigs in association with their top matched GenBank accession, we observe that the longest contig, approximately 11kb, represented full-genome coverage of the nearest GenBank accession (<strong><a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2F</a></strong>).</p></div><div id="sec-21" class="subsection"><h4>Sample 0002: A case of known Streptococcus pneumoniae meningitis</h4><p id="p-43">In sample CHRF_0002, IDseq associated 1,927,505 reads by NT with the independently verified pathogen, <em>Streptococcus pneumoniae</em>, of which 98.1% were assembled into 143 contigs (<strong><a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a></strong>). The average alignment length across all contigs and reads was 75,289.8 bp - driven largely by alignment of long contigs. Despite the large number of contigs and long alignment lengths, the GenBank accession with the greatest coverage (1.8mb LR216026.1 Streptococcus pneumoniae strain 2245STDY5775485 genome assembly, chromosome: 1) had 87.3% coverage breadth. This exemplifies a frequently observed pattern (which is even more pronounced in lower-coverage samples) - whereby coverage of larger bacterial genomes is lower than virtual genomes even for samples with a high proportion of mNGS reads associated with a particular microbe. For many cases, low coverage from mNGS data can preclude confident strain identification in bacterial species that may be useful in a clinical context. Furthermore, low coverage of the transcriptome (via RNA mNGS) may produce a large proportion of alignments in conserved rRNA regions which may be challenging to disambiguate.</p></div><div id="sec-22" class="subsection"><h4>Sample 0000: A water control</h4><p id="p-44">In sample CHRF_0000, the duplicate compression ratio (DCR) of 4.67 indicates the possibility of over-amplification of low biomass nucleic acid input (<strong><a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a></strong>). This is common for water samples where low input nucleic acid is expected. The use of ERCC controls in the library preparation of these samples enabled back-calculation of the total input RNA concentration. This sample was determined to have 3.7 pg of total input RNA, while the two infected samples (CHRF_0094 and CHRF_0002) had 29.6pg and 213.6 pg, respectively. Thus, while the relative abundance values appear comparable to those in the infected sample, they represent significantly smaller quantities of raw nucleic acid (<strong><a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2E</a></strong>). In the original study all water and non-infectious controls (which had low white cell counts and therefore little host or pathogen nucleic acid) had input RNA quantities &lt; 4 pg, enabling the use of an input nucleic acid threshold for inclusion in downstream analyses. Additionally, the top four organisms (by NT rPM) include <em>Providencia, Cutibacterium, Streptococcus</em>, and <em>Escherichia</em> - many of which are known environmental contaminants [<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">33</a>,<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>,<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>]. The 36 total rows (with NT rPM &gt; 10, NR rPM &gt; 10, and NT z-score &gt; 1) are all present at relatively similar and low abundance levels, characteristic of background contaminants [<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">33</a>,<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">34</a>].</p></div></div><div id="sec-23" class="subsection"><h3>APPLICATION II. Real-time Detection of Novel Coronavirus</h3><p id="p-45">IDseq is a globally accessible pipeline for mNGS analysis that has been shown through simulation and practice to be effective in identifying novel and divergent viruses. As an additional real-world example of this utility, we provide a vignette from the recent SARS-CoV-2 coronavirus outbreak. On January 30, 2020, a team of researchers from the CNM-NIAID (National Center for Parasitology, Entomology, and Malaria Control - National Institute of Allergy and Infectious Disease) collaboration in Cambodia obtained a nasopharyngeal swab sample from a patient with PCR-confirmed SARS-CoV-2 infection. The library preparation and sequencing were completed in-country by February 1, 2020 [<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>]. Analysis of the sample (4.5 million single reads) using IDseq against an NCBI database version from 2019-09-17, which did not contain the known sequences for SARS-CoV-2 that have since been deposited on NCBI, identified 571 reads aligned to the genus <em>Betacoronvirus</em>, with an average amino acid percent identity of 92.3% (by NR). The sample took 14 minutes to analyze end-to-end and the most abundant species was severe acute respiratory syndrome-related coronavirus, with 542 NT reads (22 contigs) and 571 NR reads (24 contigs), representing ∼33% genome coverage. To quantify the IDseq pipeline’s recall for SARS-CoV-2 sequences, we built a BLAST database from the 54 sequences associated with SARS-CoV-2, which had been deposited in NCBI between January and February 2, 2020 as a result of widespread efforts by the global science community. By BLASTing all non-host reads from the sample against the known SARS-CoV-2 sequence database, we identified 584 reads mapping to SARS-CoV-2. As compared to this ground-truth value, IDseq demonstrated 97.8% read-level recall. This indicates that for an emerging threat, IDseq was able to successfully provide information on the presence of a pathogen prior to the existence of full reference genomes associated with the organism. This identification was of paramount public health importance given unclear diagnostic accuracy in the beginning pre-pandemic state.</p></div></div><div class="section" id="sec-24"><h2 class="">DISCUSSION</h2><p id="p-46">We have introduced IDseq, an open source cloud-based pipeline and analysis service for metagenomic pathogen detection and monitoring. We described the pipeline analysis steps and demonstrated that the IDseq pipeline achieves comparable performance for taxonomic identification and relative abundance estimation as other tools in the field. We showed that IDseq is uniquely suited for detection of divergent viruses and has high sensitivity for detecting human pathogens. Finally, we have shown through two case-studies, how the IDseq portal enables researchers to rapidly generate insights into their samples’ quality, microbial content, and cohort trends. We further highlighted its real-time utility by describing how IDseq was used to analyze sequences associated with the emerging coronavirus SARS-Cov-2 prior to deposition of SARS-CoV-2 sequences into public data repositories. The IDseq web portal provides an easy-to-use access point for computationally intensive analysis of mNGS data. Its sample-type agnostic implementation enables its application for a broad range of research questions related to understanding distribution of microbes in a sample. The IDseq pipeline has been a key component in recent studies to understand undiagnosed causes of infection and survey the landscape of circulating pathogens, both in humans and animals [<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>,<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>].</p><p id="p-47">Benchmarking of mNGS tools is a well-recognized challenge within the field [<a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">39</a>,<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>,<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a>]. The choices of tools, parameters, databases, and datasets may all influence the conclusions. Our aim in this study was simply to test performance relative to other tools. We compare IDseq’s default database (NCBI nt and nr) against the default databases for all other tools included by Ye <em>et. al</em>. Though it is possible that other tools’ performance would improve given a comparably large database, configuring these details requires computational expertise that directly opposes the readily usable nature of IDseq. IDseq continues to use the full NCBI nt and nr database given their advantages for detecting divergent viruses and incorporating data on novel bacterial pathogens. However, the large database size results in longer run-times and the lack of curation induces the potential for noise in alignment results due to errant sequence assignment errors upon upload to the NCBI databases. There is ongoing work by many researchers to evaluate curated databases for mNGS analyses, but for now IDseq continues to update its database biannually. To support continued benchmarking of IDseq and empower researchers to test IDseq’s performance for their particular applications, we have released the open-source idseq-bench tool, which was used to generate the divergent virus dataset and for evaluating the per-read recall results.</p><p id="p-48">Beyond the informatics nuances between tools, IDseq provides clear advantages for researchers new to mNGS and computational data analysis. First, IDseq is designed and maintained by a team of engineers and managed as a software-as-a-service product, where user support is a key component. User support enables researchers to have confidence that they will obtain results in a timely fashion. Secondly, the tool’s user interface provides a series of advantages for users with limited computational expertise by reducing the challenges associated with installation and configuration as well as providing meaningful metrics for quality control and interpretation. It maintains transparency on individual pipeline steps through documentation (<a href="https://help.idseq.net">https://help.idseq.net</a>), the pipeline visualization tool (<strong><a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure S2</a></strong>), and availability of downloads from intermediate files. Together, these resources help researchers new to mNGS get started quickly, while also providing tools to enhance skills in computational biology. Thirdly, the pipeline provides assurance of computational reproducibility, which is an increasingly appreciated priority within the scientific community as dataset sizes and analytical complexity increase. Lastly, the web-based user interface provides an access point for collaboration and networking – enabling researchers to collaborate seamlessly across countries and institutions, thereby building global networks of expertise that can be accessed by those in resource-scarce settings.</p><p id="p-49">Finally, we highlight that IDseq is not a clinical tool and intended for research-only purposes. IDseq aims to be a valuable resource for researchers in the infectious diseases field but does not intend to become clinically validated. While IDseq can yield insights that inform public health policies, laboratory testing priorities, and real-time decisions for confirmatory clinical testing, clinical validation of the pipeline requires locking of the system for adherence to strict guidelines. IDseq will remain under continued development in order to 1) improve the computational efficiency and accuracy of the results; 2) expand the integration with other tools to enable researchers’ flexibility in the downstream analysis of their processed results; and 3) support the expanding number of mNGS sequencing platforms that will be used by researchers for pathogen detection globally. Some possible future directions for improvements to the IDseq pipeline have been discussed throughout. Notably, IDseq’s current assembly-based alignment steps results in failure to automatically identify divergent viruses beyond 70% divergent, while BLASTx of IDseq-generated contigs can enable detection down to 55% divergent. Automating full NCBI BLASTx of putative viral contigs would simplify offline analyses. Similarly, we showed that IDseq NR had reduced precision, which made relative abundance estimation of low-abundance taxa challenging. Allowing for non-species-specific mappings or propagating estimates of species-level ambiguity to increase species-level resolution for low-abundance taxa may provide another avenue for continued development. Finally, continued integration with other analysis tools and sequencing technologies will further enhance the usability of IDseq for mNGS data analysis.</p><p id="p-50">IDseq reduces the need for much of the computational expertise and access to large-scale computing resources that have traditionally been barriers for conducting mNGS data analysis. The IDseq portal provides an easy-to-use interface that enables researchers around the world to upload samples and generate hypotheses with relevant implications for global health and infectious disease tracking as diseases emerge.</p></div><div class="section" id="sec-25"><h2 class="">METHODS</h2><div id="sec-26" class="subsection"><h3>Raw Pipeline Commands</h3><p id="p-51">The IDseq pipeline uses several publicly available academic bioinformatics tools. The raw commands and parameters used for each step in the pipeline are available for each pipeline version in the pipeline visualization (<strong><a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure S1</a></strong>), which can be viewed for any sample in IDseq. Technical documentation is available here: <a href="https://github.com/chanzuckerberg/idseq-dag/wiki">https://github.com/chanzuckerberg/idseq-dag/wiki</a>.</p></div><div id="sec-27" class="subsection"><h3>Automatic ERCC Quantification</h3><p id="p-52">The External RNA Controls Consortium (ERCC) developed a common set of external RNA controls that can be used to control for a variety of sources of variation on RNA expression attributed to experimental factors (including the quality of the starting material, the level of cellularity and RNA yield, the sequencing platform, and the person performing the experiment). In the context of pathogen detection, mNGS libraries often contain extremely low quantities of RNA input. It has been shown that during library preparation, samples with low input experience amplification background contaminants [<a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">34</a>]. ERCC controls can be used to mitigate the effect of low input libraries and to quantify the total input. To enable researchers to rapidly assess the quality of their libraries and the limit of detection, IDseq provides ERCC counts for each sample. During the host filtering steps, the raw sequencing reads are aligned to the ERCC reference sequences and counts are generated by STAR –genecounts option [<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a>]. These values are then available for download, as well as visualized in the user interface (<strong><a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2B</a></strong>).</p></div><div id="sec-28" class="subsection"><h3>IDseq Z-score and Aggregate Score Metrics</h3><p id="p-53">Given the sensitivity of mNGS, it is common to identify contaminating microbial sequences derived from laboratory contaminants, reagents, collection tubes, etc. There exist numerous approaches to assist in distinguishing background contaminants from true microbes [<a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">34</a>,<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">53</a>,<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a>]. IDseq implements a previously described z-score method for background correction [<a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">53</a>]. Researchers can create a background model by selecting control samples sequenced via their standard laboratory protocols or select from a default set of publicly available water controls. From the selected set of samples, the distribution of reads for each taxon is computed. The z-score field of the IDseq sample report is calculated as the z-score for each taxonomic ID based on its prevalence in the selected background model. If a particular taxonomic ID is not found in the set of control samples, then the z-score will be set to 100. If the taxonomic ID is not found in the sample, the z-score will be set to -100. The z-score metric also feeds into the “aggregate score”, which combines information from NT rPM, NR rPM, NT z-score, and NR z-score to provide an estimate of “microbial importance” for a particular sample based on the relative abundance both with the sample as well as in the background. This experimental metric aims to rank rare organisms that may be implicated in an infection higher, even if they are present only at low abundance.</p></div><div id="sec-29" class="subsection"><h3>External Benchmarks - Datasets and Metrics</h3><p id="p-54">Datasets evaluated by Ye <em>et al</em>. in their benchmark analysis of 20 mNGS tools were downloaded from &lt;gs://metax-bakeoff-2019&gt;. The raw .fastq files were uploaded to IDseq (<strong>Table S2</strong>). The truth files for each of the datasets were obtained from <a href="https://github.com/yesimon/metax_bakeoff_2019">https://github.com/yesimon/metax_bakeoff_2019</a> and are available in the <em>Notes</em> field of the IDseq metadata. The code developed by Ye <em>et al</em>. was downloaded from the GitHub repository. IDseq sample reports were downloaded upon completion and processed to produce species-level relative abundance estimates for each sample – specifically, the proportion of total reads (by NT and NR) was computed and used as input to the script. The IDseq results were processed in parallel with the data analyzed for the Ye <em>et al</em>. paper. The scripts used to run this analysis are available here <a href="https://github.com/katrinakalantar/idseq-benchmark-manuscript">https://github.com/katrinakalantar/idseq-benchmark-manuscript</a>. Modifications to the original script are annotated as “##IDseq EDIT”. The computed metrics (AUPR, L2 distance, precision, recall, and f1-score) were then output as .csv files and plotted (<strong><a id="xref-fig-3-11" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure S3</a></strong>).</p></div><div id="sec-30" class="subsection"><h3>idseq-bench: IDseq Benchmarking Tool</h3><p id="p-55">The idseq-bench tool (<a href="https://github.com/chanzuckerberg/idseq-bench">https://github.com/chanzuckerberg/idseq-bench</a>) was developed as a resource to enable the IDseq team to benchmark datasets internally [<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a>]. The tool is open source and available for external users to generate benchmarks appropriate for their particular use case. Full documentation can be found on GitHub. Briefly, the tool enables users to simulate NGS sequencing data from known microbes. By indicating the GenBank reference accession, idseq-bench uses the InSilicoSeq simulation tool [<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a>] to generate reads in accordance with known sequencing error models. The true organism from which each read was simulated contains a tag indicating the known accession and species-, genus-, and family-level taxonomic IDs. The idseq-bench tool then uses this information to characterize performance of the IDseq pipeline results. The tool provides metrics for read-level-recall at the species-, genus-, and family-level, as well as sample-level AUPR, L2, precision, recall, etc. For samples that were not simulated internally, the tool enables users to supply a gold standard file (comparable to those obtained for the Cell Benchmarks Datasets) and compute sample-level metrics against that file.</p></div><div id="sec-31" class="subsection"><h3>Internal Benchmarks - Divergent Virus Simulation and Analysis</h3><p id="p-56">A reference genome for Rhinovirus C (RefSeq NC_009996.1) was identified and the associated coding sequence .fasta file was downloaded from RefSeq. VIRAPOPS forward viral simulation [<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a>] was used to simulate 5000 generations of viral evolution using default parameters. From the simulated data, sequences were selected at intervals of 5% nucleotide sequence identity to the original reference and compiled into a fasta file. This was then used as input to the idseq-bench simulation tool for benchmark simulation, which used InSilicoSeq [<a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">63</a>] to simulate 10,000 sequencing reads of length 126 for each divergent virus genome according to a HiSeq error model. This resulted in 195.8x coverage of each divergent viral genome, consistent with the relatively high coverage of viral genomes seen by IDseq analysis of samples with high viral load. The simulated fastq files were then uploaded to IDseq project HRhinoC Simulation (Samples HRC_100, HRC_99, HRC_95, … HRC_025, <strong>Table S3</strong>).</p><p id="p-57">To evaluate the limit of detection for divergent viruses, the total recall of Rhinovirus C reads was evaluated at each level of simulated divergence, for each tool. Additionally, the number of reads aligning to false-positive species was tracked. Offline analysis was done using the contigs generated by IDseq for samples where IDseq failed to identify Rhinovirus C. For simulated samples HRC_070 through HRC_025, the “unmapped contigs” were downloaded and aligned via BLASTx in the NCBI BLAST web interface using default parameters [<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">37</a>]. Samples for which the BLASTx result returned Rhinovirus C were marked as “potentially possible” and the greatest level of divergence was recorded.</p></div><div id="sec-32" class="subsection"><h3>Internal Benchmarks - Running Kraken2</h3><p id="p-58">To compare internal benchmark samples against Kraken2 [<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>], a Kraken2 database was generated from the NCBI NT sequence database [<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a>]. The following command line parameters were used to download and build the reference database. Finally, simulated sequencing files were run via the following commands.</p><p id="p-59">Download the NCBI Database:</p><p id="p-60">kraken2-build --download-library nt --db db_ncbi_nt</p><p id="p-61">Build the Kraken2 NCBI Database:</p><p id="p-62">kraken2-build --build --db db_ncbi_nt --threads 8</p><p id="p-63">Run Kraken2 on benchmark datasets:</p><p id="p-64"># classify: running kraken changes slightly based on the sample being</p><p id="p-65">compressed/decompressed or single/double pair</p><p id="p-66">BENCHMARK=&lt;benchmark_name_minus-R#&gt; FORMAT=fastq bash -c ‘/usr/local/sbin/kraken2</p><p id="p-67">--db databases/kraken2/ncbi_nt --threads 8 --gzip-compressed --classified-out</p><p id="p-68">results/kraken2/$BENCHMARK.classified_seqs#.fq --unclassified-out</p><p id="p-69">results/kraken2/$BENCHMARK.unclassified_seqs#.fq –output</p><p id="p-70">results/kraken2/$BENCHMARK.kraken2.out –paired</p><p id="p-71">benchmarks/${BENCHMARK}_R1.$FORMAT.gz</p><p id="p-72">benchmarks/${BENCHMARK}_R2.$FORMAT.gz &gt;</p></div><div id="sec-33" class="subsection"><h3>Application I - Data Processing</h3><p id="p-73">In collaboration with Saha <em>et al</em>. [<a id="xref-ref-1-6" class="xref-bibr" href="#ref-1">1</a>], three samples were identified (CHRF_0000, CHRF_0094, CHRF_0002, from the original NCBI Sequence Read Archive dataset under BioProject PRJNA516582) and re-run on pipeline version 3.13. The pipeline results were filtered using a conservative set of filters, which required NT_rPM &gt; 10 and NT_zscore &gt; 1. The z-score was computed with respect to the public background model CHRF_RNA_Negative, which was used in the original manuscript. The background model was generated based on RNA-seq data from water samples and negative controls. Metrics were compiled into <strong><a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a></strong> and a heatmap was generated using IDseq, with the same filters (<strong><a id="xref-fig-2-12" class="xref-fig" href="#F2">Figure 2D</a></strong>).</p></div><div id="sec-34" class="subsection"><h3>Application II - Data Processing</h3><p id="p-74">In collaboration with Manning <em>et al</em>. [<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">56</a>], RNA was extracted from a sample obtained from a symptomatic patient meeting criteria for possible COVID-19 pneumonia. Libraries were prepared for sequencing as described in Manning <em>et al</em> and sequenced on an Illumina iSeq100. The raw .fastq files were uploaded to IDseq from the CNM-NIH lab in Phnom Penh, via Illumina BaseSpace, on January 31, 2020 using an NCBI index from September 2019. An NCBI database update was then done on February 2, 2020 by the IDseq team and the results were evaluated. These samples were run on IDseq pipeline version 3.18. The data was deposited in public repositories by the original authors and is available at GISAID accession EPI_ISL_411902. IDseq results for the associated samples are available at <a href="http://public.idseq.net/covid-19?utm_source=bioarxiv&amp;utm_medium=paper&amp;utm_campaign=benchmark-paper">http://public.idseq.net/covid-19?utm_source=bioarxiv&amp;utm_medium=paper&amp;utm_campaign=benchmark-paper</a>.</p></div></div><div class="section" id="sec-35"><h2 class="">Availability of supporting source code and requirements</h2><p id="p-75">Project name: IDseq Portal</p><p id="p-76">Project home page: <a href="https://idseq.net">https://idseq.net</a></p><p id="p-77">Operating system(s): Platform independent</p><p id="p-78">Programming language: Python, Ruby, JavaScript</p><p id="p-79">Other requirements: Web browser</p><p id="p-80">License: MIT License</p></div><div class="section" id="sec-36"><h2 class="">Availability of supporting data</h2><p id="p-81">Data referenced in this manuscript has been previously published. SRA accession IDs are included in the original manuscripts [<a id="xref-ref-1-7" class="xref-bibr" href="#ref-1">1</a>, <a id="xref-ref-39-5" class="xref-bibr" href="#ref-39">39</a>, <a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">56</a>].</p></div><div class="section" id="sec-37"><h2 class="">Supplemental Text</h2><p id="p-82">Supplemental_Text.docx, contains supplemental methods and results associated with two benchmark datasets listed in the main text, as well as supplemental figures (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure S1</a> – S3) and supplemental tables indicating IDseq data availability (Table S1 – S3).</p></div><div class="section" id="sec-38"><h2 class="">Author’s Contributions</h2><p id="p-101">KLK conceived the project. KLK and JLD structured the draft and provided final editing. KLK coordinated and drafted the manuscript, and synthesized comments provided by all authors. All authors contributed critically important comments. VA, SL, SC, JAB, and JEM contributed to the generation of COVID-19 sequencing results. The IDseq Engineering Team (TC, CDB, BD, GD, RE, JH, OH, YJ, RK, AK, MM, LR, DRC, JS, JT, JW, MZ, EZ) contributed to software development. All authors read and approved the final manuscript.</p></div><div class="section" id="sec-39"><h2 class="">SUPPLEMENTAL TEXT</h2><div class="section" id="sec-40"><h3>Supplemental Methods</h3><div class="section" id="sec-41"><h4>Internal Benchmark Datasets and Analysis</h4><p id="p-102">Many closely related bacterial species have different relative pathogenicity in humans or other host organisms [<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">1</a>]. The unambiguous benchmark datasets, evaluated in the main text, do not emphasize performance at distinguishing common clinically relevant pathogens and closely related bacterial species. Here, we provide the results of an analysis of two simulated datasets containing common clinical microbes. For each dataset, we evaluated the performance of IDseq (NT and NR), as compared to Kraken2 [<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">2</a>], using per-species recall.</p></div><div class="section" id="sec-42"><h4>Internal Benchmarks - Common Clinical Microbes (CCM) Benchmark Dataset</h4><p id="p-103">Several microbes commonly identified in samples from humans and vector samples were identified in collaboration with researchers and incorporated into the CCM benchmark dataset. The selected species represent a range of bacterial, viral, fungal, and parasitic pathogens. The idseq-bench simulation tool was used to pull reference genomes from 35 specified accessions across 13 species (<em>Species name</em>, taxonomic ID; <em>Klebsiella pneumoniae</em>, 573; <em>Aspergillus fumigatus</em> Af293, 746128; <em>Plasmodium falciparum</em>, 5833; Rubella virus, 11041; Human immunodeficiency virus 1, 11676; Rhinovirus C, 463676; Chikungunya virus, 37124; <em>Staphylococcus aureus</em>, 1280; <em>Balamuthia mandrillaris</em>, 66527; <em>Elizabethkingia anophelis</em>, 1117645; <em>Neisseria meningitidis</em>, 487; Torque teno midi virus, 2065052; Hubei mosquito virus 2, 1922926) and simulate reads using the InSilicoSeq [<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">3</a>] sequence simulator with uniform coverage and sequencing error models derived from HiSeq data. The simulated sample is available on IDseq (<strong>Table S3</strong>, <a href="https://idseq.net/ak8qu">https://idseq.net/ak8qu</a>). This simulated dataset is utilized by the IDseq team to evaluate the consistency of IDseq results over time and ensure the reliability of the pipeline results.</p><p id="p-104">For IDseq NT and NR results, the idseq-bench score function was used to evaluate the per-read, species-level recall across each ground truth organism. Then, to characterize false positives, the total number of species identified was determined, along with the percentage of reads mapping to incorrect taxa. For Kraken2, the read-level recall was determined at the species level and reads mapping to higher taxonomic levels were not considered as positive matches for the species-level recall. Again, the total number of unique species was determined and the proportion of reads mapping incorrectly at the species level was determined.</p></div><div class="section" id="sec-43"><h4>Internal Benchmarks - Closely Related Bacteria (CRB) Benchmark Dataset</h4><p id="p-105">Many bacterial pathogens are closely related, therefore resulting in possible genomic overlap and ambiguity in aligning short read sequences that may map to multiple species. However, distinction of pathogens at the species-level has implications for treatment and outcomes. 10 bacterial species from 6 genera within the Enterobacteriaciae family were identified and the idseq-bench simulation tool was used to pull reference genomes (<em>Species name</em>, taxonomic ID; <em>Salmonella enterica</em>, 28901; <em>Citrobacter koseri</em>, 545; <em>Citrobacter freundii</em>, 546; <em>Klebsiella aerogenes</em>, 548; <em>Enterobacter cloacae</em>, 550; <em>Escherichia coli</em>, 562; <em>Klebsiella oxytoca</em>, 571; <em>Klebsiella pneumoniae</em>, 573; <em>Shigella boydii</em>, 621; <em>Shigella flexneri</em>, 623) and simulate reads using the InSilicoSeq [<a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">3</a>] sequence simulator with uniform coverage and sequencing error models derived from HiSeq data. The simulated sample is available on IDseq (<strong>Table S3</strong>, <a href="https://idseq.net/71wns">https://idseq.net/71wns</a>). Analysis of the CRB benchmark dataset replicated that which was done for the CCM dataset (above).</p></div></div><div class="section" id="sec-44"><h3>Supplemental Results</h3><div class="section" id="sec-45"><h4>External Benchmarks - NYCSM Outlier Dataset and IDseq NT</h4><p id="p-106">The NYCSM dataset had the lowest AUPR of all datasets tested by IDseq NT. Overall, we note that due to the relative comparable performance of tools on these particular datasets, distinctions in AUPR may be the result of just one or two missed taxa. In this case, IDseq NT failed to identify <em>Enterobacter asburiae</em> (taxID = 61645) and identified only a small number (&lt; 10) of reads aligning to <em>Pseudoalteromonas haloplanktis</em> (taxID = 228). Meanwhile, IDseq NR identified successfully recovered reads from <em>E. asburiae</em> and produced a comparably low number of reads mapping to <em>P. haloplanktis</em>. IDseq NT did identify many reads mapping to <em>Enterobacter soli</em>, which has been noted to have high genomic similarity to <em>E. asburiae</em> and <em>E. aerogenes</em> (p-distance: 1.06 and 1.19%, respectively [<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">4</a>]), both of which were included in the simulated dataset. This highlights the challenge with disambiguating short reads from taxa with a high degree of genomic similarity. We emphasize the practical importance of orthogonal validation of hits via assays (ie PCR) targeting unique regions of the genome. However, it is possible that the expansion of reference databases has improved specificity beyond the original dataset simulation. The particular GenBank accession to which all <em>E. soli</em> reads map (CP003026.1 Enterobacter soli strain LF7a, complete genome), was added to the database in April 2019, while reads were simulated in 2016 [<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">5</a>]. The same trend is observed for <em>P. haloplanktis</em>. IDseq NT and NR both identified significantly more reads mapping to <em>P. arctica</em>, a member of the <em>P. haloplanktis</em>-like group [<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">6</a>]. The associated GenBank accession to which the majority of reads map (<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=CP011025.1&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030551.1.atom">CP011025.1</a>  - Pseudoalteromonas arctica A 37-1-2 chromosome I, complete sequence) was added in September of 2017. The IDseq coverage visualization makes interrogation of microbial hits simple by linking to the NCBI taxonomy database and GenBank accessions.</p></div><div class="section" id="sec-46"><h4>Viral Divergence in Recent Human Disease</h4><p id="p-107">The ability to detect divergent viruses is a function of genomic similarity to other organisms in the database as well as genomic coverage, which influences assembled contig lengths. Many recent, emerging, diseases affecting humans do have some sequence similarity to organisms in the NCBI databases. For example, since the emergence of enterovirus EV-D68, numerous outbreaks caused by divergent sub-clades have been reported with nt similarity to other strains ∼96% [<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">7</a>]. Similarly, recent outbreaks of Dengue virus have been reported to be the result of introduction of novel DENV lineages, which are defined based on nucleotide divergence of 6-8% within each DENV serotype[<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">8</a>,<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">9</a>]. The set of nine known West Nile Virus lineages (which genomic analysis indicated diverged in the early 17th century) have an average pairwise percent identity of 77.6% (nucleotide) and 90.1% (amino acid) [<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">10</a>]. These are well within the range for detection by IDseq. Meanwhile, Zika virus, the viral species which caused the recent 2015 epidemic, was discovered in 1940 and shares, on average, 55.6% amino acid sequence identity with dengue virus and 57.0% with West Nile virus [<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">11</a>]. Had the sequence of the first Zika isolate not been in the database, IDseq would not have been able to flag the presence of this virus and researchers would be required to evaluate assembled, unmapped, contigs offline [<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">12</a>].</p></div><div class="section" id="sec-47"><h4>Internal Benchmark Datasets Results</h4><p id="p-108">Two additional datasets were simulated to evaluate IDseq’s performance for detection of common clinical microbes (CCM) as well as for disambiguating closely related bacterial species (CRB) (Methods, <strong>Table S3</strong>). The CCM dataset contains reads simulated from 13 total species, including six viral, four bacterial, and three eukaryotic pathogens. The CRB dataset contains reads simulated from 10 species of bacteria, all from the Enterobacteriaceae family. Given that IDseq’s pipeline returns a species-level assignment for each mapped read and the web interface presents results at the species level, we evaluate each of these benchmarks only at the species level. The Kraken2 algorithm assigns reads with ambiguous mappings to higher taxonomic levels. Therefore, the results shown include only the species-specific alignment results.</p><p id="p-109">For the CCM dataset, IDseq filtered out 15.3% of reads during the host and QC filtering pipeline steps. Of the remaining 99,320 non-host reads, IDseq NT showed the greatest per-species recall across the 13 species included (1.0, IQR = 0.99 – 1.0). IDseq NT per-species recall was significantly higher than that of both IDseq NR (0.87, IQR = 0.85 – 1.0, p = 0.02) and Kraken2 (0.89, IQR = 0.64 – 1.0, p = 0.005 Wilcoxon rank sum) (<strong><a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure S3A</a></strong>). All tools successfully identified the presence of all 13 microbial species. In addition to these species, IDseq NT, NR, and Kraken2 identified 63, 303, and 230 false positive species, respectively. Since the IDseq pipeline returns a species-level assignment for all mapped reads, even in cases where the species may align equally to two different species, it had a notably greater portion of the total (post-qc) reads mapping across those false positive organisms (3.0 % by nt, 10.0 % by NR) than Kraken2, which had only 0.56 % of reads mapping to the false positive species. Kraken2 avoids larger percentages of reads being associated with false-positive species calls by calling a significant portion of ambiguously mapped reads at higher levels of the taxonomic tree.</p><p id="p-110">Similar trends were observed for the CRB dataset. IDseq filtered out 15.6% of reads during the host and QC filtering pipeline steps, leaving 84,410 non-host reads for down-stream analysis. Notably, IDseq nt demonstrates the highest per-species recall (0.68, IQR = 0.43 – 0.80), significantly different than NR (0.46, IQR = 0.15 – 0.61, p = 0.005, Wilcoxon rank sum) and Kraken2 (0.18, IQR = 0.11 – 0.35, p = 0.005, Wilcoxon rank sum) (<strong><a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure S3B</a></strong>). All tools successfully identified the presence of all 13 microbial species. In addition to these species, IDseq NT, NR, and Kraken2 identified 83, 371, and 167 false positive species, respectively. Again, IDseq NT and NR had greater proportions of total reads mapping to these false-positive species (31.7% and 49.7% for NT and NR, respectively) as compared to Kraken2, with only 0.6 % of reads mapping to false-positive species and the majority of ambiguous reads mapping at higher levels of classification (70.9%).</p><p id="p-111">The impact of ambiguous reads is exaggerated in cases where we simulate reads from multiple closely related species with known genomic similarity. In many cases of an infection, with a single dominant organism, the importance of recall may outweigh the identification of lower-level false-positive species. Additionally, these simulations sample from across the genome. However, we know that ribosomal sequences can be used for typing of bacteria and studies have previously shown improved sensitivity with RNA-seq, where ribosomal RNA comprises the greatest portion of sequenced nucleic acid [<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">13</a>,<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">14</a>]. Thus, IDseq results must be interpreted by the researcher with respect to the sample type and sequencing prep.</p></div></div></div><div class="section" id="sec-48"><h2 class="">Supplemental Figure</h2><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The IDseq pipeline visualization indicates each step in the underlying pipeline and includes a description of the raw command parameters as well as the ability to download intermediate files for offline analysis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1081500886" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The IDseq pipeline visualization indicates each step in the underlying pipeline and includes a description of the raw command parameters as well as the ability to download intermediate files for offline analysis.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F5.medium.gif" width="440" height="128"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S1:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F5.medium.gif" width="440" height="128"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S1:</span> <p id="p-112" class="first-child">The IDseq pipeline visualization indicates each step in the underlying pipeline and includes a description of the raw command parameters as well as the ability to download intermediate files for offline analysis.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Performance metrics evaluated across 20 mNGS taxonomic identification tools, A) Area under the precision recall curve (AUPR), ranges from 0 to 1 (best), B) Precision, ranges from 0 to 1 (best). C) Recall, ranges from 0 to 1 (best) D) F1-score, the harmonic mean of precision and recall values. E) L2 distance, ranges from 0 (best) to 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1081500886" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Performance metrics evaluated across 20 mNGS taxonomic identification tools, &lt;strong&gt;A)&lt;/strong&gt; Area under the precision recall curve (AUPR), ranges from 0 to 1 (best), &lt;strong&gt;B)&lt;/strong&gt; Precision, ranges from 0 to 1 (best). &lt;strong&gt;C)&lt;/strong&gt; Recall, ranges from 0 to 1 (best) &lt;strong&gt;D)&lt;/strong&gt; F1-score, the harmonic mean of precision and recall values. &lt;strong&gt;E)&lt;/strong&gt; L2 distance, ranges from 0 (best) to 1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F6.medium.gif" width="440" height="314"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S2:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F6.medium.gif" width="440" height="314"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S2:</span> <p id="p-113" class="first-child">Performance metrics evaluated across 20 mNGS taxonomic identification tools, <strong>A)</strong> Area under the precision recall curve (AUPR), ranges from 0 to 1 (best), <strong>B)</strong> Precision, ranges from 0 to 1 (best). <strong>C)</strong> Recall, ranges from 0 to 1 (best) <strong>D)</strong> F1-score, the harmonic mean of precision and recall values. <strong>E)</strong> L2 distance, ranges from 0 (best) to 1.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Per-species recall values for two internal benchmark datasets, A) The common clinical microbes (CCM) dataset and B) the closely related bacteria (CRB) dataset." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1081500886" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Per-species recall values for two internal benchmark datasets, &lt;strong&gt;A)&lt;/strong&gt; The common clinical microbes (CCM) dataset and &lt;strong&gt;B)&lt;/strong&gt; the closely related bacteria (CRB) dataset.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F7.medium.gif" width="440" height="229"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S3:" src="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F7.medium.gif" width="440" height="229"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/04/09/2020.04.07.030551.1/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S3:</span> <p id="p-114" class="first-child">Per-species recall values for two internal benchmark datasets, <strong>A)</strong> The common clinical microbes (CCM) dataset and <strong>B)</strong> the closely related bacteria (CRB) dataset.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-49"><h2 class="">Supplemental Tables</h2><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1243630/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1243630/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1243630" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S1:</span> <p id="p-115" class="first-child">Github repositories containing open-source code for IDseq pipeline, web application, and benchmarking resources.</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1243631/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1243631/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1243631" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S2:</span> <p id="p-116" class="first-child">External benchmark datasets and their corresponding IDseq links.</p><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1243644/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/1243644/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S3:</span> <p id="p-117" class="first-child">Internal benchmark datasets and their corresponding IDseq links.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-100">This research was supported by the Chan Zuckerberg Initiative (CZI). The authors would like to thank all CZI team members who were involved in support for software development and all researchers who have provided input on the IDseq Web Portal throughout the course of its development. The authors would like to thank Hanna Retallack for valuable comments on the manuscript text.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-update fn-group-summary-of-updates" id="fn-1"><p id="p-6">Updated author name in bioRxiv system, full text PDF is identical.</p></li></ul></div><div class="section glossary"><h2 class="">List of Abbreviations</h2><dl id="def-list-1"><dt id="def-item-1">ASG<span class="def-item-dt-sep"></span></dt><dd>Auto-scaling Groups<span class="def-item-dd-sep"></span></dd><dt id="def-item-2">SARS-CoV-2<span class="def-item-dt-sep"></span></dt><dd>Severe Acute Respiratory Syndrome Coronavirus 2<span class="def-item-dd-sep"></span></dd><dt id="def-item-3">AUPR<span class="def-item-dt-sep"></span></dt><dd>Area Under the Precision Recall Curve<span class="def-item-dd-sep"></span></dd><dt id="def-item-4">AWS<span class="def-item-dt-sep"></span></dt><dd>Amazon Web Services<span class="def-item-dd-sep"></span></dd><dt id="def-item-5">CLI<span class="def-item-dt-sep"></span></dt><dd>Command Line Interface<span class="def-item-dd-sep"></span></dd><dt id="def-item-6">CSF<span class="def-item-dt-sep"></span></dt><dd>Cerebrospinal Fluid<span class="def-item-dd-sep"></span></dd><dt id="def-item-7">DAG<span class="def-item-dt-sep"></span></dt><dd>Directed Acyclic Graph<span class="def-item-dd-sep"></span></dd><dt id="def-item-8">DCR<span class="def-item-dt-sep"></span></dt><dd>Duplicate Compression Ratio<span class="def-item-dd-sep"></span></dd><dt id="def-item-9">EC2<span class="def-item-dt-sep"></span></dt><dd>Elastic Compute Cloud<span class="def-item-dd-sep"></span></dd><dt id="def-item-10">ERCCs<span class="def-item-dt-sep"></span></dt><dd>Externa RNA Controls Consortium<span class="def-item-dd-sep"></span></dd><dt id="def-item-11">mNGS<span class="def-item-dt-sep"></span></dt><dd>metagenomic Next-Generation Sequencing<span class="def-item-dd-sep"></span></dd><dt id="def-item-12">NCBI<span class="def-item-dt-sep"></span></dt><dd>National Center for Biotechnology Information<span class="def-item-dd-sep"></span></dd><dt id="def-item-13">QC<span class="def-item-dt-sep"></span></dt><dd>Quality Control<span class="def-item-dd-sep"></span></dd><dt id="def-item-14">taxID<span class="def-item-dt-sep"></span></dt><dd>Taxonomic identifier<span class="def-item-dd-sep"></span></dd><dt id="def-item-15">NT<span class="def-item-dt-sep"></span></dt><dd>Nucleotide, NCBI nucleotide (nt) database<span class="def-item-dd-sep"></span></dd><dt id="def-item-16">NR<span class="def-item-dt-sep"></span></dt><dd>Protein, NCBI non-redundant protein (nr) database<span class="def-item-dd-sep"></span></dd><dt id="def-item-17">rPM<span class="def-item-dt-sep"></span></dt><dd>Reads per Million Reads Sequenced</dd></dl></div><div class="section ref-list" id="ref-list-1"><h2 class="">REFERENCES</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Saha</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ramesh</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kalantar</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Malaker</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hasanuzzaman</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Khan</span>  <span class="cit-name-given-names">LM</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Unbiased metagenomic sequencing for pediatric meningitis in bangladesh reveals neuroinvasive chikungunya virus outbreak and other unrealized pathogens</span>. <abbr class="cit-jnl-abbrev">MBio</abbr>. <span class="cit-pub-date">2019</span>;<span class="cit-vol">10</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.2" data-doi="10.1093/cid/cix881"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Simner</span>  <span class="cit-name-given-names">PJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carroll</span>  <span class="cit-name-given-names">KC</span></span>. <span class="cit-article-title">Understanding the Promises and Hurdles of Metagenomic Next-Generation Sequencing as a Diagnostic Tool for Infectious Diseases</span>. <abbr class="cit-jnl-abbrev">Clin Infect Dis</abbr>. <span class="cit-pub-date">2018</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/cid/cix881</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin%2BInfect%2BDis%26rft.volume%253D66%26rft.spage%253D778%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fcid%252Fcix881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/cid/cix881&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Breitwieser</span>  <span class="cit-name-given-names">FP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thielen</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">SL</span></span>. <span class="cit-article-title">Bracken: Estimating species abundance in metagenomics data</span>. <abbr class="cit-jnl-abbrev">PeerJ Comput Sci. 2017</abbr>; <span class="cit-pub-date">2017</span>:<span class="cit-fpage">e104</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">4.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030551v2.4" data-doi="10.1101/gr.210641.116"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Breitwieser</span>  <span class="cit-name-given-names">FP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">SL</span></span>. <span class="cit-article-title">Centrifuge: Rapid and sensitive classification of metagenomic sequences</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">26</span>:<span class="cit-fpage">1721</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.210641.116%26rft_id%253Dinfo%253Apmid%252F27852649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyNi8xMi8xNzIxIjtzOjQ6ImF0b20iO3M6NTA6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDkvMjAyMC4wNC4wNy4wMzA1NTEuMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">5.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030551v2.5" data-doi="10.1093/bioinformatics/bty501"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pedamallu</span>  <span class="cit-name-given-names">CS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ojesina</span>  <span class="cit-name-given-names">AI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bullman</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sharpe</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whelan</span>  <span class="cit-name-given-names">CW</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">GATK PathSeq: a customizable computational tool for the discovery and identification of microbial sequences in libraries from eukaryotic hosts</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2018</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/bty501</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty501&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">6.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030551v2.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Břinda</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Salikhov</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pignotti</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kucherov</span>  <span class="cit-name-given-names">G.</span></span> <abbr class="cit-jnl-abbrev">karel-brinda/prophyle: ProPhyle 0.3.1.0</abbr>. <span class="cit-pub-date">2017</span>;</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">7.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030551v2.7" data-doi="10.1101/gr.225276.117"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Corvelo</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clarke</span>  <span class="cit-name-given-names">WE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Robine</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zody</span>  <span class="cit-name-given-names">MC</span></span>. <span class="cit-article-title">taxMaps: Comprehensive and highly accurate taxonomic classification of short-read data in reasonable time</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr>. <span class="cit-pub-date">2018</span>;<span class="cit-vol">28</span>:<span class="cit-fpage">751</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.225276.117%26rft_id%253Dinfo%253Apmid%252F29588360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI4LzUvNzUxIjtzOjQ6ImF0b20iO3M6NTA6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDkvMjAyMC4wNC4wNy4wMzA1NTEuMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">8.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030551v2.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Buchfink</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huson</span>  <span class="cit-name-given-names">DH</span></span>. <span class="cit-article-title">Fast and sensitive protein alignment using DIAMOND</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr>. <span class="cit-pub-date">2014</span>. p. <span class="cit-fpage">59</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">9.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030551v2.9" data-doi="10.1038/ncomms11388"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Menzel</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">KL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krogh</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Fast and sensitive taxonomic classification for metagenomics with Kaiju</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">7</span>:<span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms11388%26rft_id%253Dinfo%253Apmid%252F27009409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms11388&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27009409&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030551.1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">10.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030551v2.10" data-doi="10.1093/bioinformatics/btw006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hauser</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Steinegger</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sö Ding</span>  <span class="cit-name-given-names">J.</span></span> <abbr class="cit-jnl-abbrev">MMseqs software suite for fast and deep clustering and searching of large protein sequence sets</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btw006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMMseqs%2Bsoftware%2Bsuite%2Bfor%2Bfast%2Band%2Bdeep%2Bclustering%2Band%2Bsearching%2Bof%2Blarge%2Bprotein%2Bsequence%2Bsets%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw006%26rft_id%253Dinfo%253Apmid%252F26743509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26743509&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030551.1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">11.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030551v2.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Truong</span>  <span class="cit-name-given-names">DT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Franzosa</span>  <span class="cit-name-given-names">EA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tickle</span>  <span class="cit-name-given-names">TL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scholz</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weingart</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pasolli</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">MetaPhlAn2 for enhanced metagenomic taxonomic profiling</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr>. <span class="cit-pub-date">2015</span>. p. <span class="cit-fpage">902</span>–<span class="cit-lpage">3</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">12.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030551v2.12" data-doi="10.1038/s41467-019-09078-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Milanese</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mende</span>  <span class="cit-name-given-names">DR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Paoli</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Salazar</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ruscheweyh</span>  <span class="cit-name-given-names">HJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cuenca</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Microbial abundance, activity and population genomic profiling with mOTUs2</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr>. <span class="cit-pub-date">2019</span>;<span class="cit-vol">10</span>:<span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D10%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-019-09078-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-019-09078-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">13.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030551v2.13" data-doi="10.1186/s12864-015-1419-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ounit</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wanamaker</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Close</span>  <span class="cit-name-given-names">TJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lonardi</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">CLARK: fast and accurate classification of metagenomic and genomic sequences using discriminative k-mers</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr>. <span class="cit-pub-date">2015</span>;<span class="cit-vol">16</span>:<span class="cit-fpage">236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D236%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1419-2%26rft_id%253Dinfo%253Apmid%252F25879410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1419-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25879410&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030551.1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">14.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030551v2.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ounit</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lonardi</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Higher classification sensitivity of short metagenomic reads with CLARK-S</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">32</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.15" data-doi="10.1186/s13059-019-1891-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">DE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">Improved metagenomic analysis with Kraken 2</span>. <abbr class="cit-jnl-abbrev">Genome Biol</abbr>. <span class="cit-pub-date">2019</span>;<span class="cit-vol">20</span>:<span class="cit-fpage">257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D20%26rft.spage%253D257%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-019-1891-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-019-1891-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">16.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030551v2.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">DE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">SL</span></span>. <span class="cit-article-title">Kraken: ultrafast metagenomic sequence classification using exact alignments</span>. <abbr class="cit-jnl-abbrev">Genome Biology</abbr>. <span class="cit-pub-date">2014</span>;<span class="cit-vol">304</span>:<span class="cit-fpage">66</span>–<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiology%26rft.volume%253D304%26rft.spage%253D66%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">17.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030551v2.17" data-doi="10.1186/s13059-018-1568-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Breitwieser</span>  <span class="cit-name-given-names">FP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">DN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">SL</span></span>. <span class="cit-article-title">KrakenUniq: confident and fast metagenomics classification using unique k-mer counts</span>. <abbr class="cit-jnl-abbrev">Genome Biol</abbr>. <span class="cit-pub-date">2018</span>;<span class="cit-vol">19</span>:<span class="cit-fpage">198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D19%26rft.spage%253D198%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-018-1568-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-018-1568-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">18.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030551v2.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ainsworth</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sternberg</span>  <span class="cit-name-given-names">MJE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Come</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Butcher</span>  <span class="cit-name-given-names">SA</span></span>. <span class="cit-article-title">k-SLAM: Accurate and Ultra-Fast Taxonomic Classification and Gene Identification for Large Metagenomic Data Sets - PubMed</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">45</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">19.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030551v2.19" data-doi="10.1093/bioinformatics/btn322"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Morgulis</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coulouris</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Raytselis</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Madden</span>  <span class="cit-name-given-names">TL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Agarwala</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schäffer</span>  <span class="cit-name-given-names">AA</span></span>. <span class="cit-article-title">Database indexing for production MegaBLAST searches</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2008</span>;<span class="cit-vol">24</span>:<span class="cit-fpage">1757</span>–<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn322%26rft_id%253Dinfo%253Apmid%252F18567917%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btn322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18567917&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030551.1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258471500040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elliott</span>  <span class="cit-name-given-names">CF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Sequence analysis A novel data structure to support ultra-fast taxonomic classification of metagenomic sequences with k-mer signatures</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2018</span>;<span class="cit-vol">34</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chiu</span>  <span class="cit-name-given-names">CY</span></span>. <span class="cit-article-title">Clinical Metagenomic Next-Generation Sequencing for Pathogen Detection</span>. <abbr class="cit-jnl-abbrev">Annu Rev Pathol Mech Dis</abbr>. <span class="cit-pub-date">2019</span>;<span class="cit-vol">14</span>:<span class="cit-fpage">319</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu%2BRev%2BPathol%2BMech%2BDis%26rft.volume%253D14%26rft.spage%253D319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.07.030551v2.22"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">One Codex [Internet]. [cited 2020 Mar 11]</span>. Available from: <a href="https://www.onecodex.com/">https://www.onecodex.com/</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.23" data-doi="10.1186/s40168-019-0658-x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Clarke</span>  <span class="cit-name-given-names">EL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">LJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Connell</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fett</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Sunbeam: An extensible pipeline for analyzing metagenomic sequencing experiments</span>. <abbr class="cit-jnl-abbrev">Microbiome</abbr>. <span class="cit-pub-date">2019</span>;<span class="cit-vol">7</span>:<span class="cit-fpage">46</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiome%26rft.volume%253D7%26rft.spage%253D46%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs40168-019-0658-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s40168-019-0658-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.24" data-doi="10.1101/gr.171934.113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Naccache</span>  <span class="cit-name-given-names">SN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Federman</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Veeraraghavan</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zaharia</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Samayoa</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">A cloud-compatible bioinformatics pipeline for ultrarapid pathogen identification from next-generation sequencing of clinical samples</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr>. <span class="cit-pub-date">2014</span>;<span class="cit-vol">24</span>:<span class="cit-fpage">1180</span>–<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.171934.113%26rft_id%253Dinfo%253Apmid%252F24899342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI0LzcvMTE4MCI7czo0OiJhdG9tIjtzOjUwOiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA5LzIwMjAuMDQuMDcuMDMwNTUxLjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.25" data-doi="10.1371/journal.pntd.0001485"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yozwiak</span>  <span class="cit-name-given-names">NL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Skewes-Cox</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stenglein</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Balmaseda</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">DeRisi</span>  <span class="cit-name-given-names">JL</span></span>. <span class="cit-article-title">Virus Identification in Unknown Tropical Febrile Illness Cases Using Deep Sequencing</span>. <abbr class="cit-jnl-abbrev">PLoS Negl Trop Dis</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">6</span>:<span class="cit-fpage">e1485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BNegl%2BTrop%2BDis%26rft.aulast%253DYozwiak%26rft.auinit1%253DN.%2BL.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253De1485%26rft.epage%253De1485%26rft.atitle%253DVirus%2Bidentification%2Bin%2Bunknown%2Btropical%2Bfebrile%2Billness%2Bcases%2Busing%2Bdeep%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pntd.0001485%26rft_id%253Dinfo%253Apmid%252F22347512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pntd.0001485&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22347512&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030551.1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.26" data-doi="10.1093/bioinformatics/bts635"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dobin</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schlesinger</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drenkow</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zaleski</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jha</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Sequence analysis STAR: ultrafast universal RNA-seq aligner</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2013</span>;<span class="cit-vol">29</span>:<span class="cit-fpage">15</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts635%26rft_id%253Dinfo%253Apmid%252F23104886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts635&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23104886&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030551.1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312654600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.27" data-doi="10.1073/pnas.1809700115"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Langelier</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kalantar</span>  <span class="cit-name-given-names">KL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moazed</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crawford</span>  <span class="cit-name-given-names">ED</span></span>, <span class="cit-auth"><span class="cit-name-surname">Deiss</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Integrating host response and unbiased microbe detection for lower respiratory tract infection diagnosis in critically ill adults</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci</abbr>. <span class="cit-pub-date">2018</span>;<span class="cit-vol">115</span>:<span class="cit-fpage">E12353</span>–<span class="cit-lpage">62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1809700115%26rft_id%253Dinfo%253Apmid%252F30482864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMzoiMTE1LzUyL0UxMjM1MyI7czo0OiJhdG9tIjtzOjUwOiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA5LzIwMjAuMDQuMDcuMDMwNTUxLjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.28" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bolger</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Usadel</span>  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2014</span>;<span class="cit-vol">30</span>:<span class="cit-fpage">2114</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24695404&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030551.1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340049100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ruby</span>  <span class="cit-name-given-names">JG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bellare</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">DeRisi</span>  <span class="cit-name-given-names">JL</span></span>. <span class="cit-article-title">PRICE: Software for the targeted assembly of components of (Meta) genomic sequence data</span>. <abbr class="cit-jnl-abbrev">G3 Genes, Genomes, Genet</abbr>. <span class="cit-pub-date">2013</span>;<span class="cit-vol">3</span>:<span class="cit-fpage">865</span>–<span class="cit-lpage">80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2BGenes%252C%2BGenomes%252C%2BGenet%26rft.volume%253D3%26rft.spage%253D865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.30" data-doi="10.1093/bioinformatics/btl158"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Ã W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Godzik</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2006</span>;<span class="cit-vol">22</span>:<span class="cit-fpage">1658</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl158%26rft_id%253Dinfo%253Apmid%252F16731699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btl158&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16731699&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030551.1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238905700017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.31" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">SL</span></span>. <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2</span>. <abbr class="cit-jnl-abbrev">Nat Methods</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">9</span>:<span class="cit-fpage">357</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030551.1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.32" data-doi="10.1093/bioinformatics/btq057"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">TD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nacu</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Sequence analysis Fast and SNP-tolerant detection of complex variants and splicing in short reads</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2010</span>;<span class="cit-vol">26</span>:<span class="cit-fpage">873</span>–<span class="cit-lpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq057%26rft_id%253Dinfo%253Apmid%252F20147302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20147302&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030551.1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276045800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.33" data-doi="10.1186/s40168-019-0702-x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zinter</span>  <span class="cit-name-given-names">MS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mayday</span>  <span class="cit-name-given-names">MY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ryckman</span>  <span class="cit-name-given-names">KK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jelliffe-Pawlowski</span>  <span class="cit-name-given-names">LL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Derisi</span>  <span class="cit-name-given-names">JL</span></span>. <span class="cit-article-title">Towards precision quantification of contamination in metagenomic sequencing experiments</span>. <abbr class="cit-jnl-abbrev">Microbiome</abbr>. <span class="cit-pub-date">2019</span>;<span class="cit-vol">7</span>:<span class="cit-fpage">1</span>–<span class="cit-lpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiome%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs40168-019-0702-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s40168-019-0702-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.34" data-doi="10.1186/s40168-018-0605-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">NM</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Proctor</span> <span class="cit-name-surname">DiM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">SP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Relman</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Callahan</span>  <span class="cit-name-given-names">BJ</span></span>. <span class="cit-article-title">Simple statistical identification and removal of contaminant sequences in marker-gene and metagenomics data</span>. <abbr class="cit-jnl-abbrev">Microbiome</abbr>. <span class="cit-pub-date">2018</span>;<span class="cit-vol">6</span>:<span class="cit-fpage">226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiome%26rft.volume%253D6%26rft.spage%253D226%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs40168-018-0605-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s40168-018-0605-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.35" data-doi="10.1186/1471-2105-12-159"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">JH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span> <span class="cit-article-title">RAPSearch: A fast protein similarity search tool for short reads</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr>. <span class="cit-pub-date">2011</span>;<span class="cit-vol">12</span>:<span class="cit-fpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DYe%26rft.auinit1%253DY.%26rft.volume%253D12%26rft.spage%253D159%26rft.epage%253D159%26rft.atitle%253DRAPSearch%253A%2Ba%2Bfast%2Bprotein%2Bsimilarity%2Bsearch%2Btool%2Bfor%2Bshort%2Breads.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-159%26rft_id%253Dinfo%253Apmid%252F21575167%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-159&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21575167&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030551.1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.07.030551v2.36"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Index of /pub/taxonomy/accession2taxid [Internet]</span>. [cited 2020 Feb 26]. Available from: <a href="ftp://ftp.ncbi.nih.gov/pub/taxonomy/accession2taxid">ftp://ftp.ncbi.nih.gov/pub/taxonomy/accession2taxid</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.37" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">SF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">EW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">DJ</span></span>. <span class="cit-article-title">Basic local alignment search tool</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr>. <span class="cit-pub-date">1990</span>;<span class="cit-vol">215</span>:<span class="cit-fpage">403</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030551.1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.38" data-doi="10.1089/cmb.2012.0021"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kulikov</span>  <span class="cit-name-given-names">AS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Prjibelski</span>  <span class="cit-name-given-names">AD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tesler</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vyahhi</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sirotkin</span>  <span class="cit-name-given-names">A V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pham</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">SPAdes: A New Genome Assembly Algorithm and Its Applications to Single-Cell Sequencing</span>. <abbr class="cit-jnl-abbrev">J Comput Biol</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">19</span>:<span class="cit-fpage">455</span>–<span class="cit-lpage">77</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DBankevich%26rft.auinit1%253DA.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D455%26rft.epage%253D477%26rft.atitle%253DSPAdes%253A%2Ba%2Bnew%2Bgenome%2Bassembly%2Balgorithm%2Band%2Bits%2Bapplications%2Bto%2Bsingle-cell%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2012.0021%26rft_id%253Dinfo%253Apmid%252F22506599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/cmb.2012.0021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22506599&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030551.1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">SH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Siddle</span>  <span class="cit-name-given-names">KJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">DJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sabeti</span>  <span class="cit-name-given-names">PC</span></span>. <span class="cit-article-title">Benchmarking Metagenomics Tools for Taxonomic Classification</span>. <abbr class="cit-jnl-abbrev">Cell</abbr>. <span class="cit-pub-date">2019</span>. p. <span class="cit-fpage">779</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.40" data-doi="10.1038/nbt.3319"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knight</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Siljander</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knip</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xavier</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gevers</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">ConStrains identifies microbial strains in metagenomic datasets</span>. <abbr class="cit-jnl-abbrev">Nat Biotechnol</abbr>. <span class="cit-pub-date">2015</span>;<span class="cit-vol">33</span>:<span class="cit-fpage">1045</span>–<span class="cit-lpage">52</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BBiotechnol%26rft.volume%253D33%26rft.spage%253D1045%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3319%26rft_id%253Dinfo%253Apmid%252F26344404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3319&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26344404&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030551.1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">41.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030551v2.41" data-doi="10.1038/nmeth.3802"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Scholz</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">D V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pasolli</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tolio</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zolfo</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Asnicar</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Strain-level microbial epidemiology and population genomics from shotgun metagenomics</span>. <abbr class="cit-jnl-abbrev">Nat Methods</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">13</span>:<span class="cit-fpage">435</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D13%26rft.spage%253D435%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3802%26rft_id%253Dinfo%253Apmid%252F26999001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3802&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26999001&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030551.1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.42" data-doi="10.1101/gr.216242.116"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Truong</span>  <span class="cit-name-given-names">DT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tett</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pasolli</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huttenhower</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Segata</span>  <span class="cit-name-given-names">N.</span></span> <span class="cit-article-title">Microbial strain-level population structure &amp; genetic diversity from metagenomes</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">27</span>:<span class="cit-fpage">626</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.216242.116%26rft_id%253Dinfo%253Apmid%252F28167665%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI3LzQvNjI2IjtzOjQ6ImF0b20iO3M6NTA6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDkvMjAyMC4wNC4wNy4wMzA1NTEuMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.43" data-doi="10.1146/annurev.mi.41.100187.002205"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Steinhauer</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">JJ</span></span>. <span class="cit-article-title">Rapid Evolution of RNA Viruses</span>. <abbr class="cit-jnl-abbrev">Annu Rev Microbiol</abbr>. <span class="cit-pub-date">1987</span>;<span class="cit-vol">41</span>:<span class="cit-fpage">409</span>–<span class="cit-lpage">31</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bmicrobiology%26rft.stitle%253DAnnu%2BRev%2BMicrobiol%26rft.aulast%253DSteinhauer%26rft.auinit1%253DD.%26rft.volume%253D41%26rft.issue%253D1%26rft.spage%253D409%26rft.epage%253D433%26rft.atitle%253DRapid%2BEvolution%2BOf%2BRNA%2BViruses%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.mi.41.100187.002205%26rft_id%253Dinfo%253Apmid%252F3318675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.mi.41.100187.002205&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3318675&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030551.1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987K194900017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Woolhouse</span>  <span class="cit-name-given-names">MEJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brierley</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">McCaffery</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lycett</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Assessing the epidemic potential of RNA and DNA viruses. Emerg Infect Dis</span>. <abbr class="cit-jnl-abbrev">Centers for Disease Control and Prevention (CDC)</abbr>; <span class="cit-pub-date">2016</span>;<span class="cit-vol">22</span>:<span class="cit-fpage">2037</span>–<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCenters%2Bfor%2BDisease%2BControl%2Band%2BPrevention%2B%2528CDC%2529%26rft.volume%253D22%26rft.spage%253D2037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">45.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030551v2.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schuffenecker</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Iteman</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Michault</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murri</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frangeul</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vaney</span>  <span class="cit-name-given-names">M-C</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Genome Microevolution of Chikungunya Viruses Causing the Indian Ocean Outbreak</span>. <abbr class="cit-jnl-abbrev">PLoS Med [Internet]</abbr>. <span class="cit-pub-date">2006</span>;<span class="cit-vol">3</span>:<span class="cit-fpage">e263</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BMed%2B%255BInternet%255D%26rft.volume%253D3%26rft.spage%253D263e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">46.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030551v2.46" data-doi="10.1073/pnas.1422456112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">RA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Evolution of the H9N2 influenza genotype that facilitated the genesis of the novel H7N9 virus</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci</abbr>. <span class="cit-pub-date">2015</span>;<span class="cit-vol">112</span>:<span class="cit-fpage">548</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1422456112%26rft_id%253Dinfo%253Apmid%252F25548189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMTIvMi81NDgiO3M6NDoiYXRvbSI7czo1MDoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wNC8wOS8yMDIwLjA0LjA3LjAzMDU1MS4xLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">A Novel Coronavirus from Patients with Pneumonia in China, 2019</span>. <abbr class="cit-jnl-abbrev">N Engl J Med</abbr>. <span class="cit-pub-date">2020</span>;</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.48"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chappell</span>  <span class="cit-name-given-names">JG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Byaruhanga</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tsoleridis</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ball</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Patrick</span> <span class="cit-name-surname">McClure</span></span> C. <span class="cit-article-title">Identification of infectious agents in high-throughput sequencing data sets is easily achievable using free, cloud-based bioinformatics platforms</span>. <abbr class="cit-jnl-abbrev">J. Clin. Microbiol</abbr> <span class="cit-pub-date">2019</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.49"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">B</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Ramesh</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nakielny</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hsu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kyohere</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Byaruhanga</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Bourcy</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Metagenomic next-generation sequencing of samples from pediatric febrile illness in Tororo, Uganda</span>. <span class="cit-ed"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">B</span></span>, editor. <abbr class="cit-jnl-abbrev">PLoS One</abbr>. <span class="cit-pub-date">2019</span>;<span class="cit-vol">14</span>:<span class="cit-fpage">e0218318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D14%26rft.spage%253De0218318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">50.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.04.07.030551v2.50" data-doi="10.1093/cid/ciz960"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Crawford</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kamm</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">LM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Caldera</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lyden</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Investigating Transfusion-related Sepsis Using Culture-Independent Metagenomic Sequencing</span>. <abbr class="cit-jnl-abbrev">Clin Infect Dis</abbr>. <span class="cit-pub-date">2019</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/cid/ciz960</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin%2BInfect%2BDis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fcid%252Fciz960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/cid/ciz960&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.51"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hasan</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sundararaju</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tsui</span>  <span class="cit-name-given-names">K-M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lopez</span>  <span class="cit-name-given-names">AP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Janahi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">A Metagenomics – Based Diagnostic Approach for Central Nervous System Infections in Hospital Acute Care Setting</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>. <span class="cit-pub-date">2019</span>;<span class="cit-vol">752618</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.52"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ruppé</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schrenzel</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Messages from the second International Conference on Clinical Metagenomics (ICCMg2)</span>. <abbr class="cit-jnl-abbrev">Microbes Infect</abbr>. <span class="cit-pub-date">2018</span>. p. <span class="cit-fpage">222</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.53"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Donovan</span>  <span class="cit-name-given-names">BD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gelfand</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sample</span>  <span class="cit-name-given-names">HA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chow</span>  <span class="cit-name-given-names">FC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Betjemann</span>  <span class="cit-name-given-names">JP</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Chronic meningitis investigated via metagenomic next-generation sequencing</span>. <abbr class="cit-jnl-abbrev">JAMA Neurol</abbr>. <span class="cit-pub-date">2018</span>;<span class="cit-vol">75</span>:<span class="cit-fpage">947</span>–<span class="cit-lpage">55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJAMA%2BNeurol%26rft.volume%253D75%26rft.spage%253D947%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.54" data-doi="10.1128/JCM.02723-15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mollerup</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Friis-Nielsen</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vinner</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">TA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Richter</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fridholm</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Propionibacterium acnes: Disease-causing agent or common contaminant? detection in diverse patient samples by next-generation sequencing</span>. <abbr class="cit-jnl-abbrev">J Clin Microbiol</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">54</span>:<span class="cit-fpage">980</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BClin%2BMicrobiol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.02723-15%26rft_id%253Dinfo%253Apmid%252F26818667%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjg6IjU0LzQvOTgwIjtzOjQ6ImF0b20iO3M6NTA6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDQvMDkvMjAyMC4wNC4wNy4wMzA1NTEuMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.55" data-doi="10.1371/journal.pone.0097876"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">T</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Laurence</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hatzis</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brash</span>  <span class="cit-name-given-names">DE</span></span>. <span class="cit-article-title">Common Contaminants in Next-Generation Sequencing That Hinder Discovery of Low-Abundance Microbes</span>. <span class="cit-ed"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">T</span></span>, editor. <abbr class="cit-jnl-abbrev">PLoS One</abbr>. <span class="cit-pub-date">2014</span>;<span class="cit-vol">9</span>:<span class="cit-fpage">e97876</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De97876%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0097876%26rft_id%253Dinfo%253Apmid%252F24837716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0097876&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24837716&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030551.1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.56" data-doi="10.1101/2020.03.02.968818"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Manning</span>  <span class="cit-name-given-names">JE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bohl</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lay</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chea</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sovann</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sengdoeurn</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Rapid metagenomic characterization of a case of imported COVID-19 in Cambodia</span>. <abbr class="cit-jnl-abbrev">Biorxiv</abbr>. <span class="cit-pub-date">2020</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1101/2020.03.02.968818</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiorxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2020.03.02.968818%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czoxOToiMjAyMC4wMy4wMi45Njg4MTh2MSI7czo0OiJhdG9tIjtzOjUwOiIvYmlvcnhpdi9lYXJseS8yMDIwLzA0LzA5LzIwMjAuMDQuMDcuMDMwNTUxLjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.57"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Retallack</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Okihiro</span>  <span class="cit-name-given-names">MS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Britton</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Sommeran S</span> <span class="cit-name-surname">Van</span></span>, <span class="cit-auth"><span class="cit-name-surname">DeRisi</span>  <span class="cit-name-given-names">JL</span></span>. <span class="cit-article-title">Metagenomic Next-Generation Sequencing Reveals Miamiensis avidus (Ciliophora: Scuticocilitida) in the 2017 Epizootic of Leopard Sharks (Triakis semifasciata) in San Francisco Bay, California, USA</span>. <abbr class="cit-jnl-abbrev">J Wildl Dis</abbr>. <span class="cit-pub-date">2019</span>;<span class="cit-vol">55</span>:<span class="cit-fpage">375</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BWildl%2BDis%26rft.volume%253D55%26rft.spage%253D375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.58"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Batson</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dudas</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haas-Stapleton</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kistler</span>  <span class="cit-name-given-names">AL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">LM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Logan</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Single mosquito metatranscriptomics recovers mosquito species, blood meal sources, and microbial cargo, including viral dark matter</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>. <span class="cit-pub-date">2020</span>;2020.02.10.942854.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.59" data-doi="10.1038/nmeth.4458"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sczyrba</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hofmann</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Belmann</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koslicki</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Janssen</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dröge</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Critical Assessment of Metagenome Interpretation - A benchmark of metagenomics software</span>. <abbr class="cit-jnl-abbrev">Nat Methods</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">14</span>:<span class="cit-fpage">1063</span>–<span class="cit-lpage">71</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D14%26rft.spage%253D1063%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.4458%26rft_id%253Dinfo%253Apmid%252F28967888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.4458&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28967888&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030551.1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.60" data-doi="10.1186/s13059-016-1145-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McIntyre</span>  <span class="cit-name-given-names">ABR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ounit</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Afshinnekoo</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Prill</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hénaff</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Comprehensive benchmarking and ensemble approaches for metagenomic classifiers</span>. <abbr class="cit-jnl-abbrev">Genome Biol</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">18</span>:<span class="cit-fpage">1</span>–<span class="cit-lpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D18%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-016-1145-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-016-1145-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.61"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zinter</span>  <span class="cit-name-given-names">MS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dvorak</span>  <span class="cit-name-given-names">CC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mayday</span>  <span class="cit-name-given-names">MY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Iwanaga</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ly</span>  <span class="cit-name-given-names">NP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mcgarry</span>  <span class="cit-name-given-names">ME</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Clinical Infectious Diseases Pulmonary Metagenomic Sequencing Suggests Missed Infections in Immunocompromised Children</span>. <abbr class="cit-jnl-abbrev">Clin Infect Dis</abbr>. <span class="cit-pub-date">2019</span>:<span class="cit-vol">1847</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">62.</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.07.030551v2.62"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">GitHub - chanzuckerberg/idseq-bench: IDseq infectious disease benchmarking tools. [cited 2020 Mar 11]</span>. Available from: <a href="https://github.com/chanzuckerberg/idseq-bench">https://github.com/chanzuckerberg/idseq-bench</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">63.</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.07.030551v2.63"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gourlé</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Karlsson-Lindsjö</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hayer</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bongcam-Rudloff</span>  <span class="cit-name-given-names">E.</span></span> <span class="cit-source">Simulating Illumina metagenomic data with InSilicoSeq. [cited 2020 Feb 26]; Available from</span>: <a href="https://github.com/HadrienG/InSilicoSeq">https://github.com/HadrienG/InSilicoSeq</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">64.</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.64"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Petitjean</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vanet</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">VIRAPOPS2 supports the influenza virus reassortments</span>. <abbr class="cit-jnl-abbrev">Biol Med</abbr>. <span class="cit-pub-date">2014</span>;<span class="cit-vol">9</span>:<span class="cit-fpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol%2BMed%26rft.volume%253D9%26rft.spage%253D18%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">65.</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.65"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">O’leary</span>  <span class="cit-name-given-names">NA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">MW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brister</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ciufo</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haddad</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mcveigh</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Reference sequence (RefSeq) database at NCBI: current status, taxonomic expansion, and functional annotation</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-pub-date">2015</span>;<span class="cit-vol">44</span>:<span class="cit-fpage">733</span>–<span class="cit-lpage">45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D44%26rft.spage%253D733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><div class="section ref-list" id="ref-list-2"><h2 class="">Supplemental References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 1. in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.66"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guentzel</span>  <span class="cit-name-given-names">MN</span></span>. <span class="cit-article-title">Escherichia, Klebsiella, Enterobacter, Serratia, Citrobacter, and Proteus</span>. <abbr class="cit-jnl-abbrev">Med. Microbiol</abbr>. <span class="cit-pub-date">1996</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference 2. in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.67" data-doi="10.1186/s13059-019-1891-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">DE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">Improved metagenomic analysis with Kraken 2</span>. <abbr class="cit-jnl-abbrev">Genome Biol</abbr>. <span class="cit-pub-date">2019</span>;<span class="cit-vol">20</span>:<span class="cit-fpage">257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D20%26rft.spage%253D257%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-019-1891-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-019-1891-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference 3. in text" id="ref-68">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.07.030551v2.68"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gourlé</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Karlsson-Lindsjö</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hayer</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bongcam-Rudloff</span>  <span class="cit-name-given-names">E.</span></span> <span class="cit-source">Simulating Illumina metagenomic data with InSilicoSeq. [cited 2020 Feb 26]; Available from</span>: <a href="https://github.com/HadrienG/InSilicoSeq">https://github.com/HadrienG/InSilicoSeq</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference 4. in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.69" data-doi="10.1007/s00284-010-9809-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Manter</span>  <span class="cit-name-given-names">DK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">WJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vivanco</span>  <span class="cit-name-given-names">JM</span></span>. <span class="cit-article-title">Enterobacter soli sp. nov.: A lignin-degrading γ-Proteobacteria isolated from soil</span>. <abbr class="cit-jnl-abbrev">Curr Microbiol</abbr>. <span class="cit-pub-date">2011</span>;<span class="cit-vol">62</span>:<span class="cit-fpage">1044</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bmicrobiology%26rft.stitle%253DCurr%2BMicrobiol%26rft.aulast%253DManter%26rft.auinit1%253DD.%2BK.%26rft.volume%253D62%26rft.issue%253D3%26rft.spage%253D1044%26rft.epage%253D1049%26rft.atitle%253DEnterobacter%2Bsoli%2Bsp.%2Bnov.%253A%2Ba%2Blignin-degrading%2B%25CE%25B3-proteobacteria%2Bisolated%2Bfrom%2Bsoil.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00284-010-9809-9%26rft_id%253Dinfo%253Apmid%252F21104086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00284-010-9809-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21104086&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030551.1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference 5. in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.70" data-doi="10.1093/bioinformatics/btw542"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ounit</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lonardi</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Higher classification sensitivity of short metagenomic reads with CLARK-S</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">32</span>:<span class="cit-fpage">3823</span>–<span class="cit-lpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw542%26rft_id%253Dinfo%253Apmid%252F27540266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw542&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27540266&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F04%2F09%2F2020.04.07.030551.1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference 6. in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.71" data-doi="10.1186/s12864-016-3382-y"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bosi</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fondi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Orlandini</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Perrin</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maida</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Pascale</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">The pangenome of (Antarctic) Pseudoalteromonas bacteria: Evolutionary and functional insights</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">18</span>:<span class="cit-fpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D18%26rft.spage%253D93%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-016-3382-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-016-3382-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference 7. in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.72"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pellegrinelli</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Giardina</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lunghi</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Renteria</span>  <span class="cit-name-given-names">SCU</span></span>, <span class="cit-auth"><span class="cit-name-surname">Greco</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fratini</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Emergence of divergent enterovirus (EV) D68 sub-clade D1 strains, northern Italy, September to October 2018</span>. <abbr class="cit-jnl-abbrev">Eurosurveillance</abbr>. <span class="cit-pub-date">2019</span>;<span class="cit-vol">24</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference 8. in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.73"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ahamed</span>  <span class="cit-name-given-names">SF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rosario</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Britto</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dias</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nayak</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chandele</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Emergence of new genotypes and lineages of dengue viruses during the 2012–15 epidemics in southern India</span>. <abbr class="cit-jnl-abbrev">Int J Infect Dis</abbr>. <span class="cit-pub-date">2019</span>;<span class="cit-vol">84</span>:<span class="cit-fpage">S34</span>–<span class="cit-lpage">43</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt%2BJ%2BInfect%2BDis%26rft.volume%253D84%26rft.spage%253DS34%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference 9. in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.74"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Jesus JG</span> <span class="cit-name-surname">de</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dutra</span>  <span class="cit-name-given-names">KR</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Salles FC</span> <span class="cit-name-surname">da S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Claro</span>  <span class="cit-name-given-names">IM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Terzian</span>  <span class="cit-name-given-names">AC</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Candido D</span> <span class="cit-name-surname">da S</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Early identification of dengue virus lineage replacement in Brazil using portable genomic surveillance</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>. <span class="cit-pub-date">2019</span>;<span class="cit-vol">716159</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference 10. in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.75"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fall</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Di Paola</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Faye</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dia</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Freire CC</span> <span class="cit-name-surname">de M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Loucoubar</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Biological and phylogenetic characteristics of West African lineages of West Nile virus</span>. <abbr class="cit-jnl-abbrev">PLoS Negl Trop Dis</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">11</span>:<span class="cit-fpage">e0006078</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BNegl%2BTrop%2BDis%26rft.volume%253D11%26rft.spage%253De0006078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference 11. in text" id="ref-76">↵</a><div class="cit ref-cit ref-book" id="cit-2020.04.07.030551v2.76"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">HH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">RG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bond</span>  <span class="cit-name-given-names">PJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grad</span>  <span class="cit-name-given-names">YH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Camerini</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maurer-Stroh</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> <span class="cit-source">Systematic analysis of protein identity between Zika virus and other arthropod-borne viruses</span>. <span class="cit-publ-name">World Health Organization</span>; <span class="cit-pub-date">2017</span>;<span class="cit-vol">95</span>:<span class="cit-fpage">517</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2Banalysis%2Bof%2Bprotein%2Bidentity%2Bbetween%2BZika%2Bvirus%2Band%2Bother%2Barthropod-borne%2Bviruses%26rft.volume%253D95%26rft.spage%253D517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference 12. in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.77"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Batson</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dudas</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haas-Stapleton</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kistler</span>  <span class="cit-name-given-names">AL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">LM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Logan</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Single mosquito metatranscriptomics recovers mosquito species, blood meal sources, and microbial cargo, including viral dark matter</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>. <span class="cit-pub-date">2020</span>;2020.02.10.942854.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference 13. in text" id="ref-78">↵</a><div class="cit ref-cit ref-web" id="cit-2020.04.07.030551v2.78"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Ribosomal RNA - an overview | ScienceDirect Topics [Internet]</span>. [cited 2020 Feb 26]. Available from: <a href="https://www.sciencedirect.com/topics/neuroscience/ribosomal-rna">https://www.sciencedirect.com/topics/neuroscience/ribosomal-rna</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference 14. in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.04.07.030551v2.79"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cottier</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Srinivasan</span>  <span class="cit-name-given-names">KG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yurieva</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liao</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poidinger</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zolezzi</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Advantages of meta-total RNA sequencing (MeTRS) over shotgun metagenomics and amplicon-based sequencing in the profiling of complex microbial communities</span>. <abbr class="cit-jnl-abbrev">Biofilms Microbiomes</abbr>. <span class="cit-pub-date">2018</span>;<span class="cit-vol">4</span>:<span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiofilms%2BMicrobiomes%26rft.volume%253D4%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.04.07.030551v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.04.07.030551v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.04.09.034009v1" title="Metagenomic Analysis of Bacterial Community Associated with Rhizosphere and Phyllosphere of Basmati Rice" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2020.03.05.976712v2" title="Active Vision in Immersive, 360° Real-World Environments" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;April 09, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.04.07.030551v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.04.07.030551v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.04.07.030551v2" class="active" data-icon-position="" data-hide-link-title="0">IDseq – An Open Source Cloud-based Pipeline and Analysis Service for Metagenomic Pathogen Detection and Monitoring</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1237378" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-oOR7olxvfYCmRxmVMq4NCBVDf6Mj_oygZ7lXNcgYC1U" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1237378" id="node1237378--2" data-pisa="biorxiv;2020.04.07.030551v2" data-pisa-master="biorxiv;2020.04.07.030551" data-seqnum="1237378" data-apath="/biorxiv/early/2020/04/09/2020.04.07.030551.1.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">IDseq – An Open Source Cloud-based Pipeline and Analysis Service for Metagenomic Pathogen Detection and Monitoring</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Katrina L.</span> <span class="nlm-surname">Kalantar</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Tiago</span> <span class="nlm-surname">Carvalho</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Charles F.A.</span> <span class="nlm-surname">de Bourcy</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Boris</span> <span class="nlm-surname">Dimitrov</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Greg</span> <span class="nlm-surname">Dingle</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Rebecca</span> <span class="nlm-surname">Egger</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Julie</span> <span class="nlm-surname">Han</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Olivia B.</span> <span class="nlm-surname">Holmes</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Yun-Fang</span> <span class="nlm-surname">Juan</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Ryan</span> <span class="nlm-surname">King</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Andrey</span> <span class="nlm-surname">Kislyuk</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Maria</span> <span class="nlm-surname">Mariano</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Lucia V.</span> <span class="nlm-surname">Reynoso</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">David Rissato</span> <span class="nlm-surname">Cruz</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Jonathan</span> <span class="nlm-surname">Sheu</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Jennifer</span> <span class="nlm-surname">Tang</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">James</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="17"><span class="nlm-given-names">Mark A.</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="18"><span class="nlm-given-names">Emily</span> <span class="nlm-surname">Zhong</span></span>, <span class="highwire-citation-author" data-delta="19"><span class="nlm-given-names">Vida</span> <span class="nlm-surname">Ahyong</span></span>, <span class="highwire-citation-author" data-delta="20"><span class="nlm-given-names">Sreyngim</span> <span class="nlm-surname">Lay</span></span>, <span class="highwire-citation-author" data-delta="21"><span class="nlm-given-names">Sophana</span> <span class="nlm-surname">Chea</span></span>, <span class="highwire-citation-author" data-delta="22"><span class="nlm-given-names">Jennifer A.</span> <span class="nlm-surname">Bohl</span></span>, <span class="highwire-citation-author" data-delta="23"><span class="nlm-given-names">Jessica E.</span> <span class="nlm-surname">Manning</span></span>, <span class="highwire-citation-author" data-delta="24"><span class="nlm-given-names">Cristina M.</span> <span class="nlm-surname">Tato</span></span>, <span class="highwire-citation-author" data-delta="25"><span class="nlm-given-names">Joseph L.</span> <span class="nlm-surname">DeRisi</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.04.07.030551; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.04.07.030551 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.04.07.030551v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.04.07.030551v2&amp;title=IDseq%20%E2%80%93%20An%20Open%20Source%20Cloud-based%20Pipeline%20and%20Analysis%20Service%20for%20Metagenomic%20Pathogen%20Detection%20and%20Monitoring" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.07.030551v2&amp;title=IDseq%20%E2%80%93%20An%20Open%20Source%20Cloud-based%20Pipeline%20and%20Analysis%20Service%20for%20Metagenomic%20Pathogen%20Detection%20and%20Monitoring" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.07.030551v2&amp;text=IDseq%20%E2%80%93%20An%20Open%20Source%20Cloud-based%20Pipeline%20and%20Analysis%20Service%20for%20Metagenomic%20Pathogen%20Detection%20and%20Monitoring" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.07.030551v2&amp;title=IDseq%20%E2%80%93%20An%20Open%20Source%20Cloud-based%20Pipeline%20and%20Analysis%20Service%20for%20Metagenomic%20Pathogen%20Detection%20and%20Monitoring" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.04.07.030551v2&amp;t=IDseq%20%E2%80%93%20An%20Open%20Source%20Cloud-based%20Pipeline%20and%20Analysis%20Service%20for%20Metagenomic%20Pathogen%20Detection%20and%20Monitoring" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.04.07.030551v2&amp;title=IDseq%20%E2%80%93%20An%20Open%20Source%20Cloud-based%20Pipeline%20and%20Analysis%20Service%20for%20Metagenomic%20Pathogen%20Detection%20and%20Monitoring" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.07.030551v2&amp;title=IDseq%20%E2%80%93%20An%20Open%20Source%20Cloud-based%20Pipeline%20and%20Analysis%20Service%20for%20Metagenomic%20Pathogen%20Detection%20and%20Monitoring" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1237378" id="node1237378--3" data-pisa="biorxiv;2020.04.07.030551v2" data-pisa-master="biorxiv;2020.04.07.030551" data-seqnum="1237378" data-apath="/biorxiv/early/2020/04/09/2020.04.07.030551.1.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">IDseq – An Open Source Cloud-based Pipeline and Analysis Service for Metagenomic Pathogen Detection and Monitoring</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Katrina L.</span> <span class="nlm-surname">Kalantar</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Tiago</span> <span class="nlm-surname">Carvalho</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Charles F.A.</span> <span class="nlm-surname">de Bourcy</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Boris</span> <span class="nlm-surname">Dimitrov</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Greg</span> <span class="nlm-surname">Dingle</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Rebecca</span> <span class="nlm-surname">Egger</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Julie</span> <span class="nlm-surname">Han</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Olivia B.</span> <span class="nlm-surname">Holmes</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Yun-Fang</span> <span class="nlm-surname">Juan</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Ryan</span> <span class="nlm-surname">King</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Andrey</span> <span class="nlm-surname">Kislyuk</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Maria</span> <span class="nlm-surname">Mariano</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Lucia V.</span> <span class="nlm-surname">Reynoso</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">David Rissato</span> <span class="nlm-surname">Cruz</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Jonathan</span> <span class="nlm-surname">Sheu</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Jennifer</span> <span class="nlm-surname">Tang</span></span>, <span class="highwire-citation-author" data-delta="16"><span class="nlm-given-names">James</span> <span class="nlm-surname">Wang</span></span>, <span class="highwire-citation-author" data-delta="17"><span class="nlm-given-names">Mark A.</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="18"><span class="nlm-given-names">Emily</span> <span class="nlm-surname">Zhong</span></span>, <span class="highwire-citation-author" data-delta="19"><span class="nlm-given-names">Vida</span> <span class="nlm-surname">Ahyong</span></span>, <span class="highwire-citation-author" data-delta="20"><span class="nlm-given-names">Sreyngim</span> <span class="nlm-surname">Lay</span></span>, <span class="highwire-citation-author" data-delta="21"><span class="nlm-given-names">Sophana</span> <span class="nlm-surname">Chea</span></span>, <span class="highwire-citation-author" data-delta="22"><span class="nlm-given-names">Jennifer A.</span> <span class="nlm-surname">Bohl</span></span>, <span class="highwire-citation-author" data-delta="23"><span class="nlm-given-names">Jessica E.</span> <span class="nlm-surname">Manning</span></span>, <span class="highwire-citation-author" data-delta="24"><span class="nlm-given-names">Cristina M.</span> <span class="nlm-surname">Tato</span></span>, <span class="highwire-citation-author" data-delta="25"><span class="nlm-given-names">Joseph L.</span> <span class="nlm-surname">DeRisi</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.04.07.030551; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.04.07.030551 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=IDseq%20%E2%80%93%20An%20Open%20Source%20Cloud-based%20Pipeline%20and%20Analysis%20Service%20for%20Metagenomic%20Pathogen%20Detection%20and%20Monitoring&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Kalantar%2C+Katrina+L.&amp;rft.au=Carvalho%2C+Tiago&amp;rft.au=de+Bourcy%2C+Charles+F.A.&amp;rft.au=Dimitrov%2C+Boris&amp;rft.au=Dingle%2C+Greg&amp;rft.au=Egger%2C+Rebecca&amp;rft.au=Han%2C+Julie&amp;rft.au=Holmes%2C+Olivia+B.&amp;rft.au=Juan%2C+Yun-Fang&amp;rft.au=King%2C+Ryan&amp;rft.au=Kislyuk%2C+Andrey&amp;rft.au=Mariano%2C+Maria&amp;rft.au=Reynoso%2C+Lucia+V.&amp;rft.au=Cruz%2C+David+Rissato&amp;rft.au=Sheu%2C+Jonathan&amp;rft.au=Tang%2C+Jennifer&amp;rft.au=Wang%2C+James&amp;rft.au=Zhang%2C+Mark+A.&amp;rft.au=Zhong%2C+Emily&amp;rft.au=Ahyong%2C+Vida&amp;rft.au=Lay%2C+Sreyngim&amp;rft.au=Chea%2C+Sophana&amp;rft.au=Bohl%2C+Jennifer+A.&amp;rft.au=Manning%2C+Jessica+E.&amp;rft.au=Tato%2C+Cristina+M.&amp;rft.au=DeRisi%2C+Joseph+L."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1237378/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1237378/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1237378/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1237378/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1237378/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1237378/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1237378/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1237378/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1237378/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1237378/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1237378/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1237378/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.04.07.030551v2&amp;count=horizontal&amp;via=&amp;text=IDseq%20%E2%80%93%20An%20Open%20Source%20Cloud-based%20Pipeline%20and%20Analysis%20Service%20for%20Metagenomic%20Pathogen%20Detection%20and%20Monitoring&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.04.07.030551v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.04.07.030551v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.04.07.030551v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">Bioinformatics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
