<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2019/11/26/856518/embed/graphic-2.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/11/26/856518.atom" />
<meta name="HW.pisa" content="biorxiv;856518v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Quantitative Delineation of Herpesviruses in Bats for use in Ecological Studies" />
<meta name="DC.Identifier" content="10.1101/856518" />
<meta name="DC.Date" content="2019-11-26" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Public health concerns about recent viral epidemics have motivated researchers to seek transdisciplinary understanding of infection in wildlife hosts. With its deep history devoted to explaining the abundance and distribution of organisms, ecology can augment current methods for studying viral dynamics. However, datasets allowing ecological explorations of viral communities are lacking, and common methods for delineating viral operational taxonomic units (OTUs), or “species”, are subjective. Here, we comprehensively sampled 1,086 bats from two Puerto Rican caves and tested them for infection with herpesviruses. Using percent identity of nucleotides and a machine learning algorithm, we categorized herpesviruses into 41 OTUs, representing approximately 80% of all herpesviruses in the host community. Although 13 OTUs were detected in multiple host species, OTUs generally exhibited host specificity by infecting a core host species at a significantly higher prevalence than in all other species combined. Only two OTUs showed significantly different prevalence between host sexes. This work is the first exploration of viral community ecology in a community of wildlife hosts." />
<meta name="DC.Contributor" content="Anna R. Sjodin" />
<meta name="DC.Contributor" content="Michael R. Willig" />
<meta name="DC.Contributor" content="Simon J. Anthony" />
<meta name="article:published_time" content="2019-11-26" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Quantitative Delineation of Herpesviruses in Bats for use in Ecological Studies" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Public health concerns about recent viral epidemics have motivated researchers to seek transdisciplinary understanding of infection in wildlife hosts. With its deep history devoted to explaining the abundance and distribution of organisms, ecology can augment current methods for studying viral dynamics. However, datasets allowing ecological explorations of viral communities are lacking, and common methods for delineating viral operational taxonomic units (OTUs), or “species”, are subjective. Here, we comprehensively sampled 1,086 bats from two Puerto Rican caves and tested them for infection with herpesviruses. Using percent identity of nucleotides and a machine learning algorithm, we categorized herpesviruses into 41 OTUs, representing approximately 80% of all herpesviruses in the host community. Although 13 OTUs were detected in multiple host species, OTUs generally exhibited host specificity by infecting a core host species at a significantly higher prevalence than in all other species combined. Only two OTUs showed significantly different prevalence between host sexes. This work is the first exploration of viral community ecology in a community of wildlife hosts.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;856518v1" />
<meta name="citation_id" content="856518v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/856518v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/856518v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/856518v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/11/26/856518.full.pdf" />
<meta name="citation_doi" content="10.1101/856518" />
<meta name="citation_num_pages" content="26" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="856518" />
<meta name="citation_author" content="Anna R. Sjodin" />
<meta name="citation_author_institution" content="Department of Ecology &amp; Evolutionary Biology, University of Connecticut" />
<meta name="citation_author_institution" content="Department of Biological Sciences, University of Idaho" />
<meta name="citation_author_email" content="annasjodin03@gmail.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5044-4193" />
<meta name="citation_author" content="Michael R. Willig" />
<meta name="citation_author_institution" content="Department of Ecology &amp; Evolutionary Biology, University of Connecticut" />
<meta name="citation_author_institution" content="Center for Environmental Sciences &amp; Engineering and Institute of the Environment, University of Connecticut" />
<meta name="citation_author" content="Simon J. Anthony" />
<meta name="citation_author_institution" content="Center for Infection and Immunity, Columbia University" />
<meta name="citation_reference" content="Anthony SJ, Epstein JH, Murray KA, Navarrete-Macias I, Zambrana-Torrelio CM, Solovyov A, Ojeda-Flores R, Arrigo NC, Islam A, Khan SA, Hosseini P, Bogich TL, Olival KJ, Sanchez-Leon MD, Karesh WB, Goldstein T, Luby SP, Morse SS, Mazet JAK, Daszak P, Lipkin WI (2013) A strategy to estimate unknown viral diversity in mammals. mBio 4: e00598–13." />
<meta name="citation_reference" content="Anthony SJ, Islam A, Johnson C, Navarrete-Macias I, Liang E, Jain K, Hitchens PL, Che X, Soloyvov A, Hicks AL, Ojeda-Flores R, Zambrana-Torrelio C, Ulrich W, Rostal MK, Petrosov A, Garcia J, Haider N, Wolfe N, Goldstein T, Morse SS, Rahman M, Epstein JH, Mazet JK, Daszak P, Lipkin WI (2015) Non-random patterns in viral diversity. Nature Communications 6: 8147." />
<meta name="citation_reference" content="Anthony SJ, Johnson CK, Greig DJ, Kramer S, Che X, Wells H, Hicks AL, Joly DO, Wolfe ND, Daszak P, Karesh W, Lipkin WI, Morse SS, PREDICT Consortium, Mazet JAK, Goldstein T (2017) Global patterns in coronavirus diversity. Virus Evolution 3: vex012." />
<meta name="citation_reference" content="Azab W, Dayaram A, Greenwood AD, Osterrieder N (2018) How host specific are herpesviruses? Lessons from herpesviruses infecting wild and endangered mammals. Annual Review of Virology 5: 53–68." />
<meta name="citation_reference" content="Bodenhofer U, Kothmeier A, Hochreiter S (2011) APCluster: a R package for affinity propagation clustering. Bioinformatics 17: 2463–2464." />
<meta name="citation_reference" content="Bush AO, Holmes JC (1986) Intestinal helminths of lesser scaup ducks: an interactive community. Canadian Journal of Zoology 64: 142–152." />
<meta name="citation_reference" content="Chao A (1987) Estimating the population size for capture-recapture data with unequal catchability. Biometrics 43: 783–791." />
<meta name="citation_reference" content="Cortez PP, Carvalheira J, Pauperio S, Thompson G (2008) Prevalence of ovine herpesvirus type 2 in north-west Portugal. Veterinary Record 162: 282–284." />
<meta name="citation_reference" content="De Benedictis P, Schultz-Cherry S, Burnham A, Cattoli G (2011) Astrovirus infections in humans and animals – molecular biology, genetic diversity, and interspecies transmissions. Infection, Genetics and Evolution 11: 1529–1544." />
<meta name="citation_reference" content="Escalera-Zamudio M, Rojas-Anaya E, Kolokotronis SO, Taboada B, Loza-Rubio E, Mendez-Ojeda ML, Arias CF, Osterrieder N, Greenwood AD (2016) Bats, primates, and the evolutionary origins and diversification of mammalian Gammaherpesviruses. mBio 7: e01425–16." />
<meta name="citation_reference" content="Fischer S, Freuling CM, Muller T, Pfaff F, Bodenhofer U, Hoper D, Fischer M, Marston DA, Fooks AR, Mettenleiter TC, Conraths FJ, and Homeier-Bachmann T (2017) Defining objective clusters for rabies virus sequences using affinity propagation clustering. PLoS Neglected Tropical Diseases 12: e0006182." />
<meta name="citation_reference" content="Frey BJ and Dueck D (2007) Clustering by passing messages between data points. Science 315: 972–976." />
<meta name="citation_reference" content="Gannon MR, Kurta A, Rodríguez-Duran A, and Willig MR (2005) Bats of Puerto Rico: An Island Focus and a Caribbean Perspective. Lubbock, TX: Texas Tech University Press." />
<meta name="citation_reference" content="Greenblatt RJ, Quackenbush SL, Casey RN, Rovnak J, Balazs GH, Work TM, Casey JW, Sutton CA (2005) Genomic variation of the fibropapilloma-associated marine turtle herpesvirus across seven geographic areas and three host species. Journal of Virology 79: 1125–1132." />
<meta name="citation_reference" content="Henaux V, Samuel MD (2011) Avian influenza shedding patterns in waterfowl: implications for surveillance, environmental transmission, and diseases spread. Journal of Wildlife Diseases 47: 566–578." />
<meta name="citation_reference" content="Holz PH, Lumsden LF, Druce J, Legione AR, Vaz P, Devlin JM, Hufschmid J (2018) Virus survey in populations of two subspecies of bent-winged bats (Miniopterus orianae bassanii and oceanensis) in south-eastern Australia reveals a high prevalence of diverse herpesviruses. PLoS ONE 13: e0197625." />
<meta name="citation_reference" content="Hsieh TC, Ma KH, Chao A (2016) iNEXT: an R package for rarefaction and extrapolation of species diversity (Hill numbers). Methods in Ecology and Evolution 7: 1451–1456." />
<meta name="citation_reference" content="Jolles AE, Ezenwa VO, Etienne RS, Turner WC, Olff H (2008) Interactions between macroparasites and microparasites drive infection patterns in free-ranging African buffalo. Ecology 89: 2239–2250." />
<meta name="citation_reference" content="King AMQ, Adams MJ, Carstens EB, Lefkowitz EJ (2012) Virus Taxonomy: Classification and Nomenclature of Viruses. Ninth report of the International Committee on Taxonomy of Viruses, San Diego, CA: Academic Press." />
<meta name="citation_reference" content="Klenk K, Snow J, Morgan K, Bowen R, Stephens M, Foster F, Gordy P, Beckett S, Komar N, Gubler D, Bunning M (2004) Alligators as West Nile virus amplifiers. Emerging Infectious Diseases 10: 2150–2155." />
<meta name="citation_reference" content="Lee N, Chan PKS, Hui DSC, Rainer TH, Wong E, Choi KW, Lui GCY, Wong BCK, Wong RYK, Lam WY, Chu IMT, Lai RWM, Cockram CS, Sung JJY (2009) Viral loads and duration of viral shedding in adult patients hospitalized with influenza. The Journal of Infectious Diseases 200: 492–500." />
<meta name="citation_reference" content="Maes P, Klempa B, Clement J, Matthijnssens J, Gajdusek DC, Kruger DH, Van Ranst M (2009) A proposal for new criteria for the classification of hantaviruses, based on S and M segment protein sequences. Infection, Genetics and Evolution 9: 813–820." />
<meta name="citation_reference" content="Pages H, Aboyoun P, Gentleman R, DebRoy S (2017) Biostrings: Efficient manipulation of biological strings. R package version 2.46.0." />
<meta name="citation_reference" content="Pedersen AB, Fenton A (2006) Emphasizing the ecology in parasite community ecology. Trends in Ecology and Evolution 22: 133–139." />
<meta name="citation_reference" content="Phalen DN, Alvarado C, Grillo V, Mason P, Dobson E, Holz P (2017) Prevalence of columbid herpesvirus infection in feral pigeons from New South Wales and Victoria, Australia, with spillover into a wild powerful owl (Ninox struena). Journal of Wildlife Diseases 53: 543–551." />
<meta name="citation_reference" content="Poulin R, Mouillot D (2005) Combining phylogenetic and ecological information into a new index of host specificity. Journal of Parasitology 91: 511–514." />
<meta name="citation_reference" content="Pozo F, Juste J, Vazquez-Moron S, Aznar-Lopez C, Ibanez C, Garin I, Aihartza J, Casas I, Tenorio A, Echevarria JE (2016) Identification of novel betaherpesviruses in Iberian bats reveals parallel evolution. PLoS ONE 11: e0169153." />
<meta name="citation_reference" content="Razafindratsimandresy R, Jeanmaire EM, Counor D, Vasconcelos PF, Sall AA, Reynes JM (2009) Partial molecular characterization of alphaherpesviruses isolated from tropical bats. Journal of General Virology 90: 44–47." />
<meta name="citation_reference" content="citation_title=In Island Bats: Evolution, Ecology, and Conservation;citation_pages=265-280;citation_year=2009" />
<meta name="citation_reference" content="Russell GC, Stewart JP, Haig DM (2009) Malignant catarrhal fever: a review. The Veterinary Journal 179: 324–335." />
<meta name="citation_reference" content="Sasaki M, Setiyono A, Handharyani E, Kobayashi S, Rahmadani I, Taha S, Adiani S, Subangkit M, Nakamura I, Sawa H, Kimura T (2014) Isolation and characterization of a novel alphaherpesvirus in fruit bats. Journal of Virology 88: 9819–9829." />
<meta name="citation_reference" content="Seabloom EW, Borer ET, Gross K, Kendig AE, Lacroix C, Mitchell CE, Modrecai EA, Power AG (2015) The community ecology of pathogens: coinfection, coexistence and community composition. Ecology Letters 18: 401–415." />
<meta name="citation_reference" content="Tandler B (1996) Cytomegalovirus in the principal submandibular gland of the little brown bat, Myotis lucifugus. Journal of Comparative Pathology 114: 1–9." />
<meta name="citation_reference" content="Tang XC, Zhang JX, Zhang SY, Wang P, Fan XH, Li LF, Li G, Dong BQ, Liu W, Cheung CL, Xu KM, Song WJ, Vijaykrishna D, Poon LLM, Peiris JSM, Smith GJD, Chen H, Guan Y (2006) Prevalence and genetic diversity of coronaviruses in bats from China. Journal of Virology 80: 7481–7490." />
<meta name="citation_reference" content="Tazikeh A, Raoofi A, Madadgar O, Akbarein H, Mashhadi AG. 2019. A survey of equine herpes virus 4 infection in four provinces of Iran using Real Time PCR Taqman assay. Journal of Veterinary Research 73: 483–490." />
<meta name="citation_reference" content="Telfer S, Lambin X, Birtles R, Beldomenico P, Burthe S, Paterson S, Begon M (2010) Species interactions in a parasite community drive infection risk in a wildlife population. Science 330: 243–246." />
<meta name="citation_reference" content="Van De Vanter DR, Warrener P, Bennett L, Schultz ER, Coulter S, Garber RL, Rose TM (1996) Detection and analysis of diverse herpesviral species by consensus primer PCR. Journal of Clinical Microbiology 34: 1666–1671." />
<meta name="citation_reference" content="Vetter P, Fischer II WA, Schibler M, Jacobs M, Bausch DG, Kaiser L (2016) Ebola virus shedding and transmission: review of current evidence. The Journal of Infectious Diseases 214: S177–S184." />
<meta name="citation_reference" content="Wacharapluesadee S, Boongrid K, Wanghongsa S, Ratanasetyuth N, Supavongwong P, Saengsen D, Gongal GN, Hemachudha T (2010) A longitudinal study of the prevalence of Nipah virus in Pteropus lylei bats in Thailand: evidence for seasonal preference in disease transmission. Vector-Borne and Zoonotic Diseases 10: 183–190." />
<meta name="citation_reference" content="Wada Y, Sasaki M, Setiyono A, Handharyani E, Rahmadani I, Taha S, Adiani S, Latief M, Kholilullah ZA, Subangkit M, Kobayashi S, Nakamura I, Kimura T, Orba Y, Sawa H (2018) Detection of novel gammaherpesviruses from fruit bats in Indonesia. Journal of Medical Microbiology 67: 415–422." />
<meta name="citation_reference" content="Wassertheil-Smoller S, Smoller J (2015) Biostatistics and Epidemiology: A Primer for Health and Biomedical Professionals Fourth Edition, New York, NY: Springer." />
<meta name="citation_reference" content="Weigler BJ (1992) Biology of B virus in macaque and human hosts: a review. Clinical Infectious Diseases 14: 555–567." />
<meta name="citation_reference" content="Wibbelt G, Kurth A, Yasmum N, Bannert M, Nagel S, Nitsche A, Ehlers B (2007) Discovery of herpesviruses in bats. Journal of General Virology 88: 2651–2655." />
<meta name="citation_reference" content="Wibbelt G, Moore MS, Schountz T, Voigt CC (2010) Emerging diseases in Chiroptera: why bats? Biology Letters 6: 438–440." />
<meta name="citation_reference" content="Winker K, Spackman E, Swayne DE. Rarity of Influenza A virus in spring shorebirds, southern Alaska. Emerging Infectious Diseases 14: 1314–1316." />
<meta name="citation_reference" content="Zhang H, Todd S, Tachedijian M, Barr JA, Luo M, Yu M, Marsh GA, Crameri G, Wang L (2012) A novel bat herpesvirus encodes homologues of major histocompatibility complex class I and II, c-type lectin, and a unique family of immune-related genes. Journal of Virology 86: 8014–8030." />
<meta name="citation_reference" content="Zheng X, Qiu M, Chen S, Ziao J, Ma L, Liu S, Zhou J, Zhang Q, Li X, Chen Z, Wu Y, Chen H, Jiang L, Xiong Y, Ma S, Zhong X, Huo S, Ge J, Cen S, Chen Q (2016) High prevalence and diversity of viruses in the subfamily Gammaherpesvirinae, family Herpesviridae, in fecal specimens from bats of different species in southern China. Archives of Virology 161: 135–140." />
<meta name="twitter:title" content="Quantitative Delineation of Herpesviruses in Bats for use in Ecological Studies" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Public health concerns about recent viral epidemics have motivated researchers to seek transdisciplinary understanding of infection in wildlife hosts. With its deep history devoted to explaining the abundance and distribution of organisms, ecology can augment current methods for studying viral dynamics. However, datasets allowing ecological explorations of viral communities are lacking, and common methods for delineating viral operational taxonomic units (OTUs), or “species”, are subjective. Here, we comprehensively sampled 1,086 bats from two Puerto Rican caves and tested them for infection with herpesviruses. Using percent identity of nucleotides and a machine learning algorithm, we categorized herpesviruses into 41 OTUs, representing approximately 80% of all herpesviruses in the host community. Although 13 OTUs were detected in multiple host species, OTUs generally exhibited host specificity by infecting a core host species at a significantly higher prevalence than in all other species combined. Only two OTUs showed significantly different prevalence between host sexes. This work is the first exploration of viral community ecology in a community of wildlife hosts." />
<meta name="og-title" property="og:title" content="Quantitative Delineation of Herpesviruses in Bats for use in Ecological Studies" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/856518v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Public health concerns about recent viral epidemics have motivated researchers to seek transdisciplinary understanding of infection in wildlife hosts. With its deep history devoted to explaining the abundance and distribution of organisms, ecology can augment current methods for studying viral dynamics. However, datasets allowing ecological explorations of viral communities are lacking, and common methods for delineating viral operational taxonomic units (OTUs), or “species”, are subjective. Here, we comprehensively sampled 1,086 bats from two Puerto Rican caves and tested them for infection with herpesviruses. Using percent identity of nucleotides and a machine learning algorithm, we categorized herpesviruses into 41 OTUs, representing approximately 80% of all herpesviruses in the host community. Although 13 OTUs were detected in multiple host species, OTUs generally exhibited host specificity by infecting a core host species at a significantly higher prevalence than in all other species combined. Only two OTUs showed significantly different prevalence between host sexes. This work is the first exploration of viral community ecology in a community of wildlife hosts." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-11-26" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/856518v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/856518v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/856518v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/856518v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1010156" />
    <title>Quantitative Delineation of Herpesviruses in Bats for use in Ecological Studies | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q8qadd" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__uourhtZYS_vAk2DtacwQIGmpE376HP4afgwMlmHZpq0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__78WRxe8YsWkYp6RcQ_VwTjFG__x9mi_NTPqoxAwFUsA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__dAeSAHswzRXxn-2n4D6zNPmcZDmwaqqLzE-2ab_8jKw__f_9pwrTE8mmshoJRfurwkQ36QonD_Jiwbop2sscnkrE__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q8qadd"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"dNWgXRRPXIZu_k80HYO39EQr2ZPlhCnKlp8cwFWSRSs","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"markup":[{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;856518v1"},{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;856518v1"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;856518v1"}],"nid":"1010156","apath":"\/biorxiv\/early\/2019\/11\/26\/856518.atom","pisa":"biorxiv;856518v1","processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/856518v1","title":"Quantitative Delineation of Herpesviruses in Bats for use in Ecological Studies","identifier":"node\/1010156"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ZziYlc4W6THKaBOSCygGYZ8RV4Lsl0I\\\\\\\/bAraFdOj+7MLcH8YvYEfpZ6HWsPrnfQ\\\\\\\/g84Ihs57WEC7EINpS3KDdgSmWEarGADovSdxfqVuc4OhfJYHti8Oac2WBKP7UneR2gRaRZz3Fz8gTv5BEstiX\\\\\\\/l0yOkTuWo1wTR99ki5B3wGeV5UxOeLQIi3knTviN1ceEHFjiYuBF0vIETH9JO46iFSeD\\\\\\\/rbRHl1xqMD7gYX0vlKR7S1Fd6lEOwGSd44VEUjMn14S7tj8e7SA78\\\\\\\/+\\\\\\\/qrw2yaUxRVBsMS\\\\\\\/cF7xz4kSTdA2\\\\\\\/nZf3Gf2KadHCHPBt+4\\\\\\\/AOjEYpLdjdN0Buc7b0WCl2p0xeWGsTjGQzZcqHW8BHw6XaSVvYYq1xXGsE9+muviaXf4rTIVWLX3xwNIllDK9EZORoFGWeE9AZEeKGeDVVm7WZwpbK45YQKPp0yQfjvQNznf0\\\\\\\/evatuK8LrZNelQzU9Kp7obT20iFVWQCPgAKHP6XIXEYFpCvqeLb70q2J64rcI5eNK22Fo+6wETJhtycPkM0tVFSbc8cGGsgTGUFGTOStsSHPqHcbUKDpjETvqlY1vbZG9rRjMFAHUC7+0GOa8i\\\\\\\/Zy7xxtj1tSgWbqJhE9m5\\\\\\\/G9+BN9VJ1qENEDkc96RhQVyUT4U1T2ZCnsvSJE\\\\\\\/sjA0Haf7PMCzLacjoXN4n4hRpfvb9EV78dvBMW+xBemqm9XtctIGrO+vBXyJUHSOjyVbjkmykYQY6s75yLiHs38Qs\\\\\\\/fGun7Tndwi3\\\\\\\/8bTgFvigdZXgW3FNJ+xSvklk99re5vR4tssDNdQzZM7uJfV3SkqTSDVqyIxukwCwF4PIVcaFTVsaPn96yVH5+H0gPkCBOLNvJyO2Su+ktk271dw+PZ2GGv7emeKcQcFxln5PuA\\\\\\\/y4T2QeYbrsYoZngWUb7gk5lPj1Y59mECjNRJzMMaxhPwzgCF4rUW74GEamLbcUedCKHj0JyzlaYpreOi0WIfPOgqlwmtee+GNVCQwPGOrMCkiH\\\\\\\/OuoVJj8Lvji3XnNtb9cLk1lGj4rqTelmRjVkQBOEa3qgCo52XpNLY5D4+qavHnh+CyKL1IdWT1ACGk2Sf7HMQRqyrrRoslnvQXH33XTvgzK6spTga7nzGqRUdMHT0xjXoM70fljzXB+OWyuSkSbvLzS0Ex82J7Lkw0Xh3Ec0b6t11Y71cMdAxTnAF92a153q6ydGCf9qxbv\\\\\\\/ZE3qIM2UKds495U8xOnuKUz4zjyQNK8DXNakYAySdMBnGmDxch1MoFzBEeT\\\\\\\/uNDDquyEHoQRwDnonJ9FZE\\\\\\\/fq6LvDJMXlwUaXdrpOEeuUB7ay3QaXhYhvKFsQxsncNRwf\\\u0022,\\\u0022iv\\\u0022:\\\u0022bEXNc8V41CWhYU1KjOf8Qg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002251327f588fff153d5f5b753317e774e573619907aae786895231c3840eda3b53\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/856518v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1010156 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/856518v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vB4Wuwn7vr_kJ7fSx8uiiShotqOAiNeSTgUSCNe9Ge0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1010156" id="node1010156" data-pisa="biorxiv;856518v1" data-pisa-master="biorxiv;856518" data-apath="/biorxiv/early/2019/11/26/856518.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Quantitative Delineation of Herpesviruses in Bats for use in Ecological Studies</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5044-4193" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Anna R.</span> <span class="nlm-surname">Sjodin</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Michael R.</span> <span class="nlm-surname">Willig</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Simon J.</span> <span class="nlm-surname">Anthony</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/856518 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1010156" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Anna R. Sjodin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Ecology &#38; Evolutionary Biology, University of Connecticut</span>, Storrs, CT 06269, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>Department of Biological Sciences, University of Idaho</span>, Moscow, ID 83843, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anna%2BR.%2BSjodin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sjodin%20AR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnna%2BR.%2BSjodin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5044-4193" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Anna R. Sjodin</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:annasjodin03@gmail.com" class="" data-icon-position="" data-hide-link-title="0">annasjodin03@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Michael R. Willig </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>Department of Ecology &#38; Evolutionary Biology, University of Connecticut</span>, Storrs, CT 06269, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Center for Environmental Sciences &#38; Engineering and Institute of the Environment, University of Connecticut</span>, Storrs, CT 06269, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BR.%2BWillig%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Willig%20MR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BR.%2BWillig%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Simon J. Anthony </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Center for Infection and Immunity, Columbia University</span>, New York, NY 10032, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Simon%2BJ.%2BAnthony%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Anthony%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASimon%2BJ.%2BAnthony%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/856518v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1010156" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1010156/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/856518v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1010156" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1010156/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/856518v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1010156" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1010156/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/856518v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1010156" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1010156/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/856518v1.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:1010156" data-trigger="external-links" data-url-enabled="1">Data/Code</a><a href="/panels_ajax_tab/article_tab_data_code/node:1010156/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/856518v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1010156" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1010156/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Public health concerns about recent viral epidemics have motivated researchers to seek transdisciplinary understanding of infection in wildlife hosts. With its deep history devoted to explaining the abundance and distribution of organisms, ecology can augment current methods for studying viral dynamics. However, datasets allowing ecological explorations of viral communities are lacking, and common methods for delineating viral operational taxonomic units (OTUs), or “species”, are subjective. Here, we comprehensively sampled 1,086 bats from two Puerto Rican caves and tested them for infection with herpesviruses. Using percent identity of nucleotides and a machine learning algorithm, we categorized herpesviruses into 41 OTUs, representing approximately 80% of all herpesviruses in the host community. Although 13 OTUs were detected in multiple host species, OTUs generally exhibited host specificity by infecting a core host species at a significantly higher prevalence than in all other species combined. Only two OTUs showed significantly different prevalence between host sexes. This work is the first exploration of viral community ecology in a community of wildlife hosts.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-4">Public health concerns associated with recent and ongoing outbreaks of viruses (e.g., Ebola virus and Zika virus) have motivated researchers to investigate viral diversity in wildlife hosts from previously unexplored perspectives. Ecology has a long history of quantifying the abundance and distribution of organisms and has been increasingly integrated into viral studies (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Anthony et al. 2015</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Seabloom et al. 2015</a>). Traditionally, such efforts have focused on isolated cases of single host-single pathogen relationships (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Telfer et al. 2010</a>). However, infected host individuals generally have multiple pathogens, and infection with one pathogen can affect infection, virulence, or disease emergence of the others (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Pedersen and Fenton 2006</a>, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Jolles et al. 2008</a>, <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Telfer et al. 2010</a>, <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Seabloom et al. 2015</a>). This suggests that perspectives and approaches from community ecology can complement current methods in virology.</p><p id="p-5">Three major challenges exist when integrating community ecology and virology. First, most viruses are rare (i.e. exist at low prevalence; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Tang et al. 2006</a>, Winker et al. 2008, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Wacharapluesadee et al. 2010</a>, <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Anthony et al. 2015</a>), so the probability of infection in a host individual is low. Consequently, large datasets of comprehensively sampled host populations are needed to address high levels of uncertainty associated with low viral prevalence. Few such datasets exist. Second, most wildlife viruses are unknown to science (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Anthony et al. 2013</a>, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Anthony et al. 2017</a>, Carlson et al. 2019), and the process of defining viral species is time-consuming (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">King et al 2012</a>). This means that waiting for official designations of viral species for use in community-level analyses does not address the urgency of current health challenges posed by wildlife viruses. Alternatives to official designation of viral species include delineating operational taxonomic units (OTUs) using monophyletic groups (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Anthony et al 2015</a>) or percent identity histograms (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Maes et al. 2009</a>, <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Anthony et al. 2017</a>). However, both such methods rely on subjectively chosen cutoff points to differentiate OTUs. Finally, many viral infections last only hours to days, making detection difficult (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Klenk et al. 2004</a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Lee et al. 2009</a>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Henaux and Samuel 2011</a>, <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">King et al. 2012</a>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Vetter et al. 2016</a>).</p><p id="p-6">Herpesviruses are large, enveloped, double-stranded DNA viruses that infect all vertebrate classes and mollusks (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">King et al. 2012</a>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Azab et al. 2018</a>). In contrast to many viruses, herpesviruses generally occur at high prevalence (Cone et al. 1993, Kidd et al. 1996, <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cortez et al. 2008</a>, Imbronito et al. 2008, Tenorio de Franca et al. 2012, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Phalen et al. 2017</a>, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Tazikeh et al. 2019</a>), and they establish latent, long-term (often for the entirety of the host’s life) infections (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">King et al. 2012</a>). Using herpesviruses as the subject of ecological analyses can therefore minimize two challenges that stymie advancements in viral ecology.</p><p id="p-7">Herpesviruses were first detected in bats in 1996 (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Tandler 1996</a>). Subsequent work shows that bats likely played a critical role in the evolution and diversification of herpesviruses (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Escalera-Zamudio et al. 2016</a>). Bats generally have a negative reputation as carriers of deadly viruses that cause human diseases such as rabies, Nipah, and SARS (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Wibbelt et al. 2010</a>), and consequently, most research on bat viruses has focused on these pathogens of concern for public health. Although a number of studies have also detected herpesviruses in bats (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Wibbelt et al. 2007</a>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Razafindratsimandresy et al. 2009</a>, <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Zhang et al. 2012</a>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Sasaki et al. 2014</a>, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Pozo et al. 2016</a>, <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Zheng et al. 2016</a>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Holz et al. 2018</a>, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Wada et al. 2018</a>), little is known about viral shedding, host specificity, or the diversity of herpesviruses in bats (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Zheng et al. 2016</a>).</p><p id="p-8">Herpesviruses are generally considered to be host-specific (<a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">King et al. 2012</a>). However, inter-host transmission has been documented when close contact occurs between individuals of different species, especially when host species are closely related (e.g. within primates: <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Weigler 1992</a>; within turtles: <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Greenblatt et al. 2005</a>; within ungulates: <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Russell et al. 2009</a>). Multiple bat species come into close proximity within caves where they roost, presenting an opportunity for cross-species viral dispersal. Indeed, cross-species transmission and host switching are common, and have likely been the norm throughout the evolutionary history of the herpesvirus lineage, especially in bat hosts (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Escalera-Zamudio et al. 2016</a>, <a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Zheng et al. 2016</a>, <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Azab et al. 2018</a>, <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Wada et al. 2018</a>).</p><p id="p-9">The first goal of this research was to intensively sample bat communities in two Puerto Rican caves, with the intent of saturating viral discovery of herpesviruses for use in ecological analyses. The second goal was to determine a quantitative technique to differentiate operational taxonomic units (OTUs) of herpesviruses as a fast, objective alternative to defining viral species. Finally, we aimed to describe patterns of host preference for herpesviruses. Given recent advancements in the understanding of host sharing throughout the history of the herpesvirus lineage, we hypothesized that particular OTUs would occur in multiple bat species but would have significantly higher prevalence in one of the species (i.e. viruses will have a core host species; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bush and Holmes 1986</a>); in all satellite (i.e. “secondary”) hosts combined, viruses will have lower prevalence.</p></div><div class="section" id="sec-2"><h2 class="">Materials and Methods</h2><div id="sec-3" class="subsection"><h3>Sample Collection</h3><p id="p-10">Field work was conducted at two adjacent caves on the Mata de Plátano Nature Reserve in Arecibo, Puerto Rico (18° 24.868’ N, 66° 43.531’ W). The 13 bat species on the island are well-documented and have been studied extensively (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gannon et al. 2005</a>). Mata de Plátano Nature Reserve (operated by InterAmerican University, Bayamon, Puerto Rico) is in the north-central, karstic region of the island, an area dominated by a multitude of caves that are suitable for roosting bats.</p><p id="p-11">Culebrones Cave is structurally complex and hot, with temperatures reaching 40° C and relative humidity at 100%. It is home to an estimated 300,000 individual bats representing six species: <em>Pteronotus quadridens, P. portoricensis, Mormoops blainvillii, Monophyllus redmani, Erophylla sezekorni</em>, and <em>Brachyphylla cavernarum</em> (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Rodríguez-Duran 2009</a>). Bats were sampled for 28 nights between June and August 2017. A harp trap was placed at sunset (approximately 18:00) immediately outside of the opening, and monitored continually.</p><p id="p-12">Larva Cave is cooler (ambient temperature), smaller, and less structurally complex compared to Culebrones Cave. It hosts a small total number of bats (30-200; ARS personal observation) representing two species, <em>Artibeus jamaicensis</em> and <em>Eptesicus fuscus</em>. Bats were sampled from Larva Cave on seven instances between June and August 2017, using two different techniques. After sunset, mist nets were placed along a trail outside of the cave entrance to catch exiting bats and were checked at least every ten minutes. Hand nets were also used to capture bats roosting inside the cave.</p><p id="p-13">Regardless of sampling method, each captured bat was placed into a porous cotton holding bag. Individuals were identified to species following <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Gannon et al. (2005)</a>, and a clean cotton-tipped swab was used to collect saliva from the bat’s mouth. Swabs were placed in individual cryovials containing viral transport medium and sent to Columbia University’s Center for Infection and Immunity, in a dry shipper, and stored at −80°C. All methods were approved by the University of Connecticut Institutional Animal Care and Use Committee (IACUC, protocol A15-032).</p></div><div id="sec-4" class="subsection"><h3>Viral Screening</h3><p id="p-14">Total nucleic acids were extracted from each swab using the EasyMag platform (bioMerieux, Inc.), and cDNA was synthesized using SuperScript III first-strand synthesis supermix (Invitrogen). Nested consensus polymerase chain reaction (cPCR) was performed twice on each sample, targeting a region of the highly conserved catalytic subunit of the DNA polymerase gene (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Van deVanter et al. 1996</a>). cPCR is a broadly reactive method that allows detection of novel viruses by using nonspecific primers targeted at viral families or genera (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Anthony et al. 2013</a>, <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">2015</a>). A constructed synthetic plasmid was used as a positive control to confirm effective implementation of the assay and to detect possible contamination. PCR gel products (1% agarose) of expected size were cloned into Strataclone PCR cloning vector, and twelve colonies were sequenced to confirm identity and detection of co-occurring viruses. Each clone was cross-referenced using the GenBank nucleotide database to confirm that sequences were indeed herpesviruses. Sequence data will be available at github.com/ARSjodin upon publication.</p></div><div id="sec-5" class="subsection"><h3>OTU Delineation</h3><p id="p-15">A nucleotide alignment of 3,131 herpesvirus clones was conducted using the ClustalW alignment in Geneious version 11.0.5, and a pairwise percent identity (PID) matrix was calculated based on percent similarity of nucleotides in each sequence pair. A PID histogram was then generated (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Maes et al. 2009</a>). To evaluate the robustness of cutoffs to errors in alignment, the R package Biostrings (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Pages et al. 2017</a>) was used to conduct pairwise alignments of clones. A second PID histogram was generated, and frequency clusters were compared between the two PIDs. The number of OTUs was estimated using the cutoff extremes for each level of classification, based on total alignment and pairwise alignment (74, 81, 90 and 93 PID cutoffs, see Results and <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/26/856518/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Percent Identity (PID) histograms of herpesviruses using the alignment of all clones (a) and pairwise alignment of clones (b). The x-axis represents the percent identity of clone sequences compared pairwise. The y-axis represents the number of clone sequences. Vertical lines represent cutoff points corresponding to two distinct peaks, with blue lines representing the higher-level cutoff and the red lines representing the lower-level cutoff. Solid lines represent the low end of the cutoff ranges, and the dashed lines represent the upper end of the cutoff ranges. The distinct peaks are consistent regardless of alignment method." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-728581716" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Percent Identity (PID) histograms of herpesviruses using the alignment of all clones (a) and pairwise alignment of clones (b). The x-axis represents the percent identity of clone sequences compared pairwise. The y-axis represents the number of clone sequences. Vertical lines represent cutoff points corresponding to two distinct peaks, with blue lines representing the higher-level cutoff and the red lines representing the lower-level cutoff. Solid lines represent the low end of the cutoff ranges, and the dashed lines represent the upper end of the cutoff ranges. The distinct peaks are consistent regardless of alignment method.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/26/856518/F1.medium.gif" width="440" height="239"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2019/11/26/856518/F1.medium.gif" width="440" height="239"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/26/856518/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/26/856518/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <p id="p-16" class="first-child">Percent Identity (PID) histograms of herpesviruses using the alignment of all clones (a) and pairwise alignment of clones (b). The x-axis represents the percent identity of clone sequences compared pairwise. The y-axis represents the number of clone sequences. Vertical lines represent cutoff points corresponding to two distinct peaks, with blue lines representing the higher-level cutoff and the red lines representing the lower-level cutoff. Solid lines represent the low end of the cutoff ranges, and the dashed lines represent the upper end of the cutoff ranges. The distinct peaks are consistent regardless of alignment method.</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">To statistically support which OTU cutoff to use in subsequent analyses, the R package apcluster (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bodenhofer et al. 2011</a>) was used to perform affinity propagation on the PID matrix of the total clone alignment. Affinity propagation is an algorithm that examines all data points in a similarity matrix to identify exemplars (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Frey and Dueck 2007</a>). The algorithm then clusters remaining data around an optimum number of exemplars by iteratively exchanging two types of “messages” between data points. In the context of viral sequences, the first type of message accumulates evidence for sequence <em>k</em> as an exemplar for sequence <em>i</em>, based on genetic distance between <em>k</em> and <em>i</em> (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Frey and Dueck 2007</a>). The second type of message reflects how many sequences besides <em>i</em> support <em>k</em> as the best choice of exemplar (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Frey and Dueck 2007</a>). When inputting the PID matrix of all clones, the number of clusters identified through affinity propagation can be interpreted as an optimal number of viral OTUs. Although this method has been used to define clusters of a single viral species (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fischer et al. 2017</a>), this is the first instance in which it has been used to delineate viral “species” for the purpose of community-level analyses. To conduct affinity propagation analyses, the apcluster() command was run using the R package apcluster (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bodenhofer et al. 2011</a>), with default values of input parameters (input data = NA, input preference = NA, q = 0.5, convits = 100, maxits = 1000, lam = 0.9). Finally, OTUs were refined manually based on PID and affinity (see Results).</p><p id="p-18">Finally, sequences representing the polymerase gene of formally described beta-and gammaherpesvirus species, as defined by the International Committee on the Taxonomy of Viruses (ICTV; Order Herpesvirales Figure 5, <a id="xref-ref-19-6" class="xref-bibr" href="#ref-19">King et al. 2012</a>) were retrieved from GenBank. Matrices comparing PID of defined viral species were calculated for comparison to empirical values. For ease of exposition, “species” will be used moving forward to reference host taxa or formally described viral taxa, whereas “OTU” will be used to reference empirical viral taxa.</p></div><div id="sec-6" class="subsection"><h3>Viral Discovery Curves</h3><p id="p-19">To determine how well the collected samples represent the metacommunity of herpesviruses infecting bats at Mata de Plátano Nature Reserve, viral discovery curves were created using the iNEXT package in R (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hsieh et al. 2016</a>). By defining the metacommunity as all of the herpesviruses infecting all of the host species at Mata de Plátano Nature Reserve, an assumption is made that herpesviruses are shared between caves and among host species (i.e. no host specificity). Because this may not be a valid assumption, separate curves were generated for each bat species, as well as for each cave. Additionally, Chao2 estimates (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chao 1987</a>) were calculated for the viral metacommunities of all bats at Mata de Plátano Nature Reserve, as well as for each cave and for each bat species separately.</p></div><div id="sec-7" class="subsection"><h3>Host Preference</h3><p id="p-20">For each OTU that infected more than one host species, the host specificity index S<sub>TD</sub>* was calculated (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Poulin and Mouillot 2005</a>) as
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/26/856518/embed/graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/26/856518/embed/graphic-2.gif"/></noscript></span>

</span>
where <em>i</em> and <em>j</em> represent index values of host species, <em>p</em> represents OTU prevalence within host populations, and ω<sub>ij</sub> is the phylogenetic distinctiveness of hosts. To measure phylogenetic distinctness, hosts of the same genus are assigned a value of one, hosts of the same family but different genera are assigned a value of two, and hosts of different families have a value of three (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Poulin and Mouillot 2005</a>). S<sub>TD</sub>* thus weighs host preference by the relatedness of hosts (incorporating evolution) and the prevalence of the pathogen in each of its hosts (incorporating ecology). The extent of viral sharing among hosts species was further examined by determining whether particular OTUs had a core host species. A core host species was defined as a single host species for which the OTU had significantly higher prevalence than in all other host species combined. To do this, the prevalence of each OTU in its core host and its combined prevalence in all other hosts were compared using Fisher’s exact test (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Wassertheil-Smoller and Smoller 2015</a>). For each OTU that infected more than one host individual, viral preference for host sex was examined by comparing OTU prevalence between male and females of the core host species, using Fisher’s exact test. All analyses were done using R version 3.4.3, and code used for all methods can be found at github.com/ARSjodin.</p></div></div><div class="section" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Viral Screening</h3><p id="p-21">Oral swabs were collected from 1,086 bats representing eight species (<em>A. jamaicensis</em> and <em>E. fuscus</em> in Larva Cave, and <em>P. quadridens, P. portoricensis, Mor. blainvillii, Mon. redmani, E. sezekorni</em>, and <em>B. cavernarum</em> in Culebrones Cave; <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Three hundred and thirty host individuals tested positive for herpesvirus, for a community-level prevalence of 30.4% (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Prevalence was similar in Culebrones (30.0%) and in Larva (35.3%) Caves. Herpesvirus prevalence at the host population-level ranged from 12.9% in <em>P. portoricensis</em> to 42.1% in <em>A. jamaicensis</em>. At least one individual from each host species tested positive, with the exception of <em>E. fuscus</em> (0/11). In females, prevalence ranged from 12.5% in <em>P. portoricensis</em> to 41.9% in <em>Mor. blainvillii</em>. In males, prevalence ranged from 14.3% in <em>P. portoricensis</em> to 42.4% in <em>A. jamaicensis</em>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1013747/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1013747/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1013747" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1:</span> <p id="p-22" class="first-child">Summary of capture numbers and virus screening results. Numbers of host individuals sampled (N), number testing positive for herpesvirus (N positive), and prevalence of herpesviruses are calculated for each host species and for each cave, as well as separately for males and females of each host species. Number of herpesvirus OTUs detected empirically (S) and estimate of herpesvirus OTU richness (Chao2), are compared to estimate the percentage of herpesvirus OTUs that was detected in this study (% completeness). No male <em>Eptesicus fuscus</em> were captured, so NA values are produced.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>OTU Delineation</h3><p id="p-23">Both alignment techniques resulted in two similar, clearly differentiated potential cutoff ranges, as demonstrated by consistent peaks in the histograms (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). The lower level of taxonomic grouping of OTUs was defined as those with a cutoff between 90 – 92%. This means that two sequences with less than 90% genetic similarity would be considered to be different OTUs. The higher level of taxonomic grouping was defined as those with a cutoff between 74 and 80%. As such, two sequences with less than 74% genetic similarity would be considered to be different OTUs. The number of OTUs resulting from the cutoff extremes for both levels of classification, when total alignment and pairwise alignment were considered (74, 81, 90 and 93 PID cutoffs; <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>), were 29, 31, 42, and 47 OTUs, respectively.</p><p id="p-24">Affinity propagation resulted in 45 OTUs. Although OTU richness did not equal any values generated using the PID histograms, 45 OTUs is within the range of OTUs generated using the lower-level taxonomic cutoff (i.e. 42 – 47 OTUs at 90 – 93% PID cutoff), suggesting this cutoff is more strongly supported by affinity propagation than is the higher-level cutoff. Additionally, no two described herpesvirus species were greater than 90% similar. However, Ateline herpesvirus 3 and Saimirine herpesvirus 2, two gammaherpesvirus species, and Cercopithecine herpesvirus 5 and Macacine herpesvirus 3, two betaherpesvirus species, differed by 83.6% and 78.5%, respectively. Additionally, two strains of Human herpesvirus 6, a betaherpesvirus, and Human herpesvirus 4, a gammaherpesvirus, differed by 96.2 and 99.1%, respectively. These PIDs of established viral species further support the use of the lower-level taxonomic cutoff.</p><p id="p-25">To refine the exact number of OTUs, two neighbor-joining trees were generated from the exemplar sequences, as determined using affinity propagation, one for the gammaherpesviruses and one for the betaherpesviruses. (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). When exemplar sequences were greater than 90% similar (the most conservative cutoff within the lower taxonomic range), clusters were merged to represent a single OTU. Additionally, each OTU represented by a single occurrence (i.e. a single clone sequence) was examined individually. If the singleton sequence was greater than 74% similar to the exemplar of another OTU, the least conservative cutoff point demonstrated by PID histograms, it was clustered with that OTU. Singleton sequences less than or equal to 74% similar to each of the other exemplars were considered to be unique OTUs. After refinement, 41 OTUs remained. Preliminary data collected from Mata de Plátano Nature Reserve in 2016 corroborated the existence of OTUs 12, 17, and 25. These OTUs were found in four host individuals in 2016, representing host species <em>Mor. blainvillii</em> (OTUs 12 and 25; two individuals) and <em>A. jamaicensis</em> (OTU 17; two individuals). All OTUs found in 2016 were detected in 2017.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/26/856518/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Neighbor-joining trees of betaherpesvirine (a) and gammaherpesvirine (b) exemplars, as defined using affinity propagation. OTUs are named to reflect manual refinement of OTUs after affinity propagation. For example, affinity propagation separated OTU 28, a gammaherpesvirus, into four separate clusters. However, the similarity of the exemplars suggested that these clusters should all be grouped into a single OTU. These trees provide biological support for the manual refinement of OTUs. OTUs 39-41, all betaherpesviruses, were not included in the tree. These OTUs were represented by three highly divergent sequences (&lt; 50% genetic similarity), and their inclusion in the tree impaired visualization of all other relationships." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-728581716" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Neighbor-joining trees of betaherpesvirine (a) and gammaherpesvirine (b) exemplars, as defined using affinity propagation. OTUs are named to reflect manual refinement of OTUs after affinity propagation. For example, affinity propagation separated OTU 28, a gammaherpesvirus, into four separate clusters. However, the similarity of the exemplars suggested that these clusters should all be grouped into a single OTU. These trees provide biological support for the manual refinement of OTUs. OTUs 39-41, all betaherpesviruses, were not included in the tree. These OTUs were represented by three highly divergent sequences (&lt; 50% genetic similarity), and their inclusion in the tree impaired visualization of all other relationships.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/26/856518/F2.medium.gif" width="440" height="231"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2019/11/26/856518/F2.medium.gif" width="440" height="231"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/26/856518/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/26/856518/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <p id="p-26" class="first-child">Neighbor-joining trees of betaherpesvirine (a) and gammaherpesvirine (b) exemplars, as defined using affinity propagation. OTUs are named to reflect manual refinement of OTUs after affinity propagation. For example, affinity propagation separated OTU 28, a gammaherpesvirus, into four separate clusters. However, the similarity of the exemplars suggested that these clusters should all be grouped into a single OTU. These trees provide biological support for the manual refinement of OTUs. OTUs 39-41, all betaherpesviruses, were not included in the tree. These OTUs were represented by three highly divergent sequences (&lt; 50% genetic similarity), and their inclusion in the tree impaired visualization of all other relationships.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">No OTU existed at a prevalence greater than 6.2% in the entire host community (i.e. across all host species). When viral metacommunities were examined at the level of host population (i.e. in a single species), however, OTU prevalence reached a peak of 26.3% (OTU 17 in <em>A. jamaicensis</em>; <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). At the population-level, four OTUs reached a prevalence greater than 20%, and 29 OTUs existed at prevalence values between 1% and 10%. Five OTUs existed at a prevalence less than 1% in all of their host populations. OTU richness per host population ranged from three to 15. In comparison, humans are the only host species for which comprehensive data are available on total herpesvirus richness, and only eight known herpesvirus species infect humans, despite their cosmopolitan distribution.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1013745/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/1013745/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2:</span> <p id="p-28" class="first-child">Percent of hosts infected (prevalence) of each herpesvirus OTU at Mata de Plátano Nature Reserve (all bats), in each cave, and in each host population. Total number of sampled host individuals (N) is listed for each calculation. Continued on the next page.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Viral Discovery Curves</h3><p id="p-29">The Chao2 estimate of herpesvirus richness in the bats of both caves was 51.1 OTUs (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). As such, the detected OTUs (N = 41) represent about 80% of those in the entire herpesvirus metacommunity (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Viral discovery curves were more saturated in Culebrones (85%) than in Larva (61%) Cave, and host species-level saturation ranged from 55% (<em>P. quadridens</em>) to 93% (<em>B. cavernarum</em>) (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>; <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). A viral discovery curve was not generated for the herpesvirus metacommunity infecting <em>P. portoricensis</em>, as only three OTUs were detected.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/26/856518/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Viral discovery curves comparing number of detected herpesvirus (OTU richness) with number of sampled host bats for all bats at Mata de Pl&#xE1;tano Nature Reserve (top panel) and for each host species (bottom six panels). The black lines represent the empirical pattern of OTU accumulation. The dashed red lines represent extrapolation of undiscovered herpesviruses in the metacommunities, with red shading demonstrating error associated with the estimates. Empirical viral richness per metacommunity (S) is depicted by red triangles. Chao2 numbers estimate true viral richness in each community and is used to calculate the percent of total estimated OTUs that were detected in samples (percent completeness). No discovery curve was generated for viruses infecting P. portoricensis, as only three OTUs were detected in this metacommunity." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-728581716" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Viral discovery curves comparing number of detected herpesvirus (OTU richness) with number of sampled host bats for all bats at Mata de Pl&#xE1;tano Nature Reserve (top panel) and for each host species (bottom six panels). The black lines represent the empirical pattern of OTU accumulation. The dashed red lines represent extrapolation of undiscovered herpesviruses in the metacommunities, with red shading demonstrating error associated with the estimates. Empirical viral richness per metacommunity (S) is depicted by red triangles. Chao2 numbers estimate true viral richness in each community and is used to calculate the percent of total estimated OTUs that were detected in samples (percent completeness). No discovery curve was generated for viruses infecting &lt;em&gt;P. portoricensis&lt;/em&gt;, as only three OTUs were detected in this metacommunity.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/26/856518/F3.medium.gif" width="271" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2019/11/26/856518/F3.medium.gif" width="271" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/26/856518/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/26/856518/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <p id="p-30" class="first-child">Viral discovery curves comparing number of detected herpesvirus (OTU richness) with number of sampled host bats for all bats at Mata de Plátano Nature Reserve (top panel) and for each host species (bottom six panels). The black lines represent the empirical pattern of OTU accumulation. The dashed red lines represent extrapolation of undiscovered herpesviruses in the metacommunities, with red shading demonstrating error associated with the estimates. Empirical viral richness per metacommunity (S) is depicted by red triangles. Chao2 numbers estimate true viral richness in each community and is used to calculate the percent of total estimated OTUs that were detected in samples (percent completeness). No discovery curve was generated for viruses infecting <em>P. portoricensis</em>, as only three OTUs were detected in this metacommunity.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Host Preference</h3><p id="p-31">Thirteen OTUs were detected in one or more host species (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). S<sub>TD</sub>* for each of these OTUs ranged from one (infecting two hosts from the same genus) to three (infecting three hosts of two families). Nine OTUs had significantly higher prevalence in a core host species than in all other host species combined (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). Only two OTUs showed significantly higher prevalence in one host sex than in the other (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). OTU 1 infected a significantly higher percentage of male than female <em>P. quadridens</em> (p = 0.021), and OTU 21 infected a significantly higher percentage of female than male <em>Mon. redmani</em> (p = 0.023).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1013739/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1013739/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1013739" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3:</span> <p id="p-32" class="first-child">Host specificity index (S<sub>TD</sub>*) for each of the 13 herpesvirus OTUs that infect more than one host species. OTU prevalence is calculated for each host species, with number of sampled individuals in parentheses (N). Prevalence in the primary host species is bolded for each OTU.</p><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1013733/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1013733/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1013733" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4:</span> <p id="p-33" class="first-child">For each of the 13 herpesvirus OTUs that infect more than one host species, comparison of OTU prevalence between core host species and all other host species for which it is found, using Fisher’s exact test. Significance at an alpha of 0.01 are marked with a single asterisk, while those significant at an alpha of 0.001 are marked with two asterisks.</p><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1013744/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1013744/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1013744" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5:</span> <p id="p-34" class="first-child">For the 32 herpesvirus OTUs infecting more than one host individual, prevalence is compared between males and females using Fisher’s exact test. The number of host individuals that tested positive is in the numerator, while the total number of captured host individuals is the denominator. Significance at the alpha level of 0.05 is shown with an asterisk, and rows with significant differences are bolded. For all OTUs that are not represented in this table, only a single individual tested positive.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-13"><h2 class="">Discussion</h2><p id="p-35">Our extensive quantification of viral infection in bats represents the first comprehensive study of the virome of a wildlife host community and suggests that Puerto Rican bats offer a previously untapped resource for advancing the field of viral ecology. Even in the absence of full genome sequences, viral OTUs can be robustly differentiated and used to understand the natural history and host specificity of viruses.</p><p id="p-36">The application of tools and perspectives from community ecology to better understand viral infection dynamics necessitates an objective, quantitatively-supported delineation of viral “species”. Officially designating novel viral species requires substantial data, oftentimes full genome sequences, and approval by the International Committee for the Taxonomy of Viruses (<a id="xref-ref-19-7" class="xref-bibr" href="#ref-19">King et al. 2012</a>). Consequently, OTUs are commonly used as proxies for species. Although multiple methods exist for delineating OTUs (e.g. histograms using percent genetic identity, <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Maes et al. 2009</a>; or monophyletic groups, <a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Anthony et al. 2015</a>), subjectivity is generally required to choose the best cutoff point in terms of genetic similarity (i.e. the histogram method) or evolutionary time (i.e. the monophyletic groups method). Affinity propagation is a machine-learning algorithm developed to cluster a broad range of data (e.g. images of faces, genes in a microarray, heavily travelled cities) and identify representative examples of each cluster (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Frey and Dueck 2007</a>). The method has been used to cluster strains of rabies viruses (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Fischer et al. 2017</a>), but has never been used to identify viral OTUs for analyses in the context of community ecology. Here, we show that affinity propagation can be used as a quantitative complement, in congruence with currently defined herpesvirus species, to objectively support current methods of viral OTU delineation.</p><p id="p-37">A limit to working with viruses in an ecological context is the short viral lifespan or activation cycle (e.g. lytic stage), as compared to that in free-living or macroparasitic organisms. Short lifespans affect temporal turnover of detected OTUs and could influence interpretation of results. Although it is common for some viral infections to last only hours or days, the lytic cycle of herpesvirus infections is typically longer than that of most other viruses (<a id="xref-ref-19-8" class="xref-bibr" href="#ref-19">King et al. 2012</a>). This represents an advantage in the use of herpesviruses for preliminary exploration of viral ecology, as the duration during which OTUs can be detected will be greater than that of other taxa. Indeed, all OTUs detected during preliminary sampling in 2016 (i.e. OTUs 12, 17, and 25) were detected again in 2017, in the same core host species (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-38">Another advantage to studying herpesviruses from an ecological context is that this viral family often exists at high prevalence levels (Cone et al. 1993, Kidd et al. 1996, <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Cortez et al. 2008</a>, Imbronito et al. 2008, Tenorio de Franca et al. 2012, <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Phalen et al. 2017</a>, <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Tazikeh et al. 2019</a>). While some surveys (e.g. Cone et al. 1993, Kidd et al. 1996, <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Phalen et al. 2017</a>) show prevalence values greater than 70% for individual viral species, the population-level prevalence levels of OTUs in Puerto Rican bats are consistent with other surveyed populations of healthy humans (Tenorio de Franca et al. 2012) and non-human animals (Corez et al. 2008, <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Tazikeh et al. 2019</a>).</p><p id="p-39">Generally, herpesviruses are highly host specific, but when host-virus relationships are examined at an evolutionary timescale, host switching has been the norm (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Escalera-Zamudio et al. 2016</a>, <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Azab et al. 2018</a>). As such, we expected herpesviruses that infected multiple host species to have core hosts. Overall, we found evidence of host sharing among bat species at Mata de Plátano Nature Reserve, regardless of cave. The existence of host sharing is further supported by viral prevalence and viral discovery curves. A higher prevalence at the population-level (i.e. a single species) than the community-level, as seen here (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>), is consistent with host-specificity. Also, at the largest scale (i.e. the entire nature reserve), sampling completeness reached 80%. This is in contrast to accumulation curves for viral metacommunities infecting particular host populations, which were as low as 55% (<em>P. quadridens</em>). This suggests that undiscovered OTUs at the scale of host population have already been discovered at the scale of nature reserve (i.e. have already been discovered in another host species). This supports the hypothesis that certain OTUs that are common in a core host are rare in all other host species. Indeed, nine of the 13 multi-host OTUs had significantly higher prevalence in a single host than in all other hosts in which it occurred combined (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). Those that did not have significantly different prevalence were rare OTUs in all infected hosts (OTU 7, OTU 15, OTU 26), or were compared between host species with drastically different sample sizes (OTU 1, found in <em>P. quadridens</em> and <em>P. portoricensis</em>; with 288 and 31 sampled individuals, respectively).</p><p id="p-40">In conclusion, ecological tools and perspectives can be used to understand viral infection and transmission, generating a multitude of hypotheses in the emerging field of viral ecology. Future studies with goals of monitoring public health should increase the number of sampled individuals for each host species to enhance the likelihood of uncovering patterns of host preference and reservoir status. Studies aimed at viral ecology will likely require focus on more prevalent viral taxa, such as herpesviruses, instead of rare viral taxa of more zoonotic concern, such as coronaviruses or filoviruses. Further ecological research should prioritize sampling host individuals from multiple field locations, addressing viral infection and transmission at multiple spatial scales, while decoupling the spatial and ecological mechanisms that shape viral community assembly.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-41">We gratefully acknowledge Armando Rodriguez-Duran for invaluable logistical help in the field, as well as the intrepid field team, including Sarah Stankavich, Jose Rivera, Melanie Hodge, Brian Springall, Elspeth Pierce, Emily Stanford, Juan Contreras, Ismael Ribot, and Frank Sjodin. Isamara Navarrete-Macias and Eliza Liang trained ARS on laboratory methods, and Heather Wells provided invaluable lab and bioinformatics help. Cale Basaraba provided coding assistance. We also thank Janine Caira, Sarah Knutie, and Mark Urban for constructive discussions that advanced the intellectual development of the work. ARS was funded by the National Science Foundation Graduate Research Fellowship Program and a Jorgensen Fellowship from the University of Connecticut. MRW was funded by a grant from NSF (DEB-1546686 and DEB-1831952). Fieldwork was supported by Sigma Xi; the Royal Society of Tropical Medicine and Hygiene; and the University of Connecticut’s Department of Ecology and Evolutionary Biology, El Instituto via the Tinker Foundation, and the Office of the Vice President for Research. This project also benefited from support from the USAID PREDICT project.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-dataset fn-group-external-links" id="fn-1"><p id="p-3">
<a href="https://github.com/arsjodin">https://github.com/arsjodin</a>
</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.1" data-doi="10.1128/mBio.00598-13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anthony</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Epstein</span>  <span class="cit-name-given-names">JH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">KA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Navarrete-Macias</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zambrana-Torrelio</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Solovyov</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ojeda-Flores</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arrigo</span>  <span class="cit-name-given-names">NC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Islam</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Khan</span>  <span class="cit-name-given-names">SA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hosseini</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bogich</span>  <span class="cit-name-given-names">TL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Olival</span>  <span class="cit-name-given-names">KJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sanchez-Leon</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Karesh</span>  <span class="cit-name-given-names">WB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luby</span>  <span class="cit-name-given-names">SP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morse</span>  <span class="cit-name-given-names">SS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mazet</span>  <span class="cit-name-given-names">JAK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Daszak</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lipkin</span>  <span class="cit-name-given-names">WI</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">A strategy to estimate unknown viral diversity in mammals</span>. <abbr class="cit-jnl-abbrev">mBio</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e00598</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DmBio%26rft.volume%253D4%26rft.spage%253De00598%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.00598-13%26rft_id%253Dinfo%253Apmid%252F24003179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/mBio.00598-13&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24003179&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F26%2F856518.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anthony</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Islam</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Navarrete-Macias</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hitchens</span>  <span class="cit-name-given-names">PL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Che</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Soloyvov</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hicks</span>  <span class="cit-name-given-names">AL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ojeda-Flores</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zambrana-Torrelio</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ulrich</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rostal</span>  <span class="cit-name-given-names">MK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Petrosov</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haider</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morse</span>  <span class="cit-name-given-names">SS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rahman</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Epstein</span>  <span class="cit-name-given-names">JH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mazet</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Daszak</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lipkin</span>  <span class="cit-name-given-names">WI</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Non-random patterns in viral diversity</span>. <abbr class="cit-jnl-abbrev">Nature Communications</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">8147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D6%26rft.spage%253D8147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.3" data-doi="10.1093/ve/vex012"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anthony</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">CK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Greig</span>  <span class="cit-name-given-names">DJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kramer</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Che</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wells</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hicks</span>  <span class="cit-name-given-names">AL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joly</span>  <span class="cit-name-given-names">DO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">ND</span></span>, <span class="cit-auth"><span class="cit-name-surname">Daszak</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Karesh</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lipkin</span>  <span class="cit-name-given-names">WI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morse</span>  <span class="cit-name-given-names">SS</span></span>, <span class="cit-auth cit-collab">PREDICT Consortium</span>, <span class="cit-auth"><span class="cit-name-surname">Mazet</span>  <span class="cit-name-given-names">JAK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">T</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Global patterns in coronavirus diversity</span>. <abbr class="cit-jnl-abbrev">Virus Evolution</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">vex012</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirus%2BEvolution%26rft.volume%253D3%26rft.spage%253Dvex012%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fve%252Fvex012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/ve/vex012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Azab</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dayaram</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Greenwood</span>  <span class="cit-name-given-names">AD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Osterrieder</span>  <span class="cit-name-given-names">N</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">How host specific are herpesviruses? Lessons from herpesviruses infecting wild and endangered mammals</span>. <abbr class="cit-jnl-abbrev">Annual Review of Virology</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">68</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BVirology%26rft.volume%253D5%26rft.spage%253D53%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bodenhofer</span>  <span class="cit-name-given-names">U</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kothmeier</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hochreiter</span>  <span class="cit-name-given-names">S</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">APCluster: a R package for affinity propagation clustering</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">2463</span>–<span class="cit-lpage">2464</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D17%26rft.spage%253D2463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bush</span>  <span class="cit-name-given-names">AO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">JC</span></span> (<span class="cit-pub-date">1986</span>) <span class="cit-article-title">Intestinal helminths of lesser scaup ducks: an interactive community</span>. <abbr class="cit-jnl-abbrev">Canadian Journal of Zoology</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">142</span>–<span class="cit-lpage">152</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCanadian%2BJournal%2Bof%2BZoology%26rft.volume%253D64%26rft.spage%253D142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.7" data-doi="10.2307/2531532"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">A</span></span> (<span class="cit-pub-date">1987</span>) <span class="cit-article-title">Estimating the population size for capture-recapture data with unequal catchability</span>. <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">783</span>–<span class="cit-lpage">791</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DChao%26rft.auinit1%253DA.%26rft.volume%253D43%26rft.issue%253D4%26rft.spage%253D783%26rft.epage%253D791%26rft.atitle%253DEstimating%2Bthe%2Bpopulation%2Bsize%2Bfor%2Bcapture-recapture%2Bdata%2Bwith%2Bunequal%2Bcatchability.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2531532%26rft_id%253Dinfo%253Apmid%252F3427163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2531532&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3427163&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F26%2F856518.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987L357600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.8" data-doi="10.1136/vr.162.9.282"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cortez</span>  <span class="cit-name-given-names">PP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carvalheira</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pauperio</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">G</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">Prevalence of ovine herpesvirus type 2 in north-west Portugal</span>. <abbr class="cit-jnl-abbrev">Veterinary Record</abbr> <span class="cit-vol">162</span>: <span class="cit-fpage">282</span>–<span class="cit-lpage">284</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVeterinary%2BRecord%26rft.stitle%253DThe%2BVeterinary%2BRecord%26rft.aulast%253DCortez%26rft.auinit1%253DP.%2BP.%26rft.volume%253D162%26rft.issue%253D9%26rft.spage%253D282%26rft.epage%253D284%26rft.atitle%253DPrevalence%2Bof%2Bovine%2Bherpesvirus%2Btype%2B2%2Bin%2Bnorth-west%2BPortugal%26rft_id%253Dinfo%253Adoi%252F10.1136%252Fvr.162.9.282%26rft_id%253Dinfo%253Apmid%252F18310564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NjoidmV0cmVjIjtzOjU6InJlc2lkIjtzOjk6IjE2Mi85LzI4MiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzExLzI2Lzg1NjUxOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-856518v1.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">De Benedictis</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schultz-Cherry</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burnham</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cattoli</span>  <span class="cit-name-given-names">G</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Astrovirus infections in humans and animals – molecular biology, genetic diversity, and interspecies transmissions</span>. <abbr class="cit-jnl-abbrev">Infection, Genetics and Evolution</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1529</span>–<span class="cit-lpage">1544</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%252C%2BGenetics%2Band%2BEvolution%26rft.volume%253D11%26rft.spage%253D1529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Escalera-Zamudio</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rojas-Anaya</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kolokotronis</span>  <span class="cit-name-given-names">SO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taboada</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Loza-Rubio</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mendez-Ojeda</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arias</span>  <span class="cit-name-given-names">CF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Osterrieder</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Greenwood</span>  <span class="cit-name-given-names">AD</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Bats, primates, and the evolutionary origins and diversification of mammalian Gammaherpesviruses</span>. <abbr class="cit-jnl-abbrev">mBio</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e01425</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DmBio%26rft.volume%253D7%26rft.spage%253De01425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Freuling</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pfaff</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bodenhofer</span>  <span class="cit-name-given-names">U</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoper</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marston</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fooks</span>  <span class="cit-name-given-names">AR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mettenleiter</span>  <span class="cit-name-given-names">TC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Conraths</span>  <span class="cit-name-given-names">FJ</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Homeier-Bachmann</span>  <span class="cit-name-given-names">T</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Defining objective clusters for rabies virus sequences using affinity propagation clustering</span>. <abbr class="cit-jnl-abbrev">PLoS Neglected Tropical Diseases</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e0006182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BNeglected%2BTropical%2BDiseases%26rft.volume%253D12%26rft.spage%253D0006182e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.12" data-doi="10.1126/science.1136800"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Frey</span>  <span class="cit-name-given-names">BJ</span></span> <span class="cit-auth"><span class="cit-name-surname">and Dueck</span>  <span class="cit-name-given-names">D</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">Clustering by passing messages between data points</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">315</span>: <span class="cit-fpage">972</span>–<span class="cit-lpage">976</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1136800%26rft_id%253Dinfo%253Apmid%252F17218491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTUvNTgxNC85NzIiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMS8yNi84NTY1MTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-book" id="cit-856518v1.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gannon</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kurta</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rodríguez-Duran</span>  <span class="cit-name-given-names">A</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Willig</span>  <span class="cit-name-given-names">MR</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-chapter-title">Bats of Puerto Rico: An Island Focus and a Caribbean Perspective</span>. <span class="cit-publ-loc">Lubbock, TX</span>: <span class="cit-publ-name">Texas Tech University Press</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.14" data-doi="10.1128/JVI.79.2.1125-1132.2005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Greenblatt</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Quackenbush</span>  <span class="cit-name-given-names">SL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Casey</span>  <span class="cit-name-given-names">RN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rovnak</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Balazs</span>  <span class="cit-name-given-names">GH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Work</span>  <span class="cit-name-given-names">TM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Casey</span>  <span class="cit-name-given-names">JW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sutton</span>  <span class="cit-name-given-names">CA</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">Genomic variation of the fibropapilloma-associated marine turtle herpesvirus across seven geographic areas and three host species</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">1125</span>–<span class="cit-lpage">1132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DGreenblatt%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D79%26rft.issue%253D2%26rft.spage%253D1125%26rft.epage%253D1132%26rft.atitle%253DGenomic%2BVariation%2Bof%2Bthe%2BFibropapilloma-Associated%2BMarine%2BTurtle%2BHerpesvirus%2Bacross%2BSeven%2BGeographic%2BAreas%2Band%2BThree%2BHost%2BSpecies%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.79.2.1125-1132.2005%26rft_id%253Dinfo%253Apmid%252F15613340%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijc5LzIvMTEyNSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzExLzI2Lzg1NjUxOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Henaux</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Samuel</span>  <span class="cit-name-given-names">MD</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Avian influenza shedding patterns in waterfowl: implications for surveillance, environmental transmission, and diseases spread</span>. <abbr class="cit-jnl-abbrev">Journal of Wildlife Diseases</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">566</span>–<span class="cit-lpage">578</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BWildlife%2BDiseases%26rft.stitle%253DJ%2BWildl%2BDis%26rft.aulast%253DHenaux%26rft.auinit1%253DV.%26rft.volume%253D47%26rft.issue%253D3%26rft.spage%253D566%26rft.epage%253D578%26rft.atitle%253DAVIAN%2BINFLUENZA%2BSHEDDING%2BPATTERNS%2BIN%2BWATERFOWL%253A%2BIMPLICATIONS%2BFOR%2BSURVEILLANCE%252C%2BENVIRONMENTAL%2BTRANSMISSION%252C%2BAND%2BDISEASE%2BSPREAD%26rft_id%253Dinfo%253Apmid%252F21719821%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21719821&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F26%2F856518.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292798500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Holz</span>  <span class="cit-name-given-names">PH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lumsden</span>  <span class="cit-name-given-names">LF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Druce</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Legione</span>  <span class="cit-name-given-names">AR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vaz</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Devlin</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hufschmid</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">Virus survey in populations of two subspecies of bent-winged bats (<em>Miniopterus orianae bassanii</em> and <em>oceanensis</em>) in south-eastern Australia reveals a high prevalence of diverse herpesviruses</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">e0197625</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D13%26rft.spage%253De0197625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.17" data-doi="10.1111/2041-210X.12613"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hsieh</span>  <span class="cit-name-given-names">TC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">KH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">A</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">iNEXT: an R package for rarefaction and extrapolation of species diversity (Hill numbers)</span>. <abbr class="cit-jnl-abbrev">Methods in Ecology and Evolution</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1451</span>–<span class="cit-lpage">1456</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D7%26rft.spage%253D1451%26rft_id%253Dinfo%253Adoi%252F10.1111%252F2041-210X.12613%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/2041-210X.12613&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.18" data-doi="10.1890/07-0995.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jolles</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ezenwa</span>  <span class="cit-name-given-names">VO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Etienne</span>  <span class="cit-name-given-names">RS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">WC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Olff</span>  <span class="cit-name-given-names">H</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">Interactions between macroparasites and microparasites drive infection patterns in free-ranging African buffalo</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">2239</span>–<span class="cit-lpage">2250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.stitle%253DEcology%26rft.aulast%253DJolles%26rft.auinit1%253DA.%2BE.%26rft.volume%253D89%26rft.issue%253D8%26rft.spage%253D2239%26rft.epage%253D2250%26rft.atitle%253DInteractions%2Bbetween%2Bmacroparasites%2Band%2Bmicroparasites%2Bdrive%2Binfection%2Bpatterns%2Bin%2Bfree-ranging%2BAfrican%2Bbuffalo.%26rft_id%253Dinfo%253Adoi%252F10.1890%252F07-0995.1%26rft_id%253Dinfo%253Apmid%252F18724734%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/07-0995.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18724734&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F26%2F856518.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258236400018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-book" id="cit-856518v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">AMQ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carstens</span>  <span class="cit-name-given-names">EB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lefkowitz</span>  <span class="cit-name-given-names">EJ</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-chapter-title">Virus Taxonomy: Classification and Nomenclature of Viruses. Ninth report of the International Committee on Taxonomy of Viruses</span>, <span class="cit-publ-loc">San Diego, CA</span>: <span class="cit-publ-name">Academic Press</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.20" data-doi="10.3201/eid1012.040264"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Klenk</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snow</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bowen</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Foster</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gordy</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beckett</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Komar</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gubler</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bunning</span>  <span class="cit-name-given-names">M</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">Alligators as West Nile virus amplifiers</span>. <abbr class="cit-jnl-abbrev">Emerging Infectious Diseases</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">2150</span>–<span class="cit-lpage">2155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEmerging%2Binfectious%2Bdiseases%26rft.stitle%253DEmerg%2BInfect%2BDis%26rft.aulast%253DKlenk%26rft.auinit1%253DK.%26rft.volume%253D10%26rft.issue%253D12%26rft.spage%253D2150%26rft.epage%253D2155%26rft.atitle%253DAlligators%2Bas%2BWest%2BNile%2Bvirus%2Bamplifiers.%26rft_id%253Dinfo%253Adoi%252F10.3201%252Feid1012.040264%26rft_id%253Dinfo%253Apmid%252F15663852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3201/eid1012.040264&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15663852&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F26%2F856518.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225528000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.21" data-doi="10.1086/600383"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">PKS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hui</span>  <span class="cit-name-given-names">DSC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rainer</span>  <span class="cit-name-given-names">TH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">KW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lui</span>  <span class="cit-name-given-names">GCY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">BCK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">RYK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">WY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">IMT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">RWM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cockram</span>  <span class="cit-name-given-names">CS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sung</span>  <span class="cit-name-given-names">JJY</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Viral loads and duration of viral shedding in adult patients hospitalized with influenza</span>. <abbr class="cit-jnl-abbrev">The Journal of Infectious Diseases</abbr> <span class="cit-vol">200</span>: <span class="cit-fpage">492</span>–<span class="cit-lpage">500</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BInfectious%2BDiseases%26rft.stitle%253DThe%2BJournal%2Bof%2BInfectious%2BDisease%26rft.aulast%253DLee%26rft.auinit1%253DN.%26rft.volume%253D200%26rft.issue%253D4%26rft.spage%253D492%26rft.epage%253D500%26rft.atitle%253DViral%2Bloads%2Band%2Bduration%2Bof%2Bviral%2Bshedding%2Bin%2Badult%2Bpatients%2Bhospitalized%2Bwith%2Binfluenza.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F600383%26rft_id%253Dinfo%253Apmid%252F19591575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/600383&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19591575&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F26%2F856518.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268009700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maes</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klempa</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clement</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matthijnssens</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gajdusek</span>  <span class="cit-name-given-names">DC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kruger</span>  <span class="cit-name-given-names">DH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van Ranst</span>  <span class="cit-name-given-names">M</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">A proposal for new criteria for the classification of hantaviruses, based on S and M segment protein sequences</span>. <abbr class="cit-jnl-abbrev">Infection, Genetics and Evolution</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">813</span>–<span class="cit-lpage">820</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%252C%2BGenetics%2Band%2BEvolution%26rft.volume%253D9%26rft.spage%253D813%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pages</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aboyoun</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gentleman</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">DebRoy</span>  <span class="cit-name-given-names">S</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Biostrings: Efficient manipulation of biological strings</span>. R package version 2.46.0.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pedersen</span>  <span class="cit-name-given-names">AB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fenton</span>  <span class="cit-name-given-names">A</span></span> (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">Emphasizing the ecology in parasite community ecology</span>. <abbr class="cit-jnl-abbrev">Trends in Ecology and Evolution</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">133</span>–<span class="cit-lpage">139</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D22%26rft.spage%253D133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Phalen</span>  <span class="cit-name-given-names">DN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alvarado</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grillo</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mason</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dobson</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holz</span>  <span class="cit-name-given-names">P</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Prevalence of columbid herpesvirus infection in feral pigeons from New South Wales and Victoria, Australia, with spillover into a wild powerful owl (<em>Ninox struena</em>)</span>. <abbr class="cit-jnl-abbrev">Journal of Wildlife Diseases</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">543</span>–<span class="cit-lpage">551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BWildlife%2BDiseases%26rft.volume%253D53%26rft.spage%253D543%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.26" data-doi="10.1645/GE-398R"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Poulin</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mouillot</span>  <span class="cit-name-given-names">D</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">Combining phylogenetic and ecological information into a new index of host specificity</span>. <abbr class="cit-jnl-abbrev">Journal of Parasitology</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">511</span>–<span class="cit-lpage">514</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BParasitology%26rft.stitle%253DThe%2BJournal%2Bof%2BParasitology%26rft.aulast%253DPoulin%26rft.auinit1%253DR.%26rft.volume%253D91%26rft.issue%253D3%26rft.spage%253D511%26rft.epage%253D514%26rft.atitle%253DCombining%2Bphylogenetic%2Band%2Becological%2Binformation%2Binto%2Ba%2Bnew%2Bindex%2Bof%2Bhost%2Bspecificity.%26rft_id%253Dinfo%253Adoi%252F10.1645%252FGE-398R%26rft_id%253Dinfo%253Apmid%252F16108540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1645/GE-398R&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16108540&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F26%2F856518.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230217100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.27" data-doi="10.1371/journal.pone.0169153"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pozo</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Juste</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vazquez-Moron</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aznar-Lopez</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ibanez</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garin</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aihartza</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Casas</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tenorio</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Echevarria</span>  <span class="cit-name-given-names">JE</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Identification of novel betaherpesviruses in Iberian bats reveals parallel evolution</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0169153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D11%26rft.spage%253De0169153%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0169153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0169153&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.28" data-doi="10.1099/vir.0.006825-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Razafindratsimandresy</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jeanmaire</span>  <span class="cit-name-given-names">EM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Counor</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vasconcelos</span>  <span class="cit-name-given-names">PF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sall</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reynes</span>  <span class="cit-name-given-names">JM</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Partial molecular characterization of alphaherpesviruses isolated from tropical bats</span>. <abbr class="cit-jnl-abbrev">Journal of General Virology</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">44</span>–<span class="cit-lpage">47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DRazafindratsimandresy%26rft.auinit1%253DR.%26rft.volume%253D90%26rft.issue%253D1%26rft.spage%253D44%26rft.epage%253D47%26rft.atitle%253DPartial%2Bmolecular%2Bcharacterization%2Bof%2Balphaherpesviruses%2Bisolated%2Bfrom%2Btropical%2Bbats%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.006825-0%26rft_id%253Dinfo%253Apmid%252F19088271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.006825-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19088271&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F26%2F856518.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262263000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-book" id="cit-856518v1.29"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Fleming</span>  <span class="cit-name-given-names">TH</span></span> and </li><li><span class="cit-ed"><span class="cit-name-surname">Racey</span>  <span class="cit-name-given-names">PA</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Rodríguez-Duran</span>,  <span class="cit-name-given-names">A</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-chapter-title">Bat assemblages in the West Indies: the role of caves</span>. <span class="cit-source">In Island Bats: Evolution, Ecology, and Conservation</span>, <span class="cit-ed"><span class="cit-name-surname">Fleming</span>  <span class="cit-name-given-names">TH</span></span> and <span class="cit-ed"><span class="cit-name-surname">Racey</span>  <span class="cit-name-given-names">PA</span></span> (editors) <span class="cit-publ-loc">Chicago, IL</span>: <span class="cit-publ-name">University of Chicago Press</span>, pp <span class="cit-fpage">265</span>–<span class="cit-lpage">280</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">GC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">JP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haig</span>  <span class="cit-name-given-names">DM</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Malignant catarrhal fever: a review</span>. <abbr class="cit-jnl-abbrev">The Veterinary Journal</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">324</span>–<span class="cit-lpage">335</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BVeterinary%2BJournal%26rft.volume%253D179%26rft.spage%253D324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.31" data-doi="10.1128/JVI.01277-14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sasaki</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Setiyono</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Handharyani</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kobayashi</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rahmadani</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taha</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Adiani</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Subangkit</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nakamura</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sawa</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">T</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Isolation and characterization of a novel alphaherpesvirus in fruit bats</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">9819</span>–<span class="cit-lpage">9829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01277-14%26rft_id%253Dinfo%253Apmid%252F24942567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4OC8xNy85ODE5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTEvMjYvODU2NTE4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.32" data-doi="10.1111/ele.12418"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Seabloom</span>  <span class="cit-name-given-names">EW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Borer</span>  <span class="cit-name-given-names">ET</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gross</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kendig</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lacroix</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">CE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Modrecai</span>  <span class="cit-name-given-names">EA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Power</span>  <span class="cit-name-given-names">AG</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">The community ecology of pathogens: coinfection, coexistence and community composition</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">401</span>–<span class="cit-lpage">415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%2BLetters%26rft.volume%253D18%26rft.spage%253D401%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fele.12418%26rft_id%253Dinfo%253Apmid%252F25728488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ele.12418&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25728488&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F26%2F856518.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.33" data-doi="10.1016/S0021-9975(96)80057-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tandler</span>  <span class="cit-name-given-names">B</span></span> (<span class="cit-pub-date">1996</span>) <span class="cit-article-title">Cytomegalovirus in the principal submandibular gland of the little brown bat, <em>Myotis lucifugus</em></span>. <abbr class="cit-jnl-abbrev">Journal of Comparative Pathology</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomparative%2Bpathology%26rft.stitle%253DJ%2BComp%2BPathol%26rft.aulast%253DTandler%26rft.auinit1%253DB.%26rft.volume%253D114%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D9%26rft.atitle%253DCytomegalovirus%2Bin%2Bthe%2Bprincipal%2Bsubmandibular%2Bgland%2Bof%2Bthe%2Blittle%2Bbrown%2Bbat%252C%2BMyotis%2Blucifugus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0021-9975%252896%252980057-0%26rft_id%253Dinfo%253Apmid%252F8729075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0021-9975(96)80057-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8729075&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F26%2F856518.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.34" data-doi="10.1128/JVI.00697-06"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">XC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">JX</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">SY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">XH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">LF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">BQ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">CL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">KM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">WJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vijaykrishna</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poon</span>  <span class="cit-name-given-names">LLM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peiris</span>  <span class="cit-name-given-names">JSM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">GJD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">Y</span></span> (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">Prevalence and genetic diversity of coronaviruses in bats from China</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">7481</span>–<span class="cit-lpage">7490</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DTang%26rft.auinit1%253DX.%2BC.%26rft.volume%253D80%26rft.issue%253D15%26rft.spage%253D7481%26rft.epage%253D7490%26rft.atitle%253DPrevalence%2Band%2BGenetic%2BDiversity%2Bof%2BCoronaviruses%2Bin%2BBats%2Bfrom%2BChina%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00697-06%26rft_id%253Dinfo%253Apmid%252F16840328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4MC8xNS83NDgxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTEvMjYvODU2NTE4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tazikeh</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Raoofi</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Madadgar</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Akbarein</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mashhadi</span>  <span class="cit-name-given-names">AG.</span></span> <span class="cit-pub-date">2019</span>. <span class="cit-article-title">A survey of equine herpes virus 4 infection in four provinces of Iran using Real Time PCR Taqman assay</span>. <abbr class="cit-jnl-abbrev">Journal of Veterinary Research</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">490</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVeterinary%2BResearch%26rft.volume%253D73%26rft.spage%253D483%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.36" data-doi="10.1126/science.1190333"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Telfer</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lambin</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Birtles</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beldomenico</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burthe</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Begon</span>  <span class="cit-name-given-names">M</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Species interactions in a parasite community drive infection risk in a wildlife population</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">330</span>: <span class="cit-fpage">243</span>–<span class="cit-lpage">246</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTelfer%26rft.auinit1%253DS.%26rft.volume%253D330%26rft.issue%253D6001%26rft.spage%253D243%26rft.epage%253D246%26rft.atitle%253DSpecies%2BInteractions%2Bin%2Ba%2BParasite%2BCommunity%2BDrive%2BInfection%2BRisk%2Bin%2Ba%2BWildlife%2BPopulation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1190333%26rft_id%253Dinfo%253Apmid%252F20929776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzAvNjAwMS8yNDMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMS8yNi84NTY1MTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Van De Vanter</span>  <span class="cit-name-given-names">DR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Warrener</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schultz</span>  <span class="cit-name-given-names">ER</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coulter</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garber</span>  <span class="cit-name-given-names">RL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">TM</span></span> (<span class="cit-pub-date">1996</span>) <span class="cit-article-title">Detection and analysis of diverse herpesviral species by consensus primer PCR</span>. <abbr class="cit-jnl-abbrev">Journal of Clinical Microbiology</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">1666</span>–<span class="cit-lpage">1671</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BMicrobiology%26rft.stitle%253DJ.%2BClin.%2BMicrobiol.%26rft.aulast%253DVanDevanter%26rft.auinit1%253DD.%2BR.%26rft.volume%253D34%26rft.issue%253D7%26rft.spage%253D1666%26rft.epage%253D1671%26rft.atitle%253DDetection%2Band%2Banalysis%2Bof%2Bdiverse%2Bherpesviral%2Bspecies%2Bby%2Bconsensus%2Bprimer%2BPCR%26rft_id%253Dinfo%253Apmid%252F8784566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjk6IjM0LzcvMTY2NiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzExLzI2Lzg1NjUxOC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.38" data-doi="10.1093/infdis/jiw254"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vetter</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fischer</span>   <span class="cit-name-suffix">II</span>  <span class="cit-name-given-names">WA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schibler</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jacobs</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bausch</span>  <span class="cit-name-given-names">DG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kaiser</span>  <span class="cit-name-given-names">L</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Ebola virus shedding and transmission: review of current evidence</span>. <abbr class="cit-jnl-abbrev">The Journal of Infectious Diseases</abbr> <span class="cit-vol">214</span>: <span class="cit-fpage">S177</span>–<span class="cit-lpage">S184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BInfectious%2BDiseases%26rft_id%253Dinfo%253Adoi%252F10.1093%252Finfdis%252Fjiw254%26rft_id%253Dinfo%253Apmid%252F27443613%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/infdis/jiw254&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27443613&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F26%2F856518.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wacharapluesadee</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boongrid</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wanghongsa</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ratanasetyuth</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Supavongwong</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saengsen</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gongal</span>  <span class="cit-name-given-names">GN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hemachudha</span>  <span class="cit-name-given-names">T</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">A longitudinal study of the prevalence of Nipah virus in <em>Pteropus lylei</em> bats in Thailand: evidence for seasonal preference in disease transmission</span>. <abbr class="cit-jnl-abbrev">Vector-Borne and Zoonotic Diseases</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">183</span>–<span class="cit-lpage">190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVector-Borne%2Band%2BZoonotic%2BDiseases%26rft.volume%253D10%26rft.spage%253D183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wada</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sasaki</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Setiyono</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Handharyani</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rahmadani</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taha</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Adiani</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Latief</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kholilullah</span>  <span class="cit-name-given-names">ZA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Subangkit</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kobayashi</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nakamura</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Orba</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sawa</span>  <span class="cit-name-given-names">H</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">Detection of novel gammaherpesviruses from fruit bats in Indonesia</span>. <abbr class="cit-jnl-abbrev">Journal of Medical Microbiology</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">415</span>–<span class="cit-lpage">422</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMedical%2BMicrobiology%26rft.volume%253D67%26rft.spage%253D415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-book" id="cit-856518v1.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wassertheil-Smoller</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smoller</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-chapter-title">Biostatistics and Epidemiology: A Primer for Health and Biomedical Professionals</span> <span class="cit-edition">Fourth Edition</span>, <span class="cit-publ-loc">New York, NY</span>: <span class="cit-publ-name">Springer</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.42" data-doi="10.1093/clinids/14.2.555"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weigler</span>  <span class="cit-name-given-names">BJ</span></span> (<span class="cit-pub-date">1992</span>) <span class="cit-article-title">Biology of B virus in macaque and human hosts: a review</span>. <abbr class="cit-jnl-abbrev">Clinical Infectious Diseases</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">555</span>–<span class="cit-lpage">567</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2BInfectious%2BDiseases%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fclinids%252F14.2.555%26rft_id%253Dinfo%253Apmid%252F1313312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/clinids/14.2.555&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1313312&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F26%2F856518.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HD51800024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.43" data-doi="10.1099/vir.0.83045-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wibbelt</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kurth</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yasmum</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bannert</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nagel</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nitsche</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ehlers</span>  <span class="cit-name-given-names">B</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">Discovery of herpesviruses in bats</span>. <abbr class="cit-jnl-abbrev">Journal of General Virology</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">2651</span>–<span class="cit-lpage">2655</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BGeneral%2BVirology%26rft.stitle%253DJ.%2BGen.%2BVirol.%26rft.aulast%253DWibbelt%26rft.auinit1%253DG.%26rft.volume%253D88%26rft.issue%253D10%26rft.spage%253D2651%26rft.epage%253D2655%26rft.atitle%253DDiscovery%2Bof%2Bherpesviruses%2Bin%2Bbats%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fvir.0.83045-0%26rft_id%253Dinfo%253Apmid%252F17872515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/vir.0.83045-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17872515&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F26%2F856518.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250157000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wibbelt</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">MS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schountz</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Voigt</span>  <span class="cit-name-given-names">CC</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Emerging diseases in Chiroptera: why bats</span>? <abbr class="cit-jnl-abbrev">Biology Letters</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">438</span>–<span class="cit-lpage">440</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiology%2BLetters%26rft.volume%253D6%26rft.spage%253D438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-856518v1.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Winker</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spackman</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Swayne</span>  <span class="cit-name-given-names">DE.</span></span> <span class="cit-article-title">Rarity of Influenza A virus in spring shorebirds, southern Alaska</span>. <abbr class="cit-jnl-abbrev">Emerging Infectious Diseases</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1314</span>–<span class="cit-lpage">1316</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.46" data-doi="10.1128/JVI.00723-12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Todd</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tachedijian</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barr</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marsh</span>  <span class="cit-name-given-names">GA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crameri</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">A novel bat herpesvirus encodes homologues of major histocompatibility complex class I and II, c-type lectin, and a unique family of immune-related genes</span>. <abbr class="cit-jnl-abbrev">Journal of Virology</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">8014</span>–<span class="cit-lpage">8030</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00723-12%26rft_id%253Dinfo%253Apmid%252F22623774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Ni8xNS84MDE0IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTEvMjYvODU2NTE4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-856518v1.47" data-doi="10.1007/s00705-015-2614-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qiu</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziao</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiong</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huo</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ge</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cen</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Q</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">High prevalence and diversity of viruses in the subfamily Gammaherpesvirinae, family Herpesviridae, in fecal specimens from bats of different species in southern China</span>. <abbr class="cit-jnl-abbrev">Archives of Virology</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">135</span>–<span class="cit-lpage">140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArchives%2Bof%2BVirology%26rft.volume%253D161%26rft.spage%253D135%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00705-015-2614-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00705-015-2614-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/856518v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F856518v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/856393v1" title="Chemical signals act as the main reproductive barrier between sister and mimetic Heliconius butterflies" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;November 26, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/856518v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/early/2019/11/26/856518.external-links" target="_self" class="link-icon"><i class="icon-file"></i> <span class="title">Data/Code</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/856518v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/856518v1" class="active" data-icon-position="" data-hide-link-title="0">Quantitative Delineation of Herpesviruses in Bats for use in Ecological Studies</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1010156" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ZeNsg4CrZoPTy36yD43DCZf3Qk8mpSlWB_Oz7pscp-8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1010156" id="node1010156--2" data-pisa="biorxiv;856518v1" data-pisa-master="biorxiv;856518" data-seqnum="1010156" data-apath="/biorxiv/early/2019/11/26/856518.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Quantitative Delineation of Herpesviruses in Bats for use in Ecological Studies</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Anna R.</span> <span class="nlm-surname">Sjodin</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Michael R.</span> <span class="nlm-surname">Willig</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Simon J.</span> <span class="nlm-surname">Anthony</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">856518; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/856518 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/856518v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/856518v1&amp;title=Quantitative%20Delineation%20of%20Herpesviruses%20in%20Bats%20for%20use%20in%20Ecological%20Studies" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/856518v1&amp;title=Quantitative%20Delineation%20of%20Herpesviruses%20in%20Bats%20for%20use%20in%20Ecological%20Studies" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/856518v1&amp;text=Quantitative%20Delineation%20of%20Herpesviruses%20in%20Bats%20for%20use%20in%20Ecological%20Studies" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/856518v1&amp;title=Quantitative%20Delineation%20of%20Herpesviruses%20in%20Bats%20for%20use%20in%20Ecological%20Studies" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/856518v1&amp;t=Quantitative%20Delineation%20of%20Herpesviruses%20in%20Bats%20for%20use%20in%20Ecological%20Studies" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/856518v1&amp;title=Quantitative%20Delineation%20of%20Herpesviruses%20in%20Bats%20for%20use%20in%20Ecological%20Studies" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/856518v1&amp;title=Quantitative%20Delineation%20of%20Herpesviruses%20in%20Bats%20for%20use%20in%20Ecological%20Studies" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1010156" id="node1010156--3" data-pisa="biorxiv;856518v1" data-pisa-master="biorxiv;856518" data-seqnum="1010156" data-apath="/biorxiv/early/2019/11/26/856518.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Quantitative Delineation of Herpesviruses in Bats for use in Ecological Studies</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Anna R.</span> <span class="nlm-surname">Sjodin</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Michael R.</span> <span class="nlm-surname">Willig</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Simon J.</span> <span class="nlm-surname">Anthony</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">856518; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/856518 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Quantitative%20Delineation%20of%20Herpesviruses%20in%20Bats%20for%20use%20in%20Ecological%20Studies&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Sjodin%2C+Anna+R.&amp;rft.au=Willig%2C+Michael+R.&amp;rft.au=Anthony%2C+Simon+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1010156/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1010156/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1010156/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1010156/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1010156/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1010156/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1010156/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1010156/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1010156/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1010156/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1010156/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1010156/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/856518v1&amp;count=horizontal&amp;via=&amp;text=Quantitative%20Delineation%20of%20Herpesviruses%20in%20Bats%20for%20use%20in%20Ecological%20Studies&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/856518v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/856518v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/856518v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/ecology" class="highlight" data-icon-position="" data-hide-link-title="0">Ecology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1789)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(3141)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(2136)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(11098)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4575)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3584)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(5072)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(137)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(3114)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4915)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2044)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7708)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5844)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(7321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(8398)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3345)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(20276)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(147)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(529)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(867)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1311)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(3026)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(736)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(976)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(3039)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(503)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__qVkFI2DFUJCJKlH2laKecaY5NNjWWYhnp4WnclrNmRk.js"></script>
  </body>
</html>
