<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Microbiol Resour Announc</span><span class="iso-abbrev" title="iso-abbrev">Microbiol Resour Announc</span><span class="hwp" title="hwp">ga</span><span class="pmc" title="pmc">mra</span><span class="publisher-id" title="publisher-id">MRA</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Microbiology Resource Announcements</span></div><span class="issn-epub" tagx="issn" title="issn-epub">2576-098X</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">American Society for Microbiology</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">1752 N St., N.W., Washington, DC</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6952674">6952674</a></span><span class="publisher-id" title="publisher-id">MRA01452-19</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1128/MRA.01452-19">10.1128/MRA.01452-19</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Genome Sequences</span></div> <div class="title-group"> <div class="article-title" title="article-title">
Genome Sequences of Chikungunya Virus Strains from Bangladesh and Thailand</div> <div class="alt-title" title="alt-title">
Microbiology Resource Announcement</div> <div class="alt-title" title="alt-title">
Pyke et al.</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0003-3512-4321</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pyke</span><span class="given-names" tagx="given-names" title="given-names">Alyssa T.</span></span><a href="#aff1"><sup>a</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McMahon</span><span class="given-names" tagx="given-names" title="given-names">Jamie</span></span><a href="#aff1"><sup>a</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burtonclay</span><span class="given-names" tagx="given-names" title="given-names">Peter</span></span><a href="#aff1"><sup>a</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nair</span><span class="given-names" tagx="given-names" title="given-names">Neelima</span></span><a href="#aff1"><sup>a</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">De Jong</span><span class="given-names" tagx="given-names" title="given-names">Amanda</span></span><a href="#aff1"><sup>a</sup></a></span><span class="citation_author_institution" id="aff1">[a], <span class="addr-line" title="addr-line">Public Health Virology Laboratory, Forensic and Scientific Services, Coopers Plains, Queensland, Australia</span></span></div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Roux</span><span class="given-names" tagx="given-names" title="given-names">Simon</span></span><span class="role" tagx="role" title="role">Editor</span><span class="citation_author_institution" id="">[], </span></span></div> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
Address correspondence to Alyssa T. Pyke, <span class="email" tagx="email" title="email">Alyssa.Pyke@health.qld.gov.au</span>.</div> <div class="fn-type-other" title="other"> <p><b>Citation</b> Pyke AT, McMahon J, Burtonclay P, Nair N, De Jong A. 2020. Genome sequences of chikungunya virus strains from Bangladesh and Thailand. Microbiol Resour Announc 9:e01452-19. <a href="https://doi.org/10.1128/MRA.01452-19">https://doi.org/10.1128/MRA.01452-19</a>.</p> </div> </div><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020-1</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-1-1</span></span><span class="volume" tagx="volume" title="volume">9</span><span class="issue" tagx="issue" title="issue">2</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e01452-19</span><span class="history" title="history"><span class="received" title="received">received: 2019-11-21</span><span class="accepted" title="accepted">accepted: 2019-12-6</span></span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Crown</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open-access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International license</a>.</span></span></div><a class="self-uri" href="MRA.01452-19.pdf" title="self-uri">MRA.01452-19.pdf</a><div class="abstract" title="abstract"> <p>We sequenced the genomes of two chikungunya virus isolates obtained from viremic patients who had traveled to Australia. The first patient acquired the infection in Bangladesh in 2017, and the second was infected in Thailand in 2019. Phylogenetic sequence analysis demonstrated that both isolates belonged to the East/Central/South African genotype.</p> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
ABSTRACT</div> <p>We sequenced the genomes of two chikungunya virus isolates obtained from viremic patients who had traveled to Australia. The first patient acquired the infection in Bangladesh in 2017, and the second was infected in Thailand in 2019. Phylogenetic sequence analysis demonstrated that both isolates belonged to the East/Central/South African genotype.</p> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="equation-count" title="equation-count">equation-count: </span><span class="ref-count" title="ref-count">ref-count: </span><span class="page-count" title="page-count">page-count: </span><span class="word-count" title="word-count">word-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="cover-date" value="January 2020" /> </div> </div> </div> <div class="body" title="body"> <div class="announcement" title="sec"> <div class="title" tagx="title" title="title">
ANNOUNCEMENT</div> <p>Within the last 20 years, the reemergence and global spread of the arthropod-borne chikungunya virus (CHIKV) (genus <i>Alphavirus</i>, family <i>Togaviridae</i>) have resulted in explosive epidemics throughout the Pacific, Asia, and the Americas, affecting millions of people (<a href="#B1">1</a>). Large-scale CHIKV outbreaks were recently recorded in Dhaka, Bangladesh, in 2017 (<a href="#B2">2</a>) and Thailand in 2019 (<a href="#B3">3</a>). Areas of nonendemicity, such as Queensland, Australia, which harbor populations of <span class="named-content" title="named-content">Aedes aegypti</span> and <span class="named-content" title="named-content">Aedes albopictus</span> mosquitoes, are at risk of CHIKV importation by viremic travelers and subsequent autochthonous transmission (<a href="#B1">1</a>, <a href="#B4">4</a>).</p> <p>In July 2017, a febrile male patient traveled to Brisbane, Australia, from Bangladesh. Similarly, in September 2019, a female patient with fever and joint pains arrived in Brisbane after recent travel in Thailand. Acute-phase sera were analyzed using a specific CHIKV reverse-transcription real-time PCR assay (<a href="#B4">4</a>) which detected CHIKV RNA in each patient sample. The same patient sera were inoculated onto <i>A. albopictus</i> C6/36 cell monolayers. Two CHIKV isolates, namely, Bangladesh 2017 and Thail 2019, were recovered and subsequently used for whole-genome sequencing (WGS) and phylogenetic analysis.</p> <p>To support ongoing and enhanced CHIKV surveillance within Australasia and provide contemporary sequences for continued scrutiny of existing molecular diagnostic assays, we performed WGS as previously described (<a href="#B5">5</a>, <a href="#B6">6</a>). Briefly, total RNA was extracted from passage 1 C6/36 culture supernatants from each of the Bangladesh 2017 and Thail 2019 isolates using the QIAamp viral RNA extraction kit (Qiagen, Chadstone, Australia) without carrier RNA. Host and potentially contaminating microbial DNA was removed with DNase treatment (Heat&amp;amp;Run kit; ArcticZymes, Scientifix, South Yarra, Australia). RNA was converted to cDNA using the ProtoScript II first-strand cDNA kit (New England Biolabs), and second-strand cDNA synthesis was performed using an enzyme cocktail of <span class="named-content" title="named-content">Escherichia coli</span> DNA ligase, DNA polymerase I, and RNase H (New England Biolabs). The Nextera XT kit was used for cDNA library construction, and paired-end (2 × 151 nucleotides [nt]) sequencing was performed using the V2 midoutput kit on a NextSeq 500 machine (Illumina, San Diego, CA).</p> <p>Illumina sequencing yielded 10,299,664 and 16,336,098 reads for Bangladesh 2017 and Thail 2019, respectively, and raw sequence reads were processed using Geneious R10 version 10.2.6 software (<a href="#B7">7</a>). Complete genome sequence assembly was achieved by mapping to a reference CHIKV genome sequence (strain CHIK31, GenBank accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/EU564335">EU564335</a>) that included 5′ and 3′ untranslated terminal sequences, using default parameters and the low-sensitivity setting. For Bangladesh 2017 (11,812 nt with 50.1% G+C content), a total of 4,908,657 reads with an average coverage depth of 45,504× were mapped to strain CHIK31. Similarly, a total of 8,240,564 reads with an average coverage depth of 78,217× were mapped to strain CHIK31 for Thail 2019 (11,812 nt with 50.1% G+C content).</p> <p>Genome sequence identities were determined using the online version of blastn (<a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">https://blast.ncbi.nlm.nih.gov/Blast.cgi</a>). Bangladesh 2017 and Thail 2019 shared 99.83% sequence identity with and were most closely related to a Chinese 2017 isolate, QZ0823 (GenBank accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/MH400249">MH400249</a>), which shared 99.95% and 99.85% sequence identity with Bangladesh 2017 and Thail 2019, respectively. Further sequence and phylogenetic analysis (FastTree software version 2.1.11 [<a href="#B8">8</a>] and the general time-reversible [GTR] nucleotide substitution model) of the region encompassing the putative nonstructural polyprotein (nsP) (nsP1-nsP2-nsP3 termination product or nsP1-nsP2-nsP3-nsP4 read-through product), the subgenomic 26S RNA promoter region, and the putative structural polyprotein (C-E3-E2-6K-E1) indicated that both Bangladesh 2017 and Thail 2019 grouped within the Indian Ocean lineage of the East/Central/South African (ECSA) genotype (<a href="#fig1">Fig. 1</a>). However, the E1-A226V mutation first detected during the 2005–2006 CHIKV epidemic on La Réunion Island (<a href="#B9">9</a>) was not found in either strain.</p> <div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">FIG 1</span></a></div>   <p>Approximately maximum-likelihood phylogenetic tree (midpoint rooted for branch visibility only) inferred for 63 CHIKV genome sequences (excluding 5′ and 3′ untranslated regions) using FastTree version 2.1.11 software and the GTR nucleotide substitution model with the default setting of 20 for rate categories of sites (<a href="#B8">8</a>). Percentage Shimodaira-Hasegawa-like local support values are shown for key nodes. Multiple sequence alignments were performed using the Multiple Alignment using Fast Fourier Transform (MAFFT) program version 7.450 and Geneious version 10.2.6 software. The three major CHIKV genotypes (West African, Asian, and ECSA) are shown, including grouping of the Bangladesh 2017 (GenBank accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/MF773566">MF773566</a>) and Thail 2019 (GenBank accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/MN630017">MN630017</a>) isolates (red font) within the ECSA-Indian Ocean lineage.</p>   </div> <div class="dataavailability." title="sec"> <div class="title" tagx="title" title="title">
Data availability.</div> <p>Raw sequencing reads were deposited in the Sequence Read Archive under the accession numbers <a href="https://www.ncbi.nlm.nih.gov/sra/?term=SAMN13164600">SAMN13164600</a> (Bangladesh 2017) and <a href="https://www.ncbi.nlm.nih.gov/sra/?term=SAMN13164601">SAMN13164601</a> (Thail 2019) and BioProject number <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA583228">PRJNA583228</a>. The genome sequences have been deposited in GenBank under the accession numbers <a href="https://www.ncbi.nlm.nih.gov/nuccore/MF773566">MF773566</a> (Bangladesh 2017) and <a href="https://www.ncbi.nlm.nih.gov/nuccore/MN630017">MN630017</a> (Thail 2019).</p> </div> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
ACKNOWLEDGMENTS</div> <p>We are grateful to staff at the Royal Brisbane and Women’s Hospital, the Princess Alexandra Hospital, and the Public Health Virology Laboratory, Forensic and Scientific Services, who contributed to the collection of patient sera and clinical and travel histories.</p> <p>This work received ethical clearance from the Forensic and Scientific Services Human Ethics Committee.</p> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
REFERENCES</div> <li tag="ref"><a name="B1" /><span class="label" tagx="label" title="label">1.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pyke</span><span class="given-names" tagx="given-names" title="given-names">AT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moore</span><span class="given-names" tagx="given-names" title="given-names">PR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McMahon</span><span class="given-names" tagx="given-names" title="given-names">J</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">New insights into chikungunya virus emergence and spread from Southeast Asia</span>. <span class="source" tagx="source" title="source">Emerg Microbes Infect</span><span class="volume" tagx="volume" title="volume">7</span>:<span class="fpage" tagx="fpage" title="fpage">26</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1038/s41426-018-0024-2">10.1038/s41426-018-0024-2</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29535302">29535302</a></span></span></li> <li tag="ref"><a name="B2" /><span class="label" tagx="label" title="label">2.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hossain</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hasan</span><span class="given-names" tagx="given-names" title="given-names">MM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Islam</span><span class="given-names" tagx="given-names" title="given-names">MS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Islam</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mozaffor</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khan</span><span class="given-names" tagx="given-names" title="given-names">MAS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ahmed</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Akhtar</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chowdhury</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arafat</span><span class="given-names" tagx="given-names" title="given-names">SMY</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khaleque</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Khan</span><span class="given-names" tagx="given-names" title="given-names">ZJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dipta</span><span class="given-names" tagx="given-names" title="given-names">TF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Asna</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hossain</span><span class="given-names" tagx="given-names" title="given-names">MA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Aziz</span><span class="given-names" tagx="given-names" title="given-names">KS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mosabbir</span><span class="given-names" tagx="given-names" title="given-names">AA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raheem</span><span class="given-names" tagx="given-names" title="given-names">E</span></span></span><span class="year" tagx="year" title="year">2018</span><span class="mixed-article-title" title="mixed-article-title">Chikungunya outbreak (2017) in Bangladesh: clinical profile, economic impact and quality of life during the acute phase of the disease</span>. <span class="source" tagx="source" title="source">PLoS Negl Trop Dis</span><span class="volume" tagx="volume" title="volume">12</span>:<span class="elocation-id" tagx="elocation-id" title="elocation-id">e0006561</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pntd.0006561">10.1371/journal.pntd.0006561</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29874242">29874242</a></span></span></li> <li tag="ref"><a name="B3" /><span class="label" tagx="label" title="label">3.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Intayot</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Phumee</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Boonserm</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sor-Suwan</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buathong</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wacharapluesadee</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brownell</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Poovorawan</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Siriyasatien</span><span class="given-names" tagx="given-names" title="given-names">P</span></span></span><span class="year" tagx="year" title="year">2019</span><span class="mixed-article-title" title="mixed-article-title">Genetic characterization of chikungunya virus in field-caught Aedes aegypti mosquitoes collected during the recent outbreaks in 2019, Thailand</span>. <span class="source" tagx="source" title="source">Pathogens</span><span class="volume" tagx="volume" title="volume">8</span>:<span class="fpage" tagx="fpage" title="fpage">121</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.3390/pathogens8030121">10.3390/pathogens8030121</a></span>.</span></li> <li tag="ref"><a name="B4" /><span class="label" tagx="label" title="label">4.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">van den Hurk</span><span class="given-names" tagx="given-names" title="given-names">AF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hall-Mendelin</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pyke</span><span class="given-names" tagx="given-names" title="given-names">AT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Smith</span><span class="given-names" tagx="given-names" title="given-names">GA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mackenzie</span><span class="given-names" tagx="given-names" title="given-names">JS</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">Vector competence of Australian mosquitoes for chikungunya virus</span>. <span class="source" tagx="source" title="source">Vector Borne Zoonotic Dis</span><span class="volume" tagx="volume" title="volume">10</span>:<span class="fpage" tagx="fpage" title="fpage">489</span>–<span class="lpage" tagx="lpage" title="lpage">495</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1089/vbz.2009.0106">10.1089/vbz.2009.0106</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19877822">19877822</a></span></span></li> <li tag="ref"><a name="B5" /><span class="label" tagx="label" title="label">5.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pyke</span><span class="given-names" tagx="given-names" title="given-names">AT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McMahon</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Warrilow</span><span class="given-names" tagx="given-names" title="given-names">D</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Complete coding sequence of a case of chikungunya virus imported into Australia</span>. <span class="source" tagx="source" title="source">Genome Announc</span><span class="volume" tagx="volume" title="volume">5</span>:<span class="elocation-id" tagx="elocation-id" title="elocation-id">e00310-17</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1128/genomeA.00310-17">10.1128/genomeA.00310-17</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28495775">28495775</a></span></span></li> <li tag="ref"><a name="B6" /><span class="label" tagx="label" title="label">6.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pyke</span><span class="given-names" tagx="given-names" title="given-names">AT</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Warrilow</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moore</span><span class="given-names" tagx="given-names" title="given-names">PR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McMahon</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harrower</span><span class="given-names" tagx="given-names" title="given-names">B</span></span></span><span class="year" tagx="year" title="year">2017</span><span class="mixed-article-title" title="mixed-article-title">Complete genome sequence of a highly divergent dengue virus type 2 strain, imported into Australia from Sabah, Malaysia</span>. <span class="source" tagx="source" title="source">Genome Announc</span><span class="volume" tagx="volume" title="volume">5</span>:<span class="elocation-id" tagx="elocation-id" title="elocation-id">e00546-17</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1128/genomeA.00546-17">10.1128/genomeA.00546-17</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28729258">28729258</a></span></span></li> <li tag="ref"><a name="B7" /><span class="label" tagx="label" title="label">7.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kearse</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moir</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wilson</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stones-Havas</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cheung</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sturrock</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Buxton</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cooper</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Markowitz</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Duran</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Thierer</span><span class="given-names" tagx="given-names" title="given-names">T</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ashton</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Meintjes</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Drummond</span><span class="given-names" tagx="given-names" title="given-names">A</span></span></span><span class="year" tagx="year" title="year">2012</span><span class="mixed-article-title" title="mixed-article-title">Geneious Basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data</span>. <span class="source" tagx="source" title="source">Bioinformatics</span><span class="volume" tagx="volume" title="volume">28</span>:<span class="fpage" tagx="fpage" title="fpage">1647</span>–<span class="lpage" tagx="lpage" title="lpage">1649</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1093/bioinformatics/bts199">10.1093/bioinformatics/bts199</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22543367">22543367</a></span></span></li> <li tag="ref"><a name="B8" /><span class="label" tagx="label" title="label">8.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Price</span><span class="given-names" tagx="given-names" title="given-names">MN</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dehal</span><span class="given-names" tagx="given-names" title="given-names">PS</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Arkin</span><span class="given-names" tagx="given-names" title="given-names">AP</span></span></span><span class="year" tagx="year" title="year">2010</span><span class="mixed-article-title" title="mixed-article-title">FastTree 2—approximately maximum-likelihood trees for large alignments</span>. <span class="source" tagx="source" title="source">PLoS One</span><span class="volume" tagx="volume" title="volume">5</span>:<span class="elocation-id" tagx="elocation-id" title="elocation-id">e9490</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pone.0009490">10.1371/journal.pone.0009490</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20224823">20224823</a></span></span></li> <li tag="ref"><a name="B9" /><span class="label" tagx="label" title="label">9.</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsetsarkin</span><span class="given-names" tagx="given-names" title="given-names">KA</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Vanlandingham</span><span class="given-names" tagx="given-names" title="given-names">DL</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McGee</span><span class="given-names" tagx="given-names" title="given-names">CE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Higgs</span><span class="given-names" tagx="given-names" title="given-names">S</span></span></span><span class="year" tagx="year" title="year">2007</span><span class="mixed-article-title" title="mixed-article-title">A single mutation in chikungunya virus affects vector specificity and epidemic potential</span>. <span class="source" tagx="source" title="source">PLoS Pathog</span><span class="volume" tagx="volume" title="volume">3</span>:<span class="elocation-id" tagx="elocation-id" title="elocation-id">e201</span>. doi:<span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.0030201">10.1371/journal.ppat.0030201</a></span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18069894">18069894</a></span></span></li> </ul> </div> </div>  </body></html>