<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">PLoS Pathog</span><span class="iso-abbrev" title="iso-abbrev">PLoS Pathog</span><span class="publisher-id" title="publisher-id">plos</span><span class="pmc" title="pmc">plospath</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">PLoS Pathogens</span></div><span class="issn-ppub" tagx="issn" title="issn-ppub">1553-7366</span><span class="issn-epub" tagx="issn" title="issn-epub">1553-7374</span><div class="publisher" tagx="publisher" title="publisher"><span class="publisher-name" tagx="publisher-name" title="publisher-name">Public Library of Science</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">San Francisco, CA USA</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7272002">7272002</a></span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1371/journal.ppat.1008607">10.1371/journal.ppat.1008607</a></span><span class="publisher-id" title="publisher-id">PPATHOGENS-D-20-00730</span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Opinion</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Cell Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Cellular Types</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Animal Cells</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Blood Cells</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">White Blood Cells</span>: <span class="subject" title="subject">T Cells</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Cell Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Cellular Types</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Animal Cells</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Immune Cells</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">White Blood Cells</span>: <span class="subject" title="subject">T Cells</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Immunology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Immune Cells</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">White Blood Cells</span>: <span class="subject" title="subject">T Cells</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Immunology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Immune Cells</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">White Blood Cells</span>: <span class="subject" title="subject">T Cells</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Immunology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Vaccination and Immunization</span>: <span class="subject" title="subject">Vaccine Development</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Immunology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Vaccination and Immunization</span>: <span class="subject" title="subject">Vaccine Development</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Public and Occupational Health</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Preventive Medicine</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Vaccination and Immunization</span>: <span class="subject" title="subject">Vaccine Development</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Immunology</span>: <span class="subject" title="subject">Immune Response</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Immunology</span>: <span class="subject" title="subject">Immune Response</span></div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Cell Biology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Cellular Types</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Animal Cells</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Immune Cells</span>: <span class="subject" title="subject">Antigen-Presenting Cells</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Immunology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Immune Cells</span>: <span class="subject" title="subject">Antigen-Presenting Cells</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Immunology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Immune Cells</span>: <span class="subject" title="subject">Antigen-Presenting Cells</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Physiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Immune Physiology</span>: <span class="subject" title="subject">Antigens</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Physiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Immune Physiology</span>: <span class="subject" title="subject">Antigens</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Immunology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Immune System Proteins</span>: <span class="subject" title="subject">Antigens</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Immunology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Immune System Proteins</span>: <span class="subject" title="subject">Antigens</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and Life Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biochemistry</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Proteins</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Immune System Proteins</span>: <span class="subject" title="subject">Antigens</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Research and Analysis Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Database and Informatics Methods</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Bioinformatics</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Sequence Analysis</span>: <span class="subject" title="subject">Sequence Motif Analysis</span></div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and Health Sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Infectious Diseases</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Infectious Disease Control</span>: <span class="subject" title="subject">Vaccines</span></div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">DNA viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Herpesviruses</span>: <span class="subject" title="subject">Epstein-Barr virus</span></div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medical microbiology</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Herpesviruses</span>: <span class="subject" title="subject">Epstein-Barr virus</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Medicine and health sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathology and laboratory medicine</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbial pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Herpesviruses</span>: <span class="subject" title="subject">Epstein-Barr virus</span></div> </div> </div> </div> </div> </div> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Biology and life sciences</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Organisms</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viruses</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Viral pathogens</span><div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Herpesviruses</span>: <span class="subject" title="subject">Epstein-Barr virus</span></div> </div> </div> </div> </div> </div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
In silico veritas? Potential limitations for SARS-CoV-2 vaccine development based on T-cell epitope prediction</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Silva-Arrieta</span><span class="given-names" tagx="given-names" title="given-names">Sandra</span></span><a href="#aff001"><sup>1</sup></a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goulder</span><span class="given-names" tagx="given-names" title="given-names">Philip J. R.</span></span><a href="#aff002"><sup>2</sup></a><a href="#aff003"><sup>3</sup></a><a href="#aff004"><sup>4</sup></a></span><span class="contrib" title="contrib"><span class="contrib-id" tagx="contrib-id" title="contrib-id">http://orcid.org/0000-0002-0548-5778</span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brander</span><span class="given-names" tagx="given-names" title="given-names">Christian</span></span><a href="#aff001"><sup>1</sup></a><a href="#aff005"><sup>5</sup></a><a href="#aff006"><sup>6</sup></a><a href="#cor001">*</a></span></div><span class="citation_author_institution" id="aff001">[1], <span class="addr-line" title="addr-line">IrsiCaixa AIDS Research Institute, Hospital Germans Trias i Pujol, Institute for Health Science Research Germans Trias i Pujol (IGTP), Badalona, Spain</span></span><span class="citation_author_institution" id="aff002">[2], <span class="addr-line" title="addr-line">Department of Paediatrics, University of Oxford, Oxford, United Kingdom</span></span><span class="citation_author_institution" id="aff003">[3], <span class="addr-line" title="addr-line">HIV Pathogenesis Programme, The Doris Duke Medical Research Institute, University of KwaZulu-Natal, Durban, South Africa</span></span><span class="citation_author_institution" id="aff004">[4], <span class="addr-line" title="addr-line">Ragon Institute of MGH, MIT, and Harvard, Cambridge, Massachusetts, United States of America</span></span><span class="citation_author_institution" id="aff005">[5], <span class="addr-line" title="addr-line">University of Vicâ€"Central University of Catalonia, Catalonia, Vic, Spain</span></span><span class="citation_author_institution" id="aff006">[6], <span class="addr-line" title="addr-line">Catalan Institution for Research and Advanced Studies (ICREA), Barcelona, Spain</span></span><div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haldar</span><span class="given-names" tagx="given-names" title="given-names">Kasturi</span></span><span class="role" tagx="role" title="role">Editor</span><a href="#edit1" /></span></div><span class="citation_author_institution" id="edit1">[], <span class="addr-line" title="addr-line">University of Notre Dame, UNITED STATES</span></span><div class="author-notes" title="author-notes"> <div class="fn-type-COI-statement" title="COI-statement"> <p>The authors have declared that no competing interests exist.</p> </div> <div class="corresp" title="corresp">
* E-mail: <span class="email" tagx="email" title="email">cbrander@irsicaixa.es</span></div> </div><span class="pub-date-epub" title="pub-date-epub">epub: <span>2020-6-6</span></span><span class="pub-date-collection" title="pub-date-collection">collection: <span>2020-6</span></span><span class="volume" tagx="volume" title="volume">16</span><span class="issue" tagx="issue" title="issue">6</span><span class="elocation-id" tagx="elocation-id" title="elocation-id">e1008607</span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Silva-Arrieta et al</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</span></span></div><a class="self-uri" href="ppat.1008607.pdf" title="self-uri">ppat.1008607.pdf</a><div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="funding-statement" title="funding-statement">
The present work was in parts support by grants from the Wellcome Trust WT104748MA and NIH RO1 AI133673 to PJRG and by the European Unionâ€™s Horizon 2020 research and innovation program under grant agreement 681137-EAVI2020 and NIH grant P01-AI131568 to CB. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</div> </div> <div class="counts" title="counts"><span class="fig-count" title="fig-count">fig-count: </span><span class="table-count" title="table-count">table-count: </span><span class="page-count" title="page-count">page-count: </span></div> <div class="custom-meta-group" title="custom-meta-group"> <meta name="Outbreaks" value="COVID-19" /> </div> </div> </div> <div class="body" title="body"> <p>The development of a vaccine able to prevent infection or severe disease course of SARS-CoV-2 is a priority to stem the current COVID-19 pandemic and to be better prepared for future flare-ups. To accelerate T-cell immunogen design, many current approaches are employing epitope prediction strategies. Although such approaches have great merit, it is also important that unbiased approaches to characterizing the T-cell response to SARS-CoV-2 are incorporated into vaccine design, in order to generate a comprehensive picture of the total virus-specific T-cell response and to define correlates of protective immunity against the virus.</p> <p>Ever since the first identification of binding motifs for T-cell antigens presented by HLA class I molecules by Rammensee and colleagues almost 30 years ago, epitope identification has been greatly facilitated by epitope prediction algorithms [<a href="#ppat.1008607.ref001">1</a>]. Over the years, many vaccines designs targeting infectious pathogens as well as cancer neoantigens have been based on in silico prediction of potential HLA class Iâ€"restricted epitopes, and a series of vaccine candidates that apply such strategies to SARS-CoV-2 are currently in development. However, even though numerous prediction algorithms have been developed and gradually improved, there are several considerations that may threaten or limit the success of such approaches.</p> <div class="epitopelengthishighlyvariableandnotproperlycapturedbypredictionalgorithms" title="sec"> <div class="title" tagx="title" title="title">
Epitope length is highly variable and not properly captured by prediction algorithms</div> <p>Although the initial characterization of eluted HLA class I epitopes by the Rammensee lab in 1991 produced predominantly 9-mer epitopes, there was already the notion of longer and shorter sequences that can be bound and effectively presented by HLA class I molecules [<a href="#ppat.1008607.ref001">1</a>]. This observation has been validated many times over since, with some HLA class I alleles for which the 9-mer seems to be rather the exceptional length than the rule (such as HLA-B8, B35, B57, B27, and others). In fact, even the first identified epitopes in influenza and HIV infection, defined by other approaches a couple of years before the first binding motif was described, turned out to be 13 and 10 amino acids in length, respectively [<a href="#ppat.1008607.ref002">2</a>,<a href="#ppat.1008607.ref003">3</a>]. Especially for HIV, we have equally fallen victim to consider â€œoptimalâ€� HLA class I epitopes those for which functional experiments have identified the shortest reactive peptide and thereby unwillingly contributed to cementing the dogma of 9-mer optimal epitope length as the rule [<a href="#ppat.1008607.ref004">4</a>]. However, based on elution studies by many laboratories and crystallographic structural analyses, it is evident that many epitope variants presented by HLA class I molecules can exceed the 9-mer length [<a href="#ppat.1008607.ref005">5</a>,<a href="#ppat.1008607.ref006">6</a>]. In fact, even for members of the HLA-A11 allele family that have very strict and limited C-anchor binding requirements, elution studies have demonstrated that only about 20% of the (self and viral) epitopes are exactly 9 amino acids in length [<a href="#ppat.1008607.ref007">7</a>]. Of note, this was done in a system that uses soluble class I molecules secreted from the antigen-presenting cells and thus limits the bias to also identify (longer) processing intermediates. Similar observations have been made for other HLA class I allelesâ€"for instance, HLA-B8, which consistently presents peptides that are shorter than 9-mers; or HLA-B57, for which the immunodominant and protective HIV-specific epitope is an 11-mer; or where epitopes have been found to be entirely embedded in other epitopes [<a href="#ppat.1008607.ref008">8</a>], all cases that are generally not identified or scored as independent epitopes by prediction algorithms.</p> </div> <div class="multiplespecificitiestowardthesameepitoperegionscanbeelicitedbyallowingforlengthvariants" title="sec"> <div class="title" tagx="title" title="title">
Multiple specificities toward the same epitope regions can be elicited by allowing for length variants</div> <p>One major limitation of using predicted epitope sequences of 9 amino acids in T-cell vaccine designs is that T-cell responses to only that specific region will be induced by the vaccine. However, the example of HIV infection (which likely has the best-characterized epitope â€œlandscapeâ€� described) shows how this can limit the vaccine-induced response: by (1) screening just 5 amino acids up- and downstream of each described optimal (mostly 9-mer) epitope for suitable anchor positions satisfying the currently known allele-specific binding motif and (2) allowing for an epitope length of 9 to 12 amino acids, the number of potential independent epitopes that could bind the given HLA class I molecule and induce T-cell responses to the same region more than doubles [<a href="#ppat.1008607.ref005">5</a>]. Although this may be of paramount importance to cope with HIV variability and to prevent rapid cytotoxic T-lymphocyte (CTL) escape, other viruses, even genetically robust DNA-based viruses such as Epstein-Barr virus (EBV), have been shown to present as swarms of quasi-species and to be able to develop effective CTL escape variants as well [<a href="#ppat.1008607.ref009">9</a>,<a href="#ppat.1008607.ref010">10</a>]. Thus, it seems critical that SARS-CoV-2 vaccine development take this aspect into account and test immunogen designs that can target the same epitopic region by multiple specificities, ideally composed of a polyclonal T-cell receptor repertoire to (1) broaden the response and thus increase the chances of eliciting more potent clonotypes and (2) more effectively prevent T-cell escape.</p> </div> <div class="hlaclassibindingispromiscuousandmotifsarepoorlydefinedforlesswell-studiedhlaclassialleles" title="sec"> <div class="title" tagx="title" title="title">
HLA class I binding is promiscuous and motifs are poorly defined for less well-studied HLA class I alleles</div> <p>There are to date more than 20,000 HLA class I alleles described that translate into more than 11,000 different HLA class A, B, and C molecules. For many of these alleles, sequence similarities and structural analyses have allowed grouping them into larger, so-called HLA supertypes that share epitope binding similarities [<a href="#ppat.1008607.ref011">11</a>]. However, there is extensive binding promiscuity even by short 9-mer peptides that go well beyond the specific allele and its assigned supertype and for which commonly used minimal cutoffs of binding affinities would not identify the reactive peptides [<a href="#ppat.1008607.ref012">12</a>]. In fact, screening data using several hundred epitopes derived from different viral infections suggest that the current prediction algorithm may miss a large number of reactive â€œoptimalâ€� epitopes and possibly an even larger proportion of responses when predicting potential epitopes on a full protein sequence [<a href="#ppat.1008607.ref012">12</a>]. An extreme example of this may be epitopes predicted (and presented) by HLA-E, for which binding peptides showed an unexpected broad permissiveness at anchor positions and extensive structural freedom to bind to the presenting class I molecule [<a href="#ppat.1008607.ref013">13</a>]. The current urgency by which the scientific community attempts to achieve a viable T-cell vaccine to SARS-CoV-2 will, however, not allow generation of larger training sets that could improve these predictions and thus may mislead the design into regions of the virus that are rich in epitopes fulfilling binding motifs of the most commonly studied class I alleles. In addition, these approaches will not be able to sufficiently cover the genetically different, and often more numerous, populations with less well-characterized genetics and, in parallel, less well-established healthcare systems in which vaccines may be the only way to stem against the pandemic.</p> </div> <div class="antigen-processingpreferencesandtap-mediatedpeptidetranslocation" title="sec"> <div class="title" tagx="title" title="title">
Antigen-processing preferences and TAP-mediated peptide translocation</div> <p>Another critical limitation that is not properly addressed by most epitope prediction tools is the need to identify appropriate antigen-processing sites that could give rise to predicted, presented epitopes. As the composition of the antigen-processing machinery changes upon proinflammatory signals and can differ between different cell types, currently known prediction algorithms will likely not be suited to estimate the processing efficacy of viral antigens in the context of an infected target cell. Thus, without functional validation, there exists the risk that predicted epitopes may represent great theoretical or actual binder to HLA class I molecules, but they are either not processed at all or not processed at sufficient levels to induce a strong immune response and to sensitize infected target cells for CTL-mediated killing. In addition, the allelic diversity of TAP (transporter associated with antigen processing) genes, which encode for proteins that form a bottleneck for the translocation of processed antigen into the lumen of the endoplasmatic reticulum, are only partly integrated into peptide prediction algorithms.</p> </div> <div class="functionalcharacterization,cross-reactivity,andimmunopathology" title="sec"> <div class="title" tagx="title" title="title">
Functional characterization, cross-reactivity, and immunopathology</div> <p>Finally, there are evident advantages in identifying antigens not only to protect from SARS-CoV-2 infection but to develop vaccine candidates that could act against past and future coronavirus outbreaks. This would call for focus on particularly conserved regions between isolates and different coronaviruses, something that can be readily implemented in epitope prediction strategies, even if conservation on an epitope level does not need to be complete. Whether such cross-reactive T-cell responses could indeed mediate cross-protection and are associated with effective control of different coronaviruses, remains to be seen. Of note, emerging data start providing some insights into T-cell effector function profiles that may be protective and those that may be associated with a more severe clinical course of SARS-CoV-2 infection [<a href="#ppat.1008607.ref014">14</a>]. This is reminiscent of the situation in Dengue virus infection, in which some cross-reactive T-cell responses have been associated with the observed immunopathological consequence [<a href="#ppat.1008607.ref015">15</a>]. Thus, a functional assessment of T-cell responses in individuals with mild, moderate, and severe COVID-19 disease courses may be indicated, so that potential detrimental effects of preexisting, cross-reactive T-cell immunity can be avoided. However, without an unbiased screening for total virus-specific T-cell responses with classical and alternative T-cell effector functions [<a href="#ppat.1008607.ref016">16</a>], it will be difficult to differentiate most beneficial from potentially harmful T-cell specificities.</p> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
Conclusions</div> <p>With the urgency to develop effective measures to control the COVID-19 pandemic and to design vaccine strategies useful to confront future outbreaks and epidemics of SARS-CoV-2 and related coronaviruses, accelerated programs for T-cell immunogen design are needed. T-cell epitope prediction algorithms are an effective tool to narrow down the potential immunogen cargo in a future SARS-CoV-2 vaccine from the total viral proteome. However, approaches exclusively based on 9-mer epitope prediction will potentially miss critically important responses, and even those based on 9-mer and 10-mer epitope prediction have similar shortcomings, for the reasons described above. In addition, a relatively unbiased approach to characterizing the T-cell response using overlapping peptides will facilitate understanding of immune correlates of SARS-CoV-2 control versus disease. The benefit of the predicted 9-mer/10-mer approach would be that, when used in combination with panels of overlapping peptides spanning the viral proteome, optimal epitopes will be more rapidly identified and, at the same time, immune correlates of disease protection evaluated in an unbiased fashion. These considerations should not be overlooked, as invaluable time and resources could be directed in directions that may not yield the desired success.</p> </div> </div> <div class="back" title="back"> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="ppat.1008607.ref001" /><span class="label" tagx="label" title="label">1</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Falk</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">RÃ¶tzschke</span><span class="given-names" tagx="given-names" title="given-names">O</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stevanovic</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jung</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rammensee</span><span class="given-names" tagx="given-names" title="given-names">HG</span></span>. <span class="mixed-article-title" title="mixed-article-title">Allele-specific motifs revealed by sequencing of self-peptides eluted from MHC molecules</span>. <span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">1991</span>;<span class="volume" tagx="volume" title="volume">351</span>(<span class="issue" tagx="issue" title="issue">6324</span>):<span class="fpage" tagx="fpage" title="fpage">290</span>â€"<span class="lpage" tagx="lpage" title="lpage">6</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1709722">1709722</a></span></span></li> <li tag="ref"><a name="ppat.1008607.ref002" /><span class="label" tagx="label" title="label">2</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gotch</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rothbard</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Howland</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Townsend</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McMichael</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>. <span class="mixed-article-title" title="mixed-article-title">Cytotoxic T lymphocytes recognize a fragment of influenza virus matrix protein in association with HLA-A2</span>. <span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">1987</span>;<span class="volume" tagx="volume" title="volume">326</span>(<span class="issue" tagx="issue" title="issue">6116</span>):<span class="fpage" tagx="fpage" title="fpage">881</span>â€"<span class="lpage" tagx="lpage" title="lpage">2</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2437457">2437457</a></span></span></li> <li tag="ref"><a name="ppat.1008607.ref003" /><span class="label" tagx="label" title="label">3</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Nixon</span><span class="given-names" tagx="given-names" title="given-names">DF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Townsend</span><span class="given-names" tagx="given-names" title="given-names">AR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Elvin</span><span class="given-names" tagx="given-names" title="given-names">JG</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rizza</span><span class="given-names" tagx="given-names" title="given-names">CR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gallwey</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McMichael</span><span class="given-names" tagx="given-names" title="given-names">AJ</span></span>. <span class="mixed-article-title" title="mixed-article-title">HIV-1 gag-specific cytotoxic T lymphocytes defined with recombinant vaccinia virus and synthetic peptides</span><span class="source" tagx="source" title="source">Nature</span>. <span class="year" tagx="year" title="year">1988</span>;<span class="volume" tagx="volume" title="volume">336</span>(<span class="issue" tagx="issue" title="issue">6198</span>):<span class="fpage" tagx="fpage" title="fpage">484</span>â€"<span class="lpage" tagx="lpage" title="lpage">7</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2461519">2461519</a></span></span></li> <li tag="ref"><a name="ppat.1008607.ref004" /><span class="label" tagx="label" title="label">4</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brander</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walker</span><span class="given-names" tagx="given-names" title="given-names">BD</span></span>. <span class="chapter-title" tagx="chapter-title" title="chapter-title">The HLA-class I restricted CTL response in HIV-1 infection: Identification of optimal epitopes</span> In: <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Korber</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brander</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walker</span><span class="given-names" tagx="given-names" title="given-names">BD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koup</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moore</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haynes</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Myers</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, editors. <span class="source" tagx="source" title="source">HIV Molecular Immunology Database 1995</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Los Alamos, NM</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Los Alamos National Laboratory: Theoretical Biology and Biophysics</span>; <span class="year" tagx="year" title="year">1995</span> p. <span class="fpage" tagx="fpage" title="fpage">1</span>â€"<span class="lpage" tagx="lpage" title="lpage">9</span>.</span></li> <li tag="ref"><a name="ppat.1008607.ref005" /><span class="label" tagx="label" title="label">5</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Llano</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">CedeÃ±o</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Silva-Arrieta</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brander</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>. <span class="chapter-title" tagx="chapter-title" title="chapter-title">The 2019 Optimal HIV CTL epitopes update: Growing diversity in epitope length and HLA restriction</span> In: <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yusim</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Korber</span><span class="given-names" tagx="given-names" title="given-names">BTM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brander</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barouch</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Boer</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haynes</span><span class="given-names" tagx="given-names" title="given-names">BF</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koup</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Moore</span><span class="given-names" tagx="given-names" title="given-names">JP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walker</span><span class="given-names" tagx="given-names" title="given-names">BD</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Watkins</span><span class="given-names" tagx="given-names" title="given-names">DI</span></span>, editors. <span class="source" tagx="source" title="source">HIV Molecular Immunology 2018</span>. <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Los Alamos, NM</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Los Alamos National Laboratory, Theoretical Biology and Biophysics</span>; <span class="year" tagx="year" title="year">2019</span> p. <span class="fpage" tagx="fpage" title="fpage">3</span>â€"<span class="lpage" tagx="lpage" title="lpage">27</span>.</span></li> <li tag="ref"><a name="ppat.1008607.ref006" /><span class="label" tagx="label" title="label">6</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Burrows</span><span class="given-names" tagx="given-names" title="given-names">SR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rossjohn</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">McCluskey</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>. <span class="mixed-article-title" title="mixed-article-title">Have we cut ourselves too short in mapping CTL epitopes?</span><span class="source" tagx="source" title="source">Trends Immunol</span>. <span class="year" tagx="year" title="year">2006</span>;<span class="volume" tagx="volume" title="volume">27</span>:<span class="fpage" tagx="fpage" title="fpage">11</span>â€"<span class="lpage" tagx="lpage" title="lpage">6</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16297661">16297661</a></span></span></li> <li tag="ref"><a name="ppat.1008607.ref007" /><span class="label" tagx="label" title="label">7</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yaciuk</span><span class="given-names" tagx="given-names" title="given-names">JC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Skaley</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bardet</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schafer</span><span class="given-names" tagx="given-names" title="given-names">F</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mojsilovic</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cate</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Direct interrogation of viral peptides presented by the class I HLA of HIV-infected T cells</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2014</span>;<span class="volume" tagx="volume" title="volume">88</span>(<span class="issue" tagx="issue" title="issue">22</span>):<span class="fpage" tagx="fpage" title="fpage">12992</span>â€"<span class="lpage" tagx="lpage" title="lpage">3004</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1128/JVI.01914-14">10.1128/JVI.01914-14</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25165114">25165114</a></span></span></li> <li tag="ref"><a name="ppat.1008607.ref008" /><span class="label" tagx="label" title="label">8</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Goulder</span><span class="given-names" tagx="given-names" title="given-names">PJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tang</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pelton</span><span class="given-names" tagx="given-names" title="given-names">SI</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walker</span><span class="given-names" tagx="given-names" title="given-names">BD</span></span>. <span class="mixed-article-title" title="mixed-article-title">HLA-B57-restricted cytotoxic T-lymphocyte activity in a single infected subject toward two optimal epitopes, one of which is entirely contained within the other</span>. <span class="source" tagx="source" title="source">J Virol</span>. <span class="year" tagx="year" title="year">2000</span>;<span class="volume" tagx="volume" title="volume">74</span>(<span class="issue" tagx="issue" title="issue">11</span>):<span class="fpage" tagx="fpage" title="fpage">5291</span>â€"<span class="lpage" tagx="lpage" title="lpage">9</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10799606">10799606</a></span></span></li> <li tag="ref"><a name="ppat.1008607.ref009" /><span class="label" tagx="label" title="label">9</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">de Campos-Lima</span><span class="given-names" tagx="given-names" title="given-names">PO</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gavioli</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Zhang</span><span class="given-names" tagx="given-names" title="given-names">QJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wallace</span><span class="given-names" tagx="given-names" title="given-names">LE</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dolcetti</span><span class="given-names" tagx="given-names" title="given-names">R</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rowe</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">HLA-A11 epitope loss isolates of Epstein-Barr virus from a highly A111 population</span>. <span class="source" tagx="source" title="source">Science</span>. <span class="year" tagx="year" title="year">1993</span>;<span class="volume" tagx="volume" title="volume">260</span>:<span class="fpage" tagx="fpage" title="fpage">98</span>â€"<span class="lpage" tagx="lpage" title="lpage">100</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7682013">7682013</a></span></span></li> <li tag="ref"><a name="ppat.1008607.ref010" /><span class="label" tagx="label" title="label">10</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">GutiÃ©rrez</span><span class="given-names" tagx="given-names" title="given-names">MI</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Spangler</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kingma</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Raffeld</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guerrero</span><span class="given-names" tagx="given-names" title="given-names">I</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Misad</span><span class="given-names" tagx="given-names" title="given-names">O</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Epstein-Barr virus in nasal lymphomas contains multiple ongoing mutations in the EBNA-1 gene</span>. <span class="source" tagx="source" title="source">Blood</span>. <span class="year" tagx="year" title="year">1998</span>;<span class="volume" tagx="volume" title="volume">92</span>(<span class="issue" tagx="issue" title="issue">2</span>):<span class="fpage" tagx="fpage" title="fpage">600</span>â€"<span class="lpage" tagx="lpage" title="lpage">6</span>.<span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9657761">9657761</a></span></span></li> <li tag="ref"><a name="ppat.1008607.ref011" /><span class="label" tagx="label" title="label">11</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sidney</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Peters</span><span class="given-names" tagx="given-names" title="given-names">B</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frahm</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brander</span><span class="given-names" tagx="given-names" title="given-names">C</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sette</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>. <span class="mixed-article-title" title="mixed-article-title">HLA class I supertypes: a revised and updated classification</span>. <span class="source" tagx="source" title="source">BMC Immunol</span>. <span class="year" tagx="year" title="year">2008</span>;<span class="volume" tagx="volume" title="volume">9</span>:<span class="fpage" tagx="fpage" title="fpage">1</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1471-2172-9-1">10.1186/1471-2172-9-1</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18211710">18211710</a></span></span></li> <li tag="ref"><a name="ppat.1008607.ref012" /><span class="label" tagx="label" title="label">12</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Frahm</span><span class="given-names" tagx="given-names" title="given-names">N</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yusim</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Suscovich</span><span class="given-names" tagx="given-names" title="given-names">TJ</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Adams</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sidney</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Hraber</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Extensive HLA class I allele promiscuity among viral CTL epitopes</span>. <span class="source" tagx="source" title="source">Eur J Immunol</span>. <span class="year" tagx="year" title="year">2007</span>;<span class="volume" tagx="volume" title="volume">37</span>(<span class="issue" tagx="issue" title="issue">9</span>):<span class="fpage" tagx="fpage" title="fpage">2419</span>â€"<span class="lpage" tagx="lpage" title="lpage">33</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17705138">17705138</a></span></span></li> <li tag="ref"><a name="ppat.1008607.ref013" /><span class="label" tagx="label" title="label">13</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Walters</span><span class="given-names" tagx="given-names" title="given-names">LC</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harlos</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Brackenridge</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rozbesky</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barrett</span><span class="given-names" tagx="given-names" title="given-names">JR</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jain</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Pathogen-derived HLA-E bound epitopes reveal broad primary anchor pocket tolerability and conformationally malleable peptide binding</span>. <span class="source" tagx="source" title="source">Nat Commun</span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">9</span>(<span class="issue" tagx="issue" title="issue">1</span>):<span class="fpage" tagx="fpage" title="fpage">3137</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/s41467-018-05459-z">10.1038/s41467-018-05459-z</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30087334">30087334</a></span></span></li> <li tag="ref"><a name="ppat.1008607.ref014" /><span class="label" tagx="label" title="label">14</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chen</span><span class="given-names" tagx="given-names" title="given-names">G</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wu</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Guo</span><span class="given-names" tagx="given-names" title="given-names">W</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cao</span><span class="given-names" tagx="given-names" title="given-names">Y</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Huang</span><span class="given-names" tagx="given-names" title="given-names">D</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wang</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Clinical and immunologic features in severe and moderate Coronavirus Disease 2019</span>. <span class="source" tagx="source" title="source">J Clin Invest</span>. <span class="year" tagx="year" title="year">2020</span>;<span class="volume" tagx="volume" title="volume">130</span>(<span class="issue" tagx="issue" title="issue">5</span>)<span class="fpage" tagx="fpage" title="fpage">2620</span>â€"<span class="lpage" tagx="lpage" title="lpage">2629</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1172/JCI137244">10.1172/JCI137244</a></span> Epub 2020 Mar 27. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/32217835">32217835</a></span></span></li> <li tag="ref"><a name="ppat.1008607.ref015" /><span class="label" tagx="label" title="label">15</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Friberg</span><span class="given-names" tagx="given-names" title="given-names">H</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Beaumier</span><span class="given-names" tagx="given-names" title="given-names">CM</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Park</span><span class="given-names" tagx="given-names" title="given-names">S</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pazoles</span><span class="given-names" tagx="given-names" title="given-names">P</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Endy</span><span class="given-names" tagx="given-names" title="given-names">TP</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Mathew</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Protective versus pathologic pre-exposure cytokine profiles in dengue virus infection</span>. <span class="source" tagx="source" title="source">PLoS Negl Trop Dis</span>. <span class="year" tagx="year" title="year">2018</span>;<span class="volume" tagx="volume" title="volume">12</span>(<span class="issue" tagx="issue" title="issue">12</span>):<span class="fpage" tagx="fpage" title="fpage">e0006975</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.pntd.0006975">10.1371/journal.pntd.0006975</a></span> eCollection 2018 Dec. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30557313">30557313</a></span></span></li> <li tag="ref"><a name="ppat.1008607.ref016" /><span class="label" tagx="label" title="label">16</span><span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ruiz-Riol</span><span class="given-names" tagx="given-names" title="given-names">M</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Llano</span><span class="given-names" tagx="given-names" title="given-names">A</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ibarrondo</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">ZamarreÃ±o</span><span class="given-names" tagx="given-names" title="given-names">J</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yusim</span><span class="given-names" tagx="given-names" title="given-names">K</span></span>, <span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bach</span><span class="given-names" tagx="given-names" title="given-names">V</span></span>, <span class="etal" title="etal"><i>et al.</i></span><span class="mixed-article-title" title="mixed-article-title">Alternative effector-function profiling identifies broad HIV-specific T-cell responses in highly HIV-exposed individuals who remain uninfected</span>. <span class="source" tagx="source" title="source">J Infect Dis</span>. <span class="year" tagx="year" title="year">2015</span>;<span class="volume" tagx="volume" title="volume">211</span>(<span class="issue" tagx="issue" title="issue">6</span>):<span class="fpage" tagx="fpage" title="fpage">936</span>â€"<span class="lpage" tagx="lpage" title="lpage">46</span>. <span class="pub-id"><a href="https://dx.doi.org/10.1093/infdis/jiu534">10.1093/infdis/jiu534</a></span> Epub 2014 Sep 23. <span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25249264">25249264</a></span></span></li> </ul> </div> </div>  </body></html>