<html xmlns="http://www.w3.org/1999/xhtml"> <head> <style type="text/css">
      	     		 
      	    a                          {background : #ffffff; }
      	    article                    {border-style : dotted; border-width : 2px; }
		 	
		 	div                        {background : #ffffcc;}
		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
		 	div.ack                    {border-style : solid ; border-color : red; margin : 2em; }
		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.boxed-text             {margin : 5em; border-style : solid;}
		 	div.contrib-group          {margin : 1em; }
		 	div.fig                    {border-style : solid; border-width : 2px; margin : 2em; }
		 	div.funding                {font-weight : bold ; font-size : 16pt;}
		 	div.given-names            {font-style : italic;}
		 	div.intro                  {border-style : inset; margin : 5px;}
		 	div.introduction           {border-style : inset; margin : 5px;}
		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin 1em;}
		 	div.journal-title-group    {background : #ffeeee;}
		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
		 	div.kwd                    {font-style : italic}
		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
		 	div.name                   {font-weight : bold;}
		 	div.alt-title              {font-style : italic; font-size : 12px;}
		 	
		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
		 	div.title                  {font-family : courier; font-weight : bold;
		 	                            font-size : 16pt; margin : 5px;}
		 	
		 	div.materials_methods      {border-style : double; margin : 5px; }
		 	div.methods                {border-style : double; margin : 5px; }
		 	
		 	div.results                {border-style : solid; margin : 5px;}
		 	div.background             {border-style : dotted; margin : 5px;}
		 	
		 	div.discussion             {border-style : groove; margin : 5px;}
		 	div.conclusion             {border-style : ridge; margin : 5px;}
		 	div.conclusions             {border-style : ridge; margin : 5px;}
		 	div.supplementary-material {border-style : inset; margin : 5px;}
		 	div.abbreviations          {border-style : double; border-color : red; margin : 5px;}
		 	div.competinginterests     {border-style : double; border-color : blue; margin : 5px;}
		 	div.acknowledgements       {border-style : double; border-color : green; margin : 5px;}
		 	div.authors_contributions   {border-style : double; border-color : purple; margin : 5px;}
		 	
		 	div.publisher              {border-style : outset; margin : 5px;}
		 	div.fn-type-conflict       {background : #f88; }
		 	div.fn-type-con            {background : #ddf; }
		 	div.fn-type-other          {background : #ddd; }
		 	
		 	div.unknown                {background : #ffd;
									 	  border-style : solid;
									 	  border-width : 1px;
									 	  padding : 2 px;}
		 	  
      	    table                      {background : #ffffdd;}
		 	tr                         {background : #ddddff; padding : 1px;}
		 	
		 	span                       {background : #ffcccc;}
		 	
		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
		 	span.collab                {background : #ddffff; }
		 	span.comment               {font-family : courier; font-size : 6px; background : #ffaaff;}
		 	span.contrib               {background : #ffffff;}
		 	span.corresp               {background : #ddffdd; }
		 	span.doi                   {background : #ffffff;}
		 	span.email                 {font-family : courier; }
		 	span.etal                  {font-style : italic;}
		 	span.fpage                 {font-family : courier;}
		 	span.given-names           {background : #ffffff;}
		 	span.iso-abbrev            {background : #ffffff;}
		 	span.issn-epub             {background : #ffffff;}
		 	span.issn-ppub             {background : #ffffff;}
		 	span.journal-title         {background : #ffffff;}
		 	span.lpage                 {font-family : courier;}
		 	span.mixed-article-title   {font-style : italic ;}
		 	span.nlm-ta                {background : #ffffff;}
		 	span.pmc                   {background : #ffffff;}
		 	span.pmcid                 {background : #ffffff;}
		 	span.pmid                  {background : #ffffff;}
		 	span.publisher             {background : #ffffff;}
		 	span.publisher-id          {background : #ffffff;}
		 	span.publisher-name        {background : #ffffff;}
		 	span.source                {background : #ffffff;}
		 	span.subject               {background : #ffffff;}
		 	span.surname               {background : #ffffff;}
		 	span.volume                {font-family : courier; font-weight : bold;}
		 	span.year                  {font-family : courier ; font-style : italic;}
			</style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"><span class="nlm-ta" title="nlm-ta">Emerging and Reemerging Viral Pathogens</span><div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"><span class="journal-title" tagx="journal-title" title="journal-title">Emerging and Reemerging Viral Pathogens</span></div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"><span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/7149643">7149643</a></span><span class="publisher-id" title="publisher-id">B978-0-12-819400-3.00004-1</span><span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1016/B978-0-12-819400-3.00004-1">10.1016/B978-0-12-819400-3.00004-1</a></span><div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span></div> <div class="title-group" tagx="title-group" title="title-group"> <div class="article-title" title="article-title">
Human Viruses: Emergence and Evolution</div> </div> <div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sudhan</span><span class="given-names" tagx="given-names" title="given-names">Shashi S.</span></span><a href="#aff1">1</a></span><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Sharma</span><span class="given-names" tagx="given-names" title="given-names">Preeti</span></span><a href="#aff2">2</a></span></div><span class="citation_author_institution" id="aff1">[1], </span><span class="citation_author_institution" id="aff2">[2], </span><div class="contrib-group" title="contrib-group"><span class="contrib" title="contrib"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ennaji</span><span class="given-names" tagx="given-names" title="given-names">Moulay Mustapha</span></span></span></div><span class="citation_author_institution" id="">[], </span><span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2019-9-9</span></span><span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2020</span></span><span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-9-9</span></span><span class="fpage" tagx="fpage" title="fpage">53</span><span class="lpage" tagx="lpage" title="lpage">68</span><div class="permissions"><span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Elsevier Inc.</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2020</span></span><span class="license" title="license"><span class="license-p" title="license-p">Since January 2020 Elsevier has created a COVID-19 resource centre with free information in English and Mandarin on the novel coronavirus COVID-19. The COVID-19 resource centre is hosted on Elsevier Connect, the company's public news and information website. Elsevier hereby grants permission to make all its COVID-19-related research that is available on the COVID-19 resource centre - including this research content - immediately available in PubMed Central and other publicly funded repositories, such as the WHO COVID database with rights for unrestricted research re-use and analyses in any form or by any means with acknowledgement of the original source. These permissions are granted for free by Elsevier for as long as the COVID-19 resource centre remains active.</span></span></div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Many new viruses have emerged in the last five decades. These newer genetically active agents have a major impact on the public health systems worldwide. Most emerging infections appear to be caused by pathogens already present in the environment, brought out of obscurity or given a selective advantage by changing conditions and afforded an opportunity to infect new host populations. Also on rare occasions, a new variant may evolve and cause a new disease. Altered virus transmission because of deforestation and environment change, ecological changes and agricultural development, commerce, technology, microbial adaptation and change, breakdown of public health measures, and deficiencies in public health infrastructure are the reasons for emergence and reemergence of infectious diseases in general and viral infections in particular.</p> <p>Specific factors precipitating disease emergence can be identified in virtually all cases. Moreover, these factors are increasing in prevalence. This increase, together with the ongoing evolution of viral and microbial variants and selection for drug resistance, suggests that infections will continue to emerge and probably increase and emphasize the urgent need for effective surveillance and control. These viruses are rich source of emerging diseases due to the introduction of infections from other species in the zoonotic pool. In the near future, there is an urgent need to monitor collaboration between humanâ€"animal interface so that public health risks can be understood. A number of activities increase microbial traffic from animals to humans or disseminate microbes from isolated groups into new populations and as a result promote emergence and epidemics. In some cases, including many of the most novel infections, the agents are zoonotic crossing from their natural hosts into the human population because of many similarities. Vector-borne diseases also have a natural advantage of dissemination.</p> </div> <div class="kwd-group"> <div class="title" tagx="title" title="title">
Keywords</div> <div class="kwd" title="kwd">
Emerging infections</div> <div class="kwd" title="kwd">
novel viruses</div> <div class="kwd" title="kwd">
viral outbreaks</div> <div class="kwd" title="kwd">
virus evolution</div> <div class="kwd" title="kwd">
public health</div> <div class="kwd" title="kwd">
risk factors</div> <div class="kwd" title="kwd">
surveillance</div> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p id="p0010"> Following on from the discovery of Tobacco Mosaic Virus (TMV) in 1892 and Foot-and-Mouth disease virus in 1898, the first â€œfilterable agentâ€� to be discovered in humans was Yellow Fever (YF) virus in 1901 (<a href="#bib22">Woolhouse et al., 2001</a>). Landmark discoveries later were Rinderpest, Vaccinia and Rabies (1902â€"06), Avian leukosis and Poliomyelitis (1908â€"11), Bacteriophages (1915â€"17), and Spanish flu (1918â€"22; due to which 50 million people died). More than 65 viruses were described in the years 1920â€"27 (<a href="#bib15">Li et al., 2009</a>). Thereafter cultures of human Herpes Simplex Virus (HSV) and then YF virus were done, and first attenuated vaccine for YF was prepared (1929â€"31). First chorioallantoic membrane inoculation was done with fowlpox virus in 1931. Subsequently, nucleoprotein nature of viruses (1934â€"36), crystallization of TMV (1935â€"37), culture of polio viruses (1949), and DNA as genetic material of phages (1950â€"63) by Hershey Chase experiment were achieved and killed and live polio vaccines were prepared (<a href="#bib5">Chua et al., 2007</a>). Reoviruses were discovered (1953â€"64), single stranded Ribonucleic acid (ssRNA) of TMV was translated into viral proteins (1962), and in vitro synthesis of both single stranded deoxyribo nucleicacid (ssDNA) and ssRNA bacteriophage genome was successful (1967). Later in 1979â€"80, discovery and characterization of infectious naked RNA viroids were shown to be circular ssRNA, complete genomic sequence of hepatitis, cauliflower mosaic virus (CMV) (1982) and complete genomic sequence of TMV (1986â€"89) was achieved. Historically, major developments in virological technology mark filtration in 1890, complement fixation test in 1929, tissue culture in 1948, monoclonal antibodies in the 1970s, polymerase chain reaction (PCR) in 1985 and high-throughput sequencing in the 2000s (<a href="#bib2">Antia et al., 2003</a>).</p> <p id="p0015">New species of human viruses are still being identified, at a rate of three or four per year. Viruses make up about two-thirds of all new human pathogens. These new viruses differ widely in their importance, ranging from the rare and mild illness due to Menangle virus to the devastating public health impact of Human immunodeficiency virus 1 (HIV-1). On the contrary, a highly significant overrepresentation has been given by most human pathogen species of bacteria, fungi, or helminthes. Actually, it was the discovery of the HIV in the early 1980s that initiated a worldwide awareness and research interest in emerging novel viral pathogens (<a href="#bib1">Anderson and Tong, 2010</a>). Over the past several decades, new outbreaks of infections have led to the discovery of a diverse array of highly pathogenic viruses mainly those belonging to the Filoviridae, Arenaviridae, Bunyaviridae, Paramyxoviridae, Coronaviridae, Flaviviridae, Togaviridae, and Hepeviridae families. Some examples include BK virus, JC virus, Merkel cell polyomavirus (MCV or MCPyV), severe fever with thrombocytopenia syndrome virus (SFTSV), hantavirus (HTNV), and Sin Nombre virus (<a href="#bib3">Bebber et al., 2007</a>). Following the fatal cases of Lujo virus in southern Africa in 2008 (<a href="#bib4">Briese et al., 2009</a>), another arena virus, the Lassa virus, was first reported in Nigeria in 1969, reemerged in Guinea, Liberia, and Mali in 2009, Ghana in 2011, and Benin in 2014. Human metapneumovirus was first identified in the Netherlands in 2001 and was subsequently linked to an acute lower respiratory tract infection in children, similar to respiratory syncytial virus (RSV). Recently, in 2013, a novel avian influenza A strain (H7N9) of â€œbird fluâ€� in China (<a href="#bib19">Wolfe et al., 2004</a>) and the Middle East respiratory syndrome related Coronavirus (MERS)-CoV have been identified (<a href="#bib8">Corman et al., 2012</a>). Of note, while 2015 was threatened by the reemergence of the Ebola virus, 2015/2016 has been challenged with the resurgence of the Zika virus (ZIKV). Despite substantial advancements in the understanding of the biology of pathogens, the breakthrough in prevention, their effects on public health and the global economy, and the emergence of novel pandemic viruses remains an enduring puzzle. The intricate â€œhostâ€"pathogenâ€"environmentâ€� relationship remains the key to understanding the emergence/reemergence of pathogenic versus ecological approach to study the diversity of human viruses (defined as viruses for which there is evidence of natural infection of humans) (<a href="#bib11">Jones et al., 2008</a>). First, the temporal, geographical, and taxonomic patterns in the discovery of human viruses need to be described and co related. Second, the processes by which new human viruses emerge need to be understood. There are a number of definitions of â€œemergence.â€� All the stages of the process by which a virus shifts from not infecting humans at all to becoming a major human pathogen need to be understood. As experiences with HIV-1 and new variants of influenza A (and also with novel animal pathogens such as canine parvovirus) show, this shift can occur rapidly, over time scales of decades, years, or even months. Not all newly identified human virus species are â€œnewâ€� in the sense that they have only recently started to infect humans; many of them have been present in humans for a considerable time but have only recently been recognized (<a href="#bib7">Cleaveland et al., 2007</a>). Moreover, the â€œspeciesâ€� itself is an imprecise designation, especially in influenza-like viruses where different serotypes can have different epidemiologies and health impacts. Indeed, the demarcation between genus, species complex, species, and serotype (or other designations of subspecific variation) can be somewhat arbitrary (<a href="#bib6">Cleaveland et al., 2001</a>). Nonetheless, a study of currently recognized â€œspeciesâ€� is a natural starting point for attempts to characterize and interpret patterns of virus diversity. A comprehensive knowledge of important emerging/reemerging viral infections worldwide with their possible origins, evolution, natural reservoirs, human adaptations, and risk factors needs to be developed for individual novel agents for a better insight of the biologic and evolutionary mechanisms (<a href="#bib12">Keeling and Rohani, 2008</a>) (<a href="#f0010">Fig. 4.1</a> ). </p><div class="fig" title="fig"> <div class="figure" title="figure"><a href="http://doi.org/"><span class="label" tagx="label" title="label">Figure 4.1</span></a></div>   <p>A depiction of important emerging/reemerging viral infections and their possible origins, evolutionary drivers, natural reservoirs, and risk factors. <i>CHIKV</i>, Chikungunya virus; <i>DENV</i>, dengue virus; <i>HBV</i>, Hepatitis B virus; <i>HEV</i>, hepatitis E virus; <i>HIV</i>, human immunodeficiency virus; <i>TTV</i>, Torque teno virus; <i>WNV</i>, West Nile virus; <i>ZIKV</i>, Zika virus.</p>   </div> <p /> </div> <div class="origin" title="sec"> <div class="title" tagx="title" title="title">
Origin</div> <p id="p0020">Of the known viruses that infect humans, about 80% perpetuate naturally in nonhuman â€œreservoirs,â€� largely farm mammals and poultry and, to a lesser extent, in wild animals and arthropods. It is estimated that zoonotic infectious agents constitute about 60% of the known human pathogens and up to 75% of â€œemergingâ€� human pathogens. The knowledge of such zoonosis and the diversity of these viruses in their known reservoirs are limited. Also the data on some of the domestic mammals harboring dozens of virus species is limited, and there is insufficient knowledge about the wild animals that are estimated to host thousands of virus species (<a href="#bib13">Keusch et al., 2009</a>). Emerging human viruses, such as novel influenza strains, human (h)CoV, Hendra virus, Nipah virus, and several others, are all linked to zoonosis. Recently, the deadly outbreak of MERS-CoV was linked to its zoonotic origin because of its close genetic homology to bat CoV, but not to any other known hCoV (<a href="#bib14">Kilpatrick and Randolph, 2012</a>). Current data shows that bats harbor the greatest diversity of CoVs, which vary from species to species and region to region. The Lyssaviruses of Rhabdoviridae are zoonotic human pathogens that cause fatal encephalitic disease. In addition, European bat lyssavirus type-1/-2, Bokeloh bat lyssavirus (<a href="#bib20">Wolfe et al., 2007</a>), and Australian bat lyssaviruses have been implicated in human fatalities. Rabies virus, the Archetypal lyssavirus, is historically one of the most dreaded viruses, with zoonotic potential in dogs, cats, and ferrets, and includes other domestic and wild mammals. Globally, several mammalian species, including the deer mouse, rice rat, and cotton rat, are recognized as potential reservoir of HTNV. In China, the HTNV and Seoul virus are zoonotically linked to the striped field mouse and the Norway rat, respectively. Also in China, the Fugong virus, a novel HTNV, has recently been identified in small oriental voles. HTNVs have also been detected in shrews and bats, but their link to human illness remains to be established. Moreover, in addition to humans and pigs, there is a growing chain of mammalian hosts for the hepatitis E virus (HEV) that includes deer, boar, mongoose, rabbit, rat, goat, camel, bat, ferret, moose, etc. Likewise, the highly prevalent Torque teno virus, of the newly established virus family, Anelloviridae, also infects pigs, cows, sheep, cats, dogs, and chickens. Arboviruses, such as dengue virus, chikungunya virus (CHIKV), ZIKV, and West Nile virus (WNV), are the arthropod-borne viruses that have reemerged in many tropical and subtropical regions in the last two decades (<a href="#bib17">Taylor et al., 2001</a>). Notably, ZIKV was known as a neglected tropical disease confined to Africa and Asia until an outbreak was reported in 2007 on Yap Island and in 2013/2014 on islands in the Pacific, thus expanding its geographical territory. The WNV remains the most important mosquito-borne encephalitis pathogen in North America, involving <i>Culex</i> sp. and the American Robin in its transmission cycle. Since its emergence in the West in 1999, it has undergone adaptive genetic changes as it spreads throughout North America (<a href="#bib21">Woolhouse and Antia 2008</a>). Furthermore, Crimeanâ€"Congo hemorrhagic fever virus (CCHFV) is considered to be one of the major emerging diseases spreading to and within the European nations, following the expanding distribution of anthropophilic ticks. Every year, &amp;gt;1000 cases of CCHFV, due to human-to-human transmission, are reported in southeastern Europe, including Turkey. SFTSV, a previously unidentified tick-borne bunyavirus, has recently emerged in China with fatality rates as high as 30% (<a href="#bib1">Anderson and Tong, 2010</a>) (<a href="#t0010">Table 4.1</a> ). <div class="table-wrap_UNKNOWN" position="float" id="t0010"><span class="label" tagx="label" title="label">Table 4.1</span> </div></p><p>Examples of Putative New Human Virus Species Reported From 2005 to 2009</p>  <table frame="hsides" rules="groups"> <thead> <tr> <th valign="bottom">Virus</th> <th valign="bottom">Family</th> </tr> </thead> <tbody> <tr> <td>Human bocavirus</td> <td>Parvoviridae</td> </tr> <tr> <td>Parvovirus 4</td> <td>Parvoviridae</td> </tr> <tr> <td>KI polyomavirus</td> <td>Polyomaviridae</td> </tr> <tr> <td>Melaka virus</td> <td>Reoviridae</td> </tr> <tr> <td>WU polyomavirus</td> <td>Polyomaviridae</td> </tr> <tr> <td>Astrovirus MLB1</td> <td>Astroviridae</td> </tr> <tr> <td>Bundibugyo ebolavirus</td> <td>Filoviridae</td> </tr> <tr> <td>Human bocavirus 2</td> <td>Parvoviridae</td> </tr> <tr> <td>Human cosaviruses Aâ€"D</td> <td>Picornaviridae</td> </tr> <tr> <td>Human cosavirus E1</td> <td>Picornaviridae</td> </tr> <tr> <td>Astrovirus VA1</td> <td>Astroviridae</td> </tr> <tr> <td>Human papilloma virus 116</td> <td>Papillomaviridae</td> </tr> <tr> <td>Klassevirus</td> <td>Picornaviridae</td> </tr> <tr> <td>Lujo virus</td> <td>Arenaviridae</td> </tr> </tbody> </table>  <p /> </div> <div class="evolutionandadaptationtothehumanhost" title="sec"> <div class="title" tagx="title" title="title">
Evolution and Adaptation to the Human Host</div> <p id="p0025">While DNA viruses are believed to have been evolving and diversifying for millions of years, most RNA viruses are likely to have a much more recent evolution and â€œhuman-adaptationâ€� for only thousands of years. Owing to their error-prone polymerase/reverse-transcriptase (approx. 10<sup>âˆ’4</sup>/site/replication cycle) that seems to operate at optimal fidelity, RNA viruses exist as more genetically diversified populations than DNA viruses. Nevertheless, of the hitherto recognized 158 species of human RNA viruses consisting of 47 genera and 17 families, only a minority has adapted to humans (<a href="#bib16">Simmonds, 2001</a>). In contrast, of the known 91 DNA virus species with 22 genera and 8 families, nearly 87% are adapted to human hosts. In the human-adaptation process, the viral genetic mutations, reassortment, or virusâ€"host genetic recombination might lead to the establishment of stable virus lineages in human populations. So, there is the possibility that such human-adapted viruses could circulate in the body asymptomatically and remain undiagnosed till their new clinical manifestations are reported. To understand this clearly, a currently isolated HEV genotype 3 from a chronic hepatitis E patient containing a recombinant virusâ€"host RNA genome was shown to infect cultured human, pig and deer hepatocytes (<a href="#bib19">Wolfe et al., 2004</a>, <a href="#bib20">Wolfe et al., 2007</a>). It is assumed that there is a pool of human virus species still to be discovered. The composition of such a viral pool is dynamic, changing over time, that is, while some virus species tend to become extinct and others continue to evolve in their natural hosts. More commonly, new species arise as a result of jumps from one host to another, thus crossing the species barrier. Humans are, therefore, no more than â€œincidentalâ€� or â€œspilloverâ€� hosts for pathogens. However, only a minority of such viruses are capable of persisting in certain human populations (endemics) or spreading across populations (epidemics) in the absence of a reservoir. Differential host factors, such as age, health, physiology, nutritional status, exposure history, simultaneous infection with &amp;gt;1 pathogen, immunocompetence and genetics, are determinants to human susceptibility to an infection. The field of phylodynamics, combining a modeling framework for host, epidemiological, and molecular data, especially for RNA viruses, shows particular promise for understanding the patterns of viral evolution during epidemics (<a href="#bib9">Gaynor et al., 2007</a>). Moreover, the underappreciated aspect of growing human populations, global changes in land usage and the introduction of anthropophilic vectors create selective pressure on hosts and reservoirs. For example, both WNV and CHIKV evolved rapidly after being introduced to new locations and encountering new vectors. When <i>Aedes albopictus</i> rather than <i>A. aegypti</i> became the main vector of CHIKV in the Indian subcontinent after the 2004â€"09 epidemic, the same viral strain spread rapidly, and the subsequent mutants seemed to circulate and persist more efficiently. A unique molecular signature of the CHIKV epidemic was a single amino acid substitution (A226V) in the envelope protein (E1). This mutation was identified in 90% of the strains reported during the later phase and was associated with a high epidemic potential for CHIKV (<a href="#bib14">Kilpatrick and Randolph, 2012</a>). ZIKV, initially known to be transmitted by mosquitoes, was recently reported as being transmitted, albeit more rarely, via sexual contact, saliva, breast milk, blood transfusions and from mother to child (<a href="#bib6">Cleaveland et al., 2001</a>).</p> </div> <div class="evolutionarydrivers" title="sec"> <div class="title" tagx="title" title="title">
Evolutionary Drivers</div> <p id="p0030">The key to understanding the emergence/reemergence of novel viruses is to know the intricate â€œhostâ€"pathogenâ€"environmentâ€� relationship in the evolution of pathogens. This emergence of infectious diseases in naive regions is caused due to the movement of pathogens by travel and trade, while local emergence is driven by a combination of environmental and social and environmental change. It is noted that virus transmission rates are higher in dense than in sparse populations and the spread is often enhanced by air travel or migration. Pathogens introduced into novel regions often cause explosive epidemics, followed by a declining incidence, whereas those that emerge locally due to land usage or social changes usually show consistent increases (<a href="#bib2">Antia et al., 2003</a>). A recent example is the emergence of ZIKV in Brazil in 2015. Phylogenetic studies suggested that ZIKV from the Pacific islands outbreak in 2013/2014 was probably introduced into Brazil during the FIFA World Cup or the 2014 FIA World Endurance Championship auto racing series. The dispersal of the virus in Brazil resulted in an explosive epidemic of Zika fever, and the infection spread to other countries due to frequent travel. While most human infections are known to have zoonotic origins, it is certain that alterations in the environment, due to industrialization and urbanization, are an important but completely neglected factor. Though the origins of most of the human viruses are not known so far, the great majority can be categorized as â€œcrowd diseasesâ€� that require a relatively high host density to persist (<a href="#bib13">Keusch et al., 2009</a>).</p> <p id="p0035">Notably, the recent outbreaks of H1N1, hCoV, Hendra virus, Nipah virus, and MERS-CoV suggest that the Asia-Pacific region is the global hot spot for the emergence of novel RNA viruses. In this case an order-of-magnitude estimation of 1 such event per 100 years is broadly consistent with human demographic history. Notably, RNA viruses are known to incorporate drastic mutations in their genome, an example being the large duplication events in the G protein gene of Respiratory Syncytial Virus (RSV). Two such remarkable events were the 60-bp duplication in group B RSV in Argentina in 1999 and the 72-bp duplication in group A RSV in Canada in 2011. These new genotypes of RSV with their duplications, known as BA and ON1, respectively, spread to different geographical regions across the globe due to immunologically naive travelers. The mathematics of these spreading events is well known today and a sophisticated array of computational and mathematical models can be used to accurately back-predict such events. An example of this is the first case clusters of the severe acute respiratory syndrome (SARS) outbreak and the subsequent global spread, including the country-by-country distribution of human cases (<a href="#bib22">Woolhouse et al., 2001</a>). Recent investigations have evaluated the transmission dynamics of ZIKV infection using mathematical models. We may not ignore that wild animals constitute an important but poorly understood reservoir for known and undiscovered human pathogens, including viruses. Furthermore, the relative importance of an animal species as a source of human infection is a function of the prevalence of zoonotic agents in that species and the probability of close contact (direct or indirect) with susceptible humans. Clearly, these factors vary geographically, and changes in the patterns of human and animal disease will continue to result from socioeconomic and ecological changes at the human-to-animal interface.</p> </div> <div class="emergenceasabiologicalprocess" title="sec"> <div class="title" tagx="title" title="title">
Emergence as a Biological Process</div> <div class="nonhumanreservoirs" title="sec"> <div class="title" tagx="title" title="title">
Nonhuman Reservoirs</div> <p id="p0040">More than two-thirds of human virus species are zoonotic, that is, they are capable of infecting vertebrate hosts other than Homo sapiens. By far the most important nonhuman host taxa are other mammals, with rodents and ungulates most commonly identified as alternative hosts, followed by primates, carnivores, and bats. A minority of the zoonotic viruses (less than 20%) is also known to infect birds; very few have been reported from vertebrates other than mammals or birds (<a href="#bib2">Antia et al., 2003</a>).</p> <p id="p0045">The remaining viruses only naturally infect humans (these are sometimes referred to as â€œspecialistâ€� human pathogens). Some of these (e.g., hepatitis B) may have coevolved with humans over very long time periods; others (e.g., HIV-1) have much more recent origins. Some of both kinds are believed to have originated in other mammal or bird species, including HIV-1 (derived from a simian immunodeficiency virus found in chimpanzees); HIV-2 (sooty mangabeys); SARS virus (horseshoe bats); hepatitis B, human T-lymphotropic virus-1 and -2, dengue, and yellow fever (all primates); human coronavirus OC43, measles, mumps, and smallpox (all livestock); and influenza A (wildfowl). However, the origins of the majority of specialist human viruses is still not known and a gap in knowledge has prompted calls for an â€œorigins initiativeâ€� (<a href="#bib16">Simmonds, 2001</a>). </p> </div> <div class="pathogenpyramid" title="sec"> <div class="title" tagx="title" title="title">
Pathogen Pyramid</div> <p id="p0050">A useful conceptual framework for thinking about the emergence of novel viruses is the pathogen pyramid. The pyramid has four levels.</p> <p id="p0055">Each level represents a different degree of interaction between pathogens and humans, ranging from exposure through to epidemic spread. Some pathogens are able to progress from one level to the next:</p> <p id="p0060">Level 1 represents the exposure of humans to a novel pathogen, such as a virus. The source of viruses of interest is most likely to be other mammals or birds and â€œexposureâ€� implies any route by which a particular viral infection might be acquired, whether by contact with blood, saliva, or feces; contamination of food and water; or via an arthropod vector. The rate of such exposure is determined by a combination of the distribution and ecology of the nonhuman host and human activities. It is likely that exposure to nonhuman viruses occurs commonly: a process referred to as â€œchatter.â€�</p> <p id="p0065">Level 2 represents the subset of viruses that are capable of infecting humansâ€"that is, overcoming the â€œspecies barrier.â€� This is likely to reflect both the molecular biology of the virus (for example, is it capable of entering and replicating in human cells?) and the physiology of the exposed human (especially immunocompetence).</p> <p id="p0070">Level 3 represents the subset of viruses that cannot only infect humans but can also be transmitted from one human to another (by whatever route, including via arthropod vectors). Again, this will mainly reflect the hostâ€"pathogen interaction, especially whether it is possible for the virus to access tissues from which it can exit the host, such as the upper respiratory tract, lower gut, urogenital tract, skin, or blood (for some transmission routes).</p> <p id="p0075">Level 4 represents the subset of viruses that are sufficiently transmissible between humans to cause major outbreaks and/or become endemic in human populations without the requirement of a nonhuman reservoir. This equates with the epidemiological condition <i>R</i><sub><i>O</i></sub>&amp;gt;1, that is, a single primary case generates, on average, more than one secondary case. This is a function of both the transmissibility of the virus (how infectious an infected host is and for how long) and properties of the human population (how human demography and behavior affect opportunities for transmission). From previous reviews of the literature, it is possible to put approximate numbers of virus species at each level of the pyramid. There are greater than 200 viruses at least at level 2. It has been reported that of the virus species known to infect domestic animals (livestock and companion animals)â€"to which humans are presumably routinely exposedâ€"roughly one-third are also capable of infecting humans. The species barrier exists: but it is clearly very leaky. Based on data from roughly 50% of the viruses that can infect humans can also be transmitted by humans (level 3) and roughly 50% of those are sufficiently transmissible that <i>R</i><sub><i>O</i></sub> may exceed one (level 4). That a significant minority of viruses (mammalian or avian) should be capable of extensive spread within human populations [or of rapidly becoming so is consistent with experience: there are several examples within the past hundred years alone (HIV-1, SARS, plus variants of influenza A) and many more in the past few millennia (e.g., measles, mumps, rubella, smallpox)]. It is noteworthy that the â€œshapeâ€� of the pathogen pyramid for viruses is very different to that for other kinds of pathogen (bacteria, fungi, protozoa, or helminths), of which much smaller fractions are capable of extensive spread in human populations (<a href="#bib18">Towner et al., 2008</a>). The most straightforward explanation for this is the much more rapid evolution of viruses (especially RNA viruses), allowing them to adapt to a new (human) host much more quickly than other kinds of pathogen.</p> </div> <div class="driversofemergence" title="sec"> <div class="title" tagx="title" title="title">
Drivers of Emergence</div> <p id="p0080">â€œDriversâ€� of the emergence of novel viruses or other pathogens have been studied recently. These constitute a diverse set of environmental and biological factors, many of whichâ€"such as â€œurbanizationâ€� or â€œland useâ€�â€"seem intuitively reasonable but are too broad to relate to mechanistic causes of emergence. Moreover, identification of drivers is usually a subjective exercise: there are very few formal tests of the idea that a specific driver is associated with the emergence of a specific pathogen or set of pathogens. In many cases, this would be a challenging exercise: many drivers have only indirect effects on emergence (e.g., climate change, which is often linked with changing distributions of disease vectors); and often an emergence event has multiple causes (good examples would be the emergence of Nipah virus or SARS coronavirus). Other ideas about the drivers of emergence are even harder to test formally. One such is that we are currently living through a â€œperfect stormâ€� in which many potential drivers of emergence events (such as population growth, urbanization, global travel and trade, and intensification of livestock production) are acting in concert. Upward trends in many drivers can be quantified, but it is not entirely clear that the frequency of emergence events is increasing: one recent study suggested that it increased during the first decade of the HIV/AIDS pandemic but has been decreased thereafter. A slightly different way of thinking about the drivers of emergence is to draw an analogy between emerging pathogens and weeds. The idea here is that there is a sufficient diversity of pathogens availableâ€"each with their own biology and epidemiologyâ€"that any change in the human environment (but especially in the way that humans interact with other animals, domestic or wild) is likely to favor one pathogen or another, which responds by invading the newly accessible habitat. This implies that emerging pathogens possess different life-history characteristic to established, long-term endemic pathogens (<a href="#bib14">Kilpatrick and Randolph, 2012</a>).</p> </div> <div class="speciesjumps" title="sec"> <div class="title" tagx="title" title="title">
Species Jumps</div> <p id="p0085">For viruses, one of the key steps in the emergence process is the jump between one host species and humans. (For other kinds of pathogens, there may be other sources of human exposure, notably environmental sources or the normally commensal skin or gut flora.) Various factors have been examined in terms of their relationship with a pathogenâ€™s ability to jump into a new host species; these include taxonomic relatedness of the hosts, geographical overlap, and host range. The roles of both host relatedness and geographical proximity have been clearly established. Experimentally associations between the degree of cross-species transmission of bat lyssaviruses and both the geographical overlap between bat populations across the United States and the phylogenetically relatedness of the bat species involved have been established. Primate species tended to share more parasitic species if they were both more closely related and had sympatric distributions. A broad host range is also associated with the likelihood of a pathogen emerging or reemerging in human populations. An illustrative case study is bovine spongiform encephalopathy (BSE). After BSEâ€™s emergence in the 1980s, well before it was found to infect humans as Variant Creutzfeldt-Jakob disease (vCJD), it rapidly became apparent that it could infect a wide range of hosts, including carnivores. This was in marked contrast to a much more familiar prion disease, scrapie, which was naturally restricted to sheep and goats. With hindsight, this observation might have led to public health concerns about BSE being raised earlier than they were. Host range is a highly variable trait among viruses: some, such as rabies, can infect a very wide range of mammals; others, such as mumps, specialize on a single species (humans). Moreover, for pathogens generally, host range seems to be phylogenetically labile, with even closely related species having very different host ranges. Hence, the biological basis of host range is relevant to understanding pathogen emergence (<a href="#bib11">Jones et al., 2008</a>).</p> </div> <div class="cellreceptorusageandhostrange" title="sec"> <div class="title" tagx="title" title="title">
Cell Receptor Usage and Host Range</div> <p id="p0090">One likely biological determinant of the ability of a virus to jump between species is whether or not they use a cell receptor that is highly conserved across different (mammalian) hosts. The viruses that use conserved receptors ought to be more likely to have a broad host range. A comprehensive review identified that, for 88 human virus species, only one cell receptor was identified. Although this was only 40% of the species of interest, 21 (out of 23) families were illustrated; this set constitutes a good cross section of corresponding taxonomic diversity. Of these 88 species, 22 use nonprotein receptors (e.g., heparin sulfate) and, of the remainder, 2 of the proteins were not entered in the UniProt database (making it impossible to determine whether the protein was â€œconservedâ€� or not), leaving 64 species from 16 families. On the basis of a published studies of virus host ranges, these viruses have been accorded either a â€œnarrowâ€� host range (if the only nonhuman hosts they were known to infect were other primates) or a â€œbroadâ€� host range (if they were known to infect also other kinds of mammals or birds). Using the UniProt database it can be determined whether the cell receptor protein was â€œconservedâ€� by quantifying the amino acid sequence homology between humans and mice. (For the subset of proteins where amino acid sequences data were also available for cows, pigs, or dogs, very similar patterns were found) The most prominent characteristic of the plot is that there are no instances of human viruses with wide host ranges that do not use highly conserved cell receptors (i.e., more than 90% amino acid sequence homology). Statistical analysis needs correction for phylogenetic correlation; viruses of the common family are both more likely to use the same cell receptor. This can be crudely (but conservatively) allowed by testing for an association between host range and receptor homology at the family; not species, level. This gives a statistically significant result (<i>P</i>=.015). Notably, the use of a conserved receptor is a necessary but not sufficient condition for a virus to have a broad host range encompassing different mammalian orders. It follows that a useful piece of knowledge about a novel mammalian virus, helping to predict whether or not it poses a risk to humans, would be to identify the cell receptor it uses. However, this may not always be practicable: at present, the cell receptor used by over half the viruses that infect humans is not known, and this fraction is considerably smaller for those that infect other mammals. The emergence of new human viruses is a long-standing and ongoing biological process. The first line of defense against emerging viruses is effective surveillance. There are three important key points in surveillance (<a href="#bib10">Greninger et al., 2009</a>). First, emerging viruses are everyoneâ€™s problem: the ease with which viruses can disperse, potentially worldwide within days, coupled with the very wide geographical distribution of emergence events, means that a coordinated, global surveillance network is essential if we are to ensure rapid detection of novel viruses. This immediately highlights the enormous national and regional differences in detection capacity, with the vast majority of suitable facilities located in Europe or North America. Second, reporting of unusual disease events is patchy, even once detected, reflecting both governance issues and lack of incentives. Third, we need to consider extending the surveillance effort to other mammal populations as well as humans, because these are the most likely source of new human viruses. Improving the situation will require both political will and considerable investment in infrastructure, human capacity and new tools. However, the benefits are potentially enormous (<a href="#bib9">Gaynor et al., 2007</a>). It is possible to forestall an emerging disease event, as experience with SARS has shown. However, our ability to achieve this is closely linked to our ability to detect such an event, and deliver effective interventions, as rapidly as possible. A better understanding of the emergence of new human viruses as a biological and ecological process will allow us to refine our currently very crude notions of the kinds of pathogens, or the kinds of circumstances, we should be most concerned about, and so direct our efforts at detection and prevention more efficiently.</p> </div> </div> <div class="conclusionsandfutureprospects" title="sec"> <div class="title" tagx="title" title="title">
Conclusions and Future Prospects</div> <p id="p0095">The frequent emergence or reemergence of deadly viral infections has significantly affected human health despite extraordinary progress in biomedical knowledge. High population density, rampant constructions, poor sanitation, changing climate, and the introduction of anthropophilic vectors create selective pressure on hosts and pathogen reservoirs. Although we know that a substantial fraction of well-identified mammalian viruses is responsible for human etiology, there are large numbers of evolving viruses waiting to infect and adapt. The unpredictable nature of novel infections, the rare occasions of outbreaks, the small numbers of confirmed cases, the asymptomatic occurrence, and occurrence in remote areas severely hamper the assessment of control and preventive modalities. Surveillance of the emerging human viruses must include livestock, wild animals, and potential arthropod vectors as well as their environments at an international level of coordination. Furthermore, individuals, such as livestock herders, zookeepers, hunters, rangers, and veterinarians working with reservoir or high-risk animals, must take hygienic and general universal containment measures.</p> <p id="p0100">Efficient vaccines are already available against important human viruses, such as HBV and human papillomavirus. Even though drug resistance in HBV and HIV has accelerated alarmingly, new generations of broad-spectrum antiviral agents as combination regimens hold promise. Moreover, specific geographical regions or interfaces between humans, livestock, wildlife, and the environment should be the targets for intense surveillance. In this way, further extensive research will surely enhance our understanding and capacity for predicting new pandemics and preparing control measures in advance. Furthermore, viral pathogen discovery is of critical importance to clinical microbiology, infectious diseases, and public health. Genomic approaches for pathogen discovery, including consensus PCR, microarrays, and unbiased next-generation sequencing (NGS), have the capacity to comprehensively identify novel microbes present in clinical samples. Although numerous challenges remain to be addressed, including the bioinformatics analysis and interpretation of large datasets, these technologies have been successful in rapidly identifying emerging outbreak threats, screening vaccines and other biological products for microbial contamination, and discovering novel viruses associated with both acute and chronic illnesses. Downstream studies such as genome assembly, epidemiologic screening, and cultures system or animal model of infection are necessary to establish an association of a candidate pathogen with disease. Although sometimes derided as a â€œfishing expeditionâ€� or a needle in haystack approach, pathogen discovery is, in actuality, a highly worthwhile scientific endeavor. Without a cause identified for many presumed infectious diseases, it is not possible to conduct downstream investigations in pathogenesis and hostâ€"microbial interactions nor is it possible to design effective vaccines or antimicrobial drugs to combat the associated illness. Potential applications of pathogen discovery range from outbreak investigation of emerging pathogens, to screening of blood products, vaccines, and other biologics for viral contaminants, to clinical diagnosis of unknown acute or chronic infectious diseases. The current availability of state-of-the-art genomic technologies such as pan-microbial microarrays and NGS provides an unprecedented opportunity to â€œcast a wide netâ€� and surveys the full breadth of as-yet undiscovered pathogens in nature that pose significant threats to human health.</p> </div> <div class="acknowledgments" title="sec"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p id="p0105">We acknowledge the total support provided by the Resource Centre National Institute of Virology, Pune, India, under the Indian Council of Medical Research, Directorate of Health Research and VIral Research and Diagnostic Laboratory (VRDL) India, in establishing VRDL at Department of Microbiology, Govt. Medical College, Jammu in Jammu and Kashmir, India.</p> </div> </div> <div class="back" title="back"> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"><a name="bib1" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Anderson</span><span class="given-names" tagx="given-names" title="given-names">L.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tong</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Update on SARS research and other possibly zoonotic coronaviruses</span><span class="source" tagx="source" title="source">Int. J. Antimicrob. Agents</span><span class="volume" tagx="volume" title="volume">36</span><span class="year" tagx="year" title="year">2010</span><span class="fpage" tagx="fpage" title="fpage">S21</span><span class="lpage" tagx="lpage" title="lpage">S25</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20801001">20801001</a></span></span></li> <li tag="ref"><a name="bib2" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antia</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Regoes</span><span class="given-names" tagx="given-names" title="given-names">R.R.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koella</span><span class="given-names" tagx="given-names" title="given-names">J.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bergstrom</span><span class="given-names" tagx="given-names" title="given-names">C.T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The role of evolution in the emergence of infectious diseases</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">426</span><span class="year" tagx="year" title="year">2003</span><span class="fpage" tagx="fpage" title="fpage">658</span><span class="lpage" tagx="lpage" title="lpage">661</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature02104">10.1038/nature02104</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14668863">14668863</a></span></span></li> <li tag="ref"><a name="bib3" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bebber</span><span class="given-names" tagx="given-names" title="given-names">D.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Marriot</span><span class="given-names" tagx="given-names" title="given-names">F.H.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gaston</span><span class="given-names" tagx="given-names" title="given-names">K.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Harris</span><span class="given-names" tagx="given-names" title="given-names">S.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scotland</span><span class="given-names" tagx="given-names" title="given-names">R.W.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Predicting unknown species numbers using discovery curves</span><span class="source" tagx="source" title="source">Proc. R. Soc. B</span><span class="volume" tagx="volume" title="volume">274</span><span class="year" tagx="year" title="year">2007</span><span class="fpage" tagx="fpage" title="fpage">1651</span><span class="lpage" tagx="lpage" title="lpage">1658</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rspb.2007.0464">10.1098/rspb.2007.0464</a></span></span></li> <li tag="ref"><a name="bib4" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Briese</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Genetic detection and characterization of Lujo virus, a new hemorrhagic fever-associated arenavirus from Southern Africa</span><span class="source" tagx="source" title="source">PLoS Pathog.</span><span class="volume" tagx="volume" title="volume">5</span><span class="year" tagx="year" title="year">2009</span><span class="fpage" tagx="fpage" title="fpage">e1000455</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1000455">10.1371/journal.ppat.1000455</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19478873">19478873</a></span></span></li> <li tag="ref"><a name="bib5" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chua</span><span class="given-names" tagx="given-names" title="given-names">K.B.</span></span></span><span class="mixed-article-title" title="mixed-article-title">A previously unknown reovirus of bat origin is associated with an acute respiratory disease in humans</span><span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U.S.A.</span><span class="volume" tagx="volume" title="volume">104</span><span class="year" tagx="year" title="year">2007</span><span class="fpage" tagx="fpage" title="fpage">11 424</span><span class="lpage" tagx="lpage" title="lpage">11 429</span><span class="pub-id"><a href="https://dx.doi.org/10.1073/pnas.0701372104">10.1073/pnas.0701372104</a></span></span></li> <li tag="ref"><a name="bib6" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cleaveland</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Laurenson</span><span class="given-names" tagx="given-names" title="given-names">M.K.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taylor</span><span class="given-names" tagx="given-names" title="given-names">L.H.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Diseases of humans and their domestic mammals: pathogen characteristics, host range and the risk of emergence</span><span class="source" tagx="source" title="source">Philos. Trans. R. Soc. Lond. B</span><span class="volume" tagx="volume" title="volume">356</span><span class="year" tagx="year" title="year">2001</span><span class="fpage" tagx="fpage" title="fpage">991</span><span class="lpage" tagx="lpage" title="lpage">999</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rstb.2001.0889">10.1098/rstb.2001.0889</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11516377">11516377</a></span></span></li> <li tag="ref"><a name="bib7" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Cleaveland</span><span class="given-names" tagx="given-names" title="given-names">S.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haydon</span><span class="given-names" tagx="given-names" title="given-names">D.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taylor</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Overviews of pathogen emergence: which pathogens emerge, when and why?</span><span class="source" tagx="source" title="source">Curr. Top. Microbiol. Immunol.</span><span class="volume" tagx="volume" title="volume">315</span><span class="year" tagx="year" title="year">2007</span><span class="fpage" tagx="fpage" title="fpage">85</span><span class="lpage" tagx="lpage" title="lpage">111</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17848062">17848062</a></span></span></li> <li tag="ref"><a name="bib8" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Corman</span><span class="given-names" tagx="given-names" title="given-names">V.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Eckerle</span><span class="given-names" tagx="given-names" title="given-names">I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Bleicker</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Detection of a novel human coronavirus by real-time reverse-transcription polymerase chain reaction</span><span class="source" tagx="source" title="source">Euro Surveill.</span><span class="volume" tagx="volume" title="volume">17</span><span class="issue" tagx="issue" title="issue">39</span><span class="year" tagx="year" title="year">2012</span><span class="comment">pii: 20285</span></span></li> <li tag="ref"><a name="bib9" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gaynor</span><span class="given-names" tagx="given-names" title="given-names">A.M.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of a novel polyomavirus from patients with acute respiratory tract infections</span><span class="source" tagx="source" title="source">PLoS Pathog.</span><span class="volume" tagx="volume" title="volume">3</span><span class="year" tagx="year" title="year">2007</span><span class="fpage" tagx="fpage" title="fpage">e64</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.0030064">10.1371/journal.ppat.0030064</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17480120">17480120</a></span></span></li> <li tag="ref"><a name="bib10" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Greninger</span><span class="given-names" tagx="given-names" title="given-names">A.I.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Runckel</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Chiu</span><span class="given-names" tagx="given-names" title="given-names">C.Y.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haggerty</span><span class="given-names" tagx="given-names" title="given-names">T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Parsonnet</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Ganem</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">The complete genome of klassevirusâ€"a novel picornavirus in pediatric stool</span><span class="source" tagx="source" title="source">Virol. J.</span><span class="volume" tagx="volume" title="volume">6</span><span class="year" tagx="year" title="year">2009</span><span class="fpage" tagx="fpage" title="fpage">82</span><span class="pub-id"><a href="https://dx.doi.org/10.1186/1743-422X-6-82">10.1186/1743-422X-6-82</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19538752">19538752</a></span></span></li> <li tag="ref"><a name="bib11" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Jones</span><span class="given-names" tagx="given-names" title="given-names">K.E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Patel</span><span class="given-names" tagx="given-names" title="given-names">N.G.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Levy</span><span class="given-names" tagx="given-names" title="given-names">M.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Storeygard</span><span class="given-names" tagx="given-names" title="given-names">A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Balk</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gittleman</span><span class="given-names" tagx="given-names" title="given-names">J.L.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Global trends in emerging infectious diseases</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">451</span><span class="year" tagx="year" title="year">2008</span><span class="fpage" tagx="fpage" title="fpage">990</span><span class="lpage" tagx="lpage" title="lpage">994</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature06536">10.1038/nature06536</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18288193">18288193</a></span></span></li> <li tag="ref"><a name="bib12" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keeling</span><span class="given-names" tagx="given-names" title="given-names">M.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Rohani</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="source" tagx="source" title="source">Modelling Infectious Diseases in Humans and Animals</span><span class="year" tagx="year" title="year">2008</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Princeton University Press</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Princeton, NJ</span></span></li> <li tag="ref"><a name="bib13" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Keusch</span><span class="given-names" tagx="given-names" title="given-names">G.T.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Pappaioanou</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Gonzalez</span><span class="given-names" tagx="given-names" title="given-names">M.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Scott</span><span class="given-names" tagx="given-names" title="given-names">K.A.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Tsai</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="source" tagx="source" title="source">Sustaining Global Surveillance and Response to Emerging Zoonotic Diseases</span><span class="year" tagx="year" title="year">2009</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">The National Academies Press</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Washington, DC</span></span></li> <li tag="ref"><a name="bib14" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Kilpatrick</span><span class="given-names" tagx="given-names" title="given-names">A.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Randolph</span><span class="given-names" tagx="given-names" title="given-names">S.E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Drivers, dynamics, and control of emerging vectorborne zoonotic diseases</span><span class="source" tagx="source" title="source">Lancet</span><span class="volume" tagx="volume" title="volume">380</span><span class="year" tagx="year" title="year">2012</span><span class="fpage" tagx="fpage" title="fpage">1946</span><span class="lpage" tagx="lpage" title="lpage">1955</span><span class="comment">17</span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23200503">23200503</a></span></span></li> <li tag="ref"><a name="bib15" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Li</span><span class="given-names" tagx="given-names" title="given-names">L.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Barry</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Yeh</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Glaser</span><span class="given-names" tagx="given-names" title="given-names">C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Schnurr</span><span class="given-names" tagx="given-names" title="given-names">D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Delwart</span><span class="given-names" tagx="given-names" title="given-names">E.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Identification of a novel human gammapapillomavirus species</span><span class="source" tagx="source" title="source">J. Gen. Virol.</span><span class="volume" tagx="volume" title="volume">90</span><span class="year" tagx="year" title="year">2009</span><span class="fpage" tagx="fpage" title="fpage">2413</span><span class="lpage" tagx="lpage" title="lpage">2417</span><span class="pub-id"><a href="https://dx.doi.org/10.1099/vir.0.012344-0">10.1099/vir.0.012344-0</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19570953">19570953</a></span></span></li> <li tag="ref"><a name="bib16" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Simmonds</span><span class="given-names" tagx="given-names" title="given-names">P.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Reconstructing the origins of human hepatitis viruses</span><span class="source" tagx="source" title="source">Philos. Trans. R. Soc. Lond. B</span><span class="volume" tagx="volume" title="volume">356</span><span class="year" tagx="year" title="year">2001</span><span class="fpage" tagx="fpage" title="fpage">1013</span><span class="lpage" tagx="lpage" title="lpage">1026</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rstb.2001.0890">10.1098/rstb.2001.0890</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11516379">11516379</a></span></span></li> <li tag="ref"><a name="bib17" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taylor</span><span class="given-names" tagx="given-names" title="given-names">L.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Latham</span><span class="given-names" tagx="given-names" title="given-names">S.M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woolhouse</span><span class="given-names" tagx="given-names" title="given-names">M.E.J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Risk factors for human disease emergence</span><span class="source" tagx="source" title="source">Philos. Trans. R. Soc. Lond. B</span><span class="volume" tagx="volume" title="volume">356</span><span class="year" tagx="year" title="year">2001</span><span class="fpage" tagx="fpage" title="fpage">983</span><span class="lpage" tagx="lpage" title="lpage">989</span><span class="pub-id"><a href="https://dx.doi.org/10.1098/rstb.2001.0888">10.1098/rstb.2001.0888</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11516376">11516376</a></span></span></li> <li tag="ref"><a name="bib18" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Towner</span><span class="given-names" tagx="given-names" title="given-names">J.S.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Newly discovered Ebola virus associated with hemorrhagic fever outbreak in Uganda</span><span class="source" tagx="source" title="source">PLoS Pathog.</span><span class="volume" tagx="volume" title="volume">4</span><span class="year" tagx="year" title="year">2008</span><span class="fpage" tagx="fpage" title="fpage">e1000212</span><span class="pub-id"><a href="https://dx.doi.org/10.1371/journal.ppat.1000212">10.1371/journal.ppat.1000212</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19023410">19023410</a></span></span></li> <li tag="ref"><a name="bib19" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wolfe</span><span class="given-names" tagx="given-names" title="given-names">N.D.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Naturally acquired simian retrovirus infections in central African hunters</span><span class="source" tagx="source" title="source">Lancet</span><span class="volume" tagx="volume" title="volume">363</span><span class="year" tagx="year" title="year">2004</span><span class="fpage" tagx="fpage" title="fpage">932</span><span class="lpage" tagx="lpage" title="lpage">937</span><span class="pub-id"><a href="https://dx.doi.org/10.1016/S0140-6736(04)15787-5">10.1016/S0140-6736(04)15787-5</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15043960">15043960</a></span></span></li> <li tag="ref"><a name="bib20" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Wolfe</span><span class="given-names" tagx="given-names" title="given-names">N.D.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Dunavan</span><span class="given-names" tagx="given-names" title="given-names">C.P.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Diamond</span><span class="given-names" tagx="given-names" title="given-names">J.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Origins of major human infectious diseases</span><span class="source" tagx="source" title="source">Nature</span><span class="volume" tagx="volume" title="volume">447</span><span class="year" tagx="year" title="year">2007</span><span class="fpage" tagx="fpage" title="fpage">279</span><span class="lpage" tagx="lpage" title="lpage">283</span><span class="pub-id"><a href="https://dx.doi.org/10.1038/nature05775">10.1038/nature05775</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17507975">17507975</a></span></span></li> <li tag="ref"><a name="bib21" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woolhouse</span><span class="given-names" tagx="given-names" title="given-names">M.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Antia</span><span class="given-names" tagx="given-names" title="given-names">R.</span></span></span><span class="chapter-title" tagx="chapter-title" title="chapter-title">Emergence of new infectious diseases</span><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Stearns</span><span class="given-names" tagx="given-names" title="given-names">S.C.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Koella</span><span class="given-names" tagx="given-names" title="given-names">J.K.</span></span></span><span class="source" tagx="source" title="source">Evolution in Health and Disease</span><span class="edition" tagx="edition" title="edition">second ed.</span><span class="year" tagx="year" title="year">2008</span><span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford University Press</span><span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Oxford, UK</span><span class="fpage" tagx="fpage" title="fpage">215</span><span class="lpage" tagx="lpage" title="lpage">228</span></span></li> <li tag="ref"><a name="bib22" /><span class="element-citation'"><span class="person-group'"><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Woolhouse</span><span class="given-names" tagx="given-names" title="given-names">M.E.J.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Taylor</span><span class="given-names" tagx="given-names" title="given-names">L.H.</span></span><span class="name" tagx="name" title="name"><span class="surname" tagx="surname" title="surname">Haydon</span><span class="given-names" tagx="given-names" title="given-names">D.T.</span></span></span><span class="mixed-article-title" title="mixed-article-title">Population biology of multi-host pathogens</span><span class="source" tagx="source" title="source">Science</span><span class="volume" tagx="volume" title="volume">292</span><span class="year" tagx="year" title="year">2001</span><span class="fpage" tagx="fpage" title="fpage">1109</span><span class="lpage" tagx="lpage" title="lpage">1112</span><span class="pub-id"><a href="https://dx.doi.org/10.1126/science.1059026">10.1126/science.1059026</a></span><span class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11352066">11352066</a></span></span></li> </ul> </div> </div>  </body></html>